## The microbiome and cytosolic innate immune receptors.
 13.07.2020, _T Liwinski, D Zheng, E Elinav_


The discovery of innate immune sensors (pattern recognition receptors, PRRs) has profoundly transformed the notion of innate immunity, in providing a mechanistic basis for host immune interactions with a wealth of environmental signals, leading to a variety of immune-mediated outcomes including instruction and activation of the adaptive immune arm. As part of this growing understanding of host-environmental cross talk, an intimate connection has been unveiled between innate immune sensors and signals perceived from the commensal microbiota, which may be regarded as a hub integrating a variety of environmental cues. Among cytosolic PRRs impacting on host homeostasis by interacting with the commensal microbiota are nucleotide-binding domain, leucine-rich repeat-containing protein receptors (NLRs), together with a number of cytosolic DNA sensors and the family of absent in melanoma (AIM)-like receptors (ALRs). NLR sensors have been a particular focus of research, and some NLRs have emerged as key orchestrators of inflammatory responses and host homeostasis. Some NLRs achieve this through the formation of cytoplasmic multiprotein complexes termed inflammasomes. More recently discovered PRRs include retinoic acid-inducible gene-I (RIG-I)-like receptors (RLRs), cyclic GMP-AMP synthase (cGAS), and STING. In the present review, they summarize recent advancements in knowledge on structure and function of cytosolic PRRs and their roles in host-microbiota cross talk and immune surveillance. In addition, we discuss their relevance for human health and disease and future therapeutic applications involving modulation of their activation and signaling.

[Source](https://www.nature.com/articles/s41579-020-0367-2)

10.1111/imr.12901

---

## Host–microbiota interactions in immune-mediated diseases

[Source](https://www.nature.com/articles/s41579-020-0367-2)

---

## Diversity within species: interpreting strains in microbiomes.
 12.08.2020, _T Van Rossum, P Ferretti, OM Maistrenko, P Bork_


Studying within-species variation has traditionally been limited to culturable bacterial isolates and low-resolution microbial community fingerprinting. Metagenomic sequencing and technical advances have enabled culture-free, high-resolution strain and subspecies analyses at high throughput and in complex environments. This holds great scientific promise but has also led to an overwhelming number of methods and terms to describe infraspecific variation. This Review aims to clarify these advances by focusing on the diversity within bacterial and archaeal species in the context of microbiomics. We cover foundational microevolutionary concepts relevant to population genetics and summarize how within-species variation can be studied and stratified directly within microbial communities with a focus on metagenomics. Finally, we describe how common applications of within-species variation can be achieved using metagenomic data. We aim to guide the selection of appropriate terms and analytical approaches to facilitate researchers in benefiting from the increasing availability of large, high-resolution microbiome genetic sequencing data.

[Source](https://www.nature.com/articles/s41579-020-0368-1)

10.1038/s41579-020-0368-1

---

## Vast Differences in Strain-Level Diversity in the Gut Microbiota of Two Closely Related Honey Bee Species.
 25.07.2020, _KM Ellegaard, S Suenami, R Miyazaki, P Engel_


Most bacterial species encompass strains with vastly different gene content. Strain diversity in microbial communities is therefore considered to be of functional importance. Yet little is known about the extent to which related microbial communities differ in diversity at this level and which underlying mechanisms may constrain and maintain strain-level diversity. Here, we used shotgun metagenomics to characterize and compare the gut microbiota of two honey bee species, Apis mellifera and Apis cerana, which diverged about 6 mya. Although the host species are colonized largely by the same bacterial 16S rRNA phylotypes, we find that their communities are host specific when analyzed with genomic resolution. Moreover, despite their similar ecology, A. mellifera displayed a much higher diversity of strains and functional gene content in the microbiota compared to A. cerana, both per colony and per individual bee. In particular, the gene repertoire for polysaccharide degradation was massively expanded in the microbiota of A. mellifera relative to A. cerana. Bee management practices, divergent ecological adaptation, or habitat size may have contributed to the observed differences in microbiota genomic diversity of these key pollinator species. Our results illustrate that the gut microbiota of closely related animal hosts can differ vastly in genomic diversity while displaying similar levels of diversity based on the 16S rRNA gene. Such differences are likely to have consequences for gut microbiota functioning and host-symbiont interactions, highlighting the need for metagenomic studies to understand the ecology and evolution of microbial communities.

[Source](https://www.sciencedirect.com/science/article/pii/S0960982220305868)

10.1016/j.cub.2020.04.070

---

## Oral and vaginal microbiota in selected field mice of the genus Apodemus: a wild population study.
 11.08.2020, _T Matějková, P Hájková, R Stopková, M Stanko, JF Martin, J Kreisinger, P Stopka_


Animal-associated microbiota is expected to impose crucial effects on the host's fitness-related performance, including reproduction. Most research to date has focused on interactions between the host with its gut microbiota; however, there remain considerable gaps in knowledge regarding microbial consortia in other organs, including interspecific divergence, temporal stability, variation drivers, and their effects on the host. To fill these gaps, we examined oral and vaginal microbiota composition in four free-living mouse species of the genus Apodemus, each varying in the degree of female promiscuity. To assess temporal stability and microbiota resistance to environmental change, we exposed one of the species, Apodemus uralensis, to standardized captive conditions and analyzed longitudinal changes in its microbiota structure. Our results revealed the existence of a "core" oral microbiota that was not only shared among all four species but also persisted almost unchanged in captivity. On the other hand, vaginal microbiota appears to be more plastic in captive conditions and less species-specific in comparison with oral microbiota. This study is amongst the first to describe oral microbiota dynamics. Furthermore, the vaginal microbiota results are especially surprising in light of the well-known role of stable vaginal microbiota as a defense against pathogens. The results indicate the existence of diverse mechanisms that shape each microbiota. On the other hand, our data provides somewhat ambiguous support for the systematic effect of phylogeny and social system on both oral and vaginal microbiota structures.

[Source](https://www.nature.com/articles/s41598-020-70249-x)

10.1038/s41598-020-70249-x

---

## Characterization of Cutaneous Bacterial Microbiota from Superficial Pyoderma Forms in Atopic Dogs

[Source](https://www.mdpi.com/2076-0817/9/8/638/htm)

---

## Cryptic fungal diversity revealed in deep- sea sediments associated with whale-fall chemosynthetic ecosystems

[Source](https://www.tandfonline.com/doi/full/10.1080/21501203.2020.1799879)

---

## Feature selection and causal analysis for microbiome studies in the presence of confounding using standardization

[Source](https://www.biorxiv.org/content/10.1101/2020.08.09.243188v1)

---

## A Distance-Based Framework for the Characterization of Metabolic Heterogeneity in Large Sets of Genome-Scale Metabolic Models

[Source](https://www.sciencedirect.com/science/article/pii/S2666389920301082)

---

## FORENSIC: an Online Platform for Fecal Source Identification

[Source](an Online Platform for Fecal Source Identification)

---

## Bacterial mock communities as standards for reproducible cytometric microbiome analysis

[Source](https://www.nature.com/articles/s41596-020-0362-0)

---

