## Association between gut microbiota and influenza: a bidirectional two-sample mendelian randomization study.
 18.10.2023, _F Xu, X Gan, Y Tao, D Li, P Xie, F Liu, F Yang, Y Ma_


Previous observational studies have indicated a correlation between the gut microbiota and influenza; however, the exact nature of the bidirectional causal connection remains uncertain.

[Source](https://bmcinfectdis.biomedcentral.com/articles/10.1186/s12879-023-08706-x)

10.1186/s12879-023-08706-x

---

## LACC1 regulates changes in the intestinal flora in a mouse model of inflammatory bowel disease

[Source](https://bmcgastroenterol.biomedcentral.com/articles/10.1186/s12876-023-02971-5)

---

## Sex-specific gut bacteria manipulation impacts offspring production in the honeybee parasite small hive beetle, Aethina tumida Murray

[Source](https://www.biorxiv.org/content/10.1101/2023.10.15.562436v1.abstract)

---

## Selective enrichment of founding reproductive microbiomes allows extensive vertical transmission in a fungus-farming termite

[Source](https://royalsocietypublishing.org/doi/10.1098/rspb.2023.1559)

---

## Microbiome variations among age classes and diets of captive Asian elephants (Elephas maximus) in Thailand using full-length 16S rRNA nanopore sequencing.
 18.10.2023, _W Klinsawat, P Uthaipaisanwong, P Jenjaroenpun, S Sripiboon, T Wongsurawat, K Kusonmano_


Asian elephant (Elephas maximus) is the national symbol of Thailand and linked to Thai history and culture for centuries. The elephant welfare improvement is one of the major components to achieve sustainable captive management. Microbiome inhabiting digestive tracts have been shown with symbiotic relations to host health. This work provided high-resolution microbiome profiles of 32 captive elephants at a species level by utilizing full-length 16S rRNA gene nanopore sequencing. Eleven common uncultured bacterial species were found across elephants fed with solid food including uncultured bacterium Rikenellaceae RC9 gut group, Kiritimatiellae WCHB1-41, Phascolarctobacterium, Oscillospiraceae NK4A214 group, Christensenellaceae R-7 group, Oribacterium, Oscillospirales UCG-010, Lachnospiraceae, Bacteroidales F082, uncultured rumen Rikenellaceae RC9 gut group, and Lachnospiraceae AC2044 group. We observed microbiome shifts along the age classes of baby (0-2 years), juvenile (2-10 years), and adult (&gt; 10 years). Interestingly, we found distinct microbiome profiles among adult elephants fed with a local palm, Caryota urens, as a supplement. Potential beneficial microbes have been revealed according to the age classes and feed diets. The retrieved microbiome data could be provided as good baseline microbial profiles for monitoring elephant health, suggesting further studies towards dietary selection suitable for each age class and the use of local supplementary diets.

[Source](https://www.nature.com/articles/s41598-023-44981-z)

10.1038/s41598-023-44981-z

---

## Temporal and spatial variations in body mass and thermogenic capacity associated with alterations in the gut microbiota and host transcriptome in mammalian herbivores.
 17.10.2023, _S Ren, L Zhang, X Tang, Y Zhao, Q Cheng, JR Speakman, Y Zhang_


Most wild animals follow Bergmann's rule and grow in body size as cold stress increases. However, the underlying thermogenic strategies and their relationship with the gut microbiota have not been comprehensively elucidated. Herein, we used the plateau pikas as a model to investigate body mass, thermogenic capacity, host transcriptome, gut microbiota and metabolites collected from seven sites ranging from 3100 to 4700 m on the Qinghai-Tibetan Plateau (QTP) in summer and winter to test the seasonal thermogenesis strategy in small herbivorous mammals. The results showed that the increase in pika body mass with altitude followed Bergmann's rule in summer and an inverted parabolic shape was observed in winter. However, physiological parameters and transcriptome profiles indicated that the thermogenic capacity of pikas increased with altitude in summer and decreased with altitude in winter. The abundance of Firmicutes declined, whereas that of Bacteroidetes significantly increased with altitude in summer. Phenylalanine, tyrosine, and proline were enriched in summer, whereas carnitine and succinate were enriched in winter. Spearman's correlation analysis revealed significant positive correlations between Prevotella, Bacteroides, Ruminococcus, Alistipes and Akkermansia and metabolites of amino acids, pika physiological parameters, and transcriptome profiles. Moreover, metabolites of amino acids further showed significant positive correlations with pika physiological parameters and transcriptome profiles. Our study highlights that the changes in body mass and thermogenic capacity with altitude distinctly differentiate small herbivorous mammals between summer and winter on the QTP, and that the gut microbiota may regulate host thermogenesis through its metabolites.

[Source](https://www.sciencedirect.com/science/article/pii/S0048969723064033)

10.1016/j.scitotenv.2023.167776

---

## Metagenome sequencing and recovery of 444 metagenome-assembled genomes from the biofloc aquaculture system.
 18.10.2023, _M Rajeev, I Jung, Y Lim, S Kim, I Kang, JC Cho_


Biofloc technology is increasingly recognised as a sustainable aquaculture method. In this technique, bioflocs are generated as microbial aggregates that play pivotal roles in assimilating toxic nitrogenous substances, thereby ensuring high water quality. Despite the crucial roles of the floc-associated bacterial (FAB) community in pathogen control and animal health, earlier microbiota studies have primarily relied on the metataxonomic approaches. Here, we employed shotgun sequencing on eight biofloc metagenomes from a commercial aquaculture system. This resulted in the generation of 106.6 Gbp, and the reconstruction of 444 metagenome-assembled genomes (MAGs). Among the recovered MAGs, 230 were high-quality (≥90% completeness, ≤5% contamination), and 214 were medium-quality (≥50% completeness, ≤10% contamination). Phylogenetic analysis unveiled Rhodobacteraceae as dominant members of the FAB community. The reported metagenomes and MAGs are crucial for elucidating the roles of diverse microorganisms and their functional genes in key processes such as nitrification, denitrification, and remineralization. This study will contribute to scientific understanding of phylogenetic diversity and metabolic capabilities of microbial taxa in aquaculture environments.

[Source](https://www.nature.com/articles/s41597-023-02622-0)

10.1038/s41597-023-02622-0

---

## Net-spinning caddisflies create denitrifier-enriched niches in the stream microbiome.
 18.10.2023, _AD Bertagnolli, AJ Maritan, BB Tumolo, SF Fritz, HC Oakland, EJ Mohr, GC Poole, LK Albertson, FJ Stewart_


Larval net-spinning caddisflies (Hydropsychidae) function as ecosystem engineers in streams where they construct protective retreats composed of organic and inorganic material affixed with silk filtration nets that alter streambed hydrology. We hypothesized that hydropsychid bio-structures (retreats, nets) are microhabitats for microbes with oxygen-sensitive metabolisms, and therefore increase the metabolic heterogeneity of streambed microbial assemblages. Metagenomic and 16 S rRNA gene amplicon analysis of samples from a montane stream (Cherry Creek, Montana, USA) revealed that microbiomes of caddisfly bio-structures are taxonomically and functionally distinct from those of the immediately adjacent rock biofilm (~2 cm distant) and enriched in microbial taxa with established roles in denitrification, nitrification, and methane production. Genes for denitrification, high oxygen affinity terminal oxidases, hydrogenases, oxidative dissimilatory sulfite reductases, and complete ammonia oxidation are significantly enriched in caddisfly bio-structures. The results suggest a novel ecosystem engineering effect of caddisflies through the creation of low-oxygen, denitrifier-enriched niches in the stream microbiome. Facilitation of metabolic diversity in streambeds may be a largely unrecognized mechanism by which caddisflies alter whole-stream biogeochemistry.

[Source](https://www.nature.com/articles/s43705-023-00315-8)

10.1038/s43705-023-00315-8

---

## Effects of heat tolerance on the gut microbiota of Sarcophaga peregrina (Diptera: Sarcophagidae) and impacts on the life history traits.
 18.10.2023, _L Ren, X Zhang, F Yang, NF Jocelin, Y Shang, Q Wang, Z Liu, Y Guo_


Heat tolerance is a distinct abiotic factor affecting the distribution and abundance of insects. Gut microbiota can contribute to host fitness, thereby increasing resistance to abiotic stress conditions. In this study, Sarcophaga peregrina is closely associated with human life in ecological habits and shows remarkable adaptability to daily and seasonal temperature fluctuations. To date, the role of gut microbiota in S. peregrina response to heat stress and its influence on the host phenotypic variability remain poorly studied.

[Source](https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-023-05973-0)

10.1186/s13071-023-05973-0

---

## Comprehensive Identification of Pathogenic Microbes and Antimicrobial Resistance Genes in Food Products Using Nanopore Sequencing-Based Metagenomics

[Source](https://www.biorxiv.org/content/10.1101/2023.10.15.562131v1.abstract)

---

## Causal inference on microbiome-metabolome relations in observational host-microbiome data via in silico in vivo association pattern analyses.
 17.10.2023, _J Hertel, A Heinken, D Fässler, I Thiele_


Understanding the effects of the microbiome on the host's metabolism is core to enlightening the role of the microbiome in health and disease. Herein, we develop the paradigm of in silico in vivo association pattern analyses, combining microbiome metabolome association studies with in silico constraint-based community modeling. Via theoretical dissection of confounding and causal paths, we show that in silico in vivo association pattern analyses allow for causal inference on microbiome-metabolome relations in observational data. We justify the corresponding theoretical criterion by structural equation modeling of host-microbiome systems, integrating deterministic microbiome community modeling into population statistics approaches. We show the feasibility of our approach on a published multi-omics dataset (n = 347), demonstrating causal microbiome-metabolite relations for 26 out of 54 fecal metabolites. In summary, we generate a promising approach for causal inference in metabolic host-microbiome interactions by integrating hypothesis-free screening association studies with knowledge-based in silico modeling.

[Source](https://www.sciencedirect.com/science/article/pii/S266723752300276X)

10.1016/j.crmeth.2023.100615

---

## Rank selection for non-negative matrix factorization.
 17.10.2023, _Y Cai, H Gu, T Kenney_


Non-Negative Matrix Factorization (NMF) is a widely used dimension reduction method that factorizes a non-negative data matrix into two lower dimensional non-negative matrices: one is the basis or feature matrix which consists of the variables and the other is the coefficients matrix which is the projections of data points to the new basis. The features can be interpreted as sub-structures of the data. The number of sub-structures in the feature matrix is also called the rank. This parameter controls the model complexity and is the only tuning parameter for the NMF model. An appropriate rank will extract the key latent features while minimizing the noise from the original data. However due to the large amount of optimization error always present in the NMF computation, the rank selection has been a difficult problem. We develop a novel rank selection method based on hypothesis testing, using a deconvolved bootstrap distribution to assess the significance level accurately. Through simulations, we compare our method with a rank selection method based on hypothesis testing using bootstrap distribution without deconvolution and a method based on cross-validation; we demonstrate that our method is not only accurate at estimating the true ranks for NMF, especially when the features are hard to distinguish, but also efficient at computation. When applied to real microbiome data (eg, OTU data and functional metagenomic data), our method also shows the ability to extract interpretable subcommunities in the data.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/sim.9934)

10.1002/sim.9934

---

