## An Open Resource for Non-human Primate Optogenetics.
 20.10.2020, _S Tremblay, L Acker, A Afraz, DL Albaugh, H Amita, AR Andrei, A Angelucci, A Aschner, PF Balan, MA Basso, G Benvenuti, MO Bohlen, MJ Caiola, R Calcedo, J Cavanaugh, Y Chen, S Chen, MM Chernov, AM Clark, J Dai, SR Debes, K Deisseroth, R Desimone, V Dragoi, SW Egger, MAG Eldridge, HG El-Nahal, F Fabbrini, F Federer, CR Fetsch, MG Fortuna, RM Friedman, N Fujii, A Gail, A Galvan, S Ghosh, MA Gieselmann, RA Gulli, O Hikosaka, EA Hosseini, X Hu, J Hüer, KI Inoue, R Janz, M Jazayeri, R Jiang, N Ju, K Kar, C Klein, A Kohn, M Komatsu, K Maeda, JC Martinez-Trujillo, M Matsumoto, JHR Maunsell, D Mendoza-Halliday, IE Monosov, RS Muers, L Nurminen, M Ortiz-Rios, DJ O'Shea, S Palfi, CI Petkov, S Pojoga, R Rajalingham, C Ramakrishnan, ED Remington, C Revsine, AW Roe, PN Sabes, RC Saunders, H Scherberger, MC Schmid, W Schultz, E Seidemann, YS Senova, MN Shadlen, DL Sheinberg, C Siu, Y Smith, SS Solomon, MA Sommer, JL Spudich, WR Stauffer, M Takada, S Tang, A Thiele, S Treue, W Vanduffel, R Vogels, MP Whitmire, T Wichmann, RH Wurtz, H Xu, A Yazdan-Shahmorad, KV Shenoy, JJ DiCarlo, ML Platt_


Optogenetics has revolutionized neuroscience in small laboratory animals, but its effect on animal models more closely related to humans, such as non-human primates (NHPs), has been mixed. To make evidence-based decisions in primate optogenetics, the scientific community would benefit from a centralized database listing all attempts, successful and unsuccessful, of using optogenetics in the primate brain. We contacted members of the community to ask for their contributions to an open science initiative. As of this writing, 45 laboratories around the world contributed more than 1,000 injection experiments, including precise details regarding their methods and outcomes. Of those entries, more than half had not been published. The resource is free for everyone to consult and contribute to on the Open Science Framework website. Here we review some of the insights from this initial release of the database and discuss methodological considerations to improve the success of optogenetic experiments in NHPs.

[Source](https://bit.ly/359vU4A)

10.1016/j.neuron.2020.09.027

---

## website

[Source](https://bit.ly/359vU4A)

---

## The Reactive Species Interactome: Evolutionary Emergence, Biological Significance, and Opportunities for Redox Metabolomics and Personalized Medicine.
 13.11.2018, _MM Cortese-Krott, A Koning, GGC Kuhnle, P Nagy, CL Bianco, A Pasch, DA Wink, JM Fukuto, AA Jackson, H van Goor, KR Olson, M Feelisch_


_Adaptation, Physiological, Animals, Humans, Hydrogen Sulfide, Metabolomics, Oxidation-Reduction, Precision Medicine, Reactive Nitrogen Species, Reactive Oxygen Species, Sulfides_
Oxidative stress is thought to account for aberrant redox homeostasis and contribute to aging and disease. However, more often than not, administration of antioxidants is ineffective, suggesting that our current understanding of the underlying regulatory processes is incomplete. Recent Advances: Similar to reactive oxygen species and reactive nitrogen species, reactive sulfur species are now emerging as important signaling molecules, targeting regulatory cysteine redox switches in proteins, affecting gene regulation, ion transport, intermediary metabolism, and mitochondrial function. To rationalize the complexity of chemical interactions of reactive species with themselves and their targets and help define their role in systemic metabolic control, we here introduce a novel integrative concept defined as the reactive species interactome (RSI). The RSI is a primeval multilevel redox regulatory system whose architecture, together with the physicochemical characteristics of its constituents, allows efficient sensing and rapid adaptation to environmental changes and various other stressors to enhance fitness and resilience at the local and whole-organism level.

[Source](http://www.aginganddisease.org/article/0000/2152-5250/ad-0-0-0-2009220912-1.shtml)

10.1089/ars.2017.7083

---

## Metabolomics Signatures of Aging: Recent Advances

[Source](http://www.aginganddisease.org/article/0000/2152-5250/ad-0-0-0-2009220912-1.shtml)

---

## The human microbiome in the 21st century.
 20.10.2020, _E Rackaityte, SV Lynch_



[Source](https://www.nature.com/articles/s41467-020-18983-8)

10.1038/s41467-020-18983-8

---

## The intestinal microbiome, weight, and metabolic changes in women treated by adjuvant chemotherapy for breast and gynecological malignancies

[Source](https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-020-01751-2)

---

## The Role of the Gut Microbiota in Coronary Heart Disease.
 16.10.2020, _H Liu, J Zhuang, P Tang, J Li, X Xiong, H Deng_


This review focuses on recent evidence examining the role gut microbiota play in coronary heart disease. It also provides a succinct overview of current and future therapies targeting the gut microbiota for coronary heart disease risk reduction.

[Source](https://link.springer.com/article/10.1007/s11883-020-00892-2)

10.1007/s11883-020-00892-2

---

## Gut microbial molecules in behavioural and neurodegenerative conditions.
 17.10.2020, _BD Needham, R Kaddurah-Daouk, SK Mazmanian_


Mounting evidence suggests that the gut microbiome impacts brain development and function. Gut-brain connections may be mediated by an assortment of microbial molecules that are produced in the gastrointestinal tract, which can subsequently permeate many organs, including sometimes the brain. Studies in animal models have identified molecular cues propagated from intestinal bacteria to the brain that can affect neurological function and/or neurodevelopmental and neurodegenerative conditions. Herein, we describe bacterial metabolites with known or suspected neuromodulatory activity, define mechanisms of signalling pathways from the gut microbiota to the brain and discuss direct effects that gut bacterial molecules are likely exerting on specific brain cells. Many discoveries are recent, and the findings described in this Perspective are largely novel and yet to be extensively validated. However, expanding research into the dynamic molecular communications between gut microorganisms and the CNS continues to uncover critical and previously unappreciated clues in understanding the pathophysiology of behavioural, psychiatric and neurodegenerative diseases.

[Source](https://www.nature.com/articles/s41583-020-00381-0)

10.1038/s41583-020-00381-0

---

## Interactions between host genetics and gut microbiota determine susceptibility to CNS autoimmunity.
 20.10.2020, _TL Montgomery, A Künstner, JJ Kennedy, Q Fang, L Asarian, R Culp-Hill, A D'Alessandro, C Teuscher, H Busch, DN Krementsov_


Multiple sclerosis (MS) is an autoimmune disease of the central nervous system. The etiology of MS is multifactorial, with disease risk determined by genetics and environmental factors. An emerging risk factor for immune-mediated diseases is an imbalance in the gut microbiome. However, the identity of gut microbes associated with disease risk, their mechanisms of action, and the interactions with host genetics remain obscure. To address these questions, we utilized the principal autoimmune model of MS, experimental autoimmune encephalomyelitis (EAE), together with a genetically diverse mouse model representing 29 unique host genotypes, interrogated by microbiome sequencing and targeted microbiome manipulation. We identified specific gut bacteria and their metabolic functions associated with EAE susceptibility, implicating short-chain fatty acid metabolism as a key element conserved across multiple host genotypes. In parallel, we used a reductionist approach focused on two of the most disparate phenotypes identified in our screen. Manipulation of the gut microbiome by transplantation and cohousing demonstrated that transfer of these microbiomes into genetically identical hosts was sufficient to modulate EAE susceptibility and systemic metabolite profiles. Parallel bioinformatic approaches identified Lactobacillus reuteri as a commensal species unexpectedly associated with exacerbation of EAE in a genetically susceptible host, which was functionally confirmed by bacterial isolation and commensal colonization studies. These results reveal complex interactions between host genetics and gut microbiota modulating susceptibility to CNS autoimmunity, providing insights into microbiome-directed strategies aimed at lowering the risk for autoimmune disease and underscoring the need to consider host genetics and baseline gut microbiome composition.

[Source](https://doi.org/10.1073/pnas.2002817117)

10.1073/pnas.2002817117

---

## Bifidobacterium alters the gut microbiota and modulates the functional metabolism of T regulatory cells in the context of immune checkpoint blockade

[Source](https://doi.org/10.1073/pnas.1921223117)

---

## Bugs and drugs: a systems biology approach to characterising the effect of moxidectin on the horse’s faecal microbiome

[Source](https://link.springer.com/article/10.1186/s42523-020-00056-2)

---

## A pioneer calf foetus microbiome

[Source](https://www.nature.com/articles/s41598-020-74677-7)

---

## Global diversity of microbial communities in marine sediment

[Source](https://doi.org/10.1073/pnas.1919139117)

---

## Statistical and Machine-Learning Analyses in Nutritional Genomics Studies

[Source](https://www.mdpi.com/2072-6643/12/10/3140)

---

## A need to consider the evolutionary genetics of host‐symbiont mutualisms

[Source](https://onlinelibrary.wiley.com/doi/abs/10.1111/jeb.13715)

---

## Postmortomics: The Potential of Untargeted Metabolomics to Highlight Markers for Time Since Death

[Source](https://www.liebertpub.com/doi/full/10.1089/omi.2020.0084)

---

