## Human microbiome variation associated with race and ethnicity emerges as early as 3 months of age

[Source](https://doi.org/10.1371/journal.pbio.3002230)

---

## Mapping the T cell repertoire to a complex gut bacterial community

[Source](https://doi.org/10.1038/s41586-023-06431-8)

---

## Comparison of the relative impacts of acute consumption of an inulin-enriched diet, milk kefir or a commercial probiotic product on the human gut microbiome and metabolome

[Source](https://www.nature.com/articles/s41538-023-00216-z)

---

## High-resolution strain-level microbiome composition analysis from short reads

[Source](https://doi.org/10.1186/s40168-023-01615-w)

---

## Profiling novel lateral gene transfer events in the human microbiome

[Source](https://www.biorxiv.org/content/10.1101/2023.08.08.552500v2.abstract)

---

## Major data analysis errors invalidate cancer microbiome findings

[Source](https://www.biorxiv.org/content/10.1101/2023.07.28.550993v1)

---

## Microdiversity of the vaginal microbiome is associated with preterm birth

[Source](https://doi.org/10.1038/s41467-023-40719-7)

---

## Dysbiosis of gut microbiota during fecal stream diversion in patients with colorectal cancer

[Source](https://doi.org/10.1186/s13099-023-00566-9)

---

## The gut microbiome contributes to somatic morphine withdrawal behavior and implicates a TLR2 mediated mechanism

[Source](https://doi.org/10.1080/19490976.2023.2242610)

---

## Association of the human gut microbiota with vascular stiffness

[Source](https://doi.org/10.1038/s41598-023-40178-6)

---

## Gut microbiota and intervertebral disc degeneration: a bidirectional two-sample Mendelian randomization study

[Source](https://doi.org/10.1186/s13018-023-04081-0)

---

## Microbiome alterations in autism spectrum disorder

[Source](https://doi.org/10.1038/s41564-023-01455-2)

---

## Rosmarinic acid alleviates intestinal inflammatory damage and inhibits endoplasmic reticulum stress and smooth muscle contraction abnormalities in intestinal tissues by regulating gut microbiota

[Source](https://doi.org/10.1128/spectrum.01914-23)

---

## <em>Prevotella copri </em>alleviates sarcopenia via attenuating muscle mass loss and function decline

[Source](https://onlinelibrary.wiley.com/doi/full/10.1002/jcsm.13313)

---

## The Influence of Protein Secretomes of <em>Enterococcus durans</em> on ex vivo Human Gut Microbiome

[Source](https://doi.org/10.1007/s12602-023-10136-9)

---

## Kombucha tea as an anti-hyperglycemic agent in humans with diabetes - a randomized controlled pilot investigation.
 18.08.2023, _C Mendelson, S Sparkes, DJ Merenstein, C Christensen, V Sharma, S Desale, JM Auchtung, CR Kok, HE Hallen-Adams, R Hutkins_


Kombucha is a popular fermented tea that has attracted considerable attention due, in part, to its suggested health benefits. Previous results from animal models led us to hypothesize kombucha may reduce blood sugar levels in humans with diabetes. The objective of this pilot clinical study was to evaluate kombucha for its anti-hyperglycemic activities in adults with diabetes mellitus type II.

[Source](https://doi.org/10.3389/fnut.2023.1190248)

10.3389/fnut.2023.1190248

---

## The unique gut microbiome of giant pandas involved in protein metabolism contributes to the host’s dietary adaption to bamboo

[Source](https://doi.org/10.1186/s40168-023-01603-0)

---

## Systematic review of cnidarian microbiomes reveals insights into the structure, specificity, and fidelity of marine associations.
 22.08.2023, _M McCauley, TL Goulet, CR Jackson, S Loesgen_


_Animals, RNA, Ribosomal, 16S, Microbiota, Bacteria, Archaea, Anthozoa, Phylogeny_
Microorganisms play essential roles in the health and resilience of cnidarians. Understanding the factors influencing cnidarian microbiomes requires cross study comparisons, yet the plethora of protocols used hampers dataset integration. We unify 16S rRNA gene sequences from cnidarian microbiome studies under a single analysis pipeline. We reprocess 12,010 cnidarian microbiome samples from 186 studies, alongside 3,388 poriferan, 370 seawater samples, and 245 cultured Symbiodiniaceae, unifying ~6.5 billion sequence reads. Samples are partitioned by hypervariable region and sequencing platform to reduce sequencing variability. This systematic review uncovers an incredible diversity of 86 archaeal and bacterial phyla associated with Cnidaria, and highlights key bacteria hosted across host sub-phylum, depth, and microhabitat. Shallow (&lt; 30 m) water Alcyonacea and Actinaria are characterized by highly shared and relatively abundant microbial communities, unlike Scleractinia and most deeper cnidarians. Utilizing the V4 region, we find that cnidarian microbial composition, richness, diversity, and structure are primarily influenced by host phylogeny, sampling depth, and ocean body, followed by microhabitat and sampling date. We identify host and geographical generalist and specific Endozoicomonas clades within Cnidaria and Porifera. This systematic review forms a framework for understanding factors governing cnidarian microbiomes and creates a baseline for assessing stress associated dysbiosis.

[Source](https://doi.org/10.1038/s41467-023-39876-6)

10.1038/s41467-023-39876-6

---

## Toward an Integrated Understanding of the Lepidoptera Microbiome.
 16.08.2023, _Y Shao, CJ Mason, GW Felton_


Research over the past 30 years has led to a widespread acceptance that insects establish widespread and diverse associations with microorganisms. More recently, microbiome research has been accelerating in lepidopteran systems, leading to a greater understanding of both endosymbiont and gut microorganisms and how they contribute to integral aspects of the host. Lepidoptera are associated with a robust assemblage of microorganisms, some of which may be stable and routinely detected in larval and adult hosts, while others are ephemeral and transient. Certain microorganisms that populate Lepidoptera can contribute significantly to the hosts' performance and fitness, while others are inconsequential. We emphasize the context-dependent nature of the interactions between players. While our review discusses the contemporary literature, there are major avenues yet to be explored to determine both the fundamental aspects of host-microbe interactions and potential applications for the lepidopteran microbiome; we describe these avenues after our synthesis. Expected final online publication date for the Annual Review of Entomology, Volume 69 is January 2024. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.

[Source](https://doi.org/10.1146/annurev-ento-020723-102548)

10.1146/annurev-ento-020723-102548

---

## Characterization of bacterial community in tobacco leaves at flue-curing and redrying processing stages.
 19.08.2023, _Y Yang, R Xu, M Yang, Q Xu, C Miao, J Guo, W Mou, H Du, G Wei, L Hu, Z Hu_


_Humans, Tobacco, Plant Leaves, Proteobacteria, Aging, Cancer Vaccines_
During the processing of tobacco leaves, flue-curing and redrying can affect the structure of bacterial community, having an effect on the aging quality of tobacco leaves. In order to characterize the effects of flue-curing and redrying on the bacterial community of tobacco leaves, the bacterial community of samples at different processing stages (before flue-curing, after flue-curing, before redrying and after redrying) was analyzed using Illumina sequencing. A total of 33 phyla, 79 classes, 195 orders, 344 families, 826 genera and 7922 ASVs were obtained from 36 samples. There was no significant difference in the core bacterial groups of tobacco leaf at four processing stages. Proteobacteria dominated at the phylum level. Sphingomonas, Pseudomonas and Methylobacterium were the main genera shared by all samples. The functional prediction by PICRUSt showed an increase in the relative abundance of pathway related to metabolism after flue-curing and pathway related to environmental information processing after redrying. This study, we analyzed the changes of bacterial community and structural composition of tobacco leaves from flue-curing to redrying, and found that flue-curing had a greater effect on the microbial community than redrying. This is conducive for the exploration of microbial resources and improvement of tobacco leaf quality.

[Source](https://doi.org/10.1038/s41598-023-40502-0)

10.1038/s41598-023-40502-0

---

## Inter-comparison of marine microbiome sampling protocols.
 19.08.2023, _F Pascoal, MP Tomasino, R Piredda, GM Quero, L Torgo, J Poulain, PE Galand, JA Fuhrman, A Mitchell, T Tinta, T Turk Dermastia, A Fernandez-Guerra, A Vezzi, R Logares, F Malfatti, H Endo, AM Dąbrowska, F De Pascale, P Sánchez, N Henry, B Fosso, B Wilson, S Toshchakov, GK Ferrant, I Grigorov, FRJ Vieira, R Costa, S Pesant, C Magalhães_


Research on marine microbial communities is growing, but studies are hard to compare because of variation in seawater sampling protocols. To help researchers in the inter-comparison of studies that use different seawater sampling methodologies, as well as to help them design future sampling campaigns, we developed the EuroMarine Open Science Exploration initiative (EMOSE). Within the EMOSE framework, we sampled thousands of liters of seawater from a single station in the NW Mediterranean Sea (Service d'Observation du Laboratoire Arago [SOLA], Banyuls-sur-Mer), during one single day. The resulting dataset includes multiple seawater processing approaches, encompassing different material-type kinds of filters (cartridge membrane and flat membrane), three different size fractionations (&gt;0.22 µm, 0.22-3 µm, 3-20 µm and &gt;20 µm), and a number of different seawater volumes ranging from 1 L up to 1000 L. We show that the volume of seawater that is filtered does not have a significant effect on prokaryotic and protist diversity, independently of the sequencing strategy. However, there was a clear difference in alpha and beta diversity between size fractions and between these and "whole water" (with no pre-fractionation). Overall, we recommend care when merging data from datasets that use filters of different pore size, but we consider that the type of filter and volume should not act as confounding variables for the tested sequencing strategies. To the best of our knowledge, this is the first time a publicly available dataset effectively allows for the clarification of the impact of marine microbiome methodological options across a wide range of protocols, including large-scale variations in sampled volume.

[Source](https://doi.org/10.1038/s43705-023-00278-w)

10.1038/s43705-023-00278-w

---

## Discovery of a novel bacterial class with the capacity to drive sulfur cycling and microbiome structure in a paleo-ocean analog

[Source](https://doi.org/10.1038/s43705-023-00287-9)

---

## MT-MAG: Accurate and interpretable machine learning for complete or partial taxonomic assignments of metagenomeassembled genomes.
 21.08.2023, _W Li, L Kari, Y Yu, LA Hug_


_Animals, Cattle, Female, Humans, Metagenome, Benchmarking, Computational Biology, Gastrointestinal Microbiome, Machine Learning_
We propose MT-MAG, a novel machine learning-based software tool for the complete or partial hierarchically-structured taxonomic classification of metagenome-assembled genomes (MAGs). MT-MAG is alignment-free, with k-mer frequencies being the only feature used to distinguish a DNA sequence from another (herein k = 7). MT-MAG is capable of classifying large and diverse metagenomic datasets: a total of 245.68 Gbp in the training sets, and 9.6 Gbp in the test sets analyzed in this study. In addition to complete classifications, MT-MAG offers a "partial classification" option, whereby a classification at a higher taxonomic level is provided for MAGs that cannot be classified to the Species level. MT-MAG outputs complete or partial classification paths, and interpretable numerical classification confidences of its classifications, at all taxonomic ranks. To assess the performance of MT-MAG, we define a "weighted classification accuracy," with a weighting scheme reflecting the fact that partial classifications at different ranks are not equally informative. For the two benchmarking datasets analyzed (genomes from human gut microbiome species, and bacterial and archaeal genomes assembled from cow rumen metagenomic sequences), MT-MAG achieves an average of 87.32% in weighted classification accuracy. At the Species level, MT-MAG outperforms DeepMicrobes, the only other comparable software tool, by an average of 34.79% in weighted classification accuracy. In addition, MT-MAG is able to completely classify an average of 67.70% of the sequences at the Species level, compared with DeepMicrobes which only classifies 47.45%. Moreover, MT-MAG provides additional information for sequences that it could not classify at the Species level, resulting in the partial or complete classification of 95.13%, of the genomes in the datasets analyzed. Lastly, unlike other taxonomic assignment tools (e.g., GDTB-Tk), MT-MAG is an alignment-free and genetic marker-free tool, able to provide additional bioinformatics analysis to confirm existing or tentative taxonomic assignments.

[Source](https://doi.org/10.1371/journal.pone.0283536)

10.1371/journal.pone.0283536

---

## BinaRena: a dedicated interactive platform for human-guided exploration and binning of metagenomes.
 21.08.2023, _MJ Pavia, A Chede, Z Wu, H Cadillo-Quiroz, Q Zhu_


_Humans, Metagenome, Microbiota, Algorithms, Biological Evolution, Diarrhea_
Exploring metagenomic contigs and "binning" them into metagenome-assembled genomes (MAGs) are essential for the delineation of functional and evolutionary guilds within microbial communities. Despite the advances in automated binning algorithms, their capabilities in recovering MAGs with accuracy and biological relevance are so far limited. Researchers often find that human involvement is necessary to achieve representative binning results. This manual process however is expertise demanding and labor intensive, and it deserves to be supported by software infrastructure.

[Source](https://doi.org/10.1186/s40168-023-01625-8)

10.1186/s40168-023-01625-8

---

## Detection of viable commensal bacteria in murine melanoma tumors by culturomics.
 14.08.2023, _CM Phelps, JH Shapira, CR Laughlin, M Meisel_


Emerging evidence suggests the tumor microbiome at gut-distal sites can modulate tumor immunity and response to cancer immunotherapy. However, detection of commensal bacteria at gut-distal tumor sites is challenging given their low abundance. Here, we present a culturomics approach to facilitate recovery of phylogenetically diverse live commensal bacteria within gut-distal melanoma tumors. We describe steps for media preparation, tissue isolation, tissue homogenization, and host cell lysis. We then detail broth expansion culture followed by agar culture and single-colony 16S rRNA sequencing. For complete details on the use and execution of this protocol, please refer to Bender and McPherson et al. (2023).1.

[Source](https://doi.org/10.1016/j.xpro.2023.102492)

10.1016/j.xpro.2023.102492

---

