## Particular genomic and virulence traits associated with preterm infant-derived toxigenic <em>Clostridium perfringens</em> strains

[Source](https://doi.org/10.1038/s41564-023-01385-z)

---

## Detailed mapping of <em>Bifidobacterium</em> strain transmission from mother to infant via a dual culture-based and metagenomic approach

[Source](https://doi.org/10.1038/s41467-023-38694-0)

---

## Expanding the repertoire of counterselection markers for markerless gene deletion in the human gut bacterium <em>Phocaeicola vulgatus</em>

[Source](https://doi.org/10.1016/j.anaerobe.2023.102742)

---

## Gut microbiota, circulating inflammatory markers and metabolites, and carotid artery atherosclerosis in HIV infection

[Source](https://doi.org/10.1186/s40168-023-01566-2)

---

## Symbiotic microbiome <em>Staphylococcus epidermidis</em> restricts IL-33 production in allergic nasal epithelium via limiting the cellular necroptosis

[Source](https://doi.org/10.1186/s12866-023-02898-7)

---

## Gut microbiome associated with low anterior resection syndrome after rectal cancer surgery

[Source](https://doi.org/10.1038/s41598-023-34557-2)

---

## Metabolomics Reveals Gut Microbiota Contribute to PPARα Deficiency-Induced Alcoholic Liver Injury

[Source](https://doi.org/10.1021/acs.jproteome.3c00093)

---

## Following your gut: the emerging role of the gut microbiota in predicting and treating immune-related adverse events

[Source](https://doi.org/10.1097/CCO.0000000000000957)

---

## Respiratory eukaryotic virome expansion and bacteriophage deficiency characterize childhood asthma

[Source](https://doi.org/10.1038/s41598-023-34730-7)

---

## The causal links between gut microbiota and COVID-19: A Mendelian randomization study

[Source](https://doi.org/10.1002/jmv.28784)

---

## The Sexome&nbsp;– A proof of concept study into microbial transfer between heterosexual couples after sexual intercourse

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0379073823001615?via%3Dihub)

---

## A systematic review of the protein composition of whole saliva in subjects with healthy periodontium compared with chronic periodontitis

[Source](https://doi.org/10.1371/journal.pone.0286079)

---

## Metagenomic analysis and antibacterial activity of kefir microorganisms

[Source](https://doi.org/10.1111/1750-3841.16614)

---

## Fecal Metabolites as Biomarkers for Predicting Food Intake by Healthy Adults

[Source](https://doi.org/10.1093/jn/nxac195)

---

## Taxonogenomics of <em>Culturomica massiliensis</em> gen. nov., sp. nov., and <em>Emergencia timonensis</em> gen. nov., sp. nov. new bacteria isolated from human stool microbiota

[Source](https://doi.org/10.1038/s41598-023-35443-7)

---

## Gut microbiome signatures of extreme environment adaption in Tibetan pig

[Source](https://doi.org/10.1038/s41522-023-00395-3)

---

## Holobiont Evolution: Population Theory for the Hologenome

[Source](https://doi.org/10.1086/723782)

---

## Integration of Phenotypes in Microbiome Networks for Designing Synthetic Communities: a Study of Mycobiomes in the Grafted Tomato System

[Source](https://doi.org/10.1128/aem.01843-22)

---

## Root Niches of Blueberry Imprint Increasing Bacterial-Fungal Interkingdom Interactions along the Soil-Rhizosphere-Root Continuum

[Source](https://doi.org/10.1128/spectrum.05333-22)

---

## MDM-Pred: a freely available web application for predicting the metabolism of drug-like compounds by the gut microbiota

[Source](https://doi.org/10.1080/1062936X.2023.2214375)

---

## HONMF: integration analysis of multi-omics microbiome data via matrix factorization and hypergraph

[Source](https://doi.org/10.1093/bioinformatics/btad335)

---

## Bayesian compositional regression with microbiome features via variational inference.
 25.05.2023, _DAV Scott, E Benavente, J Libiseller-Egger, D Fedorov, J Phelan, E Ilina, P Tikhonova, A Kudryavstev, J Galeeva, T Clark, A Lewin_


_Humans, Bayes Theorem, Linear Models, Microbiota, Markov Chains, Gastrointestinal Microbiome, Monte Carlo Method_
The microbiome plays a key role in the health of the human body. Interest often lies in finding features of the microbiome, alongside other covariates, which are associated with a phenotype of interest. One important property of microbiome data, which is often overlooked, is its compositionality as it can only provide information about the relative abundance of its constituting components. Typically, these proportions vary by several orders of magnitude in datasets of high dimensions. To address these challenges we develop a Bayesian hierarchical linear log-contrast model which is estimated by mean field Monte-Carlo co-ordinate ascent variational inference (CAVI-MC) and easily scales to high dimensional data. We use novel priors which account for the large differences in scale and constrained parameter space associated with the compositional covariates. A reversible jump Monte Carlo Markov chain guided by the data through univariate approximations of the variational posterior probability of inclusion, with proposal parameters informed by approximating variational densities via auxiliary parameters, is used to estimate intractable marginal expectations. We demonstrate that our proposed Bayesian method performs favourably against existing frequentist state of the art compositional data analysis methods. We then apply the CAVI-MC to the analysis of real data exploring the relationship of the gut microbiome to body mass index.

[Source](https://doi.org/10.1186/s12859-023-05219-x)

10.1186/s12859-023-05219-x

---

## The Pathfinder plasmid toolkit for genetically engineering newly isolated bacteria enables the study of Drosophila-colonizing Orbaceae

[Source](https://doi.org/10.1038/s43705-023-00255-3)

---

## Unravelling animal-microbiota evolution on a chip.
 22.05.2023, _O Aizpurua, K Blijleven, U Trivedi, MTP Gilbert, A Alberdi_


Whether and how microorganisms have shaped the evolution of their animal hosts is a major question in biology. Although many animal evolutionary processes appear to correlate with changes in their associated microbial communities, the mechanistic processes leading to these patterns and their causal relationships are still far from being resolved. Gut-on-a-chip models provide an innovative approach that expands beyond the potential of conventional microbiome profiling to study how different animals sense and react to microbes by comparing responses of animal intestinal tissue models to different microbial stimuli. This complementary knowledge can contribute to our understanding of how host genetic features facilitate or prevent different microbiomes from being assembled, and in doing so elucidate the role of host-microbiota interactions in animal evolution.

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(23)00148-8)

10.1016/j.tim.2023.04.010

---

