## Thousands of previously unknown phages discovered in whole-community human gut metagenomes.
 02.04.2021, _S Benler, N Yutin, D Antipov, M Rayko, S Shmakov, AB Gussow, P Pevzner, EV Koonin_


Double-stranded DNA bacteriophages (dsDNA phages) play pivotal roles in structuring human gut microbiomes; yet, the gut virome is far from being fully characterized, and additional groups of phages, including highly abundant ones, continue to be discovered by metagenome mining. A multilevel framework for taxonomic classification of viruses was recently adopted, facilitating the classification of phages into evolutionary informative taxonomic units based on hallmark genes. Together with advanced approaches for sequence assembly and powerful methods of sequence analysis, this revised framework offers the opportunity to discover and classify unknown phage taxa in the human gut.

[Source](http://click.springernature.com/f/a/NubvWTsHEMMSYTbw6djWZA~~/AAEGiQA~/RgRiTVguP0RPaHR0cHM6Ly9taWNyb2Jpb21lam91cm5hbC5iaW9tZWRjZW50cmFsLmNvbS9hcnRpY2xlcy8xMC4xMTg2L3M0MDE2OC0wMjEtMDEwMTctd1cDc3BjQgpgZy7TamBJoYVuUhJlbGllc2Jpa0BnbWFpbC5jb21YBAAAAAA~)

10.1186/s40168-021-01017-w

---

## Elevated rates of horizontal gene transfer in the industrialized human microbiome.
 01.04.2021, _M Groussin, M Poyet, A Sistiaga, SM Kearney, K Moniz, M Noel, J Hooker, SM Gibbons, L Segurel, A Froment, RS Mohamed, A Fezeu, VA Juimo, S Lafosse, FE Tabe, C Girard, D Iqaluk, LTT Nguyen, BJ Shapiro, J Lehtimäki, L Ruokolainen, PP Kettunen, T Vatanen, S Sigwazi, A Mabulla, M Domínguez-Rodrigo, YA Nartey, A Agyei-Nkansah, A Duah, YA Awuku, KA Valles, SO Asibey, MY Afihene, LR Roberts, A Plymoth, CA Onyekwere, RE Summons, RJ Xavier, EJ Alm_


Industrialization has impacted the human gut ecosystem, resulting in altered microbiome composition and diversity. Whether bacterial genomes may also adapt to the industrialization of their host populations remains largely unexplored. Here, we investigate the extent to which the rates and targets of horizontal gene transfer (HGT) vary across thousands of bacterial strains from 15 human populations spanning a range of industrialization. We show that HGTs have accumulated in the microbiome over recent host generations and that HGT occurs at high frequency within individuals. Comparison across human populations reveals that industrialized lifestyles are associated with higher HGT rates and that the functions of HGTs are related to the level of host industrialization. Our results suggest that gut bacteria continuously acquire new functionality based on host lifestyle and that high rates of HGT may be a recent development in human history linked to industrialization.

[Source](https://www.sciencedirect.com/science/article/pii/S0092867421002415)

10.1016/j.cell.2021.02.052

---

## Combined effects of composting and antibiotic administration on cattle manure-borne antibiotic resistance genes.
 02.04.2021, _I Keenum, RK Williams, P Ray, ED Garner, KF Knowlton, A Pruden_


Research is needed to delineate the relative and combined effects of different antibiotic administration and manure management practices in either amplifying or attenuating the potential for antibiotic resistance to spread. Here, we carried out a comprehensive parallel examination of the effects of small-scale (&gt; 55 °C × 3 days) static and turned composting of manures from dairy and beef cattle collected during standard antibiotic administration (cephapirin/pirlimycin or sulfamethazine/chlortetracycline/tylosin, respectively), versus from untreated cattle, on "resistomes" (total antibiotic resistance genes (ARGs) determined via shotgun metagenomic sequencing), bacterial microbiota, and indicator ARGs enumerated via quantitative polymerase chain reaction. To gain insight into the role of the thermophilic phase, compost was also externally heated to &gt; 55 °C × 15 days.

[Source](http://click.springernature.com/f/a/2_qRZa_KF0zxUivGyV2VLQ~~/AAEGiQA~/RgRiTVguP0RPaHR0cHM6Ly9taWNyb2Jpb21lam91cm5hbC5iaW9tZWRjZW50cmFsLmNvbS9hcnRpY2xlcy8xMC4xMTg2L3M0MDE2OC0wMjEtMDEwMDYtelcDc3BjQgpgZy7TamBJoYVuUhJlbGllc2Jpa0BnbWFpbC5jb21YBAAAAAA~)

10.1186/s40168-021-01006-z

---

## Swine growth promotion with antibiotics or alternatives can increase antibiotic resistance gene mobility potential.
 25.03.2021, _J Muurinen, J Richert, CL Wickware, B Richert, TA Johnson_


Even though the use of antibiotics for food-producing animals may contribute to the emergence of antimicrobial resistance, antibiotics are still used as growth promoters. Due to consumer and regulatory pressures, the use of alternatives to antibiotics as growth promoters is increasing, thus more information is needed on their capability to disseminate antimicrobial resistance compared to antibiotics. We investigated the impacts of carbadox (antibiotic), copper sulfate and zinc oxide (metals) and mushroom powder (natural product) on the pig fecal resistome and microbiome. Antibiotic resistance gene (ARG) and mobile genetic element (MGE) abundances were measured using a high-throughput qPCR array with 382 primer pairs. Bacterial community composition was determined by 16S rRNA gene sequencing. More ARGs co-occurred with MGEs in the growth promoter group samples than in the control group samples. Community composition could not be linked to resistome in the growth promoter group samples, indicating a potential decoupling of ARGs and phylogeny. Additionally, machine-learning methods aided in defining the community and resistome differences in response to treatments. Since increased ARG mobility potential was the primary response to the dietary additives used in this study, we suggest that ARG mobility should be considered when designing antimicrobial use policies and antimicrobial resistance surveillances.

[Source](https://www.nature.com/articles/s41598-021-84759-9)

10.1038/s41598-021-84759-9

---

## Using high-abundance proteins as guides for fast and effective peptide/protein identification from human gut metaproteomic data.
 02.04.2021, _M Stamboulian, S Li, Y Ye_


A few recent large efforts significantly expanded the collection of human-associated bacterial genomes, which now contains thousands of entities including reference complete/draft genomes and metagenome assembled genomes (MAGs). These genomes provide useful resource for studying the functionality of the human-associated microbiome and their relationship with human health and diseases. One application of these genomes is to provide a universal reference for database search in metaproteomic studies, when matched metagenomic/metatranscriptomic data are unavailable. However, a greater collection of reference genomes may not necessarily result in better peptide/protein identification because the increase of search space often leads to fewer spectrum-peptide matches, not to mention the drastic increase of computation time. Video Abstract METHODS: Here, we present a new approach that uses two steps to optimize the use of the reference genomes and MAGs as the universal reference for human gut metaproteomic MS/MS data analysis. The first step is to use only the high-abundance proteins (HAPs) (i.e., ribosomal proteins and elongation factors) for metaproteomic MS/MS database search and, based on the identification results, to derive the taxonomic composition of the underlying microbial community. The second step is to expand the search database by including all proteins from identified abundant species. We call our approach HAPiID (HAPs guided metaproteomics IDentification).

[Source](http://click.springernature.com/f/a/rYM1wUyFyTAr_hTelPG-MA~~/AAEGiQA~/RgRiTVguP0RPaHR0cHM6Ly9taWNyb2Jpb21lam91cm5hbC5iaW9tZWRjZW50cmFsLmNvbS9hcnRpY2xlcy8xMC4xMTg2L3M0MDE2OC0wMjEtMDEwMzUtOFcDc3BjQgpgZy7TamBJoYVuUhJlbGllc2Jpa0BnbWFpbC5jb21YBAAAAAA~)

10.1186/s40168-021-01035-8

---

## High throughput sequencing provides exact genomic locations of inducible prophages and accurate phage-to-host ratios in gut microbial strains.
 02.04.2021, _M Zünd, HJ Ruscheweyh, CM Field, N Meyer, M Cuenca, D Hoces, WD Hardt, S Sunagawa_


Temperate phages influence the density, diversity and function of bacterial populations. Historically, they have been described as carriers of toxins. More recently, they have also been recognised as direct modulators of the gut microbiome, and indirectly of host health and disease. Despite recent advances in studying prophages using non-targeted sequencing approaches, methodological challenges in identifying inducible prophages in bacterial genomes and quantifying their activity have limited our understanding of prophage-host interactions.

[Source](http://click.springernature.com/f/a/1ZyWYBuOj2s4j2Dx28zCQg~~/AAEGiQA~/RgRiTVguP0RPaHR0cHM6Ly9taWNyb2Jpb21lam91cm5hbC5iaW9tZWRjZW50cmFsLmNvbS9hcnRpY2xlcy8xMC4xMTg2L3M0MDE2OC0wMjEtMDEwMzMtd1cDc3BjQgpgZy7TamBJoYVuUhJlbGllc2Jpa0BnbWFpbC5jb21YBAAAAAA~)

10.1186/s40168-021-01033-w

---

## A comprehensive metagenomics framework to characterize organisms relevant for planetary protection.
 02.04.2021, _DC Danko, MA Sierra, JN Benardini, L Guan, JM Wood, N Singh, A Seuylemezian, DJ Butler, K Ryon, K Kuchin, D Meleshko, C Bhattacharya, KJ Venkateswaran, CE Mason_


Clean rooms of the Space Assembly Facility (SAF) at the Jet Propulsion Laboratory (JPL) at NASA are the final step of spacecraft cleaning and assembly before launching into space. Clean rooms have stringent methods of air-filtration and cleaning to minimize microbial contamination for exoplanetary research and minimize the risk of human pathogens, but they are not sterile. Clean rooms make a selective environment for microorganisms that tolerate such cleaning methods. Previous studies have attempted to characterize the microbial cargo through sequencing and culture-dependent protocols. However, there is not a standardized metagenomic workflow nor analysis pipeline for spaceflight hardware cleanroom samples to identify microbial contamination. Additionally, current identification methods fail to characterize and profile the risk of low-abundance microorganisms.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01020-1)

10.1186/s40168-021-01020-1

---

