## Metagenomic analysis of mother-infant gut microbiome reveals global distinct and shared microbial signatures.
 07.05.2021, _S Wang, S Zeng, M Egan, P Cherry, C Strain, E Morais, P Boyaval, CA Ryan, E M Dempsey, RP Ross, C Stanton_


Emerging evidence indicates maternal microbiota as one major reservoir for pioneering microbes in infants. However, the global distinct and identical features of mother-infant gut microbiota at various taxonomic resolutions and metabolic functions across cohorts and potential of infant microbial prediction based on their paired mother's gut microbiota remain unclear. Here, we analyzed 376 mother-infant dyads (468 mother and 1024 infant samples) of eight studies from six countries and observed higher diversity at species and strain levels in maternal gut microbiota but not their metabolic functions. A number of 290 species were shared in at least one mother-infant dyad, with 26 species (five at strain level) observed across cohorts. The profile of mother-infant shared species and strains was further influenced by delivery mode and feeding regimen. The mother-sourced species in infants exhibited similar strain heterogeneity but more metabolic functions compared to other-sourced species, suggesting the comparable stability and fitness of shared and non-shared species and the potential role of shared species in the early gut microbial community, respectively. Predictive models showed moderate performance accuracy for shared species and strains occurrences in infants. These generalized mother-infant shared species and strains may be considered as the primary targets for future work toward infant microbiome development and probiotics exploration.

[Source](https://doi.org/10.1080/19490976.2021.1911571)

10.1080/19490976.2021.1911571

---

## Intratumour microbiome associated with the infiltration of cytotoxic CD8+ T cells and patient survival in cutaneous melanoma.
 12.05.2021, _G Zhu, H Su, CH Johnson, SA Khan, H Kluger, L Lu_


The gut microbiome plays an important role in systemic inflammation and immune response. Microbes can translocate and reside in tumour niches. However, it is unclear how the intratumour microbiome affects immunity in human cancer. The purpose of this study was to investigate the association between intratumour bacteria, infiltrating CD8+ T cells and patient survival in cutaneous melanoma.

[Source](https://doi.org/10.1016/j.ejca.2021.03.053)

10.1016/j.ejca.2021.03.053

---

## Lipopolysaccharide from the commensal microbiota of the breast enhances cancer growth: role of S100A7 and TLR4.
 10.05.2021, _T Wilkie, AK Verma, H Zhao, M Charan, DK Ahirwar, S Kant, V Pancholi, S Mishra, RK Ganju_


The role of commensal bacterial microbiota in the pathogenesis of human malignancies has been a research field of incomparable progress in recent years. Although breast tissue is commonly assumed to be sterile, recent studies suggest that human breast tissue may contain a bacterial microbiota. In this study, we used an immune-competent orthotopic breast cancer mouse model to explore the existence of a unique and independent bacterial microbiota in breast tumors. We observed some similarities in breast cancer microbiota with skin; however, breast tumor microbiota was mainly enriched with gram-negative bacteria, serving as a primary source of lipopolysaccharide (LPS). In addition, dextran sulfate sodium (DSS) treatment in late-stage tumor lesions increased LPS levels in the breast tissue environment. We also discovered an increased expression of S100A7 and low level of TLR4 in late-stage tumors with or without DSS as compared to early-stage tumor lesions. The treatment of breast cancer cells with LPS increased the expression of S100A7 in breast cancer cells in vitro. Furthermore, S100A7 overexpression downregulated TLR4 and upregulated RAGE expression in breast cancer cells. Analysis of human breast cancer samples also highlighted the inverse correlation between S100A7 and TLR4 expression. Overall, these findings suggest that the commensal microbiota of breast tissue may enhance breast tumor burden through a novel LPS/S100A7/TLR4/RAGE signaling axis.

[Source](https://doi.org/10.1002/1878-0261.12975)

10.1002/1878-0261.12975

---

## Intestinal microbiota influences clinical outcome and side effects of early breast cancer treatment.
 08.05.2021, _S Terrisse, L Derosa, V Iebba, F Ghiringhelli, I Vaz-Luis, G Kroemer, M Fidelle, S Christodoulidis, N Segata, AM Thomas, AL Martin, A Sirven, S Everhard, F Aprahamian, N Nirmalathasan, R Aarnoutse, M Smidt, J Ziemons, C Caldas, S Loibl, C Denkert, S Durand, C Iglesias, F Pietrantonio, B Routy, F André, E Pasolli, S Delaloge, L Zitvogel_


The prognosis of early breast cancer (BC) relies on cell autonomous and immune parameters. The impact of the intestinal microbiome on clinical outcome has not yet been evaluated. Shotgun metagenomics was used to determine the composition of the fecal microbiota in 121 specimens from 76 early BC patients, 45 of whom were paired before and after chemotherapy. These patients were enrolled in the CANTO prospective study designed to record the side effects associated with the clinical management of BC. We analyzed associations between baseline or post-chemotherapy fecal microbiota and plasma metabolomics with BC prognosis, as well as with therapy-induced side effects. We examined the clinical relevance of these findings in immunocompetent mice colonized with BC patient microbiota that were subsequently challenged with histo-compatible mouse BC and chemotherapy. We conclude that specific gut commensals that are overabundant in BC patients compared with healthy individuals negatively impact BC prognosis, are modulated by chemotherapy, and may influence weight gain and neurological side effects of BC therapies. These findings obtained in adjuvant and neoadjuvant settings warrant prospective validation.

[Source](https://doi.org/10.1038/s41418-021-00784-1)

10.1038/s41418-021-00784-1

---

## Fecal microbiota composition associates with the capacity of human peripheral blood monocytes to differentiate into immunogenic dendritic cells in vitro.
 10.05.2021, _D Radojević, S Tomić, D Mihajlović, M Tolinački, B Pavlović, D Vučević, S Bojić, N Golić, M Čolić, J Đokić_


Although promising for active immunization in cancer patients, dendritic cells (DCs) vaccines generated in vitro display high inter-individual variability in their immunogenicity, which mostly limits their therapeutic efficacy. Gut microbiota composition is a key emerging factor affecting individuals' immune responses, but it is unknown how it affects the variability of donors' precursor cells to differentiate into immunogenic DCs in vitro. By analyzing gut microbiota composition in 14 healthy donors, along with the phenotype and cytokines production by monocyte-derived DCs, we found significant correlations between immunogenic properties of DC and microbiota composition. Namely, donors who had higher α-diversity of gut microbiota and higher abundance of short-chain fatty acid (SCFAs) and SCFA-producing bacteria in feces, displayed lower expression of CD1a on immature (im)DC and higher expression of ILT-3, costimulatory molecules (CD86, CD40) proinflammatory cytokines (TNF-α, IL-6, IL-8) and IL-12p70/IL-10 ratio, all of which correlated with their lower maturation potential and immunogenicity upon stimulation with LPS/IFNγ, a well-known Th1 polarizing cocktail. In contrast, imDCs generated from donors with lower α-diversity and higher abundance of Bifidobacterium and Collinsella in feces displayed higher CD1a expression and higher potential to up-regulate CD86 and CD40, increase TNF-α, IL-6, IL-8 production, and IL-12p70/IL-10 ratio upon stimulation. These results emphasize the important role of gut microbiota on the capacity of donor precursor cells to differentiate into immunogenic DCs suitable for cancer therapy, which could be harnessed for improving the actual and future DC-based cancer therapies.

[Source](https://doi.org/10.1080/19490976.2021.1921927)

10.1080/19490976.2021.1921927

---

## Substrate-driven differences in tryptophan catabolism by gut microbiota and Aryl hydrocarbon Receptor activation.
 08.05.2021, _Z Huang, T Schoones, JM Wells, V Fogliano, E Capuano_


This study aims to investigate the effect of tryptophan sources on tryptophan catabolism by gut microbiota and the Aryl hydrocarbon Receptor (AhR) activation.

[Source](https://doi.org/10.1002/mnfr.202100092)

10.1002/mnfr.202100092

---

## The evolution and changing ecology of the African hominid oral microbiome.
 11.05.2021, _JA Fellows Yates, IM Velsko, F Aron, C Posth, CA Hofman, RM Austin, CE Parker, AE Mann, K Nägele, KW Arthur, JW Arthur, CC Bauer, I Crevecoeur, C Cupillard, MC Curtis, L Dalén, M Díaz-Zorita Bonilla, JC Díez Fernández-Lomana, DG Drucker, E Escribano Escrivá, M Francken, VE Gibbon, MR González Morales, A Grande Mateu, K Harvati, AG Henry, L Humphrey, M Menéndez, D Mihailović, M Peresani, S Rodríguez Moroder, M Roksandic, H Rougier, S Sázelová, JT Stock, LG Straus, J Svoboda, B Teßmann, MJ Walker, RC Power, CM Lewis, K Sankaranarayanan, K Guschanski, RW Wrangham, FE Dewhirst, DC Salazar-García, J Krause, A Herbig, C Warinner_


The oral microbiome plays key roles in human biology, health, and disease, but little is known about the global diversity, variation, or evolution of this microbial community. To better understand the evolution and changing ecology of the human oral microbiome, we analyzed 124 dental biofilm metagenomes from humans, including Neanderthals and Late Pleistocene to present-day modern humans, chimpanzees, and gorillas, as well as New World howler monkeys for comparison. We find that a core microbiome of primarily biofilm structural taxa has been maintained throughout African hominid evolution, and these microbial groups are also shared with howler monkeys, suggesting that they have been important oral members since before the catarrhine-platyrrhine split ca. 40 Mya. However, community structure and individual microbial phylogenies do not closely reflect host relationships, and the dental biofilms of Homo and chimpanzees are distinguished by major taxonomic and functional differences. Reconstructing oral metagenomes from up to 100 thousand years ago, we show that the microbial profiles of both Neanderthals and modern humans are highly similar, sharing functional adaptations in nutrient metabolism. These include an apparent Homo-specific acquisition of salivary amylase-binding capability by oral streptococci, suggesting microbial coadaptation with host diet. We additionally find evidence of shared genetic diversity in the oral bacteria of Neanderthal and Upper Paleolithic modern humans that is not observed in later modern human populations. Differences in the oral microbiomes of African hominids provide insights into human evolution, the ancestral state of the human microbiome, and a temporal framework for understanding microbial health and disease.

[Source](https://doi.org/10.1073/pnas.2021655118)

10.1073/pnas.2021655118

---

## Response of the microbiome-gut-brain axis in Drosophila to amino acid deficit.
 06.05.2021, _B Kim, MI Kanai, Y Oh, M Kyung, EK Kim, IH Jang, JH Lee, SG Kim, GSB Suh, WJ Lee_


A balanced intake of macronutrients-protein, carbohydrate and fat-is essential for the well-being of organisms. An adequate calorific intake but with insufficient protein consumption can lead to several ailments, including kwashiorkor1. Taste receptors (T1R1-T1R3)2 can detect amino acids in the environment, and cellular sensors (Gcn2 and Tor)3 monitor the levels of amino acids in the cell. When deprived of dietary protein, animals select a food source that contains a greater proportion of protein or essential amino acids (EAAs)4. This suggests that food selection is geared towards achieving the target amount of a particular macronutrient with assistance of the EAA-specific hunger-driven response, which is poorly understood. Here we show in Drosophila that a microbiome-gut-brain axis detects a deficit of EAAs and stimulates a compensatory appetite for EAAs. We found that the neuropeptide CNMamide (CNMa)5 was highly induced in enterocytes of the anterior midgut during protein deprivation. Silencing of the CNMa-CNMa receptor axis blocked the EAA-specific hunger-driven response in deprived flies. Furthermore, gnotobiotic flies bearing an EAA-producing symbiotic microbiome exhibited a reduced appetite for EAAs. By contrast, gnotobiotic flies with a mutant microbiome that did not produce leucine or other EAAs showed higher expression of CNMa and a greater compensatory appetite for EAAs. We propose that gut enterocytes sense the levels of diet- and microbiome-derived EAAs and communicate the EAA-deprived condition to the brain through CNMa.

[Source](https://doi.org/10.1038/s41586-021-03522-2)

10.1038/s41586-021-03522-2

---

## Coadaptation between host genome and microbiome under long-term xenobiotic-induced selection.
 06.05.2021, _GH Wang, J Dittmer, B Douglas, L Huang, RM Brucker_


One of the most difficult experimental challenges today is testing the evolutionary dynamics shaping complex host-microbiome interactions. We investigated host-microbiome codiversification in response to xenobiotic-induced selection using an experimental evolution approach. To this end, we exposed the parasitoid wasp Nasonia vitripennis to sublethal concentrations of the widely used herbicide atrazine for 85 generations. Our results reveal that atrazine exposure not only mediated adaptive changes within the microbiome, which conferred host resistance to atrazine toxicity, but also exerted selective pressure on the host genome and altered host gene expression and immune response. Furthermore, microbiome transplant experiments reveal a decreased survival of adults from the control population after exposure to the evolved microbiome of the atrazine-exposed population, while no such decrease occurred in the reciprocal transplant. These results indicate that xenobiotic-induced selection mediated host-microbiome coadaptation, ultimately leading to a new host genome-microbiome equilibrium.

[Source](https://advances.sciencemag.org/content/7/19/eabd4473)

10.1126/sciadv.abd4473

---

## Predicted Metabolic Function of the Gut Microbiota of Drosophila melanogaster.
 05.05.2021, _NYD Ankrah, BE Barker, J Song, C Wu, JG McMullen, AE Douglas_


An important goal for many nutrition-based microbiome studies is to identify the metabolic function of microbes in complex microbial communities and their impact on host physiology. This research can be confounded by poorly understood effects of community composition and host diet on the metabolic traits of individual taxa. Here, we investigated these multiway interactions by constructing and analyzing metabolic models comprising every combination of five bacterial members of the Drosophila gut microbiome (from single taxa to the five-member community of Acetobacter and Lactobacillus species) under three nutrient regimes. We show that the metabolic function of Drosophila gut bacteria is dynamic, influenced by community composition, and responsive to dietary modulation. Furthermore, we show that ecological interactions such as competition and mutualism identified from the growth patterns of gut bacteria are underlain by a diversity of metabolic interactions, and show that the bacteria tend to compete for amino acids and B vitamins more frequently than for carbon sources. Our results reveal that, in addition to fermentation products such as acetate, intermediates of the tricarboxylic acid (TCA) cycle, including 2-oxoglutarate and succinate, are produced at high flux and cross-fed between bacterial taxa, suggesting important roles for TCA cycle intermediates in modulating Drosophila gut microbe interactions and the potential to influence host traits. These metabolic models provide specific predictions of the patterns of ecological and metabolic interactions among gut bacteria under different nutrient regimes, with potentially important consequences for overall community metabolic function and nutritional interactions with the host.IMPORTANCEDrosophila is an important model for microbiome research partly because of the low complexity of its mostly culturable gut microbiota. Our current understanding of how Drosophila interacts with its gut microbes and how these interactions influence host traits derives almost entirely from empirical studies that focus on individual microbial taxa or classes of metabolites. These studies have failed to capture fully the complexity of metabolic interactions that occur between host and microbe. To overcome this limitation, we reconstructed and analyzed 31 metabolic models for every combination of the five principal bacterial taxa in the gut microbiome of Drosophila This revealed that metabolic interactions between Drosophila gut bacterial taxa are highly dynamic and influenced by cooccurring bacteria and nutrient availability. Our results generate testable hypotheses about among-microbe ecological interactions in the Drosophila gut and the diversity of metabolites available to influence host traits.

[Source](https://msystems.asm.org/content/6/3/e01369-20)

10.1128/mSystems.01369-20

---

## The microbiome of deep-sea fish reveals new microbial species and a sparsity of antibiotic resistance genes

[Source](https://doi.org/10.1080/19490976.2021.1921924)

---

## Compartmentalization of bacterial and fungal microbiomes in the gut of adult honeybees.
 10.05.2021, _M Callegari, E Crotti, M Fusi, R Marasco, E Gonella, I De Noni, D Romano, S Borin, G Tsiamis, A Cherif, A Alma, D Daffonchio_


The core gut microbiome of adult honeybee comprises a set of recurring bacterial phylotypes, accompanied by lineage-specific, variable, and less abundant environmental bacterial phylotypes. Several mutual interactions and functional services to the host, including the support provided for growth, hormonal signaling, and behavior, are attributed to the core and lineage-specific taxa. By contrast, the diversity and distribution of the minor environmental phylotypes and fungal members in the gut remain overlooked. In the present study, we hypothesized that the microbial components of forager honeybees (i.e., core bacteria, minor environmental phylotypes, and fungal members) are compartmentalized along the gut portions. The diversity and distribution of such three microbial components were investigated in the context of the physico-chemical conditions of different gut compartments. We observed that changes in the distribution and abundance of microbial components in the gut are consistently compartment-specific for all the three microbial components, indicating that the ecological and physiological interactions among the host and microbiome vary with changing physico-chemical and metabolic conditions of the gut.

[Source](https://doi.org/10.1038/s41522-021-00212-9)

10.1038/s41522-021-00212-9

---

## Seasonal Dynamics of the Honey Bee Gut Microbiota in Colonies Under Subtropical Climate : Seasonal Dynamics of Honey Bee Gut Microbiota.
 11.05.2021, _L Castelli, B Branchiccela, H Romero, P Zunino, K Antúnez_


Honey bees (Apis mellifera) provide invaluable benefits for food production and maintenance of biodiversity of natural environments through pollination. They are widely spread across the world, being adapted to different climatic conditions. To survive the winter in cold temperate regions, honey bees developed different strategies including storage of honey and pollen, confinement of individuals during the winter, and an annual cycle of colony growth and reproduction. Under these conditions, winter honey bees experience physiological changes, including changes in immunity and the composition of honey bee gut microbiota. However, under tropical or subtropical climates, the life cycle can experience alterations, i.e., queens lay eggs during almost all the year and new honey bees emerge constantly. In the present study, we characterized nurses' honey bee gut microbiota in colonies under subtropical region through a year, combining qPCR, PCR-DGGE, and 16S rDNA high-throughput sequencing. We also identified environmental variables involved in those changes. Our results showed that under the mentioned conditions, the number of bacteria is stable throughout the year. Diversity of gut microbiota is higher in spring and lower in summer and winter. Gradual changes in compositions occur between seasons: Lactobacillus spp. predominate in spring while Gilliamella apicola and Snodgrasella alvi predominate in summer and winter. Environmental variables (mainly precipitations) affected the composition of the honey bee gut microbiota. Our findings provide new insights into the dynamics of honey bee gut microbiota and may be useful to understand the adaptation of bees to different environmental conditions.

[Source](https://doi.org/10.1007/s00248-021-01756-1)

10.1007/s00248-021-01756-1

---

## Seasonal Dynamics of the Honey Bee Gut Microbiota in Colonies Under Subtropical Climate.

[Source](https://doi.org/10.1007/s00248-021-01756-1)

---

## Impact of cropping systems on the functional diversity of rhizosphere microbial communities associated with maize plant: a shotgun approach.
 11.05.2021, _AE Fadiji, JO Kanu, OO Babalola_


Understanding the functions carried out by rhizosphere microbiomes will further explore their importance in biotechnological improvement and agricultural sustainability. This study presents one of the foremost attempts to understand the functional diversity of the rhizosphere microbiome in mono-cropping and crop rotation farming sites using shotgun metagenomic techniques. We hypothesized that the functional diversity would vary in the cropping sites and more abundant in the rotational cropping site. Hence, we carried out complete DNA extraction from the bulk and rhizospheric soils associated with maize plant cultivated on the mono-cropping farm (LT and LTc) and the crop rotation farm (VD and VDc), respectively, and sequenced employing shotgun approach. Using the SEED subsystem, our result revealed that a total of 24 functional categories dominated the rotational cropping site, while four functional categories dominated the mono-cropping sites. Alpha diversity assessment showed that no significant difference (p &gt; 0.05) was observed across the cropping sites, while beta diversity assessment revealed a significant difference. Going by the high abundance of functional groups observed in the samples from the crop rotational site, it is evident that cropping systems influenced the functions of soil microbiomes. Worthy of note is the high abundance of unknown functions associated with these maize rhizosphere microbiomes. This is an indication that there are still some under-investigated functional genes associated with the maize rhizosphere microbiome. It is, therefore, imperative that further studies explore these functional genes for their agricultural and biotechnological potentials.

[Source](https://doi.org/10.1007/s00203-021-02354-y)

10.1007/s00203-021-02354-y

---

## Meta-transcriptomic profiling of functional variation of freshwater microbial communities induced by an antidepressant sertraline hydrochloride.
 12.05.2021, _H Cui, W Zhou, Y Deng, B Zheng, Q Zhang, Z Zhang, T Lu, H Qian_


Sertraline hydrochloride (Ser-HCl) is an effective and commonly used antidepressant drug, which is also frequently detected in aquatic environments. Our previous research showed that Ser-HCl changes the community composition of aquatic microbiome, but the understanding of the expression of functional pathways in microbial communities is still incomplete; to address this knowledge gap, we used meta-transcriptomics analysis to evaluate the toxicity of Ser-HCl to natural aquatic microbial communities cultured in laboratory microcosms. Meta-transcriptomic results show that a 15-day exposure to 50 μg/L Ser-HCl significantly changed the functional expression activity of aquatic microbial communities. Pathways related to lipid metabolism, energy metabolism, membrane transport function, and genetic information processing in the aquatic microbial community were severely inhibited under Ser-HCl treatment, but metabolism of cofactors and vitamins to alleviate biological toxicity after Ser-HCl exposure was enhanced. Our study thus reveals details of the effects of sertraline on the functioning of aquatic microbiome. Due to the extensive use of Ser-HCl and its strong biological activity, it should not continue to be an overlooked pollutant. Therefore, more attention should be paid to the negative effects of such biologically active drugs on the expression of aquatic microbiome.

[Source](https://doi.org/10.1016/j.scitotenv.2021.147434)

10.1016/j.scitotenv.2021.147434

---

## Aquatic microbial community is partially functionally redundant: Insights from an in situ reciprocal transplant experiment.

[Source](https://doi.org/10.1016/j.scitotenv.2021.147433)

---

## Towards enhancing coral heat tolerance: a "microbiome transplantation" treatment using inoculations of homogenized coral tissues.
 11.05.2021, _T Doering, M Wall, L Putchim, T Rattanawongwan, R Schroeder, U Hentschel, A Roik_


Microbiome manipulation could enhance heat tolerance and help corals survive the pressures of ocean warming. We conducted coral microbiome transplantation (CMT) experiments using the reef-building corals, Pocillopora and Porites, and investigated whether this technique can benefit coral heat resistance while modifying the bacterial microbiome. Initially, heat-tolerant donors were identified in the wild. We then used fresh homogenates made from coral donor tissues to inoculate conspecific, heat-susceptible recipients and documented their bleaching responses and microbiomes by 16S rRNA gene metabarcoding.

[Source](https://doi.org/10.1186/s40168-021-01053-6)

10.1186/s40168-021-01053-6

---

## The Bacterial Community Diversity of Bathroom Hot Tap Water Was Significantly Lower Than That of Cold Tap and Shower Water.
 10.05.2021, _C Zhang, K Qin, I Struewing, H Buse, J Santo Domingo, D Lytle, J Lu_


Microbial drinking water quality in premise plumbing systems (PPSs) strongly affects public health. Bacterial community structure is the essential aspect of microbial water quality. Studies have elucidated the microbial community structure in cold tap water, while the microbial community structures in hot tap and shower water are poorly understood. We sampled cold tap, hot tap, and shower water from a simulated PPS monthly for 16 consecutive months and assessed the bacterial community structures in those samples via high-throughput sequencing of bacterial 16S rRNA genes. The total relative abundance of the top five most abundant phyla (Proteobacteria, Actinobacteria, Bacteroidetes, Cyanobacteria, and Firmicutes) was greater than 90% among the 24 identified phyla. The most abundant families were Burkholderiaceae, Sphingomonadaceae, unclassified Alphaproteobacteria, unclassified Corynebacteriales, and Mycobacteriaceae. A multiple linear regression suggests that the bacterial community diversity increased with water temperature and the age of the simulated PPS, decreased with total chlorine residual concentration, and had a limited seasonal variation. The bacterial community in hot tap water had significantly lower Shannon and Inverse Simpson diversity indices (p &lt; 0.05) and thus a much lower diversity than those in cold tap and shower water. The paradoxical results (i.e., diversity increased with water temperature, but hot tap water bacterial community was less diverse) were presumably because (1) other environmental factors made hot tap water bacterial community less diverse, (2) the diversity of bacterial communities in all types of water samples increased with water temperature, and (3) the first draw samples of hot tap water could have a comparable or even lower temperature than shower water samples and the second draw samples of cold tap water. In both a three-dimensional Non-metric multidimensional scaling ordination plot and a phylogenetic dendrogram, the samples of cold tap and shower water cluster and are separate from hot tap water samples (p &lt; 0.05). In summary, the bacterial community in hot tap water in the simulated PPS had a distinct structure from and a much lower diversity than those in cold tap and shower water.

[Source](https://doi.org/10.3389/fmicb.2021.625324)

10.3389/fmicb.2021.625324

---

## A Protocol for Teaching Basic Next Generation Sequencing (NGS) Analysis Skills to Undergraduate Students Using Bash and R.

[Source](https://link.springer.com/protocol/10.1007%2F978-1-0716-1099-2_16)

---

## Microdroplet-based system for culturing of environmental microorganisms using FNAP-sort.
 08.05.2021, _K Saito, Y Ota, DM Tourlousse, S Matsukura, H Fujitani, M Morita, S Tsuneda, N Noda_


Droplet microfluidics has emerged as a powerful technology for improving the culturing efficiency of environmental microorganisms. However, its widespread adoption has been limited due to considerable technical challenges, especially related to identification and manipulation of individual growth-positive droplets. Here, we combined microfluidic droplet technology with on-chip "fluorescent nucleic acid probe in droplets for bacterial sorting" (FNAP-sort) for recovery of growth-positive droplets and droplet microdispensing to establish an end-to-end workflow for isolation and culturing of environmental microbes. As a proof-of-concept, we demonstrate the ability of our technique to yield high-purity cultures of rare microorganisms from a representative complex environmental microbiome. As our system employs off-the-shelf commercially available equipment, we believe that it can be readily adopted by others and may thus find widespread use toward culturing the high proportion of as-of-yet uncultured microorganisms in different biomes.

[Source](https://doi.org/10.1038/s41598-021-88974-2)

10.1038/s41598-021-88974-2

---

