## Enterotypes of the human gut mycobiome.
 15.08.2023, _S Lai, Y Yan, Y Pu, S Lin, JG Qiu, BH Jiang, MI Keller, M Wang, P Bork, WH Chen, Y Zheng, XM Zhao_


_Humans, Aged, Mycobiome, Microbiota, Gastrointestinal Microbiome, Candida, Aging_
The fungal component of the human gut microbiome, also known as the mycobiome, plays a vital role in intestinal ecology and human health. However, the overall structure of the gut mycobiome as well as the inter-individual variations in fungal composition remains largely unknown. In this study, we collected a total of 3363 fungal sequencing samples from 16 cohorts across three continents, including 572 newly profiled samples from China.

[Source](https://doi.org/10.1186/s40168-023-01586-y)

10.1186/s40168-023-01586-y

---

## Effect of butyrate-producing enterobacteria on advanced hepatocellular carcinoma treatment with atezolizumab and bevacizumab

[Source](https://doi.org/10.1002/cam4.6416)

---

## <em>Parvimonas micra</em> can translocate from the subgingival sulcus of the human oral cavity to colorectal adenocarcinoma

[Source](https://doi.org/10.1002/1878-0261.13506)

---

## An Oral Microbial Biomarker for Early Detection of Recurrence of Oral Squamous Cell Carcinoma.
 11.08.2023, _WN Lyu, MC Lin, CY Shen, LH Chen, YH Lee, SK Chen, LC Lai, EY Chuang, PJ Lou, MH Tsai_


Changes in the oral microbiome are associated with oral squamous cell carcinoma (OSCC). Oral microbe-derived signatures have been utilized as markers of OSCC. However, the structure of the oral microbiome during OSCC recurrence and biomarkers for the prediction of OSCC recurrence remains unknown. To identify OSCC recurrence-associated microbial biomarkers for the prediction of OSCC recurrence, we performed 16S rRNA amplicon sequencing on 54 oral swab samples from OSCC patients. Differences in bacterial compositions were observed in patients with vs without recurrence. We found that Granulicatella, Peptostreptococcus, Campylobacter, Porphyromonas, Oribacterium, Actinomyces, Corynebacterium, Capnocytophaga, and Dialister were enriched in OSCC recurrence. Functional analysis of the oral microbiome showed altered functions associated with OSCC recurrence compared with nonrecurrence. A random forest prediction model was constructed with five microbial signatures including Leptotrichia trevisanii, Capnocytophaga sputigena, Capnocytophaga, Cardiobacterium, and Olsenella to discriminate OSCC recurrence from original OSCC (accuracy = 0.963). Moreover, we validated the prediction model in another independent cohort (46 OSCC patients), achieving an accuracy of 0.761. We compared the accuracy of the prediction of OSCC recurrence between the five microbial signatures and two clinicopathological parameters, including resection margin and lymph node counts. The results predicted by the model with five microbial signatures showed a higher accuracy than those based on the clinical outcomes from the two clinicopathological parameters. This study demonstrated the validity of using recurrence-related microbial biomarkers, a noninvasive and effective method for the prediction of OSCC recurrence. Our findings may contribute to the prognosis and treatment of OSCC recurrence.

[Source](https://doi.org/10.1021/acsinfecdis.3c00269)

10.1021/acsinfecdis.3c00269

---

## Multilocus-sequence typing reveals clonality of <em>Staphylococcus aureus</em> in atopic dermatitis

[Source](https://doi.org/10.1093/ced/llad262)

---

## Metabarcoding approach to identify bacterial community profiling related to nosocomial infection and bacterial trafficking-routes in hospital environments.
 11.08.2023, _B Gimenes de Castro, B Mari Fredi, R Dos Santos Bezerra, QA Alcantara, CE Milani Neme, DE Mascarelli, AS Carvalho Tahyra, D Dos-Santos, CR Nappi, F Santos de Oliveira, F Pereira Freire, G Ballestero, JB Menuci Lima, J de Andrade Bolsoni, J Lourenço Gebenlian, N Lopes Bibo, N Soares Silva, N de Carvalho Santos, V Simionatto Zucherato, K Chagas Peronni, D Guariz Pinheiro, E Dias-Neto, G Gambero Gaspar, V Roberto Bollela, V da Silva Silveira, A Maria Fontes, N Maria Martinez-Rossi, S Nanev Slavov, J Paulo Bianchi Ximenez, F Barbosa, W Araújo Silva_


Nosocomial infections (NIs) appear in patients under medical care in the hospital. The surveillance of the bacterial communities employing high-resolution 16S rRNA profiling, known as metabarcoding, represents a reliable method to establish factors that may influence the composition of the bacterial population during NIs. The present study aimed to utilize high-resolution 16S rRNA profiling to identify high bacterial diversity by analyzing 11 inside and 10 outside environments from the General Hospital of Ribeirão Preto Medical School, Brazil. Our results identified a high bacterial diversity, and among these, the most abundant bacterial genera linked to NIs were Cutibacterium, Streptococcus, Staphylococcus, and Corynebacterium. A Acinetobacter was detected in cafeterias, bus stops, and adult and pediatric intensive care units (ICUs). Data suggest an association between transport and alimentation areas proximal to the hospital ICU environment. Interestingly, the correlation and clusterization analysis showed the potential of the external areas to directly influence the ICU pediatric department microbial community, including the outpatient's clinic, visitor halls, patient reception, and the closest cafeterias. Our results demonstrate that high-resolution 16S rRNA profiling is a robust and reliable tool for bacterial genomic surveillance. In addition, the metabarcoding approach might help elaborate decontamination policies, and consequently reduce NIs.

[Source](https://doi.org/10.1080/15287394.2023.2243978)

10.1080/15287394.2023.2243978

---

## Identification of host gene-microbiome associations in colorectal cancer patients using mendelian randomization.
 14.08.2023, _Y Xiang, C Zhang, J Wang, Y Cheng, L Wang, Y Tong, D Yan_


_Humans, Mendelian Randomization Analysis, Microbiota, Gastrointestinal Microbiome, East Asian People, Genome-Wide Association Study, Colorectal Neoplasms_
There are many studies indicating that alterations in the abundance of certain gut microbiota are associated with colorectal cancer (CRC). However, a causal relationship has not been identified due to confounding factors such as lifestyle, environmental, and possible reverse causal associations between the two. Furthermore, certain host gene mutations can also contribute to the development of CRC. However, the association between genes and gut microbes in patients with CRC has not been extensively studied.

[Source](https://doi.org/10.1186/s12967-023-04335-9)

10.1186/s12967-023-04335-9

---

## Bacterial amylases enable glycogen degradation by the vaginal microbiome.
 10.08.2023, _DJ Jenkins, BM Woolston, MI Hood-Pishchany, P Pelayo, AN Konopaski, M Quinn Peters, MT France, J Ravel, CM Mitchell, S Rakoff-Nahoum, C Whidbey, EP Balskus_


The human vaginal microbiota is frequently dominated by lactobacilli and transition to a more diverse community of anaerobic microbes is associated with health risks. Glycogen released by lysed epithelial cells is believed to be an important nutrient source in the vagina. However, the mechanism by which vaginal bacteria metabolize glycogen is unclear, with evidence implicating both bacterial and human enzymes. Here we biochemically characterize six glycogen-degrading enzymes (GDEs), all of which are pullanases (PulA homologues), from vaginal bacteria that support the growth of amylase-deficient Lactobacillus crispatus on glycogen. We reveal variations in their pH tolerance, substrate preferences, breakdown products and susceptibility to inhibition. Analysis of vaginal microbiome datasets shows that these enzymes are expressed in all community state types. Finally, we confirm the presence and activity of bacterial and human GDEs in cervicovaginal fluid. This work establishes that bacterial GDEs can participate in the breakdown of glycogen, providing insight into metabolism that may shape the vaginal microbiota.

[Source](https://doi.org/10.1038/s41564-023-01447-2)

10.1038/s41564-023-01447-2

---

## Babies, bugs and brains: How the early microbiome associates with infant brain and behavior development.
 11.08.2023, _S Hunter, E Flaten, C Petersen, J Gervain, JF Werker, LJ Trainor, BB Finlay_


_Humans, Infant, Pilot Projects, Gastrointestinal Microbiome, Microbiota, Bacteria, Feces, Brain_
Growing evidence is demonstrating the connection between the microbiota gut-brain axis and neurodevelopment. Microbiota colonization occurs before the maturation of many neural systems and is linked to brain health. Because of this it has been hypothesized that the early microbiome interactions along the gut-brain axis evolved to promote advanced cognitive functions and behaviors. Here, we performed a pilot study with a multidisciplinary approach to test if the microbiota composition of infants is associated with measures of early cognitive development, in particular neural rhythm tracking; language (forward speech) versus non-language (backwards speech) discrimination; and social joint attention. Fecal samples were collected from 56 infants between four and six months of age and sequenced by shotgun metagenomic sequencing. Of these, 44 performed the behavioral Point and Gaze test to measure joint attention. Infants were tested on either language discrimination using functional near-infrared spectroscopy (fNIRS; 25 infants had usable data) or neural rhythm tracking using electroencephalogram (EEG; 15 had usable data). Infants who succeeded at the Point and Gaze test tended to have increased Actinobacteria and reduced Firmicutes at the phylum level; and an increase in Bifidobacterium and Eggerthella along with a reduction in Hungatella and Streptococcus at the genus level. Measurements of neural rhythm tracking associated negatively to the abundance of Bifidobacterium and positively to the abundance of Clostridium and Enterococcus for the bacterial abundances, and associated positively to metabolic pathways that can influence neurodevelopment, including branched chain amino acid biosynthesis and pentose phosphate pathways. No associations were found for the fNIRS language discrimination measurements. Although the tests were underpowered due to the small pilot sample sizes, potential associations were identified between the microbiome and measurements of early cognitive development that are worth exploring further.

[Source](https://doi.org/10.1371/journal.pone.0288689)

10.1371/journal.pone.0288689

---

## A retrospective analysis of vitamin B6 deficiency and associated changes of gut microbes in Crohn's disease.
 07.08.2023, _Z Feng, J Hua, F Guo, Z Liu, Y Zhao, W Wu_


Patients with inflammatory bowel diseases (IBD) are at risk of micronutrient deficiencies, particularly during flares. Vitamin B6 is required for the proper development of brain, nerves, and many other parts of the body. However, limited studies are available to describe the prevalence, relevance and consequences of vitamin B6 deficiencies in IBD. We aim to estimate the prevalence of vitamin B6 deficiencies in Crohn's disease (CD) patients, to identify associated risk factors and to explore the alteration of intestinal microbiota related to vitamin B6 status.

[Source](https://doi.org/10.1038/s41430-023-01324-5)

10.1038/s41430-023-01324-5

---

## The intestinal microbiota predicts COVID-19 severity and fatality regardless of hospital feeding method.
 07.08.2023, _V Bucci, DV Ward, S Bhattarai, M Rojas-Correa, A Purkayastha, D Holler, MD Qu, WG Mitchell, J Yang, S Fountain, A Zeamer, CS Forconi, G Fujimori, B Odwar, C Cawley, AM Moormann, M Wessolossky, A Maldonado-Contreras_


SARS-CoV-2-positive patients exhibit gut and oral microbiome dysbiosis, which is associated with various aspects of COVID-19 disease (1-4). Here, we aim to identify gut and oral microbiome markers that predict COVID-19 severity in hospitalized patients, specifically severely ill patients compared to moderately ill ones. Moreover, we investigate whether hospital feeding (solid versus enteral), an important cofounder, influences the microbial composition of hospitalized COVID-19 patients. We used random forest classification machine learning models with interpretable secondary analyses. The gut, but not the oral microbiota, was a robust predictor of both COVID-19-related fatality and severity of hospitalized patients, with a higher predictive value than most clinical variables. In addition, perturbations of the gut microbiota due to enteral feeding did not associate with species that were predictive of COVID-19 severity. IMPORTANCE SARS-CoV-2 infection leads to wide-ranging, systemic symptoms with sometimes unpredictable morbidity and mortality. It is increasingly clear that the human microbiome plays an important role in how individuals respond to viral infections. Our study adds to important literature about the associations of gut microbiota and severe COVID-19 illness during the early phase of the pandemic before the availability of vaccines. Increased understanding of the interplay between microbiota and SARS-CoV-2 may lead to innovations in diagnostics, therapies, and clinical predictions.

[Source](https://doi.org/10.1128/msystems.00310-23)

10.1128/msystems.00310-23

---

## The importance of the gut microbiome in the pathogenesis and transmission of SARS-CoV-2.
 13.08.2023, _C Brogna, V Viduto, M Fabrowski, S Cristoni, G Marino, L Montano, M Piscopo_


Zhou et al. study nicely traces a significant topic in COVID-19 infection: the persistence of the virus within the intestinal tract. Many pathological mechanisms have been noted in the current literature about the mode of infection and propagation of SARS-CoV-2 in the human body. Nevertheless, there are still many concerns about this: only some things seem well understood. We present a different point of view by illustrating the importance of the gut microbiome in the pathogenesis of COVID-19 disorders.

[Source](https://doi.org/10.1080/19490976.2023.2244718)

10.1080/19490976.2023.2244718

---

## Perturbations in gut microbiota composition in patients with polycystic ovary syndrome: a systematic review and meta-analysis.
 12.08.2023, _P Li, P Shuai, S Shen, H Zheng, P Sun, R Zhang, S Lan, Z Lan, T Jayawardana, Y Yang, J Zhao, Y Liu, X Chen, EM El-Omar, Z Wan_


_Humans, Female, Gastrointestinal Microbiome, Polycystic Ovary Syndrome, Dysbiosis, Phylogeny, Microbiota_
The results of human observational studies on the correlation between gut microbiota perturbations and polycystic ovary syndrome (PCOS) have been contradictory. This study aimed to perform the first systematic review and meta-analysis to evaluate the specificity of the gut microbiota in PCOS patients compared to healthy women.

[Source](https://doi.org/10.1186/s12916-023-02975-8)

10.1186/s12916-023-02975-8

---

## Caution regarding the specificities of pan-cancer microbial structure.
 15.08.2023, _A Gihawi, CS Cooper, DS Brewer_


_Humans, Reproducibility of Results, Microbiota, Machine Learning, Neoplasms_
Results published in an article by Poore et al. (Nature. 2020;579:567-574) suggested that machine learning models can almost perfectly distinguish between tumour types based on their microbial composition using machine learning models. Whilst we believe that there is the potential for microbial composition to be used in this manner, we have concerns with the paper that make us question the certainty of the conclusions drawn. We believe there are issues in the areas of the contribution of contamination, handling of batch effects, false positive classifications and limitations in the machine learning approaches used. This makes it difficult to identify whether the authors have identified true biological signal and how robust these models would be in use as clinical biomarkers. We commend Poore et al. on their approach to open data and reproducibility that has enabled this analysis. We hope that this discourse assists the future development of machine learning models and hypothesis generation in microbiome research.

[Source](https://doi.org/10.1099/mgen.0.001088)

10.1099/mgen.0.001088

---

## Gut microbial beta-glucuronidase: a vital regulator in female estrogen metabolism.
 14.08.2023, _S Hu, Q Ding, W Zhang, M Kang, J Ma, L Zhao_


A growing amount of evidence has supported that gut microbiota plays a vital role in the reproductive endocrine system throughout a woman's whole life, and gut microbial β-glucuronidase (gmGUS) is a key factor in regulating host estrogen metabolism. Moreover, estrogen levels also influence the composition as well as the diversity of gut microbiota. In normal condition, the gmGUS-estrogen crosstalk maintains body homeostasis of physiological estrogen level. Once this homeostasis is broken, the estrogen metabolism will be disturbed, resulting in estrogen-related diseases, such as gynecological cancers, menopausal syndrome, etc. together with gut microbial dysbiosis, which may accelerate these pathological processes. In this review, we highlight the regulatory role of gmGUS on the physical estrogen metabolism and estrogen-related diseases, summarize the present evidence of the interaction between gmGUS and estrogen metabolism, and unwrap the potential mechanisms behind them. Finally, gmGUS may become a potential biomarker for early diagnosis of estrogen-induced diseases. Regulating gmGUS activity or transplanting gmGUS-producing microbes shows promise for treating estrogen-related diseases.

[Source](https://doi.org/10.1080/19490976.2023.2236749)

10.1080/19490976.2023.2236749

---

## Three specific gut bacteria in the occurrence and development of colorectal cancer: a concerted effort.
 10.08.2023, _D Gong, AG Adomako-Bonsu, M Wang, J Li_


_Humans, Colorectal Neoplasms, Escherichia coli, Bacteria, Rectum_
Colorectal cancer (CRC), which develops from the gradual evolution of tubular adenomas and serrated polyps in the colon and rectum, has a poor prognosis and a high mortality rate. In addition to genetics, lifestyle, and chronic diseases, intestinal integrity and microbiota (which facilitate digestion, metabolism, and immune regulation) could promote CRC development. For example, enterotoxigenic Bacteroides fragilis, genotoxic Escherichia coli (pks+ E. coli), and Fusobacterium nucleatum, members of the intestinal microbiota, are highly correlated in CRC. This review describes the roles and mechanisms of these three bacteria in CRC development. Their interaction during CRC initiation and progression has also been proposed. Our view is that in the precancerous stage of colorectal cancer, ETBF causes inflammation, leading to potential changes in intestinal ecology that may provide the basic conditions for pks+ E. coli colonization and induction of oncogenic mutations, when cancerous intestinal epithelial cells can further recruit F. nucleatum to colonise the lesion site and F. nucleatum may contribute to CRC advancement by primarily the development of cancer cells, stemization, and proliferation, which could create new and tailored preventive, screening and therapeutic interventions. However, there is the most dominant microbiota in each stage of CRC development, not neglecting the possibility that two or even all three bacteria could be engaged at any stage of the disease. The relationship between the associated gut microbiota and CRC development may provide important information for therapeutic strategies to assess the potential use of the associated gut microbiota in CRC studies, antibiotic therapy, and prevention strategies.

[Source](https://doi.org/10.7717/peerj.15777)

10.7717/peerj.15777

---

## Machine learning and metagenomics reveal shared antimicrobial resistance profiles across multiple chicken farms and abattoirs in China

[Source](https://doi.org/10.1038/s43016-023-00814-w)

---

## The enhancement in toxic potency of oxidized functionalized polyethylene-microplastics in mice gut and Caco-2 cells.
 08.08.2023, _J Wang, H Tian, Y Shi, Y Yang, F Yu, H Cao, L Gao, M Liu_


Microplastics (MPs) are inevitably oxidized in the environment, however, to date, no studies have discussed the biological toxicity of oxidized polyethylene (Ox-PE) MPs. In this study, oxidized low-density polyethylene (Ox-LDPE), a representative Ox-PE, was prepared using a selective oxidation method. The difference in toxicity between LDPE-MPs and Ox-LDPE-MPs were evaluated in C57BL/6 mice and Caco-2 cells. The proton nuclear magnetic resonance (1H NMR) and Fourier transform infrared (FTIR) spectroscopy analyses revealed that some hydrocarbon-containing groups were transformed into carboxyl and ketone groups during selective oxidation. In vivo experiment results showed that LDPE-MPs and Ox-LDPE-MPs exists in the intestinal (duodenum and colon) of mice, and Ox-LDPE-MPs caused more severe intestinal histological changes, oxidative stress, and inflammatory response. The gut microbiota data showed that the relative abundance of Lactobacillus decreased significantly in the LDPE-MP- and Ox-LDPE-MP-exposed groups (P &lt; 0.05). The predicted Kyoto Encyclopedia of Genes and Genomes (KEGG) metabolic pathway suggested that exposure to LDPE-MPs or Ox-LDPE-MPs inhibited glycan biosynthesis and metabolism in the flora (P &lt; 0.05). In vitro experiment results showed that selective oxidation to LDPE promoted its uptake by cells and aggravated adverse effects on cells, including reduced cell viability, damaged cell membrane, oxidative stress, and mitochondrial depolarization. The major mechanism of the increased toxicity of Ox-LDPE-MPs may be its easier accumulation and the ionic effect of oxygen-containing functional groups. Overall, these findings provide insights on the differences in toxicity between LDPE-MPs and Ox-LDPE-MPs. They also provide new perspectives for understanding the biohazards of MPs, which are necessary to accurately assess the potential environmental and health risks of these plastic pollutants.

[Source](https://doi.org/10.1016/j.scitotenv.2023.166057)

10.1016/j.scitotenv.2023.166057

---

## Geroprotective potential of microbiome modulators in the <em>Caenorhabditis elegans</em> model

[Source](https://doi.org/10.1007/s11357-023-00901-7)

---

## Macroevolutionary constraints on global microbial diversity.
 11.08.2023, _FJ Fishman, JT Lennon_


Biologists have long sought to quantify the number of species on Earth. Often missing from these efforts is the contribution of microorganisms, the smallest but most abundant form of life on the planet. Despite recent large-scale sampling efforts, estimates of global microbial diversity span many orders of magnitude. It is important to consider how speciation and extinction over the last 4 billion years constrain inventories of biodiversity. We parameterized macroevolutionary models based on birth-death processes that assume constant and universal speciation and extinction rates. The models reveal that richness beyond 1012 species is feasible and in agreement with empirical predictions. Additional simulations suggest that mass extinction events do not place hard limits on modern-day microbial diversity. Together, our study provides independent support for a massive global-scale microbiome while shedding light on the upper limits of life on Earth.

[Source](https://doi.org/10.1002/ece3.10403)

10.1002/ece3.10403

---

## Evolutionary dynamics on sequential temporal networks.
 07.08.2023, _A Sheng, A Li, L Wang_


Population structure is a well-known catalyst for the evolution of cooperation and has traditionally been considered to be static in the course of evolution. Conversely, real-world populations, such as microbiome communities and online social networks, frequently show a progression from tiny, active groups to huge, stable communities, which is insufficient to be captured by constant structures. Here, we propose sequential temporal networks to characterize growing networked populations, and we extend the theory of evolutionary games to these temporal networks with arbitrary structures and growth rules. We derive analytical rules under which a sequential temporal network has a higher fixation probability for cooperation than its static counterpart. Under neutral drift, the rule is simply a function of the increment of nodes and edges in each time step. But if the selection is weak, the rule is related to coalescence times on networks. In this case, we propose a mean-field approximation to calculate fixation probabilities and critical benefit-to-cost ratios with lower calculation complexity. Numerical simulations in empirical datasets also prove the cooperation-promoting effect of population growth. Our research stresses the significance of population growth in the real world and provides a high-accuracy approximation approach for analyzing the evolution in real-life systems.

[Source](https://doi.org/10.1371/journal.pcbi.1011333)

10.1371/journal.pcbi.1011333

---

## Nutritional and host environments determine community ecology and keystone species in a synthetic gut bacterial community

[Source](https://doi.org/10.1038/s41467-023-40372-0)

---

## Earth to Mars: A Protocol for Characterizing Permafrost in the Context of Climate Change as an Analog for Extraplanetary Exploration

[Source](https://doi.org/10.1089/ast.2022.0155)

---

## Bioleaching of polymetallic sulphidic mining residues: influence of increasing solid concentration on microbial community dynamics and metal dissolution.
 07.08.2023, _C Joulian, A Hubau, D Pino-Herrera, AG Guezennec_


Within the European research project NEMO, a bioleaching strategy was developed for efficient metal extraction from bioleach residue currently heap-leached at Sotkamo (Finland) that still contains sulphidic minerals and valuable metals (Ni, Zn, Co, Cu). The strategy of gradually increasing the solid content with 5% steps allowed the adaptation of the consortium up to 20%(w/w) solid content, with efficient metal dissolution and same dominant bacteria. Largest proportions of Sb. thermosulfidooxidans while Eh increased suggested it to be most involved in iron oxidation. At. caldus was rather found when pH stabilized, in line with a production of protons from sulphur oxidation that maintained low pH. 'Acidithiomicrobium' P2 was favoured towards the end of the runs and at 20%(w/w) solids possibly due to its tolerance to Ni. The use of gene abundance to evaluate biomass in the pulp provided complementary results to classical cell counts in the liquid phase, and suggested a key role of bacteria associated to mineral particles in iron oxidation. Scaling-up in 21-L STR at 20%(w/w) solids had no detrimental effect on bioleaching and confirmed metal extraction rates. 'Acidithiomicrobium' P2 and Sb. thermosulfidooxidans remained main actors. However, the biological activity was considerably reduced at 30%(w/w) solid concentration, which may be due to a too drastic environmental change for the bacteria to adapt to higher solid concentration. Efficient bioleaching of Sotkamo bioleaching residue at high solid concentration was demonstrated, as well as the robustness of the selected moderately thermophilic consortium, at laboratory and pilot scales.

[Source](https://doi.org/10.1016/j.resmic.2023.104112)

10.1016/j.resmic.2023.104112

---

## Multidimensional specialization and generalization are pervasive in soil prokaryotes.
 07.08.2023, _DJ Hernandez, KN Kiesewetter, BK Almeida, D Revillini, ME Afkhami_


Habitat specialization underpins biological processes from species distributions to speciation. However, organisms are often described as specialists or generalists based on a single niche axis, despite facing complex, multidimensional environments. Here, we analysed 236 environmental soil microbiomes across the United States and demonstrate that 90% of &gt;1,200 prokaryotes followed one of two trajectories: specialization on all niche axes (multidimensional specialization) or generalization on all axes (multidimensional generalization). We then documented that this pervasive multidimensional specialization/generalization had many ecological and evolutionary consequences. First, multidimensional specialization and generalization are highly conserved with very few transitions between these two trajectories. Second, multidimensional generalists dominated communities because they were 73 times more abundant than specialists. Lastly, multidimensional specialists played important roles in community structure with ~220% more connections in microbiome networks. These results indicate that multidimensional generalization and specialization are evolutionarily stable with multidimensional generalists supporting larger populations and multidimensional specialists playing important roles within communities, probably stemming from their overrepresentation among pollutant detoxifiers and nutrient cyclers. Taken together, we demonstrate that the vast majority of soil prokaryotes are restricted to one of two multidimensional niche trajectories, multidimensional specialization or multidimensional generalization, which then has far-reaching consequences for evolutionary transitions, microbial dominance and community roles.

[Source](https://doi.org/10.1038/s41559-023-02149-y)

10.1038/s41559-023-02149-y

---

## Metagenome Analysis of Speleothem Microbiome from Subterranean Cave Reveals Insight into Community Structure, Metabolic Potential, and BGCs Diversity.
 11.08.2023, _B Samanta, S Sharma, R Budhwar_


_Caves, Metagenome, Microbiota, Phylogeny, Sulfur, Bacteria_
The Borra caves, the second largest subterranean karst cave ecosystem in the Indian sub-continent, are located at the Ananthagiri hills of Araku Valley in the Alluri district of Andhra Pradesh, India. The present investigation applied a shotgun metagenomic approach to gain insights into the microbial community structure, metabolic potential, and biosynthetic gene cluster (BGC) diversity of the microbes colonizing the surface of the speleothems from the aphotic zone of Borra caves. The taxonomic analysis of the metagenome data illustrated that the speleothem-colonizing core microbial community was dominated mainly by Alpha-, Beta-, and Gamma-Proteobacteria, Actinobacteria, Firmicutes, and Bacteroidetes. The key energy metabolic pathways analysis provides strong evidence of chemolithoautotrophic and chemoheterotrophic modes of nutrition in the speleothem-colonizing microbial community. Metagenome data suggests that sulfur reducers and sulfur-disproportionating microbes might play a vital role in energy generation in this ecosystem. Our metagenome data also suggest that the dissimilatory nitrifiers and nitrifying denitrifiers might play an essential role in conserving nitrogen pools in the ecosystem. Furthermore, metagenome-wide BGCs mining retrieved 451 putative BGCs; NRPS was the most abundant (24%). Phylogenetic analysis of the C domain of NRPS showed that sequences were distributed across all six function categories of the known C domain, including several novel subclades. For example, a novel subclade had been recovered within the LCL domain clade as a sister subclade of immunosuppressant cyclosporin encoding C domain sequences. Our result suggested that subterranean cave microbiomes might be a potential reservoir of novel microbial metabolites.

[Source](https://doi.org/10.1007/s00284-023-03431-9)

10.1007/s00284-023-03431-9

---

## Effects of marine sediment as agricultural substrate on soil microbial diversity: an amplicon sequencing study

[Source](https://doi.org/10.1186/s40793-023-00519-4)

---

## New microbial tools to boost restoration and soil organic matters.
 08.08.2023, _T Sáez-Sandino, M Delgado-Baquerizo, E Egidi, BK Singh_


Anthropogenic activities are causing unprecedented rates of soil and ecosystem degradation, and the current restoration practices take decades and are prone to high rates of failure. Here we propose, the development and application of emerging microbiome tools that can potentially improve the contents and diversity of soil organic matters, enhancing the efficacy and consistency of restoration outcomes.

[Source](https://doi.org/10.1111/1751-7915.14325)

10.1111/1751-7915.14325

---

## Optimization of fecal sample homogenization for untargeted metabolomics.
 14.08.2023, _K Tarazona Carrillo, SL Nam, AP de la Mata, OM de Bruin, E Doukhanine, J Harynuk_


_Metabolomics, Reproducibility of Results, Metabolome, Feces, Solvents_
Fecal samples are highly complex and heterogeneous, containing materials at various stages of digestion. The heterogeneity and complexity of feces make stool metabolomics inherently challenging. The level of homogenization influences the outcome of the study, affecting the metabolite profiles and reproducibility; however, there is no consensus on how fecal samples should be prepared to overcome the topographical discrepancy and obtain data representative of the stool as a whole.

[Source](https://doi.org/10.1007/s11306-023-02036-4)

10.1007/s11306-023-02036-4

---

## metaSpectraST: an unsupervised and database-independent analysis workflow for metaproteomic MS/MS data using spectrum clustering.
 15.08.2023, _C Hao, JE Elias, PKH Lee, H Lam_


_Animals, Mice, Tandem Mass Spectrometry, Workflow, Proteomics, Microbiota, Peptides_
The high diversity and complexity of the microbial community make it a formidable challenge to identify and quantify the large number of proteins expressed in the community. Conventional metaproteomics approaches largely rely on accurate identification of the MS/MS spectra to their corresponding short peptides in the digested samples, followed by protein inference and subsequent taxonomic and functional analysis of the detected proteins. These approaches are dependent on the availability of protein sequence databases derived either from sample-specific metagenomic data or from public repositories. Due to the incompleteness and imperfections of these protein sequence databases, and the preponderance of homologous proteins expressed by different bacterial species in the community, this computational process of peptide identification and protein inference is challenging and error-prone, which hinders the comparison of metaproteomes across multiple samples.

[Source](https://doi.org/10.1186/s40168-023-01602-1)

10.1186/s40168-023-01602-1

---

## sccomp: Robust differential composition and variability analysis for single-cell data.
 10.08.2023, _S Mangiola, AJ Roth-Schulze, M Trussart, E Zozaya-Valdés, M Ma, Z Gao, AF Rubin, TP Speed, H Shim, AT Papenfuss_


_Genomics, Proteomics, Computer Simulation, Microbiota, Algorithms_
Cellular omics such as single-cell genomics, proteomics, and microbiomics allow the characterization of tissue and microbial community composition, which can be compared between conditions to identify biological drivers. This strategy has been critical to revealing markers of disease progression, such as cancer and pathogen infection. A dedicated statistical method for differential variability analysis is lacking for cellular omics data, and existing methods for differential composition analysis do not model some compositional data properties, suggesting there is room to improve model performance. Here, we introduce sccomp, a method for differential composition and variability analyses that jointly models data count distribution, compositionality, group-specific variability, and proportion mean-variability association, being aware of outliers. sccomp provides a comprehensive analysis framework that offers realistic data simulation and cross-study knowledge transfer. Here, we demonstrate that mean-variability association is ubiquitous across technologies, highlighting the inadequacy of the very popular Dirichlet-multinomial distribution. We show that sccomp accurately fits experimental data, significantly improving performance over state-of-the-art algorithms. Using sccomp, we identified differential constraints and composition in the microenvironment of primary breast cancer.

[Source](https://doi.org/10.1073/pnas.2203828120)

10.1073/pnas.2203828120

---

## Metagenome sequencing and 103 microbial genomes from ballast water and sediments.
 14.08.2023, _Z Xue, Y Han, W Tian, W Zhang_


_Archaea, Bacteria, Genome, Microbial, Metagenome, Metagenomics, Microbiota_
The great threat of microbes carried by ballast water calls for figuring out the species composition of the ballast-tank microbial community, where the dark, cold, and anoxic tank environment might select special taxa. In this study, we reconstructed 103 metagenome-assembled genomes (MAGs), including 102 bacteria and one archaea, from four vessels on international voyages. Of these MAGs, 60 were 'near complete' (completeness &gt;90%), 34 were &gt;80% complete, and nine were &gt;75% complete. Phylogenomic analysis revealed that over 70% (n = 74) of these MAGs represented new taxa at different taxonomical levels, including one order, three families, 12 genera, and 58 species. The species composition of these MAGs was most consistent with the previous reports, with the most abundant phyla being Proteobacteria (n = 69), Bacteroidota (n = 17), and Actinobacteriota (n = 7). These draft genomes provided novel data on species diversity and function in the ballast-tank microbial community, which will facilitate ballast water and sediments management.

[Source](https://doi.org/10.1038/s41597-023-02447-x)

10.1038/s41597-023-02447-x

---

