## Examining horizontal gene transfer in microbial communities.
 13.04.2021, _IL Brito_


Bacteria acquire novel DNA through horizontal gene transfer (HGT), a process that enables an organism to rapidly adapt to changing environmental conditions, provides a competitive edge and potentially alters its relationship with its host. Although the HGT process is routinely exploited in laboratories, there is a surprising disconnect between what we know from laboratory experiments and what we know from natural environments, such as the human gut microbiome. Owing to a suite of newly available computational algorithms and experimental approaches, we have a broader understanding of the genes that are being transferred and are starting to understand the ecology of HGT in natural microbial communities. This Review focuses on these new technologies, the questions they can address and their limitations. As these methods are applied more broadly, we are beginning to recognize the full extent of HGT possible within a microbiome and the punctuated dynamics of HGT, specifically in response to external stimuli. Furthermore, we are better characterizing the complex selective pressures on mobile genetic elements and the mechanisms by which they interact with the bacterial host genome.

[Source](https://doi.org/10.1038/s41579-021-00534-7)

10.1038/s41579-021-00534-7

---

## Evidence for a billion-years arms race between viruses, virophages and eukaryotes

[Source](https://doi.org/10.1101/2021.04.20.440574)

---

## Unifying community detection across scales from genomes to landscapes

[Source](https://doi.org/10.1111/oik.08393)

---

## Adaptive ecological processes and metabolic independence drive microbial colonization and resilience in the human gut

[Source](https://doi.org/10.1101/2021.03.02.433653)

---

## Diversity and characteristics of raw milk microbiota from Korean dairy farms using metagenomic and culturomic analysis

[Source](https://doi.org/10.1016/j.foodcont.2021.108160)

---

## Reclassification of seven honey bee symbiont strains as <em>Bombella apis</em>

[Source](https://doi.org/10.1101/2020.05.06.081802)

---

## Symbiotic associations of the deepest recorded photosynthetic scleractinian coral (172 m depth)

[Source](https://doi.org/10.1038/s41396-020-00857-y)

---

## Metagenomic shotgun sequencing reveals host species as an important driver of virome composition in mosquitoes.
 23.04.2021, _P Thongsripong, JA Chandler, P Kittayapong, BA Wilcox, DD Kapan, SN Bennett_


High-throughput nucleic acid sequencing has greatly accelerated the discovery of viruses in the environment. Mosquitoes, because of their public health importance, are among those organisms whose viromes are being intensively characterized. Despite the deluge of sequence information, our understanding of the major drivers influencing the ecology of mosquito viromes remains limited. Using methods to increase the relative proportion of microbial RNA coupled with RNA-seq we characterize RNA viruses and other symbionts of three mosquito species collected along a rural to urban habitat gradient in Thailand. The full factorial study design allows us to explicitly investigate the relative importance of host species and habitat in structuring viral communities. We found that the pattern of virus presence was defined primarily by host species rather than by geographic locations or habitats. Our result suggests that insect-associated viruses display relatively narrow host ranges but are capable of spreading through a mosquito population at the geographical scale of our study. We also detected various single-celled and multicellular microorganisms such as bacteria, alveolates, fungi, and nematodes. Our study emphasizes the importance of including ecological information in viromic studies in order to gain further insights into viral ecology in systems where host specificity is driving both viral ecology and evolution.

[Source](https://doi.org/10.1038/s41598-021-87122-0)

10.1038/s41598-021-87122-0

---

## Aerobic and anaerobic iron oxidizers together drive denitrification and carbon cycling at marine iron-rich hydrothermal vents.
 23.04.2021, _SM McAllister, R Vandzura, JL Keffer, SW Polson, CS Chan_


In principle, iron oxidation can fuel significant primary productivity and nutrient cycling in dark environments such as the deep sea. However, we have an extremely limited understanding of the ecology of iron-based ecosystems, and thus the linkages between iron oxidation, carbon cycling, and nitrate reduction. Here we investigate iron microbial mats from hydrothermal vents at Lō'ihi Seamount, Hawai'i, using genome-resolved metagenomics and metatranscriptomics to reconstruct potential microbial roles and interactions. Our results show that the aerobic iron-oxidizing Zetaproteobacteria are the primary producers, concentrated at the oxic mat surface. Their fixed carbon supports heterotrophs deeper in the mat, notably the second most abundant organism, Candidatus Ferristratum sp. (uncultivated gen. nov.) from the uncharacterized DTB120 phylum. Candidatus Ferristratum sp., described using nine high-quality metagenome-assembled genomes with similar distributions of genes, expressed nitrate reduction genes narGH and the iron oxidation gene cyc2 in situ and in response to Fe(II) in a shipboard incubation, suggesting it is an anaerobic nitrate-reducing iron oxidizer. Candidatus Ferristratum sp. lacks a full denitrification pathway, relying on Zetaproteobacteria to remove intermediates like nitrite. Thus, at Lō'ihi, anaerobic iron oxidizers coexist with and are dependent on aerobic iron oxidizers. In total, our work shows how key community members work together to connect iron oxidation with carbon and nitrogen cycling, thus driving the biogeochemistry of exported fluids.

[Source](https://doi.org/10.1038/s41396-020-00849-y)

10.1038/s41396-020-00849-y

---

## Microbial niche differentiation explains nitrite oxidation in marine oxygen minimum zones.
 23.04.2021, _X Sun, C Frey, E Garcia-Robledo, A Jayakumar, BB Ward_


Nitrite is a pivotal component of the marine nitrogen cycle. The fate of nitrite determines the loss or retention of fixed nitrogen, an essential nutrient for all organisms. Loss occurs via anaerobic nitrite reduction to gases during denitrification and anammox, while retention occurs via nitrite oxidation to nitrate. Nitrite oxidation is usually represented in biogeochemical models by one kinetic parameter and one oxygen threshold, below which nitrite oxidation is set to zero. Here we find that the responses of nitrite oxidation to nitrite and oxygen concentrations vary along a redox gradient in a Pacific Ocean oxygen minimum zone, indicating niche differentiation of nitrite-oxidizing assemblages. Notably, we observe the full inhibition of nitrite oxidation by oxygen addition and nitrite oxidation coupled with nitrogen loss in the absence of oxygen consumption in samples collected from anoxic waters. Nitrite-oxidizing bacteria, including novel clades with high relative abundance in anoxic depths, were also detected in the same samples. Mechanisms corresponding to niche differentiation of nitrite-oxidizing bacteria across the redox gradient are considered. Implementing these mechanisms in biogeochemical models has a significant effect on the estimated fixed nitrogen budget.

[Source](https://doi.org/10.1038/s41396-020-00852-3)

10.1038/s41396-020-00852-3

---

## Morphological complexity affects the diversity of marine microbiomes.
 23.04.2021, _MA Lemay, MY Chen, F Mazel, KR Hind, S Starko, PJ Keeling, PT Martone, LW Parfrey_


Large eukaryotes support diverse communities of microbes on their surface-epibiota-that profoundly influence their biology. Alternate factors known to structure complex patterns of microbial diversity-host evolutionary history and ecology, environmental conditions and stochasticity-do not act independently and it is challenging to disentangle their relative effects. Here, we surveyed the epibiota from 38 sympatric seaweed species that span diverse clades and have convergent morphology, which strongly influences seaweed ecology. Host identity explains most of the variation in epibiont communities and deeper host phylogenetic relationships (e.g., genus level) explain a small but significant portion of epibiont community variation. Strikingly, epibiota community composition is significantly influenced by host morphology and epibiota richness increases with morphological complexity of the seaweed host. This effect is robust after controlling for phylogenetic non-independence and is strongest for crustose seaweeds. We experimentally validated the effect of host morphology by quantifying bacterial community assembly on latex sheets cut to resemble three seaweed morphologies. The patterns match those observed in our field survey. Thus, biodiversity increases with habitat complexity in host-associated microbial communities, mirroring patterns observed in animal communities. We suggest that host morphology and structural complexity are underexplored mechanisms structuring microbial communities.

[Source](https://doi.org/10.1038/s41396-020-00856-z)

10.1038/s41396-020-00856-z

---

## Abundance and Microbial Diversity from Surface to Deep Water Layers Over the Rio Grande Rise, South Atlantic

[Source](https://doi.org/10.1101/2021.04.22.441028)

---

## Supervised extraction of near-complete genomes from metagenomic samples: A new service in PATRIC.
 22.04.2021, _B Parrello, R Butler, P Chlenski, GD Pusch, R Overbeek_


Large amounts of metagenomically-derived data are submitted to PATRIC for analysis. In the future, we expect even more jobs submitted to PATRIC will use metagenomic data. One in-demand use case is the extraction of near-complete draft genomes from assembled contigs of metagenomic origin. The PATRIC metagenome binning service utilizes the PATRIC database to furnish a large, diverse set of reference genomes. We provide a new service for supervised extraction and annotation of high-quality, near-complete genomes from metagenomically-derived contigs. Reference genomes are assigned to putative draft genome bins based on the presence of single-copy universal marker roles in the sample, and contigs are sorted into these bins by their similarity to reference genomes in PATRIC. Each set of binned contigs represents a draft genome that will be annotated by RASTtk in PATRIC. A structured-language binning report is provided containing quality measurements and taxonomic information about the contig bins. The PATRIC metagenome binning service emphasizes extraction of high-quality genomes for downstream analysis using other PATRIC tools and services. Due to its supervised nature, the binning service is not appropriate for mining novel or extremely low-coverage genomes from metagenomic samples.

[Source](https://doi.org/10.1371/journal.pone.0250092)

10.1371/journal.pone.0250092

---

## A roadmap for metagenomic enzyme discovery

[Source](https://doi.org/10.1039/D1NP00006C)

---

## Synergistic effect of short- and long-read sequencing on functional meta-omics

[Source](https://doi.org/10.1101/2021.04.22.440869)

---

## H3ABioNet genomic medicine and microbiome data portals hackathon proceedings

[Source](https://doi.org/10.1093/database/baab016)

---

