## Mechanoelectronic stimulation of autologous extracellular vesicle biosynthesis implant for gut microbiota modulation.
 21.04.2024, _S Wan, K Wang, P Huang, X Guo, W Liu, Y Li, J Zhang, Z Li, J Song, W Yang, X Zhang, X Ding, DT Leong, L Wang_


Pathogenic gut microbiota is responsible for a few debilitating gastrointestinal diseases. While the host immune cells do produce extracellular vesicles to counteract some deleterious effects of the microbiota, the extracellular vesicles are of insufficient doses and at unreliable exposure times. Here we use mechanical stimulation of hydrogel-embedded macrophage in a bioelectronic controller that on demand boost production of up to 20 times of therapeutic extracellular vesicles to ameliorate the microbes' deleterious effects in vivo. Our miniaturized wireless bioelectronic system termed inducible mechanical activation for in-situ and sustainable generating extracellular vesicles (iMASSAGE), leverages on wireless electronics and responsive hydrogel to impose mechanical forces on macrophages to produce extracellular vesicles that rectify gut microbiome dysbiosis and ameliorate colitis. This in vivo controllable extracellular vesicles-produced system holds promise as platform to treat various other diseases.

[Source](https://doi.org/10.1038/s41467-024-47710-w)

10.1038/s41467-024-47710-w

---

## Oral bacterial insights from a comparative study between healthy and comorbid diseased human individuals.
 21.04.2024, _A Ravichandran, J Sivapackiam, S Periasamy_


The human oral cavity is colonized by a diverse microbial community, which includes both native and transient colonizers. The microbial composition is crucial for maintaining oral homeostasis, but due to overgrowth or imbalances of these microbial communities, dysbiosis can occur. There is a lack of understanding of the research of native and transient colonizers in the oral cavity of the Indian subpopulation Therefore, in our present study, we explored the role and prevalence of transient and native colonizers between healthy and comorbid oral diseased human individuals. Culture-dependent techniques and culture-independent 16S r DNA metagenomic analyses were employed to isolate and study the interactions of native and transient colonizers from human oral samples. Among the 66 human individuals of both healthy and comorbid individuals, the most abundant isolate was found to be Bacillus amyloliquefaciens MCC 4424. In addition, the more prevalent culturable isolate from the healthy samples was Streptococcus salivarius MTCC 13009, whereas in comorbid samples Staphylococcus pasteuri MTCC 13076, Rothia dentocariosa MTCC 13010 and Pseudomonas aeruginosa MTCC 13077 were prevalent to a greater extent. 16S rDNA metagenomic analyses revealed the prevalence and abundance of genera such as Bacteroidetes and Proteobacteria in healthy individuals; consequently, Fusobacteria and Firmicutes were observed mostly in comorbid individuals. The significant differences in bacterial population density were observed in terms of the Shannon index (p = 0.5145) and Simpson index (p = 0.9061) between the healthy and comorbid groups. B. amyloliquefaciens MCC 4424 exhibits antagonistic behavior when grown as a dual-species with native and transient colonizers. This result is very consistent with the findings of antibiofilm studies using confocal laser scanning microscopy, which revealed a significant reduction in biofilm biovolume (73 %) and maximum thickness (80 %) and an increase in the rough coefficient of biofilms (30 %). Our data suggested that B. amyloliquefaciens MCC 4424 can be a native colonizer of Indian sub-populations. It may act as a novel candidate for oral healthcare applications and greatly aids in the regulation of transient species in the oral cavity.

[Source](https://doi.org/10.1016/j.micpath.2024.106643)

10.1016/j.micpath.2024.106643

---

## Dietary fiber guar gum-induced shift in gut microbiota metabolism and intestinal immune activity enhances susceptibility to colonic inflammation.
 21.04.2024, _D Paudel, DVT Nair, S Tian, F Hao, UK Goand, G Joseph, E Prodes, Z Chai, CEM Robert, B Chassaing, AD Patterson, V Singh_


_Animals, Mice, Gastrointestinal Microbiome, Interleukin-18, Inflammation, Colitis, Dietary Fiber, Lactic Acid, Succinates, Galactans, Mannans, Plant Gums_
With an increasing interest in dietary fibers (DFs) to promote intestinal health and the growth of beneficial gut bacteria, there is a continued rise in the incorporation of refined DFs in processed foods. It is still unclear how refined fibers, such as guar gum, affect the gut microbiota activity and pathogenesis of inflammatory bowel disease (IBD). Our study elucidated the effect and underlying mechanisms of guar gum, a fermentable DF (FDF) commonly present in a wide range of processed foods, on colitis development. We report that guar gum containing diet (GuD) increased the susceptibility to colonic inflammation. Specifically, GuD-fed group exhibited severe colitis upon dextran sulfate sodium (DSS) administration, as evidenced by reduced body weight, diarrhea, rectal bleeding, and shortening of colon length compared to cellulose-fed control mice. Elevated levels of pro-inflammatory markers in both serum [serum amyloid A (SAA), lipocalin 2 (Lcn2)] and colon (Lcn2) and extensive disruption of colonic architecture further affirmed that GuD-fed group exhibited more severe colitis than control group upon DSS intervention. Amelioration of colitis in GuD-fed group pre-treated with antibiotics suggest a vital role of intestinal microbiota in GuD-mediated exacerbation of intestinal inflammation. Gut microbiota composition and metabolite analysis in fecal and cecal contents, respectively, revealed that guar gum primarily enriches Actinobacteriota, specifically Bifidobacterium. Guar gum also altered multiple genera belonging to phyla Bacteroidota and Firmicutes. Such shift in gut microbiota composition favored luminal accumulation of intermediary metabolites succinate and lactate in the GuD-fed mice. Colonic IL-18 and tight junction markers were also decreased in the GuD-fed group. Importantly, GuD-fed mice pre-treated with recombinant IL-18 displayed attenuated colitis. Collectively, unfavorable changes in gut microbiota activity leading to luminal accumulation of lactate and succinate, reduced colonic IL-18, and compromised gut barrier function following guar gum feeding contributed to increased colitis susceptibility.

[Source](https://doi.org/10.1080/19490976.2024.2341457)

10.1080/19490976.2024.2341457

---

## Highly specific vaginal microbiome signature for gynecological cancers.
 19.04.2024, _M Han, N Wang, W Han, X Liu, T Sun, J Xu_


To investigate the vaginal microbiota signature of patients with gynecologic cancer and evaluate its diagnostic biomarker potential. We incorporated vaginal 16S rRNA-seq data from 529 women and utilized VSEARCH to analyze the raw data. α-Diversity was evaluated utilizing the Chao1, Shannon, and Simpson indices, and β-diversity was evaluated through principal component analysis using Bray-Curtis distances. Linear discriminant analysis effect size (LEfSe) was utilized to determine species differences between groups. A bacterial co-abundance network was constructed utilizing Spearman correlation analysis. A random forest model of gynecologic tumor risk based on genus was constructed and validated to test its diagnostic efficacy. In gynecologic cancer patients, vaginal α-diversity was significantly greater than in controls, and vaginal β-diversity was significantly separated from that of controls; there was no correlation between these characteristics and menopause status among the subject women. Women diagnosed with gynecological cancer exhibited a reduction in the abundance of vaginal Firmicutes and Lactobacillus, while an increase was observed in the proportions of Bacteroidetes, Proteobacteria, Prevotella, Streptococcus, and Anaerococcus. A random forest model constructed based on 56 genus achieved high accuracy (area under the curve = 84.96%) in gynecological cancer risk prediction. Furthermore, there were discrepancies observed in the community complexity of co-abundance networks between gynecologic cancer patients and the control group. Our study provides evidence that women with gynecologic cancer have a unique vaginal flora structure and microorganisms may be involved in the gynecologic carcinogenesis process. A gynecological cancer risk prediction model based on characteristic genera has good diagnostic value.

[Source](https://doi.org/10.1515/biol-2022-0850)

10.1515/biol-2022-0850

---

## Incretin hormones, obesity and gut microbiota.
 18.04.2024, _G Angelini, S Russo, G Mingrone_


Over the past 40 years, the prevalence of obesity has risen dramatically, reaching epidemic proportions. By 2030 the number of people affected by obesity will reach 1.12 billion worldwide. Gastrointestinal hormones, namely incretins, play a vital role in the pathogenesis of obesity and its comorbidities. GIP (glucose-dependent insulinotropic polypeptide) and GLP-1 (glucagon-like peptide-1), which are secreted from the intestine after nutrient intake and stimulate insulin secretion from pancreatic β cells, influence lipid metabolism, gastric empting, appetite and body weight. The gut microbiota plays an important role in various metabolic conditions, including obesity and type 2 diabetes and influences host metabolism through the interaction with enteroendocrine cells that modulate incretins secretion. Gut microbiota metabolites, such as short-chain fatty acids (SCFAs) and indole, directly stimulate the release of incretins from colonic enteroendocrine cells influencing host satiety and food intake. Moreover, bariatric surgery and incretin-based therapies are associated with increase gut bacterial richness and diversity. Understanding the role of incretins, gut microbiota, and their metabolites in regulating metabolic processes is crucial to develop effective strategies for the management of obesity and its associated comorbidities.

[Source](https://doi.org/10.1016/j.peptides.2024.171216)

10.1016/j.peptides.2024.171216

---

## Role of imbalanced gut microbiota in promoting CRC metastasis: from theory to clinical application.
 21.04.2024, _S Fan, L Zhou, W Zhang, D Wang, D Tang_


Metastasis poses a major challenge in colorectal cancer (CRC) treatment and remains a primary cause of mortality among patients with CRC. Recent investigations have elucidated the involvement of disrupted gut microbiota homeostasis in various facets of CRC metastasis, exerting a pivotal influence in shaping the metastatic microenvironment, triggering epithelial-mesenchymal transition (EMT), and so on. Moreover, therapeutic interventions targeting the gut microbiota demonstrate promise in enhancing the efficacy of conventional treatments for metastatic CRC (mCRC), presenting novel avenues for mCRC clinical management. Grounded in the "seed and soil" hypothesis, this review consolidates insights into the mechanisms by which imbalanced gut microbiota promotes mCRC and highlights recent strides in leveraging gut microbiota modulation for the clinical prevention and treatment of mCRC. Emphasis is placed on the considerable potential of manipulating gut microbiota within clinical settings for managing mCRC.

[Source](https://doi.org/10.1186/s12964-024-01615-9)

10.1186/s12964-024-01615-9

---

## The Role of the Gut Microbiome in Health and Disease in the Elderly.
 20.04.2024, _LA Chen, K Boyle_


Growing evidence supports the contribution of age in the composition and function of the gut microbiome, with specific findings associated with health in old age and longevity.

[Source](https://doi.org/10.1007/s11894-024-00932-w)

10.1007/s11894-024-00932-w

---

## Application of Microbiome-Based Therapies in Chronic Respiratory Diseases.
 18.04.2024, _SH Lee, JH Lee, SW Lee_


The application of microbiome-based therapies in various areas of human disease has recently increased. In chronic respiratory disease, microbiome-based clinical applications are considered compelling options due to the limitations of current treatments. The lung microbiome is ecologically dynamic and affected by various conditions, and dysbiosis is associated with disease severity, exacerbation, and phenotype as well as with chronic respiratory disease endotype. However, it is not easy to directly modulate the lung microbiome. Additionally, studies have shown that chronic respiratory diseases can be improved by modulating gut microbiome and administrating metabolites. Although the composition, diversity, and abundance of the microbiome between the gut and lung are considerably different, modulation of the gut microbiome could improve lung dysbiosis. The gut microbiome influences that of the lung via bacterial-derived components and metabolic degradation products, including short-chain fatty acids. This phenomenon might be associated with the cross-talk between the gut microbiome and lung, called gut-lung axis. There are multiple alternatives to modulate the gut microbiome, such as prebiotics, probiotics, and postbiotics ingestion and fecal material transplantation. Several studies have shown that high-fiber diets, for example, present beneficial effects through the production of short-chain fatty acids. Additionally, genetically modified probiotics to secrete some beneficial molecules might also be utilized to treat chronic respiratory diseases. Further studies on microbial modulation to regulate immunity and potentiate conventional pharmacotherapy will improve microbiome modulation techniques, which will develop as a new therapeutic area in chronic respiratory diseases.

[Source](https://doi.org/10.1007/s12275-024-00124-1)

10.1007/s12275-024-00124-1

---

## Apical root canal microbiome associated with primary and posttreatment apical periodontitis: A systematic review.
 18.04.2024, _JF Siqueira, WO Silva, K Romeiro, LF Gominho, FRF Alves, IN Rôças_


Microorganisms colonizing the apical root canal system are conceivably the ones directly involved with the causation and maintenance of apical periodontitis.

[Source](https://doi.org/10.1111/iej.14071)

10.1111/iej.14071

---

## An insider's perspective about the pathogenic relevance of gut bacterial transportomes.
 18.04.2024, _H Zafar, MH Saier_


The gut microbiome is integral to host health, hosting complex interactions between the host and numerous microbial species in the gastrointestinal tract. Key among the molecular mechanisms employed by gut bacteria are transportomes, consisting of diverse transport proteins crucial for bacterial adaptation to the dynamic, nutrient-rich environment of the mammalian gut. These transportomes facilitate the movement of a wide array of molecules, impacting both the host and the microbial community.

[Source](https://doi.org/10.1159/000538779)

10.1159/000538779

---

## Butyrate as a potential therapeutic agent for neurodegenerative disorders.
 19.04.2024, _P Chakraborty, HKAH Gamage, AS Laird_


Maintaining an optimum microbial community within the gastrointestinal tract is intricately linked to human metabolic, immune and brain health. Disturbance to these microbial populations perturbs the production of vital bioactive compounds synthesised by the gut microbiome, such as short-chain fatty acids (SCFAs). Of the SCFAs, butyrate is known to be a major source of energy for colonocytes and has valuable effects on the maintenance of intestinal epithelium and blood brain barrier integrity, gut motility and transit, anti-inflammatory effects, and autophagy induction. Inducing endogenous butyrate production is likely to be beneficial for gut-brain homeostasis and for optimal neuronal function. For these reasons, butyrate has gained interest as a potential therapy for not only metabolic and immunological disorders, but also conditions related to the brain, including neurodegenerative diseases. While direct and indirect sources of butyrate, including prebiotics, probiotics, butyrate pro-drugs and glucosidase inhibitors, offer a promising therapeutic avenue, their efficacy and dosage in neurodegenerative conditions remain largely unknown. Here, we review current literature on effects of butyrate relevant to neuronal function, the impact of butyrate in a range of neurodegenerative diseases and related treatments that may have potential for the treatment of neurodegenerative diseases.

[Source](https://doi.org/10.1016/j.neuint.2024.105745)

10.1016/j.neuint.2024.105745

---

## Adherence to the Mediterranean diet can beneficially affect the gut microbiota composition: a systematic review

[Source](https://doi.org/10.1186/s12920-024-01861-3)

---

## Hi-C metagenomics facilitate comparative genome analysis of bacteria and yeast from spontaneous beer and cider.
 18.04.2024, _IV Sonets, MA Solovyev, VA Ivanova, PA Vasiluev, AV Kachalkin, SD Ochkalova, AI Korobeynikov, SV Razin, SV Ulianov, AV Tyakht_


Sequence-based analysis of fermented foods and beverages' microbiomes offers insights into their impact on taste and consumer health. High-throughput metagenomics provide detailed taxonomic and functional community profiling, but bacterial and yeast genome reconstruction and mobile genetic elements tracking are to be improved. We established a pipeline for exploring fermented foods microbiomes using metagenomics coupled with chromosome conformation capture (Hi-C metagenomics). The approach was applied to analyze a collection of spontaneously fermented beers and ciders (n = 12). The Hi-C reads were used to reconstruct the metagenome-assembled genomes (MAGs) of bacteria and yeasts facilitating subsequent comparative genomic analysis, assembly scaffolding and exploration of "plasmid-bacteria" links. For a subset of beverages, yeasts were isolated and characterized phenotypically. The reconstructed Hi-C MAGs primarily belonged to the Lactobacillaceae family in beers, along with Acetobacteraceae and Enterobacteriaceae in ciders, exhibiting improved quality compared to conventional metagenomic MAGs. Comparative genomic analysis of Lactobacillaceae Hi-C MAGs revealed clustering by niche and suggested genetic determinants of survival and probiotic potential. For Pediococcus damnosus, Hi-C-based networks of contigs enabled linking bacteria with plasmids. Analyzing phylogeny and accessory genes in the context of known reference genomes offered insights into the niche specialization of beer lactobacilli. The subspecies-level diversity of cider Tatumella spp. was disentangled using a Hi-C-based graph. We obtained highly complete yeast Hi-C MAGs primarily represented by Brettanomyces and Saccharomyces, with Hi-C-facilitated chromosome-level genome assembly for the former. Utilizing Hi-C metagenomics to unravel the genomic content of individual species can provide a deeper understanding of the ecological interactions within the food microbiome, aid in bioprospecting beneficial microorganisms, improving quality control and improving innovative fermented products.

[Source](https://doi.org/10.1016/j.fm.2024.104520)

10.1016/j.fm.2024.104520

---

## New evidence for gut-muscle axis: Lactic acid bacteria-induced gut microbiota regulates duck meat flavor.
 18.04.2024, _L Xu, T Mao, M Xia, W Wu, J Chen, C Jiang, T Zeng, Y Tian, L Lu, Z Cai_


The interaction between gut microbiota and muscles through the gut-muscle axis has received increasing attention. This study attempted to address existing research gaps by investigating the effects of gut microbiota on meat flavor. Specifically, lactic acid bacteria were administered to ducks, and the results of e-nose and e-tongue showed significantly enhanced meat flavor in the treatment group. Further analyses using GC-MS revealed an increase in 6 characteristic volatile flavor compounds, including pentanal, hexanal, heptanal, 1-octen-3-ol, 2,3-octanedione, and 2-pentylfuran. Linoleic acid was identified as the key fatty acid that influences meat flavor. Metagenomic and transcriptomic results further confirmed that cecal microbiota affects the duck meat flavor by regulating the metabolic pathways of fatty acids and amino acids, especially ACACB was related to fatty acid biosynthesis and ACAT2, ALDH1A1 with fatty acid degradation. This study sheds light on a novel approach to improving the flavor of animal-derived food.

[Source](https://doi.org/10.1016/j.foodchem.2024.139354)

10.1016/j.foodchem.2024.139354

---

## Safety evaluation and effects of dietary phlorotannins on the growth, health, and intestinal microbiota of <em>Litopenaeus vannamei</em>

[Source](https://doi.org/10.1016/j.fsi.2024.109569)

---

## A longitudinal study of queen health in honey bees reveals tissue specific response to seasonal changes and pathogen pressure.
 21.04.2024, _DC Copeland, VA Ricigliano, BM Mott, OL Kortenkamp, RJ Erickson, J Gorrochategui-Ortega, KE Anderson_


The health of honey bee queens is crucial for colony success, particularly during stressful periods like overwintering. To accompany a previous longitudinal study of colony and worker health, we explored niche-specific gut microbiota, host gene expression, and pathogen prevalence in honey bee queens overwintering in a warm southern climate. We found differential gene expression and bacterial abundance with respect to various pathogens throughout the season. Biologically older queens had larger microbiotas, particularly enriched in Bombella and Bifidobacterium. Both Deformed Wing Virus A and B subtypes were highest in the fat body tissue in January, correlating with colony Varroa levels, and Deformed Wing Virus titers in workers. High viral titers in queens were associated with decreased vitellogenin expression, suggesting a potential trade-off between immune function and reproductive capacity. Additionally, we found a complex and dynamic relationship between these viral loads and immune gene expression, indicating a possible breakdown in the coordinated immune response as the season progressed. Our study also revealed a potential link between Nosema and Melissococcus plutonius infections in queens, demonstrating that seasonal opportunism is not confined to just workers. Overall, our findings highlight the intricate interplay between pathogens, metabolic state, and immune response in honey bee queens. Combined with worker and colony-level metrics from the same colonies, our findings illustrate the social aspect of queen health and resilience over the winter dearth.

[Source](https://doi.org/10.1038/s41598-024-58883-1)

10.1038/s41598-024-58883-1

---

## The microbiome of a Pacific moon jellyfish Aurelia coerulea.
 20.04.2024, _AH Ohdera, M Mansbridge, M Wang, P Naydenkov, B Kamel, L Goentoro_


The impact of microbiome in animal physiology is well appreciated, but characterization of animal-microbe symbiosis in marine environments remains a growing need. This study characterizes the microbial communities associated with the moon jellyfish Aurelia coerulea, first isolated from the East Pacific Ocean and has since been utilized as an experimental system. We find that the microbiome of this Pacific Aurelia culture is dominated by two taxa, a Mollicutes and Rickettsiales. The microbiome is stable across life stages, although composition varies. Mining the host sequencing data, we assembled the bacterial metagenome-assembled genomes (MAGs). The bacterial MAGs are highly reduced, and predict a high metabolic dependence on the host. Analysis using multiple metrics suggest that both bacteria are likely new species. We therefore propose the names Ca. Mariplasma lunae (Mollicutes) and Ca. Marinirickettsia aquamalans (Rickettsiales). Finally, comparison with studies of Aurelia from other geographical populations suggests the association with Ca. Mariplasma lunae occurs in Aurelia from multiple geographical locations. The low-diversity microbiome of Aurelia provides a relatively simple system to study host-microbe interactions.

[Source](https://doi.org/10.1371/journal.pone.0298002)

10.1371/journal.pone.0298002

---

## The rhizosphere and root selections intensify fungi-bacteria interaction in abiotic stress-resistant plants.
 20.04.2024, _F Huang, M Lei, W Li_


The microbial communities, inhabiting around and in plant roots, are largely influenced by the compartment effect, and in turn, promote the growth and stress resistance of the plant. However, how soil microbes are selected to the rhizosphere, and further into the roots is still not well understood. Here, we profiled the fungal, bacterial communities and their interactions in the bulk soils, rhizosphere soils and roots of eleven stress-resistant plant species after six months of growth. The results showed that the root selection (from the rhizosphere soils to the roots) was stronger than the rhizosphere selection (from the bulk soils to the rhizosphere soils) in: (1) filtering stricter on the fungal (28.5% to 40.1%) and bacterial (48.9% to 68.1%) amplicon sequence variants (ASVs), (2) depleting more shared fungal (290 to 56) and bacterial (691 to 2) ASVs measured by relative abundance, and (3) increasing the significant fungi-bacteria crosskingdom correlations (142 to 110). In addition, the root selection, but not the rhizosphere selection, significantly increased the fungi to bacteria ratios (f:b) of the observed species and shannon diversity index, indicating unbalanced effects to the fungal and bacteria communities exerted by the root selection. Based on the results of network analysis, the unbalanced root selection effects were associated with increased numbers of negative interaction (140 to 99) and crosskingdom interaction (123 to 92), suggesting the root selection intensifies the negative fungi-bacteria interactions in the roots. Our findings provide insights into the complexity of crosskingdom interactions and improve the understanding of microbiome assembly in the rhizosphere and roots.

[Source](https://doi.org/10.7717/peerj.17225)

10.7717/peerj.17225

---

## Exploring microbiome and plankton responses and interactions in the mangrove ecosystem through eDNA and network analysis.
 19.04.2024, _J Meng, F Xu, H Yang, X Li, P Zhao_


The comprehensive analysis of multiple biological communities is essential for assessing diversities within mangrove ecosystems, yet such studies are infrequent. Environmental DNA (eDNA) facilitates the simultaneous exploration of organisms across various levels within a single ecosystem. In this investigation, 16S rRNA, cytochrome C oxidase I (COI), and Mito-fish primers were employed to characterize the microbiome, eukaryotic plankton, and fish communities, along with their intricate interactions, across 24 samples from three Chinese mangrove reservoirs. The resulting dataset encompasses 3779 taxonomic groups (genus level), spanning from the microbiome to vertebrates. Diversity analysis unveiled a higher level of stability in the microbiome community compared to plankton, underscoring the superior site-specificity of plankton. The association analysis revealed that biodiversity was primarily affected by temperature, turbidity, and fluorescent dissolved organic matter (fDOM). Notably, the physicochemical factors, turbidity, and fDOM had a more pronounced impact on the microbiome than on plankton, explaining their distinct sensitivities to site-specific conditions. Network analysis constructed 15 biological interaction subnetworks representing various community connections. The most connected genera in each subnetwork, highly responsive to different environmental factors, could serve as potential indicators of distinct ecosystem states. In summary, our findings represent the first comparison of the response sensitivities of different communities and the construction of their interaction networks in mangrove environments. These results contribute valuable insights into marine ecosystem dynamics and the role of environmental factors in shaping biodiversity.

[Source](https://doi.org/10.1016/j.scitotenv.2024.172581)

10.1016/j.scitotenv.2024.172581

---

## Quantifying genome specific carbon fixation in a 750 meter deep subsurface hydrothermal microbial community

[Source](https://doi.org/10.1093/femsec/fiae062)

---

## A generic approach to infer community-level fitness of microbial genes.
 18.04.2024, _T Wang, A Weiss, L You_


The gene content in a metagenomic pool defines the function potential of a microbial community. Natural selection, operating on the level of genomes or genes, shapes the evolution of community functions by enriching some genes while depriving the others. Despite the importance of microbiomes in the environment and health, a general metric to evaluate the community-wide fitness of microbial genes remains lacking. In this work, we adapt the classic neutral model of species and use it to predict how the abundances of different genes will be shaped by selection, regardless of at which level the selection acts. We establish a simple metric that quantitatively infers the average survival capability of each gene in a microbiome. We then experimentally validate the predictions using synthetic communities of barcoded Escherichia coli strains undergoing neutral assembly and competition. We further show that this approach can be applied to publicly available metagenomic datasets to gain insights into the environment-function interplay of natural microbiomes.

[Source](https://doi.org/10.1073/pnas.2318380121)

10.1073/pnas.2318380121

---

## Randomized feature selection based semi-supervised latent Dirichlet allocation for microbiome analysis.
 20.04.2024, _N Pais, N Ravishanker, S Rajasekaran, G Weinstock, DB Tran_


_Humans, Algorithms, Microbiota, Gastrointestinal Microbiome_
Health and disease are fundamentally influenced by microbial communities and their genes (the microbiome). An in-depth analysis of microbiome structure that enables the classification of individuals based on their health can be crucial in enhancing diagnostics and treatment strategies to improve the overall well-being of an individual. In this paper, we present a novel semi-supervised methodology known as Randomized Feature Selection based Latent Dirichlet Allocation (RFSLDA) to study the impact of the gut microbiome on a subject's health status. Since the data in our study consists of fuzzy health labels, which are self-reported, traditional supervised learning approaches may not be suitable. As a first step, based on the similarity between documents in text analysis and gut-microbiome data, we employ Latent Dirichlet Allocation (LDA), a topic modeling approach which uses microbiome counts as features to group subjects into relatively homogeneous clusters, without invoking any knowledge of observed health status (labels) of subjects. We then leverage information from the observed health status of subjects to associate these clusters with the most similar health status making it a semi-supervised approach. Finally, a feature selection technique is incorporated into the model to improve the overall classification performance. The proposed method provides a semi-supervised topic modelling approach that can help handle the high dimensionality of the microbiome data in association studies. Our experiments reveal that our semi-supervised classification algorithm is effective and efficient in terms of high classification accuracy compared to popular supervised learning approaches like SVM and multinomial logistic model. The RFSLDA framework is attractive because it (i) enhances clustering accuracy by identifying key bacteria types as indicators of health status, (ii) identifies key bacteria types within each group based on estimates of the proportion of bacteria types within the groups, and (iii) computes a measure of within-group similarity to identify highly similar subjects in terms of their health status.

[Source](https://doi.org/10.1038/s41598-024-59682-4)

10.1038/s41598-024-59682-4

---

## Deep learning methods in metagenomics: a review.
 19.04.2024, _G Roy, E Prifti, E Belda, JD Zucker_


_Humans, Deep Learning, Microbiota, Metagenome, Gastrointestinal Microbiome, Metagenomics_
The ever-decreasing cost of sequencing and the growing potential applications of metagenomics have led to an unprecedented surge in data generation. One of the most prevalent applications of metagenomics is the study of microbial environments, such as the human gut. The gut microbiome plays a crucial role in human health, providing vital information for patient diagnosis and prognosis. However, analysing metagenomic data remains challenging due to several factors, including reference catalogues, sparsity and compositionality. Deep learning (DL) enables novel and promising approaches that complement state-of-the-art microbiome pipelines. DL-based methods can address almost all aspects of microbiome analysis, including novel pathogen detection, sequence classification, patient stratification and disease prediction. Beyond generating predictive models, a key aspect of these methods is also their interpretability. This article reviews DL approaches in metagenomics, including convolutional networks, autoencoders and attention-based models. These methods aggregate contextualized data and pave the way for improved patient care and a better understanding of the microbiome's key role in our health.

[Source](https://doi.org/10.1099/mgen.0.001231)

10.1099/mgen.0.001231

---

