## Unveiling the Origin and Transmission of 2019-nCoV.
 25.03.2020, _Y Xu_


_Animals, Betacoronavirus, Chiroptera, Coronavirus Infections, Disease Outbreaks, Humans, Livestock, Melia azedarach, Middle East Respiratory Syndrome Coronavirus, Pneumonia, Viral, Public Health, SARS Virus, Spike Glycoprotein, Coronavirus, Zoonoses_
A novel coronavirus has caused thousands of human infections in China since December 2019, raising a global public health concern. Recent studies (Huang et al., Chan et al., and Zhou et al.) have provided timely insights into its origin and ability to spread among humans, informing infection prevention and control practices.

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30025-1)

10.1016/j.tim.2020.02.001

---

## The spike glycoprotein of the new coronavirus 2019-nCoV contains a furin-like cleavage site absent in CoV of the same clade.
 25.03.2020, _B Coutard, C Valle, X de Lamballerie, B Canard, NG Seidah, E Decroly_


_Amino Acid Sequence, Betacoronavirus, Furin, Genome, Viral, Protein Conformation, Spike Glycoprotein, Coronavirus_
In 2019, a new coronavirus (2019-nCoV) infecting Humans has emerged in Wuhan, China. Its genome has been sequenced and the genomic information promptly released. Despite a high similarity with the genome sequence of SARS-CoV and SARS-like CoVs, we identified a peculiar furin-like cleavage site in the Spike protein of the 2019-nCoV, lacking in the other SARS-like CoVs. In this article, we discuss the possible functional consequences of this cleavage site in the viral cycle, pathogenicity and its potential implication in the development of antivirals.

[Source](https://www.sciencedirect.com/science/article/pii/S0166354220300528)

10.1016/j.antiviral.2020.104742

---

## CD-sACE2 Inclusion Compounds: An Effective Treatment for Corona Virus Disease 2019 (COVID-19).
 01.04.2020, _P Sun, X Lu, C Xu, Y Wang, W Sun, J Xi_


ACE2 is a metalloproteinase and a homolog of carboxypeptidase ACE. It hydrolyzes a variety of angiotensin peptides and is the main active peptide of the renin-angiotensin system (RAS).[3] ACE2, which is made of 805 amino acids, is a type I transmembrane glycoprotein with a single extracellular catalytic domain. This article is protected by copyright. All rights reserved.

[Source](https://onlinelibrary.wiley.com/doi/epdf/10.1002/jmv.25804)

10.1002/jmv.25804

---

## When bacteria meet mitochondria: The strange case of the tick symbiont Midichloria mitochondriiâ€ .
 18.03.2020, _F Stavru, J Riemer, A Jex, D Sassera_


Mitochondria are key eukaryotic organelles that perform several essential functions. Not surprisingly, many intracellular bacteria directly or indirectly target mitochondria, interfering with innate immunity, energy production or apoptosis, to make the host cell a more hospitable niche for bacterial replication. The alphaproteobacterium Midichloria mitochondrii has taken mitochondrial targeting to another level by physically colonising mitochondria, as shown by transmission electron micrographs of bacteria residing in the mitochondrial intermembrane space. This unique localization provokes a number of questions around the mechanisms allowing, and reasons driving intramitochondrial tropism. We suggest possible scenarios that could lead to this peculiar localization and hypothesize potential costs and benefits of mitochondrial colonisation for the bacterium and its host.

[Source](https://onlinelibrary.wiley.com/doi/full/10.1111/cmi.13189)

10.1111/cmi.13189

---

## Horizontal transfer of antibiotic resistance genes in the human gut microbiome.
 10.03.2020, _RS McInnes, GE McCallum, LE Lamberte, W van Schaik_


Infections caused by antibiotic-resistant bacteria are a major threat to public health. The pathogens causing these infections can acquire antibiotic resistance genes in a process termed horizontal gene transfer (HGT). HGT is a common event in the human gut microbiome, that is, the microbial ecosystem of the human intestinal tract. HGT in the gut microbiome can occur via different mechanisms of which transduction and conjugation have been best characterised. Novel bioinformatic tools and experimental approaches have been developed to determine the association of antibiotic resistance genes with their microbial hosts and to quantify the extent of HGT in the gut microbiome. Insights from studies into HGT in the gut microbiome may lead to the development of novel interventions to minimise the spread of antibiotic resistance genes among commensals and opportunistic pathogens.

[Source](https://www.sciencedirect.com/science/article/pii/S1369527420300254)

10.1016/j.mib.2020.02.002

---

## Gut microbiome composition and diversity-related to human personality traits&nbsp;&nbsp;

[Source](https://www.sciencedirect.com/science/article/pii/S2452231719300181)

---

## Rapid multi-residue screening of antibiotics in muscle from different animal species by microbiological inhibition method

[Source](https://www.sciencedirect.com/science/article/pii/S0026265X1932586X)

---

## Binding of pDNA with cDNA using hybridization strategy towards monitoring of Haemophilus influenza genome in human plasma samples.
 31.03.2020, _A Saadati, S Hassanpour, M Hasanzadeh, N Shadjou_


Haemophilus Influenza leads to respiratory infections such as sinusitis, acute otitis media, pneumonia and bronchitis. In addition, it causes invasive infections such as cellulite, septic arthritis, and meningitis. Therefore, quick and sensitive detection of H. influenza is of great importance in medical microbiology. In this study, a novel DNA-based bioassay was developed to the monitoring of Haemophilus influenza genome in human plasma samples using binding of pDNA with cDNA. DNA hybridization strategy was used to investigation of DNAs binding. For this purpose, silver nanoparticle doped graphene quantum dots inks functionalized by D-penicillamine (Ag NPs-DPA-GQDs) were synthesized and deposited on the surface of glass carbon electrode (GCE). Also, gold nanoparticles functionalized with cysteamine (CysA-AuNPs) were deposited on the surface of the Ag-DPA-GQDs modified GCE. Afterward, thiolated DNA probe was immobilized on the surface of the modified electrode. DNA hybridization was monitored using square wave voltammetry (SWV) technique. Engineered genosensor indicated good performance with high specificity and sensitivity for detection of Haemophilus influenza genome. Under optimal conditions, linear range and low limit of quantitation (LLOQ) were obtained as target concentrations ranging from 1 pM-1 ZM and 1 ZM, respectively. The designed biosensor also showed high capability of discriminating one-base, two-base and three-base mismatched sequences. Also, the prepared genosensor could be easily regenerated and reused to evaluate hybridization process.

[Source](https://www.sciencedirect.com/science/article/pii/S0141813019403723)

10.1016/j.ijbiomac.2020.02.062

---

## Unravelling spatiotemporal variability of arbuscular mycorrhizal fungi in a temperate grassland plot

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/full/10.1111/1462-2920.14653)

---

## Comprehensive evaluation of complex polymicrobial specimens using next generation sequencing and standard microbiological culture.
 31.03.2020, _LA Cummings, DR Hoogestraat, SL Rassoulian-Barrett, CA Rosenthal, SJ Salipante, BT Cookson, NG Hoffman_


Optimal clinical decision-making depends on identification of clinically relevant organisms present in a sample. Standard microbiological culture may fail to identify unusual or fastidious organisms and can misrepresent relative abundance of sample constituents. Culture-independent methods have improved our ability to deconvolute polymicrobial patient samples. We used next-generation 16S rRNA gene sequencing (NGS16S) to determine how often cultivatable organisms in complex polymicrobial samples are not reported by standard culture. Twenty consecutive bronchoalveolar lavage (BAL) samples were plated to standard and additional media; bacteria were identified by NGS16S analysis of DNA extracted directly from samples or from washed culture plates. 96% of organisms identified were cultivable, but only 21% were reported by standard culture, indicating that standard work-up provides an incomplete assessment of microbial constituents. Direct NGS16S correlated well with standard culture, identifying the same predominant organism in 50% of samples. When predominant organisms differed, NGS16S most often detected anaerobes, whose growth is unsupported by standard culture conditions for this specimen. NGS16S identified more organisms per sample and allowed identification of fastidious organisms, while culture was better at capturing organisms when bacterial load was low, and allowed incidental recovery of non-bacterial pathogens. Molecular and culture-based methods together detect more organisms than either method alone.

[Source](https://www.nature.com/articles/s41598-020-62424-x)

10.1038/s41598-020-62424-x

---

## Identification and quantification of bacterial genomes carrying antibiotic resistance genes and virulence factor genes for aquatic microbiological risk assessment.
 06.11.2019, _J Liang, G Mao, X Yin, L Ma, L Liu, Y Bai, T Zhang, J Qu_


_Anti-Bacterial Agents, Drug Resistance, Microbial, Ecosystem, Genes, Bacterial, Genome, Bacterial, Reproducibility of Results, Risk Assessment, Virulence Factors_
Aquatic ecosystems have been increasingly threatened by anthropogenic activities, e.g., wastewater discharge and farm operation. Several methods are adopted to evaluate the effects of anthropogenic activities on biological risk in the environment, such as qPCR and amplicon next-generation sequencing. However, these methods fall short of providing genomic information of target species, which is vital for risk assessment from genomic aspect. Here, we developed a novel approach integrating metagenomic analysis and flow cytometry to identify and quantify potential pathogenic antibiotic resistant bacteria (PARB; carrying both antibiotic resistance genes (ARGs) and virulence factor genes (VFGs)) in the environment, which are of particular concern due to their infection ability and antibiotic resistance. Based on the abundance/density of PARB, we evaluated microbiological risk in a river impacted by both municipal drainage and agriculture runoff. We collected samples upstream (mountainous area) as the control. Results showed that 81.8% of dominant PARB (33) recovered using our approach were related to known pathogenic taxa. In addition, intragenomic ARGs-VFGs coexistence patterns in the dominant Pseudomonas genomes (20 out of 71 PARB) showed high similarity with the most closely related Pseudomonas genomes from the NCBI RefSeq database. These results reflect acceptable reliability of the approach for (potential) pathogen identification in environmental samples. According to the PARB density, microbiological risk in samples from the agricultural area was significantly higher than in samples from the urban area. We speculated that this was due to the higher antibiotic usage in agriculture as well as intragenomic ARGs-VFGs co-evolution under antibiotic selective pressure. This study provides an alternative approach for the identification and quantification of PARB in aquatic environments, which can be applied for microbiological risk assessment.

[Source](https://www.sciencedirect.com/science/article/pii/S0043135419309340)

10.1016/j.watres.2019.115160

---

