## Obesity is the main driver of altered gut microbiome functions in the metabolically unhealthy.
 11.09.2023, _J de la Cuesta-Zuluaga, KE Huus, ND Youngblut, JS Escobar, RE Ley_


_Adult, Humans, Gastrointestinal Microbiome, Microbiota, Obesity, Adipose Tissue, Anthropometry_
Obesity (OB) and cardiometabolic disease are major public health issues linked to changes in the gut microbiome. OB and poor cardiometabolic health status (CHS) are often comorbid, which hinders efforts to identify components of the microbiome uniquely linked to either one. Here, we used a deeply phenotyped cohort of 408 adults from Colombia, including subjects with OB, unhealthy CHS, or both, to validate previously reported features of gut microbiome function and diversity independently correlated with OB or CHS using fecal metagenomes. OB was defined by body mass index, waist circumference, and body fat; CHS as healthy or unhealthy according to blood biochemistry and anthropometric data. We found that OB, more so than metabolic status, drove associations with gut microbiome structure and functions. The microbiome of obese individuals with and without co-existing unhealthy CHS was characterized by reduced metagenomic diversity, reduced fermentative potential and elevated capacity to respond to oxidative stress and produce bacterial antigens. Disease-linked features were correlated with increased host blood pressure and inflammatory markers, and were mainly contributed by members of the family Enterobacteriaceae. Our results link OB with a microbiome able to tolerate an inflammatory and oxygenated gut state, and suggest that OB is the main driver of microbiome functional differences when poor CHS is a comorbidity.

[Source](https://doi.org/10.1080/19490976.2023.2246634)

10.1080/19490976.2023.2246634

---

## Elucidating the Role of the Microbiome in Ocular Diseases.
 08.09.2023, _W Luo, D Skondra_



[Source](https://ajp.amjpathol.org/article/S0002-9440(23)00313-9/fulltext)

10.1016/j.ajpath.2023.08.006

---

## A binary module for microbiota-mediated regulation of γδ17 cells, hallmarked by microbiota-driven expression of programmed cell death protein 1.
 08.09.2023, _HI Huang, Y Xue, ML Jewell, CY Tan, B Theriot, N Aggarwal, J Dockterman, YD Lin, EA Schroeder, D Wang, N Xiong, J Coers, ML Shinohara, NK Surana, GE Hammer_



[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(23)01155-5)

10.1016/j.celrep.2023.113143

---

## Resistant starch decreases intrahepatic triglycerides in patients with NAFLD via gut microbiome alterations.
 08.09.2023, _Y Ni, L Qian, SL Siliceo, X Long, E Nychas, Y Liu, MJ Ismaiah, H Leung, L Zhang, Q Gao, Q Wu, Y Zhang, X Jia, S Liu, R Yuan, L Zhou, X Wang, Q Li, Y Zhao, H El-Nezami, A Xu, G Xu, H Li, G Panagiotou, W Jia_


_Animals, Mice, Non-alcoholic Fatty Liver Disease, Gastrointestinal Microbiome, Resistant Starch, Microbiota, Triglycerides_
Non-alcoholic fatty liver disease (NAFLD) is a hepatic manifestation of metabolic dysfunction for which effective interventions are lacking. To investigate the effects of resistant starch (RS) as a microbiota-directed dietary supplement for NAFLD treatment, we coupled a 4-month randomized placebo-controlled clinical trial in individuals with NAFLD (ChiCTR-IOR-15007519) with metagenomics and metabolomics analysis. Relative to the control (n = 97), the RS intervention (n = 99) resulted in a 9.08% absolute reduction of intrahepatic triglyceride content (IHTC), which was 5.89% after adjusting for weight loss. Serum branched-chain amino acids (BCAAs) and gut microbial species, in particular Bacteroides stercoris, significantly correlated with IHTC and liver enzymes and were reduced by RS. Multi-omics integrative analyses revealed the interplay among gut microbiota changes, BCAA availability, and hepatic steatosis, with causality supported by fecal microbiota transplantation and monocolonization in mice. Thus, RS dietary supplementation might be a strategy for managing NAFLD by altering gut microbiota composition and functionality.

[Source](https://www.cell.com/cell-metabolism/fulltext/S1550-4131(23)00297-8)

10.1016/j.cmet.2023.08.002

---

## Microbial signature of plaque and gut in acute coronary syndrome.
 11.09.2023, _E Pisano, F Bugli, A Severino, D Pedicino, F Paroni Sterbini, C Martini, F De Maio, R Vinci, A Sacconi, F Canonico, A D'Aiello, A Bonanni, L Proto, P Ciampi, M Ponzo, MC Grimaldi, A Urbani, A Primiano, J Gervasoni, R Montone, F Crea, M Sanguinetti, G Liuzzo_


_Humans, Acute Coronary Syndrome, RNA, Ribosomal, 16S, Heart, Angioplasty, Balloon, Carnobacteriaceae_
Gut microbiota is an emerging editable cardiovascular risk factor. We aim to investigate gut and coronary plaque microbiota, using fecal samples and angioplasty balloons from patients with acute coronary syndrome (ACS), chronic coronary syndrome (CCS) and control subjects. We examined bacterial communities in gut and coronary plaques by 16S rRNA sequencing and we performed droplet digital PCR analysis to investigate the gut relative abundance of the bacterial genes CutC/CntA involved in trimethylamine N-oxide synthesis. Linear discriminant analysis effect size (LEfSe) at the genus and species levels displayed gut enrichment in Streptococcus, Granulicatella and P. distasonis in ACS compared with CCS and controls; Roseburia, C. aerofaciens and F. prausnitzii were more abundant in controls than in patients. Principal component analysis (PCA) of 41 differentially abundant gut taxa showed a clustering of the three groups. In coronary plaque, LEfSe at the genus level revealed an enrichment of Staphylococcus and Streptococcus in ACS, and Paracoccus in CCS, whereas PCA of 15 differentially abundant plaque taxa exhibited clustering of ACS and CCS patients. CutC and CntA genes were more abundant in ACS and CCS than in controls while no significant difference emerged between ACS and CCS. Our results indicate that ACS and CCS exhibit a different gut and plaque microbial signature, suggesting a possible role of these microbiotas in coronary plaque instability.

[Source](https://doi.org/10.1038/s41598-023-41867-y)

10.1038/s41598-023-41867-y

---

## Impact of the lipase inhibitor orlistat on the human gut microbiota.
 07.09.2023, _Y Uehira, H Ueno, J Miyamoto, I Kimura, Y Ishizawa, H Iijima, S Muroga, T Fujita, S Sakai, Y Samukawa, Y Tanaka, S Murayama, H Sakoda, M Nakazato_


Orlistat, an anti-obesity agent, inhibits the metabolism and absorption of dietary fat by inactivating pancreatic lipase in the gut. The effect of orlistat on the gut microbiota of Japanese individuals with obesity is unknown. This study aimed to explore the effects of orlistat on the gut microbiota and fatty acid metabolism of Japanese individuals with obesity. Fourteen subjects with visceral fat obesity (waist circumference ≥85 cm) took orlistat orally at a dose of 60 mg, 3 times a day for 8 weeks. Body weight; waist circumference; visceral fat area; levels of short-chain fatty acids, gut microbiota, fatty acid metabolites in the feces, and gastrointestinal hormones; and adverse events were evaluated. Body weight, waist circumference, and blood leptin concentrations were significantly lower after orlistat treatment (mean ± standard deviation, 77.8 ± 9.1 kg; 91.9 ± 8.7 cm; and 4546 ± 3211 pg/mL, respectively) compared with before treatment (79.4 ± 9.0 kg; 94.4 ± 8.0 cm; and 5881 ± 3526 pg/mL, respectively). Significant increases in fecal levels of fatty acid metabolites (10-hydroxy-cis-12-octadecenoic acid, 10-oxo-cis-12-octadecenoic acid, and 10-oxo-trans-11-octadecenoic acid) were detected. Meanwhile, no significant changes were found in abdominal computed tomography parameters, blood marker levels, or short-chain fatty acid levels in the feces. Gut microbiota analysis revealed that some study subjects had decreased abundance of Firmicutes, increased abundance of Bacteroidetes, and increased α-diversity indices (Chao1 and ACE) after 8 weeks of treatment. The levels of Lactobacillus genus and Lactobacillus gasseri were significantly higher after 8 weeks of treatment. None of the subjects discontinued treatment or experienced severe adverse events. This study suggested that orlistat might alter gut microbiota composition and affect the body through fatty acid metabolites produced by the modified gut bacteria.

[Source](https://www.sciencedirect.com/science/article/pii/S1871403X23000704?via%3Dihub)

10.1016/j.orcp.2023.08.005

---

## Metabolic network construction reveals probiotic-specific alterations in the metabolic activity of a synthetic small intestinal community.
 05.09.2023, _J Jansma, AC Chatziioannou, K Castricum, S van Hemert, S El Aidy_


The gut microbiota plays a crucial role in maintaining overall health and probiotics have emerged as a promising microbiota-targeted therapy for improving human health. However, the molecular mechanisms of probiotics action in general and the targeting of small intestinal microbiota by probiotics are not well understood. To address this, we constructed a synthetic community of three species, which resembles the upper small intestinal microbiota. Our results indicate that probiotic supplementation can directly affect the metabolism of the community, resulting in colonization resistance in a probiotic-specific manner. Supplementation with Streptococcus thermophilus led to increased lactate production and a decrease in pH, while Lactobacillus casei supplementation increased the resistance to perturbations and nutrient utilization without affecting lactate production or pH. Additionally, when combined with kynurenine, Lactobacillus casei enhanced the kynurenine pathway metabolism resulting in elevated kynurenic acid levels and possibly indirect colonization resistance. Overall, our study reveals how selecting probiotics with distinct functional capacities can unlock the full potential of microbiota-targeted therapies.IMPORTANCEThe development of probiotic therapies targeted at the small intestinal microbiota represents a significant advancement in the field of probiotic interventions. This region poses unique opportunities due to its low number of gut microbiota, along with the presence of heightened immune and metabolic host responses. However, progress in this area has been hindered by a lack of detailed understanding regarding the molecular mechanisms through which probiotics exert their effects in the small intestine. Our study, utilizing a synthetic community of three small intestinal bacterial strains and the addition of two different probiotic species, and kynurenine as a representative dietary or endogenously produced compound, highlights the importance of selecting probiotic species with diverse genetic capabilities that complement the functional capacity of the resident microbiota, or alternatively, constructing a multispecies formula. This approach holds great promise for the development of effective probiotic therapies and underscores the need to consider the functional capacity of probiotic species when designing interventions.

[Source](https://doi.org/10.1128/msystems.00332-23)

10.1128/msystems.00332-23

---

## Colonic phosphocholine is correlated with <em>Candida tropicalis</em> and promotes diarrhea and pathogen clearance

[Source](https://doi.org/10.1038/s41522-023-00433-0)

---

## Differential dependence on microbiota of IL-23/IL-22-dependent gene expression between the small- and large-intestinal epithelia.
 08.09.2023, _A Nakatani, R Okumura, A Ishibashi, S Okamoto, K Sakaki, Y Ito, D Okuzaki, H Inohara, K Takeda_


In the intestine, interleukin (IL)-23 and IL-22 from immune cells in the lamina propria contribute to maintenance of the gut epithelial barrier through the induction of antimicrobial production and the promotion of epithelial cell proliferation. Several previous studies suggested that some of the functions of the IL-23/IL-22 axis on intestinal epithelial cells are shared between the small and large intestines. However, the similarities and differences of the IL-23/IL-22 axis on epithelial cells between these two anatomical sites remain unclear. Here, we comprehensively analyzed the gene expression of intestinal epithelial cells in the ileum and colon of germ-free, Il23-/- , and Il22-/- mice by RNA-sequencing. We found that while the IL-23/IL-22 axis is largely dependent on gut microbiota in the small intestine, it is much less dependent on it in the large intestine. In addition, the negative regulation of lipid metabolism in the epithelial cells by IL-23 and IL-22 in the small intestine was revealed, whereas the positive regulation of epithelial cell proliferation by IL-23 and IL-22 in the large intestine was highlighted. These findings shed light on the intestinal site-specific role of the IL-23/IL-22 axis in maintaining the physiological functions of intestinal epithelial cells.

[Source](https://doi.org/10.1111/gtc.13065)

10.1111/gtc.13065

---

## Identification of microbial markers associated with lung cancer based on multi-cohort 16 s rRNA analyses: A systematic review and meta-analysis.
 07.09.2023, _W Han, N Wang, M Han, X Liu, T Sun, J Xu_


The relationship between commensal microbiota and lung cancer (LC) has been studied extensively. However, developing replicable microbiological markers for early LC diagnosis across multiple populations has remained challenging. Current studies are limited to a single region, single LC subtype, and small sample size. Therefore, we aimed to perform the first large-scale meta-analysis for identifying micro biomarkers for LC screening by integrating gut and respiratory samples from multiple studies and building a machine-learning classifier.

[Source](https://doi.org/10.1002/cam4.6503)

10.1002/cam4.6503

---

## Targeting microbial quorum sensing: the next frontier to hinder bacterial driven gastrointestinal infections.
 11.09.2023, _Y Su, T Ding_


_Humans, Quorum Sensing, Gastrointestinal Microbiome, Bacteria, Gastrointestinal Diseases, Homeostasis_
Bacteria synchronize social behaviors via a cell-cell communication and interaction mechanism termed as quorum sensing (QS). QS has been extensively studied in monocultures and proved to be intensively involved in bacterial virulence and infection. Despite the role QS plays in pathogens during laboratory engineered infections has been proved, the potential functions of QS related to pathogenesis in context of microbial consortia remain poorly understood. In this review, we summarize the basic molecular mechanisms of QS, primarily focusing on pathogenic microbes driving gastrointestinal (GI) infections. We further discuss how GI pathogens disequilibrate the homeostasis of the indigenous microbial consortia, rebuild a realm dominated by pathogens, and interact with host under worsening infectious conditions via pathogen-biased QS signaling. Additionally, we present recent applications and main challenges of manipulating QS network in microbial consortia with the goal of better understanding GI bacterial sociality and facilitating novel therapies targeting bacterial infections.

[Source](https://doi.org/10.1080/19490976.2023.2252780)

10.1080/19490976.2023.2252780

---

## Illuminating the oral microbiome and its host interactions: recent Advancements in omics and bioinformatics technologies in the context of oral microbiome research.
 05.09.2023, _JL Baker_


The oral microbiota has an enormous impact on human health, with oral dysbiosis now linked to many oral and systemic diseases. Recent advancements in sequencing, mass spectrometry, bioinformatics, computational biology, and machine learning are revolutionizing oral microbiome research, enabling analysis at an unprecedented scale and level of resolution using omics approaches. This review contains a comprehensive perspective of the current state-of-the-art tools available to perform genomics, metagenomics, phylogenomics, pangenomics, transcriptomics, proteomics, metabolomics, lipidomics, and multi-omics analysis on (all) microbiomes, and then provides examples of how the techniques have been applied to research of the oral microbiome, specifically. Key findings of these studies and remaining challenges for the field are highlighted. Although the methods discussed here are placed in the context of their contributions to oral microbiome research specifically, they are pertinent to the study of any microbiome, and the intended audience of this includes researchers would simply like to get an introduction to microbial omics and/or an update on the latest omics methods. Continued research of the oral microbiota using omics approaches is crucial and will lead to dramatic improvements in human health, longevity, and quality of life.

[Source](https://doi.org/10.1093/femsre/fuad051)

10.1093/femsre/fuad051

---

## Association between primary Sjögren's syndrome and gut microbiota disruption: a systematic review and meta-analysis.
 08.09.2023, _Y Shen, X Yu, Q Wang, X Yao, D Lu, D Zhou, X Wang_


Evidence of gut microbiota disruption for numerous autoimmune diseases has accumulated. Recently, the relationship between the microbiota and primary Sjögren's disease has been increasingly investigated but has yet to be systematically elucidated. Therefore, a meta-analysis of publications dealing on topic was conducted. Case-control studies comparing primary Sjögren's syndrome patients and healthy controls (HCs) were systematically searched in nine databases from inception to March 1, 2023. The primary result quantitatively evaluated in this meta-analysis was the α-diversity. The secondary results qualitatively extracted and analyzed were the β-diversity and relative abundance. In total, 22 case-control studies covering 915 pSS patients and 2103 HCs were examined. The quantitative analysis revealed a slight reduction in α-diversity in pSS patients compared to HCs, with a lower Shannon-Wiener index (SMD = - 0.46, (- 0.68, - 0.25), p &lt; 0.0001, I2 = 71%), Chao1 richness estimator (SMD = - 0.59, (- 0.86, - 0.32), p &lt; 0.0001, I2 = 81%), and ACE index (SMD = - 0.92, (- 1.64, - 0.19), p = 0.01, I2 = 86%). However, the Simpson index (SMD = 0.01, (- 0.43, 0.46) p = 0.95, I2 = 86%) was similar in the two groups. The β-diversity significantly differed between pSS patients and HCs. Variations in the abundance of specific microbes and their metabolites and potential functions contribute to the pSS pathogenesis. Notably, the abundance of the phylum Firmicutes decreased, while that of Proteobacteria increased. SCFA-producing microbes including Ruminococcaceae, Lachnospiraceae, Faecalibacterium, Butyricicoccus, and Eubacterium hallii were depleted. In addition to diversity, the abundances of some specific microbes were related to clinical parameters. According to this systematic review and meta-analysis, gut microbiota dysbiosis, including reduced diversity, was associated with proinflammatory bacterium enrichment and anti-inflammatory bacterium depletion in pSS patients. Further research on the relationship between the gut microbiota and pSS is warranted.

[Source](https://doi.org/10.1007/s10067-023-06754-x)

10.1007/s10067-023-06754-x

---

## Washed microbiota transplantation for <em>Clostridioides difficile</em> infection: a national multi-center real-world study

[Source](https://doi.org/10.1111/1751-2980.13227)

---

## <em>Lactobacillus johnsonii </em>N6.2 phospholipids induce immature-like dendritic cells with a migratory-regulatory-like transcriptional signature

[Source](https://doi.org/10.1080/19490976.2023.2252447)

---

## <em>Akkermansia muciniphila</em> supplementation prevents cognitive impairment in sleep-deprived mice by modulating microglial engulfment of synapses

[Source](https://doi.org/10.1080/19490976.2023.2252764)

---

## Metagenomics and metabolomics of Toddy, an Indian fermented date palm beverage

[Source](https://doi.org/10.1016/j.foodres.2023.113205)

---

## Gut microbiota from patients with COVID-19 cause alterations in mice that resemble post-COVID symptoms

[Source](https://doi.org/10.1080/19490976.2023.2249146)

---

## Ingestion and biodegradation of disposable surgical masks by yellow mealworms <em>Tenebrio molitor</em> larvae: Differences in mask layers and effects on the larval gut microbiome

[Source](https://doi.org/10.1016/j.scitotenv.2023.166808)

---

## Assessment of beneficial effects and identification of host adaptation-associated genes of <em>Ligilactobacillus salivarius</em> isolated from badgers

[Source](https://doi.org/10.1186/s12864-023-09623-8)

---

## Microbiota-mediated competition between <em>Drosophila</em> species

[Source](https://doi.org/10.1186/s40168-023-01617-8)

---

## Microbial signature profiles of <em>Penaeus vannamei</em> larvae in low-survival hatchery tanks affected by vibriosis

[Source](https://doi.org/10.7717/peerj.15795)

---

## Fiber deprivation and microbiome-borne curli shift gut bacterial populations and accelerate disease in a mouse model of Parkinson’s disease

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(23)01082-3)

---

## Microbial communities associated with resin canal discoloration in mango fruit.
 07.09.2023, _M Umar, JP Bowman, C Asis, C McConchie, A Eyles, R Stanley, A Gracie_


Resin canal discoloration (RCD) severely impacts the fruit quality of mango, diminishes consumer confidence and reduces sales, but the biological cause is still unclear. Using next-generation sequencing, the overall microbial community composition of RCD + and visually healthy mango fruit was determined for the first time to examine the possible role of bacterial and fungal pathogens in RCD. The diversity profile of bacterial and fungal communities was determined using primers targeting the 16S rRNA gene and ITS regions. Results showed that bacterial communities in healthy fruits are clustered together and significantly different from those in RCD + fruits. Tatumella and Pantoea species were the most abundant bacterial taxa on RCD + fruit, and both have been linked to disease outbreaks in a variety of fruit crops. Fungal communities were generally similar between RCD + and normal samples though non-pathogenic yeasts Meyerozyma and Naganishia tended to dominate the fungal communities on RCD + fruit. The study indicates that bacteria rather than fungal organisms are more likely to be associated with RCD in mango. This finding will facilitate the isolation and confirmation of RCD-causing organisms and the development of control strategies to manage RCD problem in mango.

[Source](https://doi.org/10.1093/lambio/ovad104)

10.1093/lambio/ovad104

---

## Studying the microbiome of suppressive soils against vascular wilt, caused by <em>Fusarium oxysporum</em> in cape gooseberry (<em>Physalis peruviana</em>)

[Source](https://doi.org/10.1111/1758-2229.13195)

---

## Antarctic snow algae: unraveling the processes underlying microbial community assembly during blooms formation.
 07.09.2023, _DF Soto, I Gómez, P Huovinen_


_Antarctic Regions, Phylogeny, RNA, Ribosomal, 16S, Eukaryota, Microbiota_
At the West Antarctic Peninsula, snow algae blooms are composed of complex microbial communities dominated by green microalgae and bacteria. During their progression, the assembly of these microbial communities occurs under harsh environmental conditions and variable nutrient content due to fast snow melting. To date, it is still unclear what are the ecological mechanisms governing the composition and abundance of microorganisms during the formation of snow algae blooms. In this study, we aim to examine the main ecological mechanisms governing the assembly of snow algae blooms from early stages to colorful stages blooms.

[Source](https://doi.org/10.1186/s40168-023-01643-6)

10.1186/s40168-023-01643-6

---

## Understanding how various forms of phosphorus stress affect microbiome functions and boost plant disease resistance: Insights from metagenomic analysis.
 10.09.2023, _J Kang, W Qiu, W Zhang, J Liu, Z Yang, Z Wu, J Ge_


The plant's response to phosphorus (P) starvation suppresses its immunity and regulates rhizosphere microbial colonization. However, the impact of various P forms on plant disease resistance and microbial composition remains underreported. This paper examines the soybean rhizosphere microbiome facing co-stress from Fusarium oxysporum and diverse P forms. Macrogenomic analysis evaluates whether P addition enhances plant disease resistance and rhizosphere microbial function, and if such effects relate to P forms. Results show that different P forms mitigate F. oxysporum-induced plant inhibition by promoting P turnover. P forms predominantly affect microbial composition, followed by soil and plant properties. In soybean, the phosphate transport strategy (ugpA/Q) was selected to maintain high P to enhance immunity in the KH2PO4 treatment, while organo-P mineralization (phnH/F/W/G) was selected for superphosphate treatment. The Frankiales, a P-turnover microorganism, copiotrophic microorganisms, and indicator bacteria of plant properties, initially increase after F. oxysporum inoculation and then decrease post P addition, regardless of P forms. Additionally, the rhizosphere microbial community's metabolic activities and compounds significantly aid soybean defense against F. oxysporum, with functional types depending on P forms. Therefore, these findings establish a novel approach to enhance host defense against soil-borne diseases through P nutrition regulation to mediate host-driven metabolic activities of microbial communities.

[Source](https://doi.org/10.1016/j.scitotenv.2023.166899)

10.1016/j.scitotenv.2023.166899

---

## Reductive soil disinfestation and Fe amendment improve soil microbial composition and <em>Fritillaria</em> production

[Source](https://doi.org/10.1007/s00253-023-12766-z)

---

## How Benthic Sediment Microbial Communities Respond to Glyphosate and Its Metabolite: a Microcosm Experiment.
 06.09.2023, _CM Cornish, P Bergholz, K Schmidt, J Sweetman_


Glyphosate is the most commonly used agricultural herbicide in the world. In aquatic ecosystems, glyphosate often adsorbs to benthic substrates or is metabolized and degraded by microorganisms. The effects of glyphosate on microbial communities vary widely as microorganisms respond differently to exposure. To help understand the impacts of glyphosate on the sediment microbiome, we conducted a microcosm experiment examining the responses of benthic sediment microbial communities to herbicide treatments. Sediments from a prairie pothole wetland were collected, and 16S rRNA gene sequencing was used to analyze community composition 2-h and 14-days after a single treatment of low (0.07 ppm), medium (0.7 ppm), or high (7 ppm) glyphosate, aminomethylphosphonic acid (glyphosate metabolite), or a glyphosate-based commercial formula. We found no significant differences in microbial community composition across treatments, concentration levels, or day of sampling. These findings suggest that microbial species in the Prairie Pothole Region of North America may be tolerant to glyphosate exposure.

[Source](https://doi.org/10.1007/s00248-023-02296-6)

10.1007/s00248-023-02296-6

---

## Functional conservation of microbial communities determines composition predictability in anaerobic digestion.
 04.09.2023, _Q Lin, L Li, J De Vrieze, C Li, X Fang, X Li_


A major challenge in managing and engineering microbial communities is determining whether and how microbial community responses to environmental alterations can be predicted and explained, especially in microorganism-driven systems. We addressed this challenge by monitoring microbial community responses to the periodic addition of the same feedstock throughout anaerobic digestion, a typical microorganism-driven system where microorganisms degrade and transform the feedstock. The immediate and delayed response consortia were assemblages of microorganisms whose abundances significantly increased on the first or third day after feedstock addition. The immediate response consortia were more predictable than the delayed response consortia and showed a reproducible and predictable order-level composition across multiple feedstock additions. These results stood in both present (16 S rRNA gene) and potentially active (16 S rRNA) microbial communities and in different feedstocks with different biodegradability and were validated by simulation modeling. Despite substantial species variability, the immediate response consortia aligned well with the reproducible CH4 production, which was attributed to the conservation of expressed functions by the response consortia throughout anaerobic digestion, based on metatranscriptomic data analyses. The high species variability might be attributed to intraspecific competition and contribute to biodiversity maintenance and functional redundancy. Our results demonstrate reproducible and predictable microbial community responses and their importance in stabilizing system functions.

[Source](https://doi.org/10.1038/s41396-023-01505-x)

10.1038/s41396-023-01505-x

---

## Development of an antibody fused with an antimicrobial peptide targeting <em>Pseudomonas aeruginosa</em>: A new approach to prevent and treat bacterial infections

[Source](https://doi.org/10.1371/journal.ppat.1011612)

---

## Clustering microbiome data using mixtures of logistic normal multinomial models.
 11.09.2023, _Y Fang, S Subedi_


_Bayes Theorem, Cluster Analysis, Computational Biology, High-Throughput Nucleotide Sequencing, Microbiota_
Discrete data such as counts of microbiome taxa resulting from next-generation sequencing are routinely encountered in bioinformatics. Taxa count data in microbiome studies are typically high-dimensional, over-dispersed, and can only reveal relative abundance therefore being treated as compositional. Analyzing compositional data presents many challenges because they are restricted to a simplex. In a logistic normal multinomial model, the relative abundance is mapped from a simplex to a latent variable that exists on the real Euclidean space using the additive log-ratio transformation. While a logistic normal multinomial approach brings flexibility for modeling the data, it comes with a heavy computational cost as the parameter estimation typically relies on Bayesian techniques. In this paper, we develop a novel mixture of logistic normal multinomial models for clustering microbiome data. Additionally, we utilize an efficient framework for parameter estimation using variational Gaussian approximations (VGA). Adopting a variational Gaussian approximation for the posterior of the latent variable reduces the computational overhead substantially. The proposed method is illustrated on simulated and real datasets.

[Source](https://doi.org/10.1038/s41598-023-41318-8)

10.1038/s41598-023-41318-8

---

## A new spike-in-based method for quantitative metabarcoding of soil fungi and bacteria.
 06.09.2023, _M Camacho-Sanchez_


Metabarcoding is a powerful tool to characterize biodiversity in biological samples. The interpretation of taxonomic profiles from metabarcoding data has been hindered by their compositional nature. Several strategies have been proposed to transform compositional data into quantitative data, but they have intrinsic limitations. Here, I propose a workflow based on bacterial and fungal cellular internal standards (spike-ins) for absolute quantification of the microbiota in soil samples. These standards were added to the samples before DNA extraction in amounts estimated after qPCRs, to target around 1-2% coverage in the sequencing run. In bacteria, proportions of spike-in reads in the sequencing run were very similar (&lt; 2-fold change) to those predicted by the qPCR assessment, and for fungi they differed up to 40-fold. The low variation among replicates highlights the reproducibility of the method. Estimates based on multiple bacterial spike-ins were highly correlated (r = 0.99). Procrustes analysis evidenced significant biological effects on the community composition when normalizing compositional data. A protocol based on qPCR estimation of input amounts of cellular spikes is proposed as a cheap and reliable strategy for quantitative metabarcoding of biological samples.

[Source](https://doi.org/10.1007/s10123-023-00422-5)

10.1007/s10123-023-00422-5

---

## Microbiome screening platform finds drugs for bugs.
 06.09.2023, _K Kingwell_



[Source](https://doi.org/10.1038/d41573-023-00146-1)

10.1038/d41573-023-00146-1

---

