## Lichen Planopilaris: The first biopsy layer microbiota inspection.
 22.07.2022, _D Pinto, FM Calabrese, M De Angelis, G Celano, G Giuliani, F Rinaldi_


_Alopecia, Biopsy, Humans, Inflammation, Interleukin-23, Lichen Planus, Microbiota, RNA, Ribosomal, 16S_
Lichen Planopilaris (LPP) is a lymphatic disease affecting the scalp that is characterized by a chronic and destructive inflammation process, named as 'cicatricial alopecia' in which the hair follicles are targeted and may involve predominantly lymphocytes or neutrophils. Scalp and biopsy layers have never been used to investigate microbial community composition and its relative taxa abundances in LPP. We sought to examine the significant taxa of this chronic relapsing inflammatory skin disease, together with inspect the existing connections with metabolic pathways featuring this microbial community. We used a multilevel analysis based on 16S rRNA marker sequencing in order to detect OTU abundances in pathologic/healthy samples, real time PCR for measuring the levels of IL-23 interleukin expression and urinary metabolomics to find out volatile organic metabolites (VOMs). By using a linear regression model, we described peculiar taxa that significantly differentiated LPP and healthy samples. We inspected taxa abundances and interleukin mRNA levels and the Microbacteriaceae family resulted negatively correlated with the IL-23 expression. Moreover, starting from 16S taxa abundances, we predicted the metabolic pathways featuring this microbial community. By inspecting microbial composition, sample richness, metabolomics profiles and the relative metabolic pathways in a cohort of LPP and healthy samples we deepened the contribution of significant taxa that are connected to inflammation maintenance and microbiota plasticity in LPP pathology.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0269933)

10.1371/journal.pone.0269933

---

## Adaptation of the infant gut microbiome during the complementary feeding transition.
 22.07.2022, _S McKeen, NC Roy, JA Mullaney, H Eriksen, A Lovell, M Kussman, W Young, K Fraser, CR Wall, WC McNabb_


_Feces, Gastrointestinal Microbiome, Humans, Infant, Infant Nutritional Physiological Phenomena, Metagenome, Pilot Projects_
The infant gut microbiome progresses in composition and function during the introduction of solid foods throughout the first year of life. The purpose of this study was to characterize changes in healthy infant gut microbiome composition, metagenomic functional capacity, and associated metabolites over the course of the complementary feeding period. Fecal samples were obtained at three 'snapshot' timepoints from infants participating in the 'Nourish to Flourish' pilot study: before the introduction of solid foods at approximately 4 months of age, after introducing solid foods at 9 months of age, and after continued diet diversification at 12 months of age. KEGG and taxonomy assignments were correlated with LC-MS metabolomic profiles to identify patterns of co-abundance. The composition of the microbiome diversified during the first year of life, while the functional capacity present in the gut microbiome remained stable. The introduction of solid foods between 4 and 9 months of age corresponded to a larger magnitude of change in relative abundance of sequences assigned to KEGG pathways and taxonomic assignments, as well as to stronger correlations with metabolites, compared to the magnitude of changes and number of correlations seen during continued diet diversification between 9 and 12 months of age. Changes in aqueous fecal metabolites were more strongly correlated with KEGG pathway assignments, while changes in lipid metabolites associated with taxonomic assignments, particularly between 9 and 12 months of age. This study establishes trends in microbiome composition and functional capacity occurring during the complementary feeding period and identifies potential metabolite targets for future investigations.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0270213)

10.1371/journal.pone.0270213

---

## Metagenomic insights into the microbe-mediated B and K2 vitamin biosynthesis in the gastrointestinal microbiome of ruminants.
 25.07.2022, _Q Jiang, L Lin, F Xie, W Jin, W Zhu, M Wang, Q Qiu, Z Li, J Liu, S Mao_


_Animals, Cattle, Gastrointestinal Microbiome, Metagenomics, Rumen, Ruminants, Vitamin B 12, Vitamins_
B and K2 vitamins, essential nutrients in host metabolism, can be synthesized by the rumen microbiome in ruminants and subsequently absorbed by the host. However, the B and K2 vitamin biosynthesis by the whole gastrointestinal microbiome and their abundances in different dietary strategies are largely unknown. Here, we reanalyzed our previous large-scale metagenomic data on the gastrointestinal microbiome of seven ruminant species and recruited 17,425 nonredundant microbial genomes from published datasets to gain a comprehensive understanding of the microbe-mediated B and K2 vitamin biosynthesis in ruminants.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01298-9)

10.1186/s40168-022-01298-9

---

## Comparative analysis of the gut microbiota of sand fly vectors of zoonotic visceral leishmaniasis (ZVL) in Iran; host-environment interplay shapes diversity

[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0010609)

---

## Metagenomic analysis of the dynamical conversion of photosynthetic bacterial communities in different crop fields over different growth periods.
 22.07.2022, _JE Cheng, P Su, ZH Zhang, LM Zheng, ZY Wang, MR Hamid, JP Dai, XH Du, LJ Chen, ZY Zhai, XT Kong, Y Liu, DY Zhang_


_Bacteria, Crops, Agricultural, High-Throughput Nucleotide Sequencing, Metagenome, Metagenomics, Microbiota, Soil Microbiology_
Photosynthetic bacteria are beneficial to plants, but knowledge of photosynthetic bacterial community dynamics in field crops during different growth stages is scarce. The factors controlling the changes in the photosynthetic bacterial community during plant growth require further investigation. In this study, 35 microbial community samples were collected from the seedling, flowering, and mature stages of tomato, cucumber, and soybean plants. 35 microbial community samples were assessed using Illumina sequencing of the photosynthetic reaction center subunit M (pufM) gene. The results revealed significant alpha diversity and community structure differences among the three crops at the different growth stages. Proteobacteria was the dominant bacterial phylum, and Methylobacterium, Roseateles, and Thiorhodococcus were the dominant genera at all growth stages. PCoA revealed clear differences in the structure of the microbial populations isolated from leaf samples collected from different crops at different growth stages. In addition, a dissimilarity test revealed significant differences in the photosynthetic bacterial community among crops and growth stages (P&lt;0.05). The photosynthetic bacterial communities changed during crop growth. OTUs assigned to Methylobacterium were present in varying abundances among different sample types, which we speculated was related to the function of different Methylobacterium species in promoting plant growth development and enhancing plant photosynthetic efficiency. In conclusion, the dynamics observed in this study provide new research ideas for the detailed assessments of the relationship between photosynthetic bacteria and different growth stages of plants.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0262517)

10.1371/journal.pone.0262517

---

## A metagenomic investigation of spatial and temporal changes in sewage microbiomes across a university campus

[Source](https://www.biorxiv.org/content/10.1101/2022.07.13.499996v1.abstract?%3Fcollection=)

---

