## mSphere of Influence: Translating Gut Microbiome Studies To Benefit Human Health.
 09.07.2020, _AM Seekatz_


Anna M. Seekatz works in the field of the gut microbiome as it related to infectious diseases. In this "mSphere of Influence" article, she reflects on how two studies, "The impact of a consortium of fermented milk strains on the gut microbiome of gnotobiotic mice and monozygotic twins" (N. P. McNulty, T. Yatsunenko, A. Hsiao, et al., Sci Transl Med 3:106ra106, 2011) and "High-throughput DNA sequence analysis reveals stable engraftment of gut microbiota following transplantation of previously frozen fecal bacteria" (M. J. Hamilton, A. R. Weingarden, T. Unno, A. Khoruts, and M. J. Sadowsky, Gut Microbes 4:125-135, 2013), shaped how she approaches interpreting microbiome studies.

[Source](https://msphere.asm.org/content/5/4/e00592-20)

10.1128/mSphere.00592-20

---

## A metagenomics-based diagnostic approach for central nervous system infections in hospital acute care setting.
 09.07.2020, _MR Hasan, S Sundararaju, P Tang, KM Tsui, AP Lopez, M Janahi, R Tan, P Tilley_


The etiology of central nervous system (CNS) infections such as meningitis and encephalitis remains unknown in a large proportion of cases partly because the diversity of pathogens that may cause CNS infections greatly outnumber available test methods. We developed a metagenomic next generation sequencing (mNGS)-based approach for broad-range detection of pathogens associated with CNS infections suitable for application in the acute care hospital setting. The analytical sensitivity of mNGS performed on an Illumina MiSeq was assessed using simulated cerebrospinal fluid (CSF) specimens (n = 9). mNGS data were then used as a training dataset to optimize a bioinformatics workflow based on the IDseq pipeline. For clinical validation, residual CSF specimens (n = 74) from patients with suspected CNS infections previously tested by culture and/or PCR, were analyzed by mNGS. In simulated specimens, the NGS reads aligned to pathogen genomes in IDseq were correlated to qPCR CT values for the respective pathogens (R = 0.96; p &lt; 0.0001), and the results were highly specific for the spiked pathogens. In clinical samples, the diagnostic accuracy, sensitivity and specificity of the mNGS with reference to conventional methods were 100%, 95% and 96%, respectively. The clinical application of mNGS holds promise to benefit patients with CNS infections of unknown etiology.

[Source](https://www.nature.com/articles/s41598-020-68159-z)

10.1038/s41598-020-68159-z

---

## Metagenomic analysis of gut microbiome and resistome of diarrheal fecal samples from Kolkata, India, reveals the core and variable microbiota including signatures of microbial dark matter

[Source](https://gutpathogens.biomedcentral.com/articles/10.1186/s13099-020-00371-8)

---

## Bladder cancer-related microbiota: examining differences in urine and tissue samples.
 10.07.2020, _B Mansour, Á Monyók, N Makra, M Gajdács, I Vadnay, B Ligeti, J Juhász, D Szabó, E Ostorházi_


The microbiota isolated from the urine of bladder carcinoma patients exhibits significantly increased compositional abundance of some bacterial genera compared to the urine of healthy patients. Our aim was to compare the microbiota composition of cancerous tissues and urine samples collected from the same set of patients in order to improve the accuracy of diagnostic measures. Tissue samples were collected from patients during cancer tissue removal by transurethral resection. In parallel, urine samples were obtained by transurethral resectoscopy from the same patients. The V3-V4 region of the bacterial 16S rRNA gene was sequenced and analyzed using the Kraken pipeline. In the case of four patients, duplicate microbiota analysis from distant parts of the cancerous tissues was highly reproducible, and independent of the site of tissue collection of any given patient. Akkermansia, Bacteroides, Clostridium sensu stricto, Enterobacter and Klebsiella, as "five suspect genera", were over-represented in tissue samples compared to the urine. To our knowledge, this is the first study comparing urinary and bladder mucosa-associated microbiota profiles in bladder cancer patients. More accurate characterization of changes in microbiota composition during bladder cancer progression could provide new opportunities in the development of appropriate screening or monitoring methods.

[Source](https://www.nature.com/articles/s41598-020-67443-2)

10.1038/s41598-020-67443-2

---

## <span style="color:#008000;">Daily Vaginal Microbiota Fluctuations Associated with Natural Hormonal Cycle, Contraceptives, Diet, and Exercise</span>

[Source](https://msphere.asm.org/content/5/4/e00593-20)

---

## Dynamics of the lung microbiome in intensive care patients with chronic obstructive pulmonary disease and community-acquired pneumonia.
 10.07.2020, _WC Huang, MF Wu, CC Huang, SY Liu, HC Chen, YY Chen, JY Hsu, CC Huang_


Little is known about the composition and clinical implications of lung microbiome in patients with chronic obstructive pulmonary disease (COPD) and community-acquired pneumonia requiring invasive mechanical ventilation and intensive care unit admission. Therefore, this study aimed to explore the longitudinal changes in microbial airway composition and its variations between COPD patients with different weaning outcomes. Fifty-one endotracheal aspirate samples from 21 participants and 5 saline samples were collected as the patient and control group, respectively. Sequence analysis revealed significant increases and upward trends in the relative abundance of the Acinetobacter genus and Acinetobacter baumannii complex species in paired comparisons of sampling points and over time, respectively, in patients with failed weaning (p for trend = 0.012 and 0.012, respectively) but not in those with successful weaning (p for trend = 0.335 and 0.426, respectively). Furthermore, significant changes in the composition of the bacterial community were observed in paired comparisons of sampling points in patients with failed weaning compared with those with successful weaning. The alpha diversity did not differ between the patients with different weaning outcomes. These results further the understanding of longitudinal airway microbiome structure analysis and its clinical implications when managing critically ill patients with and without COPD.

[Source](https://www.nature.com/articles/s41598-020-68100-4)

10.1038/s41598-020-68100-4

---

## The polyphenol-rich extract from chokeberry (<em>Aronia melanocarpa</em>&nbsp;L<em>.</em>) modulates gut microbiota and improves lipid metabolism in diet-induced obese rats

[Source](https://nutritionandmetabolism.biomedcentral.com/articles/10.1186/s12986-020-00473-9)

---

## Changes in the gut microbiota mediate the differential regulatory effects of two glucose oxidases produced by&nbsp;<em>Aspergillus niger</em>&nbsp;and&nbsp;<em>Penicillium amagasakiense</em>&nbsp;on the meat quality and growth performance of broilers

[Source](https://jasbsci.biomedcentral.com/articles/10.1186/s40104-020-00480-z)

---

## The effects of atrazine on the microbiome of the eastern oyster:&nbsp;

[Source](https://www.nature.com/articles/s41598-020-67851-4)

---

## Crassostrea virginica

[Source](https://www.nature.com/articles/s41598-020-67851-4)

---

## How being synanthropic affects the gut bacteriome and mycobiome: comparison of two mouse species with contrasting ecologies

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-020-01859-8)

---

## Exploring bacteria diversity in commercialized table olive biofilms by metataxonomic and compositional data analysis

[Source](https://www.nature.com/articles/s41598-020-68305-7)

---

## Deconstructing the Soil Microbiome into Reduced-Complexity Functional Modules

[Source](https://mbio.asm.org/content/11/4/e01349-20)

---

## Metagenomic water quality monitoring with a portable laboratory

[Source](http://www.sciencedirect.com/science/article/pii/S0043135420306497)

---

## Exploring neighborhoods in large metagenome assembly graphs using spacegraphcats reveals hidden sequence diversity

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02066-4)

---

