## Metagenomic analysis exploring taxonomic and functional diversity of bacterial communities of a Himalayan urban fresh water lake

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0248116)

---

## A distinct clade of Bifidobacterium longum in the gut of Bangladeshi children thrives during weaning

[Source](https://www.sciencedirect.com/science/article/pii/S0092867422013265)

---

## Recent advances in systemic lupus erythematosus and microbiota: from bench to bedside

[Source](https://link.springer.com/article/10.1007/s11684-022-0957-7)

---

## Identification of shared bacterial strains in the vaginal microbiota of related and unrelated reproductive-age mothers and daughters using genome-resolved metagenomic

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0275908)

---

## Gut microbiome dysbiosis in antibiotic-treated COVID-19 patients is associated with microbial translocation and bacteremia

[Source](https://www.nature.com/articles/s41467-022-33395-6)

---

## Gut microbiota and myocardial fibrosis

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0014299922006161)

---

## Assessing the relationship between physical activity and the gut microbiome in a large, population-based sample of Wisconsin adults

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0276684)

---

## Challenges and costs of donor screening for fecal microbiota transplantations

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0276323)

---

## Gut colonization by Proteobacteria alters host metabolism and modulates cocaine neurobehavioral responses

[Source](https://www.sciencedirect.com/science/article/pii/S193131282200470X)

---

## Enriching captivity conditions with natural elements does not prevent the loss of wild-like gut microbiota but shapes its compositional variation in two small mamma

[Source](https://onlinelibrary.wiley.com/doi/10.1002/mbo3.1318)

---

## Understanding the Dynamic of POMS Infection and the Role of Microbiota Composition in the Survival of Pacific Oysters, Crassostrea gigas

[Source](https://journals.asm.org/doi/10.1128/spectrum.01959-22)

---

## Changes in Gut Microbiota Structure: A Potential Pathway for Silver Nanoparticles to Affect the Host Metabolism

[Source](https://pubs.acs.org/doi/10.1021/acsnano.2c07924)

---

## Skin microbiota of oxazolone-induced contact hypersensitivity mouse model

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0276071)

---

## Effects of a farm-specific fecal microbial transplant (FMT) product on clinical outcomes and fecal microbiome composition in preweaned dairy calves

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0276638)

---

## Multiomics Characterization of the Canada Goose Fecal Microbiome Reveals Selective Efficacy of Simulated Metagenomes

[Source](https://journals.asm.org/doi/10.1128/spectrum.02384-22)

---

## Uropygial gland microbiota differ between free-living and captive songbirds

[Source](https://www.nature.com/articles/s41598-022-22425-4)

---

## Crop management shapes the diversity and activity of DNA and RNA viruses in the rhizosphere

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01371-3)

---

## CRAMdb: a comprehensive database for composition and roles of microbiome in animals

[Source](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac973/6786202)

---

## DNA extraction bias is more pronounced for microbial eukaryotes than for prokaryotes.
 01.11.2022, _A Brauer, MM Bengtsson_


_Eukaryota, Archaea, Bacteria, Microbiota, DNA_
DNA extraction and preservation bias is a recurring topic in DNA sequencing-based microbial ecology. The different methodologies can lead to distinct outcomes, which has been demonstrated especially in studies investigating prokaryotic community composition. Eukaryotic microbes are ubiquitous, diverse, and increasingly a subject of investigation in addition to bacteria and archaea. However, little is known about how the choice of DNA preservation and extraction methodology impacts perceived eukaryotic community composition. In this study, we compared the effect of two DNA preservation methods and six DNA extraction methods on the community profiles of both eukaryotes and prokaryotes in phototrophic biofilms on seagrass (Zostera marina) leaves from the Baltic Sea. We found that, whereas both DNA preservation and extraction method caused significant bias in perceived community composition for both eukaryotes and prokaryotes, extraction bias was more pronounced for eukaryotes than for prokaryotes. In particular, soft-bodied and hard-shelled eukaryotes like nematodes and diatoms, respectively, were differentially abundant depending on the extraction method. We conclude that careful consideration of DNA preservation and extraction methodology is crucial to achieving representative community profiles of eukaryotes in marine biofilms and likely all other habitats containing diverse eukaryotic microbial communities.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/mbo3.1323)

10.1002/mbo3.1323

---

## synDNA-a Synthetic DNA Spike-in Method for Absolute Quantification of Shotgun Metagenomic Sequencing.
 01.11.2022, _LS Zaramela, M Tjuanta, O Moyne, M Neal, K Zengler_


Microbiome studies have the common goal of determining which microbial taxa are present, respond to specific conditions, or promote phenotypic changes in the host. Most of these studies rely on relative abundance measurements to drive conclusions. Inherent limitations of relative values are the inability to determine whether an individual taxon is more or less abundant and the magnitude of this change between the two samples. These limitations can be overcome by using absolute abundance quantifications, which can allow for a more complete understanding of community dynamics by measuring variations in total microbial loads. Obtaining absolute abundance measurements is still technically challenging. Here, we developed synthetic DNA (synDNA) spike-ins that enable precise and cost-effective absolute quantification of microbiome data by adding defined amounts of synDNAs to the samples. We designed 10 synDNAs with the following features: 2,000-bp length, variable GC content (26, 36, 46, 56, or 66% GC), and negligible identity to sequences found in the NCBI database. Dilution pools were generated by mixing the 10 synDNAs at different concentrations. Shotgun metagenomic sequencing showed that the pools of synDNAs with different percentages of GC efficiently reproduced the serial dilution, showing high correlation (r = 0.96; R2 â‰¥ 0.94) and significance (P &lt; 0.01). Furthermore, we demonstrated that the synDNAs can be used as DNA spike-ins to generate linear models and predict with high accuracy the absolute number of bacterial cells in complex microbial communities. IMPORTANCE The synDNAs designed in this study enable accurate and reproducible measurements of absolute amount and fold changes of bacterial species in complex microbial communities. The method proposed here is versatile and promising as it can be applied to bacterial communities or genomic features like genes and operons, in addition to being easily adaptable by other research groups at a low cost. We also made the synDNAs' sequences and the plasmids available to encourage future application of the proposed method in the study of microbial communities.

[Source](https://journals.asm.org/doi/10.1128/msystems.00447-22)

10.1128/msystems.00447-22

---

