## Colonic Transendoscopic Enteral Tubing Is a New Pathway to Microbial Therapy, Colonic Drainage, and Host-Microbiota Interaction Research.
 14.02.2023, _W Wang, G Lu, X Wu, Q Wen, F Zhang_


The limitation of traditional delivery methods for fecal microbiota transplantation (FMT) gave birth to colonic transendoscopic enteral tubing (TET) to address the requirement of frequent FMTs. Colonic TET as a novel endoscopic intervention has received increasing attention in practice since 2015 in China. Emerging studies from multiple centers indicate that colonic TET is a promising, safe, and practical delivery method for microbial therapy and administering medication with high patient satisfaction. Intriguingly, colonic TET has been used to rescue endoscopy-related perforations by draining colonic air and fluid through the TET tube. Recent research based on collecting ileocecal samples through a TET tube has contributed to demonstrating community dynamics in the intestine, and it is expected to be a novel delivery of proof-of-concept in host-microbiota interactions and pharmacological research. The present article aims to review the concept and techniques of TET and to explore microbial therapy, colonic drainage, and microbial research based on colonic TET.

[Source](https://www.sciencedirect.com/journal/current-opinion-in-microbiology/special-issue/10GJB726PJ3)

10.3390/jcm12030780

---

## a microbiota review collection

[Source](https://www.sciencedirect.com/journal/current-opinion-in-microbiology/special-issue/10GJB726PJ3)

---

## @PhilHugenholtz

[Source](https://twitter.com/PhilHugenholtz?ref_src=twsrc%5Etfw)

---

## GTDB-Tk v2: memory friendly classification with the genome taxonomy database.
 23.12.2022, _PA Chaumeil, AJ Mussig, P Hugenholtz, DH Parks_


_Documentation, Software_
The Genome Taxonomy Database (GTDB) and associated taxonomic classification toolkit (GTDB-Tk) have been widely adopted by the microbiology community. However, the growing size of the GTDB bacterial reference tree has resulted in GTDB-Tk requiring substantial amounts of memory (∼320 GB) which limits its adoption and ease of use. Here, we present an update to GTDB-Tk that uses a divide-and-conquer approach where user genomes are initially placed into a bacterial reference tree with family-level representatives followed by placement into an appropriate class-level subtree comprising species representatives. This substantially reduces the memory requirements of GTDB-Tk while having minimal impact on classification.

[Source](https://twitter.com/donovan_parks?ref_src=twsrc%5Etfw)

10.1093/bioinformatics/btac672

---

## @donovan_parks

[Source](https://twitter.com/donovan_parks?ref_src=twsrc%5Etfw)

---

## https://t.co/aXdGhRGDTJ

[Source](https://t.co/aXdGhRGDTJ)

---

## https://t.co/7dDFaI1F6Y

[Source](https://t.co/7dDFaI1F6Y)

---

## https://t.co/LUIkwO1CaS

[Source](https://t.co/LUIkwO1CaS)

---

## pic.twitter.com/U83qPOcDyi

[Source](https://t.co/U83qPOcDyi)

---

## February 2, 2023

[Source](https://twitter.com/MicrobiomeVIF/status/1621163307456663556?ref_src=twsrc%5Etfw)

---

## <em>Gardnerella </em>Diversity and Ecology in Pregnancy and Preterm Birth

[Source](https://www.biorxiv.org/content/10.1101/2023.02.03.527032v1)

---

## Heterogeneity of the group B streptococcal type VII secretion system and influence on colonization of the female genital tract 

[Source](https://www.biorxiv.org/content/10.1101/2023.01.25.525443v1)

---

## Comparative Genomic Study of <em>Streptococcus anginosus </em>Reveals Distinct Group of Urinary Strains

[Source](https://journals.asm.org/doi/10.1128/msphere.00687-22)

---

## Gut microbes predominantly act as symbiotic partners rather than raw nutrients

[Source](https://www.biorxiv.org/content/10.1101/2023.02.08.527620v1)

---

## https://t.co/7QNjiRpSZm

[Source](https://t.co/7QNjiRpSZm)

---

## February 9, 2023

[Source](https://twitter.com/mariaelemartino/status/1623609217721503745?ref_src=twsrc%5Etfw)

---

## Antibiotic resistance in the commensal human gut microbiota

[Source](https://www.sciencedirect.com/science/article/pii/S1369527422000340)

---

## Identifying glycan consumers in human gut microbiota samples using metabolic labeling coupled with fluorescence-activated cell sorting

[Source](https://www.nature.com/articles/s41467-023-36365-8)

---

## Over supplementation with vitamin B12 alters microbe-host interactions in the gut leading to accelerated <em>Citrobacter rodentium </em>colonization and pathogenesis in mice

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01461-w)

---

## Disturbance of lipid metabolism in germ-free mice transplanted with gut microbiota of DSS-induced colitis mice 

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0280850)

---

## https://t.co/9ABypLiRAj

[Source](https://t.co/9ABypLiRAj)

---

## February 8, 2023

[Source](https://twitter.com/BigDataBiology/status/1623370927281999891?ref_src=twsrc%5Etfw)

---

## Accurate and fast graph-based pangenome annotation and clustering with ggCaller

[Source](https://www.biorxiv.org/content/10.1101/2023.01.24.524926v1)

---

## uBin - a manual refining tool for genomes from metagenomes.
 10.02.2023, _TLV Bornemann, SP Esser, TL Stach, T Burg, AJ Probst_


Resolving bacterial and archaeal genomes from metagenomes has revolutionized our understanding of Earth's biomes yet producing high quality genomes from assembled fragments has been an ever-standing problem. While automated binning software and their combination produce prokaryotic bins in high throughput, their manual refinement has been slow, sometimes difficult or missing entirely facilitating error propagation in public databases. Here, we present uBin, a GUI-based, standalone bin refiner that runs on all major operating platforms and was additionally designed for educational purposes. When applied to the public CAMI dataset, refinement of bins using GC content, coverage and taxonomy was able to improve 78.9% of bins by decreasing their contamination. We also applied the bin refiner as a standalone binner to public metagenomes from the International Space Station and demonstrate the recovery of near-complete genomes, whose replication indices indicate active proliferation of microbes in Earth's lower orbit. uBin is an easy to install software for bin refinement, binning of simple metagenomes and communication of metagenomic results to other scientists and in classrooms. The software and its helper scripts are open source and available under https://github.com/ProbstLab/uBin. This article is protected by copyright. All rights reserved.

[Source](https://ami-journals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.16351)

10.1111/1462-2920.16351

---

## In-situ metagenomics: A platform for rapid sequencing and analysis of metagenomes in less than one day

[Source](https://www.biorxiv.org/content/10.1101/2023.01.25.525498v2)

---

## The virtual microbiome: A computational framework to evaluate microbiome analyses.
 10.02.2023, _B Serrano-Antón, F Rodríguez-Ventura, P Colomer-Vidal, RA Cigliano, CF Arias, F Bertocchini_


_Animals, Reproducibility of Results, Microbiota, Bacteria, Larva, Moths_
Microbiomes have been the focus of a substantial research effort in the last decades. The composition of microbial populations is normally determined by comparing DNA sequences sampled from those populations with the sequences stored in genomic databases. Therefore, the amount of information available in databanks should be expected to constrain the accuracy of microbiome analyses. Albeit normally ignored in microbiome studies, this constraint could severely compromise the reliability of microbiome data. To test this hypothesis, we generated virtual bacterial populations that exhibit the ecological structure of real-world microbiomes. Confronting the analyses of virtual microbiomes with their original composition revealed critical issues in the current approach to characterizing microbiomes, issues that were empirically confirmed by analyzing the microbiome of Galleria mellonella larvae. To reduce the uncertainty of microbiome data, the effort in the field must be channeled towards significantly increasing the amount of available genomic information and optimizing the use of this information.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0280391)

10.1371/journal.pone.0280391

---

## @Letstalkmicro1

[Source](https://twitter.com/Letstalkmicro1?ref_src=twsrc%5Etfw)

---

## https://t.co/x2ZhgNYdaT

[Source](https://t.co/x2ZhgNYdaT)

---

## February 8, 2023

[Source](https://twitter.com/svetlana_up/status/1623307075458633730?ref_src=twsrc%5Etfw)

---

