## A relational framework for microbiome research with Indigenous communities.
 03.10.2023, _AC Bader, EM Van Zuylen, M Handsley-Davis, RA Alegado, A Benezra, RM Pollet, H Ehau-Taumaunu, LS Weyrich, MZ Anderson_


_Humans, Population Groups, Microbiota_
Ethical practices in human microbiome research have failed to keep pace with scientific advances in the field. Researchers seeking to 'preserve' microbial species associated with Indigenous groups, but absent from industrialized populations, have largely failed to include Indigenous people in knowledge co-production or benefit, perpetuating a legacy of intellectual and material extraction. We propose a framework centred on relationality among Indigenous peoples, researchers and microbes, to guide ethical microbiome research. Our framework centres accountability to flatten historical power imbalances that favour researcher perspectives and interests to provide space for Indigenous worldviews in pursuit of Indigenous research sovereignty. Ethical inclusion of Indigenous communities in microbiome research can provide health benefits for all populations and reinforce mutually beneficial partnerships between researchers and the public.

[Source](https://doi.org/10.1038/s41564-023-01471-2)

10.1038/s41564-023-01471-2

---

## Microbiome ownership for Indigenous peoples.
 03.10.2023, _M Handsley-Davis, MZ Anderson, AC Bader, H Ehau-Taumaunu, K Fox, E Kowal, LS Weyrich_


_Humans, Ownership, Indigenous Peoples_
Several studies have reported increased microbial diversity, or distinct microbial community compositions, in the microbiomes of Indigenous peoples around the world. However, there is a widespread failure to include Indigenous cultures and perspectives in microbiome research programmes, and ethical issues pertaining to microbiome research involving Indigenous participants have not received enough attention. We discuss the benefits and risks arising from microbiome research involving Indigenous peoples and analyse microbiome ownership as an ethical concept in this context. We argue that microbiome ownership represents an opportunity for Indigenous peoples to steward and protect their resident microbial communities at every stage of research.

[Source](https://doi.org/10.1038/s41564-023-01470-3)

10.1038/s41564-023-01470-3

---

## Expanded microbiome niches of RAG-deficient patients

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00372-5)

---

## Low dietary fiber intake impairs small intestinal Th17 and intraepithelial T&nbsp;cell development over generations

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(23)01152-X)

---

## Iron acquisition by a commensal bacterium modifies host nutritional immunity during <em>Salmonella</em> infection

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312823003438?via%3Dihub)

---

## Microbiome profile and calprotectin levels as markers of risk of recurrent <em>Clostridioides difficile</em> infection

[Source](https://doi.org/10.3389/fcimb.2023.1237500)

---

## Safety and tolerability of frozen, capsulized autologous faecal microbiota transplantation. A randomized double blinded phase I clinical trial

[Source](https://doi.org/10.1371/journal.pone.0292132)

---

## Metabolism of azo food dyes by bacterial members of the human gut microbiome

[Source](https://doi.org/10.1016/j.anaerobe.2023.102783)

---

## Genetic risk score in multiple sclerosis is associated with unique gut microbiome

[Source](https://doi.org/10.1038/s41598-023-43217-4)

---

## Antisense inhibition of RNA polymerase α subunit of <em>Clostridioides difficile</em>

[Source](https://doi.org/10.1128/spectrum.01755-23)

---

## Metabolic signature for a dysbiotic microbiome in the female genital tract: A systematic review and meta-analysis

[Source](https://doi.org/10.1111/aji.13781)

---

## Influence of Plant-Based Diet on the Cardiovascular System: A Narrative Review

[Source](https://doi.org/10.1097/CRD.0000000000000613)

---

## Bile acid and its bidirectional interactions with gut microbiota: a review

[Source](https://doi.org/10.1080/1040841X.2023.2262020)

---

## Population mixing mediates the intestinal flora composition and facilitates invasiveness in a globally invasive fruit fly

[Source](https://doi.org/10.1186/s40168-023-01664-1)

---

## Diet-related factors strongly shaped the gut microbiota of Japanese macaques

[Source](https://doi.org/10.1002/ajp.23555)

---

## Skin and gut microbiomes of tadpoles vary differently with host and water environment: a short-term experiment using 16S metabarcoding

[Source](https://doi.org/10.1038/s41598-023-43340-2)

---

## Multi-omics revealed the long-term effect of ruminal keystone bacteria and the microbial metabolome on lactation performance in adult dairy goats.
 03.10.2023, _D Wang, L Chen, G Tang, J Yu, J Chen, Z Li, Y Cao, X Lei, L Deng, S Wu, LL Guan, J Yao_


_Humans, Animals, Female, Adolescent, Lactation, Diet, Propionates, Multiomics, Bacteria, Metabolome, Goats, Carbohydrates, Rumen, Fermentation, Animal Feed_
The increased growth rate of young animals can lead to higher lactation performance in adult goats; however, the effects of the ruminal microbiome on the growth of young goats, and the contribution of the early-life rumen microbiome to lifelong growth and lactation performance in goats has not yet been well defined. Hence, this study assessed the rumen microbiome in young goats with different average daily gains (ADG) and evaluated its contribution to growth and lactation performance during the first lactation period.

[Source](https://doi.org/10.1186/s40168-023-01652-5)

10.1186/s40168-023-01652-5

---

## The multi-kingdom microbiome of the goat gastrointestinal tract.
 03.10.2023, _Y Cao, T Feng, Y Wu, Y Xu, L Du, T Wang, Y Luo, Y Wang, Z Li, Z Xuan, S Chen, N Yao, NL Gao, Q Xiao, K Huang, X Wang, K Cui, SU Rehman, X Tang, D Liu, H Han, Y Li, WH Chen, Q Liu_


_Animals, Humans, Goats, Microbiota, Gastrointestinal Tract, Bacteria, Archaea, Methane_
Goat is an important livestock worldwide, which plays an indispensable role in human life by providing meat, milk, fiber, and pelts. Despite recent significant advances in microbiome studies, a comprehensive survey on the goat microbiomes covering gastrointestinal tract (GIT) sites, developmental stages, feeding styles, and geographical factors is still unavailable. Here, we surveyed its multi-kingdom microbial communities using 497 samples from ten sites along the goat GIT.

[Source](https://doi.org/10.1186/s40168-023-01651-6)

10.1186/s40168-023-01651-6

---

## Multi-breed host rumen epithelium transcriptome and microbiome associations and their relationship with beef cattle feed efficiency.
 30.09.2023, _PAS Fonseca, S Lam, Y Chen, SM Waters, LL Guan, A Cánovas_


Understanding host-microbial interactions in the rumen and its influence on desirable production traits may lead to potential microbiota manipulation or genetic selection for improved cattle feed efficiency. This study investigated the host transcriptome and its correlation with the rumen archaea and bacteria differential abundance of two pure beef cattle breeds (Angus and Charolais) and one composite beef hybrid (Kinsella) divergent for residual feed intake (RFI; low-RFI vs. high-RFI). Using RNA-Sequencing of rumen tissue and 16S rRNA gene amplicon sequencing, differentially expressed genes (FDR ≤ 0.05, |log2(Fold-change) &gt;|2) and differentially abundant (p-value &lt; 0.05) archaea and bacteria amplicon sequence variants (ASV) were determined. Significant correlations between gene expression and ASVs (p-value &lt; 0.05) were determine using Spearman correlation. Interesting associations with muscle contraction and the modulation of the immune system were observed for the genes correlated with bacterial ASVs. Potential functional candidate genes for feed efficiency status were identified for Angus (CCL17, CCR3, and CXCL10), Charolais (KCNK9, GGT1 and IL6), and Kinsella breed (ESR2). The results obtained here provide more insights regarding the applicability of target host and rumen microbial traits for the selection and breeding of more feed efficient beef cattle.

[Source](https://doi.org/10.1038/s41598-023-43097-8)

10.1038/s41598-023-43097-8

---

## Select symbionts drive high IgA levels in the mouse intestine.
 30.09.2023, _S Zhang, Y Han, W Schofield, M Nicosia, PE Karell, KP Newhall, JY Zhou, RJ Musich, S Pan, A Valujskikh, N Sangwan, M Dwidar, Q Lu, TS Stappenbeck_


Immunoglobulin A (IgA) is an important factor in maintaining homeostasis at mucosal surfaces, yet luminal IgA levels vary widely. Total IgA levels are thought to be driven by individual immune responses to specific microbes. Here, we found that the prebiotic, pectin oligosaccharide (pec-oligo), induced high IgA levels in the small intestine in a T cell-dependent manner. Surprisingly, this IgA-high phenotype was retained after cessation of pec-oligo treatment, and microbiome transmission either horizontally or vertically was sufficient to retain high IgA levels in the absence of pec-oligo. Interestingly, the bacterial taxa enriched in the overall pec-oligo bacterial community differed from IgA-coated microbes in this same community. Rather, a group of ethanol-resistant microbes, highly enriched for Lachnospiraceae bacterium A2, drove the IgA-high phenotype. These findings support a model of intestinal adaptive immunity in which a limited number of microbes can promote durable changes in IgA directed to many symbionts.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312823003682?via%3Dihub)

10.1016/j.chom.2023.09.001

---

## Blocked conversion of <em>Lactobacillus johnsonii</em> derived acetate to butyrate mediates copper-induced epithelial barrier damage in a pig model

[Source](https://doi.org/10.1186/s40168-023-01655-2)

---

## Laboratory mice with a wild microbiota generate strong allergic immune responses.
 02.10.2023, _J Ma, E Urgard, S Runge, CH Classon, L Mathä, JM Stark, L Cheng, JA Álvarez, S von Zedtwitz, A Baleviciute, S Martinez Hoyer, M Li, AM Gernand, L Osbelt, AA Bielecka, TR Lesker, HJ Huang, S Vrtala, L Boon, R Beyaert, M Adner, I Martinez Gonzalez, T Strowig, J Du, S Nylén, SP Rosshart, JM Coquet_


_Mice, Animals, Th2 Cells, Hypersensitivity, Cytokines, Allergens, Immunity_
Allergic disorders are caused by a combination of hereditary and environmental factors. The hygiene hypothesis postulates that early-life microbial exposures impede the development of subsequent allergic disease. Recently developed "wildling" mice are genetically identical to standard laboratory specific pathogen-free (SPF) mice but are housed under seminatural conditions and have rich microbial exposures from birth. Thus, by comparing conventional SPF mice with wildlings, we can uncouple the impact of lifelong microbial exposures from genetic factors on the allergic immune response. We found that wildlings developed larger populations of antigen-experienced T cells than conventional SPF mice, which included interleukin-10-producing CD4 T cells specific for commensal Lactobacilli strains and allergy-promoting T helper 2 (TH2) cells. In models of airway exposure to house dust mite (HDM), recombinant interleukin-33, or Alternaria alternata, wildlings developed strong allergic inflammation, characterized by eosinophil recruitment, goblet cell metaplasia, and antigen-specific immunoglobulin G1 (IgG1) and IgE responses. Wildlings developed robust de novo TH2 cell responses to incoming allergens, whereas preexisting TH2 cells could also be recruited into the allergic immune response in a cytokine-driven and TCR-independent fashion. Thus, wildling mice, which experience diverse and lifelong microbial exposures, were not protected from developing pathological allergic immune responses. Instead, wildlings mounted robust allergic responses to incoming allergens, shedding new light on the hygiene hypothesis.

[Source](https://doi.org/10.1126/sciimmunol.adf7702)

10.1126/sciimmunol.adf7702

---

## Host genetic variation drives the differentiation in the ecological role of the native <em>Miscanthus</em> root-associated microbiome

[Source](https://doi.org/10.1186/s40168-023-01646-3)

---

## Changes in structure and assembly of a species-rich soil natural community with contrasting nutrient availability upon establishment of a plant-beneficial <em>Pseudomonas</em> in the wheat rhizosphere

[Source](https://doi.org/10.1186/s40168-023-01660-5)

---

## Microbially produced fertilizer provides rhizobacteria to hydroponic tomato roots by forming beneficial biofilms

[Source](https://doi.org/10.1007/s00253-023-12794-9)

---

## Terpenes modulate bacterial and fungal growth and sorghum rhizobiome communities

[Source](https://doi.org/10.1128/spectrum.01332-23)

---

## Rhizosphere and detritusphere habitats modulate expression of soil N-cycling genes during plant development

[Source](https://doi.org/10.1128/msystems.00315-23)

---

## The presence of benzene ring activating CoA ligases for aromatics degradation in the ANaerobic MEthanotrophic (ANME) archaea

[Source](https://doi.org/10.1128/spectrum.01766-23)

---

## Microbial biogeography of pit mud from an artificial brewing ecosystem on a large time scale: all roads lead to Rome

[Source](https://doi.org/10.1128/msystems.00564-23)

---

## Stable microbial community in compacted bentonite after 5 years of exposure to natural granitic groundwater

[Source](https://doi.org/10.1128/msphere.00048-23)

---

## Impact of Experimental Bias on Compositional Analysis of Microbiome Data

[Source](https://doi.org/10.3390/genes14091777)

---

## happi: a hierarchical approach to pangenomics inference

[Source](https://doi.org/10.1186/s13059-023-03040-6)

---

## Integrating a tailored recurrent neural network with Bayesian experimental design to optimize microbial community functions

[Source](https://doi.org/10.1371/journal.pcbi.1011436)

---

## Uncultured prokaryotic genomes in the spotlight: An examination of publicly available data from metagenomics and single-cell genomics

[Source](https://www.csbj.org/article/S2001-0370(23)00322-7/fulltext)

---

