## CRISPR screens in the era of microbiomes.
 28.08.2020, _F Rousset, D Bikard_


Recent advances in genomics have uncovered the tremendous diversity and richness of microbial ecosystems. New functional genomics methods are now needed to probe gene function in high-throughput and provide mechanistic insights. Here, we review how the CRISPR toolbox can be used to inactivate, repress or overexpress genes in a sequence-specific manner and how this offers diverse attractive solutions to identify gene function in high-throughput. Developed both in eukaryotes and prokaryotes, CRISPR screening technologies have already provided meaningful insights in microbiology and host-pathogen interactions. In the era of microbiomes, the versatility and the functional diversity of CRISPR-derived tools has the potential to significantly improve our understanding of microbial communities and their interaction with the host.

[Source](https://www.sciencedirect.com/science/article/pii/S1369527420300904?via%3Dihub)

10.1016/j.mib.2020.07.009

---

## Genetic Transformation of Fusobacterium nucleatum.
 20.08.2020, _A Yoshida, A Ikegami_


Fusobacterium nucleatum is a human periodontal pathogen that causes opportunistic infections. It has been implicated in preterm birth and has as a pathogen of colorectal cancer. However, it is a common member of the oral microbiota and can have a symbiotic relationship with its hosts. To date, studies of F. nucleatum have been hindered by a lack of effective genetic tools, and the transformation of F. nucleatum has not been investigated. In this chapter, protocols for the transformation of F. nucleatum strain 12230 using sonoporation are presented. We also include a genetic complementation protocol for a F. nucleatum knockout mutant.

[Source](https://link.springer.com/protocol/10.1007%2F978-1-0716-0939-2_5)

10.1007/978-1-0716-0939-2_5

---

## Gut microbiota: a potential therapeutic target for Parkinson's disease.
 29.08.2020, _JF Baizabal-Carvallo_



[Source](http://www.nrronline.org/article.asp?issn=1673-5374;year=2021;volume=16;issue=2;spage=287;epage=288;aulast=Baizabal%2DCarvallo)

10.4103/1673-5374.290896

---

## Processing has differential effects on microbiota accessible carbohydrates in whole grains during&nbsp;<em>in vitro</em>&nbsp;fermentation

[Source](https://aem.asm.org/content/early/2020/08/24/AEM.01705-20)

---

## Gut microbiota and oral contraceptive use in overweight and obese patients with polycystic ovary syndrome.
 29.08.2020, _ND Eyupoglu, K Ergunay, A Acikgoz, Y Akyon, E Yilmaz, BO Yildiz_


Polycystic ovary syndrome (PCOS) is a common and complex endocrine disorder. Emerging animal and human data point out to various changes in microbiota that could be linked with the syndrome. However, the effects of therapeutic approaches on gut microbial composition in women with PCOS remain unknown.

[Source](https://pubmed.ncbi.nlm.nih.gov/32860695/)

10.1210/clinem/dgaa600

---

## Diversity analysis of gut microbiota between healthy controls and those with atopic dermatitis in a Chinese population

[Source](https://onlinelibrary.wiley.com/doi/abs/10.1111/1346-8138.15530)

---

## Faecal microbiota transplantation for the treatment of diarrhoea induced by tyrosine-kinase inhibitors in patients with metastatic renal cell carcinoma.
 29.08.2020, _G Ianiro, E Rossi, AM Thomas, G Schinzari, L Masucci, G Quaranta, CR Settanni, LR Lopetuso, F Armanini, A Blanco-Miguez, F Asnicar, C Consolandi, R Iacovelli, M Sanguinetti, G Tortora, A Gasbarrini, N Segata, G Cammarota_


Diarrhoea is one of the most burdensome and common adverse events of chemotherapeutics, and has no standardised therapy to date. Increasing evidence suggests that the gut microbiome can influence the development of chemotherapy-induced diarrhoea. Here we report findings from a randomised clinical trial of faecal microbiota transplantation (FMT) to treat diarrhoea induced by tyrosine kinase inhibitors (TKI) in patients with metastatic renal cell carcinoma (ClinicalTrials.gov number: NCT04040712). The primary outcome is the resolution of diarrhoea four weeks after the end of treatments. Twenty patients are randomised to receive FMT from healthy donors or placebo FMT (vehicle only). Donor FMT is more effective than placebo FMT in treating TKI-induced diarrhoea, and a successful engraftment is observed in subjects receiving donor faeces. No serious adverse events are observed in both treatment arms. The trial meets pre-specified endpoints. Our findings suggest that the therapeutic manipulation of gut microbiota may become a promising treatment option to manage TKI-dependent diarrhoea.

[Source](https://www.nature.com/articles/s41467-020-18127-y)

10.1038/s41467-020-18127-y

---

## Integrated microbiota and metabolite profiles link Crohn's disease to sulfur metabolism.
 29.08.2020, _A Metwaly, A Dunkel, N Waldschmitt, ACD Raj, I Lagkouvardos, AM Corraliza, A Mayorgas, M Martinez-Medina, S Reiter, M Schloter, T Hofmann, M Allez, J Panes, A Salas, D Haller_


Gut microbial and metabolite alterations have been linked to the pathogenesis of inflammatory bowel diseases. Here we perform a multi-omics microbiome and metabolite analysis of a longitudinal cohort of Crohn's disease patients undergoing autologous hematopoietic stem cell transplantation, and investigational therapy that induces drug free remission in a subset of patients. Via comparison of patients who responded and maintained remission, responded but experienced disease relapse and patients who did not respond to therapy, we identify shared functional signatures that correlate with disease activity despite the variability of gut microbiota profiles at taxonomic level. These signatures reflect the disease state when transferred to gnotobiotic mice. Taken together, the integration of microbiome and metabolite profiles from human cohort and mice improves the predictive modelling of disease outcome, and allows the identification of a network of bacteria-metabolite interactions involving sulfur metabolism as a key mechanism linked to disease activity in Crohn's disease.

[Source](https://www.nature.com/articles/s41467-020-17956-1)

10.1038/s41467-020-17956-1

---

## The potential role of the gut microbiota in shaping host energetics and metabolic rate.
 28.08.2020, _EC Lindsay, NB Metcalfe, MS Llewellyn_


It is increasingly recognised that symbiotic microbiota (especially those present in the gut) have important influences on the functioning of their host. Here we review the interplay between this microbial community and the growth, metabolic rate and nutritional energy harvest of the host. We show how recent developments in experimental and analytical methods have allowed much easier characterisation of the nature, and increasingly the functioning, of the gut microbiota. Manipulation studies that remove or augment gut microorganisms or transfer them between hosts have allowed unprecedented insights into their impact. While much of the information to date has come from studies of laboratory model organisms, recent studies have used a more diverse range of host species, including those living in natural conditions, revealing their ecological relevance. The gut microbiota can provide the host with dietary nutrients that would be otherwise unobtainable, as well as allow the host flexibility in its capacity to cope with changing environments. The composition of the gut microbial community of a species can vary seasonally or when the host moves between environments (e.g. fresh and sea water in the case of migratory fish). It can also change with host diet choice, metabolic rate (or demands) and life stage. These changes in gut microbial community composition enable the host to live within different environments, adapt to seasonal changes in diet and maintain performance throughout its entire life history, highlighting the ecological relevance of the gut microbiota. While it is evident that gut microbes can underpin host metabolic plasticity, the causal nature of associations between particular microorganisms and host performance is not always clear unless a manipulative approach has been used. Many studies have focussed on a correlative approach by characterising microbial community composition, but there is now a need for more experimental studies in both wild and laboratory-based environments, to reveal the true role of gut microbiota in influencing the functioning of their hosts, including its capacity to tolerate environmental change. We highlight areas where these would be particularly fruitful in the context of ecological energetics.

[Source](https://besjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1365-2656.13327)

10.1111/1365-2656.13327

---

## Microbiological analysis of endodontically treated teeth with apical periodontitis before and after endodontic retreatment.
 29.08.2020, _M Barbosa-Ribeiro, R Arruda-Vasconcelos, LM Louzada, DG Dos Santos, FD Andreote, BPFA Gomes_


To characterize the microbiota of teeth with endodontic treatment failure by 16S ribosomal RNA genetic sequencing (GS) and PCR at the different phases of the endodontic retreatment and to associate the presence of specific bacteria with clinical and radiographic features in teeth with apical periodontitis.

[Source](https://link.springer.com/article/10.1007/s00784-020-03510-2)

10.1007/s00784-020-03510-2

---

## 2000-year-old pathogen genomes reconstructed from metagenomic analysis of Egyptian mummified individuals.
 29.08.2020, _J Neukamm, S Pfrengle, M Molak, A Seitz, M Francken, P Eppenberger, C Avanzi, E Reiter, C Urban, B Welte, PW Stockhammer, B Te√ümann, A Herbig, K Harvati, K Nieselt, J Krause, VJ Schuenemann_


Recent advances in sequencing have facilitated large-scale analyses of the metagenomic composition of different samples, including the environmental microbiome of air, water, and soil, as well as the microbiome of living humans and other animals. Analyses of the microbiome of ancient human samples may provide insights into human health and disease, as well as pathogen evolution, but the field is still in its very early stages and considered highly challenging.

[Source](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-020-00839-8)

10.1186/s12915-020-00839-8

---

## Comparing the nasal bacterial microbiome diversity of allergic rhinitis, chronic rhinosinusitis and control subjects.
 29.08.2020, _W Gan, F Yang, J Meng, F Liu, S Liu, J Xian_


Until now, the microbiome of the nasal cavity and its contribution to nasal mucosal disease has remained poorly understood. The advent of cultivation-free molecular methods makes it possible to characterize the total microbiome of the nasal cavity. We sought to assess the microbial diversity and composition of the middle meatus in allergic rhinitis (AR) patients, chronic rhinosinusitis patients without polyps (CRSsNP) and a control population to determine the microbiota associated with the pathogenesis of AR and CRSsNP.

[Source](https://link.springer.com/article/10.1007%2Fs00405-020-06311-1)

10.1007/s00405-020-06311-1

---

## Maternal Oral Health Influences Infant Salivary Microbiome.
 29.08.2020, _K Ramadugu, D Bhaumik, T Luo, RE Gicquelais, KH Lee, EB Stafford, CF Marrs, K Neiswanger, DW McNeil, ML Marazita, B Foxman_


Oral microbiomes vary in cariogenic potential; these differences may be established early in life. A major concern is whether mothers transmit cariogenic bacteria to their children. Here we characterize early salivary microbiome development and the potential associations of that development with route of delivery, breastfeeding, and mother's oral health, and we evaluate transmission of microbes between mother and child. We analyzed saliva and metadata from the Center for Oral Health Research in Appalachia. For this cohort study, we sequenced the V6 region of the 16S rRNA gene and used quantitative polymerase chain reaction to detect Streptococcus mitis, Streptococcus sobrinus, Streptococcus mutans, Streptococcus oralis, and Candida albicans in the saliva from mothers and their infants, collected at 2, 9, and 12 mo (Pennsylvania site) and 2, 12, and 24 mo (West Virginia site). Breastfed children had lower relative abundances of Prevotella and Veillonella. If mothers had decayed, missing, or filled teeth, children had greater abundances of Veillonella and Actinomyces. There was little evidence of maternal transmission of selected microbes. At 12 mo, children's microbiomes were more similar to other children's than to their mothers'. Infants' salivary microbiomes became more adult-like with age but still differed with mothers' microbiomes at 12 mo. There was little evidence supporting transmission of selected microbes from mothers to children, but risk of colonization was associated with tooth emergence. Children are likely to acquire cariogenic bacteria from a variety of sources, including foods and contact with other children and adults.

[Source](https://journals.sagepub.com/doi/10.1177/0022034520947665)

10.1177/0022034520947665

---

## Dietary supplementation of sodium sulfate improves rumen fermentation, fiber digestibility and plasma metabolome through modulating rumen bacterial communities in steers.
 29.08.2020, _Y Zhao, B Xie, J Gao, G Zhao_


Six steers were used to study the effects of dietary supplementation with sodium sulfate (Na2SO4) on rumen fermentation, nutrient digestion, rumen microbiota and plasma metabolites. The animals were fed a basal ration added with Na2SO4 at 0 [sulfur (S) content 0.115% DM], 20 (S 0.185% DM) and 40 g/d (S 0.255%DM), respectively, in a replicate 3 √ó 3 Latin square design. Results indicated that supplementing Na2SO4 increased the ruminal concentration of total volatile fatty acids, the molar proportions of acetate and butyrate, the ruminal concentrations of microbial protein, SO42--S and S2--S and the digestibility of fiber while it decreased the molar proportion of propionate and the ruminal concentration of ammonia nitrogen. Supplementing Na2SO4 increased the diversity and the richness of rumen microbiota and the relative abundances of phylum Firmicutes, and genera Ruminococcus 2, Rikenellaceae RC9 gut group and Desulfovibrio whereas it decreased that of phylum Bacteroidetes and genera Prevotella 1, Prevotellaceae UCG-001 and Treponema 2 Supplementing Na2SO4 also increased the plasma concentrations of amino acids (L-arginine, L-methionine, L-cysteine, L-lysine), purine derivatives (xanthine, hypoxanthine), vitamins (thiamin, biotin) and lipids (acetylcarnitine and L-carnitine). It was concluded supplementation with Na2SO4 to steer ration was beneficial to improve the rumen fermentation, fiber digestibility and nutrient metabolism through modulating rumen microbial community.IMPORTANCE Essential elements such as nitrogen and sulfur greatly affect rumen fermentation and metabolism in ruminants. However, little knowledge is available on the effects of sulfur on rumen microbiota and plasma metabolome. Results of the present trial demonstrated that supplementation with sodium sulfate to steer ration markedly improved rumen fermentation, fiber digestibility and metabolism of amino acids, purine derivatives and vitamins through the impacts on the ruminal microbiome. The facts obtained from the present trial clarified the possible mechanisms of the positive effects of sulfur on rumen fermentation and nutrient utilization.

[Source](https://pubmed.ncbi.nlm.nih.gov/32859601/)

10.1128/AEM.01412-20

---

## <em>Salmonella enterica</em>&nbsp;serovar Typhimurium Temporally Modulates the Enteric Microbiota and Host Responses to Overcome Colonization Resistance in Swine

[Source](https://aem.asm.org/content/early/2020/08/24/AEM.01569-20)

---

## Germ-Free Conditions Modulate Host Purine Metabolism, Exacerbating Adenine-Induced Kidney Damage.
 29.08.2020, _E Mishima, M Ichijo, T Kawabe, K Kikuchi, Y Akiyama, T Toyohara, T Suzuki, C Suzuki, A Asao, N Ishii, S Fukuda, T Abe_


Alterations in microbiota are known to affect kidney disease conditions. We have previously shown that germ-free conditions exacerbated adenine-induced kidney damage in mice; however, the mechanism by which this occurs has not been elucidated. To explore this mechanism, we examined the influence of germ-free conditions on purine metabolism and renal immune responses involved in the kidney damage. Germ-free mice showed higher expression levels of purine-metabolizing enzymes such as xanthine dehydrogenase, which converts adenine to a nephrotoxic byproduct 2,8-dihydroxyadenine (2,8-DHA). The germ-free mice also showed increased urinary excretion of allantoin, indicating enhanced purine metabolism. Metabolome analysis demonstrated marked differences in the purine metabolite levels in the feces of germ-free mice and mice with microbiota. Furthermore, unlike the germ-free condition, antibiotic treatment did not increase the expression of purine-metabolizing enzymes or exacerbate adenine-induced kidney damage. Considering renal immune responses, the germ-free mice displayed an absence of renal IL-17A expression. However, the adenine-induced kidney damage in wild-type mice was comparable to that in IL-17A-deficient mice, suggesting that IL-17A does not play a major role in the disease condition. Our results suggest that the enhanced host purine metabolism in the germ-free mice potentially promotes the conversion of the administered adenine into 2,8-DHA, resulting in exacerbated kidney damage. This further suggests a role of the microbiota in regulating host purine metabolism.

[Source](https://www.mdpi.com/2072-6651/12/9/547)

10.3390/toxins12090547

---

## Conventional Co-Housing Modulates Murine Gut Microbiota and Hematopoietic Gene Expression.
 29.08.2020, _J Chen, S Zhang, X Feng, Z Wu, W Dubois, V Thovarai, S Ahluwalia, S Gao, J Chen, T Peat, SK Sen, G Trinchieri, NS Young, BA Mock_


Specific-pathogen-free (SPF) mice have improved hematopoietic characteristics relative to germ-free mice, however, it is not clear whether improvements in hematopoietic traits will continue when the level of microorganism exposure is further increased. We co-housed SPF C57BL/6 mice in a conventional facility (CVT) and found a significant increase in gut microbiota diversity along with increased levels of myeloid cells and T cells, especially effector memory T cells. Through single cell RNA sequencing of sorted KL (c-Kit+Lin-) cells, we imputed a decline in long-term hematopoietic stem cells and an increase in granulocyte-monocyte progenitors in CVT mice with up-regulation of genes associated with cell survival. Bone marrow transplantation through competitive repopulation revealed a significant increase in KSL (c-Kit+Sca-1+Lin-) cell reconstitution in recipients of CVT donor cells which occurred when donors were co-housed for both one and twelve months. However, there was minimal to no gain in mature blood cell engraftment in recipients of CVT donor cells relative to those receiving SPF donor cells. We conclude that co-housing SPF mice with mice born in a conventional facility increased gut microbiota diversity, augmented myeloid cell production and T cell activation, stimulated KSL cell reconstitution, and altered hematopoietic gene expression.

[Source](https://www.mdpi.com/1422-0067/21/17/6143)

10.3390/ijms21176143

---

## Suppression of rice planthopper populations without affecting the rice microbiota by the entomopathogenic fungus&nbsp;

[Source](https://aem.asm.org/content/early/2020/08/24/AEM.01337-20)

---

## Suppression of rice planthopper populations without affecting the rice microbiota by the entomopathogenic fungus Metarhizium anisopliae.
 29.08.2020, _Y Peng, J Tang, M Hong, J Xie_


Entomopathogenic fungi can regulate insect populations and function as crucial biological control agents against insect pests, but their impacts on nontarget microorganisms are poorly understood. In this study, we investigated the potential of the fungal strain Metarhizium anisopliae CQMa421 to control rice planthoppers under field conditions and its effects on rice microbiota. This fungus suppressed rice planthoppers during this period, and its control efficiency was more than 60% 7 days post application and did not significantly differ from that of the chemical treatment except in 2019. Both treatments showed a lower population of rice planthoppers than the controls. After application, M. anisopliae was maintained on rice plants for approximately 14 days, showing a decreasing trend over time. Furthermore, the results showed that the bacterial and fungal richness (operational taxonomic units) and diversity (Shannon index) did not significantly differ between the fungal treatment and the controls after application. The major bacterial taxa of Proteobacteria (including Alphaproteobacteria, Betaproteobacteria, Gammaproteobacteria, and Deltaproteobacteria), Actinobacteria, Bacteroidetes and Cyanobacteria accounted for more than 80% of the bacterial community after fungal application, and the major fungal taxa of Ascomycota (including Eurotiomycetes, Dothideomycetes and Sordariomycetes), and Basidiomycota (including Ustilaginomycetes) represented more than 90% of fungal community. However, the microbial communities of the rice phyllosphere did not significantly change after entomopathogenic agent application, indicating that the indigenous microbial communities may adapt to fungal insecticide application. Taken together, the results suggest that this fungal agent has good potential for rice planthopper control with no substantial effects on rice microbial communities.IMPORTANCE Entomopathogenic fungi may be used as crucial biocontrol agents for the control of insect pests, but few effective fungal strains have been reported for the control of the rice planthopper, a major pest of rice. More importantly, the impacts of fungal insecticide application on nontarget microorganisms have not been well evaluated, especially under field conditions. Therefore, in this study, we investigated the effects of the fungal strain M. anisopliae CQMa421 on rice planthopper populations from 2017 to 2019 and evaluated its potential impacts on the microbiota of rice plants after application. The results suggested that this fungal agent has good potential for use in the control of rice planthoppers with no significant effects on rice microbial communities, representing an alternative strategy for the control of rice pests.

[Source](https://aem.asm.org/content/early/2020/08/24/AEM.01337-20)

10.1128/AEM.01337-20

---

## Metarhizium anisopliae

[Source](https://aem.asm.org/content/early/2020/08/24/AEM.01337-20)

---

## Dissolved microcystin release coincident with lysis of a&nbsp;<em>Microcystis</em>-dominated bloom in western Lake Erie attributed to a novel cyanophage

[Source](https://pubmed.ncbi.nlm.nih.gov/32859600/)

---

## Ultrafast and accurate 16S rRNA microbial community analysis using Kraken 2.
 29.08.2020, _J Lu, SL Salzberg_


For decades, 16S ribosomal RNA sequencing has been the primary means for identifying the bacterial species present in a sample with unknown composition. One of the most widely used tools for this purpose today is the QIIME (Quantitative Insights Into Microbial Ecology) package. Recent results have shown that the newest release, QIIME 2, has higher accuracy than QIIME, MAPseq, and mothur when classifying bacterial genera from simulated human gut, ocean, and soil metagenomes, although QIIME 2 also proved to be the most computationally expensive. Kraken, first released in 2014, has been shown to provide exceptionally fast and accurate classification for shotgun metagenomics sequencing projects. Bracken, released in 2016, then provided users with the ability to accurately estimate species or genus relative abundances using Kraken classification results. Kraken 2, which matches the accuracy and speed of Kraken 1, now supports 16S rRNA databases, allowing for direct comparisons to QIIME and similar systems.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00900-2)

10.1186/s40168-020-00900-2

---

## Predicting Host Phenotype Based on Gut Microbiome Using a Convolutional Neural Network Approach.
 18.08.2020, _D Reiman, AM Farhat, Y Dai_


Accurate prediction of the host phenotypes from a microbial sample and identification of the associated microbial markers are important in understanding the impact of the microbiome on the pathogenesis and progression of various diseases within the host. A deep learning tool, PopPhy-CNN, has been developed for the task of predicting host phenotypes using a convolutional neural network (CNN). By representing samples as annotated taxonomic trees and further representing these trees as matrices, PopPhy-CNN utilizes the CNN's innate ability to explore locally similar microbes on the taxonomic tree. Furthermore, PopPhy-CNN can be used to evaluate the importance of each taxon in the prediction of host status. Here, we describe the underlying methodology, architecture, and core utility of PopPhy-CNN. We also demonstrate the use of PopPhy-CNN on a microbial dataset.

[Source](https://link.springer.com/protocol/10.1007%2F978-1-0716-0826-5_12)

10.1007/978-1-0716-0826-5_12

---

## Application of Next Generation Sequencing in Laboratory Medicine.
 28.08.2020, _Y Zhong, F Xu, J Wu, J Schubert, MM Li_


The rapid development of next-generation sequencing (NGS) technology, including advances in sequencing chemistry, sequencing technologies, bioinformatics, and data interpretation, has facilitated its wide clinical application in precision medicine. This review describes current sequencing technologies, including short- and long-read sequencing technologies, and highlights the clinical application of NGS in inherited diseases, oncology, and infectious diseases. We review NGS approaches and clinical diagnosis for constitutional disorders; summarize the application of U.S. Food and Drug Administration-approved NGS panels, cancer biomarkers, minimal residual disease, and liquid biopsy in clinical oncology; and consider epidemiological surveillance, identification of pathogens, and the importance of host microbiome in infectious diseases. Finally, we discuss the challenges and future perspectives of clinical NGS tests.

[Source](http://www.annlabmed.org/journal/view.html?volume=41&number=1&spage=25)

10.3343/alm.2021.41.1.25

---

## Application of Next Generation Sequencing in Laboratory Medicine ‚Äì Yiming Zhong

[Source](http://www.annlabmed.org/journal/view.html?volume=41&number=1&spage=25)

---

