## Multi-kingdom ecological drivers of microbiota assembly in preterm infants.
 25.02.2021, _C Rao, KZ Coyte, W Bainter, RS Geha, CR Martin, S Rakoff-Nahoum_


The gut microbiota of preterm infants develops predictably1-7, with pioneer species colonizing the gut after birth, followed by an ordered succession of microorganisms. The gut microbiota is vital to the health of preterm infants8,9, but the forces that shape these predictable dynamics of microbiome assembly are unknown. The environment, the host and interactions between microorganisms all potentially shape the dynamics of the microbiota, but in such a complex ecosystem, identifying the specific role of any individual factor is challenging10-14. Here we use multi-kingdom absolute abundance quantification, ecological modelling and experimental validation to address this challenge. We quantify the absolute dynamics of bacteria, fungi and archaea in a longitudinal cohort of 178 preterm infants. We uncover microbial blooms and extinctions, and show that there is an inverse correlation between bacterial and fungal loads in the infant gut. We infer computationally and demonstrate experimentally in vitro and in vivo that predictable assembly dynamics may be driven by directed, context-dependent interactions between specific microorganisms. Mirroring the dynamics of macroscopic ecosystems15-17, a late-arriving member of the microbiome, Klebsiella, exploits the pioneer microorganism, Staphylococcus, to gain a foothold within the gut. Notably, we find that interactions between different kingdoms can influence assembly, with a single fungal species-Candida albicans-inhibiting multiple dominant genera of gut bacteria. Our work reveals the centrality of simple microbe-microbe interactions in shaping host-associated microbiota, which is critical both for our understanding of microbiota ecology and for targeted microbiota interventions.

[Source](https://www.nature.com/articles/s41586-021-03241-8)

10.1038/s41586-021-03241-8

---

## A randomization-based causal inference framework for uncovering environmental exposure effects on human gut microbiota

[Source](https://www.biorxiv.org/content/10.1101/2021.02.24.432662v1.abstract)

---

## Fiber-associated Lachnospiraceae reduce colon tumorigenesis by modulation of the tumor-immune microenvironment

[Source](https://www.biorxiv.org/content/10.1101/2021.02.24.432654v1.abstract)

---

## NLRP6-associated host microbiota composition impacts in the intestinal barrier to systemic dissemination of Brucella abortus.
 22.02.2021, _M Rungue, V Melo, D Martins, PC Campos, G Leles, I Galvão, V Mendes, M Aganetti, Á Pedersen, NRG Assis, R Santos, GD Cassali, ALB Godard, FS Martins, SC Oliveira, AT Vieira_


Brucella abortus is a Gram-negative bacterium responsible for a worldwide zoonotic infection-Brucellosis, which has been associated with high morbidity rate in humans and severe economic losses in infected livestock. The natural route of infection is through oral and nasal mucosa but the invasion process through host gut mucosa is yet to be understood. Studies have examined the role of NLRP6 (NOD-like receptor family pyrin domain-containing-6 protein) in gut homeostasis and defense against pathogens. Here, we investigated the impact of gut microbiota and NLRP6 in a murine model of Ba oral infection. Nlrp6-/- and wild-type (WT) mice were infected by oral gavage with Ba and tissues samples were collected at different time points. Our results suggest that Ba oral infection leads to significant alterations in gut microbiota. Moreover, Nlrp6-/- mice were more resistant to infection, with decreased CFU in the liver and reduction in gut permeability when compared to the control group. Fecal microbiota transplantation from WT and Nlrp6-/- into germ-free mice reflected the gut permeability phenotype from the donors. Additionally, depletion of gut microbiota by broad-spectrum-antibiotic treatment prevented Ba replication in WT while favoring bacterial growth in Nlrp6-/-. Finally, we observed higher eosinophils in the gut and leukocytes in the blood of infected Nlrp6-/- compared to WT-infected mice, which might be associated to the Nlrp6-/- resistance phenotype. Altogether, these results indicated that gut microbiota composition is the major factor involved in the initial stages of pathogen host replication and partially also by the resistance phenotype observed in Nlrp6 -/- mice regulating host inflammation against Ba infection.

[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0009171)

10.1371/journal.pntd.0009171

---

## Effects of sodium butyrate supplementation on inflammation, gut microbiota, and short-chain fatty acids in Helicobacter pylori-infected mice.
 20.02.2021, _Y Huang, Y Ding, H Xu, C Shen, X Chen, C Li_


Inflammation induced by Helicobacter pylori (H. pylori) infection is the basis for the pathogenesis of H. pylori. Butyric acid, a diet-related microbial-associated metabolite, is connected to inflammation, metabolic syndrome, and other diseases. Several studies have indicated the effects of sodium butyrate (SB) against bacteria; however, the effects of SB on the main virulence factors of H. pylori, H. pylori-induced inflammation, and gut microbiota composition remain unclear.

[Source](https://onlinelibrary.wiley.com/doi/full/10.1111/hel.12785)

10.1111/hel.12785

---

## The porcine corneal surface bacterial microbiome: A distinctive niche within the ocular surface

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0247392)

---

## Dysbiosis of fecal microbiota in cats with naturally occurring and experimentally induced Tritrichomonas foetus infection.
 22.02.2021, _M Bierlein, BA Hedgespeth, MA Azcarate-Peril, SH Stauffer, JL Gookin_


The protozoal pathogen Tritrichomonas foetus infects the colon of domestic cats and is a major cause of chronic colitis and diarrhea. Treatment failure is common, but antibiotics may improve clinical signs in a subset of cats, leading researchers to question involvement of the colonic microbiota in disease pathogenesis. Studies performed in women with venereal Trichomonas vaginalis infections have revealed that dysbiosis of host microbiota contributes to pathogenicity with similar findings also found in mice with intestinal Tritrichomonas musculis The aim of this study was to characterize differences in the fecal microbiota of cats with and without naturally occurring T. foetus infection and in a group of kittens prior to and after experimentally induced infection. Archived fecal DNA from cats undergoing testing for T. foetus infection (n = 89) and experimentally infected kittens (n = 4; at pre-, 2 weeks, and 9 weeks post-infection) were analyzed by sequencing of 16S rRNA genes. Amongst the naturally infected population, the genera Megamonas and Helicobacter were significantly increased in prevalence and abundance in cats testing positive for T. foetus infection. In the group of four experimentally infected kittens, fecal samples post-infection had significantly lower abundance of genus Dialister and Megamonas and greater abundance of the class Betaproteobacteria and family Succinivibrionaceae. We hypothesize that T. foetus promotes dysbiosis by competition for fermentable substrates used by these bacteria and that metabolic byproducts may contribute to the pathogenesis of colonic inflammation and diarrhea. Future studies are warranted for the measurement of fecal concentrations of microbial and protozoal metabolites in cats with T. foetus infection for the identification of potential therapeutic targets.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0246957)

10.1371/journal.pone.0246957

---

## Hologenome analysis reveals dual symbiosis in the deep-sea hydrothermal vent snail Gigantopelta aegis.
 22.02.2021, _Y Lan, J Sun, C Chen, Y Sun, Y Zhou, Y Yang, W Zhang, R Li, K Zhou, WC Wong, YH Kwan, A Cheng, S Bougouffa, CL Van Dover, JW Qiu, PY Qian_


Animals endemic to deep-sea hydrothermal vents often form obligatory symbioses with bacteria, maintained by intricate host-symbiont interactions. Most genomic studies on holobionts have not investigated both sides to similar depths. Here, we report dual symbiosis in the peltospirid snail Gigantopelta aegis with two gammaproteobacterial endosymbionts: a sulfur oxidiser and a methane oxidiser. We assemble high-quality genomes for all three parties, including a chromosome-level host genome. Hologenomic analyses reveal mutualism with nutritional complementarity and metabolic co-dependency, highly versatile in transporting and using chemical energy. Gigantopelta aegis likely remodels its immune system to facilitate dual symbiosis. Comparisons with Chrysomallon squamiferum, a confamilial snail with a single sulfur-oxidising gammaproteobacterial endosymbiont, show that their sulfur-oxidising endosymbionts are phylogenetically distant. This is consistent with previous findings that they evolved endosymbiosis convergently. Notably, the two sulfur-oxidisers share the same capabilities in biosynthesising nutrients lacking in the host genomes, potentially a key criterion in symbiont selection.

[Source](https://www.nature.com/articles/s41467-021-21450-7)

10.1038/s41467-021-21450-7

---

## The bacterial communities of Alaskan mosses and their contributions to N2-fixation.
 24.02.2021, _H Holland-Moritz, JEM Stuart, LR Lewis, SN Miller, MC Mack, JM Ponciano, SF McDaniel, N Fierer_


Mosses in high-latitude ecosystems harbor diverse bacterial taxa, including N2-fixers which are key contributors to nitrogen dynamics in these systems. Yet the relative importance of moss host species, and environmental factors, in structuring these microbial communities and their N2-fixing potential remains unclear. We studied 26 boreal and tundra moss species across 24 sites in Alaska, USA, from 61 to 69° N. We used cultivation-independent approaches to characterize the variation in moss-associated bacterial communities as a function of host species identity and site characteristics. We also measured N2-fixation rates via 15N2 isotopic enrichment and identified potential N2-fixing bacteria using available literature and genomic information.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01001-4)

10.1186/s40168-021-01001-4

---

## Ecosystem engineers drive differing microbial community composition in intertidal estuarine sediments.
 22.02.2021, _AJ Wyness, I Fortune, AJ Blight, P Browne, M Hartley, M Holden, DM Paterson_


Intertidal systems are complex and dynamic environments with many interacting factors influencing biochemical characteristics and microbial communities. One key factor are the actions of resident fauna, many of which are regarded as ecosystem engineers because of their bioturbation, bioirrigation and sediment stabilising activities. The purpose of this investigation was to elucidate the evolutionary implications of the ecosystem engineering process by identifying, if any, aspects that act as selection pressures upon microbial communities. A mesocosm study was performed using the well characterised intertidal ecosystem engineers Corophium volutator, Hediste diversicolor, and microphytobenthos, in addition to manual turbation of sediments to compare effects of bioturbation, bioirrigation and stabilisation. A range of sediment functions and biogeochemical gradients were measured in conjunction with 16S rRNA sequencing and diatom taxonomy, with downstream bacterial metagenome function prediction, to identify selection pressures that incited change to microbial community composition and function. Bacterial communities were predominantly Proteobacteria, with the relative abundance of Bacteroidetes, Alphaproteobacteria and Verrucomicrobia being partially displaced by Deltaproteobacteria, Acidobacteria and Chloroflexi as dissolved oxygen concentration and redox potential decreased. Bacterial community composition was driven strongly by biogeochemistry; surface communities were affected by a combination of sediment functions and overlying water turbidity, and subsurface communities by biogeochemical gradients driven by sediment reworking. Diatom communities were dominated by Nitzschia laevis and Achnanthes sp., and assemblage composition was influenced by overlying water turbidity (manual or biogenic) rather than direct infaunal influences such as grazing.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0240952)

10.1371/journal.pone.0240952

---

## Rare genera differentiate urban green space soil bacterial communities in three cities across the world

[Source](https://www.biorxiv.org/content/10.1101/2021.02.21.432167v1)

---

## Nitrogen and water addition regulate fungal community and microbial co-occurrence network complexity in the rhizosphere of Alhagi sparsifolia seedlings

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0929139321000615)

---

## Metabolic profiling and antioxidant activity during flower development in Agastache rugosa

[Source](https://link.springer.com/article/10.1007/s12298-021-00945-z)

---

## Long-term dynamic changes in attached and planktonic microbial communities in a contaminated aquifer

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0269749121003456)

---

## Virome composition in marine fish revealed by meta-transcriptomics.
 25.02.2021, _JL Geoghegan, F Di Giallonardo, M Wille, AS Ortiz-Baez, VA Costa, T Ghaly, JCO Mifsud, OMH Turnbull, DR Bellwood, JE Williamson, EC Holmes_


Revealing the determinants of virome composition is central to placing disease emergence in a broader evolutionary context. Fish are the most species-rich group of vertebrates and so provide an ideal model system to study the factors that shape virome compositions and their evolution. We characterized the viromes of nineteen wild-caught species of marine fish using total RNA sequencing (meta-transcriptomics) combined with analyses of sequence and protein structural homology to identify divergent viruses that often evade characterization. From this, we identified twenty-five new vertebrate-associated viruses and a further twenty-two viruses likely associated with fish diet or their microbiomes. The vertebrate-associated viruses identified here included the first fish virus in the Matonaviridae (single-strand, negative-sense RNA virus). Other viruses fell within the Astroviridae, Picornaviridae, Arenaviridae, Reoviridae, Hepadnaviridae, Paramyxoviridae, Rhabdoviridae, Hantaviridae, Filoviridae, and Flaviviridae, and were sometimes phylogenetically distinct from known fish viruses. We also show how key metrics of virome composition-viral richness, abundance, and diversity-can be analysed along with host ecological and biological factors as a means to understand virus ecology. Accordingly, these data suggest that that the vertebrate-associated viromes of the fish sampled here are predominantly shaped by the phylogenetic history (i.e. taxonomic order) of their hosts, along with several biological factors including water temperature, habitat depth, community diversity and swimming behaviour. No such correlations were found for viruses associated with porifera, molluscs, arthropods, fungi, and algae, that are unlikely to replicate in fish hosts. Overall, these data indicate that fish harbour particularly large and complex viromes and the vast majority of fish viromes are undescribed.

[Source](https://academic.oup.com/ve/article/7/1/veab005/6128534)

10.1093/ve/veab005

---

## Comparison of beta diversity measures in clustering the high-dimensional microbial data.
 18.02.2021, _B Chen, X He, B Pan, X Zou, N You_


The heterogeneity of disease is a major concern in medical research and is commonly characterized as subtypes with different pathogeneses exhibiting distinct prognoses and treatment effects. The classification of a population into homogeneous subgroups is challenging, especially for complex diseases. Recent studies show that gut microbiome compositions play a vital role in disease development, and it is of great interest to cluster patients according to their microbial profiles. There are a variety of beta diversity measures to quantify the dissimilarity between the compositions of different samples for clustering. However, using different beta diversity measures results in different clusters, and it is difficult to make a choice among them. Considering microbial compositions from 16S rRNA sequencing, which are presented as a high-dimensional vector with a large proportion of extremely small or even zero-valued elements, we set up three simulation experiments to mimic the microbial compositional data and evaluate the performance of different beta diversity measures in clustering. It is shown that the Kullback-Leibler divergence-based beta diversity, including the Jensen-Shannon divergence and its square root, and the hypersphere-based beta diversity, including the Bhattacharyya and Hellinger, can capture compositional changes in low-abundance elements more efficiently and can work stably. Their performance on two real datasets demonstrates the validity of the simulation experiments.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0246893)

10.1371/journal.pone.0246893

---

