## Genome Analysis of Epsilon CrAss-like Phages.
 27.04.2024, _IV Babkin, AY Tikunov, IK Baykov, VV Morozova, NV Tikunova_


_Genome, Viral, Bacteriophages, Phylogeny, Viral Proteins, Retroelements, Genetic Variation, Prophages, DNA, Viral, DNA Primase, Genomics, RNA-Directed DNA Polymerase_
CrAss-like phages play an important role in maintaining ecological balance in the human intestinal microbiome. However, their genetic diversity and lifestyle are still insufficiently studied. In this study, a novel CrAssE-Sib phage genome belonging to the epsilon crAss-like phage genomes was found. Comparative analysis indicated that epsilon crAss-like phages are divided into two putative genera, which were proposed to be named Epsilonunovirus and Epsilonduovirus; CrAssE-Sib belongs to the former. The crAssE-Sib genome contains a diversity-generating retroelement (DGR) cassette with all essential elements, including the reverse transcriptase (RT) and receptor binding protein (RBP) genes. However, this RT contains the GxxxSP motif in its fourth domain instead of the usual GxxxSQ motif found in all known phage and bacterial DGRs. RBP encoded by CrAssE-Sib and other Epsilonunoviruses has an unusual structure, and no similar phage proteins were found. In addition, crAssE-Sib and other Epsilonunoviruses encode conserved prophage repressor and anti-repressors that could be involved in lysogenic-to-lytic cycle switches. Notably, DNA primase sequences of epsilon crAss-like phages are not included in the monophyletic group formed by the DNA primases of all other crAss-like phages. Therefore, epsilon crAss-like phage substantially differ from other crAss-like phages, indicating the need to classify these phages into a separate family.

[Source](https://doi.org/10.3390/v16040513)

10.3390/v16040513

---

## Cervical microbiota dysbiosis associated with high-risk Human Papillomavirus infection.
 28.04.2024, _N Zeber-Lubecka, M Kulecka, M Dabrowska, K Baginska-Drabiuk, M Glowienka-Stodolak, A Nowakowski, A Slabuszewska-Jozwiak, B Bednorz, I Jędrzejewska, M Piasecka, J Pawelec, E Wojciechowska-Lampka, J Ostrowski_


_Humans, Female, Papillomavirus Infections, Dysbiosis, Microbiota, Adult, Cervix Uteri, Lactobacillus, RNA, Ribosomal, 16S, Middle Aged, Uterine Cervical Neoplasms, Human papillomavirus 16, Case-Control Studies, Human papillomavirus 18, Uterine Cervical Dysplasia_
High-risk Human Papillomavirus (HR-HPV) genotypes, specifically HPV16 and HPV18, pose a significant risk for the development of cervical intraepithelial neoplasia and cervical cancer. In the multifaceted cervical microenvironment, consisting of immune cells and diverse microbiota, Lactobacillus emerges as a pivotal factor, wielding significant influence in both stabilizing and disrupting the microbiome of the reproductive tract. To analyze the distinction between the cervical microbiota and Lactobacillus-dominant/non-dominant status of HR-HPV and non-infected healthy women, sixty-nine cervical swab samples were analyzed, included 44 with HR-HPV infection and healthy controls. All samples were recruited from Human Papillomavirus-based cervical cancer screening program and subjected to 16s rRNA sequencing analysis. Alpha and beta diversity analyses reveal no significant differences in the cervical microbiota of HR-HPV-infected women, including 16 and 18 HPV genotypes, and those with squamous intraepithelial lesion (SIL), compared to a control group. In this study we identified significantly lower abundance of Lactobacillus mucosae in women with HR-HPV infection compared to the control group. Furthermore, changes in bacterial diversity were noted in Lactobacillus non-dominant (LND) samples compared to Lactobacillus-dominant (LD) in both HR-HPV-infected and control groups. LND samples in HR-HPV-infected women exhibited a cervical dysbiotic state, characterized by Lactobacillus deficiency. In turn, the LD HR-HPV group showed an overrepresentation of Lactobacillus helveticus. In summary, our study highlighted the distinctive roles of L. mucosae and L. helveticus in HR-HPV infections, signaling a need for further research to demonstrate potential clinical implications of cervical microbiota dysbiosis.

[Source](https://doi.org/10.1371/journal.pone.0302270)

10.1371/journal.pone.0302270

---

## Association of the trimethylamine N-oxide with cardiovascular risk and vascular alterations in middle-aged patients with risk factors for cardiovascular diseases.
 26.04.2024, _N Spasova, D Somleva, B Krastev, R Ilieva, A Borizanova, D Svinarov, E Kinova, A Goudev_


Trimethylamine N-oxide (TMAO) is synthesized by the intestinal microbiota and is an independent predictor of cardiovascular disease (CVD). However, its underlying mechanisms remain unclear. We investigated TMAO levels across different CVD-risk patient groups, and evaluated associations between TMAO and vascular alterations (e.g., arterial stiffness, intima-media thickness (IMT), and the presence and grade of carotid artery plaques (CAPs)). Methods: We examined 95 patients (58.5 ± 7.3 years): 40 with clinical atherosclerotic cardiovascular disease (ASCVD), 40 with atherosclerosis risk factors (RF), and 15 controls. Arterial stiffness was measured by Carotid-Femoral Pulse Wave Velocity (C-F PWV). B-mode ultrasound was used to evaluate the presence and grade of CAPs and carotid IMT (CIMT). TMAO was measured by high performance liquid chromatography-tandem mass spectrometry (LC-MS/MS) and results were presented as the median (interquartile range). Results: TMAO levels were higher in patients with ASCVD (251.5 (164.5) µg/l) when compared to patients with RFs (194.0 (174) µg/l, p = 0.04) and controls (122.0 (77) µg/l, p &lt; 0.001). A significant correlation was observed between TMAO and PWV (r = 0.31, p = 0.003), which was not confirmed after adjustment for RFs. TMAO levels were significantly correlated with plaque score (r = 0.46, p &lt; 0.001) and plaque height (r = 0.41, p = 0.003), and were independent predictors for grade III plaques (odds ratio (OR) = 1.002, confidence interval (CI) 95%: 1.000047-1.003, p = 0.044). Conclusions: TMAO levels are increased with expanded CVD risk. Across different types of vascular damage, TMAO is associated with atherosclerotic changes.

[Source](https://doi.org/10.1042/BSR20232090)

10.1042/BSR20232090

---

## Insights into microorganisms, associated factors, and the oral microbiome in infective endocarditis patients.
 27.04.2024, _A Ismail, A Yogarajah, JL Falconer, R Dworakowski, S Watson, J Breeze, M Gunning, H Khan, A Hussain, JP Howard, P Cheong, M Shah, L Nibali, V Sousa_


Infective Endocarditis (IE) is a rare, life-threatening infection of the endocardium with multisystem effects. Culprit microorganisms derived from different niches circulate through the bloodstream and attach to the endocardium, particularly the heart valves. This study aimed to investigate culprit microorganisms among a cross-sectional cohort of IE patients, their associated factors, and to explore the potential relationship to the oral microbiome.

[Source](https://doi.org/10.3389/froh.2024.1270492)

10.3389/froh.2024.1270492

---

## Alterations in the fecal microbiota of methamphetamine users with bad sleep quality during abstinence.
 28.04.2024, _Z Deng, L Liu, W Liu, R Liu, T Ma, Y Xin, Y Xie, Y Zhang, Y Zhou, Y Tang_


_Humans, Gastrointestinal Microbiome, Methamphetamine, Male, Adult, Feces, Amphetamine-Related Disorders, Female, Sleep Quality, RNA, Ribosomal, 16S, Young Adult, Sleep Wake Disorders_
Methamphetamine (MA) abuse has resulted in a plethora of social issues. Sleep disturbance is a prominent issue about MA addiction, which serve as a risk factor for relapse, and the gut microbiota could play an important role in the pathophysiological mechanisms of sleep disturbances. Therefore, improving sleep quality can be beneficial for treating methamphetamine addiction, and interventions addressing the gut microbiota may represent a promising approach.

[Source](https://doi.org/10.1186/s12888-024-05773-5)

10.1186/s12888-024-05773-5

---

## Gut microbiome predicts cognitive function and depressive symptoms in late life.
 25.04.2024, _A Kolobaric, C Andreescu, E Jašarević, CH Hong, HW Roh, JY Cheong, YK Kim, TS Shin, CS Kang, CO Kwon, SY Yoon, SW Hong, HJ Aizenstein, HT Karim, SJ Son_


Depression in older adults with cognitive impairment increases progression to dementia. Microbiota is associated with current mood and cognition, but the extent to which it predicts future symptoms is unknown. In this work, we identified microbial features that reflect current and predict future cognitive and depressive symptoms. Clinical assessments and stool samples were collected from 268 participants with varying cognitive and depressive symptoms. Seventy participants underwent 2-year follow-up. Microbial community diversity, structure, and composition were assessed using high-resolution 16 S rRNA marker gene sequencing. We implemented linear regression to characterize the relationship between microbiome composition, current cognitive impairment, and depressive symptoms. We leveraged elastic net regression to discover features that reflect current or future cognitive function and depressive symptoms. Greater microbial community diversity associated with lower current cognition in the whole sample, and greater depression in participants not on antidepressants. Poor current cognitive function associated with lower relative abundance of Bifidobacterium, while greater GABA degradation associated with greater current depression severity. Future cognitive decline associated with lower cognitive function, lower relative abundance of Intestinibacter, lower glutamate degradation, and higher baseline histamine synthesis. Future increase in depressive symptoms associated with higher baseline depression and anxiety, lower cognitive function, diabetes, lower relative abundance of Bacteroidota, and lower glutamate degradation. Our results suggest cognitive dysfunction and depression are unique states with an overall biological effect detectable through gut microbiota. The microbiome may present a noninvasive readout and prognostic tool for cognitive and psychiatric states.

[Source](https://doi.org/10.1038/s41380-024-02551-3)

10.1038/s41380-024-02551-3

---

## Microbial signatures in human periodontal disease: a metatranscriptome meta-analysis.
 26.04.2024, _A Ovsepian, FS Kardaras, A Skoulakis, AG Hatzigeorgiou_


The characterization of oral microbial communities and their functional potential has been shaped by metagenomics and metatranscriptomics studies. Here, a meta-analysis of four geographically and technically diverse oral shotgun metatranscriptomics studies of human periodontitis was performed. In total, 54 subgingival plaque samples, 27 healthy and 27 periodontitis, were analyzed. The core microbiota of the healthy and periodontitis group encompassed 40 and 80 species, respectively, with 38 species being common to both microbiota. The differential abundance analysis identified 23 genera and 26 species, that were more abundant in periodontitis. Our results not only validated previously reported genera and species associated with periodontitis with heightened statistical significance, but also elucidated additional genera and species that were overlooked in the individual studies. Functional analysis revealed a significant up-regulation in the transcription of 50 gene families (UniRef-90) associated with transmembrane transport and secretion, amino acid metabolism, surface protein and flagella synthesis, energy metabolism, and DNA supercoiling in periodontitis samples. Notably, the overwhelming majority of the identified gene families did not exhibit differential abundance when examined across individual datasets. Additionally, 4 bacterial virulence factor genes, including TonB dependent receptor from P. gingivalis, surface antigen BspA from T. forsynthia, and adhesin A (PsaA) and Type I glyceraldehyde-3-phosphate dehydrogenase (GAPDH) from the Streptococcus genus, were also found to be significantly more transcribed in periodontitis group. Microbial co-occurrence analysis demonstrated that the periodontitis microbial network was less dense compared to the healthy network, but it contained more positive correlations between the species. Furthermore, there were discernible disparities in the patterns of interconnections between the species in the two networks, denoting the rewiring of the whole microbial network during the transition to the disease state. In summary, our meta-analysis has provided robust insights into the oral active microbiome and transcriptome in both health and disease.

[Source](https://doi.org/10.3389/fmicb.2024.1383404)

10.3389/fmicb.2024.1383404

---

## Gut microbiome composition reveals the distinctiveness between the Bengali people and the Indigenous ethnicities in Bangladesh.
 28.04.2024, _I Ahammad, A Bhattacharjee, ZM Chowdhury, A Rahman, MU Hossain, G Dewan, S Talukder, KC Das, CA Keya, M Salimullah_


_Bangladesh, Humans, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Feces, Ethnicity, Bacteria, Female, Adult, Male, Indigenous Peoples, South Asian People_
Ethnicity has a significant role in shaping the composition of the gut microbiome, which has implications in human physiology. This study intends to investigate the gut microbiome of Bengali people as well as several indigenous ethnicities (Chakma, Marma, Khyang, and Tripura) residing in the Chittagong Hill Tracts areas of Bangladesh. Following fecal sample collection from each population, part of the bacterial 16 s rRNA gene was amplified and sequenced using Illumina NovaSeq platform. Our findings indicated that Bangladeshi gut microbiota have a distinct diversity profile when compared to other countries. We also found out that Bangladeshi indigenous communities had a higher Firmicutes to Bacteroidetes ratio than the Bengali population. The investigation revealed an unclassified bacterium that was differentially abundant in Bengali samples while the genus Alistipes was found to be prevalent in Chakma samples. Further research on these bacteria might help understand diseases associated with these populations. Also, the current small sample-sized pilot study hindered the comprehensive understanding of the gut microbial diversity of the Bangladeshi population and its potential health implications. However, our study will help establish a basic understanding of the gut microbiome of the Bangladeshi population.

[Source](https://doi.org/10.1038/s42003-024-06191-9)

10.1038/s42003-024-06191-9

---

## Comparative analysis of endometrial, vaginal, and gut microbiota in patients with and without adenomyosis.
 25.04.2024, _M Valdés-Bango, M Gracia, E Rubio, A Vergara, C Casals-Pascual, C Ros, M Rius, MÁ Martínez-Zamora, E Mension, L Quintas, F Carmona_


Alterations in microbiota composition have been implicated in a variety of human diseases. Patients with adenomyosis present immune dysregulation leading to a persistent chronic inflammatory response. In this context, the hypothesis that alterations in the microbiota may be involved in the pathogenesis of adenomyosis, by affecting the epigenetic, immunologic, and biochemical functions of the host, has recently been postulated. The aim of the present study was to compare the microbiota composition in the vagina, endometrium, and gut of individuals with and without adenomyosis.

[Source](https://doi.org/10.1111/aogs.14847)

10.1111/aogs.14847

---

## Exploring the Causal Effects of Gut Microbiota on Diabetic Nephropathy: A Two-Sample Mendelian Randomization Study.
 27.04.2024, _K Chen, X Wang, Z Shang, Q Li, W Yao, S Guo, Y Guan_


In recent years, an increasing number of studies have indicated a bidirectional relationship between gut microbiota and the kidneys (the gut-kidney axis). Currently, the potential causal relationship between gut microbiota and diabetic nephropathy remains unclear. This study explores the causal effects of gut microbiota on diabetic nephropathy through Mendelian randomization.

[Source](https://doi.org/10.1186/s13098-024-01327-7)

10.2174/0113862073311197240425073859

---

## Causal effect of gut microbiota and diabetic nephropathy: a Mendelian randomization study

[Source](https://doi.org/10.1186/s13098-024-01327-7)

---

## The characteristics of microbiome in the upper respiratory tract of COVID-19 patients.
 27.04.2024, _X Zhang, N Nurxat, J Aili, Y Yasen, Q Wang, Q Liu_


_Humans, COVID-19, Microbiota, Male, Candida albicans, Female, Sputum, Middle Aged, SARS-CoV-2, Bacteria, Respiratory System, Aged, RNA, Ribosomal, 16S, Adult, Coinfection_
Co-infection with other pathogens in coronavirus disease 2019 (COVID-19) patients exacerbates disease severity and impacts patient prognosis. Clarifying the exact pathogens co-infected with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is premise of the precise treatment for COVID-19 patients.

[Source](https://doi.org/10.1186/s12866-024-03281-w)

10.1186/s12866-024-03281-w

---

## Interplay of human gastrointestinal microbiota metabolites: Short-chain fatty acids and their correlation with Parkinson's disease.
 28.04.2024, _J Liu, Q Chen, R Su_


_Humans, Parkinson Disease, Gastrointestinal Microbiome, Fatty Acids, Volatile, Brain-Gut Axis, Oxidative Stress_
Short-chain fatty acids (SCFAs) are, the metabolic byproducts of intestinal microbiota that, are generated through anaerobic fermentation of undigested dietary fibers. SCFAs play a pivotal role in numerous physiological functions within the human body, including maintaining intestinal mucosal health, modulating immune functions, and regulating energy metabolism. In recent years, extensive research evidence has indicated that SCFAs are significantly involved in the onset and progression of Parkinson disease (PD). However, the precise mechanisms remain elusive. This review comprehensively summarizes the progress in understanding how SCFAs impact PD pathogenesis and the underlying mechanisms. Primarily, we delve into the synthesis, metabolism, and signal transduction of SCFAs within the human body. Subsequently, an analysis of SCFA levels in patients with PD is presented. Furthermore, we expound upon the mechanisms through which SCFAs induce inflammatory responses, oxidative stress, abnormal aggregation of alpha-synuclein, and the intricacies of the gut-brain axis. Finally, we provide a critical analysis and explore the potential therapeutic role of SCFAs as promising targets for treating PD.

[Source](https://doi.org/10.1097/MD.0000000000037960)

10.1097/MD.0000000000037960

---

## Natural products from the human microbiome: an emergent frontier in organic synthesis and drug discovery.
 26.04.2024, _S Sengupta, S Pabbaraja, G Mehta_


Often referred to as the "second genome", the human microbiome is at the epicenter of complex inter-habitat biochemical networks like the "gut-brain axis", which has emerged as a significant determinant of cognition, overall health and well-being, as well as resistance to antibiotics and susceptibility to diseases. As part of a broader understanding of the nexus between the human microbiome, diseases and microbial interactions, whether encoded secondary metabolites (natural products) play crucial signalling roles has been the subject of intense scrutiny in the recent past. A major focus of these activities involves harvesting the genomic potential of the human microbiome via bioinformatics guided genome mining and culturomics. Through these efforts, an impressive number of structurally intriguing antibiotics, with enhanced chemical diversity vis-à-vis conventional antibiotics have been isolated from human commensal bacteria, thereby generating considerable interest in their total synthesis and expanding their therapeutic space for drug discovery. These developments augur well for the discovery of new drugs and antibiotics, particularly in the context of challenges posed by mycobacterial resistance and emerging new diseases. The current landscape of various synthetic campaigns and drug discovery initiatives on antibacterial natural products from the human microbiome is captured in this review with an intent to stimulate further activities in this interdisciplinary arena among the new generation.

[Source](https://doi.org/10.1039/d4ob00236a)

10.1039/d4ob00236a

---

## <em>Oligella </em>spp.: A systematic review on an uncommon urinary pathogen

[Source](https://doi.org/10.1007/s10096-024-04797-9)

---

## The potential importance of the built-environment microbiome and its impact on human health.
 25.04.2024, _TCG Bosch, M Wigley, B Colomina, B Bohannan, F Meggers, KR Amato, MB Azad, MJ Blaser, K Brown, MG Dominguez-Bello, SD Ehrlich, E Elinav, BB Finlay, K Geddie, N Geva-Zatorsky, T Giles-Vernick, P Gros, K Guillemin, LP Haraoui, E Johnson, F Keck, J Lorimer, MJ McFall-Ngai, M Nichter, S Pettersson, H Poinar, T Rees, C Tropini, EA Undurraga, L Zhao, MK Melby_


_Humans, Built Environment, Microbiota, Animals_
There is increasing evidence that interactions between microbes and their hosts not only play a role in determining health and disease but also in emotions, thought, and behavior. Built environments greatly influence microbiome exposures because of their built-in highly specific microbiomes coproduced with myriad metaorganisms including humans, pets, plants, rodents, and insects. Seemingly static built structures host complex ecologies of microorganisms that are only starting to be mapped. These microbial ecologies of built environments are directly and interdependently affected by social, spatial, and technological norms. Advances in technology have made these organisms visible and forced the scientific community and architects to rethink gene-environment and microbe interactions respectively. Thus, built environment design must consider the microbiome, and research involving host-microbiome interaction must consider the built-environment. This paradigm shift becomes increasingly important as evidence grows that contemporary built environments are steadily reducing the microbial diversity essential for human health, well-being, and resilience while accelerating the symptoms of human chronic diseases including environmental allergies, and other more life-altering diseases. New models of design are required to balance maximizing exposure to microbial diversity while minimizing exposure to human-associated diseases. Sustained trans-disciplinary research across time (evolutionary, historical, and generational) and space (cultural and geographical) is needed to develop experimental design protocols that address multigenerational multispecies health and health equity in built environments.

[Source](https://doi.org/10.1073/pnas.2313971121)

10.1073/pnas.2313971121

---

## Vitamin D regulates microbiome-dependent cancer immunity.
 25.04.2024, _E Giampazolias, M Pereira da Costa, KC Lam, KHJ Lim, A Cardoso, C Piot, P Chakravarty, S Blasche, S Patel, A Biram, T Castro-Dopico, MD Buck, RR Rodrigues, GJ Poulsen, SA Palma-Duran, NC Rogers, MA Koufaki, CM Minutti, P Wang, A Vdovin, B Frederico, E Childs, S Lee, B Simpson, A Iseppon, S Omenetti, G Kelly, R Goldstone, E Nye, A Suárez-Bonnet, SL Priestnall, JI MacRae, S Zelenay, KR Patil, K Litchfield, JC Lee, T Jess, RS Goldszmid, C Reis E Sousa_


_Animals, Mice, Vitamin D, Gastrointestinal Microbiome, Humans, Bacteroides fragilis, Immune Checkpoint Inhibitors, Neoplasms, Mice, Inbred C57BL, Immunotherapy, Female, Intestinal Mucosa, Male_
A role for vitamin D in immune modulation and in cancer has been suggested. In this work, we report that mice with increased availability of vitamin D display greater immune-dependent resistance to transplantable cancers and augmented responses to checkpoint blockade immunotherapies. Similarly, in humans, vitamin D-induced genes correlate with improved responses to immune checkpoint inhibitor treatment as well as with immunity to cancer and increased overall survival. In mice, resistance is attributable to the activity of vitamin D on intestinal epithelial cells, which alters microbiome composition in favor of Bacteroides fragilis, which positively regulates cancer immunity. Our findings indicate a previously unappreciated connection between vitamin D, microbial commensal communities, and immune responses to cancer. Collectively, they highlight vitamin D levels as a potential determinant of cancer immunity and immunotherapy success.

[Source](https://doi.org/10.1126/science.adh7954)

10.1126/science.adh7954

---

## HMOs Impact the Gut Microbiome of Children and Adults Starting from Low Predicted Daily Doses.
 28.04.2024, _D Bajic, F Wiens, E Wintergerst, S Deyaert, A Baudot, PVD Abbeele_


Recent studies suggest that the dietary intake of human milk oligosaccharides (HMOs) provides health benefits from infancy up to adulthood. Thus far, beneficial changes in the adult gut microbiome have been observed at oral doses of 5-20 g/day of HMOs. Efficacy of lower doses has rarely been tested. We assessed four HMO molecular species-2'Fucosyllactose (2'FL), Lacto-N-neotetraose (LNnT), 3'Sialyllactose (3'SL), and 6'Sialyllactose (6'SL)-at predicted doses from 0.3 to 5 g/day for 6-year-old children and adults (n = 6 each), using ex vivo SIFR® technology (Cryptobiotix, Ghent, Belgium). This technology employing bioreactor fermentation on fecal samples enables us to investigate microbial fermentation products that are intractable in vivo given their rapid absorption/consumption in the human gut. We found that HMOs significantly increased short-chain fatty acids (SCFAs), acetate, propionate (in children/adults), and butyrate (in adults) from predicted doses of 0.3-0.5 g/day onwards, with stronger effects as dosing increased. The fermentation of 6'SL had the greatest effect on propionate, LNnT most strongly increased butyrate, and 2'FL and 3'SL most strongly increased acetate. An untargeted metabolomic analysis revealed that HMOs enhanced immune-related metabolites beyond SCFAs, such as aromatic lactic acids (indole-3-lactic acid/3-phenyllactic acid) and 2-hydroxyisocaproic acid, as well as gut-brain-axis-related metabolites (γ-aminobutyric acid/3-hydroxybutyric acid/acetylcholine) and vitamins. The effects of low doses of HMOs potentially originate from the highly specific stimulation of keystone species belonging to, for example, the Bifidobacteriaceae family, which had already significantly increased at doses of only 0.5 g/day LNnT (adults) and 1 g/day 2'FL (children/adults).

[Source](https://doi.org/10.3390/metabo14040239)

10.3390/metabo14040239

---

## Effects of microbiome-based interventions on neurodegenerative diseases: a systematic review and meta-analysis.
 28.04.2024, _ZSW Chui, LML Chan, EWH Zhang, S Liang, EPH Choi, KYW Lok, HM Tun, JYY Kwok_


_Humans, Neurodegenerative Diseases, Gastrointestinal Microbiome, Fecal Microbiota Transplantation, Probiotics, Microbiota_
Neurodegenerative diseases (NDDs) are characterized by neuronal damage and progressive loss of neuron function. Microbiome-based interventions, such as dietary interventions, biotics, and fecal microbiome transplant, have been proposed as a novel approach to managing symptoms and modulating disease progression. Emerging clinical trials have investigated the efficacy of interventions modulating the GM in alleviating or reversing disease progression, yet no comprehensive synthesis have been done. A systematic review of the literature was therefore conducted to investigate the efficacy of microbiome-modulating methods. The search yielded 4051 articles, with 15 clinical trials included. The overall risk of bias was moderate in most studies. Most microbiome-modulating interventions changed the GM composition. Despite inconsistent changes in GM composition, the meta-analysis showed that microbiome-modulating interventions improved disease burden (SMD, - 0.57; 95% CI - 0.93 to - 0.21; I2 = 42%; P = 0.002) with a qualitative trend of improvement in constipation. However, current studies have high methodological heterogeneity and small sample sizes, requiring more well-designed and controlled studies to elucidate the complex linkage between microbiome, microbiome-modulating interventions, and NDDs.

[Source](https://doi.org/10.1038/s41598-024-59250-w)

10.1038/s41598-024-59250-w

---

## A randomized, double-blinded, placebo-controlled clinical trial on <em>Lactobacillus</em>-containing cultured milk drink as adjuvant therapy for depression in irritable bowel syndrome

[Source](https://doi.org/10.1038/s41598-024-60029-2)

---

## Understanding the Diversity and Roles of the Ruminal Microbiome.
 25.04.2024, _GB Keum, S Pandey, ES Kim, H Doo, J Kwak, S Ryu, Y Choi, J Kang, S Kim, HB Kim_


The importance of ruminal microbiota in ruminants is emphasized, not only as a special symbiotic relationship with ruminants but also as an interactive and dynamic ecosystem established by the metabolites of various rumen microorganisms. Rumen microbial community is essential for life maintenance and production as they help decompose and utilize fiber that is difficult to digest, supplying about 70% of the energy needed by the host and 60-85% of the amino acids that reach the small intestine. Bacteria are the most abundant in the rumen, but protozoa, which are relatively large, account for 40-50% of the total microorganisms. However, the composition of these ruminal microbiota is not conserved or constant throughout life and is greatly influenced by the host. It is known that the initial colonization of calves immediately after birth is mainly influenced by the mother, and later changes depending on various factors such as diet, age, gender and breed. The initial rumen microbial community contains aerobic and facultative anaerobic bacteria due to the presence of oxygen, but as age increases, a hypoxic environment is created inside the rumen, and anaerobic bacteria become dominant in the rumen microbial community. As calves grow, taxonomic diversity increases, especially as they begin to consume solid food. Understanding the factors affecting the rumen microbial community and their effects and changes can lead to the early development and stabilization of the microbial community through the control of rumen microorganisms, and is expected to ultimately help improve host productivity and efficiency.

[Source](https://doi.org/10.1007/s12275-024-00121-4)

10.1007/s12275-024-00121-4

---

## Impacts of continuous cropping on the rhizospheric and endospheric microbial communities and root exudates of <em>Astragalus mongholicus</em>

[Source](https://doi.org/10.1186/s12870-024-05024-5)

---

## Comparative analysis of root associated microbes in tropical cultivated and weedy rice (<em>Oryza</em> spp.) and temperate cultivated rice

[Source](https://doi.org/10.1038/s41598-024-60384-0)

---

## Identifying causes and consequences of rhizosphere microbiome heritability.
 28.04.2024, _MR Wagner_


_Rhizosphere, Microbiota, Hordeum, Soil Microbiology, Genotype, Bacteria, Plant Roots_
Host genotype affects microbiome composition in many plants, but the mechanisms and implications of this phenomenon are understudied. New work in PLOS Biology illustrates how host genotype leads to differential gene expression and fitness in bacteria of the barley rhizosphere.

[Source](https://doi.org/10.1371/journal.pbio.3002604)

10.1371/journal.pbio.3002604

---

## Acidithiobacillia class members originating at sites within the Pacific Ring of Fire and other tectonically active locations and description of the novel genus ‘<em>Igneacidithiobacillus</em>

[Source](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2024.1360268/full)

---

## Exploring changes in microplastic-associated bacterial communities with time, location, and polymer type in Liusha Bay, China.
 24.04.2024, _T Wang, F Lu, C Yang, C Wang, Y Liao, R Mkuye, Y Deng_


Microplastics have become a widespread concern within marine environments and are particularly evident in aquaculture regions that are characterized by plastic accumulation. This study employed 16 S rDNA sequencing to investigate the dynamic succession of microbial communities colonizing polyvinyl chloride (PVC), polystyrene (PS), and polyamide (PA) microplastics in seawater, when subjected to varying exposure durations in the Liusha Bay aquaculture region. Results revealed that the composition of microplastics microbial communities varied remarkably across geographical locations and exposure times. With an increase in exposure duration, both the diversity and richness of bacterial communities colonizing microplastics significantly increased, microbial communities show adaptations to the plastisphere. The type of microplastics had a significant effect on the community structure characteristicsof bacteria attached to their surfaces, with inconsistent trends in the relative abundance of different genera on different substrates. Notably, microplastic surfaces harbored a significant abundance of hydrocarbon-degrading bacteria, exemplified by Erythrobacter. These findings underscore the potential of microplastics as unique microbial niches. Meanwhile, long-term exposure experiments also offer the possibility of screening for plastic-degrading bacteria. In addition, the presence of the pathogenic bacterium Vibrio was detected in all microplastic samples, implying that microplastics could serve as carriers for pathogenic dissemination. This underscores the urgency of addressing the risk posed by the proliferation of harmful bacteria on microplastic surfaces. Overall, this study enhances our understanding of microbial community dynamics on microplastics under diverse conditions. It contributes to the broader comprehension of plastisphere microbial ecosystems in the marine environment, thereby addressing critical environmental implications.

[Source](https://doi.org/10.1016/j.marenvres.2024.106525)

10.1016/j.marenvres.2024.106525

---

## Genomic and environmental controls on <em>Castellaniella </em>biogeography in an anthropogenically disturbed subsurface

[Source](https://doi.org/10.1186/s40793-024-00570-9)

---

## Phytoplankton-derived polysaccharides and microbial peptidoglycans are key nutrients for deep-sea microbes in the Mariana Trench

[Source](https://doi.org/10.1186/s40168-024-01789-x)

---

## Microbiome compositional data analysis for survival studies.
 27.04.2024, _M Pujolassos, A Susín, ML Calle_


The growing interest in studying the relationship between the human microbiome and our health has also extended to time-to-event studies where researchers explore the connection between the microbiome and the occurrence of a specific event of interest. The analysis of microbiome obtained through high throughput sequencing techniques requires the use of specialized Compositional Data Analysis (CoDA) methods designed to accommodate its compositional nature. There is a limited availability of statistical tools for microbiome analysis that incorporate CoDA, and this is even more pronounced in the context of survival analysis. To fill this methodological gap, we present coda4microbiome for survival studies, a new methodology for the identification of microbial signatures in time-to-event studies. The algorithm implements an elastic-net penalized Cox regression model adapted to compositional covariates. We illustrate coda4microbiome algorithm for survival studies with a case study about the time to develop type 1 diabetes for non-obese diabetic mice. Our algorithm identified a bacterial signature composed of 21 genera associated with diabetes development. coda4microbiome for survival studies is integrated in the R package coda4microbiome as an extension of the existing functions for cross-sectional and longitudinal studies.

[Source](https://doi.org/10.1093/nargab/lqae038)

10.1093/nargab/lqae038

---

## Periportal macrophages protect against commensal-driven liver inflammation.
 27.04.2024, _Y Miyamoto, J Kikuta, T Matsui, T Hasegawa, K Fujii, D Okuzaki, YC Liu, T Yoshioka, S Seno, D Motooka, Y Uchida, E Yamashita, S Kobayashi, H Eguchi, E Morii, K Tryggvason, T Shichita, H Kayama, K Atarashi, J Kunisawa, K Honda, K Takeda, M Ishii_


The liver is the main gateway from the gut, and the unidirectional sinusoidal flow from portal to central veins constitutes heterogenous zones, including the periportal vein (PV) and the pericentral vein zones1-5. However, functional differences in the immune system in each zone remain poorly understood. Here intravital imaging revealed that inflammatory responses are suppressed in PV zones. Zone-specific single-cell transcriptomics detected a subset of immunosuppressive macrophages enriched in PV zones that express high levels of interleukin-10 and Marco, a scavenger receptor that sequesters pro-inflammatory pathogen-associated molecular patterns and damage-associated molecular patterns, and consequently suppress immune responses. Induction of Marco+ immunosuppressive macrophages depended on gut microbiota. In particular, a specific bacterial family, Odoribacteraceae, was identified to induce this macrophage subset through its postbiotic isoallolithocholic acid. Intestinal barrier leakage resulted in inflammation in PV zones, which was markedly augmented in Marco-deficient conditions. Chronic liver inflammatory diseases such as primary sclerosing cholangitis (PSC) and non-alcoholic steatohepatitis (NASH) showed decreased numbers of Marco+ macrophages. Functional ablation of Marco+ macrophages led to PSC-like inflammatory phenotypes related to colitis and exacerbated steatosis in NASH in animal experimental models. Collectively, commensal bacteria induce Marco+ immunosuppressive macrophages, which consequently limit excessive inflammation at the gateway of the liver. Failure of this self-limiting system promotes hepatic inflammatory disorders such as PSC and NASH.

[Source](https://doi.org/10.1038/s41586-024-07372-6)

10.1038/s41586-024-07372-6

---

## A case study to engage students in the research design and ethics of high-throughput metagenomics.
 27.04.2024, _CC Alexander, MM Gaudier-Diaz, AJ Kleinschmit, PJ Dihle, SA Salger, N Vega, SD Robertson_


Case studies present students with an opportunity to learn and apply course content through problem solving and critical thinking. Supported by the High-throughput Discovery Science &amp; Inquiry-based Case Studies for Today's Students (HITS) Research Coordination Network, our interdisciplinary team designed, implemented, and assessed two case study modules entitled "You Are What You Eat." Collectively, the case study modules present students with an opportunity to engage in experimental research design and the ethical considerations regarding microbiome research and society. In this manuscript, we provide instructors with tools for adopting or adapting the research design and/or the ethics modules. To date, the case has been implemented using two modalities (remote and in-person) in three courses (Microbiology, Physiology, and Neuroscience), engaging over 200 undergraduate students. Our assessment data demonstrate gains in content knowledge and students' perception of learning following case study implementation. Furthermore, when reflecting on our experiences and student feedback, we identified ways in which the case study could be modified for different settings. In this way, we hope that the "You Are What You Eat" case study modules can be implemented widely by instructors to promote problem solving and critical thinking in the traditional classroom or laboratory setting when discussing next-generation sequencing and/or metagenomics research.

[Source](https://doi.org/10.1128/jmbe.00074-23)

10.1128/jmbe.00074-23

---

