## Multi-omic factors associated with future wheezing in infants.
 27.09.2022, _R Beheshti, ES Halstead, D McKeone, SD Hicks_


The pathophysiology of wheezing is multifactorial, impacted by medical, demographic, environmental, and immunologic factors. We hypothesized that multi-omic analyses of host and microbial factors in saliva would enhance the ability to identify infants at risk for wheezing.

[Source](https://www.nature.com/articles/s41390-022-02318-y)

10.1038/s41390-022-02318-y

---

## Race is a key determinant of the human intratumor microbiome.
 03.10.2022, _M Luo, Y Liu, LC Hermida, EM Gertz, Z Zhang, Q Li, L Diao, E Ruppin, L Han_


_Ethnicity, Humans, Microbiota, Neoplasms, Racial Groups_

[Source](https://www.sciencedirect.com/science/article/pii/S1535610822003713)

10.1016/j.ccell.2022.08.007

---

## Alterations in the respiratory tract microbiome in COVID-19: current observations and potential significance

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01342-8)

---

## Maternal weight status and the composition of the human milk microbiome: A scoping review.
 04.10.2022, _K Daiy, V Harries, K Nyhan, UM Marcinkowska_


The human milk microbiome is thought to partly contribute to the assembly of the infant gut microbiome, a microbial community with important implications for infant health and development. While obesity has well-established links with the adult gut microbiome, less is known about how it affects the human milk microbiome. In this scoping review, we synthesize the current literature on the microbial composition of human milk by maternal weight status, defined broadly as BMI (prepregnancy and postpartum) and gestational weight gain (GWG). This study followed the a priori protocol published in Prospero (registration #: CRD42020165633). We searched the following databases for studies reporting maternal weight status and a characterization of milk microbiota through culture-dependent and culture-independent methods: MEDLINE, Embase, Web of Science, CINAHL, and Scopus. After screening 6,365 studies, we found 20 longitudinal and cross-sectional studies investigating associations between maternal weight status and the composition of the milk microbiome. While some studies reported no associations, many others reported that women with a pre-pregnancy or postpartum BMI characterized as overweight or obese, or with excessive GWG, had higher abundances of the genus Staphylococcus, lower Bifidobacterium abundance, and lower alpha diversity (within-sample diversity). This review suggests that maternal weight status is minorly associated with the composition of the milk microbiome in various ways. We offer potential explanations for these findings, as well as suggestions for future research.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274950)

10.1371/journal.pone.0274950

---

## Functional changes in the oral microbiome after use of fluoride and arginine containing dentifrices: a metagenomic and metatranscriptomic study.
 01.10.2022, _M Carda-Diéguez, R Moazzez, A Mira_


_Adult, Ammonia, Arginine, Bacteria, Cariostatic Agents, Dental Caries, Dental Plaque, Dentifrices, Double-Blind Method, Fluorides, Humans, Metagenome, Microbiota, Periodontitis, Phosphates, Tooth Remineralization, Toothpastes_
Tooth decay is one of the most prevalent diseases worldwide, and efficient tooth brushing with a fluoride-containing dentifrice is considered fundamental to caries prevention. Fluoride-containing dentifrices have been extensively studied in relation to enamel resistance to demineralization. Arginine (Arg) has also been proposed as a promising prebiotic to promote pH buffering through ammonia production. Here, we present the first metagenomic (DNA sequencing of the whole microbial community) and metatranscriptomic (RNAseq of the same community) analyses of human dental plaque to evaluate the effect of brushing with fluoride (Fl) and a Fl+Arg containing dentifrices on oral microbial composition and activity. Fifty-three patients were enrolled in a longitudinal clinical intervention study with two arms, including 26 caries-active and 27 caries-free adults. After a minimum 1-week washout period, dental plaque samples were collected at this post-washout baseline, 3 months after the use of a 1450-ppm fluoride dentifrice, and after 6 months of using a 1450-ppm fluoride with 1.5% arginine dentifrice.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01338-4)

10.1186/s40168-022-01338-4

---

## Remembering St. Louis individual—structural violence and acute bacterial infections in a historical anatomical collection

[Source](https://www.nature.com/articles/s42003-022-03890-z)

---

## SARS-CoV-2-associated gut microbiome alteration; A new contributor to colorectal cancer pathogenesis.
 03.10.2022, _SA Mozaffari, A Salehi, E Mousavi, BA Zaman, AE Nassaj, F Ebrahimzadeh, H Nasiri, Z Valedkarimi, A Adili, G Asemani, M Akbari_


The emergence of a novel coronavirus, COVID-19, in December 2019 led to a global pandemic with more than 170 million confirmed infections and more than 6 million deaths (by July 2022). Studies have shown that infection with SARS-CoV-2 in cancer patients has a higher mortality rate than in people without cancer. Here, we have reviewed the evidence showing that gut microbiota plays an important role in health and is linked to colorectal cancer development. Studies have shown that SARS-CoV-2 infection leads to a change in gut microbiota, which modify intestinal inflammation and barrier permeability and affects tumor-suppressor or oncogene genes, proposing SARS-CoV-2 as a potential contributor to CRC pathogenesis.

[Source](https://www.sciencedirect.com/science/article/pii/S0344033822003752)

10.1016/j.prp.2022.154131

---

## Population structure discovery in meta-analyzed microbial communities and inflammatory bowel disease using MMUPHin.
 03.10.2022, _S Ma, D Shungin, H Mallick, M Schirmer, LH Nguyen, R Kolde, E Franzosa, H Vlamakis, R Xavier, C Huttenhower_


Microbiome studies of inflammatory bowel diseases (IBD) have achieved a scale for meta-analysis of dysbioses among populations. To enable microbial community meta-analyses generally, we develop MMUPHin for normalization, statistical meta-analysis, and population structure discovery using microbial taxonomic and functional profiles. Applying it to ten IBD cohorts, we identify consistent associations, including novel taxa such as Acinetobacter and Turicibacter, and additional exposure and interaction effects. A single gradient of dysbiosis severity is favored over discrete types to summarize IBD microbiome population structure. These results provide a benchmark for characterization of IBD and a framework for meta-analysis of any microbial communities.

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02753-4)

10.1186/s13059-022-02753-4

---

## Microbiota imbalance induced by dietary sugar disrupts immune-mediated protection from metabolic syndrome.
 01.10.2022, _Y Kawano, M Edwards, Y Huang, AM Bilate, LP Araujo, T Tanoue, K Atarashi, MS Ladinsky, SL Reiner, HH Wang, D Mucida, K Honda, II Ivanov_


_Animals, Diet, High-Fat, Dietary Sugars, Interleukin-17, Intestinal Mucosa, Lipids, Metabolic Syndrome, Mice, Mice, Inbred C57BL, Microbiota, Obesity, Th17 Cells_
How intestinal microbes regulate metabolic syndrome is incompletely understood. We show that intestinal microbiota protects against development of obesity, metabolic syndrome, and pre-diabetic phenotypes by inducing commensal-specific Th17 cells. High-fat, high-sugar diet promoted metabolic disease by depleting Th17-inducing microbes, and recovery of commensal Th17 cells restored protection. Microbiota-induced Th17 cells afforded protection by regulating lipid absorption across intestinal epithelium in an IL-17-dependent manner. Diet-induced loss of protective Th17 cells was mediated by the presence of sugar. Eliminating sugar from high-fat diets protected mice from obesity and metabolic syndrome in a manner dependent on commensal-specific Th17 cells. Sugar and ILC3 promoted outgrowth of Faecalibaculum rodentium that displaced Th17-inducing microbiota. These results define dietary and microbiota factors posing risk for metabolic syndrome. They also define a microbiota-dependent mechanism for immuno-pathogenicity of dietary sugar and highlight an elaborate interaction between diet, microbiota, and intestinal immunity in regulation of metabolic disorders.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(22)00992-8)

10.1016/j.cell.2022.08.005

---

## Rural environment reduces allergic inflammation by modulating the gut microbiota.
 04.10.2022, _Z Yang, Z Chen, X Lin, S Yao, M Xian, X Ning, W Fu, M Jiang, N Li, X Xiao, M Feng, Z Lian, W Yang, X Ren, Z Zheng, J Zhao, N Wei, W Lu, M Roponen, B Schaub, GWK Wong, Z Su, C Wang, J Li_


Rural environments and microbiota are linked to a reduction in the prevalence of allergies. However, the mechanism underlying the reduced allergies modulated by rural residency is unclear. Here, we assessed gut bacterial composition and metagenomics in urban and rural children in the EuroPrevall-INCO cohort. Airborne dusts, including mattress and rural henhouse dusts, were profiled for bacterial and fungal composition by amplicon sequencing. Mice were repeatedly exposed to intranasal dust extracts and evaluated for their effects on ovalbumin (OVA)-induced allergic airway inflammation, and gut microbiota restoration was validated by fecal microbiota transplant (FMT) from dust-exposed donor mice. We found that rural children had fewer allergies and unique gut microbiota with fewer Bacteroides and more Prevotella. Indoor dusts in rural environments harbored higher endotoxin level and diversity of bacteria and fungi, whereas indoor urban dusts were enriched with Aspergillus and contained elevated pathogenic bacteria. Intranasal administration of rural dusts before OVA sensitization reduced respiratory eosinophils and blood IgE level in mice and also led to a recovery of gut bacterial diversity and Ruminiclostridium in the mouse model. FMT restored the protective effect by reducing OVA-induced lung eosinophils in recipient mice. Together, these results support a cause-effect relationship between exposure to dust microbiota and allergy susceptibility in children and mice. Specifically, rural environmental exposure modulated the gut microbiota, which was essential in reducing allergy in children from Southern China. Our findings support the notion that the modulation of gut microbiota by exposure to rural indoor dust may improve allergy prevention.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2022.2125733)

10.1080/19490976.2022.2125733

---

## Six types of tea extracts attenuated high-fat diet-induced metabolic syndrome via modulating gut microbiota in rats.
 04.10.2022, _F Zhou, MZ Zhu, JY Tang, J Ou-Yang, BH Shang, CW Liu, J Wang, Q Liu, JA Huang, ZH Liu_


An important puzzle for tea consumers is which type of tea is effective in treating metabolic syndrome (MS). In this study, the effects of six types of tea extracts (TEs) on high-fat diet (HFD)-induced MS, as well as chemical components of six TEs, were investigated and compared. Each TE consisted of representative tea originated from different places in China to avoid one-sidedness of sampling. All six TEs were found to attenuate MS and ameliorate intestinal barrier function in HFD-fed rats. Further, white tea performed better in body weight control, while dark tea had more advantages in protecting intestinal barrier. Moreover, all six TEs alleviated the gut microbiota dysbiosis, which was manifested by decreased Firmicutes/Bacteroidetes ratio and enriched beneficial bacteria, such as Akkermansia, Bacteroides, and Bifidobacterium. Together, all six TEs attenuate HFD-induced MS although their efficiency varies, and this therapeutic effect is related to the modulation of gut microbiota.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0963996922008468)

10.1016/j.foodres.2022.111788

---

## Adaptation of the gut holobiont to malnutrition during mouse pregnancy depends on the type of nutritional adversity.
 04.10.2022, _KL Connor, E Bloise, TZ DeSantis, SJ Lye_


Malnutrition can influence maternal physiology and programme offspring development. Yet, in pregnancy, little is known about how dietary challenges that influence maternal phenotype affect gut structure and function. Emerging evidence suggests that interactions between the environment, multidrug resistance (MDR) transporters and microbes may influence maternal adaptation to pregnancy and regulate fetoplacental development. We hypothesised that the gut holobiont (host and microbes) during pregnancy adapts differently to suboptimal maternal diets, evidenced by changes in the gut microenvironment, morphology, and expression of key protective MDR transporters during pregnancy. Mice were fed a control diet (CON) during pregnancy, or undernourished (UN) by 30% of control intake from gestational day (GD)5.5-18.5, or fed 60% high fat diet (HF) for eight weeks before and during pregnancy. At GD18.5, maternal small intestinal (SI) architecture (H&amp;E), proliferation (Ki67), P-glycoprotein (P-gp - encoded by Abcb1a/b) and breast cancer resistance protein (BCRP/Abcg2) MDR transporter expression and levels of pro-inflammatory biomarkers were assessed. Circulating inflammatory biomarkers and maternal caecal microbiome composition (G3 PhyloChipTM) were measured. MDR transporter expression was also assessed in fetal gut. HF diet increased maternal SI crypt depth and proinflammatory load, and decreased SI expression of Abcb1a mRNA, whilst UN increased SI villi proliferation and Abcb1a, but decreased Abcg2, mRNA expression. There were significant associations between Abcb1a and Abcg2 mRNA levels with relative abundance of specific microbial taxa. Using a systems physiology approach we report that common nutritional adversities provoke adaptations in the pregnancy holobiont in mice, and reveal new mechanisms that could influence reproductive outcomes and fetal development.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0955286322002406)

10.1016/j.jnutbio.2022.109172

---

## A global phylogenomic and metabolic reconstruction of the large intestine bacterial community of domesticated cattle.
 30.09.2022, _S Teseo, S Otani, C Brinch, S Leroy, P Ruiz, M Desvaux, E Forano, FM Aarestrup, P Sapountzis_


_Amino Acids, Animals, Bacteria, Carbohydrates, Cattle, Intestine, Large, Mammals, Metagenomics, Phylogeny_
The large intestine is a colonization site of beneficial microbes complementing the nutrition of cattle but also of zoonotic and animal pathogens. Here, we present the first global gene catalog of cattle fecal microbiomes, a proxy of the large intestine microbiomes, from 436 metagenomes from six countries.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01357-1)

10.1186/s40168-022-01357-1

---

## Comparative multiomics study of the effects of Ellagic acid on the gut environment in young and adult mice

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0963996922008778)

---

## Crosstalk between the microbiota and insect postembryonic development.
 27.09.2022, _M Girard, P Luis, C Valiente Moro, G Minard_


Insect sequential development evolves from a simple molt towards complete metamorphosis. Like any multicellular host, insects interact with a complex microbiota. In this review, factors driving the microbiota dynamics were pointed out along their development. Special focus was put on tissue renewal, shift in insect ecology, and microbial interactions. Conversely, how the microbiota modulates its host development through nutrient acquisition, hormonal control, and cellular or tissue differentiation was exemplified. Such modifications might have long-term carry-over effects on insect physiology. Finally, remarkable microbe-driven control of insect behaviors along their life cycle was highlighted. Increasing knowledge of those interactions might offer new insights on how insects respond to their environment as well as perspectives on pest- or vector-control strategies.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0966842X22002463)

10.1016/j.tim.2022.08.013

---

## Culturomic, metagenomic, and transcriptomic-based characterization of commensal lactic acid bacteria isolated from domestic dogs using Caenorhabditis elegans as a model for aging.
 04.10.2022, _AN Kang, D Mun, S Ryu, JJ Lee, S Oh, MK Kim, M Song, S Oh, Y Kim_


In tandem with the fast expansion of the pet-economy industry, the present ageing research has been noticing the function of probiotics in extending the healthy lifetime of domestic animals. In this study, we aimed to understand the bacterial compositions of canine feces and isolating lactic acid bacteria as commensal lactic acid bacteria (LAB) as novel potential probiotics for the use of anti-aging using Caenorhabditis elegans surrogate animal model. Under an anaerobic culturomic and metagenomic analysis, a total of 305 commensal LAB were isolated from diverse domestic dogs, and four strains, Lactobacillus amylolyticus, L. salivarius, Enterococcus hirae and E. faecium, made prominence as commensal LAB by enhancing C. elegans life span and restored neuronal degeneration induced by aging by up-regulating skn-1, ser-7 and odr-3, 7, 10. Importantly, whole transcriptome results and integrative network analysis revealed extensive mRNA encoding protein domains and functional pathways of naturally aging C. elegans was examined and we built the gene informatics basis. Taken together, our finding proposed that a specific gene network corresponding to the pathways differentially expressed during the aging and selected commensal LAB as potential probiotic strains could be provided beneficial effects in the aging of domestic animals by modulating dynamics of gut microbiota.

[Source](https://academic.oup.com/jas/advance-article-abstract/doi/10.1093/jas/skac323/6748171)

10.1093/jas/skac323

---

## Recruitment of specific microbes through exudates affects cadmium activation and accumulation in Brassica napu

[Source](https://www.sciencedirect.com/science/article/abs/pii/S030438942201860X)

---

## Novel trends of genome evolution in highly complex tropical sponge microbiomes

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01359-z)

---

## Marine DNA methylation patterns are associated with microbial community composition and inform virus-host dynamics.
 30.09.2022, _HJ Seong, S Roux, CY Hwang, WJ Sul_


_Bacteriophages, DNA, DNA Methylation, Metagenome, Metagenomics, Methyltransferases, Microbiota_
DNA methylation in prokaryotes is involved in many different cellular processes including cell cycle regulation and defense against viruses. To date, most prokaryotic methylation systems have been studied in culturable microorganisms, resulting in a limited understanding of DNA methylation from a microbial ecology perspective. Here, we analyze the distribution patterns of several microbial epigenetics marks in the ocean microbiome through genome-centric metagenomics across all domains of life.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01340-w)

10.1186/s40168-022-01340-w

---

## Untangling microbial diversity and assembly patterns in rare earth element mine drainage in South China.
 03.10.2022, _Z Chen, YH Fei, WS Liu, K Ding, J Lu, X Cai, T Cui, YT Tang, S Wang, Y Chao, R Qiu_


Ion-adsorption rare earth element (REE) deposits are the main reservoirs of REEs worldwide, and are widely exploited in South China. Microbial diversity is essential for maintaining the performance and function of mining ecosystems. Investigating the ecological patterns underlying the REE mine microbiome is essential to understand ecosystem responses to environmental changes and to improve the bioremediation of mining areas. We applied 16S rRNA and ITS gene sequence analyses to investigate the composition characteristics of prokaryotic (bacteria, archaea) and fungal communities in a river impacted by REE acid mine drainage (REE-AMD). The river formed a unique micro-ecosystem, including the main prokaryotic taxa of Proteobacteria, Acidobacteria, Crenarchaeota, and Euryarchaeota, as well as the main fungal taxa of Ascomycota, Basidiomycota, and Chytridiomycota. Analysis of microbial diversity showed that, unlike prokaryotic communities that responded drastically to pollution disturbances, fungal communities were less affected by REE-AMD, but fluctuated significantly in different seasons. Ecological network analysis revealed that fungal communities have lower connectivity and centrality, and higher modularity than prokaryotic networks, indicating that fungal communities have more stable network structures. The introduction of REE-AMD mainly reduced the complexity of the community network and the number of keystone species, while the proportion of negative prokaryotic-fungal associations in the network increased. Ecological process analysis revealed that, compared to the importance of environmental selection for prokaryotes, stochastic processes might have contributed primarily to fungal communities in REE mining areas. These findings confirm that the different assembly mechanisms of prokaryotic and fungal communities are key to the differences in their responses to environmental perturbations. The findings also provide the first insights into microbiota assembly patterns in REE-AMD and important ecological knowledge for the formation and development of microbial communities in REE mining areas.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0043135422011174)

10.1016/j.watres.2022.119172

---

## Benchmarking Community-Wide Estimates of Growth Potential from Metagenomes Using Codon Usage Statistics.
 03.10.2022, _JL Weissman, M Peras, TP Barnum, JA Fuhrman_


Trait inference from mixed-species assemblages is a central problem in microbial ecology. Frequently, sequencing information from an environment is available, but phenotypic measurements from individual community members are not. With the increasing availability of molecular data for microbial communities, bioinformatic approaches that map metagenome to (meta)phenotype are needed. Recently, we developed a tool, gRodon, that enables the prediction of the maximum growth rate of an organism from genomic data on the basis of codon usage patterns. Our work and that of other groups suggest that such predictors can be applied to mixed-species communities in order to derive estimates of the average community-wide maximum growth rate. Here, we present an improved maximum growth rate predictor designed for metagenomes that corrects a persistent GC bias in the original gRodon model for metagenomic prediction. We benchmark this predictor with simulated metagenomic data sets to show that it has superior performance on mixed-species communities relative to earlier models. We go on to provide guidance on data preprocessing and show that calling genes from assembled contigs rather than directly from reads dramatically improves performance. Finally, we apply our predictor to large-scale metagenomic data sets from marine and human microbiomes to illustrate how community-wide growth prediction can be a powerful approach for hypothesis generation. Altogether, we provide an updated tool with clear guidelines for users about the uses and pitfalls of metagenomic prediction of the average community-wide maximal growth rate. IMPORTANCE Microbes dominate nearly every known habitat, and therefore tools to survey the structure and function of natural microbial communities are much needed. Metagenomics, in which the DNA content of an entire community of organisms is sequenced all at once, allows us to probe the genetic diversity contained in a habitat. Yet, mapping metagenomic information to the actual traits of community members is a difficult and largely unsolved problem. Here, we present and validate a tool that allows users to predict the average maximum growth rate of a microbial community directly from metagenomic data. Maximum growth rate is a fundamental characteristic of microbial species that can give us a great deal of insight into their ecological role, and by applying our community-level predictor to large-scale metagenomic data sets from marine and human-associated microbiomes, we show how community-wide growth prediction can be a powerful approach for hypothesis generation.

[Source](https://journals.asm.org/doi/10.1128/msystems.00745-22)

10.1128/msystems.00745-22

---

## PhyloDM is a high-performance library that converts a phylogenetic tree into a pairwise distance matrix.

[Source](https://github.com/aaronmussig/PhyloDM)

---

