## A human gut <em>Faecalibacterium prausnitzii</em> fatty acid amide hydrolase

[Source](https://doi.org/10.1126/science.ado6828)

---

## A ubiquitous mobile genetic element changes the antagonistic weaponry of a human gut symbiont.
 29.10.2024, _ML Sheahan, K Flores, MJ Coyne, L García-Bayona, M Chatzidaki-Livanis, AQ Holst, RC Smith, A Sundararajan, B Barquera, LE Comstock_


_Humans, Bacteroides fragilis, Conjugation, Genetic, Gastrointestinal Microbiome, Gene Transfer, Horizontal, Genetic Fitness, Genome, Bacterial, Interspersed Repetitive Sequences, Symbiosis, Type VI Secretion Systems, DNA, Bacterial_
DNA transfer is ubiquitous in the human gut microbiota, especially among species of the order Bacteroidales. In silico analyses have revealed hundreds of mobile genetic elements shared between these species, yet little is known about the phenotypes they encode, their effects on fitness, or pleiotropic consequences for the recipient's genome. In this work, we show that acquisition of a ubiquitous integrative conjugative element (ICE) encoding a type VI secretion system (T6SS) shuts down the native T6SS of Bacteroides fragilis. Despite inactivating this T6SS, ICE acquisition increases the fitness of the B. fragilis transconjugant over its progenitor by arming it with the new T6SS. DNA transfer causes the strain to change allegiances so that it no longer targets ecosystem members with the same element yet is armed for communal defense.

[Source](https://doi.org/10.1126/science.adj9504)

10.1126/science.adj9504

---

## Exploring alterations of gut/blood microbes in addressing iron overload-induced gut dysbiosis and cognitive impairment in thalassemia patients.
 25.10.2024, _K Suparan, K Trirattanapa, P Piriyakhuntorn, S Sriwichaiin, C Thonusin, W Nawara, S Kerdpoo, N Chattipakorn, A Tantiworawit, SC Chattipakorn_


_Humans, Dysbiosis, Gastrointestinal Microbiome, Male, Female, Thalassemia, Adult, Iron Overload, Cognitive Dysfunction, Young Adult, Middle Aged, Brain-Gut Axis, Case-Control Studies_
Iron overload causes cognitive impairment in thalassemia patients. The gut-brain axis plays an important role in cognitive function. However, the association between gut/blood microbiome, cognition, and iron burden in thalassemia patients has not been thoroughly investigated. We aimed to determine those associations in thalassemia patients with different blood-transfusion regimens. Sixty participants: healthy controls, transfusion-dependent thalassemia (TDT) patients, and non-transfusion-dependent (NTDT) patients, were recruited to evaluate iron overload, cognition, and gut/blood microbiome. TDT patients exhibited greater iron overload than NTDT patients. Most thalassemia patients developed gut dysbiosis, and approximately 25% of the patients developed minor cognitive impairment. Increased Fusobacteriota and Verrucomicrobiota with decreased Fibrobacterota were observed in both TDT and NTDT groups. TDT patients showed more abundant beneficial bacteria: Verrucomicrobia. Iron overload was correlated with cognitive impairment. Increased Butyricimonas and decreased Paraclostridium were associated with higher cognitive function. No trace of blood microbiota was observed. Differences in blood bacterial profiles of thalassemia patients and controls were insignificant. These findings suggest iron overload plays a role in the imbalance of gut microbiota and impaired cognitive function in thalassemia patients. Harnessing probiotic potential from those microbes could prevent the gut-brain disturbance in thalassemia patients.

[Source](https://doi.org/10.1038/s41598-024-76684-4)

10.1038/s41598-024-76684-4

---

## Effects of bile acids on the growth, composition and metabolism of gut bacteria.
 25.10.2024, _YL Peng, SH Wang, YL Zhang, MY Chen, K He, Q Li, WH Huang, W Zhang_


_Gastrointestinal Microbiome, Bile Acids and Salts, Bacteria, Animals, Deoxycholic Acid, Mice, Amino Acids, Metabolomics, Humans, Ribosomes_
Bile acids (BAs) exert a profound influence on the body's pathophysiology by intricately shaping the composition of gut bacteria. However, the complex interplay between BAs and gut microbiota has impeded a systematic exploration of their impact on intestinal bacteria. Initially, we investigated the effects of 21 BAs on the growth of 65 gut bacterial strains in vitro. Subsequently, we examined the impact of BAs on the overall composition of intestinal bacteria, both in vivo and in vitro. The results unveiled distinct effects of various BAs on different intestinal strains and their diverse impacts on the composition of gut bacteria. Mechanistically, the inhibition of intestinal strains by BAs occurs through the accumulation of these acids within the strains. The intracellular accumulation of deoxycholic acid (DCA) significantly influenced the growth of intestinal bacteria by impacting ribosome transcription and amino-acid metabolism. The metabolomic analysis underscores the pronounced impact of DCA on amino-acid profiles in both in vivo and in vitro settings. This study not only elucidates the effects of BAs on a diverse range of bacterial strains and their role in shaping the gut microbiota but also reveals underlying mechanisms essential for understanding and maintaining a healthy gut microbiota.

[Source](https://doi.org/10.1038/s41522-024-00566-w)

10.1038/s41522-024-00566-w

---

## Gut-first Parkinson's disease is encoded by gut dysbiome.
 30.10.2024, _MF Munoz-Pinto, E Candeias, I Melo-Marques, AR Esteves, A Maranha, JD Magalhães, DR Carneiro, M Sant'Anna, AR Pereira-Santos, AE Abreu, D Nunes-Costa, S Alarico, I Tiago, A Morgadinho, J Lemos, PN Figueiredo, C Januário, N Empadinhas, SM Cardoso_


_Animals, Gastrointestinal Microbiome, Parkinson Disease, Mice, Dysbiosis, Male, Mice, Inbred C57BL, Humans, Fecal Microbiota Transplantation_
In Parkinson's patients, intestinal dysbiosis can occur years before clinical diagnosis, implicating the gut and its microbiota in the disease. Recent evidence suggests the gut microbiota may trigger body-first Parkinson Disease (PD), yet the underlying mechanisms remain unclear. This study aims to elucidate how a dysbiotic microbiome through intestinal immune alterations triggers PD-related neurodegeneration.

[Source](https://doi.org/10.1186/s13024-024-00766-0)

10.1186/s13024-024-00766-0

---

## Metaproteomics reveals diet-induced changes in gut microbiome function according to Crohn's disease location.
 30.10.2024, _S Levi Mortera, V Marzano, F Rapisarda, C Marangelo, I Pirona, P Vernocchi, M Di Michele, F Del Chierico, MA Quintero, I Fernandez, H Hazime, RM Killian, N Solis, M Ortega, OM Damas, S Proksell, DH Kerman, AR Deshpande, L Garces, F Scaldaferri, A Gasbarrini, MT Abreu, L Putignani_


_Humans, Crohn Disease, Gastrointestinal Microbiome, Male, Female, Feces, Adult, Proteomics, Middle Aged, Diet, Dietary Fiber, Bacteria, Colon, Young Adult, Faecalibacterium_
Crohn's disease (CD) is characterized by chronic intestinal inflammation. Diet is a key modifiable factor influencing the gut microbiome (GM) and a risk factor for CD. However, the impact of diet modulation on GM function in CD patients is understudied. Herein, we evaluated the effect of a high-fiber, low-fat diet (the Mi-IBD diet) on GM function in CD patients. All participants were instructed to follow the Mi-IBD diet for 8 weeks. One group of CD patients received one-time diet counseling only (Gr1); catered food was supplied for the other three groups, including CD patients (Gr2) and dyads of CD patients and healthy household controls (HHCs) residing within the same household (Gr3-HHC dyads). Stool samples were collected at baseline, week 8, and week 36, and analyzed by liquid chromatography-tandem mass spectrometry.

[Source](https://doi.org/10.1186/s40168-024-01927-5)

10.1186/s40168-024-01927-5

---

## Examining the healthy human microbiome concept

[Source](https://doi.org/10.1038/s41579-024-01107-0)

---

## Host-pathobiont interactions in Crohn’s disease

[Source](https://doi.org/10.1038/s41575-024-00997-y)

---

## Species- and subspecies-level characterization of health-associated bacterial consortia that colonize the human gut during infancy

[Source](https://doi.org/10.1080/19490976.2024.2414975)

---

## Metal availability shapes early life microbial ecology and community succession

[Source](https://doi.org/10.1128/mbio.01534-24)

---

## Oral reovirus reshapes the gut microbiome and enhances antitumor immunity in colon cancer

[Source](https://doi.org/10.1038/s41467-024-53347-6)

---

## Pooled analysis of oral microbiome profiles defines robust signatures associated with periodontitis

[Source](https://doi.org/10.1128/msystems.00930-24)

---

## Type VI secretion systems promote intraspecific competition and host interactions in a bee gut symbiont

[Source](https://doi.org/10.1073/pnas.2414882121)

---

## Gut bacteria facilitate leaf beetles in adapting to dietary specialization by enhancing larval fitness

[Source](https://doi.org/10.1038/s41522-024-00587-5)

---

## The dietary effects of two strain probiotics (<em>Leuconostoc mesenteroides</em>, <em>Lactococcus lactis</em>) on growth performance, immune response and gut microbiota in Nile tilapia (<em>Oreochromis niloticus</em>)

[Source](https://doi.org/10.1371/journal.pone.0312580)

---

## Effects of <em>Lactiplantibacillus plantarum</em> CBT LP3 and<em> Bifidobacterium breve</em> CBT BR3 supplementation on weight loss and gut microbiota of overweight dogs

[Source](https://doi.org/10.1038/s41598-024-75594-9)

---

## Maternal gastrointestinal microbiome shapes gut microbial function and resistome of newborns in a cow-to-calf model

[Source](https://doi.org/10.1186/s40168-024-01943-5)

---

## Integrated analysis of metabolome and microbiome in a rat model of perimenopausal syndrome

[Source](https://doi.org/10.1128/msystems.00623-24)

---

## Maternal dysbiosis produces long-lasting behavioral changes in offspring

[Source](https://doi.org/10.1038/s41380-024-02794-0)

---

## DNA metabarcoding analyses reveal fine-scale microbiome structures on Western Canadian bat wings.
 22.10.2024, _C Insuk, N Cheeptham, C Lausen, J Xu_


Healthy wings are vital for the survival and reproduction of bats, and wing microbiome is a key component of bat wing health. However, relatively little is known about the wing microbiome of bats in western Canada where the white nose syndrome has become an increasing threat. Here, we used DNA metabarcoding to investigate the bacterial and fungal communities on the wings of three bat species: the big brown bat (Eptesicus fuscus), the Yuma myotis (Myotis yumanensis), and the little brown myotis (M. lucifugus) from four field sites in Lillooet, British Columbia, Canada. The bacterial 16S rRNA metabarcoding revealed a total of 4,167 amplicon sequence variants (ASVs) belonging to 27 phyla, 639 genera, and 533 known and 2,423 unknown species. The wing bacteria were dominated by phyla Proteobacteria, Firmicutes, Bacteroides, and Actinobacteria, and the most common genera were Delftia, Bordetella, Sphingomonas, Phyllobacterium, Bradyrhizobium, Pseudomonas, and Corynebacterium. The fungal internal transcribed spacer (ITS) metabarcoding revealed a total of 11,722 ASVs belonging to 16 phyla, 806 genera, and 1,420 known and 10,302 unknown species. The wing fungi were dominated by phyla Ascomycota, Basidiomycota, and Motierellomycota, and the most common genera were Cladosporium, Aspergillus, and Mycosphaerella. Principal coordinates analysis showed that both bat species and field sites contributed variably to the diversity and distribution of bacterial and fungal communities on bat wings. Interestingly, both positive and negative correlations were found in their relative abundances among several groups of microbial taxa. We discuss the implications of our results for bat health, including the management of P. destructans infection and white-nose syndrome spread.

[Source](https://doi.org/10.1128/spectrum.00376-24)

10.1128/spectrum.00376-24

---

## Aquificae overcomes competition by archaeal thermophiles, and crowding by bacterial mesophiles, to dominate the boiling vent-water of a Trans-Himalayan sulfur-borax spring.
 27.10.2024, _N Mondal, S Dutta, S Chatterjee, J Sarkar, M Mondal, C Roy, R Chakraborty, W Ghosh_


_Hot Springs, RNA, Ribosomal, 16S, Bacteria, Sulfur, Archaea, Phylogeny, Microbiota, Boron, Water Microbiology_
Trans-Himalayan hot spring waters rich in boron, chlorine, sodium and sulfur (but poor in calcium and silicon) are known based on PCR-amplified 16S rRNA gene sequence data to harbor high diversities of infiltrating bacterial mesophiles. Yet, little is known about the community structure and functions, primary productivity, mutual interactions, and thermal adaptations of the microorganisms present in the steaming waters discharged by these geochemically peculiar spring systems. We revealed these aspects of a bacteria-dominated microbiome (microbial cell density ~8.5 × 104 mL-1; live:dead cell ratio 1.7) thriving in the boiling (85°C) fluid vented by a sulfur-borax spring called Lotus Pond, situated at 4436 m above the mean sea-level, in the Puga valley of eastern Ladakh, on the Changthang plateau. Assembly, annotation, and population-binning of &gt;15-GB metagenomic sequence illuminated the numeral predominance of Aquificae. While members of this phylum accounted for 80% of all 16S rRNA-encoding reads within the metagenomic dataset, 14% of such reads were attributed to Proteobacteria. Post assembly, only 25% of all protein-coding genes identified were attributable to Aquificae, whereas 41% was ascribed to Proteobacteria. Annotation of metagenomic reads encoding 16S rRNAs, and/or PCR-amplified 16S rRNA genes, identified 163 bacterial genera, out of which 66 had been detected in past investigations of Lotus Pond's vent-water via 16S amplicon sequencing. Among these 66, Fervidobacterium, Halomonas, Hydrogenobacter, Paracoccus, Sulfurihydrogenibium, Tepidimonas, Thermus and Thiofaba (or their close phylogenomic relatives) were presently detected as metagenome-assembled genomes (MAGs). Remarkably, the Hydrogenobacter related MAG alone accounted for ~56% of the entire metagenome, even though only 15 out of the 66 genera consistently present in Lotus Pond's vent-water have strains growing in the laboratory at &gt;45°C, reflecting the continued existence of the mesophiles in the ecosystem. Furthermore, the metagenome was replete with genes crucial for thermal adaptation in the context of Lotus Pond's geochemistry and topography. In terms of sequence similarity, a majority of those genes were attributable to phylogenetic relatives of mesophilic bacteria, while functionally they rendered functions such as encoding heat shock proteins, molecular chaperones, and chaperonin complexes; proteins controlling/modulating/inhibiting DNA gyrase; universal stress proteins; methionine sulfoxide reductases; fatty acid desaturases; different toxin-antitoxin systems; enzymes protecting against oxidative damage; proteins conferring flagellar structure/function, chemotaxis, cell adhesion/aggregation, biofilm formation, and quorum sensing. The Lotus Pond Aquificae not only dominated the microbiome numerically but also acted potentially as the main primary producers of the ecosystem, with chemolithotrophic sulfur oxidation (Sox) being the fundamental bioenergetic mechanism, and reductive tricarboxylic acid (rTCA) cycle the predominant carbon fixation pathway. The Lotus Pond metagenome contained several genes directly or indirectly related to virulence functions, biosynthesis of secondary metabolites including antibiotics, antibiotic resistance, and multi-drug efflux pumping. A large proportion of these genes being attributable to Aquificae, and Proteobacteria (very few were ascribed to Archaea), it could be worth exploring in the future whether antibiosis helped the Aquificae overcome niche overlap with other thermophiles (especially those belonging to Archaea), besides exacerbating the bioenergetic costs of thermal endurance for the mesophilic intruders of the ecosystem.

[Source](https://doi.org/10.1371/journal.pone.0310595)

10.1371/journal.pone.0310595

---

## Metagenome-assembled genomes of Macrocystis-associated bacteria.
 22.10.2024, _EG Aguirre, JA Schwartzman_


Microbes are important for the development of kelp, but little is understood about the functions encoded by these microbes. Here, we assembled 1,794 metagenome-associated genomes (MAGs) from bacteria that colonize gametophytes of the giant kelp Macrocystis pyrifera. MAGs were grouped into 149 dereplicated strains. Functional annotation is also presented.

[Source](https://doi.org/10.1128/mra.00715-24)

10.1128/mra.00715-24

---

## Metagenomic time-series reveals a western English Channel viral community dominated by members with strong seasonal signals.
 23.10.2024, _LM Bolaños, M Michelsen, B Temperton_


Marine viruses are key players of ocean biogeochemistry, profoundly influencing microbial community ecology and evolution. Despite their importance, few studies have explored continuous inter-seasonal viral metagenomic time-series in marine environments. Viral dynamics are complex, influenced by multiple factors such as host population dynamics and environmental conditions. To disentangle the complexity of viral communities, we developed an unsupervised machine learning framework to classify viral contigs into "chronotypes" based on temporal abundance patterns. Analysing an inter-seasonal monthly time-series of surface viral metagenomes from the Western English Channel, we identified chronotypes and compared their functional and evolutionary profiles. Results revealed a consistent annual cycle with steep compositional changes from winter to summer and steadier transitions from summer to winter. Seasonal chronotypes were enriched in potential auxiliary metabolic genes of the ferrochelatases and 2OG-Fe(II) oxygenase orthologous groups compared to non-seasonal types. Chronotypes clustered into four groups based on their correlation profiles with environmental parameters, primarily driven by temperature and nutrients. Viral contigs exhibited a rapid turnover of polymorphisms, akin to Red Queen dynamics. However, within seasonal chronotypes, some sequences exhibited annual polymorphism recurrence, suggesting that a fraction of the seasonal viral populations evolve more slowly. Classification into chronotypes revealed viral genomic signatures linked to temporal patterns, likely reflecting metabolic adaptations to environmental fluctuations and host dynamics. This novel framework enables the identification of long-term trends in viral composition, environmental influences on genomic structure, and potential viral interactions.

[Source](https://doi.org/10.1093/ismejo/wrae216)

10.1093/ismejo/wrae216

---

## Volatile-mediated interspecific plant interaction promotes root colonization by beneficial bacteria via induced shifts in root exudation.
 23.10.2024, _X Zhou, J Zhang, J Shi, M Khashi U Rahman, H Liu, Z Wei, F Wu, F Dini-Andreote_


_Volatile Organic Compounds, Plant Roots, Rhizosphere, Solanum lycopersicum, Bacteria, Soil Microbiology, Plant Exudates, Microbiota, Solanum tuberosum, Pseudomonas_
Volatile organic compounds (VOCs) released by plants can act as signaling molecules mediating ecological interactions. Therefore, the study of VOCs mediated intra- and interspecific interactions with downstream plant physiological responses is critical to advance our understanding of mechanisms underlying information exchange in plants. Here, we investigated how plant-emitted VOCs affect the performance of an interspecific neighboring plant via induced shifts in root exudate chemistry with implications for root-associated microbiota recruitment.

[Source](https://doi.org/10.1186/s40168-024-01914-w)

10.1186/s40168-024-01914-w

---

## Microbial community acclimation during anaerobic digestion of high-oil food waste

[Source](https://doi.org/10.1038/s41598-024-77136-9)

---

## The choice of 16S rRNA gene sequence analysis impacted characterization of highly variable surface microbiota in dairy processing environments.
 21.10.2024, _SE Daly, J Feng, D Daeschel, J Kovac, AB Snyder_


Accurate knowledge of the microbiota collected from surfaces in food processing environments is important for food quality and safety. This study assessed discrepancies in taxonomic composition and alpha and beta diversity values generated from eight different bioinformatic workflows for the analysis of 16S rRNA gene sequences extracted from the microbiota collected from surfaces in dairy processing environments. We found that the microbiota collected from environmental surfaces varied widely in density (0-9.09 log10 CFU/cm2) and Shannon alpha diversity (0.01-3.40). Consequently, depending on the sequence analysis method used, characterization of low-abundance genera (i.e., below 1% relative abundance) and the number of genera identified (114-173 genera) varied considerably. Some low-abundance genera, including Listeria, varied between the amplicon sequence variant (ASV) and operational taxonomic unit (OTU) methods. Centered log-ratio transformation inflated alpha and beta diversity values compared to rarefaction. Furthermore, the ASV method also inflated alpha and beta diversity values compared to the OTU method (P &lt; 0.05). Therefore, for sparse, uneven, low-density data sets, the OTU method and rarefaction are better for taxonomic and ecological characterization of surface microbiota.IMPORTANCECulture-dependent environmental monitoring programs are used by the food industry to identify foodborne pathogens and spoilage biota on surfaces in food processing environments. The use of culture-independent 16S rRNA amplicon sequencing to characterize this surface microbiota has been proposed as a tool to enhance environmental monitoring. However, there is no consensus on the most suitable bioinformatic analyses to accurately capture the diverse levels and types of bacteria on surfaces in food processing environments. Here, we quantify the impact of different bioinformatic analyses on the results and interpretation of 16S rRNA amplicon sequences collected from three cultured dairy facilities in New York State. This study provides guidance for the selection of appropriate 16S rRNA analysis procedures for studying environmental microbiota in dairy processing environments.

[Source](https://doi.org/10.1128/msystems.00620-24)

10.1128/msystems.00620-24

---

## A single-stranded based library preparation method for virome characterization.
 30.10.2024, _X Zhai, A Gobbi, W Kot, L Krych, DS Nielsen, L Deng_


_Virome, Genome, Viral, Gastrointestinal Microbiome, Gene Library, Humans, DNA, Viral, DNA, Single-Stranded, High-Throughput Nucleotide Sequencing, Reproducibility of Results, Sequence Analysis, DNA_
The gut virome is an integral component of the gut microbiome, playing a crucial role in maintaining gut health. However, accurately depicting the entire gut virome is challenging due to the inherent diversity of genome types (dsDNA, ssDNA, dsRNA, and ssRNA) and topologies (linear, circular, or fragments), with subsequently biases associated with current sequencing library preparation methods. To overcome these problems and improve reproducibility and comparability across studies, universal or standardized virome sequencing library construction methods are highly needed in the gut virome study.

[Source](https://doi.org/10.1186/s40168-024-01935-5)

10.1186/s40168-024-01935-5

---

## Microbiome-based correction for random errors in nutrient profiles derived from self-reported dietary assessments.
 29.10.2024, _T Wang, Y Fu, M Shuai, JS Zheng, L Zhu, AT Chan, Q Sun, FB Hu, ST Weiss, YY Liu_


_Humans, Self Report, Gastrointestinal Microbiome, Nutrients, Diet Records, Diet, Deep Learning, Nutrition Assessment, Female, Male_
Since dietary intake is challenging to directly measure in large-scale cohort studies, we often rely on self-reported instruments (e.g., food frequency questionnaires, 24-hour recalls, and diet records) developed in nutritional epidemiology. Those self-reported instruments are prone to measurement errors, which can lead to inaccuracies in the calculation of nutrient profiles. Currently, few computational methods exist to address this problem. In the present study, we introduce a deep-learning approach-Microbiome-based nutrient profile corrector (METRIC), which leverages gut microbial compositions to correct random errors in self-reported dietary assessments using 24-hour recalls or diet records. We demonstrate the excellent performance of METRIC in minimizing the simulated random errors, particularly for nutrients metabolized by gut bacteria in both synthetic and three real-world datasets. Additionally, we find that METRIC can still correct the random errors well even without including gut microbial compositions. Further research is warranted to examine the utility of METRIC to correct actual measurement errors in self-reported dietary assessment instruments.

[Source](https://doi.org/10.1038/s41467-024-53567-w)

10.1038/s41467-024-53567-w

---

## Fecal samples and rectal swabs adequately reflect the human colonic luminal microbiota.
 27.10.2024, _J Rode, L Brengesjö Johnson, J König, I Rangel, L Engstrand, D Repsilber, RJ Brummer_


_Humans, Feces, Rectum, Gastrointestinal Microbiome, Male, Colon, Adult, Female, Specimen Handling, Middle Aged, Bacteria, Colonoscopy, Young Adult_
The appropriateness of the fecal microbiota to adequately reflect the gut microbiota composition from more difficult to access luminal content at different colonic locations has been debated. Here, in a healthy population, luminal samples were collected from terminal ileum to rectum using an unique sampling technique without the need of prior bowel cleansing/preparation. Rectal swabs were collected immediately prior colonoscopy by an experienced physician, and fecal samples were collected at home by the participants themselves. Microbiota composition was evaluated as relative abundance, α-diversity and Bray-Curtis dissimilarities. Our data suggest that fecal samples and rectal swabs present noninvasive, easily accessible, low-cost sampling tools that are accurate proxies to characterize luminal large intestinal microbiota composition.

[Source](https://doi.org/10.1080/19490976.2024.2416912)

10.1080/19490976.2024.2416912

---

## Optimal 16S rRNA gene amplicon sequencing analysis for oral microbiota to avoid the potential bias introduced by trimming length, primer, and database.
 29.10.2024, _T Nagai, T Shiba, K Komatsu, T Watanabe, T Nemoto, S Maekawa, R Kobayashi, S Matsumura, Y Ohsugi, S Katagiri, Y Takeuchi, T Iwata_


16S rRNA gene amplicon sequencing analysis is used to investigate bacterial communities; however, the estimated bacterial composition can differ from the original due to experimental and analytical biases. Therefore, this study determines the optimal conditions to minimize the potential biases in 16S rRNA gene amplicon sequencing analysis using different DNA samples, trimming lengths, primers, and databases. The results of the mock1 community with 250 and 300 bp paired-end (PE), comprising 15 bacteria from various environments, showed the highest similarity between the theoretical value and the data using the Greengenes2 and the V3 region at the genus level and V5-V6 at the species level. In the 300 bp PE sequencing analysis of the mock2 community, comprising six major oral bacteria, the data using the V3-V4, V4, and V5-V6 regions with the SILVA, Greengenes2, and the Human Oral Microbiome Database (HOMD) showed the highest similarity to the theoretical values at the genus level. At the species level, the data using the V3-V4 and V4 regions with Greengenes2 and the data using the V1-V2 with HOMD exhibited the highest alignment with the theoretical values. In the species analysis of the dental calculus samples with 300 bp PE, the Shannon index value was higher with the V1-V2 region and HOMD than with others. Our results suggest that the optimal conditions for oral microbiome analysis are the combinations of 300 bp PE and the V3-V4 or V4 region with the Greengenes2 and the V1-V2 with HOMD.IMPORTANCEThe 16S rRNA gene amplicon sequencing analysis is frequently used in oral microbiome research. However, this method can have biases that distort the experimental data, which depend on the methodological steps, including sequencing length, trimming length, selected amplification regions, and referenced databases. In this study, a combination of 300 bp PE and the primer targeting the V3-V4 or V4 regions with the Greengenes2 and the V1-V2 region with the HOMD was the most bias-minimizing condition for oral microbiota analysis. In addition, this is the first report of such analyses in modern Japanese dental calculus. The methods used in this study will aid in setting appropriate conditions for sequence analysis of microbiota obtained not only from the oral cavity but also from any environment.

[Source](https://doi.org/10.1128/spectrum.03512-23)

10.1128/spectrum.03512-23

---

## Moving Toward Metaproteogenomics: A Computational Perspective on Analyzing Microbial Samples via Proteogenomics.
 22.10.2024, _F Singer, M Kuhring, BY Renard, T Muth_


_Proteogenomics, Microbiota, Computational Biology, Proteomics, Software, Databases, Protein, Metagenomics, Algorithms, Metagenome, Humans, High-Throughput Nucleotide Sequencing, Mass Spectrometry, Proteome_
Microbial sample analysis has received growing attention within the last decade, driven by important findings in microbiome research and promising applications in the biotechnological field. Modern mass spectrometry-based methodology has been established in this context, providing sufficient sensitivity, resolution, dynamic range, and throughput to analyze the so-called metaproteome of complex microbial mixtures from clinical or environmental samples. While proteomic analyses were previously restricted to common model organisms, next-generation sequencing technologies nowadays allow for the rapid and cost-efficient characterization of whole metagenomes of microbial consortia and specific genomes from non-model organisms to which microbes contribute by significant amounts. This proteogenomic approach, meaning the combined application of genomic and proteomic methods, enables researchers to create a protein database that presents a tailored blueprint of the microbial sample under investigation. This contribution provides an overview of the computational challenges and opportunities in proteogenomics and metaproteomics as of January 2018. For practical application, we first showcase an integrative proteogenomic method that circumvents existing reference databases by creating sample-specific transcripts. The underlying algorithm uses a graph network approach that combines RNA-Seq and peptide information. As a second example, we provide a tutorial for a simulation tool that estimates the computational limits of detecting microbial non-model organisms. This method evaluates the potential influence of error-tolerant searches and proteogenomic approaches on databases of interest. Finally, we discuss recommendations for developing future strategies that may help overcome present limitations by combining the strengths of genome- and proteome-based methods and moving toward an integrated metaproteogenomics approach.

[Source](https://doi.org/10.1007/978-1-0716-4152-1_17)

10.1007/978-1-0716-4152-1_17

---

