## A 3D-printed transepidermal microprojection array for human skin microbiome sampling.
 22.07.2022, _K Liang, C Leong, JM Loh, N Chan, L Lim, YI Lam, TL Dawson, HL Tey_


Skin microbiome sampling is currently performed with tools such as swabs and tape strips to collect microbes from the skin surface. However, these conventional approaches may be unable to detect microbes deeper in the epidermis or in epidermal invaginations. We describe a sampling tool with a depth component, a transepidermal microprojection array (MPA), which captures microbial biomass from both the epidermal surface and deeper skin layers. We leveraged the rapid customizability of 3D printing to enable systematic optimization of MPA for human skin sampling. Evaluation of sampling efficacy on human scalp revealed the optimized MPA was comparable in sensitivity to swab and superior to tape strip, especially for nonstandard skin surfaces. We observed differences in species diversity, with the MPA detecting clinically relevant fungi more often than other approaches. This work delivers a tool in the complex field of skin microbiome sampling to potentially address gaps in our understanding of its role in health and disease.

[Source](https://doi.org/10.1073/pnas.2203556119)

10.1073/pnas.2203556119

---

## Analysis of Intestinal Mycobiota of Patients with <em>Clostridioides difficile</em> Infection among a Prospective Inpatient Cohort

[Source](https://doi.org/10.1128/spectrum.01362-22)

---

## Role of gastrointestinal microbial populations, a terra incognita of the human body in the management of intestinal bowel disease and metabolic disorders.
 22.07.2022, _JEV Bucheli, SD Todorov, WH Holzapfel_


Intestinal bowel disease (IBD) is a chronic immune-mediated clinical condition that affects the gastrointestinal tract and is mediated by an inflammatory response. Although it has been extensively studied, the multifactorial aetiology of this disorder makes it difficult to fully understand all the involved mechanisms in its development and therefore its treatment. In recent years, the fundamental role played by the human microbiota in the pathogenesis of IBD has been emphasised. Microbial imbalances in the gut bacterial communities and a lower species diversity in patients suffering from inflammatory gastrointestinal disorders compared to healthy individuals have been reported as principal factors in the development of IBD. These served to support scientific arguments for the use of probiotic microorganisms in alternative approaches for the prevention and treatment of IBD. In a homeostatic environment, the presence of bacteria (including probiotics) on the intestinal epithelial surface activates a cascade of processes by which immune responses inhibited and thereby commensal organisms maintained. At the same time these processes may support activities against specific pathogenic bacteria. In dysbiosis, these underlying mechanisms will serve to provoke a proinflammatory response, that, in combination with the use of antibiotics and the genetic predisposition of the host, will culminate in the development of IBD. In this review, we summarised the main causes of IBD, the physiological mechanisms involved and the related bacterial groups most frequently associated with these processes. The intention was to enable a better understanding of the interaction between the intestinal microbiota and the host, and to suggest possibilities by which this knowledge can be useful for the development of new therapeutic treatments.

[Source](https://doi.org/10.3920/BM2022.0022)

10.3920/BM2022.0022

---

## The Gut Bacterial Community Potentiates <em>Clostridioides difficile</em> Infection Severity

[Source](https://doi.org/10.1128/mbio.01183-22)

---

## The interplay of gut microbiota between donors and recipients determines the efficacy of fecal microbiota transplantation.
 24.07.2022, _R He, P Li, J Wang, B Cui, F Zhang, F Zhao_


_Bacteria, Clostridioides difficile, Clostridium Infections, Fecal Microbiota Transplantation, Feces, Gastrointestinal Microbiome, Humans, Inflammatory Bowel Diseases, Treatment Outcome_
Fecal microbiota transplantation (FMT) is a promising treatment for microbiota dysbiosis associated diseases, such as Clostridioides difficile infection (CDI) and inflammatory bowel disease (IBD). The engraftment of donor bacteria is essential for the effectiveness of FMT, which to some extent depends on the matching of donors and recipients. However, how different types of donor-derived bacteria affect FMT efficacy has not been fully dissected. We recruited two longitudinal IBD cohorts of 103 FMT recipients and further analyzed 1,280 microbiota datasets from 14 public CDI and IBD studies to uncover the effect of donor-derived microbiota in recipients. We found that two enterotypes, RCPT/E and RCPT/B (dominated by Enterobacteriaceae and Bacteroides, respectively), consistently exist in both CDI and IBD patients. Based on a time-course-based multi-cohort analysis of FMT fecal samples, we observed the interplay between recipient and donor-derived microbiota during FMT, in which the FMT outcome was significantly associated with the enterotype and microbiota distance between donor and recipient after FMT. We proposed a new measurement, the ratio of colonizers to residents after FMT (C2R), to quantify the engraftment of donor-derived bacteria in the recipients, and then constructed an enterotype-based statistical model for donor-recipient matching, which was validated by both cross-validation and an additional IBD FMT cohort (n = 42). We believe that with the accumulation of FMT multi-omics datasets, machine learning-based methods will be helpful for rational donor selection for improving efficacy and precision FMT practices.

[Source](https://doi.org/10.1080/19490976.2022.2100197)

10.1080/19490976.2022.2100197

---

## Gut commensals expand vitamin A metabolic capacity of the mammalian host.
 21.07.2022, _M Bonakdar, LC Czuba, G Han, G Zhong, H Luong, N Isoherrannen, S Vaishnava_


Conversion of dietary vitamin A (VA) into retinoic acid (RA) is essential for many biological processes and thus far studied largely in mammalian cells. Using targeted metabolomics, we found that commensal bacteria in the mouse gut lumen produced a high concentration of the active retinoids, all-trans-retinoic acid (atRA) and 13-cis-retinoic acid (13cisRA), as well as the principal circulating retinoid, retinol. Ablation of anerobic bacteria significantly reduced retinol, atRA, and 13cisRA, whereas introducing these bacteria into germ-free mice significantly enhanced retinoids. Remarkably, cecal bacterial supplemented with VA produced active retinoids in vitro, establishing that gut bacteria encode metabolic machinery necessary for multistep conversion of dietary VA into its active forms. Finally, gut bacteria Lactobacillus intestinalis metabolized VA and specifically restored RA levels in the gut of vancomycin-treated mice. Our work establishes vitamin A metabolism as an emergent property of the gut microbiome and lays the groundwork for developing probiotic-based retinoid therapy.

[Source](https://doi.org/10.1016/j.chom.2022.06.011)

10.1016/j.chom.2022.06.011

---

## A reappraisal of microbiome dysbiosis during experimental periodontitis".
 21.07.2022, _M Arce, N Endo, N Dutzan, L Abusleme_


Periodontitis is a chronic inflammatory disease associated with the presence of dysbiotic microbial communities. Several studies interrogating periodontitis pathogenesis have utilized the murine ligature-induced periodontitis (LIP) model, and have further examined the ligature-associated microbiome relying on 16SrRNA-based sequencing techniques. However, it is often very challenging to compare microbial profiles across studies, due to important differences in bioinformatic processing and databases used for taxonomic assignment. Thus, our study aim was to re-analyze microbiome sequencing data-sets from studies utilizing the LIP model, through a standardized bioinformatic analysis pipeline, generating a comprehensive overview of the microbial dysbiosis during experimental periodontitis. We conducted a re-analysis of 16SrRNA gene sequencing datasets from 9 published studies utilizing the LIP model. Reads were grouped according the hypervariable region of the 16SrRNA gene amplified (V1-V3 and V4), preprocessed, binned into Operational Taxonomical Units (OTUs) and classified utilizing relevant databases. Alpha and beta-diversity analyses were conducted, alongside relative abundance profiling of microbial communities. Our findings revealed similar microbial richness and diversity across studies, and determined shifts in microbial community structure determined by periodontitis induction and study of origin. Clear variations in the relative abundance of bacterial taxa were observed starting day 5 after ligation and onwards, consistent with a distinct microbial composition during health and experimental periodontitis. We also uncovered differentially represented bacterial taxa across studies, dominating periodontal health and LIP-associated communities. Collectively, this re-analysis provides a unified overview of microbial dysbiosis during the LIP model, providing new insights that aim to inform further studies dedicated to unravelling oral host-microbial interactions. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1111/omi.12382)

10.1111/omi.12382

---

## Activity-based protein profiling identifies alternating activation of enzymes involved in the bifidobacterium shunt pathway or mucin degradation in the gut microbiome response to soluble dietary fiber.
 24.07.2022, _BJ Killinger, C Whidbey, NC Sadler, AJ DeLeon, N Munoz, YM Kim, AT Wright_


_Animals, Bacteria, Bifidobacterium, Carrier Proteins, Dietary Fiber, Feces, Gastrointestinal Microbiome, Mice, Mucins, Sugars_
While deprivation of dietary fiber has been associated with adverse health outcomes, investigations concerning the effect of dietary fiber on the gut microbiome have been largely limited to compositional sequence-based analyses or utilize a defined microbiota not native to the host. To extend understanding of the microbiome's functional response to dietary fiber deprivation beyond correlative evidence from sequence-based analyses, approaches capable of measuring functional enzymatic activity are needed. In this study, we use an activity-based protein profiling (ABPP) approach to identify sugar metabolizing and transport proteins in native mouse gut microbiomes that respond with differential activity to the deprivation or supplementation of the soluble dietary fibers inulin and pectin. We found that the microbiome of mice subjected to a high fiber diet high in soluble fiber had increased functional activity of multiple proteins, including glycoside hydrolases, polysaccharide lyases, and sugar transport proteins from diverse taxa. The results point to an increase in activity of the Bifidobacterium shunt metabolic pathway in the microbiome of mice fed high fiber diets. In those subjected to a low fiber diet, we identified a shift from the degradation of dietary fibers to that of gut mucins, in particular by the recently isolated taxon "Musculibacterium intestinale", which experienced dramatic growth in response to fiber deprivation. When combined with metabolomics and shotgun metagenomics analyses, our findings provide a functional investigation of dietary fiber metabolism in the gut microbiome and demonstrates the power of a combined ABPP-multiomics approach for characterizing the response of the gut microbiome to perturbations.

[Source](https://doi.org/10.1038/s41522-022-00313-z)

10.1038/s41522-022-00313-z

---

## Activity-based protein profiling identifies alternating activation of enzymes involved in the <em>bifidobacterium</em> shunt pathway or mucin degradation in the gut microbiome response to soluble dietary fiber

[Source](https://doi.org/10.1038/s41522-022-00313-z)

---

## Cooperative action of gut-microbiota-accessible carbohydrates improves host metabolic function.
 22.07.2022, _S Tomioka, N Seki, Y Sugiura, M Akiyama, J Uchiyama, G Yamaguchi, K Yakabe, R Ejima, K Hattori, T Kimizuka, Y Fujimura, H Sato, M Gondo, S Ozaki, Y Honme, M Suematsu, I Kimura, N Inohara, G Núñez, K Hase, YG Kim_


_Animals, Arabinose, Bacteroides, Carbohydrates, Fatty Acids, Volatile, Gastrointestinal Microbiome, Mice, Microbiota, Obesity, Sucrose_
Microbiota-accessible carbohydrates (MACs) exert health-promoting effects, but how each MAC impacts gut microbiota and regulates host physiology remains unclear. Here, we show that l-arabinose and sucrose cooperatively act on gut microbiota and exert anti-obesogenic effects. Specifically, l-arabinose, a monosaccharide that is poorly absorbed in the gut and inhibits intestinal sucrase, suppresses diet-induced obesity in mice in the presence of sucrose. Additionally, the suppressive effect of l-arabinose on adiposity is abrogated in mice lacking the short-chain fatty acid (SCFA) receptors GPR43 and GPR41. Mechanistically, l-arabinose increases the relative abundance of acetate and propionate producers (e.g., Bacteroides), while sucrose enhances SCFA production. Furthermore, l-arabinose and sucrose activate the glycolytic and pentose phosphate pathways of Bacteroides, respectively, indicating that they synergistically promote acetate production through distinct pathways. These findings suggest that each MAC has a unique property and thus may serve as a precision gut-microbiota modulator to promote host homeostasis.

[Source](https://doi.org/10.1016/j.celrep.2022.111087)

10.1016/j.celrep.2022.111087

---

## Key features of the genetic architecture and evolution of host-microbe interactions revealed by high-resolution genetic mapping of the mucosa-associated gut microbiome in hybrid mice.
 22.07.2022, _S Doms, H Fokt, MC Rühlemann, CJ Chung, A Kuenstner, SM Ibrahim, A Franke, LM Turner, JF Baines_


Determining the forces that shape diversity in host-associated bacterial communities is critical to understanding the evolution and maintenance of metaorganisms. To gain deeper understanding of the role of host genetics in shaping gut microbial traits, we employed a powerful genetic mapping approach using inbred lines derived from the hybrid zone of two incipient house mouse species. Furthermore, we uniquely performed our analysis on microbial traits measured at the gut mucosal interface, which is in more direct contact with host cells and the immune system. Several mucosa-associated bacterial taxa have high heritability estimates, and interestingly, 16S rRNA transcript-based heritability estimates are positively correlated with cospeciation rate estimates. Genome-wide association mapping identifies 428 loci influencing 120 taxa, with narrow genomic intervals pinpointing promising candidate genes and pathways. Importantly, we identified an enrichment of candidate genes associated with several human diseases, including inflammatory bowel disease, and functional categories including innate immunity and G-protein-coupled receptors. These results highlight key features of the genetic architecture of mammalian host-microbe interactions and how they diverge as new species form.

[Source](https://doi.org/10.7554/eLife.75419)

10.7554/eLife.75419

---

## Metagenomic insights into the microbe-mediated B and K(2) vitamin biosynthesis in the gastrointestinal microbiome of ruminants

[Source](https://doi.org/10.1186/s40168-022-01298-9)

---

## Exploring the Transient Microbe Population on Citrus Butterfly Wings.
 20.07.2022, _PD Kamala Jayanthi, M Vyas_


Microbes carve out dwelling niches in unusual environments. Insects, in general, have been hosts to microbes in different ways. Some insects incorporate microbes as endosymbionts that help with metabolic functions, while some vector pathogenic microbes that cause serious plant and animal diseases, including humans. Microbes isolated from insect sources have been beneficial and a huge information repository. The fascinating and evolutionarily successful insect community has survived mass extinctions as a result of their unique biological traits. Wings have been one of the most important factors contributing to the evolutionary success of insects. In the current study, wings of Papilio polytes, a citrus butterfly, were investigated for the presence of ecologically significant microbes within hours of eclosing under aseptic conditions. Scanning electron microscopy (SEM) revealed the presence of bacteria dwelling in crevices created by a specific arrangement of scales on the butterfly wing. A total of 38 bacterial isolates were obtained from the patched wings of the citrus butterfly, and Bacillus spp. were predominant among them. We probed the occurrence of these microbes to assess their significance to the insect. Many of the isolates displayed antibacterial, antifungal, and biosurfactant properties. Interestingly, one of the isolates displayed entomopathogenic potential toward the notorious agricultural pest mealybug. All the wing isolates were seen to cluster together consistently in a phylogenetic analysis, except for one isolate of Bacillus zhangzhouensis (Papilio polytes isolate [Pp] no. 28), suggesting they are distinct strains. IMPORTANCE This is a first study reporting the presence of culturable microbes on an unusual ecological niche such as butterfly wings. Our findings also establish that microbes inhabit these niches before the butterfly has contact with the environment. The findings in this report have opened up a new area of research which will not only help understand the microbiome of insect wings but might prove beneficial in other specialized studies.

[Source](https://doi.org/10.1128/spectrum.02055-21)

10.1128/spectrum.02055-21

---

## Methods and Strategies to Uncover Coral-Associated Microbial Dark Matter.
 21.07.2022, _J Schultz, F Modolon, AS Rosado, CR Voolstra, M Sweet, RS Peixoto_


The vast majority of environmental microbes have not yet been cultured, and most of the knowledge on coral-associated microbes (CAMs) has been generated from amplicon sequencing and metagenomes. However, exploring cultured CAMs is key for a detailed and comprehensive characterization of the roles of these microbes in shaping coral health and, ultimately, for their biotechnological use as, for example, coral probiotics and other natural products. Here, the strategies and technologies that have been used to access cultured CAMs are presented, while advantages and disadvantages associated with each of these strategies are discussed. We highlight the existing gaps and potential improvements in culture-dependent methodologies, indicating several possible alternatives (including culturomics and in situ diffusion devices) that could be applied to retrieve the CAM "dark matter" (i.e., the currently undescribed CAMs). This study provides the most comprehensive synthesis of the methodologies used to recover the cultured coral microbiome to date and draws suggestions for the development of the next generation of CAM culturomics.

[Source](https://doi.org/10.1128/msystems.00367-22)

10.1128/msystems.00367-22

---

## Plant genetic effects on microbial hubs impact host fitness in repeated field trials.
 22.07.2022, _B Brachi, D Filiault, H Whitehurst, P Darme, P Le Gars, M Le Mentec, TC Morton, E Kerdaffrec, F Rabanal, A Anastasio, MS Box, S Duncan, F Huang, R Leff, P Novikova, M Perisin, T Tsuchimatsu, R Woolley, C Dean, M Nordborg, S Holm, J Bergelson_


Although complex interactions between hosts and microbial associates are increasingly well documented, we still know little about how and why hosts shape microbial communities in nature. In addition, host genetic effects on microbial communities vary widely depending on the environment, obscuring conclusions about which microbes are impacted and which plant functions are important. We characterized the leaf microbiota of 200 Arabidopsis thaliana genotypes in eight field experiments and detected consistent host effects on specific, broadly distributed microbial species (operational taxonomic unit [OTUs]). Host genetic effects disproportionately influenced central ecological hubs, with heritability of particular OTUs declining with their distance from the nearest hub within the microbial network. These host effects could reflect either OTUs preferentially associating with specific genotypes or differential microbial success within them. Host genetics associated with microbial hubs explained over 10% of the variation in lifetime seed production among host genotypes across sites and years. We successfully cultured one of these microbial hubs and demonstrated its growth-promoting effects on plants in sterile conditions. Finally, genome-wide association mapping identified many putatively causal genes with small effects on the relative abundance of microbial hubs across sites and years, and these genes were enriched for those involved in the synthesis of specialized metabolites, auxins, and the immune system. Using untargeted metabolomics, we corroborate the consistent association between variation in specialized metabolites and microbial hubs across field sites. Together, our results reveal that host genetic variation impacts the microbial communities in consistent ways across environments and that these effects contribute to fitness variation among host genotypes.

[Source](https://doi.org/10.1073/pnas.2201285119)

10.1073/pnas.2201285119

---

## Beneficial and pathogenic plant-microbe interactions during flooding stress.
 22.07.2022, _C Martínez-Arias, J Witzell, A Solla, JA Martín, JR Calcerrada_


The number and intensity of flood events will likely increase in the future, raising the risk of flooding stress in terrestrial plants. Understanding flood effects on plant physiology and plant-associated microbes is key to alleviate flooding stress in sensitive species and ecosystems. Reduced oxygen supply is the main constrain to the plant and its associated microbiome. Hypoxic conditions hamper root aerobic respiration and, consequently, hydraulic conductance, nutrient uptake, and plant growth and development. Hypoxia favors the presence of anaerobic microbes in the rhizosphere and roots with potential negative effects to the plant due to their pathogenic behavior or their soil denitrification ability. Moreover, plant physiological and metabolic changes induced by flooding stress may also cause dysbiotic changes in endosphere and rhizosphere microbial composition. The negative effects of flooding stress on the holobiont (i.e. the host plant and its associated microbiome) can be mitigated once the plant displays adaptive responses to increase oxygen uptake. Stress relief could also arise from the positive effect of certain beneficial microbes, such as mycorrhiza or dark septate endophytes. More research is needed to explore the spiraling, feedback flood responses of plant and microbes if we want to promote plant flood tolerance from a holobiont perspective. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1111/pce.14403)

10.1111/pce.14403

---

## Harnessing the microbiome to prevent global biodiversity loss.
 21.07.2022, _RS Peixoto, CR Voolstra, M Sweet, CM Duarte, S Carvalho, H Villela, JE Lunshof, L Gram, DC Woodhams, J Walter, A Roik, U Hentschel, RV Thurber, B Daisley, B Ushijima, D Daffonchio, R Costa, T Keller-Costa, JS Bowman, AS Rosado, G Reid, CE Mason, JB Walke, T Thomas, G Berg_


Global biodiversity loss and mass extinction of species are two of the most critical environmental issues the world is currently facing, resulting in the disruption of various ecosystems central to environmental functions and human health. Microbiome-targeted interventions, such as probiotics and microbiome transplants, are emerging as potential options to reverse deterioration of biodiversity and increase the resilience of wildlife and ecosystems. However, the implementation of these interventions is urgently needed. We summarize the current concepts, bottlenecks and ethical aspects encompassing the careful and responsible management of ecosystem resources using the microbiome (termed microbiome stewardship) to rehabilitate organisms and ecosystem functions. We propose a real-world application framework to guide environmental and wildlife probiotic applications. This framework details steps that must be taken in the upscaling process while weighing risks against the high toll of inaction. In doing so, we draw parallels with other aspects of contemporary science moving swiftly in the face of urgent global challenges.

[Source](https://doi.org/10.1038/s41564-022-01173-1)

10.1038/s41564-022-01173-1

---

## Proteome Exploration of <em>Legionella pneumophila</em> To Identify Novel Therapeutics: a Hierarchical Subtractive Genomics and Reverse Vaccinology Approach

[Source](https://doi.org/10.1128/spectrum.00373-22)

---

## LOCOM: A logistic regression model for testing differential abundance in compositional microbiome data with false discovery rate control.
 22.07.2022, _Y Hu, GA Satten, YJ Hu_


Compositional analysis is based on the premise that a relatively small proportion of taxa are differentially abundant, while the ratios of the relative abundances of the remaining taxa remain unchanged. Most existing methods use log-transformed data, but log-transformation of data with pervasive zero counts is problematic, and these methods cannot always control the false discovery rate (FDR). Further, high-throughput microbiome data such as 16S amplicon or metagenomic sequencing are subject to experimental biases that are introduced in every step of the experimental workflow. McLaren et al. [eLife 8, e46923 (2019)] have recently proposed a model for how these biases affect relative abundance data. Motivated by this model, we show that the odds ratios in a logistic regression comparing counts in two taxa are invariant to experimental biases. With this motivation, we propose logistic compositional analysis (LOCOM), a robust logistic regression approach to compositional analysis, that does not require pseudocounts. Inference is based on permutation to account for overdispersion and small sample sizes. Traits can be either binary or continuous, and adjustment for confounders is supported. Our simulations indicate that LOCOM always preserved FDR and had much improved sensitivity over existing methods. In contrast, analysis of composition of microbiomes (ANCOM) and ANCOM with bias correction (ANCOM-BC)/ANOVA-Like Differential Expression tool (ALDEx2) had inflated FDR when the effect sizes were small and large, respectively. Only LOCOM was robust to experimental biases in every situation. The flexibility of our method for a variety of microbiome studies is illustrated by the analysis of data from two microbiome studies. Our R package LOCOM is publicly available.

[Source](https://doi.org/10.1073/pnas.2122788119)

10.1073/pnas.2122788119

---

## A metagenomic DNA sequencing assay that is robust against environmental DNA contamination.
 21.07.2022, _O Mzava, AP Cheng, A Chang, S Smalling, LK Djomnang, JS Lenz, R Longman, A Steadman, LG Gómez-Escobar, EJ Schenck, M Salvatore, MJ Satlin, M Suthanthiran, JR Lee, CE Mason, D Dadhania, I De Vlaminck_


Metagenomic DNA sequencing is a powerful tool to characterize microbial communities but is sensitive to environmental DNA contamination, in particular when applied to samples with low microbial biomass. Here, we present Sample-Intrinsic microbial DNA Found by Tagging and sequencing (SIFT-seq) a metagenomic sequencing assay that is robust against environmental DNA contamination introduced during sample preparation. The core idea of SIFT-seq is to tag the DNA in the sample prior to DNA isolation and library preparation with a label that can be recorded by DNA sequencing. Any contaminating DNA that is introduced in the sample after tagging can then be bioinformatically identified and removed. We applied SIFT-seq to screen for infections from microorganisms with low burden in blood and urine, to identify COVID-19 co-infection, to characterize the urinary microbiome, and to identify microbial DNA signatures of sepsis and inflammatory bowel disease in blood.

[Source](https://doi.org/10.1038/s41467-022-31654-0)

10.1038/s41467-022-31654-0

---

## Bioinformatic Analysis to Investigate Metaproteome Composition Using Trans-Proteomic Pipeline.
 22.07.2022, _S He, S Ranganathan_


With evidence emerging that the microbiome has a role in the onset of many human diseases, including cancer, analyzing these microbial communities and their proteins (i.e., the metaproteome) has become a powerful research tool. The Trans-Proteomic Pipeline (TPP) is a free, comprehensive software suite that facilitates the analysis of mass spectrometry (MS) data. By utilizing available microbial proteomes, TPP can identify microbial proteins and species, with an acceptable peptide false-discovery rate (FDR). An application to a publicly available oral cancer dataset is presented as an example to identify the viral metaproteome on the oral cancer invasive tumor front. © 2022 The Authors. Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: Collection of data and resources Basic Protocol 2: Analysis of MS data using TPP Basic Protocol 3: Analysis of TPP output using R in RStudio.

[Source](https://doi.org/10.1002/cpz1.506)

10.1002/cpz1.506

---

