## https://www.outreachy.org/apply/project-selection/#bioconductor

[Source](https://www.outreachy.org/apply/project-selection/#bioconductor)

---

## Successional action of Bacteroidota and Firmicutes in decomposing straw polymers in a paddy soil.
 14.10.2023, _J Huang, K Gao, L Yang, Y Lu_


Decomposition of plant biomass is vital for carbon cycling in terrestrial ecosystems. In waterlogged soils including paddy fields and natural wetlands, plant biomass degradation generates the largest natural source of global methane emission. However, the intricate process of plant biomass degradation by diverse soil microorganisms remains poorly characterized. Here we report a chemical and metagenomic investigation into the mechanism of straw decomposition in a paddy soil.

[Source](https://twitter.com/hashtag/microbiome?src=hash&ref_src=twsrc%5Etfw)

10.1186/s40793-023-00533-6

---

## #microbiome

[Source](https://twitter.com/hashtag/microbiome?src=hash&ref_src=twsrc%5Etfw)

---

## #bugsigsdb

[Source](https://twitter.com/hashtag/bugsigsdb?src=hash&ref_src=twsrc%5Etfw)

---

## @outreachy

[Source](https://twitter.com/outreachy?ref_src=twsrc%5Etfw)

---

## Extended-spectrum β-lactamase- producing gram-negative bacterial infections in severely ill COVID-19 patients admitted in a national referral hospital, Kenya.
 14.10.2023, _JM Mutua, JM Njeru, AM Musyoki_


Bacterial infections in COVID-19 patients, especially those caused by multidrug-resistant gram-negative strains, are associated with increased morbidity, hospital stay and mortality. However, there is limited data on the epidemiology of extended-spectrum β-lactamase (ESBL)-producing bacteria in COVID-19 patients. Here, we assessed the prevalence and the factors associated with ESBL-producing gram-negative bacterial (GNB) infections among severely ill COVID-19 patients admitted in Kenyatta National Hospital (KNH), Kenya.

[Source](https://twitter.com/svetlana_up/status/1711656036132610217?ref_src=twsrc%5Etfw)

10.1186/s12941-023-00641-8

---

## October 10, 2023

[Source](https://twitter.com/svetlana_up/status/1711656036132610217?ref_src=twsrc%5Etfw)

---

## From hype to hope: considerations in conducting robust microbiome science.
 08.10.2023, _AJ McGuinness, LF Stinson, M Snelson, A Loughman, A Stringer, AJ Hannan, C S M Cowan, HA Jama, JA Caparros-Martin, ML West, HR Wardill_


Microbiome science has been one of the most exciting and rapidly evolving research fields in the past two decades. Breakthroughs in technologies including DNA sequencing have meant that the trillions of microbes (particularly bacteria) inhabiting human biological niches (particularly the gut) can be profiled and analysed in exquisite detail. This microbiome profiling has profound impacts across many fields of research, especially biomedical science, with implications for how we understand and ultimately treat a wide range of human disorders. However, like many great scientific frontiers in human history, the pioneering nature of microbiome research comes with a multitude of challenges and potential pitfalls. These include the reproducibility and robustness of microbiome science, especially in its applications to human health outcomes. In this article, we address the enormous promise of microbiome science and its many challenges, proposing constructive solutions to enhance the reproducibility and robustness of research in this nascent field. The optimisation of microbiome science spans research design, implementation and analysis, and we discuss specific aspects such as the importance of ecological principals and functionality, challenges with microbiome-modulating therapies and the consideration of confounding, alternative options for microbiome sequencing, and the potential of machine learning and computational science to advance the field. The power of microbiome science promises to revolutionise our understanding of many diseases and provide new approaches to prevention, early diagnosis, and treatment.

[Source](https://www.sciencedirect.com/science/article/pii/S0889159123002854)

10.1016/j.bbi.2023.09.022

---

## Unraveling the functional dark matter through global metagenomics

[Source](https://www.nature.com/articles/s41586-023-06583-7)

---

## Integration of 168,000 samples reveals global patterns of the human gut microbiome

[Source](https://www.biorxiv.org/content/10.1101/2023.10.11.560955v1)

---

## Analysis of strain, sex, and diet-dependent modulation of gut microbiota reveals candidate keystone organisms driving microbial diversity in response to American and ketogenic diets.
 13.10.2023, _AC Salvador, MN Huda, D Arends, AM Elsaadi, CA Gacasan, GA Brockmann, W Valdar, BJ Bennett, DW Threadgill_


_Animals, Mice, Diet, Ketogenic, Gastrointestinal Microbiome, Mice, Inbred C57BL, Diet, Bacteroidetes, Carbohydrates_
The gut microbiota is modulated by a combination of diet, host genetics, and sex effects. The magnitude of these effects and interactions among them is important to understanding inter-individual variability in gut microbiota. In a previous study, mouse strain-specific responses to American and ketogenic diets were observed along with several QTLs for metabolic traits. In the current study, we searched for genetic variants underlying differences in the gut microbiota in response to American and ketogenic diets, which are high in fat and vary in carbohydrate composition, between C57BL/6 J (B6) and FVB/NJ (FVB) mouse strains.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01588-w)

10.1186/s40168-023-01588-w

---

## Gut microbial genomes with paired isolates from China signify probiotic and cardiometabolic effects

[Source](https://www.biorxiv.org/content/10.1101/2023.09.25.559444v1)

---

## Gut microbiota profiles in feces and paired tumor and non-tumor tissues from Colorectal Cancer patients. Relationship to the Body Mass Index

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0292551)

---

## Tracking gut microbiome and bloodstream infection in critically ill adults.
 12.10.2023, _CH Gu, LA Khatib, AS Fitzgerald, J Graham-Wooten, CA Ittner, S Sherrill-Mix, Y Chuang, LJ Glaser, NJ Meyer, FD Bushman, RG Collman_


_Humans, Adult, Gastrointestinal Microbiome, Bacteremia, Critical Illness, Sepsis, Bacteria_
The gut microbiome is believed to contribute to bloodstream infection (BSI) via translocation of dominant gut bacteria in vulnerable patient populations. However, conclusively linking gut and blood organisms requires stringent approaches to establish strain-level identity.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0289923)

10.1371/journal.pone.0289923

---

## Human milk oligosaccharides modify the strength of priority effects in the Bifidobacterium community assembly during infancy.
 10.10.2023, _MF Laursen, HM Roager_


Despite the significant role of the gut microbiota in infant health and development, little is known about the ecological processes determining gut microbial community assembly. According to ecology theory, the timing and order of arrival of microbial species into an ecosystem affect microbial community assembly, a phenomenon termed priority effects. Bifidobacterium species are recognized as highly abundant early colonizers of the infant's gut, partly due to their ability to selectively utilize human milk oligosaccharides (HMOs) from breast milk. However, the role of priority effects in Bifidobacterium community assembly remains unclear. Here, we investigated the Bifidobacterium community assembly in the gut of 25 breastfed Danish infants longitudinally sampled throughout the first 6 months of life. Our results showed that the breastfed infants were often initially, but temporarily, dominated by suboptimal HMO-utilizing Bifidobacterium taxa, such as B. longum subsp. longum, before more efficient HMO-utilizers such as B. longum subsp. infantis, replaced the first colonizer as the dominant Bifidobacterium taxon. Subsequently, we validated this observation using gnotobiotic mice sequentially colonized with B. longum subsp. longum and B. longum subsp. infantis or vice versa, with or without supplementation of HMOs in the drinking water. The results showed that in the absence of HMOs, order of arrival determined dominance. Yet, when mice were supplemented with HMOs the strength of priority effects diminished, and B. longum subsp. infantis dominated regardless of colonization order. Our data demonstrate that the arrival order of Bifidobacterium taxa and the deterministic force of breast milk-derived HMOs, dictate Bifidobacterium community assembly in the infant's gut.

[Source](https://www.nature.com/articles/s41396-023-01525-7)

10.1038/s41396-023-01525-7

---

## Metagenome meta-analysis reveals an increase in the abundance of some multidrug efflux pumps and mobile genetic elements in chemically polluted environments.
 20.09.2023, _J Subirats, H Sharpe, V Tai, M Fruci, E Topp_


Many human activities contaminate terrestrial and aquatic environments with numerous chemical pollutants that not only directly alter the environment but also affect microbial communities in ways that are potentially concerning to human health, such as selecting for the spread of antibiotic-resistance genes (ARGs) through horizontal gene transfer. In the present study, metagenomes available in the public domain from polluted (with antibiotics, with petroleum, with metal mining, or with coal-mining effluents) and unpolluted terrestrial and aquatic environments were compared to examine whether pollution has influenced the abundance and composition of ARGs and mobile elements, with specific focus on IS26 and class 1 integrons (intI1). When aggregated together, polluted environments had a greater relative abundance of ARGs than unpolluted environments and a greater relative abundance of IS26 and intI1. In general, chemical pollution, notably with petroleum, was associated with an increase in the prevalence of ARGs linked to multidrug efflux pumps. Included in the suite of efflux pumps were mexK, mexB, mexF, and mexW that are polyspecific and whose substrate ranges include multiple classes of critically important antibiotics. Also, in some instances, β-lactam resistance (TEM181 and OXA-541) genes increased, and genes associated with rifampicin resistance (RNA polymerases subunits rpoB and rpoB2) decreased in relative abundance. This meta-analysis suggests that different types of chemical pollution can enrich populations that carry efflux pump systems associated with resistance to multiple classes of medically critical antibiotics. IMPORTANCE The United Nations has identified chemical pollution as being one of the three greatest threats to environmental health, through which the evolution of antimicrobial resistance, a seminally important public health challenge, may be favored. While this is a very plausible outcome of continued chemical pollution, there is little evidence or research evaluating this risk. The objective of the present study was to examine existing metagenomes from chemically polluted environments and evaluate whether there is evidence that pollution increases the relative abundance of genes and mobile genetic elements that are associated with antibiotic resistance. The key finding is that for some types of pollution, particularly in environments exposed to petroleum, efflux pumps are enriched, and these efflux pumps can confer resistance to multiple classes of medically important antibiotics that are typically associated with Pseudomonas spp. or other Gram-negative bacteria. This finding makes clear the need for more investigation on the impact of chemical pollution on the environmental reservoir of ARGs and their association with mobile genetic elements that can contribute to horizontal gene transfer events.

[Source](https://journals.asm.org/doi/10.1128/aem.01047-23)

10.1128/aem.01047-23

---

## Characterization of Differences of Seed Endophytic Microbiome in Conventional and Organic Rice by Amplicon-based Sequencing and Culturing Methods

[Source](https://www.biorxiv.org/content/10.1101/2023.10.10.561698v1)

---

## Microbiome species diversity and seasonal stability of two temperate marine sponges <em>Hymeniacidon perlevis</em> and <em>Suberites massa</em> 

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-023-00508-7)

---

## An observational study of the presence and variability of the microbiota composition of goat herd milk related to mainstream and artisanal farm management.
 10.10.2023, _RAH Hoving-Bolink, AFG Antonis, MFW Te Pas, D Schokker_


_Animals, Milk, Farms, Bacteria, Microbiota, Lactobacillales, Goats_
Goat milk is produced on mainstream and artisanal farms. It was expected that the farm management may influence the microbial population of the milk. Therefore, we investigated the bacterial content and microbiota composition of raw milk in relation to Dutch goat farm management. After amplicon sequencing we analyzed the taxa at phylum and genus levels, and used the relative values enabling to provide information about the variation among the different samples. On ten farms our results indicated that the number of bacterial colony forming units and microbiota composition of the milk, directly after milking was variable among farms and not related to the farm management system. At the phylum level the phyla Firmicutes, Actinobacteria, Proteobacteria, and to a minor extend Bacteriodota were the dominant phyla in the raw goat milk, together usually comprising 90% of the total bacterial phyla. The most dominant genera were Staphylococcus, Pseudomonas, Lactococcus, Microbacteria, Acinetobacteria, and Corinebacteria. The number of bacterial phyla and genera does not differ between the mainstream and artisanal farms, although the Shannon index may be numerically higher in the mainstream farms as compared to artisanal farms. In addition, the variability is higher among artisanal farms, which may be due to less standardization of the management. The milk microbiota composition differed among farms. Repeated sampling of a farm showed that this changed over time. The lactic acid producing bacteria showed a similar pattern. Variable microbiota richness amount and diversity of microorganisms were present in different farming systems. We concluded that farm-specific management and sampling moment were the major determining factors for the milk microbiota composition.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0292650)

10.1371/journal.pone.0292650

---

## Cultivating efficiency: High-throughput growth analysis of anaerobic bacteria in compact microplate readers

[Source](https://www.biorxiv.org/content/10.1101/2023.10.10.561742v1)

---

## @ElectricBiome

[Source](https://twitter.com/ElectricBiome?ref_src=twsrc%5Etfw)

---

## https://t.co/2wgoi4FaGo

[Source](https://t.co/2wgoi4FaGo)

---

## Effects of rehabilitation therapy based on exercise prescription on motor function and complications after hip fracture surgery in elderly patients.
 14.10.2023, _YJ Che, Z Qian, Q Chen, R Chang, X Xie, YF Hao_


Exercise rehabilitation training is an important measure for improving the prognosis of patients with hip fractures. However, the particular program that works effectively and the efficiency of exercise therapy are still controversial.

[Source](https://twitter.com/Paul_Carini/status/1712070258293665999?ref_src=twsrc%5Etfw)

10.1186/s12891-023-06806-y

---

## October 11, 2023

[Source](https://twitter.com/Paul_Carini/status/1712070258293665999?ref_src=twsrc%5Etfw)

---

## On the relationship between protist metabarcoding and protist metagenome-assembled genomes

[Source](https://www.biorxiv.org/content/10.1101/2023.10.09.561583v1)

---

## Track, Analyze, Visualize: Unravel Your Microbiome’s Temporal Tale with MicrobiomeStat

[Source](https://www.microbiomestat.wiki/)

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/brHZE8qmTQ

[Source](https://t.co/brHZE8qmTQ)

---

## PARTIE: a partition engine to separate metagenomic and amplicon projects in the Sequence Read Archive.
 16.03.2022, _PJ Torres, RA Edwards, KA McNair_


_Bacteria, Humans, Metagenome, Metagenomics, Microbiota, Molecular Sequence Annotation, Sequence Analysis, DNA, Software, Viruses_
The Sequence Read Archive (SRA) contains raw data from many different types of sequence projects. As of 2017, the SRA contained approximately ten petabases of DNA sequence (10 16 bp). Annotations of the data are provided by the submitter, and mining the data in the SRA is complicated by both the amount of data and the detail within those annotations. Here, we introduce PARTIE, a partition engine optimized to differentiate sequence read data into metagenomic (random) and amplicon (targeted) sequence data sets.

[Source](https://twitter.com/linsalrob?ref_src=twsrc%5Etfw)

10.1093/bioinformatics/btx184

---

## @linsalrob

[Source](https://twitter.com/linsalrob?ref_src=twsrc%5Etfw)

---

## @Flinders

[Source](https://twitter.com/Flinders?ref_src=twsrc%5Etfw)

---

## @TurveyLab

[Source](https://twitter.com/TurveyLab?ref_src=twsrc%5Etfw)

---

## @avcarrology

[Source](https://twitter.com/avcarrology?ref_src=twsrc%5Etfw)

---

## @minghao_chia

[Source](https://twitter.com/minghao_chia?ref_src=twsrc%5Etfw)

---

## A Male Case of Kagami-Ogata Syndrome Caused by Paternal Unipaternal Disomy 14 as a Result of a Robertsonian Translocation.
 28.09.2020, _X Wang, H Pang, BA Shah, H Gu, L Zhang, H Wang_


Kagami-Ogata syndrome (KOS) is a rare imprinting disorder characterized by skeletal abnormalities, dysmorphic facial features, growth retardation and developmental delay. The genetic etiology of KOS includes paternal uniparental disomy 14 [upd(14)pat], epimutations and microdeletions affecting the maternally derived imprinted region of chromosome 14q32.2. More than seventy KOS cases have been reported thus far; however, only 10, including two familial, are associated with upd(14)pat harboring Robertsonian translocation (ROB). Here, we reported a male infant with clinical manifestations of facial dysmorphism, bell-shaped small thorax, and omphalocele. Karyotype analyses identify a balanced ROB involving the long arms of chromosomes 13 and 14 both in the patient and his father. We further confirm the pattern of upd(14)pat utilizing DNA polymorphic markers. In conclusion, our case report provides a new male KOS case caused by upd(14)pat with paternally inherited Robertsonian translocation, which represents the second male case officially reported. Notably, a KOS case due to upd(14)pat and ROB is rare. An accurate diagnosis requires not only the identification of the characteristic clinical features but also systemic cytogenetic and molecular studies.

[Source](https://twitter.com/Official_UPD?ref_src=twsrc%5Etfw)

10.3389/fped.2020.00088

---

## @Official_UPD

[Source](https://twitter.com/Official_UPD?ref_src=twsrc%5Etfw)

---

## @NITDgp1955

[Source](https://twitter.com/NITDgp1955?ref_src=twsrc%5Etfw)

---

## pic.twitter.com/KzaL5AGDOq

[Source](https://t.co/KzaL5AGDOq)

---

## October 7, 2023

[Source](https://twitter.com/MicrobiomeVIF/status/1710613202654888399?ref_src=twsrc%5Etfw)

---

