## Microbiome Virtual International Forum

[Source](https://events.ringcentral.com/events/microbiome-vif-n-28)

---

## Longitudinal multi-omics analysis uncovers the altered landscape of gut microbiota and plasma metabolome in response to high altitude.
 08.04.2024, _Y Han, X Liu, Q Jia, J Xu, J Shi, X Li, G Xie, X Zhao, K He_


_Animals, Male, Humans, Gastrointestinal Microbiome, Altitude, Multiomics, Metabolome, East Asian People_
Gut microbiota is significantly influenced by altitude. However, the dynamics of gut microbiota in relation to altitude remains undisclosed.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01781-5)

10.1186/s40168-024-01781-5

---

## Illuminating microflora: shedding light on the potential of blue light to modulate the cutaneous microbiome

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2024.1307374/full)

---

## Evolutionary investigations of the biosynthetic diversity in the skin microbiome using lsaBGC.
 27.05.2023, _R Salamzade, JZA Cheong, S Sandstrom, MH Swaney, RM Stubbendieck, NL Starr, CR Currie, AM Singh, LR Kalan_


_Microbiota, Metagenome, Biological Evolution_
Bacterial secondary metabolites, synthesized by enzymes encoded in biosynthetic gene clusters (BGCs), can underlie microbiome homeostasis and serve as commercialized products, which have historically been mined from a select group of taxa. While evolutionary approaches have proven beneficial for prioritizing BGCs for experimental characterization efforts to uncover new natural products, dedicated bioinformatics tools designed for comparative and evolutionary analysis of BGCs within focal taxa are limited. We thus developed lineage specific analysis of BGCs (lsaBGC; https://github.com/Kalan-Lab/lsaBGC) to aid exploration of microdiversity and evolutionary trends across homologous groupings of BGCs, gene cluster families (GCFs), in any bacterial taxa of interest. lsaBGC enables rapid and direct identification of GCFs in genomes, calculates evolutionary statistics and conservation for BGC genes, and builds a framework to allow for base resolution mining of novel variants through metagenomic exploration. Through application of the suite to four genera commonly found in skin microbiomes, we uncover new insights into the evolution and diversity of their BGCs. We show that the BGC of the virulence-associated carotenoid staphyloxanthin in  Staphylococcus aureus  is ubiquitous across the genus  Staphylococcus . While one GCF encoding the biosynthesis of staphyloxanthin showcases evidence for plasmid-mediated horizontal gene transfer (HGT) between species, another GCF appears to be transmitted vertically amongst a sub-clade of skin-associated  Staphylococcus . Further, the latter GCF, which is well conserved in  S. aureus , has been lost in most  Staphylococcus epidermidis , which is the most common  Staphylococcus  species on human skin and is also regarded as a commensal. We also identify thousands of novel single-nucleotide variants (SNVs) within BGCs from the  Corynebacterium tuberculostearicum  sp. complex, a narrow, multi-species clade that features the most prevalent  Corynebacterium  in healthy skin microbiomes. Although novel SNVs were approximately 10 times as likely to correspond to synonymous changes when located in the top five percentile of conserved sites, lsaBGC identified SNVs that defied this trend and are predicted to underlie amino acid changes within functionally key enzymatic domains. Ultimately, beyond supporting evolutionary investigations of BGCs, lsaBGC also provides important functionalities to aid efforts for the discovery or directed modification of natural products.

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000988)

10.1099/mgen.0.000988

---

## A pan-cancer analysis of the microbiome in metastatic cancer.
 10.04.2024, _TW Battaglia, IL Mimpen, JJH Traets, A van Hoeck, LJ Zeverijn, BS Geurts, GF de Wit, M Noë, I Hofland, JL Vos, S Cornelissen, M Alkemade, A Broeks, CL Zuur, E Cuppen, L Wessels, J van de Haar, E Voest_


Microbial communities are resident to multiple niches of the human body and are important modulators of the host immune system and responses to anticancer therapies. Recent studies have shown that complex microbial communities are present within primary tumors. To investigate the presence and relevance of the microbiome in metastases, we integrated mapping and assembly-based metagenomics, genomics, transcriptomics, and clinical data of 4,160 metastatic tumor biopsies. We identified organ-specific tropisms of microbes, enrichments of anaerobic bacteria in hypoxic tumors, associations between microbial diversity and tumor-infiltrating neutrophils, and the association of Fusobacterium with resistance to immune checkpoint blockade (ICB) in lung cancer. Furthermore, longitudinal tumor sampling revealed temporal evolution of the microbial communities and identified bacteria depleted upon ICB. Together, we generated a pan-cancer resource of the metastatic tumor microbiome that may contribute to advancing treatment strategies.

[Source](https://www.cell.com/cell/pdf/S0092-8674(24)00312-X.pdf)

10.1016/j.cell.2024.03.021

---

## A <em>Lacticaseibacillus rhamnosus</em> secretome induces immunoregulatory transcriptional, functional and immunometabolic signatures in human THP-1 monocytes

[Source](https://www.nature.com/articles/s41598-024-56420-8)

---

## The oral cavity and intestinal microbiome in children with functional constipation.
 11.04.2024, _M Kwiatkowska, M Gołębiewski, M Sikora, EŁ Rycharska, A Krogulska_


_Humans, Child, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Constipation, Microbiota, Feces, Mouth, Bacteria, Lactobacillales, Bacteroidetes_
Constipation is a widespread problem in paediatric practice, affecting almost 30% of children. One of the key causal factors of constipation may be disturbances in the homeostasis of the gastrointestinal microbiome. The aim of the study was to determine whether the oral and fecal microbiomes differ between children with and without constipation. A total of 91 children over three years of age were included in the study. Of these, 57 were qualified to a group with constipation, and 34 to a group without. The saliva and stool microbiomes were evaluated using 16S rRNA gene amplicon sequencing. Functional constipation was associated with characteristic bacterial taxa in the fecal microbiota. Statistically significant differences were found at the family level: Burkholderiaceae (q = 0.047), Christensenellaceae (q = 0.047), Chlostridiaceae (q = 0.047) were significantly less abundant in the constipation group, while the Tannerellaceae (q = 0.007) were more abundant. At the genus level, the significant differences were observed for rare genera, including Christensenellaceae r-7 (q = 2.88 × 10-2), Fusicatenibacter (q = 2.88 × 10-2), Parabacteroides (q = 1.63 × 10-2), Romboutsia (q = 3.19 × 10-2) and Subdoligranulum (q = 1.17 × 10-2). All of them were less abundant in children with constipation. With the exception of significant taxonomic changes affecting only feces, no differences were found in the alpha and beta diversity of feces and saliva. Children with functional constipation demonstrated significant differences in the abundance of specific bacteria in the stool microbiome compared to healthy children. It is possible that the rare genera identified in our study which were less abundant in the constipated patients (Christensellaceae r-7, Fusicatenibacter, Parabacteroides, Romboutsia and Subdoligranulum) may play a role in protection against constipation. No significant differences were observed between the two groups with regard to the saliva microbiome.

[Source](https://www.nature.com/articles/s41598-024-58642-2)

10.1038/s41598-024-58642-2

---

## Characterization of the gut bacterial and viral microbiota in latent autoimmune diabetes in adults.
 11.04.2024, _CS Poulsen, D Hesse, GR Fernandes, TH Hansen, T Kern, A Linneberg, L Van Espen, T Jørgensen, T Nielsen, AC Alibegovic, J Matthijnssens, O Pedersen, H Vestergaard, T Hansen, MK Andersen_


_Adult, Humans, Diabetes Mellitus, Type 1, Diabetes Mellitus, Type 2, Latent Autoimmune Diabetes in Adults, Gastrointestinal Microbiome, Adenosine Deaminase, RNA, Ribosomal, 16S, Intercellular Signaling Peptides and Proteins, Insulin, Glucose Intolerance_
Latent autoimmune diabetes in adults (LADA) is a heterogeneous disease characterized by autoantibodies against insulin producing pancreatic beta cells and initial lack of need for insulin treatment. The aim of the present study was to investigate if individuals with LADA have an altered gut microbiota relative to non-diabetic control subjects, individuals with type 1 diabetes (T1D), and individuals with type 2 diabetes (T2D). Bacterial community profiling was performed with primers targeting the variable region 4 of the 16S rRNA gene and sequenced. Amplicon sequence variants (ASVs) were generated with DADA2 and annotated to the SILVA database. The gut virome was sequenced, using a viral particle enrichment and metagenomics approach, assembled, and quantified to describe the composition of the viral community. Comparison of the bacterial alpha- and beta-diversity measures revealed that the gut bacteriome of individuals with LADA resembled that of individuals with T2D. Yet, specific genera were found to differ in abundance in individuals with LADA compared with T1D and T2D, indicating that LADA has unique taxonomical features. The virome composition reflected the stability of the most dominant order Caudovirales and the families Siphoviridae, Podoviridae, and Inoviridae, and the dominant family Microviridae. Further studies are needed to confirm these findings.

[Source](https://www.nature.com/articles/s41598-024-58985-w)

10.1038/s41598-024-58985-w

---

## Colonization during a key developmental window reveals microbiota-dependent shifts in growth and immunity during undernutrition.
 10.04.2024, _YA Serrano Matos, J Cano, H Shafiq, C Williams, J Sunny, CA Cowardin_


_Child, Infant, Humans, Animals, Mice, Gastrointestinal Microbiome, Malnutrition, Microbiota, Intestine, Small, Growth Disorders_
Childhood undernutrition is a major global health challenge with devastating lifelong consequences. Linear growth stunting due to undernutrition has been linked to poor health outcomes, and mothers who experience growth stunting in childhood are more likely to give birth to stunted children later in life. Based on these findings, we hypothesized that intergenerational colonization of mice with microbiota from human donors with undernutrition may recapitulate certain immune and growth changes observed in this disorder.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01783-3)

10.1186/s40168-024-01783-3

---

## Dermal injury drives a skin to gut axis that disrupts the intestinal microbiome and intestinal immune homeostasis in mice.
 10.04.2024, _T Dokoshi, Y Chen, KJ Cavagnero, G Rahman, D Hakim, S Brinton, H Schwarz, EA Brown, A O'Neill, Y Nakamura, F Li, NH Salzman, R Knight, RL Gallo_


_Mice, Animals, Gastrointestinal Microbiome, Hyaluronic Acid, Colitis, Intestinal Mucosa, Fecal Microbiota Transplantation, Dextran Sulfate, Mice, Inbred C57BL, Disease Models, Animal, Colon_
The composition of the microbial community in the intestine may influence the functions of distant organs such as the brain, lung, and skin. These microbes can promote disease or have beneficial functions, leading to the hypothesis that microbes in the gut explain the co-occurrence of intestinal and skin diseases. Here, we show that the reverse can occur, and that skin directly alters the gut microbiome. Disruption of the dermis by skin wounding or the digestion of dermal hyaluronan results in increased expression in the colon of the host defense genes Reg3 and Muc2, and skin wounding changes the composition and behavior of intestinal bacteria. Enhanced expression Reg3 and Muc2 is induced in vitro by exposure to hyaluronan released by these skin interventions. The change in the colon microbiome after skin wounding is functionally important as these bacteria penetrate the intestinal epithelium and enhance colitis from dextran sodium sulfate (DSS) as seen by the ability to rescue skin associated DSS colitis with oral antibiotics, in germ-free mice, and fecal microbiome transplantation to unwounded mice from mice with skin wounds. These observations provide direct evidence of a skin-gut axis by demonstrating that damage to the skin disrupts homeostasis in intestinal host defense and alters the gut microbiome.

[Source](https://www.nature.com/articles/s41467-024-47072-3)

10.1038/s41467-024-47072-3

---

## A folate inhibitor exploits metabolic differences in <em>Pseudomonas aeruginosa</em> for narrow-spectrum targeting

[Source](https://www.nature.com/articles/s41564-024-01665-2#Abs1)

---

## A genome-centric view of the role of the <em>Acropora kenti</em> microbiome in coral health and resilience

[Source](https://www.nature.com/articles/s41467-024-46905-5)

---

## Probiotics reshape the coral microbiome in situ without detectable off-target effects in the surrounding environment.
 11.04.2024, _N Delgadillo-Ordoñez, N Garcias-Bonet, I Raimundo, FC García, H Villela, EO Osman, EP Santoro, J Curdia, JGD Rosado, P Cardoso, A Alsaggaf, A Barno, CP Antony, C Bocanegra, ML Berumen, CR Voolstra, F Benzoni, S Carvalho, RS Peixoto_


_Animals, Anthozoa, Vibrio, Microbiota, Probiotics, Bacillus_
Beneficial microorganisms for corals (BMCs), or probiotics, can enhance coral resilience against stressors in laboratory trials. However, the ability of probiotics to restructure the coral microbiome in situ is yet to be determined. As a first step to elucidate this, we inoculated putative probiotic bacteria (pBMCs) on healthy colonies of Pocillopora verrucosa in situ in the Red Sea, three times per week, during 3 months. pBMCs significantly influenced the coral microbiome, while bacteria of the surrounding seawater and sediment remained unchanged. The inoculated genera Halomonas, Pseudoalteromonas, and Bacillus were significantly enriched in probiotic-treated corals. Furthermore, the probiotic treatment also correlated with an increase in other beneficial groups (e.g., Ruegeria and Limosilactobacillus), and a decrease in potential coral pathogens, such as Vibrio. As all corals (treated and non-treated) remained healthy throughout the experiment, we could not track health improvements or protection against stress. Our data indicate that healthy, and therefore stable, coral microbiomes can be restructured in situ, although repeated and continuous inoculations may be required in these cases. Further, our study provides supporting evidence that, at the studied scale, pBMCs have no detectable off-target effects on the surrounding microbiomes of seawater and sediment near inoculated corals.

[Source](https://www.nature.com/articles/s42003-024-06135-3)

10.1038/s42003-024-06135-3

---

## Substantial viral diversity in bats and rodents from East Africa: insights into evolution, recombination, and cocirculation.
 10.04.2024, _D Wang, X Yang, Z Ren, B Hu, H Zhao, K Yang, P Shi, Z Zhang, Q Feng, CV Nawenja, V Obanda, K Robert, B Nalikka, CN Waruhiu, GO Ochola, SO Onyuok, H Ochieng, B Li, Y Zhu, H Si, J Yin, K Kristiansen, X Jin, X Xu, M Xiao, B Agwanda, S Ommeh, J Li, ZL Shi_


Zoonotic viruses cause substantial public health and socioeconomic problems worldwide. Understanding how viruses evolve and spread within and among wildlife species is a critical step when aiming for proactive identification of viral threats to prevent future pandemics. Despite the many proposed factors influencing viral diversity, the genomic diversity and structure of viral communities in East Africa are largely unknown.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01782-4)

10.1186/s40168-024-01782-4

---

## Root volatiles manipulate bacterial biofilms.
 09.04.2024, _W Raza, G Jiang_



[Source](https://www.nature.com/articles/s41559-024-02403-x)

10.1038/s41559-024-02403-x

---

## Analysis of endophytic bacterial diversity in seeds of different genotypes of cotton and the suppression of <em>Verticillium wilt </em>pathogen infection by a synthetic microbial community

[Source](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-024-04910-2)

---

## Holomics - a user-friendly R shiny application for multi-omics data integration and analysis.
 07.03.2024, _K Munk, D Ilina, L Ziemba, G Brader, EM Molin_


_Multiomics, Plant Breeding, Computational Biology, Beta vulgaris, Cluster Analysis_
An organism's observable traits, or phenotype, result from intricate interactions among genes, proteins, metabolites and the environment. External factors, such as associated microorganisms, along with biotic and abiotic stressors, can significantly impact this complex biological system, influencing processes like growth, development and productivity. A comprehensive analysis of the entire biological system and its interactions is thus crucial to identify key components that support adaptation to stressors and to discover biomarkers applicable in breeding programs or disease diagnostics. Since the genomics era, several other 'omics' disciplines have emerged, and recent advances in high-throughput technologies have facilitated the generation of additional omics datasets. While traditionally analyzed individually, the last decade has seen an increase in multi-omics data integration and analysis strategies aimed at achieving a holistic understanding of interactions across different biological layers. Despite these advances, the analysis of multi-omics data is still challenging due to their scale, complexity, high dimensionality and multimodality. To address these challenges, a number of analytical tools and strategies have been developed, including clustering and differential equations, which require advanced knowledge in bioinformatics and statistics. Therefore, this study recognizes the need for user-friendly tools by introducing Holomics, an accessible and easy-to-use R shiny application with multi-omics functions tailored for scientists with limited bioinformatics knowledge. Holomics provides a well-defined workflow, starting with the upload and pre-filtering of single-omics data, which are then further refined by single-omics analysis focusing on key features. Subsequently, these reduced datasets are subjected to multi-omics analyses to unveil correlations between 2-n datasets. This paper concludes with a real-world case study where microbiomics, transcriptomics and metabolomics data from previous studies that elucidate factors associated with improved sugar beet storability are integrated using Holomics. The results are discussed in the context of the biological background, underscoring the importance of multi-omics insights. This example not only highlights the versatility of Holomics in handling different types of omics data, but also validates its consistency by reproducing findings from preceding single-omics studies.

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-024-05719-4)

10.1186/s12859-024-05719-4

---

