## gutSMASH

[Source](https://gutsmash.bioinformatics.nl/)

---

## AMPSphere

[Source](https://zenodo.org/record/4574469#.YEI4HhERWV4)

---

##  Green Algorithms calculator

[Source](http://www.green-algorithms.org)

---

## a reproducible research appendix

[Source](https://pratheepaj.github.io/diffTop/)

---

## The Transbiome project

[Source](http://gigasciencejournal.com/blog/transbiome-guest-post/)

---

## here

[Source](https://en.goteo.org/project/transbiome)

---

## A systematic analysis of metabolic pathways in the human gut microbiota

[Source](https://www.biorxiv.org/content/10.1101/2021.02.25.432841v1.full.pdf)

---

## Strain inheritance and neonatal gut microbiota development: A meta-analysis

[Source](https://www.sciencedirect.com/science/article/pii/S1438422121000126)

---

## Adaptive ecological processes and metabolic independence drive microbial colonization and resilience in the human gut

[Source](https://www.biorxiv.org/content/10.1101/2021.03.02.433653v1.full.pdf)

---

## Effect of fecal microbiota transplantation on neurological restoration in a spinal cord injury mouse model: involvement of brain-gut axis

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01007-y)

---

## Association between antibiotics and gut microbiome dysbiosis in children: systematic review and meta-analysis

[Source](https://www.tandfonline.com/doi/epub/10.1080/19490976.2020.1870402?needAccess=true)

---

## Salmonella versus the Microbiome

[Source](https://mmbr.asm.org/content/mmbr/85/1/e00027-19.full.pdf)

---

## Strain-Resolved Dynamics of the Lung Microbiome in Patients with Cystic Fibrosis 

[Source](https://mbio.asm.org/content/mbio/12/2/e02863-20.full.pdf)

---

## Resilience of the oral microbiome

[Source](https://onlinelibrary.wiley.com/doi/epdf/10.1111/prd.12365)

---

## Skin microbiota and clinical associations in Netherton sindrome

[Source](https://www.jidinnovations.org/action/showPdf?pii=S2667-0267%2821%2900008-4)

---

## Long-term stability of the urogenital microbiota of asymptomatic European women

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-021-02123-3)

---

## <em>Limosilactobacillus urinaemulieris </em>sp. nov. and <em>Limosilactobacillus portuensis</em> sp. nov. isolated from urine of healthy women

[Source](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.004726)

---

## The structure and diversity of strain-level variation in vaginal bacteria

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000543)

---

## Obesity is associated with lower bacterial vaginosis prevalence in menopausal but not pre-menopausal women in a retrospective analysis of the Women’s Interagency HIV Study 

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0248136)

---

## Bacterial biofilm formation on vaginal ring pessaries used for pelvic organ prolapse.
 04.03.2021, _FG Gould, MP Carey, EL Plummer, GL Murray, JA Danielewski, SN Tabrizi, SM Garland_


The objective of this study was to characterize the bacterial biofilm on vaginal ring pessaries used to treat pelvic organ prolapse and investigate the relationship between biofilm phenotype and patient symptoms and clinical signs that are suggestive of inflammation.

[Source](https://link.springer.com/article/10.1007/s00192-021-04717-x)

10.1007/s00192-021-04717-x

---

## Long-term storage of feces at −80 °C versus −20 °C is negligible for 16S rRNA amplicon profiling of the equine bacterial microbiome

[Source](https://peerj.com/articles/10837/)

---

## Differences in the gut microbiomes of dogs and wolves: roles of antibiotics and starch.
 12.03.2021, _Y Liu, B Liu, C Liu, Y Hu, C Liu, X Li, X Li, X Zhang, DM Irwin, Z Wu, Z Chen, Q Jin, S Zhang_


Dogs are domesticated wolves. Change of living environment, such as diet and veterinary care may affect the gut bacterial flora of dogs. The aim of this study was to assess the gut bacterial diversity and function in dogs compared with captive wolves. We surveyed the gut bacterial diversity of 27 domestic dogs, which were fed commercial dog food, and 31 wolves, which were fed uncooked meat, by 16S rRNA sequencing. In addition, we collected fecal samples from 5 dogs and 5 wolves for shotgun metagenomic sequencing to explore changes in the functions of their gut microbiome.

[Source](https://bmcvetres.biomedcentral.com/articles/10.1186/s12917-021-02815-y)

10.1186/s12917-021-02815-y

---

## An Expanded Gene Catalog of Mouse Gut Metagenomes.
 25.02.2021, _J Zhu, H Ren, H Zhong, X Li, Y Zou, M Han, M Li, L Madsen, K Kristiansen, L Xiao_


High-quality and comprehensive reference gene catalogs are essential for metagenomic research. The rather low diversity of samples used to construct existing catalogs of the mouse gut metagenome limits the numbers of identified genes in existing catalogs. We therefore established an expanded catalog of genes in the mouse gut metagenome (EMGC) containing &gt;5.8 million genes by integrating 88 newly sequenced samples, 86 mouse gut-related bacterial genomes, and 3 existing gene catalogs. EMGC increases the number of nonredundant genes by more than 1 million genes compared to the so-far most extensive catalog. More than 60% of the genes in EMGC were assigned to Bacteria, with 54.20% being assigned to a phylum and 35.33% to a genus, while 30.39% were annotated at the KEGG orthology level. Nine hundred two metagenomic species (MGS) assigned to 122 taxa are identified based on the EMGC. The EMGC-based analysis of samples from groups of mice originating from different animal providers, housing laboratories, and genetic strains substantiated that diet is a major contributor to differences in composition and functional potential of the gut microbiota irrespective of differences in environment and genetic background. We envisage that EMGC will serve as a valuable reference data set for future metagenomic studies in mice.IMPORTANCE We established an expanded gene catalog of the mouse gut metagenome not only to increase the sample size compared to that in existing catalogs but also to provide a more comprehensive reference data set of the mouse gut microbiome for bioinformatic analysis. The expanded gene catalog comprises more than 5.8 million unique genes, as well as a wide range of taxonomic and functional information. Particularly, the analysis of metagenomic species with the expanded gene catalog reveals a great novelty of mouse gut-inhabiting microbial species. We envisage that the expanded gene catalog of the mouse gut metagenome will serve as a valuable bioinformatic resource for future gut metagenomic studies in mice.

[Source](https://msphere.asm.org/content/msph/6/1/e01119-20.full.pdf)

10.1128/mSphere.01119-20

---

## Long-term evolution and short-term adaptation of microbiota strains and sub-strains in mice.
 04.03.2021, _B Yilmaz, C Mooser, I Keller, H Li, J Zimmermann, L Bosshard, T Fuhrer, M Gomez de Agüero, NF Trigo, H Tschanz-Lischer, JP Limenitakis, WD Hardt, KD McCoy, B Stecher, L Excoffier, U Sauer, SC Ganal-Vonarburg, AJ Macpherson_


Isobiotic mice, with an identical stable microbiota composition, potentially allow models of host-microbial mutualism to be studied over time and between different laboratories. To understand microbiota evolution in these models, we carried out a 6-year experiment in mice colonized with 12 representative taxa. Increased non-synonymous to synonymous mutation rates indicate positive selection in multiple taxa, particularly for genes annotated for nutrient acquisition or replication. Microbial sub-strains that evolved within a single taxon can stably coexist, consistent with niche partitioning of ecotypes in the complex intestinal environment. Dietary shifts trigger rapid transcriptional adaptation to macronutrient and micronutrient changes in individual taxa and alterations in taxa biomass. The proportions of different sub-strains are also rapidly altered after dietary shift. This indicates that microbial taxa within a mouse colony adapt to changes in the intestinal environment by long-term genomic positive selection and short-term effects of transcriptional reprogramming and adjustments in sub-strain proportions.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(21)00047-0)

10.1016/j.chom.2021.02.001

---

## The gut mycobiome of healthy mice is shaped by the environment and correlates with metabolic outcomes in response to diet.
 10.03.2021, _TS Mims, QA Abdallah, JD Stewart, SP Watts, CT White, TV Rousselle, A Gosain, A Bajwa, JC Han, KA Willis, JF Pierre_


As an active interface between the host and their diet, the gut microbiota influences host metabolic adaptation; however, the contributions of fungi have been overlooked. Here, we investigate whether variations in gut mycobiome abundance and composition correlate with key features of host metabolism. We obtained animals from four commercial sources in parallel to test if differing starting mycobiomes can shape host adaptation in response to processed diets. We show that the gut mycobiome of healthy mice is shaped by the environment, including diet, and significantly correlates with metabolic outcomes. We demonstrate that exposure to processed diet leads to persistent differences in fungal communities that significantly associate with differential deposition of body mass in male mice compared to mice fed standardized diet. Fat deposition in the liver, transcriptional adaptation of metabolically active tissues and serum metabolic biomarker levels are linked with alterations in fungal community diversity and composition. Specifically, variation in fungi from the genera Thermomyces and Saccharomyces most strongly associate with metabolic disturbance and weight gain. These data suggest that host-microbe metabolic interactions may be influenced by variability in the mycobiome. This work highlights the potential significance of the gut mycobiome in health and has implications for human and experimental metabolic studies.

[Source](https://www.nature.com/articles/s42003-021-01820-z)

10.1038/s42003-021-01820-z

---

## Functional genetics of human gut commensal <em>Bacteroides thetaiotaomicron</em> reveals metabolic requirements for growth across environments

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247%2821%2900103-0#%20)

---

## Tea polyphenols regulate gut microbiota dysbiosis induced by antibiotic in mice.
 01.03.2021, _J Li, C Chen, H Yang, X Yang_


Tea polyphenols (TPs) are now widely used in foods for various biological activities. However, they are rarely used in foods to regulate gut microbiota dysbiosis induced by antibiotics. We assessed the regulation of TPs on gut microbiota with an antibiotic-induced intestinal flora disorder mouse model. The mice were orally administered with cefixime for 8 days, then received TPs for 28 days. We found that the antibiotic had a profound impact on the gut microbiota. Compared with the normal group, significant decreases in the species richness and diversity and the production of short-chain fatty acids (SCFAs) were still observed 28 days after the antibiotic treatment, although there was no significant difference in the colonic mucosa. TPs significantly alleviated the decrease of the richness and diversity of gut microbiota caused by the antibiotic treatment, and significantly increased the relative abundance of beneficial microbes such as Lactobacillus, Akkermansia, Blautia, Roseburia, and Eubacterium. The function prediction showed that TPs significantly decreased the relative abundance of genes related to human diseases, yet significantly increased the relative abundance of genes related to cell growth and death, cell motility, and energy metabolism. These showed that TPs could regulate the gut microbiota dysbiosis induced by antibiotics, thus decreasing the risk of diseases such as obesity, cancer, and diabetes. These suggest that TPs have a great potential to be used as a functional food ingredient to prevent or reduce adverse effects of antibiotics.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0963996921000508)

10.1016/j.foodres.2021.110153

---

## Designing a home for beneficial plant microbiomes.
 08.03.2021, _C Song, K Jin, JM Raaijmakers_


The plant microbiome comprises a highly diverse community of saprotrophic, mutualistic, and pathogenic microbes that can affect plant growth and plant health. There is substantial interest to exploit beneficial members of plant microbiomes for new sustainable management strategies in crop production. However, poor survival and colonization of plant tissues by introduced microbial isolates as well as lack of expression of the plant growth-promoting or disease-suppressive traits at the right time and place are still major limitations for successful implementation of microbiomes in future agricultural practices and plant breeding programs. Similar to building a home for humans, we discuss different strategies of building a home for beneficial plant microbiomes, here referred to as the 'MicrobiHome'.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S136952662100025X)

10.1016/j.pbi.2021.102025

---

## Polymicrobial interaction between <em>Lactobacillus</em> and <em>Saccharomyces cerevisiae</em>: coexistence-relevant mechanisms

[Source](https://www.tandfonline.com/doi/abs/10.1080/1040841X.2021.1893265)

---

## Viromes outperform total metagenomes in revealing the spatiotemporal patterns of agricultural soil viral communities.
 22.02.2021, _C Santos-Medellin, LA Zinke, AM Ter Horst, DL Gelardi, SJ Parikh, JB Emerson_


Viruses are abundant yet understudied members of soil environments that influence terrestrial biogeochemical cycles. Here, we characterized the dsDNA viral diversity in biochar-amended agricultural soils at the preplanting and harvesting stages of a tomato growing season via paired total metagenomes and viral size fraction metagenomes (viromes). Size fractionation prior to DNA extraction reduced sources of nonviral DNA in viromes, enabling the recovery of a vaster richness of viral populations (vOTUs), greater viral taxonomic diversity, broader range of predicted hosts, and better access to the rare virosphere, relative to total metagenomes, which tended to recover only the most persistent and abundant vOTUs. Of 2961 detected vOTUs, 2684 were recovered exclusively from viromes, while only three were recovered from total metagenomes alone. Both viral and microbial communities differed significantly over time, suggesting a coupled response to rhizosphere recruitment processes and/or nitrogen amendments. Viral communities alone were also structured along an 18 m spatial gradient. Overall, our results highlight the utility of soil viromics and reveal similarities between viral and microbial community dynamics throughout the tomato growing season yet suggest a partial decoupling of the processes driving their spatial distributions, potentially due to differences in dispersal, decay rates, and/or sensitivities to soil heterogeneity.

[Source](https://www.nature.com/articles/s41396-021-00897-y)

10.1038/s41396-021-00897-y

---

## Biases in Viral Metagenomics-Based Detection, Cataloguing and Quantification of Bacteriophage Genomes in Human Faeces, a Review

[Source](https://www.mdpi.com/2076-2607/9/3/524)

---

## High molecular weight DNA extraction strategies for long-read sequencing of complex metagenomes

[Source](https://www.biorxiv.org/content/10.1101/2021.03.03.433801v1.full.pdf)

---

## AMPSphere: the worldwide survey of prokaryotic antimicrobial peptides

[Source](https://zenodo.org/record/4574469#.YEBJOBERWV4)

---

## A Statistical Perspective on the Challenges in Molecular Microbial Biology

[Source](https://arxiv.org/abs/2103.04198)

---

## The carbon footprint of bioinformatics

[Source](https://www.biorxiv.org/content/10.1101/2021.03.08.434372v1.full.pdf)

---

## Make Interactive Complex Heatmaps in R 

[Source](https://www.biorxiv.org/content/10.1101/2021.03.08.434289v2.full.pdf)

---

## Machine learning applications in microbial ecology, human microbiome studies, and environmental monitoring 

[Source](https://www.sciencedirect.com/science/article/pii/S2001037021000325)

---

## Tutorial: assessing metagenomics software with the CAMI benchmarking toolkit

[Source](https://www.nature.com/articles/s41596-020-00480-3)

---

## STATegra: Multi-Omics Data Integration – A Conceptual Scheme With a Bioinformatics Pipeline

[Source](https://www.frontiersin.org/articles/10.3389/fgene.2021.620453/full)

---

## Exploring bacterial diversity via a curated and searchable snapshot of archived DNA sequences

[Source](https://www.biorxiv.org/content/10.1101/2021.03.02.433662v1.full.pdf)

---

## clinker &amp; clustermap.js: automatic generation of gene cluster comparison figures

[Source](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btab007/6103786)

---

## gapseq: informed prediction of bacterial metabolic pathways and reconstruction of accurate metabolic models

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02295-1)

---

## mSystems Thinking Series: Diversity, Equity, and Inclusion in Microbiology 

[Source](https://asmusa-org.zoom.us/webinar/register/WN_9ZIMp_o6QUapQpZWDnmTaA)

---

## The Microbiome Centers Consortium Seminar Series 

[Source](https://docs.google.com/document/d/1FZWTdsocs9wGpbABF4KOjq6CfIWqtF18RI8_r7gWYfI/edit)

---

## Americas Webinar: Mother’s Milk &amp; Baby’s Bacteria: Discoveries from the CHILD Cohort Study

[Source](https://www.isrhml.com/i4a/ams/meetings/index.cfm?controller=meetings&action=startRegistration&conferenceID=94&reginit=1&pageID=3436)

---

## Get R Done: R Tutorials video series

[Source](https://www.youtube.com/playlist?list=PLmNgrNF3pZqg2GPmcIAWDa1A2-cAfbX37)

---

## Machine Learning in Genomics: Tools, Resources, Clinical Applications and Ethics

[Source](https://www.genome.gov/event-calendar/Machine-Learning-in-Genomics-Tools-Resources-Clinical-Applications-and-Ethics)

---

## February 2021 Pod: Global Chang‪e‬

[Source](https://podcasts.apple.com/se/podcast/february-2021-pod-global-change/id1509550298?i=1000511103116)

---

## Enhancers with Molly Gasperini 

[Source](https://bioinformatics.chat/enhancers)

---

## Matthew Schechter: Metagenomics History Part 1 and 2

[Source](https://anchor.fm/bioinfopod/episodes/Matthew-Schechter-Metagenomics-History-Part-1-eppqeh)

---

## Ep. 33 Oyster Microbiome with Dr. Rachel Diner 

[Source](https://planetb612.fm/ep-33-oyster-microbiome-with-dr-rachel-diner/)

---

## UMich Microbial Masterpieces 

[Source](https://sites.google.com/umich.edu/microbialmasterpieces)

---

