## Evolution of the Gut Microbiome in HIV-Exposed Uninfected and Unexposed Infants during the First Year of Life.
 08.09.2022, _CL Jackson, DN Frank, CE Robertson, D Ir, JM Kofonow, MP Montlha, EAML Mutsaerts, MC Nunes, SA Madhi, D Ghosh, A Weinberg_


HIV-exposed uninfected infants (HEU) have abnormal immunologic functions and increased infectious morbidity in the first 6 months of life, which gradually decreases thereafter. The mechanisms underlying HEU immune dysfunctions are unknown. We hypothesized that unique characteristics of the HEU gut microbiota associated with maternal HIV status may underlie the HEU immunologic dysfunctions. We characterized the infant gut, maternal gut, and breast milk microbiomes of mother-infant pairs, including 123 with HEU and 117 with HIV-uninfected infants (HUU), from South Africa. Pan-bacterial 16S rRNA gene sequencing was performed on (i) infant stool at 6, 28, and 62 weeks; (ii) maternal stool at delivery and 62 weeks; and (iii) breast milk at 6 weeks. Infant gut alpha and beta diversities were similar between groups. Microbial composition significantly differed, including 12 genera, 5 families and 1 phylum at 6 weeks; 12 genera and 2 families at 28 weeks; and 2 genera and 2 families at 62 weeks of life. Maternal gut microbiomes significantly differed in beta diversity and microbial composition, and breast milk microbiomes differed in microbial composition only. Infant gut microbiotas extensively overlapped with maternal gut and minimally with breast milk microbiotas. Nevertheless, exclusively breastfed HEU and HUU had less divergent microbiomes than nonexclusively breastfed infants. Feeding pattern and maternal gut microbiome imprint the HEU gut microbiome. Compared to HUU, the HEU gut microbiome prominently differs in early infancy, including increased abundance of taxa previously observed to be present in excess in adults with HIV. The HEU and HUU gut microbiome compositions converge over time, mirroring the kinetics of HEU infectious morbidity risk. IMPORTANCE HIV-exposed uninfected infants (HEU) are highly vulnerable to infections in the first 6 months of life, and this vulnerability decreases to the age of 24 months. Because the microbiome plays a critical role in the education of the infant immune system, which protects them against infections, we characterized the gut microbiomes of HEU and HIV-unexposed infants (HUU) in the first year of life. The HEU and HUU gut microbiomes showed prominent differences at 6 and 28 weeks of life but converged at 62 weeks of life, mirroring the time course of the HEU excess infectious morbidity and suggesting a potential association between the infant gut microbiome structure and susceptibility to infections. Infant gut microbiotas extensively overlapped with maternal gut and minimally with breast milk microbiotas. Moreover, exclusively breastfed HEU and HUU had less divergent microbiomes at 6 and 28 weeks than nonexclusively breastfed HEU and HUU. The factors that affect the HEU gut microbiome, maternal gut microbiome and exclusive breastfeeding, may be targeted by interventions.

[Source](https://doi.org/10.1128/mbio.01229-22)

10.1128/mbio.01229-22

---

## Gastrointestinal microbiota profile and clinical correlations in advanced EGFR-WT and EGFR-mutant non-small cell lung cancer.
 08.09.2022, _W Saifon, I Sensorn, N Trachu, S Oranratnachai, A Charoenyingwattana, C Runcharoen, N Monnamo, W Sukkasem, P Inchareon, T Suwatanapongched, P Chansriwong, T Ativitavas, R Panvichian, W Chantratita, T Reungwetwattana_


Difference in clinical responses to cancer therapy in each patient is from several factors. Gastrointestinal microbiota is one of the reasons. However, this correlation remains unknown. This study aims to explore correlation between gastrointestinal microbiota profile and clinical outcomes in Thai advanced non-small cell lung cancer (NSCLC) according to epidermal growth factor receptor (EGFR) status.

[Source](https://doi.org/10.1186/s12885-022-10050-3)

10.1186/s12885-022-10050-3

---

## Hybrid metagenome assemblies link carbohydrate structure with function in the human gut microbiome.
 08.09.2022, _A Ravi, P Troncoso-Rey, J Ahn-Jarvis, KR Corbin, S Harris, H Harris, A Aydin, GL Kay, T Le Viet, R Gilroy, MJ Pallen, AJ Page, J O'Grady, FJ Warren_


Complex carbohydrates that escape small intestinal digestion, are broken down in the large intestine by enzymes encoded by the gut microbiome. This is a symbiotic relationship between microbes and host, resulting in metabolic products that influence host health and are exploited by other microbes. However, the role of carbohydrate structure in directing microbiota community composition and the succession of carbohydrate-degrading microbes, is not fully understood. In this study we evaluate species-level compositional variation within a single microbiome in response to six structurally distinct carbohydrates in a controlled model gut using hybrid metagenome assemblies. We identified 509 high-quality metagenome-assembled genomes (MAGs) belonging to ten bacterial classes and 28 bacterial families. Bacterial species identified as carrying genes encoding starch binding modules increased in abundance in response to starches. The use of hybrid metagenomics has allowed identification of several uncultured species with the functional potential to degrade starch substrates for future study.

[Source](https://doi.org/10.1038/s42003-022-03865-0)

10.1038/s42003-022-03865-0

---

## Genetic Determinants of Hydrogen Sulfide Biosynthesis in <em>Fusobacterium nucleatum</em> Are Required for Bacterial Fitness, Antibiotic Sensitivity, and Virulence

[Source](https://doi.org/10.1128/mbio.01936-22)

---

## Genomic and ecologic characteristics of the airway microbial-mucosal complex

[Source](https://doi.org/10.1101/2022.09.08.507073)

---

## <em>Lactiplantibacillus plantarum</em> Regulated Intestinal Microbial Community and Cytokines to Inhibit <em>Salmonella typhimurium </em>Infection

[Source](https://doi.org/10.1007/s12602-022-09987-5)

---

## Exploration of rumen microbial and carbohydrate-active enzyme profiles in cattle fed coir a lignin-rich diet using a metagenomic approach.
 08.09.2022, _M Khatoon, SJ Jakhesara, DN Rank, CG Joshi, AP Kunjadiya_


Lignocellulosic biomass is a rich source of feed for cattle. Amongst them, coconut coir may be the potential source of feed supplements. To assess, the effect of various concentrations of coconut coir (0%, 21% and 40 %) as a feed supplement on the rumen microbiome of cattle (Kankrej breed), a metagenomic (16S rRNA gene amplicon and shotgun sequencing) study was performed. The Alpha diversity estimation from the amplicon study suggested that the group of cattle fed food without the coconut coir has a higher number of genera than the cattle fed with mixed ration. Within the liquid fraction, bacterial phyla Bacteroidetes were abundant followed by Firmicutes and Fibrobacteres, whereas the proportion of Tenericutes, TM7, SRI, Verrucomicrobia, Lentisphaerae, and Elusimicrobia had decreased with the rise in the coir concentration. While within the solid fractions, the proportion of Elusimicrobia increased, but the count of Bacteriodetes, Firmicutes, Fibrobacteres Tenericutes, TM7, SRI, Verrucomicrobia, and Lentisphaerae decreased with an increase in coir percentages. The results obtained from shotgun sequencing show similar results for bacterial diversity. The functions associated with carbohydrate metabolism were abundant in both the treatments as compared to the control. Functions related to glycoside hydrolases, glycosyltransferases and carbohydrate-binding modules were abundant in both the treatments as compared to control. Thus, the study indicates that the microbiome does alter after feeding coir as a supplement and may be used as feed for cattle.

[Source](https://doi.org/10.1016/j.gene.2022.146868)

10.1016/j.gene.2022.146868

---

## The core bacterial microbiome of banana (<em>Musa</em> spp.)

[Source](https://doi.org/10.1186/s40793-022-00442-0)

---

## Higher pH is associated with enhanced co-occurrence network complexity, stability and nutrient cycling functions in the rice rhizosphere microbiome.
 08.09.2022, _Y Guo, B Song, A Li, Q Wu, H Huang, N Li, Y Yang, JM Adams, L Yang_


The rice rhizosphere microbiota is crucial for crop yields and nutrient use efficiency. However, little is known about how co-occurrence patterns, keystone taxa and functional gene assemblages relate to soil pH in the rice rhizosphere soils. Using shotgun metagenome analysis, the rice rhizosphere microbiome was investigated across 28 rice fields in east-central China. At higher pH sites, the taxonomic co-occurrence network of rhizosphere soils was more complex and compact, as defined by higher average degree, graph density and complexity. Network stability was greatest at medium pH (6.5 &lt; pH &lt; 7.5), followed by high pH (7.5 &lt; pH). Keystone taxa were more abundant at higher pH and correlated significantly with key ecosystem functions. Overall functional genes involved in C, N, P and S cycling were at a higher relative abundance in higher pH rhizosphere soils, excepting C degradation genes (e.g., key genes involved in starch-, cellulose-, chitin- and lignin- degradation). Our results suggest that the rice rhizosphere soil microbial network is more complex and stable at higher pH, possibly indicating increased efficiency of nutrient cycling. These observations may indicate routes towards more efficient soil management and understanding of the potential effects of soil acidification on the rice rhizosphere system.

[Source](https://doi.org/10.1111/1462-2920.16185)

10.1111/1462-2920.16185

---

## Virome analysis in lakes of the South Shetland Islands, Antarctica - 2020.
 08.09.2022, _T Prado, ML Brandão, TM Fumian, L Freitas, M Chame, L Leomil, MGP Magalhães, WMS Degrave, JPG Leite, MP Miagostovich_


Polar freshwater ecosystems are characterized by a distinct microbiota. However, little is known about viral diversity and abundance, especially regarding the ecology of RNA viruses. We used shotgun metagenomic analysis on samples from Antarctic ecosystems, and report here the characterization of the virome fraction, from different lakes located in the South Shetland Islands (Penguin, Ardley, Deception and King George Island) in the Peninsula Antarctica, in the summer season 2020. DNA viruses (99.4 %) prevailed over RNA viruses (0.6 %) in the lake samples. Six viral orders were identified in the metagenomic libraries: Caudovirales (dsDNA), which was prevalent in most lakes; Picornavirales (ssRNA+); Sobelivirales (ssRNA+); Tolivirales (ssRNA+); Petitvirales (ssDNA) and Baphyvirales (ssDNA), including eight viral families (Herelleviridae, Siphoviridae, Myoviridae, Microviridae, Marnaviridae, Bacilladnaviridae, Barnaviridae and Tombusviridae) and several other, mainly non-classified ssRNA(+) viruses in the lakes of Ardley Island. Bacteriophages (dsDNA) (Herelleviridae family) infecting the phylum Firmicutes and Siphoviridae were predominant in most lakes evaluated. Functional analysis demonstrated a prevalence of unknown proteins (68 %) in the virome. Our prospective study provides virome analysis data from different lakes in the South Shetland Islands, Antarctica, opening exploratory lines for future research related to the biodiversity and viral ecology in this extreme ecosystem.

[Source](https://doi.org/10.1016/j.scitotenv.2022.158537)

10.1016/j.scitotenv.2022.158537

---

## Investigating differential abundance methods in microbiome data: A benchmark study.
 08.09.2022, _M Cappellato, G Baruzzo, B Di Camillo_


The development of increasingly efficient and cost-effective high throughput DNA sequencing techniques has enhanced the possibility of studying complex microbial systems. Recently, researchers have shown great interest in studying the microorganisms that characterise different ecological niches. Differential abundance analysis aims to find the differences in the abundance of each taxa between two classes of subjects or samples, assigning a significance value to each comparison. Several bioinformatic methods have been specifically developed, taking into account the challenges of microbiome data, such as sparsity, the different sequencing depth constraint between samples and compositionality. Differential abundance analysis has led to important conclusions in different fields, from health to the environment. However, the lack of a known biological truth makes it difficult to validate the results obtained. In this work we exploit metaSPARSim, a microbial sequencing count data simulator, to simulate data with differential abundance features between experimental groups. We perform a complete comparison of recently developed and established methods on a common benchmark with great effort to the reliability of both the simulated scenarios and the evaluation metrics. The performance overview includes the investigation of numerous scenarios, studying the effect on methods' results on the main covariates such as sample size, percentage of differentially abundant features, sequencing depth, feature variability, normalisation approach and ecological niches. Mainly, we find that methods show a good control of the type I error and, generally, also of the false discovery rate at high sample size, while recall seem to depend on the dataset and sample size.

[Source](https://doi.org/10.1371/journal.pcbi.1010467)

10.1371/journal.pcbi.1010467

---

## A rapid and standardized workflow for functional assessment of bacterial biosensors in fecal samples.
 08.09.2022, _A Zúñiga, G Muñoz-Guamuro, L Boivineau, P Mayonove, I Conejero, GP Pageaux, R Altwegg, J Bonnet_


Gut metabolites are pivotal mediators of host-microbiome interactions and provide an important window on human physiology and disease. However, current methods to monitor gut metabolites rely on heavy and expensive technologies such as liquid chromatography-mass spectrometry (LC-MS). In that context, robust, fast, field-deployable, and cost-effective strategies for monitoring fecal metabolites would support large-scale functional studies and routine monitoring of metabolites biomarkers associated with pathological conditions. Living cells are an attractive option to engineer biosensors due to their ability to detect and process many environmental signals and their self-replicating nature. Here we optimized a workflow for feces processing that supports metabolite detection using bacterial biosensors. We show that simple centrifugation and filtration steps remove host microbes and support reproducible preparation of a physiological-derived media retaining important characteristics of human feces, such as matrix effects and endogenous metabolites. We measure the performance of bacterial biosensors for benzoate, lactate, anhydrotetracycline, and bile acids, and find that they are highly sensitive to fecal matrices. However, encapsulating the bacteria in hydrogel helps reduce this inhibitory effect. Sensitivity to matrix effects is biosensor-dependent but also varies between individuals, highlighting the need for case-by-case optimization for biosensors' operation in feces. Finally, by detecting endogenous bile acids, we demonstrate that bacterial biosensors could be used for future metabolite monitoring in feces. This work lays the foundation for the optimization and use of bacterial biosensors for fecal metabolites monitoring. In the future, our method could also allow rapid pre-prototyping of engineered bacteria designed to operate in the gut, with applications to in situ diagnostics and therapeutics.

[Source](https://doi.org/10.3389/fbioe.2022.859600)

10.3389/fbioe.2022.859600

---

