## A benchmark of algorithms for the analysis of pooled CRISPR screens.
 20.03.2020, _S Bodapati, TP Daley, X Lin, J Zou, LS Qi_


Genome-wide pooled CRISPR-Cas-mediated knockout, activation, and repression screens are powerful tools for functional genomic investigations. Despite their increasing importance, there is currently little guidance on how to design and analyze CRISPR-pooled screens. Here, we provide a review of the commonly used algorithms in the computational analysis of pooled CRISPR screens. We develop a comprehensive simulation framework to benchmark and compare the performance of these algorithms using both synthetic and real datasets. Our findings inform parameter choices of CRISPR screens and provide guidance to researchers on the design and analysis of pooled CRISPR screens.

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01972-x)

10.1186/s13059-020-01972-x

---

## Evaluation of liquid and powdered forms of polyclonal antibody preparation against Streptococcus bovis and Fusobacterium necrophorum in cattle adapted or not adapted to highly fermentable carbohydrate diets.
 16.04.2020, _ECO Cassiano, FP Júnior, TA Barros, CT Marino, RDL Pacheco, FA Ferreira, DD Millen, MF Martins, SMP Pugine, MP de Melo, K Beauchemin, PM Meyer, MB Arrigoni, PHM Rodrigues_


Feed additives that modify rumen fermentation can be used to prevent metabolic disturbances such as acidosis and optimize beef cattle production. The study evaluated the effects of liquid and powdered forms of polyclonal antibody preparation (PAP) against Streptococcus bovis and Fusobacterium necrophorum on rumen fermentation parameters in ruminally cannulated non-lactating dairy cows that were adapted or unadapted to a high concentrate diet.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231533#sec001)

10.5713/ajas.19.0761

---

## Comparison of the ruminal and fecal microbiotas in beef calves supplemented or not with concentrate

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231533#sec001)

---

## The trisaccharide melezitose impacts honey bees and their intestinal microbiota.
 15.04.2020, _VC Seeburger, P D'Alvise, B Shaaban, K Schweikert, G Lohaus, A Schroeder, M Hasselmann_


In general, honey bees (Apis mellifera L.) feed on honey produced from collected nectar. In the absence of nectar, during certain times of the year or in monocultural landscapes, honey bees forage on honeydew. Honeydew is excreted by different herbivores of the order Hemiptera that consume phloem sap of plant species. In comparison to nectar, honeydew is composed of a higher variety of sugars and additional sugars with higher molecular weight, like the trisaccharide melezitose that can be a major constituent of honeydew. However, melezitose-containing honey is known to cause malnutrition in overwintering honey bees. Following the hypothesis that melezitose may be the cause for the so called 'honeydew flow disease', three independent feeding experiments with caged bees were conducted in consecutive years. Bees fed with melezitose showed increased food uptake, higher gut weights and elevated mortality compared to bees fed a control diet. Moreover, severe disease symptoms, such as swollen abdomen, abdomen tipping and impaired movement were observed in melezitose-fed bees. 16S-amplicon sequencing indicated that the melezitose diet changed the species composition of the lactic acid bacteria community within the gut microbiota. Based on these results, we conclude that melezitose cannot be easily digested by the host and may accumulate in the hindgut. Within cages or during winter, when there is no opportunity for excretion, the accumulated melezitose can cause severe intestinal symptoms and death of the bees, probably as result of poor melezitose metabolism capabilities in the intestinal microbiota. These findings confirm the causal relation between the trisaccharide melezitose and the honeydew flow disease and indicate a possible mechanism of pathogenesis.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0230871)

10.1371/journal.pone.0230871

---

## Fecal shedding of Salmonella spp., Clostridium perfringens, and Clostridioides difficile in dogs fed raw meat-based diets in Brazil and their owners' motivation.
 14.04.2020, _FM Viegas, CP Ramos, RGC Xavier, EO Lopes, CAO Júnior, RM Bagno, AN Diniz, FCF Lobato, ROS Silva_


The present study aimed to explore the motivations of Brazilian dog owners and their knowledge about the risks related to raw meat-based diets (RMBD) as well as to evaluate important enteropathogens such as Salmonella spp., C. perfringens, and C. difficile, in feces of dogs fed different diets. The majority of the pet owners (69.3%) reported to have chosen this diet for their dogs, considering it to be more "natural". A large number of owners declared that RMBD do not pose health risks for their animals (87.9%) or humans (98.8%), even though almost one third of the respondents (34.8%) declared having at least one individual at high risk of infection in contact with RMBD-fed dogs. Stool samples from 46 RMBD-fed dogs and 192 dogs fed commercial dry feed were collected. The present study revealed that dogs fed raw meat diets were almost 30 times more likely to be positive for Salmonella spp. than dogs on a conventional diet. Some of the serovars detected were commonly associated with human salmonellosis, such as S. Typhimurium and S. Saintpaul, and were multidrug resistant. RMBD-fed dogs were more likely to be positive for C. perfringens type A (p = 0.008) and one C. perfringens type F was isolated from these animals. Two toxigenic strains (4.3%) of C. difficile were isolated only from raw meat-fed dogs, all of which were under antibiotic therapy. These toxigenic C. difficile isolates were classified as RT106/ST54 and RT600/ST149, previously associated with infection in dogs and humans. The present work revealed that the owners have a tendency to ignore or are unaware of the risks associated with raw meat diets for dogs. Also, the higher fecal shedding of important enteropathogens in dogs fed RMBD suggests that this diet poses a risk for the animals and the people in contact with them.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231275)

10.1371/journal.pone.0231275

---

## Mechanistic molecular responses of the giant clam Tridacna crocea to Vibrio coralliilyticus challenge.
 15.04.2020, _D Xu, Z Zhao, Z Zhou, Y Lin, X Zhang, Y Zhang, Y Zhang, J Li, F Mao, S Xiao, H Ma, X Zhiming, Z Yu_


Vibrio coralliilyticus is a pathogen of coral and mollusk, contributing to dramatic losses worldwide. In our study, we found that V. coralliilyticus challenge could directly affect adult Tridacna crocea survival; there were dead individuals appearing at 6 h post infection, and there were 45.56% and 56.78% mortality rates in challenged groups after 36 h of infection. The apoptosis rate of hemocytes was significantly increased by 1.8-fold at 6 h after V. coralliilyticus injection. To shed light on the mechanistic molecular responses of T. crocea to V. coralliilyticus infection, we used transcriptome sequencing analysis and other relevant techniques to analyze T. crocea hemocytes at 0 h, 6 h, 12 h and 24 h after V. coralliilyticus challenge. Our results revealed that the total numbers of unigenes and DEGs were 195651 and 3446, respectively. Additional details were found by KEGG pathway enrichment analysis, where DEGs were significantly enriched in immune-related signaling pathways, such as the TLR signaling pathway, and some were associated with signaling related to apoptosis. Quantitative validation results illustrated that with exposure to V. coralliilyticus, the expression of TLR pathway members, TLR, MyD88, IRAK4, TRAF6, and IкB-α, were significantly upregulated (by 22.9-, 9.6-, 4.0-, 3.6-, and 3.9-fold, respectively) at 6 h. The cytokine-related gene IL-17 exhibited an increase of 6.3-fold and 10.5-fold at 3 h and 6 h, respectively. The apoptosis-related gene IAP1 was dramatically increased by 2.99-fold at 6 h. These results indicate that adult T. crocea could initiate the TLR pathway to resist V. coralliilyticus, which promotes the release of inflammatory factors such as IL-17 and leads to the activation of a series of outcomes, such as apoptosis. The response mechanism is related to the T. crocea immunoreaction stimulated by V. coralliilyticus, providing a theoretical basis for understanding T. crocea immune response mechanisms.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231399)

10.1371/journal.pone.0231399

---

## Metagenomic analysis of <em>Aedes aegypti</em> and <em>Culex quinquefasciatus</em> mosquitoes from Grenada, West Indies

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231047)

---

## Detection of novel coronaviruses in bats in Myanmar.
 15.04.2020, _MT Valitutto, O Aung, KYN Tun, ME Vodzak, D Zimmerman, JH Yu, YT Win, MT Maw, WZ Thein, HH Win, J Dhanota, V Ontiveros, B Smith, A Tremeau-Brevard, T Goldstein, CK Johnson, S Murray, J Mazet_


_Anal Canal, Animals, Chiroptera, Coronavirus, Feces, Mouth, Myanmar, Population Surveillance_
The recent emergence of bat-borne zoonotic viruses warrants vigilant surveillance in their natural hosts. Of particular concern is the family of coronaviruses, which includes the causative agents of severe acute respiratory syndrome (SARS), Middle East respiratory syndrome (MERS), and most recently, Coronavirus Disease 2019 (COVID-19), an epidemic of acute respiratory illness originating from Wuhan, China in December 2019. Viral detection, discovery, and surveillance activities were undertaken in Myanmar to identify viruses in animals at high risk contact interfaces with people. Free-ranging bats were captured, and rectal and oral swabs and guano samples collected for coronaviral screening using broadly reactive consensus conventional polymerase chain reaction. Sequences from positives were compared to known coronaviruses. Three novel alphacoronaviruses, three novel betacoronaviruses, and one known alphacoronavirus previously identified in other southeast Asian countries were detected for the first time in bats in Myanmar. Ongoing land use change remains a prominent driver of zoonotic disease emergence in Myanmar, bringing humans into ever closer contact with wildlife, and justifying continued surveillance and vigilance at broad scales.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0230802)

10.1371/journal.pone.0230802

---

## Plant microbiome analysis after Metarhizium amendment reveals increases in abundance of plant growth-promoting organisms and maintenance of disease-suppressive soil.
 15.04.2020, _L Barelli, AS Waller, SW Behie, MJ Bidochka_


The microbial community in the plant rhizosphere is vital to plant productivity and disease resistance. Alterations in the composition and diversity of species within this community could be detrimental if microbes suppressing the activity of pathogens are removed. Species of the insect-pathogenic fungus, Metarhizium, commonly employed as biological control agents against crop pests, have recently been identified as plant root colonizers and provide a variety of benefits (e.g. growth promotion, drought resistance, nitrogen acquisition). However, the impact of Metarhizium amendment on the rhizosphere microbiome has yet to be elucidated. Using Illumina sequencing, we examined the community profiles (bacteria and fungi) of common bean (Phaseolus vulgaris) rhizosphere (loose soil and plant root) after amendment with M. robertsii conidia, in the presence and absence of an insect host. Although alpha diversity was not significantly affected overall, there were numerous examples of plant growth-promoting organisms that significantly increased with Metarhizium amendment (Bradyrhizobium, Flavobacterium, Chaetomium, Trichoderma). Specifically, the abundance of Bradyrhizobium, a group of nitrogen-fixing bacteria, was confirmed to be increased using a qPCR assay with genus-specific primers. In addition, the ability of the microbiome to suppress the activity of a known bean root pathogen was assessed. The development of disease symptoms after application with Fusarium solani f. sp. phaseoli was visible in the hypocotyl and upper root of plants grown in sterilized soil but was suppressed during growth in microbiome soil and soil treated with M. robertsii. Successful amendment of agricultural soils with biocontrol agents such as Metarhizium necessitates a comprehensive understanding of the effects on the diversity of the rhizosphere microbiome. Such research is fundamentally important towards sustainable agricultural practices to improve overall plant health and productivity.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231150)

10.1371/journal.pone.0231150

---

## The genome evolution and domestication of tropical fruit mango.
 20.03.2020, _P Wang, Y Luo, J Huang, S Gao, G Zhu, Z Dang, J Gai, M Yang, M Zhu, H Zhang, X Ye, A Gao, X Tan, S Wang, S Wu, EB Cahoon, B Bai, Z Zhao, Q Li, J Wei, H Chen, R Luo, D Gong, K Tang, B Zhang, Z Ni, G Huang, S Hu, Y Chen_


Mango is one of the world's most important tropical fruits. It belongs to the family Anacardiaceae, which includes several other economically important species, notably cashew, sumac and pistachio from other genera. Many species in this family produce family-specific urushiols and related phenols, which can induce contact dermatitis.

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01959-8)

10.1186/s13059-020-01959-8

---

## Environmental control on the distribution of metabolic strategies of benthic microbial mats in Lake Fryxell, Antarctica

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231053)

---

## Experimentally-validated correlation analysis reveals new anaerobic methane oxidation partnerships with consortium-level heterogeneity in diazotrophy

[Source](https://www.biorxiv.org/content/10.1101/2020.04.12.038331v1?%3Fcollection=)

---

## A comparative analysis of drinking water employing metagenomics.
 15.04.2020, _KD Brumfield, NA Hasan, MB Leddy, JA Cotruvo, SM Rashed, RR Colwell, A Huq_


The microbiological content of drinking water traditionally is determined by employing culture-dependent methods that are unable to detect all microorganisms, especially those that are not culturable. High-throughput sequencing now makes it possible to determine the microbiome of drinking water. Thus, the natural microbiota of water and water distribution systems can now be determined more accurately and analyzed in significantly greater detail, providing comprehensive understanding of the microbial community of drinking water applicable to public health. In this study, shotgun metagenomic analysis was performed to determine the microbiological content of drinking water and to provide a preliminary assessment of tap, drinking fountain, sparkling natural mineral, and non-mineral bottled water. Predominant bacterial species detected were members of the phyla Actinobacteria and Proteobacteria, notably the genera Alishewanella, Salmonella, and Propionibacterium in non-carbonated non-mineral bottled water, Methyloversatilis and Methylibium in sparkling natural mineral water, and Mycobacterium and Afipia in tap and drinking fountain water. Fecal indicator bacteria, i.e., Escherichia coli or enterococci, were not detected in any samples examined in this study. Bacteriophages and DNA encoding a few virulence-associated factors were detected but determined to be present only at low abundance. Antibiotic resistance markers were detected only at abundance values below our threshold of confidence. DNA of opportunistic plant and animal pathogens was identified in some samples and these included bacteria (Mycobacterium spp.), protozoa (Acanthamoeba mauritaniensis and Acanthamoeba palestinensis), and fungi (Melampsora pinitorqua and Chryosporium queenslandicum). Archaeal DNA (Candidatus Nitrosoarchaeum) was detected only in sparkling natural mineral water. This preliminary study reports the complete microbiome (bacteria, viruses, fungi, and protists) of selected types of drinking water employing whole-genome high-throughput sequencing and bioinformatics. Investigation into activity and function of the organisms detected is in progress.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231210)

10.1371/journal.pone.0231210

---

## Prevalence and pollution characteristics of antibiotic resistant genes in one high anthropogenically-impacted river.
 15.04.2020, _Q Li, Q Zhang_


The objectives of this study were to comprehensively investigate the occurrence, distribution, and mobility of antibiotic resistant genes (ARGs) in the biofilm, water, and sediment from a section of the Weihe-river, in the northern Henan province, China. The abundances of nine ARGs belonging to four commonly used antibiotic classes (tetracyclines, sulfonamides, fluoroquinolones, and multidrug) and class 1 integron-integrase gene (intI1) were quantified. Sulfonamides gene (sulI) accounted for the highest percentage of detected ARGs in most sampling sites, including in water, biofilm, and sediment. Among the resistance genes, IntI1 and sul1 were significantly correlated (r&gt;0.800, p&lt;0.01) with a fecal coliform (FC) detected in the biofilm, and there was also a significantly positive correlation between the abundances of 16SrRNA and intI1 in the biofilms. Compared with the sediment and water samples, the biofilms contained sufficient nutrients to promote bacterial reproduction. Under sufficient total nitrogen and phosphorus concentrations, the horizontal gene transfer due to intI1 plays a key role in the formation and migration of ARGs within biofilms.

[Source](https://journals.plos.org/plosone/article/comments?id=10.1371/journal.pone.0231128)

10.1371/journal.pone.0231128

---

## Longitudinal survey of microbiome associated with particulate matter in a megacity

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01964-x)

---

## Ozone efficacy for the control of airborne viruses: Bacteriophage and norovirus models.
 15.04.2020, _ME Dubuis, N Dumont-Leblond, C Laliberté, M Veillette, N Turgeon, J Jean, C Duchaine_


This study was designed to test the efficacy of an air treatment using ozone and relative humidity (RH) for the inactivation of airborne viruses. Four phages (φX174, PR772, MS2 and φ6) and one eukaryotic virus (murine norovirus MNV-1) were exposed to low ozone concentrations (1.23 ppm for phages and 0.23 ppm for MNV-1) and various levels of RH for 10 to 70 minutes. The inactivation of these viruses was then assessed to determine which of the tested conditions provided the greatest reduction in virus infectivity. An inactivation of at least two orders of magnitude for φX174, MS2 and MNV-1 was achieved with an ozone exposure of 40 minutes at 85% RH. For PR772 and φ6, exposure to the reference condition at 20% RH for 10 minutes yielded the same results. These findings suggest that ozone used at a low concentration is a powerful disinfectant for airborne viruses when combined with a high RH. Air treatment could therefore be implemented inside hospital rooms ventilated naturally.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231164)

10.1371/journal.pone.0231164

---

## Decode-seq: a practical approach to improve differential gene expression analysis.
 26.03.2020, _Y Li, H Yang, H Zhang, Y Liu, H Shang, H Zhao, T Zhang, Q Tu_


Many differential gene expression analyses are conducted with an inadequate number of biological replicates. We describe an easy and effective RNA-seq approach using molecular barcoding to enable profiling of a large number of replicates simultaneously. This approach significantly improves the performance of differential gene expression analysis. Using this approach in medaka (Oryzias latipes), we discover novel genes with sexually dimorphic expression and genes necessary for germ cell development. Our results also demonstrate why the common practice of using only three replicates in differential gene expression analysis should be abandoned.

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01966-9)

10.1186/s13059-020-01966-9

---

## WhatsGNU: a tool for identifying proteomic novelty.
 21.03.2020, _AM Moustafa, PJ Planet_


To understand diversity in enormous collections of genome sequences, we need computationally scalable tools that can quickly contextualize individual genomes based on their similarities and identify features of each genome that make them unique. We present WhatsGNU, a tool based on exact match proteomic compression that, in seconds, classifies any new genome and provides a detailed report of protein alleles that may have novel functional differences. We use this technique to characterize the total allelic diversity (panallelome) of Salmonella enterica, Mycobacterium tuberculosis, Pseudomonas aeruginosa, and Staphylococcus aureus. It could be extended to others. WhatsGNU is available from https://github.com/ahmedmagds/WhatsGNU.

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01965-w)

10.1186/s13059-020-01965-w

---

## A Zipf-plot based normalization method for high-throughput RNA-seq data.
 11.04.2020, _B Wang_


Normalization is crucial in RNA-seq data analyses. Due to the existence of excessive zeros and a large number of small measures, it is challenging to find reliable linear rescaling normalization parameters. We propose a Zipf plot based normalization method (ZN) assuming that all gene profiles have similar upper tail behaviors in their expression distributions. The new normalization method uses global information of all genes in the same profile without gene-level expression alteration. It doesn't require the majority of genes to be not differentially expressed (DE), and can be applied to data where the majority of genes are weakly or not expressed. Two normalization schemes are implemented with ZN: a linear rescaling scheme and a non-linear transformation scheme. The linear rescaling scheme can be applied alone or together with the non-linear normalization scheme. The performance of ZN is benchmarked against five popular linear normalization methods for RNA-seq data. Results show that the linear rescaling normalization scheme by itself works well and is robust. The non-linear normalization scheme can further improve the normalization outcomes and is optional if the Zipf plots show parallel patterns.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0230594)

10.1371/journal.pone.0230594

---

