## Intestinal commensal microbiota and cytokines regulate Fut2+ Paneth cells for gut defense.
 14.01.2022, _M Kamioka, Y Goto, K Nakamura, Y Yokoi, R Sugimoto, S Ohira, Y Kurashima, S Umemoto, S Sato, J Kunisawa, Y Takahashi, SE Domino, JC Renauld, S Nakae, Y Iwakura, PB Ernst, T Ayabe, H Kiyono_


Paneth cells are intestinal epithelial cells that release antimicrobial peptides, such as α-defensin as part of host defense. Together with mesenchymal cells, Paneth cells provide niche factors for epithelial stem cell homeostasis. Here, we report two subtypes of murine Paneth cells, differentiated by their production and utilization of fucosyltransferase 2 (Fut2), which regulates α(1,2)fucosylation to create cohabitation niches for commensal bacteria and prevent invasion of the intestine by pathogenic bacteria. The majority of Fut2- Paneth cells were localized in the duodenum, whereas the majority of Fut2+ Paneth cells were in the ileum. Fut2+ Paneth cells showed higher granularity and structural complexity than did Fut2- Paneth cells, suggesting that Fut2+ Paneth cells are involved in host defense. Signaling by the commensal bacteria, together with interleukin 22 (IL-22), induced the development of Fut2+ Paneth cells. IL-22 was found to affect the α-defensin secretion system via modulation of Fut2 expression, and IL-17a was found to increase the production of α-defensin in the intestinal tract. Thus, these intestinal cytokines regulate the development and function of Fut2+ Paneth cells as part of gut defense.

[Source](https://doi.org/10.1073/pnas.2115230119)

10.1073/pnas.2115230119

---

## Gut microbiota promotes cholesterol gallstone formation by modulating bile acid composition and biliary cholesterol secretion.
 12.01.2022, _H Hu, W Shao, Q Liu, N Liu, Q Wang, J Xu, X Zhang, Z Weng, Q Lu, L Jiao, C Chen, H Sun, Z Jiang, X Zhang, A Gu_


Cholesterol gallstone disease is a worldwide common disease. Cholesterol supersaturation in gallbladder bile is the prerequisite for its pathogenesis, while the mechanism is not completely understood. In this study, we find enrichment of gut microbiota (especially Desulfovibrionales) in patients with gallstone disease. Fecal transplantation of gut microbiota from gallstone patients to gallstone-resistant strain of mice can induce gallstone formation. Carrying Desulfovibrionales is associated with enhanced cecal secondary bile acids production and increase of bile acid hydrophobicity facilitating intestinal cholesterol absorption. Meanwhile, the metabolic product of Desulfovibrionales, H2S increase and is shown to induce hepatic FXR and inhibit CYP7A1 expression. Mice carrying Desulfovibrionales present induction of hepatic expression of cholesterol transporters Abcg5/g8 to promote biliary secretion of cholesterol as well. Our study demonstrates the role of gut microbiota, Desulfovibrionales, as an environmental regulator contributing to gallstone formation through its influence on bile acid and cholesterol metabolism.

[Source](https://doi.org/10.1038/s41467-021-27758-8)

10.1038/s41467-021-27758-8

---

## Non-typeable <em>Haemophilus influenzae</em>–<em>Moraxella catarrhalis</em> vaccine for the prevention of exacerbations in chronic obstructive pulmonary disease: a multicentre, randomised, placebo-controlled, observer-blinded, proof-of-concept, phase 2b trial

[Source](https://doi.org/10.1016/S2213-2600(21)00502-6)

---

## PET microplastics affect human gut microbiota communities during simulated gastrointestinal digestion, first evidence of plausible polymer biodegradation during human digestion.
 15.01.2022, _A Tamargo, N Molinero, JJ Reinosa, V Alcolea-Rodriguez, R Portela, MA Bañares, JF Fernández, MV Moreno-Arribas_


Microplastics (MPs) are a widely recognized global problem due to their prevalence in natural environments and the food chain. However, the impact of microplastics on human microbiota and their possible biotransformation in the gastrointestinal tract have not been well reported. To evaluate the potential risks of microplastics at the digestive level, completely passing a single dose of polyethylene terephthalate (PET) through the gastrointestinal tract was simulated by combining a harmonized static model and the dynamic gastrointestinal simgi model, which recreates the different regions of the digestive tract in physiological conditions. PET MPs started several biotransformations in the gastrointestinal tract and, at the colon, appeared to be structurally different from the original particles. We report that the feeding with microplastics alters human microbial colonic community composition and hypothesize that some members of the colonic microbiota could adhere to MPs surface promoting the formation of biofilms. The work presented here indicates that microplastics are indeed capable of digestive-level health effects. Considering this evidence and the increasing exposure to microplastics in consumer foods and beverages, the impact of plastics on the functionality of the gut microbiome and their potential biodegradation through digestion and intestinal bacteria merits critical investigation.

[Source](https://doi.org/10.1038/s41598-021-04489-w)

10.1038/s41598-021-04489-w

---

## Ear microbiota and middle ear disease: a longitudinal pilot study of Aboriginal children in a remote south Australian setting

[Source](https://doi.org/10.1186/s12866-022-02436-x)

---

## Comparative analysis of DNA extraction and PCR product purification methods for cervicovaginal microbiome analysis using cpn60 microbial profiling.
 13.01.2022, _E Shvartsman, MEI Richmond, JJ Schellenberg, A Lamont, C Perciani, JNH Russell, V Poliquin, A Burgener, W Jaoko, P Sandstrom, KS MacDonald_


The microbiota of the lower female genital tract plays an important role in women's health. Microbial profiling using the chaperonin60 (cpn60) universal target (UT) improves resolution of vaginal species associated with negative health outcomes compared to the more commonly used 16S ribosomal DNA target. However, the choice of DNA extraction and PCR product purification methods may bias sequencing-based microbial studies and should be optimized for the sample type and molecular target used. In this study, we compared two commercial DNA extraction kits and two commercial PCR product purification kits for the microbial profiling of cervicovaginal samples using the cpn60 UT.

[Source](https://doi.org/10.1371/journal.pone.0262355)

10.1371/journal.pone.0262355

---

## Microbial hydrogen economy alleviates colitis by reprogramming colonocyte metabolism and reinforcing intestinal barrier.
 15.01.2022, _L Ge, J Qi, B Shao, Z Ruan, Y Ren, S Sui, X Wu, X Sun, S Liu, S Li, C Xu, W Song_


With the rapid development and high therapeutic efficiency and biosafety of gas-involving theranostics, hydrogen medicine has been particularly outstanding because hydrogen gas (H2), a microbial-derived gas, has potent anti-oxidative, anti-apoptotic, and anti-inflammatory activities in many disease models. Studies have suggested that H2-enriched saline/water alleviates colitis in murine models; however, the underlying mechanism remains poorly understood. Despite evidence demonstrating the importance of the microbial hydrogen economy, which reflects the balance between H2-producing (hydrogenogenic) and H2-utilizing (hydrogenotrophic) microbes in maintaining colonic mucosal ecosystems, minimal efforts have been exerted to manipulate relevant H2-microbe interactions for colonic health. Consistent with previous studies, we found that administration of hydrogen-rich saline (HS) ameliorated dextran sulfate sodium-induced acute colitis in a mouse model. Furthermore, we demonstrated that HS administration can increase the abundance of intestinal-specific short-chain fatty acid (SCFA)-producing bacteria and SCFA production, thereby activating the intracellular butyrate sensor peroxisome proliferator-activated receptor γ signaling and decreasing the epithelial expression of Nos2, consequently promoting the recovery of the colonic anaerobic environment. Our results also indicated that HS administration ameliorated disrupted intestinal barrier functions by modulating specific mucosa-associated mucolytic bacteria, leading to substantial inhibition of opportunistic pathogenic Escherichia coli expansion as well as a significant increase in the expression of interepithelial tight junction proteins and a decrease in intestinal barrier permeability in mice with colitis. Exogenous H2 reprograms colonocyte metabolism by regulating the H2-gut microbiota-SCFAs axis and strengthens the intestinal barrier by modulating specific mucosa-associated mucolytic bacteria, wherein improved microbial hydrogen economy alleviates colitis.

[Source](https://doi.org/10.1080/19490976.2021.2013764)

10.1080/19490976.2021.2013764

---

## Single-gene long-read sequencing illuminates <em>Escherichia coli</em> strain dynamics in the human intestinal microbiome

[Source](https://doi.org/10.1016/j.celrep.2021.110239)

---

## Draft Genome Sequence of <em>Bifidobacterium adolescentis</em> 4-2, Isolated from Healthy Human Feces

[Source](https://doi.org/10.1128/mra.00846-21)

---

## Diet, microbe, and autism: Cause or consequence?

[Source](https://doi.org/10.1016/j.chom.2021.12.018)

---

## Gut microbiota in patients with Alzheimer’s disease spectrum: a systematic review and meta-analysis

[Source](https://doi.org/10.18632/aging.203826)

---

## Characteristics of the gut microbiome in women with gestational diabetes mellitus: A systematic review

[Source](https://doi.org/10.1371/journal.pone.0262618)

---

## Respiratory tract infections and gut microbiome modifications: A systematic review

[Source](https://doi.org/10.1371/journal.pone.0262057)

---

## Repercussions of intermittent fasting on the intestinal microbiota community and body composition: a systematic review

[Source](https://doi.org/10.1093/nutrit/nuab108)

---

## Skin Deep: The Role of the Microbiota in Cutaneous Autoimmunity

[Source](https://doi.org/10.1016/j.jid.2021.12.005)

---

## Current advances in the anti-inflammatory effects and mechanisms of natural polysaccharides

[Source](https://doi.org/10.1080/10408398.2022.2025535)

---

## The gut microbiota of environmentally enriched mice regulates visual cortical plasticity

[Source](https://doi.org/10.1016/j.celrep.2021.110212)

---

## Characterization of the juvenile koala gut microbiome across wild populations

[Source](https://doi.org/10.1111/1462-2920.15884)

---

## Specific gut bacterial responses to natural diets of tropical birds

[Source](https://doi.org/10.1038/s41598-022-04808-9)

---

## Effect of dietary supplementation of lauric acid on growth performance, antioxidative capacity, intestinal development and gut microbiota on black sea bream (<em>Acanthopagrus schlegelii</em>)

[Source](https://doi.org/10.1371/journal.pone.0262427)

---

## Establishment and resilience of transplanted gut microbiota in aged mice

[Source](https://doi.org/10.1016/j.isci.2021.103654)

---

## Identification of Gut Bacteria such as <em>Lactobacillus johnsonii</em> that Disseminate to Systemic Tissues of Wild Type and MyD88-/- Mice

[Source](https://doi.org/10.1080/19490976.2021.2007743)

---

## Responsive changes of rumen microbiome and metabolome in dairy cows with different susceptibility to subacute ruminal acidosis.
 15.01.2022, _T Zhang, Y Mu, R Zhang, Y Xue, C Guo, W Qi, J Zhang, S Mao_


Subacute ruminal acidosis (SARA) represents one of the most important digestive disorders in intensive dairy farms, and dairy cows are individually different in the severity of SARA risk. The objectives of the current study were to investigate differences in the ruminal bacterial community and metabolome in dairy cattle with different susceptibility to SARA. In the present study, 12 cows were initially enrolled in the experiment. Based on average ruminal pH, 4 cows with the lowest ruminal pH were assigned to the susceptible group (SUS, pH = 5.76, n = 4) and 4 cows with the highest ruminal pH assigned to the tolerant group (TOL, pH = 6.10, n = 4). Rumen contents from susceptible (SUS, n = 4) and tolerant (TOL, n = 4) dairy cows were collected through rumen fistula to systematically reveal the rumen microbial and metabolic alterations of dairy cows with different susceptibility to SARA using multi-omics approaches (16S and 18S rRNA gene sequencing and metabolome). The results showed that despite being fed the same diet, SUS cows had lower ruminal pH and higher concentrations of total volatile fatty acids (VFA) and propionate than TOL cows (P &lt; 0.05). No significant differences were observed in dry matter intake, milk yield, and other milk compositions between the SUS and TOL groups (P &gt; 0.05). The principal coordinates analysis based on the analysis of molecular variance indicated a significant difference in bacterial composition between the two groups (P = 0.01). More specifically, the relative abundance of starch-degrading bacteria (Prevotella spp.) was greater (P &lt; 0.05), while the proportion of fiber-degrading bacteria (unclassified Ruminococcaceae spp., Ruminococcus spp., Papillibacter, and unclassified Family_XIII) was lower in the rumen of SUS cows compared with TOL cows (P &lt; 0.05). Community analysis of protozoa showed that there were no significant differences in the diversity, richness, and community structure (P &gt; 0.05). Metabolomics analysis revealed that the concentrations of organic acids (such as lactic acid), biogenic amines (such as histamine), and bacterial degradation products (such as hypoxanthine) were significantly higher in the SUS group compared to the TOL group (P &lt; 0.05). These findings revealed that the higher proportion of starch-degrading bacteria/lower fiber-degrading bacteria in the rumen of SUS cows resulted in higher VFA-producing capacity, in particular propionate. This caused a disruption in metabolic homeostasis in the rumen which might be the reason for the higher susceptibility to SARA. Overall, these findings enhanced our understanding of the ruminal microbiome and metabolic changes in cows susceptible to SARA.

[Source](https://doi.org/10.1016/j.aninu.2021.10.009)

10.1016/j.aninu.2021.10.009

---

## Torula yeast in the diet of Atlantic salmon Salmo salar and the impact on growth performance and gut microbiome.
 13.01.2022, _A Leeper, R Ekmay, S Knobloch, S Skírnisdóttir, M Varunjikar, M Dubois, BÖ Smárason, J Árnason, W Koppe, D Benhaïm_


Atlantic salmon aquaculture is expanding, and with it, the need to find suitable replacements for conventional protein sources used in formulated feeds. Torula yeast (Cyberlindnera jadinii), has been identified as a promising alternative protein for feed and can be sustainably cultivated on lignocellulosic biomasses. The present study investigated the impact of torula yeast on the growth performance and gut microbiome of freshwater Atlantic salmon. A marine protein base diet and a mixed marine and plant protein base diet were tested, where conventional proteins were replaced with increasing inclusion levels of torula yeast, (0%, 10%, 20%). This study demonstrated that 20% torula yeast can replace fish meal without alteration to growth performance while leading to potential benefits for the gut microbiome by increasing the presence of bacteria positively associated with the host. However, when torula yeast replaced plant meal in a mixed protein diet, results suggested that 10% inclusion of yeast produced the best growth performance results but at the 20% inclusion level of yeast, potentially negative changes were observed in the gut microbial community, such as a decrease in lactic acid bacteria. This study supports the continued investigation of torula yeast for Atlantic salmon as a partial replacement for conventional proteins.

[Source](https://doi.org/10.1038/s41598-021-04413-2)

10.1038/s41598-021-04413-2

---

## Distinct gut microbiomes in two polar bear subpopulations inhabiting different sea ice ecoregions.
 15.01.2022, _M Franz, L Whyte, TC Atwood, KL Laidre, D Roy, SE Watson, E Góngora, MA McKinney_


Gut microbiomes were analyzed by 16S rRNA gene metabarcoding for polar bears (Ursus maritimus) from the southern Beaufort Sea (SB), where sea ice loss has led to increased use of land-based food resources by bears, and from East Greenland (EG), where persistent sea ice has allowed hunting of ice-associated prey nearly year-round. SB polar bears showed a higher number of total (940 vs. 742) and unique (387 vs. 189) amplicon sequence variants and higher inter-individual variation compared to EG polar bears. Gut microbiome composition differed significantly between the two subpopulations and among sex/age classes, likely driven by diet variation and ontogenetic shifts in the gut microbiome. Dietary tracer analysis using fatty acid signatures for SB polar bears showed that diet explained more intrapopulation variation in gut microbiome composition and diversity than other tested variables, i.e., sex/age class, body condition, and capture year. Substantial differences in the SB gut microbiome relative to EG polar bears, and associations between SB gut microbiome and diet, suggest that the shifting foraging habits of SB polar bears tied to sea ice loss may be altering their gut microbiome, with potential consequences for nutrition and physiology.

[Source](https://doi.org/10.1038/s41598-021-04340-2)

10.1038/s41598-021-04340-2

---

## Microbiome Resilience despite a Profound Loss of Minority Microbiota following Clindamycin Challenge in Humanized Gnotobiotic Mice.
 12.01.2022, _L Liu, ME Kirst, L Zhao, E Li, GP Wang_


Antibiotics are known to induce gut dysbiosis and increase the risk of antibiotic resistance. While antibiotic exposure is a known risk factor leading to compromised colonization resistance against enteric pathogens such as Clostridioides difficile, the extent and consequences of antibiotic perturbation on the human gut microbiome remain poorly understood. Human studies on impacts of antibiotics are complicated by the tremendous variability of gut microbiome among individuals, even between identical twins. Furthermore, antibiotic challenge experiments cannot be replicated in human subjects for a given gut microbiome. Here, we transplanted feces from three unrelated human donors into groups of identical germfree (GF) Swiss-Webster mice, and examined the temporal responses of the transplanted microbiome to oral clindamycin challenge in gnotobiotic isolators over 7 weeks. Analysis of 177 longitudinal fecal samples revealed that 59% to 81% of human microbiota established a stable configuration rapidly and stably in GF mice. Microbiome responses to clindamycin challenge was highly reproducible and microbiome-dependent. A short course of clindamycin was sufficient to induce a profound loss (∼one third) of the microbiota by disproportionally eliminating minority members of the transplanted microbiota. However, it was inadequate to disrupt the global microbial community structure or function, which rebounded rapidly to resemble its pre-treatment state after clindamycin discontinuation. Furthermore, the response of individual microbes was community-dependent. Taken together, these results suggest that the overall gut microbiome structure is resilient to antibiotic perturbation, the functional consequences of which warrant further investigation. IMPORTANCE Antibiotics cause imbalance of gut microbiota, which in turn increase our susceptibility to gastrointestinal infections. However, how antibiotics disrupt gut bacterial communities is not well understood, and exposing healthy volunteers to unnecessary antibiotics for research purposes carries clinical and ethical concerns. In this study, we used genetically identical mice transplanted with the same human gut microbiota to control for both genetic and environmental variables. We found that a short course of oral clindamycin was sufficient to eliminate one third of the gut bacteria by disproportionally eliminating minority members of the transplanted microbiota, but it was inadequate to disrupt the overall microbial community structure and function, which rebounded rapidly to its pre-treatment state. These results suggest that gut microbiome is highly resilient to antibiotic challenge and degradation of the human gut ecosystem may require repeated or prolonged antibiotic exposure.

[Source](https://doi.org/10.1128/spectrum.01960-21)

10.1128/spectrum.01960-21

---

## Bovine respiratory microbiota of feedlot cattle and its association with disease.
 13.01.2022, _J Chai, SF Capik, B Kegley, JT Richeson, JG Powell, J Zhao_


Bovine respiratory disease (BRD), as one of the most common and costly diseases in the beef cattle industry, has significant adverse impacts on global food security and the economic stability of the industry. The bovine respiratory microbiome is strongly associated with health and disease and may provide insights for alternative therapy when treating BRD. The niche-specific microbiome communities that colonize the inter-surface of the upper and the lower respiratory tract consist of a dynamic and complex ecological system. The correlation between the disequilibrium in the respiratory ecosystem and BRD has become a hot research topic. Hence, we summarize the pathogenesis and clinical signs of BRD and the alteration of the respiratory microbiota. Current research techniques and the biogeography of the microbiome in the healthy respiratory tract are also reviewed. We discuss the process of resident microbiota and pathogen colonization as well as the host immune response. Although associations between the microbiota and BRD have been revealed to some extent, interpreting the development of BRD in relation to respiratory microbial dysbiosis will likely be the direction for upcoming studies, which will allow us to better understand the importance of the airway microbiome and its contributions to animal health and performance.

[Source](https://doi.org/10.1186/s13567-021-01020-x)

10.1186/s13567-021-01020-x

---

## Plant neighborhood shapes diversity and reduces interspecific variation of the phyllosphere microbiome.
 13.01.2022, _KM Meyer, R Porch, IE Muscettola, ALS Vasconcelos, JK Sherman, CJE Metcalf, SE Lindow, B Koskella_


Microbial communities associated with plant leaf surfaces (i.e., the phyllosphere) are increasingly recognized for their role in plant health. While accumulating evidence suggests a role for host filtering of its microbiota, far less is known about how community composition is shaped by dispersal, including from neighboring plants. We experimentally manipulated the local plant neighborhood within which tomato, pepper, or bean plants were grown in a 3-month field trial. Focal plants were grown in the presence of con- or hetero-specific neighbors (or no neighbors) in a fully factorial combination. At 30-day intervals, focal plants were harvested and replaced with a new age- and species-matched cohort while allowing neighborhood plants to continue growing. Bacterial community profiling revealed that the strength of host filtering effects (i.e., interspecific differences in composition) decreased over time. In contrast, the strength of neighborhood effects increased over time, suggesting dispersal from neighboring plants becomes more important as neighboring plant biomass increases. We next implemented a cross-inoculation study in the greenhouse using inoculum generated from the field plants to directly test host filtering of microbiomes while controlling for directionality and source of dispersal. This experiment further demonstrated that focal host species, the host from which the microbiome came, and in one case the donor hosts' neighbors, contribute to variation in phyllosphere bacterial composition. Overall, our results suggest that local dispersal is a key factor in phyllosphere assembly, and that demographic factors such as nearby neighbor identity and biomass or age are important determinants of phyllosphere microbiome diversity.

[Source](https://doi.org/10.1038/s41396-021-01184-6)

10.1038/s41396-021-01184-6

---

## Sulfate-dependant microbially induced corrosion of mild steel in the deep sea: a 10-year microbiome study.
 14.01.2022, _P Rajala, DQ Cheng, SA Rice, FM Lauro_


Metal corrosion in seawater has been extensively studied in surface and shallow waters. However, infrastructure is increasingly being installed in deep-sea environments, where extremes of temperature, salinity, and high hydrostatic pressure increase the costs and logistical challenges associated with monitoring corrosion. Moreover, there is currently only a rudimentary understanding of the role of microbially induced corrosion, which has rarely been studied in the deep-sea. We report here an integrative study of the biofilms growing on the surface of corroding mooring chain links that had been deployed for 10 years at ~2 km depth and developed a model of microbially induced corrosion based on flux-balance analysis.

[Source](https://doi.org/10.1186/s40168-021-01196-6)

10.1186/s40168-021-01196-6

---

## Microbial community composition and glyphosate degraders of two soils under the influence of temperature, total organic carbon and pH.
 15.01.2022, _AM Muskus, A Miltner, U Hamer, KM Nowak_


Glyphosate can be degraded by soil microorganisms rapidly and is impacted by temperature and soil properties. Enhanced temperature and total organic carbon (TOC) as well as reduced pH increased the rate of 13C315N-glyphosate conversion to CO2 and biogenic non-extractable residues (bioNERs) in a Haplic Chernozem (Muskus et al., 2019) and in a Humic Cambisol (Muskus et al., 2020). To date; however, the combined effect of temperature and TOC or pH on microbial community composition and glyphosate degraders in these two soils has not been investigated. Phospholipid fatty acid [PLFA] biomarker analysis combined with 13C labeling was employed to investigate the effect of two soil properties (pH, TOC) and of three temperatures (10 °C, 20 °C, 30 °C) on soil microorganisms. Before incubation, the properties of a Haplic Chernozem and a Humic Cambisol were adjusted to obtain five treatments: (a) Control (Haplic Chernozem: 2.1% TOC and pH 6.6; Humic Cambisol: 3% TOC and pH 7.0), (b) 3% TOC (Haplic Chernozem) or 4% TOC (Humic Cambisol), (c) 4% TOC (Haplic Chernozem) or 5% TOC (Humic Cambisol), (d) pH 6.0 (Haplic Chernozem) or pH 6.5 (Humic Cambisol), and (e) pH 5.5 for both soils. All treatments were amended with 50 mg kg-1 glyphosate and incubated at 10 °C, 20 °C or 30 °C. We observed an increase in respiration, microbial biomass and glyphosate mineralization with incubation temperature. Although respiration and microbial biomass in the Humic Cambisol was higher, the microorganisms in the Haplic Chernozem were more active in glyphosate degradation. Increased TOC shifted the microbiome and the 13C-glyphosate degraders towards Gram-positive bacteria in both soils. However, the abundance of 13C-PLFAs indicative for the starvation of Gram-negative bacteria increased with increasing TOC or decreasing pH at higher temperatures. Gram-negative bacteria thus may have been involved in earlier stages of glyphosate degradation.

[Source](https://doi.org/10.1016/j.envpol.2022.118790)

10.1016/j.envpol.2022.118790

---

## Increased Tea Saponin Content Influences the Diversity and Function of Plantation Soil Microbiomes.
 12.01.2022, _S Zhang, J Kong, L Chen, K Guo, X Zhou_


Plant secondary metabolites (PSMs) can affect the structures and functions of soil microbiomes. However, the core bacteria associated with PSMs, and their corresponding functions have not been explored extensively. In this study, soil physicochemical properties, tea saponin (TS) contents, microbial community compositions, and microbial community functions of different-age Camellia oleifera plantation soils from representative regions were analyzed. We evaluated the effects of plantation age increase on PSM accumulation, and the subsequent consequences on the structures and functions of soil microbiomes. Plantation ages increase positively correlated with accumulated TS contents, negative effects on soil physicochemical properties, and soil microbiome structures and functions. Clearly, the core functions of soil microbiomes transitioned to those associated with PSM metabolisms, while microbial pathways involved in cellulose degradation were inhibited. Our study systematically explored the influences of PSMs on soil microbiomes via the investigation of key bacterial populations and their functional pathways. With the increase in planting years, increased TS content simplified soil microbiome diversity, inhibited the degradation of organic matter, and enriched the genes related to the degradation of TS. These findings significantly advance our understanding on PSMs-microbiome interactions and could provide fundamental and important data for sustainable management of Camellia plantations. IMPORTANCE Plant secondary metabolites (PSMs) contained in plant litter will be released into soil with the decomposition process, which will affect the diversity and function of soil microbiomes. The response of soil microbiomes to PSMs in terms of diversity and function can provide an important theoretical basis for plantations to put forward rational soil ecological management measures. The effects of planting years on PSM content, soil physicochemical properties, microbial diversity, and function, as well as the interaction between each index in Camellia oleifera plantation soil are still unclear. We found that, with planting years increased, the accumulation of tea saponin (TS) led to drastic changes in the diversity and function of soil microbiomes, which hindered the decomposition of organic matter and enriched many genes related to PSM degradation. We first found that soil bacteria, represented by Acinetobacter, were significantly associated with TS degradation. Our results provide important data for proposing rational soil management measures for pure forest plantations.

[Source](https://doi.org/10.1128/spectrum.02324-21)

10.1128/spectrum.02324-21

---

## Identifying temporal and spatial patterns of variation from multimodal data using MEFISTO.
 14.01.2022, _B Velten, JM Braunger, R Argelaguet, D Arnol, J Wirbel, D Bredikhin, G Zeller, O Stegle_


Factor analysis is a widely used method for dimensionality reduction in genome biology, with applications from personalized health to single-cell biology. Existing factor analysis models assume independence of the observed samples, an assumption that fails in spatio-temporal profiling studies. Here we present MEFISTO, a flexible and versatile toolbox for modeling high-dimensional data when spatial or temporal dependencies between the samples are known. MEFISTO maintains the established benefits of factor analysis for multimodal data, but enables the performance of spatio-temporally informed dimensionality reduction, interpolation, and separation of smooth from non-smooth patterns of variation. Moreover, MEFISTO can integrate multiple related datasets by simultaneously identifying and aligning the underlying patterns of variation in a data-driven manner. To illustrate MEFISTO, we apply the model to different datasets with spatial or temporal resolution, including an evolutionary atlas of organ development, a longitudinal microbiome study, a single-cell multi-omics atlas of mouse gastrulation and spatially resolved transcriptomics.

[Source](https://doi.org/10.1038/s41592-021-01343-9)

10.1038/s41592-021-01343-9

---

## Inferring microbiota functions from taxonomic genes: a review.
 13.01.2022, _C Djemiel, PA Maron, S Terrat, S Dequiedt, A Cottin, L Ranjard_


Deciphering microbiota functions is crucial to predict ecosystem sustainability in response to global change. High-throughput sequencing at the individual or community level has revolutionized our understanding of microbial ecology, leading to the big data era and improving our ability to link microbial diversity with microbial functions. Recent advances in bioinformatics have been key for developing functional prediction tools based on DNA metabarcoding data and using taxonomic gene information. This cheaper approach in every aspect serves as an alternative to shotgun sequencing. Although these tools are increasingly used by ecologists, an objective evaluation of their modularity, portability, and robustness is lacking. Here, we reviewed 100 scientific papers on functional inference and ecological trait assignment to rank the advantages, specificities, and drawbacks of these tools, using a scientific benchmarking. To date, inference tools have been mainly devoted to bacterial functions, and ecological trait assignment tools, to fungal functions. A major limitation is the lack of reference genomes-compared with the human microbiota-especially for complex ecosystems such as soils. Finally, we explore applied research prospects. These tools are promising and already provide relevant information on ecosystem functioning, but standardized indicators and corresponding repositories are still lacking that would enable them to be used for operational diagnosis.

[Source](https://doi.org/10.1093/gigascience/giab090)

10.1093/gigascience/giab090

---

