## Microbiome sequencing in high-host and low biomass samples

[Source](https://www.nature.com/webcasts/event/microbiome-sequencing-in-high-host-and-low-biomass-samples/)

---

## Webinar Series on the Gut-Brain Axis and the Microbiome

[Source](https://ilsina.org/event/webinar-series-ilsi-north-america-and-division-of-nutritional-sciences-university-of-illinois/)

---

## Gut microbiota and neurological effects of glyphosate.
 30.04.2020, _L Rueda-Ruzafa, F Cruz, P Roman, D Cardona_


_Animals, Dysbiosis, Gastrointestinal Microbiome, Glycine, Herbicides, Humans, Nervous System Diseases_
There are currently various concerns regarding certain environmental toxins and the possible impact they can have on developmental diseases. Glyphosate (Gly) is the most utilised herbicide in agriculture, although its widespread use is generating controversy in the scientific world because of its probable carcinogenic effect on human cells. Gly performs as an inhibitor of 5-enolpyruvylshikimate-3-phospate synthase (EPSP synthase), not only in plants, but also in bacteria. An inhibiting effect on EPSP synthase from intestinal microbiota has been reported, affecting mainly beneficial bacteria. To the contrary, Clostridium spp. and Salmonella strains are shown to be resistant to Gly. Consequently, researchers have suggested that Gly can cause dysbiosis, a phenomenon which is characterised by an imbalance between beneficial and pathogenic microorganisms. The overgrowth of bacteria such as clostridia generates high levels of noxious metabolites in the brain, which can contribute to the development of neurological deviations. This work reviews the impact of Gly-induced intestinal dysbiosis on the central nervous system, focusing on emotional, neurological and neurodegenerative disorders. A wide variety of factors were investigated in relation to brain-related changes, including highlighting genetic abnormalities, pregnancy-associated problems, diet, infections, vaccines and heavy metals. However, more studies are required to determine the implication of the most internationally used herbicide, Gly, in behavioural disorders.

[Source](https://www.biocodexmicrobiotafoundation.com/international-call-projects-2021)

10.1016/j.neuro.2019.08.006

---

## Functional effects of the human gut microbiome on autism

[Source](https://www.biocodexmicrobiotafoundation.com/international-call-projects-2021)

---

## Focus on Microbiome in Health and Disease – A special microbiome issue in Cell Host Microbe

[Source](https://www.cell.com/cell-host-microbe/issue?pii=S1931-3128(19)X0009-8)

---

## The Impact of the Microbiome on Immunity to Vaccination in Humans.
 21.08.2020, _SE de Jong, A Olin, B Pulendran_


Vaccines are the most effective means available for preventing infectious diseases. However, vaccine-induced immune responses are highly variable between individuals and between populations in different regions of the world. Understanding the basis of this variation is, thus, of fundamental importance to human health. Although the factors that are associated with intra- and inter-population variation in vaccine responses are manifold, emerging evidence points to a key role for the gut microbiome in controlling immune responses to vaccination. Much of this evidence comes from studies in mice, and causal evidence for the impact of the microbiome on human immunity is sparse. However, recent studies on vaccination in subjects treated with broad-spectrum antibiotics have provided causal evidence and mechanistic insights into how the microbiota controls immune responses in humans.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30355-3)

10.1016/j.chom.2020.06.014

---

## The Gut Microbiota and Unhealthy Aging: Disentangling Cause from Consequence.
 21.08.2020, _EN DeJong, MG Surette, DME Bowdish_


The gut microbiota changes with age, but it is not clear to what degree these changes are due to physiologic changes, age-associated inflammation or immunosenescence, diet, medications, or chronic health conditions. Observational studies in humans find that there are profound differences between the microbiomes of long-lived and frail individuals, but the degree to which these differences promote or prevent late-life health is unclear. Studies in model organisms demonstrate that age-related microbial dysbiosis causes intestinal permeability, systemic inflammation, and premature mortality, but identifying causal relationships have been challenging. Herein, we review how physiological and immune changes contribute to microbial dysbiosis and the degree to which microbial dysbiosis contributes to late-life health conditions. We discuss the features of the aging microbiota that make it more amenable to diet and pre- and probiotic interventions. Health interventions that promote a diverse microbiome could influence the health of older adults.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30409-1)

10.1016/j.chom.2020.07.013

---

## Helminth coinfection and COVID-19: An alternate hypothesis.
 21.08.2020, _R Hays, D Pierce, P Giacomin, A Loukas, P Bourke, R McDermott_



[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0008628)

10.1371/journal.pntd.0008628

---

## Mitochondria, the gut microbiome and ROS.
 18.08.2020, _JWO Ballard, SG Towarnicki_


In this review, we discuss the connections between mitochondria and the gut microbiome provided by reactive oxygen species (ROS). We examine the mitochondrion as an endosymbiotic organelle that is a hub for energy production, signaling, and cell homeostasis. Maintaining a diverse gut microbiome is generally associated with organismal fitness, intestinal health and resistance to environmental stress. In contrast, gut microbiome imbalance, termed dysbiosis, is linked to a reduction in organismal well-being. ROS are essential signaling molecules but can be damaging when present in excess. Increasing ROS levels have been shown to influence human health, homeostasis of gut cells, and the gastrointestinal microbial community's biodiversity. Reciprocally, gut microbes can affect ROS levels, mitochondrial homeostasis, and host health. We propose that mechanistic understanding of the suite of bi-directional interactions between mitochondria and the gut microbiome will facilitate innovative interdisciplinary studies examining evolutionary divergence and provide novel treatments and therapeutics for disease. GLOSS: In this review, we focus on the nexus between mitochondria and the gut microbiome provided by reactive oxygen species (ROS). Mitochondria are a cell organelle that is derived from an ancestral alpha-proteobacteria. They generate around 80% of the adenosine triphosphate that an organism needs to function and release a range of signaling molecules essential for cellular homeostasis. The gut microbiome is a suite of microorganisms that are commensal, symbiotic and pathogenic to their host. ROS are one predominant group of essential signaling molecules that can be harmful in excess. We suggest that the mitochondria- microbiome nexus is a frontier of research that has cross-disciplinary benefits in understanding genetic divergence and human well-being.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S089865682030214X)

10.1016/j.cellsig.2020.109737

---

## Mitochondria, the gut microbiome and ROS. and ROS

[Source](https://www.sciencedirect.com/science/article/abs/pii/S089865682030214X)

---

## Stool Microbiome Profiling of Patients with Metastatic Renal Cell Carcinoma Receiving Anti–PD-1 Immune Checkpoint Inhibitors

[Source](https://www.europeanurology.com/article/S0302-2838(20)30543-1/fulltext)

---

## Programing of an Intravascular Immune Firewall by the Gut Microbiota Protects against Pathogen Dissemination during Infection.
 18.08.2020, _B McDonald, AZ Zucoloto, IL Yu, R Burkhard, K Brown, MB Geuking, KD McCoy_


Eradication of pathogens from the bloodstream is critical to prevent disseminated infections and sepsis. Kupffer cells in the liver form an intravascular firewall that captures and clears pathogens from the blood. Here, we show that the catching and killing of circulating pathogens by Kupffer cells in vivo are promoted by the gut microbiota through commensal-derived D-lactate that reaches the liver via the portal vein. The integrity of this Kupffer cell-mediated intravascular firewall requires continuous crosstalk with gut commensals, as microbiota depletion with antibiotics leads to a failure of pathogen clearance and overwhelming disseminated infection. Furthermore, administration of purified D-lactate to germ-free mice, or gnotobiotic colonization with D-lactate-producing commensals, restores Kupffer cell-mediated pathogen clearance by the liver firewall. Thus, the gut microbiota programs an intravascular immune firewall that protects against the spread of bacterial infections via the bloodstream.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30410-8)

10.1016/j.chom.2020.07.014

---

## Analysis of microbial changes in the tonsillar formalin-fixed paraffin-embedded tissue of Chinese patients with IgA nephropathy

[Source](https://www.sciencedirect.com/science/article/abs/pii/S034403382032029X)

---

## Breastmilk Feeding Practices Are Associated with the Co-Occurrence of Bacteria in Mothers' Milk and the Infant Gut: the CHILD Cohort Study.
 21.08.2020, _K Fehr, S Moossavi, H Sbihi, RCT Boutin, L Bode, B Robertson, C Yonemitsu, CJ Field, AB Becker, PJ Mandhane, MR Sears, E Khafipour, TJ Moraes, P Subbarao, BB Finlay, SE Turvey, MB Azad_


Gut microbiota play a critical role in infant health. It is now accepted that breastmilk contains live bacteria from endogenous and exogenous sources, but it remains unclear whether these bacteria transfer to the infant gut and whether this process is influenced by breastmilk feeding practices. Here, we show that certain bacteria, including Streptococcus spp. and Veillonella dispar, co-occur in mothers' milk and their infants' stool, and co-occurrence is reduced when infants receive pumped breastmilk. The relative abundances of commonly shared species are positively correlated between breastmilk and stool. Overall, gut microbiota composition is strongly associated with breastfeeding exclusivity and duration but not breastmilk feeding mode (nursing versus pumping). Moreover, breastmilk bacteria contributed to overall gut microbiota variation to a similar extent as other modifiers of the infant microbiome, such as birth mode. These results provide evidence that breastmilk may transfer bacteria to the infant gut and influence microbiota development.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30350-4)

10.1016/j.chom.2020.06.009

---

## Effects of Urbanization and Landscape on Gut Microbiomes in White-Crowned Sparrows.
 17.08.2020, _M Berlow, JN Phillips, EP Derryberry_


Habitats are changing rapidly around the globe and urbanization is one of the primary drivers. Urbanization changes food availability, environmental stressors, and the prevalence of disease for many species. These changes can lead to divergence in phenotypic traits, including behavioral, physiological, and morphological features between urban and rural populations. Recent research highlights that urbanization is also changing the gut microbial communities found in a diverse group of host species. These changes have not been uniform, leaving uncertainty as to how urban habitats are shaping gut microbial communities. To better understand these effects, we investigated the gut bacterial communities of White-Crowned Sparrow (Zonotrichia leucophrys) populations along an urbanization gradient in the San Francisco Bay area. We examined how gut bacterial communities vary with the local environment and host morphological characteristics. We found direct effects of environmental factors, including urban noise levels and territory land cover, as well as indirect effects through body size and condition, on alpha and beta diversity of gut microbial communities. We also found that urban and rural birds' microbiomes differed in which variables predicted their diversity, with urban communities driven by host morphology, and rural communities driven by environmental factors. Elucidating these effects provides a better understanding of how urbanization affects wild avian physiology.

[Source](https://link.springer.com/article/10.1007/s00248-020-01569-8)

10.1007/s00248-020-01569-8

---

## Passenger-surface microbiome interactions in the subway of Mexico City.
 19.08.2020, _D Vargas-Robles, C Gonzalez-Cedillo, AM Hernandez, LD Alcaraz, M Peimbert_


Interaction between hands and the environment permits the interchange of microorganisms. The Mexico City subway is used daily by millions of passengers that get in contact with its surfaces. In this study, we used 16S rRNA gene sequencing to characterize the microbiomes of frequently touched surfaces and compare regular and women-only wagons. We also explored the effect of surface cleaning on microbial resettling. Finally, we studied passenger behavior and characterized microbial changes after traveling. Most passengers (99%), showed some type of surface interaction during a wagon trip, mostly with the hands (92%). We found microbiome differences associated with surfaces, probably reflecting diverse surface materials and usage frequency. The platform floor was the most bacterial diverse surface, while the stair handrail and pole were the least diverse ones. After pole cleaning, the resettling of microbial diversity was fast (5-30 minutes); however, it did not resemble the initial composition. After traveling, passengers significantly increased their hand microbial diversity and converged to a similar microbial composition among passengers. Additionally, passenger hand microbiomes resembled subway surfaces in diversity. However, microbial fingerprints were preserved within passengers after traveling.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237272)

10.1371/journal.pone.0237272

---

## Plant resistance and leaf chemical characteristic jointly shape phyllosphere bacterial community.
 18.08.2020, _T Xueliang, X Dan, S Tingting, Z Songyu, L Ying, W Diandong_


Phyllosphere bacteria have an important role in plant growth and resistance to pathogen infection and are partially influenced by plant genotype and leaf environment. How plant resistance to pathogens and leaf chemical characteristics shape the phyllosphere bacterial communities is unclear. In this study, the phyllosphere bacterial communities of maize hybrids with various resistance to Setosphaeria turcica were compared using the high-throughput sequencing and large-scale culturing methods. The results showed that Shannon and Simpson indices of phyllosphere bacterial communities were markedly higher in the highly resistant hybrid (HR) compared with the susceptible one. Hierarchical clustering analysis, unweighted UniFrac principal component analysis (PCoA) and the analysis of similarities (ANOSIM) demonstrated that the phyllosphere bacterial communities were significantly distinct between resistant and susceptible hybrids. The redundancy analysis (RDA) demonstrated that leaf chemical characteristics, including nitrogen and phosphorus concentration, and disease resistance play an important role in shaping the phyllosphere bacterial community. Linear discriminant effect size (LEfSe) analysis indicated that Bacillus, Pseudomonas and Tumebacillus were the biomarker species in the phyllosphere of HR. Biocontrol bacteria against S. turcica (such as Pseudomonas and Bacillus) were isolated from the phyllosphere of HR by large-scale culturing. The work contributes to understanding of the phyllosphere bacterial community assembly and provides a new clue to screening for strong biocontrol bacteria from HR and to facilitating future breeding efforts for enhancing disease resistance.

[Source](https://link.springer.com/article/10.1007/s11274-020-02908-0)

10.1007/s11274-020-02908-0

---

## The role of land management and elevation in shaping soil microbial communities: Insights from the Central European Alps

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0038071720302479)

---

## Composition, predicted functions, and co-occurrence networks of fungal and bacterial communities_ Links to soil organic carbon under long-term fertilization in a rice-wheat cropping system

[Source](https://www.sciencedirect.com/science/article/abs/pii/S116455632030296X)

---

## Diversity and structure of prokaryotic communities within organic and conventional farming systems in central highlands of Kenya.
 21.08.2020, _EN Karanja, A Fliessbach, N Adamtey, AK Kambura, M Musyoka, K Fiaboe, R Mwirichia_


Management practices such as tillage, crop rotation, irrigation, organic and inorganic inputs application are known to influence diversity and function of soil microbial populations. In this study, we investigated the effect of conventional versus organic farming systems at low and high input levels on structure and diversity of prokaryotic microbial communities. Soil samples were collected from the ongoing long-term farming system comparison trials established in 2007 at Chuka and Thika in Kenya. Physicochemical parameters for each sample were analyzed. Total DNA and RNA amplicons of variable region (V4-V7) of the 16S rRNA gene were generated on an Illumina platform using the manufacturer's instructions. Diversity indices and statistical analysis were done using QIIME2 and R packages, respectively. A total of 29,778,886 high quality reads were obtained and assigned to 16,176 OTUs at 97% genetic distance across both 16S rDNA and 16S rRNA cDNA datasets. The results pointed out a histrionic difference in OTUs based on 16S rDNA and 16S rRNA cDNA. Precisely, while 16S rDNA clustered by site, 16S rRNA cDNA clustered by farming systems. In both sites and systems, dominant phylotypes were affiliated to phylum Actinobacteria, Proteobacteria and Acidobacteria. Conventional farming systems showed a higher species richness and diversity compared to organic farming systems, whilst 16S rRNA cDNA datasets were similar. Physiochemical factors were associated differently depending on rRNA and rDNA. Soil pH, electrical conductivity, organic carbon, nitrogen, potassium, aluminium, zinc, iron, boron and micro-aggregates showed a significant influence on the observed microbial diversity. The observed higher species diversity in the conventional farming systems can be attributed to the integration of synthetic and organic agricultural inputs. These results show that the type of inputs used in a farming system not only affect the soil chemistry but also the microbial population dynamics and eventually the functional roles of these microbes.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0236574)

10.1371/journal.pone.0236574

---

## Turning harmful algal biomass to electricity by microbial fuel cell: A sustainable approach for waste management

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0269749120360619)

---

## Combining physicochemical properties and microbiome data to evaluate the water quality of South African drinking water production plants.
 21.08.2020, _TE Maguvu, CC Bezuidenhout, R Kritzinger, K Tsholo, M Plaatjie, LG Molale-Tom, CM Mienie, RD Coertze_


Anthropogenic activities in catchments used for drinking water production largely contaminates source waters, and this may impact the quality of the final drinking water product. These contaminants may also affect taxonomic and functional profiles of the bacterial communities in the drinking water. Here, we report an integrated insight into the microbiome and water quality of four water treatment plants (NWC, NWE, WCA and NWG) that supply portable water to communities in South Africa. A new scoring system based on combined significant changes of physicochemical parameters and microbial abundance from raw to treated water was used to evaluate the effectiveness of the treatment plants at water purification. Physicochemical parameters which include total soluble solids, turbidity, pH, nitrites and phosphorus among others, were measured in source, treated, and distributed water. There were general statistically significant (P ≤ 0.05) differences between raw and treated water, demonstrating the effectiveness of the purification process. Illumina sequencing of the 16S rRNA gene was used for taxonomic profiling of the microbial communities and this data was used to infer functional attributes of the communities. Structure and composition of the bacterial communities differed significantly (P &lt; 0.05) among the treatment plants, only NWE and NWG showed no significant differences (P &gt; 0.05), this correlated with the predicted functional profile of the microbial communities obtained from Phylogenetic Investigation of Communities by Reconstruction of Observed States (PICRUSt), as well as the likely pollutants of source water. Bacteroidetes, Chlorobi and Fibrobacteres significantly differed (P &lt; 0.05) between raw and distributed water. PICRUSt inferred a number of pathways involved in the degradation of xenobiotics such as Dichlorodiphenyltrichloroethane, atrazine and polycyclic aromatic hydrocarbons. More worryingly, was the presence of pathways involved in beta-lactam resistance, potential pathogenic Escherichia coli infection, Vibrio cholerae infection, and Shigellosis. Also present in drinking and treated water were OTUs associated with a number of opportunistic pathogens.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237335)

10.1371/journal.pone.0237335

---

## They are what you eat: Shaping of viral populations through nutrition and consequences for virulence.
 21.08.2020, _R Honce, S Schultz-Cherry_



[Source](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1008711)

10.1371/journal.ppat.1008711

---

## Assembly methods for nanopore-based metagenomic sequencing: a comparative study.
 18.08.2020, _A Latorre-Pérez, P Villalba-Bermell, J Pascual, C Vilanova_


Metagenomic sequencing has allowed for the recovery of previously unexplored microbial genomes. Whereas short-read sequencing platforms often result in highly fragmented metagenomes, nanopore-based sequencers could lead to more contiguous assemblies due to their potential to generate long reads. Nevertheless, there is a lack of updated and systematic studies evaluating the performance of different assembly tools on nanopore data. In this study, we have benchmarked the ability of different assemblers to reconstruct two different commercially-available mock communities that have been sequenced using Oxford Nanopore Technologies platforms. Among the tested tools, only metaFlye, Raven, and Canu performed well in all the datasets. These tools retrieved highly contiguous genomes (or even complete genomes) directly from the metagenomic data. Despite the intrinsic high error of nanopore sequencing, final assemblies reached high accuracy (~ 99.5 to 99.8% of consensus accuracy). Polishing strategies demonstrated to be necessary for reducing the number of indels, and this had an impact on the prediction of biosynthetic gene clusters. Correction with high quality short reads did not always result in higher quality draft assemblies. Overall, nanopore metagenomic sequencing data-adapted to MinION's current output-proved sufficient for assembling and characterizing low-complexity microbial communities.

[Source](https://www.nature.com/articles/s41598-020-70491-3)

10.1038/s41598-020-70491-3

---

## Completion of draft bacterial genomes by long-read sequencing of synthetic genomic pools.
 05.08.2020, _H Derakhshani, SP Bernier, VA Marko, MG Surette_


Illumina technology currently dominates bacterial genomics due to its high read accuracy and low sequencing cost. However, the incompleteness of draft genomes generated by Illumina reads limits their application in comprehensive genomics analyses. Alternatively, hybrid assembly using both Illumina short reads and long reads generated by single molecule sequencing technologies can enable assembly of complete bacterial genomes, yet the high per-genome cost of long-read sequencing limits the widespread use of this approach in bacterial genomics. Here we developed a protocol for hybrid assembly of complete bacterial genomes using miniaturized multiplexed Illumina sequencing and non-barcoded PacBio sequencing of a synthetic genomic pool (SGP), thus significantly decreasing the overall per-genome cost of sequencing.

[Source](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-06910-6)

10.1186/s12864-020-06910-6

---

## Modeling microbial cross-feeding at intermediate scale portrays community dynamics and species coexistence.
 18.08.2020, _C Liao, T Wang, S Maslov, JB Xavier_


Social interaction between microbes can be described at many levels of details: from the biochemistry of cell-cell interactions to the ecological dynamics of populations. Choosing an appropriate level to model microbial communities without losing generality remains a challenge. Here we show that modeling cross-feeding interactions at an intermediate level between genome-scale metabolic models of individual species and consumer-resource models of ecosystems is suitable to experimental data. We applied our modeling framework to three published examples of multi-strain Escherichia coli communities with increasing complexity: uni-, bi-, and multi-directional cross-feeding of either substitutable metabolic byproducts or essential nutrients. The intermediate-scale model accurately fit empirical data and quantified metabolic exchange rates that are hard to measure experimentally, even for a complex community of 14 amino acid auxotrophies. By studying the conditions of species coexistence, the ecological outcomes of cross-feeding interactions, and each community's robustness to perturbations, we extracted new quantitative insights from these three published experimental datasets. Our analysis provides a foundation to quantify cross-feeding interactions from experimental data, and highlights the importance of metabolic exchanges in the dynamics and stability of microbial communities.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008135)

10.1371/journal.pcbi.1008135

---

## Bacterial modification of the host glycosaminoglycan heparan sulfate modulates SARS-CoV-2 infectivity

[Source](https://www.biorxiv.org/content/10.1101/2020.08.17.238444v1)

---

## Evaluation of Compatibility of 16S rRNA V3V4 and V4 Amplicon Libraries for Clinical Microbiome Profiling

[Source](https://www.biorxiv.org/content/10.1101/2020.08.18.256818v1.abstract)

---

## François-Joseph Lapointe, when biology meets art

[Source](https://www.convergenceinitiative.org/post/françois-joseph-lapointe-when-biology-meets-art)

---

## Early-life microbiota diversification facilitates enterocyte proliferation

[Source](https://naturemicrobiologycommunity.nature.com/posts/early-life-microbiota-diversification-facilitates-enterocyte-proliferation)

---

## ecology and evolution of metabolic cross-feeding interactions in soil-living bacterial communities

[Source](https://twitter.com/KostChristian/status/1295265650525560832)

---

## gut microbiota in protecting the host against intestinal colonisation by multidrug-resistant organisms

[Source](https://twitter.com/JulieAKMcDonald/status/1296364661009453056)

---

