## The human microbiome and COVID-19: A systematic review.
 02.07.2021, _S Yamamoto, M Saito, A Tamura, D Prawisuda, T Mizutani, H Yotsuyanagi_


_Bronchoalveolar Lavage Fluid, COVID-19, Dysbiosis, Feces, Gastrointestinal Microbiome, Humans, Microbiota, Nasopharynx_
Human microbiotas are communities of microorganisms living in symbiosis with humans. They play an important role in the host immune response to respiratory viral infection. However, evidence on the human microbiome and coronavirus disease (COVID-19) relationship is insufficient. The aim of this systematic literature review was to evaluate existing evidence on the association between the microbiome and COVID-19 in humans and summarize these data in the pandemic era.

[Source](https://doi.org/10.1371/journal.pone.0253293)

10.1371/journal.pone.0253293

---

## The gut microbiome and type 2 diabetes status in the Multiethnic Cohort.
 23.06.2021, _G Maskarinec, P Raquinio, BS Kristal, VW Setiawan, LR Wilkens, AA Franke, U Lim, L Le Marchand, TW Randolph, JW Lampe, MAJ Hullar_


The gut microbiome may play a role in inflammation associated with type 2 diabetes (T2D) development. This cross-sectional study examined its relation with glycemic status within a subset of the Multiethnic Cohort (MEC) and estimated the association of circulating bacterial endotoxin (measured as plasma lipopolysaccharide-binding protein (LBP)) with T2D, which may be mediated by C-reactive protein (CRP).

[Source](https://doi.org/10.1371/journal.pone.0250855)

10.1371/journal.pone.0250855

---

## Niche partitioning facilitates coexistence of closely related gut bacteria

[Source](https://doi.org/10.1101/2021.03.12.434400)

---

## Successional Stages in Infant Gut Microbiota Maturation

[Source](https://doi.org/10.1101/2021.06.25.450009)

---

## Fetal abdominal tumors and cysts.
 01.07.2021, _DL Cass_


This article reviews the contemporary diagnosis and management of the most common abdominal neoplasms and cystic lesions diagnosed in the fetus. Fetal tumors discussed include teratomas (sacrococcygeal, cervical or mediastinal), mesoblastic nephroma, nephroblastoma (Wilms' tumor), neuroblastoma, and hepatoblastoma. Fetal abdominal cystic lesions discussed include ovarian cyst, choledochal cyst, intestinal duplication cyst, mesenteric cyst, simple hepatic cyst, and meconium pseudocyst. We discuss the rare indications for fetal intervention or fetal surgery and other perinatal management, including prenatal interventions and fetal surgery for sacrococcygeal teratoma. The lesions reviewed are detected by widespread use of screening ultrasonography during pregnancy. Work-up for these abnormalities may include fetal MRI which enhances the diagnostic accuracy of abdominal tumors and cystic lesions and can aid in characterization of the lesion in relationship to surrounding anatomic structures. Accurate prenatal diagnosis of such lesions permits recommendations for optimal location and timing of delivery, and inclusion of appropriate caregivers and expertise to facilitate postnatal management. Perinatal management of the fetus with a neoplasm requires consideration of the optimal timing and mode of delivery, and pediatric oncology and surgical specialty care. The majority of tumors diagnosed antenatally have good prognosis with current multimodality treatment.

[Source](https://doi.org/10.1038/s41564-021-00904-0)

10.21037/tp-20-440

---

## Fetal meconium does not have a detectable microbiota before birth

[Source](https://doi.org/10.1038/s41564-021-00904-0)

---

## Developmental intestinal microbiome alterations in canine fading puppy syndrome: a prospective observational study.
 24.06.2021, _S Tal, E Tikhonov, I Aroch, L Hefetz, S Turjeman, O Koren, S Kuzi_


Fading puppy syndrome (FPS) is a fatal condition in neonatal dogs. Intestinal microbial alterations, although never investigated, may be involved in its pathophysiology. The study examined the occurrence of FPS and its associations with dam, puppy, and husbandry characteristics, compared the intestinal microbial diversity of healthy puppies and those with FPS, and examined whether intestinal microbiomes are predictive of FPS. Day 1 and 8 post-partum (PP) rectal swabs were collected from healthy puppies and puppies which later developed FPS. Microbial compositional structure, including alpha and beta diversities and relative abundance of specific taxa were compared between groups, and microbial data was applied to a machine-learning model to assess the predictive performance of microbial indices of FPS or death. FPS occurred in 22/165 puppies (13%), with a 100% mortality rate. FPS was associated (P &lt; 0.001) with decreased Day 1 PP puppy activity. Day 1 (P = 0.003) and 8 (P = 0.005) PP rectal beta diversities were different in puppies with FPS vs healthy ones. Increased Proteobacteria/Firmicutes ratio, increased relative abundance of Pasteurellaceae, and decreased relative abundance of Clostridia and Enterococcus were associated with FPS. A machine-learning model showed that Day 1 PP rectal microbiome composition accurately predicted FPS-related death. We found that specific rectal microbial phenotypes are associated with FPS, reflecting the significant role of microbiome alterations in this phenomenon. These findings may serve as useful microbial indices for early diagnosis of puppies at risk of FPS and may provide specific therapeutic targets.

[Source](https://doi.org/10.1038/s41522-021-00222-7)

10.1038/s41522-021-00222-7

---

## Gut microbiome alterations in high-fat-diet-fed mice are associated with antibiotic tolerance.
 30.06.2021, _Y Liu, K Yang, Y Jia, J Shi, Z Tong, D Fang, B Yang, C Su, R Li, X Xiao, Z Wang_


Antibiotic tolerance, the ability of a typically susceptible microorganism to survive extended periods of exposure to antibiotics, has a critical role in chronic and recurrent bacterial infections, and facilitates the evolution of antibiotic resistance. However, the physiological factors that contribute to the development of antibiotic tolerance, particularly in vivo, are not fully known. Despite the fact that a high-fat diet (HFD) is implicated in several human diseases, the relationship between HFD and antibiotic efficacy is still poorly understood. Here, we evaluated the efficacy of multiple clinically relevant bactericidal antibiotics in HFD-fed mice infected with methicillin-resistant Staphylococcus aureus (MRSA) or Escherichia coli. We found that HFD-fed mice had higher bacterial burdens and these bacteria displayed lower susceptibility to bactericidal antibiotic treatment compared with mice that were fed a standard diet, while microbiota-depleted standard-diet- or HFD-fed mice showed similar susceptibility. Faecal microbiota transplantation from HFD-fed mice impaired antibiotic activity in mice fed a standard diet, indicating that alteration of the gut microbiota and related metabolites in HFD-fed mice may account for the decreased antibiotic activity. 16S rRNA sequencing and metabolomics analysis of faecal samples revealed decreased microbial diversity and differential metabolite profiles in HFD-fed mice. Notably, the tryptophan metabolite indole-3-acetic acid (IAA) was significantly decreased in HFD-fed mice. Further in vitro studies showed that IAA supplementation inhibited the formation of bacterial persisters and promoted the elimination of persisters in combination with antibiotic treatment, potentially through the activation of bacterial metabolic pathways. In vivo, the combination of IAA and ciprofloxacin increased the survival rate of HFD-fed mice infected with MRSA persisters. Overall, our data reveal that a HFD has an antagonistic effect on antibiotic treatment in a mouse model, and this is associated with the alteration of the gut microbiota and IAA production.

[Source](https://doi.org/10.1038/s41564-021-00912-0)

10.1038/s41564-021-00912-0

---

## Enteric viruses nucleic acids distribution along the digestive tract of rhesus macaques with idiopathic chronic diarrhea

[Source](https://doi.org/10.1101/2021.06.24.449827)

---

## Microbiomes of commercially-available pine nuts and sesame seeds.
 02.07.2021, _M Fay, JK Salazar, P Ramachandran, D Stewart_


Metagenomic analysis of food is becoming more routine and can provide important information pertaining to the shelf life potential and the safety of these products. However, less information is available on the microbiomes associated with low water activity foods. Pine nuts and sesame seeds, and food products which contain these ingredients, have been associated with recalls due to contamination with bacterial foodborne pathogens. The objective of this study was to identify the microbial community of pine nuts and sesame seeds using targeted 16S rRNA sequencing technology. Ten different brands of each seed type were assessed, and core microbiomes were determined. A total of 21 and 16 unique taxa with proportional abundances &gt;1% in at least one brand were identified in the pine nuts and sesame seeds, respectively. Members of the core pine nut microbiome included the genera Alishewanella, Aminivibrio, Mycoplasma, Streptococcus, and unassigned OTUs in the families of Desulfobacteraceae and Xanthomonadaceae. For sesame seeds, the core microbiome included Aminivibrio, Chryseolina, Okibacterium, and unassigned OTUs in the family Flavobacteriaceae. The microbiomes of these seeds revealed that these products are dominated by environmental bacterial genera commonly isolated from soil, water, and plants; bacterial genera containing species known as commensal organisms were also identified. Understanding these microbiomes can aid in the risk assessment of these products by identifying food spoilage potential and community members which may co-enrich with foodborne bacterial pathogens.

[Source](https://doi.org/10.1371/journal.pone.0252605)

10.1371/journal.pone.0252605

---

## Correction to: Spatial patterns in phage-Rhizobium coevolutionary interactions across regions of common bean domestication.
 01.07.2021, _J Van Cauwenberghe, RI Santamaría, P Bustos, S Juárez, MA Ducci, TF Fleming, AV Etcheverry, V González_



[Source](https://doi.org/10.1038/s41396-021-00907-z)

10.1038/s41396-021-00963-5

---

## Spatial patterns in phage-Rhizobium coevolutionary interactions across regions of common bean domestication

[Source](https://doi.org/10.1038/s41396-021-00907-z)

---

## <span style="color:#ff6600;">Genomic insights into diverse bacterial taxa that degrade extracellular DNA in marine sediments</span>

[Source](https://doi.org/10.1038/s41564-021-00917-9)

---

## High viral abundance and low diversity are associated with increased CRISPR-Cas prevalence across microbial ecosystems

[Source](https://doi.org/10.1101/2021.06.24.449667)

---

## Metagenomic compendium of 189,680 DNA viruses from the human gut microbiome

[Source](https://doi.org/10.1038/s41564-021-00928-6)

---

## SCAPP: an algorithm for improved plasmid assembly in metagenomes.
 01.07.2021, _D Pellow, A Zorea, M Probst, O Furman, A Segal, I Mizrahi, R Shamir_


_Algorithms, Humans, Metagenome, Metagenomics, Plasmids, Sequence Analysis, DNA, Software_
Metagenomic sequencing has led to the identification and assembly of many new bacterial genome sequences. These bacteria often contain plasmids: usually small, circular double-stranded DNA molecules that may transfer across bacterial species and confer antibiotic resistance. These plasmids are generally less studied and understood than their bacterial hosts. Part of the reason for this is insufficient computational tools enabling the analysis of plasmids in metagenomic samples.

[Source](https://doi.org/10.1186/s40168-021-01068-z)

10.1186/s40168-021-01068-z

---

## A standardized archaeal taxonomy for the Genome Taxonomy Database.
 30.06.2021, _C Rinke, M Chuvochina, AJ Mussig, PA Chaumeil, AA Davín, DW Waite, WB Whitman, DH Parks, P Hugenholtz_


The accrual of genomic data from both cultured and uncultured microorganisms provides new opportunities to develop systematic taxonomies based on evolutionary relationships. Previously, we established a bacterial taxonomy through the Genome Taxonomy Database. Here, we propose a standardized archaeal taxonomy that is derived from a 122-concatenated-protein phylogeny that resolves polyphyletic groups and normalizes ranks based on relative evolutionary divergence. The resulting archaeal taxonomy, which forms part of the Genome Taxonomy Database, is stable for a range of phylogenetic variables including marker gene selection, inference methods, corrections for rate heterogeneity and compositional bias, tree rooting scenarios and expansion of the genome database. Rank normalization is shown to robustly correct for substitution rates varying up to 30-fold using simulated datasets. Taxonomic curation follows the rules of the International Code of Nomenclature of Prokaryotes while taking into account proposals to formally recognize the rank of phylum and to use genome sequences as type material. This taxonomy is based on 2,392 archaeal genomes, 93.3% of which required one or more changes to their existing taxonomy, mainly owing to incomplete classification. We identify 16 archaeal phyla and reclassify 3 major monophyletic units from the former Euryarchaeota and one phylum that unites the Thaumarchaeota-Aigarchaeota-Crenarchaeota-Korarchaeota (TACK) superphylum into a single phylum.

[Source](https://doi.org/10.1038/s41564-021-00918-8)

10.1038/s41564-021-00918-8

---

## Is the open access citation advantage real? A systematic review of the citation of open access and subscription-based articles

[Source](https://doi.org/10.1371/journal.pone.0253129)

---

