## An exosome mRNA-related gene risk model to evaluate the tumor microenvironment and predict prognosis in hepatocellular carcinoma.
 19.11.2024, _Z Du, X Han, L Zhu, L Li, L Castellano, J Stebbing, L Peng, Z Wang_


_Humans, Carcinoma, Hepatocellular, Exosomes, RNA, Messenger, Tumor Microenvironment, Phosphatidylinositol 3-Kinases, Liver Neoplasms, Prognosis, Risk Factors, Microfilament Proteins, Vesicular Transport Proteins_
The interplay between exosomes and the tumor microenvironment (TME) remains unclear. We investigated the influence of exosomes on the TME in hepatocellular carcinoma (HCC), focusing on their mRNA expression profile.

[Source](https://www.nature.com/articles/d41586-024-03784-6)

10.1186/s12920-024-01865-z

---

## https://www.nature.com/articles/d41586-024-03784-6

[Source](https://www.nature.com/articles/d41586-024-03784-6)

---

## NeurosurGen, Inc.: an academic ancestry database for neurosurgery.
 29.11.2024, _AA Gajjar, CH Alleyne_


The mission of NeurosurGen, Inc., is to compile and maintain genealogical data on every neurosurgeon in the US and, eventually, the world.

[Source](https://cassyni.com/s/mvif-34)

10.3171/2024.7.JNS232995

---

## https://cassyni.com/s/mvif-34

[Source](https://cassyni.com/s/mvif-34)

---

## Microbiome Virtual International Forum (@microbiomevif.bsky.social)

[Source](https://bsky.app/profile/did:plc:dyrbb3iiuynmhsho3hywdakx?ref_src=embed)

---

## Presence of dengue virus RNA in urine and oral fluid of laboratory-confirmed dengue patients: Implications for wastewater surveillance.
 29.11.2024, _C Stauber, LC Jacob-Nascimento, C Grosch, MDS Sousa, MM Portilho, RO Anjos, MA Brinton, U Kitron, MG Reis, GS Ribeiro_


Dengue cases in the Americas in 2024 have reached record highs, especially in Brazil. However, surveillance remains suboptimal and new methods are needed to monitor Dengue Virus (DENV) spread. To assess whether wastewater-based epidemiology would be a useful tool, we investigated the presence of DENV RNA in dengue patients' urine and oral fluid from an endemic area to inform how shedding in these fluids occurs and provide insight for wastewater surveillance.

[Source](https://bsky.app/profile/did:plc:dyrbb3iiuynmhsho3hywdakx/post/3lbelftyhn22t?ref_src=embed)

10.1016/j.bjid.2024.104484

---

## 2024-11-20T08:57:41.034Z

[Source](https://bsky.app/profile/did:plc:dyrbb3iiuynmhsho3hywdakx/post/3lbelftyhn22t?ref_src=embed)

---

## Citizen science as an instrument for women’s health research

[Source](https://www.nature.com/articles/s41591-024-03371-2)

---

## Engineering microbial consortia for controllable outputs.
 02.12.2018, _SR Lindemann, HC Bernstein, HS Song, JK Fredrickson, MW Fields, W Shou, DR Johnson, AS Beliaev_


_Bioengineering, Genomics, Microbial Consortia_
Much research has been invested into engineering microorganisms to perform desired biotransformations; nonetheless, these efforts frequently fall short of expected results due to the unforeseen effects of biofeedback regulation and functional incompatibility. In nature, metabolic function is compartmentalized into diverse organisms assembled into robust consortia, in which the division of labor is thought to lead to increased community efficiency and productivity. Here we consider whether and how consortia can be designed to perform bioprocesses of interest beyond the metabolic flexibility limitations of a single organism. Advances in post-genomic analysis of microbial consortia and application of high-resolution global measurements now offer the promise of systems-level understanding of how microbial consortia adapt to changes in environmental variables and inputs of carbon and energy. We argue that, when combined with appropriate modeling frameworks, systems-level knowledge can markedly improve our ability to predict the fate and functioning of consortia. Here we articulate our collective perspective on the current and future state of microbial community engineering and control while placing specific emphasis on ecological principles that promote control over community function and emergent properties.

[Source](https://www.nature.com/articles/s41564-024-01850-3)

10.1038/ismej.2016.26

---

## Perspective: A framework for understanding collective microbiome metabolism

[Source](https://www.nature.com/articles/s41564-024-01850-3)

---

## http://www.nature.com/articles/s41&amp;#8230

[Source](http://www.nature.com/articles/s41&#8230)

---

## Influence of rubber particle inputs on nitrogen removal efficiency of bioretention systems.
 29.11.2024, _T Hu, W Li, J Xiong, J Zhou, Q Xia_


_Nitrogen, Rubber, Water Pollutants, Chemical, Waste Disposal, Fluid, Denitrification, Bacteria, Ethylenes, Elastomers_
Bioretention systems effectively capture rubber particles and other microplastics in stormwater runoff. However, it is uncertain whether long-term particle accumulation affects pollutant removal efficacy. This study investigated the impact of various concentrations of ethylene-propylene-diene-monomer (EPDM) particles (0, 50, 100, and 400 mg/L) on bioretention system nitrogen removal performance. The input of EPDM during short-duration (2 h) rainfall favored the removal of nitrogen, and the total nitrogen effluent concentration of the bioretention system with EPDM was reduced by 0.59-1.52 mg/L compared with that of the system without EPDM. In addition, the input of EPDM reduced the negative effects of drought. During long-duration (24 h) rainfall, higher concentrations of EPDM led to lower nitrate-nitrogen concentrations in the effluent. The bioretention system with EPDM required less time for nitrate-nitrogen removal to reach 50% than that without EPDM input. Microbial community analysis showed that EPDM increased the relative total abundance of denitrifying bacteria (such as Dechloromonas, Zoogloea, Ramlibacter, and Aeromonas) by 7.25-10.26%, which improved the denitrification capacity of the system.

[Source](https://bsky.app/profile/did:plc:5r53rhz7nxiurzw4mfcmfewf?ref_src=embed)

10.2166/wst.2024.363

---

## Microbial Systems Ecology Lab (@micsysecolab.bsky.social)

[Source](https://bsky.app/profile/did:plc:5r53rhz7nxiurzw4mfcmfewf?ref_src=embed)

---

## Catalase-like activity of perylene diimide based radical anion: Chromogenic substrate for achieving glucose sensing.
 29.11.2024, _R Kaur, P Singh_


In this work, perylene diimide based radical anion (PH2-) is synthesized and characterized using optical, NOBF4 methods; cyclic voltammetry (CV) and differential pulse voltammetry (DPV) techniques. The PH2- is stable for 120 min (2 h) in oxygenated environment and 273 min (4.5 h) in hypoxic conditions. The PH2- showed catalase-like activity to reduce H2O2 to H2O with turn-over number (TON) = 20 and turn-over frequency (TOF) = 40 h-1. The catalase-like activity can be measured using optical and electrochemical methods by monitoring the changes at 726 nm (absorbance); 585 nm (emission) and at 0.27 V. We were able to quantitatively monitor the ultra low-level concentrations of the H2O2 as low as 320 fM (absorbance) and 200 fM (emission). Moreover, PH2- could be used as a chromogenic and fluorogenic substrate for monitoring the low-level concentrations of the glucose using GOx based biochemical assay. We have demonstrated the development and validation of the glucose assay kit for the detection of glucose as low as 3.6/2.8 nM in aqueous medium and 6.2/5.6 nM in blood serum.

[Source](https://bsky.app/profile/did:plc:5r53rhz7nxiurzw4mfcmfewf/post/3lbu2xf2wps2o?ref_src=embed)

10.1016/j.saa.2024.125438

---

## 2024-11-26T12:45:51.587Z

[Source](https://bsky.app/profile/did:plc:5r53rhz7nxiurzw4mfcmfewf/post/3lbu2xf2wps2o?ref_src=embed)

---

## Large scale capsid-mediated mobilisation of bacterial genomic DNA in the gut microbiome

[Source](https://www.biorxiv.org/content/10.1101/2024.11.15.623857v3)

---

## Gut microbiota markers in early childhood are linked to farm living, pets in household and allergy.
 27.11.2024, _A Ljung, M Gio-Batta, B Hesselmar, H Imberg, H Rabe, FL Nowrouzian, S Johansen, CJ Törnhage, G Lindhagen, M Ceder, AC Lundell, A Rudin, AE Wold, I Adlerberth_


_Humans, Pets, Infant, Hypersensitivity, Farms, Animals, Female, Male, Child, Preschool, Gastrointestinal Microbiome, Feces, Child, Infant, Newborn, Family Characteristics, Biomarkers_
Children growing up on farms or with pets have a lower risk of developing allergy, which may be linked to their gut microbiota development during infancy.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0313078)

10.1371/journal.pone.0313078

---

## Gut physiology and environment explain variations in human gut microbiome composition and metabolism 

[Source](https://www.nature.com/articles/s41564-024-01856-x)

---

## Gut physiology and environment explain variations in human gut microbiome composition and metabolism.
 28.11.2024, _N Procházková, MF Laursen, G La Barbera, E Tsekitsidi, MS Jørgensen, MA Rasmussen, J Raes, TR Licht, LO Dragsted, HM Roager_


_Humans, Gastrointestinal Microbiome, Feces, Adult, Hydrogen-Ion Concentration, Male, Female, Bacteria, Gastrointestinal Transit, Young Adult, Middle Aged, Diet, Fermentation, Gastrointestinal Tract, Methane, Healthy Volunteers_
The human gut microbiome is highly personal. However, the contribution of gut physiology and environment to variations in the gut microbiome remains understudied. Here we performed an observational trial using multi-omics to profile microbiome composition and metabolism in 61 healthy adults for 9 consecutive days. We assessed day-to-day changes in gut environmental factors and measured whole-gut and segmental intestinal transit time and pH using a wireless motility capsule in a subset of 50 individuals. We observed substantial daily fluctuations, with intra-individual variations in gut microbiome and metabolism associated with changes in stool moisture and faecal pH, and inter-individual variations accounted for by whole-gut and segmental transit times and pH. Metabolites derived from microbial carbohydrate fermentation correlated negatively with the gut passage time and pH, while proteolytic metabolites and breath methane showed a positive correlation. Finally, we identified associations between segmental transit time/pH and coffee-, diet-, host- and microbial-derived metabolites. Our work suggests that gut physiology and environment are key to understanding the individuality of the human gut microbial composition and metabolism.

[Source](https://bsky.app/profile/did:plc:jyh2n3qwawf2uk6pyg4ulujg?ref_src=embed)

10.1038/s41564-024-01856-x

---

## Henrik M. Roager (@hroager.bsky.social)

[Source](https://bsky.app/profile/did:plc:jyh2n3qwawf2uk6pyg4ulujg?ref_src=embed)

---

## Catalase-like activity of perylene diimide based radical anion: Chromogenic substrate for achieving glucose sensing.
 29.11.2024, _R Kaur, P Singh_


In this work, perylene diimide based radical anion (PH2-) is synthesized and characterized using optical, NOBF4 methods; cyclic voltammetry (CV) and differential pulse voltammetry (DPV) techniques. The PH2- is stable for 120 min (2 h) in oxygenated environment and 273 min (4.5 h) in hypoxic conditions. The PH2- showed catalase-like activity to reduce H2O2 to H2O with turn-over number (TON) = 20 and turn-over frequency (TOF) = 40 h-1. The catalase-like activity can be measured using optical and electrochemical methods by monitoring the changes at 726 nm (absorbance); 585 nm (emission) and at 0.27 V. We were able to quantitatively monitor the ultra low-level concentrations of the H2O2 as low as 320 fM (absorbance) and 200 fM (emission). Moreover, PH2- could be used as a chromogenic and fluorogenic substrate for monitoring the low-level concentrations of the glucose using GOx based biochemical assay. We have demonstrated the development and validation of the glucose assay kit for the detection of glucose as low as 3.6/2.8 nM in aqueous medium and 6.2/5.6 nM in blood serum.

[Source](https://bsky.app/profile/did:plc:jyh2n3qwawf2uk6pyg4ulujg/post/3lbwe7op4gk26?ref_src=embed)

10.1016/j.saa.2024.125438

---

## 2024-11-27T10:36:53.274Z

[Source](https://bsky.app/profile/did:plc:jyh2n3qwawf2uk6pyg4ulujg/post/3lbwe7op4gk26?ref_src=embed)

---

## Significant difference in gut microbiota <em>Bifidobacterium </em>species but not <em>Lactobacillus</em> species in colorectal cancer patients in comparison with healthy volunteers using quantitative real-time PCR

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0294053)

---

## Gut microbiome strain-sharing within isolated village social networks.
 20.11.2024, _F Beghini, J Pullman, M Alexander, SV Shridhar, D Prinster, A Singh, R Matute Juárez, EM Airoldi, IL Brito, NA Christakis_


When humans assemble into face-to-face social networks, they create an extended social environment that permits exposure to the microbiome of others, thereby shaping the composition and diversity of the microbiome at individual and population levels1-6. Here we use comprehensive social network mapping and detailed microbiome sequencing data in 1,787 adults within 18 isolated villages in Honduras7 to investigate the relationship between network structure and gut microbiome composition. Using both species-level and strain-level data, we show that microbial sharing occurs between many relationship types, notably including non-familial and non-household connections. Furthermore, strain-sharing extends to second-degree social connections, suggesting the relevance of a person's broader network. We also observe that socially central people are more microbially similar to the overall village than socially peripheral people. Among 301 people whose microbiome was re-measured 2 years later, we observe greater convergence in strain-sharing in connected versus otherwise similar unconnected co-villagers. Clusters of species and strains occur within clusters of people in village social networks, meaning that social networks provide the social niches within which microbiome biology and phenotypic impact are manifested.

[Source](https://www.nature.com/articles/s41586-024-08222-1)

10.1038/s41586-024-08222-1

---

## Coffee consumption is associated with intestinal<em> Lawsonibacter asaccharolyticus</em> abundance and prevalence across multiple cohort

[Source](https://www.nature.com/articles/s41564-024-01858-9)

---

## Inter-facility characterization of bacteria in seafood processing plants: Exploring potential reservoirs of spoilage organisms and the resistome.
 30.07.2024, _KF Corral-Jara, S Skírnisdóttir, S Knobloch, H Briem, JF Cobo-Díaz, N Carlino, P Bergsten, F Armanini, F Asnicar, F Pinto, A Alvarez-Ordóñez, N Segata, VÞ Marteinsson_


A study was conducted in fish processing facilities to investigate the microbial composition, microbial metabolic potential, and distribution of antibiotic resistance genes. Whole metagenomic sequencing was used to analyze microbial communities from different processing rooms, operators and fish products. Taxonomic analyses identified the genera Pseudomonas and Psychrobacter as the most prevalent bacteria. A Principal Component Analysis revealed a distinct separation between fish product and environmental samples, as well as differences between fish product samples from companies processing either Gadidae or Salmonidae fish. Some particular bacterial genera and species were associated with specific processing rooms and operators. Metabolic analysis of metagenome assembled genomes demonstrated variations in microbiota metabolic profiles of microbiota across rooms and fish products. The study also examined the presence of antibiotic-resistance genes in fish processing environments, contributing to the understanding of microbial dynamics, metabolic potential, and implications for fish spoilage.

[Source](https://bsky.app/profile/did:plc:pcld5no3yjxnsues74juglri?ref_src=embed)

10.1016/j.heliyon.2024.e33866

---

## Segata Lab (@cibiocm.bsky.social)

[Source](https://bsky.app/profile/did:plc:pcld5no3yjxnsues74juglri?ref_src=embed)

---

## Assessing the evolution of pre-hospital combat casualty care: A comparative study of two conflicts a decade apart.
 29.11.2024, _D Epstein, I Radimislensky, AM Lipsky, S Minha, S Kiso, Y Sagie, M Zer, Y Yosef, A Givon, E Katorza, G Twig, B Gordon, S Shapira_


Combat casualty care has advanced significantly with the implementation of evidence-based protocols designed to lower combat-related mortality. Over the last decade, two major urban conflicts in southern Israel have challenged the evolving military trauma system. This study aimed to assess differences in prehospital care and compare the outcomes of aeromedically evacuated casualties from the 2014 and 2023 conflicts.

[Source](https://bsky.app/profile/did:plc:pcld5no3yjxnsues74juglri/post/3lb7soxsvrs2k?ref_src=embed)

10.1016/j.ajem.2024.11.077

---

## 2024-11-18T11:24:44.554Z

[Source](https://bsky.app/profile/did:plc:pcld5no3yjxnsues74juglri/post/3lb7soxsvrs2k?ref_src=embed)

---

## Insights into respiratory microbiome composition and systemic inflammatory biomarkers of bronchiectasis patients.
 13.11.2024, _A Konovalovas, J Armalytė, L Klimkaitė, T Liveikis, B Jonaitytė, E Danila, D Bironaitė, D Mieliauskaitė, E Bagdonas, R Aldonytė_


The human microbiomes, including the ones present in the respiratory tract, are described and characterized in an increasing number of studies. However, the composition and the impact of the healthy and/or impaired microbiome on pulmonary health and its interaction with the host tissues remain enigmatic. In chronic airway diseases, bronchiectasis stands out as a progressive condition characterized by microbial colonization and infection. In this study, we aimed to investigate the microbiome of the lower airways and lungs of bronchiectasis patients together with their serum cytokine and chemokine content, and gain novel insights into the pathogenesis of bronchiectasis. The microbiome of 47 patients was analyzed by sequencing of full-length 16S rRNA gene using amplicon sequencing Oxford Nanopore technologies. Their serum inflammatory mediators content was quantified in parallel. Several divergently composed microbiome groups were identified and characterized, the majority of patients displayed one dominant bacterial species, whereas others had a more diverse microbiome. The analysis of systemic immune biomarkers revealed two distinct inflammatory response groups, i.e., low and high response groups, each associated with a specific array of clinical symptoms, microbial composition, and diversity. Moreover, we have identified some microbiome compositions associated with high inflammatory response, i.e., high levels of pro- and anti-inflammatory cytokines, whereas other microbiomes were in correlation with low inflammatory responses. Although bronchiectasis pathogenetic mechanisms remain to be elucidated, it is clear that addressing microbiome composition in the airways is a valuable resource not only for diagnosis but also for personalized disease management.

[Source](https://journals.asm.org/doi/10.1128/spectrum.04144-23)

10.1128/spectrum.04144-23

---

## Urinary biochemical ecology reveals microbiome-metabolite interactions and metabolic markers of recurrent urinary tract infection.
 01.11.2024, _ML Neugent, NV Hulyalkar, D Ghosh, PE Zimmern, V Shulaev, NJ De Nisco_


Recurrent urinary tract infections (rUTIs) are a major clinical challenge in postmenopausal women and their increasing prevalence underscores the need to define interactions between the host and the urinary microbiome that may underlie rUTI susceptibility. A body of work has identified the taxonomic profile of the female urinary microbiome associate with aging, menopause, and urinay disease. However, how this microbial community engages with the host niche, including the local biochemical environment of the urogenital tract, in health and disease is yet to be fully defined. This study directly assesses differences in the biochemical environment of the urine, or biochemical ecology, associated with recurrent urinary tract infection (UTI) and defines a microbe-metabolite association network of the female urinary microbiome. By integrating metagenomic and metabolomic data collected from a controlled cohort of women with rUTI, we find that distinct metabolites, such as methionine sulfoxide (Met-SO) and trimethylamine oxide (TMAO), are associated with differences in urinary microbiome diversity. We observe associations between microbial and biochemical beta diversity and unique metabolic networks of uropathogenic Escherichia coli and uroprotective Lactobacillus species, highlighting potential metabolite-driven ecological shifts that may influence UTI susceptibility. We identify a urinary lipid signature of active rUTI that can accurately distinguish (AUC = 0.987) cases controls. Finally, using time-to-relapse data we identify deoxycholic acid (DCA) as a new prognostic indicator for rUTI recurrence. Together these findings suggest that systemic metabolic processes may influence susceptibility, opening new avenues for therapeutic intervention and the development of more accurate diagnostic and prognostic to improve patient outcomes.

[Source](https://www.biorxiv.org/content/10.1101/2024.10.22.619727v1)

10.1101/2024.10.22.619727

---

## Distinct maternofetal immune signatures delineate preterm birth onset following urinary tract infection.
 27.11.2024, _S Ottinger, AB Larson, V Mercado-Evans, H Branthoover, JJ Zulk, C Serchejian, ME Mejia, ZA Hameed, R Walde, RC Fleck, AE Shea, KA Patras_


Preterm birth is the leading cause of infant mortality resulting in over one million neonatal deaths annually. Maternal urinary tract infection (UTI) during pregnancy increases risk for preterm birth; however, biological processes mediating UTI-associated preterm birth are not well-described. We established a murine maternal UTI model in which challenge with uropathogenic E. coli resulted in preterm birth in about half of dams. Dams experiencing preterm birth displayed excessive bladder inflammation and altered uteroplacental T cell polarization compared to non-laboring infected dams, with no differences in bacterial burdens. Additional factors associated with preterm birth included higher proportions of male fetuses and lower maternal serum IL-10. Furthermore, exogenous maternal IL-10 treatment absolved UTI-associated preterm birth but contributed to fetal growth restriction in this model. Using urine samples from a cohort of human pregnancies with or without UTI, we correlated urinary cytokines with birth outcomes and urine culture status. These analyses yielded a non-invasive, highly predictive three-model system for evaluating preterm birth risk implicating cytokines IL-10, IL-15, IL-1β, and IL-1RA. Our unique bimodal murine model coupled with patient samples provides a platform to investigate immunological and microbial factors governing UTI-associated preterm birth, revealing novel therapeutic opportunities to predict or prevent preterm birth.

[Source](https://www.biorxiv.org/content/10.1101/2024.10.22.619711v1)

10.1101/2024.10.22.619711

---

## Glucose alters the evolutionary response to gentamicin in uropathogenic <em>Escherichia coli</em>

[Source](https://www.biorxiv.org/content/10.1101/2024.11.26.625364v1)

---

## Longitudinal genomic surveillance of a UK intensive care unit shows a lack of patient colonisation by multi-drug-resistant Gram-negative bacterial pathogens.
 06.11.2024, _AE Snaith, RA Moran, RJ Hall, A Casey, L Ratcliffe, W van Schaik, T Whitehouse, A McNally_


_Humans, Intensive Care Units, Drug Resistance, Multiple, Bacterial, United Kingdom, COVID-19, Escherichia coli, Whole Genome Sequencing, SARS-CoV-2, Cross Infection, Genome, Bacterial, Anti-Bacterial Agents, Gram-Negative Bacteria, Genomics_
Vulnerable patients in an intensive care unit (ICU) setting are at high risk of infection from bacteria including gut-colonising Escherichia coli and Klebsiella species. Complex ICU procedures often depend on successful antimicrobial treatment, underscoring the importance of understanding the extent of patient colonisation by multi-drug-resistant organisms (MDROs) in large UK ICUs. Previous work on ICUs globally uncovered high rates of colonisation by transmission of MDROs, but the situation in UK ICUs is less understood. Here, we investigated the diversity and antibiotic resistance gene (ARG) carriage of bacteria present in one of the largest UK ICUs at the Queen Elizabeth Hospital Birmingham (QEHB), focusing primarily on E. coli as both a widespread commensal and a globally disseminated multi-drug-resistant pathogen. Samples were taken during highly restrictive coronavirus disease 2019 (COVID-19) control measures from May to December 2021. Whole-genome and metagenomic sequencing were used to detect and report strain-level colonisation of patients, focusing on E. coli sequence types (STs), their colonisation dynamics and antimicrobial resistance gene carriage. We found a lack of multi-drug resistance (MDR) in the QEHB. Only one carbapenemase-producing organism was isolated, a Citrobacter carrying bla KPC-2. There was no evidence supporting the spread of this strain, and there was little evidence overall of nosocomial acquisition or circulation of colonising E. coli. Whilst 22 different E. coli STs were identified, only 1 strain of the pandemic ST131 lineage was isolated. This ST131 strain was non-MDR and was found to be a clade A strain, associated with low levels of antibiotic resistance. Overall, the QEHB ICU had very low levels of pandemic or MDR strains, a result that may be influenced in part by the strict COVID-19 control measures in place at the time. Employing some of these infection prevention and control measures where reasonable in all ICUs might therefore assist in maintaining low levels of nosocomial MDR.

[Source](https://bsky.app/profile/did:plc:vf4j77c7bvnyn7lmeeu3byyo?ref_src=embed)

10.1099/mgen.0.001314

---

## Rebecca J Hall (@drrebeccajhall.bsky.social)

[Source](https://bsky.app/profile/did:plc:vf4j77c7bvnyn7lmeeu3byyo?ref_src=embed)

---

## Au nanoflower/Au island hybrid substrate as high-performance SERS platforms.
 29.11.2024, _X Mi, X Zhao, Y Luo, Y Li, Y Wu, S Xue, X Yang, Z Zhang, H Qiao_


Surface-enhanced Raman scattering (SERS) offers significant enhancements to weak Raman signals, which has been widely used for the detection of ultra-low concentrations of molecules. Obtaining repeatable SERS substrates with high density "hot spots" is one of the main challenges for quantitative analysis using SERS. Herein, a SERS substrate based on Au nanoflower/nanoisland (NF/NI) hybrid substrate is constructed. The Au NF/NI substrate can generate "hot spots" by the nanogap between nanotip of Au NF and Au NI, which has been proved to have high stability and sensitivity by experiments, the limit of detection is determined to be down to 10-11 M for Rhodamine 6G (R6G) molecules. In addition, the Au NF/NI hybrid substrate as template can be further extended to prepare bimetallic SERS substrates. This synthetic pathway will not only contribute to the design of new type of SERS substrate, but also promote understanding of the growth mechanisms of multicomponent nanostructure.

[Source](https://bsky.app/profile/did:plc:vf4j77c7bvnyn7lmeeu3byyo/post/3lbw44njddk2q?ref_src=embed)

10.1016/j.saa.2024.125483

---

## 2024-11-27T08:11:55.853Z

[Source](https://bsky.app/profile/did:plc:vf4j77c7bvnyn7lmeeu3byyo/post/3lbw44njddk2q?ref_src=embed)

---

## Ileal microbial microbiome and its secondary bile acids modulate susceptibility to nonalcoholic steatohepatitis in dairy goats.
 25.11.2024, _Y Wang, X Chen, SA Huws, G Xu, J Li, J Ren, J Xu, LL Guan, J Yao, S Wu_


_Animals, Non-alcoholic Fatty Liver Disease, Goats, Bile Acids and Salts, Gastrointestinal Microbiome, Ileum, Liver, Mice, Bacteria, Disease Susceptibility, Goat Diseases, Disease Models, Animal, Female_
Liver damage from nonalcoholic steatohepatitis (NASH) presents a significant challenge to the health and productivity of ruminants. However, the regulatory mechanisms behind variations in NASH susceptibility remain unclear. The gut‒liver axis, particularly the enterohepatic circulation of bile acids (BAs), plays a crucial role in regulating the liver diseases. Since the ileum is the primary site for BAs reabsorption and return to the liver, we analysed the ileal metagenome and metabolome, liver and serum metabolome, and liver single-nuclei transcriptome of NASH-resistant and susceptible goats together with a mice validation model to explore how ileal microbial BAs metabolism affects liver metabolism and immunity, uncovering the key mechanisms behind varied NASH pathogenesis in dairy goats.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01964-0)

10.1186/s40168-024-01964-0

---

## Effects of prebiotic and multispecies probiotic supplementation on the gut microbiota, immune function, and growth performance of weaned piglets.
 23.11.2024, _SOB Huaman, FA de Souza, MA Bonato, CP Dias, MA Callegari, A Oba, RH de Carvalho, CA da Silva_


_Animals, Probiotics, Swine, Prebiotics, Gastrointestinal Microbiome, Weaning, Dietary Supplements, Animal Feed, Saccharomyces cerevisiae, Cytokines_
In this study, we evaluated the impact of yeast cell wall prebiotics and multispecies probiotics on the gut microbiota, immune response, and growth performance of weaned piglets, as alternatives to antibiotics as growth promoters (AGPs). A randomized complete block design was employed, involving 160 piglets divided into four treatment groups during the nursery phase. The treatments applied throughout the experimental period were as follows: CONT+ = basal diet with halquinol (AGP); YCW = basal diet with yeast cell wall (cell wall of Saccharomyces cerevisiae yeast); SIM+ = basal diet with yeast cell wall + multispecies probiotic (Bacillus subtilis (2.0 x 109 CFU/g), Bacillus coagulans (5.0 x 108 CFU/g), Clostridium butyricum (5.0 x 107 CFU/g), and Bacillus licheniformis (2.0 x 109 CFU/g)); SIM- = basal diet with yeast cell wall + multispecies probiotic (half dose). The parameters assessed included daily feed intake, weight gain, feed conversion ratio (FCR), diarrhea score, serum cytokine levels, and chemokine concentrations, as well as microbiota analysis. During the 21 to 63-day study period, only FCR differed significantly (p = 0.0076). CONT+ and PREB had superior FCRs of 1.543 and 1.585, while SIM- had the least favorable FCR at 1.654. At 35 days, IL-10 levels were greater in the SIM- group, showing a 271.25% increase over those in the other groups. By 49 days, the IL-8 concentration was lower in the PREB group than in the CONT+ group, with a reduction of 247%, while the IL-8 concentrations in the SIM+ and SIM- groups were not significantly different from those in the other groups. The Firmicutes/Bacteroidetes (F/B) ratio in the CONT+ group was lower than that in the PREB, SIM+, and SIM- treatment groups. The Lactobacillaceae family was more abundant in the SIM+ treatment, followed by the SIM- and PREB treatments. The CONT+ treatment had the lowest abundance. The abundance of the genus Lactobacillus differed between the CONT+ group and the PREB, SIM+, and SIM- treatment groups. Prebiotics, used either alone or combined with probiotics, serve as effective substitutes for AGPs, boosting piglets' health and performance throughout the nursery phase.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0313475)

10.1371/journal.pone.0313475

---

## Western diet promotes endometriotic lesion growth in mice and induces depletion of <em>Akkermansia muciniphila</em> in intestinal microbiota 

[Source](https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-024-03738-9)

---

## <em>In vitro </em>validation of colon delivery of vitamin B2 through a food grade multi-unit particle system

[Source](https://brill.com/view/journals/bm/aop/article-10.1163-18762891-bja00045/article-10.1163-18762891-bja00045.xml)

---

## Development of a novel complex inflammatory bowel disease mouse model: Reproducing human inflammatory bowel disease etiologies in mice.
 23.11.2024, _SM Seo, NW Kim, ES Yoo, JH Lee, AR Kang, HB Jeong, WY Shim, DH Kim, YJ Park, K Bae, KA Yoon, YK Choi_


_Animals, Disease Models, Animal, Mice, Inflammatory Bowel Diseases, Humans, Dextran Sulfate, Diet, High-Fat, Cytokines, Mice, Inbred C57BL, Citrobacter rodentium, Dysbiosis, Gastrointestinal Microbiome, Colon, Mice, Knockout, Male_
Inflammatory bowel disease (IBD), caused by environmental factors associated with the host's genetic traits, is represented by Crohn's disease and ulcerative colitis. Despite the increasing number of patients with IBD, the current treatment is limited to symptomatic therapy. A complex IBD model mimicking the human IBD etiology is required to overcome this limitation. Herein, we developed novel complex IBD models using interleukin 2 receptor subunit gamma (Il2rg)-deficient mice, high-fat diet, dextran sodium sulfate, and Citrobacter rodentium. The more IBD factors applied complexly, colon length shortened and inflammation worsened. The levels of pro-inflammatory cytokines increased with increased IBD factors. Anti-inflammatory cytokine decreased in all factors application but increased in Il2rg deficiency and Westernized diet combination. Additionally, the pro-inflammatory transcription factors and leaky intestinal epithelial marker were upregulated by a combination of IBD factors. Species diversity decreased with IBD factors. Phylogenetic diversity decreased as IBD factors were applied but increased with combined Il2rg deficiency and Westernized diet. The more IBD factors applied complexly, the more severe the dysbiosis. Finally, we developed a novel complex IBD model using various IBD factors. This model more closely mimic human IBD based on colonic inflammation and dysbiosis than the previous models. Based on these results, our novel complex IBD model could be a valuable tool for further IBD research.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0311310)

10.1371/journal.pone.0311310

---

## Applications of marine microbial community models in the nature-based economy 

[Source](https://journals.plos.org/sustainabilitytransformation/article?id=10.1371/journal.pstr.0000145)

---

## Soil microbiomes show consistent and predictable responses to extreme events.
 28.11.2024, _CG Knight, O Nicolitch, RI Griffiths, T Goodall, B Jones, C Weser, H Langridge, J Davison, A Dellavalle, N Eisenhauer, KB Gongalsky, A Hector, E Jardine, P Kardol, FT Maestre, M Schädler, M Semchenko, C Stevens, MΑ Tsiafouli, O Vilhelmsson, W Wanek, FT de Vries_


Increasing extreme climatic events threaten the functioning of terrestrial ecosystems1,2. Because soil microbes govern key biogeochemical processes, understanding their response to climate extremes is crucial in predicting the consequences for ecosystem functioning3,4. Here we subjected soils from 30 grasslands across Europe to four contrasting extreme climatic events under common controlled conditions (drought, flood, freezing and heat), and compared the response of soil microbial communities and their functioning with those of undisturbed soils. Soil microbiomes exhibited a small, but highly consistent and phylogenetically conserved, response under the imposed extreme events. Heat treatment most strongly impacted soil microbiomes, enhancing dormancy and sporulation genes and decreasing metabolic versatility. Microbiome response to heat in particular could be predicted by local climatic conditions and soil properties, with soils that do not normally experience the extreme conditions being imposed being most vulnerable. Our results suggest that soil microbiomes from different climates share unified responses to extreme climatic events, but that predicting the extent of community change may require knowledge of the local microbiome. These findings advance our understanding of soil microbial responses to extreme events, and provide a first step for making general predictions about the impact of extreme climatic events on soil functioning.

[Source](https://www.nature.com/articles/s41586-024-08185-3)

10.1038/s41586-024-08185-3

---

## A metagenome-assembled genome inventory for children reveals early-life gut bacteriome and virome dynamics.
 26.11.2024, _Y Peng, J Zhu, S Wang, Y Liu, X Liu, O DeLeon, W Zhu, Z Xu, X Zhang, S Zhao, S Liang, H Li, B Ho, JY Ching, CP Cheung, TF Leung, WH Tam, TY Leung, EB Chang, FKL Chan, L Zhang, SC Ng, HM Tun_


Existing microbiota databases are biased toward adult samples, hampering accurate profiling of the infant gut microbiome. Here, we generated a metagenome-assembled genome inventory for children (MAGIC) from a large collection of bulk and viral-like particle-enriched metagenomes from 0 to 7 years of age, encompassing 3,299 prokaryotic and 139,624 viral species-level genomes, 8.5% and 63.9% of which are unique to MAGIC. MAGIC improves early-life microbiome profiling, with the greatest improvement in read mapping observed in Africans. We then identified 54 candidate keystone species, including several Bifidobacterium spp. and four phages, forming guilds that fluctuated in abundance with time. Their abundances were reduced in preterm infants and were associated with childhood allergies. By analyzing the B. longum pangenome, we found evidence of phage-mediated evolution and quorum sensing-related ecological adaptation. Together, the MAGIC database recovers genomes that enable characterization of the dynamics of early-life microbiomes, identification of candidate keystone species, and strain-level study of target species.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312824004025?via%3Dihub)

10.1016/j.chom.2024.10.017

---

## Lefser: Implementation of metagenomic biomarker discovery tool, LEfSe, in R

[Source](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btae707/7908399?searchresult=1)

---

## Evaluation of DNA extraction kits for long-read shotgun metagenomics using Oxford Nanopore sequencing for rapid taxonomic and antimicrobial resistance detection.
 28.11.2024, _S Purushothaman, M Meola, T Roloff, AM Rooney, A Egli_


_Metagenomics, Nanopore Sequencing, Drug Resistance, Bacterial, DNA, Bacterial, Humans, Bacteria_
During a bacterial infection or colonization, the detection of antimicrobial resistance (AMR) is critical, but slow due to culture-based approaches for clinical and screening samples. Culture-based phenotypic AMR detection and confirmation require up to 72 hours (h) or even weeks for slow-growing bacteria. Direct shotgun metagenomics by long-read sequencing using Oxford Nanopore Technologies (ONT) may reduce the time for bacterial species and AMR gene identification. However, screening swabs for metagenomics is complex due to the range of Gram-negative and -positive bacteria, diverse AMR genes, and host DNA present in the samples. Therefore, DNA extraction is a critical initial step. We aimed to compare the performance of different DNA extraction protocols for ONT applications to reliably identify species and AMR genes using a shotgun long-read metagenomic approach. We included three different sample types: ZymoBIOMICS Microbial Community Standard, an in-house mock community of ESKAPE pathogens including Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Escherichia coli (ESKAPE Mock), and anonymized clinical swab samples. We processed all sample types with four different DNA extraction kits utilizing different lysis (enzymatic vs. mechanical) and purification (spin-column vs. magnetic beads) methods. We used kits from Qiagen (QIAamp DNA Mini and QIAamp PowerFecal Pro DNA) and Promega (Maxwell RSC Cultured Cells and Maxwell RSC Buccal Swab DNA). After extraction, samples were subject to the Rapid Barcoding Kit (RBK004) for library preparation followed by sequencing on the GridION with R9.4.1 flow cells. The fast5 files were base called to fastq files using Guppy in High Accuracy (HAC) mode with the inbuilt MinKNOW software. Raw read quality was assessed using NanoPlot and human reads were removed using Minimap2 alignment against the Hg38 genome. Taxonomy identification was performed on the raw reads using Kraken2 and on assembled contigs using Minimap2. The AMR genes were identified using Minimap2 with alignment against the CARD database on both the raw reads and assembled contigs. We identified all bacterial species present in the Zymo Mock Community (8/8) and ESKAPE Mock (6/6) with Qiagen PowerFecal Pro DNA kit (chemical and mechanical lysis) at read and assembly levels. Enzymatic lysis retrieved fewer aligned bases for the Gram-positive species (Staphylococcus aureus and Enterococcus faecium) from the ESKAPE Mock on the assembly level compared to the mechanical lysis. We detected the AMR genes from Gram-negative and -positive species in the ESKAPE Mock with the QIAamp PowerFecal Pro DNA kit on reads level with a maximum median time of 1.9 h of sequencing. Long-read metagenomics with ONT may reduce the turnaround time in screening for AMR genes. Currently, the QIAamp PowerFecal Pro DNA kit (chemical and mechanical lysis) for DNA extraction along with the Rapid Barcoding Kit for the ONT sequencing captured the best taxonomy and AMR identification for our specific use case.

[Source](https://www.nature.com/articles/s41598-024-80660-3)

10.1038/s41598-024-80660-3

---

## Gardnerella pickettii sp. nov. (formerly Gardnerella genomic species 3) and Gardnerella greenwoodii sp. nov. (formerly Gardnerella genomic species 8) isolated from female urinary microbiome.
 06.11.2023, _M Sousa, M Ksiezarek, SU Perovic, T Antunes-Lopes, F Grosso, TG Ribeiro, L Peixe_


_Female, Humans, Gardnerella, Fatty Acids, Sequence Analysis, DNA, Phylogeny, RNA, Ribosomal, 16S, Base Composition, DNA, Bacterial, Bacterial Typing Techniques, Genomics, Nucleic Acid Hybridization, Microbiota_
During an ongoing female urinary microbiome research study, strains c17Ua_112T and c31Ua_26T isolated from urine samples of a patient diagnosed with overactive bladder and a healthy postmenopausal woman, respectively, could not be allocated to any Gardnerella species with valid names. In this work, we aimed to characterize these strains. The 16S rRNA gene sequences confirmed that these strains are members of the genus Gardnerella. Phylogenetic analysis based on cpn60 strongly supported two clades, one encompassing c17Ua_112T and nine other strains from the public database, and the other including c31Ua_26T and three other strains, which were distinct from currently recognized species of the genus Gardnerella. Likewise, the phylogenomic tree also showed that strains c17Ua_112T and c31Ua_26T formed independent and robust clusters. Average nucleotide identity (ANI) and digital DNA-DNA hybridization (dDDH) values between c17Ua_112T and c31Ua_26T were 79.27 and 27.4 %, respectively. Strain c17Ua_112T showed the highest ANI (94.8 %) and dDDH values (59.8 %) with Gardnerella piotii UGent 18.01T, and strain c31Ua_26T revealed highest ANI (84.2 %) and dDDH (29.1 %) values with Gardnerella swidsinskii GS 9838-1T. Based on the data presented here, the two strains c17Ua_112T and c31Ua_26T represent two novel species of the genus Gardnerella, for which the names Gardnerella pickettii (c17Ua_112T=DSM 113414T=CCP 71T) and Gardnerella greenwoodii (c31Ua_26T=DSM 113415T=CCP 72T) are proposed.

[Source](https://bsky.app/profile/did:plc:hffa64ymkjg2wqddo4q3jjrs?ref_src=embed)

10.1099/ijsem.0.006140

---

## Svetlana Ugarcina Perovic (@svetlanaup.bsky.social)

[Source](https://bsky.app/profile/did:plc:hffa64ymkjg2wqddo4q3jjrs?ref_src=embed)

---

## A new difunctional nanopolypropylene surface molecularly imprinted polyacrylamide catalytic probe for determination of trace sulfadiazine with resonance Rayleigh scattering technique.
 29.11.2024, _L Lin, A Liang, G Wen, Z Jiang_


A new nanopolypropylene (PP) surface molecularly imprinted polyacrylamide (PP@MIP) probe was synthesized facilely for the Resonance Rayleigh Scattering (RRS) determination of trace sulfadiazine (SDZ), using PP as nanosubstrate, SDZ as template molecule, and the acrylamide as functional monomer. The catalytic effect of PP@MIP on the reduction of HAuCl4 by hydrazine hydrate to form AuNP, a nano-indicated reaction, was investigated. The generated AuNPs had a strong RRS effect, and the addition of SDZ resulted in the selective formation of PP@MIP-SDZ complex, which further promoted the generation of AuNP and increased the RRS signal. Therefore, there was developed a novel RRS assay platform for rapid, selective and sensitive detection of SDZ with a linearity range of 0.0125-0.150 nmol/L and limit of detection of 0.004 nmol/L. The recoveries were in the range of 93.3-109.9 % with the relative standard deviations (RSDs) in the range of 2.55-9.14 % in the real sample analysis.

[Source](https://bsky.app/profile/did:plc:hffa64ymkjg2wqddo4q3jjrs/post/3lbihe4otjs2l?ref_src=embed)

10.1016/j.saa.2024.125487

---

## 2024-11-21T21:55:47.051Z

[Source](https://bsky.app/profile/did:plc:hffa64ymkjg2wqddo4q3jjrs/post/3lbihe4otjs2l?ref_src=embed)

---

