## Diagnostic and prognostic potential of the oral and gut microbiome for lung adenocarcinoma.
 02.10.2021, _MY Lim, S Hong, KH Hwang, EJ Lim, JY Han, YD Nam_



[Source](https://onlinelibrary.wiley.com/doi/10.1002/ctm2.508)

10.1002/ctm2.508

---

## Individuals with substance use disorders have a distinct oral microbiome pattern

[Source](https://www.sciencedirect.com/science/article/pii/S2666354621000740)

---

## The Dynamics of Skin Microbiota in Shoulder Surgery Infections.
 29.09.2021, _N Fatima, T Bjarnsholt, L Bay_


Post-surgical infections arise due to various contributing factors. Most important is the presence of potential pathogenic microorganisms in the skin complemented by the patient's health status. Cutibacterium acnes is commonly present in the pilosebaceous glands and hair follicle funnels in human skin. After surgical intervention, these highly prevalent, slow-growing bacteria can be found in the deeper tissues and in proximity of implants. C. acnes is frequently implicated in post-surgical infections, often resulting in the need for revision surgery. This review summarizes the current understanding of microbial dynamics in shoulder surgical infections. In particular, we shed light on the contribution of C. acnes to post-surgical shoulder infections as well as their colonization and immune modulatory potential. Despite being persistently found in post-surgical tissues, C. acnes is often underestimated as a causative organism due to its slow growth and the inefficient detection methods. We discuss the role of the skin environment constituted by microbial composition and host cellular status in influencing C. acnes recolonization potential. Future mapping of the individual skin microbiome in shoulder surgery patients using advanced molecular methods would be a useful approach for determining the risk of post-operative infections.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/apm.13185)

10.1111/apm.13185

---

## Mucins, gut microbiota, and postbiotics role in colorectal cancer.
 30.09.2021, _R Pothuraju, S Chaudhary, S Rachagani, S Kaur, HK Roy, M Bouvet, SK Batra_


An imbalance in the crosstalk between the host and gut microbiota affects the intestinal barrier function, which results in inflammatory diseases and colorectal cancer. The colon epithelium protects itself from a harsh environment and various pathogenic organisms by forming a double mucus layer, primarily comprising mucins. Recent studies are focusing on how dietary patterns alter the gut microbiota composition, which in turn regulates mucin expression and maintains the intestinal layers. In addition, modulation of gut microbiota by microbiotic therapy (involving fecal microbiota transplantation) has emerged as a significant factor in the pathologies associated with dysbiosis. Therefore, proper communication between host and gut microbiota via different dietary patterns (prebiotics and probiotics) is needed to maintain mucus composition, mucin synthesis, and regulation. Here, we review how the interactions between diet and gut microbiota and bacterial metabolites (postbiotics) regulate mucus layer functionalities and mucin expression in human health and disease.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2021.1974795)

10.1080/19490976.2021.1974795

---

## Age and the aging process significantly alter the small bowel microbiome.
 30.09.2021, _G Leite, M Pimentel, GM Barlow, C Chang, A Hosseini, J Wang, G Parodi, R Sedighi, A Rezaie, R Mathur_


Gut microbial diversity decreases with aging, but existing studies have used stool samples, which do not represent the entire gut. We analyzed the duodenal microbiome in 251 subjects aged 18-35 (n = 32), 36-50 (n = 41), 51-65 (n = 96), and 66-80 (n = 82). Decreased duodenal microbial diversity in older subjects is associated with combinations of chronological age, number of concomitant diseases, and number of medications used, and also correlated with increasing coliform numbers (p &lt; 0.0001). Relative abundance (RA) of phylum Proteobacteria increases in older subjects, with increased RA of family Enterobacteriaceae and coliform genera Escherichia and Klebsiella, and is associated with alterations in the RA of other duodenal microbial taxa and decreased microbial diversity. Increased RA of specific genera are associated with chronological age only (Escherichia, Lactobacillus, and Enterococcus), number of medications only (Klebsiella), or number of concomitant diseases only (Clostridium and Bilophila). These findings indicate the small intestinal microbiome changes significantly with age and the aging process.

[Source](https://www.sciencedirect.com/science/article/pii/S2211124721012195)

10.1016/j.celrep.2021.109765

---

## Guidelines for reporting on animal fecal transplantation (GRAFT) studies: recommendations from a systematic review of murine transplantation protocols.
 30.09.2021, _KR Secombe, GH Al-Qadami, CB Subramaniam, JM Bowen, J Scott, YZA Van Sebille, M Snelson, C Cowan, G Clarke, CE Gheorghe, JF Cryan, HR Wardill_


Fecal microbiota transplant (FMT) is a powerful tool used to connect changes in gut microbial composition with a variety of disease states and pathologies. While FMT enables potential causal relationships to be identified, the experimental details reported in preclinical FMT protocols are highly inconsistent and/or incomplete. This limitation reflects a current lack of authoritative guidance on reporting standards that would facilitate replication efforts and ultimately reproducible science. We therefore systematically reviewed all FMT protocols used in mouse models with the goal of formulating recommendations on the reporting of preclinical FMT protocols. Search strategies were applied across three databases (PubMed, EMBASE, and Ovid Medline) until June 30, 2020. Data related to donor attributes, stool collection, processing/storage, recipient preparation, administration, and quality control were extracted. A total of 1753 papers were identified, with 241 identified for data extraction and analysis. Of the papers included, 92.5% reported a positive outcome with FMT intervention. However, the vast majority of studies failed to address core methodological aspects including the use of anaerobic conditions (91.7% of papers lacked information), storage (49.4%), homogenization (33.6%), concentration (31.5%), volume (19.9%) and administration route (5.3%). To address these reporting limitations, we developed theGuidelines for Reporting Animal Fecal Transplant (GRAFT) that guide reporting standards for preclinical FMT. The GRAFT recommendations will enable robust reporting of preclinical FMT design, and facilitate high-quality peer review, improving the rigor and translation of knowledge gained through preclinical FMT studies.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2021.1979878)

10.1080/19490976.2021.1979878

---

## Alpha-synuclein alters the faecal viromes of rats in a gut-initiated model of Parkinson’s disease

[Source](https://www.nature.com/articles/s42003-021-02666-1)

---

## First trimester gut microbiome induces Inflammation-dependent gestational diabetes phenotype in mice

[Source](https://www.medrxiv.org/content/10.1101/2021.09.17.21262268v1)

---

## High-resolution genetic mapping of the mucosa-associated gut microbiome in hybrid mice provides novel insight on the impact of host genetics

[Source](https://www.biorxiv.org/content/10.1101/2021.09.28.462095v1)

---

## Effects of microplastics on soil microbiome: The impacts of polymer type, shape, and concentration.
 03.10.2021, _Y Sun, C Duan, N Cao, X Li, X Li, Y Chen, Y Huang, J Wang_


Increasing research has recognized that the ubiquitous presence of microplastics in terrestrial environments is undeniable, which potentially alters the soil ecosystem properties and processes. The fact that microplastics with diverse characteristics enter into the soil may induce distinct effects on soil ecosystems. Our knowledge of the impacts of microplastics with different polymers, shapes, and concentrations on soil bacterial communities is still limited. To address this, we examined the effects of spherical microplastics (150 μm) with different polymers (i.e., polyethylene (PE), polystyrene (PS), and polypropylene (PP)) and four shapes of PP microplastics (i.e., fiber, film, foam, and particle) at a constant concentration (1%, w/w) on the soil bacterial community in an agricultural soil over 60 days. Treatments with different concentrations (0.01-20%, w/w) of PP microplastic particles (150 μm) were also included. The bacterial communities in PE and PP treatments showed a similar pattern but separated from those in PS-treated soils, indicating the polymer backbone structure is an important factor modulating the soil bacterial responses. Fiber, foam, and film microplastics significantly affected the soil bacterial composition as compared to the particle. The community dissimilarity of soil bacteria was significantly (R2 = 0.592, p &lt; 0.001) correlated with the changes of microplastic concentration. The random forest model identified that certain bacteria belonging to Patescibacteria were closely linked to microplastic contamination. Additionally, analysis of the predicted function further showed that microplastics with different characteristics caused distinct effects on microbial community function. Our findings suggested that the idiosyncrasies of microplastics should not be neglected when studying their effects on terrestrial ecosystems.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969721055935)

10.1016/j.scitotenv.2021.150516

---

## Long-read metagenomics of multiple displacement amplified DNA of low-biomass human gut phageomes by SACRA preprocessing chimeric reads.
 29.09.2021, _Y Kiguchi, S Nishijima, N Kumar, M Hattori, W Suda_


The human gut bacteriophage community (phageome) plays an important role in the host's health and disease; however, the entire structure is poorly understood, partly owing to the generation of many incomplete genomes in conventional short-read metagenomics. Here, we show long-read metagenomics of amplified DNA of low-biomass phageomes with multiple displacement amplification (MDA), involving the development of a novel bioinformatics tool, SACRA, that efficiently preprocessed numerous chimeric reads generated through MDA. Using five samples, SACRA markedly reduced the average chimera ratio from 72% to 1.5% in PacBio reads with an average length of 1.8-kb. De novo assembly of chimera-less PacBio long reads reconstructed contigs of ≥ 5-kb with an average proportion of 27%, which was 1% in contigs from MiSeq short reads, thereby dramatically improving contig length and genome completeness. Comparison of PacBio and MiSeq contigs found MiSeq contig fragmentations frequently near local repeats and hypervariable regions in the phage genomes, and those caused by multiple homologous phage genomes coexisting in the community. We also developed a reference-independent method to assess the completeness of the linear phage genomes. Overall, we established a SACRA-coupled long-read metagenomics robust to highly diverse gut phageomes, identifying high-quality circular and linear phage genomes with adequate sequence quantity.

[Source](https://academic.oup.com/dnaresearch/advance-article/doi/10.1093/dnares/dsab019/6377780)

10.1093/dnares/dsab019

---

## BiG-MAP: an Automated Pipeline To Profile Metabolic Gene Cluster Abundance and Expression in Microbiomes.
 28.09.2021, _V Pascal Andreu, HE Augustijn, K van den Berg, JJJ van der Hooft, MA Fischbach, MH Medema_


Microbial gene clusters encoding the biosynthesis of primary and secondary metabolites play key roles in shaping microbial ecosystems and driving microbiome-associated phenotypes. Although effective approaches exist to evaluate the metabolic potential of such bacteria through identification of these metabolic gene clusters in their genomes, no automated pipelines exist to profile the abundance and expression levels of such gene clusters in microbiome samples to generate hypotheses about their functional roles, and to find associations with phenotypes of interest. Here, we describe BiG-MAP, a bioinformatic tool to profile abundance and expression levels of gene clusters across metagenomic and metatranscriptomic data and evaluate their differential abundance and expression under different conditions. To illustrate its usefulness, we analyzed 96 metagenomic samples from healthy and caries-associated human oral microbiome samples and identified 252 gene clusters, including unreported ones, that were significantly more abundant in either phenotype. Among them, we found the muc operon, a gene cluster known to be associated with tooth decay. Additionally, we found a putative reuterin biosynthetic gene cluster from a Streptococcus strain to be enriched but not exclusively found in healthy samples; metabolomic data from the same samples showed masses with fragmentation patterns consistent with (poly)acrolein, which is known to spontaneously form from the products of the reuterin pathway and has been previously shown to inhibit pathogenic Streptococcus mutans strains. Thus, we show how BiG-MAP can be used to generate new hypotheses on potential drivers of microbiome-associated phenotypes and prioritize the experimental characterization of relevant gene clusters that may mediate them. IMPORTANCE Microbes play an increasingly recognized role in determining host-associated phenotypes by producing small molecules that interact with other microorganisms or host cells. The production of these molecules is often encoded in syntenic genomic regions, also known as gene clusters. With the increasing numbers of (multi)omics data sets that can help in understanding complex ecosystems at a much deeper level, there is a need to create tools that can automate the process of analyzing these gene clusters across omics data sets. This report presents a new software tool called BiG-MAP, which allows assessing gene cluster abundance and expression in microbiome samples using metagenomic and metatranscriptomic data. Here, we describe the tool and its functionalities, as well as its validation using a mock community. Finally, using an oral microbiome data set, we show how it can be used to generate hypotheses regarding the functional roles of gene clusters in mediating host phenotypes.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00937-21)

10.1128/mSystems.00937-21

---

## Increasing the power of interpretation for soil metaproteomics data.
 30.09.2021, _V Jouffret, G Miotello, K Culotta, S Ayrault, O Pible, J Armengaud_


Soil and sediment microorganisms are highly phylogenetically diverse but are currently largely under-represented in public molecular databases. Their functional characterization by means of metaproteomics is usually performed using metagenomic sequences acquired for the same sample. However, such hugely diverse metagenomic datasets are difficult to assemble; in parallel, theoretical proteomes from isolates available in generic databases are of high quality. Both these factors advocate for the use of theoretical proteomes in metaproteomics interpretation pipelines. Here, we examined a number of database construction strategies with a view to increasing the outputs of metaproteomics studies performed on soil samples.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01139-1)

10.1186/s40168-021-01139-1

---

