## The longitudinal and cross-sectional heterogeneity of the intestinal microbiota.
 24.08.2021, _BM Miller, MJ Liou, JY Lee, AJ Bäumler_


A central goal of microbiome research is to understand the factors that balance gut-associated microbial communities, thereby creating longitudinal and cross-sectional heterogeneity in their composition and density. Whereas the diet dictates taxa dominance, microbial communities are linked intimately to host physiology through digestive and absorptive functions that generate longitudinal heterogeneity in nutrient availability. Additionally, the host differentially controls the access to electron acceptors along the longitudinal axis of the intestine to drive the development of microbial communities that are dominated by facultatively anaerobic bacteria in the small intestine or obligately anaerobic bacteria in the large intestine. By secreting mucus and antimicrobials, the host further constructs microhabitats that generate cross-sectional heterogeneity in the colonic microbiota composition. Here we will review how understanding the host factors involved in generating longitudinal and cross-sectional microbiota heterogeneity helps define physiological states that are characteristic of or appropriate to a homeostatic microbiome.

[Source](https://www.sciencedirect.com/science/article/pii/S1369527421001089)

10.1016/j.mib.2021.08.004

---

## [REVIEW]The longitudinal and cross-sectional heterogeneity of the intestinal microbiota

[Source](https://www.sciencedirect.com/science/article/pii/S1369527421001089)

---

## Early life gut microbiota is associated with rapid infant growth in Hispanics from Southern California.
 23.08.2021, _TL Alderete, RB Jones, JP Shaffer, EA Holzhausen, WB Patterson, E Kazemian, L Chatzi, R Knight, JF Plows, PK Berger, MI Goran_


We aimed to determine if the newborn gut microbiota is an underlying determinant of early life growth trajectories. 132 Hispanic infants were recruited at 1-month postpartum. The infant gut microbiome was characterized using 16S rRNA amplicon sequencing. Rapid infant growth was defined as a weight-for-age z-score (WAZ) change greater than 0.67 between birth and 12-months of age. Measures of infant growth included change in WAZ, weight-for-length z-score (WLZ), and body mass index (BMI) z-scores from birth to 12-months and infant anthropometrics at 12-months (weight, skinfold thickness). Of the 132 infants, 40% had rapid growth in the first year of life. Multiple metrics of alpha-diversity predicted rapid infant growth, including a higher Shannon diversity (OR = 1.83; 95% CI: 1.07-3.29; p = .03), Faith's phylogenic diversity (OR = 1.41, 95% CI: 1.05-1.94; p = .03), and richness (OR = 1.04, 95% CI: 1.01-1.08; p = .02). Many of these alpha-diversity metrics were also positively associated with increases in WAZ, WLZ, and BMI z-scores from birth to 12-months (pall&lt;0.05). Importantly, we identified subsets of microbial consortia whose abundance were correlated with these same measures of infant growth. We also found that rapid growers were enriched in multiple taxa belonging to genera such as Acinetobacter, Collinsella, Enterococcus, Neisseria, and Parabacteroides. Moreover, measures of the newborn gut microbiota explained up to an additional 5% of the variance in rapid growth beyond known clinical predictors (R2 = 0.37 vs. 0.32, p &lt; .01). These findings indicate that a more mature gut microbiota, characterized by increased alpha-diversity, at as early as 1-month of age, may influence infant growth trajectories in the first year of life.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2021.1961203)

10.1080/19490976.2021.1961203

---

## Characterization of the Human Oropharyngeal Microbiomes in SARS-CoV-2 Infection and Recovery Patients.
 23.08.2021, _M Gao, H Wang, H Luo, Y Sun, L Wang, S Ding, H Ren, J Gang, B Rao, S Liu, X Wang, X Gao, M Li, Y Zou, C Liu, C Yuan, J Sun, G Cui, Z Ren_


Respiratory tract microbiome is closely related to respiratory tract infections, while characterization of oropharyngeal microbiome in recovered coronavirus disease 2019 (COVID-19) patients is not studied. Herein, oropharyngeal swabs are collected from confirmed cases (CCs) with COVID-19 (73 subjects), suspected cases (SCs) (36), confirmed cases who recovered (21), suspected cases who recovered (36), and healthy controls (Hs) (140) and then completed MiSeq sequencing. Oropharyngeal microbial α-diversity is markedly reduced in CCs versus Hs. Opportunistic pathogens are increased, while butyrate-producing genera are decreased in CCs versus Hs. The classifier based on eight optimal microbial markers is constructed through a random forest model and reached great diagnostic efficacy in both discovery and validation cohorts. Notably, the classifier successfully diagnosed SCs with positive IgG antibody as CCs and is demonstrated from the perspective of the microbiome. Importantly, several genera with significant differences gradually increase and decrease along with recovery from COVID-19. Forty-four oropharyngeal operational taxonomy units (OTUs) are closely correlated with 11 clinical indicators of SARS-CoV-2 infection and Hs based on Spearman correlation analysis. Together, this research is the first to characterize oropharyngeal microbiota in recovered COVID-19 cases and suspected cases, to successfully construct and validate the diagnostic model for COVID-19 and to depict the correlations between microbial OTUs and clinical indicators.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/advs.202102785)

10.1002/advs.202102785

---

## Metagenome-wide association study revealed disease-specific landscape of the gut microbiome of systemic lupus erythematosus in Japanese.
 24.08.2021, _Y Tomofuji, Y Maeda, E Oguro-Igashira, T Kishikawa, K Yamamoto, K Sonehara, D Motooka, Y Matsumoto, H Matsuoka, M Yoshimura, M Yagita, T Nii, S Ohshima, S Nakamura, H Inohara, K Takeda, A Kumanogoh, Y Okada_


Alteration of the gut microbiome has been linked to the pathogenesis of systemic lupus erythematosus (SLE). However, a comprehensive view of the gut microbiome in SLE and its interaction with the host remains to be revealed. This study aimed to reveal SLE-associated changes in the gut microbiome and its interaction with the host by a comprehensive metagenome-wide association study (MWAS) followed by integrative analysis.

[Source](https://ard.bmj.com/content/early/2021/08/17/annrheumdis-2021-220687)

10.1136/annrheumdis-2021-220687

---

## Gut microbes participate in food preference alterations during obesity.
 23.08.2021, _A de Wouters d'Oplinter, M Rastelli, M Van Hul, NM Delzenne, PD Cani, A Everard_


Hypothalamic regulations of food intake are altered during obesity. The dopaminergic mesocorticolimbic system, responsible for the hedonic response to food intake, is also affected. Gut microbes are other key players involved in obesity. Therefore, we investigated whether the gut microbiota plays a causal role in hedonic food intake alterations contributing to obesity. We transferred fecal material from lean or diet-induced obese mice into recipient mice and evaluated the hedonic food intake using a food preference test comparing the intake of control and palatable diets (HFHS, High-Fat High-Sucrose) in donor and recipient mice. Obese mice ate 58% less HFHS during the food preference test (p &lt; 0.0001) than the lean donors, suggesting a dysregulation of the hedonic food intake during obesity. Strikingly, the reduction of the pleasure induced by eating during obesity was transferable through gut microbiota transplantation since obese gut microbiota recipient mice exhibited similar reduction in HFHS intake during the food preference test (40% reduction as compared to lean gut microbiota recipient mice, p &lt; 0.01). This effect was associated with a consistent trend in modifications of dopaminergic markers expression in the striatum. We also pinpointed a highly positive correlation between HFHS intake and Parabacteroides (p &lt; 0.0001), which could represent a potential actor involved in hedonic feeding probably through the gut-to-brain axis. We further demonstrated the key roles played by gut microbes in this paradigm since depletion of gut microbiota using broad-spectrum antibiotics also altered HFHS intake during food preference test in lean mice. In conclusion, we discovered that gut microbes regulate hedonic aspects of food intake. Our data demonstrate that gut microbiota modifications associated with obesity participate in dysregulations of the reward and hedonic components of the food intake. These data provide evidence that gut microbes could be an interesting therapeutic target to tackle hedonic disorders related to obesity.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2021.1959242)

10.1080/19490976.2021.1959242

---

## Altered gut microbiome compositions are associated with the severity of asthma.
 24.08.2021, _Z Wang, Z Lai, X Zhang, P Huang, J Xie, Q Jiang, Q Zhang, KF Chung_


Despite substantial evidence on the contribution of the diversity of the gut microbiome to the pathogenesis of asthma and allergic diseases, little is known about their relationship with asthma severity and/or clinical phenotypes. We analyzed the difference in composition of the gut microbiome between subjects with asthma and healthy subjects and explored its role in the development of asthma.

[Source](https://jtd.amegroups.com/article/view/54245/html)

10.21037/jtd-20-2189

---

## Evolution in a Community Context: towards Understanding the Causes and Consequences of Adaptive Evolution in the Human Gut Microbiota over Short Time Scales

[Source](https://journals.asm.org/doi/10.1128/mSystems.00832-21)

---

## Propamocarb exposure has the potential to accelerate the formation of atherosclerosis in both WT and ApoE−/− mice accompanied by gut microbiota dysbiosis

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969721046775)

---

## Inflammation-relevant microbiome signature of the stroke brain, gut, spleen, and thymus and the impact of exercise.
 24.08.2021, _C Kingsbury, A Shear, M Heyck, N Sadanandan, H Zhang, B Gonzales-Portillo, B Cozene, M Sheyner, L Navarro-Torres, J García-Sánchez, JY Lee, CV Borlongan_


Stroke remains a significant unmet need in the clinic with few therapeutic options. We, and others, have implicated the role of inflammatory microbiota in stroke secondary cell death. Elucidating this inflammation microbiome as a biomarker may improve stroke diagnosis and treatment. Here, adult Sprague-Dawley rats performed 30 minutes of exercise on a motorized treadmill for 3 consecutive days prior to transient middle cerebral artery occlusion (MCAO). Stroke animals that underwent exercise showed 1) robust behavioral improvements, 2) significantly smaller infarct sizes and increased peri-infarct cell survival and 3) decreasing trends of inflammatory microbiota BAC303, EREC482, and LAB158 coupled with significantly reduced levels of inflammatory markers ionized calcium binding adaptor molecule 1, tumor necrosis factor alpha, and mouse monoclonal MHC Class II RT1B in the brain, gut, spleen, and thymus compared to non-exercised stroke rats. These results suggest that a specific set of inflammatory microbiota exists in central and peripheral organs and can serve as a disease biomarker and a therapeutic target for stroke.

[Source](https://journals.sagepub.com/doi/10.1177/0271678X211039598)

10.1177/0271678X211039598

---

## Comparative Analysis of Skin Bacterial Diversity and Its Potential Antifungal Function Between Desert and Pine Forest Populations of Boreal Toads Anaxyrus boreas.
 24.08.2021, _MD Basanta, EA Rebollar, MG García-Castillo, G Parra Olea_


The skin microbiome in amphibians has gained a lot of attention as some of its members play a protective role against pathogens such as the fungus Batrachochytrium dendrobatidis (Bd). The composition of skin bacterial communities has been suggested as one of the factors explaining differences in susceptibility to Bd among amphibian species and populations. The boreal toad Anaxyrus boreas is known to be susceptible to Bd, and severe population declines in its southeastern range have been documented. However, throughout A. boreas distribution, populations present differences in susceptibility to Bd infections which may be associated with differences in skin microbial diversity. This study compared the skin bacterial diversity and Bd infection levels of A. boreas in one desert population and one pine forest population from Baja California, Mexico. We found that desert and pine forest toad populations exhibit differences in skin bacterial community structure but show similar Bd infection levels. Using a predictive method, we found that the abundance of bacteria with potential Bd-inhibitory properties differed between uninfected and infected individuals but not between populations. Our data suggest that several bacteria in the skin community may be offering protection from Bd infections in these A. boreas populations. This study provides foundational evidence for future studies seeking to understand the skin-microbial variation among boreal toads' populations and its relation with Bd susceptibility.

[Source](https://link.springer.com/article/10.1007%2Fs00248-021-01845-1)

10.1007/s00248-021-01845-1

---

## Herbivory shapes the rhizosphere bacterial microbiota in potato plants.
 24.08.2021, _A Malacrinò, M Wang, S Caul, AJ Karley, AE Bennett_


Plant-associated microbiomes assist their host in a variety of activities, spanning from nutrition to defence against herbivores and diseases. Previous research showed that plant-associated microbiomes shift their composition when plants are exposed to stressors, including herbivory. However, existing studies explored only single herbivore-plant combinations, whereas plants are often attacked by several different herbivores, but the effects of multiple herbivore types on the plant microbiome remain to be determined. Here, we first tested whether feeding by different herbivores (aphids, nematodes and slugs) produces a shift in the rhizosphere bacterial microbiota associated with potato plants. Then, we expanded this question asking whether the identity of the herbivore produces different effects on the rhizosphere microbial community. While we found shifts in microbial diversity and structure due to herbivory, we observed that the herbivore identity does not influence the diversity or community structure of bacteria thriving in the rhizosphere. However, a deeper analysis revealed that the herbivores differentially affected the structure of the network of microbial co-occurrences. Our results have the potential to increase our ability to predict how plant microbiomes assemble and aid our understanding of the role of plant microbiome in plant responses to biotic stress.

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1758-2229.12998)

10.1111/1758-2229.12998

---

## Teasing apart the host-related, nutrient-related and temperature-related effects shaping the phenology and microbiome of the tropical seagrass Halophila stipulacea

[Source](https://www.biorxiv.org/content/10.1101/2021.08.21.457214v1.abstract)

---

## The Effect of Common Viral Inactivation Techniques on 16S rRNA Amplicon-Based Analysis of the Gut Microbiota

[Source](https://www.mdpi.com/2076-2607/9/8/1755)

---

## Comprehensive Wet-Bench and Bioinformatics Workflow for Complex Microbiota Using Oxford Nanopore Technologies.
 24.08.2021, _C Ammer-Herrmenau, N Pfisterer, T van den Berg, I Gavrilova, A Amanzada, SK Singh, A Khalil, R Alili, E Belda, K Clement, A Abd El Wahed, EE Gady, M Haubrock, T Beißbarth, V Ellenrieder, A Neesse_


The advent of high-throughput sequencing techniques has recently provided an astonishing insight into the composition and function of the human microbiome. Next-generation sequencing (NGS) has become the gold standard for advanced microbiome analysis; however, 3rd generation real-time sequencing, such as Oxford Nanopore Technologies (ONT), enables rapid sequencing from several kilobases to &gt;2 Mb with high resolution. Despite the wide availability and the enormous potential for clinical and translational applications, ONT is poorly standardized in terms of sampling and storage conditions, DNA extraction, library creation, and bioinformatic classification. Here, we present a comprehensive analysis pipeline with sampling, storage, DNA extraction, library preparation, and bioinformatic evaluation for complex microbiomes sequenced with ONT. Our findings from buccal and rectal swabs and DNA extraction experiments indicate that methods that were approved for NGS microbiome analysis cannot be simply adapted to ONT. We recommend using swabs and DNA extractions protocols with extended washing steps. Both 16S rRNA and metagenomic sequencing achieved reliable and reproducible results. Our benchmarking experiments reveal thresholds for analysis parameters that achieved excellent precision, recall, and area under the precision recall values and is superior to existing classifiers (Kraken2, Kaiju, and MetaMaps). Hence, our workflow provides an experimental and bioinformatic pipeline to perform a highly accurate analysis of complex microbial structures from buccal and rectal swabs. IMPORTANCE Advanced microbiome analysis relies on sequencing of short DNA fragments from microorganisms like bacteria, fungi, and viruses. More recently, long fragment DNA sequencing of 3rd generation sequencing has gained increasing importance and can be rapidly conducted within a few hours due to its potential real-time sequencing. However, the analysis and correct identification of the microbiome relies on a multitude of factors, such as the method of sampling, DNA extraction, sequencing, and bioinformatic analysis. Scientists have used different protocols in the past that do not allow us to compare results across different studies and research fields. Here, we provide a comprehensive workflow from DNA extraction, sequencing, and bioinformatic workflow that allows rapid and accurate analysis of human buccal and rectal swabs with reproducible protocols. This workflow can be readily applied by many scientists from various research fields that aim to use long-fragment microbiome sequencing.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00750-21)

10.1128/mSystems.00750-21

---

