## A Sedentary Lifestyle Changes the Composition and Predicted Functions of the Gut Bacterial and Fungal Microbiota of Subjects from the Same Company.
 16.10.2023, _L Xu, W Li, L Ling, Z Zhang, Z Cui, J Ge, Y Wang, Q Meng, Y Wang, K Liu, J Zhou, F Zeng, J Wang, J Wu_


A sedentary lifestyle affects the diversity and composition of the gut microbiota, but previous studies have mainly focused on bacteria instead of fungi. Here, we compared both the fecal bacterial and fungal microbiota compositions and functions in sedentary persons and controls. Subjects from the China Railway Corporation, including 99 inspectors and 88 officials, were enrolled in our study. Fecal microbiota communities were analyzed using 16S rRNA gene sequencing for bacteria and ITS sequencing for fungi. We found that the diversity of the gut microbiota of the sedentary group was significantly lower than that of the control group (P &lt; 0.05). The sedentary group had a higher abundance of Firmicutes, a lower abundance of Actinobacteria and Proteobacteria and a higher abundance of Ascomycota, and a lower abundance of Basidiomycota. Furthermore, functional prediction analysis of the fungal microbiota revealed more L-tryptophan degradation to 2-amino-3-carboxymuconate semialdehyde, more phospholipid remodeling (phosphatidylethanolamine, yeast), and more L-tyrosine degradation I, as well as less pentose phosphate pathway (non-oxidative branch), less adenosine nucleotide biosynthesis and less L-valine biosynthesis in the sedentary group (P &lt; 0.05). Thus, a sedentary lifestyle changes the composition and function of the gut microbiota. It may change the pentose phosphate pathway (non-oxidative branch), nucleic acid and amino acid biosynthesis and phospholipid metabolism in fungi.

[Source](https://doi.org/10.1007/s00284-023-03480-0)

10.1007/s00284-023-03480-0

---

## Effect of urbanization and parasitism on the gut microbiota of Darwin's finch nestlings.
 14.10.2023, _G Solomon, AC Love, GJ Vaziri, J Harvey, T Verrett, K Chernicky, S Simons, L Albert, JA Chaves, SA Knutie_


Host-associated microbiota can be affected by factors related to environmental change, such as urbanization and invasive species. For example, urban areas often affect food availability for animals, which can change their gut microbiota. Invasive parasites can also influence microbiota through competition or indirectly through a change in the host immune response. These interacting factors can have complex effects on host fitness, but few studies have disentangled the relationship between urbanization and parasitism on an organism's gut microbiota. To address this gap in knowledge, we investigated the effects of urbanization and parasitism by the invasive avian vampire fly (Philornis downsi) on the gut microbiota of nestling small ground finches (Geospiza fuliginosa) on San Cristóbal Island, Galápagos. We conducted a factorial study in which we experimentally manipulated parasite presence in an urban and nonurban area. Faeces were then collected from nestlings to characterize the gut microbiota (i.e. bacterial diversity and community composition). Although we did not find an interactive effect of urbanization and parasitism on the microbiota, we did find main effects of each variable. We found that urban nestlings had lower bacterial diversity and different relative abundances of taxa compared to nonurban nestlings, which could be mediated by introduction of the microbiota of the food items or changes in host physiology. Additionally, parasitized nestlings had lower bacterial richness than nonparasitized nestlings, which could be mediated by a change in the immune system. Overall, this study advances our understanding of the complex effects of anthropogenic stressors on the gut microbiota of birds.

[Source](https://doi.org/10.1111/mec.17164)

10.1111/mec.17164

---

## Metabarcoding using nanopore long-read sequencing for the unbiased characterization of apicomplexan haemoparasites.
 14.10.2023, _LG Huggins, V Colella, ND Young, RJ Traub_


Apicomplexan haemoparasites generate significant morbidity and mortality in humans and other animals, particularly in many low-to-middle income countries. Malaria caused by Plasmodium remains responsible for some of the highest numbers of annual deaths of any human pathogen, whilst piroplasmids, such as Babesia and Theileria can have immense negative economic effects through livestock loss. Diagnosing haemoparasites via traditional methods like microscopy is challenging due to low-level and transient parasitaemia. PCR-based diagnostics overcome these limitations by being both highly sensitive and specific, but they may be unable to accurately detect coinfections or identify novel species. In contrast, next-generation sequencing (NGS)-based methods can characterize all pathogens from a group of interest concurrently, although, the short-read platforms previously used have been limited in the taxonomic resolution achievable. Here, we used Oxford Nanopore Technologies' (ONT) long-read MinION™ sequencer to conduct apicomplexan haemoparasite metabarcoding via sequencing the near full-length 18S ribosomal RNA gene, demonstrating its ability to detect Babesia, Hepatozoon, Neospora, Plasmodium, Theileria and Toxoplasma species. This method was tested on blood-extracted DNA from 100 dogs and the results benchmarked against qPCR and Illumina-based metabarcoding. For two common haemoparasites, nanopore sequencing performed as well as qPCR (kappa agreement statistics &gt; 0.98), whilst also detecting one pathogen, Hepatozoon felis, missed by the other techniques. The long-reads obtained by nanopore sequencing provide an improved species-level taxonomic resolution whilst the method's broad applicability mean it can be used to explore apicomplexan communities from diverse mammalian hosts, on a portable sequencer that easily permits adaptation to field use.

[Source](https://doi.org/10.1111/1755-0998.13878)

10.1111/1755-0998.13878

---

## Bacterial networks in Atlantic salmon with Piscirickettsiosis

[Source](https://doi.org/10.1038/s41598-023-43345-x)

---

## Metagenomic analysis reveals a dynamic microbiome with diversified adaptive functions that respond to ovulation regulation in the mouse endometrium

[Source](https://doi.org/10.1186/s12864-023-09712-8)

---

## Probiotic-fermented tomato alleviates high-fat diet-induced obesity in mice: Insights from microbiome and metabolomics

[Source](https://doi.org/10.1016/j.foodchem.2023.137719)

---

## <em>Lactobacillus rhamnosus</em> Modulates Lung Inflammation and Mitigates Gut Dysbiosis in a Murine Model of Asthma-COPD Overlap Syndrome

[Source](https://doi.org/10.1007/s12602-023-10167-2)

---

## Fermented Goat’s Milk Contributes to the Recovery of Iron Deficiency Anemia via Modulation of the Gut Microbiome

[Source](https://doi.org/10.1021/acs.jafc.3c05560)

---

## Coexistence of specialist and generalist species within mixed plastic derivative-utilizing microbial communities

[Source](https://doi.org/10.1186/s40168-023-01645-4)

---

## A critical review of the recent trends in source tracing of microplastics in the environment

[Source](https://doi.org/10.1016/j.envres.2023.117394)

---

## Keystone taxa of phoD-harboring bacteria mediate alkaline phosphatase activity during biochar remediation of Cd-contaminated soil

[Source](https://doi.org/10.1016/j.scitotenv.2023.167726)

---

## Maintaining beneficial alga-associated bacterial communities under heat stress: Insights from controlled co-culture experiments using antibiotic-resistant bacterial strains.
 13.10.2023, _E Karimi, SM Dittami_


Brown algae, like many eukaryotes, possess diverse microbial communities. Ectocarpus - a model brown alga- relies on these communities for essential processes, such as growth development. Controlled laboratory systems are needed for functional studies of these algal-bacterial interactions. We selected bacterial strains based on their metabolic networks to provide optimal completion of the algal metabolism, rendered them resistant to two antibiotics, and inoculate them to establish controlled co-cultures with Ectocarpus under continuous antibiotic treatment. We then monitored the stability of the resulting associations under control conditions and heat stress using 16S metabarcoding. Antibiotics strongly reduced bacterial diversity both in terms of taxonomy and predicted metabolic functions. In the inoculated sample, 63-69% of reads corresponded to the inoculated strains, and the communities remained stable during temperature stress. They also partially restored the predicted metabolic functions of the natural community. Overall, the development of antibiotic-resistant helper cultures offers a promising route to fully controlled laboratory experiments with algae and microbiota and thus represents an important step towards generating experimental evidence for specific host-microbe interactions in the systems studied. Further work will be required to achieve full control and progressively expand our repertoire of helper strains including those currently "unculturable".

[Source](https://doi.org/10.1093/femsec/fiad130)

10.1093/femsec/fiad130

---

## Deep discovery informs difficult deployment in plant microbiome science.
 13.10.2023, _D Russ, CR Fitzpatrick, PJPL Teixeira, JL Dangl_


Plant-associated microbiota can extend plant immune system function, improve nutrient acquisition and availability, and alleviate abiotic stresses. Thus, naturally beneficial microbial therapeutics are enticing tools to improve plant productivity. The basic definition of plant microbiota across species and ecosystems, combined with the development of reductionist experimental models and the manipulation of plant phenotypes with microbes, has fueled interest in its translation to agriculture. However, the great majority of microbes exhibiting plant-productivity traits in the lab and greenhouse fail in the field. Therapeutic microbes must reach détente, the establishment of uneasy homeostasis, with the plant immune system, invade heterogeneous pre-established plant-associated communities, and persist in a new and potentially remodeled community. Environmental conditions can alter community structure and thus impact the engraftment of therapeutic microbes. We survey recent breakthroughs, challenges, and opportunities in translating beneficial microbes from the lab to the field.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(23)00969-8)

10.1016/j.cell.2023.08.035

---

## Plant cell wall component induced bacterial development.
 14.10.2023, _ÁT Kovács_


Plant-microbiome functioning depends on intricate signaling pathways including plant-derived excretions that induce microbial gene expression. Marc Ongena and his team (Boubsi et al.) dissect how the pectin backbone homogalacturonan promotes bacterial differentiation programs of Bacillus velezensis, potentially facilitating its establishment in the rhizosphere.

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(23)00292-5)

10.1016/j.tim.2023.10.003

---

## MGS2AMR: a gene-centric mining of metagenomic sequencing data for pathogens and their antimicrobial resistance profile.
 14.10.2023, _PJ Van Camp, VBS Prasath, DB Haslam, A Porollo_


Identification of pathogenic bacteria from clinical specimens and evaluating their antimicrobial resistance (AMR) are laborious tasks that involve in vitro cultivation, isolation, and susceptibility testing. Recently, a number of methods have been developed that use machine learning algorithms applied to the whole-genome sequencing data of isolates to approach this problem. However, making AMR assessments from more easily available metagenomic sequencing data remains a big challenge.

[Source](https://doi.org/10.1186/s40168-023-01674-z)

10.1186/s40168-023-01674-z

---

## Subgroup Identification Using Virtual Twins for Human Microbiome Studies.
 13.10.2023, _H Koh_


Even when the same treatment is employed, some patients are cured, while others are not. The patients that are cured may have beneficial microbes in their body that can boost treatment effects, but it is vice versa for the patients that are not cured. That is, treatment effects can vary depending on the patient's microbiome. If the effects of candidate treatments are well-predicted based on the patient's microbiome, we can select a treatment that is suited to the patient's microbiome or alter the patient's microbiome to improve treatment effects. Here, I introduce a streamlined analytic method, microbiome virtual twins (MiVT), to probe for the interplay between microbiome and treatment. MiVT employs a new prediction method, distance-based machine learning (dML), to improve prediction accuracy in microbiome studies and a new significance test, bootstrap-based test for regression tree (BoRT), to test if each subgroup's treatment effect is the same with the overall treatment effect. MiVT will serve as a useful guideline in microbiome-based personalized medicine to select the therapy that is most suited to the patient's microbiome or to tune the patient's microbiome to be suited to the treatment. MiVT can be implemented using an R package, MiVT, at https://github.com/hk1785/MiVT.

[Source](https://doi.org/10.1109/TCBB.2023.3324139)

10.1109/TCBB.2023.3324139

---

