## Microbiome diversity: A barrier to the environmental spread of antimicrobial resistance?

[Source](https://www.biorxiv.org/content/10.1101/2023.03.30.534382v2.abstract)

---

## First-Day-of-Life Rectal Swabs Fail To Represent Meconial Microbiota Composition and Underestimate the Presence of Antibiotic Resistance Genes.
 25.04.2023, _S Graspeuntner, M Lupatsii, L Dashdorj, A Rody, J Rupp, V Bossung, C Härtel_


The human gut microbiome plays a vital role in health and disease. In particular, the first days of life provide a unique window of opportunity for development and establishment of microbial community. Currently, stool samples are known to be the most widely used sampling approach for studying the gut microbiome. However, complicated sample acquisition at certain time points, challenges in transportation, and patient discomfort underline the need for development of alternative sampling approaches. One of the alternatives is rectal swabs, shown to be a reliable proxy for gut microbiome analysis when obtained from adults. Here, we compare the usability of rectal swabs and meconium paired samples collected from infants on the first days of life. Our results indicate that the two sampling approaches display significantly distinct patterns in microbial composition and alpha and beta diversity as well as detection of resistance genes. Moreover, the dissimilarity between the two collection methods was greater than the interindividual variation. Therefore, we conclude that rectal swabs are not a reliable proxy compared to stool samples for gut microbiome analysis when collected on the first days of a newborn's life. IMPORTANCE Currently, there are numerous suggestions on how to ease the notoriously complex and error-prone methodological setups to study the gut microbiota of newborns during the first days of life. Especially, meconium samples are regularly failing to yield meaningful data output and therefore have been suggested to be replaced by rectal swabs as done in adults as well. We find this development toward a simplified method to be producing dramatically erroneous results, skewing data interpretation away from the real aspects to be considered for neonatal health during the first days of life. We have put together our knowledge on this critical aspect with careful consideration and identified the failure of rectal swabs to be a replacement for sampling of meconium in term-born newborns.

[Source](https://journals.asm.org/doi/10.1128/spectrum.05254-22)

10.1128/spectrum.05254-22

---

## Mucin-microbiome signatures shape the tumor microenvironment in gastric cancer.
 25.04.2023, _B Oosterlinck, H Ceuleers, W Arras, JG De Man, K Geboes, H De Schepper, M Peeters, S Lebeer, J Skieceviciene, GL Hold, J Kupcinskas, A Link, BY De Winter, A Smet_


_Humans, Stomach Neoplasms, Mucin-2, Tumor Microenvironment, RNA, Ribosomal, 16S, Mucin-6, Adenocarcinoma, Microbiota, Phenotype_
We aimed to identify mucin-microbiome signatures shaping the tumor microenvironment in gastric adenocarcinomas and clinical outcomes.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01534-w)

10.1186/s40168-023-01534-w

---

## Fecal microbiota transplantation for recurrent C. difficile infection in patients with inflammatory bowel disease: A systematic review and meta-analysis.
 25.04.2023, _S Porcari, SMD Baunwall, AS Occhionero, MR Ingrosso, AC Ford, CL Hvas, A Gasbarrini, G Cammarota, G Ianiro_


Fecal microbiota transplantation (FMT) is known to be highly effective in patients with recurrent Clostridioides difficile infection (rCDI), but its role in patients who also suffer from inflammatory bowel disease (IBD) is unclear. Therefore, we performed a systematic review and meta-analysis to evaluate the efficacy and safety of FMT for the treatment of rCDI in patients with IBD. We searched the available literature until November 22, 2022 to identify studies that included patients with IBD treated with FMT for rCDI, reporting efficacy outcomes after at least 8 weeks of follow-up. The proportional effect of FMT was summarized with a generalized linear mixed-effect model fitting a logistic regression accounting for different intercepts among studies. We identified 15 eligible studies, containing 777 patients. Overall, FMT achieved high cure rates of rCDI, 81% for single FMT, based on all included studies and patients, and 92% for overall FMT, based on nine studies with 354 patients, respectively. We found a significant advantage of overall FMT over single FMT in improving cure rates of rCDI (from 80% to 92%, p = 0.0015). Serious adverse events were observed in 91 patients (12% of the overall population), with the most common being hospitalisation, IBD-related surgery, or IBD flare. In conclusion, in our meta-analysis FMT achieved high cure rates of rCDI in patients with IBD, with a significant advantage of overall FMT over single FMT, similar to data observed in patients without IBD. Our findings support the use of FMT as a treatment for rCDI in patients with IBD.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0896841123000458)

10.1016/j.jaut.2023.103036

---

## Alcohol, the gut microbiome, and liver disease.
 25.04.2023, _MH Jew, CL Hsu_


The microorganisms inhabiting our gastrointestinal tract are critical for human health. Chronic heavy alcohol use can modulate the composition and function of the gut microbiota, thereby exacerbating end-organ damage via the gut-brain axis and the gut-liver axis. In this review, we summarize the bacterial, fungal, and viral gut microbial compositional changes associated with alcohol use and alcohol-associated liver disease and discuss the mechanisms of action by which gut dysbiosis reinforces alcohol use behavior and liver inflammation and injury. We also highlight important pre-clinical and clinical trials that target gut microbial-specific mechanisms for the treatment of alcohol use disorder and alcohol-associated liver disease.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/jgh.16199)

10.1111/jgh.16199

---

## Microbial metabolites in colorectal tumorigenesis and cancer therapy.
 26.04.2023, _Y Liu, HC Lau, J Yu_


_Humans, Gastrointestinal Microbiome, Carcinogenesis, Intestines, Colorectal Neoplasms_
Trillions of microbes are indigenous to the human gastrointestinal tract, together forming an ecological community known as the gut microbiota. The gut microbiota is involved in dietary digestion to produce various metabolites. In healthy condition, microbial metabolites have unneglectable roles in regulating host physiology and intestinal homeostasis. However, increasing studies have reported the correlation between metabolites and the development of colorectal cancer (CRC), with the identification of oncometabolites. Meanwhile, metabolites can also influence the efficacy of cancer treatments. In this review, metabolites derived from microbes-mediated metabolism of dietary carbohydrates, proteins, and cholesterol, are introduced. The roles of pro-tumorigenic (secondary bile acids and polyamines) and anti-tumorigenic (short-chain fatty acids and indole derivatives) metabolites in CRC development are then discussed. The impacts of metabolites on chemotherapy and immunotherapy are further elucidated. Collectively, given the importance of microbial metabolites in CRC, therapeutic approaches that target metabolites may be promising to improve patient outcome.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2023.2203968)

10.1080/19490976.2023.2203968

---

## Indirect maternal effects via nest microbiome composition drive gut colonization in altricial chicks.
 25.04.2023, _D Diez-Méndez, KH Bodawatta, I Freiberga, I Klečková, KA Jønsson, M Poulsen, K Sam_


Gut microbial communities are complex and heterogeneous and play critical roles for animal hosts. Early-life disruptions to microbiome establishment can negatively impact host fitness and development. However, the consequences of such early-life disruptions remain unknown in wild birds. To help fill this gap, we investigated the effect of continuous early-life gut microbiome disruptions on the establishment and development of gut communities in wild Great tit (Parus major) and Blue tit (Cyanistes caeruleus) nestlings by applying antibiotics and probiotics. Treatment neither affected nestling growth nor their gut microbiome composition. Independent of treatment, nestling gut microbiomes of both species grouped by brood, which shared the highest numbers of bacterial taxa with both nest environment and their mother. Although fathers showed different gut communities than their nestlings and nests, they still contributed to structuring chick microbiomes. Lastly, we observed that the distance between nests increased inter-brood microbiome dissimilarity, but only in Great tits, indicating that species-specific foraging behaviour and/or microhabitat influence gut microbiomes. Overall, the strong maternal effect, driven by continuous recolonization from the nest environment and vertical transfer of microbes during feeding, appears to provide resilience towards early-life disruptions in nestling gut microbiomes.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/mec.16959)

10.1111/mec.16959

---

## The gut microbiome of solitary bees is mainly affected by pathogen assemblage and partially by land use.
 25.04.2023, _G Fernandez De Landa, D Alberoni, L Baffoni, M Fernandez De Landa, PD Revainera, LP Porrini, C Brasesco, S Quintana, F Zumpano, MJ Eguaras, MD Maggi, D Di Gioia_


Pollinators, including solitary bees, are drastically declining worldwide. Among the factors contributing to this decline, bee pathogens and different land uses are of relevance. The link between the gut microbiome composition and host health has been recently studied for social pollinators (e.g. honeybees), whereas the information related to solitary bees is sparse. This work aimed at the characterization of the gut microbiome of the solitary bees Xylocopa augusti, Eucera fervens and Lasioglossum and attempted to correlate the gut microbial composition with the presence and load of different pathogens and land uses. Solitary bees were sampled in different sites (i.e. a farm, a natural reserve, and an urban plant nursery) showing different land uses. DNA was extracted from the gut, 16S rRNA gene amplified and sequenced. Eight pathogens, known for spillover from managed bees to wild ones, were quantified with qPCR. The results showed that the core microbiome profile of the three solitary bees significantly varied in the different species. Pseudomonas was found as the major core taxa in all solitary bees analyzed, whereas Lactobacillus, Spiroplasma and Sodalis were the second most abundant taxa in X. augusti, E. fervens and Lasioglossum, respectively. The main pathogens detected with qPCR were Nosema ceranae, Nosema bombi and Crithidia bombi, although differently abundant in the different bee species and sampling sites. Most microbial taxa did not show any correlation with the land use, apart from Snodgrassella and Nocardioides, showing higher abundances on less anthropized sites. Conversely, the pathogens species and load strongly affected the gut microbial composition, with Bifidobacterium, Apibacter, Serratia, Snodgrassella and Sodalis abundance that positively or negatively correlated with the detected pathogens load. Therefore, pathogens presence and load appear to be the main factor shaping the gut microbiome of solitary bees in Argentina.

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-023-00494-w)

10.1186/s40793-023-00494-w

---

## Spatial Succession Underlies Microbial Contribution to Food Digestion in the Gut of an Algivorous Sea Urchin.
 25.04.2023, _M Masasa, A Kushmaro, D Nguyen, H Chernova, N Shashar, L Guttman_


Dietary influence on the microbiome in algivorous sea urchins such as Tripneustes gratilla elatensis suggests a bacterial contribution to the digestion of fiber-rich seaweed. An ecological insight into the spatial arrangement in the gut bacterial community will improve our knowledge of host-microbe relations concerning the involved taxa, their metabolic repertoire, and the niches of activity. Toward this goal, we investigated the bacterial communities in the esophagus, stomach, and intestine of Ulva-fed sea urchins through 16S rRNA amplicon sequencing, followed by the prediction of their functional genes. We revealed communities with distinct features, especially those in the esophagus and intestine. The esophageal community was less diverse and was poor in food digestive or fermentation genes. In contrast, bacteria that can contribute to the digestion of the dietary Ulva were common in the stomach and intestine and consisted of genes for carbohydrate decomposition, fermentation, synthesis of short-chain fatty acids, and various ways of N and S metabolism. Bacteroidetes and Firmicutes were found as the main phyla in the gut and are presumably also necessary in food digestion. The abundant sulfate-reducing bacteria in the stomach and intestine from the genera Desulfotalea, Desulfitispora, and Defluviitalea may aid in removing the excess sulfate from the decomposition of the algal polysaccharides. Although these sea urchins were fed with Ulva, genes for the degradation of polysaccharides of other algae and plants were present in this sea urchin gut microbiome. We conclude that the succession of microbial communities along the gut obtained supports the hypothesis on bacterial contribution to food digestion. IMPORTANCE Alga grazing by the sea urchin Tripneustes gratilla elatensis is vital for nutrient recycling and constructing new reefs. This research was driven by the need to expand the knowledge of bacteria that may aid this host in alga digestion and their phylogeny, roles, and activity niches. We hypothesized alterations in the bacterial compositional structure along the gut and their association with the potential contribution to food digestion. The current spatial insight into the sea urchin's gut microbiome ecology is novel and reveals how distinct bacterial communities are when distant from each other in this organ. It points to keynote bacteria with genes that may aid the host in the digestion of the complex sulfated polysaccharides in dietary Ulva by removing the released sulfates and fermentation to provide energy. The gut bacteria's genomic arsenal may also help to gain energy from diets of other algae and plants.

[Source](https://journals.asm.org/doi/10.1128/spectrum.00514-23)

10.1128/spectrum.00514-23

---

## The microbiota in feces of domestic pigeons in Seoul, Korea.
 25.04.2023, _S Oh, SH Park, JH Choi, SL Kim, M Kim, S Lee, MH Yi, IY Lee, TS Yong, JY Kim_


In Korea, feral pigeons pose significant public health risks because they carry various zoonotic pathogens. Human population density is a significant factor in zoonotic disease events. Seoul is one of the largest cities by population density among developed countries and where most of the homeless population in Korea exists. We designed this study to compare the microbiota of pigeon feces by regional characteristics and the presence of homeless individuals. Therefore, this study used 16S rRNA amplicon sequencing to detect possible pathogenic microbes and assess the current risk of zoonosis in Seoul, South Korea. Pigeon fecal samples (n = 144) obtained from 19 public sites (86 and 58 fecal samples from regions in and outside Seoul, respectively) were examined. Potentially pathogenic bacteria were also detected in the fecal samples; Campylobacter spp. was found in 19 samples from 13 regions, Listeriaceae was found in seven samples, and Chlamydia spp. was found in three samples from two regions. Principal coordinates analysis and permutational multivariate analysis of variance revealed a significant difference in bacterial composition between the regions in Seoul (n = 86) and outside Seoul (n = 58) and between the regions with (n = 81) and without (n = 63) homeless individuals. Overall, this study identified various potentially pathogenic microorganisms in pigeon feces at public sites in South Korea. Moreover, this study demonstrates that the microbial composition was influenced by regional characteristics and homelessness. Taken together, this study provides important information for public health strategic planning and disease control.

[Source](https://www.sciencedirect.com/science/article/pii/S2405844023022041)

10.1016/j.heliyon.2023.e14997

---

## Characterization of the Rhizosphere Bacterial Microbiome and Coffee Bean Fermentation in the Castillo-Tambo and Bourbon Varieties in the Popayán-Colombia Plateau.
 25.04.2023, _AFS Pino, ZYD Espinosa, EVR Cabrera_


The microbial biodiversity and the role of microorganisms in the fermentation of washed coffee in Colombia were investigated using the Bourbon and Castillo coffee varieties. DNA sequencing was used to evaluate the soil microbial biota and their contribution to fermentation. The potential benefits of these microorganisms were analyzed, including increased productivity and the need to understand the rhizospheric bacterial species to optimize these benefits.

[Source](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-023-04182-2)

10.1186/s12870-023-04182-2

---

## Hydrochemical and Seasonally Conditioned Changes of Microbial Communities in the Tufa-Forming Freshwater Network Ecosystem.
 25.04.2023, _A Čačković, K Kajan, L Selak, T Marković, A Brozičević, P Pjevac, S Orlić_


Freshwater network ecosystems consist of interconnected lotic and lentic environments within the same catchment area. Using Plitvice Lakes as an example, we studied the changes in environmental conditions and microbial communities (bacteria and fungi) that occur with downstream flow. Water samples from tributaries, interlake streams, connections of the cascading lakes, and the Korana River, the main outflow of the system, were characterized using amplicon sequencing of bacterial 16S rRNA and fungal ITS2 genes. Our results show that different environmental conditions and bacterial and fungal communities prevail among the three stream types within the freshwater network ecosystem during multiple sampling seasons. Microbial community differences were also confirmed along the longitudinal gradient between the most distant sampling sites. The higher impact of "mass effect" was evident during spring and winter, while "species sorting" and "environmental selection" was more pronounced during summer. Prokaryotic community assembly was majorly influenced by deterministic processes, while fungal community assembly was highly dominated by stochastic processes, more precisely by the undominated fraction, which is not dominated by any process. Despite the differences between stream types, the microbial community of Plitvice Lakes is shown to be very stable by the core microbiome that makes up the majority of stream communities. Our results suggest microbial community succession along the river-lake continuum of microbial communities in small freshwater network ecosystems with developed tufa barriers. IMPORTANCE Plitvice Lakes represent a rare freshwater ecosystem consisting of a complex network of lakes and waterfalls connecting them, as well as rivers and streams supplying water to the lake basin. The unique geomorphological, hydrological, biogeochemical, and biological phenomenon of Plitvice Lakes lies in the biodynamic process of forming tufa barriers. In addition to microbial communities, abiotic water factors also have a major influence on the formation of tufa. Therefore, it is important to understand how changes in environmental conditions and microbial community assembly affect the functioning of the ecosystem of a freshwater network with developed tufa barriers.

[Source](https://journals.asm.org/doi/10.1128/msphere.00602-22)

10.1128/msphere.00602-22

---

## ggpicrust2: an R package for PICRUSt2 predicted functional profile analysis and visualization

[Source](https://arxiv.org/abs/2303.10388)

---

