## Questioning the fetal microbiome illustrates pitfalls of low-biomass microbial studies.
 27.01.2023, _KM Kennedy, MC de Goffau, ME Perez-Muñoz, MC Arrieta, F Bäckhed, P Bork, T Braun, FD Bushman, J Dore, WM de Vos, AM Earl, JA Eisen, MA Elovitz, SC Ganal-Vonarburg, MG Gänzle, WS Garrett, LJ Hall, MW Hornef, C Huttenhower, L Konnikova, S Lebeer, AJ Macpherson, RC Massey, AC McHardy, O Koren, TD Lawley, RE Ley, L O'Mahony, PW O'Toole, EG Pamer, J Parkhill, J Raes, T Rattei, A Salonen, E Segal, N Segata, F Shanahan, DM Sloboda, GCS Smith, H Sokol, TD Spector, MG Surette, GW Tannock, AW Walker, M Yassour, J Walter_


_Pregnancy, Female, Animals, Humans, Biomass, Placenta, Microbiota, Amniotic Fluid, Prenatal Care, Mammals_
Whether the human fetus and the prenatal intrauterine environment (amniotic fluid and placenta) are stably colonized by microbial communities in a healthy pregnancy remains a subject of debate. Here we evaluate recent studies that characterized microbial populations in human fetuses from the perspectives of reproductive biology, microbial ecology, bioinformatics, immunology, clinical microbiology and gnotobiology, and assess possible mechanisms by which the fetus might interact with microorganisms. Our analysis indicates that the detected microbial signals are likely the result of contamination during the clinical procedures to obtain fetal samples or during DNA extraction and DNA sequencing. Furthermore, the existence of live and replicating microbial populations in healthy fetal tissues is not compatible with fundamental concepts of immunology, clinical microbiology and the derivation of germ-free mammals. These conclusions are important to our understanding of human immune development and illustrate common pitfalls in the microbial analyses of many other low-biomass environments. The pursuit of a fetal microbiome serves as a cautionary example of the challenges of sequence-based microbiome studies when biomass is low or absent, and emphasizes the need for a trans-disciplinary approach that goes beyond contamination controls by also incorporating biological, ecological and mechanistic concepts.

[Source](https://www.nature.com/articles/s41586-022-05546-8)

10.1038/s41586-022-05546-8

---

## Microbial foods for improving human and planetary health.
 19.01.2023, _LJ Jahn, VM Rekdal, MOA Sommer_


The current food production system is negatively impacting planetary and human health. A transition to a sustainable and fair food system is urgently needed. Microorganisms are likely enablers of this process, as they can produce delicious and healthy microbial foods with low environmental footprints. We review traditional and current approaches to microbial foods, such as fermented foods, microbial biomass, and food ingredients derived from microbial fermentations. We discuss how future advances in science-driven fermentation, synthetic biology, and sustainable feedstocks enable a new generation of microbial foods, potentially impacting the sustainability, resilience, and health effects of our food system.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(22)01515-X)

10.1016/j.cell.2022.12.002

---

## A tissue atlas of ulcerative colitis revealing evidence of sex-dependent differences in disease-driving inflammatory cell types and resistance to TNF inhibitor therapy

[Source](https://www.science.org/doi/10.1126/sciadv.add1166)

---

## Associations of the skin, oral and gut microbiome with aging, frailty and infection risk reservoirs in older adults.
 22.11.2022, _PJ Larson, W Zhou, A Santiago, S Driscoll, E Fleming, AY Voigt, OK Chun, JJ Grady, GA Kuchel, JT Robison, J Oh_


Older adults represent a vulnerable population with elevated risk for numerous morbidities. To explore the association of the microbiome with aging and age-related susceptibilities including frailty and infectious disease risk, we conducted a longitudinal study of the skin, oral, and gut microbiota in 47 community- or skilled nursing facility-dwelling older adults vs. younger adults. We found that microbiome changes were not associated with chronological age so much as frailty: we identified prominent changes in microbiome features associated with susceptibility to pathogen colonization and disease risk, including diversity, stability, heterogeneity, and biogeographic determinism, which were moreover associated with a loss of Cutibacterium (C.) acnes in the skin microbiome. Strikingly, the skin microbiota were also the primary reservoir for antimicrobial resistance, clinically important pathobionts, and nosocomial strains, suggesting a potential role particularly for the skin microbiome in disease risk and dissemination of multidrug resistant pathogens.

[Source](https://www.nature.com/articles/s43587-022-00287-9)

10.1038/s43587-022-00287-9

---

## Phage therapy: From biological mechanisms to future directions.
 11.01.2023, _SA Strathdee, GF Hatfull, VK Mutalik, RT Schooley_


_Phage Therapy, Bacteriophages, Bacteria_
Increasing antimicrobial resistance rates have revitalized bacteriophage (phage) research, the natural predators of bacteria discovered over 100 years ago. In order to use phages therapeutically, they should (1) preferably be lytic, (2) kill the bacterial host efficiently, and (3) be fully characterized to exclude side effects. Developing therapeutic phages takes a coordinated effort of multiple stakeholders. Herein, we review the state of the art in phage therapy, covering biological mechanisms, clinical applications, remaining challenges, and future directions involving naturally occurring and genetically modified or synthetic phages.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(22)01461-1)

10.1016/j.cell.2022.11.017

---

## Mining metatranscriptomes reveals a vast world of viroid-like circular RNAs

[Source](https://www.cell.com/cell/fulltext/S0092-8674(22)01582-3)

---

## High-resolution metagenomic reconstruction of the freshwater spring bloom.
 28.01.2023, _VS Kavagutti, PA Bulzu, CM Chiriac, MM Salcher, I Mukherjee, T Shabarova, V Grujčić, M Mehrshad, V Kasalický, AS Andrei, J Jezberová, J Seďa, P Rychtecký, P Znachor, K Šimek, R Ghai_


_Metagenome, Fresh Water, Bacteria, Plankton, Viruses, Eukaryota, Water_
The phytoplankton spring bloom in freshwater habitats is a complex, recurring, and dynamic ecological spectacle that unfolds at multiple biological scales. Although enormous taxonomic shifts in microbial assemblages during and after the bloom have been reported, genomic information on the microbial community of the spring bloom remains scarce.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01451-4)

10.1186/s40168-022-01451-4

---

## Closed genomes uncover a saltwater species of Candidatus Electronema and shed new light on the boundary between marine and freshwater cable bacteria.
 25.01.2023, _M Sereika, F Petriglieri, TBN Jensen, A Sannikov, M Hoppe, PH Nielsen, IPG Marshall, A Schramm, M Albertsen_


Cable bacteria of the Desulfobulbaceae family are centimeter-long filamentous bacteria, which are capable of conducting long-distance electron transfer. Currently, all cable bacteria are classified into two candidate genera: Candidatus Electronema, typically found in freshwater environments, and Candidatus Electrothrix, typically found in saltwater environments. This taxonomic framework is based on both 16S rRNA gene sequences and metagenome-assembled genome (MAG) phylogenies. However, most of the currently available MAGs are highly fragmented, incomplete, and thus likely miss key genes essential for deciphering the physiology of cable bacteria. Also, a closed, circular genome of cable bacteria has not been published yet. To address this, we performed Nanopore long-read and Illumina short-read shotgun sequencing of selected environmental samples and a single-strain enrichment of Ca. Electronema aureum. We recovered multiple cable bacteria MAGs, including two circular and one single-contig. Phylogenomic analysis, also confirmed by 16S rRNA gene-based phylogeny, classified one circular MAG and the single-contig MAG as novel species of cable bacteria, which we propose to name Ca. Electronema halotolerans and Ca. Electrothrix laxa, respectively. The Ca. Electronema halotolerans, despite belonging to the previously recognized freshwater genus of cable bacteria, was retrieved from brackish-water sediment. Metabolic predictions showed several adaptations to a high salinity environment, similar to the "saltwater" Ca. Electrothrix species, indicating how Ca. Electronema halotolerans may be the evolutionary link between marine and freshwater cable bacteria lineages.

[Source](https://www.nature.com/articles/s41396-023-01372-6)

10.1038/s41396-023-01372-6

---

## Enhanced correlation-based linking of biosynthetic gene clusters to their metabolic products through chemical class matching.
 26.01.2023, _JJR Louwen, MH Medema, JJJ van der Hooft_


_Tandem Mass Spectrometry, Biosynthetic Pathways, Genomics, Metabolomics, Multigene Family_
It is well-known that the microbiome produces a myriad of specialised metabolites with diverse functions. To better characterise their structures and identify their producers in complex samples, integrative genome and metabolome mining is becoming increasingly popular. Metabologenomic co-occurrence-based correlation scoring methods facilitate the linking of metabolite mass fragmentation spectra (MS/MS) to their cognate biosynthetic gene clusters (BGCs) based on shared absence/presence patterns of metabolites and BGCs in paired omics datasets of multiple strains. Recently, these methods have been made more readily accessible through the NPLinker platform. However, co-occurrence-based approaches usually result in too many candidate links to manually validate. To address this issue, we introduce a generic feature-based correlation method that matches chemical compound classes between BGCs and MS/MS spectra.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01444-3)

10.1186/s40168-022-01444-3

---

## PLSDA-batch: a multivariate framework to correct for batch effects in microbiome data.
 18.01.2023, _Y Wang, KA Lê Cao_


Microbial communities are highly dynamic and sensitive to changes in the environment. Thus, microbiome data are highly susceptible to batch effects, defined as sources of unwanted variation that are not related to and obscure any factors of interest. Existing batch effect correction methods have been primarily developed for gene expression data. As such, they do not consider the inherent characteristics of microbiome data, including zero inflation, overdispersion and correlation between variables. We introduce new multivariate and non-parametric batch effect correction methods based on Partial Least Squares Discriminant Analysis (PLSDA). PLSDA-batch first estimates treatment and batch variation with latent components, then subtracts batch-associated components from the data. The resulting batch-effect-corrected data can then be input in any downstream statistical analysis. Two variants are proposed to handle unbalanced batch x treatment designs and to avoid overfitting when estimating the components via variable selection. We compare our approaches with popular methods managing batch effects, namely, removeBatchEffect, ComBat and Surrogate Variable Analysis, in simulated and three case studies using various visual and numerical assessments. We show that our three methods lead to competitive performance in removing batch variation while preserving treatment variation, especially for unbalanced batch $\times $ treatment designs. Our downstream analyses show selections of biologically relevant taxa. This work demonstrates that batch effect correction methods can improve microbiome research outputs. Reproducible code and vignettes are available on GitHub.

[Source](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbac622/6991121)

10.1093/bib/bbac622

---

## Genome-scale metabolic reconstruction of 7,302 human microorganisms for personalized medicine.
 19.01.2023, _A Heinken, J Hertel, G Acharya, DA Ravcheev, M Nyga, OE Okpala, M Hogan, S Magnúsdóttir, F Martinelli, B Nap, G Preciat, JN Edirisinghe, CS Henry, RMT Fleming, I Thiele_


The human microbiome influences the efficacy and safety of a wide variety of commonly prescribed drugs. Designing precision medicine approaches that incorporate microbial metabolism would require strain- and molecule-resolved, scalable computational modeling. Here, we extend our previous resource of genome-scale metabolic reconstructions of human gut microorganisms with a greatly expanded version. AGORA2 (assembly of gut organisms through reconstruction and analysis, version 2) accounts for 7,302 strains, includes strain-resolved drug degradation and biotransformation capabilities for 98 drugs, and was extensively curated based on comparative genomics and literature searches. The microbial reconstructions performed very well against three independently assembled experimental datasets with an accuracy of 0.72 to 0.84, surpassing other reconstruction resources and predicted known microbial drug transformations with an accuracy of 0.81. We demonstrate that AGORA2 enables personalized, strain-resolved modeling by predicting the drug conversion potential of the gut microbiomes from 616 patients with colorectal cancer and controls, which greatly varied between individuals and correlated with age, sex, body mass index and disease stages. AGORA2 serves as a knowledge base for the human microbiome and paves the way to personalized, predictive analysis of host-microbiome metabolic interactions.

[Source](https://www.nature.com/articles/s41587-022-01628-0)

10.1038/s41587-022-01628-0

---

## Infection infidelities drive innate immunity.
 26.01.2023, _JC Kagan_


Unsuccessful pathogenic activities may be key to activating host defenses to infection.

[Source](https://www.science.org/doi/10.1126/science.ade9733)

10.1126/science.ade9733

---

## Functional T cells are capable of supernumerary cell division and longevity.
 18.01.2023, _AG Soerens, M Künzli, CF Quarnstrom, MC Scott, L Swanson, JJ Locquiao, HE Ghoneim, D Zehn, B Youngblood, V Vezys, D Masopust_


Differentiated somatic mammalian cells putatively exhibit species-specific division limits that impede cancer but may constrain lifespans1-3. To provide immunity, transiently stimulated CD8+ T cells undergo unusually rapid bursts of numerous cell divisions, and then form quiescent long-lived memory cells that remain poised to reproliferate following subsequent immunological challenges. Here we addressed whether T cells are intrinsically constrained by chronological or cell-division limits. We activated mouse T cells in vivo using acute heterologous prime-boost-boost vaccinations4, transferred expanded cells to new mice, and then repeated this process iteratively. Over 10 years (greatly exceeding the mouse lifespan)5 and 51 successive immunizations, T cells remained competent to respond to vaccination. Cells required sufficient rest between stimulation events. Despite demonstrating the potential to expand the starting population at least 1040-fold, cells did not show loss of proliferation control and results were not due to contamination with young cells. Persistent stimulation by chronic infections or cancer can cause T cell proliferative senescence, functional exhaustion and death6. We found that although iterative acute stimulations also induced sustained expression and epigenetic remodelling of common exhaustion markers (including PD1, which is also known as PDCD1, and TOX) in the cells, they could still proliferate, execute antimicrobial functions and form quiescent memory cells. These observations provide a model to better understand memory cell differentiation, exhaustion, cancer and ageing, and show that functionally competent T cells can retain the potential for extraordinary population expansion and longevity well beyond their organismal lifespan.

[Source](https://www.nature.com/articles/s41586-022-05626-9)

10.1038/s41586-022-05626-9

---

## Genetic adaptation to pathogens and increased risk of inflammatory disorders in post-Neolithic Europe

[Source](https://www.cell.com/cell-genomics/fulltext/S2666-979X(22)00211-7)

---

