## A Deep Learning Approach to Antibiotic Discovery.
 24.02.2020, _JM Stokes, K Yang, K Swanson, W Jin, A Cubillos-Ruiz, NM Donghia, CR MacNair, S French, LA Carfrae, Z Bloom-Ackerman, VM Tran, A Chiappino-Pepe, AH Badran, IW Andrews, EJ Chory, GM Church, ED Brown, TS Jaakkola, R Barzilay, JJ Collins_


Due to the rapid emergence of antibiotic-resistant bacteria, there is a growing need to discover new antibiotics. To address this challenge, we trained a deep neural network capable of predicting molecules with antibacterial activity. We performed predictions on multiple chemical libraries and discovered a molecule from the Drug Repurposing Hub-halicin-that is structurally divergent from conventional antibiotics and displays bactericidal activity against a wide phylogenetic spectrum of pathogens including Mycobacterium tuberculosis and carbapenem-resistant Enterobacteriaceae. Halicin also effectively treated Clostridioides difficile and pan-resistant Acinetobacter baumannii infections in murine models. Additionally, from a discrete set of 23 empirically tested predictions from &gt;107 million molecules curated from the ZINC15 database, our model identified eight antibacterial compounds that are structurally distant from known antibiotics. This work highlights the utility of deep learning approaches to expand our antibiotic arsenal through the discovery of structurally distinct antibacterial molecules.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(20)30102-1?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867420301021%3Fshowall%3Dtrue#.XlAdBt9DSM4.twitter)

[Sci-hub](http://scihub.bban.top/10.1016/j.cell.2020.01.021)

---

## Novel Outlook in Microbial Ecology: Nonmutualistic Interspecies Electron Transfe

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0966842X2030010X)

---

## A solution to minimum sample size for regressions.
 28.02.2020, _DG Jenkins, PF Quintana-Ascencio_


Regressions and meta-regressions are widely used to estimate patterns and effect sizes in various disciplines. However, many biological and medical analyses use relatively low sample size (N), contributing to concerns on reproducibility. What is the minimum N to identify the most plausible data pattern using regressions? Statistical power analysis is often used to answer that question, but it has its own problems and logically should follow model selection to first identify the most plausible model. Here we make null, simple linear and quadratic data with different variances and effect sizes. We then sample and use information theoretic model selection to evaluate minimum N for regression models. We also evaluate the use of coefficient of determination (R2) for this purpose; it is widely used but not recommended. With very low variance, both false positives and false negatives occurred at N &lt; 8, but data shape was always clearly identified at N ≥ 8. With high variance, accurate inference was stable at N ≥ 25. Those outcomes were consistent at different effect sizes. Akaike Information Criterion weights (AICc wi) were essential to clearly identify patterns (e.g., simple linear vs. null); R2 or adjusted R2 values were not useful. We conclude that a minimum N = 8 is informative given very little variance, but minimum N ≥ 25 is required for more variance. Alternative models are better compared using information theory indices such as AIC but not R2 or adjusted R2. Insufficient N and R2-based model selection apparently contribute to confusion and low reproducibility in various disciplines. To avoid those problems, we recommend that research based on regressions or meta-regressions use N ≥ 25.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229345)

[Sci-hub](http://scihub.bban.top/10.1371/journal.pone.0229345)

---

## The relationship between the host genome, microbiome, and host phenotype.
 18.02.2020, _H Brüssow_



[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1462-2920.14938?campaign=wolearlyview)

[Sci-hub](http://scihub.bban.top/10.1111/1462-2920.14938)

---

## A comprehensive non-redundant gene catalog reveals extensive within-community intraspecies diversity in the human vagina.
 27.02.2020, _B Ma, MT France, J Crabtree, JB Holm, MS Humphrys, RM Brotman, J Ravel_


Analysis of metagenomic and metatranscriptomic data is complicated and typically requires extensive computational resources. Leveraging a curated reference database of genes encoded by members of the target microbiome can make these analyses more tractable. In this study, we assemble a comprehensive human vaginal non-redundant gene catalog (VIRGO) that includes 0.95 million non-redundant genes. The gene catalog is functionally and taxonomically annotated. We also construct a vaginal orthologous groups (VOG) from VIRGO. The gene-centric design of VIRGO and VOG provides an easily accessible tool to comprehensively characterize the structure and function of vaginal metagenome and metatranscriptome datasets. To highlight the utility of VIRGO, we analyze 1,507 additional vaginal metagenomes, and identify a high degree of intraspecies diversity within and across vaginal microbiota. VIRGO offers a convenient reference database and toolkit that will facilitate a more in-depth understanding of the role of vaginal microorganisms in women's health and reproductive outcomes.

[Source](https://www.nature.com/articles/s41467-020-14677-3)

[Sci-hub](http://scihub.bban.top/10.1038/s41467-020-14677-3)

---

## Increasing the proportion of plasma MUFA, as a result of dietary intervention, is associated with a modest improvement in insulin sensitivity.
 23.02.2020, _I Johns, G Frost, A Dornhorst_


The effect of modifying dietary fatty acid (FA) composition on insulin sensitivity remains unclear. We aimed to investigate whether changes in plasma phospholipid (PL) FA composition, as a result of dietary intervention, correspond with changes in insulin sensitivity. The RISCK study was a 6-month randomised controlled dietary intervention study, which assessed the effect of modifying dietary fat and the glycaemic index (GI) of carbohydrates on insulin sensitivity. Total NEFA levels, fasting plasma PL FA profiles and an insulin sensitivity index (Si), derived from intravenous glucose tolerance minimal-model analysis, were available from 533 participants, all at elevated risk of type 2 diabetes. Bivariate correlations between changes in saturated PL FA (SFA), MUFA (as a percentage of total plasma NEFA) and changes in Si were assessed according to treatment group. Age, sex, ethnicity, percentage change in body mass and change in dietary GI were controlled for. Increasing total NEFA concentration was associated with worsening Si (r -0·152; P = 0·001). In the high-MUFA/low-GI diet group, change in PL-MUFA was positively and independently associated with change in Si (r 0·297; P = 0·002). Among MUFA, change in oleic acid (18 : 1) was most strongly correlated with change in Si (r 0·266; P = 0·005), as was change in minor FA 24 : 1 (r 0·244; P = 0·011) and 17 : 1 (r 0·196; P = 0·042). In the high-SFA/high-GI group, change in SFA concentration was not significantly associated with change in Si. In conclusion, increases in the proportion of plasma PL-MUFA following a high-MUFA dietary intervention were associated with improvements in insulin sensitivity.

[Source](https://www.nature.com/articles/s41598-020-58000-y)

[Sci-hub](http://scihub.bban.top/10.1017/jns.2019.29)

---

## Gut microbiota plasticity is correlated with sustained weight loss on a low-carb or low-fat dietary intervention

[Source](https://www.nature.com/articles/s41598-020-58000-y)

---

## Analysis of healthy reference ranges for clinically relevant gut microbial indicators: A large web-based cohort study in Japan

[Source](https://www.biorxiv.org/content/10.1101/2020.02.25.965459v1.abstract?%3Fcollection=)

---

## Short chain fatty acids produced by colonizing intestinal commensal bacterial interaction with expressed breast milk are anti-inflammatory in human immature enterocytes.
 28.02.2020, _N Zheng, Y Gao, W Zhu, D Meng, WA Walker_


Necrotizing enterocolitis (NEC) is a devastating intestinal emergency that affects ten percent of very low birth weight premature babies and costs society in both expense and heartache. It is probably caused by an inappropriate interaction of colonizing bacteria with an immature intestine. A possible preventative measure is to feed prematures their mother's expressed breast milk in conjunction with a probiotic. This synbiotic prevention reduces the severity and incidence of this condition. This study was designed to determine the mechanism of the synbiotic effect in human and mouse fetal intestine. Breast milk interacting with a NEC preventative probiotic such as Bifidobacterium infantis can produce increased levels of short chain fatty acids (acetate, propionate and butyrate) (SCFAs). SCFAs are known to be anti-inflammatory in mature enterocytes and immunocytes. Very little is known about their role in immature intestine. When exposed to a human fetal cell line, fetal intestinal organoids and fetal mouse intestine, these SCFAs were anti-inflammatory. Their mechanism of anti-inflammation differed from those reported for mature cells by involving the G-protein coupled receptor (GPR 109A) and inhibiting histone deacetylase 4 and 5. These bacterial metabolites may help explain the synbiotic anti-inflammatory effect of breast milk and probiotics given to premature infants at risk for NEC.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229283)

[Sci-hub](http://scihub.bban.top/10.1371/journal.pone.0229283)

---

## Characterization of gut microbiomes of household pets in the United States using a direct-to-consumer approach.
 28.02.2020, _AR Jha, J Shmalberg, J Tanprasertsuk, L Perry, D Massey, RW Honaker_


The role of gut microbiomes as important regulators of mammalian health is increasingly recognized, although feline and canine gut microbiomes remain poorly characterized. In this proof-of-concept study, we assessed the utility of a direct-to-consumer approach to executing pet microbiome studies. We characterized the gut microbiomes of 238 pets (46 cats and 192 dogs) by generating ~11 million merged reads that were mapped to the V4 region of 16S ribosomal RNA gene at a sequencing depth of 45,806 (±22,325) reads per sample. Analyses of these reads revealed that both feline and canine gut microbiomes are dominated by three major phyla, namely Firmicutes, Proteobacteria, and Bacteroides and that alpha diversity is higher in the feline gut. In addition to interspecies differences between the feline and canine gut, we also detected appreciable intraspecies bacterial variation within the canine population. While the dogs in this dataset could be assigned to three distinct clusters based on their gut microbiome, no clustering was observed within the feline population. Integration of additional data obtained from survey questionnaires revealed that geography and body weight may be associated with canine gut microbiome composition. Furthermore, we found that both the inter and intraspecies differences are more pronounced at finer taxonomic levels, indicating that strain-level investigations may be necessary in the future. This study demonstrates that the direct-to-consumer approach overcomes existing limitations in pet microbiome research, for example, it allows collection of large numbers of pet samples. The direct-to-consumer approach has proven successful in human genomics as well as human microbiomics and this study demonstrates that by building partnerships with an engaged general public this approach can also propel the field of pet microbiomics forward.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0227289)

[Sci-hub](http://scihub.bban.top/10.1371/journal.pone.0227289)

---

## Repeated sleep disruption in mice leads to persistent shifts in the fecal microbiome and metabolome.
 28.02.2020, _SJ Bowers, F Vargas, A González, S He, P Jiang, PC Dorrestein, R Knight, KP Wright, CA Lowry, M Fleshner, MH Vitaterna, FW Turek_


It has been established in recent years that the gut microbiome plays a role in health and disease, potentially via alterations in metabolites that influence host physiology. Although sleep disruption and gut dysbiosis have been associated with many of the same diseases, studies investigating the gut microbiome in the context of sleep disruption have yielded inconsistent results, and have not assessed the fecal metabolome. We exposed mice to five days of sleep disruption followed by four days of ad libitum recovery sleep, and assessed the fecal microbiome and fecal metabolome at multiple timepoints using 16S rRNA gene amplicons and untargeted LC-MS/MS mass spectrometry. We found global shifts in both the microbiome and metabolome in the sleep-disrupted group on the second day of recovery sleep, when most sleep parameters had recovered to baseline levels. We observed an increase in the Firmicutes:Bacteroidetes ratio, along with decreases in the genus Lactobacillus, phylum Actinobacteria, and genus Bifidobacterium in sleep-disrupted mice compared to control mice. The latter two taxa remained low at the fourth day post-sleep disruption. We also identified multiple classes of fecal metabolites that were differentially abundant in sleep-disrupted mice, some of which are physiologically relevant and commonly influenced by the microbiome. This included bile acids, and inference of microbial functional gene content suggested reduced levels of the microbial bile salt hydrolase gene in sleep-disrupted mice. Overall, this study adds to the evidence base linking disrupted sleep to the gut microbiome and expands it to the fecal metabolome, identifying sleep disruption-sensitive bacterial taxa and classes of metabolites that may serve as therapeutic targets to improve health after poor sleep.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229001)

[Sci-hub](http://scihub.bban.top/10.1371/journal.pone.0229001)

---

## Engineering bacterial symbionts of nematodes improves biocontrol potential of the western corn rootworm.
 18.02.2020, _RAR Machado, L Thönen, CCM Arce, V Theepan, F Prada, D Wüthrich, CAM Robert, E Vogiatzaki, YM Shi, OP Schaeren, M Notter, R Bruggmann, S Hapfelmeier, HB Bode, M Erb_


The western corn rootworm (WCR) decimates maize crops worldwide. One potential way to control this pest is treatment with entomopathogenic nematodes (EPNs) that harbor bacterial symbionts that are pathogenic to insects. However, WCR larvae sequester benzoxazinoid secondary metabolites that are produced by maize and use them to increase their resistance to the nematodes and their symbionts. Here we report that experimental evolution and selection for bacterial symbionts that are resistant to benzoxazinoids improve the ability of a nematode-symbiont pair to kill WCR larvae. We isolated five Photorhabdus symbionts from different nematodes and increased their benzoxazinoid resistance through experimental evolution. Benzoxazinoid resistance evolved through multiple mechanisms, including a mutation in the aquaporin-like channel gene aqpZ. We reintroduced benzoxazinoid-resistant Photorhabdus strains into their original EPN hosts and identified one nematode-symbiont pair that was able to kill benzoxazinoid-sequestering WCR larvae more efficiently. Our results suggest that modification of bacterial symbionts might provide a generalizable strategy to improve biocontrol of agricultural pests.

[Source](https://www.nature.com/articles/s41587-020-0419-1)

[Sci-hub](http://scihub.bban.top/10.1038/s41587-020-0419-1)

---

## Genome wide association study reveals plant loci controlling heritability of the rhizosphere microbiome

[Source](https://www.biorxiv.org/content/10.1101/2020.02.21.960377v1)

---

## Differentiation strategies of soil rare and abundant microbial taxa in response to changing climatic regimes.
 27.02.2020, _Y Liang, X Xiao, EE Nuccio, M Yuan, N Zhang, K Xue, FM Cohan, J Zhou, B Sun_


Despite the important roles of soil microbes, especially the most diverse rare taxa in maintaining community diversity and multifunctionality, how different climate regimes alter the stability and functions of the rare microbial biosphere remains unknown. We reciprocally transplanted field soils across a latitudinal gradient to simulate climate change and sampled the soils annually after harvesting the maize over the following 6 years (from 2005 to 2011). By sequencing microbial 16S ribosomal RNA gene amplicons, we found that changing climate regimes significantly altered the composition and dynamics of soil microbial communities. A continuous succession of the rare and abundant communities was observed. Rare microbial communities were more stable under changing climatic regimes, with lower variations in temporal dynamics, and higher stability and constancy of diversity. More nitrogen cycling genes were detected in the rare members than in the abundant members, including amoA, napA, nifH, nirK, nirS, norB and nrfA. Random forest analysis and receiver operating characteristics analysis showed that rare taxa may act as potential contributors to maize yield under changing climatics. The study indicates that the taxonomically and functionally diverse rare biosphere has the potential to increase functional redundancy and enhance the ability of soil communities to counteract environmental disturbances. With ongoing global climate change, exploring the succession process and functional changes of rare taxa may be important in elucidating the ecosystem stability and multifunctionality that are mediated by microbial communities.

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1462-2920.14945?campaign=wolacceptedarticle)

[Sci-hub](http://scihub.bban.top/10.1111/1462-2920.14945)

---

## Network analysis infers the wilt pathogen invasion associated with non-detrimental bacteria.
 19.02.2020, _Q Hu, L Tan, S Gu, Y Xiao, X Xiong, WA Zeng, K Feng, Z Wei, Y Deng_


The microbiota colonizing the root endophytic compartment and surrounding rhizosphere soils contribute to plant growth and health. However, the key members of plant soil and endophytic microbial communities involved in inhibiting or assisting pathogen invasion remain elusive. By utilizing 16S high-throughput sequencing and a molecular ecological network (MEN) approach, we systematically studied the interactions within bacterial communities in plant endophytic compartments (stem and root) and the surrounding soil (bulk and rhizosphere) during bacterial wilt invasion. The endophytic communities were found to be strongly influenced by pathogen invasion according to analysis of microbial diversity and community structure and composition. Endophytic communities of the infected plants were primarily derived from soil communities, as assessed by the SourceTracker program, but with rare migration from soil communities to endophytic communities observed in healthy plants. Soil and endophytic microbiomes from infected plants showed modular topology and greater complexity in network analysis, and a higher number of interactions than those in healthy plants. Furthermore, interactions among microbial members revealed that pathogenic Ralstonia members were positively correlated with several bacterial genera, including Delftia, Stenotrophomonas, Bacillus, Clostridium XlVa, Fontibacillus, Acidovorax, Herminiimonas, and three unclassified bacterial genera, in infected plant roots. Our findings indicated that the pathogen invasion in the rhizosphere and endophytic compartments may be highly associated with bacteria that are normally not detrimental, and sometimes even beneficial, to plants.

[Source](https://www.nature.com/articles/s41522-020-0117-2?utm_source=npjbiofilms_etoc&utm_medium=email&utm_campaign=toc_41522_6_1&utm_content=20200227&WT.ec_id=NPJBIOFILMS-202002&sap-outbound-id=406233CA0E331B8EFCFEFF4D68E39EBAD56535DE&mkt-key=005056B0331B1ED791CFE57296368737)

[Sci-hub](http://scihub.bban.top/10.1038/s41522-020-0117-2)

---

## Exploring the resistome, virulome and microbiome of drinking water in environmental and clinical settings.
 02.03.2020, _MF Dias, G da Rocha Fernandes, M Cristina de Paiva, A Christina de Matos Salim, AB Santos, AM Amaral Nascimento_


Aquatic ecosystems harbor a vast pool of antibiotic resistance genes (ARGs), which can suffer mutation, recombination and selection events. Here, we explored the diversity of ARGs, virulence factors and the bacterial community composition in water samples before (surface raw water, RW) and after (disinfected water, DW) drinking water conventional treatment, as well as in tap water (TW) and ultrafiltration membranes (UM, recovered from hemodialysis equipment) through metagenomics. A total of 852 different ARGs were identified, 21.8% of them only in RW, which might reflect the impact of human activities on the river at the sampling point. Although a similar resistance profile has been observed between the samples, significant differences in the frequency of clinically relevant antibiotic classes (penam and peptide) were identified. Resistance determinants to last resort antibiotics, including sequences related to mcr, optrA and poxtA and clinically relevant beta-lactamase genes (i.e. blaKPC, blaGES, blaIMP, blaVIM, blaSPM and blaNDM) were detected. 830 coding sequences (CDSs - related to 217 different ARGs) were embedded in contigs associated with mobile genetic elements, specially plasmids, of which 68% in RW, DW and TW, suggesting the importance of water environments in resistance dissemination. Shifts in bacterial pathogens genera were observed, such as a significant increase in Mycobacterium after treatment and distribution. In UM, the potentially pathogenic genus Halomonas predominated. Its draft genome was closely related to H. stevensii, hosting mainly multidrug efflux pumps. These results broaden our understanding of the global ARGs diversity and stress the importance of tracking the ever-expanding environmental resistome.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0043135420301664)

[Sci-hub](http://scihub.bban.top/10.1016/j.watres.2020.115630)

---

## Niche partitioning of methane-oxidizing bacteria along the oxygen-methane counter gradient of stratified lakes.
 26.02.2020, _MJ Mayr, M Zimmermann, C Guggenheim, A Brand, H Bürgmann_


Lakes are a significant source of atmospheric methane, although methane-oxidizing bacteria consume most methane diffusing upward from anoxic sediments. Diverse methane-oxidizing bacteria form an effective methane filter in the water column of stratified lakes, yet, niche partitioning of different methane-oxidizing bacteria along the oxygen-methane counter gradient remains poorly understood. In our study, we reveal vertical distribution patterns of active methane-oxidizing bacteria along the oxygen-methane counter gradient of four lakes, based on amplicon sequencing analysis of 16S rRNA and pmoA genes, and 16S rRNA and pmoA transcripts, and potential methane oxidation rates. Differential distribution patterns indicated that ecologically different methane-oxidizing bacteria occupied the methane-deficient and oxygen-deficient part above and below the oxygen-methane interface. The interface sometimes harbored additional taxa. Within the dominant Methylococcales, an uncultivated taxon (CABC2E06) occurred mainly under methane-deficient conditions, whereas Crenothrix-related taxa preferred oxygen-deficient conditions. Candidatus Methylomirabilis limnetica (NC10 phylum) abundantly populated the oxygen-deficient part in two of four lakes. We reason that the methane filter in lakes is structured and that methane-oxidizing bacteria may rely on niche-specific adaptations for methane oxidation along the oxygen-methane counter gradient. Niche partitioning of methane-oxidizing bacteria might support greater overall resource consumption, contributing to the high effectivity of the lacustrine methane filter.

[Source](https://www.nature.com/articles/s41396-019-0515-8?utm_source=feedburner&utm_medium=feed&utm_campaign=Feed%3A+ismej%2Frss%2Fcurrent+%28The+ISME+Journal+-+Current%29)

[Sci-hub](http://scihub.bban.top/10.1038/s41396-019-0515-8)

---

## A metagenomic assessment of microbial eukaryotic diversity in the global ocean.
 17.02.2020, _A Obiol, CR Giner, P Sánchez, CM Duarte, SG Acinas, R Massana_


Surveying microbial diversity and function is accomplished by combining complementary molecular tools. Among them, metagenomics is a PCR free approach that contains all genetic information from microbial assemblages and is today performed at a relatively large scale and reasonable cost, mostly based on very short reads. Here we investigated the potential of metagenomics to provide taxonomic reports of marine microbial eukaryotes. We prepared a curated database with reference sequences of the V4 region of 18S rDNA clustered at 97% similarity and used this database to extract and classify metagenomic reads. More than half of them were unambiguously affiliated to a unique reference whilst the rest could be assigned to a given taxonomic group. The overall diversity reported by metagenomics was similar to that obtained by amplicon sequencing of the V4 and V9 regions of the 18S rRNA gene, although either one or both of these amplicon surveys performed poorly for groups like Excavata, Amoebozoa, Fungi and Haptophyta. We then studied the diversity of picoeukaryotes and nanoeukaryotes using 91 metagenomes from surface down to bathypelagic layers in different oceans, unveiling a clear taxonomic separation between size fractions and depth layers. Finally, we retrieved long rDNA sequences from assembled metagenomes that improved phylogenetic reconstructions of particular groups. Overall, this study shows metagenomics as an excellent resource for taxonomic exploration of marine microbial eukaryotes.

[Source](https://onlinelibrary.wiley.com/doi/abs/10.1111/1755-0998.13147)

[Sci-hub](http://scihub.bban.top/10.1111/1755-0998.13147)

---

## Ocean acidification affects microbial community and invertebrate settlement on biofilms.
 01.03.2020, _KS Nelson, F Baltar, MD Lamare, SE Morales_


Increased atmospheric CO2 is driving ocean acidification (OA), and potential changes in marine ecosystems. Research shows that both planktonic and benthic communities are affected, but how these changes are linked remains unresolved. Here we show experimentally that decreasing seawater pH (from pH 8.1 to 7.8 and 7.4) leads to reduced biofilm formation and lower primary producer biomass within biofilms. These changes occurred concurrently with a re-arrangement of the biofilm microbial communities. Changes suggest a potential shift from autotrophic to heterotrophic dominated biofilms in response to reduced pH. In a complimentary experiment, biofilms reared under reduced pH resulted in altered larval settlement for a model species (Galeolaria hystrix). These findings show that there is a potential cascade of impacts arising from OA effects on biofilms that may drive important community shifts through altered settlement patterns of benthic species.

[Source](https://www.nature.com/articles/s41598-020-60023-4)

[Sci-hub](http://scihub.bban.top/10.1038/s41598-020-60023-4)

---

## Understanding and creating biocementing beachrocks via biostimulation of indigenous microbial communities.
 25.02.2020, _AL Ramachandran, P Polat, A Mukherjee, NK Dhami_


Bacterially induced precipitation of minerals leading to cementation of natural geological formations has been well recorded in a variety of environments. A range of microbial pathways and geochemical processes have been found to influence the cementation processes; but detailed formation mechanisms and biogeochemical relationships are still not very clear. There has been a growing demand for the application of bacterially driven biocementation in a number of geotechnical engineering applications recently. Here, we aimed to unpin the mechanisms behind the formation of actively mineralising beachrock sediments at Lucky Bay in Western Australia to understand the natural accretionary processes and potential of indigenous bacterial communities in biocementation. We observed ferruginous, aluminosilicate and carbonate cements along with extensive extra polymeric substances, borings with possible microbial activities in certain sections of native beachrock sediments. Cement precipitation under calcium- and iron-rich microenvironments sourced from seawater and iron creek seems to be driven by both biogenic and abiogenic processes in nature. Native microbial communities with a dominance of the genera Halococcus and Marinobacter were recorded. Enrichment of native bacterial communities under seawater media conditions was conducted which lead to successful biomineralisation of calcitic and ferruginous cements under in vitro conditions although the community composition changed significantly. Nanomechanical properties of natural and laboratory synthesised cement crystals showed that engineered biocement is highly promising. The results of this study clearly demonstrate biological influence in the formation of natural cements and hint significant potential of biostimulation which can be harnessed for different engineering applications including coastal erosion.

[Source](https://link.springer.com/article/10.1007/s00253-020-10474-6)

[Sci-hub](http://scihub.bban.top/10.1007/s00253-020-10474-6)

---

