## Development of point-of-need colourimetric, isothermal diagnostic assays for specific detection of Bacillus subtilis using shikimate dehydrogenase gene.
 27.09.2024, _N S, M C, SK T, RR S, K A, PM K, P D, SS N_


The largest obstacle in the promotion of biopesticides is the existence of counterfeit products available in the market. Identification and quantification of antagonistic organisms in biopesticide products are the key to the reduction of spurious microbial pesticides. In this study, we have developed a simple, sensitive, isothermal-based colourimetric assay for specific detection of Bacillus subtilis from the biopesticide formulations and soil samples. A region specific to B. subtilis which codes for shikimate dehydrogenase was identified through in silico analysis. We employed conventional PCR, loop-mediated isothermal amplification (LAMP), recombinase polymerase amplification (RPA), and qPCR for specific detection of B. subtilis in soil samples and biopesticide formulations. Specificity tests showed that the PCR primers amplified an amplicon of 521 bp in four strains of B. subtilis only, and no amplification was found in negative control samples. Similarly, the LAMP assay showed sky blue colour in all four strains of B. subtilis and violet colour in negative control samples. Whereas in the RPA assay, upon the addition of SYBR Green dye, a bright green colour was seen in B. subtilis strains, while a brick-red colour was observed in negative control samples by visualizing under a UV transilluminator. The qPCR assay showed specific amplifications with a Ct value of 12 for B. subtilis strains and no amplification in negative control samples. In the sensitivity test, PCR could amplify DNA of B. subtilis up to 500 pg/µL. DNA concentration as low as 10 pg/µL was enough to show the colour change in the LAMP as well as the RPA assays, whereas the qPCR assay showed sensitivity till 100 pg/µL. All four diagnostic assays developed in the study have been validated in soil samples and B. subtilis-based biopesticides. Compared to conventional PCR, the qPCR assay has the advantage of quantification and visualizing the result in real-time, whereas LAMP and RPA assays have the benefits of being colourimetric and less time-consuming. The other advantages are that the results can be visualized with the naked eye, and these assays do not require a costly thermal cycler and gel documentation system. Hence, LAMP and RPA assays are highly suitable for developing point-of-need diagnostic kits and, in turn, help regulators assess the quality of biopesticides in the market.

[Source](https://www.microbiologyresearch.org/content/microbial-primers)

10.1007/s12223-024-01201-z

---

## Microbial Primers

[Source](https://www.microbiologyresearch.org/content/microbial-primers)

---

## Phages reconstitute NAD+ to counter bacterial immunity.
 25.09.2024, _I Osterman, H Samra, F Rousset, E Loseva, M Itkin, S Malitsky, E Yirmiya, A Millman, R Sorek_


Bacteria defend against phage infection through a variety of antiphage defence systems1. Many defence systems were recently shown to deplete cellular nicotinamide adenine dinucleotide (NAD+) in response to infection, by cleaving NAD+ into ADP-ribose (ADPR) and nicotinamide2-7. It was demonstrated that NAD+ depletion during infection deprives the phage of this essential molecule and impedes phage replication. Here we show that a substantial fraction of phages possess enzymatic pathways allowing reconstitution of NAD+ from its degradation products in infected cells. We describe NAD+ reconstitution pathway 1 (NARP1), a two-step pathway in which one enzyme phosphorylates ADPR to generate ADPR pyrophosphate (ADPR-PP), and the second enzyme conjugates ADPR-PP and nicotinamide to generate NAD+. Phages encoding NARP1 can overcome a diverse set of defence systems, including Thoeris, DSR1, DSR2, SIR2-HerA and SEFIR, all of which deplete NAD+ as part of their defensive mechanism. Phylogenetic analyses show that NARP1 is primarily encoded on phage genomes, suggesting a phage-specific function in countering bacterial defences. A second pathway, NARP2, allows phages to overcome bacterial defences by building NAD+ using metabolites different from ADPR-PP. Our findings reveal a unique immune evasion strategy in which viruses rebuild molecules depleted by defence systems, thus overcoming host immunity.

[Source](https://www.nature.com/articles/s41586-024-07986-w)

10.1038/s41586-024-07986-w

---

## The direct inhibitory effects of Lactobacillus acidophilus, a commensal urinary bacterium, on calcium oxalate stone development.
 21.09.2024, _C Noonin, A Putpim, V Thongboonkerd_


_Lactobacillus acidophilus, Calcium Oxalate, Escherichia coli, Humans, Bacterial Adhesion, Kidney Calculi, Crystallization, Membrane Glycoproteins, Flagella_
Lactobacillus acidophilus is a commensal urinary bacterium found more abundantly in healthy individuals than in stone patients. Hence, it has been proposed to play an inhibitory role in kidney stone disease (KSD) but with unclear mechanisms. We therefore investigated the direct effects of L. acidophilus on calcium oxalate (CaOx) stone development compared with Escherichia coli, which is known to promote CaOx stone formation.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01877-y)

10.1186/s40168-024-01877-y

---

## Neonatal gut microbiota profile and the association with retinopathy of prematurity in preterm infants.
 25.09.2024, _YH Chang, YM Yeh, CC Lee, CH Chiu, HC Chen, YJ Hsueh, CW Lee, R Lien, SM Chu, MC Chiang, EY Kang, KJ Chen, NK Wang, L Liu, YS Hwang, CC Lai, WC Wu_


To explore the role of gut microbiota in preterm infants at high risk of developing retinopathy of prematurity (ROP).

[Source](https://onlinelibrary.wiley.com/doi/10.1111/ceo.14441)

10.1111/ceo.14441

---

## The wound microbiota: microbial mechanisms of impaired wound healing and infection.
 16.07.2024, _A Uberoi, A McCready-Vangi, EA Grice_


_Humans, Wound Healing, Microbiota, Wound Infection, Skin, Biofilms, Animals_
The skin barrier protects the human body from invasion by exogenous and pathogenic microorganisms. A breach in this barrier exposes the underlying tissue to microbial contamination, which can lead to infection, delayed healing, and further loss of tissue and organ integrity. Delayed wound healing and chronic wounds are associated with comorbidities, including diabetes, advanced age, immunosuppression and autoimmune disease. The wound microbiota can influence each stage of the multi-factorial repair process and influence the likelihood of an infection. Pathogens that commonly infect wounds, such as Staphylococcus aureus and Pseudomonas aeruginosa, express specialized virulence factors that facilitate adherence and invasion. Biofilm formation and other polymicrobial interactions contribute to host immunity evasion and resistance to antimicrobial therapies. Anaerobic organisms, fungal and viral pathogens, and emerging drug-resistant microorganisms present unique challenges for diagnosis and therapy. In this Review, we explore the current understanding of how microorganisms present in wounds impact the process of skin repair and lead to infection through their actions on the host and the other microbial wound inhabitants.

[Source](https://www.nature.com/articles/s41579-024-01035-z)

10.1038/s41579-024-01035-z

---

## Altered oral microbiota of drug-resistant organism carriers exhibit impaired gram-negative pathogen inhibition

[Source](https://www.biorxiv.org/content/10.1101/2024.09.24.614756v1.abstract)

---

## Emergence of community behaviors in the gut microbiota upon drug treatment.
 25.09.2024, _S Garcia-Santamarina, M Kuhn, S Devendran, L Maier, M Driessen, A Mateus, E Mastrorilli, AR Brochado, MM Savitski, KR Patil, M Zimmermann, P Bork, A Typas_


Pharmaceuticals can directly inhibit the growth of gut bacteria, but the degree to which such interactions manifest in complex community settings is an open question. Here, we compared the effects of 30 drugs on a 32-species synthetic community with their effects on each community member in isolation. While most individual drug-species interactions remained the same in the community context, communal behaviors emerged in 26% of all tested cases. Cross-protection during which drug-sensitive species were protected in community was 6 times more frequent than cross-sensitization, the converse phenomenon. Cross-protection decreased and cross-sensitization increased at higher drug concentrations, suggesting that the resilience of microbial communities can collapse when perturbations get stronger. By metabolically profiling drug-treated communities, we showed that both drug biotransformation and bioaccumulation contribute mechanistically to communal protection. As a proof of principle, we molecularly dissected a prominent case: species expressing specific nitroreductases degraded niclosamide, thereby protecting both themselves and sensitive community members.

[Source](https://www.sciencedirect.com/science/article/pii/S0092867424009668)

10.1016/j.cell.2024.08.037

---

## Microbial transformation of dietary xenobiotics shapes gut microbiome composition.
 25.09.2024, _EJ Culp, NT Nelson, AA Verdegaal, AL Goodman_


Diet is a major determinant of gut microbiome composition, and variation in diet-microbiome interactions may contribute to variation in their health consequences. To mechanistically understand these relationships, here we map interactions between ∼150 small-molecule dietary xenobiotics and the gut microbiome, including the impacts of these compounds on community composition, the metabolic activities of human gut microbes on dietary xenobiotics, and interindividual variation in these traits. Microbial metabolism can toxify and detoxify these compounds, producing emergent interactions that explain community-specific remodeling by dietary xenobiotics. We identify the gene and enzyme responsible for detoxification of one such dietary xenobiotic, resveratrol, and demonstrate that this enzyme contributes to interindividual variation in community remodeling by resveratrol. Together, these results systematically map interactions between dietary xenobiotics and the gut microbiome and connect toxification and detoxification to interpersonal differences in microbiome response to diet.

[Source](https://www.sciencedirect.com/science/article/pii/S009286742400967X)

10.1016/j.cell.2024.08.038

---

## Microglia and Gut Microbiota: A Double-Edged Sword in Alzheimer's Disease.
 25.09.2024, _N Bano, S Khan, S Ahamad, JS Kanshana, NJ Dar, S Khan, A Nazir, SA Bhat_


The strong association between gut microbiota (GM) and brain functions such as mood, behaviour, and cognition has been well documented. Gut-brain axis is a unique bidirectional communication system between the gut and brain, in which gut microbes play essential role in maintaining various molecular and cellular processes. GM interacts with the brain through various pathways and processes including, metabolites, vagus nerve, HPA axis, endocrine system, and immune system to maintain brain homeostasis. GM dysbiosis, or an imbalance in GM, is associated with several neurological disorders, including anxiety, depression, and Alzheimer's disease (AD). Conversely, AD is sustained by microglia-mediated neuroinflammation and neurodegeneration. Further, GM and their products also affect microglia-mediated neuroinflammation and neurodegeneration. Despite the evidence connecting GM dysbiosis and AD progression, the involvement of GM in modulating microglia-mediated neuroinflammation in AD remains elusive. Importantly, deciphering the mechanism/s by which GM regulates microglia-dependent neuroinflammation may be helpful in devising potential therapeutic strategies to mitigate AD. Herein, we review the current evidence regarding the involvement of GM dysbiosis in microglia activation and neuroinflammation in AD. We also discuss the possible mechanisms through which GM influences the functioning of microglia and its implications for therapeutic intervention. Further, we explore the potential of microbiota-targeted interventions, such as prebiotics, probiotics, faecal microbiota transplantation, etc., as a novel therapeutic strategy to mitigate neuroinflammation and AD progression. By understanding and exploring the gut-brain axis, we aspire to revolutionize the treatment of neurodegenerative disorders, many of which share a common theme of microglia-mediated neuroinflammation and neurodegeneration.

[Source](https://www.sciencedirect.com/science/article/pii/S1568163724003337)

10.1016/j.arr.2024.102515

---

## What defines a healthy gut microbiome?
 25.09.2024, _M Van Hul, PD Cani, C Petifils, WM De Vos, H Tilg, EM El Omar_


The understanding that changes in microbiome composition can influence chronic human diseases and the efficiency of therapies has driven efforts to develop microbiota-centred therapies such as first and next generation probiotics, prebiotics and postbiotics, microbiota editing and faecal microbiota transplantation. Central to microbiome research is understanding how disease impacts microbiome composition and vice versa, yet there is a problematic issue with the term 'dysbiosis', which broadly links microbial imbalances to various chronic illnesses without precision or definition. Another significant issue in microbiome discussions is defining 'healthy individuals' to ascertain what characterises a healthy microbiome. This involves questioning who represents the healthiest segment of our population-whether it is those free from illnesses, athletes at peak performance, individuals living healthily through regular exercise and good nutrition or even elderly adults or centenarians who have been tested by time and achieved remarkable healthy longevity.This review advocates for delineating 'what defines a healthy microbiome?' by considering a broader range of factors related to human health and environmental influences on the microbiota. A healthy microbiome is undoubtedly linked to gut health. Nevertheless, it is very difficult to pinpoint a universally accepted definition of 'gut health' due to the complexities of measuring gut functionality besides the microbiota composition. We must take into account individual variabilities, the influence of diet, lifestyle, host and environmental factors. Moreover, the challenge in distinguishing causation from correlation between gut microbiome and overall health is presented.The review also highlights the resource-heavy nature of comprehensive gut health assessments, which hinders their practicality and broad application. Finally, we call for continued research and a nuanced approach to better understand the intricate and evolving concept of gut health, emphasising the need for more precise and inclusive definitions and methodologies in studying the microbiome.

[Source](https://gut.bmj.com/content/early/2024/09/25/gutjnl-2024-333378)

10.1136/gutjnl-2024-333378

---

## The gut microbiome and chronic pain.
 27.09.2024, _C Watson_


_Animals, Humans, Mice, Chronic Pain, Gastrointestinal Microbiome, Endometriosis, Female_

[Source](https://www.nature.com/articles/d41586-024-03006-z)

10.1038/d41586-024-03006-z

---

## Acarbose enhances the efficacy of immunotherapy against solid tumours by modulating the gut microbiota.
 25.09.2024, _SL Zhang, X Wang, QQ Cai, C Chen, ZY Zhang, YY Xu, MX Yang, QA Jia, Y Wang, ZM Wang_


The crucial role of gut microbiota in shaping immunotherapy outcomes has prompted investigations into potential modulators. Here we show that oral administration of acarbose significantly increases the anti-tumour response to anti-PD-1 therapy in female tumour-bearing mice. Acarbose modulates the gut microbiota composition and tryptophan metabolism, thereby contributing to changes in chemokine expression and increased T cell infiltration within tumours. We identify CD8+ T cells as pivotal components determining the efficacy of the combined therapy. Further experiments reveal that acarbose promotes CD8+ T cell recruitment through the CXCL10-CXCR3 pathway. Faecal microbiota transplantation and gut microbiota depletion assays indicate that the effects of acarbose are dependent on the gut microbiota. Specifically, acarbose enhances the efficacy of anti-PD-1 therapy via the tryptophan catabolite indoleacetate, which promotes CXCL10 expression and thus facilitates CD8+ T cell recruitment, sensitizing tumours to anti-PD-1 therapy. The bacterial species Bifidobacterium infantis, which is enriched by acarbose, also improves response to anti-PD-1 therapy. Together, our study endorses the potential combination of acarbose and anti-PD-1 for cancer immunotherapy.

[Source](https://www.nature.com/articles/s42255-024-01137-1)

10.1038/s42255-024-01137-1

---

## Trait-based study predicts glycerol/diol dehydratases as a key function of the gut microbiota of hindgut-fermenting carnivores.
 22.09.2024, _Q Li, HJ Ruscheweyh, LH Østergaard, M Libertella, KS Simonsen, S Sunagawa, A Scoma, C Schwab_


_Animals, Gastrointestinal Microbiome, Feces, Fermentation, Glycerol, Metagenomics, Hydro-Lyases, Propylene Glycols, Vitamin B 12, Bacteria, Phylogeny, Animals, Wild_
Microbial pdu and cob-cbi-hem gene clusters encode the key enzyme glycerol/diol dehydratase (PduCDE), which mediates the transformation of dietary nutrients glycerol and 1,2-propanediol (1,2-PD) to a variety of metabolites, and enzymes for cobalamin synthesis, a co-factor and shared good of microbial communities. It was the aim of this study to relate pdu as a multipurpose functional trait to environmental conditions and microbial community composition. We collected fecal samples from wild animal species living in captivity with different gut physiology and diet (n = 55, in total 104 samples), determined occurrence and diversity of pdu and cob-cbi-hem using a novel approach combining metagenomics with quantification of metabolic and genetic biomarkers, and conducted in vitro fermentations to test for trait-based activity.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01863-4)

10.1186/s40168-024-01863-4

---

## Microbial community storm dynamics signal sources of "old" stream water.
 26.09.2024, _DR URycki, SP Good, BC Crump, NC Ceperley, JR Brooks_


_Rivers, Microbiota, RNA, Ribosomal, 16S, Water Microbiology, Oregon, Water Movements, Hydrology, Bacteria, Oxygen Isotopes_
Accurate characterization of the movement of water through catchments, particularly during precipitation event response, is critical for hydrological efforts such as contaminant transport modeling or prediction of extreme flows. Abiotic hydrogeochemical tracers are commonly used to track sources and ages of surface waters but provide limited details about transit pathways or the spatial dynamics of water storage and release. Alternatively, biotic material in streams is derived from thousands of taxa originating from a variety of environments within watersheds, including groundwater, sediment, and upslope terrestrial environments, and this material can be characterized with genetic sequencing and bioinformatics. We analyzed the stable water isotopes (δ18O and δ2H) and microbiome composition (16S rRNA gene amplicon sequencing) of the Marys River of western Oregon, USA during an early season storm to describe the processes, storage, and flowpaths that shape surface water hydrology. Stable water isotopes (δ18O and δ2H) typified an event response in which stream water is composed largely of 'old' water introduced to the catchment before the storm, a common though not well understood phenomenon. In contrast, microbial biodiversity spiked during the storm, consisting of early- and late-event communities clearly distinguishable from pre-event communities. We applied concentration-discharge (cQ) analysis to individual microbial taxa and found that most Alphaproteobacteria sequences were positively correlated (i.e., were mobilized) with discharge, whereas most sequences from phyla Gammaproteobacteria and Bacteroidota were negatively correlated with discharge (i.e., were diluted). Source predictions using the prokaryote habitat preference database ProkAtlas found that freshwater-associated microbes composed a smaller fraction of the microbial community during the stream rise and a larger fraction during the recession, while soil and biofilm-associated microbes increased during the storm and remained high during recession. This suggests that the "old" water discharged during the storm was likely stored and released from, or passed through, soil- and biofilm-rich environments, demonstrating that this approach adds new, biologically derived tracer information about the hydrologic pathways active during and after this event. Overall, this study demonstrates an approach for integrating information-rich DNA into water resource investigations, incorporating tools from both hydrology and microbiology to demonstrate that microbial DNA is useful not only as an indicator of biodiversity but also functions as an innovative hydrologic tracer.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0306896)

10.1371/journal.pone.0306896

---

## Hyperexpansion of genetic diversity and metabolic capacity of extremophilic bacteria and archaea in ancient Andean lake sediments.
 21.09.2024, _MÁ Lezcano, TLV Bornemann, L Sánchez-García, D Carrizo, PS Adam, SP Esser, NA Cabrol, AJ Probst, V Parro_


_Lakes, Archaea, Geologic Sediments, Bacteria, Genetic Variation, Chile, Phylogeny, Microbiota, Extremophiles, RNA, Ribosomal, 16S_
The Andean Altiplano hosts a repertoire of high-altitude lakes with harsh conditions for life. These lakes are undergoing a process of desiccation caused by the current climate, leaving terraces exposed to extreme atmospheric conditions and serving as analogs to Martian paleolake basins. Microbiomes in Altiplano lake terraces have been poorly studied, enclosing uncultured lineages and a great opportunity to understand environmental adaptation and the limits of life on Earth. Here we examine the microbial diversity and function in ancient sediments (10.3-11 kyr BP (before present)) from a terrace profile of Laguna Lejía, a sulfur- and metal/metalloid-rich saline lake in the Chilean Altiplano. We also evaluate the physical and chemical changes of the lake over time by studying the mineralogy and geochemistry of the terrace profile.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01878-x)

10.1186/s40168-024-01878-x

---

## Bronze Age cheese reveals human-Lactobacillus interactions over evolutionary history.
 26.09.2024, _Y Liu, B Miao, W Li, X Hu, F Bai, Y Abuduresule, Y Liu, Z Zheng, W Wang, Z Chen, S Zhu, X Feng, P Cao, W Ping, R Yang, Q Dai, F Liu, C Tian, Y Yang, Q Fu_


Despite the long history of consumption of fermented dairy, little is known about how the fermented microbes were utilized and evolved over human history. Here, by retrieving ancient DNA of Bronze Age kefir cheese (∼3,500 years ago) from the Xiaohe cemetery, we explored past human-microbial interactions. Although it was previously suggested that kefir was spread from the Northern Caucasus to Europe and other regions, we found an additional spreading route of kefir from Xinjiang to inland East Asia. Over evolutionary history, the East Asian strains gained multiple gene clusters with defensive roles against environmental stressors, which can be a result of the adaptation of Lactobacillus strains to various environmental niches and human selection. Overall, our results highlight the role of past human activities in shaping the evolution of human-related microbes, and such insights can, in turn, provide a better understanding of past human behaviors.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(24)00899-7)

10.1016/j.cell.2024.08.008

---

## A realistic benchmark for differential abundance testing and confounder adjustment in human microbiome studies.
 25.09.2024, _J Wirbel, M Essex, SK Forslund, G Zeller_


_Humans, Microbiota, Benchmarking, RNA, Ribosomal, 16S, Computer Simulation_
In microbiome disease association studies, it is a fundamental task to test which microbes differ in their abundance between groups. Yet, consensus on suitable or optimal statistical methods for differential abundance testing is lacking, and it remains unexplored how these cope with confounding. Previous differential abundance benchmarks relying on simulated datasets did not quantitatively evaluate the similarity to real data, which undermines their recommendations.

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03390-9)

10.1186/s13059-024-03390-9

---

## Quantifying Data Distortion in Bar Graphs in Biological Research

[Source](https://www.biorxiv.org/content/10.1101/2024.09.20.609464v1.abstract)

---

