## <em>Akkermansia muciniphila </em>and <em>Parabacteroides distasonis </em>as prognostic markers for relapse in ulcerative colitis patients

[Source](https://doi.org/10.3389/fcimb.2024.1367998)

---

## Obesity-enriched gut microbe degrades myo-inositol and promotes lipid absorption.
 12.07.2024, _C Wu, F Yang, H Zhong, J Hong, H Lin, M Zong, H Ren, S Zhao, Y Chen, Z Shi, X Wang, J Shen, Q Wang, M Ni, B Chen, Z Cai, M Zhang, Z Cao, K Wu, A Gao, J Li, C Liu, M Xiao, Y Li, J Shi, Y Zhang, X Xu, W Gu, Y Bi, G Ning, W Wang, J Wang, R Liu_


Numerous studies have reported critical roles for the gut microbiota in obesity. However, the specific microbes that causally contribute to obesity and the underlying mechanisms remain undetermined. Here, we conducted shotgun metagenomic sequencing in a Chinese cohort of 631 obese subjects and 374 normal-weight controls and identified a Megamonas-dominated, enterotype-like cluster enriched in obese subjects. Among this cohort, the presence of Megamonas and polygenic risk exhibited an additive impact on obesity. Megamonas rupellensis possessed genes for myo-inositol degradation, as demonstrated in vitro and in vivo, and the addition of myo-inositol effectively inhibited fatty acid absorption in intestinal organoids. Furthermore, mice colonized with M. rupellensis or E. coli heterologously expressing the myo-inositol-degrading iolG gene exhibited enhanced intestinal lipid absorption, thereby leading to obesity. Altogether, our findings uncover roles for M. rupellensis as a myo-inositol degrader that enhances lipid absorption and obesity, suggesting potential strategies for future obesity management.

[Source](https://doi.org/10.1016/j.chom.2024.06.012)

10.1016/j.chom.2024.06.012

---

## Gut microbiome in endometriosis: a cohort study on 1000 individuals

[Source](https://doi.org/10.1186/s12916-024-03503-y)

---

## Metagenomic analysis reveals altered gut virome and diagnostic potential in pancreatic cancer

[Source](https://doi.org/10.1002/jmv.29809)

---

## The tumor microbiome as a predictor of outcomes in patients with metastatic melanoma treated with immune checkpoint inhibitors

[Source](https://doi.org/10.1158/2767-9764.CRC-23-0170)

---

## Gut microbiome diversity within Clostridia is negatively associated with human obesity

[Source](https://doi.org/10.1128/msystems.00627-24)

---

## Early depletion of gut microbiota shape oligodendrocyte response after traumatic brain injury

[Source](https://doi.org/10.1186/s12974-024-03158-9)

---

## Effect of progestin-based contraceptives on HIV-associated vaginal immune biomarkers and microbiome in adolescent girls

[Source](https://doi.org/10.1371/journal.pone.0306237)

---

## Microbiome and pancreatic cancer: time to think about chemotherapy

[Source](https://doi.org/10.1080/19490976.2024.2374596)

---

## Host control of the microbiome: Mechanisms, evolution, and disease

[Source](https://doi.org/10.1126/science.adi3338)

---

## Connecting microbial community assembly and function

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1369527424000882?via%3Dihub)

---

## Microbiota of Saliva: A Non-invasive Diagnostic Tool

[Source](https://doi.org/10.1007/s12088-024-01219-4)

---

## Topical prebiotic nitrate: optimizing the ‘hang-time’, source and dose for specific oral or systemic effects

[Source](https://doi.org/10.1038/s41522-024-00528-2)

---

## An improved diet-based nutritional interventions can improve childhood obesity with the synergistic regulation of gut microbiota

[Source](https://brill.com/view/journals/bm/aop/article-10.1163-18762891-bja00019/article-10.1163-18762891-bja00019.xml)

---

## Intermittent fasting, calorie restriction, and a ketogenic diet improve mitochondrial function by reducing lipopolysaccharide signaling in monocytes during obesity: A randomized clinical trial

[Source](https://www.clinicalnutritionjournal.com/article/S0261-5614(24)00228-0/fulltext)

---

## Influence of the initial microbiota on eggplant <em>shibazuke </em>pickle and eggplant juice fermentation

[Source](https://doi.org/10.1128/spectrum.00464-24)

---

## Metatranscriptomics reveals microbial community function succession and characteristic flavor formation mechanisms during black rice wine fermentation

[Source](https://doi.org/10.1016/j.foodchem.2024.140428)

---

## Gα13 restricts nutrient driven proliferation in mucosal germinal centers

[Source](https://doi.org/10.1038/s41590-024-01910-0)

---

## Gut microbiota mediates ambient PM(2.5) exposure-induced abnormal glucose metabolism via short-chain fatty acids

[Source](https://doi.org/10.1016/j.jhazmat.2024.135096)

---

## Domestication and evolutionary histories of specialized gut symbionts across cephalotine ants

[Source](https://doi.org/10.1111/mec.17454)

---

## The interplay between gut symbionts and behavioral variation in social insects

[Source](https://doi.org/10.1016/j.cois.2024.101233)

---

## Meta-transcriptomic analysis of companion animal infectomes reveals their diversity and potential roles in animal and human disease.
 16.07.2024, _W-C Wu, Y-F Pan, W-D Zhou, Y-Q Liao, M-W Peng, G-Y Luo, G-Y Xin, Y-N Peng, T An, B Li, H Luo, VR Barrs, JA Beatty, EC Holmes, W Zhao, M Shi, Y Shu_


Companion animals such as cats and dogs harbor diverse microbial communities that can potentially impact human health due to close and frequent contact. To better characterize their total infectomes and assess zoonotic risks, we characterized the overall infectomes of companion animals (cats and dogs) and evaluated their potential zoonotic risks. Meta-transcriptomic analyses were performed on 239 samples from cats and dogs collected across China, identifying 24 viral species, 270 bacterial genera, and two fungal genera. Differences in the overall microbiome and infectome composition were compared across different animal species (cats or dogs), sampling sites (rectal or oropharyngeal), and health status (healthy or diseased). Diversity analyses revealed that viral abundance was generally higher in diseased animals compared to healthy ones, while differences in microbial composition were mainly driven by sampling site, followed by animal species and health status. Disease association analyses validated the pathogenicity of known pathogens and suggested potential pathogenic roles of previously undescribed bacteria and newly discovered viruses. Cross-species transmission analyses identified seven pathogens shared between cats and dogs, such as alphacoronavirus 1, which was detected in both oropharyngeal and rectal swabs albeit with differential pathogenicity. Further analyses showed that some viruses, like alphacoronavirus 1, harbored multiple lineages exhibiting distinct pathogenicity, tissue, or host preferences. Ultimately, a systematic evolutionary screening identified 27 potential zoonotic pathogens in this sample set, with far more bacterial than viral species, implying potential health threats to humans. Overall, our meta-transcriptomic analysis reveals a landscape of actively transcribing microorganisms in major companion animals, highlighting key pathogens, those with the potential for cross-species transmission, and possible zoonotic threats.

[Source](https://doi.org/10.1128/msphere.00439-24)

10.1128/msphere.00439-24

---

## Alcohol consumption during pregnancy differentially affects the fecal microbiota of dams and offspring

[Source](https://doi.org/10.1038/s41598-024-64313-z)

---

## Meta-omics analysis reveals the marine arsenic cycle driven by bacteria.
 18.07.2024, _ZJ Teng, J Li, P Wang, CY Li, M Peng, QL Qin, XL Chen, Y Chen, HH Fu, N Wang, YZ Zhang_


Arsenic is a toxic element widely distributed in the Earth's crust and ranked as a class I human carcinogen. Microbial metabolism makes significant contributions to arsenic detoxification, migration and transformation. Nowadays, research on arsenic is primarily in areas affected by arsenic pollution associated with human health activities. However, the biogeochemical traits of arsenic in the global marine ecosystem remain to be explicated. In this study, we revealed that seawater environments were primarily governed by the process of arsenate reduction to arsenite, while arsenite methylation was predominant in marine sediments which may serve as significant sources of arsenic emission into the atmosphere. Significant disparities existed in the distribution patterns of the arsenic cycle between surface and deep seawaters at middle and low latitudes, whereas these situations tend to be similar in the Arctic and Antarctic oceans. Significant variations were also observed in the taxonomic diversity and core microbial community of arsenic cycling across different marine environments. Specifically, γ-proteobacteria played a pivotal role in the arsenic cycle in the whole marine environment. Temperature, dissolved oxygen and phosphate were the crucial factors that related to these differentiations in seawater environments. Overall, our study contributes to a deeper understanding of the marine arsenic cycle.

[Source](https://doi.org/10.1016/j.jhazmat.2024.135137)

10.1016/j.jhazmat.2024.135137

---

## Using Machine Learning to Identify Environmental Factors that Collectively Determine Microbial Community Structure of Activated Sludge.
 18.07.2024, _L Wang, W Lu, Y Song, S Liu, YU Vincent Fu_


Activated sludge (AS) microbial communities are influenced by various environmental variables. However, a comprehensive analysis of how these variables jointly and nonlinearly shape the AS microbial community remains challenging. In this study, we employed advanced machine learning techniques to elucidate the collective effects of environmental variables on the structure and function of AS microbial communities. Applying Dirichlet Multinomial Mixtures analysis to 311 global AS samples, we identified four distinct microbial community types (AS-types), each characterized by unique microbial compositions and metabolic profiles. We used 14 classical linear and nonlinear machine learning methods to select a baseline model. The Extremely Randomized Trees demonstrated optimal performance in learning the relationship between environmental factors and AS types (with an accuracy of 71.43%). Feature selection identified critical environmental factors and their importance rankings, including latitude (Lat), longitude (Long), precipitation during sampling (Precip), solids retention time (SRT), effluent total nitrogen (Effluent TN), average temperature during sampling month (Avg Temp), mixed liquor temperature (Mixed Temp), influent biochemical oxygen demand (Influent BOD), and annual precipitation (Annual Precip). Significantly, Lat, Long, Precip, Avg Temp, and Annual Precip, influenced metabolic variations among AS types. These findings emphasize the pivotal role of environmental variables in shaping microbial community structures and enhancing metabolic pathways within activated sludge. Our study encourages the application of machine learning techniques to design artificial activated sludge microbial communities for specific environmental purposes.

[Source](https://doi.org/10.1016/j.envres.2024.119635)

10.1016/j.envres.2024.119635

---

## Microplastic pollution promotes soil respiration: A global-scale meta-analysis.
 16.07.2024, _S Zhao, MC Rillig, H Bing, Q Cui, T Qiu, Y Cui, J Penuelas, B Liu, S Bian, FA Monikh, J Chen, L Fang_


_Soil Microbiology, Microplastics, Soil, Carbon Dioxide, Soil Pollutants, Microbiota, Biomass, Carbon_
Microplastic (MP) pollution likely affects global soil carbon (C) dynamics, yet it remains uncertain how and to what extent MP influences soil respiration. Here, we report on a global meta-analysis to determine the effects of MP pollution on the soil microbiome and CO2 emission. We found that MP pollution significantly increased the contents of soil organic C (SOC) (21%) and dissolved organic C (DOC) (12%), the activity of fluorescein diacetate hydrolase (FDAse) (10%), and microbial biomass (17%), but led to a decrease in microbial diversity (3%). In particular, increases in soil C components and microbial biomass further promote CO2 emission (25%) from soil, but with a much higher effect of MPs on these emissions than on soil C components and microbial biomass. The effect could be attributed to the opposite effects of MPs on microbial biomass vs. diversity, as soil MP accumulation recruited some functionally important bacteria and provided additional C substrates for specific heterotrophic microorganisms, while inhibiting the growth of autotrophic taxa (e.g., Chloroflexi, Cyanobacteria). This study reveals that MP pollution can increase soil CO2 emission by causing shifts in the soil microbiome. These results underscore the potential importance of plastic pollution for terrestrial C fluxes, and thus climate feedbacks.

[Source](https://doi.org/10.1111/gcb.17415)

10.1111/gcb.17415

---

## SMMP: A Deep-Coverage Marine Metaproteome Method for Microbial Community Analysis throughout the Water Column Using 1 L of Seawater.
 13.07.2024, _S Wang, Z Zhang, K Yang, J Zhao, W Zhang, Z Wang, Z Liang, Y Zhang, Y Zhang, J Liu, L Zhang_


Marine microbes drive pivotal transformations in planetary-scale elemental cycles and have crucial impacts on global biogeochemical processes. Metaproteomics is a powerful tool for assessing the metabolic diversity and function of marine microbes. However, hundreds of liters of seawater are required for normal metaproteomic analysis due to the sparsity of microbial populations in seawater, which poses a substantial challenge to the widespread application of marine metaproteomics, particularly for deep seawater. Herein, a sensitive marine metaproteomics workflow, named sensitive marine metaproteome analysis (SMMP), was developed by integrating polycarbonate filter-assisted microbial enrichment, solid-phase alkylation-based anti-interference sample preparation, and narrow-bore nanoLC column for trace peptide separation and characterization. The method provided more than 8500 proteins from 1 L of bathypelagic seawater samples, which covered diverse microorganisms and crucial functions, e.g., the detection of key enzymes associated with the Wood-Ljungdahl pathway. Then, we applied SMMP to investigate vertical variations in the metabolic expression patterns of marine microorganisms from the euphotic zone to the bathypelagic zone. Methane oxidation and carbon monoxide (CO) oxidation were active processes, especially in the bathypelagic zone, which provided a remarkable energy supply for the growth and proliferation of heterotrophic microorganisms. In addition, marker protein profiles detected related to ammonia transport, ammonia oxidation, and carbon fixation highlighted that Thaumarchaeota played a critical role in primary production based on the coupled carbon-nitrogen process, contributing to the storage of carbon and nitrogen in the bathypelagic regions. SMMP has low microbial input requirements and yields in-depth metaproteome analysis, making it a prospective approach for comprehensive marine metaproteomic investigations.

[Source](https://doi.org/10.1021/acs.analchem.4c02079)

10.1021/acs.analchem.4c02079

---

## Metagenomic insights into microbial community structure and metabolism in alpine permafrost on the Tibetan Plateau.
 17.07.2024, _L Kang, Y Song, R Mackelprang, D Zhang, S Qin, L Chen, L Wu, Y Peng, Y Yang_


_Permafrost, Tibet, Soil Microbiology, RNA, Ribosomal, 16S, Metagenomics, Microbiota, Bacteria, Soil, Metagenome, Ecosystem, Climate Change, Biodiversity, Phylogeny_
Permafrost, characterized by its frozen soil, serves as a unique habitat for diverse microorganisms. Understanding these microbial communities is crucial for predicting the response of permafrost ecosystems to climate change. However, large-scale evidence regarding stratigraphic variations in microbial profiles remains limited. Here, we analyze microbial community structure and functional potential based on 16S rRNA gene amplicon sequencing and metagenomic data obtained from an ∼1000 km permafrost transect on the Tibetan Plateau. We find that microbial alpha diversity declines but beta diversity increases down the soil profile. Microbial assemblages are primarily governed by dispersal limitation and drift, with the importance of drift decreasing but that of dispersal limitation increasing with soil depth. Moreover, genes related to reduction reactions (e.g., ferric iron reduction, dissimilatory nitrate reduction, and denitrification) are enriched in the subsurface and permafrost layers. In addition, microbial groups involved in alternative electron accepting processes are more diverse and contribute highly to community-level metabolic profiles in the subsurface and permafrost layers, likely reflecting the lower redox potential and more complicated trophic strategies for microorganisms in deeper soils. Overall, these findings provide comprehensive insights into large-scale stratigraphic profiles of microbial community structure and functional potentials in permafrost regions.

[Source](https://doi.org/10.1038/s41467-024-50276-2)

10.1038/s41467-024-50276-2

---

## Top-down and bottom-up microbiome engineering approaches to enable biomanufacturing from waste biomass.
 13.07.2024, _X Lyu, M Nuhu, P Candry, J Wolfanger, M Betenbaugh, A Saldivar, C Zuniga, Y Wang, S Shrestha_


Growing environmental concerns and the need to adopt a circular economy have highlighted the importance of waste valorization for resource recovery. Microbial consortia-enabled biotechnologies have made significant developments in the biomanufacturing of valuable resources from waste biomass that serve as suitable alternatives to petrochemical-derived products. These microbial consortia are designed following a top-down or bottom-up engineering approach. The top-down approach is a classical method that uses environmental variables to selectively steer an existing microbial consortium to achieve a target function. While high-throughput sequencing has enabled microbial community characterization, the major challenge is to disentangle complex microbial interactions and manipulate the structure and function accordingly. Microbial consortia design through a bottom-up approach uses prior knowledge of the metabolic pathway and possible interactions among consortium partners to design and engineer synthetic microbial consortia. This strategy offers some control over the composition and function of the consortium for targeted bioprocesses, but challenges remain in optimal assembly methods and long-term stability. In this review, we present the recent advancements, challenges, and opportunities for further improvement using top-down and bottom-up approaches for microbiome engineering. As the bottom-up approach is relatively a new concept for waste valorization, this review explores the assembly and design of synthetic microbial consortia, ecological engineering principles to optimize microbial consortia, and metabolic engineering approaches for efficient conversion. Integration of top-down and bottom-up approaches along with developments in metabolic modeling to predict and optimize consortia function are also highlighted.

[Source](https://doi.org/10.1093/jimb/kuae025)

10.1093/jimb/kuae025

---

## GSearch: ultra-fast and scalable genome search by combining K-mer hashing with hierarchical navigable small world graphs.
 16.07.2024, _J Zhao, JP Both, LM Rodriguez-R, KT Konstantinidis_


Genome search and/or classification typically involves finding the best-match database (reference) genomes and has become increasingly challenging due to the growing number of available database genomes and the fact that traditional methods do not scale well with large databases. By combining k-mer hashing-based probabilistic data structures (i.e. ProbMinHash, SuperMinHash, Densified MinHash and SetSketch) to estimate genomic distance, with a graph based nearest neighbor search algorithm (Hierarchical Navigable Small World Graphs, or HNSW), we created a new data structure and developed an associated computer program, GSearch, that is orders of magnitude faster than alternative tools while maintaining high accuracy and low memory usage. For example, GSearch can search 8000 query genomes against all available microbial or viral genomes for their best matches (n = ∼318 000 or ∼3 000 000, respectively) within a few minutes on a personal laptop, using ∼6 GB of memory (2.5 GB via SetSketch). Notably, GSearch has an O(log(N)) time complexity and will scale well with billions of genomes based on a database splitting strategy. Further, GSearch implements a three-step search strategy depending on the degree of novelty of the query genomes to maximize specificity and sensitivity. Therefore, GSearch solves a major bottleneck of microbiome studies that require genome search and/or classification.

[Source](https://doi.org/10.1093/nar/gkae609)

10.1093/nar/gkae609

---

## Multi-omics machine learning to study host-microbiome interactions in early-onset colorectal cancer.
 20.07.2024, _TT Jayakrishnan, N Sangwan, SV Barot, N Farha, A Mariam, S Xiang, F Aucejo, M Conces, KG Nair, SS Krishnamurthi, SL Schmit, D Liska, DM Rotroff, AA Khorana, SD Kamath_


The incidence of early-onset colorectal cancer (eoCRC) is rising, and its pathogenesis is not completely understood. We hypothesized that machine learning utilizing paired tissue microbiome and plasma metabolome features could uncover distinct host-microbiome associations between eoCRC and average-onset CRC (aoCRC). Individuals with stages I-IV CRC (n = 64) were categorized as eoCRC (age ≤ 50, n = 20) or aoCRC (age ≥ 60, n = 44). Untargeted plasma metabolomics and 16S rRNA amplicon sequencing (microbiome analysis) of tumor tissue were performed. We fit DIABLO (Data Integration Analysis for Biomarker Discovery using Latent variable approaches for Omics studies) to construct a supervised machine-learning classifier using paired multi-omics (microbiome and metabolomics) data and identify associations unique to eoCRC. A differential association network analysis was also performed. Distinct clustering patterns emerged in multi-omic dimension reduction analysis. The metabolomics classifier achieved an AUC of 0.98, compared to AUC 0.61 for microbiome-based classifier. Circular correlation technique highlighted several key associations. Metabolites glycerol and pseudouridine (higher abundance in individuals with aoCRC) had negative correlations with Parasutterella, and Ruminococcaceae (higher abundance in individuals with eoCRC). Cholesterol and xylitol correlated negatively with Erysipelatoclostridium and Eubacterium, and showed a positive correlation with Acidovorax with higher abundance in individuals with eoCRC. Network analysis revealed different clustering patterns and associations for several metabolites e.g.: urea cycle metabolites and microbes such as Akkermansia. We show that multi-omics analysis can be utilized to study host-microbiome correlations in eoCRC and demonstrates promising biomarker potential of a metabolomics classifier. The distinct host-microbiome correlations for urea cycle in eoCRC may offer opportunities for therapeutic interventions.

[Source](https://doi.org/10.1038/s41698-024-00647-1)

10.1038/s41698-024-00647-1

---

## 3D printed rectal swabs for assessing the gut microbiome, metabolome and inflammation.
 21.07.2024, _RW Perry, BH Mullish, JL Alexander, R Shah, NP Danckert, JM Blanco, L Roberts, Z Liu, D Chrysostomou, ST Radhakrishnan, S Balarajah, R Barry, LC Hicks, HRT Williams, JR Marchesi_


_Humans, Gastrointestinal Microbiome, Feces, Metabolome, Inflammatory Bowel Diseases, Printing, Three-Dimensional, RNA, Ribosomal, 16S, Male, Female, Adult, Rectum, Leukocyte L1 Antigen Complex, Inflammation, Middle Aged, Specimen Handling_
Investigating the gut microbiome and metabolome frequently requires faecal samples, which can be difficult to obtain. Previous studies have shown that rectal swabs are comparable to faecal samples for analysing gut microbiota composition and key metabolites. In this study, 3D printed rectal swabs were compared with conventional flocked swabs and faecal samples, due to the potential advantages 3D printing as a technique offers for swab production and development. 16S rRNA gene sequencing, qPCR and metabolite profiling (using 1H-NMR spectroscopy) were performed on swab and faecal samples from healthy participants. Faecal calprotectin and total protein analysis were performed on samples from inflammatory bowel disease (IBD) patients. There were no significant differences between both swab types and faecal samples when assessing key measures of alpha and beta diversity, and differences in the abundance of major phyla. There was a strong correlation between both swab types and faecal samples for all combined metabolites detected by NMR. In IBD patients, there was no significant difference in faecal calprotectin and total protein levels between both swab types and faecal samples. These data lead us to conclude that 3D printed swabs are equivalent to flocked swabs for the analysis of the gut microbiome, metabolome and inflammation.

[Source](https://doi.org/10.1038/s41598-024-67457-0)

10.1038/s41598-024-67457-0

---

## AFITbin: a metagenomic contig binning method using aggregate l-mer frequency based on initial and terminal nucleotides.
 19.07.2024, _A Darabi, S Sobhani, R Aghdam, C Eslahchi_


_Metagenomics, Algorithms, Nucleotides, High-Throughput Nucleotide Sequencing, Software, Microbiota, Sequence Analysis, DNA, Cluster Analysis, Contig Mapping, Metagenome_
Using next-generation sequencing technologies, scientists can sequence complex microbial communities directly from the environment. Significant insights into the structure, diversity, and ecology of microbial communities have resulted from the study of metagenomics. The assembly of reads into longer contigs, which are then binned into groups of contigs that correspond to different species in the metagenomic sample, is a crucial step in the analysis of metagenomics. It is necessary to organize these contigs into operational taxonomic units (OTUs) for further taxonomic profiling and functional analysis. For binning, which is synonymous with the clustering of OTUs, the tetra-nucleotide frequency (TNF) is typically utilized as a compositional feature for each OTU.

[Source](https://doi.org/10.1186/s12859-024-05859-7)

10.1186/s12859-024-05859-7

---

## JASPER: Fast, powerful, multitrait association testing in structured samples gives insight on pleiotropy in gene expression.
 18.07.2024, _J Mbatchou, MS McPeek_


Joint association analysis of multiple traits with multiple genetic variants can provide insight into genetic architecture and pleiotropy, improve trait prediction, and increase power for detecting association. Furthermore, some traits are naturally high-dimensional, e.g., images, networks, or longitudinally measured traits. Assessing significance for multitrait genetic association can be challenging, especially when the sample has population sub-structure and/or related individuals. Failure to adequately adjust for sample structure can lead to power loss and inflated type 1 error, and commonly used methods for assessing significance can work poorly with a large number of traits or be computationally slow. We developed JASPER, a fast, powerful, robust method for assessing significance of multitrait association with a set of genetic variants, in samples that have population sub-structure, admixture, and/or relatedness. In simulations, JASPER has higher power, better type 1 error control, and faster computation than existing methods, with the power and speed advantage of JASPER increasing with the number of traits. JASPER is potentially applicable to a wide range of association testing applications, including for multiple disease traits, expression traits, image-derived traits, and microbiome abundances. It allows for covariates, ascertainment, and rare variants and is robust to phenotype model misspecification. We apply JASPER to analyze gene expression in the Framingham Heart Study, where, compared to alternative approaches, JASPER finds more significant associations, including several that indicate pleiotropic effects, most of which replicate previous results, while others have not previously been reported. Our results demonstrate the promise of JASPER for powerful multitrait analysis in structured samples.

[Source](https://www.cell.com/ajhg/abstract/S0002-9297(24)00216-7)

10.1016/j.ajhg.2024.06.010

---

## The nature of the last universal common ancestor and its impact on the early Earth system.
 12.07.2024, _ERR Moody, S Álvarez-Carretero, TA Mahendrarajah, JW Clark, HC Betts, N Dombrowski, LL Szánthó, RA Boyle, S Daines, X Chen, N Lane, Z Yang, GA Shields, GJ Szöllősi, A Spang, D Pisani, TA Williams, TM Lenton, PCJ Donoghue_


The nature of the last universal common ancestor (LUCA), its age and its impact on the Earth system have been the subject of vigorous debate across diverse disciplines, often based on disparate data and methods. Age estimates for LUCA are usually based on the fossil record, varying with every reinterpretation. The nature of LUCA's metabolism has proven equally contentious, with some attributing all core metabolisms to LUCA, whereas others reconstruct a simpler life form dependent on geochemistry. Here we infer that LUCA lived ~4.2 Ga (4.09-4.33 Ga) through divergence time analysis of pre-LUCA gene duplicates, calibrated using microbial fossils and isotope records under a new cross-bracing implementation. Phylogenetic reconciliation suggests that LUCA had a genome of at least 2.5 Mb (2.49-2.99 Mb), encoding around 2,600 proteins, comparable to modern prokaryotes. Our results suggest LUCA was a prokaryote-grade anaerobic acetogen that possessed an early immune system. Although LUCA is sometimes perceived as living in isolation, we infer LUCA to have been part of an established ecological system. The metabolism of LUCA would have provided a niche for other microbial community members and hydrogen recycling by atmospheric photochemistry could have supported a modestly productive early ecosystem.

[Source](https://doi.org/10.1038/s41559-024-02461-1)

10.1038/s41559-024-02461-1

---

