## Airway microecology in rifampicin-resistant and rifampicin-sensitive pulmonary tuberculosis patients.
 29.11.2022, _X Cai, Y Luo, Y Zhang, Y Lin, B Wu, Z Cao, Z Hu, X Wu, S Tan_


Pulmonary tuberculosis is a chronic infectious disease of the respiratory system. It is still one of the leading causes of death from a single infectious disease, but it has been stuck in the study of a single pathogen. Recent studies have shown that many diseases are associated with disruption of the native microbiota. In this study we investigated the occurrence of tuberculosis and the correlation between drug resistance and respiratory flora. High-throughput 16 S rRNA gene sequencing was used to characterize the respiratory microbiota composition of 30 tuberculosis (TB) affected patients and compared with 30 healthy (H) controls. According to their Gene Xpert results, 30 pulmonary tuberculosis patients were divided into 12 persons in the drug-sensitive group (DS0) and 18 persons in the drug-resistant group (DR0). The microbial flora of the two were compared with the H group.

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-022-02705-9)

10.1186/s12866-022-02705-9

---

## Oral Bacteriome and Mycobiome across Stages of Oral Carcinogenesis.
 29.11.2022, _W Heng, W Wang, T Dai, P Jiang, Y Lu, R Li, M Zhang, R Xie, Y Zhou, M Zhao, N Duan, Z Ye, F Yan, X Wang_


Oral microbial dysbiosis contributes to the development of oral squamous cell carcinoma (OSCC). Numerous studies have focused on variations in the oral bacterial microbiota of patients with OSCC. However, similar studies on fungal microbiota, another integral component of the oral microbiota, are scarce. Moreover, there is an evidence gap regarding the role that microecosystems play in different niches of the oral cavity at different stages of oral carcinogenesis. Here, we catalogued the microbial communities in the human oral cavity by profiling saliva, gingival plaque, and mucosal samples at different stages of oral carcinogenesis. We analyzed the oral bacteriome and mycobiome along the health-premalignancy-carcinoma sequence. Some species, including Prevotella intermedia, Porphyromonas endodontalis, Acremonium exuviarum, and Aspergillus fumigatus, were enriched, whereas others, such as Streptococcus salivarius subsp. salivarius, Scapharca broughtonii, Mortierella echinula, and Morchella septimelata, were depleted in OSCC. These findings suggest that an array of signature species, including bacteria and fungi, are closely associated with oral carcinogenesis. OSCC-associated diversity differences, species distinction, and functional alterations were most remarkable in mucosal samples, not in gingival plaque or saliva samples, suggesting an urgent need to define oral carcinogenesis-associated microbial dysbiosis based on the spatial microbiome. IMPORTANCE Abundant oral microorganisms constitute a complex microecosystem within the oral environment of the host, which plays a critical role in the adjustment of various physiological and pathological states of the oral cavity. In this study, we demonstrated that variations in the "core microbiome" may be used to predict carcinogenesis. In addition, sample data collected from multiple oral sites along the health-premalignancy-carcinoma sequence increase our understanding of the microecosystems of different oral niches and their specific changes during oral carcinogenesis. This work provides insight into the roles of bacteria and fungi in OSCC and may contribute to the development of early diagnostic assays and novel treatments.

[Source](https://journals.asm.org/doi/10.1128/spectrum.02737-22)

10.1128/spectrum.02737-22

---

## Double-edged sword: impact of fecal microbiome transplants on the gut resistome

[Source](https://journals.lww.com/co-gastroenterology/Abstract/9900/Double_edged_sword__impact_of_fecal_microbiome.36.aspx)

---

## Targeted suppression of human IBD-associated gut microbiota commensals by phage consortia for treatment of intestinal inflammation.
 31.10.2022, _S Federici, S Kredo-Russo, R Valdés-Mas, D Kviatcovsky, E Weinstock, Y Matiuhin, Y Silberberg, K Atarashi, M Furuichi, A Oka, B Liu, M Fibelman, IN Weiner, E Khabra, N Cullin, N Ben-Yishai, D Inbar, H Ben-David, J Nicenboim, N Kowalsman, W Lieb, E Kario, T Cohen, YF Geffen, L Zelcbuch, A Cohen, U Rappo, I Gahali-Sass, M Golembo, V Lev, M Dori-Bachash, H Shapiro, C Moresi, A Cuevas-Sierra, G Mohapatra, L Kern, D Zheng, SP Nobs, J Suez, N Stettner, A Harmelin, N Zak, S Puttagunta, M Bassan, K Honda, H Sokol, C Bang, A Franke, C Schramm, N Maharshak, RB Sartor, R Sorek, E Elinav_


_Animals, Bacteriophages, Colitis, Gastrointestinal Microbiome, Humans, Inflammation, Inflammatory Bowel Diseases, Klebsiella pneumoniae, Mice_
Human gut commensals are increasingly suggested to impact non-communicable diseases, such as inflammatory bowel diseases (IBD), yet their targeted suppression remains a daunting unmet challenge. In four geographically distinct IBD cohorts (n = 537), we identify a clade of Klebsiella pneumoniae (Kp) strains, featuring a unique antibiotics resistance and mobilome signature, to be strongly associated with disease exacerbation and severity. Transfer of clinical IBD-associated Kp strains into colitis-prone, germ-free, and colonized mice enhances intestinal inflammation. Stepwise generation of a lytic five-phage combination, targeting sensitive and resistant IBD-associated Kp clade members through distinct mechanisms, enables effective Kp suppression in colitis-prone mice, driving an attenuated inflammation and disease severity. Proof-of-concept assessment of Kp-targeting phages in an artificial human gut and in healthy volunteers demonstrates gastric acid-dependent phage resilience, safety, and viability in the lower gut. Collectively, we demonstrate the feasibility of orally administered combination phage therapy in avoiding resistance, while effectively inhibiting non-communicable disease-contributing pathobionts.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0092867422008509)

10.1016/j.cell.2022.07.003

---

## Gut microbiome alterations in preclinical Alzheimer’s disease

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0278276)

---

## Sex, puberty, and the gut microbiome

[Source](https://rep.bioscientifica.com/view/journals/rep/aop/rep-22-0303/rep-22-0303.xml)

---

## Lactobacillus reuteri tryptophan metabolism promotes host susceptibility to CNS autoimmunity.
 30.11.2022, _TL Montgomery, K Eckstrom, KH Lile, S Caldwell, ER Heney, KG Lahue, A D'Alessandro, MJ Wargo, DN Krementsov_


_Mice, Animals, Lactobacillus reuteri, Autoimmunity, Tryptophan, Multiple Sclerosis, Indoles_
Dysregulation of gut microbiota-associated tryptophan metabolism has been observed in patients with multiple sclerosis. However, defining direct mechanistic links between this apparent metabolic rewiring and individual constituents of the gut microbiota remains challenging. We and others have previously shown that colonization with the gut commensal and putative probiotic species, Lactobacillus reuteri, unexpectedly enhances host susceptibility to experimental autoimmune encephalomyelitis (EAE), a murine model of multiple sclerosis. To identify underlying mechanisms, we characterized the genome of commensal L. reuteri isolates, coupled with in vitro and in vivo metabolomic profiling, modulation of dietary substrates, and gut microbiota manipulation.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01408-7)

10.1186/s40168-022-01408-7

---

## Genomic and Environmental Factors Shape the Active Gill Bacterial Community of an Amazonian Teleost Holobiont.
 29.11.2022, _FÉ Sylvain, N Leroux, É Normandeau, A Holland, S Bouslama, PL Mercier, A Luis Val, N Derome_


Fish bacterial communities provide functions critical for their host's survival in contrasting environments. These communities are sensitive to environmental-specific factors (i.e., physicochemical parameters, bacterioplankton), and host-specific factors (i.e., host genetic background). The relative contribution of these factors shaping Amazonian fish bacterial communities is largely unknown. Here, we investigated this topic by analyzing the gill bacterial communities of 240 wild flag cichlids (Mesonauta festivus) from 4 different populations (genetic clusters) distributed across 12 sites in 2 contrasting water types (ion-poor/acidic black water and ion-rich/circumneutral white water). Transcriptionally active gill bacterial communities were characterized by a 16S rRNA metabarcoding approach carried on RNA extractions. They were analyzed using comprehensive data sets from the hosts genetic background (Genotyping-By-Sequencing), the bacterioplankton (16S rRNA) and a set of 34 environmental parameters. Results show that the taxonomic structure of 16S rRNA gene transcripts libraries were significantly different between the 4 genetic clusters and also between the 2 water types. However, results suggest that the contribution of the host's genetic background was relatively weak in comparison to the environment-related factors in structuring the relative abundance of different active gill bacteria species. This finding was also confirmed by a mixed-effects modeling analysis, which indicated that the dissimilarity between the taxonomic structure of bacterioplanktonic communities possessed the best explicative power regarding the dissimilarity between gill bacterial communities' structure, while pairwise fixation indexes (FST) from the hosts' genetic data only had a weak explicative power. We discuss these results in terms of bacterial community assembly processes and flag cichlid fish ecology. IMPORTANCE Host-associated microbial communities respond to factors specific to the host physiology, genetic backgrounds, and life history. However, these communities also show different degrees of sensitivity to environment-dependent factors, such as abiotic physico-chemical parameters and ecological interactions. The relative importance of host- versus environment-associated factors in shaping teleost bacterial communities is still understudied and is paramount for their conservation and aquaculture. Here, we studied the relative importance of host- and environment-associated factors structuring teleost bacterial communities using gill samples from a wild Amazonian teleost model (Mesonauta festivus) sampled in contrasting habitats along a 1500 km section of the Amazonian basin, thus ensuring high genetic diversity. Results showed that the contribution of the host's genetic background was weak compared to environment-related bacterioplanktonic communities in shaping gill bacterial assemblages, thereby suggesting that our understanding of teleost microbiome assembly could benefit from further studies focused on the ecological interplay between host-associated and free-living communities.

[Source](https://journals.asm.org/doi/10.1128/spectrum.02064-22)

10.1128/spectrum.02064-22

---

## Lack of host phylogenetic structure in the gut bacterial communities of New Zealand cicadas and their interspecific hybrids.
 29.11.2022, _D Haji, J Vailionis, M Stukel, E Gordon, EM Lemmon, AR Lemmon, C Simon_


Host-microbe interactions are intimately linked to eukaryotic evolution, particularly in sap-sucking insects that often rely on obligate microbial symbionts for nutrient provisioning. Cicadas (Cicadidae: Auchenorrhyncha) specialize on xylem fluid and derive many essential amino acids and vitamins from intracellular bacteria or fungi (Hodgkinia, Sulcia, and Ophiocordyceps) that are propagated via transmission from mothers to offspring. Despite the beneficial role of these non-gut symbionts in nutrient provisioning, the role of beneficial microbiota within the gut remains unclear. Here, we investigate the relative abundance and impact of host phylogeny and ecology on gut microbial diversity in cicadas using 16S ribosomal RNA gene amplicon sequencing data from 197 wild-collected cicadas and new mitochondrial genomes across 38 New Zealand cicada species, including natural hybrids between one pair of two species. We find low abundance and a lack of phylogenetic structure and hybrid effects but a significant role of elevation in explaining variation in gut microbiota.

[Source](https://www.nature.com/articles/s41598-022-24723-3)

10.1038/s41598-022-24723-3

---

## Unveiling the Wheat Microbiome under Varied Agricultural Field Conditions.
 29.11.2022, _S Jaiswal, B Aneja, J Jagannadham, B Pandey, RS Chhokar, SC Gill, OP Ahlawat, A Kumar, UB Angadi, A Rai, R Tiwari, MA Iquebal, D Kumar_


Wheat being the important staple food crop plays a significant role in nutritional security. A wide variety of microbial communities beneficial to plants and contributing to plant health and production are found in the rhizosphere. The wheat microbiome encompasses an extensive variety of microbial species playing a key role in sustaining the physiology of the crop, nutrient uptake, and biotic/abiotic stress resilience. This report presents wheat microbiome analysis under six different farm practices, namely, organic (Org), timely sown (TS), wheat after pulse crop (WAPC), temperature-controlled phenotyping facility (TCPF), maize-wheat cropping system (MW), and residue burnt field (Bur), using 16S rRNA sequencing methodology. The soil samples collected from either side of the wheat row were mixed to get a final sample set for DNA extraction under each condition. After the data preprocessing, microbial community analysis was performed, followed by functional analysis and annotation. An abundance of the phylum Proteobacteria was observed, followed by Acidobacteria, Actinobacteria, and Gemmatimonadetes in the majority of the samples, while relative abundance was found to vary at the genus level. Analysis against the Carbohydrate-Active Enzymes (CAZy) database showed a high number of glycoside hydrolase genes in the TS, TCPF, and WAPC samples, while the Org, MW, and Bur samples predominantly had glycosyltransferase genes and carbohydrate esterase genes were in the lowest numbers. Also, the Org and TCPF samples showed lower diversity, while rare and abundant species ranged from 12 to 25% and 20 to 32% of the total bacterial species in all the sets, respectively. These variations indicate that the different cropping sequence had a significant impact on soil microbial diversity and community composition, which characterizes its economic and environmental value as a sustainable agricultural approach to maintaining food security and ecosystem health. IMPORTANCE This investigation examined the wheat microbiome under six different agricultural field conditions to understand the role of cropping pattern on soil microbial diversity. This study also elaborated the community composition, which has importance in economic (role of beneficial community leading to higher production) and environmental (role of microbial diversity/community in safeguarding the soil health, etc.) arenas. This could lead to a sustainable farming approach for food security and improved ecosystem health. Also, the majority of the microbes are unculturable; hence, technology-based microcultivation will be a potential approach for harnessing other cultured microorganisms, leading to unique species for commercial production. The outcome of this research-accelerated work can provide an idea to the scientists/breeders/agronomists/pathologists under the mentioned field conditions regarding their influence over their crops.

[Source](https://journals.asm.org/doi/10.1128/spectrum.02633-22)

10.1128/spectrum.02633-22

---

## Changes of the microbial community in kiwifruit during storage after postharvest application of Wickerhamomyces anomalus.
 30.11.2022, _Q Zhao, Y Shi, G Legrand Ngolong Ngea, X Zhang, Q Yang, Q Zhang, X Xu, H Zhang_


_Fruit, Actinidia, Saccharomycetales, Microbiota_
High-throughput sequencing techniques can provide important information for understanding the interaction between exogenous microbial agents and fruit microbial communities, and explain how it controls postharvest fungal diseases. In this study, we found that Wickerhamomyces anomalus could control the postharvest disease of kiwifruit. Meanwhile, high-throughput sequencing technology results showed that the composition and structure changes of the fungal community in microbial flora were significantly greater than those of bacteria after W. anomalus treated. W. anomalus could colonize inside the fruit and regulate the community composition of bacteria to reduce the abundance of pathogens and eventually maintain the healthy state of the fruit. The dominant genus in the microbiota of kiwifruit after application of W. anomalus showed an increased ability to interact. Some fungi or bacteria are positively associated with yeast in the epiphytic and endophytic sample communities, guiding the synthesis of compound biocontrol strains for kiwifruit postharvest diseases.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0308814622025559)

10.1016/j.foodchem.2022.134593

---

## A dark matter in sake brewing: Origin of microbes producing a Kimoto-style fermentation starter

[Source](https://www.biorxiv.org/content/10.1101/2022.11.28.518188v1.abstract)

---

