## Incorporating plasmid biology and metagenomics into a holistic model of the human gut microbiome.
 01.04.2023, _SR Stockdale, C Hill_


The human gut microbiome is often described as the collection of bacteria, archaea, fungi, protists, and viruses associated with an individual, with no acknowledgement of the plasmid constituents. However, like viruses, plasmids are autonomous intracellular replicating entities that can influence the genotype and phenotype of their host and mediate trans-kingdom interactions. Plasmids are frequently noted as vehicles for horizontal gene transfer and for spreading antibiotic resistance, yet their multifaceted contribution to mutualistic and antagonistic interactions within the human microbiome and impact on human health is overlooked. In this review, we highlight the importance of plasmids and their biological properties as overlooked components of microbiomes. Subsequent human microbiome studies should include dedicated analyses of plasmids, particularly as a holistic understanding of human-microbial interactions is required before effective and safe interventions can be implemented to improve human well-being.

[Source](https://www.sciencedirect.com/science/article/pii/S1369527423000449?via%3Dihub)

10.1016/j.mib.2023.102307

---

## No evidence for a common blood microbiome based on a population study of 9,770 healthy humans

[Source](https://doi.org/10.1038/s41564-023-01350-w)

---

## Genetic correlations between Alzheimer's disease and gut microbiome genera.
 04.04.2023, _D Cammann, Y Lu, MJ Cummings, ML Zhang, JM Cue, J Do, J Ebersole, X Chen, EC Oh, JL Cummings, J Chen_


_Humans, Alzheimer Disease, Gastrointestinal Microbiome, Genome-Wide Association Study, Microbiota, Apolipoproteins E_
A growing body of evidence suggests that dysbiosis of the human gut microbiota is associated with neurodegenerative diseases like Alzheimer's disease (AD) via neuroinflammatory processes across the microbiota-gut-brain axis. The gut microbiota affects brain health through the secretion of toxins and short-chain fatty acids, which modulates gut permeability and numerous immune functions. Observational studies indicate that AD patients have reduced microbiome diversity, which could contribute to the pathogenesis of the disease. Uncovering the genetic basis of microbial abundance and its effect on AD could suggest lifestyle changes that may reduce an individual's risk for the disease. Using the largest genome-wide association study of gut microbiota genera from the MiBioGen consortium, we used polygenic risk score (PRS) analyses with the "best-fit" model implemented in PRSice-2 and determined the genetic correlation between 119 genera and AD in a discovery sample (ADc12 case/control: 1278/1293). To confirm the results from the discovery sample, we next repeated the PRS analysis in a replication sample (GenADA case/control: 799/778) and then performed a meta-analysis with the PRS results from both samples. Finally, we conducted a linear regression analysis to assess the correlation between the PRSs for the significant genera and the APOE genotypes. In the discovery sample, 20 gut microbiota genera were initially identified as genetically associated with AD case/control status. Of these 20, three genera (Eubacterium fissicatena as a protective factor, Collinsella, and Veillonella as a risk factor) were independently significant in the replication sample. Meta-analysis with discovery and replication samples confirmed that ten genera had a significant correlation with AD, four of which were significantly associated with the APOE rs429358 risk allele in a direction consistent with their protective/risk designation in AD association. Notably, the proinflammatory genus Collinsella, identified as a risk factor for AD, was positively correlated with the APOE rs429358 risk allele in both samples. Overall, the host genetic factors influencing the abundance of ten genera are significantly associated with AD, suggesting that these genera may serve as biomarkers and targets for AD treatment and intervention. Our results highlight that proinflammatory gut microbiota might promote AD development through interaction with APOE. Larger datasets and functional studies are required to understand their causal relationships.

[Source](https://doi.org/10.1038/s41598-023-31730-5)

10.1038/s41598-023-31730-5

---

## Limitations of using 16S rRNA microbiome sequencing to predict oral squamous cell carcinoma.
 31.03.2023, _C Delaney, CLR Veena, MC Butcher, W McLean, SMA Shaban, CJ Nile, G Ramage_


A new era of next generation sequencing has changed our perception of the oral microbiome in health and disease, and with this there is a growing understanding that the oral microbiome is a contributing factor to oral squamous cell carcinoma (OSCC), a malignancy of the oral cavity. This study aimed to analyse the trends and relevant literature based on the 16S rRNA oral microbiome in head and neck cancer using next generation sequencing technologies, and to conduct a meta-analysis of the studies with OSCC cases and healthy controls.

[Source](https://doi.org/10.1111/apm.13315)

10.1111/apm.13315

---

## Mendelian randomization analyses reveal causal relationships between the human microbiome and longevity.
 31.03.2023, _X Liu, L Zou, C Nie, Y Qin, X Tong, J Wang, H Yang, X Xu, X Jin, L Xiao, T Zhang, J Min, Y Zeng, H Jia, Y Hou_


_Aged, 80 and over, Humans, Longevity, Genome-Wide Association Study, Mendelian Randomization Analysis, Microbiota, Lactobacillus acidophilus_
Although recent studies have revealed the association between the human microbiome especially gut microbiota and longevity, their causality remains unclear. Here, we assess the causal relationships between the human microbiome (gut and oral microbiota) and longevity, by leveraging bidirectional two-sample Mendelian randomization (MR) analyses based on genome-wide association studies (GWAS) summary statistics of the gut and oral microbiome from the 4D-SZ cohort and longevity from the CLHLS cohort. We found that some disease-protected gut microbiota such as Coriobacteriaceae and Oxalobacter as well as the probiotic Lactobacillus amylovorus were related to increased odds of longevity, whereas the other gut microbiota such as colorectal cancer pathogen Fusobacterium nucleatum, Coprococcus, Streptococcus, Lactobacillus, and Neisseria were negatively associated with longevity. The reverse MR analysis further revealed genetically longevous individuals tended to have higher abundances of Prevotella and Paraprevotella but lower abundances of Bacteroides and Fusobacterium species. Few overlaps of gut microbiota-longevity interactions were identified across different populations. We also identified abundant links between the oral microbiome and longevity. The additional analysis suggested that centenarians genetically had a lower gut microbial diversity, but no difference in oral microbiota. Our findings strongly implicate these bacteria to play a role in human longevity and underscore the relocation of commensal microbes among different body sites that would need to be monitored for long and healthy life.

[Source](https://doi.org/10.1038/s41598-023-31115-8)

10.1038/s41598-023-31115-8

---

## Genomics of Invasive <em>Cutibacterium acnes</em> Isolates from Deep-Seated Infections

[Source](https://doi.org/10.1128/spectrum.04740-22)

---

## Vaginal Microbiome Metagenome Inference Accuracy: Differential Measurement Error according to Community Composition.
 31.03.2023, _KA Carter, AA Fodor, JE Balkus, A Zhang, MG Serrano, GA Buck, SM Engel, MC Wu, S Sun_


Several studies have compared metagenome inference performance in different human body sites; however, none specifically reported on the vaginal microbiome. Findings from other body sites cannot easily be generalized to the vaginal microbiome due to unique features of vaginal microbial ecology, and investigators seeking to use metagenome inference in vaginal microbiome research are "flying blind" with respect to potential bias these methods may introduce into analyses. We compared the performance of PICRUSt2 and Tax4Fun2 using paired 16S rRNA gene amplicon sequencing and whole-metagenome sequencing data from vaginal samples from 72 pregnant individuals enrolled in the Pregnancy, Infection, and Nutrition (PIN) cohort. Participants were selected from those with known birth outcomes and adequate 16S rRNA gene amplicon sequencing data in a case-control design. Cases experienced early preterm birth (&lt;32 weeks of gestation), and controls experienced term birth (37 to 41 weeks of gestation). PICRUSt2 and Tax4Fun2 performed modestly overall (median Spearman correlation coefficients between observed and predicted KEGG ortholog [KO] relative abundances of 0.20 and 0.22, respectively). Both methods performed best among Lactobacillus crispatus-dominated vaginal microbiotas (median Spearman correlation coefficients of 0.24 and 0.25, respectively) and worst among Lactobacillus iners-dominated microbiotas (median Spearman correlation coefficients of 0.06 and 0.11, respectively). The same pattern was observed when evaluating correlations between univariable hypothesis test P values generated with observed and predicted metagenome data. Differential metagenome inference performance across vaginal microbiota community types can be considered differential measurement error, which often causes differential misclassification. As such, metagenome inference will introduce hard-to-predict bias (toward or away from the null) in vaginal microbiome research. IMPORTANCE Compared to taxonomic composition, the functional potential within a bacterial community is more relevant to establishing mechanistic understandings and causal relationships between the microbiome and health outcomes. Metagenome inference attempts to bridge the gap between 16S rRNA gene amplicon sequencing and whole-metagenome sequencing by predicting a microbiome's gene content based on its taxonomic composition and annotated genome sequences of its members. Metagenome inference methods have been evaluated primarily among gut samples, where they appear to perform fairly well. Here, we show that metagenome inference performance is markedly worse for the vaginal microbiome and that performance varies across common vaginal microbiome community types. Because these community types are associated with sexual and reproductive outcomes, differential metagenome inference performance will bias vaginal microbiome studies, obscuring relationships of interest. Results from such studies should be interpreted with substantial caution and the understanding that they may over- or underestimate associations with metagenome content.

[Source](https://doi.org/10.1128/msystems.01003-22)

10.1128/msystems.01003-22

---

## Interspecies Interactions within the Host: the Social Network of Group B <em>Streptococcus</em>

[Source](https://doi.org/10.1128/iai.00440-22)

---

## Kefir as a therapeutic agent in clinical research: a scoping review.
 30.03.2023, _MK Bessa, GR Bessa, RR Bonamigo_


Increasing research has been conducted on the role of probiotics in disease treatment. Kefir, a safe, low-cost probiotic fermented milk drink, has been investigated in many in vitro and animal studies, although parameters for human therapeutic dose or treatment time have not yet been determined. To perform a scoping review of clinical studies that have used kefir as a therapeutic agent, compiling the results for perspectives to support and direct further research. This review was based on Joanna Briggs Institute guidelines, including studies on the effects of kefir-fermented milk in humans. Using the term KEFIR, the main international databases were searched for studies published in English, Spanish, or Portuguese until March 9, 2022. A total of 5835 articles were identified in the 4 databases, with 44 eligible for analysis. The research areas were classified as metabolic syndrome and type 2 diabetes, gastrointestinal health/disorders, maternal/child health and pediatrics, dentistry, oncology, women's and geriatric health, and dermatology. The many study limitations hampered generalization of the results. The small sample sizes, methodological variation, and differences in kefir types, dosage, and treatment duration prevented clear conclusions about its benefits for specific diseases. We suggest using a standard therapeutic dose of traditionally prepared kefir in milliliters according to body weight, making routine consumption more feasible. The studies showed that kefir is safe for people without serious illnesses.

[Source](https://doi.org/10.1017/S0954422423000070)

10.1017/S0954422423000070

---

## Interaction between β-glucans and gut microbiota: a comprehensive review.
 28.03.2023, _R Karimi, M Homayoonfal, N Malekjani, MS Kharazmi, SM Jafari_


Gut microbiota (GMB) in humans plays a crucial role in health and diseases. Diet can regulate the composition and function of GMB which are associated with different human diseases. Dietary fibers can induce different health benefits through stimulation of beneficial GMB. β-glucans (BGs) as dietary fibers have gained much interest due to their various functional properties. They can have therapeutic roles on gut health based on modulation of GMB, intestinal fermentation, production of different metabolites, and so on. There is an increasing interest in food industries in commercial application of BG as a bioactive substance into food formulations. The aim of this review is considering the metabolizing of BGs by GMB, effects of BGs on the variation of GMB population, influence of BGs on the gut infections, prebiotic effects of BGs in the gut, in vivo and in vitro fermentation of BGs and effects of processing on BG fermentability.

[Source](https://doi.org/10.1080/10408398.2023.2192281)

10.1080/10408398.2023.2192281

---

## Combination of vitamin D and probiotics inhibits chemically induced colorectal carcinogenesis in Wistar rats.
 01.04.2023, _CS de Oliveira, MM Baptistella, AP Siqueira, MO Carvalho, LF Ramos, BS Souto, LA de Almeida, EG Dos Santos, RD Novaes, ESC Nogueira, PF de Oliveira_


The modulation of inflammatory elements, cell differentiation and proliferation by vitamin D and the role of probiotics in the intestinal microbiota and immunogenic response have sparked interest in the application of both in chemotherapeutics and chemoprevention of colorectal tumors.

[Source](https://doi.org/10.1016/j.lfs.2023.121617)

10.1016/j.lfs.2023.121617

---

## Recent advancements in coral health, microbiome interactions and climate change.
 03.04.2023, _N Ashraf, A Anas, V Sukumaran, G Gopinath, KK Idrees Babu, PK Dinesh Kumar_


Corals are the visible indicators of the disasters induced by global climate change and anthropogenic activities and have become a highly vulnerable ecosystem on the verge of extinction. Multiple stressors could act individually or synergistically which results in small to large scale tissue degradation, reduced coral covers, and makes the corals vulnerable to various diseases. The coralline diseases are like the Chicken pox in humans because they spread hastily throughout the coral ecosystem and can devastate the coral cover formed over centuries in an abbreviated time. The extinction of the entire reef ecosystem will alter the ocean and earth's amalgam of biogeochemical cycles causing a threat to the entire planet. The current manuscript provides an overview of the recent advancement in coral health, microbiome interactions and climate change. Culture dependent and independent approaches in studying the microbiome of corals, the diseases caused by microorganisms, and the reservoirs of coral pathogens are also discussed. Finally, we discuss the possibilities of protecting the coral reefs from diseases through microbiome transplantation and the capabilities of remote sensing in monitoring their health status.

[Source](https://doi.org/10.1016/j.scitotenv.2023.163085)

10.1016/j.scitotenv.2023.163085

---

## Detection of specific uncultured bacteriophages by fluorescence in situ hybridisation in pig microbiome.
 03.04.2023, _LJ Ostenfeld, P Munk, FM Aarestrup, S Otani_


_Animals, Bacteria, Bacteriophages, Fluorescence, Metagenome, Microbiota, Swine_
Microbial communities have huge impacts on their ecosystems and local environments spanning from marine and soil communities to the mammalian gut. Bacteriophages (phages) are important drivers of population control and diversity in the community, but our understanding of complex microbial communities is halted by biased detection techniques. Metagenomics have provided a method of novel phage discovery independent of in vitro culturing techniques and have revealed a large proportion of understudied phages. Here, five jumbophage genomes, that were previously assembled in silico from pig faecal metagenomes, are detected and observed directly in their natural environment using a modified phageFISH approach, and combined with methods to decrease bias against large-sized phages (e.g., jumbophages). These phages are uncultured with unknown hosts. The specific phages were detected by PCR and fluorescent in situ hybridisation in their original faecal samples as well as across other faecal samples. Co-localisation of bacterial signals and phage signals allowed detection of the different stages of phage life cycle. All phages displayed examples of early infection, advanced infection, burst, and free phages. To our knowledge, this is the first detection of jumbophages in faeces, which were investigated independently of culture, host identification, and size, and based solely on the genome sequence. This approach opens up opportunities for characterisation of novel in silico phages in vivo from a broad range of gut microbiomes.

[Source](https://doi.org/10.1371/journal.pone.0283676)

10.1371/journal.pone.0283676

---

## Microbial Species-Area Relationships on the Skins of Amphibian Hosts.
 30.03.2023, _F Yang, Z Liu, J Zhou, X Guo, Y Chen_


Unlike species-area relationships (SARs) that have been widely reported for plants and animals on Earth, there is no clear understanding of the SARs for microorganisms. In this study, 358 specimens of 10 amphibian host species collected from the rural Chengdu region of southwest China were selected as island models for evaluating SAR curve shapes and assessing the skin microbiota from different amphibian species. The results showed that skin microbial diversity, measured using Hill's number, presented significant differences between hosts, but the difference was insignificant between habitat-specific classifications of hosts. As for microbial SARs, other than the classical power-law (PL) model describing an expected steady increase in microbial diversity as sampled skin area increases, two additional trends were observed: (i) microbial diversity first rises and gradually decreases after reaching a maximum accrual diversity (MaxAD) and (ii) microbial diversity decreases and starts to rise after reaching the minimum accrual diversity (MinAD). Among the four SAR statistical models compared, it was consistently found that the models that can describe MaxAD were favorably selected in the highest frequency. Models that can describe MinAD and PL model also performed reasonably well. However, PL had the poorest fitting power, implying the necessity of introducing biologically meaningful complex SAR models in microbial diversity research. In conclusion, through multihost analyses, our study provided compelling evidence that microbial SARs are complex and nonlinear. A variety of ecological mechanisms may be used for explaining these, including, but not limited to, community saturation, small-island effects, or sampling heterogeneity. IMPORTANCE In this study, we investigate species-area relationships (SARs) for skin-borne symbiotic microbes of wildlife hosts. Unlike the traditional SARs for plants and animals, symbiotic microbial SARs were complex. We found that both U-shaped and inverted U-shaped SAR models were widely favored for microbial taxa than the well-known power-law model in different host species. These favored models presented interesting statistical features, including minimal or maximal accrual diversity or inflection point. We provide intuitive derivations of these statistical properties. We showed that different habitat-specific amphibian hosts did not present distinct microbial diversity and skin-related SAR patterns. We predicted that approximately 600 to 1,400 cm2 (in two-dimensional [2D] measurement) or approximately 1,200 to 3 500 cm2 (in 3D measurement) are the skin area threshold range that can allow the emergence of minimal or maximal accrual microbial diversity with high chances. Finally, we list a variety of ecological mechanisms that may be used for explaining the observed nonlinear SAR trends.

[Source](https://doi.org/10.1128/spectrum.01771-22)

10.1128/spectrum.01771-22

---

## Intestinal microbiota of Nearctic-Neotropical migratory birds vary more over seasons and years than between host species.
 28.03.2023, _HR Skeen, DE Willard, AW Jones, BM Winger, EF Gyllenhaal, BR Tsuru, SJ Hackett, J Novembre_


Seasonal migration of Nearctic-Neotropical passerine birds may have profound effects on the diversity and abundance of their host-associated microbiota. Migratory birds experience seasonal change in environments and diets throughout the course of the annual cycle that, along with recurrent biological events such as reproduction, may significantly impact their microbiota. In this study, we characterize the intestinal microbiota of four closely related species of migratory Catharus thrushes at three time points of their migratory cycle: during spring migration, on the summer breeding territories and during fall migration. Using observations replicated over 3 years, we determined that microbial community diversity of Catharus thrushes was significantly different across distinct time periods of the annual cycle, whereas community composition was more similar within than across years. Elevated alpha diversity in the summer birds compared to either migratory period indicated that birds may harbour a reduced microbiota during active migration. We also found that community composition of the microbiota did not substantially differ between host species. Finally, we recovered two phyla, Cyanobacteria and Planctomycetota, which are not commonly described from birds, that were in relatively high abundance in specific years. This study contributes to our growing understanding of how microbiota in wild birds vary throughout disparate ecological conditions and reveals potential axes across which an animal's microbial flexibility adapts to variable environments and recurrent biological conditions throughout the annual cycle.

[Source](https://doi.org/10.1111/mec.16915)

10.1111/mec.16915

---

## A unique case in which Kimoto-style fermentation was completed with <em>Leuconostoc</em> as the dominant genus without transitioning to <em>Lactobacillus</em>

[Source](https://doi.org/10.1016/j.jbiosc.2023.03.007)

---

## Holobiont Urbanism: sampling urban beehives reveals cities' metagenomes.
 31.03.2023, _E Hénaff, D Najjar, M Perez, R Flores, C Woebken, CE Mason, K Slavin_


Over half of the world's population lives in urban areas with, according to the United Nations, nearly 70% expected to live in cities by 2050. Our cities are built by and for humans, but are also complex, adaptive biological systems involving a diversity of other living species. The majority of these species are invisible and constitute the city's microbiome. Our design decisions for the built environment shape these invisible populations, and as inhabitants we interact with them on a constant basis. A growing body of evidence shows us that human health and well-being are dependent on these interactions. Indeed, multicellular organisms owe meaningful aspects of their development and phenotype to interactions with the microorganisms-bacteria or fungi-with which they live in continual exchange and symbiosis. Therefore, it is meaningful to establish microbial maps of the cities we inhabit. While the processing and sequencing of environmental microbiome samples can be high-throughput, gathering samples is still labor and time intensive, and can require mobilizing large numbers of volunteers to get a snapshot of the microbial landscape of a city.

[Source](https://doi.org/10.1186/s40793-023-00467-z)

10.1186/s40793-023-00467-z

---

## Contamination detection and microbiome exploration with GRIMER.
 01.04.2023, _VC Piro, BY Renard_


_Microbiota, Biomass, Metadata_
Contamination detection is a important step that should be carefully considered in early stages when designing and performing microbiome studies to avoid biased outcomes. Detecting and removing true contaminants is challenging, especially in low-biomass samples or in studies lacking proper controls. Interactive visualizations and analysis platforms are crucial to better guide this step, to help to identify and detect noisy patterns that could potentially be contamination. Additionally, external evidence, like aggregation of several contamination detection methods and the use of common contaminants reported in the literature, could help to discover and mitigate contamination.

[Source](https://doi.org/10.1093/gigascience/giad017)

10.1093/gigascience/giad017

---

## Annotation-free discovery of functional groups in microbial communities.
 30.03.2023, _X Shan, A Goyal, R Gregor, OX Cordero_


Recent studies have shown that microbial communities are composed of groups of functionally cohesive taxa whose abundance is more stable and better-associated with metabolic fluxes than that of any individual taxon. However, identifying these functional groups in a manner that is independent of error-prone functional gene annotations remains a major open problem. Here we tackle this structure-function problem by developing a novel unsupervised approach that coarse-grains taxa into functional groups, solely on the basis of the patterns of statistical variation in species abundances and functional read-outs. We demonstrate the power of this approach on three distinct datasets. On data of replicate microcosms with heterotrophic soil bacteria, our unsupervised algorithm recovered experimentally validated functional groups that divide metabolic labour and remain stable despite large variation in species composition. When leveraged against the ocean microbiome data, our approach discovered a functional group that combines aerobic and anaerobic ammonia oxidizers whose summed abundance tracks closely with nitrate concentrations in the water column. Finally, we show that our framework can enable the detection of species groups that are probably responsible for the production or consumption of metabolites abundant in animal gut microbiomes, serving as a hypothesis-generating tool for mechanistic studies. Overall, this work advances our understanding of structure-function relationships in complex microbiomes and provides a powerful approach to discover functional groups in an objective and systematic manner.

[Source](https://www.nature.com/articles/s41559-023-02021-z)

10.1038/s41559-023-02021-z

---

## ViroProfiler: a containerized bioinformatics pipeline for viral metagenomic data analysis.
 03.04.2023, _J Ru, M Khan Mirzaei, J Xue, X Peng, L Deng_


_Software, Gastrointestinal Microbiome, Reproducibility of Results, Metagenome, Microbiota, Metagenomics, Computational Biology, Data Analysis_
Bacteriophages play central roles in the maintenance and function of most ecosystems by regulating bacterial communities. Yet, our understanding of their diversity remains limited due to the lack of robust bioinformatics standards. Here we present ViroProfiler, an in-silico workflow for analyzing shotgun viral metagenomic data. ViroProfiler can be executed on a local Linux computer or cloud computing environments. It uses the containerization technique to ensure computational reproducibility and facilitate collaborative research. ViroProfiler is freely available at https://github.com/deng-lab/viroprofiler.

[Source](https://doi.org/10.1080/19490976.2023.2192522)

10.1080/19490976.2023.2192522

---

## Sequence-Based Characterization of Microalgal Microbiomes: Impact of DNA Extraction Protocol on Yield and Community Composition.
 28.03.2023, _L Roager, EC Sonnenschein, L Gram_


The bacterial communities associated with microalgae are vital for the growth and health of the host, and engineering algal microbiomes can enhance the fitness of the algae. Characterization of these microbiomes mostly relies on sequencing of DNA, which can be extracted with an array of protocols that potentially impact DNA quantity and quality and thus potentially affect subsequent analyses of microbiome composition. Here, we extracted DNA from Isochrysis galbana, Tetraselmis suecica, and Conticribra weissflogii microbiomes using four different protocols. DNA yield and quality was greatly impacted by the choice of extraction protocol, whereas microbiome composition determined by 16S rRNA gene amplicon sequencing was only impacted to a minor degree, with microalgal host species being the main determinant of microbiome composition. The I. galbana microbiome was dominated by the genus Alteromonas, whereas the microbiome associated with T. suecica was dominated by Marinobacteraceae and Rhodobacteraceae family members. While these two families were also prevalent in the microbiome associated with C. weissflogii, Flavobacteriaceae and Cryomorphaceae were also highly dominant. Phenol-chloroform extraction resulted in higher DNA quality and quantity compared to commercial kits; however, because they have other advantages such as high throughput and low toxicity, commercial kits can be employed to great benefit for the characterization of microalgal microbiomes. IMPORTANCE Microalgae are very important as primary producers in the ocean, but also as forthcoming sustainable producers of biotechnologically interesting compounds. Accordingly, the bacterial microbiomes associated with microalgae are attracting increasing attention due to their effects on the growth and health of microalgae. Since most members of these microbiomes cannot be cultured, knowledge about community composition is best obtained using sequencing-based methods. This study evaluates the impact of DNA extraction methods on DNA quantity and quality along with sequence-based characterization of the bacterial microbiome composition of three microalgae: Isochrysis galbana, Tetraselmis suecica, and Conticribra weissflogii.

[Source](https://doi.org/10.1128/spectrum.03408-22)

10.1128/spectrum.03408-22

---

## Identifying keystone species in microbial communities using deep learning.
 30.03.2023, _XW Wang, Z Sun, H Jia, S Michel-Mata, MT Angulo, L Dai, X He, ST Weiss, YY Liu_


Previous studies suggested that microbial communities harbor keystone species whose removal can cause a dramatic shift in microbiome structure and functioning. Yet, an efficient method to systematically identify keystone species in microbial communities is still lacking. This is mainly due to our limited knowledge of microbial dynamics and the experimental and ethical difficulties of manipulating microbial communities. Here, we propose a Data-driven Keystone species Identification (DKI) framework based on deep learning to resolve this challenge. Our key idea is to implicitly learn the assembly rules of microbial communities from a particular habitat by training a deep learning model using microbiome samples collected from this habitat. The well-trained deep learning model enables us to quantify the community-specific keystoneness of each species in any microbiome sample from this habitat by conducting a thought experiment on species removal. We systematically validated this DKI framework using synthetic data generated from a classical population dynamics model in community ecology. We then applied DKI to analyze human gut, oral microbiome, soil, and coral microbiome data. We found that those taxa with high median keystoneness across different communities display strong community specificity, and many of them have been reported as keystone taxa in literature. The presented DKI framework demonstrates the power of machine learning in tackling a fundamental problem in community ecology, paving the way for the data-driven management of complex microbial communities.

[Source](https://doi.org/10.1101/2023.03.15.532858)

10.1101/2023.03.15.532858

---

