## Extending and improving metagenomic taxonomic profiling with uncharacterized species using MetaPhlAn 4.
 24.02.2023, _A Blanco-Míguez, F Beghini, F Cumbo, LJ McIver, KN Thompson, M Zolfo, P Manghi, L Dubois, KD Huang, AM Thomas, WA Nickols, G Piccinno, E Piperni, M Punčochář, M Valles-Colomer, A Tett, F Giordano, R Davies, J Wolf, SE Berry, TD Spector, EA Franzosa, E Pasolli, F Asnicar, C Huttenhower, N Segata_


Metagenomic assembly enables new organism discovery from microbial communities, but it can only capture few abundant organisms from most metagenomes. Here we present MetaPhlAn 4, which integrates information from metagenome assemblies and microbial isolate genomes for more comprehensive metagenomic taxonomic profiling. From a curated collection of 1.01 M prokaryotic reference and metagenome-assembled genomes, we define unique marker genes for 26,970 species-level genome bins, 4,992 of them taxonomically unidentified at the species level. MetaPhlAn 4 explains ~20% more reads in most international human gut microbiomes and &gt;40% in less-characterized environments such as the rumen microbiome and proves more accurate than available alternatives on synthetic evaluations while also reliably quantifying organisms with no cultured isolates. Application of the method to &gt;24,500 metagenomes highlights previously undetected species to be strong biomarkers for host conditions and lifestyles in human and mouse microbiomes and shows that even previously uncharacterized species can be genetically profiled at the resolution of single microbial strains.

[Source](https://doi.org/10.1038/s41587-023-01688-w)

10.1038/s41587-023-01688-w

---

## Gut microbial metabolism of 5-ASA diminishes its clinical efficacy in inflammatory bowel disease.
 24.02.2023, _RS Mehta, JR Mayers, Y Zhang, A Bhosle, NR Glasser, LH Nguyen, W Ma, S Bae, T Branck, K Song, L Sebastian, JA Pacheco, HS Seo, C Clish, S Dhe-Paganon, AN Ananthakrishnan, EA Franzosa, EP Balskus, AT Chan, C Huttenhower_


For decades, variability in clinical efficacy of the widely used inflammatory bowel disease (IBD) drug 5-aminosalicylic acid (5-ASA) has been attributed, in part, to its acetylation and inactivation by gut microbes. Identification of the responsible microbes and enzyme(s), however, has proved elusive. To uncover the source of this metabolism, we developed a multi-omics workflow combining gut microbiome metagenomics, metatranscriptomics and metabolomics from the longitudinal IBDMDB cohort of 132 controls and patients with IBD. This associated 12 previously uncharacterized microbial acetyltransferases with 5-ASA inactivation, belonging to two protein superfamilies: thiolases and acyl-CoA N-acyltransferases. In vitro characterization of representatives from both families confirmed the ability of these enzymes to acetylate 5-ASA. A cross-sectional analysis within the discovery cohort and subsequent prospective validation within the independent SPARC IBD cohort (n = 208) found three of these microbial thiolases and one acyl-CoA N-acyltransferase to be epidemiologically associated with an increased risk of treatment failure among 5-ASA users. Together, these data address a longstanding challenge in IBD management, outline a method for the discovery of previously uncharacterized gut microbial activities and advance the possibility of microbiome-based personalized medicine.

[Source](https://doi.org/10.1038/s41591-023-02217-7)

10.1038/s41591-023-02217-7

---

## Interpersonal variability of the human gut virome confounds disease signal detection in IBD.
 01.03.2023, _SR Stockdale, AN Shkoporov, EV Khokhlova, KM Daly, SA McDonnell, O O' Regan, JA Nolan, TDS Sutton, AG Clooney, FJ Ryan, D Sheehan, A Lavelle, LA Draper, F Shanahan, RP Ross, C Hill_


_Humans, Virome, Gastrointestinal Microbiome, Viruses, Colitis, Ulcerative, Inflammatory Bowel Diseases_
Viruses are increasingly recognised as important components of the human microbiome, fulfilling numerous ecological roles including bacterial predation, immune stimulation, genetic diversification, horizontal gene transfer, microbial interactions, and augmentation of metabolic functions. However, our current view of the human gut virome is tainted by previous sequencing requirements that necessitated the amplification of starting nucleic acids. In this study, we performed an original longitudinal analysis of 40 healthy control, 19 Crohn's disease, and 20 ulcerative colitis viromes over three time points without an amplification bias, which revealed and highlighted the interpersonal individuality of the human gut virome. In contrast to a 16 S rRNA gene analysis of matched samples, we show that α- and β-diversity metrics of unamplified viromes are not as efficient at discerning controls from patients with inflammatory bowel disease. Additionally, we explored the intrinsic properties of unamplified gut viromes and show there is considerable interpersonal variability in viral taxa, infrequent longitudinal persistence of intrapersonal viruses, and vast fluctuations in the abundance of temporal viruses. Together, these properties of unamplified faecal viromes confound the ability to discern disease associations but significantly advance toward an unbiased and accurate representation of the human gut virome.

[Source](https://doi.org/10.1038/s42003-023-04592-w)

10.1038/s42003-023-04592-w

---

## Clinical characteristics, diagnosis, outcomes and lung microbiome analysis of invasive pulmonary aspergillosis in the community-acquired pneumonia patients.
 02.03.2023, _Z Ao, H Xu, M Li, H Liu, M Deng, Y Liu_


_Humans, Invasive Pulmonary Aspergillosis, Retrospective Studies, Sensitivity and Specificity, Bronchoalveolar Lavage Fluid, Aspergillosis, Lung, Pneumonia, Microbiota_
Invasive pulmonary aspergillosis (IPA) remains underestimated in patients with community-acquired pneumonia (CAP). This study aims to describe clinical features and outcomes of IPA in CAP patients, assess diagnostic performance of metagenomic next-generation sequencing (mNGS) for IPA and analyse lung microbiome via mNGS data.

[Source](https://doi.org/10.1136/bmjresp-2022-001358)

10.1136/bmjresp-2022-001358

---

## Comparing the gut microbiome of obese, African American, older adults with and without mild cognitive impairment.
 28.02.2023, _A McLeod, B Penalver Bernabe, Y Xia, J Sanchez-Flack, M Lamar, L Schiffer, K Castellanos, G Fantuzzi, P Maki, M Fitzgibbon, L Tussing-Humphreys_


_Aged, Female, Humans, Middle Aged, Black or African American, Case-Control Studies, Cognitive Dysfunction, Dementia, Gastrointestinal Microbiome, Inflammation, Obesity, RNA, Ribosomal, 16S, Male_
Those with mild cognitive impairment (MCI), a precursor to dementia, have a gut microbiome distinct from healthy individuals, but this has only been shown in healthy individuals, not in those exhibiting several risk factors for dementia. Using amplicon 16S rRNA gene sequencing in a case-control study of 60 older (ages 55-76), obese, predominately female, African American adults, those with MCI (cases) had different gut microbiota profiles than controls. While microbial community diversity was similar between cases and controls, the abundances of specific microbial taxa weren't, such as Parabacteroides distasonis (lower in cases) and Dialister invisus (higher in cases). These differences disappeared after adjusting for markers of oxidative stress and systemic inflammation. Cognitive scores were positively correlated with levels of Akkermansia muciniphila, a bacterium associated with reduced inflammation. Our study shows that gut microbial composition may be associated with inflammation, oxidative stress, and MCI in those at high risk for dementia.

[Source](https://doi.org/10.1371/journal.pone.0280211)

10.1371/journal.pone.0280211

---

## Capacity of a Microbial Synbiotic To Rescue the In Vitro Metabolic Activity of the Gut Microbiome following Perturbation with Alcohol or Antibiotics

[Source](https://doi.org/10.1128/aem.01880-22)

---

## A microbiota and dietary metabolite integrates DNA repair and cell death to regulate embryo viability and aneuploidy during aging

[Source](https://doi.org/10.1126/sciadv.ade8653)

---

## A longitudinal survey in the wild reveals major shifts in fish host microbiota after parasite infection

[Source](https://doi.org/10.1111/mec.16901)

---

## Matrine alleviates depressive-like behaviors via modulating microbiota-gut-brain axis in CUMS-induced mice

[Source](https://doi.org/10.1186/s12967-023-03993-z)

---

## Variations in gut microbiome and metabolites of dogs with acute diarrhea in poodles and Labrador retrievers

[Source](https://doi.org/10.1007/s00203-023-03439-6)

---

## Enzyme adaptation to habitat thermal legacy shapes the thermal plasticity of marine microbiomes

[Source](https://doi.org/10.1038/s41467-023-36610-0)

---

## Computational methods and challenges in analyzing intratumoral microbiome data

[Source](https://doi.org/10.1016/j.tim.2023.01.011)

---

## Robust Bioinformatics Approaches Result in the First Polygenic Risk Score for BMI in Greek Adults.
 28.02.2023, _M Kafyra, IP Kalafati, M Dimitriou, E Grigoriou, A Kokkinos, L Rallidis, G Kolovou, G Trovas, E Marouli, P Deloukas, P Moulos, GV Dedoussis_


Quantifying the role of genetics via construction of polygenic risk scores (PRSs) is deemed a resourceful tool to enable and promote effective obesity prevention strategies. The present paper proposes a novel methodology for PRS extraction and presents the first PRS for body mass index (BMI) in a Greek population. A novel pipeline for PRS derivation was used to analyze genetic data from a unified database of three cohorts of Greek adults. The pipeline spans various steps of the process, from iterative dataset splitting to training and test partitions, calculation of summary statistics and PRS extraction, up to PRS aggregation and stabilization, achieving higher evaluation metrics. Using data from 2185 participants, implementation of the pipeline enabled consecutive repetitions in splitting training and testing samples and resulted in a 343-single nucleotide polymorphism PRS yielding an R2 = 0.3241 (beta = 1.011, p-value = 4 × 10-193) for BMI. PRS-included variants displayed a variety of associations with known traits (i.e., blood cell count, gut microbiome, lifestyle parameters). The proposed methodology led to creation of the first-ever PRS for BMI in Greek adults and aims at promoting a facilitating approach to reliable PRS development and integration in healthcare practice.

[Source](https://doi.org/10.3390/jpm13020327)

10.3390/jpm13020327

---

## Development of a tongue image-based machine learning tool for the diagnosis of gastric cancer: a prospective multicentre clinical cohort study.
 27.02.2023, _L Yuan, L Yang, S Zhang, Z Xu, J Qin, Y Shi, P Yu, Y Wang, Z Bao, Y Xia, J Sun, W He, T Chen, X Chen, C Hu, Y Zhang, C Dong, P Zhao, Y Wang, N Jiang, B Lv, Y Xue, B Jiao, H Gao, K Chai, J Li, H Wang, X Wang, X Guan, X Liu, G Zhao, Z Zheng, J Yan, H Yu, L Chen, Z Ye, H You, Y Bao, X Cheng, P Zhao, L Wang, W Zeng, Y Tian, M Chen, Y You, G Yuan, H Ruan, X Gao, J Xu, H Xu, L Du, S Zhang, H Fu, X Cheng_


Tongue images (the colour, size and shape of the tongue and the colour, thickness and moisture content of the tongue coating), reflecting the health state of the whole body according to the theory of traditional Chinese medicine (TCM), have been widely used in China for thousands of years. Herein, we investigated the value of tongue images and the tongue coating microbiome in the diagnosis of gastric cancer (GC).

[Source](https://doi.org/10.1016/j.eclinm.2023.101834)

10.1016/j.eclinm.2023.101834

---

## Latent dirichlet allocation for double clustering (LDA-DC): discovering patients phenotypes and cell populations within a single Bayesian framework.
 27.02.2023, _EE Hachem, N Sokolovska, H Soula_


_Humans, Bayes Theorem, Cluster Analysis_
Current clinical routines rely more and more on "omics" data such as flow cytometry data from host and microbiota. Cohorts variability in addition to patients' heterogeneity and huge dimensions make it difficult to understand underlying structure of the data and decipher pathologies. Patients stratification and diagnostics from such complex data are extremely challenging. There is an acute need to develop novel statistical machine learning methods that are robust with respect to the data heterogeneity, efficient from the computational viewpoint, and can be understood by human experts.

[Source](https://doi.org/10.1186/s12859-023-05177-4)

10.1186/s12859-023-05177-4

---

