## A global metagenomic map of urban microbiomes and antimicrobial resistance

[Source](https://doi.org/10.1016/j.cell.2021.05.002)

---

## Ecology and molecular targets of hypermutation in the global microbiome.
 25.05.2021, _S Roux, BG Paul, SC Bagby, S Nayfach, MA Allen, G Attwood, R Cavicchioli, L Chistoserdova, RJ Gruninger, SJ Hallam, ME Hernandez, M Hess, WT Liu, TA McAllister, MA O'Malley, X Peng, VI Rich, SR Saleska, EA Eloe-Fadrosh_


Changes in the sequence of an organism's genome, i.e., mutations, are the raw material of evolution. The frequency and location of mutations can be constrained by specific molecular mechanisms, such as diversity-generating retroelements (DGRs). DGRs have been characterized from cultivated bacteria and bacteriophages, and perform error-prone reverse transcription leading to mutations being introduced in specific target genes. DGR loci were also identified in several metagenomes, but the ecological roles and evolutionary drivers of these DGRs remain poorly understood. Here, we analyze a dataset of &gt;30,000 DGRs from public metagenomes, establish six major lineages of DGRs including three primarily encoded by phages and seemingly used to diversify host attachment proteins, and demonstrate that DGRs are broadly active and responsible for &gt;10% of all amino acid changes in some organisms. Overall, these results highlight the constraints under which DGRs evolve, and elucidate several distinct roles these elements play in natural communities.

[Source](https://doi.org/10.1038/s41467-021-23402-7)

10.1038/s41467-021-23402-7

---

## A cocoa (<em>Theobroma cacao L.</em>) extract impairs the growth, virulence properties, and inflammatory potential of <em>Fusobacterium nucleatum</em> and improves oral epithelial barrier function

[Source](https://doi.org/10.1371/journal.pone.0252029)

---

## Composition of cutaneous bacterial microbiome in seborrheic dermatitis patients: A cross-sectional study.
 24.05.2021, _MGH Sanders, T Nijsten, J Verlouw, R Kraaij, LM Pardo_


Seborrheic dermatitis (SD) is a chronic inflammatory skin disease with a multifactorial aetiology. Malassezia yeasts have been associated with the disease but the role of bacterial composition in SD has not been thoroughly investigated.

[Source](https://doi.org/10.1371/journal.pone.0251136)

10.1371/journal.pone.0251136

---

## Gene-level metagenomic architectures across diseases yield high-resolution microbiome diagnostic indicators.
 26.05.2021, _BT Tierney, Y Tan, AD Kostic, CJ Patel_


We propose microbiome disease "architectures": linking &gt;1 million microbial features (species, pathways, and genes) to 7 host phenotypes from 13 cohorts using a pipeline designed to identify associations that are robust to analytical model choice. Here, we quantify conservation and heterogeneity in microbiome-disease associations, using gene-level analysis to identify strain-specific, cross-disease, positive and negative associations. We find coronary artery disease, inflammatory bowel diseases, and liver cirrhosis to share gene-level signatures ascribed to the Streptococcus genus. Type 2 diabetes, by comparison, has a distinct metagenomic signature not linked to any one specific species or genus. We additionally find that at the species-level, the prior-reported connection between Solobacterium moorei and colorectal cancer is not consistently identified across models-however, our gene-level analysis unveils a group of robust, strain-specific gene associations. Finally, we validate our findings regarding colorectal cancer and inflammatory bowel diseases in independent cohorts and identify that features inversely associated with disease tend to be less reproducible than features enriched in disease. Overall, our work is not only a step towards gene-based, cross-disease microbiome diagnostic indicators, but it also illuminates the nuances of the genetic architecture of the human microbiome, including tension between gene- and species-level associations.

[Source](https://doi.org/10.1038/s41467-021-23029-8)

10.1038/s41467-021-23029-8

---

## Altered faecal microbiome and metabolome in IgG4-related sclerosing cholangitis and primary sclerosing cholangitis.
 26.05.2021, _Q Liu, B Li, Y Li, Y Wei, B Huang, J Liang, Z You, Y Li, Q Qian, R Wang, J Zhang, R Chen, Z Lyu, Y Chen, M Shi, X Xiao, Q Wang, Q Miao, JY Fang, ME Gershwin, M Lian, X Ma, R Tang_


Multiple clinical similarities exist between IgG4-related sclerosing cholangitis (IgG4-SC) and primary sclerosing cholangitis (PSC), and while gut dysbiosis has been extensively studied in PSC, the role of the gut microbiota in IgG4-SC remains unknown. Herein, we aimed to evaluate alterations of the gut microbiome and metabolome in IgG4-SC and PSC.

[Source](https://doi.org/10.1136/gutjnl-2020-323565)

10.1136/gutjnl-2020-323565

---

## Gut microbiome variation modulates the effects of dietary fiber on host metabolism.
 24.05.2021, _SM Murga-Garrido, Q Hong, TL Cross, ER Hutchison, J Han, SP Thomas, EI Vivas, J Denu, DG Ceschin, ZZ Tang, FE Rey_


_Animals, Diet, Dietary Fiber, Gastrointestinal Microbiome, Germ-Free Life, Inulin, Mice_
There is general consensus that consumption of dietary fermentable fiber improves cardiometabolic health, in part by promoting mutualistic microbes and by increasing production of beneficial metabolites in the distal gut. However, human studies have reported variations in the observed benefits among individuals consuming the same fiber. Several factors likely contribute to this variation, including host genetic and gut microbial differences. We hypothesized that gut microbial metabolism of dietary fiber represents an important and differential factor that modulates how dietary fiber impacts the host.

[Source](https://doi.org/10.1186/s40168-021-01061-6)

10.1186/s40168-021-01061-6

---

## Real-time monitoring of ruminal microbiota reveals their roles in dairy goats during subacute ruminal acidosis.
 18.05.2021, _X Chen, X Su, J Li, Y Yang, P Wang, F Yan, J Yao, S Wu_


Ruminal microbiota changes frequently with high grain diets and the occurrence of subacute ruminal acidosis (SARA). A grain-induced goat model of SARA, with durations of a significant decrease in the rumen pH value to less than 5.6 and an increase in the rumen lipopolysaccharides concentration, is constructed for real-time monitoring of bacteria alteration. Using 16 S rRNA gene sequencing, significant bacterial differences between goats from the SARA and healthy groups are identified at every hour for six continuous hours after feeding. Moreover, 29 common differential genera between two groups over 6 h after feeding are all related to the altered pH and lipopolysaccharides. Transplanting the microbiota from donor goats with SARA could induce colonic inflammation in antibiotic-pretreated mice. Overall, significant differences in the bacterial community and rumen fermentation pattern between the healthy and SARA dairy goats are real-time monitored, and then tested using ruminal microbe transplantation to antibiotic-treated mice.

[Source](https://doi.org/10.1038/s41522-021-00215-6)

10.1038/s41522-021-00215-6

---

## The microbiome and mosquito vectorial capacity: rich potential for discovery and translation.
 24.05.2021, _C Cansado-Utrilla, SY Zhao, PJ McCall, KL Coon, GL Hughes_


_Animals, Culicidae, Gastrointestinal Microbiome, Humans, Microbial Interactions, Microbiota, Mosquito Vectors_
Microbiome research has gained considerable interest due to the emerging evidence of its impact on human and animal health. As in other animals, the gut-associated microbiota of mosquitoes affect host fitness and other phenotypes. It is now well established that microbes can alter pathogen transmission in mosquitoes, either positively or negatively, and avenues are being explored to exploit microbes for vector control. However, less attention has been paid to how microbiota affect phenotypes that impact vectorial capacity. Several mosquito and pathogen components, such as vector density, biting rate, survival, vector competence, and the pathogen extrinsic incubation period all influence pathogen transmission. Recent studies also indicate that mosquito gut-associated microbes can impact each of these components, and therefore ultimately modulate vectorial capacity. Promisingly, this expands the options available to exploit microbes for vector control by also targeting parameters that affect vectorial capacity. However, there are still many knowledge gaps regarding mosquito-microbe interactions that need to be addressed in order to exploit them efficiently. Here, we review current evidence of impacts of the microbiome on aspects of vectorial capacity, and we highlight likely opportunities for novel vector control strategies and areas where further studies are required. Video abstract.

[Source](https://doi.org/10.1186/s40168-021-01073-2)

10.1186/s40168-021-01073-2

---

## A metabolomics pipeline enables mechanistic interrogation of the gut microbiome

[Source](https://doi.org/10.1101/2021.05.25.445684)

---

## Comparison study of differential abundance testing methods using two large Parkinson disease gut microbiome datasets derived from 16S amplicon sequencing.
 27.05.2021, _ZD Wallen_


_Gastrointestinal Microbiome, Humans, Microbiota, Parkinson Disease, RNA, Ribosomal, 16S_
Testing for differential abundance of microbes in disease is a common practice in microbiome studies. Numerous differential abundance (DA) testing methods exist and range from traditional statistical tests to methods designed for microbiome data. Comparison studies of DA testing methods have been performed, but none performed on microbiome datasets collected for the study of real, complex disease. Due to this, DA testing was performed here using various DA methods in two large, uniformly collected gut microbiome datasets on Parkinson disease (PD), and their results compared.

[Source](https://doi.org/10.1186/s12859-021-04193-6)

10.1186/s12859-021-04193-6

---

## Reads Binning Improves the Assembly of Viral Genome Sequences From Metagenomic Samples

[Source](https://doi.org/10.3389/fmicb.2021.664560)

---

## Standard Sample Storage Conditions Impact on Inferred Microbiome Composition and Antimicrobial Resistance Patterns

[Source](https://doi.org/10.1101/2021.05.24.445395)

---

