## Gut microbiome lipid metabolism and its impact on host physiology.
 18.02.2023, _EM Brown, J Clardy, RJ Xavier_


_Animals, Gastrointestinal Microbiome, Lipid Metabolism, Immune System, Lipids, Mammals_
Metabolites produced by commensal gut microbes impact host health through their recognition by the immune system and their influence on numerous metabolic pathways. Notably, the gut microbiota can both transform and synthesize lipids as well as break down dietary lipids to generate secondary metabolites with host modulatory properties. Although lipids have largely been consigned to structural roles, particularly in cell membranes, recent research has led to an increased appreciation of their signaling activities, with potential impacts on host health and physiology. This review focuses on studies that highlight the functions of bioactive lipids in mammalian physiology, with a special emphasis on immunity and metabolism.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(23)00034-3)

10.1016/j.chom.2023.01.009

---

## Modelling human immune responses using microbial exposures in rodents.
 16.02.2023, _Y Li, MT Baldridge_



[Source](https://www.nature.com/articles/s41564-023-01334-w)

10.1038/s41564-023-01334-w

---

## Gut commensal&nbsp;<em>Parabacteroides distasonis</em>&nbsp;alleviates inflammatory arthritis

[Source](https://gut.bmj.com/content/early/2023/02/16/gutjnl-2022-327756)

---

## An open label, non-randomized study assessing a prebiotic fiber intervention in a small cohort of Parkinson’s disease participants

[Source](https://www.nature.com/articles/s41467-023-36497-x)

---

## Antibiotic use and inflammatory bowel disease: number needed to harm?
 22.02.2023, _JF Ludvigsson_



[Source](https://gut.bmj.com/content/early/2023/02/20/gutjnl-2023-329575)

10.1136/gutjnl-2023-329575

---

## Oral mucosal breaks trigger anti-citrullinated bacterial and human protein antibody responses in rheumatoid arthritis.
 24.02.2023, _RC Brewer, TV Lanz, CR Hale, GD Sepich-Poore, C Martino, AD Swafford, TS Carroll, S Kongpachith, LK Blum, SE Elliott, NE Blachere, S Parveen, J Fak, V Yao, O Troyanskaya, MO Frank, MS Bloom, S Jahanbani, AM Gomez, R Iyer, NS Ramadoss, O Sharpe, S Chandrasekaran, LB Kelmenson, Q Wang, H Wong, HL Torres, M Wiesen, DT Graves, KD Deane, VM Holers, R Knight, RB Darnell, WH Robinson, DE Orange_


_Humans, Autoantibodies, Mouth Mucosa, Antibody Formation, Arthritis, Rheumatoid, Epitopes, Periodontal Diseases, Bacteria_
Periodontal disease is more common in individuals with rheumatoid arthritis (RA) who have detectable anti-citrullinated protein antibodies (ACPAs), implicating oral mucosal inflammation in RA pathogenesis. Here, we performed paired analysis of human and bacterial transcriptomics in longitudinal blood samples from RA patients. We found that patients with RA and periodontal disease experienced repeated oral bacteremias associated with transcriptional signatures of ISG15+HLADRhi and CD48highS100A2pos monocytes, recently identified in inflamed RA synovia and blood of those with RA flares. The oral bacteria observed transiently in blood were broadly citrullinated in the mouth, and their in situ citrullinated epitopes were targeted by extensively somatically hypermutated ACPAs encoded by RA blood plasmablasts. Together, these results suggest that (i) periodontal disease results in repeated breaches of the oral mucosa that release citrullinated oral bacteria into circulation, which (ii) activate inflammatory monocyte subsets that are observed in inflamed RA synovia and blood of RA patients with flares and (iii) activate ACPA B cells, thereby promoting affinity maturation and epitope spreading to citrullinated human antigens.

[Source](https://www.science.org/doi/10.1126/scitranslmed.abq8476)

10.1126/scitranslmed.abq8476

---

## Sputum bacterial load and bacterial composition correlate with lung function and are altered by long-term azithromycin treatment in children with HIV-associated chronic lung disease

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01460-x)

---

## Genetic mapping of microbial and host traits reveals production of immunomodulatory lipids by Akkermansia muciniphila in the murine gut.
 14.02.2023, _Q Zhang, V Linke, KA Overmyer, LL Traeger, K Kasahara, IJ Miller, DE Manson, TJ Polaske, RL Kerby, JH Kemis, EA Trujillo, TR Reddy, JD Russell, KL Schueler, DS Stapleton, ME Rabaglia, M Seldin, DM Gatti, GR Keele, DT Pham, JP Gerdt, EI Vivas, AJ Lusis, MP Keller, GA Churchill, HE Blackwell, KW Broman, AD Attie, JJ Coon, FE Rey_


The molecular bases of how host genetic variation impacts the gut microbiome remain largely unknown. Here we used a genetically diverse mouse population and applied systems genetics strategies to identify interactions between host and microbe phenotypes including microbial functions, using faecal metagenomics, small intestinal transcripts and caecal lipids that influence microbe-host dynamics. Quantitative trait locus (QTL) mapping identified murine genomic regions associated with variations in bacterial taxa; bacterial functions including motility, sporulation and lipopolysaccharide production and levels of bacterial- and host-derived lipids. We found overlapping QTL for the abundance of Akkermansia muciniphila and caecal levels of ornithine lipids. Follow-up in vitro and in vivo studies revealed that A. muciniphila is a major source of these lipids in the gut, provided evidence that ornithine lipids have immunomodulatory effects and identified intestinal transcripts co-regulated with these traits including Atf3, which encodes for a transcription factor that plays vital roles in modulating metabolism and immunity. Collectively, these results suggest that ornithine lipids are potentially important for A. muciniphila-host interactions and support the role of host genetics as a determinant of responses to gut microbes.

[Source](https://www.nature.com/articles/s41564-023-01326-w)

10.1038/s41564-023-01326-w

---

## Microbe-mediated intestinal NOD2 stimulation improves linear growth of undernourished infant mice.
 23.02.2023, _M Schwarzer, UK Gautam, K Makki, A Lambert, T Brabec, A Joly, D Šrůtková, P Poinsot, T Novotná, S Geoffroy, P Courtin, PP Hermanová, RC Matos, JJM Landry, C Gérard, AL Bulteau, T Hudcovic, H Kozáková, D Filipp, MP Chapot-Chartier, M Šinkora, N Peretti, IG Boneca, M Chamaillard, H Vidal, F De Vadder, F Leulier_


The intestinal microbiota is known to influence postnatal growth. We previously found that a strain of Lactiplantibacillus plantarum (strain LpWJL) buffers the adverse effects of chronic undernutrition on the growth of juvenile germ-free mice. Here, we report that LpWJL sustains the postnatal growth of malnourished conventional animals and supports both insulin-like growth factor-1 (IGF-1) and insulin production and activity. We have identified cell walls isolated from LpWJL, as well as muramyl dipeptide and mifamurtide, as sufficient cues to stimulate animal growth despite undernutrition. Further, we found that NOD2 is necessary in intestinal epithelial cells for LpWJL-mediated IGF-1 production and for postnatal growth promotion in malnourished conventional animals. These findings indicate that, coupled with renutrition, bacteria cell walls or purified NOD2 ligands have the potential to alleviate stunting.

[Source](https://www.science.org/doi/10.1126/science.ade9767)

10.1126/science.ade9767

---

## Gamma-delta T cells modulate the microbiota and fecal micro-RNAs to maintain mucosal tolerance.
 24.02.2023, _RM Rezende, LM Cox, TG Moreira, S Liu, S Boulenouar, F Dhang, DS LeServe, BN Nakagaki, JR Lopes, BK Tatematsu, L Lemos, J Mayrink, ELC Lobo, L Guo, MG Oliveira, C Kuhn, HL Weiner_


_Mice, Animals, T-Lymphocytes, MicroRNAs, Receptors, Antigen, T-Cell, gamma-delta, Intestines, Intestinal Mucosa, Microbiota, Immunity, Mucosal_
Gamma-delta (γδ) T cells are a major cell population in the intestinal mucosa and are key mediators of mucosal tolerance and microbiota composition. Little is known about the mechanisms by which intestinal γδ T cells interact with the gut microbiota to maintain tolerance.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01478-1)

10.1186/s40168-023-01478-1

---

## Bacteriophage discovery to advance biotechnology and biotherapeutics.
 22.02.2023, _A Mu, TD Lawley_



[Source](https://www.nature.com/articles/s41579-023-00870-w)

10.1038/s41579-023-00870-w

---

## Impact of microbial genome completeness on metagenomic functional inference.
 19.02.2023, _R Eisenhofer, I Odriozola, A Alberdi_


Inferring the functional capabilities of bacteria from metagenome-assembled genomes (MAGs) is becoming a central process in microbiology. Here we show that the completeness of genomes has a significant impact on the recovered functional signal, spanning all domains of metabolic functions. We identify factors that affect this relationship between genome completeness and function fullness, and provide baseline knowledge to guide efforts to correct for this overlooked bias in metagenomic functional inference.

[Source](https://www.nature.com/articles/s43705-023-00221-z)

10.1038/s43705-023-00221-z

---

## Pharokka: a fast scalable bacteriophage annotation tool.
 22.02.2023, _G Bouras, R Nepal, G Houtak, AJ Psaltis, PJ Wormald, S Vreugde_


_Bacteriophages, Genomics, Genome, Computational Biology, Software_
In recent years, there has been an increasing interest in bacteriophages, which has led to growing numbers of bacteriophage genomic sequences becoming available. Consequently, there is a need for a rapid and consistent genomic annotation tool dedicated for bacteriophages. Existing tools either are not designed specifically for bacteriophages or are web- and email-based and require significant manual curation, which makes their integration into bioinformatic pipelines challenging. Pharokka was created to provide a tool that annotates bacteriophage genomes easily, rapidly and consistently with standards compliant outputs. Moreover, Pharokka requires only two lines of code to install and use and takes under 5 min to run for an average 50-kb bacteriophage genome.

[Source](https://academic.oup.com/bioinformatics/article/39/1/btac776/6858464)

10.1093/bioinformatics/btac776

---

## A systems immunology study comparing innate and adaptive immune responses in adults to COVID-19 mRNA and adenovirus vectored vaccines

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00076-9)

---

## Influenza-trained mucosal-resident alveolar macrophages confer long-term antitumor immunity in the lungs.
 22.02.2023, _T Wang, J Zhang, Y Wang, Y Li, L Wang, Y Yu, Y Yao_


Respiratory viral infections reprogram pulmonary macrophages with altered anti-infectious functions. However, the potential function of virus-trained macrophages in antitumor immunity in the lung, a preferential target of both primary and metastatic malignancies, is not well understood. Using mouse models of influenza and lung metastatic tumors, we show here that influenza trains respiratory mucosal-resident alveolar macrophages (AMs) to exert long-lasting and tissue-specific antitumor immunity. Trained AMs infiltrate tumor lesions and have enhanced phagocytic and tumor cell cytotoxic functions, which are associated with epigenetic, transcriptional and metabolic resistance to tumor-induced immune suppression. Generation of antitumor trained immunity in AMs is dependent on interferon-γ and natural killer cells. Notably, human AMs with trained immunity traits in non-small cell lung cancer tissue are associated with a favorable immune microenvironment. These data reveal a function for trained resident macrophages in pulmonary mucosal antitumor immune surveillance. Induction of trained immunity in tissue-resident macrophages might thereby be a potential antitumor strategy.

[Source](https://www.nature.com/articles/s41590-023-01428-x)

10.1038/s41590-023-01428-x

---

