## Commensal microbiota divergently affect myeloid subsets in the mammalian central nervous system during homeostasis and disease.
 08.10.2021, _R Sankowski, J Ahmari, C Mezö, AL Hrabě de Angelis, V Fuchs, O Utermöhlen, T Buch, T Blank, M Gomez de Agüero, AJ Macpherson, D Erny_


The immune cells of the central nervous system (CNS) comprise parenchymal microglia and at the CNS border regions meningeal, perivascular, and choroid plexus macrophages (collectively called CNS-associated macrophages, CAMs). While previous work has shown that microglial properties depend on environmental signals from the commensal microbiota, the effects of microbiota on CAMs are unknown. By combining several microbiota manipulation approaches, genetic mouse models, and single-cell RNA-sequencing, we have characterized CNS myeloid cell composition and function. Under steady-state conditions, the transcriptional profiles and numbers of choroid plexus macrophages were found to be tightly regulated by complex microbiota. In contrast, perivascular and meningeal macrophages were affected to a lesser extent. An acute perturbation through viral infection evoked an attenuated immune response of all CAMs in germ-free mice. We further assessed CAMs in a more chronic pathological state in 5xFAD mice, a model for Alzheimer's disease, and found enhanced amyloid beta uptake exclusively by perivascular macrophages in germ-free 5xFAD mice. Our results aid the understanding of distinct microbiota-CNS macrophage interactions during homeostasis and disease, which could potentially be targeted therapeutically.

[Source](https://www.embopress.org/doi/full/10.15252/embj.2021108605)

10.15252/embj.2021108605

---

## Coordination of root auxin with the fungus <em>Piriformospora indica </em>and bacterium <em>Bacillus cereus</em> enhances rice rhizosheath formation under soil drying

[Source](https://doi.org/10.1038/s41396-021-01133-3)

---

## The role of <em>Akkermansia muciniphila</em> in obesity, diabetes and atherosclerosis

[Source](https://doi.org/10.1099/jmm.0.001435)

---

## How can probiotic improve irritable bowel syndrome symptoms?
 09.10.2021, _I Benjak Horvat, I Gobin, A Kresović, G Hauser_


The onset and manifestations of irritable bowel syndrome (IBS) is associated with several factors, and the pathophysiology involves various central and peripheral mechanisms. Most studies indicate that the management of gut microbiota could significantly affect the improvement of subjective disorders in patients with IBS. Numerous clinical trials have assessed the efficacy of probiotics for IBS with controversial conclusions. Several clinical trials have suggested that probiotics can improve global IBS symptoms, while others only improve individual IBS symptoms, such as bloating scores and abdominal pain scores. Only a few clinical trials have found no apparent effect of probiotics on IBS symptoms. Generally, probiotics appear to be safe for patients with IBS. However, the question of which probiotics should be used for certain IBS subtypes remains unresolved. In everyday practice, the dose of the recommended probiotic remains questionable, as well as how long the probiotic should be used in therapy. The use of probiotics in the M subtype and non-classified IBS is particularly problematic, in which combination therapy should be recommended due to the change in symptoms. Therefore, new approaches are needed in the design of clinical studies that should address certain subtypes of IBS.

[Source](https://doi.org/10.4240/wjgs.v13.i9.923)

10.4240/wjgs.v13.i9.923

---

## Meta-Analysis of the Composition of Human Intestinal Gases.
 08.10.2021, _A Modesto, NR Cameron, C Varghese, N Peters, B Stokes, A Phillips, I Bissett, G O'Grady_


Understanding intestinal gases volume and composition may contribute to diagnosing digestive diseases and the microbiome's status. This meta-analysis aimed to define the composition of human intestinal gases and changes associated with diet.

[Source](https://link.springer.com/article/10.1007/s10620-021-07254-1)

10.1007/s10620-021-07254-1

---

## Effectiveness of Probiotics and Prebiotics Against Acute Liver Injury: A Meta-Analysis.
 09.10.2021, _S Xu, M Zhao, Q Wang, Z Xu, B Pan, Y Xue, Z Dai, S Wang, Z Xue, F Wang, C Xu_


Background and Aims: Acute liver injury (ALI) is a clinical syndrome characterized by rapid loss of liver function, which may progress to life-threatening liver failure. We conducted this meta-analysis to examine the evidence on the effects of probiotics or prebiotics on ALI. Methods and Results: Several databases, including PubMed, EMBASE, and Cochrane Library, were scrutinized from the inception through February 2021 by combining key search terms, yielding 26 eligible studies, which concluded that modulation of gut microbiota significantly decreased aspartate transaminase [standardized mean difference (SMD): -1.51, 95% confidence interval (CI): -2.03 to -1.00], alanine aminotransferase (SMD: -1.42, 95% CI: -1.85 to -0.98), and bilirubin (SMD: -0.91, 95% CI: -1.33 to -0.49). In addition, administration of probiotics or prebiotics also promoted proliferation of Bifidobacterium (SMD: 1.21, 95% CI: -0.18 to 2.60) and inhibited Enterococcus (SMD: -1.00, 95% CI: -1.39 to -0.61), contributing to lower levels of endotoxin (SMD: -2.14, 95% CI: -2.91 to -1.37). Tight junction protein ZO-1 (SMD: 1.95, 95% CI: 0.14 to 3.76) was upregulated after intervention, thereby reducing bacterial translocation to the liver [odds ratio (OR) = 0.23, 95% CI: 0.13-0.44] and mesenteric lymph node (OR = 0.14, 95% CI: 0.08 to 0.26), with decreased tumor necrosis factor-α (SMD: -2.84, 95% CI: -3.76 to -1.93) and interleukin-6 (SMD: -2.62, 95% CI: -4.14 to -1.10). Oxidative stress was also relieved by reducing malondialdehyde (SMD: -1.83, 95% CI: -2.55 to -1.10) while elevating superoxide dismutase (SMD: 1.78, 95% CI: 1.00-2.55) and glutathione (SMD: 1.83, 95% CI: 0.76-2.91). Conclusion: Our findings suggest that probiotics and prebiotics could be a promising therapeutic strategy in ALI and possess a potential for clinical applications. Systematic Review Registration:https://www.crd.york.ac.uk/PROSPERO/display_record.php?RecordID=255888, CRD42021255888.

[Source](https://doi.org/10.3389/fmed.2021.739337)

10.3389/fmed.2021.739337

---

## The bacterial gut microbiome of probiotic-treated very-preterm infants: changes from admission to discharge.
 08.10.2021, _JAF Westaway, R Huerlimann, Y Kandasamy, CM Miller, R Norton, KM Staunton, D Watson, D Rudd_


Preterm birth is associated with the development of acute and chronic disease, potentially, through the disruption of normal gut microbiome development. Probiotics may correct for microbial imbalances and mitigate disease risk. Here, we used amplicon sequencing to characterise the gut microbiome of probiotic-treated premature infants. We aimed to identify and understand variation in bacterial gut flora from admission to discharge and in association with clinical variables.

[Source](https://doi.org/10.1038/s41390-021-01738-6)

10.1038/s41390-021-01738-6

---

## Discovery of β-N-acetylglucosaminidases from screening metagenomic libraries and their use as thioglycoligase mutants.
 08.10.2021, _G Tegl, P Rahfeld, K Ostmann, J Hanson, SG Withers_


β-N-Acetylhexosaminidases (HexNAcases) are versatile biocatalysts that cleave terminal N-acetylhexosamine units from various glycoconjugates. Established strategies to generate glycoside-forming versions of the wild type enzymes rely on the mutation of their catalytic residues; however, successful examples of synthetically useful HexNAcase mutants are scarce. In order to expand the range of HexNAcases available as targets for enzyme engineering, we functionally screened a metagenomic library derived from a human gut microbiome. From a pool of hits, we characterized four of the more active candidates by sequence analysis and phylogenetic mapping, and found that they all belonged to CAZy family GH20. After detailed kinetic analysis and characterization of their substrate specificities, active site mutants were generated which resulted in the identification of two new thioglycoligases. BvHex E294A and AsHex E301A catalyzed glycosyl transfer to all three of the 3-, 4- and 6-thio-N-acetylglucosaminides (thio-GlcNAcs) that were tested. Both mutant enzymes also catalyzed glycosyl transfer to a cysteine-containing variant of the model peptide Tab1, with AsHex E301A also transferring GlcNAc onto a thiol-containing protein. This work illustrates how large scale functional screening of expressed gene libraries allows the relatively rapid development of useful new glycoside-forming mutants of HexNAcases, expanding the pool of biocatalysts for carbohydrate synthesis.

[Source](https://doi.org/10.1039/d1ob01246k)

10.1039/d1ob01246k

---

## Comparing the gut microbiome along the gastrointestinal tract of three sympatric species of wild rodents.
 08.10.2021, _JL Anders, MAM Moustafa, WMA Mohamed, T Hayakawa, R Nakao, I Koizumi_


Host-microbe interactions within the gastrointestinal tract (GIT) play a pivotal role in shaping host physiology, ecology, and life history. However, these interactions vary across gut regions due to changes in the physical environment or host immune system activity, thereby altering the microbial community. Each animal species may harbor their own unique microbial community due to host species-specific ecological traits such as dietary habits, micro-habitat preferences, and mating behavior as well as physiological traits. While the gut microbiota in wild animals has received much attention over the last decade, most studies comparing closely related species only utilized fecal or colon samples. In this study, we first compared the gut microbial community from the small intestine, cecum, colon, and rectum within three sympatric species of wild rodents (i.e. Apodemus speciosus, A. argenteus, and Myodes rufocanus). We then compared each gut region among host species to determine the effect of both gut region and host species on the gut microbiota. We found that the small intestine harbored a unique microbiome as compared to the lower GIT in all three host species, with the genus Lactobacillus in particular having higher abundance in the small intestine of all three host species. There were clear interspecific differences in the microbiome within all gut regions, although some similarity in alpha diversity and community structure within the small intestine was found. Finally, fecal samples may be appropriate for studying the lower GIT in these species, but not the small intestine.

[Source](https://doi.org/10.1038/s41598-021-99379-6)

10.1038/s41598-021-99379-6

---

## Changes in rumen microbial community composition in yak in response to seasonal variations.
 08.10.2021, _X Huang, J Mi, SE Denman, B Wangdui, P Zhandui, Q Zhang, R Long, CS McSweeney_


Yak is a dominant ruminant, well-adapted to grazing on pasture year around in the harsh climate of the 3,000-meter-high Qinghai-Tibetan Plateau. The complex microbial community that resides within the yak rumen is responsible for fermentation and contributes to its climatic adaptation. This study aimed to characterize the rumen microbiota responses to wide seasonal variations, especially those necessary for survival in the cold seasons.

[Source](https://doi.org/10.1111/jam.15322)

10.1111/jam.15322

---

## <em>Faecalicatena faecalis</em> sp. nov., a moderately alkaliphilic bacterial strain isolated from swine faeces

[Source](https://doi.org/10.1007/s10482-021-01663-y)

---

## Capacity of soil bacteria to reach the phyllosphere and convergence of floral communities despite soil microbiota variation.
 08.10.2021, _J Massoni, M Bortfeld-Miller, A Widmer, JA Vorholt_


Leaves and flowers are colonized by diverse bacteria that impact plant fitness and evolution. Although the structure of these microbial communities is becoming well-characterized, various aspects of their environmental origin and selection by plants remain uncertain, such as the relative proportion of soilborne bacteria in phyllosphere communities. Here, to address this issue and to provide experimental support for bacteria being filtered by flowers, we conducted common-garden experiments outside and under gnotobiotic conditions. We grew Arabidopsis thaliana in a soil substitute and added two microbial communities from natural soils. We estimated that at least 25% of the phyllosphere bacteria collected from the plants grown in the open environment were also detected in the controlled conditions, in which bacteria could reach leaves and flowers only from the soil. These taxa represented more than 40% of the communities based on amplicon sequencing. Unsupervised hierarchical clustering approaches supported the convergence of all floral microbiota, and 24 of the 28 bacteria responsible for this pattern belonged to the Burkholderiaceae family, which includes known plant pathogens and plant growth-promoting members. We anticipate that our study will foster future investigations regarding the routes used by soil microbes to reach leaves and flowers, the ubiquity of the environmental filtering of Burkholderiaceae across plant species and environments, and the potential functional effects of the accumulation of these bacteria in the reproductive organs of flowering plants.

[Source](https://doi.org/10.1073/pnas.2100150118)

10.1073/pnas.2100150118

---

## Distinct Endophytic Bacterial Communities Inhabiting Seagrass Seeds.
 09.10.2021, _F Tarquinio, O Attlan, MA Vanderklift, O Berry, A Bissett_


Seagrasses are marine angiosperms that can live completely or partially submerged in water and perform a variety of significant ecosystem services. Like terrestrial angiosperms, seagrasses can reproduce sexually and, the pollinated female flower develop into fruits and seeds, which represent a critical stage in the life of plants. Seed microbiomes include endophytic microorganisms that in terrestrial plants can affect seed germination and seedling health through phytohormone production, enhanced nutrient availability and defence against pathogens. However, the characteristics and origins of the seagrass seed microbiomes is unknown. Here, we examined the endophytic bacterial community of six microenvironments (flowers, fruits, and seeds, together with leaves, roots, and rhizospheric sediment) of the seagrass Halophila ovalis collected from the Swan Estuary, in southwestern Australia. An amplicon sequencing approach (16S rRNA) was used to characterize the diversity and composition of H. ovalis bacterial microbiomes and identify core microbiome bacteria that were conserved across microenvironments. Distinct communities of bacteria were observed within specific seagrass microenvironments, including the reproductive tissues (flowers, fruits, and seeds). In particular, bacteria previously associated with plant growth promoting characteristics were mainly found within reproductive tissues. Seagrass seed-borne bacteria that exhibit growth promoting traits, the ability to fix nitrogen and anti-pathogenic potential activity, may play a pivotal role in seed survival, as is common for terrestrial plants. We present the endophytic community of the seagrass seeds as foundation for the identification of potential beneficial bacteria and their selection in order to improve seagrass restoration.

[Source](https://doi.org/10.3389/fmicb.2021.703014)

10.3389/fmicb.2021.703014

---

## Widespread bacterial diversity within the bacteriome of fungi.
 08.10.2021, _AJ Robinson, GL House, DP Morales, JM Kelliher, V Gallegos-Graves, ES LeBrun, KW Davenport, F Palmieri, A Lohberger, D Bregnard, A Estoppey, M Buffi, C Paul, T Junier, V Hervé, G Cailleau, S Lupini, HN Nguyen, AO Zheng, LJ Gimenes, S Bindschedller, DF Rodrigues, JH Werner, JD Young, P Junier, PSG Chain_


Knowledge of associations between fungal hosts and their bacterial associates has steadily grown in recent years as the number and diversity of examinations have increased, but current knowledge is predominantly limited to a small number of fungal taxa and bacterial partners. Here, we screened for potential bacterial associates in over 700 phylogenetically diverse fungal isolates, representing 366 genera, or a tenfold increase compared with previously examined fungal genera, including isolates from several previously unexplored phyla. Both a 16 S rDNA-based exploration of fungal isolates from four distinct culture collections spanning North America, South America and Europe, and a bioinformatic screen for bacterial-specific sequences within fungal genome sequencing projects, revealed that a surprisingly diverse array of bacterial associates are frequently found in otherwise axenic fungal cultures. We demonstrate that bacterial associations with diverse fungal hosts appear to be the rule, rather than the exception, and deserve increased consideration in microbiome studies and in examinations of microbial interactions.

[Source](https://doi.org/10.1038/s42003-021-02693-y)

10.1038/s42003-021-02693-y

---

## Genomic and metabolic adaptations of biofilms to ecological windows of opportunities in glacier-fed streams

[Source](https://www.biorxiv.org/content/10.1101/2021.10.07.463499v1.abstract?%3Fcollection=)

---

