## Genome-scale metabolic modeling of <em>Aspergillus fumigatus</em> strains reveals growth dependencies on the lung microbiome

[Source](https://doi.org/10.1038/s41467-023-39982-5)

---

## Depletion of butyrate-producing microbes of the Firmicutes predicts nonresponse to FMT therapy in patients with recurrent <em>Clostridium difficile</em> infection

[Source](https://doi.org/10.1080/19490976.2023.2236362)

---

## Chagas disease is related to structural changes of the gut microbiota in adults with chronic infection (TRIPOBIOME Study)

[Source](https://doi.org/10.1371/journal.pntd.0011490)

---

## Study of the gut microbiome in Egyptian patients with Parkinson’s Disease

[Source](https://doi.org/10.1186/s12866-023-02933-7)

---

## Intestinal microbiota controls graft-versus-host disease independent of donor-host genetic disparity

[Source](https://www.cell.com/immunity/fulltext/S1074-7613(23)00281-9)

---

## A systematic review exploring the association between the human gut microbiota and brain connectivity in health and disease

[Source](https://doi.org/10.1038/s41380-023-02146-4)

---

## A macroecological perspective on genetic diversity in the human gut microbiome

[Source](https://doi.org/10.1371/journal.pone.0288926)

---

## Gut microbiota metabolites mediate the interplay between childhood maltreatment and psychopathology in patients with eating disorders

[Source](https://doi.org/10.1038/s41598-023-38665-x)

---

## In-depth metataxonomic investigation reveals low richness, high intervariability, and diverse phylotype candidates of archaea&nbsp;in the human urogenital tract

[Source](https://doi.org/10.1038/s41598-023-38710-9)

---

## Host microbiome-pathogen interactions in pediatric infections

[Source](https://doi.org/10.1097/QCO.0000000000000949)

---

## Intratumoural microbiome can predict the prognosis of hepatocellular carcinoma after surgery

[Source](https://doi.org/10.1002/ctm2.1331)

---

## A shotgun metagenomic analysis of the fecal microbiome in humans infected with <em>Giardia duodenalis</em>

[Source](https://doi.org/10.1186/s13071-023-05821-1)

---

## Microbiome-derived cobalamin and succinyl-CoA as biomarkers for improved screening of anal cancer

[Source](https://doi.org/10.1038/s41591-023-02407-3)

---

## Bacterial clade-specific analysis identifies distinct epithelial responses in inflammatory bowel disease.
 23.07.2023, _GL D'Adamo, M Chonwerawong, LJ Gearing, VR Marcelino, JA Gould, EL Rutten, SM Solari, PWR Khoo, TJ Wilson, T Thomason, EL Gulliver, PJ Hertzog, EM Giles, SC Forster_


_Humans, Inflammatory Bowel Diseases, Colon, Biopsy, Intestinal Mucosa, Epithelial Cells_
Abnormal immune responses to the resident gut microbiome can drive inflammatory bowel disease (IBD). Here, we combine high-resolution, culture-based shotgun metagenomic sequencing and analysis with matched host transcriptomics across three intestinal sites (terminal ileum, cecum, rectum) from pediatric IBD (PIBD) patients (n = 58) and matched controls (n = 42) to investigate this relationship. Combining our site-specific approach with bacterial culturing, we establish a cohort-specific bacterial culture collection, comprising 6,620 isolates (170 distinct species, 32 putative novel), cultured from 286 mucosal biopsies. Phylogeny-based, clade-specific metagenomic analysis identifies key, functionally distinct Enterococcus clades associated with either IBD or health. Strain-specific in vitro validation demonstrates differences in cell cytotoxicity and inflammatory signaling in intestinal epithelial cells, consistent with the colonic mucosa-specific response measured in patients with IBD. This demonstrates the importance of strain-specific phenotypes and consideration of anatomical sites in exploring the dysregulated host-bacterial interactions in IBD.

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00252-5)

10.1016/j.xcrm.2023.101124

---

## Multimodal immune cell phenotyping in GI biopsies reveals microbiome-related T cell modulations in human GvHD.
 23.07.2023, _S Jarosch, J Köhlen, S Ghimire, ET Orberg, M Hammel, D Gaag, M Evert, KP Janssen, A Hiergeist, A Gessner, D Weber, E Meedt, H Poeck, E D'Ippolito, E Holler, DH Busch_


_Humans, Graft vs Host Disease, Microbiota, Hematopoietic Stem Cell Transplantation, Gastrointestinal Tract, CD8-Positive T-Lymphocytes_
Acute graft-versus-host disease (aGvHD) is a significant complication after allogeneic hematopoietic stem cell transplantation (aHSCT), but major factors determining disease severity are not well defined yet. By combining multiplexed tissue imaging and single-cell RNA sequencing on gastrointestinal biopsies from aHSCT-treated individuals with fecal microbiome analysis, we link high microbiome diversity and the abundance of short-chain fatty acid-producing bacteria to the sustenance of suppressive regulatory T cells (Tregs). Furthermore, aGvHD severity strongly associates with the clonal expansion of mainly CD8 T cells, which we find distributed over anatomically distant regions of the gut, persistent over time, and inversely correlated with the presence of suppressive Tregs. Overall, our study highlights the pathophysiological importance of expanded CD8 T cell clones in the progression of aGvHD toward more severe clinical manifestations and strongly supports the further development of microbiome interventions as GvHD treatment via repopulation of the gut Treg niche to suppress inflammation.

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00253-7)

10.1016/j.xcrm.2023.101125

---

## Prebiotics and the risk of upper digestive tract and stomach cancers: the PrebiotiCa study.
 19.07.2023, _F Turati, F Concina, P Bertuccio, F Fiori, M Parpinel, W Garavello, A Crispo, M Libra, E Negri, D Serraino, C La Vecchia_


Fiber intake may lower digestive tract cancer risk, possibly by modulating the composition of gut microbiota. However, no data is available about the role of specific fiber fractions with prebiotic activity, e.g., inulin-type fructans (ITFs), fructo-oligosaccharides (FOSs) and galactooligosaccharides (GOSs), on the risk lower digestive tract cancer.

[Source](https://doi.org/10.1016/j.jand.2023.07.008)

10.1016/j.jand.2023.07.008

---

## Disturbance of eucalypt forests alters the composition, function, and assembly of soil microbial communities.
 23.07.2023, _ED Osburn, C Moon, T Stephenson, K Kittipalawattanapol, M Jones, MS Strickland, LM Lynch_


Forest disturbance has well-characterized effects on soil microbial communities in tropical and northern hemisphere ecosystems, but little is known regarding effects of disturbance in temperate forests of the southern hemisphere. To address this question, we collected soils from intact and degraded Eucalyptus forests along an east-west transect across Tasmania, Australia, and characterized prokaryotic and fungal communities using amplicon sequencing. Forest degradation altered soil microbial community composition and function, with consistent patterns across soil horizons and regions of Tasmania. Responses of prokaryotic communities included decreased relative abundance of Acidobacteriota, nitrifying archaea, and methane-oxidizing prokaryotes in the degraded forest sites, while fungal responses included decreased relative abundance of some saprotrophic taxa (e.g. litter saprotrophs). Forest degradation also reduced network connectivity in prokaryotic communities and increased the importance of dispersal limitation in assembling both prokaryotic and fungal communities, suggesting recolonization dynamics drive microbial composition following disturbance. Further, changes in microbial functional groups reflected changes in soil chemical properties-reductions in nitrifying microorganisms corresponded with reduced NO3-N pools in the degraded soils. Overall, our results show that soil microbiota are highly responsive to forest degradation in eucalypt forests and demonstrate that microbial responses to degradation will drive changes in key forest ecosystem functions.

[Source](https://doi.org/10.1093/femsec/fiad085)

10.1093/femsec/fiad085

---

## Reciprocal influence of soil, phyllosphere, and aphid microbiomes.
 24.07.2023, _A Wolfgang, AJM Tack, G Berg, A Abdelfattah_


The effect of soil on the plant microbiome is well-studied. However, less is known about the impact of the soil microbiome in multitrophic systems. Here we examined the effect of soil on plant and aphid microbiomes, and the reciprocal effect of aphid herbivory on the plant and soil microbiomes. We designed microcosms, which separate below and aboveground compartments, to grow oak seedlings with and without aphid herbivory in soils with three different microbiomes. We used amplicon sequencing and qPCR to characterize the bacterial and fungal communities in soils, phyllospheres, and aphids.

[Source](https://doi.org/10.1186/s40793-023-00515-8)

10.1186/s40793-023-00515-8

---

## Co-evolution within the plant holobiont drives host performance.
 20.07.2023, _F Mesny, S Hacquard, BP Thomma_


Plants interact with a diversity of microorganisms that influence their growth and resilience, and they can therefore be considered as ecological entities, namely "plant holobionts," rather than as singular organisms. In a plant holobiont, the assembly of above- and belowground microbiota is ruled by host, microbial, and environmental factors. Upon microorganism perception, plants activate immune signaling resulting in the secretion of factors that modulate microbiota composition. Additionally, metabolic interdependencies and antagonism between microbes are driving forces for community assemblies. We argue that complex plant-microbe and intermicrobial interactions have been selected for during evolution and may promote the survival and fitness of plants and their associated microorganisms as holobionts. As part of this process, plants evolved metabolite-mediated strategies to selectively recruit beneficial microorganisms in their microbiota. Some of these microbiota members show host-adaptation, from which mutualism may rapidly arise. In the holobiont, microbiota members also co-evolved antagonistic activities that restrict proliferation of microbes with high pathogenic potential and can therefore prevent disease development. Co-evolution within holobionts thus ultimately drives plant performance.

[Source](https://doi.org/10.15252/embr.202357455)

10.15252/embr.202357455

---

## May microbial ecological baseline exist in continental groundwater?
 26.07.2023, _S Zhong, S Zhou, S Liu, J Wang, C Dang, Q Chen, J Hu, S Yang, C Deng, W Li, J Liu, AGL Borthwick, J Ni_


_Bacteria, Biota, Groundwater, Microbiota, Water_
Microbes constitute almost the entire biological community in subsurface groundwater and play an important role in ecological evolution and global biogeochemical cycles. Ecological baseline as a fundamental reference with less human interference has been investigated in surface ecosystems such as soils, rivers, and ocean, but the existence of groundwater microbial ecological baseline (GMEB) is still an open question so far.

[Source](https://doi.org/10.1186/s40168-023-01572-4)

10.1186/s40168-023-01572-4

---

## Evolutionary patterns of archaea predominant in acidic environment.
 21.07.2023, _R Bargiela, AA Korzhenkov, OA McIntosh, SV Toshchakov, MM Yakimov, PN Golyshin, OV Golyshina_


Archaea of the order Thermoplasmatales are widely distributed in natural acidic areas and are amongst the most acidophilic prokaryotic organisms known so far. These organisms are difficult to culture, with currently only six genera validly published since the discovery of Thermoplasma acidophilum in 1970. Moreover, known great diversity of uncultured Thermoplasmatales represents microbial dark matter and underlines the necessity of efforts in cultivation and study of these archaea. Organisms from the order Thermoplasmatales affiliated with the so-called "alphabet-plasmas", and collectively dubbed "E-plasma", were the focus of this study. These archaea were found predominantly in the hyperacidic site PM4 of Parys Mountain, Wales, UK, making up to 58% of total metagenomic reads. However, these archaea escaped all cultivation attempts.

[Source](https://doi.org/10.1186/s40793-023-00518-5)

10.1186/s40793-023-00518-5

---

## Machine learning-based approaches for cancer prediction using microbiome data.
 24.07.2023, _P Freitas, F Silva, JV Sousa, RM Ferreira, C Figueiredo, T Pereira, HP Oliveira_


_Humans, Colonic Neoplasms, Rectal Neoplasms, Stomach Neoplasms, Machine Learning, Microbiota_
Emerging evidence of the relationship between the microbiome composition and the development of numerous diseases, including cancer, has led to an increasing interest in the study of the human microbiome. Technological breakthroughs regarding DNA sequencing methods propelled microbiome studies with a large number of samples, which called for the necessity of more sophisticated data-analytical tools to analyze this complex relationship. The aim of this work was to develop a machine learning-based approach to distinguish the type of cancer based on the analysis of the tissue-specific microbial information, assessing the human microbiome as valuable predictive information for cancer identification. For this purpose, Random Forest algorithms were trained for the classification of five types of cancer-head and neck, esophageal, stomach, colon, and rectum cancers-with samples provided by The Cancer Microbiome Atlas database. One versus all and multi-class classification studies were conducted to evaluate the discriminative capability of the microbial data across increasing levels of cancer site specificity, with results showing a progressive rise in difficulty for accurate sample classification. Random Forest models achieved promising performances when predicting head and neck, stomach, and colon cancer cases, with the latter returning accuracy scores above 90% across the different studies conducted. However, there was also an increased difficulty when discriminating esophageal and rectum cancers, failing to differentiate with adequate results rectum from colon cancer cases, and esophageal from head and neck and stomach cancers. These results point to the fact that anatomically adjacent cancers can be more complex to identify due to microbial similarities. Despite the limitations, microbiome data analysis using machine learning may advance novel strategies to improve cancer detection and prevention, and decrease disease burden.

[Source](https://doi.org/10.1038/s41598-023-38670-0)

10.1038/s41598-023-38670-0

---

## Bioinformatic mining for RiPP biosynthetic gene clusters in Bacteroidales reveals possible new subfamily architectures and novel natural products.
 21.07.2023, _MV Fernandez-Cantos, D Garcia-Morena, Y Yi, L Liang, E Gómez-Vázquez, OP Kuipers_


The Bacteroidales order, widely distributed among diverse human populations, constitutes a key component of the human microbiota. Members of this Gram-negative order have been shown to modulate the host immune system, play a fundamental role in the gut's microbial food webs, or be involved in pathogenesis. Bacteria inhabiting such a complex environment as the human microbiome are expected to display social behaviors and, hence, possess factors that mediate cooperative and competitive interactions. Different types of molecules can mediate interference competition, including non-ribosomal peptides (NRPs), polyketides, and bacteriocins. The present study investigates the potential of Bacteroidales bacteria to biosynthesize class I bacteriocins, which are ribosomally synthesized and post-translationally modified peptides (RiPPs). For this purpose, 1,136 genome-sequenced strains from this order were mined using BAGEL4. A total of 1,340 areas of interest (AOIs) were detected. The most commonly identified enzymes involved in RiPP biosynthesis were radical S-adenosylmethionine (rSAM), either alone or in combination with other biosynthetic enzymes such as YcaO. A more comprehensive analysis of a subset of 9 biosynthetic gene clusters (BGCs) revealed a consistent association in Bacteroidales BGCs between peptidase-containing ATP-binding transporters (PCATs) and precursor peptides with GG-motifs. This finding suggests a possibly shared mechanism for leader peptide cleavage and transport of mature products. Notably, human metagenomic studies showed a high prevalence and abundance of the RiPP BGCs from Phocaeicola vulgatus and Porphyromonas gulae. The mature product of P. gulae BGC is hypothesized to display γ-thioether linkages and a C-terminal backbone amidine, a potential new combination of post-translational modifications (PTM). All these findings highlight the RiPP biosynthetic potential of Bacteroidales bacteria, as a rich source of novel peptide structures of possible relevance in the human microbiome context.

[Source](https://doi.org/10.3389/fmicb.2023.1219272)

10.3389/fmicb.2023.1219272

---

## A bioinformatic analysis of zinc transporters in intestinal Lactobacillaceae.
 18.07.2023, _U Huynh, HN Nguyen, BK Trinh, J Elhaj, ML Zastrow_


As the second most abundant transition element and a crucial cofactor for many proteins, zinc is essential for the survival of all living organisms. To maintain required zinc levels and prevent toxic overload, cells and organisms have a collection of metal transport proteins for uptake and efflux of zinc. In bacteria, metal transport proteins are well defined for model organisms and many pathogens, but fewer studies have explored metal transport proteins, including those for zinc, in commensal bacteria from the gut microbiota. The healthy human gut microbiota is comprised of hundreds of species and among these, bacteria from the Lactobacillaceae family are well documented to have various beneficial effects on health. Furthermore, changes in dietary metal intake, such as for zinc and iron, are frequently correlated with changes in abundance of Lactobacillaceae. Few studies have explored zinc requirements and zinc homeostasis mechanisms in Lactobacillaceae, however. Here we applied a bioinformatics approach to identify and compare predicted zinc uptake and efflux proteins in several Lactobacillaceae genera of intestinal relevance. Few Lactobacillaceae had zinc transporters currently annotated in proteomes retrieved from the UniProt database, but protein sequence-based (BLASTP) homology searches revealed that high affinity ABC transporter genes are likely common, albeit with genus-specific domain features. P-type ATPase transporters are probably also common and some Lactobacillaceae genera code for predicted zinc efflux cation diffusion facilitators. This analysis confirms that Lactobacillaceae harbor genes for various zinc transporter homologs, and provides a foundation for systematic experimental studies to elucidate zinc homeostasis mechanisms in these bacteria.

[Source](https://doi.org/10.1093/mtomcs/mfad044)

10.1093/mtomcs/mfad044

---

## Uncovering the Carboxylated Metabolome in Gut Microbiota-Host Co-metabolism: A Chemical Derivatization-Molecular Networking Approach.
 20.07.2023, _YZ Wang, YY Chen, XZ Wu, PR Bai, N An, XL Liu, QF Zhu, YQ Feng_


Gut microbiota-host co-metabolites serve as essential mediators of communication between the host and gut microbiota. They provide nutrient sources for host cells and regulate gut microenvironment, which are associated with a variety of diseases. Analysis of gut microbiota-host co-metabolites is of great significance to explore the host-gut microbiota interaction. In this study, we integrated chemical derivatization, liquid chromatography-mass spectrometry, and molecular networking (MN) to establish a novel CD-MN strategy for the analysis of carboxylated metabolites in gut microbial-host co-metabolism. Using this strategy, 261 carboxylated metabolites from mouse feces were detected, which grouped to various classes including fatty acids, bile acids, N-acyl amino acids, benzoheterocyclic acids, aromatic acids, and other unknown small-scale molecular clusters in MN. Based on the interpretation of the bile acid cluster, a novel type of phenylacetylated conjugates of host bile acids was identified, which were mediated by gut microbiota and exhibited a strong binding ability to Farnesoid X receptor and Takeda G protein-coupled receptor 5. Our proposed strategy offers a promising platform for uncovering carboxylated metabolites in gut microbial-host co-metabolism.

[Source](https://doi.org/10.1021/acs.analchem.3c02353)

10.1021/acs.analchem.3c02353

---

