## Gut microbiota wellbeing index predicts overall health in a cohort of 1000 infants

[Source](https://www.nature.com/articles/s41467-024-52561-6)

---

## Pangenomes of human gut microbiota uncover links between genetic diversity and stress response.
 01.10.2024, _S Shoer, L Reicher, C Zhao, KS Pollard, Y Pilpel, E Segal_


The genetic diversity of the gut microbiota has a central role in host health. Here, we created pangenomes for 728 human gut prokaryotic species, quadrupling the genes of strain-specific genomes. Each of these species has a core set of a thousand genes, differing even between closely related species, and an accessory set of genes unique to the different strains. Functional analysis shows high strain variability associates with sporulation, whereas low variability is linked with antibiotic resistance. We further map the antibiotic resistome across the human gut population and find 237 cases of extreme resistance even to last-resort antibiotics, with a predominance among Enterobacteriaceae. Lastly, the presence of specific genes in the microbiota relates to host age and sex. Our study underscores the genetic complexity of the human gut microbiota, emphasizing its significant implications for host health. The pangenomes and antibiotic resistance map constitute a valuable resource for further research.

[Source](https://www.cell.com/cell-host-microbe/abstract/S1931-3128(24)00324-X)

10.1016/j.chom.2024.08.017

---

## The human gut microbiome in health and disease: time for a new chapter?
 30.09.2024, _J-Y Lee, DJ Bays, HP Savage, AJ Bäumler_


The gut microbiome, composed of the colonic microbiota and their host environment, is important for many aspects of human health. A gut microbiome imbalance (gut dysbiosis) is associated with major causes of human morbidity and mortality. Despite the central part our gut microbiome plays in health and disease, mechanisms that maintain homeostasis and properties that demarcate dysbiosis remain largely undefined. Here we discuss that sorting taxa into meaningful ecological units reveals that the availability of respiratory electron acceptors, such as oxygen, in the host environment has a dominant influence on gut microbiome health. During homeostasis, host functions that limit the diffusion of oxygen into the colonic lumen shelter a microbial community dominated by primary fermenters from atmospheric oxygen. In turn, primary fermenters break down unabsorbed nutrients into fermentation products that support host nutrition. This symbiotic relationship is disrupted when host functions that limit the luminal availability of host-derived electron acceptors become weakened. The resulting changes in the host environment drive alterations in the microbiota composition, which feature an elevated abundance of facultatively anaerobic microbes. Thus, the part of the gut microbiome that becomes imbalanced during dysbiosis is the host environment, whereas changes in the microbiota composition are secondary to this underlying cause. This shift in our understanding of dysbiosis provides a novel starting point for therapeutic strategies to restore microbiome health. Such strategies can either target the microbes through metabolism-based editing or strengthen the host functions that control their environment.

[Source](https://journals.asm.org/doi/10.1128/iai.00302-24)

10.1128/iai.00302-24

---

## Metabolic interactions shape emergent biofilm structures in a conceptual model of gut mucosal bacterial communities 

[Source](https://www.nature.com/articles/s41522-024-00572-y)

---

## A population-scale analysis of 36 gut microbiome studies reveals universal species signatures for common diseases.
 03.10.2024, _W Sun, Y Zhang, R Guo, S Sha, C Chen, H Ullah, Y Zhang, J Ma, W You, J Meng, Q Lv, L Cheng, S Fan, R Li, X Mu, S Li, Q Yan_


_Humans, Gastrointestinal Microbiome, Feces, Case-Control Studies, Bacteria, Metagenome, Metagenomics, China, Biodiversity_
The gut microbiome has been implicated in various human diseases, though findings across studies have shown considerable variability. In this study, we reanalyzed 6314 publicly available fecal metagenomes from 36 case-control studies on different diseases to investigate microbial diversity and disease-shared signatures. Using a unified analysis pipeline, we observed reduced microbial diversity in many diseases, while some exhibited increased diversity. Significant alterations in microbial communities were detected across most diseases. A meta-analysis identified 277 disease-associated gut species, including numerous opportunistic pathogens enriched in patients and a depletion of beneficial microbes. A random forest classifier based on these signatures achieved high accuracy in distinguishing diseased individuals from controls (AUC = 0.776) and high-risk patients from controls (AUC = 0.825), and it also performed well in external cohorts. These results offer insights into the gut microbiome's role in common diseases in the Chinese population and will guide personalized disease management strategies.

[Source](https://www.nature.com/articles/s41522-024-00567-9)

10.1038/s41522-024-00567-9

---

## A single amplified genome catalog reveals the dynamics of mobilome and resistome in the human microbiome.
 02.10.2024, _T Kawano-Sugaya, K Arikawa, T Saeki, T Endoh, K Kamata, A Matsuhashi, M Hosokawa_


_Humans, Bacteria, Gastrointestinal Microbiome, Metagenome, Mouth, Genome, Bacterial, Interspersed Repetitive Sequences, Microbiota, Drug Resistance, Bacterial, Metagenomics, Phylogeny_
The increase in metagenome-assembled genomes (MAGs) has advanced our understanding of the functional characterization and taxonomic assignment within the human microbiome. However, MAGs, as population consensus genomes, often aggregate heterogeneity among species and strains, thereby obfuscating the precise relationships between microbial hosts and mobile genetic elements (MGEs). In contrast, single amplified genomes (SAGs) derived via single-cell genome sequencing can capture individual genomic content, including MGEs.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01903-z)

10.1186/s40168-024-01903-z

---

## Stratification of human gut microbiomes by succinotype is associated with inflammatory bowel disease status.
 03.10.2024, _L Anthamatten, PR von Bieberstein, C Menzi, JN Zünd, C Lacroix, T de Wouters, GE Leventhal_


_Humans, Gastrointestinal Microbiome, Feces, Inflammatory Bowel Diseases, Succinic Acid, Bacteria, RNA, Ribosomal, 16S, Adult, Male, Female_
The human gut microbiome produces and consumes a variety of compounds that interact with the host and impact health. Succinate is of particular interest as it intersects with both host and microbiome metabolism. However, which gut bacteria are most responsible for the consumption of intestinal succinate is poorly understood.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01897-8)

10.1186/s40168-024-01897-8

---

## Impact of complementary feeding on infant gut microbiome, metabolites and early development.
 02.10.2024, _Y Shi, R Yin, J Pang, Y Chen, Z Li, S Su, Y Wen_


Introducing complementary foods is critical for promoting infant health and development. During the weaning period, the dietary patterns provide essential nutrients and facilitate the development of a diverse gut microbiome, which plays significant roles in the regulation of immune, metabolic, and neurological functions. This study enrolled 200 families to assess the impact of complementary feeding on infant growth and health outcomes. Data included detailed records of feeding practices, infant growth measurements, health assessments, and fecal samples and breast milk collected between weeks 12 and 32 postpartum. The gut microbiome was analyzed using 16S rRNA sequencing, while metabolites such as human milk oligosaccharides (HMOs), monosaccharides, and short-chain fatty acids (SCFAs) were measured using chromatography-mass spectrometry. Results revealed a high prevalence of breastfeeding, with complementary food introduced at around 16 weeks. Significant alterations in the infant gut microbiome were observed, particularly in the genera Lactobacillus, Akkermansia, and Staphylococcus. Additionally, the levels of HMOs, monosaccharides, and SCFAs were found to be influenced by the introduction of complementary foods. Significant correlations emerged between complementary feeding practices, gut microbiome diversity, specific bacterial genera (e.g., Streptococcus, Lactobacillus, Bifidobacterium, and Clostridioides), and key metabolites (such as lacto-N-tetraose, lacto-N-neotetraose, mannose, and butyric acid). This study offers valuable insights into the complex interactions between complementary feeding, gut microbiome development, and metabolite profiles during early infant growth. Future research with larger cohorts and targeted dietary interventions is recommended to further elucidate the underlying mechanisms.

[Source](https://pubs.rsc.org/en/content/articlelanding/2024/fo/d4fo03948c)

10.1039/d4fo03948c

---

## Can longer lifespan be associated with gut microbiota involvement in lipid metabolism?
 02.10.2024, _S Šardzíková, M Gajewska, N Gałka, M Štefánek, A Baláž, M Garaiová, R Holič, W Świderek, K Šoltys_


Biological aging is linked to altered body composition and reduced neuroactive steroid hormones like dehydroepiandrosterone sulfate (DHEAS), which can stimulate the GABA signaling pathway via gut microbiota. Our study examined the association of gut microbiota with lifespan in mice through comprehensive analysis of its composition and functional involvement in cholesterol sulfate, a precursor of DHEAS, metabolism. We used 16S rRNA and metagenomic sequencing, followed by metabolic pathway prediction and TLC and MALDI-TOF cholesterol sulfate identification. Significant increases in bacteria such as Bacteroides, typical for long-lived and Odoribacter and Colidextribacter, specific for short-lived mice were detected. Furthermore, for males (Rikenella, Alloprevotella) and females (Lactobacillus, Bacteroides), specific bacterial groups emerged as predictors (AUC=1), highlighting sex-specific patterns. Long-lived mice showed a strong correlation of Bacteroides (0.918) with lipid and steroid hormone metabolism, while a negative correlation of GABAergic synapse with body weight (-0.589). We found that several Bacteroides species harboring the sulfotransferase gene and gene cluster for sulfonate donor synthesis are involved in converting cholesterol to cholesterol sulfate, significantly higher in the feces of long-lived individuals. Overall, we suggest that increased involvement of gut bacteria, mainly Bacteroides spp., in cholesterol sulfate synthesis could ameliorate aging through lipid metabolism.

[Source](https://academic.oup.com/femsec/advance-article/doi/10.1093/femsec/fiae135/7804367)

10.1093/femsec/fiae135

---

## Preventive effect of probiotics on oral mucositis induced by anticancer therapy: a systematic review and meta-analysis of randomized controlled trials.
 03.10.2024, _B Yang, W Li, J Shi_


_Humans, Antineoplastic Agents, Neoplasms, Probiotics, Quality of Life, Randomized Controlled Trials as Topic, Stomatitis_
Oral mucositis (OM) is a prevalent and painful complication in patients undergoing anticancer treatment, which significantly impacts patients' quality of life (QoL) and adherence to therapy. The use of oral probiotics as a preventive strategy for OM has shown promise, but the clinical evidence remains inconclusive. This meta-analysis of randomized controlled trials (RCTs) aims to evaluate the efficacy of probiotics in preventing OM caused by radiotherapy and/or chemotherapy.

[Source](https://bmcoralhealth.biomedcentral.com/articles/10.1186/s12903-024-04955-7)

10.1186/s12903-024-04955-7

---

## Bacteria from the Amycolatopsis genus associated with a toxic bird secrete protective secondary metabolites.
 02.10.2024, _E Seibel, S Um, KH Bodawatta, AJ Komor, T Decker, J Fricke, R Murphy, G Maiah, B Iova, H Maus, T Schirmeister, KA Jønsson, M Poulsen, C Beemelmanns_


_Animals, Secondary Metabolism, Phylogeny, Amycolatopsis, Birds, Microbiota, Lipopeptides, Feathers, Tandem Mass Spectrometry, Multigene Family_
Uropygial gland secretions of birds consist of host and bacteria derived compounds and play a major sanitary and feather-protective role. Here we report on our microbiome studies of the New Guinean toxic bird Pachycephala schlegelii and the isolation of a member of the Amycolatopsis genus from the uropygial gland secretions. Bioactivity studies in combination with co-cultures, MALDI imaging and HR-MS/MS-based network analyses unveil the basis of its activity against keratinolytic bacteria and fungal skin pathogens. We trace the protective antimicrobial activity of Amycolatopsis sp. PS_44_ISF1 to the production of rifamycin congeners, ciromicin A and of two yet unreported compound families. We perform NMR and HR-MS/MS studies to determine the relative structures of six members belonging to a yet unreported lipopeptide family of pachycephalamides and of one representative of the demiguisins, a new hexapeptide family. We then use a combination of phylogenomic, transcriptomic and knock-out studies to identify the underlying biosynthetic gene clusters responsible for the production of pachycephalamides and demiguisins. Our metabolomics data allow us to map molecular ion features of the identified metabolites in extracts of P. schlegelii feathers, verifying their presence in the ecological setting where they exert their presumed active role for hosts. Our study shows that members of the Actinomycetota may play a role in avian feather protection.

[Source](https://www.nature.com/articles/s41467-024-52316-3)

10.1038/s41467-024-52316-3

---

## Impact of virginiamycin on the ruminal microbiota of feedlot cattle

[Source](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11418755/)

---

## From carrion to soil: microbial recycling of animal carcasses

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(24)00229-4)

---

## Microbiome–metabolite linkages drive greenhouse gas dynamics over a permafrost thaw gradient 

[Source](https://www.nature.com/articles/s41564-024-01800-z)

---

## Prescribed fire regimes influence responses of fungal and bacterial communities on new litter substrates in a brackish tidal marsh

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0311230)

---

## The phyllosphere of Nigerian medicinal plants, Euphorbia lateriflora and Ficus thonningii is inhabited by a specific microbiota.
 03.10.2024, _AO Oaikhena, ME Coker, D Cyril-Okoh, WA Wicaksono, E Olimi, G Berg, IN Okeke_


_Euphorbia, Ficus, Microbiota, Plants, Medicinal, Bacteria, RNA, Ribosomal, 16S, Plant Leaves, Fungi, Nigeria, Phylogeny_
The microbiota of medicinal plants is known to be highly specific and can contribute to medicinal activity. However, the majority of plant species have not yet been studied. Here, we investigated the phyllosphere composition of two common Nigerian medicinal plants, Euphorbia lateriflora and Ficus thonningii, by a polyphasic approach combining analyses of metagenomic DNA and isolates. Microbial abundance estimated via qPCR using specific marker gene primers showed that all leaf samples were densely colonized, with up to 108 per gram of leaf, with higher bacterial and fungal abundance than Archaea. While no statistically significant differences between both plant species were found for abundance, amplicon sequencing of 16S rRNA and ITS genes revealed distinct microbiota compositions. Only seven of the 27 genera isolated were represented on both plants, e.g. dominant Sphingomonas spp., and numerous members of Xanthomonadaceae and Enterobacteriaceae. The most dominant fungal families on both plants were Cladosporiaceae, Mycosphaerellaceae and Trichosphaeriaceae. In addition, 225 plant-specific isolates were identified, with Pseudomonadota and Enterobacteriaceae being dominant. Interestingly, 29 isolates are likely species previously unknown, and 14 of these belong to Burkholderiales. However, a high proportion, 56% and 40% of the isolates from E. lateriflora and F. thonningii, respectively, were characterized as various Escherichia coli. The growth of most of the bacterial isolates was not influenced by extractable secondary metabolites of plants. Our results suggest that a specific and diverse microbial community inhabits the leaves of both E. lateriflora and F. thonningii, including potentially new species and producers of antimicrobials.

[Source](https://www.nature.com/articles/s41598-024-68001-w)

10.1038/s41598-024-68001-w

---

## Water‒soil-air‒plant mutual feedback mechanism under the application of red bed composite polymers.
 01.10.2024, _T Chen, G Cui, C Zhou, Z Liu_


_Soil, Water, Polymers, Air, Temperature, Hydrogen-Ion Concentration, Germination, Environmental Restoration and Remediation, Carbon Dioxide_
Red bed composite polymers composed of weathered red bed soil, adhesive materials, and water-retaining materials have been applied as a new type of material for environmental restoration. However, the promotion and application of this material has been limited by a lack of understanding of its action mechanism in environmental restoration. The objective of this study is to innovatively propose a water‒soil-air‒plant mutual feedback mechanism based on this material. Therefore, water‒soil-air‒plant mutual feedback tests were conducted in this study under 3 initial water contents and 10 red bed composite polymers ratios. Key parameters, namely, water content, soil conductivity, pH, temperature, O2 and CO2 contents, pigeon pea (Cajanus cajan) germination number and plant height were monitored and analyzed. As the results, a mutual feedback mechanism driving water retention, soil consolidation, air retention, and plant rooting was revealed under the application of red bed composite polymers. And, suitable environments and optimal compositions for this material are proposed. The study results provide a theoretical basis for the large-scale application of red bed composite polymers.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0310172)

10.1371/journal.pone.0310172

---

## Diversity and community distribution of soil bacterial in the Yellow River irrigation area of Ningxia, China.
 03.10.2024, _X Wu, J Cai, Z Wang, W Li, G Chen, Y Bai_


_China, Soil Microbiology, Rivers, Bacteria, RNA, Ribosomal, 16S, Biodiversity, Soil, Agricultural Irrigation, Phylogeny_
The bacterial community performs an essential ecological role in maintaining agriculture systems. The roles of bacteria in the forest, marine, and agricultural systems have been studied extensively and intensively. However, similar studies in the areas irrigated by the Yellow River remain limited. In this study, we used Illumina sequencing analysis with the 16S rRNA method to analyze the bacterial diversity, community structure, and influencing factors in soil samples from eight regions of the Yellow River irrigation area in northwestern China. The bacterial community structure and diversity varied among samples from the eight regions. The samples differed significantly in terms of the bacterial community composition. Proteobacteria (approximately 12.4%-55.7%) accounted for the largest proportion and was the dominant bacteria, followed by Actinobacteria (approximately 9.2%-39.7%), Bacteroidetes (approximately 1.8%-21.5%), and Chloroflexi (approximately 2.7%-12.6%). Among the physicochemical variables, the soil pH in the eight regions was mildly alkaline, and the total nitrogen, total phosphorus, and total potassium contents in the soils differed significantly. However, the trend in the variations of the above variables was essentially similar. Soil bacteria in Yongning county had greater Chao1, Shannon-Wiener, and Simpson indices than those in the other regions. Notably, soil moisture, organic matter, and total nitrogen were recognized as the primary factors influencing the bacterial community in the Yellow River irrigation area. Our results revealed the laws of variation in soil bacterial diversity and community composition in the Yellow River irrigation area. Our findings could be beneficial for maintaining sustainable ecological practices in the Yellow River irrigation area.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0311087)

10.1371/journal.pone.0311087

---

## Strain-resolved de-novo metagenomic assembly of viral genomes and microbial 16S rRNAs.
 01.10.2024, _A Jochheim, FA Jochheim, A Kolodyazhnaya, É Morice, M Steinegger, J Söding_


_RNA, Ribosomal, 16S, Genome, Viral, Metagenomics, Bacteria, Software, Humans, Bayes Theorem, Viruses, Metagenome_
Metagenomics is a powerful approach to study environmental and human-associated microbial communities and, in particular, the role of viruses in shaping them. Viral genomes are challenging to assemble from metagenomic samples due to their genomic diversity caused by high mutation rates. In the standard de Bruijn graph assemblers, this genomic diversity leads to complex k-mer assembly graphs with a plethora of loops and bulges that are challenging to resolve into strains or haplotypes because variants more than the k-mer size apart cannot be phased. In contrast, overlap assemblers can phase variants as long as they are covered by a single read.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01904-y)

10.1186/s40168-024-01904-y

---

