## Metabolomic and microbiome profiling reveals personalized risk factors for coronary artery disease.
 18.02.2022, _Y Talmor-Barkan, N Bar, AA Shaul, N Shahaf, A Godneva, Y Bussi, M Lotan-Pompan, A Weinberger, A Shechter, C Chezar-Azerrad, Z Arow, Y Hammer, K Chechi, SK Forslund, S Fromentin, ME Dumas, SD Ehrlich, O Pedersen, R Kornowski, E Segal_


Complex diseases, such as coronary artery disease (CAD), are often multifactorial, caused by multiple underlying pathological mechanisms. Here, to study the multifactorial nature of CAD, we performed comprehensive clinical and multi-omic profiling, including serum metabolomics and gut microbiome data, for 199 patients with acute coronary syndrome (ACS) recruited from two major Israeli hospitals, and validated these results in a geographically distinct cohort. ACS patients had distinct serum metabolome and gut microbial signatures as compared with control individuals, and were depleted in a previously unknown bacterial species of the Clostridiaceae family. This bacterial species was associated with levels of multiple circulating metabolites in control individuals, several of which have previously been linked to an increased risk of CAD. Metabolic deviations in ACS patients were found to be person specific with respect to their potential genetic or environmental origin, and to correlate with clinical parameters and cardiovascular outcomes. Moreover, metabolic aberrations in ACS patients linked to microbiome and diet were also observed to a lesser extent in control individuals with metabolic impairment, suggesting the involvement of these aberrations in earlier dysmetabolic phases preceding clinically overt CAD. Finally, a metabolomics-based model of body mass index (BMI) trained on the non-ACS cohort predicted higher-than-actual BMI when applied to ACS patients, and the excess BMI predictions independently correlated with both diabetes mellitus (DM) and CAD severity, as defined by the number of vessels involved. These results highlight the utility of the serum metabolome in understanding the basis of risk-factor heterogeneity in CAD.

[Source](https://www.nature.com/articles/s41591-022-01686-6)

10.1038/s41591-022-01686-6

---

## Microbiome and metabolome features of the cardiometabolic disease spectrum.
 18.02.2022, _S Fromentin, SK Forslund, K Chechi, J Aron-Wisnewsky, R Chakaroun, T Nielsen, V Tremaroli, B Ji, E Prifti, A Myridakis, J Chilloux, P Andrikopoulos, Y Fan, MT Olanipekun, R Alves, S Adiouch, N Bar, Y Talmor-Barkan, E Belda, R Caesar, LP Coelho, G Falony, S Fellahi, P Galan, N Galleron, G Helft, L Hoyles, R Isnard, E Le Chatelier, H Julienne, L Olsson, HK Pedersen, N Pons, B Quinquis, C Rouault, H Roume, JE Salem, TSB Schmidt, S Vieira-Silva, P Li, M Zimmermann-Kogadeeva, C Lewinter, NB Søndertoft, TH Hansen, D Gauguier, JP Gøtze, L Køber, R Kornowski, H Vestergaard, T Hansen, JD Zucker, S Hercberg, I Letunic, F Bäckhed, JM Oppert, J Nielsen, J Raes, P Bork, M Stumvoll, E Segal, K Clément, ME Dumas, SD Ehrlich, O Pedersen_


Previous microbiome and metabolome analyses exploring non-communicable diseases have paid scant attention to major confounders of study outcomes, such as common, pre-morbid and co-morbid conditions, or polypharmacy. Here, in the context of ischemic heart disease (IHD), we used a study design that recapitulates disease initiation, escalation and response to treatment over time, mirroring a longitudinal study that would otherwise be difficult to perform given the protracted nature of IHD pathogenesis. We recruited 1,241 middle-aged Europeans, including healthy individuals, individuals with dysmetabolic morbidities (obesity and type 2 diabetes) but lacking overt IHD diagnosis and individuals with IHD at three distinct clinical stages-acute coronary syndrome, chronic IHD and IHD with heart failure-and characterized their phenome, gut metagenome and serum and urine metabolome. We found that about 75% of microbiome and metabolome features that distinguish individuals with IHD from healthy individuals after adjustment for effects of medication and lifestyle are present in individuals exhibiting dysmetabolism, suggesting that major alterations of the gut microbiome and metabolome might begin long before clinical onset of IHD. We further categorized microbiome and metabolome signatures related to prodromal dysmetabolism, specific to IHD in general or to each of its three subtypes or related to escalation or de-escalation of IHD. Discriminant analysis based on specific IHD microbiome and metabolome features could better differentiate individuals with IHD from healthy individuals or metabolically matched individuals as compared to the conventional risk markers, pointing to a pathophysiological relevance of these features.

[Source](https://www.nature.com/articles/s41591-022-01688-4)

10.1038/s41591-022-01688-4

---

## Gut metagenome associations with extensive digital health data in a volunteer-based Estonian microbiome cohort

[Source](https://www.nature.com/articles/s41467-022-28464-9)

---

## Drivers of gut microbiome variation within and between groups of a wild Malagasy primate.
 16.02.2022, _K Rudolph, D Schneider, C Fichtel, R Daniel, M Heistermann, PM Kappeler_


Various aspects of sociality can benefit individuals' health. The host social environment and its relative contributions to the host-microbiome relationship have emerged as key topics in microbial research. Yet, understanding the mechanisms that lead to structural variation in the social microbiome, the collective microbial metacommunity of an animal's social network, remains difficult since multiple processes operate simultaneously within and among animal social networks. Here, we examined the potential drivers of the convergence of the gut microbiome on multiple scales among and within seven neighbouring groups of wild Verreaux's sifakas (Propithecus verreauxi) - a folivorous primate of Madagascar.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01223-6)

10.1186/s40168-021-01223-6

---

## Integrated meta-omics reveals new ruminal microbial features associated with feed efficiency in dairy cattle.
 17.02.2022, _MY Xue, YY Xie, Y Zhong, XJ Ma, HZ Sun, JX Liu_


As the global population continues to grow, competition for resources between humans and livestock has been intensifying. Increasing milk protein production and improving feed efficiency are becoming increasingly important to meet the demand for high-quality dairy protein. In a previous study, we found that milk protein yield in dairy cows was associated with the rumen microbiome. The objective of this study was to elucidate the potential microbial features that underpins feed efficiency in dairy cows using metagenomics, metatranscriptomics, and metabolomics.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01228-9)

10.1186/s40168-022-01228-9

---

## The gut microbiome and antibiotic resistome of chronic diarrhea rhesus macaques (Macaca mulatta) and its similarity to the human gut microbiome 

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01218-3)

---

## Independent host- and bacterium-based determinants protect a model symbiosis from phage predation

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(22)00097-3)

---

## Virome characterization of game animals in China reveals a spectrum of emerging pathogens – 

[Source](https://www.sciencedirect.com/science/article/pii/S0092867422001945)

---

## Microeukaryotic gut parasites in wastewater treatment plants: diversity, activity, and removal

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01225-y)

---

## METABOLIC: high-throughput profiling of microbial genomes for functional traits, metabolism, biogeochemistry, and community-scale functional networks

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01213-8)

---

## Sustained software development, not number of citations or journal choice, is indicative of accurate bioinformatic software

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02625-x)

---

## The post-Theranos world

[Source](https://www.nature.com/articles/s41587-022-01242-0)

---

