## Faecalibacterium prausnitzii increases following fecal microbiota transplantation in recurrent Clostridioides difficile infection.
 09.04.2021, _O Björkqvist, I Rangel, L Serrander, C Magnusson, J Halfvarson, T Norén, M Bergman-Jungeström_


Fecal microbiota transplantation (FMT) is a highly effective treatment for Clostridioides difficile infection (CDI). However, the fecal transplant's causal components translating into clearance of the CDI are yet to be identified. The commensal bacteria Faecalibacterium prausnitzii may be of great interest in this context, since it is one of the most common species of the healthy gut microbiota and produces metabolites with anti-inflammatory properties. Although there is mounting evidence that F. prausnitzii is an important regulator of intestinal homeostasis, data about its role in CDI and FMT are relatively scarce.

[Source](https://doi.org/10.1371/journal.pone.0249861)

10.1371/journal.pone.0249861

---

## Comprehensive profiling of the gut microbiota in patients with chronic obstructive pulmonary disease of varying severity.
 09.04.2021, _YC Chiu, SW Lee, CW Liu, RC Lin, YC Huang, TY Lan, LS Wu_


Chronic obstructive pulmonary disease (COPD) is a chronic respiratory disease that reduces lung and respiratory function, with a high mortality rate. Severe and acute deterioration of COPD can easily lead to respiratory failure, resulting in personal, social, and medical burden. Recent studies have shown a high correlation between the gut microbiota and lung inflammation. In this study, we investigated the relationship between gut microbiota and COPD severity. A total of 60 COPD patients with varying severity according to GOLD guidelines were enrolled in this study. DNA was extracted from patients' stool and 16S rRNA data analysis conducted using high-throughput sequencing followed by bioinformatics analysis. The richness of the gut microbiota was not associated with COPD severity. The gut microbiome is more similar in stage 1 and 2 COPD than stage 3+4 COPD. Fusobacterium and Aerococcus were more abundant in stage 3+4 COPD. Ruminococcaceae NK4A214 group and Lachnoclostridium were less abundant in stage 2-4, and Tyzzerella 4 and Dialister were less abundant in stage 1. However, the abundance of a Bacteroides was associated with blood eosinophils and lung function. This study suggests that no distinctive gut microbiota pattern is associated with the severity of COPD. The gut microbiome could affect COPD by gut inflammation shaping the host immune system.

[Source](https://doi.org/10.1371/journal.pone.0249944)

10.1371/journal.pone.0249944

---

## Nutrition and the microbiota post-COVID-19.
 09.04.2021, _GGF Alberca, RW Alberca_



[Source](https://doi.org/10.4103/sjg.sjg_75_21)

10.4103/sjg.sjg_75_21

---

## From "Leaky Gut" to Impaired Glia-Neuron Communication in Depression.
 09.04.2021, _L Rudzki, M Maes_


In the last three decades, the robust scientific data emerged, demonstrating that the immune-inflammatory response is a fundamental component of the pathophysiology of major depressive disorder (MDD). Psychological stress and various inflammatory comorbidities contribute to such immune activation. Still, this is not uncommon that patients with depression do not have defined inflammatory comorbidities, and alternative mechanisms of immune activation need to take place. The gastrointestinal (GI) tract, along with gut-associated lymphoid tissue (GALT), constitutes the largest lymphatic organ in the human body and forms the biggest surface of contact with the external environment. It is also the most significant source of bacterial and food-derived antigenic material. There is a broad range of reciprocal interactions between the GI tract, intestinal microbiota, increased intestinal permeability, activation of immune-inflammatory response, and the CNS that has crucial implications in brain function and mental health. This intercommunication takes place within the microbiota-gut-immune-glia (MGIG) axis, and glial cells are the main orchestrator of this communication. A broad range of factors, including psychological stress, inflammation, dysbiosis, may compromise the permeability of this barrier. This leads to excessive bacterial translocation and the excessive influx of food-derived antigenic material that contributes to activation of the immune-inflammatory response and depressive psychopathology. This chapter summarizes the role of increased intestinal permeability in MDD and mechanisms of how the "leaky gut" may contribute to immune-inflammatory response in this disorder.

[Source](https://doi.org/10.1007/978-981-33-6044-0_9)

10.1007/978-981-33-6044-0_9

---

## Skin microbiome of patients with interdigital tinea pedis: Corynebacterium striatum is more abundant in the patients.
 08.04.2021, _M Miyamoto, R Tsuboi, K Harada, O Cho, T Sugita_



[Source](https://doi.org/10.1111/1346-8138.15877)

10.1111/1346-8138.15877

---

## Selected Clostridia Strains from The Human Microbiota and their Metabolite, Butyrate, Improve Experimental Autoimmune Encephalomyelitis.
 08.04.2021, _L Calvo-Barreiro, H Eixarch, T Cornejo, C Costa, M Castillo, L Mestre, C Guaza, MDC Martínez-Cuesta, T Tanoue, K Honda, JJ González-López, X Montalban, C Espejo_


Gut microbiome studies in multiple sclerosis (MS) patients are unravelling some consistent but modest patterns of gut dysbiosis. Among these, a significant decrease of Clostridia cluster IV and XIVa has been reported. In the present study, we investigated the therapeutic effect of a previously selected mixture of human gut-derived 17 Clostridia strains, which belong to Clostridia clusters IV, XIVa, and XVIII, on the clinical outcome of experimental autoimmune encephalomyelitis (EAE). The observed clinical improvement was related to lower demyelination and astrocyte reactivity as well as a tendency to lower microglia reactivity/infiltrating macrophages and axonal damage in the central nervous system (CNS), and to an enhanced immunoregulatory response of regulatory T cells in the periphery. Transcriptome studies also highlighted increased antiinflammatory responses related to interferon beta in the periphery and lower immune responses in the CNS. Since Clostridia-treated mice were found to present higher levels of the immunomodulatory short-chain fatty acid (SCFA) butyrate in the serum, we studied if this clinical effect could be reproduced by butyrate administration alone. Further EAE experiments proved its preventive but slight therapeutic impact on CNS autoimmunity. Thus, this smaller therapeutic effect highlighted that the Clostridia-induced clinical effect was not exclusively related to the SCFA and could not be reproduced by butyrate administration alone. Although it is still unknown if these Clostridia strains will have the same effect on MS patients, gut dysbiosis in MS patients could be partially rebalanced by these commensal bacteria and their immunoregulatory properties could have a beneficial effect on MS clinical course.

[Source](https://doi.org/10.1007/s13311-021-01016-7)

10.1007/s13311-021-01016-7

---

## Gut Microbiota Composition and Epigenetic Molecular Changes Connected to the Pathogenesis of Alzheimer's Disease.
 08.04.2021, _P Nagu, A Parashar, T Behl, V Mehta_


Alzheimer's disease (AD) is a neurodegenerative disorder, and its pathogenesis is not fully known. Although there are several hypotheses, such as neuroinflammation, tau hyperphosphorylation, amyloid-β plaques, neurofibrillary tangles, and oxidative stress, none of them completely explain the origin and progression of AD. Emerging evidence suggests that gut microbiota and epigenetics can directly influence the pathogenesis of AD via their effects on multiple pathways, including neuroinflammation, oxidative stress, and amyloid protein. Various gut microbes such as Actinobacteria, Bacteroidetes, E. coli, Firmicutes, Proteobacteria, Tenericutes, and Verrucomicrobia are known to play a crucial role in the pathogenesis of AD. These microbes and their metabolites modulate various physiological processes that contribute to AD pathogenesis, such as neuroinflammation and other inflammatory processes, amyloid deposition, cytokine storm syndrome, altered BDNF and NMDA signaling, impairing neurodevelopmental processes. Likewise, epigenetic markers associated with AD mainly include histone modifications and DNA methylation, which are under the direct control of a variety of enzymes, such as acetylases and methylases. The activity of these enzymes is dependent upon the metabolites generated by the host's gut microbiome, suggesting the significance of epigenetics in AD pathogenesis. It is interesting to know that both gut microbiota and epigenetics are dynamic processes and show a high degree of variation according to diet, stressors, and environmental factors. The bidirectional relation between the gut microbiota and epigenetics suggests that they might work in synchrony to modulate AD representation, its pathogenesis, and progression. They both also provide numerous targets for early diagnostic biomarkers and for the development of AD therapeutics. This review discusses the gut microbiota and epigenetics connection in the pathogenesis of AD and aims to highlight vast opportunities for diagnosis and therapeutics of AD.

[Source](https://doi.org/10.1007/s12031-021-01829-3)

10.1007/s12031-021-01829-3

---

## Global climate change, diet, and the complex relationship between human host and microbiome: Towards an integrated picture.
 08.04.2021, _F Catania, J Baedke, A Fábregas-Tejeda, A Nieves Delgado, V Vitali, LAN Long_


Dietary changes can alter the human microbiome with potential detrimental consequences for health. Given that environment, health, and evolution are interconnected, we ask: Could diet-driven microbiome perturbations have consequences that extend beyond their immediate impact on human health? We address this question in the context of the urgent health challenges posed by global climate change. Drawing on recent studies, we propose that not only can diet-driven microbiome changes lead to dysbiosis, they can also shape life-history traits and fuel human evolution. We posit that dietary shifts prompt mismatched microbiome-host genetics configurations that modulate human longevity and reproductive success. These mismatches can also induce a heritable intra-holobiont stress response, which encourages the holobiont to re-establish equilibrium within the changed nutritional environment. Thus, while mismatches between climate change-related genetic and epigenetic configurations within the holobiont increase the risk and severity of diseases, they may also affect life-history traits and facilitate adaptive responses. These propositions form a framework that can help systematize and address climate-related dietary challenges for policy and health interventions.

[Source](https://doi.org/10.1002/bies.202100049)

10.1002/bies.202100049

---

## Global research trends in the microbiome related to irritable bowel syndrome: A bibliometric and visualized study.
 10.04.2021, _SH Zyoud, S Smale, WS Waring, W Sweileh, SW Al-Jabi_


Irritable bowel syndrome (IBS) is a common functional gastrointestinal disorder. Dysregulation of the gut-brain axis plays a central role in the pathophysiology of IBS. It is increasingly clear that the microbiome plays a key role in the development and normal functioning of the gut-brain axis.

[Source](https://doi.org/10.3748/wjg.v27.i13.1341)

10.3748/wjg.v27.i13.1341

---

## Coronavirus disease-2019 and the intestinal tract: An overview.
 10.04.2021, _GGF Alberca, RL Solis-Castro, ME Solis-Castro, RW Alberca_


Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) infection can progress to a severe respiratory and systemic disease named coronavirus disease-2019 (COVID-19). The most common symptoms are fever and respiratory discomfort. Nevertheless, gastrointestinal infections have been reported, with symptoms such as diarrhea, nausea, vomiting, abdominal pain, and lack of appetite. Importantly, SARS-CoV-2 can remain positive in fecal samples after nasopharyngeal clearance. After gastrointestinal SARS-CoV-2 infection and other viral gastrointestinal infections, some patients may develop alterations in the gastrointestinal microbiota. In addition, some COVID-19 patients may receive antibiotics, which may also disturb gastrointestinal homeostasis. In summary, the gastrointestinal system, gut microbiome, and gut-lung axis may represent an important role in the development, severity, and treatment of COVID-19. Therefore, in this review, we explore the current pieces of evidence of COVID-19 gastrointestinal manifestations, possible implications, and interventions.

[Source](https://doi.org/10.3748/wjg.v27.i13.1255)

10.3748/wjg.v27.i13.1255

---

## Comparison of Argentinean microbiota with other geographical populations reveals different taxonomic and functional signatures associated with obesity.
 11.04.2021, _SA Pesoa, N Portela, E Fernández, O Elbarcha, M Gotteland, F Magne_


Accumulating evidence suggests that various genetic and environmental factors contribute to the development of obesity. Among the latter, the gut microbiota has emerged as a critical player in the regulation of human metabolism and health and the development of non-communicable chronic diseases. Considering that no information on this matter is available in Argentina, our aim was to identify the microorganisms associated with obesity as well as their potential functionality. Using high throughput sequencing of 16SrRNA bacterial gene and diverse bioinformatics tools, we observed that the gut microbiota of obese and overweight individuals differs qualitatively and quantitatively from that from their lean counterparts. The comparison of the gut microbiota composition in obese subjects from Argentina, US and UK showed that the beta diversity significantly differs among the three countries, indicating that obesity-associated microbiota composition changes according to the geographical origin of the individuals. Moreover, four distinct microbiotypes were identified in obese individuals, whose prevalence and metabolic pathway signature differed according to the country, indicating that obesity associated dysbiosis would comprise several structures. In summary, identification of distinct taxonomic signatures associated with obesity might be a novel promising tool to stratify patients based on their microbiome configuration to design strategies for managing obesity.

[Source](https://doi.org/10.1038/s41598-021-87365-x)

10.1038/s41598-021-87365-x

---

## Six-month follow-up of gut microbiota richness in patients with COVID-19.
 09.04.2021, _Y Chen, S Gu, Y Chen, H Lu, D Shi, J Guo, WR Wu, Y Yang, Y Li, KJ Xu, C Ding, R Luo, C Huang, L Yu, M Xu, P Yi, J Liu, JJ Tao, H Zhang, L Lv, B Wang, J Sheng, L Li_



[Source](https://doi.org/10.1136/gutjnl-2021-324090)

10.1136/gutjnl-2021-324090

---

## The combined therapy of fecal microbiota transplantation and laxatives for functional constipation in adults: A systematic review and meta-analysis of randomized controlled trials.
 09.04.2021, _S Fang, S Wu, L Ji, Y Fan, X Wang, K Yang_


Functional constipation is a prevalent, burdensome gastrointestinal disorder whose treatment remains challenging. Combined therapy uniting multiple treatments may be promising. Fecal microbiota transplantation (FMT) which tends to be an etiological treatment has been increasingly investigated in its management. Meanwhile, laxatives are widely used to relieve constipation temporarily, but their overall efficacy is poor. Therefore, we performed meta-analyses of randomized controlled trials to evaluate the joint efficacy of FMT and laxatives in functional constipation.

[Source](https://doi.org/10.1097/md.0000000000025390)

10.1097/MD.0000000000025390

---

## Vaginal metabolic profiles during pregnancy: Changes between first and second trimester.
 08.04.2021, _L Laghi, S Zagonari, G Patuelli, C Zhu, C Foschi, S Morselli, MF Pedna, V Sambri, A Marangoni_


During pregnancy, the vaginal microbiome plays an important role in both maternal and neonatal health outcomes. Throughout pregnancy, the vaginal microbial composition undergoes significant changes, including a decrease in overall diversity and enrichment with Lactobacillus spp. In turn, the modifications in the microbial profiles are associated with shifts in the composition of vaginal metabolites. In this study, we characterized the vaginal metabolic profiles throughout pregnancy at two different gestational ages, correlating them with a microscopic evaluation of the vaginal bacterial composition. A total of 67 Caucasian pregnant women presenting to the Family Advisory Health Centres of Ravenna (Italy) were enrolled and a vaginal swab was collected at gestational ages 9-13 weeks (first trimester) and 20-24 weeks (second trimester). The composition of the vaginal microbiome was assessed by Nugent score and women were divided in 'H' (normal lactobacilli-dominated microbiota), 'I' (intermediate microbiota), and 'BV' (bacterial vaginosis) groups. Starting from the cell-free supernatants of the vaginal swabs, a metabolomic analysis was performed by means of a 1H-NMR spectroscopy. From the first to the second trimester, a greater number of women showed a normal lactobacilli-dominated microbiota, with a reduction of cases of dysbiosis. These microbial shifts were associated with profound changes in the vaginal metabolic profiles. Over the weeks, a significant reduction in the levels of BV-associated metabolites (e.g. acetate, propionate, tyramine, methylamine, putrescine) was observed. At the same time, the vaginal metabolome was characterized by higher concentrations of lactate and of several amino acids (e.g. tryptophan, threonine, isoleucine, leucine), typically found in healthy vaginal conditions. Over time, the vaginal metabolome became less diverse and more homogeneous: in the second trimester, women with BV showed metabolic profiles more similar to the healthy/intermediate groups, compared to the first trimester. Our data could help unravel the role of vaginal metabolites in the pathophysiology of pregnancy.

[Source](https://doi.org/10.1371/journal.pone.0249925)

10.1371/journal.pone.0249925

---

## Microbiome profiling of uncinate tissue and nasal polyps in patients with chronic rhinosinusitis using swab and tissue biopsy.
 08.04.2021, _SW Cho, DY Kim, S Choi, S Won, HR Kang, H Yi_


Chronic rhinosinusitis (CRS) is characterized according to the presence or absence of nasal polyps (NPs) and displays nasal microbiota dysbiosis. However, optimal sampling methods of the nasal microbiome in CRS have not been identified. We aimed to assess the microbial composition in patients with CRS, comparing different sampling methods (swab and tissue biopsy), tissue types (uncinate tissue and NP), and disease subtypes. Samples were obtained by swabbing the middle meatus and taking a biopsy of uncinate tissue (UT) in patients with CRS with (CRSwNP, N = 8) or without NP (CRSsNP, N = 6) and controls (N = 8). NPs were also harvested in CRSwNP. DNAs were extracted from fifty-two samples and analyzed by 16S rRNA gene amplicon sequencing. As a result, a great interpersonal variance was observed in nasal swabs, while UT samples presented distinct microbiome with low inter-personal differences. Moreover, the UT microbiomes were further differentiated into three clusters which are associated with disease status (control, CRSsNP, and CRSwNP). Compared to UT, NP revealed a unique microbiome profile with significantly less bacterial diversity. Prevotella was the genus whose abundance was negatively correlated with disease severity in NP. In conclusion, tissue samples are better specimens than nasal swabs for assessing the microbiomes of CRS patients. Several bacteria in UT and NP tissues revealed an association with clinical severity of CRSwNP.

[Source](https://doi.org/10.1371/journal.pone.0249688)

10.1371/journal.pone.0249688

---

## Patient attitudes towards faecal sampling for gut microbiome studies and clinical care reveal positive engagement and room for improvement.
 08.04.2021, _LA Bolte, MAY Klaassen, V Collij, A Vich Vila, J Fu, TA van der Meulen, JJ de Haan, GJ Versteegen, A Dotinga, A Zhernakova, C Wijmenga, RK Weersma, F Imhann_


Faecal sample collection is crucial for gut microbiome research and its clinical applications. However, while patients and healthy volunteers are routinely asked to provide stool samples, their attitudes towards sampling remain largely unknown. Here, we investigate the attitudes of 780 Dutch patients, including participants in a large Inflammatory Bowel Disease (IBD) gut microbiome cohort and population controls, in order to identify barriers to sample collection and provide recommendations for gut microbiome researchers and clinicians. We sent questionnaires to 660 IBD patients and 112 patients with other disorders who had previously been approached to participate in gut microbiome studies. We also conducted 478 brief interviews with participants in our general population cohort who had collected stool samples. Statistical analysis of the data was performed using R. 97.4% of respondents reported that they had willingly participated in stool sample collection for gut microbiome research, and most respondents (82.9%) and interviewees (95.6%) indicated willingness to participate again, with their motivations for participating being mainly altruistic (57.0%). Responses indicated that storing stool samples in the home freezer for a prolonged time was the main barrier to participation (52.6%), but clear explanations of the sampling procedures and their purpose increased participant willingness to collect and freeze samples (P = 0.046, P = 0.003). To account for participant concerns, gut microbiome researchers establishing cohorts and clinicians trying new faecal tests should provide clear instructions, explain the rationale behind their protocol, consider providing a small freezer and inform patients about study outcomes. By assessing the attitudes, motives and barriers surrounding participation in faecal sample collection, we provide important information that will contribute to the success of gut microbiome research and its near-future clinical applications.

[Source](https://doi.org/10.1371/journal.pone.0249405)

10.1371/journal.pone.0249405

---

## Microbiome research in general and business newspapers: How many microbiome articles are published and which study designs make the news the most?
 09.04.2021, _A Prados-Bo, G Casino_


The microbiome is a matter of interest for science, consumers and business. Our objective is to quantify that interest in academic journals and newspapers, both quantitatively and by study design. We calculated the number of articles on the microbiome from the total number of biomedicine articles featured in both PubMed and Spanish science news agency SINC, from 2008 to 2018. We used the Factiva database to identify news stories on microbiome papers in three general newspapers (The New York Times, The Times and El País) and three business newspapers (The Wall Street Journal, the Financial Times and Expansión), from 2007 to 2019. Then, we compared news stories with microbiome papers in PubMed, while also analyzing the frequencies of five study design types, both in the newspapers and in the papers themselves. Microbiome papers represented 0.8% of biomedicine papers in PubMed from 2008 to 2018 (increasing from 0.4% to 1.4%), while microbiome news published by SINC represented 1.6% of total biomedical news stories during the same period (increasing from 0.2% to 2.2%). The number of news stories on microbiome papers correlated with the number of microbiome papers (0.91, p &lt; 0.001) featured in general newspapers, but not in business ones. News stories on microbiome papers represented 78.9% and 42.7% of all microbiome articles in general and business newspapers, respectively. Both media outlet types tended to over-report observational studies in humans while under-reporting environmental studies, while the representation of systematic reviews of randomized controlled trials, randomized controlled trials and animal/laboratory studies was similar when comparing newspapers and PubMed. The microbiome is receiving increasing attention in academic journals and newspapers. News stories on the microbiome in general and business newspapers are mostly based on research findings and are more interested in observational studies in humans and less in environmental studies compared to PubMed.

[Source](https://doi.org/10.1371/journal.pone.0249835)

10.1371/journal.pone.0249835

---

## The ecology of wine fermentation: a model for the study of complex microbial ecosystems

[Source](https://doi.org/10.1007/s00253-021-11270-6)

---

## The rumen eukaryotome is a source of novel antimicrobial peptides with therapeutic potential.
 09.04.2021, _LA Onime, LB Oyama, BJ Thomas, J Gani, P Alexander, KE Waddams, A Cookson, N Fernandez-Fuentes, CJ Creevey, SA Huws_


The rise of microbial antibiotic resistance is a leading threat to the health of the human population. As such, finding new approaches to tackle these microbes, including development of novel antibiotics is vital.

[Source](https://doi.org/10.1186/s12866-021-02172-8)

10.1186/s12866-021-02172-8

---

## The skin microbiota in equine pastern dermatitis: a case-control study of horses in Switzerland.
 08.04.2021, _S Kaiser-Thom, M Hilty, S Axiak, V Gerber_


Equine pastern dermatitis (EPD), a multifactorial syndrome, manifests as skin lesions of variable severity in the pastern area. Despite the widespread use of antibacterial therapy for treating this condition, little is known about the contributing bacteria.

[Source](https://doi.org/10.1111/vde.12955)

10.1111/vde.12955

---

## Association between aging-dependent gut microbiome dysbiosis and dry eye severity in C57BL/6 male mouse model: a pilot study.
 11.04.2021, _CH Yoon, JS Ryu, J Moon, MK Kim_


While aging is a potent risk factor of dry eye disease, age-related gut dysbiosis is associated with inflammation and chronic geriatric diseases. Emerging evidence have demonstrated that gut dysbiosis contributes to the pathophysiology or exacerbation of ocular diseases including dry eye disease. However, the relationship between aging-related changes in gut microbiota and dry eye disease has not been elucidated. In this pilot study, we investigated the association between aging-dependent microbiome changes and dry eye severity in C57BL/6 male mice.

[Source](https://doi.org/10.1186/s12866-021-02173-7)

10.1186/s12866-021-02173-7

---

## Black Yeast Genomes Assembled from Plastic Fabric Metagenomes Reveal an Abundance of Hydrocarbon Degradation Genes.
 09.04.2021, _O Radwan, ON Ruiz_


We report the assembly and annotation of 10 different black yeast genomes from microbiome metagenomic data derived from biofouled plastic fabrics. The draft genomes are estimated to be 9 to 33.2 Mb, with 357 to 5,108 contigs and G+C contents of 43.9% to 57.4%, and they harbor multiple genes for hydrocarbon adaptation and degradation.

[Source](https://doi.org/10.1128/mra.01459-20)

10.1128/MRA.01459-20

---

## Effects of supplemental feeding on the fecal bacterial communities of Rocky Mountain elk in the Greater Yellowstone Ecosystem.
 08.04.2021, _CE Couch, BL Wise, BM Scurlock, JD Rogerson, RK Fuda, EK Cole, KE Szcodronski, AJ Sepulveda, PR Hutchins, PC Cross_


Supplemental feeding of wildlife is a common practice often undertaken for recreational or management purposes, but it may have unintended consequences for animal health. Understanding cryptic effects of diet supplementation on the gut microbiomes of wild mammals is important to inform conservation and management strategies. Multiple laboratory studies have demonstrated the importance of the gut microbiome for extracting and synthesizing nutrients, modulating host immunity, and many other vital host functions, but these relationships can be disrupted by dietary perturbation. The well-described interplay between diet, the microbiome, and host health in laboratory and human systems highlights the need to understand the consequences of supplemental feeding on the microbiomes of free-ranging animal populations. This study describes changes to the gut microbiomes of wild elk under different supplemental feeding regimes. We demonstrated significant cross-sectional variation between elk at different feeding locations and identified several relatively low-abundance bacterial genera that differed between fed versus unfed groups. In addition, we followed four of these populations through mid-season changes in supplemental feeding regimes and demonstrated a significant shift in microbiome composition in a single population that changed from natural forage to supplementation with alfalfa pellets. Some of the taxonomic shifts in this population mirrored changes associated with ruminal acidosis in domestic livestock. We discerned no significant changes in the population that shifted from natural forage to hay supplementation, or in the populations that changed from one type of hay to another. Our results suggest that supplementation with alfalfa pellets alters the native gut microbiome of elk, with potential implications for population health.

[Source](https://doi.org/10.1371/journal.pone.0249521)

10.1371/journal.pone.0249521

---

## A community-supported metaproteomic pipeline for improving peptide identifications in hydrothermal vent microbiota.
 09.04.2021, _Y Chang, Q Fan, J Hou, Y Zhang, J Li_


Microorganisms in deep-sea hydrothermal vents provide valuable insights into life under extreme conditions. Mass spectrometry-based proteomics has been widely used to identify protein expression and function. However, the metaproteomic studies in deep-sea microbiota have been constrained largely by the low identification rates of protein or peptide. To improve the efficiency of metaproteomics for hydrothermal vent microbiota, we firstly constructed a microbial gene database (HVentDB) based on 117 public metagenomic samples from hydrothermal vents and proposed a metaproteomic analysis strategy, which takes the advantages of not only the sample-matched metagenome, but also the metagenomic information released publicly in the community of hydrothermal vents. A two-stage false discovery rate method was followed up to control the risk of false positive. By applying our community-supported strategy to a hydrothermal vent sediment sample, about twice as many peptides were identified when compared with the ways against the sample-matched metagenome or the public reference database. In addition, more enriched and explainable taxonomic and functional profiles were detected by the HVentDB-based approach exclusively, as well as many important proteins involved in methane, amino acid, sugar, glycan metabolism and DNA repair, etc. The new metaproteomic analysis strategy will enhance our understanding of microbiota, including their lifestyles and metabolic capabilities in extreme environments. The database HVentDB is freely accessible from http://lilab.life.sjtu.edu.cn:8080/HventDB/main.html.

[Source](https://doi.org/10.1093/bib/bbab052)

10.1093/bib/bbab052

---

## DisBalance: a platform to automatically build balance-based disease prediction models and discover microbial biomarkers from microbiome data.
 09.04.2021, _F Yang, Q Zou_


How best to utilize the microbial taxonomic abundances in regard to the prediction and explanation of human diseases remains appealing and challenging, and the relative nature of microbiome data necessitates a proper feature selection method to resolve the compositional problem. In this study, we developed an all-in-one platform to address a series of issues in microbiome-based human disease prediction and taxonomic biomarkers discovery. We prioritize the interpretation, runtime and classification accuracy of the distal discriminative balances analysis (DBA-distal) method in selecting a set of distal discriminative balances, and develop DisBalance, a comprehensive platform, to integrate and streamline the workflows of disease model building, disease risk prediction and disease-related biomarker discovery for microbiome-based binary classifications. DisBalance allows the de novo model-building and disease risk prediction in a very fast and convenient way. To facilitate the model-driven and knowledge-driven discoveries, DisBalance dedicates multiple strategies for the mining of microbial biomarkers. The independent validation of the models constructed by the DisBalance pipeline is performed on seven microbiome datasets from the original article of DBA-distal. The implementation of the DisBalance platform is demonstrated by a complete analysis of a shotgun metagenomic dataset of Ulcerative Colitis (UC). As a free and open-source, DisBlance can be accessed at http://lab.malab.cn/soft/DisBalance. The source code and demo data for Disbalance are available at https://github.com/yangfenglong/DisBalance.

[Source](https://doi.org/10.1093/bib/bbab094)

10.1093/bib/bbab094

---

## Metagenomic Data Assembly - The Way of Decoding Unknown Microorganisms.
 10.04.2021, _AL Lapidus, AI Korobeynikov_


Metagenomics is a segment of conventional microbial genomics dedicated to the sequencing and analysis of combined genomic DNA of entire environmental samples. The most critical step of the metagenomic data analysis is the reconstruction of individual genes and genomes of the microorganisms in the communities using metagenomic assemblers - computational programs that put together small fragments of sequenced DNA generated by sequencing instruments. Here, we describe the challenges of metagenomic assembly, a wide spectrum of applications in which metagenomic assemblies were used to better understand the ecology and evolution of microbial ecosystems, and present one of the most efficient microbial assemblers, SPAdes that was upgraded to become applicable for metagenomics.

[Source](https://doi.org/10.3389/fmicb.2021.613791)

10.3389/fmicb.2021.613791

---

## Analyzing indoor mycobiomes through a large-scale citizen science study in Norway.
 08.04.2021, _PM Martin-Sanchez, EF Estensmo, LN Morgado, S Maurice, IB Engh, I Skrede, H Kauserud_


In the built environment, fungi can cause important deterioration of building materials and adverse health effects on occupants. Increased knowledge about indoor mycobiomes from different regions of the world, and their main environmental determinants, will enable improved indoor air quality management and identification of health risks. This is the first citizen science study about indoor mycobiomes at a large geographical scale in Europe, including 271 houses from Norway and 807 dust samples from three house compartments: outside of the building, living room and bathroom. The fungal community composition determined by DNA metabarcoding was clearly different between indoor and outdoor samples, but there were no significant differences between the two indoor compartments. The 32 selected variables, related to the outdoor environment, building features and occupant characteristics, accounted for 15% of the overall variation in community composition, with the house compartment as the key factor (7.6%). Next, the climate was the main driver of the dust mycobiomes (4.2%), while building and occupant variables had significant but minor influences (1.4% and 1.1%, respectively). The house-dust mycobiomes were dominated by ascomycetes (⁓70%) with Capnodiales and Eurotiales as the most abundant orders. Compared to the outdoor samples, the indoor mycobiomes showed higher species richness, which is probably due to the mixture of fungi from outdoor and indoor sources. The main indoor indicator fungi belonged to two ecological groups with allergenic potential: xerophilic molds and skin-associated yeasts. Our results suggest that citizen science is a successful approach for unraveling the built microbiome at large geographical scales.

[Source](https://doi.org/10.1111/mec.15916)

10.1111/mec.15916

---

## Microbiomes in forensic botany: a review.
 08.04.2021, _S Ishak, E Dormontt, JM Young_


Fragments of botanical material can often be found at crime scenes (on live and dead bodies, or on incriminating objects) and can provide circumstantial evidence on various aspects of forensic investigations such as determining crime scene locations, times of death or possession of illegal species. Morphological and genetic analysis are the most commonly applied methods to analyze plant fragment evidence but are limited by their low capacity to differentiate between potential source locations, especially at local scales. Here, we review the current applications and limitations of current plant fragment analysis for forensic investigations and introduce the potential of microbiome analysis to complement the existing forensic plant fragment analysis toolkit. The potential for plant fragment provenance identification at geographic scales meaningful to forensic investigations warrants further investigation of the phyllosphere microbiome in this context. To that end we identify three key areas of future research: 1) Retrieval of microbial DNA of sufficient quality and quantity from botanical material; 2) Variability of the phyllosphere microbiome at different taxonomic and spatial scales, with explicit reference to assignment capacity; 3) Impacts on assignment capacity of time, seasonality and movement of fragments between locations. The development of robust microbiome analysis tools for forensic purposes in botanical material could increase the evidentiary value of the botanical evidence commonly encountered in casework, aiding in the identification of crime scene locations.

[Source](https://doi.org/10.1007/s12024-021-00362-4)

10.1007/s12024-021-00362-4

---

## Untargeted Stable Isotope Probing of the Gut Microbiota Metabolome Using 13C-Labeled Dietary Fibers.
 08.04.2021, _P Deng, T Valentino, MD Flythe, HNB Moseley, JR Leachman, AJ Morris, B Hennig_


The gut microbiome generates numerous metabolites that exert local effects and enter the circulation to affect the functions of many organs. Despite extensive sequencing-based characterization of the gut microbiome, there remains a lack of understanding of microbial metabolism. Here, we developed an untargeted stable isotope-resolved metabolomics (SIRM) approach for the holistic study of gut microbial metabolites. Viable microbial cells were extracted from fresh mice feces and incubated anaerobically with 13C-labeled dietary fibers including inulin or cellulose. High-resolution mass spectrometry was used to monitor 13C enrichment in metabolites associated with glycolysis, the Krebs cycle, the pentose phosphate pathway, nucleotide synthesis, and pyruvate catabolism in both microbial cells and the culture medium. We observed the differential use of inulin and cellulose as substrates for biosynthesis of essential and non-essential amino acids, neurotransmitters, vitamin B5, and other coenzymes. Specifically, the use of inulin for these biosynthetic pathways was markedly more efficient than the use of cellulose, reflecting distinct metabolic pathways of dietary fibers in the gut microbiome, which could be related with host effects. This technology facilitates deeper and holistic insights into the metabolic function of the gut microbiome (Metabolomic Workbench Study ID: ST001651).

[Source](https://pubs.acs.org/doi/10.1021/acs.jproteome.1c00124)

10.1021/acs.jproteome.1c00124

---

## Metaproteomic sample preparation methods bias the recovery of host and microbial proteins according to taxa and cellular compartment.
 08.04.2021, _PG Gavin, J Wong, D Loo, D Zipris, MM Hill, EE Hamilton-Williams_


Faecal proteomics studies have focussed on identification of microbial proteins, however; stool represents a valuable resource to interrogate the host interactions with the microbiota without the need for invasive intestinal biopsies. As the widely used enrichment method (differential centrifugation, DC) enriches for microbial proteins, we compared two other methods for enrichment of host proteins, termed 'host enriched' (HE) and ALL (all proteins). The HE and ALL protocols identified 1.8-fold more host proteins than DC while detecting a similar number of microbial proteins, but the methods had limited overlap in the specific microbial proteins detected. To maximize identification of both host and microbial proteins, samples were subjected to HE and the remaining material was used to perform DC. These two fractions displayed large differences in relative taxonomic abundance and cellular compartmentalization, with proteins from Bacteroidales and extracellular vesicles were enriched in the soluble HE component. The combination of data generated from these two fractions may allow identification of more distinct proteins than simply performing samples in duplicate or more complex fractionation techniques, or a single fraction could be chosen to suit the experimental hypothesis. SIGNIFICANCE: We compared how different stool protein preparation methods influenced the taxonomic and functional characteristics of microbial and host proteins identified. Surprisingly, a method designed to enrich for host proteins recovered a similar number of microbial protein groups to the method that specifically enriched intact bacterial cells. However, the taxonomic and subcellular origin of the microbial proteins differed considerably between the methods. By implementing a two-step method, we could maximize recovery of both host and microbial proteins derived from different cellular compartments and taxa.

[Source](https://doi.org/10.1016/j.jprot.2021.104219)

10.1016/j.jprot.2021.104219

---

