## A 16th century Escherichia coli draft genome associated with an opportunistic bile infection.
 20.06.2022, _GS Long, J Klunk, AT Duggan, M Tapson, V Giuffra, L Gazz√®, A Fornaciari, S Duchene, G Fornaciari, O Clermont, E Denamur, GB Golding, H Poinar_


_Animals, Bile, Escherichia coli, Escherichia coli Infections, Genome, Bacterial, Mice, Opportunistic Infections_
Escherichia coli - one of the most characterized bacteria and a major public health concern - remains invisible across the temporal landscape. Here, we present the meticulous reconstruction of the first ancient E. coli genome from a 16th century gallstone from an Italian mummy with chronic cholecystitis. We isolated ancient DNA and reconstructed the ancient E. coli genome. It consisted of one chromosome of 4446 genes and two putative plasmids with 52 genes. The E. coli strain belonged to the phylogroup A and an exceptionally rare sequence type 4995. The type VI secretion system component genes appears to be horizontally acquired from Klebsiella aerogenes, however we could not identify any pathovar specific genes nor any acquired antibiotic resistances. A sepsis mouse assay showed that a closely related contemporary E. coli strain was avirulent. Our reconstruction of this ancient E. coli helps paint a more complete picture of the burden of opportunistic infections of the past.

[Source](https://www.nature.com/articles/s42003-022-03527-1)

10.1038/s42003-022-03527-1

---

## Strong pathogen competition in neonatal gut colonisation

[Source](https://www.biorxiv.org/content/10.1101/2022.06.19.496579v1.abstract)

---

## Dysbiosis of Skin Microbiota with Increased Fungal Diversity is Associated with Severity of Disease in Atopic Dermatitis

[Source](https://onlinelibrary.wiley.com/doi/10.1111/jdv.18347)

---

## Loss of Novel Diversity in Human Gut Microbiota Associated with Ongoing Urbanization in China

[Source](https://journals.asm.org/doi/10.1128/msystems.00200-22)

---

## A comprehensive analysis of the microbiota composition and host driver gene mutations in colorectal cancer

[Source](https://link.springer.com/article/10.1007/s10637-022-01263-1)

---

## Vegetarianism, Microbiota and Cardiovascular health: Looking back, and forward

[Source](https://academic.oup.com/eurjpc/advance-article-abstract/doi/10.1093/eurjpc/zwac128/6612717)

---

## Effects of a low FODMAP diet on the colonic microbiome in irritable bowel syndrome: a systematic review with meta-analysis

[Source](https://academic.oup.com/ajcn/advance-article/doi/10.1093/ajcn/nqac176/6612725)

---

## Profiling gut microbiota and bile acid metabolism in critically ill children

[Source](https://www.nature.com/articles/s41598-022-13640-0)

---

## Super-taxon in human microbiome are identified to be associated with colorectal cancer

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-04786-9)

---

## The gut metagenomics and metabolomics signature in patients with inflammatory bowel disease

[Source](https://gutpathogens.biomedcentral.com/articles/10.1186/s13099-022-00499-9)

---

## Immune checkpoint inhibitors unleash pathogenic immune responses against the microbiota

[Source](https://www.pnas.org/doi/full/10.1073/pnas.2200348119)

---

## Deletion of both Dectin-1 and Dectin-2 affects the bacterial but not fungal gut microbiota and susceptibility to colitis in mice

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01273-4)

---

## Fecal microbiome of horses transitioning between warm-season and cool-season grass pasture within integrated rotational grazing systems

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-022-00192-x)

---

## Full-Length 16S rRNA Gene Sequences from Raw Sewage Samples Spanning Geographic and Seasonal Gradients in Conveyance Systems across the United States

[Source](https://journals.asm.org/doi/10.1128/mra.00319-22)

---

## SRS-FISH: A high-throughput platform linking microbiome metabolism to identity at the single-cell level.
 21.06.2022, _X Ge, FC Pereira, M Mitteregger, D Berry, M Zhang, B Hausmann, J Zhang, A Schintlmeister, M Wagner, JX Cheng_


One of the biggest challenges in microbiome research in environmental and medical samples is to better understand functional properties of microbial community members at a single-cell level. Single-cell isotope probing has become a key tool for this purpose, but the current detection methods for determination of isotope incorporation into single cells do not allow high-throughput analyses. Here, we report on the development of an imaging-based approach termed stimulated Raman scattering-two-photon fluorescence in situ hybridization (SRS-FISH) for high-throughput metabolism and identity analyses of microbial communities with single-cell resolution. SRS-FISH offers an imaging speed of 10 to 100 ms per cell, which is two to three orders of magnitude faster than achievable by state-of-the-art methods. Using this technique, we delineated metabolic responses of 30,000 individual cells to various mucosal sugars in the human gut microbiome via incorporation of deuterium from heavy water as an activity marker. Application of SRS-FISH to investigate the utilization of host-derived nutrients by two major human gut microbiome taxa revealed that response to mucosal sugars tends to be dominated by Bacteroidales, with an unexpected finding that Clostridia can outperform Bacteroidales at foraging fucose. With high sensitivity and speed, SRS-FISH will enable researchers to probe the fine-scale temporal, spatial, and individual activity patterns of microbial cells in complex communities with unprecedented detail.

[Source](https://www.pnas.org/doi/full/10.1073/pnas.2203519119)

10.1073/pnas.2203519119

---

