## The perfect condition for the rising of superbugs: person-to-person contagion and antibiotic use are the key factors responsible for the positive correlation between antibiotic resistance gene diversity and virulence gene diversity in human metagenomes

[Source](https://www.biorxiv.org/content/10.1101/2020.04.25.061853v3?%3Fcollection=)

---

## Parallel and non-parallel changes of the gut microbiota during trophic diversification in repeated young adaptive radiations of sympatric cichlid fish.
 30.10.2020, _A Härer, J Torres-Dowdall, SJ Rometsch, E Yohannes, G Machado-Schiaffino, A Meyer_


Recent increases in understanding the ecological and evolutionary roles of microbial communities have underscored the importance of their hosts' biology. Yet, little is known about gut microbiota dynamics during the early stages of ecological diversification and speciation. We sequenced the V4 region of the 16s rRNA gene to study the gut microbiota of Nicaraguan Midas cichlid fish (Amphilophus cf. citrinellus). Specifically, we tested the hypothesis that parallel divergence in trophic ecology in extremely young adaptive radiations from two crater lakes is associated with parallel changes of their gut microbiota.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00897-8)

10.1186/s40168-020-00897-8

---

## Exploring the resident gut microbiota of stranded odontocetes: high similarities between two dolphin species <em>Tursiops truncatus</em> and <em>Stenella coeruleoalba</em>

[Source](https://www.cambridge.org/core/journals/journal-of-the-marine-biological-association-of-the-united-kingdom/article/exploring-the-resident-gut-microbiota-of-stranded-odontocetes-high-similarities-between-two-dolphin-species-tursiops-truncatus-and-stenella-coeruleoalba/77EBF3C07CF3D80E992506D65BAB480C)

---

## Investigating the cow skin and teat canal microbiomes of the bovine udder using different sampling and sequencing approaches.
 02.11.2020, _CJ Dean, IB Slizovskiy, KK Crone, AX Pfennig, BJ Heins, LS Caixeta, NR Noyes_


There is a need for standardized, efficient, and practical sampling methods to support large population-based studies of the internal and external epithelial microbiomes of the bovine udder. The primary objective of this study was to evaluate different sampling devices for the isolation of microbial DNA originating from the internal and external teat epithelium. Secondary objectives were to survey and compare the microbial diversity of external and teat canal epithelial microbiomes using amplicon and shotgun metagenomic sequencing approaches. To address these objectives, we enrolled a convenience sample of 24 Holstein dairy cows and collected samples from the external epithelium at the base of udder, the external teat barrel epithelium, the external teat apex epithelium, and the teat canal epithelium. Extracted DNA was quantified and subjected to PCR amplification of the V4 hypervariable region of the 16S rRNA gene and sequenced on the Illumina MiSeq platform (Illumina Inc., San Diego, CA). A subset of samples was subjected to a shallow shotgun metagenomic assay on the Illumina HiSeq platform. For samples collected from the external teat epithelium, we found that gauze squares consistently yielded more DNA than swabs, and Simpson's reciprocal index of diversity was higher for gauze than for swabs. The teat canal epithelial samples exhibited significantly lower diversity than the external sampling locations, but there were no significant differences in diversity between teat apex, teat barrel, and base of the udder samples. There were, however, differences in the microbial distribution and abundances of specific bacteria across external epithelial surfaces. The proportion of shotgun sequence reads classified as Bos taurus was highly variable between sampling locations, ranging from 0.33% in teat apex samples to 99.91% in teat canal samples. These results indicate that gauze squares should be considered for studying the microbiome of the external epithelium of the bovine udder, particularly if DNA yield must be maximized. Further, the relative proportion of host to non-host DNA present in samples collected from the internal and external teat epithelium should be considered when designing studies that utilize shotgun metagenomic sequencing.

[Source](https://www.journalofdairyscience.org/article/S0022-0302(20)30860-2/fulltext)

10.3168/jds.2020-18277

---

## Defining Coral Bleaching as a Microbial Dysbiosis within the Coral Holobiont

[Source](https://www.mdpi.com/2076-2607/8/11/1682)

---

## Free Iron and Iron-Reducing Microorganisms in Permafrost and Permafrost-Affected Soils of Northeastern Siberia

[Source](https://link.springer.com/article/10.1134/S1064229320100166)

---

## Behaviors of cadmium in rhizosphere soils and its interaction with microbiome communities in phytoremediation

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0045653520329635)

---

## Uncovering the genomic potential of the Amazon River microbiome to degrade rainforest organic matter

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00930-w)

---

## Microbiome variability across the native and invasive range of the ascidian <em>Clavelina oblonga</em>

[Source](https://aem.asm.org/content/early/2020/10/26/AEM.02233-20.abstract)

---

## Microbiome of the Black Sea water column analyzed by genome centric metagenomics

[Source](https://www.biorxiv.org/content/10.1101/2020.10.30.362129v1?%3Fcollection=)

---

## Understanding building-occupant-microbiome interactions toward healthy built environments

[Source](https://link.springer.com/article/10.1007/s11783-020-1357-3)

---

## Molecular repertoire of <em>Deinococcus radiodurans</em> after 1 year of exposure outside the International Space Station within the Tanpopo mission

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00927-5)

---

## negative binomial and zero-inflated mixed models, with application to microbiome/metagenomics data analysis

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03803-z)

---

## microbiota—active substance interactions database

[Source](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkaa924/5943835)

---

