## Microbiome sequencing in high-host and low biomass samples

[Source](https://www.nature.com/webcasts/event/microbiome-sequencing-in-high-host-and-low-biomass-samples/)

---

## Webinar Series on the Gut-Brain Axis and the Microbiome

[Source](https://ilsina.org/event/webinar-series-ilsi-north-america-and-division-of-nutritional-sciences-university-of-illinois/)

---

## Functional effects of the human gut microbiome on autism

[Source](https://www.biocodexmicrobiotafoundation.com/international-call-projects-2021)

---

## Focus on Microbiome in Health and Disease – A special microbiome issue in Cell Host Microbe

[Source](https://www.cell.com/cell-host-microbe/issue?pii=S1931-3128(19)X0009-8)

---

## The Impact of the Microbiome on Immunity to Vaccination in Humans

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30355-3)

---

## The Gut Microbiota and Unhealthy Aging: Disentangling Cause from Consequence

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30409-1)

---

## Helminth coinfection and COVID-19: An alternate hypothesis

[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0008628)

---

## Mitochondria, the gut microbiome and ROS. and ROS

[Source](https://www.sciencedirect.com/science/article/abs/pii/S089865682030214X)

---

## Stool Microbiome Profiling of Patients with Metastatic Renal Cell Carcinoma Receiving Anti–PD-1 Immune Checkpoint Inhibitors

[Source](https://www.europeanurology.com/article/S0302-2838(20)30543-1/fulltext)

---

## Programing of an Intravascular Immune Firewall by the Gut Microbiota Protects against Pathogen Dissemination during Infection

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30410-8)

---

## Analysis of microbial changes in the tonsillar formalin-fixed paraffin-embedded tissue of Chinese patients with IgA nephropathy

[Source](https://www.sciencedirect.com/science/article/abs/pii/S034403382032029X)

---

## Breastmilk Feeding Practices Are Associated with the Co-Occurrence of Bacteria in Mothers’ Milk and the Infant Gut: the CHILD Cohort Study

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30350-4)

---

## Effects of Urbanization and Landscape on Gut Microbiomes in White-Crowned Sparrows

[Source](https://link.springer.com/article/10.1007/s00248-020-01569-8)

---

## Passenger-surface microbiome interactions in the subway of Mexico City

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237272)

---

## Plant resistance and leaf chemical characteristic jointly shape phyllosphere bacterial community

[Source](https://link.springer.com/article/10.1007/s11274-020-02908-0)

---

## The role of land management and elevation in shaping soil microbial communities: Insights from the Central European Alps

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0038071720302479)

---

## Composition, predicted functions, and co-occurrence networks of fungal and bacterial communities_ Links to soil organic carbon under long-term fertilization in a rice-wheat cropping system

[Source](https://www.sciencedirect.com/science/article/abs/pii/S116455632030296X)

---

## Diversity and structure of prokaryotic communities within organic and conventional farming systems in central highlands of Kenya

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0236574)

---

## Turning harmful algal biomass to electricity by microbial fuel cell: A sustainable approach for waste management

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0269749120360619)

---

## Combining physicochemical properties and microbiome data to evaluate the water quality of South African drinking water production plants.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237335)

---

## They are what you eat: Shaping of viral populations through nutrition and consequences for virulence

[Source](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1008711)

---

## Assembly methods for nanopore-based metagenomic sequencing: a comparative study.
 18.08.2020, _A Latorre-Pérez, P Villalba-Bermell, J Pascual, C Vilanova_


Metagenomic sequencing has allowed for the recovery of previously unexplored microbial genomes. Whereas short-read sequencing platforms often result in highly fragmented metagenomes, nanopore-based sequencers could lead to more contiguous assemblies due to their potential to generate long reads. Nevertheless, there is a lack of updated and systematic studies evaluating the performance of different assembly tools on nanopore data. In this study, we have benchmarked the ability of different assemblers to reconstruct two different commercially-available mock communities that have been sequenced using Oxford Nanopore Technologies platforms. Among the tested tools, only metaFlye, Raven, and Canu performed well in all the datasets. These tools retrieved highly contiguous genomes (or even complete genomes) directly from the metagenomic data. Despite the intrinsic high error of nanopore sequencing, final assemblies reached high accuracy (~ 99.5 to 99.8% of consensus accuracy). Polishing strategies demonstrated to be necessary for reducing the number of indels, and this had an impact on the prediction of biosynthetic gene clusters. Correction with high quality short reads did not always result in higher quality draft assemblies. Overall, nanopore metagenomic sequencing data-adapted to MinION's current output-proved sufficient for assembling and characterizing low-complexity microbial communities.

[Source](https://www.nature.com/articles/s41598-020-70491-3)

10.1038/s41598-020-70491-3

---

## Completion of draft bacterial genomes by long-read sequencing of synthetic genomic pools.
 05.08.2020, _H Derakhshani, SP Bernier, VA Marko, MG Surette_


Illumina technology currently dominates bacterial genomics due to its high read accuracy and low sequencing cost. However, the incompleteness of draft genomes generated by Illumina reads limits their application in comprehensive genomics analyses. Alternatively, hybrid assembly using both Illumina short reads and long reads generated by single molecule sequencing technologies can enable assembly of complete bacterial genomes, yet the high per-genome cost of long-read sequencing limits the widespread use of this approach in bacterial genomics. Here we developed a protocol for hybrid assembly of complete bacterial genomes using miniaturized multiplexed Illumina sequencing and non-barcoded PacBio sequencing of a synthetic genomic pool (SGP), thus significantly decreasing the overall per-genome cost of sequencing.

[Source](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-06910-6)

10.1186/s12864-020-06910-6

---

## Modeling microbial cross-feeding at intermediate scale portrays community dynamics and species coexistence.
 18.08.2020, _C Liao, T Wang, S Maslov, JB Xavier_


Social interaction between microbes can be described at many levels of details: from the biochemistry of cell-cell interactions to the ecological dynamics of populations. Choosing an appropriate level to model microbial communities without losing generality remains a challenge. Here we show that modeling cross-feeding interactions at an intermediate level between genome-scale metabolic models of individual species and consumer-resource models of ecosystems is suitable to experimental data. We applied our modeling framework to three published examples of multi-strain Escherichia coli communities with increasing complexity: uni-, bi-, and multi-directional cross-feeding of either substitutable metabolic byproducts or essential nutrients. The intermediate-scale model accurately fit empirical data and quantified metabolic exchange rates that are hard to measure experimentally, even for a complex community of 14 amino acid auxotrophies. By studying the conditions of species coexistence, the ecological outcomes of cross-feeding interactions, and each community's robustness to perturbations, we extracted new quantitative insights from these three published experimental datasets. Our analysis provides a foundation to quantify cross-feeding interactions from experimental data, and highlights the importance of metabolic exchanges in the dynamics and stability of microbial communities.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008135)

10.1371/journal.pcbi.1008135

---

## Bacterial modification of the host glycosaminoglycan heparan sulfate modulates SARS-CoV-2 infectivity

[Source](https://www.biorxiv.org/content/10.1101/2020.08.17.238444v1)

---

## Evaluation of Compatibility of 16S rRNA V3V4 and V4 Amplicon Libraries for Clinical Microbiome Profiling

[Source](https://www.biorxiv.org/content/10.1101/2020.08.18.256818v1.abstract)

---

## François-Joseph Lapointe, when biology meets art

[Source](https://www.convergenceinitiative.org/post/françois-joseph-lapointe-when-biology-meets-art)

---

## Early-life microbiota diversification facilitates enterocyte proliferation

[Source](https://naturemicrobiologycommunity.nature.com/posts/early-life-microbiota-diversification-facilitates-enterocyte-proliferation)

---

## ecology and evolution of metabolic cross-feeding interactions in soil-living bacterial communities

[Source](https://twitter.com/KostChristian/status/1295265650525560832)

---

## gut microbiota in protecting the host against intestinal colonisation by multidrug-resistant organisms

[Source](https://twitter.com/JulieAKMcDonald/status/1296364661009453056)

---

