## Amino acid auxotrophies in human gut bacteria are linked to higher microbiome diversity and long-term stability.
 27.10.2023, _S Starke, DMM Harris, J Zimmermann, S Schuchardt, M Oumari, D Frank, C Bang, P Rosenstiel, S Schreiber, N Frey, A Franke, K Aden, S Waschina_


Amino acid auxotrophies are prevalent among bacteria. They can govern ecological dynamics in microbial communities and indicate metabolic cross-feeding interactions among coexisting genotypes. Despite the ecological importance of auxotrophies, their distribution and impact on the diversity and function of the human gut microbiome remain poorly understood. This study performed the first systematic analysis of the distribution of amino acid auxotrophies in the human gut microbiome using a combined metabolomic, metagenomic, and metabolic modeling approach. Results showed that amino acid auxotrophies are ubiquitous in the colon microbiome, with tryptophan auxotrophy being the most common. Auxotrophy frequencies were higher for those amino acids that are also essential to the human host. Moreover, a higher overall abundance of auxotrophies was associated with greater microbiome diversity and stability, and the distribution of auxotrophs was found to be related to the human host's metabolome, including trimethylamine oxide, small aromatic acids, and secondary bile acids. Thus, our results suggest that amino acid auxotrophies are important factors contributing to microbiome ecology and host-microbiome metabolic interactions.

[Source](https://doi.org/10.1038/s41396-023-01537-3)

10.1038/s41396-023-01537-3

---

## Gut microbiota impacts bone via <em>Bacteroides vulgatus</em>-valeric acid-related pathways

[Source](https://doi.org/10.1038/s41467-023-42005-y)

---

## The global burden of antimicrobial resistance - urinary tract infections.
 27.10.2023, _S Von Vietinghoff, O Shevchuk, U Dobrindt, DR Engel, SK Jorch, C Kurts, T Miethke, F Wagenlehner_


Antimicrobial resistance (AMR) emerged as a significant global healthcare problem. Antibiotic use has accelerated the physiologic process of AMR, particularly in Gram-negative pathogens. Urinary tract infections (UTI) are predominantly of Gram-negative nature. Uropathogens are evolutionarily highly adapted and selected strains with specific virulence factors, suggesting common mechanisms how bacterial cells acquire virulence and AMR factors. The simultaneous increase in resistance and virulence is a complex and context-dependent phenomenon. Amongst known AMR mechanisms, the plenitude of different Î²-lactamases is especially prominent. The risk for AMR in UTI varies in different patient populations. History of antibiotic consumption and physiology of urinary flow are major factors that shape AMR prevalence. The urinary tract is in close crosstalk with the microbiome of other compartments, such as the gut or the genital tracts. In addition, pharmacokinetic properties and the physiochemical composition of urinary compartments can contribute to emergence of AMR. Alternatives to antibiotic treatment and a broader approach to address bacterial infections are needed. Among the various alternatives studied, antimicrobial peptides (AMPs) and bacteriophage treatment appear to be highly promising approaches. We here summarize the present knowledge of clinical and microbiological AMR in UTI and discuss innovative approaches, namely new risk prediction tools and the use of non-antibiotic approaches to defend against uropathogenic microbes.

[Source](https://doi.org/10.1093/ndt/gfad233)

10.1093/ndt/gfad233

---

## Class I HLA Allele Predicted Restricted Antigenic Coverages for Fap2 Protein of <em>Fusobacterium nucleatum</em> Are Associated with Colorectal Cancer Incidence

[Source](https://doi.org/10.31557/APJCP.2023.24.10.3629)

---

## Microbial-dependent recruitment of immature myeloid cells promotes intestinal regeneration.
 28.10.2023, _Z Jiang, QT Waterbury, E Malagola, N Fu, W Kim, Y Ochiai, F Wu, C Guha, CJ Shawber, KS Yan, TC Wang_


The intestinal epithelium functions both in nutrient absorption and as a barrier, separating the luminal contents from a network of vascular, fibroblastic, and immune cells underneath. Following injury to the intestine, multiple cell populations cooperate to drive regeneration of the mucosal barrier, including lymphatic endothelial cells (LECs). A population of granulocytic immature myeloid cells (IMCs), marked by histidine decarboxylase (Hdc), participate in regeneration of multiple organs such as the colon and central nervous system, but their contribution to intestinal regeneration was unknown.

[Source](https://doi.org/10.1016/j.jcmgh.2023.10.007)

10.1016/j.jcmgh.2023.10.007

---

## Prior antibiotic administration disrupts anti-PD-1 responses in advanced gastric cancer by altering the gut microbiome and systemic immune response.
 27.10.2023, _CG Kim, JY Koh, SJ Shin, JH Shin, M Hong, HC Chung, SY Rha, HS Kim, CK Lee, JH Lee, Y Han, H Kim, X Che, UJ Yun, H Kim, JH Kim, SY Lee, SK Park, S Park, H Kim, JY Ahn, HC Jeung, JS Lee, YD Nam, M Jung_


Evidence on whether prior antibiotic (pATB) administration modulates outcomes of programmed cell death protein-1 (PD-1) inhibitors in advanced gastric cancer (AGC) is scarce. In this study, we find that pATB administration is consistently associated with poor progression-free survival (PFS) and overall survival (OS) in multiple cohorts consisting of patients with AGC treated with PD-1 inhibitors. In contrast, pATB does not affect outcomes among patients treated with irinotecan. Multivariable analysis of the overall patients treated with PD-1 inhibitors confirms that pATB administration independently predicts worse PFS and OS. Administration of pATBs is associated with diminished gut microbiome diversity, reduced abundance of Lactobacillus gasseri, and disproportional enrichment of circulating exhaustive CD8+ T cells, all of which are associated with worse outcomes. Considering the inferior treatment response and poor survival outcomes by pATB administration followed by PD-1 blockade, ATBs should be prescribed with caution in patients with AGC who are planning to receive PD-1 inhibitors.

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00428-7)

10.1016/j.xcrm.2023.101251

---

## Gut microbiome dynamics in index patients colonized with extended-spectrum beta-lactamase (ESBL)-producing Enterobacterales after hospital discharge and their household contacts.
 27.10.2023, _J Top, TD Verschuuren, MC Viveen, ME Riccio, S Harbarth, JAJW Kluytmans, RJL Willems, FL Paganelli_


This study investigated the gut microbiome dynamics of index patients colonized with extended-spectrum beta-lactamase-producing Escherichia coli (ESBL-Ec) (n = 5) or extended-spectrum beta-lactamase-producing Klebsiella pneumoniae (ESBL-Kp) (n = 3) and their household contacts (n = 9) up to 4 months after hospital discharge of the index patient using 16S rRNA amplicon gene sequencing. Samples were collected at the day of hospital discharge of the index patient, 1 week and 2 and 4 months after discharge. Compared to household contacts, we observed a significant lower alpha diversity (P &lt; 0.001) among index patients and significant (P &lt; 0.05) separation between the two groups for beta diversity. Principal component analysis of the samples from each household (i.e., index patient and respective household contact) showed a clear shift in microbiome composition, in 4/8 index patients, from dissimilar to more similar to the household contact group. This suggests recovery of the microbiome to a healthier status, which was also reflected by de novo colonization of (health-associated) taxa. In contrast, the four time-point samples of the household contacts clustered together indicating a stable microbiome composition over time irrespective of low-level ESBL-Ec (n = 3) or ESBL-Kp (n = 2) colonization. In conclusion, here we show that, at the day of hospital discharge, the microbiome composition of index patients is dissimilar from that of household contacts. Over time, signals of microbiome recovery were observed in half of the index patients. The stable microbiome composition in household contacts irrespective of low-level ESBL-Ec or ESBL-Kp colonization suggests that the gut microbiome in these healthy people provided colonization resistance against ESBL-PE outgrowth. IMPORTANCE Colonization with extended-spectrum beta-lactamase-producing Enterobacterales (ESBL-PE) often precedes infections and is therefore considered as a great threat for public health. Here, we studied the gut microbiome dynamics in eight index patients colonized with ESBL-PE after hospital discharge and the impact of exposure to this index patient on the gut microbiome dynamics of their household contacts. We showed that the microbiome composition from index patients is different from their household contacts upon hospital discharge and that, in some of the index patients, their microbiome composition over time shifted toward the composition of their household contacts. In contrast, household contacts showed a stable microbiome composition over time irrespective of low-level extended-spectrum beta-lactamase-producing Escherichia coli (ESBL-Ec) or extended-spectrum beta-lactamase-producing Klebsiella pneumoniae (ESBL-Kp) gut colonization, suggesting that, in healthy microbiomes, colonization resistance is able to prevent ESBL-PE expansion.

[Source](https://doi.org/10.1128/spectrum.01275-23)

10.1128/spectrum.01275-23

---

## White-nose syndrome restructures bat skin microbiomes.
 27.10.2023, _M Ange-Stark, KL Parise, TL Cheng, JR Hoyt, KE Langwig, WF Frick, AM Kilpatrick, J Gillece, MD MacManes, JT Foster_


The skin microbiome is an essential line of host defense against pathogens, yet our understanding of microbial communities and how they change when hosts become infected is limited. We investigated skin microbial composition in three North American bat species (Myotis lucifugus, Eptesicus fuscus, and Perimyotis subflavus) that have been impacted by the infectious disease, white-nose syndrome, caused by an invasive fungal pathogen, Pseudogymnoascus destructans. We compared bacterial and fungal composition from 154 skin swab samples and 70 environmental samples using a targeted 16S rRNA and internal transcribed spacer amplicon approach. We found that for M. lucifugus, a species that experiences high mortality from white-nose syndrome, bacterial microbiome diversity was dramatically lower when P. destructans was present. Key bacterial families-including those potentially involved in pathogen defense-significantly differed in abundance in bats infected with P. destructans compared to uninfected bats. However, skin bacterial diversity was not lower in E. fuscus or P. subflavus when P. destructans was present despite populations of the latter species declining sharply from white-nose syndrome. The fungal species present on bats substantially overlapped with the fungal taxa present in the environment at the site where the bat was sampled, but fungal community composition was unaffected by the presence of P. destructans for any of the three bat species. This species-specific alteration in bat skin bacterial microbiomes after pathogen invasion may suggest a mechanism for the severity of white-nose syndrome in M. lucifugus but not for other bat species impacted by the disease.IMPORTANCEInherent complexities in the composition of microbiomes can often preclude investigations of microbe-associated diseases. Instead of single organisms being associated with disease, community characteristics may be more relevant. Longitudinal microbiome studies of the same individual bats as pathogens arrive and infect a population are the ideal experiment but remain logistically challenging; therefore, investigations like our approach that are able to correlate invasive pathogens to alterations within a microbiome may be the next best alternative. The results of this study potentially suggest that microbiome-host interactions may determine the likelihood of infection. However, the contrasting relationship between Pd and the bacterial microbiomes of Myotis lucifugus and Perimyotis subflavus indicate that we are just beginning to understand how the bat microbiome interacts with a fungal invader such as Pd.

[Source](https://doi.org/10.1128/spectrum.02715-23)

10.1128/spectrum.02715-23

---

## Linking Clinical Blood Metabogram and Gut Microbiota.
 29.10.2023, _PG Lokhov, EE Balashova, DL Maslov, OP Trifonova, AV Lisitsa, YM Markova, VV Stetsenko, AS Polyanina, SA Sheveleva, KK Sharafetdinov, DB Nikityuk, VA Tutelyan, AI Archakov_


Recently, a clinical blood metabogram was developed as a fast, low-cost and reproducible test that allows the implementation of metabolomics in clinical practice. The components of the metabogram are functionally related groups of blood metabolites associated with humoral regulation, the metabolism of lipids, carbohydrates and amines, lipid intake into the organism, and liver function, thereby providing clinically relevant information. It is known that the gut microbiota affects the blood metabolome, and the components of the blood metabolome may affect the composition of the gut microbiota. Therefore, before using the metabogram in the clinic, the link between the metabogram components and the level of gut microorganisms should be established. For this purpose, the metabogram and microbiota data were obtained in this work for the same individuals. Metabograms of blood plasma were obtained by direct mass spectrometry of blood plasma, and the gut microbiome was determined by a culture-based method and real-time polymerase chain reaction (PCR). This study involved healthy volunteers and individuals with varying degrees of deviation in body weight (n = 44). A correlation analysis determined which metabogram components are linked to which gut microorganisms and the strength of this link. Moreover, diagnostic parameters (sensitivity, specificity and accuracy) confirmed the capacity of metabogram components to be used for diagnosing gut microbiota alterations. Therefore, the obtained results allow the use of the metabogram in a clinical setting, taking into account its relationship with gut microbiota.

[Source](https://doi.org/10.3390/metabo13101095)

10.3390/metabo13101095

---

## A citizen-science-enabled catalogue of the vaginal microbiome and associated factors.
 26.10.2023, _S Lebeer, S Ahannach, T Gehrmann, S Wittouck, T Eilers, E Oerlemans, S Condori, J Dillen, I Spacova, L Vander Donck, C Masquillier, CN Allonsius, PA Bron, W Van Beeck, C De Backer, G Donders, V Verhoeven_


Understanding the composition and function of the vaginal microbiome is crucial for reproductive and overall health. Here we established the Isala citizen-science project to analyse the vaginal microbiomes of 3,345 women in Belgium (18-98 years) through self-sampling, 16S amplicon sequencing and extensive questionnaires. The overall vaginal microbiome composition was strongly tied to age, childbirth and menstrual cycle phase. Lactobacillus species dominated 78% of the vaginal samples. Specific bacterial taxa also showed to co-occur in modules based on network correlation analysis. Notably, the module containing Lactobacillus crispatus, Lactobacillus jensenii and Limosilactobacillus taxa was positively linked to oestrogen levels and contraceptive use and negatively linked to childbirth and breastfeeding. Other modules, named after abundant taxa (Gardnerella, Prevotella and Bacteroides), correlated with multiple partners, menopause, menstrual hygiene and contraceptive use. With this resource-rich vaginal microbiome map and associated health, life-course, lifestyle and dietary factors, we provide unique data and insights for follow-up clinical and mechanistic research.

[Source](https://doi.org/10.1038/s41564-023-01500-0)

10.1038/s41564-023-01500-0

---

## Low-dose ethanol consumption inhibits neutrophil extracellular traps formation to alleviate rheumatoid arthritis.
 30.10.2023, _L Jin, Z Zhang, P Pan, Y Zhao, M Zhou, L Liu, Y Zhai, H Wang, L Xu, D Mei, H Zhang, Y Yang, J Hua, X Zhang, L Zhang_


_Humans, Mice, Animals, Extracellular Traps, Arthritis, Rheumatoid, Neutrophils, Arthritis, Experimental, Acetates_
Rheumatoid arthritis (RA) is a chronic systemic autoimmune disease. Ethanol consumption has been reported to reduce morbidity in RA patients, but the mechanism behind it remains unclear. Our results showed that Muribaculaceae was predominant in the gut microbiota of mice after ethanol treatment, and the levels of microbiota metabolite acetate were increased. Acetate reduced arthritis severity in collagen-induced arthritis (CIA) mice, which was associated with a decrease in the articular neutrophils and the myeloperoxidase-deoxyribonucleic acid complex in serum. Meanwhile, in vitro experiments confirmed that acetate affected neutrophil activity by acting on G-protein-coupled receptor 43, which reduced endoplasmic reticulum stress in neutrophils and inhibited neutrophil extracellular traps formation. Furthermore, exogenous acetate reversed CIA mice with exacerbated gut microbial disruption, further confirming that the effect of gut microbial metabolite acetate on neutrophils in vivo is crucial for the immune regulation. Our findings illuminate the metabolic and cellular mechanisms of the gut-joint axis in the regulation of autoimmune arthritis, and may offer alternative avenues to replicate or induce the joint-protective benefits of ethanol without associated detrimental effects.

[Source](https://doi.org/10.1038/s42003-023-05473-y)

10.1038/s42003-023-05473-y

---

## Community composition and the environment modulate the population dynamics of type VI secretion in human gut bacteria.
 26.10.2023, _S Robitaille, EL Simmons, AJ Verster, EA McClure, DB Royce, E Trus, K Swartz, D Schultz, CD Nadell, BD Ross_


Understanding the relationship between the composition of the human gut microbiota and the ecological forces shaping it is of great importance; however, knowledge of the biogeographical and ecological relationships between physically interacting taxa is limited. Interbacterial antagonism may play an important role in gut community dynamics, yet the conditions under which antagonistic behaviour is favoured or disfavoured by selection in the gut are not well understood. Here, using genomics, we show that a species-specific type VI secretion system (T6SS) repeatedly acquires inactivating mutations in Bacteroides fragilis in the human gut. This result implies a fitness cost to the T6SS, but we could not identify laboratory conditions under which such a cost manifests. Strikingly, experiments in mice illustrate that the T6SS can be favoured or disfavoured in the gut depending on the strains and species in the surrounding community and their susceptibility to T6SS antagonism. We use ecological modelling to explore the conditions that could underlie these results and find that community spatial structure modulates interaction patterns among bacteria, thereby modulating the costs and benefits of T6SS activity. Our findings point towards new integrative models for interrogating the evolutionary dynamics of type VI secretion and other modes of antagonistic interaction in microbiomes.

[Source](https://doi.org/10.1038/s41559-023-02230-6)

10.1038/s41559-023-02230-6

---

## Extension of the <em>Segatella copri</em> complex to 13 species with distinct large extrachromosomal elements and associations with host conditions

[Source](https://doi.org/10.1016/j.chom.2023.09.013)

---

## Gut microbe-metabolite profiles are associated with microbial pathways of longevity in women: a cross-sectional study conducted in China

[Source](https://doi.org/10.1159/000534681)

---

## Longitudinal analysis of vaginal microbiota during IVF fresh embryo transfer and in early pregnancy.
 26.10.2023, _S VÃ¤inÃ¤mÃ¶, S Saqib, I Kalliala, K Kervinen, K Luiro, M NiinimÃ¤ki, M Halttunen-Nieminen, S Virtanen, P Nieminen, A Salonen, T Holster_


Non-Lactobacillus-dominated vaginal microbiota has been associated with poor gynecologic health and complications during pregnancy. Lactobacilli and especially Lactobacillus crispatus associate with good reproductive health and dominate the microbiota during healthy pregnancy. We examined whether the composition of vaginal microbiota at the time of fresh embryo transfer (ET) has an impact on the success of in vitro fertilization (IVF) and whether vaginal microbiota changes from IVF-ET to early pregnancy within individuals. Vaginal swab samples were collected from subfertile women at the time of IVF-ET (n = 76) and at the eighth gestational week (n = 21) from those who achieved clinical pregnancy. The microbiota composition was analyzed using 16S rRNA gene amplicon sequencing. L. crispatus was more abundant among the 30 women who achieved clinical pregnancy (46.9% vs. 19.1%, q = 0.039) and the 26 women who had live birth (43.3% vs. 23.1%, q = 0.32) compared to those who did not. Lactobacilli, mainly L. crispatus (76.2%), dominated all early pregnancy samples. Microbiota remained the same, i.e., Lactobacillus-dominated type in 52% (11/21), shifted from one Lactobacillus-dominated type to another in 24% (5/21), or shifted from mixed community to Lactobacillus-dominated type in 24% (5/21) women, but never from Lactobacillus dominance to non-lactobacilli dominance. Our results emphasize the role of L. crispatus in the success of IVF-ET and in early pregnancy. During pregnancy, the microbiota shifted toward L. crispatus dominance even if it was undetectable before pregnancy, indicating that most women hold a reservoir of this beneficial Lactobacillus in their reproductive tract.IMPORTANCEInfertility is a global public health issue which leads many couples to seek fertility treatments, of which in vitro fertilization (IVF) is considered to be the most effective. Still, only about one-third of the women achieve live birth after the first IVF embryo transfer (IVF-ET). Factors affecting embryo implantation are poorly known, but the female reproductive tract microbiota may play a key role. Our study confirms the beneficial role of vaginal lactobacilli, especially Lactobacillus crispatus, in the probability of achieving clinical pregnancy and live birth following IVF-ET. Our findings regarding the intra-individual shift of vaginal microbiota between non-pregnancy and pregnancy states are novel and provide new information about the dynamics of microbiota in the early steps of human reproduction. These findings may help clinicians in their attempts to optimize the conditions for ET by microbiota screening or modulation and timing the ET when the microbiota is the most favorable.

[Source](https://doi.org/10.1128/spectrum.01650-23)

10.1128/spectrum.01650-23

---

## Machine Learning Algorithms Applied to Predict Autism Spectrum Disorder Based on Gut Microbiome Composition.
 28.10.2023, _JM Olaguez-Gonzalez, I Chairez, L Breton-Deval, M Alfaro-Ponce_


The application of machine learning (ML) techniques stands as a reliable method for aiding in the diagnosis of complex diseases. Recent studies have related the composition of the gut microbiota to the presence of autism spectrum disorder (ASD), but until now, the results have been mostly contradictory. This work proposes using machine learning to study the gut microbiome composition and its role in the early diagnosis of ASD. We applied support vector machines (SVMs), artificial neural networks (ANNs), and random forest (RF) algorithms to classify subjects as neurotypical (NT) or having ASD, using published data on gut microbiome composition. Naive Bayes, k-nearest neighbors, ensemble learning, logistic regression, linear regression, and decision trees were also trained and validated; however, the ones presented showed the best performance and interpretability. All the ML methods were developed using the SAS Viya software platform. The microbiome's composition was determined using 16S rRNA sequencing technology. The application of ML yielded a classification accuracy as high as 90%, with a sensitivity of 96.97% and specificity reaching 85.29%. In the case of the ANN model, no errors occurred when classifying NT subjects from the first dataset, indicating a significant classification outcome compared to traditional tests and data-based approaches. This approach was repeated with two datasets, one from the USA and the other from China, resulting in similar findings. The main predictors in the obtained models differ between the analyzed datasets. The most important predictors identified from the analyzed datasets are Bacteroides, Lachnospira, Anaerobutyricum, and Ruminococcus torques. Notably, among the predictors in each model, there is the presence of bacteria that are usually considered insignificant in the microbiome's composition due to their low relative abundance. This outcome reinforces the conventional understanding of the microbiome's influence on ASD development, where an imbalance in the composition of the microbiota can lead to disrupted host-microbiota homeostasis. Considering that several previous studies focused on the most abundant genera and neglected smaller (and frequently not statistically significant) microbial communities, the impact of such communities has been poorly analyzed. The ML-based models suggest that more research should focus on these less abundant microbes. A novel hypothesis explains the contradictory results in this field and advocates for more in-depth research to be conducted on variables that may not exhibit statistical significance. The obtained results seem to contribute to an explanation of the contradictory findings regarding ASD and its relation with gut microbiota composition. While some research correlates higher ratios of Bacillota/Bacteroidota, others find the opposite. These discrepancies are closely linked to the minority organisms in the microbiome's composition, which may differ between populations but share similar metabolic functions. Therefore, the ratios of Bacillota/Bacteroidota regarding ASD may not be determinants in the manifestation of ASD.

[Source](https://doi.org/10.1186/s13099-023-00575-8)

10.3390/biomedicines11102633

---

## Gut microbiota and autism spectrum disorders: where do we stand?

[Source](https://doi.org/10.1186/s13099-023-00575-8)

---

## Host fecal DNA specific methylation signatures mark gut dysbiosis and inflammation in children affected by autism spectrum disorder.
 27.10.2023, _M Cuomo, L Coretti, D Costabile, R Della Monica, G De Riso, M Buonaiuto, F Trio, C Bravaccio, R Visconti, R Berni Canani, L Chiariotti, F Lembo_


_Humans, Child, Child, Preschool, Autism Spectrum Disorder, Gastrointestinal Microbiome, Dysbiosis, DNA Methylation, Inflammation_
The gut-brain axis involves several bidirectional pathway communications including microbiome, bacterial metabolites, neurotransmitters as well as immune system and is perturbed both in brain and in gastrointestinal disorders. Consistently, microbiota-gut-brain axis has been found altered in autism spectrum disorder (ASD). We reasoned that such alterations occurring in ASD may impact both on methylation signatures of human host fecal DNA (HFD) and possibly on the types of human cells shed in the stools from intestinal tract giving origin to HFD. To test this hypothesis, we have performed whole genome methylation analysis of HFD from an age-restricted cohort of young children with ASD (N = 8) and healthy controls (N = 7). In the same cohort we have previously investigated the fecal microbiota composition and here we refined such analysis and searched for eventual associations with data derived from HFD methylome analysis. Our results showed that specific epigenetic signatures in human fecal DNA, especially at genes related to inflammation, associated with the disease. By applying methylation-based deconvolution algorithm, we found that the HFD derived mainly from immune cells and the relative abundance of those differed between patients and controls. Consistently, most of differentially methylated regions fitted with genes involved in inflammatory response. Interestingly, using Horvath epigenetic clock, we found that ASD affected children showed both epigenetic and microbiota age accelerated. We believe that the present unprecedented approach may be useful for the identification of the ASD associated HFD epigenetic signatures and may be potentially extended to other brain disorders and intestinal inflammatory diseases.

[Source](https://doi.org/10.1038/s41598-023-45132-0)

10.1038/s41598-023-45132-0

---

## Gut microbiome-derived ammonia modulates stress vulnerability in the host.
 23.10.2023, _P Wang, PF Wu, HJ Wang, F Liao, F Wang, JG Chen_


Ammonia has been long recognized as a metabolic waste product with well-known neurotoxic effects. However, little is known about the beneficial function of endogenous ammonia. Here, we show that gut ammonia links microbe nitrogen metabolism to host stress vulnerability by maintaining brain glutamine availability in male mice. Chronic stress decreases blood ammonia levels by altering gut urease-positive microbiota. A representative urease-producing strain, Streptococcus thermophilus, can reverse depression-like behaviours induced by gut microbiota that was altered by stress, whereas pharmacological inhibition of gut ammonia production increases stress vulnerability. Notably, abnormally low blood ammonia levels limit the brain's availability of glutamine, a key metabolite produced by astrocytes that is required for presynaptic Î³-aminobutyric acid (GABA) replenishment and confers stress vulnerability through cortical GABAergic dysfunction. Of therapeutic interest, ammonium chloride (NH4Cl), a commonly used expectorant in the clinic, can rescue behavioural abnormalities and GABAergic deficits in mouse models of depression. In sum, ammonia produced by the gut microbiome can help buffer stress in the host, providing a gut-brain signalling basis for emotional behaviour.

[Source](https://doi.org/10.1038/s42255-023-00909-5)

10.1038/s42255-023-00909-5

---

## Nanobodies against <em>C. difficile</em> TcdA and TcdB reveal unexpected neutralizing epitopes and provide a toolkit for toxin quantitation in vivo

[Source](https://doi.org/10.1371/journal.ppat.1011496)

---

## Microbiome-based therapies for periodontitis and peri-implantitis.
 27.10.2023, _X Wei, S Qian, Y Yang, J Mo_


Periodontitis and peri-implantitis are oral infectious-inflammatory diseases associated with oral microbial dysbiosis. Microbiome-based therapies, characterized by manipulation of the microbiota, are emerging as promising therapeutic approaches to resolve the microbial dysbiosis and associated dysregulation of immune system. This review aims at summarizing recent progress on microbiome-based therapies in periodontitis and peri-implantitis, promoting a further understanding of the related therapeutic mechanisms.

[Source](https://doi.org/10.1111/odi.14782)

10.1111/odi.14782

---

## Distinct gut microbial communities and functional predictions in divergent ophiuroid species: host differentiation, ecological niches, and adaptation to cold-water habitats.
 27.10.2023, _Y Dong, Y Li, M Ge, T Takatsu, Z Wang, X Zhang, D Ding, Q Xu_


Gastrointestinal microorganisms play a crucial role in host survival and adaptation, but information on host-specific selection or environmental factors that shape the microbiome in natural populations is limited. In this study, we employed 16S rRNA gene amplicon sequencing to investigate the composition and predicted the functions of gut microbiota of two ophiuroid species (Ophiura sarsii and its subspecies O. sarsii vadicola) from cold-water habitats across three geographically distant sea areas in the Northern Pacific Ocean. We also explored the potential influence of the host and environment on the microbiota. The two ophiuroids possessed diverse microbial communities, with Proteobacteria being the most dominant phylum in all four groups. Aliivibrio was the predominant genus in O. sarsii from the Bering Sea, while Candidatus Hepatoplasma was the dominant genus in O. sarsii from the Japan Sea and O. sarsii vadicola from the Yellow Sea. Predicted bacterial functions indicated that most of the pathways with significant differences belonged to metabolism and genetic information processing. Notably, no significant difference was observed between the two coexisting ophiuroid species in the Japan Sea. The high similarity in microbial communities between O. sarsii from the Japan Sea and O. sarsii vadicola from the Yellow Sea may be attributed to their analogous ecological niche. The host and the environment jointly shape the composition of the gut microbial community. The presence of specific microorganisms supports the ecological success of ophiuroids across diverse environments, providing a foundation for host adaptation to cold-water habitats.IMPORTANCEGastrointestinal microorganisms are critical to the survival and adaptation of hosts, and there are few studies on the differences and functions of gastrointestinal microbes in widely distributed species. This study investigated the gut microbes of two ophiuroid species (Ophiura sarsii and its subspecies O. sarsii vadicola) in cold-water habitats of the Northern Pacific Ocean. The results showed that a combination of host and environmental factors shapes the intestinal microbiota of ophiuroids. There was a high similarity in microbial communities between the two groups living in different regions, which may be related to their similar ecological niches. These microorganisms played a vital role in the ecological success of ophiuroids as the foundation for their adaptation to cold-water environments. This study revealed the complex relationship between hosts and their gut microbes, providing insights into the role they play in the adaptation and survival of marine species.

[Source](https://doi.org/10.1128/spectrum.02073-23)

10.1128/spectrum.02073-23

---

## Integrated multi-omics analysis reveals the positive leverage of citrus flavonoids on hindgut microbiota and host homeostasis by modulating sphingolipid metabolism in mid-lactation dairy cows consuming a high-starch diet.
 28.10.2023, _Y Zhao, S Yu, H Zhao, L Li, Y Li, M Liu, L Jiang_


_Female, Humans, Cattle, Animals, Starch, Flavonoids, Interleukin-6, Lipopolysaccharides, Multiomics, Tumor Necrosis Factor-alpha, Fermentation, Diet, Lactation, Microbiota, Homeostasis, Sphingolipids, Rumen, Animal Feed_
Modern dairy diets have shifted from being forage-based to grain and energy dense. However, feeding high-starch diets can lead to a metabolic disturbance that is linked to dysregulation of the gastrointestinal microbiome and systemic inflammatory response. Plant flavonoids have recently attracted extensive interest due to their anti-inflammatory effects in humans and ruminants. Here, multi-omics analysis was conducted to characterize the biological function and mechanisms of citrus flavonoids in modulating the hindgut microbiome of dairy cows fed a high-starch diet.

[Source](https://doi.org/10.1186/s40168-023-01661-4)

10.1186/s40168-023-01661-4

---

## Update on novel validly published taxa of bacteria isolated from domestic animals described in 2022.
 27.10.2023, _SD Lawhon, CR Burbick, E Munson, E Thelen, A Zapp, A Wilson_


Expansion of our knowledge of the microbial world continues to progress at a rapid rate and carries with it an associated need for recognizing and understanding the implications of those changes. Here, we describe additions of novel taxa from domestic animals published in 2022 that are validly published per the International Code of Nomenclature of Prokaryotes. These included new members of Staphylococcaceae, Moraxella nasovis sp. nov. in sheep with respiratory disease, three additions to Campylobacteraceae (including one from chickens with spotty liver disease), and multiple additions of organisms from the microbiota of dogs, pigs, and especially honeybees and other important pollinators. Noteworthy additions were associated with diseases of cattle, including mastitis, endocarditis, orchitis, and endometritis. Also described in 2022 was Pseudochrobactrum algeriense sp. nov., a member of the Brucellaceae family, isolated from the mammary lymph nodes of cows.

[Source](https://doi.org/10.1128/jcm.00839-23)

10.1128/jcm.00839-23

---

## Location: root architecture structures rhizosphere microbial associations.
 26.10.2023, _T Galindo-CastaÃ±eda, M Hartmann, JP Lynch_


Root architectural phenotypes are promising targets for crop breeding, but root architectural effects on microbial associations in agricultural fields are not well understood. Architecture determines the location of microbial associations within root systems, which when integrated with soil vertical gradients, determines the functions and the metabolic capability of rhizosphere microbial communities. We argue that variation in root architecture in crops has important implications for root exudation, microbial recruitment and function, and the decomposition and fate of root tissues and exudates. Recent research has shown that the root microbiome changes along root axes and among root classes, that root tips have a unique microbiome, and that root exudates change within the root system depending on soil physicochemical conditions. Although fresh exudates are produced in larger amounts in root tips, the rhizosphere of mature root segments also plays a role in influencing soil vertical gradients. We argue that more research is needed to understand specific root phenotypes that structure microbial associations and discuss candidate root phenotypes that may determine the location of microbial hotspots within root systems with relevance to agricultural systems.

[Source](https://doi.org/10.1093/jxb/erad421)

10.1093/jxb/erad421

---

## Virus diversity and activity is driven by snowmelt and host dynamics in a high-altitude watershed soil ecosystem

[Source](https://doi.org/10.1186/s40168-023-01666-z)

---

## Shotgun metagenomics and computational profiling of the plastisphere microbiome: unveiling the potential of enzymatic production and plastic degradation.
 30.10.2023, _M Saleem, S Yahya, SA Razzak, S Khawaja, A Ali_


_Bacteria, Microbiota, Proteobacteria, Archaea, Metagenome, Metagenomics_
Plastic pollution is one of the most resilient types of pollution and is considered a global environmental threat, particularly in the marine environment. This study aimed to identify plastic-degrading bacteria from the plastisphere and their pharmaceutical and therapeutic potential. We collected samples from soil and aquatic plastisphere to identify the bacterial communities using shotgun metagenomic sequencing and bioinformatic tools. Results showed that the microbiome comprised 93% bacteria, 0.29% archaea, and 3.87% unidentified microbes. Of these 93% of bacteria, 54% were Proteobacteria, 23.9% were Firmicutes, 13% were Actinobacteria, and 2.1% were other phyla. We found that the plastisphere microbiome was involved in degrading synthetic and polyhydroxy alkanoate (PHA) plastic, biosurfactant production, and can thrive under high temperatures. However, no association existed between thermophiles, synthetic plastic or PHA degraders, and biosurfactant-producing bacterial species except for Pseudomonas. Other plastisphere inhabiting plastic degrading microbes include Streptomyces, Bacillus, Achromobacter, Azospirillum, Bacillus, Brevundimonas, Clostridium, Paenibacillus, Rhodococcus, Serratia, Staphylococcus, Thermobifida, and Thermomonospora. However, the plastisphere microbiome showed potential for producing secondary metabolites that were found to act as anticancer, antitumor, anti-inflammatory, antimicrobial, and enzyme stabilizers. These results revealed that the plastisphere microbiome upholds clinical and environmental significance as it can open future portals in a multi-directional way.

[Source](https://doi.org/10.1007/s00203-023-03701-x)

10.1007/s00203-023-03701-x

---

## Machine-learning analysis of cross-study samples according to the gut microbiome in 12 infant cohorts

[Source](https://doi.org/10.1128/msystems.00364-23)

---

## Analyzing microbial evolution through gene and genome phylogenies.
 28.10.2023, _S Teichman, MD Lee, AD Willis_


Microbiome scientists critically need modern tools to explore and analyze microbial evolution. Often this involves studying the evolution of microbial genomes as a whole. However, different genes in a single genome can be subject to different evolutionary pressures, which can result in distinct gene-level evolutionary histories. To address this challenge, we propose to treat estimated gene-level phylogenies as data objects, and present an interactive method for the analysis of a collection of gene phylogenies. We use a local linear approximation of phylogenetic tree space to visualize estimated gene trees as points in low-dimensional Euclidean space, and address important practical limitations of existing related approaches, allowing an intuitive visualization of complex data objects. We demonstrate the utility of our proposed approach through microbial data analyses, including by identifying outlying gene histories in strains of Prevotella, and by contrasting Streptococcus phylogenies estimated using different gene sets. Our method is available as an open-source R package, and assists with estimating, visualizing, and interacting with a collection of bacterial gene phylogenies.

[Source](https://doi.org/10.1093/biostatistics/kxad025)

10.1093/biostatistics/kxad025

---

## AnimalMetaOmics: a multi-omics data resources for exploring animal microbial genomes and microbiomes.
 28.10.2023, _R Hu, F Li, Y Chen, C Liu, J Li, Z Ma, Y Wang, C Cui, C Luo, P Zhou, W Ni, QY Yang, S Hu_


The Animal Meta-omics landscape database (AnimalMetaOmics, https://yanglab.hzau.edu.cn/animalmetaomics#/) is a comprehensive and freely available resource that includes metagenomic, metatranscriptomic, and metaproteomic data from various non-human animal species and provides abundant information on animal microbiomes, including cluster analysis of microbial cognate genes, functional gene annotations, active microbiota composition, gene expression abundance, and microbial protein identification. In this work, 55 898 microbial genomes were annotated from 581 animal species, including 42 924 bacterial genomes, 12 336 virus genomes, 496 archaea genomes and 142 fungi genomes. Moreover, 321 metatranscriptomic datasets were analyzed from 31 animal species and 326 metaproteomic datasets from four animal species, as well as the pan-genomic dynamics and compositional characteristics of 679 bacterial species and 13 archaea species from animal hosts. Researchers can efficiently access and acquire the information of cross-host microbiota through a user-friendly interface, such as species, genomes, activity levels, expressed protein sequences and functions, and pan-genome composition. These valuable resources provide an important reference for better exploring the classification, functional diversity, biological process diversity and functional genes of animal microbiota.

[Source](https://doi.org/10.1093/nar/gkad931)

10.1093/nar/gkad931

---

## SPIRE: a Searchable,&nbsp;Planetary-scale mIcrobiome REsource

[Source](https://doi.org/10.1093/nar/gkad943)

---

## ACR: metagenome-assembled prokaryotic and eukaryotic genome refinement tool.
 30.10.2023, _HJ Seong, JJ Kim, WJ Sul_


_Metagenome, Eukaryota, Microbiota, Algorithms, Metagenomics, Cluster Analysis_
Microbial genome recovery from metagenomes can further explain microbial ecosystem structures, functions and dynamics. Thus, this study developed the Additional Clustering Refiner (ACR) to enhance high-purity prokaryotic and eukaryotic metagenome-assembled genome (MAGs) recovery. ACR refines low-quality MAGs by subjecting them to iterative k-means clustering predicated on contig abundance and increasing bin purity through validated universal marker genes. Synthetic and real-world metagenomic datasets, including short- and long-read sequences, evaluated ACR's effectiveness. The results demonstrated improved MAG purity and a significant increase in high- and medium-quality MAG recovery rates. In addition, ACR seamlessly integrates with various binning algorithms, augmenting their strengths without modifying core features. Furthermore, its multiple sequencing technology compatibilities expand its applicability. By efficiently recovering high-quality prokaryotic and eukaryotic genomes, ACR is a promising tool for deepening our understanding of microbial communities through genome-centric metagenomics.

[Source](https://doi.org/10.1093/bib/bbad381)

10.1093/bib/bbad381

---

## dbAPIS: a database of anti-prokaryotic immune system genes.
 27.10.2023, _Y Yan, J Zheng, X Zhang, Y Yin_


Anti-prokaryotic immune system (APIS) proteins, typically encoded by phages, prophages, and plasmids, inhibit prokaryotic immune systems (e.g. restriction modification, toxin-antitoxin, CRISPR-Cas). A growing number of APIS genes have been characterized and dispersed in the literature. Here we developed dbAPIS (https://bcb.unl.edu/dbAPIS), as the first literature curated data repository for experimentally verified APIS genes and their associated protein families. The key features of dbAPIS include: (i) experimentally verified APIS genes with their protein sequences, functional annotation, PDB or AlphaFold predicted structures, genomic context, sequence and structural homologs from different microbiome/virome databases; (ii) classification of APIS proteins into sequence-based families and construction of hidden Markov models (HMMs); (iii) user-friendly web interface for data browsing by the inhibited immune system types or by the hosts, and functions for searching and batch downloading of pre-computed data; (iv) Inclusion of all types of APIS proteins (except for anti-CRISPRs) that inhibit a variety of prokaryotic defense systems (e.g. RM, TA, CBASS, Thoeris, Gabija). The current release of dbAPIS contains 41 verified APIS proteins and â¼4400 sequence homologs of 92 families and 38 clans. dbAPIS will facilitate the discovery of novel anti-defense genes and genomic islands in phages, by providing a user-friendly data repository and a web resource for an easy homology search against known APIS proteins.

[Source](https://doi.org/10.1093/nar/gkad932)

10.1093/nar/gkad932

---

## Cellsnake: a user-friendly tool for single-cell RNA sequencing analysis.
 30.10.2023, _SU Umu, K Rapp Vander-Elst, VT Karlsen, M Chouliara, ES BÃ¦kkevold, FL Jahnsen, D Domanska_


_Software, Transcriptome, Single-Cell Analysis, Workflow, Sequence Analysis, RNA, Gene Expression Profiling, RNA_
Single-cell RNA sequencing (scRNA-seq) provides high-resolution transcriptome data to understand the heterogeneity of cell populations at the single-cell level. The analysis of scRNA-seq data requires the utilization of numerous computational tools. However, nonexpert users usually experience installation issues, a lack of critical functionality or batch analysis modes, and the steep learning curves of existing pipelines.

[Source](https://doi.org/10.1093/gigascience/giad091)

10.1093/gigascience/giad091

---

## aMeta: an accurate and memory-efficient ancient metagenomic profiling workflow.
 30.10.2023, _Z Pochon, N Bergfeldt, E KÄ±rdÃ¶k, M Vicente, T Naidoo, T van der Valk, NE AltÄ±nÄ±ÅÄ±k, M KrzewiÅska, L DalÃ©n, A GÃ¶therstrÃ¶m, C Mirabello, P Unneberg, N Oskolkov_


_Workflow, Metagenome, Metagenomics, Archaeology, DNA, Ancient_
Analysis of microbial data from archaeological samples is a growing field with great potential for understanding ancient environments, lifestyles, and diseases. However, high error rates have been a challenge in ancient metagenomics, and the availability of computational frameworks that meet the demands of the field is limited. Here, we propose aMeta, an accurate metagenomic profiling workflow for ancient DNA designed to minimize the amount of false discoveries and computer memory requirements. Using simulated data, we benchmark aMeta against a current state-of-the-art workflow and demonstrate its superiority in microbial detection and authentication, as well as substantially lower usage of computer memory.

[Source](https://doi.org/10.1186/s13059-023-03083-9)

10.1186/s13059-023-03083-9

---

