## Microbial determinants of effective donors in faecal microbiota transplantation for UC.
 25.07.2022, _C Haifer, LDW Luu, S Paramsothy, TJ Borody, RW Leong, NO Kaakoush_


Faecal microbiota transplantation (FMT) has variable efficacy in treating UC. Recently, oral lyophilised FMT was found to induce remission in patients with UC, with one donor having 100% efficacy compared with a second donor (36% efficacy). We characterised differences in the gut microbiota of these two donors with the aim of improving FMT donor selection.

[Source](https://doi.org/10.1136/gutjnl-2022-327742)

10.1136/gutjnl-2022-327742

---

## A gut microbial peptide and molecular mimicry in the pathogenesis of type 1 diabetes.
 25.07.2022, _K Girdhar, Q Huang, IT Chow, T Vatanen, C Brady, A Raisingani, P Autissier, MA Atkinson, WW Kwok, CR Kahn, E Altindis_


Type 1 diabetes (T1D) is an autoimmune disease characterized by the destruction of pancreatic β-cells. One of the earliest aspects of this process is the development of autoantibodies and T cells directed at an epitope in the B-chain of insulin (insB:9-23). Analysis of microbial protein sequences with homology to the insB:9-23 sequence revealed 17 peptides showing &gt;50% identity to insB:9-23. Of these 17 peptides, the hprt4-18 peptide, found in the normal human gut commensal Parabacteroides distasonis, activated both human T cell clones from T1D patients and T cell hybridomas from nonobese diabetic (NOD) mice specific to insB:9-23. Immunization of NOD mice with P. distasonis insB:9-23 peptide mimic or insB:9-23 peptide verified immune cross-reactivity. Colonization of female NOD mice with P. distasonis accelerated the development of T1D, increasing macrophages, dendritic cells, and destructive CD8+ T cells, while decreasing FoxP3+ regulatory T cells. Western blot analysis identified P. distasonis-reacting antibodies in sera of NOD mice colonized with P. distasonis and human T1D patients. Furthermore, adoptive transfer of splenocytes from P. distasonis-treated mice to NOD/SCID mice enhanced disease phenotype in the recipients. Finally, analysis of human children gut microbiome data from a longitudinal DIABIMMUNE study revealed that seroconversion rates (i.e., the proportion of individuals developing two or more autoantibodies) were consistently higher in children whose microbiome harbored sequences capable of producing the hprt4-18 peptide compared to individuals who did not harbor it. Taken together, these data demonstrate the potential role of a gut microbiota-derived insB:9-23-mimic peptide as a molecular trigger of T1D pathogenesis.

[Source](https://doi.org/10.1073/pnas.2120028119)

10.1073/pnas.2120028119

---

## Normal gut microbiome in NMDA receptor encephalitis.
 01.10.2020, _J Herken, C Bang, MC Rühlemann, C Finke, J Klag, A Franke, H Prüss_


_Acute Disease, Adult, Anti-N-Methyl-D-Aspartate Receptor Encephalitis, Feces, Female, Gastrointestinal Microbiome, Humans, Male, Middle Aged_
To determine whether the gut microbiota shows overabundance of commensal bacteria species in patients with anti-NMDA receptor (NMDAR) encephalitis, similar to patients with MS or neuromyelitis optica where they potentially balance pro- and anti-inflammatory immune responses or participate in disease pathogenesis by molecular mimicry.

[Source](https://doi.org/10.1186/s12883-022-02804-0)

10.1212/NXI.0000000000000632

---

## Gut microbiome changes in anti-N-methyl-D-aspartate receptor encephalitis patients

[Source](https://doi.org/10.1186/s12883-022-02804-0)

---

## Effect of a bacteriocin-producing<em> Streptococcus salivarius</em> on the pathogen <em>Fusobacterium nucleatum</em> in a model of the human distal colon

[Source](https://doi.org/10.1080/19490976.2022.2100203)

---

## Identification of Gut Microbiome Metabolites via Network Pharmacology Analysis in Treating Alcoholic Liver Disease.
 26.07.2022, _KK Oh, YR Choi, H Gupta, R Ganesan, SP Sharma, SM Won, JJ Jeong, SB Lee, MG Cha, GH Kwon, DJ Kim, KT Suk_


Alcoholic liver disease (ALD) is linked to a broad spectrum of diseases, including diabetes, hypertension, atherosclerosis, and even liver carcinoma. The ALD spectrum includes alcoholic fatty liver disease (AFLD), alcoholic hepatitis, and cirrhosis. Most recently, some reports demonstrated that the pathogenesis of ALD is strongly associated with metabolites of human microbiota. AFLD was the onset of disease among ALDs, the initial cause of which is alcohol consumption. Thus, we analyzed the significant metabolites of microbiota against AFLD via the network pharmacology concept. The metabolites from microbiota were retrieved by the gutMGene database; sequentially, AFLD targets were identified by public databases (DisGeNET, OMIM). The final targets were utilized for protein-protein interaction (PPI) networks and signaling pathway analyses. Then, we performed a molecular docking test (MDT) to verify the affinity between metabolite(s) and target(s) utilizing the Autodock 1.5.6 tool. From a holistic viewpoint, we integrated the relationships of microbiota-signaling pathways-targets-metabolites (MSTM) using the R Package. We identified the uppermost six key targets (TLR4, RELA, IL6, PPARG, COX-2, and CYP1A2) against AFLD. The PPI network analysis revealed that TLR4, RELA, IL6, PPARG, and COX-2 had equivalent degrees of value (4); however, CYP1A2 had no associations with the other targets. The bubble chart showed that the PI3K-Akt signaling pathway in nine signaling pathways might be the most significant mechanism with antagonistic functions in the treatment of AFLD. The MDT confirmed that Icaritin is a promising agent to bind stably to RELA (known as NF-Κb). In parallel, Bacterium MRG-PMF-1, the PI3K-Akt signaling pathway, RELA, and Icaritin were the most significant components against AFLD in MSTM networks. In conclusion, we showed that the Icaritin-RELA complex on the PI3K-Akt signaling pathway by bacterial MRG-PMF-1 might have promising therapeutic effects against AFLD, providing crucial evidence for further research.

[Source](https://doi.org/10.3390/cimb44070224)

10.3390/cimb44070224

---

## The Gut Microbiota: A Double Edge Sword in Endometriosis.
 25.07.2022, _C Talwar, V Singh, R Kommagani_


Endometriosis that afflicts 1 in 10 women of reproductive age is characterized by growth of endometrial tissue in the extra-uterine sites and encompasses metabolic-, immunologic- and endocrine-disruption. Importantly, several comorbidities are associated with endometriosis, especially autoimmune disorders such as inflammatory bowel disease. Primarily thought of as a condition arising from retrograde menstruation, emerging evidence uncovered a functional link between the gut microbiota and endometriosis. Specifically, recent findings revealed altered gut microbiota profiles in endometriosis and in turn this altered microbiota appears to be causal in the disease progression, implying a bi-directional crosstalk. In this review, we discuss the complex etiology and pathogenesis of endometriosis emphasizing on this recently recognized role of gut microbiome. We review the gut microbiome structure and functions and its complex network of interactions with the host for maintenance of homeostasis that is crucial for disease prevention. We highlight the underlying mechanisms on how some bacteria promotes disease progression and others protects against endometriosis. Further, we highlight the areas that require future emphases in the gut microbiome-endometriosis nexus and the potential microbiome-based therapies for amelioration of endometriosis.

[Source](https://doi.org/10.1093/biolre/ioac147)

10.1093/biolre/ioac147

---

## Hair Microbiome Diversity within and across Primate Species.
 25.07.2022, _C Kitrinos, RB Bell, BJ Bradley, JM Kamilar_


Primate hair and skin are substrates upon which social interactions occur and are host-pathogen interfaces. While human hair and skin microbiomes display body site specificity and immunological significance, little is known about the nonhuman primate (NHP) hair microbiome. Here, we collected hair samples (n = 158) from 8 body sites across 12 NHP species housed at three zoological institutions in the United States to examine the following: (1) the diversity and composition of the primate hair microbiome and (2) the factors predicting primate hair microbiome diversity and composition. If both environmental and evolutionary factors shape the microbiome, then we expect significant differences in microbiome diversity across host body sites, sexes, institutions, and species. We found our samples contained high abundances of gut-, respiratory-, and environment-associated microbiota. In addition, multiple factors predicted microbiome diversity and composition, although host species identity outweighed sex, body site, and institution as the strongest predictor. Our results suggest that hair microbial communities are affected by both evolutionary and environmental factors and are relatively similar across nonhuman primate body sites, which differs from the human condition. These findings have important implications for understanding the biology and conservation of wild and captive primates and the uniqueness of the human microbiome. IMPORTANCE We created the most comprehensive primate hair and skin data set to date, including data from 12 nonhuman primate species sampled from 8 body regions each. We find that the nonhuman primate hair microbiome is distinct from the human hair and skin microbiomes in that it is relatively uniform-as opposed to distinct-across body regions and is most abundant in gut-, environment-, and respiratory-associated microbiota rather than human skin-associated microbiota. Furthermore, we found that the nonhuman primate hair microbiome varies with host species identity, host sex, host environment, and host body site, with host species identity being the strongest predictor. This result demonstrates that nonhuman primate hair microbiome diversity varies with both evolutionary and environmental factors and within and across primate species. These findings have important implications for understanding the biology and conservation of wild and captive primates and the uniqueness of the human microbiome.

[Source](https://doi.org/10.1128/msystems.00478-22)

10.1128/msystems.00478-22

---

## Seasonal variations in the composition and diversity of gut microbiota in white-lipped deer (<em>Cervus albirostris</em>)

[Source](https://doi.org/10.7717/peerj.13753)

---

## Effects of Domestication on Plant-Microbiome Interactions.
 25.07.2022, _A Gutierrez, MA Grillo_


Through the process of domestication selection is targeted on a limited number of plant traits that are typically associated with yield. As an unintended consequence, domesticated plants often perform poorly compared to their wild progenitors for a multitude of traits that were not under selection during domestication, including abiotic and biotic stress tolerance. Over the past decade advances in sequencing technology have allowed for the rigorous characterization of host-associated microbial communities, termed the microbiome. It is now clear that nearly every conceivable plant interaction with the environment is mediated by interactions with the microbiome. For this reason, plant-microbiome interactions are an area of great promise for plant breeding and crop improvement. Here we review the literature to assess the potential impact that domestication has had on plant microbiome interactions and the current understanding of the genetic basis of microbiome variation to inform plant breeding efforts. Overall, we find limited evidence that domestication impacts the diversity of microbiomes, but domestication is often associated with shifts in the abundance and composition of microbial communities, including taxa of known functional significance. Moreover, GWAS and mutant analysis has not revealed a consistent set of core candidate genes or genetic pathways that confer variation in microbiomes across systems. However, such studies do implicate a consistent role for plant immunity, root traits, root and leaf exudates, and cell wall integrity as key traits that control microbiome colonization and assembly. Therefore, selection on these key traits may pose the most immediate promise for enhancing plant-microbiome interactions through breeding.

[Source](https://doi.org/10.1093/pcp/pcac108)

10.1093/pcp/pcac108

---

## Evaluation of protein extraction methods for metaproteomic analyses of root-associated microbes.
 25.07.2022, _F Salvato, S Vintila, OM Finkel, J Dangl, M Kleiner_


Metaproteomics is a powerful tool for the characterization of metabolism, physiology and functional interactions in microbial communities, including plant-associated microbiota. However, the metaproteomic methods that have been used to study plant-associated microbiota are very laborious and require large amounts of plant tissue; hindering, wider application of these methods. We optimized and evaluated different protein extraction methods for metaproteomics of plant-associated microbiota in two different plant species (Arabidopsis and maize). Our main goal was to identify a method that would work with low amounts of input material (40-70 mg) and that would maximize the number of identified microbial proteins. We tested eight protocols, each comprising a different combination of physical lysis method, extraction buffer, and cell enrichment method on roots from plants grown with synthetic microbial communities. We assessed the performance of the extraction protocols by LC-MS/MS-based metaproteomics and found that the optimal extraction method differed between the two species. For Arabidopsis roots protein extraction by beating whole roots with small beads provided the highest number of identified microbial proteins and improved the identification of proteins from gram-positive bacteria. For maize, vortexing root pieces in the presence of large glass beads yielded the highest number of microbial proteins identified. Based on these data we recommend the use of these two methods for metaproteomics with Arabidopsis and maize. Furthermore, detailed descriptions of the eight tested protocols will enable future optimization of protein extraction for metaproteomics in other dicot and monocot plants.

[Source](https://doi.org/10.1094/MPMI-05-22-0116-TA)

10.1094/MPMI-05-22-0116-TA

---

## Absence of stress-promoted facilitation coupled with a competition decrease in the microbiome of ephemeral saline lakes.
 25.07.2022, _M Menéndez-Serra, VJ Ontiveros, A Barberán, EO Casamayor_


Salinity fluctuations constitute a well-known high stress factor strongly shaping global biological distributions and abundances. However, there is a knowledge gap regarding how increasing saline stress affects microbial biological interactions. We applied the combination of a probabilistic method for estimating significant co-occurrences/exclusions and a conceptual framework for filtering out associations potentially linked to environmental and/or spatial factors, in a series of connected ephemeral (hyper) saline lakes. We carried out a network analysis over the full aquatic microbiome -bacteria, eukarya and archaea- under severe salinity fluctuations. Most of the observed co-occurrences/exclusions were potentially explained by environmental niche and/or dispersal limitation. Co-occurrences assigned to potential biological interactions remained stable, suggesting that the salt gradient was not promoting interspecific facilitation processes. Conversely, co-exclusions assigned to potential biological interactions decreased along the gradient both in number and network complexity, pointing to a decrease of interspecies competition as salinity increased. Overall, higher saline stress reduced microbial co-exclusions while co-occurrences remained stable suggesting decreasing competition coupled with lack of stress-gradient promoted facilitation in the microbiome of ephemeral saline lakes.

[Source](https://doi.org/10.1002/ecy.3834)

10.1002/ecy.3834

---

## Insights into the Antimicrobial Activities and Metabolomes of <em>Aquimarina</em> (Flavobacteriaceae, Bacteroidetes) Species from the Rare Marine Biosphere

[Source](https://doi.org/10.3390/md20070423)

---

## International travel, the gut microbiome, and ESBL-E coli carriage.
 25.07.2022, _Y Peng, DZY Sin, HM Tun_



[Source](https://doi.org/10.1016/S2666-5247(22)00201-4)

10.1016/S2666-5247(22)00201-4

---

## International travel, the gut microbiome, and ESBL-<em>E coli </em>carriage

[Source](https://doi.org/10.1016/S2666-5247(22)00201-4)

---

## An Engineered λ Phage Enables Enhanced and Strain-Specific Killing of Enterohemorrhagic <em>Escherichia coli</em>

[Source](https://doi.org/10.1128/spectrum.01271-22)

---

## MOCHI, a comprehensive cross-platform tool for amplicon-based microbiota analysis.
 25.07.2022, _JJ Zheng, PW Wang, TW Huang, YJ Yang, HS Chiu, P Sumazin, TW Chen_


Microbiota analyses have important implications for health and science. These analyses make use of 16S/18S rRNA gene sequencing to identify taxa and predict species diversity. However, most available tools for analyzing microbiota data require adept programming skills and in-depth statistical knowledge for proper implementation. While long-read amplicon sequencing can lead to more accurate taxa predictions and is quickly becoming more common, practitioners have no easily accessible tools with which to perform their analyses.

[Source](https://doi.org/10.1093/bioinformatics/btac494)

10.1093/bioinformatics/btac494

---

