## Human cytomegalovirus in breast milk is associated with milk composition and the infant gut microbiome and growth.
 26.07.2024, _KE Johnson, N Hernandez-Alvarado, M Blackstad, T Heisel, M Allert, DA Fields, E Isganaitis, KM Jacobs, D Knights, EF Lock, MC Rudolph, CA Gale, MR Schleiss, FW Albert, EW Demerath, R Blekhman_


_Humans, Milk, Human, Gastrointestinal Microbiome, Female, Cytomegalovirus, Cytomegalovirus Infections, Infant, Infant, Newborn, Kynurenine, Breast Feeding, Viral Load, Male, Adult, Indoleamine-Pyrrole 2,3,-Dioxygenase, Tryptophan, Metabolome_
Human cytomegalovirus (CMV) is a highly prevalent herpesvirus that is often transmitted to the neonate via breast milk. Postnatal CMV transmission can have negative health consequences for preterm and immunocompromised infants, but any effects on healthy term infants are thought to be benign. Furthermore, the impact of CMV on the composition of the hundreds of bioactive factors in human milk has not been tested. Here, we utilize a cohort of exclusively breastfeeding full-term mother-infant pairs to test for differences in the milk transcriptome and metabolome associated with CMV, and the impact of CMV in breast milk on the infant gut microbiome and infant growth. We find upregulation of the indoleamine 2,3-dioxygenase (IDO) tryptophan-to-kynurenine metabolic pathway in CMV+ milk samples, and that CMV+ milk is associated with decreased Bifidobacterium in the infant gut. Our data indicate two opposing CMV-associated effects on infant growth; with kynurenine positively correlated, and CMV viral load negatively correlated, with infant weight-for-length at 1 month of age. These results suggest CMV transmission, CMV-related changes in milk composition, or both may be modulators of full-term infant development.

[Source](https://www.nature.com/articles/s41467-024-50282-4)

10.1038/s41467-024-50282-4

---

## First-Year Dynamics of the Anaerobic Microbiome and Archaeome in Infants Oral and Gastrointestinal Systems

[Source](https://www.biorxiv.org/content/10.1101/2024.07.24.604926v1)

---

## A Mendelian randomization analysis reveals the multifaceted role of the skin microbiota in liver cancer

[Source](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2024.1422132/full)

---

## Metagenomic sequencing of the skin microbiota of the scalp predicting the risk of surgical site infections following surgery of traumatic brain injury in sub-Saharan Africa.
 24.07.2024, _HM Lekuya, DP Kateete, G Olweny, E Kigozi, LK Kamabu, SP Mudekereza, R Nantambi, R Mbiine, F Makumbi, S Cose, J Vandersteene, E Baert, JO Kalala, M Galukande_


_Humans, Male, Surgical Wound Infection, Female, Scalp, Adult, Microbiota, Metagenomics, Brain Injuries, Traumatic, Prospective Studies, Africa South of the Sahara, Skin, Young Adult, Adolescent, RNA, Ribosomal, 16S, Uganda, Middle Aged, Risk Factors, Metagenome_
Surgical site infections (SSI) are a significant concern following traumatic brain injury (TBI) surgery and often stem from the skin's microbiota near the surgical site, allowing bacteria to penetrate deeper layers and potentially causing severe infections in the cranial cavity. This study investigated the relationship between scalp skin microbiota composition and the risk of SSI after TBI surgery in sub-Saharan Africa (SSA).

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0303483)

10.1371/journal.pone.0303483

---

## The interplay between diet and the gut microbiome: implications for health and disease.
 15.07.2024, _FC Ross, D Patangia, G Grimaud, A Lavelle, EM Dempsey, RP Ross, C Stanton_


Diet has a pivotal role in shaping the composition, function and diversity of the gut microbiome, with various diets having a profound impact on the stability, functionality and diversity of the microbial community within our gut. Understanding the profound impact of varied diets on the microbiome is crucial, as it will enable us not only to make well-informed dietary decisions for better metabolic and intestinal health, but also to prevent and slow the onset of specific diet-related diseases that stem from suboptimal diets. In this Review, we explore how geographical location affects the gut microbiome and how different diets shape its composition and function. We examine the mechanisms by which whole dietary regimes, such as the Mediterranean diet, high-fibre diet, plant-based diet, high-protein diet, ketogenic diet and Western diet, influence the gut microbiome. Furthermore, we underscore the need for exhaustive studies to better understand the causal relationship between diet, host and microorganisms for the development of precision nutrition and microbiome-based therapies.

[Source](https://www.nature.com/articles/s41579-024-01068-4)

10.1038/s41579-024-01068-4

---

## Microbiota composition effect on immunotherapy outcomes in colorectal cancer patients: A systematic review.
 25.07.2024, _SM Ajab, SH Zoughbor, LA Labania, LM Ã–stlundh, HS Orsud, MA Olanda, O Alkaabi, SH Alkuwaiti, SM Alnuaimi, Z Al Rasbi_


_Humans, Colorectal Neoplasms, Gastrointestinal Microbiome, Immunotherapy, Immune Checkpoint Inhibitors, Treatment Outcome, Feces_
Immune checkpoint inhibitors (ICIs) have emerged as an effective treatment for colorectal cancer (CRC). Studies indicate that the composition of gut microbiota could potentially serve as a biomarker for predicting the clinical effectiveness of immune checkpoint inhibitors.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0307639)

10.1371/journal.pone.0307639

---

## Altitude-dependent agro-ecologies impact the microbiome diversity of scavenging indigenous chicken in Ethiopia.
 26.07.2024, _L Glendinning, X Jia, A Kebede, SO Oyola, JE Park, W Park, A Assiri, JB Holm, K Kristiansen, J Han, O Hanotte_


_Animals, Chickens, Ethiopia, Altitude, Bacteria, Gastrointestinal Microbiome, Metagenome, Cecum, Microbiota, Biodiversity, Phylogeny_
Scavenging indigenous village chickens play a vital role in sub-Saharan Africa, sustaining the livelihood of millions of farmers. These chickens are exposed to vastly different environments and feeds compared to commercial chickens. In this study, we analysed the caecal microbiota of 243 Ethiopian village chickens living in different altitude-dependent agro-ecologies.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01847-4)

10.1186/s40168-024-01847-4

---

## Effect of soybean and seaweed-based diets on growth performance, feed utilization, and gut microbiota of tilapia: A systematic review and meta-analysis.
 25.07.2024, _LDA Viagem, JN Hakizimana, C Rumisha, BDS Cerozi, G Misinzo_


_Animals, Seaweed, Gastrointestinal Microbiome, Animal Feed, Glycine max, Tilapia, Aquaculture, Diet_
Tilapia, a significant aquaculture species globally, relies heavily on feed for its production. While numerous studies have investigated the impact of soybean and seaweed-based diets on tilapia, a comprehensive understanding remains elusive. This review aimed at evaluating and synthesizing the existing literature on these diets' effects, focusing on growth performance, feed utilization, and gut microbiota. A systematic search of databases was conducted using Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines and a total of 57 studies were included in the qualitative analysis and 24 in the meta-analysis. The results indicated that soybean-based diets, at a 59.4% inclusion level improved the Specific Growth Rate (SGR) of tilapia with an effect size of -2.14 (95% CI: -2.92, -1.37; p &lt; 0.00001; I2 = 99%) and did not improve the feed conversion rate (FCR), as the effect size was 1.80 (95% CI: 0.72, 2.89; p = 0.001; I2 = 100%). For seaweed-based diets, at a 15,9% inclusion level did not improve SGR, with an effect size of -0.74 (95% CI: -1.70, 0.22; p = 0.13; I2 = 99%), and the FCR with an effect size of -0.70 (95% CI: -1.94, 0.54; p = 0.27; I2 = 100%). Regarding the gut microbiota, was noted a lack of studies meeting the inclusion criteria for tilapia. However, findings from studies on other farmed fishes suggested that soybean and seaweed-based diets could have diverse effects on gut microbiota composition and promote the growth of beneficial microbiota. This study suggests that incorporating soybean-based diets at 59.4% inclusion can improve the SGR of tilapia. Seaweed-based diets, while not demonstrating improvement in the analyzed parameters with an inclusion level of 15.9%, have the potential to contribute to the sustainability of the aquaculture industry when incorporated at lower levels.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0293775)

10.1371/journal.pone.0293775

---

## Plant species influences the composition of root system microbiome and its antibiotic resistance profile in a constructed wetland receiving primary treated wastewater

[Source](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2024.1436122/full)

---

## Endogenous retroviruses mediate transcriptional rewiring in response to oncogenic signaling in colorectal cancer.
 19.07.2024, _A Ivancevic, DM Simpson, OM Joyner, SM Bagby, LL Nguyen, BG Bitler, TM Pitts, EB Chuong_


_Humans, Colorectal Neoplasms, Endogenous Retroviruses, Gene Expression Regulation, Neoplastic, Signal Transduction, Enhancer Elements, Genetic, Cell Line, Tumor, Transcription, Genetic, Animals, Carcinogenesis, Gene Regulatory Networks_
Cancer cells exhibit rewired transcriptional regulatory networks that promote tumor growth and survival. However, the mechanisms underlying the formation of these pathological networks remain poorly understood. Through a pan-cancer epigenomic analysis, we found that primate-specific endogenous retroviruses (ERVs) are a rich source of enhancers displaying cancer-specific activity. In colorectal cancer and other epithelial tumors, oncogenic MAPK/AP1 signaling drives the activation of enhancers derived from the primate-specific ERV family LTR10. Functional studies in colorectal cancer cells revealed that LTR10 elements regulate tumor-specific expression of multiple genes associated with tumorigenesis, such as ATG12 and XRCC4. Within the human population, individual LTR10 elements exhibit germline and somatic structural variation resulting from a highly mutable internal tandem repeat region, which affects AP1 binding activity. Our findings reveal that ERV-derived enhancers contribute to transcriptional dysregulation in response to oncogenic signaling and shape the evolution of cancer-specific regulatory networks.

[Source](https://www.science.org/doi/10.1126/sciadv.ado1218)

10.1126/sciadv.ado1218

---

## The plasma viral communities associate with clinical profiles in a large-scale haematological patients cohort

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01855-4)

---

## Viruses of free-roaming and hunting dogs in Uganda show elevated prevalence, richness and abundance across a gradient of contact with wildlife.
 24.07.2024, _DS Tayebwa, D Hyeroba, CD Dunn, E Dunay, JC Richard, S Biryomumaisho, JO Acai, TL Goldberg_


_Animals, Dogs, Uganda, Dog Diseases, Prevalence, Animals, Wild, Virome, Viruses, Metagenomics, Anelloviridae, Humans, Virus Diseases_
Domestic dogs (Canis lupus familiaris) live with humans, frequently contact other animals and may serve as intermediary hosts for the transmission of viruses. Free-roaming dogs, which account for over 70% of the world's domestic dog population, may pose a particularly high risk in this regard. We conducted an epidemiological study of dog viromes in three locations in Uganda, representing low, medium and high rates of contact with wildlife, ranging from dogs owned specifically for traditional hunting in a biodiversity and disease 'hotspot' to pets in an affluent suburb. We quantified rates of contact between dogs and wildlife through owner interviews and conducted canine veterinary health assessments. We then applied broad-spectrum viral metagenomics to blood plasma samples, from which we identified 46 viruses, 44 of which were previously undescribed, in three viral families, Sedoreoviridae, Parvoviridae and Anelloviridae. All 46 viruses (100 %) occurred in the high-contact population of dogs compared to 63 % and 39 % in the medium- and low-contact populations, respectively. Viral prevalence ranged from 2.1 % to 92.0 % among viruses and was highest, on average, in the high-contact population (22.3 %), followed by the medium-contact (12.3 %) and low-contact (4.8 %) populations. Viral richness (number of viruses per dog) ranged from 0 to 27 and was markedly higher, on average, in the high-contact population (10.2) than in the medium-contact (5.7) or low-contact (2.3) populations. Viral richness was strongly positively correlated with the number of times per year that a dog was fed wildlife and negatively correlated with the body condition score, body temperature and packed cell volume. Viral abundance (cumulative normalized metagenomic read density) varied 124-fold among dogs and was, on average, 4.1-fold higher and 2.4-fold higher in the high-contact population of dogs than in the low-contact or medium-contact populations, respectively. Viral abundance was also strongly positively correlated with the number of times per year that a dog was fed wildlife, negatively correlated with packed cell volume and positively correlated with white blood cell count. These trends were driven by nine viruses in the family Anelloviridae, genus Thetatorquevirus, and by one novel virus in the family Sedoreoviridae, genus Orbivirus. The genus Orbivirus contains zoonotic viruses and viruses that dogs can acquire through ingestion of infected meat. Overall, our findings show that viral prevalence, richness and abundance increased across a gradient of contact between dogs and wildlife and that the health status of the dog modified viral infection. Other ecological, geographic and social factors may also have contributed to these trends. Our finding of a novel orbivirus in dogs with high wildlife contact supports the idea that free-roaming dogs may serve as intermediary hosts for viruses of medical importance to humans and other animals.

[Source](https://www.microbiologyresearch.org/content/journal/jgv/10.1099/jgv.0.002011)

10.1099/jgv.0.002011

---

