## An ethical way forward for Indigenous microbiome research.
 02.09.2024, _S Jones_



[Source](https://www.nature.com/immersive/d41586-024-02792-w/index.html)

10.1038/d41586-024-02792-w

---

## Interplay between particle size and microbial ecology in the gut microbiome.
 03.09.2024, _J Letourneau, VM Carrion, J Zeng, S Jiang, OW Osborne, ZC Holmes, A Fox, P Epstein, CY Tan, M Kirtley, NK Surana, LA David_


Physical particles can serve as critical abiotic factors that structure the ecology of microbial communities. For non-human vertebrate gut microbiomes, fecal particle size (FPS) has been known to be shaped by chewing efficiency and diet. However, little is known about what drives FPS in the human gut. Here, we analyzed FPS by laser diffraction across a total of 76 individuals and found FPS to be strongly individualized. Contrary to our initial hypothesis, a behavioral intervention with 41 volunteers designed to increase chewing efficiency did not impact FPS. Dietary patterns could also not be associated with FPS. Instead, we found evidence that human and mouse gut microbiomes shaped FPS. Fecal samples from germ-free and antibiotic-treated mice exhibited increased FPS relative to colonized mice. In humans, markers of longer transit time were correlated with smaller FPS. Gut microbiota diversity and composition were also associated with FPS. Finally, ex vivo culture experiments using human fecal microbiota from distinct donors showed that differences in microbiota community composition can drive variation in particle size. Together, our results support an ecological model in which the human gut microbiome plays a key role in reducing the size of food particles during digestion. This finding has important implications for our understanding of energy extraction and subsequent uptake in gastrointestinal tract. FPS may therefore be viewed as an informative functional readout, providing new insights into the metabolic state of the gut microbiome.

[Source](https://academic.oup.com/ismej/advance-article/doi/10.1093/ismejo/wrae168/7746109)

10.1093/ismejo/wrae168

---

## Unexplored microbial diversity from 2,500 food metagenomes and links with the human microbiome 

[Source](https://www.cell.com/cell/fulltext/S0092-8674(24)00833-X)

---

## A streamlined culturomics case study for the human gut microbiota research

[Source](https://www.nature.com/articles/s41598-024-71370-x)

---

## Gut microbiota patterns associated with duration of diarrhea in children under five years of age in Ethiopia

[Source](https://www.nature.com/articles/s41467-024-51464-w)

---

## Association of maternal gut microbial metabolites with gestational diabetes mellitus: evidence from an original case-control study, meta-analysis, and Mendelian randomization

[Source](https://www.nature.com/articles/s41430-024-01502-z)

---

## The Microleaks study: 16S community profile and metagenomic shotgun sequencing signatures associated with anastomotic leak

[Source](https://www.nature.com/articles/s44355-024-00006-8)

---

## Targeted isolation of Methanobrevibacter strains from fecal samples expands the cultivated human archaeome

[Source](https://www.nature.com/articles/s41467-024-52037-7)

---

## Characterization of gut microbiota in the Uyghur osteopenia population 

[Source](https://www.nature.com/articles/s41598-024-71077-z)

---

## Spaceflight alters host-gut microbiota interactions 

[Source](https://www.nature.com/articles/s41522-024-00545-1)

---

## Bacterial association with metals enables in vivo monitoring of urogenital microbiota using magnetic resonance imaging

[Source](https://www.nature.com/articles/s42003-024-06783-5)

---

## Urine metabolomic profiles of autism and autistic traits – A twin study

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0308224)

---

## Massive expansion of the pig gut virome based on global metagenomic mining

[Source](https://www.nature.com/articles/s41522-024-00554-0)

---

## Marine sponge microbe provides insights into evolution and virulence of the tubercle bacillus

[Source](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1012440)

---

## Exploring the interactions between Nosema ceranae infection and the honey bee gut microbiome

[Source](https://www.nature.com/articles/s41598-024-67796-y)

---

## Metagenomic characterisation of canine skin reveals a core healthy skin microbiome 

[Source](https://www.nature.com/articles/s41598-024-63999-5)

---

## CHIKV infection drives shifts in the gastrointestinal microbiome and metabolites in rhesus monkeys 

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01895-w)

---

## Effects of dietary restriction and one-carbon metabolite supplementation during the first 63 days of gestation on the maternal gut, vaginal, and blood microbiota in cattle 

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-024-00335-2)

---

## Aloe arborescens supplementation in drying-off dairy cows: influence on rumen, rectum and milk microbiomes

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-024-00336-1)

---

## Comparative genomics identifies key adaptive traits of sponge-associated microbial symbionts 

[Source](https://enviromicro-journals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.16690)

---

## Deciphering the role of rhizosphere microbiota in modulating disease resistance in cabbage varieties.
 03.09.2024, _X Ping, RAA Khan, S Chen, Y Jiao, X Zhuang, L Jiang, L Song, Y Yang, J Zhao, Y Li, Z Mao, B Xie, J Ling_


_Rhizosphere, Brassica, Disease Resistance, Plant Diseases, Soil Microbiology, Fusarium, Microbiota, Bacteria, Plant Roots, Fungi_
Cabbage Fusarium wilt (CFW) is a devastating disease caused by the soil-borne fungus Fusarium oxysporum f. sp. conglutinans (Foc). One of the optimal measures for managing CFW is the employment of tolerant/resistant cabbage varieties. However, the interplay between plant genotypes and the pathogen Foc in shaping the rhizosphere microbial community, and the consequent influence of these microbial assemblages on biological resistance, remains inadequately understood.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01883-0)

10.1186/s40168-024-01883-0

---

## Oak seedling microbiome assembly under climate warming and drought.
 02.09.2024, _D Hoefle, M Sommer, B Wassermann, M Faticov, D Serra, G Berg, AJM Tack, A Abdelfattah_


Despite that climate change is currently one of the most pervasive challenges, its effects on the plant-associated microbiome is still poorly studied. The aim of this study was to evaluate the impact of the independent and combinatory effect of climate warming and drought on the microbiome assembly of oak from seed to seedling. In a multifactorial experimental set up, acorns were subjected to different temperatures (15 °C, 20 °C, and 25 °C) and soil moisture levels (drought (15%) and control (60%)) from germination until the seedling stage, after which the bacterial and fungal communities associated to the rhizosphere and phyllosphere were characterized by amplicon sequencing and qPCR. The results showed a stronger effect of temperature on fungal than on bacterial diversity and the effect was more pronounced in the phyllosphere. Under drought condition, temperature had a significantly negative effect on phyllosphere fungal diversity. In the rhizosphere, temperature had a significant effect on the fungal community composition which was primarily caused by species turnover. Regardless of temperature, Actinobacteriota was significantly enriched in drought, a group of bacteria known to increase plant drought tolerance. This study provides new insights into the effect of climate change on the plant microbiome in natural ecosystems.

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-024-00602-4)

10.1186/s40793-024-00602-4

---

## The ancestral environment of teosinte populations shapes their root microbiome.
 01.09.2024, _CJ Barnes, MS Bünner, MR Ramírez-Flores, IB Nielsen, J Ramos-Madrigal, D Zharikova, CM McLaughlin, MT Gilbert, RJH Sawers_


The composition of the root microbiome affects the host's growth, with variation in the host genome associated with microbiome variation. However, it is not known whether this intra-specific variation of root microbiomes is a consequence of plants performing targeted manipulations of them to adapt to their local environment or varying passively with other traits. To explore the relationship between the genome, environment and microbiome, we sampled seeds from teosinte populations across its native range in Mexico. We then grew teosinte accessions alongside two modern maize lines in a common garden experiment. Metabarcoding was performed using universal bacterial and fungal primers to profile their root microbiomes.

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-024-00606-0)

10.1186/s40793-024-00606-0

---

## Prokaryotic-virus-encoded auxiliary metabolic genes throughout the global oceans.
 03.09.2024, _F Tian, JM Wainaina, C Howard-Varona, G Domínguez-Huerta, B Bolduc, MC Gazitúa, G Smith, MR Gittrich, O Zablocki, DR Cronin, D Eveillard, SJ Hallam, MB Sullivan_


_Oceans and Seas, Seawater, Metagenomics, Bacteria, Viruses, Prokaryotic Cells, Metagenome, Metabolic Networks and Pathways, Gene Transfer, Horizontal, Phosphatidylethanolamines_
Prokaryotic microbes have impacted marine biogeochemical cycles for billions of years. Viruses also impact these cycles, through lysis, horizontal gene transfer, and encoding and expressing genes that contribute to metabolic reprogramming of prokaryotic cells. While this impact is difficult to quantify in nature, we hypothesized that it can be examined by surveying virus-encoded auxiliary metabolic genes (AMGs) and assessing their ecological context.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01876-z)

10.1186/s40168-024-01876-z

---

## Global distribution, diversity, and ecological niche of Picozoa, a widespread and enigmatic marine protist lineage.
 05.09.2024, _P Huber, D De Angelis, H Sarmento, S Metz, CR Giner, C Vargas, L Maiorano, R Massana, R Logares_


_Phylogeny, Eukaryota, Biodiversity, Aquatic Organisms, Ecosystem, Seawater_
The backbone of the eukaryotic tree of life contains taxa only found in molecular surveys, of which we still have a limited understanding. Such is the case of Picozoa, an enigmatic lineage of heterotrophic picoeukaryotes within the supergroup Archaeplastida, which has emerged as a significant component of marine microbial planktonic communities. To enhance our understanding of the diversity, distribution, and ecology of Picozoa, we conduct a comprehensive assessment at different levels, from assemblages to taxa, employing phylogenetic analysis, species distribution modeling, and ecological niche characterization.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01874-1)

10.1186/s40168-024-01874-1

---

## Microbial core communities in activated sludge plants are strongly affected by immigration and geography.
 02.09.2024, _SZ Vestergaard, G Dottorini, M Peces, A Murguz, MKD Dueholm, M Nierychlo, PH Nielsen_


The microbiota in wastewater treatment plants (WWTPs) and incoming wastewater is critical for the treatment process, the preservation of natural ecosystems and human health, and for the recovery of resources and achievement of sustainability goals. Both core species and conditionally rare and abundant taxa (CRAT) are considered process-critical but little is known about identity as well as true functional and ecological importance. Here, we present a comprehensive investigation of the microbiota of 84 municipal activated sludge (AS) plants with nutrient removal treating ~ 70% of all wastewater within a confined geographical area, Denmark (43,000 km2). With the use of an ecosystem-specific database (MiDAS 5.2), species-level classification allowed us to investigate the core and CRAT species, whether they were active, and important factors determining their presence.

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-024-00604-2)

10.1186/s40793-024-00604-2

---

## The eukaryome of modern microbialites reveals distinct colonization across aquatic ecosystems.
 06.09.2024, _AM Bonacolta, PT Visscher, J Del Campo, RA White Iii_


_Ecosystem, Salinity, British Columbia, Eukaryota, Biodiversity, Fresh Water, Biofilms, Seawater, Australia, Diatoms, Geologic Sediments_
Protists are less studied for their role and diversity in ecosystems. Notably, protists have played and still play an important role in microbialites. Microbialites, or lithified microbial mats, represent the oldest evidence of fossil biofilms (~3.5 Gyr). Modern microbialites may offer a unique proxy to study the potential role of protists within a geological context. We examined protist diversity in freshwater (Kelly and Pavilion Lake in British Columbia, Canada) and marine (Highborne Cay, Bahamas) to hypersaline (Shark Bay, Australia) microbialites to decipher their geomicrobiological role. The freshwater microbialite communities were clearly distinct from their marine and hypersaline counterparts. Chlorophytes had higher numerical abundance in freshwater microbialites; whereas pennate diatoms dominated numerically in marine microbialites. Despite the differences, protists across ecosystems may have adopted similar roles and functions. We suggest a consistent biogeochemical role of protists across microbialites globally; but that salinity may shape protist composition and evolution in these ecosystems.

[Source](https://www.nature.com/articles/s41522-024-00547-z)

10.1038/s41522-024-00547-z

---

## Domestic freezer storage: a solution for preserving stool microbiota integrity for at-home collection

[Source](https://www.biorxiv.org/content/10.1101/2024.09.02.610874v1.full.pdf+html)

---

## COVID-19 prevention behaviors and dietary habits among undergraduate students: A health belief model approach

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0309623)

---

## A catalog of small proteins from the global microbiome.
 02.09.2024, _Y Duan, CD Santos-Júnior, TS Schmidt, A Fullam, BLS de Almeida, C Zhu, M Kuhn, XM Zhao, P Bork, LP Coelho_


_Microbiota, Open Reading Frames, Bacteria, Metagenome, Archaea, Molecular Sequence Annotation, Bacterial Proteins_
Small open reading frames (smORFs) shorter than 100 codons are widespread and perform essential roles in microorganisms, where they encode proteins active in several cell functions, including signal pathways, stress response, and antibacterial activities. However, the ecology, distribution and role of small proteins in the global microbiome remain unknown. Here, we construct a global microbial smORFs catalog (GMSC) derived from 63,410 publicly available metagenomes across 75 distinct habitats and 87,920 high-quality isolate genomes. GMSC contains 965 million non-redundant smORFs with comprehensive annotations. We find that archaea harbor more smORFs proportionally than bacteria. We moreover provide a tool called GMSC-mapper to identify and annotate small proteins from microbial (meta)genomes. Overall, this publicly-available resource demonstrates the immense and underexplored diversity of small proteins.

[Source](https://www.nature.com/articles/s41467-024-51894-6#Sec10)

10.1038/s41467-024-51894-6

---

