## The cancer chemotherapeutic 5-fluorouracil is a potent <em>Fusobacterium nucleatum </em>inhibitor and its activity is modified by intratumoral microbiota

[Source](https://doi.org/10.1016/j.celrep.2022.111625)

---

## Enterococci enhance <em>Clostridioides difficile</em> pathogenesis

[Source](https://doi.org/10.1038/s41586-022-05438-x)

---

## Host biology, ecology and the environment influence microbial biomass and diversity in 101 marine fish species

[Source](https://doi.org/10.1038/s41467-022-34557-2)

---

## Genomic ancestry, diet and microbiomes of Upper Palaeolithic hunter-gatherers from San Teodoro cave

[Source](https://doi.org/10.1038/s42003-022-04190-2)

---

## Gut microbiota analysis for prediction of clinical relapse in Crohn's disease.
 22.11.2022, _S Buffet-Bataillon, G Bouguen, F Fleury, V Cattoir, Y Le Cunff_


_Humans, Gastrointestinal Microbiome, Crohn Disease, Fatty Acids, Volatile, Feces, Clostridiales, Chronic Disease, Firmicutes, Recurrence_
The role of intestinal bacterial microbiota has been described as key in the pathophysiology of Crohn's disease (CD). CD is characterized by frequent relapses after periods of remission which are not entirely understood. In this paper, we investigate whether the heterogeneity in microbiota profiles in CD patients could be a suitable predictor for these relapses. This prospective observational study involved 259 CD patients, in which 41 provided an additional total of 62 consecutive fecal samples, with an average interval of 25 weeks in between each of these samples. Fecal microbiota was analyzed by massive genomic sequencing through 16 S rRNA amplicon sampling. We found that our 259 CD patients could be split into three distinct subgroups of microbiota (G1, G2, G3). From G1 to G3, we noticed a progressive decrease in alpha diversity (p ≤ 0.0001) but no change in the fecal calprotectin (FC) level. Focusing on the 103 consecutive samples from 41 CD patients, we showed that the patients microbiota profiles were remarkably stable over time and associated with increasing symptom severity. Investigating further this microbiota/severity association revealed that the first signs of aggravation are (1) a loss of the main anti-inflammatory Short-Chain Fatty Acids (SCFAs) Roseburia, Eubacterium, Subdoligranumum, Ruminococcus (P &lt; 0.05), (2) an increase in pro-inflammatory pathogens Proteus, Finegoldia (P &lt; 0.05) while (3) an increase of other minor SCFA producers such as Ezakiella, Anaerococcus, Megasphaera, Anaeroglobus, Fenollaria (P &lt; 0.05). Further aggravation of clinical signs is significantly linked to the subsequent loss of these minor SCFAs species and to an increase in other proinflammatory Proteobacteria such as Klebsiella, Pseudomonas, Salmonella, Acinetobacter, Hafnia and proinflammatory Firmicutes such as Staphylococcus, Enterococcus, Streptococcus. (P &lt; 0.05). To our knowledge, this is the first study (1) specifically identifying subgroups of microbiota profiles in CD patients, (2) relating these groups to the evolution of symptoms over time and (3) showing a two-step process in CD symptoms' worsening. This paves the way towards a better understanding of patient-to-patient heterogeneity, as well as providing early warning signals of future aggravation of the symptoms and eventually adapting empirically treatments.

[Source](https://doi.org/10.1038/s41598-022-23757-x)

10.1038/s41598-022-23757-x

---

## Implications of oral streptococcal bacteriophages in autism spectrum disorder

[Source](https://doi.org/10.1038/s41522-022-00355-3)

---

## Reproducible changes in the anorexia nervosa gut microbiota following inpatient therapy remain distinct from non-eating disorder controls.
 22.11.2022, _F Fouladi, EC Bulik-Sullivan, EM Glenny, LM Thornton, KK Reed, S Thomas, S Kleiman, A Watters, J Oakes, EY Huh, Q Tang, J Liu, Z Djukic, L Harper, Y Trillo-Ordoñez, S Sun, I Blakely, PS Mehler, AA Fodor, LM Tarantino, CM Bulik, IM Carroll_


_Humans, Gastrointestinal Microbiome, Anorexia Nervosa, Inpatients, Feces, Microbiota_
The composition of the gut microbiota in patients with anorexia nervosa (AN), and the ability of this microbial community to influence the host, remains uncertain. To achieve a broader understanding of the role of the intestinal microbiota in patients with AN, we collected fecal samples before and following clinical treatment at two geographically distinct eating disorder units (Center of Excellence for Eating Disorders [UNC-CH] and ACUTE Center for Eating Disorders [Denver Health]). Gut microbiotas were characterized in patients with AN, before and after inpatient treatment, and in non-eating disorder (non-ED) controls using shotgun metagenomic sequencing. The impact of inpatient treatment on the AN gut microbiota was remarkably consistent between eating disorder units. Although weight in patients with AN showed improvements, AN microbiotas post-treatment remained distinct from non-ED controls. Additionally, AN gut microbiotas prior to treatment exhibited more fermentation pathways and a lower ability to degrade carbohydrates than non-ED controls. As the intestinal microbiota can influence nutrient metabolism, our data highlight the complex microbial communities in patients with AN as an element needing further attention post inpatient treatment. Additionally, this study defines the effects of renourishment on the AN gut microbiota and serves as a platform to develop precision nutrition approaches to potentially mitigate impediments to recovery.

[Source](https://doi.org/10.1080/19490976.2022.2143217)

10.1080/19490976.2022.2143217

---

## Nasal microbiome disruption and recovery after mupirocin treatment in <em>Staphylococcus aureus</em> carriers and noncarriers

[Source](https://doi.org/10.1038/s41598-022-21453-4)

---

## BlpC-mediated selfish program leads to rapid loss of <em>Streptococcus pneumoniae</em> clonal diversity during infection

[Source](https://doi.org/10.1016/j.chom.2022.10.015)

---

## Treasures from trash in cancer research

[Source](https://doi.org/10.18632/oncotarget.28308)

---

## Human Gut Metagenomes Encode Diverse GH156 Sialidases.
 17.11.2022, _E Mann, S Shekarriz, MG Surette_


The intestinal lining is protected by a mucous barrier composed predominantly of complex carbohydrates. Gut microbes employ diverse glycoside hydrolases (GHs) to liberate mucosal sugars as a nutrient source to facilitate host colonization. Intensive catabolism of mucosal glycans, however, may contribute to barrier erosion, pathogen encroachment, and inflammation. Sialic acid is an acidic sugar featured at terminal positions of host glycans. Characterized sialidases from the microbiome belong to the GH33 family, according to CAZy (Carbohydrate-Active enZYmes Database). In 2018 a functional metagenomics screen using thermal spring DNA uncovered the founding member of the GH156 sialidase family, the presence of which has yet to be reported in the context of the human microbiome. A subset of GH156 sequences from the CAZy database containing key sialidase residues was used to build a hidden Markov model. HMMsearch against public databases revealed ~10× more putative GH156 sialidases than currently cataloged by CAZy. Represented phyla include Bacteroidota, Verrucomicrobiota, and Firmicutes_A from human microbiomes, all of which play notable roles in carbohydrate fermentation. Analyses of metagenomic data sets revealed that GH156s are frequently encoded in metagenomes, with a greater variety and abundance of GH156 genes observed in traditional hunter-gatherer or agriculturalist societies than in industrialized societies, particularly relative to individuals with inflammatory bowel disease (IBD). Nineteen GH156s were recombinantly expressed and assayed for sialidase activity. The five GH156 sialidases identified here share limited sequence identity to each other or the founding GH156 family member and are representative of a large subset of the family. IMPORTANCE Sialic acids occupy terminal positions of human glycans where they act as receptors for microbes, toxins, and immune signaling molecules. Microbial enzymes that remove sialic acids, sialidases, are abundant in the human microbiome where they may contribute to shaping the microbiota community structure or contribute to pathology. Furthermore, sialidases have proven to hold therapeutic potential for cancer therapy. Here, we examined the sequence space of a sialidase family of enzymes, GH156, previously unknown in the human gut environment. Our analyses suggest that human populations with disparate dietary practices harbor distinct varieties and abundances of GH156-encoding genes. Furthermore, we demonstrate the sialidase activity of 5 gut-derived GH156s. These results expand the diversity of sialidases that may contribute to host glycan degradation, and these sequences may have biotechnological or clinical utility.

[Source](https://doi.org/10.1128/aem.01755-22)

10.1128/aem.01755-22

---

## Species-Specific Patterns of Gut Metabolic Modules in Dutch Individuals with Different Dietary Habits.
 17.11.2022, _SA Shetty, PB Stege, J Hordijk, E Gijsbers, CM Dierikx, E van Duijkeren, E Franz, RJL Willems, FL Paganelli, S Fuentes_


Diet is an important determinant of the human gut microbiome. Here, we analyzed fecal metagenomes of Dutch adults following omnivorous, pescatarian, vegan, and vegetarian diets. We compared the taxonomic composition of individuals from our study with publicly available gut metagenomes from westernized and non-westernized societies. We observed that, despite long-term transition to diets rich in plant fibers (vegan or vegetarian), the microbiomes of these were typical of westernized populations, and similar in composition to omnivores. Although there were no major differences in metabolic modules, we identified differences in the species that contributed to particular functions, such as carbohydrate degradation and short-chain fatty acid metabolism. Overall, this study shows functional redundancy of the microbiomes among westernized populations, which is independent of long-term individual dietary habits. IMPORTANCE Diet is an important modulator of the human gut microbiome, which is susceptible to increased consumption of plant fibers in vegan or vegetarian lifestyles. To investigate this, we compared the gut microbiome of Dutch adults following omnivorous, pescatarian, vegan and vegetarian diets. We did not observe major differences in the gut microbiome composition and function between individuals with different dietary habits. However, we observed differences in the species that contribute to the core functions of the gut microbiome. Our study thus emphasizes the need to better understand the species-specific functional changes associated with dietary habits in the human gut microbiome.

[Source](https://doi.org/10.1128/msphere.00512-22)

10.1128/msphere.00512-22

---

## Effect of the intratumoral microbiota on spatial and cellular heterogeneity in cancer.
 20.11.2022, _JL Galeano Niño, H Wu, KD LaCourse, AG Kempchinsky, A Baryiames, B Barber, N Futran, J Houlton, C Sather, E Sicinska, A Taylor, SS Minot, CD Johnston, S Bullman_


The tumour-associated microbiota is an intrinsic component of the tumour microenvironment across human cancer types1,2. Intratumoral host-microbiota studies have so far largely relied on bulk tissue analysis1-3, which obscures the spatial distribution and localized effect of the microbiota within tumours. Here, by applying in situ spatial-profiling technologies4 and single-cell RNA sequencing5 to oral squamous cell carcinoma and colorectal cancer, we reveal spatial, cellular and molecular host-microbe interactions. We adapted 10x Visium spatial transcriptomics to determine the identity and in situ location of intratumoral microbial communities within patient tissues. Using GeoMx digital spatial profiling6, we show that bacterial communities populate microniches that are less vascularized, highly immuno‑suppressive and associated with malignant cells with lower levels of Ki-67 as compared to bacteria-negative tumour regions. We developed a single-cell RNA-sequencing method that we name INVADEseq (invasion-adhesion-directed expression sequencing) and, by applying this to patient tumours, identify cell-associated bacteria and the host cells with which they interact, as well as uncovering alterations in transcriptional pathways that are involved in inflammation, metastasis, cell dormancy and DNA repair. Through functional studies, we show that cancer cells that are infected with bacteria invade their surrounding environment as single cells and recruit myeloid cells to bacterial regions. Collectively, our data reveal that the distribution of the microbiota within a tumour is not random; instead, it is highly organized in microniches with immune and epithelial cell functions that promote cancer progression.

[Source](https://doi.org/10.1038/s41586-022-05435-0)

10.1038/s41586-022-05435-0

---

## Broad-spectrum antibiotics associated gut microbiome disturbance impairs T cell immunity and promotes lung cancer metastasis: a retrospective study.
 21.11.2022, _K Xu, J Cai, J Xing, X Li, B Wu, Z Zhu, Z Zhang_


_Humans, Lung Neoplasms, Carcinoma, Non-Small-Cell Lung, Gastrointestinal Microbiome, Retrospective Studies, Dysbiosis, Anti-Bacterial Agents, Neoplastic Processes, T-Lymphocytes, DNA, Ribosomal_
Gut microbiome has been linked to a regulatory role in cancer progression. However, whether broad-spectrum antibiotics (ATB) associated gut microbiome dysbiosis contributes to an impaired T cell immune function, and ultimately promotes lung cancer metastasis is not well known.

[Source](https://doi.org/10.1186/s12885-022-10307-x)

10.1186/s12885-022-10307-x

---

## Diet-induced modifications to human microbiome reshape colonic homeostasis in irritable bowel syndrome.
 21.11.2022, _H Bootz-Maoz, A Pearl, E Melzer, S Malnick, E Sharon, Y Bennet, R Tsentsarevsky, S Abuchatzera, S Amidror, E Aretz, S Azriel, C Gam Ze Letova, M Naama, I Shoval, O Yaron, S Karako-Lampert, S Bel, N Yissachar_


_Humans, Irritable Bowel Syndrome, Diet, Carbohydrate-Restricted, Gastrointestinal Microbiome, Homeostasis_
Changes in microbiome composition are associated with a wide array of human diseases, turning the human microbiota into an attractive target for therapeutic intervention. Yet, clinical translation of these findings requires the establishment of causative connections between specific microbial taxa and their functional impact on host tissues. Here, we infuse gut organ cultures with longitudinal microbiota samples collected from therapy-naive patients with irritable bowel syndrome (IBS) under a low-fermentable oligo-, di-, mono-saccharides and polyols (FODMAP) diet. We show that post-diet microbiota regulates intestinal expression of inflammatory and neuro-muscular gene sets. Specifically, we identify Bifidobacterium adolescentis as a diet-sensitive pathobiont that alters tight junction integrity and disrupts gut barrier functions. Collectively, we present a pathway discovery platform for mechanistic dissection and identification of functional diet-host-microbiota modules. Our data support the hypothesis that the gut microbiota mediates the beneficial effects of a low-FODMAP diet and reinforce the potential feasibility of microbiome-based therapies in IBS.

[Source](https://doi.org/10.1016/j.celrep.2022.111657)

10.1016/j.celrep.2022.111657

---

## The dynamic lung microbiome in health and disease.
 22.11.2022, _JG Natalini, S Singh, LN Segal_


New methods and technologies within the field of lung biology are beginning to shed new light into the microbial world of the respiratory tract. Long considered to be a sterile environment, it is now clear that the human lungs are frequently exposed to live microbes and their by-products. The nature of the lung microbiome is quite distinct from other microbial communities inhabiting our bodies such as those in the gut. Notably, the microbiome of the lung exhibits a low biomass and is dominated by dynamic fluxes of microbial immigration and clearance, resulting in a bacterial burden and microbiome composition that is fluid in nature rather than fixed. As our understanding of the microbial ecology of the lung improves, it is becoming increasingly apparent that certain disease states can disrupt the microbial-host interface and ultimately affect disease pathogenesis. In this Review, we provide an overview of lower airway microbial dynamics in health and disease and discuss future work that is required to uncover novel therapeutic targets to improve lung health.

[Source](https://doi.org/10.1038/s41579-022-00821-x)

10.1038/s41579-022-00821-x

---

## Modulation of gut microbiota, blood metabolites, and disease resistance by dietary β-glucan in rainbow trout (<em>Oncorhynchus mykiss</em>)

[Source](https://doi.org/10.1186/s42523-022-00209-5)

---

## Living in honey: bacterial and fungal communities in honey of sympatric populations of <em>Apis mellifera</em> and the stingless bee <em>Melipona beecheii</em>, in Yucatan, Mexico

[Source](https://doi.org/10.1007/s00203-022-03319-5)

---

## The involvement of organic acids in soil fertility, plant health and environment sustainability.
 22.11.2022, _SS Sindhu, A Sehrawat, BR Glick_


_Humans, Soil, Soil Microbiology, Plant Roots, Plants, Microbiota_
Increasing demand for safe food by an ever-growing human population emphasizes the urgency for increasing crop yields and reducing the losses caused by abiotic and biotic stresses; a partial solution to this problem is to develop a better understanding of plant-microbe interactions. Plant roots continuously release a wide range of compounds including organic acids in root exudates. These root exudates stimulate growth of specific microbial communities in the rhizosphere, which affect complex biological and physico-chemical interactions occurring between plant roots and the surrounding soil environment. In addition, organic acids are also released by different microbes and during decomposition of organic matter and plant residues in the soil. Interestingly, the available organic acids in the rhizosphere play crucial roles in various physio-chemical processes including the chemoattraction of microbes (both beneficial and pathogenic), mineralization and solubilization of complex minerals (P, K and Zn), biocontrol of phytopathogens, induction of systemic resistance, biogas formation, mitigation of abiotic stresses and, detoxification of metals and residual pesticides. Thus, organic acids play a significant role in the sustainable management of the soil ecosystem and in environmental sustainability. This review discusses the role of organic acids in the stimulation or enrichment of specific root-associated microbial communities and their effect on plant-microbe interactions at the root surface. In addition, the potential for root microbiome modification to enhance nutrient cycling and nutrient acquisition, and in amelioration of environmental stresses for increasing food production is discussed.

[Source](https://doi.org/10.1007/s00203-022-03321-x)

10.1007/s00203-022-03321-x

---

## Investigating the impact of database choice on the accuracy of metagenomic read classification for the rumen microbiome.
 21.11.2022, _RH Smith, L Glendinning, AW Walker, M Watson_


Microbiome analysis is quickly moving towards high-throughput methods such as metagenomic sequencing. Accurate taxonomic classification of metagenomic data relies on reference sequence databases, and their associated taxonomy. However, for understudied environments such as the rumen microbiome many sequences will be derived from novel or uncultured microbes that are not present in reference databases. As a result, taxonomic classification of metagenomic data from understudied environments may be inaccurate. To assess the accuracy of taxonomic read classification, this study classified metagenomic data that had been simulated from cultured rumen microbial genomes from the Hungate collection. To assess the impact of reference databases on the accuracy of taxonomic classification, the data was classified with Kraken 2 using several reference databases. We found that the choice and composition of reference database significantly impacted on taxonomic classification results, and accuracy. In particular, NCBI RefSeq proved to be a poor choice of database. Our results indicate that inaccurate read classification is likely to be a significant problem, affecting all studies that use insufficient reference databases. We observed that adding cultured reference genomes from the rumen to the reference database greatly improved classification rate and accuracy. We also demonstrated that metagenome-assembled genomes (MAGs) have the potential to further enhance classification accuracy by representing uncultivated microbes, sequences of which would otherwise be unclassified or incorrectly classified. However, classification accuracy was strongly dependent on the taxonomic labels assigned to these MAGs. We therefore highlight the importance of accurate reference taxonomic information and suggest that, with formal taxonomic lineages, MAGs have the potential to improve classification rate and accuracy, particularly in environments such as the rumen that are understudied or contain many novel genomes.

[Source](https://doi.org/10.1186/s42523-022-00207-7)

10.1186/s42523-022-00207-7

---

## dbCAN-seq update: CAZyme gene clusters and substrates in microbiomes.
 18.11.2022, _J Zheng, B Hu, X Zhang, Q Ge, Y Yan, J Akresi, V Piyush, L Huang, Y Yin_


Carbohydrate Active EnZymes (CAZymes) are significantly important for microbial communities to thrive in carbohydrate rich environments such as animal guts, agricultural soils, forest floors, and ocean sediments. Since 2017, microbiome sequencing and assembly have produced numerous metagenome assembled genomes (MAGs). We have updated our dbCAN-seq database (https://bcb.unl.edu/dbCAN_seq) to include the following new data and features: (i) ∼498 000 CAZymes and ∼169 000 CAZyme gene clusters (CGCs) from 9421 MAGs of four ecological (human gut, human oral, cow rumen, and marine) environments; (ii) Glycan substrates for 41 447 (24.54%) CGCs inferred by two novel approaches (dbCAN-PUL homology search and eCAMI subfamily majority voting) (the two approaches agreed on 4183 CGCs for substrate assignments); (iii) A redesigned CGC page to include the graphical display of CGC gene compositions, the alignment of query CGC and subject PUL (polysaccharide utilization loci) of dbCAN-PUL, and the eCAMI subfamily table to support the predicted substrates; (iv) A statistics page to organize all the data for easy CGC access according to substrates and taxonomic phyla; and (v) A batch download page. In summary, this updated dbCAN-seq database highlights glycan substrates predicted for CGCs from microbiomes. Future work will implement the substrate prediction function in our dbCAN2 web server.

[Source](https://doi.org/10.1093/nar/gkac1068)

10.1093/nar/gkac1068

---

## Gene co-expression network analysis of the human gut commensal bacterium <em>Faecalibacterium prausnitzii</em> in R-Shiny

[Source](https://doi.org/10.1371/journal.pone.0271847)

---

