## @CTMRBio

[Source](https://twitter.com/CTMRBio?ref_src=twsrc%5Etfw)

---

## A large scale prediction of bacteriocin gene blocks suggests a wide functional spectrum for bacteriocins.
 02.12.2018, _JT Morton, SD Freed, SW Lee, I Friedberg_


_Anti-Bacterial Agents, Bacteriocins, Chromosome Mapping, Genome, Archaeal, Genome, Bacterial, Operon, Software_
Bacteriocins are peptide-derived molecules produced by bacteria, whose recently-discovered functions include virulence factors and signaling molecules as well as their better known roles as antibiotics. To date, close to five hundred bacteriocins have been identified and classified. Recent discoveries have shown that bacteriocins are highly diverse and widely distributed among bacterial species. Given the heterogeneity of bacteriocin compounds, many tools struggle with identifying novel bacteriocins due to their vast sequence and structural diversity. Many bacteriocins undergo post-translational processing or modifications necessary for the biosynthesis of the final mature form. Enzymatic modification of bacteriocins as well as their export is achieved by proteins whose genes are often located in a discrete gene cluster proximal to the bacteriocin precursor gene, referred to as context genes in this study. Although bacteriocins themselves are structurally diverse, context genes have been shown to be largely conserved across unrelated species.

[Source](https://twitter.com/jamietmorton?ref_src=twsrc%5Etfw)

10.1186/s12859-015-0792-9

---

## @jamietmorton

[Source](https://twitter.com/jamietmorton?ref_src=twsrc%5Etfw)

---

## @JournalSpectrum

[Source](https://twitter.com/JournalSpectrum?ref_src=twsrc%5Etfw)

---

## https://t.co/AvVIkPLCgw

[Source](https://t.co/AvVIkPLCgw)

---

## April 13, 2023

[Source](https://twitter.com/JWoodelius/status/1646525435625488389?ref_src=twsrc%5Etfw)

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## @MurdochUni

[Source](https://twitter.com/MurdochUni?ref_src=twsrc%5Etfw)

---

## @lisafstinson

[Source](https://twitter.com/lisafstinson?ref_src=twsrc%5Etfw)

---

## @OHSUBCB

[Source](https://twitter.com/OHSUBCB?ref_src=twsrc%5Etfw)

---

## @ClareLou9

[Source](https://twitter.com/ClareLou9?ref_src=twsrc%5Etfw)

---

## @kfrayman

[Source](https://twitter.com/kfrayman?ref_src=twsrc%5Etfw)

---

## https://t.co/e5zd1rzCEp

[Source](https://t.co/e5zd1rzCEp)

---

## pic.twitter.com/b5G0ZDvlka

[Source](https://t.co/b5G0ZDvlka)

---

## A prospective pilot study to assess for histologic changes on vulvar biopsies in postmenopausal women with lichen sclerosus treated with fractionated CO2 laser therapy.
 16.04.2023, _AA Dieter, CB Iglesia, JH Lee, MJ Etcheverry, MK Gonzales, AI Sokol, E Tefera, MA Cardis_


To investigate the histologic characteristics of vulvar tissues before and after completion of fractionated carbon dioxide (CO2 ) laser therapy (FxCO2) for vulvar lichen sclerosus (LS). The secondary objective was to assess subjective improvement in symptoms via the Skindex-16 questionnaire.

[Source](https://twitter.com/MicrobiomeVIF/status/1643235758361198592?ref_src=twsrc%5Etfw)

10.1002/lsm.23669

---

## April 4, 2023

[Source](https://twitter.com/MicrobiomeVIF/status/1643235758361198592?ref_src=twsrc%5Etfw)

---

## Characterization of Antibiotic Resistance Gene Abundance and Microbiota Composition in Feces of Organic and Conventional Pigs from Four EU Countries.
 13.11.2018, _L Gerzova, V Babak, K Sedlar, M Faldynova, P Videnska, D Cejkova, AN Jensen, M Denis, A Kerouanton, A Ricci, V Cibin, J Ã–sterberg, I Rychlik_


_Animal Husbandry, Animals, Drug Resistance, Bacterial, European Union, Feces, Genes, Bacterial, Microbiota, Organic Agriculture, RNA, Ribosomal, 16S, Swine_
One of the recent trends in animal production is the revival of interest in organic farming. The increased consumer interest in organic animal farming is mainly due to concerns about animal welfare and the use of antibiotics in conventional farming. On the other hand, providing animals with a more natural lifestyle implies their increased exposure to environmental sources of different microorganisms including pathogens. To address these concerns, we determined the abundance of antibiotic resistance and diversity within fecal microbiota in pigs kept under conventional and organic farming systems in Sweden, Denmark, France and Italy. The abundance of sul1, sul2, strA, tet(A), tet(B) and cat antibiotic resistance genes was determined in 468 samples by real-time PCR and the fecal microbiota diversity was characterized in 48 selected samples by pyrosequencing of V3/V4 regions of 16S rRNA. Contrary to our expectations, there were no extensive differences between the abundance of tested antibiotic resistance genes in microbiota originating from organic or conventionally housed pigs within individual countries. There were also no differences in the microbiota composition of organic and conventional pigs. The only significant difference was the difference in the abundance of antibiotic resistance genes in the samples from different countries. Fecal microbiota in the samples originating from southern European countries (Italy, France) exhibited significantly higher antibiotic resistance gene abundance than those from northern parts of Europe (Denmark, Sweden). Therefore, the geographical location of the herd influenced the antibiotic resistance in the fecal microbiota more than farm's status as organic or conventional.

[Source](https://www.biorxiv.org/content/10.1101/2023.04.07.536071v1)

10.1371/journal.pone.0132892

---

## Global trends in antimicrobial resistance on organic and conventional farms 

[Source](https://www.biorxiv.org/content/10.1101/2023.04.07.536071v1)

---

## Microbiome diversity: A barrier to the environmental spread of antimicrobial resistance?

[Source](https://www.biorxiv.org/content/10.1101/2023.03.30.534382v1)

---

## Mother-to-infant microbiota transmission and infant microbiota development across multiple body sites.
 22.03.2023, _D Bogaert, GJ van Beveren, EM de Koff, P Lusarreta Parga, CE Balcazar Lopez, L Koppensteiner, M Clerc, R Hasrat, K Arp, MLJN Chu, PCM de Groot, EAM Sanders, MA van Houten, WAA de Steenhuijsen Piters_


_Female, Pregnancy, Humans, Infant, Mothers, Microbiota, Delivery, Obstetric, Cesarean Section, Feces_
Early-life microbiota seeding and subsequent development is crucial to future health. Cesarean-section (CS) birth, as opposed to vaginal delivery, affects early mother-to-infant transmission of microbes. Here, we assess mother-to-infant microbiota seeding and early-life microbiota development across six maternal and four infant niches over the first 30 days of life in 120 mother-infant pairs. Across all infants, we estimate that on average 58.5% of the infant microbiota composition can be attributed to any of the maternal source communities. All maternal source communities seed multiple infant niches. We identify shared and niche-specific host/environmental factors shaping the infant microbiota. In CS-born infants, we report reduced seeding of infant fecal microbiota by maternal fecal microbes, whereas colonization with breastmilk microbiota is increased when compared with vaginally born infants. Therefore, our data suggest auxiliary routes of mother-to-infant microbial seeding, which may compensate for one another, ensuring that essential microbes/microbial functions are transferred irrespective of disrupted transmission routes.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(23)00043-4)

10.1016/j.chom.2023.01.018

---

## The Local Tumor Microbiome Is Associated with Survival in Late-Stage Colorectal Cancer Patients.
 12.04.2023, _JW Debelius, L Engstrand, A Matussek, N Brusselaers, JT Morton, M Stenmarker, RS Olsen_


The gut microbiome is associated with survival in colorectal cancer. Single organisms have been identified as markers of poor prognosis. However, in situ imaging of tumors demonstrate a polymicrobial tumor-associated community. To understand the role of these polymicrobial communities in survival, we conducted a nested case-control study in late-stage cancer patients undergoing resection for primary adenocarcinoma. The microbiome of paired tumor and adjacent normal tissue samples was profiled using 16S rRNA sequencing. We found a consistent difference in the microbiome between paired tumor and adjacent tissue, despite strong individual microbial identities. Furthermore, a larger difference between normal and tumor tissue was associated with prognosis: patients with shorter survival had a larger difference between normal and tumor tissue. Within the tumor tissue, we identified a 39-member community statistic associated with survival; for every log2-fold increase in this value, an individual's odds of survival increased by 20% (odds ratio survival 1.20; 95% confidence interval = 1.04 to 1.33). Our results suggest that a polymicrobial tumor-specific microbiome is associated with survival in late-stage colorectal cancer patients. IMPORTANCE Microbiome studies in colorectal cancer (CRC) have primarily focused on the role of single organisms in cancer progression. Recent work has identified specific organisms throughout the intestinal tract, which may affect survival; however, the results are inconsistent. We found differences between the tumor microbiome and the microbiome of the rest of the intestine in patients, and the magnitude of this difference was associated with survival, or, the more like a healthy gut a tumor looked, the better a patient's prognosis. Our results suggest that future microbiome-based interventions to affect survival in CRC will need to target the tumor community.

[Source](https://journals.asm.org/doi/10.1128/spectrum.05066-22)

10.1128/spectrum.05066-22

---

## Exploring <em>Bifidobacterium </em>species community and functional variations with human gut microbiome structure and health beyond infancy 

[Source](https://oaepublish.com/mrr/article/view/5583)

---

## The microbial dark matter and "wanted list" in worldwide wastewater treatment plants.
 13.04.2023, _Y Zhang, Y Wang, M Tang, J Zhou, T Zhang_


_Sewage, Wastewater, RNA, Ribosomal, 16S, Microbiota, Water Purification_
Wastewater treatment plants (WWTPs) are one of the largest biotechnology applications in the world and are of critical importance to modern urban societies. An accurate evaluation of the microbial dark matter (MDM, microorganisms whose genomes remain uncharacterized) proportions in WWTPs is of great value, while there is no such research yet. This study conducted a global meta-analysis of MDM in WWTPs with 317,542 prokaryotic genomes from the Genome Taxonomy Database and proposed a "wanted list" for priority targets in further investigations of activated sludge.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01503-3)

10.1186/s40168-023-01503-3

---

## Microbiome, resistome and mobilome of chlorine-free drinking water treatment systems.
 14.04.2023, _D CalderÃ³n-Franco, F Corbera-Rubio, M Cuesta-Sanz, B Pieterse, D de Ridder, MCM van Loosdrecht, D van Halem, M Laureni, DG Weissbrodt_


Drinking water treatment plants (DWTPs) are designed to remove physical, chemical, and biological contaminants. However, until recently, the role of DWTPs in minimizing the cycling of antibiotic resistance determinants has got limited attention. In particular, the risk of selecting antibiotic-resistant bacteria (ARB) is largely overlooked in chlorine-free DWTPs where biological processes are applied. Here, we combined high-throughput quantitative PCR and metagenomics to analyze the abundance and dynamics of microbial communities, antibiotic resistance genes (ARGs), and mobile genetic elements (MGEs) across the treatment trains of two chlorine-free DWTPs involving dune-based and reservoir-based systems. The microbial diversity of the water increased after all biological unit operations, namely rapid and slow sand filtration (SSF), and granular activated carbon filtration. Both DWTPs reduced the concentration of ARGs and MGEs in the water by circa 2.5 log gene copies mL-1, despite their relative increase in the disinfection sub-units (SSF in dune-based and UV treatment in reservoir-based DWTPs). The total microbial concentration was also reduced (2.5 log units), and none of the DWTPs enriched for bacteria containing genes linked to antibiotic resistance. Our findings highlight the effectiveness of chlorine-free DWTPs in supplying safe drinking water while reducing the concentration of antibiotic resistance determinants. To the best of our knowledge, this is the first study that monitors the presence and dynamics of antibiotic resistance determinants in chlorine-free DWTPs.

[Source](https://www.sciencedirect.com/science/article/pii/S004313542300341X)

10.1016/j.watres.2023.119905

---

## https://t.co/bDnLZQ4TTR

[Source](https://t.co/bDnLZQ4TTR)

---

## Acquisition of Antibody Against Cytomegalovirus After Kidney Transplantation in Seronegative Recipients.
 15.04.2023, _S Ishikawa, M Tasaki, K Saito, Y Nakagawa, M Ikeda, K Takahashi, Y Tomita_


Cytomegalovirus (CMV) infection is one of the most important infectious diseases affecting recipients of kidney transplantation (KTx). However, the timing of seroconversion for CMV infection in seronegative recipients remains unclear. We evaluated CMV infections in CMV-seronegative recipients and the time to acquire antibodies against CMV.

[Source](https://twitter.com/FrancescCorbera/status/1640652069265629184?ref_src=twsrc%5Etfw)

10.1016/j.transproceed.2023.03.007

---

## March 28, 2023

[Source](https://twitter.com/FrancescCorbera/status/1640652069265629184?ref_src=twsrc%5Etfw)

---

## Probiotics of <em>Lacticaseibacillus paracasei</em> SD1 and<em> Lacticaseibacillus rhamnosus </em>SD11 attenuate inflammation and Î²-cell death in streptozotocin-induced type 1 diabetic mice

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0284303)

---

## Antibiotic treatment using amoxicillin-clavulanic acid impairs gut mycobiota development through modification of the bacterial ecosystem.
 13.04.2023, _M Spatz, G Da Costa, R Ventin-Holmberg, J Planchais, C Michaudel, Y Wang, C Danne, A Lapiere, ML Michel, KL Kolho, P Langella, H Sokol, ML Richard_


_Humans, Mice, Animals, Amoxicillin-Potassium Clavulanate Combination, Anti-Bacterial Agents, Gastrointestinal Tract, Microbiota, Fungi, Bacteria_
Effects of antibiotics on gut bacteria have been widely studied, but very little is known about the consequences of such treatments on the fungal microbiota (mycobiota). It is commonly believed that fungal load increases in the gastrointestinal tract following antibiotic treatment, but better characterization is clearly needed of how antibiotics directly or indirectly affect the mycobiota and thus the entire microbiota.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01516-y)

10.1186/s40168-023-01516-y

---

## Experimental Factors Influence Diversity Metrics of the Gut Microbiome in Laboratory Mice

[Source](https://www.biorxiv.org/content/10.1101/2023.04.08.536089v1)

---

## @FZMarques

[Source](https://twitter.com/FZMarques?ref_src=twsrc%5Etfw)

---

## @biorxiv

[Source](https://twitter.com/biorxiv?ref_src=twsrc%5Etfw)

---

## https://t.co/fE1pkVNfya

[Source](https://t.co/fE1pkVNfya)

---

## April 12, 2023

[Source](https://twitter.com/rikeishm/status/1645986663292870659?ref_src=twsrc%5Etfw)

---

## Expanding known viral diversity in the healthy infant gut

[Source](https://www.nature.com/articles/s41564-023-01345-7)

---

## Large-scale phage cultivation for commensal human gut bacteria.
 13.04.2023, _J Shen, J Zhang, L Mo, Y Li, Y Li, C Li, X Kuang, Z Tao, Z Qu, L Wu, J Chen, S Liu, L Zeng, Z He, Z Chen, Y Deng, T Zhang, B Li, L Dai, Y Ma_


Phages are highly abundant in the human gut, yet most of them remain uncultured. Here, we present a gut phage isolate collection (GPIC) containing 209 phages for 42 commensal human gut bacterial species. Genome analysis of the phages identified 34 undescribed genera. We discovered 22 phages from the Salasmaviridae family that have small genomes (âˆ¼10-20 kbp) and infect Gram-positive bacteria. Two phages from a candidate family, Paboviridae, with high prevalence in the human gut were also identified. Infection assays showed that Bacteroides and Parabacteroides phages are specific to a bacterial species, and strains of the same species also exhibit substantial variations in phage susceptibility. A cocktail of 8 phages with a broad host range for Bacteroides fragilis strains effectively reduced their abundance in complex host-derived communities in vitro. Our study expands the diversity of cultured human gut bacterial phages and provides a valuable resource for human microbiome engineering.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(23)00117-8)

10.1016/j.chom.2023.03.013

---

## https://t.co/ulVv8lwvXt

[Source](https://t.co/ulVv8lwvXt)

---

## Trends in SARS-CoV-2 cycle threshold values in the Czech Republic from April 2020 to April 2022.
 15.04.2023, _D Musalkova, L Piherova, O Kwasny, Z Dindova, L Stancik, H Hartmannova, O Slama, P Peckova, J Pargac, G Minarik, T Zima, AJ Bleyer, M Radina, M Pohludka, S Kmoch_


The inability to predict the evolution of the COVID-19 epidemic hampered abilities to respond to the crisis effectively. The cycle threshold (Ct) from the standard SARS-CoV-2 quantitative reverse transcription-PCR (RT-qPCR) clinical assay is inversely proportional to the amount of SARS-CoV-2 RNA in the sample. We were interested to see if population Ct values could predict future increases in COVID-19 cases as well as subgroups that would be more likely to be affected. This information would have been extremely helpful early in the COVID-19 epidemic. We therefore conducted a retrospective analysis of demographic data and Ct values from 2,076,887 nasopharyngeal swab RT-qPCR tests that were performed at a single diagnostic laboratory in the Czech Republic from April 2020 to April 2022 and from 221,671 tests that were performed as a part of a mandatory school surveillance testing program from March 2021 to March 2022. We found that Ct values could be helpful predictive tools in the real-time management of viral epidemics. First, early measurement of Ct values would have indicated the low viral load in children, equivalent viral load in males and females, and higher viral load in older individuals. Second, rising or falling median Ct values and differences in Ct distribution indicated changes in the transmission in the population. Third, monitoring Ct values and positivity rates would have provided early evidence as to whether prevention measures are effective. Health system authorities should thus consider collecting weekly median Ct values of positively tested samples from major diagnostic laboratories for regional epidemic surveillance.

[Source](https://twitter.com/leidai_CAS/status/1646359030208790529?ref_src=twsrc%5Etfw)

10.1038/s41598-023-32953-2

---

## April 13, 2023

[Source](https://twitter.com/leidai_CAS/status/1646359030208790529?ref_src=twsrc%5Etfw)

---

## The effect of storage conditions on microbial communities in stool.
 09.04.2020, _K Nel Van Zyl, AC Whitelaw, M Newton-Foot_


_Child, Preschool, DNA, Bacterial, Enterobacteriaceae, Feces, Fungi, Humans, Microbiota, RNA, Ribosomal, 16S, Specimen Handling_
Microbiome research has experienced a surge of interest in recent years due to the advances and reduced cost of next-generation sequencing technology. The production of high quality and comparable data is dependent on proper sample collection and storage and should be standardized as far as possible. However, this becomes challenging when samples are collected in the field, especially in resource-limited settings. We investigated the impact of different stool storage methods common to the TB-CHAMP clinical trial on the microbial communities in stool. Ten stool samples were subjected to DNA extraction after 48-hour storage at -80Â°C, room temperature and in a cooler-box, as well as immediate DNA extraction. Three stool DNA extraction kits were evaluated based on DNA yield and quality. Quantitative PCR was performed to determine the relative abundance of the two major gut phyla Bacteroidetes and Firmicutes, and other representative microbial groups. The bacterial populations in the frozen group closely resembled the immediate extraction group, supporting previous findings that storage at -80Â°C is equivalent to the gold standard of immediate DNA extraction. More variation was seen in the room temperature and cooler-box groups, which may be due to the growth temperature preferences of certain bacterial populations. However, for most bacterial populations, no significant differences were found between the storage groups. As seen in other microbiome studies, the variation between participant samples was greater than that related to differences in storage. We determined that the risk of introducing bias to microbial community profiling through differences in storage will likely be minimal in our setting.

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2023.1094800/full)

10.1371/journal.pone.0227486

---

## Reducing bias in microbiome research: Comparing methods from sample collection to sequencing

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2023.1094800/full)

---

## In Vitro Modelling of Oral Microbial Invasion in the Human Colon.
 15.04.2023, _L Etienne-Mesmin, V Meslier, O Uriot, E Fournier, C Deschamps, S Denis, A David, S Jegou, C Morabito, B Quinquis, F Thirion, F Plaza OÃ±ate, E Le Chatelier, SD Ehrlich, S Blanquet-Diot, M Almeida_


Recent advances in the human microbiome characterization have revealed significant oral microbial detection in stools of dysbiotic patients. However, little is known about the potential interactions of these invasive oral microorganisms with commensal intestinal microbiota and the host. In this proof-of-concept study, we proposed a new model of oral-to-gut invasion by the combined use of an in vitro model simulating both the physicochemical and microbial (lumen- and mucus-associated microbes) parameters of the human colon (M-ARCOL), a salivary enrichment protocol, and whole-metagenome shotgun sequencing. Oral invasion of the intestinal microbiota was simulated by injection of enriched saliva in the in vitro colon model inoculated with a fecal sample from the same healthy adult donor. The mucosal compartment of M-ARCOL was able to retain the highest species richness levels over time, while species richness levels decreased in the luminal compartment. This study also showed that oral microorganisms preferably colonized the mucosal microenvironment, suggesting potential oral-to-intestinal mucosal competitions. This new model of oral-to-gut invasion can provide useful mechanistic insights into the role of oral microbiome in various disease processes. IMPORTANCE Here, we propose a new model of oral-to-gut invasion by the combined use of an in vitro model simulating both the physicochemical and microbial (lumen- and mucus-associated microbes) parameters of the human colon (M-ARCOL), a salivary enrichment protocol, and whole-metagenome shotgun sequencing. Our study revealed the importance of integrating the mucus compartment, which retained higher microbial richness during fermentation, showed the preference of oral microbial invaders for the mucosal resources, and indicated potential oral-to-intestinal mucosal competitions. It also underlined promising opportunities to further understand mechanisms of oral invasion into the human gut microbiome, define microbe-microbe and mucus-microbe interactions in a compartmentalized fashion, and help to better characterize the potential of oral microbial invasion and their persistence in the gut.

[Source](https://journals.asm.org/doi/pdf/10.1128/spectrum.04344-22)

10.1128/spectrum.04344-22

---

## Improved eukaryotic detection compatible with large-scale automated analysis of metagenomes.
 13.04.2023, _W Bazant, AS Blevins, K Crouch, DP Beiting_


_Humans, Metagenome, Eukaryota, Microbiota, Bacteria, Archaea, Metagenomics_
Eukaryotes such as fungi and protists frequently accompany bacteria and archaea in microbial communities. Unfortunately, their presence is difficult to study with "shotgun" metagenomic sequencing since prokaryotic signals dominate in most environments. Recent methods for eukaryotic detection use eukaryote-specific marker genes, but they do not incorporate strategies to handle the presence of eukaryotes that are not represented in the reference marker gene set, and they are not compatible with web-based tools for downstream analysis.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01505-1)

10.1186/s40168-023-01505-1

---

## @MicrobiomDigest

[Source](https://twitter.com/MicrobiomDigest?ref_src=twsrc%5Etfw)

---

## @satyx

[Source](https://twitter.com/satyx?ref_src=twsrc%5Etfw)

---

## #DataRadicals

[Source](https://twitter.com/hashtag/DataRadicals?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/M55fYKvLXY

[Source](https://t.co/M55fYKvLXY)

---

## #Podcasts

[Source](https://twitter.com/hashtag/Podcasts?src=hash&ref_src=twsrc%5Etfw)

---

## pic.twitter.com/NIjzhFNH37

[Source](https://t.co/NIjzhFNH37)

---

## Timing of seasonal influenza epidemics for 25 countries in Africa during 2010-19: a retrospective analysis.
 15.04.2023, _LS Igboh, K Roguski, P Marcenac, GO Emukule, MD Charles, S Tempia, B Herring, K Vandemaele, A Moen, SJ Olsen, DE Wentworth, R Kondor, JA Mott, S Hirve, JS Bresee, P Mangtani, P Nguipdop-Djomo, E Azziz-Baumgartner_


Using country-specific surveillance data to describe influenza epidemic activity could inform decisions on the timing of influenza vaccination. We analysed surveillance data from African countries to characterise the timing of seasonal influenza epidemics to inform national vaccination strategies.

[Source](https://twitter.com/Alation/status/1646185262597242887?ref_src=twsrc%5Etfw)

10.1016/S2214-109X(23)00109-2

---

## April 12, 2023

[Source](https://twitter.com/Alation/status/1646185262597242887?ref_src=twsrc%5Etfw)

---

