## Vascular endothelial injury exacerbates coronavirus disease 2019: The role of endothelial glycocalyx protection.
 13.08.2020, _H Okada, S Yoshida, A Hara, S Ogura, H Tomita_


The potential for a rapid increase in severity is among the most frightening aspects of severe acute respiratory syndrome coronavirus 2 infection. Evidence increasingly suggests that the symptoms of coronavirus disease-2019 (COVID-19)-related acute respiratory distress syndrome (ARDS) differ from those of classic ARDS. Recently, the severity of COVID-19 has been attributed to a systemic, thrombotic, and inflammatory disease that damages not only the lungs but multiple organs, including the heart, brain, toes, and liver. This systemic form of COVID-19 may be due to inflammation and vascular endothelial cell injury. The vascular endothelial glycocalyx comprises glycoproteins, and plays an important role in systemic capillary homeostasis maintenance. The glycocalyx covers the entire vascular endothelium, and its thickness varies among organs. The endothelial glycocalyx is very thin in the pulmonary capillaries, where it is affected by gaseous exchange with the alveoli and the low intravascular pressure in the pulmonary circulation. Despite the clearly important roles of the glycocalyx in vascular endothelial injury, thrombosis, vasculitis, and inflammation, the link between this structure and vascular endothelial cell dysfunction in COVID-19 remains unclear. In this prospective review, we summarize the importance of the glycocalyx and its potential as a therapeutic target in cases of systemic COVID-19.

[Source](https://academic.oup.com/cid/advance-article/doi/10.1093/cid/ciaa1191/5891289)

10.1111/micc.12654

---

## Gut microbiota and COVID-19: A superfluous diagnostic biomarker or therapeutic target?

[Source](https://academic.oup.com/cid/advance-article/doi/10.1093/cid/ciaa1191/5891289)

---

## Engineered Live Biotherapeutics: Progress and Challenges.
 09.08.2020, _Y Tan, J Shen, T Si, CL Ho, Y Li, L Dai_


The human microbiome plays an important role in human health, from metabolism to immunity. In the last few decades, advances in synthetic biology have enabled scientists to design and engineer live microorganisms for therapeutic purposes. In this review, we outline major strategies for manipulating the microbiome, which include three emerging areas with promising therapeutic applications: engineered commensal bacteria, synthetic microbial consortia and targeted modulation by phages. Furthermore, we highlight the applications of engineered live biotherapeutics in treating a variety of human diseases, including pathogenic infections, metabolic disorders, inflammatory bowel disease and colorectal cancer. Finally, we provide an overview of the challenges and opportunities in the future development of engineered live biotherapeutics. This article is protected by copyright. All rights reserved.

[Source](https://onlinelibrary.wiley.com/doi/abs/10.1002/biot.202000155)

10.1002/biot.202000155

---

## Gut microbial co-abundance networks show specificity in inflammatory bowel disease and obesity.
 12.08.2020, _L Chen, V Collij, M Jaeger, ICL van den Munckhof, A Vich Vila, A Kurilshikov, R Gacesa, T Sinha, M Oosting, LAB Joosten, JHW Rutten, NP Riksen, RJ Xavier, F Kuipers, C Wijmenga, A Zhernakova, MG Netea, RK Weersma, J Fu_


The gut microbiome is an ecosystem that involves complex interactions. Currently, our knowledge about the role of the gut microbiome in health and disease relies mainly on differential microbial abundance, and little is known about the role of microbial interactions in the context of human disease. Here, we construct and compare microbial co-abundance networks using 2,379 metagenomes from four human cohorts: an inflammatory bowel disease (IBD) cohort, an obese cohort and two population-based cohorts. We find that the strengths of 38.6% of species co-abundances and 64.3% of pathway co-abundances vary significantly between cohorts, with 113 species and 1,050 pathway co-abundances showing IBD-specific effects and 281 pathway co-abundances showing obesity-specific effects. We can also replicate these IBD microbial co-abundances in longitudinal data from the IBD cohort of the integrative human microbiome (iHMP-IBD) project. Our study identifies several key species and pathways in IBD and obesity and provides evidence that altered microbial abundances in disease can influence their co-abundance relationship, which expands our current knowledge regarding microbial dysbiosis in disease.

[Source](https://www.nature.com/articles/s41467-020-17840-y?)

10.1038/s41467-020-17840-y

---

## Multi-omics data integration in anorexia nervosa patients before and after weight regain: A microbiome-metabolomics investigation.
 12.08.2020, _AM Monteleone, J Troisi, A Fasano, R Dalle Grave, F Marciello, G Serena, S Calugi, G Scala, G Corrivetti, G Cascino, P Monteleone, M Maj_


We have recently reported specific fecal metabolomic changes in acute and short-term weight restored patients with anorexia nervosa (AN). In this study we explored the association between those metabolomic changes and patients' gut microbiome composition.

[Source](https://www.sciencedirect.com/science/article/pii/S0261561420303861?via%3Dihub)

10.1016/j.clnu.2020.07.021

---

## Gastric Microbiome Diversities in Gastric Cancer Patients from Europe and Asia Mimic the Human Population Structure and Are Partly Driven by Microbiome Quantitative Trait Loci.
 12.08.2020, _B Cavadas, R Camacho, JC Ferreira, RM Ferreira, C Figueiredo, A Brazma, NA Fonseca, L Pereira_


The human gastrointestinal tract harbors approximately 100 trillion microorganisms with different microbial compositions across geographic locations. In this work, we used RNASeq data from stomach samples of non-disease (164 individuals from European ancestry) and gastric cancer patients (137 from Europe and Asia) from public databases. Although these data were intended to characterize the human expression profiles, they allowed for a reliable inference of the microbiome composition, as confirmed from measures such as the genus coverage, richness and evenness. The microbiome diversity (weighted UniFrac distances) in gastric cancer mimics host diversity across the world, with European gastric microbiome profiles clustering together, distinct from Asian ones. Despite the confirmed loss of microbiome diversity from a healthy status to a cancer status, the structured profile was still recognized in the disease condition. In concordance with the parallel host-bacteria population structure, we found 16 human loci (non-synonymous variants) in the European-descendent cohorts that were significantly associated with specific genera abundance. These microbiome quantitative trait loci display heterogeneity between population groups, being mainly linked to the immune system or cellular features that may play a role in enabling microbe colonization and inflammation.

[Source](https://www.mdpi.com/2076-2607/8/8/1196)

10.3390/microorganisms8081196

---

## Faecal microbiota signatures of IBD and their relation to diagnosis, disease phenotype, inflammation, treatment escalation and anti-TNF response in a European Multicentre Study (IBD-Character) 

[Source](https://www.tandfonline.com/doi/full/10.1080/00365521.2020.1803396)

---

## A randomized crossover trial on the effect of plant-based compared with animal-based meat on trimethylamine-N-oxide and cardiovascular disease risk factors in generally healthy adults: Study With Appetizing Plantfood-Meat Eating Alternative Trial (SWAP-MEAT).
 11.08.2020, _A Crimarco, S Springfield, C Petlura, T Streaty, K Cunanan, J Lee, P Fielding-Singh, MM Carter, MA Topf, HC Wastyk, ED Sonnenburg, JL Sonnenburg, CD Gardner_


Despite the rising popularity of plant-based alternative meats, there is limited evidence of the health effects of these products.

[Source](https://academic.oup.com/ajcn/advance-article/doi/10.1093/ajcn/nqaa203/5890315)

10.1093/ajcn/nqaa203

---

## Whole-blood transcriptome profiling reveals signatures of metformin and its therapeutic response.
 11.08.2020, _M Ustinova, L Ansone, I Silamikelis, V Rovite, I Elbere, L Silamikele, I Kalnina, D Fridmanis, J Sokolovska, I Konrade, V Pirags, J Klovins_


Metformin, a biguanide agent, is the first-line treatment for type 2 diabetes mellitus due to its glucose-lowering effect. Despite its wide application in the treatment of multiple health conditions, the glycemic response to metformin is highly variable, emphasizing the need for reliable biomarkers. We chose the RNA-Seq-based comparative transcriptomics approach to evaluate the systemic effect of metformin and highlight potential predictive biomarkers of metformin response in drug-naïve volunteers with type 2 diabetes in vivo. The longitudinal blood-derived transcriptome analysis revealed metformin-induced differential expression of novel and previously described genes involved in cholesterol homeostasis (SLC46A1 and LRP1), cancer development (CYP1B1, STAB1, CCR2, TMEM176B), and immune responses (CD14, CD163) after administration of metformin for three months. We demonstrate for the first time a transcriptome-based molecular discrimination between metformin responders (delta HbA1c ≥ 1% or 12.6 mmol/mol) and non-responders (delta HbA1c &lt; 1% or 12.6 mmol/mol), that is determined by expression levels of 56 genes, explaining 13.9% of the variance in the therapeutic efficacy of the drug. Moreover, we found a significant upregulation of IRS2 gene (log2FC 0.89) in responders compared to non-responders before the use of metformin. Finally, we provide evidence for the mitochondrial respiratory complex I as one of the factors related to the high variability of the therapeutic response to metformin in patients with type 2 diabetes mellitus.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237400)

10.1371/journal.pone.0237400

---

## Microbial Profile During Pericoronitis and Microbiota Shift After Treatment

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.01888/full)

---

## Unraveling the camel rumen microbiome through metaculturomics approach for agriculture waste hydrolytic potential.
 10.08.2020, _S Srivastava, NA Dafale, SJ Jakhesara, CG Joshi, NV Patil, HJ Purohit_


Cellulose is the most abundant natural polymer present on Earth in the form of agriculture waste. Hydrolysis of agriculture waste for simple fermentable reducing sugars is the bottleneck in the area of biofuel generation and other value-added products. The present study aims to utilize the camel rumen as a bioreactor for potent cellulolytic and hemicellulolytic bacteria by altering the feed types with varying cellulosic concentrations. A total of 6716 bacterial cultures were subjected to three layers of screening, where plate zymography and chromophoric substrate screening served as primary screening method for cellulolytic and hemicellulolytic potential. The potential isolates were genetically grouped using RAPD, and 51 representative isolates from each group were subjected to molecular identification through 16S rDNA sequencing, followed by quantification of various cellulolytic and hemicellulolytic enzymes. Out of 51 potent isolates, 5 isolates had high endoglucanase activity ranging from 0.3 to 0.48 U/ml. The selected five key isolates identified as Pseudomonas, Paenibacillus, Citrobacter, Bacillus subtilis, and Enterobacter were employed for hydrolyzing the various agriculture residues and resulted in approximately 0.4 mg/ml of reducing sugar. Furthermore, the metaculturomics approach was implemented to deduce the total cultured diversity through 16S rRNA amplicon library sequencing. The metaculturomics data revealed the dominance of proteobacteria and unidentified bacterial population in all four feed types, which indicates the possibility of culturing novel cellulose-deconstructing bacteria. Moreover, the presence of diverse hydrolytic enzymes in cultured isolates supports the usage of these bacteria in bio-processing of agriculture waste residues and obtaining the biofuels and other value-added products.

[Source](https://link.springer.com/article/10.1007/s00203-020-02010-x)

10.1007/s00203-020-02010-x

---

## Bacterial polyphosphates interfere with the innate host defense to infection.
 13.08.2020, _J Roewe, G Stavrides, M Strueve, A Sharma, F Marini, A Mann, SA Smith, Z Kaya, B Strobl, M Mueller, C Reinhardt, JH Morrissey, M Bosmann_


Polyphosphates are linear polymers and ubiquitous metabolites. Bacterial polyphosphates are long chains of hundreds of phosphate units. Here, we report that mouse survival of peritoneal Escherichia coli sepsis is compromised by long-chain polyphosphates, and improves with bacterial polyphosphatekinase deficiency or neutralization using recombinant exopolyphosphatase. Polyphosphate activities are chain-length dependent, impair pathogen clearance, antagonize phagocyte recruitment, diminish phagocytosis and decrease production of iNOS and cytokines. Macrophages bind and internalize polyphosphates, in which their effects are independent of P2Y1 and RAGE receptors. The M1 polarization driven by E. coli derived LPS is misdirected by polyphosphates in favor of an M2 resembling phenotype. Long-chain polyphosphates modulate the expression of more than 1800 LPS/TLR4-regulated genes in macrophages. This interference includes suppression of hundreds of type I interferon-regulated genes due to lower interferon production and responsiveness, blunted STAT1 phosphorylation and reduced MHCII expression. In conclusion, prokaryotic polyphosphates disturb multiple macrophage functions for evading host immunity.

[Source](https://www.nature.com/articles/s41467-020-17639-x?)

10.1038/s41467-020-17639-x

---

## Prominent members of the human gut microbiota express endo-acting O-glycanases to initiate mucin breakdown.
 12.08.2020, _LI Crouch, MV Liberato, PA Urbanowicz, A Baslé, CA Lamb, CJ Stewart, K Cooke, M Doona, S Needham, RR Brady, JE Berrington, K Madunic, M Wuhrer, P Chater, JP Pearson, R Glowacki, EC Martens, F Zhang, RJ Linhardt, DIR Spencer, DN Bolam_


The thick mucus layer of the gut provides a barrier to infiltration of the underlying epithelia by both the normal microbiota and enteric pathogens. Some members of the microbiota utilise mucin glycoproteins as a nutrient source, but a detailed understanding of the mechanisms used to breakdown these complex macromolecules is lacking. Here we describe the discovery and characterisation of endo-acting enzymes from prominent mucin-degrading bacteria that target the polyLacNAc structures within oligosaccharide side chains of both animal and human mucins. These O-glycanases are part of the large and diverse glycoside hydrolase 16 (GH16) family and are often lipoproteins, indicating that they are surface located and thus likely involved in the initial step in mucin breakdown. These data provide a significant advance in our knowledge of the mechanism of mucin breakdown by the normal microbiota. Furthermore, we also demonstrate the potential use of these enzymes as tools to explore changes in O-glycan structure in a number of intestinal disease states.

[Source](https://www.nature.com/articles/s41467-020-17847-5?)

10.1038/s41467-020-17847-5

---

## Nostoc sphaeroids Kütz polysaccharide and powder enrich a core bacterial community on C57BL/6j mice

[Source](https://www.sciencedirect.com/science/article/pii/S0141813020341131)

---

## Maternal dietary omega-3 deficiency worsens the deleterious effects of prenatal inflammation on the gut-brain axis in the offspring across lifetime.
 11.08.2020, _Q Leyrolle, F Decoeur, G Briere, C Amadieu, ARAA Quadros, I Voytyuk, C Lacabanne, A Benmamar-Badel, J Bourel, A Aubert, A Sere, F Chain, L Schwendimann, B Matrot, T Bourgeois, S Grégoire, JG Leblanc, A De Moreno De Leblanc, P Langella, GR Fernandes, L Bretillon, C Joffre, R Uricaru, P Thebault, P Gressens, JM Chatel, S Layé, A Nadjar_


Maternal immune activation (MIA) and poor maternal nutritional habits are risk factors for the occurrence of neurodevelopmental disorders (NDD). Human studies show the deleterious impact of prenatal inflammation and low n-3 polyunsaturated fatty acid (PUFA) intake on neurodevelopment with long-lasting consequences on behavior. However, the mechanisms linking maternal nutritional status to MIA are still unclear, despite their relevance to the etiology of NDD. We demonstrate here that low maternal n-3 PUFA intake worsens MIA-induced early gut dysfunction, including modification of gut microbiota composition and higher local inflammatory reactivity. These deficits correlate with alterations of microglia-neuron crosstalk pathways and have long-lasting effects, both at transcriptional and behavioral levels. This work highlights the perinatal period as a critical time window, especially regarding the role of the gut-brain axis in neurodevelopment, elucidating the link between MIA, poor nutritional habits, and NDD. Fig. 1 EFFECT OF N-3 PUFA DEFICIENCY ON MIA-INDUCED BEHAVIORAL DEFICITS IN NEONATES AND IN ADULT OFFSPRING.: All graphs show Means ± SEM. a Experimental setup. b Average time spent by pups to achieve the Fox battery tests (negative geotaxis and righting reflex; 3 trials per day from PND4 to PND6). N = 14-19. Two-way ANOVA: MIA effect, F(1,62) = 11.67, p = 0.0011. c Average vocalization time (15-min sessions at PND7-8). N = 14-19. Kruskal-Wallis test followed by Mann-Whitney comparison; n-3 sufficient-Saline vs n-3 sufficient-LPS, **p &lt; 0.01. d Neonate average locomotion measured as the distance traveled (in cm/min) during 1 min-session from PND5 to PND8. N = 14-19. Kruskal-Wallis test followed by Mann-Whitney comparison; n-3 sufficient-Saline vs n-3 deficient-saline, **p = 0.009, n-3 deficient-Saline vs n-3 deficient-LPS, ***p &lt; 0.001. e Time course of locomotor activity of newborns from PND5 to PND8. N = 14-19. Two-way ANOVA on repeated measures followed by Bonferroni's multiple comparisons test: n-3 deficient-Saline vs n-3 deficient-LPS, *p = 0.02. f Time course of the distance traveled in the Morris Water Maze during the learning phase (in cm). N = 10. Two-way ANOVA on repeated measures: diet effect, F(1,36) = 9.22, p = 0.004. g Percentage of time spent in the target quadrant. N = 10. One-sample t test; n-3 sufficient-Saline, ***p &lt; 0.001; n-3 sufficient-LPS, **p = 0.004; n-3 deficient-saline, ***p &lt; 0.001; n-3 deficient-LPS, p = 0.11. h. Time course of the distance traveled in the Morris Water Maze during the reversal learning phase (in cm). N = 10. Two-way ANOVA on repeated measures: MIA effect, F(1,36) = 3.27, p = 0.008; time effect, F(1,36) = 19, p &lt; 0.001. i Basal locomotor activity (in cm). N = 12. Kruskal-Wallis test followed by Mann-Whitney comparison; n-3 deficient-Saline vs n-3 deficient-LPS, **p = 0.008. j Time spent in the light box (anxiogenic area) of the light-dark test. N = 11. Kruskal-Wallis test followed by Mann-Whitney comparison. k Percentage of time spent in the center of the open-field arena (anxiogenic area). N = 8-11. Kruskal-Wallis test followed by Mann-Whitney comparison. Fig. 2 DIETARY N-3 PUFA DEFICIENCY EXACERBATES MIA-INDUCED ALTERATIONS OF THE HIPPOCAMPAL LIPID AND TRANSCRIPTIONAL PROFILES IN ADULTHOOD.: Quantification of the levels of total PUFAs (a), n-6 PUFAs (b), DTA n-6 (c) or DPAn-6 (d) in the hippocampus of adult mice, expressed as the percentage of total fatty acids. All graphs show Means ± SEM. N = 6. Two-way ANOVA followed by Bonferroni's multiple comparisons test: Total PUFAs: n-3 sufficient-Saline vs n-3 deficient-Saline, ***p = 0.0001; n-3 deficient-Saline vs n-3 deficient-LPS, *p = 0.0156; n-3 deficient-Saline vs n-3 sufficient-LPS, ***p = 0.0003. Total n-6 PUFAs: n-3 sufficient-Saline vs n-3 deficient-Saline, ***p &lt; 0.0001; n-3 deficient-Saline vs n-3 deficient-LPS, **p = 0.0055; n-3 deficient-Saline vs n-3 sufficient-LPS, ***p &lt; 0.0001; n-3 sufficient-Saline vs n-3 deficient LPS, ***p &lt; 0.0001; n-3 deficient-LPS vs n-3 sufficient-LPS, ***p &lt; 0.0001. DTA n-6: n-3 sufficient-Saline vs n-3 deficient-Saline, ***p &lt; 0.0001; n-3 deficient-Saline vs n-3 deficient-LPS, **p = 0.0013; n-3 deficient-Saline vs n-3 sufficient-LPS, ***p &lt; 0.0001; n-3 sufficient-Saline vs n-3 deficient LPS, ***p = 0.0004; n-3 deficient-LPS vs n-3 sufficient-LPS, *p = 0.0158. DPA n-6: n-3 sufficient-Saline vs n-3 deficient-Saline, ***p &lt; 0.0001; n-3 deficient-Saline vs n-3 deficient-LPS, **p = 0.0045; n-3 deficient-Saline vs n-3 sufficient-LPS, ***p &lt; 0.0001; n-3 sufficient-Saline vs n-3 deficient LPS, ***p &lt; 0.0001; n-3 deficient-LPS vs n-3 sufficient-LPS, ***p &lt; 0.0001. e Venn diagram highlighting the number of genes that were modulated by MIA in the hippocampi of adult n-3 sufficient (blue) or n-3 deficient (red) mice. Lower panel: Number of genes that were up- or down-regulated in n-3 sufficient and n-3 deficient mice. Representation of the 20 most significantly dysregulated genes in n-3 sufficient (f) and n-3 deficient (g) mice. Genes that appear in both n-3 sufficient and n-3 deficient mice are bold. h PCA analysis of MIA-induced differentially expressed genes (DEG) in both dietary groups. Confidence ellipses appear around each group. i, j Gene Ontology analysis of DEGs (light red and blue: up-regulated genes; dark red and blue: down-regulated genes). Fig. 3 EFFECT OF N-3 PUFA DEFICIENCY AND MIA ON MICROGLIA-NEURON CROSSTALK PATHWAYS, SPINE DENSITY, OLIGODENDROCYTE AND MYELIN PROTEIN EXPRESSION.: All graphs show Means ± SEM. a Colocalization of Iba-1 and PSD95 proteins immunoreactivity in the CA1 region of the hippocampus of PND14 pups. Representative confocal image of Iba-1 (green) PSD95 (red) costaining (Top panel: scale bar = 10 µm) and Imaris 3D reconstruction (Bottom panel, scale bar = 1 µm). N = 72-122. Kruskal-Wallis test followed by Mann-Whitney comparisons; n-3 deficient-Saline vs n-3 sufficient-Saline, ***p &lt; 0.0001; n-3 deficient-LPS vs n-3 sufficient-LPS, ***p &lt; 0.0001. b qRT-PCR quantification of microglia-neuron interaction mRNA markers in the hippocampus of PND14 mice (data normalized to the saline group, dotted line). N = 4-6. Kruskal-Wallis test followed by Mann-Whitney comparisons; *p &lt; 0.05, **p &lt; 0.01 (all comparisons in Table S2). c Quantification and representative images of Golgi staining spine density in the CA1 region of the hippocampus at PND28. N = 8-21. Kruskal-Wallis test followed by Mann-Whitney comparisons; n-3 deficient-Saline vs n-3 deficient-LPS, ***p &lt; 0.0001; n-3 deficient-Saline vs n-3 sufficient-Saline, **p = 0.001; n-3 deficient-LPS vs n-3 sufficient-LPS, *p = 0.025. d Western blot-based quantification and representative images of PSD95 protein expression in the hippocampus of PND28 mice. N = 4-8. Kruskal-Wallis test followed by Mann-Whitney comparisons; n-3 deficient-Saline vs n-3 deficient-LPS, **p = 0.004; n-3 deficient-LPS vs n-3 sufficient-LPS, *p = 0.03. Quantification of Olig2 (e), PLP (f), APC (g), MAG (h) and MBP (i) immunoreactivity in the hippocampus of PND14 mice. N = 4-7. Two-way ANOVA. Olig2: diet effect, F(1,20) = 3.48, p = 0.08; MIA effect, F(1,20) = 4.78, p = 0.041. PLP: MIA effect, F(1,22) = 5.01, p = 0.036. APC: diet effect, F(1,17) = 4.96, p = 0.0397. Fig. 4 EFFECT OF N-3 PUFA DEFICIENCY AND MIA ON GUT MICROBIOTA COMPOSITION AT PND14 AND PND21.: All graphs show Means ± SEM. a 16S rRNA-sequencing-based alpha diversity analysis of the microbiota, measured by Shannon index in PND14 mice. N = 8-12. Two-way ANOVA: diet effect, F(1,39) = 12.76, p &lt; 0.001; MIA effect, F(1,39) = 5.39, p = 0.026. Bacteria phyla (b) and family (c) observed in all experimental groups at PND14. d PCA of all subjects at PND14. Confidence ellipses appear around each group. e 16S rRNA-sequencing-based alpha diversity analysis, measured by Shannon index in PND21 mice. N=8-13. Two-way ANOVA: n-3 sufficient-Saline vs n-3 sufficient-LPS, ***p = 0.0005; n-3 sufficient-LPS vs n-3 deficient-LPS, *p = 0.019; n-3 sufficient-Saline vs n-3 deficient-Saline, **p = 0.0078. Bacteria phyla (f) and family (g) observed in all experimental groups at PND21. h PCA of all subjects at PND14. Confidence ellipses appear around each group. Quantification of MLN lymphocytes cytokine release measured by ELISA at PND14 (i) and PND21 (j). N = 6-15; Kruskal-Wallis test followed by Mann-Whitney comparisons; *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001 (all comparisons in Table S2). Z-score of T cells inflammatory reactivity in PND14 (k) and PND21 (l) mice. N = 7-15. Kruskal-Wallis test followed by Mann-Whitney comparisons; PND14: n-3 deficient-Saline vs n-3 deficient-LPS, ***p &lt; 0.001. PND21: n-3 sufficient-Saline vs n-3 sufficient-LPS, **p = 0.0011, n-3 sufficient-LPS vs n-3 deficient-LPS, ***p = &lt;0.0004. Fig. 5 CORRELATIONS BETWEEN MICROBIAL MODIFICATIONS, GUT INFLAMMATION, AND NEUROBIOLOGICAL PARAMETERS.: a Spearman's correlation matrix between gut immune cells reactivity (e.g cytokine release after T-cells stimulation) and bacterial genera in PND14 mice (*p = 0.05). b Spearman's correlation matrix between neurobiological measurements (PLP, Olig2, Iba-1 and MAP2) and bacterial genera in PND14 mice (*p = 0.05). c Spearman's correlation matrix between gut immune cells reactivity (e.g cytokine release after T-cells stimulation) and bacterial genera in PND21 mice (*p = 0.05). d Spearman's correlation matrix between neurobiological measurements (PLP, Olig2, Iba-1, and MAP2) and bacterial genera in PND21 mice (*p = 0.05). e Spearman's correlation between gut immune cells reactivity (e.g released cytokines after stimulation) and neurobiological parameters in PND21 mice (*p = 0.05). N = 20-22. Escherichia-Shig: Escherichia-Shigella; Eubacterium copro: Eubacterium coprostanoligenes group; Lachno NK4A136: Lachnospiraceae NK4A136 group; Lachno UCG-008: Lachnospiraceae UCG-008; Prevo UCG-001: Prevotellaceae UCG-001; Rikenellaceae RC9: Rikenellaceae RC9 gut group; Ruminococcus gg: Ruminococcus gnavus group. f Schematic summarizing the main findings. Exposure of n-3 PUFA deficient dams to MIA alters the gut microbiota composition and increases the inflammatory reactivity of the gut T-lymphocytes in the offspring during the post-natal period. This is correlated with an impairment in microglia-neuron crosstalk during this phase, with consequences on hippocampus function and memory abilities later in life.

[Source](https://www.nature.com/articles/s41386-020-00793-7)

10.1038/s41386-020-00793-7

---

## Influence of the polar light cycle on seasonal dynamics of an Antarctic lake microbial community.
 12.08.2020, _P Panwar, MA Allen, TJ Williams, AM Hancock, S Brazendale, J Bevington, S Roux, D Páez-Espino, S Nayfach, M Berg, F Schulz, IA Chen, M Huntemann, N Shapiro, NC Kyrpides, T Woyke, EA Eloe-Fadrosh, R Cavicchioli_


Cold environments dominate the Earth's biosphere and microbial activity drives ecosystem processes thereby contributing greatly to global biogeochemical cycles. Polar environments differ to all other cold environments by experiencing 24-h sunlight in summer and no sunlight in winter. The Vestfold Hills in East Antarctica contains hundreds of lakes that have evolved from a marine origin only 3000-7000 years ago. Ace Lake is a meromictic (stratified) lake from this region that has been intensively studied since the 1970s. Here, a total of 120 metagenomes representing a seasonal cycle and four summers spanning a 10-year period were analyzed to determine the effects of the polar light cycle on microbial-driven nutrient cycles.

[Source](https://link.springer.com/article/10.1186/s40168-020-00889-8)

10.1186/s40168-020-00889-8

---

## Light into the darkness: Unifying the known and unknown coding sequence space in microbiome analyses

[Source](https://www.biorxiv.org/content/10.1101/2020.06.30.180448v3.abstract?%3Fcollection=)

---

## Metagenomic next-generation sequencing of rectal swabs for the surveillance of antimicrobial-resistant organisms on the Illumina Miseq and Oxford MinION platforms.
 12.08.2020, _R Yee, FP Breitwieser, S Hao, BNA Opene, RE Workman, PD Tamma, J Dien-Bard, W Timp, PJ Simner_


Antimicrobial resistance (AMR) is a public health threat where efficient surveillance is needed to prevent outbreaks. Existing methods for detection of gastrointestinal colonization of multidrug-resistant organisms (MDRO) are limited to specific organisms or resistance mechanisms. Metagenomic next-generation sequencing (mNGS) is a more rapid and agnostic diagnostic approach for microbiome and resistome investigations. We determined if mNGS can detect MDRO from rectal swabs in concordance with standard microbiology results. We performed and compared mNGS performance on short-read Illumina MiSeq (N = 10) and long-read Nanopore MinION (N = 5) platforms directly from rectal swabs to detect vancomycin-resistant enterococci (VRE) and carbapenem-resistant Gram-negative organisms (CRO). We detected Enterococcus faecium (N = 8) and Enterococcus faecalis (N = 2) with associated van genes (9/10) in concordance with VRE culture-based results. We studied the microbiome and identified CRO, Pseudomonas aeruginosa (N = 1), Enterobacter cloacae (N = 1), and KPC-producing Klebsiella pneumoniae (N = 1). Nanopore real-time analysis detected the blaKPC gene in 2.3 min and provided genetic context (blaKPC harbored on pKPC_Kp46 IncF plasmid). Illumina sequencing provided accurate allelic variant determination (i.e., blaKPC-2) and strain typing of the K. pneumoniae (ST-15). We demonstrated an agnostic approach for surveillance of MDRO, examining advantages of both short- and long-read mNGS methods for AMR detection.

[Source](https://link.springer.com/article/10.1007%2Fs10096-020-03996-4)

10.1007/s10096-020-03996-4

---

