## agmicroPapers

[Source](https://twitter.com/AgmicroPapers)

---

## Mucosal or systemic microbiota exposures shape the B cell repertoire.
 14.09.2020, _H Li, JP Limenitakis, V Greiff, B Yilmaz, O Schären, C Urbaniak, M Zünd, MAE Lawson, ID Young, S Rupp, M Heikenwälder, KD McCoy, S Hapfelmeier, SC Ganal-Vonarburg, AJ Macpherson_


_Administration, Intravenous, Administration, Oral, Animals, B-Lymphocytes, Clostridiales, Escherichia coli, Female, Germ-Free Life, Immunity, Mucosal, Immunoglobulin A, Immunoglobulin G, Immunoglobulin Heavy Chains, Immunologic Memory, Intestinal Mucosa, Male, Mice, Mice, Inbred C57BL, Plasma Cells, Repetition Priming, Symbiosis_
Colonization by the microbiota causes a marked stimulation of B cells and induction of immunoglobulin, but mammals colonized with many taxa have highly complex and individualized immunoglobulin repertoires1,2. Here we use a simplified model of defined transient exposures to different microbial taxa in germ-free mice3 to deconstruct how the microbiota shapes the B cell pool and its functional responsiveness. We followed the development of the immunoglobulin repertoire in B cell populations, as well as single cells by deep sequencing. Microbial exposures at the intestinal mucosa generated oligoclonal responses that differed from those of germ-free mice, and from the diverse repertoire that was generated after intravenous systemic exposure to microbiota. The IgA repertoire-predominantly to cell-surface antigens-did not expand after dose escalation, whereas increased systemic exposure broadened the IgG repertoire to both microbial cytoplasmic and cell-surface antigens. These microbial exposures induced characteristic immunoglobulin heavy-chain repertoires in B cells, mainly at memory and plasma cell stages. Whereas sequential systemic exposure to different microbial taxa diversified the IgG repertoire and facilitated alternative specific responses, sequential mucosal exposure produced limited overlapping repertoires and the attrition of initial IgA binding specificities. This shows a contrast between a flexible response to systemic exposure with the need to avoid fatal sepsis, and a restricted response to mucosal exposure that reflects the generic nature of host-microbial mutualism in the mucosa.

[Source](https://www.biorxiv.org/content/10.1101/2020.11.05.370106v1)

10.1038/s41586-020-2564-6

---

## Memory shapes microbial populations

[Source](https://www.biorxiv.org/content/10.1101/2020.11.05.370106v1)

---

## Bacterial-fungal interactions revealed by genome-wide analysis of bacterial mutant fitness.
 03.11.2020, _EC Pierce, M Morin, JC Little, RB Liu, J Tannous, NP Keller, K Pogliano, BE Wolfe, LM Sanchez, RJ Dutton_


Microbial interactions are expected to be major determinants of microbiome structure and function. Although fungi are found in diverse microbiomes, their interactions with bacteria remain largely uncharacterized. In this work, we characterize interactions in 16 different bacterial-fungal pairs, examining the impacts of 8 different fungi isolated from cheese rind microbiomes on 2 bacteria (Escherichia coli and a cheese-isolated Pseudomonas psychrophila). Using random barcode transposon-site sequencing with an analysis pipeline that allows statistical comparisons between different conditions, we observed that fungal partners caused widespread changes in the fitness of bacterial mutants compared to growth alone. We found that all fungal species modulated the availability of iron and biotin to bacterial species, which suggests that these may be conserved drivers of bacterial-fungal interactions. Species-specific interactions were also uncovered, a subset of which suggested fungal antibiotic production. Changes in both conserved and species-specific interactions resulted from the deletion of a global regulator of fungal specialized metabolite production. This work highlights the potential for broad impacts of fungi on bacterial species within microbiomes.

[Source](https://www.nature.com/articles/s41564-020-00800-z)

10.1038/s41564-020-00800-z

---

## Environmental and Physiological Factors Affecting High-Throughput Measurements of Bacterial Growth.
 22.10.2020, _E Atolia, S Cesar, HA Arjes, M Rajendram, H Shi, BD Knapp, S Khare, A Aranda-Díaz, RE Lenski, KC Huang_


Bacterial growth under nutrient-rich and starvation conditions is intrinsically tied to the environmental history and physiological state of the population. While high-throughput technologies have enabled rapid analyses of mutant libraries, technical and biological challenges complicate data collection and interpretation. Here, we present a framework for the execution and analysis of growth measurements with improved accuracy over that of standard approaches. Using this framework, we demonstrate key biological insights that emerge from consideration of culturing conditions and history. We determined that quantification of the background absorbance in each well of a multiwell plate is critical for accurate measurements of maximal growth rate. Using mathematical modeling, we demonstrated that maximal growth rate is dependent on initial cell density, which distorts comparisons across strains with variable lag properties. We established a multiple-passage protocol that alleviates the substantial effects of glycerol on growth in carbon-poor media, and we tracked growth rate-mediated fitness increases observed during a long-term evolution of Escherichia coli in low glucose concentrations. Finally, we showed that growth of Bacillus subtilis in the presence of glycerol induces a long lag in the next passage due to inhibition of a large fraction of the population. Transposon mutagenesis linked this phenotype to the incorporation of glycerol into lipoteichoic acids, revealing a new role for these envelope components in resuming growth after starvation. Together, our investigations underscore the complex physiology of bacteria during bulk passaging and the importance of robust strategies to understand and quantify growth.IMPORTANCE How starved bacteria adapt and multiply under replete nutrient conditions is intimately linked to their history of previous growth, their physiological state, and the surrounding environment. While automated equipment has enabled high-throughput growth measurements, data interpretation and knowledge gaps regarding the determinants of growth kinetics complicate comparisons between strains. Here, we present a framework for growth measurements that improves accuracy and attenuates the effects of growth history. We determined that background absorbance quantification and multiple passaging cycles allow for accurate growth rate measurements even in carbon-poor media, which we used to reveal growth-rate increases during long-term laboratory evolution of Escherichia coli Using mathematical modeling, we showed that maximum growth rate depends on initial cell density. Finally, we demonstrated that growth of Bacillus subtilis with glycerol inhibits the future growth of most of the population, due to lipoteichoic acid synthesis. These studies highlight the challenges of accurate quantification of bacterial growth behaviors.

[Source](https://mbio.asm.org/content/11/5/e01378-20)

10.1128/mBio.01378-20

---

## Host variables confound gut microbiota studies of human disease.
 06.11.2020, _I Vujkovic-Cvijin, J Sklar, L Jiang, L Natarajan, R Knight, Y Belkaid_


Low concordance between studies that examine the role of microbiota in human diseases is a pervasive challenge that limits the capacity to identify causal relationships between host-associated microorganisms and pathology. The risk of obtaining false positives is exacerbated by wide interindividual heterogeneity in microbiota composition1, probably due to population-wide differences in human lifestyle and physiological variables2 that exert differential effects on the microbiota. Here we infer the greatest, generalized sources of heterogeneity in human gut microbiota profiles and also identify human lifestyle and physiological characteristics that, if not evenly matched between cases and controls, confound microbiota analyses to produce spurious microbial associations with human diseases. We identify alcohol consumption frequency and bowel movement quality as unexpectedly strong sources of gut microbiota variance that differ in distribution between healthy participants and participants with a disease and that can confound study designs. We demonstrate that for numerous prevalent, high-burden human diseases, matching cases and controls for confounding variables reduces observed differences in the microbiota and the incidence of spurious associations. On this basis, we present a list of host variables that we recommend should be captured in human microbiota studies for the purpose of matching comparison groups, which we anticipate will increase robustness and reproducibility in resolving the members of the gut microbiota that are truly associated with human disease.

[Source](https://www.nature.com/articles/s41586-020-2881-9)

10.1038/s41586-020-2881-9

---

## Infant Skin Bacterial Communities Vary by Skin Site and Infant Age across Populations in Mexico and the United States.
 04.11.2020, _MB Manus, S Kuthyar, AG Perroni-Marañón, A Núñez-de la Mora, KR Amato_


Daily practices put humans in close contact with the surrounding environment, and differences in these practices have an impact on human physiology, development, and health. There is mounting evidence that the microbiome represents an interface that mediates interactions between the human body and the environment. In particular, the skin microbiome serves as the primary interface with the external environment and aids in host immune function by contributing as the first line of defense against pathogens. Despite these important connections, we have only a basic understanding of how the skin microbiome is first established, or which environmental factors contribute to its development. To this end, this study compared the skin bacterial communities of infants (n = 47) living in four populations in Mexico and the United States that span the socioeconomic gradient, where we predicted that variation in physical and social environments would shape the infant skin microbiome. Results of 16S rRNA bacterial gene sequencing on 119 samples (armpit, hand, and forehead) showed that infant skin bacterial diversity and composition are shaped by population-level factors, including those related to socioeconomic status and household composition, and vary by skin site and infant age. Differences in infant-environment interactions, including with other people, appear to vary across the populations, likely influencing infant microbial exposures and, in turn, the composition of infant skin bacterial communities. These findings suggest that variation in microbial exposures stemming from the local environment in infancy can impact the establishment of the skin microbiome across body sites, with implications for developmental and health outcomes.IMPORTANCE This study contributes to the sparse literature on the infant skin microbiome in general, and the virtually nonexistent literature on the infant skin microbiome in a field setting. While microbiome research often addresses patterns at a national scale, this study addresses the influence of population-level factors, such as maternal socioeconomic status and contact with caregivers, on infant skin bacterial communities. This approach strengthens our understanding of how local variables influence the infant skin microbiome, and paves the way for additional studies to combine biological sample collection with questionnaires to adequately capture how specific behaviors dictate infant microbial exposures. Work in this realm has implications for infant care and health, as well as for investigating how the microbial communities of different body sites develop over time, with applications to specific health outcomes associated with the skin microbiome (e.g., immune system development or atopic dermatitis).

[Source](https://msystems.asm.org/content/msys/5/6/e00834-20.full.pdf)

10.1128/mSystems.00834-20

---

## Microbiome manipulation by a soil-borne fungal plant pathogen using effector proteins.
 03.11.2020, _NC Snelders, H Rovenich, GC Petti, M Rocafort, GCM van den Berg, JA Vorholt, JR Mesters, MF Seidl, R Nijland, BPHJ Thomma_


During colonization of their hosts, pathogens secrete effector proteins to promote disease development through various mechanisms. Increasing evidence shows that the host microbiome plays a crucial role in health, and that hosts actively shape their microbiomes to suppress disease. We proposed that pathogens evolved to manipulate host microbiomes to their advantage in turn. Here, we show that the previously identified virulence effector VdAve1, secreted by the fungal plant pathogen Verticillium dahliae, displays antimicrobial activity and facilitates colonization of tomato and cotton through the manipulation of their microbiomes by suppressing antagonistic bacteria. Moreover, we show that VdAve1, and also the newly identified antimicrobial effector VdAMP2, are exploited for microbiome manipulation in the soil environment, where the fungus resides in absence of a host. In conclusion, we demonstrate that a fungal plant pathogen uses effector proteins to modulate microbiome compositions inside and outside the host, and propose that pathogen effector catalogues represent an untapped resource for new antibiotics.

[Source](https://www.nature.com/articles/s41477-020-00799-5)

10.1038/s41477-020-00799-5

---

## Assembly and variation of root-associated microbiota of rice during their vegetative growth phase with and without lindane pollutant

[Source](https://link.springer.com/article/10.1007/s42832-020-0063-1)

---

## Restoring degraded microbiome function with self-assembled communities.
 05.11.2020, _CFG Landazuri, JS Gomez, JM Raaijmakers, BO Oyserman_


The natural microbial functions of many soils are severely degraded. Current state-of-the-art technology to restore these functions is through the isolation, screening, formulation and application of microbial inoculants and synthetic consortia. These approaches have inconsistent success, in part due to the incompatibility between the biofertilizer, crop, climate, existing soil microbiome, and physicochemical characteristics of the soils. Here, we review the current state-of-the-art in biofertilization and identify two key deficiencies in current strategies: the difficulty in designing complex multispecies biofertilizers and the bottleneck scaling the production of complex multispecies biofertilizers. To address the challenge of producing scalable, multi-species biofertilizers, we propose to merge ecological theory with bioprocess engineering to produce 'self-assembled communities' enriched for particular functional guilds and adapted to a target soil and host plant. Using the nitrogen problem as an anchor, we review relevant ecology (microbial, plant and environmental), as well as reactor design strategies and operational parameters for the production of functionally enriched self-assembled communities. The use of self-assembled communities for biofertilization addresses two major hurdles in microbiome engineering: the importance of enriching microbes indigenous to (and targeted for) a specific environment, and the recognized potential benefits of microbial consortia over isolates (e.g. functional redundancy). The proposed community enrichment model could also be instrumental for other microbial functions such as P-solubilization, plant growth promotion or disease suppression.

[Source](https://pubmed.ncbi.nlm.nih.gov/33150935/)

10.1093/femsec/fiaa225

---

## Multi-species relationships in legume roots: From pairwise legume-symbiont interactions to the plant – microbiome – soil continuum

[Source](https://pubmed.ncbi.nlm.nih.gov/33155054/)

---

## Evidence supporting the microbiota–gut–brain axis in a songbird

[Source](https://royalsocietypublishing.org/doi/full/10.1098/rsbl.2020.0430)

---

## Comparative Analysis of the Fecal Microbiota of Wild and Captive Beal’s Eyed Turtle (<em>Sacalia bealei</em>) by 16S rRNA Gene Sequencing

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.570890/full)

---

## High-Throughput Cultivation for the Selective Isolation of Acidobacteria From Termite Nests

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.597628/full)

---

## EUKulele: Taxonomic annotation of the unsung<br>eukaryotic microbes

[Source](https://arxiv.org/ftp/arxiv/papers/2011/2011.00089.pdf)

---

## The antiSMASH database version 3: increased taxonomic coverage and new query features for modular enzymes

[Source](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkaa978/5957162)

---

## Microbiome: How to Navigate Through Matlab to Perform Machine Learning Analysis on Metagenomics Data

[Source](https://www.youtube.com/watch?v=Q3ZCwV2touQ&feature=youtu.be)

---

##  Natural Products – Data Repositories Questionnair

[Source](https://docs.google.com/forms/d/e/1FAIpQLSc9IP4BOaeenNFar0sFpxzDNhy9WjCoFc46jt0rXm1r7v2HQw/viewform)

---

