## A cryptic plasmid is among the most numerous genetic elements in the human gut.
 11.03.2024, _EC Fogarty, MS Schechter, K Lolans, ML Sheahan, I Veseli, RM Moore, E Kiefl, T Moody, PA Rice, MK Yu, M Mimee, EB Chang, HJ Ruscheweyh, S Sunagawa, SL Mclellan, AD Willis, LE Comstock, AM Eren_


_Humans, Bacteria, Bacteroidetes, Feces, Metagenome, Plasmids, Gastrointestinal Tract_
Plasmids are extrachromosomal genetic elements that often encode fitness-enhancing features. However, many bacteria carry "cryptic" plasmids that do not confer clear beneficial functions. We identified one such cryptic plasmid, pBI143, which is ubiquitous across industrialized gut microbiomes and is 14 times as numerous as crAssphage, currently established as the most abundant extrachromosomal genetic element in the human gut. The majority of mutations in pBI143 accumulate in specific positions across thousands of metagenomes, indicating strong purifying selection. pBI143 is monoclonal in most individuals, likely due to the priority effect of the version first acquired, often from one's mother. pBI143 can transfer between Bacteroidales, and although it does not appear to impact bacterial host fitness in vivo, it can transiently acquire additional genetic content. We identified important practical applications of pBI143, including its use in identifying human fecal contamination and its potential as an alternative approach to track human colonic inflammatory states.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(24)00107-7)

10.1016/j.cell.2024.01.039

---

## Associations between bacterial and fungal communities in the human gut microbiota and their implications for nutritional status and body weigh t.
 11.03.2024, _R García-Gamboa, O Díaz-Torres, C Senés-Guerrero, MS Gradilla-Hernández, A Moya, V Pérez-Brocal, A Garcia-Gonzalez, M González-Avila_


_Humans, Gastrointestinal Microbiome, Overweight, Nutritional Status, Mycobiome, Bacteria, Obesity, Bacteroidetes, Firmicutes, Saccharomycetales_
This study examined the interplay between bacterial and fungal communities in the human gut microbiota, impacting on nutritional status and body weight. Cohorts of 10 participants of healthy weight, 10 overweight, and 10 obese individuals, underwent comprehensive analysis, including dietary, anthropometric, and biochemical evaluations. Microbial composition was studied via gene sequencing of 16S and ITS rDNA regions, revealing bacterial (bacteriota) and fungal (mycobiota) profiles. Bacterial diversity exceeded fungal diversity. Statistically significant differences in bacterial communities were found within healthy-weight, overweight, and obese groups. The Bacillota/Bacteroidota ratio (previously known as the Firmicutes/Bacteroidetes ratio) correlated positively with body mass index. The predominant fungal phyla were Ascomycota and Basidiomycota, with the genera Nakaseomyces, Kazachstania, Kluyveromyces, and Hanseniaspora, inversely correlating with weight gain; while Saccharomyces, Debaryomyces, and Pichia correlated positively with body mass index. Overweight and obese individuals who harbored a higher abundance of Akkermansia muciniphila, demonstrated a favorable lipid and glucose profiles in contrast to those with lower abundance. The overweight group had elevated Candida, positively linked to simple carbohydrate consumption. The study underscores the role of microbial taxa in body mass index and metabolic health. An imbalanced gut bacteriota/mycobiota may contribute to obesity/metabolic disorders, highlighting the significance of investigating both communities.

[Source](https://doi.org/10.1038/s41598-024-54782-7)

10.1038/s41598-024-54782-7

---

## Multicomponent (bio)markers for obesity risk prediction: a scoping review protocol.
 11.03.2024, _F Vahid, C Dessenne, JA Tur, C Bouzas, Y Devaux, L Malisoux, M Monserrat-Mesquida, A Sureda, MS Desai, JD Turner, E Lamy, M Perez-Jimenez, G Ravn-Haren, R Andersen, S Forberger, R Nagrani, Y Ouzzahra, MF Fontefrancesco, MG Onorati, GG Bonetti, T de-Magistris, T Bohn_


_Humans, Obesity, Anthropometry, Cognition, Databases, Factual, Biomarkers, Research Design, Systematic Reviews as Topic, Review Literature as Topic_
Despite international efforts, the number of individuals struggling with obesity is still increasing. An important aspect of obesity prevention relates to identifying individuals at risk at early stage, allowing for timely risk stratification and initiation of countermeasures. However, obesity is complex and multifactorial by nature, and one isolated (bio)marker is unlikely to enable an optimal risk stratification and prognosis for the individual; rather, a combined set is required. Such a multicomponent interpretation would integrate biomarkers from various domains, such as classical markers (eg, anthropometrics, blood lipids), multiomics (eg, genetics, proteomics, metabolomics), lifestyle and behavioural attributes (eg, diet, physical activity, sleep patterns), psychological traits (mental health status such as depression) and additional host factors (eg, gut microbiota diversity), also by means of advanced interpretation tools such as machine learning. In this paper, we will present a protocol that will be employed for a scoping review that attempts to summarise and map the state-of-the-art in the area of multicomponent (bio)markers related to obesity, focusing on the usability and effectiveness of such biomarkers.

[Source](https://doi.org/10.1136/bmjopen-2023-083558)

10.1136/bmjopen-2023-083558

---

## Gut microbiome-derived bacterial extracellular vesicles in patients with solid tumours.
 08.03.2024, _S Mishra, MV Tejesvi, J Hekkala, J Turunen, N Kandikanti, A Kaisanlahti, M Suokas, S Leppä, P Vihinen, H Kuitunen, K Sunela, J Koivunen, A Jukkola, I Kalashnikov, P Auvinen, OS Kääriäinen, T Peñate Medina, O Peñate Medina, J Saarnio, S Meriläinen, T Rautio, R Aro, R Häivälä, J Suojanen, M Laine, P Erawijattari, L Lahti, P Karihtala, TS Ruuska, J Reunanen_


Gut microbiome-derived nanoparticles, known as bacterial extracellular vesicles (bEVs), have garnered interest as promising tools for studying the link between the gut microbiome and human health. The diverse composition of bEVs, including their proteins, mRNAs, metabolites, and lipids, makes them useful for investigating diseases such as cancer. However, conventional approaches for studying gut microbiome composition alone may not be accurate in deciphering host-gut microbiome communication. In clinical microbiome research, there is a gap in the knowledge on the role of bEVs in solid tumor patients.

[Source](https://doi.org/10.1016/j.jare.2024.03.003)

10.1016/j.jare.2024.03.003

---

## Comparative analysis of the vaginal bacteriome and virome in healthy women living in high-altitude and sea-level areas.
 11.03.2024, _C Li, S Jin, O Lv, G Wang, Y Zhang, S Li, W Zhang, F Long, Z Shen, S Bai, D Zhaxi, F Kong, Q Yan, Z Xiao_


_Pregnancy, Female, Humans, Virome, Altitude, Vagina, Microbiota, Sexually Transmitted Diseases, Viruses_
The vaginal microbiota plays an important role in the health of the female reproductive tract and is closely associated with various pregnancy outcomes and sexually transmitted diseases. Plenty of internal and external factors have strong influence on the changes in a woman's vaginal microbiome. However, the effect of a high-altitude on female vaginal microbiota has not been described. In this study, we characterized the vaginal bacteriome and virome of 13 and 34 healthy women living in high-altitude and sea-level areas, using whole-metagenome shotgun sequencing of their vaginal mucus samples. The results revealed that the vaginal bacteriomes of high-altitude individuals are featured by a significant increase of species diversity, depletion of Lactobacillus crispatus, and more abundant of some anaerobic bacteria, such as Chlamydia trachomatis, Mageeibacillus indolicus, Dialister micraerophilus, and Sneathia amnii). In addition, the vagina samples of sea-level subjects harbor more Lactobacillus strains, whereas the anaerobic bacteroidetes strains mostly appeared in high-altitude subjects. Identified and assembled 191 virus operational taxonomic units (vOTUs), there were significant differences in the abundance of 107 vOTUs between the two groups. Together, the results of this study raised the understanding of bacteriome and virome in the vagina of women at different elevations, and demonstrated that the vaginal microbiome is related to the high-altitude geographic adaptation.

[Source](https://doi.org/10.1186/s40001-023-01391-1)

10.1186/s40001-023-01391-1

---

## Cas9-assisted biological containment of a genetically engineered human commensal bacterium and genetic elements.
 11.03.2024, _N Hayashi, Y Lai, J Fuerte-Stone, M Mimee, TK Lu_


_Humans, CRISPR-Cas Systems, Containment of Biohazards, Genetic Engineering, Bacteria, Thymidine_
Sophisticated gene circuits built by synthetic biology can enable bacteria to sense their environment and respond predictably. Engineered biosensing bacteria outfitted with such circuits can potentially probe the human gut microbiome to prevent, diagnose, or treat disease. To provide robust biocontainment for engineered bacteria, we devised a Cas9-assisted auxotrophic biocontainment system combining thymidine auxotrophy, an Engineered Riboregulator (ER) for controlled gene expression, and a CRISPR Device (CD). The CD prevents the engineered bacteria from acquiring thyA via horizontal gene transfer, which would disrupt the biocontainment system, and inhibits the spread of genetic elements by killing bacteria harboring the gene cassette. This system tunably controlled gene expression in the human gut commensal bacterium Bacteroides thetaiotaomicron, prevented escape from thymidine auxotrophy, and blocked transgene dissemination. These capabilities were validated in vitro and in vivo. This biocontainment system exemplifies a powerful strategy for bringing genetically engineered microorganisms safely into biomedicine.

[Source](https://doi.org/10.1038/s41467-024-45893-w)

10.1038/s41467-024-45893-w

---

## Unlocking the secrets: exploring the influence of the aryl hydrocarbon receptor and microbiome on cancer development.
 09.03.2024, _M Rayan, TS Sayed, OJ Hussein, L Therachiyil, ZH Maayah, C Maccalli, S Uddin, JHM Prehn, HM Korashy_


_Humans, Receptors, Aryl Hydrocarbon, Neoplasms, Microbiota, Carcinogenesis, Environmental Pollutants, Tumor Microenvironment_
Gut microbiota regulates various aspects of human physiology by producing metabolites, metabolizing enzymes, and toxins. Many studies have linked microbiota with human health and altered microbiome configurations with the occurrence of several diseases, including cancer. Accumulating evidence suggests that the microbiome can influence the initiation and progression of several cancers. Moreover, some microbiotas of the gut and oral cavity have been reported to infect tumors, initiate metastasis, and promote the spread of cancer to distant organs, thereby influencing the clinical outcome of cancer patients. The gut microbiome has recently been reported to interact with environmental factors such as diet and exposure to environmental toxicants. Exposure to environmental pollutants such as polycyclic aromatic hydrocarbons (PAHs) induces a shift in the gut microbiome metabolic pathways, favoring a proinflammatory microenvironment. In addition, other studies have also correlated cancer incidence with exposure to PAHs. PAHs are known to induce organ carcinogenesis through activating a ligand-activated transcriptional factor termed the aryl hydrocarbon receptor (AhR), which metabolizes PAHs to highly reactive carcinogenic intermediates. However, the crosstalk between AhR and the microbiome in mediating carcinogenesis is poorly reviewed. This review aims to discuss the role of exposure to environmental pollutants and activation of AhR on microbiome-associated cancer progression and explore the underlying molecular mechanisms involved in cancer development.

[Source](https://doi.org/10.1186/s11658-024-00538-0)

10.1186/s11658-024-00538-0

---

## Multi-omics and gut microbiome: Unveiling the pathogenic mechanisms of early-life pesticide exposure.
 11.03.2024, _Y Yue, X Sun, S Tian, S Yan, W Sun, J Miao, S Huang, J Diao, Z Zhou, W Zhu_


_Child, Humans, Animals, Rats, Mice, Pesticides, Gastrointestinal Microbiome, Multiomics, Zebrafish, Insecticides_
The extensive application of pesticides in agricultural production has raised significant concerns about its impact on human health. Different pesticides, including fungicides, insecticides, and herbicides, cause environmental pollution and health problems for non-target organisms. Infants and young children are so vulnerable to the harmful effects of pesticide exposure that early-life exposure to pesticides deserves focused attention. Recent research lays emphasis on understanding the mechanism between negative health impacts and early-life exposure to various pesticides. Studies have explored the impacts of exposure to these pesticides on model organisms (zebrafish, rats, and mice), as well as the mechanism of negative health effects, based on advanced methodologies like gut microbiota and multi-omics. These methodologies help comprehend the pathogenic mechanisms associated with early-life pesticide exposure. In addition to presenting health problems stemming from early-life exposure to pesticides and their pathogenic mechanisms, this review proposes expectations for future research. These proposals include focusing on identifying biomarkers that indicate early-life pesticide exposure, investigating transgenerational effects, and seeking effective treatments for diseases arising from such exposure. This review emphasizes how to understand the pathogenic mechanisms of early-life pesticide exposure through gut microbiota and multi-omics, as well as the adverse health effects of such exposure.

[Source](https://doi.org/10.1016/j.pestbp.2024.105770)

10.1016/j.pestbp.2024.105770

---

## Identification of microbial pathogens in Neolithic Scandinavian humans.
 11.03.2024, _N Bergfeldt, E Kırdök, N Oskolkov, C Mirabello, P Unneberg, H Malmström, M Fraser, F Sanchez-Quinto, R Jorgensen, B Skar, K Lidén, M Jakobsson, J Storå, A Götherström_


_Humans, Agriculture, DNA, Mitochondrial, Europe, History, Ancient, Yersinia, Microbiota_
With the Neolithic transition, human lifestyle shifted from hunting and gathering to farming. This change altered subsistence patterns, cultural expression, and population structures as shown by the archaeological/zooarchaeological record, as well as by stable isotope and ancient DNA data. Here, we used metagenomic data to analyse if the transitions also impacted the microbiome composition in 25 Mesolithic and Neolithic hunter-gatherers and 13 Neolithic farmers from several Scandinavian Stone Age cultural contexts. Salmonella enterica, a bacterium that may have been the cause of death for the infected individuals, was found in two Neolithic samples from Battle Axe culture contexts. Several species of the bacterial genus Yersinia were found in Neolithic individuals from Funnel Beaker culture contexts as well as from later Neolithic context. Transmission of e.g. Y. enterocolitica may have been facilitated by the denser populations in agricultural contexts.

[Source](https://doi.org/10.1038/s41598-024-56096-0)

10.1038/s41598-024-56096-0

---

## A comparison of short-read, HiFi long-read, and hybrid strategies for genome-resolved metagenomics.
 07.03.2024, _R Eisenhofer, J Nesme, L Santos-Bay, A Koziol, SJ Sørensen, A Alberdi, O Aizpurua_


Shotgun metagenomics enables the reconstruction of complex microbial communities at a high level of detail. Such an approach can be conducted using both short-read and long-read sequencing data, as well as a combination of both. To assess the pros and cons of these different approaches, we used 22 fecal DNA extracts collected weekly for 11 weeks from two respective lab mice to study seven performance metrics over four combinations of sequencing depth and technology: (i) 20 Gbp of Illumina short-read data, (ii) 40 Gbp of short-read data, (iii) 20 Gbp of PacBio HiFi long-read data, and (iv) 40 Gbp of hybrid (20 Gbp of short-read +20 Gbp of long-read) data. No strategy was best for all metrics; instead, each one excelled across different metrics. The long-read approach yielded the best assembly statistics, with the highest N50 and lowest number of contigs. The 40 Gbp short-read approach yielded the highest number of refined bins. Finally, the hybrid approach yielded the longest assemblies and the highest mapping rate to the bacterial genomes. Our results suggest that while long-read sequencing significantly improves the quality of reconstructed bacterial genomes, it is more expensive and requires deeper sequencing than short-read approaches to recover a comparable amount of reconstructed genomes. The most optimal strategy is study-specific and depends on how researchers assess the trade-off between the quantity and quality of recovered genomes.IMPORTANCEMice are an important model organism for understanding the gut microbiome. When studying these gut microbiomes using DNA techniques, researchers can choose from technologies that use short or long DNA reads. In this study, we perform an extensive benchmark between short- and long-read DNA sequencing for studying mice gut microbiomes. We find that no one approach was best for all metrics and provide information that can help guide researchers in planning their experiments.

[Source](https://doi.org/10.1128/spectrum.03590-23)

10.1128/spectrum.03590-23

---

## OMEinfo: global geographic metadata for -omics experiments.
 09.03.2024, _M Crown, M Bashton_


Microbiome studies increasingly associate geographical features like rurality and climate with microbiomes. It is essential to correctly integrate rich geographical metadata; and inconsistent definitions of rurality, can hinder cross-study comparisons. We address this with OMEinfo, a tool for automated retrieval of consistent geographical metadata from user-provided location data. OMEinfo leverages open data sources such as the Global Human Settlement Layer, and Open-Data Inventory for Anthropogenic Carbon dioxide. OMEinfo's web-app enables users to visualize and investigate the spatial distribution of metadata features. OMEinfo promotes reproducibility and consistency in microbiome metadata through a standardized metadata retrieval approach. To demonstrate utility, OMEinfo is used to replicate the results of a previous study linking population density to bacterial diversity. As the field explores relationships between microbiomes and geographical features, tools like OMEinfo will prove vital in developing a robust, accurate, and interconnected understanding of these interactions, whilst having applicability beyond this field to any studies utilizing location-based metadata. Finally, we release the OMEinfo annotation dataset of 5.3 million OMEinfo annotated samples from the ENA, for use in retrospective analyses of sequencing samples, and suggest several ways researchers and sequencing read repositories can improve the quality of underlying metadata submitted to these public stores.

[Source](https://doi.org/10.1093/bioadv/vbae025)

10.1093/bioadv/vbae025

---

## Successful topical treatment of human biofilms using multiple antibiotic elution from a collagen-rich hydrogel.
 11.03.2024, _AD Sharma, EH Jarman, K Kuppalli, MJ Murphy, MT Longaker, G Gurtner, PM Fox_


_Animals, Humans, Anti-Bacterial Agents, Hydrogels, Wound Healing, Collagen, Diabetic Foot, Biofilms, Mammals_
Chronic non-healing wounds significantly strain modern healthcare systems, affecting 1-2% of the population in developed countries with costs ranging between $28.1 and $96.8 billion annually. Additionally, it has been established that chronic wounds resulting from comorbidities, such as peripheral vascular disease and diabetes mellitus, tend to be polymicrobial in nature. Treatment of polymicrobial chronic wounds with oral and IV antibiotics can result in antimicrobial resistance, leading to more difficult-to-treat wounds. Ideally, chronic ulcers would be topically treated with antibiotic combinations tailored to the microbiome of a patient's wound. We have previously shown that a topical collagen-rich hydrogel (cHG) can elute single antibiotics to inhibit bacterial growth in a manner that is nontoxic to mammalian cells. Here, we analyzed the microbiology of cultures taken from human patients diagnosed with diabetes mellitus suffering from chronic wounds present for more than 6 weeks. Additionally, we examined the safety of the elution of multiple antibiotics from collagen-rich hydrogel in mammalian cells in vivo. Finally, we aimed to create tailored combinations of antibiotics impregnated into cHG to successfully target and treat infections and eradicate biofilms cultured from human chronic diabetic wound tissue. We found that the majority of human chronic wounds in our study were polymicrobial in nature. The elution of multiple antibiotics from cHG was well-tolerated in mammalian cells, making it a potential topical treatment of the polymicrobial chronic wound. Finally, combinations of antibiotics tailored to each patient's microbiome eluted from a collagen-rich hydrogel successfully treated bacterial cultures isolated from patient samples via an in vitro assay.

[Source](https://doi.org/10.1038/s41598-024-54477-z)

10.1038/s41598-024-54477-z

---

## The astounding exhaustiveness and speed of the Astral mass analyzer for highly complex samples is a quantum leap in the functional analysis of microbiomes.
 11.03.2024, _T Dumas, R Martinez Pinna, C Lozano, S Radau, O Pible, L Grenga, J Armengaud_


_Humans, Tandem Mass Spectrometry, Proteomics, Peptides, Microbiota, Databases, Protein_
By analyzing the proteins which are the workhorses of biological systems, metaproteomics allows us to list the taxa present in any microbiota, monitor their relative biomass, and characterize the functioning of complex biological systems.

[Source](https://doi.org/10.1186/s40168-024-01766-4)

10.1186/s40168-024-01766-4

---

