## here

[Source](https://t.co/jAWAD5OGGI?amp=1)

---

## Symbiosis and stress: how plant microbiomes affect host evolution.
 10.08.2020, _CV Hawkes, JJ Bull, JA Lau_


Existing paradigms for plant microevolution rarely acknowledge the potential impacts of diverse microbiomes on evolutionary processes. Many plant-associated microorganisms benefit the host via access to resources, protection from pathogens, or amelioration of abiotic stress. In doing so, they alter the plant's perception of the environment, potentially reducing the strength of selection acting on plant stress tolerance or defence traits or altering the traits that are the target of selection. We posit that the microbiome can affect plant microevolution via (1) manipulation of plant phenotypes in ways that increase plant fitness under stress and (2) direct microbial responses to the environment that benefit the plant. Both mechanisms might favour plant genotypes that attract or stimulate growth of the most responsive microbial populations or communities. We provide support for these scenarios using infectious disease and quantitative genetics models. Finally, we discuss how beneficial plant-microbiome associations can evolve if traditional mechanisms maintaining cooperation in pairwise symbioses, namely partner fidelity, partner choice and fitness alignment, also apply to the interactions between plants and diverse foliar and soil microbiomes. To understand the role of the plant microbiome in host evolution will require a broad ecological understanding of plant-microbe interactions across both space and time. This article is part of the theme issue 'The role of the microbiome in host evolution'.

[Source](https://royalsocietypublishing.org/toc/rstb/2020/375/1808)

10.1098/rstb.2019.0590

---

## The role of the microbiome in host evolution

[Source](https://royalsocietypublishing.org/toc/rstb/2020/375/1808)

---

## Evidence for contamination as the origin for bacteria found in human placenta rather than a microbiota.
 10.08.2020, _R Gschwind, T Fournier, S Kennedy, V Tsatsaris, AG Cordier, F Barbut, MJ Butel, S Wydau-Dematteis_


Until recently the in utero environment of pregnant women was considered sterile. Recent high-sensitivity molecular techniques and high-throughput sequencing lead to some evidence for a low-biomass microbiome associated with the healthy placenta. Other studies failed to reveal evidence for a consistent presence of bacteria using either culture or molecular based techniques. Comparing conflicting "placental microbiome" studies is complicated by the use of varied and inconsistent protocols. Given this situation, we undertook an evaluation of the in utero environment sterility using several controlled methods, in the same study, to evaluate the presence or absence of bacteria and to explain contradictions present in the literature. Healthy pregnant women (n = 38) were recruited in three maternity wards. Placenta were collected after cesarean section with or without Alexis® and vaginal delivery births. For this study we sampled fetal membranes, umbilical cord and chorionic villi. Bacterial presence was analyzed using bacterial culture and qPCR on 34 fetal membranes, umbilical cord and chorionic villi samples. Shotgun metagenomics was performed on seven chorionic villi samples. We showed that the isolation of meaningful quantities of viable bacteria or bacterial DNA was possible only outside the placenta (fetal membranes and umbilical cords) highlighting the importance of sampling methods in studying the in utero environment. Bacterial communities described by metagenomics analysis were similar in chorionic villi samples and in negative controls and were dependent on the database chosen for the analysis. We conclude that the placenta does not harbor a specific, consistent and functional microbiota.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237232)

10.1371/journal.pone.0237232

---

## SARS-CoV-2 in human milk is inactivated by Holder pasteurization but not cold storage

[Source](https://onlinelibrary.wiley.com/doi/10.1111/jpc.15065#)

---

## Oral microbiome geography: Micron-scale habitat and nich

[Source](https://www.sciencedirect.com/science/article/pii/S1931312820304054?dgcid=author)

---

## Epidemiology and associated microbiota changes in deployed military personnel at high risk of traveler’s diarrhea

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0236703)

---

## Differential alteration in gut microbiome profiles during acquisition, extinction, and reinstatement of morphine-induced CPP

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0278584620303742)

---

## Modulation of the intestinal microbiota of broilers supplemented with monensin or functional oils in response to challenge by <em>Eimeria</em> spp

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237118)

---

## .

[Source](https://u2730683.ct.sendgrid.net/ls/click?upn=1I40RAgHmCEvmIsjUbeAh3he4lcJCb-2FC42rEKZ3c8FpZbQ7EuAY74hTnLYpV3-2BhEpcNnEZ-2FOOdoD6UfzQj3ytZMxdFSOVQ21OJYQX4MuE6o-3Dcy16_Ca-2F6HwZe6r-2Bcd3JI9-2BmwvRvL6Gtvob0rhw-2B8-2BTbob13WFjyK90bUuEh6IKyiWRpu9xUt6CK-2Fm4HShMnkHWJUj4Sc9tPNZd0D7VNOv5y-2BXVdnBSNhH0HJtCUwfla6BvF9b-2BEtXZxNRilKk2XqtzbgFATZXP-2FyoG-2BOa37n32I2vwBZr1lJE5Grvp3e8al9CdbsAgOjDsn-2Fff-2BaBZbMbHlj6hLOdY0x9pbyAqKyi-2BbWgNrRoc3M3iseqdZRNu5RGyLF02cPqavJBazoaiCMD3Hcog-3D-3D)

---

## Herpes simplex virus infection, Acyclovir and IVIG treatment all independently cause gut dysbiosis

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237189)

---

## Raw milk and fecal microbiota of commercial Alpine dairy cows varies with herd, fat content and diet

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237262)

---

## Hydrolytic bacteria associated with natural helminth infection in the midgut of Red Sea marbled spinefoot rabbit fish <em>Siganus rivulatus</em>

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0882401020307701)

---

## Book: Agriculturally Important Fungi for Sustainable Agriculture Vol 2: Functional Annotation for Crop Protection

[Source](https://link.springer.com/book/10.1007/978-3-030-48474-3)

---

## Patient propagules: Do soil archives preserve the legacy of fungal and prokaryotic communities?

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237368)

---

## Labile carbon limits late winter microbial activity near Arctic treeline

[Source](https://www.nature.com/articles/s41467-020-17790-5)

---

## Enhanced biodegradation of crude oil by constructed bacterial consortium comprising salt-tolerant petroleum degraders and biosurfactant producers 

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0964830520306478)

---

## Plant–archaea relationships: a potential means to improve crop production in arid and semi-arid regions

[Source](https://link.springer.com/article/10.1007/s11274-020-02910-6)

---

## First report of Fusarium equiseti causing crown rot and damping-off on durum wheat in Algeria

[Source](https://www.tandfonline.com/doi/abs/10.1080/03235408.2020.1804303)

---

## Soil&nbsp;microbial&nbsp;diversity and composition: Links to soil texture and associated properties

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0038071720302492)

---

## Effects of different habitats on the bacterial community composition in the water and sediments of Lake Taihu, China

[Source](https://link.springer.com/article/10.1007/s11356-020-10376-0)

---

## Genomic and enzymatic evidence of acetogenesis by anaerobic methanotrophic archaea

[Source](https://www.nature.com/articles/s41467-020-17860-8)

---

## Undinarchaeota illuminate DPANN phylogeny and the impact of gene transfer on archaeal evolution

[Source](https://www.nature.com/articles/s41467-020-17408-w)

---

## Occurrence, identification and antibiogram signatures of selected Enterobacteriaceae from Tsomo and Tyhume rivers in the Eastern Cape Province, Republic of South Africa

[Source](https://www.biorxiv.org/content/10.1101/2020.08.11.246025v1.abstract?%3Fcollection=)

---

## The&nbsp;microbiome&nbsp;of drinking water biofilters is influenced by environmental factors and engineering decisions but has little influence on the&nbsp;microbiome&nbsp;of the filtrate

[Source](https://pubs.acs.org/doi/abs/10.1021/acs.est.0c01730)

---

## <em>In-situ</em> remediation of acid mine drainage from abandoned coal mine by filed pilot-scale passive treatment system: Performance and response of microbial communities to low pH and elevated Fe

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0960852420312578)

---

## De novo sequence assembly requires bioinformatic checking of chimeric sequences.
 10.08.2020, _LS Arroyo Mühr, C Lagheden, SS Hassan, SN Kleppe, E Hultin, J Dillner_


De novo assembly of sequence reads from next generation sequencing platforms is a common strategy for detecting presence and sequencing of viruses in biospecimens. Amplification artifacts and presence of several related viruses in the same specimen can lead to assembly of erroneous, chimeric sequences. We now report that such chimeras can also occur between viral and non-viral biological sequences incorrectly joined together which may cause erroneous detection of viruses, highlighting the importance of performing a chimera checking step in bioinformatics pipelines. Using Illumina NextSeq and metagenomic sequencing, we analyzed 80 consecutive non-melanoma skin cancers (NMSCs) from 11 immunosuppressed patients together with 11 NMSCs from patients who had only developed 1 NMSC. We aligned high-quality reads against a Human Papillomavirus (HPV) database and found HPV sequences in 9/91 specimens. A previous bioinformatic analysis of the same crude sequencing data from some of these samples had found an additional 3 specimens to be HPV-positive after performing de novo assembly. The reason for the discrepancy was investigated and found to be mostly caused by chimeric sequences containing both viral and non-viral sequences. Non-viral sequences were present in these 3 samples. To avoid erroneous detection of HPV when performing sequencing, we thus developed a novel script to identify HPV chimeric sequences.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237455)

10.1371/journal.pone.0237455

---

## Review on matrix-assisted laser desorption/ionization time-of-flight mass spectrometry for the rapid screening of&nbsp;microbial&nbsp;species: A promising bioanalytical tool

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0026265X2032302X)

---

## New machine learning tool predicts devastating intestinal disease in premature infants

[Source](https://www.sciencedaily.com/releases/2020/08/200812094842.htm)

---

## Does retraction after misconduct have an impact on citations? A pre-post study

[Source](https://www.biorxiv.org/content/10.1101/2020.08.11.246637v1.abstract?%3Fcollection=)

---

