## The abundance of Ruminococcus bromii is associated with faecal butyrate levels and atopic dermatitis in infancy

[Source](https://onlinelibrary.wiley.com/doi/10.1111/all.15440)

---

## Smokeless tobacco consumption induces dysbiosis of oral mycobiome: a pilot study.
 01.08.2022, _M Sajid, P Sharma, S Srivastava, R Hariprasad, H Singh, M Bharadwaj_


Smokeless tobacco (SLT) alters the oral microbiome of smokeless tobacco users. Dysbiosis of oral bacteriome has been determined; however, the mycobiome of SLT users has not been characterized. The oral mycobiome was assayed by amplification and sequencing of the fungal internal transcribed spacer (ITS1) region from oral swab samples of non-SLT users, SLT users (with or without oral lesions), and SLT with alcohol users. We observed that the richness and diversity of oral mycobiome were significantly decreased in SLT with oral lesions users than in non-users. The β-diversity analysis showed significant dissimilarity of oral mycobiome between non-users and SLT with oral lesions users. Linear discriminant analysis effect size and random forest analysis of oral mycobiome affirm that the genus Pichia was typical for SLT with oral lesions users. Prevalence of the fungal genus Pichia correlates positively with Starmerella, Mortierella, Fusarium, Calonectria, and Madurella, but is negatively correlated with Pyrenochaeta, Botryosporium, and Alternaria. Further, the determination of oral mycobiome functionality showed a high abundance of pathotroph-saprotroph-symbiotroph and animal pathogen-endophyte-epiphyte-undefined saprotroph at trophic and guild levels, respectively, indicating possibly major changes in normal growth repression of types of fungi. The oral mycobiome in SLT users was identified and comprehensively analyzed for the first time. SLT intake is associated with oral mycobiome dysbiosis and such alterations of the oral mycobiome may contribute to oral carcinogenesis in SLT users. This study will provide a basis for further large-scale investigations on the potential role of the mycobiome in SLT-induced oral cancer. KEY POINTS: • SLT induces dysbiosis of the oral microbiome that can contribute to oral cancer. • Oral mycobiome diversity is noticeably reduced in SLT users having oral lesions. • Occurrence of Pichia can be used as a biomarker for SLT users having oral lesions.

[Source](https://link.springer.com/article/10.1007/s00253-022-12096-6)

10.1007/s00253-022-12096-6

---

## The Proteome of Extracellular Vesicles Produced by the Human Gut Bacteria Bacteroides thetaiotaomicron In Vivo Is Influenced by Environmental and Host-Derived Factors.
 02.08.2022, _R Stentz, E Jones, R Juodeikis, U Wegmann, M Guirro, AJ Goldson, A Brion, C Booth, P Sudhakar, IR Brown, T Korcsmáros, SR Carding_


Bacterial extracellular vesicles (BEVs) released from both Gram-negative and Gram-positive bacteria provide an effective means of communication and trafficking of cell signaling molecules. In the gastrointestinal tract (GIT) BEVs produced by members of the intestinal microbiota can impact host health by mediating microbe-host cell interactions. A major unresolved question, however, is what factors influence the composition of BEV proteins and whether the host influences protein packaging into BEVs and secretion into the GIT. To address this, we have analyzed the proteome of BEVs produced by the major human gut symbiont Bacteroides thetaiotaomicron both in vitro and in vivo in the murine GIT in order to identify proteins specifically enriched in BEVs produced in vivo. We identified 113 proteins enriched in BEVs produced in vivo, the majority (62/113) of which accumulated in BEVs in the absence of any changes in their expression by the parental cells. Among these selectively enriched proteins, we identified dipeptidyl peptidases and an asparaginase and confirmed their increased activity in BEVs produced in vivo. We also showed that intact BEVs are capable of degrading bile acids via a bile salt hydrolase. Collectively these findings provide additional evidence for the dynamic interplay of host-microbe interactions in the GIT and the existence of an active mechanism to drive and enrich a selected group of proteins for secretion into BEVs in the GIT. IMPORTANCE The gastrointestinal tract (GIT) harbors a complex community of microbes termed the microbiota that plays a role in maintaining the host's health and wellbeing. How this comes about and the nature of microbe-host cell interactions in the GIT is still unclear. Recently, nanosized vesicles naturally produced by bacterial constituents of the microbiota have been shown to influence responses of different host cells although the molecular basis and identity of vesicle-born bacterial proteins that mediate these interactions is unclear. We show here that bacterial extracellular vesicles (BEVs) produced by the human symbiont Bacteroides thetaiotaomicron in the GIT are enriched in a set of proteins and enzymes, including dipeptidyl peptidases, an asparaginase and a bile salt hydrolase that can influence host cell biosynthetic pathways. Our results provide new insights into the molecular basis of microbiota-host interactions that are central to maintaining GIT homeostasis and health.

[Source](https://journals.asm.org/doi/10.1128/aem.00533-22)

10.1128/aem.00533-22

---

## Improved gut microbiome recovery following drug therapy is linked to abundance and replication of probiotic strains.
 03.08.2022, _J FitzGerald, S Patel, J Eckenberger, E Guillemard, P Veiga, F Schäfer, J Walter, MJ Claesson, M Derrien_


Probiotics have been used for decades to alleviate the negative side-effects of oral antibiotics, but our mechanistic understanding on how they work is so far incomplete. Here, we performed a metagenomic analysis of the fecal microbiota in participants who underwent a 14-d Helicobacter pylori eradication therapy with or without consumption of a multi-strain probiotic intervention (L. paracasei CNCM I-1518, L. paracasei CNCM I-3689, L. rhamnosus CNCM I-3690, and four yogurt strains) in a randomized, double-blinded, controlled clinical trial. Using a strain-level analysis for detection and metagenomic determination of replication rate, ingested strains were detected and replicated transiently in fecal samples and in the gut during and following antibiotic administration. Consumption of the fermented milk product led to a significant, although modest, improvement in the recovery of microbiota composition. Stratification of participants into two groups based on the degree to which their microbiome recovered showed i) a higher fecal abundance of the probiotic L. paracasei and L. rhamnosus strains and ii) an elevated replication rate of one strain (L. paracasei CNCMI-1518) in the recovery group. Collectively, our findings show a small but measurable benefit of a fermented milk product on microbiome recovery after antibiotics, which was linked to the detection and replication of specific probiotic strains. Such functional insight can form the basis for the development of probiotic-based intervention aimed to protect gut microbiome from drug treatments.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2022.2094664)

10.1080/19490976.2022.2094664

---

## Emerging frontiers of antibiotics use and their impacts on the human gut microbiome.
 03.08.2022, _R Kumari, Y Yadav, R Misra, U Das, U Das Adhikari, P Malakar, GP Dubey_


Antibiotics, the primary drugs used to cure bacterial diseases, are increasingly becoming ineffective due to the emergence of multiple drug resistance (MDR) leading to recurrence of previously sensitive pathogens. Human gut microbiome (GM), known to play an important role in various physiological processes, consists of pool of diverse microbes. Indiscriminate use of antibiotics during the life span of an individual may lead to development of resistant microbes e.g. Vibrio, Acinetobacter, Escherichia, Klebsiella, Clostridia, etc. in the human GM. Transmission of antibiotic resistant genes (ARGs) between pathogenic and commensal bacteria occurs more frequently in microbiome communities wherein bacteria communicate and exchange cellular constituents both among themselves and with the host. Additionally, co-factors like 'early vs. late' exposure, type of antibiotics and duration of treatment modulate the adverse effects of antibiotics on GM maturation. Furthermore, factors like mode of birth, ethnicity, malnutrition, demography, diet, lifestyle, etc., which influence GM composition, can also indirectly alter the host response to antibiotics. Currently, advanced 'omics' and culturomics approaches are revealing novel avenues to study the interplay between antibiotics and the microbiome and to identify resistant genes in these bacterial communities. Here, we discuss the recent developments that have given insights into the effects of antibiotics on the homeostatic balance of the gut microbiome and thus on human health.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0944501322001677)

10.1016/j.micres.2022.127127

---

## Microbiota of the Pregnant Mouse: Characterization of the Bacterial Communities in the Oral Cavity, Lung, Intestine, and Vagina through Culture and DNA Sequencing.
 02.08.2022, _JM Greenberg, R Romero, AD Winters, J Galaz, V Garcia-Flores, M Arenas-Hernandez, J Panzer, Z Shaffer, DJ Kracht, N Gomez-Lopez, KR Theis_


Mice are frequently used as animal models for mechanistic studies of infection and obstetrical disease, yet characterization of the murine microbiota during pregnancy is lacking. The objective of this study was to characterize the microbiotas of distinct body sites of the pregnant mouse-vagina, oral cavity, intestine, and lung-that harbor microorganisms that could potentially invade the murine amniotic cavity, thus leading to adverse pregnancy outcomes. The microbiotas of these body sites were characterized through anoxic, hypoxic, and oxic culture as well as through 16S rRNA gene sequencing. With the exception of the vagina, the cultured microbiotas of each body site varied by atmosphere, with the greatest diversity in the cultured microbiota appearing under anoxic conditions. Only cultures of the vagina were comprehensively representative of the microbiota observed through direct DNA sequencing of body site samples, primarily due to the predominance of two Rodentibacter strains. Identified as Rodentibacter pneumotropicus and Rodentibacter heylii, these isolates exhibited predominance patterns similar to those of Lactobacillus crispatus and Lactobacillus iners in the human vagina. Whole-genome sequencing of these Rodentibacter strains revealed shared genomic features, including the ability to degrade glycogen, an abundant polysaccharide in the vagina. In summary, we report body site-specific microbiotas in the pregnant mouse with potential ecological parallels to those of humans. Importantly, our findings indicate that the vaginal microbiotas of pregnant mice can be readily cultured, suggesting that mock vaginal microbiotas can be tractably generated and maintained for experimental manipulation in future mechanistic studies of host vaginal-microbiome interactions. IMPORTANCE Mice are widely utilized as animal models of obstetrical complications; however, the characterization of the murine microbiota during pregnancy has been neglected. Microorganisms from the vagina, oral cavity, intestine, and lung have been found in the intra-amniotic space, where their presence threatens the progression of gestation. Here, we characterized the microbiotas of pregnant mice and established the appropriateness of culture in capturing the microbiota at each site. The high relative abundance of Rodentibacter observed in the vagina is similar to that of Lactobacillus in humans, suggesting potential ecological parallels. Importantly, we report that the vaginal microbiota of the pregnant mouse can be readily cultured under hypoxic conditions, demonstrating that mock microbial communities can be utilized to test the potential ecological parallels between microbiotas in human and murine pregnancy and to evaluate the relevance of the structure of these microbiotas for adverse pregnancy outcomes, especially intra-amniotic infection and preterm birth.

[Source](https://journals.asm.org/doi/10.1128/spectrum.01286-22)

10.1128/spectrum.01286-22

---

## Viral biogeography of the mammalian gut and parenchymal organs.
 02.08.2022, _AN Shkoporov, SR Stockdale, A Lavelle, I Kondova, C Heuston, A Upadrasta, EV Khokhlova, I van der Kamp, B Ouwerling, LA Draper, JAM Langermans, R Paul Ross, C Hill_


The mammalian virome has been linked to health and disease but our understanding of how it is structured along the longitudinal axis of the mammalian gastrointestinal tract (GIT) and other organs is limited. Here, we report a metagenomic analysis of the prokaryotic and eukaryotic virome occupying luminal and mucosa-associated habitats along the GIT, as well as parenchymal organs (liver, lung and spleen), in two representative mammalian species, the domestic pig and rhesus macaque (six animals per species). Luminal samples from the large intestine of both mammals harboured the highest loads and diversity of bacteriophages (class Caudoviricetes, family Microviridae and others). Mucosal samples contained much lower viral loads but a higher proportion of eukaryotic viruses (families Astroviridae, Caliciviridae, Parvoviridae). Parenchymal organs contained bacteriophages of gut origin, in addition to some eukaryotic viruses. Overall, GIT virome composition was specific to anatomical region and host species. Upper GIT and mucosa-specific viruses were greatly under-represented in distal colon samples (a proxy for faeces). Nonetheless, certain viral and phage species were ubiquitous in all samples from the oral cavity to the distal colon. The dataset and its accompanying methodology may provide an important resource for future work investigating the biogeography of the mammalian gut virome.

[Source](https://www.nature.com/articles/s41564-022-01178-w)

10.1038/s41564-022-01178-w

---

## Namco: a microbiome explorer.
 03.08.2022, _A Dietrich, MS Matchado, M Zwiebel, B Ölke, M Lauber, I Lagkouvardos, J Baumbach, D Haller, B Brandl, T Skurk, H Hauner, S Reitmeier, M List_


16S rRNA gene profiling is currently the most widely used technique in microbiome research and allows the study of microbial diversity, taxonomic profiling, phylogenetics, functional and network analysis. While a plethora of tools have been developed for the analysis of 16S rRNA gene data, only a few platforms offer a user-friendly interface and none comprehensively covers the whole analysis pipeline from raw data processing down to complex analysis. We introduce Namco, an R shiny application that offers a streamlined interface and serves as a one-stop solution for microbiome analysis. We demonstrate Namco's capabilities by studying the association between a rich fibre diet and the gut microbiota composition. Namco helped to prove the hypothesis that butyrate-producing bacteria are prompted by fibre-enriched intervention. Namco provides a broad range of features from raw data processing and basic statistics down to machine learning and network analysis, thus covering complex data analysis tasks that are not comprehensively covered elsewhere. Namco is freely available at https://exbio.wzw.tum.de/namco/.

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000852)

10.1099/mgen.0.000852

---

## MiCloud: A unified web platform for comprehensive microbiome data analysis.
 03.08.2022, _W Gu, J Moon, C Chisina, B Kang, T Park, H Koh_


_Cross-Sectional Studies, Data Analysis, Gastrointestinal Microbiome, High-Throughput Nucleotide Sequencing, Humans, Microbiota_
The recent advance in massively parallel sequencing has enabled accurate microbiome profiling at a dramatically lowered cost. Then, the human microbiome has been the subject of intensive investigation in public health and medicine. In the meanwhile, researchers have developed lots of microbiome data analysis methods, protocols, and/or tools. Among those, especially, the web platforms can be highlighted because of the user-friendly interfaces and streamlined protocols for a long sequence of analytic procedures. However, existing web platforms can handle only a categorical trait of interest, cross-sectional study design, and the analysis with no covariate adjustment. We therefore introduce here a unified web platform, named MiCloud, for a binary or continuous trait of interest, cross-sectional or longitudinal/family-based study design, and with or without covariate adjustment. MiCloud handles all such types of analyses for both ecological measures (i.e., alpha and beta diversity indices) and microbial taxa in relative abundance on different taxonomic levels (i.e., phylum, class, order, family, genus and species). Importantly, MiCloud also provides a unified analytic protocol that streamlines data inputs, quality controls, data transformations, statistical methods and visualizations with vastly extended utility and flexibility that are suited to microbiome data analysis. We illustrate the use of MiCloud through the United Kingdom twin study on the association between gut microbiome and body mass index adjusting for age. MiCloud can be implemented on either the web server (http://micloud.kr) or the user's computer (https://github.com/wg99526/micloudgit).

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0272354)

10.1371/journal.pone.0272354

---

