## Cross-talk between the microbiome and chronic inflammation in esophageal cancer: potential driver of oncogenesis.
 05.05.2022, _T Sharma, A Gupta, R Chauhan, AA Bhat, S Nisar, S Hashem, S Akhtar, A Ahmad, M Haris, M Singh, S Uddin_


Esophageal cancer (EC) is frequently considered a lethal malignancy and is often identified at a later stage. It is one of the major causes of cancer-related deaths globally. The conventional treatment methods like chemotherapy, radiotherapy, and surgery offer limited efficacy and poor clinical outcome with a less than 25% 5-year survival rate. The poor prognosis of EC persists despite the growth in the development of diagnostic and therapeutic modalities to treat EC. This underlines the need to elucidate the complex molecular mechanisms that drive esophageal oncogenesis. Apart from the role of the tumor microenvironment and its structural and cellular components in tumorigenesis, mounting evidence points towards the involvement of the esophageal microbiome, inflammation, and their cross-talk in promoting esophageal cancer. The current review summarizes recent research that delineates the underlying molecular mechanisms by which the microbiota and inflammation promote the pathophysiology of esophageal cancer, thus unraveling targets for potential therapeutic intervention.

[Source](https://link.springer.com/article/10.1007/s10555-022-10026-6)

10.1007/s10555-022-10026-6

---

## Systematic and quantitative view of the antiviral arsenal of prokaryotes.
 12.05.2022, _F Tesson, A Hervé, E Mordret, M Touchon, C d'Humières, J Cury, A Bernheim_


_Antiviral Agents, Archaea, Bacteria, Genomics, Prokaryotic Cells_
Bacteria and archaea have developed multiple antiviral mechanisms, and genomic evidence indicates that several of these antiviral systems co-occur in the same strain. Here, we introduce DefenseFinder, a tool that automatically detects known antiviral systems in prokaryotic genomes. We use DefenseFinder to analyse 21000 fully sequenced prokaryotic genomes, and find that antiviral strategies vary drastically between phyla, species and strains. Variations in composition of antiviral systems correlate with genome size, viral threat, and lifestyle traits. DefenseFinder will facilitate large-scale genomic analysis of antiviral defense systems and the study of host-virus interactions in prokaryotes.

[Source](https://www.nature.com/articles/s41467-022-30269-9)

10.1038/s41467-022-30269-9

---

## A randomized controlled trial of the effects of whole grains versus refined grains diets on the microbiome in pregnancy.
 10.05.2022, _H Sun, P Yamada, A Paetow, M Chan, A Arslan, R Landberg, MG Dominguez-Bello, BK Young_


_Diet, Dietary Carbohydrates, Female, Gestational Weight Gain, Humans, Infant, Newborn, Male, Microbiota, Pregnancy, Whole Grains_
Dietary whole grain consumption has been postulated to have metabolic benefits. The purpose of this study was to compare a pregnancy diet containing 75% of total carbohydrates as refined grains with a diet of 75% of total carbohydrates as whole grains for pregnancy outcomes and effects on the microbiome. Gestational weight gain, glucose tolerance and newborn outcomes were measured on 248 enrolled compliant women from whom a subset of 103 women consented to give 108 vaginal and 109 anal swabs. The data presented here are limited to the patients from whom the vaginal and anal swabs were obtained in order to study the microbiome. A microbiome-16SrRNA survey-was characterized in these samples. Samples and measurements were obtained at the first obstetrical visit, before beginning a prescribed diet (T1-baseline) and after 17-32 weeks on the prescribed diet (T3). Food frequency questionnaires and total plasma alkylresorcinols were used as a measure of whole grain consumption. There were no dietary differences in maternal weight gain, birth weight, or glucose tolerance test. Mothers consuming the whole grains diet showed a trend of gestational decrease in vaginal bacterial alpha diversity, with increasing Lactobacillus-dominance. No significant difference was observed for the anal microbiome. The results suggest that diet modulations of the vaginal microbiome during gestation may have important implications for maternal and neonatal health and in the intergenerational transfer of maternal microbiome. Trial registration: ClinicalTrials.gov Identifier: NCT03232762.

[Source](https://www.nature.com/articles/s41598-022-11571-4)

10.1038/s41598-022-11571-4

---

## Should we modulate the neonatal microbiome and what should be the goal?
 12.05.2022, _N van Best, MG Dominguez-Bello, MW Hornef, E Jašarević, K Korpela, TD Lawley_


_Gastrointestinal Microbiome, Goals, Microbiota_

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01281-4)

10.1186/s40168-022-01281-4

---

## Space travel and early childhood gut microbiome: is space dirty enough to raise a child?

[Source](https://www.futuremedicine.com/doi/10.2217/fmb-2022-0036)

---

## On-person adaptive evolution of Staphylococcus aureus during treatment for atopic dermatitis

[Source](https://www.biorxiv.org/content/10.1101/2021.03.24.436824v2.abstract)

---

## Oral microbiome of Crohn’s Disease patients with and without oral manifestations

[Source](https://academic.oup.com/ecco-jcc/advance-article/doi/10.1093/ecco-jcc/jjac063/6577053)

---

## Integrating the salivary microbiome in the forensic toolkit by 16S rRNA gene: potential application in body fluid identification and biogeographic inference

[Source](https://link.springer.com/article/10.1007/s00414-022-02831-z)

---

## An approach for evaluating the effects of dietary fiber polysaccharides on the human gut microbiome and plasma proteome.
 11.05.2022, _O Delannoy-Bruno, C Desai, JJ Castillo, G Couture, RA Barve, V Lombard, B Henrissat, J Cheng, N Han, DK Hayashi, A Meynier, S Vinoy, CB Lebrilla, S Marion, AC Heath, MJ Barratt, JI Gordon_


_Dietary Fiber, Gastrointestinal Microbiome, Humans, Microbiota, Polysaccharides, Proteome_
SignificanceDietary fibers contain complex mixtures of biomolecules, making it difficult to develop/test hypotheses about how different fiber-types impact different components of the human gut microbiome and how microbiome changes that they produce are linked to human physiology. Here, we analyze microbiome and plasma proteome responses to consumption of two fiber-enriched snacks in two human studies. We use a variety of computational methods to correlate their effects on gut microbiome genes encoding enzymes that degrade complex fiber-associated polysaccharides, the microbial products of polysaccharide degradation, and plasma proteins representing diverse physiological processes. This approach can be used to guide the design of fiber-containing snacks that more precisely manipulate microbiome features in ways that improve nutritional and health status.

[Source](https://www.pnas.org/doi/full/10.1073/pnas.2123411119)

10.1073/pnas.2123411119

---

## A randomization-based causal inference framework for uncovering environmental exposure effects on human gut microbiota.
 09.05.2022, _AJ Sommer, A Peters, M Rommel, J Cyrys, H Grallert, D Haller, CL Müller, MC Bind_


Statistical analysis of microbial genomic data within epidemiological cohort studies holds the promise to assess the influence of environmental exposures on both the host and the host-associated microbiome. However, the observational character of prospective cohort data and the intricate characteristics of microbiome data make it challenging to discover causal associations between environment and microbiome. Here, we introduce a causal inference framework based on the Rubin Causal Model that can help scientists to investigate such environment-host microbiome relationships, to capitalize on existing, possibly powerful, test statistics, and test plausible sharp null hypotheses. Using data from the German KORA cohort study, we illustrate our framework by designing two hypothetical randomized experiments with interventions of (i) air pollution reduction and (ii) smoking prevention. We study the effects of these interventions on the human gut microbiome by testing shifts in microbial diversity, changes in individual microbial abundances, and microbial network wiring between groups of matched subjects via randomization-based inference. In the smoking prevention scenario, we identify a small interconnected group of taxa worth further scrutiny, including Christensenellaceae and Ruminococcaceae genera, that have been previously associated with blood metabolite changes. These findings demonstrate that our framework may uncover potentially causal links between environmental exposure and the gut microbiome from observational data. We anticipate the present statistical framework to be a good starting point for further discoveries on the role of the gut microbiome in environmental health.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010044)

10.1371/journal.pcbi.1010044

---

## Microbiota and gastric cancer.
 09.05.2022, _E Bessède, F Mégraud_


The discovery of Helicobacter pylori in 1982 drew to an end the stomach being considered as a sterile organ. Later, the progress in molecular methods, especially Next Generation Sequencing and metagenomics, has highlighted the fact that a diverse microbiota including five major phyla could also be present in the stomach. However, when present, H. pylori is the essential species and it influences the other bacterial communities in terms of richness and evenness. It is now well accepted that H. pylori is the main risk factor for gastric cancer, especially the strains harboring the cag pathogenicity island and the CagA oncoprotein, but the need for other factors from the host and the environment can explain the important difference between those infected and those developing gastric cancer. Several studies showed a difference between the gastric microbiota of patients at various stages of development of gastric premalignant and malignant lesions, showing globally a reduced microbial diversity and an increase in the presence of intestinal commensals, especially with nitrosative functions. Other studies showed an increase in oral microbiota. These data suggest that the gastric microbiota other than H. pylori may play a role in the last steps of gastric carcinogenesis. It must also be noted that in a limited number of cases, a virus: the Epstein Barr Virus is responsible for the evolution toward gastric cancer, while in others the mycobiota also needs to be explored. Finally, the use of mice models allowed an exploration of the role of different gastric microbiota in addition to H. pylori.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1044579X22001043)

10.1016/j.semcancer.2022.05.001

---

## Host-microbial interactions in metabolic diseases: from diet to immunity.
 05.05.2022, _JH Lee, JH Park_


Growing evidence suggests that the gut microbiome is an important contributor to metabolic diseases. Alterations in microbial communities are associated with changes in lipid metabolism, glucose homeostasis, intestinal barrier functions, and chronic inflammation, all of which can lead to metabolic disorders. Therefore, the gut microbiome may represent a novel therapeutic target for obesity, type 2 diabetes, and nonalcoholic fatty liver disease. This review discusses how gut microbes and their products affect metabolic diseases and outlines potential treatment approaches via manipulation of the gut microbiome. Increasing our understanding of the interactions between the gut microbiome and host metabolism may help restore the healthy symbiotic relationship between them.

[Source](https://link.springer.com/article/10.1007/s12275-022-2087-y)

10.1007/s12275-022-2087-y

---

## A randomized controlled trial for response of microbiome network to exercise and diet intervention in patients with nonalcoholic fatty liver disease.
 12.05.2022, _R Cheng, L Wang, S Le, Y Yang, C Zhao, X Zhang, X Yang, T Xu, L Xu, P Wiklund, J Ge, D Lu, C Zhang, L Chen, S Cheng_


_Diet, Exercise, Humans, Liver, Microbiota, Non-alcoholic Fatty Liver Disease, Prediabetic State_
Exercise and diet are treatments for nonalcoholic fatty liver disease (NAFLD) and prediabetes, however, how exercise and diet interventions impact gut microbiota in patients is incompletely understood. We previously reported a 8.6-month, four-arm (Aerobic exercise, n = 29; Diet, n = 28; Aerobic exercise + Diet, n = 29; No intervention, n = 29) randomized, singe blinded (for researchers), and controlled intervention in patients with NAFLD and prediabetes to assess the effect of interventions on the primary outcomes of liver fat content and glucose metabolism. Here we report the third primary outcome of the trial-gut microbiota composition-in participants who completed the trial (22 in Aerobic exercise, 22 in Diet, 23 in Aerobic exercise + Diet, 18 in No Intervention). We show that combined aerobic exercise and diet intervention are associated with diversified and stabilized keystone taxa, while exercise and diet interventions alone increase network connectivity and robustness between taxa. No adverse effects were observed with the interventions. In addition, in exploratory ad-hoc analyses we find that not all subjects responded to the intervention in a similar manner, when using differentially altered gut microbe amplicon sequence variants abundance to classify the responders and low/non-responders. A personalized gut microbial network at baseline could predict the individual responses in liver fat to exercise intervention. Our findings suggest an avenue for developing personalized intervention strategies for treatment of NAFLD based on host-gut microbiome ecosystem interactions, however, future studies with large sample size are needed to validate these discoveries. The Trial Registration Number is ISRCTN 42622771.

[Source](https://www.nature.com/articles/s41467-022-29968-0)

10.1038/s41467-022-29968-0

---

## Sex-dependent host-microbiome dynamics in zebrafish: Implications for toxicology and gastrointestinal physiology.
 09.05.2022, _CJ Martyniuk, AN Buerger, H Vespalcova, B Rudzanova, SR Sohag, AT Hanlon, PE Ginn, SL Craft, S Smetanova, E Budinska, JH Bisesi, O Adamovsky_


The physiology of males and females can be vastly different, complicating interpretation of toxicological and physiological data. The objectives of this study were to elucidate the sex differences in the microbiome-gastrointestinal (GI) transcriptome of adult zebrafish. We compared microbial composition and diversity in both males and females fed the same diet and housed in the same environment. There were no sex-specific differences in weight gain nor gastrointestinal morphology based on histopathology. There was no difference in gut microbial diversity, richness (Shannon and Chao1 index) nor predicted functional composition of the microbiome between males and females. Prior to post-hoc correction, male zebrafish showed higher abundance for the bacterial families Erythrobacteraceae and Lamiaceae, both belonging to the phyla Actinobacteria and Proteobacteria. At the genus level, Lamia and Altererythrobacter were more dominant in males and an unidentified genus in Bacteroidetes was more abundant in females. There were 16 unique differentially expressed transcripts in the gastrointestinal tissue between male and female zebrafish (FDR corrected, p &lt; 0.05). Relative to males, the mRNA expression for trim35-9, slc25a48, chchd3b, csad, and hsd17b3 were lower in female GI while cyp2k6, adra2c, and bckdk were higher in the female GI. Immune and lipid-related gene network expression differed between the sexes (i.e., cholesterol export and metabolism) as well as networks related to gastric motility, gastrointestinal system absorption and digestion. Such data provide clues as to putative differences in gastrointestinal physiology between male and female zebrafish. This study identifies host-transcriptome differences that can be considered when interpreting the microgenderome of zebrafish in studies investigating GI physiology and toxicology of fishes.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1744117X22000351)

10.1016/j.cbd.2022.100993

---

## Comprehensive 16S rRNA and metagenomic data from the gut microbiome of aging and rejuvenation mouse models.
 13.05.2022, _J Shin, JR Noh, D Choe, N Lee, Y Song, S Cho, EJ Kang, MJ Go, SK Ha, JH Kim, YH Kim, KS Kim, BC Kim, CH Lee, BK Cho_


_Aging, Animals, Disease Models, Animal, Gastrointestinal Microbiome, Metagenomics, Mice, RNA, Ribosomal, 16S, Rejuvenation_
The gut microbiota is associated with the health and longevity of the host. A few methods, such as fecal microbiota transplantation and oral administration of probiotics, have been applied to alter the gut microbiome and promote healthy aging. The changes in host microbiomes still remain poorly understood. Here, we characterized both the changes in gut microbial communities and their functional potential derived from colon samples in mouse models during aging. We achieved this through four procedures including co-housing, serum injection, parabiosis, and oral administration of Akkermansia muciniphila as probiotics using bacterial 16 S rRNA sequencing and shotgun metagenomic sequencing. The dataset comprised 16 S rRNA sequencing (36,249,200 paired-end reads, 107 sequencing data) and metagenomic sequencing data (307,194,369 paired-end reads, 109 sequencing data), characterizing the taxonomy of bacterial communities and their functional potential during aging and rejuvenation. The generated data expand the resources of the gut microbiome related to aging and rejuvenation and provide a useful dataset for research on developing therapeutic strategies to achieve healthy active aging.

[Source](https://www.nature.com/articles/s41597-022-01308-3)

10.1038/s41597-022-01308-3

---

## A total infectome approach to understand the etiology of infectious disease in pigs.
 12.05.2022, _X Huang, W Wu, X Tian, X Hou, X Cui, Y Xiao, Q Jiao, P Zhou, L Liu, W Shi, L Chen, Y Sun, Y Yang, J Chen, G Zhang, J Liu, EC Holmes, X Cai, T An, M Shi_


_Animals, Bacteria, Coinfection, Communicable Diseases, DNA Viruses, RNA Viruses, Swine_
The global pork industry is continuously affected by infectious diseases that can result in large-scale mortality, trade restrictions, and major reductions in production. Nevertheless, the cause of many infectious diseases in pigs remains unclear, largely because commonly used diagnostic tools fail to capture the full diversity of potential pathogens and because pathogen co-infection is common.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01265-4)

10.1186/s40168-022-01265-4

---

## Synergistic interaction of gut microbiota enhances the growth of nematode through neuroendocrine signaling.
 12.05.2022, _WS Lo, Z Han, H Witte, W Röseler, RJ Sommer_


_Animals, Bacteria, Caenorhabditis elegans, Gastrointestinal Microbiome, Nematoda, Rhabditida, Signal Transduction, Transforming Growth Factor beta_
Animals are associated with a diverse bacterial community that impacts host physiology. It is well known that nutrients and enzymes synthesized by bacteria largely expand host metabolic capacity. Bacteria also impact a wide range of animal physiology that solely depends on host genetics through direct interaction. However, studying the synergistic effects of the bacterial community remains challenging due to its complexity. The omnivorous nematode Pristionchus pacificus has limited digestive efficiency on bacteria. Therefore, we established a bacterial collection that represents the natural gut microbiota that are resistant to digestion. Using this collection, we show that the bacterium Lysinibacillus xylanilyticus by itself provides limited nutritional value, but in combination with Escherichia coli, it significantly promotes life-history traits of P. pacificus by regulating the neuroendocrine peptide in sensory neurons. This gut-to-brain communication depends on undigested L. xylanilyticus providing Pristionchus nematodes a specific fitness advantage to compete with nematodes that rupture bacteria efficiently. Using RNA-seq and CRISPR-induced mutants, we show that 1-h exposure to L. xylanilyticus is sufficient to stimulate the expression of daf-7-type TGF-β signaling ligands, which induce a global transcriptome change. In addition, several effects of L. xylanilyticus depend on TGF-β signaling, including olfaction, body size regulation, and a switch of energy allocation from lipid storage to reproduction. Our results reveal the beneficial effects of a gut bacterium to modify life-history traits and maximize nematode survival in natural habitats.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0960982222004900)

10.1016/j.cub.2022.03.056

---

## Unraveling global and diazotrophic bacteriomes of boreal forest floor feather mosses and their environmental drivers at the ecosystem and at the plant scale in North America.
 09.05.2022, _M Renaudin, I Laforest-Lapointe, JP Bellenger_


Feather mosses are abundant cryptogams of the boreal forest floor and shelter a broad diversity of bacteria who have important ecological functions (e.g., decomposition, nutrient cycling). In particular, nitrogen (N2-) fixation performed by feather moss-associated diazotrophs constitutes an important entry of nitrogen in the boreal forest ecosystem. However, the composition of the feather moss bacteriome and its environmental drivers are still unclear. Using cDNA amplicon sequencing of the 16S rRNA and nifH genes and cyanobacterial biomass quantification, we explored the active global and diazotrophic bacterial communities of two dominant feather moss species (i) at the ecosystem scale, along a 500-km climatic and nutrient deposition gradient in the North American boreal forest, and (ii) at the plant scale, along the moss shoot senescence gradient. We found that cyanobacteria were major actors of the feather moss bacteriome, accounting for 33% of global bacterial communities and 65% of diazotrophic communities, and that several cyanobacterial and methanotrophic genera were contributing to N2-fixation. Moreover, we showed that bacteria were occupying ecological niches along the moss shoot, with phototrophs being dominant in the apical part and methanotrophs being dominant in the basal part. Finally, climate (temperature, precipitation), environmental variables (moss species, month, tree density) and nutrients (nitrogen, phosphorus, molybdenum, vanadium, iron) strongly shaped global and diazotrophic bacteriomes. In summary, this work presents evidence that the feather moss bacteriome plays crucial roles in supporting moss growth, health, and decomposition, as well as in the boreal forest carbon and nitrogen cycles. This study also highlights the substantial effects of climate and nutrients on the feather moss bacteriome, suggesting the importance of understanding the impacts of global change on moss-associated bacterial growth and activity.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969722028583)

10.1016/j.scitotenv.2022.155761

---

## Antimicrobial resistance in Antarctica: is it still a pristine environment?

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01250-x)

---

## Evaluating metagenomic assembly approaches for biome-specific gene catalogues.
 10.05.2022, _LF Delgado, AF Andersson_


_Algorithms, Cluster Analysis, Ecosystem, Metagenome, Metagenomics_
For many environments, biome-specific microbial gene catalogues are being recovered using shotgun metagenomics followed by assembly and gene calling on the assembled contigs. The assembly is typically conducted either by individually assembling each sample or by co-assembling reads from all the samples. The co-assembly approach can potentially recover genes that display too low abundance to be assembled from individual samples. On the other hand, combining samples increases the risk of mixing data from closely related strains, which can hamper the assembly process. In this respect, assembly on individual samples followed by clustering of (near) identical genes is preferable. Thus, both approaches have potential pros and cons, but it remains to be evaluated which assembly strategy is most effective. Here, we have evaluated three assembly strategies for generating gene catalogues from metagenomes using a dataset of 124 samples from the Baltic Sea: (1) assembly on individual samples followed by clustering of the resulting genes, (2) co-assembly on all samples, and (3) mix assembly, combining individual and co-assembly.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01259-2)

10.1186/s40168-022-01259-2

---

## Compare the performance of multiple binary classification models in microbial high-throughput sequencing datasets.
 10.05.2022, _N Xu, Z Zhang, Y Shen, Q Zhang, Z Liu, Y Yu, Y Wang, C Lei, M Ke, D Qiu, T Lu, Y Chen, J Xiong, H Qian_


The development of machine learning and deep learning provided solutions for predicting microbiota response on environmental change based on microbial high-throughput sequencing. However, there were few studies specifically clarifying the performance and practical of two types of binary classification models to find a better algorithm for the microbiota data analysis. Here, for the first time, we evaluated the performance, accuracy and running time of the binary classification models built by three machine learning methods - random forest (RF), support vector machine (SVM), logistic regression (LR), and one deep learning method - back propagation neural network (BPNN). The built models were based on the microbiota datasets that removed low-quality variables and solved the class imbalance problem. Additionally, we optimized the models by tuning. Our study demonstrated that dataset pre-processing was a necessary process for model construction. Among these 4 binary classification models, BPNN and RF were the most suitable methods for constructing microbiota binary classification models. Using these 4 models to predict multiple microbial datasets, BPNN showed the highest accuracy and the most robust performance, while the RF method was ranked second. We also constructed the optimal models by adjusting the epochs of BPNN and the n_estimators of RF for six times. The evaluation related to performances of models provided a road map for the application of artificial intelligence to assess microbial ecology.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969722029047)

10.1016/j.scitotenv.2022.155807

---

