## Systematic Review: The Gut Microbiome and Its Potential Clinical Application in Inflammatory Bowel Disease.
 05.05.2021, _L Aldars-García, M Chaparro, JP Gisbert_


Inflammatory bowel disease (IBD) is a chronic relapsing-remitting systemic disease of the gastrointestinal tract. It is well established that the gut microbiome has a profound impact on IBD pathogenesis. Our aim was to systematically review the literature on the IBD gut microbiome and its usefulness to provide microbiome-based biomarkers. A systematic search of the online bibliographic database PubMed from inception to August 2020 with screening in accordance with the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines was conducted. One-hundred and forty-four papers were eligible for inclusion. There was a wide heterogeneity in microbiome analysis methods or experimental design. The IBD intestinal microbiome was generally characterized by reduced species richness and diversity, and lower temporal stability, while changes in the gut microbiome seemed to play a pivotal role in determining the onset of IBD. Multiple studies have identified certain microbial taxa that are enriched or depleted in IBD, including bacteria, fungi, viruses, and archaea. The two main features in this sense are the decrease in beneficial bacteria and the increase in pathogenic bacteria. Significant differences were also present between remission and relapse IBD status. Shifts in gut microbial community composition and abundance have proven to be valuable as diagnostic biomarkers. The gut microbiome plays a major role in IBD, yet studies need to go from casualty to causality. Longitudinal designs including newly diagnosed treatment-naïve patients are needed to provide insights into the role of microbes in the onset of intestinal inflammation. A better understanding of the human gut microbiome could provide innovative targets for diagnosis, prognosis, treatment and even cure of this relevant disease.

[Source](https://doi.org/10.3390/microorganisms9050977)

10.3390/microorganisms9050977

---

## Millet-based supplement restored gut microbial diversity of acute malnourished pigs.
 07.05.2021, _X Li, Y Hui, B Leng, J Ren, Y Song, L Che, X Peng, B Huang, S Liu, L Li, DS Nielsen, Y Li, X Dai, S Zhao_


The tight association between malnutrition and gut microbiota (GM) dysbiosis enables microbiota-targeting intervention to be a promising strategy. Thus, we used a malnourished pig model to investigate the host response and GM alterations under different diet supplementation strategies. Pigs at age of 4 weeks were fed with pure maize diet to induce malnutrition symptoms, and followed by continuous feeding with maize (Maize, n = 8) or re-feeding using either corn-soy-blend (CSB+, n = 10) or millet-soy-blend based (MSB+, n = 10) supplementary food for 3 weeks. Meanwhile, 8 pigs were fed on a standard formulated ration as control (Ref). The effect of nutritional supplementation was assessed by the growth status, blood chemistry, gastrointestinal pathology, mucosal microbiota composition and colon production of short-chain fatty acids. Compared with purely maize-fed pigs, both CSB+ and MSB+ elevated the concentrations of total protein and globulin in blood. These pigs still showed most malnutrition symptoms after the food intervention period. MSB+ had superior influence on the GM development, exhibiting better performance in both structural and functional aspects. MSB+ pigs were colonized by less Proteobacteria but more Bacteroidetes, Firmicutes and Lachnospira spp. Pearson's correlation analysis indicated a strong correlation between the abundance of mucosal e.g., Faecalibacterium and Lachnospira spp. and body weight, crown-rump length and total serum protein. In conclusion, the malnutrition symptoms were accompanied by an aberrant GM, and millet-based nutritional supplementation showed promising potentials to restore the reduced GM diversity implicated in pig malnutrition.

[Source](https://doi.org/10.1371/journal.pone.0250423)

10.1371/journal.pone.0250423

---

## A mixture of quebracho and chestnut tannins drives butyrate-producing bacteria populations shift in the gut microbiota of weaned piglets.
 07.05.2021, _F Miragoli, V Patrone, A Prandini, S Sigolo, M Dell'Anno, L Rossi, M Barbato, A Senizza, L Morelli, ML Callegari_


Weaning is a critical period for piglets, in which unbalanced gut microbiota and/or pathogen colonisation can contribute to diseases that interfere with animal performance. Tannins are natural compounds that could be used as functional ingredients to improve gut health in pig farming thanks to their antibacterial, antioxidant, and antidiarrhoeal properties. In this study, a mixture of quebracho and chestnut tannins (1.25%) was evaluated for its efficacy in reducing the negative weaning effects on piglet growth. Microbiota composition was assessed by Illumina MiSeq 16S rRNA gene sequencing of DNA extracted from stools at the end of the trial. Sequence analysis revealed an increase in the genera Shuttleworthia, Pseudobutyrivibrio, Peptococcus, Anaerostipes, and Solobacterium in the tannin-supplemented group. Conversely, this dietary intervention reduced the abundance of the genera Syntrophococcus, Atopobium, Mitsuokella, Sharpea, and Prevotella. The populations of butyrate-producing bacteria were modulated by tannin, and higher butyrate concentrations in stools were detected in the tannin-fed pigs. Co-occurrence analysis revealed that the operational taxonomic units (OTUs) belonging to the families Veillonellaceae, Lachnospiraceae, and Coriobacteriaceae occupied the central part of the network in both the control and the tannin-fed animals. Instead, in the tannin group, the OTUs belonging to the families Acidaminococcaceae, Alcaligenaceae, and Spirochaetaceae characterised its network, whereas Family XIII Incertae Sedis occupied a more central position than in the control group. Conversely, the presence of Desulfovibrionaceae characterised the network of the control group, and this family was not present in the network of the tannin group. Moreover, the prediction of metabolic pathways revealed that the gut microbiome of the tannin group possessed an enhanced potential for carbohydrate transport and metabolism, as well as a lower abundance of pathways related to cell wall/membrane/envelope biogenesis and inorganic ion transport. In conclusion, the tested tannins seem to modulate the gut microbiota, favouring groups of butyrate-producing bacteria.

[Source](https://doi.org/10.1371/journal.pone.0250874)

10.1371/journal.pone.0250874

---

## Dynamic changes occur in the DNA gut virome of female cynomolgus macaques during aging

[Source](https://doi.org/10.1002/mbo3.1186)

---

## Virome Diversity among Mosquito Populations in a Sub-Urban Region of Marseille, France

[Source](https://doi.org/10.3390/v13050768)

---

## Mapping the microbial diversity and natural resistome of north Antarctica soils

[Source](https://doi.org/10.1101/2021.05.05.442734)

---

## Community dynamics in rhizosphere microorganisms at different development stages of wheat growing in confined isolation environments

[Source](https://doi.org/10.1007/s00253-021-11283-1)

---

## Bioprospecting Extreme Ecosystems Before They Vanish: The (Poorly Studied) Microbiology of Tropical Glaciers

[Source](https://www.taylorfrancis.com/chapters/edit/10.1201/9780429343452-1/bioprospecting-extreme-ecosystems-vanish-poorly-studied-microbiology-tropical-glaciers-luis-andr%C3%A9s-yarz%C3%A1bal)

---

## Investigating variability in microbial community composition in replicate environmental DNA samples down lake sediment cores

[Source](https://doi.org/10.1371/journal.pone.0250783)

---

## Benchmarking metagenomics classifiers on ancient viral DNA: a simulation study

[Source](https://doi.org/10.1101/2021.04.30.442132)

---

## Network analysis methods for studying microbial communities: A mini review

[Source](https://doi.org/10.1016/j.csbj.2021.05.001)

---

## Improving metagenomic binning results with overlapped bins using assembly graphs

[Source](https://doi.org/10.1186/s13015-021-00185-6)

---

## Current challenges in microbiome metadata collection

[Source](https://doi.org/10.1101/2021.05.05.442781)

---

