## Mechanism of Dyspnea during Exercise in Children with Corrected Congenital Heart Disease.
 24.01.2022, _M Chlif, MM Ammar, NB Said, L Sergey, S Ahmaidi, F Alassery, H Hamam_


_Child, Dyspnea, Exercise, Exercise Test, Heart Defects, Congenital, Humans, Oxygen Consumption_
This study will evaluate cardiorespiratory and peripheral muscle function and their relationship with subjective dyspnea threshold after the surgical correction of congenital heart disease in children. Thirteen children with surgically repaired congenital heart disease were recruited. Each participant performed an incremental exercise test on a cycle ergometer until exhaustion. Gas exchanges were continuously sampled to measure the maximal aerobic parameters and ventilatory thresholds. The functional capacity of the subjects was assessed with a 6 min walk test. At the end of the exercise test, isokinetic Cybex Norm was used to evaluate the strength and endurance of the knee extensor muscle in the leg. Dyspnea was subjectively scored with a visual analog scale during the last 15 s of each exercise step. Oxygen consumption measured at the dyspnea score/VO2 relationship located at the dyspnea threshold, at which dyspnea suddenly increased. Results: The maximal and submaximal values of the parameters describing the exercise and the peripheral muscular performances were: VO2 Peak: 33.8 ± 8.9 mL·min-1·kg-1; HR: 174 ± 9 b·min-1; VEmax: 65.68 ± 15.9 L·min-1; P max: 117 ± 27 W; maximal voluntary isometric force MVIF: 120.8 ± 41.9 N/m; and time to exhaustion Tlim: 53 ± 21 s. Oxygen consumption measured at the dyspnea threshold was related to VO2 Peak (R2 = 0.74; p &lt; 0.01), Tlim (R2 = 0.78; p &lt; 0.01), and the distance achieved during the 6MWT (R2 = 0.57; p &lt; 0.05). Compared to the theoretical maximal values for the power output, VO2, and HR, the surgical correction did not repair the exercise performance. After the surgical correction of congenital heart disease, exercise performance was impeded by alterations of the cardiorespiratory function and peripheral local factors. A subjective evaluation of the dyspnea threshold is a reliable criterion that allows the prediction of exercise capacity in subjects suffering from congenital heart disease.

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

10.3390/ijerph19010099

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## @AmyDWillis

[Source](https://twitter.com/AmyDWillis?ref_src=twsrc%5Etfw)

---

## @clannabel7

[Source](https://twitter.com/clannabel7?ref_src=twsrc%5Etfw)

---

## @bahti_yil

[Source](https://twitter.com/bahti_yil?ref_src=twsrc%5Etfw)

---

## @LabAllaband

[Source](https://twitter.com/LabAllaband?ref_src=twsrc%5Etfw)

---

## @v_heidrich

[Source](https://twitter.com/v_heidrich?ref_src=twsrc%5Etfw)

---

## https://t.co/Qf7MuAWEyF

[Source](https://t.co/Qf7MuAWEyF)

---

## pic.twitter.com/Frm2FPjqEd

[Source](https://t.co/Frm2FPjqEd)

---

## December 6, 2022

[Source](https://twitter.com/MicrobiomeVIF/status/1600084727561527296?ref_src=twsrc%5Etfw)

---

## Experimental temperatures shape host microbiome diversity and composition

[Source](https://onlinelibrary.wiley.com/doi/10.1111/gcb.16429)

---

## Global biogeography and projection of soil antibiotic resistance genes.
 02.12.2022, _D Zheng, G Yin, M Liu, L Hou, Y Yang, TP Van Boeckel, Y Zheng, Y Li_


_Humans, Soil, Anti-Bacterial Agents, Soil Microbiology, Genes, Bacterial, Drug Resistance, Microbial_
Although edaphic antibiotic resistance genes (ARGs) pose serious threats to human well-being, their spatially explicit patterns and responses to environmental constraints at the global scale are not well understood. This knowledge gap is hindering the global action plan on antibiotic resistance launched by the World Health Organization. Here, a global analysis of 1088 soil metagenomic samples detected 558 ARGs in soils, where ARG abundance in agricultural habitats was higher than that in nonagricultural habitats. Soil ARGs were mostly carried by clinical pathogens and gut microbes that mediated the control of climatic and anthropogenic factors to ARGs. We generated a global map of soil ARG abundance, where the identified microbial hosts, agricultural activities, and anthropogenic factors explained ARG hot spots in India, East Asia, Western Europe, and the United States. Our results highlight health threats from soil clinical pathogens carrying ARGs and determine regions prioritized to control soil antibiotic resistance worldwide.

[Source](https://www.science.org/doi/10.1126/sciadv.abq8015)

10.1126/sciadv.abq8015

---

## Genomic analysis of sewage from 101 countries reveals global landscape of antimicrobial resistance

[Source](https://www.nature.com/articles/s41467-022-34312-7)

---

## #globalsewage

[Source](https://twitter.com/hashtag/globalsewage?src=hash&ref_src=twsrc%5Etfw)

---

## #AMR

[Source](https://twitter.com/hashtag/AMR?src=hash&ref_src=twsrc%5Etfw)

---

## #sewage

[Source](https://twitter.com/hashtag/sewage?src=hash&ref_src=twsrc%5Etfw)

---

## Salt tolerance evolution facilitates antibiotic resistome in soil microbiota: Evidences from dissemination evaluation, hosts identification and co-occurrence exploration.
 08.12.2022, _Y Xu, G You, J Yin, M Zhang, D Peng, J Xu, S Yang, J Hou_


Salinity is considered as one of the vital factors affecting the profiles of antibiotic resistance genes (ARGs) in soils, whereby its roles in shaping the antibiotic resistome were still poorly understood. Here, metagenomic analysis was conducted to track the ARGs distributions and dissemination in soils during salt accumulation and desalinization processes. Neutral-salt accumulation for 45 and 90 days significantly increased the relative abundances of ARGs and mobile genetic elements (MGEs) carrying antibiotic resistance contigs (ARCs). The ARGs within antibiotic efflux and target protection families primarily carried by Streptomyces, Nocardioides, Rhodanobacter and Monashia were largely enriched by salinity. The ARGs subtypes of the resistance-nodulation-division (RND) family, ATP-binding cassette (ABC) family, rRNA methyltransferase and other efflux were closely associated with MGEs, contributing to the enrichment of ARGs. Moreover, the ARGs subtypes and transposons were genetically linked with the salt-tolerance mechanisms of organic osmolyte transporters and K+ uptake proteins on the same ARC, demonstrating the coselection of ARGs and halotolerant genes. Furthermore, the antibiotic resistome could recover to a normal state after the prolonged incubation by alleviating salt stress. Nevertheless, the acquisition of ARGs by opportunistic pathogens after salt treatment was increased, serving to prioritize further efforts on the health risks correlated with resistance propagation and human exposure in saline soils.

[Source](https://twitter.com/hashtag/ARGs?src=hash&ref_src=twsrc%5Etfw)

10.1016/j.envpol.2022.120830

---

## #ARGs

[Source](https://twitter.com/hashtag/ARGs?src=hash&ref_src=twsrc%5Etfw)

---

## @novonordiskfond

[Source](https://twitter.com/novonordiskfond?ref_src=twsrc%5Etfw)

---

## @EuropeVeo

[Source](https://twitter.com/EuropeVeo?ref_src=twsrc%5Etfw)

---

## https://t.co/rQLoADLEEv

[Source](https://t.co/rQLoADLEEv)

---

## pic.twitter.com/TZ1oeja7qS

[Source](https://t.co/TZ1oeja7qS)

---

## December 1, 2022

[Source](https://twitter.com/pat_munk/status/1598353862447943690?ref_src=twsrc%5Etfw)

---

## No evidence for a common blood microbiome based on a population study of 9,770 healthy humans

[Source](https://www.biorxiv.org/content/10.1101/2022.07.29.502098v2)

---

## Effects of dog ownership on the gut microbiota of elderly owners.
 08.12.2022, _C Jiang, Z Cui, P Fan, G Du_


Dog owners are usually in close contact with dogs. Whether dogs can affect the gut microbiota of elderly dog owners is worth studying. Data from 54 elderly (over 65 years of age) dog owners were screened from the American Gut Project. Owning a dog did not affect the α-diversity of the gut microbiota of the dog owner. Dog ownership significantly modulated the composition of the gut microbiota of the dog owner. The abundance of Actinobacteria was significantly increased. The abundances of Bifidobacteriaceae and Ruminococcaceae were significantly increased, while the abundance of Moracellaceae was significantly suppressed. In general, dog ownership can regulate the composition of gut microbiota and has a more significant effect on elderly males.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0278105)

10.1371/journal.pone.0278105

---

## Prokaryotic and viral genomes recovered from 787 Japanese gut metagenomes revealed microbial features linked to diets, populations, and diseases

[Source](https://www.cell.com/cell-genomics/fulltext/S2666-979X(22)00172-0)

---

## Gut microbiome-wide association study of depressive symptoms.
 06.12.2022, _D Radjabzadeh, JA Bosch, AG Uitterlinden, AH Zwinderman, MA Ikram, JBJ van Meurs, AI Luik, M Nieuwdorp, A Lok, CM van Duijn, R Kraaij, N Amin_


Depression is one of the most poorly understood diseases due to its elusive pathogenesis. There is an urgency to identify molecular and biological mechanisms underlying depression and the gut microbiome is a novel area of interest. Here we investigate the relation of fecal microbiome diversity and composition with depressive symptoms in 1,054 participants from the Rotterdam Study cohort and validate these findings in the Amsterdam HELIUS cohort in 1,539 subjects. We identify association of thirteen microbial taxa, including genera Eggerthella, Subdoligranulum, Coprococcus, Sellimonas, Lachnoclostridium, Hungatella, Ruminococcaceae (UCG002, UCG003 and UCG005), LachnospiraceaeUCG001, Eubacterium ventriosum and Ruminococcusgauvreauiigroup, and family Ruminococcaceae with depressive symptoms. These bacteria are known to be involved in the synthesis of glutamate, butyrate, serotonin and gamma amino butyric acid (GABA), which are key neurotransmitters for depression. Our study suggests that the gut microbiome composition may play a key role in depression.

[Source](https://www.nature.com/articles/s41467-022-34502-3)

10.1038/s41467-022-34502-3

---

## Gut Microbiome and Its Impact on Obesity and Obesity-Related Disorders.
 05.12.2022, _S Sankararaman, K Noriega, S Velayuthan, T Sferra, R Martindale_


The prevalence of overweight and obesity has been increasing worldwide at an alarming rate. Gut microbiota intimately influence host energy metabolism, and immune response. Studies indicate a prominent role of gut dysbiosis in propagating inflammation that is associated with the development of obesity and obesity-related disorders such as type 2 diabetes mellitus, metabolic syndrome, and non-alcoholic fatty liver disease. This article will review the current literature on gut microbiome and its impact on obesity and obesity-related disorders.

[Source](https://www.biorxiv.org/content/10.1101/2022.12.02.518833v1)

10.1007/s11894-022-00859-0

---

## Development of non-alcoholic steatohepatitis is associated with gut microbiota but not oxysterol synthesis

[Source](https://www.biorxiv.org/content/10.1101/2022.12.02.518833v1)

---

## The microbiome's fiber degradation profile and its relationship with the host diet.
 07.12.2022, _Y Cohen, E Borenstein_


_Humans, Dietary Fiber, Diet, Gastrointestinal Microbiome, Microbiota, Metagenome_
The relationship between the gut microbiome and diet has been the focus of numerous recent studies. Such studies aim to characterize the impact of diet on the composition of the microbiome, as well as the microbiome's ability to utilize various compounds in the diet and produce metabolites that may be beneficial for the host. Consumption of dietary fibers (DFs)-polysaccharides that cannot be broken down by the host's endogenous enzymes and are degraded primarily by members of the microbiome-is known to have a profound effect on the microbiome. Yet, a comprehensive characterization of microbiome compositional and functional shifts in response to the consumption of specific DFs is still lacking.

[Source](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-022-01461-6)

10.1186/s12915-022-01461-6

---

## <em>Lactobacillus reuteri </em>improves the development and maturation of fecal microbiota in piglets through mother-to-infant microbe and metabolite vertical transmission

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01336-6)

---

## Development of a live biotherapeutic throat spray with lactobacilli targeting respiratory viral infections.
 05.12.2022, _I Spacova, I De Boeck, E Cauwenberghs, L Delanghe, PA Bron, T Henkens, A Simons, I Gamgami, L Persoons, I Claes, MFL van den Broek, D Schols, P Delputte, S Coenen, V Verhoeven, S Lebeer_


Respiratory viruses such as influenza viruses, respiratory syncytial virus (RSV), and coronaviruses initiate infection at the mucosal surfaces of the upper respiratory tract (URT), where the resident respiratory microbiome has an important gatekeeper function. In contrast to gut-targeting administration of beneficial bacteria against respiratory viral disease, topical URT administration of probiotics is currently underexplored, especially for the prevention and/or treatment of viral infections. Here, we report the formulation of a throat spray with live lactobacilli exhibiting several in vitro mechanisms of action against respiratory viral infections, including induction of interferon regulatory pathways and direct inhibition of respiratory viruses. Rational selection of Lactobacillaceae strains was based on previously documented beneficial properties, up-scaling and industrial production characteristics, clinical safety parameters, and potential antiviral and immunostimulatory efficacy in the URT demonstrated in this study. Using a three-step selection strategy, three strains were selected and further tested in vitro antiviral assays and in formulations: Lacticaseibacillus casei AMBR2 as a promising endogenous candidate URT probiotic with previously reported barrier-enhancing and anti-pathogenic properties and the two well-studied model strains Lacticaseibacillus rhamnosus GG and Lactiplantibacillus plantarum WCFS1 that display immunomodulatory capacities. The three strains and their combination significantly reduced the cytopathogenic effects of RSV, influenza A/H1N1 and B viruses, and HCoV-229E coronavirus in co-culture models with bacteria, virus, and host cells. Subsequently, these strains were formulated in a throat spray and human monocytes were employed to confirm the formulation process did not reduce the interferon regulatory pathway-inducing capacity. Administration of the throat spray in healthy volunteers revealed that the lactobacilli were capable of temporary colonization of the throat in a metabolically active form. Thus, the developed spray with live lactobacilli will be further explored in the clinic as a potential broad-acting live biotherapeutic strategy against respiratory viral diseases.

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1751-7915.14189)

10.1111/1751-7915.14189

---

## Differential microbial assembly processes and co-occurrence networks in the soil-root continuum along an environmental gradient 

[Source](https://onlinelibrary.wiley.com/doi/10.1002/imt2.18)

---

## Co-occurrence and cooperation between comammox and anammox bacteria in a full-scale attached growth municipal wastewater treatment process

[Source](https://www.biorxiv.org/content/10.1101/2022.12.05.519185v1)

---

## @KatieCommits

[Source](https://twitter.com/KatieCommits?ref_src=twsrc%5Etfw)

---

## @irmarie_25

[Source](https://twitter.com/irmarie_25?ref_src=twsrc%5Etfw)

---

## @HRSDVA

[Source](https://twitter.com/HRSDVA?ref_src=twsrc%5Etfw)

---

## #comammox

[Source](https://twitter.com/hashtag/comammox?src=hash&ref_src=twsrc%5Etfw)

---

## #anammox

[Source](https://twitter.com/hashtag/anammox?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/5NAFuhpLcU

[Source](https://t.co/5NAFuhpLcU)

---

## December 7, 2022

[Source](https://twitter.com/watermicrobe/status/1600348215592644608?ref_src=twsrc%5Etfw)

---

## Full-scale activated sludge transplantation reveals a highly resilient community structure 

[Source](https://www.sciencedirect.com/science/article/pii/S0043135422013999)

---

## Towards complete representation of bacterial contents in metagenomic samples

[Source](https://arxiv.org/abs/2210.00098)

---

## Cultivation-independent genomes greatly expand taxonomic-profiling capabilities of mOTUs across various environments.
 07.12.2022, _HJ Ruscheweyh, A Milanese, L Paoli, N Karcher, Q Clayssen, MI Keller, J Wirbel, P Bork, DR Mende, G Zeller, S Sunagawa_


_Swine, Cattle, Animals, RNA, Ribosomal, 16S, Metagenome, Metagenomics, Microbiota, Soil_
Taxonomic profiling is a fundamental task in microbiome research that aims to detect and quantify the relative abundance of microorganisms in biological samples. Available methods using shotgun metagenomic data generally depend on the deposition of sequenced and taxonomically annotated genomes, usually from cultures of isolated strains, in reference databases (reference genomes). However, the majority of microorganisms have not been cultured yet. Thus, a substantial fraction of microbial community members remains unaccounted for during taxonomic profiling, particularly in samples from underexplored environments. To address this issue, we developed the mOTU profiler, a tool that enables reference genome-independent species-level profiling of metagenomes. As such, it supports the identification and quantification of both "known" and "unknown" species based on a set of select marker genes.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01410-z)

10.1186/s40168-022-01410-z

---

## @MicrobiomeJ

[Source](https://twitter.com/MicrobiomeJ?ref_src=twsrc%5Etfw)

---

## @hjruscheweyh

[Source](https://twitter.com/hjruscheweyh?ref_src=twsrc%5Etfw)

---

## @alessio_mila

[Source](https://twitter.com/alessio_mila?ref_src=twsrc%5Etfw)

---

## @BorkLab

[Source](https://twitter.com/BorkLab?ref_src=twsrc%5Etfw)

---

## The emergence of the tetrathionate reductase operon in the Escherichia coli/Shigella pan-genome.
 08.12.2022, _FG Adsit, TA Randall, J Locklear, DM Kurtz_


Escherichia coli pathogenic variants (pathovars) are generally characterized by defined virulence traits and are susceptible to the evolution of hybridized identities due to the considerable plasticity of the E. coli genome. We have isolated a strain from a purified diet intended for research animals that further demonstrates the ability of E. coli to acquire novel genetic elements leading potentially to emergent new pathovars. Utilizing next generation sequencing to obtain a whole genome profile, we report an atypical strain of E. coli, EcoFA807-17, possessing a tetrathionate reductase (ttr) operon, which enables the utilization of tetrathionate as an electron acceptor, thus facilitating respiration in anaerobic environments such as the mammalian gut. The ttr operon is a potent virulence factor for several enteric pathogens, most prominently Salmonella enterica. However, the presence of chromosomally integrated tetrathionate reductase genes does not appear to have been previously reported in wild-type E. coli or Shigella. Accordingly, it is possible that the appearance of this virulence factor may signal the evolution of new mechanisms of pathogenicity in E. coli and Shigella and may potentially alter the effectiveness of existing assays using tetrathionate reductase as a unique marker for the detection of Salmonella enterica.

[Source](https://twitter.com/pangenomics?ref_src=twsrc%5Etfw)

10.1002/mbo3.1333

---

## @pangenomics

[Source](https://twitter.com/pangenomics?ref_src=twsrc%5Etfw)

---

## @ZellerGroup

[Source](https://twitter.com/ZellerGroup?ref_src=twsrc%5Etfw)

---

## https://t.co/GGRWY2zW84

[Source](https://t.co/GGRWY2zW84)

---

## December 5, 2022

[Source](https://twitter.com/SunagawaLab/status/1599757566506323968?ref_src=twsrc%5Etfw)

---

## Database Resources of the National Genomics Data Center, China National Center for Bioinformation in 2023 – CNCB-NGDC Members and Partners 

[Source](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac1073/6845434?login=false)

---

## Nanopore long-read-only metagenomics enables complete and high-quality genome reconstruction from mock and complex metagenomes.
 05.12.2022, _L Liu, Y Yang, Y Deng, T Zhang_


_Metagenome, Metagenomics, Nanopores, RNA, Ribosomal, 16S, Sewage, Microbiota, Prophages_
The accurate and comprehensive analyses of genome-resolved metagenomics largely depend on the reconstruction of reference-quality (complete and high-quality) genomes from diverse microbiomes. Closing gaps in draft genomes have been approaching with the inclusion of Nanopore long reads; however, genome quality improvement requires extensive and time-consuming high-accuracy short-read polishing.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01415-8)

10.1186/s40168-022-01415-8

---

## @chris_zie

[Source](https://twitter.com/chris_zie?ref_src=twsrc%5Etfw)

---

## https://t.co/5mCdDF8faB

[Source](https://t.co/5mCdDF8faB)

---

## @AcademicChatter

[Source](https://twitter.com/AcademicChatter?ref_src=twsrc%5Etfw)

---

## #phdchat

[Source](https://twitter.com/hashtag/phdchat?src=hash&ref_src=twsrc%5Etfw)

---

## December 7, 2022

[Source](https://twitter.com/Prokaryota/status/1600473298365607937?ref_src=twsrc%5Etfw)

---

