## Affordable hands-on DNA sequencing and genotyping: an exercise for teaching DNA analysis to undergraduates.
 22.11.2013, _K Shah, S Thomas, A Stein_


_Base Sequence, DNA, Genotype, Genotyping Techniques, Humans, Molecular Biology, Molecular Sequence Data, Polymerase Chain Reaction, Polymorphism, Single Nucleotide, Reproducibility of Results, Research, Sequence Analysis, DNA, Students, Teaching, Universities_
In this report, we describe a 5-week laboratory exercise for undergraduate biology and biochemistry students in which students learn to sequence DNA and to genotype their DNA for selected single nucleotide polymorphisms (SNPs). Students use miniaturized DNA sequencing gels that require approximately 8 min to run. The students perform G, A, T, C Sanger sequencing reactions. They prepare and run the gels, perform Southern blots (which require only 10 min), and detect sequencing ladders using a colorimetric detection system. Students enlarge their sequencing ladders from digital images of their small nylon membranes, and read the sequence manually. They compare their reads with the actual DNA sequence using BLAST2. After mastering the DNA sequencing system, students prepare their own DNA from a cheek swab, polymerase chain reaction-amplify a region of their DNA that encompasses a SNP of interest, and perform sequencing to determine their genotype at the SNP position. A family pedigree can also be constructed. The SNP chosen by the instructor was rs17822931, which is in the ABCC11 gene and is the determinant of human earwax type. Genotypes at the rs178229931 site vary in different ethnic populations.

[Source](https://apply.interfolio.com/85943)

10.1002/bmb.20732

---

## Biological Sciences Laboratory Instructor Position

[Source](https://apply.interfolio.com/85943)

---

## Phylogenetic Core Groups: a promising concept in search of a consistent methodological framework : Comment to ``A conceptual framework for the phylogenetically-constrained assembly of microbial communities''.
 28.03.2021, _A Pascual-García_


In this comment, we analyse the conceptual framework proposed by Aguirre de Cárcer (Microbiome 7:142, 2019), introducing the novel concept of Phylogenetic Core Groups (PCGs). This notion aims to complement the traditional classification in operational taxonomic units (OTUs), widely used in microbial ecology, to provide a more intrinsic taxonomical classification which avoids the use of pre-determined thresholds. However, to introduce this concept, the author frames his proposal in a wider theoretical framework based on a conceptualization of selection that we argue is a tautology. This blurs the subsequent formulation of an assembly principle for microbial communities, favouring that some contradictory examples introduced to support the framework appear aligned in their conclusions. And more importantly, under this framework and its derived methodology, it is not possible to infer PCGs from data in a consistent way. We reanalyse the proposal to identify its logical and methodological flaws and, through the analysis of synthetic scenarios, we propose a number of methodological refinements to contribute towards the determination of PCGs in a consistent way. We hope our analysis will promote the exploration of PCGs as a potentially valuable tool, helping to bridge the gap between environmental conditions and community composition in microbial ecology. Video Abstract.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01023-y)

10.1186/s40168-021-01023-y

---

## Phylogenetic Core Groups: a promising concept in search of a consistent methodological framework

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01023-y)

---

## Differential viral RNA methylation contributes to pathogen blocking in <em>Wolbachia</em>-colonized arthropod

[Source](https://www.biorxiv.org/content/10.1101/2021.03.26.437201v1)

---

## Isolation and characterization of a novel Wolbachia bacteriophage from <em>Allonemobius socius</em> crickets in Missouri

[Source](https://www.biorxiv.org/content/10.1101/2021.03.31.437854v1)

---

## From Plants to Ants: Fungal Modification of Leaf Lipids for Nutrition and Communication in the Leaf-Cutter Ant Fungal Garden Ecosystem.
 24.03.2021, _L Khadempour, JE Kyle, BM Webb-Robertson, CD Nicora, FB Smith, RD Smith, MS Lipton, CR Currie, ES Baker, KE Burnum-Johnson_


Lipids are essential to all living organisms, as an energy source, as an important cellular structural component, and as a communication tool. In this study, we used global lipidomic methods to evaluate the lipids in leaf-cutter ant fungal gardens. Leaf-cutter ants and their coevolved fungal cultivar, Leucoagaricus gongylophorus, are a model mutualistic system. The fungus enzymatically digests fresh plant material that the ants cut and deliver, converting energy and nutrients from plants and providing them to the ants through specialized hyphal swellings called gongylidia. Using combined liquid chromatography, ion mobility spectrometry, and tandem mass spectrometry, we evaluated differences between the molecular species of lipids in the leaf-cutter ant fungal garden ecosystem. This lipidomic study characterized leaves that are fed to the gardens, gongylidia that are produced by the fungus to feed the ants, and spatially resolved regions of the fungal garden through stages of leaf degradation. Lipids containing alpha-linolenic acid (18:3) were enriched in leaves and the top of the gardens but not dominant in the middle or bottom regions. Gongylidia were dominated by lipids containing linoleic acid (18:2). To evaluate the communicative potential of the lipids in fungal gardens, we conducted a behavioral experiment that showed Atta leaf-cutter ants responded differently to 18:3 and 18:2 fatty acids, with aggression toward 18:3 and attraction for 18:2. This work demonstrates the role of lipids in both the transfer of energy and as an interkingdom communication tool in leaf-cutter ant fungal gardens.IMPORTANCE In this work, we examined the role of lipids in the mutualism between leaf-cutter ants and fungus. These ants cut fresh leaf material, which they provide to their fungal cultivar, that converts energy and nutrients from the plants and provides it to the ants in specialized hyphal swellings called gongylidia. This work constitutes the first example of a global lipidomics study of a symbiotic system and provides insights as to how the fungus modifies plant lipids into a usable source for the ants. Through a behavioral experiment, this work also demonstrates how lipids can be used as an interkingdom communication tool, in this case, as an attractant rather than as a repellant, which is more often seen.

[Source](https://msystems.asm.org/content/6/2/e01307-20)

10.1128/mSystems.01307-20

---

## From plants to ants: Fungal modification of leaf lipids for nutrition and communication in the leaf-cutter ant fungal garden ecosystem

[Source](https://msystems.asm.org/content/6/2/e01307-20)

---

## Experimental evolution-driven identification of Arabidopsis rhizosphere competence genes in <em>Pseudomonas protegens</em>

[Source](https://www.biorxiv.org/content/10.1101/2020.12.01.407551v3.abstract)

---

## Metagenomic profile of microbial communities in a drinking water storage tank sediment after sequential exposure to monochloramine, free chlorine, and monochloramine

[Source](https://pubs.acs.org/doi/10.1021/acsestwater.1c00016)

---

## A 16S rRNA Gene and Draft Genome Database for the Murine Oral Bacterial Community.
 21.02.2021, _S Joseph, J Aduse-Opoku, A Hashim, E Hanski, R Streich, SCL Knowles, AB Pedersen, WG Wade, MA Curtis_


A curated murine oral microbiome database to be used as a reference for mouse-based studies has been constructed using a combination of bacterial culture, 16S rRNA gene amplicon, and whole-genome sequencing. The database comprises a collection of nearly full-length 16S rRNA gene sequences from cultured isolates and draft genomes from representative taxa collected from a range of sources, including specific-pathogen-free laboratory mice, wild Mus musculusdomesticus mice, and formerly wild wood mouse Apodemus sylvaticus At present, it comprises 103 mouse oral taxa (MOT) spanning four phyla-Firmicutes, Proteobacteria, Actinobacteria, and Bacteroidetes-including 12 novel undescribed species-level taxa. The key observations from this study are (i) the low diversity and predominantly culturable nature of the laboratory mouse oral microbiome and (ii) the identification of three major murine-specific oral bacterial lineages, namely, Streptococcus danieliae (MOT10), Lactobacillus murinus (MOT93), and Gemella species 2 (MOT43), which is one of the novel, still-unnamed taxa. Of these, S. danieliae is of particular interest, since it is a major component of the oral microbiome from all strains of healthy and periodontally diseased laboratory mice, as well as being present in wild mice. It is expected that this well-characterized database should be a useful resource for in vitro experimentation and mouse model studies in the field of oral microbiology.IMPORTANCE Mouse model studies are frequently used in oral microbiome research, particularly to investigate diseases such as periodontitis and caries, as well as other related systemic diseases. We have reported here the details of the development of a curated reference database to characterize the oral microbial community in laboratory and some wild mice. The genomic information and findings reported here can help improve the outcomes and accuracy of host-microbe experimental studies that use murine models to understand health and disease. Work is also under way to make the reference data sets publicly available on a web server to enable easy access and downloading for researchers across the world.

[Source](https://msystems.asm.org/content/6/1/e01222-20)

10.1128/mSystems.01222-20

---

## A 16S rRNA gene and draft genome database for the murine oral bacterial community

[Source](https://msystems.asm.org/content/6/1/e01222-20)

---

