## Resistome expansion in disease-associated human gut microbiomes.
 01.08.2023, _S Fredriksen, S de Warle, P van Baarlen, J Boekhorst, JM Wells_


_Humans, Gastrointestinal Microbiome, Microbiota, Anti-Bacterial Agents, Drug Resistance, Microbial, Genes, Bacterial_
The resistome, the collection of antibiotic resistance genes (ARGs) in a microbiome, is increasingly recognised as relevant to the development of clinically relevant antibiotic resistance. Many metagenomic studies have reported resistome differences between groups, often in connection with disease and/or antibiotic treatment. However, the consistency of resistome associations with antibiotic- and non-antibiotic-treated diseases has not been established. In this study, we re-analysed human gut microbiome data from 26 case-control studies to assess the link between disease and the resistome.

[Source](https://doi.org/10.1186/s40168-023-01610-1)

10.1186/s40168-023-01610-1

---

## A microbial causal mediation analytic tool for health disparity and applications in body mass index

[Source](https://doi.org/10.1186/s40168-023-01608-9)

---

## Alterations of gut microbiota in biopsy-proven diabetic nephropathy and a long history of diabetes without kidney damage

[Source](https://doi.org/10.1038/s41598-023-39444-4)

---

## Assessment of infant outgrowth of cowâ€™s milk allergy in relation to the faecal microbiome and metaproteome

[Source](https://doi.org/10.1038/s41598-023-39260-w)

---

## Analysis of bacterial vaginosis, the vaginal microbiome, and sexually transmitted infections following the provision of menstrual cups in Kenyan schools: Results of a nested study within a cluster randomized controlled trial

[Source](https://doi.org/10.1371/journal.pmed.1004258)

---

## Evolutionary modeling suggests that addictions may be driven by competition-induced microbiome dysbiosis

[Source](https://doi.org/10.1038/s42003-023-05099-0)

---

## Alterations in the gut microbiome implicate key taxa and metabolic pathways across inflammatory arthritis phenotypes

[Source](https://doi.org/10.1126/scitranslmed.abn4722)

---

## Increased genital mucosal cytokines in Canadian women associate with higher antigen-presenting cells, inflammatory metabolites, epithelial barrier disruption, and the depletion of <em>L. crispatus</em>

[Source](https://doi.org/10.1186/s40168-023-01594-y)

---

## Insulin alleviates murine colitis through microbiome alterations and bile acid metabolism

[Source](https://doi.org/10.1186/s12967-023-04214-3)

---

## The Vaginal Microbiota of Pregnant Women Varies with Gestational Age, Maternal Age, and Parity

[Source](https://doi.org/10.1128/spectrum.03429-22)

---

## Unique microbial landscape in the human oropharynx during different types of acute respiratory tract infections

[Source](https://doi.org/10.1186/s40168-023-01597-9)

---

## Probiotics to prevent necrotising enterocolitis in very preterm or very low birth weight infants

[Source](https://doi.org/10.1002/14651858.CD005496.pub6)

---

## A gut feeling for drugs that have metabolic benefits

[Source](https://doi.org/10.1038/s41467-023-40167-3)

---

## Antibiotic perturbations to the gut microbiome

[Source](https://doi.org/10.1038/s41579-023-00933-y)

---

## The global human gut microbiome: genes, lifestyles, and diet

[Source](https://doi.org/10.1016/j.molmed.2023.07.002)

---

## 5-Aminosalicylic acid alters the gut microbiota and altered microbiota transmitted vertically to offspring have protective effects against colitis

[Source](https://doi.org/10.1038/s41598-023-39491-x)

---

## Presence of an ultra-small microbiome in fermented cabbages

[Source](https://doi.org/10.7717/peerj.15680)

---

## The long-term gut bacterial signature of a wild primate is associated with a timing effect of pre- and postnatal maternal glucocorticoid levels

[Source](https://doi.org/10.1186/s40168-023-01596-w)

---

## Early-life prophylactic antibiotic treatment disturbs the stability of the gut microbiota and increases susceptibility to H9N2 AIV in chicks

[Source](https://doi.org/10.1186/s40168-023-01609-8)

---

## Human-influenced diets affect the gut microbiome of wild baboons

[Source](https://doi.org/10.1038/s41598-023-38895-z)

---

## Expanding the range of the respiratory infectome in Australian feedlot cattle with and without respiratory disease using metatranscriptomics

[Source](https://doi.org/10.1186/s40168-023-01591-1)

---

## Social network position is a major predictor of ant behavior, microbiota composition, and brain gene expression

[Source](https://doi.org/10.1371/journal.pbio.3002203)

---

## The root microbiome: techniques for exploration and agricultural applications

[Source](https://doi.org/10.2144/btn-2023-0057)

---

## Recently-formed Antarctic lakes host less diverse benthic bacterial and diatom communities than their older counterparts

[Source](https://doi.org/10.1093/femsec/fiad087)

---

## Greengenes2 unifies microbial data in a single reference tree

[Source](https://doi.org/10.1038/s41587-023-01845-1)

---

## CheckM2: a rapid, scalable and accurate tool for assessing microbial genome quality using machine learning

[Source](https://doi.org/10.1038/s41592-023-01940-w)

---

## Lectin-Seq: A method to profile lectin-microbe interactions in native communities.
 02.08.2023, _RL McPherson, CR Isabella, RL Walker, D Sergio, S Bae, T Gaca, S Raman, LTT Nguyen, DA Wesener, M Halim, MG Wuo, A Dugan, R Kerby, S Ghosh, FE Rey, C Dhennezel, G Pishchany, V Lensch, H Vlamakis, EJ Alm, RJ Xavier, LL Kiessling_


_Humans, Lectins, Immunity, Innate_
Soluble human lectins are critical components of innate immunity. Genetic models suggest that lectins influence host-resident microbiota, but their specificity for commensal and mutualist species is understudied. Elucidating lectins' roles in regulating microbiota requires an understanding of which microbial species they bind within native communities. To profile human lectin recognition, we developed Lectin-Seq. We apply Lectin-Seq to human fecal microbiota using the soluble mannose-binding lectin (MBL) and intelectin-1 (hItln1). Although each lectin binds a substantial percentage of the samples (10 to 20%), the microbial interactomes of MBL and hItln1 differ markedly in composition and diversity. MBL binding is highly selective for a small subset of species commonly associated with humans. In contrast, hItln1's interaction profile encompasses a broad range of lower-abundance species. Our data uncover stark differences in the commensal recognition properties of human lectins.

[Source](https://doi.org/10.1126/sciadv.add8766)

10.1126/sciadv.add8766

---

## A mixed-model approach for estimating drivers of microbiota community composition and differential taxonomic abundance.
 25.07.2023, _AR Sweeny, H Lemon, A Ibrahim, KA Watt, K Wilson, DZ Childs, DH Nussey, A Free, L McNally_


Next-generation sequencing (NGS) and metabarcoding approaches are increasingly applied to wild animal populations, but there is a disconnect between the widely applied generalized linear mixed model (GLMM) approaches commonly used to study phenotypic variation and the statistical toolkit from community ecology typically applied to metabarcoding data. Here, we describe the suitability of a novel GLMM-based approach for analyzing the taxon-specific sequence read counts derived from standard metabarcoding data. This approach allows decomposition of the contribution of different drivers to variation in community composition (e.g., age, season, individual) via interaction terms in the model random-effects structure. We provide guidance to implementing this approach and show how these models can identify how responsible specific taxonomic groups are for the effects attributed to different drivers. We applied this approach to two cross-sectional data sets from the Soay sheep population of St. Kilda. GLMMs showed agreement with dissimilarity-based approaches highlighting the substantial contribution of age and minimal contribution of season to microbiota community compositions, and simultaneously estimated the contribution of other technical and biological factors. We further used model predictions to show that age effects were principally due to increases in taxa of the phylum Bacteroidetes and declines in taxa of the phylum Firmicutes. This approach offers a powerful means for understanding the influence of drivers of community structure derived from metabarcoding data. We discuss how our approach could be readily adapted to allow researchers to estimate contributions of additional factors such as host or microbe phylogeny to answer emerging questions surrounding the ecological and evolutionary roles of within-host communities. IMPORTANCE NGS and fecal metabarcoding methods have provided powerful opportunities to study the wild gut microbiome. A wealth of data is, therefore, amassing across wild systems, generating the need for analytical approaches that can appropriately investigate simultaneous factors at the host and environmental scale that determine the composition of these communities. Here, we describe a generalized linear mixed-effects model (GLMM) approach to analyze read count data from metabarcoding of the gut microbiota, allowing us to quantify the contributions of multiple host and environmental factors to within-host community structure. Our approach provides outputs that are familiar to a majority of field ecologists and can be run using any standard mixed-effects modeling packages. We illustrate this approach using two metabarcoding data sets from the Soay sheep population of St. Kilda investigating age and season effects as worked examples.

[Source](https://doi.org/10.1128/msystems.00040-23)

10.1128/msystems.00040-23

---

