## No evidence of human genome integration of SARS-CoV-2 found by long-read DNA sequencing.
 13.08.2021, _N Smits, J Rasmussen, GO Bodea, AA Amarilla, P Gerdes, FJ Sanchez-Luque, P Ajjikuttira, N Modhiran, B Liang, J Faivre, IW Deveson, AA Khromykh, D Watterson, AD Ewing, GJ Faulkner_


A recent study proposed that severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) hijacks the LINE-1 (L1) retrotransposition machinery to integrate into the DNA of infected cells. If confirmed, this finding could have significant clinical implications. Here, we apply deep (&gt;50×) long-read Oxford Nanopore Technologies (ONT) sequencing to HEK293T cells infected with SARS-CoV-2 and do not find the virus integrated into the genome. By examining ONT data from separate HEK293T cultivars, we completely resolve 78 L1 insertions arising in vitro in the absence of L1 overexpression systems. ONT sequencing applied to hepatitis B virus (HBV)-positive liver cancer tissues located a single HBV insertion. These experiments demonstrate reliable resolution of retrotransposon and exogenous virus insertions by ONT sequencing. That we find no evidence of SARS-CoV-2 integration suggests that such events are, at most, extremely rare in vivo and therefore are unlikely to drive oncogenesis or explain post-recovery detection of the virus.

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(21)00961-X)

10.1016/j.celrep.2021.109530

---

## Ultrapotent antibodies against diverse and highly transmissible SARS-CoV-2 variants.
 13.08.2021, _L Wang, T Zhou, Y Zhang, ES Yang, CA Schramm, W Shi, A Pegu, OK Oloniniyi, AR Henry, S Darko, SR Narpala, C Hatcher, DR Martinez, Y Tsybovsky, E Phung, OM Abiona, A Antia, EM Cale, LA Chang, M Choe, KS Corbett, RL Davis, AT DiPiazza, IJ Gordon, SH Hait, T Hermanus, P Kgagudi, F Laboune, K Leung, T Liu, RD Mason, AF Nazzari, L Novik, S O'Connell, S O'Dell, AS Olia, SD Schmidt, T Stephens, CD Stringham, CA Talana, IT Teng, DA Wagner, AT Widge, B Zhang, M Roederer, JE Ledgerwood, TJ Ruckwardt, MR Gaudinski, PL Moore, NA Doria-Rose, RS Baric, BS Graham, AB McDermott, DC Douek, PD Kwong, JR Mascola, NJ Sullivan, J Misasi_


The emergence of highly transmissible SARS-CoV-2 variants of concern (VOCs) that are resistant to therapeutic antibodies highlights the need for continuing discovery of broadly reactive antibodies. We identified four receptor binding domain-targeting antibodies from three early-outbreak convalescent donors with potent neutralizing activity against 23 variants, including the B.1.1.7, B.1.351, P.1, B.1.429, B.1.526, and B.1.617 VOCs. Two antibodies are ultrapotent, with subnanomolar neutralization titers [half-maximal inhibitory concentration (IC50) 0.3 to 11.1 nanograms per milliliter; IC80 1.5 to 34.5 nanograms per milliliter). We define the structural and functional determinants of binding for all four VOC-targeting antibodies and show that combinations of two antibodies decrease the in vitro generation of escape mutants, suggesting their potential in mitigating resistance development.

[Source](https://science.sciencemag.org/content/373/6556/eabh1766)

10.1126/science.abh1766

---

## Long-term perturbation of the peripheral immune system months after SARS-CoV-2 infection

[Source](https://www.medrxiv.org/content/10.1101/2021.07.30.21261234v1)

---

## Bacterial colonization dynamics and antibiotic resistance gene dissemination in the hospital environment after first patient occupancy: a longitudinal metagenetic study.
 17.08.2021, _TE Klassert, R Leistner, C Zubiria-Barrera, M Stock, M López, R Neubert, D Driesch, P Gastmeier, H Slevogt_


Humans spend the bulk of their time in indoor environments. This space is shared with an indoor ecosystem of microorganisms, which are in continuous exchange with the human inhabitants. In the particular case of hospitals, the environmental microorganisms may influence patient recovery and outcome. An understanding of the bacterial community structure in the hospital environment is pivotal for the prevention of hospital-acquired infections and the dissemination of antibiotic resistance genes. In this study, we performed a longitudinal metagenetic approach in a newly opened ward at the Charité Hospital (Berlin) to characterize the dynamics of the bacterial colonization process in the hospital environment after first patient occupancy.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01109-7)

10.1186/s40168-021-01109-7

---

## Acute SARS-CoV-2 infection is associated with an expansion of bacteria pathogens in the nose including Pseudomonas Aeruginosa.
 25.05.2021, _NS Rhoades, A Pinski, AN Monsibais, A Jankeel, BM Doratt, IR Cinco, I Ibraim, I Messaoudi_


Much of the research conducted on SARS-CoV-2 and COVID-19 has focused on the systemic host response, especially that generated by severely ill patients. Very few studies have investigated the impact of acute SARS-CoV-2 within the nasopharynx, the site of initial infection and viral replication. In this study we profiled changes in the nasal microbial communities as well as in host transcriptional profile during acute SARS-CoV-2 infection using 16S amplicon sequencing and RNA sequencing. These analyses were coupled to viral genome sequencing. Our microbiome analysis revealed that the nasal microbiome of COVID patients was unique and was marked by an expansion of bacterial pathogens. Some of these microbes (i.e. Acinetobacter ) were shared with COVID negative health care providers from the same medical center but absent in COVID negative outpatients seeking care at the same institutions suggesting acquisition of nosocomial respiratory pathogens. Specifically, we report a distinct increase in the prevalence and abundance of the pathogen Pseudomonas aeruginosa in COVID patients that correlated with viral RNA load. These data suggest that the inflammatory environment caused by SARS-CoV-2 infection and potentially exposure to the hospital environment leads to an expansion of bacterial pathogens in the nasal cavity that could contribute to increased incidence of secondary bacterial infections. Additionally, we observed a robust host transcriptional response in the nasal epithelia of COVID patients, indicative of an antiviral innate immune repones and neuronal damage. Finally, analysis of viral genomes did not reveal an association between viral loads and viral sequences.

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(21)01080-9)

10.1101/2021.05.20.445008

---

## Acute SARS-CoV-2 infection is associated with an increased abundance of bacterial pathogens, including Pseudomonas aeruginosa in the nose

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(21)01080-9)

---

## Meta-analysis of Clinical Microbiome Studies in Urolithiasis Reveal Age, Stone Composition, and Study Location as the Predominant Factors in Urolithiasis-Associated Microbiome Composition.
 10.08.2021, _N Kachroo, D Lange, KL Penniston, J Stern, G Tasian, P Bajic, AJ Wolfe, M Suryavanshi, A Ticinesi, T Meschi, M Monga, AW Miller_


To determine whether functionally relevant questions associated with the urinary or gut microbiome and urinary stone disease (USD) can be answered from metagenome-wide association studies (MWAS), we performed the most comprehensive meta-analysis of published clinical MWAS in USD to date, using publicly available data published prior to April 2021. Six relevant studies met inclusion criteria. For alpha-diversity, significant differences were noted between USD status, stone composition, sample type, study location, age, diet, and sex. For beta-diversity, significant differences were noted by USD status, stone composition, sample type, study location, antibiotic use (30 days and 12 months before sampling), sex, hypertension, water intake, body habitus, and age. Prevotella and Lactobacillus in the gut and urinary tract, respectively, were associated with healthy individuals, while Enterobacteriaceae was associated with USD in the urine and stones. Paradoxically, other Prevotella strains were also strongly associated with USD in the gut microbiome. When data were analyzed together, USD status, stone composition, age group, and study location were the predominant factors associated with microbiome composition. Meta-analysis showed significant microbiome differences based on USD status, stone composition, age group or study location. However, analyses were limited by a lack of public data from published studies, metadata collected, and differing study protocols. Results highlight the need for field-specific standardization of experimental protocols in terms of sample collection procedures and the anatomical niches to assess, as well as in defining clinically relevant metadata and subphenotypes such as stone composition. IMPORTANCE Studies focused on the microbiome broadly support the hypothesis that the microbiome influences the onset of chronic diseases such as urinary stone disease. However, it is unclear what environmental factors shape the microbiome in ways that increase the risk for chronic disease. In addition, it is unclear how differences in study methodology can impact the results of clinical metagenome-wide association studies. In the current meta-analysis, we show that age, stone composition, and study location are the predominant factors that associate with the microbiome and USD status. Furthermore, we reveal differences in results based on specific analytical protocols, which impacts the interpretation of any microbiome study.

[Source](https://journals.asm.org/doi/10.1128/mBio.02007-21)

10.1128/mBio.02007-21

---

## The fecal microbiome and rotavirus vaccine immunogenicity in rural Zimbabwean infants

[Source](https://www.sciencedirect.com/science/article/pii/S0264410X21009816)

---

## Gut microbiome ADP-ribosyltransferases are widespread phage-encoded fitness factors

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(21)00344-9)

---

## Maternal γδ T Cells Shape Offspring Pulmonary Type-2 Immunity In A Microbiota-Dependent Manner

[Source](https://www.biorxiv.org/content/10.1101/2021.08.13.456265v1)

---

## Functional strain redundancy and persistent phage infection in Swiss hard cheese starter cultures

[Source](https://www.nature.com/articles/s41396-021-01071-0)

---

## Ecogenomics and Adaptation Strategies of Southern Ocean Viral Communities

[Source](https://journals.asm.org/doi/10.1128/mSystems.00396-21)

---

## Bog ecosystems as a playground for plant–microbe coevolution: bryophytes and vascular plants harbour functionally adapted bacteria

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01117-7)

---

## HoPhage: an ab initio tool for identifying hosts of phage fragments from metaviromes

[Source](https://academic.oup.com/bioinformatics/advance-articles)

---

## Sparse least trimmed squares regression with compositional covariates for high dimensional data.

[Source](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btab572/6343442)

---

## Ribovore: ribosomal RNA sequence analysis for GenBank submissions and database curation

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04316-z)

---

## Daily energy expenditure through the human life course

[Source](https://science.sciencemag.org/content/373/6556/808)

---

## Analysis of scientific society honors reveals disparities

[Source](https://www.cell.com/cell-systems/fulltext/S2405-4712(21)00285-4)

---

