## In Vitro and In Silico Based Approaches to Identify Potential Novel Bacteriocins from the Athlete Gut Microbiome of an Elite Athlete Cohort

[Source](https://www.mdpi.com/2076-2607/10/4/701)

---

## Cancer pharmacomicrobiomics: targeting microbiota to optimise cancer therapy outcomes.
 12.03.2022, _NL Ting, HC Lau, J Yu_


Despite the promising advances in novel cancer therapy such as immune checkpoint inhibitors (ICIs), limitations including therapeutic resistance and toxicity remain. In recent years, the relationship between gut microbiota and cancer has been extensively studied. Accumulating evidence reveals the role of microbiota in defining cancer therapeutic efficacy and toxicity. Unlike host genetics, microbiota can be easily modified via multiple strategies, including faecal microbiota transplantation (FMT), probiotics and antibiotics. Preclinical studies have identified the mechanisms on how microbes influence cancer treatment outcomes. Clinical trials have also demonstrated the potential of microbiota modulation in cancer treatments. Herein, we review the mechanistic insights of gut microbial interactions with chemotherapy and ICIs, particularly focusing on the interplay between gut bacteria and the pharmacokinetics (eg, metabolism, enzymatic degradation) or pharmacodynamics (eg, immunomodulation) of cancer treatment. The translational potential of basic findings in clinical settings is then explored, including using microbes as predictive biomarkers and microbial modulation by antibiotics, probiotics, prebiotics, dietary modulations and FMT. We further discuss the current limitations of gut microbiota modulation in patients with cancer and suggest essential directions for future study. In the era of personalised medicine, it is crucial to understand the microbiota and its interactions with cancer. Manipulating the gut microbiota to augment cancer therapeutic responses can provide new insights into cancer treatment.

[Source](https://gut.bmj.com/content/early/2022/03/10/gutjnl-2021-326264)

10.1136/gutjnl-2021-326264

---

## Inhibition of lung microbiota-derived proapoptotic peptides ameliorates acute exacerbation of pulmonary fibrosis.
 24.03.2022, _CN D'Alessandro-Gabazza, T Yasuma, T Kobayashi, M Toda, AM Abdel-Hamid, H Fujimoto, O Hataji, H Nakahara, A Takeshita, K Nishihama, T Okano, H Saiki, Y Okano, A Tomaru, V Fridman D'Alessandro, M Shiraishi, A Mizoguchi, R Ono, J Ohtsuka, M Fukumura, T Nosaka, X Mi, D Shukla, K Kataoka, Y Kondoh, M Hirose, T Arai, Y Inoue, Y Yano, RI Mackie, I Cann, EC Gabazza_


Idiopathic pulmonary fibrosis is an incurable disease of unknown etiology. Acute exacerbation of idiopathic pulmonary fibrosis is associated with high mortality. Excessive apoptosis of lung epithelial cells occurs in pulmonary fibrosis acute exacerbation. We recently identified corisin, a proapoptotic peptide that triggers acute exacerbation of pulmonary fibrosis. Here, we provide insights into the mechanism underlying the processing and release of corisin. Furthermore, we demonstrate that an anticorisin monoclonal antibody ameliorates lung fibrosis by significantly inhibiting acute exacerbation in the human transforming growth factorβ1 model and acute lung injury in the bleomycin model. By investigating the impact of the anticorisin monoclonal antibody in a general model of acute lung injury, we further unravel the potential of corisin to impact such diseases. These results underscore the role of corisin in the pathogenesis of acute exacerbation of pulmonary fibrosis and acute lung injury and provide a novel approach to treating this incurable disease.

[Source](https://www.nature.com/articles/s41467-022-29064-3)

10.1038/s41467-022-29064-3

---

## Gut microbiota and fermentation-derived branched chain hydroxy acids mediate health benefits of yogurt consumption in obese mice.
 16.03.2022, _N Daniel, RT Nachbar, TTT Tran, A Ouellette, TV Varin, A Cotillard, L Quinquis, A Gagné, P St-Pierre, J Trottier, B Marcotte, M Poirel, M Saccareau, MJ Dubois, P Joubert, O Barbier, H Koutnikova, A Marette_


Meta-analyses suggest that yogurt consumption reduces type 2 diabetes incidence in humans, but the molecular basis of these observations remains unknown. Here we show that dietary yogurt intake preserves whole-body glucose homeostasis and prevents hepatic insulin resistance and liver steatosis in a dietary mouse model of obesity-linked type 2 diabetes. Fecal microbiota transplantation studies reveal that these effects are partly linked to the gut microbiota. We further show that yogurt intake impacts the hepatic metabolome, notably maintaining the levels of branched chain hydroxy acids (BCHA) which correlate with improved metabolic parameters. These metabolites are generated upon milk fermentation and concentrated in yogurt. Remarkably, diet-induced obesity reduces plasma and tissue BCHA levels, and this is partly prevented by dietary yogurt intake. We further show that BCHA improve insulin action on glucose metabolism in liver and muscle cells, identifying BCHA as cell-autonomous metabolic regulators and potential mediators of yogurt's health effects.

[Source](https://www.nature.com/articles/s41467-022-29005-0)

10.1038/s41467-022-29005-0

---

## Gut microbiota determines the social behavior of mice and induces metabolic and inflammatory changes in their adipose tissue.
 22.09.2021, _O Agranyoni, S Meninger-Mordechay, A Uzan, O Ziv, M Salmon-Divon, D Rodin, O Raz, I Koman, O Koren, A Pinhasov, S Navon-Venezia_


_Adipose Tissue, Animals, Bacteria, Behavior, Animal, Body Weight, Depression, Fecal Microbiota Transplantation, Female, Gastrointestinal Microbiome, Germ-Free Life, Male, Mice, Phylogeny, RNA, Ribosomal, 16S, Sequence Analysis, RNA, Social Behavior_
The link between the gut microbiota and social behavior has been demonstrated, however the translational impact of a certain microbiota composition on stable behavioral patterns is yet to be elucidated. Here we employed an established social behavior mouse model of dominance (Dom) or submissiveness (Sub). A comprehensive 16S rRNA gene sequence analysis of Dom and Sub mice revealed a significantly different gut microbiota composition that clearly distinguishes between the two behavioral modes. Sub mice gut microbiota is significantly less diverse than that of Dom mice, and their taxa composition uniquely comprised the genera Mycoplasma and Anaeroplasma of the Tenericutes phylum, in addition to the Rikenellaceae and Clostridiaceae families. Conversely, the gut microbiota of Dom mice includes the genus Prevotella of the Bacteriodetes phylum, significantly less abundant in Sub mice. In addition, Sub mice show lower body weight from the age of 2 weeks and throughout their life span, accompanied with lower epididymis white adipose tissue (eWAT) mass and smaller adipocytes together with substantially elevated expression of inflammation and metabolic-related eWAT adipokines. Finally, fecal microbiota transplantation into germ-free mice show that Sub-transplanted mice acquired Sub microbiota and adopted their behavioral and physiological features, including depressive-like and anti-social behaviors alongside reduced eWAT mass, smaller adipocytes, and a Sub-like eWAT adipokine profile. Our findings demonstrate the critical role of the gut microbiome in determining dominance vs. submissiveness and suggest an association between gut microbiota, the eWAT metabolic and inflammatory profile, and the social behavior mode.

[Source](https://www.nature.com/articles/s41522-021-00193-9)

10.1038/s41522-021-00193-9

---

## The microbiota restrains neurodegenerative microglia in a model of amyotrophic lateral sclerosis

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01232-z)

---

## Spatial organization of the kelp microbiome at micron scales.
 29.03.2022, _ST Ramírez-Puebla, BL Weigel, L Jack, C Schlundt, CA Pfister, JL Mark Welch_


Elucidating the spatial structure of host-associated microbial communities is essential for understanding taxon-taxon interactions within the microbiota and between microbiota and host. Macroalgae are colonized by complex microbial communities, suggesting intimate symbioses that likely play key roles in both macroalgal and bacterial biology, yet little is known about the spatial organization of microbes associated with macroalgae. Canopy-forming kelp are ecologically significant, fixing teragrams of carbon per year in coastal kelp forest ecosystems. We characterized the micron-scale spatial organization of bacterial communities on blades of the kelp Nereocystis luetkeana using fluorescence in situ hybridization and spectral imaging with a probe set combining phylum-, class-, and genus-level probes to localize and identify &gt; 90% of the microbial community.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01235-w)

10.1186/s40168-022-01235-w

---

## Metagenomics reveals global-scale contrasts in nitrogen cycling and cyanobacterial light-harvesting mechanisms in glacier cryoconite.
 29.03.2022, _T Murakami, N Takeuchi, H Mori, Y Hirose, A Edwards, T Irvine-Fynn, Z Li, S Ishii, T Segawa_


Cryoconite granules are mineral-microbial aggregates found on glacier surfaces worldwide and are hotspots of biogeochemical reactions in glacier ecosystems. However, despite their importance within glacier ecosystems, the geographical diversity of taxonomic assemblages and metabolic potential of cryoconite communities around the globe remain unclear. In particular, the genomic content of cryoconite communities on Asia's high mountain glaciers, which represent a substantial portion of Earth's ice masses, has rarely been reported. Therefore, in this study, to elucidate the taxonomic and ecological diversities of cryoconite bacterial consortia on a global scale, we conducted shotgun metagenomic sequencing of cryoconite acquired from a range of geographical areas comprising Polar (Arctic and Antarctic) and Asian alpine regions.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01238-7)

10.1186/s40168-022-01238-7

---

## Phages and their satellites encode hotspots of antiviral systems.
 22.03.2022, _F Rousset, F Depardieu, S Miele, J Dowding, AL Laval, E Lieberman, D Garry, EPC Rocha, A Bernheim, D Bikard_


Bacteria carry diverse genetic systems to defend against viral infection, some of which are found within prophages where they inhibit competing viruses. Phage satellites pose additional pressures on phages by hijacking key viral elements to their own benefit. Here, we show that E. coli P2-like phages and their parasitic P4-like satellites carry hotspots of genetic variation containing reservoirs of anti-phage systems. We validate the activity of diverse systems and describe PARIS, an abortive infection system triggered by a phage-encoded anti-restriction protein. Antiviral hotspots participate in inter-viral competition and shape dynamics between the bacterial host, P2-like phages, and P4-like satellites. Notably, the anti-phage activity of satellites can benefit the helper phage during competition with virulent phages, turning a parasitic relationship into a mutualistic one. Anti-phage hotspots are present across distant species and constitute a substantial source of systems that participate in the competition between mobile genetic elements.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(22)00104-4)

10.1016/j.chom.2022.02.018

---

## Fungal sensing enhances neutrophil metabolic fitness by regulating antifungal Glut1 activity.
 22.03.2022, _DD Li, CV Jawale, C Zhou, L Lin, GJ Trevejo-Nunez, SA Rahman, SJ Mullet, J Das, SG Wendell, GM Delgoffe, MS Lionakis, SL Gaffen, PS Biswas_


Combating fungal pathogens poses metabolic challenges for neutrophils, key innate cells in anti-Candida albicans immunity, yet how host-pathogen interactions cause remodeling of the neutrophil metabolism is unclear. We show that neutrophils mediate renal immunity to disseminated candidiasis by upregulating glucose uptake via selective expression of glucose transporter 1 (Glut1). Mechanistically, dectin-1-mediated recognition of β-glucan leads to activation of PKCδ, which triggers phosphorylation, localization, and early glucose transport by a pool of pre-formed Glut1 in neutrophils. These events are followed by increased Glut1 gene transcription, leading to more sustained Glut1 accumulation, which is also dependent on the β-glucan/dectin-1/CARD9 axis. Card9-deficient neutrophils show diminished glucose incorporation in candidiasis. Neutrophil-specific Glut1-ablated mice exhibit increased mortality in candidiasis caused by compromised neutrophil phagocytosis, reactive oxygen species (ROS), and neutrophil extracellular trap (NET) formation. In human neutrophils, β-glucan triggers metabolic remodeling and enhances candidacidal function. Our data show that the host-pathogen interface increases glycolytic activity in neutrophils by regulating Glut1 expression, localization, and function.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(22)00103-2)

10.1016/j.chom.2022.02.017

---

## The primate gut mycobiome-bacteriome interface is impacted by environmental and subsistence factors.
 18.03.2022, _AK Sharma, S Davison, B Pafco, JB Clayton, JM Rothman, MR McLennan, M Cibot, T Fuh, R Vodicka, CJ Robinson, K Petrzelkova, A Gomez_


The gut microbiome of primates is known to be influenced by both host genetic background and subsistence strategy. However, these inferences have been made mainly based on adaptations in bacterial composition - the bacteriome and have commonly overlooked the fungal fraction - the mycobiome. To further understand the factors that shape the gut mycobiome of primates and mycobiome-bacteriome interactions, we sequenced 16 S rRNA and ITS2 markers in fecal samples of four different nonhuman primate species and three human groups under different subsistence patterns (n = 149). The results show that gut mycobiome composition in primates is still largely unknown but highly plastic and weakly structured by primate phylogeny, compared with the bacteriome. We find significant gut mycobiome overlap between captive apes and human populations living under industrialized subsistence contexts; this is in contrast with contemporary hunter-gatherers and agriculturalists, who share more mycobiome traits with diverse wild-ranging nonhuman primates. In addition, mycobiome-bacteriome interactions were specific to each population, revealing that individual, lifestyle and intrinsic ecological factors affect structural correspondence, number, and kind of interactions between gut bacteria and fungi in primates. Our findings indicate a dominant effect of ecological niche, environmental factors, and diet over the phylogenetic background of the host, in shaping gut mycobiome composition and mycobiome-bacteriome interactions in primates.

[Source](https://www.nature.com/articles/s41522-022-00274-3)

10.1038/s41522-022-00274-3

---

## Swapping Metagenomics Preprocessing Pipeline Components Offers Speed and Sensitivity Increases.
 16.03.2022, _G Armstrong, C Martino, J Morris, B Khaleghi, J Kang, J DeReus, Q Zhu, D Roush, D McDonald, A Gonazlez, JP Shaffer, C Carpenter, M Estaki, S Wandro, S Eilert, A Akel, J Eno, K Curewitz, AD Swafford, N Moshiri, T Rosing, R Knight_


Increasing data volumes on high-throughput sequencing instruments such as the NovaSeq 6000 leads to long computational bottlenecks for common metagenomics data preprocessing tasks such as adaptor and primer trimming and host removal. Here, we test whether faster recently developed computational tools (Fastp and Minimap2) can replace widely used choices (Atropos and Bowtie2), obtaining dramatic accelerations with additional sensitivity and minimal loss of specificity for these tasks. Furthermore, the taxonomic tables resulting from downstream processing provide biologically comparable results. However, we demonstrate that for taxonomic assignment, Bowtie2's specificity is still required. We suggest that periodic reevaluation of pipeline components, together with improvements to standardized APIs to chain them together, will greatly enhance the efficiency of common bioinformatics tasks while also facilitating incorporation of further optimized steps running on GPUs, FPGAs, or other architectures. We also note that a detailed exploration of available algorithms and pipeline components is an important step that should be taken before optimization of less efficient algorithms on advanced or nonstandard hardware. IMPORTANCE In shotgun metagenomics studies that seek to relate changes in microbial DNA across samples, processing the data on a computer often takes longer than obtaining the data from the sequencing instrument. Recently developed software packages that perform individual steps in the pipeline of data processing in principle offer speed advantages, but in practice they may contain pitfalls that prevent their use, for example, they may make approximations that introduce unacceptable errors in the data. Here, we show that differences in choices of these components can speed up overall data processing by 5-fold or more on the same hardware while maintaining a high degree of correctness, greatly reducing the time taken to interpret results. This is an important step for using the data in clinical settings, where the time taken to obtain the results may be critical for guiding treatment.

[Source](https://journals.asm.org/doi/10.1128/msystems.01378-21)

10.1128/msystems.01378-21

---

