## Oral Microbiota and Immune System Crosstalk: A Translational Research.
 20.06.2020, _A Ballini, G Dipalma, CG Isacco, M Boccellino, M Di Domenico, L Santacroce, KCD Nguyễn, S Scacco, M Calvani, A Boddi, F Corcioli, L Quagliuolo, S Cantore, FS Martelli, F Inchingolo_


Oral pathogens may exert the ability to trigger differently the activation of local macrophage immune responses, for instance Porphyromonas gingivalis and Aggregatibacter actinomycetemcomitans induce predominantly pro-inflammatory (M1-like phenotypes) responses, while oral commensal microbiota primarily elicits macrophage functions consistent with the anti-inflammatory (M2-like phenotypes).

[Source](https://doi.org/10.1002/cncr.33030)

10.3390/biology9060131

---

## The microbiome in pediatric oncology

[Source](https://doi.org/10.1002/cncr.33030)

---

## Metagenomic analysis of the lung microbiome in pulmonary tuberculosis - a pilot study.
 19.06.2020, _Y Hu, M Cheng, B Liu, J Dong, L Sun, J Yang, F Yang, X Chen, Q Jin_


Background.  The lung microbiome plays an important role in the pathophysiological processes associated with pulmonary tuberculosis (PTB). However, only a few studies using 16S rDNA amplicon sequencing have been reported, and the interactions between Mycobacterium tuberculosis (MTB) and the lung microbiome remain poorly understood.  Methods.  Patients with respiratory symptoms and imaging abnormalities compatible with tuberculosis (TB) were enrolled. We analyzed the lung microbiome in bronchoalveolar lavage (BAL) samples from 30 MTB-positive (MTB+) subjects and 30 MTB negative (MTB-) subjects by shotgun metagenomic sequencing.  Results.  Alpha diversity tended to be lower in the MTB+ group than in the MTB- group. There was a significant difference in beta diversity between the MTB+ and MTB- subjects. MTB+ lung samples were dominated by MTB, while MTB- samples were enriched with Streptococcus, Prevotella, Nesseria, Selenomonas and Bifidobacterium, which more closely resemble the microbial composition of a healthy lung. Network analysis suggested that MTB could greatly impact the microbial community structure. MTB+ and MTB- communities showed distinct functional signatures. Fungal communities were also found to be associated with the presence or absence of MTB. Furthermore, it was confirmed that 16S rDNA amplicon sequencing underrepresents Mycobacterium.  Conclusions.  This pilot study is the first to explore the interplay between MTB and the host microbiome by using metagenomic sequencing. MTB dominates the lung microbiome of MTB+ subjects, while MTB- subjects have a Streptococcus-enriched microbiome. The 16S approach underrepresents Mycobacterium and is not the best way to study the TB-associated microbiome.

[Source](https://doi.org/10.1080/22221751.2020.1783188)

10.1080/22221751.2020.1783188

---

## Taxonomic and Functional Differences in Cervical Microbiome Associated with Cervical Cancer Development.
 19.06.2020, _CN Tango, SS Seo, M Kwon, DO Lee, HK Chang, MK Kim_


The cervical microbiome is associated with cervical cancer risk, but how microbial diversity and functional profiles change in cervical cancer remains unclear. Herein, we investigated microbial-compositional and functional differences between a control group and a high-grade cervical intraepithelial neoplasia and cervical cancer (CIN2/3-CC) group. After retrospective collection of 92 cervical swab samples, we carried out 16S rRNA amplicon sequencing on 50 and 42 samples from the control and CIN2/3-CC groups, respectively. The EzBioCloud pipeline was applied to identify the genomic features associated with the groups using 16S rRNA data. A linear discriminant analysis effect size (LEfSe) was performed to assess the enrichment in the assigned taxonomic and functional profiles. We found a lower richness in the control group relative to the CIN2/3-CC group; however, the β-diversity tended to be similar between the groups. The LEfSe analysis showed that a phylum Sacchaaribacteria_TM7, 11 genera, and 21 species were more abundant in the CIN2/3-CC group and that one uncharacterized Gardnerella species was more abundant only in the control group. Further characterization of the functional pathways using EzBioCloud showed that the 4 KEGG orthologs (Phosphotransferase system [PTS] sucrose-specific IIA, IIB, IIC components and PTS cellubiose-specific IIC component) were involved in the KEGG pathway of starch and sucrose metabolism. The two pathways of folate biosynthesis and oxidative phosphorylation were more abundant in the CIN2/3-CC group. Further confirmation of these results in larger samples can help to elucidate the potential association between the cervical microbiome and cervical cancer.

[Source](https://doi.org/10.1038/s41598-020-66607-4)

10.1038/s41598-020-66607-4

---

## Potential Role of Gut Microbiota in Induction and Regulation of Innate Immune Memory.
 04.05.2020, _S Negi, DK Das, S Pahari, S Nadeem, JN Agrewala_


The gut microbiota significantly regulates the development and function of the innate and adaptive immune system. The attribute of immunological memory has long been linked only with adaptive immunity. Recent evidence indicates that memory is also present in the innate immune cells such as monocytes/macrophages and natural killer cells. These cells exhibit pattern recognition receptors (PRRs) that recognize microbe- or pathogen-associated molecular patterns (MAMPs or PAMPs) expressed by the microbes. Interaction between PRRs and MAMPs is quite crucial since it triggers the sequence of signaling events and epigenetic rewiring that not only play a cardinal role in modulating the activation and function of the innate cells but also impart a sense of memory response. We discuss here how gut microbiota can influence the generation of innate memory and functional reprogramming of bone marrow progenitors that helps in protection against infections. This article will broaden our current perspective of association between the gut microbiome and innate memory. In the future, this knowledge may pave avenues for development and designing of novel immunotherapies and vaccination strategies.

[Source](https://doi.org/10.1101/2020.06.16.153809)

10.3389/fimmu.2019.02441

---

## The gut microbiome regulates memory function

[Source](https://doi.org/10.1101/2020.06.16.153809)

---

## Zebrafish microbiome studies make waves.
 16.06.2020, _K Stagaman, TJ Sharpton, K Guillemin_


Zebrafish have a 50-year history as a model organism for studying vertebrate developmental biology and more recently have emerged as a powerful model system for studying vertebrate microbiome assembly, dynamics and function. In this Review, we discuss the strengths of the zebrafish model for both observational and manipulative microbiome studies, and we highlight some of the important insights gleaned from zebrafish gut microbiome research.

[Source](https://doi.org/10.1038/s41684-020-0573-6)

10.1038/s41684-020-0573-6

---

## Revealing antimicrobial resistance in stormwater with MinION

[Source](https://doi.org/10.1016/j.chemosphere.2020.127392)

---

## Towards an understanding of the avian virome.
 10.06.2020, _S François, OG Pybus_


The last two decades have seen the rise of viromics, the study of viral communities through the detection and characterization of virus genome sequences. Here we systematically review and summarize the scope and limitations of our current understanding of avian viromes, in both domesticated and wild-bird populations. We compare this viromic work to the broader literature on avian prokaryotic microbiomes, and highlight the growing importance of structured sampling and experimental design for testing explanatory hypotheses. We provide a number of recommendations for sample collection and preliminary data analysis to guide the development of avian viromics. Avian viromes have the potential to inform disease surveillance in poultry and improve our understanding of the risk of zoonotic viruses to human health.

[Source](https://www.microbiologyresearch.org/content/journal/jgv/10.1099/jgv.0.001447?crawler=true#abstract_content)

10.1099/jgv.0.001447

---

## Initial Mapping of the New York City Wastewater Virome.
 17.06.2020, _K Gulino, J Rahman, M Badri, J Morton, R Bonneau, E Ghedin_


Bacteriophages are abundant members of all microbiomes studied to date, influencing microbial communities through interactions with their bacterial hosts. Despite their functional importance and ubiquity, phages have been underexplored in urban environments compared to their bacterial counterparts. We profiled the viral communities in New York City (NYC) wastewater using metagenomic data collected in November 2014 from 14 wastewater treatment plants. We show that phages accounted for the largest viral component of the sewage samples and that specific virus communities were associated with local environmental conditions within boroughs. The vast majority of the virus sequences had no homology matches in public databases, forming an average of 1,700 unique virus clusters (putative genera). These new clusters contribute to elucidating the overwhelming proportion of data that frequently goes unidentified in viral metagenomic studies. We assigned potential hosts to these phages, which appear to infect a wide range of bacterial genera, often outside their presumed host. We determined that infection networks form a modular-nested pattern, indicating that phages include a range of host specificities, from generalists to specialists, with most interactions organized into distinct groups. We identified genes in viral contigs involved in carbon and sulfur cycling, suggesting functional importance of viruses in circulating pathways and gene functions in the wastewater environment. In addition, we identified virophage genes as well as a nearly complete novel virophage genome. These findings provide an understanding of phage abundance and diversity in NYC wastewater, previously uncharacterized, and further examine geographic patterns of phage-host association in urban environments.IMPORTANCE Wastewater is a rich source of microbial life and contains bacteria, viruses, and other microbes found in human waste as well as environmental runoff sources. As part of an effort to characterize the New York City wastewater metagenome, we profiled the viral community of sewage samples across all five boroughs of NYC and found that local sampling sites have unique sets of viruses. We focused on bacteriophages, or viruses of bacteria, to understand how they may influence the microbial ecology of this system. We identified several new clusters of phages and successfully associated them with bacterial hosts, providing insight into virus-host interactions in urban wastewater. This study provides a first look into the viral communities present across the wastewater system in NYC and points to their functional importance in this environment.

[Source](https://doi.org/10.1128/mSystems.00876-19)

10.1128/mSystems.00876-19

---

## Higher order Markov models for metagenomic sequence classification.
 09.06.2020, _DJ Burks, RK Azad_


Alignment-free, stochastic models derived from k-mer distributions representing reference genome sequences have a rich history in the classification of DNA sequences. In particular, the variants of Markov models have previously been used extensively. Higher order Markov models have been used with caution, perhaps sparingly, primarily because of the lack of enough training data and computational power. Advances in sequencing technology and computation have enabled exploitation of the predictive power of higher order models. We therefore revisited higher order Markov models and assessed their performance in classifying metagenomic sequences.

[Source](https://doi.org/10.1093/bioinformatics/btaa562)

10.1093/bioinformatics/btaa562

---

## Combinatorial Algorithms for Strain Level Metagenomic Microbial Detection and Quantification

[Source](https://doi.org/10.1101/2020.06.12.149245)

---

## Ultra-rapid metagenotyping of the human gut microbiome

[Source](https://doi.org/10.1101/2020.06.12.149336)

---

## Information Theory in Computational Biology: Where We Stand Today

[Source](https://doi.org/10.3390/e22060627)

---

