## Translating Microbiome Research From and To the Clinic

[Source](https://doi.org/10.1146/annurev-micro-041020-022206)

---

## High-throughput, single-microbe genomics with strain resolution, applied to a human gut microbiome.
 02.06.2022, _W Zheng, S Zhao, Y Yin, H Zhang, DM Needham, ED Evans, CL Dai, PJ Lu, EJ Alm, DA Weitz_


Characterizing complex microbial communities with single-cell resolution has been a long-standing goal of microbiology. We present Microbe-seq, a high-throughput method that yields the genomes of individual microbes from complex microbial communities. We encapsulate individual microbes in droplets with microfluidics and liberate their DNA, which we then amplify, tag with droplet-specific barcodes, and sequence. We explore the human gut microbiome, sequencing more than 20,000 microbial single-amplified genomes (SAGs) from a single human donor and coassembling genomes of almost 100 bacterial species, including several with multiple subspecies strains. We use these genomes to probe microbial interactions, reconstructing the horizontal gene transfer (HGT) network and observing HGT between 92 species pairs; we also identify a significant in vivo host-phage association between crAssphage and one strain of Bacteroides vulgatus. Microbe-seq contributes high-throughput culture-free capabilities to investigate genomic blueprints of complex microbial communities with single-microbe resolution.

[Source](https://doi.org/10.1126/science.abm1483)

10.1126/science.abm1483

---

## Gut microbiota and atopic dermatitis in children: a scoping review

[Source](https://doi.org/10.1186/s12887-022-03390-3)

---

## Gut microbiome and autoimmune disorders.
 02.06.2022, _WA Shaheen, MN Quraishi, TH Iqbal_


Autoimmune diseases have long been known to share a common pathogenesis involving a dysregulated immune system with failure to recognize self from non-self antigens. This immune dysregulation is now increasingly understood to be induced by environmental triggers in genetically predisposed individuals. Although several external environmental triggers have been defined in different autoimmune diseases, much attention is being paid to the role of the internal micro-environment occupied by the microbiome which was once termed "the forgotten organ". In this regard, the gut microbiome, serving as an intermediary between some of those external environmental effectors and the immune system helps programming of the immune system to be tolerant to innocent external and self antigens. However, in the presence of perturbed gut microbiota (dysbiosis), the immune system could be erroneously directed in favor of pro-inflammatory pathways to instigate different autoimmune processes. An accumulating body of evidence, including both experimental and human studies (observational and interventional) points to a role of gut microbiome in different autoimmune diseases. Such evidence could provide a rationale for gut microbiome manipulation with therapeutic and even preventative intents in patients with established or predisposed to autoimmune diseases respectively. Perturbations of the gut microbiome have been delineated in some immune mediated diseases, IBD in particular. However, such patterns of disturbance (microbiome signatures) and related pathogenetic roles of the gut microbiome are context dependent and cannot be generalized in the same exact way to other autoimmune disorders and the contribution of gut microbiome to different disease phenotypes has to be precisely defined. In this review, we revise the evidence for a role of gut microbiome in various autoimmune diseases and possible mechanisms mediating such a role.

[Source](https://doi.org/10.1093/cei/uxac057)

10.1093/cei/uxac057

---

## Altered Vaginal Microbiota Composition Correlates With Human Papillomavirus and Mucosal Immune Responses in Women With Symptomatic Cervical Ectopy

[Source](https://doi.org/10.3389/fcimb.2022.884272)

---

## Profile of the Nasopharyngeal Microbiota Affecting the Clinical Course in COVID-19 Patients

[Source](https://doi.org/10.3389/fmicb.2022.871627)

---

## Machine learning aided construction of the quorum sensing communication network for human gut microbiota

[Source](https://doi.org/10.1038/s41467-022-30741-6)

---

## Metagenomic assembled plasmids of the human microbiome vary across disease cohorts

[Source](https://doi.org/10.1038/s41598-022-13313-y)

---

## Gut microbial metabolites in Parkinson’s disease: Association with lifestyle, disease characteristics, and treatment status

[Source](https://doi.org/10.1016/j.nbd.2022.105780)

---

## Mechanistic insights into accelerated α-synuclein aggregation mediated by human microbiome-associated functional amyloids

[Source](https://doi.org/10.1016/j.jbc.2022.102088)

---

## Gut bacterial isoamylamine promotes age-related cognitive dysfunction by promoting microglial cell death

[Source](https://doi.org/10.1016/j.chom.2022.05.005)

---

## A Review: Cereals on Modulating the Microbiota/Metabolome for Metabolic Health

[Source](https://doi.org/10.1007/s13668-022-00424-1)

---

## Anti-Diabetic Potential of <em>Chlorella pyrenoidosa</em>-Based Mixture and its Regulation of Gut Microbiota

[Source](https://doi.org/10.1007/s11130-022-00968-1)

---

## Exploring the Ecological Effects of Naturally Antibiotic-Insensitive Bifidobacteria in the Recovery of the Resilience of the Gut Microbiota during and after Antibiotic Treatment.
 02.06.2022, _C Argentini, L Mancabelli, G Alessandri, C Tarracchini, M Barbetti, L Carnevali, G Longhi, A Viappiani, R Anzalone, C Milani, A Sgoifo, D van Sinderen, M Ventura, F Turroni_


Amoxicillin-clavulanic acid (AMC) is the most widely used antibiotic, being frequently prescribed to infants. Particular members of the genus Bifidobacterium are among the first microbial colonizers of the infant gut, and it has been demonstrated that they exhibit various activities beneficial for their human host, including promotion/maintenance of the human gut microbiota homeostasis. It has been shown that natural resistance of bifidobacteria to AMC is limited to a small number of strains. In the current study, we investigated the mitigation effects of AMC-resistant bifidobacteria in diversity preservation of the gut microbiota during AMC treatment. To this end, an in vitro coculture experiment based on infant fecal samples and an in vivo study employing a rodent model were performed. The results confirmed the ability of AMC-resistant bifidobacterial strains to bolster gut microbiota resilience, while specific covariance analysis revealed strain-specific and variable impacts on the microbiota composition by individual bifidobacterial taxa. IMPORTANCE The first microbial colonizers of the infant gut are members of the genus Bifidobacterium, which exhibit different activities beneficial to their host. Amoxicillin-clavulanic acid (AMC) is the most frequently prescribed antibiotic during infancy, and few strains of bifidobacteria are known to show a natural resistance to this antibiotic. In the present work, we evaluated the possible positive effects of AMC-resistant bifidobacterial strains in maintaining gut microbiota diversity during AMC exposure, performing an in vitro and in vivo experiment based on an infant gut model and a rodent model, respectively. Our results suggested the ability of AMC-resistant bifidobacterial strains to support gut microbiota restoration.

[Source](https://doi.org/10.1128/aem.00522-22)

10.1128/aem.00522-22

---

## Synchrony and idiosyncrasy in the gut microbiome of wild baboons

[Source](https://doi.org/10.1038/s41559-022-01773-4)

---

## A synergistic bacterial pool decomposes tebuthiuron in soil

[Source](https://doi.org/10.1038/s41598-022-13147-8)

---

## The microbiome of cryospheric ecosystems

[Source](https://doi.org/10.1038/s41467-022-30816-4)

---

## Bacterial diversity and community structure in the rhizosphere of the halophyte <em>Halocnemum strobilaceum</em> in an Algerian arid saline soil

[Source](https://doi.org/10.1007/s00792-022-01268-x)

---

## Multiscale analysis of count data through topic alignment

[Source](https://doi.org/10.1093/biostatistics/kxac018)

---

## SAMBA: A Multicolor Digital Melting PCR Platform for Rapid Microbiome Profiling.
 02.06.2022, _Y Luo, X Cui, E Cheruba, YK Chua, C Ng, RZ Tan, KK Tan, LF Cheow_


During the past decade, breakthroughs in sequencing technology have provided the basis for studies of the myriad ways in which microbial communities in and on the human body influence human health and disease. In almost every medical specialty, there is now a growing interest in accurate and quantitative profiling of the microbiota for use in diagnostic and therapeutic applications. However, the current next-generation sequencing approach for microbiome profiling is costly, requires laborious library preparation, and is challenging to scale up for routine diagnostics. Split, Amplify, and Melt analysis of BActeria-community (SAMBA), a novel multicolor digital melting polymerase chain reaction platform with unprecedented multiplexing capability is presented, and the capability to distinguish and quantify 16 bacteria species in mixtures is demonstrated. Subsequently, SAMBA is applied to measure the compositions of bacteria in the gut microbiome to identify microbial dysbiosis related to colorectal cancer. This rapid, low cost, and high-throughput approach will enable the implementation of microbiome diagnostics in clinical laboratories and routine medical practice.

[Source](https://doi.org/10.1002/smtd.202200185)

10.1002/smtd.202200185

---

