## Integrated Bacterial Identification and Antimicrobial Susceptibility Testing for Polymicrobial Infections Using Digital PCR and Digital High-Resolution Melt in a Microfluidic Array Platform.
 28.09.2020, _P Athamanolap, K Hsieh, AT Wang_


_Algorithms, Anti-Infective Agents, Bacteria, Coinfection, Humans, Machine Learning, Microbial Sensitivity Tests, Microfluidics, Polymerase Chain Reaction, RNA, Ribosomal, 16S_
In diagnosing bacterial infection, rapid bacterial identification (ID) and antimicrobial susceptibility testing (AST) are critical to clinicians in order to provide an effective treatment in a timely manner. The gold standard, culture-based approach provides both ID and antimicrobial susceptibility but requires several days of turnaround time. Especially in polymicrobial infections, where there are more than one organisms interacting collectively that can complicate the treatment. Here, we demonstrate a rapid bacterial diagnostic approach that is capable of bacterial ID/AST in heterogeneous samples within less than 4 hours by using digital PCR (dPCR) and digital high-resolution melt via microfluidic devices. By utilizing dPCR, we are able to quantify amount of nucleic acid, which correlates to phenotypic responses of {\bf individual pathogens in a mixed sample and also shorten the required time of antibiotic exposure. In addition, we employ a machine learning algorithm to automatically identify bacterial species based on melt profiles of 16S rRNA gene.

[Source](https://www.biorxiv.org/content/10.1101/2021.11.10.468026v1.abstract?%3Fcollection=)

10.1109/EMBC.2018.8513472

---

## Rapid antibiotic susceptibility testing and species identification for mixed infections

[Source](https://www.biorxiv.org/content/10.1101/2021.11.10.468026v1.abstract?%3Fcollection=)

---

## Bacterial quorum sensing allows graded and bimodal cellular responses to variations in population density

[Source](https://www.biorxiv.org/content/10.1101/850297v2.abstract?%3Fcollection=)

---

## Phage-encoded sigma factors alter bacterial dormancy

[Source](https://www.biorxiv.org/content/10.1101/2021.11.12.468384v1.abstract?%3Fcollection=)

---

## <em>Aspergillus fumigatus</em> drives tissue damage via iterative assaults upon mucosal integrity and immune homeostasis

[Source](https://www.biorxiv.org/content/10.1101/2021.11.09.468003v1.abstract?%3Fcollection=)

---

## Cobamide sharing drives skin microbiome dynamics

[Source](https://www.biorxiv.org/content/10.1101/2020.12.02.407395v2.abstract?%3Fcollection=)

---

## Toxic effects of shale gas fracturing flowback fluid on microbial communities in polluted soil.
 11.11.2021, _X Mei, F Zeng, F Xu, H Su_


_Environmental Monitoring, Hydraulic Fracking, Microbiota, Natural Gas, Soil, Waste Water_
A large amount of shale gas fracturing flowback fluid (FFBF) from the process of shale gas exploitation causes obvious ecological harm to health of soil and water. However, biological hazard of soil microbial populations by fracturing flowback fluid remains rarely reported. In this study, the microbiological compositions were assessed via analyzing diversity of microbial populations. The results showed significant differences between polluted soil by fracturing flowback fluid and unpolluted soil in different pH and temperature conditions. And then, the microbe-index of biological integrity (M-IBI) was used to evaluate the toxicity of the fracturing flowback fluid based on analysis of microbial integrity. The results showed that polluted soil lacks key microbial species known to be beneficial to soil health, including denitrifying bacteria and cellulose-decomposing bacteria, and 35 °C is a critical value for estimating poor and sub-healthy level of damage to microbial integrity by fracturing flowback fluid. Our results provide a valuable reference for the evaluation of soil damage by fracturing flowback fluid.

[Source](https://link.springer.com/article/10.1007%2Fs10661-021-09544-7)

10.1007/s10661-021-09544-7

---

## Colonial choanoflagellate isolated from Mono Lake harbors a microbiome

[Source](https://www.biorxiv.org/content/10.1101/2021.03.30.437421v5.abstract?%3Fcollection=)

---

## Linking extreme seasonality and gene expression in arctic marine protists

[Source](https://www.biorxiv.org/content/10.1101/2021.11.11.467955v1.abstract?%3Fcollection=)

---

## Biosorption potential and molecular characterization of metal resistant autochthonous microbes from tannery solid waste

[Source](https://www.biorxiv.org/content/10.1101/2021.11.12.468455v1.abstract?%3Fcollection=)

---

## Plasma Electrolytic Polishing of Nitinol: Investigation of Functional Properties.
 15.11.2021, _K Navickaitė, L Ianniciello, J Tušek, K Engelbrecht, CRH Bahl, M Penzel, K Nestler, F Böttger-Hiller, H Zeidler_


A novel, environmentally friendly, fast, and flexible polishing process for Nitinol parts is presented in this study. Nitinol samples with both superelastic and shape memory properties at room temperature were investigated. The chemical contamination and surface roughness of superelastic Nitinol plates were examined before and after plasma electrolytic polishing. The shift in phase transformation temperature and tensile strength before and after the polishing process were analysed using Nitinol wire with shape memory properties. The obtained experimental results were compared to the data obtained on reference samples examined in the as-received condition. It was found that plasma electrolytic polishing, when the right process parameters are applied, is capable of delivering Nitinol parts with extremely high surface quality. Moreover, it was experimentally proven that plasma electrolytic polishing does not have a negative impact on functionality or mechanical properties of polished parts.

[Source](https://www.biorxiv.org/content/10.1101/2021.10.27.466057v2.abstract?%3Fcollection=)

10.3390/ma14216450

---

## Oxford Nanopore R10.4 long-read sequencing enables near-perfect bacterial genomes from pure cultures and metagenomes without short-read or reference polishing

[Source](https://www.biorxiv.org/content/10.1101/2021.10.27.466057v2.abstract?%3Fcollection=)

---

## Going through phages: A Computational approach to Revealing the role of prophage in <em>Staphylococcus aureus</em>

[Source](https://www.biorxiv.org/content/10.1101/2021.11.10.468171v1.abstract?%3Fcollection=)

---

