## Highly multiplexed spatial mapping of microbial communities.
 03.12.2020, _H Shi, Q Shi, B Grodner, JS Lenz, WR Zipfel, IL Brito, I De Vlaminck_


Mapping the complex biogeography of microbial communities in situ with high taxonomic and spatial resolution poses a major challenge because of the high density1 and rich diversity2 of species in environmental microbiomes and the limitations of optical imaging technology3-6. Here we introduce high-phylogenetic-resolution microbiome mapping by fluorescence in situ hybridization (HiPR-FISH), a versatile technology that uses binary encoding, spectral imaging and decoding based on machine learning to create micrometre-scale maps of the locations and identities of hundreds of microbial species in complex communities. We show that 10-bit HiPR-FISH can distinguish between 1,023 isolates of Escherichia coli, each fluorescently labelled with a unique binary barcode. HiPR-FISH, in conjunction with custom algorithms for automated probe design and analysis of single-cell images, reveals the disruption of spatial networks in the mouse gut microbiome in response to treatment with antibiotics, and the longitudinal stability of spatial architectures in the human oral plaque microbiome. Combined with super-resolution imaging, HiPR-FISH shows the diverse strategies of ribosome organization that are exhibited by taxa in the human oral microbiome. HiPR-FISH provides a framework for analysing the spatial ecology of environmental microbial communities at single-cell resolution.

[Source](https://www.nature.com/articles/s41586-020-2983-4)

10.1038/s41586-020-2983-4

---

## Comparison of the human microbiome in adults and children with chronic rhinosinusitis.
 02.12.2020, _IH Park, JS Lee, JH Park, SH Kang, SM Hong, IS Park, JH Yoon, SJ Hong_


We hypothesized that differences in the microbiome could be a cause of the substantial differences in the symptoms of and treatment options for adult and pediatric patients with chronic rhinosinusitis (CRS). First, we characterized the differences in the nasal microbiomes of pediatric and adult CRS patients. Swabs were obtained from 19 patients with chronic rhinosinusitis (9 children and 10 adults). The bacterial 16S rRNA gene was pyrosequenced to compare the microbiota of the middle meatus. No significant differences were found in species richness and alpha-diversity indices between the two groups. However, in the comparison of diversity between groups using the unweighted pair group method with arithmetic mean (UPGMA) clustering of microbiome taxonomic profiles, we observed a relatively clear separation between the adult and pediatric groups. Actinobacteria had a significantly higher relative abundance in the adult group than in the pediatric group at the phylum level. At the genus level, Corynebacterium showed significantly higher relative abundance in the adult group than in the pediatric group. This is a comparative study between the microbiomes of adult and pediatric CRS patients. We expect this study to be the first step in understanding the pathogenesis of CRS in adults and children using microbiome analysis.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0242770)

10.1371/journal.pone.0242770

---

## Gut mycobiomes are altered in people with type 2 Diabetes Mellitus and Diabetic Retinopathy.
 01.12.2020, _R Jayasudha, T Das, S Kalyana Chakravarthy, G Sai Prashanthi, A Bhargava, M Tyagi, PK Rani, RR Pappuru, S Shivaji_


Studies have documented dysbiosis in the gut mycobiome in people with Type 2 diabetes mellitus (T2DM). However, it is not known whether dysbiosis in the gut mycobiome of T2DM patients would be reflected in people with diabetic retinopathy (DR) and if so, is the observed mycobiome dysbiosis similar in people with T2DM and DR. Gut mycobiomes were generated from healthy controls (HC), people with T2DM and people with DR through Illumina sequencing of ITS2 region. Data were analysed using QIIME and R software. Dysbiotic changes were observed in people with T2DM and DR compared to HC at the phyla and genera level. Mycobiomes of HC, T2DM and DR could be discriminated by heat map analysis, Beta diversity analysis and LEfSE analysis. Spearman correlation of fungal genera indicated more negative correlation in HC compared to T2DM and DR mycobiomes. This study demonstrates dysbiosis in the gut mycobiomes in people with T2DM and DR compared to HC. These differences were significant both at the phyla and genera level between people with T2DM and DR as well. Such studies on mycobiomes may provide new insights and directions to identification of specific fungi associated with T2DM and DR and help developing novel therapies for Diabetes Mellitus and DR.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0243077)

10.1371/journal.pone.0243077

---

## Mucosa-Associated Microbiota in Patients with Irritable Bowel Syndrome: A Comparison of Subtypes.
 03.12.2020, _H Matsumoto, A Shiotani, R Katsumata, S Fukushima, Y Handa, M Osawa, T Murao, O Handa, E Umegaki, R Inoue, Y Naito_


Most studies on gut microbiome of irritable bowel syndrome (IBS) have focused on fecal microbiota, instead of mucosa-associated microbiota (MAM).

[Source](https://www.karger.com/Article/Abstract/512167)

10.1159/000512167

---

## The Role of the Gut Microbiota and the Immune System in the Development of Autism.
 03.12.2020, _RM Lungba, SZA Khan, U Ajibawo-Aganbi, MV Perez Bastidas, S Veliginti, S Saleem, I Cancarevic_


Autism‌ ‌spectrum‌ ‌disorders‌ ‌(ASDs)‌ ‌are‌ ‌neurodevelopmental‌ ‌disorders‌ ‌that‌ ‌present‌ ‌with‌ ‌social‌ skills‌ ‌and‌ ‌communication‌ ‌challenges,‌ ‌restricted‌ ‌interest,‌ ‌and‌ ‌repetitive‌ ‌behavior.‌ ‌The‌ ‌specific‌ cause ‌of‌ ‌autism‌ ‌is‌ ‌not‌ ‌well‌ ‌understood‌ ‌yet.‌ ‌However,‌ ‌numerous‌ ‌studies‌ ‌indicated‌ ‌that‌ ‌environmental‌ ‌and‌ ‌genetic‌ ‌factors,‌ ‌dysregulated‌ ‌immune‌ ‌response,‌ ‌and‌ ‌alterations‌ ‌to‌ ‌the‌ ‌balance‌ ‌and‌ ‌content‌ ‌of‌ ‌the‌ ‌gut‌ ‌microbiota‌ ‌are‌ ‌implemented‌ ‌in‌ ‌the‌ ‌development‌ ‌of‌ ‌autism.‌ ‌Many‌ ‌non-pharmacological‌ ‌interventions‌ ‌are‌ ‌nominated‌ ‌to‌ ‌manage ‌autism,‌ ‌including‌ ‌family‌ ‌support‌ ‌services‌ ‌and‌ ‌psychoeducational‌ ‌methods‌. Moreover,‌ ‌different‌ ‌pharmacological‌ ‌therapy‌ ‌modalities‌ ‌are‌ ‌recommended‌ ‌for‌ ‌children‌ ‌with‌ ‌ASD.‌ ‌Learning‌ ‌more‌ ‌about‌ ‌the‌ ‌brain,‌ ‌immune‌ ‌system, ‌and‌ ‌gut‌ ‌connections‌ ‌could‌ ‌assist‌ ‌in early‌ ‌diagnosis‌ ‌and‌ ‌treatment‌ ‌of‌ ‌this‌ ‌devastating‌ ‌neurodevelopmental‌ ‌disorders‌ ‌as‌ ‌an‌ ‌early‌ ‌intervention‌ ‌in‌ ‌ASD‌ ‌could‌ ‌improve‌ ‌a‌ ‌child's‌ ‌overall‌ ‌development.‌ We‌ ‌gathered‌ ‌data‌ ‌from‌ ‌relevant‌ ‌previously‌ ‌published‌ ‌articles‌ ‌on‌ ‌PubMed‌ ‌to‌ ‌evaluate ‌the‌ ‌role‌ ‌of‌ ‌the‌ ‌gut‌ ‌microbiota‌ ‌and‌ ‌the‌ ‌immune‌ ‌system‌ ‌on‌ ‌the‌ ‌development‌ ‌of‌ ‌autism.‌.

[Source](https://www.cureus.com/articles/43567-the-role-of-the-gut-microbiota-and-the-immune-system-in-the-development-of-autism)

10.7759/cureus.11226

---

## A shotgun metagenomic investigation of the microbiota of udder cleft dermatitis in comparison to healthy skin in dairy cows.
 02.12.2020, _L Ekman, E Bagge, A Nyman, K Persson Waller, M Pringle, B Segerman_


Udder cleft dermatitis (UCD) is a skin condition affecting the fore udder attachment of dairy cows. UCD may be defined as mild (eczematous skin changes) or severe (open wounds, large skin changes). Our aims were to compare the microbiota of mild and severe UCD lesions with the microbiota of healthy skin from the fore udder attachment of control cows, and to investigate whether mastitis-causing pathogens are present in UCD lesions. Samples were obtained from cows in six dairy herds. In total, 36 UCD samples categorized as mild (n = 17) or severe (n = 19) and 13 control samples were sequenced using a shotgun metagenomic approach and the reads were taxonomically classified based on their k-mer content. The Wilcoxon rank sum test was used to compare the abundance of different taxa between different sample types, as well as to compare the bacterial diversity between samples. A high proportion of bacteria was seen in all samples. Control samples had a higher proportion of archaeal reads, whereas most samples had low proportions of fungi, protozoa and viruses. The bacterial microbiota differed between controls and mild and severe UCD samples in both composition and diversity. Subgroups of UCD samples were visible, characterized by increased proportion of one or a few bacterial genera or species, e.g. Corynebacterium, Staphylococcus, Brevibacterium luteolum, Trueperella pyogenes and Fusobacterium necrophorum. Bifidobacterium spp. were more common in controls compared to UCD samples. The bacterial diversity was higher in controls compared to UCD samples. Bacteria commonly associated with mastitis were uncommon. In conclusion, a dysbiosis of the microbiota of mild and severe UCD samples was seen, characterized by decreased diversity and an increased proportion of certain bacteria. There was no evidence of a specific pathogen causing UCD or that UCD lesions are important reservoirs for mastitis-causing bacteria.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0242880)

10.1371/journal.pone.0242880

---

## Gut microbiota is associated with protection against Marek's disease virus infection in chickens.
 03.12.2020, _J Bavananthasivam, J Astill, A Matsuyama-Kato, K Taha-Abdelaziz, B Shojadoost, S Sharif_


Marek's Disease Virus (MDV) infects chickens via respiratory route and causes lymphomas in internal organs including gastrointestinal tract. MDV infection causes a shift in the gut microbiota composition. However, interactions between the gut microbiota and immune responses against MDV infection are not well understood. Therefore, the current study was performed to understand the effect of the gut microbiota on Marek's disease (MD) pathogenesis. The findings showed that depletion of gut microbiota increased the severity of MD in infected chickens. In addition, an increase in the transcription of interferon (IFN)-α, IFN-β and IFN-γ in the bursa of Fabricius at 4 days post-infection (dpi) was observed in the gut microbiota depleted chickens. The observations in this study shed more light on the association between the gut microbiota and MDV infection in chickens. More research is needed to explore the mechanisms of involvement of the gut microbiota in immunity against MD in chickens.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0042682220302191)

10.1016/j.virol.2020.10.011

---

## Stingless bees and microbial interactions.
 03.12.2020, _GT de Paula, C Menezes, MT Pupo, CA Rosa_


Stingless bees (Meliponini) are a monophyletic group of eusocial insects inhabiting tropical and subtropical regions. These insects represent the most abundant and diversified group of corbiculate bees. Meliponini mostly rely on fermentation by symbiont microbes to preserve honey and transform pollen in stored food. Bee nests harbor diverse microbiota that includes bacteria, yeasts, filamentous fungi, and viruses. These microorganisms may interact with the bees through symbiotic relationships, or they may act as food for the insects, or produce biomolecules that aid in the biotransformation of bee products, such as honey and bee bread. Certain microbial species can also produce antimicrobial compounds that inhibit opportunistic bee pathogens.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S2214574520301504)

10.1016/j.cois.2020.11.006

---

## CoMA - an intuitive and user-friendly pipeline for amplicon-sequencing data analysis.
 02.12.2020, _S Hupfauf, M Etemadi, M Fernández-Delgado Juárez, M Gómez-Brandón, H Insam, SM Podmirseg_


In recent years, there has been a veritable boost in next-generation sequencing (NGS) of gene amplicons in biological and medical studies. Huge amounts of data are produced and need to be analyzed adequately. Various online and offline analysis tools are available; however, most of them require extensive expertise in computer science or bioinformatics, and often a Linux-based operating system. Here, we introduce "CoMA-Comparative Microbiome Analysis" as a free and intuitive analysis pipeline for amplicon-sequencing data, compatible with any common operating system. Moreover, the tool offers various useful services including data pre-processing, quality checking, clustering to operational taxonomic units (OTUs), taxonomic assignment, data post-processing, data visualization, and statistical appraisal. The workflow results in highly esthetic and publication-ready graphics, as well as output files in standardized formats (e.g. tab-delimited OTU-table, BIOM, NEWICK tree) that can be used for more sophisticated analyses. The CoMA output was validated by a benchmark test, using three mock communities with different sample characteristics (primer set, amplicon length, diversity). The performance was compared with that of Mothur, QIIME and QIIME2-DADA2, popular packages for NGS data analysis. Furthermore, the functionality of CoMA is demonstrated on a practical example, investigating microbial communities from three different soils (grassland, forest, swamp). All tools performed well in the benchmark test and were able to reveal the majority of all genera in the mock communities. Also for the soil samples, the results of CoMA were congruent to those of the other pipelines, in particular when looking at the key microbial players.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0243241)

10.1371/journal.pone.0243241

---

