## An ethical way forward for Indigenous microbiome research.
 02.09.2024, _S Jones_



[Source](https://www.nature.com/immersive/d41586-024-02792-w/index.html)

10.1038/d41586-024-02792-w

---

## Interplay between particle size and microbial ecology in the gut microbiome.
 03.09.2024, _J Letourneau, VM Carrion, J Zeng, S Jiang, OW Osborne, ZC Holmes, A Fox, P Epstein, CY Tan, M Kirtley, NK Surana, LA David_


Physical particles can serve as critical abiotic factors that structure the ecology of microbial communities. For non-human vertebrate gut microbiomes, fecal particle size (FPS) has been known to be shaped by chewing efficiency and diet. However, little is known about what drives FPS in the human gut. Here, we analyzed FPS by laser diffraction across a total of 76 individuals and found FPS to be strongly individualized. Contrary to our initial hypothesis, a behavioral intervention with 41 volunteers designed to increase chewing efficiency did not impact FPS. Dietary patterns could also not be associated with FPS. Instead, we found evidence that human and mouse gut microbiomes shaped FPS. Fecal samples from germ-free and antibiotic-treated mice exhibited increased FPS relative to colonized mice. In humans, markers of longer transit time were correlated with smaller FPS. Gut microbiota diversity and composition were also associated with FPS. Finally, ex vivo culture experiments using human fecal microbiota from distinct donors showed that differences in microbiota community composition can drive variation in particle size. Together, our results support an ecological model in which the human gut microbiome plays a key role in reducing the size of food particles during digestion. This finding has important implications for our understanding of energy extraction and subsequent uptake in gastrointestinal tract. FPS may therefore be viewed as an informative functional readout, providing new insights into the metabolic state of the gut microbiome.

[Source](https://academic.oup.com/ismej/advance-article/doi/10.1093/ismejo/wrae168/7746109)

10.1093/ismejo/wrae168

---

## Unexplored microbial diversity from 2,500 food metagenomes and links with the human microbiome 

[Source](https://www.cell.com/cell/fulltext/S0092-8674(24)00833-X)

---

## A streamlined culturomics case study for the human gut microbiota research.
 05.09.2024, _H Park, S Yeo, CB Ryu, CS Huh_


_Humans, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Feces, Bacteria, DNA, Bacterial_
Bacterial culturomics is a set of techniques to isolate and identify live bacteria from complex microbial ecosystems. Despite its potential to revolutionize microbiome research, bacterial culturomics has significant challenges when applied to human gut microbiome studies due to its labor-intensive nature. Therefore, we established a streamlined culturomics approach with minimal culture conditions for stool sample preincubation. We evaluated the suitability of non-selective medium candidates for maintaining microbial diversity during a 30-day incubation period based on 16S rRNA gene amplicon analysis. Subsequently, we applied four culture conditions (two preincubation media under an aerobic/anaerobic atmosphere) to isolate gut bacteria on a large scale from eight stool samples of healthy humans. We identified 8141 isolates, classified into 263 bacterial species, including 12 novel species candidates. Our analysis of cultivation efficiency revealed that seven days of aerobic and ten days of anaerobic incubation captured approximately 91% and 95% of the identified species within each condition, respectively, with a synergistic effect confirmed when selected preincubation media were combined. Moreover, our culturomics findings expanded the coverage of gut microbial diversity compared to 16S rRNA gene amplicon sequencing results. In conclusion, this study demonstrated the potential of a streamlined culturomics approach for the efficient isolation of gut bacteria from human stool samples. This approach might pave the way for the broader adoption of culturomics in human gut microbiome studies, ultimately leading to a more comprehensive understanding of this complex microbial ecosystem.

[Source](https://www.nature.com/articles/s41598-024-71370-x)

10.1038/s41598-024-71370-x

---

## Gut microbiota patterns associated with duration of diarrhea in children under five years of age in Ethiopia.
 05.09.2024, _G Tesfaw, DS Siraj, A Abdissa, RR Jakobsen, ØH Johansen, M Zangenberg, K Hanevik, Z Mekonnen, N Langeland, O Bjørang, N Safdar, AC Mapes, A Kates, L Krych, JL Castro-Mejía, DS Nielsen_


_Humans, Gastrointestinal Microbiome, Ethiopia, Diarrhea, Child, Preschool, Infant, Case-Control Studies, Male, Female, Feces, Bacteria_
Diarrhea claims &gt;500,000 lives annually among children under five years of age in low- and middle-income countries. Mortality due to acute diarrhea (&lt;7 days' duration) is decreasing, but prolonged (7-13 days) and persistent (≥14 days of duration) diarrhea remains a massive challenge. Here, we use a case-control study to decipher if fecal gut microbiota compositional differences between Ethiopian children with acute (n=554) or prolonged/persistent (n=95) diarrhea and frequency-matched non-diarrheal controls (n=663) are linked to diarrheal etiology. We show that diarrhea cases are associated with lower bacterial diversity and enriched in Escherichia spp., Campylobacter spp., and Streptococcus spp. Further, diarrhea cases are depleted in gut commensals such as Prevotella copri, Faecalibacterium prausnitzii, and Dialister succinatiphilus, with depletion being most pronounced in prolonged/persistent cases, suggesting that prolonged duration of diarrhea is accompanied by depletion of gut commensals and that re-establishing these via e.g., microbiota-directed food supplements offer a potential treatment strategy.

[Source](https://www.nature.com/articles/s41467-024-51464-w)

10.1038/s41467-024-51464-w

---

## Association of maternal gut microbial metabolites with gestational diabetes mellitus: evidence from an original case-control study, meta-analysis, and Mendelian randomization.
 02.09.2024, _M Yao, Y Xiao, Y Sun, B Zhang, Y Ding, Q Ma, F Liang, Z Yang, W Ge, S Liu, L Xin, J Yin, X Zhu_


The associations of gut microbial metabolites, such as trimethylamine N-oxide (TMAO), its precursors, and phenylacetylglutamine (PAGln), with the risk of gestational diabetes mellitus (GDM) remain unclear.

[Source](https://www.nature.com/articles/s41430-024-01502-z)

10.1038/s41430-024-01502-z

---

## The Microleaks study: 16S community profile and metagenomic shotgun sequencing signatures associated with anastomotic leak

[Source](https://www.nature.com/articles/s44355-024-00006-8)

---

## Targeted isolation of Methanobrevibacter strains from fecal samples expands the cultivated human archaeome.
 03.09.2024, _S Duller, S Vrbancic, Ł Szydłowski, A Mahnert, M Blohs, M Predl, C Kumpitsch, V Zrim, C Högenauer, T Kosciolek, RA Schmitz, A Eberhard, M Dragovan, L Schmidberger, T Zurabischvili, V Weinberger, AM Moser, D Kolb, D Pernitsch, R Mohammadzadeh, T Kühnast, T Rattei, C Moissl-Eichinger_


_Methanobrevibacter, Humans, Feces, Gastrointestinal Microbiome, Genome, Archaeal, Methane, Phylogeny, Adult, Male, Female, Gastrointestinal Tract_
Archaea are vital components of the human microbiome, yet their study within the gastrointestinal tract (GIT) is limited by the scarcity of cultured representatives. Our study presents a method for the targeted enrichment and isolation of methanogenic archaea from human fecal samples. The procedure combines methane breath testing, in silico metabolic modeling, media optimization, FACS, dilution series, and genomic sequencing through Nanopore technology. Additional analyzes include the co-cultured bacteriome, comparative genomics of archaeal genomes, functional comparisons, and structure-based protein function prediction of unknown differential traits. Successful establishment of stable archaeal cultures from 14 out of 16 fecal samples yielded nine previously uncultivated strains, eight of which are absent from a recent archaeome genome catalog. Comparative genomic and functional assessments of Methanobrevibacter smithii and Candidatus Methanobrevibacter intestini strains from individual donors revealed features potentially associated with gastrointestinal diseases. Our work broadens available archaeal representatives for GIT studies, and offers insights into Candidatus Methanobrevibacter intestini genomes' adaptability in critical microbiome contexts.

[Source](https://www.nature.com/articles/s41467-024-52037-7)

10.1038/s41467-024-52037-7

---

## Characterization of gut microbiota in the Uyghur osteopenia population.
 03.09.2024, _K Teng, X Li, T Huang, S Zhang, Q Zhang, N Rousitemu, T Lan, Y Wen_


_Humans, Gastrointestinal Microbiome, Bone Diseases, Metabolic, Middle Aged, Female, Male, China, RNA, Ribosomal, 16S, Feces, Bone Density, Adult, Aged, Bacteria_
The objectives of this study were to investigate the composition of gut microbiota and its relationship with bone loss in the Uyghur osteopenia population, identify potential disease-related taxa and collect information for the prevention and treatment of osteopenia in different people by regulating gut microbiota. We selected Uyghur residents, measured their heel BMD, collected faeces and general information, grouped them by BMD level, obtained faecal 16S rRNA sequences, and compared and analysed the differences between the groups. This study showed that the numbers of OTUs and species in the gut microbiota in the osteopenia group were higher than those in the control. At the phylum level, Erysipelotrichia was more abundant in the osteopenia group. At the genus level, Phascolarctobacterium was less abundant, and Ruminiclostridium_5 was more abundant in the osteopenia group compared to the control. Phascolarctobacterium and Z-score were positively correlated, and Ruminiclostridium_5 was negatively correlated with T and Z score. The different composition of the gut microbiota in Uyghur osteopenia patients and controls found in this study fills a knowledge gap in this ethnic group. The relationship between Uyghur osteopenia and BMD-associated bacterial genera deserves further exploration.

[Source](https://www.nature.com/articles/s41598-024-71077-z)

10.1038/s41598-024-71077-z

---

## Spaceflight alters host-gut microbiota interactions.
 01.09.2024, _E Gonzalez, MD Lee, BT Tierney, N Lipieta, P Flores, M Mishra, L Beckett, A Finkelstein, A Mo, P Walton, F Karouia, R Barker, RJ Jansen, SJ Green, S Weging, J Kelliher, NK Singh, D Bezdan, J Galazska, NJB Brereton_


_Gastrointestinal Microbiome, Animals, Space Flight, Mice, Bacteria, Liver, Host Microbial Interactions, Metagenomics, Colon, Bile Acids and Salts, Energy Metabolism, Male, Humans, Mice, Inbred C57BL_
The ISS rodent habitat has provided crucial insights into the impact of spaceflight on mammals, inducing symptoms characteristic of liver disease, insulin resistance, osteopenia, and myopathy. Although these physiological responses can involve the microbiome on Earth, host-microbiota interactions during spaceflight are still being elucidated. We explore murine gut microbiota and host gene expression in the colon and liver after 29 and 56 days of spaceflight using multiomics. Metagenomics revealed significant changes in 44 microbiome species, including relative reductions in bile acid and butyrate metabolising bacteria like Extibacter muris and Dysosmobacter welbionis. Functional prediction indicate over-representation of fatty acid and bile acid metabolism, extracellular matrix interactions, and antibiotic resistance genes. Host gene expression described corresponding changes to bile acid and energy metabolism, and immune suppression. These changes imply that interactions at the host-gut microbiome interface contribute to spaceflight pathology and that these interactions might critically influence human health and long-duration spaceflight feasibility.

[Source](https://www.nature.com/articles/s41522-024-00545-1)

10.1038/s41522-024-00545-1

---

## Bacterial association with metals enables in vivo monitoring of urogenital microbiota using magnetic resonance imaging.
 06.09.2024, _SC Donnelly, GE Varela-Mattatall, S Hassan, Q Sun, N Gelman, JD Thiessen, RT Thompson, FS Prato, JP Burton, DE Goldhawk_


_Humans, Magnetic Resonance Imaging, Microbiota, Female, Bacteria, Metals, Urogenital System, Manganese_
Bacteria constitute a significant part of the biomass of the human microbiota, but their interactions are complex and difficult to replicate outside the host. Exploiting the superior resolution of magnetic resonance imaging (MRI) to examine signal parameters of selected human isolates may allow tracking of their dispersion throughout the body. Here we investigate longitudinal and transverse MRI relaxation rates and found significant differences between several bacterial strains. Common commensal strains of lactobacilli display notably high MRI relaxation rates, partially explained by elevated cellular manganese content, while other species contain more iron than manganese. Lactobacillus crispatus show particularly high values, 4-fold greater than any other species; up to 60-fold greater signal than relevant tissue background; and a linear relationship between relaxation rate and fraction of live cells. Different bacterial strains have detectable, repeatable MRI relaxation rates that in the future may enable monitoring of their persistence in the human body for enhanced molecular imaging.

[Source](https://www.nature.com/articles/s42003-024-06783-5)

10.1038/s42003-024-06783-5

---

## Urine metabolomic profiles of autism and autistic traits-A twin study.
 06.09.2024, _A Arora, F Mastropasqua, S Bölte, K Tammimies_


_Humans, Autistic Disorder, Male, Female, Metabolomics, Child, Biomarkers, Adolescent, Metabolome, Adult, Twins, Monozygotic, Child, Preschool_
Currently, there are no reliable biomarkers for autism diagnosis. The heterogeneity of autism and several co-occurring conditions are key challenges to establishing these. Here, we used untargeted mass spectrometry-based urine metabolomics to investigate metabolic differences for autism diagnosis and autistic traits in a well-characterized twin cohort (N = 105). We identified 208 metabolites in the urine samples of the twins. No clear, significant metabolic drivers for autism diagnosis were detected when controlling for other neurodevelopmental conditions. However, we identified nominally significant changes for several metabolites. For instance, phenylpyruvate (p = 0.019) and taurine (p = 0.032) were elevated in the autism group, while carnitine (p = 0.047) was reduced. We furthermore accounted for the shared factors, such as genetics within the twin pairs, and report additional metabolite differences. Based on the nominally significant metabolites for autism diagnosis, the arginine and proline metabolism pathway (p = 0.024) was enriched. We also investigated the association between quantitative autistic traits, as measured by the Social Responsiveness Scale 2nd Edition, and metabolite differences, identifying a greater number of nominally significant metabolites and pathways. A significant positive association between indole-3-acetate and autistic traits was observed within the twin pairs (adjusted p = 0.031). The utility of urine biomarkers in autism, therefore, remains unclear, with mixed findings from different study populations.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0308224)

10.1371/journal.pone.0308224

---

## Massive expansion of the pig gut virome based on global metagenomic mining.
 01.09.2024, _J Mi, X Jing, C Ma, Y Yang, Y Li, Y Zhang, R Long, H Zheng_


_Animals, Swine, Virome, Metagenomics, Gastrointestinal Microbiome, Viruses, Genome, Viral, Data Mining, Metagenome, Phylogeny_
The pig gut virome plays a vital role in the gut microbial ecosystem of pigs. However, a comprehensive understanding of their diversity and a reference database for the virome are currently lacking. To address this gap, we established a Pig Virome Database (PVD) that comprised of 5,566,804 viral contig sequences from 4650 publicly available gut metagenomic samples using a pipeline designated "metav". By clustering sequences, we identified 48,299 viral operational taxonomic units (vOTUs) genomes of at least medium quality, of which 92.83% of which were not found in existing major databases. The majority of vOTUs were identified as Caudoviricetes (72.21%). The PVD database contained a total of 2,362,631 protein-coding genes across the above medium-quality vOTUs genomes that can be used to explore the functional potential of the pig gut virome. These findings highlight the extensive diversity of viruses in the pig gut and provide a pivotal reference dataset for forthcoming research concerning the pig gut virome.

[Source](https://www.nature.com/articles/s41522-024-00554-0)

10.1038/s41522-024-00554-0

---

## Marine sponge microbe provides insights into evolution and virulence of the tubercle bacillus.
 03.09.2024, _SJ Pidot, S Klatt, LS Ates, W Frigui, F Sayes, L Majlessi, H Izumi, IR Monk, JL Porter, V Bennett-Wood, T Seemann, A Otter, G Taiaroa, GM Cook, N West, NJ Tobias, JA Fuerst, MD Stutz, M Pellegrini, M McConville, R Brosch, TP Stinear_


_Animals, Mice, Virulence, Porifera, Mycobacterium tuberculosis, Tuberculosis, Virulence Factors, Female, Biological Evolution, Humans, Phylogeny, Mycobacterium_
Reconstructing the evolutionary origins of Mycobacterium tuberculosis, the causative agent of human tuberculosis, has helped identify bacterial factors that have led to the tubercle bacillus becoming such a formidable human pathogen. Here we report the discovery and detailed characterization of an exceedingly slow growing mycobacterium that is closely related to M. tuberculosis for which we have proposed the species name Mycobacterium spongiae sp. nov., (strain ID: FSD4b-SM). The bacterium was isolated from a marine sponge, taken from the waters of the Great Barrier Reef in Queensland, Australia. Comparative genomics revealed that, after the opportunistic human pathogen Mycobacterium decipiens, M. spongiae is the most closely related species to the M. tuberculosis complex reported to date, with 80% shared average nucleotide identity and extensive conservation of key M. tuberculosis virulence factors, including intact ESX secretion systems and associated effectors. Proteomic and lipidomic analyses showed that these conserved systems are functional in FSD4b-SM, but that it also produces cell wall lipids not previously reported in mycobacteria. We investigated the virulence potential of FSD4b-SM in mice and found that, while the bacteria persist in lungs for 56 days after intranasal infection, no overt pathology was detected. The similarities with M. tuberculosis, together with its lack of virulence, motivated us to investigate the potential of FSD4b-SM as a vaccine strain and as a genetic donor of the ESX-1 genetic locus to improve BCG immunogenicity. However, neither of these approaches resulted in superior protection against M. tuberculosis challenge compared to BCG vaccination alone. The discovery of M. spongiae adds to our understanding of the emergence of the M. tuberculosis complex and it will be another useful resource to refine our understanding of the factors that shaped the evolution and pathogenesis of M. tuberculosis.

[Source](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1012440)

10.1371/journal.ppat.1012440

---

## Exploring the interactions between Nosema ceranae infection and the honey bee gut microbiome.
 03.09.2024, _E Lau, J Maccaro, QS McFrederick, JC Nieh_


_Bees, Animals, Nosema, Gastrointestinal Microbiome, Microsporidiosis, Spores, Fungal, Host-Pathogen Interactions_
Managed colonies of the European honey bee, Apis mellifera, have faced considerable losses in recent years. A widespread contributing factor is a microsporidian pathogen, Nosema ceranae, which occurs worldwide, is increasingly resistant to antibiotic treatment, and can alter the host's immune response and nutritional uptake. These obligate gut pathogens share their environment with a natural honey bee microbiome whose composition can affect pathogen resistance. We tested the effect of N. ceranae infection on this microbiome by feeding 5 day-old adult bees that had natural, fully developed microbiomes with live N. ceranae spores (40,000 per bee) or a sham inoculation, sterile 2.0 M sucrose solution. We caged and reared these bees in a controlled lab environment and tracked their mortality over 12 d, after which we dissected them, measured their infection levels (gut spore counts), and analyzed their microbiomes. Bees fed live spores had two-fold higher mortality by 12 d and 36.5-fold more spores per bee than controls. There were also strong colony effects on infection levels, and 9% of spore-inoculated bees had no spore counts at all (defined as fed-spores-but-not-infected). Nosema ceranae infection had significant but subtle effects on the gut microbiomes of experimentally infected bees, bees with different infection levels, and fed-spores-but-not-infected vs. bees with gut spores. Specific bacteria, including Gilliamella ASVs, were positively associated with infection, indicating that multiple strains of core gut microbes either facilitate or resist N. ceranae infection. Future studies on the interactions between bacterial, pathogen, and host genotypes would be illuminating.

[Source](https://www.nature.com/articles/s41598-024-67796-y)

10.1038/s41598-024-67796-y

---

## Metagenomic characterisation of canine skin reveals a core healthy skin microbiome.
 03.09.2024, _MJ Whittle, J Castillo-Fernandez, GCA Amos, P Watson_


_Dogs, Animals, Skin, Microbiota, Metagenomics, Metagenome, RNA, Ribosomal, 16S, Bacteria, High-Throughput Nucleotide Sequencing_
Furthering our knowledge of the skin microbiome is essential to understand health and disease in canines. To date, studies into the canine skin microbiome have focused on 16S rRNA high throughput sequencing however, these lack the granularity of species and strain level taxonomic characterisation and their associated functions. The aim of this study was to provide a comprehensive assessment of the skin microbiome by analysing the skin microbiome of 72 healthy adult colony dogs, across four distinct skin sites and four breeds, using metagenomic sequencing. Our analysis revealed that breed and skin site are drivers of variation, and a core group of taxa and genes are present within the skin microbiome of healthy dogs, comprising 230 taxa and 1219 gene families. We identified 15 species within the core microbiome that are represented by more than one strain. The biosynthesis of secondary metabolites pathway was enriched in the core microbiome suggesting the skin microbiome may play a role in colonisation resistance and protection from invading pathogens. Additionally, we uncovered the novelty of the canine skin microbiome and show that further investigation is required to increase the suitability of current databases for metagenomic sequencing of canine skin samples.

[Source](https://www.nature.com/articles/s41598-024-63999-5)

10.1038/s41598-024-63999-5

---

## CHIKV infection drives shifts in the gastrointestinal microbiome and metabolites in rhesus monkeys.
 06.09.2024, _H Chen, J Shi, C Tang, J Xu, B Li, J Wang, Y Zhou, Y Yang, H Yang, Q Huang, W Yu, H Wang, D Wu, Y Hu, H Zhou, Q Sun, S Lu_


_Animals, Gastrointestinal Microbiome, Macaca mulatta, Feces, Chikungunya Fever, Chikungunya virus, Bacteria, Dysbiosis, Inflammation, Inflammasomes, Disease Models, Animal, Interleukin-17, Gastrointestinal Tract, Cytokines_
Many studies have demonstrated the association between intestinal microbiota and joint diseases. The "gut-joint axis" also has potential roles in chikungunya virus (CHIKV) infection. Pro-inflammatory arthritis after CHIKV infection might disrupt host homeostasis and lead to dysbacteriosis. This study investigated the characteristics of fecal and gut microbiota, intestinal metabolites, and the changes in gene regulation of intestinal tissues after CHIKV infection using multi-omics analysis to explore the involvement of gut microbiota in the pathogenesis of CHIKV infection.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01895-w)

10.1186/s40168-024-01895-w

---

## Effects of dietary restriction and one-carbon metabolite supplementation during the first 63 days of gestation on the maternal gut, vaginal, and blood microbiota in cattle.
 02.09.2024, _SM Luecke, G Aryee, DB Holman, KN Schmidt, LE King, MS Crouse, AK Ward, CR Dahlen, JS Caton, S Amat_


Maternal diet quality and quantity have significant impacts on both maternal and fetal health and development. The composition and function of the maternal gut microbiome is also significantly influenced by diet; however, little is known about the impact of gestational nutrient restriction on the bovine maternal microbiome during early gestation, which is a critical stage for maternal microbiome-mediated fetal programming to take place. The objective of the present study was to evaluate the impacts of diet restriction and one-carbon metabolite (OCM) supplementation during early gestation on maternal ruminal, vaginal, and blood microbiota in cattle. Thirty-three beef heifers (approx. 14 months old) were used in a 2 × 2 factorial experiment with main factors of target gain (control [CON]; targeted 0.45 kg/d gain vs restricted [RES]; targeted - 0.23 kg/d gain), and OCM supplementation (+ OCM vs - OCM; n = 8/treatment; except n = 9 for RES-OCM). Heifers were individually fed, starting treatment at breeding (d 0) and concluding at d 63 of gestation. Ruminal fluid and vaginal swabs were collected on d - 2, d 35, and d 63 (at necropsy) and whole blood was collected on d 63 (necropsy). Bacterial microbiota was assessed using 16S rRNA gene (V3-V4) sequencing.

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-024-00335-2)

10.1186/s42523-024-00335-2

---

## Aloe arborescens supplementation in drying-off dairy cows: influence on rumen, rectum and milk microbiomes.
 03.09.2024, _P Cremonesi, F Biscarini, G Conte, F Piccioli-Cappelli, S Morandi, T Silvetti, S Tringali, E Trevisi, B Castiglioni, M Brasca_


In the context of the RABOLA project, which aimed to identify operational practices that lead to the reduction of antibiotic use in dairy cattle farming, lyophilised Aloe arborescens was administered orally to cows during the dry-off and peripartum periods. In this specific paper we wanted to examine whether oral administration of Aloe arborescens, in combination with the topical application of a teat sealant could exert an effect on the microbial populations of three cow microbiomes (rumen, milk, rectum), between dry-off and peripartum. Dry-off and peripartum are critical physiological phases of the cow's life, where both the mammary gland and the gastrointestinal tract undergo dramatic modifications, hence the relevance of evaluating the effects of dietary treatments.

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-024-00336-1)

10.1186/s42523-024-00336-1

---

## Comparative genomics identifies key adaptive traits of sponge-associated microbial symbionts.
 04.09.2024, _PA O'Brien, SJ Robbins, S Tan, L Rix, DJ Miller, NS Webster, G Zhang, DG Bourne_


_Symbiosis, Porifera, Animals, Genomics, Bacteria, Microbiota, Phylogeny, Genome, Bacterial_
Sponge microbiomes are often highly diverse making it difficult to determine which lineages are important for maintaining host health and homeostasis. Characterising genomic traits associated with symbiosis can improve our knowledge of which lineages have adapted to their host and what functions they might provide. Here we examined five microbial families associated with sponges that have previously shown evidence of cophylogeny, including Endozoicomonadaceae, Nitrosopumilaceae, Spirochaetaceae, Microtrichaceae and Thermoanaerobaculaceae, to better understand the mechanisms behind their symbiosis. We compared sponge-associated genomes to genomes found in other environments and found that sponge-specific clades were enriched in genes encoding many known mechanisms for symbiont survival, such as avoiding phagocytosis and defence against foreign genetic elements. We expand on previous knowledge to show that glycosyl hydrolases with sulfatases and sulfotransferases likely form multienzyme degradation pathways to break and remodel sulfated polysaccharides and reveal an enrichment in superoxide dismutase that may prevent damage from free oxygen radicals produced by the host. Finally, we identified novel traits in sponge-associated symbionts, such as urea metabolism in Spirochaetaceae which was previously shown to be rare in the phylum Spirochaetota. These results identify putative mechanisms by which symbionts have adapted to living in association with sponges.

[Source](https://enviromicro-journals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.16690)

10.1111/1462-2920.16690

---

## Deciphering the role of rhizosphere microbiota in modulating disease resistance in cabbage varieties.
 03.09.2024, _X Ping, RAA Khan, S Chen, Y Jiao, X Zhuang, L Jiang, L Song, Y Yang, J Zhao, Y Li, Z Mao, B Xie, J Ling_


_Rhizosphere, Brassica, Disease Resistance, Plant Diseases, Soil Microbiology, Fusarium, Microbiota, Bacteria, Plant Roots, Fungi_
Cabbage Fusarium wilt (CFW) is a devastating disease caused by the soil-borne fungus Fusarium oxysporum f. sp. conglutinans (Foc). One of the optimal measures for managing CFW is the employment of tolerant/resistant cabbage varieties. However, the interplay between plant genotypes and the pathogen Foc in shaping the rhizosphere microbial community, and the consequent influence of these microbial assemblages on biological resistance, remains inadequately understood.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01883-0)

10.1186/s40168-024-01883-0

---

## Oak seedling microbiome assembly under climate warming and drought.
 02.09.2024, _D Hoefle, M Sommer, B Wassermann, M Faticov, D Serra, G Berg, AJM Tack, A Abdelfattah_


Despite that climate change is currently one of the most pervasive challenges, its effects on the plant-associated microbiome is still poorly studied. The aim of this study was to evaluate the impact of the independent and combinatory effect of climate warming and drought on the microbiome assembly of oak from seed to seedling. In a multifactorial experimental set up, acorns were subjected to different temperatures (15 °C, 20 °C, and 25 °C) and soil moisture levels (drought (15%) and control (60%)) from germination until the seedling stage, after which the bacterial and fungal communities associated to the rhizosphere and phyllosphere were characterized by amplicon sequencing and qPCR. The results showed a stronger effect of temperature on fungal than on bacterial diversity and the effect was more pronounced in the phyllosphere. Under drought condition, temperature had a significantly negative effect on phyllosphere fungal diversity. In the rhizosphere, temperature had a significant effect on the fungal community composition which was primarily caused by species turnover. Regardless of temperature, Actinobacteriota was significantly enriched in drought, a group of bacteria known to increase plant drought tolerance. This study provides new insights into the effect of climate change on the plant microbiome in natural ecosystems.

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-024-00602-4)

10.1186/s40793-024-00602-4

---

## The ancestral environment of teosinte populations shapes their root microbiome.
 01.09.2024, _CJ Barnes, MS Bünner, MR Ramírez-Flores, IB Nielsen, J Ramos-Madrigal, D Zharikova, CM McLaughlin, MT Gilbert, RJH Sawers_


The composition of the root microbiome affects the host's growth, with variation in the host genome associated with microbiome variation. However, it is not known whether this intra-specific variation of root microbiomes is a consequence of plants performing targeted manipulations of them to adapt to their local environment or varying passively with other traits. To explore the relationship between the genome, environment and microbiome, we sampled seeds from teosinte populations across its native range in Mexico. We then grew teosinte accessions alongside two modern maize lines in a common garden experiment. Metabarcoding was performed using universal bacterial and fungal primers to profile their root microbiomes.

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-024-00606-0)

10.1186/s40793-024-00606-0

---

## Prokaryotic-virus-encoded auxiliary metabolic genes throughout the global oceans.
 03.09.2024, _F Tian, JM Wainaina, C Howard-Varona, G Domínguez-Huerta, B Bolduc, MC Gazitúa, G Smith, MR Gittrich, O Zablocki, DR Cronin, D Eveillard, SJ Hallam, MB Sullivan_


_Oceans and Seas, Seawater, Metagenomics, Bacteria, Viruses, Prokaryotic Cells, Metagenome, Metabolic Networks and Pathways, Gene Transfer, Horizontal, Phosphatidylethanolamines_
Prokaryotic microbes have impacted marine biogeochemical cycles for billions of years. Viruses also impact these cycles, through lysis, horizontal gene transfer, and encoding and expressing genes that contribute to metabolic reprogramming of prokaryotic cells. While this impact is difficult to quantify in nature, we hypothesized that it can be examined by surveying virus-encoded auxiliary metabolic genes (AMGs) and assessing their ecological context.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01876-z)

10.1186/s40168-024-01876-z

---

## Global distribution, diversity, and ecological niche of Picozoa, a widespread and enigmatic marine protist lineage.
 05.09.2024, _P Huber, D De Angelis, H Sarmento, S Metz, CR Giner, C Vargas, L Maiorano, R Massana, R Logares_


_Phylogeny, Eukaryota, Biodiversity, Aquatic Organisms, Ecosystem, Seawater_
The backbone of the eukaryotic tree of life contains taxa only found in molecular surveys, of which we still have a limited understanding. Such is the case of Picozoa, an enigmatic lineage of heterotrophic picoeukaryotes within the supergroup Archaeplastida, which has emerged as a significant component of marine microbial planktonic communities. To enhance our understanding of the diversity, distribution, and ecology of Picozoa, we conduct a comprehensive assessment at different levels, from assemblages to taxa, employing phylogenetic analysis, species distribution modeling, and ecological niche characterization.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01874-1)

10.1186/s40168-024-01874-1

---

## Microbial core communities in activated sludge plants are strongly affected by immigration and geography.
 02.09.2024, _SZ Vestergaard, G Dottorini, M Peces, A Murguz, MKD Dueholm, M Nierychlo, PH Nielsen_


The microbiota in wastewater treatment plants (WWTPs) and incoming wastewater is critical for the treatment process, the preservation of natural ecosystems and human health, and for the recovery of resources and achievement of sustainability goals. Both core species and conditionally rare and abundant taxa (CRAT) are considered process-critical but little is known about identity as well as true functional and ecological importance. Here, we present a comprehensive investigation of the microbiota of 84 municipal activated sludge (AS) plants with nutrient removal treating ~ 70% of all wastewater within a confined geographical area, Denmark (43,000 km2). With the use of an ecosystem-specific database (MiDAS 5.2), species-level classification allowed us to investigate the core and CRAT species, whether they were active, and important factors determining their presence.

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-024-00604-2)

10.1186/s40793-024-00604-2

---

## The eukaryome of modern microbialites reveals distinct colonization across aquatic ecosystems.
 06.09.2024, _AM Bonacolta, PT Visscher, J Del Campo, RA White Iii_


_Ecosystem, Salinity, British Columbia, Eukaryota, Biodiversity, Fresh Water, Biofilms, Seawater, Australia, Diatoms, Geologic Sediments_
Protists are less studied for their role and diversity in ecosystems. Notably, protists have played and still play an important role in microbialites. Microbialites, or lithified microbial mats, represent the oldest evidence of fossil biofilms (~3.5 Gyr). Modern microbialites may offer a unique proxy to study the potential role of protists within a geological context. We examined protist diversity in freshwater (Kelly and Pavilion Lake in British Columbia, Canada) and marine (Highborne Cay, Bahamas) to hypersaline (Shark Bay, Australia) microbialites to decipher their geomicrobiological role. The freshwater microbialite communities were clearly distinct from their marine and hypersaline counterparts. Chlorophytes had higher numerical abundance in freshwater microbialites; whereas pennate diatoms dominated numerically in marine microbialites. Despite the differences, protists across ecosystems may have adopted similar roles and functions. We suggest a consistent biogeochemical role of protists across microbialites globally; but that salinity may shape protist composition and evolution in these ecosystems.

[Source](https://www.nature.com/articles/s41522-024-00547-z)

10.1038/s41522-024-00547-z

---

## Domestic freezer storage: a solution for preserving stool microbiota integrity for at-home collection

[Source](https://www.biorxiv.org/content/10.1101/2024.09.02.610874v1.full.pdf+html)

---

## COVID-19 prevention behaviors and dietary habits among undergraduate students: A health belief model approach

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0309623)

---

## A catalog of small proteins from the global microbiome.
 02.09.2024, _Y Duan, CD Santos-Júnior, TS Schmidt, A Fullam, BLS de Almeida, C Zhu, M Kuhn, XM Zhao, P Bork, LP Coelho_


_Microbiota, Open Reading Frames, Bacteria, Metagenome, Archaea, Molecular Sequence Annotation, Bacterial Proteins_
Small open reading frames (smORFs) shorter than 100 codons are widespread and perform essential roles in microorganisms, where they encode proteins active in several cell functions, including signal pathways, stress response, and antibacterial activities. However, the ecology, distribution and role of small proteins in the global microbiome remain unknown. Here, we construct a global microbial smORFs catalog (GMSC) derived from 63,410 publicly available metagenomes across 75 distinct habitats and 87,920 high-quality isolate genomes. GMSC contains 965 million non-redundant smORFs with comprehensive annotations. We find that archaea harbor more smORFs proportionally than bacteria. We moreover provide a tool called GMSC-mapper to identify and annotate small proteins from microbial (meta)genomes. Overall, this publicly-available resource demonstrates the immense and underexplored diversity of small proteins.

[Source](https://www.nature.com/articles/s41467-024-51894-6#Sec10)

10.1038/s41467-024-51894-6

---

