## Staphylococcus epidermidis and its dual lifestyle in skin health and infection.
 31.08.2022, _MM Severn, AR Horswill_


The coagulase-negative bacterium Staphylococcus epidermidis is a member of the human skin microbiota. S. epidermidis is not merely a passive resident on skin but actively primes the cutaneous immune response, maintains skin homeostasis and prevents opportunistic pathogens from causing disease via colonization resistance. However, it is now appreciated that S. epidermidis and its interactions with the host exist on a spectrum of potential pathogenicity derived from its high strain-level heterogeneity. S. epidermidis is the most common cause of implant-associated infections and is a canonical opportunistic biofilm former. Additional emerging evidence suggests that some strains of S. epidermidis may contribute to the pathogenesis of common skin diseases. Here, we highlight new developments in our understanding of S. epidermidis strain diversity, skin colonization dynamics and its multifaceted interactions with the host and other members of the skin microbiota.

[Source](https://www.nature.com/articles/s41579-022-00780-3)

10.1038/s41579-022-00780-3

---

## Effect of chewing gum containing Xylitol and blackberry powder on oral bacteria: A randomized controlled crossover trial.
 29.08.2022, _CS Miller, RJ Danaher, S Kirakodu, CR Carlson, RJ Mumper_


The aim was to determine the effect of chewing gum containing xylitol and freeze-dried blackberry powder on oral bacteria.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0003996922001807)

10.1016/j.archoralbio.2022.105523

---

## Antimicrobial peptide production in response to gut microbiota imbalance.
 29.08.2022, _MH Cardoso, BT Meneguetti, NG Oliveira-Júnior, MLR Macedo, OL Franco_


The gut microbiota presents essential functions in the immune response. The gut epithelium acts as a protective barrier and, therefore, can produce several antimicrobial peptides (AMPs) that can act against pathogenic microorganisms, including bacteria. Several factors cause a disturbance in gut microbiota, including the exacerbated and erroneous use of antibiotics. Antibiotic therapy has been closely related to bacterial resistance and is also correlated with undesired side-effects to the host, including the eradication of commensal bacteria. Consequently, this results in gut microbiota imbalance and inflammatory bowel diseases (IBD) development. In this context, AMPs in the gut epithelium play a restructuring role for gut microbiota. Some naturally occurring AMPs are selective for pathogenic bacteria, thus preserving the health microbiota. Therefore, AMPs produced by the host's epithelial cells represent effective molecules in treating gut bacterial infections. Bearing this in mind, this review focused on describing the importance of the host's AMPs in gut microbiota modulation and their role as anti-infective agents against pathogenic bacteria.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0196978122001310)

10.1016/j.peptides.2022.170865

---

## ROLE OF GUT MICROBIOTA IN DEPRESSION: UNDERSTANDING MOLECULAR PATHWAYS, RECENT RESEARCH, AND FUTURE DIRECTION

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0166432822003497)

---

## Gut-derived metabolites influence neurodevelopmental gene expression and Wnt signaling events in a germ-free zebrafish model

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01302-2)

---

## Age as a primary driver of the gut microbial composition and function in wild harbor seals

[Source](https://www.nature.com/articles/s41598-022-18565-2)

---

## Biogeographical survey of soil microbiomes across sub-Saharan Africa: structure, drivers, and predicted climate-driven changes.
 30.08.2022, _DA Cowan, PH Lebre, C Amon, RW Becker, HI Boga, A Boulangé, TL Chiyaka, T Coetzee, PC de Jager, O Dikinya, F Eckardt, M Greve, MA Harris, DW Hopkins, HB Houngnandan, P Houngnandan, K Jordaan, E Kaimoyo, AK Kambura, G Kamgan-Nkuekam, TP Makhalanyane, G Maggs-Kölling, E Marais, H Mondlane, E Nghalipo, BW Olivier, M Ortiz, LR Pertierra, JB Ramond, M Seely, I Sithole-Niang, A Valverde, G Varliero, S Vikram, DH Wall, A Zeze_


_Biodiversity, Desert Climate, Ecosystem, Microbiota, Soil, Soil Microbiology_
Top-soil microbiomes make a vital contribution to the Earth's ecology and harbor an extraordinarily high biodiversity. They are also key players in many ecosystem services, particularly in arid regions of the globe such as the African continent. While several recent studies have documented patterns in global soil microbial ecology, these are largely biased towards widely studied regions and rely on models to interpolate the microbial diversity of other regions where there is low data coverage. This is the case for sub-Saharan Africa, where the number of regional microbial studies is very low in comparison to other continents.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01297-w)

10.1186/s40168-022-01297-w

---

## Soil fungal communities show more specificity than bacteria for plant species composition in a temperate forest in China

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-022-02591-1)

---

## Machine learning algorithm to characterize antimicrobial resistance associated with the International Space Station surface microbiome.
 30.08.2022, _P Madrigal, NK Singh, JM Wood, E Gaudioso, F Hernández-Del-Olmo, CE Mason, K Venkateswaran, A Beheshti_


_Algorithms, Anti-Bacterial Agents, Drug Resistance, Bacterial, Humans, Machine Learning, Metagenomics, Microbiota, Spacecraft_
Antimicrobial resistance (AMR) has a detrimental impact on human health on Earth and it is equally concerning in other environments such as space habitat due to microgravity, radiation and confinement, especially for long-distance space travel. The International Space Station (ISS) is ideal for investigating microbial diversity and virulence associated with spaceflight. The shotgun metagenomics data of the ISS generated during the Microbial Tracking-1 (MT-1) project and resulting metagenome-assembled genomes (MAGs) across three flights in eight different locations during 12 months were used in this study. The objective of this study was to identify the AMR genes associated with whole genomes of 226 cultivable strains, 21 shotgun metagenome sequences, and 24 MAGs retrieved from the ISS environmental samples that were treated with propidium monoazide (PMA; viable microbes).

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01332-w)

10.1186/s40168-022-01332-w

---

## Extensive diversity and rapid turnover of phage defense repertoires in cheese-associated bacterial communities.
 30.08.2022, _V Somerville, T Schowing, H Chabas, RS Schmidt, U von Ah, R Bruggmann, P Engel_


_Bacteria, Bacteriophages, Cheese, Genome, Bacterial, Metagenome_
Phages are key drivers of genomic diversity in bacterial populations as they impose strong selective pressure on the evolution of bacterial defense mechanisms across closely related strains. The pan-immunity model suggests that such diversity is maintained because the effective immune system of a bacterial species is the one distributed across all strains present in the community. However, only few studies have analyzed the distribution of bacterial defense systems at the community-level, mostly focusing on CRISPR and comparing samples from complex environments. Here, we studied 2778 bacterial genomes and 188 metagenomes from cheese-associated communities, which are dominated by a few bacterial taxa and occur in relatively stable environments.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01328-6)

10.1186/s40168-022-01328-6

---

## Crowdsourced benchmarking of taxonomic metagenome profilers: lessons learned from the sbv IMPROVER Microbiomics challenge.
 30.08.2022, _C Poussin, L Khachatryan, N Sierro, VK Narsapuram, F Meyer, V Kaikala, V Chawla, U Muppirala, S Kumar, V Belcastro, JND Battey, E Scotti, S Boué, AC McHardy, MC Peitsch, NV Ivanov, J Hoeng_


Selection of optimal computational strategies for analyzing metagenomics data is a decisive step in determining the microbial composition of a sample, and this procedure is complex because of the numerous tools currently available. The aim of this research was to summarize the results of crowdsourced sbv IMPROVER Microbiomics Challenge designed to evaluate the performance of off-the-shelf metagenomics software as well as to investigate the robustness of these results by the extended post-challenge analysis. In total 21 off-the-shelf taxonomic metagenome profiling pipelines were benchmarked for their capacity to identify the microbiome composition at various taxon levels across 104 shotgun metagenomics datasets of bacterial genomes (representative of various microbiome samples) from public databases. Performance was determined by comparing predicted taxonomy profiles with the gold standard.

[Source](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-022-08803-2)

10.1186/s12864-022-08803-2

---

## Newly Renamed Prokaryote Phyla Cause Uproar

[Source](https://www.the-scientist.com/news-opinion/newly-renamed-prokaryote-phyla-cause-uproar-69578)

---

