## The Microbestiary

[Source](http://microbestiary.org/)

---

## Genomic determinants of pathogenicity in SARS-CoV-2 and other human coronaviruses

[Source](https://www.biorxiv.org/content/10.1101/2020.04.05.026450v2)

---

## Nelfinavir inhibits replication of severe acute respiratory syndrome coronavirus 2 in vitro

[Source](https://www.biorxiv.org/content/10.1101/2020.04.06.026476v1.abstract?%3Fcollection=)

---

## Understanding the B and T cells epitopes of spike protein of severe respiratory syndrome coronavirus-2: A computational way to predict the immunogens

[Source](https://www.biorxiv.org/content/10.1101/2020.04.08.013516v1)

---

## From iron to antibiotics: Identification of conserved bacterial-fungal interactions across diverse partners.

[Source](https://www.biorxiv.org/content/10.1101/2020.03.19.999193v2)

---

## Unraveling Host-Gut Microbiota Dialogue and Its Impact on Cholesterol Levels

[Source](https://www.frontiersin.org/articles/10.3389/fphar.2020.00278/full)

---

## Metagenomics analysis reveals features unique to Indian distal gut microbiota.
 08.04.2020, _K Kaur, I Khatri, A Akhtar, S Subramanian, TNC Ramya_


Various factors including diet, age, geography, culture and socio-economic status have a role in determining the composition of the human gut microbiota. The human gut microbial composition is known to be altered in disease conditions. Considering the important role of the gut microbiome in maintaining homeostasis and overall health, it is important to understand the microbial diversity and the functional metagenome of the healthy gut. Here, we characterized the microbiota of 31 fecal samples from healthy individuals of Indian ethnic tribes from Ladakh, Jaisalmer and Khargone by shotgun metagenomic sequencing. Sequence analysis revealed that Bifidobacterium and Prevotella were the key microbes contributing to the differences among Jaisalmer, Khargone and Ladakh samples at the genus level. Our correlation network study identified carbohydrate-active enzymes and carbohydrate binding proteins that are associated with specific genera in the different Indian geographical regions studied. Network analysis of carbohydrate-active enzymes and genus abundance revealed that the presence of different carbohydrate-active enzymes is driven by differential abundance of genera. The correlation networks were different in the different geographical regions, and these interactions suggest the role of less abundant genera in shaping the gut environment. We compared our data with samples from different countries and found significant differences in taxonomic composition and abundance of carbohydrate-active enzymes in the gut microbiota as compared to the other countries.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231197)

10.1371/journal.pone.0231197

---

## Gut Microbiota Modulate CD8 T Cell Responses to Influence Colitis-Associated Tumorigenesis.
 08.04.2020, _AI Yu, L Zhao, KA Eaton, S Ho, J Chen, S Poe, J Becker, A Gonzalez, D McKinstry, M Hasso, J Mendoza-Castrejon, J Whitfield, C Koumpouras, PD Schloss, EC Martens, GY Chen_


There is increasing evidence that gut microbiome perturbations, also known as dysbiosis, can influence colorectal cancer development. To understand the mechanisms by which the gut microbiome modulates cancer susceptibility, we examine two wild-type mouse colonies with distinct gut microbial communities that develop significantly different tumor numbers using a mouse model of inflammation-associated tumorigenesis. We demonstrate that adaptive immune cells contribute to the different tumor susceptibilities associated with the two microbial communities. Mice that develop more tumors have increased colon lamina propria CD8+ IFNγ+ T cells before tumorigenesis but reduced CD8+ IFNγ+ T cells in tumors and adjacent tissues compared with mice that develop fewer tumors. Notably, intratumoral T cells in mice that develop more tumors exhibit increased exhaustion. Thus, these studies suggest that microbial dysbiosis can contribute to colon tumor susceptibility by hyperstimulating CD8 T cells to promote chronic inflammation and early T cell exhaustion, which can reduce anti-tumor immunity.

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(20)30349-1)

10.1016/j.celrep.2020.03.035

---

## Characterization of the oral microbiome of medically controlled type-2 diabetes patients

[Source](https://www.biorxiv.org/content/10.1101/2020.04.07.031070v1)

---

## Cervicovaginal microbiome and natural history of HPV in a longitudinal study

[Source](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1008376)

---

## Meta-Analysis of Vaginal Microbiome Data Provides New Insights Into Preterm Birth

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.00476/full)

---

## Microbiota characterization of Exaiptasia diaphana from the Great Barrier Reef

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-020-00029-5)

---

## Microbial Diversity and Metabolic Potential in the Stratified Sansha Yongle Blue Hole in the South China Sea.
 06.04.2020, _P He, L Xie, X Zhang, J Li, X Lin, X Pu, C Yuan, Z Tian, J Li_


The Sansha Yongle Blue Hole is the world's deepest (301 m) underwater cave and has a sharp redox gradient, with oligotrophic, anoxic, and sulfidic bottom seawater. In order to discover the microbial communities and their special biogeochemical pathways in the blue hole, we analyzed the 16S ribosomal RNA amplicons and metagenomes of microbials from seawater depths with prominent physical, chemical, and biological features. Redundancy analysis showed that dissolved oxygen was the most important factor affecting the microbial assemblages of the blue hole and surrounding open sea waters, and significantly explained 44.7% of the total variation, followed by silicate, temperature, sulfide, ammonium, methane, nitrous oxide, nitrate, dissolved organic carbon, salinity, particulate organic carbon, and chlorophyll a. We identified a bloom of Alteromonas (34.9%) at the primary nitrite maximum occurring in close proximity to the chlorophyll a peak in the blue hole. Genomic potential for nitrate reduction of Alteromonas might contribute to this maximum under oxygen decrease. Genes that would allow for aerobic ammonium oxidation, complete denitrification, and sulfur-oxidization were enriched at nitrate/nitrite-sulfide transition zone (90 and 100 m) of the blue hole, but not anammox pathways. Moreover, γ-Proteobacterial clade SUP05, ε-Proteobacterial genera Sulfurimonas and Arcobacter, and Chlorobi harbored genes for sulfur-driven denitrification process that mediated nitrogen loss and sulfide removal. In the anoxic bottom seawater (100-300 m), high levels of sulfate reducers and dissimilatory sulfite reductase gene (dsrA) potentially created a sulfidic zone of ~200 m thickness. Our findings suggest that in the oligotrophic Sansha Yongle Blue Hole, O2 deficiency promotes nitrogen- and sulfur-cycling processes mediated by metabolically versatile microbials.

[Source](https://www.nature.com/articles/s41598-020-62411-2)

10.1038/s41598-020-62411-2

---

## Metabolic versatility of freshwater sedimentary archaea feeding on different organic carbon sources.
 08.04.2020, _S Compte-Port, M Fillol, F Gich, CM Borrego_


Members of the phylum Bathyarchaeota and the class Thermoplasmata are widespread in marine and freshwater sediments where they have been recognized as key players in the carbon cycle. Here, we tested the responsiveness of archaeal communities on settled plant debris and sediment from a karstic lake to different organic carbon amendments (amino acids, plant-derived carbohydrates, and aromatics) using a lab-scale microcosm. Changes in the composition and abundance of sediment and biofilm archaeal communities in both DNA and RNA fractions were assessed by 16S rRNA gene amplicon sequencing and qPCR, respectively, after 7 and 30 days of incubation. Archaeal communities showed compositional changes in terms of alpha and beta diversity in relation to the type of carbon source (amino acids vs. plant-derived compounds), the nucleic acid fraction (DNA vs. RNA), and the incubation time (7 vs. 30 days). Distinct groups within the Bathyarchaeota (Bathy-15 and Bathy-6) and the Thermoplasmata (MBG-D) differently reacted to carbon supplements as deduced from the analysis of RNA libraries. Whereas Bathyarchaeota in biofilms showed a long-term positive response to humic acids, their counterparts in the sediment were mainly stimulated by the addition of tryptophan, suggesting the presence of different subpopulations in both habitats. Overall, our work presents an in vitro assessment of the versatility of archaea inhabiting freshwater sediments towards organic carbon and introduces settled leaf litter as a new habitat for the Bathyarchaeota and the Thermoplasmata.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231238)

10.1371/journal.pone.0231238

---

## The skin microbiome facilitates adaptive tetrodotoxin production in poisonous newts.
 09.04.2020, _PM Vaelli, KR Theis, JE Williams, LA O'Connell, JA Foster, HL Eisthen_


Rough-skinned newts (Taricha granulosa) use tetrodotoxin (TTX) to block voltage-gated sodium (Nav) channels as a chemical defense against predation. Interestingly, newts exhibit extreme population-level variation in toxicity attributed to a coevolutionary arms race with TTX-resistant predatory snakes, but the source of TTX in newts is unknown. Here, we investigated whether symbiotic bacteria isolated from toxic newts could produce TTX. We characterized the skin-associated microbiota from a toxic and non-toxic population of newts and established pure cultures of isolated bacterial symbionts from toxic newts. We then screened bacterial culture media for TTX using LC-MS/MS and identified TTX-producing bacterial strains from four genera, including Aeromonas, Pseudomonas, Shewanella, and Sphingopyxis. Additionally, we sequenced the Nav channel gene family in toxic newts and found that newts expressed Nav channels with modified TTX binding sites, conferring extreme physiological resistance to TTX. This study highlights the complex interactions among adaptive physiology, animal-bacterial symbiosis, and ecological context.

[Source](https://elifesciences.org/articles/53898)

10.7554/eLife.53898

---

## Metatranscriptomic virome assessment of Rhipicephalus microplus from Colombia

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1877959X19301761)

---

## Metabolic differences between symbiont subpopulations in the deep-sea tubeworm Riftia pachyptila

[Source](https://www.biorxiv.org/content/10.1101/2020.04.08.032177v1.abstract?%3Fcollection=)

---

## Similar gut bacterial microbiota in two fruit-feeding moth pests collected from different host species and locations

[Source](https://www.biorxiv.org/content/10.1101/2020.04.06.028886v2.abstract?%3Fcollection=)

---

## The preceding root system drives the composition and function of the rhizosphere microbiome.
 07.04.2020, _Y Zhou, DR Coventry, VVSR Gupta, D Fuentes, A Merchant, BN Kaiser, J Li, Y Wei, H Liu, Y Wang, S Gan, MD Denton_


The soil environment is responsible for sustaining most terrestrial plant life, yet we know surprisingly little about the important functions carried out by diverse microbial communities in soil. Soil microbes that inhabit the channels of decaying root systems, the detritusphere, are likely to be essential for plant growth and health, as these channels are the preferred locations of new root growth. Understanding the microbial metagenome of the detritusphere, and how it responds to agricultural management such as crop rotations and soil tillage, is vital for improving global food production.

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-01999-0)

10.1186/s13059-020-01999-0

---

## Microbial Co-Occurrence in Floral Nectar Affects Metabolites and Attractiveness to a Generalist Pollinator.
 04.04.2020, _CC Rering, RL Vannette, RN Schaeffer, JJ Beck_


Microbial metabolism can shape cues important for animal attraction in service-resource mutualisms. Resources are frequently colonized by microbial communities, but experimental assessment of animal-microbial interactions often focus on microbial monocultures. Such an approach likely fails to predict effects of microbial assemblages, as microbe-microbe interactions may affect in a non-additive manner microbial metabolism and resulting chemosensory cues. Here, we compared effects of microbial mono- and cocultures on growth of constituent microbes, volatile metabolite production, sugar catabolism, and effects on pollinator foraging across two nectar environments that differed in sugar concentration. Growth in co-culture decreased the abundance of the yeast Metschnikowia reukaufii, but not the bacterium Asaia astilbes. Volatile emissions differed significantly between microbial treatments and with nectar concentration, while sugar concentration was relatively similar among mono- and cocultures. Coculture volatile emission closely resembled an additive combination of monoculture volatiles. Despite differences in microbial growth and chemosensory cues, honey bee feeding did not differ between microbial monocultures and assemblages. Taken together, our results suggest that in some cases, chemical and ecological effects of microbial assemblages are largely predictable from those of component species, but caution that more work is necessary to predict under what circumstances non-additive effects are important.

[Source](https://link.springer.com/article/10.1007/s10886-020-01169-3)

10.1007/s10886-020-01169-3

---

## Impact of phages on soil bacterial communities and nitrogen availability under different assembly scenarios.
 07.04.2020, _LPP Braga, A Spor, W Kot, MC Breuil, LH Hansen, JC Setubal, L Philippot_


Bacteriophages, the viruses infecting bacteria, are biological entities that can control their host populations. The ecological relevance of phages for microbial systems has been widely explored in aquatic environments, but the current understanding of the role of phages in terrestrial ecosystems remains limited. Here, our objective was to quantify the extent to which phages drive the assembly and functioning of soil bacterial communities. We performed a reciprocal transplant experiment using natural and sterilized soil incubated with different combinations of two soil microbial communities, challenged against native and non-native phage suspensions as well as against a cocktail of phage isolates. We tested three different community assembly scenarios by adding phages: (a) during soil colonization, (b) after colonization, and (c) in natural soil communities. One month after inoculation with phage suspensions, bacterial communities were assessed by 16S rRNA amplicon gene sequencing.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00822-z)

10.1186/s40168-020-00822-z

---

## A predatory myxobacterium controls cucumber Fusarium wilt by regulating the soil microbial community.
 07.04.2020, _X Ye, Z Li, X Luo, W Wang, Y Li, R Li, B Zhang, Y Qiao, J Zhou, J Fan, H Wang, Y Huang, H Cao, Z Cui, R Zhang_


Myxobacteria are micropredators in the soil ecosystem with the capacity to move and feed cooperatively. Some myxobacterial strains have been used to control soil-borne fungal phytopathogens. However, interactions among myxobacteria, plant pathogens, and the soil microbiome are largely unexplored. In this study, we aimed to investigate the behaviors of the myxobacterium Corallococcus sp. strain EGB in the soil and its effect on the soil microbiome after inoculation for controlling cucumber Fusarium wilt caused by Fusarium oxysporum f. sp. cucumerinum (FOC).

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00824-x)

10.1186/s40168-020-00824-x

---

## A plant genetic network for preventing dysbiosis in the phyllosphere

[Source](https://www.nature.com/articles/s41586-020-2185-0)

---

## Fungal communities differentially respond to warming and drought in tropical grassland soil.
 03.04.2020, _TB de Oliveira, RC de Lucas, AS de Almeida Scarcella, AG Contato, TM Pasin, CA Martinez, M de Lourdes Teixeira de Moraes Polizeli_


Climate change is predicted to cause more extreme events, such as heatwaves, and different precipitation patterns. The effects of warming and short-term drought on soil microbial communities, in particular fungal communities, remain largely unexplored under field conditions. Here, we evaluated how the fungal community of a tropical grassland soil responds to these changes. A field experiment was carried out in a temperature free-air controlled enhancement (T-FACE) facility in Ribeirão Preto, Brazil. The isolated and combined effects of drought and a 2 °C increase in temperature were investigated. Based on metabarcoding of the ITS2 region, a total of 771 OTUs were observed. While warming affected the community structure, drought affected the alpha diversity, and the interaction between warming and drought affected both diversity and structure. The change in community composition driven by warming affected only the less abundant species (&gt;1% of the total sequences). The most affected aspect of the fungal communities was the diversity, which was increased by drought (P&lt;0.05), mostly by reducing the dominance of a single species, as observed in the watered plots. In a phylogenetic context, some fungal taxa were favored by changes in temperature (Hypocreales) and drought (Sordariales) or disadvantaged by both (Pleosporales). It is worth highlighting that a water deficit increased the abundance of phytopathogenic fungi, such as Curvularia, Thielavia, and Fusarium species. Overall, our results provide evidence that fungal communities in tropical grassland soils have greater sensitivity to drought than to temperature, which might increase the incidence of certain soil-borne diseases.

[Source](https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.15423)

10.1111/mec.15423

---

## An in-depth survey of the microbial landscape of the walls of a neonatal operating room

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0230957)

---

## Defining the resilience of the human salivary microbiota by a 520 days longitudinal study in confined environment: the Mars500 mission

[Source](https://www.biorxiv.org/content/10.1101/2020.04.08.031401v1.abstract?%3Fcollection=)

---

## Phage cocktail containing Podoviridae and Myoviridae bacteriophages inhibits the growth of Pectobacterium spp. under in vitro and in vivo conditions.
 10.04.2020, _MA Zaczek-Moczydłowska, GK Young, J Trudgett, C Plahe, CC Fleming, K Campbell, R O' Hanlon_


Globally, there is a high economic burden caused by pre- and post-harvest losses in vegetables, fruits and ornamentals due to soft rot diseases. At present, the control methods for these diseases are limited, but there is some promise in developing biological control products for use in Integrated Pest Management. This study sought to formulate a phage cocktail which would be effective against soft rot Pectobacteriaceae species affecting potato (Solanum tuberosum L.), with potential methods of application in agricultural systems, including vacuum-infiltration and soil drench, also tested. Six bacteriophages were isolated and characterized using transmission electron microscopy, and tested against a range of Pectobacterium species that cause soft rot/blackleg of potato. Isolated bacteriophages of the family Podoviridae and Myoviridae were able to control isolates of the Pectobacterium species: Pectobacterium atrosepticum and Pectobacterium carotovorum subsp. carotovorum. Genomic analysis of three Podoviridae phages did not indicate host genes transcripts or proteins encoding toxin or antibiotic resistance genes. These bacteriophages were formulated as a phage cocktail and further experiments showed high activity in vitro and in vivo to suppress Pectobacterium growth, potentially indicating their efficacy in formulation as a microbial pest control agent to use in planta.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0230842)

10.1371/journal.pone.0230842

---

## Dynamic genome evolution and complex virocell metabolism of globally-distributed giant viruses.
 09.04.2020, _M Moniruzzaman, CA Martinez-Gutierrez, AR Weinheimer, FO Aylward_


The discovery of eukaryotic giant viruses has transformed our understanding of the limits of viral complexity, but the extent of their encoded metabolic diversity remains unclear. Here we generate 501 metagenome-assembled genomes of Nucleo-Cytoplasmic Large DNA Viruses (NCLDV) from environments around the globe, and analyze their encoded functional capacity. We report a remarkable diversity of metabolic genes in widespread giant viruses, including many involved in nutrient uptake, light harvesting, and nitrogen metabolism. Surprisingly, numerous NCLDV encode the components of glycolysis and the TCA cycle, suggesting that they can re-program fundamental aspects of their host's central carbon metabolism. Our phylogenetic analysis of NCLDV metabolic genes and their cellular homologs reveals distinct clustering of viral sequences into divergent clades, indicating that these genes are virus-specific and were acquired in the distant past. Overall our findings reveal that giant viruses encode complex metabolic capabilities with evolutionary histories largely independent of cellular life, strongly implicating them as important drivers of global biogeochemical cycles.

[Source](https://www.nature.com/articles/s41467-020-15507-2?utm_source=ncomms_etoc&utm_medium=email&utm_campaign=toc_41467_11_1&utm_content=20200408&WT.ec_id=NCOMMS-20200408&sap-outbound-id=1FE7F29FB15AAFBF1CA0E0CE511FCC90AC6ED4B3&mkt-key=42010A0550671EEA9BA133EAFD9034AA)

10.1038/s41467-020-15507-2

---

## Infection with novel Bacteroides phage BV01 alters host transcriptome and bile acid metabolism in a common human gut microbe

[Source](https://www.biorxiv.org/content/10.1101/2020.04.06.028910v2.abstract?%3Fcollection=)

---

## Integrative and Conjugative Elements (ICE) and Associated Cargo Genes within and across Hundreds of Bacterial Genera

[Source](https://www.biorxiv.org/content/10.1101/2020.04.07.030320v1.abstract?%3Fcollection=)

---

## A restriction enzyme reduced representation sequencing approach for low-cost, high-throughput metagenome profiling.
 10.04.2020, _MK Hess, SJ Rowe, TC Van Stijn, HM Henry, SM Hickey, R Brauning, AF McCulloch, AS Hess, MR Kirk, S Kumar, C Pinares-Patiño, S Kittelmann, GR Wood, PH Janssen, JC McEwan_


Microbial community profiles have been associated with a variety of traits, including methane emissions in livestock. These profiles can be difficult and expensive to obtain for thousands of samples (e.g. for accurate association of microbial profiles with traits), therefore the objective of this work was to develop a low-cost, high-throughput approach to capture the diversity of the rumen microbiome. Restriction enzyme reduced representation sequencing (RE-RRS) using ApeKI or PstI, and two bioinformatic pipelines (reference-based and reference-free) were compared to bacterial 16S rRNA gene sequencing using repeated samples collected two weeks apart from 118 sheep that were phenotypically extreme (60 high and 58 low) for methane emitted per kg dry matter intake (n = 236). DNA was extracted from freeze-dried rumen samples using a phenol chloroform and bead-beating protocol prior to RE-RRS. The resulting sequences were used to investigate the repeatability of the rumen microbial community profiles, the effect of laboratory and analytical method, and the relationship with methane production. The results suggested that the best method was PstI RE-RRS analyzed with the reference-free approach, which accounted for 53.3±5.9% of reads, and had repeatabilities of 0.49±0.07 and 0.50±0.07 for the first two principal components (PC1 and PC2), phenotypic correlations with methane yield of 0.43±0.06 and 0.46±0.06 for PC1 and PC2, and explained 41±8% of the variation in methane yield. These results were significantly better than for bacterial 16S rRNA gene sequencing of the same samples (p&lt;0.05) except for the correlation between PC2 and methane yield. A Sensitivity study suggested approximately 2000 samples could be sequenced in a single lane on an Illumina HiSeq 2500, meaning the current work using 118 samples/lane and future proposed 384 samples/lane are well within that threshold. With minor adaptations, our approach could be used to obtain microbial profiles from other metagenomic samples.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0219882)

10.1371/journal.pone.0219882

---

## Open microbiome dominated by Clostridium and Eubacterium converts methanol into i-butyrate and n-butyrate.
 05.04.2020, _S Huang, R Kleerebezem, K Rabaey, R Ganigué_


Isobutyrate (i-butyrate) is a versatile platform chemical, whose acid form is used as a precursor of plastic and emulsifier. It can be produced microbially either using genetically engineered organisms or via microbiomes, in the latter case starting from methanol and short-chain carboxylates. This opens the opportunity to produce i-butyrate from non-sterile feedstocks. Little is known on the ecology and process conditions leading to i-butyrate production. In this study, we steered i-butyrate production in a bioreactor fed with methanol and acetate under various conditions, achieving maximum i-butyrate productivity of 5.0 mM day-1, with a concurrent production of n-butyrate of 7.9 mM day-1. The production of i-butyrate was reversibly inhibited by methanogenic inhibitor 2-bromoethanesulfonate. The microbial community data revealed the co-dominance of two major OTUs during co-production of i-butyrate and n-butyrate in two distinctive phases throughout a period of 54 days and 28 days, respectively. The cross-comparison of product profile with microbial community composition suggests that the relative abundance of Clostridium sp. over Eubacterium sp. is correlated with i-butyrate productivity over n-butyrate productivity.

[Source](https://link.springer.com/article/10.1007%2Fs00253-020-10551-w)

10.1007/s00253-020-10551-w

---

## The Microbestiary

[Source](http://Microbestiary.org)

---

## A browser-based tool for visualization and analysis of diffusion MRI data.
 09.03.2020, _JD Yeatman, A Richie-Halford, JK Smith, A Keshavan, A Rokem_


Human neuroscience research faces several challenges with regards to reproducibility. While scientists are generally aware that data sharing is important, it is not always clear how to share data in a manner that allows other labs to understand and reproduce published findings. Here we report a new open source tool, AFQ-Browser, that builds an interactive website as a companion to a diffusion MRI study. Because AFQ-Browser is portable-it runs in any web-browser-it can facilitate transparency and data sharing. Moreover, by leveraging new web-visualization technologies to create linked views between different dimensions of the dataset (anatomy, diffusion metrics, subject metadata), AFQ-Browser facilitates exploratory data analysis, fueling new discoveries based on previously published datasets. In an era where Big Data is playing an increasingly prominent role in scientific discovery, so will browser-based tools for exploring high-dimensional datasets, communicating scientific discoveries, aggregating data across labs, and publishing data alongside manuscripts.

[Source](https://www.uwyo.edu/uw/news/2020/04/uw-student-one-of-40-selected-for-arctic-circle-expedition.html)

10.1038/s41467-018-03297-7

---

## an article published on the UW website

[Source](https://www.uwyo.edu/uw/news/2020/04/uw-student-one-of-40-selected-for-arctic-circle-expedition.html)

---

