## <em>Bacteroides vulgatus</em> and <em>Bacteroides dorei</em> predict immune-related adverse events in immune checkpoint blockade treatment of metastatic melanoma

[Source](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-021-00974-z)

---

## A Genome-Phenome Association study in native microbiomes identifies a mechanism for cytosine modification in DNA and RNA.
 08.11.2021, _W Yang, YC Lin, W Johnson, N Dai, R Vaisvila, P Weigele, YJ Lee, IR Corrêa, I Schildkraut, L Ettwiller_


Shotgun metagenomic sequencing is a powerful approach to study microbiomes in an unbiased manner and of increasing relevance for identifying novel enzymatic functions. However, the potential of metagenomics to relate from microbiome composition to function has thus far been underutilized. Here, we introduce the Metagenomics Genome-Phenome Association (MetaGPA) study framework, which allows linking genetic information in metagenomes with a dedicated functional phenotype. We applied MetaGPA to identify enzymes associated with cytosine modifications in environmental samples. From the 2365 genes that met our significance criteria, we confirm known pathways for cytosine modifications and proposed novel cytosine-modifying mechanisms. Specifically, we characterized and identified a novel nucleic acid modifying enzyme, 5-hydroxymethylcytosine carbamoyltransferase, that catalyzes the formation of a previously unknown cytosine modification, 5-carbamoyloxymethylcytosine, in DNA and RNA. Our work introduces MetaGPA as a novel and versatile tool for advancing functional metagenomics.

[Source](https://doi.org/10.7554/eLife.70021)

10.7554/eLife.70021

---

## The Gut Microbiome Modifies the Association between a Mediterranean Diet and Diabetes in US Hispanic / Latino Population.
 08.11.2021, _DD Wang, Q Qi, Z Wang, M Usyk, D Sotres-Alvarez, J Mattei, M Tamez, MD Gellman, M Daviglus, FB Hu, MJ Stampfer, C Huttenhower, R Knight, RD Burk, RC Kaplan_


The interrelationships among the gut microbiome, the MedDiet and a clinical endpoint of diabetes is unknown.

[Source](https://doi.org/10.1210/clinem/dgab815)

10.1210/clinem/dgab815

---

## Proteomics analysis of the gut-brain axis in a gut microbiota-dysbiosis model of depression

[Source](https://doi.org/10.1038/s41398-021-01689-w)

---

## The Capacity to Produce Hydrogen Sulfide (H(2)S) via Cysteine Degradation Is Ubiquitous in the Human Gut Microbiome

[Source](https://doi.org/10.3389/fmicb.2021.705583)

---

## Identification of a novel cancer microbiome signature for predicting prognosis of human breast cancer patients.
 06.11.2021, _AW Mao, H Barck, J Young, A Paley, J- Mao, H Chang_


Prognosis of breast cancer (BC) patients differs considerably and identifying reliable prognostic biomarker(s) is imperative. With evidence that the microbiome plays a critical role in the response to cancer therapies, we aimed to identify a cancer microbiome signature for predicting the prognosis of BC patients.

[Source](https://doi.org/10.1007/s12094-021-02725-3)

10.1007/s12094-021-02725-3

---

## Gut microbiota of adults with asthma is broadly similar to non-asthmatics in a large population with varied ethnic origins

[Source](https://doi.org/10.1080/19490976.2021.1995279)

---

## Effects of fecal microbiota transplant on DNA methylation in subjects with metabolic syndrome.
 08.11.2021, _EWJ van der Vossen, D Bastos, D Stols-Gonçalves, MC de Goffau, M Davids, JPB Pereira, AYF Li Yim, P Henneman, MG Netea, WM de Vos, W de Jonge, AK Groen, M Nieuwdorp, E Levin_


Accumulating evidence shows that microbes with their theater of activity residing within the human intestinal tract (i.e., the gut microbiome) influence host metabolism. Some of the strongest results come from recent fecal microbial transplant (FMT) studies that relate changes in intestinal microbiota to various markers of metabolism as well as the pathophysiology of insulin resistance. Despite these developments, there is still a limited understanding of the multitude of effects associated with FMT on the general physiology of the host, beyond changes in gut microbiome composition. We examined the effect of either allogenic (lean donor) or autologous FMTs on the gut microbiome, plasma metabolome, and epigenomic (DNA methylation) reprogramming in peripheral blood mononuclear cells in individuals with metabolic syndrome measured at baseline (pre-FMT) and after 6 weeks (post-FMT). Insulin sensitivity was determined with a stable isotope-based 2 step hyperinsulinemic clamp and multivariate machine learning methodology was used to uncover discriminative microbes, metabolites, and DNA methylation loci. A larger gut microbiota shift was associated with an allogenic than with autologous FMT. Furthemore, the data results of the the allogenic FMT group data indicates that the introduction of new species can potentially modulate the plasma metabolome and (as a result) the epigenome. Most notably, the introduction of Prevotella ASVs directly correlated with methylation of AFAP1, a gene involved in mitochondrial function, insulin sensitivity, and peripheral insulin resistance (Rd, rate of glucose disappearance). FMT was found to have notable effects on the gut microbiome but also on the host plasma metabolome and the epigenome of immune cells providing new avenues of inquiry in the context of metabolic syndrome treatment for the manipulation of host physiology to achieve improved insulin sensitivity.

[Source](https://doi.org/10.1080/19490976.2021.1993513)

10.1080/19490976.2021.1993513

---

## The Mucosally-Adherent Rectal Microbiota Contains Features Unique to Alcohol-Related Cirrhosis.
 08.11.2021, _TD Shen, SG Daniel, S Patel, E Kaplan, L Phung, K Lemelle-Thomas, L Chau, L Herman, C Trisolini, A Stonelake, E Toal, V Khungar, K Bittinger, KR Reddy, GD Wu_


Most studies examining correlations between the gut microbiota and disease states focus on fecal samples due to ease of collection, yet there are distinct differences when compared to samples collected from the colonic mucosa. Although fecal microbiota has been reported to be altered in cirrhosis, correlation with mucosal microbiota characterized via rectal swab has not been previously described in this patient population. We conducted a cross-sectional analysis using 39 stool and 39 rectal swabs from adult patients with cirrhosis of different etiologies and performed shotgun metagenomic sequencing. Bacterial growth studies were performed with Escherichia coli. Two asaccharolytic bacterial taxa, Finegoldia magna and Porphyromonas asaccharolytica, were increased in rectal swabs relative to stool (FDR &lt; 0.01). Genomic analysis of the microbiome revealed 58 genes and 16 pathways that differed between stool and rectal swabs (FDR &lt; 0.05), where rectal swabs were enriched for pathways associated with protein synthesis and cellular proliferation but decreased in carbohydrate metabolism. Although no features in the fecal microbiome differentiated cirrhosis etiologies, the mucosal microbiome revealed decreased abundances of E. coli and Enterobacteriaceae in alcohol-related cirrhosis relative to non-alcohol related cirrhosis (FDR &lt; 0.05). In vitro bacterial culture studies showed that physiological concentrations of ethanol and its oxidative metabolites inhibited E. coli growth in a pH- and concentration-dependent manner. Characterization of the mucosally associated gut microbiome via rectal swab revealed findings consistent with amino acid/nitrogen abundance versus carbohydrate limitation in the mucosal microenvironment as well as unique features of alcohol-related cirrhosis possibly consistent with the influence of host-derived metabolites on the composition of mucosally adherent microbiota.

[Source](https://doi.org/10.1080/19490976.2021.1987781)

10.1080/19490976.2021.1987781

---

## Multiple bacterial virulence factors focused on adherence and biofilm formation associate with outcomes in cirrhosis.
 08.11.2021, _JS Bajaj, A Shamsaddini, C Acharya, A Fagan, M Sikaroodi, E Gavis, S McGeorge, A Khoruts, M Fuchs, RK Sterling, H Lee, PM Gillevet_


Altered gut microbiota is associated with poor outcomes in cirrhosis, including infections and hepatic encephalopathy (HE). However, the role of bacterial virulence factors (VFs) is unclear. Aim: Define association of VFs with cirrhosis severity and infections, their linkage with outcomes, and impact of fecal microbiota transplant (FMT).

[Source](https://doi.org/10.1080/19490976.2021.1993584)

10.1080/19490976.2021.1993584

---

## Long-term dynamics of the human oral microbiome during clinical disease progression.
 07.11.2021, _A Duran-Pinedo, J Solbiati, F Teles, R Teles, Y Zang, J Frias-Lopez_


Oral microbiome dysbiosis is linked to overt inflammation of tooth-supporting tissues, leading to periodontitis, an oral condition that can cause tooth and bone loss. Microbiome dysbiosis has been described as a disruption in the symbiotic microbiota composition's stability that could adversely affect the host's health status. However, the precise microbiome dynamics that lead to dysbiosis and the progression of the disease are largely unknown. The objective of our study was to investigate the long-term dynamics of periodontitis progression and its connection to dysbiosis.

[Source](https://doi.org/10.1186/s12915-021-01169-z)

10.1186/s12915-021-01169-z

---

## Microbiome characteristics and the key biochemical reactions identified on stone world cultural heritage under different climate conditions.
 06.11.2021, _X Ding, W Lan, A Yan, Y Li, Y Katayama, JD Gu_


The surfaces of historical stone monuments are visibly covered with a layer of colonizing microorganisms and their degradation products. In this study, a metadata analysis was conducted using the microbial sequencing data available from NCBI database to determine the diversity, biodeterioration potential and functionality of the stone microbiome on important world cultural heritage sites under four different climatic conditions. The retrieved stone microbial community composition in these metagenomes shows a clear association between climate types of the historical monuments and the diversity and taxonomic composition of the stone microbiomes. Shannon diversity values showed that microbial communities on stone monuments exposed to dry climate were more diverse than those under humid ones. In particular, functions associated with photosynthesis and UV resistance were identified from geographical locations under different climate types. The distribution of key microbial determinants responsible for stone deterioration was linked to survival under extreme environmental conditions and biochemical capabilities and reactions. Among them, biochemical reactions of the microbial nitrogen and sulfur cycles were most predominant. These stone-dwelling microbiomes on historical stone monuments were highly diverse and self-sustaining driven by energy metabolism and biomass accumulation. And metabolic products of the internal geomicrobiological nitrogen cycling on these ancient monuments play a unique role in the biodeterioration of stone monuments. These results highlight the significance of identifying the essential microbial biochemical reactions to advance the understanding of stone biodeterioration for protection management.

[Source](https://doi.org/10.1016/j.jenvman.2021.114041)

10.1016/j.jenvman.2021.114041

---

## Fecal biomarkers of environmental enteric dysfunction and the gut microbiota of rural Malawian children: An observational study.
 08.11.2021, _D Chaima, H Pickering, JD Hart, SE Burr, KM Maleta, K Kalua, RL Bailey, MJ Holland_


Environmental enteric dysfunction (EED) is a subclinical condition of the gut characterized by changes in morphology and function with underlying chronic inflammatory responses. This study characterized composition and diversity of the gut microbiota in rural Malawian children with and without signs of EED. Fecal samples were collected from children aged 1-59 months. Neopterin, myeloperoxidase and alpha-1 antitrypsin concentrations were quantified by ELISA and combined to form a composite EED score using principal component analysis. DNA was extracted from fecal samples and V4-16S rRNA gene sequencing was used to characterize the gut microbiota. The concentrations of all three biomarkers decreased with increasing age, which is consistent with other studies of children living in similar low-income settings. Firmicutes, Bacteroidetes, Proteobacteria and Actinobacteria were the dominant phyla while Faecalibacterium and Bifidobacterium were the most prevalent genera. Increased alpha diversity was associated with a reduction in neopterin concentration. Microbiota composition was different between fecal samples with low and high composite EED scores; increased abundance of Succinivibrio was associated with reduced composite EED scores.

[Source](https://doi.org/10.1016/j.heliyon.2021.e08194)

10.1016/j.heliyon.2021.e08194

---

## The Evolving Role of Consensus Molecular Subtypes: a Step Beyond Inpatient Selection for Treatment of Colorectal Cancer.
 08.11.2021, _J Ros, I Baraibar, G Martini, F Salvà, N Saoudi, JL Cuadra-Urteaga, R Dienstmann, J Tabernero, E Élez_


The heterogenous nature of colorectal cancer (CRC) renders it a major clinical challenge. Increasing genomic understanding of CRC has improved our knowledge of this heterogeneity and the main cancer drivers, with significant improvements in clinical outcomes. Comprehensive molecular characterization has allowed clinicians a more precise range of treatment options based on biomarker selection. Furthermore, this deep molecular understanding likely extends therapeutic options to a larger number of patients. The biological associations of consensus molecular subtypes (CMS) with clinical outcomes in localized CRC have been validated in retrospective clinical trials. The prognostic role of CMS has also been confirmed in the metastatic setting, with CMS2 having the best prognosis, whereas CMS1 tumors are associated with a higher risk of progression and death after chemotherapy. Similarly, according to mesenchymal features and immunosuppressive molecules, CMS1 responds to immunotherapy, whereas CMS4 has a poorer prognosis, suggesting that a CMS1 signature could identify patients who may benefit from immune checkpoint inhibitors regardless of microsatellite instability (MSI) status. The main goal of these comprehensive analyses is to switch from "one marker-one drug" to "multi-marker drug combinations" allowing oncologists to give "the right drug to the right patient." Despite the revealing data from transcriptomic analyses, the high rate of intra-tumoral heterogeneity across the different CMS subgroups limits its incorporation as a predictive biomarker. In clinical practice, when feasible, comprehensive genomic tests should be performed to identify potentially targetable alterations, particularly in RAS/BRAF wild-type, MSI, and right-sided tumors. Furthermore, CMS has not only been associated with clinical outcomes and specific tumor and patient phenotypes but also with specific microbiome patterns. Future steps will include the integration of clinical features, genomics, transcriptomics, and microbiota to select the most accurate biomarkers to identify optimal treatments, improving individual clinical outcomes. In summary, CMS is context specific, identifies a level of heterogeneity beyond standard genomic biomarkers, and offers a means of maximizing personalized therapy.

[Source](https://doi.org/10.1007/s11864-021-00913-5)

10.1007/s11864-021-00913-5

---

## The role of the host microbiome in autism and neurodegenerative disorders and effect of epigenetic procedures in the brain functions.
 07.11.2021, _B Yousefi, P Kokhaei, F Mehranfar, A Bahar, A Abdolshahi, A Emadi, M Eslami_


Autism Spectrum Disorder (ASD) is a severe neurological/neurodegenerative syndrome that results in cognitive and communication disorders. The degree of dysbiosis is related to the severity of ASD signs. The gut is conferred with a variety of sensory receptors that cooperate with effector systems including the endocrine, nervous and gut immune systems of the intestine. Gut dysbiosis causes amplified inflammation, the launch of the HPA axis, changed levels of neurotransmitters and bacterial metabolites; these may donate to abnormal signaling throughout the Vagus nerve in ASD. Decreased integrity of the gastrointestinal barrier led to extreme leakage of substances as of the intestine in early life and inflammation followed by disruption of BBB integrity maybe increase the risk of ASD. Microbiota, by controlling the barrier permeability, regulate the quantity and types of bioactive materials that are transferred from the intestine to the brain. Exposure to metabolites and microbial products regulate significant procedures in the CNS, including glial cell role, myelination, synaptic pruning, and play a role in neurobehavioral, neurodegenerative, psychiatric, and metabolic syndrome.

[Source](https://doi.org/10.1016/j.neubiorev.2021.10.046)

10.1016/j.neubiorev.2021.10.046

---

## The Oral Microbiota and its Role in Carcinogenesis.
 07.11.2021, _M Stasiewicz, TM Karpiński_


Despite decades of research, cancer continues to be a major global health concern. In recent years, the role played by microorganisms in the development and progression of cancer has come under increased scrutiny. The aim of the present review is to highlight the main associations between members of the human oral microbiota and various cancers. The PubMed database was searched for available literature to outline the current state of understanding regarding the role of the oral microbiota and a variety of human cancers. Oral squamous cell carcinoma (OSCC) is associated with carriage of a number of oral bacteria (e.g., Porphyromonas gingivalis, Fusobacterium nucleatum, Streptococcus sp.), certain viruses (e.g., human papilloma virus, human herpes virus 8, herpes simplex virus 1 and Epstein-Barr virus) and yeast Candida albicans. Moreover, members of the oral microbiota are associated with cancers of the esophagus, stomach, pancreas, colon/rectum and lung. Furthermore, the present review outlines a number of the carcinogenic mechanisms underlying the presented microbial associations with cancer. Such information may one day help clinicians to diagnose neoplastic diseases at earlier stages and prescribe treatments that take into account the possible microbial nature of carcinogenesis.

[Source](https://doi.org/10.1016/j.semcancer.2021.11.002)

10.1016/j.semcancer.2021.11.002

---

## Gut microbiota: sculptors of the intestinal stem cell niche in health and inflammatory bowel disease.
 08.11.2021, _M Markandey, A Bajaj, NE Ilott, S Kedia, S Travis, F Powrie, V Ahuja_


Intestinal epithelium represents a dynamic and diverse cellular system that continuously interacts with gut commensals and external cues. Intestinal stem cells, which lie at the heart of epithelial renewal and turnover, proliferate to maintain a steady stem cell population and differentiate to form functional epithelial cell types. This rather sophisticated assembly-line is maintained by an elaborate micro-environment, sculpted by a myriad of host and gut microbiota-derived signals, forming an intestinal stem cell niche. This complex, yet crucial signaling niche undergoes dynamic changes during homeostasis and chronic intestinal inflammation. Inflammatory bowel disease refers to a chronic inflammatory response toward pathogenic or commensal microbiota, in a genetically susceptible host. Compositional and functional alterations in gut microbiota are pathognomonic of IBD.The present review highlights the modulatory role of gut microbiota on the intestinal stem cell niche during homeostasis and inflammatory bowel disease. We discuss the mechanisms of direct action of gut commensals (through microbiota-derived or microbiota-influenced metabolites) on ISCs, followed by their effects via other epithelial and immune cell types.

[Source](https://doi.org/10.1080/19490976.2021.1990827)

10.1080/19490976.2021.1990827

---

## The gastrointestinal microbiota in colorectal cancer cell migration and invasion.
 08.11.2021, _C Henstra, J van Praagh, P Olinga, A Nagelkerke_


Colorectal carcinoma is the third most common cancer in developed countries and the second leading cause of cancer-related mortality. Interest in the influence of the intestinal microbiota on CRC emerged rapidly in the past few years, and the close presence of microbiota to the tumour mass creates a unique microenvironment in CRC. The gastrointestinal microbiota secrete factors that can contribute to CRC metastasis by influencing, for example, epithelial-to-mesenchymal transition. Although the role of EMT in metastasis is well-studied, mechanisms by which gastrointestinal microbiota contribute to the progression of CRC remain poorly understood. In this review, we will explore bacterial factors that contribute to the migration and invasion of colorectal carcinoma and the mechanisms involved. Bacteria involved in the induction of metastasis in primary CRC include Fusobacterium nucleatum, Enterococcus faecalis, enterotoxigenic Bacteroides fragilis, Escherichia coli and Salmonella enterica. Examples of prominent bacterial factors secreted by these bacteria include Fusobacterium adhesin A and Bacteroides fragilis Toxin. Most of these factors induce EMT-like properties in carcinoma cells and, as such, contribute to disease progression by affecting cell-cell adhesion, breakdown of the extracellular matrix and reorganisation of the cytoskeleton. It is of utmost importance to elucidate how bacterial factors promote CRC recurrence and metastasis to increase patient survival. So far, mainly animal models have been used to demonstrate this interplay between the host and microbiota. More human-based models are needed to study the mechanisms that promote migration and invasion and mimic the progression and recurrence of CRC.

[Source](https://doi.org/10.1007/s10585-021-10130-x)

10.1007/s10585-021-10130-x

---

## Zebrafish Axenic Larvae Colonization with Human Intestinal Microbiota.
 03.07.2018, _N Arias-Jayo, L Alonso-Saez, A Ramirez-Garcia, MA Pardo_


_Animals, Feces, Gastrointestinal Microbiome, High-Throughput Nucleotide Sequencing, Humans, Intestines, Larva, Models, Animal, Zebrafish_
The human intestine hosts a vast and complex microbial community that is vital for maintaining several functions related with host health. The processes that determine the gut microbiome composition are poorly understood, being the interaction between species, the external environment, and the relationship with the host the most feasible. Animal models offer the opportunity to understand the interactions between the host and the microbiota. There are different gnotobiotic mice or rat models colonized with the human microbiota, however, to our knowledge, there are no reports on the colonization of germ-free zebrafish with a complex human intestinal microbiota. In the present study, we have successfully colonized 5 days postfertilization germ-free zebrafish larvae with the human intestinal microbiota previously extracted from a donor and analyzed by high-throughput sequencing the composition of the transferred microbial communities that established inside the zebrafish gut. Thus, we describe for first time which human bacteria phylotypes are able to colonize the zebrafish digestive tract. Species with relevant interest because of their linkage to dysbiosis in different human diseases, such as Akkermansia muciniphila, Eubacterium rectale, Faecalibacterium prausnitzii, Prevotella spp., or Roseburia spp. have been successfully transferred inside the zebrafish digestive tract.

[Source](https://doi.org/10.1016/j.anaerobe.2021.102472)

10.1089/zeb.2017.1460

---

## <em>Akkermansia muciniphila</em> and host interaction within the intestinal tract

[Source](https://doi.org/10.1016/j.anaerobe.2021.102472)

---

## Effects of anti-aging interventions on intestinal microbiota.
 08.11.2021, _Y Du, Y Gao, B Zeng, X Fan, D Yang, M Yang_


Identifying ways to deal with the challenges presented by aging is an urgent task, as we are facing an aging society. External factors such as diet, exercise and drug therapy have proven to be major elements in controlling healthy aging and prolonging life expectancy. More recently, the intestinal microbiota has also become a key factor in the anti-aging process. As the intestinal microbiota changes with aging, an imbalance in intestinal microorganisms can lead to many age-related degenerative diseases and unhealthy aging. This paper reviews recent research progress on the relationship between intestinal microorganisms and anti-aging effects, focusing on the changes and beneficial effects of intestinal microorganisms under dietary intervention, exercise and drug intervention. In addition, bacteriotherapy has been used to prevent frailty and unhealthy aging. Most of these anti-aging approaches improve the aging process and age-related diseases by regulating the homeostasis of intestinal flora and promoting a healthy intestinal environment. Intervention practices based on intestinal microorganisms show great potential in the field of anti-aging medicine.

[Source](https://doi.org/10.1080/19490976.2021.1994835)

10.1080/19490976.2021.1994835

---

## Comparative Study of the Gut Microbiota Among Four Different Marine Mammals in an Aquarium.
 08.11.2021, _S Bai, P Zhang, C Zhang, J Du, X Du, C Zhu, J Liu, P Xie, S Li_


Despite an increasing appreciation in the importance of host-microbe interactions in ecological and evolutionary processes, information on the gut microbial communities of some marine mammals is still lacking. Moreover, whether diet, environment, or host phylogeny has the greatest impact on microbial community structure is still unknown. To fill part of this knowledge gap, we exploited a natural experiment provided by an aquarium with belugas (Delphinapterus leucas) affiliated with family Monodontidae, Pacific white-sided dolphins (Lagenorhynchus obliquidens) and common bottlenose dolphin (Tursiops truncatus) affiliated with family Delphinidae, and Cape fur seals (Arctocephalus pusillus pusillus) affiliated with family Otariidae. Results show significant differences in microbial community composition of whales, dolphins, and fur seals and indicate that host phylogeny (family level) plays the most important role in shaping the microbial communities, rather than food and environment. In general, the gut microbial communities of dolphins had significantly lower diversity compared to that of whales and fur seals. Overall, the gut microbial communities were mainly composed of Firmicutes and Gammaproteobacteria, together with some from Bacteroidetes, Fusobacteria, and Epsilonbacteraeota. However, specific bacterial lineages were differentially distributed among the marine mammal groups. For instance, Lachnospiraceae, Ruminococcaceae, and Peptostreptococcaceae were the dominant bacterial lineages in the gut of belugas, while for Cape fur seals, Moraxellaceae and Bacteroidaceae were the main bacterial lineages. Moreover, gut microbial communities in both Pacific white-sided dolphins and common bottlenose dolphins were dominated by a number of pathogenic bacteria, including Clostridium perfringens, Vibrio fluvialis, and Morganella morganii, reflecting the poor health condition of these animals. Although there is a growing recognition of the role microorganisms play in the gut of marine mammals, current knowledge about these microbial communities is still severely lacking. Large-scale research studies should be undertaken to reveal the roles played by the gut microbiota of different marine mammal species.

[Source](https://doi.org/10.1016/j.jped.2021.10.001)

10.3389/fmicb.2021.769012

---

## Impact of the environment on the microbiome

[Source](https://doi.org/10.1016/j.jped.2021.10.001)

---

## Progress and prospects of the healthy human gut virome.
 06.11.2021, _SR Stockdale, C Hill_


Not all viruses associated with humans cause disease. Non-pathogenic human-infecting viruses are predicted as important for immune system induction and preparation. Phages that infect bacteria are the most abundant predators of the human microbial ecosystem, promoting and maintaining bacterial diversity. Metagenomic analyses of the human gut virome and microbiome are unravelling the intricate predator-prey dynamics of phage-bacteria co-existence, co-evolution, and their interplay with the human host. While most adults harbour a distinctly individualistic and persistent community of virulent phages, new-borns are dominated by temperate phages heavily influenced by environmental exposures. The future development of microbiome-based interventions, therapeutics, and manipulation, will require a greater understanding of the human microbiome and the virome.

[Source](https://doi.org/10.1016/j.coviro.2021.10.001)

10.1016/j.coviro.2021.10.001

---

## Integrating the human microbiome in the forensic toolkit: Current bottlenecks and future solutions.
 06.11.2021, _C Díez López, A Vidaki, M Kayser_


Over the last few years, advances in massively parallel sequencing technologies (also referred to next generation sequencing) and bioinformatics analysis tools have boosted our knowledge on the human microbiome. Such insights have brought new perspectives and possibilities to apply human microbiome analysis in many areas, particularly in medicine. In the forensic field, the use of microbial DNA obtained from human materials is still in its infancy but has been suggested as a potential alternative in situations when other human (non-microbial) approaches present limitations. More specifically, DNA analysis of a wide variety of microorganisms that live in and on the human body offers promises to answer various forensically relevant questions, such as post-mortem interval estimation, individual identification, and tissue/body fluid identification, among others. However, human microbiome analysis currently faces significant challenges that need to be considered and overcome via future forensically oriented human microbiome research to provide the necessary solutions. In this perspective article, we discuss the most relevant biological, technical and data-related issues and propose future solutions that will pave the way towards the integration of human microbiome analysis in the forensic toolkit.

[Source](https://doi.org/10.1016/j.fsigen.2021.102627)

10.1016/j.fsigen.2021.102627

---

## Bioactives and their metabolites from <em>Tetrastigma hemsleyanum</em> leaves ameliorate DSS-induced colitis via protecting the intestinal barrier, mitigating oxidative stress and regulating the gut microbiota

[Source](https://doi.org/10.1039/d1fo02588k)

---

## Bacterial membrane vesicles and phages in blood after consumption of <em>Lacticaseibacillus rhamnosus</em> JB-1

[Source](https://doi.org/10.1080/19490976.2021.1993583)

---

## β-Glucan-triggered <em>Akkermansia muciniphila</em> expansion facilitates the expulsion of intestinal helminth via TLR2 in mice

[Source](https://doi.org/10.1016/j.carbpol.2021.118719)

---

## The impact of a low-calorie, low-glycemic diet on systemic lupus erythematosus: a systematic review.
 07.11.2021, _AM Imoto, LB Gottems, AL Salomon, HECE Silva, IL Júnior, MS Peccin, FF Amorim, LA Santana_


Diet plays a critical role in Systemic Lupus Erythematosus (SLE) patients, impacting on the microbiota composition and, consequently, on the immune response. The objective was to analyze and verify the diet effect on SLE patients.

[Source](https://doi.org/10.1186/s42358-021-00224-1)

10.1186/s42358-021-00224-1

---

## Recent advances in the concept of paraprobiotics: Nutraceutical/functional properties for promoting children health.
 08.11.2021, _H Kumar, F Schütz, K Bhardwaj, R Sharma, E Nepovimova, DS Dhanjal, R Verma, D Kumar, K Kuča, N Cruz-Martins_


Probiotics consumption has been associated with various health promoting benefits, including disease prevention and even treatment by modulating gut microbiota. Contrary to this, probiotics may also overstimulate the immune system, trigger systemic infections, harmful metabolic activities, and promote gene transfer. In children, the fragile immune system and impaired intestinal barrier may boost the occurrence of adverse effects following probiotics' consumption. To overcome these health challenges, the key focus has been shifted toward non-viable probiotics, also called paraprobiotics. Cell wall polysaccharides, peptidoglycans, surface proteins and teichoic acid present on cell's surface are involved in the interaction of paraprobiotics with the host, ultimately providing health benefits. Among other benefits, paraprobiotics possess the ability to regulate innate and adaptive immunity, exert anti-adhesion, anti-biofilm, anti-hypertensive, anti-inflammatory, antioxidant, anti-proliferative, and antagonistic effects against pathogens, while also enhance clinical impact and general safety when administered in children in comparison to probiotics. Clinical evidence have underlined the paraprobiotics impact in children and young infants against atopic dermatitis, respiratory and gastrointestinal infections, in addition to be useful for immunocompromised individuals. Therefore, this review focuses on probiotics-related issues in children's health and also discusses the Lactobacillus and Bifidobacterium spp. qualities for qualifying as paraprobiotics and their role in promoting the children's health.

[Source](https://doi.org/10.1080/10408398.2021.1996327)

10.1080/10408398.2021.1996327

---

## Breastfeeding Contributes to Physiological Immune Programming in the Newborn.
 08.11.2021, _A Camacho-Morales, M Caba, M García-Juárez, MD Caba-Flores, R Viveros-Contreras, C Martínez-Valenzuela_


The first 1,000 days in the life of a human being are a vulnerable stage where early stimuli may program adverse health outcomes in future life. Proper maternal nutrition before and during pregnancy modulates the development of the fetus, a physiological process known as fetal programming. Defective programming promotes non-communicable chronic diseases in the newborn which might be prevented by postnatal interventions such as breastfeeding. Breast milk provides distinct bioactive molecules that contribute to immune maturation, organ development, and healthy microbial gut colonization, and also secures a proper immunological response that protects against infection and inflammation in the newborn. The gut microbiome provides the most critical immune microbial stimulation in the newborn in early life, allowing a well-trained immune system and efficient metabolic settings in healthy subjects. Conversely, negative fetal programming by exposing mothers to diets rich in fat and sugar has profound effects on breast milk composition and alters the immune profiles in the newborn. At this new stage, newborns become vulnerable to immune compromise, favoring susceptibility to defective microbial gut colonization and immune response. This review will focus on the importance of breastfeeding and its immunological biocomponents that allow physiological immune programming in the newborn. We will highlight the importance of immunological settings by breastfeeding, allowing proper microbial gut colonization in the newborn as a window of opportunity to secure effective immunological response.

[Source](https://doi.org/10.3389/fped.2021.744104)

10.3389/fped.2021.744104

---

## Electroacupuncture could balance the gut microbiota and improve the learning and memory abilities of Alzheimer's disease animal model.
 08.11.2021, _J Jiang, H Liu, Z Wang, H Tian, S Wang, J Yang, J Ren_


Alzheimer's disease (AD), as one of most common dementia, mainly affects older people from the worldwide. In this study, we intended to explore the possible mechanism of improving cognitive function and protecting the neuron effect by electroacupuncture.

[Source](https://doi.org/10.1371/journal.pone.0259530)

10.1371/journal.pone.0259530

---

## Protective Effects of <em>Bacillus amyloliquefaciens</em> 40 Against <em>Clostridium perfringens</em> Infection in Mice

[Source](https://doi.org/10.3389/fnut.2021.733591)

---

## Evidence of MHC class I and II influencing viral and helminth infection via the microbiome in a non-human primate

[Source](https://doi.org/10.1371/journal.ppat.1009675)

---

## Metagenomic and network analysis revealed wide distribution of antibiotic resistance genes in monkey gut microbiota.
 06.11.2021, _Y Yan, H Li, A Fayyaz, Y Gai_


The emergence and spread of drug-resistant microorganisms that have acquired new resistance mechanisms, leading to antibiotic resistance, continue to threaten the health of humans and animals worldwide. Non-human primates (NHPs), as close living relatives of human beings in the world, have a high degree of genetic and physiological similarity to humans. However, despite its importance, we lack a comprehensive characterization or understanding of the similarities and differences of the antibiotic resistance genes of the gut microbiome carried by non-human primates and humans. In the present study, the diversity and abundance of antibiotic resistance genes carried by the gut microbiota of cynomolgus monkeys (Macaca fascicularis) were investigated by metagenomic analysis. In total, 60 resistance types conferring resistance to 11 categories of antibiotics were identified in the gut microbiome of cynomolgus monkeys. Interestingly, the composition and abundance of ARGs carried by the gut microbiota of cynomolgus monkeys can be significantly affected by dietary changes. Moreover, we found that all ARG types carried by humans are also present in cynomolgus monkeys. The tetracycline resistance gene tet(37) is evolutionarily conserved and highly homologous. Taken together, our study provides a comprehensive overview of the diversity and richness of ARGs in the gut microbiota of cynomolgus monkeys and underlines the potentially crucial role of diet in the gut health of monkeys and humans.

[Source](https://doi.org/10.1016/j.micres.2021.126895)

10.1016/j.micres.2021.126895

---

## Comparative Study of the Gut Microbiota Among Four Different Marine Mammals in an Aquarium

[Source](https://doi.org/10.3389/fmicb.2021.769012)

---

## Bacterial Communities of Lab and Field Northern House Mosquitoes (Diptera: Culicidae) Throughout Diapause.
 08.11.2021, _EM Didion, M Doyle, JB Benoit_


Diapause is a hormonally driven response which is triggered by environmental cues that signal impending adverse conditions and prompts metabolic, developmental, and behavioral changes to allow survival until the return of favorable conditions. Microbial symbionts have been shown to influence the metabolism, development, and behavior of their host organisms, all of which are common diapause-associated characteristics. Surveys of bacterial components in relation to diapause have been examined in few systems, of which the species are usually inactive during dormancy, such as eggs or pupae. This is specifically intriguing as adult female diapause in Culex pipiens (Diptera: Culicidae) can last between 4 and 7 mo and females remain mobile within their hibernacula. Furthermore, it is unknown how microbiota changes associated with prolonged dormancy are different between the lab and field for insect systems. This study aims to characterize how the microbiota of C. pipiens changes throughout diapause under both field and lab settings when provided identical food and water resources. Based on these studies, C. pipiens microbiota shifts as diapause progresses and there are considerable differences between field and lab individuals even when provided the same carbohydrate and water sources. Specific bacterial communities have more association with different periods of diapause, field and lab rearing conditions, and nutritional reserve levels. These studies highlight that diapausing mosquito microbiota studies ideally should occur in field mesocosms and at multiple locations, to increase applicability to wild C. pipiens as prolonged exposure to artificial rearing conditions could impact metrics related to diapause-microbiome interactions. Additionally, these findings suggest that it would be worthwhile to establish if the microbiota shift during diapause impacts host physiology and whether this shift is critical to diapause success.

[Source](https://doi.org/10.1093/jme/tjab184)

10.1093/jme/tjab184

---

## Gut Microbiome of Wild Baltic Salmon (<em>Salmo salar</em> L.) Parr

[Source](https://doi.org/10.1007/s00248-021-01910-9)

---

## Concerted impacts of antiherbivore defenses and opportunistic <em>Serratia pathogens</em> on the fall armyworm (<em>Spodoptera frugiperda</em>)

[Source](https://doi.org/10.1007/s00442-021-05072-w)

---

## Sculpting the soil microbiota

[Source](https://doi.org/10.1111/tpj.15568)

---

## Identification of bacterial communities in extreme sites of Pakistan using high throughput barcoded amplicon sequencing.
 08.11.2021, _A Fariq, A Yasmin, JC Blazier, S Jannat_


Microorganisms thrive nearly everywhere including extreme environments where few other forms of life can exist. Geochemistry of extreme sites plays a major role in shaping these microbial communities and microbes thriving in such harsh conditions are untapped sources of novel biomolecules. To understand the structure and composition of such microbial communities, culture-independent bacterial diversity was characterised for two extreme sites in Pakistan, Khewra salt range and Murtazaabad hot spring. Barcoded amplicon sequencing technique was used to study the microbial communities. Physicochemical analysis of these sites was also conducted to study the dynamics of microbial communities under stressed conditions. Metagenomic sequencing of salt range soil samples yielded of 40,433 16S rRNA sequences, while hot spring sediments produced 76,449 16S rRNA sequence reads. Proteobacteria were predominant in saline soil while Firmicutes were most abundant in hot spring sediment. The taxonomic analysis of saline samples revealed 914 operational taxonomic units (OTUs) while that of hot spring sequences were clustered into 726 distinct OTUs. OTUs from genus Alkalibacillus were most abundant in hot spring sediments, whereas Haloarcula were more prevalent in saline soil. Some unidentified sequences were also present at each taxonomic level. Multivariate analysis indicated that electrical conductivity and pH are the major environmental factors involved in modelling microbial communities. This study revealed a poly-extremophilic microbial community in the Murtazaabad hot spring and characterised the unexplored halophilic microbial diversity of saline soil of Pakistan.

[Source](https://www.nature.com/articles/s41579-021-00648-y)

10.3897/BDJ.9.e68929

---

## Microbial diversity in extreme environments

[Source](https://www.nature.com/articles/s41579-021-00648-y)

---

## A large-scale genome-based survey of acidophilic Bacteria suggests that genome streamlining is an adaption for life at low pH

[Source](https://www.biorxiv.org/content/10.1101/2021.11.05.467356v1)

---

## Bacterial Community Structure and Metabolic Function Succession During the Composting of Distilled Grain Waste.
 08.11.2021, _SP Wang, ZY Sun, ST Wang, HW Yuan, MZ An, ZY Xia, YQ Tang, CH Shen, K Kida_


Distilled grain waste (DGW) can be converted to organic fertilizer via aerobic composting process without inoculating exogenous microorganisms. To illustrate the material conversion mechanism, this study investigated the dynamic changes of bacterial community structure and metabolic function involved in DGW composting. Results showed that a significant increase in microbial community alpha diversity was observed during DGW composting. Moreover, unique community structures occurred at each composting stage. The dominant phyla were Firmicutes, Proteobacteria, Actinobacteriota, Bacteroidota, Myxococcota, and Chloroflexi, whose abundance varied according to different composting stages. Keystone microbes can be selected as biomarkers for each stage, and Microbispora, Chryseolinea, Steroidobacter, Truepera, and Luteimonas indicating compost maturity. Co-occurrence network analysis revealed a significant relationship between keystone microbes and environmental factors. The carbohydrate and amino acid metabolism were confirmed as the primary metabolic pathways by metabolic function profiles. Furthermore, nitrogen metabolism pathway analysis indicated that denitrification and NH3 volatilization induced higher nitrogen loss during DGW composting. This study can provide new understanding of the microbiota for organic matter and nitrogen conversion in the composting process of DGW.

[Source](https://doi.org/10.1007/s12010-021-03731-5)

10.1007/s12010-021-03731-5

---

## AMDB: a database of animal gut microbial communities with manually curated metadata.
 08.11.2021, _J Yang, J Park, Y Jung, J Chun_


Variations in gut microbiota can be explained by animal host characteristics, including host phylogeny and diet. However, there are currently no databases that allow for easy exploration of the relationship between gut microbiota and diverse animal hosts. The Animal Microbiome Database (AMDB) is the first database to provide taxonomic profiles of the gut microbiota in various animal species. AMDB contains 2530 amplicon data from 34 projects with manually curated metadata. The total data represent 467 animal species and contain 10 478 bacterial taxa. This novel database provides information regarding gut microbiota structures and the distribution of gut bacteria in animals, with an easy-to-use interface. Interactive visualizations are also available, enabling effective investigation of the relationship between the gut microbiota and animal hosts. AMDB will contribute to a better understanding of the gut microbiota of animals. AMDB is publicly available without login requirements at http://leb.snu.ac.kr/amdb.

[Source](https://doi.org/10.1093/nar/gkab1009)

10.1093/nar/gkab1009

---

## RESCRIPt: Reproducible sequence taxonomy reference database management.
 08.11.2021, _MS Robeson, DR O'Rourke, BD Kaehler, M Ziemski, MR Dillon, JT Foster, NA Bokulich_


Nucleotide sequence and taxonomy reference databases are critical resources for widespread applications including marker-gene and metagenome sequencing for microbiome analysis, diet metabarcoding, and environmental DNA (eDNA) surveys. Reproducibly generating, managing, using, and evaluating nucleotide sequence and taxonomy reference databases creates a significant bottleneck for researchers aiming to generate custom sequence databases. Furthermore, database composition drastically influences results, and lack of standardization limits cross-study comparisons. To address these challenges, we developed RESCRIPt, a Python 3 software package and QIIME 2 plugin for reproducible generation and management of reference sequence taxonomy databases, including dedicated functions that streamline creating databases from popular sources, and functions for evaluating, comparing, and interactively exploring qualitative and quantitative characteristics across reference databases. To highlight the breadth and capabilities of RESCRIPt, we provide several examples for working with popular databases for microbiome profiling (SILVA, Greengenes, NCBI-RefSeq, GTDB), eDNA and diet metabarcoding surveys (BOLD, GenBank), as well as for genome comparison. We show that bigger is not always better, and reference databases with standardized taxonomies and those that focus on type strains have quantitative advantages, though may not be appropriate for all use cases. Most databases appear to benefit from some curation (quality filtering), though sequence clustering appears detrimental to database quality. Finally, we demonstrate the breadth and extensibility of RESCRIPt for reproducible workflows with a comparison of global hepatitis genomes. RESCRIPt provides tools to democratize the process of reference database acquisition and management, enabling researchers to reproducibly and transparently create reference materials for diverse research applications. RESCRIPt is released under a permissive BSD-3 license at https://github.com/bokulich-lab/RESCRIPt.

[Source](https://doi.org/10.1371/journal.pcbi.1009581)

10.1371/journal.pcbi.1009581

---

## Reprocessing 16S rRNA Gene Amplicon Sequencing Studies: (Meta)Data Issues, Robustness, and Reproducibility.
 08.11.2021, _X Kang, DM Deng, W Crielaard, BW Brandt_


High-throughput sequencing technology provides an efficient method for evaluating microbial ecology. Different bioinformatics pipelines can be used to convert 16S ribosomal RNA gene amplicon sequencing data into an operational taxonomic unit (OTU) table that is used to analyze microbial communities. It is important to assess the robustness of these pipelines, each with specific algorithms and/or parameters, and their influence on the outcome of statistical tests. Articles with publicly available datasets on the oral microbiome were searched for, and five datasets were retrieved. These were from studies on changes in microbiota related to smoking, oral cancer, caries, diabetes, or periodontitis. Next, the data was processed with four pipelines based on VSEARCH, USEARCH, mothur, and UNOISE3. OTU tables were rarefied, and differences in α-diversity and β-diversity were tested for different groups in a dataset. Finally, these results were checked for consistency among these example pipelines. Of articles that deposited data, only 57% made all sequencing and metadata available. When processing the datasets, issues were encountered, caused by read characteristics and differences between tools and their defaults in combination with a lack of detail in the methodology of the articles. In general, the four mainstream pipelines provided similar results, but importantly, P-values sometimes differed between pipelines beyond the significance threshold. Our results indicated that for published articles, the description of bioinformatics methods and data deposition should be improved, and regarding reproducibility, that analysis of multiple subsamples is required when using rarefying as library-size normalization method.

[Source](https://doi.org/10.3389/fcimb.2021.720637)

10.3389/fcimb.2021.720637

---

