## Metagenomic analysis exploring taxonomic and functional diversity of bacterial communities of a Himalayan urban fresh water lake

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0248116)

---

## A distinct clade of Bifidobacterium longum in the gut of Bangladeshi children thrives during weaning

[Source](https://www.sciencedirect.com/science/article/pii/S0092867422013265)

---

## Recent advances in systemic lupus erythematosus and microbiota: from bench to bedside.
 01.11.2022, _Y Zhan, Q Liu, B Zhang, X Huang, Q Lu_


Systemic lupus erythematosus (SLE) is a complicated autoimmune disease affecting multiple systems and organs. It is highly heterogeneous, and it preferentially affects women at childbearing age, causing worldwide social burden. The pathogenesis of SLE mostly involves genetic predisposition, epigenetic dysregulation, overactivation of the immune system, and environment factors. Human microbiome, which is mostly composed of microbiota colonized in the gut, skin, and oral cavity, provides a natural microbiome barrier against environmental risks. The past decade of research has demonstrated a strong association between microbiota and metabolic diseases or gastrointestinal diseases. However, the role of microbiota in autoimmunity remains largely unknown until recently, when the technological and methodological progress facilitates further microbiota research in SLE. In this review, the latest research about the role and mechanisms of microbiota in SLE and the advances in the development of diagnostic and therapeutic strategies based on microbiota for SLE were summarized.

[Source](https://link.springer.com/article/10.1007/s11684-022-0957-7)

10.1007/s11684-022-0957-7

---

## Identification of shared bacterial strains in the vaginal microbiota of related and unrelated reproductive-age mothers and daughters using genome-resolved metagenomic

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0275908)

---

## Gut microbiome dysbiosis in antibiotic-treated COVID-19 patients is associated with microbial translocation and bacteremia

[Source](https://www.nature.com/articles/s41467-022-33395-6)

---

## Clostridium butyricum and Bifidobacterium pseudolongum Attenuate the Development of Cardiac Fibrosis in Mice.
 01.11.2022, _J Wang, J Chen, L Li, H Zhang, D Pang, H Ouyang, X Jin, X Tang_


Cardiac fibrosis is an integral aspect of every form of cardiovascular diseases, which is one of the leading causes of death worldwide. It is urgent to explore new effective drugs and treatments. In this paper, transverse aortic constriction (TAC)-induced cardiac fibrosis was significantly alleviated by a cocktail of antibiotics to clear the intestinal flora, indicating that the gut microbiota was associated with the disease process of cardiac fibrosis. We transplanted feces from sham-operated and TAC-treated mice to mice treated with a cocktail of antibiotics. We found that TAC-treated gut microbiota dysbiosis cannot cause cardiac fibrosis on its own. Interestingly, healthy fecal microbiota transplantation could alleviate cardiac fibrosis, indicating that targeted probiotics and related metabolite intervention may restore a normal microenvironment for the treatment or prevention of cardiac fibrosis. We used 16S rRNA sequencing of fecal samples and discovered that butyric acid-producing bacteria and Bifidobacterium pseudolongum were the dominant bacteria in the group with the lowest degree of cardiac fibrosis. Moreover, we demonstrated that sodium butyrate prevented the development of cardiac fibrosis. The effect of Clostridium butyricum (butyric acid-producing bacteria) was better than that of B. pseudolongum on cardiac fibrosis. Surprisingly, the cocktail of two probiotics had a stronger ability than a single probiotic. In conclusion, therapies targeting the gut microbiota and metabolites such as probiotics present new strategies for treating cardiovascular disease. IMPORTANCE Cardiac fibrosis is a basic process in cardiac remodeling. It is related to almost all types of cardiovascular diseases (CVD) and has become an important global health problem. Basic research and a number of clinical studies have shown that myocardial fibrosis can be prevented and reversed to a certain extent. It is urgent to explore new effective drugs and treatments. We indicated a causal relationship between cardiac fibrosis and gut microbiota. Gut microbiota dysbiosis cannot cause cardiac fibrosis on its own. Interestingly, healthy fecal microbiota transplantation could alleviate cardiac fibrosis. According to our findings, the combined use of butyric acid-producing bacteria and B. pseudolongum can help prevent cardiac fibrosis. Therapies targeting the gut microbiota and metabolites, such as probiotics, represent new strategies for treating cardiovascular disease.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0014299922006161)

10.1128/spectrum.02524-22

---

## Gut microbiota and myocardial fibrosis

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0014299922006161)

---

## Assessing the relationship between physical activity and the gut microbiome in a large, population-based sample of Wisconsin adults

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0276684)

---

## Challenges and costs of donor screening for fecal microbiota transplantations.
 24.10.2022, _MV Bénard, CMA de Bruijn, AC Fenneman, K Wortelboer, J Zeevenhoven, B Rethans, HJ Herrema, T van Gool, M Nieuwdorp, MA Benninga, CY Ponsioen_


_Humans, Adolescent, Young Adult, Adult, Middle Aged, Aged, Fecal Microbiota Transplantation, Donor Selection, SARS-CoV-2, COVID-19, Feces, Clostridium Infections_
The increasing interest to perform and investigate the efficacy of fecal microbiota transplantation (FMT) has generated an urge for feasible donor screening. We report our experience with stool donor recruitment, screening, follow-up, and associated costs in the context of clinical FMT trials.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0276323)

10.1371/journal.pone.0276323

---

## Gut colonization by Proteobacteria alters host metabolism and modulates cocaine neurobehavioral responses

[Source](https://www.sciencedirect.com/science/article/pii/S193131282200470X)

---

## Enriching captivity conditions with natural elements does not prevent the loss of wild-like gut microbiota but shapes its compositional variation in two small mammals.
 01.11.2022, _A Koziol, I Odriozola, L Nyholm, A Leonard, C San José, J Pauperio, C Ferreira, AJ Hansen, O Aizpurua, MTP Gilbert, A Alberdi_


_Animals, Gastrointestinal Microbiome, Bacteria, Microbiota, Feces, Mammals, RNA, Ribosomal, 16S_
As continued growth in gut microbiota studies in captive and model animals elucidates the importance of their role in host biology, further pursuit of how to retain a wild-like microbial community is becoming increasingly important to obtain representative results from captive animals. In this study, we assessed how the gut microbiota of two wild-caught small mammals, namely Crocidura russula (Eulipotyphla, insectivore) and Apodemus sylvaticus (Rodentia, omnivore), changed when bringing them into captivity. We analyzed fecal samples of 15 A. sylvaticus and 21 C. russula, immediately after bringing them into captivity and 5 weeks later, spread over two housing treatments: a "natural" setup enriched with elements freshly collected from nature and a "laboratory" setup with sterile artificial elements. Through sequencing of the V3-V4 region of the 16S recombinant RNA gene, we found that the initial microbial diversity dropped during captivity in both species, regardless of treatment. Community composition underwent a change of similar magnitude in both species and under both treatments. However, we did observe that the temporal development of the gut microbiome took different trajectories (i.e., changed in different directions) under different treatments, particularly in C. russula, suggesting that C. russula may be more susceptible to environmental change. The results of this experiment do not support the use of microbially enriched environments to retain wild-like microbial diversities and compositions, yet show that specific housing conditions can significantly affect the drift of microbial communities under captivity.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/mbo3.1318)

10.1002/mbo3.1318

---

## Understanding the Dynamic of POMS Infection and the Role of Microbiota Composition in the Survival of Pacific Oysters, Crassostrea gigas.
 31.10.2022, _L Delisle, O Laroche, Z Hilton, JF Burguin, A Rolton, J Berry, X Pochon, P Boudry, J Vignier_


For over a decade, Pacific oyster mortality syndrome (POMS), a polymicrobial disease, induced recurring episodes of massive mortality affecting Crassostrea gigas oysters worldwide. Recent studies evidenced a combined infection of the ostreid herpesvirus (OsHV-1 μVar) and opportunistic bacteria in affected oysters. However, the role of the oyster microbiota in POMS is not fully understood. While some bacteria can protect hosts from infection, even minor changes to the microbial communities may also facilitate infection and worsen disease severity. Using a laboratory-based experimental infection model, we challenged juveniles from 10 biparental oyster families with previously established contrasted genetically based ability to survive POMS in the field. Combining molecular analyses and 16S rRNA gene sequencing with histopathological observations, we described the temporal kinetics of POMS and characterized the changes in microbiota during infection. By associating the microbiota composition with oyster mortality rate, viral load, and viral gene expression, we were able to identify both potentially harmful and beneficial bacterial amplicon sequence variants (ASVs). We also observed a delay in viral infection resulting in a later onset of mortality in oysters compared to previous observations and a lack of evidence of fatal dysbiosis in infected oysters. Overall, these results provide new insights into how the oyster microbiome may influence POMS disease outcomes and open new perspectives on the use of microbiome composition as a complementary screening tool to determine shellfish health and potentially predict oyster vulnerability to POMS. IMPORTANCE For more than a decade, Pacific oyster mortality syndrome (POMS) has severely impacted the Crassostrea gigas aquaculture industry, at times killing up to 100% of young farmed Pacific oysters, a key commercial species that is cultivated globally. These disease outbreaks have caused major financial losses for the oyster aquaculture industry. Selective breeding has improved disease resistance in oysters, but some levels of mortality persist, and additional knowledge of the disease progression and pathogenicity is needed to develop complementary mitigation strategies. In this holistic study, we identified some potentially harmful and beneficial bacteria that can influence the outcome of the disease. These results will contribute to advance disease management and aquaculture practices by improving our understanding of the mechanisms behind genetic resistance to POMS and assisting in predicting oyster vulnerability to POMS.

[Source](https://journals.asm.org/doi/10.1128/spectrum.01959-22)

10.1128/spectrum.01959-22

---

## Changes in Gut Microbiota Structure: A Potential Pathway for Silver Nanoparticles to Affect the Host Metabolism.
 31.10.2022, _XL Wang, N Yu, C Wang, HR Zhou, C Wu, L Yang, S Wei, AJ Miao_


Silver nanoparticles (AgNPs) are one of the most widely used NPs. Their adverse effects on either the host or its gut microbiota (GM) have been examined. Nevertheless, whether the GM plays any role in AgNP toxicity to the host remains unclear. In the present study, AgNPs were administered to mice by oral gavage once a day for 120 days. A significant dose-dependent accumulation of Ag in the liver was observed, with a steady state reached within 21 days. The AgNPs changed the structure of the GM, mainly with respect to microorganisms involved in the metabolism of energy, amino acids, organic acids, and lipids, as predicted in a PICRUST analysis. Effects of the AgNPs on liver metabolism were also demonstrated, as a KEGG pathway analysis showed the enrichment of pathways responsible for the metabolism of amino acids, purines and pyrimidine, lipids, and energy. More interestingly, the changes in GM structure and liver metabolism were highly correlated, evidenced by the correlation between ∼23% of the differential microorganisms at the genus level and ∼60% of the differential metabolites. This implies that the metabolic variations in liver as affected by AgNPs were partly attributable to NP-induced changes of GM structure. Therefore, our results demonstrate the importance of considering the roles of GM in the toxicity of NPs to the host in evaluations of the health risks of NPs.

[Source](https://pubs.acs.org/doi/10.1021/acsnano.2c07924)

10.1021/acsnano.2c07924

---

## Skin microbiota of oxazolone-induced contact hypersensitivity mouse model.
 24.10.2022, _K Mäenpää, S Wang, M Ilves, H El-Nezami, H Alenius, H Sinkko, P Karisola_


_Mice, Animals, Oxazolone, RNA, Ribosomal, 16S, Olive Oil, Acetone, Microbiota, Disease Models, Animal, Dermatitis, Allergic Contact, Inflammation, Bacteria_
Contact allergy is a common skin allergy, which can be studied utilising contact hypersensitivity (CHS) animal model. However, it is not clear, whether CHS is a suitable model to investigate skin microbiota interactions. We characterised the effect of contact dermatitis on the skin microbiota and studied the biological effects of oxazolone (OXA) -induced inflammation on skin thickness, immune cell numbers and changes of the microbiota in CHS mouse model (n = 72) for 28 days. Through 16S rRNA gene sequencing we defined the composition of bacterial communities and associations of bacteria with inflammation. We observed that the vehicle solution of acetone and olive oil induced bacterial community changes on day 1, and OXA-induced changes were observed mainly on day 7. Many of the notably enriched bacteria present in the OXA-challenged positive group represented the genus Faecalibaculum which were most likely derived from the cage environment. Additionally, skin inflammation correlated negatively with Streptococcus, which is considered a native skin bacterium, and positively with Muribacter muris, which is typical in oral environment. Skin inflammation favoured colonisation of cage-derived faecal bacteria, and additionally mouse grooming transferred oral bacteria on the skin. Due to the observed changes, we conclude that CHS model could be used for certain skin microbiome-related research set-ups. However, since vehicle exposure can alter the skin microbiome as such, future studies should include considerations such as careful control sampling and statistical tests to account for potential confounding factors.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0276071)

10.1371/journal.pone.0276071

---

## Effects of a farm-specific fecal microbial transplant (FMT) product on clinical outcomes and fecal microbiome composition in preweaned dairy calves.
 25.10.2022, _GS Slanzon, BJ Ridenhour, LM Parrish, SC Trombetta, DA Moore, WM Sischo, CS McConnel_


_Cattle, Animals, Farms, Fecal Microbiota Transplantation, Feces, Microbiota, Anti-Bacterial Agents_
Gastrointestinal disease (GI) is the most common illness in pre-weaned dairy calves. Therefore, effective strategies to manipulate the microbiome of dairy calves under commercial dairy operations are of great importance to improve animal health and reduce antimicrobial usage. The objective of this study was to develop a farm-specific FMT product and to investigate its effects on clinical outcomes and fecal microbial composition of dairy calves. The FMT product was derived from feces from healthy donors (5-24 days of age) raised in the same calf ranch facility as the FMT recipients. Healthy and diarrheic calves were randomly enrolled to a control (n = 115) or FMT (n = 112) treatment group (~36 g of processed fecal matter once daily for 3 days). Fecal samples were collected at enrollment and again 9 days later after the first FMT dose. Although the FMT product was rich in organisms typically known for their beneficial probiotic properties, the FMT therapy did not prevent or ameliorate GI disease in dairy calves. In fact, calves that received FMT were less likely to recover from GI disease, and more likely to die due to GI disease complications. Fecal microbial community analysis revealed an increase in the alpha-diversity in FMT calves; however, no major differences across treatment groups were observed in the beta-diversity analysis. Calves that received FMT had higher relative abundance of an uncultured organism of the genus Lactobacillus and Lactobacillus reuteri on day 10. Moreover, FMT calves had lower relative abundance of Clostridium nexile and Bacteroides vulgatus on day 10. Our results indicate the need to have an established protocol when developing FMT products, based on rigorous inclusion and exclusion criteria for the selection of FMT donors free of potential pathogens, no history of disease or antibiotic treatment.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0276638)

10.1371/journal.pone.0276638

---

## Multiomics Characterization of the Canada Goose Fecal Microbiome Reveals Selective Efficacy of Simulated Metagenomes.
 01.11.2022, _JC Gil, SM Hird_


16S rRNA amplicon sequences are predominantly used to identify the taxonomic composition of a microbiome, but they can also be used to generate simulated metagenomes to circumvent costly empirical shotgun sequencing. The effectiveness of using "simulated metagenomes" (shotgun metagenomes simulated from 16S rRNA amplicons using a database of full genomes closely related to the amplicons) in nonmodel systems is poorly known. We sought to determine the accuracy of simulated metagenomes in a nonmodel organism, the Canada goose (Branta canadensis), by comparing metagenomes and metatranscriptomes to simulated metagenomes derived from 16S amplicon sequencing. We found significant differences between the metagenomes, metatranscriptomes, and simulated metagenomes when comparing enzymes, KEGG orthologies (KO), and metabolic pathways. The simulated metagenomes accurately identified the majority (&gt;70%) of the total enzymes, KOs, and pathways. The simulated metagenomes accurately identified the majority of the short-chain fatty acid metabolic pathways crucial to folivores. When narrowed in scope to specific genes of interest, the simulated metagenomes overestimated the number of antimicrobial resistance genes and underestimated the number of genes related to the breakdown of plant matter. Our results suggest that simulated metagenomes should not be used in lieu of empirical sequencing when studying the functional potential of a nonmodel organism's microbiome. Regarding the function of the Canada goose microbiome, we found unexpected amounts of fermentation pathways, and we found that a few taxa are responsible for large portions of the functional potential of the microbiome. IMPORTANCE The taxonomic composition of a microbiome is predominately identified using amplicon sequencing of 16S rRNA genes, but as a single marker, it cannot identify functions (genes). Metagenome and metatranscriptome sequencing can determine microbiome function but can be cost prohibitive. Therefore, computational methods have been developed to generate simulated metagenomes derived from 16S rRNA sequences and databases of full-length genomes. Simulated metagenomes can be an effective alternative to empirical sequencing, but accuracy depends on the genomic database used and whether the database contains organisms closely related to the 16S sequences. These tools are effective in well-studied systems, but the accuracy of these predictions in a nonmodel system is less known. Using a nonmodel bird species, we characterized the function of the microbiome and compared the accuracy of 16S-derived simulated metagenomes to sequenced metagenomes. We found that the simulated metagenomes reflect most but not all functions of empirical metagenome sequencing.

[Source](https://journals.asm.org/doi/10.1128/spectrum.02384-22)

10.1128/spectrum.02384-22

---

## Uropygial gland microbiota differ between free-living and captive songbirds.
 02.11.2022, _LA Grieves, CLJ Bottini, GB Gloor, EA MacDougall-Shackleton_


_Animals, Scent Glands, Songbirds, Microbiota, Symbiosis, Sebaceous Glands, Passeriformes, Mammals_
Symbiotic microbes can affect host behavior and fitness. Gut microbiota have received the most study, with less attention to other important microbial communities like those of scent-producing glands such as mammalian anal glands and the avian uropygial gland. However, mounting evidence suggests that microbes inhabiting scent-producing glands play an important role in animal behavior by contributing to variation in chemical signals. Free-living and captive conditions typically differ in social environment, food diversity and availability, disease exposure, and other factors-all of which can translate into differences in gut microbiota. However, whether extrinsic factors such as captivity alter microbial communities in scent glands remains an open question. We compared the uropygial gland microbiota of free-living and captive song sparrows (Melospiza melodia) and tested for an effect of dietary manipulations on the gland microbiota of captive birds. As predicted, the uropygial gland microbiota was significantly different between free-living and captive birds. Surprisingly, microbial diversity was higher in captive than free-living birds, and we found no effect of dietary treatments on captive bird microbiota. Identifying the specific factors responsible for microbial differences among groups and determining whether changes in symbiotic microbiota alter behavior and fitness are important next steps in this field.

[Source](https://www.nature.com/articles/s41598-022-22425-4)

10.1038/s41598-022-22425-4

---

## Crop management shapes the diversity and activity of DNA and RNA viruses in the rhizosphere.
 28.10.2022, _G Muscatt, S Hilton, S Raguideau, G Teakle, IDEA Lidbury, EMH Wellington, C Quince, A Millard, GD Bending, E Jameson_


_Rhizosphere, Soil Microbiology, Plant Roots, Microbiota, Soil, Bacteria, RNA Viruses, Bacteriophages, Brassica napus, DNA_
The rhizosphere is a hotspot for microbial activity and contributes to ecosystem services including plant health and biogeochemical cycling. The activity of microbial viruses, and their influence on plant-microbe interactions in the rhizosphere, remains undetermined. Given the impact of viruses on the ecology and evolution of their host communities, determining how soil viruses influence microbiome dynamics is crucial to build a holistic understanding of rhizosphere functions.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01371-3)

10.1186/s40168-022-01371-3

---

## CRAMdb: a comprehensive database for composition and roles of microbiome in animals.
 01.11.2022, _B Lei, Y Xu, Y Lei, C Li, P Zhou, L Wang, Q Yang, X Li, F Li, C Liu, C Cui, T Chen, W Ni, S Hu_


CRAMdb (a database for composition and roles of animal microbiome) is a comprehensive resource of curated and consistently annotated metagenomes for non-human animals. It focuses on the composition and roles of the microbiome in various animal species. The main goal of the CRAMdb is to facilitate the reuse of animal metagenomic data, and enable cross-host and cross-phenotype comparisons. To this end, we consistently annotated microbiomes (including 16S, 18S, ITS and metagenomics sequencing data) of 516 animals from 475 projects spanning 43 phenotype pairs to construct the database that is equipped with 9430 bacteria, 278 archaea, 2216 fungi and 458 viruses. CRAMdb provides two main contents: microbiome composition data, illustrating the landscape of the microbiota (bacteria, archaea, fungi, and viruses) in various animal species, and microbiome association data, revealing the relationships between the microbiota and various phenotypes across different animal species. More importantly, users can quickly compare the composition of the microbiota of interest cross-host or body site and the associated taxa that differ between phenotype pairs cross-host or cross-phenotype. CRAMdb is freely available at (http://www.ehbio.com/CRAMdb).

[Source](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac973/6786202)

10.1093/nar/gkac973

---

## DNA extraction bias is more pronounced for microbial eukaryotes than for prokaryotes.
 01.11.2022, _A Brauer, MM Bengtsson_


_Eukaryota, Archaea, Bacteria, Microbiota, DNA_
DNA extraction and preservation bias is a recurring topic in DNA sequencing-based microbial ecology. The different methodologies can lead to distinct outcomes, which has been demonstrated especially in studies investigating prokaryotic community composition. Eukaryotic microbes are ubiquitous, diverse, and increasingly a subject of investigation in addition to bacteria and archaea. However, little is known about how the choice of DNA preservation and extraction methodology impacts perceived eukaryotic community composition. In this study, we compared the effect of two DNA preservation methods and six DNA extraction methods on the community profiles of both eukaryotes and prokaryotes in phototrophic biofilms on seagrass (Zostera marina) leaves from the Baltic Sea. We found that, whereas both DNA preservation and extraction method caused significant bias in perceived community composition for both eukaryotes and prokaryotes, extraction bias was more pronounced for eukaryotes than for prokaryotes. In particular, soft-bodied and hard-shelled eukaryotes like nematodes and diatoms, respectively, were differentially abundant depending on the extraction method. We conclude that careful consideration of DNA preservation and extraction methodology is crucial to achieving representative community profiles of eukaryotes in marine biofilms and likely all other habitats containing diverse eukaryotic microbial communities.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/mbo3.1323)

10.1002/mbo3.1323

---

## synDNA-a Synthetic DNA Spike-in Method for Absolute Quantification of Shotgun Metagenomic Sequencing.
 01.11.2022, _LS Zaramela, M Tjuanta, O Moyne, M Neal, K Zengler_


Microbiome studies have the common goal of determining which microbial taxa are present, respond to specific conditions, or promote phenotypic changes in the host. Most of these studies rely on relative abundance measurements to drive conclusions. Inherent limitations of relative values are the inability to determine whether an individual taxon is more or less abundant and the magnitude of this change between the two samples. These limitations can be overcome by using absolute abundance quantifications, which can allow for a more complete understanding of community dynamics by measuring variations in total microbial loads. Obtaining absolute abundance measurements is still technically challenging. Here, we developed synthetic DNA (synDNA) spike-ins that enable precise and cost-effective absolute quantification of microbiome data by adding defined amounts of synDNAs to the samples. We designed 10 synDNAs with the following features: 2,000-bp length, variable GC content (26, 36, 46, 56, or 66% GC), and negligible identity to sequences found in the NCBI database. Dilution pools were generated by mixing the 10 synDNAs at different concentrations. Shotgun metagenomic sequencing showed that the pools of synDNAs with different percentages of GC efficiently reproduced the serial dilution, showing high correlation (r = 0.96; R2 ≥ 0.94) and significance (P &lt; 0.01). Furthermore, we demonstrated that the synDNAs can be used as DNA spike-ins to generate linear models and predict with high accuracy the absolute number of bacterial cells in complex microbial communities. IMPORTANCE The synDNAs designed in this study enable accurate and reproducible measurements of absolute amount and fold changes of bacterial species in complex microbial communities. The method proposed here is versatile and promising as it can be applied to bacterial communities or genomic features like genes and operons, in addition to being easily adaptable by other research groups at a low cost. We also made the synDNAs' sequences and the plasmids available to encourage future application of the proposed method in the study of microbial communities.

[Source](https://journals.asm.org/doi/10.1128/msystems.00447-22)

10.1128/msystems.00447-22

---

