## The impact of maternal and early life malnutrition on health: a diet-microbe perspective.
 12.05.2020, _AJ Forgie, KM Drall, SL Bourque, CJ Field, AL Kozyrskyj, BP Willing_


Early-life malnutrition may have long-lasting effects on microbe-host interactions that affect health and disease susceptibility later in life. Diet quality and quantity in conjunction with toxin and pathogen exposure are key contributors to microbe-host physiology and malnutrition. Consequently, it is important to consider both diet- and microbe-induced pathologies as well as their interactions underlying malnutrition.

[Source](https://doi.org/10.1186/s12916-020-01584-z)

10.1186/s12916-020-01584-z

---

## Microbial Diagnostics for Cancer: A Step Forward but Not Prime Time Yet

[Source](https://doi.org/10.1016/j.ccell.2020.04.010)

---

## Oral-lung microbiome interactions in lung diseases.
 11.05.2020, _MJ Mammen, FA Scannapieco, S Sethi_


The proximity and continuity of the oral cavity and the lower respiratory tract allows the oropharyngeal microbiome to be a major determinant of the lung microbiome. In addition, host-pathogen interactions related to the oropharyngeal microbiome or its metabolites could propagate systemic inflammation or modulate host defense mechanisms that could affect other organs, including the lung. There is increasing appreciation of the pathophysiologic significance of the lung microbiome, not only in the classical infection-related diseases, pneumonia, bronchiectasis, and cystic fibrosis, but also in chronic noninfectious lung diseases, such as chronic obstructive pulmonary disease, asthma, and pulmonary fibrosis. In this review, we will explore the relationship of the oral microbiome with lung diseases, such as pneumonia, chronic obstructive pulmonary disease, asthma, and cystic fibrosis.

[Source](https://doi.org/10.1111/prd.12301)

10.1111/prd.12301

---

## The composition and richness of the gut microbiota differentiate the top Polish endurance athletes from sedentary controls.
 13.05.2020, _M Kulecka, B Fraczek, M Mikula, N Zeber-Lubecka, J Karczmarski, A Paziewska, F Ambrozkiewicz, K Jagusztyn-Krynicka, P Cieszczyk, J Ostrowski_


Little data are available on the subject of gut microbiota composition in endurance athletes as well as connections between diet and specific bacteria abundance. However, most studies suggest that athletes' microbiota undergoes major alterations, which may contribute to increased physical performance. Therefore, we decided to investigate differences in gut microbiota between healthy controls and endurance athletes.

[Source](https://doi.org/10.1186/s12970-020-00353-w)

10.1080/19490976.2020.1758009

---

## The athletic gut microbiota

[Source](https://doi.org/10.1186/s12970-020-00353-w)

---

## Colorectal cancer occurrence and treatment based on changes in intestinal flora.
 14.05.2020, _H Si, Q Yang, H Hu, C Ding, H Wang, X Lin_


Colorectal cancer (CRC) is the most common cancer in the world, and its incidence has been increasing in recent years. The occurrence of CRC is believed to be related to a variety of factors. Epidemiological data indicate that CRC is mainly affected by environmental factors, eating habits, physical activity and genetic factors. As a newly recognized functional component, the intestinal microbiota plays important roles in preventing CRC formation and maintaining intestinal immunity. In this review, we summarize the mechanisms by which the gut microbiota causes CRC through alterations to immune function, focusing on the mechanisms by which intestinal microbial dysfunction promotes CRC. Furthermore, we describe the changes in the intestinal flora observed in CRC and their potential for CRC treatment with the goal of facilitating future research on the roles of the intestinal flora.

[Source](https://doi.org/10.1016/j.semcancer.2020.05.004)

10.1016/j.semcancer.2020.05.004

---

## Microbial communities modulating brain functioning and behaviors in zebrafish: A mechanistic approach

[Source](https://doi.org/10.1016/j.micpath.2020.104251)

---

## Multi-omics reveals that the rumen microbiome and its metabolome together with the host metabolome contribute to individualized dairy cow performance.
 13.05.2020, _MY Xue, HZ Sun, XH Wu, JX Liu, LL Guan_


Recently, we reported that some dairy cows could produce high amounts of milk with high amounts of protein (defined as milk protein yield [MPY]) when a population was raised under the same nutritional and management condition, a potential new trait that can be used to increase high-quality milk production. It is unknown to what extent the rumen microbiome and its metabolites, as well as the host metabolism, contribute to MPY. Here, analysis of rumen metagenomics and metabolomics, together with serum metabolomics was performed to identify potential regulatory mechanisms of MPY at both the rumen microbiome and host levels.

[Source](https://doi.org/10.1186/s40168-020-00819-8)

10.1186/s40168-020-00819-8

---

## Rumen protozoa shape microbiome composition and metabolic output of fermentation

[Source](https://doi.org/10.1101/2020.05.15.080218)

---

## Changes in the gut microbiota during Asian particolored bat (<em>Vespertilio sinensis</em>) development

[Source](https://doi.org/10.7717/peerj.9003)

---

## After the bite: bacterial transmission from grey seals (<em>Halichoerus grypus</em>) to harbour porpoises (<em>Phocoena phocoena</em>)

[Source](http://dx.doi.org/10.1098/rsos.192079)

---

## Exploration of the virome of the European brown shrimp (<em>Crangon crangon</em>)

[Source](https://doi.org/10.1099/jgv.0.001412)

---

## The Role of Rhizosphere Bacteriophages in Plant Health.
 17.05.2020, _AA Pratama, J Terpstra, ALM de Oliveria, JF Salles_


Microbiomes and their hosts influence each other; for instance, the microbiome improves host fitness, whereas the host supports microbiome nutrition. Most studies on this topic have focused on the role of bacteria and fungi, although research on viruses that infect bacteria, known as 'bacteriophages' (phages), has gained importance due to the potential role bacteriophages play in the resilience and functionality of the gut microbiome. Like the gut microbiome, the rhizosphere harbors a complex microbiome, but little is known about the role of phages in this ecosystem. In this opinion, we extend the knowledge gained in human gut virus metagenomics (viromics) to disentangle the potential role of phages in driving the resilience and functionality of the rhizosphere microbiome. We propose that future comparative studies across environments are necessary to unravel the underlying mechanisms through which phages drive the composition and functionality of the rhizosphere microbiome and its interaction with the plant host. Importantly, such understanding might generate strategies to improve plant resistance and resilience in the context of climate change.

[Source](https://doi.org/10.1016/j.tim.2020.04.005)

10.1016/j.tim.2020.04.005

---

## Analyzing bacterial community in pit mud of Yibin Baijiu in China using high throughput sequencing

[Source](https://doi.org/10.7717/peerj.9122)

---

## Shifts of bacterial community structure and function in long-term soybean monoculture

[Source](https://doi.org/10.1080/03650340.2020.1759797)

---

## Impact of Land-use Change on Vertical Soil Bacterial Communities in Sabah.
 11.01.2019, _HS Tin, K Palaniveloo, J Anilik, M Vickneswaran, Y Tashiro, CS Vairappan, K Sakai_


_Agriculture, Bacteria, Biodiversity, Ecosystem, Forests, Malaysia, Phylogeny, Soil, Soil Microbiology_
Decline in forest productivity due to forest conversion is defining the Bornean landscape. Responses of bacterial communities due to land-use changes are vital and could define our understanding of ecosystem functions. This study reports the changes in bacterial community structure in organic soil (0-5 cm; O-Horizon) and organic-mineral soil (5-15 cm; A-Horizon) across Maliau Basin Conservation Area old growth forest (MBOG), Fragment E logged forest (FELF) located in Kalabakan Forest Reserve to Benta Wawasan oil palm plantation (BWOP) using two-step PCR amplicon analysis of bacteria DNA on Illumina Miseq next generation sequencing. A total of 30 soil samples yielded 893,752-OTU reads at ≥97% similarity from 5,446,512 good quality sequences. Soil from BWOP plantation showed highest unshared OTUs for organic (49.2%) and organic-mineral (50.9%) soil. MBOG soil showed a drop in unshared OTUs between organic (48.6%) and organic-mineral (33.9%). At phylum level, Proteobacteria dominated MBOG but shifted to Actinobacteria in logged and plantation soil. Present findings also indicated that only FELF exhibited change in bacterial communities along the soil depth, moving from the organic to the organic-mineral layer. Both layers of BWOP plantation soils deviated from other forests' soil in β-diversity analysis. To our knowledge, this is the first report on transitions of bacterial community structures with different soil horizons in the tropical rainforest including Borneo, Sabah. Borneo tropical soils form a large reservoir for soil bacteria and future exploration is needed for fully understanding the diversity structure and their bacterial functional properties.

[Source](https://doi.org/10.1134/S1064229320040080)

10.1007/s00248-017-1043-6

---

## Bacterial Diversity in Peat Soils of Forest Ecosystems and Oil Palm Plantation

[Source](https://doi.org/10.1134/S1064229320040080)

---

## Space Is More Important than Season when Shaping Soil Microbial Communities at a Large Spatial Scale

[Source](https://msystems.asm.org/content/5/3/e00783-19)

---

## Effects of cadmium perturbation on the microbial community structure and heavy metal resistome of a tropical agricultural soil

[Source](https://doi.org/10.1186/s40643-020-00314-w)

---

## Competition for iron drives phytopathogen control by natural rhizosphere microbiomes.
 12.05.2020, _S Gu, Z Wei, Z Shao, VP Friman, K Cao, T Yang, J Kramer, X Wang, M Li, X Mei, Y Xu, Q Shen, R Kümmerli, A Jousset_


Plant pathogenic bacteria cause high crop and economic losses to human societies1-3. Infections by such pathogens are challenging to control as they often arise through complex interactions between plants, pathogens and the plant microbiome4,5. Experimental studies of this natural ecosystem at the microbiome-wide scale are rare, and consequently we have a poor understanding of how the taxonomic and functional microbiome composition and the resulting ecological interactions affect pathogen growth and disease outbreak. Here, we combine DNA-based soil microbiome analysis with in vitro and in planta bioassays to show that competition for iron via secreted siderophore molecules is a good predictor of microbe-pathogen interactions and plant protection. We examined the ability of 2,150 individual bacterial members of 80 rhizosphere microbiomes, covering all major phylogenetic lineages, to suppress the bacterium Ralstonia solanacearum, a global phytopathogen capable of infecting various crops6,7. We found that secreted siderophores altered microbiome-pathogen interactions from complete pathogen suppression to strong facilitation. Rhizosphere microbiome members with growth-inhibitory siderophores could often suppress the pathogen in vitro as well as in natural and greenhouse soils, and protect tomato plants from infection. Conversely, rhizosphere microbiome members with growth-promotive siderophores were often inferior in competition and facilitated plant infection by the pathogen. Because siderophores are a chemically diverse group of molecules, with each siderophore type relying on a compatible receptor for iron uptake8-12, our results suggest that pathogen-suppressive microbiome members produce siderophores that the pathogen cannot use. Our study establishes a causal mechanistic link between microbiome-level competition for iron and plant protection and opens promising avenues to use siderophore-mediated interactions as a tool for microbiome engineering and pathogen control.

[Source](https://doi.org/10.1038/s41564-020-0719-8)

10.1038/s41564-020-0719-8

---

## Tara Oceans: towards global ocean ecosystems biology

[Source](https://doi.org/10.1038/s41579-020-0364-5)

---

## Microbial genomics amidst the Arctic crisis.
 11.05.2020, _A Edwards, KA Cameron, JM Cook, AR Debbonaire, E Furness, MC Hay, SME Rassner_


The Arctic is warming - fast. Microbes in the Arctic play pivotal roles in feedbacks that magnify the impacts of Arctic change. Understanding the genome evolution, diversity and dynamics of Arctic microbes can provide insights relevant for both fundamental microbiology and interdisciplinary Arctic science. Within this synthesis, we highlight four key areas where genomic insights to the microbial dimensions of Arctic change are urgently required: the changing Arctic Ocean, greenhouse gas release from the thawing permafrost, 'biological darkening' of glacial surfaces, and human activities within the Arctic. Furthermore, we identify four principal challenges that provide opportunities for timely innovation in Arctic microbial genomics. These range from insufficient genomic data to develop unifying concepts or model organisms for Arctic microbiology to challenges in gaining authentic insights to the structure and function of low-biomass microbiota and integration of data on the causes and consequences of microbial feedbacks across scales. We contend that our insights to date on the genomics of Arctic microbes are limited in these key areas, and we identify priorities and new ways of working to help ensure microbial genomics is in the vanguard of the scientific response to the Arctic crisis.

[Source](https://doi.org/10.1099/mgen.0.000375)

10.1099/mgen.0.000375

---

## Exceptional but vulnerable microbial diversity in coral reef animal surface microbiomes.
 12.05.2020, _M Chiarello, JC Auguet, NAJ Graham, T Claverie, E Sucré, C Bouvier, F Rieuvilleneuve, CX Restrepo-Ortiz, Y Bettarel, S Villéger, T Bouvier_


Coral reefs host hundreds of thousands of animal species that are increasingly threatened by anthropogenic disturbances. These animals host microbial communities at their surface, playing crucial roles for their fitness. However, the diversity of such microbiomes is mostly described in a few coral species and still poorly defined in other invertebrates and vertebrates. Given the diversity of animal microbiomes, and the diversity of host species inhabiting coral reefs, the contribution of such microbiomes to the total microbial diversity of coral reefs could be important, yet potentially vulnerable to the loss of animal species. Analysis of the surface microbiome from 74 taxa, including teleost fishes, hard and soft corals, crustaceans, echinoderms, bivalves and sponges, revealed that more than 90% of their prokaryotic phylogenetic richness was specific and not recovered in surrounding plankton. Estimate of the total richness associated with coral reef animal surface microbiomes reached up to 2.5% of current estimates of Earth prokaryotic diversity. Therefore, coral reef animal surfaces should be recognized as a hotspot of marine microbial diversity. Loss of the most vulnerable reef animals expected under present-day scenarios of reef degradation would induce an erosion of 28% of the prokaryotic richness, with unknown consequences on coral reef ecosystem functioning.

[Source](https://doi.org/10.1098/rspb.2020.0642)

10.1098/rspb.2020.0642

---

## A powerful microbial group association test based on the higher criticism analysis for sparse microbial association signals.
 12.05.2020, _H Koh, N Zhao_


In human microbiome studies, it is crucial to evaluate the association between microbial group (e.g., community or clade) composition and a host phenotype of interest. In response, a number of microbial group association tests have been proposed, which account for the unique features of the microbiome data (e.g., high-dimensionality, compositionality, phylogenetic relationship). These tests generally fall in the class of aggregation tests which amplify the overall group association by combining all the underlying microbial association signals, and, therefore, they are powerful when many microbial species are associated with a given host phenotype (i.e., low sparsity). However, in practice, the microbial association signals can be highly sparse, and this is especially the situation where we have a difficulty to discover the microbial group association.

[Source](https://doi.org/10.1186/s40168-020-00834-9)

10.1186/s40168-020-00834-9

---

## A practical guide to amplicon and metagenomic analysis of microbiome data.
 12.05.2020, _YX Liu, Y Qin, T Chen, M Lu, X Qian, X Guo, Y Bai_


Advances in high-throughput sequencing (HTS) have fostered rapid developments in the field of microbiome research, and massive microbiome datasets are now being generated. However, the diversity of software tools and the complexity of analysis pipelines make it difficult to access this field. Here, we systematically summarize the advantages and limitations of microbiome methods. Then, we recommend specific pipelines for amplicon and metagenomic analyses, and describe commonly-used software and databases, to help researchers select the appropriate tools. Furthermore, we introduce statistical and visualization methods suitable for microbiome analysis, including alpha- and beta-diversity, taxonomic composition, difference comparisons, correlation, networks, machine learning, evolution, source tracing, and common visualization styles to help researchers make informed choices. Finally, a step-by-step reproducible analysis guide is introduced. We hope this review will allow researchers to carry out data analysis more effectively and to quickly select the appropriate tools in order to efficiently mine the biological significance behind the data.

[Source](https://doi.org/10.1007/s13238-020-00724-8)

10.1007/s13238-020-00724-8

---

## TAMA: improved metagenomic sequence classification through meta-analysis.
 13.05.2020, _M Sim, J Lee, D Lee, D Kwon, J Kim_


Microorganisms are important occupants of many different environments. Identifying the composition of microbes and estimating their abundance promote understanding of interactions of microbes in environmental samples. To understand their environments more deeply, the composition of microorganisms in environmental samples has been studied using metagenomes, which are the collections of genomes of the microorganisms. Although many tools have been developed for taxonomy analysis based on different algorithms, variability of analysis outputs of existing tools from the same input metagenome datasets is the main obstacle for many researchers in this field.

[Source](https://doi.org/10.1186/s12859-020-3533-7)

10.1186/s12859-020-3533-7

---

## nf-rnaSeqMetagen: A nextflow metagenomics pipeline for identifying and characterizing microbial sequences from RNA-seq data

[Source](https://doi.org/10.1016/j.medmic.2020.100011)

---

## FAIR Data Sharing: The Roles of Common Data Elements and Harmonization.
 14.05.2020, _RD Kush, D Warzel, MA Kush, A Sherman, EA Navarro, R Fitzmartin, F Pétavy, J Galvez, LB Becnel, FL Zhou, N Harmon, B Jauregui, T Jackson, L Hudson_


The value of robust and responsible data sharing in clinical research and healthcare is recognized by patients, patient advocacy groups, researchers, journal editors, and the healthcare industry globally. Privacy and security concerns acknowledged, the act of exchanging data (interoperability) along with its meaning (semantic interoperability) across studies and between partners has been difficult, if not elusive. For shared data to retain its value, a recommendation has been made to follow the Findable, Accessible, Interoperable, Reusable (FAIR) principles. Without applying appropriate data exchange standards with domain-relevant content standards and accessible rich metadata that uses applicable terminologies, interoperability is burdened by the need for transformation and/or mapping. These obstacles to interoperability limit the findability, accessibility and reusability of data, thus diminishing its value and making it impossible to adhere to FAIR principles. One effort to standardize data collection has been through common data elements (CDEs). CDEs are data collection units comprising one or more questions together with a set of valid values. Some CDEs contain standardized terminology concepts that define the meaning of the data, and others include links to unique terminology concept identifiers and unique identifiers for each CDE; however, usually CDEs are defined for specific projects or collaborations and lack traceable or machine readable semantics. While the name implies that these are 'common', this has not necessarily been a requirement, and many CDEs have not been commonly used. The National Institutes of Health (NIH) CDEs are, in fact, a conglomerate of CDEs developed in silos by various NIH institutes. Therefore, CDEs have not brought the anticipated benefit to the industry through widescale interoperability, nor is there widespread reuse of CDEs. Certain institutes in the NIH recommend, albeit do not enforce, institute-specific preferred CDEs; however, at the NIH level a preponderance of choice and a lack of any overarching harmonization of CDEs or consistency in linking them to controlled terminology or common identifiers create confusion for researchers in their efforts to identify the best CDEs for their protocol. The problem of comparing data among studies is exacerbated when researchers select different CDEs for the same variable or data collection field. This manuscript explores reasons for the disappointingly low adoption of CDEs and the inability of CDEs or other clinical research standards to broadly solve the interoperability and data sharing problems. Recommendations are offered for rectifying this situation to enable responsible data sharing that will help in adherence to FAIR principles and the realization of Learning Health Systems for the sake of all of us as patients.

[Source](https://doi.org/10.1016/j.jbi.2020.103421)

10.1016/j.jbi.2020.103421

---

