## The oral microbiome and lung cancer risk: An analysis of 3 prospective cohort studies.
 05.08.2022, _E Vogtmann, X Hua, G Yu, V Purandare, AG Hullings, D Shao, Y Wan, S Li, CL Dagnall, K Jones, BD Hicks, A Hutchinson, JG Caporaso, W Wheeler, DP Sandler, LEB Freeman, LM Liao, WY Huang, ND Freedman, NE Caporaso, R Sinha, MH Gail, J Shi, CC Abnet_


Previous studies suggested associations between the oral microbiome and lung cancer, but studies were predominantly cross-sectional and underpowered.

[Source](https://doi.org/10.1093/jnci/djac149)

10.1093/jnci/djac149

---

## Antibiotic resistance genes in the gut microbiota of mothers and linked neonates with or without sepsis from low- and middle-income countries

[Source](https://doi.org/10.1038/s41564-022-01184-y)

---

## Crosstalk between mucosal microbiota, host gene expression, and sociomedical factors in the progression of colorectal cancer.
 04.08.2022, _N Kim, JA Gim, BJ Lee, BI Choi, HS Yoon, SH Kim, MK Joo, JJ Park, C Kim_


Various omics-based biomarkers related to the occurrence, progression, and prognosis of colorectal cancer (CRC) have been identified. In this study, we attempted to identify gut microbiome-based biomarkers and detect their association with host gene expression in the initiation and progression of CRC by integrating analysis of the gut mucosal metagenome, RNA sequencing, and sociomedical factors. We performed metagenome and RNA sequencing on colonic mucosa samples from 13 patients with advanced CRC (ACRC), 10 patients with high-risk adenoma (HRA), and 7 normal control (NC) individuals. All participants completed a questionnaire on sociomedical factors. The interaction and correlation between changes in the microbiome and gene expression were assessed using bioinformatic analysis. When comparing HRA and NC samples, which can be considered to represent the process of tumor initiation, 28 genes and five microbiome species were analyzed with correlation plots. When comparing ACRC and HRA samples, which can be considered to represent the progression of CRC, seven bacterial species and 21 genes were analyzed. When comparing ACRC and NC samples, 16 genes and five bacterial species were analyzed, and four correlation plots were generated. A network visualizing the relationship between bacterial and host gene expression in the initiation and progression of CRC indicated that Clostridium spiroforme and Tyzzerella nexilis were hub bacteria in the development and progression of CRC. Our study revealed the interactions of and correlation between the colonic mucosal microbiome and host gene expression to identify potential roles of the microbiome in the initiation and progression of CRC. Our results provide gut microbiome-based biomarkers that may be potential diagnostic markers and therapeutic targets in patients with CRC.

[Source](https://doi.org/10.1038/s41598-022-17823-7)

10.1038/s41598-022-17823-7

---

## Using Culture-Enriched Phenotypic Metagenomics for Targeted High-Throughput Monitoring of the Clinically Important Fraction of the β-Lactam Resistome.
 05.08.2022, _Z Zhang, G Zhang, F Ju_


High bacterial community diversity and complexity greatly challenge the cost-efficient monitoring of clinically prevalent antibiotic-resistant bacteria, which are usually present as rare and important populations involved in the environmental dissemination of clinical resistance. Here, we introduce culture-enriched phenotypic metagenomics that integrates culture enrichment, phenotypic screening, and metagenomic analyses as an emerging standardized methodology for targeted resistome monitoring and apply it to decipher the extended-spectrum β-lactam resistome in a municipal wastewater treatment plant (WWTP) and its receiving river. The results showed that clinically prevalent carbapenemase genes (e.g., the NDM and KPC families) and extended-spectrum β-lactamase genes (e.g., the CTX-M, TEM, and OXA families) were prevalent in the WWTP and showed prominent potential in horizontal dissemination. Strikingly, carbapenem and polymyxin resistance genes co-occurred in the highly virulent nosocomial pathogens Enterobacter kobei and Citrobacter freundii. Overall, this study exemplifies phenotypic metagenomics for high-throughput surveillance of a targeted clinically important fraction of antibiotic resistomes and substantially expands current knowledge on extended-spectrum β-lactam resistance in WWTPs.

[Source](https://doi.org/10.1021/acs.est.2c03627)

10.1021/acs.est.2c03627

---

## <em>Porphyromonas gingivalis</em> and dental stem cells crosstalk amplify inflammation and bone loss in the periodontitis niche

[Source](https://doi.org/10.1002/jcp.30848)

---

## On the effect of flavonoids and dietary fibre in lingonberries on atherosclerotic plaques, lipid profiles and gut microbiota composition in <em>Apoe(-/-)</em> mice

[Source](https://doi.org/10.1080/09637486.2022.2106358)

---

## The Gut-Brain-Microbiome Connection: Can Probiotics Decrease Anxiety and Depression?

[Source](https://doi.org/10.1080/01612840.2022.2106525)

---

## The effect of <em>Lactobacillus rhamnosus</em> B10 on alcoholic liver injury and intestinal microbiota in alcohol-induced mice model

[Source](https://doi.org/10.1111/jfbc.14372)

---

## The effect of young blood plasma administration on gut microbiota in middle-aged rats

[Source](https://doi.org/10.1007/s00203-022-03154-8)

---

## Quantitative microbial population study reveals geographical differences in bacterial symbionts of <em>Ixodes ricinus</em>

[Source](https://doi.org/10.1186/s40168-022-01276-1)

---

## Host taxonomy determines the composition, structure, and diversity of the earthworm cast microbiome under homogenous feeding conditions

[Source](https://doi.org/10.1093/femsec/fiac093)

---

## Ruminal Microbiota Determines the High-Fiber Utilization of Ruminants: Evidence from the Ruminal Microbiota Transplant

[Source](https://doi.org/10.1128/spectrum.00446-22)

---

## Computational approach to modeling microbiome landscapes associated with chronic human disease progression

[Source](https://doi.org/10.1371/journal.pcbi.1010373)

---

## Guideline for the analysis of the microbial communities of the human upper airways

[Source](https://doi.org/10.1080/20002297.2022.2103282)

---

