## Exploring the Influence of Gut Microbiome on Energy Metabolism in Humans.
 09.04.2023, _J Montenegro, AM Armet, B Willing, EC Deehan, PG Fassini, JF Mota, J Walter, CM Prado_


The gut microbiome has a profound influence on host physiology, including energy metabolism, which is the process by which energy from nutrients is transformed to other forms of energy to be used by the body. However, mechanistic evidence for how the microbiome influences energy metabolism is derived from animal models. In this narrative review, we included human studies investigating the relation between gut microbiome and energy metabolism - i.e., energy expenditure in humans and energy harvest by the gut microbiome. Studies have found no consistent gut microbiome patterns associated with energy metabolism, and most interventions were not effective in modulating the gut microbiome to influence energy metabolism. To date, cause-and-effect relationships, and mechanistic evidence on the impact of the gut microbiome on energy expenditure have not been established in humans. Future longitudinal observational studies and randomized controlled trials utilizing robust methodologies and advanced statistical analysis are needed. Such knowledge would potentially inform the design of therapeutic avenues and specific dietary recommendations to improve energy metabolism through gut microbiome modulation.

[Source](https://doi.org/10.1016/j.advnut.2023.03.015)

10.1016/j.advnut.2023.03.015

---

## Ruminococcus gnavus: friend or foe for human health.
 04.04.2023, _EH Crost, E Coletto, A Bell, N Juge_


Ruminococcus gnavus was first identified in 1974 as a strict anaerobe in the gut of healthy individuals, and for several decades, its study has been limited to specific enzymes or bacteriocins. With the advent of metagenomics, R. gnavus has been associated both positively and negatively with an increasing number of intestinal and extra-intestinal diseases from inflammatory bowel diseases to neurological disorders. This prompted renewed interest in understanding the adaptation mechanisms of R. gnavus to the gut, and the molecular mediators affecting its association with health and disease. From ca. 250 publications citing R. gnavus since 1990, 94% were published in the last 10 years. In this review, we describe the biological characterisation of R. gnavus, its occurrence in the infant and adult gut microbiota and the factors influencing its colonisation of the gastrointestinal tract; we also discuss the current state of our knowledge on its role in host health and disease. We highlight gaps in knowledge and discuss the hypothesis that differential health outcomes associated with R. gnavus in the gut are strain and niche specific.

[Source](https://doi.org/10.1093/femsre/fuad014)

10.1093/femsre/fuad014

---

## <em>Ruminococcus gnavus</em>: friend or foe for human health

[Source](https://doi.org/10.1093/femsre/fuad014)

---

## Uncovering the special microbiota associated with occurrence and progression of gastric cancer by using RNA-sequencing.
 12.04.2023, _B Ai, Y Mei, D Liang, T Wang, H Cai, D Yu_


_Humans, Stomach Neoplasms, Gastrointestinal Microbiome, Microbiota, Carcinogenesis, RNA, Ribosomal, 16S_
Gastric cancer (GC) has been identified as the third deadly cancer in the world. Accumulating researches suggest a potential role of microorganisms in tumorigenesis. However, the composition of microbiota in GC tissues is not clear and it changes throughout the different stages of GC remain mostly elusive. Our study integrated RNA-Seq data of 727 samples derived from gastric tissues across four datasets and revealed its microbial composition. In order to remove the false positive results, core taxa were defined and characterized. Based on it, we analyzed the influence of biological factors on its composition. The pan-microbiome of gastric tissues was estimated to be over than 1400 genera. Seventeen core genera were identified. Among them, Helicobacter, Lysobacter were significantly enriched in normal tissues, while Pseudomonas was enriched in tumor tissues. Interestingly, Acinetobacter, Pasteurella, Streptomyces, Chlamydia, and Lysobacter, showed a significant increase trend during tumor development and formed strong intra/inter-correlations among them or with other genera. Furthermore, we found that tumor stage played an important role in altering the microbial composition of GC tissues. This study provides support for the in-depth study of tumor microbiome, and the specific microbiome excavated provides a possibility for the subsequent identification of potential biomarkers for GC.

[Source](https://doi.org/10.1038/s41598-023-32809-9)

10.1038/s41598-023-32809-9

---

## Intermittent fasting modulates the intestinal microbiota and improves obesity and host energy metabolism.
 12.04.2023, _X Hu, K Xia, M Dai, X Han, P Yuan, J Liu, S Liu, F Jia, J Chen, F Jiang, J Yu, H Yang, J Wang, X Xu, X Jin, K Kristiansen, L Xiao, W Chen, M Han, S Duan_


_Humans, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Intermittent Fasting, Obesity, Metagenome_
Intermittent fasting (IF) is a promising paradigm for weight loss which has been shown to modulate the gut microbiota based on 16S rRNA gene amplicon sequencing. Here, 72 Chinese volunteers with a wide range of body mass index (BMI) participated in a three-week IF program during which an average loss of 3.67 kg body weight accompanied with improved clinical parameters was observed irrespective of initial anthropometric and gut microbiota status. Fecal samples were collected before and after the intervention and subjected to shotgun metagenomic sequencing. De novo assembly yielded 2934 metagenome-assembled genomes (MAGs). Profiling revealed significant enrichment of Parabacteroides distasonis and Bacteroides thetaiotaomicron after the intervention, with inverse correlations between their relative abundances and parameters related to obesity and atherosclerotic cardiovascular diseases (ASCVD). MAGs enriched after the intervention showed high richness and diversity of carbohydrate-active enzymes, with an increased relative abundances of genes related to succinate production and glutamate fermentation.

[Source](https://doi.org/10.1038/s41522-023-00386-4)

10.1038/s41522-023-00386-4

---

## The secondary bile acid isoursodeoxycholate correlates with post-prandial lipemia, inflammation, and appetite and changes post-bariatric surgery.
 06.04.2023, _P Louca, AS Meijnikman, A Nogal, F Asnicar, I Attaye, A Vijay, A Kouraki, A Visconti, K Wong, SE Berry, ER Leeming, O Mompeo, F Tettamanzi, AF Baleanu, M Falchi, G Hadjigeorgiou, J Wolf, YIZ Acherman, AW Van de Laar, VEA Gerdes, GA Michelotti, PW Franks, N Segata, M Mangino, TD Spector, WJ Bulsiewicz, M Nieuwdorp, AM Valdes, C Menni_


Primary and secondary bile acids (BAs) influence metabolism and inflammation, and the gut microbiome modulates levels of BAs. We systematically explore the host genetic, gut microbial, and habitual dietary contribution to a panel of 19 serum and 15 stool BAs in two population-based cohorts (TwinsUK, n = 2,382; ZOE PREDICT-1, n = 327) and assess changes post-bariatric surgery and after nutritional interventions. We report that BAs have a moderately heritable genetic component, and the gut microbiome accurately predicts their levels in serum and stool. The secondary BA isoursodeoxycholate (isoUDCA) can be explained mostly by gut microbes (area under the receiver operating characteristic curve [AUC] = ∼80%) and associates with post-prandial lipemia and inflammation (GlycA). Furthermore, circulating isoUDCA decreases significantly 1 year after bariatric surgery (β = -0.72, p = 1 × 10-5) and in response to fiber supplementation (β = -0.37, p &lt; 0.03) but not omega-3 supplementation. In healthy individuals, isoUDCA fasting levels correlate with pre-meal appetite (p &lt; 1 × 10-4). Our findings indicate an important role for isoUDCA in lipid metabolism, appetite, and, potentially, cardiometabolic risk.

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00099-X)

10.1016/j.xcrm.2023.100993

---

## Forging the microbiome to help us live long and prosper.
 05.04.2023, _RR Rock, PJ Turnbaugh_


Aging is often accompanied by an increased risk of an array of diseases spanning the cardiovascular, nervous, and immune systems, among others. Despite remarkable progress in understanding the cellular and molecular mechanisms involved in aging, the role of the microbiome remains understudied. In this Essay, we highlight recent progress towards understanding if and how the microbiome contributes to aging and age-associated diseases. Furthermore, we discuss the need to consider sexually dimorphic phenotypes in the context of aging and the microbiome. We also highlight the broad implications for this emerging area of interdisciplinary research to address long-standing questions about host-microbiome interactions across the life span.

[Source](https://doi.org/10.1371/journal.pbio.3002087)

10.1371/journal.pbio.3002087

---

## Meta-analysis reveals <em>Helicobacter pylori</em> mutual exclusivity and reproducible gastric microbiome alterations during gastric carcinoma progression

[Source](https://doi.org/10.1080/19490976.2023.2197835)

---

## <em>Lacticaseibacillus rhamnosus</em> Probio-M9 enhanced the antitumor response to anti-PD-1 therapy by modulating intestinal metabolites

[Source](https://www.thelancet.com/journals/ebiom/article/PIIS2352-3964(23)00098-1/fulltext)

---

## Quantifying Shared and Unique Gene Content across 17 Microbial Ecosystems.
 06.04.2023, _S Zimmerman, BT Tierney, CJ Patel, AD Kostic_


Measuring microbial diversity is traditionally based on microbe taxonomy. Here, in contrast, we aimed to quantify heterogeneity in microbial gene content across 14,183 metagenomic samples spanning 17 ecologies, including 6 human associated, 7 nonhuman host associated, and 4 in other nonhuman host environments. In total, we identified 117,629,181 nonredundant genes. The vast majority of genes (66%) occurred in only one sample (i.e., "singletons"). In contrast, we found 1,864 sequences present in every metagenome, but not necessarily every bacterial genome. Additionally, we report data sets of other ecology-associated genes (e.g., abundant in only gut ecosystems) and simultaneously demonstrated that prior microbiome gene catalogs are both incomplete and inaccurately cluster microbial genetic life (e.g., at gene sequence identities that are too restrictive). We provide our results and the sets of environmentally differentiating genes described above at http://www.microbial-genes.bio. IMPORTANCE The amount of shared genetic elements has not been quantified between the human microbiome and other host- and non-host-associated microbiomes. Here, we made a gene catalog of 17 different microbial ecosystems and compared them. We show that most species shared between environment and human gut microbiomes are pathogens and that prior gene catalogs described as "nearly complete" are far from it. Additionally, over two-thirds of all genes only appear in a single sample, and only 1,864 genes (0.001%) are found in all types of metagenomes. These results highlight the large diversity between metagenomes and reveal a new, rare class of genes, those found in every type of metagenome, but not every microbial genome.

[Source](https://doi.org/10.1128/msystems.00118-23)

10.1128/msystems.00118-23

---

## New carbohydrate binding domains identified by phage display based functional metagenomic screens of human gut microbiota.
 11.04.2023, _A Akhtar, M Lata, S Sunsunwal, A Yadav, K Lnu, S Subramanian, TNC Ramya_


_Humans, Gastrointestinal Microbiome, Metagenomics, Genomics, Carbohydrates, Bacteriophages_
Uncultured microbes represent a huge untapped biological resource of novel genes and gene products. Although recent genomic and metagenomic sequencing efforts have led to the identification of numerous genes that are homologous to existing annotated genes, there remains, yet, an enormous pool of unannotated genes that do not find significant sequence homology to existing annotated genes. Functional metagenomics offers a way to identify and annotate novel gene products. Here, we use functional metagenomics to mine novel carbohydrate binding domains that might aid human gut commensals in adherence, gut colonization, and metabolism of complex carbohydrates. We report the construction and functional screening of a metagenomic phage display library from healthy human fecal samples against dietary, microbial and host polysaccharides/glycoconjugates. We identify several protein sequences that do not find a hit to any known protein domain but are predicted to contain carbohydrate binding module-like folds. We heterologously express, purify and biochemically characterize some of these protein domains and demonstrate their carbohydrate-binding function. Our study reveals several previously unannotated carbohydrate-binding domains, including a levan binding domain and four complex N-glycan binding domains that might be useful for the labeling, visualization, and isolation of these glycans.

[Source](https://doi.org/10.1038/s42003-023-04718-0)

10.1038/s42003-023-04718-0

---

## Genome-resolving metagenomics reveals wild western capercaillies (<em>Tetrao urogallus</em>) as avian hosts for antibiotic-resistance bacteria and their interactions with the gut-virome community

[Source](https://doi.org/10.1016/j.micres.2023.127372)

---

## Cofunctioning of bacterial exometabolites drives root microbiota establishment.
 09.04.2023, _F Getzke, MA Hassani, M Crüsemann, M Malisic, P Zhang, Y Ishigaki, N Böhringer, A Jiménez Fernández, L Wang, J Ordon, KW Ma, T Thiergart, CJ Harbort, H Wesseler, S Miyauchi, R Garrido-Oter, K Shirasu, TF Schäberle, S Hacquard, P Schulze-Lefert_


_Bacteria, Microbiota, Symbiosis, Arabidopsis, Microbial Interactions, Plant Roots, Soil Microbiology_
Soil-dwelling microbes are the principal inoculum for the root microbiota, but our understanding of microbe-microbe interactions in microbiota establishment remains fragmentary. We tested 39,204 binary interbacterial interactions for inhibitory activities in vitro, allowing us to identify taxonomic signatures in bacterial inhibition profiles. Using genetic and metabolomic approaches, we identified the antimicrobial 2,4-diacetylphloroglucinol (DAPG) and the iron chelator pyoverdine as exometabolites whose combined functions explain most of the inhibitory activity of the strongly antagonistic Pseudomonas brassicacearum R401. Microbiota reconstitution with a core of Arabidopsis thaliana root commensals in the presence of wild-type or mutant strains revealed a root niche-specific cofunction of these exometabolites as root competence determinants and drivers of predictable changes in the root-associated community. In natural environments, both the corresponding biosynthetic operons are enriched in roots, a pattern likely linked to their role as iron sinks, indicating that these cofunctioning exometabolites are adaptive traits contributing to pseudomonad pervasiveness throughout the root microbiota.

[Source](https://doi.org/10.1073/pnas.2221508120)

10.1073/pnas.2221508120

---

## Unveiling the role of emerging metagenomics for the examination of hypersaline environments.
 05.04.2023, _K Dindhoria, V Manyapu, A Ali, R Kumar_


Hypersaline ecosystems are distributed all over the globe. They are subjected to poly-extreme stresses and are inhabited by halophilic microorganisms possessing multiple adaptations. The halophiles have many biotechnological applications such as nutrient supplements, antioxidant synthesis, salt tolerant enzyme production, osmolyte synthesis, biofuel production, electricity generation etc. However, halophiles are still underexplored in terms of complex ecological interactions and functions as compared to other niches. The advent of metagenomics and the recent advancement of next-generation sequencing tools have made it feasible to investigate the microflora of an ecosystem, its interactions and functions. Both target gene and shotgun metagenomic approaches are commonly employed for the taxonomic, phylogenetic, and functional analyses of the hypersaline microbial communities. This review discusses different types of hypersaline niches, their residential microflora, and an overview of the metagenomic approaches used to investigate them. Various applications, hurdles and the recent advancements in metagenomic approaches have also been focused on here for their better understanding and utilization in the study of hypersaline microbiome.

[Source](https://doi.org/10.1080/02648725.2023.2197717)

10.1080/02648725.2023.2197717

---

## Multi-scale Adaptive Differential Abundance Analysis in Microbial Compositional Data.
 05.04.2023, _S Wang_


Differential abundance analysis is an essential and commonly used tool to characterize the difference between microbial communities. However, identifying differentially abundant microbes remains a challenging problem because the observed microbiome data is inherently compositional, excessive sparse, and distorted by experimental bias. Besides these major challenges, the results of differential abundance analysis also depend largely on the choice of analysis unit, adding another practical complexity to this already complicated problem.

[Source](https://doi.org/10.1093/bioinformatics/btad178)

10.1093/bioinformatics/btad178

---

## The selection of software and database for metagenomics sequence analysis impacts the outcome of microbial profiling and pathogen detection.
 12.04.2023, _R Xu, S Rajeev, LCM Salvador_


_Animals, Mice, Metagenomics, Software, Microbiota, Algorithms, Metagenome, Sequence Analysis, DNA_
Shotgun metagenomic sequencing analysis is widely used for microbial profiling of biological specimens and pathogen detection. However, very little is known about the technical biases caused by the choice of analysis software and databases on the biological specimen. In this study, we evaluated different direct read shotgun metagenomics taxonomic profiling software to characterize the microbial compositions of simulated mice gut microbiome samples and of biological samples collected from wild rodents across multiple taxonomic levels. Using ten of the most widely used metagenomics software and four different databases, we demonstrated that obtaining an accurate species-level microbial profile using the current direct read metagenomics profiling software is still a challenging task. We also showed that the discrepancies in results when different databases and software were used could lead to significant variations in the distinct microbial taxa classified, in the characterizations of the microbial communities, and in the differentially abundant taxa identified. Differences in database contents and read profiling algorithms are the main contributors for these discrepancies. The inclusion of host genomes and of genomes of the interested taxa in the databases is important for increasing the accuracy of profiling. Our analysis also showed that software included in this study differed in their ability to detect the presence of Leptospira, a major zoonotic pathogen of one health importance, especially at the species level resolution. We concluded that using different databases and software combinations can result in confounding biological conclusions in microbial profiling. Our study warrants that software and database selection must be based on the purpose of the study.

[Source](https://doi.org/10.1371/journal.pone.0284031)

10.1371/journal.pone.0284031

---

## An <em>in vitro </em>platform for study of the human gut microbiome under an oxygen gradient

[Source](https://doi.org/10.1007/s10544-023-00653-3)

---

## Machine learning model to predict obesity using gut metabolite and brain microstructure data.
 11.04.2023, _V Osadchiy, R Bal, EA Mayer, R Kunapuli, T Dong, P Vora, D Petrasek, C Liu, J Stains, A Gupta_


_Humans, Overweight, Obesity, Brain, Gastrointestinal Microbiome, Metabolomics, Feces_
A growing body of preclinical and clinical literature suggests that brain-gut-microbiota interactions may contribute to obesity pathogenesis. In this study, we use a machine learning approach to leverage the enormous amount of microstructural neuroimaging and fecal metabolomic data to better understand key drivers of the obese compared to overweight phenotype. Our findings reveal that although gut-derived factors play a role in this distinction, it is primarily brain-directed changes that differentiate obese from overweight individuals. Of the key gut metabolites that emerged from our model, many are likely at least in part derived or influenced by the gut-microbiota, including some amino-acid derivatives. Remarkably, key regions outside of the central nervous system extended reward network emerged as important differentiators, suggesting a role for previously unexplored neural pathways in the pathogenesis of obesity.

[Source](https://doi.org/10.1038/s41598-023-32713-2)

10.1038/s41598-023-32713-2

---

