## @PeterFabianLab

[Source](https://twitter.com/PeterFabianLab)

---

## Dynamization-Posterior Lumbar Interbody Fusion for Hemodialysis-Related Spondyloarthropathy: Evaluation of the Radiographic Outcomes and Reoperation Rate within 2 Years Postoperatively.
 07.03.2022, _T Yasukawa, J Ohya, N Kawamura, Y Yoshida, Y Onishi, K Kohata, Y Kakuta, S Nagatani, Y Kudo, T Shirahata, J Kunogi_


Clinical case series.

[Source](https://twitter.com/PeterFabianLab/status/1499818919560769539)

10.31616/asj.2021.0312

---

## March 4, 2022

[Source](https://twitter.com/PeterFabianLab/status/1499818919560769539)

---

## Microbial invasion of a toxic medium is facilitated by a resident community but inhibited as the community co-evolves

[Source](https://www.biorxiv.org/content/10.1101/2022.03.03.482806v1)

---

## @PhilippePiccar1

[Source](https://twitter.com/PhilippePiccar1?ref_src=twsrc%5Etfw)

---

## https://t.co/rUKz4ApqoP

[Source](https://t.co/rUKz4ApqoP)

---

## March 4, 2022

[Source](https://twitter.com/saramitri/status/1499672946046083075?ref_src=twsrc%5Etfw)

---

## Could a Focus on the “Why” of Taxonomy Help Taxonomy Better Respond to the Needs of Science and Society?

[Source](https://www.preprints.org/manuscript/202203.0069/v1)

---

## Cysteine dependence of<em> Lactobacillus iners</em> is a potential therapeutic target for vaginal microbiota modulation 

[Source](https://www.nature.com/articles/s41564-022-01070-7)

---

## Citizen-science map of the vaginal microbiome

[Source](https://www.researchsquare.com/article/rs-1350465/v1)

---

## Insight into the ecology of vaginal bacteria through integrative analyses of metagenomic and metatranscriptomic data

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02635-9)

---

## #microbiome

[Source](https://twitter.com/hashtag/microbiome?src=hash&ref_src=twsrc%5Etfw)

---

## #TheMarieProject

[Source](https://twitter.com/hashtag/TheMarieProject?src=hash&ref_src=twsrc%5Etfw)

---

## #awareness

[Source](https://twitter.com/hashtag/awareness?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/cjxTHtZggQ

[Source](https://t.co/cjxTHtZggQ)

---

## #WomensHealth

[Source](https://twitter.com/hashtag/WomensHealth?src=hash&ref_src=twsrc%5Etfw)

---

## pic.twitter.com/B9MLNWF6Ex

[Source](https://t.co/B9MLNWF6Ex)

---

## March 4, 2022

[Source](https://twitter.com/themarieprojec1/status/1499637011916886016?ref_src=twsrc%5Etfw)

---

## The gut, vaginal, and urine microbiome in overactive bladder: a systematic review.
 03.03.2022, _C Sze, M Pressler, JR Lee, B Chughtai_


The objective was to systemically review the current literature on the association of gut, vaginal, and urinary dysbiosis in female patients with overactive bladder (OAB).

[Source](https://link.springer.com/article/10.1007/s00192-022-05127-3)

10.1007/s00192-022-05127-3

---

## Identification of antimicrobial peptides from the human gut microbiome using deep learning.
 04.03.2022, _Y Ma, Z Guo, B Xia, Y Zhang, X Liu, Y Yu, N Tang, X Tong, M Wang, X Ye, J Feng, Y Chen, J Wang_


The human gut microbiome encodes a large variety of antimicrobial peptides (AMPs), but the short lengths of AMPs pose a challenge for computational prediction. Here we combined multiple natural language processing neural network models, including LSTM, Attention and BERT, to form a unified pipeline for candidate AMP identification from human gut microbiome data. Of 2,349 sequences identified as candidate AMPs, 216 were chemically synthesized, with 181 showing antimicrobial activity (a positive rate of &gt;83%). Most of these peptides have less than 40% sequence homology to AMPs in the training set. Further characterization of the 11 most potent AMPs showed high efficacy against antibiotic-resistant, Gram-negative pathogens and demonstrated significant efficacy in lowering bacterial load by more than tenfold against a mouse model of bacterial lung infection. Our study showcases the potential of machine learning approaches for mining functional peptides from metagenome data and accelerating the discovery of promising AMP candidate molecules for in-depth investigations.

[Source](https://www.nature.com/articles/s41587-022-01226-0)

10.1038/s41587-022-01226-0

---

## Microbiome risk profiles as biomarkers for inflammatory and metabolic disorders.
 22.02.2022, _A Metwaly, S Reitmeier, D Haller_


The intestine harbours a complex array of microorganisms collectively known as the gut microbiota. The past two decades have witnessed increasing interest in studying the gut microbiota in health and disease, largely driven by rapid innovation in high-throughput multi-omics technologies. As a result, microbial dysbiosis has been linked to many human pathologies, including type 2 diabetes mellitus and inflammatory bowel disease. Integrated analyses of multi-omics data, including metagenomics and metabolomics along with measurements of host response and cataloguing of bacterial isolates, have identified many bacteria and bacterial products that are correlated with disease. Nevertheless, insight into the mechanisms through which microbes affect intestinal health requires going beyond correlation to causation. Current understanding of the contribution of the gut microbiota to disease causality remains limited, largely owing to the heterogeneity of microbial community structures, interindividual differences in disease evolution and incomplete understanding of the mechanisms that integrate microbiota-derived signals into host signalling pathways. In this Review, we provide a broad insight into the microbiome signatures linked to inflammatory and metabolic disorders, discuss outstanding challenges in this field and propose applications of multi-omics technologies that could lead to an improved mechanistic understanding of microorganism-host interactions.

[Source](https://www.nature.com/articles/s41575-022-00581-2)

10.1038/s41575-022-00581-2

---

## Temporal variations shape the gut microbiome ecology of the moth<em> Brithys crini </em>

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1462-2920.15952)

---

## Systematically assessing microbiome-disease associations identifies drivers of inconsistency in metagenomic research.
 05.03.2022, _BT Tierney, Y Tan, Z Yang, B Shui, MJ Walker, BM Kent, AD Kostic, CJ Patel_


Evaluating the relationship between the human gut microbiome and disease requires computing reliable statistical associations. Here, using millions of different association modeling strategies, we evaluated the consistency-or robustness-of microbiome-based disease indicators for 6 prevalent and well-studied phenotypes (across 15 public cohorts and 2,343 individuals). We were able to discriminate between analytically robust versus nonrobust results. In many cases, different models yielded contradictory associations for the same taxon-disease pairing, some showing positive correlations and others negative. When querying a subset of 581 microbe-disease associations that have been previously reported in the literature, 1 out of 3 taxa demonstrated substantial inconsistency in association sign. Notably, &gt;90% of published findings for type 1 diabetes (T1D) and type 2 diabetes (T2D) were particularly nonrobust in this regard. We additionally quantified how potential confounders-sequencing depth, glucose levels, cholesterol, and body mass index, for example-influenced associations, analyzing how these variables affect the ostensible correlation between Faecalibacterium prausnitzii abundance and a healthy gut. Overall, we propose our approach as a method to maximize confidence when prioritizing findings that emerge from microbiome association studies.

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001556)

10.1371/journal.pbio.3001556

---

## @PLOSBiology

[Source](https://twitter.com/PLOSBiology?ref_src=twsrc%5Etfw)

---

## https://t.co/MV5xvVQ2aF

[Source](https://t.co/MV5xvVQ2aF)

---

## March 2, 2022

[Source](https://twitter.com/BradenTierney/status/1499118231944151047?ref_src=twsrc%5Etfw)

---

## MultiBaC: An R package to remove batch effects in multi-omic experiments

[Source](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btac132/6541627)

---

## HiCBin: binning metagenomic contigs and recovering metagenome-assembled genomes using Hi-C contact maps

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02626-w)

---

## #microbiome

[Source](https://twitter.com/hashtag/microbiome)

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF)

---

## https://hopin.com/events/microbiome-vif-n-7

[Source](https://hopin.com/events/microbiome-vif-n-7})

---

## https://www.microbiome-vif.org/program/

[Source](https://www.microbiome-vif.org/program/})

---

## @MicrobiomeVIF

[Source](https://twitter.com/MicrobiomeVIF)

---

## February 17, 2022

[Source](https://twitter.com/MicrobiomeVIF/status/1494340672995966977)

---

