## Dynamic metabolic adaptation can promote species coexistence in competitive communities.
 07.05.2020, _L Pacciani-Mori, A Giometto, S Suweis, A Maritan_


Microbes are capable of physiologically adapting to diverse environmental conditions by differentially varying the rates at which they uptake different nutrients. In particular, microbes can switch hierarchically between different energy sources, consuming first those that ensure the highest growth rate. Experimentally, this can result in bi-phasic growth curves called "diauxic shifts" that typically arise when microbes are grown in media containing several nutrients. Despite these observations are well-known in microbiology and molecular biology, the mathematical models generally used to describe the population dynamics of microbial communities do not account for dynamic metabolic adaptation, thus implicitly assuming that microbes cannot switch dynamically from one resource to another. Here, we introduce dynamic metabolic adaptation in the framework of consumer-resource models, which are commonly used to describe competitive microbial communities, allowing each species to temporally change its preferred energy source to maximize its own relative fitness. We show that dynamic metabolic adaptation enables the community to self-organize, allowing several species to coexist even in the presence of few resources, and to respond optimally to a time-dependent environment, thus showing that dynamic metabolic adaptation could be an important mechanism for maintaining high levels of diversity even in environments with few energy sources. We show that introducing dynamic metabolic strategies in consumer-resource models is necessary for reproducing experimental growth curves of the baker's yeast Saccharomyces cerevisiae growing in the presence of two carbon sources. Even though diauxic shifts emerge naturally from the model when two resources are qualitatively very different, the model predicts that the existence of such shifts is not a prerequisite for species coexistence in competitive communities.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007896)

10.1371/journal.pcbi.1007896

---

## Toll-Like Receptor Signaling and Immune Regulatory Lymphocytes in Periodontal Disease.
 13.05.2020, _Y Gu, X Han_


Periodontitis is known to be initiated by periodontal microbiota derived from biofilm formation. The microbial dysbiotic changes in the biofilm trigger the host immune and inflammatory responses that can be both beneficial for the protection of the host from infection, and detrimental to the host, causing tissue destruction. During this process, recognition of Pathogen-Associated Molecular Patterns (PAMPs) by the host Pattern Recognition Receptors (PRRs) such as Toll-like receptors (TLRs) play an essential role in the host-microbe interaction and the subsequent innate as well as adaptive responses. If persistent, the adverse interaction triggered by the host immune response to the microorganisms associated with periodontal biofilms is a direct cause of periodontal inflammation and bone loss. A large number of T and B lymphocytes are infiltrated in the diseased gingival tissues, which can secrete inflammatory mediators and activate the osteolytic pathways, promoting periodontal inflammation and bone resorption. On the other hand, there is evidence showing that immune regulatory T and B cells are present in the diseased tissue and can be induced for the enhancement of their anti-inflammatory effects. Changes and distribution of the T/B lymphocytes phenotype seem to be a key determinant of the periodontal disease outcome, as the functional activities of these cells not only shape up the overall immune response pattern, but may directly regulate the osteoimmunological balance. Therefore, interventional strategies targeting TLR signaling and immune regulatory T/B cells may be a promising approach to rebalance the immune response and alleviate bone loss in periodontal disease. In this review, we will examine the etiological role of TLR signaling and immune cell osteoclastogenic activity in the pathogenesis of periodontitis. More importantly, the protective effects of immune regulatory lymphocytes, particularly the activation and functional role of IL-10 expressing regulatory B cells, will be discussed.

[Source](https://onlinelibrary.wiley.com/doi/abs/10.1111/prd.12296)

10.3390/ijms21093329

---

## The role of the microbiota in periodontal disease

[Source](https://onlinelibrary.wiley.com/doi/abs/10.1111/prd.12296)

---

## Oral-lung microbiome interactions in lung diseases.
 11.05.2020, _MJ Mammen, FA Scannapieco, S Sethi_


The proximity and continuity of the oral cavity and the lower respiratory tract allows the oropharyngeal microbiome to be a major determinant of the lung microbiome. In addition, host-pathogen interactions related to the oropharyngeal microbiome or its metabolites could propagate systemic inflammation or modulate host defense mechanisms that could affect other organs, including the lung. There is increasing appreciation of the pathophysiologic significance of the lung microbiome, not only in the classical infection-related diseases, pneumonia, bronchiectasis, and cystic fibrosis, but also in chronic noninfectious lung diseases, such as chronic obstructive pulmonary disease, asthma, and pulmonary fibrosis. In this review, we will explore the relationship of the oral microbiome with lung diseases, such as pneumonia, chronic obstructive pulmonary disease, asthma, and cystic fibrosis.

[Source](https://onlinelibrary.wiley.com/doi/abs/10.1111/prd.12301)

10.1111/prd.12301

---

## Microbiome and Schizophrenia: Current Evidence and Future Challenges

[Source](https://link.springer.com/article/10.1007/s40473-020-00206-5)

---

## Patients infected with <em>Mycobacterium africanum</em> versus <em>Mycobacterium tuberculosis</em> possess distinct intestinal microbiota

[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0008230)

---

## A systematic machine learning and data type comparison yields metagenomic predictors of infant age, sex, breastfeeding, antibiotic usage, country of origin, and delivery type.
 11.05.2020, _A Le Goallec, BT Tierney, JM Luber, EM Cofer, AD Kostic, CJ Patel_


The microbiome is a new frontier for building predictors of human phenotypes. However, machine learning in the microbiome is fraught with issues of reproducibility, driven in large part by the wide range of analytic models and metagenomic data types available. We aimed to build robust metagenomic predictors of host phenotype by comparing prediction performances and biological interpretation across 8 machine learning methods and 4 different types of metagenomic data. Using 1,570 samples from 300 infants, we fit 7,865 models for 6 host phenotypes. We demonstrate the dependence of accuracy on algorithm choice and feature definition in microbiome data and propose a framework for building microbiome-derived indicators of host phenotype. We additionally identify biological features predictive of age, sex, breastfeeding status, historical antibiotic usage, country of origin, and delivery type. Our complete results can be viewed at http://apps.chiragjpgroup.org/ubiome_predictions/.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007895)

10.1371/journal.pcbi.1007895

---

## The role of small intestinal bacterial overgrowth and false positive diagnosis of lactose intolerance in southwest Hungary-A retrospective observational study.
 14.05.2020, _P Varjú, B Ystad, N Gede, P Hegyi, D Pécsi, J Czimmer_


Lactose intolerance is a frequent gastrointestinal disease affecting 47% of the Eastern European population. Small intestinal bacterial overgrowth (SIBO) leads to carbohydrate malabsorption and therefore to false results during lactose breath and tolerance tests.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0230784)

10.1371/journal.pone.0230784

---

## The role of small intestinal bacterial overgrowth and false positive diagnosis of lactose intolerance in southwest Hungary 

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0230784)

---

## Retention of Primary Bile Acids by Lupin Cell Wall Polysaccharides Under In Vitro Digestion Conditions.
 24.02.2020, _S Naumann, U Schweiggert-Weisz, D Haller, P Eisner_


_Adsorption, Bile Acids and Salts, Cell Wall, Cotyledon, Dietary Fiber, Digestion, Elasticity, Hydrophobic and Hydrophilic Interactions, Lupinus, Polysaccharides, Viscosity_
Interference of dietary fibres with the enterohepatic circulation of bile acids is proposed as a mechanism for lowering cholesterol. We investigated how lupin hull and cotyledon dietary fibres interact with primary bile acids using an in vitro model under simulated upper gastrointestinal conditions. Cell wall polysaccharides were isolated and extracted to separate pectin-like, hemicellulosic, and lignocellulosic structures. Lupin hull consisted mainly of structural components rich in cellulose. The viscosity of the in vitro digesta of lupin hull was low, showing predominantly liquid-like viscoelastic properties. On the other hand, lupin cotyledon fibre retarded bile acid release due to increased viscosity of the in vitro digesta, which was linked with high contents of pectic polymers forming an entangled network. Molecular interactions with bile acids were not measured for the hull but for the cotyledon, as follows: A total of 1.29&amp;nbsp;µmol/100&amp;nbsp;mg&amp;nbsp;DM of chenodesoxycholic acids were adsorbed. Molecular interactions of cholic and chenodesoxycholic acids were evident for lignin reference material but did not account for the adsorption of the lupin cotyledon. Furthermore, none of the isolated and fractionated cell wall materials showed a significant adsorptive capacity, thus disproving a major role of lupin cell wall polysaccharides in bile acid adsorption.

10.3390/nu11092117

---

## Péter&nbsp;

---

## The Bacterial Gut Microbiota of Adult Patients Infected, Colonized or Noncolonized by Clostridioides difficile.
 11.05.2020, _MJT Crobach, QR Ducarmon, EM Terveer, C Harmanus, IMJG Sanders, KM Verduin, EJ Kuijper, RD Zwittink_


: Gut microbiota composition in patients with Clostridioides difficile colonization is not well investigated. We aimed to identify bacterial signatures associated with resistance and susceptibility to C. difficile colonization (CDC) and infection (CDI). Therefore, gut microbiota composition from patients with CDC (n = 41), with CDI (n = 41), and without CDC (controls, n = 43) was determined through 16S rRNA gene amplicon sequencing. Bacterial diversity was decreased in CDC and CDI patients (p&lt;0.01). Overall microbiota composition was significantly different between control, CDC, and CDI patients (p = 0.001). Relative abundance of Clostridioides (most likely C. difficile) increased stepwise from controls to CDC and CDI patients. In addition, differential abundance analysis revealed that CDI patients' gut microbiota was characterized by significantly higher relative abundance of Bacteroides and Veillonella than CDC patients and controls. Control patients had significantly higher Eubacterium hallii and Fusicatenibacter abundance than colonized patients. Network analysis indicated that Fusicatenibacter was negatively associated with Clostridioides in CDI patients, while Veillonella was positively associated with Clostridioides in CDC patients. Bacterial microbiota diversity decreased in both CDC and CDI patients, but harbored a distinct microbiota. Eubacterium hallii and Fusicatenibacter may indicate resistance against C. difficile colonization and subsequent infection, while Veillonella may indicate susceptibility to colonization and infection by C. difficile.

[Source](https://www.mdpi.com/2076-2607/8/5/677)

10.3390/microorganisms8050677

---

## The Bacterial Gut Microbiota of Adult Patients Infected, Colonized or Noncolonized by <em>Clostridioides difficile</em>

[Source](https://www.mdpi.com/2076-2607/8/5/677)

---

## Effect of age and the individual on the gastrointestinal bacteriome of ponies fed a high-starch diet.
 14.05.2020, _PK Morrison, CJ Newbold, E Jones, HJ Worgan, DH Grove-White, AH Dugdale, C Barfoot, PA Harris, CM Argo_


Bacteria residing in the gastrointestinal tract of mammals are crucial for the digestion of dietary nutrients. Bacterial community composition is modified by age and diet in other species. Although horses are adapted to consuming fibre-based diets, high-energy, often high-starch containing feeds are increasingly used. The current study assessed the impact of age on the faecal bacteriome of ponies transitioning from a hay-based diet to a high-starch diet. Over two years, 23 Welsh Section A pony mares were evaluated (Controls, 5-15 years, n = 6/year, 12 in total; Aged, ≥19 years, n = 6 Year 1; n = 5 Year 2, 11 in total). Across the same 30-week (May to November) period in each year, animals were randomly assigned to a 5-week period of study and were individually fed the same hay to maintenance (2% body mass as daily dry matter intake) for 4-weeks. During the final week, 2g starch per kg body mass (micronized steam-flaked barley) was incorporated into the diet (3-day transition and 5 days at maximum). Faecal samples were collected for 11 days (final 3 days hay and 8 days hay + barley feeding). Bacterial communities were determined using Ion Torrent Sequencing of amplified V1-V2 hypervariable regions of 16S rRNA. Age had a minimal effect on the bacteriome response to diet. The dietary transition increased Candidatus Saccharibacteria and Firmicutes phyla abundance and reduced Fibrobactres abundance. At the genera level, Streptococcus abundance was increased but not consistently across individual animals. Bacterial diversity was reduced during dietary transition in Streptococcus 'responders'. Faecal pH and VFA concentrations were modified by diet but considerable inter-individual variation was present. The current study describes compositional changes in the faecal bacteriome associated with the transition from a fibre-based to a high-starch diet in ponies and emphasises the individual nature of dietary responses, which may reflect functional differences in the bacterial populations present in the hindgut.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0232689)

10.1371/journal.pone.0232689

---

## Gut microbial pulse provides nutrition for parental provisioning in incipient termite colonies

[Source](https://link.springer.com/article/10.1007/s00265-020-02843-y)

---

## The role of the gut microbiome in mediating standard metabolic rate after dietary shifts in the viviparous cockroach, <em>Diploptera punctata</em>

[Source](https://jeb.biologists.org/content/early/2020/05/08/jeb.218271)

---

## Genome reconstruction of the non-culturable spinach downy mildew <em>Peronospora effusa</em> by metagenome filtering

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0225808)

---

## Joël&nbsp;

---

## Unraveling the Microbiota of Natural Black cv. Kalamata Fermented Olives through 16S and ITS Metataxonomic Analysis

[Source](https://www.mdpi.com/2076-2607/8/5/672)

---

## Quantifying Plant-Borne Carbon Assimilation by Root-Associating Bacteria.
 13.05.2020, _S Waller, SL Wilder, MJ Schueller, AB Housh, RA Ferrieri_


Herbaspirillum seropedicae is a rhizobacteria that occupies a specialized ecological niche in agriculture. As an endophyte and prolific grass root colonizer it has the potential to promote plant growth, enhancing crop yield in many cereal crops. While the mechanisms for plant growth promotion are controversial, the one irrefutable fact is these microorganisms rely heavily on plant-borne carbon as their main energy source in support of their biological functions. Unfortunately, the tools and technology enabling researchers to trace carbon exchange between plants and the microorganisms associating with them has been limiting. Here, we demonstrate that radioactive 11CO2 administered to intact maize leaves with translocation of 11C-photosynthates to roots can provide a 'traceable' source of carbon whose assimilation by microbial organisms can be quantified with enormous sensitivity. Fluorescence root imaging of RAM10, a green fluorescent protein (GFP) reporting strain of H. seropedicae, was used to identify regions of high microbial colonization. Microbes were mechanically removed from these regions via sonication in saline solution and extracts were subjected to fluorescence measurement and gamma counting to correlate carbon-11 atoms with numbers of colony forming units. The method has potential to translate to other microorganisms provided they possess an optical reporting trait.

[Source](https://www.mdpi.com/2076-2607/8/5/700)

10.3390/microorganisms8050700

---

## Community-level signatures of ecological succession in natural bacterial communities.
 14.05.2020, _A Pascual-García, T Bell_


A central goal in microbial ecology is to simplify the extraordinary biodiversity that inhabits natural environments into ecologically coherent units. We profiled (16S rRNA sequencing) &gt; 700 semi-aquatic bacterial communities while measuring their functional capacity when grown in laboratory conditions. This approach allowed us to investigate the relationship between composition and function excluding confounding environmental factors. Simulated data allowed us to reject the hypothesis that stochastic processes were responsible for community assembly, suggesting that niche effects prevailed. Consistent with this idea we identified six distinct community classes that contained samples collected from distant locations. Structural equation models showed there was a functional signature associated with each community class. We obtained a more mechanistic understanding of the classes using metagenomic predictions (PiCRUST). This approach allowed us to show that the classes contained distinct genetic repertoires reflecting community-level ecological strategies. The ecological strategies resemble the classical distinction between r- and K-strategists, suggesting that bacterial community assembly may be explained by simple ecological mechanisms.

[Source](https://www.nature.com/articles/s41467-020-16011-3)

10.1038/s41467-020-16011-3

---

## Instigation of indigenous thermophilic bacterial consortia for enhanced oil recovery from high temperature oil reservoirs.
 12.05.2020, _N Sharma, M Lavania, V Kukreti, B Lal_


The purpose of the study involves the development of an anaerobic, thermophilic microbial consortium TERIK from the high temperature reservoir of Gujarat for enhance oil recovery. To isolate indigenous microbial consortia, anaerobic baltch media were prepared and inoculated with the formation water; incubated at 65°C for 10 days. Further, the microbial metabolites were analyzed by gas chromatography, FTIR and surface tension. The efficiency of isolated consortia towards enhancing oil recovery was analyzed through core flood assay. The novelty of studied consortia was that, it produces biomass (600 mg/l), bio-surfactant (325 mg/l), and volatile fatty acids (250 mg/l) at 65°C in the span of 10 days, that are adequate to alter the surface tension (70 to 34 mNm -1) and sweep efficiency of zones facilitating the displacement of oil. TERIK was identified as Clostridium sp. The FTIR spectra of biosurfactant indicate the presence of N-H stretch, amides and polysaccharide. A core flooding assay was designed to explore the potential of TERIK towards enhancing oil recovery. The results showed an effective reduction in permeability at residual oil saturation from 2.14 ± 0.1 to 1.39 ± 0.05 mD and 19% incremental oil recovery.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229889)

10.1371/journal.pone.0229889

---

## Microplastics affect sedimentary microbial communities and nitrogen cycling.
 13.05.2020, _ME Seeley, B Song, R Passie, RC Hale_


Microplastics are ubiquitous in estuarine, coastal, and deep sea sediments. The impacts of microplastics on sedimentary microbial ecosystems and biogeochemical carbon and nitrogen cycles, however, have not been well reported. To evaluate if microplastics influence the composition and function of sedimentary microbial communities, we conducted a microcosm experiment using salt marsh sediment amended with polyethylene (PE), polyvinyl chloride (PVC), polyurethane foam (PUF) or polylactic acid (PLA) microplastics. We report that the presence of microplastics alters sediment microbial community composition and nitrogen cycling processes. Compared to control sediments without microplastic, PUF- and PLA-amended sediments promote nitrification and denitrification, while PVC amendment inhibits both processes. These results indicate that nitrogen cycling processes in sediments can be significantly affected by different microplastics, which may serve as organic carbon substrates for microbial communities. Considering this evidence and increasing microplastic pollution, the impact of plastics on global ecosystems and biogeochemical cycling merits critical investigation.

[Source](https://www.nature.com/articles/s41467-020-16235-3)

10.1038/s41467-020-16235-3

---

## Peatland Microbial Community Composition Is Driven by a Natural Climate Gradient.
 10.05.2020, _J Seward, MA Carson, LJ Lamit, N Basiliko, JB Yavitt, E Lilleskov, CW Schadt, DS Smith, J Mclaughlin, N Mykytczuk, S Willims-Johnson, N Roulet, T Moore, L Harris, S Bräuer_


Peatlands are important players in climate change-biosphere feedbacks via long-term net carbon (C) accumulation in soil organic matter and as potential net C sources including the potent greenhouse gas methane (CH4). Interactions of climate, site-hydrology, plant community, and groundwater chemical factors influence peatland development and functioning, including C dioxide (CO2) and CH4 fluxes, but the role of microbial community composition is not well understood. To assess microbial functional and taxonomic dissimilarities, we used high throughput sequencing of the small subunit ribosomal DNA (SSU rDNA) to determine bacterial and archaeal community composition in soils from twenty North American peatlands. Targeted DNA metabarcoding showed that although Proteobacteria, Acidobacteria, and Actinobacteria were the dominant phyla on average, intermediate and rich fens hosted greater diversity and taxonomic richness, as well as an array of candidate phyla when compared with acidic and nutrient-poor poor fens and bogs. Moreover, pH was revealed to be the strongest predictor of microbial community structure across sites. Predictive metagenome content (PICRUSt) showed increases in specific genes, such as purine/pyrimidine and amino-acid metabolism in mid-latitude peatlands from 38 to 45° N, suggesting a shift toward utilization of microbial biomass over utilization of initial plant biomass in these microbial communities. Overall, there appears to be noticeable differences in community structure between peatland classes, as well as differences in microbial metabolic activity between latitudes. These findings are in line with a predicted increase in the decomposition and accelerated C turnover, and suggest that peatlands north of 37° latitude may be particularly vulnerable to climate change.

[Source](https://link.springer.com/article/10.1007/s00248-020-01510-z)

10.1007/s00248-020-01510-z

---

## Confocal Raman microscopy to identify bacteria in oral subgingival biofilm models.
 11.05.2020, _LS Kriem, K Wright, RA Ccahuana-Vasquez, S Rupp_


The study of oral disease progression, in relation to the accumulation of subgingival biofilm in gingivitis and periodontitis is limited, due to either the ability to monitor plaque in vitro. When compared, optical spectroscopic techniques offer advantages over traditional destructive or biofilm staining approaches, making it a suitable alternative for the analysis and continued development of three-dimensional structures. In this work, we have developed a confocal Raman spectroscopy analysis approach towards in vitro subgingival plaque models. The main objective of this study was to develop a method for differentiating multiple oral subgingival bacterial species in planktonic and biofilm conditions, using confocal Raman microscopy. Five common subgingival bacteria (Fusobacterium nucleatum, Streptococcus mutans, Veillonella dispar, Actinomyces naeslundii and Prevotella nigrescens) were used and differentiated using a 2-way orthogonal Partial Least Square with Discriminant Analysis (O2PLS-DA) for the collected spectral data. In addition to planktonic growth, mono-species biofilms cultured using the 'Zürich Model' were also analyzed. The developed method was successfully used to predict planktonic and mono-species biofilm species in a cross validation setup. The results show differences in the presence and absence of chemical bands within the Raman spectra. The O2PLS-DA model was able to successfully predict 100% of all tested planktonic samples and 90% of all mono-species biofilm samples. Using this approach we have shown that Confocal Raman microscopy can analyse and predict the identity of planktonic and mono-species biofilm species, thus enabling its potential as a technique to map oral multi-species biofilm models.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0232912)

10.1371/journal.pone.0232912

---

