## #microber2022

[Source](https://twitter.com/hashtag/microber2022?src=hashtag_click&f=live)

---

## Mechanism of Dyspnea during Exercise in Children with Corrected Congenital Heart Disease.
 24.01.2022, _M Chlif, MM Ammar, NB Said, L Sergey, S Ahmaidi, F Alassery, H Hamam_


_Child, Dyspnea, Exercise, Exercise Test, Heart Defects, Congenital, Humans, Oxygen Consumption_
This study will evaluate cardiorespiratory and peripheral muscle function and their relationship with subjective dyspnea threshold after the surgical correction of congenital heart disease in children. Thirteen children with surgically repaired congenital heart disease were recruited. Each participant performed an incremental exercise test on a cycle ergometer until exhaustion. Gas exchanges were continuously sampled to measure the maximal aerobic parameters and ventilatory thresholds. The functional capacity of the subjects was assessed with a 6 min walk test. At the end of the exercise test, isokinetic Cybex Norm was used to evaluate the strength and endurance of the knee extensor muscle in the leg. Dyspnea was subjectively scored with a visual analog scale during the last 15 s of each exercise step. Oxygen consumption measured at the dyspnea score/VO2 relationship located at the dyspnea threshold, at which dyspnea suddenly increased. Results: The maximal and submaximal values of the parameters describing the exercise and the peripheral muscular performances were: VO2 Peak: 33.8 ± 8.9 mL·min-1·kg-1; HR: 174 ± 9 b·min-1; VEmax: 65.68 ± 15.9 L·min-1; P max: 117 ± 27 W; maximal voluntary isometric force MVIF: 120.8 ± 41.9 N/m; and time to exhaustion Tlim: 53 ± 21 s. Oxygen consumption measured at the dyspnea threshold was related to VO2 Peak (R2 = 0.74; p &lt; 0.01), Tlim (R2 = 0.78; p &lt; 0.01), and the distance achieved during the 6MWT (R2 = 0.57; p &lt; 0.05). Compared to the theoretical maximal values for the power output, VO2, and HR, the surgical correction did not repair the exercise performance. After the surgical correction of congenital heart disease, exercise performance was impeded by alterations of the cardiorespiratory function and peripheral local factors. A subjective evaluation of the dyspnea threshold is a reliable criterion that allows the prediction of exercise capacity in subjects suffering from congenital heart disease.

[Source](https://twitter.com/search?q=%23MVIF&src=typed_query&f=live)

10.3390/ijerph19010099

---

## #MVIF

[Source](https://twitter.com/search?q=%23MVIF&src=typed_query&f=live)

---

## Mechanism of Dyspnea during Exercise in Children with Corrected Congenital Heart Disease.
 24.01.2022, _M Chlif, MM Ammar, NB Said, L Sergey, S Ahmaidi, F Alassery, H Hamam_


_Child, Dyspnea, Exercise, Exercise Test, Heart Defects, Congenital, Humans, Oxygen Consumption_
This study will evaluate cardiorespiratory and peripheral muscle function and their relationship with subjective dyspnea threshold after the surgical correction of congenital heart disease in children. Thirteen children with surgically repaired congenital heart disease were recruited. Each participant performed an incremental exercise test on a cycle ergometer until exhaustion. Gas exchanges were continuously sampled to measure the maximal aerobic parameters and ventilatory thresholds. The functional capacity of the subjects was assessed with a 6 min walk test. At the end of the exercise test, isokinetic Cybex Norm was used to evaluate the strength and endurance of the knee extensor muscle in the leg. Dyspnea was subjectively scored with a visual analog scale during the last 15 s of each exercise step. Oxygen consumption measured at the dyspnea score/VO2 relationship located at the dyspnea threshold, at which dyspnea suddenly increased. Results: The maximal and submaximal values of the parameters describing the exercise and the peripheral muscular performances were: VO2 Peak: 33.8 ± 8.9 mL·min-1·kg-1; HR: 174 ± 9 b·min-1; VEmax: 65.68 ± 15.9 L·min-1; P max: 117 ± 27 W; maximal voluntary isometric force MVIF: 120.8 ± 41.9 N/m; and time to exhaustion Tlim: 53 ± 21 s. Oxygen consumption measured at the dyspnea threshold was related to VO2 Peak (R2 = 0.74; p &lt; 0.01), Tlim (R2 = 0.78; p &lt; 0.01), and the distance achieved during the 6MWT (R2 = 0.57; p &lt; 0.05). Compared to the theoretical maximal values for the power output, VO2, and HR, the surgical correction did not repair the exercise performance. After the surgical correction of congenital heart disease, exercise performance was impeded by alterations of the cardiorespiratory function and peripheral local factors. A subjective evaluation of the dyspnea threshold is a reliable criterion that allows the prediction of exercise capacity in subjects suffering from congenital heart disease.

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

10.3390/ijerph19010099

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## @MicrobiomDigest

[Source](https://twitter.com/MicrobiomDigest?ref_src=twsrc%5Etfw)

---

## @CornetLuc3

[Source](https://twitter.com/CornetLuc3?ref_src=twsrc%5Etfw)

---

## @agustinmolinna

[Source](https://twitter.com/agustinmolinna?ref_src=twsrc%5Etfw)

---

## @anniegwest

[Source](https://twitter.com/anniegwest?ref_src=twsrc%5Etfw)

---

## https://t.co/Af8tkmNw5J

[Source](https://t.co/Af8tkmNw5J)

---

## #microbiome

[Source](https://twitter.com/hashtag/microbiome?src=hash&ref_src=twsrc%5Etfw)

---

## pic.twitter.com/uvXiZXL1yQ

[Source](https://t.co/uvXiZXL1yQ)

---

## October 10, 2022

[Source](https://twitter.com/MicrobiomeVIF/status/1579351022333575169?ref_src=twsrc%5Etfw)

---

## Review: Microbiome epidemiology and association studies in human health 

[Source](https://www.nature.com/articles/s41576-022-00529-x)

---

## The effects of fermented vegetable consumption on the composition of the intestinal microbiota and levels of inflammatory markers in women: A pilot and feasibility study 

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0275275)

---

## Functional screens of barcoded expression libraries uncover new gene functions in carbon utilization among gut Bacteroidales

[Source](https://www.biorxiv.org/content/10.1101/2022.10.10.511384v2)

---

## @AdamArkinLab

[Source](https://twitter.com/AdamArkinLab?ref_src=twsrc%5Etfw)

---

## #functionalgenomics

[Source](https://twitter.com/hashtag/functionalgenomics?src=hash&ref_src=twsrc%5Etfw)

---

## #microbiome

[Source](https://twitter.com/hashtag/microbiome?src=hash&ref_src=twsrc%5Etfw)

---

## #DNAbarcoding

[Source](https://twitter.com/hashtag/DNAbarcoding?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/tMKlN5cHQU

[Source](https://t.co/tMKlN5cHQU)

---

## October 12, 2022

[Source](https://twitter.com/_YHuang/status/1580107332012298241?ref_src=twsrc%5Etfw)

---

## Advancing human gut microbiota research by considering gut transit time

[Source](https://gut.bmj.com/content/early/2022/09/28/gutjnl-2022-328166)

---

## Food Additive Emulsifiers and Their Impact on Gut Microbiome, Permeability, and Inflammation: Mechanistic Insights in Inflammatory Bowel Disease

[Source](https://academic.oup.com/ecco-jcc/article/15/6/1068/6041235)

---

## Influence of the microbiome, diet and genetics on inter-individual variation in the human plasma metabolome

[Source](https://www.nature.com/articles/s41591-022-02014-8)

---

## @NatureMedicine

[Source](https://twitter.com/NatureMedicine?ref_src=twsrc%5Etfw)

---

## https://t.co/yFzCjJnOMB

[Source](https://t.co/yFzCjJnOMB)

---

## October 10, 2022

[Source](https://twitter.com/LianminChen/status/1579492045055741953?ref_src=twsrc%5Etfw)

---

## N-glycosylation of cervicovaginal fluid reflects microbial community, immune activity, and pregnancy status

[Source](https://www.nature.com/articles/s41598-022-20608-7)

---

## More than just hitchhikers: a survey of bacterial communities associated with diatoms originating from sea turtles Get access Arrow 

[Source](https://academic.oup.com/femsec/article/doi/10.1093/femsec/fiac104/6693935)

---

## @FEMSmicro

[Source](https://twitter.com/FEMSmicro?ref_src=twsrc%5Etfw)

---

## @ASSEMBLE_Plus

[Source](https://twitter.com/ASSEMBLE_Plus?ref_src=twsrc%5Etfw)

---

## @hrzz_science

[Source](https://twitter.com/hrzz_science?ref_src=twsrc%5Etfw)

---

## @Turtle_BIOME

[Source](https://twitter.com/Turtle_BIOME?ref_src=twsrc%5Etfw)

---

## https://t.co/w3z4SjEzuc

[Source](https://t.co/w3z4SjEzuc)

---

## October 10, 2022

[Source](https://twitter.com/Klara_Fil/status/1579554757945221120?ref_src=twsrc%5Etfw)

---

## Next generation tools for crop-microbiome manipulation to mitigate the impact of climate change.
 10.10.2022, _BD Batista, BK Singh_



[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1462-2920.16231)

10.1111/1462-2920.16231

---

## Defending Earth's terrestrial microbiome.
 03.10.2022, _C Averill, MA Anthony, P Baldrian, F Finkbeiner, J van den Hoogen, T Kiers, P Kohout, E Hirt, GR Smith, TW Crowther_


Microbial life represents the majority of Earth's biodiversity. Across disparate disciplines from medicine to forestry, scientists continue to discover how the microbiome drives essential, macro-scale processes in plants, animals and entire ecosystems. Yet, there is an emerging realization that Earth's microbial biodiversity is under threat. Here we advocate for the conservation and restoration of soil microbial life, as well as active incorporation of microbial biodiversity into managed food and forest landscapes, with an emphasis on soil fungi. We analyse 80 experiments to show that native soil microbiome restoration can accelerate plant biomass production by 64% on average, across ecosystems. Enormous potential also exists within managed landscapes, as agriculture and forestry are the dominant uses of land on Earth. Along with improving and stabilizing yields, enhancing microbial biodiversity in managed landscapes is a critical and underappreciated opportunity to build reservoirs, rather than deserts, of microbial life across our planet. As markets emerge to engineer the ecosystem microbiome, we can avert the mistakes of aboveground ecosystem management and avoid microbial monocultures of single high-performing microbial strains, which can exacerbate ecosystem vulnerability to pathogens and extreme events. Harnessing the planet's breadth of microbial life has the potential to transform ecosystem management, but it requires that we understand how to monitor and conserve the Earth's microbiome.

[Source](https://www.nature.com/articles/s41564-022-01228-3)

10.1038/s41564-022-01228-3

---

## GTDB-Tk v2: memory friendly classification with the Genome Taxonomy Database.
 11.10.2022, _PA Chaumeil, AJ Mussig, P Hugenholtz, DH Parks_


The Genome Taxonomy Database (GTDB) and associated taxonomic classification toolkit (GTDB-Tk) have been widely adopted by the microbiology community. However, the growing size of the GTDB bacterial reference tree has resulted in GTDB-Tk requiring substantial amounts of memory (∼320 GB) which limits its adoption and ease of use. Here we present an update to GTDB-Tk that uses a divide-and-conquer approach where user genomes are initially placed into a bacterial reference tree with family-level representatives followed by placement into an appropriate class-level subtree comprising species representatives. This substantially reduces the memory requirements of GTDB-Tk while having minimal impact on classification.

[Source](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btac672/6758240)

10.1093/bioinformatics/btac672

---

## binny: an automated binning algorithm to recover high-quality genomes from complex metagenomic datasets.
 14.10.2022, _O Hickl, P Queirós, P Wilmes, P May, A Heintz-Buschart_


The reconstruction of genomes is a critical step in genome-resolved metagenomics and for multi-omic data integration from microbial communities. Here, we present binny, a binning tool that produces high-quality metagenome-assembled genomes (MAG) from both contiguous and highly fragmented genomes. Based on established metrics, binny outperforms or is highly competitive with commonly used and state-of-the-art binning methods and finds unique genomes that could not be detected by other methods. binny uses k-mer-composition and coverage by metagenomic reads for iterative, nonlinear dimension reduction of genomic signatures as well as subsequent automated contig clustering with cluster assessment using lineage-specific marker gene sets. When compared with seven widely used binning algorithms, binny provides substantial amounts of uniquely identified MAGs and almost always recovers the most near-complete ($\gt 95\%$ pure, $\gt 90\%$ complete) and high-quality ($\gt 90\%$ pure, $\gt 70\%$ complete) genomes from simulated datasets from the Critical Assessment of Metagenome Interpretation initiative, as well as substantially more high-quality draft genomes, as defined by the Minimum Information about a Metagenome-Assembled Genome standard, from a real-world benchmark comprised of metagenomes from various environments than any other tested method.

[Source](https://www.biorxiv.org/content/10.1101/2021.12.22.473795v6)

10.1093/bib/bbac431

---

## VEBA: a modular end-to-end suite for in silico recovery, clustering, and analysis of prokaryotic, microeukaryotic, and viral genomes from metagenomes.
 14.10.2022, _JL Espinoza, CL Dupont_


_Archaea, Bacteria, Cluster Analysis, Ecosystem, Eukaryota, Genome, Viral, Humans, Metagenome, Metagenomics_
With the advent of metagenomics, the importance of microorganisms and how their interactions are relevant to ecosystem resilience, sustainability, and human health has become evident. Cataloging and preserving biodiversity is paramount not only for the Earth's natural systems but also for discovering solutions to challenges that we face as a growing civilization. Metagenomics pertains to the in silico study of all microorganisms within an ecological community in situ, however, many software suites recover only prokaryotes and have limited to no support for viruses and eukaryotes.

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-04973-8)

10.1186/s12859-022-04973-8

---

## #microber2022

[Source](https://twitter.com/hashtag/microber2022?src=hash&ref_src=twsrc%5Etfw)

---

## pic.twitter.com/Xk1ntqA76a

[Source](https://t.co/Xk1ntqA76a)

---

## October 11, 2022

[Source](https://twitter.com/LizahvdAart/status/1579840049776308225?ref_src=twsrc%5Etfw)

---

