## Manipulating denitrifying sulfide removal of <em>Pseudomonas sp.</em> C27 with nitrite as sole nitrogen source: Shotgun proteomics analysis

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0960852420313468)

---

## <em>Pseudomonas aeruginosa</em> – Candida interplay: Effect on in vitro antibiotic susceptibility of <em>Pseudomonas aeruginosa</em> when grown in the presence <em>Candida</em> culture

[Source](https://www.tandfonline.com/doi/abs/10.1080/09674845.2020.1819003?journalCode=tbbs20)

---

## Impact of DNA extraction on whole genome sequencing analysis for characterization and relatedness of Shiga toxin-producing <em>Escherichia coli</em> isolates

[Source](https://www.nature.com/articles/s41598-020-71207-3)

---

## Evolution of diversity explains the impact of pre-adaptation of a focal species on the structure of a natural microbial community

[Source](https://www.nature.com/articles/s41396-020-00755-3)

---

## Biomass-degrading glycoside hydrolases of archaeal origin

[Source](https://biotechnologyforbiofuels.biomedcentral.com/articles/10.1186/s13068-020-01792-y)

---

## Mining for novel antibiotics in the age of antimicrobial resistance.
 03.09.2020, _Z Udaondo, MA Matilla_


The misuse of antimicrobials is causing an alarming increase in the appearance of antibiotic-resistant microorganisms. In this context, the identification of novel antibiotics against new targets and with low rates of resistance development is a major global challenge. In this article, we highlight a number of recent articles that exploit a variety of in vitro, in vivo and in silico state-of-the-art approaches to identify and develop new antimicrobials. Rapid progress in this research field will be crucial to combating a global health problem, antimicrobial resistance, that is expected to be the leading cause of death by 2050.

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/pdfdirect/10.1111/1751-7915.13662)

10.1111/1751-7915.13662

---

## Association between the microbiomes of tonsil and saliva samples isolated from pediatric patients subjected to tonsillectomy for the treatment of tonsillar hyperplasia.
 05.09.2020, _DH Choi, J Park, JK Choi, KE Lee, WH Lee, J Yang, JY Lee, YJ Park, C Oh, HR Won, BS Koo, JW Chang, YS Park_


Oral microbes have the capacity to spread throughout the gastrointestinal system and are strongly associated with multiple diseases. Given that tonsils are located between the oral cavity and the laryngopharynx at the gateway of the alimentary and respiratory tracts, tonsillar tissue may also be affected by microbiota from both the oral cavity (saliva) and the alimentary tract. Here, we analyzed the distribution and association of the microbial communities in the saliva and tonsils of Korean children subjected to tonsillectomy because of tonsil hyperplasia (n = 29). The microbiome profiles of saliva and tonsils were established via 16S rRNA gene sequencing. Based on the alpha diversity indices, the microbial communities of the two groups showed high similarities. According to Spearman's ranking correlation analysis, the distribution of Treponema, the causative bacterium of periodontitis, in saliva and tonsils was found to have a significant positive correlation. Two representative microbes, Prevotella in saliva and Alloprevotella in tonsils, were negatively correlated, while Treponema 2 showed a strong positive correlation between saliva and tonsils. Taken together, strong similarities in the microbial communities of the tonsils and saliva are evident in terms of diversity and composition. The saliva microbiome is expected to significantly affect the tonsil microbiome. Furthermore, we suggest that our study creates an opportunity for tonsillar microbiome research to facilitate the development of novel microbiome-based therapeutic strategies.

[Source](https://www.nature.com/articles/s12276-020-00487-6)

10.1038/s12276-020-00487-6

---

## The impact of the human microbiome in forensic sciences: A systematic review.
 05.09.2020, _MG García, MD Pérez-Cárceles, E Osuna, I Legaz_


Numerous studies relate differences in microbial communities to human health and disease, however little is known about microbial changes that occur post mortem or the possible applications of microbiome analysis in the field of forensic science. The aim of this review was study the microbiome and its applications in forensic sciences, to determine the main lines of investigation that are emerging, as well as its possible contributions in the forensic field. A systematic review of the human microbiome in relation to forensic science was carried out following PRISMA guidelines. This study throws light on the role of microbiome research in the postmortem interval during the process of decomposition; identifying death caused by drowning or sudden death; locating the geographical location of death; establishing a connection between human microbiome and personal items, sexual contact, and the identification of individuals. Actinomycetaceae, Bacteroidaceae, Alcaligenaceae and Bacilli play an important role in determining the postmortem interval. Aeromonas can be used to determine the cause of death, and Corynebacterium or Helicobacter pylori to ascertain personal identity or geographical location. Several studies point to a promising future for microbiome analysis in the different fields of forensic science, opening up an important new area of research.

[Source](https://aem.asm.org/content/early/2020/09/01/AEM.01451-20.abstract)

10.1128/AEM.01451-20

---

## The evolving role of the lung microbiome in pulmonary fibrosis.
 02.09.2020, _JH Lipinski, BB Moore, DN O'Dwyer_


Mucosal surfaces are constantly exposed to a microbiome consisting of micro-organisms that heavily influence human immunity and health. In the lung these micro-organisms consist of bacteria, viruses and fungi and exist in a relatively low biomass state. Bacterial communities of the lung modulate local inflammation and correlate with changes in pulmonary physiology and clinical outcomes in patients with lung disease. Instrumental to this progress has been the study of these bacterial communities in the pathogenesis of pulmonary fibrosis, a fatal and progressive disease culminating in respiratory failure. Key pathophysiological mechanisms in pulmonary fibrosis include recurrent idiopathic alveolar epithelial injury, unchecked collagen deposition, mucociliary dysfunction due to muc5b overexpression, hypoxia, and altered host defense. These key mechanisms and their related consequences promote severe progressive architectural lung destruction and loss of local homeostasis. As such, pulmonary fibrosis is an appropriate target disease for the study of the lung microbiome. Herein, we discuss recent advances in our understanding of the role of the lung microbiome in the pathogenesis of pulmonary fibrosis. We highlight fundamental clinical observations, mechanistic insights and identify crucial areas for further discovery science. An improved understanding of how the lung microbiome acts to influence outcomes in patients with pulmonary fibrosis will lead to enhanced therapies for this devastating lung disease.

[Source](https://journals.physiology.org/doi/abs/10.1152/ajplung.00258.2020)

10.1152/ajplung.00258.2020

---

## Recent Advances in Understanding the Ecology of the Lung Microbiota and Deciphering the Gut-Lung Axis.
 02.09.2020, _KA Willis, JD Stewart, N Ambalavanan_


A rapidly expanding new field of lung research has been produced by the emergence of culture-independent next-generation sequencing technologies. While pulmonary microbiome research lags behind the exploration of the microbiome in other organ systems, the field is maturing and has recently produced multiple exciting discoveries. In this Mini-Review, we will explore recent advances in our understanding of the lung microbiome and the gut-lung axis from an ecological perspective.

[Source](https://journals.physiology.org/doi/abs/10.1152/ajplung.00360.2020)

10.1152/ajplung.00360.2020

---

## Recent advances in understanding the ecology of the lung microbiota and deciphering the gut-lung axis

[Source](https://journals.physiology.org/doi/abs/10.1152/ajplung.00360.2020)

---

## Altered gut microbiota correlated with systemic inflammation in children with Kawasaki disease.
 06.09.2020, _J Chen, Y Yue, L Wang, Z Deng, Y Yuan, M Zhao, Z Yuan, C Tan, Y Cao_


Kawasaki disease (KD) is a multi-systemic vasculitis of unknown etiology that occurs mainly in children, and the disturbance of gut microbiota is generally believed to cause a hyperimmune reaction triggering KD. The aim of the study was to investigate the alterations in the fecal microbiota and assess its relationship with systemic inflammation. Totally 30 KD children were enrolled and followed up for 6 months, with another group of 30 age- and sex-matched healthy children as controls. Phylotype profiles of fecal microbial communities were analyzed using 16S rRNA gene sequencing. Serum inflammatory markers were detected by flow cytometer. We showed that KD children exhibited a significant reduction in fecal microbial diversity in the acute phase compared with the healthy controls. Enterococcus, Acinetobacter, Helicobacter, Lactococcus, Staphylococcus and Butyricimonas in acute KD children were significantly higher than the healthy children. Levels of systemic inflammation biomarkers, including IL-2, IL-4, IL-6, IL-10, TNF-α, and INF-γ, were significantly elevated in the acute KD children. Altered microbiota genera Enterococcus and Helicobacter abundances were shown to be correlated positively with IL-6, which were never previously reported in KD. This study suggested that gut microbiota alteration is closely associated with systemic inflammation, which provides a new perspective on the etiology and pathogenesis of KD.

[Source](https://www.nature.com/articles/s41598-020-71371-6)

10.1038/s41598-020-71371-6

---

## Gut microbiota in human metabolic health and disease – Fan and Pedersen

[Source](https://www.nature.com/articles/s41579-020-0433-9)

---

## Lactose-reduced infant formula with added corn syrup solids is associated with a distinct gut microbiota in Hispanic infants.
 07.09.2020, _RB Jones, PK Berger, JF Plows, TL Alderete, J Millstein, J Fogel, SN Iablokov, DA Rodionov, AL Osterman, L Bode, MI Goran_


Infant formula feeding, compared with human milk, has been associated with development of a distinct infant gut microbiome, but no previous study has examined effects of formula with added sugars. This work examined differences in gut microbiota among 91 Hispanic infants who consumed human milk [at breast (BB) vs. pumped in bottle (BP)] and 2 kinds of infant formula [(traditional lactose-based (TF) vs. lactose-reduced with added sugar (ASF)]. At 1 and 6 months, infant stool was collected to characterize gut microbiota. At 6 months, mothers completed 24-hour dietary recalls and questionnaires to determine infant consumption of human milk (BB vs. BP) or formula (TF vs. ASF). Linear regression models were used to determine associations of milk consumption type and microbial features at 6 months. Infants in the formula groups exhibited a significantly more 'mature' microbiome than infants in the human milk groups with the most pronounced differences observed between the ASF vs. BB groups. In the ASF group, we observed reduced log-normalized abundance of Bifidobacteriaceae (TF-BB Mean Difference = -0.71, ASF-BB Mean Difference = -1.10), and increased abundance of Lachnospiraceae (TF-BB Mean Difference = +0.89, ASF-BB Mean Difference = +1.20). We also observed a higher Community Phenotype Index of propionate, most likely produced by Lachnospiraceae, in the ASF group (TF-BB Mean Difference = +0.27, ASF-BB Mean Difference = +0.36). This study provides the first evidence that consumption of infant formula with added sugar may have a stronger association than birth delivery mode, infant caloric intake, and maternal BMI on the infant's microbiome at 6 months of age.

[Source](https://www.tandfonline.com/doi/pdf/10.1080/19490976.2020.1813534)

10.1080/19490976.2020.1813534

---

## Gut bacterial communities and their contribution to performance of specialist <em>Altica</em> flea beetles

[Source](https://link.springer.com/article/10.1007/s00248-020-01590-x)

---

## Dynamic changes of gut microbiota of discus fish (<em>Symphysodon haraldi</em>) at different feeding stages

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0044848620319724)

---

## Diversity of cultivable microbes from soil of the Fildes Peninsula, Antarctica, and their potential application

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.570836/full)

---

## Structural and functional microbial diversity of sandy soil under cropland and grassland

[Source](https://peerj.com/articles/9501/)

---

## Microbiome–environment interactions in antimony-contaminated rice paddies and the correlation of core microbiome with arsenic and antimony contamination

[Source](https://www.sciencedirect.com/science/article/abs/pii/S004565352032422X)

---

## Implications of endophytic microbiota in <em>Camellia Sinensis</em>: A review on current understanding and future insights

[Source](https://www.tandfonline.com/doi/pdf/10.1080/21655979.2020.1816788)

---

## Characterizing the nectar microbiome of the non-native tropical milkweed, <em>Asclepias curassavica</em>, in an urban environment 

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237561)

---

## Prokaryotic diversity in stream sediments affected by acid mine drainage

[Source](https://link.springer.com/article/10.1007/s00792-020-01196-8)

---

## Combination of genetic tools to discern <em>Bacillus</em> species isolated from hot springs in South Africa

[Source](https://academicjournals.org/journal/AJMR/article-full-text-pdf/94619F264699)

---

## Bioleaching of pyritic coal wastes: bioprospecting and efficiency of selected consortia

[Source](https://www.sciencedirect.com/science/article/pii/S0923250820300814)

---

## Genotypic and phylogenetic diversity of new isolates of terrestrial Xanthophyceae (Stramenopiles) from maritime sandy habitats

[Source](https://www.tandfonline.com/doi/abs/10.1080/00318884.2020.1802950)

---

## <em>Paucihalobacter ruber</em> gen. nov., sp. nov., isolated from a haloalkaline lake sediment sample

[Source](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.004421)

---

## Comparative analysis of bacterial diversity and community structure in the rhizosphere and root endosphere of two halophytes, <em>Salicornia europaea</em> and <em>Glaux maritima</em>, collected from two brackish lakes in Japan 

[Source](https://www.jstage.jst.go.jp/article/jsme2/35/3/35_ME20072/_pdf)

---

## Fungal diversity in deep-sea sediments from Magellan seamounts environment of the western Pacific revealed by high-throughput Illumina sequencing

[Source](https://link.springer.com/article/10.1007/s12275-020-0198-x)

---

## Effect of Ferrous Iron Addition on Ammonium Nitrogen Removal and Microbial Communities in Horizontal Subsurface Flow Constructed Wetlands

[Source](https://link.springer.com/article/10.1007/s13157-020-01343-z)

---

## Microplastics negatively affect soil fauna but stimulate microbial activity: insights from a field-based microplastic addition experiment

[Source](https://royalsocietypublishing.org/doi/pdf/10.1098/rspb.2020.1268)

---

## Shared bicycle microbial community: a potential antibiotic-resistant bacteria warehouse

[Source](https://link.springer.com/article/10.1007/s12223-020-00820-6)

---

## Microbial carrying capacity and carbon biomass of plastic marine debris

[Source](https://www.nature.com/articles/s41396-020-00756-2)

---

## Proteolysis, lipolysis, texture and sensory properties of cheese ripened by <em>Monascus fumeus</em>

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0963996920306827)

---

## <span class="has-inline-color has-vivid-green-cyan-color">Best practices for successfully writing and publishing a genome announcement in microbiology resource announcements</span>

[Source](https://mra.asm.org/content/ga/9/36/e00763-20.full.pdf)

---

## Flow-through stable isotope probing (Flow-SIP) minimizes cross-feeding in complex microbial communities

[Source](https://www.nature.com/articles/s41396-020-00761-5)

---

