## Microbiome definition re-visited: old concepts and new challenges.
 03.07.2020, _G Berg, D Rybakova, D Fischer, T Cernava, MC Vergès, T Charles, X Chen, L Cocolin, K Eversole, GH Corral, M Kazou, L Kinkel, L Lange, N Lima, A Loy, JA Macklin, E Maguin, T Mauchline, R McClure, B Mitter, M Ryan, I Sarand, H Smidt, B Schelkle, H Roume, GS Kiran, J Selvin, RSC Souza, L van Overbeek, BK Singh, M Wagner, A Walsh, A Sessitsch, M Schloter_


The field of microbiome research has evolved rapidly over the past few decades and has become a topic of great scientific and public interest. As a result of this rapid growth in interest covering different fields, we are lacking a clear commonly agreed definition of the term "microbiome." Moreover, a consensus on best practices in microbiome research is missing. Recently, a panel of international experts discussed the current gaps in the frame of the European-funded MicrobiomeSupport project. The meeting brought together about 40 leaders from diverse microbiome areas, while more than a hundred experts from all over the world took part in an online survey accompanying the workshop. This article excerpts the outcomes of the workshop and the corresponding online survey embedded in a short historical introduction and future outlook. We propose a definition of microbiome based on the compact, clear, and comprehensive description of the term provided by Whipps et al. in 1988, amended with a set of novel recommendations considering the latest technological developments and research findings. We clearly separate the terms microbiome and microbiota and provide a comprehensive discussion considering the composition of microbiota, the heterogeneity and dynamics of microbiomes in time and space, the stability and resilience of microbial networks, the definition of core microbiomes, and functionally relevant keystone species as well as co-evolutionary principles of microbe-host and inter-species interactions within the microbiome. These broad definitions together with the suggested unifying concepts will help to improve standardization of microbiome studies in the future, and could be the starting point for an integrated assessment of data resulting in a more rapid transfer of knowledge from basic science into practice. Furthermore, microbiome standards are important for solving new challenges associated with anthropogenic-driven changes in the field of planetary health, for which the understanding of microbiomes might play a key role. Video Abstract.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00875-0)

10.1186/s40168-020-00875-0

---

## Competition, not cooperation, dominates interactions among culturable microbial species.
 21.11.2013, _KR Foster, T Bell_


_Bacteria, Bacterial Physiological Phenomena, Carbon Dioxide, Water Microbiology_
Microbial cells secrete numerous enzymes, scavenging molecules, and signals that can promote the growth and survival of other cells around them [1-4]. This observation is consistent with the evolution of cooperation within species [5], and there is now an increasing emphasis on the importance of cooperation between different microbial species [4, 6]. We lack, however, a systematic test of the importance of mutually positive interactions between different species, which is vital for assessing the commonness and importance of cooperative evolution in natural communities. Here, we study the extent of mutually positive interaction among bacterial strains isolated from a common aquatic environment. Using data collected from two independent experiments evaluating community productivity across diversity gradients, we show that (1) in pairwise species combinations, the great majority of interactions are net negative and (2) there is no evidence that strong higher-order positive effects arise when more than two species are mixed together. Our data do not exclude the possibility of positive effects in one direction where one species gains at the expense of another, i.e., predator-prey-like interactions. However, these do not constitute cooperation and our analysis suggests that the typical result of adaptation to other microbial species will be competitive, rather than cooperative, phenotypes.

[Source](https://www.biorxiv.org/content/10.1101/2020.06.24.169474v1)

10.1016/j.cub.2012.08.005

---

## Positive interactions are common among culturable bacteria

[Source](https://www.biorxiv.org/content/10.1101/2020.06.24.169474v1)

---

## An emerging role of<em> Prevotella histicola</em> on estrogen-deficiency induced bone loss through the gut microbiota-bone axis

[Source](https://www.biorxiv.org/content/10.1101/2020.06.03.133082v1.full.pdf)

---

## Identifying novel high-impact rare disease-causing mutations, genes and pathways in exomes of Ashkenazi Jewish inflammatory bowel disease patients

[Source](https://www.medrxiv.org/content/10.1101/2020.07.01.20143750v1.full.pdf)

---

## Early prediction of liver disease using conventional risk factors and gut microbiome-augmented gradient boosting 

[Source](https://www.medrxiv.org/content/10.1101/2020.06.24.20138933v1.full.pdf)

---

## GWAS of stool frequency reveals genes, pathways, and cell types relevant to human gastrointestinal motility and irritable bowel syndrome

[Source](https://www.medrxiv.org/content/10.1101/2020.06.17.20132555v1.full.pdf)

---

## Relationships of gut microbiota, short-chain fatty acids, inflammation, and the gut barrier in Parkinson’s disease

[Source](https://www.medrxiv.org/content/10.1101/2020.06.15.20131011v1.full.pdf)

---

## Genetically diverse<em> Pseudomonas aeruginosa</em> populations display similar transcriptomic profiles in a cystic fibrosis explanted lung

[Source](https://www.nature.com/articles/s41467-019-11414-3)

---

## Architecture and function of human uromodulin filaments in urinary tract infections

[Source](https://science.sciencemag.org/content/early/2020/07/01/science.aaz9866)

---

## The structure and diversity of strain level variation in vaginal bacteria

[Source](https://www.biorxiv.org/content/10.1101/2020.06.26.173922v1)

---

## Synchronous genitourinary lichen sclerosus signals a distinct urinary microbiome profile in men with urethral stricture disease

[Source](https://link.springer.com/article/10.1007/s00345-020-03198-9)

---

## Human urine 1H NMR metabolomics reveals alterations of the protein and carbohydrate metabolism when comparing habitual Average Danish diet vs. healthy New Nordic diet

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0899900720301507)

---

## <em>Lactobacillus rhamnosus</em> probiotic prevents airway function deterioration and promotes gut microbiome resilience in a murine asthma model

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2020.1766345)

---

## The Sourdough Microbiome

[Source](https://asm.org/Articles/2020/June/The-Sourdough-Microbiome)

---

## Strain-Level Diversity Impacts Cheese Rind Microbiome Assembly and Function

[Source](https://msystems.asm.org/content/5/3/e00149-20)

---

## Light into the darkness: Unifying the known and unknown coding sequence space in microbiome analyses

[Source](https://www.biorxiv.org/content/10.1101/2020.06.30.180448v1)

---

## Welcome to the dark side of genomes and metagenomes

[Source](http://merenlab.org/2020/07/01/dark-side/)

---

## GraphBin: refined binning of metagenomic contigs using assembly graphs.
 03.06.2020, _V Mallawaarachchi, A Wickramarachchi, Y Lin_


The field of metagenomics has provided valuable insights into the structure, diversity and ecology within microbial communities. One key step in metagenomics analysis is to assemble reads into longer contigs which are then binned into groups of contigs that belong to different species present in the metagenomic sample. Binning of contigs plays an important role in metagenomics and most available binning algorithms bin contigs using genomic features such as oligonucleotide/k-mer composition and contig coverage. As metagenomic contigs are derived from the assembly process, they are output from the underlying assembly graph which contains valuable connectivity information between contigs that can be used for binning.

[Source](https://academic.oup.com/bioinformatics/article-abstract/36/11/3307/5804980)

10.1093/bioinformatics/btaa180

---

## NG-meta-profiler: fast processing of metagenomes using NGLess, a domain-specific language.
 07.05.2020, _LP Coelho, R Alves, P Monteiro, J Huerta-Cepas, AT Freitas, P Bork_


_Algorithms, Computational Biology, High-Throughput Nucleotide Sequencing, Metagenome, Metagenomics, Programming Languages, Reproducibility of Results, Sequence Analysis, DNA_
Shotgun metagenomes contain a sample of all the genomic material in an environment, allowing for the characterization of a microbial community. In order to understand these communities, bioinformatics methods are crucial. A common first step in processing metagenomes is to compute abundance estimates of different taxonomic or functional groups from the raw sequencing data. Given the breadth of the field, computational solutions need to be flexible and extensible, enabling the combination of different tools into a larger pipeline.

[Source](https://bioinformatics.chat/ngless)

10.1186/s40168-019-0684-8

---

## Reproducible pipelines and NGLess with Luis Pedro Coelho

[Source](https://bioinformatics.chat/ngless)

---

