## Protection against Clostridioides difficile disease by a naturally avirulent strain.
 03.12.2024, _Q Dong, S Harper, E McSpadden, SS Son, MM Allen, H Lin, RC Smith, C Metcalfe, V Burgo, C Woodson, A Sundararajan, A Rose, M McMillin, D Moran, J Little, MW Mullowney, AM Sidebottom, LC Fortier, A Shen, EG Pamer_


Clostridioides difficile is a leading cause of healthcare infections. Gut dysbiosis promotes C. difficile infection (CDI) and CDIs promote gut dysbiosis, leading to frequent CDI recurrence. Although therapies preventing recurrent CDI have been developed, including live biotherapeutic products, existing therapies are costly and do not prevent primary infections. Here, we show that an avirulent C. difficile isolate, ST1-75, protects mice from developing colitis induced by a virulent R20291 strain when coinfected at a 1:1 ratio. In metabolic analyses, avirulent ST1-75 depletes amino acids more rapidly than virulent R20291 and supplementation with amino acids ablates this competitive advantage, indicating that ST1-75 limits the growth of virulent R20291 through amino acid depletion. Overall, our study identifies inter-strain nutrient depletion as a potentially exploitable mechanism to reduce the incidence of CDI and reveals that the ST1-75 strain may be a biotherapeutic agent that can prevent CDI in high-risk patients.

[Source](https://www.cell.com/cell-host-microbe/abstract/S1931-3128(24)00409-8)

10.1016/j.chom.2024.11.003

---

## The host genotype actively shapes its microbiome across generations in laboratory mice

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01954-2)

---

## Prenatal antibiotics reduce breast milk IgA and induce dysbiosis in mouse offspring, increasing neonatal susceptibility to bacterial sepsis.
 27.11.2024, _C Pietrasanta, C Carlosama, M Lizier, G Fornasa, TR Jost, S Carloni, S Giugliano, A Silvestri, P Brescia, B De Ponte Conti, D Braga, M Mihula, L Morosi, A Bernardinello, A Ronchi, G Martano, F Mosca, G Penna, F Grassi, L Pugni, M Rescigno_


Antibiotics (Abx) are administered to 20%-30% of pregnant women, but their effects on neonatal immune development are poorly understood. We show that newborn mice born to Abx-treated dams are more susceptible to late-onset sepsis. This susceptibility is linked to lower maternal breast milk immunoglobulin A (IgA), neonatal fecal IgA, and IgA coating of intestinal bacteria, thus causing the translocation of intestinal pathobionts. Weaned young adults born to Abx-treated mothers had reduced IgA+ plasma cells in the ileum and colon, fecal secretory IgA (SIgA), colonic CD4+ T regulatory lymphocytes and T helper 17-like lymphocytes, and a less diverse fecal microbiome. However, treatment with apyrase, which restores SIgA secretion, prompted IgA production in breast milk and protected pups from sepsis. Additionally, breast milk from untreated mothers rescued the phenotypes of pups born to Abx-treated mothers. Our data highlight the impact of prenatal Abx on breast milk IgA and their long-term influence on intestinal mucosal immune function mediated by breastfeeding.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(24)00407-4)

10.1016/j.chom.2024.11.001

---

## Gingival spatial analysis reveals geographic immunological variation in a microbiota-dependent and -independent manner 

[Source](https://www.nature.com/articles/s41522-024-00625-2)

---

## Host DNA depletion on frozen human respiratory samples enables successful metagenomic sequencing for microbiome studies.
 03.12.2024, _M Kim, RC Parrish, MJ Tisza, VS Shah, T Tran, M Ross, J Cormier, A Baig, CY Huang, L Brenner, I Neuringer, K Whiteson, JK Harris, AD Willis, PS Lai_


_Humans, Metagenomics, Microbiota, Sputum, Cystic Fibrosis, Bronchoalveolar Lavage Fluid, Freezing, Adult, Cryopreservation, Female, High-Throughput Nucleotide Sequencing, Bacteria, Sequence Analysis, DNA, Male, Respiratory System_
Most respiratory microbiome studies use amplicon sequencing due to high host DNA. Metagenomics sequencing offers finer taxonomic resolution, phage assessment, and functional characterization. We evaluated five host DNA depletion methods on frozen nasal swabs from healthy adults, sputum from people with cystic fibrosis (pwCF), and bronchoalveolar lavage (BAL) from critically ill patients. Median sequencing depth was 76.4 million reads per sample. Untreated nasal, sputum, and BAL had 94.1%, 99.2%, and 99.7% host reads, respectively. Host depletion effects varied by sample type, generally increasing microbial reads, species and functional richness; this was mediated by higher effective sequencing depth. Rarefaction curves showed species richness saturation at 0.5-2 million microbial reads. Most methods did not change Morisita-Horn dissimilarity for BAL and nasal samples although the proportion of gram-negative bacteria decreased for sputum from pwCF. Freezing did not affect the viability of Staphylococcus aureus but reduced the viability of Pseudomonas aeruginosa and Enterobacter spp.; this was mitigated by adding a cryoprotectant. QIAamp-based host depletion minimally impacted gram-negative viability even in non-cryoprotected frozen isolates. While some host depletion methods may shift microbial composition, metagenomics sequencing without host depletion severely underestimates microbial diversity of respiratory samples due to shallow effective sequencing depth and is not recommended.

[Source](https://www.nature.com/articles/s42003-024-07290-3)

10.1038/s42003-024-07290-3

---

## The respiratory microbiome is linked to the severity of RSV infections and the persistence of symptoms in children

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(24)00607-4)

---

## Plant microbiota feedbacks through dose-responsive expression of general non-self response genes.
 03.12.2024, _A Keppler, M Roulier, S Pfeilmeier, GC Petti, A Sintsova, BA Maier, M Bortfeld-Miller, S Sunagawa, C Zipfel, JA Vorholt_


The ability of plants to perceive and react to biotic and abiotic stresses is critical for their health. We recently identified a core set of genes consistently induced by members of the leaf microbiota, termed general non-self response (GNSR) genes. Here we show that GNSR components conversely impact leaf microbiota composition. Specific strains that benefited from this altered assembly triggered strong plant responses, suggesting that the GNSR is a dynamic system that modulates colonization by certain strains. Examination of the GNSR to live and inactivated bacteria revealed that bacterial abundance, cellular composition and exposure time collectively determine the extent of the host response. We link the GNSR to pattern-triggered immunity, as diverse microbe- or danger-associated molecular patterns cause dynamic GNSR gene expression. Our findings suggest that the GNSR is the result of a dose-responsive perception and signalling system that feeds back to the leaf microbiota and contributes to the intricate balance of plant-microbiome interactions.

[Source](https://www.nature.com/articles/s41477-024-01856-z)

10.1038/s41477-024-01856-z

---

## Dynamics of rice seed-borne bacteria from acquisition to seedling colonization.
 05.12.2024, _L Chen, H Bao, J Yang, Y Huo, J Zhang, R Fang, L Zhang_


_Oryza, Seeds, Seedlings, Bacteria, RNA, Ribosomal, 16S, Microbiota, Plant Roots, Germination, Soil Microbiology_
Rice cultivation relies on planting grains harboring beneficial microbiota. However, the origination, distribution, and transmission dynamics of grain-borne bacteria remain unclear.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01978-8)

10.1186/s40168-024-01978-8

---

## Managing false positives during detection of pathogen sequences in shotgun metagenomics datasets.
 03.12.2024, _LM Bradford, C Carrillo, A Wong_


_Metagenomics, Salmonella, Software, False Positive Reactions, Sensitivity and Specificity, Computational Biology, Sequence Analysis, DNA, Food Microbiology_
Culture-independent diagnostic tests are gaining popularity as tools for detecting pathogens in food. Shotgun sequencing holds substantial promise for food testing as it provides abundant information on microbial communities, but the challenge is in analyzing large and complex sequencing datasets with a high degree of both sensitivity and specificity. Falsely classifying sequencing reads as originating from pathogens can lead to unnecessary food recalls or production shutdowns, while low sensitivity resulting in false negatives could lead to preventable illness.

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-024-05952-x)

10.1186/s12859-024-05952-x

---

## TaxSEA: an R package for rapid interpretation of differential abundance analysis output

[Source](https://www.biorxiv.org/content/10.1101/2024.11.20.624438v1.abstract)

---

