## Microbiome-derived inosine modulates response to checkpoint inhibitor immunotherapy.
 18.09.2020, _LF Mager, R Burkhard, N Pett, NCA Cooke, K Brown, H Ramay, S Paik, J Stagg, RA Groves, M Gallo, IA Lewis, MB Geuking, KD McCoy_


Several species of intestinal bacteria have been associated with enhanced efficacy of checkpoint blockade immunotherapy, but the underlying mechanisms by which the microbiome enhances antitumor immunity are unclear. In this study, we isolated three bacterial species-Bifidobacterium pseudolongum, Lactobacillus johnsonii, and Olsenella species-that significantly enhanced efficacy of immune checkpoint inhibitors in four mouse models of cancer. We found that intestinal B. pseudolongum modulated enhanced immunotherapy response through production of the metabolite inosine. Decreased gut barrier function induced by immunotherapy increased systemic translocation of inosine and activated antitumor T cells. The effect of inosine was dependent on T cell expression of the adenosine A2A receptor and required costimulation. Collectively, our study identifies a previously unknown microbial metabolite immune pathway activated by immunotherapy that may be exploited to develop microbial-based adjuvant therapies.

[Source](https://science.sciencemag.org/content/369/6510/1481)

10.1126/science.abc3421

---

## Messengers from the microbiota.
 18.09.2020, _FY Shaikh, CL Sears_



[Source](https://doi.org/10.1126/science.abe0709)

10.1126/science.abe0709

---

## Diet posttranslationally modifies the mouse gut microbial proteome to modulate renal function.
 18.09.2020, _L Lobel, YG Cao, K Fenn, JN Glickman, WS Garrett_


Associations between chronic kidney disease (CKD) and the gut microbiota have been postulated, yet questions remain about the underlying mechanisms. In humans, dietary protein increases gut bacterial production of hydrogen sulfide (H2S), indole, and indoxyl sulfate. The latter are uremic toxins, and H2S has diverse physiological functions, some of which are mediated by posttranslational modification. In a mouse model of CKD, we found that a high sulfur amino acid-containing diet resulted in posttranslationally modified microbial tryptophanase activity. This reduced uremic toxin-producing activity and ameliorated progression to CKD in the mice. Thus, diet can tune microbiota function to support healthy host physiology through posttranslational modification without altering microbial community composition.

[Source](https://science.sciencemag.org/content/369/6510/1518)

10.1126/science.abb3763

---

## Diet posttranslationally modifies the mouse gut microbial proteome to modulate renal function.
 18.09.2020, _L Lobel, YG Cao, K Fenn, JN Glickman, WS Garrett_


Associations between chronic kidney disease (CKD) and the gut microbiota have been postulated, yet questions remain about the underlying mechanisms. In humans, dietary protein increases gut bacterial production of hydrogen sulfide (H2S), indole, and indoxyl sulfate. The latter are uremic toxins, and H2S has diverse physiological functions, some of which are mediated by posttranslational modification. In a mouse model of CKD, we found that a high sulfur amino acid-containing diet resulted in posttranslationally modified microbial tryptophanase activity. This reduced uremic toxin-producing activity and ameliorated progression to CKD in the mice. Thus, diet can tune microbiota function to support healthy host physiology through posttranslational modification without altering microbial community composition.

[Source](https://doi.org/10.1126/science.abd8344)

10.1126/science.abb3763

---

## The gut microbiota in kidney disease

[Source](https://doi.org/10.1126/science.abd8344)

---

## Fecal microbiota transplantation in inflammatory bowel disease patients: A systematic review and meta-analysis.
 18.09.2020, _LF Caldeira, HH Borba, FS Tonin, A Wiens, F Fernandez-Llimos, R Pontarolo_


Current evidence on fecal microbiota transplantation for inflammatory bowel disease is inconclusive. We conducted a systematic review to gather evidence on the efficacy and safety of fecal microbiota transplantation for inflammatory bowel disease.

[Source](https://doi.org/10.1371/journal.pone.0238910)

10.1371/journal.pone.0238910

---

## Antimicrobial peptides in the gut-brain axis: A straightforward review to unravel some missing links.
 18.09.2020, _F Pinheiro da Silva, E Vinicius Macarini Bruzaferro, N Olsen Saraiva Câmara_


Antimicrobial peptides (AMPs) are intriguing molecules, able to directly kill several microorganisms and to regulate multiple aspects of the immune response. Despite the extensive studies on the role of AMPs in the epithelial barrier, placing them as a pivotal line of defense against pathogen invasion, little attention has been directed to their role in the maintenance and modulation of the gut microbiota and, by consequence, of the homeostasis of extra intestinal tissues. Here, we review the recent literature about the microbiome-gut-brain axis, focusing on the role of AMPs in this scenario. We provide a straightforward revision of current data in order to provide an overview of the subject, discussing more in depth some points that, in our opinion, are crucial and have received little attention.

[Source](https://doi.org/10.1002/jnr.24729)

10.1002/jnr.24729

---

## Microbiome analysis of contact lens care solutions and tear fluids of contact lens wearers: Possible involvement of streptococcal antigens in allergic symptoms related to contact lens wear.
 18.09.2020, _F Hotta, H Eguchi, H Nakayama-Imaohji, T Kuwahara, A Tada, H Yagi, Y Shimomura, S Kusaka_


The present study elucidated the pathogenesis of allergic symptoms (AS) related to contact lens (CL) wear by assaying CL care solutions in lens storage cases and tears from subjects with AS using molecular biology techniques. A total of 15 CL storage cases were collected from subjects with AS (n=9) and healthy, asymptomatic control CL wearers (n=6). Bacterial populations in CL care solutions and tears were assayed by culture and 16S rDNA sequencing. Histamine levels in tears were measured by high‑performance liquid chromatography. Western blot analysis was performed to identify the bacteria recognized by tear IgE from subjects with AS. No significant differences were found in the culture results between the subjects with AS and asymptomatic subjects. Histamine was detected in 2 subjects with AS. Meta‑16S rDNA sequencing identified a cluster of 4 subjects with AS that were distinguished from others by principal coordinate analysis. Detailed population analysis revealed that the abundance of Gram‑positive bacteria in the microbiomes of CL care solutions used by the subjects with AS were higher than those of asymptomatic subjects (42.24±9.47 vs. 16.85±22.76% abundance). Among these, Streptococcus was the dominant genus (12.1‑18.3% abundance). Tear microbiome analysis revealed that the abundance of Streptococcus in the subjects with AS was significantly higher than that in other subjects (19.02±5.50 vs. 3.08±3.35%, P&lt;0.01). Western blot analysis demonstrated that the tear IgE in all subjects with AS reacted with Streptococcus (100%), but not with Staphylococcus. On the whole, these results provide novel insight into the pathogenesis of AS and identify Streptococcus as an important factor in AS associated with CL wear.

[Source](https://doi.org/10.3892/ijmm.2020.4678)

10.3892/ijmm.2020.4678

---

## Longitudinal Changes in Skin Microbiome and Microbes Associated with Change in Skin Status in Patients with Psoriasis.
 18.09.2020, _H Wang, MWM Chan, HH Chan, H Pang_


The aim of this study was to identify key microbes associated with change in skin status in patients with psoriasis over time. Longitudinal changes in the skin microbiome between patients with psoriasis and healthy family controls living in the same household were studied using whole genome metagenomic shotgun sequencing at 4 time-points. There were significant changes in abundance of the pathogen Campylobacter jejuni and its higher taxonomic levels when the skin status of patients with psoriasis changed. There were significant longitudinal variations in alpha and beta diversity of the skin microbiota in patients with psoriasis, but not in the control group. These results indicate a role for a balanced and dynamically stable skin microbiome in healthy skin, and will serve as the basis for future temporal studies of the skin microbiome and probiotic therapeutics.

[Source](https://doi.org/10.2340/00015555-3638)

10.2340/00015555-3638

---

## Changes in fecal short-chain fatty acids following fecal microbiota transplantation in patients with irritable bowel syndrome.
 18.09.2020, _M El-Salhy, J Valeur, T Hausken, J Gunnar Hatlebakk_


Short-chain fatty acids (SCFAs) may play a role in the pathophysiology of irritable bowel syndrome (IBS). This study analyzed fecal SCFAs after performing fecal microbiota transplantation (FMT) in the IBS patients who were included in our previous study of the efficacy of FMT.

[Source](https://doi.org/10.1111/nmo.13983)

10.1111/nmo.13983

---

## Analysis of airway microbiota in adults from a Brazilian cystic fibrosis center.
 18.09.2020, _CCF Leite, FAD de Freitas, M de Cássia Firmida, RS Leão, RM Albano, EA Marques_


The application of next-generation sequencing tools revealed that the cystic fibrosis respiratory tract is a polymicrobial environment. We have characterized the airway bacterial microbiota of five adult patients with cystic fibrosis during a 14-month period by 16S rRNA tag sequencing using the Illumina technology. Microbial diversity, estimated by the Shannon index, varied among patient samples collected throughout the follow-up period. The beta diversity analysis revealed that the composition of the airway microbiota was highly specific for each patient, showing little variation among the samples of each patient analyzed over time. The composition of the bacterial microbiota did not reveal any emerging pathogen predictor of pulmonary disease in cystic fibrosis or of its unfavorable clinical progress, except for unveiling the presence of anaerobic microorganisms, even without any established clinical association. Our results could potentialy help us to translate and develop strategies in response to the pathobiology of this disease, particularly because it represents an innovative approach for CF centers in Brazil.

[Source](https://doi.org/10.1007/s42770-020-00381-3)

10.1007/s42770-020-00381-3

---

## Bacterial diversity and inflammatory response at first-time visit in younger and older individuals with diabetic foot infection (DFI).
 18.09.2020, _S Dörr, F Freier, M Schlecht, R Lobmann_


Infection of a diabetic foot ulcer (DFU) is common. More than the half of DFUs become infected and 15-20% of them necessitate amputation in course of treatment. Diabetic foot infection (DFI) is therefore the major cause for non-traumatic lower limb amputation in Germany. Prompt and effective treatment of DFI is mandatory to safe limbs and lives. We investigated if there are relevant differences in evoked inflammatory response between different species and age-separated groups. We further investigated if there is an impact of ulcer localization on bacterial diversity.

[Source](https://doi.org/10.1007/s00592-020-01587-5)

10.1007/s00592-020-01587-5

---

## Harnessing the strategy of metagenomics for exploring the intestinal microecology of sable (Martes zibellina), the national first-level protected animal.
 18.09.2020, _J Yan, X Wu, J Chen, Y Chen, H Zhang_


Sable (Martes zibellina), a member of family Mustelidae, order Carnivora, is primarily distributed in the cold northern zone of Eurasia. The purpose of this study was to explore the intestinal flora of the sable by metagenomic library-based techniques. Libraries were sequenced on an Illumina HiSeq 4000 instrument. The effective sequencing data of each sample was above 6000 M, and the ratio of clean reads to raw reads was over 98%. The total ORF length was approximately 603,031, equivalent to 347.36 Mbp. We investigated gene functions with the KEGG database and identified 7140 KEGG ortholog (KO) groups comprising 129,788 genes across all of the samples. We selected a subset of genes with the highest abundances to construct cluster heat maps. From the results of the KEGG metabolic pathway annotations, we acquired information on gene functions, as represented by the categories of metabolism, environmental information processing, genetic information processing, cellular processes and organismal systems. We then investigated gene function with the CAZy database and identified functional carbohydrate hydrolases corresponding to genes in the intestinal microorganisms of sable. This finding is consistent with the fact that the sable is adapted to cold environments and requires a large amount of energy to maintain its metabolic activity. We also investigated gene functions with the eggNOG database; the main functions of genes included gene duplication, recombination and repair, transport and metabolism of amino acids, and transport and metabolism of carbohydrates. In this study, we attempted to identify the complex structure of the microbial population of sable based on metagenomic sequencing methods, which use whole metagenomic data, and to map the obtained sequences to known genes or pathways in existing databases, such as CAZy, KEGG, and eggNOG. We then explored the genetic composition and functional diversity of the microbial community based on the mapped functional categories.

[Source](https://doi.org/10.1186/s13568-020-01103-6)

10.1186/s13568-020-01103-6

---

## The Gut Microbial Community Structure of the North American River Otter (Lontra canadensis) in the Alberta Oil Sands Region in Canada: Relationship with Local Environmental Variables and Metal Body Burden.
 18.09.2020, _G Guo, KM Eccles, M McMillan, PJ Thomas, HM Chan, AJ Poulain_


The Alberta Oil Sands Region (AOSR) in Canada is home to one of the largest oil bitumen deposits in the world. Being a top predator with a small home range and sensitive to disturbances, the North American river otter (Lontra canadensis) has been designated as a sentinel species for the potential impacts of the natural resource exploitation on freshwater ecosystems in the AOSR. With an increasing interest in non-invasive biomarkers, recent work suggests that gut microbiota can be used as a potential biomarker of early biological effect on aquatic wildlife. The goal of the present study was to determine the river otter gut microbial structure related to environmental variables characterizing mining activities and metal body burden. We obtained 18 trapped animals from and surrounding the surface mineable area of the AOSR. The gut microbial community structure was characterized using high-throughput sequencing of 16S rRNA gene amplicon analyses. Trace metals concentrations in the liver were measured by ICP-MS. Our study revealed that the gut bacteria of river otters in the AOSR clustered in four groups dominated by Peptostreptococcaceae, Carnobacteriaceae, Enterobacteriaceae, Clostridiaceae and Nostocaceae, respectively. We show that As, Ba, Rb, liver-body weight ratio and δ15 N were associated with each cluster. When comparing affected vs. less affected sites, we showed that river otter gut bacterial community and structure was significantly related to trophic level of the river otter but not to ASOR mining activities. Our study reveals that the gut bacterial dynamics can provide insights into the diet and habitat use of river otters but that more work is needed to use it as a pollution biomarker. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1002/etc.4876)

10.1002/etc.4876

---

## Dynamic changes of the fecal bacterial community in dairy cows during early lactation.
 18.09.2020, _S Huang, S Ji, F Wang, J Huang, GM Alugongo, S Li_


The dynamics of the community structure and composition of the dairy cow fecal bacterial communities during early lactation is unclear, therefore this study was conducted to characterize the fecal bacterial communities in dairy cows during early lactation using 16S rRNA gene sequencing. Feces were sampled from 20 healthy fresh Holstein dairy cows on day 1 (Fresh1d group) and day 14 (Fresh14d group) after calving. After calving, cows were fed the same fresh diet. The dominant phyla Firmicutes and Proteobacteria were decreased (P ≤ 0.01) with lactating progress and phyla Bacteroidetes were increased (P = 0.008) with lactating progress and dietary transition. At family level, the predominant families were Ruminococcaceae (35.23%), Lachnospiraceae (11.46%), Rikenellaceae (10.44%) and Prevotellaceae (6.89%). A total of 14 genera were different between fecal samples from Fresh1d and Fresh14d, included the predominant genera, such as Ruminococcaceae_UCG-005 (P = 0.008), Rikenellaceae_RC9_gut_group (P = 0.043) and Christensenellaceae_R-7_group (P = 0.008). All fecal bacterial communities shared members of the genera Ruminococcaceae_UCG-005, Bacteroides and Rikenellaceae_RC9_gut_group. These findings help to improve our understanding of the composition and structure of the fecal microbial community in fresh cows and may provide insight into bacterial adaptation time and dietary in lactating cows.

[Source](https://doi.org/10.1186/s13568-020-01106-3)

10.1186/s13568-020-01106-3

---

## Comparative analysis of the pulmonary microbiome in healthy and diseased pigs.
 18.09.2020, _Z Li, X Wang, D Di, R Pan, Y Gao, C Xiao, B Li, J Wei, K Liu, Y Qiu, Z Ma_


The lungs possess an effective antimicrobial system and a strong ability to eliminate microorganisms in healthy organisms, and were once considered sterile. With the development of culture-independent sequencing technology, the richness and diversity of porcine lung microbiota have been gaining attention. In order to study the relationship between lung microbiota and porcine respiratory disease complex (PRDC), the lung microbiota in healthy and diseased swine bronchoalveolar lavage fluids were analyzed and compared using the Illumina MiSeq sequencing platform. The predominant microbial communities of healthy and diseased swine were similar at the phylum level, mainly composed of Proteobacteria, Firmicutes, Tenericutes, and Bacteroidetes. However, the bacterial taxonomic communities of healthy and diseased swine differed at the genus level. The higher relative abundances of Lactococcus, Enterococcus, Staphylococcus, and Lactobacillus genera in healthy swine might provide more benefits for lung health, while the enhanced richness of Streptococcus, Haemophilus, Pasteurella, and Bordetella genera in diseased swine might be closely related to pathogen invasion and the occurrence of respiratory disease. In conclusion, the observed differences in the richness and diversity of lung microbiota can provide novel insights into their relationship with PRDC. Analyses of swine lung microbiota communities might produce an effective strategy for the control and prevention of respiratory tract infections.

[Source](https://doi.org/10.1007/s00438-020-01722-5)

10.1007/s00438-020-01722-5

---

## Functional gene categories differentiate maize leaf drought-related microbial epiphytic communities.
 18.09.2020, _BA Methe, D Hiltbrand, J Roach, W Xu, SG Gordon, BW Goodner, AE Stapleton_


The phyllosphere epiphytic microbiome is composed of microorganisms that colonize the external aerial portions of plants. Relationships of plant responses to specific microorganisms-both pathogenic and beneficial-have been examined, but the phyllosphere microbiome functional and metabolic profile responses are not well described. Changing crop growth conditions, such as increased drought, can have profound impacts on crop productivity. Also, epiphytic microbial communities provide a new target for crop yield optimization. We compared Zea mays leaf microbiomes collected under drought and well-watered conditions by examining functional gene annotation patterns across three physically disparate locations each with and without drought treatment, through the application of short read metagenomic sequencing. Drought samples exhibited different functional sequence compositions at each of the three field sites. Maize phyllosphere functional profiles revealed a wide variety of metabolic and regulatory processes that differed in drought and normal water conditions and provide key baseline information for future selective breeding.

[Source](https://doi.org/10.1371/journal.pone.0237493)

10.1371/journal.pone.0237493

---

## Soy bioactive peptides and the gut microbiota modulation.
 18.09.2020, _TJ Ashaolu_


The balance of protein, carbohydrate, and fat affect the composition and functions of the gut microbiota. The complexities involved thereof require insights into the roles and impacts of individual dietary components due to the difficulty of defining such in a group of others. Peptides and proteins from several animal and plant sources have been widely explored in relation to the gut microbiome modulation, but the effects of soy peptides and other soy derivatives on the gut microbiota are largely unexplored. This piece considered an overview of the production and interventions of soy bioactive peptides on gut, as they affect the composition and functions of the gut microorganisms. A mini review on the production of soy protein hydrolysates/peptides and highlights of the most recent knowledge regarding their physiological effects on host's gut microbiota cum health were investigated. Overall deductions and research gaps were critically evaluated for futuristic interventions and relevance. Key points • Diet affects the composition of gut microorganisms. • Modulation of the gut microbiota by soy biopeptides is described. • Critical deductions on personal and commercial use are provided.

[Source](https://doi.org/10.1007/s00253-020-10799-2)

10.1007/s00253-020-10799-2

---

