## Maternal antibiotic prophylaxis during cesarean section has a limited impact on the infant gut microbiome

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(24)00268-3)

---

## Microbial assimilatory sulfate reduction-mediated H2S: an overlooked role in Crohn's disease development.
 19.08.2024, _W Luo, M Zhao, M Dwidar, Y Gao, L Xiang, X Wu, MH Medema, S Xu, X Li, H Schäfer, M Chen, R Feng, Y Zhu_


_Crohn Disease, Humans, Gastrointestinal Microbiome, Hydrogen Sulfide, Animals, Mice, Sulfates, Escherichia coli, Feces, Dysbiosis, Colon, Metagenomics, Oxidation-Reduction, Disease Models, Animal, Female_
H2S imbalances in the intestinal tract trigger Crohn's disease (CD), a chronic inflammatory gastrointestinal disorder characterized by microbiota dysbiosis and barrier dysfunction. However, a comprehensive understanding of H2S generation in the gut, and the contributions of both microbiota and host to systemic H2S levels in CD, remain to be elucidated. This investigation aimed to enhance comprehension regarding the sulfidogenic potential of both the human host and the gut microbiota.

[Source](https://doi.org/10.1186/s40168-024-01873-2)

10.1186/s40168-024-01873-2

---

## Characterization of the vaginal microbiota in Italian women with endometriosis: preliminary study.
 16.08.2024, _R Sessa, S Filardo, MF Viscardi, G Brandolino, L Muzii, M Di Pietro, MG Porpora_


This cross-sectional study aims to assess the interplay between the vaginal microbiota and endometriosis.

[Source](https://doi.org/10.1007/s00404-024-07631-x)

10.1007/s00404-024-07631-x

---

## Profiles and interactions of gut microbiome and intestinal microRNAs in pediatric Crohn's disease.
 16.08.2024, _Y Lv, C Zhen, A Liu, Y Hu, G Yang, C Xu, Y Lou, Q Cheng, Y Luo, J Yu, Y Fang, H Zhao, K Peng, Y Yu, J Lou, J Chen, Y Ni_


Gut dysbiosis is closely related to dysregulated microRNAs (miRNAs) in the intestinal epithelial cells, which plays an important role in the pathogenesis of Crohn's disease (CD). We investigated the relationship between fecal gut microbiome (GM) and intestinal tissue miRNAs in different stages of pediatric CD. Metagenomic analysis and miRNA sequencing were conducted to examine the GM and intestinal miRNA profiles of CD patients before and after clinical induction therapy and the controls. Twenty-seven newly diagnosed, therapy-naïve pediatric patients with active CD and 11 non-inflammatory bowel disease (IBD) controls were recruited in this study. Among CD patients, 11 patients completed induction treatment and reached clinical remission. Both GM and miRNA profiles were significantly changed between CD patients and controls. Seven key bacteria were identified at species level including Defluviitalea raffinosedens, Thermotalea metallivorans, Roseburia intestinalis, Dorea sp. AGR2135, Escherichia coli, Shigella sonnei, and Salmonella enterica, the exact proportions of which were further validated by real-time quantitative PCR analysis. Eight key miRNAs were also identified including hsa-miR-215-5p, hsa-miR-194-5p, hsa-miR-12135, hsa-miR-509-3-5p, hsa-miR-212-5p, hsa-miR-4448, hsa-miR-501-3p, and hsa-miR-503-5p. The functional enrichment analysis of differential miRNAs indicated the significantly altered cyclin protein, cyclin-dependent protein, and cell cycle pathway. The close interactions between seven key bacteria and eight key miRNAs were further investigated by miRNA target prediction. The association between specific miRNA expressions and key gut bacteria at different stages of CD supported their important roles as potential molecular biomarkers. Understanding the relationship between them will help us to explore the molecular mechanisms of CD.

[Source](https://doi.org/10.1128/msystems.00783-24)

10.1128/msystems.00783-24

---

## Regional variation and adaptive evolution in <em>Bifidobacterium pseudocatenulatum</em>: Insights into genomic and functional diversity in human gut

[Source](https://doi.org/10.1016/j.foodres.2024.114840)

---

## Effects of physical exercise on <em>Akkermansia muciniphila</em>: a systematic review of human and animal studies

[Source](https://doi.org/10.1163/18762891-bja00031)

---

## Microbiome modification for personalized treatment of dysbiotic diseases.
 20.08.2024, _RB Sartor_


_Humans, Fecal Microbiota Transplantation, Dysbiosis, Precision Medicine, Gastrointestinal Microbiome, Clostridium Infections, Inflammation, Clostridioides difficile, Biomarkers_
Fecal microbial transplantation (FMT) for inflammatory diseases or refractory immune checkpoint inhibitor therapy is less effective than for preventing recurrent Clostridioides difficile infection. This commentary outlines strategies to use biomarkers of successful FMT to guide newer approaches to restore microbial homeostasis in individuals with dysbiosis-mediated inflammation.

[Source](https://www.cell.com/cell-host-microbe/abstract/S1931-3128(24)00282-8)

10.1016/j.chom.2024.07.023

---

## Combining Gut Microbiota Modulation and Enzymatic-Triggered Colonic Delivery by Prebiotic Nanoparticles Improves Mouse Colitis Therapy.
 15.08.2024, _H Li, Y Cheng, L Cui, Z Yang, J Wang, Z Zhang, K Chen, C Zhao, N He, S Li_


The efficacy of ulcerative colitis (UC) therapy is closely connected to the composition of gut microbiota in the gastrointestinal tract. Prebiotic-based nanoparticles (NPs) provide a more precise approach to alleviate UC via modulating gut microbiota dysbiosis. The present study develops an efficient prebiotic-based colon-targeted drug delivery system (PCDDS) by using prebiotic pectin (Pcn) and chitosan (Csn) polysaccharides as a prebiotic shell, with the anti-inflammatory drug sulfasalazine (SAS) loaded into a poly(lactic-co-glycolic acid) (PLGA) core to construct SAS@PLGA-Csn-Pcn NPs. Then, we examine its characterization, cellular uptake, and in vivo therapeutic efficacy. The results of our study indicate that the Pcn/Csn shell confers efficient pH-sensitivity properties. The gut microbiota-secreted pectinase serves as the trigger agent for Pcn/Csn shell degradation, and the resulting Pcn oligosaccharides possess a substantial prebiotic property. Meanwhile, the formed PCDDSs exhibit robust biodistribution and accumulation in the colon tissue, rapid cellular uptake, efficient in vivo therapeutic efficacy, and modulation of gut microbiota dysbiosis in a mouse colitis model. Collectively, our synthetic PCDDSs demonstrate a promising and synergistic strategy for UC therapy.

[Source](https://doi.org/10.34133/bmr.0062)

10.34133/bmr.0062

---

## Effect of chronic alcohol consumption on oral microbiota in rats with periodontitis.
 17.08.2024, _Z Zhao, X Zhang, W Zhao, J Wang, Y Peng, X Liu, N Liu, Q Liu_


_Animals, Rats, Wistar, Male, Periodontitis, Microbiota, Rats, RNA, Ribosomal, 16S, Mouth, Alcohol Drinking, Disease Models, Animal_
The imbalance of oral microbiota can contribute to various oral disorders and potentially impact general health. Chronic alcohol consumption beyond a certain threshold has been implicated in influencing both the onset and progression of periodontitis. However, the mechanism by which chronic alcohol consumption affects periodontitis and its association with changes in the oral microbial community remains unclear.

[Source](https://doi.org/10.7717/peerj.17795)

10.7717/peerj.17795

---

## Fecal microbiota transplantation accelerates restoration of florfenicol-disturbed intestinal microbiota in a fish model

[Source](https://doi.org/10.1038/s42003-024-06727-z)

---

## Structural, Biochemical, and Phylogenetic Analysis of Bacterial and Fungal Carbohydrate Esterase Family 15 Glucuronoyl Esterases in the Rumen.
 17.08.2024, _RJ Gruninger, M Kevorkova, KE Low, DR Jones, L Worrall, TA McAllister, DW Abbott_


Glucuronoyl esterases (GEs) are carbohydrate active enzymes in carbohydrate esterase family 15 which are involved in the hydrolysis of lignin-carbohydrate complexes. They are encoded by a wide range of aerobic and anaerobic fungi and bacteria inhabiting diverse environments. The rumen microbiome is a complex microbial community with a wide array of enzymes that specialize in deconstructing plant cell wall carbohydrates. Enzymes from the rumen tend to show low similarity to homologues found in other environments, making the rumen microbiome a promising source for the discovery of novel enzymes. Using a combination of phylogenetic and structural analysis, we investigated the structure-function relationship of GEs from the rumen bacteria Fibrobacter succinogenes and Ruminococcus flavefaciens, and from the rumen fungus, Piromyces rhizinflata. All adopt a canonical α/β hydrolase fold and possess a structurally conserved Ser-His-Glu/Asp catalytic triad. Structural variations in the enzymes are localized to loops surrounding the active site. Analysis of the active site structures in these enzymes emphasized the importance of structural plasticity in GEs with non-canonical active site conformations. We hypothesize that interkingdom HGT events may have contributed to the diversity of GEs in the rumen, and this is demonstrated by the phylogenetic and structural similarity observed between rumen bacterial and fungal GEs. This study advances our understanding of the structure-function relationship in glucuronoyl esterases and illuminates the evolutionary dynamics that contribute to enzyme diversity in the rumen microbiome.

[Source](https://doi.org/10.1007/s10930-024-10221-0)

10.1007/s10930-024-10221-0

---

## Harnessing the plant microbiome for sustainable crop production.
 15.08.2024, _S Compant, F Cassan, T Kostić, L Johnson, G Brader, F Trognitz, A Sessitsch_


Global research on the plant microbiome has enhanced our understanding of the complex interactions between plants and microorganisms. The structure and functions of plant-associated microorganisms, as well as the genetic, biochemical, physical and metabolic factors that influence the beneficial traits of plant microbiota have also been intensively studied. Harnessing the plant microbiome has led to the development of various microbial applications to improve crop productivity in the face of a range of challenges, for example, climate change, abiotic and biotic stresses, and declining soil properties. Microorganisms, particularly nitrogen-fixing rhizobia as well as mycorrhizae and biocontrol agents, have been applied for decades to improve plant nutrition and health. Still, there are limitations regarding efficacy and consistency under field conditions. Also, the wealth of expanding knowledge on microbiome diversity, functions and interactions represents a huge source of information to exploit for new types of application. In this Review, we explore plant microbiome functions, mechanisms, assembly and types of interaction, and discuss current applications and their pitfalls. Furthermore, we elaborate on how the latest findings in plant microbiome research may lead to the development of new or more advanced applications. Finally, we discuss research gaps to fully leverage microbiome functions for sustainable plant production.

[Source](https://doi.org/10.1038/s41579-024-01079-1)

10.1038/s41579-024-01079-1

---

## The impact of filamentous plant pathogens on the host microbiota.
 18.08.2024, _VM Flores-Nunez, EH Stukenbrock_


_Microbiota, Plant Diseases, Host-Pathogen Interactions, Plants, Fungi_
When a pathogen invades a plant, it encounters a diverse microbiota with some members contributing to the health and growth of the plant host. So far, the relevance of interactions between pathogens and the plant microbiota are poorly understood; however, new lines of evidence suggest that pathogens play an important role in shaping the microbiome of their host during invasion. This review aims to summarize recent findings that document changes in microbial community composition during the invasion of filamentous pathogens in plant tissues. We explore the known mechanisms of interaction between plant pathogens and the host microbiota that underlie these changes, particularly the pathogen-encoded traits that are produced to target specific microbes. Moreover, we discuss the limitations of current strategies and shed light on new perspectives to study the complex interaction networks between filamentous pathogens and the plant microbiome.

[Source](https://doi.org/10.1186/s12915-024-01965-3)

10.1186/s12915-024-01965-3

---

## Metagenomic insights into the prokaryotic communities of heavy metal-contaminated hypersaline soils.
 16.08.2024, _C Galisteo, F Puente-Sánchez, RR de la Haba, S Bertilsson, C Sánchez-Porro, A Ventosa_


Saline soils and their microbial communities have recently been studied in response to ongoing desertification of agricultural soils caused by anthropogenic impacts and climate change. Here we describe the prokaryotic microbiota of hypersaline soils in the Odiel Saltmarshes Natural Area of Southwest Spain. This region has been strongly affected by mining and industrial activity and feature high levels of certain heavy metals. We sequenced 18 shotgun metagenomes through Illumina NovaSeq from samples obtained from three different areas in 2020 and 2021. Taxogenomic analyses demonstrate that these soils harbored equal proportions of archaea and bacteria, with Methanobacteriota, Pseudomonadota, Bacteroidota, Gemmatimonadota, and Balneolota as most abundant phyla. Functions related to the transport of heavy metal outside the cytoplasm are among the most relevant features of the community (i.e., ZntA and CopA enzymes). They seem to be indispensable to avoid the increase of zinc and copper concentration inside the cell. Besides, the archaeal phylum Methanobacteriota is the main arsenic detoxifier within the microbiota although arsenic related genes are widely distributed in the community. Regarding the osmoregulation strategies, "salt-out" mechanism was identified in part of the bacterial population, whereas "salt-in" mechanism was present in both domains, Bacteria and Archaea. De novo biosynthesis of two of the most universal compatible solutes was detected, with predominance of glycine betaine biosynthesis (betAB genes) over ectoine (ectABC genes). Furthermore, doeABCD gene cluster related to the use of ectoine as carbon and energy source was solely identified in Pseudomonadota and Methanobacteriota.

[Source](https://doi.org/10.1016/j.scitotenv.2024.175497)

10.1016/j.scitotenv.2024.175497

---

## Ecological phage therapy: Can bacteriophages help rapidly restore the soil microbiome?
 16.08.2024, _T Davies, C Cando-Dumancela, C Liddicoat, R Dresken, RH Damen, RA Edwards, SA Ramesh, MF Breed_


Soil microbiota underpin ecosystem functionality yet are rarely targeted during ecosystem restoration. Soil microbiota recovery following native plant revegetation can take years to decades, while the effectiveness of soil inoculation treatments on microbiomes remains poorly explored. Therefore, innovative restoration treatments that target soil microbiota represent an opportunity to accelerate restoration outcomes. Here, we introduce the concept of ecological phage therapy-the application of phage for the targeted reduction of the most abundant and dominant bacterial taxa present in degraded ecosystems. We propose that naturally occurring bacteriophages-viruses that infect bacteria-could help rapidly shift soil microbiota towards target communities. Bacteriophages sculpt the microbiome by lysis of specific bacteria, and if followed by the addition of reference soil microbiota, such treatments could facilitate rapid reshaping of soil microbiota. Here, we experimentally tested this concept in a pilot study. We collected five replicate pre-treatment degraded soil samples, then three replicate soil samples 48 hours after phage, bacteria, and control treatments. Bacterial 16S rDNA sequencing showed that phage-treated soils had reduced bacterial diversity; however, when we combined ecological phage therapy with reference soil inoculation, we did not see a shift in soil bacterial community composition from degraded soil towards a reference-like community. Our pilot study provides early evidence that ecological phage therapy could help accelerate the reshaping of soil microbiota with the ultimate aim of reducing timeframes for ecosystem recovery. We recommend the next steps for ecological phage therapy be (a) developing appropriate risk assessment and management frameworks, and (b) focussing research effort on its practical application to maximise its accessibility to restoration practitioners.

[Source](https://doi.org/10.1002/ece3.70185)

10.1002/ece3.70185

---

## Do restoration strategies in mangroves recover microbial diversity? A case study in the Yucatan peninsula.
 18.08.2024, _D Esguerra-Rodríguez, A De León-Lorenzana, C Teutli, A Prieto-Davó, JQ García-Maldonado, J Herrera-Silveira, LI Falcón_


_Mexico, Biodiversity, Wetlands, Geologic Sediments, Microbiota, RNA, Ribosomal, 16S, Bacteria, Proteobacteria, Conservation of Natural Resources, Ecosystem_
Mangrove forests are fundamental coastal ecosystems for the variety of services they provide, including green-house gas regulation, coastal protection and home to a great biodiversity. Mexico is the fourth country with the largest extension of mangroves of which 60% occurs in the Yucatan Peninsula. Understanding the microbial component of mangrove forests is necessary for their critical roles in biogeochemical cycles, ecosystem health, function and restoration initiatives. Here we study the relation between the microbial community from sediments and the restoration process of mangrove forests, comparing conserved, degraded and restored mangroves along the northern coast of the Yucatan peninsula. Results showed that although each sampling site had a differentiated microbial composition, the taxa belonged predominantly to Proteobacteria (13.2-23.6%), Desulfobacterota (7.6-8.3%) and Chloroflexi (9-15.7%) phyla, and these were similar between rainy and dry seasons. Conserved mangroves showed significantly higher diversity than degraded ones, and restored mangroves recovered their microbial diversity from the degraded state (Dunn test p-value Benjamini-Hochberg adjusted = 0.0034 and 0.0071 respectively). The structure of sediment microbial β-diversity responded significantly to the mangrove conservation status and physicochemical parameters (organic carbon content, redox potential, and salinity). Taxa within Chloroflexota, Desulfobacterota and Thermoplasmatota showed significantly higher abundance in degraded mangrove samples compared to conserved ones. This study can help set a baseline that includes the microbial component in health assessment and restoration strategies of mangrove forests.

[Source](https://doi.org/10.1371/journal.pone.0307929)

10.1371/journal.pone.0307929

---

## Metagenome sequencing and 107 microbial genomes from seamount sediments along the Yap and Mariana trenches.
 18.08.2024, _Y Zhang, H Jing_


_Metagenome, Geologic Sediments, Phylogeny, Bacteria, Archaea, Genome, Microbial, Genome, Archaeal, Genome, Bacterial_
Microbes in the sediments across a series of seamounts along the island arc of the Yap and Mariana trenches were investigated by metagenome. In this study, we reconstructed 107 metagenome-assembled genomes (MAGs), including 100 bacteria and 7 archaea. All the MAGs exhibited &gt;75% completeness and &lt;10% contamination, with 26 MAGs being classified as 'nearly complete' (completeness &gt;90%), while 50 falling within 80-90% range and 31 between 75-80% complete. Phylogenomic analysis revealed that 86% (n = 92) of these MAGs represented new taxa at different taxonomical levels. The species composition of these MAGs was most consistent with the previous reports, with the most abundant phyla being Proteobacteria (n = 39), Methylomirabilota (n = 27), and Nitrospirota (n = 7). These draft genomes provided novel data on species diversity and function in the seamount microbial community, which will provide reference data for extensive comparative genomic studies across crucial phylogenetic groups worldwide.

[Source](https://doi.org/10.1038/s41597-024-03762-7)

10.1038/s41597-024-03762-7

---

## Metrological evaluation of DNA extraction method effects on the bacterial microbiome and resistome in sputum.
 16.08.2024, _A Benčič, N Toplak, S Koren, A Bogožalec Košir, M Milavec, V Tomič, D Lužnik, T Dreo_


Targeted high-throughput sequencing (HTS) has revolutionized the way we look at bacterial communities. It can be used for the species-specific detection of bacteria as well as for the determination of the microbiome and resistome and can be applied to samples from almost any environment. However, the results of targeted HTS can be influenced by many factors, which poses a major challenge for its use in clinical diagnostics. In this study, we investigated the impact of the DNA extraction method on the determination of the bacterial microbiome and resistome by targeted HTS using principles from metrology and diagnostics such as repeatability and analytical sensitivity. Sputum samples spiked with Acinetobacter baumannii, Klebsiella pneumoniae, and Pseudomonas aeruginosa at three different concentrations (103-106 cells/mL) were used. DNA was extracted from each sample on 2 separate days in three replicates each using three different extraction methods based on cetrimonium bromide, magnetic beads, and silica membranes. All three spiked bacteria were detected in sputum, and the DNA extraction method had no significant effect on detection. However, the DNA extraction method had significant effects on the composition of the microbiome and the resistome. The sequencing results were repeatable in the majority of cases. The silica membrane-based DNA extraction kit provided the most repeatable results and the highest diversity of the microbiome and resistome. Targeted HTS has been shown to be a reliable tool for determining the microbiome and resistome; however, the method of DNA extraction should be carefully selected to minimize its impact on the results.

[Source](https://doi.org/10.1128/msystems.00735-24)

10.1128/msystems.00735-24

---

## AI in microbiome research: Where have we been, where are we going?
 15.08.2024, _GK Gerber_


_Artificial Intelligence, Humans, Microbiota, Software_
Artificial intelligence (AI), a subdiscipline of computer science that develops machines or software that mimic characteristically human cognitive functions, is currently undergoing a revolution. In this commentary article, I will give my perspective on the evolution of the field and my thoughts on how AI may impact microbiome research in the next decade.

[Source](https://www.cell.com/cell-host-microbe/abstract/S1931-3128(24)00280-4)

10.1016/j.chom.2024.07.021

---

## Modeling Microbial Communities: Perspective and Challenges.
 16.08.2024, _L Raajaraam, K Raman_


_Microbiota, Models, Biological, Machine Learning_
Microbial communities are immensely important due to their widespread presence and profound impact on various facets of life. Understanding these complex systems necessitates mathematical modeling, a powerful tool for simulating and predicting microbial community behavior. This review offers a critical analysis of metabolic modeling and highlights key areas that would greatly benefit from broader discussion and collaboration. Moreover, we explore the challenges and opportunities linked to the intricate nature of these communities, spanning data generation, modeling, and validation. We are confident that ongoing advancements in modeling techniques, such as machine learning, coupled with interdisciplinary collaborations, will unlock the full potential of microbial communities across diverse applications.

[Source](https://doi.org/10.1021/acssynbio.4c00116)

10.1021/acssynbio.4c00116

---

