## <em>Neisseria</em> species as pathobionts in bronchiectasis

[Source](https://doi.org/10.1016/j.chom.2022.08.005)

---

## The gut microbiome molecular complex in human health and disease.
 15.09.2022, _P Wilmes, C Martin-Gallausiaux, M Ostaszewski, VTE Aho, PV Novikova, CC Laczny, JG Schneider_


The human gut microbiome produces a functional complex of biomolecules, including nucleic acids, (poly)peptides, structural molecules, and metabolites. This impacts human physiology in multiple ways, especially by triggering inflammatory pathways in disease. At present, much remains to be learned about the identity of key effectors and their causal roles.

[Source](https://doi.org/10.1016/j.chom.2022.08.016)

10.1016/j.chom.2022.08.016

---

## Codiversification of gut microbiota with humans.
 15.09.2022, _TA Suzuki, JL Fitzstevens, VT Schmidt, H Enav, KE Huus, M Mbong Ngwese, A Grießhammer, A Pfleiderer, BR Adegbite, JF Zinsou, M Esen, TP Velavan, AA Adegnika, LH Song, TD Spector, AL Muehlbauer, N Marchi, H Kang, L Maier, R Blekhman, L Ségurel, G Ko, ND Youngblut, P Kremsner, RE Ley_


The gut microbiomes of human populations worldwide have many core microbial species in common. However, within a species, some strains can show remarkable population specificity. The question is whether such specificity arises from a shared evolutionary history (codiversification) between humans and their microbes. To test for codiversification of host and microbiota, we analyzed paired gut metagenomes and human genomes for 1225 individuals in Europe, Asia, and Africa, including mothers and their children. Between and within countries, a parallel evolutionary history was evident for humans and their gut microbes. Moreover, species displaying the strongest codiversification independently evolved traits characteristic of host dependency, including reduced genomes and oxygen and temperature sensitivity. These findings all point to the importance of understanding the potential role of population-specific microbial strains in microbiome-mediated disease phenotypes.

[Source](https://doi.org/10.1126/science.abm7759)

10.1126/science.abm7759

---

## Gut microbiome of multiple sclerosis patients and paired household healthy controls reveal associations with disease risk and course

[Source](https://doi.org/10.1016/j.cell.2022.08.021)

---

## Composition and diverse differences of intestinal microbiota in ulcerative colitis patients

[Source](https://doi.org/10.3389/fcimb.2022.953962)

---

## A systematic review on omics data (metagenomics, metatranscriptomics, and metabolomics) in the role of microbiome in gallbladder disease

[Source](https://doi.org/10.3389/fphys.2022.888233)

---

## Variability of strain engraftment and predictability of microbiome composition after fecal microbiota transplantation across different diseases.
 15.09.2022, _G Ianiro, M Punčochář, N Karcher, S Porcari, F Armanini, F Asnicar, F Beghini, A Blanco-Míguez, F Cumbo, P Manghi, F Pinto, L Masucci, G Quaranta, S De Giorgi, GD Sciumè, S Bibbò, F Del Chierico, L Putignani, M Sanguinetti, A Gasbarrini, M Valles-Colomer, G Cammarota, N Segata_


Fecal microbiota transplantation (FMT) is highly effective against recurrent Clostridioides difficile infection and is considered a promising treatment for other microbiome-related disorders, but a comprehensive understanding of microbial engraftment dynamics is lacking, which prevents informed applications of this therapeutic approach. Here, we performed an integrated shotgun metagenomic systematic meta-analysis of new and publicly available stool microbiomes collected from 226 triads of donors, pre-FMT recipients and post-FMT recipients across eight different disease types. By leveraging improved metagenomic strain-profiling to infer strain sharing, we found that recipients with higher donor strain engraftment were more likely to experience clinical success after FMT (P = 0.017) when evaluated across studies. Considering all cohorts, increased engraftment was noted in individuals receiving FMT from multiple routes (for example, both via capsules and colonoscopy during the same treatment) as well as in antibiotic-treated recipients with infectious diseases compared with antibiotic-naïve patients with noncommunicable diseases. Bacteroidetes and Actinobacteria species (including Bifidobacteria) displayed higher engraftment than Firmicutes except for six under-characterized Firmicutes species. Cross-dataset machine learning predicted the presence or absence of species in the post-FMT recipient at 0.77 average AUROC in leave-one-dataset-out evaluation, and highlighted the relevance of microbial abundance, prevalence and taxonomy to infer post-FMT species presence. By exploring the dynamics of microbiome engraftment after FMT and their association with clinical variables, our study uncovered species-specific engraftment patterns and presented machine learning models able to predict donors that might optimize post-FMT specific microbiome characteristics for disease-targeted FMT protocols.

[Source](https://doi.org/10.1038/s41591-022-01964-3)

10.1038/s41591-022-01964-3

---

## Drivers and determinants of strain dynamics following fecal microbiota transplantation.
 15.09.2022, _TSB Schmidt, SS Li, OM Maistrenko, W Akanni, LP Coelho, S Dolai, A Fullam, AM Glazek, R Hercog, H Herrema, F Jung, S Kandels, A Orakov, R Thielemann, M von Stetten, T Van Rossum, V Benes, TJ Borody, WM de Vos, CY Ponsioen, M Nieuwdorp, P Bork_


Fecal microbiota transplantation (FMT) is a therapeutic intervention for inflammatory diseases of the gastrointestinal tract, but its clinical mode of action and subsequent microbiome dynamics remain poorly understood. Here we analyzed metagenomes from 316 FMTs, sampled pre and post intervention, for the treatment of ten different disease indications. We quantified strain-level dynamics of 1,089 microbial species, complemented by 47,548 newly constructed metagenome-assembled genomes. Donor strain colonization and recipient strain resilience were mostly independent of clinical outcomes, but accurately predictable using LASSO-regularized regression models that accounted for host, microbiome and procedural variables. Recipient factors and donor-recipient complementarity, encompassing entire microbial communities to individual strains, were the main determinants of strain population dynamics, providing insights into the underlying processes that shape the post-FMT gut microbiome. Applying an ecology-based framework to our findings indicated parameters that may inform the development of more effective, targeted microbiome therapies in the future, and suggested how patient stratification can be used to enhance donor microbiota colonization or the displacement of recipient microbes in clinical practice.

[Source](https://doi.org/10.1038/s41591-022-01913-0)

10.1038/s41591-022-01913-0

---

## Long-term ecological and evolutionary dynamics in the gut microbiomes of carbapenemase-producing Enterobacteriaceae colonized subjects.
 15.09.2022, _JTL Kang, JJY Teo, D Bertrand, A Ng, A Ravikrishnan, M Yong, OT Ng, K Marimuthu, SL Chen, KR Chng, YH Gan, N Nagarajan_


Long-term colonization of the gut microbiome by carbapenemase-producing Enterobacteriaceae (CPE) is a growing area of public health concern as it can lead to community transmission and rapid increase in cases of life-threatening CPE infections. Here, leveraging the observation that many subjects are decolonized without interventions within a year, we used longitudinal shotgun metagenomics (up to 12 timepoints) for detailed characterization of ecological and evolutionary dynamics in the gut microbiome of a cohort of CPE-colonized subjects and family members (n = 46; 361 samples). Subjects who underwent decolonization exhibited a distinct ecological shift marked by recovery of microbial diversity, key commensals and anti-inflammatory pathways. In addition, colonization was marked by elevated but unstable Enterobacteriaceae abundances, which exhibited distinct strain-level dynamics for different species (Escherichia coli and Klebsiella pneumoniae). Finally, comparative analysis with whole-genome sequencing data from CPE isolates (n = 159) helped identify substrain variation in key functional genes and the presence of highly similar E. coli and K. pneumoniae strains with variable resistance profiles and plasmid sharing. These results provide an enhanced view into how colonization by multi-drug-resistant bacteria associates with altered gut ecology and can enable transfer of resistance genes, even in the absence of overt infection and antibiotic usage.

[Source](https://doi.org/10.1038/s41564-022-01221-w)

10.1038/s41564-022-01221-w

---

## Structural and functional dysbiosis of gut microbiota in Tibetan subjects with coronary heart disease.
 17.09.2022, _Q Cheng, C Fan, F Liu, Y Li, H Hou, Y Ma, Y Tan, Y Li, Y Hai, T Wu, L Zhang, Y Zhang_


The gut microbiota plays a crucial role in coronary heart disease (CHD). However, only a few studies focusing on the relationship between gut microbiota and CHD in ethnic populations are available. Here, we employed shotgun sequencing of the gut metagenome to analyze the taxonomic composition and functional annotation of the gut microbiota of 14 CHD patients, 13 patients with non-stenosis coronary heart disease (NCHD), and 18 healthy controls (HT) in Tibetan subjects. We found that the α-diversity of the gut microbiota was not significantly different among the three groups., whereas β-diversity was significantly altered in the CHD group compared with HT. Based on the receiver operating characteristic curve (ROC) analysis, the relative abundance of Proteobacteria species effectively distinguished patients with CHD from the control group. Most of the enriched species belonged to Proteobacteria. The pathways that contributed the most to the differences between groups were amino acid metabolism-related pathways, especially lysine biosynthesis. The enzymes of the lysine biosynthesis pathway, including K01714 and K00821, were significantly decreased in the CHD group. Our findings increase the understanding of the association between CHD pathogenesis and gut microbiota in the Tibetan population, thus paving the way for the development of improved diagnostic methods and treatments for Tibetan patients with CHD.

[Source](https://doi.org/10.1016/j.ygeno.2022.110483)

10.1016/j.ygeno.2022.110483

---

## Growth of succinate consumer <em>Dialister hominis</em> is supported by <em>Bacteroides thetaiotaomicron</em>

[Source](https://doi.org/10.1016/j.anaerobe.2022.102642)

---

## RhoB affects colitis through modulating cell signaling and intestinal microbiome.
 16.09.2022, _J Yang, G Pei, X Sun, Y Xiao, C Miao, L Zhou, B Wang, L Yang, M Yu, ZS Zhang, ET Keller, Z Yao, Q Wang_


The pathogenesis of inflammatory bowel diseases (IBD) is multifactorial, and diagnostic and treatment strategies for IBD remain to be developed. RhoB regulates multiple cell functions; however, its role in colitis is unexplored.

[Source](https://doi.org/10.1186/s40168-022-01347-3)

10.1186/s40168-022-01347-3

---

## Diet fuelling inflammatory bowel diseases: preclinical and clinical concepts.
 16.09.2022, _TE Adolph, J Zhang_


The diet and gut microbiota have been extensively interrogated as a fuel for gut inflammation in inflammatory bowel diseases (IBDs) in the last few years. Here, we review how specific nutrients, typically enriched in a Western diet, instigate or deteriorate experimental gut inflammation in a genetically susceptible host and we discuss microbiota-dependent and independent mechanisms. We depict the study landscape of nutritional trials in paediatric and adult IBD and delineate common grounds for dietary advice. Conclusively, the diet reflects a critical rheostat of microbial dysbiosis and gut inflammation in IBD. Dietary restriction by exclusive enteral nutrition, with or without a specific exclusion diet, is effectively treating paediatric Crohn's disease, while adult IBD trials are less conclusive. Insights into molecular mechanisms of nutritional therapy will change the perception of IBD and will allow us to enter the era of precision nutrition. To achieve this, we discuss the need for carefully designed nutritional trials with scientific rigour comparable to medical trials, which also requires action from stake holders. Establishing evidence-based dietary therapy for IBD does not only hold promise to avoid long-term immunosuppression, but to provide a widely accessible therapy at low cost. Identification of dietary culprits disturbing gut health also bears the potential to prevent IBD and allows informed decision making in food politics.

[Source](https://doi.org/10.1136/gutjnl-2021-326575)

10.1136/gutjnl-2021-326575

---

## Recent findings in Akkermansia muciniphila-regulated metabolism and its role in intestinal diseases.
 16.09.2022, _MJ Liu, JY Yang, ZH Yan, S Hu, JQ Li, ZX Xu, YP Jian_


The mammalian gastrointestinal tract is colonized with a majority of gut microbes, affecting host metabolism and homeostasis. Gut microbiota plays a vital role in nutrient exchange, signaling transduction between intestinal epithelial cells, and resistance to pathogen invasion. Gut microbiota is divided into mucus layer bacteria and intestinal lumen bacteria based on the colonization distribution. Akkermansia muciniphila (A. muciniphila) prefers to colonize in the intestinal mucus layer, and specifically degrades mucins to produce short-chain fatty acids, providing energy for the host and promoting colonization of the bacterium itself. Degradation of mucins prompts the host to compensate for the production of more mucins, thereby maintaining the dynamics of these proteins. In the intestinal micro-ecosystem, A. muciniphila is non-pathogenic, and its colonization with suitable abundance contributes to the development of immune system, thus promoting intestinal health. The mechanisms by which A. muciniphila bears a protective role in the host intestine are currently unclear. In this review, we summarize the microenvironment for the colonization of A. muciniphila, physiological characteristics and pathophysiological impact of A. muciniphila on intestinal diseases, such as irritable bowel syndrome, inflammatory bowel diseases, and intestinal tumors. We also provided updates for current studies on signals that A. muciniphila enhances intestinal barrier integrity and regulates immune response. Together, we conclude that A. muciniphila is a promising probiotic, which could be a microbial target for the treatment of multiple intestinal diseases.

[Source](https://doi.org/10.1016/j.clnu.2022.08.029)

10.1016/j.clnu.2022.08.029

---

## Recent findings in <em>Akkermansia muciniphila</em>-regulated metabolism and its role in intestinal diseases

[Source](https://doi.org/10.1016/j.clnu.2022.08.029)

---

## Comprehensive analysis of microbiome, metabolome and transcriptome revealed the mechanisms of <em>Moringa oleifera</em> polysaccharide on preventing ulcerative colitis

[Source](https://doi.org/10.1016/j.ijbiomac.2022.09.100)

---

## Probiotics ameliorates pulmonary inflammation via modulating gut microbiota and rectifying Th17/Treg imbalance in a rat model of PM2.5 induced lung injury

[Source](https://doi.org/10.1016/j.ecoenv.2022.114060)

---

## Short-term supplementation with ω-3 polyunsaturated fatty acids modulates primarily mucolytic species from the gut luminal mucin niche in a human fermentation system

[Source](https://doi.org/10.1080/19490976.2022.2120344)

---

## Interactive Effect of Combined Intermittent and Sustained Hypoxia and High-Fat Diet on the Colonic Mucosal Microbiome and Host Gene Expression in Mice.
 17.09.2022, _S Mashaqi, D Laubitz, EJD Morales, R De Armond, H Alameddin, FK Ghishan, PR Kiela, S Parthasarathy_


Gut dysbiosis can cause cardiometabolic disease. Gut dysbiosis can be independently caused by high-fat diet (HFD) and intermittent hypoxia (IH; characterizing obstructive sleep apnea), but the interactive effect of combined intermittent and sustained hypoxia (IH+SH) (characterizing obesity hypoventilation syndrome) and HFD on gut dysbiosis is unclear. We aimed to investigate the interactive effect of a combination of IH and SH and HFD on proximal colonic microbiota and colonic gene expression pattern.

[Source](https://doi.org/10.2147/NSS.S370957)

10.2147/NSS.S370957

---

## Fungi are more transient than bacteria in caterpillar gut microbiomes.
 16.09.2022, _M Šigut, P Pyszko, H Šigutová, D Višňovská, M Kostovčík, N Kotásková, O Dorňák, M Kolařík, P Drozd_


Despite an increasing number of studies on caterpillar (Insecta: Lepidoptera) gut microbiota, bacteria have been emphasized more than fungi. Therefore, we lack data on whether fungal microbiota is resident or transient and shaped by factors similar to those of bacteria. We sampled nine polyphagous caterpillar species from several tree species at multiple sites to determine the factors shaping leaf and gut bacterial and fungal microbiota as well as the extent to which caterpillars acquire microbiota from their diet. We performed 16S and ITS2 DNA metabarcoding of the leaves and guts to determine the composition and richness of the respective microbiota. While spatial variables shaped the bacterial and fungal microbiota of the leaves, they only affected fungi in the guts, whereas the bacteria were shaped primarily by caterpillar species, with some species harboring more specific bacterial consortia. Leaf and gut microbiota significantly differed; in bacteria, this difference was more pronounced. The quantitative similarity between leaves and guts significantly differed among caterpillar species in bacteria but not fungi, suggesting that some species have more transient bacterial microbiota. Our results suggest the complexity of the factors shaping the gut microbiota, while highlighting interspecific differences in microbiota residency within the same insect functional group.

[Source](https://www.nature.com/articles/s41598-022-19855-5)

10.1038/s41598-022-19855-5

---

## Adapting to climate with limited genetic diversity: Nucleotide, DNA methylation and microbiome variation among populations of the social spider <em>Stegodyphus dumicola</em>

[Source](https://doi.org/10.1111/mec.16696)

---

## Filtering ASVs/OTUs via mutual information-based microbiome network analysis.
 16.09.2022, _EB Mokhtari, BJ Ridenhour_


Microbial communities are widely studied using high-throughput sequencing techniques, such as 16S rRNA gene sequencing. These techniques have attracted biologists as they offer powerful tools to explore microbial communities and investigate their patterns of diversity in biological and biomedical samples at remarkable resolution. However, the accuracy of these methods can negatively affected by the presence of contamination. Several studies have recognized that contamination is a common problem in microbial studies and have offered promising computational and laboratory-based approaches to assess and remove contaminants. Here we propose a novel strategy, MI-based (mutual information based) filtering method, which uses information theoretic functionals and graph theory to identify and remove contaminants. We applied MI-based filtering method to a mock community data set and evaluated the amount of information loss due to filtering taxa. We also compared our method to commonly practice traditional filtering methods. In a mock community data set, MI-based filtering approach maintained the true bacteria in the community without significant loss of information. Our results indicate that MI-based filtering method effectively identifies and removes contaminants in microbial communities and hence it can be beneficial as a filtering method to microbiome studies. We believe our filtering method has two advantages over traditional filtering methods. First, it does not required an arbitrary choice of threshold and second, it is able to detect true taxa with low abundance.

[Source](https://doi.org/10.1186/s12859-022-04919-0)

10.1186/s12859-022-04919-0

---

## Batch effects removal for microbiome data via conditional quantile regression.
 15.09.2022, _W Ling, J Lu, N Zhao, A Lulla, AM Plantinga, W Fu, A Zhang, H Liu, H Song, Z Li, J Chen, TW Randolph, WLA Koay, JR White, LJ Launer, AA Fodor, KA Meyer, MC Wu_


Batch effects in microbiome data arise from differential processing of specimens and can lead to spurious findings and obscure true signals. Strategies designed for genomic data to mitigate batch effects usually fail to address the zero-inflated and over-dispersed microbiome data. Most strategies tailored for microbiome data are restricted to association testing or specialized study designs, failing to allow other analytic goals or general designs. Here, we develop the Conditional Quantile Regression (ConQuR) approach to remove microbiome batch effects using a two-part quantile regression model. ConQuR is a comprehensive method that accommodates the complex distributions of microbial read counts by non-parametric modeling, and it generates batch-removed zero-inflated read counts that can be used in and benefit usual subsequent analyses. We apply ConQuR to simulated and real microbiome datasets and demonstrate its advantages in removing batch effects while preserving the signals of interest.

[Source](https://doi.org/10.1038/s41467-022-33071-9)

10.1038/s41467-022-33071-9

---

## Phylogenetic Analysis of Anti-CRISPR and Member Addition in the Families.
 15.09.2022, _S Nidhi, P Tripathi, V Tripathi_


CRISPR-Cas is a widespread anti-viral adaptive immune system in the microorganisms. Viruses living in bacteria or some phages carry anti-CRISPR proteins to evade immunity by CRISPR-Cas. The anti-CRISPR proteins are prevalent in phages capable of lying dormant in a CRISPR-carrying host, while their orthologs frequently found in virulent phages. Here, we propose a probabilistic strategy of ancestral sequence reconstruction (ASR) and Hidden Markov Model (HMM) profile search to fish out sequences of anti-CRISPR proteins from environmental metagenomic, human microbiome metagenomic, human microbiome reference genome, and NCBI's non-redundant databases. Our results revealed that the metagenome database dark matter might contain anti-CRISPR encoding genes.

[Source](https://doi.org/10.1007/s12033-022-00558-1)

10.1007/s12033-022-00558-1

---

## Shallow shotgun sequencing of the microbiome recapitulates 16S amplicon results and provides functional insights.
 16.09.2022, _MR Stothart, PD McLoughlin, J Poissant_


Prevailing 16S rRNA gene-amplicon methods for characterizing the bacterial microbiome of wildlife are economical, but result in coarse taxonomic classifications, are subject to primer and 16S copy number biases, and do not allow for direct estimation of microbiome functional potential. While deep shotgun metagenomic sequencing can overcome many of these limitations, it is prohibitively expensive for large sample sets. We evaluated the ability of shallow shotgun metagenomic sequencing to characterize taxonomic and functional patterns in the fecal microbiome of a model population of feral horses (Sable Island, Canada). Since 2007, this unmanaged population has been the subject of an individual-based, long-term ecological study. Using deep shotgun metagenomic sequencing, we determined the sequencing depth required to accurately characterize the horse microbiome. In comparing conventional versus high-throughput shotgun metagenomic library preparation techniques, we validate the use of more cost-effective lab methods. Finally, we characterize similarities between 16S amplicon and shallow shotgun characterization of the microbiome, and demonstrate that the latter recapitulates biological patterns first described in a published amplicon dataset. Unlike amplicon data, we further demonstrate how shallow shotgun metagenomic data provide useful insights about microbiome functional potential which support previously hypothesized diet effects in this study system.

[Source](https://doi.org/10.1111/1755-0998.13713)

10.1111/1755-0998.13713

---

## SnapShot: The Expobiome Map.
 15.09.2022, _VTE Aho, M Ostaszewski, C Martin-Gallausiaux, CC Laczny, JG Schneider, P Wilmes_


The human gut microbiome is intricately connected to health and disease. Microbiome-derived molecules are implicated in many chronic diseases involving inflammation. Herein, we summarize the diverse complex of such immunogenic molecules, including nucleic acids, (poly)peptides, structural molecules, and metabolites. The interactions between this "expobiome" and human immune pathways are specifically illustrated in the context of chronic diseases. To view this SnapShot, open or download the PDF.

[Source](https://doi.org/10.1016/j.chom.2022.08.015)

10.1016/j.chom.2022.08.015

---

