## The cancer chemotherapeutic 5-fluorouracil is a potent <em>Fusobacterium nucleatum </em>inhibitor and its activity is modified by intratumoral microbiota

[Source](https://doi.org/10.1016/j.celrep.2022.111625)

---

## Enterococci enhance <em>Clostridioides difficile</em> pathogenesis

[Source](https://doi.org/10.1038/s41586-022-05438-x)

---

## Host biology, ecology and the environment influence microbial biomass and diversity in 101 marine fish species

[Source](https://doi.org/10.1038/s41467-022-34557-2)

---

## Genomic ancestry, diet and microbiomes of Upper Palaeolithic hunter-gatherers from San Teodoro cave

[Source](https://doi.org/10.1038/s42003-022-04190-2)

---

## Gut microbiota analysis for prediction of clinical relapse in Crohn’s disease

[Source](https://doi.org/10.1038/s41598-022-23757-x)

---

## Implications of oral streptococcal bacteriophages in autism spectrum disorder

[Source](https://doi.org/10.1038/s41522-022-00355-3)

---

## Reproducible changes in the anorexia nervosa gut microbiota following inpatient therapy remain distinct from non-eating disorder controls

[Source](https://doi.org/10.1080/19490976.2022.2143217)

---

## Nasal microbiome disruption and recovery after mupirocin treatment in <em>Staphylococcus aureus</em> carriers and noncarriers

[Source](https://doi.org/10.1038/s41598-022-21453-4)

---

## BlpC-mediated selfish program leads to rapid loss of <em>Streptococcus pneumoniae</em> clonal diversity during infection

[Source](https://doi.org/10.1016/j.chom.2022.10.015)

---

## Treasures from trash in cancer research

[Source](https://doi.org/10.18632/oncotarget.28308)

---

## Human Gut Metagenomes Encode Diverse GH156 Sialidases

[Source](https://doi.org/10.1128/aem.01755-22)

---

## Species-Specific Patterns of Gut Metabolic Modules in Dutch Individuals with Different Dietary Habits

[Source](https://doi.org/10.1128/msphere.00512-22)

---

## Effect of the intratumoral microbiota on spatial and cellular heterogeneity in cancer

[Source](https://doi.org/10.1038/s41586-022-05435-0)

---

## Broad-spectrum antibiotics associated gut microbiome disturbance impairs T cell immunity and promotes lung cancer metastasis: a retrospective study

[Source](https://doi.org/10.1186/s12885-022-10307-x)

---

## Diet-induced modifications to human microbiome reshape colonic homeostasis in irritable bowel syndrome.
 21.11.2022, _H Bootz-Maoz, A Pearl, E Melzer, S Malnick, E Sharon, Y Bennet, R Tsentsarevsky, S Abuchatzera, S Amidror, E Aretz, S Azriel, C Gam Ze Letova, M Naama, I Shoval, O Yaron, S Karako-Lampert, S Bel, N Yissachar_


_Humans, Irritable Bowel Syndrome, Diet, Carbohydrate-Restricted, Gastrointestinal Microbiome, Homeostasis_
Changes in microbiome composition are associated with a wide array of human diseases, turning the human microbiota into an attractive target for therapeutic intervention. Yet, clinical translation of these findings requires the establishment of causative connections between specific microbial taxa and their functional impact on host tissues. Here, we infuse gut organ cultures with longitudinal microbiota samples collected from therapy-naive patients with irritable bowel syndrome (IBS) under a low-fermentable oligo-, di-, mono-saccharides and polyols (FODMAP) diet. We show that post-diet microbiota regulates intestinal expression of inflammatory and neuro-muscular gene sets. Specifically, we identify Bifidobacterium adolescentis as a diet-sensitive pathobiont that alters tight junction integrity and disrupts gut barrier functions. Collectively, we present a pathway discovery platform for mechanistic dissection and identification of functional diet-host-microbiota modules. Our data support the hypothesis that the gut microbiota mediates the beneficial effects of a low-FODMAP diet and reinforce the potential feasibility of microbiome-based therapies in IBS.

[Source](https://doi.org/10.1016/j.celrep.2022.111657)

10.1016/j.celrep.2022.111657

---

## The dynamic lung microbiome in health and disease.
 22.11.2022, _JG Natalini, S Singh, LN Segal_


New methods and technologies within the field of lung biology are beginning to shed new light into the microbial world of the respiratory tract. Long considered to be a sterile environment, it is now clear that the human lungs are frequently exposed to live microbes and their by-products. The nature of the lung microbiome is quite distinct from other microbial communities inhabiting our bodies such as those in the gut. Notably, the microbiome of the lung exhibits a low biomass and is dominated by dynamic fluxes of microbial immigration and clearance, resulting in a bacterial burden and microbiome composition that is fluid in nature rather than fixed. As our understanding of the microbial ecology of the lung improves, it is becoming increasingly apparent that certain disease states can disrupt the microbial-host interface and ultimately affect disease pathogenesis. In this Review, we provide an overview of lower airway microbial dynamics in health and disease and discuss future work that is required to uncover novel therapeutic targets to improve lung health.

[Source](https://doi.org/10.1038/s41579-022-00821-x)

10.1038/s41579-022-00821-x

---

## Modulation of gut microbiota, blood metabolites, and disease resistance by dietary β-glucan in rainbow trout (<em>Oncorhynchus mykiss</em>)

[Source](https://doi.org/10.1186/s42523-022-00209-5)

---

## Living in honey: bacterial and fungal communities in honey of sympatric populations of <em>Apis mellifera</em> and the stingless bee <em>Melipona beecheii</em>, in Yucatan, Mexico

[Source](https://doi.org/10.1007/s00203-022-03319-5)

---

## The involvement of organic acids in soil fertility, plant health and environment sustainability.
 22.11.2022, _SS Sindhu, A Sehrawat, BR Glick_


_Humans, Soil, Soil Microbiology, Plant Roots, Plants, Microbiota_
Increasing demand for safe food by an ever-growing human population emphasizes the urgency for increasing crop yields and reducing the losses caused by abiotic and biotic stresses; a partial solution to this problem is to develop a better understanding of plant-microbe interactions. Plant roots continuously release a wide range of compounds including organic acids in root exudates. These root exudates stimulate growth of specific microbial communities in the rhizosphere, which affect complex biological and physico-chemical interactions occurring between plant roots and the surrounding soil environment. In addition, organic acids are also released by different microbes and during decomposition of organic matter and plant residues in the soil. Interestingly, the available organic acids in the rhizosphere play crucial roles in various physio-chemical processes including the chemoattraction of microbes (both beneficial and pathogenic), mineralization and solubilization of complex minerals (P, K and Zn), biocontrol of phytopathogens, induction of systemic resistance, biogas formation, mitigation of abiotic stresses and, detoxification of metals and residual pesticides. Thus, organic acids play a significant role in the sustainable management of the soil ecosystem and in environmental sustainability. This review discusses the role of organic acids in the stimulation or enrichment of specific root-associated microbial communities and their effect on plant-microbe interactions at the root surface. In addition, the potential for root microbiome modification to enhance nutrient cycling and nutrient acquisition, and in amelioration of environmental stresses for increasing food production is discussed.

[Source](https://doi.org/10.1007/s00203-022-03321-x)

10.1007/s00203-022-03321-x

---

## Investigating the impact of database choice on the accuracy of metagenomic read classification for the rumen microbiome.
 21.11.2022, _RH Smith, L Glendinning, AW Walker, M Watson_


Microbiome analysis is quickly moving towards high-throughput methods such as metagenomic sequencing. Accurate taxonomic classification of metagenomic data relies on reference sequence databases, and their associated taxonomy. However, for understudied environments such as the rumen microbiome many sequences will be derived from novel or uncultured microbes that are not present in reference databases. As a result, taxonomic classification of metagenomic data from understudied environments may be inaccurate. To assess the accuracy of taxonomic read classification, this study classified metagenomic data that had been simulated from cultured rumen microbial genomes from the Hungate collection. To assess the impact of reference databases on the accuracy of taxonomic classification, the data was classified with Kraken 2 using several reference databases. We found that the choice and composition of reference database significantly impacted on taxonomic classification results, and accuracy. In particular, NCBI RefSeq proved to be a poor choice of database. Our results indicate that inaccurate read classification is likely to be a significant problem, affecting all studies that use insufficient reference databases. We observed that adding cultured reference genomes from the rumen to the reference database greatly improved classification rate and accuracy. We also demonstrated that metagenome-assembled genomes (MAGs) have the potential to further enhance classification accuracy by representing uncultivated microbes, sequences of which would otherwise be unclassified or incorrectly classified. However, classification accuracy was strongly dependent on the taxonomic labels assigned to these MAGs. We therefore highlight the importance of accurate reference taxonomic information and suggest that, with formal taxonomic lineages, MAGs have the potential to improve classification rate and accuracy, particularly in environments such as the rumen that are understudied or contain many novel genomes.

[Source](https://doi.org/10.1186/s42523-022-00207-7)

10.1186/s42523-022-00207-7

---

## dbCAN-seq update: CAZyme gene clusters and substrates in microbiomes

[Source](https://doi.org/10.1093/nar/gkac1068)

---

## Gene co-expression network analysis of the human gut commensal bacterium <em>Faecalibacterium prausnitzii</em> in R-Shiny

[Source](https://doi.org/10.1371/journal.pone.0271847)

---

