## Microbiome research in Africa must be based on equitable partnerships.
 23.05.2024, _OH Oduaran, MO Foláyan, AW Kamng'ona, A Nakimuli, LM Mwapagha, ME Setati, M Owusu, N Mulder, TP Makhalanyane, S Kouidhi_



[Source](https://doi.org/10.1038/s41591-024-03026-2)

10.1038/s41591-024-03026-2

---

## IL-33 controls IL-22-dependent antibacterial defense by modulating the microbiota

[Source](https://doi.org/10.1073/pnas.2310864121)

---

## Meta-analysis of shotgun sequencing of gut microbiota in Parkinson’s disease

[Source](https://doi.org/10.1038/s41531-024-00724-z)

---

## Systematic review of associations between gut microbiome composition and stunting in under-five children

[Source](https://doi.org/10.1038/s41522-024-00517-5)

---

## Production of deoxycholic acid by low-abundant microbial species is associated with impaired glucose metabolism

[Source](https://doi.org/10.1038/s41467-024-48543-3)

---

## Altered gut microbiota is associated with the formation of occult hepatitis B virus infection

[Source](https://doi.org/10.1128/spectrum.00239-24)

---

## Metagenome-assembled genomes (MAGs) suggest an acetate-driven protective role in gut microbiota disrupted by <em>Clostridioides difficile</em>

[Source](https://doi.org/10.1016/j.micres.2024.127739)

---

## From gut microbiota to brain: implications on binge eating disorders

[Source](https://doi.org/10.1080/19490976.2024.2357177)

---

## Pathogenesis and therapeutic opportunities of gut microbiome dysbiosis in critical illness

[Source](https://doi.org/10.1080/19490976.2024.2351478)

---

## Pathobiological signatures of dysbiotic lung injury in pediatric patients undergoing stem cell transplantation

[Source](https://doi.org/10.1038/s41591-024-02999-4)

---

## Leveraging existing 16S rRNA gene surveys to decipher microbial signatures and dysbiosis in cervical carcinogenesis

[Source](https://doi.org/10.1038/s41598-024-62531-z)

---

## Targeting <em>Megasphaera</em> species to promote cervicovaginal health

[Source](https://www.cell.com/trends/microbiology/abstract/S0966-842X(24)00131-8)

---

## Prodrug-conjugated tumor-seeking commensals for targeted cancer therapy

[Source](https://doi.org/10.1038/s41467-024-48661-y)

---

## Fecal microbiota transplantation significantly improved respiratory failure of amyotrophic lateral sclerosis

[Source](https://doi.org/10.1080/19490976.2024.2353396)

---

## The use of probiotics and prebiotics in decolonizing pathogenic bacteria from the gut; a systematic review and meta-analysis of clinical outcomes

[Source](https://doi.org/10.1080/19490976.2024.2356279)

---

## Advanced microbiome therapeutics as a novel modality for oral delivery of peptides to manage metabolic diseases

[Source](https://www.cell.com/trends/endocrinology-metabolism/fulltext/S1043-2760(24)00115-2)

---

## Identification of the mutual gliding locus as a factor for gut colonization in non-native bee hosts using the ARTP mutagenesis

[Source](https://doi.org/10.1186/s40168-024-01813-0)

---

## Symbiotic bacteria and fungi proliferate in diapause and may enhance overwintering survival in a solitary bee

[Source](https://doi.org/10.1093/ismejo/wrae089)

---

## Phages enhance both phytopathogen density control and rhizosphere microbiome suppressiveness

[Source](https://doi.org/10.1128/mbio.03016-23)

---

## Global epistasis and the emergence of function in microbial consortia

[Source](https://www.cell.com/cell/fulltext/S0092-8674(24)00411-2)

---

## Bacterial dynamics of the plastisphere microbiome exposed to sub-lethal antibiotic pollution

[Source](https://doi.org/10.1186/s40168-024-01803-2)

---

## Global soil metagenomics reveals distribution and predominance of Deltaproteobacteria in nitrogen-fixing microbiome.
 27.05.2024, _Y Masuda, K Mise, Z Xu, Z Zhang, Y Shiratori, K Senoo, H Itoh_


_Soil Microbiology, Nitrogen Fixation, Metagenomics, Microbiota, Deltaproteobacteria, Soil, Phylogeny, Nitrogen, Metagenome_
Biological nitrogen fixation is a fundamental process sustaining all life on earth. While distribution and diversity of N2-fixing soil microbes have been investigated by numerous PCR amplicon sequencing of nitrogenase genes, their comprehensive understanding has been hindered by lack of de facto standard protocols for amplicon surveys and possible PCR biases. Here, by fully leveraging the planetary collections of soil shotgun metagenomes along with recently expanded culture collections, we evaluated the global distribution and diversity of terrestrial diazotrophic microbiome.

[Source](https://doi.org/10.1186/s40168-024-01812-1)

10.1186/s40168-024-01812-1

---

## A global initiative for ecological and evolutionary hologenomics

[Source](https://www.cell.com/trends/ecology-evolution/abstract/S0169-5347(24)00074-0)

---

## Fungal diversity differences in the indoor dust microbiome from built environments on earth and in space.
 27.05.2024, _N Nastasi, SR Haines, A Bope, ME Meyer, JM Horack, KC Dannemiller_


_Dust, Fungi, Microbiota, Humans, Spacecraft, Air Pollution, Indoor, Built Environment, Bacteria, Air Microbiology, Earth, Planet, Humidity_
Human occupied built environments are no longer confined to Earth. In fact, there have been humans living and working in low-Earth orbit on the International Space Station (ISS) since November 2000. With NASA's Artemis missions and the age of commercial space stations set to begin, more human-occupied spacecraft than ever will be in Earth's orbit and beyond. On Earth and in the ISS, microbes, especially fungi, can be found in dust and grow when unexpected, elevated moisture conditions occur. However, we do not yet know how indoor microbiomes in Earth-based homes and in the ISS differ due to their unique set of environmental conditions. Here we show that bacterial and fungal communities are different in dust collected from vacuum bags on Earth and the ISS, with Earth-based homes being more diverse (465 fungal OTUs and 237 bacterial ASVs) compared to the ISS (102 fungal OTUs and 102 bacterial ASVs). When dust from these locations were exposed to varying equilibrium relative humidity conditions (ERH), there were also significant fungal community composition changes as ERH and time elevated increased (Bray Curtis: R2 = 0.35, P = 0.001). These findings can inform future spacecraft design to promote healthy indoor microbiomes that support crew health, spacecraft integrity, and planetary protection.

[Source](https://doi.org/10.1038/s41598-024-62191-z)

10.1038/s41598-024-62191-z

---

## A novel clinical metaproteomics workflow enables bioinformatic analysis of host-microbe dynamics in disease.
 23.05.2024, _K Do, S Mehta, R Wagner, D Bhuming, AT Rajczewski, APN Skubitz, JE Johnson, TJ Griffin, PD Jagtap_


Clinical metaproteomics has the potential to offer insights into the host-microbiome interactions underlying diseases. However, the field faces challenges in characterizing microbial proteins found in clinical samples, usually present at low abundance relative to the host proteins. As a solution, we have developed an integrated workflow coupling mass spectrometry-based analysis with customized bioinformatic identification, quantification, and prioritization of microbial proteins, enabling targeted assay development to investigate host-microbe dynamics in disease. The bioinformatics tools are implemented in the Galaxy ecosystem, offering the development and dissemination of complex bioinformatic workflows. The modular workflow integrates MetaNovo (to generate a reduced protein database), SearchGUI/PeptideShaker and MaxQuant [to generate peptide-spectral matches (PSMs) and quantification], PepQuery2 (to verify the quality of PSMs), Unipept (for taxonomic and functional annotation), and MSstatsTMT (for statistical analysis). We have utilized this workflow in diverse clinical samples, from the characterization of nasopharyngeal swab samples to bronchoalveolar lavage fluid. Here, we demonstrate its effectiveness via analysis of residual fluid from cervical swabs. The complete workflow, including training data and documentation, is available via the Galaxy Training Network, empowering non-expert researchers to utilize these powerful tools in their clinical studies.

[Source](https://doi.org/10.1128/msphere.00793-23)

10.1128/msphere.00793-23

---

## A comparative study of supervised and unsupervised machine learning algorithms applied to human microbiome.
 20.05.2024, _E Kalluçi, B Preni, X Dhamo, E Noka, S Bardhi, A Macchia, G Bonetti, K Dhuli, K Donato, M Bertelli, LJM Zambrano, S Janaqi_


_Humans, Supervised Machine Learning, Unsupervised Machine Learning, Microbiota, RNA, Ribosomal, 16S, Colorectal Neoplasms, Gastrointestinal Microbiome, Algorithms, Feces, Adenoma_
The human microbiome, consisting of diverse bacte-rial, fungal, protozoan and viral species, exerts a profound influence on various physiological processes and disease susceptibility. However, the complexity of microbiome data has presented significant challenges in the analysis and interpretation of these intricate datasets, leading to the development of specialized software that employs machine learning algorithms for these aims.

[Source](https://doi.org/10.7417/CT.2024.5051)

10.7417/CT.2024.5051

---

