## The Multiomics Analyses of Fecal Matrix and Its Significance to Coeliac Disease Gut Profiling

[Source](https://doi.org/10.3390/ijms22041965)

---

## <span style="color:#993300;">Exploring the interaction network of a synthetic gut bacterial community</span>

[Source](https://doi.org/10.1101/2021.02.25.432904)

---

## Environmental concentrations of antibiotics alter the zebrafish gut microbiome structure and potential functions

[Source](https://doi.org/10.1016/j.envpol.2021.116760)

---

## Fish farm effluents as a source of antibiotic resistance gene dissemination on Jeju Island, South Korea

[Source](https://doi.org/10.1016/j.envpol.2021.116764)

---

## Influence of environmental factors and bacterial community diversity in pond water on health of Chinese perch through Gut Microbiota change

[Source](https://doi.org/10.1016/j.aqrep.2021.100629)

---

## The diversity, composition, and metabolic pathways of archaea in pigs

[Source](10.20944/preprints202102.0508.v1)

---

## Stony Coral Tissue Loss Disease biomarker bacteria identified in corals and overlying waters using a rapid field-based sequencing approach

[Source](https://doi.org/10.1101/2021.02.17.431614)

---

## <span style="color:#993300;">Microbial analyses of blood spot surfaces collected from a laboratory and the bathroom of a female single-person household under different environmental conditions</span>

[Source](https://doi.org/10.1093/femsle/fnab023)

---

## Rapid emergence of virus-host mutualism under stress.
 18.02.2021, _VV Dolja_



[Source](https://doi.org/10.1073/pnas.2100936118)

10.1073/pnas.2100936118

---

## Patterns of virus growth across the diversity of life.
 22.02.2021, _T Jin, J Yin_


Although viruses in their natural habitats add up to less than 10% of the biomass, they contribute more than 90% of the genome sequences [1]. These viral sequences or 'viromes' encode viruses that populate the Earth's oceans [2, 3] and terrestrial environments [4, 5], where their infections impact life across diverse ecological niches and scales [6, 7], including humans [8-10]. Most viruses have yet to be isolated and cultured [11-13], and surprisingly few efforts have explored what analysis of available data might reveal about their nature. Here, we compiled and analyzed seven decades of one-step growth and other data for viruses from six major families, including their infections of archaeal, bacterial and eukaryotic hosts [14-191]. We found that the use of host cell biomass for virus production was highest for archaea at 10%, followed by bacteria at 1% and eukarya at 0.01%, highlighting the degree to which viruses of archaea and bacteria exploit their host cells. For individual host cells, the yield of virus progeny spanned a relatively narrow range (10-1000 infectious particles per cell) compared with the million-fold difference in size between the smallest and largest cells. Furthermore, healthy and infected host cells were remarkably similar in the time they needed to multiply themselves or their virus progeny. Specifically, the doubling time of healthy cells and the delay time for virus release from infected cells were not only correlated (r = 0.71, p &lt; 10-10, n = 101); they also spanned the same range from tens of minutes to about a week. These results have implications for better understanding the growth, spread and persistence of viruses in complex natural habitats that abound with diverse hosts, including humans and their associated microbes.

[Source](https://doi.org/10.1093/intbio/zyab001)

10.1093/intbio/zyab001

---

## Standard Bacteriophage Purification Procedures Cause Loss in Numbers and Activity

[Source](https://doi.org/10.3390/v13020328)

---

## Misinformation, Perceptions Towards COVID-19 and Willingness to be Vaccinated: A Population-Based Survey in Yemen

[Source](https://doi.org/10.1101/2021.02.25.432838)

---

