## A practice of using five-colour chart to guide the control of COVID-19 and resumption of work in Zhejiang Province, China.
 14.06.2021, _F He, X Shang, F Ling, Z Chen, T Fu, J Lin, Z Wang_


_COVID-19, China, Color, Disease Outbreaks, Epidemics, Humans, Pandemics, Quarantine, Risk Assessment_
Since the outbreak of COVID-19 in December 2019 in Wuhan, Zhejiang has become the province with the largest number of cases. The aim of this article is to present Zhejiang province's experience of establishing an accurate and smart control mechanism for epidemic prevention and control and resumption of work and production using a 'five-colour epidemic chart'. The number of confirmed cases, proportion of local cases, and occurrence of clustered outbreaks were used as evaluation indicators to calculate the county-level epidemic risk and were assigned different weight coefficients; the absence of cases for 3 and 7 consecutive days was used as the adjustment index. When the first chart was published on February 9, there were 1 very-high-risk, 12 high-risk, and 12 low-risk counties. Under the five-colour chart, Zhejiang began to adopt precise measures to prevent and control the epidemic and resume work and production. By February 24, the low-risk counties had expanded to 82, with no high-risk and very-high-risk counties. The epidemic situation in Zhejiang province has been effectively controlled. The experience of epidemic prevention and control in Zhejiang is worthy to be emulated and learned by other countries and regions.

[Source](https://doi.org/10.3785/j.issn.1008-9292.2020.02.02)

10.1038/s41598-021-90808-0

---

## Management Of COVID-19: The Zhejiang Experience

[Source](https://doi.org/10.3785/j.issn.1008-9292.2020.02.02)

---

## Metatranscriptomic Characterization of COVID-19 Identified A Host Transcriptional Classifier Associated With Immune Signaling.
 26.06.2020, _H Zhang, JW Ai, W Yang, X Zhou, F He, S Xie, W Zeng, Y Li, Y Yu, X Gou, Y Li, X Wang, H Su, T Xu, W Zhang_


The recent identification of a novel coronavirus, also known as SARS-CoV-2, has caused a global outbreak of respiratory illnesses. The rapidly developing pandemic has posed great challenges to diagnosis of this novel infection. However, little is known about the metatranscriptomic characteristics of patients with Coronavirus Disease 2019 (COVID-19).

[Source](https://doi.org/10.1093/cid/ciaa663)

10.1093/cid/ciaa663

---

## The lung tissue microbiota features of 20 deceased patients with COVID-19.
 10.01.2021, _J Fan, X Li, Y Gao, J Zhou, S Wang, B Huang, J Wu, Q Cao, Y Chen, Z Wang, D Luo, T Zhou, R Li, Y Shang, X Nie_


_Betacoronavirus, COVID-19, Coronavirus Infections, Critical Illness, Humans, Microbiota, Pandemics, Pneumonia, Viral, SARS-CoV-2_

[Source](https://doi.org/10.1016/j.jinf.2020.06.047)

10.1016/j.jinf.2020.06.047

---

## The Lung Tissue Microbiota Features Of 20 Deceased Patients With COVID-19

[Source](https://doi.org/10.1016/j.jinf.2020.06.047)

---

## Nasopharyngeal Microbiota Profiling of SARS-CoV-2 Infected Patients.
 28.09.2020, _F De Maio, B Posteraro, FR Ponziani, P Cattani, A Gasbarrini, M Sanguinetti_


We analyzed the bacterial communities of the nasopharynx in 40 SARS-CoV-2 infected and uninfected patients. All infected patients had a mild COVID-19 disease. We did not find statistically significant differences in either bacterial richness and diversity or composition. These findings suggest a nasopharyngeal microbiota at least early resilient to SARS-CoV-2 infection.

[Source](https://doi.org/10.1186/s12575-020-00131-7)

10.1186/s12575-020-00131-7

---

## Metagenomic Next-Generation Sequencing of Nasopharyngeal Specimens Collected from Confirmed and Suspect COVID-19 Patients.
 14.12.2020, _HH Mostafa, JA Fissel, B Fanelli, Y Bergman, V Gniazdowski, M Dadlani, KC Carroll, RR Colwell, PJ Simner_


_Bacteria, COVID-19, Coinfection, Computational Biology, High-Throughput Nucleotide Sequencing, Humans, Metagenome, Metagenomics, Microbiota, Nasopharynx, RNA, Viral, SARS-CoV-2, Specimen Handling_
Metagenomic next-generation sequencing (mNGS) offers an agnostic approach for emerging pathogen detection directly from clinical specimens. In contrast to targeted methods, mNGS also provides valuable information on the composition of the microbiome and might uncover coinfections that may associate with disease progression and impact prognosis. To evaluate the use of mNGS for detecting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and/or other infecting pathogens, we applied direct Oxford Nanopore long-read third-generation metatranscriptomic and metagenomic sequencing. Nasopharyngeal (NP) swab specimens from 50 patients under investigation for CoV disease 2019 (COVID-19) were sequenced, and the data were analyzed by the CosmosID bioinformatics platform. Further, we characterized coinfections and the microbiome associated with a four-point severity index. SARS-CoV-2 was identified in 77.5% (31/40) of samples positive by RT-PCR, correlating with lower cycle threshold (Ct) values and fewer days from symptom onset. At the time of sampling, possible bacterial or viral coinfections were detected in 12.5% of SARS-CoV-2-positive specimens. A decrease in microbial diversity was observed among COVID-19-confirmed patients (Shannon diversity index, P = 0.0082; Chao richness estimate, P = 0.0097; Simpson diversity index, P = 0.018), and differences in microbial communities were linked to disease severity (P = 0.022). Furthermore, statistically significant shifts in the microbiome were identified among SARS-CoV-2-positive and -negative patients, in the latter of whom a higher abundance of Propionibacteriaceae (P = 0.028) and a reduction in the abundance of Corynebacterium accolens (P = 0.025) were observed. Our study corroborates the growing evidence that increased SARS-CoV-2 RNA detection from NP swabs is associated with the early stages rather than the severity of COVID-19. Further, we demonstrate that SARS-CoV-2 causes a significant change in the respiratory microbiome. This work illustrates the utility of mNGS for the detection of SARS-CoV-2, for diagnosing coinfections without viral target enrichment or amplification, and for the analysis of the respiratory microbiome.IMPORTANCE SARS-CoV-2 has presented a rapidly accelerating global public health crisis. The ability to detect and analyze viral RNA from minimally invasive patient specimens is critical to the public health response. Metagenomic next-generation sequencing (mNGS) offers an opportunity to detect SARS-CoV-2 from nasopharyngeal (NP) swabs. This approach also provides information on the composition of the respiratory microbiome and its relationship to coinfections or the presence of other organisms that may impact SARS-CoV-2 disease progression and prognosis. Here, using direct Oxford Nanopore long-read third-generation metatranscriptomic and metagenomic sequencing of NP swab specimens from 50 patients under investigation for COVID-19, we detected SARS-CoV-2 sequences by applying the CosmosID bioinformatics platform. Further, we characterized coinfections and detected a decrease in the diversity of the microbiomes in these patients. Statistically significant shifts in the microbiome were identified among COVID-19-positive and -negative patients, in the latter of whom a higher abundance of Propionibacteriaceae and a reduction in the abundance of Corynebacterium accolens were observed. Our study also corroborates the growing evidence that increased SARS-CoV-2 RNA detection from NP swabs is associated with the early stages of disease rather than with severity of disease. This work illustrates the utility of mNGS for the detection and analysis of SARS-CoV-2 from NP swabs without viral target enrichment or amplification and for the analysis of the respiratory microbiome.

[Source](https://doi.org/10.1128/mBio.01969-20)

10.1128/mBio.01969-20

---

## Feasibility of using alternative swabs and storage solutions for paired SARS-CoV-2 detection and microbiome analysis in the hospital environment.
 26.05.2021, _JJ Minich, F Ali, C Marotz, P Belda-Ferre, L Chiang, JP Shaffer, CS Carpenter, D McDonald, J Gilbert, SM Allard, EE Allen, R Knight, DA Sweeney, AD Swafford_


_Biological Transport, COVID-19 Nucleic Acid Testing, Ethanol, Feasibility Studies, Humans, Intensive Care Units, Limit of Detection, Microbiota, RNA, Ribosomal, 16S, RNA, Viral, Ribonucleases, SARS-CoV-2, Specimen Handling_
Determining the role of fomites in the transmission of SARS-CoV-2 is essential in the hospital setting and will likely be important outside of medical facilities as governments around the world make plans to ease COVID-19 public health restrictions and attempt to safely reopen economies. Expanding COVID-19 testing to include environmental surfaces would ideally be performed with inexpensive swabs that could be transported safely without concern of being a source of new infections. However, CDC-approved clinical-grade sampling supplies and techniques using a synthetic swab are expensive, potentially expose laboratory workers to viable virus and prohibit analysis of the microbiome due to the presence of antibiotics in viral transport media (VTM). To this end, we performed a series of experiments comparing the diagnostic yield using five consumer-grade swabs (including plastic and wood shafts and various head materials including cotton, synthetic, and foam) and one clinical-grade swab for inhibition to RNA. For three of these swabs, we evaluated performance to detect SARS-CoV-2 in twenty intensive care unit (ICU) hospital rooms of patients including COVID-19+ patients. All swabs were placed in 95% ethanol and further evaluated in terms of RNase activity. SARS-CoV-2 was measured both directly from the swab and from the swab eluent.

[Source](https://doi.org/10.1186/s40168-020-00960-4)

10.1186/s40168-020-00960-4

---

## Variability in digestive and respiratory tract Ace2 expression is associated with the microbiome.
 27.03.2021, _ST Koester, N Li, DM Lachance, NM Morella, N Dey_


_Adenomatous Polyposis Coli Protein, Angiotensin-Converting Enzyme 2, Animals, Colon, Female, Gastrointestinal Microbiome, Gene Expression, Interleukin-10, Intestine, Small, Lung, Male, Mice, Mice, Inbred C57BL, Mice, Knockout_
COVID-19 (coronavirus disease 2019) patients exhibiting gastrointestinal symptoms are reported to have worse prognosis. Ace2 (angiotensin-converting enzyme 2), the gene encoding the host protein to which SARS-CoV-2 spike proteins bind, is expressed in the gut and therefore may be a target for preventing or reducing severity of COVID-19. Here we test the hypothesis that Ace2 expression in the gastrointestinal and respiratory tracts is modulated by the microbiome. We used quantitative PCR to profile Ace2 expression in germ-free mice, conventional raised specific pathogen-free mice, and gnotobiotic mice colonized with different microbiota. Intestinal Ace2 expression levels were significantly higher in germ-free mice compared to conventional mice. A similar trend was observed in the respiratory tract. Intriguingly, microbiota depletion via antibiotics partially recapitulated the germ-free phenotype, suggesting potential for microbiome-mediated regulation of Ace2 expression. Variability in intestinal Ace2 expression was observed in gnotobiotic mice colonized with different microbiota, partially attributable to differences in microbiome-encoded proteases and peptidases. Together, these data suggest that the microbiome may be one modifiable factor determining COVID-19 infection risk and disease severity.

[Source](https://doi.org/10.1371/journal.pone.0248730)

10.1371/journal.pone.0248730

---

## Variability In Digestive And Respiratory Tract Ace2 expression Is associated with the microbiome

[Source](https://doi.org/10.1371/journal.pone.0248730)

---

## Diversity and genomic determinants of the microbiomes associated with COVID-19 and non-COVID respiratory diseases.
 25.05.2021, _MN Hoque, MS Rahman, R Ahmed, MS Hossain, MS Islam, T Islam, MA Hossain, AZ Siddiki_


The novel coronavirus disease 2019 (COVID-19) is a rapidly emerging and highly transmissible disease caused by the Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2). Understanding the microbiomes associated with the upper respiratory tract infection (URTI), chronic obstructive pulmonary disease (COPD) and COVID-19 diseases has clinical interest. We hypothesize that microbiome diversity and composition, and their genomic features are associated with different pathological conditions of these human respiratory tract diseases. To test this hypothesis, we analyzed 21 RNASeq metagenomic data including eleven COVID-19 (BD = 6 and China = 5), six COPD (UK = 6) and four URTI (USA = 4) samples to unravel the microbiome diversity and related genomic metabolic functions. The metagenomic data mapped to 534 bacterial, 60 archaeal and 61 viral genomes with distinct variation in the microbiome composition across the samples (COVID-19 &gt; COPD &gt; URTI). Notably, 94.57%, 80.0% and 24.59% bacterial, archaeal and viral genera shared between the COVID-19 and non-COVID samples, respectively. However, the COVID-19 related samples had sole association with 16 viral genera other than SARS-CoV-2. Strain-level virome profiling revealed 660 and 729 strains in COVID-19 and non-COVID samples, respectively, and of them 34.50% strains shared between the conditions. Functional annotation of the metagenomic data identified the association of several biochemical pathways related to basic metabolism (amino acid and energy), ABC transporters, membrane transport, virulence, disease and defense, regulation of virulence, programmed cell death, and primary immunodeficiency. We also detected 30 functional gene groups/classes associated with resistance to antibiotics and toxic compounds (RATC) in both COVID-19 and non-COVID microbiomes. Furthermore, we detected comparatively higher abundance of cobalt-zinc-cadmium resistance (CZCR) and multidrug resistance to efflux pumps (MREP) genes in COVID-19 metagenome. The profiles of microbiome diversity and associated microbial genomic features found in both COVID-19 and non-COVID (COPD and URTI) samples might be helpful in developing microbiome-based diagnostics and therapeutics for COVID-19 and non-COVID respiratory diseases. However, future studies might be carried out to explore the microbiome dynamics and the cross-talk between host and microbiomes employing larger volume of samples from different ethnic groups and geoclimatic conditions.

[Source](https://doi.org/10.1038/s41598-021-85750-0)

10.1016/j.genrep.2021.101200

---

## Functional Profiling Of COVID-19 Respiratory Tract Microbiomes

[Source](https://doi.org/10.1038/s41598-021-85750-0)

---

## Signatures of COVID-19 severity and immune response in the respiratory tract microbiome.
 29.05.2021, _C Merenstein, G Liang, SA Whiteside, AG Cobián-Güemes, MS Merlino, LJ Taylor, A Glascock, K Bittinger, C Tanes, J Graham-Wooten, LA Khatib, AS Fitzgerald, S Reddy, AE Baxter, JR Giles, DA Oldridge, NJ Meyer, EJ Wherry, JE McGinniss, FD Bushman, RG Collman_


Viral infection of the respiratory tract can be associated with propagating effects on the airway microbiome, and microbiome dysbiosis may influence viral disease.

[Source](https://doi.org/10.1101/2021.04.02.21254514)

10.1101/2021.04.02.21254514

---

## Signatures Of COVID-19 Severity And Immune Response In The Respiratory Tract microbiome

[Source](https://doi.org/10.1101/2021.04.02.21254514)

---

## Plasma microbiome in COVID-19 subjects: an indicator of gut barrier defects and dysbiosis.
 14.04.2021, _R Prasad, MJ Patton, JL Floyd, CP Vieira, S Fortmann, M DuPont, A Harbour, CS Jeremy, J Wright, R Lamendella, BR Stevens, MB Grant_


The gut is a well-established route of infection and target for viral damage by SARS-CoV-2. This is supported by the clinical observation that about half of COVID-19 patients exhibit gastrointestinal ( GI ) symptoms. We asked whether the analysis of plasma could provide insight into gut barrier dysfunction in patients with COVID-19 infection. Plasma samples of COVID-19 patients (n=30) and healthy control (n=16) were collected during hospitalization. Plasma microbiome was analyzed using 16S rRNA sequencing, metatranscriptomic analysis, and gut permeability markers including FABP-2, PGN and LPS in both patient cohorts. Almost 65% (9 out 14) COVID-19 patients showed abnormal presence of gut microbes in their bloodstream. Plasma samples contained predominately Proteobacteria, Firmicutes, and Actinobacteria . The abundance of gram-negative bacteria ( Acinetobacter, Nitrospirillum, Cupriavidus, Pseudomonas, Aquabacterium, Burkholderia, Caballeronia, Parabhurkholderia, Bravibacterium, and Sphingomonas ) was higher than the gram-positive bacteria ( Staphylococcus and Lactobacillus ) in COVID-19 subjects. The levels of plasma gut permeability markers FABP2 (1282±199.6 vs 838.1±91.33; p=0.0757), PGN (34.64±3.178 vs 17.53±2.12; p&lt;0.0001), and LPS (405.5±48.37 vs 249.6±17.06; p=0.0049) were higher in COVID-19 patients compared to healthy subjects. These findings support that the intestine may represent a source for bacteremia and may contribute to worsening COVID-19 outcomes. Therapies targeting the gut and prevention of gut barrier defects may represent a strategy to improve outcomes in COVID-19 patients.

[Source](https://doi.org/10.1101/2021.04.06.438634)

10.1101/2021.04.06.438634

---

## Plasma Microbiome In COVID-19 Subjects: An Indicator Of Gut Barrier Defects And dysbiosis

[Source](https://doi.org/10.1101/2021.04.06.438634)

---

## Evaluation of superinfection, antimicrobial usage, and airway microbiome with metagenomic sequencing in COVID-19 patients: A cohort study in Shanghai.
 11.06.2021, _Q Miao, Y Ma, Y Ling, W Jin, Y Su, Q Wang, J Pan, Y Zhang, H Chen, J Yuan, H Wu, B Hu_


In COVID-19 patients, information regarding superinfection, antimicrobial assessment, and the value of metagenomic sequencing (MS) could help develop antimicrobial stewardship.

[Source](https://doi.org/10.1016/j.jmii.2021.03.015)

10.1016/j.jmii.2021.03.015

---

## Evaluation Of Superinfection, Antimicrobial Usage, And Airway Microbiome With metagenomic sequencing in COVID-19 patients: A cohort study in Shanghai

[Source](https://doi.org/10.1016/j.jmii.2021.03.015)

---

## COVID-19 Influences Lung Microbiota Dynamics And Favors The Emergence Of Rare infectious diseases: A case report of <em>Hafnia alvei</em> pneumonia

[Source](https://doi.org/10.1016/j.jcrc.2021.04.008)

---

## Impaired immune signaling and changes in the lung microbiome precede secondary bacterial pneumonia in COVID-19.
 27.05.2021, _A Tsitsiklis, B Zha, A Byrne, C DeVoe, S Levan, E Rackaityte, S Sunshine, E Mick, R Ghale, A Jauregui, N Neff, A Sarma, P Serpa, T Deiss, A Kistler, S Carrillo, KM Ansel, A Leligdowicz, S Christenson, N Jones, B Wu, S Darmanis, M Matthay, S Lynch, J DeRisi, C Consortium, C Hendrickson, K Kangelaris, M Krummel, P Woodruff, D Erle, O Rosenberg, C Calfee, C Langelier_


Secondary bacterial infections, including ventilator-associated pneumonia (VAP), lead to worse clinical outcomes and increased mortality following viral respiratory infections including in patients with coronavirus disease 2019 (COVID-19). Using a combination of tracheal aspirate bulk and single-cell RNA sequencing (scRNA-seq) we assessed lower respiratory tract immune responses and microbiome dynamics in 28 COVID-19 patients, 15 of whom developed VAP, and eight critically ill uninfected controls. Two days before VAP onset we observed a transcriptional signature of bacterial infection. Two weeks prior to VAP onset, following intubation, we observed a striking impairment in immune signaling in COVID-19 patients who developed VAP. Longitudinal metatranscriptomic analysis revealed disruption of lung microbiome community composition in patients with VAP, providing a connection between dysregulated immune signaling and outgrowth of opportunistic pathogens. These findings suggest that COVID-19 patients who develop VAP have impaired antibacterial immune defense detectable weeks before secondary infection onset.

[Source](https://doi.org/10.1038/s41598-021-89516-6)

10.21203/rs.3.rs-380803/v1

---

## The Lower Respiratory Tract Microbiome Of Critically Ill Patients With COVID-19

[Source](https://doi.org/10.1038/s41598-021-89516-6)

---

## Metagenomic analysis reveals oropharyngeal microbiota alterations in patients with COVID-19.
 31.05.2021, _S Ma, F Zhang, F Zhou, H Li, W Ge, R Gan, H Nie, B Li, Y Wang, M Wu, D Li, D Wang, Z Wang, Y You, Z Huang_


_Adult, Bacteria, COVID-19, Female, Humans, Male, Metagenomics, Microbiota, Middle Aged, Oropharynx, SARS-CoV-2_
COVID-19 remains a serious emerging global health problem, and little is known about the role of oropharynx commensal microbes in infection susceptibility and severity. Here, we present the oropharyngeal microbiota characteristics identified by shotgun metagenomic sequencing analyses of oropharynx swab specimens from 31 COVID-19 patients, 29 influenza B patients, and 28 healthy controls. Our results revealed a distinct oropharyngeal microbiota composition in the COVID-19 patients, characterized by enrichment of opportunistic pathogens such as Veillonella and Megasphaera and depletion of Pseudopropionibacterium, Rothia, and Streptococcus. Based on the relative abundance of the oropharyngeal microbiome, we built a microbial classifier to distinguish COVID-19 patients from flu patients and healthy controls with an AUC of 0.889, in which Veillonella was identified as the most prominent biomarker for COVID-19 group. Several members of the genus Veillonella, especially Veillonella parvula which was highly enriched in the oropharynx of our COVID-19 patients, were also overrepresented in the BALF of COVID-19 patients, indicating that the oral cavity acts as a natural reservoir for pathogens to induce co-infections in the lungs of COVID-19 patients. We also found the increased ratios of Klebsiella sp., Acinetobacter sp., and Serratia sp. were correlated with both disease severity and elevated systemic inflammation markers (neutrophil-lymphocyte ratio, NLR), suggesting that these oropharynx microbiota alterations may impact COVID-19 severity by influencing the inflammatory response. Moreover, the oropharyngeal microbiome of COVID-19 patients exhibited a significant enrichment in amino acid metabolism and xenobiotic biodegradation and metabolism. In addition, all 26 drug classes of antimicrobial resistance genes were detected in the COVID-19 group, and were significantly enriched in critical cases. In conclusion, we found that oropharyngeal microbiota alterations and functional differences were associated with COVID-19 severity.

[Source](https://doi.org/10.1038/s41392-021-00614-3)

10.1038/s41392-021-00614-3

---

## Metagenomic Analysis Reveals Oropharyngeal Microbiota Alterations In Patients With COVID-19

[Source](https://doi.org/10.1038/s41392-021-00614-3)

---

## Acute SARS-CoV-2 Infection Is Associated With An Expansion Of Bacteria Pathogens In the nose including <em>Pseudomonas aeruginosa</em>

[Source](https://doi.org/10.1101/2021.05.20.445008)

---

## Validation of High-Sensitivity Severe Acute Respiratory Syndrome Coronavirus 2 Testing for Stool-Toward the New Normal for Fecal Microbiota Transplantation.
 17.06.2021, _A Babiker, JM Ingersoll, MW Adelman, AS Webster, KJ Broder, V Stittleburg, JJ Waggoner, CS Kraft, MH Woodworth_


_COVID-19, COVID-19 Testing, Centers for Disease Control and Prevention, U.S., Fecal Microbiota Transplantation, Feces, Humans, Nasopharynx, RNA, Viral, Reverse Transcriptase Polymerase Chain Reaction, SARS-CoV-2, Tissue Donors, United States_
Mounting evidence demonstrates potential for fecal-oral transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The US Food and Drug Administration now requires SARS-CoV-2 testing of potential feces donors before the use of stool manufactured for fecal microbiota transplantation. We sought to develop and validate a high-sensitivity SARS-CoV-2 reverse transcriptase polymerase chain reaction (RT-PCR) procedure for testing stool specimens.

[Source](https://doi.org/10.1186/s12575-021-00148-6)

10.14309/ctg.0000000000000363

---

## Nasopharyngeal Microbiota In SARS-CoV-2 Positive And Negative Patients

[Source](https://doi.org/10.1186/s12575-021-00148-6)

---

## SARS-CoV-2 detection status associates with bacterial community composition in patients and the hospital environment.
 12.06.2021, _C Marotz, P Belda-Ferre, F Ali, P Das, S Huang, K Cantrell, L Jiang, C Martino, RE Diner, G Rahman, D McDonald, G Armstrong, S Kodera, S Donato, G Ecklu-Mensah, N Gottel, MC Salas Garcia, LY Chiang, RA Salido, JP Shaffer, MK Bryant, K Sanders, G Humphrey, G Ackermann, N Haiminen, KL Beck, HC Kim, AP Carrieri, L Parida, Y Vázquez-Baeza, FJ Torriani, R Knight, J Gilbert, DA Sweeney, SM Allard_


_COVID-19, Hospitals, Humans, Pandemics, Phylogeny, RNA, Ribosomal, 16S, RNA, Viral, SARS-CoV-2_
SARS-CoV-2 is an RNA virus responsible for the coronavirus disease 2019 (COVID-19) pandemic. Viruses exist in complex microbial environments, and recent studies have revealed both synergistic and antagonistic effects of specific bacterial taxa on viral prevalence and infectivity. We set out to test whether specific bacterial communities predict SARS-CoV-2 occurrence in a hospital setting.

[Source](https://doi.org/10.1186/s40168-021-01083-0)

10.1186/s40168-021-01083-0

---

## SARS-CoV-2 Detection Status Associates With Bacterial Community Composition In patients and the hospital environment

[Source](https://doi.org/10.1186/s40168-021-01083-0)

---

## Gnotobiotic Rats Reveal That Gut Microbiota Regulates Colonic mRNA of Ace2, the Receptor for SARS-CoV-2 Infectivity.
 18.12.2020, _T Yang, S Chakraborty, P Saha, B Mell, X Cheng, JY Yeo, X Mei, G Zhou, J Mandal, R Golonka, BS Yeoh, V Putluri, DWB Piyarathna, N Putluri, CG McCarthy, CF Wenceslau, A Sreekumar, AT Gewirtz, M Vijay-Kumar, B Joe_


_Angiotensin-Converting Enzyme 2, Animals, Betacoronavirus, COVID-19, Colitis, Colon, Coronavirus Infections, Correlation of Data, Gastrointestinal Microbiome, Gene Expression Regulation, Germ-Free Life, Pandemics, Peptidyl-Dipeptidase A, Pneumonia, Viral, RNA, Messenger, Rats, SARS-CoV-2_

[Source](https://doi.org/10.1161/HYPERTENSIONAHA.120.15360)

10.1161/HYPERTENSIONAHA.120.15360

---

## Temporal landscape of human gut RNA and DNA virome in SARS-CoV-2 infection and severity.
 27.04.2021, _T Zuo, Q Liu, F Zhang, YK Yeoh, Y Wan, H Zhan, GCY Lui, Z Chen, AYL Li, CP Cheung, N Chen, W Lv, RWY Ng, EYK Tso, KSC Fung, V Chan, L Ling, G Joynt, DSC Hui, FKL Chan, PKS Chan, SC Ng_


_COVID-19, Child, Preschool, DNA, Gastrointestinal Microbiome, Humans, RNA, SARS-CoV-2, Virome_
Coronavirus disease 2019 (COVID-19) caused by the enveloped RNA virus SARS-CoV-2 primarily affects the respiratory and gastrointestinal tracts. SARS-CoV-2 was isolated from fecal samples, and active viral replication was reported in human intestinal cells. The human gut also harbors an enormous amount of resident viruses (collectively known as the virome) that play a role in regulating host immunity and disease pathophysiology. Understanding gut virome perturbation that underlies SARS-CoV-2 infection and severity is an unmet need.

[Source](https://doi.org/10.1053/j.gastro.2020.05.048)

10.1186/s40168-021-01008-x

---

## Alterations In Gut Microbiota Of Patients With COVID-19 During Time Of Hospitalization

[Source](https://doi.org/10.1053/j.gastro.2020.05.048)

---

## Alterations in Fecal Fungal Microbiome of Patients With COVID-19 During Time of Hospitalization until Discharge.
 05.05.2021, _T Zuo, H Zhan, F Zhang, Q Liu, EYK Tso, GCY Lui, N Chen, A Li, W Lu, FKL Chan, PKS Chan, SC Ng_


_Adult, Aged, Aspergillus flavus, Aspergillus niger, Betacoronavirus, COVID-19, Candida, Candida albicans, Case-Control Studies, Community-Acquired Infections, Coronavirus Infections, DNA, Fungal, Feces, Female, Fungi, Gastrointestinal Microbiome, Humans, Male, Metagenomics, Middle Aged, Mycobiome, Nasopharynx, Pandemics, Patient Discharge, Pneumonia, Pneumonia, Viral, SARS-CoV-2, Time Factors, Young Adult_
Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infects intestinal cells, and might affect the intestinal microbiota. We investigated changes in the fecal fungal microbiomes (mycobiome) of patients with SARS-CoV-2 infection during hospitalization and on recovery.

[Source](https://doi.org/10.1053/j.gastro.2020.06.048)

10.1053/j.gastro.2020.06.048

---

## Gut microbiota composition reflects disease severity and dysfunctional immune responses in patients with COVID-19.
 15.03.2021, _YK Yeoh, T Zuo, GC Lui, F Zhang, Q Liu, AY Li, AC Chung, CP Cheung, EY Tso, KS Fung, V Chan, L Ling, G Joynt, DS Hui, KM Chow, SSS Ng, TC Li, RW Ng, TC Yip, GL Wong, FK Chan, CK Wong, PK Chan, SC Ng_


_Adult, Bacteria, C-Reactive Protein, COVID-19, Cytokines, DNA, Bacterial, Dysbiosis, Female, Gastrointestinal Microbiome, Gastrointestinal Tract, Hong Kong, Humans, Immunity, Male, SARS-CoV-2, Severity of Illness Index, Transferases_
Although COVID-19 is primarily a respiratory illness, there is mounting evidence suggesting that the GI tract is involved in this disease. We investigated whether the gut microbiome is linked to disease severity in patients with COVID-19, and whether perturbations in microbiome composition, if any, resolve with clearance of the SARS-CoV-2 virus.

[Source](https://doi.org/10.1136/gutjnl-2020-323020)

10.1136/gutjnl-2020-323020

---

## Gut Microbiota Composition Reflects Disease Severity And Dysfunctional Immune responses in patients with COVID-19

[Source](https://doi.org/10.1136/gutjnl-2020-323020)

---

## The faecal metabolome in COVID-19 patients is altered and associated with clinical features and gut microbes.
 09.03.2021, _L Lv, H Jiang, Y Chen, S Gu, J Xia, H Zhang, Y Lu, R Yan, L Li_


_Adult, Bacteria, COVID-19, Cohort Studies, Dysbiosis, Feces, Female, Fungi, Gas Chromatography-Mass Spectrometry, Gastrointestinal Microbiome, Humans, Male, Metabolome, Middle Aged, SARS-CoV-2_
Although SARS-CoV-2 can invade the intestine, though its effect on digestion and absorption is not fully understood. In the present study, 56 COVID-19 patients and 47 age- and sex-matched healthy subjects were divided into a discovery cohort and a validation cohort. Blood, faeces and clinical information were collected from the patients in the hospital and at discharge. The faecal metabolome was analysed using gas chromatography-mass spectrometry, and Spearman's correlation analyses of clinical features, the serum metabolome, and the faecal micro- and mycobiota were conducted. The results showed that, the faeces of COVID-19 patients were enriched with important nutrients that should be metabolized or absorbed, such as sucrose and 2-palmitoyl-glycerol; diet-related components that cannot be synthesized by humans, such as 1,5-anhydroglucitol and D-pinitol; and harmful metabolites, such as oxalate, were also detected. In contrast, purine metabolites such as deoxyinosine and hypoxanthine, low-water-soluble long-chain fatty alcohols/acids such as behenic acid, compounds rarely occurring in nature such as D-allose and D-arabinose, and microbe-related compounds such as 2,4-di-tert-butylphenol were depleted in the faeces of COVID-19 patients. Moreover, these metabolites significantly correlated with altered serum metabolites such as oxalate and gut microbesincluding Ruminococcaceae, Actinomyces, Sphingomonas and Aspergillus. Although levels of several faecal metabolites, such as sucrose, 1,5-anhydroglucitol and D-pinitol, of discharged patients were not different from those of healthy controls (HCs), those of oxalate and 2-palmitoyl-glycerol did differ. Therefore, alterations in the faecal metabolome of COVID-19 patients may reflect malnutrition and intestinal inflammation and warrant greater attention. The results of present study provide new insights into the pathogenesis and treatment of COVID-19.

[Source](https://doi.org/10.1016/j.aca.2021.338267)

10.1016/j.aca.2021.338267

---

## The Faecal Metabolome In COVID-19 Patients Is Altered And Associated With Clinical features and gut microbes

[Source](https://doi.org/10.1016/j.aca.2021.338267)

---

## 16S rRNA Gene Sequencing Of Rectal Swab In Patients Affected By COVID-19

[Source](https://doi.org/10.1371/journal.pone.0247041)

---

## Investigation of Nasal/Oropharyngeal Microbial Community of COVID-19 Patients by 16S rDNA Sequencing.
 12.03.2021, _M Rueca, A Fontana, B Bartolini, P Piselli, A Mazzarelli, M Copetti, E Binda, F Perri, CEM Gruber, E Nicastri, L Marchioni, G Ippolito, MR Capobianchi, A Di Caro, V Pazienza_


_Adult, Aged, Bacteria, COVID-19, Female, Humans, Male, Microbiota, Middle Aged, Nose, Oropharynx, RNA, Ribosomal, 16S, Young Adult_
Since December 2019, SARS-CoV-2 infection has been still rapidly spreading, resulting in a pandemic, followed by an increasing number of cases in countries throughout the world. The severity of the disease depends on the patient's overall medical condition but no appropriate markers are available to establish the prognosis of the patients. We performed a 16S rRNA gene sequencing, revealing an altered composition of the nasal/oropharyngeal (NOP) microbiota in 21 patients affected by COVID-19, paucisymptomatic or in an Intensive Care Unit (ICU), as compared to 10 controls negative for COVID-19 or eight affected by a different Human Coronavirus (HKU, NL63 and OC43). A significant decrease in Chao1 index was observed when patients affected by COVID-19 (in ICU) were compared to paucisymptomatic. Furthermore, patients who were in ICU, paucisymptomatic or affected by other Coronaviruses all displayed a decrease in the Chao1 index when compared to controls, while Shannon index significantly decreased only in patients under ICU as compared to controls and paucisymptomatic patients. At the phylum level, Deinococcus-Thermus was present only in controls as compared to SARS-CoV-2 patients admitted to ICU, paucisymptomatic or affected by other coronaviruses. Candidatus Saccharibacteria (formerly known as TM7) was strongly increased in negative controls and SARS-CoV-2 paucisymptomatic patients as compared to SARS-CoV-2 ICU patients. Other modifications were observed at a lower taxonomy level. Complete depletion of Bifidobacterium and Clostridium was exclusively observed in ICU SARS-CoV-2 patients, which was the only group characterized by the presence of Salmonella, Scardovia, Serratia and Pectobacteriaceae. In conclusion, our preliminary results showed that nasal/oropharyngeal microbiota profiles of patients affected with SARS-CoV-2 may provide valuable information in order to facilitate the stratification of patients and may open the way to new interventional strategies in order to ameliorate the outcome of the patients.

[Source](https://doi.org/10.3390/ijerph18042174)

10.3390/ijerph18042174

---

## Integrated gut virome and bacteriome dynamics in COVID-19 patients.
 28.04.2021, _J Cao, C Wang, Y Zhang, G Lei, K Xu, N Zhao, J Lu, F Meng, L Yu, J Yan, C Bai, S Zhang, N Zhang, Y Gong, Y Bi, Y Shi, Z Chen, L Dai, J Wang, P Yang_


_Adolescent, Adult, Aged, Aged, 80 and over, Animals, Anti-Bacterial Agents, COVID-19, Case-Control Studies, China, Disease Models, Animal, Dysbiosis, Female, Gastrointestinal Microbiome, Genome, Viral, Humans, Male, Mice, Mice, Inbred C57BL, MicroRNAs, Middle Aged, Transcriptome, Virome_
SARS-CoV-2 is the cause of the current global pandemic of COVID-19; this virus infects multiple organs, such as the lungs and gastrointestinal tract. The microbiome in these organs, including the bacteriome and virome, responds to infection and might also influence disease progression and treatment outcome. In a cohort of 13 COVID-19 patients in Beijing, China, we observed that the gut virome and bacteriome in the COVID-19 patients were notably different from those of five healthy controls. We identified a bacterial dysbiosis signature by observing reduced diversity and viral shifts in patients, and among the patients, the bacterial/viral compositions were different between patients of different severities, although these differences are not entirely distinguishable from the effect of antibiotics. Severe cases of COVID-19 exhibited a greater abundance of opportunistic pathogens but were depleted for butyrate-producing groups of bacteria compared with mild to moderate cases. We replicated our findings in a mouse COVID-19 model, confirmed virome differences and bacteriome dysbiosis due to SARS-CoV-2 infection, and observed that immune/infection-related genes were differentially expressed in gut epithelial cells during infection, possibly explaining the virome and bacteriome dynamics. Our results suggest that the components of the microbiome, including the bacteriome and virome, are affected by SARS-CoV-2 infections, while their compositional signatures could reflect or even contribute to disease severity and recovery processes.

[Source](https://doi.org/10.1080/19490976.2021.1887722)

10.1080/19490976.2021.1887722

---

## Integrated Gut Virome And Bacteriome Dynamics In COVID-19 Patients

[Source](https://doi.org/10.1080/19490976.2021.1887722)

---

## Six-month follow-up of gut microbiota richness in patients with COVID-19.
 09.04.2021, _Y Chen, S Gu, Y Chen, H Lu, D Shi, J Guo, WR Wu, Y Yang, Y Li, KJ Xu, C Ding, R Luo, C Huang, L Yu, M Xu, P Yi, J Liu, JJ Tao, H Zhang, L Lv, B Wang, J Sheng, L Li_



[Source](https://doi.org/10.1136/gutjnl-2021-324090)

10.1136/gutjnl-2021-324090

---

## Six-month Follow-up Of Gut Microbiota Richness In Patients With COVID-19

[Source](https://doi.org/10.1136/gutjnl-2021-324090)

---

## Gut mycobiota alterations in patients with COVID-19 and H1N1 infections and their associations with clinical features.
 10.05.2021, _L Lv, S Gu, H Jiang, R Yan, Y Chen, Y Chen, R Luo, C Huang, H Lu, B Zheng, H Zhang, J Xia, L Tang, G Sheng, L Li_


_Adult, Aged, Bacteria, COVID-19, Dysbiosis, Feces, Female, Fungi, Gastrointestinal Microbiome, Humans, Influenza A Virus, H1N1 Subtype, Influenza, Human, Male, Middle Aged, SARS-CoV-2, Sequence Analysis, DNA_
The relationship between gut microbes and COVID-19 or H1N1 infections is not fully understood. Here, we compared the gut mycobiota of 67 COVID-19 patients, 35 H1N1-infected patients and 48 healthy controls (HCs) using internal transcribed spacer (ITS) 3-ITS4 sequencing and analysed their associations with clinical features and the bacterial microbiota. Compared to HCs, the fungal burden was higher. Fungal mycobiota dysbiosis in both COVID-19 and H1N1-infected patients was mainly characterized by the depletion of fungi such as Aspergillus and Penicillium, but several fungi, including Candida glabrata, were enriched in H1N1-infected patients. The gut mycobiota profiles in COVID-19 patients with mild and severe symptoms were similar. Hospitalization had no apparent additional effects. In COVID-19 patients, Mucoromycota was positively correlated with Fusicatenibacter, Aspergillus niger was positively correlated with diarrhoea, and Penicillium citrinum was negatively correlated with C-reactive protein (CRP). In H1N1-infected patients, Aspergillus penicilloides was positively correlated with Lachnospiraceae members, Aspergillus was positively correlated with CRP, and Mucoromycota was negatively correlated with procalcitonin. Therefore, gut mycobiota dysbiosis occurs in both COVID-19 patients and H1N1-infected patients and does not improve until the patients are discharged and no longer require medical attention.

[Source](https://doi.org/10.1038/s42003-021-02036-x)

10.1038/s42003-021-02036-x

---

## Gut Mycobiota Alterations In Patients With COVID-19 And H1N1 Infections And Their associations with clinical features

[Source](https://doi.org/10.1038/s42003-021-02036-x)

---

## Gut microbiome and resistome changes during the first wave of the COVID-19 pandemic in comparison with pre-pandemic travel-related changes.
 22.05.2021, _Y Peng, D Zhang, T Chen, Y Xia, P Wu, WK Seto, AL Kozyrskyj, BJ Cowling, J Zhao, HM Tun_



[Source](https://doi.org/10.1093/jtm/taab067)

10.1093/jtm/taab067

---

## Gut Microbiome And Resistome Changes During The First Wave Of The COVID-19 Pandemic in comparison with pre-pandemic travel-related changes

[Source](https://doi.org/10.1093/jtm/taab067)

---

## Bacterial communities in the rumen and feces of lactating Holstein dairy cows are not affected when fed reduced-fat dried distillers' grains with solubles.
 21.06.2021, _AS Dankwa, U Humagain, SL Ishaq, CJ Yeoman, S Clark, DC Beitz, ED Testroet_


Reduced-fat dried distillers' grains with solubles (RF-DDGSs) are co-products of ethanol production and contain less fat than traditional distillers' grains. The fat in corn is ~91% unsaturated, and it is toxic to rumen microorganisms so it could influence the composition of the rumen microbiome. It has been demonstrated that RF-DDGS is a suitable ration ingredient to support the high-producing dairy cow, and this feedstuff is a promising alternative protein source for lactating dairy cows. The current study aims to better understand the effect of RF-DDGS on the rumen and fecal bacterial composition in lactating dairy cows. Thirty-six multiparous (two or three), mid-lactation Holstein cows (BW = 680 ± 11 kg; 106 ± 27 DIM) were randomly assigned to two groups which were fed a control diet made up of corn, corn silage, and alfalfa hay supplemented with expeller soybean meal or with added RF-DDGS (20% of the DM) containing approximately 6.0% fat. Whole rumen contents (rumen fluid and digesta; esophageal tubing method) and feces (free-catch method) were collected on day 35 of the experimental period, after the 14-d acclimation period. Rumen contents and feces from each cow were used for DNA extraction. The bacterial community composition in rumen and fecal samples was assessed via the 16S rRNA gene by using the Illumina MiSeq sequencing platform. Bacteroidetes, Actinobacteria, and Firmicutes were the most abundant phyla in rumen contents. The fecal microbiota was dominated by the phyla Firmicutes and Bacteroidetes, as well as Actinobacteria and Chloroflexi. RF-DGGS increased bacterial richness, evenness, and Shannon diversity in both rumen and fecal samples and was associated with several taxa that had different abundance in treatment versus control comparisons. The RF-DGGS, however, did not significantly alter the bacterial community in the rumen or feces. In general, these findings demonstrated that dietary inclusion of RF-DDGS did not impose any serious short-term (within 30 days) health or production consequences, as would be expected. With this study, we present further evidence that inclusion of 20% (DM basis) RF-DDGS in the diet of lactating dairy cows can be done without consequence on the microbiome of the rumen.

[Source](https://doi.org/10.3389/fnut.2021.638825)

10.1016/j.animal.2021.100281

---

## Gut Microbiota May Not Be Fully Restored In Recovered COVID-19 Patients After 3-Month Recovery

[Source](https://doi.org/10.3389/fnut.2021.638825)

---

## The gut microbiome of COVID-19 recovered patients returns to uninfected status in a minority-dominated United States cohort.
 21.06.2021, _RC Newsome, J Gauthier, MC Hernandez, GE Abraham, TO Robinson, HB Williams, M Sloan, A Owings, H Laird, T Christian, Y Pride, KJ Wilson, M Hasan, A Parker, M Senitko, SC Glover, RZ Gharaibeh, C Jobin_


_Aged, Bacteria, COVID-19, Cohort Studies, Feces, Female, Gastrointestinal Microbiome, Humans, Male, Middle Aged, RNA, Ribosomal, 16S, RNA, Viral, SARS-CoV-2, Severity of Illness Index, United States_
To investigate the relationship between intestinal microbiota and SARS-CoV-2-mediated pathogenicity in a United States, majority African American cohort. We prospectively collected fecal samples from 50 SARS-CoV-2 infected patients, 9 SARS-CoV-2 recovered patients, and 34 uninfected subjects seen by the hospital with unrelated respiratory medical conditions (controls). 16S rRNA sequencing and qPCR analysis was performed on fecal DNA/RNA. The fecal microbial composition was found to be significantly different between SARS-CoV-2 patients and controls (PERMANOVA FDR-P = .004), independent of antibiotic exposure. Peptoniphilus, Corynebacterium and Campylobacter were identified as the three most significantly enriched genera in COVID-19 patients compared to controls. Actively infected patients were also found to have a different gut microbiota than recovered patients (PERMANOVA FDR-P = .003), and the most enriched genus in infected patients was Campylobacter, with Agathobacter and Faecalibacterium being enriched in the recovered patients. No difference in microbial community structure between recovered patients and uninfected controls was observed, nor a difference in alpha diversity between the three groups. 24 of the 50 COVID-19 patients (48%) tested positive via RT-qPCR for fecal SARS-CoV-2 RNA. A significant difference in gut microbial composition between SARS-CoV-2 positive and negative samples was observed, with Klebsiella and Agathobacter being enriched in the positive cohort. No significant associations between microbiome composition and disease severity was found. The intestinal microbiota is sensitive to the presence of SARS-CoV-2, with increased relative abundance of genera (Campylobacter, Klebsiella) associated with gastrointestinal (GI) disease. Further studies are needed to investigate the functional impact of SARS-CoV-2 on GI health.

[Source](https://doi.org/10.1053/j.gastro.2021.05.006)

10.1080/19490976.2021.1926840

---

## Gut Microbiome Composition Is Associated With COVID-19 Disease Severity

[Source](https://doi.org/10.1053/j.gastro.2021.05.006)

---

## The Gut Microbiome Of COVID-19 Recovered Patients Returns To Uninfected Status In A minority-dominated United States cohort

[Source](https://doi.org/10.1080/19490976.2021.1926840)

---

## Intestinal permeability changes with bacterial translocation as key events modulating systemic host immune response to SARS-CoV-2: A working hypothesis.
 10.01.2021, _V Cardinale, G Capurso, G Ianiro, A Gasbarrini, PG Arcidiacono, D Alvaro_


_Alarmins, Angiotensin-Converting Enzyme 2, Bacterial Translocation, COVID-19, Cytokine Release Syndrome, Disease Progression, Gastrointestinal Microbiome, Humans, Immunity, Inflammation, Interleukin-6, Intestinal Mucosa, Lipopolysaccharides, Liver, Lung, Microbiota, Pathogen-Associated Molecular Pattern Molecules, Permeability, SARS-CoV-2, Serine Endopeptidases_
The microbiota-gut-liver-lung axis plays a bidirectional role in the pathophysiology of a number of infectious diseases. During the course of severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and 2 (SARS-CoV-2) infection, this pathway is unbalanced due to intestinal involvement and systemic inflammatory response. Moreover, there is convincing preliminary evidence linking microbiota-gut-liver axis perturbations, proinflammatory status, and endothelial damage in noncommunicable preventable diseases with coronavirus disease 2019 (Covid-19) severity. Intestinal damage due to SARS-CoV-2 infection, systemic inflammation-induced dysfunction, and IL-6-mediated diffuse vascular damage may increase intestinal permeability and precipitate bacterial translocation. The systemic release of damage- and pathogen-associated molecular patterns (e.g. lipopolysaccharides) and consequent immune-activation may in turn auto-fuel vicious cycles of systemic inflammation and tissue damage. Thus, intestinal bacterial translocation may play an additive/synergistic role in the cytokine release syndrome in Covid-19. This review provides evidence on gut-liver axis involvement in Covid-19 as well as insights into the hypothesis that intestinal endotheliitis and permeability changes with bacterial translocation are key pathophysiologic events modulating systemic inflammatory response. Moreover, it presents an overview of readily applicable measures for the modulation of the gut-liver axis and microbiota in clinical practice.

[Source](https://doi.org/10.1016/j.dld.2020.09.009)

10.1016/j.dld.2020.09.009

---

## Intestinal Permeability Changes With Bacterial Translocation As Key Events modulating systemic host immune response to SARS-CoV-2: A working hypothesis

[Source](https://doi.org/10.1016/j.dld.2020.09.009)

---

## The potential application of probiotics and prebiotics for the prevention and treatment of COVID-19.
 22.10.2020, _AN Olaimat, I Aolymat, M Al-Holy, M Ayyash, M Abu Ghoush, AA Al-Nabulsi, T Osaili, V Apostolopoulos, SQ Liu, NP Shah_


COVID-19 is a pandemic disease caused by the novel coronavirus severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). This new viral infection was first identified in China in December 2019, and it has subsequently spread globally. The lack of a vaccine or curative treatment for COVID-19 necessitates a focus on other strategies to prevent and treat the infection. Probiotics consist of single or mixed cultures of live microorganisms that can beneficially affect the host by maintaining the intestinal or lung microbiota that play a major role in human health. At present, good scientific evidence exists to support the ability of probiotics to boost human immunity, thereby preventing colonization by pathogens and reducing the incidence and severity of infections. Herein, we present clinical studies of the use of probiotic supplementation to prevent or treat respiratory tract infections. These data lead to promising benefits of probiotics in reducing the risk of COVID-19. Further studies should be conducted to assess the ability of probiotics to combat COVID-19.

[Source](https://doi.org/10.1038/s41538-020-00078-9)

10.1038/s41538-020-00078-9

---

## The Potential Application Of Probiotics And Prebiotics For The Prevention And treatment of COVID-19

[Source](https://doi.org/10.1038/s41538-020-00078-9)

---

## The gut microbiome: an under-recognised contributor to the COVID-19 pandemic?
 08.12.2020, _JP Segal, JWY Mak, BH Mullish, JL Alexander, SC Ng, JR Marchesi_


The novel coronavirus infection (COVID-19) caused by the SARS-CoV-2 virus has spread rapidly across the globe, culminating in major global morbidity and mortality. As such, there has been a rapid escalation in scientific and clinical activity aimed at increasing our comprehension of this virus. This volume of work has led to early insights into risk factors associated with severity of disease, and mechanisms that underpin the virulence and dynamics involved in viral transmission. These insights ultimately may help guide potential therapeutics to reduce the human, economic and social impact of this pandemic. Importantly, the gastrointestinal (GI) tract has emerged as an important organ influencing propensity to, and potentially severity of, COVID-19 infection. Furthermore, the gut microbiome has been linked to a variety of risk factors for COVID-19 infection, and manipulation of the gut microbiome is an attractive potential therapeutic target for a number of diseases. While data profiling the gut microbiome in COVID-19 infection to date are limited, they support the possibility of several routes of interaction between COVID-19, the gut microbiome, angiotensin converting enzyme 2 (ACE-2) expression in the small bowel and colon and gut inflammation. This article will explore the evidence that implicates the gut microbiome as a contributing factor to the pathogenesis, severity and disease course of COVID-19, and speculate about the gut microbiome's capability as a therapeutic avenue against COVID-19.

[Source](https://doi.org/10.1177/1756284820974914)

10.1177/1756284820974914

---

## The Gut Microbiome: An Under-recognised Contributor To The COVID-19 Pandemic?

[Source](https://doi.org/10.1177/1756284820974914)

---

## Is Gut Microbiota Dysbiosis a Predictor of Increased Susceptibility to Poor Outcome of COVID-19 Patients? An Update.
 26.01.2021, _C Ferreira, SD Viana, F Reis_


The scientific knowledge already attained regarding the way severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infects human cells and the clinical manifestations and consequences for Coronavirus Disease 2019 (COVID-19) patients, especially the most severe cases, brought gut microbiota into the discussion. It has been suggested that intestinal microflora composition plays a role in this disease because of the following: (i) its relevance to an efficient immune system response; (ii) the fact that 5-10% of the patients present gastrointestinal symptoms; and (iii) because it is modulated by intestinal angiotensin-converting enzyme 2 (ACE2) (which is the virus receptor). In addition, it is known that the most severely affected patients (those who stay longer in hospital, who require intensive care, and who eventually die) are older people with pre-existing cardiovascular, metabolic, renal, and pulmonary diseases, the same people in which the prevalence of gut microflora dysbiosis is higher. The COVID-19 patients presenting poor outcomes are also those in which the immune system's hyperresponsiveness and a severe inflammatory condition (collectively referred as "cytokine storm") are particularly evident, and have been associated with impaired microbiota phenotype. In this article, we present the evidence existing thus far that may suggest an association between intestinal microbiota composition and the susceptibility of some patients to progress to severe stages of the disease.

[Source](https://doi.org/10.3390/microorganisms90100530)

10.3390/microorganisms9010053

---

## Do An Altered Gut Microbiota And An Associated Leaky Gut Affect COVID-19 Severity?

[Source](https://doi.org/10.1128/mBio.03022-20)

---

## The microbiome of bat guano: for what is this knowledge important?
 22.02.2021, _I Dimkić, D Fira, T Janakiev, J Kabić, M Stupar, M Nenadić, N Unković, ML Grbić_


_Animals, Biodiversity, Chiroptera, Communicable Diseases, Emerging, Conservation of Natural Resources, Disease Reservoirs, Feces, Humans, Microbiota_
Bats as flying mammals are potent vectors and natural reservoir hosts for many infectious viruses, bacteria, and fungi, also detected in their excreta such as guano. Accelerated deforestation, urbanization, and anthropization hastily lead to overpopulation of the bats in urban areas allowing easy interaction with other animals, expansion, and emergence of new zoonotic disease outbreaks potentially harmful to humans. Therefore, getting new insights in the microbiome of bat guano from different places represents an imperative for the future. Furthermore, the use of novel high-throughput sequencing technologies allows better insight in guano microbiome and potentially indicated that some species could be typical guano-dwelling members. Bats are well known as a natural reservoir of many zoonotic viruses such as Ebola, Nipah, Marburg, lyssaviruses, rabies, henipaviruses, and many coronaviruses which caused a high number of outbreaks including ongoing COVID-19 pandemic. Additionally, many bacterial and fungal pathogens were identified as common guano residents. Thus, the presence of multi-drug-resistant bacteria as environmental reservoirs of extended spectrum β-lactamases and carbapenemase-producing strains has been confirmed. Bat guano is the most suitable substrate for fungal reproduction and dissemination, including pathogenic yeasts and keratinophilic and dimorphic human pathogenic fungi known as notorious causative agents of severe endemic mycoses like histoplasmosis and fatal cryptococcosis, especially deadly in immunocompromised individuals. This review provides an overview of bat guano microbiota diversity and the significance of autochthonous and pathogenic taxa for humans and the environment, highlighting better understanding in preventing emerging diseases. KEY POINTS: Bat guano as reservoir and source for spreading of autochthonous and pathogenic microbiota Bat guano vs. novel zoonotic disease outbreaks Destruction of bat natural habitats urgently demands increased human awareness.

[Source](https://doi.org/10.1007/s00253-021-11143-y)

10.1007/s00253-021-11143-y

---

## The Microbiome Of Bat Guano: For What Is This Knowledge Important?

[Source](https://doi.org/10.1007/s00253-021-11143-y)

---

## Microbiota Modulation of the Gut-Lung Axis in COVID-19.
 18.03.2021, _GLV de Oliveira, CNS Oliveira, CF Pinzan, LVV de Salis, CRB Cardoso_


_COVID-19, Dysbiosis, Gastrointestinal Microbiome, Humans, Intestines, Lung, Probiotics, SARS-CoV-2_
COVID-19 is an infectious disease caused by the Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2), and according to the World Health Organization (WHO), to date, SARS-CoV-2 has already infected more than 91.8 million people worldwide with 1,986,871 deaths. This virus affects mainly the respiratory system, but the gastrointestinal tract (GIT) is also a target, meanwhile SARS-CoV-2 was already detected in oesophagus, stomach, duodenum, rectum, and in fecal samples from COVID-19 patients. Prolonged GIT manifestations in COVID-19, mainly the diarrhea, were correlated with decreased richness and diversity of the gut microbiota, immune deregulation and delayed SARS-CoV-2 clearance. So, the bidirectional interactions between the respiratory mucosa and the gut microbiota, known as gut-lung axis, are supposed to be involved in the healthy or pathologic immune responses to SARS-CoV-2. In accordance, the intestinal dysbiosis is associated with increased mortality in other respiratory infections, due to an exacerbated inflammation and decreased regulatory or anti-inflammatory mechanisms in the lungs and in the gut, pointing to this important relationship between both mucosal compartments. Therefore, since the mucous membranes from the respiratory and gastrointestinal tracts are affected, in addition to dysbiosis and inflammation, it is plausible to assume that adjunctive therapies based on the modulation of the gut microbiota and re-establishment of eubiosis conditions could be an important therapeutic approach for constraining the harmful consequences of COVID-19. Then, in this review, we summarized studies showing the persistence of SARS-CoV-2 in the gastrointestinal system and the related digestive COVID-19 manifestations, in addition to the literature demonstrating nasopharyngeal, pulmonary and intestinal dysbiosis in COVID-19 patients. Lastly, we showed the potential beneficial role of probiotic administration in other respiratory infections, and discuss the possible role of probiotics as an adjunctive therapy in SARS-CoV-2 infection.

[Source](https://doi.org/10.3389/fimmu.2021.635471)

10.3389/fimmu.2021.635471

---

## Longitudinal Peripheral Blood Transcriptional Analysis Reveals Molecular Signatures of Disease Progression in COVID-19 Patients.
 03.05.2021, _Q Yan, P Li, X Ye, X Huang, B Feng, T Ji, Z Chen, F Li, Y Zhang, K Luo, F Chen, X Mo, J Wang, L Feng, F Hu, C Lei, L Qu, L Chen_


_Adult, Aged, COVID-19, Cytokines, Disease Progression, Female, Gene Expression Regulation, Humans, Inflammation Mediators, Leukocytes, Mononuclear, Longitudinal Studies, Male, Middle Aged, RNA-Seq, SARS-CoV-2_
Coronavirus disease 2019 (COVID-19) is caused by a novel coronavirus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), with some patients developing severe illness or even death. Disease severity has been associated with increased levels of proinflammatory cytokines and lymphopenia. To elucidate the atlas of peripheral immune response and pathways that might lead to immunopathology during COVID-19 disease course, we performed a peripheral blood RNA sequencing analysis of the same patient's samples collected from symptom onset to full recovery. We found that PBMCs at different disease stages exhibited unique transcriptome characteristics. We observed that SARS-CoV-2 infection caused excessive release of inflammatory cytokines and lipid mediators as well as an aberrant increase of low-density neutrophils. Further analysis revealed an increased expression of RNA sensors and robust IFN-stimulated genes expression but a repressed type I IFN production. SARS-CoV-2 infection activated T and B cell responses during the early onset but resulted in transient adaptive immunosuppression during severe disease state. Activation of apoptotic pathways and functional exhaustion may contribute to the reduction of lymphocytes and dysfunction of adaptive immunity, whereas increase in IL2, IL7, and IL15 may facilitate the recovery of the number and function of lymphocytes. Our study provides comprehensive transcriptional signatures of peripheral blood response in patients with moderate COVID-19.

[Source](https://doi.org/10.3389/fcimb.2021.632490)

10.4049/jimmunol.2001325

---

## Next-Generation Sequencing Reveals The Progression Of COVID-19

[Source](https://doi.org/10.3389/fcimb.2021.632490)

---

## Metagenomics Approaches to Investigate the Gut Microbiome of COVID-19 Patients.
 02.04.2021, _S Sehli, I Allali, R Chahboune, Y Bakri, N Al Idrissi, S Hamdi, C Nejjari, S Amzazi, H Ghazal_


Over the last decade, it has become increasingly apparent that the microbiome is a central component in human well-being and illness. However, to establish innovative therapeutic methods, it is crucial to learn more about the microbiota. Thereby, the area of metagenomics and associated bioinformatics methods and tools has become considerable in the study of the human microbiome biodiversity. The application of these metagenomics approaches to studying the gut microbiome in COVID-19 patients could be one of the promising areas of research in the fight against the SARS-CoV-2 infection and disparity. Therefore, understanding how the gut microbiome is affected by or could affect the SARS-CoV-2 is very important. Herein, we present an overview of approaches and methods used in the current published studies on COVID-19 patients and the gut microbiome. The accuracy of these researches depends on the appropriate choice and the optimal use of the metagenomics bioinformatics platforms and tools. Interestingly, most studies reported that COVID-19 patients' microbiota are enriched with opportunistic microorganisms. The choice and use of appropriate computational tools and techniques to accurately investigate the gut microbiota is therefore critical in determining the appropriate microbiome profile for diagnosis and the most reliable antiviral or preventive microbial composition.

[Source](https://doi.org/10.1177/1177932221999428)

10.1177/1177932221999428

---

## https://www.frontiersin.org/articles/10.3389/fmicb.2020.01388/full

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.01388/full)

---

## https://www.frontiersin.org/articles/10.3389/fcimb.2020.576551/full

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2020.576551/full)

---

## https://www.sciencedirect.com/science/article/pii/S0306987720333673

[Source](https://www.sciencedirect.com/science/article/pii/S0306987720333673)

---

## https://www.dldjournalonline.com/article/S1590-8658(20)30474-6/fulltext

[Source](https://www.dldjournalonline.com/article/S1590-8658(20)30474-6/fulltext)

---

## https://theconversation.com/a-healthy-microbiome-builds-a-strong-immune-system-that-could-help-defeat-covid-19-145668

[Source](https://theconversation.com/a-healthy-microbiome-builds-a-strong-immune-system-that-could-help-defeat-covid-19-145668)

---

## The COVID-19 Vaccine and the Black Community: Addressing the Justified Questions.
 18.06.2021, _CT Laurencin, H Valantine, C Yancy, CP Jones, C Bright_


Established in 2019, the Roundtable on Black Men and Black Women in STEM convenes a broad array of stakeholders that focus on the barriers and opportunities encountered by Black men and Black women as they navigate the pathways from K-12 and postsecondary education to careers in science, engineering, and medicine. Through meetings, public workshops, and publications, the Roundtable advances discussions that raise awareness and/or highlight promising practices for increasing the representation, retention, and inclusiveness of Black men and Black women in STEM. In keeping with the charge of the Roundtable, Roundtable leadership and leaders of the COVID-19 action group conducted an informational video in January 2021 to provide an in-depth discussion around common, justified questions in the Black community pertaining to the COVID-19 vaccine. The manuscript addresses selected questions and answers relating to the different types of COVID-19 vaccines and their development, administration, and effectiveness. Discussion focuses on addressing vaccine misconceptions, misinformation, mistrust, and hesitancy; challenges in prioritizing vaccinations in diverse populations and communities; dealing with racism in medicine and public health; optimizing communication and health education; and offering practical strategies and recommendations for improving vaccine acceptance by clinicians, health care workers, and the Black community. This manuscript summarizes the content in the YouTube video ( https://www.youtube.com/watch?v=wdEC9c48A_k ).

[Source](https://www.medrxiv.org/content/10.1101/2021.01.05.20249061v1)

10.1007/s40615-021-01082-9

---

## https://www.medrxiv.org/content/10.1101/2021.01.05.20249061v1

[Source](https://www.medrxiv.org/content/10.1101/2021.01.05.20249061v1)

---

## https://www.newsweek.com/gut-microbes-could-predict-how-seriously-ill-covid-19-patient-might-get-preliminary-study-suggests-1500568

[Source](https://www.newsweek.com/gut-microbes-could-predict-how-seriously-ill-covid-19-patient-might-get-preliminary-study-suggests-1500568)

---

## https://theconversation.com/gut-reaction-how-the-gut-microbiome-may-influence-the-severity-of-covid-19-139094

[Source](https://theconversation.com/gut-reaction-how-the-gut-microbiome-may-influence-the-severity-of-covid-19-139094)

---

## https://www.frontiersin.org/articles/10.3389/fmed.2021.652464/full

[Source](https://www.frontiersin.org/articles/10.3389/fmed.2021.652464/full)

---

## <span>Like</span>

[Source](https://microbiomedigest.com/2021/06/21/microbiome-and-covid19-some-research-highlights/?like_comment=27894&_wpnonce=02168e6f44)

---

## https://www.frontiersin.org/articles/10.3389/fmicb.2021.637430/full

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2021.637430/full)

---

## Sulfurized Polyacrylonitrile for High-Performance Lithium-Sulfur Batteries: In-Depth Computational Approach Revealing Multiple Sulfur's Reduction Pathways and Hidden Li+ Storage Mechanisms for Extra Discharge Capacity.
 14.01.2021, _S Perez Beltran, PB Balbuena_


Like no other sulfur host material, polyacrylonitrile-derived sulfurized carbon (SPAN) promises improved electrochemical performance for lithium-sulfur batteries, based on its compatibility with carbonate solvents and ability to prevent self-discharge and shuttle effect. However, a complete understanding of the SPAN's lithiation mechanism is still missing because its structural features vary widely with synthesis conditions, and its electrochemical performance deviates from elemental sulfur. This study continues our research on the elucidation of the SPAN's structural characteristics and lithiation mechanisms via computational approaches. Our models reproduce most experimental data regarding carbon's graphitization level and conjugated ordering, sulfur-carbon covalent bonding, sulfur loading, and elemental composition. Our simulations emulate the discharge voltage observed in experiments for the first discharge, which reveals that sulfur follows multiple reduction pathways based on its interaction with the carbon backbone. Sulfur reduction takes place above 1.0 V vs Li/Li+ mostly in the SPAN-like material, with no long-chain lithium polysulfide formation. Below 1.0 V vs Li/Li+, the backbone's electrochemical activity occurs via multiple C-Li and N-Li interactions, mostly with edge carbon atoms and pyridinic nitrogen. Moreover, we identify Li+ binding sites throughout the graphitized backbone that might lead to prohibited energy costs for Li+ deintercalation, which may explain the irreversible capacity loss between the first and second discharges. This work improves understanding of lithiation mechanisms in sulfurized carbon, which is useful for rationally designing SPAN synthesis pathways tailored to increase sulfur loading and enhanced electrochemical performance.

[Source](https://microbiomedigest.com/2021/06/21/microbiome-and-covid19-some-research-highlights/?like_comment=27895&_wpnonce=29e67beadf)

10.1021/acsami.0c17537

---

## <span>Like</span>

[Source](https://microbiomedigest.com/2021/06/21/microbiome-and-covid19-some-research-highlights/?like_comment=27895&_wpnonce=29e67beadf)

---

