## Microbial Tracking-2, a metagenomics analysis of bacteria and fungi onboard the International Space Station

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01293-0)

---

## Engineering the human gut commensal Bacteroides thetaiotaomicron with synthetic biology.
 27.06.2022, _Y Lai, N Hayashi, TK Lu_


The role of the microbiome in health and disease is attracting the attention of researchers seeking to engineer microorganisms for diagnostic and therapeutic applications. Recent progress in synthetic biology may enable the dissection of host-microbiota interactions. Sophisticated genetic circuits that can sense, compute, memorize, and respond to signals have been developed for the stable commensal bacterium Bacteroides thetaiotaomicron, dominant in the human gut. In this review, we highlight recent advances in expanding the genetic toolkit for B. thetaiotaomicron and foresee several applications of this species for microbiome engineering. We provide our perspective on the challenges and future opportunities for the engineering of human gut-associated bacteria as living therapeutic agents.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1367593122000631)

10.1016/j.cbpa.2022.102178

---

## Factors shaping the abundance and diversity of the gut archaeome across the animal kingdom.
 14.06.2022, _CM Thomas, E Desmond-Le Quéméner, S Gribaldo, G Borrel_


_Animals, Archaea, Bacteria, Dietary Fiber, Euryarchaeota, Mammals, Methanobacteriaceae, Methanobrevibacter, Phylogeny, RNA, Ribosomal, 16S_
Archaea are common constituents of the gut microbiome of humans, ruminants, and termites but little is known about their diversity and abundance in other animals. Here, we analyse sequencing and quantification data of archaeal and bacterial 16S rRNA genes from 250 species of animals covering a large taxonomic spectrum. We detect the presence of archaea in 175 animal species belonging to invertebrates, fish, amphibians, birds, reptiles and mammals. We identify five dominant gut lineages, corresponding to Methanobrevibacter, Methanosphaera, Methanocorpusculum, Methanimicrococcus and "Ca. Methanomethylophilaceae". Some archaeal clades, notably within Methanobrevibacter, are associated to certain hosts, suggesting specific adaptations. The non-methanogenic lineage Nitrososphaeraceae (Thaumarchaeota) is frequently present in animal samples, although at low abundance, but may have also adapted to the gut environment. Host phylogeny, diet type, fibre content, and intestinal tract physiology are major drivers of the diversity and abundance of the archaeome in mammals. The overall abundance of archaea is more influenced by these factors than that of bacteria. Methanogens reducing methyl-compounds with H2 can represent an important fraction of the overall methanogens in many animals. Together with CO2-reducing methanogens, they are influenced by diet and composition of gut bacteria. Our results provide key elements toward our understanding of the ecology of archaea in the gut, an emerging and important field of investigation.

[Source](https://www.nature.com/articles/s41467-022-31038-4)

10.1038/s41467-022-31038-4

---

## Increase in Bifidobacterium is a characteristic of the difference in the salivary microbiota of pregnant and non-pregnant women

[Source](https://bmcoralhealth.biomedcentral.com/articles/10.1186/s12903-022-02293-0)

---

## Concordance of Urinary Microbiota Detected by 16S rRNA Amplicon Sequencing Versus Expanded Quantitative Urine Culture.
 28.06.2022, _MH Vaughan, GE Zemtsov, EM Dahl, L Karstens, L Ma, NY Siddiqui_



[Source](https://www.sciencedirect.com/science/article/abs/pii/S0002937822004859)

10.1016/j.ajog.2022.06.031

---

## Dietary variety relates to gut microbiota diversity and abundance in humans

[Source](https://link.springer.com/article/10.1007/s00394-022-02929-5)

---

## The gut microbiota composition in patients with right- and left-sided colorectal cancer and after curative colectomy, as analyzed by 16S rRNA gene amplicon sequencing.
 28.06.2022, _D Suga, H Mizutani, S Fukui, M Kobayashi, Y Shimada, Y Nakazawa, Y Nishiura, Y Kawasaki, I Moritani, Y Yamanaka, H Inoue, E Ojima, Y Mohri, H Nakagawa, K Dohi, K Takaba, H Wada, K Shiraki_


_Carcinoma, Renal Cell, Colectomy, Colonic Neoplasms, Colorectal Neoplasms, Gastrointestinal Microbiome, Genes, rRNA, Humans, Kidney Neoplasms, RNA, Ribosomal, 16S_
Gut pathological microbial imbalance or dysbiosis is closely associated with colorectal cancer. Although there are observable differences in molecular and clinical characteristics between patients with right- and left-sided colon cancer, differences in their gut microbiomes have not been thoroughly investigated. Furthermore, subsequent changes in microbiota status after partial colectomy remain unknown. We examined the human gut microbiota composition to determine its relationship with colon cancer and partial colon resection according to location.

[Source](https://bmcgastroenterol.biomedcentral.com/articles/10.1186/s12876-022-02382-y)

10.1186/s12876-022-02382-y

---

## Long-Term Dietary Effects on Human Gut Microbiota Composition Employing Shotgun Metagenomics Data Analysis.
 27.06.2022, _A Troci, P Rausch, S Waschina, W Lieb, A Franke, C Bang_


The gut microbiome regulates various metabolic pathways in the host and its dysbiosis is involved in the pathogenesis of diverse diseases. Multiple factors modulate the composition of the microbiome and how it physiologically functions, but one of the major factors triggering gut microbiome establishment is diet. We aim to unravel interactions and changes between diet and gut microbiome over a period of 3 years.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/mnfr.202101098)

10.1002/mnfr.202101098

---

## Functional changes of the gastric bypass microbiota reactivate thermogenic adipose tissue and systemic glucose control via intestinal FXR-TGR5 crosstalk in diet-induced obesity.
 27.06.2022, _J Münzker, N Haase, A Till, R Sucher, SB Haange, L Nemetschke, T Gnad, E Jäger, J Chen, SJ Riede, R Chakaroun, L Massier, P Kovacs, M Ost, U Rolle-Kampczyk, N Jehmlich, J Weiner, JT Heiker, N Klöting, G Seeger, M Morawski, V Keitel, A Pfeifer, M von Bergen, J Heeren, U Krügel, WK Fenske_


_Adipose Tissue, Animals, Bile Acids and Salts, Blood Glucose, Diabetes Mellitus, Type 2, Diet, Gastric Bypass, Microbiota, Obesity, Rats, Receptors, G-Protein-Coupled, Taurine, Thermogenesis_
Bariatric surgery remains the most effective therapy for adiposity reduction and remission of type 2 diabetes. Although different bariatric procedures associate with pronounced shifts in the gut microbiota, their functional role in the regulation of energetic and metabolic benefits achieved with the surgery are not clear.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01264-5)

10.1186/s40168-022-01264-5

---

## Perinatal transmission of a probiotic Bifidobacterium strain protects against early life stress-induced mood and gastrointestinal motility disorders.
 28.06.2022, _H Zhu, P Tian, X Qian, L Gu, J Zhao, G Wang, W Chen_


Early life stress can considerably interfere in gut microbiome formation and nervous system development. Specific probiotic strains have been proved to exert anti-stress effects by modulating the gut-brain axis. However, little is known about whether probiotic treatment during pregnancy can protect the offspring from early life stress. In this study, Bifidobacterium breve CCFM1025, previously proven to exert microbial and neurobiological regulation effects, was given to pregnant mice. The offspring's gut and brain functions were evaluated when challenged with maternal separation. Intriguingly, treatment with probiotics during pregnancy protected the offspring from maternal separation-induced neurobiological and gastrointestinal disorders such as depression-like behaviour and delayed defecation. Quantification of CCFM1025 was performed, and perinatal transmission of CCFM1025 was further validated, which also explained the reason for increased levels of colonic 5-hydroxytryptamine and caecal short-chain fatty acids in the offspring. Our findings indicated that the effects of probiotics can be perinatally transmitted through gut microbes and that probiotic treatment during pregnancy may have great potential in managing health risks in early life.

[Source](https://pubs.rsc.org/en/content/articlelanding/2022/FO/D2FO01164F)

10.1039/d2fo01164f

---

## Leveraging Existing 16SrRNA Microbial Data to Define a Composite Biomarker for Autism Spectrum Disorder.
 28.06.2022, _Y Xu, Y Wang, J Xu, Y Song, B Liu, Z Xiong_


Cumulative studies have utilized high-throughput sequencing of the 16SrRNA gene to characterize the composition and structure of the microbiota in autism spectrum disorder (ASD). However, they do not always obtain consistent results; thus, conducting cross-study comparisons is necessary. This study sought to analyze the alteration of fecal microbiota and the diagnostic capabilities of gut microbiota biomarkers in individuals with ASD using the existing 16SrRNA microbial data and explore heterogeneity among studies. The raw sequence and metadata from 10 studies, including 1,019 samples, were reanalyzed. Results showed no significant difference in alpha diversity of fecal microbiota between ASD and the control group. However, a significant difference in the composition structure of fecal microbiota was observed. Given the large differences in sample selection and technical differences, the separation of fecal microbiota between ASD and controls was not observed. Subgroup analysis was performed on the basis of different country of origin, hypervariable regions, and sequencing platforms, and the dominant genera in ASD and healthy control groups were determined by linear discriminant analysis (LDA) of the effect size (LEfSe) algorithm and Wilcoxon rank-sum test. Machine learning analyses were carried out to determine the diagnostic capabilities of potential microbial biomarkers. A total of 12 genera were identified to distinguish ASD from control, and the AUC of the training set and verification set was 0.757 and 0.761, respectively. Despite cohort heterogeneity, gut microbial dysbiosis of ASD has been proven to be a widespread phenomenon. Therefore, fecal microbial markers are of great significance in diagnosing ASD diseases and possible candidates for further mechanistic study of the role of intestinal microbiota in ASD. IMPORTANCE This study provides an updated analysis to characterize the gut microbiota in ASD using 16SrRNA gene high-throughput sequencing data from 10 publicly available studies. Our analysis suggests an association between the fecal microbiota and ASD. Sample selection and technical differences between studies may interfere with the species composition analysis of the ASD group and control group. By summarizing the results of 16SrRNA gene sequencing from multiple fecal samples, we can provide evidence to support the use of microbial biomarkers to diagnose the occurrence of ASD. Our study provides a new perspective for further revealing the correlation between gut microbiota and ASD from the perspective of 16SrRNA sequencing in larger samples.

[Source](https://journals.asm.org/doi/10.1128/spectrum.00331-22)

10.1128/spectrum.00331-22

---

## Gut microbiome of century-old snail specimens stable across time in preservation

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01286-z)

---

## Soil-derived bacteria endow Camellia weevil with more ability to resist plant chemical defense.
 28.06.2022, _S Zhang, Z Li, J Shu, H Xue, K Guo, X Zhou_


_Animals, Bacteria, Camellia, Insecta, Saponins, Soil, Weevils_
Herbivorous insects acquire their gut microbiota from diverse sources, and these microorganisms play significant roles in insect hosts' tolerance to plant secondary defensive compounds. Camellia weevil (Curculio chinensis) (CW) is an obligate seed parasite of Camellia oleifera plants. Our previous study linked the CW's gut microbiome to the tolerance of the tea saponin (TS) in C. oleifera seeds. However, the source of these gut microbiomes, the key bacteria involved in TS tolerance, and the degradation functions of these bacteria remain unresolved.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01290-3)

10.1186/s40168-022-01290-3

---

## A genome and gene catalog of glacier microbiomes.
 27.06.2022, _Y Liu, M Ji, T Yu, J Zaugg, AM Anesio, Z Zhang, S Hu, P Hugenholtz, K Liu, P Liu, Y Chen, Y Luo, T Yao_


Glaciers represent a unique inventory of microbial genetic diversity and a record of evolution. The Tibetan Plateau contains the largest area of low-latitude glaciers and is particularly vulnerable to global warming. By sequencing 85 metagenomes and 883 cultured isolates from 21 Tibetan glaciers covering snow, ice and cryoconite habitats, we present a specialized glacier microbial genome and gene catalog to archive glacial genomic and functional diversity. This comprehensive Tibetan Glacier Genome and Gene (TG2G) catalog includes 883 genomes and 2,358 metagenome-assembled genomes, which represent 968 candidate species spanning 30 phyla. The catalog also contains over 25 million non-redundant protein-encoding genes, the utility of which is demonstrated by the exploration of secondary metabolite biosynthetic potentials, virulence factor identification and global glacier metagenome comparison. The TG2G catalog is a valuable resource that enables enhanced understanding of the structure and functions of Tibetan glacial microbiomes.

[Source](https://www.nature.com/articles/s41587-022-01367-2)

10.1038/s41587-022-01367-2

---

## Large Scale Genome-Centric Metagenomic Data from the Gut Microbiome of Food-Producing Animals and Humans.
 28.06.2022, _LN Lemos, FM de Carvalho, FF Santos, TB Valiatti, DC Corsi, AC de Oliveira Silveira, A Gerber, APC Guimarães, C de Oliveira Souza, DM Brasiliense, DSC Maia Castelo-Branco, EK Anzai, FO Bessa-Neto, GM de Melo, GH de Souza, LFC Ferraz, M de Nazaré Miranda Bahia, MS Mattos, RGB da Silva, R Veiga, S Simionatto, WAP Monteiro, WA de Oliveira Lima, CRV Kiffer, R Cayô, AC Gales, ATR de Vasconcelos_


_Animals, Archaea, Cattle, Gastrointestinal Microbiome, Humans, Metagenome, Metagenomics, Pilot Projects, Swine_
The One Health concept is a global strategy to study the relationship between human and animal health and the transfer of pathogenic and non-pathogenic species between these systems. However, to the best of our knowledge, no data based on One Health genome-centric metagenomics are available in public repositories. Here, we present a dataset based on a pilot-study of 2,915 metagenome-assembled genomes (MAGs) of 107 samples from the human (N = 34), cattle (N = 28), swine (N = 15) and poultry (N = 30) gut microbiomes. Samples were collected from the five Brazilian geographical regions. Of the draft genomes, 1,273 were high-quality drafts (≥90% of completeness and ≤5% of contamination), and 1,642 were medium-quality drafts (≥50% of completeness and ≤10% of contamination). Taxonomic predictions were based on the alignment and concatenation of single-marker genes, and the most representative phyla were Bacteroidota, Firmicutes, and Proteobacteria. Many of these species represent potential pathogens that have already been described or potential new families, genera, and species with potential biotechnological applications. Analyses of this dataset will highlight discoveries about the ecology and functional role of pathogens and uncultivated Archaea and Bacteria from food-producing animals and humans. Furthermore, it also represents an opportunity to describe new species from underrepresented taxonomic groups.

[Source](https://www.nature.com/articles/s41597-022-01465-5)

10.1038/s41597-022-01465-5

---

## Research Note: Choice of microbiota database affects data analysis and interpretation in chicken cecal microbiota.
 27.06.2022, _PM Campos, N Darwish, J Shao, M Proszkowiec-Weglarz_


The chicken microbiota is often analyzed to address questions about the effects of diet or disease on poultry health. To analyze the microbiota, bioinformatic platforms such as QIIME 2 and mothur are used, which incorporate public taxonomic databases such as Greengenes, the ribosomal database project (RDP), and SILVA to assign taxonomies to bacterial sequences. Many chicken microbiota studies continue to incorporate the Greengenes database, which has not been updated since 2013. To determine whether a choice of database could affect results, this study compared the results of bioinformatic analyses obtained using the Greengenes, RDP, and SILVA databases on a cecal luminal microbiome dataset. The QIIME 2 platform was used to process 16S bacterial sequences and assign taxonomies with Greengenes, RDP, and SILVA. Linear discriminant analysis effect size (LEfSe) was performed, allowing for the comparison of taxonomies considered significantly differentially abundant between the three databases. Some notable differences between databases were observed in results, in particular the ability of SILVA database to classify members of the family Lachnospiraceae into separate genera, while these members remained in one group of unclassified Lachnospiraceae through Greengenes and RDP. LEfSe analyses showed that the SILVA database produced more differentially abundant genera, in large part due to the classification of these separate Lachnospiraceae genera. Additionally, the relative abundance of unclassified Lachnospiraceae in SILVA results was significantly lower than in RDP results. Our results show the choice of taxonomic database can influence the results of a microbiota study at the genus level, potentially affecting the interpretation of the results. The use of the SILVA database is recommended over Greengenes in chicken microbiota studies, as more specific classifications at the genus level may provide more accurate interpretations of changes in the microbiota.

[Source](https://www.sciencedirect.com/science/article/pii/S0032579122002632)

10.1016/j.psj.2022.101971

---

## Predicting metabolomic profiles from microbial composition through neural ordinary differential equations

[Source](https://www.biorxiv.org/content/10.1101/2022.06.23.497381v1.abstract)

---

