## Intracellular <em>Staphylococcus aureus</em> persisters upon antibiotic exposure

[Source](https://doi.org/10.1038/s41467-020-15966-7)

---

## Investigation of the human nasal microbiome in persons with long- and short-term exposure to methicillin-resistant <em>Staphylococcus aureus</em> and other bacteria from the pig farm environment

[Source](https://doi.org/10.1371/journal.pone.0232456)

---

## Microbial predictors of healing and short-term effect of debridement on the microbiome of chronic wounds.
 07.05.2020, _S Verbanic, Y Shen, J Lee, JM Deacon, IA Chen_


Chronic wounds represent a large and growing disease burden. Infection and biofilm formation are two of the leading impediments of wound healing, suggesting an important role for the microbiome of these wounds. Debridement is a common and effective treatment for chronic wounds. We analyzed the bacterial content of the wound surface from 20 outpatients with chronic wounds before and immediately after debridement, as well as healthy skin. Given the large variation observed among different wounds, we introduce a Bayesian statistical method that models patient-to-patient variability and identify several genera that were significantly enriched in wounds vs. healthy skin. We found no difference between the microbiome of the original wound surface and that exposed by a single episode of sharp debridement, suggesting that this debridement did not directly alter the wound microbiome. However, we found that aerobes and especially facultative anaerobes were significantly associated with wounds that did not heal within 6 months. The facultative anaerobic genus Enterobacter was significantly associated with lack of healing. The results suggest that an abundance of facultative anaerobes is a negative prognostic factor in the chronic wound microbiome, possibly due to the increased robustness of such communities to different metabolic environments.

[Source](https://doi.org/10.1038/s41522-020-0130-5)

10.1038/s41522-020-0130-5

---

## Systemic short chain fatty acids limit antitumor effect of CTLA-4 blockade in hosts with cancer.
 06.05.2020, _C Coutzac, JM Jouniaux, A Paci, J Schmidt, D Mallardo, A Seck, V Asvatourian, L Cassard, P Saulnier, L Lacroix, PL Woerther, A Vozy, M Naigeon, L Nebot-Bral, M Desbois, E Simeone, C Mateus, L Boselli, J Grivel, E Soularue, P Lepage, F Carbonnel, PA Ascierto, C Robert, N Chaput_


Gut microbiota composition influences the clinical benefit of immune checkpoints in patients with advanced cancer but mechanisms underlying this relationship remain unclear. Molecular mechanism whereby gut microbiota influences immune responses is mainly assigned to gut microbial metabolites. Short-chain fatty acids (SCFA) are produced in large amounts in the colon through bacterial fermentation of dietary fiber. We evaluate in mice and in patients treated with anti-CTLA-4 blocking mAbs whether SCFA levels is related to clinical outcome. High blood butyrate and propionate levels are associated with resistance to CTLA-4 blockade and higher proportion of Treg cells. In mice, butyrate restrains anti-CTLA-4-induced up-regulation of CD80/CD86 on dendritic cells and ICOS on T cells, accumulation of tumor-specific T cells and memory T cells. In patients, high blood butyrate levels moderate ipilimumab-induced accumulation of memory and ICOS + CD4 + T cells and IL-2 impregnation. Altogether, these results suggest that SCFA limits anti-CTLA-4 activity.

[Source](https://doi.org/10.1038/s41467-020-16079-x)

10.1038/s41467-020-16079-x

---

## Functional and phylogenetic alterations in gut microbiome are linked to graft-versus-host disease severity.
 30.04.2020, _M Payen, I Nicolis, M Robin, D Michonneau, J Delannoye, C Mayeur, N Kapel, B Berçot, MJ Butel, J Le Goff, G Socié, C Rousseau_


Acute graft-versus-host disease (aGVHD) is the main complication of hematopoietic stem cell transplantation (HSCT). Changes in gut microbiota composition have been associated with subsequent aGVHD, and reconstitution of healthy microbiota is currently being explored as a therapeutic approach. However, the specific actors in the intestinal ecosystem involved in the pathologic process at the time of aGVHD onset are not yet fully known. We prospectively collected stool samples from patients who underwent allogeneic HSCT. Patients sampled at aGVHD onset were compared with non-GVHD patients. To identify phylogenetic and functional signatures of the disease process, we determined fecal short-chain fatty acid (SFCA) profiles and used high-throughput DNA sequencing and real-time quantitative polymerase chain reaction to assess the microbiota composition. Microbiota alterations were highly specific of gastrointestinal (GI) aGVHD severity. Bacterial biomass and α-diversity were lower in severe aGVHD. We identified several bacterial signatures associated with severe aGVHD at disease onset; a negative correlation was observed with anaerobic bacteria of the Lachnospiraceae, especially the Blautia genus, and Ruminococcaceae families. In parallel, in severe aGVHD patients, we showed a dramatic decrease in the levels of the main SFCAs: acetate (75.8%), propionate (95.8%), and butyrate (94.6%). Mild aGVHD patients were characterized by conserved levels of propionate and Blautia propionate producers. Butyrate was significantly decreased in all GI aGVHD stages, representing a potential diagnostic marker of the disease. Specific microbiota and metabolic alterations were thus associated with aGVHD severity and may be useful for diagnostic and pathophysiologic purposes.

[Source](https://doi.org/10.1182/bloodadvances.2020001531)

10.1182/bloodadvances.2020001531

---

## Metagenomic alterations in gut microbiota precede and predict onset of colitis in the IL10 gene-deficient murine model

[Source](https://doi.org/10.1101/2020.05.04.078022)

---

## Dietary Fiber Protects against Diabetic Nephropathy through Short-Chain Fatty Acid-Mediated Activation of G Protein-Coupled Receptors GPR43 and GPR109A.
 02.05.2020, _YJ Li, X Chen, TK Kwan, YW Loh, J Singer, Y Liu, J Ma, J Tan, L Macia, CR Mackay, SJ Chadban, H Wu_


Studies have reported "dysbiotic" changes to gut microbiota, such as depletion of gut bacteria that produce short-chain fatty acids (SCFAs) through gut fermentation of fiber, in CKD and diabetes. Dietary fiber is associated with decreased inflammation and mortality in CKD, and SCFAs have been proposed to mediate this effect.

[Source](https://doi.org/10.1681/ASN.2019101029)

10.1681/ASN.2019101029

---

## Probiotic strain Lactobacillus plantarum YYC-3 prevents colon cancer in mice by regulating the tumour microenvironment.
 07.05.2020, _Y Yue, K Ye, J Lu, X Wang, S Zhang, L Liu, B Yang, K Nassar, X Xu, X Pang, J Lv_


The gut microbiota plays important roles in chronic inflammation and colon cancer. Lactobacillus is a gut-resident probiotic with benefits to host health. We recently identified Lactobacillus plantarum strain YYC-3 with strong inhibition against two colon cancer cell lines (HT-29 and Caco2). However, the inhibitory effect of YYC-3 against colon cancer in vivo has not been verified. Thus, in the present study, we explored the probiotic function of strain YYC-3 and its cell-free supernatant (YYCS) respectively in the APCMin/+ mouse model of colon cancer during tumour development and growth, and the underlying anti-cancer mechanism. Treatment of both strain YYC-3 and the YYCS prevented the occurrence of colon tumours and mucosal damage in APCMin/+ mice fed a high-fat diet, although YYC-3 had a stronger anti-cancer effect. The mechanism involved modulation of the immune system and downregulated expression of the inflammatory cytokines interleukin (IL)-6, IL-17 F, and IL-22, along with reduced infiltration of inflammatory cells. Moreover, YYC-3 suppressed activation of the NF-κB and Wnt signalling pathways, and restored the altered gut microbiota composition to closely match that of wild-type mice. These results lay a theoretical foundation for application of YYC-3 in colon cancer prevention.

[Source](https://doi.org/10.1016/j.biopha.2020.110159)

10.1016/j.biopha.2020.110159

---

## Effect of stevia on the gut microbiota and glucose tolerance in a murine model of diet-induced obesity

[Source](https://doi.org/10.1093/femsec/fiaa079)

---

## Sea cucumber intestinal regeneration reveals deterministic assembly of the gut microbiome.
 02.05.2020, _BL Weigel_


The gut microbiome has far-reaching effects on host organism health, thus understanding the processes that underlie microbial community assembly in the developing gut is a current research priority. Here, a holothurian (a.k.a., sea cucumber; phylum Echinodermata) host is explored as promising model system for studying the assembly of the gut microbiome. Holothurians have a unique capacity for evisceration (expulsion of the internal organs) followed by rapid regeneration of the gut, decoupling host ontogeny from gut tissue development and permitting experimental manipulation of the gut microbiome in mature host individuals. Here, evisceration was induced in the sea cucumber Sclerodactyla briareus, and regenerating stomach and intestine microbiomes were characterized before and on days 0, 13, 17 and 20 post-evisceration using Illumina sequencing of 16S rRNA genes. Regenerating stomach and intestine microbial communities had significantly different community structure and composition compared to mature microbial communities, with much higher alpha diversity and evenness of taxa in regenerating tissues. Despite immersion in a diverse pool of sediment and seawater microbes in flow-through seawater aquaria, regenerating gut microbiomes differed at each stage of regeneration and displayed highly similar community structure among replicates, providing evidence for deterministic host selection of a specific microbial consortium. Moreover, regenerating gut tissues acquired a microbiome that likely conferred energetic and immune advantages to the sea cucumber host, including microbes that can fix carbon and degrade invading pathogens.Importance The gut microbiome is pertinent to many aspects of animal health, and there is a great need for natural but tractable experimental systems to examine the processes shaping gut microbiome assembly. Here, the holothurian (sea cucumber) Sclerodactyla briareus is explored as an experimental system to study microbial colonization in the gut, as S. briareus has the ability to completely eviscerate and rapidly regenerate their digestive organs. After induced evisceration, microbial community assembly was characterized over 20 days in regenerating animals. This study demonstrated that colonization of the sea cucumber gut was deterministic; despite immersion in a diverse consortium of environmental microbes, a specific subset of microbes proliferated in the gut, including taxa that likely conferred energetic and immune advantages to the host. Sea cucumbers have the potential to revolutionize our understanding of gut microbiome assembly, as rapid and repeatable gut tissue regeneration provides a promising and tractable experimental system.

[Source](https://doi.org/10.1128/AEM.00489-20)

10.1128/AEM.00489-20

---

## Mating changes the genital microbiome in both sexes of the common bedbug Cimex lectularius across populations.
 29.04.2020, _S Bellinvia, PR Johnston, S Mbedi, O Otti_


Many bacteria live on host surfaces, in cells and in specific organ systems. In comparison with gut microbiomes, the bacterial communities of reproductive organs (genital microbiomes) have received little attention. During mating, male and female genitalia interact and copulatory wounds occur, providing an entrance for sexually transmitted microbes. Besides being potentially harmful to the host, invading microbes might interact with resident genital microbes and affect immunity. Apart from the investigation of sexually transmitted symbionts, few studies have addressed how mating changes genital microbiomes. We dissected reproductive organs from virgin and mated common bedbugs, Cimex lectularius L., and sequenced their microbiomes to investigate composition and mating-induced changes. We show that mating changes the genital microbiomes, suggesting bacteria are sexually transmitted. Also, genital microbiomes varied between populations and the sexes. This provides evidence for local and sex-specific adaptation of bacteria and hosts, suggesting bacteria might play an important role in shaping the evolution of reproductive traits. Coadaptation of genital microbiomes and reproductive traits might further lead to reproductive isolation between populations, giving reproductive ecology an important role in speciation. Future studies should investigate the transmission dynamics between the sexes and populations to uncover potential reproductive barriers.

[Source](https://doi.org/10.1098/rspb.2020.0302)

10.1098/rspb.2020.0302

---

## Mutualist and pathogen traits interact to affect plant community structure in a spatially explicit model

[Source](https://doi.org/10.1038/s41467-020-16047-5)

---

## Seasonal dynamics of methane cycling microbial communities in Amazonian floodplain sediments

[Source](https://doi.org/10.1101/2020.05.04.076356)

---

## Community succession of the grapevine fungal microbiome in the annual growth cycle

[Source](https://doi.org/10.1101/2020.05.03.075457)

---

## Characterizing changes in soil microbiome abundance and diversity due to different cover crop techniques.
 05.05.2020, _CH Wang, L Wu, Z Wang, MS Alabady, D Parson, Z Molumo, SC Fankhauser_


Soil-based microorganisms assume a direct and crucial role in the promotion of soil health, quality and fertility, all factors known to contribute heavily to the quality and yield of agricultural products. Cover cropping, used in both traditional and organic farming, is a particularly efficient and environmentally favorable tool for manipulating microbiome composition in agricultural soils and has had clear benefits for soil quality and crop output. Several long-term investigations have evaluated the influence of multi-mix (multiple species) cover crop treatments on soil health and microbial diversity. The present study investigated the short-term effects of a seven species multi-mix cover crop treatment on soil nutrient content and microbial diversity, compared to a single-mix cover crop treatment and control. Analysis of 16S sequencing data of isolated soil DNA revealed that the single-mix cover crop treatment decreased overall microbial abundance and diversity, whereas the control and multi-mix treatments altered the overall microbial composition in similar fluctuating trends. Furthermore, we observed significant changes in specific bacteria belonging to the phyla Acidobacteria, Actinobacteria, Planctomycetes, Proteobacteria and Verrucombicrobia for all treatments, but only the single-mix significantly decreased in abundance of the selected bacteria over time. Our findings indicate that the control and multi-mix treatments are better at maintaining overall microbial composition and diversity compared to the single-mix. Further study is required to elucidate the specific difference between the treatment effect of the multi-mix treatment and the control, given that their microbial composition changes over time were similar but they diverge into two populations of unique bacterial types by the end of this short-term study.

[Source](https://doi.org/10.1371/journal.pone.0232453)

10.1371/journal.pone.0232453

---

## Specific and conserved patterns of microbiota-structuring by maize benzoxazinoids in the field

[Source](https://doi.org/10.1101/2020.05.03.075135)

---

## The role of changing temperature in microbial metabolic processes during permafrost thaw.
 30.04.2020, _KS Messan, RM Jones, SJ Doherty, K Foley, TA Douglas, RA Barbato_


Approximately one fourth of the Earth's Northern Hemisphere is underlain by permafrost, earth materials (soil, organic matter, or bedrock), that has been continuously frozen for at least two consecutive years. Numerous studies point to evidence of accelerated climate warming in the Arctic and sub-Arctic where permafrost is located. Changes to permafrost biochemical processes may critically impact ecosystem processes at the landscape scale. Here, we sought to understand how the permafrost metabolome responds to thaw and how this response differs based on location (i.e. chronosequence of permafrost formation constituting diverse permafrost types). We analyzed metabolites from microbial cells originating from Alaskan permafrost. Overall, permafrost thaw induced a shift in microbial metabolic processes. Of note were the dissimilarities in biochemical structure between frozen and thawed samples. The thawed permafrost metabolomes from different locations were highly similar. In the intact permafrost, several metabolites with antagonist properties were identified, illustrating the competitive survival strategy required to survive a frozen state. Interestingly, the intensity of these antagonistic metabolites decreased with warmer temperature, indicating a shift in ecological strategies in thawed permafrost. These findings illustrate the impact of change in temperature and spatial variability as permafrost undergoes thaw, knowledge that will become crucial for predicting permafrost biogeochemical dynamics as the Arctic and Antarctic landscapes continue to warm.

[Source](https://doi.org/10.1371/journal.pone.0232169)

10.1371/journal.pone.0232169

---

## Microbial biofilm community structure and composition on the lithic substrates of Herculaneum Suburban Baths.
 04.05.2020, _A De Natale, BH Mele, P Cennamo, A Del Mondo, M Petraretti, A Pollio_


In this work, we want to investigate the impact of different substrates and different environmental condition on the biofilm communities growing on plaster, marble, and mortar substrates inside the Herculaneum Suburban Baths. To do so, we measured environmental conditions and sampled biofilm communities along the walls of the baths and used culture-dependent and -independent molecular techniques (DGGE) to identify the species at each sampling sites. We used the species pool to infer structure and richness of communities within each site in each substrate, and confocal light scanning microscopy to assess the three-dimensional structure of the sampled biofilms. To gather further insights, we built a meta-community network and used its local realizations to analyze co-occurrence patterns of species. We found that light is a limiting factor in the baths environment, that moving along sites equals moving along an irradiation gradient, and that such gradient shapes the community structure, de facto separating a dark community, rich in Bacteria, Fungi and cyanobacteria, from two dim communities, rich in Chlorophyta. Almost all sites are dominated by photoautotrophs, with Fungi and Bacteria relegated to the role of rare species., and structural properties of biofilms are not consistent within the same substrate. We conclude that the Herculaneum suburban baths are an environment-shaped community, where one dark community (plaster) and one dim community (mortar) provides species to a "midway" community (marble).

[Source](https://doi.org/10.1371/journal.pone.0232512)

10.1371/journal.pone.0232512

---

## Comparative effectiveness of probiotic-based formulations on cecal microbiota modulation in broilers.
 05.05.2020, _DR Rodrigues, W Briggs, A Duff, K Chasser, R Murugesan, C Pender, S Ramirez, L Valenzuela, LR Bielke_


The potential of probiotics to manipulate the intestinal microbial ecosystem toward commensal bacteria growth offers great opportunity for enhancing health and performance in poultry. This study aimed to evaluate the efficacy of five probiotic-based formulations in modulating cecal microbiota in broilers at 21 and 42 days of age. Probiotics investigated included a synbiotic (SYNBIO), a yeast (YEAST), and three single-strain formulations of Bacillus amyloliquefaciens (SINGLE1), B. subtilis (SINGLE2) and B. licheniformis (SINGLE3). Alpha-diversity analyses showed that cecal microbiota of SINGLE1, SINGLE2, and YEAST had low diversity compared to the control diet with no feed additive (CON) at 21d. At the same age, weighted Unifrac distance measure showed significant differences between samples from SYNBIO and CON (P = 0.02). However, by analyzing principal coordinates analysis (PCoA) with unweighted Unifrac, there was no evidence of clustering between CON and probiotic treatments. By 42d, there were no differences in alpha or beta-diversity in the microbiota of probiotic treatments compared to CON. Similarly, taxonomic microbial profiling did not show major changes in cecal microbial taxa. In conclusion, not all probiotic-based formulations tested had a core benefit on the modulation of microbiota. However, based on the quantitative beta diversity results, SYNBIO greatly influenced the cecal microbial community structure attributable to transient variations in relative taxon abundance.

[Source](https://doi.org/10.1371/journal.pone.0225871)

10.1371/journal.pone.0225871

---

## Weirdo19ES is a novel singleton mycobacteriophage that selects for glycolipid deficient phage-resistant <em>M. smegmatis</em> mutants

[Source](https://doi.org/10.1371/journal.pone.0231881)

---

## Decoding the language of microbiomes using word-embedding techniques, and applications in inflammatory bowel disease.
 04.05.2020, _CA Tataru, MM David_


Microbiomes are complex ecological systems that play crucial roles in understanding natural phenomena from human disease to climate change. Especially in human gut microbiome studies, where collecting clinical samples can be arduous, the number of taxa considered in any one study often exceeds the number of samples ten to one hundred-fold. This discrepancy decreases the power of studies to identify meaningful differences between samples, increases the likelihood of false positive results, and subsequently limits reproducibility. Despite the vast collections of microbiome data already available, biome-specific patterns of microbial structure are not currently leveraged to inform studies. Here, we derive microbiome-level properties by applying an embedding algorithm to quantify taxon co-occurrence patterns in over 18,000 samples from the American Gut Project (AGP) microbiome crowdsourcing effort. We then compare the predictive power of models trained using properties, normalized taxonomic count data, and another commonly used dimensionality reduction method, Principal Component Analysis in categorizing samples from individuals with inflammatory bowel disease (IBD) and healthy controls. We show that predictive models trained using property data are the most accurate, robust, and generalizable, and that property-based models can be trained on one dataset and deployed on another with positive results. Furthermore, we find that properties correlate significantly with known metabolic pathways. Using these properties, we are able to extract known and new bacterial metabolic pathways associated with inflammatory bowel disease across two completely independent studies. By providing a set of pre-trained embeddings, we allow any V4 16S amplicon study to apply the publicly informed properties to increase the statistical power, reproducibility, and generalizability of analysis.

[Source](https://doi.org/10.1371/journal.pcbi.1007859)

10.1371/journal.pcbi.1007859

---

## Sequence count data are poorly fit by the negative binomial distribution.
 30.04.2020, _S Hawinkel, JCW Rayner, L Bijnens, O Thas_


Sequence count data are commonly modelled using the negative binomial (NB) distribution. Several empirical studies, however, have demonstrated that methods based on the NB-assumption do not always succeed in controlling the false discovery rate (FDR) at its nominal level. In this paper, we propose a dedicated statistical goodness of fit test for the NB distribution in regression models and demonstrate that the NB-assumption is violated in many publicly available RNA-Seq and 16S rRNA microbiome datasets. The zero-inflated NB distribution was not found to give a substantially better fit. We also show that the NB-based tests perform worse on the features for which the NB-assumption was violated than on the features for which no significant deviation was detected. This gives an explanation for the poor behaviour of NB-based tests in many published evaluation studies. We conclude that nonparametric tests should be preferred over parametric methods.

[Source](https://doi.org/10.1371/journal.pone.0224909)

10.1371/journal.pone.0224909

---

## sPepFinder expedites genome-wide identification of small proteins in bacteria

[Source](https://doi.org/10.1101/2020.05.05.079178)

---

## Logicome Profiler: Exhaustive detection of statistically significant logic relationships from comparative omics data.
 04.05.2020, _T Fukunaga, W Iwasaki_


Logic relationship analysis is a data mining method that comprehensively detects item triplets that satisfy logic relationships from a binary matrix dataset, such as an ortholog table in comparative genomics. Thanks to recent technological advancements, many binary matrix datasets are now being produced in genomics, transcriptomics, epigenomics, metagenomics, and many other fields for comparative purposes. However, regardless of presumed interpretability and importance of logic relationships, existing data mining methods are not based on the framework of statistical hypothesis testing. That means, the type-1 and type-2 error rates are neither controlled nor estimated. Here, we developed Logicome Profiler, which exhaustively detects statistically significant triplet logic relationships from a binary matrix dataset (Logicome means ome of logics). To test all item triplets in a dataset while avoiding false positives, Logicome Profiler adjusts a significance level by the Bonferroni or Benjamini-Yekutieli method for the multiple testing correction. Its application to an ocean metagenomic dataset showed that Logicome Profiler can effectively detect statistically significant triplet logic relationships among environmental microbes and genes, which include those among urea transporter, urease, and photosynthesis-related genes. Beyond omics data analysis, Logicome Profiler is applicable to various binary matrix datasets in general for finding significant triplet logic relationships. The source code is available at https://github.com/fukunagatsu/LogicomeProfiler.

[Source](https://doi.org/10.1371/journal.pone.0232106)

10.1371/journal.pone.0232106

---

