## Custom scoring based on ecological topology of gut microbiota associated with cancer immunotherapy outcome.
 21.06.2024, _L Derosa, V Iebba, CAC Silva, G Piccinno, G Wu, L Lordello, B Routy, N Zhao, C Thelemaque, R Birebent, F Marmorino, M Fidelle, M Messaoudene, AM Thomas, G Zalcman, S Friard, J Mazieres, C Audigier-Valette, DM Sibilot, F Goldwasser, A Scherpereel, H Pegliasco, F Ghiringhelli, N Bouchard, C Sow, I Darik, S Zoppi, P Ly, A Reni, R Daillère, E Deutsch, KA Lee, LA Bolte, JR Björk, RK Weersma, F Barlesi, L Padilha, A Finzel, ML Isaksen, B Escudier, L Albiges, D Planchard, F André, C Cremolini, S Martinez, B Besse, L Zhao, N Segata, J Wojcik, G Kroemer, L Zitvogel_


_Humans, Gastrointestinal Microbiome, Immunotherapy, Carcinoma, Non-Small-Cell Lung, Female, Lung Neoplasms, Male, Dysbiosis, Feces, Middle Aged, Metagenomics, Immune Checkpoint Inhibitors, Treatment Outcome, Aged, Melanoma, Akkermansia, Colorectal Neoplasms, Neoplasms_
The gut microbiota influences the clinical responses of cancer patients to immunecheckpoint inhibitors (ICIs). However, there is no consensus definition of detrimental dysbiosis. Based on metagenomics (MG) sequencing of 245 non-small cell lung cancer (NSCLC) patient feces, we constructed species-level co-abundance networks that were clustered into species-interacting groups (SIGs) correlating with overall survival. Thirty-seven and forty-five MG species (MGSs) were associated with resistance (SIG1) and response (SIG2) to ICIs, respectively. When combined with the quantification of Akkermansia species, this procedure allowed a person-based calculation of a topological score (TOPOSCORE) that was validated in an additional 254 NSCLC patients and in 216 genitourinary cancer patients. Finally, this TOPOSCORE was translated into a 21-bacterial probe set-based qPCR scoring that was validated in a prospective cohort of NSCLC patients as well as in colorectal and melanoma patients. This approach could represent a dynamic diagnosis tool for intestinal dysbiosis to guide personalized microbiota-centered interventions.

[Source](https://doi.org/10.1016/j.cell.2024.05.029)

10.1016/j.cell.2024.05.029

---

## Gut microbiome discovery provides roadmap for life-saving cancer therapies.
 21.06.2024, _G Guglielmi_



[Source](https://doi.org/10.1038/d41586-024-02070-9)

10.1038/d41586-024-02070-9

---

## Faecal microbiota transplantation for sleep disturbance in post-acute COVID-19 syndrome.
 22.06.2024, _RI Lau, Q Su, JYL Ching, RN Lui, TT Chan, MTL Wong, LHS Lau, YK Wing, RNY Chan, HYH Kwok, AHY Ho, YK Tse, CP Cheung, MKT Li, WY Siu, C Liu, W Lu, Y Wang, EOL Chiu, PK Cheong, FKL Chan, SC Ng_


Post-acute COVID-19 syndrome (PACS) is associated with sleep disturbance but treatment options are limited. The aetiology of PACS may be secondary to alterations in the gut microbiome. Here, we report the efficacy of faecal microbiota transplantation (FMT) in alleviating post-COVID insomnia symptoms in a non-randomised, open-label prospective interventional study.

[Source](https://doi.org/10.1016/j.cgh.2024.06.004)

10.1016/j.cgh.2024.06.004

---

## Pan-cancer atlas of tumor-resident microbiome, immunity and prognosis.
 22.06.2024, _D Sheng, C Jin, K Yue, M Yue, Y Liang, X Xue, P Li, G Zhao, L Zhang_


The existence of microbiome in human tumors has been determined widely, but evaluating the contribution of intratumoral bacteria and fungi to tumor immunity and prognosis from a pan-cancer perspective remains absent. We designed an improved microbial analysis pipeline to reduce interference from host sequences, complemented with integration analysis of intratumoral microbiota at species level with clinical indicators, tumor microenvironment, and prognosis across cancer types. We found that intratumoral microbiota is associated with immunophenotyping, with high-immunity subtypes showing greater bacterial and fungal richness compared to low-immunity groups. We also noted that the combination of fungi and bacteria demonstrated promising prognostic value across cancer types. We, thus, present The Cancer Microbiota (TCMbio), an interactive platform that provides the intratumoral bacteria and fungi data, and a comprehensive analysis module for 33 types of cancers. This led to the discovery of clinical and prognostic significance of intratumoral microbes.

[Source](https://doi.org/10.1016/j.canlet.2024.217077)

10.1016/j.canlet.2024.217077

---

## Gut metagenomic analysis of gastric cancer patients reveals <em>Akkermansia</em>, Gammaproteobacteria, and <em>Veillonella</em> microbiota as potential non-invasive biomarkers

[Source](https://doi.org/10.1186/s44342-024-00001-8)

---

## A Bioinformatic Analysis of Gut Microbiota Related with Immune Cell Infiltration in Colorectal Cancer.
 21.06.2024, _D Liu, J Zhou, Q Fu, Y Zhao, P Wang, Y Zheng, M Cui, H Zhang_


The composition of microbiota which correlates with infiltrating immune cells and clinical signatures is not clarified in CRC.

[Source](https://doi.org/10.1080/07357907.2024.2368233)

10.1080/07357907.2024.2368233

---

## An alternative broad-specificity pathway for glycan breakdown in bacteria.
 22.06.2024, _SA Nasseri, AC Lazarski, IL Lemmer, CY Zhang, E Brencher, HM Chen, L Sim, D Panwar, L Betschart, LJ Worrall, H Brumer, NCJ Strynadka, SG Withers_


The vast majority of glycosidases characterized to date follow one of the variations of the 'Koshland' mechanisms1 to hydrolyse glycosidic bonds through substitution reactions. Here we describe a large-scale screen of a human gut microbiome metagenomic library using an assay that selectively identifies non-Koshland glycosidase activities2. Using this, we identify a cluster of enzymes with extremely broad substrate specificities and thoroughly characterize these, mechanistically and structurally. These enzymes not only break glycosidic linkages of both α and β stereochemistry and multiple connectivities, but also cleave substrates that are not hydrolysed by standard glycosidases. These include thioglycosides, such as the glucosinolates from plants, and pseudoglycosidic bonds of pharmaceuticals such as acarbose. This is achieved through a distinct mechanism of hydrolysis that involves oxidation/reduction and elimination/hydration steps, each catalysed by enzyme modules that are in many cases interchangeable between organisms and substrate classes. Homologues of these enzymes occur in both Gram-positive and Gram-negative bacteria associated with the gut microbiome and other body parts, as well as other environments, such as soil and sea. Such alternative step-wise mechanisms appear to constitute largely unrecognized but abundant pathways for glycan degradation as part of the metabolism of carbohydrates in bacteria.

[Source](https://doi.org/10.1038/s41586-024-07574-y)

10.1038/s41586-024-07574-y

---

## The association of maternal factors with the neonatal microbiota and health.
 22.06.2024, _B Zhu, DJ Edwards, KM Spaine, L Edupuganti, A Matveyev, MG Serrano, GA Buck_


_Humans, Female, Infant, Newborn, Pregnancy, Feces, Microbiota, Adult, Cesarean Section, Premature Birth, Gastrointestinal Microbiome, Mouth, Rectum, Male_
The human microbiome plays a crucial role in human health. However, the influence of maternal factors on the neonatal microbiota remains obscure. Herein, our observations suggest that the neonatal microbiotas, particularly the buccal microbiota, change rapidly within 24-48 h of birth but begin to stabilize by 48-72 h after parturition. Network analysis clustered over 200 maternal factors into thirteen distinct groups, and most associated factors were in the same group. Multiple maternal factor groups were associated with the neonatal buccal, rectal, and stool microbiotas. Particularly, a higher maternal inflammatory state and a lower maternal socioeconomic position were associated with a higher alpha diversity of the neonatal buccal microbiota and beta diversity of the neonatal stool microbiota was influenced by maternal diet and cesarean section by 24-72 h postpartum. The risk of admission of a neonate to the newborn intensive care unit was associated with preterm birth as well as higher cytokine levels and probably higher alpha diversity of the maternal buccal microbiota.

[Source](https://doi.org/10.1038/s41467-024-49160-w)

10.1038/s41467-024-49160-w

---

## Revealing the gut microbiome mystery: A meta-analysis revealing differences between individuals with autism spectrum disorder and neurotypical children.
 19.06.2024, _C Yang, H Xiao, H Zhu, Y Du, L Wang_


The brain-gut axis intricately links gut microbiota (GM) dysbiosis to the development or worsening of autism spectrum disorder (ASD). However, the precise GM composition in ASD and the effectiveness of probiotics are unclear. To address this, we performed a thorough meta-analysis of 28 studies spanning PubMed, PsycINFO, Web of Science, Scopus, and MEDLINE, involving 1,256 children with ASD and 1042 neurotypical children, up to February 2024. Using Revman 5.3, we analyzed the relative abundance of 8 phyla and 64 genera. While individuals with ASD did not exhibit significant differences in included phyla, they exhibited elevated levels of Parabacteroides, Anaerostipes, Faecalibacterium, Clostridium, Dorea, Phascolarctobacterium, Lachnoclostridium, Catenibacterium, and Collinsella along with reduced percentages of Barnesiella, Odoribacter, Paraprevotella, Blautia, Turicibacter, Lachnospira, Pseudomonas, Parasutterella, Haemophilus, and Bifidobacterium. Notably, discrepancies in Faecalibacterium, Clostridium, Dorea, Phascolarctobacterium, Catenibacterium, Odoribacter, and Bifidobacterium persisted even upon systematic exclusion of individual studies. Consequently, the GM of individuals with ASD demonstrates an imbalance, with potential increases or decreases in both beneficial and harmful bacteria. Therefore, personalized probiotic interventions tailored to ASD specifics are imperative, rather than a one-size-fits-all approach.

[Source](https://doi.org/10.5582/bst.2024.01123)

10.5582/bst.2024.01123

---

## Propionic acid promotes neurite recovery in damaged multiple sclerosis neurons.
 20.06.2024, _B Gisevius, A Duscha, G Poschmann, K Stühler, J Motte, AL Fisse, S Augustyniak, A Rehm, P Renk, C Böse, D Hubert, K Peters, M Jagst, A Gömer, D Todt, V Bader, M Tokic, S Hirschberg, C Krogias, N Trampe, C Coutourier, C Winnesberg, E Steinmann, K Winklhofer, R Gold, A Haghikia_


Neurodegeneration in the autoimmune disease multiple sclerosis still poses a major therapeutic challenge. Effective drugs that target the inflammation can only partially reduce accumulation of neurological deficits and conversion to progressive disease forms. Diet and the associated gut microbiome are currently being discussed as crucial environmental risk factors that determine disease onset and subsequent progression. In people with multiple sclerosis, supplementation of the short-chain fatty acid propionic acid, as a microbial metabolite derived from the fermentation of a high-fiber diet, has previously been shown to regulate inflammation accompanied by neuroprotective properties. We set out to determine whether the neuroprotective impact of propionic acid is a direct mode of action of short-chain fatty acids on CNS neurons. We analysed neurite recovery in the presence of the short-chain fatty acid propionic acid and butyric acid in a reverse-translational disease-in-a-dish model of human-induced primary neurons differentiated from people with multiple sclerosis-derived induced pluripotent stem cells. We found that recovery of damaged neurites is induced by propionic acid and butyric acid. We could also show that administration of butyric acid is able to enhance propionic acid-associated neurite recovery. Whole-cell proteome analysis of induced primary neurons following recovery in the presence of propionic acid revealed abundant changes of protein groups that are associated with the chromatin assembly, translational, and metabolic processes. We further present evidence that these alterations in the chromatin assembly were associated with inhibition of histone deacetylase class I/II following both propionic acid and butyric acid treatment, mediated by free fatty acid receptor signalling. While neurite recovery in the presence of propionic acid is promoted by activation of the anti-oxidative response, administration of butyric acid increases neuronal ATP synthesis in people with multiple sclerosis-specific induced primary neurons.

[Source](https://doi.org/10.1093/braincomms/fcae182)

10.1093/braincomms/fcae182

---

## What is the impact of microbiota on dry eye: a literature review of the gut-eye axis.
 22.06.2024, _J Song, H Dong, T Wang, H Yu, J Yu, S Ma, X Song, Q Sun, Y Xu, M Liu_


_Dry Eye Syndromes, Humans, Gastrointestinal Microbiome, Animals, Tears_
Dry eye is a chronic and multifactorial ocular surface disease caused by tear film instability or imbalance in the microenvironment of the ocular surface. It can lead to various discomforts such as inflammation of the ocular surface and visual issues. However, the mechanism of dry eye is not clear, which results in dry eye being only relieved but not cured in clinical practice. Finding multiple environmental pathways for dry eye and exploring the pathogenesis of dry eye have become the focus of research. Studies have found that changes in microbiota may be related to the occurrence and development of dry eye disease.

[Source](https://doi.org/10.1186/s12886-024-03526-2)

10.1186/s12886-024-03526-2

---

## Cross-feeding of bifidobacteria promotes intestinal homeostasis: a lifelong perspective on the host health.
 22.06.2024, _M Xiao, C Zhang, H Duan, A Narbad, J Zhao, W Chen, Q Zhai, L Yu, F Tian_


_Humans, Bifidobacterium, Homeostasis, Gastrointestinal Microbiome, Polysaccharides, Host Microbial Interactions, Animals, Fermentation_
Throughout the life span of a host, bifidobacteria have shown superior colonization and glycan abilities. Complex glycans, such as human milk oligosaccharides and plant glycans, that reach the colon are directly internalized by the transport system of bifidobacteria, cleaved into simple structures by extracellular glycosyl hydrolase, and transported to cells for fermentation. The glycan utilization of bifidobacteria introduces cross-feeding activities between bifidobacterial strains and other microbiota, which are influenced by host nutrition and regulate gut homeostasis. This review discusses bifidobacterial glycan utilization strategies, focusing on the cross-feeding involved in bifidobacteria and its potential health benefits. Furthermore, the impact of cross-feeding on the gut trophic niche of bifidobacteria and host health is also highlighted. This review provides novel insights into the interactions between microbe-microbe and host-microbe.

[Source](https://doi.org/10.1038/s41522-024-00524-6)

10.1038/s41522-024-00524-6

---

## Peyer's Patch: Possible target for modulating the Gut-Brain-Axis through microbiota.
 20.06.2024, _R Asgari, MA Bazzazan, A Karimi Jirandehi, S Yousefzadeh, M Alaei, S Keshavarz Shahbaz_


The gastrointestinal (GI) tract and the brain form bidirectional nervous, immune, and endocrine communications known as the gut-brain axis. Several factors can affect this axis; among them, various studies have focused on the microbiota and imply that alterations in microbiota combinations can influence both the brain and GI. Also, many studies have shown that the immune system has a vital role in varying gut microbiota combinations. In the current paper, we will review the multidirectional effects of gut microbiota, immune system, and nervous system on each other. Specifically, this review mainly focuses on the impact of Peyer's patches as a critical component of the gut immune system on the gut-brain axis through affecting the gut's microbial composition. In this way, some factors were discussed as proposed elements of missing gaps in this field.

[Source](https://doi.org/10.1016/j.cellimm.2024.104844)

10.1016/j.cellimm.2024.104844

---

## Alterations in gut microbiota and inflammatory cytokines after administration of antibiotics in mice.
 20.06.2024, _W Gao, X Liu, S Zhang, J Wang, B Qiu, J Shao, W Huang, Y Huang, M Yao, L-L Tang_


Antibiotics are widely used to treat bacterial infection and reduce the mortality rate, while antibiotic overuse can cause gut microbiota dysbiosis. The impact of antibiotics on gut microbiota is not fully understood. In our study, four commonly used antibiotics (ceftazidime, cefoperazone-sulbactam, imipenem-cilastatin, and moxifloxacin) were given subcutaneously to mice, and their impacts on the gut microbiota composition and serum cytokine levels were evaluated through 16S rRNA analysis and a multiplex immunoassay. Antibiotic treatment markedly reduced gut microbiota diversity and changed gut microbiota composition. Antibiotic treatment significantly increased and decreased the abundance of Firmicutes and Bacteroidota, respectively. The antibiotic treatments increased the abundance of opportunistic pathogens such as Enterococcus and decreased that of Lachnospiraceae and Muribaculaceae. For moxifloxacin, the significantly high abundance of Enterococcus and Klebsiella was observed after 14 and 21 days of treatment. However, a relatively low abundance of opportunistic pathogens was found after 14 days of imipenem-cilastatin treatment. Additionally, the serum levels of various pro-inflammatory cytokines, such as IL-1β, IL-12 (p70), and IL-17, significantly increased after 21 days of antibiotic treatments. Overall, these results provide a guide for rational use of antibiotics in clinical settings: short-term use of moxifloxacin is recommended with regard to gut microbiota health, and the 14-day use of imipenem-cilastatin may have a less severe impact than other antibiotics.IMPORTANCEAntibiotic treatments are directly associated with changes in gut microbiota and are effective against both pathogens and beneficial bacteria. Gut microbiota dysbiosis induced by antibiotic treatment could increase the risk of some diseases. Therefore, an adequate understanding of gut microbiota changes after antibiotic use is crucial. In this study, we investigated the effects of continuous treatment with antibiotics on gut microbiota, serum cytokines, and intestinal inflammatory response. Our results suggest that short-term use of moxifloxacin is recommended, and the 14-day use of imipenem-cilastatin may have a less severe effect on gut microbiota health than cefoperazone-sulbactam. These results provide useful guidance on the rational use of antibiotics with regard to gut microbiota health.

[Source](https://doi.org/10.1128/spectrum.03095-23)

10.1128/spectrum.03095-23

---

## A global atlas of soil viruses reveals unexplored biodiversity and potential biogeochemical impacts

[Source](https://doi.org/10.1038/s41564-024-01686-x)

---

## Resistance to freezing conditions of endemic Antarctic polychaetes is enhanced by cryoprotective proteins produced by their microbiome.
 21.06.2024, _E Buschi, A Dell'Anno, M Tangherlini, M Candela, S Rampelli, S Turroni, G Palladino, E Esposito, ML Martire, L Musco, S Stefanni, C Munari, J Fiori, R Danovaro, C Corinaldesi_


_Polychaeta, Animals, Antarctic Regions, Microbiota, Freezing, Phylogeny, Bacterial Proteins_
The microbiome plays a key role in the health of all metazoans. Whether and how the microbiome favors the adaptation processes of organisms to extreme conditions, such as those of Antarctica, which are incompatible with most metazoans, is still unknown. We investigated the microbiome of three endemic and widespread species of Antarctic polychaetes: Leitoscoloplos geminus, Aphelochaeta palmeri, and Aglaophamus trissophyllus. We report here that these invertebrates contain a stable bacterial core dominated by Meiothermus and Anoxybacillus, equipped with a versatile genetic makeup and a unique portfolio of proteins useful for coping with extremely cold conditions as revealed by pangenomic and metaproteomic analyses. The close phylosymbiosis between Meiothermus and Anoxybacillus and these Antarctic polychaetes indicates a connection with their hosts that started in the past to support holobiont adaptation to the Antarctic Ocean. The wide suite of bacterial cryoprotective proteins found in Antarctic polychaetes may be useful for the development of nature-based biotechnological applications.

[Source](https://doi.org/10.1126/sciadv.adk9117)

10.1126/sciadv.adk9117

---

## Microbial co-occurrence network demonstrates spatial and climatic trends for global soil diversity.
 22.06.2024, _N Pechlivanis, G Karakatsoulis, K Kyritsis, M Tsagiopoulou, S Sgardelis, I Kappas, F Psomopoulos_


_Soil Microbiology, Microbiota, Climate, Bacteria, Biodiversity_
Despite recent research efforts to explore the co-occurrence patterns of diverse microbes within soil microbial communities, a substantial knowledge-gap persists regarding global climate influences on soil microbiota behaviour. Comprehending co-occurrence patterns within distinct geoclimatic groups is pivotal for unravelling the ecological structure of microbial communities, that are crucial for preserving ecosystem functions and services. Our study addresses this gap by examining global climatic patterns of microbial diversity. Using data from the Earth Microbiome Project, we analyse a meta-community co-occurrence network for bacterial communities. This method unveils substantial shifts in topological features, highlighting regional and climatic trends. Arid, Polar, and Tropical zones show lower diversity but maintain denser networks, whereas Temperate and Cold zones display higher diversity alongside more modular networks. Furthermore, it identifies significant co-occurrence patterns across diverse climatic regions. Central taxa associated with different climates are pinpointed, highlighting climate's pivotal role in community structure. In conclusion, our study identifies significant correlations between microbial interactions in diverse climatic regions, contributing valuable insights into the intricate dynamics of soil microbiota.

[Source](https://doi.org/10.1038/s41597-024-03528-1)

10.1038/s41597-024-03528-1

---

## Strain dynamics of contaminating bacteria modulate the yield of ethanol biorefineries.
 22.06.2024, _FS de Oliveira Lino, S Garg, SS Li, MA Misiakou, K Kang, BL Vale da Costa, TS Beyer-Pedersen, TG Giacon, TO Basso, G Panagiotou, MOA Sommer_


_Ethanol, Fermentation, Bacteria, Microbiota, Biofuels, Metagenomics, Industrial Microbiology, Temperature_
Bioethanol is a sustainable energy alternative and can contribute to global greenhouse-gas emission reductions by over 60%. Its industrial production faces various bottlenecks, including sub-optimal efficiency resulting from bacteria. Broad-spectrum removal of these contaminants results in negligible gains, suggesting that the process is shaped by ecological interactions within the microbial community. Here, we survey the microbiome across all process steps at two biorefineries, over three timepoints in a production season. Leveraging shotgun metagenomics and cultivation-based approaches, we identify beneficial bacteria and find improved outcome when yeast-to-bacteria ratios increase during fermentation. We provide a microbial gene catalogue which reveals bacteria-specific pathways associated with performance. We also show that Limosilactobacillus fermentum overgrowth lowers production, with one strain reducing yield by ~5% in laboratory fermentations, potentially due to its metabolite profile. Temperature is found to be a major driver for strain-level dynamics. Improved microbial management strategies could unlock environmental and economic gains in this US $ 60 billion industry enabling its wider adoption.

[Source](https://doi.org/10.1038/s41467-024-49683-2)

10.1038/s41467-024-49683-2

---

## A multi-kingdom collection of 33,804 reference genomes for the human vaginal microbiome.
 21.06.2024, _L Huang, R Guo, S Li, X Wu, Y Zhang, S Guo, Y Lv, Z Xiao, J Kang, J Meng, P Zhou, J Ma, W You, Y Zhang, H Yu, J Zhao, G Huang, Z Duan, Q Yan, W Sun_


The human vagina harbours diverse microorganisms-bacteria, viruses and fungi-with profound implications for women's health. Genome-level analysis of the vaginal microbiome across multiple kingdoms remains limited. Here we utilize metagenomic sequencing data and fungal cultivation to establish the Vaginal Microbial Genome Collection (VMGC), comprising 33,804 microbial genomes spanning 786 prokaryotic species, 11 fungal species and 4,263 viral operational taxonomic units. Notably, over 25% of prokaryotic species and 85% of viral operational taxonomic units remain uncultured. This collection significantly enriches genomic diversity, especially for prevalent vaginal pathogens such as BVAB1 (an uncultured bacterial vaginosis-associated bacterium) and Amygdalobacter spp. (BVAB2 and related species). Leveraging VMGC, we characterize functional traits of prokaryotes, notably Saccharofermentanales (an underexplored yet prevalent order), along with prokaryotic and eukaryotic viruses, offering insights into their niche adaptation and potential roles in the vagina. VMGC serves as a valuable resource for studying vaginal microbiota and its impact on vaginal health.

[Source](https://doi.org/10.1038/s41564-024-01751-5)

10.1038/s41564-024-01751-5

---

## Deep learning for predicting 16S rRNA gene copy number.
 22.06.2024, _J Miao, T Chen, M Misir, Y Lin_


_RNA, Ribosomal, 16S, Deep Learning, Gene Dosage, Phylogeny, Computational Biology, Algorithms, Microbiota, Neural Networks, Computer_
Culture-independent 16S rRNA gene metabarcoding is a commonly used method for microbiome profiling. To achieve more quantitative cell fraction estimates, it is important to account for the 16S rRNA gene copy number (hereafter 16S GCN) of different community members. Currently, there are several bioinformatic tools available to estimate the 16S GCN values, either based on taxonomy assignment or phylogeny. Here we present a novel approach ANNA16, Artificial Neural Network Approximator for 16S rRNA gene copy number, a deep learning-based method that estimates the 16S GCN values directly from the 16S gene sequence strings. Based on 27,579 16S rRNA gene sequences and gene copy number data from the rrnDB database, we show that ANNA16 outperforms the commonly used 16S GCN prediction algorithms. Interestingly, Shapley Additive exPlanations (SHAP) shows that ANNA16 can identify unexpected informative positions in 16S rRNA gene sequences without any prior phylogenetic knowledge, which suggests potential applications beyond 16S GCN prediction.

[Source](https://doi.org/10.1038/s41598-024-64658-5)

10.1038/s41598-024-64658-5

---

## GEM-based computational modeling for exploring metabolic interactions in a microbial community

[Source](https://doi.org/10.1371/journal.pcbi.1012233)

---

## Techniques, procedures, and applications in microbiome analysis

[Source](https://doi.org/10.1016/bs.adgen.2024.01.003)

---

