## Time series strain tracking analysis post fecal transplantation identifies individual specific patterns of fecal dominant donor, recipient, and unrelated microbial strains.
 19.09.2022, _H Koo, CD Morrow_


_Anti-Bacterial Agents, Fecal Microbiota Transplantation, Feces, Nucleotides, Time Factors_
Fecal microbial transplantation (FMT) has been used with the therapeutic intent to change the functions of the gut microbial community in metabolism and host immunity. For most of these therapies, the recipients are not given antibiotics to eliminate the microbial community prior to transplant with donor fecal microbes resulting in the initial gut microbial community following FMT consisting of a consortium of donor and recipient microbes. The detailed analysis of the fecal samples from these FMT over time provides a unique opportunity to study the changes in the gut microbial strain community that occurs following the introduction of new microbial strains (donor) into an established community (recipient).

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274633)

10.1371/journal.pone.0274633

---

## Microbiome complexity shapes metabolism.
 21.09.2022, _L Litichevskiy, CA Thaiss_


Equipped with a novel isolator-housed metabolic cage system, a study in PLOS Biology assessed how the metabolism of mice harboring a defined minimal microbial community (OligoMM12) differs from that of germ-free and conventionally colonized mice.

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001793)

10.1371/journal.pbio.3001793

---

## Minimal overall divergence of the gut microbiome in an adaptive radiation of <em>Cyprinodon </em>pupfishes despite potential adaptive enrichment for scale-eating

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0273177)

---

## Molecular characterization and biofilm-formation analysis of<em> Listeria monocytogenes</em>, <em>Salmonella </em>spp., and <em>Escherichia coli</em> isolated from Brazilian swine slaughterhouses

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274636)

---

## Ecology and distribution of <em>Leptospira</em> spp., reservoir hosts and environmental interaction in Sri Lanka, with identification of a new strain

[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0010757#sec002)

---

## A simple kit to use computational notebooks for more openness, reproducibility, and productivity in research

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010356)

---

## Bacterial concentration and <em>Campylobacter</em> spp. quantification differ when fresh or ultra-frozen samples are analysed over time using molecular biology and culture-based methods

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274682)

---

## Robust inference of population size histories from genomic sequencing data

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010419)

---

## Rapid genotyping of targeted viral samples using Illumina short-read sequencing data

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274414)

---

## Improved assessments of bulk milk microbiota composition via sample preparation and DNA extraction methods.
 21.09.2022, _Z Xue, ML Marco_


_Animals, Bacteria, Cattle, DNA, Bacterial, Endopeptidase K, Glycerol, Microbiota, Milk, RNA, Ribosomal, 16S, Reproducibility of Results, Ribonuclease, Pancreatic_
Although bacterial detection by 16S rRNA gene amplicon DNA sequencing is a widely-applied technique, standardized methods for sample preparation and DNA extraction are needed to ensure accuracy, reproducibility, and scalability for automation. To develop these methods for bovine bulk milk, we assembled and tested a bacterial cell mock community (BCMC) containing bacterial species commonly found in milk. The following protocol variations were examined:: BCMC enumeration (colony enumeration or microscopy), sample volume (200 μl to 30 ml), sample storage condition (frozen in PBS or 25% glycerol or exposure to freeze-thaw cycles), cell lysis method (bead-beating, vortex, enzymatic), and DNA extraction procedure (MagMAX Total, MagMAX CORE, and MagMAX Ultra 2.0, with and without either Proteinase K or RNase A). Cell enumeration by microscopy was more accurate for quantification of the BCMC contents. We found that least 10 mL (≥ 104 cells in high quality milk) is needed for reproducible bacterial detection by 16S rRNA gene amplicon DNA sequencing, whereas variations in storage conditions caused minor differences in the BCMC. For DNA extraction and purification, a mild lysis step (bead-beating for 10 s at 4 m/s or vortexing at 1800 rpm for 10 s) paired with the MagMAX Total kit and Proteinase K digestion provided the most accurate representation of the BCMC. Cell lysis procedures conferred the greatest changes to milk microbiota composition and these effects were confirmed to provide similar results for commercial milk samples. Overall, our systematic approach with the BCMC is broadly applicable to other milk, food, and environmental samples therefore recommended for improving accuracy of culture-independent, DNA sequence-based analyses of microbial composition in different habitats.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0267992)

10.1371/journal.pone.0267992

---

