## Microbiome definition re-visited: old concepts and new challenges.
 03.07.2020, _G Berg, D Rybakova, D Fischer, T Cernava, MC Vergès, T Charles, X Chen, L Cocolin, K Eversole, GH Corral, M Kazou, L Kinkel, L Lange, N Lima, A Loy, JA Macklin, E Maguin, T Mauchline, R McClure, B Mitter, M Ryan, I Sarand, H Smidt, B Schelkle, H Roume, GS Kiran, J Selvin, RSC Souza, L van Overbeek, BK Singh, M Wagner, A Walsh, A Sessitsch, M Schloter_


The field of microbiome research has evolved rapidly over the past few decades and has become a topic of great scientific and public interest. As a result of this rapid growth in interest covering different fields, we are lacking a clear commonly agreed definition of the term "microbiome." Moreover, a consensus on best practices in microbiome research is missing. Recently, a panel of international experts discussed the current gaps in the frame of the European-funded MicrobiomeSupport project. The meeting brought together about 40 leaders from diverse microbiome areas, while more than a hundred experts from all over the world took part in an online survey accompanying the workshop. This article excerpts the outcomes of the workshop and the corresponding online survey embedded in a short historical introduction and future outlook. We propose a definition of microbiome based on the compact, clear, and comprehensive description of the term provided by Whipps et al. in 1988, amended with a set of novel recommendations considering the latest technological developments and research findings. We clearly separate the terms microbiome and microbiota and provide a comprehensive discussion considering the composition of microbiota, the heterogeneity and dynamics of microbiomes in time and space, the stability and resilience of microbial networks, the definition of core microbiomes, and functionally relevant keystone species as well as co-evolutionary principles of microbe-host and inter-species interactions within the microbiome. These broad definitions together with the suggested unifying concepts will help to improve standardization of microbiome studies in the future, and could be the starting point for an integrated assessment of data resulting in a more rapid transfer of knowledge from basic science into practice. Furthermore, microbiome standards are important for solving new challenges associated with anthropogenic-driven changes in the field of planetary health, for which the understanding of microbiomes might play a key role. Video Abstract.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00875-0)

10.1186/s40168-020-00875-0

---

## Large-scale metabolic interaction network of the mouse and human gut microbiota.
 01.07.2020, _R Lim, JJT Cabatbat, TLP Martin, H Kim, S Kim, J Sung, CM Ghim, PJ Kim_


The role of our gut microbiota in health and disease is largely attributed to the collective metabolic activities of the inhabitant microbes. A system-level framework of the microbial community structure, mediated through metabolite transport, would provide important insights into the complex microbe-microbe and host-microbe chemical interactions. This framework, if adaptable to both mouse and human systems, would be useful for mechanistic interpretations of the vast amounts of experimental data from gut microbiomes in murine animal models, whether humanized or not. Here, we constructed a literature-curated, interspecies network of the mammalian gut microbiota for mouse and human hosts, called NJC19. This network is an extensive data resource, encompassing 838 microbial species (766 bacteria, 53 archaea, and 19 eukaryotes) and 6 host cell types, interacting through 8,224 small-molecule transport and macromolecule degradation events. Moreover, we compiled 912 negative associations between organisms and metabolic compounds that are not transportable or degradable by those organisms. Our network may facilitate experimental and computational endeavors for the mechanistic investigations of host-associated microbial communities.

[Source](https://www.nature.com/articles/s41597-020-0516-5)

10.1038/s41597-020-0516-5

---

## Very early-life exposure to microbiota-induced TNF drives the maturation of neonatal pre-cDC1.
 17.06.2020, _A Köhler, S Delbauve, J Smout, D Torres, V Flamand_


Induction of immune protection against pathogens is particularly crucial during the neonatal period dominated by anti-inflammatory and tolerance immunity. The preclinical study was carried out to determine whether environmental factors such as microbiota may influence early life immunity by impacting the development and the functional maturation of precursors of type 1 conventional dendritic cells (pre-cDC1), endowed with regulatory properties.

[Source](https://gut.bmj.com/content/early/2020/06/15/gutjnl-2019-319700)

10.1136/gutjnl-2019-319700

---

## Arrhythmic Gut Microbiome Signatures Predict Risk of Type 2 Diabetes.
 03.07.2020, _S Reitmeier, S Kiessling, T Clavel, M List, EL Almeida, TS Ghosh, K Neuhaus, H Grallert, J Linseisen, T Skurk, B Brandl, TA Breuninger, M Troll, W Rathmann, B Linkohr, H Hauner, M Laudes, A Franke, CI Le Roy, JT Bell, T Spector, J Baumbach, PW O'Toole, A Peters, D Haller_


Lifestyle, obesity, and the gut microbiome are important risk factors for metabolic disorders. We demonstrate in 1,976 subjects of a German population cohort (KORA) that specific microbiota members show 24-h oscillations in their relative abundance and identified 13 taxa with disrupted rhythmicity in type 2 diabetes (T2D). Cross-validated prediction models based on this signature similarly classified T2D. In an independent cohort (FoCus), disruption of microbial oscillation and the model for T2D classification was confirmed in 1,363 subjects. This arrhythmic risk signature was able to predict T2D in 699 KORA subjects 5 years after initial sampling, being most effective in combination with BMI. Shotgun metagenomic analysis functionally linked 26 metabolic pathways to the diurnal oscillation of gut bacteria. Thus, a cohort-specific risk pattern of arrhythmic taxa enables classification and prediction of T2D, suggesting a functional link between circadian rhythms and the microbiome in metabolic diseases.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312820303437)

10.1016/j.chom.2020.06.004

---

## Ranking microbiome variance in inflammatory bowel disease: a large longitudinal intercontinental study.
 15.06.2020, _AG Clooney, J Eckenberger, E Laserna-Mendieta, KA Sexton, MT Bernstein, K Vagianos, M Sargent, FJ Ryan, C Moran, D Sheehan, RD Sleator, LE Targownik, CN Bernstein, F Shanahan, MJ Claesson_


The microbiome contributes to the pathogenesis of inflammatory bowel disease (IBD) but the relative contribution of different lifestyle and environmental factors to the compositional variability of the gut microbiota is unclear.

[Source](https://gut.bmj.com/content/early/2020/06/11/gutjnl-2020-321106)

10.1136/gutjnl-2020-321106

---

## Cholesterol Metabolism by Uncultured Human Gut Bacteria Influences Host Cholesterol Level.
 16.06.2020, _DJ Kenny, DR Plichta, D Shungin, N Koppel, AB Hall, B Fu, RS Vasan, SY Shaw, H Vlamakis, EP Balskus, RJ Xavier_


The human microbiome encodes extensive metabolic capabilities, but our understanding of the mechanisms linking gut microbes to human metabolism remains limited. Here, we focus on the conversion of cholesterol to the poorly absorbed sterol coprostanol by the gut microbiota to develop a framework for the identification of functional enzymes and microbes. By integrating paired metagenomics and metabolomics data from existing cohorts with biochemical knowledge and experimentation, we predict and validate a group of microbial cholesterol dehydrogenases that contribute to coprostanol formation. These enzymes are encoded by ismA genes in a clade of uncultured microorganisms, which are prevalent in geographically diverse human cohorts. Individuals harboring coprostanol-forming microbes have significantly lower fecal cholesterol levels and lower serum total cholesterol with effects comparable to those attributed to variations in lipid homeostasis genes. Thus, cholesterol metabolism by these microbes may play important roles in reducing intestinal and serum cholesterol concentrations, directly impacting human health.

[Source](https://www.sciencedirect.com/science/article/pii/S193131282030295X)

10.1016/j.chom.2020.05.013

---

## Amending microbiota by targeting intestinal inflammation with TNF blockade attenuates development of colorectal cancer

[Source](https://www.nature.com/articles/s43018-020-0078-7)

---

## A Systematic Analysis of Mosquito-Microbiome Biosynthetic Gene Clusters Reveals Antimalarial Siderophores that Reduce Mosquito Reproduction Capacity.
 03.07.2020, _JG Ganley, A Pandey, K Sylvester, KY Lu, M Toro-Moreno, S Rütschlin, JM Bradford, CJ Champion, T Böttcher, J Xu, ER Derbyshire_


Advances in infectious disease control strategies through genetic manipulation of insect microbiomes have heightened interest in microbially produced small molecules within mosquitoes. Herein, 33 mosquito-associated bacterial genomes were mined and over 700 putative biosynthetic gene clusters (BGCs) were identified, 135 of which belong to known classes of BGCs. After an in-depth analysis of the 135 BGCs, iron-binding siderophores were chosen for further investigation due to their high abundance and well-characterized bioactivities. Through various metabolomic strategies, eight siderophore scaffolds were identified in six strains of mosquito-associated bacteria. Among these, serratiochelin A and pyochelin were found to reduce female Anopheles gambiae overall fecundity likely by lowering their blood-feeding rate. Serratiochelin A and pyochelin were further found to inhibit the Plasmodium parasite asexual blood and liver stages in vitro. Our work supplies a bioinformatic resource for future mosquito-microbiome studies and highlights an understudied source of bioactive small molecules.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S2451945620302270)

10.1016/j.chembiol.2020.06.004

---

## Microbial succession during the transition from active to inactive stages of deep-sea hydrothermal vent sulfide chimneys.
 03.07.2020, _J Hou, SM Sievert, Y Wang, JS Seewald, VP Natarajan, F Wang, X Xiao_


Deep-sea hydrothermal vents are highly productive biodiversity hotspots in the deep ocean supported by chemosynthetic microorganisms. Prominent features of these systems are sulfide chimneys emanating high-temperature hydrothermal fluids. While several studies have investigated the microbial diversity in both active and inactive sulfide chimneys that have been extinct for up to thousands of years, little is known about chimneys that have ceased activity more recently, as well as the microbial succession occurring during the transition from active to inactive chimneys.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00851-8)

10.1186/s40168-020-00851-8

---

## Phase-variable capsular polysaccharides and lipoproteins modify bacteriophage susceptibility in Bacteroides thetaiotaomicron.
 30.06.2020, _NT Porter, AJ Hryckowian, BD Merrill, JJ Fuentes, JO Gardner, RWP Glowacki, S Singh, RD Crawford, ES Snitkin, JL Sonnenburg, EC Martens_


A variety of cell surface structures dictate interactions between bacteria and their environment, including their viruses (bacteriophages). Members of the human gut Bacteroidetes characteristically produce several phase-variable capsular polysaccharides (CPSs), but their contributions to bacteriophage interactions are unknown. To begin to understand how CPSs have an impact on Bacteroides-phage interactions, we isolated 71 Bacteroides thetaiotaomicron-infecting bacteriophages from two locations in the United States. Using B. thetaiotaomicron strains that express defined subsets of CPSs, we show that CPSs dictate host tropism for these phages and that expression of non-permissive CPS variants is selected under phage predation, enabling survival. In the absence of CPSs, B. thetaiotaomicron escapes bacteriophage predation by altering expression of eight distinct phase-variable lipoproteins. When constitutively expressed, one of these lipoproteins promotes resistance to multiple bacteriophages. Our results reveal important roles for Bacteroides CPSs and other cell surface structures that allow these bacteria to persist under bacteriophage predation, and hold important implications for using bacteriophages therapeutically to target gut symbionts.

[Source](https://www.nature.com/articles/s41564-020-0746-5)

10.1038/s41564-020-0746-5

---

## ATLAS: a Snakemake workflow for assembly, annotation, and genomic binning of metagenome sequence data.
 25.06.2020, _S Kieser, J Brown, EM Zdobnov, M Trajkovski, LA McCue_


Metagenomics studies provide valuable insight into the composition and function of microbial populations from diverse environments; however, the data processing pipelines that rely on mapping reads to gene catalogs or genome databases for cultured strains yield results that underrepresent the genes and functional potential of uncultured microbes. Recent improvements in sequence assembly methods have eased the reliance on genome databases, thereby allowing the recovery of genomes from uncultured microbes. However, configuring these tools, linking them with advanced binning and annotation tools, and maintaining provenance of the processing continues to be challenging for researchers.

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03585-4)

10.1186/s12859-020-03585-4

---

## High-Throughput Stool Metaproteomics: Method and Application to Human Specimens.
 01.07.2020, _CG Gonzalez, HC Wastyk, M Topf, CD Gardner, JL Sonnenburg, JE Elias_


Stool-based proteomics is capable of significantly augmenting our understanding of host-gut microbe interactions. However, compared to competing technologies, such as metagenomics and 16S rRNA sequencing, it is underutilized due to its low throughput and the negative impact sample contaminants can have on highly sensitive mass spectrometry equipment. Here, we present a new stool proteomic processing pipeline that addresses these shortcomings in a highly reproducible and quantitative manner. Using this method, 290 samples from a dietary intervention study were processed in approximately 1.5 weeks, largely done by a single researcher. These data indicated a subtle but distinct monotonic increase in the number of significantly altered proteins between study participants on fiber- or fermented food-enriched diets. Lastly, we were able to classify study participants based on their diet-altered proteomic profiles and demonstrated that classification accuracies of up to 89% could be achieved by increasing the number of subjects considered. Taken together, this study represents the first high-throughput proteomic method for processing stool samples in a technically reproducible manner and has the potential to elevate stool-based proteomics as an essential tool for profiling host-gut microbiome interactions in a clinical setting.IMPORTANCE Widely available technologies based on DNA sequencing have been used to describe the kinds of microbes that might correlate with health and disease. However, mechanistic insights might be best achieved through careful study of the dynamic proteins at the interface between the foods we eat, our microbes, and ourselves. Mass spectrometry-based proteomics has the potential to revolutionize our understanding of this complex system, but its application to clinical studies has been hampered by low-throughput and laborious experimentation pipelines. In response, we developed SHT-Pro, the first high-throughput pipeline designed to rapidly handle large stool sample sets. With it, a single researcher can process over one hundred stool samples per week for mass spectrometry analysis, conservatively approximately 10× to 100× faster than previous methods, depending on whether isobaric labeling is used or not. Since SHT-Pro is fairly simple to implement using commercially available reagents, it should be easily adaptable to large-scale clinical studies.

[Source](https://msystems.asm.org/content/5/3/e00200-20)

10.1128/mSystems.00200-20

---

## Interindividual Variation in Dietary Carbohydrate Metabolism by Gut Bacteria Revealed with Droplet Microfluidic Culture.
 01.07.2020, _MM Villa, RJ Bloom, JD Silverman, HK Durand, S Jiang, A Wu, EP Dallow, S Huang, L You, LA David_


Culture and screening of gut bacteria enable testing of microbial function and therapeutic potential. However, the diversity of human gut microbial communities (microbiota) impedes comprehensive experimental studies of individual bacterial taxa. Here, we combine advances in droplet microfluidics and high-throughput DNA sequencing to develop a platform for separating and assaying growth of microbiota members in picoliter droplets (MicDrop). MicDrop enabled us to cultivate 2.8 times more bacterial taxa than typical batch culture methods. We then used MicDrop to test whether individuals possess similar abundances of carbohydrate-degrading gut bacteria, using an approach which had previously not been possible due to throughput limitations of traditional bacterial culture techniques. Single MicDrop experiments allowed us to characterize carbohydrate utilization among dozens of gut bacterial taxa from distinct human stool samples. Our aggregate data across nine healthy stool donors revealed that all of the individuals harbored gut bacterial species capable of degrading common dietary polysaccharides. However, the levels of richness and abundance of polysaccharide-degrading species relative to monosaccharide-consuming taxa differed by up to 2.6-fold and 24.7-fold, respectively. Additionally, our unique dataset suggested that gut bacterial taxa may be broadly categorized by whether they can grow on single or multiple polysaccharides, and we found that this lifestyle trait is correlated with how broadly bacterial taxa can be found across individuals. This demonstration shows that it is feasible to measure the function of hundreds of bacterial taxa across multiple fecal samples from different people, which should in turn enable future efforts to design microbiota-directed therapies and yield new insights into microbiota ecology and evolution.IMPORTANCE Bacterial culture and assay are components of basic microbiological research, drug development, and diagnostic screening. However, community diversity can make it challenging to comprehensively perform experiments involving individual microbiota members. Here, we present a new microfluidic culture platform that makes it feasible to measure the growth and function of microbiota constituents in a single set of experiments. As a proof of concept, we demonstrate how the platform can be used to measure how hundreds of gut bacterial taxa drawn from different people metabolize dietary carbohydrates. Going forward, we expect this microfluidic technique to be adaptable to a range of other microbial assay needs.

[Source](https://msystems.asm.org/content/5/3/e00864-19.abstract)

10.1128/mSystems.00864-19

---

