## Selective and cross-reactive SARS-CoV-2 T cell epitopes in unexposed humans.
 05.10.2020, _J Mateus, A Grifoni, A Tarke, J Sidney, SI Ramirez, JM Dan, ZC Burger, SA Rawlings, DM Smith, E Phillips, S Mallal, M Lammers, P Rubiro, L Quiambao, A Sutherland, ED Yu, R da Silva Antunes, J Greenbaum, A Frazier, AJ Markmann, L Premkumar, A de Silva, B Peters, S Crotty, A Sette, D Weiskopf_


Many unknowns exist about human immune responses to the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus. SARS-CoV-2-reactive CD4+ T cells have been reported in unexposed individuals, suggesting preexisting cross-reactive T cell memory in 20 to 50% of people. However, the source of those T cells has been speculative. Using human blood samples derived before the SARS-CoV-2 virus was discovered in 2019, we mapped 142 T cell epitopes across the SARS-CoV-2 genome to facilitate precise interrogation of the SARS-CoV-2-specific CD4+ T cell repertoire. We demonstrate a range of preexisting memory CD4+ T cells that are cross-reactive with comparable affinity to SARS-CoV-2 and the common cold coronaviruses human coronavirus (HCoV)-OC43, HCoV-229E, HCoV-NL63, and HCoV-HKU1. Thus, variegated T cell memory to coronaviruses that cause the common cold may underlie at least some of the extensive heterogeneity observed in coronavirus disease 2019 (COVID-19) disease.

[Source](https://science.sciencemag.org/content/370/6512/89)

10.1126/science.abd3871

---

## The major genetic risk factor for severe COVID-19 is inherited from Neanderthals.
 30.09.2020, _H Zeberg, S Pääbo_


A recent genetic association study1 identified a gene cluster on chromosome 3 as a risk locus for respiratory failure upon SARS-CoV-2 infection. A new study2 comprising 3,199 hospitalized COVID-19 patients and controls finds that this is the major genetic risk factor for severe SARS-CoV-2 infection and hospitalization (COVID-19 Host Genetics Initiative). Here, we show that the risk is conferred by a genomic segment of ~50 kb that is inherited from Neanderthals and is carried by ~50% of people in South Asia and ~16% of people in Europe today.

[Source](https://www.nature.com/articles/s41586-020-2818-3)

10.1038/s41586-020-2818-3

---

## Massive and rapid COVID-19 testing is feasible by extraction-free SARS-CoV-2 RT-PCR.
 01.10.2020, _I Smyrlaki, M Ekman, A Lentini, N Rufino de Sousa, N Papanicolaou, M Vondracek, J Aarum, H Safari, S Muradrasoli, AG Rothfuchs, J Albert, B Högberg, B Reinius_


_Benchmarking, Betacoronavirus, Clinical Laboratory Techniques, Coronavirus Infections, DNA Primers, Hot Temperature, Humans, Pandemics, Pneumonia, Viral, RNA, Viral, Reverse Transcriptase Polymerase Chain Reaction, Sensitivity and Specificity, Sweden, Viral Plaque Assay_
Coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is commonly diagnosed by reverse transcription polymerase chain reaction (RT-PCR) to detect viral RNA in patient samples, but RNA extraction constitutes a major bottleneck in current testing. Methodological simplification could increase diagnostic availability and efficiency, benefitting patient care and infection control. Here, we describe methods circumventing RNA extraction in COVID-19 testing by performing RT-PCR directly on heat-inactivated or lysed samples. Our data, including benchmarking using 597 clinical patient samples and a standardised diagnostic system, demonstrate that direct RT-PCR is viable option to extraction-based tests. Using controlled amounts of active SARS-CoV-2, we confirm effectiveness of heat inactivation by plaque assay and evaluate various generic buffers as transport medium for direct RT-PCR. Significant savings in time and cost are achieved through RNA-extraction-free protocols that are directly compatible with established PCR-based testing pipelines. This could aid expansion of COVID-19 testing.

[Source](https://www.nature.com/articles/s41467-020-18611-5)

10.1038/s41467-020-18611-5

---

## Identification of new enterosynes using prebiotics: roles of bioactive lipids and mu-opioid receptor signalling in humans and mice

[Source](https://gut.bmj.com/content/early/2020/09/29/gutjnl-2019-320230)

---

## Assessing the Variation within the Oral Microbiome of Healthy Adults.
 02.10.2020, _JT Nearing, V DeClercq, J Van Limbergen, MGI Langille_


More than 1,000 different species of microbes have been found to live within the human oral cavity, where they play important roles in maintaining both oral and systemic health. Several studies have identified the core members of this microbial community; however, the factors that determine oral microbiome composition are not well understood. In this study, we exam the salivary oral microbiome of 1,049 Atlantic Canadians using 16S rRNA gene sequencing to determine which dietary, lifestyle, and anthropometric features play a role in shaping microbial community composition. Features that were identified as being significantly associated with overall composition then were additionally examined for genera, amplicon sequence variants, and predicted pathway abundances that were associated with these features. Several associations were replicated in an additional secondary validation data set. Overall, we found that several anthropometric measurements, including waist-hip ratio (WHR), height, and fat-free mass, as well as age and sex, were associated with overall oral microbiome structure in both our exploratory and validation data sets. We were unable to validate any dietary impacts on overall taxonomic oral microbiome composition but did find evidence to suggest potential contributions from factors such as the number of vegetable and refined grain servings an individual consumes. Interestingly, each one of these factors on its own was associated with only minor shifts in the overall taxonomic composition of the oral microbiome, suggesting that future biomarker identification for several diseases associated with the oral microbiome can be undertaken without the worry of confounding factors obscuring biological signals.IMPORTANCE The human oral cavity is inhabited by a diverse community of microbes, known as the human oral microbiome. These microbes play a role in maintaining both oral and systemic health and, as such, have been proposed to be useful biomarkers of disease. However, to identify these biomarkers, we first need to determine the composition and variation of the healthy oral microbiome. In this report, we investigate the oral microbiome of 1,049 healthy individuals to determine which genera and amplicon sequence variants are commonly found between individual oral microbiomes. We then further investigate how lifestyle, anthropometric, and dietary choices impact overall microbiome composition. Interestingly, the results from this investigation showed that while many features were significantly associated with oral microbiome composition, no single biological factor explained a variation larger than 2%. These results indicate that future work on biomarker detection may be encouraged by the lack of strong confounding factors.

[Source](https://msphere.asm.org/content/5/5/e00451-20)

10.1128/mSphere.00451-20

---

## Strain-Level Analysis of Bifidobacterium spp. from Gut Microbiomes of Adults with Differing Lactase Persistence Genotypes.
 03.10.2020, _V Schmidt, H Enav, TD Spector, ND Youngblut, RE Ley_


One of the strongest associations between human genetics and the gut microbiome is a greater relative abundance of Bifidobacterium in adults with lactase gene (LCT) single nucleotide polymorphisms (SNPs) associated with lactase nonpersistence (GG genotypes), versus lactase persistence (AA/AG genotypes). To gain a finer-grained phylogenetic resolution of this association, we interrogated 1,680 16S rRNA libraries and 245 metagenomes from gut microbiomes of adults with various lactase persistence genotypes. We further employed a novel genome-capture-based enrichment of Bifidobacterium DNA from a subset of these metagenomes, including monozygotic (MZ) twin pairs, each sampled 2 or 3 times. B. adolescentis and B. longum were the most abundant Bifidobacterium species regardless of host LCT genotype. LCT genotypes could not be discriminated based on relative abundances of Bifidobacterium species or Bifidobacterium community structure. Three distinct metagenomic analysis methods of Bifidobacterium-enriched DNA revealed intraindividual temporal stability of B. longum, B. adolescentis, and B. bifidum strains against the background of a changeable microbiome. Two of our three methods also observed greater strain sharing within MZ twin pairs than within unrelated individuals for B. adolescentis, while no method revealed an effect of host LCT genotype on Bifidobacterium strain composition. Our results support a "rising tide lifts all boats" model for the dominant bifidobacteria in the adult gut: their higher abundance in lactase-nonpersistent than in lactase-persistent individuals results from an expansion at the genus level. Bifidobacterium species are known to be transmitted from mother to child and stable within individuals in infancy and childhood: our results extend this stability into adulthood.IMPORTANCE When humans domesticated animals, some adapted genetically to digest milk into adulthood (lactase persistence). The gut microbiomes of people with lactase-persistent genotypes (AA or AG) differ from those with lactase-nonpersistent genotypes (GG) by containing fewer bacteria belonging to the bifidobacteria, a group which contains beneficial species. Here, we asked if the gut microbiomes of adults with GG and AA/AG genotypes differ in the species of bifidobacteria present. In particular, we used a novel technique which allowed us to compare bifidobacteria in adults at the strain level, without the traditional need for culturing. Our results show that the GG genotype enhances the abundance of bifidobacteria regardless of species. We also noted that a person's specific strains are recoverable several years later, and twins can share the same ones. Given that bifidobacteria are inherited from mother to child, strain stability over time in adulthood suggests long-term, multigenerational inheritance.

[Source](https://msystems.asm.org/content/5/5/e00911-20)

10.1128/mSystems.00911-20

---

## Description of a new member of the family Erysipelotrichaceae: Dakotella fusiforme gen. nov., sp. nov., isolated from healthy human feces

[Source](https://peerj.com/articles/10071/)

---

## Distinct signatures of gut microbiome and metabolites associated with significant fibrosis in non-obese NAFLD

[Source](https://www.nature.com/articles/s41467-020-18754-5)

---

## Fecal microbiota in client-owned obese dogs changes after weight loss with a high-fiber-high-protein diet

[Source](https://peerj.com/articles/9706/)

---

## Achieving similar root microbiota composition in neighbouring plants through airborne signalling

[Source](https://www.nature.com/articles/s41396-020-00759-z)

---

## Human-Gut-DNA Virome Variations across Geography, Ethnicity, and Urbanization

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30458-3)

---

## Phage gene expression and host responses lead to infection-dependent costs of CRISPR immunity

[Source](https://www.nature.com/articles/s41396-020-00794-w)

---

## Interaction of bacterial metagenome and virome in patients with cirrhosis and hepatic encephalopathy 

[Source](https://gut.bmj.com/content/early/2020/09/29/gutjnl-2020-322470)

---

## Stability of the Virome in Lab- and Field-Collected Aedes albopictus Mosquitoes across Different Developmental Stages and Possible Core Viruses in the Publicly Available Virome Data of Aedes Mosquitoes

[Source](https://msystems.asm.org/content/5/5/e00640-20)

---

