## A 16th century Escherichia coli draft genome associated with an opportunistic bile infection.
 20.06.2022, _GS Long, J Klunk, AT Duggan, M Tapson, V Giuffra, L Gazz√®, A Fornaciari, S Duchene, G Fornaciari, O Clermont, E Denamur, GB Golding, H Poinar_


_Animals, Bile, Escherichia coli, Escherichia coli Infections, Genome, Bacterial, Mice, Opportunistic Infections_
Escherichia coli - one of the most characterized bacteria and a major public health concern - remains invisible across the temporal landscape. Here, we present the meticulous reconstruction of the first ancient E. coli genome from a 16th century gallstone from an Italian mummy with chronic cholecystitis. We isolated ancient DNA and reconstructed the ancient E. coli genome. It consisted of one chromosome of 4446 genes and two putative plasmids with 52 genes. The E. coli strain belonged to the phylogroup A and an exceptionally rare sequence type 4995. The type VI secretion system component genes appears to be horizontally acquired from Klebsiella aerogenes, however we could not identify any pathovar specific genes nor any acquired antibiotic resistances. A sepsis mouse assay showed that a closely related contemporary E. coli strain was avirulent. Our reconstruction of this ancient E. coli helps paint a more complete picture of the burden of opportunistic infections of the past.

[Source](https://www.nature.com/articles/s42003-022-03527-1)

10.1038/s42003-022-03527-1

---

## Strong pathogen competition in neonatal gut colonisation

[Source](https://www.biorxiv.org/content/10.1101/2022.06.19.496579v1.abstract)

---

## Dysbiosis of Skin Microbiota with Increased Fungal Diversity is Associated with Severity of Disease in Atopic Dermatitis

[Source](https://onlinelibrary.wiley.com/doi/10.1111/jdv.18347)

---

## Loss of Novel Diversity in Human Gut Microbiota Associated with Ongoing Urbanization in China.
 21.06.2022, _S Sun, H Wang, AG Howard, J Zhang, C Su, Z Wang, S Du, AA Fodor, P Gordon-Larsen, B Zhang_


Recent rapid and large-scale urbanization has had a profound impact on human lifestyles and is associated with an increased risk of many diseases. Recent studies have revealed large differences in the human gut microbiota across populations in countries at different stages of urbanization. However, few studies have analyzed the impact of ongoing urbanization within the same geographic region. In this study, we sampled 214 participants in communities of different urbanization levels within two provinces of China and reconstructed draft prokaryotic genomes with metagenome sequences. The genomes were clustered into 447 species-level operational taxonomic units (OTUs), among which 196 did not have genomes in public reference databases according to the GTDB-Tk pipeline. The novel OTUs comprised 19.1% abundance in rural participants and 16.0% in urban participants, increasing the proportion of classified reads from 47.6% to 65.3% across all samples. Among the unknown OTUs, 26 OTUs present in rural samples were absent in urban participants, while 70 unknown OTUs were more abundant in rural than urban participants, suggesting potential loss and growth suppression of novel human symbionts during urbanization. Moreover, there were a higher number of genes, especially transporters, identified in genomes assembled from urban samples. This change in gene functionality indicates that urbanization not only altered the community structure of the human gut microbiota but also impacted its functional capacity. Taken together, these data show a dramatic change in the microbiota with urbanization and suggest the importance of cataloging microbial diversity from rural populations while these communities still exist. IMPORTANCE Previous studies have reported the differences in human gut microbiota across populations of different urbanization levels, but most of the studies focused on populations across different geographic regions. In this study, we analyzed the impact of ongoing urbanization in neighborhoods within the same geographic region. By assembling shotgun metagenome sequences, we reconstructed prokaryotic genomes from human gut microbiota and found that the novel bacterial OTUs were less abundant and less prevalent in urban participants than in rural participants, indicating potential loss and suppression of novel human symbionts during urbanization. Genes, including transporters and antibiotic resistance genes, were enriched in genomes of urban origins, suggesting change in functional potential of the microbiota. These findings suggest the significant influence of urbanization on human gut microbiota and the necessity of exploring the microbial diversity of rural populations.

[Source](https://journals.asm.org/doi/10.1128/msystems.00200-22)

10.1128/msystems.00200-22

---

## A comprehensive analysis of the microbiota composition and host driver gene mutations in colorectal cancer.
 21.06.2022, _D Yuan, Y Tao, H Wang, J Wang, Y Cao, W Cao, S Pan, Z Yu_


Studies of both, microbiota and target therapy associated with gene mutations in colorectal cancer, (CRC) have attracted increasing attention. However, only a few of them analyzed the combined effects on CRC. we analyzed differences in intestinal microbiota of 44 colorectal cancer patients and 20 healthy controls (HC) using 16S rRNA gene sequencing of fecal samples. For 39 of the CRC patients, targeted Next Generation Sequencing (NGS) was carried out at formalin fixed paraffin embedded (FFPE) samples to identify somatic mutation profiles. Compared to the HC group, the microbial diversity of CRC patients was significantly lower. In the CRC group, we found a microbiome that was significantly enriched for strains of Bifidobacterium, Bacteroides, and Megasphaera whereas in the HC group the abundance of Collinsella, Faecalibacterium, and Agathobacter strains was higher. Among the mutations detected in the CRC group, the APC gene had the highest mutation rate (77%, 30/39). We found that the KRAS mutant type was closely associated with Faecalibacterium, Roseburia, Megamonas, Lachnoclostridium, and Harryflintia. Notably, Spearman correlation analysis showed that KRAS mutations were negatively correlated with the existence of Bifidobacterium and positively correlated with Faecalibacterium. By employing 16S rRNA gene sequencing, we identified more unique features of microbiota profiles in CRC patients. For the first time, our study showed that gene mutations could directly be linked to the microbiota composition of CRC patients. We hypothesize that the effect of a targeted colorectal cancer therapy is also closely related to the colorectal flora, however, this requires further investigation.

[Source](https://link.springer.com/article/10.1007/s10637-022-01263-1)

10.1007/s10637-022-01263-1

---

## Vegetarianism, Microbiota and Cardiovascular health: Looking back, and forward.
 21.06.2022, _A Kumar, V Chidambaram, JL Mehta_


Cardiovascular diseases (CVD) are the leading cause of death globally, with over 17.9 million attributed deaths in 2019. Unhealthy diet is an often-overlooked major modifiable risk factor for CVD. Global Burden of Disease (GBD) estimates suggest that unhealthy diets account for nearly 26% of all deaths, of which 84% were attributed to CVD. Plant-based diets (PBDs), which are a diverse group of dietary patterns focused on plant produce, with flexibility for varying levels of vegetarianism, have been suggested to decrease the incidence of various cardiovascular and cardiometabolic diseases. In this review, we aim to delve into the spectrum of PBDs, revisit objective definitions and classifications, and compare them with standard non-vegetarian diets. We examine plausible mechanisms underlying the cardiovascular benefits of PBDs with a particular focus on the dietary manipulation of gut microbiota-host interaction and its effect on energy metabolism, and local and systemic inflammation. In addition, we explore the evidence on the impact of PBDs on cardiovascular disease, examine the challenges and limitations associated with dietary intervention studies, and devise strategies to draw valid conclusions. Dietary interventions, such as PBDs are one of the most powerful, attainable, cost-effective tools for health and environmental protection at the population level. We conclude with a clear appreciation for PBDs in environmental sustainability, climate change, and animal welfare.

[Source](https://academic.oup.com/eurjpc/advance-article-abstract/doi/10.1093/eurjpc/zwac128/6612717)

10.1093/eurjpc/zwac128

---

## Effects of a low FODMAP diet on the colonic microbiome in irritable bowel syndrome: a systematic review with meta-analysis.
 21.06.2022, _D So, A Loughman, HM Staudacher_


A low FODMAP (fermentable oligosaccharides, disaccharides, monosaccharides and polyols) diet is increasingly used to manage symptoms in irritable bowel syndrome (IBS). While this approach may alter the colonic microbiome, the nature of these changes has not been comprehensively synthesized.

[Source](https://academic.oup.com/ajcn/advance-article/doi/10.1093/ajcn/nqac176/6612725)

10.1093/ajcn/nqac176

---

## Multi-Compartment Profiling of Bacterial and Host Metabolites Identifies Intestinal Dysbiosis and Its Functional Consequences in the Critically Ill Child.
 29.01.2022, _A Wijeyesekera, J Wagner, M De Goffau, S Thurston, A Rodrigues Sabino, S Zaher, D White, J Ridout, MJ Peters, P Ramnarayan, RG Branco, ME Torok, F Valla, R Meyer, N Klein, G Frost, J Parkhill, E Holmes, N Pathan_


_Adolescent, Child, Child, Preschool, Chromatography, Liquid, Cresols, Critical Illness, Dysbiosis, Fatty Acids, Volatile, Feces, Female, Formates, Gastrointestinal Microbiome, Hippurates, Humans, Infant, Intensive Care Units, Pediatric, Magnetic Resonance Imaging, Male, Mass Spectrometry, Metabolomics, Prospective Studies, RNA, Ribosomal, 16S, Respiration, Artificial, Severity of Illness Index, Sulfuric Acid Esters, Time Factors, United Kingdom, Urine_
Adverse physiology and antibiotic exposure devastate the intestinal microbiome in critical illness. Time and cost implications limit the immediate clinical potential of microbial sequencing to identify or treat intestinal dysbiosis. Here, we examined whether metabolic profiling is a feasible method of monitoring intestinal dysbiosis in critically ill children.

[Source](https://www.nature.com/articles/s41598-022-13640-0)

10.1097/CCM.0000000000003841

---

## Profiling gut microbiota and bile acid metabolism in critically ill children

[Source](https://www.nature.com/articles/s41598-022-13640-0)

---

## A comprehensive analysis of the microbiota composition and host driver gene mutations in colorectal cancer.
 21.06.2022, _D Yuan, Y Tao, H Wang, J Wang, Y Cao, W Cao, S Pan, Z Yu_


Studies of both, microbiota and target therapy associated with gene mutations in colorectal cancer, (CRC) have attracted increasing attention. However, only a few of them analyzed the combined effects on CRC. we analyzed differences in intestinal microbiota of 44 colorectal cancer patients and 20 healthy controls (HC) using 16S rRNA gene sequencing of fecal samples. For 39 of the CRC patients, targeted Next Generation Sequencing (NGS) was carried out at formalin fixed paraffin embedded (FFPE) samples to identify somatic mutation profiles. Compared to the HC group, the microbial diversity of CRC patients was significantly lower. In the CRC group, we found a microbiome that was significantly enriched for strains of Bifidobacterium, Bacteroides, and Megasphaera whereas in the HC group the abundance of Collinsella, Faecalibacterium, and Agathobacter strains was higher. Among the mutations detected in the CRC group, the APC gene had the highest mutation rate (77%, 30/39). We found that the KRAS mutant type was closely associated with Faecalibacterium, Roseburia, Megamonas, Lachnoclostridium, and Harryflintia. Notably, Spearman correlation analysis showed that KRAS mutations were negatively correlated with the existence of Bifidobacterium and positively correlated with Faecalibacterium. By employing 16S rRNA gene sequencing, we identified more unique features of microbiota profiles in CRC patients. For the first time, our study showed that gene mutations could directly be linked to the microbiota composition of CRC patients. We hypothesize that the effect of a targeted colorectal cancer therapy is also closely related to the colorectal flora, however, this requires further investigation.

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-04786-9)

10.1007/s10637-022-01263-1

---

## Super-taxon in human microbiome are identified to be associated with colorectal cancer

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-04786-9)

---

## The gut metagenomics and metabolomics signature in patients with inflammatory bowel disease

[Source](https://gutpathogens.biomedcentral.com/articles/10.1186/s13099-022-00499-9)

---

## Immune checkpoint inhibitors unleash pathogenic immune responses against the microbiota.
 21.06.2022, _ZI Hu, VM Link, DS Lima-Junior, J Delaleu, N Bouladoux, SJ Han, N Collins, Y Belkaid_


Immune checkpoint inhibitors (ICIs) are essential components of the cancer therapeutic armamentarium. While ICIs have demonstrated remarkable clinical responses, they can be accompanied by immune-related adverse events (irAEs). These inflammatory side effects are of unclear etiology and impact virtually all organ systems, with the most common being sites colonized by the microbiota such as the skin and gastrointestinal tract. Here, we establish a mouse model of commensal bacteria-driven skin irAEs and demonstrate that immune checkpoint inhibition unleashes commensal-specific inflammatory T cell responses. These aberrant responses were dependent on production of IL-17 by commensal-specific T cells and induced pathology that recapitulated the cutaneous inflammation seen in patients treated with ICIs. Importantly, aberrant T cell responses unleashed by ICIs were sufficient to perpetuate inflammatory memory responses to the microbiota months following the cessation of treatment. Altogether, we have established a mouse model of skin irAEs and reveal that ICIs unleash aberrant immune responses against skin commensals, with long-lasting inflammatory consequences.

[Source](https://www.pnas.org/doi/full/10.1073/pnas.2200348119)

10.1073/pnas.2200348119

---

## Deletion of both Dectin-1 and Dectin-2 affects the bacterial but not fungal gut microbiota and susceptibility to colitis in mice

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01273-4)

---

## Fecal microbiome of horses transitioning between warm-season and cool-season grass pasture within integrated rotational grazing systems

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-022-00192-x)

---

## Full-Length 16S rRNA Gene Sequences from Raw Sewage Samples Spanning Geographic and Seasonal Gradients in Conveyance Systems across the United States.
 21.06.2022, _EL LaMartina, AL Schmoldt, RJ Newton_


Wastewater microbiome research often relies on sequencing of hypervariable regions of 16S rRNA genes, which are difficult to classify at refined taxonomic levels. Here, we introduce a data set of near-full-length 16S rRNA genes from samples designed to capture known geographic and seasonal variations in municipal wastewater microbial communities.

[Source](https://journals.asm.org/doi/10.1128/mra.00319-22)

10.1128/mra.00319-22

---

## SRS-FISH: A high-throughput platform linking microbiome metabolism to identity at the single-cell level.
 21.06.2022, _X Ge, FC Pereira, M Mitteregger, D Berry, M Zhang, B Hausmann, J Zhang, A Schintlmeister, M Wagner, JX Cheng_


One of the biggest challenges in microbiome research in environmental and medical samples is to better understand functional properties of microbial community members at a single-cell level. Single-cell isotope probing has become a key tool for this purpose, but the current detection methods for determination of isotope incorporation into single cells do not allow high-throughput analyses. Here, we report on the development of an imaging-based approach termed stimulated Raman scattering-two-photon fluorescence in situ hybridization (SRS-FISH) for high-throughput metabolism and identity analyses of microbial communities with single-cell resolution. SRS-FISH offers an imaging speed of 10 to 100 ms per cell, which is two to three orders of magnitude faster than achievable by state-of-the-art methods. Using this technique, we delineated metabolic responses of 30,000 individual cells to various mucosal sugars in the human gut microbiome via incorporation of deuterium from heavy water as an activity marker. Application of SRS-FISH to investigate the utilization of host-derived nutrients by two major human gut microbiome taxa revealed that response to mucosal sugars tends to be dominated by Bacteroidales, with an unexpected finding that Clostridia can outperform Bacteroidales at foraging fucose. With high sensitivity and speed, SRS-FISH will enable researchers to probe the fine-scale temporal, spatial, and individual activity patterns of microbial cells in complex communities with unprecedented detail.

[Source](https://www.pnas.org/doi/full/10.1073/pnas.2203519119)

10.1073/pnas.2203519119

---

