## Crohn's disease in endoscopic remission, obesity, and cases of high genetic risk demonstrates overlapping shifts in the colonic mucosal-luminal interface microbiome.
 15.08.2022, _JP Jacobs, M Goudarzi, V Lagishetty, D Li, T Mak, M Tong, P Ruegger, T Haritunians, C Landers, P Fleshner, E Vasiliauskas, A Ippoliti, G Melmed, D Shih, S Targan, J Borneman, AJ Fornace, DPB McGovern, J Braun_


Crohn's disease (CD) patients demonstrate distinct intestinal microbial compositions and metabolic characteristics compared to unaffected controls. However, the impact of inflammation and underlying genetic risk on these microbial profiles and their relationship to disease phenotype are unclear. We used lavage sampling to characterize the colonic mucosal-luminal interface (MLI) microbiome of CD patients in endoscopic remission and unaffected controls relative to obesity, disease genetics, and phenotype.

[Source](https://doi.org/10.1186/s13073-022-01099-7)

10.1186/s13073-022-01099-7

---

## Preservation of fecal microbiome is associated with reduced severity of Graft-versus-Host Disease.
 15.08.2022, _M Burgos da Silva, DM Ponce, A Dai, SM Devlin, AL Gomes, GF Moore, J Slingerland, R Shouval, GK Armijo, S DeWolf, T Fei, A Clurman, E Fontana, LA Amoretti, RJ Wright, H Andrlova, O Miltiadous, MA Perales, Y Taur, JU Peled, MRM van den Brink_


Following allogeneic hematopoietic cell transplantation (allo-HCT), the gastrointestinal (GI) tract is frequently affected by acute graft-versus-host disease (aGVHD), the pathophysiology of which is associated with a dysbiotic microbiome. Since microbial composition varies along the length of the GI tract, we hypothesized that microbiome features correlate with the pattern of organ involvement after allo-HCT. We evaluated 266 allo-HCT recipients from whom 1,303 stool samples were profiled by 16S ribosomal gene sequencing. Patients were classified according to which organs were affected by aGVHD. In the 20 days prior to disease onset, GVHD patients had lower abundances of members of the class Clostridia, lower counts of butyrate producers, and lower ratios of strict-to-facultative anaerobic bacteria (S/F anaerobe ratio) compared to allograft recipients who were free of GVHD. GI-GVHD patients showed significant reduction in microbial diversity pre-onset. Patients with lower GI-aGVHD had lower S/F anaerobe ratios compared to those with isolated upper GI-aGVHD. In the 20 days after disease onset, dysbiosis was observed only in GVHD patients with GI involvement, particularly those with lower-tract disease. Importantly, Clostridial and butyrate-producer abundance, as well as S/F anaerobe ratio were predictors of longer overall survival; higher abundance of butyrate producers, and higher S/F anaerobe ratio were associated with decreased risk of GVHD-related death. These findings suggest that the intestinal microbiome can serve as a biomarker for outcomes of allo-HCT patients with GVHD.

[Source](https://doi.org/10.1182/blood.2021015352)

10.1182/blood.2021015352

---

## Cobamide Sharing Is Predicted in the Human Skin Microbiome.
 15.08.2022, _MH Swaney, S Sandstrom, LR Kalan_


The skin microbiome is a key player in human health, with diverse functions ranging from defense against pathogens to education of the immune system. While recent studies have begun to shed light on the valuable role that skin microorganisms have in maintaining the skin barrier, a detailed understanding of the complex interactions that shape healthy skin microbial communities is limited. Cobamides, the vitamin B12 class of cofactor, are essential for organisms across the tree of life. Because this vitamin is only produced by a limited fraction of prokaryotes, cobamide sharing is predicted to mediate community dynamics within microbial communities. Here, we provide the first large-scale metagenomic assessment of cobamide biosynthesis and utilization in the skin microbiome. We show that while numerous and diverse taxa across the major bacterial phyla on the skin encode cobamide-dependent enzymes, relatively few species encode de novo cobamide biosynthesis. We show that cobamide producers and users are integrated into the network structure of microbial communities across the different microenvironments of the skin and that changes in microbiome community structure and diversity are associated with the abundance of cobamide producers in the Corynebacterium genus, for both healthy and diseased skin states. Finally, we find that de novo cobamide biosynthesis is enriched only in Corynebacterium species associated with hosts, including those prevalent on human skin. We confirm that the cofactor is produced in excess through quantification of cobamide production by human skin-associated species isolated in the laboratory. Taken together, our results reveal the potential for cobamide sharing within skin microbial communities, which we hypothesize mediates microbiome community dynamics and host interactions. IMPORTANCE The skin microbiome is essential for maintaining skin health and function. However, the microbial interactions that dictate microbiome structure, stability, and function are not well understood. Here, we investigate the biosynthesis and use of cobamides, a cofactor needed by many organisms but only produced by select prokaryotes, within the human skin microbiome. We found that while a large proportion of skin taxa encode cobamide-dependent enzymes, only a select few encode de novo cobamide biosynthesis. Further, the abundance of cobamide-producing Corynebacterium species is associated with skin microbiome diversity and structure, and within this genus, de novo biosynthesis is enriched in host-associated species compared to environment-associated species. These findings identify cobamides as a potential mediator of skin microbiome dynamics and skin health.

[Source](https://doi.org/10.1128/msystems.00677-22)

10.1128/msystems.00677-22

---

## Postnatal exposure to ambient air pollutants is associated with the composition of the infant gut microbiota at 6-months of age.
 16.08.2022, _MJ Bailey, EA Holzhausen, ZEM Morgan, N Naik, JP Shaffer, D Liang, HH Chang, J Sarnat, S Sun, PK Berger, KA Schmidt, F Lurmann, MI Goran, TL Alderete_


_Adult, Air Pollutants, Environmental Pollutants, Gastrointestinal Microbiome, Humans, Infant, Nitrogen Dioxide, Particulate Matter, RNA, Ribosomal, 16S_
Epidemiological studies in adults have shown that exposure to ambient air pollution (AAP) is associated with the composition of the adult gut microbiome, but these relationships have not been examined in infancy. We aimed to determine if 6-month postnatal AAP exposure was associated with the infant gut microbiota at 6 months of age in a cohort of Latino mother-infant dyads from the Southern California Mother's Milk Study (n = 103). We estimated particulate matter (PM2.5 and PM10) and nitrogen dioxide (NO2) exposure from birth to 6-months based on residential address histories. We characterized the infant gut microbiota using 16S rRNA amplicon sequencing at 6-months of age. At 6-months, the gut microbiota was dominated by the phyla Bacteroidetes, Firmicutes, Proteobacteria, and Actinobacteria. Our results show that, after adjusting for important confounders, postnatal AAP exposure was associated with the composition of the gut microbiota. As an example, PM10 exposure was positively associated with Dialister, Dorea, Acinetobacter, and Campylobacter while PM2.5 was positively associated with Actinomyces. Further, exposure to PM10 and PM2.5 was inversely associated with Alistipes and NO2 exposure was positively associated with Actinomyces, Enterococcus, Clostridium, and Eubacterium. Several of these taxa have previously been linked with systemic inflammation, including the genera Dialister and Dorea. This study provides the first evidence of significant associations between exposure to AAP and the composition of the infant gut microbiota, which may have important implications for future infant health and development.

[Source](https://doi.org/10.1080/19490976.2022.2105096)

10.1080/19490976.2022.2105096

---

## Shotgun metagenomic sequencing reveals skin microbial variability from different facial sites.
 16.08.2022, _Q Wei, Z Li, Z Gu, X Liu, J Krutmann, J Wang, J Xia_


Biogeography (body site) is known to be one of the main factors influencing the composition of the skin microbial community. However, site-associated microbial variability at a fine-scale level was not well-characterized since there was a lack of high-resolution recognition of facial microbiota across kingdoms by shotgun metagenomic sequencing. To investigate the explicit microbial variance in the human face, 822 shotgun metagenomic sequencing data from Han Chinese recently published by our group, in combination with 97 North American samples from NIH Human Microbiome Project (HMP), were reassessed. Metagenomic profiling of bacteria, fungi, and bacteriophages, as well as enriched function modules from three facial sites (forehead, cheek, and the back of the nose), was analyzed. The results revealed that skin microbial features were more alike in the forehead and cheek while varied from the back of the nose in terms of taxonomy and functionality. Analysis based on biogeographic theories suggested that neutral drift with niche selection from the host could possibly give rise to the variations. Of note, the abundance of porphyrin-producing species, i.e., Cutibacterium acnes, Cutibacterium avidum, Cutibacterium granulosum, and Cutibacterium namnetense, was all the highest in the back of the nose compared with the forehead/cheek, which was consistent with the highest porphyrin level on the nose in our population. Sequentially, the site-associated microbiome variance was confirmed in American populations; however, it was not entirely consistent. Furthermore, our data revealed correlation patterns between Propionibacterium acnes bacteriophages with genus Cutibacterium at different facial sites in both populations; however, C. acnes exhibited a distinct correlation with P. acnes bacteriophages in Americans/Chinese. Taken together, in this study, we explored the fine-scale facial site-associated changes in the skin microbiome and provided insight into the ecological processes underlying facial microbial variations.

[Source](https://doi.org/10.3389/fmicb.2022.933189)

10.3389/fmicb.2022.933189

---

## Recent insights into the role of microbiome in the pathogenesis of obesity.
 16.08.2022, _EWJ van der Vossen, MC de Goffau, E Levin, M Nieuwdorp_


Obesity is a risk factor for many chronic diseases and its rising prevalence the last couple of decades is a healthcare concern in many countries. Obesity is a multifactorial problem that is not only limited in its causation by diet and lack of exercise. Genetics but also environmental factors such as the gut microbiome should similarly be taken into account. A plethora of articles have been published, that from various different angles, attempt to disentangle the complex interaction between gut microbiota and obesity. Examples range from the effect of the gut microbiota on the host immune system to the pathophysiological pathways in which microbial-derived metabolites affect obesity. Various discordant gut microbiota findings are a result of this complexity. In this review, in addition to summarizing the classical role of the gut microbiome in the pathogenesis of obesity, we attempt to view both the healthy and obesogenic effects of the gut microbiota as a consequence of the presence or absence of collective guilds/trophic networks. Lastly, we propose avenues and strategies for the future of gut microbiome research concerning obesity.

[Source](https://doi.org/10.1177/17562848221115320)

10.1177/17562848221115320

---

## Impact of habitual chewing on gut motility via microbiota transition.
 15.08.2022, _F Yaoita, K Watanabe, I Kimura, M Miyazawa, S Tsuchiya, M Kanzaki, M Tsuchiya, K Tan-No_


The gut environment, including the microbiota and its metabolites and short-chain fatty acids (SCFA), is essential for health maintenance. It is considered that functional recovery treatment for masticatory dysphagia affects the composition of the gut microbiota, indicating that habitual mastication, depending on the hardness of the food, may affect the gut microbiota and environment. However, the impact of chronic powdered diet feeding on the colonic condition and motility remains unclear. Here, we evaluated various colonic features in mice fed with powdered diets for a long-term and a mouse model with masticatory behavior. We observed a decreased abundance of the SCFA-producing bacterial genera in the ceca of the powdered diet-fed mice. Based on the importance of SCFAs in gut immune homeostasis and motility, interestingly, powdered diet feeding also resulted in constipation-like symptoms due to mild colitis, which were ameliorated by the administration of a neutrophil-depleting agent and neutrophil elastase inhibitors. Lastly, the suppressed colonic motility in the powdered diet-fed mice was significantly improved by loading masticatory activity for 2 h. Thus, feeding habits with appropriate masticatory activity and stimulation may play a key role in providing a favorable gut environment based on interactions between the gut microbiota and host immune system.

[Source](https://doi.org/10.1038/s41598-022-18095-x)

10.1038/s41598-022-18095-x

---

## Resources to Facilitate Use of the Altered Schaedler Flora (ASF) Mouse Model to Study Microbiome Function.
 15.08.2022, _A Proctor, S Parvinroo, T Richie, X Jia, STM Lee, PD Karp, S Paley, AD Kostic, JF Pierre, MJ Wannemuehler, GJ Phillips_


Animals colonized with a defined microbiota represent useful experimental systems to investigate microbiome function. The altered Schaedler flora (ASF) represents a consortium of eight murine bacterial species that have been used for more than 4 decades where the study of mice with a reduced microbiota is desired. In contrast to germ-free mice, or mice colonized with only one or two species, ASF mice show the normal gut structure and immune system development. To further expand the utility of the ASF, we have developed technical and bioinformatic resources to enable a systems-based analysis of microbiome function using this model. Here, we highlighted four distinct applications of these resources that enable and improve (i) measurements of the abundance of each ASF member by quantitative PCR; (ii) exploration and comparative analysis of ASF genomes and the metabolic pathways they encode that comprise the entire gut microbiome; (iii) global transcriptional profiling to identify genes whose expression responds to environmental changes within the gut; and (iv) discovery of genetic changes resulting from the evolutionary adaptation of the microbiota. These resources were designed to be accessible to a broad community of researchers that, in combination with conventionally-reared mice (i.e., with complex microbiome), should contribute to our understanding of microbiome structure and function. IMPORTANCE Improved experimental systems are needed to advance our understanding of how the gut microbiome influences processes of the mammalian host as well as microbial community structure and function. An approach that is receiving considerable attention is the use of animal models that harbor a stable microbiota of known composition, i.e., defined microbiota, which enables control over an otherwise highly complex and variable feature of mammalian biology. The altered Schaedler flora (ASF) consortium is a well-established defined microbiota model, where mice are stably colonized with 8 distinct murine bacterial species. To take better advantage of the ASF, we established new experimental and bioinformatics resources for researchers to make better use of this model as an experimental system to study microbiome function.

[Source](https://doi.org/10.1128/msystems.00293-22)

10.1128/msystems.00293-22

---

## The microbiome of a bacterivorous marine choanoflagellate contains a resource-demanding obligate bacterial associate.
 15.08.2022, _DM Needham, C Poirier, C Bachy, EE George, S Wilken, CCM Yung, AJ Limardo, M Morando, L Sudek, RR Malmstrom, PJ Keeling, AE Santoro, AZ Worden_


Microbial predators such as choanoflagellates are key players in ocean food webs. Choanoflagellates, which are the closest unicellular relatives of animals, consume bacteria and also exhibit marked biological transitions triggered by bacterial compounds, yet their native microbiomes remain uncharacterized. Here we report the discovery of a ubiquitous, uncultured bacterial lineage we name Candidatus Comchoanobacterales ord. nov., related to the human pathogen Coxiella and physically associated with the uncultured marine choanoflagellate Bicosta minor. We analyse complete 'Comchoano' genomes acquired after sorting single Bicosta cells, finding signatures of obligate host-dependence, including reduction of pathways encoding glycolysis, membrane components, amino acids and B-vitamins. Comchoano encode the necessary apparatus to import energy and other compounds from the host, proteins for host-cell associations and a type IV secretion system closest to Coxiella's that is expressed in Pacific Ocean metatranscriptomes. Interactions between choanoflagellates and their microbiota could reshape the direction of energy and resource flow attributed to microbial predators, adding complexity and nuance to marine food webs.

[Source](https://doi.org/10.1038/s41564-022-01174-0)

10.1038/s41564-022-01174-0

---

## Characterization of presence and activity of microRNAs in the rumen of cattle hints at possible host-microbiota cross-talk mechanism.
 15.08.2022, _S Ricci, RM Petri, C Pacífico, E Castillo-Lopez, R Rivera-Chacon, A Sener-Aydemir, N Reisinger, Q Zebeli, S Kreuzer-Redmer_


MicroRNAs (miRNAs), as important post-transcriptional regulators, are ubiquitous in various tissues. The aim of this exploratory study was to determine the presence of miRNAs in rumen fluid, and to investigate the possibility of miRNA-mediated cross-talk within the ruminal ecosystem. Rumen fluid samples from four cannulated Holstein cows were collected during two feeding regimes (forage and high-grain diet) and DNA and RNA were extracted for amplicon and small RNA sequencing. Epithelial biopsies were simultaneously collected to investigate the co-expression of miRNAs in papillae and rumen fluid. We identified 377 miRNAs in rumen fluid and 638 in rumen papillae, of which 373 were shared. Analysis of microbiota revealed 20 genera to be differentially abundant between the two feeding regimes, whereas no difference in miRNAs expression was detected. Correlations with at least one genus were found for 170 miRNAs, of which, 39 were highly significant (r &gt; |0.7| and P &lt; 0.01). Both hierarchical clustering of the correlation matrix and WGCNA analysis identified two main miRNA groups. Putative target and functional prediction analysis for the two groups revealed shared pathways with the predicted metabolic activities of the microbiota. Hence, our study supports the hypothesis of a cross-talk within the rumen at least partly mediated by miRNAs.

[Source](https://doi.org/10.1038/s41598-022-17445-z)

10.1038/s41598-022-17445-z

---

## Full-length 16S rRNA amplicon sequencing reveals the variation of epibiotic microbiota associated with two shrimp species of Alvinocarididae: possibly co-determined by environmental heterogeneity and specific recognition of hosts.
 16.08.2022, _M Hui, A Wang, J Cheng, Z Sha_


Shrimps of the family Alvinocarididae, endemic species to deep sea chemosynthetic ecosystems, harbor epibiotic microbes on gills which probably play important roles in the survival of the shrimps. Among them, Alvinocaris longirostris and Shinkaicaris leurokolos occupy different ecological niches within the same hydrothermal vent in Okinawa Trough, and A. longirostris also exists in a methane seep of the South China Sea. In this study, full-length 16S rRNA sequences of the gill associated bacteria of two alvinocaridid species from different chemosynthetically ecological niches were first captured by single-molecule real-time sequencing. Totally, 120,792 optimized circular consensus sequences with ∼1,450 bp in length were obtained and clustered into 578 operational taxonomic units. Alpha diversity analysis showed seep A. longirostris had the highest species richness and evenness (average Chao1 = 213.68, Shannon = 3.39). Beta diversity analysis revealed that all samples were clearly divided into three groups, and microbial community of A. longirostris from seep and vent were more related than the other comparisons. By permutational multivariate analysis of variance, the most significant community compositional variance was detected between seep A. longirostris and vent S. leurokolos (R 2 = 0.731, P = 0.001). The taxon tags were further classified into 21 phyla, 40 classes, 89 orders, 124 families and 135 genera. Overall, the microbial communities were dominated by Campylobacteria and Gammaproteobacteria. Alphaproteobacteria, Bacteroidia, Verrucomicrobiae, Bacilli and other minor groups were also detected at lower abundance. Taxonomic groups recovered from the vent S. leurokolos samples were only dominated by Sulfurovaceae (94.06%). In comparison, gill-associated microbiota of vent A. longirostris consisted of more diverse sulfur-oxidizing bacteria, including Sulfurovaceae (69.21%), Thiotrichaceae (6.77%) and a putative novel Gammaproteobacteria group (14.37%), while in seep A. longirostris, Gammaproteobacteria un-group (44.01%) constituted the major component, following the methane-oxidizing bacteria Methylomonadaceae (19.38%), and Sulfurovaceae (18.66%). Therefore, the gill associated bacteria composition and abundance of alvinocaridid shrimps are closely related to the habitat heterogeneity and the selection of microbiota by the host. However, the interaction between these alvinocaridid shrimps and the epibiotic communities requires further study based on metagenome sequencing and fluorescence in situ hybridization.

[Source](https://doi.org/10.7717/peerj.13758)

10.7717/peerj.13758

---

## Systematic profiling of the chicken gut microbiome reveals dietary supplementation with antibiotics alters expression of multiple microbial pathways with minimal impact on community structure.
 16.08.2022, _A Zou, K Nadeau, X Xiong, PW Wang, JK Copeland, JY Lee, JS Pierre, M Ty, B Taj, JH Brumell, DS Guttman, S Sharif, D Korver, J Parkinson_


_Animals, Anti-Bacterial Agents, Chickens, DNA, Ribosomal, Dietary Supplements, Gastrointestinal Microbiome_
The emergence of antimicrobial resistance is a major threat to global health and has placed pressure on the livestock industry to eliminate the use of antibiotic growth promotants (AGPs) as feed additives. To mitigate their removal, efficacious alternatives are required. AGPs are thought to operate through modulating the gut microbiome to limit opportunities for colonization by pathogens, increase nutrient utilization, and reduce inflammation. However, little is known concerning the underlying mechanisms. Previous studies investigating the effects of AGPs on the poultry gut microbiome have largely focused on 16S rDNA surveys based on a single gastrointestinal (GI) site, diet, and/or timepoint, resulting in an inconsistent view of their impact on community composition.

[Source](https://doi.org/10.1186/s40168-022-01319-7)

10.1186/s40168-022-01319-7

---

## Analysis of the anaerobic digestion metagenome under environmental stresses stimulating prophage induction.
 16.08.2022, _A Rossi, MS Morlino, M Gaspari, A Basile, P Kougias, L Treu, S Campanaro_


_Anaerobiosis, Archaea, Metagenome, Microbiota, Virus Activation, Viruses_
The viral community has the potential to influence the structure of the microbiome and thus the yield of the anaerobic digestion process. However, the virome composition in anaerobic digestion is still under-investigated. A viral induction experiment was conducted on separate batches undergoing a series of DNA-damaging stresses, in order to coerce temperate viruses to enter the lytic cycle.

[Source](https://doi.org/10.1186/s40168-022-01316-w)

10.1186/s40168-022-01316-w

---

## Microbiome and Metabiotic Properties of Kefir Grains and Kefirs Based on Them

[Source](https://doi.org/10.1134/S0026261722100885)

---

## Profiling the role of microorganisms in quality improvement of the aged flue-cured tobacco

[Source](https://doi.org/10.1186/s12866-022-02597-9)

---

