## Clinical medicine journals lag behind science journals with regards to "microbiota sequence" data availability.
 07.12.2021, _J Pryor, GD Eslick, NJ Talley, K Duncanson, S Keely, EC Hoedt_



[Source](https://onlinelibrary.wiley.com/doi/10.1002/ctm2.656)

10.1002/ctm2.656

---

## Role of the Microbiome in Regulating Bone Metabolism and Susceptibility to Osteoporosis.
 06.12.2021, _O Cronin, SA Lanham-New, BM Corfe, CL Gregson, AL Darling, KR Ahmadi, PS Gibson, JH Tobias, KA Ward, MH Traka, M Rossi, C Williams, NC Harvey, C Cooper, K Whelan, AG Uitterlinden, PW O'Toole, C Ohlsson, JE Compston, SH Ralston_


The human microbiota functions at the interface between diet, medication-use, lifestyle, host immune development and health. It is therefore closely aligned with many of the recognised modifiable factors that influence bone mass accrual in the young, and bone maintenance and skeletal decline in older populations. While understanding of the relationship between micro-organisms and bone health is still in its infancy, two decades of broader microbiome research and discovery supports a role of the human gut microbiome in the regulation of bone metabolism and pathogenesis of osteoporosis as well as its prevention and treatment. Pre-clinical research has demonstrated biological interactions between the microbiome and bone metabolism. Furthermore, observational studies and randomized clinical trials have indicated that therapeutic manipulation of the microbiota by oral administration of probiotics may influence bone turnover and prevent bone loss in humans. In this paper, we summarize the content, discussion and conclusions of a workshop held by the Osteoporosis and Bone Research Academy of the Royal Osteoporosis Society in October, 2020. We provide a detailed review of the literature examining the relationship between the microbiota and bone health in animal models and in humans, as well as formulating the agenda for key research priorities required to advance this field. We also underscore the potential pitfalls in this research field that should be avoided and provide methodological recommendations to facilitate bridging the gap from promising concept to a potential cause and intervention target for osteoporosis.

[Source](https://link.springer.com/article/10.1007%2Fs00223-021-00924-2)

10.1007/s00223-021-00924-2

---

## Defining and quantifying the core microbiome: Challenges and prospects.
 06.12.2021, _AT Neu, EE Allen, K Roy_


The term "core microbiome" has become widely used in microbial ecology over the last decade. Broadly, the core microbiome refers to any set of microbial taxa, or the genomic and functional attributes associated with those taxa, that are characteristic of a host or environment of interest. Most commonly, core microbiomes are measured as the microbial taxa shared among two or more samples from a particular host or environment. Despite the popularity of this term and its growing use, there is little consensus about how a core microbiome should be quantified in practice. Here, we present a brief history of the core microbiome concept and use a representative sample of the literature to review the different metrics commonly used for quantifying the core. Empirical analyses have used a wide range of metrics for quantifying the core microbiome, including arbitrary occurrence and abundance cutoff values, with the focal taxonomic level of the core ranging from phyla to amplicon sequence variants. However, many of these metrics are susceptible to sampling and other biases. Developing a standardized set of metrics for quantifying the core that accounts for such biases is necessary for testing specific hypotheses about the functional and ecological roles of core microbiomes.

[Source](https://www.pnas.org/content/118/51/e2104429118)

10.1073/pnas.2104429118

---

## Clean room microbiome complexity impacts planetary protection bioburden.
 07.12.2021, _R Hendrickson, C Urbaniak, JJ Minich, HS Aronson, C Martino, R Stepanauskas, R Knight, K Venkateswaran_


The Spacecraft Assembly Facility (SAF) at the NASA's Jet Propulsion Laboratory is the primary cleanroom facility used in the construction of some of the planetary protection (PP)-sensitive missions developed by NASA, including the Mars 2020 Perseverance Rover that launched in July 2020. SAF floor samples (n=98) were collected, over a 6-month period in 2016 prior to the construction of the Mars rover subsystems, to better understand the temporal and spatial distribution of bacterial populations (total, viable, cultivable, and spore) in this unique cleanroom.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01159-x)

10.1186/s40168-021-01159-x

---

## Community Composition of Bacteria Isolated from Swiss Banknotes Varies Depending on Collection Environment

[Source](https://www.biorxiv.org/content/10.1101/2021.12.01.470750v2.abstract)

---

## Microbiota and stress: a loop that impacts memory

[Source](https://www.sciencedirect.com/science/article/pii/S0306453021004686)

---

## Species- and site-specific genome editing in complex bacterial communities

[Source](https://www.nature.com/articles/s41564-021-01014-7)

---

## Integrated metabolic and microbial analysis reveals host-microbial interactions in IgE-mediated childhood asthma.
 07.12.2021, _CY Chiu, ML Cheng, MH Chiang, CJ Wang, MH Tsai, G Lin_


A metabolomics-based approach to address the molecular mechanism of childhood asthma with immunoglobulin E (IgE) or allergen sensitization related to microbiome in the airways remains lacking. Fifty-three children with lowly sensitized non-atopic asthma (n = 15), highly sensitized atopic asthma (n = 13), and healthy controls (n = 25) were enrolled. Blood metabolomic analysis with 1H-nuclear magnetic resonance (NMR) spectroscopy and airway microbiome composition analysis by bacterial 16S rRNA sequencing were performed. An integrative analysis of their associations with allergen-specific IgE levels for lowly and highly sensitized asthma was also assessed. Four metabolites including tyrosine, isovalerate, glycine, and histidine were uniquely associated with lowly sensitized asthma, whereas one metabolite, acetic acid, was strongly associated with highly sensitized asthma. Metabolites associated with highly sensitized asthma (valine, isobutyric acid, and acetic acid) and lowly sensitized asthma (isovalerate, tyrosine, and histidine) were strongly correlated each other (P &lt; 0.01). Highly sensitized asthma associated metabolites were mainly enriched in pyruvate and acetyl-CoA metabolisms. Metabolites associated with highly sensitized atopic asthma were mostly correlated with microbiota in the airways. Acetic acid, a short-chain fatty acid (SCFA), was negatively correlated with the genus Atopobium (P &lt; 0.01), but positively correlated with the genus Fusobacterium (P &lt; 0.05). In conclusion, metabolomics reveals microbes-related metabolic pathways associated with IgE responses to house dust mite allergens in childhood asthma. A strong correlation of metabolites related to highly sensitized atopic asthma with airway microbiota provides linkages between the host-microbial interactions and asthma endotypes.

[Source](https://www.nature.com/articles/s41598-021-02925-5)

10.1038/s41598-021-02925-5

---

## Longitudinal Study of Oral Microbiome Variation in Twins.
 14.05.2021, _M Freire, A Moustafa, DM Harkins, MG Torralba, Y Zhang, P Leong, R Saffery, M Bockmann, C Kuelbs, T Hughes, JM Craig, KE Nelson_


_Aging, Child, Female, Habits, Humans, Longitudinal Studies, Male, Microbiota, Mouth, Oral Hygiene, Twins_
Humans are host to a multitude of microorganisms that rapidly populate the body at birth, subject to a complex interplay that is dependent on host genetics, lifestyle, and environment. The host-associated microbiome, including the oral microbiome, presents itself in a complex ecosystem important to health and disease. As the most common chronic disease globally, dental caries is induced by host-microbial dysbiosis in children and adults. Multiple biological and environmental factors are likely to impact disease predisposition, onset, progression, and severity, yet longitudinal studies able to capture these influences are missing. To investigate how host genetics and environment influenced the oral microbial communities over time, we profiled supragingival plaque microbiomes of dizygotic and monozygotic twins during 3 visits over 12-months. Dental plaque DNA samples were amplified by targeting the 16S rRNA gene V4 region, and microbial findings were correlated with clinical, diet and genetic metadata. We observed that the oral microbiome variances were shaped primarily by the environment when compared to host genetics. Among the environmental factors shaping microbial changes of our subjects, significant metadata included age of the subject, and the age by which subjects initiated brushing habits, and the types of actions post-brushing. Relevant heritability of the microbiome included Actinomyces and Capnocytophaga in monozygotic twins and Kingella in dizygotic twins. Corynebacterium and Veillonella abundances were associated with age, whereas Aggregatibacter was associated with younger subjects. Streptococcus abundance showed an inverse association over time, and Selenomonas abundances increased with brushing frequency per day. Unraveling the exact biological mechanisms in caries has the potential to reveal novel host-microbial biomarkers, pathways, and targets important to effective preventive measures, and early disease control in children.

[Source](https://www.nature.com/articles/s41421-021-00356-0)

10.1038/s41598-020-64747-1

---

## Metagenome-genome-wide association studies reveal human genetic impact on the oral microbiome

[Source](https://www.nature.com/articles/s41421-021-00356-0)

---

## Interaction Between Intestinal Microbiota, Immunology Molecular Mimicry, and AID

[Source](https://www.hindawi.com/journals/jir/2021/5578958/)

---

## The Gut Microbiome and Hepatocellular Carcinoma.
 06.12.2021, _H Akkız_


The microbiome modulates key processes in metabolism, inflammation, and immunity and plays pivotal roles in many gastrointestinal and liver diseases. Recent experimental studies have demonstrated a key role of the microbiome in hepatocarcinogenesis. Dysfunctions of the gut bacterial flora have a significant effect on liver disease. Dysbiosis is found to be associated with chronic liver diseases. Hepatocellular carcinoma (HCC) is the sixth most common cancer and the third leading cause of cancer-related mortality. The majority of HCC develops in patients with chronic liver disease, caused by chronic viral hepatitis, nonalcoholic fatty liver disease (NAFLD), and alcohol-related fatty liver disease. This review discusses molecular mechanisms of gut microbiome-related hepatocarcinogenesis and the impact of dysbiosis on chronic liver disease progression.

[Source](https://link.springer.com/article/10.1007%2Fs12029-021-00748-1)

10.1007/s12029-021-00748-1

---

## Anaerostipes hominis sp. nov., a novel butyrate-producing bacteria isolated from faeces of a patient with Crohn's disease.
 07.12.2021, _JY Lee, W Kang, NR Shin, DW Hyun, PS Kim, HS Kim, JY Lee, EJ Tak, H Sung, JW Bae_


Cultivation and isolation of gut bacteria are necessary for understanding their role in the intestinal ecosystem. We isolated a novel bacterium, designated strain BG01T, from the faeces of a patient with Crohn's disease. Strain BG01T was a strictly anaerobic, rod-shaped, Gram-variable and endospore-forming bacterium. Strain BG01T possessed C12 : 0, C18 : 0 dimethyl aldehyde (DMA) and C18 : 1  ω9c DMA as predominant cellular fatty acids and meso-diaminopimelic acid as a diagnostic diamino acid. Strain BG01T grew at 15-45 °C (optimum, 37 °C), with 0-4 % (w/v) NaCl (optimum, 0-1 %), at pH 6-10 (optimum, pH 7) and was resistant to bile salt, but not to ampicillin, metronidazole, vancomycin and cefoperazone. Butyrate, propionate, oxalacetate and fumarate were produced as fermentation end products from Gifu anaerobic medium broth. Strain BG01T showed 97.7 % 16S rRNA gene sequence similarity, and 92.0 and 48.5 % of average nucleotide identity and digital DNA-DNA hybridization values, respectively, with Anaerostipes caccae KCTC 15019T. Genomic analysis indicated that strain BG01T had a butyrate-producing pathway. The genomic G+C content of the strain was 43.5 mol%. Results of the phenotypic, phylogenetic and genotypic analyses indicated that strain BG01T represents a novel butyrate-producing species of the genus Anaerostipes, for which the name Anaerostipes hominis sp. nov. is proposed. The type strain is BG01T (=KCTC 15617T=JCM 32275T).

[Source](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.005129)

10.1099/ijsem.0.005129

---

## Gut microbiome is associated with the clinical response to anti-PD-1 based immunotherapy in hepatobiliary cancers

[Source](https://jitc.bmj.com/content/9/12/e003334)

---

## Time-restricted feeding induces Lactobacillus- and Akkermansia-specific functional changes in the rat fecal microbiota.
 06.12.2021, _A Palomba, A Tanca, M Abbondio, R Sau, M Serra, F Marongiu, C Fraumene, D Pagnozzi, E Laconi, S Uzzau_


Diet is a key factor influencing gut microbiota (GM) composition and functions, which in turn affect host health. Among dietary regimens, time-restricted (TR) feeding has been associated to numerous health benefits. The impact of TR feeding on the GM composition has been mostly explored by means of metagenomic sequencing. To date, however, little is known about the modulation of GM functions by this dietary regimen. Here, we analyzed the effects of TR feeding on GM functions by evaluating protein expression changes in a rat model through a metaproteomic approach. We observed that TR feeding has a relevant impact on GM functions, specifically leading to an increased abundance of several enzymes involved in carbohydrate and protein metabolism and expressed by Lactobacillus spp. and Akkermansia muciniphila. Taken together, these results contribute to deepening our knowledge about the key relationship between diet, GM, and health.

[Source](https://www.nature.com/articles/s41522-021-00256-x)

10.1038/s41522-021-00256-x

---

## Diet-driven mercury contamination is associated with polar bear gut microbiota.
 07.12.2021, _SE Watson, MA McKinney, M Pindo, MJ Bull, TC Atwood, HC Hauffe, SE Perkins_


The gut microbiota may modulate the disposition and toxicity of environmental contaminants within a host but, conversely, contaminants may also impact gut bacteria. Such contaminant-gut microbial connections, which could lead to alteration of host health, remain poorly known and are rarely studied in free-ranging wildlife. The polar bear (Ursus maritimus) is a long-lived, wide-ranging apex predator that feeds on a variety of high trophic position seal and cetacean species and, as such, is exposed to among the highest levels of biomagnifying contaminants of all Arctic species. Here, we investigate associations between mercury (THg; a key Arctic contaminant), diet, and the diversity and composition of the gut microbiota of polar bears inhabiting the southern Beaufort Sea, while accounting for host sex, age class and body condition. Bacterial diversity was negatively associated with seal consumption and mercury, a pattern seen for both Shannon and Inverse Simpson alpha diversity indices (adjusted R2 = 0.35, F1,18 = 8.00, P = 0.013 and adjusted R2 = 0.26, F1,18 = 6.04, P = 0.027, respectively). No association was found with sex, age class or body condition of polar bears. Bacteria known to either be involved in THg methylation or considered to be highly contaminant resistant, including Lactobacillales, Bacillales and Aeromonadales, were significantly more abundant in individuals that had higher THg concentrations. Conversely, individuals with higher THg concentrations showed a significantly lower abundance of Bacteroidales, a bacterial order that typically plays an important role in supporting host immune function by stimulating intraepithelial lymphocytes within the epithelial barrier. These associations between diet-acquired mercury and microbiota illustrate a potentially overlooked outcome of mercury accumulation in polar bears.

[Source](https://www.nature.com/articles/s41598-021-02657-6)

10.1038/s41598-021-02657-6

---

## Maternal inheritance of the koala gut microbiome and its compositional and functional maturation during juvenile development.
 04.12.2021, _MDJ Blyton, RM Soo, P Hugenholtz, BD Moore_


The acquisition and maturation of the gastrointestinal microbiome is a crucial aspect of mammalian development, particularly for specialist herbivores such as the koala (Phascolarctos cinereus). Joey koalas are thought to be inoculated with microorganisms by feeding on specialised maternal faeces (pap). We found that compared to faeces, pap has higher microbial density, higher microbial evenness and a higher proportion of rare taxa, which may facilitate the establishment of those taxa in joey koalas. We show that the microbiomes of captive joey koalas were on average more similar to those of their mothers than to other koalas, indicating strong maternal inheritance of the faecal microbiome, which can lead to intergenerational gut dysbiosis when the mother is ill. Directly after pap feeding, the joey koalas' microbiomes were enriched for milk-associated bacteria including Bacteroides fragilis, suggesting a conserved role for this species across mammalian taxa. The joeys' microbiomes then changed slowly over five months to resemble those of adults by one year of age. The relative abundance of fibrolytic bacteria and genes involved in the degradation of plant cell walls also increased in the infants over this time, likely in response to an increased proportion of Eucalyptus leaves in their diets. This article is protected by copyright. All rights reserved.

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.15858)

10.1111/1462-2920.15858

---

## Relationships between pond water and tilapia skin microbiomes in aquaculture ponds in Malawi

[Source](https://www.biorxiv.org/content/10.1101/2021.12.06.470702v1.abstract)

---

## Age-Based Variations in the Gut Microbiome of the Shennongjia (Hubei) Golden Snub-Nosed Monkey (Rhinopithecus roxellana hubeiensis).
 26.05.2021, _L Yao, X Li, Z Zhou, D Shi, Z Li, S Li, H Yao, J Yang, H Yu, Y Xiao_


_Aging, Bacteria, Colobinae, Gastrointestinal Microbiome_
The gut microbiota represents a source of genetic and metabolic diversity of a complex polymicrobial ecosystem within its host. To investigate age-based variations of the gut microbiota among Shennongjia golden snub-nosed monkeys (Rhinopithecus roxellana hubeiensis), we characterized the microbial species in fecal samples from 18 Shennongjia golden snub-nosed monkeys evenly pooled into 3 aged groups (Group 1, 1-3 years; Group 2, 5-8 years; Group 3, above 12 years) in Shennongjia, Hubei Province, China. Genomic DNA was extracted from fecal samples, and the 16S rRNA gene V4 region was sequenced using the Illumina high-throughput MiSeq platform PE250. A total of 28 microbial phyla were identified in the gut microbiome of these monkeys with the ten most abundant phyla (i.e., Firmicutes, Bacteroidetes, Verrucomicrobia, Spirochaetes, Tenericutes, Proteobacteria, Planctomycetes, Fibrobacteres, Cyanobacteria, and Euryarchaeota). A total of 1,469 (of 16 phyla and 166 genera), 1,381 (of 16 phyla and 157 genera), and 1,931 (of 19 phyla and 190 genera) operational taxonomic units (OTUs) were revealed in Groups 1, 2, and 3, respectively, with Group 3 containing the most diverse groups of OTUs as revealed by the species relative abundance clustering analysis. These results suggest that the gut microbiota in these monkeys maintain a dynamic status, starting from the early developmental stages of life with the species relative abundance increasing with age. This is the first study to comprehensively characterize the gut microbiota and provide valuable information for monitoring the health and nutritional needs of this endangered primate at different ages.

[Source](https://www.hindawi.com/journals/bmri/2021/6667715/)

10.1155/2021/6667715

---

## Looking through the FOG: microbiome characterization and lipolytic bacteria isolation from a fatberg site.
 07.12.2021, _EK Court, RR Chaudhuri, RV Kapoore, RX Villa, J Pandhal, CA Biggs, GP Stafford_


Sewer systems are complex physical, chemical and microbial ecosystems where fats, oils and grease (FOG) present a major problem for sewer management. Their accumulation can lead to blockages ('Fatbergs'), sewer overflows and disruption of downstream wastewater treatment. Further advancements of biological FOG treatments need to be tailored to degrade the FOG, and operate successfully within the sewer environment. In this study we developed a pipeline for isolation of lipolytic strains directly from two FOG blockage sites in the UK, and isolated a range of highly lipolytic bacteria. We selected the five most lipolytic strains using Rhodamine B agar plates and pNP-Fatty acid substrates, with two Serratia spp., two Klebsiella spp. and an environmental Acinetobacter strain that all have the capacity to grow on FOG-based carbon sources. Their genome sequences identified the genetic capacity for fatty acid harvesting (lipases), catabolism and utilization (Fad genes). Furthermore, we performed a preliminary molecular characterization of the microbial community at these sites, showing a diverse community of environmental bacteria at each site, but which did include evidence of sequences related to our isolates. This study provides proof of concept to isolation strategies targeting Fatberg sites to yield candidate strains with bioremediation potential for FOG in the wastewater network. Our work sets the foundation for development of novel bioadditions tailored to the environment with non-pathogenic Acinetobacter identified as a candidate for this purpose.

[Source](https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.001117)

10.1099/mic.0.001117

---

## The Baltic Sea methane pockmark microbiome: The new insights into the patterns of relative abundance and ANME niche separation

[Source](https://www.sciencedirect.com/science/article/pii/S0141113621002890)

---

## 3MCor: an integrative web server for metabolome-microbiome-metadata correlation analysis

[Source](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab818/6454942)

---

## Microbiome Methods in Experimental Autoimmune Encephalomyelitis.
 07.12.2021, _DP Daberkow, K Hoffman, HM Kohl, T Long, TO Kirby, J Ochoa-Repáraz_


Microbiome composition studies are increasingly shedding light on animal models of disease. This paper describes a protocol for analyzing the gut microbiome composition prior to and after the induction of mice to experimental autoimmune encephalomyelitis (EAE), the principal animal model of the human neuroinflammatory demyelinating disease multiple sclerosis (MS). We also address and provide data assessing the impact of mice reared in different animal facilities on EAE induction. Furthermore, we discuss potential regulators of the gut-microbiome-brain axis (GMBA) in relation to neuroinflammation and implications on demyelinating disease states. Our results suggest that mice reared in different animal facilities produce different levels of EAE induction. These results highlight the importance of accounting for consistent environmental conditions when inducing EAE and other animal models of disease. © 2021 Wiley Periodicals LLC. Basic Protocol 1: Study of the composition of the gut microbiome in the neuroinflammatory model of experimental autoimmune encephalomyelitis Basic Protocol 2: Experimental procedures for DNA extraction and microbiome analysis.

[Source](https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/cpz1.314)

10.1002/cpz1.314

---

## A wipe-based stool collection and preservation kit for microbiome community profiling

[Source](https://www.biorxiv.org/content/10.1101/2021.12.03.471072v1.abstract)

---

