## Microbiota regulate social behaviour via stress response neurons in the brain.
 01.07.2021, _WL Wu, MD Adame, CW Liou, JT Barlow, TT Lai, G Sharon, CE Schretter, BD Needham, MI Wang, W Tang, J Ousey, YY Lin, TH Yao, R Abdel-Haq, K Beadle, V Gradinaru, RF Ismagilov, SK Mazmanian_


Social interactions among animals mediate essential behaviours, including mating, nurturing, and defence1,2. The gut microbiota contribute to social activity in mice3,4, but the gut-brain connections that regulate this complex behaviour and its underlying neural basis are unclear5,6. Here we show that the microbiome modulates neuronal activity in specific brain regions of male mice to regulate canonical stress responses and social behaviours. Social deviation in germ-free and antibiotic-treated mice is associated with elevated levels of the stress hormone corticosterone, which is primarily produced by activation of the hypothalamus-pituitary-adrenal (HPA) axis. Adrenalectomy, antagonism of glucocorticoid receptors, or pharmacological inhibition of corticosterone synthesis effectively corrects social deficits following microbiome depletion. Genetic ablation of glucocorticoid receptors in specific brain regions or chemogenetic inactivation of neurons in the paraventricular nucleus of the hypothalamus that produce corticotrophin-releasing hormone (CRH) reverse social impairments in antibiotic-treated mice. Conversely, specific activation of CRH-expressing neurons in the paraventricular nucleus induces social deficits in mice with a normal microbiome. Via microbiome profiling and in vivo selection, we identify a bacterial species, Enterococcus faecalis, that promotes social activity and reduces corticosterone levels in mice following social stress. These studies suggest that specific gut bacteria can restrain the activation of the HPA axis, and show that the microbiome can affect social behaviours through discrete neuronal circuits that mediate stress responses in the brain.

[Source](https://www.nature.com/articles/s41586-021-03669-y)

10.1038/s41586-021-03669-y

---

## Defining the resilience of the human salivary microbiota by a 520-day longitudinal study in a confined environment: the Mars500 mission

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01070-5)

---

## A MicroRNA Gene Panel Predicts the Vaginal Microbiota Composition.
 05.05.2021, _L Cheng, D Kaźmierczak, J Norenhag, M Hamsten, E Fransson, I Schuppe-Koistinen, M Olovsson, L Engstrand, P Hydbring, J Du_


The vaginal microbiota plays an essential role in vaginal health. The vaginas of many reproductive-age women are dominated by one of the Lactobacillus species. However, the vaginas of a large number of women are characterized by the colonization of several other anaerobes. Notably, some women with the non-Lactobacillus-dominated vaginal microbiota develop bacterial vaginosis, which has been correlated with sexually transmitted infections and other adverse outcomes. However, interactions and mechanisms linking the vaginal microbiota to host response are still under investigation. There are studies suggesting a link between human microRNAs and gut microbiota, but limited analysis has been carried out on the interplay of microRNAs and vaginal microbiota. In this study, we performed a microRNA expression array profiling on 67 vaginal samples from young Swedish women. MicroRNAs were clustered into distinct groups according to vaginal microbiota composition. Interestingly, 182 microRNAs were significantly elevated in their expression in the non-Lactobacillus-dominated community, suggesting an antagonistic relationship between Lactobacillus and microRNAs. Of the elevated microRNAs, 10 microRNAs displayed excellent diagnostic potential, visualized by receiver operating characteristics analysis. We further validated our findings in 34 independent samples where expression of top microRNA candidates strongly separated the Lactobacillus-dominated community from the non-Lactobacillus-dominated community in the vaginal microbiota. Notably, the Lactobacillus crispatus-dominated community showed the most profound differential microRNA expression compared with the non-Lactobacillus-dominated community. In conclusion, we demonstrate a strong relationship between the vaginal microbiota and numerous genital microRNAs, which may facilitate a deeper mechanistic interplay in this biological niche.IMPORTANCE Vaginal microbiota is correlated with women's health, where a non-Lactobacillus-dominated community predisposes women to a higher risk of disease, including human papillomavirus (HPV). However, the molecular relationship between the vaginal microbiota and host is largely unexplored. In this study, we investigated a link between the vaginal microbiota and host microRNAs in a group of young women. We uncovered an inverse correlation of the expression of microRNAs with the abundance of Lactobacillus species in the vaginal microbiota. Particularly, the expression of microRNA miR-23a-3p and miR-130a-3p, displaying significantly elevated levels in non-Lactobacillus-dominated communities, predicted the bacterial composition of the vaginal microbiota in an independent validation group. Since targeting of microRNAs is explored in the clinical setting, our results warrant investigation of whether microRNA modulation could be used for treating vaginosis recurrence and vaginosis-related diseases. Conversely, commensal bacteria could be used for treating diseases with microRNA aberrations.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00175-21)

10.1128/mSystems.00175-21

---

## Gut microbiota mediate the FGF21 adaptive stress response to chronic dietary protein-restriction in mice.
 28.06.2021, _A Martin, G Ecklu-Mensah, CWY Ha, G Hendrick, DK Layman, J Gilbert, S Devkota_


Chronic dietary protein-restriction can create essential amino acid deficiencies and induce metabolic adaptation through the hepatic FGF21 pathway which serves to maintain host fitness during prolonged states of nutritional imbalance. Similarly, the gut microbiome undergoes metabolic adaptations when dietary nutrients are added or withdrawn. Here we confirm previous reports that dietary protein-restriction triggers the hepatic FGF21 adaptive metabolic pathway and further demonstrate that this response is mediated by the gut microbiome and can be tuned through dietary supplementation of fibers that alter the gut microbiome. In the absence of a gut microbiome, we discover that FGF21 is de-sensitized to the effect of protein-restriction. These data suggest that host-intrinsic adaptive pathways to chronic dietary protein-restriction, such as the hepatic FGF21 pathway, may in-fact be responding first to adaptive metabolic changes in the gut microbiome.

[Source](https://www.nature.com/articles/s41467-021-24074-z)

10.1038/s41467-021-24074-z

---

## Gut microbes impact stroke severity via the trimethylamine N-oxide pathway.
 17.06.2021, _W Zhu, KA Romano, L Li, JA Buffa, N Sangwan, P Prakash, AN Tittle, XS Li, X Fu, C Androjna, AJ DiDonato, K Brinson, BD Trapp, MA Fischbach, FE Rey, AM Hajjar, JA DiDonato, SL Hazen_


Clinical studies have demonstrated associations between circulating levels of the gut-microbiota-derived metabolite trimethylamine-N-oxide (TMAO) and stroke incident risk. However, a causal role of gut microbes in stroke has not yet been demonstrated. Herein we show that gut microbes, through dietary choline and TMAO generation, directly impact cerebral infarct size and adverse outcomes following stroke. Fecal microbial transplantation from low- versus high-TMAO-producing human subjects into germ-free mice shows that both TMAO generation and stroke severity are transmissible traits. Furthermore, employing multiple murine stroke models and transplantation of defined microbial communities with genetically engineered human commensals into germ-free mice, we demonstrate that the microbial cutC gene (an enzymatic source of choline-to-TMA transformation) is sufficient to transmit TMA/TMAO production, heighten cerebral infarct size, and lead to functional impairment. We thus reveal that gut microbiota in general, specifically the metaorganismal TMAO pathway, directly contributes to stroke severity.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(21)00230-4)

10.1016/j.chom.2021.05.002

---

## Dietary Fiber Hierarchical Specificity: the Missing Link for Predictable and Strong Shifts in Gut Bacterial Communities.
 29.06.2021, _TM Cantu-Jungles, N Bulut, E Chambry, A Ruthes, M Iacomini, A Keshavarzian, TA Johnson, BR Hamaker_


Most dietary fibers used to shape the gut microbiota present different and unpredictable responses, presumably due to the diverse microbial communities of people. Recently, we proposed that fibers can be classified in a hierarchical way where fibers of high specificity (i.e., structurally complex and utilized by a narrow group of gut bacteria) could have more similar interindividual responses than those of low specificity (i.e., structurally simple and utilized by many gut bacteria). To test this hypothesis, we evaluated microbiota fermentation of fibers tentatively classified as low (fructooligosaccharides), low-to-intermediate (type 2 resistant starch), intermediate (pectin), and high (insoluble β-1,3-glucan) specificity, utilizing fecal inoculum from distinct subjects, regarding interindividual similarity/dissimilarity in fiber responses. Individual shifts in target bacteria (as determined by linear discriminant analysis) confirmed that divergent fiber responses occur when utilizing both of the low-specificity dietary fibers, but fibers of intermediate and high specificity lead to more similar responses across subjects in support of targeted bacteria. The high-specificity insoluble β-glucan promoted a large increase of the target bacteria (from 0.3 to 16.5% average for Anaerostipes sp. and 2.5 to 17.9% average for Bacteroides uniformis), which were associated with increases in ratios of related metabolites (butyrate and propionate, respectively) in every microbial community in which these bacteria were present. Also, high-specificity dietary fibers promoted more dramatic changes in microbial community structure than low-specificity ones relative to the initial microbial communities. IMPORTANCE In the face of interindividual variability and complexity of gut microbial communities, prediction of outcomes from a given fiber utilized by many microbes would require a sophisticated comprehension of all competitive interactions that occur in the gut. Results presented here suggest that high-specificity fibers potentially circumvent the competitive scope in the gut for fiber utilization, providing a promising path to targeted and predictable microbial shifts in different individuals. These findings are the first to indicate that fiber specificity is related to similarity and intensity of response in distinct human gut microbiota communities.

[Source](https://journals.asm.org/doi/10.1128/mBio.01028-21)

10.1128/mBio.01028-21

---

## Bifidobacteria-mediated immune system imprinting early in life.
 18.06.2021, _BM Henrick, L Rodriguez, T Lakshmikanth, C Pou, E Henckel, A Arzoomand, A Olin, J Wang, J Mikes, Z Tan, Y Chen, AM Ehrlich, AK Bernhardsson, CH Mugabo, Y Ambrosiani, A Gustafsson, S Chew, HK Brown, J Prambs, K Bohlin, RD Mitchell, MA Underwood, JT Smilowitz, JB German, SA Frese, P Brodin_


Immune-microbe interactions early in life influence the risk of allergies, asthma, and other inflammatory diseases. Breastfeeding guides healthier immune-microbe relationships by providing nutrients to specialized microbes that in turn benefit the host's immune system. Such bacteria have co-evolved with humans but are now increasingly rare in modern societies. Here we show that a lack of bifidobacteria, and in particular depletion of genes required for human milk oligosaccharide (HMO) utilization from the metagenome, is associated with systemic inflammation and immune dysregulation early in life. In breastfed infants given Bifidobacterium infantis EVC001, which expresses all HMO-utilization genes, intestinal T helper 2 (Th2) and Th17 cytokines were silenced and interferon β (IFNβ) was induced. Fecal water from EVC001-supplemented infants contains abundant indolelactate and B. infantis-derived indole-3-lactic acid (ILA) upregulated immunoregulatory galectin-1 in Th2 and Th17 cells during polarization, providing a functional link between beneficial microbes and immunoregulation during the first months of life.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(21)00660-7)

10.1016/j.cell.2021.05.030

---

## Temporal patterns in Ixodes ricinus microbial communities: an insight into tick-borne microbe interactions.
 04.07.2021, _E Lejal, J Chiquet, J Aubert, S Robin, A Estrada-Peña, O Rue, C Midoux, M Mariadassou, X Bailly, A Cougoul, P Gasqui, JF Cosson, K Chalvet-Monfray, M Vayssier-Taussat, T Pollet_


Ticks transmit pathogens of medical and veterinary importance and are an increasing threat to human and animal health. Assessing disease risk and developing new control strategies requires identifying members of the tick-borne microbiota as well as their temporal dynamics and interactions.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01051-8)

10.1186/s40168-021-01051-8

---

## Microbiome diversity and host immune functions influence survivorship of sponge holobionts under future ocean conditions.
 04.07.2021, _N Posadas, JIP Baquiran, MAL Nada, M Kelly, C Conaco_


The sponge-associated microbial community contributes to the overall health and adaptive capacity of the sponge holobiont. This community is regulated by the environment and the immune system of the host. However, little is known about the effect of environmental stress on the regulation of host immune functions and how this may, in turn, affect sponge-microbe interactions. In this study, we compared the bacterial diversity and immune repertoire of the demosponge, Neopetrosia compacta, and the calcareous sponge, Leucetta chagosensis, under varying levels of acidification and warming stress based on climate scenarios predicted for 2100. Neopetrosia compacta harbors a diverse microbial community and possesses a rich repertoire of scavenger receptors while L. chagosensis has a less diverse microbiome and an expanded range of pattern recognition receptors and immune response-related genes. Upon exposure to RCP 8.5 conditions, the microbiome composition and host transcriptome of N. compacta remained stable, which correlated with high survival (75%). In contrast, tissue necrosis and low survival (25%) of L. chagosensis was accompanied by microbial community shifts and downregulation of host immune-related pathways. Meta-analysis of microbiome diversity and immunological repertoire across poriferan classes further highlights the importance of host-microbe interactions in predicting the fate of sponges under future ocean conditions.

[Source](https://www.nature.com/articles/s41396-021-01050-5)

10.1038/s41396-021-01050-5

---

## A Community Effort: Combining Functional Amplicon Sequencing and Metagenomics Reveals Potential Biosynthetic Gene Clusters Associated with Protective Phenotypes in Rhizosphere Microbiomes.
 30.06.2021, _JM Winter_


Soil-dwelling microorganisms associated with plant roots carry out essential processes that promote plant growth and productivity. In addition to these beneficial functions, the rhizosphere microbiome also serves as the first line of defense against many plant pathogens. While many rhizobacteria are capable of producing antifungal natural products, fungal pathogens, such as those belonging to the genus Fusarium, continue to be a major threat to agricultural crops worldwide. In this issue, Tracanna and coworkers (V. Tracanna, A. Ossowicki, M. L. C. Petrus, S. Overduin, et al., mSystems 6:e01116-20, 2021, https://doi.org/10.1128/mSystems.01116-20) implement a targeted amplicon sequencing approach to identify conserved domains and specific metabolic pathways shared among soil samples with antagonistic activities against Fusarium culmorum. They also introduce dom2BGC, an open-source annotation platform that builds co-occurrence networks of natural product-associated domains across samples and aids in putative gene cluster reconstruction. When coupled with metagenomics, functional amplicon sequencing and the dom2BGC pipeline can aid in identifying mechanisms and potential metabolites associated with particular microbiome-associated phenotypes.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00587-21)

10.1128/mSystems.00587-21

---

## METAnnotatorX2: a Comprehensive Tool for Deep and Shallow Metagenomic Data Set Analyses.
 29.06.2021, _C Milani, GA Lugli, F Fontana, L Mancabelli, G Alessandri, G Longhi, R Anzalone, A Viappiani, F Turroni, D van Sinderen, M Ventura_


The use of bioinformatic tools for read-based taxonomic and functional analyses of metagenomic data sets, including their assembly and management, is rather fragmentary due to the absence of an accepted gold standard. Moreover, most currently available software tools need input of millions of reads and rely on approximations in data analysis in order to reduce computing times. These issues result in suboptimal results in terms of accuracy, sensitivity, and specificity when used either for the reconstruction of taxonomic or functional profiles through read analysis or analysis of genomes reconstructed by metagenomic assembly. Moreover, the recent introduction of novel DNA sequencing technologies that generate long reads, such as Nanopore and PacBio, represent a valuable data resource that still suffers from a lack of dedicated tools to perform integrated hybrid analysis alongside short read data. In order to overcome these limitations, here we describe a comprehensive bioinformatic platform, METAnnotatorX2, aimed at providing an optimized user-friendly resource which maximizes output quality, while also allowing user-specific adaptation of the pipeline and straightforward integrated analysis of both short and long read data. To further improve performance quality and accuracy of taxonomic assignment of reads and contigs, custom preprocessed and taxonomically revised genomic databases for viruses, prokaryotes, and various eukaryotes were developed. The performance of METAnnotatorX2 was tested by analysis of artificial data sets encompassing viral, archaeal, bacterial, and eukaryotic (fungal) sequence reads that simulate different biological matrices. Moreover, real biological samples were employed to validate in silico results. IMPORTANCE We developed a novel tool, i.e., METAnnotatorX2, that includes a number of new advanced features for analysis of deep and shallow metagenomic data sets and is accompanied by (regularly updated) customized databases for archaea, bacteria, fungi, protists, and viruses. Both software and databases were developed so as to maximize sensitivity and specificity while including support for shallow metagenomic data sets. Through extensive tests performed on Illumina and Nanopore artificial data sets, we demonstrated the high performance of the software to not only extract taxonomic and functional information from sequence reads but also to assemble and process genomes from metagenomic data. The robustness of these functionalities was validated using "real-life" data sets obtained from Illumina and Nanopore sequencing of biological samples. Furthermore, the performance of METAnnotatorX2 was compared to other available software tools for analysis of shotgun metagenomics data.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00583-21)

10.1128/mSystems.00583-21

---

