## Microbiome definition re-visited: old concepts and new challenges.
 03.07.2020, _G Berg, D Rybakova, D Fischer, T Cernava, MC Vergès, T Charles, X Chen, L Cocolin, K Eversole, GH Corral, M Kazou, L Kinkel, L Lange, N Lima, A Loy, JA Macklin, E Maguin, T Mauchline, R McClure, B Mitter, M Ryan, I Sarand, H Smidt, B Schelkle, H Roume, GS Kiran, J Selvin, RSC Souza, L van Overbeek, BK Singh, M Wagner, A Walsh, A Sessitsch, M Schloter_


The field of microbiome research has evolved rapidly over the past few decades and has become a topic of great scientific and public interest. As a result of this rapid growth in interest covering different fields, we are lacking a clear commonly agreed definition of the term "microbiome." Moreover, a consensus on best practices in microbiome research is missing. Recently, a panel of international experts discussed the current gaps in the frame of the European-funded MicrobiomeSupport project. The meeting brought together about 40 leaders from diverse microbiome areas, while more than a hundred experts from all over the world took part in an online survey accompanying the workshop. This article excerpts the outcomes of the workshop and the corresponding online survey embedded in a short historical introduction and future outlook. We propose a definition of microbiome based on the compact, clear, and comprehensive description of the term provided by Whipps et al. in 1988, amended with a set of novel recommendations considering the latest technological developments and research findings. We clearly separate the terms microbiome and microbiota and provide a comprehensive discussion considering the composition of microbiota, the heterogeneity and dynamics of microbiomes in time and space, the stability and resilience of microbial networks, the definition of core microbiomes, and functionally relevant keystone species as well as co-evolutionary principles of microbe-host and inter-species interactions within the microbiome. These broad definitions together with the suggested unifying concepts will help to improve standardization of microbiome studies in the future, and could be the starting point for an integrated assessment of data resulting in a more rapid transfer of knowledge from basic science into practice. Furthermore, microbiome standards are important for solving new challenges associated with anthropogenic-driven changes in the field of planetary health, for which the understanding of microbiomes might play a key role. Video Abstract.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00875-0)

10.1186/s40168-020-00875-0

---

## Competition, not cooperation, dominates interactions among culturable microbial species.
 21.11.2013, _KR Foster, T Bell_


_Bacteria, Bacterial Physiological Phenomena, Carbon Dioxide, Water Microbiology_
Microbial cells secrete numerous enzymes, scavenging molecules, and signals that can promote the growth and survival of other cells around them [1-4]. This observation is consistent with the evolution of cooperation within species [5], and there is now an increasing emphasis on the importance of cooperation between different microbial species [4, 6]. We lack, however, a systematic test of the importance of mutually positive interactions between different species, which is vital for assessing the commonness and importance of cooperative evolution in natural communities. Here, we study the extent of mutually positive interaction among bacterial strains isolated from a common aquatic environment. Using data collected from two independent experiments evaluating community productivity across diversity gradients, we show that (1) in pairwise species combinations, the great majority of interactions are net negative and (2) there is no evidence that strong higher-order positive effects arise when more than two species are mixed together. Our data do not exclude the possibility of positive effects in one direction where one species gains at the expense of another, i.e., predator-prey-like interactions. However, these do not constitute cooperation and our analysis suggests that the typical result of adaptation to other microbial species will be competitive, rather than cooperative, phenotypes.

[Source](https://www.biorxiv.org/content/10.1101/2020.06.24.169474v1)

10.1016/j.cub.2012.08.005

---

## Positive interactions are common among culturable bacteria

[Source](https://www.biorxiv.org/content/10.1101/2020.06.24.169474v1)

---

## An emerging role of<em> Prevotella histicola</em> on estrogen-deficiency induced bone loss through the gut microbiota-bone axis

[Source](https://www.biorxiv.org/content/10.1101/2020.06.03.133082v1.full.pdf)

---

## Identifying novel high-impact rare disease-causing mutations, genes and pathways in exomes of Ashkenazi Jewish inflammatory bowel disease patients

[Source](https://www.medrxiv.org/content/10.1101/2020.07.01.20143750v1.full.pdf)

---

## Early prediction of liver disease using conventional risk factors and gut microbiome-augmented gradient boosting 

[Source](https://www.medrxiv.org/content/10.1101/2020.06.24.20138933v1.full.pdf)

---

## GWAS of stool frequency reveals genes, pathways, and cell types relevant to human gastrointestinal motility and irritable bowel syndrome

[Source](https://www.medrxiv.org/content/10.1101/2020.06.17.20132555v1.full.pdf)

---

## Plasma Short-Chain Fatty Acids in Patients With Parkinson's Disease.
 22.06.2020, _C Shin, Y Lim, H Lim, TB Ahn_


Short-chain fatty acids are exclusively produced by gut microbiota and are reduced in feces of patients with Parkinson's disease (PD). The objective of this study was to conduct a case-control study on peripheral concentration of short-chain fatty acids based on evidence of pathologic changes in the blood-brain barrier in PD and the possible role of short-chain fatty acids in blood-brain barrier permeability.

[Source](https://www.medrxiv.org/content/10.1101/2020.06.15.20131011v1.full.pdf)

10.1002/mds.28016

---

## Relationships of gut microbiota, short-chain fatty acids, inflammation, and the gut barrier in Parkinson’s disease

[Source](https://www.medrxiv.org/content/10.1101/2020.06.15.20131011v1.full.pdf)

---

## Genetically diverse<em> Pseudomonas aeruginosa</em> populations display similar transcriptomic profiles in a cystic fibrosis explanted lung

[Source](https://www.nature.com/articles/s41467-019-11414-3)

---

## Architecture and function of human uromodulin filaments in urinary tract infections.
 03.07.2020, _GL Weiss, JJ Stanisich, MM Sauer, CW Lin, J Eras, DS Zyla, J Trück, O Devuyst, M Aebi, M Pilhofer, R Glockshuber_


Uromodulin is the most abundant protein in human urine and forms filaments that antagonize the adhesion of uropathogens; however, filament structure and mechanism of protection remain poorly understood. We used cryo-electron tomography to show that the uromodulin filament consists of a zigzag-shaped backbone with laterally protruding arms. N-glycosylation mapping and biophysical assays revealed that uromodulin acts as a multivalent ligand for the bacterial type 1 pilus adhesin, presenting specific epitopes on the regularly spaced arms. Imaging of uromodulin-uropathogen interactions in vitro and in patient urine showed that uromodulin filaments associate with uropathogens and mediate bacterial aggregation, which likely prevents adhesion and allows clearance by micturition. These results provide a framework for understanding uromodulin in urinary tract infections and in its more enigmatic roles in physiology and disease.

[Source](https://science.sciencemag.org/content/early/2020/07/01/science.aaz9866)

10.1126/science.aaz9866

---

## The structure and diversity of strain level variation in vaginal bacteria

[Source](https://www.biorxiv.org/content/10.1101/2020.06.26.173922v1)

---

## Synchronous genitourinary lichen sclerosus signals a distinct urinary microbiome profile in men with urethral stricture disease.
 10.04.2020, _AJ Cohen, TW Gaither, S Srirangapatanam, ER Castellanos, A Enriquez, KB Fergus, D Fadrosh, S Lynch, NA Mmonu, BN Breyer_


Alterations in the urinary microbiome have been associated with urological diseases. The microbiome of patients with urethral stricture disease (USD) remains unknown. Our objective is to examine the microbiome of USD with a focus on inflammatory USD caused by lichen sclerosus (LS).

[Source](https://link.springer.com/article/10.1007/s00345-020-03198-9)

10.1007/s00345-020-03198-9

---

## Human urine 1H NMR metabolomics reveals alterations of the protein and carbohydrate metabolism when comparing habitual Average Danish diet vs. healthy New Nordic diet.
 03.07.2020, _A Trimigno, B Khakimov, F Savorani, SK Poulsen, A Astrup, LO Dragsted, SB Engelsen_


The aim of this study was to investigate the alteration of the human urine metabolome by means of diet and to compare the metabolic effects of the nutritionally healthy New Nordic Diet (NND) with an Average Danish Diet (ADD). The NND was designed a decade ago by scientists and chefs, based on local and sustainable foods, including fish, shellfish, vegetables, roots, fruit, and berries. The NND has been proven to lower blood pressure, reduce glycemia, and lead to weight loss.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0899900720301507)

10.1016/j.nut.2020.110867

---

## <em>Lactobacillus rhamnosus</em> probiotic prevents airway function deterioration and promotes gut microbiome resilience in a murine asthma model

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2020.1766345)

---

## Molecular analysis of the dominant lactic acid bacteria of chickpea liquid starters and doughs and propagation of chickpea sourdoughs with selected Weissella confusa.
 17.06.2020, _CP Boyaci Gunduz, R Gaglio, E Franciosi, L Settanni, H Erten_


Fermented chickpea liquid is used as a leavening agent in chickpea bread production. In the present study, traditional chickpea liquid starter and dough samples were collected from bakeries in Turkey and microbiologically investigated. Culture-independent analysis for microbiota diversity, performed by MiSeq Illumina, identified Clostridium perfringens as major group in all samples, while Weissella spp. Dominated LAB community. A culture-dependent methodology was applied and 141 isolates were confirmed to be members of the LAB group based on 16s rRNA gene sequence analysis. In particular, 11 different LAB species were identified confirming the high frequency of isolation of weissellas, since Weissella confusa and Weissella cibaria constituted 47.8 and 12.4%, respectively, of total LAB isolated. The other species were Enterococcus faecium, Enterococcus lactis, Lactobacillus brevis, Lactobacillus plantarum, Leuconostoc mesenteroides, Leuconostoc mesenteroides subsp. Dextranium, Pediococcus acidilactici, Pediococcus pentosaceus and Streptococcus lutetiensis. Due to high frequency of isolation, W. confusa strains were investigated at technological level and W. confusa RL1139 was used as mono-culture starter in the experimental chickpea sourdough production. Chemical and microbiological properties, as well as volatile organic compounds (VOCs) of the chickpea liquid starters and doughs were subjected to a multivariate analysis. Control and W. confusa inoculated chickpea liquid starter and dough samples were close to each other in terms of some characteristics related to chemical, microbiological and VOCs profile, but the inoculated sourdough showed a higher generation of certain VOCs, like butanoic acid (81.52%) and ethyl acetate (8.15%) than control sourdough. This is important in order to maintain typical characteristics of the traditional chickpea dough, but at the same time improving the aroma profile. This work demonstrated that W. confusa RL1139 can be applied at large scale production level without compromising the typical characteristics of the final product.

[Source](https://asm.org/Articles/2020/June/The-Sourdough-Microbiome)

10.1016/j.fm.2020.103490

---

## The Sourdough Microbiome

[Source](https://asm.org/Articles/2020/June/The-Sourdough-Microbiome)

---

## Strain-Level Diversity Impacts Cheese Rind Microbiome Assembly and Function.
 26.06.2020, _BA Niccum, EK Kastman, N Kfoury, A Robbat, BE Wolfe_


Diversification can generate genomic and phenotypic strain-level diversity within microbial species. This microdiversity is widely recognized in populations, but the community-level consequences of microbial strain-level diversity are poorly characterized. Using the cheese rind model system, we tested whether strain diversity across microbiomes from distinct geographic regions impacts assembly dynamics and functional outputs. We first isolated the same three bacterial species (Staphylococcus equorum, Brevibacterium auranticum, and Brachybacterium alimentarium) from nine cheeses produced in different regions of the United States and Europe to construct nine synthetic microbial communities consisting of distinct strains of the same three bacterial species. Comparative genomics identified distinct phylogenetic clusters and significant variation in genome content across the nine synthetic communities. When we assembled each synthetic community with initially identical compositions, community structure diverged over time, resulting in communities with different dominant taxa. The taxonomically identical communities showed differing responses to abiotic (high salt) and biotic (the fungus Penicillium) perturbations, with some communities showing no response and others substantially shifting in composition. Functional differences were also observed across the nine communities, with significant variation in pigment production (light yellow to orange) and in composition of volatile organic compound profiles emitted from the rinds (nutty to sulfury).IMPORTANCE Our work demonstrated that the specific microbial strains used to construct a microbiome could impact the species composition, perturbation responses, and functional outputs of that system. These findings suggest that 16S rRNA gene taxonomic profiles alone may have limited potential to predict the dynamics of microbial communities because they usually do not capture strain-level diversity. Observations from our synthetic communities also suggest that strain-level diversity has the potential to drive variability in the aesthetics and quality of surface-ripened cheeses.

[Source](https://msystems.asm.org/content/5/3/e00149-20)

10.1128/mSystems.00149-20

---

## Light into the darkness: Unifying the known and unknown coding sequence space in microbiome analyses

[Source](https://www.biorxiv.org/content/10.1101/2020.06.30.180448v1)

---

## Welcome to the dark side of genomes and metagenomes

[Source](http://merenlab.org/2020/07/01/dark-side/)

---

## GraphBin: refined binning of metagenomic contigs using assembly graphs

[Source](https://academic.oup.com/bioinformatics/article-abstract/36/11/3307/5804980)

---

## Reproducible pipelines and NGLess with Luis Pedro Coelho

[Source](https://bioinformatics.chat/ngless)

---

