## The impact of prenatal dog keeping on infant gut microbiota development.
 14.03.2023, _AR Panzer, AR Sitarik, D Fadrosh, SL Havstad, K Jones, B Davidson, S Finazzo, GR Wegienka, K Woodcroft, NW Lukacs, AM Levin, DR Ownby, CC Johnson, SV Lynch, EM Zoratti_


Prenatal and early-life dog exposure has been linked to reduced childhood allergy and asthma. A potential mechanism includes altered early immune development in response to changes in the gut microbiome among dog-exposed infants. We thus sought to determine whether infants born into homes with indoor dog(s) exhibit altered gut microbiome development.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/cea.14303)

10.1111/cea.14303

---

## Host microbiome interactions in apical periodontitis: the endodontic microbiome in relation to circulatory immunologic markers.
 14.03.2023, _AC Georgiou, SV van der Waal, MJ Buijs, W Crielaard, E Zaura, BW Brandt_


To explore microbial differences in the endodontic infection of teeth with primary and secondary apical periodontitis, with or without symptomatology. Additionally, to investigate if these differences are depicted in immunologic markers in blood.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/iej.13912)

10.1111/iej.13912

---

## Microbiota dysbiosis and myasthenia gravis: Do all roads lead to Rome?
 16.03.2023, _B Kapoor, M Gulati, R Gupta, RK Singla_


Dysregulated immune system with a failure to recognize self from non-self-antigens is one of the common pathogeneses seen in autoimmune diseases. The complex interplay of genetic and environmental factors is important for the occurrence and development of the disease. Among the environmental factors, disturbed gut microbiota (gut dysbiosis) has recently attracted particular attention, especially with advancement in human microbiome research. Although the alterations in microbiota have been seen in various autoimmune diseases, including those of nervous system, there is paucity of information on neuromuscular system diseases. Myasthenia gravis (MG) is one such rare autoimmune disease of neuromuscular junction, and is caused by generation of pathogenic autoantibodies to components of the postsynaptic muscle endplate. In the recent years, accumulating evidences have endorsed the key role of host microbiota, particularly those of gut, in the pathogenesis of MG. Differential microbiota composition, characterized by increased abundance of Fusobacteria, Bacteroidetes, and Proteobacteria, and decreased abundance of Actinobacteria and Firmicutes, has been seen in MG patients in comparison to healthy subjects. Disturbance of microbiota composition, particularly reduced ratio of Firmicutes/Bacteroidetes, alter the gut permeability, subsequently triggering the immunological response. Resultant reduction in levels of short chain fatty acids (SCFAs) is another factor contributing to the immunological response in MG patients. Modulation of gut microbiota via intervention of probiotics, prebiotics, synbiotics, postbiotics (metabiotics), and fecal microbiota transplantation (FMT) is considered to be the futuristic approach for the management of MG. This review summarizes the role of gut microbiota and their metabolites (postbiotics) in the progression of MG. Also, various bacteriotherapeutic approaches involving gut microbiota are discussed for the prevention of MG progression.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1568997223000472)

10.1016/j.autrev.2023.103313

---

## Metabolomic interplay between gut microbiome and plasma metabolome in cardiac surgery-associated acute kidney injury.
 14.03.2023, _Y Bai, W Huang, X Jiang, W Xu, Y Li, Y Wang, S Huang, K Wu, L Hu, C Chen_


Cardiac surgery-associated acute kidney injury (CSA-AKI) is a prevalent complication of cardiac surgery, which may be associated with a great risk of developing chronic kidney disease and mortality. This study aimed to investigate the possible links between gut microbiota metabolism and CSA-AKI.

[Source](https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/rcm.9504)

10.1002/rcm.9504

---

## A non-antibiotic-disrupted gut microbiome is associated with clinical responses to CD19-CAR-T cell cancer immunotherapy.
 14.03.2023, _CK Stein-Thoeringer, NY Saini, E Zamir, V Blumenberg, ML Schubert, U Mor, MA Fante, S Schmidt, E Hayase, T Hayase, R Rohrbach, CC Chang, L McDaniel, I Flores, R Gaiser, M Edinger, D Wolff, M Heidenreich, P Strati, R Nair, D Chihara, LE Fayad, S Ahmed, SP Iyer, RE Steiner, P Jain, LJ Nastoupil, J Westin, R Arora, ML Wang, J Turner, M Menges, M Hidalgo-Vargas, K Reid, P Dreger, A Schmitt, C MÃ¼ller-Tidow, FL Locke, ML Davila, RE Champlin, CR Flowers, EJ Shpall, H Poeck, SS Neelapu, M Schmitt, M Subklewe, MD Jain, RR Jenq, E Elinav_


Increasing evidence suggests that the gut microbiome may modulate the efficacy of cancer immunotherapy. In a B cell lymphoma patient cohort from five centers in Germany and the United States (Germany, n = 66; United States, n = 106; total, n = 172), we demonstrate that wide-spectrum antibiotics treatment ('high-risk antibiotics') prior to CD19-targeted chimeric antigen receptor (CAR)-T cell therapy is associated with adverse outcomes, but this effect is likely to be confounded by an increased pretreatment tumor burden and systemic inflammation in patients pretreated with high-risk antibiotics. To resolve this confounding effect and gain insights into antibiotics-masked microbiome signals impacting CAR-T efficacy, we focused on the high-risk antibiotics non-exposed patient population. Indeed, in these patients, significant correlations were noted between pre-CAR-T infusion Bifidobacterium longum and microbiome-encoded peptidoglycan biosynthesis, and CAR-T treatment-associated 6-month survival or lymphoma progression. Furthermore, predictive pre-CAR-T treatment microbiome-based machine learning algorithms trained on the high-risk antibiotics non-exposed German cohort and validated by the respective US cohort robustly segregated long-term responders from non-responders. Bacteroides, Ruminococcus, Eubacterium and Akkermansia were most important in determining CAR-T responsiveness, with Akkermansia also being associated with pre-infusion peripheral T cell levels in these patients. Collectively, we identify conserved microbiome features across clinical and geographical variations, which may enable cross-cohort microbiome-based predictions of outcomes in CAR-T cell immunotherapy.

[Source](https://www.nature.com/articles/s41591-023-02234-6)

10.1038/s41591-023-02234-6

---

## The gut microbiome modulates the transformation of microglial subtypes.
 14.03.2023, _Y Huang, J Wu, H Zhang, Y Li, L Wen, X Tan, K Cheng, Y Liu, J Pu, L Liu, H Wang, W Li, SW Perry, ML Wong, J Licinio, P Zheng, P Xie_


Clinical and animal studies have shown that gut microbiome disturbances can affect neural function and behaviors via the microbiota-gut-brain axis, and may be implicated in the pathogenesis of several brain diseases. However, exactly how the gut microbiome modulates nervous system activity remains obscure. Here, using a single-cell nucleus sequencing approach, we sought to characterize the cell type-specific transcriptomic changes in the prefrontal cortex and hippocampus derived from germ-free (GF), specific pathogen free, and colonized-GF mice. We found that the absence of gut microbiota resulted in cell-specific transcriptomic changes. Furthermore, microglia transcriptomes were preferentially influenced, which could be effectively reversed by microbial colonization. Significantly, the gut microbiome modulated the mutual transformation of microglial subpopulations in the two regions. Cross-species analysis showed that the transcriptome changes of these microglial subpopulations were mainly associated with Alzheimer's disease (AD) and major depressive disorder (MDD), which were further supported by animal behavioral tests. Our findings demonstrate that gut microbiota mainly modulate the mutual transformation of microglial subtypes, which may lead to new insights into the pathogenesis of AD and MDD.

[Source](https://www.nature.com/articles/s41380-023-02017-y)

10.1038/s41380-023-02017-y

---

## Multi-omics insights into the interplay between gut microbiota and colorectal cancer in the "microworld" age.
 14.03.2023, _AJ Wang, D Song, YM Hong, NN Liu_


Colorectal cancer (CRC) is a multifactorial heterogeneous disease largely due to both genetic predisposition and environmental factors including the gut microbiota, a dynamic microbial ecosystem inhabiting the gastrointestinal tract. Elucidation of the molecular mechanisms by which the gut microbiota interacts with the host may contribute to the pathogenesis, diagnosis, and promotion of CRC. However, deciphering the influence of genetic variants and interactions with the gut microbial ecosystem is rather challenging. Despite recent advancements in single omics analysis, the application of multi-omics approaches to integrate multiple layers of information in the microbiome and host to introduce effective prevention, diagnosis, and treatment strategies is still in its infancy. Here, we integrate host- and microbe-based multi-omics studies, respectively, to provide a strategy to explore potential causal relationships between gut microbiota and colorectal cancer. Specifically, we summarize the recent multi-omics studies such as metagenomics combined with metabolomics and metagenomics combined with genomics. Meanwhile, the sample size and sample types commonly used in multi-omics research, as well as the methods of data analysis, were also generalized. We highlight multiple layers of information from multi-omics that need to be verified by different types of models. Together, this review provides new insights into the clinical diagnosis and treatment of colorectal cancer patients.

[Source](https://pubs.rsc.org/en/content/articlelanding/2023/MO/D2MO00288D)

10.1039/d2mo00288d

---

## A Longitudinal Characterization of the Seminal Microbiota and Antibiotic Resistance in Yearling Beef Bulls Subjected to Different Rates of Gain.
 14.03.2023, _EM Webb, DB Holman, KN Schmidt, MS Crouse, CR Dahlen, RA Cushman, AP Snider, KL McCarthy, S Amat_


In this study, we evaluated the seminal and fecal microbiota in yearling beef bulls fed a common diet to achieve moderate (1.13 kg/day) or high (1.80 kg/day) rates of weight gain. Semen samples were collected on days 0 and 112 of dietary intervention (n = 19/group) as well as postbreeding (n = 6/group) using electroejaculation, and the microbiota was assessed using 16S rRNA gene sequencing, quantitative PCR (qPCR), and culturing. The fecal microbiota was also evaluated, and its similarity with seminal microbiota was assessed. A subset of seminal bacterial isolates (n = 33) was screened for resistance against 28 antibiotics. A complex and dynamic microbiota was detected in bovine semen, and the community structure was affected by sampling time (R2 = 0.16, P &lt; 0.001). Microbial richness increased significantly from day 0 to day 112, and diversity increased after breeding (P &gt; 0.05). Seminal microbiota remained unaffected by the differential rates of gain, and its overall composition was distinct from fecal microbiota, with only 6% of the taxa shared between them. A total of 364 isolates from 49 different genera were recovered under aerobic and anaerobic culturing. Among these seminal isolates were pathogenic species and those resistant to several antibiotics. Overall, our results suggest that bovine semen harbors a rich and complex microbiota which changes over time and during the breeding season but appears to be resilient to differential gains achieved via a common diet. Seminal microbiota is distinct from the fecal microbiota and harbors potentially pathogenic and antibiotic-resistant bacterial species. IMPORTANCE Increasing evidence from human and other animal species supports the existence of a commensal microbiota in semen and that this seminal microbiota may influence not only sperm quality and fertility but also female reproduction. Seminal microbiota in bulls and its evolution and factors shaping this community, however, remain largely underexplored. In this study, we characterized the seminal microbiota of yearling beef bulls and its response to the bull age, different weight gains, and mating activity. We compared bacterial composition between seminal and fecal microbiota and evaluated the diversity of culturable seminal bacteria and their antimicrobial resistance. Our results obtained from sequencing, culturing, and antibiotic susceptibility testing provide novel information on the taxonomic composition, evolution, and factors shaping the seminal microbiota of yearling beef bulls. This information will serve as an important basis for further understanding of the seminal microbiome and its involvement in reproductive health and fertility in cattle.

[Source](https://journals.asm.org/doi/10.1128/spectrum.05180-22)

10.1128/spectrum.05180-22

---

## Abiotic selection of microbial genome size in the global ocean.
 17.03.2023, _DK Ngugi, SG Acinas, P SÃ¡nchez, JM Gasol, S Agusti, DM Karl, CM Duarte_


_Genome Size, Oceans and Seas, Genome, Microbial, Metagenome, Microbiota, Seawater_
Strong purifying selection is considered a major evolutionary force behind small microbial genomes in the resource-poor photic ocean. However, very little is currently known about how the size of prokaryotic genomes evolves in the global ocean and whether patterns reflect shifts in resource availability in the epipelagic and relatively stable deep-sea environmental conditions. Using 364 marine microbial metagenomes, we investigate how the average genome size of uncultured planktonic prokaryotes varies across the tropical and polar oceans to the hadal realm. We find that genome size is highest in the perennially cold polar ocean, reflecting elongation of coding genes and gene dosage effects due to duplications in the interior ocean microbiome. Moreover, the rate of change in genome size due to temperature is 16-fold higher than with depth up to 200 m. Our results demonstrate how environmental factors can influence marine microbial genome size selection and ecological strategies of the microbiome.

[Source](https://www.nature.com/articles/s41467-023-36988-x)

10.1038/s41467-023-36988-x

---

## Benchmarking genome assembly methods on metagenomic sequencing data.
 14.03.2023, _Z Zhang, C Yang, WP Veldsman, X Fang, L Zhang_


Metagenome assembly is an efficient approach to reconstruct microbial genomes from metagenomic sequencing data. Although short-read sequencing has been widely used for metagenome assembly, linked- and long-read sequencing have shown their advancements in assembly by providing long-range DNA connectedness. Many metagenome assembly tools were developed to simplify the assembly graphs and resolve the repeats in microbial genomes. However, there remains no comprehensive evaluation of metagenomic sequencing technologies, and there is a lack of practical guidance on selecting the appropriate metagenome assembly tools. This paper presents a comprehensive benchmark of 19 commonly used assembly tools applied to metagenomic sequencing datasets obtained from simulation, mock communities or human gut microbiomes. These datasets were generated using mainstream sequencing platforms, such as Illumina and BGISEQ short-read sequencing, 10x Genomics linked-read sequencing, and PacBio and Oxford Nanopore long-read sequencing. The assembly tools were extensively evaluated against many criteria, which revealed that long-read assemblers generated high contig contiguity but failed to reveal some medium- and high-quality metagenome-assembled genomes (MAGs). Linked-read assemblers obtained the highest number of overall near-complete MAGs from the human gut microbiomes. Hybrid assemblers using both short- and long-read sequencing were promising methods to improve both total assembly length and the number of near-complete MAGs. This paper also discussed the running time and peak memory consumption of these assembly tools and provided practical guidance on selecting them.

[Source](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbad087/7077274)

10.1093/bib/bbad087

---

