## BugSigDB - a database for identifying unusual abundance patterns in human microbiome studies.
 11.09.2023, __



[Source](https://bugsigdb.org/Main_Page)

10.1038/s41587-023-01930-5

---

##  BugSigDB

[Source](https://bugsigdb.org/Main_Page)

---

## Computational exploration of the global microbiome for antibiotic discovery.
 15.09.2023, _CD Santos-Júnior, MDT Torres, Y Duan, Á Rodríguez Del Río, TSB Schmidt, H Chong, A Fullam, K Michael, C Zhu, A Houseman, J Somborski, A Vines, XM Zhao, P Bork, J Huerta-Cepas, C de la Fuente-Nunez, LP Coelho_


Novel antibiotics are urgently needed to combat the antibiotic-resistance crisis. We present a machine learning-based approach to predict prokaryotic antimicrobial peptides (AMPs) by leveraging a vast dataset of 63,410 metagenomes and 87,920 microbial genomes. This led to the creation of AMPSphere, a comprehensive catalog comprising 863,498 non-redundant peptides, the majority of which were previously unknown. We observed that AMP production varies by habitat, with animal-associated samples displaying the highest proportion of AMPs compared to other habitats. Furthermore, within different human-associated microbiota, strain-level differences were evident. To validate our predictions, we synthesized and experimentally tested 50 AMPs, demonstrating their efficacy against clinically relevant drug-resistant pathogens both in vitro and in vivo. These AMPs exhibited antibacterial activity by targeting the bacterial membrane. Additionally, AMPSphere provides valuable insights into the evolutionary origins of peptides. In conclusion, our approach identified AMP sequences within prokaryotic microbiomes, opening up new avenues for the discovery of antibiotics.

[Source](https://ampsphere.big-data-biology.org/home)

10.1101/2023.08.31.555663

---

## AMPSphere

[Source](https://ampsphere.big-data-biology.org/home)

---

## online conferencing and how to do it better

[Source](http://[https://www.nature.com/articles/d41586-023-02507-7])

---

## Mechanism of Dyspnea during Exercise in Children with Corrected Congenital Heart Disease.
 24.01.2022, _M Chlif, MM Ammar, NB Said, L Sergey, S Ahmaidi, F Alassery, H Hamam_


_Child, Dyspnea, Exercise, Exercise Test, Heart Defects, Congenital, Humans, Oxygen Consumption_
This study will evaluate cardiorespiratory and peripheral muscle function and their relationship with subjective dyspnea threshold after the surgical correction of congenital heart disease in children. Thirteen children with surgically repaired congenital heart disease were recruited. Each participant performed an incremental exercise test on a cycle ergometer until exhaustion. Gas exchanges were continuously sampled to measure the maximal aerobic parameters and ventilatory thresholds. The functional capacity of the subjects was assessed with a 6 min walk test. At the end of the exercise test, isokinetic Cybex Norm was used to evaluate the strength and endurance of the knee extensor muscle in the leg. Dyspnea was subjectively scored with a visual analog scale during the last 15 s of each exercise step. Oxygen consumption measured at the dyspnea score/VO2 relationship located at the dyspnea threshold, at which dyspnea suddenly increased. Results: The maximal and submaximal values of the parameters describing the exercise and the peripheral muscular performances were: VO2 Peak: 33.8 ± 8.9 mL·min-1·kg-1; HR: 174 ± 9 b·min-1; VEmax: 65.68 ± 15.9 L·min-1; P max: 117 ± 27 W; maximal voluntary isometric force MVIF: 120.8 ± 41.9 N/m; and time to exhaustion Tlim: 53 ± 21 s. Oxygen consumption measured at the dyspnea threshold was related to VO2 Peak (R2 = 0.74; p &lt; 0.01), Tlim (R2 = 0.78; p &lt; 0.01), and the distance achieved during the 6MWT (R2 = 0.57; p &lt; 0.05). Compared to the theoretical maximal values for the power output, VO2, and HR, the surgical correction did not repair the exercise performance. After the surgical correction of congenital heart disease, exercise performance was impeded by alterations of the cardiorespiratory function and peripheral local factors. A subjective evaluation of the dyspnea threshold is a reliable criterion that allows the prediction of exercise capacity in subjects suffering from congenital heart disease.

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

10.3390/ijerph19010099

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/mf96F35ETY

[Source](https://t.co/mf96F35ETY)

---

## @conTaminatedsci

[Source](https://twitter.com/conTaminatedsci?ref_src=twsrc%5Etfw)

---

## @HillLabSask

[Source](https://twitter.com/HillLabSask?ref_src=twsrc%5Etfw)

---

## @IlianaSerghiou

[Source](https://twitter.com/IlianaSerghiou?ref_src=twsrc%5Etfw)

---

## @SalamMicrobes

[Source](https://twitter.com/SalamMicrobes?ref_src=twsrc%5Etfw)

---

## @DegraeveAl

[Source](https://twitter.com/DegraeveAl?ref_src=twsrc%5Etfw)

---

## Extracellular Vesicles and Endocannabinoid Signaling in Patients with COVID-19.
 15.09.2023, _F Brandes, AM Keiler, B Kirchner, M Borrmann, JN Billaud, M Reithmair, M Klein, P Campolongo, D Thieme, MW Pfaffl, G Schelling, AS Meidert_


Introduction: Endocannabinoids in COVID-19 have immunomodulatory and anti-inflammatory properties but the functional role and the regulation of endocannabinoid signaling in this pandemic disorder is controversial. To exercise their biologic function, endocannabinoids need to travel across the intercellular space and within the blood stream to reach their target cells. How the lipophilic endocannabinoids are transported in the vascular system and how these hydrophobic compounds cross cell membranes is still unclear. Extracellular vesicles (EVs) are released and incorporated by many cell types including immune cells. EVs are small lipid-membrane covered particles and contain RNA, lipids and proteins. They play an important role in intercellular communication by transporting these signaling molecules from their cells of origin to specific target cells. EVs may represent ideal transport vehicles for lipophilic signaling molecules like endocannabinoids and this effect could also be evident in COVID-19. Materials and Methods: We measured the endocannabinoids anandamide, 2-AG, SEA, PEA and OEA in patients with COVID-19 in EVs and plasma. RNA sequencing of microRNAs (miRNAs) derived from EVs (EV-miRNAs) and mRNA transcripts from blood cells was used for the construction of signaling networks reflecting endocannabinoid and miRNA communication by EVs to target immune cells. Results: With the exception of anandamide, endocannabinoid concentrations were significantly enriched in EVs in comparison to plasma and increased with disease severity. No enrichment in EVs was seen for the more hydrophilic steroid hormones cortisol and testosterone. High EV-endocannabinoid concentrations were associated with downregulation of CNR2 (CB2) by upregulated EV-miRNA miR-146a-5p and upregulation of MGLL by downregulated EV-miR-199a-5p and EV-miR-370-5p suggesting counterregulatory effects. In contrast, low EV-levels of anandamide were associated with upregulation of CNR1 by downregulation of EV-miR-30c-5p and miR-26a-5p along with inhibition of FAAH. Immunologically active molecules in immune cells regulated by endocannabinoid signaling included VEGFA, GNAI2, IGF1, BDNF, IGF1R and CREB1 and CCND1 among others. Discussion and Conclusions: EVs carry immunologically functional endocannabinoids in COVID-19 along with miRNAs which may regulate the expression of mRNA transcripts involved in the regulation of endocannabinoid signaling and metabolism. This mechanism could fine-tune and adapt endocannabinoid effects in recipient cells in relationship to the present biological context.

[Source](https://twitter.com/QIAGEN?ref_src=twsrc%5Etfw)

10.1089/can.2023.0040

---

## @QIAGEN

[Source](https://twitter.com/QIAGEN?ref_src=twsrc%5Etfw)

---

## @NSURPorg

[Source](https://twitter.com/NSURPorg?ref_src=twsrc%5Etfw)

---

## pic.twitter.com/l0Sz0N2745

[Source](https://t.co/l0Sz0N2745)

---

## September 11, 2023

[Source](https://twitter.com/MicrobiomeVIF/status/1701235383210508355?ref_src=twsrc%5Etfw)

---

## The landscape of antibiotic resistance genes in the nasopharynx of preterm infants: Prolonged signature of hospitalization and effects by antibiotics

[Source](https://www.biorxiv.org/content/10.1101/2023.08.09.552435v2)

---

## Integrating current analyses of the breast cancer microbiome.
 15.09.2023, _S Sohail, MB Burns_


_Humans, Cancer Survivors, Databases, Factual, Microbiota, Problem Solving, Research Design, Neoplasms_
Many cancer types have significant associations with their resident microbial communities-emerging evidence suggests that breast cancers also interact with the local tissue-associated microbiota. Microbiome research advances rapidly and analysis pipelines and databases are updated frequently. This dynamic environment makes comparative evaluations challenging. Here, we have integrated all publicly available studies related to breast cancer and the mammary microbiome in light of advances in this rapidly progressing field. Based on alpha diversity, beta diversity, proportional abundance, and statistical analyses, we observed differences between our modern analytical approaches and the original findings. We were able to classify and identify additional taxa across samples through abundance analyses and identify previously unidentified statistically significant taxa. In our updated analyses there were more taxa identified as statistically significant in comparison to the original studies' results. In the re-analysis for The Microbiome of Aseptically Collected Human Breast Tissue in Benign and Malignant Disease by Hieken et al., there were twelve statistically significant differentially abundant taxa identified in breast tissue microbiota in benign and invasive cancer disease states. In the re-analysis for The Microbiota of Breast Tissue and Its Association with Breast Cancer by Urbaniak et al., there were 18 taxa identified as statistically significant. In the re-analysis for Characterization of the microbiome of nipple aspirate fluid of breast cancer survivors by Chan et al., there were three genera identified as statistically significant in the skin and fluid samples. Our work has discovered that reanalyses are necessary for microbiome studies, especially older 16S studies. Through our re-analysis, we classified and identified more phyla and genera across studies, which supports the notion that reanalyses provide new insights to the microbiome field and help to assess robusticity of previously published findings by using new and updated tools and databases.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0291320)

10.1371/journal.pone.0291320

---

## Diet influences community dynamics following vaginal group B <em>Streptococcus</em> colonization 

[Source](https://www.biorxiv.org/content/biorxiv/early/2023/09/10/2023.09.10.556920.full.pdf)

---

## Gut microbes exacerbate systemic inflammation and behavior disorders in neurologic disease CADASIL.
 11.09.2023, _S Liu, X Men, Y Guo, W Cai, R Wu, R Gao, W Zhong, H Guo, H Ruan, S Chou, J Mai, S Ping, C Jiang, H Zhou, X Mou, W Zhao, Z Lu_


_Animals, Mice, CADASIL, Gastrointestinal Microbiome, Mental Disorders, Cytokines, gamma-Aminobutyric Acid_
Cerebral autosomal dominant arteriopathy with subcortical infarcts and leukoencephalopathy (CADASIL) is a cerebral small vessel disease that carries mutations in NOTCH3. The clinical manifestations are influenced by genetic and environmental factors that may include gut microbiome.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01638-3)

10.1186/s40168-023-01638-3

---

## Exploring novel alkane-degradation pathways in uncultured bacteria from the North Atlantic Ocean.
 13.09.2023, _M Vázquez Rosas Landa, V De Anda, RR Rohwer, A Angelova, G Waldram, T Gutierrez, BJ Baker_


Microbes play a significant role in the degradation of petroleum hydrocarbons in the oceans, yet little is known about the native bacteria that metabolize hydrocarbons before an oil spill. The Faroe-Shetland Channel (FSC) is a deepwater subarctic region of the North Atlantic with prominent oil production and a diverse microbial community associated with the degradation of petroleum. Here, we combine DNA-based stable-isotope probing (DNA-SIP) with metagenomics to elucidate the metabolic underpinnings of native alkane-degrading bacteria from the FSC. From two 13C n-hexadecane SIP experiments using seawater from 5 and 700 m depths in the FSC, we obtained 42 metagenome-assembled genomes (MAGs) belonging to 19 genera, including two previously overlooked hydrocarbon-degrading bacteria, Lentibacter (Alphaproteobacteria) and Dokdonia (Bacteroidetes). Diversity surveys indicated Lentibacter were dominant members of the FSC, constituting up to 17% of these communities. Many of the SIP-enriched MAGs (20/42) encoded a complete alkane oxidation pathway, including alkane monooxygenase (AlkB), rubredoxin reductase (AlkT), and rubredoxin-2 (AlkG). Fourteen Aphaproteobacteria MAGs lacked AlkG for electron transfer. Instead, they encoded novel disulfide isomerases with iron-binding cysteine motifs conserved across rubredoxins. Dokdonia lacked AlkT and AlkG, however, their central alkane-degradation catabolic pathways were complete. We describe previously unrecognized bacteria capable of hydrocarbon degradation, including the dominant genera Lentibacter, which may continuously purge hydrocarbons released from oil exploration activities in the FSC. This advances the understanding of the diversity and physiologies of alkane degradation in the North Atlantic and provides evidence of new mechanisms used to metabolize alkanes. IMPORTANCE Petroleum pollution in the ocean has increased because of rapid population growth and modernization, requiring urgent remediation. Our understanding of the metabolic response of native microbial communities to oil spills is not well understood. Here, we explored the baseline hydrocarbon-degrading communities of a subarctic Atlantic region to uncover the metabolic potential of the bacteria that inhabit the surface and subsurface water. We conducted enrichments with a 13C-labeled hydrocarbon to capture the fraction of the community actively using the hydrocarbon. We then combined this approach with metagenomics to identify the metabolic potential of this hydrocarbon-degrading community. This revealed previously undescribed uncultured bacteria with unique metabolic mechanisms involved in aerobic hydrocarbon degradation, indicating that temperature may be pivotal in structuring hydrocarbon-degrading baseline communities. Our findings highlight gaps in our understanding of the metabolic complexity of hydrocarbon degradation by native marine microbial communities.

[Source](https://journals.asm.org/doi/10.1128/msystems.00619-23)

10.1128/msystems.00619-23

---

## Plastics shape the black soldier fly larvae gut microbiome and select for biodegrading functions.
 15.09.2023, _F De Filippis, M Bonelli, D Bruno, G Sequino, A Montali, M Reguzzoni, E Pasolli, D Savy, S Cangemi, V Cozzolino, G Tettamanti, D Ercolini, M Casartelli, S Caccia_


_Animals, Larva, Gastrointestinal Microbiome, Plastics, RNA, Ribosomal, 16S, Diptera_
In the last few years, considerable attention has been focused on the plastic-degrading capability of insects and their gut microbiota in order to develop novel, effective, and green strategies for plastic waste management. Although many analyses based on 16S rRNA gene sequencing are available, an in-depth analysis of the insect gut microbiome to identify genes with plastic-degrading potential is still lacking.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01649-0)

10.1186/s40168-023-01649-0

---

## Enhancing Recovery from Gut Microbiome Dysbiosis and Alleviating DSS-Induced Colitis in Mice with a Consortium of Rare Short-Chain Fatty Acid-Producing Bacteria

[Source](https://www.biorxiv.org/content/10.1101/2023.09.11.556543v1)

---

## BugSigDB captures patterns of differential abundance across a broad range of host-associated microbial signatures.
 11.09.2023, _L Geistlinger, C Mirzayi, F Zohra, R Azhar, S Elsafoury, C Grieve, J Wokaty, SD Gamboa-Tuz, P Sengupta, I Hecht, A Ravikrishnan, RS Gonçalves, E Franzosa, K Raman, V Carey, JB Dowd, HE Jones, S Davis, N Segata, C Huttenhower, L Waldron_


The literature of human and other host-associated microbiome studies is expanding rapidly, but systematic comparisons among published results of host-associated microbiome signatures of differential abundance remain difficult. We present BugSigDB, a community-editable database of manually curated microbial signatures from published differential abundance studies accompanied by information on study geography, health outcomes, host body site and experimental, epidemiological and statistical methods using controlled vocabulary. The initial release of the database contains &gt;2,500 manually curated signatures from &gt;600 published studies on three host species, enabling high-throughput analysis of signature similarity, taxon enrichment, co-occurrence and coexclusion and consensus signatures. These data allow assessment of microbiome differential abundance within and across experimental conditions, environments or body sites. Database-wide analysis reveals experimental conditions with the highest level of consistency in signatures reported by independent studies and identifies commonalities among disease-associated signatures, including frequent introgression of oral pathobionts into the gut.

[Source](https://www.nature.com/articles/s41587-023-01872-y)

10.1038/s41587-023-01872-y

---

## Computational exploration of the global microbiome for antibiotic discovery.
 15.09.2023, _CD Santos-Júnior, MDT Torres, Y Duan, Á Rodríguez Del Río, TSB Schmidt, H Chong, A Fullam, K Michael, C Zhu, A Houseman, J Somborski, A Vines, XM Zhao, P Bork, J Huerta-Cepas, C de la Fuente-Nunez, LP Coelho_


Novel antibiotics are urgently needed to combat the antibiotic-resistance crisis. We present a machine learning-based approach to predict prokaryotic antimicrobial peptides (AMPs) by leveraging a vast dataset of 63,410 metagenomes and 87,920 microbial genomes. This led to the creation of AMPSphere, a comprehensive catalog comprising 863,498 non-redundant peptides, the majority of which were previously unknown. We observed that AMP production varies by habitat, with animal-associated samples displaying the highest proportion of AMPs compared to other habitats. Furthermore, within different human-associated microbiota, strain-level differences were evident. To validate our predictions, we synthesized and experimentally tested 50 AMPs, demonstrating their efficacy against clinically relevant drug-resistant pathogens both in vitro and in vivo. These AMPs exhibited antibacterial activity by targeting the bacterial membrane. Additionally, AMPSphere provides valuable insights into the evolutionary origins of peptides. In conclusion, our approach identified AMP sequences within prokaryotic microbiomes, opening up new avenues for the discovery of antibiotics.

[Source](https://www.biorxiv.org/content/biorxiv/early/2023/09/11/2023.08.31.555663.full.pdf)

10.1101/2023.08.31.555663

---

## https://t.co/r7E8jWQEjK

[Source](https://t.co/r7E8jWQEjK)

---

## Metrics of Anterior Sclera in Normal Chinese Adults: Anterior Segment Imaging Using the Swept-Source Optical Coherence Tomography.
 15.09.2023, _N Lin, Y Sun, M Yang, B Chen, J Mao, Y Liu, R Deng_


To measure the corneoscleral limbus and anterior sclera parameters of normal Chinese adults by swept-source optical coherence tomography (OCT).

[Source](https://twitter.com/svetlana_up/status/1699086351348515120?ref_src=twsrc%5Etfw)

10.1097/ICL.0000000000001032

---

## September 5, 2023

[Source](https://twitter.com/svetlana_up/status/1699086351348515120?ref_src=twsrc%5Etfw)

---

