## Delivery mode impacts gut bacteriophage colonization during infancy.
 28.11.2023, _P Subramanian, HN Romero-Soto, DB Stern, GL Maxwell, S Levy, SK Hourigan_


Cesarean section delivery is associated with altered early-life bacterial colonization and later adverse inflammatory and immune health outcomes. Although gut bacteriophages can alter gut microbiome composition and impact host immune responses, little is known about how delivery mode impacts bacteriophage colonization over time. To begin to address this we examined how delivery mode affected bacteriophage colonization over the first two years of life.

[Source](https://www.medrxiv.org/content/10.1101/2023.11.13.23298307v1)

10.1101/2023.11.13.23298307

---

## Oral microbiota analyses of paediatric Saudi population reveals signatures of dental caries.
 29.11.2023, _YM Alyousef, S Piotrowski, FA Alonaizan, A Alsulaiman, AA Alali, NN Almasood, C Vatte, L Hamilton, D Gandla, H Lad, FL Robinson, C Cyrus, RC Meng, A Dowdell, B Piening, BJ Keating, AK Al-Ali_


_Male, Child, Female, Humans, Dental Caries, Saudi Arabia, RNA, Ribosomal, 16S, Microbiota, Saliva_
Oral microbiome sequencing has revealed key links between microbiome dysfunction and dental caries. However, these efforts have largely focused on Western populations, with few studies on the Middle Eastern communities. The current study aimed to identify the composition and abundance of the oral microbiota in saliva samples of children with different caries levels using machine learning approaches.

[Source](https://bmcoralhealth.biomedcentral.com/articles/10.1186/s12903-023-03448-3)

10.1186/s12903-023-03448-3

---

## Gastric coinfection with thiopeptide-positive Cutibacterium acnes decreases FOXM1 and pro-inflammatory biomarker expression in a murine model of Helicobacter pylori-induced gastric cancer

[Source](https://journals.asm.org/doi/10.1128/spectrum.03450-23)

---

## Butterflies in the gut: the interplay between intestinal microbiota and stress.
 29.11.2023, _TT Lai, CW Liou, YH Tsai, YY Lin, WL Wu_


_Animals, Humans, Gastrointestinal Microbiome, Butterflies_
Psychological stress is a global issue that affects at least one-third of the population worldwide and increases the risk of numerous psychiatric disorders. Accumulating evidence suggests that the gut and its inhabiting microbes may regulate stress and stress-associated behavioral abnormalities. Hence, the objective of this review is to explore the causal relationships between the gut microbiota, stress, and behavior. Dysbiosis of the microbiome after stress exposure indicated microbial adaption to stressors. Strikingly, the hyperactivated stress signaling found in microbiota-deficient rodents can be normalized by microbiota-based treatments, suggesting that gut microbiota can actively modify the stress response. Microbiota can regulate stress response via intestinal glucocorticoids or autonomic nervous system. Several studies suggest that gut bacteria are involved in the direct modulation of steroid synthesis and metabolism. This review provides recent discoveries on the pathways by which gut microbes affect stress signaling and brain circuits and ultimately impact the host's complex behavior.

[Source](https://jbiomedsci.biomedcentral.com/articles/10.1186/s12929-023-00984-6)

10.1186/s12929-023-00984-6

---

## Genetics, Nutrition, and Health: A New Frontier in Disease Prevention.
 28.11.2023, _P Agrawal, J Kaur, J Singh, P Rasane, K Sharma, V Bhadariya, S Kaur, V Kumar_


The field of nutrition research has traditionally focused on the effects of macronutrients and micronutrients on the body. However, it has become evident that individuals have unique genetic makeups that influence their response to food. Nutritional genomics, which includes nutrigenetics and nutrigenomics, explores the interaction between an individual's genetic makeup, diet, and health outcomes. Nutrigenetics studies the impact of genetic variation on an individual's response to dietary nutrients, while nutrigenomics investigates how dietary components affect gene regulation and expression. These disciplines seek to understand the impact of diet on the genome, transcriptome, proteome, and metabolome. It provides insights into the mechanisms underlying the effect of diet on gene expression. Nutrients can cause the modification of genetic expression through epigenetic changes, such as DNA methylation and histone modifications. The aim of nutrigenomics is to create personalized diets based on the unique metabolic profile of an individual, gut microbiome, and genetic makeup to prevent diseases and promote health. Nutrigenomics has the potential to revolutionize the field of nutrition by combining the practicality of personalized nutrition with knowledge of genetic factors underlying health and disease. Thus, nutrigenomics offers a promising approach to improving health outcomes (in terms of disease prevention) through personalized nutrition strategies based on an individual's genetic and metabolic characteristics.

[Source](https://www.tandfonline.com/doi/abs/10.1080/27697061.2023.2284997)

10.1080/27697061.2023.2284997

---

## Age-dependent changes in the gut microbiota and serum metabolome correlate with renal function and human aging.
 28.11.2023, _L Sun, Z Li, C Hu, J Ding, Q Zhou, G Pang, Z Wu, R Yang, S Li, J Li, J Cai, Y Sun, R Li, H Zhen, S Sun, J Zhang, M Fang, Z Chen, Y Lv, Q Cao, Y Sun, R Gong, Z Huang, Y Duan, H Liu, J Dong, J Li, J Ruan, H Lu, B He, N Li, T Li, W Xue, Y Li, J Shen, F Yang, C Zhao, Q Liang, M Zhang, C Chen, H Gong, Y Hou, J Wang, Y Zhang, H Yang, S Zhu, L Xiao, Z Jin, H Guo, P Zhao, S Brix, X Xu, H Jia, K Kristiansen, Z Yang, C Nie_


Human aging is invariably accompanied by a decline in renal function, a process potentially exacerbated by uremic toxins originating from gut microbes. Based on a registered household Chinese Guangxi longevity cohort (n = 151), we conducted comprehensive profiling of the gut microbiota and serum metabolome of individuals from 22 to 111 years of age and validated the findings in two independent East Asian aging cohorts (Japan aging cohort n = 330, Yunnan aging cohort n = 80), identifying unique age-dependent differences in the microbiota and serum metabolome. We discovered that the influence of the gut microbiota on serum metabolites intensifies with advancing age. Furthermore, mediation analyses unveiled putative causal relationships between the gut microbiota (Escherichia coli, Odoribacter splanchnicus, and Desulfovibrio piger) and serum metabolite markers related to impaired renal function (p-cresol, N-phenylacetylglutamine, 2-oxindole, and 4-aminohippuric acid) and aging. The fecal microbiota transplantation experiment demonstrated that the feces of elderly individuals could influence markers related to impaired renal function in the serum. Our findings reveal novel links between age-dependent alterations in the gut microbiota and serum metabolite markers of impaired renal function, providing novel insights into the effects of microbiota-metabolite interplay on renal function and healthy aging.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/acel.14028)

10.1111/acel.14028

---

## The changes of gut microbiota and metabolites in different drug-induced liver injuries.
 29.11.2023, _R Zhu, Y Gao, J Dong, Z Li, Z Ren_


_Humans, Gastrointestinal Microbiome, Chemical and Drug Induced Liver Injury, Multiomics_
The increasing incidence of drug-induced liver injury (DILI) has become a major concern. Gut microbiota, as another organ of the human body, has been studied in various tumors, cardiovascular metabolic diseases, inflammatory bowel disease and human immunity. The studies mentioned above have confirmed its important impact on the occurrence and development of DILI. The gut-liver axis explains the close relationship between the gut and the liver, and it may be a pathway by which gut microbes contribute to DILI. In addition, the interaction between drugs and gut microbes affects both separately, which in turn may have positive or negative effects on the body, including DILI. There are both common and specific changes in liver injury caused by different drugs. The alteration of metabolites in DILI is also a new direction of therapeutic exploration. The application of microbiomics, metabolomics and other multi-omics to DILI has also explored new ideas for DILI. In this review, we conclude the alterations of gut microbes and metabolites under different DILI, and the significance of applying gut microbiome-metabolomics to DILI, so as to explore the metabolic characteristics of DILI and possible novel metabolic biomarkers.

[Source](https://www.microbiologyresearch.org/content/journal/jmm/10.1099/jmm.0.001778)

10.1099/jmm.0.001778

---

## Comparison of the Activity of Fecal Enzymes and Concentration of SCFA in Healthy and Overweight Children.
 01.03.2023, _K Śliżewska, M Włodarczyk, M Sobczak, R Barczyńska, J Kapuśniak, P Socha, A Wierzbicka-Rucińska, A Kotowska_


_Child, Humans, alpha-Galactosidase, alpha-Glucosidases, Cellulases, Fatty Acids, Volatile, Feces, Glucuronidase, Overweight, Pediatric Obesity_
In modern societies obesity has become a serious issue which must be urgently addressed. The health implications of neglected obesity are substantial, as not only does it affect individuals' everyday lives, but it also leads to significantly increased mortality due to the development of several disorders such as type-2 diabetes, cardiovascular diseases, cancers, and depression. The objective of this research was to investigate the alterations in selected health markers caused by overweight and obesity in children. The measured parameters were the activity of the fecal enzymes, the concentration of short-chain fatty acids (SCFAs), and the concentration of branched-chain fatty acids (BCFAs). The activity of the fecal enzymes, specifically α-glucosidase, α-galactosidase, β-glucosidase, β-galactosidase, and β-glucuronidase, was determined using spectrophotometry at a wavelength of 400 nm. Furthermore, concentrations of lactic acid, SCFAs (formic, acetic, propionic, butyric, and valeric acids), and BCFAs (isobutyric and isovaleric acids) were determined using the HPLC method. The obtained results reveal that obese children have different fecal enzyme activity and a different profile of fatty acids from children of normal weight. The group of obese children, when compared to children of normal weight, had increased concentrations of BCFAs (p &lt; 0.05) and higher activity of potentially harmful enzymes such as β-glucosidase and β-glucuronidase (p &lt; 0.05). In comparison, children of normal weight exhibited significantly increased concentrations of lactic acid and SCFAs (especially formic and butyric acids) (p &lt; 0.05). Furthermore, their α-glucosidase and α-galactosidase activity were higher when compared to the group of obese children (p &lt; 0.05). These results suggest that the prevalence of obesity has a significant impact on metabolites produced in the gastrointestinal tract, which might result in a higher chance of developing serious diseases.

[Source](https://www.biorxiv.org/content/10.1101/2023.11.27.568948v1.abstract)

10.3390/nu15040987

---

## The differing prevalences of propionate and butyrate producing bacteria in the human gut microbiota

[Source](https://www.biorxiv.org/content/10.1101/2023.11.27.568948v1.abstract)

---

## The Parkinson's drug entacapone disrupts gut microbiome homeostasis via iron sequestration.
 28.11.2023, _FC Pereira, X Ge, JM Kristensen, RH Kirkegaard, K Maritsch, Y Zhu, M Decorte, B Hausmann, D Berry, K Wasmund, A Schintlmeister, T Boettcher, JX Cheng, M Wagner_


Increasing evidence shows that many human-targeted drugs alter the gut microbiome, leading to implications for host health. However, much less is known about the mechanisms by which drugs target the microbiome and how drugs affect microbial function. Here we combined quantitative microbiome profiling, long-read metagenomics, stable isotope probing and single cell chemical imaging to investigate the impact of two widely prescribed nervous system targeted drugs on the gut microbiome. Ex vivo supplementation of physiologically relevant concentrations of entacapone or loxapine succinate to faecal samples significantly impacted the abundance of up to one third of the microbial species present. Importantly, we demonstrate that the impact of these drugs on microbial metabolism is much more pronounced than their impact on abundances, with low concentrations of drugs reducing the activity, but not the abundance of key microbiome members like Bacteroides, Ruminococcus or Clostridium species. We further demonstrate that entacapone impacts the microbiome due to its ability to complex and deplete available iron, and that microbial growth can be rescued by replenishing levels of microbiota-accessible iron. Remarkably, entacapone-induced iron starvation selected for iron-scavenging organisms carrying antimicrobial resistance and virulence genes. Collectively, our study unveils the impact of two under-investigated drugs on whole microbiomes and identifies metal sequestration as a mechanism of drug-induced microbiome disturbance.

[Source](https://www.biorxiv.org/content/10.1101/2023.11.12.566429v2)

10.1101/2023.11.12.566429

---

## Mutual modulation of gut microbiota and the immune system in type 1 diabetes models.
 29.11.2023, _E Rosell-Mases, A Santiago, M Corral-Pujol, F Yáñez, E Varela, L Egia-Mendikute, B Arpa, C Cosovanu, A Panosa, G Serrano-Gómez, C Mora, J Verdaguer, C Manichanh_


_Mice, Animals, Diabetes Mellitus, Type 1, Mice, Inbred NOD, Gastrointestinal Microbiome, B-Lymphocytes, B-Lymphocyte Subsets_
The transgenic 116C-NOD mouse strain exhibits a prevalent Th17 phenotype, and reduced type 1 diabetes (T1D) compared to non-obese diabetic (NOD) mice. A cohousing experiment between both models revealed lower T1D incidence in NOD mice cohoused with 116C-NOD, associated with gut microbiota changes, reduced intestinal permeability, shifts in T and B cell subsets, and a transition from Th1 to Th17 responses. Distinct gut bacterial signatures were linked to T1D in each group. Using a RAG-2-/- genetic background, we found that T cell alterations promoted segmented filamentous bacteria proliferation in young NOD and 116C-NOD, as well as in immunodeficient NOD.RAG-2-/- and 116C-NOD.RAG-2-/- mice across all ages. Bifidobacterium colonization depended on lymphocytes and thrived in a non-diabetogenic environment. Additionally, 116C-NOD B cells in 116C-NOD.RAG-2-/- mice enriched the gut microbiota in Adlercreutzia and reduced intestinal permeability. Collectively, these results indicate reciprocal modulation between gut microbiota and the immune system in rodent T1D models.

[Source](https://www.nature.com/articles/s41467-023-43652-x)

10.1038/s41467-023-43652-x

---

## Microbiota-dependent early life programming of gastrointestinal motility.
 28.11.2023, _ME Frith, PC Kashyap, DR Linden, B Theriault, EB Chang_


Gastrointestinal microbes modulate peristalsis and stimulate the enteric nervous system (ENS), whose development, as in the central nervous system (CNS), continues into the murine postweaning period. Given that adult CNS function depends on stimuli received during critical periods of postnatal development, we hypothesized that adult ENS function, namely motility, depends on microbial stimuli during similar critical periods. We gave fecal microbiota transplantation (FMT) to germ-free mice at weaning or as adults and found that only the mice given FMT at weaning recovered normal transit, while those given FMT as adults showed limited improvements. RNAseq of colonic muscularis propria revealed enrichments in neuron developmental pathways in mice exposed to gut microbes earlier in life, while mice exposed later - or not at all - showed exaggerated expression of inflammatory pathways. These findings highlight a microbiota-dependent sensitive period in ENS development, pointing to potential roles of the early life microbiome in later life dysmotility.

[Source](https://www.biorxiv.org/content/10.1101/2023.11.08.566304v1)

10.1101/2023.11.08.566304

---

## Horizontal transmission of gut microbiota attenuates mortality in lung fibrosis.
 28.11.2023, _SJ Gurczynski, JH Lipinski, JY Strauss, S Alam, GB Huffnagle, P Ranjan, LH Kennedy, BB Moore, DN O'Dwyer_


Pulmonary fibrosis is a chronic and often fatal disease. The pathogenesis is characterized by aberrant repair of lung parenchyma resulting in loss of physiological homeostasis, respiratory failure and death. The immune response in pulmonary fibrosis is dysregulated. The gut microbiome is a key regulator of immunity. The role of the gut microbiome in regulating the pulmonary immunity in lung fibrosis is poorly understood. Here, we determine the impact of gut microbiota on pulmonary fibrosis in C57BL/6 mice derived from different vendors (C57BL/6J and C57BL/6NCrl). We use germ free models, fecal microbiota transplantation and cohousing to transmit gut microbiota. Metagenomic studies of feces establish keystone species between sub-strains. Pulmonary fibrosis is microbiota dependent in C57BL/6 mice. Gut microbiota are distinct by β diversity (PERMANOVA P&lt;0.001) and α diversity (P&lt;0.0001). Mortality and lung fibrosis are attenuated in C57BL/6NCrl mice. Elevated CD4+ IL-10+ T cells and lower IL-6 occur in C57BL/6NCrl mice. Horizontal transmission of microbiota by cohousing attenuates mortality in C57BL/6J mice and promotes a transcriptionally altered pulmonary immunity. Temporal changes in lung and gut microbiota demonstrates that gut microbiota contribute largely to immunological phenotype. Key regulatory gut microbiota contribute to lung fibrosis generating rationale for human studies.

[Source](https://insight.jci.org/articles/view/164572)

10.1172/jci.insight.164572

---

## High-altitude and low-altitude adapted chicken gut-microbes have different functional diversity.
 29.11.2023, _NR Bhagat, P Chauhan, P Verma, A Mishra, VK Bharti_


_Animals, Altitude, Chickens, Glutamic Acid, Gastrointestinal Microbiome, Carbon_
Recently, there has been considerable interest in the functions of gut microbiota in broiler chickens in relation to their use as feed additives. However, the gut-microbiota of chickens reared at different altitudes are not well documented for their potential role in adapting to prevailing conditions and functional changes. In this context, the present study investigates the functional diversity of gut-microbes in high-altitude (HACh) and low-altitude adapted chickens (LACh), assessing their substrate utilization profile through Biolog Ecoplates technology. This will help in the identification of potential microbes or their synthesized metabolites, which could be beneficial for the host or industrial applications. Results revealed that among the 31 different types of studied substrates, only polymers, carbohydrates, carboxylic acids, and amine-based substrates utilization varied significantly (p &lt; 0.05) among the chickens reared at two different altitudes where gut-microbes of LACh utilized a broad range of substrates than the HACh. Further, diversity indices (Shannon and MacIntosh) analysis in LACh samples showed significant (p &lt; 0.05) higher richness and evenness of microbes as compared to the HACh samples. However, no significant difference was observed in the Simpson diversity index in gut microbes of lowversus high-altitude chickens. In addition, the Principal Component Analysis elucidated variation in substrate preferences of gut-microbes, where 13 and 8 carbon substrates were found to constitute PC1 and PC2, respectively, where γ-aminobutyric acid, D-glucosaminic acid, i-erythritol and tween 40 were the most relevant substrates that had a major effect on PC1, however, alpha-ketobutyric acid and glycyl-L-glutamic acid affected PC2. Hence, this study concludes that the gut-microbes of high and low-altitudes adapted chickens use different carbon substrates so that they could play a vital role in the health and immunity of an animal host based on their geographical location. Consequently, this study substantiates the difference in the substrate utilization and functional diversity of the microbial flora in chickens reared at high and low altitudes due to altitudinal changes.

[Source](https://www.nature.com/articles/s41598-023-48147-9)

10.1038/s41598-023-48147-9

---

## Introducing an environmental microbiome to axenic Aedes aegypti mosquitoes documents bacterial responses to a blood meal.
 28.11.2023, _JC LaReau, J Hyde, DE Brackney, B Steven_


The blood meal of the female mosquito serves as a nutrition source to support egg development, so is an important aspect of its biology. Yet, the roles the microbiome may play in blood digestion are poorly characterized. We employed axenic mosquitoes to investigate how the microbiome differs between mosquitoes reared in the insectary versus mosquitoes that acquire their microbiome from the environment. Environmental microbiomes were more diverse and showed larger temporal shifts over the course of blood digestion. Importantly, only bacteria from the environmental microbiome performed hemolysis in culture, pointing to functional differences between bacterial populations. These data highlight that taxonomic differences between the microbiomes of insectary-reared and wild mosquitoes are potentially also related to their functional ecology. Thus, axenic mosquitoes colonized with environmental bacteria offer a way to investigate the role of bacteria from the wild in mosquito processes such as blood digestion, under controlled laboratory conditions.

[Source](https://journals.asm.org/doi/10.1128/aem.00959-23)

10.1128/aem.00959-23

---

## Altered microbiota, antimicrobial resistance genes, and functional enzyme profiles in the rumen of yak calves fed with milk replacer.
 28.11.2023, _Y Zhuang, W Guo, K Cui, Y Tu, Q Diao, N Zhang, Y Bi, T Ma_


Yaks, as ruminants inhabiting high-altitude environments, possess a distinct rumen microbiome and are resistant to extreme living conditions. This study investigated the microbiota, resistome, and functional gene profiles in the rumen of yaks fed milk or milk replacer (MR), providing insights into the regulation of the rumen microbiome and the intervention of antimicrobial resistance in yaks through dietary methods. The abundance of Prevotella members increased significantly in response to MR. Tetracycline resistance was the most predominant. The rumen of yaks contained multiple antimicrobial resistance genes (ARGs) originating from different bacteria, which could be driven by MR, and these ARGs displayed intricate and complex interactions. MR also induced changes in functional genes. The enzymes associated with fiber degradation and butyrate metabolism were activated and showed close correlations with Prevotella members and butyrate concentration. This study allows us to deeply understand the ruminal microbiome and ARGs of yaks and their relationship with rumen bacteria in response to different milk sources.

[Source](https://journals.asm.org/doi/10.1128/spectrum.01314-23)

10.1128/spectrum.01314-23

---

## Global patterns in gene content of soil microbiomes emerge from microbial interactions.
 28.11.2023, _K Crocker, KK Lee, M Chakraverti-Wuerthwein, Z Li, M Tikhonov, M Mani, K Gowda, S Kuehn_


Microbial metabolism sustains life on Earth. Sequencing surveys of communities in hosts, oceans, and soils have revealed ubiquitous patterns linking the microbes present, the genes they possess, and local environmental conditions. One prominent explanation for these patterns is environmental filtering: local conditions select strains with particular traits. However, filtering assumes ecological interactions do not influence patterns, despite the fact that interactions can and do play an important role in structuring communities. Here, we demonstrate the insufficiency of the environmental filtering hypothesis for explaining global patterns in topsoil microbiomes. Using denitrification as a model system, we find that the abundances of two characteristic genotypes trade-off with pH; nar gene abundances increase while nap abundances decrease with declining pH. Contradicting the filtering hypothesis, we show that strains possessing the Nar genotype are enriched in low pH conditions but fail to grow alone. Instead, the dominance of Nar genotypes at low pH arises from an ecological interaction with Nap genotypes that alleviates nitrite toxicity. Our study provides a roadmap for dissecting how global associations between environmental variables and gene abundances arise from environmentally modulated community interactions.

[Source](https://www.biorxiv.org/content/10.1101/2023.05.31.542950v2)

10.1101/2023.05.31.542950

---

## Metagenomic analysis reveals diverse microbial community and potential functional roles in Baner rivulet, India.
 28.11.2023, _B Brar, R Kumar, D Sharma, AK Sharma, K Thakur, D Mahajan, R Kumar_


The health index of any population is directly correlated with the water quality, which in turn depends upon physicochemical characteristics and the microbiome of that aquatic source. For maintaining the water quality, knowledge of microbial diversity is a must. The present investigation attempts to evaluate the microflora of Baner. Metagenomics has been proven to be the technique for examining the genetic diversity of unculturable microbiota without using traditional culturing techniques. The microbial profile of Baner is analyzed using metagenomics for the first time to the best of our knowledge.

[Source](https://jgeb.springeropen.com/articles/10.1186/s43141-023-00601-x)

10.1186/s43141-023-00601-x

---

## mb-PHENIX: Diffusion and Supervised Uniform Manifold Approximation for denoising microbiota data.
 28.11.2023, _C Padron-Manrique, A Vázquez-Jiménez, DA Esquivel-Hernandez, YE Martinez Lopez, D Neri-Rosario, JP Sánchez-Castañeda, D Giron-Villalobos, O Resendis-Antonio_


Microbiota data encounters challenges arising from technical noise and the curse of dimensionality, which affect the reliability of scientific findings. Furthermore, abundance matrices exhibit a zero-inflated distribution due to biological and technical influences. Consequently, there is a growing demand for advanced algorithms that can effectively recover missing taxa while also considering the preservation of data structure.

[Source](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btad706/7453374)

10.1093/bioinformatics/btad706

---

## Facilitating bioinformatics reproducibility with QIIME 2 provenance Replay.
 27.11.2023, _CR Keefe, MR Dillon, E Gehret, C Herman, M Jewell, CV Wood, E Bolyen, JG Caporaso_


Study reproducibility is essential to corroborate, build on, and learn from the results of scientific research but is notoriously challenging in bioinformatics, which often involves large data sets and complex analytic workflows involving many different tools. Additionally, many biologists are not trained in how to effectively record their bioinformatics analysis steps to ensure reproducibility, so critical information is often missing. Software tools used in bioinformatics can automate provenance tracking of the results they generate, removing most barriers to bioinformatics reproducibility. Here we present an implementation of that idea, Provenance Replay, a tool for generating new executable code from results generated with the QIIME 2 bioinformatics platform, and discuss considerations for bioinformatics developers who wish to implement similar functionality in their software.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011676)

10.1371/journal.pcbi.1011676

---

## High-throughput identification and quantification of bacterial cells in the microbiota based on 16S rRNA sequencing with single-base accuracy using BarBIQ.
 27.11.2023, _J Jin, R Yamamoto, K Shiroguchi_


Bacteria often function as a community, called the microbiota, consisting of many different bacterial species. The accurate identification of bacterial types and the simultaneous quantification of the cells of each bacterial type will advance our understanding of microbiota; however, this cannot be performed by conventional 16S rRNA sequencing methods as they only identify and quantify genes, which do not always represent cells. Here, we present a protocol for our developed method, barcoding bacteria for identification and quantification (BarBIQ). In BarBIQ, the 16S rRNA genes of single bacterial cells are amplified and attached to a unique cellular barcode in a droplet. Sequencing the tandemly linked cellular barcodes and 16S rRNA genes from many droplets (representing many cells with unique cellular barcodes) and clustering the sequences using the barcodes determines both the bacterial type for each cell based on 16S rRNA gene and the number of cells for each bacterial type based on the quantity of barcode types sequenced. Single-base accuracy for 16S rRNA sequencing is achieved via the barcodes and by avoiding chimera formation from 16S rRNA genes of different bacteria using droplets. For data processing, an easy-to-use bioinformatic pipeline is available ( https://github.com/Shiroguchi-Lab/BarBIQ_Pipeline_V1_2_0 ). This protocol allows researchers with experience in molecular biology but without bioinformatics experience to perform the process in ~2 weeks. We show the application of BarBIQ in mouse gut microbiota analysis as an example; however, this method is also applicable to other microbiota samples, including those from the mouth and skin, marine environments, soil and plants, as well as those from other terrestrial environments.

[Source](https://www.nature.com/articles/s41596-023-00906-8)

10.1038/s41596-023-00906-8

---

