## The hallmarks of dietary intervention-resilient gut microbiome.
 08.10.2022, _NS Klimenko, VE Odintsova, A Revel-Muroz, AV Tyakht_


Maintaining equilibrium of the gut microbiome is crucial for human health. Diet represents an important and generally accessible natural channel of controlling the nutrients supply to the intestinal microorganisms. Although many studies showed that dietary interventions can specifically modulate gut microbiome composition, further progress of the approach is complicated by interindividual variability of the microbial community response. The reported causes of this variability include the baseline microbiome composition features, but it is unclear whether any of them are intervention-specific. Here, we applied a unified computational framework to investigate the variability of microbiome response measured as beta diversity in eight various dietary interventions using previously published 16S rRNA sequencing datasets. We revealed a number of baseline microbiome features which determine the microbiome response in an intervention-independent manner. One of the most stable associations, reproducible for different interventions and enterotypes, was a negative dependence of the response on the average number of genes per microorganism in the community-an indicator of the community functional redundancy. Meanwhile, many revealed microbiome response determinants were enterotype-specific. In Bact1 and Rum enterotypes, the response was negatively correlated with the baseline abundance of their main drivers. Additionally, we proposed a method for preliminary assessment of the microbiome response. Our study delineats the universal features determining microbiome response to diverse interventions. The proposed approach is promising for understanding the mechanisms of gut microbiome stability and improving the efficacy of personalised microbiome-tailored interventions.

[Source](https://doi.org/10.1038/s41522-022-00342-8)

10.1038/s41522-022-00342-8

---

## Nutrition and Microbiome Interactions in Human Cancer.
 08.10.2022, _PJ Daschner, S Ross, H Seifried, A Kumar, R Flores_


Individual physiologic responses and cancer risk can vary widely to changes in dietary patterns and are driven by multiple host-specific factors (e.g., genetics, epigenetics, inflammatory and metabolic states, and the colonizing microbiome). Emerging evidence indicates that diet-induced microbiota alterations are key modulators of several host functions important to tumor etiology, progression, and response to cancer therapy. Thus, diet may potentially be used to target alterations of the microbiota as an effective means to improve outcomes across the cancer continuum (from cancer prevention to tumor development and progression, to effects on treatment and survivorship). This review will focus on recent examples of functional interactions between dietary components (nutrients and non-nutrients) and the gastrointestinal (GI) microbiome, which are two critical and malleable environmental variables in cancer risk that affect host immune, metabolic, and cell signaling functions and which may provide insights for novel cancer therapeutic and preventive strategies.

[Source](https://doi.org/10.1016/j.jand.2022.10.004)

10.1016/j.jand.2022.10.004

---

## Differences in the microbiota of oral rinse, lesion, and normal site samples from patients with mucosal abnormalities on the tongue.
 07.10.2022, _Y Shitozawa, K Haro, M Ogawa, A Miyawaki, M Saito, K Fukuda_


The oral microbiota associated with mucosal diseases, including oral squamous cell carcinoma and oral potentially malignant disorders, have been extensively analyzed at the phylum and genus levels. However, the details of the oral microbiota remain unclear at the species and operational taxonomic unit (OTU) levels. We aimed to determine differences in the microbiota of oral rinse, lesion and normal site swab samples of patients with mucosal abnormalities on the tongues. Oral samples were obtained from 10 patients with oral mucosal abnormalities. Alpha and beta diversity at the OTU and genus levels of the microbiota samples were analyzed using OTUs clustered with 99.6% similarity based on 16S rRNA gene sequences obtained using the Sanger method. At the OTU level, the microbiota of the lesions were the least diverse but were different from those of the normal site and oral rinse samples. The OTUs corresponding to Streptococcus infantis and Haemophilus parainfluenzae were suggested to contribute to the differences between the microbiota of the lesions and normal sites. At the genus level, no significant differences between these microbiota were observed. In conclusion, strict OTU-level microbiota analysis might be able to discriminate lesions from normal sites of patients with mucosal abnormalities.

[Source](https://doi.org/10.1038/s41598-022-21031-8)

10.1038/s41598-022-21031-8

---

## Dynamic changes in host immune system and gut microbiota are associated with the production of SARS-CoV-2 antibodies.
 07.10.2022, _M Han, Y Huang, H Gui, Y Xiao, M He, J Liu, X Cao, M Zheng, M Lu, W Jia, H Li, X Wang, N Zhang, SA Kong, X Liu, Y Wu, F Wu, S Huang_



[Source](https://doi.org/10.1136/gutjnl-2022-327561)

10.1136/gutjnl-2022-327561

---

## Antibiotics and probiotics impact gut antimicrobial resistance gene reservoir in COVID-19 patients.
 09.10.2022, _Q Su, Q Liu, L Zhang, Z Xu, C Liu, W Lu, JY Ching, A Li, JWY Mak, GCY Lui, SSS Ng, KM Chow, DS Hui, PK Chan, FKL Chan, SC Ng_


Dysbiosis of gut microbiota is well-described in patients with coronavirus 2019 (COVID-19), but the dynamics of antimicrobial resistance genes (ARGs) reservoir, known as resistome, is less known. Here, we performed longitudinal fecal metagenomic profiling of 142 patients with COVID-19, characterized the dynamics of resistome from diagnosis to 6 months after viral clearance, and reported the impact of antibiotics or probiotics on the ARGs reservoir. Antibiotic-naive patients with COVID-19 showed increased abundance and types, and higher prevalence of ARGs compared with non-COVID-19 controls at baseline. Expansion in resistome was mainly driven by tetracycline, vancomycin, and multidrug-resistant genes and persisted for at least 6 months after clearance of SARS-CoV-2. Patients with expanded resistome exhibited increased prevalence of Klebsiella sp. and post-acute COVID-19 syndrome. Antibiotic treatment resulted in further increased abundance of ARGs whilst oral probiotics (synbiotic formula, SIM01) significantly reduced the ARGs reservoir in the gut microbiota of COVID-19 patients during the acute infection and recovery phase. Collectively, these findings shed new insights on the dynamic of ARGs reservoir in COVID-19 patients and the potential role of microbiota-directed therapies in reducing the burden of accumulated ARGs.

[Source](https://doi.org/10.1080/19490976.2022.2128603)

10.1080/19490976.2022.2128603

---

## Fibre is good for the microbiome: but what is the evidence?
 08.10.2022, _K Whelan, HM Staudacher_



[Source](https://doi.org/10.1016/S2468-1253(22)00310-7)

10.1016/S2468-1253(22)00310-7

---

## Enhanced cultured diversity of the mouse gut microbiota enables custom-made synthetic communities.
 08.10.2022, _A Afrizal, SAV Jennings, TCA Hitch, T Riedel, M Basic, A Panyot, N Treichel, FT Hager, EO Wong, B Wolter, A Viehof, A von Strempel, C Eberl, EM Buhl, B Abt, A Bleich, R Tolba, LM Blank, WW Navarre, F Kiessling, HP Horz, N Torow, V Cerovic, B Stecher, T Strowig, J Overmann, T Clavel_


Microbiome research needs comprehensive repositories of cultured bacteria from the intestine of mammalian hosts. We expanded the mouse intestinal bacterial collection (www.dsmz.de/miBC) to 212 strains, all publicly available and taxonomically described. This includes strain-level diversity, small-sized bacteria, and previously undescribed taxa (one family, 10 genera, and 39 species). This collection enabled metagenome-educated prediction of synthetic communities (SYNs) that capture key functional differences between microbiomes, notably identifying communities associated with either resistance or susceptibility to DSS-induced colitis. Additionally, nine species were used to amend the Oligo-Mouse Microbiota (OMM)12 model, yielding the OMM19.1 model. The added strains compensated for phenotype differences between OMM12 and specific pathogen-free mice, including body composition and immune cells in the intestine and associated lymphoid tissues. Ready-to-use OMM stocks are available for future studies. In conclusion, this work improves our knowledge of gut microbiota diversity in mice and enables functional studies via the modular use of isolates.

[Source](https://doi.org/10.1016/j.chom.2022.09.011)

10.1016/j.chom.2022.09.011

---

## Convergent evolution of the gut microbiome in marine carnivores.
 09.10.2022, _X Wang, X Wu, Y Shang, X Mei, S Zhou, Q Wei, G Sun, Y Dong, H Zhang_


The gut microbiome can help the host adapt to a variety of environments and is affected by many factors. Marine carnivores have unique habitats in extreme environments. The question of whether marine habitats surpass phylogeny to drive the convergent evolution of the gut microbiome in marine carnivores remains unanswered. In the present study, we compared the gut microbiomes of 16 species from different habitats. Principal component analysis (PCA) and principal coordinate analysis (PCoA) separated three groups according to their gut microbiomes: marine carnivores, terrestrial carnivores, and terrestrial herbivores. The alpha diversity and niche breadth of the gut microbiome of marine carnivores were lower than those of the gut microbiome of terrestrial carnivores and terrestrial herbivores. The gut microbiome of marine carnivores harbored many marine microbiotas, including those belonging to the phyla Planctomycetes, Cyanobacteria, and Proteobacteria, and the genus Peptoclostridium. Collectively, these results revealed that marine habitats drive the convergent evolution of the gut microbiome of marine carnivores. This study provides a new perspective on the adaptive evolution of marine carnivores.

[Source](https://doi.org/10.1002/ece3.9373)

10.1002/ece3.9373

---

## Gut microbiota shift in layer pullets fed on black soldier fly larvae-based feeds towards enhancing healthy gut microbial community.
 09.10.2022, _EW Ndotono, FM Khamis, JL Bargul, CM Tanga_


Globally, most gut microbiota-related studies have focused on broilers due to their diverse microbial communities compared to that of layer chicken. However, in Africa few studies have been undertaken despite the increasing benefits to the poultry industry. The utilization of Insect-Based diets to improve the gastrointestinal function and gut health in poultry is increasingly gaining global attention. Here, we evaluated the potential roles of commercial black soldier fly larvae-based feeds (BSFLF) in reshaping the abundance, composition and diversity of the gut microbiota of layer chickens using high throughput Oxford nanopore Minion sequencing of the full length bacterial 16S rRNA gene. Two hundred and fifty ISA Brown layer chicks were reared in pens for a period of 20 weeks. The layer pullets were allotted 5 dietary treatments that were formulated as follows: control diet (T1): 100% FM + 0% BSFL, T2: 25% BSFL + 75% FM; T3: 50% BSFL + 50% FM; T4: 75% BSFL + 25% FM, and T5: 100% BSFL + 0% FM. Sampling was done from the eight major regions including oesophagus, crop, proventriculus, gizzard, duodenum, ileum, large intestines and ceca. Out of the 400,064 classified reads analyzed, the most dominant phyla identified across the feed treatments were Firmicutes, Bacteroidetes, Proteobacteria and Actinobacteria. The diet treatment with 100% inclusion levels of BSFL showed the highest intra-species alpha diversity and richness according to Chao1 and Shannon index. Intra-species beta diversity assessment revealed that the diet types significantly influenced the abundance of the microbiota, but differences between most abundant taxa were similar. There was increase in abundance of potentially beneficial bacteria (Lactobacillus, Bacteroides and Enterococcus) with increased inclusion levels of BSFLF in layer pullets diets. Across the different gut segments, Lactobacillus dominated all the eight regions and the ceca was the most diverse segment. Our findings unravel complex gut microbial shift in laying hen fed BSFLF and therefore underpins the potential roles of beneficial bacteria as promising prebiotics and probiotics in reshaping of the gut microbiota to maintain good gut health.

[Source](https://doi.org/10.1038/s41598-022-20736-0)

10.1038/s41598-022-20736-0

---

## Succession of the intestinal bacterial community in Pacific bluefin tuna (<em>Thunnus orientalis</em>) larvae

[Source](https://doi.org/10.1371/journal.pone.0275211)

---

## Bacterial community in soil and tree roots of <em>Picea abies</em> shows little response to clearcutting

[Source](https://doi.org/10.1093/femsec/fiac118)

---

## Size-fractionated microbiome observed during an eight-month long sampling in Jiaozhou Bay and the Yellow Sea

[Source](https://doi.org/10.1038/s41597-022-01734-3)

---

## Microbiome of rehydrated corn and sorghum grain silages treated with microbial inoculants in different fermentation periods

[Source](https://doi.org/10.1038/s41598-022-21461-4)

---

## Quantification and characterization of mouse and human tissue-resident microbiota by qPCR and 16S sequencing

[Source](https://doi.org/10.1016/j.xpro.2022.101765)

---

