## Signature of Alzheimer's Disease in Intestinal Microbiome: Results From the AlzBiom Study.
 07.05.2022, _C Laske, S Müller, O Preische, V Ruschil, MHJ Munk, I Honold, S Peter, U Schoppmeier, M Willmann_


Changes in intestinal microbiome composition have been described in animal models of Alzheimer's disease (AD) and AD patients. Here we investigated how well taxonomic and functional intestinal microbiome data and their combination with clinical data can be used to discriminate between amyloid-positive AD patients and cognitively healthy elderly controls.

[Source](https://doi.org/10.3389/fnins.2022.792996)

10.3389/fnins.2022.792996

---

## The Fungal Gut Microbiome Exhibits Reduced Diversity and Increased Relative Abundance of Ascomycota in Severe COVID-19 Illness and Distinct Interconnected Communities in SARS-CoV-2 Positive Patients.
 09.05.2022, _J Reinold, F Farahpour, AK Schoerding, C Fehring, S Dolff, M Konik, J Korth, L van Baal, J Buer, O Witzke, AM Westendorf, J Kehrmann_


_Ascomycota, Bacteria, COVID-19, Dysbiosis, Gastrointestinal Microbiome, Humans, Mycobiome, SARS-CoV-2_
Clinical and experimental studies indicate that the bacterial and fungal gut microbiota modulates immune responses in distant organs including the lungs. Immune dysregulation is associated with severe SARS-CoV-2 infection, and several groups have observed gut bacterial dysbiosis in SARS-CoV-2 infected patients, while the fungal gut microbiota remains poorly defined in these patients. We analyzed the fungal gut microbiome from rectal swabs taken prior to anti-infective treatment in 30 SARS-CoV-2 positive (21 non-severe COVID-19 and 9 developing severe/critical COVID-19 patients) and 23 SARS-CoV-2 negative patients by ITS2-sequencing. Pronounced but distinct interconnected fungal communities distinguished SARS-CoV-2 positive and negative patients. Fungal gut microbiota in severe/critical COVID-19 illness was characterized by a reduced diversity, richness and evenness and by an increase of the relative abundance of the Ascomycota phylum compared with non-severe COVID-19 illness. A dominance of a single fungal species with a relative abundance of &gt;75% was a frequent feature in severe/critical COVID-19. The dominating fungal species were highly variable between patients even within the groups. Several fungal taxa were depleted in patients with severe/critical COVID-19.The distinct compositional changes of the fungal gut microbiome in SARS-CoV-2 infection, especially in severe COVID-19 illness, illuminate the necessity of a broader approach to investigate whether the differences in the fungal gut microbiome are consequences of SARS-CoV-2 infection or a predisposing factor for critical illness.

[Source](https://doi.org/10.3389/fcimb.2022.848650)

10.3389/fcimb.2022.848650

---

## The associations between low abundance of <em>Mycoplasma hominis</em> and female fecundability: a pregnancy-planning cohort study

[Source](https://doi.org/10.1186/s12866-022-02545-7)

---

## Insights into the Profile of the Human Expiratory Microbiota and Its Associations with Indoor Microbiotas.
 05.05.2022, _Y Zhang, F Shen, Y Yang, M Niu, D Chen, L Chen, S Wang, Y Zheng, Y Sun, F Zhou, H Qian, Y Wu, T Zhu_


Microorganisms residing in the human respiratory tract can be exhaled, and they constitute a part of environmental microbiotas. However, the expiratory microbiota community and its associations with environmental microbiotas remain poorly understood. Here, expiratory bacteria and fungi and the corresponding microbiotas from the living environments were characterized by DNA amplicon sequencing of residents' exhaled breath condensate (EBC) and environmental samples collected from 14 residences in Nanjing, China. The microbiotas of EBC samples, with a substantial heterogeneity, were found to be as diverse as those of skin, floor dust, and airborne microbiotas. Model fitting results demonstrated the role of stochastic processes in the assembly of the expiratory microbiota. Using a fast expectation-maximization algorithm, microbial community analysis revealed that expiratory microbiotas were differentially associated with other types of microbiotas in a type-dependent and residence-specific manner. Importantly, the expiratory bacteria showed a composition similarity with airborne bacteria in the bathroom and kitchen environments with an average of 12.60%, while the expiratory fungi showed a 53.99% composition similarity with the floor dust fungi. These differential patterns indicate different relationships between expiratory microbiotas and the airborne microbiotas and floor dust microbiotas. The results here illustrated for the first time the associations between expiratory microbiotas and indoor microbiotas, showing a potential microbial exchange between the respiratory tract and indoor environment. Thus, improved hygiene and ventilation practices can be implemented to optimize the indoor microbial exposome, especially in indoor bathrooms and kitchens.

[Source](https://doi.org/10.1021/acs.est.2c00688)

10.1021/acs.est.2c00688

---

## Oral microbiome of Crohn's Disease patients with and without oral manifestations.
 05.05.2022, _S Hu, J Mok, M Gowans, DEH Ong, JL Hartono, JWJ Lee_


Microbiome dysbiosis is associated with inflammatory destruction in Crohn's Disease (CD). Although gut microbiome dysbiosis is well-established in CD, the oral microbiome is comparatively under-studied. This study aims to characterize the oral microbiome of CD patients with/without oral manifestations.

[Source](https://doi.org/10.1093/ecco-jcc/jjac063)

10.1093/ecco-jcc/jjac063

---

## Addressing virtual reality misclassification: A hardware-based qualification matrix for virtual reality technology.
 25.11.2021, _M Takac, J Collett, R Conduit, A De Foe_


_Humans, Reproducibility of Results, Technology, Virtual Reality, Virtual Reality Exposure Therapy_
Through its unique sensory synchronized design, virtual reality (VR) provides a convincing, user-centred experience of highly controllable scenarios. Importantly, VR is a promising modality for healthcare, where treatment efficacy has been recognized for a range of conditions. It is equally valuable across wider research disciplines. However, there is a lack of suitable criteria and consistent terminology with which to define VR technology. A considerable number of studies have misclassified VR hardware (e.g. defining laptops as VR), hindering validity and research comparisons. This review addresses these limitations and establishes a standardized VR qualification framework. As a result of a comprehensive theoretical and literature review, the hardware-based VR qualification matrix is proposed. The matrix criteria consist of (1) three-dimensional (3D) synchronized sensory stimulation; (2) degrees of freedom tracking; and (3) visual suppression of physical stimuli. To validate the model and quantify the current scale/diversity of VR misclassification, a 2019 sectional review of health-related studies was conducted. Of the 115 studies examined against standardized criteria, 35.7% utilized VR, 31.3% misclassified VR, 18.3% were considered quasi-VR, and 14.8% omitted critical specifications. The proposed model demonstrates good validity and reliability for qualifying and classifying VR. Key Practitioner Messages Virtual reality (VR) therapy has gained rapid empirical support, although many practitioners do not understand the difference between genuine and less-realistic VR variations. That has resulted from an evident lack of suitable criteria to define VR across a range of studies and protocols. Our proposed hardware-based virtual reality qualification matrix addresses issues to do with misclassification, via the introduction of standardised criteria. Applying the matrix to existing literature has revealed that more than 30% of VR studies use hardware that does not fit the high standards of rigour required for immersion in a simulated space. The model is a practical tool researchers and practitioners can use to quality and verify VR standards across research studies.

[Source](https://doi.org/10.1099/jmm.0.001520)

10.1002/cpp.2624

---

## SARS-CoV-2 and <em>Prevotella</em> spp.: friend or foe? A systematic literature review

[Source](https://doi.org/10.1099/jmm.0.001520)

---

## The Game for Three: Salmonella-Host-Microbiota Interaction Models.
 07.05.2022, _K Grzymajlo_


Colonization of the gastrointestinal (GI) tract by enteric pathogens occurs in a context strongly determined by host-specific gut microbiota, which can significantly affect the outcome of infection. The complex gameplay between the trillions of microbes that inhabit the GI tract, the host, and the infecting pathogen defines a specific triangle of interaction; therefore, a complete model of infection should consider all of these elements. Many different infection models have been developed to explain the complexity of these interactions. This review sheds light on current knowledge, along with the strengths and limitations of in vitro and in vivo models utilized in the study of Salmonella-host-microbiome interactions. These models range from the simplest experiment simulating environmental conditions using dedicated growth media through in vitro interaction with cell lines and 3-D organoid structure, and sophisticated "gut on a chip" systems, ending in various animal models. Finally, the challenges facing this field of research and the important future directions are outlined.

[Source](https://doi.org/10.3389/fmicb.2022.854112)

10.3389/fmicb.2022.854112

---

## Host gene effects on gut microbiota in type 1 diabetes.
 06.05.2022, _K Guo, J Huang, Z Zhou_


Type 1 diabetes (T1D) is an organ-specific autoimmune disease characterized by progressive pancreatic β-cell loss. Both a predisposing genetic background, that may encompass mutations in several genes, as well as exposure to environmental factors can affect the progression of autoimmune responses to multiple pancreatic islet autoantigens. Many genetic variants that increase the risk of T1D are found in immunity genes involved in sensing and responding to microorganisms. Although increasing evidence indicates that the gut microbiome composition may promote or prevent T1D development, little is known about the link between gut microbiota and T1D susceptibility genes in patients with T1D. Recent studies in the inbred non-obese diabetic (NOD) mouse, a widely used model of T1D, have suggested that many genetic loci can influence gut microbiome composition to modulate islet autoimmunity. This review summarizes evidence that examines the effect of host genes on gut microbiota diversity and function during T1D development. Knowledge of the host gene-gut microbiota interactions at play during T1D progression may help us identify new diagnostic and prognostic tools and help also design effective strategies for disease treatment.

[Source](https://doi.org/10.1042/BST20220004)

10.1042/BST20220004

---

## Host-microbial interactions in metabolic diseases: from diet to immunity.
 05.05.2022, _JH Lee, JH Park_


Growing evidence suggests that the gut microbiome is an important contributor to metabolic diseases. Alterations in microbial communities are associated with changes in lipid metabolism, glucose homeostasis, intestinal barrier functions, and chronic inflammation, all of which can lead to metabolic disorders. Therefore, the gut microbiome may represent a novel therapeutic target for obesity, type 2 diabetes, and nonalcoholic fatty liver disease. This review discusses how gut microbes and their products affect metabolic diseases and outlines potential treatment approaches via manipulation of the gut microbiome. Increasing our understanding of the interactions between the gut microbiome and host metabolism may help restore the healthy symbiotic relationship between them.

[Source](https://doi.org/10.1007/s12275-022-2087-y)

10.1007/s12275-022-2087-y

---

## Bacterial association with metals enables in vivo tracking of microbiota using magnetic resonance imaging

[Source](https://doi.org/10.1101/2022.05.05.490857)

---

## Natural Garlic Organosulfur Compounds Prevent Metabolic Disorder of Lipid and Glucose by Increasing Gut Commensal <em>Bacteroides acidifaciens</em>

[Source](https://doi.org/10.1021/acs.jafc.2c00555)

---

## Circulating gut microbiota-related metabolites influence endothelium plaque lesion formation in ApoE knockout rats.
 09.05.2022, _HL Chuang, CC Chiu, C Lo, CC Hsu, JY Liu, SW Hung, SC Tsai, HH Sung, CL Wang, YT Huang_


Atherosclerosis is the main cause of cardiac and peripheral vessel infarction in developed countries. Recent studies have established that gut microbiota and their metabolites play important roles in the development and progression of cardiovascular disease; however, the underlying mechanisms remain unclear. The present study aimed to investigate endothelium plaque lesion formation in ApoE-deficient rats fed a normal chow diet under germ-free (GF) and specific-pathogen-free (SPF) conditions at various time points. There was no difference in serum cholesterol and triglyceride levels between SPF-rats and GF-rats. Histological studies revealed that the GF-rats developed endothelium plaques in the aorta from 26 to 52 weeks, but this was not observed in SPF-rats. GF-rat coronary arteries had moderate-to-severe endothelium lesions during this time period, but SPF-rat coronary arteries had only mild lesion formation. Immunohistochemical staining showed higher accumulation of CD68-positive and arginase-negative foamy-like macrophages on the arterial walls of GF-rats, and expression of TNF-α and IL-6 in foam cells was only observed in GF-rats. In addition, microbial metabolites, including equol derivatives, enterolactone derivatives, indole-3-propionate, indole-3-acrylic acid, cholic acid, hippuric acid, and isoquinolone, were significantly higher in the SPF group than in the GF group. In conclusion, our results indicate that gut microbiota may attenuate atherosclerosis development.

[Source](https://doi.org/10.1371/journal.pone.0264934)

10.1371/journal.pone.0264934

---

## Depletion of the gut microbiota enhances the blood pressure-lowering effect of captopril: implication of the gut microbiota in resistant hypertension.
 05.05.2022, _J Kyoung, T Yang_


The role of the gut microbiota in the initiation and progression of hypertension has been newly identified, suggesting that targeting the gut microbiota may provide a new treatment strategy. This entails a complicated interaction between the gut microbiota and different host systems (e.g., immune system) or organs (e.g., gut, spleen) that contribute to blood pressure control. The significance of the gut microbiota in treatment-resistant hypertension is still unknown, owing to a lack of appropriate animal models. Given that the gut microbiota has a variety of enzymatic activities, we hypothesized that the gut microbiota may be involved in the metabolism of antihypertensive medications, causing treatment-resistant hypertension. We investigated this hypothesis in a simple, new hypertension paradigm and found that hypertensive rats pretreated with antibiotics to reduce the gut microbiota had a better response to the angiotensin-converting enzyme inhibitor captopril. This is a simple rodent model for testing the effectiveness of antihypertensive medications. Further mechanistic research may shed light on the pathogenic function of the gut microbiota in resistant hypertension. Our method presents a novel model that has the potential to be employed in the research of resistant hypertension.

[Source](https://doi.org/10.1038/s41440-022-00921-4)

10.1038/s41440-022-00921-4

---

## Fecal microbiota and diets of muskox female adults and calves.
 07.05.2022, _JY Cheon, H Cho, M Kim, HJ Park, TS Park, WY Lee_


In mammals, the gut microbiome is vertically transmitted during maternal lactation at birth. In this study, we investigated the gut microbiome and diets of muskox, a large herbivore inhabiting in the high Arctic. We compared the microbiota composition using bacterial 16S rRNA gene sequencing and diets using stable isotope analysis of muskox feces of six female adults and four calves on Ella Island, East Greenland. Firmicutes were the most abundant bacterial phylum in both the adults and calves, comprising 94.36% and 94.03%, respectively. Significant differences were observed in the relative abundance of the two Firmicutes families. The adults were primarily dominated by Ruminococcaceae (73.90%), and the calves were dominated by both Ruminococcaceae (56.25%) and Lachnospiraceae (24.00%). Stable isotope analysis of the feces in the study area revealed that both adults and calves had similar ranges of 13C and 15N, likely derived from the dominant diet plants. Despite their similar diets, the different gut microbiome compositions in muskox adults and calves indicate that the gut microbiome of the calves may not be fully colonized to the extent of that of the adults.

[Source](https://doi.org/10.1002/ece3.8879)

10.1002/ece3.8879

---

## Temporal Changes in Gut Microbiota Composition and Pollen Diet Associated with Colony Weakness of a Stingless Bee.
 05.05.2022, _KL Haag, L Caesar, M da Silveira Regueira-Neto, DR de Sousa, V Montenegro Marcelino, V de Queiroz Balbino, A Torres Carvalho_


Compared to honeybees and bumblebees, the effect of diet on the gut microbiome of Neotropical corbiculate bees such as Melipona spp. is largely unknown. These bees have been managed for centuries, but recently an annual disease is affecting M. quadrifasciata, an endangered species kept exclusively by management in Southern Brazil. Here we report the results of a longitudinal metabarcoding study involving the period of M. quadrifasciata colony weakness, designed to monitor the gut microbiota and diet changes preceding an outbreak. We found increasing amounts of bacteria associated to the gut of forager bees 2 months before the first symptoms have been recorded. Simultaneously, forager bees showed decreasing body weight. The accelerated growth of gut-associated bacteria was uneven among taxa, with Bifidobacteriaceae dominating, and Lactobacillaceae decreasing in relative abundance within the bacterial community. Dominant fungi such as Candida and Starmerella also decreased in numbers, and the stingless bee obligate symbiont Zygosaccharomyces showed the lowest relative abundance during the outbreak period. Such changes were associated with pronounced diet shifts, i.e., the rise of Eucalyptus spp. pollen amount in forager bees' guts. Furthermore, there was a negative correlation between the amount of Eucalyptus pollen in diets and the abundance of some bacterial taxa in the gut-associated microbiota. We conclude that diet and subsequent interactions with the gut microbiome are key environmental components of the annual disease and propose the use of diet supplementation as means to sustain the activity of stingless bee keeping as well as native bee pollination services.

[Source](https://doi.org/10.1007/s00248-022-02027-3)

10.1007/s00248-022-02027-3

---

## Interaction between MHC diversity and constitution, gut microbiota and Astrovirus infections in a neotropical bat

[Source](https://doi.org/10.1111/mec.16491)

---

## Symbiont transmission in marine sponges: reproduction, development, and metamorphosis.
 09.05.2022, _TJ Carrier, M Maldonado, L Schmittmann, L Pita, TCG Bosch, U Hentschel_


Marine sponges (phylum Porifera) form symbioses with diverse microbial communities that can be transmitted between generations through their developmental stages. Here, we integrate embryology and microbiology to review how symbiotic microorganisms are transmitted in this early-diverging lineage. We describe that vertical transmission is widespread but not universal, that microbes are vertically transmitted during a select developmental window, and that properties of the developmental microbiome depends on whether a species is a high or low microbial abundance sponge. Reproduction, development, and symbiosis are thus deeply rooted, but why these partnerships form remains the central and elusive tenet of these developmental symbioses.

[Source](https://doi.org/10.1186/s12915-022-01291-6)

10.1186/s12915-022-01291-6

---

## A new microbiological weapon against lepidopteran pests

[Source](https://doi.org/10.1101/2022.05.06.490868)

---

## Associations between bacterial communities and microplastics from surface seawater of the Northern Patagonian area of Chile.
 05.05.2022, _P Aguila-Torres, M González, JE Maldonado, R Miranda, L Zhang, R González-Stegmaier, LA Rojas, A Gaete_


The presence of microplastics in oceans and coastlines has increased during recent years due anthropogenic activities and represents a serious environmental problem. The establishment and assembly of microbial communities in these microplastics, specifically located near aquaculture activities, is not well understood. In this study, we analyzed unique and core members of bacterial communities attached to microplastics collected from three coastal environments of the South Pacific, which represent low, medium and high anthropogenic activity derived from the aquaculture industry. Microplastics were analyzed with Fourier-transform infrared spectroscopy, scanning electron microscopy, and next-generation sequencing to assess the prevailing microplastics types, and to characterize microbial communities attached to them. We identified four main types of microplastics (polypropylene, polyethylene, nylon and polystyrene) and 3102 Operational Taxonomic Units (OTUs) at the sampled sites, which were dominated by the phylum Cyanobacteria, Bacteroidetes and Proteobacteria (mainly Alpha and Gammaproteobacteria). Similarity index analysis showed that bacterial communities in microplastics differed from those found in the surrounding seawaters, and also that they varied among locations, suggesting a role of the environment and level of anthropogenic activities on the plastisphere taxa. Despite this difference, 222 bacterial OTUs were shared among the three sites representing between 34 and 51% of OTUs of each sampled site, and thus constituted a core microbiome of microplastics. Comparison of the core microbiome with bacterial communities of the surrounding seawater suggested that the plastisphere constituted a selective habitat for diverse microbial communities. Computational predictions also provided evidence of significantly enriched functions in the core microbiome. Co-occurrence networks revealed that putative ecological interactions among microplastics OTUs was dominated by positive correlations. To the best of our knowledge, this is the first study that evaluated the composition of microbial communities found in microplastics from the Patagonia region of the Southern Pacific Ocean.

[Source](https://doi.org/10.1016/j.envpol.2022.119313)

10.1016/j.envpol.2022.119313

---

## Identification of beneficial and detrimental bacteria impacting sorghum responses to drought using multi-scale and multi-system microbiome comparisons.
 06.05.2022, _M Qi, JC Berry, KW Veley, L O'Connor, OM Finkel, I Salas-González, M Kuhs, J Jupe, E Holcomb, T Glavina Del Rio, C Creech, P Liu, SG Tringe, JL Dangl, DP Schachtman, RS Bart_


Drought is a major abiotic stress limiting agricultural productivity. Previous field-level experiments have demonstrated that drought decreases microbiome diversity in the root and rhizosphere. How these changes ultimately affect plant health remains elusive. Toward this end, we combined reductionist, transitional and ecological approaches, applied to the staple cereal crop sorghum to identify key root-associated microbes that robustly affect drought-stressed plant phenotypes. Fifty-three Arabidopsis-associated bacteria were applied to sorghum seeds and their effect on root growth was monitored. Two Arthrobacter strains caused root growth inhibition (RGI) in Arabidopsis and sorghum. In the context of synthetic communities, Variovorax strains were able to protect plants from Arthrobacter-caused RGI. As a transitional system, high-throughput phenotyping was used to test the synthetic communities. During drought stress, plants colonized by Arthrobacter had reduced growth and leaf water content. Plants colonized by both Arthrobacter and Variovorax performed as well or better than control plants. In parallel, we performed a field trial wherein sorghum was evaluated across drought conditions. By incorporating data on soil properties into the microbiome analysis, we accounted for experimental noise with a novel method and were able to observe the negative correlation between the abundance of Arthrobacter and plant growth. Having validated this approach, we cross-referenced datasets from the high-throughput phenotyping and field experiments and report a list of bacteria with high confidence that positively associated with plant growth under drought stress. In conclusion, a three-tiered experimental system successfully spanned the lab-to-field gap and identified beneficial and deleterious bacterial strains for sorghum under drought.

[Source](https://doi.org/10.1038/s41396-022-01245-4)

10.1038/s41396-022-01245-4

---

## A metagenomic study of the bacteria in snow algae microbiomes.
 05.05.2022, _KM Yakimovich, LM Quarmby_


The bacterial communities found in snow algae blooms have been described in terms of their 16S rRNA gene community profiles, but little information exists on their metabolic potential. Previously, we reported that several bacterial taxa are common across snow algae blooms in the southwestern mountains of the Coast Range in British Columbia, Canada. Here, we further this work by reporting a partial bacterial metagenome from the same snow algal microbiomes. Using shotgun metagenomic data, we constructed metagenomically assembled bacterial genomes (MAGs). Of the total 54 binned MAGs, 28 were bacterial and estimated to be at least 50% complete based on single copy core genes. The 28 MAGs fell into five Classes: Actinomycetia, Alphaproteobacteria, Bacteroidia, Betaproteobacteria and Gammaproteobacteria. All MAGs were assigned to a class, 27 to an order, 25 to family, 18 to genus, and none to species. MAGs showed the potential to support algal growth by synthesizing B-vitamins and growth hormones. There was also widespread adaptation to the low oxygen environment of biofilms, including synthesis of high-affinity terminal oxidases and anaerobic pathways for cobalamin synthesis. Also notable, was the absence of N2 fixation, and the presence of incomplete denitrification pathways suggestive of NO signalling within the microbiome.

[Source](https://doi.org/10.1139/cjm-2021-0313)

10.1139/cjm-2021-0313

---

## Soil microbial diversity and functional profiling of a tropical rainforest of a highly dissected low hill from the upper Itaya river basin revealed by analysis of shotgun metagenomics sequencing data.
 07.05.2022, _M Cobos, SL Estela, HN Rodríguez, CG Castro, M Grandez, JC Castro_


The tropical rainforest of a highly dissected low hill from the upper Itaya river basin belongs to the western Amazonia region. Some investigations on the biodiversity of these rainforests were more focused on animals and plants diversity. The soils of this region are composed of moderately fertile sediments deposited recently from the initiation of the Andean orogenesis in the Miocene until now. However, scientific information about the soil microbial and functional diversity is still missing. This report presents shotgun metagenomics sequencing data from soils of this rainforest type. A composite loamy soil sample was collected from a primary forest, and metagenomic DNA was purified with standardized methods. Furthermore, libraries were prepared and paired-end sequenced on the Illumina NextSeq 550 platform. Raw Illumina paired-end reads have been uploaded and analysed in the Metagenomics RAST server (MG-RAST). The raw sequence data in fastq format is available at NCBI's Sequence Read Archive (SRA) with accession number SRX12846710.

[Source](https://doi.org/10.1016/j.dib.2022.108205)

10.1016/j.dib.2022.108205

---

## Accessible analysis of longitudinal data with linear mixed effects models.
 09.05.2022, _JI Murphy, NE Weaver, AE Hendricks_


_Animals, Computer Simulation, Linear Models, Longitudinal Studies, Mice, Models, Statistical_
Longitudinal studies are commonly used to examine possible causal factors associated with human health and disease. However, the statistical models, such as two-way ANOVA, often applied in these studies do not appropriately model the experimental design, resulting in biased and imprecise results. Here, we describe the linear mixed effects (LME) model and how to use it for longitudinal studies. We re-analyze a dataset published by Blanton et al. in 2016 that modeled growth trajectories in mice after microbiome implantation from nourished or malnourished children. We compare the fit and stability of different parameterizations of ANOVA and LME models; most models found that the nourished versus malnourished growth trajectories differed significantly. We show through simulation that the results from the two-way ANOVA and LME models are not always consistent. Incorrectly modeling correlated data can result in increased rates of false positives or false negatives, supporting the need to model correlated data correctly. We provide an interactive Shiny App to enable accessible and appropriate analysis of longitudinal data using LME models.

[Source](https://doi.org/10.1242/dmm.048025)

10.1242/dmm.048025

---

## Scalable in-memory processing of omics workflows.
 07.05.2022, _V Elisseev, LJ Gardiner, R Krishna_


We present a proof of concept implementation of the in-memory computing paradigm that we use to facilitate the analysis of metagenomic sequencing reads. In doing so we compare the performance of POSIX™file systems and key-value storage for omics data, and we show the potential for integrating high-performance computing (HPC) and cloud native technologies. We show that in-memory key-value storage offers possibilities for improved handling of omics data through more flexible and faster data processing. We envision fully containerized workflows and their deployment in portable micro-pipelines with multiple instances working concurrently with the same distributed in-memory storage. To highlight the potential usage of this technology for event driven and real-time data processing, we use a biological case study focused on the growing threat of antimicrobial resistance (AMR). We develop a workflow encompassing bioinformatics and explainable machine learning (ML) to predict life expectancy of a population based on the microbiome of its sewage while providing a description of AMR contribution to the prediction. We propose that in future, performing such analyses in 'real-time' would allow us to assess the potential risk to the population based on changes in the AMR profile of the community.

[Source](https://doi.org/10.1016/j.csbj.2022.04.014)

10.1016/j.csbj.2022.04.014

---

## DBSCAN-SWA: An Integrated Tool for Rapid Prophage Detection and Annotation.
 07.05.2022, _R Gan, F Zhou, Y Si, H Yang, C Chen, C Ren, J Wu, F Zhang_


As an intracellular form of a bacteriophage in the bacterial host genome, a prophage usually integrates into bacterial DNA with high specificity and contributes to horizontal gene transfer (HGT). With the exponentially increasing number of microbial sequences uncovered in genomic or metagenomics studies, there is a massive demand for a tool that is capable of fast and accurate identification of prophages. Here, we introduce DBSCAN-SWA, a command line software tool developed to predict prophage regions in bacterial genomes. DBSCAN-SWA runs faster than any previous tools. Importantly, it has great detection power based on analysis using 184 manually curated prophages, with a recall of 85% compared with Phage_Finder (63%), VirSorter (74%), and PHASTER (82%) for (Multi-) FASTA sequences. Moreover, DBSCAN-SWA outperforms the existing standalone prophage prediction tools for high-throughput sequencing data based on the analysis of 19,989 contigs of 400 bacterial genomes collected from Human Microbiome Project (HMP) project. DBSCAN-SWA also provides user-friendly result visualizations including a circular prophage viewer and interactive DataTables. DBSCAN-SWA is implemented in Python3 and is available under an open source GPLv2 license from https://github.com/HIT-ImmunologyLab/DBSCAN-SWA/.

[Source](https://doi.org/10.3389/fgene.2022.885048)

10.3389/fgene.2022.885048

---

## A New Approach to Testing Mediation of the Microbiome at Both the Community and Individual Taxon Levels.
 05.05.2022, _Y Yue, YJ Hu_


Understanding whether and which microbes played a mediating role between an exposure and a disease outcome are essential for researchers to develop clinical interventions to treat the disease by modulating the microbes. Existing methods for mediation analysis of the microbiome are often limited to a global test of community-level mediation or selection of mediating microbes without control of the false discovery rate (FDR). Further, while the null hypothesis of no mediation at each microbe is a composite null that consists of three types of null, most existing methods treat the microbes as if they were all under the same type of null, leading to excessive false positive results.

[Source](https://doi.org/10.1093/bioinformatics/btac310)

10.1093/bioinformatics/btac310

---

