## Face masks: what the data say.
 09.10.2020, _L Peeples_


_Animals, Betacoronavirus, Coronavirus, Coronavirus Infections, Cricetinae, Masks, Mesocricetus, Pandemics, Pneumonia, Viral_

[Source](https://www.nature.com/articles/d41586-020-02801-8?WT.ec_id=NATURE-20201008)

10.1038/d41586-020-02801-8

---

## Can we prevent allergic disease? Understanding the links between the early life microbiome and allergic diseases of childhood.
 07.10.2020, _C Petersen, SE Turvey_


The microbiome and immune system are intrinsically linked, and during infancy these crucial biological systems undergo a concurrent and expansive maturation process. As these maturation processes progress, some children develop a sequence of IgE-mediated immune disorders termed the 'Allergic March', and unfortunately the prevalence of these lifelong and burdensome allergic conditions has increased over the past half century. Our current treatment strategies are unable to prevent or cure components of the Allergic March. However, recent discoveries have enhanced our mechanistic understanding of early-life microbiota-immune interactions with exciting implications for preventing these allergic disorders.

[Source](https://journals.lww.com/co-pediatrics/Abstract/9000/Can_we_prevent_allergic_disease__Understanding_the.98719.aspx)

10.1097/MOP.0000000000000956

---

## Atopic Dermatitis and the Role of the Skin Microbiome in Choosing Prevention, Treatment, and Maintenance Options.
 07.10.2020, _H Baldwin, C Aguh, A Andriessen, L Benjamin, AS Ferberg, D Hooper, JL Jarizzo, PA Lio, B Tlougan, HC Woolery-Lloyd, J Zeichner_


Atopic dermatitis (AD) is a common skin condition characterized by disturbed barrier function, skin inflammation, and cutaneous dysbiosis. Clinically, it manifests as chronic-recurrent xerosis, pruritus, and erythematous lesions. Its pathophysiology is complex, making the selection of appropriate treatment options a task.

[Source](https://jddonline.com/articles/dermatology/S1545961620P0935X)

10.36849/JDD.2020.10.36849/JDD.2020.5393

---

## Western-style diet impedes colonization and clearance of gut bacterial pathogens

[Source](https://www.biorxiv.org/content/10.1101/2020.10.06.329144v1.abstract)

---

## Taxonomic profiling of individual nematodes isolated from copse soils using deep amplicon sequencing of four distinct regions of the 18S ribosomal RNA gene.
 07.10.2020, _H Kenmotsu, K Uchida, Y Hirose, T Eki_


Nematodes are representative soil metazoans with diverged species that play crucial roles in nutrient recycling in the pedosphere. Qualitative and quantitative information on nematode communities is useful for assessing soil quality, and DNA barcode-mediated taxonomic analysis is a powerful tool to investigate taxonomic compositions and changes in nematode communities. Here, we investigated four regions (regions 1-4) of the 18S small subunit ribosomal RNA (SSU) gene as PCR targets of deep amplicon sequencing for the taxonomic profiling of individual soil nematodes. We determined the sequence variants (SVs) of 4 SSU regions for 96 nematodes (total 384 amplicons) isolated from copse soils and assigned their taxonomy using the QIIME2 software with dada2 or deblur algorithm and the SILVA database. Dada2 detected approximately 2-fold more nematode-derived SVs than deblur, and a larger number of SVs were obtained in regions 1 and 4 than those in other regions. These results and sufficient reference sequence coverage in region 4 indicated that DNA barcoding using a primer set for region 4 followed by dada2-based analysis would be most suitable for soil nematode taxonomic analysis. Eighteen SSU-derived operational taxonomic units (rOTUs) were obtained from 68 isolates, and their orders were determined based on the phylogenetic trees built by four regional sequences of rOTUs and 116 nematode reference species as well as the BLASTN search. The majority of the isolates were derived from three major orders Dorylaimida (6 rOTUs, 51.5% in 68 isolates), Rhabditida (4 rOTUs, 29.4%), and Triplonchida (7 rOTUs, 17.6%). The predicted feeding types of the isolates were fungivores (38.2% in total nematodes), plant feeders (32.4%), and 14.7% for both bacterivores and omnivores/predators. Additionally, we attempted to improve the branch structure of phylogenetic trees by using long nucleotide sequences artificially prepared by connecting regional sequences, but the effect was limited.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0240336)

10.1371/journal.pone.0240336

---

## The red flour beetle Tribolium castaneum: A model for host-microbiome interactions

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0239051)

---

## Meta-transcriptomic identification of divergent Amnoonviridae in fish

[Source](https://www.biorxiv.org/content/10.1101/2020.10.06.329003v1.abstract)

---

## Metagenome analysis from the sediment of river Ganga and Yamuna: In search of beneficial microbiome

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0239594)

---

## Thousands of previously unknown phages discovered in whole-community human gut metagenomes

[Source](https://www.biorxiv.org/content/10.1101/2020.10.07.330464v1.abstract)

---

## Metabolic pathway inference using multi-label classification with rich pathway features

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008174)

---

## Big Data in metagenomics: Apache Spark vs MPI

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0239741)

---

## <span>Like</span>

[Source](https://microbiomedigest.com/2020/10/08/october-09-2020/?like_comment=25513&_wpnonce=916262fee0)

---

