## Microbiome-associated human genetic variants impact phenome-wide disease risk.
 26.07.2022, _RHG Markowitz, AL LaBella, M Shi, A Rokas, JA Capra, JF Ferguson, JD Mosley, SR Bordenstein_


_Disease, Gastrointestinal Microbiome, Genetic Variation, Genome, Human, Humans, Phenomics, Phenotype_
Human genetic variation associates with the composition of the gut microbiome, yet its influence on clinical traits remains largely unknown. We analyzed the consequences of nearly a thousand gut microbiome-associated variants (MAVs) on phenotypes reported in electronic health records from tens of thousands of individuals. We discovered and replicated associations of MAVs with neurological, metabolic, digestive, and circulatory diseases. Five significant MAVs in these categories correlate with the relative abundance of microbes down to the strain level. We also demonstrate that these relationships are independently observed and concordant with microbe by disease associations reported in case-control studies. Moreover, a selective sweep and population differentiation impacted some disease-linked MAVs. Combined, these findings establish triad relationships among the human genome, microbiome, and disease. Consequently, human genetic influences may offer opportunities for precision diagnostics of microbiome-associated diseases but also highlight the relevance of genetic background for microbiome modulation and therapeutics.

[Source](https://www.pnas.org/doi/full/10.1073/pnas.2200551119)

10.1073/pnas.2200551119

---

## Microbial risk score for capturing microbial characteristics, integrating multi-omics data, and predicting disease risk.
 09.08.2022, _C Wang, LN Segal, J Hu, B Zhou, RB Hayes, J Ahn, H Li_


_COVID-19, Crohn Disease, Disease Susceptibility, Humans, Microbiota, Risk Factors_
With the rapid accumulation of microbiome-wide association studies, a great amount of microbiome data are available to study the microbiome's role in human disease and advance the microbiome's potential use for disease prediction. However, the unique features of microbiome data hinder its utility for disease prediction.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01310-2)

10.1186/s40168-022-01310-2

---

## A framework for testing the impact of co-infections on host gut microbiomes

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-022-00198-5)

---

## Vaginal microbiota and personal risk factors associated with HPV status conversion—A new approach to reduce the risk of cervical cancer?

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0270521)

---

## The role of the gut microbiota in multiple sclerosis

[Source](https://www.nature.com/articles/s41582-022-00697-8)

---

## Specific gut microbiota alterations in essential tremor and its difference from Parkinson’s disease

[Source](https://www.nature.com/articles/s41531-022-00359-y)

---

## Identification of clinical and ecological determinants of strain engraftment after fecal microbiota transplantation using metagenomics

[Source](https://www.sciencedirect.com/science/article/pii/S2666379122002543)

---

## Alterations in gut microbiota of esophageal squamous cell carcinoma patients

[Source](https://onlinelibrary.wiley.com/doi/full/10.1111/jgh.15941)

---

## Targeted suppression of human IBD-associated gut microbiota commensals by phage consortia for treatment of intestinal inflammation

[Source](https://www.sciencedirect.com/science/article/pii/S0092867422008509)

---

## Camel milk microbiota: A culture-independent assessment

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0963996922006871?)

---

## Effect of processing on the anti-inflammatory efficacy of cocoa in a high fat diet-induced mouse model of obesity

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0955286322001851)

---

## The impacts of viral infection and subsequent antimicrobials on the microbiome-resistome of growing pigs

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01312-0)

---

## Soil microbial community structures are shaped by agricultural systems revealing little temporal variation.
 08.08.2022, _A Fox, F Widmer, A Lüscher_


Many studies in soil microbial ecology are undertaken with a single sampling event, with the influence of temporal progression rarely being considered. Under field conditions, soil samples were taken from different agricultural systems; a sown grassland to maize rotation (MC), an intensively managed permanent grassland (INT), as well as extensively managed permanent grasslands with high (EXT_HP), low to sufficient (EXT_LP) and deficient available P (EXT_DP), six times throughout the 2017 growing season. Thus, this study aimed to determine if any differences in soil microbiome structures between both sharply contrasting (MC - INT - EXT), slightly differing (EXT_HP - EXT_DP) and quite similar (EXT_HP - EXT_LP and EXT_LP - EXT_DP) agricultural systems persist through changing growth conditions within the growing season. For both fungal and bacterial community structure, the influence of agricultural system (√CV = 0.256 and 0.145, respectively, both at least P &lt; 0.01) was much greater than that of temporal progression (√CV = 0.065 and 0.042, respectively, both P &lt; 0.001). Importantly, nearly all agricultural systems persistently harbored significantly distinct fungal community structures across each of the six sampling events (all at least P &lt; 0.05). There were not as many pairwise differences in bacterial community structure between the agricultural systems, but some did persist (MC and EXT_HP ∼ EXT_DP, all P &lt; 0.001). Additionally, persistent indicator fungal OTUs (IndVal &gt;0.7, P ≤ 0.05) associated to each agricultural system (except EXT_LP) were found in each of the six sampling events. These results highlight the temporal stability of pairwise differences in soil microbiome structures between established agricultural systems through changing plant growth conditions, even between those with a comparable management regime. This is a highly relevant finding in informing the sampling strategy of studies in soil microbial ecology as well as for designing efficient soil biodiversity monitoring systems.

[Source](https://www.sciencedirect.com/science/article/pii/S0013935122012427)

10.1016/j.envres.2022.113915

---

## The streamwater microbiome encodes hydrologic data across scales.
 09.08.2022, _DR URycki, M Bassiouni, SP Good, BC Crump, B Li_


Many fundamental questions in hydrology remain unanswered due to the limited information that can be extracted from existing data sources. Microbial communities constitute a novel type of environmental data, as they are comprised of many thousands of taxonomically and functionally diverse groups known to respond to both biotic and abiotic environmental factors. As such, these microscale communities reflect a range of macroscale conditions and characteristics, some of which also drive hydrologic regimes. Here, we assess the extent to which streamwater microbial communities (as characterized by 16S gene amplicon sequence abundance) encode information about catchment hydrology across scales. We analyzed 64 summer streamwater DNA samples collected from subcatchments within the Willamette, Deschutes, and John Day river basins in Oregon, USA, which range 0.03-29,000 km2 in area and 343-2334 mm/year of precipitation. We applied information theory to quantify the breadth and depth of information about common hydrologic metrics encoded within microbial taxa. Of the 256 microbial taxa that spanned all three watersheds, we found 9.6 % (24.5/256) of taxa, on average, shared information with a given hydrologic metric, with a median 15.6 % (range = 12.4-49.2 %) reduction in uncertainty of that metric based on knowledge of the microbial biogeography. All of the hydrologic metrics we assessed, including daily discharge at different time lags, mean monthly discharge, and seasonal high and low flow durations were encoded within the microbial community. Summer microbial taxa shared the most information with winter mean flows. Our study demonstrates quantifiable relationships between streamwater microbial taxa and hydrologic metrics at different scales, likely resulting from the integration of multiple overlapping drivers of each. Streamwater microbial communities are rich sources of information that may contribute fresh insight to unresolved hydrologic questions.

[Source](https://www.sciencedirect.com/science/article/pii/S0048969722050100)

10.1016/j.scitotenv.2022.157911

---

## Lake iTag measurements over nineteen years, introducing the limony dataset

[Source](https://www.biorxiv.org/content/10.1101/2022.08.04.502869v1.abstract)

---

