## Public disclosure of clinical trial results at Clinical Trial Registry of India- Need for transparency in research!
 16.06.2023, _R Munshi, C Pilliwar, MR Maurya_


Since June 15, 2009, clinical trial registration in the Clinical Trial Registry-India (CTRI) has been made mandatory by the Drugs Controller General of India to improve transparency, accountability, conform to accepted ethical standards and reporting of all relevant results of registered trials. In this study, we planned to evaluate the compliance of Indian and global sponsors with clinical trials conducted in India in terms of reporting of clinical trial results at the CTRI.

[Source](https://twitter.com/svetlana_up/status/1668170517487009792?ref_src=twsrc%5Etfw)

10.4103/picr.picr_39_22

---

## June 12, 2023

[Source](https://twitter.com/svetlana_up/status/1668170517487009792?ref_src=twsrc%5Etfw)

---

## Mechanism of Dyspnea during Exercise in Children with Corrected Congenital Heart Disease.
 24.01.2022, _M Chlif, MM Ammar, NB Said, L Sergey, S Ahmaidi, F Alassery, H Hamam_


_Child, Dyspnea, Exercise, Exercise Test, Heart Defects, Congenital, Humans, Oxygen Consumption_
This study will evaluate cardiorespiratory and peripheral muscle function and their relationship with subjective dyspnea threshold after the surgical correction of congenital heart disease in children. Thirteen children with surgically repaired congenital heart disease were recruited. Each participant performed an incremental exercise test on a cycle ergometer until exhaustion. Gas exchanges were continuously sampled to measure the maximal aerobic parameters and ventilatory thresholds. The functional capacity of the subjects was assessed with a 6 min walk test. At the end of the exercise test, isokinetic Cybex Norm was used to evaluate the strength and endurance of the knee extensor muscle in the leg. Dyspnea was subjectively scored with a visual analog scale during the last 15 s of each exercise step. Oxygen consumption measured at the dyspnea score/VO2 relationship located at the dyspnea threshold, at which dyspnea suddenly increased. Results: The maximal and submaximal values of the parameters describing the exercise and the peripheral muscular performances were: VO2 Peak: 33.8 ± 8.9 mL·min-1·kg-1; HR: 174 ± 9 b·min-1; VEmax: 65.68 ± 15.9 L·min-1; P max: 117 ± 27 W; maximal voluntary isometric force MVIF: 120.8 ± 41.9 N/m; and time to exhaustion Tlim: 53 ± 21 s. Oxygen consumption measured at the dyspnea threshold was related to VO2 Peak (R2 = 0.74; p &lt; 0.01), Tlim (R2 = 0.78; p &lt; 0.01), and the distance achieved during the 6MWT (R2 = 0.57; p &lt; 0.05). Compared to the theoretical maximal values for the power output, VO2, and HR, the surgical correction did not repair the exercise performance. After the surgical correction of congenital heart disease, exercise performance was impeded by alterations of the cardiorespiratory function and peripheral local factors. A subjective evaluation of the dyspnea threshold is a reliable criterion that allows the prediction of exercise capacity in subjects suffering from congenital heart disease.

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

10.3390/ijerph19010099

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/cncTaOU5IS

[Source](https://t.co/cncTaOU5IS)

---

## Tuberculosis infection and hypertension: Prevalence estimates from the US National Health and Nutrition Examination Survey.
 16.06.2023, _AD Salindri, SC Auld, UP Gujral, EM Urbina, JR Andrews, MA Huaman, MJ Magee_


Latent Tuberculosis infection (LTBI) is marked by dynamic host-pathogen interactions with persistent low-grade inflammation and is associated with increased risk of cardiovascular diseases (CVD) including acute coronary syndrome, myocardial infarction, and stroke. However, few studies assess the relationship between LTBI and hypertension, an intermediate of CVD. We sought to determine the association between LTBI and hypertension using data representative of the adult US population.

[Source](https://twitter.com/Stanford?ref_src=twsrc%5Etfw)

10.1101/2023.05.12.23289899

---

## @Stanford

[Source](https://twitter.com/Stanford?ref_src=twsrc%5Etfw)

---

## Pollination: Orchids attract unusual pollinators by means of novel chemical compounds.
 17.03.2022, _KJRP Byers_


_Animals, Bees, Flowers, Orchidaceae, Pollination, Wasps_
Sexually deceptive pollination is widespread in orchids, yet studies of its chemical basis have been mostly limited to species pollinated by bees and wasps. Two new studies in orchids demonstrate the novel chemistry of deceptive pollination by unusual pollinators.

[Source](https://twitter.com/kelsey_EJ?ref_src=twsrc%5Etfw)

10.1016/j.cub.2021.02.058

---

## @kelsey_EJ

[Source](https://twitter.com/kelsey_EJ?ref_src=twsrc%5Etfw)

---

## @UMNews

[Source](https://twitter.com/UMNews?ref_src=twsrc%5Etfw)

---

## Determination of Tr1 cell populations correlating with distinct activation states in acute IAV infection.
 15.06.2023, _CA Abbott, EL Freimayer, TS Tyllis, TS Norton, M Alsharifi, A H S Heng, SM Pederson, Z Qu, M Armstrong, GR Hill, SR McColl, I Comerford_


Type I regulatory (Tr1) cells are defined as FOXP3-IL-10-secreting CD4+ T cells that contribute to immune suppression and typically express the markers LAG-3 and CD49b and other co-inhibitory receptors. These cells have not been studied in detail the context of the resolution of acute infection in the lung. Here, we identify FOXP3-IL-10+ CD4+ T cells transiently accumulating in the lung parenchyma during resolution of the response to sublethal influenza A virus (IAV) infection in mice. These cells were dependent on IL-27Rα, which was required for timely recovery from IAV-induced weight loss. LAG-3 and CD49b were not generally co-expressed by FOXP3- IL-10+ CD4+ T cells in this model and four populations of these cells based on LAG-3 and CD49b co-expression were apparent (LAG-3-CD49b- [DN], LAG-3+CD49b+ [DP], LAG-3+CD49b- [LAG-3+], LAG-3-CD49b+ [CD49b+]). However, each population exhibited suppressive potential consistent with the definition of Tr1 cells. Notably, differences between these populations of Tr1 cells were apparent including differential dependence on IL-10 to mediate suppression and expression of markers indicative of different activation states and terminal differentiation. Sort-transfer experiments indicated that LAG-3+ Tr1 cells exhibited the capacity to convert to DN and DP Tr1 cells, indicative of plasticity between these populations. Together, these data determine the features and suppressive potential of Tr1 cells in the resolution of IAV infection and identify four populations delineated by LAG-3 and CD49b, which likely correspond to different Tr1 cell activation states.

[Source](https://twitter.com/telethonkids?ref_src=twsrc%5Etfw)

10.1016/j.mucimm.2023.06.003

---

## @telethonkids

[Source](https://twitter.com/telethonkids?ref_src=twsrc%5Etfw)

---

## Obesity: Policy and Practice Recommendations for High-Risk Populations Influenced by the COVID-19 Pandemic.
 22.12.2022, _KN Robinson, DA Saber_


_Humans, United States, COVID-19, SARS-CoV-2, Pandemics, Influenza A Virus, H1N1 Subtype, Obesity, Policy_
Obesity has been linked to higher morbidity and mortality in those that contract novel viruses, such as H1N1 and SARS-CoV-2. Loss of life and the high cost of obesity highlight the need to focus on preventative measures. The state of Maine (United States) is profiled as a rural state with high rates of obesity and low health literacy that benefits from programs to improve health. However, more multidirectional efforts are needed. Four recommendations for building a healthier, more resilient patient population are discussed here: (i) state incentives and subsidies, (ii) employers to incentivize healthy living, (iii) federal incentives and initiatives, and (iv) personal responsibility for health and wellness.

[Source](https://twitter.com/UMaine?ref_src=twsrc%5Etfw)

10.1128/msystems.00089-22

---

## @UMaine

[Source](https://twitter.com/UMaine?ref_src=twsrc%5Etfw)

---

## @OHSUNews

[Source](https://twitter.com/OHSUNews?ref_src=twsrc%5Etfw)

---

## https://t.co/VeplbbgGqf

[Source](https://t.co/VeplbbgGqf)

---

## pic.twitter.com/KLCsbs63gr

[Source](https://t.co/KLCsbs63gr)

---

## June 5, 2023

[Source](https://twitter.com/MicrobiomeVIF/status/1665656925969604608?ref_src=twsrc%5Etfw)

---

## Guild and Niche Determination Enable Targeted Alteration of the Microbiome

[Source](https://www.biorxiv.org/content/10.1101/2023.05.11.540389v1)

---

## https://t.co/PgEq0DSqFe

[Source](https://t.co/PgEq0DSqFe)

---

## Implementation of an Emergency Department Back Pain Clinical Management Tool on the Early Diagnosis and Testing of Spinal Epidural Abscess.
 16.06.2023, _AG Cai, MS Zocchi, JN Carlson, J Bedolla, JM Pines_


Spinal epidural abscess (SEA) is a rare, catastrophic condition for which diagnostic delays are common. Our national group develops evidence-based guidelines, known as Clinical Management Tools (CMT), to reduce high risk misdiagnoses. We study whether implementation of our back pain CMT improved SEA diagnostic timeliness and testing rates in the emergency department (ED).

[Source](https://twitter.com/OrianeMoyne/status/1657155454093410304?ref_src=twsrc%5Etfw)

10.1111/acem.14765

---

## May 12, 2023

[Source](https://twitter.com/OrianeMoyne/status/1657155454093410304?ref_src=twsrc%5Etfw)

---

## Biomolecular Markers of Recurrent Implantation Failure-A Review.
 19.11.2021, _AE Mrozikiewicz, M Ożarowski, P Jędrzejczak_


_Adult, Biomarkers, Blastocyst, Embryo Implantation, Endometrium, Estrogens, Female, Glycodelin, Humans, Immune System, Immunologic Factors, Infertility, Female, Leukemia Inhibitory Factor, Microbiota, Neovascularization, Pathologic, Progesterone, Receptors, Estrogen, Receptors, Progesterone, Recurrence, Risk Factors, Vagina_
Currently, infertility affects 8-12% of reproductive age couples worldwide, a problem that also affects women suffering from recurrent implantation failure (RIF). RIF is a complex condition resulting from many physiological and molecular mechanisms involving dynamic endometrium-blastocyst interaction. The most important are the endometrial receptivity process, decidualization, trophoblast invasion, and blastocyst nesting. Although the exact multifactorial pathogenesis of RIF remains unclear, many studies have suggested the association between hormone level imbalance, disturbances of angiogenic and immunomodulatory factors, certain genetic polymorphisms, and occurrence of RIF. These studies were performed in quite small groups. Additionally, the results are inconsistent between ethnicities. The present review briefly summarizes the importance of factors involved in RIF development that could also serve as diagnostic determinants. Moreover, our review could constitute part of a new platform for discovery of novel diagnostic and therapeutic solutions for RIF.

[Source](https://www.biorxiv.org/content/10.1101/2023.06.14.544778v1)

10.3390/ijms221810082

---

## Interaction of genital microbiota in infertile couples

[Source](https://www.biorxiv.org/content/10.1101/2023.06.14.544778v1)

---

## Therapy of replenishing and regulating for manic episode in bipolar disorder: study protocol for a prospective, double-blind, randomized controlled trial.
 26.05.2023, _Y Yang, Y Haining, J Hongxiao, N Yanzhe, W Di, Z Lei, Y Kaijuan, G Yumeng, W Fei, S Weishuang, C Pei_


_Humans, Adolescent, Bipolar Disorder, Mania, Quality of Life, Prospective Studies, Double-Blind Method, Treatment Outcome, Randomized Controlled Trials as Topic_
Bipolar disorder (BD) is a chronic and recurrent disorder characterized by biphasic mood episodes of mania or hypomania and depression. It affects more than 1% of the global population and is a leading cause of disability in young people. Currently available treatments for BD are still fairly limited in terms of efficacy, with high rates of non-adherence, non-response, and undesirable side effects. Traditional Chinese medicine (TCM) has a long history and rich experience in stabilizing mania and improving quality of life. Aiming at rebalancing and in BD, therapy of replenishing and regulating (RYRY therapy) has been in clinical use for years in China. The present prospective, double-blind, randomized controlled trial is designed to investigate the efficacy and safety of RYRY therapy for bipolar mania and its possible mechanism from the point of regulating gut microbiota and anti-inflammation. A total of 60 eligible participants will be recruited from Beijing Anding Hospital. They will be randomized to either the study group or the control group in a ratio of 1∶1. Participants allocated to the study group will receive RYRY granule, while placebo granule will be applied in the control group. Participants in both groups will be prescribed conventional therapy for manic episode in BD. Four scheduled visits will be conducted over 4 weeks. Outcome measurements include Young Mania Rating Scale, TCM Symptom Pattern Rating Scale, Treatment Emergent Symptom Scale, levels of C-reactive protein, interleukin-6 and tumor necrosis factor-α and the gut microbial community profile of stool samples. Safety outcomes and adverse events will also be recorded. In this study, we set a number of scientific and objective assessments to evaluate the efficacy of RYRY therapy and study into its possible mechanism, hopefully offering clinicians an alternative approach to BD.

[Source](https://www.biorxiv.org/content/10.1101/2023.06.13.544832v1)

10.19852/j.cnki.jtcm.20230328.002

---

## Emergence of community behaviors in the gut microbiota upon drug treatment

[Source](https://www.biorxiv.org/content/10.1101/2023.06.13.544832v1)

---

## @BorkLab

[Source](https://twitter.com/BorkLab?ref_src=twsrc%5Etfw)

---

## https://t.co/EuZH2kkJNE

[Source](https://t.co/EuZH2kkJNE)

---

## June 15, 2023

[Source](https://twitter.com/SarSarela/status/1669238869437431808?ref_src=twsrc%5Etfw)

---

## Skin microbiome differentiates into distinct cutotypes with unique metabolic functions upon exposure to polycyclic aromatic hydrocarbons

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01564-4)

---

## Altitude-dependent agro-ecologies impact the microbiome diversity of scavenging indigenous chicken in Ethiopia

[Source](https://www.biorxiv.org/content/10.1101/2023.06.12.544316v1)

---

## @OlivierHanotte

[Source](https://twitter.com/OlivierHanotte?ref_src=twsrc%5Etfw)

---

## #microbiota

[Source](https://twitter.com/hashtag/microbiota?src=hash&ref_src=twsrc%5Etfw)

---

## #poultry

[Source](https://twitter.com/hashtag/poultry?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/gWabXzr6oQ

[Source](https://t.co/gWabXzr6oQ)

---

## Review on adherence of the literature to official recommendations on albuminuria harmonization and standardization.
 16.06.2023, _M Mussap, M Sortino, E Monteverde, R Tomaiuolo, G Banfi, M Locatelli, A Carobene_


Albuminuria standardization is a key issue to produce reliable and equivalent results between laboratories. We investigated whether official recommendations on albuminuria harmonization are followed in the literature. The PubMed database was searched from June 1 to September 26, 2021. The search terms included urine albumin, UACR, and albuminuria. A total of 159 articles were considered eligible; 50.9 % reported the type of urine collection. Specifically, 58.1 % collected a random spot urine specimen, 21 % collected a first morning void, and 6.2 % collected a 24-h specimen. Overall, 15 % of articles reported data on sample shipping, storage, and centrifugation and 13.3 % mentioned the preanalytical phase without any data on albuminuria. The method for albuminuria was properly described in 31.4 % of articles; of these, 54.9 % used immunological methods, and 8.9 % contained errors or missing data. Most articles (76.7 %) expressed test results as albuminuria-to-creatininuria ratio. Different decision levels were utilized in 130 articles; of these, 36 % used a decision level of ≤30 mg/g creatininuria and 23.7 % used three decision levels (≤30, 30-300, and ≥300 mg/g). The failure to follow guidelines on albuminuria harmonization was mainly found in the preanalytical phase. The poor awareness of the importance of preanalytical steps on test result may be a possible explanation.

[Source](https://twitter.com/Microbes4ever/status/1668524788044767233?ref_src=twsrc%5Etfw)

10.1515/cclm-2023-0408

---

## June 13, 2023

[Source](https://twitter.com/Microbes4ever/status/1668524788044767233?ref_src=twsrc%5Etfw)

---

## Effects of fenbendazole on fecal microbiome in BPH/5 mice, a model of hypertension and obesity, a brief report 

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0287145)

---

## Gut microbiota and cognitive development in infant mice: Quantity and source of potable water.
 16.06.2023, _CS Kim, DM Shin_


_Animals, Mice, Drinking Water, Gastrointestinal Microbiome, Dehydration, Cognition, Cognitive Dysfunction, Bacteroidetes, Firmicutes_
Not only the water quantity consumed but also the source of drinking water has been considered for their health benefits, but there is limited evidence. We aimed to determine whether the amount and type of drinking water affect physiological and biological functions, including brain function, by confirming how it affects gut microbiota which has an important regulatory role in host physiology. Three-week-old infant mice were subjected to 1) a water restriction experiment (control group, ad libitum consumption of distilled water; dehydration group, time-limited access to distilled water [15 min/day]) and 2) different water source experiment (distilled water, purified water, spring water, and tap water groups). The gut microbiota and cognitive development were analyzed using the 16S ribosomal ribonucleic acid sequencing method and the Barnes maze, respectively. The relative abundance of Firmicutes and Bacteroidetes and the Firmicutes-to-Bacteroidetes ratio (F/B ratio) changed depending on age (juveniles vs. infants). Insufficient water intake reversed these developmental changes, showing that the relative abundances of Bacteroidetes and Firmicutes and the F/B ratio in dehydrated juvenile mice were similar to those in normal infant mice. Additionally, clustering analysis revealed no significant differences in the intestinal flora in the mice from the different drinking water sources; however, dehydration significantly altered the composition of the genera compared to the other water source groups wherein water was provided ad libitum. Moreover, cognitive development was significantly disrupted by insufficient water intake, although the type of drinking water had no significant influence. Cognitive decline, measured by relative latency, was positively associated with the relative abundance of unclassified Erysipelotrichaceae that were in significantly high relative abundance in the dehydration group. These results suggest that the water quantity consumed, rather than the mineral content of drinking water, is imperative for shaping the early gut microbiota associated with cognitive development during infancy.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0286951)

10.1371/journal.pone.0286951

---

## Globally consistent response of plant microbiome diversity across hosts and continents to soil nutrients and herbivores.
 16.06.2023, _EW Seabloom, MC Caldeira, KF Davies, L Kinkel, JMH Knops, KJ Komatsu, AS MacDougall, G May, M Millican, JL Moore, LI Perez, AJ Porath-Krause, SA Power, SM Prober, AC Risch, C Stevens, ET Borer_


_Herbivory, Biomass, Microbiota, Nutrients, Soil_
All multicellular organisms host a diverse microbiome composed of microbial pathogens, mutualists, and commensals, and changes in microbiome diversity or composition can alter host fitness and function. Nonetheless, we lack a general understanding of the drivers of microbiome diversity, in part because it is regulated by concurrent processes spanning scales from global to local. Global-scale environmental gradients can determine variation in microbiome diversity among sites, however an individual host's microbiome also may reflect its local micro-environment. We fill this knowledge gap by experimentally manipulating two potential mediators of plant microbiome diversity (soil nutrient supply and herbivore density) at 23 grassland sites spanning global-scale gradients in soil nutrients, climate, and plant biomass. Here we show that leaf-scale microbiome diversity in unmanipulated plots depended on the total microbiome diversity at each site, which was highest at sites with high soil nutrients and plant biomass. We also found that experimentally adding soil nutrients and excluding herbivores produced concordant results across sites, increasing microbiome diversity by increasing plant biomass, which created a shaded microclimate. This demonstration of consistent responses of microbiome diversity across a wide range of host species and environmental conditions suggests the possibility of a general, predictive understanding of microbiome diversity.

[Source](https://www.nature.com/articles/s41467-023-39179-w)

10.1038/s41467-023-39179-w

---

## Achieving pan-microbiome biological insights via the dbBact knowledge base.
 16.06.2023, _A Amir, E Ozel, Y Haberman, N Shental_


16S rRNA amplicon sequencing provides a relatively inexpensive culture-independent method for studying microbial communities. Although thousands of such studies have examined diverse habitats, it is difficult for researchers to use this vast trove of experiments when interpreting their own findings in a broader context. To bridge this gap, we introduce dbBact - a novel pan-microbiome resource. dbBact combines manually curated information from studies across diverse habitats, creating a collaborative central repository of 16S rRNA amplicon sequence variants (ASVs), which are assigned multiple ontology-based terms. To date dbBact contains information from more than 1000 studies, which include 1500000 associations between 360000 ASVs and 6500 ontology terms. Importantly, dbBact offers a set of computational tools allowing users to easily query their own datasets against the database. To demonstrate how dbBact augments standard microbiome analysis we selected 16 published papers, and reanalyzed their data via dbBact. We uncovered novel inter-host similarities, potential intra-host sources of bacteria, commonalities across different diseases and lower host-specificity in disease-associated bacteria. We also demonstrate the ability to detect environmental sources, reagent-borne contaminants, and identify potential cross-sample contaminations. These analyses demonstrate how combining information across multiple studies and over diverse habitats leads to better understanding of underlying biological processes.

[Source](https://twitter.com/hashtag/diverse?src=hash&ref_src=twsrc%5Etfw)

10.1093/nar/gkad527

---

## #diverse

[Source](https://twitter.com/hashtag/diverse?src=hash&ref_src=twsrc%5Etfw)

---

## Achieving pan-microbiome biological insights via the dbBact knowledge base.
 16.06.2023, _A Amir, E Ozel, Y Haberman, N Shental_


16S rRNA amplicon sequencing provides a relatively inexpensive culture-independent method for studying microbial communities. Although thousands of such studies have examined diverse habitats, it is difficult for researchers to use this vast trove of experiments when interpreting their own findings in a broader context. To bridge this gap, we introduce dbBact - a novel pan-microbiome resource. dbBact combines manually curated information from studies across diverse habitats, creating a collaborative central repository of 16S rRNA amplicon sequence variants (ASVs), which are assigned multiple ontology-based terms. To date dbBact contains information from more than 1000 studies, which include 1500000 associations between 360000 ASVs and 6500 ontology terms. Importantly, dbBact offers a set of computational tools allowing users to easily query their own datasets against the database. To demonstrate how dbBact augments standard microbiome analysis we selected 16 published papers, and reanalyzed their data via dbBact. We uncovered novel inter-host similarities, potential intra-host sources of bacteria, commonalities across different diseases and lower host-specificity in disease-associated bacteria. We also demonstrate the ability to detect environmental sources, reagent-borne contaminants, and identify potential cross-sample contaminations. These analyses demonstrate how combining information across multiple studies and over diverse habitats leads to better understanding of underlying biological processes.

[Source](https://twitter.com/hashtag/microbiome?src=hash&ref_src=twsrc%5Etfw)

10.1093/nar/gkad527

---

## #microbiome

[Source](https://twitter.com/hashtag/microbiome?src=hash&ref_src=twsrc%5Etfw)

---

## Real-time genomics for One Health.
 16.06.2023, _L Urban, A Perlas, O Francino, J Martí-Carreras, BA Muga, JW Mwangi, L Boykin Okalebo, JL Stanton, A Black, N Waipara, C Fontsere, D Eccles, H Urel, T Reska, HE Morales, M Palmada-Flores, T Marques-Bonet, M Watsa, Z Libke, G Erkenswick, C van Oosterhout_


The ongoing degradation of natural systems and other environmental changes has put our society at a crossroad with respect to our future relationship with our planet. While the concept of One Health describes how human health is inextricably linked with environmental health, many of these complex interdependencies are still not well-understood. Here, we describe how the advent of real-time genomic analyses can benefit One Health and how it can enable timely, in-depth ecosystem health assessments. We introduce nanopore sequencing as the only disruptive technology that currently allows for real-time genomic analyses and that is already being used worldwide to improve the accessibility and versatility of genomic sequencing. We showcase real-time genomic studies on zoonotic disease, food security, environmental microbiome, emerging pathogens, and their antimicrobial resistances, and on environmental health itself - from genomic resource creation for wildlife conservation to the monitoring of biodiversity, invasive species, and wildlife trafficking. We stress why equitable access to real-time genomics in the context of One Health will be paramount and discuss related practical, legal, and ethical limitations.

[Source](https://twitter.com/hashtag/biodiversity?src=hash&ref_src=twsrc%5Etfw)

10.15252/msb.202311686

---

## #biodiversity

[Source](https://twitter.com/hashtag/biodiversity?src=hash&ref_src=twsrc%5Etfw)

---

## Achieving pan-microbiome biological insights via the dbBact knowledge base.
 16.06.2023, _A Amir, E Ozel, Y Haberman, N Shental_


16S rRNA amplicon sequencing provides a relatively inexpensive culture-independent method for studying microbial communities. Although thousands of such studies have examined diverse habitats, it is difficult for researchers to use this vast trove of experiments when interpreting their own findings in a broader context. To bridge this gap, we introduce dbBact - a novel pan-microbiome resource. dbBact combines manually curated information from studies across diverse habitats, creating a collaborative central repository of 16S rRNA amplicon sequence variants (ASVs), which are assigned multiple ontology-based terms. To date dbBact contains information from more than 1000 studies, which include 1500000 associations between 360000 ASVs and 6500 ontology terms. Importantly, dbBact offers a set of computational tools allowing users to easily query their own datasets against the database. To demonstrate how dbBact augments standard microbiome analysis we selected 16 published papers, and reanalyzed their data via dbBact. We uncovered novel inter-host similarities, potential intra-host sources of bacteria, commonalities across different diseases and lower host-specificity in disease-associated bacteria. We also demonstrate the ability to detect environmental sources, reagent-borne contaminants, and identify potential cross-sample contaminations. These analyses demonstrate how combining information across multiple studies and over diverse habitats leads to better understanding of underlying biological processes.

[Source](https://twitter.com/hashtag/experiment?src=hash&ref_src=twsrc%5Etfw)

10.1093/nar/gkad527

---

## #experiment

[Source](https://twitter.com/hashtag/experiment?src=hash&ref_src=twsrc%5Etfw)

---

## @NatureComms

[Source](https://twitter.com/NatureComms?ref_src=twsrc%5Etfw)

---

## @umncbs

[Source](https://twitter.com/umncbs?ref_src=twsrc%5Etfw)

---

## @niooknaw

[Source](https://twitter.com/niooknaw?ref_src=twsrc%5Etfw)

---

## @NutNetGlobal

[Source](https://twitter.com/NutNetGlobal?ref_src=twsrc%5Etfw)

---

## Divergence in cell cycle progression is associated with shifted phenology in a multivoltine moth: the European corn borer, Ostrinia nubilalis.
 15.06.2023, _Q Xia, C Chen, EB Dopman, DA Hahn_


_Animals, Moths, Zea mays, Larva, Diapause, Cell Division_
Evolutionary change in diapause timing can be an adaptive response to changing seasonality, and even result in ecological speciation. However, the molecular and cellular mechanisms regulating shifts in diapause timing remain poorly understood. One of the hallmarks of diapause is a massive slowdown in the cell cycle of target organs such as the brain and primordial imaginal structures, and resumption of cell cycle proliferation is an indication of diapause termination and resumption of development. Characterizing cell cycle parameters between lineages differing in diapause life history timing may help identify molecular mechanisms associated with alterations of diapause timing. We tested the extent to which progression of the cell cycle differs across diapause between two genetically distinct European corn borer strains that differ in their seasonal diapause timing. We show the cell cycle slows down during larval diapause with a significant decrease in the proportion of cells in S phase. Brain-subesophageal complex cells slow primarily in G0/G1 phase whereas most wing disc cells are in G2 phase. Diapausing larvae of the earlier emerging, bivoltine E-strain (BE) suppressed cell cycle progression less than the later emerging, univoltine Z-strain (UZ) individuals, with a greater proportion of cells in S phase across both tissues during diapause. Additionally, resumption of cell cycle proliferation occurred earlier in the BE strain than in the UZ strain after exposure to diapause-terminating conditions. We propose that regulation of cell cycle progression rates ultimately drives differences in larval diapause termination, and adult emergence timing, between early- and late-emerging European corn borer strains.

[Source](https://twitter.com/e_borer?ref_src=twsrc%5Etfw)

10.1242/jeb.245244

---

## @e_borer

[Source](https://twitter.com/e_borer?ref_src=twsrc%5Etfw)

---

## https://t.co/TEhvN2Fxge

[Source](https://t.co/TEhvN2Fxge)

---

## Review on adherence of the literature to official recommendations on albuminuria harmonization and standardization.
 16.06.2023, _M Mussap, M Sortino, E Monteverde, R Tomaiuolo, G Banfi, M Locatelli, A Carobene_


Albuminuria standardization is a key issue to produce reliable and equivalent results between laboratories. We investigated whether official recommendations on albuminuria harmonization are followed in the literature. The PubMed database was searched from June 1 to September 26, 2021. The search terms included urine albumin, UACR, and albuminuria. A total of 159 articles were considered eligible; 50.9 % reported the type of urine collection. Specifically, 58.1 % collected a random spot urine specimen, 21 % collected a first morning void, and 6.2 % collected a 24-h specimen. Overall, 15 % of articles reported data on sample shipping, storage, and centrifugation and 13.3 % mentioned the preanalytical phase without any data on albuminuria. The method for albuminuria was properly described in 31.4 % of articles; of these, 54.9 % used immunological methods, and 8.9 % contained errors or missing data. Most articles (76.7 %) expressed test results as albuminuria-to-creatininuria ratio. Different decision levels were utilized in 130 articles; of these, 36 % used a decision level of ≤30 mg/g creatininuria and 23.7 % used three decision levels (≤30, 30-300, and ≥300 mg/g). The failure to follow guidelines on albuminuria harmonization was mainly found in the preanalytical phase. The poor awareness of the importance of preanalytical steps on test result may be a possible explanation.

[Source](https://twitter.com/e_seabloom/status/1669298473085140992?ref_src=twsrc%5Etfw)

10.1515/cclm-2023-0408

---

## June 15, 2023

[Source](https://twitter.com/e_seabloom/status/1669298473085140992?ref_src=twsrc%5Etfw)

---

## Processing and Storage Methods Affect Oral and Gut Microbiome Composition 

[Source](https://www.biorxiv.org/content/10.1101/2023.06.13.544865v1)

---

## QMRA of beach water by Nanopore sequencing-based viability-metagenomics absolute quantification.
 18.04.2023, _Y Yang, Y Deng, X Shi, L Liu, X Yin, W Zhao, S Li, C Yang, T Zhang_


_Humans, Microbial Viability, Metagenomics, Reproducibility of Results, Nanopore Sequencing, Propidium, Azides, Risk Assessment, Bacteria, Escherichia coli_
The majority of the current regulatory practices for routine monitoring of beach water quality rely on the culture-based enumeration of faecal indicator bacteria (FIB) to develop criteria for promoting the general public's health. To address the limitations of culture methods and the arguable reliability of FIB in indicating health risks, we developed a Nanopore metagenomic sequencing-based viable cell absolute quantification workflow to rapidly and accurately estimate a broad range of microbes in beach waters by a combination of propidium monoazide (PMA) and cellular spike-ins. Using the simple synthetic bacterial communities mixed with viable and heat-killed cells, we observed near-complete relic DNA removal by PMA with minimal disturbance to the composition of viable cells, demonstrating the feasibility of PMA treatment in profiling viable cells by Nanopore sequencing. On a simple mock community comprised of 15 prokaryotic species, our results showed high accordance between the expected and estimated concentrations, suggesting the accuracy of our method in absolute quantification. We then further assessed the accuracy of our method for counting viable Escherichia coli and Vibrio spp. in beach waters by comparing to culture-based method, which were also in high agreement. Furthermore, we demonstrated that 1 Gb sequences obtained within 2 h would be sufficient to quantify a species having a concentration of ≥ 10 cells/mL in beach waters. Using our viability-resolved quantification workflow to assess the microbial risk of the beach water, we conducted (1) screening-level quantitative microbial risk assessment (QMRA) to investigate human illness risk and site-specific risk patterns that might guide risk management efforts and (2) metagenomics-based resistome risk assessment to evaluate another layer of risk caused by difficult illness treatment due to antimicrobial resistance (AMR). In summary, our metagenomic workflow for the rapid absolute quantification of viable bacteria demonstrated its great potential in paving new avenues toward holistic microbial risk assessment.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0043135423002932)

10.1016/j.watres.2023.119858

---

## ResFinderFG v2.0: a database of antibiotic resistance genes obtained by functional metagenomics.
 19.05.2023, _R Gschwind, S Ugarcina Perovic, M Weiss, M Petitjean, J Lao, LP Coelho, E Ruppé_


Metagenomics can be used to monitor the spread of antibiotic resistance genes (ARGs). ARGs found in databases such as ResFinder and CARD primarily originate from culturable and pathogenic bacteria, while ARGs from non-culturable and non-pathogenic bacteria remain understudied. Functional metagenomics is based on phenotypic gene selection and can identify ARGs from non-culturable bacteria with a potentially low identity shared with known ARGs. In 2016, the ResFinderFG v1.0 database was created to collect ARGs from functional metagenomics studies. Here, we present the second version of the database, ResFinderFG v2.0, which is available on the Center of Genomic Epidemiology web server (https://cge.food.dtu.dk/services/ResFinderFG/). It comprises 3913 ARGs identified by functional metagenomics from 50 carefully curated datasets. We assessed its potential to detect ARGs in comparison to other popular databases in gut, soil and water (marine + freshwater) Global Microbial Gene Catalogues (https://gmgc.embl.de). ResFinderFG v2.0 allowed for the detection of ARGs that were not detected using other databases. These included ARGs conferring resistance to beta-lactams, cycline, phenicol, glycopeptide/cycloserine and trimethoprim/sulfonamide. Thus, ResFinderFG v2.0 can be used to identify ARGs differing from those found in conventional databases and therefore improve the description of resistomes.

[Source](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkad384/7173762)

10.1093/nar/gkad384

---

## ResFinderFG v2.0: a database of antibiotic resistance genes obtained by functional metagenomics – Rémi Gschwind 

[Source](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkad384/7173762)

---

## Achieving pan-microbiome biological insights via the dbBact knowledge base.
 16.06.2023, _A Amir, E Ozel, Y Haberman, N Shental_


16S rRNA amplicon sequencing provides a relatively inexpensive culture-independent method for studying microbial communities. Although thousands of such studies have examined diverse habitats, it is difficult for researchers to use this vast trove of experiments when interpreting their own findings in a broader context. To bridge this gap, we introduce dbBact - a novel pan-microbiome resource. dbBact combines manually curated information from studies across diverse habitats, creating a collaborative central repository of 16S rRNA amplicon sequence variants (ASVs), which are assigned multiple ontology-based terms. To date dbBact contains information from more than 1000 studies, which include 1500000 associations between 360000 ASVs and 6500 ontology terms. Importantly, dbBact offers a set of computational tools allowing users to easily query their own datasets against the database. To demonstrate how dbBact augments standard microbiome analysis we selected 16 published papers, and reanalyzed their data via dbBact. We uncovered novel inter-host similarities, potential intra-host sources of bacteria, commonalities across different diseases and lower host-specificity in disease-associated bacteria. We also demonstrate the ability to detect environmental sources, reagent-borne contaminants, and identify potential cross-sample contaminations. These analyses demonstrate how combining information across multiple studies and over diverse habitats leads to better understanding of underlying biological processes.

[Source](https://twitter.com/NAR_Open?ref_src=twsrc%5Etfw)

10.1093/nar/gkad527

---

## @NAR_Open

[Source](https://twitter.com/NAR_Open?ref_src=twsrc%5Etfw)

---

## https://t.co/xhd0DqpImV

[Source](https://t.co/xhd0DqpImV)

---

## https://t.co/zyHksEbpfs

[Source](https://t.co/zyHksEbpfs)

---

## May 22, 2023

[Source](https://twitter.com/GschwindRemi/status/1660622312511619075?ref_src=twsrc%5Etfw)

---

