## Phenotypic and genomic diversification in complex carbohydrate degrading human gut bacteria

[Source](https://www.biorxiv.org/content/10.1101/2021.07.15.452266v1.abstract?%3Fcollection=)

---

## Effect of biochanin A on the rumen microbial community of Holstein steers consuming a high fiber diet and subjected to a subacute acidosis challenge.
 21.07.2021, _BE Harlow, MD Flythe, JL Klotz, DL Harmon, GE Aiken_


Subacute rumen acidosis (SARA) occurs when highly fermentable carbohydrates are introduced into the diet, decreasing pH and disturbing the microbial ecology of the rumen. Rumen amylolytic bacteria rapidly catabolize starch, fermentation acids accumulate in the rumen and reduce environmental pH. Historically, antibiotics (e.g., monensin, MON) have been used in the prevention and treatment of SARA. Biochanin A (BCA), an isoflavone produced by red clover (Trifolium pratense), mitigates changes associated with starch fermentation ex vivo. The objective of the study was to determine the effect of BCA on amylolytic bacteria and rumen pH during a SARA challenge. Twelve rumen fistulated steers were assigned to 1 of 4 treatments: HF CON (high fiber control), SARA CON, MON (200 mg d-1), or BCA (6 g d-1). The basal diet consisted of corn silage and dried distiller's grains ad libitum. The study consisted of a 2-wk adaptation, a 1-wk HF period, and an 8-d SARA challenge (d 1-4: 40% corn; d 5-8: 70% cracked corn). Samples for pH and enumeration were taken on the last day of each period (4 h). Amylolytic, cellulolytic, and amino acid/peptide-fermenting bacteria (APB) were enumerated. Enumeration data were normalized by log transformation and data were analyzed by repeated measures ANOVA using the MIXED procedure of SAS. The SARA challenge increased total amylolytics and APB, but decreased pH, cellulolytics, and in situ DMD of hay (P &lt; 0.05). BCA treatment counteracted the pH, microbiological, and fermentative changes associated with SARA challenge (P &lt; 0.05). Similar results were also observed with MON (P &lt; 0.05). These results indicate that BCA may be an effective alternative to antibiotics for mitigating SARA in cattle production systems.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0253754)

10.1371/journal.pone.0253754

---

## Ear wound healing in MRL/MpJ mice is associated with gut microbiome composition and is transferable to non-healer mice via microbiome transplantation.
 20.07.2021, _C Velasco, C Dunn, C Sturdy, V Izda, J Martin, A Rivas, J McNaughton, MA Jeffries_


Adult elastic cartilage has limited repair capacity. MRL/MpJ (MRL) mice, by contrast, are capable of spontaneously healing ear punctures. This study was undertaken to characterize microbiome differences between healer and non-healer mice and to evaluate whether this healing phenotype can be transferred via gut microbiome transplantation.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0248322)

10.1371/journal.pone.0248322

---

## Increased microbial diversity and decreased prevalence of common pathogens in the gut microbiomes of wild turkeys compared to domestic turkeys

[Source](https://www.biorxiv.org/content/10.1101/2021.07.16.452759v1.abstract?%3Fcollection=)

---

## Interspecific variation and functional traits of the gut microbiome in spiders from the wild: The largest effort so far.
 21.07.2021, _K Tyagi, I Tyagi, V Kumar_


Spiders being one of the most diverse group in phylum arthropod are of great importance due to their role as predators, silk producer, and in medicinal applications. Spiders in prey-predator relationships play a crucial role in balancing the food-chain of any ecosystem; therefore it is essential to characterize the gut microbiota of spiders collected from natural environments. In the present work, the largest effort so far has been made to characterize the gut microbiota of 35 spider species belonging to four different families using 16S amplicon targeting sequencing. Further, we compared the gut microbiota composition including endosymbiont abundance in spider species collected from different geographical locations. The results obtained revealed the presence of genera like Acinetobacter (15%), V7clade (9%), Wolbachia (8%), Pseudomonas (5%), Bacillus (6%). Although comparative analysis revealed that the gut bacterial composition in all the spider families has a similar pattern, in terms of community richness and evenness. The bacterial diversity in the spider family, Lycosidae are more diverse than in Salticidae, Tetragnathidae and Araneidae. Furthermore, it was observed that the abundance of endosymbiont genera, i.e. Wolbachia and Rickettsia, leads to shift in the abundance of other bacterial taxa and may cause sexual alterations in spider species. Moreover, predicted functional analysis based on PICRUSt2 reveals that gut microbiota of spider species were involved in functions like metabolism of carbohydrates, cofactors and vitamins, amino acids; biosynthesis of organic compounds, fatty acids, lipids etc. Based on the results obtained, it can be said that different locations do not correlate with community composition of gut microbiota in spider species collected from natural environments.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0251790)

10.1371/journal.pone.0251790

---

## Tillage intensity and plant rhizosphere selection shape bacterial-archaeal assemblage diversity and nitrogen cycling genes

[Source](https://www.biorxiv.org/content/10.1101/2021.07.16.452714v1.abstract?%3Fcollection=)

---

## An Ohio State Scenic River Shows Elevated Antibiotic Resistance Genes, Including Acinetobacter Tetracycline and Macrolide Resistance, Downstream of Wastewater Treatment Plant Effluent

[Source](https://www.biorxiv.org/content/10.1101/2021.04.26.441562v3.abstract?%3Fcollection=)

---

## Biogeochemistry of the Antrim Shale Natural Gas Reservoir

[Source](https://pubs.acs.org/doi/full/10.1021/acsearthspacechem.1c00087)

---

## Novel municipal sewage-associated bacterial genomes and their potential in source tracking

[Source](https://www.biorxiv.org/content/10.1101/2021.07.15.452399v1.abstract?%3Fcollection=)

---

## Structure of bacterial communities in Japanese-style bathrooms: Comparative sequencing of bacteria in shower water and showerhead biofilms using a portable nanopore sequencer

[Source](https://www.biorxiv.org/content/10.1101/2021.07.14.452346v1.abstract?%3Fcollection=)

---

## Dynamic surveillance of mosquitoes and their viromes in Wuhan during 2020

[Source](https://www.biorxiv.org/content/10.1101/2021.07.17.452774v1.abstract?%3Fcollection=)

---

## Simple mapping-based quantification of a mock microbial community using total RNA-seq data.
 19.07.2021, _S Moriya_


Most microbes in the natural environment are difficult to cultivate. Thus, culture-independent analysis for microbial community structure is important for the understanding of its ecological functions. An immense ribosomal RNA sequence collection is available from phylogenetic research on organisms in all domains. These sequences are available for use in genetic research. However, the amplicon-seq process using PCR requires the construction of a sequence library. Construction can introduce bias into quantitative analyses, and each domain of species needs its own primer set. Total RNA sequencing has the advantage of analyzing an entire microbial community, including bacteria, archea, and eukaryote, at once. Such analysis yields large amounts of ribosomal RNA sequences that can be used for analysis without PCR bias. Evaluation using total RNA-seq for quantitative analysis of microbial communities and comparison with amplicon-seq is still rare. In the present study, we developed a mapping-based total RNA-seq analysis to obtain quantitative information on microbial community structure and compared our results with ordinary amplicon-seq methods. We read total RNA sequences from a commercially available mock community (ATCC MSA-2003) and divided reads into small subunit ribosomal RNA (ssrRNA) origin reads and others, such as mRNA origin reads. We then mapped ssrRNA origin reads on annotated assembled contigs and obtained quantitative results under several analysis strategies. Removal of low complexity sequences, sorting ssrRNA with paired-in mode, and performing homology-based taxonomical assignments (BLAST+ or vsearch) showed superior outcomes to other strategies. Results with this approach showed a median relative abundance among ten mock community members of ~10%; ordinary amplicon-seq showed a much lower percentage. Thus, total RNA-seq can be a powerful tool for analyzing microbial community structure and is not limited to analyzing gene expression profiling of microbiomes.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0254556)

10.1371/journal.pone.0254556

---

## Evaluation of DNA extraction protocols from liquid-based cytology specimens for studying cervical microbiota

[Source](https://www.biorxiv.org/content/10.1101/2020.01.27.921619v3.abstract?%3Fcollection=)

---

