## Longitudinal changes in the nasopharyngeal resistome of South African infants using shotgun metagenomic sequencing.
 22.04.2020, _RI Manenzhe, FS Dube, M Wright, K Lennard, HJ Zar, S Mounaud, WC Nierman, MP Nicol, C Moodley_


Nasopharyngeal (NP) colonization with antimicrobial-resistant bacteria is a global public health concern. Antimicrobial-resistance (AMR) genes carried by the resident NP microbiota may serve as a reservoir for transfer of resistance elements to opportunistic pathogens. Little is known about the NP antibiotic resistome. This study longitudinally investigated the composition of the NP antibiotic resistome in Streptococcus-enriched samples in a South African birth cohort.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231887)

10.1371/journal.pone.0231887

---

## Resident microbial communities inhibit growth and antibiotic-resistance evolution of Escherichia coli in human gut microbiome samples.
 20.04.2020, _M Baumgartner, F Bayer, KR Pfrunder-Cardozo, A Buckling, AR Hall_


Countering the rise of antibiotic-resistant pathogens requires improved understanding of how resistance emerges and spreads in individual species, which are often embedded in complex microbial communities such as the human gut microbiome. Interactions with other microorganisms in such communities might suppress growth and resistance evolution of individual species (e.g., via resource competition) but could also potentially accelerate resistance evolution via horizontal transfer of resistance genes. It remains unclear how these different effects balance out, partly because it is difficult to observe them directly. Here, we used a gut microcosm approach to quantify the effect of three human gut microbiome communities on growth and resistance evolution of a focal strain of Escherichia coli. We found the resident microbial communities not only suppressed growth and colonisation by focal E. coli but also prevented it from evolving antibiotic resistance upon exposure to a beta-lactam antibiotic. With samples from all three human donors, our focal E. coli strain only evolved antibiotic resistance in the absence of the resident microbial community, even though we found resistance genes, including a highly effective resistance plasmid, in resident microbial communities. We identified physical constraints on plasmid transfer that can explain why our focal strain failed to acquire some of these beneficial resistance genes, and we found some chromosomal resistance mutations were only beneficial in the absence of the resident microbiota. This suggests, depending on in situ gene transfer dynamics, interactions with resident microbiota can inhibit antibiotic-resistance evolution of individual species.

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000465)

10.1371/journal.pbio.3000465

---

## Comparative analysis of beneficial effects of Vancomycin treatment on Th1- and Th2-biased mice and role of gut microbiota

[Source](https://www.biorxiv.org/content/10.1101/516898v4)

---

## Assessment of dietary supplementation with galactomannan oligosaccharides and phytogenics on gut microbiota of European sea bass (Dicentrarchus Labrax) fed low fishmeal and fish oil based diet.
 22.04.2020, _S Rimoldi, S Torrecillas, D Montero, E Gini, A Makol, V Valdenegro V, M Izquierdo, G Terova_


There is an increasing interest from the aquafeed industry in functional feeds containing selected additives that improve fish growth performance and health status. Functional feed additives include probiotics, prebiotics, organic acids, and phytogenics (substances derived from plants and their extracts). This study evaluated the effects of dietary inclusion of a mucilage extract rich in galactomannan oligosaccharides (GMOS), a mixture of garlic and labiatae-plants oils (PHYTO), and a combination of them (GMOSPHYTO), on gut microbiota composition of European sea bass (Dicentrarchus labrax) fed with a low fishmeal (FM) and fish oil (FO) diet. Three experimental diets and a control diet (plant-based formulation with 10% FM and 6% FO) were tested in a 63-days feeding trial. To analyze the microbiota associated to feeds and the intestinal autochthonous (mucosa-adhered) and allochthonous (transient) microbial communities, the Illumina MiSeq platform for sequencing of 16S rRNA gene and QIIME2 pipeline were used. Metabarcoding analysis of feed-associated bacteria showed that the microbial communities of control (CTRL) feed deeply differed from those of experimental diets. The number of reads was significantly lower in CTRL feed than in other feeds. The OTU (operational taxonomic unit) number was instead similar between the feeds, ranging from 42 to 50 OTUs. The variation of resident gut microbiota induced by diet was lower than the variation of transient intestinal microbiota, because feedstuffs are a major source of allochthonous bacteria, which can temporarily integrate into the gut transient microbiome. However, the composition of transient bacterial communities was not simply a mirror of feed-borne bacteria. Indeed, the microbial profile of feeds was different from both faecal and mucosa profiles. Our findings suggest that the dietary inclusion of GMOS (0.5%) and PHYTO (0.02%) in a low FM and FO diet induces changes in gut microbiota composition of European sea bass. However, if on allochthonous microbiota the combined inclusion of GMOS and PHYTO showed an antagonistic effect on bactericidal activity against Vibrionales, at mucosa level, only GMOSPHYTO diet increased the relative abundance of Bacteroidales, Lactobacillales, and Clostridiales resident bacterial orders. The main beneficial effects of GMOS and PHYTO on gut microbiota are the reduction of coliforms and Vibrionales bacteria, which include several potentially pathogenic species for fish, and the enrichment of gut microbiota composition with butyrate producer taxa. Therefore, these functional ingredients have a great potential to be used as health-promoting agents in the farming of European sea bass and other marine fish.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231494)

10.1371/journal.pone.0231494

---

## Lactobacillus reuteri attenuated allergic inflammation induced by HDM in the mouse and modulated gut microbes.
 21.04.2020, _L Li, Z Fang, X Liu, W Hu, W Lu, YK Lee, J Zhao, H Zhang, W Chen_


Gut microbiome plays an essential role in asthma development, and probiotic-based manipulation of the gut microbiome has been proposed to prevent asthma. Although the preventive effect of Lactobacillus supplementation against allergies has been reported, the precise Lactobacillus species beneficial for effective prevention of asthma remain unidentified and the underlying mechanisms remain unclear. Therefore, we aimed to investigate the efficacy of oral administration of six Lactobacillus species and the mechanism underlying asthma prevention via gut microbiome modulation. We investigated the effects of oral administration of L. rhamnosus, L. fermentum, L. casei, L. gasseri, L. salivarius, and L. reuteri (five strains of each species) on asthma and gut microbiome of house dust mite (HDM)-treated murine models of asthma. Of these, L. reuteri administration was the most effective: it alleviated airway inflammation, decreased total IgE and HDM-IgG1, and reduced Th2-associated pro-inflammatory cytokines. Moreover, modulation of specific microbial genera by L. reuteri was more effective in asthma prevention than the modulation of the overall microbiota composition. Lactobacillus and Enterococcus were enriched after L. reuteri supplementation and were closely associated with total IgE and IL-13 production. Furthermore, L. reuteri specifically altered the gut microbial function toward butyrate generation. Thus, L. reuteri may reduce the risk of asthma development by modulating specific gut microbiota to improve the lung immune environment. Our study suggests a novel option for gut microbiome manipulation via L. reuteri supplementation for suppression of asthma and other allergic diseases.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231865)

10.1371/journal.pone.0231865

---

## Occurrence of Mycoplasma gallisepticum in wild birds: A systematic review and meta-analysis.
 22.04.2020, _A Sawicka, M Durkalec, G Tomczyk, O Kursa_


Mycoplasma gallisepticum is one of the most important poultry pathogens that can also infect wild birds, but knowledge of potential non-poultry hosts that could be reservoirs of M. gallisepticum is limited. For the paper presented here, we screened three databases (PubMed, Scopus, and the Web of Knowledge) to find articles on the occurrence of M. gallisepticum in different wild bird species that were published between 1951 and 2018. Among 314 studies found, we selected and included 50 original articles that met the pre-established criteria. From those publications we extracted the following information: name of the first author, year of publication, year of sample isolation, country, region, number of birds sampled, number of birds tested by each method, number of positive samples, diagnostic criteria, and if birds were wild or captive. Because different detection techniques were used to confirm the presence of M. gallisepticum in one animal, we decided to perform the meta analyses separately for each method. The estimated prevalence of M. gallisepticum in wild birds was different by each method of detection. Our summary revealed that M. gallisepticum was present in 56 species of bird belonging to 11 different orders, of which 21 species were reported suffering both past and current infection. Our work provides information on wild bird species that could be considered potential reservoirs or carriers of M. gallisepticum and could be helpful to set the direction for future research on the spread and phylogeny of M. gallisepticum in different hosts.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231545)

10.1371/journal.pone.0231545

---

## A fungal pathogen induces systemic susceptibility and systemic shifts in wheat metabolome and microbiome composition.
 21.04.2020, _H Seybold, TJ Demetrowitsch, MA Hassani, S Szymczak, E Reim, J Haueisen, L Lübbers, M Rühlemann, A Franke, K Schwarz, EH Stukenbrock_


Yield losses caused by fungal pathogens represent a major threat to global food production. One of the most devastating fungal wheat pathogens is Zymoseptoria tritici. Despite the importance of this fungus, the underlying mechanisms of plant-pathogen interactions are poorly understood. Here we present a conceptual framework based on coinfection assays, comparative metabolomics, and microbiome profiling to study the interaction of Z. tritici in susceptible and resistant wheat. We demonstrate that Z. tritici suppresses the production of immune-related metabolites in a susceptible cultivar. Remarkably, this fungus-induced immune suppression spreads within the leaf and even to other leaves, a phenomenon that we term "systemic induced susceptibility". Using a comparative metabolomics approach, we identify defense-related biosynthetic pathways that are suppressed and induced in susceptible and resistant cultivars, respectively. We show that these fungus-induced changes correlate with changes in the wheat leaf microbiome. Our findings suggest that immune suppression by this hemibiotrophic pathogen impacts specialized plant metabolism, alters its associated microbial communities, and renders wheat vulnerable to further infections.

[Source](https://www.nature.com/articles/s41467-020-15633-x)

10.1038/s41467-020-15633-x

---

## SMOOT libraries and phage-induced directed evolution of Cas9 to engineer reduced off-target activity.
 22.04.2020, _D Cerchione, K Loveluck, EL Tillotson, F Harbinski, J DaSilva, CP Kelley, E Keston-Smith, CA Fernandez, VE Myer, H Jayaram, BE Steinberg_


RNA-guided endonucleases such as Cas9 provide efficient on-target genome editing in cells but may also cleave at off-target loci throughout the genome. Engineered variants of Streptococcus pyogenes Cas9 (SpCas9) have been developed to globally reduce off-target activity, but individual off-targets may remain, or on-target activity may be compromised. In order to evolve against activity at specific off-targets while maintaining strong on-target editing, we developed a novel M13 bacteriophage-mediated selection method. Using this method, sequential rounds of positive and negative selection are used to identify mutations to Cas9 that enhance or diminish editing activity at particular genomic sequences. We also introduce scanning mutagenesis of oligo-directed targets (SMOOT), a comprehensive mutagenesis method to create highly diverse libraries of Cas9 variants that can be challenged with phage-based selection. Our platform identifies novel SpCas9 mutants which mitigate cleavage against off-targets both in biochemical assays and in T-cells while maintaining higher on-target activity than previously described variants. We describe an evolved variant, S. pyogenes Adapted to Reduce Target Ambiguity Cas9 (SpartaCas), composed of the most enriched mutations, each of unknown function. This evolved Cas9 mutant reduces off-target cleavage while preserving efficient editing at multiple therapeutically relevant targets. Directed evolution of Cas9 using our system demonstrates an improved structure-independent methodology to effectively engineer nuclease activity.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231716)

10.1371/journal.pone.0231716

---

## Virome Analysis of Aphid Populations That Infest the Barley Field: The Discovery of Two Novel Groups of Nege/Kita-Like Viruses and Other Novel RNA Viruses.
 22.04.2020, _H Kondo, M Fujita, H Hisano, K Hyodo, IB Andika, N Suzuki_


Aphids (order Hemiptera) are important insect pests of crops and are also vectors of many plant viruses. However, little is known about aphid-infecting viruses, particularly their diversity and relationship to plant viruses. To investigate the aphid viromes, we performed deep sequencing analyses of the aphid transcriptomes from infested barley plants in a field in Japan. We discovered virus-like sequences related to nege/kita-, flavi-, tombus-, phenui-, mononega-, narna-, chryso-, partiti-, and luteoviruses. Using RT-PCR and sequence analyses, we determined almost complete sequences of seven nege/kitavirus-like virus genomes; one of which was a variant of the Wuhan house centipede virus (WHCV-1). The other six seem to belong to four novel viruses distantly related to Wuhan insect virus 9 (WhIV-9) or Hubei nege-like virus 4 (HVLV-4). We designated the four viruses as barley aphid RNA virus 1 to 4 (BARV-1 to -4). Moreover, some nege/kitavirus-like sequences were found by searches on the transcriptome shotgun assembly (TSA) libraries of arthropods and plants. Phylogenetic analyses showed that BARV-1 forms a clade with WHCV-1 and HVLV-4, whereas BARV-2 to -4 clustered with WhIV-9 and an aphid virus, Aphis glycines virus 3. Both virus groups (tentatively designated as Centivirus and Aphiglyvirus, respectively), together with arthropod virus-like TSAs, fill the phylogenetic gaps between the negeviruses and kitaviruses lineages. We also characterized the flavi/jingmen-like and tombus-like virus sequences as well as other RNA viruses, including six putative novel viruses, designated as barley aphid RNA viruses 5 to 10. Interestingly, we also discovered that some aphid-associated viruses, including nege/kita-like viruses, were present in different aphid species, raising a speculation that these viruses might be distributed across different aphid species with plants being the reservoirs. This study provides novel information on the diversity and spread of nege/kitavirus-related viruses and other RNA viruses that are associated with aphids.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231814)

10.3389/fmicb.2020.00509

---

## BOLD and GenBank revisited – Do identification errors arise in the lab or in the sequence libraries?

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0231814)

---

## Collaborative Learning Tools To Foster Inclusive Participation and Sense of Belonging in a Microbiology Outreach Partnership.
 21.04.2020, _ED Sprowls_


For secondary outreach programs to meet the goals of enhancing science education and attracting future scientists from underrepresented populations, we need an inclusive approach that integrates students' knowledge and experiences in the process of doing science. I present three pedagogical tools designed by developing equitable, inclusive collaboration among microbiologist outreach mentors and high school biology students. These activities aim to foster a sense of belonging in a scientific community and serve as an entry point to the practice of inclusion. Over a one-semester course at an alternative high school, ten secondary students and their scientist mentors met weekly to design and conduct microbiology experiments together. This group of students and scientists participated in structured collaborative learning activities to: i) understand each other's ideas about science; ii) collectively analyze their research findings; and iii) offer peer feedback. I modified the following three learning tools for use in my secondary science classroom from protocols of the National School Reform Faculty: 1) the Quotes Introduction Activity set the stage for equitable discourse between high school students and scientist mentors, while initiating important conversations about the process of biological research; 2) the Data Analysis Protocol allowed both students and mentors to contribute to the scientific process; and 3) the Feedback Carousel Activity engaged students and scientists alike in reviewing and refining poster presentations. This inclusive engagement in the social aspects of learning science can help students feel a sense of belonging and imagine their futures in the scientific community, key steps towards inclusion. The supportive system of structured feedback in these collaborative learning activities created a safe, inclusive space for secondary students to try on the role of microbiology expert, and for scientist volunteers to practice inclusive mentorship. Drawing from inclusive pedagogical tools in secondary education will help expand our capacity for inclusive science outreach and bring us closer to the goals of improving biology education and attracting future biologists at the university level.

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000650)

10.1128/jmbe.v21i1.2070

---

## A pedagogical approach to science outreach

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000650)

---

## Trypanosoma brucei colonizes the tsetse gut via an immature peritrophic matrix in the proventriculus.
 21.04.2020, _C Rose, A Casas-Sánchez, NA Dyer, C Solórzano, AJ Beckett, B Middlehurst, M Marcello, LR Haines, J Lisack, M Engstler, MJ Lehane, IA Prior, Á Acosta-Serrano_


The peritrophic matrix of blood-feeding insects is a chitinous structure that forms a protective barrier against oral pathogens and abrasive particles1. Tsetse flies transmit Trypanosoma brucei, which is the parasite that causes human sleeping sickness and is also partially responsible for animal trypanosomiasis in Sub-Saharan Africa. For this parasite to establish an infection in flies, it must first colonize the area between the peritrophic matrix and gut epithelium called the ectoperitrophic space. Although unproven, it is generally accepted that trypanosomes reach the ectoperitrophic space by penetrating the peritrophic matrix in the anterior midgut2-4. Here, we revisited this event using fluorescence- and electron-microscopy methodologies. We show that trypanosomes penetrate the ectoperitrophic space in which the newly made peritrophic matrix is synthesized by the proventriculus. Our model describes how these proventriculus-colonizing parasites can either migrate to the ectoperitrophic space or become trapped within peritrophic matrix layers to form cyst-like bodies that are passively pushed along the gut as the matrix gets remodelled. Furthermore, early proventricular colonization seems to be promoted by factors in trypanosome-infected blood that cause higher salivary gland infections and potentially increase parasite transmission.

[Source](https://www.nature.com/articles/s41564-020-0707-z)

10.1038/s41564-020-0707-z

---

