## Cultivation-independent and cultivation-dependent metagenomes reveal genetic and enzymatic potential of microbial community involved in the degradation of a complex microbial polymer.
 08.06.2020, _OYA Costa, M de Hollander, A Pijl, B Liu, EE Kuramae_


Cultivation-independent methods, including metagenomics, are tools for the exploration and discovery of biotechnological compounds produced by microbes in natural environments. Glycoside hydrolases (GHs) enzymes are extremely desired and important in the industry of production for goods and biofuel and removal of problematic biofilms and exopolysaccharide (EPS). Biofilms and EPS are complex, requiring a wide range of enzymes for a complete degradation. The aim of this study was to identify potential GH microbial producers and GH genes with biotechnological potential, using EPS-complex structure (WH15EPS) of Acidobacteria Granulicella sp. strain WH15 as an enrichment factor, in cultivation-independent and cultivation-dependent methods. We performed stable isotope probing (SIP) combined with metagenomics on topsoil litter amended with WH15EPS and coupled solid culture-EPS amended medium with metagenomics.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00836-7)

10.1186/s40168-020-00836-7

---

## Conceptualizing the vertebrate sterolbiome.
 06.06.2020, _JM Ridlon_


Vertebrates synthesize a diverse set of steroids and bile acids that undergo bacterial biotransformations. The endocrine literature has principally focused on the biochemistry and molecular biology of host synthesis and tissue-specific metabolism of steroids. Host-associated microbiota possess a co-evolved set of steroid and bile acid modifying enzymes that match the majority of host peripheral biotransformations in addition to unique capabilities. The set of host-associated microbial genes encoding enzymes involved in steroid transformations is known as the sterolbiome. This review focuses on the current knowledge of the sterolbiome as well as its importance in medicine and agriculture.

[Source](https://aem.asm.org/content/early/2020/06/01/AEM.00641-20.long)

10.1128/AEM.00641-20

---

## Human milk fungi: environmental determinants and inter-kingdom associations with milk bacteria in the CHILD Cohort Study.
 08.06.2020, _S Moossavi, K Fehr, H Derakhshani, H Sbihi, B Robertson, L Bode, J Brook, SE Turvey, TJ Moraes, AB Becker, PJ Mandhane, MR Sears, E Khafipour, P Subbarao, MB Azad_


Fungi constitute an important yet frequently neglected component of the human microbiota with a possible role in health and disease. Fungi and bacteria colonise the infant gastrointestinal tract in parallel, yet most infant microbiome studies have ignored fungi. Milk is a source of diverse and viable bacteria, but few studies have assessed the diversity of fungi in human milk.

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-020-01829-0)

10.1186/s12866-020-01829-0

---

## Skin microbiota analysis-inspired development of novel anti-infectives.
 08.06.2020, _Y Liu, Y Liu, Z Du, L Zhang, J Chen, Z Shen, Q Liu, J Qin, H Lv, H Wang, L He, J Liu, Q Huang, Y Sun, M Otto, M Li_


The alarming spread of antimicrobial resistance requires the development of novel anti-infective drugs. Despite the recent research focus on the human microbiome and its likely value to understand and exploit inter-bacterial inhibitory phenomena as a source for antimicrobial strategies, the human microbiota has barely been investigated for the purpose of drug development.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00866-1)

10.1186/s40168-020-00866-1

---

## Vaginal microbiota of adolescents and their mothers: A preliminary study of vertical transmission and persistence

[Source](https://www.biorxiv.org/content/10.1101/768598v2)

---

## Personalized whole-body models integrate metabolism, physiology, and the gut microbiome.
 29.05.2020, _I Thiele, S Sahoo, A Heinken, J Hertel, L Heirendt, MK Aurich, RM Fleming_


Comprehensive molecular-level models of human metabolism have been generated on a cellular level. However, models of whole-body metabolism have not been established as they require new methodological approaches to integrate molecular and physiological data. We developed a new metabolic network reconstruction approach that used organ-specific information from literature and omics data to generate two sex-specific whole-body metabolic (WBM) reconstructions. These reconstructions capture the metabolism of 26 organs and six blood cell types. Each WBM reconstruction represents whole-body organ-resolved metabolism with over 80,000 biochemical reactions in an anatomically and physiologically consistent manner. We parameterized the WBM reconstructions with physiological, dietary, and metabolomic data. The resulting WBM models could recapitulate known inter-organ metabolic cycles and energy use. We also illustrate that the WBM models can predict known biomarkers of inherited metabolic diseases in different biofluids. Predictions of basal metabolic rates, by WBM models personalized with physiological data, outperformed current phenomenological models. Finally, integrating microbiome data allowed the exploration of host-microbiome co-metabolism. Overall, the WBM reconstructions, and their derived computational models, represent an important step toward virtual physiological humans.

[Source](https://www.embopress.org/doi/full/10.15252/msb.20198982)

10.15252/msb.20198982

---

## The gut microbiota profile of adults with kidney disease and kidney stones: a systematic review of the literature.
 08.06.2020, _J Stanford, K Charlton, A Stefoska-Needham, R Ibrahim, K Lambert_


There is mounting evidence that individuals with kidney disease and kidney stones have an abnormal gut microbiota composition. No studies to date have summarised the evidence to categorise how the gut microbiota profile of these individuals may differ from controls. Synthesis of this evidence is essential to inform future clinical trials. This systematic review aims to characterise differences of the gut microbial community in adults with kidney disease and kidney stones, as well as to describe the functional capacity of the gut microbiota and reporting of diet as a confounder in these studies.

[Source](https://bmcnephrol.biomedcentral.com/articles/10.1186/s12882-020-01805-w)

10.1186/s12882-020-01805-w

---

## Gut microbes and their genes are associated with brain development and cognitive function in healthy children

[Source](https://www.biorxiv.org/content/10.1101/2020.02.13.944181v3)

---

## Microbial environment shapes immune function and cloacal microbiota dynamics in zebra finches Taeniopygia guttata

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-020-00039-3)

---

## Large scale metagenome assembly reveals novel animal-associated microbial genomes, biosynthetic gene clusters, and other genetic diversity 

[Source](https://www.biorxiv.org/content/10.1101/2020.06.05.135962v1)

---

## Novel insights into the Thaumarchaeota in the deepest oceans: their metabolism and potential adaptation mechanisms

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00849-2)

---

## The role of metal contamination in shaping microbial communities in heavily polluted marine sediments

[Source](https://www.sciencedirect.com/science/article/pii/S0269749120317590)

---

## Engineering CRISPR/Cas9 to mitigate abundant host contamination for 16S rRNA gene-based amplicon sequencing

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00859-0)

---

## In silico design and characterization of multiepitopes vaccine for SARS-CoV from its Spike proteins

[Source](https://www.biorxiv.org/content/10.1101/2020.06.03.131755v2)

---

