## Toward an integrative framework for microbial community coalescence.
 01.10.2023, _GF Custer, L Bresciani, F Dini-Andreote_


Community coalescence is defined as the mixing of intact ecological communities. From river confluences to fecal microbiota transplantation, community coalescence constitutes a common ecological occurrence affecting natural and engineered microbial systems. In this opinion article, we propose an integrative framework for microbial community coalescence to guide advances in our understanding of this important - yet underexplored - ecological phenomenon. We start by aligning community coalescence with the unified framework of biological invasion and enumerate commonalities and idiosyncrasies between these two analogous processes. Then, we discuss how organismal interactions and cohesive establishment affect coalescence outcomes with direct implications for community functioning. Last, we propose the use of ecological null modeling to study the interplay of ecological processes structuring community reassembly following coalescence.

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(23)00262-7)

10.1016/j.tim.2023.09.001

---

## Statistically learning the functional landscape of microbial communities.
 04.10.2023, _A Skwara, K Gowda, M Yousef, J Diaz-Colunga, AS Raman, A Sanchez, M Tikhonov, S Kuehn_


Microbial consortia exhibit complex functional properties in contexts ranging from soils to bioreactors to human hosts. Understanding how community composition determines function is a major goal of microbial ecology. Here we address this challenge using the concept of community-function landscapes-analogues to fitness landscapes-that capture how changes in community composition alter collective function. Using datasets that represent a broad set of community functions, from production/degradation of specific compounds to biomass generation, we show that statistically inferred landscapes quantitatively predict community functions from knowledge of species presence or absence. Crucially, community-function landscapes allow prediction without explicit knowledge of abundance dynamics or interactions between species and can be accurately trained using measurements from a small subset of all possible community compositions. The success of our approach arises from the fact that empirical community-function landscapes appear to be not rugged, meaning that they largely lack high-order epistatic contributions that would be difficult to fit with limited data. Finally, we show that this observation holds across a wide class of ecological models, suggesting community-function landscapes can be efficiently inferred across a broad range of ecological regimes. Our results open the door to the rational design of consortia without detailed knowledge of abundance dynamics or interactions.

[Source](https://www.nature.com/articles/s41559-023-02197-4)

10.1038/s41559-023-02197-4

---

## Skin microbiota secretomes modulate cutaneous innate immunity against Borrelia burgdorferi s.s.
 05.10.2023, _F Baquer, B Jaulhac, C Barthel, M Paz, J Wolfgramm, A Müller, N Boulanger, A Grillon_


_Animals, Humans, Borrelia burgdorferi, Interleukin-8, Secretome, Lyme Disease, Inflammation, Immunity, Innate, Ixodes_
In Lyme borreliosis, the skin constitutes a major interface for the host, the bacteria and the tick. Skin immunity is provided by specialized immune cells but also by the resident cells: the keratinocytes and the fibroblasts. Discoveries on the role of the microbiome in the modulation of skin inflammation and immunity have reinforced the potential importance of the skin in vector-borne diseases. In this study, we analyzed in vitro the interaction of human primary keratinocytes and fibroblasts with Borrelia burgdorferi sensu stricto N40 in presence or absence of bacterial commensal supernatants. We aimed to highlight the role of resident skin cells and skin microbiome on the inflammation induced by B. burgdorferi s.s.. The secretomes of Staphylococcus epidermidis, Corynebacterium striatum and Cutibacterium acnes showed an overall increase in the expression of IL-8, CXCL1, MCP-1 and SOD-2 by fibroblasts, and of IL-8, CXCL1, MCP-1 and hBD-2 in the undifferentiated keratinocytes. Commensal bacteria showed a repressive effect on the expression of IL-8, CXCL1 and MCP-1 by differentiated keratinocytes. Besides the inflammatory effect observed in the presence of Borrelia on all cell types, the cutaneous microbiome appears to promote a rapid innate response of resident skin cells during the onset of Borrelia infection.

[Source](https://www.nature.com/articles/s41598-023-43566-0)

10.1038/s41598-023-43566-0

---

## Temporal changes in fecal microbiota of patients infected with COVID-19: a longitudinal cohort

[Source](https://bmcinfectdis.biomedcentral.com/articles/10.1186/s12879-023-08511-6)

---

## Association Between Bowel-Movement Pattern and Cognitive Function: Prospective Cohort Study and a Metagenomic Analysis of the Gut Microbiome

[Source](https://n.neurology.org/content/early/2023/09/29/WNL.0000000000207849.abstract)

---

## The gut microbiome tango in the progression of chronic kidney disease and potential therapeutic strategies

[Source](https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-023-04455-2)

---

## Microbiota-gut-brain axis drives overeating disorder

[Source](https://www.cell.com/cell-metabolism/fulltext/S1550-4131(23)00335-2)

---

## Exploring gut microbiota in adult Atlantic salmon (Salmo salar L.): Associations with gut health and dietary prebiotics

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-023-00269-1)

---

## Using wasps as a tool to restore a functioning vine grape mycobiota and preserve the mycobial “terroir”

[Source](https://www.nature.com/articles/s41598-023-43541-9)

---

## Soil microbes mediate the effects of resource variability on plant invasion

[Source](https://esajournals.onlinelibrary.wiley.com/doi/10.1002/ecy.4154)

---

## Deepurify: a multi-modal deep language model to remove contamination from metagenome-assembled genomes

[Source](https://www.biorxiv.org/content/10.1101/2023.09.27.559668v1)

---

## BATTER: Accurate Prediction of Rho-dependent and Rho-independent Transcription Terminators in Metagenomes

[Source](https://www.biorxiv.org/content/10.1101/2023.10.02.560326v1)

---

## Gut microbial determinants of clinically important improvement in patients with rheumatoid arthritis.
 21.02.2022, _VK Gupta, KY Cunningham, B Hur, U Bakshi, H Huang, KJ Warrington, V Taneja, E Myasoedova, JM Davis, J Sung_


_Aged, Aged, 80 and over, Arthritis, Rheumatoid, Clostridiales, Cohort Studies, Dysbiosis, Female, Gastrointestinal Microbiome, Humans, Male, Metagenome, Metagenomics, Middle Aged, RNA, Ribosomal, 16S, Retrospective Studies, Severity of Illness Index_
Rapid advances in the past decade have shown that dysbiosis of the gut microbiome is a key hallmark of rheumatoid arthritis (RA). Yet, the relationship between the gut microbiome and clinical improvement in RA disease activity remains unclear. In this study, we explored the gut microbiome of patients with RA to identify features that are associated with, as well as predictive of, minimum clinically important improvement (MCII) in disease activity.

[Source](https://www.biorxiv.org/content/10.1101/2023.09.30.560294v1)

10.1186/s13073-021-00957-0

---

## Gut Microbiome Wellness Index 2 for Enhanced Health Status Prediction from Gut Microbiome Taxonomic Profiles

[Source](https://www.biorxiv.org/content/10.1101/2023.09.30.560294v1)

---

## INVADEseq to identify cell-adherent or invasive bacteria and the associated host transcriptome at single-cell-level resolution.
 03.10.2023, _JL Galeano Niño, H Wu, KD LaCourse, H Srinivasan, M Fitzgibbon, SS Minot, C Sather, CD Johnston, S Bullman_


Single-cell RNA sequencing (scRNAseq) technologies have been beneficial in revealing and describing cellular heterogeneity within mammalian tissues, including solid tumors. However, many of these techniques apply poly(A) selection of RNA, and thus have primarily focused on determining the gene signatures of eukaryotic cellular components of the tumor microenvironment. Microbiome analysis has revealed the presence of microbial ecosystems, including bacteria and fungi, within human tumor tissues from major cancer types. Imaging data have revealed that intratumoral bacteria may be located within epithelial and immune cell types. However, as bacterial RNA typically lacks a poly(A) tail, standard scRNAseq approaches have limited ability to capture this microbial component of the tumor microenvironment. To overcome this, we describe the invasion-adhesion-directed expression sequencing (INVADEseq) approach, whereby we adapt 10x Genomics 5' scRNAseq protocol by introducing a primer that targets a conserved region of the bacterial 16S ribosomal RNA gene in addition to the standard primer for eukaryotic poly(A) RNA selection. This 'add-on' approach enables the generation of eukaryotic and bacterial DNA libraries at eukaryotic single-cell level resolution, utilizing the 10x barcode to identify single cells with intracellular bacteria. The INVADEseq method takes 30 h to complete, including tissue processing, sequencing and computational analysis. As an output, INVADEseq has shown to be a reliable tool in human cancer cell lines and patient tumor specimens by detecting the proportion of human cells that harbor bacteria and the identities of human cells and intracellular bacteria, along with identifying host transcriptional programs that are modulated on the basis of associated bacteria.

[Source](https://www.nature.com/articles/s41596-023-00888-7)

10.1038/s41596-023-00888-7

---

