## Particular genomic and virulence traits associated with preterm infant-derived toxigenic <em>Clostridium perfringens</em> strains

[Source](https://doi.org/10.1038/s41564-023-01385-z)

---

## Detailed mapping of <em>Bifidobacterium</em> strain transmission from mother to infant via a dual culture-based and metagenomic approach

[Source](https://doi.org/10.1038/s41467-023-38694-0)

---

## Expanding the repertoire of counterselection markers for markerless gene deletion in the human gut bacterium <em>Phocaeicola vulgatus</em>

[Source](https://doi.org/10.1016/j.anaerobe.2023.102742)

---

## Gut microbiota, circulating inflammatory markers and metabolites, and carotid artery atherosclerosis in HIV infection

[Source](https://doi.org/10.1186/s40168-023-01566-2)

---

## Symbiotic microbiome <em>Staphylococcus epidermidis</em> restricts IL-33 production in allergic nasal epithelium via limiting the cellular necroptosis

[Source](https://doi.org/10.1186/s12866-023-02898-7)

---

## Gut microbiome associated with low anterior resection syndrome after rectal cancer surgery.
 29.05.2023, _MJ Kim, S Park, JW Park, J Choi, HJ Kim, HK Lim, SB Ryoo, KJ Park, Y Ji, SY Jeong_


_Humans, Rectal Neoplasms, Low Anterior Resection Syndrome, Postoperative Complications, Gastrointestinal Microbiome, Rectum, Quality of Life_
This study aimed to assess the likely association of gut microbiome with low anterior resection syndrome (LARS) symptoms. Postoperative stool samples from patients with minor or major LARS after sphincter-preserving surgery (SPS) for rectal cancer were collected and analyzed using 16S ribosomal RNA sequencing method. The symptom patterns of LARS were classified into two groups (PC1LARS, PC2LARS) using principal component analysis. The dichotomized sum of questionnaire items (sub1LARS, sub2LARS) was used to group patients according to the main symptoms. According to microbial diversity, enterotype, and taxa, PC1LARS and sub1LARS were associated with frequency-dominant LARS symptoms and patients, while PC2LARS and sub2LARS were grouped as incontinence-dominant LARS symptoms and patients. Butyricicoccus levels decreased while overall LARS scores increased. The α-diversity richness index Chao1 showed a significantly negative correlation in sub1LARS and a positive correlation in sub2LARS. In sub1LARS, the severe group showed a lower Prevotellaceae enterotype and higher Bacteroidaceae enterotype than the mild group. Subdoligranulum and Flavonifractor showed a negative and a positive correlation with PC1LARS, respectively, while showing a negative relationship with PC2LARS. Lactobacillus and Bifidobacterium were negatively correlated to PC1LARS. Frequency-dominant LARS had decreased diversity of gut microbiome and showed lower levels of lactic acid-producing bacteria.

[Source](https://doi.org/10.1038/s41598-023-34557-2)

10.1038/s41598-023-34557-2

---

## Metabolomics Reveals Gut Microbiota Contribute to PPARα Deficiency-Induced Alcoholic Liver Injury.
 26.05.2023, _T Zhang, L Bao, Q Zhao, ZE Wu, M Dai, Q Rao, F Li_


Incidence and mortality rates of alcoholic liver disease (ALD) is one of the highest in the world. In the present study, we found that the genetic knockout nuclear receptor the peroxisome proliferator-activated receptor α (PPARα) exacerbated ALD. Lipidomics of the liver revealed changed levels of lipid species encompassing phospholipids, ceramides (CM), and long-chain fatty acids in Ppara-null mice induced by ethanol. Moreover, 4-hydroxyphenylacetic acid (4-HPA) was changed as induced by ethanol in the metabolome of urine. Moreover, the phylum level analysis showed a decrease in the level of Bacteroidetes and an increase in the level of Firmicutes after alcohol feeding in Ppara-null mice, while there was no change in wild-type mice. In Ppara-null mice, the level of Clostridium_sensu_stricto_1 and Romboutsia were upregulated after alcohol feeding. These data revealed that PPARα deficiency potentiated alcohol-induced liver injury through promotion of lipid accumulation, changing the metabolome of urine, and increasing the level of Clostridium_sensu_stricto_1 and Romboutsia. 4-HPA could improve ALD in mice by regulating inflammation and lipid metabolism. Therefore, our findings suggest a novel approach to the treatment of ALD: focusing on the gut microbiota and its metabolites. Data are available via ProteomeXchange (PXD 041465).

[Source](https://doi.org/10.1021/acs.jproteome.3c00093)

10.1021/acs.jproteome.3c00093

---

## Following your gut: the emerging role of the gut microbiota in predicting and treating immune-related adverse events.
 24.05.2023, _A Desilets, A Elkrief_


Although immune checkpoint inhibition has reshaped the therapeutic landscape leading to improved outcomes across an array of both solid and hematologic malignancies, a significant source of morbidity is caused by immune-related adverse events (irAEs) caused by these agents.

[Source](https://doi.org/10.1097/CCO.0000000000000957)

10.1097/CCO.0000000000000957

---

## Respiratory eukaryotic virome expansion and bacteriophage deficiency characterize childhood asthma.
 02.06.2023, _S Megremis, B Constantinides, P Xepapadaki, CF Yap, AG Sotiropoulos, C Bachert, S Finotto, T Jartti, A Tapinos, T Vuorinen, E Andreakos, DL Robertson, NG Papadopoulos_


_Child, Humans, Child, Preschool, Eukaryota, Bacteriophages, Virome, Eukaryotic Cells, Asthma, Anelloviridae, Asymptomatic Diseases_
Asthma development and exacerbation is linked to respiratory virus infections. There is limited information regarding the presence of viruses during non-exacerbation/infection periods. We investigated the nasopharyngeal/nasal virome during a period of asymptomatic state, in a subset of 21 healthy and 35 asthmatic preschool children from the Predicta cohort. Using metagenomics, we described the virome ecology and the cross-species interactions within the microbiome. The virome was dominated by eukaryotic viruses, while prokaryotic viruses (bacteriophages) were independently observed with low abundance. Rhinovirus B species consistently dominated the virome in asthma. Anelloviridae were the most abundant and rich family in both health and asthma. However, their richness and alpha diversity were increased in asthma, along with the co-occurrence of different Anellovirus genera. Bacteriophages were richer and more diverse in healthy individuals. Unsupervised clustering identified three virome profiles that were correlated to asthma severity and control and were independent of treatment, suggesting a link between the respiratory virome and asthma. Finally, we observed different cross-species ecological associations in the healthy versus the asthmatic virus-bacterial interactome, and an expanded interactome of eukaryotic viruses in asthma. Upper respiratory virome "dysbiosis" appears to be a novel feature of pre-school asthma during asymptomatic/non-infectious states and merits further investigation.

[Source](https://doi.org/10.1038/s41598-023-34730-7)

10.1038/s41598-023-34730-7

---

## The causal links between gut microbiota and COVID-19: A Mendelian randomization study.
 24.05.2023, _J Song, Y Wu, X Yin, H Ma, J Zhang_


_Humans, Gastrointestinal Microbiome, Mendelian Randomization Analysis, COVID-19, Microbiota, Dietary Supplements, Genome-Wide Association Study, Polymorphism, Single Nucleotide_
Several studies have shown a possible correlation between gut microbiota and COVID-19. However, the cause-and-effect relationship between the two has not been investigated. We conducted a two-sample Mendelian randomization study (MR) study using publicly available GWAS data. Inverse variance weighted (IVW) analysis was the main MR analysis technique and was supplemented with other sensitivity analyses. Forty-two bacterial genera were associated with COVID-19 susceptibility, hospitalization, and severity in the IVW method. Among these gut microbiota, five gut microbiota (genus unknowngenus [id.1000005472], family unknownfamily [id.1000005471], genus Tyzzerella3, order MollicutesRF9.id.11579, and phylum Actinobacteria) were significantly associated with COVID-19 hospitalization and severity. Three gut microbiota (class Negativicutes, order Selenomonadales, and class Actinobacteria) were significantly associated with COVID-19 hospitalization and susceptibility, while two microbiota (class Negativicutes and order Selenomonadales) were significantly associated with COVID-19 hospitalization and severity, and susceptibility. Sensitivity analysis did not detect any heterogeneity and horizontal pleiotropy. Our findings demonstrated that several microorganisms were causally linked to COVID-19, and improved our understanding of the relationship between gut microbiota and COVID-19 pathology.

[Source](https://doi.org/10.1002/jmv.28784)

10.1002/jmv.28784

---

## The Sexome&nbsp;– A proof of concept study into microbial transfer between heterosexual couples after sexual intercourse

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0379073823001615?via%3Dihub)

---

## A systematic review of the protein composition of whole saliva in subjects with healthy periodontium compared with chronic periodontitis.
 28.05.2023, _AG Sánchez-Medrano, RE Martinez-Martinez, R Soria-Guerra, D Portales-Perez, H Bach, F Martinez-Gutierrez_


_Humans, Chronic Periodontitis, Proteomics, Saliva, Periodontium, Periodontal Ligament, Calgranulin A, Calgranulin B_
Periodontitis is a chronic multifactorial inflammatory disease linked to oral microbiota dysbiosis. This disease progresses to infection that stimulates a host immune/inflammatory response, with progressive destruction of the tooth-supporting structures.

[Source](https://doi.org/10.1371/journal.pone.0286079)

10.1371/journal.pone.0286079

---

## Metagenomic analysis and antibacterial activity of kefir microorganisms.
 24.05.2023, _BD González-Orozco, I García-Cano, A Escobar-Zepeda, R Jiménez-Flores, VB Álvarez_


The microbiota composition of kefir grain and milk kefir was assessed via a metagenomic approach. Significant microorganisms were isolated and identified using molecular methods. A safety assessment was conducted based on antibiotic susceptibility and blood hemolysis. Probiotic traits such as resistance to gastric tract conditions, surface characteristics, adhesion to intestinal cells, and antibacterial activity were also assessed. Metagenomic analysis revealed that kefir grains are a more stable community with clear dominant species as compared to milk kefir. Lactobacillus kefiranofaciens BDGO-A1, Lactobacillus helveticus BDGO-AK2, and Lactobacillu kefiri strains showed tolerance to acidic pH and the presence of bile salts, adhesion capability to Caco-2 cells, in vitro antibacterial activity, and the production of antibacterial proteins. In the metagenomic analysis, contigs associated with these species showed the presence of genes involved in exporting polyketide antibiotics and bacteriocin production. To fully exploit the potential probiotic properties of these microorganisms to help human health, further investigation is necessary to elucidate the mechanisms behind the biological activity and the genotypic characteristics of the isolated strains.

[Source](https://doi.org/10.1111/1750-3841.16614)

10.1111/1750-3841.16614

---

## Fecal Metabolites as Biomarkers for Predicting Food Intake by Healthy Adults.
 29.05.2023, _LM Shinn, A Mansharamani, DJ Baer, JA Novotny, CS Charron, NA Khan, R Zhu, HD Holscher_


_Humans, Adult, Diet, Metabolomics, Metabolome, Juglans, Edible Grain, Biomarkers, Eating_
The fecal metabolome is affected by diet and includes metabolites generated by human and microbial metabolism. Advances in -omics technologies and analytic approaches have allowed researchers to identify metabolites and better utilize large data sets to generate usable information. One promising aspect of these advancements is the ability to determine objective biomarkers of food intake.

[Source](https://doi.org/10.1093/jn/nxac195)

10.1093/jn/nxac195

---

## Taxonogenomics of <em>Culturomica massiliensis</em> gen. nov., sp. nov., and <em>Emergencia timonensis</em> gen. nov., sp. nov. new bacteria isolated from human stool microbiota

[Source](https://doi.org/10.1038/s41598-023-35443-7)

---

## Gut microbiome signatures of extreme environment adaption in Tibetan pig.
 28.05.2023, _F Zhao, L Yang, T Zhang, D Zhuang, Q Wu, J Yu, C Tian, Z Zhang_


_Swine, Animals, Gastrointestinal Microbiome, Tibet, Ultraviolet Rays, Acclimatization, Acetic Acid, Extreme Environments, Mammals_
Tibetan pigs (TPs) can adapt to the extreme environments in the Tibetan plateau implicated by their self-genome signals, but little is known about roles of the gut microbiota in the host adaption. Here, we reconstructed 8210 metagenome-assembled genomes from TPs (n = 65) living in high-altitude and low-altitude captive pigs (87 from China-CPs and 200 from Europe-EPs) that were clustered into 1050 species-level genome bins (SGBs) at the threshold of 95% average nucleotide identity. 73.47% of SGBs represented new species. The gut microbial community structure analysis based on 1,048 SGBs showed that TPs was significantly different from low-altitude captive pigs. TP-associated SGBs enabled to digest multiple complex polysaccharides, including cellulose, hemicellulose, chitin and pectin. Especially, we found TPs showed the most common enrichment of phyla Fibrobacterota and Elusimicrobia, which were involved in the productions of short- and medium-chain fatty acids (acetic acid, butanoate and propanoate; octanomic, decanoic and dodecanoic acids), as well as in the biosynthesis of lactate, 20 essential amino acids, multiple B vitamins (B1, B2, B3, B5, B7 and B9) and cofactors. Unexpectedly, Fibrobacterota solely showed powerful metabolic capacity, including the synthesis of acetic acid, alanine, histidine, arginine, tryptophan, serine, threonine, valine, B2, B5, B9, heme and tetrahydrofolate. These metabolites might contribute to host adaptation to high-altitude, such as energy harvesting and resistance against hypoxia and ultraviolet radiation. This study provides insights into understanding the role of gut microbiome played in mammalian high-altitude adaptation and discovers some potential microbes as probiotics for improving animal health.

[Source](https://doi.org/10.1038/s41522-023-00395-3)

10.1038/s41522-023-00395-3

---

## Holobiont Evolution: Population Theory for the Hologenome.
 29.05.2023, _J Roughgarden_


_Biological Evolution, Symbiosis, Microbiota, Population Dynamics, Cell Nucleus_
AbstractThis article develops mathematical theory for the population dynamics of microbiomes with their hosts and for holobiont evolution caused by holobiont selection. The objective is to account for the formation of microbiome-host integration. Microbial population dynamic parameters must mesh with the host's for coexistence. A horizontally transmitted microbiome is a genetic system with "collective inheritance." The microbial source pool in the environment corresponds to the gamete pool for nuclear genes. Poisson sampling of the microbial source pool corresponds to binomial sampling of the gamete pool. However, holobiont selection on the microbiome does not lead to a counterpart of the Hardy-Weinberg law or to directional selection that always fixes microbial genes conferring the highest holobiont fitness. A microbe might strike an optimal fitness balance between lowering its within-host fitness while increasing holobiont fitness. Such microbes are replaced by otherwise identical microbes that contribute nothing to holobiont fitness. This replacement can be reversed by hosts that initiate immune responses to nonhelpful microbes. This discrimination leads to microbial species sorting. Host-orchestrated species sorting followed by microbial competition, rather than coevolution or multilevel selection, is predicted to be the cause of microbiome-host integration.

[Source](https://doi.org/10.1086/723782)

10.1086/723782

---

## Integration of Phenotypes in Microbiome Networks for Designing Synthetic Communities: a Study of Mycobiomes in the Grafted Tomato System.
 24.05.2023, _R Poudel, A Jumpponen, MM Kennelly, C Rivard, L Gomez-Montano, KA Garrett_


Understanding factors influencing microbial interactions, and designing methods to identify key taxa that are candidates for synthetic communities, or SynComs, are complex challenges for achieving microbiome-based agriculture. Here, we study how grafting and the choice of rootstock influences root-associated fungal communities in a grafted tomato system. We studied three tomato rootstocks (BHN589, RST-04-106, and Maxifort) grafted to a BHN589 scion and profiled the fungal communities in the endosphere and rhizosphere by sequencing the internal transcribed spacer (ITS2). The data provided evidence for a rootstock effect (explaining ~2% of the total captured variation, P &lt; 0.01) on the fungal community. Moreover, the most productive rootstock, Maxifort, supported greater fungal species richness than the other rootstocks or controls. We then constructed a phenotype-operational taxonomic unit (OTU) network analysis (PhONA) using an integrated machine learning and network analysis approach based on fungal OTUs and associated tomato yield as the phenotype. PhONA provides a graphical framework to select a testable and manageable number of OTUs to support microbiome-enhanced agriculture. We identified differentially abundant OTUs specific to each rootstock in both endosphere and rhizosphere compartments. Subsequent analyses using PhONA identified OTUs that were directly associated with tomato fruit yield and others that were indirectly linked to yield through their links to these OTUs. Fungal OTUs that are directly or indirectly linked with tomato yield may represent candidates for synthetic communities to be explored in agricultural systems. IMPORTANCE The realized benefits of microbiome analyses for plant health and disease management are often limited by the lack of methods to select manageable and testable synthetic microbiomes. We evaluated the composition and diversity of root-associated fungal communities from grafted tomatoes. We then constructed a phenotype-OTU network analysis (PhONA) using these linear and network models. By incorporating yield data in the network, PhONA identified OTUs that were directly predictive of tomato yield and others that were indirectly linked to yield through their links to these OTUs. Follow-up functional studies of taxa associated with effective rootstocks, identified using approaches such as PhONA, could support the design of synthetic fungal communities for microbiome-based crop production and disease management. The PhONA framework is flexible for incorporation of other phenotypic data, and the underlying models can readily be generalized to accommodate other microbiome or 'omics data.

[Source](https://doi.org/10.1128/aem.01843-22)

10.1128/aem.01843-22

---

## Root Niches of Blueberry Imprint Increasing Bacterial-Fungal Interkingdom Interactions along the Soil-Rhizosphere-Root Continuum.
 24.05.2023, _J Che, Y Wu, H Yang, S Wang, W Wu, L Lyu, X Wang, W Li_


Plant root-associated microbiomes play critical roles in promoting plant health, productivity, and tolerance to biotic/abiotic stresses. Blueberry (Vaccinium spp.) is adapted to acidic soils, while the interactions of the root-associated microbiomes in this specific habitat under various root microenvironments remain elusive. Here, we investigated the diversity and community composition of bacterial and fungal communities in various blueberry root niches (bulk soil, rhizosphere soil, and root endosphere). The results showed that blueberry root niches significantly affected root-associated microbiome diversity and community composition compared to those of the three host cultivars. Deterministic processes gradually increased along the soil-rhizosphere-root continuum in both bacterial and fungal communities. The co-occurrence network topological features showed that both bacterial and fungal community complexity and intensive interactions decreased along the soil-rhizosphere-root continuum. Different compartment niches clearly influenced bacterial-fungal interkingdom interactions, which were significantly higher in the rhizosphere, and positive interactions gradually dominated the co-occurrence networks from the bulk soil to the endosphere. The functional predictions showed that rhizosphere bacterial and fungal communities may have higher cellulolysis and saprotrophy capacities, respectively. Collectively, the root niches not only affected microbial diversity and community composition but also enhanced the positive interkingdom interactions between bacterial and fungal communities along the soil-rhizosphere-root continuum. This provides an essential basis for manipulating synthetic microbial communities for sustainable agriculture. IMPORTANCE The blueberry root-associated microbiome plays an essential role in its adaptation to acidic soils and in limiting the uptake of soil nutrients by its poor root system. Studies on the interactions of the root-associated microbiome in the various root niches may deepen our understanding of the beneficial effects in this particular habitat. Our study extended the research on the diversity and composition of microbial communities in different blueberry root compartment niches. Root niches dominated the root-associated microbiome compared to that of the host cultivar, and deterministic processes increased from the bulk soil to the endosphere. In addition, bacterial-fungal interkingdom interactions were significantly higher in the rhizosphere, and those positive interactions progressively dominated the co-occurrence network along the soil-rhizosphere-root continuum. Collectively, root niches dominantly affected the root-associated microbiome and the positive interkingdom interactions increased, potentially providing benefits for the blueberry.

[Source](https://doi.org/10.1128/spectrum.05333-22)

10.1128/spectrum.05333-22

---

## MDM-Pred: a freely available web application for predicting the metabolism of drug-like compounds by the gut microbiota

[Source](https://doi.org/10.1080/1062936X.2023.2214375)

---

## HONMF: integration analysis of multi-omics microbiome data via matrix factorization and hypergraph.
 22.05.2023, _Y Ma, L Liu, Y Ma, S Zhang_


The accumulation of multi-omics microbiome data provides an unprecedented opportunity to understand the diversity of bacterial, fungal and viral components from different conditions. The changes in the composition of viruses, bacteria and fungi communities have been associated with environments and critical illness. However, identifying and dissecting the heterogeneity of microbial samples and cross-kingdom interactions remains challenging.

[Source](https://doi.org/10.1093/bioinformatics/btad335)

10.1093/bioinformatics/btad335

---

## Bayesian compositional regression with microbiome features via variational inference.
 25.05.2023, _DAV Scott, E Benavente, J Libiseller-Egger, D Fedorov, J Phelan, E Ilina, P Tikhonova, A Kudryavstev, J Galeeva, T Clark, A Lewin_


_Humans, Bayes Theorem, Linear Models, Microbiota, Markov Chains, Gastrointestinal Microbiome, Monte Carlo Method_
The microbiome plays a key role in the health of the human body. Interest often lies in finding features of the microbiome, alongside other covariates, which are associated with a phenotype of interest. One important property of microbiome data, which is often overlooked, is its compositionality as it can only provide information about the relative abundance of its constituting components. Typically, these proportions vary by several orders of magnitude in datasets of high dimensions. To address these challenges we develop a Bayesian hierarchical linear log-contrast model which is estimated by mean field Monte-Carlo co-ordinate ascent variational inference (CAVI-MC) and easily scales to high dimensional data. We use novel priors which account for the large differences in scale and constrained parameter space associated with the compositional covariates. A reversible jump Monte Carlo Markov chain guided by the data through univariate approximations of the variational posterior probability of inclusion, with proposal parameters informed by approximating variational densities via auxiliary parameters, is used to estimate intractable marginal expectations. We demonstrate that our proposed Bayesian method performs favourably against existing frequentist state of the art compositional data analysis methods. We then apply the CAVI-MC to the analysis of real data exploring the relationship of the gut microbiome to body mass index.

[Source](https://doi.org/10.1186/s12859-023-05219-x)

10.1186/s12859-023-05219-x

---

## The Pathfinder plasmid toolkit for genetically engineering newly isolated bacteria enables the study of Drosophila-colonizing Orbaceae

[Source](https://doi.org/10.1038/s43705-023-00255-3)

---

## Unravelling animal-microbiota evolution on a chip.
 22.05.2023, _O Aizpurua, K Blijleven, U Trivedi, MTP Gilbert, A Alberdi_


Whether and how microorganisms have shaped the evolution of their animal hosts is a major question in biology. Although many animal evolutionary processes appear to correlate with changes in their associated microbial communities, the mechanistic processes leading to these patterns and their causal relationships are still far from being resolved. Gut-on-a-chip models provide an innovative approach that expands beyond the potential of conventional microbiome profiling to study how different animals sense and react to microbes by comparing responses of animal intestinal tissue models to different microbial stimuli. This complementary knowledge can contribute to our understanding of how host genetic features facilitate or prevent different microbiomes from being assembled, and in doing so elucidate the role of host-microbiota interactions in animal evolution.

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(23)00148-8)

10.1016/j.tim.2023.04.010

---

