## Maternal and infant microbiome and birth anthropometry.
 11.10.2024, _S Padiyar, V Nandakumar, S Kollikonda, S Karnati, N Sangwan, H Aly_


Preterm birth is the leading cause of neonatal mortality and morbidity. Microbiome dysbiosis in the mother and infant may contribute to their adverse outcomes. 16S rRNA amplicon sequencing was performed on all samples. Phyloseq, microbiomeSeq, and NetCoMi were utilized for bioinformatics analysis. Statistical tests included the Wilcoxon test, ANOVA, permutational multivariate analysis of variance (PERMANOVA), and linear regression. Statistical significance was set at p value &lt;0.05. The establishment of an infant's microbiome most likely begins in utero and is influenced by the maternal microbiome. Infants' samples were enriched with Salmonella. There is a complex interplay among the microbial taxa noticeable at birth, exhibiting variability in interaction within the same host and across different hosts. Both maternal and infant microbiomes influence the anthropometric measures determined at birth, and a sex-based difference in correlation exists. This study highlights the potential role of maternal and infant microbiomes in improving pregnancy and neonatal outcomes.

[Source](https://www.sciencedirect.com/science/article/pii/S2589004224015372)

10.1016/j.isci.2024.110312

---

## Tumor-colonized Streptococcus mutans metabolically reprograms tumor microenvironment and promotes oral squamous cell carcinoma.
 08.10.2024, _J Zhou, Z Hu, L Wang, Q Hu, Z Chen, T Lin, R Zhou, Y Cai, Z Wu, Z Zhang, Y Yang, C Zhang, G Li, L Zeng, K Su, H Li, Q Su, G Zeng, B Cheng, T Wu_


_Tumor Microenvironment, Streptococcus mutans, Humans, Mouth Neoplasms, Animals, Rats, Saliva, Carcinoma, Squamous Cell, Male, Squamous Cell Carcinoma of Head and Neck, Female_
Oral squamous cell carcinoma (OSCC) remains a major death cause in head and neck cancers, but the exact pathogenesis mechanisms of OSCC are largely unclear.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01907-9)

10.1186/s40168-024-01907-9

---

## Causal relationship between 150 skin microbiomes and prostate cancer: insights from bidirectional mendelian randomization and meta-analysis

[Source](https://www.frontiersin.org/journals/immunology/articles/10.3389/fimmu.2024.1463309/full)

---

## Microbiome and metabolome patterns after lung transplantation reflect underlying disease and chronic lung allograft dysfunction.
 12.10.2024, _C Martin, KS Mahan, TD Wiggen, AJ Gilbertsen, MI Hertz, RC Hunter, RA Quinn_


_Humans, Lung Transplantation, Metabolome, Microbiota, Bronchoalveolar Lavage Fluid, Male, Female, Middle Aged, Adult, RNA, Ribosomal, 16S, Allografts, Aged, Bacteria, Lung, Metabolomics, Lung Diseases, Cystic Fibrosis, Pulmonary Disease, Chronic Obstructive_
Progression of chronic lung disease may lead to the requirement for lung transplant (LTx). Despite improvements in short-term survival after LTx, chronic lung allograft dysfunction (CLAD) remains a critical challenge for long-term survival. This study investigates the molecular and microbial relationships between underlying lung disease and the development of CLAD in bronchoalveolar lavage fluid (BALF) from subjects post-LTx, which is crucial for tailoring treatment strategies specific to allograft dysfunctions.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01893-y)

10.1186/s40168-024-01893-y

---

## Visceral adiposity in postmenopausal women is associated with a pro-inflammatory gut microbiome and immunogenic metabolic endotoxemia.
 07.10.2024, _M Gaber, AS Wilson, AE Millen, KM Hovey, MJ LaMonte, J Wactawski-Wende, HM Ochs-Balcom, KL Cook_


_Female, Gastrointestinal Microbiome, Endotoxemia, Humans, Postmenopause, Animals, Aged, Mice, Lipopolysaccharides, Intra-Abdominal Fat, Inflammation, Aged, 80 and over, Mice, Inbred C57BL, Adiposity, Bacteria, Acute-Phase Proteins, Feces, Obesity, Abdominal, Absorptiometry, Photon, Carrier Proteins, Membrane Glycoproteins_
Obesity, and in particular abdominal obesity, is associated with an increased risk of developing a variety of chronic diseases. Obesity, aging, and menopause are each associated with differential shifts in the gut microbiome. Obesity causes chronic low-grade inflammation due to increased lipopolysaccharide (LPS) levels which is termed "metabolic endotoxemia." We examined the association of visceral adiposity tissue (VAT) area, circulating endotoxemia markers, and the gut bacterial microbiome in a cohort of aged postmenopausal women.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01901-1)

10.1186/s40168-024-01901-1

---

## Abundance measurements reveal the balance between lysis and lysogeny in the human gut microbiome.
 10.10.2024, _J Lopez, S McKeithen-Mead, H Shi, TH Nguyen, KC Huang, BH Good_


The human gut contains diverse communities of bacteriophage, whose interactions with the broader microbiome and potential roles in human health are only beginning to be uncovered. Here, we combine multiple types of data to quantitatively estimate gut phage population dynamics and lifestyle characteristics in human subjects. Unifying results from previous studies, we show that an average human gut contains a low ratio of phage particles to bacterial cells (~1:100), but a much larger ratio of phage genomes to bacterial genomes (~4:1), implying that most gut phage are effectively temperate (e.g., integrated prophage, phage-plasmids, etc.). By integrating imaging and sequencing data with a generalized model of temperate phage dynamics, we estimate that phage induction and lysis occurs at a low average rate (~0.001-0.01 per bacterium per day), imposing only a modest fitness burden on their bacterial hosts. Consistent with these estimates, we find that the phage composition of a diverse synthetic community in gnotobiotic mice can be quantitatively predicted from bacterial abundances alone, while still exhibiting phage diversity comparable to native human microbiomes. These results provide a foundation for interpreting existing and future studies on links between the gut virome and human health.

[Source](https://www.biorxiv.org/content/10.1101/2024.09.27.614587v1)

10.1101/2024.09.27.614587

---

## The role of gut microbiota in prostate inflammation and benign prostatic hyperplasia and its therapeutic implications.
 11.10.2024, _J Chen, B Chen, B Lin, Y Huang, J Li, J Li, Z Chen, P Wang, B Ran, J Yang, H Huang, L Liu, Q Wei, J Ai, D Cao_


The gut microbiota thrives in a complex ecological environment and its dynamic balance is closely related to host health. Recent studies have shown that the occurrence of various diseases including prostate inflammation is related to the dysregulation of the gut microbiome.

[Source](https://www.sciencedirect.com/science/article/pii/S2405844024143332)

10.1016/j.heliyon.2024.e38302

---

## Failure of colonization following gut microbiota transfer exacerbates DSS-induced colitis

[Source](https://www.biorxiv.org/content/10.1101/2024.09.25.614792v1)

---

## Gut microbiome in two high-altitude bird populations showed heterogeneity in sex and life stage.
 11.10.2024, _M Sun, N Halimubieke, B Fang, JO Valdebenito, X Xu, SK Sheppard, T Székely, T Zhang, S He, R Lu, S Ward, AO Urrutia, Y Liu_


Gut microbiotas have important impacts on host health, reproductive success, and survival. While extensive research in mammals has identified the exogenous (e.g. environment) and endogenous (e.g. phylogeny, sex, and age) factors that shape the gut microbiota composition and functionality, yet avian systems remain comparatively less understood. Shorebirds, characterized by a well-resolved phylogeny and diverse life-history traits, present an ideal model for dissecting the factors modulating gut microbiota dynamics. Here, we provide an insight into the composition of gut microbiota in two high-altitude (ca. 3200 m above sea level) breeding populations of Kentish plover (Charadrius alexandrinus) and Tibetan sand plover (Charadrius altrifrons) in the Qinghai-Tibetan Plateau, China. By analysing faecal bacterial communities using 16S rRNA sequencing technology, we find a convergence in gut microbial communities between the two species, dominated by Firmicutes, Proteobacteria, and Bacteroidetes. This suggests that the shared breeding environment potentially acts as a significant determinant shaping their gut microbiota. We also show sex- and age-specific patterns of gut microbiota: female adults maintain a higher diversity than males, and juveniles are enriched in Rhizobiaceae and Exiguobacterium due to their vegetative food resource. Our study not only provides a comprehensive descriptive information for future investigations on the diversity, functionality, and determinants of avian microbiomes, but also underscores the importance of microbial communities in broader ecological contexts.

[Source](https://academic.oup.com/femsmicrobes/article/doi/10.1093/femsmc/xtae020/7706307)

10.1093/femsmc/xtae020

---

## Analysis of rhizosphere fungal diversity in lavender at different planting years based on high-throughput sequencing technology

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0310929)

---

## Soil microbiome perturbation impedes growth of Bouteloua curtipendula and increases relative abundance of soil microbial pathogens

[Source](https://www.biorxiv.org/content/10.1101/2024.10.05.616815v1.abstract)

---

## Adaptation to space conditions of novel bacterial species isolated from the International Space Station revealed by functional gene annotations and comparative genome analysis.
 06.10.2024, _LM Szydlowski, AA Bulbul, AC Simpson, DE Kaya, NK Singh, UO Sezerman, PP Łabaj, T Kosciolek, K Venkateswaran_


_Genome, Bacterial, Space Flight, Weightlessness, Bacteria, Adaptation, Physiological, Molecular Sequence Annotation, Spacecraft, Proteome, Phylogeny, Genomics, Humans_
The extreme environment of the International Space Station (ISS) puts selective pressure on microorganisms unintentionally introduced during its 20+ years of service as a low-orbit science platform and human habitat. Such pressure leads to the development of new features not found in the Earth-bound relatives, which enable them to adapt to unfavorable conditions.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01916-8)

10.1186/s40168-024-01916-8

---

## Metabolomics survey of uropathogenic bacteria in human urine

[Source](https://www.biorxiv.org/content/10.1101/2024.10.07.617107v1.abstract)

---

## Genomic and Functional Portrait of a Highly Virulent, CTX-M-15-Producing H30-Rx Subclone of Escherichia coli Sequence Type 131.
 18.03.2022, _A Ranjan, S Shaik, A Hussain, N Nandanwar, T Semmler, S Jadhav, LH Wieler, N Ahmed_


_Cell Line, Drug Resistance, Multiple, Bacterial, Escherichia coli, Humans, Virulence, beta-Lactamases_
Escherichia coli sequence type 131 (ST131) is a pandemic clone associated with multidrug-resistant, extraintestinal infections, attributable to the presence of the CTX-M-15 extended-spectrum β-lactamase gene and mutations entailing fluoroquinolone resistance. Studies on subclones within E. coli ST131 are critically required for targeting and implementation of successful control efforts. Our study comprehensively analyzed the genomic and functional attributes of the H30-Rx subclonal strains NA097 and NA114, belonging to the ST131 lineage. We carried out whole-genome sequencing, comparative analysis, phenotypic virulence assays, and profiling of the antibacterial responses of THP1 cells infected with these subclones. Phylogenomic analysis suggested that the strains were clonal in nature and confined entirely to a single clade. Comparative genomic analysis revealed that the virulence and resistance repertoires were comparable among the H30-Rx ST131 strains except for the commensal ST131 strain SE15. Similarly, seven phage-specific regions were found to be strongly associated with the H30-Rx strains but were largely absent in the genome of SE15. Phenotypic analysis confirmed the virulence and resistance similarities between the two strains. However, NA097 was found to be more robust than NA114 in terms of virulence gene carriage (dra operon), invasion ability (P &lt; 0.05), and antimicrobial resistance (streptomycin resistance). RT(2) gene expression profiling revealed generic upregulation of key proinflammatory responses in THP1 cells, irrespective of ST131 lineage status. In conclusion, our study provides comprehensive, genome-inferred insights into the biology and immunological properties of ST131 strains and suggests clonal diversification of genomic and phenotypic features within the H30-Rx subclone of E. coli ST131.

[Source](https://www.biorxiv.org/content/10.1101/2024.10.07.617009v1.abstract)

10.1128/AAC.01447-15

---

## FungAMR: A comprehensive portrait of antimicrobial resistance mutations in fungi

[Source](https://www.biorxiv.org/content/10.1101/2024.10.07.617009v1.abstract)

---

## argNorm: Normalization of antibiotic resistance gene annotations to the Antibiotic Resistance Ontology (ARO)

[Source](https://eprints.qut.edu.au/252448/)

---

## BEREN: A bioinformatic tool for recovering Giant viruses, Polinton-like Viruses, and Virophages in metagenomic data

[Source](https://www.biorxiv.org/content/10.1101/2024.10.09.617401v1)

---

## Dataset of 16S rRNA gene sequences of 111 healthy and Newcastle disease infected caecal samples from multiple chicken breeds of Pakistan.
 11.10.2024, _A Ameer, F Saleem, C Keating, O Gundogdu, UZ Ijaz, S Javed_


The article presents a processed dataset from amplicon sequencing of the V4 region of the 16S rRNA gene to recover bacterial and archaeal taxa from the caeca of multiple chicken breeds of Pakistan. These include chicken breeds commonly raised at commercial level, Naked Neck, Black Australorp, Rhode Island Red, White Layer, and Broiler. All the breeds were challenged with Newcastle Disease Virus (NDV), with vaccination against the disease also explored. This resulted in samples belonging to four treatment groups as: Control; Vaccinated; Vaccinated and Challenged; and Non-vaccinated and Challenged. These were raised on an antibiotic free diet in a semi-controlled farming setup. 16S rRNA gene amplicon sequencing of caecal DNA from day old and mature chicken samples (22 weeks for Naked Neck, Black Australorp, Rhode Island Red and White Layer; 8 weeks for Broiler) of the four groups was performed. The paired-end reads from all the samples were quality trimmed, error corrected, and overlapped, on which unique Operational Taxonomic Units (OTUs) were obtained at 99 % similarity. Using predictive modelling, the MetaCyc functional pathways, as well as KEGG orthologs were also recovered. The generated data may be used to explore microbial interactions in gastrointestinal tract with respect to NDV vaccination and infection, together with increased understanding of chicken health and productivity.

[Source](https://www.sciencedirect.com/science/article/pii/S2352340924009193)

10.1016/j.dib.2024.110957

---

## High accuracy meets high throughput for near full-length 16S ribosomal RNA amplicon sequencing on the Nanopore platform.
 11.10.2024, _X Lin, K Waring, H Ghezzi, C Tropini, J Tyson, RM Ziels_


Small subunit (SSU) ribosomal RNA (rRNA) gene amplicon sequencing is a foundational method in microbial ecology. Currently, short-read platforms are commonly employed for high-throughput applications of SSU rRNA amplicon sequencing, but at the cost of poor taxonomic classification due to limited fragment lengths. The Oxford Nanopore Technologies (ONT) platform can sequence full-length SSU rRNA genes, but its lower raw-read accuracy has so-far limited accurate taxonomic classification and de novo feature generation. Here, we present a sequencing workflow, termed ssUMI, that combines unique molecular identifier (UMI)-based error correction with newer (R10.4+) ONT chemistry and sample barcoding to enable high throughput near full-length SSU rRNA (e.g. 16S rRNA) amplicon sequencing. The ssUMI workflow generated near full-length 16S rRNA consensus sequences with 99.99% mean accuracy using a minimum subread coverage of 3×, surpassing the accuracy of Illumina short reads. The consensus sequences generated with ssUMI were used to produce error-free de novo sequence features with no false positives with two microbial community standards. In contrast, Nanopore raw reads produced erroneous de novo sequence features, indicating that UMI-based error correction is currently necessary for high-accuracy microbial profiling with R10.4+ ONT sequencing chemistries. We showcase the cost-competitive scalability of the ssUMI workflow by sequencing 87 time-series wastewater samples and 27 human gut samples, obtaining quantitative ecological insights that were missed by short-read amplicon sequencing. ssUMI, therefore, enables accurate and low-cost full-length 16S rRNA amplicon sequencing on Nanopore, improving accessibility to high-resolution microbiome science.

[Source](https://academic.oup.com/pnasnexus/article/3/10/pgae411/7816023)

10.1093/pnasnexus/pgae411

---

