## Gut metagenomes of Asian octogenarians reveal metabolic potential expansion and distinct microbial species associated with aging phenotypes.
 10.09.2024, _A Ravikrishnan, I Wijaya, E Png, KR Chng, EXP Ho, AHQ Ng, AN Mohamed Naim, JS Gounot, SP Guan, JL Hanqing, L Guan, C Li, JY Koh, PF de Sessions, WP Koh, L Feng, TP Ng, A Larbi, AB Maier, BK Kennedy, N Nagarajan_


_Humans, Gastrointestinal Microbiome, Singapore, Male, Aged, 80 and over, Female, Aging, Metagenome, Asian People, Phenotype, Metagenomics, Bacteria, Bacteroides, Cohort Studies, Feces_
While rapid demographic changes in Asia are driving the incidence of chronic aging-related diseases, the limited availability of high-quality in vivo data hampers our ability to understand complex multi-factorial contributions, including gut microbial, to healthy aging. Leveraging a well-phenotyped cohort of community-living octogenarians in Singapore, we used deep shotgun-metagenomic sequencing for high-resolution taxonomic and functional characterization of their gut microbiomes (n = 234). Joint species-level analysis with other Asian cohorts identified distinct age-associated shifts characterized by reduction in microbial richness, and specific Alistipes and Bacteroides species enrichment (e.g., Alistipes shahii and Bacteroides xylanisolvens). Functional analysis confirmed these changes correspond to metabolic potential expansion in aging towards alternate pathways synthesizing and utilizing amino-acid precursors, vis-à-vis dominant microbial guilds producing butyrate in gut from pyruvate (e.g., Faecalibacterium prausnitzii, Roseburia inulinivorans). Extending these observations to key clinical markers helped identify &gt;10 robust microbial associations to inflammation, cardiometabolic and liver health, including potential probiotic species (e.g., Parabacteroides goldsteinii) and pathobionts (e.g., Klebsiella pneumoniae), highlighting the microbiome's role as biomarkers and potential targets for promoting healthy aging.

[Source](https://doi.org/10.1038/s41467-024-52097-9)

10.1038/s41467-024-52097-9

---

## Integrated analysis of facial microbiome and skin physio-optical properties unveils cutotype-dependent aging effects.
 07.09.2024, _C Sun, G Hu, L Yi, W Ge, Q Yang, X Yang, Y He, Z Liu, WH Chen_


_Humans, Microbiota, Skin, Face, Middle Aged, Skin Aging, Female, Adult, Male, Bacteria, Aged, Aging, Metagenome, Young Adult, High-Throughput Nucleotide Sequencing, Sebum_
Our facial skin hosts millions of microorganisms, primarily bacteria, crucial for skin health by maintaining the physical barrier, modulating immune response, and metabolizing bioactive materials. Aging significantly influences the composition and function of the facial microbiome, impacting skin immunity, hydration, and inflammation, highlighting potential avenues for interventions targeting aging-related facial microbes amidst changes in skin physiological properties.

[Source](https://doi.org/10.1186/s40168-024-01891-0)

10.1186/s40168-024-01891-0

---

## Regulation of <em>Bacteroides acidifaciens</em> by the aryl hydrocarbon receptor in IL-22-producing immune cells has sex-dependent consequential impact on colitis

[Source](https://doi.org/10.3389/fimmu.2024.1444045)

---

## The role of KLF5 in gut microbiota and lung adenocarcinoma: unveiling programmed cell death pathways and prognostic biomarkers.
 10.09.2024, _Q Fang, M Xu, W Yao, R Wu, R Han, S Kawakita, A Shen, S Guan, J Zhang, X Sun, M Zhou, N Li, Q Sun, CS Dong_


Lung adenocarcinoma (LUAD) is the most important subtype of lung cancer. It is well known that the gut microbiome plays an important role in the pathophysiology of various diseases, including cancer, but little research has been done on the intestinal microbiome associated with LUAD. Utilizing bioinformatics tools and data analysis, we identified novel potential prognostic biomarkers for LUAD. To integrate differentially expressed genes and clinical significance modules, we used a weighted correlation network analysis system. According to the Peryton database and the gutMGene database, the composition and structure of gut microbiota in LUAD patients differed from those in healthy individuals. LUAD was associated with 150 gut microbiota and 767 gut microbiota targets, with Krüppel-like factor 5 (KLF5) being the most closely related. KLF5 was associated with immune status and correlated well with the prognosis of LUAD patients. The identification of KLF5 as a potential prognostic biomarker suggests its utility in improving risk stratification and guiding personalized treatment strategies for LUAD patients. Altogether, KLF5 could be a potential prognostic biomarker in LUAD.

[Source](https://doi.org/10.1007/s12672-024-01257-w)

10.1007/s12672-024-01257-w

---

## Healthy first-degree relatives from multiplex families versus simplex families have a higher subclinical intestinal inflammation, a distinct fecal microbial signature, and harbor a higher risk of developing Crohn’s disease

[Source](https://www.gastrojournal.org/article/S0016-5085(24)05414-3/pdf)

---

## Reversions mask the contribution of adaptive evolution in microbiomes.
 10.09.2024, _PA Torrillo, TD Lieberman_


_Mutation, Evolution, Molecular, Selection, Genetic, Genome, Bacterial, Microbiota, Gastrointestinal Microbiome, Bacteroides, Adaptation, Physiological, Models, Genetic, Bacteria_
When examining bacterial genomes for evidence of past selection, the results depend heavily on the mutational distance between chosen genomes. Even within a bacterial species, genomes separated by larger mutational distances exhibit stronger evidence of purifying selection as assessed by dN/dS, the normalized ratio of nonsynonymous to synonymous mutations. Here, we show that the classical interpretation of this scale dependence, weak purifying selection, leads to problematic mutation accumulation when applied to available gut microbiome data. We propose an alternative, adaptive reversion model with opposite implications for dynamical intuition and applications of dN/dS. Reversions that occur and sweep within-host populations are nearly guaranteed in microbiomes due to large population sizes, short generation times, and variable environments. Using analytical and simulation approaches, we show that adaptive reversion can explain the dN/dS decay given only dozens of locally fluctuating selective pressures, which is realistic in the context of Bacteroides genomes. The success of the adaptive reversion model argues for interpreting low values of dN/dS obtained from long timescales with caution as they may emerge even when adaptive sweeps are frequent. Our work thus inverts the interpretation of an old observation in bacterial evolution, illustrates the potential of mutational reversions to shape genomic landscapes over time, and highlights the importance of studying bacterial genomic evolution on short timescales.

[Source](https://doi.org/10.7554/eLife.93146)

10.7554/eLife.93146

---

## The hidden base of the iceberg: gut peptidoglycome dynamics is foundational to its influence on the host.
 11.09.2024, _R Wheeler, I Gomperts Boneca_


_Humans, Gastrointestinal Microbiome, Peptidoglycan, Bacteria, Animals, Host Microbial Interactions, Homeostasis, Cell Wall_
The intestinal microbiota of humans includes a highly diverse range of bacterial species. All these bacteria possess a cell wall, composed primarily of the macromolecule peptidoglycan. As such, the gut also harbors an abundant and varied peptidoglycome. A remarkable range of host physiological pathways are regulated by peptidoglycan fragments that originate from the gut microbiota and enter the host system. Interactions between the host system and peptidoglycan can influence physiological development and homeostasis, promote health, or contribute to inflammatory disease. Underlying these effects is the interplay between microbiota composition and enzymatic processes that shape the intestinal peptidoglycome, dictating the types of peptidoglycan generated, that subsequently cross the gut barrier. In this review, we highlight and discuss the hidden and emerging functional aspects of the microbiome, i.e. the hidden base of the iceberg, that modulate the composition of gut peptidoglycan, and how these fundamental processes are drivers of physiological outcomes for the host.

[Source](https://doi.org/10.1080/19490976.2024.2395099)

10.1080/19490976.2024.2395099

---

## Benefits and concerns of probiotics: an overview of the potential genotoxicity of the colibactin-producing <em>Escherichia coli </em>Nissle 1917 strain

[Source](https://doi.org/10.1080/19490976.2024.2397874)

---

## Microbial Extracellular Vesicles in Host-Microbiota Interactions.
 06.09.2024, _S Abubaker, S Miri, W Mottawea, R Hammami_


_Extracellular Vesicles, Humans, Animals, Microbiota, Host Microbial Interactions_
Extracellular vesicles have emerged as key players in cellular communication, influencing various physiological processes and pathophysiological progression, including digestion, immune response, and tissue repairs. Recently, a class of EVs derived from microbial communities has gained significant attention due to their pivotal role in intercellular communication and their potential as biomarkers and biotherapeutic agents. Microbial EVs are membrane-bound molecules encapsulating bioactive metabolites that modulate host physiological and pathological processes. This chapter discusses the evolving history of microbiota-produced EVs, including their discovery, characterization, current research status, and their diverse mechanisms of interaction with other microbes and hosts. This review also highlights the importance of EVs in health and disease and discusses recent research that shows promising results for the therapeutic potential of EVs.

[Source](https://doi.org/10.1007/978-3-031-62036-2_19)

10.1007/978-3-031-62036-2_19

---

## A probiotic <em>Limosilactobacillus fermentum </em>GR-3 mitigates colitis-associated tumorigenesis in mice via modulating gut microbiome

[Source](https://doi.org/10.1038/s41538-024-00307-5)

---

## Pathophysiological and Clinical Potential of Human Microbiome: Microbe-based Therapeutic Insights.
 06.09.2024, _S Sanyal, K Nigam, S Singh, P Lohani, M Dwivedi_


The human microbiota represents the community and diverse population of microbes within the human body, which comprises approximately 100 trillion micro-organisms. They exist in the human gastrointestinal tract and various other organs and are now considered virtual body organs. It is mainly represented by bacteria but also includes viruses, fungi, and protozoa. Although there is a heritable component to the gut microbiota, environmental factors related to diet, drugs, and anthropometry determine the composition of the microbiota. Besides the gastrointestinal tract, the human body also harbours microbial communities in the skin, oral and nasal cavities, and reproductive tract. The current review demonstrates the role of gut microbiota and its involvement in processing food, drugs, and immune responses. The discussion focuses on the implications of human microbiota in developing several diseases, such as gastrointestinal infections, metabolic disorders, malignancies, etc., through symbiotic relationships. The microbial population may vary depending on the pathophysiological condition of an individual and thus may be exploited as a therapeutic and clinical player. Further, we need a more thorough investigation to establish the correlation between microbes and pathophysiology in humans and propose them as potential therapeutic targets.

[Source](https://doi.org/10.2174/0113892010314433240823113111)

10.2174/0113892010314433240823113111

---

## Soil volatilomics uncovers tight linkage between soybean presence and soil omics profiles in agricultural fields.
 07.09.2024, _H Kuchikata, M Sano, F Fujiwara, K Murashima, K Kumaishi, M Narukawa, Y Nose, M Kobayashi, S Hamamoto, NI Kobayashi, A Sugiyama, N Nihei, Y Ichihashi, M Kusano_


_Glycine max, Volatile Organic Compounds, Soil, Agriculture, Metabolomics, Soil Microbiology, Microbiota_
Securing a stable food supply and achieving sustainable agricultural production are essential for mitigating future food insecurity. Soil metabolomics is a promising tool for capturing soil status, which is a critical issue for future sustainable food security. This study aims to provide deeper insights into the status of soybean-grown fields under varying soil conditions over three years by employing comprehensive soil volatile organic compound (VOC) profiling, also known as soil volatilomics. Profiling identified approximately 200 peaks in agricultural fields. The soil of soybean-presented plots exhibited markedly higher VOC levels than those of non-soybean plots during the flowering season. Pentanoic acid, 2,2,4-trimethyl-3-carboxyisopropyl, isobutyl ester, a discriminative soil VOC, was identified through multivariate data analysis as a distinctively present VOC in fields with or without soybean plants during the flowering period. Soil VOC profiles exhibited strong correlations with soil-related omics datasets (soil ionome, microbiome, metabolome, and physics) and no significant correlations with root microbiome and rhizosphere chemicals. These findings indicate that soil VOC profiles could serve as a valuable indicator for assessing soil status, thereby supporting efforts to ensure future global food security.

[Source](https://doi.org/10.1038/s41598-024-70873-x)

10.1038/s41598-024-70873-x

---

## Meddling with the microbiota: Fungal tricks to infect plant hosts.
 06.09.2024, _F Mesny, M Bauer, J Zhu, BPHJ Thomma_


Plants associate with a wealth of microbes, collectively referred to as the plant microbiota, whose composition is determined by host plant genetics, immune responses, environmental factors and intermicrobial relations. Unsurprisingly, microbiota compositions change during disease development. Recent evidence revealed that some of these changes can be attributed to effector proteins with antimicrobial activities that are secreted by plant pathogens to manipulate host microbiota to their advantage. Intriguingly, many of these effectors have ancient origins, predating land plant emergence, and evolved over long evolutionary trajectories to acquire selective antimicrobial activities to target microbial antagonists in host plant microbiota. Thus, we argue that host-pathogen co-evolution likely involved arms races within the host-associated microbiota.

[Source](https://doi.org/10.1016/j.pbi.2024.102622)

10.1016/j.pbi.2024.102622

---

## A comprehensive catalog encompassing 1376 species-level genomes reveals the core community and functional diversity of anammox microbiota.
 05.09.2024, _D Wang, Y Meng, LN Huang, XX Zhang, X Luo, F Meng_


Research on the microbial community and function of the anammox process for environmentally friendly wastewater treatment has achieved certain success, which may mean more universal insights are needed. However, the comprehensive understanding of the anammox process is constrained by the limited taxonomic assignment and functional characterization of anammox microbiota, primarily due to the scarcity of high-quality genomes for most organisms. This study reported a global genome catalog of anammox microbiotas based on numerous metagenomes obtained from both lab- and full-scale systems. A total of 1376 candidate species from 7474 metagenome-assembled genomes were used to construct the genome catalog, providing extensive microbial coverage (averaged of 92.40 %) of anammox microbiota. Moreover, a total of 64 core genera and 44 core species were identified, accounting for approximately 64.25 % and 43.97 %, respectively, of anammox microbiota. The strict core genera encompassed not only functional bacteria (e.g., Brocadia, Desulfobacillus, Zeimonas, and Nitrosomonas) but also two candidate genera (UBA12294 and OLB14) affiliated with the order Anaerolineales. In particular, core denitrifying bacteria with observably taxonomic diversity exhibited diverse functional profiles; for instance, the potential of carbohydrate metabolism in Desulfobacillus and Zeimonas likely improves the mixotrophic lifestyle of anammox microbiota. Besides, a noteworthy association was detected between anammox microbiota and system type. Microbiota in coupling system exhibited complex diversity and interspecies interactions by limiting numerous core denitrifying bacteria. In summary, the constructed catalog substantially expands our understanding of the core community and their functions of anammox microbiota, providing a valuable resource for future studies on anammox systems.

[Source](https://doi.org/10.1016/j.watres.2024.122356)

10.1016/j.watres.2024.122356

---

## Revealing the intricate temporal dynamics and adaptive responses of prokaryotic and eukaryotic microbes in the coastal South China Sea.
 06.09.2024, _W Xu, Y Xu, R Sun, E Rey Redondo, KK Leung, SH Wan, J Li, CCM Yung_


This comprehensive two-year investigation in the coastal South China Sea has advanced our understanding of marine microbes at both community and genomic levels. By combining metagenomics and metatranscriptomics, we have revealed the intricate temporal dynamics and remarkable adaptability of microbial communities and phytoplankton metagenome-assembled genomes (MAGs) in response to environmental fluctuations. We observed distinct seasonal shifts in microbial community composition and function: cyanobacteria were predominant during warmer months, whereas photosynthetic protists were more abundant during colder seasons. Notably, metabolic marker KOs of photosynthesis were consistently active throughout the year, underscoring the persistent role of these processes irrespective of seasonal changes. Our analysis reveals that environmental parameters such as temperature, salinity, and nitrate concentrations profoundly influence microbial community composition, while temperature and silicate have emerged as crucial factors shaping their functional traits. Through the recovery and analysis of 37 phytoplankton MAGs, encompassing nine prokaryotic cyanobacteria and 28 eukaryotic protists from diverse phyla, we have gained insights into their genetic diversity and metabolic capabilities. Distinct profiles of photosynthesis-related pathways including carbon fixation, carotenoid biosynthesis, photosynthesis-antenna proteins, and photosynthesis among the MAGs indicated their genetic adaptations to changing environmental conditions. This study not only enhances our understanding of microbial dynamics in coastal marine ecosystems but also sheds light on the ecological roles and adaptive responses of different microbial groups to environmental changes.

[Source](https://doi.org/10.1016/j.scitotenv.2024.176019)

10.1016/j.scitotenv.2024.176019

---

## Biodegradation of low-density polyethylene by mixed fungi composed of <em>Alternaria </em>sp. and <em>Trametes </em>sp. isolated from landfill sites

[Source](https://doi.org/10.1186/s12866-024-03477-0)

---

## Using patterns of shared taxa to infer bacterial dispersal in human living environment in urban and rural areas

[Source](https://doi.org/10.1128/aem.00903-24)

---

## Annotation-free prediction of microbial dioxygen utilization.
 04.09.2024, _AI Flamholz, JE Goldford, PA Richter, EM Larsson, A Jinich, WW Fischer, DK Newman_


Aerobes require dioxygen (O2) to grow; anaerobes do not. However, nearly all microbes-aerobes, anaerobes, and facultative organisms alike-express enzymes whose substrates include O2, if only for detoxification. This presents a challenge when trying to assess which organisms are aerobic from genomic data alone. This challenge can be overcome by noting that O2 utilization has wide-ranging effects on microbes: aerobes typically have larger genomes encoding distinctive O2-utilizing enzymes, for example. These effects permit high-quality prediction of O2 utilization from annotated genome sequences, with several models displaying ≈80% accuracy on a ternary classification task for which blind guessing is only 33% accurate. Since genome annotation is compute-intensive and relies on many assumptions, we asked if annotation-free methods also perform well. We discovered that simple and efficient models based entirely on genomic sequence content-e.g., triplets of amino acids-perform as well as intensive annotation-based classifiers, enabling rapid processing of genomes. We further show that amino acid trimers are useful because they encode information about protein composition and phylogeny. To showcase the utility of rapid prediction, we estimated the prevalence of aerobes and anaerobes in diverse natural environments cataloged in the Earth Microbiome Project. Focusing on a well-studied O2 gradient in the Black Sea, we found quantitative correspondence between local chemistry (O2:sulfide concentration ratio) and the composition of microbial communities. We, therefore, suggest that statistical methods like ours might be used to estimate, or "sense," pivotal features of the chemical environment using DNA sequencing data.IMPORTANCEWe now have access to sequence data from a wide variety of natural environments. These data document a bewildering diversity of microbes, many known only from their genomes. Physiology-an organism's capacity to engage metabolically with its environment-may provide a more useful lens than taxonomy for understanding microbial communities. As an example of this broader principle, we developed algorithms that accurately predict microbial dioxygen utilization directly from genome sequences without annotating genes, e.g., by considering only the amino acids in protein sequences. Annotation-free algorithms enable rapid characterization of natural samples, highlighting quantitative correspondence between sequences and local O2 levels in a data set from the Black Sea. This example suggests that DNA sequencing might be repurposed as a multi-pronged chemical sensor, estimating concentrations of O2 and other key facets of complex natural settings.

[Source](https://doi.org/10.1128/msystems.00763-24)

10.1128/msystems.00763-24

---

## MRGM: an enhanced catalog of mouse gut microbial genomes substantially broadening taxonomic and functional landscapes.
 10.09.2024, _N Kim, CY Kim, J Ma, S Yang, DJ Park, SJ Ha, P Belenky, I Lee_


_Animals, Gastrointestinal Microbiome, Mice, Bacteria, Genome, Bacterial, Humans, Phylogeny, Diet_
Mouse gut microbiome research is pivotal for understanding the human gut microbiome, providing insights into disease modeling, host-microbe interactions, and the dietary influence on the gut microbiome. To enhance the translational value of mouse gut microbiome studies, we need detailed and high-quality catalogs of mouse gut microbial genomes. We introduce the Mouse Reference Gut Microbiome (MRGM), a comprehensive catalog with 42,245 non-redundant mouse gut bacterial genomes across 1,524 species. MRGM marks a 40% increase in the known taxonomic diversity of mouse gut microbes, capturing previously underrepresented lineages through refined genome quality assessment techniques. MRGM not only broadens the taxonomic landscape but also enriches the functional landscape of the mouse gut microbiome. Using deep learning, we have elevated the Gene Ontology annotation rate for mouse gut microbial proteins from 3.2% with orthology to 60%, marking an over 18-fold increase. MRGM supports both DNA- and marker-based taxonomic profiling by providing custom databases, surpassing previous catalogs in performance. Finally, taxonomic and functional comparisons between human and mouse gut microbiota reveal diet-driven divergences in their taxonomic composition and functional enrichment. Overall, our study highlights the value of high-quality microbial genome catalogs in advancing our understanding of the co-evolution between gut microbes and their host.

[Source](https://doi.org/10.1080/19490976.2024.2393791)

10.1080/19490976.2024.2393791

---

## A unified web cloud computing platform MiMedSurv for microbiome causal mediation analysis with survival responses.
 07.09.2024, _H Jang, H Koh_


_Humans, Microbiota, Cloud Computing, Internet, Software, Survival Analysis_
In human microbiome studies, mediation analysis has recently been spotlighted as a practical and powerful analytic tool to survey the causal roles of the microbiome as a mediator to explain the observed relationships between a medical treatment/environmental exposure and a human disease. We also note that, in a clinical research, investigators often trace disease progression sequentially in time; as such, time-to-event (e.g., time-to-disease, time-to-cure) responses, known as survival responses, are prevalent as a surrogate variable for human health or disease. In this paper, we introduce a web cloud computing platform, named as microbiome mediation analysis with survival responses (MiMedSurv), for comprehensive microbiome mediation analysis with survival responses on user-friendly web environments. MiMedSurv is an extension of our prior web cloud computing platform, named as microbiome mediation analysis (MiMed), for survival responses. The two main features that are well-distinguished are as follows. First, MiMedSurv conducts some baseline exploratory non-mediational survival analysis, not involving microbiome, to survey the disparity in survival response between medical treatments/environmental exposures. Then, MiMedSurv identifies the mediating roles of the microbiome in various aspects: (i) as a microbial ecosystem using ecological indices (e.g., alpha and beta diversity indices) and (ii) as individual microbial taxa in various hierarchies (e.g., phyla, classes, orders, families, genera, species). To illustrate its use, we survey the mediating roles of the gut microbiome between antibiotic treatment and time-to-type 1 diabetes. MiMedSurv is freely available on our web server ( http://mimedsurv.micloud.kr ).

[Source](https://doi.org/10.1038/s41598-024-71852-y)

10.1038/s41598-024-71852-y

---

## A quantitative approach to measure and predict microbiome response to antibiotics.
 04.09.2024, _V Tu, Y Ren, C Tanes, S Mukhopadhyay, SG Daniel, H Li, K Bittinger_


Although antibiotics induce sizable perturbations in the human microbiome, we lack a systematic and quantitative method to measure and predict the microbiome's response to specific antibiotics. Here, we introduce such a method, which takes the form of a microbiome response index (MiRIx) for each antibiotic. Antibiotic-specific MiRIx values quantify the overall susceptibility of the microbiota to an antibiotic, based on databases of bacterial phenotypes and published data on intrinsic antibiotic susceptibility. We applied our approach to five published microbiome studies that carried out antibiotic interventions with vancomycin, metronidazole, ciprofloxacin, amoxicillin, and doxycycline. We show how MiRIx can be used in conjunction with existing microbiome analytical approaches to gain a deeper understanding of the microbiome response to antibiotics. Finally, we generate antibiotic response predictions for the oral, skin, and gut microbiome in healthy humans. Our approach is implemented as open-source software and is readily applied to microbiome data sets generated by 16S rRNA marker gene sequencing or shotgun metagenomics.

[Source](https://doi.org/10.1128/msphere.00488-24)

10.1128/msphere.00488-24

---

## Inhibited hypoxia-inducible factor by intraoperative hyperglycemia increased postoperative delirium of aged patients: A review.
 11.09.2024, _Y Han, B Ji, Y Leng, C Xie_


_Humans, Hyperglycemia, Aged, Postoperative Complications, Delirium, Hypoxia-Inducible Factor 1, alpha Subunit, Hypoxia-Inducible Factor 1, Anesthesia, General_
The underlying mechanism of postoperative delirium (POD) in elderly people remains unclear. Perioperative hyperglycemia (POHG) is an independent risk indicator for POD, particularly in the elderly. Under cerebral desaturation (hypoxia) during general anesthesia, hypoxia-inducible factor (HIF) is neuroprotective during cerebral hypoxia via diverse pathways, like glucose metabolism and angiogenesis. Hyperglycemia can repress HIF expression and activity. On the other hand, POHG occurred among patients undergoing surgery. For surgical stress, hypothalamic-pituitary-adrenal activation and sympathoadrenal activation may increase endogenous glucose production via gluconeogenesis and glycogenolysis. Thus, under the setting of cerebral hypoxia during general anesthesia, we speculate that POHG prevents HIF-1α levels and function in the brain of aged patients, thus exacerbating the hypoxic response of HIF-1 and potentially contributing to POD. This paper sketches the underlying mechanisms of HIF in POD in elderly patients and offers novel insights into targets for preventing or treating POD in the same way as POHG.

[Source](https://doi.org/10.1093/bioinformatics/btae397)

10.1097/MD.0000000000038349

---

## Metagenomic functional profiling: to sketch or not to sketch?

[Source](https://doi.org/10.1093/bioinformatics/btae397)

---

## TolRad, a model for predicting radiation tolerance using Pfam annotations, identifies novel radiosensitive bacterial species from reference genomes and MAGs

[Source](https://doi.org/10.1128/spectrum.03838-23)

---

## Improving viral annotation with artificial intelligence.
 04.09.2024, _ZN Flamholz, C Li, L Kelly_


Viruses of bacteria, "phages," are fundamental, poorly understood components of microbial community structure and function. Additionally, their dependence on hosts for replication positions phages as unique sensors of ecosystem features and environmental pressures. High-throughput sequencing approaches have begun to give us access to the diversity and range of phage populations in complex microbial community samples, and metagenomics is currently the primary tool with which we study phage populations. The study of phages by metagenomic sequencing, however, is fundamentally limited by viral diversity, which results in the vast majority of viral genomes and metagenome-annotated genomes lacking annotation. To harness bacteriophages for applications in human and environmental health and disease, we need new methods to organize and annotate viral sequence diversity. We recently demonstrated that methods that leverage self-supervised representation learning can supplement statistical sequence representations for remote viral protein homology detection in the ocean virome and propose that consideration of the functional content of viral sequences allows for the identification of similarity in otherwise sequence-diverse viruses and viral-like elements for biological discovery. In this review, we describe the potential and pitfalls of large language models for viral annotation. We describe the need for new approaches to annotate viral sequences in metagenomes, the fundamentals of what protein language models are and how one can use them for sequence annotation, the strengths and weaknesses of these models, and future directions toward developing better models for viral annotation more broadly.

[Source](https://doi.org/10.1128/mbio.03206-23)

10.1128/mbio.03206-23

---

## Sampling the Maize (<em>Zea mays</em>) Leaf Microbiome

[Source](https://doi.org/10.1101/pdb.prot108581)

---

## Sampling Root-Associated Microbiome Communities of Maize (<em>Zea mays</em>)

[Source](https://doi.org/10.1101/pdb.prot108580)

---

## Sampling and Analysis of the Maize Microbiome

[Source](https://doi.org/10.1101/pdb.top108463)

---

## Manipulating the Maize (<em>Zea mays</em>) Microbiome

[Source](https://doi.org/10.1101/pdb.prot108584)

---

