## The Metaproteomics Initiative: a coordinated approach for propelling the functional characterization of microbiomes.
 24.12.2021, _T Van Den Bossche, MØ Arntzen, D Becher, D Benndorf, VGH Eijsink, C Henry, PD Jagtap, N Jehmlich, C Juste, BJ Kunath, B Mesuere, T Muth, PB Pope, J Seifert, A Tanca, S Uzzau, P Wilmes, RL Hettich, J Armengaud_


Through connecting genomic and metabolic information, metaproteomics is an essential approach for understanding how microbiomes function in space and time. The international metaproteomics community is delighted to announce the launch of the Metaproteomics Initiative (www.metaproteomics.org), the goal of which is to promote dissemination of metaproteomics fundamentals, advancements, and applications through collaborative networking in microbiome research. The Initiative aims to be the central information hub and open meeting place where newcomers and experts interact to communicate, standardize, and accelerate experimental and bioinformatic methodologies in this field. We invite the entire microbiome community to join and discuss potential synergies at the interfaces with other disciplines, and to collectively promote innovative approaches to gain deeper insights into microbiome functions and dynamics. Video Abstract.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01176-w)

10.1186/s40168-021-01176-w

---

## Directional Gaussian Mixture Models of the Gut Microbiome Elucidate Microbial Spatial Structure.
 22.12.2021, _AP Pasarkar, TA Joseph, I Pe'er_


The gut microbiome is spatially heterogeneous, with environmental niches contributing to the distribution and composition of microbial populations. A recently developed mapping technology, MaPS-seq, aims to characterize the spatial organization of the gut microbiome by providing data about local microbial populations. However, information about the global arrangement of these populations is lost by MaPS-seq. To address this, we propose a class of Gaussian mixture models (GMM) with spatial dependencies between mixture components in order to computationally recover the relative spatial arrangement of microbial communities. We demonstrate on synthetic data that our spatial models can identify global spatial dynamics, accurately cluster data, and improve parameter inference over a naive GMM. We applied our model to three MaPS-seq data sets taken from various regions of the mouse intestine. On cecal and distal colon data sets, we find our model accurately recapitulates known spatial behaviors of the gut microbiome, including compositional differences between mucus and lumen-associated populations. Our model also seems to capture the role of a pH gradient on microbial populations in the mouse ileum and proposes new behaviors as well. IMPORTANCE The spatial arrangement of the microbes in the gut microbiome is a defining characteristic of its behavior. Various experimental studies have attempted to provide glimpses into the mechanisms that contribute to microbial arrangements. However, many of these descriptions are qualitative. We developed a computational method that takes microbial spatial data and learns many of the experimentally validated spatial factors. We can then use our model to propose previously unknown spatial behaviors. Our results demonstrate that the gut microbiome, while exceptionally large, has predictable spatial patterns that can be used to help us understand its role in health and disease.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00817-21)

10.1128/mSystems.00817-21

---

## Human gut-microbiome-derived propionate coordinates proteasomal degradation via HECTD2 upregulation to target EHMT2 in colorectal cancer.
 01.01.2022, _TY Ryu, K Kim, TS Han, MO Lee, J Lee, J Choi, KB Jung, EJ Jeong, DM An, CR Jung, JH Lim, J Jung, K Park, MS Lee, MY Kim, SJ Oh, K Hur, R Hamamoto, DS Park, DS Kim, MY Son, HS Cho_


The human microbiome plays an essential role in the human immune system, food digestion, and protection from harmful bacteria by colonizing the human intestine. Recently, although the human microbiome affects colorectal cancer (CRC) treatment, the mode of action between the microbiome and CRC remains unclear. This study showed that propionate suppressed CRC growth by promoting the proteasomal degradation of euchromatic histone-lysine N-methyltransferase 2 (EHMT2) through HECT domain E3 ubiquitin protein ligase 2 (HECTD2) upregulation. In addition, EHMT2 downregulation reduced the H3K9me2 level on the promoter region of tumor necrosis factor α-induced protein 1 (TNFAIP1) as a novel direct target of EHMT2. Subsequently, TNFAIP1 upregulation induced the apoptosis of CRC cells. Furthermore, using Bacteroides thetaiotaomicron culture medium, we confirmed EHMT2 downregulation via upregulation of HECTD2 and TNFAIP1 upregulation. Finally, we observed the synergistic effect of propionate and an EHMT2 inhibitor (BIX01294) in 3D spheroid culture models. Thus, we suggest the anticancer effects of propionate and EHMT2 as therapeutic targets for colon cancer treatment and may provide the possibility for the synergistic effects of an EHMT2 inhibitor and microbiome in CRC treatment.

[Source](https://www.nature.com/articles/s41396-021-01119-1)

10.1038/s41396-021-01119-1

---

## Gut microbiomes from Gambian infants reveal the development of a non-industrialized Prevotella-based trophic network.
 05.01.2022, _MC de Goffau, AT Jallow, C Sanyang, AM Prentice, N Meagher, DJ Price, PA Revill, J Parkhill, DIA Pereira, J Wagner_


Distinct bacterial trophic networks exist in the gut microbiota of individuals in industrialized and non-industrialized countries. In particular, non-industrialized gut microbiomes tend to be enriched with Prevotella species. To study the development of these Prevotella-rich compositions, we investigated the gut microbiota of children aged between 7 and 37 months living in rural Gambia (616 children, 1,389 stool samples, stratified by 3-month age groups). These infants, who typically eat a high-fibre, low-protein diet, were part of a double-blind, randomized iron intervention trial (NCT02941081) and here we report the secondary outcome. We found that child age was the largest discriminating factor between samples and that anthropometric indices (collection time points, season, geographic collection site, and iron supplementation) did not significantly influence the gut microbiome. Prevotella copri, Faecalibacterium prausnitzii and Prevotella stercorea were, on average, the most abundant species in these 1,389 samples (35%, 11% and 7%, respectively). Distinct bacterial trophic network clusters were identified, centred around either P. stercorea or F. prausnitzii and were found to develop steadily with age, whereas P. copri, independently of other species, rapidly became dominant after weaning. This dataset, set within a critical gut microbial developmental time frame, provides insights into the development of Prevotella-rich gut microbiomes, which are typically understudied and are underrepresented in western populations.

[Source](https://www.nature.com/articles/s41564-021-01023-6)

10.1038/s41564-021-01023-6

---

## Reuterin in the healthy gut microbiome suppresses colorectal cancer growth through altering redox balance.
 24.12.2021, _HN Bell, RJ Rebernick, J Goyert, R Singhal, M Kuljanin, SA Kerk, W Huang, NK Das, A Andren, S Solanki, SL Miller, PK Todd, ER Fearon, CA Lyssiotis, SP Gygi, JD Mancias, YM Shah_


Microbial dysbiosis is a colorectal cancer (CRC) hallmark and contributes to inflammation, tumor growth, and therapy response. Gut microbes signal via metabolites, but how the metabolites impact CRC is largely unknown. We interrogated fecal metabolites associated with mouse models of colon tumorigenesis with varying mutational load. We find that microbial metabolites from healthy mice or humans are growth-repressive, and this response is attenuated in mice and patients with CRC. Microbial profiling reveals that Lactobacillus reuteri and its metabolite, reuterin, are downregulated in mouse and human CRC. Reuterin alters redox balance, and reduces proliferation and survival in colon cancer cells. Reuterin induces selective protein oxidation and inhibits ribosomal biogenesis and protein translation. Exogenous Lactobacillus reuteri restricts colon tumor growth, increases tumor reactive oxygen species, and decreases protein translation in vivo. Our findings indicate that a healthy microbiome and specifically, Lactobacillus reuteri, is protective against CRC through microbial metabolite exchange.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1535610821006139)

10.1016/j.ccell.2021.12.001

---

## Population study of the gut microbiome: associations with diet, lifestyle, and cardiometabolic disease.
 22.12.2021, _RL Walker, H Vlamakis, JWJ Lee, LA Besse, V Xanthakis, RS Vasan, SY Shaw, RJ Xavier_


The human gut harbors trillions of microbes that play dynamic roles in health. While the microbiome contributes to many cardiometabolic traits by modulating host inflammation and metabolism, there is an incomplete understanding regarding the extent that and mechanisms by which individual microbes impact risk and development of cardiovascular disease (CVD). The Framingham Heart Study (FHS) is a multi-generational observational study following participants over decades to identify risk factors for CVD by correlating genetic and phenotypic factors with clinical outcomes. As a large-scale population-based cohort with extensive clinical phenotyping, FHS provides a rich landscape to explore the relationships between the gut microbiome and cardiometabolic traits.

[Source](https://link.springer.com/article/10.1186/s13073-021-01007-5)

10.1186/s13073-021-01007-5

---

## Enteric virome negatively affects seroconversion following oral rotavirus vaccination in a longitudinally sampled cohort of Ghanaian infants

[Source](https://www.sciencedirect.com/science/article/pii/S1931312821005679)

---

## Maternal gut bacteria drive intestinal inflammation in offspring with neurodevelopmental disorders by altering the chromatin landscape of CD4+ T cells.
 08.12.2021, _E Kim, D Paik, RN Ramirez, DG Biggs, Y Park, HK Kwon, GB Choi, JR Huh_


Children with autism spectrum disorders often display dysregulated immune responses and related gastrointestinal symptoms. However, the underlying mechanisms leading to the development of both phenotypes have not been elucidated. Here, we show that mouse offspring exhibiting autism-like phenotypes due to prenatal exposure to maternal inflammation were more susceptible to developing intestinal inflammation following challenges later in life. In contrast to its prenatal role in neurodevelopmental phenotypes, interleukin-17A (IL-17A) generated immune-primed phenotypes in offspring through changes in the maternal gut microbiota that led to postnatal alterations in the chromatin landscape of naive CD4+ T cells. The transfer of stool samples from pregnant mice with enhanced IL-17A responses into germ-free dams produced immune-primed phenotypes in offspring. Our study provides mechanistic insights into why children exposed to heightened inflammation in the womb might have an increased risk of developing inflammatory diseases in addition to neurodevelopmental disorders.

[Source](https://www.cell.com/immunity/fulltext/S1074-7613(21)00496-9)

10.1016/j.immuni.2021.11.005

---

## Lactobacillus gallinarum modulates the gut microbiota and produces anti-cancer metabolites to protect against colorectal tumourigenesis.
 23.12.2021, _N Sugimura, Q Li, ESH Chu, HCH Lau, W Fong, W Liu, C Liang, G Nakatsu, ACY Su, OO Coker, WKK Wu, FKL Chan, J Yu_


Using faecal shotgun metagenomic sequencing, we identified the depletion of Lactobacillus gallinarum in patients with colorectal cancer (CRC). We aimed to determine the potential antitumourigenic role of L. gallinarum in colorectal tumourigenesis.

[Source](https://gut.bmj.com/content/early/2021/12/21/gutjnl-2020-323951)

10.1136/gutjnl-2020-323951

---

## A catalogue of 1,167 genomes from the human gut archaeome.
 05.01.2022, _CM Chibani, A Mahnert, G Borrel, A Almeida, A Werner, JF Brugère, S Gribaldo, RD Finn, RA Schmitz, C Moissl-Eichinger_


The human gut microbiome plays an important role in health, but its archaeal diversity remains largely unexplored. In the present study, we report the analysis of 1,167 nonredundant archaeal genomes (608 high-quality genomes) recovered from human gastrointestinal tract, sampled across 24 countries and rural and urban populations. We identified previously undescribed taxa including 3 genera, 15 species and 52 strains. Based on distinct genomic features, we justify the split of the Methanobrevibacter smithii clade into two separate species, with one represented by the previously undescribed 'Candidatus Methanobrevibacter intestini'. Patterns derived from 28,581 protein clusters showed significant associations with sociodemographic characteristics such as age groups and lifestyle. We additionally show that archaea are characterized by specific genomic and functional adaptations to the host and carry a complex virome. Our work expands our current understanding of the human archaeome and provides a large genome catalogue for future analyses to decipher its impact on human physiology.

[Source](https://www.nature.com/articles/s41564-021-01020-9)

10.1038/s41564-021-01020-9

---

## Variation and transmission of the human gut microbiota across multiple familial generations.
 05.01.2022, _M Valles-Colomer, R Bacigalupe, S Vieira-Silva, S Suzuki, Y Darzi, RY Tito, T Yamada, N Segata, J Raes, G Falony_


Although the composition and functional potential of the human gut microbiota evolve over the lifespan, kinship has been identified as a key covariate of microbial community diversification. However, to date, sharing of microbiota features within families has mostly been assessed between parents and their direct offspring. Here we investigate the potential transmission and persistence of familial microbiome patterns and microbial genotypes in a family cohort (n = 102) spanning 3 to 5 generations over the same female bloodline. We observe microbiome community composition associated with kinship, with seven low abundant genera displaying familial distribution patterns. While kinship and current cohabitation emerge as closely entangled variables, our explorative analyses of microbial genotype distribution and transmission estimates point at the latter as a key covariate of strain dissemination. Highest potential transmission rates are estimated between sisters and mother-daughter pairs, decreasing with increasing daughter's age and being higher among cohabiting pairs than those living apart. Although rare, we detect potential transmission events spanning three and four generations, primarily involving species of the genera Alistipes and Bacteroides. Overall, while our analyses confirm the existence of family-bound microbiome community profiles, transmission or co-acquisition of bacterial strains appears to be strongly linked to cohabitation.

[Source](https://www.nature.com/articles/s41564-021-01021-8)

10.1038/s41564-021-01021-8

---

## The Mouse Gastrointestinal Bacteria Catalogue enables translation between the mouse and human gut microbiotas via functional mapping.
 31.12.2021, _BS Beresford-Jones, SC Forster, MD Stares, G Notley, E Viciani, HP Browne, DJ Boehmler, AT Soderholm, N Kumar, K Vervier, JR Cross, A Almeida, TD Lawley, VA Pedicord_


Human health and disease have increasingly been shown to be impacted by the gut microbiota, and mouse models are essential for investigating these effects. However, the compositions of human and mouse gut microbiotas are distinct, limiting translation of microbiota research between these hosts. To address this, we constructed the Mouse Gastrointestinal Bacteria Catalogue (MGBC), a repository of 26,640 high-quality mouse microbiota-derived bacterial genomes. This catalog enables species-level analyses for mapping functions of interest and identifying functionally equivalent taxa between the microbiotas of humans and mice. We have complemented this with a publicly deposited collection of 223 bacterial isolates, including 62 previously uncultured species, to facilitate experimental investigation of individual commensal bacteria functions in vitro and in vivo. Together, these resources provide the ability to identify and test functionally equivalent members of the host-specific gut microbiotas of humans and mice and support the informed use of mouse models in human microbiota research.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(21)00568-0#secsectitle0040)

10.1016/j.chom.2021.12.003

---

## Integrating cultivation and metagenomics for a multi-kingdom view of skin microbiome diversity and functions

[Source](https://www.nature.com/articles/s41564-021-01011-w)

---

## Alterations of human skin microbiome and expansion of antimicrobial resistance after systemic antibiotics

[Source](https://www.science.org/doi/10.1126/scitranslmed.abd8077)

---

## Microbiome-mediated incapacitation of interferon lambda production in the oral mucosa

[Source](https://www.pnas.org/content/118/51/e2105170118.short)

---

## Glutamic acid reshapes the plant microbiota to protect plants against pathogens 

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01186-8)

---

## Metagenomics of Antarctic Marine Sediment Reveals Potential for Diverse Chemolithoautotrophy

[Source](https://journals.asm.org/doi/10.1128/mSphere.00770-21)

---

## Dissecting the dominant hot spring microbial populations based on community-wide sampling at single-cell genomic resolutio

[Source](https://www.nature.com/articles/s41396-021-01178-4)

---

## Fast and accurate metagenotyping of the human gut microbiome with GT-Pro

[Source](https://www.nature.com/articles/s41587-021-01102-3)

---

## Rapid screening and identification of viral pathogens in metagenomic data

[Source](https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-021-01138-z)

---

