## Circulating microbiome DNA: An emerging paradigm for cancer liquid biopsy.
 30.08.2021, _H Chen, Y Ma, Z Liu, J Li, X Li, F Yang, M Qiu_


Dysbiosis of the human microbiome has long been reported to be closely associated with various cancers. Accumulating studies have shown that microbial dysbiosis can accelerate tumorigenesis through tumor-promoting inflammation, DNA damage, and inducing immune evasion. Differential composition of microbiome could be novel biomarkers for cancer detection or biomarkers of successful immunotherapy. More importantly, emerging evidence demonstrates that alterations of circulating microbiome DNA (cmDNA) could serve as promising noninvasive biomarkers for cancer detection. It has been reported that distinct circulating bacterial DNA could distinguish prostate cancer, lung cancer, and melanoma patients from healthy populations. Therefore, in this review, we summarized current literature on microbial biomarkers for cancer detection and unraveled the potential of cmDNA as a promising cancer detection tool.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0304383521004328)

10.1016/j.canlet.2021.08.036

---

## Human-associated microbiota suppress invading bacteria even under disruption by antibiotics.
 30.08.2021, _AD Letten, M Baumgartner, KR Pfrunder-Cardozo, JM Levine, AR Hall_


In light of their adverse impacts on resident microbial communities, it is widely predicted that broad-spectrum antibiotics can promote the spread of resistance by releasing resistant strains from competition with other strains and species. We investigated the competitive suppression of a resistant strain of Escherichia coli inoculated into human-associated communities in the presence and absence of the broad and narrow spectrum antibiotics rifampicin and polymyxin B, respectively. We found strong evidence of community-level suppression of the resistant strain in the absence of antibiotics and, despite large changes in community composition and abundance following rifampicin exposure, suppression of the invading resistant strain was maintained in both antibiotic treatments. Instead, the strength of competitive suppression was more strongly associated with the source community (stool sample from individual human donor). This suggests microbiome composition strongly influences the competitive suppression of antibiotic-resistant strains, but at least some antibiotic-associated disruption can be tolerated before competitive release is observed. A deeper understanding of this association will aid the development of ecologically-aware strategies for managing antibiotic resistance.

[Source](https://www.nature.com/articles/s41396-021-00929-7)

10.1038/s41396-021-00929-7

---

## Crosstalk between γδ T cells and the microbiota.
 26.08.2021, _PH Papotto, B Yilmaz, B Silva-Santos_


The role of the microbiota in the development and function of γδ T cells-a T cell subset characterized by a T cell receptor composed of one γ-chain and one δ-chain-has been investigated in multiple organs in mice and humans. Interactions between the microbiota and γδ T cells affect both tissue homeostasis and disease pathologies. Notably, microbiota-induced interleukin-17 (IL-17)-producing-γδ T cells can mediate a range of immunological processes, from metabolic disorders to neuroinflammation via the gut-brain axis. However, the bidirectional interactions between γδ T cells and the microbiota have not been fully determined. In this Perspective, we dissect the roles of microbiota in modulating γδ T cell development and function, and evaluate the evidence for γδ T cell selection of commensal communities. We also discuss the potential implications of these cells in health and disease and the major open questions and research avenues in the field.

[Source](https://www.nature.com/articles/s41564-021-00948-2)

10.1038/s41564-021-00948-2

---

## Key bacterial taxa and metabolic pathways affecting gut short-chain fatty acid profiles in early life.
 30.08.2021, _N Tsukuda, K Yahagi, T Hara, Y Watanabe, H Matsumoto, H Mori, K Higashi, H Tsuji, S Matsumoto, K Kurokawa, T Matsuki_


Infant gut microbiota development affects the host physiology throughout life, and short-chain fatty acids (SCFAs) are promising key metabolites mediating microbiota-host relationships. Here, we investigated dense longitudinally collected faecal samples from 12 subjects during the first 2 years (n = 1048) to identify early life gut SCFA patterns and their relationships with the microbiota. Our results revealed three distinct phases of progression in the SCFA profiles: early phase characterised by low acetate and high succinate, middle-phase characterised by high lactate and formate and late-phase characterised by high propionate and butyrate. Assessment of the SCFA-microbiota relationships revealed that faecal butyrate is associated with increased Clostridiales and breastfeeding cessation, and that diverse and personalised assemblage of Clostridiales species possessing the acetyl-CoA pathway play major roles in gut butyrate production. We also found an association between gut formate and some infant-type bifidobacterial species, and that human milk oligosaccharides (HMO)-derived fucose is the substrate for formate production during breastfeeding. We identified genes upregulated in fucose and fucosylated HMO utilisation in infant-type bifidobacteria. Notably, bifidobacteria showed interspecific and intraspecific variation in the gene repertoires, and cross-feeding of fucose contributed to gut formate production. This study provides an insight into early life SCFA-microbiota relationships, which is an important step for developing strategies for modulating lifelong health.

[Source](https://www.nature.com/articles/s41396-021-00937-7)

10.1038/s41396-021-00937-7

---

## Deterministic transition of enterotypes shapes the infant gut microbiome at an early age.
 28.08.2021, _L Xiao, J Wang, J Zheng, X Li, F Zhao_


The succession of the gut microbiota during the first few years plays a vital role in human development. We elucidate the characteristics and alternations of the infant gut microbiota to better understand the correlation between infant health and microbiota maturation.

[Source](https://link.springer.com/article/10.1186/s13059-021-02463-3)

10.1186/s13059-021-02463-3

---

## Association between metabolic syndrome and periodontitis: The role of lipids, inflammatory cytokines, altered host response, and the microbiome

[Source](https://onlinelibrary.wiley.com/doi/10.1111/prd.12379)

---

## Fungal diseases: Oral dysbiosis in susceptible hosts

[Source](https://onlinelibrary.wiley.com/doi/10.1111/prd.12378)

---

## The oral microbiome: Role of key organisms and complex networks in oral health and disease

[Source](https://onlinelibrary.wiley.com/doi/10.1111/prd.12393)

---

## Human reference gut microbiome catalog including newly assembled genomes from under-represented Asian metagenomes

[Source](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-021-00950-7)

---

## The promise of the gut microbiome as part of individualized treatment strategies

[Source](https://www.nature.com/articles/s41575-021-00499-1)

---

## The gut microbiome and efficacy of cancer immunotherapy

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0163725821001753)

---

## Integrated analysis of the faecal metagenome and serum metabolome reveals the role of gut microbiome-associated metabolites in the detection of colorectal cancer and adenoma

[Source](https://gut.bmj.com/content/early/2021/08/29/gutjnl-2020-323476)

---

## Longitudinal study on the effects of growth-promoting and therapeutic antibiotics on the dynamics of chicken cloacal and litter microbiomes and resistomes

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01136-4)

---

## Social networks strongly predict the gut microbiota of wild mice

[Source](https://www.nature.com/articles/s41396-021-00949-3)

---

## Genetic identity and genotype × genotype interactions between symbionts outweigh species level effects in an insect microbiome

[Source](https://www.nature.com/articles/s41396-021-00943-9)

---

## The effects of raw-meat diets on the gastrointestinal microbiota of the cat and dog: a review

[Source](https://www.tandfonline.com/doi/abs/10.1080/00480169.2021.1975586?journalCode=tnzv20)

---

## Predicting spatial patterns of soil bacteria under current and future environmental conditions

[Source](https://www.nature.com/articles/s41396-021-00947-5)

---

## Persistent microbiome members in the common bean rhizosphere: an integrated analysis of space, time, and plant genotype

[Source](https://www.nature.com/articles/s41396-021-00955-5)

---

## Host preference and invasiveness of commensal bacteria in the Lotus and Arabidopsis root microbiota.
 26.08.2021, _K Wippel, K Tao, Y Niu, R Zgadzaj, N Kiel, R Guan, E Dahms, P Zhang, DB Jensen, E Logemann, S Radutoiu, P Schulze-Lefert, R Garrido-Oter_


Roots of different plant species are colonized by bacterial communities, that are distinct even when hosts share the same habitat. It remains unclear to what extent the host actively selects these communities and whether commensals are adapted to a specific plant species. To address this question, we assembled a sequence-indexed bacterial culture collection from roots and nodules of Lotus japonicus that contains representatives of most species previously identified using metagenomics. We analysed taxonomically paired synthetic communities from L. japonicus and Arabidopsis thaliana in a multi-species gnotobiotic system and detected signatures of host preference among commensal bacteria in a community context, but not in mono-associations. Sequential inoculation experiments revealed priority effects during root microbiota assembly, where established communities are resilient to invasion by latecomers, and that host preference of commensal bacteria confers a competitive advantage in their cognate host. Our findings show that host preference in commensal bacteria from diverse taxonomic groups is associated with their invasiveness into standing root-associated communities.

[Source](https://www.nature.com/articles/s41564-021-00941-9)

10.1038/s41564-021-00941-9

---

## Functional meta-omics provide critical insights into long- and short-read assemblies.
 28.08.2021, _V Galata, SB Busi, BJ Kunath, L de Nies, M Calusinska, R Halder, P May, P Wilmes, CC Laczny_


Real-world evaluations of metagenomic reconstructions are challenged by distinguishing reconstruction artifacts from genes and proteins present in situ. Here, we evaluate short-read-only, long-read-only and hybrid assembly approaches on four different metagenomic samples of varying complexity. We demonstrate how different assembly approaches affect gene and protein inference, which is particularly relevant for downstream functional analyses. For a human gut microbiome sample, we use complementary metatranscriptomic and metaproteomic data to assess the metagenomic data-based protein predictions. Our findings pave the way for critical assessments of metagenomic reconstructions. We propose a reference-independent solution, which exploits the synergistic effects of multi-omic data integration for the in situ study of microbiomes using long-read sequencing data.

[Source](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbab330/6358409)

10.1093/bib/bbab330

---

## BiomeHorizon: visualizing microbiome time series data in R

[Source](https://www.biorxiv.org/content/10.1101/2021.08.29.458140v1.abstract)

---

## Evaluation of DNA extraction protocols from liquid-based cytology specimens for studying cervical microbiota.
 30.08.2021, _T Shibata, M Nakagawa, HN Coleman, SM Owens, WW Greenfield, T Sasagawa, MS Robeson_


Cervical microbiota (CM) are considered an important factor affecting the progression of cervical intraepithelial neoplasia (CIN) and are implicated in the persistence of human papillomavirus (HPV). Collection of liquid-based cytology (LBC) samples is routine for cervical cancer screening and HPV genotyping and can be used for long-term cytological biobanking. We sought to determine whether it is possible to access microbial DNA from LBC specimens, and compared the performance of four different extraction protocols: (ZymoBIOMICS DNA Miniprep Kit; QIAamp PowerFecal Pro DNA Kit; QIAamp DNA Mini Kit; and IndiSpin Pathogen Kit) and their ability to capture the diversity of CM from LBC specimens. LBC specimens from 20 patients (stored for 716 ± 105 days) with CIN values of 2 or 3 were each aliquoted for each of the four kits. Loss of microbial diversity due to long-term LBC storage could not be assessed due to lack of fresh LBC samples. Comparisons with other types of cervical sampling were not performed. We observed that all DNA extraction kits provided equivalent accessibility to the cervical microbial DNA within stored LBC samples. Approximately 80% microbial genera were shared among all DNA extraction protocols. Potential kit contaminants were observed as well. Variation between individuals was a significantly greater influence on the observed microbial composition than was the method of DNA extraction. We also observed that HPV16 was significantly associated with community types that were not dominated by Lactobacillus iners.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237556)

10.1371/journal.pone.0237556

---

## MicFunPred: A conserved approach to predict functional profiles from 16S rRNA gene sequence data.
 27.08.2021, _DS Mongad, NS Chavan, NP Narwade, K Dixit, YS Shouche, DP Dhotre_


The 16S rRNA gene amplicon sequencing is a popular technique that provides accurate characterization of microbial taxonomic abundances but does not provide any functional information. Several tools are available to predict functional profiles based on 16S rRNA gene sequence data that use different genome databases and approaches. As variable regions of partially-sequenced 16S rRNA gene cannot resolve taxonomy accurately beyond the genus level, these tools may give inflated results. Here, we developed 'MicFunPred', which uses a novel approach to derive imputed metagenomes based on a set of core genes only, thereby minimizing false-positive predictions. On simulated datasets, MicFunPred showed the lowest False Positive Rate (FPR) with mean Spearman's correlation of 0.89 (SD = 0.03), while on seven real datasets the mean correlation was 0.75 (SD = 0.08). MicFunPred was found to be faster with low computational requirements and performed better or comparable when compared with other tools.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0888754321003293)

10.1016/j.ygeno.2021.08.016

---

