## Commentary on: LaGoy A, Evangelou EA, Somogyi T, DiGangi EA. Recommended practices for macerating human thyroid cartilage. J Forensic Sci 2020;65(4):1266-73. https://doi.org/10.1111/1556-4029.14289.
 06.10.2020, _LC Fulginiti_



[Source](https://www.biocodexmicrobiotafoundation.com/national-call-projects/usa)

10.1111/1556-4029.14591

---

## https://www.biocodexmicrobiotafoundation.com/national-call-projects/usa

[Source](https://www.biocodexmicrobiotafoundation.com/national-call-projects/usa)

---

## Understanding the potential of lactobacilli in recurrent UTI prevention.
 03.10.2020, _S Vagios, H Hesham, C Mitchell_


Urinary tract infections (UTIs) are one of the most common infections in women. The only proven preventive strategy for recurrent UTIs is prophylactic antibiotics. Given growing antibiotic resistance, the use of probiotics has been proposed as an alternative to antibiotics. Herein, we discuss the current evidence to support the possibility that exogenous lactobacilli may limit the pathogenicity of uropathogens such as E. coli. Probiotics appear to have a significant potential in prevention of recurrent UTI, however, additional data are needed to understand how they can be effectively used in clinical practice.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0882401020309104)

10.1016/j.micpath.2020.104544

---

## Throat microbial community structure and functional changes in post-surgery laryngeal carcinoma patients.
 03.10.2020, _CY Hsueh, H Gong, N Cong, J Sun, HC Lau, Y Guo, Q Huang, X Yuan, M Zhang, L Tao, L Zhou_


Microbial community structure and its shift after laryngectomy in laryngeal squamous cell carcinoma (LSCC) patients were investigated. Thirty swab samples of LSCC prior to laryngectomy (SLC), 18 samples after 1-weeks laryngectomy (SLCA1w), and 30 samples after 24-week laryngectomy (SLCA24w) from 30 LSCC patients were sourced. Microbial diversity was profiled through sequencing the V3-4 variable region of the 16S ribosomal ribonucleic acid (rRNA) gene. Quantitative real-time polymerase chain reaction (qPCR) was used to validate the V3-4 region of 16S rRNA sequence data. The community structure and function of throat microbiota were assessed by PICRUSt analysis. Both Alpha and Beta diversity results showed significant differences in the throat microbiota of LSCC patients before and after laryngectomy (P &lt; 0.05). Drinking index of SLC group was positively associated with the genus abundance of Prevotella (P &lt; 0.05). SLCA1w group had lower abundance of Fusobacterium, Leptotrichia, Lachnoanaerobaculum and Veillonella compared to the SLC group (P &lt; 0.05). SLCA24w group had higher abundance of Streptococcus and Leptotrichia, as well as the lower abundance of Fusobacterium and Alloprevotella compared to the SLC group (P &lt; 0.05). The different propensity of throat microbiome of SLC group could implicate human cancer signaling pathways evident through Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt) analysis (P &lt; 0.05). Our study clarifies altered throat microbial community structure and function in LSCC patients during the perioperative period and post-operative recovery period.ImportanceLaryngeal squamous cell carcinoma greatly impacts patient's livelihood, and non-invasive ways for prognostic assessment is valuable in determining the effectiveness of laryngectomy.We set to study the microbial structure changes in throat before and after laryngectomy and found the gene functions of several throat bacteria to be associated with human cancer signaling pathways.Our findings may offer insights into the disease management of patients with laryngeal squamous cell carcinoma. We hope to provide a perspective in molecular mechanisms to improve the prognosis of laryngeal cancer treatment and to facilitate relevant research.

[Source](https://aem.asm.org/content/early/2020/09/28/AEM.01849-20.abstract)

10.1128/AEM.01849-20

---

## Bacterial diversity and prevalence of antibiotic resistance genes in the oral microbiome.
 02.10.2020, _VSM Almeida, J Azevedo, HF Leal, ATL Queiroz, HP da Silva Filho, JN Reis_


This study aims to describe the oral microbiome diversity and prevalence of ARGs in periodontal health and disease.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0239664)

10.1371/journal.pone.0239664

---

## Metagenomics of the modern and historical human oral microbiome with phylogenetic studies on Streptococcus mutans and Streptococcus sobrinus.
 05.10.2020, _M Achtman, Z Zhou_


We have recently developed bioinformatic tools to accurately assign metagenomic sequence reads to microbial taxa: SPARSE for probabilistic, taxonomic classification of sequence reads; EToKi for assembling and polishing genomes from short-read sequences; and GrapeTree, a graphic visualizer of genetic distances between large numbers of genomes. Together, these methods support comparative analyses of genomes from ancient skeletons and modern humans. Here, we illustrate these capabilities with 784 samples from historical dental calculus, modern saliva and modern dental plaque. The analyses revealed 1591 microbial species within the oral microbiome. We anticipated that the oral complexes of Socransky et al., which were defined in 1998, would predominate among taxa whose frequencies differed by source. However, although some species discriminated between sources, we could not confirm the existence of the complexes. The results also illustrate further functionality of our pipelines with two species that are associated with dental caries, Streptococcus mutans and Streptococcus sobrinus. They were rare in historical dental calculus but common in modern plaque, and even more common in saliva. Reconstructed draft genomes of these two species from metagenomic samples in which they were abundant were combined with modern public genomes to provide a detailed overview of their core genomic diversity. This article is part of the theme issue 'Insights into health and disease from ancient biomolecules'.

[Source](https://royalsocietypublishing.org/doi/10.1098/rstb.2019.0573)

10.1098/rstb.2019.0573

---

## In utero human intestine harbors unique metabolomic features including bacterial metabolites.
 01.10.2020, _Y Li, JM Toothaker, S Ben-Simon, L Ozeri, R Schweitzer, BT McCourt, CC McCourt, L Werner, SB Snapper, DS Shouval, S Khatib, O Koren, S Agnihorti, G Tseng, L Konnikova_


Symbiotic microbial colonization through the establishment of the intestinal microbiome is critical to many intestinal functions including nutrient metabolism, intestinal barrier integrity and immune regulation. Recent studies suggest that education of the intestinal immunity maybe ongoing in utero. However, the drivers of this process are unknown. The microbiome and its byproducts are one potential source. Whether a fetal intestinal microbiome exists is controversial and if microbially derived metabolites are present in utero is unknown. Here, we aimed to determine whether bacterial DNA and microbially-derived metabolites can be detected in second trimester human intestinal samples. Although, we were unable to amplify bacterial DNA from fetal intestines, we report a unique fetal metabolomic intestinal profile with an abundance of bacterially derived and host derived metabolites commonly produced in response to microbiota. Though we did not directly assess their source and function, we hypothesize that these microbial associated metabolites come either from the maternal microbiome and are vertically transmitted to the fetus to prime the fetal immune system and prepare the gastrointestinal tract for postnatal microbial encounters or are produced locally by bacteria that was below our detection threshold.

[Source](https://insight.jci.org/articles/view/138751)

10.1172/jci.insight.138751

---

## nteractions between Environmental Exposures and the Microbiome: Implications for Fetal Programming

[Source](https://www.sciencedirect.com/science/article/pii/S2451965020300946)

---

## Alterations in blood microbiota after colonic cancer surgery.
 06.10.2020, _JH Søby, SK Watt, RP Vogelsang, F Servant, B Lelouvier, H Raskov, FK Knop, I Gögenur_


Mechanisms contributing to the perioperative stress response remain poorly understood. This study investigated changes in the amount of bacterial DNA in blood and the diversity of blood microbiota in the perioperative period in patients undergoing minimally invasive surgery for colonic cancer in an enhanced recovery after surgery setting.

[Source](https://bjssjournals.onlinelibrary.wiley.com/doi/10.1002/bjs5.50357)

10.1002/bjs5.50357

---

## Estimating molecular preservation of the intestinal microbiome via metagenomic analyses of latrine sediments from two medieval cities.
 05.10.2020, _S Sabin, HY Yeh, A Pluskowski, C Clamer, PD Mitchell, KI Bos_


Ancient latrine sediments, which contain the concentrated collective biological waste of past whole human communities, have the potential to be excellent proxies for human gastrointestinal health on the population level. A rich body of literature explores their use to detect the presence of gut-associated eukaryotic parasites through microscopy, immunoassays and genetics. Despite this interest, a lack of studies have explored the whole genetic content of ancient latrine sediments through consideration not only of gut-associated parasites, but also of core community gut microbiome signals that remain from the group that used the latrine. Here, we present a metagenomic analysis of bulk sediment from medieval latrines in Riga (Latvia) and Jerusalem. Our analyses reveal survival of microbial DNA representative of intestinal flora as well as numerous parasites. These data are compared against parasite taxon identifications obtained via microscopy and ELISA techniques. Together, these findings provide a first glimpse into the rich prokaryotic and eukaryotic intestinal flora of pre-industrial agricultural populations, which may give a better context for interpreting the health of modern microbiomes. This article is part of the theme issue 'Insights into health and disease from ancient biomolecules'.

[Source](https://royalsocietypublishing.org/doi/10.1098/rstb.2019.0576)

10.1098/rstb.2019.0576

---

## Gut microbiota composition in obese and non-obese adult relatives from the highlands of Papua New Guinea.
 06.10.2020, _ME Jonduo, L Wawae, G Masiria, W Suda, M Hattori, L Takayasu, MY Abdad, AR Greenhill, PF Horwood, W Pomat, M Umezaki_


Obesity is a condition that results from an imbalance between energy intake and expenditure. Recently, obesity has been linked to differences in the composition of gut microbiota. To examine this association in Papua New Guinea (PNG) highlanders, fecal samples were collected from 18 adults; nine obese participants paired with their non-obese relative. Amplification of the 16S rRNA gene targeting the V1-V2 region was performed on DNA extracts for each participant, with high-quality sequences selected and used for OTU clustering. The data showed Firmicutes and Bacteroidetes were the two dominant phyla, while at genus level Prevotella was the most dominant genus in all of the samples. Nonetheless, statistical evaluation of potential association between nutritional status and bacterial abundance at phyla and genus levels both showed no significant difference. Further studies, ideally in both rural and urban areas, are needed to evaluate the role of the gut microbiome in the occurrence of obesity in PNG and other resource-limited settings.

[Source](https://academic.oup.com/femsle/advance-article/doi/10.1093/femsle/fnaa161/5918384)

10.1093/femsle/fnaa161

---

## Fecal Bacteria as Biomarkers for Predicting Food Intake in Healthy Adults.
 06.10.2020, _LM Shinn, Y Li, A Mansharamani, LS Auvil, ME Welge, C Bushell, NA Khan, CS Charron, JA Novotny, DJ Baer, R Zhu, HD Holscher_


Diet affects the human gastrointestinal microbiota. Blood and urine samples have been used to determine nutritional biomarkers. However, there is a dearth of knowledge on the utility of fecal biomarkers, including microbes, as biomarkers of food intake.

[Source](https://academic.oup.com/jn/advance-article/doi/10.1093/jn/nxaa285/5918305)

10.1093/jn/nxaa285

---

## Inulin fermentation by lactobacilli and bifidobacteria from dairy calves.
 03.10.2020, _Y Zhu, J Liu, JM Lopez, DA Mills_


Prebiotics are increasingly examined for their ability to modulate the neonate gut microbiota of livestock and products such as inulin are commonly added to milk replacer used in calving. However, the ability of specific members of the bovine neonate microbiota to respond to inulin remains to be determined, particularly among indigenous lactobacilli and bifidobacteria, beneficial genera commonly enriched by inulin. Screening of Bifidobacterium and Lactobacillus isolates obtained from fresh feces of dairy calves revealed that lactobacilli had a higher prevalence of inulin fermentation capacity (58%) than bifidobacteria (17%). Several Ligilactobacillus agilis comb. nov. (formerly Lactobacillus agilis) isolates exhibited vigorous growth on, and complete degradation of, inulin however the phenotype was strain-specific. The most vigorous inulin fermenting strain, L. agilis YZ050, readily degraded long chain inulin not consumed by bifidobacterial isolates. Comparative genomic analysis of both L. agilis fermenter and non-fermenter strains indicated that strain YZ050 encodes an inulinase homolog, previously linked to extracellular degradation of long-chain inulin in Lacticaseibacillus paracasei comb. nov., that was strongly induced during growth on inulin. Inulin catabolism by YZ050 also generates extracellular fructose which can serve as a growth substrate to cross feed other non-inulin-fermenting lactic acid bacteria isolated from the same bovine feces. The presence of specific inulin-responsive bacterial strains within calf gut microbiome provides a mechanistic rationale for enrichment of specific lactobacilli and creates a foundation for future synbiotic applications in dairy calves aimed at improving health in early life.IMPORTANCE The gut microbiome plays an important role in animal health and is increasingly recognized as a target for diet-based manipulation. Inulin is a common prebiotic routinely added to animal feeds, however, the mechanism of inulin consumption by specific beneficial taxa in livestock is ill defined. In this study we examined Lactobacillus and Bifidobacterium isolates from calves fed inulin-containing milk replacer and characterized specific strains that robustly consume long chain inulin. In particular, a novel Ligilactobacillus agilis strain YZ050 consumed inulin via an extracellular fructosidase resulting in complete consumption of all long chain inulin. Inulin catabolism resulted in temporal release of extracellular fructose that can promote growth of other non-inulin consuming LAB strains. This work provides the mechanistic insight needed to purposely modulate the calf gut microbiome via the establishment of networks of beneficial microbes linked to specific prebiotics.

[Source](https://aem.asm.org/content/early/2020/09/28/AEM.01738-20.abstract)

10.1128/AEM.01738-20

---

## Microbial Diversity and Community Structure in Alpine Stream Soil

[Source](https://www.tandfonline.com/doi/abs/10.1080/01490451.2020.1824039)

---

## Impact of dietary fibers in infant formula on gut microbiota and the intestinal immune barrier

[Source](https://pubs.rsc.org/en/content/articlelanding/2020/fo/d0fo01700k/unauth#!divAbstract)

---

## A framework for comparing microbial networks reveals core associations

[Source](https://www.biorxiv.org/content/10.1101/2020.10.05.325860v1.abstract)

---

## Gut health benefits begin in utero

[Source](https://news.yale.edu/2020/10/06/gut-health-benefits-begin-utero)

---

## Medieval poo holds clues to health of modern-day gut microbiome

[Source](https://www.sciencefocus.com/news/medieval-poo-holds-clues-to-health-of-modern-day-gut-microbiome/)

---

