## Searching for New Microbiome-Targeted Therapeutics through a Drug Repurposing Approach.
 30.11.2021, _M Barone, S Rampelli, E Biagi, SM Bertozzi, F Falchi, A Cavalli, A Armirotti, P Brigidi, S Turroni, M Candela_


Commonly used non-antibiotic drugs have been associated with changes in gut microbiome composition, paving the way for the possibility of repurposing FDA-approved molecules as next-generation microbiome therapeutics. Herein, we developed and validated an ex vivo high-throughput screening platform─the mini gut model─to underpin human gut microbiome response to molecular modulators. Ten FDA-approved compounds, selected based on maximum structural diversity of molecular fingerprints, were screened against the gut microbiome of five healthy subjects to characterize the ability of human-targeted drugs to modulate the human gut microbiome network. Three compounds, THIP hydrochloride, methenamine, and mesna, have shown promise as novel gut microbiome therapeutics in light of their capability of promoting health-associated features of the gut microbiome. Our findings provide a resource for future research on drug-microbiome interactions and lay the foundation for a new era of more precise gut microbiome modulation through drug repurposing, aimed at targeting specific dysbiotic events.

[Source](https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c01333)

10.1021/acs.jmedchem.1c01333

---

## Current status of intratumour microbiome in cancer and engineered exogenous microbiota as a promising therapeutic strategy.
 30.11.2021, _J Chen, T Li, J Liang, Q Huang, JD Huang, Y Ke, H Sun_


Research on the relationship between microbiome and cancer has made significant progress in the past few decades. It is now known that the gut microbiome has multiple effects on tumour biology. However, the relationship between intratumoral bacteria and cancers remains unclear. Growing evidence suggests that intratumoral bacteria are important components of the microenvironment in several types of cancers. Furthermore, several studies have demonstrated that intratumoral bacteria may directly influence tumorigenesis, progression and responses to treatment. Limited studies have been conducted on intratumoral bacteria, and using intratumoral bacteria to treat tumours remains a challenge. Bacteria have been studied as anticancer therapeutics since the 19th century when William B. Coley successfully treated patients with inoperable sarcomas using Streptococcus pyogenes. With the development of synthetic biological approaches, several bacterial species have been genetically engineered to increase their applicability for cancer treatment. Genetically engineered bacteria for cancer therapy have unique properties compared to other treatment methods. They can specifically accumulate within tumours and inhibit cancer growth. In addition, genetically engineered bacteria may be used as a vector to deliver antitumour agents or combined with radiation and chemotherapy to synergise the effectiveness of cancer treatment. However, various problems in treating tumours with genetically engineered bacteria need to be addressed. In this review, we focus on the role of intratumoral bacteria on tumour initiation, progression and responses to chemotherapy or immunotherapy. Moreover, we summarised the recent progress in the treatment of tumours with genetically engineered bacteria.

[Source](https://www.sciencedirect.com/science/article/pii/S0753332221012294)

10.1016/j.biopha.2021.112443

---

## Regulation of blood-brain barrier integrity by microbiome-associated methylamines and cognition by trimethylamine N-oxide.
 29.11.2021, _L Hoyles, MG Pontifex, I Rodriguez-Ramiro, MA Anis-Alavi, KS Jelane, T Snelling, E Solito, S Fonseca, AL Carvalho, SR Carding, M Müller, RC Glen, D Vauzour, S McArthur_


Communication between the gut microbiota and the brain is primarily mediated via soluble microbe-derived metabolites, but the details of this pathway remain poorly defined. Methylamines produced by microbial metabolism of dietary choline and L-carnitine have received attention due to their proposed association with vascular disease, but their effects upon the cerebrovascular circulation have hitherto not been studied.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01181-z)

10.1186/s40168-021-01181-z

---

## Modeling host-associating microbes under selection.
 30.11.2021, _F Bansept, N Obeng, H Schulenburg, A Traulsen_


The concept of fitness is often reduced to a single component, such as the replication rate in a given habitat. For species with multi-step life cycles, this can be an unjustified oversimplification, as every step of the life cycle can contribute to the overall reproductive success in a specific way. In particular, this applies to microbes that spend part of their life cycles associated to a host. In this case, there is a selection pressure not only on the replication rates, but also on the phenotypic traits associated to migrating from the external environment to the host and vice-versa (i.e., the migration rates). Here, we investigate a simple model of a microbial lineage living, replicating, migrating and competing in and between two compartments: a host and an environment. We perform a sensitivity analysis on the overall growth rate to determine the selection gradient experienced by the microbial lineage. We focus on the direction of selection at each point of the phenotypic space, defining an optimal way for the microbial lineage to increase its fitness. We show that microbes can adapt to the two-compartment life cycle through either changes in replication or migration rates, depending on the initial values of the traits, the initial distribution across the two compartments, the intensity of competition, and the time scales involved in the life cycle versus the time scale of adaptation (which determines the adequate probing time to measure fitness). Overall, our model provides a conceptual framework to study the selection on microbes experiencing a host-associated life cycle.

[Source](https://www.nature.com/articles/s41396-021-01039-0)

10.1038/s41396-021-01039-0

---

## Bacterial variation in the oral microbiota in multiple sclerosis patients.
 30.11.2021, _Z Zangeneh, A Abdi-Ali, K Khamooshian, A Alvandi, R Abiri_


Microorganisms in oral cavity are called oral microbiota, while microbiome consists of total genome content of microorganisms in a host. Interaction between host and microorganisms is important in nervous system development and nervous diseases such as Autism, Alzheimer, Parkinson and Multiple Sclerosis (MS). Bacterial infections, as an environmental factor in MS pathogenesis play role in T helper 17(Th17) increase and it enhancing the production of pro-inflammatory cytokines such as Interlukin-21(IL-21), IL-17 and IL -22. Oral microbiota consists diverse populations of cultivable and uncultivable bacterial species. Denaturing gradient gel electrophoresis (DGGE) is an acceptable method for identification of uncultivable bacteria. In this study, we compared the bacterial population diversity in the oral cavity between MS and healthy people.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0260384)

10.1371/journal.pone.0260384

---

## Gut microbiota as a transducer of dietary cues to regulate host circadian rhythms and metabolism.
 22.11.2021, _H Choi, MC Rao, EB Chang_


_Biological Clocks, Circadian Rhythm, Diet, Diet, High-Fat, Diet, Vegetarian, Diet, Western, Dietary Fiber, Dietary Sugars, Energy Metabolism, Gastrointestinal Microbiome, Host Microbial Interactions, Humans, Obesity_
Certain members of the gut microbiota exhibit diurnal variations in relative abundance and function to serve as non-canonical drivers of host circadian rhythms and metabolism. Also known as microbial oscillators, these microorganisms entrain upon non-photic cues, primarily dietary, to modulate host metabolism by providing input to both circadian clock-dependent and clock-independent host networks. Microbial oscillators are generally promoted by plant-based, low-fat (lean) diets, and most are abolished by low-fibre, high-sugar, high-fat (Western) diets. The changes in microbial oscillators under different diets then affect host metabolism by altering central and peripheral host circadian clock functions and/or by directly affecting other metabolic targets. Here, we review the unique role of the gut microbiota as a non-photic regulator of host circadian rhythms and metabolism. We describe genetic, environmental, dietary and other host factors such as sex and gut immunity that determine the composition and behaviour of microbial oscillators. The mechanisms by which these oscillators regulate host circadian gene expression and metabolic state are further discussed. Because of the gut microbiota's unique role as a non-photic driver of host metabolism and circadian rhythms, the development and clinical application of novel gut microbiota-related diagnostics and therapeutics hold great promise for achieving and maintaining metabolic health.

[Source](https://www.nature.com/articles/s41575-021-00452-2)

10.1038/s41575-021-00452-2

---

## Gut microbiota-derived short-chain fatty acids and colorectal cancer: Ready for clinical translation?
 29.11.2021, _H Hou, D Chen, K Zhang, W Zhang, T Liu, S Wang, X Dai, B Wang, W Zhong, H Cao_


Colorectal cancer (CRC) is the third most common cancer and the second leading cause of cancer-related death worldwide. It involves the complex interactions between genetic factors, environmental exposure, and gut microbiota. Specific changes in the gut microbiome and metabolome have been described in CRC, supporting the critical role of gut microbiota dysbiosis and microbiota-related metabolites in the tumorigenesis process. Short-chain fatty acids (SCFAs), the principal metabolites generated from the gut microbial fermentation of insoluble dietary fiber, can directly activate G-protein-coupled receptors (GPCRs), inhibit histone deacetylases (HDACs), and serve as energy substrates to connect dietary patterns and gut microbiota, thereby improving the intestinal health. A significantly lower abundance of SCFAs and SCFA-producing bacteria has been demonstrated in CRC, and the supplementation of SCFA-producing probiotics can inhibit intestinal tumor development. SCFAs-guided modulation in both mouse and human CRC models augmented their responses to chemotherapy and immunotherapy. This review briefly summarizes the complex crosstalk between SCFAs and CRC, which might inspire new approaches for the diagnosis, treatment and prevention of CRC on the basis of gut microbiota-derived metabolites SCFAs.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0304383521006005)

10.1016/j.canlet.2021.11.027

---

## Analysis of changes in microbiome compositions related to the prognosis of colorectal cancer patients based on tissue-derived 16S rRNA sequences.
 30.11.2021, _S Choi, J Chung, ML Cho, D Park, SS Choi_


Comparing the microbiome compositions obtained under different physiological conditions has frequently been attempted in recent years to understand the functional influence of microbiomes in the occurrence of various human diseases.

[Source](https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-021-03154-0)

10.1186/s12967-021-03154-0

---

## The Fecal Microbiome and Metabolome of Pitt Hopkins Syndrome, a Severe Autism Spectrum Disorder.
 30.11.2021, _AH Dilmore, D McDonald, TT Nguyen, JB Adams, R Krajmalnik-Brown, E Elijah, PC Dorrestein, R Knight_


Alterations to the gut microbiome have been reported between children with autism spectrum disorders (ASDs) and typically developing (TD) children. Characterizing these differences has led to the proposal of new treatments for ASD, such as probiotic interventions and fecal matter transplants. However, no study to date has characterized the gut microbiome or metabolome in Pitt Hopkins syndrome (PTHS), a severe ASD with a high incidence of gastrointestinal (GI) disturbances such as constipation. Here, we surveyed the gut microbiome and metabolome in a cohort of PTHS individuals and their unaffected parents. We focused our analysis on Clostridium bolteae, a microbe previously associated with ASD known to chemically modify bile acids in the gut. PTHS individuals carry a higher load of C. bolteae than their parents as well as both ASD and non-ASD individuals from the American Gut Project cohort. Specific metabolites were associated with PTHS, including bile acids and sphingosines. With a metadata reanalysis tool, we found that PTHS-associated metabolites have previously been identified in inflammatory bowel disease and obesity patients. These results suggest microbial involvement in PTHS, but further research must be performed to clarify the exact mechanisms through which microbes may act. Furthermore, new associations between PTHS-specific metabolites and other conditions may lead to additional therapeutic options for PTHS individuals. IMPORTANCE GI disturbances in ASD such as severe constipation can be medically significant and often require medication. This is especially true for individuals with PTHS, suggesting that the gut microbiome may be involved in PTHS's pathology. Revealing associations between specific gut microbes and PTHS may allow the development of new therapeutics or the application of existing therapeutics to ease day-to-day challenges encountered by PTHS individuals. In this study, we characterized an association between C. bolteae and PTHS, in addition to metabolites linked to both PTHS and C. bolteae. We also identified other microbiome-involved medical conditions where PTHS-associated metabolites have been isolated. Utilizing common metabolites to identify conditions with similar phenotypes may suggest new therapeutic options for GI-related symptoms.

[Source](https://journals.asm.org/doi/10.1128/mSystems.01006-21)

10.1128/mSystems.01006-21

---

## Characterization of gut microbial structural variations as determinants of human bile acid metabolism.
 30.11.2021, _D Wang, M Doestzada, L Chen, S Andreu-Sánchez, ICL van den Munckhof, HE Augustijn, M Koehorst, AJ Ruiz-Moreno, VW Bloks, NP Riksen, JHW Rutten, LAB Joosten, MG Netea, C Wijmenga, A Zhernakova, F Kuipers, J Fu_


Bile acids (BAs) facilitate intestinal fat absorption and act as important signaling molecules in host-gut microbiota crosstalk. BA-metabolizing pathways in the microbial community have been identified, but it remains largely unknown how the highly variable genomes of gut bacteria interact with host BA metabolism. We characterized 8,282 structural variants (SVs) of 55 bacterial species in the gut microbiomes of 1,437 individuals from two cohorts and performed a systematic association study with 39 plasma BA parameters. Both variations in SV-based continuous genetic makeup and discrete clusters showed correlations with BA metabolism. Metagenome-wide association analysis identified 809 replicable associations between bacterial SVs and BAs and SV regulators that mediate the effects of lifestyle factors on BA metabolism. This is the largest microbial genetic association analysis to demonstrate the impact of bacterial SVs on human BA composition, and it highlights the potential of targeting gut microbiota to regulate BA metabolism through lifestyle intervention.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312821005096)

10.1016/j.chom.2021.11.003

---

## Microbial regulation of hexokinase 2 links mitochondrial metabolism and cell death in colitis.
 30.11.2021, _F Hinrichsen, J Hamm, M Westermann, L Schröder, K Shima, N Mishra, A Walker, N Sommer, K Klischies, D Prasse, J Zimmermann, S Kaiser, D Bordoni, A Fazio, G Marinos, G Laue, S Imm, V Tremaroli, M Basic, R Häsler, RA Schmitz, S Krautwald, A Wolf, B Stecher, P Schmitt-Kopplin, C Kaleta, J Rupp, F Bäckhed, P Rosenstiel, F Sommer_


Hexokinases (HK) catalyze the first step of glycolysis limiting its pace. HK2 is highly expressed in gut epithelium, contributes to immune responses, and is upregulated during inflammation. We examined the microbial regulation of HK2 and its impact on inflammation using mice lacking HK2 in intestinal epithelial cells (Hk2ΔIEC). Hk2ΔIEC mice were less susceptible to acute colitis. Analyzing the epithelial transcriptome from Hk2ΔIEC mice during colitis and using HK2-deficient intestinal organoids and Caco-2 cells revealed reduced mitochondrial respiration and epithelial cell death in the absence of HK2. The microbiota strongly regulated HK2 expression and activity. The microbially derived short-chain fatty acid (SCFA) butyrate repressed HK2 expression via histone deacetylase 8 (HDAC8) and reduced mitochondrial respiration in wild-type but not in HK2-deficient Caco-2 cells. Butyrate supplementation protected wild-type but not Hk2ΔIEC mice from colitis. Our findings define a mechanism how butyrate promotes intestinal homeostasis and suggest targeted HK2-inhibition as therapeutic avenue for inflammation.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1550413121005325)

10.1016/j.cmet.2021.11.004

---

## Bacterial communities from vagina of dairy healthy heifers and cows with impaired reproductive performance.
 30.11.2021, _CG Moreno, AT Luque, KN Galvão, MC Otero_


Vaginal microenvironment plays a significant role in bovine fertility since its resident microorganisms interact with the host mucosa and constitutes the first barrier against ascending pathogens in the reproductive tract. In this study, the vaginal microbiome of healthy heifers (H) and cows with impaired reproductive performance, metritis complex (MT) or repeat breeders (RB), was assessed using a 16S rRNA gene sequencing approach. Analysis revealed that even though a vaginal microbiological guild (Firmicutes, Bacteroidetes, Proteobacteria, Tenericutes, Fusobacteria and Actinobacteria) was shared among healthy heifers and cows with uterine disease; further analysis at genus level showed significant differences depending on the reproductive health status. The relative abundances of recognized uterine pathogens such as Bacteroidetes, Fusobacterium and Helcococcus were higher in MT when compared with H and RB; therefore, their presence in vagina can be considered as a risk factor for fertility. The present study describes for the first time, the composition of native bacterial communities in the vagina of cows undergoing the repeat breeding syndrome (RBS), and reports an association between this disease and the presence of Porphyromonas and unassigned genera of the Pasteurellaceae family. In addition, this work highlights the bacteria associated with a healthy vagina: genera from the families Lachnospiraceae, Rikenellaceae and the genera Acinetobacter, Bacillus, Oscillospira, CF231 and 5-7NS. Results highlighted herein, signify the potential of the evaluation of the bovine vaginal microbiome to future design therapeutic interventions to improve pregnancy rates however, further research is needed to elucidate the balance of bacterial species resulting in an optimal reproductive health.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0034528821003271)

10.1016/j.rvsc.2021.11.007

---

## Artificial Selection on Microbiomes To Breed Microbiomes That Confer Salt Tolerance to Plants.
 30.11.2021, _UG Mueller, TE Juenger, MR Kardish, AL Carlson, KM Burns, JA Edwards, CC Smith, CC Fang, DL Des Marais_


We develop a method to artificially select for rhizosphere microbiomes that confer salt tolerance to the model grass Brachypodium distachyon grown under sodium salt stress or aluminum salt stress. In a controlled greenhouse environment, we differentially propagated rhizosphere microbiomes between plants of a nonevolving, highly inbred plant population; therefore, only microbiomes evolved in our experiment, but the plants did not evolve in parallel. To maximize microbiome perpetuation when transplanting microbiomes between plants and, thus, maximize response to microbiome selection, we improved earlier methods by (i) controlling microbiome assembly when inoculating seeds at the beginning of each selection cycle; (ii) fractionating microbiomes before transfer between plants to harvest, perpetuate, and select on only bacterial and viral microbiome components; (iii) ramping of salt stress gradually from minor to extreme salt stress with each selection cycle to minimize the chance of overstressing plants; (iv) using two nonselection control treatments (e.g., nonselection microbial enrichment and null inoculation) that permit comparison to the improving fitness benefits that selected microbiomes impart on plants. Unlike previous methods, our selection protocol generated microbiomes that enhance plant fitness after only 1 to 3 rounds of microbiome selection. After nine rounds of microbiome selection, the effect of microbiomes selected to confer tolerance to aluminum salt stress was nonspecific (these artificially selected microbiomes equally ameliorate sodium and aluminum salt stresses), but the effect of microbiomes selected to confer tolerance to sodium salt stress was specific (these artificially selected microbiomes do not confer tolerance to aluminum salt stress). Plants with artificially selected microbiomes had 55 to 205% greater seed production than plants with unselected control microbiomes. IMPORTANCE We developed an experimental protocol that improves earlier methods of artificial selection on microbiomes and then tested the efficacy of our protocol to breed root-associated bacterial microbiomes that confer salt tolerance to a plant. Salt stress limits growth and seed production of crop plants, and artificially selected microbiomes conferring salt tolerance may ultimately help improve agricultural productivity. Unlike previous experiments of microbiome selection, our selection protocol generated microbiomes that enhance plant productivity after only 1 to 3 rounds of artificial selection on root-associated microbiomes, increasing seed production under extreme salt stress by 55 to 205% after nine rounds of microbiome selection. Although we artificially selected microbiomes under controlled greenhouse conditions that differ from outdoor conditions, increasing seed production by 55 to 205% under extreme salt stress is a remarkable enhancement of plant productivity compared to traditional plant breeding. We describe a series of additional experimental protocols that will advance insights into key parameters that determine efficacy and response to microbiome selection.

[Source](https://journals.asm.org/doi/10.1128/mSystems.01125-21)

10.1128/mSystems.01125-21

---

## Minnesota peat viromes reveal terrestrial and aquatic niche partitioning for local and global viral populations.
 01.12.2021, _AM Ter Horst, C Santos-Medellín, JW Sorensen, LA Zinke, RM Wilson, ER Johnston, GG Trubl, J Pett-Ridge, SJ Blazewicz, PJ Hanson, JP Chanton, CW Schadt, JE Kostka, JB Emerson_


Peatlands are expected to experience sustained yet fluctuating higher temperatures due to climate change, leading to increased microbial activity and greenhouse gas emissions. Despite mounting evidence for viral contributions to these processes in peatlands underlain with permafrost, little is known about viruses in other peatlands. More generally, soil viral biogeography and its potential drivers are poorly understood at both local and global scales. Here, 87 metagenomes and five viral size-fraction metagenomes (viromes) from a boreal peatland in northern Minnesota (the SPRUCE whole-ecosystem warming experiment and surrounding bog) were analyzed for dsDNA viral community ecological patterns, and the recovered viral populations (vOTUs) were compared with our curated PIGEON database of 266,125 vOTUs from diverse ecosystems.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01156-0)

10.1186/s40168-021-01156-0

---

## Degradation of biological macromolecules supports uncultured microbial populations in Guaymas Basin hydrothermal sediments.
 30.11.2021, _S Pérez Castro, MA Borton, K Regan, I Hrabe de Angelis, KC Wrighton, AP Teske, M Strous, SE Ruff_


Hydrothermal sediments contain large numbers of uncultured heterotrophic microbial lineages. Here, we amended Guaymas Basin sediments with proteins, polysaccharides, nucleic acids or lipids under different redox conditions and cultivated heterotrophic thermophiles with the genomic potential for macromolecule degradation. We reconstructed 20 metagenome-assembled genomes (MAGs) of uncultured lineages affiliating with known archaeal and bacterial phyla, including endospore-forming Bacilli and candidate phylum Marinisomatota. One Marinisomatota MAG had 35 different glycoside hydrolases often in multiple copies, seven extracellular CAZymes, six polysaccharide lyases, and multiple sugar transporters. This population has the potential to degrade a broad spectrum of polysaccharides including chitin, cellulose, pectin, alginate, chondroitin, and carrageenan. We also describe thermophiles affiliating with the genera Thermosyntropha, Thermovirga, and Kosmotoga with the capability to make a living on nucleic acids, lipids, or multiple macromolecule classes, respectively. Several populations seemed to lack extracellular enzyme machinery and thus likely scavenged oligo- or monomers (e.g., MAGs affiliating with Archaeoglobus) or metabolic products like hydrogen (e.g., MAGs affiliating with Thermodesulfobacterium or Desulforudaceae). The growth of methanogens or the production of methane was not observed in any condition, indicating that the tested macromolecules are not degraded into substrates for methanogenesis in hydrothermal sediments. We provide new insights into the niches, and genomes of microorganisms that actively degrade abundant necromass macromolecules under oxic, sulfate-reducing, and fermentative thermophilic conditions. These findings improve our understanding of the carbon flow across trophic levels and indicate how primary produced biomass sustains complex and productive ecosystems.

[Source](https://www.nature.com/articles/s41396-021-01026-5)

10.1038/s41396-021-01026-5

---

## Microbial community structure in hadal sediments: high similarity along trench axes and strong changes along redox gradients.
 30.11.2021, _C Schauberger, RN Glud, B Hausmann, B Trouche, L Maignien, J Poulain, P Wincker, S Arnaud-Haond, F Wenzhöfer, B Thamdrup_


Hadal trench sediments are hotspots of biogeochemical activity in the deep sea, but the biogeochemical and ecological factors that shape benthic hadal microbial communities remain unknown. Here, we sampled ten hadal sites from two trench regions with a vertical resolution of down to 1 cm. We sequenced 16S rRNA gene amplicons using universal and archaea-specific primer sets and compared the results to biogeochemical parameters. Despite bathymetric and depositional heterogeneity we found a high similarity of microbial communities within each of the two trench axes, while composition at the phylum level varied strongly with sediment depth in conjunction with the redox stratification into oxic, nitrogenous, and ferruginous zones. As a result, communities of a given sediment horizon were more similar to each other across a distance of hundreds of kilometers within each trench, than to those of adjacent horizons from the same sites separated only by centimeters. Total organic carbon content statistically only explained a small part of the variation within and between trenches, and did not explain the community differences observed between the hadal and adjacent shallower sites. Anaerobic taxa increased in abundance at the top of the ferruginous zone, seeded by organisms deposited at the sediment surface and surviving burial through the upper redox zones. While an influence of other potential factors such as geographic isolation, hydrostatic pressure, and non-steady state depositional regimes could not be discerned, redox stratification and diagenesis appear to be the main selective forces that structure community composition in hadal sediments.

[Source](https://www.nature.com/articles/s41396-021-01021-w)

10.1038/s41396-021-01021-w

---

## Disentangling environmental effects in microbial association networks.
 30.11.2021, _IM Deutschmann, G Lima-Mendez, AK Krabberød, J Raes, SM Vallina, K Faust, R Logares_


Ecological interactions among microorganisms are fundamental for ecosystem function, yet they are mostly unknown or poorly understood. High-throughput-omics can indicate microbial interactions through associations across time and space, which can be represented as association networks. Associations could result from either ecological interactions between microorganisms, or from environmental selection, where the association is environmentally driven. Therefore, before downstream analysis and interpretation, we need to distinguish the nature of the association, particularly if it is due to environmental selection or not.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01141-7)

10.1186/s40168-021-01141-7

---

## Negative Binomial factor regression with application to microbiome data analysis

[Source](https://www.biorxiv.org/content/10.1101/2021.11.29.470304v1.abstract)

---

## Novel clades of soil biphenyl degraders revealed by integrating isotope probing, multi-omics, and single-cell analyses.
 30.11.2021, _SC Chen, R Budhraja, L Adrian, F Calabrese, H Stryhanyuk, N Musat, HH Richnow, GL Duan, YG Zhu, F Musat_


Most microorganisms in the biosphere remain uncultured and poorly characterized. Although the surge in genome sequences has enabled insights into the genetic and metabolic properties of uncultured microorganisms, their physiology and ecological roles cannot be determined without direct probing of their activities in natural habitats. Here we employed an experimental framework coupling genome reconstruction and activity assays to characterize the largely uncultured microorganisms responsible for aerobic biodegradation of biphenyl as a proxy for a large class of environmental pollutants, polychlorinated biphenyls. We used 13C-labeled biphenyl in contaminated soils and traced the flow of pollutant-derived carbon into active cells using single-cell analyses and protein-stable isotope probing. The detection of 13C-enriched proteins linked biphenyl biodegradation to the uncultured Alphaproteobacteria clade UBA11222, which we found to host a distinctive biphenyl dioxygenase gene widely retrieved from contaminated environments. The same approach indicated the capacity of Azoarcus species to oxidize biphenyl and suggested similar metabolic abilities for species of Rugosibacter. Biphenyl oxidation would thus represent formerly unrecognized ecological functions of both genera. The quantitative role of these microorganisms in pollutant degradation was resolved using single-cell-based uptake measurements. Our strategy advances our understanding of microbially mediated biodegradation processes and has general application potential for elucidating the ecological roles of uncultured microorganisms in their natural habitats.

[Source](https://www.nature.com/articles/s41396-021-01022-9)

10.1038/s41396-021-01022-9

---

