## Exploring Microbiome Functional Dynamics through Space and Time with Trait-Based Theory.
 01.09.2021, _LS Bittleston, ZB Freedman, JR Bernardin, JJ Grothjan, EB Young, S Record, B Baiser, SM Gray_


Microbiomes play essential roles in the health and function of animal and plant hosts and drive nutrient cycling across ecosystems. Integrating novel trait-based approaches with ecological theory can facilitate the prediction of microbial functional traits important for ecosystem functioning and health. In particular, the yield-acquisition-stress (Y-A-S) framework considers dominant microbial life history strategies across gradients of resource availability and stress. However, microbiomes are dynamic, and spatial and temporal shifts in taxonomic and trait composition can affect ecosystem functions. We posit that extending the Y-A-S framework to microbiomes during succession and across biogeographic gradients can lead to generalizable rules for how microbiomes and their functions respond to resources and stress across space, time, and diverse ecosystems. We demonstrate the potential of this framework by applying it to the microbiomes hosted by the carnivorous pitcher plant Sarracenia purpurea, which have clear successional trajectories and are distributed across a broad climatic gradient.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00530-21)

10.1128/mSystems.00530-21

---

## Effects of antibiotic duration on the intestinal microbiota and resistome: The PIRATE RESISTANCE project, a cohort study nested within a randomized trial.
 09.09.2021, _S Leo, V Lazarevic, E von Dach, L Kaiser, V Prendki, J Schrenzel, BD Huttner, A Huttner_


Shortening antibiotic-treatment durations is a key recommendation of antibiotic-stewardship programmes, yet it is based on weak evidence. We investigated whether halving antibiotic courses would reduce antibiotic-resistance genes (ARG) in the intestinal microbiomes of patients treated for gram-negative bacteraemia.

[Source](https://www.thelancet.com/journals/ebiom/article/PIIS2352-3964(21)00359-5/fulltext)

10.1016/j.ebiom.2021.103566

---

## Stool microRNA profiles reflect different dietary and gut microbiome patterns in healthy individuals.
 28.07.2021, _S Tarallo, G Ferrero, F De Filippis, A Francavilla, E Pasolli, V Panero, F Cordero, N Segata, S Grioni, RG Pensa, B Pardini, D Ercolini, A Naccarati_


MicroRNA (miRNA) profiles have been evaluated in several biospecimens in relation to common diseases for which diet may have a considerable impact. We aimed at characterising how specific diets are associated with the miRNome in stool of vegans, vegetarians and omnivores and how this is reflected in the gut microbial composition, as this is still poorly explored.

[Source](https://gut.bmj.com/content/early/2021/07/27/gutjnl-2021-325168)

10.1136/gutjnl-2021-325168

---

## Associations between the gut microbiome and metabolome in early life.
 03.09.2021, _QP Nguyen, MR Karagas, JC Madan, E Dade, TJ Palys, HG Morrison, WW Pathmasiri, S McRitche, SJ Sumner, HR Frost, AG Hoen_


The infant intestinal microbiome plays an important role in metabolism and immune development with impacts on lifelong health. The linkage between the taxonomic composition of the microbiome and its metabolic phenotype is undefined and complicated by redundancies in the taxon-function relationship within microbial communities. To inform a more mechanistic understanding of the relationship between the microbiome and health, we performed an integrative statistical and machine learning-based analysis of microbe taxonomic structure and metabolic function in order to characterize the taxa-function relationship in early life.

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-021-02282-3)

10.1186/s12866-021-02282-3

---

## The composition of the gut microbiota following early-life antibiotic exposure affects host health and longevity in later life.
 25.08.2021, _MA Lynn, G Eden, FJ Ryan, J Bensalem, X Wang, SJ Blake, JM Choo, YT Chern, A Sribnaia, J James, SC Benson, L Sandeman, J Xie, S Hassiotis, EW Sun, AM Martin, MD Keller, DJ Keating, TJ Sargeant, CG Proud, SL Wesselingh, GB Rogers, DJ Lynn_


Studies investigating whether there is a causative link between the gut microbiota and lifespan have largely been restricted to invertebrates or to mice with a reduced lifespan because of a genetic deficiency. We investigate the effect of early-life antibiotic exposure on otherwise healthy, normal chow-fed, wild-type mice, monitoring these mice for more than 700 days in comparison with untreated control mice. We demonstrate the emergence of two different low-diversity community types, post-antibiotic microbiota (PAM) I and PAM II, following antibiotic exposure. PAM II but not PAM I mice have impaired immunity, increased insulin resistance, and evidence of increased inflammaging in later life as well as a reduced lifespan. Our data suggest that differences in the composition of the gut microbiota following antibiotic exposure differentially affect host health and longevity in later life.

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(21)00998-0)

10.1016/j.celrep.2021.109564

---

## Human gut-derived B. longum subsp. longum strains protect against aging in a D-galactose-induced aging mouse model 

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01108-8#Sec2)

---

## Arms race in a cell: genomic, transcriptomic, and proteomic insights into intracellular phage–bacteria interplay in deep-sea snail holobionts 

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01099-6)

---

## Stable-Isotope-Informed, Genome-Resolved Metagenomics Uncovers Potential Cross-Kingdom Interactions in Rhizosphere Soil.
 01.09.2021, _EP Starr, S Shi, SJ Blazewicz, BJ Koch, AJ Probst, BA Hungate, J Pett-Ridge, MK Firestone, JF Banfield_


The functioning, health, and productivity of soil are intimately tied to a complex network of interactions, particularly in plant root-associated rhizosphere soil. We conducted a stable-isotope-informed, genome-resolved metagenomic study to trace carbon from Avena fatua grown in a 13CO2 atmosphere into soil. We collected paired rhizosphere and nonrhizosphere soil at 6 and 9 weeks of plant growth and extracted DNA that was then separated by density using ultracentrifugation. Thirty-two fractions from each of five samples were grouped by density, sequenced, assembled, and binned to generate 55 unique bacterial genomes that were ≥70% complete. We also identified complete 18S rRNA sequences of several 13C-enriched microeukaryotic bacterivores and fungi. We generated 10 circularized bacteriophage (phage) genomes, some of which were the most labeled entities in the rhizosphere, suggesting that phage may be important agents of turnover of plant-derived C in soil. CRISPR locus targeting connected one of these phage to a Burkholderiales host predicted to be a plant pathogen. Another highly labeled phage is predicted to replicate in a Catenulispora sp., a possible plant growth-promoting bacterium. We searched the genome bins for traits known to be used in interactions involving bacteria, microeukaryotes, and plant roots and found DNA from heavily 13C-labeled bacterial genes thought to be involved in modulating plant signaling hormones, plant pathogenicity, and defense against microeukaryote grazing. Stable-isotope-informed, genome-resolved metagenomics indicated that phage can be important agents of turnover of plant-derived carbon in soil. IMPORTANCE Plants grow in intimate association with soil microbial communities; these microbes can facilitate the availability of essential resources to plants. Thus, plant productivity commonly depends on interactions with rhizosphere bacteria, viruses, and eukaryotes. Our work is significant because we identified the organisms that took up plant-derived organic C in rhizosphere soil and determined that many of the active bacteria are plant pathogens or can impact plant growth via hormone modulation. Further, by showing that bacteriophage accumulate CO2-derived carbon, we demonstrated their vital roles in redistribution of plant-derived C into the soil environment through bacterial cell lysis. The use of stable-isotope probing (SIP) to identify consumption (or lack thereof) of root-derived C by key microbial community members within highly complex microbial communities opens the way for assessing manipulations of bacteria and phage with potentially beneficial and detrimental traits, ultimately providing a path to improved plant health and soil carbon storage.

[Source](https://journals.asm.org/doi/10.1128/mSphere.00085-21)

10.1128/mSphere.00085-21

---

## Powerful and robust non-parametric association testing for microbiome data via a zero-inflated quantile approach (ZINQ)

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01129-3)

---

## Ancient and modern genomes unravel the evolutionary history of the rhinoceros family.
 25.08.2021, _S Liu, MV Westbury, N Dussex, KJ Mitchell, MS Sinding, PD Heintzman, DA Duchêne, JD Kapp, J von Seth, H Heiniger, F Sánchez-Barreiro, A Margaryan, R André-Olsen, B De Cahsan, G Meng, C Yang, L Chen, T van der Valk, Y Moodley, K Rookmaaker, MW Bruford, O Ryder, C Steiner, LGR Bruins-van Sonsbeek, S Vartanyan, C Guo, A Cooper, P Kosintsev, I Kirillova, AM Lister, T Marques-Bonet, S Gopalakrishnan, RR Dunn, ED Lorenzen, B Shapiro, G Zhang, PO Antoine, L Dalén, MTP Gilbert_


Only five species of the once-diverse Rhinocerotidae remain, making the reconstruction of their evolutionary history a challenge to biologists since Darwin. We sequenced genomes from five rhinoceros species (three extinct and two living), which we compared to existing data from the remaining three living species and a range of outgroups. We identify an early divergence between extant African and Eurasian lineages, resolving a key debate regarding the phylogeny of extant rhinoceroses. This early Miocene (∼16 million years ago [mya]) split post-dates the land bridge formation between the Afro-Arabian and Eurasian landmasses. Our analyses also show that while rhinoceros genomes in general exhibit low levels of genome-wide diversity, heterozygosity is lowest and inbreeding is highest in the modern species. These results suggest that while low genetic diversity is a long-term feature of the family, it has been particularly exacerbated recently, likely reflecting recent anthropogenic-driven population declines.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(21)00891-6)

10.1016/j.cell.2021.07.032

---

## Five keys to writing a reproducible lab protocol.
 07.09.2021, _M Baker_



[Source](https://www.nature.com/articles/d41586-021-02428-3)

10.1038/d41586-021-02428-3

---

