## Noninvasive assessment of gut function using transcriptional recording sentinel cells.
 16.05.2022, _F Schmidt, J Zimmermann, T Tanna, R Farouni, T Conway, AJ Macpherson, RJ Platt_


Transcriptional recording by CRISPR spacer acquisition from RNA endows engineered Escherichia coli with synthetic memory, which through Record-seq reveals transcriptome-scale records. Microbial sentinels that traverse the gastrointestinal tract capture a wide range of genes and pathways that describe interactions with the host, including quantitative shifts in the molecular environment that result from alterations in the host diet, induced inflammation, and microbiome complexity. We demonstrate multiplexed recording using barcoded CRISPR arrays, enabling the reconstruction of transcriptional histories of isogenic bacterial strains in vivo. Record-seq therefore provides a scalable, noninvasive platform for interrogating intestinal and microbial physiology throughout the length of the intestine without manipulations to host physiology and can determine how single microbial genetic differences alter the way in which the microbe adapts to the host intestinal environment.

[Source](https://doi.org/10.1126/science.abm6038)

10.1126/science.abm6038

---

## Recording bacterial responses to changes in the gut environment.
 16.05.2022, _L Zahavi, E Segal_


A CRISPR-based tool reveals intestinal microbiota gene expression through time.

[Source](https://doi.org/10.1126/science.abq1455)

10.1126/science.abq1455

---

## Contribution of the Microbiome to Western Diet-Induced Gene Expression Changes in the Intestine and Vasculature.
 16.05.2022, _AN Cavalier, TL Weir, CL Gentile, TJ LaRocca_


Western diet (high fat/sugar) is a core contributor to obesity, which is associated with increased risk for chronic diseases, including cardiovascular disease (CVD). The mechanisms through which Western diet contributes to CVD are heavily studied, and increasing evidence suggests that Western diet alters the gut microbiota leading to dysbiosis(i.e., disruption of microbial homeostasis and accompanying intestinal inflammation), contributing to development of vascular dysfunction, a strong predictor of future CVD. Still, the biological processes linking intestinal dysbiosis with the vasculature are incompletely understood. To identify potential mediators of intestinal-vascular crosstalk, we investigated the role of the gut microbiome in Western diet-induced global gene expression changes using transcriptomics. We performed total RNA sequencing on colons and arteries (aorta) from wild-type (WT) and germ-free (GF, no microbiome) mice fed standard or Western purified diets for 8 weeks. In the colon, we found ~100 differentially expressed genes in standard diet-fed WT vs. GF mice, and pathway analyses showed that these genes were associated with several important biological processes including peroxisome proliferator-activated receptor (PPAR) signaling, adipocyte lipolysis regulation, and AMP-activated protein kinase (AMPK) signaling (genes and pathways, FDR &lt; 0.1). In colons of Western diet-fed WT vs. GF mice, we found 163 differentially expressed genes related to cancer signaling, mitogen-activated protein kinase (MAPK) signaling, and viral immune response (FDR &lt; 0.1). Interestingly, in transcriptome analyses of aortas we found significantly different expression of 314 genes in standard diet-fed WT vs. GF mice, and these were related to complement/coagulation cascades, transforming growth factor beta (TGF-β), and advanced glycation end product (AGE) signaling, but most of these differences were absent with Western diet (only 4 significantly different genes in WT vs. GF mice, FDR &lt; 0.1). These data suggest that the microbiome is an important mediator of gene expression in the intestine regardless of diet, but may only be important in arteries under standard diet conditions, and that the microbiome may not be a strong mediator of intestinal-vascular crosstalk at the protein-coding gene expression level. We are currently performing in-depth analyses of non-coding RNAs (e.g., micro-RNAs) to further elucidate the mechanisms at play.

[Source](https://doi.org/10.1096/fasebj.2022.36.S1.L7713)

10.1096/fasebj.2022.36.S1.L7713

---

## The Gut Microbiota Metabolite Succinate promotes Adipose Tissue Browning in Crohn's Disease.
 13.05.2022, _D Monfort-Ferré, A Caro, M Menacho, M Martí, B Espina, A Boronat-Toscano, C Nuñez-Roa, J Seco, M Bautista, E Espín, A Megía, J Vendrell, S Fernández-Veledo, C Serena_


Crohn's disease (CD) is associated with complex microbe-host interactions, involving changes in microbial communities, and gut barrier defects, leading to the translocation of microorganisms to surrounding adipose tissue (AT). We evaluated the presence of beige AT depots in CD and questioned whether succinate and/or bacterial translocation promotes white-to-beige transition in adipocytes.

[Source](https://doi.org/10.1093/ecco-jcc/jjac069)

10.1093/ecco-jcc/jjac069

---

## Changes in Gut Microbiome Composition with Healthy Aging in Humans: Links to Vascular Endothelial Function.
 16.05.2022, _N Greenberg, ML Burnsed-Torres, A Gonzalez, AG Casso, KL Lubieniecki, BP Ziemba, MJ Rossman, EC Adam, M Chonchol, KP Davy, R Knight, DR Seals, VE Brunt_


Aging is the primary risk factor for cardiovascular diseases, primarily due to development of vascular endothelial dysfunction. The gut microbiome is a strong influencer of host physiology, but few studies have investigated how gut microbiome composition changes with primary (healthy) aging in humans, or how such changes may influence endothelial function.

[Source](https://doi.org/10.1096/fasebj.2022.36.S1.R2759)

10.1096/fasebj.2022.36.S1.R2759

---

## The Gut Microbiota Modulates Gene Expression in the Kidney.
 16.05.2022, _B Moore, J Pluznick_


Gut microbiota are a diverse array of commensal bacteria in the intestinal lumen, and shifts in the composition of gut microbes can influence host health. We have previously established a link between the gut microbiota and host kidney function by showing that gut microbial metabolites can activate G protein-coupled receptors in the kidney. Although host-microbiome interactions have been studied at the signaling level, the influence of gut microbes on gene expression at the transcriptional level is unclear. Evidence demonstrates that gut microbiota alters gene transcription in the colon (J. Allen, Genome Medicine, 2019); however, it is entirely unknown whether gut microbiota influences host gene expression at distal sites, such as the kidney. To address this question for the first time, we utilized an unbiased RNA sequencing (RNASeq) approach to compare gene expression in the kidneys of male and female mice which were either germ-free (GF; without gut microbes) or conventionalized (Conv; with gut microbes). All mice were born on a C57Bl/6 background and housed in the Johns Hopkins GF facility. At four weeks of age, a subset of mice (n=5 males, n=5 females) were conventionalized via oral gavage using a fecal slurry of mixed stool from age-matched specific-pathogen free (SPF) mice (n=3 males, n=3 females); whereas an additional cohort of mice (n=5 males, n=5 females) were maintained as GF. Eight weeks after conventionalization both GF and Conv mice were sacrificed, whole kidneys were processed for RNASeq, and fecal pellets from Conv mice were collected for 16S rRNA sequencing. For RNASeq analysis, differentially expressed genes (DEGs) were defined as having a log2fold change ≥ ± 0.5 and base mean ≥ 15 (the average of the normalized count values, dividing by size factors, taken over all samples). We identified a clear effect of the gut microbiota on renal gene expression: 578 DEGs between GF and Conv males, and 340 DEGs between GF and Conv females. Only 73 of these DEGs were common to both males and females. These data demonstrate that the gut microbiota not only impacts gene expression in the kidney, but does so in a sex-specific manner. In addition, 16S rRNA sequencing showed no differences in bacterial colonization between male and female Conv mice, establishing that the renal transcriptomic differences are not a consequence of differing gut microbes; rather, the same microbes have differential effects in males and females, presumably through differences in either signals (i.e., microbial metabolites), or differences in host responses. Furthermore, DAVID pathway analysis revealed that DEGs are involved in pathways including aldosterone synthesis and secretion, mineral absorption, proximal tubule bicarbonate reclamation, and vasopressin-regulated water reabsorption. In sum, we demonstrate that the gut microbiota influences kidney gene expression in a sex-specific manner, and that these transcriptional changes are poised to impact kidney physiology and regulation. We are now working to identify physiologically relevant signals from the microbiome and/or host that mediate these sex-specific differences, and to determine how they influence renal function.

[Source](https://doi.org/10.1096/fasebj.2022.36.S1.R2348)

10.1096/fasebj.2022.36.S1.R2348

---

## Rapid evolution and strain turnover in the infant gut microbiome.
 11.05.2022, _DW Chen, NR Garud_


While the ecological dynamics of the infant gut microbiome have been intensely studied, relatively little is known about evolutionary dynamics in the infant gut microbiome. Here we analyze longitudinal fecal metagenomic data from &gt;700 infants and their mothers over the first year of life and find that the evolutionary dynamics in infant gut microbiomes are distinct from that of adults. We find evidence for more than 10-fold increase in the rate of evolution and strain turnover in the infant gut compared to healthy adults, with the mother-infant transition at delivery being a particularly dynamic period in which gene loss dominates. Within a few months after birth, these dynamics stabilize, and gene gains become increasingly frequent as the microbiome matures. We furthermore find that evolutionary changes in infants show signatures of being seeded by a mixture of de novo mutations and transmissions of pre-evolved lineages from the broader family. Several of these evolutionary changes occur in parallel across infants, highlighting candidate genes that may play important roles in the development of the infant gut microbiome. Our results point to a picture of a volatile infant gut microbiome characterized by rapid evolutionary and ecological change in the early days of life.

[Source](https://genome.cshlp.org/content/early/2022/05/11/gr.276306.121.abstract)

10.1101/gr.276306.121

---

## The role of short-chain fatty acids in <em>Clostridioides difficile</em> infection: A review

[Source](https://doi.org/10.1016/j.anaerobe.2022.102585)

---

## Lung microbiome composition and bronchial epithelial gene expression in patients with COPD versus healthy individuals: a bacterial 16S rRNA gene sequencing and host transcriptomic analysis.
 16.05.2022, _MY Ramsheh, K Haldar, A Esteve-Codina, LF Purser, M Richardson, J Müller-Quernheim, T Greulich, A Nowinski, I Barta, M Stendardo, P Boschetto, D Korzybski, A Prasse, DG Parr, JM Hohlfeld, B Döme, T Welte, S Heath, I Gut, JA Morrissey, L Ziegler-Heitbrock, MR Barer, D Singh, CE Brightling_


Chronic obstructive pulmonary disease (COPD) is associated with airway inflammation and bacterial dysbiosis. The relationship between the airway microbiome and bronchial gene expression in COPD is poorly understood. We aimed to identify differences in the airway microbiome from bronchial brushings in patients with COPD and healthy individuals and to investigate whether any distinguishing bacteria are related to bronchial gene expression.

[Source](https://doi.org/10.1016/S2666-5247(21)00035-5)

10.1016/S2666-5247(21)00035-5

---

## Therapeutic Targeting of the Respiratory Microbiome.
 13.05.2022, _SH Chotirmall, D Bogaert, JD Chalmers, MJ Cox, PM Hansbro, YJ Huang, PL Molyneaux, DN O'Dwyer, AA Pragman, GB Rogers, LN Segal, RP Dickson_


The last decade of research has revolutionized our understanding of respiratory microbiology, revealing that the lungs and airways contain diverse and dynamic microbial communities in health and disease. This "respiratory ecosystem"-a densely interconnected environment of microbial and host interactions-represents a tremendous and under-appreciated source of biological and clinical heterogeneity across patients with acute and chronic lung disease. Unlike other major sources of heterogeneity, such as comorbidities and host genetics, the respiratory microbiome is readily modifiable by clinical interventions, and therefore represents an untapped opportunity for therapeutic manipulation. As a potential "treatable trait" in efforts to subphenotype patients and deliver precision medicine, the respiratory microbiome is a promising therapeutic target. In this Pulmonary Perspective, we identify and discuss multiple challenges, both conceptual and practical, that must be overcome before the respiratory microbiome can be effectively modulated as a therapeutic target. Barriers include: 1) the need to identify specific microbiologic and ecologic "targets" for therapeutic modulation; 2) the need for an improved understanding of the efficacy and persistence of response to respiratory microbiome-modulating interventions; 3) the need for clinicians to be able to access, understand and utilize microbiome data for sub-phenotyping patients, and 4) specific concerns in special populations (including children, patients with chronic lung disease, and critically ill patients). By delineating these barriers, we identify opportunities for prospective research to advance our understanding of the respiratory microbiome, its role in human respiratory disease, and its genuine potential as a therapeutic target.

[Source](https://doi.org/10.1164/rccm.202112-2704PP)

10.1164/rccm.202112-2704PP

---

## Temporal Dynamics of the Nasopharyngeal Microbiome and its Relationship with Childhood Asthma Exacerbation.
 13.05.2022, _J Hou, Y Song, ASY Leung, MF Tang, M Shi, EY Wang, JGS Tsun, RWY Chan, GWK Wong, SK Tsui, TF Leung_


Despite distinct nasopharyngeal microbiome (NPM) profiles between asthmatics and healthy subjects, little is known about the NPM dynamics and its relation to childhood asthma exacerbation (AE). We investigated NPM changes by longitudinally collecting 135 flocked nasopharyngeal swabs (FNPSs) from 33 school-age asthmatic children at six time points (2 to 4-week intervals) from September to December 2017 in Hong Kong. Subjects were categorized into AE and stable asthma (AS) groups according to whether they experienced any exacerbation during follow-up. One-off FNPSs from nine nonasthmatic children were included as controls. Microbiota profiles were analyzed using 16S rRNA gene sequencing. All 144 NPMs were classified into six microbiome profile groups (MPGs), each dominated by Moraxella, Corynebacterium 1, Dolosigranulum, Staphylococcus, Streptococcus, or Anoxybacillus. The microbial diversity and compositions of NPM in exacerbation samples were different from both baseline samples and those from healthy controls. Moraxella and Dolosigranulum-dominated NPM exhibited high temporal stability revealed by MPG transition analysis. NPM diversity decreased whereas microbial composition remained similar over time. The relative abundances of Moraxella increased while Corynebacterium 1, Anoxybacillus, and Pseudomonas decreased longitudinally. However, these temporal patterns did not differ between AE and AS groups, suggesting that short-term dynamic patterns were not sufficient to predict AE occurrence. Asthmatic NPM underwent Moraxella expansion during AE and presented a high microbiome resilience (recovery potential) after AE resolution. Microbial pathways involved in methane, ketone bodies, and vitamin B3 metabolisms were enhanced during AE and primarily contributed by Moraxella. IMPORTANCE Evidence on the dynamic changes of NPM in asthmatic patients remains limited. Here, we present that asthmatic NPMs deviating from a healthy status still showed resilience after disturbance. Our data imply from a longitudinal perspective that Moraxella increase is closely related to AE occurrence. The finding of functional dysbiosis (imbalance) during AE offers a plausible explanation for the known association between nasopharyngeal Moraxella expansion and increased AE risk. This work serves as a basis for future long-term prospective studies leveraging multiomics approaches to elucidate the temporal association between NPM and pediatric AE.

[Source](https://doi.org/10.1128/spectrum.00129-22)

10.1128/spectrum.00129-22

---

## Role of Oral Microbiota Biofilms in Recurrent Tonsillitis and Sleep-Disordered Breathing in Children.
 16.05.2022, _T Allain, E DeMichele, JC Stearns, W Yunker, AG Buret_


The palatine tonsils are aggregates of lymphoid tissue in the lateral oropharynx. In children, tonsillar enlargement can lead to upper airway obstruction and sleep-disordered breathing (SDB). The reason for tonsillar enlargement in some children, and not others is unknown. We have investigated the role of the tonsillar microbiota in tonsillar enlargement. The role of bacterial biofilms in RT and SDB was investigated through comparisons of tonsil histopathology, mixed-species biofilm's microbiota composition and biofilm phenotypes. We hypothesized that the SDB tonsillar phenotype has distinct biofilm forming bacterial communities driving tonsillar hyperplasia compared to recurrent tonsillitis.

[Source](https://doi.org/10.1096/fasebj.2022.36.S1.R4279)

10.1096/fasebj.2022.36.S1.R4279

---

## Oral Microbiota of Amazonian Kichwa Infants From Equador During the First Six Months of Life.
 16.05.2022, _D Maldonado Lino, B Diaz, W Vanden Berghe, JM Cevallos-Cevallos, A Orellana-Manzano_


For public health policy, this study aimed to identify the oral microbiota in babies of the Amazonian Kichwa (AK) population of Ecuador born vaginally during the first six months of life.

[Source](https://doi.org/10.1096/fasebj.2022.36.S1.0R855)

10.1096/fasebj.2022.36.S1.0R855

---

## Child saliva microbiota and caries: a randomized controlled maternal education trial in rural Uganda.
 16.05.2022, _GKM Muhoozi, K Li, P Atukunda, AB Skaare, T Willumsen, M Enersen, AC Westerberg, A Morris, AR Vieira, PO Iversen, BA Methé_


Undernutrition is a public health challenge in sub-Saharan countries, including Uganda. In a previous randomized controlled trial (RCT) with a nutrition, hygiene and stimulation education intervention among mothers of 6 months' old children, we found less caries in the intervention group when the children were 36 months of age. We now examined the effects of (i) the intervention on the microbiota, (ii) microbiota on caries, and (iii) the intervention and microbiota on caries. The original RCT comprised 511 mother/child pairs whereas in the current study we had access to data from 344/511 (67%) children aged 36 months. The saliva microbiota was determined using 16S rRNA gene sequencing. Carious lesions (a proxy for dental health) were identified using close-up intra-oral photographs of the upper front teeth. Statistical models were used to determine host-microbiota associations. The intervention had a significant effect on the microbiota, e.g. an increase in Streptococcus abundance and decreases in Alloprevotella and Tannerella. Significant associations between the microbiota and dental caries were identified: Positive associations of Capnocytophaga and Tannerella suggest that these taxa may be deleterious to dental health while negative associations of Granulicatella, Fusobacterium, and Abiotrophia suggest taxa potentially beneficial or benign contributors to dental health. Based on taxonomic profiles, the effects of the intervention and microbiota on dental health may be independent of one another. Educational interventions with emphasis on nutrition and oral hygiene may provide a feasible strategy to decrease progression of childhood caries in low-resource settings.

[Source](https://doi.org/10.1038/s41598-022-11979-y)

10.1038/s41598-022-11979-y

---

## Gut microbiota: Role and Association with Tumorigenesis in Different Malignancies.
 11.05.2022, _A Ali, A Ara, MK Kashyap_


The microbiota has been associated with different cancer and may vary from patient to patient. A specific microbial strain can alter the progression of cancer and therapeutic outcome in response to anti-cancer therapy. The variations in microbiota contributed due to the individual microbiome of the microorganism are responsible for diverse clinical outcomes. The expansion of microbiota subpopulation during dysbiosis can lead to toxin production, inducing inflammation and cancer. The microbiota can be a dual-edged sword because it can be tumor-suppressive or oncogenic in the case of the gut. The transition of cancer cells from early to late-stage also impacts the composition of the microbiota, and this alteration could change the behavior of cancer. Multi-omics platforms derived data from an individual's multi-dimensional data (DNA, mRNA, microRNA, protein, metabolite, microbiota, and microbiome), i.e., individualome, to exploit it for personalized tailored treatment for different cancers in a precise manner. A number of studies suggest the importance of microbiota and its add-in suitability to existing treatment options for different malignancies. Furthermore, in vitro, and in vivo studies and cancer clinical trials suggest that probiotics have driven modulation of gut microbiota and other sites discourage the aggressive behavior and progression of different cancers.

[Source](https://doi.org/10.1007/s11033-022-07357-6)

10.1007/s11033-022-07357-6

---

## <em>Fusobacterium nucleatum</em> as a Marker for Epithelial to Mesenchymal Transition in Colorectal Cancer

[Source](https://doi.org/10.1096/fasebj.2022.36.S1.L7993)

---

## L-norvaline affects the proliferation of breast cancer cells based on the microbiome and metabolome analysis.
 11.05.2022, _Q Zhu, H Zai, K Zhang, X Zhang, N Luo, X Li, Y Hu, Y Wu_


The altered fecal metabolites and microbiota might be involved in the development of breast cancer. We aimed to investigate the effect of differential metabolites on the proliferative activity of breast cancer cells.

[Source](https://doi.org/10.1111/jam.15620)

10.1111/jam.15620

---

## Integrative multiomics analysis reveals host-microbe-metabolite interplays associated with the aging process in Singaporeans.
 16.05.2022, _L Chen, T Zheng, Y Yang, PP Chaudhary, JPY Teh, BK Cheon, D Moses, SC Schuster, J Schlundt, J Li, PL Conway_


The age-associated alterations in microbiomes vary across populations due to the influence of genetics and lifestyles. To the best of our knowledge, the microbial changes associated with aging have not yet been investigated in Singapore adults. We conducted shotgun metagenomic sequencing of fecal and saliva samples, as well as fecal metabolomics to characterize the gut and oral microbial communities of 62 healthy adult male Singaporeans, including 32 young subjects (age, 23.1 ± 1.4 years) and 30 elderly subjects (age, 69.0 ± 3.5 years). We identified 8 gut and 13 oral species that were differentially abundant in elderly compared to young subjects. By combining the gut and oral microbiomes, 25 age-associated oral-gut species connections were identified. Moreover, oral bacteria Acidaminococcus intestine and Flavonifractor plautii were less prevalent/abundant in elderly gut samples than in young gut samples, whereas Collinsella aerofaciens and Roseburia hominis showed the opposite trends. These results indicate the varied gut-oral communications with aging. Subsequently, we expanded the association studies on microbiome, metabolome and host phenotypic parameters. In particular, Eubacterium eligens increased in elderly compared to young subjects, and was positively correlated with triglycerides, which implies that the potential role of E. eligens in lipid metabolism is altered during the aging process. Our results demonstrated aging-associated changes in the gut and oral microbiomes, as well as the connections between metabolites and host-microbe interactions, thereby deepening the understanding of alterations in the human microbiome during the aging process in a Singapore population.

[Source](https://doi.org/10.1080/19490976.2022.2070392)

10.1080/19490976.2022.2070392

---

## Association of antibiotics exposure within the first 2 years after birth with subsequent childhood type 1 diabetes.
 13.05.2022, _D Lee, S Choi, J Chang, YJ Park, JH Kim, SM Park_


Antibiotics prescription in early life can cause dysbiosis, an imbalance of gut microbiota. We aimed to reveal the relationship between antibiotics exposure during the first 2 years after birth and type 1 diabetes risk in children under 8 years of age using a nationally representative data from South Korea.

[Source](https://doi.org/10.1007/s12020-022-03042-7)

10.1007/s12020-022-03042-7

---

## The resistance within: Antibiotic disruption of the gut microbiome and resistome dynamics in infancy.
 16.05.2022, _R Thänert, SS Sawhney, DJ Schwartz, G Dantas_


Intestinal host-microbiota interactions during the first year of life are critical for infant development. Early-life antibiotic exposures disrupt stereotypical gut microbiota maturation and adversely affect childhood health. Furthermore, antibiotics increase the abundance of resistant bacteria and enrich the resistome-the compendium of antibiotic resistance genes-within the gut microbiota. Here, we discuss acute and persistent impacts of antibiotic exposure during infancy on pediatric health, the gut microbiome, and, particularly, the resistome. Reviewing our current understanding of antibiotic resistance acquisition and dissemination within and between microbiomes, we highlight open questions, which are imperative to resolve in the face of rising bacterial resistance.

[Source](https://doi.org/10.1016/j.chom.2022.03.013)

10.1016/j.chom.2022.03.013

---

## The developing infant gut microbiome: A strain-level view.
 16.05.2022, _H Enav, F Bäckhed, RE Ley_


At birth, neonates provide a vast habitat awaiting microbial colonization. Microbiome assembly is a complex process involving microbial seeding and succession driven by ecological forces and subject to environmental conditions. These successional events not only significantly affect the ecology and function of the microbiome, but also impact host health. While the establishment of the infant microbiome has been a point of interest for decades, an integrated view focusing on strain level colonization has been lacking until recently. Technological and computational advancements enabling strain-level analyses of the infant microbiome have demonstrated the immense complexity of this system and allowed for an improved understanding of how strains of the same species spread, colonize, evolve, and affect the host. Here, we review the current knowledge of the establishment and maturation of the infant gut microbiome with particular emphasis on newer discoveries achieved through strain-centric analyses.

[Source](https://doi.org/10.1016/j.chom.2022.04.009)

10.1016/j.chom.2022.04.009

---

## The healthy female microbiome across body sites: effect of hormonal contraceptives and the menstrual cycle.
 13.05.2022, _MC Krog, LW Hugerth, E Fransson, Z Bashir, A Nyboe Andersen, G Edfeldt, L Engstrand, I Schuppe-Koistinen, HS Nielsen_


How does hormonal contraceptive use and menstrual cycle phase affect the female microbiome across different body sites?

[Source](https://doi.org/10.1093/humrep/deac094)

10.1093/humrep/deac094

---

## Multi-scale study of the oral and gut environments in children with high and low threshold peanut allergy.
 13.05.2022, _L Zhang, Y Chun, HE Ho, Z Arditi, T Lo, S Sajja, R Rose, D Jones, J Wang, S Sicherer, S Bunyavanich_


The oral and gut microbiomes have each been associated with food allergy status. Within food allergy, they may also influence reaction thresholds.

[Source](https://doi.org/10.1016/j.jaci.2022.04.026)

10.1016/j.jaci.2022.04.026

---

## Supplementation with a probiotic mixture accelerates gut microbiome maturation and reduces intestinal inflammation in extremely preterm infants.
 16.05.2022, _J Samara, S Moossavi, B Alshaikh, VA Ortega, VK Pettersen, T Ferdous, SL Hoops, A Soraisham, J Vayalumkal, D Dersch-Mills, JS Gerber, S Mukhopadhyay, K Puopolo, TA Tompkins, D Knights, J Walter, H Amin, MC Arrieta_


Probiotics are increasingly administered to premature infants to prevent necrotizing enterocolitis and neonatal sepsis. However, their effects on gut microbiome assembly and immunity are poorly understood. Using a randomized intervention trial in extremely premature infants, we tested the effects of a probiotic product containing four strains of Bifidobacterium species autochthonous to the infant gut and one Lacticaseibacillus strain on the compositional and functional trajectory of microbiome. Daily administration of the mixture accelerated the transition into a mature, term-like microbiome with higher stability and species interconnectivity. Besides infant age, Bifidobacterium strains and stool metabolites were the best predictors of microbiome maturation, and structural equation modeling confirmed probiotics as a major determinant for the trajectory of microbiome assembly. Bifidobacterium-driven microbiome maturation was also linked to an anti-inflammatory intestinal immune milieu. This demonstrates that Bifidobacterium strains are ecosystem engineers that lead to an acceleration of microbiome maturation and immunological consequences in extremely premature infants.

[Source](https://doi.org/10.1016/j.chom.2022.04.005)

10.1016/j.chom.2022.04.005

---

## Effective Regulation of Gut Microbiota With Probiotics and Prebiotics May Prevent or Alleviate COVID-19 Through the Gut-Lung Axis.
 16.05.2022, _L Xu, CS Yang, Y Liu, X Zhang_


Coronavirus disease 2019 (COVID-19) can disrupt the gut microbiota balance, and patients usually have intestinal disorders. The intestine is the largest immune organ of the human body, and gut microbes can affect the immune function of the lungs through the gut-lung axis. Many lines of evidence support the role of beneficial bacteria in enhancing human immunity, preventing pathogen colonization, and thereby reducing the incidence and severity of infection. In this article, we review the possible approach of modulating microbiota to help prevent and treat respiratory tract infections, including COVID-19, and discuss the possibility of using probiotics and prebiotics for this purpose. We also discuss the mechanism by which intestinal micro-flora regulate immunity and the effects of probiotics on the intestinal micro-ecological balance. Based on this understanding, we propose the use of probiotics and prebiotics to modulate gut microbiota for the prevention or alleviation of COVID-19 through the gut-lung axis.

[Source](https://doi.org/10.3389/fphar.2022.895193)

10.3389/fphar.2022.895193

---

## A Reciprocal Link Between Gut Microbiota, Inflammation and Depression: A Place for Probiotics?
 16.05.2022, _A Eltokhi, IE Sommer_


Depression is a severe mental disorder that places a significant economic burden on public health. The reciprocal link between the trillions of bacteria in the gut, the microbiota, and depression is a controversial topic in neuroscience research and has drawn the attention of public interest and press coverage in recent years. Mounting pieces of evidence shed light on the role of the gut microbiota in depression, which is suggested to involve immune, endocrine, and neural pathways that are the main components of the microbiota-gut-brain axis. The gut microbiota play major roles in brain development and physiology and ultimately behavior. The bidirectional communication between the gut microbiota and brain function has been extensively explored in animal models of depression and clinical research in humans. Certain gut microbiota strains have been associated with the pathophysiology of depression. Therefore, oral intake of probiotics, the beneficial living bacteria and yeast, may represent a therapeutic approach for depression treatment. In this review, we summarize the findings describing the possible links between the gut microbiota and depression, focusing mainly on the inflammatory markers and sex hormones. By discussing preclinical and clinical studies on probiotics as a supplementary therapy for depression, we suggest that probiotics may be beneficial in alleviating depressive symptoms, possibly through immune modulation. Still, further comprehensive studies are required to draw a more solid conclusion regarding the efficacy of probiotics and their mechanisms of action.

[Source](https://doi.org/10.3389/fnins.2022.852506)

10.3389/fnins.2022.852506

---

## Reduced housing density improves statistical power of murine gut microbiota studies.
 16.05.2022, _A Russell, JN Copio, Y Shi, S Kang, CL Franklin, AC Ericsson_


The gut microbiome of humans and animals is critical to host health. Mice are used to investigate the microbiome and its influences; however, the predictive value of such studies is hindered by cage effects due to coprophagy. Our objectives were to evaluate the influence of cage density on the statistical power to detect treatment-dependent effects of a selective pressure on microbiome composition. C57BL/6 mice were separated into groups of 2 or 4 mice per cage and then assigned to groups receiving enrofloxacin, broad-spectrum antibiotics, or control. Fecal samples were collected at weeks 0, 1, and 4, along with contents of the jejunum and cecum. Bacterial DNA analysis examined microbiome richness, diversity, and variability within and between cages. Statistical analyses reveal that reduced housing density consistently results in comparable susceptibility to antibiotics, reduced cage effects, and increased statistical power to detect treatment-associated effects, justifying the practice of reduced housing density.

[Source](https://doi.org/10.1016/j.celrep.2022.110783)

10.1016/j.celrep.2022.110783

---

## Investigation of Effects of Hyperglycaemia on the Lung Microbiome in Diabetic Mice.
 16.05.2022, _S Vasiljevs, D Baines_


The lungs are constantly exposed to a diversity of microbes. On average, the human inhales between 0.7 and 7000 bacterial colony forming units (CFU) every minute. The airway epithelium and the airway surface liquid (ASL) which lines the luminal surface, play a vital role in the defence against these inhaled organisms. Glucose concentration in the ASL is much lower than that of blood (approximately 12.5 times lower). It was proposed that low glucose concentration in the ASL contributes to innate protection against the growth of pathogenic organisms which can utilise glucose for growth. Previous research demonstrated that a sustained increase in blood glucose concentration (such as diabetes) led to increased glucose concentration found in the ASL in both human and animals. We therefore hypothesised that the microbial population of the lung would change in the diabetic lung. Seven-week-old female db/db (BKS.Cg-+Leprdb/+Leprdb/OlaHsd) and non-diabetic littermates (BKS.Cg-(Lean)/OlaHsd) db/db mice and non-diabetic littermate controls were purchased from Envigo (UK). Mice were maintained in standard animal housing in a 12h light/dark cycle; water and standard rodent chow available ad libitumand allowed to acclimatise for three weeks before lung microbiome collection. Mice were terminated with an overdose of pentobarbital (0.2ml of 100mg/ml i.p.). Blood was collected for glucose measurement. Bronchoalveolar lavage was performed and 1 mL of solution was used to extract bacterial DNA using QIAamp DNA Microbiome Kit (Qiagen). The V3-V4-region of the 16S rRNA gene was amplified and sequenced using 300 bp paired-end reads on the Illumina MiSeq platform. Bioinformatic analysis was performed using Mothur v1.39.5 as per the MiSeq SOP pipeline. After removing of contaminant sequence reads, downstream statistical analyses were performed using R statistical software. The bacterial diversity in BAL samples was highly variable within and between diabetic and non-diabetic mice. Hyperglycaemia did not affect the a-diversity of the lung microbiome (Inverse Simpson rating). However, hyperglycaemia had a significant effect on the b-diversity of lung microbiome (analysed with AMOVA, p=0.011, n=9) with the microbiome from diabetic mice clustering together. At the genus level, bacteria of genus Staphylococcus were more abundant in the normoglycaemic mice (n=9, p=0.019). The genus Pseudomonas were more abundant in diabetic mice (n=9, p=0.028) and Corynebacterium (n=9, p=0.0018), which are frequently found in the lung microbiome as commensal organisms, were decreased. Taken together, these data indicate that sustained hyperglycaemia modifies the lung microbiome, decreasing the abundance of commensal bacteria and promoting the growth of glucose-utilising bacteria such as Pseudomonas which may include potential pathogenic species such as P. aeruginosa.

[Source](https://doi.org/10.1096/fasebj.2022.36.S1.L7744)

10.1096/fasebj.2022.36.S1.L7744

---

## Dietary Impacts on Changes in Diversity and Abundance of the Murine Microbiome During Progression and Treatment of Cancer.
 16.05.2022, _H Paden, N Kurbatfinski, J Poelstra, S Ilic_


The microbiome has been recognized as being highly impactful for cancer outcomes, both with respect to the diversity and composition of the microbial population, as well as overall abundance. Presence or absence of certain microbial species has been correlated to improved cancer outcomes, in concert with specific treatments. However, little research has been performed to consider the reciprocal of this relationship: how different treatments, or the progression of disease, impact the microbiome. Further, though correlations have been demonstrated between dietary intake of both fats and carbohydrates and the composition and breadth of the microbiome, no studies have investigated the microbiome population as it is impacted by cancer and therapeutic treatments under different dietary regiments. This study aims to investigate how dietary patterns, in concert with progression of cancer and both surgical and therapeutic interventions, impact the diversity and abundance of the murine microbiome. A total of 116 c57BI/6 female mice (7-8 weeks old) were randomized into four groups (n=29/group, each) one week post ovariectomy. Mice received a specific, semi-purified diet consisting of A) low sucrose, high omega 3 fatty acid; B) low sucrose, low omega 3 fatty acid; C) high sucrose, high omega 3 fatty acid; or D) high sucrose, high omega 3 fatty acid. For high omega 3 fatty acid groups, 2% of daily calories were supplied to the mice as EPA and DHA in a 1.5:1 ratio. One week after implementation of diets all mice were inoculated with E0771 breast cancer cells. Twelve days following inoculation all mice were subject to lumpectomy. Eight days following, measurement procedures were repeated, as at baseline. Ten days following lumpectomy, half of the mice from each dietary cohort were treated with saline, while the other half were treated with a chemotherapeutic agent (9 mg/kg body weight doxorubicin and 90 mg/kg body weight cyclophosphamide). Six days later, the experiment ended, and mice were sacrificed following IACUC guidelines. Fecal samples, which had been collected at each time point noted above, were processed using QIAmp® PowerFecal® DNA isolation kit and were sent to MCIC (OSU Wooster Campus) for 16S ribosomal genetic profiling. A genomic library was generated, using previously validated primers for the V4 region. FASTQ files were filtered, trimmed, dereplicated, and denoised. Cleaned FASTQ files were then used to infer ASVs. Over the progression of cancer, there has been a consistent decrease in the alpha diversity of the microbiome. The effect of diet was mitigated by introduction of chemotherapeutic agents, though the significance of this effect is yet to be determined. While the analysis is still ongoing, it has been determined that there is a clear dietary effect which interacts with abundance. Multiple ASVs have been shown to be positively associated with high fat consumption, though inverse effects, as well as timepoint and treatment effects, have also been noted. Akkermansia muciniphila was a highly present bacterial species in all treatments, despite it usually being noted only as a minor resident of the gut.

[Source](https://doi.org/10.1096/fasebj.2022.36.S1.0R381)

10.1096/fasebj.2022.36.S1.0R381

---

## Impact of Gut <em>R. torques</em> on Type 2 Diabetes in Selenium-deficient Mice

[Source](https://doi.org/10.1096/fasebj.2022.36.S1.L7599)

---

## Interaction between gut microbiota dysbiosis and lung infection as gut-lung axis caused by <em>Streptococcus suis</em> in mouse model

[Source](https://doi.org/10.1016/j.micres.2022.127047)

---

## The influence of maternal unhealthy diet on maturation of offspring gut microbiota in rat.
 16.05.2022, _K Hasebe, MD Kendig, NO Kaakoush, A Tajaddini, RF Westbrook, MJ Morris_


Despite well-known effects of diet on gut microbiota diversity, relatively little is known about how maternal diet quality shapes the longitudinal maturation of gut microbiota in offspring. To investigate, we fed female rats standard chow (Chow) or a western-style, high-choice cafeteria diet (Caf) prior to and during mating, gestation and lactation. At weaning (3 weeks), male and female offspring were either maintained on their mother's diet (ChowChow, CafCaf groups) or switched to the other diet (ChowCaf, CafChow). Fecal microbial composition was assessed in dams and longitudinally in offspring at 3, 7 and 14 weeks of age.

[Source](https://doi.org/10.1186/s42523-022-00185-w)

10.1186/s42523-022-00185-w

---

## Lactoferrin modulates gut microbiota and Toll-like receptors (TLRs) in mice with dysbiosis induced by antibiotics.
 12.05.2022, _A Bellés, D Aguirre-Ramírez, I Abad, M Parras-Moltó, L Sánchez, L Grasa_


Background: Antibiotic administration can result in gut microbiota and immune system alterations that impact health. Bovine lactoferrin is a milk protein with anticancer, anti-inflammatory, antimicrobial and immune modulatory activities. The aim was to study the ability of native and iron-saturated lactoferrin to reverse the effects of clindamycin on gut microbiota and intestinal Toll-like receptor (TLR) expression in a murine model. Methods: Male C57BL/6 mice were treated with vehicle, clindamycin (Clin), native bovine lactoferrin (nLf), nLf + clindamycin (nLf_Clin), iron-saturated bovine lactoferrin (sLf) and sLf + clindamycin (sLf_Clin). Fecal samples of each group were collected, and bacterial DNA was extracted. Sequencing of 16s rRNA V4 hypervariable gene regions was conducted to assess the microbial composition. mRNA expression levels of TLRs (1-9) were determined in mouse colon by qPCR. Pearson's correlation test was carried out between bacteria showing differences in abundance among samples and TLR2, TLR8 and TLR9. Results: Beta-diversity analysis showed that the microbial community of the vehicle was different from the communities of Clin, nLf_Clin and sLf_Clin. At the family level, Bacteroidaceae, Prevotellaceae and Rikenellaceae decreased in the Clin group, and treatment with nLf or sLf reverted these effects. Clin reduced the expression of TLR2, TLR8 and TLR9 and sLf reverted the decrease in the expression of these receptors. Finally, TLR8 was positively correlated with Rikenellaceae abundance. Conclusion: In a situation of intestinal dysbiosis induced by clindamycin, lactoferrin restores the normal levels of some anti-inflammatory bacteria and TLRs and, therefore, could be a good ingredient to be added to functional foods.

[Source](https://doi.org/10.1039/d2fo00287f)

10.1039/d2fo00287f

---

## Serotonin modulates insect gut bacterial community homeostasis.
 16.05.2022, _T Zeng, HA Su, YL Liu, JF Li, DX Jiang, YY Lu, YX Qi_


Metazoan guts are in permanent contact with microbial communities. However, the host mechanisms that have developed to manage the dynamic changes of these microorganisms and maintain homeostasis remain largely unknown.

[Source](https://doi.org/10.1186/s12915-022-01319-x)

10.1186/s12915-022-01319-x

---

## Natural variation in root exudation of GABA and DIMBOA impacts the maize root endosphere and rhizosphere microbiomes.
 13.05.2022, _P Wang, LD Lopes, MG Lopez-Guerrero, K van Dijk, S Alvarez, JJ Riethoven, DP Schachtman_


Root exudates are important for shaping root-associated microbiomes. However, studies on a wider range of metabolites in exudates are required for a comprehensive understanding about their influence on microbial communities. We identified maize inbred lines that differ in exudate concentrations of DIMBOA and GABA using a semi-hydroponic system. These lines were grown in the field to determine the changes in microbial diversity and gene expression due to varying concentrations of DIMBOA and GABA in exudates using 16S rRNA amplicon sequencing and metatranscriptomics. Results showed individual and interaction effects of DIMBOA and GABA on the rhizosphere and root endosphere β-diversity, most strongly at the V10 growth stage. The main bacterial families affected by both compounds were Ktedonobacteraceae and Xanthomonadaceae. Higher concentrations of DIMBOA in exudates affected the rhizosphere metatranscriptome, enriching for KEGG pathways associated with plant disease. This study validated the use of natural variation within plant species as a powerful approach for understanding the role of root exudates on microbiome selection. We also showed that a semi-hydroponic system can be used to identify maize genotypes that differ in GABA and DIMBOA exudate concentrations under field conditions. The impact of GABA exudation on root-associated microbiomes was shown for the first time.

[Source](https://doi.org/10.1093/jxb/erac202)

10.1093/jxb/erac202

---

## Beyond the snapshot: identification of the timeless, enduring indicator microbiome informing soil fertility and crop production in alkaline soils.
 16.05.2022, _J Zhang, J Dolfing, W Liu, R Chen, J Zhang, X Lin, Y Feng_


Microorganisms are known to be important drivers of biogeochemical cycling in soil and hence could act as a proxy informing on soil conditions in ecosystems. Identifying microbiomes indicative for soil fertility and crop production is important for the development of the next generation of sustainable agriculture. Earlier researches based on one-time sampling have revealed various indicator microbiomes for distinct agroecosystems and agricultural practices as well as their importance in supporting sustainable productivity. However, these microbiomes were based on a mere snapshot of a dynamic microbial community which is subject to significant changes over time. Currently true indicator microbiomes based on long-term, multi-annual monitoring are not available.

[Source](https://doi.org/10.1186/s40793-022-00420-6)

10.1186/s40793-022-00420-6

---

## The role of microbiome in carbon sequestration and environment security during wastewater treatment.
 16.05.2022, _X Zhu, C Lei, J Qi, G Zhen, X Lu, S Xu, J Zhang, H Liu, X Zhang, Z Wu_


Wastewater treatment is an essential aspect of the earth's sustainable future. However, different wastewater treatment methods are responsible for carbon discharge into the environment, raising environmental risks. Hence, such wastewater treatment methods are required that can minimize carbon release without compromising the treatment quality. Microbiome-based carbon sequestration is a potential method for achieving this goal. Limited studies have been carried out to investigate how microbes can capture and utilize CO2. This review summarizes the approaches including microbial electrolytic carbon capture, microbial electrosynthesis, microbial fuel cell, microalgae cultivation, and constructed wetlands that employ microbes to capture and utilize CO2. Electroactive Bacteria (EAB) convert carbon dioxide to carbonates and bicarbonates in subsequent steps after organic matter decomposition. Similarly, microbial electrosynthesis (MES) not only helps capture carbon but also produces secondary products (production of polyhydroxyalkanoates by Gram-negative rod Aeromonas hydrophila bacteria) of commercial importance during wastewater treatment. In addition to this, microbial carbon capture cells (MCCs) have been now utilized for energy generation and carbon sequestration at the same time during wastewater treatment. Moreover, microalgae cultivation has also been found to capture CO2 at a rapid pace while releasing O2 as a consequence of photosynthesis. Hence, microbe-based wastewater treatment has quite a potential due to two-fold benefits like carbon sequestration and by-product formation.

[Source](https://doi.org/10.1016/j.scitotenv.2022.155793)

10.1016/j.scitotenv.2022.155793

---

## Variational Approximation-Based Model Selection for Microbial Network Inference.
 13.05.2022, _S Yooseph, S Tavakoli_


Microbial associations are characterized by both direct and indirect interactions between the constituent taxa in a microbial community, and play an important role in determining the structure, organization, and function of the community. Microbial associations can be represented using a weighted graph (microbial network), whose nodes represent taxa and edges represent pairwise associations. A microbial network is typically inferred from a sample-taxa matrix that is obtained by sequencing multiple biological samples and identifying the taxa counts in each sample. However, it is known that microbial associations are impacted by environmental and/or host factors. Thus, a sample-taxa matrix generated in a microbiome study involving a wide range of values for the environmental and/or clinical metadata variables may in fact be associated with more than one microbial network. In this study, we consider the problem of inferring multiple microbial networks from a given sample-taxa count matrix. Each sample is a count vector assumed to be generated by a mixture model consisting of component distributions that are multivariate Poisson log-normal. We present a variational expectation maximization algorithm for the model selection problem to infer the correct number of components of this mixture model. Our approach involves reframing the mixture model as a latent variable model, treating only the mixing coefficients as parameters, and subsequently approximating the marginal likelihood using an evidence lower bound framework. Our algorithm is evaluated on a large simulated dataset generated using a collection of different graph structures (band, hub, cluster, random, and scale-free).

[Source](https://doi.org/10.1089/cmb.2021.0595)

10.1089/cmb.2021.0595

---

## Opportunities and challenges of using metagenomic data to bring uncultured microbes into cultivation.
 16.05.2022, _S Liu, CD Moon, N Zheng, S Huws, S Zhao, J Wang_


Although there is now an extensive understanding of the diversity of microbial life on earth through culture-independent metagenomic DNA sequence analyses, the isolation and cultivation of microbes remains critical to directly study them and confirm their metabolic and physiological functions, and their ecological roles. The majority of environmental microbes are as yet uncultured however; therefore, bringing these rare or poorly characterized groups into culture is a priority to further understand microbiome functions. Moreover, cultivated isolates may find utility in a range of applications, such as new probiotics, biocontrol agents, and agents for industrial processes. The growing abundance of metagenomic and meta-transcriptomic sequence information from a wide range of environments provides more opportunities to guide the isolation and cultivation of microbes of interest. In this paper, we discuss a range of successful methodologies and applications that have underpinned recent metagenome-guided isolation and cultivation of microbe efforts. These approaches include determining specific culture conditions to enrich for taxa of interest, to more complex strategies that specifically target the capture of microbial species through antibody engineering and genome editing strategies. With the greater degree of genomic information now available from uncultivated members, such as via metagenome-assembled genomes, the theoretical understanding of their cultivation requirements will enable greater possibilities to capture these and ultimately gain a more comprehensive understanding of the microbiomes. Video Abstract.

[Source](https://doi.org/10.1186/s40168-022-01272-5)

10.1186/s40168-022-01272-5

---

