## Dysbiosis of a microbiota-immune metasystem in critical illness is associated with nosocomial infections.
 09.03.2023, _J Schlechte, AZ Zucoloto, IL Yu, CJ Doig, MJ Dunbar, KD McCoy, B McDonald_


Critically ill patients in intensive care units experience profound alterations of their gut microbiota that have been linked to a high risk of hospital-acquired (nosocomial) infections and adverse outcomes through unclear mechanisms. Abundant mouse and limited human data suggest that the gut microbiota can contribute to maintenance of systemic immune homeostasis, and that intestinal dysbiosis may lead to defects in immune defense against infections. Here we use integrated systems-level analyses of fecal microbiota dynamics in rectal swabs and single-cell profiling of systemic immune and inflammatory responses in a prospective longitudinal cohort study of critically ill patients to show that the gut microbiota and systemic immunity function as an integrated metasystem, where intestinal dysbiosis is coupled to impaired host defense and increased frequency of nosocomial infections. Longitudinal microbiota analysis by 16s rRNA gene sequencing of rectal swabs and single-cell profiling of blood using mass cytometry revealed that microbiota and immune dynamics during acute critical illness were highly interconnected and dominated by Enterobacteriaceae enrichment, dysregulated myeloid cell responses and amplified systemic inflammation, with a lesser impact on adaptive mechanisms of host defense. Intestinal Enterobacteriaceae enrichment was coupled with impaired innate antimicrobial effector responses, including hypofunctional and immature neutrophils and was associated with an increased risk of infections by various bacterial and fungal pathogens. Collectively, our findings suggest that dysbiosis of an interconnected metasystem between the gut microbiota and systemic immune response may drive impaired host defense and susceptibility to nosocomial infections in critical illness.

[Source](https://doi.org/10.1038/s41591-023-02243-5)

10.1038/s41591-023-02243-5

---

## Altered infective competence of the human gut microbiome in COVID-19 

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01472-7)

---

## Impact of high altitude on composition and functional profiling of oral microbiome in Indian male population

[Source](https://doi.org/10.1038/s41598-023-30963-8)

---

## Incidence of Breast Cancer and <em>Enterococcus</em> Infection: A Retrospective Analysis

[Source](https://doi.org/10.14740/wjon1551)

---

## Multi-omics studies in interpreting the evolving standard model for immune functions

[Source](https://doi.org/10.1093/bfgp/elad003)

---

## Understanding the bacterial compositional network associations between oral and gut microbiome within healthy Koreans

[Source](https://doi.org/10.1080/20002297.2023.2186591)

---

## A transcriptomic insight into the human sperm microbiome through next-generation sequencing.
 10.03.2023, _C Corral-Vazquez, J Blanco, R Aiese Cigliano, S Zaida, F Vidal, E Anton_


The purpose of this study is to provide novel information through Next Generation Sequencing (NGS) for the characterization of viral and bacterial RNA cargo of human sperm cells from healthy fertile donors. For this, RNA-seq raw data of poly(A) RNA from 12 sperm samples from fertile donors were aligned to microbiome databases using the GAIA software. Species of viruses and bacteria were quantified in Operational Taxonomic Units (OTU) and filtered by minimal expression level (&gt;1% OTU in at least one sample). Mean expression values (and their standard deviation) of each species were estimated. A Hierarchical Cluster Analysis (HCA) and a Principal Component Analysis (PCA) were performed to detect common microbiome patterns among samples. Sixteen microbiome species, families, domains, and orders surpassed the established expression threshold. Of the 16 categories, nine corresponded to viruses (23.07% OTU) and seven to bacteria (2.77% OTU), among which the Herperviriales order and Escherichia coli were the most abundant, respectively. HCA and PCA displayed four clusters of samples with a differentiated microbiome fingerprint. This work represents a pilot study into the viruses and bacteria that make up the human sperm microbiome. Despite the high variability observed, some patterns of similarity among individuals were identified. Further NGS studies under standardized methodological procedures are necessary to achieve a deep knowledge of the semen microbiome and its implications in male fertility.

[Source](https://doi.org/10.1080/19396368.2023.2183912)

10.1080/19396368.2023.2183912

---

## Metformin alleviates liver fibrosis in mice by enriching <em>Lactobacillus</em> sp. MF-1 in the gut microbiota

[Source](https://doi.org/10.1016/j.bbadis.2023.166664)

---

## Microbiota alters the metabolome in an age- and sex- dependent manner in mice

[Source](https://doi.org/10.1038/s41467-023-37055-1)

---

## IL-22 alters gut microbiota composition and function to increase aryl hydrocarbon receptor activity in mice and humans

[Source](https://doi.org/10.1186/s40168-023-01486-1)

---

## Decreased Paneth cell Î±-defensins promote fibrosis in a choline-deficient L-amino acid-defined high-fat diet-induced mouse model of nonalcoholic steatohepatitis via disrupting intestinal microbiota

[Source](https://doi.org/10.1038/s41598-023-30997-y)

---

## Microbiota and functional analyses of nitrogen-fixing bacteria in root-knot nematode parasitism of plants

[Source](https://doi.org/10.1186/s40168-023-01484-3)

---

## Non-volatile metabolites mediating plant interactions with insect herbivores

[Source](https://doi.org/10.1111/tpj.16180)

---

## Comparative metagenomic analysis reveals rhizosphere microbial community composition and functions help protect grapevines against salt stress

[Source](https://doi.org/10.3389/fmicb.2023.1102547)

---

## Emerging Pathways for Engineering the Rhizosphere Microbiome for Optimal Plant Health

[Source](https://doi.org/10.1021/acs.jafc.2c08758)

---

## A bayesian zero-inflated dirichlet-multinomial regression model for multivariate compositional count data

[Source](https://doi.org/10.1111/biom.13853)

---

## Determining the most accurate 16S rRNA hypervariable region for taxonomic identification from respiratory samples

[Source](https://doi.org/10.1038/s41598-023-30764-z)

---

