## Whole genome sequencing and gene sharing network analysis powered by machine learning identifies antibiotic resistance sharing between animals, humans and environment in livestock farming.
 25.03.2022, _Z Peng, A Maciel-Guerra, M Baker, X Zhang, Y Hu, W Wang, J Rong, J Zhang, N Xue, P Barrow, D Renney, D Stekel, P Williams, L Liu, J Chen, F Li, T Dottorini_


Anthropogenic environments such as those created by intensive farming of livestock, have been proposed to provide ideal selection pressure for the emergence of antimicrobial-resistant Escherichia coli bacteria and antimicrobial resistance genes (ARGs) and spread to humans. Here, we performed a longitudinal study in a large-scale commercial poultry farm in China, collecting E. coli isolates from both farm and slaughterhouse; targeting animals, carcasses, workers and their households and environment. By using whole-genome phylogenetic analysis and network analysis based on single nucleotide polymorphisms (SNPs), we found highly interrelated non-pathogenic and pathogenic E. coli strains with phylogenetic intermixing, and a high prevalence of shared multidrug resistance profiles amongst livestock, human and environment. Through an original data processing pipeline which combines omics, machine learning, gene sharing network and mobile genetic elements analysis, we investigated the resistance to 26 different antimicrobials and identified 361 genes associated to AR phenotypes; 58 of these were known AMR-associated genes and 35 were associated to multidrug resistance. We uncovered an extensive network of genes, correlated to AMR phenotypes, shared among livestock, humans, farm and slaughterhouse environments. We also found several human, livestock and environmental isolates sharing closely related mobile genetic elements carrying AR genes across host species and environments. In a scenario where no consensus exists on how antibiotic use in the livestock may affect antibiotic resistance in the human population, our findings provide novel insights into the broader epidemiology of antimicrobial resistance in livestock farming. Moreover, our original data analysis method has the potential to uncover AR transmission pathways when applied to the study of other pathogens active in other anthropogenic environments characterised by complex interconnections between host species.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010018)

10.1371/journal.pcbi.1010018

---

## Superfast Evolution of Multi-drug Antibiotic Resistance

[Source](https://www.biorxiv.org/content/10.1101/2022.03.29.486198v1.abstract)

---

## Identification of the specific microbial community compositions in saliva associated with periodontitis during pregnancy.
 30.03.2022, _Y Narita, H Kodama_


To identify the specific microbial community compositions in saliva associated with periodontitis during pregnancy.

[Source](https://link.springer.com/article/10.1007/s00784-022-04468-z)

10.1007/s00784-022-04468-z

---

## Impact of Topical Interventions on the Vaginal Microbiota and Metabolome in Postmenopausal Women: A Secondary Analysis of a Randomized Clinical Trial.
 01.04.2022, _S Srinivasan, X Hua, MC Wu, S Proll, DJ Valint, SD Reed, KA Guthrie, AZ LaCroix, JC Larson, R Pepin, S Bhasin, D Raftery, DN Fredricks, CM Mitchell_


_Double-Blind Method, Female, Humans, Metabolome, Microbiota, Middle Aged, Postmenopause, Vagina_
Postmenopausal women with genitourinary symptoms of menopause are often prescribed vaginal estradiol or moisturizer for symptom improvement, but the impact of these treatments on the local microenvironment is poorly understood.

[Source](https://jamanetwork.com/journals/jamanetworkopen/fullarticle/2790503)

10.1001/jamanetworkopen.2022.5032

---

## Impact of Topical Interventions on the Vaginal Microbiota and Metabolome in Postmenopausal Women

[Source](https://jamanetwork.com/journals/jamanetworkopen/fullarticle/2790503)

---

## The interplay between anticancer challenges and the microbial communities from the gut.
 30.03.2022, _CA Hobson, S Bonacorsi, A Baruchel, O Tenaillon, A Birgy_


Cancer being an increasing burden on human health, the use of anticancer drugs has risen over the last decades. The physiological effects of these drugs are not only perceived by the host's cells but also by the microbial cells it harbors as commensals, notably the gut microbiota. Since the early '50 s, the cytotoxicity of anticancer chemotherapy was evaluated on bacteria revealing some antimicrobial activities that result in an established perturbation of the gut microbiota. This perturbation can affect the host's health through dysbiosis, which can lead to multiple complications, but has also been shown to have a direct effect on the treatment efficiency.We, therefore, conducted a review of literature focusing on this triangular relationship involving the microbial communities from the gut, the host's disease, and the anticancer treatment. We focused specifically on the antimicrobial effects of anticancer chemotherapy, their impact on mutagenesis in bacteria, and the perspectives of using bacteria-based tools to help in the diagnostic and treatment of cancer.

[Source](https://link.springer.com/article/10.1007/s10096-022-04435-2)

10.1007/s10096-022-04435-2

---

## Unraveling disease pathways involving the gut microbiota: the need for deep phenotyping and longitudinal data.
 30.03.2022, _KA Meyer, A Memili, DR Jacobs, P Gordon-Larsen_



[Source](https://academic.oup.com/ajcn/advance-article/doi/10.1093/ajcn/nqac052/6556172)

10.1093/ajcn/nqac052

---

## Anesthesia and Oncology: Friend or Foe?
 02.04.2022, _BS Buddeberg, MD Seeberger_


Cancer is a leading cause of death, and surgery is an important treatment modality. Laboratory research and retrospective studies have raised the suspicion that the choice of anesthetics for cancer surgery might affect the course of cancerous disease. The aim of this review is to provide a critical overview of the current state of knowledge. Inhalational anesthesia with volatiles or total intravenous anesthesia (TIVA) with propofol are the two most commonly used anesthetic techniques. Most data comparing volatile anesthetics with TIVA is from either in vitro or retrospective studies. Although conflicting, data shows a trend towards favoring propofol. Opioids are commonly used in anesthesia. Data on potential effects of opioids on growth and recurrence of cancer are scarce and conflicting. Preclinical studies have shown that opioids stimulate cancer growth through the Âµ-opioid receptor. Opioids also act as immunosuppressants and, therefore, have the potential to facilitate metastatic spread. However, the finding of an adverse effect of opioids on tumor growth and cancer recurrence by some retrospective studies has not been confirmed by prospective studies. Regional anesthesia has not been found to have a beneficial effect on the outcome of surgically treated cancer patients, but prospective studies are scarce. Local anesthetics might have a beneficial effect, as observed in animal and in vitro studies. However, prospective clinical studies strongly question such an effect. Blood products, which may be needed during extensive cancer surgery suppress the immune system, and data strongly suggest a negative impact on cancer recurrence. The potential effects of other commonly used anesthetic agents on the outcome of cancer patients have not been sufficiently studied for drawing valid conclusions. In conclusion, laboratory data and most retrospective studies suggest a potential advantage of TIVA over inhalational anesthesia on the outcome of surgical cancer patients, but prospective, randomized studies are missing. Given the state of weak scientific evidence, TIVA may be used as the preferred type of anesthesia unless there is an individual contraindication against it. Studies on the effects of other drugs frequently used in anesthesia are limited in number and quality, and have found conflicting results.

[Source](https://www.mdpi.com/2076-2607/10/4/744/htm)

10.3389/fonc.2022.802210

---

## The Effects of Stevia Consumption on Gut Bacteria: Friend or Foe?

[Source](https://www.mdpi.com/2076-2607/10/4/744/htm)

---

## Jejunoileal mucosal growth in mice with a limited microbiome.
 01.04.2022, _MP Shaughnessy, CJ Park, PS Salvi, RA Cowles_


Previous work demonstrated enhanced enterocyte proliferation and mucosal growth in gnotobiotic mice, suggesting that intestinal flora participate in mucosal homeostasis. Furthermore, broad-spectrum enteral antibiotics are known to induce near germ-free (GF) conditions in mice with conventional flora (CONV). We hypothesized that inducing near GF conditions with broad-spectrum enteral antibiotics would cause ordered small intestinal mucosal growth in CONV mice but would have no effect in GF mice with no inherent microbiome. C57BL/6J CONV and GF mice received either an antibiotic solution (Ampicillin, Ciprofloxacin, Metronidazole, Vancomycin, Meropenem) or a vehicle alone. After treatment, small intestinal villus height (VH), crypt depth (CD), mucosal surface area (MSA), crypt proliferation index (CPI), apoptosis, and villus and crypt cell types were assessed. Antibiotic-treated CONV (Abx-CONV) mice had taller villi, deeper crypts, increased CPI, increased apoptosis, and greater MSA compared to vehicle-treated CONV mice. Minor differences were noted in enterocyte and enterochromaffin cell proportions between groups, but goblet and Paneth cell proportions were unchanged in Abx-CONV mice compared to vehicle-treated CONV mice (p&gt;0.05). Antibiotics caused no significant changes in VH or MSA in GF mice when compared to vehicle-treated GF mice (p&gt;0.05). Enteral administration of broad-spectrum antibiotics to mice with a conventional microbiome stimulates ordered small intestinal mucosal growth. Mucosal growth was not seen in germ-free mice treated with antibiotics, implying that intestinal mucosal growth is associated with change in the microbiome in this model.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0266251)

10.1371/journal.pone.0266251

---

## Reproducible Propagation of Species-Rich Soil Bacterial Communities Suggests Robust Underlying Deterministic Principles of Community Formation.
 30.03.2022, _S ÄauÅ¡eviÄ, J Tackmann, V Sentchilo, C von Mering, JR van der Meer_


Microbiomes are typically characterized by high species diversity but it is poorly understood how such system-level complexity can be generated and propagated. Here, we used soil microcosms as a model to study development of bacterial communities as a function of their starting complexity and environmental boundary conditions. Despite inherent stochastic variation in manipulating species-rich communities, both laboratory-mixed medium complexity (21 soil bacterial isolates in equal proportions) and high-diversity natural top-soil communities followed highly reproducible succession paths, maintaining 16S rRNA gene amplicon signatures prominent for known soil communities in general. Development trajectories and compositional states were different for communities propagated in soil microcosms than in liquid suspension. Compositional states were maintained over multiple renewed growth cycles but could be diverged by short-term pollutant exposure. The different but robust trajectories demonstrated that deterministic taxa-inherent characteristics underlie reproducible development and self-organized complexity of soil microbiomes within their environmental boundary conditions. Our findings also have direct implications for potential strategies to achieve controlled restoration of desertified land. IMPORTANCE There is now a great awareness of the high diversity of most environmental ("free-living") and host-associated microbiomes, but exactly how diverse microbial communities form and maintain is still highly debated. A variety of theories have been put forward, but testing them has been problematic because most studies have been based on synthetic communities that fail to accurately mimic the natural composition (i.e., the species used are typically not found together in the same environment), the diversity (usually too low to be representative), or the environmental system itself (using designs with single carbon sources or solely mixed liquid cultures). In this study, we show how species-diverse soil bacterial communities can reproducibly be generated, propagated, and maintained, either from individual isolates (21 soil bacterial strains) or from natural microbial mixtures washed from top-soil. The high replicate consistency we achieve both in terms of species compositions and developmental trajectories demonstrates the strong inherent deterministic factors driving community formation from their species composition. Generating complex soil microbiomes may provide ways for restoration of damaged soils that are prevalent on our planet.

[Source](https://journals.asm.org/doi/10.1128/msystems.00160-22)

10.1128/msystems.00160-22

---

## A quantitative microbial risk assessment for touchscreen user interfaces using an asymmetric transfer gradient transmission mode.
 29.03.2022, _A Di Battista_


The ubiquitous use of public touchscreen user interfaces for commercial applications has created a credible risk for fomite-mediated disease transmission. This paper presents results from a stochastic simulation designed to assess this risk. The model incorporates a queueing network to simulate people flow and touchscreen interactions. It also describes an updated model for microbial transmission using an asymmetric gradient transfer assumption that incorporates literature reviewed empirical data concerning touch-transfer efficiency between fingers and surfaces. In addition to natural decay/die-off, pathogens are removed from the system by simulated cleaning / disinfection and personal-touching rates (e.g. face, dermal, hair and clothing). The dose response is implemented with an exponential moving average filter to model the temporal dynamics of exposure. Public touchscreens were shown to pose a considerable infection risk (â¼3%) using plausible default simulation parameters. Sensitivity of key model parameters, including the rate of surface disinfection is examined and discussed. A distinctive and important advancement of this simulation was its ability to distinguish between infection risk from a primary contaminated source and that due to the re-deposition of pathogens onto secondary, initially uncontaminated touchscreens from sequential use. The simulator is easily configurable and readily adapted to more general fomite-mediated transmission modelling and may provide a valuable framework for future research.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0265565)

10.1371/journal.pone.0265565

---

## Postmortem Skeletal Microbial Community Composition and Function in Buried Human Remains.
 30.03.2022, _AL Emmons, AZ Mundorff, KM Hoeland, J Davoren, SW Keenan, DO Carter, SR Campagna, JM DeBruyn_


Bones and teeth can provide a lasting resource to identify human remains following decomposition. Bone can support dynamic communities of micro- and macroscopic scavengers and incidental taxa, which influence the preservation of bone over time. Previously we identified key microbial taxa associated with survivability of DNA in bones of surface-decomposed human remains, observing high intra- and interindividual variation. Here we characterized the postmortem bone microbiome of skeletal remains in a multi-individual burial to better understand subsurface bone colonization and preservation. To understand microbial community origins and assembly, 16S rRNA amplicon sequences from 256 bone and 27 soil samples were compared to bone from individuals who decomposed on the ground surface, and human gut sequences from the American Gut Project. Untargeted metabolomics was applied to a subset of 41 bone samples from buried remains to examine potential microbe-metabolite interactions and infer differences related to community functionality. Results show that postmortem bone microbial communities are distinct from those of the oxic surface soils and the human gut. Microbial communities from surface-deposited bone and shallow buried bone were more similar to those from soils, while bones recovered from saturated areas deeper in the grave showed increased similarity with human gut samples with higher representation of anaerobic taxa, suggesting that the depositional environment affected the established bone microbiome. Correlations between metabolites and microbes indicate that phosphate solubilization is likely an important mechanism of microbially mediated skeletal degradation. This research expands our knowledge of microbial bone colonizers, including colonizers important in a burial environment. IMPORTANCE Understanding the microbes that colonize and degrade bone has important implications for preservation of skeletal elements and identification of unknown human remains. Current research on the postmortem bone microbiome is limited and largely focuses on archaeological or marine contexts. Our research expands our understanding of bone microbiomes in buried remains by characterizing the taxonomic and metabolic diversity of microbes that are colonizing bone after a 4-year postmortem burial interval and examines the potential impact of microbial colonization on human skeletal DNA preservation. Our results indicate that the postmortem bone microbiome is distinct from the human gut and soil. Evidence from combined metabolomic and amplicon sequencing analysis suggests that Pseudomonas and phosphate solubilization likely play a role in skeletal degradation. This work provides important insight into the types and activities of microbes controlling the preservation of buried skeletal remains.

[Source](https://journals.asm.org/doi/10.1128/msystems.00041-22)

10.1128/msystems.00041-22

---

## Microbial Identification Using rRNA Operon Region: Database and Tool for Metataxonomics with Long-Read Sequence.
 30.03.2022, _D Seol, JS Lim, S Sung, YH Lee, M Jeong, S Cho, W Kwak, H Kim_


Recent development of long-read sequencing platforms has enabled researchers to explore bacterial community structure through analysis of full-length 16S rRNA gene (â¼1,500 bp) or 16S-ITS-23S rRNA operon region (â¼4,300 bp), resulting in higher taxonomic resolution than short-read sequencing platforms. Despite the potential of long-read sequencing in metagenomics, resources and protocols for this technology are scarce. Here, we describe MIrROR, the database and analysis tool for metataxonomics using the bacterial 16S-ITS-23S rRNA operon region. We collected 16S-ITS-23S rRNA operon sequences extracted from bacterial genomes from NCBI GenBank and performed curation. A total of 97,781 16S-ITS-23S rRNA operon sequences covering 9,485 species from 43,653 genomes were obtained. For user convenience, we provide an analysis tool based on a mapping strategy that can be used for taxonomic profiling with MIrROR database. To benchmark MIrROR, we compared performance against publicly available databases and tool with mock communities and simulated data sets. Our platform showed promising results in terms of the number of species covered and the accuracy of classification. To encourage active 16S-ITS-23S rRNA operon analysis in the field, BLAST function and taxonomic profiling results with 16S-ITS-23S rRNA operon studies, which have been reported as BioProject on NCBI are provided. MIrROR (http://mirror.egnome.co.kr/) will be a useful platform for researchers who want to perform high-resolution metagenome analysis with a cost-effective sequencer such as MinION from Oxford Nanopore Technologies. IMPORTANCE Metabarcoding is a powerful tool to investigate community diversity in an economic and efficient way by amplifying a specific gene marker region. With the advancement of long-read sequencing technologies, the field of metabarcoding has entered a new phase. The technologies have brought a need for development in several areas, including new markers that long-read can cover, database for the markers, tools that reflect long-read characteristics, and compatibility with downstream analysis tools. By constructing MIrROR, we met the need for a database and tools for the 16S-ITS-23S rRNA operon region, which has recently been shown to have sufficient resolution at the species level. Bacterial community analysis using the 16S-ITS-23S rRNA operon region with MIrROR will provide new insights from various research fields.

[Source](https://journals.asm.org/doi/10.1128/spectrum.02017-21)

10.1128/spectrum.02017-21

---

## Long-Term Incubation of Lake Water Enables Genomic Sampling of Consortia Involving Planctomycetes and Candidate Phyla Radiation Bacteria.
 30.03.2022, _AL Jaffe, M Fuster, MC Schoelmerich, LX Chen, J Colombet, H Billard, T Sime-Ngando, JF Banfield_


Microbial communities in lakes can profoundly impact biogeochemical processes through their individual activities and collective interactions. However, the complexity of these communities poses challenges, particularly for studying rare organisms such as Candidate Phyla Radiation bacteria (CPR) and enigmatic entities such as aster-like nanoparticles (ALNs). Here, a reactor was inoculated with water from Lake Fargette, France, and maintained under dark conditions at 4Â°C for 31 months and enriched for ALNs, diverse Planctomycetes, and CPR bacteria. We reconstructed draft genomes and predicted metabolic traits for 12 diverse Planctomycetes and 9 CPR bacteria, some of which are likely representatives of undescribed families or genera. One CPR genome representing the little-studied lineage "Candidatus Peribacter" was curated to completion (1.239 Mbp) and unexpectedly encodes the full gluconeogenesis pathway. Metatranscriptomic data indicate that some planctomycetes and CPR bacteria were active under the culture conditions, accounting for â¼30% and â¼1% of RNA reads mapping to the genome set, respectively. We also reconstructed genomes and obtained transmission electron microscope images for numerous viruses, including one with a &gt;300-kbp genome and several predicted to infect Planctomycetes. Together, our analyses suggest that freshwater Planctomycetes are central players in a subsystem that includes ALNs, symbiotic CPR bacteria, and viruses. IMPORTANCE Laboratory incubations of natural microbial communities can aid in the study of member organisms and their networks of interaction. This is particularly important for understudied lineages for which key elements of basic biology are still emerging. Using genomics and microscopy, we found that members of the bacterial lineage Planctomycetes may be central players in a subset of a freshwater lake microbiome that includes other bacteria, archaea, viruses, and mysterious entities, called aster-like nanoparticles (ALNs), whose origin is unknown. Our results help constrain the possible origins of ALNs and provide insight into possible interactions within a complex lake ecosystem.

[Source](https://journals.asm.org/doi/10.1128/msystems.00223-22)

10.1128/msystems.00223-22

---

