## Lactocaseibacillus rhamnosus zz-1 Supplementation Mitigates Depression-Like Symptoms in Chronic Stress-Induced Depressed Mice via the Microbiota-Gut-Brain Axis.
 22.02.2023, _J Xu, M Tang, D Wang, X Zhang, X Yang, Y Ma, X Xu_


Accumulating evidence has revealed an association between depression and disordered intestinal microecology. The discovery of psychobiotics has provided a promising perspective for studying the treatment of psychiatric disorders. Here, we aimed to investigate the antidepressant abilities of Lactocaseibacillus rhamnosus zz-1 (LRzz-1) and elucidate the underlying mechanisms. The viable bacteria (2 × 109 CFU/day) were orally supplemented to depressed C57BL/6 mice induced by chronic unpredictable mild stress (CUMS), and the behavioral, neurophysiological, and intestinal microbial effects were assessed, with fluoxetine used as a positive control. The treatment with LRzz-1 effectively mitigated the depression-like behavioral disorders of depressed mice and reduced the expression of inflammatory cytokine mRNA (IL-1β, IL-6, and TNF-α) in the hippocampus. In addition, LRzz-1 treatment also improved tryptophan metabolic disorder in the mouse hippocampus, as well as its peripheral circulation. These benefits are associated with the mediation of microbiome-gut-brain bidirectional communication. CUMS-induced depression impaired the intestinal barrier integrity and microbial homeostasis in mice, neither of which was restored by fluoxetine. LRzz-1 prevented intestinal leakage and significantly ameliorated epithelial barrier permeability by up-regulating tight-junction proteins (including ZO-1, occludin, and claudin-1). In particular, LRzz-1 improved the microecological balance by normalizing the threatened bacteria (e.g., Bacteroides and Desulfovibrio), exerting beneficial regulation (e.g., Ruminiclostridium 6 and Alispites), and modifying short-chain fatty acid metabolism. In summary, LRzz-1 showed considerable antidepressant-like effects and exhibited more comprehensive intestinal microecological regulation than other drugs, which offers novel insights that can facilitate the development of depression therapeutic strategies.

[Source](https://doi.org/10.1016/j.cels.2022.12.010)

10.1021/acschemneuro.2c00580

---

## Controlling the human microbiome

[Source](https://doi.org/10.1016/j.cels.2022.12.010)

---

## Metabolic models of human gut microbiota: Advances and challenges.
 22.02.2023, _D Rios Garza, D Gonze, H Zafeiropoulos, B Liu, K Faust_


_Humans, Gastrointestinal Microbiome, Ecosystem, Models, Theoretical_
The human gut is a complex ecosystem consisting of hundreds of microbial species interacting with each other and with the human host. Mathematical models of the gut microbiome integrate our knowledge of this system and help to formulate hypotheses to explain observations. The generalized Lotka-Volterra model has been widely used for this purpose, but it does not describe interaction mechanisms and thus does not account for metabolic flexibility. Recently, models that explicitly describe gut microbial metabolite production and consumption have become popular. These models have been used to investigate the factors that shape gut microbial composition and to link specific gut microorganisms to changes in metabolite concentrations found in diseases. Here, we review how such models are built and what we have learned so far from their application to human gut microbiome data. In addition, we discuss current challenges of these models and how these can be addressed in the future.

[Source](https://doi.org/10.1016/j.cels.2022.11.002)

10.1016/j.cels.2022.11.002

---

## The fecal microbiotas of women of Pacific and New Zealand European ethnicities are characterized by distinctive enterotypes that reflect dietary intakes and fecal water content.
 22.02.2023, _N Renall, B Lawley, T Vatanen, B Merz, J Douwes, M Corbin, L Te Morenga, R Kruger, BH Breier, GW Tannock_


_Humans, Female, Ethnicity, Diabetes Mellitus, Type 2, New Zealand, Gastrointestinal Microbiome, Feces, Microbiota, Obesity, Eating_
Obesity is a complex, multifactorial condition that is an important risk factor for noncommunicable diseases including cardiovascular disease and type 2 diabetes. While prevention and management require a healthy and energy balanced diet and adequate physical activity, the taxonomic composition and functional attributes of the colonic microbiota may have a supplementary role in the development of obesity. The taxonomic composition and metabolic capacity of the fecal microbiota of 286 women, resident in Auckland New Zealand, was determined by metagenomic analysis. Associations with BMI (obese, nonobese), body fat composition, and ethnicity (Pacific, n = 125; NZ European women [NZE], n = 161) were assessed using regression analyses. The fecal microbiotas were characterized by the presence of three distinctive enterotypes, with enterotype 1 represented in both Pacific and NZE women (39 and 61%, respectively), enterotype 2 mainly in Pacific women (84 and 16%) and enterotype 3 mainly in NZE women (13 and 87%). Enterotype 1 was characterized mainly by the relative abundances of butyrate producing species, Eubacterium rectale and Faecalibacterium prausnitzii, enterotype 2 by the relative abundances of lactic acid producing species, Bifidobacterium adolescentis, Bifidobacterium bifidum, and Lactobacillus ruminis, and enterotype 3 by the relative abundances of Subdoligranulum sp., Akkermansia muciniphila, Ruminococcus bromii, and Methanobrevibacter smithii. Enterotypes were also associated with BMI, visceral fat %, and blood cholesterol. Habitual food group intake was estimated using a 5 day nonconsecutive estimated food record and a 30 day, 220 item semi-quantitative Food Frequency Questionnaire. Higher intake of 'egg' and 'dairy' products was associated with enterotype 3, whereas 'non-starchy vegetables', 'nuts and seeds' and 'plant-based fats' were positively associated with enterotype 1. In contrast, these same food groups were inversely associated with enterotype 2. Fecal water content, as a proxy for stool consistency/colonic transit time, was associated with microbiota taxonomic composition and gene pools reflective of particular bacterial biochemical pathways. The fecal microbiotas of women of Pacific and New Zealand European ethnicities are characterized by distinctive enterotypes, most likely due to differential dietary intake and fecal consistency/colonic transit time. These parameters need to be considered in future analyses of human fecal microbiotas.

[Source](https://doi.org/10.1080/19490976.2023.2178801)

10.1080/19490976.2023.2178801

---

## Identifying the Novel Gut Microbial Metabolite Contributing to Metabolic Syndrome in Children Based on Integrative Analyses of Microbiome-Metabolome Signatures.
 16.02.2023, _J Wei, W Dai, X Pan, Y Zhong, N Xu, P Ye, J Wang, J Li, F Yang, J Luo, M Luo_


The pathogenesis of gut microbiota and their metabolites in the development of metabolic syndrome (MS) remains unclear. This study aimed to evaluate the signatures of gut microbiota and metabolites as well as their functions in obese children with MS. A case-control study was conducted based on 23 MS children and 31 obese controls. The gut microbiome and metabolome were measured using 16S rRNA gene amplicon sequencing and liquid chromatography-mass spectrometry. An integrative analysis was conducted, combining the results of the gut microbiome and metabolome with extensive clinical indicators. The biological functions of the candidate microbial metabolites were validated in vitro. We identified 9 microbiota and 26 metabolites that were significantly different from the MS and the control group. The clinical indicators of MS were correlated with the altered microbiota Lachnoclostridium, Dialister, and Bacteroides, as well as with the altered metabolites all-trans-13,14-dihydroretinol, DL-dipalmitoylphosphatidylcholine (DPPC), LPC 24: 1, PC (14:1e/10:0), and 4-phenyl-3-buten-2-one, etc. The association network analysis further identified three MS-linked metabolites, including all-trans-13,14-dihydroretinol, DPPC, and 4-phenyl-3-buten-2-one, that were significantly correlated with the altered microbiota. Bio-functional validation showed that all-trans-13, 14-dihydroretinol could significantly upregulate the expression of lipid synthesis genes and inflammatory genes. This study identified a new biomarker that may contribute to MS development. These findings provided new insights regarding the development of efficient therapeutic strategies for MS. IMPORTANCE Metabolic syndrome (MS) has become a health concern worldwide. Gut microbiota and metabolites play an important role in human health. We first endeavored to comprehensively analyze the microbiome and metabolome signatures in obese children and found the novel microbial metabolites in MS. We further validated the biological functions of the metabolites in vitro and illustrated the effects of the microbial metabolites on lipid synthesis and inflammation. The microbial metabolite all-trans-13, 14-dihydroretinol may be a new biomarker in the pathogenesis of MS, especially in obese children. These findings were not available in previous studies, and they provide new insights regarding the management of metabolic syndrome.

[Source](https://doi.org/10.1128/spectrum.03771-22)

10.1128/spectrum.03771-22

---

## Gut Microbiome and Risk of Ischemic Stroke: A Comprehensive Mendelian Randomization Study.
 17.02.2023, _C Meng, P Deng, R Miao, H Tang, Y Li, J Wang, J Wu, W Wang, S Liu, J Xia, Y Lu_


Increasing evidence implicates the microbiome as a susceptibility factor for ischemic stroke (IS). Interpretation of this evidence is difficult for the composition of the microbiome is influenced by various factors and might affect differently in IS subtypes. We aim to determine if the specific gut microbiome is causally associated with IS subtypes and suggest potential approaches for stroke prevention.

[Source](https://doi.org/10.1093/eurjpc/zwad052)

10.1093/eurjpc/zwad052

---

## Metagenomic analysis of Ancient Egyptian canopic jars.
 18.02.2023, _E Rayo, J Neukamm, N Tomoum, P Eppenberger, A Breidenstein, AS Bouwman, VJ Schuenemann, FJ Rühli_


_Humans, Egypt, DNA, Ancient, Mummies, Lung, Base Sequence_
Ancient Egyptian remains have been of interest for anthropological research for decades. Despite many investigations, the ritual vessels for the internal organs removed during body preparation-liver, lungs, stomach, and intestines, of Egyptian mummies are rarely used for palaeopathological or medical investigations. These artifacts, commonly referred to as canopic jars, are the perfect combination of cultural and biological material and present an untapped resource for both Egyptological and medical fields. Nevertheless, technical challenges associated with this archeological material have prevented the application of current ancient DNA techniques for both the characterization of human and pathogenic DNA. We present shotgun-sequenced metagenomic profiles and ancient DNA degradation patterns from multiple canopic jars sampled from several European museum collections and enumerate current limitations and possible solutions for the future analysis of similar material. This is the first-ever recorded evidence of ancient human DNA found in Ancient Egyptian canopic jars and the first associated metagenomic description of bacterial taxa in these funerary artifacts.

[Source](https://doi.org/10.1002/ajpa.24600)

10.1002/ajpa.24600

---

## Vaginal microbial dynamics and pathogen colonization in a humanized microbiota mouse model

[Source](https://doi.org/10.1101/2023.02.09.527909)

---

## Multidisciplinary evaluation of plant growth promoting rhizobacteria on soil microbiome and strawberry quality.
 20.02.2023, _JH Nam, A Thibodeau, YL Qian, MC Qian, SH Park_


The natural soil environment is considered one of the most diverse habitats containing numerous bacteria, fungi, and larger organisms such as nematodes, insects, or rodents. Rhizosphere bacteria play vital roles in plant nutrition and the growth promotion of their host plant. The aim of this study was to evaluate the effects of three plant growth-promoting rhizobacteria (PGPR), Bacillus subtilis, Bacillus amyloliquefaciens, and Pseudomonas monteilii for their potential role as a biofertilizer. The effect of the PGPR was examined at a commercial strawberry farm in Dayton, Oregon. The PGPR were applied to the soil of the strawberry (Fragaria × ananassa cultivar Hood) plants in two different concentrations of PGPR, T1 (0.24% PGPR) and T2 (0.48% PGPR), and C (no PGPR). A total of 450 samples from August 2020 to May 2021 were collected, and microbiome sequencing based on the V4 region of the 16S rRNA gene was conducted. The strawberry quality was measured by sensory evaluation, total acidity (TA), total soluble solids (TSS), color (lightness and chroma), and volatile compounds. Application of the PGPR significantly increased the populations of Bacillus and Pseudomonas and promoted the growth of nitrogen-fixing bacteria. The TSS and color evaluation showed that the PGPR presumptively behaved as a ripening enhancer. The PGPR contributed to the production of fruit-related volatile compounds, while the sensory evaluation did not show significant differences among the three groups. The major finding of this study suggests that the consortium of the three PGPR have a potential role as a biofertilizer by supporting the growth of other microorganisms (nitrogen-fixing bacteria) as part of a synergetic effect and strawberry quality such as sweetness and volatile compounds.

[Source](https://doi.org/10.1186/s13568-023-01524-z)

10.1186/s13568-023-01524-z

---

## Unravelling microalgal-bacterial interactions in aquatic ecosystems through 16S rRNA gene-based co-occurrence networks.
 22.02.2023, _BLDU Pushpakumara, K Tandon, A Willis, H Verbruggen_


_RNA, Ribosomal, 16S, Microalgae, Genes, rRNA, Bacteria, Microbiota_
Interactions between microalgae and bacteria can directly influence the global biogeochemical cycles but the majority of such interactions remain unknown. 16S rRNA gene-based co-occurrence networks have potential to help identify microalgal-bacterial interactions. Here, we used data from 10 Earth microbiome projects to identify potential microalgal-bacterial associations in aquatic ecosystems. A high degree of clustering was observed in microalgal-bacterial modules, indicating densely connected neighbourhoods. Proteobacteria and Bacteroidetes predominantly co-occurred with microalgae and represented hubs of most modules. Our results also indicated that species-specificity may be a global characteristic of microalgal associated microbiomes. Several previously known associations were recovered from our network modules, validating that biologically meaningful results can be inferred using this approach. A range of previously unknown associations were recognised such as co-occurrences of Bacillariophyta with uncultured Planctomycetes OM190 and Deltaproteobacteria order NB1-j. Planctomycetes and Verrucomicrobia were identified as key associates of microalgae due to their frequent co-occurrences with several microalgal taxa. Despite no clear taxonomic pattern, bacterial associates appeared functionally similar across different environments. To summarise, we demonstrated the potential of 16S rRNA gene-based co-occurrence networks as a hypothesis-generating framework to guide more focused research on microalgal-bacterial associations.

[Source](https://doi.org/10.1038/s41598-023-27816-9)

10.1038/s41598-023-27816-9

---

## gutSMASH predicts specialized primary metabolic pathways from the human gut microbiota.
 14.02.2023, _V Pascal Andreu, HE Augustijn, L Chen, A Zhernakova, J Fu, MA Fischbach, D Dodd, MH Medema_


The gut microbiota produce hundreds of small molecules, many of which modulate host physiology. Although efforts have been made to identify biosynthetic genes for secondary metabolites, the chemical output of the gut microbiome consists predominantly of primary metabolites. Here we introduce the gutSMASH algorithm for identification of primary metabolic gene clusters, and we used it to systematically profile gut microbiome metabolism, identifying 19,890 gene clusters in 4,240 high-quality microbial genomes. We found marked differences in pathway distribution among phyla, reflecting distinct strategies for energy capture. These data explain taxonomic differences in short-chain fatty acid production and suggest a characteristic metabolic niche for each taxon. Analysis of 1,135 individuals from a Dutch population-based cohort shows that the level of microbiome-derived metabolites in plasma and feces is almost completely uncorrelated with the metagenomic abundance of corresponding metabolic genes, indicating a crucial role for pathway-specific gene regulation and metabolite flux. This work is a starting point for understanding differences in how bacterial taxa contribute to the chemistry of the microbiome.

[Source](https://doi.org/10.1038/s41587-023-01675-1)

10.1038/s41587-023-01675-1

---

## Reliability of species detection in 16S microbiome analysis: Comparison of five widely used pipelines and recommendations for a more standardized approach.
 22.02.2023, _A Hiergeist, J Ruelle, S Emler, A Gessner_


_Reproducibility of Results, High-Throughput Nucleotide Sequencing, Microbiota, Bacteria, Base Sequence, RNA, Ribosomal, 16S, Sequence Analysis, DNA_
The use of NGS-based testing of the bacterial microbiota is often impeded by inconsistent or non-reproducible results, especially when applying different analysis pipelines and reference databases. We investigated five frequently used software packages by submitting the same monobacterial datasets to them, representing the V1-2 and the V3-4 regions of the 16S-rRNA gene of 26 well characterized strains, which were sequenced by the Ion Torrent™ GeneStudio S5 system. The results obtained were divergent and calculations of relative abundance did not yield the expected 100%. We investigated these inconsistencies and were able to attribute them to failures either of the pipelines themselves or of the reference databases they rely on. On the basis of these findings, we recommend certain standards which should help to render microbiome testing more consistent and reproducible, and thus useful in clinical practice.

[Source](https://doi.org/10.1371/journal.pone.0280870)

10.1371/journal.pone.0280870

---

## Effect of in vitro cultivation on human gut microbiota composition using 16S rDNA amplicon sequencing and metabolomics approach.
 22.02.2023, _P Średnicka, MŁ Roszko, D Popowski, M Kowalczyk, M Wójcicki, P Emanowicz, M Szczepańska, D Kotyrba, E Juszczuk-Kubiak_


Gut microbiota (GM) plays many key functions and helps maintain the host's health. Consequently, the development of GM cultivation under in vitro stimulating physiological conditions has gained extreme interest in different fields. In this study, we evaluated the impact of four culture media: Gut Microbiota Medium (GMM), Schaedler Broth (SM), Fermentation Medium (FM), and Carbohydrate Free Basal Medium (CFBM) on preserving the biodiversity and metabolic activity of human GM in batch in vitro cultures using PMA treatment coupled with 16S rDNA sequencing (PMA-seq) and LC-HR-MS/MS untargeted metabolomics supplemented with GC-MS SCFA profiling. Before the experiments, we determined the possibility of using the pooled faecal samples (MIX) from healthy donors (n = 15) as inoculum to reduce the number of variables and ensure the reproducibility of in vitro cultivation tests. Results showed the suitability of pooling faecal samples for in vitro cultivation study. Non-cultured MIX inoculum was characterized by higher α-diversity (Shannon effective count, and Effective microbial richness) compared to inocula from individual donors. After 24 h of cultivation, a significant effect of culture media composition on GM taxonomic and metabolomic profiles was observed. The SM and GMM had the highest α-diversity (Shannon effective count). The highest number of core ASVs (125) shared with non-cultured MIX inoculum and total SCFAs production was observed in the SM. These results might contribute to the development of standardized protocols for human GM in vitro cultivation by preventing methodological bias in the data.

[Source](https://doi.org/10.1101/2023.02.08.527766)

10.1038/s41598-023-29637-2

---

## Impact of experimental bias on compositional analysis of microbiome data

[Source](https://doi.org/10.1101/2023.02.08.527766)

---

