## Cross-talk between the microbiome and chronic inflammation in esophageal cancer: potential driver of oncogenesis.
 05.05.2022, _T Sharma, A Gupta, R Chauhan, AA Bhat, S Nisar, S Hashem, S Akhtar, A Ahmad, M Haris, M Singh, S Uddin_


Esophageal cancer (EC) is frequently considered a lethal malignancy and is often identified at a later stage. It is one of the major causes of cancer-related deaths globally. The conventional treatment methods like chemotherapy, radiotherapy, and surgery offer limited efficacy and poor clinical outcome with a less than 25% 5-year survival rate. The poor prognosis of EC persists despite the growth in the development of diagnostic and therapeutic modalities to treat EC. This underlines the need to elucidate the complex molecular mechanisms that drive esophageal oncogenesis. Apart from the role of the tumor microenvironment and its structural and cellular components in tumorigenesis, mounting evidence points towards the involvement of the esophageal microbiome, inflammation, and their cross-talk in promoting esophageal cancer. The current review summarizes recent research that delineates the underlying molecular mechanisms by which the microbiota and inflammation promote the pathophysiology of esophageal cancer, thus unraveling targets for potential therapeutic intervention.

[Source](https://link.springer.com/article/10.1007/s10555-022-10026-6)

10.1007/s10555-022-10026-6

---

## Systematic and quantitative view of the antiviral arsenal of prokaryotes.
 12.05.2022, _F Tesson, A Hervé, E Mordret, M Touchon, C d'Humières, J Cury, A Bernheim_


_Antiviral Agents, Archaea, Bacteria, Genomics, Prokaryotic Cells_
Bacteria and archaea have developed multiple antiviral mechanisms, and genomic evidence indicates that several of these antiviral systems co-occur in the same strain. Here, we introduce DefenseFinder, a tool that automatically detects known antiviral systems in prokaryotic genomes. We use DefenseFinder to analyse 21000 fully sequenced prokaryotic genomes, and find that antiviral strategies vary drastically between phyla, species and strains. Variations in composition of antiviral systems correlate with genome size, viral threat, and lifestyle traits. DefenseFinder will facilitate large-scale genomic analysis of antiviral defense systems and the study of host-virus interactions in prokaryotes.

[Source](https://www.nature.com/articles/s41467-022-30269-9)

10.1038/s41467-022-30269-9

---

## A randomized controlled trial of the effects of whole grains versus refined grains diets on the microbiome in pregnancy.
 10.05.2022, _H Sun, P Yamada, A Paetow, M Chan, A Arslan, R Landberg, MG Dominguez-Bello, BK Young_


_Diet, Dietary Carbohydrates, Female, Gestational Weight Gain, Humans, Infant, Newborn, Male, Microbiota, Pregnancy, Whole Grains_
Dietary whole grain consumption has been postulated to have metabolic benefits. The purpose of this study was to compare a pregnancy diet containing 75% of total carbohydrates as refined grains with a diet of 75% of total carbohydrates as whole grains for pregnancy outcomes and effects on the microbiome. Gestational weight gain, glucose tolerance and newborn outcomes were measured on 248 enrolled compliant women from whom a subset of 103 women consented to give 108 vaginal and 109 anal swabs. The data presented here are limited to the patients from whom the vaginal and anal swabs were obtained in order to study the microbiome. A microbiome-16SrRNA survey-was characterized in these samples. Samples and measurements were obtained at the first obstetrical visit, before beginning a prescribed diet (T1-baseline) and after 17-32 weeks on the prescribed diet (T3). Food frequency questionnaires and total plasma alkylresorcinols were used as a measure of whole grain consumption. There were no dietary differences in maternal weight gain, birth weight, or glucose tolerance test. Mothers consuming the whole grains diet showed a trend of gestational decrease in vaginal bacterial alpha diversity, with increasing Lactobacillus-dominance. No significant difference was observed for the anal microbiome. The results suggest that diet modulations of the vaginal microbiome during gestation may have important implications for maternal and neonatal health and in the intergenerational transfer of maternal microbiome. Trial registration: ClinicalTrials.gov Identifier: NCT03232762.

[Source](https://www.nature.com/articles/s41598-022-11571-4)

10.1038/s41598-022-11571-4

---

## Should we modulate the neonatal microbiome and what should be the goal?
 12.05.2022, _N van Best, MG Dominguez-Bello, MW Hornef, E Jašarević, K Korpela, TD Lawley_


_Gastrointestinal Microbiome, Goals, Microbiota_

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01281-4)

10.1186/s40168-022-01281-4

---

## Space travel and early childhood gut microbiome: is space dirty enough to raise a child?

[Source](https://www.futuremedicine.com/doi/10.2217/fmb-2022-0036)

---

## On-person adaptive evolution of Staphylococcus aureus during treatment for atopic dermatitis

[Source](https://www.biorxiv.org/content/10.1101/2021.03.24.436824v2.abstract)

---

## Oral microbiome of Crohn’s Disease patients with and without oral manifestations

[Source](https://academic.oup.com/ecco-jcc/advance-article/doi/10.1093/ecco-jcc/jjac063/6577053)

---

## Integrating the salivary microbiome in the forensic toolkit by 16S rRNA gene: potential application in body fluid identification and biogeographic inference

[Source](https://link.springer.com/article/10.1007/s00414-022-02831-z)

---

## An approach for evaluating the effects of dietary fiber polysaccharides on the human gut microbiome and plasma proteome

[Source](https://www.pnas.org/doi/full/10.1073/pnas.2123411119)

---

## A randomization-based causal inference framework for uncovering environmental exposure effects on human gut microbiota

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010044)

---

## Microbiota and gastric cancer

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1044579X22001043)

---

## Host—microbial interactions in metabolic diseases: from diet to immunity

[Source](https://link.springer.com/article/10.1007/s12275-022-2087-y)

---

## A randomized controlled trial for response of microbiome network to exercise and diet intervention in patients with nonalcoholic fatty liver disease

[Source](https://www.nature.com/articles/s41467-022-29968-0)

---

## Sex-dependent host-microbiome dynamics in zebrafish: Implications for toxicology and gastrointestinal physiology

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1744117X22000351)

---

## Comprehensive 16S rRNA and metagenomic data from the gut microbiome of aging and rejuvenation mouse models

[Source](https://www.nature.com/articles/s41597-022-01308-3)

---

## A total infectome approach to understand the etiology of infectious disease in pigs

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01265-4)

---

## Synergistic interaction of gut microbiota enhances the growth of nematode through neuroendocrine signaling

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0960982222004900)

---

## Unraveling global and diazotrophic bacteriomes of boreal forest floor feather mosses and their environmental drivers at the ecosystem and at the plant scale in North America.
 09.05.2022, _M Renaudin, I Laforest-Lapointe, JP Bellenger_


Feather mosses are abundant cryptogams of the boreal forest floor and shelter a broad diversity of bacteria who have important ecological functions (e.g., decomposition, nutrient cycling). In particular, nitrogen (N2-) fixation performed by feather moss-associated diazotrophs constitutes an important entry of nitrogen in the boreal forest ecosystem. However, the composition of the feather moss bacteriome and its environmental drivers are still unclear. Using cDNA amplicon sequencing of the 16S rRNA and nifH genes and cyanobacterial biomass quantification, we explored the active global and diazotrophic bacterial communities of two dominant feather moss species (i) at the ecosystem scale, along a 500-km climatic and nutrient deposition gradient in the North American boreal forest, and (ii) at the plant scale, along the moss shoot senescence gradient. We found that cyanobacteria were major actors of the feather moss bacteriome, accounting for 33% of global bacterial communities and 65% of diazotrophic communities, and that several cyanobacterial and methanotrophic genera were contributing to N2-fixation. Moreover, we showed that bacteria were occupying ecological niches along the moss shoot, with phototrophs being dominant in the apical part and methanotrophs being dominant in the basal part. Finally, climate (temperature, precipitation), environmental variables (moss species, month, tree density) and nutrients (nitrogen, phosphorus, molybdenum, vanadium, iron) strongly shaped global and diazotrophic bacteriomes. In summary, this work presents evidence that the feather moss bacteriome plays crucial roles in supporting moss growth, health, and decomposition, as well as in the boreal forest carbon and nitrogen cycles. This study also highlights the substantial effects of climate and nutrients on the feather moss bacteriome, suggesting the importance of understanding the impacts of global change on moss-associated bacterial growth and activity.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969722028583)

10.1016/j.scitotenv.2022.155761

---

## Antimicrobial resistance in Antarctica: is it still a pristine environment?

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01250-x)

---

## Evaluating metagenomic assembly approaches for biome-specific gene catalogues.
 10.05.2022, _LF Delgado, AF Andersson_


_Algorithms, Cluster Analysis, Ecosystem, Metagenome, Metagenomics_
For many environments, biome-specific microbial gene catalogues are being recovered using shotgun metagenomics followed by assembly and gene calling on the assembled contigs. The assembly is typically conducted either by individually assembling each sample or by co-assembling reads from all the samples. The co-assembly approach can potentially recover genes that display too low abundance to be assembled from individual samples. On the other hand, combining samples increases the risk of mixing data from closely related strains, which can hamper the assembly process. In this respect, assembly on individual samples followed by clustering of (near) identical genes is preferable. Thus, both approaches have potential pros and cons, but it remains to be evaluated which assembly strategy is most effective. Here, we have evaluated three assembly strategies for generating gene catalogues from metagenomes using a dataset of 124 samples from the Baltic Sea: (1) assembly on individual samples followed by clustering of the resulting genes, (2) co-assembly on all samples, and (3) mix assembly, combining individual and co-assembly.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01259-2)

10.1186/s40168-022-01259-2

---

## Compare the performance of multiple binary classification models in microbial high-throughput sequencing datasets.
 10.05.2022, _N Xu, Z Zhang, Y Shen, Q Zhang, Z Liu, Y Yu, Y Wang, C Lei, M Ke, D Qiu, T Lu, Y Chen, J Xiong, H Qian_


The development of machine learning and deep learning provided solutions for predicting microbiota response on environmental change based on microbial high-throughput sequencing. However, there were few studies specifically clarifying the performance and practical of two types of binary classification models to find a better algorithm for the microbiota data analysis. Here, for the first time, we evaluated the performance, accuracy and running time of the binary classification models built by three machine learning methods - random forest (RF), support vector machine (SVM), logistic regression (LR), and one deep learning method - back propagation neural network (BPNN). The built models were based on the microbiota datasets that removed low-quality variables and solved the class imbalance problem. Additionally, we optimized the models by tuning. Our study demonstrated that dataset pre-processing was a necessary process for model construction. Among these 4 binary classification models, BPNN and RF were the most suitable methods for constructing microbiota binary classification models. Using these 4 models to predict multiple microbial datasets, BPNN showed the highest accuracy and the most robust performance, while the RF method was ranked second. We also constructed the optimal models by adjusting the epochs of BPNN and the n_estimators of RF for six times. The evaluation related to performances of models provided a road map for the application of artificial intelligence to assess microbial ecology.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969722029047)

10.1016/j.scitotenv.2022.155807

---

