## RNA-based amplicon sequencing is ineffective in measuring metabolic activity in environmental microbial communities.
 22.06.2023, _Y Wang, KN Thompson, Y Yan, MI Short, Y Zhang, EA Franzosa, J Shen, EM Hartmann, C Huttenhower_


_Humans, Escherichia coli, RNA, Ribosomal, 16S, Gene Library, Hot Temperature, Microbiota_
Characterization of microbial activity is essential to the understanding of the basic biology of microbial communities, as the function of a microbiome is defined by its biochemically active ("viable") community members. Current sequence-based technologies can rarely differentiate microbial activity, due to their inability to distinguish live and dead sourced DNA. As a result, our understanding of microbial community structures and the potential mechanisms of transmission between humans and our surrounding environments remains incomplete. As a potential solution, 16S rRNA transcript-based amplicon sequencing (16S-RNA-seq) has been proposed as a reliable methodology to characterize the active components of a microbiome, but its efficacy has not been evaluated systematically. Here, we present our work to benchmark RNA-based amplicon sequencing for activity assessment in synthetic and environmentally sourced microbial communities.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01449-y)

10.1186/s40168-022-01449-y

---

## A microbiota-modulated checkpoint directs immunosuppressive intestinal T cells into cancers.
 15.06.2023, _M Fidelle, C Rauber, C Alves Costa Silva, AL Tian, I Lahmar, AM de La Varende, L Zhao, C Thelemaque, I Lebhar, M Messaoudene, E Pizzato, R Birebent, MD Mbogning Fonkou, S Zoppi, A Reni, C Dalban, M Leduc, G Ferrere, S Durand, P Ly, A Silvin, K Mulder, CA Dutertre, F Ginhoux, S Yonekura, MP Roberti, M Tidjani-Alou, S Terrisse, J Chen, O Kepp, A Schippers, N Wagner, J Suárez-Gosálvez, S Kobold, JE Fahrner, C Richard, J Bosq, L Lordello, G Vitali, N Galleron, B Quinquis, E Le Chatelier, L Blanchard, JP Girard, A Jarry, N Gervois, E Godefroy, N Labarrière, R Koschny, R Daillère, B Besse, C Truntzer, F Ghiringhelli, N Coatnoan, V Mhanna, D Klatzmann, D Drubay, L Albiges, AM Thomas, N Segata, FX Danlos, A Marabelle, B Routy, L Derosa, G Kroemer, L Zitvogel_


_Animals, Humans, Mice, Anti-Bacterial Agents, Bacteria, Cell Adhesion Molecules, Cell Movement, Drug Resistance, Neoplasm, Fecal Microbiota Transplantation, Gastrointestinal Microbiome, Immune Checkpoint Inhibitors, Immune Tolerance, Immunologic Surveillance, Integrins, Interleukin-17, Mucoproteins, Neoplasms, Th17 Cells, Gastrointestinal Tract_
Antibiotics (ABX) compromise the efficacy of programmed cell death protein 1 (PD-1) blockade in cancer patients, but the mechanisms underlying their immunosuppressive effects remain unknown. By inducing the down-regulation of mucosal addressin cell adhesion molecule 1 (MAdCAM-1) in the ileum, post-ABX gut recolonization by Enterocloster species drove the emigration of enterotropic α4β7+CD4+ regulatory T 17 cells into the tumor. These deleterious ABX effects were mimicked by oral gavage of Enterocloster species, by genetic deficiency, or by antibody-mediated neutralization of MAdCAM-1 and its receptor, α4β7 integrin. By contrast, fecal microbiota transplantation or interleukin-17A neutralization prevented ABX-induced immunosuppression. In independent lung, kidney, and bladder cancer patient cohorts, low serum levels of soluble MAdCAM-1 had a negative prognostic impact. Thus, the MAdCAM-1-α4β7 axis constitutes an actionable gut immune checkpoint in cancer immunosurveillance.

[Source](https://www.biorxiv.org/content/10.1101/2023.07.17.549387v1.abstract)

10.1126/science.abo2296

---

## The prevalence of motility within the human oral microbiota

[Source](https://www.biorxiv.org/content/10.1101/2023.07.17.549387v1.abstract)

---

## A shotgun metagenomic analysis of the fecal microbiome in humans infected with Giardia duodenalis.
 18.07.2023, _BA McGregor, E Razmjou, H Hooshyar, DR Seeger, SA Golovko, MY Golovko, SM Singer, J Hur, S Solaymani-Mohammadi_


The mechanisms underlying the clinical outcome disparity during human infection with Giardia duodenalis are still unclear. In recent years, evidence has pointed to the roles of host factors as well as parasite's genetic heterogeneity as major contributing factors in the development of symptomatic human giardiasis. However, it remains contested as to how only a small fraction of individuals infected with G. duodenalis develop clinical gastrointestinal manifestations, whereas the majority of infected individuals remain asymptomatic. Here, we demonstrate that diversity in the fecal microbiome correlates with the clinical outcome of human giardiasis.

[Source](https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-023-05821-1)

10.1186/s13071-023-05821-1

---

## Microbiome-derived cobalamin and succinyl-CoA as biomarkers for improved screening of anal cancer.
 18.07.2023, _S Serrano-Villar, C Tincati, SC Raju, JS Sáenz, E Moreno, R Bargiela, A Cabello-Ubeda, E Sendagorta, A Kurz, JA Perez Molina, A de Benito, JR Hov, L Fernandez-Lopez, A Muriel, R Del Campo, S Moreno, M Trøseid, J Seifert, M Ferrer_


Human papillomavirus can cause preinvasive, high-grade squamous intraepithelial lesions (HSILs) as precursors to cancer in the anogenital area, and the microbiome is suggested to be a contributing factor. Men who have sex with men (MSM) living with human immunodeficiency virus (HIV) have a high risk of anal cancer, but current screening strategies for HSIL detection lack specificity. Here, we investigated the anal microbiome to improve HSIL screening. We enrolled participants living with HIV, divided into a discovery (n = 167) and validation cohort (n = 46), and who were predominantly (93.9%) cisgender MSM undergoing HSIL screening with high-resolution anoscopy and anal biopsies. We identified no microbiome composition signatures associated with HSILs, but elevated levels of microbiome-encoded proteins producing succinyl coenzyme A and cobalamin were significantly associated with HSILs in both cohorts. Measurement of these candidate biomarkers alone in anal cytobrushes outperformed anal cytology as a diagnostic indicator for HSILs, increasing the sensitivity from 91.2% to 96.6%, the specificity from 34.1% to 81.8%, and reclassifying 82% of false-positive results as true negatives. We propose that these two microbiome-derived biomarkers may improve the current strategy of anal cancer screening.

[Source](https://www.nature.com/articles/s41591-023-02407-3)

10.1038/s41591-023-02407-3

---

## Host-microbiota interactions contributing to the heterogeneous tumor microenvironment in colorectal cancer

[Source](https://www.biorxiv.org/content/10.1101/2023.07.17.549261v1.abstract)

---

## Diversity of plant DNA in stool is linked to dietary quality, age, and household income.
 10.07.2023, _BL Petrone, A Aqeel, S Jiang, HK Durand, EP Dallow, JR McCann, HK Dressman, Z Hu, CB Tenekjian, WS Yancy, PH Lin, JJ Scialla, PC Seed, JF Rawls, SC Armstrong, J Stevens, LA David_


_Adolescent, Humans, DNA, Plant, Diet, Nutritional Status, Plants, DNA Barcoding, Taxonomic_
Eating a varied diet is a central tenet of good nutrition. Here, we develop a molecular tool to quantify human dietary plant diversity by applying DNA metabarcoding with the chloroplast trnL-P6 marker to 1,029 fecal samples from 324 participants across two interventional feeding studies and three observational cohorts. The number of plant taxa per sample (plant metabarcoding richness or pMR) correlated with recorded intakes in interventional diets and with indices calculated from a food frequency questionnaire in typical diets (ρ = 0.40 to 0.63). In adolescents unable to collect validated dietary survey data, trnL metabarcoding detected 111 plant taxa, with 86 consumed by more than one individual and four (wheat, chocolate, corn, and potato family) consumed by &gt;70% of individuals. Adolescent pMR was associated with age and household income, replicating prior epidemiologic findings. Overall, trnL metabarcoding promises an objective and accurate measure of the number and types of plants consumed that is applicable to diverse human populations.

[Source](https://www.pnas.org/doi/10.1073/pnas.2304441120)

10.1073/pnas.2304441120

---

## Alterations in fecal β-defensin-3 secretion as a marker of instability of the gut microbiota.
 18.07.2023, _Z Saqib, G De Palma, J Lu, M Surette, P Bercik, SM Collins_


Compositional changes in the microbiota (dysbiosis) may be a basis for Irritable Bowel Syndrome (IBS), but biomarkers are currently unavailable to direct microbiota-directed therapy. We therefore examined whether changes in fecal β-defensin could be a marker of dysbiosis in a murine model. Experimental dysbiosis was induced using four interventions relevant to IBS: a mix of antimicrobials, westernized diets (high-fat/high-sugar and high salt diets), or mild restraint stress. Fecal mouse β-defensin-3 and 16S rRNA-based microbiome profiles were assessed at baseline and during and following these interventions. Each intervention, except for mild restraint stress, altered compositional and diversity profiles of the microbiota. Exposure to antimicrobials or a high-fat/high-sugar diet, but not mild restraint stress, resulted in decreased fecal β-defensin-3 compared to baseline. In contrast, exposure to the high salt diet increased β-defensin-3 compared to baseline. Mice exposed to the mix of antimicrobials showed the largest compositional changes and the most significant correlations between β-defensin-3 levels and bacterial diversity. The high salt diet was also associated with significant correlations between changes in β-defensin-3 and bacterial diversity, and this was not accompanied by discernible inflammatory changes in the host. Thus, dietary change or antimicrobial exposure, both recognized factors in IBS exacerbations, induced marked dysbiosis that was accompanied by changes in fecal β-defensin-3 levels. We propose that serial monitoring of fecal β-defensins may serve as a marker of dysbiosis and help identify those IBS patients who may benefit from microbiota-directed therapeutic interventions.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2023.2233679)

10.1080/19490976.2023.2233679

---

## The effect of prenatal stress on offspring depression susceptibility in relation to the gut microbiome and metabolome

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0165032723009400)

---

## Gastrointestinal and Microbiome Profiling in Rodent Models of Neurological and Psychiatric Disorders.
 18.07.2023, _G Kong, C Gubert, AJ Hannan_


Mice and other rodent models have been widely used to understand the role of the gut microbiome in various neurological and psychiatric disorders. Here we describe a protocol to characterize the structural and functional phenotype of the rodent gut and to examine the gut microbiota composition through V4 16S rRNA gene sequencing and microbiome profiling. This protocol will have utility for those investigating the gut, and associated microbiota, in a wide range of different rodent models of human disorders.

[Source](https://link.springer.com/protocol/10.1007/978-1-0716-3307-6_7)

10.1007/978-1-0716-3307-6_7

---

## Persistent effects of intramammary ceftiofur treatment on the gut microbiome and antibiotic resistance in dairy cattle

[Source](https://www.biorxiv.org/content/10.1101/2023.07.17.549362v1.abstract)

---

## A hybrid and poly-polish workflow for the complete and accurate assembly of phage genomes: a case study of ten przondoviruses.
 18.07.2023, _CKA Elek, TL Brown, T Le Viet, R Evans, DJ Baker, A Telatin, SK Tiwari, H Al-Khanaq, G Thilliez, RA Kingsley, LJ Hall, MA Webber, EM Adriaenssens_


Bacteriophages (phages) within the genus Przondovirus are T7-like podoviruses belonging to the subfamily Studiervirinae, within the family Autographiviridae, and have a highly conserved genome organisation. The genomes of these phages range from 37 to 42 kb in size, encode 50-60 genes and are characterised by the presence of direct terminal repeats (DTRs) flanking the linear chromosome. These DTRs are often deleted during short-read-only and hybrid assemblies. Moreover, long-read-only assemblies are often littered with sequencing and/or assembly errors and require additional curation. Here, we present the isolation and characterisation of ten novel przondoviruses targeting Klebsiella spp. We describe HYPPA, a HYbrid and Poly-polish Phage Assembly workflow, which utilises long-read assemblies in combination with short-read sequencing to resolve phage DTRs and correcting errors, negating the need for laborious primer walking and Sanger sequencing validation. Our assembly workflow utilised Oxford Nanopore Technologies for long-read sequencing for its accessibility, making it the more relevant long-read sequencing technology at this time, and Illumina DNA Prep for short-read sequencing, representing the most commonly used technologies globally. Our data demonstrate the importance of careful curation of phage assemblies before publication, and prior to using them for comparative genomics.

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.001065)

10.1099/mgen.0.001065

---

## Learning a deep language model for microbiomes: the power of large scale unlabeled microbiome data

[Source](https://www.biorxiv.org/content/10.1101/2023.07.17.549267v1.abstract)

---

## MBECS: Microbiome Batch Effects Correction Suite.
 06.05.2023, _M Olbrich, A Künstner, H Busch_


_Software, Algorithms, Microbiota_
Despite the availability of batch effect correcting algorithms (BECA), no comprehensive tool that combines batch correction and evaluation of the results exists for microbiome datasets. This work outlines the Microbiome Batch Effects Correction Suite development that integrates several BECAs and evaluation metrics into a software package for the statistical computation framework R.

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-023-05252-w)

10.1186/s12859-023-05252-w

---

## High-resolution profiling of bacterial and fungal communities using pangenome-informed taxon-specific amplicons and long-read sequence

[Source](https://www.biorxiv.org/content/10.1101/2023.07.17.549274v1.abstract)

---

