## Non-zero-sum microbiome immune system interactions.
 09.07.2021, _T Tuganbaev, K Honda_


Fundamental asymmetries between the host and its microbiome in enzymatic activities and nutrient storage capabilities have promoted mutualistic adaptations on both sides. As a result, the enteric immune system has evolved so as not to cause a zero-sum sterilization of non-self, but rather achieve a non-zero-sum self-reinforcing cooperation with its evolutionary partner the microbiome. In this review we attempt to integrate the accumulated knowledge of immune - microbiome interactions into an evolutionary framework and trace the pattern of positive immune - microbiome feedback loops across epithelial, enteric nervous system, innate and adaptive immune circuits. Indeed, the immune system requires commensal signals for its development and function, and reciprocally protects the microbiome from nutrient shortage and pathogen outgrowth. In turn, a healthy microbiome is the result of immune system curatorship as well as microbial ecology. The paradigms of host-microbiome asymmetry and the cooperative nature of their interactions identified in the gut are applicable across all tissues influenced by microbial activities. Incorporation of immune system influences into models of microbiome ecology will be a step forward towards defining what constitutes a healthy human microbiome and guide discoveries of novel host-microbiome mutualistic adaptations that may be harnessed for promotion of human health. This article is protected by copyright. All rights reserved.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/eji.202049065)

10.1002/eji.202049065

---

## Intersection of FcγRIIB, the microbiome, and checkpoint inhibitors in antitumor immunity.
 09.07.2021, _KM Baecher, ML Ford_


Fc receptors (FcRs) and the microbiome are both known to have an effect on the development and progression of cancers. Checkpoint inhibitors are a novel class of therapeutics which are used to combat cancer and are integrally linked to both FcRs and the microbiome. The use of checkpoint inhibitors has grown exponentially over the past decade, although many host factors affect both the efficacy and the safety of these therapeutics. Some of these host factors, including the microbiome and the expression of FcRs, are currently being investigated. Here we discuss the current understanding of FcRs (particularly the inhibitory FcγRIIB) and the microbiome in context of T cell immunity, inflammation, cancer, and checkpoint inhibition.

[Source](https://link.springer.com/article/10.1007/s00262-021-03004-4)

10.1007/s00262-021-03004-4

---

## Evaluating extraction methods to study canine urine microbiota.
 09.07.2021, _R Mrofchak, C Madden, MV Evans, VL Hale_


The urinary microbiota is the collection of microbes present in urine that may play a role in host health. Studies of urine microbiota have traditionally relied upon culturing methods aimed at identifying pathogens. However, recent culture-free sequencing studies of the urine microbiota have determined that a diverse array of microbes is present in health and disease. To study these microbes and their potential role in diseases like bladder cancer or interstitial cystitis, consistent extraction and detection of bacterial DNA from urine is critical. However, urine is a low biomass substrate, requiring sensitive methods to capture DNA and making the risk of contamination high. To address this challenge, we collected urine samples from ten healthy dogs and extracted DNA from each sample using five different commercially available extraction methods. Extraction methods were compared based on total and bacterial DNA concentrations and bacterial community composition and diversity assessed through 16S rRNA gene sequencing. Significant differences in the urinary microbiota were observed by dog and sex but not extraction method. The Bacteremia Kit yielded the highest total DNA concentrations (Kruskal-Wallis, p = 0.165, not significant) and the highest bacterial DNA concentrations (Kruskal-Wallis, p = 0.044). Bacteremia also extracted bacterial DNA from the greatest number of samples. Taken together, these results suggest that the Bacteremia kit is an effective option for studying the urine microbiota. This work lays the foundation to study the urine microbiome in a wide range of urogenital diseases in dogs and other species.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0253989)

10.1371/journal.pone.0253989

---

## The potential of outdoor environments to supply beneficial butyrate-producing bacteria to humans.
 28.05.2021, _JE Brame, C Liddicoat, CA Abbott, MF Breed_


_Adult, Bacteria, Biodiversity, Butyrates, Dietary Supplements, Gastrointestinal Microbiome, Humans_
Butyrate is an important mediator of human health and disease. The mechanisms of action of butyrate are becoming increasingly well-known. Many commensal bacteria that inhabit the human gut can synthesise butyrate, which is then absorbed into the human host. Simultaneously, several immune- and inflammatory-mediated diseases are being linked to insufficient exposure to beneficial microbes from our environment, including butyrate-producing bacteria. However, the role of outdoor environmental exposure to butyrate-producing bacteria remains poorly understood. Here we review the literature on the human exposure pathways to butyrate-producing bacteria, with a particular focus on outdoor environmental sources (e.g. associated with plants, plant-based residues, and soil), and the health implications of exposure to them. Emerging evidence suggests that environmental butyrate-producers may help supplement the human gut microbiota and represent an important component of the Biodiversity and Old Friends hypotheses. Improving our understanding of potential sources, precursors, and exposure pathways of environmental butyrate-producers that influence the gut microbiota and butyrate production offers promise to advance multiple disciplines of health and environmental science. We outline research priorities to address knowledge gaps in the outdoor environment-butyrate-health nexus and build knowledge of the potential pathways to help optimise exposure to human-beneficial butyrate-producing bacteria from the outdoor environment during childhood and adulthood.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S004896972101130X?via%3Dihub)

10.1016/j.scitotenv.2021.146063

---

## Gut microbiome heritability is nearly universal but environmentally contingent.
 10.07.2021, _L Grieneisen, M Dasari, TJ Gould, JR Björk, JC Grenier, V Yotova, D Jansen, N Gottel, JB Gordon, NH Learn, LR Gesquiere, TL Wango, RS Mututua, JK Warutere, L Siodi, JA Gilbert, LB Barreiro, SC Alberts, J Tung, EA Archie, R Blekhman_


Relatives have more similar gut microbiomes than nonrelatives, but the degree to which this similarity results from shared genotypes versus shared environments has been controversial. Here, we leveraged 16,234 gut microbiome profiles, collected over 14 years from 585 wild baboons, to reveal that host genetic effects on the gut microbiome are nearly universal. Controlling for diet, age, and socioecological variation, 97% of microbiome phenotypes were significantly heritable, including several reported as heritable in humans. Heritability was typically low (mean = 0.068) but was systematically greater in the dry season, with low diet diversity, and in older hosts. We show that longitudinal profiles and large sample sizes are crucial to quantifying microbiome heritability, and indicate scope for selection on microbiome characteristics as a host phenotype.

[Source](https://science.sciencemag.org/content/373/6551/181)

10.1126/science.aba5483

---

## Host genetics influence the gut microbiome.
 10.07.2021, _L Cortes-Ortiz, KR Amato_



[Source](https://science.sciencemag.org/content/373/6551/159)

10.1126/science.abj5287

---

## Isoflavone diet ameliorates experimental autoimmune encephalomyelitis through modulation of gut bacteria depleted in patients with multiple sclerosis.
 10.07.2021, _SN Jensen, NM Cady, SK Shahi, SR Peterson, A Gupta, KN Gibson-Corley, AK Mangalam_


The gut microbiota is a potential environmental factor that influences the development of multiple sclerosis (MS). We and others have demonstrated that patients with MS and healthy individuals have distinct gut microbiomes. However, the pathogenic relevance of these differences remains unclear. Previously, we showed that bacteria that metabolize isoflavones are less abundant in patients with MS, suggesting that isoflavone-metabolizing bacteria might provide protection against MS. Here, using a mouse model of MS, we report that an isoflavone diet provides protection against disease, which is dependent on the presence of isoflavone-metabolizing bacteria and their metabolite equol. Notably, the composition of the gut microbiome in mice fed an isoflavone diet exhibited parallels to healthy human donors, whereas the composition in those fed an isoflavone-free diet exhibited parallels to patients with MS. Collectively, our study provides evidence that dietary-induced gut microbial changes alleviate disease severity and may contribute to MS pathogenesis.

[Source](https://advances.sciencemag.org/content/7/28/eabd4595)

10.1126/sciadv.abd4595

---

## Monitoring Infection and Antibiotic Treatment in the Skin Microbiota of Farmed European Seabass (<em>Dicentrarchus Labrax</em>) Fingerlings 

[Source](https://link.springer.com/article/10.1007%2Fs00248-021-01795-8)

---

## Metagenomic Next-Generation Sequencing for the Diagnosis of&nbsp;<em>Pneumocystis jirovecii</em>&nbsp;Pneumonia in Non-HIV-Infected Patients:

[Source](https://link.springer.com/article/10.1007%2Fs40121-021-00482-y)

---

## Optimal primer selection for sinus microbiome profiling: A comparative analysis of the V1-V3 and V3-4 16S target regions 

[Source](https://onlinelibrary.wiley.com/doi/10.1002/alr.22858)

---

## Detection of&nbsp;<em>Chlamydia psittaci</em>&nbsp;in both blood and bronchoalveolar lavage fluid using metagenomic next-generation sequencing

[Source](https://journals.lww.com/md-journal/Fulltext/2021/07090/Detection_of_Chlamydia_psittaci_in_both_blood_and.22.aspx)

---

## New insights into the microbiota of wild mice

[Source](https://link.springer.com/article/10.1007%2Fs00335-021-09887-z)

---

## Bacterial communities of indoor surface of stingless bee nests

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0252933)

---

## Elucidating the optimum added dosage of Diatomite during co-composting of pig manure and sawdust: Carbon dynamics and microbial community

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969721011256?via%3Dihub)

---

## XiunaRen

---

## Punicic acid ameliorates obesity and liver steatosis by regulating gut microbiota composition in mice.
 09.07.2021, _G Yuan, M Tan, X Chen_


This study aimed to elucidate the effect of punicic acid (PUA, cis9,trans11,cis13-18 : 3) on obesity and liver steatosis in mice induced by high-fat diet (HFD), and to explore the possible mechanism. Mice were fed with either a HFD or a control diet for 8 weeks. Half of HFD-mice received daily supplementation of PUA. Supplementation with PUA ameliorated the liver steatosis and obesity in mice fed by HFD, as demonstrated by the decreased hepatic triglyceride accumulation, body weight gain and fat weight. A HFD increased the ratio of Firmicutes to Bacteroidetes, whereas supplementation with PUA effectively restored it. PUA supplementation counteracted the upregulation in family Desulfovibrionaceae and Helicobacteraceae, and the downregulation in Muribaculaceae and Bacteroidaceae induced by HFD. Correspondingly, the family of Desulfovibrionaceae was positively related, whereas Muribaculaceae was negatively related to the amount of epididymal and perirenal fat, and the level of liver triglyceride and total cholesterol. The family Helicobacteraceae was also positively related to the amount of epididymal and perirenal fat. Moreover, PUA supplementation counteracted the increase in the population of Anaerotruncus, Faecalibaculim, Mucispirillum, and the decrease in the population of Lactobacillus, Roseburia, Oscillibacter at the genus level induced by HFD. These results demonstrated that PUA can at least in part ameliorate obesity and liver steatosis in mice induced by HFD by regulating gut microbiota composition.

[Source](https://pubs.rsc.org/en/content/articlelanding/2021/FO/D1FO01152A#!divAbstract)

10.1039/d1fo01152a

---

## Arc1 and the microbiota together modulate growth and metabolic traits in&nbsp;<em>Drosophila</em>&nbsp;

[Source](https://journals.biologists.com/dev/article-abstract/doi/10.1242/dev.195222/270856/Arc1-and-the-microbiota-together-modulate-growth?redirectedFrom=fulltext)

---

## Diversity and Interactomics of Bacterial Communities Associated with Dominant Trees During Tropical Forest Recovery.
 10.07.2021, _AA Becerra-Lucio, NY Labrín-Sotomayor, PA Becerra-Lucio, FI Trujillo-Elisea, AT Chávez-Bárcenas, S Machkour-M'Rabet, YJ Peña-Ramírez_


Bacterial communities have been identified as functional key members in soil ecology. A deep relation with these communities maintains forest coverture. Trees harbor particular bacteriomes in the rhizosphere, endosphere, or phyllosphere, different from bulk-soil representatives. Moreover, the plant microbiome appears to be specific for the plant-hosting species, varies through season, and responsive to several environmental factors. This work reports the changes in bacterial communities associated with dominant pioneer trees [Tabebuia rosea and Handroanthus chrysanthus [(Bignoniaceae)] during tropical forest recovery chronosequence in the Mayan forest in Campeche, Mexico. Massive 16S sequencing approach leads to identifying phylotypes associated with rhizosphere, bulk-soil, or recovery stage. Lotka-Volterra interactome modeling suggests the presence of putative regulatory roles of some phylotypes over the rest of the community. Our results may indicate that bacterial communities associated with pioneer trees may establish more complex regulatory networks than those found in bulk-soil. Moreover, modeled regulatory networks predicted from rhizosphere samples resulted in a higher number of nodes and interactions than those found in the analysis of bulk-soil samples.

[Source](https://link.springer.com/article/10.1007%2Fs00284-021-02603-9)

10.1007/s00284-021-02603-9

---

## Land-use types shape soil microbial compositions under rapid urbanization in the Xiong'an New Area, China.
 28.05.2021, _C Wang, A Masoudi, M Wang, J Yang, Z Yu, J Liu_


_China, Humans, RNA, Ribosomal, 16S, Soil, Soil Microbiology, Urbanization_
For urban planning and infrastructural projects, considerable attention has been paid to the relationship between soil biota, especially protists, and edaphic conditions in various land-use types having different plant species in the Xiong'an New Area of China. To elucidate this relationship, we assessed edaphic variables and soil biota compositions and compared them among 5 habitat types: human-made forests, crop cultivations, arid rivers, Baiyangdian (BYD) Lake, and around oil wells. In all, 12 experimental plots from terrestrial and aquatic ecosystems were assessed using high-throughput sequencing of environmental DNA, targeting the V3-V4 region of the 16S rRNA gene, internal transcribed spacer 1, and V4 region of the 18S rRNA gene for bacteria, fungi, and protists, respectively. The abundance of bacterial and protist communities was higher than fungi, possibly because fungi prefer acidic soil conditions and likely have greater susceptibility to anthropogenic activities. Across all experimental plots, land-use types contributed the most to the β-diversity of soil biota, followed by soil moisture. Diversity and richness were significantly higher at aquatic habitats than at terrestrial habitats. Predictive metagenomic analysis of trophic groups predicted relatively high frequency of functional genes from bacterial metabolism pathways (carbohydrate and amino acid); contrary to expectation, phototrophic protists, but not fungal symbionts and protistan consumers, were the dominant group at the BYD Lake. Geographical coordinates showed significant (P &lt; 0.05) relationships with all microbiome taxa (nodes at network) from all land-use types. Moreover, soil-microbiome relationships were more complex and more intense at crop habitats. Links between protist and fungal taxa were the highest at the petroleum-contaminated sampling sites, indicating the importance of these two soil microbiomes in polluted soil. Thus, our findings suggest that human manipulation and land-use types are crucial factors for soil biota structure and composition across our sampling sites.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969721010433?via%3Dihub)

10.1016/j.scitotenv.2021.145976

---

## Insights into the Endophytic Bacterial Microbiome of Crocus sativus: Functional Characterization Leads to Potential Agents that Enhance the Plant Growth, Productivity, and Key Metabolite Content.
 09.07.2021, _T Ahmad, S Farooq, DN Mirza, A Kumar, RA Mir, S Riyaz-Ul-Hassan_


The study was undertaken to unravel the culturable endophytic bacterial microbiome of Crocus sativus L. (saffron crocus) and consequently obtain potential leads to develop plant growth-promoting and biocontrol agents for increased productivity and sustainable cultivation. The endophytes formed 47 different operational taxonomic units (OTUs), spanning over 28 genera. The host was preferentially colonized by the genus Bacillus, followed by Burkholderia and Pantoea, respectively. Several endophytes possessed potential plant growth-promoting properties and inhibitory activities against the specific fungal pathogens of saffron. The endophytes, except for Microbacterium oxydans, did not cause any disease symptoms in the pot experiments. The selected cultures, Burkholderia gladioli, Streptomyces achromogenes, and three species of Bacillus, enhanced the host plant growth significantly. Based on the pot experiment results, two isolates, Bacillus mojavensis CS4EB32 and Burkholderia gladioli E39CS3, were selected for the field experiments. We obtained an increase of 67.5%, 69.8%, and 68.3% in the production of flowers with the individual and collective treatments, respectively. The treatments also enhanced the biomass of the plant and the length and weight of stigmas significantly. The endophyte treatments induced the expression of the pathway genes, resulting in a marked increase in the concentration of apocarotenoids. The study indicates that the dominant endophytes support plant growth and development in nature and present an opportunity for developing microbial formulations for the sustainability of saffron cultivation.

[Source](https://link.springer.com/article/10.1007%2Fs00248-021-01810-y)

10.1007/s00248-021-01810-y

---

## Using a high-throughput sequencing technology to evaluate the various forage source epiphytic microbiota and their effect on fermentation quality and microbial diversity of Napier grass.
 09.07.2021, _N Ali, S Wang, J Zhao, Z Dong, J Li, M Nazar, T Shao_


Napier grass (Pennisetum purpureum) is well-known due to its high biomass production. The epiphytic microbiota was prepared from Napier, alfalfa, and red clover grass and served as an inoculum. The chopped sterilized Napier grass was inoculated with reconstituted epiphytic microbiota, and treatments were designed as: distilled water (N0); Napier grass epiphytic microbiota (NP); alfalfa epiphytic microbiota (AL); and red clover epiphytic microbiota (RC). The results reveal that the reconstituted epiphytic microbiota bacteria efficiently adapted in Napier grass silage, improved fermentation, and produced lactic acid. The alfalfa-grass inoculum rapidly dropped pH and enhanced the lactic acid (LA) and the ratio of lactic acid-to-acetic acid (LA/AA) during the entire ensiling process. However, red clover attains high lactic acid, while Napier grass produces high acetic acid-type fermentation at terminal silage. After day 60 of ensiling, Lactobacillus proportion was higher in AL (85.45%), and RC (59.44%), inocula as compared with NP (36.41%), inoculum. The NP inoculum terminal silage was diverse than AL and RC inocula and dominated by Enterobacter (16.32%) and Enterobacteriaceae (10.16%) and also significantly (p &lt; 0.05) higher in acetic acid. The present study concluded that AL and RC epiphytic microbiota successfully develop and more efficient than Napier grass microbiota. It is suggested that abundant microbiota isolate from alfala and red clover and develop more economical and efficient inocula for quality fermentation of Napier grass silage in practice.

[Source](https://link.springer.com/article/10.1007%2Fs00203-021-02296-5)

10.1007/s00203-021-02296-5

---

## Global homogenization of the structure and function in the soil microbiome of urban greenspaces.
 10.07.2021, _M Delgado-Baquerizo, DJ Eldridge, YR Liu, B Sokoya, JT Wang, HW Hu, JZ He, F Bastida, JL Moreno, AR Bamigboye, JL Blanco-Pastor, C Cano-Díaz, JG Illán, TP Makhalanyane, C Siebe, P Trivedi, E Zaady, JP Verma, L Wang, J Wang, T Grebenc, GF Peñaloza-Bojacá, TU Nahberger, AL Teixido, XQ Zhou, M Berdugo, J Duran, A Rodríguez, X Zhou, F Alfaro, S Abades, C Plaza, A Rey, BK Singh, L Tedersoo, N Fierer_


The structure and function of the soil microbiome of urban greenspaces remain largely undetermined. We conducted a global field survey in urban greenspaces and neighboring natural ecosystems across 56 cities from six continents, and found that urban soils are important hotspots for soil bacterial, protist and functional gene diversity, but support highly homogenized microbial communities worldwide. Urban greenspaces had a greater proportion of fast-growing bacteria, algae, amoebae, and fungal pathogens, but a lower proportion of ectomycorrhizal fungi than natural ecosystems. These urban ecosystems also showed higher proportions of genes associated with human pathogens, greenhouse gas emissions, faster nutrient cycling, and more intense abiotic stress than natural environments. City affluence, management practices, and climate were fundamental drivers of urban soil communities. Our work paves the way toward a more comprehensive global-scale perspective on urban greenspaces, which is integral to managing the health of these ecosystems and the well-being of human populations.

[Source](https://advances.sciencemag.org/content/7/28/eabg5809)

10.1126/sciadv.abg5809

---

## Anaerobic fermentation of hydrothermal liquefaction wastewater of dewatered sewage sludge for volatile fatty acids production with focuses on the degradation of organic components and microbial community compositions.
 28.05.2021, _Z Chen, Y Rao, M Usman, H Chen, A Białowiec, S Zhang, G Luo_


_Anaerobiosis, Bioreactors, Fatty Acids, Volatile, Fermentation, Hydrogen-Ion Concentration, Microbiota, Sewage, Waste Water_
Hydrothermal conversion (HTC) is a promising technology for the treatment of dewatered sewage sludge to produce bio-fuels including bio-oil and hydrochar. At the same time, a huge amount of wastewater (HTCWW) was produced. The present study investigated the organic compositions of HTCWW obtained at different HTC temperatures (170-320 °C) and volatile fatty acids (VFAs) yields through anaerobic fermentation. Results showed that the highest VFAs yield of 0.59 gCODVFA/gCOD was obtained from HTCWW obtained at 170 °C (HTCWW 170). Higher amount of easily biodegradable organics including proteins and carbohydrates were present in HTCWW 170 °C, which resulted in the highest VFAs yields. With the increase of HTC temperature, recalcitrant organic compounds were produced as revealed by 3D-EEM and GC-MS analysis, which resulted in lower VFAs yields. Furthermore, microbial analysis showed that different compositions in the HTCWW led to the enrichment of different microbial communities, which affected the VFAs yields.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S004896972101144X?via%3Dihub)

10.1016/j.scitotenv.2021.146077

---

## Robust covariance estimation for high-dimensional compositional data with application to microbial communities analysis.
 29.06.2021, _Y He, P Liu, X Zhang, W Zhou_


_Computer Simulation, High-Throughput Nucleotide Sequencing, Humans, Microbiota_
Microbial communities analysis is drawing growing attention due to the rapid development fire of high-throughput sequencing techniques nowadays. The observed data has the following typical characteristics: it is high-dimensional, compositional (lying in a simplex) and even would be leptokurtic and highly skewed due to the existence of overly abundant taxa, which makes the conventional correlation analysis infeasible to study the co-occurrence and co-exclusion relationship between microbial taxa. In this article, we address the challenges of covariance estimation for this kind of data. Assuming the basis covariance matrix lying in a well-recognized class of sparse covariance matrices, we adopt a proxy matrix known as centered log-ratio covariance matrix in the literature. We construct a Median-of-Means estimator for the centered log-ratio covariance matrix and propose a thresholding procedure that is adaptive to the variability of individual entries. By imposing a much weaker finite fourth moment condition compared with the sub-Gaussianity condition in the literature, we derive the optimal rate of convergence under the spectral norm. In addition, we also provide theoretical guarantee on support recovery. The adaptive thresholding procedure of the MOM estimator is easy to implement and gains robustness when outliers or heavy-tailedness exist. Thorough simulation studies are conducted to show the advantages of the proposed procedure over some state-of-the-arts methods. At last, we apply the proposed method to analyze a microbiome dataset in human gut.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/sim.8979)

10.1002/sim.8979

---

## A new pipeline for structural characterization and classification of RNA-Seq microbiome data.
 10.07.2021, _S Racedo, I Portnoy, JI Vélez, H San-Juan-Vergara, M Sanjuan, E Zurek_


High-throughput sequencing enables the analysis of the composition of numerous biological systems, such as microbial communities. The identification of dependencies within these systems requires the analysis and assimilation of the underlying interaction patterns between all the variables that make up that system. However, this task poses a challenge when considering the compositional nature of the data coming from DNA-sequencing experiments because traditional interaction metrics (e.g., correlation) produce unreliable results when analyzing relative fractions instead of absolute abundances. The compositionality-associated challenges extend to the classification task, as it usually involves the characterization of the interactions between the principal descriptive variables of the datasets. The classification of new samples/patients into binary categories corresponding to dissimilar biological settings or phenotypes (e.g., control and cases) could help researchers in the development of treatments/drugs.

[Source](https://biodatamining.biomedcentral.com/articles/10.1186/s13040-021-00266-7)

10.1186/s13040-021-00266-7

---

## Sebastian Racedo

[Source](https://biodatamining.biomedcentral.com/articles/10.1186/s13040-021-00266-7#auth-Sebastian-Racedo)

---

## Taxonomic analysis of metagenomic data with kASA.
 10.07.2021, _S Weging, A Gogol-Döring, I Grosse_


The taxonomic analysis of sequencing data has become important in many areas of life sciences. However, currently available tools for that purpose either consume large amounts of RAM or yield insufficient quality and robustness. Here, we present kASA, a k-mer based tool capable of identifying and profiling metagenomic DNA or protein sequences with high computational efficiency and a user-definable memory footprint. We ensure both high sensitivity and precision by using an amino acid-like encoding of k-mers together with a range of multiple k's. Custom algorithms and data structures optimized for external memory storage enable a full-scale taxonomic analysis without compromise on laptop, desktop, and HPCC.

[Source](https://academic.oup.com/nar/article/49/12/e68/6204649)

10.1093/nar/gkab200

---

## Rapid and simultaneous determination of histidine metabolism intermediates in human and mouse microbiota and biomatrices.
 10.07.2021, _I Acuña, A Ruiz, T Cerdó, S Cantarero, A López-Moreno, M Aguilera, C Campoy, A Suárez_


Histidine metabolism is a key pathway physiologically involved in satiety, recognition memory, skin, and neural protection and allergic diseases. Microbiologically-produced imidazole propionate induces type II diabetes and interferes with glucose lowering drugs. Despite their determinant health implications, no single method simultaneously assesses histidine metabolites in urine, feces, and microbiota. The aim of this study was to develop a simple, rapid, and sensitive method for the determination of histidine and its major bioactive metabolites histamine, N-acetylhistamine, imidazole-4-acetate, cis-urocanate, trans-urocanate, glutamate and imidazole propionate, using ultrahigh-performance liquid chromatography with electrospray ionization tandem mass spectrometry. An innovative simple extraction method from small aliquots of human and mice urine, feces and microbial cell extracts was coupled to separation in a 6.5 min chromatographic run. The successful performance allowed accurate and precise quantification of all metabolites in mouse feces, suggesting broad exchange of histidine metabolites between the gut and mice. Higher urine histamine, histamine to histidine ratio, and imidazole-4-acetate pointed to an underlying inflammatory or allergic process in mice compared to human subjects. N-acetylhistamine and imidazole propionate were detected in human and mouse feces, confirming its origin from gut microbial metabolism. Our novel and robust analytical method captured histidine metabolism in a single assay that will facilitate broad and deep histidine metabolic phenotyping assessing the impact of microbiota on host health in large-scale human observational and interventional studies.

[Source](https://pubmed.ncbi.nlm.nih.gov/34245620/)

10.1002/biof.1766

---

