## Mechanism of Dyspnea during Exercise in Children with Corrected Congenital Heart Disease.
 24.01.2022, _M Chlif, MM Ammar, NB Said, L Sergey, S Ahmaidi, F Alassery, H Hamam_


_Child, Dyspnea, Exercise, Exercise Test, Heart Defects, Congenital, Humans, Oxygen Consumption_
This study will evaluate cardiorespiratory and peripheral muscle function and their relationship with subjective dyspnea threshold after the surgical correction of congenital heart disease in children. Thirteen children with surgically repaired congenital heart disease were recruited. Each participant performed an incremental exercise test on a cycle ergometer until exhaustion. Gas exchanges were continuously sampled to measure the maximal aerobic parameters and ventilatory thresholds. The functional capacity of the subjects was assessed with a 6 min walk test. At the end of the exercise test, isokinetic Cybex Norm was used to evaluate the strength and endurance of the knee extensor muscle in the leg. Dyspnea was subjectively scored with a visual analog scale during the last 15 s of each exercise step. Oxygen consumption measured at the dyspnea score/VO2 relationship located at the dyspnea threshold, at which dyspnea suddenly increased. Results: The maximal and submaximal values of the parameters describing the exercise and the peripheral muscular performances were: VO2 Peak: 33.8 ± 8.9 mL·min-1·kg-1; HR: 174 ± 9 b·min-1; VEmax: 65.68 ± 15.9 L·min-1; P max: 117 ± 27 W; maximal voluntary isometric force MVIF: 120.8 ± 41.9 N/m; and time to exhaustion Tlim: 53 ± 21 s. Oxygen consumption measured at the dyspnea threshold was related to VO2 Peak (R2 = 0.74; p &lt; 0.01), Tlim (R2 = 0.78; p &lt; 0.01), and the distance achieved during the 6MWT (R2 = 0.57; p &lt; 0.05). Compared to the theoretical maximal values for the power output, VO2, and HR, the surgical correction did not repair the exercise performance. After the surgical correction of congenital heart disease, exercise performance was impeded by alterations of the cardiorespiratory function and peripheral local factors. A subjective evaluation of the dyspnea threshold is a reliable criterion that allows the prediction of exercise capacity in subjects suffering from congenital heart disease.

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

10.3390/ijerph19010099

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## @SarahLebeer

[Source](https://twitter.com/SarahLebeer?ref_src=twsrc%5Etfw)

---

## @MikeNeug

[Source](https://twitter.com/MikeNeug?ref_src=twsrc%5Etfw)

---

## @Josiane63293249

[Source](https://twitter.com/Josiane63293249?ref_src=twsrc%5Etfw)

---

## @_YHuang

[Source](https://twitter.com/_YHuang?ref_src=twsrc%5Etfw)

---

## @HuiminYe4

[Source](https://twitter.com/HuiminYe4?ref_src=twsrc%5Etfw)

---

## @samueldgamboa

[Source](https://twitter.com/samueldgamboa?ref_src=twsrc%5Etfw)

---

## https://t.co/kQMf0mXqSz

[Source](https://t.co/kQMf0mXqSz)

---

## pic.twitter.com/rdhuyRJh5j

[Source](https://t.co/rdhuyRJh5j)

---

## October 31, 2022

[Source](https://twitter.com/MicrobiomeVIF/status/1587044212880482304?ref_src=twsrc%5Etfw)

---

## Sharing of Antimicrobial Resistance Genes between Humans and Food Animals 

[Source](https://journals.asm.org/doi/10.1128/msystems.00775-22)

---

## Host-microbiome interactions: Gut-Liver axis and its connection with other organs 

[Source](https://www.nature.com/articles/s41522-022-00352-6)

---

## Gut microbiota bacterial strain richness is species specific and limits therapeutic engraftment

[Source](https://www.biorxiv.org/content/10.1101/2022.11.01.514782v1.abstract)

---

## Effects of different amoxicillin treatment durations on microbiome diversity and composition in the gut

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0275737)

---

## The microbiota promotes social behavior by modulating microglial remodeling of forebrain neurons

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001838)

---

## Strain-specific impacts of probiotics are a significant driver of gut microbiome development in very preterm infants.
 27.10.2022, _LC Beck, AC Masi, GR Young, T Vatanen, CA Lamb, R Smith, J Coxhead, A Butler, BJ Marsland, ND Embleton, JE Berrington, CJ Stewart_


_Anti-Bacterial Agents, Bifidobacterium, Bifidobacterium bifidum, Female, Gastrointestinal Microbiome, Humans, Infant, Infant, Newborn, Infant, Premature, Probiotics_
The development of the gut microbiome from birth plays important roles in short- and long-term health, but factors influencing preterm gut microbiome development are poorly understood. In the present study, we use metagenomic sequencing to analyse 1,431 longitudinal stool samples from 123 very preterm infants (&lt;32 weeks' gestation) who did not develop intestinal disease or sepsis over a study period of 10 years. During the study period, one cohort had no probiotic exposure whereas two cohorts were given different probiotic products: Infloran (Bifidobacterium bifidum and Lactobacillus acidophilus) or Labinic (B. bifidum, B. longum subsp. infantis and L. acidophilus). Mothers' own milk, breast milk fortifier, antibiotics and probiotics were significantly associated with the gut microbiome, with probiotics being the most significant factor. Probiotics drove microbiome transition into different preterm gut community types (PGCTs), each enriched in a different Bifidobacterium sp. and significantly associated with increased postnatal age. Functional analyses identified stool metabolites associated with PGCTs and, in preterm-derived organoids, sterile faecal supernatants impacted intestinal, organoid monolayer, gene expression in a PGCT-specific manner. The present study identifies specific influencers of gut microbiome development in very preterm infants, some of which overlap with those impacting term infants. The results highlight the importance of strain-specific differences in probiotic products and their impact on host interactions in the preterm gut.

[Source](https://www.nature.com/articles/s41564-022-01213-w)

10.1038/s41564-022-01213-w

---

## Plasticity of the adult human small intestinal stoma microbiota.
 01.11.2022, _B Yilmaz, T Fuhrer, D Morgenthaler, N Krupka, D Wang, D Spari, D Candinas, B Misselwitz, G Beldi, U Sauer, AJ Macpherson_


The human distal small intestine (ileum) has a distinct microbiota, but human studies investigating its composition and function have been limited by the inaccessibility of the ileum without purging and/or deep intubation. We investigated inherent instability, temporal dynamics, and the contribution of fed and fasted states using stoma samples from cured colorectal cancer patients as a non-invasive access route to the otherwise inaccessible small and large intestines. Sequential sampling of the ileum before and after stoma formation indicated that ileostoma microbiotas represented that of the intact small intestine. Ileal and colonic stoma microbiotas were confirmed as distinct, and two types of instability in ileal host-microbial relationships were observed: inter-digestive purging followed by the rapid postprandial blooming of bacterial biomass and sub-strain appearance and disappearance within individual taxa after feeding. In contrast to the relative stability of colonic microbiota, the human small intestinal microbiota biomass and its sub-strain composition can be highly dynamic.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(22)00515-7)

10.1016/j.chom.2022.10.002

---

## @cellhostmicrobe

[Source](https://twitter.com/cellhostmicrobe?ref_src=twsrc%5Etfw)

---

## https://t.co/3iznKsQCIp

[Source](https://t.co/3iznKsQCIp)

---

## pic.twitter.com/TL2tKdyfKU

[Source](https://t.co/TL2tKdyfKU)

---

## October 31, 2022

[Source](https://twitter.com/bahti_yil/status/1587119986832642049?ref_src=twsrc%5Etfw)

---

## MIDAS2: Metagenomic Intra-species Diversity Analysis System

[Source](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btac713/6793850)

---

## Automated identification of sequence-tailored Cas9 proteins using massive metagenomic data 

[Source](https://www.nature.com/articles/s41467-022-34213-9)

---

## microshades: An R Package for Improving Color Accessibility and Organization of Microbiome Data

[Source](https://journals.asm.org/doi/10.1128/mra.00795-22)

---

## Application of ecosystem-specific reference databases for increased taxonomic resolution in soil microbial profiling 

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2022.942396/full)

---

## pic.twitter.com/DcVaCwzEH5

[Source](https://t.co/DcVaCwzEH5)

---

## May 18, 2021

[Source](https://twitter.com/svetlana_up/status/1394613676875915268?ref_src=twsrc%5Etfw)

---

