## A human gut <em>Faecalibacterium prausnitzii</em> fatty acid amide hydrolase

[Source](https://doi.org/10.1126/science.ado6828)

---

## A ubiquitous mobile genetic element changes the antagonistic weaponry of a human gut symbiont

[Source](https://doi.org/10.1126/science.adj9504)

---

## Exploring alterations of gut/blood microbes in addressing iron overload-induced gut dysbiosis and cognitive impairment in thalassemia patients

[Source](https://doi.org/10.1038/s41598-024-76684-4)

---

## Effects of bile acids on the growth, composition and metabolism of gut bacteria.
 25.10.2024, _YL Peng, SH Wang, YL Zhang, MY Chen, K He, Q Li, WH Huang, W Zhang_


_Gastrointestinal Microbiome, Bile Acids and Salts, Bacteria, Animals, Deoxycholic Acid, Mice, Amino Acids, Metabolomics, Humans, Ribosomes_
Bile acids (BAs) exert a profound influence on the body's pathophysiology by intricately shaping the composition of gut bacteria. However, the complex interplay between BAs and gut microbiota has impeded a systematic exploration of their impact on intestinal bacteria. Initially, we investigated the effects of 21 BAs on the growth of 65 gut bacterial strains in vitro. Subsequently, we examined the impact of BAs on the overall composition of intestinal bacteria, both in vivo and in vitro. The results unveiled distinct effects of various BAs on different intestinal strains and their diverse impacts on the composition of gut bacteria. Mechanistically, the inhibition of intestinal strains by BAs occurs through the accumulation of these acids within the strains. The intracellular accumulation of deoxycholic acid (DCA) significantly influenced the growth of intestinal bacteria by impacting ribosome transcription and amino-acid metabolism. The metabolomic analysis underscores the pronounced impact of DCA on amino-acid profiles in both in vivo and in vitro settings. This study not only elucidates the effects of BAs on a diverse range of bacterial strains and their role in shaping the gut microbiota but also reveals underlying mechanisms essential for understanding and maintaining a healthy gut microbiota.

[Source](https://doi.org/10.1038/s41522-024-00566-w)

10.1038/s41522-024-00566-w

---

## Gut-first Parkinson's disease is encoded by gut dysbiome.
 29.10.2024, _MF Munoz-Pinto, E Candeias, I Melo-Marques, AR Esteves, A Maranha, JD Magalhães, DR Carneiro, M Sant'Anna, AR Pereira-Santos, AE Abreu, D Nunes-Costa, S Alarico, I Tiago, A Morgadinho, J Lemos, PN Figueiredo, C Januário, N Empadinhas, SM Cardoso_


_Animals, Gastrointestinal Microbiome, Parkinson Disease, Mice, Dysbiosis, Male, Mice, Inbred C57BL, Humans, Fecal Microbiota Transplantation_
In Parkinson's patients, intestinal dysbiosis can occur years before clinical diagnosis, implicating the gut and its microbiota in the disease. Recent evidence suggests the gut microbiota may trigger body-first Parkinson Disease (PD), yet the underlying mechanisms remain unclear. This study aims to elucidate how a dysbiotic microbiome through intestinal immune alterations triggers PD-related neurodegeneration.

[Source](https://doi.org/10.1186/s13024-024-00766-0)

10.1186/s13024-024-00766-0

---

## Metaproteomics reveals diet-induced changes in gut microbiome function according to Crohn's disease location.
 29.10.2024, _S Levi Mortera, V Marzano, F Rapisarda, C Marangelo, I Pirona, P Vernocchi, M Di Michele, F Del Chierico, MA Quintero, I Fernandez, H Hazime, RM Killian, N Solis, M Ortega, OM Damas, S Proksell, DH Kerman, AR Deshpande, L Garces, F Scaldaferri, A Gasbarrini, MT Abreu, L Putignani_


_Humans, Crohn Disease, Gastrointestinal Microbiome, Male, Female, Feces, Adult, Proteomics, Middle Aged, Diet, Dietary Fiber, Bacteria, Colon, Young Adult, Faecalibacterium_
Crohn's disease (CD) is characterized by chronic intestinal inflammation. Diet is a key modifiable factor influencing the gut microbiome (GM) and a risk factor for CD. However, the impact of diet modulation on GM function in CD patients is understudied. Herein, we evaluated the effect of a high-fiber, low-fat diet (the Mi-IBD diet) on GM function in CD patients. All participants were instructed to follow the Mi-IBD diet for 8 weeks. One group of CD patients received one-time diet counseling only (Gr1); catered food was supplied for the other three groups, including CD patients (Gr2) and dyads of CD patients and healthy household controls (HHCs) residing within the same household (Gr3-HHC dyads). Stool samples were collected at baseline, week 8, and week 36, and analyzed by liquid chromatography-tandem mass spectrometry.

[Source](https://doi.org/10.1186/s40168-024-01927-5)

10.1186/s40168-024-01927-5

---

## Examining the healthy human microbiome concept

[Source](https://doi.org/10.1038/s41579-024-01107-0)

---

## Host-pathobiont interactions in Crohn’s disease

[Source](https://doi.org/10.1038/s41575-024-00997-y)

---

## Species- and subspecies-level characterization of health-associated bacterial consortia that colonize the human gut during infancy

[Source](https://doi.org/10.1080/19490976.2024.2414975)

---

## Metal availability shapes early life microbial ecology and community succession

[Source](https://doi.org/10.1128/mbio.01534-24)

---

## Oral reovirus reshapes the gut microbiome and enhances antitumor immunity in colon cancer.
 25.10.2024, _WS Lee, SJ Lee, HJ Lee, H Yang, EJ Go, E Gansukh, KH Song, X Xiang, DG Park, T Alain, HJ Chon, C Kim_


_Animals, Colonic Neoplasms, Oncolytic Virotherapy, Gastrointestinal Microbiome, Mice, Oncolytic Viruses, Reoviridae, CTLA-4 Antigen, Administration, Oral, Humans, CD8-Positive T-Lymphocytes, Cell Line, Tumor, Dendritic Cells, Female, Mice, Inbred C57BL, Immune Checkpoint Inhibitors, Peyer's Patches_
The route of oncolytic virotherapy is pivotal for immunotherapeutic efficacy in advanced cancers. In this preclinical study, an oncolytic reovirus (RC402) is orally administered to induce antitumor immunity. Oral reovirus treatment shows no gross toxicities and effectively suppresses multifocal tumor lesions. Orally administered reovirus interacts with the host immune system in the Peyer's patch of the terminal ileum, increases IgA+ antibody-secreting cells in the lamina propria through MAdCAM-1+ blood vessels, and reshapes the gut microbiome. Oral reovirus promotes antigen presentation, type I/II interferons, and T cell activation within distant tumors, but does not reach or directly infect tumor cells beyond the gastrointestinal tract. In contrast to intratumoral reovirus injection, the presence of the gut microbiome, Batf3+ dendritic cells, type I interferons, and CD8+ T cells are indispensable for orally administered reovirus-induced antitumor immunity. Oral reovirus treatment is most effective when combined with αPD-1(L1) and/or αCTLA-4, leading to complete colon tumor regression and protective immune memory. Collectively, oral reovirus virotherapy is a feasible and effective immunotherapeutic strategy in preclinical studies.

[Source](https://doi.org/10.1038/s41467-024-53347-6)

10.1038/s41467-024-53347-6

---

## Pooled analysis of oral microbiome profiles defines robust signatures associated with periodontitis.
 24.10.2024, _A Soueidan, K Idiri, C Becchina, P Esparbès, A Legrand, Q Le Bastard, E Montassier_


Oral microbial dysbiosis has been associated with periodontitis in studies using 16S rRNA gene sequencing analysis. However, this technology is not sufficient to consistently separate the bacterial species to species level, and reproducible oral microbiome signatures are scarce. Obtaining these signatures would significantly enhance our understanding of the underlying pathophysiological processes of this condition and foster the development of improved therapeutic strategies, potentially personalized to individual patients. Here, we sequenced newly collected samples from 24 patients with periodontitis, and we collected available oral microbiome data from 24 samples in patients with periodontitis and from 214 samples in healthy individuals (n = 262). Data were harmonized, and we performed a pooled analysis of individual patient data. By metagenomic sequencing of the plaque microbiome, we found microbial signatures for periodontitis and defined a periodontitis-related complex, composed by the most discriminative bacteria. A simple two-factor decision tree, based on Tannerella forsythia and Fretibacterium fastidiosum, was associated with periodontitis with high accuracy (area under the curve: 0.94). Altogether, we defined robust oral microbiome signatures relevant to the pathophysiology of periodontitis that can help define promising targets for microbiome therapeutic modulation when caring for patients with periodontitis.

[Source](https://doi.org/10.1128/msystems.00930-24)

10.1128/msystems.00930-24

---

## Type VI secretion systems promote intraspecific competition and host interactions in a bee gut symbiont

[Source](https://doi.org/10.1073/pnas.2414882121)

---

## Gut bacteria facilitate leaf beetles in adapting to dietary specialization by enhancing larval fitness.
 25.10.2024, _M Ma, J Luo, X Chen, C Li, S Li, J Sun, L Xu_


_Animals, Larva, Gastrointestinal Microbiome, Plant Leaves, Coleoptera, Bacteria, Pantoea, Diet_
Dietary specialization between insect stages can reduce intraspecific food competition. The involvement of gut bacteria and the mechanisms underlying this phenomenon received limited attention. Plagiodera versicolora is a pest harming Salicaceae trees. Here, we confirmed dietary specialization in P. versicolora, wherein adults prefer new leaves, while larvae predominantly consume mature leaves when both types are available. We demonstrated the larval preference for mature leaves confers ecological advantages by promoting growth, development and immunity and this advantage is contingent upon the presence of gut bacteria. Gut microbiota in larvae revealed a significant enrichment of Pantoea when feeding new leaves, with P. anthophila exhibiting the most pronounced inhibitory effect on larval development. Further exploration identified specific metabolites, such as Tyrosyl-valine, with higher content in new leaves, which serve as substrates for the entomopathogenic gut bacterium to facilitate its proliferation. This study provides a fresh perspective on the ecological role of gut bacteria.

[Source](https://doi.org/10.1038/s41522-024-00587-5)

10.1038/s41522-024-00587-5

---

## The dietary effects of two strain probiotics (<em>Leuconostoc mesenteroides</em>, <em>Lactococcus lactis</em>) on growth performance, immune response and gut microbiota in Nile tilapia (<em>Oreochromis niloticus</em>)

[Source](https://doi.org/10.1371/journal.pone.0312580)

---

## Effects of <em>Lactiplantibacillus plantarum</em> CBT LP3 and<em> Bifidobacterium breve</em> CBT BR3 supplementation on weight loss and gut microbiota of overweight dogs

[Source](https://doi.org/10.1038/s41598-024-75594-9)

---

## Maternal gastrointestinal microbiome shapes gut microbial function and resistome of newborns in a cow-to-calf model

[Source](https://doi.org/10.1186/s40168-024-01943-5)

---

## Integrated analysis of metabolome and microbiome in a rat model of perimenopausal syndrome

[Source](https://doi.org/10.1128/msystems.00623-24)

---

## Maternal dysbiosis produces long-lasting behavioral changes in offspring

[Source](https://doi.org/10.1038/s41380-024-02794-0)

---

## DNA metabarcoding analyses reveal fine-scale microbiome structures on Western Canadian bat wings

[Source](https://doi.org/10.1128/spectrum.00376-24)

---

## Aquificae overcomes competition by archaeal thermophiles, and crowding by bacterial mesophiles, to dominate the boiling vent-water of a Trans-Himalayan sulfur-borax spring

[Source](https://doi.org/10.1371/journal.pone.0310595)

---

## Metagenome-assembled genomes of Macrocystis-associated bacteria

[Source](https://doi.org/10.1128/mra.00715-24)

---

## Metagenomic time-series reveals a western English Channel viral community dominated by members with strong seasonal signals

[Source](https://doi.org/10.1093/ismejo/wrae216)

---

## Volatile-mediated interspecific plant interaction promotes root colonization by beneficial bacteria via induced shifts in root exudation.
 23.10.2024, _X Zhou, J Zhang, J Shi, M Khashi U Rahman, H Liu, Z Wei, F Wu, F Dini-Andreote_


_Volatile Organic Compounds, Plant Roots, Rhizosphere, Solanum lycopersicum, Bacteria, Soil Microbiology, Plant Exudates, Microbiota, Solanum tuberosum, Pseudomonas_
Volatile organic compounds (VOCs) released by plants can act as signaling molecules mediating ecological interactions. Therefore, the study of VOCs mediated intra- and interspecific interactions with downstream plant physiological responses is critical to advance our understanding of mechanisms underlying information exchange in plants. Here, we investigated how plant-emitted VOCs affect the performance of an interspecific neighboring plant via induced shifts in root exudate chemistry with implications for root-associated microbiota recruitment.

[Source](https://doi.org/10.1186/s40168-024-01914-w)

10.1186/s40168-024-01914-w

---

## Microbial community acclimation during anaerobic digestion of high-oil food waste.
 28.10.2024, _Y Hu, Z Zhou, C Shen_


_Anaerobiosis, Methane, Sewage, Microbiota, RNA, Ribosomal, 16S, Bacteria, Fatty Acids, Volatile, High-Throughput Nucleotide Sequencing, Acclimatization, Food, Archaea, Oils, Biodegradation, Environmental, Food Loss and Waste_
Anaerobic digestion is one of the most promising options for the disposal of biodegradable food waste. However, the relatively high content of oil in food waste inhibits the conversion efficiency of anaerobic digestion because of the accumulation of long-chain fatty acids (LCFAs). In this study, activated anaerobic sludge was acclimated to accommodate high-oil conditions. The methane yield of high-oil food waste digested by the acclimated sludge increased by 24.9% compared to that digested by the raw sludge. To determine the internal changes in the acclimated sludge, the shifts in the microbial communities during the acclimation period were investigated via high-throughput sequencing (HTS) based on the 16 S rRNA gene. The results indicated that Bacteroidetes, Firmicutes, Chloroflexi and Proteobacteria were the dominant bacteria at the phylum level. The acclimation time allows some functional bacterial taxa to proliferate, such as Clostridium and Longilinea, which are able to degrade LCFAs and turn them into small organic molecules that also have nutrient value for other bacteria. Among the archaeal communities, the hydrogenotrophic methanogen Methanobacterium nearly supplanted the acetotrophic methanogen Methanosaeta. The time profiles of volatile fatty acids (VFAs) and pH during this period provided additional evidence for the success of the acclimation. This study demonstrated the effectiveness of acclimation and the dynamic of microbial communities, which further contributed to the management and resource utilization of high-oil food waste.

[Source](https://doi.org/10.1038/s41598-024-77136-9)

10.1038/s41598-024-77136-9

---

## The choice of 16S rRNA gene sequence analysis impacted characterization of highly variable surface microbiota in dairy processing environments.
 21.10.2024, _SE Daly, J Feng, D Daeschel, J Kovac, AB Snyder_


Accurate knowledge of the microbiota collected from surfaces in food processing environments is important for food quality and safety. This study assessed discrepancies in taxonomic composition and alpha and beta diversity values generated from eight different bioinformatic workflows for the analysis of 16S rRNA gene sequences extracted from the microbiota collected from surfaces in dairy processing environments. We found that the microbiota collected from environmental surfaces varied widely in density (0-9.09 log10 CFU/cm2) and Shannon alpha diversity (0.01-3.40). Consequently, depending on the sequence analysis method used, characterization of low-abundance genera (i.e., below 1% relative abundance) and the number of genera identified (114-173 genera) varied considerably. Some low-abundance genera, including Listeria, varied between the amplicon sequence variant (ASV) and operational taxonomic unit (OTU) methods. Centered log-ratio transformation inflated alpha and beta diversity values compared to rarefaction. Furthermore, the ASV method also inflated alpha and beta diversity values compared to the OTU method (P &lt; 0.05). Therefore, for sparse, uneven, low-density data sets, the OTU method and rarefaction are better for taxonomic and ecological characterization of surface microbiota.IMPORTANCECulture-dependent environmental monitoring programs are used by the food industry to identify foodborne pathogens and spoilage biota on surfaces in food processing environments. The use of culture-independent 16S rRNA amplicon sequencing to characterize this surface microbiota has been proposed as a tool to enhance environmental monitoring. However, there is no consensus on the most suitable bioinformatic analyses to accurately capture the diverse levels and types of bacteria on surfaces in food processing environments. Here, we quantify the impact of different bioinformatic analyses on the results and interpretation of 16S rRNA amplicon sequences collected from three cultured dairy facilities in New York State. This study provides guidance for the selection of appropriate 16S rRNA analysis procedures for studying environmental microbiota in dairy processing environments.

[Source](https://doi.org/10.1128/msystems.00620-24)

10.1128/msystems.00620-24

---

## A single-stranded based library preparation method for virome characterization.
 29.10.2024, _X Zhai, A Gobbi, W Kot, L Krych, DS Nielsen, L Deng_


_Virome, Genome, Viral, Gastrointestinal Microbiome, Gene Library, Humans, DNA, Viral, DNA, Single-Stranded, High-Throughput Nucleotide Sequencing, Reproducibility of Results, Sequence Analysis, DNA_
The gut virome is an integral component of the gut microbiome, playing a crucial role in maintaining gut health. However, accurately depicting the entire gut virome is challenging due to the inherent diversity of genome types (dsDNA, ssDNA, dsRNA, and ssRNA) and topologies (linear, circular, or fragments), with subsequently biases associated with current sequencing library preparation methods. To overcome these problems and improve reproducibility and comparability across studies, universal or standardized virome sequencing library construction methods are highly needed in the gut virome study.

[Source](https://doi.org/10.1186/s40168-024-01935-5)

10.1186/s40168-024-01935-5

---

## Microbiome-based correction for random errors in nutrient profiles derived from self-reported dietary assessments.
 29.10.2024, _T Wang, Y Fu, M Shuai, JS Zheng, L Zhu, AT Chan, Q Sun, FB Hu, ST Weiss, YY Liu_


_Humans, Self Report, Gastrointestinal Microbiome, Nutrients, Diet Records, Diet, Deep Learning, Nutrition Assessment, Female, Male_
Since dietary intake is challenging to directly measure in large-scale cohort studies, we often rely on self-reported instruments (e.g., food frequency questionnaires, 24-hour recalls, and diet records) developed in nutritional epidemiology. Those self-reported instruments are prone to measurement errors, which can lead to inaccuracies in the calculation of nutrient profiles. Currently, few computational methods exist to address this problem. In the present study, we introduce a deep-learning approach-Microbiome-based nutrient profile corrector (METRIC), which leverages gut microbial compositions to correct random errors in self-reported dietary assessments using 24-hour recalls or diet records. We demonstrate the excellent performance of METRIC in minimizing the simulated random errors, particularly for nutrients metabolized by gut bacteria in both synthetic and three real-world datasets. Additionally, we find that METRIC can still correct the random errors well even without including gut microbial compositions. Further research is warranted to examine the utility of METRIC to correct actual measurement errors in self-reported dietary assessment instruments.

[Source](https://doi.org/10.1038/s41467-024-53567-w)

10.1038/s41467-024-53567-w

---

## Fecal samples and rectal swabs adequately reflect the human colonic luminal microbiota.
 27.10.2024, _J Rode, L Brengesjö Johnson, J König, I Rangel, L Engstrand, D Repsilber, RJ Brummer_


_Humans, Feces, Rectum, Gastrointestinal Microbiome, Male, Colon, Adult, Female, Specimen Handling, Middle Aged, Bacteria, Colonoscopy, Young Adult_
The appropriateness of the fecal microbiota to adequately reflect the gut microbiota composition from more difficult to access luminal content at different colonic locations has been debated. Here, in a healthy population, luminal samples were collected from terminal ileum to rectum using an unique sampling technique without the need of prior bowel cleansing/preparation. Rectal swabs were collected immediately prior colonoscopy by an experienced physician, and fecal samples were collected at home by the participants themselves. Microbiota composition was evaluated as relative abundance, α-diversity and Bray-Curtis dissimilarities. Our data suggest that fecal samples and rectal swabs present noninvasive, easily accessible, low-cost sampling tools that are accurate proxies to characterize luminal large intestinal microbiota composition.

[Source](https://doi.org/10.1080/19490976.2024.2416912)

10.1080/19490976.2024.2416912

---

## Optimal 16S rRNA gene amplicon sequencing analysis for oral microbiota to avoid the potential bias introduced by trimming length, primer, and database.
 29.10.2024, _T Nagai, T Shiba, K Komatsu, T Watanabe, T Nemoto, S Maekawa, R Kobayashi, S Matsumura, Y Ohsugi, S Katagiri, Y Takeuchi, T Iwata_


16S rRNA gene amplicon sequencing analysis is used to investigate bacterial communities; however, the estimated bacterial composition can differ from the original due to experimental and analytical biases. Therefore, this study determines the optimal conditions to minimize the potential biases in 16S rRNA gene amplicon sequencing analysis using different DNA samples, trimming lengths, primers, and databases. The results of the mock1 community with 250 and 300 bp paired-end (PE), comprising 15 bacteria from various environments, showed the highest similarity between the theoretical value and the data using the Greengenes2 and the V3 region at the genus level and V5-V6 at the species level. In the 300 bp PE sequencing analysis of the mock2 community, comprising six major oral bacteria, the data using the V3-V4, V4, and V5-V6 regions with the SILVA, Greengenes2, and the Human Oral Microbiome Database (HOMD) showed the highest similarity to the theoretical values at the genus level. At the species level, the data using the V3-V4 and V4 regions with Greengenes2 and the data using the V1-V2 with HOMD exhibited the highest alignment with the theoretical values. In the species analysis of the dental calculus samples with 300 bp PE, the Shannon index value was higher with the V1-V2 region and HOMD than with others. Our results suggest that the optimal conditions for oral microbiome analysis are the combinations of 300 bp PE and the V3-V4 or V4 region with the Greengenes2 and the V1-V2 with HOMD.IMPORTANCEThe 16S rRNA gene amplicon sequencing analysis is frequently used in oral microbiome research. However, this method can have biases that distort the experimental data, which depend on the methodological steps, including sequencing length, trimming length, selected amplification regions, and referenced databases. In this study, a combination of 300 bp PE and the primer targeting the V3-V4 or V4 regions with the Greengenes2 and the V1-V2 region with the HOMD was the most bias-minimizing condition for oral microbiota analysis. In addition, this is the first report of such analyses in modern Japanese dental calculus. The methods used in this study will aid in setting appropriate conditions for sequence analysis of microbiota obtained not only from the oral cavity but also from any environment.

[Source](https://doi.org/10.1128/spectrum.03512-23)

10.1128/spectrum.03512-23

---

## Moving Toward Metaproteogenomics: A Computational Perspective on Analyzing Microbial Samples via Proteogenomics.
 22.10.2024, _F Singer, M Kuhring, BY Renard, T Muth_


_Proteogenomics, Microbiota, Computational Biology, Proteomics, Software, Databases, Protein, Metagenomics, Algorithms, Metagenome, Humans, High-Throughput Nucleotide Sequencing, Mass Spectrometry, Proteome_
Microbial sample analysis has received growing attention within the last decade, driven by important findings in microbiome research and promising applications in the biotechnological field. Modern mass spectrometry-based methodology has been established in this context, providing sufficient sensitivity, resolution, dynamic range, and throughput to analyze the so-called metaproteome of complex microbial mixtures from clinical or environmental samples. While proteomic analyses were previously restricted to common model organisms, next-generation sequencing technologies nowadays allow for the rapid and cost-efficient characterization of whole metagenomes of microbial consortia and specific genomes from non-model organisms to which microbes contribute by significant amounts. This proteogenomic approach, meaning the combined application of genomic and proteomic methods, enables researchers to create a protein database that presents a tailored blueprint of the microbial sample under investigation. This contribution provides an overview of the computational challenges and opportunities in proteogenomics and metaproteomics as of January 2018. For practical application, we first showcase an integrative proteogenomic method that circumvents existing reference databases by creating sample-specific transcripts. The underlying algorithm uses a graph network approach that combines RNA-Seq and peptide information. As a second example, we provide a tutorial for a simulation tool that estimates the computational limits of detecting microbial non-model organisms. This method evaluates the potential influence of error-tolerant searches and proteogenomic approaches on databases of interest. Finally, we discuss recommendations for developing future strategies that may help overcome present limitations by combining the strengths of genome- and proteome-based methods and moving toward an integrated metaproteogenomics approach.

[Source](https://doi.org/10.1007/978-1-0716-4152-1_17)

10.1007/978-1-0716-4152-1_17

---

