## Clinical medicine journals lag behind science journals with regards to "microbiota sequence" data availability.
 07.12.2021, _J Pryor, GD Eslick, NJ Talley, K Duncanson, S Keely, EC Hoedt_



[Source](https://onlinelibrary.wiley.com/doi/10.1002/ctm2.656)

10.1002/ctm2.656

---

## Role of the Microbiome in Regulating Bone Metabolism and Susceptibility to Osteoporosis.
 06.12.2021, _O Cronin, SA Lanham-New, BM Corfe, CL Gregson, AL Darling, KR Ahmadi, PS Gibson, JH Tobias, KA Ward, MH Traka, M Rossi, C Williams, NC Harvey, C Cooper, K Whelan, AG Uitterlinden, PW O'Toole, C Ohlsson, JE Compston, SH Ralston_


The human microbiota functions at the interface between diet, medication-use, lifestyle, host immune development and health. It is therefore closely aligned with many of the recognised modifiable factors that influence bone mass accrual in the young, and bone maintenance and skeletal decline in older populations. While understanding of the relationship between micro-organisms and bone health is still in its infancy, two decades of broader microbiome research and discovery supports a role of the human gut microbiome in the regulation of bone metabolism and pathogenesis of osteoporosis as well as its prevention and treatment. Pre-clinical research has demonstrated biological interactions between the microbiome and bone metabolism. Furthermore, observational studies and randomized clinical trials have indicated that therapeutic manipulation of the microbiota by oral administration of probiotics may influence bone turnover and prevent bone loss in humans. In this paper, we summarize the content, discussion and conclusions of a workshop held by the Osteoporosis and Bone Research Academy of the Royal Osteoporosis Society in October, 2020. We provide a detailed review of the literature examining the relationship between the microbiota and bone health in animal models and in humans, as well as formulating the agenda for key research priorities required to advance this field. We also underscore the potential pitfalls in this research field that should be avoided and provide methodological recommendations to facilitate bridging the gap from promising concept to a potential cause and intervention target for osteoporosis.

[Source](https://link.springer.com/article/10.1007%2Fs00223-021-00924-2)

10.1007/s00223-021-00924-2

---

## Defining and quantifying the core microbiome: Challenges and prospects.
 06.12.2021, _AT Neu, EE Allen, K Roy_


The term "core microbiome" has become widely used in microbial ecology over the last decade. Broadly, the core microbiome refers to any set of microbial taxa, or the genomic and functional attributes associated with those taxa, that are characteristic of a host or environment of interest. Most commonly, core microbiomes are measured as the microbial taxa shared among two or more samples from a particular host or environment. Despite the popularity of this term and its growing use, there is little consensus about how a core microbiome should be quantified in practice. Here, we present a brief history of the core microbiome concept and use a representative sample of the literature to review the different metrics commonly used for quantifying the core. Empirical analyses have used a wide range of metrics for quantifying the core microbiome, including arbitrary occurrence and abundance cutoff values, with the focal taxonomic level of the core ranging from phyla to amplicon sequence variants. However, many of these metrics are susceptible to sampling and other biases. Developing a standardized set of metrics for quantifying the core that accounts for such biases is necessary for testing specific hypotheses about the functional and ecological roles of core microbiomes.

[Source](https://www.pnas.org/content/118/51/e2104429118)

10.1073/pnas.2104429118

---

## Clean room microbiome complexity impacts planetary protection bioburden.
 07.12.2021, _R Hendrickson, C Urbaniak, JJ Minich, HS Aronson, C Martino, R Stepanauskas, R Knight, K Venkateswaran_


The Spacecraft Assembly Facility (SAF) at the NASA's Jet Propulsion Laboratory is the primary cleanroom facility used in the construction of some of the planetary protection (PP)-sensitive missions developed by NASA, including the Mars 2020 Perseverance Rover that launched in July 2020. SAF floor samples (n=98) were collected, over a 6-month period in 2016 prior to the construction of the Mars rover subsystems, to better understand the temporal and spatial distribution of bacterial populations (total, viable, cultivable, and spore) in this unique cleanroom.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01159-x)

10.1186/s40168-021-01159-x

---

## Community Composition of Bacteria Isolated from Swiss Banknotes Varies Depending on Collection Environment

[Source](https://www.biorxiv.org/content/10.1101/2021.12.01.470750v2.abstract)

---

## Microbiota and stress: a loop that impacts memory.
 09.12.2021, _N Kraimi, F Lormant, L Calandreau, F Kempf, O Zemb, J Lemarchand, P Constantin, C Parias, K Germain, S Rabot, C Philippe, A Foury, MP Moisan, AV Carvalho, V Coustham, H Dardente, P Velge, T Chaumeil, C Leterrier_


Chronic stress and the gut microbiota appear to comprise a feed-forward loop, which contributes to the development of depressive disorders. Evidence suggests that memory can also be impaired by either chronic stress or microbiota imbalance. However, it remains to be established whether these could be a part of an integrated loop model and be responsible for memory impairments. To shed light on this, we used a two-pronged approach in Japanese quail: first stress-induced alterations in gut microbiota were characterized, then we tested whether this altered microbiota could affect brain and memory function when transferred to a germ-free host. The cecal microbiota of chronically stressed quails was found to be significantly different from that of unstressed individuals with lower α and β diversities and increased Bacteroidetes abundance largely represented by the Alistipes genus, a well-known stress target in rodents and humans. The transfer of this altered microbiota into germ-free quails decreased their spatial and cue-based memory abilities as previously demonstrated in the stressed donors. The recipients also displayed increased anxiety-like behavior, reduced basal plasma corticosterone levels and differential gene expression in the brain. Furthermore, cecal microbiota transfer from a chronically stressed individual was sufficient to mimic the adverse impact of chronic stress on memory in recipient hosts and this action may be related to the Alistipes genus. Our results provide evidence of a feed-forward loop system linking the microbiota-gut-brain axis to stress and memory function and suggest that maintaining a healthy microbiota could help alleviate memory impairments linked to chronic stress.

[Source](https://www.sciencedirect.com/science/article/pii/S0306453021004686)

10.1016/j.psyneuen.2021.105594

---

## Species- and site-specific genome editing in complex bacterial communities.
 07.12.2021, _BE Rubin, S Diamond, BF Cress, A Crits-Christoph, YC Lou, AL Borges, H Shivram, C He, M Xu, Z Zhou, SJ Smith, R Rovinsky, DCJ Smock, K Tang, TK Owens, N Krishnappa, R Sachdeva, R Barrangou, AM Deutschbauer, JF Banfield, JA Doudna_


Understanding microbial gene functions relies on the application of experimental genetics in cultured microorganisms. However, the vast majority of bacteria and archaea remain uncultured, precluding the application of traditional genetic methods to these organisms and their interactions. Here, we characterize and validate a generalizable strategy for editing the genomes of specific organisms in microbial communities. We apply environmental transformation sequencing (ET-seq), in which nontargeted transposon insertions are mapped and quantified following delivery to a microbial community, to identify genetically tractable constituents. Next, DNA-editing all-in-one RNA-guided CRISPR-Cas transposase (DART) systems for targeted DNA insertion into organisms identified as tractable by ET-seq are used to enable organism- and locus-specific genetic manipulation in a community context. Using a combination of ET-seq and DART in soil and infant gut microbiota, we conduct species- and site-specific edits in several bacteria, measure gene fitness in a nonmodel bacterium and enrich targeted species. These tools enable editing of microbial communities for understanding and control.

[Source](https://www.nature.com/articles/s41564-021-01014-7)

10.1038/s41564-021-01014-7

---

## Integrated metabolic and microbial analysis reveals host-microbial interactions in IgE-mediated childhood asthma.
 07.12.2021, _CY Chiu, ML Cheng, MH Chiang, CJ Wang, MH Tsai, G Lin_


A metabolomics-based approach to address the molecular mechanism of childhood asthma with immunoglobulin E (IgE) or allergen sensitization related to microbiome in the airways remains lacking. Fifty-three children with lowly sensitized non-atopic asthma (n = 15), highly sensitized atopic asthma (n = 13), and healthy controls (n = 25) were enrolled. Blood metabolomic analysis with 1H-nuclear magnetic resonance (NMR) spectroscopy and airway microbiome composition analysis by bacterial 16S rRNA sequencing were performed. An integrative analysis of their associations with allergen-specific IgE levels for lowly and highly sensitized asthma was also assessed. Four metabolites including tyrosine, isovalerate, glycine, and histidine were uniquely associated with lowly sensitized asthma, whereas one metabolite, acetic acid, was strongly associated with highly sensitized asthma. Metabolites associated with highly sensitized asthma (valine, isobutyric acid, and acetic acid) and lowly sensitized asthma (isovalerate, tyrosine, and histidine) were strongly correlated each other (P &lt; 0.01). Highly sensitized asthma associated metabolites were mainly enriched in pyruvate and acetyl-CoA metabolisms. Metabolites associated with highly sensitized atopic asthma were mostly correlated with microbiota in the airways. Acetic acid, a short-chain fatty acid (SCFA), was negatively correlated with the genus Atopobium (P &lt; 0.01), but positively correlated with the genus Fusobacterium (P &lt; 0.05). In conclusion, metabolomics reveals microbes-related metabolic pathways associated with IgE responses to house dust mite allergens in childhood asthma. A strong correlation of metabolites related to highly sensitized atopic asthma with airway microbiota provides linkages between the host-microbial interactions and asthma endotypes.

[Source](https://www.nature.com/articles/s41598-021-02925-5)

10.1038/s41598-021-02925-5

---

## Metagenome-genome-wide association studies reveal human genetic impact on the oral microbiome.
 07.12.2021, _X Liu, X Tong, J Zhu, L Tian, Z Jie, Y Zou, X Lin, H Liang, W Li, Y Ju, Y Qin, L Zou, H Lu, S Zhu, X Jin, X Xu, H Yang, J Wang, Y Zong, W Liu, Y Hou, H Jia, T Zhang_


The oral microbiota contains billions of microbial cells, which could contribute to diseases in many body sites. Challenged by eating, drinking, and dental hygiene on a daily basis, the oral microbiota is regarded as highly dynamic. Here, we report significant human genomic associations with the oral metagenome from more than 1915 individuals, for both the tongue dorsum (n = 2017) and saliva (n = 1915). We identified five genetic loci associated with oral microbiota at study-wide significance (p &lt; 3.16 × 10-11). Four of the five associations were well replicated in an independent cohort of 1439 individuals: rs1196764 at APPL2 with Prevotella jejuni, Oribacterium uSGB 3339 and Solobacterium uSGB 315; rs3775944 at the serum uric acid transporter SLC2A9 with Oribacterium uSGB 1215, Oribacterium uSGB 489 and Lachnoanaerobaculum umeaense; rs4911713 near OR11H1 with species F0422 uSGB 392; and rs36186689 at LOC105371703 with Eggerthia. Further analyses confirmed 84% (386/455 for tongue dorsum) and 85% (391/466 for saliva) of host genome-microbiome associations including six genome-wide significant associations mutually validated between the two niches. As many of the oral microbiome-associated genetic variants lie near miRNA genes, we tentatively validated the potential of host miRNAs to modulate the growth of specific oral bacteria. Human genetics accounted for at least 10% of oral microbiome compositions between individuals. Machine learning models showed that polygenetic risk scores dominated over oral microbiome in predicting risk of dental diseases such as dental calculus and gingival bleeding. These findings indicate that human genetic differences are one explanation for a stable or recurrent oral microbiome in each individual.

[Source](https://www.nature.com/articles/s41421-021-00356-0)

10.1038/s41421-021-00356-0

---

## Interaction Between Intestinal Microbiota, Immunology Molecular Mimicry, and AID

[Source](https://www.hindawi.com/journals/jir/2021/5578958/)

---

## Overview of the microbiota in the gut-liver axis in viral B and C hepatitis.
 11.12.2021, _MA Neag, AO Mitre, A Catinean, AD Buzoianu_


Viral B and C hepatitis are a major current health issue, both diseases having a chronic damaging effect on the liver and its functions. Chronic liver disease can lead to even more severe and life-threatening conditions, such as liver cirrhosis and hepatocellular carcinoma. Recent years have uncovered an important interplay between the liver and the gut microbiome: the gut-liver axis. Hepatitis B and C infections often cause alterations in the gut microbiota by lowering the levels of 'protective' gut microorganisms and, by doing so, hinder the microbiota ability to boost the immune response. Treatments aimed at restoring the gut microbiota balance may provide a valuable addition to current practice therapies and may help limit the chronic changes observed in the liver of hepatitis B and C patients. This review aims to summarize the current knowledge on the anato-functional axis between the gut and liver and to highlight the influence that hepatitis B and C viruses have on the microbiota balance, as well as the influence of treatments aimed at restoring the gut microbiota on infected livers and disease progression.

[Source](https://link.springer.com/article/10.1007%2Fs12029-021-00748-1)

10.3748/wjg.v27.i43.7446

---

## The Gut Microbiome and Hepatocellular Carcinoma

[Source](https://link.springer.com/article/10.1007%2Fs12029-021-00748-1)

---

## Anaerostipes hominis sp. nov., a novel butyrate-producing bacteria isolated from faeces of a patient with Crohn's disease.
 07.12.2021, _JY Lee, W Kang, NR Shin, DW Hyun, PS Kim, HS Kim, JY Lee, EJ Tak, H Sung, JW Bae_


Cultivation and isolation of gut bacteria are necessary for understanding their role in the intestinal ecosystem. We isolated a novel bacterium, designated strain BG01T, from the faeces of a patient with Crohn's disease. Strain BG01T was a strictly anaerobic, rod-shaped, Gram-variable and endospore-forming bacterium. Strain BG01T possessed C12 : 0, C18 : 0 dimethyl aldehyde (DMA) and C18 : 1  ω9c DMA as predominant cellular fatty acids and meso-diaminopimelic acid as a diagnostic diamino acid. Strain BG01T grew at 15-45 °C (optimum, 37 °C), with 0-4 % (w/v) NaCl (optimum, 0-1 %), at pH 6-10 (optimum, pH 7) and was resistant to bile salt, but not to ampicillin, metronidazole, vancomycin and cefoperazone. Butyrate, propionate, oxalacetate and fumarate were produced as fermentation end products from Gifu anaerobic medium broth. Strain BG01T showed 97.7 % 16S rRNA gene sequence similarity, and 92.0 and 48.5 % of average nucleotide identity and digital DNA-DNA hybridization values, respectively, with Anaerostipes caccae KCTC 15019T. Genomic analysis indicated that strain BG01T had a butyrate-producing pathway. The genomic G+C content of the strain was 43.5 mol%. Results of the phenotypic, phylogenetic and genotypic analyses indicated that strain BG01T represents a novel butyrate-producing species of the genus Anaerostipes, for which the name Anaerostipes hominis sp. nov. is proposed. The type strain is BG01T (=KCTC 15617T=JCM 32275T).

[Source](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.005129)

10.1099/ijsem.0.005129

---

## Gut microbiome is associated with the clinical response to anti-PD-1 based immunotherapy in hepatobiliary cancers.
 07.12.2021, _J Mao, D Wang, J Long, X Yang, J Lin, Y Song, F Xie, Z Xun, Y Wang, Y Wang, Y Li, H Sun, J Xue, Y Song, B Zuo, J Zhang, J Bian, T Zhang, X Yang, L Zhang, X Sang, H Zhao_


The gut microbiome is associated with the response to immunotherapy for different cancers. However, the impact of the gut microbiome on hepatobiliary cancers receiving immunotherapy remains unknown. This study aims to investigate the relationship between the gut microbiome and the clinical response to anti-programmed cell death protein 1 (PD-1) immunotherapy in patients with advanced hepatobiliary cancers.

[Source](https://jitc.bmj.com/content/9/12/e003334)

10.1136/jitc-2021-003334

---

## Time-restricted feeding induces Lactobacillus- and Akkermansia-specific functional changes in the rat fecal microbiota.
 06.12.2021, _A Palomba, A Tanca, M Abbondio, R Sau, M Serra, F Marongiu, C Fraumene, D Pagnozzi, E Laconi, S Uzzau_


Diet is a key factor influencing gut microbiota (GM) composition and functions, which in turn affect host health. Among dietary regimens, time-restricted (TR) feeding has been associated to numerous health benefits. The impact of TR feeding on the GM composition has been mostly explored by means of metagenomic sequencing. To date, however, little is known about the modulation of GM functions by this dietary regimen. Here, we analyzed the effects of TR feeding on GM functions by evaluating protein expression changes in a rat model through a metaproteomic approach. We observed that TR feeding has a relevant impact on GM functions, specifically leading to an increased abundance of several enzymes involved in carbohydrate and protein metabolism and expressed by Lactobacillus spp. and Akkermansia muciniphila. Taken together, these results contribute to deepening our knowledge about the key relationship between diet, GM, and health.

[Source](https://www.nature.com/articles/s41522-021-00256-x)

10.1038/s41522-021-00256-x

---

## Diet-driven mercury contamination is associated with polar bear gut microbiota.
 07.12.2021, _SE Watson, MA McKinney, M Pindo, MJ Bull, TC Atwood, HC Hauffe, SE Perkins_


The gut microbiota may modulate the disposition and toxicity of environmental contaminants within a host but, conversely, contaminants may also impact gut bacteria. Such contaminant-gut microbial connections, which could lead to alteration of host health, remain poorly known and are rarely studied in free-ranging wildlife. The polar bear (Ursus maritimus) is a long-lived, wide-ranging apex predator that feeds on a variety of high trophic position seal and cetacean species and, as such, is exposed to among the highest levels of biomagnifying contaminants of all Arctic species. Here, we investigate associations between mercury (THg; a key Arctic contaminant), diet, and the diversity and composition of the gut microbiota of polar bears inhabiting the southern Beaufort Sea, while accounting for host sex, age class and body condition. Bacterial diversity was negatively associated with seal consumption and mercury, a pattern seen for both Shannon and Inverse Simpson alpha diversity indices (adjusted R2 = 0.35, F1,18 = 8.00, P = 0.013 and adjusted R2 = 0.26, F1,18 = 6.04, P = 0.027, respectively). No association was found with sex, age class or body condition of polar bears. Bacteria known to either be involved in THg methylation or considered to be highly contaminant resistant, including Lactobacillales, Bacillales and Aeromonadales, were significantly more abundant in individuals that had higher THg concentrations. Conversely, individuals with higher THg concentrations showed a significantly lower abundance of Bacteroidales, a bacterial order that typically plays an important role in supporting host immune function by stimulating intraepithelial lymphocytes within the epithelial barrier. These associations between diet-acquired mercury and microbiota illustrate a potentially overlooked outcome of mercury accumulation in polar bears.

[Source](https://www.nature.com/articles/s41598-021-02657-6)

10.1038/s41598-021-02657-6

---

## Maternal inheritance of the koala gut microbiome and its compositional and functional maturation during juvenile development.
 10.12.2021, _MDJ Blyton, RM Soo, P Hugenholtz, BD Moore_


The acquisition and maturation of the gastrointestinal microbiome is a crucial aspect of mammalian development, particularly for specialist herbivores such as the koala (Phascolarctos cinereus). Joey koalas are thought to be inoculated with microorganisms by feeding on specialized maternal faeces (pap). We found that compared to faeces, pap has higher microbial density, higher microbial evenness and a higher proportion of rare taxa, which may facilitate the establishment of those taxa in joey koalas. We show that the microbiomes of captive joey koalas were on average more similar to those of their mothers than to other koalas, indicating strong maternal inheritance of the faecal microbiome, which can lead to intergenerational gut dysbiosis when the mother is ill. Directly after pap feeding, the joey koalas' microbiomes were enriched for milk-associated bacteria including Bacteroides fragilis, suggesting a conserved role for this species across mammalian taxa. The joeys' microbiomes then changed slowly over 5 months to resemble those of adults by 1 year of age. The relative abundance of fibrolytic bacteria and genes involved in the degradation of plant cell walls also increased in the infants over this time, likely in response to an increased proportion of Eucalyptus leaves in their diets.

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.15858)

10.1111/1462-2920.15858

---

## Relationships between pond water and tilapia skin microbiomes in aquaculture ponds in Malawi

[Source](https://www.biorxiv.org/content/10.1101/2021.12.06.470702v1.abstract)

---

## Age-Based Variations in the Gut Microbiome of the Shennongjia (Hubei) Golden Snub-Nosed Monkey (Rhinopithecus roxellana hubeiensis).
 26.05.2021, _L Yao, X Li, Z Zhou, D Shi, Z Li, S Li, H Yao, J Yang, H Yu, Y Xiao_


_Aging, Bacteria, Colobinae, Gastrointestinal Microbiome_
The gut microbiota represents a source of genetic and metabolic diversity of a complex polymicrobial ecosystem within its host. To investigate age-based variations of the gut microbiota among Shennongjia golden snub-nosed monkeys (Rhinopithecus roxellana hubeiensis), we characterized the microbial species in fecal samples from 18 Shennongjia golden snub-nosed monkeys evenly pooled into 3 aged groups (Group 1, 1-3 years; Group 2, 5-8 years; Group 3, above 12 years) in Shennongjia, Hubei Province, China. Genomic DNA was extracted from fecal samples, and the 16S rRNA gene V4 region was sequenced using the Illumina high-throughput MiSeq platform PE250. A total of 28 microbial phyla were identified in the gut microbiome of these monkeys with the ten most abundant phyla (i.e., Firmicutes, Bacteroidetes, Verrucomicrobia, Spirochaetes, Tenericutes, Proteobacteria, Planctomycetes, Fibrobacteres, Cyanobacteria, and Euryarchaeota). A total of 1,469 (of 16 phyla and 166 genera), 1,381 (of 16 phyla and 157 genera), and 1,931 (of 19 phyla and 190 genera) operational taxonomic units (OTUs) were revealed in Groups 1, 2, and 3, respectively, with Group 3 containing the most diverse groups of OTUs as revealed by the species relative abundance clustering analysis. These results suggest that the gut microbiota in these monkeys maintain a dynamic status, starting from the early developmental stages of life with the species relative abundance increasing with age. This is the first study to comprehensively characterize the gut microbiota and provide valuable information for monitoring the health and nutritional needs of this endangered primate at different ages.

[Source](https://www.hindawi.com/journals/bmri/2021/6667715/)

10.1155/2021/6667715

---

## Looking through the FOG: microbiome characterization and lipolytic bacteria isolation from a fatberg site.
 07.12.2021, _EK Court, RR Chaudhuri, RV Kapoore, RX Villa, J Pandhal, CA Biggs, GP Stafford_


Sewer systems are complex physical, chemical and microbial ecosystems where fats, oils and grease (FOG) present a major problem for sewer management. Their accumulation can lead to blockages ('Fatbergs'), sewer overflows and disruption of downstream wastewater treatment. Further advancements of biological FOG treatments need to be tailored to degrade the FOG, and operate successfully within the sewer environment. In this study we developed a pipeline for isolation of lipolytic strains directly from two FOG blockage sites in the UK, and isolated a range of highly lipolytic bacteria. We selected the five most lipolytic strains using Rhodamine B agar plates and pNP-Fatty acid substrates, with two Serratia spp., two Klebsiella spp. and an environmental Acinetobacter strain that all have the capacity to grow on FOG-based carbon sources. Their genome sequences identified the genetic capacity for fatty acid harvesting (lipases), catabolism and utilization (Fad genes). Furthermore, we performed a preliminary molecular characterization of the microbial community at these sites, showing a diverse community of environmental bacteria at each site, but which did include evidence of sequences related to our isolates. This study provides proof of concept to isolation strategies targeting Fatberg sites to yield candidate strains with bioremediation potential for FOG in the wastewater network. Our work sets the foundation for development of novel bioadditions tailored to the environment with non-pathogenic Acinetobacter identified as a candidate for this purpose.

[Source](https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.001117)

10.1099/mic.0.001117

---

## The Baltic Sea methane pockmark microbiome: The new insights into the patterns of relative abundance and ANME niche separation.
 10.12.2021, _TR Iasakov, TA Kanapatskiy, SV Toshchakov, AA Korzhenkov, MO Ulyanova, NV Pimenov_


Pockmarks are important "pumps", which are believed to play a significant role in the global methane cycling and harboring a unique assemblage of very diverse prokaryotes. This study reports the results of massive sequencing of the 16S rRNA gene V4 hypervariable regions for the samples from thirteen pockmark horizons (the Baltic Sea) collected at depths from 0 to 280 cm below seafloor (cmbsf) and the rates of microbially mediated anaerobic oxidation of methane (AOM) and sulfate reduction (SR). Altogether, 76 bacterial and 12 archaeal phyla were identified, 23 of which were candidate divisions. Of the total obtained in the pockmark sequences, 84.3% of them were classified as Bacteria and 12.4% as Archaea; 3.3% of the sequences were assigned to unknown operational taxonomic units (OTUs). Members of the phyla Planctomycetota, Chloroflexota, Desulfobacterota, Caldatribacteriota, Acidobacteriota and Proteobacteria predominated across all horizons, comprising 58.5% of the total prokaryotic community. These phyla showed different types of patterns of relative abundance. Analysis of AOM-SR-mediated prokaryotes abundance and biogeochemical measurements revealed that ANME-2a-2b subcluster was predominant in sulfate-rich upper horizons (including sulfate-methane transition zone (SMTZ)) and together with sulfate-reducing bacterial group SEEP-SRB1 had a primary role in AOM coupled to SR. At deeper sulfate-depleted horizons ANME-2a-2b shifted to ANME-1a and ANME-1b which alone mediated AOM or switch to methanogenic metabolism. Shifting of the ANME subclusters depending on depth reflect a tendency for niche separation in these groups. It was shown that the abundance of Caldatribacteriota and organohalide-respiring Dehalococcoidia (Chloroflexota) exhibited a strong correlation with AOM rates. This is the first detailed study of depth profiles of prokaryotic diversity, patterns of relative abundance, and ANME niche separation in the Baltic Sea pockmark microbiomes sheds light on assembly of prokaryotes in a pockmark.

[Source](https://www.sciencedirect.com/science/article/pii/S0141113621002890)

10.1016/j.marenvres.2021.105533

---

## 3MCor: an integrative web server for metabolome-microbiome-metadata correlation analysis.
 07.12.2021, _T Sun, M Li, X Yu, D Liang, G Xie, C Sang, W Jia, T Chen_


The metabolome and microbiome disorders are highly associated with human health and there are great demands for dual-omics interaction analysis. Here, we designed and developed an integrative platform, 3MCor, for metabolome and microbiome correlation analysis under the instruction of phenotype and with the consideration of confounders.

[Source](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab818/6454942)

10.1093/bioinformatics/btab818

---

## Microbiome Methods in Experimental Autoimmune Encephalomyelitis.
 07.12.2021, _DP Daberkow, K Hoffman, HM Kohl, T Long, TO Kirby, J Ochoa-Repáraz_


Microbiome composition studies are increasingly shedding light on animal models of disease. This paper describes a protocol for analyzing the gut microbiome composition prior to and after the induction of mice to experimental autoimmune encephalomyelitis (EAE), the principal animal model of the human neuroinflammatory demyelinating disease multiple sclerosis (MS). We also address and provide data assessing the impact of mice reared in different animal facilities on EAE induction. Furthermore, we discuss potential regulators of the gut-microbiome-brain axis (GMBA) in relation to neuroinflammation and implications on demyelinating disease states. Our results suggest that mice reared in different animal facilities produce different levels of EAE induction. These results highlight the importance of accounting for consistent environmental conditions when inducing EAE and other animal models of disease. © 2021 Wiley Periodicals LLC. Basic Protocol 1: Study of the composition of the gut microbiome in the neuroinflammatory model of experimental autoimmune encephalomyelitis Basic Protocol 2: Experimental procedures for DNA extraction and microbiome analysis.

[Source](https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/cpz1.314)

10.1002/cpz1.314

---

## A wipe-based stool collection and preservation kit for microbiome community profiling

[Source](https://www.biorxiv.org/content/10.1101/2021.12.03.471072v1.abstract)

---

