## The bacterial association with oral cavity and intra-abdominal abscess after gastrectomy.
 09.11.2020, _M Nishikawa, M Honda, R Kimura, A Kobayashi, Y Yamaguchi, S Hori, H Kobayashi, M Waragai, H Kawamura, Y Nakayama, Y Todate, Y Takano, H Yamaguchi, K Hamada, S Iketani, I Seto, Y Izumi, K Seto_


Perioperative oral management has been reported to be effective for preventing postoperative infectious complications. In addition, severe periodontal disease was identified as the significant risk factor for complications after gastrointestinal surgery. We investigated the bacteriological association between the periodontal pocket, stomach mucosa and drainage fluid to determine whether oral bacteria directly cause intra-abdominal infection after gastrectomy.

[Source](https://doi.org/10.1371/journal.pone.0242091)

10.1371/journal.pone.0242091

---

## Loss of symbiont infectivity following thermal stress can be a factor limiting recovery from bleaching in cnidarians.
 06.11.2020, _M Kishimoto, AH Baird, S Maruyama, J Minagawa, S Takahashi_


Increases in seawater temperature can cause coral bleaching through loss of symbiotic algae (dinoflagellates of the family Symbiodiniaceae). Corals can recover from bleaching by recruiting algae into host cells from the residual symbiont population or from the external environment. However, the high coral mortality that often follows mass-bleaching events suggests that recovery is often limited in the wild. Here, we examine the effect of pre-exposure to heat stress on the capacity of symbiotic algae to infect cnidarian hosts using the Aiptasia (sea-anemone)-Symbiodiniaceae model system. We found that the symbiont strain Breviolum sp. CS-164 (ITS2 type B1), both free-living and in symbiosis, loses the capacity to infect the host following exposure to heat stress. This loss of infectivity is reversible, however, a longer exposure to heat stress increases the time taken for reversal. Under the same experimental conditions, the loss of infectivity was not observed in another strain Breviolum psygmophilum CCMP2459 (ITS2 type B2). Our results suggest that recovery from bleaching can be limited by the loss of symbiont infectivity following exposure to heat stress.

[Source](https://doi.org/10.1038/s41396-020-00742-8)

10.1038/s41396-020-00742-8

---

## A cosmopolitan fungal pathogen of dicots adopts an endophytic lifestyle on cereal crops and protects them from major fungal diseases.
 06.11.2020, _B Tian, J Xie, Y Fu, J Cheng, B Li, T Chen, Y Zhao, Z Gao, P Yang, MJ Barbetti, BM Tyler, D Jiang_


Fungal pathogens are seriously threatening food security and natural ecosystems; efficient and environmentally friendly control methods are essential to help safeguard such resources for increasing human populations on a global scale. Here, we find that Sclerotinia sclerotiorum, a widespread pathogen of dicotyledons, can grow endophytically in wheat, rice, barley, maize, and oat, providing protection against Fusarium head blight, stripe rust, and rice blast. Protection is also provided by disabled S. sclerotiorum strains harboring a hypovirulence virus. The disabled strain DT-8 promoted wheat yields by 4-18% in the field and consistently reduced Fusarium disease by 40-60% across multiple field trials. We term the host-dependent trophism of S. sclerotiorum, destructively pathogenic or mutualistically endophytic, as schizotrophism. As a biotroph, S. sclerotiorum modified the expression of wheat genes involved in disease resistance and photosynthesis and increased the level of IAA. Our study shows that a broad-spectrum pathogen of one group of plants may be employed as a biocontrol agent in a different group of plants where they can be utilized as beneficial microorganisms while avoiding the risk of in-field release of pathogens. Our study also raises provocative questions about the potential role of schizotrophic endophytes in natural ecosystems.

[Source](https://doi.org/10.1038/s41396-020-00744-6)

10.1038/s41396-020-00744-6

---

## Spatio-temporal variations in bacterial and fungal community associated with dust aerosol in Kuwait.
 05.11.2020, _F Al Salameen, N Habibi, S Uddin, K Al Mataqi, V Kumar, B Al Doaij, S Al Amad, E Al Ali, F Shirshikhar_


Kuwait is a country with a very high dust loading; in fact it bears the world's highest particulate matter concentration in the outdoor air. The airborne dust often has associated biological materials, including pathogenic microbes that pose a serious risk to the urban ecosystem and public health. This study has established the baseline taxonomic characterization of microbes associated with dust transported into Kuwait from different trajectories. A high volume air sampler with six-stage cascade impactor was deployed for sample collection at a remote as well as an urban site. Samples from three different seasons (autumn, spring and summer) were subjected to targeted amplicon sequencing. A set of ~ 50 and 60 bacterial and fungal genera, respectively, established the core air microbiome. The predominant bacterial genera (relative abundance ≥ 1%) were Brevundimonas (12.5%), Sphingobium (3.3%), Sphingopyxis (2.7%), Pseudomonas (2.5%), Sphingomonas (2.4%), Massilia (2.3%), Acidovorax (2.0%), Allorhizobium (1.8%), Halomonas (1.3%), and Mesorhizobium (1.1%), and the fungal taxa were Cryptococcus (12%) followed by Alternaria (9%), Aspergillus (7%), Candida (3%), Cladosporium (2.9%), Schizophyllum (1.6%), Fusarium (1.4%), Gleotinia (1.3%) and Penicillium (1.15%). Significant spatio-temporal variations were recorded in terms of relative abundances, α-diversities, and β-diversities of bacterial communities. The dissimilarities were less pronounced and instead the communities were fairly homogenous. Linear discrimant analysis revealed three fungal genera known to be significantly differentially abundant with respect to different size fractions of dust. Our results shed light on the spatio-temporal distribution of airborne microbes and their implications in general health.

[Source](https://doi.org/10.1371/journal.pone.0241283)

10.1371/journal.pone.0241283

---

## Environmental heterogeneity determines the ecological processes that govern bacterial metacommunity assembly in a floodplain river system

[Source](https://doi.org/10.1038/s41396-020-0723-2)

---

## A large-scale assessment of lakes reveals a pervasive signal of land use on bacterial communities.
 06.11.2020, _SA Kraemer, N Barbosa da Costa, BJ Shapiro, M Fradette, Y Huot, DA Walsh_


Lakes play a pivotal role in ecological and biogeochemical processes and have been described as "sentinels" of environmental change. Assessing "lake health" across large geographic scales is critical to predict the stability of their ecosystem services and their vulnerability to anthropogenic disturbances. The LakePulse research network is tasked with the assessment of lake health across gradients of land use on a continental scale. Bacterial communities are an integral and rapidly responding component of lake ecosystems, yet large-scale responses to anthropogenic activity remain elusive. Here, we assess the ecological impact of land use on bacterial communities from over 200 lakes covering more than 660,000 km2 across Eastern Canada. In addition to community variation between ecozones, land use across Eastern Canada also appeared to alter diversity, community composition, and network structure. Specifically, increasing anthropogenic impact within the watershed lowered diversity. Likewise, community composition was significantly correlated with agriculture and urban development within a watershed. Interaction networks showed decreasing complexity and fewer keystone taxa in impacted lakes. Moreover, we identified potential indicator taxa of high or low lake water quality. Together, these findings point to detectable bacterial community changes of largely unknown consequences induced by human activity within lake watersheds.

[Source](https://doi.org/10.1038/s41396-020-0733-0)

10.1038/s41396-020-0733-0

---

## Viral elements and their potential influence on microbial processes along the permanently stratified Cariaco Basin redoxcline.
 06.11.2020, _P Mara, D Vik, MG Pachiadaki, EA Suter, B Poulos, GT Taylor, MB Sullivan, VP Edgcomb_


Little is known about viruses in oxygen-deficient water columns (ODWCs). In surface ocean waters, viruses are known to act as gene vectors among susceptible hosts. Some of these genes may have metabolic functions and are thus termed auxiliary metabolic genes (AMGs). AMGs introduced to new hosts by viruses can enhance viral replication and/or potentially affect biogeochemical cycles by modulating key microbial pathways. Here we identify 748 viral populations that cluster into 94 genera along a vertical geochemical gradient in the Cariaco Basin, a permanently stratified and euxinic ocean basin. The viral communities in this ODWC appear to be relatively novel as 80 of these viral genera contained no reference viral sequences, likely due to the isolation and unique features of this system. We identify viral elements that encode AMGs implicated in distinctive processes, such as sulfur cycling, acetate fermentation, signal transduction, [Fe-S] formation, and N-glycosylation. These AMG-encoding viruses include two putative Mu-like viruses, and viral-like regions that may constitute degraded prophages that have been modified by transposable elements. Our results provide an insight into the ecological and biogeochemical impact of viruses oxygen-depleted and euxinic habitats.

[Source](https://doi.org/10.1038/s41396-020-00739-3)

10.1038/s41396-020-00739-3

---

## Metagenomic Analysis of the Enteric RNA Virome of Infants from the Oukasie Clinic, North West Province, South Africa, Reveals Diverse Eukaryotic Viruses.
 10.11.2020, _MT Mogotsi, PN Mwangi, PA Bester, MJ Mphahlele, ML Seheri, HG O'Neill, MM Nyaga_


Establishing a diverse gut microbiota after birth is essential for preventing illnesses later in life. However, little knowledge exists about the total viral population (virome) present in the gut of infants during the early developmental stage, with RNA viruses being generally overlooked. Therefore, this small pilot longitudinal study investigated the diversity and changes in the enteric RNA virome in healthy infants from South Africa. Faecal samples (n = 12) were collected from four infants at three time points (on average at 8, 13, and 25 weeks), and then sequenced on an Illumina MiSeq platform. The genomic analysis revealed a diverse population of human enteric viruses from the infants' stools, and changes in the enteric virome composition were observed over time. The Reoviridae family, more specifically the Rotavirus genus, was the most common and could be linked to viral shedding due to the administration of live-attenuated oral vaccines in South Africa, followed by the Picornaviridae family including parechoviruses, echoviruses, coxsackieviruses, enteroviruses, and polioviruses. Polioviruses were also linked to vaccine-related shedding. Astroviridae (astroviruses) and Caliciviridae (noroviruses) were present at low abundance. It is evident that an infant's gut is colonized by distinct viral populations irrespective of their health state. Further characterization of the human virome (with a larger participant pool) is imperative to provide more conclusive insights into the viral community structure and diversity that has been shown in the current study, despite the smaller sample size.

[Source](https://doi.org/10.3390/v12111260)

10.3390/v12111260

---

## PPR-Meta: a tool for identifying phages and plasmids from metagenomic fragments using deep learning.
 21.04.2020, _Z Fang, J Tan, S Wu, M Li, C Xu, Z Xie, H Zhu_


_Bacteriophages, Deep Learning, Metagenome, Metagenomics, Plasmids, Software_
Phages and plasmids are the major components of mobile genetic elements, and fragments from such elements generally co-exist with chromosome-derived fragments in sequenced metagenomic data. However, there is a lack of efficient methods that can simultaneously identify phages and plasmids in metagenomic data, and the existing tools identifying either phages or plasmids have not yet presented satisfactory performance.

[Source](https://doi.org/10.1007/s40484-019-0187-4)

10.1093/gigascience/giz066

---

## Identifying viruses from metagenomic data using deep learning

[Source](https://doi.org/10.1007/s40484-019-0187-4)

---

## Reconstructing organisms in silico: genome-scale models and their emerging applications.
 24.09.2020, _X Fang, CJ Lloyd, BO Palsson_


Escherichia coli is considered to be the best-known microorganism given the large number of published studies detailing its genes, its genome and the biochemical functions of its molecular components. This vast literature has been systematically assembled into a reconstruction of the biochemical reaction networks that underlie E. coli's functions, a process which is now being applied to an increasing number of microorganisms. Genome-scale reconstructed networks are organized and systematized knowledge bases that have multiple uses, including conversion into computational models that interpret and predict phenotypic states and the consequences of environmental and genetic perturbations. These genome-scale models (GEMs) now enable us to develop pan-genome analyses that provide mechanistic insights, detail the selection pressures on proteome allocation and address stress phenotypes. In this Review, we first discuss the overall development of GEMs and their applications. Next, we review the evolution of the most complete GEM that has been developed to date: the E. coli GEM. Finally, we explore three emerging areas in genome-scale modelling of microbial phenotypes: collections of strain-specific models, metabolic and macromolecular expression models, and simulation of stress responses.

[Source](https://doi.org/10.1038/s41579-020-00440-4)

10.1038/s41579-020-00440-4

---

## Zero-Inflated gaussian mixed models for analyzing longitudinal microbiome data.
 09.11.2020, _X Zhang, B Guo, N Yi_


The human microbiome is variable and dynamic in nature. Longitudinal studies could explain the mechanisms in maintaining the microbiome in health or causing dysbiosis in disease. However, it remains challenging to properly analyze the longitudinal microbiome data from either 16S rRNA or metagenome shotgun sequencing studies, output as proportions or counts. Most microbiome data are sparse, requiring statistical models to handle zero-inflation. Moreover, longitudinal design induces correlation among the samples and thus further complicates the analysis and interpretation of the microbiome data.

[Source](https://doi.org/10.1371/journal.pone.0242073)

10.1371/journal.pone.0242073

---

## Evaluating the number of different genomes in a metagenome by means of the compositional spectra approach

[Source](https://doi.org/10.1371/journal.pone.0237205)

---

## Survey of metaproteomics software tools for functional microbiome analysis.
 10.11.2020, _R Sajulga, C Easterly, M Riffle, B Mesuere, T Muth, S Mehta, P Kumar, J Johnson, BA Gruening, H Schiebenhoefer, CA Kolmeder, S Fuchs, BL Nunn, J Rudney, TJ Griffin, PD Jagtap_


To gain a thorough appreciation of microbiome dynamics, researchers characterize the functional relevance of expressed microbial genes or proteins. This can be accomplished through metaproteomics, which characterizes the protein expression of microbiomes. Several software tools exist for analyzing microbiomes at the functional level by measuring their combined proteome-level response to environmental perturbations. In this survey, we explore the performance of six available tools, to enable researchers to make informed decisions regarding software choice based on their research goals. Tandem mass spectrometry-based proteomic data obtained from dental caries plaque samples grown with and without sucrose in paired biofilm reactors were used as representative data for this evaluation. Microbial peptides from one sample pair were identified by the X! tandem search algorithm via SearchGUI and subjected to functional analysis using software tools including eggNOG-mapper, MEGAN5, MetaGOmics, MetaProteomeAnalyzer (MPA), ProPHAnE, and Unipept to generate functional annotation through Gene Ontology (GO) terms. Among these software tools, notable differences in functional annotation were detected after comparing differentially expressed protein functional groups. Based on the generated GO terms of these tools we performed a peptide-level comparison to evaluate the quality of their functional annotations. A BLAST analysis against the NCBI non-redundant database revealed that the sensitivity and specificity of functional annotation varied between tools. For example, eggNOG-mapper mapped to the most number of GO terms, while Unipept generated more accurate GO terms. Based on our evaluation, metaproteomics researchers can choose the software according to their analytical needs and developers can use the resulting feedback to further optimize their algorithms. To make more of these tools accessible via scalable metaproteomics workflows, eggNOG-mapper and Unipept 4.0 were incorporated into the Galaxy platform.

[Source](https://doi.org/10.1371/journal.pone.0241503)

10.1371/journal.pone.0241503

---

## Sample pooling methods for efficient pathogen screening: Practical implications.
 11.11.2020, _TN Furstenau, JH Cocking, CM Hepp, VY Fofanov_


Due to the large number of negative tests, individually screening large populations for rare pathogens can be wasteful and expensive. Sample pooling methods improve the efficiency of large-scale pathogen screening campaigns by reducing the number of tests and reagents required to accurately categorize positive and negative individuals. Such methods rely on group testing theory which mainly focuses on minimizing the total number of tests; however, many other practical concerns and tradeoffs must be considered when choosing an appropriate method for a given set of circumstances. Here we use computational simulations to determine how several theoretical approaches compare in terms of (a) the number of tests, to minimize costs and save reagents, (b) the number of sequential steps, to reduce the time it takes to complete the assay, (c) the number of samples per pool, to avoid the limits of detection, (d) simplicity, to reduce the risk of human error, and (e) robustness, to poor estimates of the number of positive samples. We found that established methods often perform very well in one area but very poorly in others. Therefore, we introduce and validate a new method which performs fairly well across each of the above criteria making it a good general use approach.

[Source](https://doi.org/10.1371/journal.pone.0236849)

10.1371/journal.pone.0236849

---

## Genomes OnLine Database (GOLD) v.8: overview and updates.
 05.11.2020, _S Mukherjee, D Stamatis, J Bertsch, G Ovchinnikova, JC Sundaramurthi, J Lee, M Kandimalla, IA Chen, NC Kyrpides, TBK Reddy_


The Genomes OnLine Database (GOLD) (https://gold.jgi.doe.gov/) is a manually curated, daily updated collection of genome projects and their metadata accumulated from around the world. The current version of the database includes over 1.17 million entries organized broadly into Studies (45 770), Organisms (387 382) or Biosamples (101 207), Sequencing Projects (355 364) and Analysis Projects (283 481). These four levels contain over 600 metadata fields, which includes 76 controlled vocabulary (CV) tables containing 3873 terms. GOLD provides an interactive web user interface for browsing and searching by a wide range of project and metadata fields. Users can enter details about their own projects in GOLD, which acts as a gatekeeper to ensure that metadata is accurately documented before submitting sequence information to the Integrated Microbial Genomes (IMG) system for analysis. In order to maintain a reference dataset for use by members of the scientific community, GOLD also imports projects from public repositories such as GenBank and SRA. The current status of the database, along with recent updates and improvements are described in this manuscript.

[Source](https://doi.org/10.1093/nar/gkaa983)

10.1093/nar/gkaa983

---

