## Population bottlenecks constrain host microbiome diversity and genetic variation impeding fitness.
 23.05.2022, _M Ørsted, E Yashiro, AA Hoffmann, TN Kristensen_


It is becoming increasingly clear that microbial symbionts influence key aspects of their host's fitness, and vice versa. This may fundamentally change our thinking about how microbes and hosts interact in influencing fitness and adaptation to changing environments. Here we explore how reductions in population size commonly experienced by threatened species influence microbiome diversity. Consequences of such reductions are normally interpreted in terms of a loss of genetic variation, increased inbreeding and associated inbreeding depression. However, fitness effects of population bottlenecks might also be mediated through microbiome diversity, such as through loss of functionally important microbes. Here we utilise 50 Drosophila melanogaster lines with different histories of population bottlenecks to explore these questions. The lines were phenotyped for egg-to-adult viability and their genomes sequenced to estimate genetic variation. The bacterial 16S rRNA gene was amplified in these lines to investigate microbial diversity. We found that 1) host population bottlenecks constrained microbiome richness and diversity, 2) core microbiomes of hosts with low genetic variation were constituted from subsets of microbiomes found in flies with higher genetic variation, 3) both microbiome diversity and host genetic variation contributed to host population fitness, 4) connectivity and robustness of bacterial networks was low in the inbred lines regardless of host genetic variation, 5) reduced microbial diversity was associated with weaker evolutionary responses of hosts in stressful environments, and 6) these effects were unrelated to Wolbachia density. These findings suggest that population bottlenecks reduce hologenomic variation (combined host and microbial genetic variation). Thus, while the current biodiversity crisis focuses on population sizes and genetic variation of eukaryotes, an additional focal point should be the microbial diversity carried by the eukaryotes, which in turn may influence host fitness and adaptability with consequences for the persistence of populations.

[Source](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1010206)

10.1371/journal.pgen.1010206

---

## Diversity, abundance, and host specificity of the human skin associated circular and single stranded DNA virome

[Source](https://www.biorxiv.org/content/10.1101/2022.05.22.492996v1.abstract)

---

## Climate change, the cutaneous microbiome and skin disease: implications for a warming world.
 22.05.2022, _MF Isler, SJ Coates, MD Boos_


The skin plays an important role in human health by providing barrier protection against environmental stressors. In addition to human skin cells, the cutaneous barrier is also home to a network of organisms that have co-evolved with humans, referred to as the cutaneous microbiome. This network has been demonstrated to play an active role in skin health and the manifestation of cutaneous disease. Here, we review how a warming world and its attendant changes in climatic variables, including temperature, humidity, ultraviolet radiation, and air pollution, influence the cutaneous microbiome and, in turn, skin health. Studies indicate that the cutaneous microbiome is affected by these factors, and these changes may influence the epidemiology and severity of cutaneous disorders including atopic dermatitis, acne vulgaris, psoriasis, and skin cancer. Further investigation into how the cutaneous microbiome changes in response to climate change and subsequently influences skin disease is needed to better anticipate future dermatologic needs and potentially generate novel therapeutic solutions in response.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/ijd.16297)

10.1111/ijd.16297

---

## Species- and strain-level assessment using rrn long-amplicons suggests donor's influence on gut microbial transference via fecal transplants in metabolic syndrome subjects.
 25.05.2022, _A Benítez-Páez, AV Hartstra, M Nieuwdorp, Y Sanz_


_Bacteria, Fecal Microbiota Transplantation, Feces, Gastrointestinal Microbiome, Humans, Metabolic Syndrome, Nucleotides_
Fecal microbiota transplantation (FMT) is currently used for treating Clostridium difficile infection and explored for other clinical applications in experimental trials. However, the effectiveness of this therapy could vary, and partly depend on the donor's bacterial species engraftment, whose evaluation is challenging because there are no cost-effective strategies for accurately tracking the microbe transference. In this regard, the precise identification of bacterial species inhabiting the human gut is essential to define their role in human health unambiguously. We used Nanopore-based device to sequence bacterial rrn operons (16S-ITS-23S) and to reveal species-level abundance changes in the human gut microbiota of a FMT trial. By assessing the donor and recipient microbiota before and after FMT, we further evaluated whether this molecular approach reveals strain-level genetic variation to demonstrate microbe transfer and engraftment. Strict control over sequencing data quality and major microbiota covariates was critical for accurately estimating the changes in gut microbial species abundance in the recipients after FMT. We detected strain-level variation via single-nucleotide variants (SNVs) at rrn regions in a species-specific manner. We showed that it was possible to explore successfully the donor-bacterial strain (e.g., Parabacteroides merdae) engraftment in recipients of the FMT by assessing the nucleotide frequencies at rrn-associated SNVs. Our findings indicate that the engraftment of donors' microbiota is to some extent correlated with the improvement of metabolic health in recipients and that parameters such as the baseline gut microbiota configuration, sex, and age of donors should be considered to ensure the success of FMT in humans. The study was prospectively registered at the Dutch Trial registry - NTR4488 (https://www.trialregister.nl/trial/4488).

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2022.2078621)

10.1080/19490976.2022.2078621

---

## Greater alteration of gut microbiota occurs in childhood obesity than adulthood obesity

[Source](https://www.biorxiv.org/content/10.1101/2022.05.22.492992v1.abstract)

---

## Higher levels of Bifidobacteria and tumor necrosis factor in children with drug-resistant epilepsy are associated with anti-seizure response to the ketogenic diet

[Source](https://www.sciencedirect.com/science/article/pii/S2352396422002420)

---

## Towards Strain-Level Complexity: Sequencing Depth Required for Comprehensive Single-Nucleotide Polymorphism Analysis of the Human Gut Microbiome.
 24.05.2022, _P Liu, S Hu, Z He, C Feng, G Dong, S An, R Liu, F Xu, Y Chen, X Ying_


Intestinal bacteria strains play crucial roles in maintaining host health. Researchers have increasingly recognized the importance of strain-level analysis in metagenomic studies. Many analysis tools and several cutting-edge sequencing techniques like single cell sequencing have been proposed to decipher strains in metagenomes. However, strain-level complexity is far from being well characterized up to date. As the indicator of strain-level complexity, metagenomic single-nucleotide polymorphisms (SNPs) have been utilized to disentangle conspecific strains. Lots of SNP-based tools have been developed to identify strains in metagenomes. However, the sufficient sequencing depth for SNP and strain-level analysis remains unclear. We conducted ultra-deep sequencing of the human gut microbiome and constructed an unbiased framework to perform reliable SNP analysis. SNP profiles of the human gut metagenome by ultra-deep sequencing were obtained. SNPs identified from conventional and ultra-deep sequencing data were thoroughly compared and the relationship between SNP identification and sequencing depth were investigated. The results show that the commonly used shallow-depth sequencing is incapable to support a systematic metagenomic SNP discovery. In contrast, ultra-deep sequencing could detect more functionally important SNPs, which leads to reliable downstream analyses and novel discoveries. We also constructed a machine learning model to provide guidance for researchers to determine the optimal sequencing depth for their projects (SNPsnp, https://github.com/labomics/SNPsnp). To conclude, the SNP profiles based on ultra-deep sequencing data extend current knowledge on metagenomics and highlights the importance of evaluating sequencing depth before starting SNP analysis. This study provides new ideas and references for future strain-level investigations.

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2022.828254/full)

10.3389/fmicb.2022.828254

---

## Comparative Gut Microbiome in Trachypithecus leucocephalus and Other Primates in Guangxi, China, Based on Metagenome Sequencing.
 24.05.2022, _T Que, X Pang, H Huang, P Chen, Y Wei, Y Hua, H Liao, J Wu, S Li, A Wu, M He, X Ruan, Y Hu_


_Animals, Calcium Carbonate, China, Colobinae, Gastrointestinal Microbiome, Metagenome_
The Trachypithecus leucocephalus (white-headed langur) is a highly endangered, karst-endemic primate species, inhabiting the karst limestone forest in Guangxi, Southwest China. How white-headed langurs adapted to karst limestone and special dietary remains unclear. It is the first time to study the correlation between the gut microbiome of primates and special dietary, and environment in Guangxi. In the study, 150 fecal samples are collected from nine primates in Guangxi, China. Metagenomic sequencing is used to analyze and compare the gut microbiome composition and diversity between white-headed langurs and other primates. Our results indicate that white-headed langurs has a higher diversity of microbiome than other primates, and the key microbiome are phylum Firmicutes, class Clostridia, family Lachnospiraceae, and genera Clostridiates and Ruminococcus, which are related to the digestion and degradation of cellulose. Ten genera are significantly more abundant in white-headed langurs and François' langur than in other primates, most of which are high-temperature microbiome. Functional analysis reveals that energy synthesis-related pathways and sugar metabolism-related pathways are less abundant in white-headed langurs and François' langur than in other primates. This phenomenon could be an adaptation mechanism of leaf-eating primates to low-energy diet. The gut microbiome of white-headed langurs is related to diet and karst limestone environment. This study could serve as a reference to design conservation breeding, manage conservation units, and determine conservation priorities.

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2022.872841/full)

10.3389/fcimb.2022.872841

---

## Bacterial Adaptation to Venom in Snakes and Arachnida.
 23.05.2022, _E Esmaeilishirazifard, L Usher, C Trim, H Denise, V Sangal, GH Tyson, A Barlow, KF Redway, JD Taylor, M Kremyda-Vlachou, S Davies, TD Loftus, MMG Lock, K Wright, A Dalby, LAS Snyder, W Wuster, S Trim, SA Moschos_


Animal venoms are considered sterile sources of antimicrobial compounds with strong membrane-disrupting activity against multidrug-resistant bacteria. However, venomous bite wound infections are common in developing nations. Investigating the envenomation organ and venom microbiota of five snake and two spider species, we observed venom community structures that depend on the host venomous animal species and evidenced recovery of viable microorganisms from black-necked spitting cobra (Naja nigricollis) and Indian ornamental tarantula (Poecilotheria regalis) venoms. Among the bacterial isolates recovered from N. nigricollis, we identified two venom-resistant, novel sequence types of Enterococcus faecalis whose genomes feature 16 virulence genes, indicating infectious potential, and 45 additional genes, nearly half of which improve bacterial membrane integrity. Our findings challenge the dogma of venom sterility and indicate an increased primary infection risk in the clinical management of venomous animal bite wounds. IMPORTANCE Notwithstanding their 3 to 5% mortality, the 2.7 million envenomation-related injuries occurring annually-predominantly across Africa, Asia, and Latin America-are also major causes of morbidity. Venom toxin-damaged tissue will develop infections in some 75% of envenomation victims, with E. faecalis being a common culprit of disease; however, such infections are generally considered to be independent of envenomation. Here, we provide evidence on venom microbiota across snakes and arachnida and report on the convergent evolution mechanisms that can facilitate adaptation to black-necked cobra venom in two independent E. faecalis strains, easily misidentified by biochemical diagnostics. Therefore, since inoculation with viable and virulence gene-harboring bacteria can occur during envenomation, acute infection risk management following envenomation is warranted, particularly for immunocompromised and malnourished victims in resource-limited settings. These results shed light on how bacteria evolve for survival in one of the most extreme environments on Earth and how venomous bites must be also treated for infections.

[Source](https://journals.asm.org/doi/10.1128/spectrum.02408-21)

10.1128/spectrum.02408-21

---

## Recent advances in understanding of fungal and oomycete effectors.
 25.05.2022, _RA Wilson, JM McDowell_


Fungal and oomycete pathogens secrete complex arrays of proteins and small RNAs to interface with plant-host targets and manipulate plant regulatory networks to the microbes' advantage. Research on these important virulence factors has been accelerated by improved genome sequences, refined bioinformatic prediction tools, and exploitation of efficient platforms for understanding effector gene expression and function. Recent studies have validated the expectation that oomycetes and fungi target many of the same sectors in immune signaling networks, but the specific host plant targets and modes of action are diverse. Effector research has also contributed to deeper understanding of the mechanisms of effector-triggered immunity.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1369526622000577)

10.1016/j.pbi.2022.102228

---

## Fungi associated with the potato taste defect in coffee beans from Rwanda.
 24.05.2022, _AR Hale, PM Ruegger, P Rolshausen, J Borneman, JI Yang_


Potato taste defect (PTD) of coffee is characterized by a raw potato like smell that leads to a lower quality taste in the brewed coffee, and harms the commercial value of some East African coffees. Although several causes for PTD have been proposed, none of them have been confirmed. Recently, high throughput sequencing techniques and bioinformatic analysis have shown great potential for identifying putative causal agents of plant diseases. Toward the goal of determining the cause of PTD, we examined raw coffee beans from Rwanda exhibiting varying PTD scores using an Illumina-based sequence analysis of the fungal rRNA ITS region.

[Source](https://as-botanicalstudies.springeropen.com/articles/10.1186/s40529-022-00346-9)

10.1186/s40529-022-00346-9

---

## Delineating the Bacteriome of Packaged and Loose Smokeless Tobacco Products Available in North India.
 23.05.2022, _S Srivastava, M Sajid, H Singh, M Bharadwaj_


Smokeless tobacco product (STP) consumption is a significant public health threat across the globe. STPs are not only a storehouse of carcinogens and toxicants but also harbor microbes that aid in the conversion of tobacco alkaloids to carcinogenic tobacco-specific nitrosamines (TSNAs), thereby posing a further threat to the health of its consumers. The present study analyzed the bacterial diversity of popular dry and loose STPs by 16S rRNA gene sequencing. This NGS-based investigation revealed four dominant phyla Actinobacteria, Bacteroidetes, Firmicutes, and Proteobacteria and identified 549 genera, Prevotella, Bacteroides, and Lactobacillus constituting the core bacteriome of these STPs. The most significantly diverse bacteriome profile was displayed by the loose STP Mainpuri kapoori. The study further predicted the functional attributes of the prevalent genera by Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt) algorithm. Genes encoding for nitrate and nitrite reduction and transport enzymes, antibiotic resistance, multi-drug transporters and efflux pumps, secretion of endo- and exotoxin, and other pro-inflammatory molecules were identified. The loose STPs showed the highest level of nitrogen metabolism genes which can contribute to the synthesis of TSNAs. This study reveals the bacteriome of Indian domestic loose STPs that stagger behind in manufacturing and storage stringencies. Our results raise an alarm that the consumption of STPs harboring pathogenic genera can potentially lead to the onset of several oral and systemic diseases. Nevertheless, an in-depth correlation analysis of the microbial diversity of STPs and their elicit impact on consumer health is warranted. KEY POINTS: • Smokeless tobacco harbors bacteria that aid in synthesis of carcinogenic nitrosamines. • Most diverse bacteriome profile was displayed by loose smokeless tobacco products. • Pathogenic genera in these products can harm the oral and systemic health of users.

[Source](https://link.springer.com/article/10.1007/s00253-022-11979-y)

10.1007/s00253-022-11979-y

---

## Diversity, distribution and ecology of fungal communities present in Antarctic lake sediments uncovered by DNA metabarcoding.
 23.05.2022, _LMD de Souza, JM Lirio, SH Coria, FAC Lopes, P Convey, M Carvalho-Silva, FS de Oliveira, CA Rosa, PEAS Câmara, LH Rosa_


_Antarctic Regions, Ascomycota, Biodiversity, DNA, DNA Barcoding, Taxonomic, DNA, Fungal, Fungi, Lakes, Mycobiome_
We assessed fungal diversity in sediments obtained from four lakes in the South Shetland Islands and James Ross Island, Antarctica, using DNA metabarcoding. We detected 218 amplicon sequence variants (ASVs) dominated by the phyla Ascomycota, Basidiomycota, Mortierellomycota, Mucoromycota and Chytridiomycota. In addition, the rare phyla Aphelidiomycota, Basidiobolomycota, Blastocladiomycota, Monoblepharomycota, Rozellomycota and Zoopagomycota as well as fungal-like Straminopila belonging to the phyla Bacillariophyta and Oomycota were detected. The fungal assemblages were dominated by unknown fungal taxa (Fungal sp. 1 and Fungal sp. 2), followed by Talaromyces rubicundus and Dactylonectria anthuriicola. In general, they displayed high diversity, richness and moderate dominance. Sequences representing saprophytic, pathogenic and symbiotic fungi were detected, including the phytopathogenic fungus D. anthuriicola that was abundant, in the relatively young Soto Lake on Deception Island. The lake sediments studied contained the DNA of rich, diverse and complex fungal communities, including both fungi commonly reported in Antarctica and other taxa considered to be rare. However, as the study was based on the use of environmental DNA, which does not unequivocally confirm the presence of active or viable organisms, further studies using other approaches such as shotgun sequencing are required to elucidate the ecology of fungi in these Antarctic lake sediments.

[Source](https://www.nature.com/articles/s41598-022-12290-6)

10.1038/s41598-022-12290-6

---

