## Assessment of microbiota in the gut and upper respiratory tract associated with SARS-CoV-2 infection.
 07.03.2023, _J Li, Q Jing, J Li, M Hua, L Di, C Song, Y Huang, J Wang, C Chen, AR Wu_


_Humans, COVID-19, SARS-CoV-2, Microbiota, Gastrointestinal Microbiome, Nose_
The human microbiome plays an important role in modulating the host metabolism and immune system. Connections and interactions have been found between the microbiome of the gut and oral pharynx in the context of SARS-CoV-2 and other viral infections; hence, to broaden our understanding of host-viral responses in general and to deepen our knowledge of COVID-19, we performed a large-scale, systematic evaluation of the effect of SARS-CoV-2 infection on human microbiota in patients with varying disease severity.

[Source](https://doi.org/10.1186/s40168-022-01447-0)

10.1186/s40168-022-01447-0

---

## Population-level impacts of antibiotic usage on the human gut microbiome.
 06.03.2023, _K Lee, S Raguideau, K Sirén, F Asnicar, F Cumbo, F Hildebrand, N Segata, CJ Cha, C Quince_


_Humans, Gastrointestinal Microbiome, Anti-Bacterial Agents, Microbiota, Metagenome, Genome, Human_
The widespread usage of antimicrobials has driven the evolution of resistance in pathogenic microbes, both increased prevalence of antimicrobial resistance genes (ARGs) and their spread across species by horizontal gene transfer (HGT). However, the impact on the wider community of commensal microbes associated with the human body, the microbiome, is less well understood. Small-scale studies have determined the transient impacts of antibiotic consumption but we conduct an extensive survey of ARGs in 8972 metagenomes to determine the population-level impacts. Focusing on 3096 gut microbiomes from healthy individuals not taking antibiotics we demonstrate highly significant correlations between both the total ARG abundance and diversity and per capita antibiotic usage rates across ten countries spanning three continents. Samples from China were notable outliers. We use a collection of 154,723 human-associated metagenome assembled genomes (MAGs) to link these ARGs to taxa and detect HGT. This reveals that the correlations in ARG abundance are driven by multi-species mobile ARGs shared between pathogens and commensals, within a highly connected central component of the network of MAGs and ARGs. We also observe that individual human gut ARG profiles cluster into two types or resistotypes. The less frequent resistotype has higher overall ARG abundance, is associated with certain classes of resistance, and is linked to species-specific genes in the Proteobacteria on the periphery of the ARG network.

[Source](https://doi.org/10.1038/s41467-023-36633-7)

10.1038/s41467-023-36633-7

---

## A bacterial sulfoglycosidase highlights mucin O-glycan breakdown in the gut ecosystem.
 02.03.2023, _T Katoh, C Yamada, MD Wallace, A Yoshida, A Gotoh, M Arai, T Maeshibu, T Kashima, A Hagenbeek, MN Ojima, H Takada, M Sakanaka, H Shimizu, K Nishiyama, H Ashida, J Hirose, M Suarez-Diez, M Nishiyama, I Kimura, KA Stubbs, S Fushinobu, T Katayama_


Mucinolytic bacteria modulate host-microbiota symbiosis and dysbiosis through their ability to degrade mucin O-glycans. However, how and to what extent bacterial enzymes are involved in the breakdown process remains poorly understood. Here we focus on a glycoside hydrolase family 20 sulfoglycosidase (BbhII) from Bifidobacterium bifidum, which releases N-acetylglucosamine-6-sulfate from sulfated mucins. Glycomic analysis showed that, in addition to sulfatases, sulfoglycosidases are involved in mucin O-glycan breakdown in vivo and that the released N-acetylglucosamine-6-sulfate potentially affects gut microbial metabolism, both of which were also supported by a metagenomic data mining analysis. Enzymatic and structural analysis of BbhII reveals the architecture underlying its specificity and the presence of a GlcNAc-6S-specific carbohydrate-binding module (CBM) 32 with a distinct sugar recognition mode that B. bifidum takes advantage of to degrade mucin O-glycans. Comparative analysis of the genomes of prominent mucinolytic bacteria also highlights a CBM-dependent O-glycan breakdown strategy used by B. bifidum.

[Source](https://doi.org/10.1038/s41589-023-01272-y)

10.1038/s41589-023-01272-y

---

## Microbiota-derived genotoxin tilimycin generates colonic stem cell mutations.
 04.03.2023, _L Pöltl, M Kitsera, S Raffl, S Schild, A Cosic, S Kienesberger, K Unterhauser, G Raber, C Lembacher-Fadum, R Breinbauer, G Gorkiewicz, C Sebastian, G Hoefler, EL Zechner_


The DNA-alkylating metabolite tilimycin is a microbial genotoxin. Intestinal accumulation of tilimycin in individuals carrying til+ Klebsiella spp. causes apoptotic erosion of the epithelium and colitis. Renewal of the intestinal lining and response to injury requires the activities of stem cells located at the base of intestinal crypts. This study interrogates the consequences of tilimycin-induced DNA damage to cycling stem cells. We charted the spatial distribution and luminal quantities of til metabolites in Klebsiella-colonized mice in the context of a complex microbial community. Loss of marker gene G6pd function indicates genetic aberrations in colorectal stem cells that became stabilized in monoclonal mutant crypts. Mice colonized with tilimycin-producing Klebsiella displayed both higher frequencies of somatic mutation and more mutations per affected individual than animals carrying a non-producing mutant. Our findings imply that genotoxic til+ Klebsiella may drive somatic genetic change in the colon and increase disease susceptibility in human hosts.

[Source](https://doi.org/10.1016/j.celrep.2023.112199)

10.1016/j.celrep.2023.112199

---

## Impact of caloric restriction on the gut microbiota.
 03.03.2023, _L Kern, D Kviatcovsky, Y He, E Elinav_


Caloric restriction (CR) and related time-restricted diets have been popularized as means of preventing metabolic disease while improving general well-being. However, evidence as to their long-term efficacy, adverse effects, and mechanisms of activity remains incompletely understood. The gut microbiota is modulated by such dietary approaches, yet causal evidence to its possible downstream impacts on host metabolism remains elusive. Herein, we discuss the positive and adverse influences of restrictive dietary interventions on gut microbiota composition and function, and their collective impacts on host health and disease risk. We highlight known mechanisms of microbiota influences on the host, such as modulation of bioactive metabolites, while discussing challenges in achieving mechanistic dietary-microbiota insights, including interindividual variability in dietary responses as well as other methodological and conceptual challenges. In all, causally understanding the impact of CR approaches on the gut microbiota may enable to better decode their overall influences on human physiology and disease.

[Source](https://doi.org/10.1016/j.mib.2023.102287)

10.1016/j.mib.2023.102287

---

## <em>Romboutsia lituseburensis</em> JCM1404 supplementation ameliorated endothelial function via gut microbiota modulation and lipid metabolisms alterations in obese rats

[Source](https://doi.org/10.1093/femsle/fnad016)

---

## Disparate population and holobiont structure of pocilloporid corals across the Red Sea gradient demonstrate species-specific evolutionary trajectories.
 04.03.2023, _C Buitrago-López, A Cárdenas, BCC Hume, T Gosselin, F Staubach, M Aranda, DJ Barshis, Y Sawall, CR Voolstra_


Global habitat degradation heightens the need to better understand patterns of genetic connectivity and diversity of marine biota across geographical ranges to guide conservation efforts. Corals across the Red Sea are subject to pronounced environmental differences, but studies so far suggest that animal populations are largely connected, excepting evidence for a genetic break between the northern-central and southern regions. Here, we investigated population structure and holobiont assemblage of two common pocilloporid corals, Pocillopora verrucosa and Stylophora pistillata, across the Red Sea. We found little evidence for population differentiation in P. verrucosa, except for the southernmost site. Conversely, S. pistillata exhibited a complex population structure with evidence for within-reef and regional genetic differentiation, in line with differences in their reproductive mode (P. verrucosa is a broadcast spawner and S. pistillata is a brooder). Analysis for genomic loci under positive selection identified 85 sites (18 of which were in coding sequences) that distinguished the southern P. verrucosa population from the remainder of the Red Sea population. By comparison, we found 128 loci (24 of which were residing in coding sequences) in S. pistillata with evidence for local adaptation at various sites. Functional annotation of the underlying proteins revealed putative roles in the response to stress, lipid metabolism, transport, cytoskeletal rearrangement, and ciliary function (among others). Microbial assemblages of both coral species showed pervasive association with microalgal symbionts from the genus Symbiodinium (former clade A) and bacteria from the genus Endozoicomonas that exhibited significant differences according to host genotype and environment. The disparity of population genetic and holobiont assemblage patterns even between closely related species (family Pocilloporidae) highlights the need for multispecies investigations to better understand the role of the environment in shaping evolutionary trajectories. It further emphasizes the importance of networks of reef reserves to achieve conservation of genetic variants critical to the future survival of coral ecosystems.

[Source](https://doi.org/10.1111/mec.16871)

10.1111/mec.16871

---

## Omics and imaging combinatorial approach reveals butyrate-induced inflammatory effects in the zebrafish gut.
 06.03.2023, _A López Nadal, J Boekhorst, C Lute, F van den Berg, MA Schorn, T Bergen Eriksen, D Peggs, C McGurk, D Sipkema, M Kleerebezem, GF Wiegertjes, S Brugman_


Prebiotic feed additives aim to improve gut health by influencing the microbiota and the gut barrier. Most studies on feed additives concentrate on one or two (monodisciplinary) outcome parameters, such as immunity, growth, microbiota or intestinal architecture. A combinatorial and comprehensive approach to disclose the complex and multifaceted effects of feed additives is needed to understand their underlying mechanisms before making health benefit claims. Here, we used juvenile zebrafish as a model species to study effects of feed additives by integrating gut microbiota composition data and host gut transcriptomics with high-throughput quantitative histological analysis. Zebrafish received either control, sodium butyrate or saponin-supplemented feed. Butyrate-derived components such as butyric acid or sodium butyrate have been widely used in animal feeds due to their immunostimulant properties, thereby supporting intestinal health. Soy saponin is an antinutritional factor from soybean meal that promotes inflammation due to its amphipathic nature.

[Source](https://doi.org/10.1186/s42523-023-00230-2)

10.1186/s42523-023-00230-2

---

## Gut microbiome is linked to functions of peripheral immune cells in transition cows during excessive lipolysis.
 07.03.2023, _F Gu, S Zhu, Y Tang, X Liu, M Jia, N Malmuthuge, TG Valencak, JW McFadden, JX Liu, HZ Sun_


_Female, Animals, Cattle, Gastrointestinal Microbiome, Lipolysis, Bacteroides, Down-Regulation, Metabolome_
Postpartum dairy cows experiencing excessive lipolysis are prone to severe immunosuppression. Despite the extensive understanding of the gut microbial regulation of host immunity and metabolism, its role during excessive lipolysis in cows is largely unknown. Herein, we investigated the potential links between the gut microbiome and postpartum immunosuppression in periparturient dairy cows with excessive lipolysis using single immune cell transcriptome, 16S amplicon sequencing, metagenomics, and targeted metabolomics.

[Source](https://doi.org/10.1186/s40168-023-01492-3)

10.1186/s40168-023-01492-3

---

## Maternal effects drive intestinal development beginning in the embryonic period on the basis of maternal immune and microbial transfer in chickens.
 07.03.2023, _H Gong, T Wang, M Wu, Q Chu, H Lan, W Lang, L Zhu, Y Song, Y Zhou, Q Wen, J Yu, B Wang, X Zheng_


_Animals, Female, Humans, Chickens, Maternal Inheritance, Child Development, Gastrointestinal Microbiome, Gene Expression Profiling_
Nutrition drives immunity and health in animals, and maternal immunity benefits offspring. In our previous study, a nutritional intervention strategy was found to promote the immunity of hens, which subsequently improved immunity and growth in offspring chicks. Maternal effects clearly exist, but how are mothers' immune advantages transferred to their offspring, and how do they benefit them?

[Source](https://doi.org/10.1186/s40168-023-01490-5)

10.1186/s40168-023-01490-5

---

## Mouse mammary tumor virus is implicated in severity of colitis and dysbiosis in the IL-10-/- mouse model of inflammatory bowel disease.
 07.03.2023, _H Armstrong, M Rahbari, H Park, D Sharon, A Thiesen, N Hotte, N Sun, H Syed, H Abofayed, W Wang, K Madsen, E Wine, A Mason_


_Animals, Mice, Mice, Inbred C3H, Mammary Tumor Virus, Mouse, Interleukin-10, Dysbiosis, Colitis, Inflammatory Bowel Diseases, Disease Models, Animal_
Following viral infection, genetically manipulated mice lacking immunoregulatory function may develop colitis and dysbiosis in a strain-specific fashion that serves as a model for inflammatory bowel disease (IBD). We found that one such model of spontaneous colitis, the interleukin (IL)-10 knockout (IL-10-/-) model derived from the SvEv mouse, had evidence of increased Mouse mammary tumor virus (MMTV) viral RNA expression compared to the SvEv wild type. MMTV is endemic in several mouse strains as an endogenously encoded Betaretrovirus that is passaged as an exogenous agent in breast milk. As MMTV requires a viral superantigen to replicate in the gut-associated lymphoid tissue prior to the development of systemic infection, we evaluated whether MMTV may contribute to the development of colitis in the IL-10-/- model.

[Source](https://doi.org/10.1186/s40168-023-01483-4)

10.1186/s40168-023-01483-4

---

## Microplastics affect arsenic bioavailability by altering gut microbiota and metabolites in a mouse model

[Source](https://doi.org/10.1016/j.envpol.2023.121376)

---

## Substantial viral and bacterial diversity at the bat-tick interface.
 06.03.2023, _AS Ortiz-Baez, TGT Jaenson, EC Holmes, JH Pettersson, P Wilhelmsson_


_Humans, Animals, Chiroptera, Argasidae, Bacteria, Disease Outbreaks, Europe_
Ticks harbour a high diversity of viruses, bacteria and protozoa. The soft tick Carios vespertilionis (Argasidae) is a common ectoparasite of bats in the Palearctic region and is suspected to be vector and reservoir of viruses and other microbial species in bat populations, some of which may act as zoonotic agents for human disease. The Soprano pipistrelle (Pipistrellus pygmaeus, Vespertilionidae) is widely distributed in Europe, where it can be found inside or close to human habitation. We used meta-transcriptomic sequencing to determine the RNA virome and common microbiota in blood-fed C. vespertilionis ticks collected from a Soprano pipistrelle bat roosting site in south-central Sweden. Our analyses identified 16 viruses from 11 virus families, of which 15 viruses were novel. For the first time in Sweden we identified Issuk-Kul virus, a zoonotic arthropod-borne virus previously associated with outbreaks of acute febrile illness in humans. Probable bat-associated and tick-borne viruses were classified within the families Nairoviridae, Caliciviridae and Hepeviridae, while other invertebrate-associated viruses included members of the Dicistroviridae, Iflaviridae, Nodaviridae, Partitiviridae, Permutotetraviridae, Polycipiviridae and Solemoviridae. Similarly, we found abundant bacteria in C. vespertilionis, including genera with known tick-borne bacteria, such as Coxiella spp. and Rickettsia spp. These findings demonstrate the remarkable diversity of RNA viruses and bacteria present in C. vespertilionis and highlight the importance of bat-associated ectoparasite surveillance as an effective and non-invasive means to track viruses and bacteria circulating in bats and ticks.

[Source](https://doi.org/10.1099/mgen.0.000942)

10.1099/mgen.0.000942

---

## Perturbation of maternal gut microbiota in mice during a critical perinatal window influences early neurobehavioral outcomes in offspring.
 04.03.2023, _C Morel, I Martinez Sanchez, Y Cherifi, N Chartrel, R Diaz Heijtz_


The gut microbiota is increasingly recognized as a key environmental factor that shapes host development and physiology, including neural circuits formation and function. Concurrently, there has been growing concern that early-life antibiotic exposure may alter brain developmental trajectories, increasing the risk for neurodevelopmental disorders such as autism spectrum disorder (ASD). Here, we assessed whether perturbation of the maternal gut microbiota in mice during a narrow critical perinatal window (last week of pregnancy and first three postnatal days), induced by exposure to a commonly used broad-spectrum oral antibiotic (ampicillin), influences offspring neurobehavioral outcomes relevant to ASD. Our results demonstrate that neonatal offspring from antibiotic-treated dams display an altered pattern of ultrasonic communication, which was more pronounced in males. Moreover, juvenile male, but not female, offspring from antibiotic-treated dams showed reduced social motivation and social interaction, as well as context-dependent anxiety-like behavior. However, no changes were observed in locomotor or exploratory activity. This behavioral phenotype of exposed juvenile males was associated with reduced gene expression of the oxytocin receptor (OXTR) and several tight-junction proteins in the prefrontal cortex, a key region involved in the regulation of social and emotional behavior, as well as a mild inflammatory response in the colon. Further, juvenile offspring from exposed dams also showed distinct alterations in several gut bacterial species, including, Lactobacillus murinus, and Parabacteroides goldsteinii. Overall, this study highlights the importance of the maternal microbiome in early-life, and how its perturbation by a widely used antibiotic could contribute to atypical social and emotional development of offspring in a sex-dependent manner.

[Source](https://doi.org/10.1016/j.neuropharm.2023.109479)

10.1016/j.neuropharm.2023.109479

---

## Local environment drives rapid shifts in composition and phylogenetic clustering of seagrass microbiomes.
 07.03.2023, _MR Kardish, JJ Stachowicz_


_Phylogeny, RNA, Ribosomal, 16S, Acclimatization, Cluster Analysis, Microbiota_
Plant microbiomes depend on environmental conditions, stochasticity, host species, and genotype identity. Eelgrass (Zostera marina) is a unique system for plant-microbe interactions as a marine angiosperm growing in a physiologically-challenging environment with anoxic sediment, periodic exposure to air at low tide, and fluctuations in water clarity and flow. We tested the influence of host origin versus environment on eelgrass microbiome composition by transplanting 768 plants among four sites within Bodega Harbor, CA. Over three months following transplantation, we sampled microbial communities monthly on leaves and roots and sequenced the V4-V5 region of the 16S rRNA gene to assess community composition. The main driver of leaf and root microbiome composition was destination site; more modest effects of host origin site did not last longer than one month. Community phylogenetic analyses suggested that environmental filtering structures these communities, but the strength and nature of this filtering varies among sites and over time and roots and leaves show opposing gradients in clustering along a temperature gradient. We demonstrate that local environmental differences create rapid shifts in associated microbial community composition with potential functional implications for rapid host acclimation under shifting environmental conditions.

[Source](https://doi.org/10.1038/s41598-023-30194-x)

10.1038/s41598-023-30194-x

---

## Experimental community coalescence sheds light on microbial interactions in soil and restores impaired functions.
 07.03.2023, _S Huet, S Romdhane, MC Breuil, D Bru, A Mounier, A Spor, L Philippot_


_Humans, Microbial Interactions, Betaproteobacteria, Coculture Techniques, Microbiota, Soil_
Microbes typically live in communities where individuals can interact with each other in numerous ways. However, knowledge on the importance of these interactions is limited and derives mainly from studies using a limited number of species grown in coculture. Here, we manipulated soil microbial communities to assess the contribution of interactions between microorganisms for assembly of the soil microbiome.

[Source](https://doi.org/10.1186/s40168-023-01480-7)

10.1186/s40168-023-01480-7

---

## Infants’ gut microbiome data: A Bayesian Marginal Zero-inflated Negative Binomial regression model for multivariate analyses of count data

[Source](https://doi.org/10.1016/j.csbj.2023.02.027)

---

## From defaults to databases: parameter and database choice dramatically impact the performance of metagenomic taxonomic classification tools.
 07.03.2023, _RJ Wright, AM Comeau, MGI Langille_


_Humans, Metagenome, Microbiota, Databases, Factual, Metagenomics_
In metagenomic analyses of microbiomes, one of the first steps is usually the taxonomic classification of reads by comparison to a database of previously taxonomically classified genomes. While different studies comparing metagenomic taxonomic classification methods have determined that different tools are 'best', there are two tools that have been used the most to-date: Kraken (k-mer-based classification against a user-constructed database) and MetaPhlAn (classification by alignment to clade-specific marker genes), the latest versions of which are Kraken2 and MetaPhlAn 3, respectively. We found large discrepancies in both the proportion of reads that were classified as well as the number of species that were identified when we used both Kraken2 and MetaPhlAn 3 to classify reads within metagenomes from human-associated or environmental datasets. We then investigated which of these tools would give classifications closest to the real composition of metagenomic samples using a range of simulated and mock samples and examined the combined impact of tool-parameter-database choice on the taxonomic classifications given. This revealed that there may not be a one-size-fits-all 'best' choice. While Kraken2 can achieve better overall performance, with higher precision, recall and F1 scores, as well as alpha- and beta-diversity measures closer to the known composition than MetaPhlAn 3, the computational resources required for this may be prohibitive for many researchers, and the default database and parameters should not be used. We therefore conclude that the best tool-parameter-database choice for a particular application depends on the scientific question of interest, which performance metric is most important for this question and the limit of available computational resources.

[Source](https://doi.org/10.1099/mgen.0.000949)

10.1099/mgen.0.000949

---

## Using strain-resolved analysis to identify contamination in metagenomics data.
 06.03.2023, _YC Lou, J Hoff, MR Olm, J West-Roberts, S Diamond, BA Firek, MJ Morowitz, JF Banfield_


_Metagenomics, Biomass, DNA Contamination, Microbiota, DNA_
Metagenomics analyses can be negatively impacted by DNA contamination. While external sources of contamination such as DNA extraction kits have been widely reported and investigated, contamination originating within the study itself remains underreported.

[Source](https://doi.org/10.1186/s40168-023-01477-2)

10.1186/s40168-023-01477-2

---

## Transkingdom Network Analysis (TkNA): a systems approach inferring causal factors underlying host-microbiota interactions.
 03.03.2023, _NK Newman, M Macovsky, RR Rodrigues, AM Bruce, JW Pederson, SS Patil, J Padiadpu, AK Dzutsev, N Shulzhenko, G Trinchieri, K Brown, A Morgun_


Technological advances have generated tremendous amounts of high-throughput omics data. Integrating data from multiple cohorts and/or several omics types from new and previously published studies can offer a holistic view of a biological system and decipher its critical players and key mechanisms. In this protocol we describe how to use TkNA, a unique causal-inference analytical framework that can perform meta-analysis of cohorts and detect master regulators (causal factors) among measured parameters (e.g., microbes or microbial genes, host genes, metabolites) that govern pathological or physiological responses of host-microbiota interactions in a particular condition or disease. The first step reconstructs the network that represents a statistical model capturing the complex relationships between the different omics of the biological system. Here, it selects differential features and their per-group correlations by identifying robust and reproducible patterns of fold change direction and sign of correlation across several cohorts. Next, using a causality-reflecting metric (correlation inequalities), statistical thresholds (per cohort p-values, combined Fisher's p-value, and BH FDR), and a set of topological criteria (e.g., network sparsity, ratio of negative to positive correlations, and strengths of correlations) it selects the final edges that form the Transkingdom Network. The second part of the analysis involves interrogating the network. Using the network's local (degree) and global (Bipartite Betweenness Centrality (BiBC)) topology metrics it detects nodes that are responsible for control of given subnetwork or control of communication between kingdoms and/or subnetworks. The underlying basis of TkNA approach involves fundamental principles including laws of causality, graph theory and information theory. Hence, TkNA can be used for causal inference via network analysis of any host and/or microbiota multi-omics data. This protocol takes approximately few hours to days to run, depending on size of available data and computational resources. It is targeted towards researchers that might not have access to high-performance computing and requires very basic familiarity with the Unix command-line environment.

[Source](https://doi.org/10.1101/2023.02.22.529449)

10.1101/2023.02.22.529449

---

