## Bioaccumulation of therapeutic drugs by human gut bacteria.
 09.09.2021, _M Klünemann, S Andrejev, S Blasche, A Mateus, P Phapale, S Devendran, J Vappiani, B Simon, TA Scott, E Kafkia, D Konstantinidis, K Zirngibl, E Mastrorilli, M Banzhaf, MT Mackmull, F Hövelmann, L Nesme, AR Brochado, L Maier, T Bock, V Periwal, M Kumar, Y Kim, M Tramontano, C Schultz, M Beck, J Hennig, M Zimmermann, DC Sévin, F Cabreiro, MM Savitski, P Bork, A Typas, KR Patil_


Bacteria in the gut can modulate the availability and efficacy of therapeutic drugs. However, the systematic mapping of the interactions between drugs and bacteria has only started recently1 and the main underlying mechanism proposed is the chemical transformation of drugs by microorganisms (biotransformation). Here we investigated the depletion of 15 structurally diverse drugs by 25 representative strains of gut bacteria. This revealed 70 bacteria-drug interactions, 29 of which had not to our knowledge been reported before. Over half of the new interactions can be ascribed to bioaccumulation; that is, bacteria storing the drug intracellularly without chemically modifying it, and in most cases without the growth of the bacteria being affected. As a case in point, we studied the molecular basis of bioaccumulation of the widely used antidepressant duloxetine by using click chemistry, thermal proteome profiling and metabolomics. We find that duloxetine binds to several metabolic enzymes and changes the metabolite secretion of the respective bacteria. When tested in a defined microbial community of accumulators and non-accumulators, duloxetine markedly altered the composition of the community through metabolic cross-feeding. We further validated our findings in an animal model, showing that bioaccumulating bacteria attenuate the behavioural response of Caenorhabditis elegans to duloxetine. Together, our results show that bioaccumulation by gut bacteria may be a common mechanism that alters drug availability and bacterial metabolism, with implications for microbiota composition, pharmacokinetics, side effects and drug responses, probably in an individual manner.

[Source](https://doi.org/10.1038/s41586-021-03891-8)

10.1038/s41586-021-03891-8

---

## Eight Metagenome-Assembled Genomes Provide Evidence for Microbial Adaptation in 20,000- to 1,000,000-Year-Old Siberian Permafrost.
 10.09.2021, _K Sipes, A Almatari, A Eddie, D Williams, E Spirina, E Rivkina, R Liang, TC Onstott, TA Vishnivetskaya, KG Lloyd_


Permafrost microbes may be metabolically active in microscopic layers of liquid brines, even in ancient soil. Metagenomics can help discern whether permafrost microbes show adaptations to this environment. Thirty-three metagenome-assembled genomes (MAGs) were obtained from six depths (3.5 m to 20 m) of freshly cored permafrost from the Siberian Kolyma-Indigirka Lowland region. These soils have been continuously frozen for ∼20,000 to 1,000,000 years. Eight of these MAGs were ≥80% complete with &lt;10% contamination and were taxonomically identified as Aminicenantes, Atribacteria, Chloroflexi, and Actinobacteria within bacteria and Thermoprofundales within archaea. MAGs from these taxa have been obtained previously from nonpermafrost environments and have been suggested to show adaptations to long-term energy starvation, but they have never been explored in ancient permafrost. The permafrost MAGs had greater proportions in the Clusters of Orthologous Groups (COGs) categories of energy production and conversion and carbohydrate transport and metabolism than did their nonpermafrost counterparts. They also contained genes for trehalose synthesis, thymine metabolism, mevalonate biosynthesis, and cellulose degradation, which were less prevalent in nonpermafrost genomes. Many of these genes are involved in membrane stabilization and osmotic stress responses, consistent with adaptation to the anoxic, high-ionic-strength, cold environments of permafrost brine films. Our results suggest that this ancient permafrost contains DNA of high enough quality to assemble MAGs from microorganisms with adaptations to survive long-term freezing in this extreme environment. IMPORTANCE Permafrost around the world is thawing rapidly. Many scientists from a variety of disciplines have shown the importance of understanding what will happen to our ecosystem, commerce, and climate when permafrost thaws. The fate of permafrost microorganisms is connected to these predicted rapid environmental changes. Studying ancient permafrost with culture-independent techniques can give a glimpse into how these microorganisms function under these extreme low-temperature and low-energy conditions. This will facilitate understanding how they will change with the environment. This study presents genomic data from this unique environment ∼20,000 to 1,000,000 years of age.

[Source](https://doi.org/10.1128/AEM.00972-21)

10.1128/AEM.00972-21

---

## Analysis the Link between Periodontal Diseases and Alzheimer's Disease: A Systematic Review.
 10.09.2021, _L Borsa, M Dubois, G Sacco, L Lupi_


The hypothesis of an infectious connection from the oro-pharyngeal sphere to the brain underlines the interest in analyzing the link between periodontal disease and Alzheimer's disease. The aim of this systematic review was to examine the link between Alzheimer's disease and periodontal disease in patients aged 65 and over. Databases (PubMed (MEDLINE), the Cochrane Library, and Embase) were analyzed for relevant references up to 21 June 2021. The authors independently selected the studies and extracted the data. The quality of included studies was checked using the National Institutes of Health's quality assessment tools. Five studies were included. The selected studies described in their results an increase in F. nucleatum in Alzheimer's disease patients (adjusted p = 0.02), and its incidence was linked to C. rectus and P. gingivalis (adjusted HR = 1.22 (1.04-1.43), p = 0.012) as well as A. naeslundii (crude HR = 2.0 (1.1-3.8)). The presence of periodontitis at baseline was associated with a six-fold increase in the rate of cognitive decline over a 6-month follow-up period (ADAS-Cog mean change = 2.9 ± 6.6). The current review suggests an association between periodontal disease and Alzheimer's disease. The treatment of periodontal disease could be a way to explore Alzheimer's disease prevention.

[Source](https://doi.org/10.3390/ijerph18179312)

10.3390/ijerph18179312

---

## Transplantation of gut microbiota derived from Alzheimer's disease mouse model impairs memory function and neurogenesis in C57BL/6 mice.
 09.09.2021, _N Kim, S Ho Jeon, I Gyoung Ju, M Sung Gee, J Do, M Sook Oh, J Kil Lee_


Alzheimer's disease (AD) is a neurodegenerative disease that causes memory and cognitive decline. Although many studies have attempted to clarify the causes of AD occurrence, it is not clearly understood. Recently, the emerging role of the gut microbiota in neurodegenerative diseases, including AD, has received much attention. The gut microbiota composition of AD patients and AD mouse models is different from that of healthy controls, and these changes may affect the brain environment. However, the specific mechanisms by which gut microbiota that influence memory decline are currently unclear. In this study, we performed fecal microbiota transplantation (FMT) to clarify the role of 5xFAD mouse-derived microbiota in memory decline. We observed that FMT from 5xFAD mice into normal C57BL/6 mice (5xFAD-FMT) decreased adult hippocampal neurogenesis and brain-derived neurotrophic factor expression and increased p21 expression, resulting in memory impairment. Microglia in the hippocampus of the 5xFAD-FMT mice were activated, which caused the elevation of pro-inflammatory cytokines (tumor necrosis factor-α and interleukin-1β). Moreover, we observed that pro-inflammatory cytokines increased in the colon and plasma of 5xFAD-FMT mice. The gut microbiota composition of the 5xFAD-FMT mice was different from that of the control mice or wild type-FMT mice. Collectively, 5xFAD mouse-derived microbiota decreased neurogenesis by increasing colonic inflammation, thereby contributing to memory loss. Our findings provide further evidence concerning the role of gut microbial dysbiosis in AD pathogenesis and suggest that targeting the gut microbiota may be a useful therapeutic strategy for the development of novel candidates for the treatment of AD.

[Source](https://doi.org/10.1016/j.bbi.2021.09.002)

10.1016/j.bbi.2021.09.002

---

## Controllers of cutaneous regulatory T&nbsp;cells: ultraviolet radiation and the skin microbiome.

[Source](https://doi.org/10.1515/hsz-2021-0252)

---

## The effects of cigarette smoking on the salivary and tongue microbiome.
 10.09.2021, _N Suzuki, Y Nakano, M Yoneda, T Hirofuji, T Hanioka_


It has been suggested that smoking affects the oral microbiome, but its effects on sites other than the subgingival microbiome remain unclear. This study investigated the composition of the salivary and tongue bacterial communities of smokers and nonsmokers in periodontally healthy adults.

[Source](https://doi.org/10.1002/cre2.489)

10.1002/cre2.489

---

## Oral Microbiota Variation: A Risk Factor for Development and Poor Prognosis of Esophageal Cancer.
 10.09.2021, _X Song, OK Greiner-Tollersrud, H Zhou_


Recent studies have shown that oral microbiota play an important role in the esophageal cancer (EC) initiation and progression, suggesting that oral microbiota is a new risk factor for EC. The composition of the microbes inhabiting the oral cavity could be perturbed with continuous factors such as smoking, alcohol consumption, and inflammation. The microbial alteration involves the decrease of beneficial species and the increase of pathogenic species. Experimental evidences suggest a significant role of oral commensal organisms in protecting hosts against EC. By contrast, oral pathogens, especially Porphyromonas gingivalis and Fusobacterium nucleatum, give rise to the risk for developing EC through their pro-inflammatory and pro-tumorigenic activities. The presences of oral dysbiosis, microbial biofilm, and periodontitis in EC patients are found to be associated with invasive cancer phenotypes and poor prognosis. The mechanism of oral bacteria in EC progression is complex, which involves a combination of cytokines, chemokines, oncogenic signaling pathways, cell surface receptors, the degradation of extracellular matrix, and cell apoptosis. From a clinical perspective, good oral hygiene, professional oral care, and rational use of antibiotics bring positive impacts on oral microbial balance, thus helping individuals reduce the risk of EC, inhibiting postoperative complications among EC patients, and improving the efficiency of chemoradiotherapy. However, current oral hygiene practices mainly focus on the oral bacteria-based predictive and preventive purposes. It is still far from implementing microbiota-dependent regulation as a therapy for EC. Further explorations are needed to render oral microbiota a potential target for treating EC.

[Source](https://doi.org/10.1007/s10620-021-07245-2)

10.1007/s10620-021-07245-2

---

## The gastric microbiota in patients with Crohn's disease; a preliminary study.
 10.09.2021, _J Ostrowski, M Kulecka, I Zawada, N Żeber-Lubecka, A Paziewska, K Graca-Pakulska, K Dąbkowski, K Skubisz, P Cybula, F Ambrożkiewicz, E Urasińska, M Mikula, T Starzyńska_


The gastric microbiota in Crohn's disease (CD) has not been studied. The purpose of the study was to evaluate differences of stomach microbiota between CD patients and controls. DNA was extracted from gastric mucosal and fluid samples, from 24 CD patients and 19 controls. 16S rRNA gene sequencing identified 1511 operational taxonomic units (OTUs), of which 239 passed the low abundance and low variance filters. All but one CD patients were HP negative. Fifteen bacterial phyla were identified in at least one mucosal or fluid site. Of these, Bacteroidota and Firmicutes accounted for 70% of all phyla. Proteobacteria, Actinobacteriota, and Fusobacteriota combined accounted for 27%. There was significant difference in the relative abundance of Bacteroidota, Proteobacteria, Fusobacteriota, and Campilobacterota between CD patients and controls only in gastric corpus samples. In gastric liquid, there was a significant difference only in Actinobacteriota. Pairwise comparison identified 67 differentially abundant OTUs in at least one site. Of these, 13 were present in more than one comparison, and four differentiating OTUs (Neisseriaceae, Neisseria, Absconditabacteriales, and Microbacteriaceae) were identified at all tested sites. The results reveal significant changes in gastric microbial profiles (beta diversity, phylum, and individual taxa levels) between H. pylori-negative CD patients and controls.

[Source](https://doi.org/10.1038/s41598-021-97261-z)

10.1038/s41598-021-97261-z

---

## Saliva-derived microcosm biofilms grown on different oral surfaces in vitro.
 10.09.2021, _X Li, L Shang, BW Brandt, MJ Buijs, S Roffel, C van Loveren, W Crielaard, S Gibbs, DM Deng_


The microbial composition of a specific oral niche could be influenced by initial bacterial adherence, nutrient and physiological property of the local surface. To investigate the influence of nutrient and surface properties on microbial composition, saliva-derived biofilms were grown in agar on three substrata: Reconstructed Human Gingiva (RHG), a hydroxyapatite (HAP) surface, and a titanium (TI) surface. Agar was mixed with either Brain Heart Infusion (BHI) or Thompson (TP) medium. After 1, 3, or 5 days, biofilm viability (by colony forming units) and microbiome profiles (by 16 S rDNA amplicon sequencing) were determined. On RHG, biofilm viability and composition were similar between BHI and TP. However, on the abiotic substrata, biofilm properties greatly depended on the type of medium and substratum. In BHI, the viability of HAP-biofilm first decreased and then increased, whereas that of TI-biofilm decreased in time until a 6-log reduction. In TP, either no or a 2-log reduction in viability was observed for HAP- or TI-biofilms respectively. Furthermore, different bacterial genera (or higher level) were differentially abundant in the biofilms on 3 substrata: Haemophilus and Porphyromonas for RHG; Bacilli for HAP and Prevotella for TI. In conclusion, RHG, the biotic substratum, is able to support a highly viable and diverse microbiome. In contrast, the viability and diversity of the biofilms on the abiotic substrata were influenced by the substrata type, pH of the environment and the richness of the growth media. These results suggest that the host (oral mucosa) plays a vital role in the oral ecology.

[Source](https://doi.org/10.1038/s41522-021-00246-z)

10.1038/s41522-021-00246-z

---

## Plasma Metabolomic and Intestinal Microbial Analyses of Patients With Severe Aplastic Anemia.
 10.09.2021, _Y Shao, W Qi, X Zhang, N Ran, C Liu, R Fu, Z Shao_


Aplastic anemia results from bone marrow failure caused by an autoimmune abnormality, but the pathogenesis of severe aplastic anemia (SAA) is not well characterized. To identify potential metabolic markers of SAA and to further elucidate the pathogenetic mechanisms of SAA, we performed a metabolomic study of plasma samples and characterized the intestinal microbiota of patients with SAA and healthy controls. Patients with SAA had more Enterobacteriales and Lactobacillales, but fewer Bacteroidales, Clostridiales, and Erysipelotrichales than healthy controls. At the species level, the abundances of Escherichia coli and others including Clostridium citroniae were higher, whereas those of Prevotella copri, Roseburia faecis, and Ruminococcus bromii were lower. Eight metabolites showed significantly different plasma concentrations in the SAA and healthy control groups. Coumaric acid, L-phenylalanine, and sulfate were present at higher concentrations in the SAA group; whereas L-glutamic γ-semialdehyde, theobromine, 3a, 7a-dihydroxy-5b-cholestane, γ-δ-dioxovaleric acid, and (12Z)-9, 10-dihydroxyoctadec-12-enoic acid were present at lower concentrations. In conclusion, patients with SAA show abnormalities in both their plasma metabolomes and intestinal microbial compositions. These differences might reflect the molecular mechanisms involved in the defective immunity that characterizes SAA.

[Source](https://doi.org/10.3389/fcell.2021.669887)

10.3389/fcell.2021.669887

---

## Gastrointestinal mucosal immunity and COVID-19.
 10.09.2021, _T Velikova, V Snegarova, A Kukov, H Batselova, A Mihova, R Nakov_


_COVID-19, COVID-19 Vaccines, Gastrointestinal Tract, Humans, Immunity, Mucosal, Mucous Membrane, SARS-CoV-2_
As the gastrointestinal tract may also be a crucial entry or interaction site of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the role of the gut mucosal immune system as a first-line physical and immunological defense is critical. Furthermore, gastrointestinal involvement and symptoms in coronavirus disease 2019 (COVID-19) patients have been linked to worse clinical outcomes. This review discusses recent data on the interactions between the virus and the immune cells and molecules in the mucosa during the infection. By carrying out appropriate investigations, the mucosal immune system role in SARS-CoV-2 infection in therapy and prevention can be established. In line with this, COVID-19 vaccines that stimulate mucosal immunity against the virus may have more advantages than the others.

[Source](https://doi.org/10.3748/wjg.v27.i30.5047)

10.3748/wjg.v27.i30.5047

---

## Exploring the global vaginal microbiome and its impact on human health.
 09.09.2021, _LA Chacra, F Fenollar_


Around the world, more than 175,000,000 women are diagnosed every year with gynaecological disease, in many cases contributing to high morbidity and mortality. For this reason, knowledge of the composition of the vaginal microbiome and its variations represents a real health challenge, as this is key to improving therapeutic management. This review traces the history of the poorly known vaginal microbiome and focuses on the latest findings concerning this ecosystem. Studies in the past decade have targeted complex bacterial communities within the vagina. However, due to the development of technology and the emergence of next generation sequencing (NGS), the exact definition of the vaginal microbiome has changed and can no longer be linked solely to the presence of bacteria. In order to reach a global view of the vaginal microbiome, it is essential to take into account all microorganisms that the vagina harbours, including fungi, viruses, archaea, and candidate phyla radiation. Although these communities represent only a minimal percentage of the vaginal microbiome, they may act as modifiers of its basic physiology and may play a key role in the maintenance of microbial communities, as well as metabolic and immune functions. Studies of the complex interactions between these different microorganisms have recently begun and are not yet fully understood. Results to date indicate that these microbial communities together constitute the first line of defence against infections. On the other hand, the slightest disturbance in this microbiome may lead to disease. For this reason, enhanced knowledge of these associations is critical to better identify predispositions to certain illnesses, which may open new therapeutic avenues. Currently however, only the tip of the iceberg is understood and current research on this ecosystem is revolutionising our knowledge and understanding of human health and disease.

[Source](https://doi.org/10.1016/j.micpath.2021.105172)

10.1016/j.micpath.2021.105172

---

## Gut microbiome in acute pancreatitis: A review based on current literature.
 10.09.2021, _BK Patel, KH Patel, M Bhatia, SG Iyer, K Madhavan, SM Moochhala_


_Acute Disease, Dysbiosis, Gastrointestinal Microbiome, Humans, Pancreatitis, Prebiotics, Probiotics_
The gut microbiome is a complex microbial community, recognized for its potential role in physiology, health, and disease. The available evidence supports the role of gut dysbiosis in pancreatic disorders, including acute pancreatitis (AP). In AP, the presence of gut barrier damage resulting in increased mucosal permeability may lead to translocation of intestinal bacteria, necrosis of pancreatic and peripancreatic tissue, and infection, often accompanied by multiple organ dysfunction syndrome. Preserving gut microbial homeostasis may reduce the systemic effects of AP. A growing body of evidence suggests the possible involvement of the gut microbiome in various pancreatic diseases, including AP. This review discusses the possible role of the gut microbiome in AP. It highlights AP treatment and supplementation with prebiotics, synbiotics, and probiotics to maintain gastrointestinal microbial balance and effectively reduce hospitalization, morbidity and mortality in an early phase. It also addresses novel therapeutic areas in the gut microbiome, personalized treatment, and provides a roadmap of human microbial contributions to AP that have potential clinical benefit.

[Source](https://doi.org/10.3748/wjg.v27.i30.5019)

10.3748/wjg.v27.i30.5019

---

## Parkinson's Disease and Gut Microbiota.
 09.09.2021, _M Hirayama, K Ohno_


Parkinson's disease (PD) is caused by abnormal aggregation of α-synuclein fibrils, called the Lewy bodies, in the central nervous system. Accumulating knowledge points to the notion that α-synuclein fibrils start from the dorsal vagal nucleus and ascend to the locus ceruleus and the substantia nigra (SN). Even in healthy elderly subjects without motor or cognitive impairment, α-synuclein fibrils are frequently observed in the brain and sometimes in the intestinal neural plexus. Enteroendocrine cells have a direct synapse to the vagal afferents, and the vagal nucleus has synaptic pathways to the SN and the striatum. Intestinal bacteria are likely to be involved in the formation of intestinal α-synuclein fibrils.

[Source](https://doi.org/10.1159/000518147)

10.1159/000518147

---

## Dietary Pterostilbene and Resveratrol Modulate the Gut Microbiota Influenced by Circadian Rhythm Dysregulation.
 10.09.2021, _YC Koh, PS Lee, YL Kuo, K Nagabhushanam, CT Ho, MH Pan_


A causal relationship between circadian misalignment and microbiota dysbiosis has been discussed recently, due to their association to pathogenesis. Herein, the possible impact of pterostilbene and resveratrol on the gut microbiota brought by chronic jet-lag in mice was investigated.

[Source](https://doi.org/10.1002/mnfr.202100434)

10.1002/mnfr.202100434

---

## <em>Lactobacillus rhamnosus</em> Affects Rat Peritoneal Cavity Cell Response to Stimulation with Gut Microbiota: Focus on the Host Innate Immunity

[Source](https://doi.org/10.1007/s10753-021-01513-z)

---

## Fisetin Improves Hyperuricemia-Induced Chronic Kidney Disease via Regulating Gut Microbiota-Mediated Tryptophan Metabolism and Aryl Hydrocarbon Receptor Activation.
 10.09.2021, _Q Ren, L Cheng, F Guo, S Tao, C Zhang, L Ma, P Fu_


The intestinal flora serves a critical role in the development of hyperuricemia-induced chronic kidney disease (CKD). We previously found that natural flavonol fisetin exhibited nephroprotective effects in hyperuricemic mice. However, the mechanism remains largely unknown. To investigate the underlying mechanism of fisetin, mice were fed with potassium oxonate and adenine to introduce hyperuricemia-induced CKD. Fisetin improved kidney function, ameliorated renal fibrosis, and restored enteric dysbacteriosis in hyperuricemia-induced CKD mice. Meanwhile, gut microbiota-derived tryptophan metabolites, especially l-kynurenine, showed correlations with nephroprotective profiles of fisetin. Additionally, the kidney expression of the aryl hydrocarbon receptor (AHR), an endogenous receptor of l-kynurenine, was enhanced in hyperuricemic mice and further reduced in fisetin-treated mice. Finally, in vitro results showed that inhibition of AHR activation attenuated l-kynurenine-induced fibrosis. These results highlighted that fisetin protected against hyperuricemia-induced CKD via modulating gut microbiota-mediated tryptophan metabolism and AHR activation.

[Source](https://doi.org/10.1021/acs.jafc.1c03449)

10.1021/acs.jafc.1c03449

---

## Comparative effects of alive and pasteurized <em>Akkermansia muciniphila</em> on normal diet-fed mice

[Source](https://doi.org/10.1038/s41598-021-95738-5)

---

## Fruits and their impact on the gut microbiota, gut motility and constipation.
 10.09.2021, _Z Katsirma, E Dimidi, A Rodriguez-Mateos, K Whelan_


Fruits are the seed-bearing product of plants and have considerable nutritional importance in the human diet. The consumption of fruits is among the dietary strategies recommended for constipation due to its potential effects on the gut microbiota and gut motility. Dietary fiber from fruits has been the subject of research on the impact on gut microbiota, gut motility and constipation, however, fruits also contain other components that impact the intestinal luminal environment that may impact these outcomes including sorbitol and (poly)phenols. This review aims to explore the mechanisms of action and effectiveness of fruits and fruit products on the gut microbiota, gut motility and constipation, with a focus on fiber, sorbitol and (poly)phenols. In vitro, animal and human studies investigating the effects of fruits on gut motility and gut microbiota were sought through electronic database searches, hand searching and consulting with experts. Various fruits have been shown to modify the microbiota in human studies including blueberry powder (lactobacilli, bifidobacteria), prunes (bifidobacteria), kiwi fruit (Bacteroides, Faecalibacterium prausnitzii) and raisins (Ruminococcus, F. prausnitzii). Prunes, raisins and apple fiber isolate have been shown to increase fecal weight in humans, whilst kiwifruit to increase small bowel and fecal water content. Apple fiber isolate, kiwifruit, fig paste, and orange extract have been shown to reduce gut transit time, while prunes have not. There is limited evidence on which fruit components play a predominant role in regulating gut motility and constipation, or whether a synergy of multiple components is responsible for such effects.

[Source](https://doi.org/10.1039/d1fo01125a)

10.1039/d1fo01125a

---

## Rare sugars: Metabolic Impacts and Mechanisms of Action – a Scoping Review

[Source](https://doi.org/10.1017/S0007114521003524)

---

## Invasive Grass Dominance over Native Forbs Is Linked to Shifts in the Bacterial Rhizosphere Microbiome.
 10.09.2021, _ML LaForgia, H Kang, CL Ettinger_


Rhizosphere microbiomes have received growing attention in recent years for their role in plant health, stress tolerance, soil nutrition, and invasion. Still, relatively little is known about how these microbial communities are altered under plant competition, and even less about whether these shifts are tied to competitive outcomes between native and invasive plants. We investigated the structure and diversity of rhizosphere bacterial and fungal microbiomes of native annual forbs and invasive annual grasses grown in a shade-house both individually and in competition using high-throughput amplicon sequencing of the bacterial 16S rRNA gene and the fungal ITS region. We assessed how differentially abundant microbial families correlate to plant biomass under competition. We find that bacterial diversity and structure differ between native forbs and invasive grasses, but fungal diversity and structure do not. Furthermore, bacterial community structures under competition are distinct from individual bacterial community structures. We also identified five bacterial families that varied in normalized abundance between treatments and that were correlated with plant biomass under competition. We speculate that invasive grass dominance over these natives may be partially due to effects on the rhizosphere community, with changes in specific bacterial families potentially benefiting invaders at the expense of natives.

[Source](https://doi.org/10.1007/s00248-021-01853-1)

10.1007/s00248-021-01853-1

---

## Coupling the endophytic microbiome with the host transcriptome in olive roots.
 10.09.2021, _AJ Fernández-González, JA Ramírez-Tejero, MP Nevado-Berzosa, F Luque, M Fernández-López, J Mercado-Blanco_


The connection between olive genetic responses to environmental and agro-climatic conditions and the composition, structure and functioning of host-associated, belowground microbiota has never been studied under the holobiont conceptual framework. Two groups of cultivars growing under the same environmental, pedological and agronomic conditions, and showing highest (AH) and lowest (AL) Actinophytocola relative abundances, were earlier identified. We aimed now to: i) compare the root transcriptome profiles of these two groups harboring significantly different relative abundances in the above-mentioned bacterial genus; ii) examine their rhizosphere and root-endosphere microbiota co-occurrence networks; and iii) connect the root host transcriptome pattern to the composition of the root microbial communities by correlation and co-occurrence network analyses. Significant differences in olive gene expression were found between the two groups. Co-occurrence networks of the root endosphere microbiota were clearly different as well. Pearson's correlation analysis enabled a first portray of the interaction occurring between the root host transcriptome and the endophytic community. To further identify keystone operational taxonomic units (OTUs) and genes, subsequent co-occurrence network analysis showed significant interactions between 32 differentially expressed genes (DEGs) and 19 OTUs. Overall, negative correlation was detected between all upregulated genes in the AH group and all OTUs except of Actinophytocola. While two groups of olive cultivars grown under the same conditions showed significantly different microbial profiles, the most remarkable finding was to unveil a strong correlation between these profiles and the differential gene expression pattern of each group. In conclusion, this study shows a holistic view of the plant-microbiome communication.

[Source](https://doi.org/10.1016/j.csbj.2021.08.035)

10.1016/j.csbj.2021.08.035

---

## Succession and potential role of bacterial communities during Pleurotus ostreatus production.
 09.09.2021, _R Bánfi, Z Pohner, A Szabó, G Herczeg, GM Kovács, A Nagy, K Márialigeti, B Vajna_


There is an increasing interest in studying bacterial-fungal interactions (BFIs), also the interactions of Pleurotus ostreatus, a model white-rot fungus and important cultivated mushroom. In Europe, P. ostreatus is produced on a wheat straw-based substrate with a characteristic bacterial community, where P. ostreatus is exposed to the microbiome during substrate colonisation. This study investigated how the bacterial community structure was affected by the introduction of P. ostreatus into the mature substrate. Based on the results obtained, the effect of the presence and absence of this microbiome on P. ostreatus production in an experimental cultivation setup was determined. 16S rRNA gene-based terminal restriction fragment length polymorphism (T-RFLP) and amplicon sequencing revealed a definite succession of the microbiome during substrate colonisation and fruiting body production: a sharp decrease in relative abundance of Thermus spp. and Actinobacteria, and the increasing dominance of Bacillales and Halomonas spp. The introduced experimental cultivation setup proved the protective role of the microbial community against competing fungi without affecting P. ostreatus growth. We could also demonstrate that this effect could be attributed to both living microbes and their secreted metabolites. These findings highlight the importance of bacterial-fungal interactions during mushroom production.

[Source](https://doi.org/10.1093/femsec/fiab125)

10.1093/femsec/fiab125

---

## Metagenomic Analysis of Antibiotic Resistance Genes in Untreated Wastewater From Three Different Hospitals.
 10.09.2021, _X Guo, N Tang, H Lei, Q Fang, L Liu, Q Zhou, C Song_


Controlling antibiotic resistance genes (ARGs) is a worldwide intervention to ensure global health. Hospital wastewater is the main pollution source of antibiotic-resistant bacteria and ARGs in the environment. Expanding our knowledge on the bacterial composition of hospital wastewater could help us to control infections in hospitals and decrease pathogen release into the environment. In this study, a high-throughput sequencing-based metagenomic approach was applied to investigate the community composition of bacteria and ARGs in untreated wastewater from three different types of hospitals [the general hospital, traditional Chinese medicine (TCM) hospital, and stomatology hospital]. In total, 130 phyla and 2,554 genera were identified from the microbiota of the wastewaters, with significantly different bacterial community compositions among the three hospitals. Total ARG analysis using the Antibiotic Resistance Genes Database (ARDB) and Comprehensive Antibiotic Resistance Database (CARD) revealed that the microbiota in the wastewaters from the three hospitals harbored different types and percentage of ARGs, and their composition was specific to the hospital type based on the correlation analysis between species and ARG abundance, some ARGs contributed to different bacterial genera with various relationships in different hospitals. In summary, our findings demonstrated a widespread occurrence of ARGs and ARG-harboring microbiota in untreated wastewaters of different hospitals, suggesting that protection measures should be applied to prevent human infections. Concurrently, hospital wastewater should be treated more specifically for the removal of pathogens before its discharge into the urban sewage system.

[Source](https://doi.org/10.3389/fmicb.2021.709051)

10.3389/fmicb.2021.709051

---

## Learning Sparse Log-Ratios for High-Throughput Sequencing Data.
 09.09.2021, _E Gordon-Rodriguez, TP Quinn, JP Cunningham_


The automatic discovery of sparse biomarkers that are associated with an outcome of interest is a central goal of bioinformatics. In the context of high-throughput sequencing (HTS) data, and compositional data (CoDa) more generally, an important class of biomarkers are the log-ratios between the input variables. However, identifying predictive log-ratio biomarkers from HTS data is a combinatorial optimization problem, which is computationally challenging. Existing methods are slow to run and scale poorly with the dimension of the input, which has limited their application to low- and moderate-dimensional metagenomic datasets.

[Source](https://doi.org/10.1093/bioinformatics/btab645)

10.1093/bioinformatics/btab645

---

## gutMGene: a comprehensive database for target genes of gut microbes and microbial metabolites.
 09.09.2021, _L Cheng, C Qi, H Yang, M Lu, Y Cai, T Fu, J Ren, Q Jin, X Zhang_


gutMGene (http://bio-annotation.cn/gutmgene), a manually curated database, aims at providing a comprehensive resource of target genes of gut microbes and microbial metabolites in humans and mice. Metagenomic sequencing of fecal samples has identified 3.3 × 106 non-redundant microbial genes from up to 1500 different species. One of the contributions of gut microbiota to host biology is the circulating pool of bacterially derived small-molecule metabolites. It has been estimated that 10% of metabolites found in mammalian blood are derived from the gut microbiota, where they can produce systemic effects on the host through activating or inhibiting gene expression. The current version of gutMGene documents 1331 curated relationships between 332 gut microbes, 207 microbial metabolites and 223 genes in humans, and 2349 curated relationships between 209 gut microbes, 149 microbial metabolites and 544 genes in mice. Each entry in the gutMGene contains detailed information on a relationship between gut microbe, microbial metabolite and target gene, a brief description of the relationship, experiment technology and platform, literature reference and so on. gutMGene provides a user-friendly interface to browse and retrieve each entry using gut microbes, disorders and intervention measures. It also offers the option to download all the entries and submit new experimentally validated associations.

[Source](https://doi.org/10.1093/nar/gkab786)

10.1093/nar/gkab786

---

## Elucidating biofilm diversity on water lily leaves through 16S rRNA amplicon analysis: Comparison of four DNA extraction kits.
 10.09.2021, _K Janssen, SL Low, Y Wang, QY Mu, G Bierbaum, CT Gee_


Within a broader study on leaf fossilization in freshwater environments, a long-term study on the development and microbiome composition of biofilms on the foliage of aquatic plants has been initiated to understand how microbes and biofilms contribute to leaf decay and preservation. Here, water lily leaves are employed as a study model to investigate the relationship between bacterial microbiomes, biodegradation, and fossilization. We compare four DNA extraction kits to reduce biases in interpretation and to identify the most suitable kit for the extraction of DNA from bacteria associated with biofilms on decaying water lily leaves for 16S rRNA amplicon analysis.

[Source](https://doi.org/10.1002/aps3.11444)

10.1002/aps3.11444

---

## Metagenome-Assembled Genome Sequences Obtained from a Reactivated Kombucha Microbial Community Exposed to a Mars-Like Environment outside the International Space Station.
 10.09.2021, _I Lee, D Barh, O Podolich, B Brenig, S Tiwari, V Azevedo, A Góes-Neto, O Reva, N Kozyrovska, JP de Vera, BS Kim_


Kombucha is a traditional tea fermented by symbiotic microbiota, and it has been known as a functional fermented product. Here, we report four microbial metagenome-assembled genome sequences (MAGs) reconstructed from the microbiomes in kombucha exposed to a Mars-like environment outside the International Space Station.

[Source](https://doi.org/10.1128/MRA.00549-21)

10.1128/MRA.00549-21

---

## A year of genomic surveillance reveals how the SARS-CoV-2 pandemic unfolded in Africa

[Source](https://www.science.org/doi/10.1126/science.abj4336)

---

