## Longitudinal multi-omics analyses link gut microbiome dysbiosis with recurrent urinary tract infections in women

[Source](https://www.nature.com/articles/s41564-022-01107-x)

---

## Unfavourable intrauterine environment contributes to abnormal gut microbiome and metabolome in twins.
 07.04.2022, _J Yang, L Hou, J Wang, L Xiao, J Zhang, N Yin, S Yao, K Cheng, W Zhang, Z Shi, J Wang, H Jiang, N Huang, Y You, M Lin, R Shang, Y Wei, Y Zhao, F Zhao_


Fetal growth restriction (FGR) is a devastating pregnancy complication that increases the risk of perinatal mortality and morbidity. This study aims to determine the combined and relative effects of genetic and intrauterine environments on neonatal microbial communities and to explore selective FGR-induced gut microbiota disruption, metabolic profile disturbances and possible outcomes.

[Source](https://gut.bmj.com/content/early/2022/04/05/gutjnl-2021-326482)

10.1136/gutjnl-2021-326482

---

## Dosing a synbiotic of human milk oligosaccharides and B. infantis leads to reversible engraftment in healthy adult microbiomes without antibiotics

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(22)00207-4)

---

## Gut microbial similarity in twins is driven by shared environment and aging.
 01.05.2022, _R Vilchez-Vargas, J Skieceviciene, K Lehr, G Varkalaite, C Thon, M Urba, E Morkūnas, L Kucinskas, K Bauraite, D Schanze, M Zenker, P Malfertheiner, J Kupcinskas, A Link_


Human gut microbiome composition is influenced by genetics, diet and environmental factors. We investigated the microbial composition in several gastrointestinal (GI) compartments to evaluate the impact of genetics, delivery mode, diet, household sharing and aging on microbial similarity in monozygotic and dizygotic twins.

[Source](https://www.thelancet.com/journals/ebiom/article/PIIS2352-3964(22)00195-5/fulltext)

10.1016/j.ebiom.2022.104011

---

## Gut Microbiome Alterations Associated with Diabetes in Mexican Americans in South Texas.
 03.05.2022, _SY Kwan, CM Sabotta, A Joon, P Wei, LE Petty, JE Below, X Wu, J Zhang, RR Jenq, ET Hawk, JB McCormick, SP Fisher-Hoch, L Beretta_


Mexican Americans have a high prevalence of diabetes and burden of diabetes-related complications, highlighting the need for novel preventive strategies and noninvasive predictors of diabetes risk tailored to this population. Changes in the gut microbiome have the potential to predict diabetes. Here, we aimed to identify alterations in the gut microbiome associated with diabetes in the high-risk population of Mexican Americans in South Texas. Stool samples were collected from 216 subjects from the population-based Cameron County Hispanic Cohort. Among them, 75 had type 2 diabetes. Taxonomic and functional profiling of the stool samples were assessed by 16S and shotgun metagenomic sequencing, and the influence of genetic factors was explored. The gut microbiome of subjects with diabetes was enriched with proinflammatory Proteobacteria members (Enterobacteriaceae, Escherichia-Shigella) and depleted of butyrate-producing Clostridiales members (Faecalibacterium prausnitzii, Peptostreptococcaceae, and Clostridium sensu stricto 1). The accompanying metagenomic changes in subjects with diabetes suggested dysregulated amino acid metabolism, reduced galacturonate and glucuronate catabolism (correlating with Faecalibacterium prausnitzii abundance), and enriched heme biosynthesis (correlating with Enterobacteriaceae abundance). Polymorphism rs7129790 near MMP27 was strongly associated with high Proteobacteria abundance and was more frequent in this cohort and in individuals of Mexican ancestry than in Europeans. In conclusion, Mexican Americans in South Texas with diabetes display distinct gut microbiome and metagenomic signatures. These signatures may have utility in risk modeling and disease prevention in this high-risk population. IMPORTANCE The gut microbiome composition varies across ethnicities and geographical locations, yet studies on diabetes-associated microbiome changes specific to high-risk Mexican Americans are lacking. Here, we aimed to identify specific alterations associated with diabetes in this population, as well as host genetic factors that may explain increased disease susceptibility in this ethnic group. Using samples from a population-based cohort of Mexican Americans with a high prevalence of obesity and diabetes, we confirmed findings from studies on other ethnicities that suggested promotion of a chronic proinflammatory environment, loss of butyrate production, and compromised intestinal barrier integrity. High abundance of proinflammatory Proteobacteria was associated with a polymorphism that was more frequent in this cohort and in individuals of Mexican ancestry than in Europeans. Validation of microbiome-based risk models for diabetes should be evaluated in prospective cohort studies.

[Source](https://journals.asm.org/doi/10.1128/msystems.00033-22)

10.1128/msystems.00033-22

---

## Metagenomic sequencing reveals time, host, and body compartment-specific viral dynamics after lung transplantation.
 29.04.2022, _S Widder, I Görzer, B Friedel, N Rahimi, S Schwarz, P Jaksch, S Knapp, E Puchhammer-Stöckl_


_Humans, Lung, Lung Transplantation, Metagenome, Metagenomics, Transplant Recipients_
The virome of lung transplant recipients (LTRs) under immunosuppressive therapy is dominated by non-pathogenic Anelloviridae and further includes several pathogenic viruses such as Herpesviruses or respiratory viruses. It is unclear whether the donor-derived virome in the transplanted lung influences recipient virome dynamics in other body compartments and if so, to which degree. Likewise, it is unknown whether dependencies exist among virus populations that mutually shape viral loads and kinetics.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01244-9)

10.1186/s40168-022-01244-9

---

## Polymicrobial Interactions of Oral Microbiota: a Historical Review and Current Perspective.
 02.05.2022, _M Zhang, M Whiteley, GR Lewin_


The oral microbiota is enormously diverse, with over 700 microbial species identified across individuals that play a vital role in the health of our mouth and our overall well-being. In addition, as oral diseases such as caries (cavities) and periodontitis (gum disease) are mediated through interspecies microbial interactions, this community serves as an important model system to study the complexity and dynamics of polymicrobial interactions. Here, we review historical and recent progress in our understanding of the oral microbiome, highlighting how oral microbiome research has significantly contributed to our understanding of microbial communities, with broad implications in polymicrobial diseases and across microbial community ecology. Further, we explore innovations and challenges associated with analyzing polymicrobial systems and suggest future directions of study. Finally, we provide a conceptual framework to systematically study microbial interactions within complex communities, not limited to the oral microbiota.

[Source](https://journals.asm.org/doi/10.1128/mbio.00235-22)

10.1128/mbio.00235-22

---

## Effects of 'Healthy' Fecal Microbiota Transplantation against the Deterioration of Depression in Fawn-Hooded Rats.
 03.05.2022, _B Hu, P Das, X Lv, M Shi, J Aa, K Wang, L Duan, JA Gilbert, Y Nie, XL Wu_


Depression is a recurrent, heterogeneous mood disorder occurring in more than 260 million people worldwide. Gut microbiome dysbiosis is associated with the development of depressive-like behaviors by modulating neuro-biochemical metabolism through the microbiome-gut-brain (MGB) axis. Fecal microbiota transplantation (FMT) has been proposed as a potential therapeutic solution for depression, but the therapeutic efficiency and mechanism are unknown. Here, we performed an FMT from Sprague-Dawley (SD) rats ('healthy' controls) to Fawn-hooded (FH) rats (depression model). Pre-FMT, the FH rats exhibited significantly elevated depressive-like behaviors and distinct neurotransmitter and cytokine levels compared with SD rats. Post-FMT, FH recipients receiving FH fecal microbiota (FH-FH rats) showed aggravated depressive-like behaviors, while the ones receiving SD microbiota (FH-SD rats) had significantly alleviated depressive symptoms, a significant increase in hippocampal neurotransmitters, and a significant decrease of some hippocampal cytokines than FH-FH rats. SD-FMT resulted in the FH-SD rats' gut microbiome resembling the SD donors, and a significant shift in the serum metabolome but not the hippocampal metabolome. Co-occurrence analysis suggests that SD-FMT prevented recipients' depression development via the significant decrease of gut microbial species such as Dialister sp., which led to the recipients' metabolic modulation in serum and hippocampus through the enteric nervous system, the intestinal barrier, and the blood-brain barrier. Our results provided new data pointing to multiple mechanisms of interaction for the impact of gut microbiome modulation on depression therapy. IMPORTANCE Depression is a chronic, recurrent mental disease, which could make the patients commit suicide in severe cases. Considering that gut microbiome dysbiosis could cause depressive symptoms in animals through the MGB axis, the modification of gut microbiota is expected to be a potential therapy for depression, but the daily administration of probiotics is invalid or transient. In this study, we demonstrated that the gut microbiome transferred from a healthy rat model to a depressive rat model could regulate the recipient's neurobiology and behavior via the systematic alternation of the depressive gut microbiota followed by the serum and hippocampal metabolism. These results underline the significance of understanding the impact of gut microbiota on mental disorders and suggest that 'healthy' microbiota transplantation with the function to solve the host's cerebral inflammation may serve as a novel therapeutic strategy for depression.

[Source](https://journals.asm.org/doi/10.1128/msystems.00218-22)

10.1128/msystems.00218-22

---

## Metagenome-assembled genomes of phytoplankton microbiomes from the Arctic and Atlantic Oceans.
 03.05.2022, _A Duncan, K Barry, C Daum, E Eloe-Fadrosh, S Roux, K Schmidt, SG Tringe, KU Valentin, N Varghese, A Salamov, IV Grigoriev, RM Leggett, V Moulton, T Mock_


_Atlantic Ocean, Chlorophyll A, Eukaryota, Metagenome, Microbiota, Phylogeny, Phytoplankton_
Phytoplankton communities significantly contribute to global biogeochemical cycles of elements and underpin marine food webs. Although their uncultured genomic diversity has been estimated by planetary-scale metagenome sequencing and subsequent reconstruction of metagenome-assembled genomes (MAGs), this approach has yet to be applied for complex phytoplankton microbiomes from polar and non-polar oceans consisting of microbial eukaryotes and their associated prokaryotes.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01254-7)

10.1186/s40168-022-01254-7

---

## Compositionally Aware Phylogenetic Beta-Diversity Measures Better Resolve Microbiomes Associated with Phenotype.
 03.05.2022, _C Martino, D McDonald, K Cantrell, AH Dilmore, Y Vázquez-Baeza, L Shenhav, JP Shaffer, G Rahman, G Armstrong, C Allaband, SJ Song, R Knight_


Microbiome data have several specific characteristics (sparsity and compositionality) that introduce challenges in data analysis. The integration of prior information regarding the data structure, such as phylogenetic structure and repeated-measure study designs, into analysis, is an effective approach for revealing robust patterns in microbiome data. Past methods have addressed some but not all of these challenges and features: for example, robust principal-component analysis (RPCA) addresses sparsity and compositionality; compositional tensor factorization (CTF) addresses sparsity, compositionality, and repeated measure study designs; and UniFrac incorporates phylogenetic information. Here we introduce a strategy of incorporating phylogenetic information into RPCA and CTF. The resulting methods, phylo-RPCA, and phylo-CTF, provide substantial improvements over state-of-the-art methods in terms of discriminatory power of underlying clustering ranging from the mode of delivery to adult human lifestyle. We demonstrate quantitatively that the addition of phylogenetic information improves effect size and classification accuracy in both data-driven simulated data and real microbiome data. IMPORTANCE Microbiome data analysis can be difficult because of particular data features, some unavoidable and some due to technical limitations of DNA sequencing instruments. The first step in many analyses that ultimately reveals patterns of similarities and differences among sets of samples (e.g., separating samples from sick and healthy people or samples from seawater versus soil) is calculating the difference between each pair of samples. We introduce two new methods to calculate these differences that combine features of past methods, specifically being able to take into account the principles that most types of microbes are not in most samples (sparsity), that abundances are relative rather than absolute (compositionality), and that all microbes have a shared evolutionary history (phylogeny). We show using simulated and real data that our new methods provide improved classification accuracy of ordinal sample clusters and increased effect size between sample groups on beta-diversity distances.

[Source](https://journals.asm.org/doi/10.1128/msystems.00050-22)

10.1128/msystems.00050-22

---

## BiomeHorizon: Visualizing Microbiome Time Series Data in R

[Source](https://journals.asm.org/doi/10.1128/msystems.01380-21)

---

## Comparative Analysis and Data Provenance for 1,113 Bacterial Genome Assemblies.
 02.05.2022, _DA Yarmosh, JG Lopera, NP Puthuveetil, PF Combs, AL Reese, C Tabron, AE Pierola, J Duncan, SR Greenfield, R Marlow, S King, MA Riojas, J Bagnoli, B Benton, JL Jacobs_


The availability of public genomics data has become essential for modern life sciences research, yet the quality, traceability, and curation of these data have significant impacts on a broad range of microbial genomics research. While microbial genome databases such as NCBI's RefSeq database leverage the scalability of crowd sourcing for growth, genomics data provenance and authenticity of the source materials used to produce data are not strict requirements. Here, we describe the de novo assembly of 1,113 bacterial genome references produced from authenticated materials sourced from the American Type Culture Collection (ATCC), each with full genomics data provenance relating to bioinformatics methods, quality control, and passage history. Comparative genomics analysis of ATCC standard reference genomes (ASRGs) revealed significant issues with regard to NCBI's RefSeq bacterial genome assemblies related to completeness, mutations, structure, strain metadata, and gaps in traceability to the original biological source materials. Nearly half of RefSeq assemblies lack details on sample source information, sequencing technology, or bioinformatics methods. Deep curation of these records is not within the scope of NCBI's core mission in supporting open science, which aims to collect sequence records that are submitted by the public. Nonetheless, we propose that gaps in metadata accuracy and data provenance represent an "elephant in the room" for microbial genomics research. Effectively addressing these issues will require raising the level of accountability for data depositors and acknowledging the need for higher expectations of quality among the researchers whose research depends on accurate and attributable reference genome data. IMPORTANCE The traceability of microbial genomics data to authenticated physical biological materials is not a requirement for depositing these data into public genome databases. This creates significant risks for the reliability and data provenance of these important genomics research resources, the impact of which is not well understood. We sought to investigate this by carrying out a comparative genomics study of 1,113 ATCC standard reference genomes (ASRGs) produced by ATCC from authenticated and traceable materials using the latest sequencing technologies. We found widespread discrepancies in genome assembly quality, genetic variability, and the quality and completeness of the associated metadata among hundreds of reference genomes for ATCC strains found in NCBI's RefSeq database. We present a comparative analysis of de novo-assembled ASRGs, their respective metadata, and variant analysis using RefSeq genomes as a reference. Although assembly quality in RefSeq has generally improved over time, we found that significant quality issues remain, especially as related to genomic data and metadata provenance. Our work highlights the importance of data authentication and provenance for the microbial genomics community, and underscores the risks of ignoring this issue in the future.

[Source](https://journals.asm.org/doi/10.1128/msphere.00077-22)

10.1128/msphere.00077-22

---

## Ancestry-inclusive dog genomics challenges popular breed stereotypes.
 03.05.2022, _K Morrill, J Hekman, X Li, J McClure, B Logan, L Goodman, M Gao, Y Dong, M Alonso, E Carmichael, N Snyder-Mackler, J Alonso, HJ Noh, J Johnson, M Koltookian, C Lieu, K Megquier, R Swofford, J Turner-Maier, ME White, Z Weng, A Colubri, DP Genereux, KA Lord, EK Karlsson_


_Animals, Dogs, Genome-Wide Association Study, Genomics, Phenotype_
Behavioral genetics in dogs has focused on modern breeds, which are isolated subgroups with distinctive physical and, purportedly, behavioral characteristics. We interrogated breed stereotypes by surveying owners of 18,385 purebred and mixed-breed dogs and genotyping 2155 dogs. Most behavioral traits are heritable [heritability (h2) &gt; 25%], and admixture patterns in mixed-breed dogs reveal breed propensities. Breed explains just 9% of behavioral variation in individuals. Genome-wide association analyses identify 11 loci that are significantly associated with behavior, and characteristic breed behaviors exhibit genetic complexity. Behavioral loci are not unusually differentiated in breeds, but breed propensities align, albeit weakly, with ancestral function. We propose that behaviors perceived as characteristic of modern breeds derive from thousands of years of polygenic adaptation that predates breed formation, with modern breeds distinguished primarily by aesthetic traits.

[Source](https://www.science.org/doi/10.1126/science.abk0639)

10.1126/science.abk0639

---

## Dietary fibre in hypertension and cardiovascular disease management: systematic review and meta-analyses.
 25.04.2022, _AN Reynolds, A Akerman, S Kumar, HT Diep Pham, S Coffey, J Mann_


_Adult, Cardiovascular Diseases, Dietary Fiber, Humans, Hypertension, Observational Studies as Topic, Primary Prevention, Prospective Studies_
Higher dietary fibre intakes are associated with a reduced risk of developing cardiovascular disease (CVD), and increasing intake has been shown to reduce blood pressure and other cardiometabolic risk factors. The extent to which dietary fibre can further reduce risk for those with CVD and treated with cardioprotective drugs has not been clearly established. We have examined the evidence for dietary fibre as adjunct therapy in those with CVD or hypertension.

[Source](https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-022-02328-x)

10.1186/s12916-022-02328-x

---

