## Paternal microbiome perturbations impact offspring fitness.
 01.05.2024, _A Argaw-Denboba, TSB Schmidt, M Di Giacomo, B Ranjan, S Devendran, E Mastrorilli, CT Lloyd, D Pugliese, V Paribeni, J Dabin, A Pisaniello, S Espinola, A Crevenna, S Ghosh, N Humphreys, O Boruc, P Sarkies, M Zimmermann, P Bork, JA Hackett_


The gut microbiota operates at the interface of host-environment interactions to influence human homoeostasis and metabolic networks1-4. Environmental factors that unbalance gut microbial ecosystems can therefore shape physiological and disease-associated responses across somatic tissues5-9. However, the systemic impact of the gut microbiome on the germline-and consequently on the F1 offspring it gives rise to-is unexplored10. Here we show that the gut microbiota act as a key interface between paternal preconception environment and intergenerational health in mice. Perturbations to the gut microbiota of prospective fathers increase the probability of their offspring presenting with low birth weight, severe growth restriction and premature mortality. Transmission of disease risk occurs via the germline and is provoked by pervasive gut microbiome perturbations, including non-absorbable antibiotics or osmotic laxatives, but is rescued by restoring the paternal microbiota before conception. This effect is linked with a dynamic response to induced dysbiosis in the male reproductive system, including impaired leptin signalling, altered testicular metabolite profiles and remapped small RNA payloads in sperm. As a result, dysbiotic fathers trigger an elevated risk of in utero placental insufficiency, revealing a placental origin of mammalian intergenerational effects. Our study defines a regulatory 'gut-germline axis' in males, which is sensitive to environmental exposures and programmes offspring fitness through impacting placenta function.

[Source](https://doi.org/10.1038/s41586-024-07336-w)

10.1038/s41586-024-07336-w

---

## Oral bacteria relative abundance in faeces increases due to gut microbiota depletion and is linked with patient outcomes.
 02.05.2024, _C Liao, T Rolling, A Djukovic, T Fei, V Mishra, H Liu, C Lindberg, L Dai, B Zhai, JU Peled, MRM van den Brink, TM Hohl, JB Xavier_


The detection of oral bacteria in faecal samples has been associated with inflammation and intestinal diseases. The increased relative abundance of oral bacteria in faeces has two competing explanations: either oral bacteria invade the gut ecosystem and expand (the 'expansion' hypothesis), or oral bacteria transit through the gut and their relative increase marks the depletion of other gut bacteria (the 'marker' hypothesis). Here we collected oral and faecal samples from mouse models of gut dysbiosis (antibiotic treatment and DSS-induced colitis) and used 16S ribosomal RNA sequencing to determine the abundance dynamics of oral bacteria. We found that the relative, but not absolute, abundance of oral bacteria increases, reflecting the 'marker' hypothesis. Faecal microbiome datasets from diverse patient cohorts, including healthy individuals and patients with allogeneic haematopoietic cell transplantation or inflammatory bowel disease, consistently support the 'marker' hypothesis and explain associations between oral bacterial abundance and patient outcomes consistent with depleted gut microbiota. By distinguishing between the two hypotheses, our study guides the interpretation of microbiome compositional data and could potentially identify cases where therapies are needed to rebuild the resident microbiome rather than protect against invading oral bacteria.

[Source](https://doi.org/10.1038/s41564-024-01680-3)

10.1038/s41564-024-01680-3

---

## Human gut microbiota and their production of endocannabinoid-like mediators are directly affected by a dietary oil.
 02.05.2024, _C Roussel, M Sola, J Lessard-Lord, N Nallabelli, P Généreux, C Cavestri, O Azeggouar Wallen, R Villano, F Raymond, N Flamand, C Silvestri, V Di Marzo_


_Humans, Gastrointestinal Microbiome, Bacteria, Endocannabinoids, Colon, Ileum, Fatty Acids, Omega-3, Plant Oils, Dietary Supplements, Adult, Male_
Dietary omega-3 polyunsaturated fatty acids (n-3 PUFAs) and the gut microbiome affect each other. We investigated the impact of supplementation with Buglossoides arvensis oil (BO), rich in stearidonic acid (SDA), on the human gut microbiome. Employing the Mucosal Simulator of the Human Intestinal Microbial Ecosystem (M-SHIME), we simulated the ileal and ascending colon microbiomes of four donors. Our results reveal two distinct microbiota clusters influenced by BO, exhibiting shared and contrasting shifts. Notably, Bacteroides and Clostridia abundance underwent similar changes in both clusters, accompanied by increased propionate production in the colon. However, in the ileum, cluster 2 displayed a higher metabolic activity in terms of BO-induced propionate levels. Accordingly, a triad of bacterial members involved in propionate production through the succinate pathway, namely Bacteroides, Parabacteroides, and Phascolarctobacterium, was identified particularly in this cluster, which also showed a surge of second-generation probiotics, such as Akkermansia, in the colon. Finally, we describe for the first time the capability of gut bacteria to produce N-acyl-ethanolamines, and particularly the SDA-derived N-stearidonoyl-ethanolamine, following BO supplementation, which also stimulated the production of another bioactive endocannabinoid-like molecule, commendamide, in both cases with variations across individuals. Spearman correlations enabled the identification of bacterial genera potentially involved in endocannabinoid-like molecule production, such as, in agreement with previous reports, Bacteroides in the case of commendamide. This study suggests that the potential health benefits on the human microbiome of certain dietary oils may be amenable to stratified nutrition strategies and extend beyond n-3 PUFAs to include microbiota-derived endocannabinoid-like mediators.

[Source](https://doi.org/10.1080/19490976.2024.2335879)

10.1080/19490976.2024.2335879

---

## Leveraging genomic signatures of oral microbiome-associated antibiotic resistance genes for diagnosing pancreatic cancer.
 02.05.2024, _X Shen, X Zhu, H Liu, R Yuan, Q Guo, P Zhao_


_Humans, Pancreatic Neoplasms, Microbiota, Carcinoma, Pancreatic Ductal, Female, Male, Mouth, Middle Aged, Drug Resistance, Microbial, Aged, Genomics_
Growing evidence has increasingly suggested a potential linkage between the oral microbiome and various diseases, including pancreatic ductal adenocarcinoma (PDAC). However, the utilization of gene-level information derived from the oral microbiome for diagnosing PDAC remains unexplored. In this study, we sought to investigate the novel potential of leveraging genomic signatures associated with antibiotic resistance genes (ARGs) within the oral microbiome for the diagnosis of PDAC. By conducting an analysis of oral microbiome samples obtained from PDAC patients, we successfully identified specific ARGs that displayed distinct sequence abundance profiles correlated with the presence of PDAC. In the healthy group, three ARGs were found to be enriched, whereas 21 ARGs were enriched in PDAC patients. Remarkably, these ARGs from oral microbiome exhibited promising diagnostic capabilities for PDAC (AUROC = 0.79), providing a non-invasive and early detection method. Our findings not only provide novel modal data for diagnosing PDAC but also shed light on the intricate interplay between the oral microbiome and PDAC.

[Source](https://doi.org/10.1371/journal.pone.0302361)

10.1371/journal.pone.0302361

---

## Deciphering the Pancreatic Cancer Microbiome in Mainland China: Impact of <em>Exiguobacterium/Bacillus </em>ratio on Tumor Progression and Prognostic Significance

[Source](https://doi.org/10.1016/j.phrs.2024.107197)

---

## Emergent antibiotic persistence in a spatially structured synthetic microbial mutualism

[Source](https://doi.org/10.1093/ismejo/wrae075)

---

## Microplastics and microbiota: Unraveling the hidden environmental challenge

[Source](https://doi.org/10.3748/wjg.v30.i16.2191)

---

## Genome plasticity shapes the ecology and evolution of <em>Phocaeicola dorei </em>and <em>Phocaeicola vulgatus</em>

[Source](https://doi.org/10.1038/s41598-024-59148-7)

---

## A catalog of ethanol-producing microbes in humans

[Source](https://www.futuremedicine.com/doi/abs/10.2217/fmb-2023-0250?journalCode=fmb)

---

## Direct metagenomics investigation of non-surgical hard-to-heal wounds: a review

[Source](https://doi.org/10.1186/s12941-024-00698-z)

---

## A consortium of <em>Hordeum vulgare </em>and gut microbiota against non-alcoholic fatty liver disease via data-driven analysis

[Source](https://doi.org/10.1080/21691401.2024.2347380)

---

## Amphibian tolerance to arsenic: microbiome-mediated insights

[Source](https://doi.org/10.1038/s41598-024-60879-w)

---

## Combined effect of seasons and life history in an anuran strengthens the response and relationship between their physiology and gut microbiota

[Source](https://doi.org/10.1038/s41598-024-60105-7)

---

## Gut microbiome and resistome characterization of pigs treated with commonly used post-weaning diarrhea treatments

[Source](https://doi.org/10.1186/s42523-024-00307-6)

---

## Effects of microbiome transplants on fly microbiome

[Source](https://doi.org/10.1038/s41684-024-01369-4)

---

## Multi-omics reveal the gut microbiota-mediated severe foraging environment adaption of small wild ruminants in the Three-River-Source National Park, China

[Source](https://doi.org/10.1111/1749-4877.12830)

---

## Sea cucumber (<em>Holothuria glaberrima</em>) intestinal microbiome dataset from Puerto Rico, generated by shotgun sequencing

[Source](https://doi.org/10.1016/j.dib.2024.110421)

---

## Depletion of protective microbiota promotes the incidence of fruit disease

[Source](https://doi.org/10.1093/ismejo/wrae071)

---

## Analysis of Salt Stress on Soil Microbial Community Composition and Its Correlation with Active Components in the Rhizosphere of <em>Acanthopanax senticosus</em>

[Source](https://doi.org/10.1007/s00284-024-03687-9)

---

## Mechanisms and implications of bacterial-fungal competition for soil resources

[Source](https://doi.org/10.1093/ismejo/wrae073)

---

## Potential coupling of microbial methane, nitrogen, and sulphur cycling in the Okinawa Trough cold seep sediments

[Source](https://doi.org/10.1128/spectrum.03490-23)

---

## mbDecoda: a debiased approach to compositional data analysis for microbiome surveys

[Source](https://doi.org/10.1093/bib/bbae205)

---

## EnsembleSeq: a workflow towards real-time, rapid, and simultaneous multi-kingdom-amplicon sequencing for holistic and resource-effective microbiome research at scale.
 30.04.2024, _S Nagpal, SS Mande, H Hooda, U Dutta, B Taneja_


Bacterial communities are often concomitantly present with numerous microorganisms in the human body and other natural environments. Amplicon-based microbiome studies have generally paid skewed attention, that too at a rather shallow genus level resolution, to the highly abundant bacteriome, with interest now forking toward the other microorganisms, particularly fungi. Given the generally sparse abundance of other microbes in the total microbiome, simultaneous sequencing of amplicons targeting multiple microbial kingdoms could be possible even with full multiplexing. Guiding studies are currently needed for performing and monitoring multi-kingdom-amplicon sequencing and data capture at scale. Aiming to address these gaps, amplification of full-length bacterial 16S rRNA gene and entire fungal internal-transcribed spacer (ITS) region was performed for human saliva samples (n = 96, including negative and positive controls). Combined amplicon DNA libraries were prepared for nanopore sequencing using a major fraction of 16S molecules and a minor fraction of ITS amplicons. Sequencing was performed in a single run of an R10.4.1 flow cell employing the latest V14 chemistry. An approach for real-time monitoring of the species saturation using dynamic rarefaction was designed as a guiding determinant of optimal run time. Real-time saturation monitoring for both bacterial and fungal species enabled the completion of sequencing within 30 hours, utilizing less than 60% of the total nanopores. Approximately 5 million high quality (HQ) taxonomically assigned reads were generated (~4.2 million bacterial and 0.7 million fungal), providing a wider (beyond bacteriome) snapshot of human oral microbiota at species-level resolution. Among the more than 400 bacterial and 240 fungal species identified in the studied samples, the species of Streptococcus (e.g., Streptococcus mitis and Streptococcus oralis) and Candida (e.g., Candida albicans and Candida tropicalis) were observed to be the dominating microbes in the oral cavity, respectively. This conformed well with the previous reports of the human oral microbiota. EnsembleSeq provides a proof-of-concept toward the identification of both fungal and bacterial species simultaneously in a single fully multiplexed nanopore sequencing run in a time- and resource-effective manner. Details of this workflow, along with the associated codebase, are provided to enable large-scale application for a holistic species-level microbiome study.

[Source](https://doi.org/10.1128/spectrum.04150-23)

10.1128/spectrum.04150-23

---

## mi-Mic: a novel multi-layer statistical test for microbiota-disease associations.
 03.05.2024, _O Shtossel, S Finkelstein, Y Louzoun_


_Humans, Microbiota, Disease, Statistics as Topic_
mi-Mic, a novel approach for microbiome differential abundance analysis, tackles the key challenges of such statistical tests: a large number of tests, sparsity, varying abundance scales, and taxonomic relationships. mi-Mic first converts microbial counts to a cladogram of means. It then applies a priori tests on the upper levels of the cladogram to detect overall relationships. Finally, it performs a Mann-Whitney test on paths that are consistently significant along the cladogram or on the leaves. mi-Mic has much higher true to false positives ratios than existing tests, as measured by a new real-to-shuffle positive score.

[Source](https://doi.org/10.1186/s13059-024-03256-0)

10.1186/s13059-024-03256-0

---

## Unveiling microbial diversity: harnessing long-read sequencing technology.
 30.04.2024, _DP Agustinho, Y Fu, VK Menon, GA Metcalf, TJ Treangen, FJ Sedlazeck_


Long-read sequencing has recently transformed metagenomics, enhancing strain-level pathogen characterization, enabling accurate and complete metagenome-assembled genomes, and improving microbiome taxonomic classification and profiling. These advancements are not only due to improvements in sequencing accuracy, but also happening across rapidly changing analysis methods. In this Review, we explore long-read sequencing's profound impact on metagenomics, focusing on computational pipelines for genome assembly, taxonomic characterization and variant detection, to summarize recent advancements in the field and provide an overview of available analytical methods to fully leverage long reads. We provide insights into the advantages and disadvantages of long reads over short reads and their evolution from the early days of long-read sequencing to their recent impact on metagenomics and clinical diagnostics. We further point out remaining challenges for the field such as the integration of methylation signals in sub-strain analysis and the lack of benchmarks.

[Source](https://doi.org/10.1038/s41592-024-02262-1)

10.1038/s41592-024-02262-1

---

