## Chemotaxis under flow disorder shapes microbial dispersion in porous media

[Source](https://www.nature.com/articles/s41567-020-1002-x)

---

## A piece of the pie: engineering microbiomes by exploiting division of labor in complex polysaccharide consumption

[Source](https://www.sciencedirect.com/science/article/abs/pii/S2211339820300502)

---

## Benchmarking microbial growth rate predictions from metagenomes

[Source](https://www.nature.com/articles/s41396-020-00773-1)

---

## Integration of absolute multi-omics reveals dynamic protein-to-RNA ratios and metabolic interplay within mixed-domain microbiomes

[Source](https://www.nature.com/articles/s41467-020-18543-0)

---

## Spatial Variance of Species Distribution Predicts the Interspecies Interactions within a Microbial Metacommunity

[Source](https://link.springer.com/article/10.1007/s00248-020-01603-9)

---

## The Social Distancing Imposed To Contain COVID-19 Can Affect Our Microbiome: a Double-Edged Sword in Human Health 

[Source](https://msphere.asm.org/content/5/5/e00716-20)

---

## Auto-reactivity against gut bacterial peptides in patients with late-onset diabetes

[Source](https://www.tandfonline.com/doi/abs/10.1080/08916934.2020.1818232)

---

## The gut microbiome in pediatric patients undergoing allogeneic hematopoietic stem cell transplantation

[Source](https://onlinelibrary.wiley.com/doi/abs/10.1002/pbc.28711)

---

## Gut Microbiota and Metabolome Alterations Associated with Parkinson’s Disease

[Source](https://msystems.asm.org/content/5/5/e00561-20)

---

## Gut Microbiota Resilience: Definition, Link to Health and Strategies for Intervention

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.572921/full)

---

## Deciphering the impact of diet and host physiology on the mammalian gut microbiome by multi-omics approaches

[Source](https://aem.asm.org/content/early/2020/09/14/AEM.01864-20)

---

## High altitude as a possible factor for dysbiosis of salivary microbiome in orthodontic patients

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0003996920302958)

---

## Changes in Gut Microorganism in Patients with Positive Immune Antibody-Associated Recurrent Abortion

[Source](https://www.hindawi.com/journals/bmri/2020/4673250/)

---

## Contrasting microbiota profiles observed in children carrying either Blastocystis spp. or the commensal amoebas Entamoeba coli or Endolimax nana

[Source](https://www.nature.com/articles/s41598-020-72286-y)

---

## Biological observations in microbiota analysis are robust to the choice of 16S rRNA gene sequencing processing algorithm: case study on human milk microbiota.
 21.09.2020, _S Moossavi, F Atakora, K Fehr, E Khafipour_


In recent years, the microbiome field has undergone a shift from clustering-based methods of operational taxonomic unit (OTU) designation based on sequence similarity to denoising algorithms that identify exact amplicon sequence variants (ASVs), and methods to identify contaminating bacterial DNA sequences from low biomass samples have been developed. Although these methods improve accuracy when analyzing mock communities, their impact on real samples and downstream analysis of biological associations is less clear.

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-020-01949-7)

10.1186/s12866-020-01949-7

---

## Metagenome analysis using serum extracellular vesicles identified distinct microbiota in asthmatics.
 16.09.2020, _JH Lee, JP Choi, J Yang, HK Won, CS Park, WJ Song, HS Kwon, TB Kim, YK Kim, HS Park, YS Cho_


Different patterns of bacterial communities have been reported in the airways and gastrointestinal tract of asthmatics when compared to healthy controls. However, the blood microbiome of asthmatics is yet to be investigated. Therefore, we aimed to determine whether a distinct serum microbiome is observed in asthmatics by metagenomic analysis of serum extracellular vesicles (EVs). We obtained serum from 190 adults with asthma and 260 healthy controls, from which EVs were isolated and analyzed. The bacterial composition of asthmatics was significantly different from that of healthy controls. Chao 1 index was significantly higher in the asthma group, while Shannon and Simpson indices were higher in the control group. At the phylum level, Bacteroidetes was more abundant in asthmatics, while Actinobacter, Verrucomicrobia, and Cyanobacteria were more abundant in healthy controls. At the genus level, 24 bacterial genera showed differences in relative abundance between asthmatics and controls, with linear discriminant analysis scores greater than 3. Further, in a diagnostic model based on these differences, a high predictive value with a sensitivity of 0.92 and a specificity of 0.93 was observed. In conclusion, we demonstrated distinct blood microbiome in asthma indicating the role of microbiome as a potential diagnostic marker of asthma.

[Source](https://www.nature.com/articles/s41598-020-72242-w)

10.1038/s41598-020-72242-w

---

## Reconsidering ventilator-associated pneumonia from a new dimension of the lung microbiome.
 23.09.2020, _L Fernández-Barat, R López-Aladid, A Torres_


Complex microbial communities that reside in the lungs, skin and gut are now appreciated for their role in maintaining organ, tissue and immune homoeostasis. As lungs are currently seen as an ecosystem, the shift in paradigm calls for the consideration of new algorithms related to lung ecology in pulmonology. Evidence of lung microbiota does not solely challenge the traditional physiopathology of ventilator-associated pneumonia (VAP); indeed, it also reinforces the need to include molecular techniques in VAP diagnosis and accelerate the use of immunomodulatory drugs, including corticosteroids, and other supplements such as probiotics for VAP prevention and/or treatment. With that stated, both microbiome and virome, including phageome, can lead to new opportunities in further understanding the relationship between health and dysbiosis in VAP. Previous knowledge may be, however, reconsidered at a microbiome scale.

[Source](https://www.sciencedirect.com/science/article/pii/S2352396420303716)

10.1016/j.ebiom.2020.102995

---

## The impact of anthelmintic treatment on gut bacterial and fungal communities in diagnosed parasite-free sika deer Cervus nippon.
 15.09.2020, _X Hu, Y Xu, G Liu, D Hu, Y Wang, W Zhang, Y Zheng_


The gut microbiota, including both bacterial and fungal communities, plays vital roles in the gut homeostasis of animals, and antibiotics can lead to disorders of these microbial communities. The use of anthelmintic treatment to control parasitic infection has long been a standard practice, although its impact on the gut microbiota of healthy sika deer is relatively unknown. This study used next-generation sequencing based on 16S/18S/ITS rRNA genes to investigate the shifts in fecal bacterial and fungal communities in parasite-free sika deer after treatment with fenbendazole and ivermectin tablets. The α-diversity of both bacterial and fungal communities was significantly decreased (P &lt; 0.05) after treatment, as were the bacterial genus Bacteroides and fungal genus Candida (P &lt; 0.05). The results of β-diversity, LEfSe analysis, core community's analysis, taxonomic composition, and functional prediction of fungal and bacterial communities confirmed the substantial impacts of anthelmintic treatment on the function and structure of the intestinal microbiota of sika deer. Nevertheless, many lines of evidence, including β-diversity, LEfSe analysis and functional prediction analysis, suggested that the anthelmintics exerted more significant influences on fungal communities than on bacterial communities, suggesting that more attention should be paid to the changes in fungal communities of sika deer under anthelmintic treatment. The present study provides evidence to support the assumption that anthelmintic drugs modify the gut microbiota of deer and serves as the first trial to test the potential effects of anthelmintics on mycobiota in ruminants using high-throughput sequencing techniques. Key Points • Anthelmintic treatment showed significant effects on the gut microbiota of sika deer. • Fungi were more strongly affected by anthelmintic treatment than bacteria. • The profile of mycobiota provides essential data that were previously absent.

[Source](https://link.springer.com/article/10.1007/s00253-020-10838-y)

10.1007/s00253-020-10838-y

---

## Host phylogeny and life history stage shape the gut microbiome in dwarf (Kogia sima) and pygmy (Kogia breviceps) sperm whales.
 17.09.2020, _ER Denison, RG Rhodes, WA McLellan, DA Pabst, PM Erwin_


Gut microbiomes perform crucial roles in host health and development, but few studies have explored cetacean microbiomes especially deep divers. We characterized the gut microbiomes of stranded dwarf (Kogia sima) and pygmy (K. breviceps) sperm whales to examine the effects of phylogeny and life stage on microbiome composition and diversity. 16S rRNA gene sequence analysis revealed diverse gut communities (averaging 674 OTUs) dominated by a few symbiont taxa (25 OTUs accounted for 64% of total relative abundance). Both phylogeny and life stage shaped community composition and diversity, with species-specific microbiome differences present early in life. Further analysis showed evidence of microbiome convergence with host maturity, albeit through different processes: symbiont 'accumulation' in K. sima and 'winnowing' in K. breviceps, indicating different methods of community assembly during host development. Furthermore, culture-based analyses yielded 116 pure cultures matching 25 OTUs, including one isolate positive for chitin utilization. Our findings indicate that kogiid gut microbiomes are highly diverse and species-specific, undergo significant shifts with host development, and can be cultivated on specialized media under anaerobic conditions. These results enhance our understanding of the kogiid gut microbiome and may provide useful information for symbiont assessment in host health.

[Source](https://www.nature.com/articles/s41598-020-72032-4)

10.1038/s41598-020-72032-4

---

## Altered gut microbiome accompanying with placenta barrier dysfunction programs pregnant complications in mice caused by graphene oxide.
 23.09.2020, _X Liu, F Zhang, Z Wang, T Zhang, C Teng, Z Wang_


The wide use of graphene oxide (GO) has raised increasing concerns about the potential risks to environmental and human health. Recent studies have shown the vital role of gut microbiome in various pathological status or even exogenous exposure, but more detailed understanding about the effects of possible gut microbiome alterations under GO exposure on reproductive toxicology evaluations in pregnant mammals remained elusive. Here we found that orally administrated GO daily during gestational day (GD) 7-16 caused dose-dependent pregnant complications of mice on the endpoint (GD19), including decreased weight of dam and live fetus, high rate of resorbed embryos and dead fetus, and skeletal development retardation. Meanwhile in placenta tissues of pregnant mice exposed to GO at dose over 10 mg/kg, the expression levels of tight junctions (Claudin1 and Occludin) and vascular endothelial growth factor (VEGFA) decreased approximately by 30%-80%, meaning impaired placenta barrier. According to the data of fecal 16s RNA sequencing in 40 mg/kg dose group and the control group, gut microbiome showed dramatically decreased α- and β-diversity, and upregulated Firmicutes/Bacteroidetes ratio owing to GO exposure. What's more, significantly differentiated abundance of Euryarchaeota is expected to be a special biomarker for failed pregnancy caused by GO. Notably, the result of Spearman correlation analysis suggested that there was a strong link (correlation coefficient&gt;0.6) between perturbed gut microbiome with both abnormally expressed factors of placenta barrier and adverse pregnant outcomes. In summary, the damages of GO exposure to placenta barrier and pregnancy were dose-dependent. And GO exposure was responsible for gut microbiome dysbiosis in mice with pregnant complications. These findings could provide referable evidence to evaluate reproductive risk of GO to mammals.

[Source](https://www.sciencedirect.com/science/article/pii/S0147651320309829)

10.1016/j.ecoenv.2020.111143

---

## Comparative analysis of the pulmonary microbiome in healthy and diseased pigs.
 18.09.2020, _Z Li, X Wang, D Di, R Pan, Y Gao, C Xiao, B Li, J Wei, K Liu, Y Qiu, Z Ma_


The lungs possess an effective antimicrobial system and a strong ability to eliminate microorganisms in healthy organisms, and were once considered sterile. With the development of culture-independent sequencing technology, the richness and diversity of porcine lung microbiota have been gaining attention. In order to study the relationship between lung microbiota and porcine respiratory disease complex (PRDC), the lung microbiota in healthy and diseased swine bronchoalveolar lavage fluids were analyzed and compared using the Illumina MiSeq sequencing platform. The predominant microbial communities of healthy and diseased swine were similar at the phylum level, mainly composed of Proteobacteria, Firmicutes, Tenericutes, and Bacteroidetes. However, the bacterial taxonomic communities of healthy and diseased swine differed at the genus level. The higher relative abundances of Lactococcus, Enterococcus, Staphylococcus, and Lactobacillus genera in healthy swine might provide more benefits for lung health, while the enhanced richness of Streptococcus, Haemophilus, Pasteurella, and Bordetella genera in diseased swine might be closely related to pathogen invasion and the occurrence of respiratory disease. In conclusion, the observed differences in the richness and diversity of lung microbiota can provide novel insights into their relationship with PRDC. Analyses of swine lung microbiota communities might produce an effective strategy for the control and prevention of respiratory tract infections.

[Source](https://link.springer.com/article/10.1007/s00438-020-01722-5)

10.1007/s00438-020-01722-5

---

## Diet posttranslationally modifies the mouse gut microbial proteome to modulate renal function.
 18.09.2020, _L Lobel, YG Cao, K Fenn, JN Glickman, WS Garrett_


Associations between chronic kidney disease (CKD) and the gut microbiota have been postulated, yet questions remain about the underlying mechanisms. In humans, dietary protein increases gut bacterial production of hydrogen sulfide (H2S), indole, and indoxyl sulfate. The latter are uremic toxins, and H2S has diverse physiological functions, some of which are mediated by posttranslational modification. In a mouse model of CKD, we found that a high sulfur amino acid-containing diet resulted in posttranslationally modified microbial tryptophanase activity. This reduced uremic toxin-producing activity and ameliorated progression to CKD in the mice. Thus, diet can tune microbiota function to support healthy host physiology through posttranslational modification without altering microbial community composition.

[Source](https://science.sciencemag.org/content/369/6510/1518.abstract)

10.1126/science.abb3763

---

## Overhauling the assessment of agrochemical-driven interferences with microbial communities for improved global ecosystem integrity

[Source](https://www.sciencedirect.com/science/article/pii/S2666498420300533)

---

## Susceptibility and resilience of the soil ciliate community to high temperatures

[Source](https://www.tandfonline.com/doi/abs/10.1080/00380768.2020.1819148?journalCode=tssp20)

---

## Novel approaches for the identification of microbial communities in kimchi: MALDI-TOF MS analysis and high-throughput sequencing

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0740002020302306)

---

## The Nutrient Removal Ability and Microbial Communities in a Pilot-Scale Horizontal Subsurface Flow Constructed Wetland Fed by Slightly Polluted Lake Water

[Source](https://link.springer.com/article/10.1007/s13157-020-01327-z)

---

## Impact of temporal pH fluctuations on the coexistence of nasal bacteria

[Source](https://www.biorxiv.org/content/10.1101/2020.09.15.298778v1)

---

## Study of growth, metabolism, and morphology of Akkermansia muciniphila with an in vitro advanced bionic intestinal reactor

[Source](https://www.biorxiv.org/content/10.1101/2020.07.06.190843v2.abstract?%3Fcollection=)

---

