## Genetic regulation of serum IgA levels and susceptibility to common immune, infectious, kidney, and cardio-metabolic traits.
 15.11.2022, _L Liu, A Khan, E Sanchez-Rodriguez, F Zanoni, Y Li, N Steers, O Balderes, J Zhang, P Krithivasan, RA LeDesma, C Fischman, SJ Hebbring, JB Harley, H Moncrieffe, LC Kottyan, B Namjou-Khales, TL Walunas, R Knevel, S Raychaudhuri, EW Karlson, JC Denny, IB Stanaway, D Crosslin, T Rauen, J Floege, F Eitner, Z Moldoveanu, C Reily, B Knoppova, S Hall, JT Sheff, BA Julian, RJ Wyatt, H Suzuki, J Xie, N Chen, X Zhou, H Zhang, L Hammarström, A Viktorin, PKE Magnusson, N Shang, G Hripcsak, C Weng, T Rundek, MSV Elkind, EC Oelsner, RG Barr, I Ionita-Laza, J Novak, AG Gharavi, K Kiryluk_


_Humans, Mice, Animals, Glomerulonephritis, IGA, Genome-Wide Association Study, Celiac Disease, Genetic Predisposition to Disease, Diabetes Mellitus, Type 2, Immunoglobulin A, Inflammatory Bowel Diseases, Kidney_
Immunoglobulin A (IgA) mediates mucosal responses to food antigens and the intestinal microbiome and is involved in susceptibility to mucosal pathogens, celiac disease, inflammatory bowel disease, and IgA nephropathy. We performed a genome-wide association study of serum IgA levels in 41,263 individuals of diverse ancestries and identified 20 genome-wide significant loci, including 9 known and 11 novel loci. Co-localization analyses with expression QTLs prioritized candidate genes for 14 of 20 significant loci. Most loci encoded genes that produced immune defects and IgA abnormalities when genetically manipulated in mice. We also observed positive genetic correlations of serum IgA levels with IgA nephropathy, type 2 diabetes, and body mass index, and negative correlations with celiac disease, inflammatory bowel disease, and several infections. Mendelian randomization supported elevated serum IgA as a causal factor in IgA nephropathy. African ancestry was consistently associated with higher serum IgA levels and greater frequency of IgA-increasing alleles compared to other ancestries. Our findings provide novel insights into the genetic regulation of IgA levels and its potential role in human disease.

[Source](https://doi.org/10.1038/s41467-022-34456-6)

10.1038/s41467-022-34456-6

---

## Kidney microbiota dysbiosis contributes to the development of hypertension

[Source](https://doi.org/10.1080/19490976.2022.2143220)

---

## Gut mycobiota dysbiosis in drug-naïve, first-episode schizophrenia.
 15.11.2022, _X Yuan, X Li, Y Kang, L Pang, G Hei, X Zhang, S Wang, X Zhao, S Zhang, Q Tao, L Zhang, Q Zhu, L Lv, X Fan, S Hu, X Song_


Bacterial dysbiosis has been demonstrated in patients with schizophrenia (SCH). The aim of the present study was to investigate alterations in mycobiota composition and fungi-bacteria correlation network in drug-naïve, first episode SCH. We recruited 205 SCH patients and 125 healthy controls (HCs), whose gut bacterial and fungal compositions were characterized by 16S and 18S ribosomal RNA gene amplicon sequencing, respectively. Fungal-bacterial relative correlation network analysis was performed using the Spearman's test and distance correlation. We also computed relative networks connectedness, which represents the ratio of significant interactions (edges) and taxa (nodes) in the network. SCH patients showed lower fungal α-diversity compared with that of HCs. Furthermore, we identified 29 differential fungal markers at multiple taxonomies between SCH patients and HCs. SCH patients also showed a significantly lower fungi-to-bacteria α-diversity ratio compared with that of HCs (p = 1.81 × 10-8). In risk prediction models, we observed that combining bacterial and fungal markers achieved higher accuracy than that of bacterial markers alone (AUC = 0.847 vs AUC = 0.739; p = 0.043). Fungal-bacterial correlation network was denser in HCs than in SCH patients and was characterized by a high number of neighbors (p &lt; 0.05). In addition, an increased abundance of Purpureocillium was associated with more severe psychiatric symptoms and poorer cognitive function in SCH patients (p &lt; 0.05). Our study demonstrated a disrupted and weakened fungi-bacteria network in SCH patients, which might be associated with their clinical manifestations. Future research on fungal-bacterial correlation network is warranted to advance our understanding about the role of mycobiota in the etiology of SCH and to explore novel intervention approaches.

[Source](https://doi.org/10.1016/j.schres.2022.10.011)

10.1016/j.schres.2022.10.011

---

## Transcriptomic analysis reveals an association of FCGBP with Parkinson’s disease

[Source](https://doi.org/10.1038/s41531-022-00415-7)

---

## Integration of stool microbiota, proteome and amino acid profiles to discriminate patients with adenomas and colorectal cancer.
 15.11.2022, _S Bosch, A Acharjee, MN Quraishi, IV Bijnsdorp, P Rojas, A Bakkali, EE Jansen, P Stokkers, J Kuijvenhoven, TV Pham, AD Beggs, CR Jimenez, EA Struys, GV Gkoutos, TG de Meij, NK de Boer_


_Humans, Proteome, Colorectal Neoplasms, Chromatography, Liquid, RNA, Ribosomal, 16S, Amino Acids, Gastrointestinal Microbiome, Tandem Mass Spectrometry, Adenoma, Feces_
Screening for colorectal cancer (CRC) reduces its mortality but has limited sensitivity and specificity. Aims We aimed to explore potential biomarker panels for CRC and adenoma detection and to gain insight into the interaction between gut microbiota and human metabolism in the presence of these lesions.

[Source](https://doi.org/10.1080/19490976.2022.2139979)

10.1080/19490976.2022.2139979

---

## Human gut homeostasis and regeneration: the role of the gut microbiota and its metabolites.
 12.11.2022, _CM Arenas-Gómez, E Garcia-Gutierrez, JS Escobar, PD Cotter_


The healthy human gut is a balanced ecosystem where host cells and representatives of the gut microbiota interact and communicate in a bidirectional manner at the gut epithelium. As a result of these interactions, many local and systemic processes necessary for host functionality, and ultimately health, take place. Impairment of the integrity of the gut epithelium diminishes its ability to act as an effective gut barrier, can contribute to conditions associated to inflammation processes and can have other negative consequences. Pathogens and pathobionts have been linked with damage of the integrity of the gut epithelium, but other components of the gut microbiota and some of their metabolites can contribute to its repair and regeneration. Here, we review what is known about the effect of bacterial metabolites on the gut epithelium and, more specifically, on the regulation of repair by intestinal stem cells and the regulation of the immune system in the gut. Additionally, we explore the potential therapeutic use of targeted modulation of the gut microbiota to maintain and improve gut homeostasis as a mean to improve health outcomes.

[Source](https://doi.org/10.1080/1040841X.2022.2142088)

10.1080/1040841X.2022.2142088

---

## Colibactin possessing <em>E. coli</em> isolates in association with colorectal cancer and their genetic diversity among Pakistani population

[Source](https://doi.org/10.1371/journal.pone.0262662)

---

## Intestinal toxicity to CTLA-4 blockade driven by IL-6 and myeloid infiltration.
 15.11.2022, _Y Zhou, YB Medik, B Patel, DB Zamler, S Chen, T Chapman, S Schneider, EM Park, RL Babcock, TT Chrisikos, LM Kahn, AM Dyevoich, JE Pineda, MC Wong, AK Mishra, SH Cass, AP Cogdill, DH Johnson, SB Johnson, K Wani, DA Ledesma, CW Hudgens, J Wang, MA Wadud Khan, CB Peterson, AY Joon, W Peng, HS Li, R Arora, X Tang, MG Raso, X Zhang, WC Foo, MT Tetzlaff, GE Diehl, K Clise-Dwyer, EM Whitley, MM Gubin, JP Allison, P Hwu, NJ Ajami, A Diab, JA Wargo, SS Watowich_


_Mice, Animals, Interleukin-6, Quality of Life, Colitis, Immunotherapy, Inflammation_
Immune checkpoint blockade (ICB) has revolutionized cancer treatment, yet quality of life and continuation of therapy can be constrained by immune-related adverse events (irAEs). Limited understanding of irAE mechanisms hampers development of approaches to mitigate their damage. To address this, we examined whether mice gained sensitivity to anti-CTLA-4 (αCTLA-4)-mediated toxicity upon disruption of gut homeostatic immunity. We found αCTLA-4 drove increased inflammation and colonic tissue damage in mice with genetic predisposition to intestinal inflammation, acute gastrointestinal infection, transplantation with a dysbiotic fecal microbiome, or dextran sodium sulfate administration. We identified an immune signature of αCTLA-4-mediated irAEs, including colonic neutrophil accumulation and systemic interleukin-6 (IL-6) release. IL-6 blockade combined with antibiotic treatment reduced intestinal damage and improved αCTLA-4 therapeutic efficacy in inflammation-prone mice. Intestinal immune signatures were validated in biopsies from patients with ICB colitis. Our work provides new preclinical models of αCTLA-4 intestinal irAEs, mechanistic insights into irAE development, and potential approaches to enhance ICB efficacy while mitigating irAEs.

[Source](https://doi.org/10.1084/jem.20221333)

10.1084/jem.20221333

---

## Genome-microbiome interplay provides insight into the determinants of the human blood metabolome.
 10.11.2022, _C Diener, CL Dai, T Wilmanski, P Baloni, B Smith, N Rappaport, L Hood, AT Magis, SM Gibbons_


Variation in the blood metabolome is intimately related to human health. However, few details are known about the interplay between genetics and the microbiome in explaining this variation on a metabolite-by-metabolite level. Here, we perform analyses of variance for each of 930 blood metabolites robustly detected across a cohort of 1,569 individuals with paired genomic and microbiome data while controlling for a number of relevant covariates. We find that 595 (64%) of these blood metabolites are significantly associated with either host genetics or the gut microbiome, with 69% of these associations driven solely by the microbiome, 15% driven solely by genetics and 16% under hybrid genome-microbiome control. Additionally, interaction effects, where a metabolite-microbe association is specific to a particular genetic background, are quite common, albeit with modest effect sizes. This knowledge will help to guide targeted interventions designed to alter the composition of the human blood metabolome.

[Source](https://doi.org/10.1038/s42255-022-00670-1)

10.1038/s42255-022-00670-1

---

## Vaginal microbiota in women with spontaneous preterm labor versus those with term labor in Kenya: a case control study.
 14.11.2022, _E Gulavi, F Mwendwa, DO Atandi, PO Okiro, M Hall, RG Beiko, RD Adam_


_Humans, Female, Infant, Newborn, Pregnancy, Infant, RNA, Ribosomal, 16S, Premature Birth, Case-Control Studies, Kenya, Placenta, Vagina, Obstetric Labor, Premature, Microbiota_
Preterm birth is a global problem with about 12% of births in sub-Saharan Africa occurring before 37 weeks of gestation. Several studies have explored a potential association between vaginal microbiota and preterm birth, and some have found an association while others have not. We performed a study designed to determine whether there is an association with vaginal microbiota and/or placental microbiota and preterm birth in an African setting.

[Source](https://doi.org/10.1186/s12866-022-02681-0)

10.1186/s12866-022-02681-0

---

## Characteristics of oral microbiome of healthcare workers in different clinical scenarios: a cross-sectional analysis.
 14.11.2022, _Z Zhang, W Yu, G Li, Y He, Z Shi, J Wu, X Ma, Y Zhu, L Zhao, S Liu, Y Wei, J Xue, S Guo, Z Gao_


_Humans, RNA, Ribosomal, 16S, Cross-Sectional Studies, Bacteria, Prospective Studies, Microbiota, Health Personnel_
The environment of healthcare institutes (HCIs) potentially affects the internal microecology of medical workers, which is reflected not only in the well-studied gut microbiome but also in the more susceptible oral microbiome. We conducted a prospective cross-sectional cohort study in four hospital departments in Central China. Oropharyngeal swabs from 65 healthcare workers were collected and analyzed using 16S rRNA gene amplicon sequencing. The oral microbiome of healthcare workers exhibited prominent deviations in diversity, microbial structure, and predicted function. The coronary care unit (CCU) samples exhibited robust features and stability, with significantly higher abundances of genera such as Haemophilus, Fusobacterium, and Streptococcus, and a lower abundance of Prevotella. Functional prediction analysis showed that vitamin, nucleotide, and amino acid metabolisms were significantly different among the four departments. The CCU group was at a potential risk of developing periodontal disease owing to the increased abundance of F. nucleatum. Additionally, oral microbial diversification of healthcare workers was related to seniority. We described the oral microbiome profile of healthcare workers in different clinical scenarios and demonstrated that community diversity, structure, and potential functions differed markedly among departments. Intense modulation of the oral microbiome of healthcare workers occurs because of their original departments, especially in the CCU.

[Source](https://doi.org/10.1186/s12903-022-02501-x)

10.1186/s12903-022-02501-x

---

## Multi-kingdom gut microbiota analyses define COVID-19 severity and post-acute COVID-19 syndrome.
 14.11.2022, _Q Liu, Q Su, F Zhang, HM Tun, JWY Mak, GC Lui, SSS Ng, JYL Ching, A Li, W Lu, C Liu, CP Cheung, DSC Hui, PKS Chan, FKL Chan, SC Ng_


_Humans, Gastrointestinal Microbiome, COVID-19, Metagenomics, Feces_
Our knowledge of the role of the gut microbiome in acute coronavirus disease 2019 (COVID-19) and post-acute COVID-19 is rapidly increasing, whereas little is known regarding the contribution of multi-kingdom microbiota and host-microbial interactions to COVID-19 severity and consequences. Herein, we perform an integrated analysis using 296 fecal metagenomes, 79 fecal metabolomics, viral load in 1378 respiratory tract samples, and clinical features of 133 COVID-19 patients prospectively followed for up to 6 months. Metagenomic-based clustering identifies two robust ecological clusters (hereafter referred to as Clusters 1 and 2), of which Cluster 1 is significantly associated with severe COVID-19 and the development of post-acute COVID-19 syndrome. Significant differences between clusters could be explained by both multi-kingdom ecological drivers (bacteria, fungi, and viruses) and host factors with a good predictive value and an area under the curve (AUC) of 0.98. A model combining host and microbial factors could predict the duration of respiratory viral shedding with 82.1% accuracy (error ± 3 days). These results highlight the potential utility of host phenotype and multi-kingdom microbiota profiling as a prognostic tool for patients with COVID-19.

[Source](https://doi.org/10.1038/s41467-022-34535-8)

10.1038/s41467-022-34535-8

---

## Characterization of antibiotic-resistance traits in <em>Akkermansia muciniphila</em> strains of human origin

[Source](https://doi.org/10.1038/s41598-022-23980-6)

---

## Causality Verification for the Correlation between the Presence of Nonstarter Bacteria and Flavor Characteristics in Soft-Type Ripened Cheeses.
 10.11.2022, _R Unno, T Suzuki, Y Osaki, M Matsutani, M Ishikawa_


Flavor characteristics of ripened cheese are established by various bacteria, such as lactic acid bacteria, Actinobacteria, and Proteobacteria, which spontaneously develop during the cheese-manufacturing process. We previously revealed the relationship between bacterial microbiota and flavor components in soft-type ripened cheeses by using a multiomics approach that combined metagenomics and metabolomics; however, we could not establish a causal relationship. This study aimed to substantiate the causal nature of the correlations revealed by the multiomics approach by using cheese-ripening tests with single isolate inoculation. The bacterial diversity and composition in surface mold-ripened cheeses from Japan and France varied, depending on the differences between the milks (pasteurized or raw), cheese positions (core or rind), and manufacturers. Although the volatile compounds did not clearly reflect the distinctive characteristics of the cheese samples, nonstarter lactic acid bacteria, Actinobacteria, and Proteobacteria positively correlated with ketones and sulfur compounds, as evidenced by a Spearman's correlation analysis. Cheese-ripening tests conducted after inoculation with single bacterial strains belonging to the above-mentioned taxa confirmed that these bacteria formed volatile compounds, in agreement with the correlations observed. In particular, various flavor compounds, such as acids, esters, ketones, and sulfur compounds, were detected in cheese inoculated with Pseudoalteromonas sp. TS-4-4 strain. These findings provide important insights into the role of nonstarter bacteria in the development of cheese flavor and into the effectiveness of the multiomics approach in screening for bacteria that can improve the quality of cheese products. IMPORTANCE Our previous study revealed that the existence of various bacteria, such as lactic acid bacteria, Actinobacteria, and Proteobacteria, clearly correlated with the abundance of flavor components, such as volatile compounds, in soft-type ripened cheeses via a multiomics approach that used 16S rRNA gene amplicon sequencing and headspace gas chromatography-mass spectrometry. However, this approach only showed correlations derived from statistical analyses rather than causal relationships. Therefore, in the present study, we performed cheese-ripening tests using nonstarter bacteria to substantiate the correlations revealed by the multiomics approach in soft-type ripened cheese. Our results suggest the capability of nonstarter bacteria, such as Proteobacteria, to impart flavor to cheese and the effectiveness of the multiomics approach in screening for microbial isolates that can improve the quality of cheese. Overall, our research provides new insights into the importance of bacteria in cheese production.

[Source](https://doi.org/10.1128/spectrum.02894-22)

10.1128/spectrum.02894-22

---

## Microbiota of healthy and bleached corals of the species Siderastrea stellata in response to river influx and seasonality in Brazilian northeast.
 12.11.2022, _GVB Paulino, CR Félix, FA da Silva Oliveira, C Gomez-Silvan, VMM Melo, GL Andersen, MF Landell_


Although coral bleaching is increasing worldwide due to warming oceans exacerbated by climate change, there has been a growing recognition that local stressors may play an additional role. Important stressors include the physicochemical and microbiological influences that are related to river runoff. Here, we investigated the microbiota associated to mucus and tissue of endemic coral Siderastrea stellata, collected from Brazilian northeast coral reefs of Barra de Santo Antônio (subject to river runoff) and Maragogi (minimal river runoff) during both the rainy and dry seasons. We sequenced the V4 region of 16S rDNA and used multiple R packages to process raw data and performed statistical analysis to reveal the microbial community structure composition and functional predictions. Major dissimilarities between microbial communities were related to seasonality, while healthy and bleached specimens were mainly associated with the enrichment of several less abundant taxa involved in specific metabolic functions, mainly related to the nitrogen cycle. We were not able to observe the dominance of groups that has been previously associated with bleachings, such as Vibrionaceae or Burkholderiaceae. The influx of freshwater appears to increase the homogeneity between individuals in Barra de Santo Antonio, especially during the rainy season. By contrast, we observed an increased homogeneity between samples in Maragogi during the dry season. Understanding the dynamics of the coral microbiota and how bleaching appears in response to specific environmental variables, in addition to determining the conditions that lead to a more robust coral microbiota, is essential for choosing the most appropriate area and conservation methods, for example.

[Source](https://doi.org/10.1007/s11356-022-23976-9)

10.1007/s11356-022-23976-9

---

## Microbiota of healthy and bleached corals of the species <em>Siderastrea stellata</em> in response to river influx and seasonality in Brazilian northeast

[Source](https://doi.org/10.1007/s11356-022-23976-9)

---

## Exploring structure, microbiota, and metagenome functions of epigean and hypogean black deposits by microscopic, molecular and bioinformatic approaches.
 15.11.2022, _B Farda, I Vaccarelli, C Ercole, R Djebaili, M Del Gallo, M Pellegrini_


_Metagenome, RNA, Ribosomal, 16S, Computational Biology, Microbiota, Bacteria, Phylogeny, Archaea_
This study revealed how Bacteria and Archaea communities and their metabolic functions differed between two groups of black deposits identified in gorge and cave environments. Scanning electron microscopy coupled with energy dispersive spectroscopy was used to analyse the presence of microbial biosignatures and the elemental composition of samples. Metabarcoding of the V3-V4 regions of 16S rRNA was used to investigate Bacteria and Archaea communities. Based on 16S rRNA sequencing results, PICRUSt software was used to predict metagenome functions. Micrographs showed that samples presented microbial biosignatures and microanalyses highlighted Mn concretions and layers on Al-Si surfaces. The 16S rRNA metabarcoding alpha-diversity metrics showed similar Simpson's and Shannon indices and different values of the Chao-1 index. The amplicon sequence variants (ASVs) analysis at the different taxonomic levels showed a diverse genera composition. However, the communities of all samples shared the presence of uncultured ASVs belonging to the Gemmatales family (Phylogenesis: Gemmataceae; Planctomycetes; Planctomycetota; Bacteria). The predicted metagenome functions analysis revealed diverse metabolic profiles of the Cave and Gorge groups. Genes coding for essential Mn metabolism were present in all samples. Overall, the findings on structure, microbiota, and predicted metagenome functions showed a similar microbial contribution to epigean and hypogean black deposits Mn metabolism.

[Source](https://doi.org/10.1038/s41598-022-24159-9)

10.1038/s41598-022-24159-9

---

## High throughput method of 16S rRNA gene sequencing library preparation for plant root microbial community profiling.
 15.11.2022, _K Kumaishi, E Usui, K Suzuki, S Kobori, T Sato, Y Toda, H Takanashi, S Shinozaki, M Noda, A Takakura, K Matsumoto, Y Yamasaki, H Tsujimoto, H Iwata, Y Ichihashi_


_RNA, Ribosomal, 16S, DNA, Bacterial, Genes, rRNA, Microbiota, High-Throughput Nucleotide Sequencing, Soil, DNA, Plant Roots, Exonucleases_
Microbiota are a major component of agroecosystems. Root microbiota, which inhabit the inside and surface of plant roots, play a significant role in plant growth and health. As next-generation sequencing technology allows the capture of microbial profiles without culturing the microbes, profiling of plant microbiota has become a staple tool in plant science and agriculture. Here, we have increased sample handling efficiency in a two-step PCR amplification protocol for 16S rRNA gene sequencing of plant root microbiota, improving DNA extraction using AMPure XP magnetic beads and PCR purification using exonuclease. These modifications reduce sample handling and capture microbial diversity comparable to that obtained by the manual method. We found a buffer with AMPure XP magnetic beads enabled efficient extraction of microbial DNA directly from plant roots. We also demonstrated that purification using exonuclease before the second PCR step enabled the capture of higher degrees of microbial diversity, thus allowing for the detection of minor bacteria compared with the purification using magnetic beads in this step. In addition, our method generated comparable microbiome profile data in plant roots and soils to that of using common commercially available DNA extraction kits, such as DNeasy PowerSoil Pro Kit and FastDNA SPIN Kit for Soil. Our method offers a simple and high-throughput option for maintaining the quality of plant root microbial community profiling.

[Source](https://doi.org/10.1038/s41598-022-23943-x)

10.1038/s41598-022-23943-x

---

## Sugarcane cultivation practices modulate rhizosphere microbial community composition and structure.
 14.11.2022, _APC Moneda, LAL de Carvalho, LG Teheran-Sierra, MIG Funnicelli, DG Pinheiro_


_Rhizosphere, Saccharum, RNA, Ribosomal, 16S, Soil Microbiology, Plant Roots, Microbiota, Edible Grain_
Sugarcane (Saccharum spp.) represents a crop of great economic importance, remarkably relevant in the food industry and energy supply chains from renewable sources. However, its conventional cultivation involves the intensive use of fertilizers, pesticides, and other agrochemical agents whose detrimental effects on the environment are notorious. Alternative systems, such as organic farming, have been presented as an environmentally friendly way of production. Still, the outcomes of different cropping systems on the microbiota associated with sugarcane-whose role in its health and growth is crucial-remain underexplored. Thus, we studied the rhizospheric microbiota of two adjacent sugarcane fields, which differ in terms of the type of farming system. For this, we used the sequencing of taxonomic markers of prokaryotes (gene 16S rRNA, subregions V3-V4) and fungi (Internal transcribed spacer 2) and evaluated the changes caused by the systems. Our results show a well-conserved microbiota composition among farming systems in the highest taxonomic ranks, such as phylum, class, and order. Also, both systems showed very similar alpha diversity indices and shared core taxa with growth-promoting capacities, such as bacteria from the Bacillus and Bradyrhizobium genera and the fungal genus Trichoderma. However, the composition at more specific levels denotes differences, such as the separation of the samples concerning beta diversity and the identification of 74 differentially abundant taxa between the systems. Of these, 60 were fungal taxa, indicating that this microbiota quota is more susceptible to changes caused by farming systems. The analysis of co-occurrence networks also showed the formation of peripheral sub-networks associated with the treatments-especially in fungi-and the presence of keystone taxa in terms of their ability to mediate relationships between other members of microbial communities. Considering that both crop fields used the same cultivar and had almost identical soil properties, we conclude that the observed findings are effects of the activities intrinsic to each system and can contribute to a better understanding of the effects of farming practices on the plant microbiome.

[Source](https://doi.org/10.1038/s41598-022-23562-6)

10.1038/s41598-022-23562-6

---

## Effects of Two <em>Bacillus velezensis</em> Microbial Inoculants on the Growth and Rhizosphere Soil Environment of <em>Prunus davidiana</em>

[Source](https://doi.org/10.3390/ijms232113639)

---

## Climate change and plant pathogens.
 15.11.2022, _MM Raza, DP Bebber_


Global food security is threatened by climate change, both directly through responses of crop physiology and productivity, and indirectly through responses of plant-associated microbiota, including plant pathogens. While the interactions between host plants, pathogens and environmental drivers can be complex, recent research is beginning to indicate certain overall patterns in how plant diseases will affect crop production in future. Here, we review the results of three methodological approaches: large-scale observational studies, process-based disease models and experimental comparisons of pathosystems under current and future conditions. We find that observational studies have tended to identify rising temperatures as the primary driver of disease impact. Process-based models suggest that rising temperatures will lead to latitudinal shifts in disease pressure, but drying conditions could mitigate disease risk. Experimental studies suggest that rising atmospheric CO2 will exacerbate disease impacts. Plant diseases may therefore counteract any crop yield increases due to climate change.

[Source](https://doi.org/10.1016/j.mib.2022.102233)

10.1016/j.mib.2022.102233

---

## Faecal microbiome-based machine learning for multi-class disease diagnosis.
 14.11.2022, _Q Su, Q Liu, RI Lau, J Zhang, Z Xu, YK Yeoh, TWH Leung, W Tang, L Zhang, JQY Liang, YK Yau, J Zheng, C Liu, M Zhang, CP Cheung, JYL Ching, HM Tun, J Yu, FKL Chan, SC Ng_


_Humans, COVID-19, Feces, Machine Learning, Microbiota_
Systemic characterisation of the human faecal microbiome provides the opportunity to develop non-invasive approaches in the diagnosis of a major human disease. However, shared microbial signatures across different diseases make accurate diagnosis challenging in single-disease models. Herein, we present a machine-learning multi-class model using faecal metagenomic dataset of 2,320 individuals with nine well-characterised phenotypes, including colorectal cancer, colorectal adenomas, Crohn's disease, ulcerative colitis, irritable bowel syndrome, obesity, cardiovascular disease, post-acute COVID-19 syndrome and healthy individuals. Our processed data covers 325 microbial species derived from 14.3 terabytes of sequence. The trained model achieves an area under the receiver operating characteristic curve (AUROC) of 0.90 to 0.99 (Interquartile range, IQR, 0.91-0.94) in predicting different diseases in the independent test set, with a sensitivity of 0.81 to 0.95 (IQR, 0.87-0.93) at a specificity of 0.76 to 0.98 (IQR 0.83-0.95). Metagenomic analysis from public datasets of 1,597 samples across different populations observes comparable predictions with AUROC of 0.69 to 0.91 (IQR 0.79-0.87). Correlation of the top 50 microbial species with disease phenotypes identifies 363 significant associations (FDR &lt; 0.05). This microbiome-based multi-disease model has potential clinical application in disease diagnostics and treatment response monitoring and warrants further exploration.

[Source](https://doi.org/10.1038/s41467-022-34405-3)

10.1038/s41467-022-34405-3

---

## Wochenende&nbsp;–&nbsp;modular and flexible alignment-based shotgun metagenome analysis

[Source](https://doi.org/10.1186/s12864-022-08985-9)

---

## De novo identification of microbial contaminants in low microbial biomass microbiomes with Squeegee.
 14.11.2022, _Y Liu, RAL Elworth, MD Jochum, KM Aagaard, TJ Treangen_


_Humans, Biomass, Microbiota, Metagenomics, Metagenome, Specimen Handling_
Computational analysis of host-associated microbiomes has opened the door to numerous discoveries relevant to human health and disease. However, contaminant sequences in metagenomic samples can potentially impact the interpretation of findings reported in microbiome studies, especially in low-biomass environments. Contamination from DNA extraction kits or sampling lab environments leaves taxonomic "bread crumbs" across multiple distinct sample types. Here we describe Squeegee, a de novo contamination detection tool that is based upon this principle, allowing the detection of microbial contaminants when negative controls are unavailable. On the low-biomass samples, we compare Squeegee predictions to experimental negative control data and show that Squeegee accurately recovers putative contaminants. We analyze samples of varying biomass from the Human Microbiome Project and identify likely, previously unreported kit contamination. Collectively, our results highlight that Squeegee can identify microbial contaminants with high precision and thus represents a computational approach for contaminant detection when negative controls are unavailable.

[Source](https://doi.org/10.1038/s41467-022-34409-z)

10.1038/s41467-022-34409-z

---

