## A multi-omic investigation of male lower urinary tract symptoms: Potential role for JC virus.
 05.03.2021, _S Thomas, CD Dunn, LJ Campbell, DW Strand, CM Vezina, DE Bjorling, KL Penniston, L Li, WA Ricke, TL Goldberg_


Male lower urinary tract symptoms (LUTS) comprise a common syndrome of aging that negatively impacts quality of life. The etiology of LUTS is multifactorial, involving benign prostatic hyperplasia, smooth muscle and neurologic dysfunction, inflammation, sexually transmitted infections, fibrosis, and potentially dysbiosis, but this aspect remains poorly explored. We investigated whether the presence of infectious agents in urine might be associated with LUTS by combining next-generation DNA sequencing for virus discovery, microbiome analysis for characterization of bacterial communities, and mass spectrometry-based metabolomics. In urine from 29 LUTS cases and 9 controls from Wisconsin, we found a statistically significant association between a diagnosis of LUTS and the presence of JC virus (JCV), a common neurotropic human polyomavirus (Polyomaviridae, Betapolyomavirus) linked to severe neurologic disease in rare cases. This association (based on metagenomics) was not borne out when specific polymerase chain reaction (PCR) testing was applied to this set of samples, likely due to the greater sensitivity of PCR. Interestingly, urine metabolomics analysis identified dysregulation of metabolites associated with key LUTS processes. Microbiome analysis found no evidence of microbial community dysbiosis in LUTS cases, but JCV-positive samples contained more Anaerococcus species, which are involved in polymicrobial infections of the urinary tract. Neither age nor body mass index were significantly associated with the presence of urinary JCV-in the initial group or in an additional, regionally distinct group. These data provide preliminary support the hypothesis that viruses such as JCV may play a role in the development or progression of LUTS, together with other infectious agents and host metabolic responses.

[Source](https://doi.org/10.1371/journal.pone.0246266)

10.1371/journal.pone.0246266

---

## Association between physical activity and changes in intestinal microbiota composition: A systematic review.
 05.03.2021, _V Aya, A Flórez, L Perez, JD Ramírez_


The intestinal microbiota comprises bacteria, fungi, archaea, protists, helminths and viruses that symbiotically inhabit the digestive system. To date, research has provided limited data on the possible association between an active lifestyle and a healthy composition of human microbiota. This review was aimed to summarize the results of human studies comparing the microbiome of healthy individuals with different physical activity amounts.

[Source](https://doi.org/10.1371/journal.pone.0247039)

10.1371/journal.pone.0247039

---

## Specialized microbiomes facilitate natural rhizosphere microbiome interactions counteracting high salinity stress in plants

[Source](https://doi.org/10.1016/j.envexpbot.2021.104430)

---

## Impact of sampling depth on pathogen detection in pit latrines.
 02.03.2021, _D Capone, P Chigwechokha, FL de Los Reyes, RH Holm, BB Risk, E Tilley, J Brown_


Wastewater based epidemiology (WBE) is increasingly used to provide decision makers with actionable data about community health. WBE efforts to date have primarily focused on sewer-transported wastewater in high-income countries, but at least 1.8 billion people in low- and middle-income countries (LMIC) use onsite sanitation systems such as pit latrines and septic tanks. Like wastewater, fecal sludges from such systems offer similar advantages in community pathogen monitoring and other epidemiological applications. To evaluate the distribution of enteric pathogens inside pit latrines-which could inform sampling methods for WBE in LMIC settings unserved by sewers-we collected fecal sludges from the surface, mid-point, and maximum-depth of 33 pit latrines in urban and peri-urban Malawi and analyzed the 99 samples for 20 common enteric pathogens via multiplex quantitative reverse transcription PCR. Using logistic regression adjusted for household population, latrine sharing, the presence of a concrete floor or slab, water source, and anal cleansing materials, we found no significant difference in the odds of detecting the 20 pathogens from the mid-point (adjusted odds ratio, aOR = 1.1; 95% confidence interval = 0.73, 1.6) and surface samples (aOR = 0.80, 95% CI = 0.54, 1.2) compared with those samples taken from the maximum depth. Our results suggest that, for the purposes of routine pathogen monitoring, pit latrine sampling depth does not strongly influence the odds of detecting enteric pathogens by molecular methods. A single sample from the pit latrines' surface, or a composite of surface samples, may be preferred as the most recent material contributed to the pit and may be easiest to collect.

[Source](https://doi.org/10.1371/journal.pntd.0009176)

10.1371/journal.pntd.0009176

---

## A novel methodology to study antimicrobial properties of high-touch surfaces used for indoor hygiene applications—A study on Cu metal

[Source](https://doi.org/10.1371/journal.pone.0247081)

---

## Functional capacities of microbial communities to carry out large scale geochemical processes are maintained during ex situ anaerobic incubation

[Source](https://doi.org/10.1371/journal.pone.0245857)

---

## Wastewater treatment works change the intestinal microbiomes of insectivorous bats.
 03.03.2021, _C Mehl, MC Schoeman, TJ Sanko, C Bezuidenhout, CMS Mienie, W Preiser, D Vosloo_


Mammals, born with a near-sterile intestinal tract, are inoculated with their mothers' microbiome during birth. Thereafter, extrinsic and intrinsic factors shape their intestinal microbe assemblage. Wastewater treatment works (WWTW), sites synonymous with pollutants and pathogens, receive influent from domestic, agricultural and industrial sources. The high nutrient content of wastewater supports abundant populations of chironomid midges (Diptera), which transfer these toxicants and potential pathogens to their predators, such as the banana bat Neoromicia nana (Vespertilionidae), thereby influencing their intestinal microbial assemblages. We used next generation sequencing and 16S rRNA gene profiling to identify and compare intestinal bacteria of N. nana at two reference sites and two WWTW sites. We describe the shared intestinal microbiome of the insectivorous bat, N. nana, consisting of seven phyla and eleven classes. Further, multivariate analyses revealed that location was the most significant driver (sex, body size and condition were not significant) of intestinal microbiome diversity. Bats at WWTW sites exhibited greater intestinal microbiota diversity than those at reference sites, likely due to wastewater exposure, stress and/or altered diet. Changes in their intestinal microbiota assemblages may allow these bats to cope with concomitant stressors.

[Source](https://doi.org/10.1371/journal.pone.0247475)

10.1371/journal.pone.0247475

---

## Microbial communities developing within bulk sediments under fish carcasses on a tidal flat.
 05.03.2021, _Y Kawamoto, H Kato, Y Nagata, J Urabe_


Animal carcasses are often brought into tidal flats where they are at the boundary between terrestrial and marine ecosystems. Since these carcasses act as microhabitats with large amounts of energy and nutrients, they likely develop unique bacterial assemblages in the ambient sediment, which in turn may stimulate colonization of other organisms such as protozoans. However, little is known about the microbial assemblages colonized in sediment around animal carcasses in the tidal zone. Herein we examined the bacterial and ciliophoran assemblages developed in association with fish carcasses by incubating the carcasses in the Higashiyachi tidal flat (Sendai, Japan). We collected sediment samples at 2, 9, and 42 days of incubation and analyzed the bacterial and ciliophoran assemblages by 16S and 18S rRNA gene amplicon sequencing. We observed significant differences in the composition and relative abundance of bacterial and ciliophoran operational taxonomic units (OTUs) between the sediments with and without the carcasses. Our analyses suggest that these unique assemblages were created through the direct effects of the carcass and indirect effects through interactions between bacteria and ciliophorans. These results also suggest that animal carcasses developed a temporally unique microbial food web in the sediments close to the carcasses, although it disappeared for several weeks.

[Source](https://doi.org/10.1371/journal.pone.0247220)

10.1371/journal.pone.0247220

---

## Host-microbe cross-talk governs amino acid chirality to regulate survival and differentiation of B cells.
 04.03.2021, _M Suzuki, T Sujino, S Chiba, Y Harada, M Goto, R Takahashi, M Mita, K Hamase, T Kanai, M Ito, MK Waldor, M Yasui, J Sasabe_


Organisms use l-amino acids (l-aa) for most physiological processes. Unlike other organisms, bacteria chiral-convert l-aa to d-configurations as essential components of their cell walls and as signaling molecules in their ecosystems. Mammals recognize microbe-associated molecules to initiate immune responses, but roles of bacterial d-amino acids (d-aa) in mammalian immune systems remain largely unknown. Here, we report that amino acid chirality balanced by bacteria-mammal cross-talk modulates intestinal B cell fate and immunoglobulin A (IgA) production. Bacterial d-aa stimulate M1 macrophages and promote survival of intestinal naïve B cells. Mammalian intestinal d-aa catabolism limits the number of B cells and restricts growth of symbiotic bacteria that activate T cell-dependent IgA class switching of the B cells. Loss of d-aa catabolism results in excessive IgA production and dysbiosis with altered IgA coating on bacteria. Thus, chiral conversion of amino acids is linked to bacterial recognition by mammals to control symbiosis with bacteria.

[Source](https://advances.sciencemag.org/content/7/10/eabd6480)

10.1126/sciadv.abd6480

---

## An integrated, modular approach to data science education in microbiology.
 05.03.2021, _KA Dill-McFarland, SG König, F Mazel, DC Oliver, LM McEwen, KY Hong, SJ Hallam_


We live in an increasingly data-driven world, where high-throughput sequencing and mass spectrometry platforms are transforming biology into an information science. This has shifted major challenges in biological research from data generation and processing to interpretation and knowledge translation. However, postsecondary training in bioinformatics, or more generally data science for life scientists, lags behind current demand. In particular, development of accessible, undergraduate data science curricula has the potential to improve research and learning outcomes as well as better prepare students in the life sciences to thrive in public and private sector careers. Here, we describe the Experiential Data science for Undergraduate Cross-Disciplinary Education (EDUCE) initiative, which aims to progressively build data science competency across several years of integrated practice. Through EDUCE, students complete data science modules integrated into required and elective courses augmented with coordinated cocurricular activities. The EDUCE initiative draws on a community of practice consisting of teaching assistants (TAs), postdocs, instructors, and research faculty from multiple disciplines to overcome several reported barriers to data science for life scientists, including instructor capacity, student prior knowledge, and relevance to discipline-specific problems. Preliminary survey results indicate that even a single module improves student self-reported interest and/or experience in bioinformatics and computer science. Thus, EDUCE provides a flexible and extensible active learning framework for integration of data science curriculum into undergraduate courses and programs across the life sciences.

[Source](https://doi.org/10.1371/journal.pcbi.1008661)

10.1371/journal.pcbi.1008661

---

## Five key aspects of metaproteomics as a tool to understand functional interactions in host-associated microbiomes.
 05.03.2021, _F Salvato, RL Hettich, M Kleiner_



[Source](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1009245)

10.1371/journal.ppat.1009245

---

