## Altered vaginal microbiome and relative co-abundance network in pregnant women with penicillin allergy

[Source](https://doi.org/10.1186/s13223-020-00475-5)

---

## Mechanisms underpinning the efficacy of faecal microbiota transplantation in treating gastrointestinal disease

[Source](https://doi.org/10.1177/1756284820946904)

---

## Deep-sea mussels from a hybrid zone on the Mid-Atlantic Ridge host genetically indistinguishable symbionts

[Source](https://doi.org/10.1101/2020.09.11.282681)

---

## Metagenomic analysis of viral communities in Baikal sponges

[Source](https://doi.org/10.31951/2658-3518-2020-A-4-777)

---

## Microbial community associated with Baicalian crustacea near gas-oil deepwater discharges

[Source](https://doi.org/10.31951/2658-3518-2020-A-4-991)

---

## Investigating the role of root exudates in recruiting Streptomyces bacteria to the <em>Arabidopsis thaliana</em> root microbiome

[Source](https://doi.org/10.1101/2020.09.09.290742)

---

## <span style="color:#ff6600;">Gene exchange networks define species-like units in marine prokaryotes</span>

[Source](https://doi.org/10.1101/2020.09.10.291518)

---

## The effects of seasonal anoxia on the microbial community structure in demosponges in a marine lake (Lough Hyne, Ireland)

[Source](https://doi.org/10.1101/2020.09.09.290791)

---

## Diversity and dynamics of fungi during spontaneous fermentations and association with unique aroma profiles in wine

[Source](https://doi.org/10.1101/2020.09.10.290536)

---

## Depicting the RNA Virome of Hematophagous Arthropods from Belgrade, Serbia

[Source](https://doi.org/10.3390/v12090975)

---

##  An Unconventional Flavivirus and other RNA Viruses in the Sea Cucumber (Holothuroidea; Echinodermata) Virome

[Source](https://www.preprints.org/manuscript/202009.0061/v1)

---

## Challenges in gene-oriented approaches for pangenome content discovery.
 07.09.2020, _V Bonnici, E Maresi, R Giugno_


Given a group of genomes, represented as the sets of genes that belong to them, the discovery of the pangenomic content is based on the search of genetic homology among the genes for clustering them into families. Thus, pangenomic analyses investigate the membership of the families to the given genomes. This approach is referred to as the gene-oriented approach in contrast to other definitions of the problem that takes into account different genomic features. In the past years, several tools have been developed to discover and analyse pangenomic contents. Because of the hardness of the problem, each tool applies a different strategy for discovering the pangenomic content. This results in a differentiation of the performance of each tool that depends on the composition of the input genomes. This review reports the main analysis instruments provided by the current state of the art tools for the discovery of pangenomic contents. Moreover, unlike previous works, the presented study compares pangenomic tools from a methodological perspective, analysing the causes that lead a given methodology to outperform other tools. The analysis is performed by taking into account different bacterial populations, which are synthetically generated by changing evolutionary parameters. The benchmarks used to compare the pangenomic tools, in addition to the computational pipeline developed for this purpose, are available at https://github.com/InfOmics/pangenes-review. Contact: V. Bonnici, R. Giugno Supplementary information: Supplementary data are available at Briefings in Bioinformatics online.

[Source](https://doi.org/10.1093/bib/bbaa198)

10.1093/bib/bbaa198

---

## A Data Driven Binning Method to Recover More Nucleotide Sequences of Species in a Metagenome

[Source](https://doi.org/10.1109/MERCon50084.2020.9185388)

---

## CasCollect: targeted assembly of CRISPR-associated operons from high-throughput sequencing data

[Source](https://doi.org/10.1093/nargab/lqaa063)

---

## Next-Generation Metagenomics: Methodological Challenges and Opportunities.
 30.04.2020, _I Laudadio, V Fulci, L Stronati, C Carissimi_


_Animals, Big Data, Biodiversity, Computational Biology, Databases, Genetic, High-Throughput Nucleotide Sequencing, Humans, Metagenome, Metagenomics, Microbiological Techniques, Microbiology, Microbiota_
Metagenomics is not only one of the newest omics system science technologies but also one that has arguably the broadest set of applications and impacts globally. Metagenomics has found vast utility not only in environmental sciences, ecology, and public health but also in clinical medicine and looking into the future, in planetary health. In line with the One Health concept, metagenomics solicits collaboration between molecular biologists, geneticists, microbiologists, clinicians, computational biologists, plant biologists, veterinarians, and other health care professionals. Almost every ecological niche of our planet hosts an extremely diverse community of organisms that are still poorly characterized. Detailed characterization of the features of such communities is instrumental to our comprehension of ecological, biological, and clinical complexity. This expert review article evaluates how metagenomics is improving our knowledge of microbiota composition from environmental to human samples. Furthermore, we offer an analysis of the common technical and methodological challenges and potential pitfalls arising from metagenomics approaches, such as metagenomics study design, data processing, and interpretation. All in all, at this critical juncture of further growth of the metagenomics field, it is time to critically reflect on the lessons learned and the future prospects of next-generation metagenomics science, technology, and conceivable applications, particularly from the standpoint of a metagenomics methodology perspective.

[Source](https://doi.org/10.1007/978-981-15-6529-8)

10.1089/omi.2019.0073

---

##  Metagenomics: Techniques, Applications, Challenges and Opportunities

[Source](https://doi.org/10.1007/978-981-15-6529-8)

---

## Big data in biology: The hope and present-day challenges in it

[Source](https://doi.org/10.1016/j.genrep.2020.100869)

---

