## Microbiome research in Africa must be based on equitable partnerships.
 23.05.2024, _OH Oduaran, MO Foláyan, AW Kamng'ona, A Nakimuli, LM Mwapagha, ME Setati, M Owusu, N Mulder, TP Makhalanyane, S Kouidhi_



[Source](https://doi.org/10.1038/s41591-024-03026-2)

10.1038/s41591-024-03026-2

---

## IL-33 controls IL-22-dependent antibacterial defense by modulating the microbiota

[Source](https://doi.org/10.1073/pnas.2310864121)

---

## Meta-analysis of shotgun sequencing of gut microbiota in Parkinson’s disease

[Source](https://doi.org/10.1038/s41531-024-00724-z)

---

## Systematic review of associations between gut microbiome composition and stunting in under-five children

[Source](https://doi.org/10.1038/s41522-024-00517-5)

---

## Production of deoxycholic acid by low-abundant microbial species is associated with impaired glucose metabolism.
 23.05.2024, _A Wahlström, A Brumbaugh, W Sjöland, L Olsson, H Wu, M Henricsson, A Lundqvist, K Makki, SL Hazen, G Bergström, HU Marschall, MA Fischbach, F Bäckhed_


_Deoxycholic Acid, Animals, Gastrointestinal Microbiome, Female, Glucose, Mice, Humans, Diabetes Mellitus, Type 2, Mice, Inbred C57BL, Clostridium, Cholic Acid, Male_
Alterations in gut microbiota composition are suggested to contribute to cardiometabolic diseases, in part by producing bioactive molecules. Some of the metabolites are produced by very low abundant bacterial taxa, which largely have been neglected due to limits of detection. However, the concentration of microbially produced metabolites from these taxa can still reach high levels and have substantial impact on host physiology. To explore this concept, we focused on the generation of secondary bile acids by 7α-dehydroxylating bacteria and demonstrated that addition of a very low abundant bacteria to a community can change the metabolic output dramatically. We show that Clostridium scindens converts cholic acid into the secondary bile acid deoxycholic acid (DCA) very efficiently even though the abundance of C. scindens is low, but still detectable by digital droplet PCR. We also show that colonization of germ-free female mice with a community containing C. scindens induces DCA production and affects host metabolism. Finally, we show that DCA correlates with impaired glucose metabolism and a worsened lipid profile in individuals with type 2 diabetes, which implies that this metabolic pathway may contribute to the development of cardiometabolic disease.

[Source](https://doi.org/10.1038/s41467-024-48543-3)

10.1038/s41467-024-48543-3

---

## Altered gut microbiota is associated with the formation of occult hepatitis B virus infection.
 24.05.2024, _B Liu, H Yang, Q Liao, M Wang, J Huang, R Xu, Z Shan, H Zhong, T Li, C Li, Y Fu, X Rong_


Hepatitis B virus (HBV), a common blood transmission pathogen worldwide, can lead to viral hepatitis, cirrhosis, liver cancer, and other liver diseases. In particular, occult hepatitis B virus infection (OBI) may be caused by an immune response leading to suppressed virus replication. Gut microbiota can change the immunity status of the human body and, therefore, affect the replication of HBV. Thus, to identify whether there are differences in gut microbiota between HBV carriers and OBI carriers, we collected fecal samples from 18 HBV carriers, 24 OBI blood donors, and also 20 healthy blood donors as negative control. After 16S sequencing, we found that the abundance of Faecalibacterium was significantly reduced in samples from OBI blood donors compared with those from healthy blood donors. Compared with samples from HBV carriers, the samples from OBI blood donors had a significantly increased abundance of Subdoligranulum, which might stimulate immune activation, thus inhibiting HBV replication and contributing to the formation of occult infection. Our findings revealed the potential role of gut microbiota in the formation of OBI and further provided a novel strategy for the treatment of HBV infection.IMPORTANCEOccult hepatitis B virus infection (OBI) is a special form of hepatitis B virus infection with hepatitis B surface antigen (HBsAg) positive and hepatitis B virus (HBV) DNA negative. Gut microbiota may contribute to the immune response leading to suppressed virus replication and, thus, participates in the development of OBI. The study on gut microbiota of OBI blood donors provides novel data considerably advancing our understanding of the immune mechanism for the determination of occult hepatitis B virus infection, which is helpful for improving the strategy of the treatment of HBV infection.

[Source](https://doi.org/10.1128/spectrum.00239-24)

10.1128/spectrum.00239-24

---

## Metagenome-assembled genomes (MAGs) suggest an acetate-driven protective role in gut microbiota disrupted by <em>Clostridioides difficile</em>

[Source](https://doi.org/10.1016/j.micres.2024.127739)

---

## From gut microbiota to brain: implications on binge eating disorders.
 26.05.2024, _W Guo, W Xiong_


_Gastrointestinal Microbiome, Humans, Binge-Eating Disorder, Brain-Gut Axis, Brain, Animals, Dysbiosis, Feeding Behavior_
The prevalence of eating disorders has been increasing over the last 50 years. Binge eating disorder (BED) and bulimia nervosa (BN) are two typical disabling, costly and life-threatening eating disorders that substantially compromise the physical well-being of individuals while undermining their psychological functioning. The distressing and recurrent episodes of binge eating are commonly observed in both BED and BN; however, they diverge as BN often involves the adoption of inappropriate compensatory behaviors aimed at averting weight gain. Normal eating behavior is coordinated by a well-regulated trade-off between intestinal and central ingestive mechanism. Conversely, despite the fact that the etiology of BED and BN remains incompletely resolved, emerging evidence corroborates the notion that dysbiosis of gastrointestinal microbiome and its metabolites, alteration of gut-brain axis, as well as malfunctioning central circuitry regulating motivation, execution and reward all contribute to the pathology of binge eating. In this review, we aim to outline the current state of knowledge pertaining to the potential mechanisms through which each component of the gut-brain axis participates in binge eating behaviors, and provide insight for the development of microbiome-based therapeutic interventions that hold promise in ameliorating patients afflicted with binge eating disorders.

[Source](https://doi.org/10.1080/19490976.2024.2357177)

10.1080/19490976.2024.2357177

---

## Pathogenesis and therapeutic opportunities of gut microbiome dysbiosis in critical illness.
 26.05.2024, _NA Cho, K Strayer, B Dobson, B McDonald_


_Dysbiosis, Humans, Critical Illness, Gastrointestinal Microbiome, Cross Infection, Intensive Care Units, Animals, Host Microbial Interactions, Bacteria_
For many years, it has been hypothesized that pathological changes to the gut microbiome in critical illness is a driver of infections, organ dysfunction, and other adverse outcomes in the intensive care unit (ICU). The advent of contemporary microbiome methodologies and multi-omics tools have allowed researchers to test this hypothesis by dissecting host-microbe interactions in the gut to better define its contribution to critical illness pathogenesis. Observational studies of patients in ICUs have revealed that gut microbial communities are profoundly altered in critical illness, characterized by markedly reduced alpha diversity, loss of commensal taxa, and expansion of potential pathogens. These key features of ICU gut dysbiosis have been associated with adverse outcomes including life-threatening hospital-acquired (nosocomial) infections. Current research strives to define cellular and molecular mechanisms connecting gut dysbiosis with infections and other outcomes, and to identify opportunities for therapeutic modulation of host-microbe interactions. This review synthesizes evidence from studies of critically ill patients that have informed our understanding of intestinal dysbiosis in the ICU, mechanisms linking dysbiosis to infections and other adverse outcomes, as well as clinical trials of microbiota-modifying therapies. Additionally, we discuss novel avenues for precision microbial therapeutics to combat nosocomial infections and other life-threatening complications of critical illness.

[Source](https://doi.org/10.1080/19490976.2024.2351478)

10.1080/19490976.2024.2351478

---

## Pathobiological signatures of dysbiotic lung injury in pediatric patients undergoing stem cell transplantation

[Source](https://doi.org/10.1038/s41591-024-02999-4)

---

## Leveraging existing 16S rRNA gene surveys to decipher microbial signatures and dysbiosis in cervical carcinogenesis

[Source](https://doi.org/10.1038/s41598-024-62531-z)

---

## Targeting <em>Megasphaera</em> species to promote cervicovaginal health

[Source](https://www.cell.com/trends/microbiology/abstract/S0966-842X(24)00131-8)

---

## Prodrug-conjugated tumor-seeking commensals for targeted cancer therapy

[Source](https://doi.org/10.1038/s41467-024-48661-y)

---

## Fecal microbiota transplantation significantly improved respiratory failure of amyotrophic lateral sclerosis

[Source](https://doi.org/10.1080/19490976.2024.2353396)

---

## The use of probiotics and prebiotics in decolonizing pathogenic bacteria from the gut; a systematic review and meta-analysis of clinical outcomes

[Source](https://doi.org/10.1080/19490976.2024.2356279)

---

## Advanced microbiome therapeutics as a novel modality for oral delivery of peptides to manage metabolic diseases.
 23.05.2024, _R Vazquez-Uribe, KA Hedin, TR Licht, M Nieuwdorp, MOA Sommer_


The rising prevalence of metabolic diseases calls for innovative treatments. Peptide-based drugs have transformed the management of conditions such as obesity and type 2 diabetes. Yet, challenges persist in oral delivery of these peptides. This review explores the potential of 'advanced microbiome therapeutics' (AMTs), which involve engineered microbes for delivery of peptides in situ, thereby enhancing their bioavailability. Preclinical work on AMTs has shown promise in treating animal models of metabolic diseases, including obesity, type 2 diabetes, and metabolic dysfunction-associated steatotic liver disease. Outstanding challenges toward realizing the potential of AMTs involve improving peptide expression, ensuring predictable colonization control, enhancing stability, and managing safety and biocontainment concerns. Still, AMTs have potential for revolutionizing the treatment of metabolic diseases, potentially offering dynamic and personalized novel therapeutic approaches.

[Source](https://www.cell.com/trends/endocrinology-metabolism/fulltext/S1043-2760(24)00115-2)

10.1016/j.tem.2024.04.021

---

## Identification of the mutual gliding locus as a factor for gut colonization in non-native bee hosts using the ARTP mutagenesis.
 25.05.2024, _Y Meng, X Zhang, Y Zhai, Y Li, Z Shao, S Liu, C Zhang, XH Xing, H Zheng_


_Animals, Bees, Gastrointestinal Microbiome, Symbiosis, Mutagenesis, Mutation_
The gut microbiota and their hosts profoundly affect each other's physiology and evolution. Identifying host-selected traits is crucial to understanding the processes that govern the evolving interactions between animals and symbiotic microbes. Current experimental approaches mainly focus on the model bacteria, like hypermutating Escherichia coli or the evolutionary changes of wild stains by host transmissions. A method called atmospheric and room temperature plasma (ARTP) may overcome the bottleneck of low spontaneous mutation rates while maintaining mild conditions for the gut bacteria.

[Source](https://doi.org/10.1186/s40168-024-01813-0)

10.1186/s40168-024-01813-0

---

## Symbiotic bacteria and fungi proliferate in diapause and may enhance overwintering survival in a solitary bee

[Source](https://doi.org/10.1093/ismejo/wrae089)

---

## Phages enhance both phytopathogen density control and rhizosphere microbiome suppressiveness.
 23.05.2024, _X Wang, S Wang, M Huang, Y He, S Guo, K Yang, N Wang, T Sun, H Yang, T Yang, Y Xu, Q Shen, V-P Friman, Z Wei_


Bacteriophages, viruses that specifically target plant pathogenic bacteria, have emerged as a promising alternative to traditional agrochemicals. However, it remains unclear how phages should be applied to achieve efficient pathogen biocontrol and to what extent their efficacy is shaped by indirect interactions with the resident microbiota. Here, we tested if the phage biocontrol efficacy of Ralstonia solanacearum phytopathogenic bacterium can be improved by increasing the phage cocktail application frequency and if the phage efficacy is affected by pathogen-suppressing bacteria already present in the rhizosphere. We find that increasing phage application frequency improves R. solanacearum density control, leading to a clear reduction in bacterial wilt disease in both greenhouse and field experiments with tomato. The high phage application frequency also increased the diversity of resident rhizosphere microbiota and enriched several bacterial taxa that were associated with the reduction in pathogen densities. Interestingly, these taxa often belonged to Actinobacteria known for antibiotics production and soil suppressiveness. To test if they could have had secondary effects on R. solanacearum biocontrol, we isolated Actinobacteria from Nocardia and Streptomyces genera and tested their suppressiveness to the pathogen in vitro and in planta. We found that these taxa could clearly inhibit R. solanacearum growth and constrain bacterial wilt disease, especially when combined with the phage cocktail. Together, our findings unravel an undiscovered benefit of phage therapy, where phages trigger a second line of defense by the pathogen-suppressing bacteria that already exist in resident microbial communities.

[Source](https://doi.org/10.1128/mbio.03016-23)

10.1128/mbio.03016-23

---

## Global epistasis and the emergence of function in microbial consortia.
 22.05.2024, _J Diaz-Colunga, A Skwara, JCC Vila, D Bajic, A Sanchez_


The many functions of microbial communities emerge from a complex web of interactions between organisms and their environment. This poses a significant obstacle to engineering microbial consortia, hindering our ability to harness the potential of microorganisms for biotechnological applications. In this study, we demonstrate that the collective effect of ecological interactions between microbes in a community can be captured by simple statistical models that predict how adding a new species to a community will affect its function. These predictive models mirror the patterns of global epistasis reported in genetics, and they can be quantitatively interpreted in terms of pairwise interactions between community members. Our results illuminate an unexplored path to quantitatively predicting the function of microbial consortia from their composition, paving the way to optimizing desirable community properties and bringing the tasks of predicting biological function at the genetic, organismal, and ecological scales under the same quantitative formalism.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(24)00411-2)

10.1016/j.cell.2024.04.016

---

## Bacterial dynamics of the plastisphere microbiome exposed to sub-lethal antibiotic pollution.
 27.05.2024, _B Joannard, C Sanchez-Cid_


_Anti-Bacterial Agents, Bacteria, Microbiota, RNA, Ribosomal, 16S, Integrons, Drug Resistance, Bacterial, Water Pollutants, Chemical, Microplastics, High-Throughput Nucleotide Sequencing, Metagenome, Plasmids, Water Microbiology, Drug Resistance, Microbial_
Antibiotics and microplastics are two major aquatic pollutants that have been associated to antibiotic resistance selection in the environment and are considered a risk to human health. However, little is known about the interaction of these pollutants at environmental concentrations and the response of the microbial communities in the plastisphere to sub-lethal antibiotic pollution. Here, we describe the bacterial dynamics underlying this response in surface water bacteria at the community, resistome and mobilome level using a combination of methods (next-generation sequencing and qPCR), sequencing targets (16S rRNA gene, pre-clinical and clinical class 1 integron cassettes and metagenomes), technologies (short and long read sequencing), and assembly approaches (non-assembled reads, genome assembly, bacteriophage and plasmid assembly).

[Source](https://doi.org/10.1186/s40168-024-01803-2)

10.1186/s40168-024-01803-2

---

## Global soil metagenomics reveals distribution and predominance of Deltaproteobacteria in nitrogen-fixing microbiome.
 27.05.2024, _Y Masuda, K Mise, Z Xu, Z Zhang, Y Shiratori, K Senoo, H Itoh_


_Soil Microbiology, Nitrogen Fixation, Metagenomics, Microbiota, Deltaproteobacteria, Soil, Phylogeny, Nitrogen, Metagenome_
Biological nitrogen fixation is a fundamental process sustaining all life on earth. While distribution and diversity of N2-fixing soil microbes have been investigated by numerous PCR amplicon sequencing of nitrogenase genes, their comprehensive understanding has been hindered by lack of de facto standard protocols for amplicon surveys and possible PCR biases. Here, by fully leveraging the planetary collections of soil shotgun metagenomes along with recently expanded culture collections, we evaluated the global distribution and diversity of terrestrial diazotrophic microbiome.

[Source](https://doi.org/10.1186/s40168-024-01812-1)

10.1186/s40168-024-01812-1

---

## A global initiative for ecological and evolutionary hologenomics

[Source](https://www.cell.com/trends/ecology-evolution/abstract/S0169-5347(24)00074-0)

---

## Fungal diversity differences in the indoor dust microbiome from built environments on earth and in space.
 27.05.2024, _N Nastasi, SR Haines, A Bope, ME Meyer, JM Horack, KC Dannemiller_


_Dust, Fungi, Microbiota, Humans, Spacecraft, Air Pollution, Indoor, Built Environment, Bacteria, Air Microbiology, Earth, Planet, Humidity_
Human occupied built environments are no longer confined to Earth. In fact, there have been humans living and working in low-Earth orbit on the International Space Station (ISS) since November 2000. With NASA's Artemis missions and the age of commercial space stations set to begin, more human-occupied spacecraft than ever will be in Earth's orbit and beyond. On Earth and in the ISS, microbes, especially fungi, can be found in dust and grow when unexpected, elevated moisture conditions occur. However, we do not yet know how indoor microbiomes in Earth-based homes and in the ISS differ due to their unique set of environmental conditions. Here we show that bacterial and fungal communities are different in dust collected from vacuum bags on Earth and the ISS, with Earth-based homes being more diverse (465 fungal OTUs and 237 bacterial ASVs) compared to the ISS (102 fungal OTUs and 102 bacterial ASVs). When dust from these locations were exposed to varying equilibrium relative humidity conditions (ERH), there were also significant fungal community composition changes as ERH and time elevated increased (Bray Curtis: R2 = 0.35, P = 0.001). These findings can inform future spacecraft design to promote healthy indoor microbiomes that support crew health, spacecraft integrity, and planetary protection.

[Source](https://doi.org/10.1038/s41598-024-62191-z)

10.1038/s41598-024-62191-z

---

## A novel clinical metaproteomics workflow enables bioinformatic analysis of host-microbe dynamics in disease.
 23.05.2024, _K Do, S Mehta, R Wagner, D Bhuming, AT Rajczewski, APN Skubitz, JE Johnson, TJ Griffin, PD Jagtap_


Clinical metaproteomics has the potential to offer insights into the host-microbiome interactions underlying diseases. However, the field faces challenges in characterizing microbial proteins found in clinical samples, usually present at low abundance relative to the host proteins. As a solution, we have developed an integrated workflow coupling mass spectrometry-based analysis with customized bioinformatic identification, quantification, and prioritization of microbial proteins, enabling targeted assay development to investigate host-microbe dynamics in disease. The bioinformatics tools are implemented in the Galaxy ecosystem, offering the development and dissemination of complex bioinformatic workflows. The modular workflow integrates MetaNovo (to generate a reduced protein database), SearchGUI/PeptideShaker and MaxQuant [to generate peptide-spectral matches (PSMs) and quantification], PepQuery2 (to verify the quality of PSMs), Unipept (for taxonomic and functional annotation), and MSstatsTMT (for statistical analysis). We have utilized this workflow in diverse clinical samples, from the characterization of nasopharyngeal swab samples to bronchoalveolar lavage fluid. Here, we demonstrate its effectiveness via analysis of residual fluid from cervical swabs. The complete workflow, including training data and documentation, is available via the Galaxy Training Network, empowering non-expert researchers to utilize these powerful tools in their clinical studies.

[Source](https://doi.org/10.1128/msphere.00793-23)

10.1128/msphere.00793-23

---

## A comparative study of supervised and unsupervised machine learning algorithms applied to human microbiome.
 20.05.2024, _E Kalluçi, B Preni, X Dhamo, E Noka, S Bardhi, A Macchia, G Bonetti, K Dhuli, K Donato, M Bertelli, LJM Zambrano, S Janaqi_


_Humans, Supervised Machine Learning, Unsupervised Machine Learning, Microbiota, RNA, Ribosomal, 16S, Colorectal Neoplasms, Gastrointestinal Microbiome, Algorithms, Feces, Adenoma_
The human microbiome, consisting of diverse bacte-rial, fungal, protozoan and viral species, exerts a profound influence on various physiological processes and disease susceptibility. However, the complexity of microbiome data has presented significant challenges in the analysis and interpretation of these intricate datasets, leading to the development of specialized software that employs machine learning algorithms for these aims.

[Source](https://doi.org/10.7417/CT.2024.5051)

10.7417/CT.2024.5051

---

