## Metagenomic prediction of antimicrobial resistance in critically ill patients with lower respiratory tract infections

[Source](https://doi.org/10.1186/s13073-022-01072-4)

---

## Alterations in gut microbiota of esophageal squamous cell carcinoma patients.
 11.07.2022, _MK Cheung, GGL Yue, S Lauw, CSY Li, MY Yung, SC Ng, HC Yip, HS Kwan, PWY Chiu, CBS Lau_


Esophageal squamous cell carcinoma (ESCC) is the most common histological subtype of esophageal cancer worldwide. Patients with ESCC display an altered esophageal microbiota compared with healthy individuals; however, little is known about the gut microbiota in ESCC.

[Source](https://doi.org/10.1111/jgh.15941)

10.1111/jgh.15941

---

## Altered Pattern of Immunoglobulin A-Targeted Microbiota in Inflammatory Bowel Disease After Fecal Transplantation.
 11.07.2022, _WQ Huang, HL Huang, W Peng, YD Liu, YL Zhou, HM Xu, LJ Zhang, C Zhao, YQ Nie_


Adaptive immune response to the gut microbiota is one of the main drivers of inflammatory bowel disease (IBD). Under inflammatory conditions, immunoglobulin (Ig)-targeted bacteria are altered. However, changes in Ig-targeted bacteria in Asian patients with IBD with ulcerative colitis (UC) remain unclear. Furthermore, changes in IgA-targeted bacteria in patients with UC treated with fecal microbiota transplantation (FMT) are unclear. Here, we analyzed fecal samples of patients with IBD and patients with UC before and after FMT by flow cytometry. We found that the percentage of IgA/G-coated bacteria can be used to assess the severity of IBD. Besides oral pharyngeal bacteria such as Streptococcus, we hypothesized that Megamonas, Acinetobacter, and, especially, Staphylococcus might play an important role in IBD pathogenesis. Moreover, we evaluated the influence of FMT on IgA-coated bacteria in patients with UC. We found that IgA-bacterial interactions were re-established in human FMT recipients and resembled those in the healthy fecal donors. Additionally, the IgA targeting was not influenced by delivery methods: gastroscopy spraying and colonic transendoscopic enteral tubing (TET). Then, we established an acute dextran sulfate sodium (DSS)-induced mouse model to explore whether FMT intervention would impact IgA/G memory B cell in the intestine. We found that after FMT, both IgA/G memory B cell and the percentage of IgA/G-targeted bacteria were restored to normal levels in DSS mice.

[Source](https://doi.org/10.3389/fmicb.2022.873018)

10.3389/fmicb.2022.873018

---

## 16S rRNA gene sequencing reveals an altered composition of gut microbiota in children with <em>Mycoplasma pneumoniae</em> pneumonia treated with azithromycin

[Source](https://doi.org/10.2323/jgam.2022.05.004)

---

## Clinical and metagenomic profiling of hormonal acne-prone skin in different populations.
 10.07.2022, _N Hrapovic, T Richard, C Messaraa, X Li, A Abbaspour, S Fabre, A Mavon, B Andersson, I Khmaladze_


Acne is one of the most common skin concerns of unknown etiology, often connected to the menstrual cycle in women, and possibly to the microbial profile and function.

[Source](https://doi.org/10.1111/jocd.15225)

10.1111/jocd.15225

---

## Microbiota diversity in nonalcoholic fatty liver disease and in drug-induced liver injury.
 11.07.2022, _C Rodriguez-Diaz, B Taminiau, A García-García, A Cueto, M Robles-Díaz, A Ortega-Alonso, F Martín-Reyes, G Daube, J Sanabria-Cabrera, M Jimenez-Perez, M Isabel Lucena, RJ Andrade, E García-Fuentes, M García-Cortes_


The gut microbiota could play a significant role in the progression of nonalcoholic fatty liver disease (NAFLD); however, its relevance in drug-induced liver injury (DILI) remains unexplored. Since the two hepatic disorders may share damage pathways, we analysed the metagenomic profile of the gut microbiota in NAFLD, with or without significant liver fibrosis, and in DILI, and we identified the main associated bacterial metabolic pathways. In the NAFLD group, we found a decrease in Alistipes, Barnesiella, Eisenbergiella, Flavonifractor, Fusicatenibacter, Gemminger, Intestinimonas, Oscillibacter, Parasutterella, Saccharoferementans and Subdoligranulum abundances compared with those in both the DILI and control groups. Additionally, we detected an increase in Enterobacter, Klebsiella, Sarcina and Turicibacter abundances in NAFLD, with significant liver fibrosis, compared with those in NAFLD with no/mild liver fibrosis. The DILI group exhibited a lower microbial bacterial richness than the control group, and lower abundances of Acetobacteroides, Blautia, Caloramator, Coprococcus, Flavobacterium, Lachnospira, Natronincola, Oscillospira, Pseudobutyrivibrio, Shuttleworthia, Themicanus and Turicibacter compared with those in the NAFLD and control groups. We found seven bacterial metabolic pathways that were impaired only in DILI, most of which were associated with metabolic biosynthesis. In the NAFLD group, most of the differences in the bacterial metabolic pathways found in relation to those in the DILI and control groups were related to fatty acid and lipid biosynthesis. In conclusion, we identified a distinct bacterial profile with specific bacterial metabolic pathways for each type of liver disorder studied. These differences can provide further insight into the physiopathology and development of NAFLD and DILI.

[Source](https://doi.org/10.1016/j.phrs.2022.106348)

10.1016/j.phrs.2022.106348

---

## The potential role of prebiotics, probiotics, and synbiotics in adjuvant cancer therapy especially colorectal cancer.
 11.07.2022, _Z Alam, X Shang, K Effat, F Kanwal, X He, Y Li, C Xu, W Niu, AR War, Y Zhang_


Cancer is a global health issue that is rising swiftly with younger people and an increased number of patients. The role of human microbiota in the pathophysiology of tumors has been paid more and more attention. Microecologics including prebiotics, probiotics, and synbiotics are among the best validated/proven resources for the application of microbiological prophylaxis and therapy. There is strong evidence that microecologics have anti-cancer activity and their potential association with cancer is significant. In this review, we will focus on the role of prebiotics, probiotics, and synbiotics in tumor suppression in maintaining the colon barrier, metabolism, immune regulation, inhibition of host tumor cell proliferation, and epidemiological-based recommendations. Besides, other signs illuminate the role of microecological agents to adjunct the cancer treatment and counter the toxic side effects of cancer drugs. In addition, we will explore their role in chemotherapy, where these probiotics can be used as an adjunct to chemotherapy, counteracting the toxic side effects of chemotherapy drugs to minimize or optimize the therapeutic effect. In the treatment of cancer, we can see the role of prebiotics, probiotics, synbiotics, and their application in cancer patients, and the effectiveness effect can be considered as a clinical benefit. PRACTICAL APPLICATIONS: A large number of studies have shown that microecologics including prebiotics, probiotics, and synbiotics play an important role in regulating intestinal microecology and contribute to the prevention and treatment of cancer, indicating that prebiotics, probiotics, and synbiotics have the potential to be used as microecological modulators in the adjuvant therapy of cancer. However, it is not clear what is the anti-tumor mechanism of these microecologics and how they antagonize the side effects of cancer chemotherapy and protect normal cells. This paper reviews the role of prebiotics, probiotics, and synbiotics in tumor suppression in maintaining the colon barrier, metabolism, immune regulation, and prevention of rapid growth of host cells, as well as their potential role in cancer chemotherapy. This review helps to better understand the relationship between prebiotics, probiotics, and synbiotics with immune regulation, intestinal microecology, metabolic regulation, and cell proliferation and provides strong evidence for their potential application as microecologics in cancer adjuvant therapy.

[Source](https://doi.org/10.1111/jfbc.14302)

10.1111/jfbc.14302

---

## Probiotics, prebiotics and postbiotics for better sleep quality: a narrative review.
 11.07.2022, _JE Haarhuis, A Kardinaal, GAM Kortman_


There is a growing prevalence of sleep problems and insomnia worldwide, urging the development of new treatments to tackle this increase. Several studies have suggested that the gut microbiome might influence sleep quality. The gut microbiome affects the host's health via the production of metabolites and compounds with neuroactive and immunomodulatory properties, which include short-chain fatty acids, secondary bile acids and neurotransmitters. Several of these metabolites and compounds are independently known as wakefulness-promoting (serotonin, epinephrine, dopamine, orexin, histamine, acetylcholine, cortisol) or sleep-promoting (gamma-aminobutyric acid, melatonin). The primary aim of this review was to evaluate the potential of pro-, pre- and postbiotic treatments to improve sleep quality. Additionally, we aimed to evaluate whether each of the treatments could ameliorate stress and anxiety, which are known to bidirectionally correlate with sleep problems. Lastly, we provided a mechanistic explanation for our findings. A literature search was conducted using PubMed, Scopus, Web of Science, and Google Scholar to compare all human trials that met our inclusion criteria and were published before November 2021. We furthermore discussed relevant findings from animal experiments to provide a mechanistic insight. While several studies found that sleep latency, sleep length, and cortisol levels improved after pro-, pre- or postbiotic treatment, others did not show any significant improvements for sleep quality, stress, or anxiety. These discrepancies can be explained by between-study variations in study designs, study populations, treatments, type and level of distress, and sex differences. We conclude that the trials discussed provide some evidence for prebiotics, postbiotics, and traditional probiotics, such as those belonging to lactobacilli and bifidobacteria, to improve sleep quality and stress, but stronger evidence might be found in the future after implementing the methodological adjustments that are suggested in this review.

[Source](https://doi.org/10.3920/BM2021.0122)

10.3920/BM2021.0122

---

## Gut Microbiota: A Novel Therapeutic Target for Parkinson's Disease.
 12.07.2022, _M Zhu, X Liu, Y Ye, X Yan, Y Cheng, L Zhao, F Chen, Z Ling_


_Animals, Dysbiosis, Fecal Microbiota Transplantation, Gastrointestinal Microbiome, Neurodegenerative Diseases, Parkinson Disease_
Parkinson's disease (PD) is the second most common neurodegenerative disease characterized by motor dysfunction. Growing evidence has demonstrated that gut dysbiosis is involved in the occurrence, development and progression of PD. Numerous clinical trials have identified the characteristics of the changed gut microbiota profiles, and preclinical studies in PD animal models have indicated that gut dysbiosis can influence the progression and onset of PD via increasing intestinal permeability, aggravating neuroinflammation, aggregating abnormal levels of α-synuclein fibrils, increasing oxidative stress, and decreasing neurotransmitter production. The gut microbiota can be considered promising diagnostic and therapeutic targets for PD, which can be regulated by probiotics, psychobiotics, prebiotics, synbiotics, postbiotics, fecal microbiota transplantation, diet modifications, and Chinese medicine. This review summarizes the recent studies in PD-associated gut microbiota profiles and functions, the potential roles, and mechanisms of gut microbiota in PD, and gut microbiota-targeted interventions for PD. Deciphering the underlying roles and mechanisms of the PD-associated gut microbiota will help interpret the pathogenesis of PD from new perspectives and elucidate novel therapeutic strategies for PD.

[Source](https://doi.org/10.3389/fimmu.2022.937555)

10.3389/fimmu.2022.937555

---

## Obesity enriches for tumor protective microbial metabolites and treatment refractory cells to confer therapy resistance in PDAC.
 11.07.2022, _K Kesh, R Mendez, B Mateo-Victoriano, VT Garrido, B Durden, VK Gupta, A Oliveras Reyes, N Merchant, J Datta, S Banerjee, S Banerjee_


Obesity causes chronic inflammation and changes in gut microbiome. However, how this contributes to poor survival and therapy resistance in patients with pancreatic cancer remain undetermined. Our current study shows that high fat diet-fed obese pancreatic tumor bearing mice do not respond to standard of care therapy with gemcitabine and paclitaxel when compared to corresponding control diet-fed mice. C57BL6 mice were put on control and high fat diet for 1 month following with pancreatic tumors were implanted in both groups. Microbiome of lean (control) and obese (high fat diet fed) mice was analyzed. Fecal matter transplant from control mice to obese mice sensitized tumors to chemotherapy and demonstrated extensive cell death. Analysis of gut microbiome showed an enrichment of queuosine (Q) producing bacteria in obese mice and an enrichment of S-adenosyl methionine (SAM) producing bacteria in control diet-fed mice. Further, supplementation of obese animals with SAM sensitized pancreatic tumors to chemotherapy. Treatment of pancreatic cancer cells with Q increased PRDX1 involved in oxidative stress protection. In parallel, tumors in obese mice showed increase in CD133+ treatment refractory tumor populations compared to control animals. These observations indicated that microbial metabolite Q accumulation in high fat diet-fed mice protected tumors from chemotherapy induced oxidative stress by upregulating PRDX1. This protection could be reversed by treatment with SAM. We conclude that relative concentration of SAM and queuosine in fecal samples of pancreatic cancer patients can be developed as a potential biomarker and therapeutic target in chemotherapy refractory pancreatic cancer.

[Source](https://doi.org/10.1080/19490976.2022.2096328)

10.1080/19490976.2022.2096328

---

## Profiling of d-alanine production by the microbial isolates of rat gut microbiota.
 12.07.2022, _CJ Lee, TA Qiu, Z Hong, Z Zhang, Y Min, L Zhang, L Dai, H Zhao, T Si, JV Sweedler_


d-alanine (d-Ala) and several other d-amino acids (d-AAs) act as hormones and neuromodulators in nervous and endocrine systems. Unlike the endogenously synthesized d-serine in animals, d-Ala may be from exogenous sources, e.g., diet and intestinal microorganisms. However, it is unclear if the capability to produce d-Ala and other d-AAs varies among different microbial strains in the gut. We isolated individual microorganisms of rat gut microbiota and profiled their d-AA production in vitro, focusing on d-Ala. Serial dilutions of intestinal contents from adult male rats were plated on agar to obtain clonal cultures. Using MALDI-TOF MS for rapid strain typing, we identified 38 unique isolates, grouped into 11 species based on 16S rRNA gene sequences. We then used two-tier screening to profile bacterial d-AA production, combining a d-amino acid oxidase-based enzymatic assay for rapid assessment of non-acidic d-AA amount and chiral LC-MS/MS to quantify individual d-AAs, revealing 19 out of the 38 isolated strains as d-AA producers. LC-MS/MS analysis of the eight top d-AA producers showed high levels of d-Ala in all strains tested, with substantial inter- and intra-species variations. Though results from the enzymatic assay and LC-MS/MS analysis aligned well, LC-MS/MS further revealed the existence of d-glutamate and d-aspartate, which are poor substrates for this enzymatic assay. We observed large inter- and intra-species variation of d-AA production profiles from rat gut microbiome species, demonstrating the importance of chemical profiling of gut microbiota in addition to sequencing, furthering the idea that microbial metabolites modulate host physiology.

[Source](https://doi.org/10.1096/fj.202101595R)

10.1096/fj.202101595R

---

## Within-community variation of interspecific divergence patterns in passerine gut microbiota.
 11.07.2022, _J Kubovčiak, L Schmiedová, T Albrecht, M Těšický, O Tomášek, T Kauzálová, J Kreisinger_


Gut microbiota (GM) often exhibit variation between different host species and co-divergence with hosts' phylogeny. Identifying these patterns is a key for understanding the mechanisms that shaped symbiosis between GM and its hosts. Therefore, both GM-host species specificity and GM-host co-divergence have been investigated by numerous studies. However, most of them neglected a possibility that different groups of bacteria within GM can vary in the tightness of their association with the host. Consequently, unlike most of these studies, we aimed to directly address how the strength of GM-host species specificity and GM-host co-divergence vary across different GM clades. We decomposed GM communities of 52 passerine species (394 individuals), characterized by 16S rRNA amplicon sequence variant (ASV) profiles, into monophyletic Binned Taxonomic units (BTUs). Subsequently, we analyzed strength of host species specificity and correlation with host phylogeny separately for resulting BTUs. We found that most BTUs exhibited significant host-species specificity in their composition. Notably, BTUs exhibiting high host-species specificity comprised bacterial taxa known to impact host's physiology and immune system. However, BTUs rarely displayed significant co-divergence with host phylogeny, suggesting that passerine GM evolution is not shaped primarily through a shared evolutionary history between the host and its gut microbes.

[Source](https://doi.org/10.1002/ece3.9071)

10.1002/ece3.9071

---

## Phage defence by deaminase-mediated depletion of deoxynucleotides in bacteria

[Source](https://doi.org/10.1038/s41564-022-01162-4)

---

## CheckM2: a rapid, scalable and accurate tool for assessing microbial genome quality using machine learning

[Source](https://www.biorxiv.org/content/10.1101/2022.07.11.499243v1)

---

## Cancer type classification using plasma cell-free RNAs derived from human and microbes

[Source](https://doi.org/10.7554/eLife.75181)

---

## Temporal Alignment of Longitudinal Microbiome Data.
 11.07.2022, _R Armoni, E Borenstein_


A major challenge in working with longitudinal data when studying some temporal process is the fact that differences in pace and dynamics might overshadow similarities between processes. In the case of longitudinal microbiome data, this may hinder efforts to characterize common temporal trends across individuals or to harness temporal information to better understand the link between the microbiome and the host. One possible solution to this challenge lies in the field of "temporal alignment" - an approach for optimally aligning longitudinal samples obtained from processes that may vary in pace. In this work we investigate the use of alignment-based analysis in the microbiome domain, focusing on microbiome data from infants in their first years of life. Our analyses center around two main use-cases: First, using the overall alignment score as a measure of the similarity between microbiome developmental trajectories, and showing that this measure can capture biological differences between individuals. Second, using the specific matching obtained between pairs of samples in the alignment to highlight changes in pace and temporal dynamics, showing that it can be utilized to predict the age of infants based on their microbiome and to uncover developmental delays. Combined, our findings serve as a proof-of-concept for the use of temporal alignment as an important and beneficial tool in future longitudinal microbiome studies.

[Source](https://doi.org/10.3389/fmicb.2022.909313)

10.3389/fmicb.2022.909313

---

## A Bottom-up Approach to Testing Hypotheses That Have a Branching Tree Dependence Structure, with Error Rate Control.
 11.07.2022, _Y Li, YJ Hu, GA Satten_


Modern statistical analyses often involve testing large numbers of hypotheses. In many situations, these hypotheses may have an underlying tree structure that both helps determine the order that tests should be conducted but also imposes a dependency between tests that must be accounted for. Our motivating example comes from testing the association between a trait of interest and groups of microbes that have been organized into operational taxonomic units (OTUs) or amplicon sequence variants (ASVs). Given p-values from association tests for each individual OTU or ASV, we would like to know if we can declare a certain species, genus, or higher taxonomic group to be associated with the trait. For this problem, a bottom-up testing algorithm that starts at the lowest level of the tree (OTUs or ASVs) and proceeds upward through successively higher taxonomic groupings (species, genus, family etc.) is required. We develop such a bottom-up testing algorithm that controls a novel error rate that we call the false selection rate. By simulation, we also show that our approach is better at finding driver taxa, the highest level taxa below which there are dense association signals. We illustrate our approach using data from a study of the microbiome among patients with ulcerative colitis and healthy controls.

[Source](https://doi.org/10.1080/01621459.2020.1799811)

10.1080/01621459.2020.1799811

---

