## <em>Akkermansia muciniphila </em>and <em>Parabacteroides distasonis </em>as prognostic markers for relapse in ulcerative colitis patients

[Source](https://doi.org/10.3389/fcimb.2024.1367998)

---

## Obesity-enriched gut microbe degrades myo-inositol and promotes lipid absorption.
 12.07.2024, _C Wu, F Yang, H Zhong, J Hong, H Lin, M Zong, H Ren, S Zhao, Y Chen, Z Shi, X Wang, J Shen, Q Wang, M Ni, B Chen, Z Cai, M Zhang, Z Cao, K Wu, A Gao, J Li, C Liu, M Xiao, Y Li, J Shi, Y Zhang, X Xu, W Gu, Y Bi, G Ning, W Wang, J Wang, R Liu_


Numerous studies have reported critical roles for the gut microbiota in obesity. However, the specific microbes that causally contribute to obesity and the underlying mechanisms remain undetermined. Here, we conducted shotgun metagenomic sequencing in a Chinese cohort of 631 obese subjects and 374 normal-weight controls and identified a Megamonas-dominated, enterotype-like cluster enriched in obese subjects. Among this cohort, the presence of Megamonas and polygenic risk exhibited an additive impact on obesity. Megamonas rupellensis possessed genes for myo-inositol degradation, as demonstrated in vitro and in vivo, and the addition of myo-inositol effectively inhibited fatty acid absorption in intestinal organoids. Furthermore, mice colonized with M. rupellensis or E. coli heterologously expressing the myo-inositol-degrading iolG gene exhibited enhanced intestinal lipid absorption, thereby leading to obesity. Altogether, our findings uncover roles for M. rupellensis as a myo-inositol degrader that enhances lipid absorption and obesity, suggesting potential strategies for future obesity management.

[Source](https://doi.org/10.1016/j.chom.2024.06.012)

10.1016/j.chom.2024.06.012

---

## Gut microbiome in endometriosis: a cohort study on 1000 individuals

[Source](https://doi.org/10.1186/s12916-024-03503-y)

---

## Metagenomic analysis reveals altered gut virome and diagnostic potential in pancreatic cancer.
 17.07.2024, _P Zhang, H Shi, R Guo, L Li, X Guo, H Yang, D Chang, Y Cheng, G Zhao, S Li, Q Zhong, H Zhang, P Zhao, C Fu, Y Song, L Yang, Y Wang, Y Zhang, J Jiang, T Wang, J Zhao, Y Li, B Wang, F Chen, H Zhao, Y Wang, J Wang, S Ma_


_Humans, Pancreatic Neoplasms, Gastrointestinal Microbiome, Virome, Metagenomics, Feces, Viruses, Metagenome, Bacteria, Middle Aged, Male, Female, Aged, Case-Control Studies_
Pancreatic cancer (PC) is a highly aggressive malignancy with a poor prognosis, making early diagnosis crucial for improving patient outcomes. While the gut microbiome, including bacteria and viruses, is believed to be essential in cancer pathogenicity, the potential contribution of the gut virome to PC remains largely unexplored. In this study, we conducted a comparative analysis of the gut viral compositional and functional profiles between PC patients and healthy controls, based on fecal metagenomes from two publicly available data sets comprising a total of 101 patients and 82 healthy controls. Our results revealed a decreasing trend in the gut virome diversity of PC patients with disease severity. We identified significant alterations in the overall viral structure of PC patients, with a meta-analysis revealing 219 viral operational taxonomic units (vOTUs) showing significant differences in relative abundance between patients and healthy controls. Among these, 65 vOTUs were enriched in PC patients, and 154 were reduced. Host prediction revealed that PC-enriched vOTUs preferentially infected bacterial members of Veillonellaceae, Enterobacteriaceae, Fusobacteriaceae, and Streptococcaceae, while PC-reduced vOTUs were more likely to infect Ruminococcaceae, Lachnospiraceae, Clostridiaceae, Oscillospiraceae, and Peptostreptococcaceae. Furthermore, we constructed random forest models based on the PC-associated vOTUs, achieving an optimal average area under the curve (AUC) of up to 0.879 for distinguishing patients from controls. Through additional 10 public cohorts, we demonstrated the reproducibility and high specificity of these viral signatures. Our study suggests that the gut virome may play a role in PC development and could serve as a promising target for PC diagnosis and therapeutic intervention. Future studies should further explore the underlying mechanisms of gut virus-bacteria interactions and validate the diagnostic models in larger and more diverse populations.

[Source](https://doi.org/10.1002/jmv.29809)

10.1002/jmv.29809

---

## The tumor microbiome as a predictor of outcomes in patients with metastatic melanoma treated with immune checkpoint inhibitors.
 17.07.2024, _CE Dravillas, SS Coleman, R Hoyd, G Caryotakis, L Denko, CHF Chan, ML Churchman, N Denko, RD Dodd, I Eljilany, S Hardikar, M Husain, AP Ikeguchi, N Jin, Q Ma, MD McCarter, AEG Osman, LA Robinson, EA Singer, G Tinoco, CM Ulrich, Y Zakharia, D Spakowicz, AA Tarhini, AC Tan_


Emerging evidence supports the important role of the tumor microbiome in oncogenesis, cancer immune phenotype, cancer progression, and treatment outcomes in many malignancies. In this study, we investigated the metastatic melanoma tumor microbiome and potential roles in association with clinical outcomes, such as survival, in patients with metastatic disease treated with immune checkpoint inhibitors (ICIs). Baseline tumor samples were collected from 71 patients with metastatic melanoma before treatment with ICIs. Bulk RNA-seq was conducted on the formalin-fixed paraffin-embedded (FFPE) and fresh frozen (FF) tumor samples. Durable clinical benefit (primary clinical endpoint) following ICIs was defined as overall survival &gt;24 months and no change to the primary drug regimen (responders). We processed RNA-seq reads to carefully identify exogenous sequences using the {exotic} tool. The 71 patients with metastatic melanoma ranged in age from 24 to 83 years, 59% were male, and 55% survived &gt;24 months following the initiation of ICI treatment. Exogenous taxa were identified in the tumor RNA-seq, including bacteria, fungi, and viruses. We found differences in gene expression and microbe abundances in immunotherapy responsive versus non-responsive tumors. Responders showed significant enrichment of bacteriophage in the phylum Uroviricota, and non-responders showed enrichment of several bacteria including Campylobacter jejuni. These microbes correlated with immune-related gene expression signatures. Finally, we found that models for predicting prolonged survival with immunotherapy using both microbe abundances and gene expression outperformed models using either dataset alone. Our findings warrant further investigation and potentially support therapeutic strategies to modify the tumor microbiome in order to improve treatment outcomes with ICIs.

[Source](https://doi.org/10.1158/2767-9764.CRC-23-0170)

10.1158/2767-9764.CRC-23-0170

---

## Gut microbiome diversity within Clostridia is negatively associated with human obesity.
 16.07.2024, _L Salazar-Jaramillo, J de la Cuesta-Zuluaga, LA Chica, M Cadavid, RE Ley, A Reyes, JS Escobar_


Clostridia are abundant in the human gut and comprise families associated with host health such as Oscillospiraceae, which has been correlated with leanness. However, culturing bacteria within this family is challenging, leading to their detection primarily through 16S rRNA amplicon sequencing, which has a limited ability to unravel diversity at low taxonomic levels, or by shotgun metagenomics, which is hindered by its high costs and complexity. In this cross-sectional study involving 114 Colombian adults, we used an amplicon-based sequencing strategy with alternative markers-gyrase subunit B (gyrB) and DNA K chaperone heat protein 70 (dnaK)-that evolve faster than the 16S rRNA gene. Comparing the diversity and abundance observed with the three markers in our cohort, we found a reduction in the diversity of Clostridia, particularly within Lachnospiraceae and Oscillospiraceae among obese individuals [as measured by the body mass index (BMI)]. Within Lachnospiraceae, the diversity of Ruminococcus_A negatively correlated with BMI. Within Oscillospiraceae, the genera CAG-170 and Vescimonas also exhibited this negative correlation. In addition, the abundance of Vescimonas was negatively correlated with BMI. Leveraging shotgun metagenomic data, we conducted a phylogenetic and genomic characterization of 120 metagenome-assembled genomes from Vescimonas obtained from a larger sample of the same cohort. We identified 17 of the 72 reported species. The functional annotation of these genomes showed the presence of multiple carbohydrate-active enzymes, particularly glycosyl transferases and glycoside hydrolases, suggesting potential beneficial roles in fiber degradation, carbohydrate metabolism, and butyrate production.

[Source](https://doi.org/10.1128/msystems.00627-24)

10.1128/msystems.00627-24

---

## Early depletion of gut microbiota shape oligodendrocyte response after traumatic brain injury.
 18.07.2024, _K Shumilov, A Ni, M Garcia-Bonilla, M Celorrio, SH Friess_


_Animals, Brain Injuries, Traumatic, Oligodendroglia, Gastrointestinal Microbiome, Mice, Mice, Inbred C57BL, Cell Proliferation, Male, T-Lymphocytes, Cells, Cultured_
White matter injury (WMI) is thought to be a major contributor to long-term cognitive dysfunctions after traumatic brain injury (TBI). This damage occurs partly due to apoptotic death of oligodendrocyte lineage cells (OLCs) after the injury, triggered directly by the trauma or in response to degenerating axons. Recent research suggests that the gut microbiota modulates the inflammatory response through the regulation of peripheral immune cell infiltration after TBI. Additionally, T-cells directly impact OLCs differentiation and proliferation. Therefore, we hypothesized that the gut microbiota plays a critical role in regulating the OLC response to WMI influencing T-cells differentiation and activation. Gut microbial depletion early after TBI chronically reduced re-myelination, acutely decreased OLCs proliferation, and was associated with increased myelin debris accumulation. Surprisingly, the absence of T-cells in gut microbiota depleted mice restored OLC proliferation and remyelination after TBI. OLCs co-cultured with T-cells derived from gut microbiota depleted mice resulted in impaired proliferation and increased expression of MHC-II compared with T cells from control-injured mice. Furthermore, MHC-II expression in OLCs appears to be linked to impaired proliferation under gut microbiota depletion and TBI conditions. Collectively our data indicates that depletion of the gut microbiota after TBI impaired remyelination, reduced OLCs proliferation with concomitantly increased OLC MHCII expression, and required the presence of T cells. This data suggests that T cells are an important mechanistic link by which the gut microbiota modulate the oligodendrocyte response and white matter recovery after TBI.

[Source](https://doi.org/10.1186/s12974-024-03158-9)

10.1186/s12974-024-03158-9

---

## Effect of progestin-based contraceptives on HIV-associated vaginal immune biomarkers and microbiome in adolescent girls.
 17.07.2024, _MA Nasr, A Aldous, J Daniels, C Joy, E Capozzi, M Yang, P Moriarty, V Emmanuel-Baker, S Malcolm, SJ Green, V Gomez-Lobo, M Ghosh_


_Humans, Female, Adolescent, Vagina, HIV Infections, Microbiota, Biomarkers, Medroxyprogesterone Acetate, Young Adult, Levonorgestrel, Desogestrel, Contraceptive Agents, Female, Longitudinal Studies, Progestins, Elafin_
Adolescent girls bear a disproportionate burden of both the HIV epidemic and unintended pregnancies; yet important questions remain unanswered regarding the effects of hormonal contraceptives on the vaginal immune microenvironment, which can impact HIV susceptibility in this group. Multiple studies report genital immune alterations associated with the progestin-based contraceptive Depot medroxyprogesterone acetate (DMPA) in adult women, but there is little available data in adolescents. The objective of this longitudinal cohort study was to evaluate the effects of short-term use of three progestin-based contraceptives, levonorgestrel intrauterine device (LNG-IUD), subdermal etonogestrel (ETNG), and injectable DMPA, on HIV-associated vaginal immune biomarkers and microbiome in adolescent girls. Fifty-nine sexually active, HIV-uninfected girls aged 15-19, were recruited from the Washington DC metro area and self-selected into Control (condoms only), combined oral contraceptive pills, LNG-IUD, ETNG and DMPA groups. Vaginal swabs were collected at baseline prior to contraceptive use and at 3-month follow-up visit. Vaginal secretions were tested for pro-inflammatory (IL-1α, IL-1β, TNF-α, IL-6, IL-8, MIP-3α, IP-10, RANTES, MIP-1α, MIP-1β) and anti-inflammatory/anti-HIV (Serpin-A1, Elafin, Beta-Defensin-2, SLPI) immune biomarkers using ELISA and for anti-HIV activity using TZM-bl assay. Vaginal microbiome was evaluated using 16S rRNA gene sequencing. Data were analyzed using SAS Version 9. Among the 34 participants who completed both visits, no significant changes in median biomarker concentrations, HIV inhibition and microbiome composition were observed between baseline and follow-up visits for any of the contraceptive groups. IL-8 (p&lt;0.01), MIP-3α (0.02), Elafin (p = 0.03) and RANTES (p&lt;0.01) differed significantly by race whereas IL-6 was significantly different by age (p = 0.03). We conclude that 3-month use of LNG-IUD, ETNG and DMPA have minimal effects on adolescent vaginal immune microenvironment, and therefore unlikely to impact HIV risk. Future studies with larger sample size and longer follow-up are recommended to continue to evaluate effects of contraceptives on the lower genital tract immunity and susceptibility to sexually transmitted infections.

[Source](https://doi.org/10.1371/journal.pone.0306237)

10.1371/journal.pone.0306237

---

## Microbiome and pancreatic cancer: time to think about chemotherapy.
 21.07.2024, _J de Castilhos, K Tillmanns, J Blessing, A Laraño, V Borisov, CK Stein-Thoeringer_


_Humans, Pancreatic Neoplasms, Gastrointestinal Microbiome, Carcinoma, Pancreatic Ductal, Tumor Microenvironment, Antineoplastic Agents, Animals, Bacteria, Precision Medicine, Drug Resistance, Neoplasm_
Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer characterized by late diagnosis, rapid progression, and a high mortality rate. Its complex biology, characterized by a dense, stromal tumor environment with an immunosuppressive milieu, contributes to resistance against standard treatments like chemotherapy and radiation. This comprehensive review explores the dynamic role of the microbiome in modulating chemotherapy efficacy and outcomes in PDAC. It delves into the microbiome's impact on drug metabolism and resistance, and the interaction between microbial elements, drugs, and human biology. We also highlight the significance of specific bacterial species and microbial enzymes in influencing drug action and the immune response in the tumor microenvironment. Cutting-edge methodologies, including artificial intelligence, low-biomass microbiome analysis and patient-derived organoid models, are discussed, offering insights into the nuanced interactions between microbes and cancer cells. The potential of microbiome-based interventions as adjuncts to conventional PDAC treatments are discussed, paving the way for personalized therapy approaches. This review synthesizes recent research to provide an in-depth understanding of how the microbiome affects chemotherapy efficacy. It focuses on elucidating key mechanisms and identifying existing knowledge gaps. Addressing these gaps is crucial for enhancing personalized medicine and refining cancer treatment strategies, ultimately improving patient outcomes.

[Source](https://doi.org/10.1080/19490976.2024.2374596)

10.1080/19490976.2024.2374596

---

## Host control of the microbiome: Mechanisms, evolution, and disease.
 19.07.2024, _J Wilde, E Slack, KR Foster_


_Animals, Humans, Biological Evolution, Homeostasis, Host Microbial Interactions, Microbiota, Selection, Genetic, Symbiosis_
Many species, including humans, host communities of symbiotic microbes. There is a vast literature on the ways these microbiomes affect hosts, but here we argue for an increased focus on how hosts affect their microbiomes. Hosts exert control over their symbionts through diverse mechanisms, including immunity, barrier function, physiological homeostasis, and transit. These mechanisms enable hosts to shape the ecology and evolution of microbiomes and generate natural selection for microbial traits that benefit the host. Our microbiomes result from a perpetual tension between host control and symbiont evolution, and we can leverage the host's evolved abilities to regulate the microbiota to prevent and treat disease. The study of host control will be central to our ability to both understand and manipulate microbiotas for better health.

[Source](https://doi.org/10.1126/science.adi3338)

10.1126/science.adi3338

---

## Connecting microbial community assembly and function.
 17.07.2024, _LS Bittleston_


Microbial ecology is moving away from purely descriptive analyses to experiments that can determine the underlying mechanisms driving changes in community assembly and function. More species-rich microbial communities generally have higher functional capabilities depending on if there is positive selection of certain species or complementarity among different species. When building synthetic communities or laboratory enrichment cultures, there are specific choices that can increase the number of species able to coexist. Higher resource complexity or the addition of physical niches are two of the many factors leading to greater biodiversity and associated increases in functional capabilities. We can use principles from community ecology and knowledge of microbial physiology to generate improved microbiomes for use in medicine, agriculture, or environmental management.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1369527424000882?via%3Dihub)

10.1016/j.mib.2024.102512

---

## Microbiota of Saliva: A Non-invasive Diagnostic Tool.
 17.07.2024, _DB Shinde, JG Mahore, PS Giram, SL Singh, A Sharda, D Choyan, S Musale_


Potential of salivary microbiota as a non-invasive diagnostic tool for various diseases are explained in the present review. Traditional diagnostic methods rely on blood, which has limitations in terms of collection and biomarker specificity. We discuss the concept of normal flora and how disruptions in oral microbiota can be indicative of diseases. Saliva, harboring a diverse microbial community, offers promise as a diagnostic biomarker source for oral and non-oral conditions. We delve into the role of microbial dysbiosis in disease pathogenesis and the prospects of using biological indicators like dysbiosis for diagnosis, prediction, and monitoring. This review also emphasizes the significance of saliva microbiota in advancing early disease detection and timely intervention. We addressed the following research question and objectives: Can the microbiota of saliva serve as a non-invasive diagnostic tool for the early detection and monitoring of both oral and non-oral diseases? To achieve this, we will explore the normal flora of microorganisms in the oral cavity, the impact of microbial dysbiosis, and the potential of using specific pathogenic microorganisms as biomarkers. Additionally, we will investigate the correlation between oral and non-oral diseases by analyzing total saliva or site-specific dental biofilms for signs of symbiosis or dysbiosis. This research seeks to contribute valuable insights into the development of a non-invasive diagnostic approach with broad applications in healthcare.

[Source](https://doi.org/10.1007/s12088-024-01219-4)

10.1007/s12088-024-01219-4

---

## The role of airways microbiota on local and systemic diseases: a rationale for probiotics delivery to the respiratory tract.
 23.07.2024, _S Glieca, E Quarta, B Bottari, VC Lal, F Sonvico, F Buttini_


Recent discoveries in the field of lung microbiota have enabled the investigation of new therapeutic interventions involving the use of inhaled probiotics.

[Source](https://doi.org/10.1038/s41522-024-00528-2)

10.1080/17425247.2024.2380334

---

## Topical prebiotic nitrate: optimizing the ‘hang-time’, source and dose for specific oral or systemic effects

[Source](https://doi.org/10.1038/s41522-024-00528-2)

---

## An improved diet-based nutritional interventions can improve childhood obesity with the synergistic regulation of gut microbiota.
 16.07.2024, _M Zhou, C Peng, Z Miao, K Wang, H Zhou, Y Li, G Xiao, X Wu_


Childhood obesity is a crucial public health concern worldwide. Dietary intervention is the most common intervention for the treatment of obesity. Therefore, we tested an improved diet-based nutritional interventions to improve the childhood obesity and its gut microbiota. Thirty obese children received a 12-week intervention with the adjust-energy-restricted dietary pattern (A-CRD). Body composition was measured by bioelectrical impedance (Inbody S10) and faecal microbes were profiled by sequencing 16S rRNA. Compared to the NTB group (at 0 week), the NTA group (at 12 weeks) had a significantly greater decrease in body weight, body mass index (BMI) and percent body fat (PBF) ( P &lt; 0.001, respectively), whereas skeletal muscle mass (SMM) and fat free mass (FFM) were not statistically significantly different ( P &gt; 0.05). The gut microbiota was found significantly different between the NTB and NTA groups based on alpha and beta diversity. Bifidobacterium, Blautia, and Streptococcus was significantly increased, whereas Bacteroides and Megamonas was significantly decreased in the NTA group ( P &lt; 0.05, respectively). Meanwhile, NTA group significantly increased the ability to produce short-chain fatty acids (SCFAs; e.g. acetic acid/total dietary energy) and changed he predictive metabolic functional features of the microbiota communities ( P &lt; 0.05, respectively) than the NTB group. In conclusion, A-CRD can significantly improve childhood obesity, and the underlying mechanism may be its effect on gut microbiota and metabolism. Therefore, the diet-based nutrition intervention targeting gut microbiota will be more effective management of body weight and prevention of obesity. Chinese Clinical Trial Register: ChiCTR2300074571.

[Source](https://brill.com/view/journals/bm/aop/article-10.1163-18762891-bja00019/article-10.1163-18762891-bja00019.xml)

10.1163/18762891-bja00019

---

## Intermittent fasting, calorie restriction, and a ketogenic diet improve mitochondrial function by reducing lipopolysaccharide signaling in monocytes during obesity: A randomized clinical trial

[Source](https://www.clinicalnutritionjournal.com/article/S0261-5614(24)00228-0/fulltext)

---

## Influence of the initial microbiota on eggplant <em>shibazuke </em>pickle and eggplant juice fermentation

[Source](https://doi.org/10.1128/spectrum.00464-24)

---

## Metatranscriptomics reveals microbial community function succession and characteristic flavor formation mechanisms during black rice wine fermentation.
 18.07.2024, _A Tang, B Peng_


Black rice wine (BRW) is a traditional Chinese rice wine with unique flavors; however, the formation pathways of flavor compounds driven by microbiota remain unclear. This study employed HPLC and GC-MS to reveal that during BRW fermentation, free amino acids increased sevenfold, volatile compounds doubled, and 28 key characteristic flavor compounds were identified. Metatranscriptomic analysis indicated that during fermentation, driven by physicochemical factors and microbial interactions, Saccharomyces gradually became the dominant active microorganism (relative abundance 87.01%-97.70%). Other dominant microorganisms (relative abundance &gt;0.1%), including Saccharomycopsis, Pediococcus, Wickerhamomyces, and Weissella, significantly decreased. Meanwhile, the microflora's signature functions underwent succession: transcription early, carbohydrate metabolism mid-stage, and autophagy late. These microbial and functional successions facilitated the accumulation of flavor compounds. Metabolic network reconstruction revealed that Saccharomyces was pivotal in substrate degradation and flavor formation, while other dominant microorganisms actively promoted these processes. This study provides insights into regulating BRW's flavor through microorganisms.

[Source](https://doi.org/10.1016/j.foodchem.2024.140428)

10.1016/j.foodchem.2024.140428

---

## Gα13 restricts nutrient driven proliferation in mucosal germinal centers.
 19.07.2024, _HT Nguyen, M Li, R Vadakath, KA Henke, TC Tran, H Li, M Yamadi, S Darbha, Y Yang, J Kabat, AR Albright, EG Centeno, JD Phelan, S Roulland, DW Huang, MC Kelly, RM Young, S Pittaluga, S Difilippantonio, JR Muppidi_


Germinal centers (GCs) that form in mucosal sites are exposed to gut-derived factors that have the potential to influence homeostasis independent of antigen receptor-driven selective processes. The G-protein Gα13 confines B cells to the GC and limits the development of GC-derived lymphoma. We discovered that Gα13-deficiency fuels the GC reaction via increased mTORC1 signaling and Myc protein expression specifically in the mesenteric lymph node (mLN). The competitive advantage of Gα13-deficient GC B cells (GCBs) in mLN was not dependent on T cell help or gut microbiota. Instead, Gα13-deficient GCBs were selectively dependent on dietary nutrients likely due to greater access to gut lymphatics. Specifically, we found that diet-derived glutamine supported proliferation and Myc expression in Gα13-deficient GCBs in the mLN. Thus, GC confinement limits the effects of dietary glutamine on GC dynamics in mucosal tissues. Gα13 pathway mutations coopt these processes to promote the gut tropism of aggressive lymphoma.

[Source](https://doi.org/10.1038/s41590-024-01910-0)

10.1038/s41590-024-01910-0

---

## Gut microbiota mediates ambient PM2.5 exposure-induced abnormal glucose metabolism via short-chain fatty acids.
 12.07.2024, _W Shao, B Pan, Z Li, R Peng, W Yang, Y Xie, D Han, X Fang, J Li, Y Zhu, Z Zhao, H Kan, Z Ying, Y Xu_


PM2.5 exposure has been found to cause gut dysbiosis and impair glucose homeostasis in human and animals, yet their underlying biological connection remain unclear. In the present study, we aim to investigate the biological significance of gut microbiota in PM2.5-induced glucose metabolic abnormalities. Our results showed that microbiota depletion by antibiotics treatment significantly alleviated PM2.5-induced glucose intolerance and insulin resistance, as indicated by the intraperitoneal glucose tolerance test, glucose-induced insulin secretion, insulin tolerance test, insulin-induced phosphorylation levels of Akt and GSK-3β in insulin sensitive tissues. In addition, faecal microbiota transplantation (FMT) from PM2.5-exposed donor mice successfully remodeled the glucose metabolism abnormalities in recipient mice, while the transplantation of autoclaved faecal materials did not. Faecal microbiota analysis demonstrated that the composition and alpha diversity of the gut bacterial community were altered by PM2.5 exposure and in FMT recipient mice. Furthermore, short-chain fatty acids levels analysis showed that the circulating acetate was significantly decreased in PM2.5-exposed donor and FMT recipient mice, and supplementation of sodium acetate for 3 months successfully improved the glucose metabolism abnormalities induced by PM2.5 exposure. These results indicate that manipulating gut microbiota or its metabolites could be a potential strategy for preventing the adverse health effects of ambient PM2.5.

[Source](https://doi.org/10.1016/j.jhazmat.2024.135096)

10.1016/j.jhazmat.2024.135096

---

## Domestication and evolutionary histories of specialized gut symbionts across cephalotine ants.
 22.07.2024, _C Cabuslay, JT Wertz, B Béchade, Y Hu, S Braganza, D Freeman, S Pradhan, M Mukhanova, S Powell, C Moreau, JA Russell_


_Animals, Symbiosis, Ants, Phylogeny, RNA, Ribosomal, 16S, Domestication, Gastrointestinal Microbiome, Bacteria, Biological Evolution_
The evolution of animals and their gut symbionts is a complex phenomenon, obscured by lability and diversity. In social organisms, transmission of symbionts among relatives may yield systems with more stable associations. Here, we study the history of a social insect symbiosis involving cephalotine ants and their extracellular gut bacteria, which come predominantly from host-specialized lineages. We perform multi-locus phylogenetics for symbionts from nine bacterial orders, and map prior amplicon sequence data to lineage-assigned symbiont genomes, studying distributions of rigorously defined symbionts across 20 host species. Based on monophyly and additional hypothesis testing, we estimate that these specialized gut bacteria belong to 18 distinct lineages, of which 15 have been successfully isolated and cultured. Several symbiont lineages showed evidence for domestication events that occurred later in cephalotine evolutionary history, and only one lineage was ubiquitously detected in all 20 host species and 48 colonies sampled with amplicon 16S rRNA sequencing. We found evidence for phylogenetically constrained distributions in four symbionts, suggesting historical or genetic impacts on community composition. Two lineages showed evidence for frequent intra-lineage co-infections, highlighting the potential for niche divergence after initial domestication. Nearly all symbionts showed evidence for occasional host switching, but four may, more often, co-diversify with their hosts. Through our further assessment of symbiont localization and genomic functional profiles, we demonstrate distinct niches for symbionts with shared evolutionary histories, prompting further questions on the forces underlying the evolution of hosts and their gut microbiomes.

[Source](https://doi.org/10.1111/mec.17454)

10.1111/mec.17454

---

## The interplay between gut symbionts and behavioral variation in social insects.
 17.07.2024, _J Liberti, P Engel, A Cabirol_


Social insects exhibit a high degree of intraspecific behavioral variation. Moreover, they often harbor specialized microbial communities in their gut. Recent studies suggest that these two characteristics of social insects are interlinked: insect behavioral phenotypes affect their gut microbiota composition, partly through exposure to different environments and diet, and in return, the gut microbiota has been shown to influence insect behavior. Here we discuss the bidirectional relationship existing between intraspecific variation in gut microbiota composition and behavioral phenotypes in social insects.

[Source](https://doi.org/10.1016/j.cois.2024.101233)

10.1016/j.cois.2024.101233

---

## Meta-transcriptomic analysis of companion animal infectomes reveals their diversity and potential roles in animal and human disease.
 16.07.2024, _W-C Wu, Y-F Pan, W-D Zhou, Y-Q Liao, M-W Peng, G-Y Luo, G-Y Xin, Y-N Peng, T An, B Li, H Luo, VR Barrs, JA Beatty, EC Holmes, W Zhao, M Shi, Y Shu_


Companion animals such as cats and dogs harbor diverse microbial communities that can potentially impact human health due to close and frequent contact. To better characterize their total infectomes and assess zoonotic risks, we characterized the overall infectomes of companion animals (cats and dogs) and evaluated their potential zoonotic risks. Meta-transcriptomic analyses were performed on 239 samples from cats and dogs collected across China, identifying 24 viral species, 270 bacterial genera, and two fungal genera. Differences in the overall microbiome and infectome composition were compared across different animal species (cats or dogs), sampling sites (rectal or oropharyngeal), and health status (healthy or diseased). Diversity analyses revealed that viral abundance was generally higher in diseased animals compared to healthy ones, while differences in microbial composition were mainly driven by sampling site, followed by animal species and health status. Disease association analyses validated the pathogenicity of known pathogens and suggested potential pathogenic roles of previously undescribed bacteria and newly discovered viruses. Cross-species transmission analyses identified seven pathogens shared between cats and dogs, such as alphacoronavirus 1, which was detected in both oropharyngeal and rectal swabs albeit with differential pathogenicity. Further analyses showed that some viruses, like alphacoronavirus 1, harbored multiple lineages exhibiting distinct pathogenicity, tissue, or host preferences. Ultimately, a systematic evolutionary screening identified 27 potential zoonotic pathogens in this sample set, with far more bacterial than viral species, implying potential health threats to humans. Overall, our meta-transcriptomic analysis reveals a landscape of actively transcribing microorganisms in major companion animals, highlighting key pathogens, those with the potential for cross-species transmission, and possible zoonotic threats.

[Source](https://doi.org/10.1128/msphere.00439-24)

10.1128/msphere.00439-24

---

## Alcohol consumption during pregnancy differentially affects the fecal microbiota of dams and offspring.
 15.07.2024, _TS Bodnar, G Ainsworth-Cruickshank, V Billy, L Wegener Parfrey, J Weinberg, C Raineki_


_Animals, Pregnancy, Female, Feces, Alcohol Drinking, Prenatal Exposure Delayed Effects, Rats, Fetal Alcohol Spectrum Disorders, Gastrointestinal Microbiome, Ethanol, Male, Disease Models, Animal, Microbiota, Bacteria_
Microbiota imbalances are linked to inflammation and disease, as well as neurodevelopmental conditions where they may contribute to behavioral, physiological, and central nervous system dysfunction. By contrast, the role of the microbiota in Fetal Alcohol Spectrum Disorder (FASD), the group of neurodevelopmental conditions that can occur following prenatal alcohol exposure (PAE), has not received similar attention. Here we utilized a rodent model of alcohol consumption during pregnancy to characterize the impact of alcohol on the microbiota of dam-offspring dyads. Overall, bacterial diversity decreased in alcohol-consuming dams and community composition differed from that of controls in alcohol-consuming dams and their offspring. Bacterial taxa and predicted biochemical pathway composition were also altered with alcohol consumption/exposure; however, there was minimal overlap between the changes in dams and offspring. These findings illuminate the potential importance of the microbiota in the pathophysiology of FASD and support investigation into novel microbiota-based interventions.

[Source](https://doi.org/10.1038/s41598-024-64313-z)

10.1038/s41598-024-64313-z

---

## Meta-omics analysis reveals the marine arsenic cycle driven by bacteria.
 18.07.2024, _ZJ Teng, J Li, P Wang, CY Li, M Peng, QL Qin, XL Chen, Y Chen, HH Fu, N Wang, YZ Zhang_


Arsenic is a toxic element widely distributed in the Earth's crust and ranked as a class I human carcinogen. Microbial metabolism makes significant contributions to arsenic detoxification, migration and transformation. Nowadays, research on arsenic is primarily in areas affected by arsenic pollution associated with human health activities. However, the biogeochemical traits of arsenic in the global marine ecosystem remain to be explicated. In this study, we revealed that seawater environments were primarily governed by the process of arsenate reduction to arsenite, while arsenite methylation was predominant in marine sediments which may serve as significant sources of arsenic emission into the atmosphere. Significant disparities existed in the distribution patterns of the arsenic cycle between surface and deep seawaters at middle and low latitudes, whereas these situations tend to be similar in the Arctic and Antarctic oceans. Significant variations were also observed in the taxonomic diversity and core microbial community of arsenic cycling across different marine environments. Specifically, γ-proteobacteria played a pivotal role in the arsenic cycle in the whole marine environment. Temperature, dissolved oxygen and phosphate were the crucial factors that related to these differentiations in seawater environments. Overall, our study contributes to a deeper understanding of the marine arsenic cycle.

[Source](https://doi.org/10.1016/j.jhazmat.2024.135137)

10.1016/j.jhazmat.2024.135137

---

## Using Machine Learning to Identify Environmental Factors that Collectively Determine Microbial Community Structure of Activated Sludge.
 18.07.2024, _L Wang, W Lu, Y Song, S Liu, YU Vincent Fu_


Activated sludge (AS) microbial communities are influenced by various environmental variables. However, a comprehensive analysis of how these variables jointly and nonlinearly shape the AS microbial community remains challenging. In this study, we employed advanced machine learning techniques to elucidate the collective effects of environmental variables on the structure and function of AS microbial communities. Applying Dirichlet Multinomial Mixtures analysis to 311 global AS samples, we identified four distinct microbial community types (AS-types), each characterized by unique microbial compositions and metabolic profiles. We used 14 classical linear and nonlinear machine learning methods to select a baseline model. The Extremely Randomized Trees demonstrated optimal performance in learning the relationship between environmental factors and AS types (with an accuracy of 71.43%). Feature selection identified critical environmental factors and their importance rankings, including latitude (Lat), longitude (Long), precipitation during sampling (Precip), solids retention time (SRT), effluent total nitrogen (Effluent TN), average temperature during sampling month (Avg Temp), mixed liquor temperature (Mixed Temp), influent biochemical oxygen demand (Influent BOD), and annual precipitation (Annual Precip). Significantly, Lat, Long, Precip, Avg Temp, and Annual Precip, influenced metabolic variations among AS types. These findings emphasize the pivotal role of environmental variables in shaping microbial community structures and enhancing metabolic pathways within activated sludge. Our study encourages the application of machine learning techniques to design artificial activated sludge microbial communities for specific environmental purposes.

[Source](https://doi.org/10.1016/j.envres.2024.119635)

10.1016/j.envres.2024.119635

---

## Microplastic pollution promotes soil respiration: A global-scale meta-analysis.
 16.07.2024, _S Zhao, MC Rillig, H Bing, Q Cui, T Qiu, Y Cui, J Penuelas, B Liu, S Bian, FA Monikh, J Chen, L Fang_


_Soil Microbiology, Microplastics, Soil, Carbon Dioxide, Soil Pollutants, Microbiota, Biomass, Carbon_
Microplastic (MP) pollution likely affects global soil carbon (C) dynamics, yet it remains uncertain how and to what extent MP influences soil respiration. Here, we report on a global meta-analysis to determine the effects of MP pollution on the soil microbiome and CO2 emission. We found that MP pollution significantly increased the contents of soil organic C (SOC) (21%) and dissolved organic C (DOC) (12%), the activity of fluorescein diacetate hydrolase (FDAse) (10%), and microbial biomass (17%), but led to a decrease in microbial diversity (3%). In particular, increases in soil C components and microbial biomass further promote CO2 emission (25%) from soil, but with a much higher effect of MPs on these emissions than on soil C components and microbial biomass. The effect could be attributed to the opposite effects of MPs on microbial biomass vs. diversity, as soil MP accumulation recruited some functionally important bacteria and provided additional C substrates for specific heterotrophic microorganisms, while inhibiting the growth of autotrophic taxa (e.g., Chloroflexi, Cyanobacteria). This study reveals that MP pollution can increase soil CO2 emission by causing shifts in the soil microbiome. These results underscore the potential importance of plastic pollution for terrestrial C fluxes, and thus climate feedbacks.

[Source](https://doi.org/10.1111/gcb.17415)

10.1111/gcb.17415

---

## SMMP: A Deep-Coverage Marine Metaproteome Method for Microbial Community Analysis throughout the Water Column Using 1 L of Seawater.
 23.07.2024, _S Wang, Z Zhang, K Yang, J Zhao, W Zhang, Z Wang, Z Liang, Y Zhang, Y Zhang, J Liu, L Zhang_


_Seawater, Proteomics, Microbiota, Proteome, Methane, Bacteria, Oxidation-Reduction, Carbon Monoxide_
Marine microbes drive pivotal transformations in planetary-scale elemental cycles and have crucial impacts on global biogeochemical processes. Metaproteomics is a powerful tool for assessing the metabolic diversity and function of marine microbes. However, hundreds of liters of seawater are required for normal metaproteomic analysis due to the sparsity of microbial populations in seawater, which poses a substantial challenge to the widespread application of marine metaproteomics, particularly for deep seawater. Herein, a sensitive marine metaproteomics workflow, named sensitive marine metaproteome analysis (SMMP), was developed by integrating polycarbonate filter-assisted microbial enrichment, solid-phase alkylation-based anti-interference sample preparation, and narrow-bore nanoLC column for trace peptide separation and characterization. The method provided more than 8500 proteins from 1 L of bathypelagic seawater samples, which covered diverse microorganisms and crucial functions, e.g., the detection of key enzymes associated with the Wood-Ljungdahl pathway. Then, we applied SMMP to investigate vertical variations in the metabolic expression patterns of marine microorganisms from the euphotic zone to the bathypelagic zone. Methane oxidation and carbon monoxide (CO) oxidation were active processes, especially in the bathypelagic zone, which provided a remarkable energy supply for the growth and proliferation of heterotrophic microorganisms. In addition, marker protein profiles detected related to ammonia transport, ammonia oxidation, and carbon fixation highlighted that Thaumarchaeota played a critical role in primary production based on the coupled carbon-nitrogen process, contributing to the storage of carbon and nitrogen in the bathypelagic regions. SMMP has low microbial input requirements and yields in-depth metaproteome analysis, making it a prospective approach for comprehensive marine metaproteomic investigations.

[Source](https://doi.org/10.1021/acs.analchem.4c02079)

10.1021/acs.analchem.4c02079

---

## Metagenomic insights into microbial community structure and metabolism in alpine permafrost on the Tibetan Plateau.
 17.07.2024, _L Kang, Y Song, R Mackelprang, D Zhang, S Qin, L Chen, L Wu, Y Peng, Y Yang_


_Permafrost, Tibet, Soil Microbiology, RNA, Ribosomal, 16S, Metagenomics, Microbiota, Bacteria, Soil, Metagenome, Ecosystem, Climate Change, Biodiversity, Phylogeny_
Permafrost, characterized by its frozen soil, serves as a unique habitat for diverse microorganisms. Understanding these microbial communities is crucial for predicting the response of permafrost ecosystems to climate change. However, large-scale evidence regarding stratigraphic variations in microbial profiles remains limited. Here, we analyze microbial community structure and functional potential based on 16S rRNA gene amplicon sequencing and metagenomic data obtained from an ∼1000 km permafrost transect on the Tibetan Plateau. We find that microbial alpha diversity declines but beta diversity increases down the soil profile. Microbial assemblages are primarily governed by dispersal limitation and drift, with the importance of drift decreasing but that of dispersal limitation increasing with soil depth. Moreover, genes related to reduction reactions (e.g., ferric iron reduction, dissimilatory nitrate reduction, and denitrification) are enriched in the subsurface and permafrost layers. In addition, microbial groups involved in alternative electron accepting processes are more diverse and contribute highly to community-level metabolic profiles in the subsurface and permafrost layers, likely reflecting the lower redox potential and more complicated trophic strategies for microorganisms in deeper soils. Overall, these findings provide comprehensive insights into large-scale stratigraphic profiles of microbial community structure and functional potentials in permafrost regions.

[Source](https://doi.org/10.1038/s41467-024-50276-2)

10.1038/s41467-024-50276-2

---

## Top-down and bottom-up microbiome engineering approaches to enable biomanufacturing from waste biomass.
 13.07.2024, _X Lyu, M Nuhu, P Candry, J Wolfanger, M Betenbaugh, A Saldivar, C Zuniga, Y Wang, S Shrestha_


Growing environmental concerns and the need to adopt a circular economy have highlighted the importance of waste valorization for resource recovery. Microbial consortia-enabled biotechnologies have made significant developments in the biomanufacturing of valuable resources from waste biomass that serve as suitable alternatives to petrochemical-derived products. These microbial consortia are designed following a top-down or bottom-up engineering approach. The top-down approach is a classical method that uses environmental variables to selectively steer an existing microbial consortium to achieve a target function. While high-throughput sequencing has enabled microbial community characterization, the major challenge is to disentangle complex microbial interactions and manipulate the structure and function accordingly. Microbial consortia design through a bottom-up approach uses prior knowledge of the metabolic pathway and possible interactions among consortium partners to design and engineer synthetic microbial consortia. This strategy offers some control over the composition and function of the consortium for targeted bioprocesses, but challenges remain in optimal assembly methods and long-term stability. In this review, we present the recent advancements, challenges, and opportunities for further improvement using top-down and bottom-up approaches for microbiome engineering. As the bottom-up approach is relatively a new concept for waste valorization, this review explores the assembly and design of synthetic microbial consortia, ecological engineering principles to optimize microbial consortia, and metabolic engineering approaches for efficient conversion. Integration of top-down and bottom-up approaches along with developments in metabolic modeling to predict and optimize consortia function are also highlighted.

[Source](https://doi.org/10.1093/jimb/kuae025)

10.1093/jimb/kuae025

---

## GSearch: ultra-fast and scalable genome search by combining K-mer hashing with hierarchical navigable small world graphs.
 16.07.2024, _J Zhao, JP Both, LM Rodriguez-R, KT Konstantinidis_


Genome search and/or classification typically involves finding the best-match database (reference) genomes and has become increasingly challenging due to the growing number of available database genomes and the fact that traditional methods do not scale well with large databases. By combining k-mer hashing-based probabilistic data structures (i.e. ProbMinHash, SuperMinHash, Densified MinHash and SetSketch) to estimate genomic distance, with a graph based nearest neighbor search algorithm (Hierarchical Navigable Small World Graphs, or HNSW), we created a new data structure and developed an associated computer program, GSearch, that is orders of magnitude faster than alternative tools while maintaining high accuracy and low memory usage. For example, GSearch can search 8000 query genomes against all available microbial or viral genomes for their best matches (n = ∼318 000 or ∼3 000 000, respectively) within a few minutes on a personal laptop, using ∼6 GB of memory (2.5 GB via SetSketch). Notably, GSearch has an O(log(N)) time complexity and will scale well with billions of genomes based on a database splitting strategy. Further, GSearch implements a three-step search strategy depending on the degree of novelty of the query genomes to maximize specificity and sensitivity. Therefore, GSearch solves a major bottleneck of microbiome studies that require genome search and/or classification.

[Source](https://doi.org/10.1093/nar/gkae609)

10.1093/nar/gkae609

---

## Multi-omics machine learning to study host-microbiome interactions in early-onset colorectal cancer.
 20.07.2024, _TT Jayakrishnan, N Sangwan, SV Barot, N Farha, A Mariam, S Xiang, F Aucejo, M Conces, KG Nair, SS Krishnamurthi, SL Schmit, D Liska, DM Rotroff, AA Khorana, SD Kamath_


The incidence of early-onset colorectal cancer (eoCRC) is rising, and its pathogenesis is not completely understood. We hypothesized that machine learning utilizing paired tissue microbiome and plasma metabolome features could uncover distinct host-microbiome associations between eoCRC and average-onset CRC (aoCRC). Individuals with stages I-IV CRC (n = 64) were categorized as eoCRC (age ≤ 50, n = 20) or aoCRC (age ≥ 60, n = 44). Untargeted plasma metabolomics and 16S rRNA amplicon sequencing (microbiome analysis) of tumor tissue were performed. We fit DIABLO (Data Integration Analysis for Biomarker Discovery using Latent variable approaches for Omics studies) to construct a supervised machine-learning classifier using paired multi-omics (microbiome and metabolomics) data and identify associations unique to eoCRC. A differential association network analysis was also performed. Distinct clustering patterns emerged in multi-omic dimension reduction analysis. The metabolomics classifier achieved an AUC of 0.98, compared to AUC 0.61 for microbiome-based classifier. Circular correlation technique highlighted several key associations. Metabolites glycerol and pseudouridine (higher abundance in individuals with aoCRC) had negative correlations with Parasutterella, and Ruminococcaceae (higher abundance in individuals with eoCRC). Cholesterol and xylitol correlated negatively with Erysipelatoclostridium and Eubacterium, and showed a positive correlation with Acidovorax with higher abundance in individuals with eoCRC. Network analysis revealed different clustering patterns and associations for several metabolites e.g.: urea cycle metabolites and microbes such as Akkermansia. We show that multi-omics analysis can be utilized to study host-microbiome correlations in eoCRC and demonstrates promising biomarker potential of a metabolomics classifier. The distinct host-microbiome correlations for urea cycle in eoCRC may offer opportunities for therapeutic interventions.

[Source](https://doi.org/10.1038/s41698-024-00647-1)

10.1038/s41698-024-00647-1

---

## 3D printed rectal swabs for assessing the gut microbiome, metabolome and inflammation.
 21.07.2024, _RW Perry, BH Mullish, JL Alexander, R Shah, NP Danckert, JM Blanco, L Roberts, Z Liu, D Chrysostomou, ST Radhakrishnan, S Balarajah, R Barry, LC Hicks, HRT Williams, JR Marchesi_


_Humans, Gastrointestinal Microbiome, Feces, Metabolome, Inflammatory Bowel Diseases, Printing, Three-Dimensional, RNA, Ribosomal, 16S, Male, Female, Adult, Rectum, Leukocyte L1 Antigen Complex, Inflammation, Middle Aged, Specimen Handling_
Investigating the gut microbiome and metabolome frequently requires faecal samples, which can be difficult to obtain. Previous studies have shown that rectal swabs are comparable to faecal samples for analysing gut microbiota composition and key metabolites. In this study, 3D printed rectal swabs were compared with conventional flocked swabs and faecal samples, due to the potential advantages 3D printing as a technique offers for swab production and development. 16S rRNA gene sequencing, qPCR and metabolite profiling (using 1H-NMR spectroscopy) were performed on swab and faecal samples from healthy participants. Faecal calprotectin and total protein analysis were performed on samples from inflammatory bowel disease (IBD) patients. There were no significant differences between both swab types and faecal samples when assessing key measures of alpha and beta diversity, and differences in the abundance of major phyla. There was a strong correlation between both swab types and faecal samples for all combined metabolites detected by NMR. In IBD patients, there was no significant difference in faecal calprotectin and total protein levels between both swab types and faecal samples. These data lead us to conclude that 3D printed swabs are equivalent to flocked swabs for the analysis of the gut microbiome, metabolome and inflammation.

[Source](https://doi.org/10.1038/s41598-024-67457-0)

10.1038/s41598-024-67457-0

---

## AFITbin: a metagenomic contig binning method using aggregate l-mer frequency based on initial and terminal nucleotides.
 19.07.2024, _A Darabi, S Sobhani, R Aghdam, C Eslahchi_


_Metagenomics, Algorithms, Nucleotides, High-Throughput Nucleotide Sequencing, Software, Microbiota, Sequence Analysis, DNA, Cluster Analysis, Contig Mapping, Metagenome_
Using next-generation sequencing technologies, scientists can sequence complex microbial communities directly from the environment. Significant insights into the structure, diversity, and ecology of microbial communities have resulted from the study of metagenomics. The assembly of reads into longer contigs, which are then binned into groups of contigs that correspond to different species in the metagenomic sample, is a crucial step in the analysis of metagenomics. It is necessary to organize these contigs into operational taxonomic units (OTUs) for further taxonomic profiling and functional analysis. For binning, which is synonymous with the clustering of OTUs, the tetra-nucleotide frequency (TNF) is typically utilized as a compositional feature for each OTU.

[Source](https://doi.org/10.1186/s12859-024-05859-7)

10.1186/s12859-024-05859-7

---

## JASPER: Fast, powerful, multitrait association testing in structured samples gives insight on pleiotropy in gene expression.
 18.07.2024, _J Mbatchou, MS McPeek_


Joint association analysis of multiple traits with multiple genetic variants can provide insight into genetic architecture and pleiotropy, improve trait prediction, and increase power for detecting association. Furthermore, some traits are naturally high-dimensional, e.g., images, networks, or longitudinally measured traits. Assessing significance for multitrait genetic association can be challenging, especially when the sample has population sub-structure and/or related individuals. Failure to adequately adjust for sample structure can lead to power loss and inflated type 1 error, and commonly used methods for assessing significance can work poorly with a large number of traits or be computationally slow. We developed JASPER, a fast, powerful, robust method for assessing significance of multitrait association with a set of genetic variants, in samples that have population sub-structure, admixture, and/or relatedness. In simulations, JASPER has higher power, better type 1 error control, and faster computation than existing methods, with the power and speed advantage of JASPER increasing with the number of traits. JASPER is potentially applicable to a wide range of association testing applications, including for multiple disease traits, expression traits, image-derived traits, and microbiome abundances. It allows for covariates, ascertainment, and rare variants and is robust to phenotype model misspecification. We apply JASPER to analyze gene expression in the Framingham Heart Study, where, compared to alternative approaches, JASPER finds more significant associations, including several that indicate pleiotropic effects, most of which replicate previous results, while others have not previously been reported. Our results demonstrate the promise of JASPER for powerful multitrait analysis in structured samples.

[Source](https://www.cell.com/ajhg/abstract/S0002-9297(24)00216-7)

10.1016/j.ajhg.2024.06.010

---

## The nature of the last universal common ancestor and its impact on the early Earth system.
 12.07.2024, _ERR Moody, S Álvarez-Carretero, TA Mahendrarajah, JW Clark, HC Betts, N Dombrowski, LL Szánthó, RA Boyle, S Daines, X Chen, N Lane, Z Yang, GA Shields, GJ Szöllősi, A Spang, D Pisani, TA Williams, TM Lenton, PCJ Donoghue_


The nature of the last universal common ancestor (LUCA), its age and its impact on the Earth system have been the subject of vigorous debate across diverse disciplines, often based on disparate data and methods. Age estimates for LUCA are usually based on the fossil record, varying with every reinterpretation. The nature of LUCA's metabolism has proven equally contentious, with some attributing all core metabolisms to LUCA, whereas others reconstruct a simpler life form dependent on geochemistry. Here we infer that LUCA lived ~4.2 Ga (4.09-4.33 Ga) through divergence time analysis of pre-LUCA gene duplicates, calibrated using microbial fossils and isotope records under a new cross-bracing implementation. Phylogenetic reconciliation suggests that LUCA had a genome of at least 2.5 Mb (2.49-2.99 Mb), encoding around 2,600 proteins, comparable to modern prokaryotes. Our results suggest LUCA was a prokaryote-grade anaerobic acetogen that possessed an early immune system. Although LUCA is sometimes perceived as living in isolation, we infer LUCA to have been part of an established ecological system. The metabolism of LUCA would have provided a niche for other microbial community members and hydrogen recycling by atmospheric photochemistry could have supported a modestly productive early ecosystem.

[Source](https://doi.org/10.1038/s41559-024-02461-1)

10.1038/s41559-024-02461-1

---

