## Altered gut microbiota as potential biomarker biomarkers for autism spectrum disorder in early childhood.
 04.06.2023, _Y Zhao, Y Wang, F Meng, X Chen, T Chang, H Huang, F He, Y Zheng_


Gastrointestinal (GI) disorders are widely recorded in autism spectrum disorder (ASD), and ASD with GI symptoms is a vital subtype of this disease. Growing evidence suggests altered gut microbiota biomarkers in ASD, but little is known about the gut microbiota of individuals with ASD with GI Symptoms, particularly in early childhood. In our study, the gut microbiota of 36 individuals with ASD along with GI symptoms and 40 typically developing (TD) children were compared using 16S rRNA gene sequencing. The microbial diversity and composition were found to differ between the two groups. Compared to TD, the gut microbiota of ASD patients with GI symptoms exhibited decreased alpha diversity and depletion of butyrate-producing bacteria (e.g., Faecalibacterium and Coprococcus). In addition, microbial functional analysis showed abnormality in several gut metabolic models and gut brain models of ASD with GI symptoms, including SCFAshort-chain fatty acid (SCFA) synthesis/degradation and neurotoxin-related p-cresol degradation, which are closely associated with ASD-related behaviors in animal models. Furthermore, we constructed a Support Vector Machine classification model, which robustly discriminated individuals with ASD and GI symptoms from TD individuals in a validation set (AUC = 0.88). Our findings provide a deep insight into the roles of the disturbed gut ecosystem in individuals with ASD and GI symptoms aged 3-6 years. Our classification model supports gut microbiota as a potential biomarker for the early identification of ASD and interventions targeting particular gut-beneficial microbiota.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0306452223002026?via%3Dihub)

10.1016/j.neuroscience.2023.04.029

---

## The composition of environmental microbiota in three tree fruit packing facilities changed over seasons and contained taxa indicative of <em>L. monocytogenes</em> contamination

[Source](https://doi.org/10.1186/s40168-023-01544-8)

---

## Succession and determinants of the early life nasopharyngeal microbiota in a South African birth cohort

[Source](https://doi.org/10.1186/s40168-023-01563-5)

---

## Secreted <em>Akkermansia muciniphila</em> threonyl-tRNA synthetase functions to monitor and modulate immune homeostasis

[Source](https://doi.org/10.1016/j.chom.2023.05.007)

---

## Gut Microbiome Composition is associated with future onset of Crohn’s Disease in Healthy First-Degree Relatives

[Source](https://www.gastrojournal.org/article/S0016-5085(23)00805-3/pdf)

---

## Metagenome Investigation of Ocular Microbiota of Cataract Patients With and Without Type 2 Diabetes

[Source](https://doi.org/10.1167/tvst.12.6.1)

---

## Longitudinal Gut Microbiota Dysbiosis Underlies Olanzapine-Induced Weight Gain

[Source](https://doi.org/10.1128/spectrum.00058-23)

---

## Metagenome-mining indicates an association between bacteriocin presence and strain diversity in the infant gut

[Source](https://doi.org/10.1186/s12864-023-09388-0)

---

## The Microbiome in Advanced Melanoma: Where Are We Now?

[Source](https://doi.org/10.1007/s11912-023-01431-3)

---

## Human gut microbiome: A primer for the clinician

[Source](https://doi.org/10.1002/jgh3.12902)

---

## Role of gut microbiota and bacterial metabolites in mucins of colorectal cancer

[Source](https://doi.org/10.3389/fcimb.2023.1119992)

---

## Bacteriophage targeting microbiota alleviates non-alcoholic fatty liver disease induced by high alcohol-producing <em>Klebsiella pneumoniae</em>

[Source](https://www.nature.com/articles/s41467-023-39028-w)

---

## <em>Clostridium butyricum</em> improves cognitive dysfunction in ICV-STZ-induced Alzheimer’s disease mice via suppressing TLR4 signaling pathway through the gut-brain axis

[Source](https://doi.org/10.1371/journal.pone.0286086)

---

## Regulation of gut microbiota by vitamin C, vitamin E and β-carotene

[Source](https://doi.org/10.1016/j.foodres.2023.112749)

---

## Diversity of the Pacific Ocean coral reef microbiome

[Source](https://doi.org/10.1038/s41467-023-38500-x)

---

## Effects of <em>Aeromonas hydrophila</em> infection on the intestinal microbiota, transcriptome, and metabolomic of common carp (<em>Cyprinus carpio</em>)

[Source](https://doi.org/10.1016/j.fsi.2023.108876)

---

## Genital tract microbiome dynamics are associated with time of <em>Chlamydia</em> infection in mice

[Source](https://doi.org/10.1038/s41598-023-36130-3)

---

## The core mangrove microbiome reveals shared taxa potentially involved in nutrient cycling and promoting host survival.
 05.06.2023, _BJ Wainwright, T Millar, L Bowen, L Semon, KJE Hickman, JN Lee, ZY Yeo, G Zahn_


Microbes have fundamental roles underpinning the functioning of our planet, they are involved in global carbon and nutrient cycling, and support the existence of multicellular life. The mangrove ecosystem is nutrient limited and if not for microbial cycling of nutrients, life in this harsh environment would likely not exist. The mangroves of Southeast Asia are the oldest and most biodiverse on the planet, and serve vital roles helping to prevent shoreline erosion, act as nursery grounds for many marine species and sequester carbon. Despite these recognised benefits and the importance of microbes in these ecosystems, studies examining the mangrove microbiome in Southeast Asia are scarce.cxs RESULTS: Here we examine the microbiome of Avicenia alba and Sonneratia alba and identify a core microbiome of 81 taxa. A further eight taxa (Pleurocapsa, Tunicatimonas, Halomonas, Marinomonas, Rubrivirga, Altererythrobacte, Lewinella, and Erythrobacter) were found to be significantly enriched in mangrove tree compartments suggesting key roles in this microbiome. The majority of those identified are involved in nutrient cycling or have roles in the production of compounds that promote host survival.

[Source](https://doi.org/10.1186/s40793-023-00499-5)

10.1186/s40793-023-00499-5

---

## Formation and Analysis of Mono-species and Polymicrobial Oral Biofilms in Flow-Cell Models.
 06.06.2023, _J Neilands, G Svensäter, G Boisen, C Robertsson, C Wickström, JR Davies_


_Humans, Biofilms, Saliva, Periodontitis, Adhesins, Bacterial, Bacteria_
The oral microbiota, which is known to include at least 600 different bacterial species, is found on the teeth and mucosal surfaces as multi-species communities or biofilms. The oral surfaces are covered with a pellicle of proteins absorbed from saliva, and biofilm formation is initiated when primary colonizers, which express surface adhesins that bind to specific salivary components, attach to the oral tissues. Further development then proceeds through co-aggregation of additional species. Over time, the composition of oral biofilms, which varies between different sites throughout the oral cavity, is determined by a combination of environmental factors such as the properties of the underlying surface, nutrient availability and oxygen levels, and bacterial interactions within the community. A complex equilibrium between biofilm communities and the host is responsible for the maintenance of a healthy biofilm phenotype (eubiosis). In the face of sustained environmental perturbation, however, biofilm homeostasis can break down giving rise to dysbiosis, which is associated with the development of oral diseases such as caries and periodontitis.In vitro models have an important part to play in increasing our understanding of the complex processes involved in biofilm development in oral health and disease, and the requirements for experimental system, microbial complexity, and analysis techniques will necessarily vary depending on the question posed. In this chapter we describe some current and well-established methods used in our laboratory for studying oral bacteria in biofilm models which can be adapted to suit the needs of individual users.

[Source](https://doi.org/10.1007/978-1-0716-3243-7_2)

10.1007/978-1-0716-3243-7_2

---

## MAIRA: Protein-based Analysis of MinION Reads on a Laptop.
 06.06.2023, _C Bağcı, B Albrecht, DH Huson_


_High-Throughput Nucleotide Sequencing, Sequence Analysis, DNA, Software, Microcomputers, Microbiota_
Third-generation sequencing technologies are being increasingly used in microbiome research and this has given rise to new challenges in computational microbiome analysis. Oxford Nanopore's MinION is a portable sequencer that streams data that can be basecalled on-the-fly. Here we give an introduction to the MAIRA software, which is designed to analyze MinION sequencing reads from a microbiome sample, as they are produced in real-time, on a laptop. The software processes reads in batches and updates the presented analysis after each batch. There are two analysis steps: First, protein alignments are calculated to determine which genera might be present in a sample. When strong evidence for a genus is found, then, in a second step, a more detailed analysis is performed by aligning the reads against the proteins of all species in the detected genus. The program presents a detailed analysis of species, antibiotic resistance genes, and virulence factors.

[Source](https://doi.org/10.1007/978-1-0716-3072-3_11)

10.1007/978-1-0716-3072-3_11

---

## Recovery and Analysis of Long-Read Metagenome-Assembled Genomes.
 06.06.2023, _K Arumugam, I Bessarab, MAS Haryono, RBH Williams_


_Metagenome, Metagenomics, Microbiota, Sequence Analysis, DNA, Base Sequence, High-Throughput Nucleotide Sequencing_
The development of long-read nucleic acid sequencing is beginning to make very substantive impact on the conduct of metagenome analysis, particularly in relation to the problem of recovering the genomes of member species of complex microbial communities. Here we outline bioinformatics workflows for the recovery and characterization of complete genomes from long-read metagenome data and some complementary procedures for comparison of cognate draft genomes and gene quality obtained from short-read sequencing and long-read sequencing.

[Source](https://doi.org/10.1007/978-1-0716-3072-3_12)

10.1007/978-1-0716-3072-3_12

---

## Evaluation of 96-well high-throughput DNA extraction methods for 16S rRNA gene metabarcoding.
 31.05.2023, _MP Chapuis, L Benoit, M Galan_


Gaining meaningful insights into bacterial communities associated with animal hosts requires the provision of high-quality nucleic acids. Although many studies have compared DNA extraction methods for samples with low bacterial biomass (e.g. water) or specific PCR inhibitors (e.g. plants), DNA extraction bias in samples without inherent technical constraint (e.g. animal samples) has received little attention. Furthermore, there is an urgent need to identify a DNA extraction methods in a high-throughput format that decreases the cost and time for processing large numbers of samples. We here evaluated five DNA extraction protocols, using silica membrane-based spin columns and a 96-well microplate format and based on either mechanical or enzymatic lysis or a combination of both, using three bacterial mock communities and Illumina sequencing of the V4 region of the 16SrRNA gene. Our results showed that none of the DNA extraction methods fully eliminated bias associated with unequal lysis efficiencies. However, we identified a DNA extraction method with a lower bias for each mock community standard. Of these methods, those including an enzymatic lysis showed biases specific to some bacteria. Altogether, these results again demonstrate the importance of DNA extraction standardization to be able to compare the microbiome results of different samples. In this attempt, we advise for the use of the 96-well DNeasy Blood and Tissue kit (Qiagen) with a zirconia bead-beating procedure, which optimizes altogether the cost, handling time and bacteria-specific effects associated with enzymatic lysis.

[Source](https://doi.org/10.1111/1755-0998.13812)

10.1111/1755-0998.13812

---

