## Development of Microbiome Biobanks – Challenges and Opportunities

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(20)30188-8)

---

## STROBE-metagenomics: a STROBE extension statement to guide the reporting of metagenomics studies.
 12.08.2020, _T Bharucha, C Oeser, F Balloux, JR Brown, EC Carbo, A Charlett, CY Chiu, ECJ Claas, MC de Goffau, JJC de Vries, M Eloit, S Hopkins, JF Huggett, D MacCannell, S Morfopoulou, A Nath, DM O'Sullivan, LB Reoma, LP Shaw, I Sidorov, PJ Simner, L Van Tan, EC Thomson, L van Dorp, MR Wilson, J Breuer, N Field_


The term metagenomics refers to the use of sequencing methods to simultaneously identify genomic material from all organisms present in a sample, with the advantage of greater taxonomic resolution than culture or other methods. Applications include pathogen detection and discovery, species characterisation, antimicrobial resistance detection, virulence profiling, and study of the microbiome and microecological factors affecting health. However, metagenomics involves complex and multistep processes and there are important technical and methodological challenges that require careful consideration to support valid inference. We co-ordinated a multidisciplinary, international expert group to establish reporting guidelines that address specimen processing, nucleic acid extraction, sequencing platforms, bioinformatics considerations, quality assurance, limits of detection, power and sample size, confirmatory testing, causality criteria, cost, and ethical issues. The guidance recognises that metagenomics research requires pragmatism and caution in interpretation, and that this field is rapidly evolving.

[Source](https://www.thelancet.com/action/showPdf?pii=S1473-3099%2820%2930199-7)

10.1016/S1473-3099(20)30199-7

---

## Schizasterid Heart Urchins Host Microorganisms in a Digestive Symbiosis of Mesozoic Origin.
 14.08.2020, _A Ziegler, AM Gilligan, JG Dillon, B Pernet_


Because of their lifestyles, abundance, and feeding habits, infaunal marine deposit feeders have a significant impact on the ocean floor. As these animals also ingest microorganisms associated with their sediment and seawater diet, their digestive tract usually contains a diverse array of bacteria. However, while most of these microorganisms are transients, some may become part of a resident gut microbiome, in particular when sheltered from the main flow of digesta in specialized gut compartments. Here, we provide an in-depth analysis of the structure and contents of the intestinal caecum (IC), a hindgut diverticulum found exclusively in schizasterid heart urchins (Echinoidea: Spatangoida: Schizasteridae). Based on specimens of Brisaster townsendi, in addition to various other schizasterid taxa, our structural characterization of the IC shows that the organ is a highly specialized gut compartment with unique structural properties. Next generation sequencing shows that the IC contains a microbial population composed predominantly of Bacteroidales, Desulfobacterales, and Spirochaetales. The microbiome of this gut compartment is significantly different in composition and lower in diversity than the microbial population in the sediment-filled main digestive tract. Inferences on the function and evolution of the IC and its microbiome suggest that this symbiosis plays a distinct role in host nutrition and that it evolved at least 66 million years ago during the final phase of the Mesozoic.

[Source](https://royalsocietypublishing.org/toc/rstb/2020/375/1808)

10.3389/fmicb.2020.01697

---

## The role of the microbiome in host evolution

[Source](https://royalsocietypublishing.org/toc/rstb/2020/375/1808)

---

## Microbiota-host interactions shape ageing dynamics.
 10.08.2020, _M Popkes, DR Valenzano_


Occupying the interface between host and environment, host-associated microbes play fundamental roles in nutrient absorption, essential metabolite synthesis, development of the immune system, defence against pathogens and pathogenesis. Microbiota composition and function is rather stable during adulthood, while it dramatically changes during early development, frailty and disease. Ageing is associated with progressive decrease of homeostasis, often resulting in disruption of the physiological balance between host and commensal microbes, ultimately leading to dysbiosis and host demise. Generally, high microbial diversity is associated with health and a youthful state, while low individual microbial diversity and larger inter-individual microbial diversity is associated with ageing and disease states. Different species are equipped with species-specific commensal, symbiotic and pathogenic microbial communities. How and whether the specific host-microbiota consortia co-evolved with host physiology to ensure homeostasis and promote individual fitness remains an open question. In this essay, we propose that the evolution of vertebrate-specific immune adaptations may have enabled the establishment of highly diverse, species-specific commensal microbial communities. We discuss how the maintenance of intact immune surveillance mechanisms, which allow discrimination between commensal and pathogenic bacteria, fail during ageing and lead to the onset of known ageing-related diseases. We discuss how host-microbiota interactions are key to maintaining homeostasis despite external perturbations, but also how they affect a range of host-specific ageing-related phenotypes. This article is part of the theme issue 'The role of the microbiome in host evolution'.

[Source](https://royalsocietypublishing.org/doi/pdf/10.1098/rstb.2019.0596)

10.1098/rstb.2019.0596

---

## Pathogen resistance may be the principal evolutionary advantage provided by the microbiome.
 10.08.2020, _MR McLaren, BJ Callahan_


To survive, plants and animals must continually defend against pathogenic microbes that would invade and disrupt their tissues. Yet they do not attempt to extirpate all microbes. Instead, they tolerate and even encourage the growth of commensal microbes, which compete with pathogens for resources and via direct inhibition. We argue that hosts have evolved to cooperate with commensals in order to enhance the pathogen resistance this competition provides. We briefly describe competition between commensals and pathogens within the host, consider how natural selection might favour hosts that tilt this competition in favour of commensals, and describe examples of extant host traits that may serve this purpose. Finally, we consider ways that this cooperative immunity may have facilitated the adaptive evolution of non-pathogen-related host traits. On the basis of these observations, we argue that pathogen resistance vies with other commensal-provided benefits for being the principal evolutionary advantage provided by the microbiome to host lineages across the tree of life. This article is part of the theme issue 'The role of the microbiome in host evolution'.

[Source](https://royalsocietypublishing.org/doi/pdf/10.1098/rstb.2019.0592)

10.1098/rstb.2019.0592

---

## A Randomized Clinical Trial of Fecal Microbiota Transplant for Alcohol Use Disorder.
 04.08.2020, _JS Bajaj, EA Gavis, A Fagan, JB Wade, LR Thacker, M Fuchs, S Patel, B Davis, J Meador, P Puri, M Sikaroodi, PM Gillevet_


Alcohol use disorder (AUD) is associated with microbial alterations that worsen with cirrhosis. Fecal microbiota transplant (FMT) could be a promising approach.

[Source](https://aasldpubs.onlinelibrary.wiley.com/doi/epdf/10.1002/hep.31496)

10.1002/hep.31496

---

## Comparative Genomic Study of <em>Lactobacillus jensenii</em> and the Newly Defined <em>Lactobacillus mulieris</em> Species Identifies Species-Specific Functionality

[Source](https://msphere.asm.org/content/5/4/e00560-20)

---

## Vaginal Estrogen Therapy Is Associated with Increased<em> Lactobacillus</em> in the Urine of Post-Menopausal Women with OAB Symptoms

[Source](https://www.ajog.org/article/S0002-9378(20)30834-6/pdf)

---

## Vaginal microbiota, genital inflammation, and neoplasia impact immune checkpoint protein profiles in the cervicovaginal microenvironment

[Source](https://www.nature.com/articles/s41698-020-0126-x)

---

## Lateral Gene Transfer Shapes Diversity of <em>Gardnerella</em> spp.

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2020.00293/full)

---

## Variation of the vaginal microbiome during and after pregnancy in Chinese women

[Source](https://www.medrxiv.org/content/10.1101/2020.07.07.20148536v1.full.pdf)

---

## Diversity of Vaginal Microbiome in Pregnancy: Deciphering the Obscurity

[Source](https://www.frontiersin.org/articles/10.3389/fpubh.2020.00326/full)

---

## Evidence for contamination as the origin for bacteria found in human placenta rather than a microbiota

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0237232)

---

## A Novel Description of the Human Sinus Archaeome During Health and Chronic Rhinosinusitis

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2020.00398/full)

---

## Temporal changes in the gut microbiota in farmed Atlantic cod (<em>Gadus morhua</em>) outweigh the response to diet supplementation with macroalgae

[Source](https://www.biorxiv.org/content/10.1101/2020.08.10.222604v1.full.pdf)

---

## Broiler Chickens and Early Life Programming: Microbiome transplant-induced cecal bacteriome dynamics and phenotypic effects

[Source](https://www.biorxiv.org/content/biorxiv/early/2020/08/14/2020.08.13.240572.full.pdf)

---

## Rumen metaproteomics: Closer to linking rumen microbial function to animal productivity traits.
 14.08.2020, _TO Andersen, BJ Kunath, LH Hagen, MØ Arntzen, PB Pope_


The rumen microbiome constitutes a dense and complex mixture of anaerobic bacteria, archaea, protozoa, virus and fungi. Collectively, rumen microbial populations interact closely in order to degrade and ferment complex plant material into nutrients for host metabolism, a process which also produces other by-products, such as methane gas. Our understanding of the rumen microbiome and its functions are of both scientific and industrial interest, as the metabolic functions are connected to animal health and nutrition, but at the same time contribute significantly to global greenhouse gas emissions. While many of the major microbial members of the rumen microbiome are acknowledged, advances in modern culture-independent meta-omic techniques, such as metaproteomics, enable deep exploration into active microbial populations involved in essential rumen metabolic functions. Meaningful and accurate metaproteomic analyses are highly dependent on representative samples, precise protein extraction and fractionation, as well as a comprehensive and high-quality protein sequence database that enables precise protein identification and quantification. This review focuses on the application of rumen metaproteomics, and its potential towards understanding the complex rumen microbiome and its metabolic functions. We present and discuss current methods in sample handling, protein extraction and data analysis for rumen metaproteomics, and finally emphasize the potential of (meta)genome-integrated metaproteomics for accurate reconstruction of active microbial populations in the rumen.

[Source](https://www.sciencedirect.com/science/article/pii/S1046202320301067)

10.1016/j.ymeth.2020.07.011

---

## Fast and Slow-Growing Management Systems: Characterisation of Broiler Caecal Microbiota Development throughout the Growing Period

[Source](https://www.mdpi.com/2076-2615/10/8/1401/htm)

---

## Desmosomal protein regulation and clinical implications in oral mucosal tissues.
 14.08.2020, _NU Huda Raza, S Ghafoor_


Cell-adhesion complex within a tissue is important for its stability, structural integrity, functioning, cellular migration and morphogenesis. Disruption of desmosomal cell-adhesions complex results in epithelial conditions such as epidermolysis bullosa and bullous pemphigoid. Desmosome assembly and disassembly is regulated post-translationally by calcium, kinase/phosphatase activity, proteolytic processing, and also through adhesive junctions. Altered functions of desmosomal proteins desmocollin and desmoglein can cause blistering disorders, such as pemphigus foliaceus and pemphigus vulgaris, and non-Hodgkin Lymphoma while defective desmoplakin can cause supra-basal clefting in epithelium and conditions such as Carvajal syndrome, palmo-plantar keratoderma etc. This review summarises major functions of demosomal complex family and how mis-regulation of demosomal structural proteins occur in pathogenesis of non-, pre- and malignant oral lesions with disrupted epithelium.

[Source](https://www.nature.com/articles/s41586-020-2564-6)

10.5455/JPMA.15798

---

## Mucosal or systemic microbiota exposures shape the B cell repertoire

[Source](https://www.nature.com/articles/s41586-020-2564-6)

---

## Screening for chronic prostatitis pathogens using high-throughput next-generation sequencing.
 15.07.2020, _Y Wu, H Jiang, M Tan, X Lu_


_Bacterial Infections, Case-Control Studies, Chronic Disease, Cluster Analysis, Fungi, Gram-Negative Bacteria, Gram-Positive Bacteria, High-Throughput Nucleotide Sequencing, Humans, Male, Mycoses, Pelvic Pain, Principal Component Analysis, Prostatitis, RNA, Ribosomal, 16S_
The pathogens responsible for chronic prostatitis/chronic pelvic pain syndrome (CP/CPPS; NIH category III) are not currently known. the present study utilized high-throughput next-generation sequencing to screen for potential pathogens associated with NIH category III CP (CP III).

[Source](https://www.biorxiv.org/content/10.1101/2020.02.29.971051v1.full.pdf)

10.1002/pros.23971

---

## The immune system fails to mount a protective response to Gram-positive or Gram-negative bacterial prostatitis

[Source](https://www.biorxiv.org/content/10.1101/2020.02.29.971051v1.full.pdf)

---

## Oncogenic Gene Fusions in Non-Neoplastic Precursors as Evidence that Bacterial Infection Initiates Prostate Cancer

[Source](https://www.biorxiv.org/content/10.1101/2020.07.27.224154v1.full.pdf)

---

## Microbiome-derived inosine modulates response to checkpoint inhibitor immunotherapy.
 14.08.2020, _LF Mager, R Burkhard, N Pett, NCA Cooke, K Brown, H Ramay, S Paik, J Stagg, RA Groves, M Gallo, IA Lewis, MB Geuking, KD McCoy_


Several species of intestinal bacteria have been associated with enhanced efficacy of checkpoint blockade immunotherapy, but the underlying mechanisms by which the microbiome enhances anti-tumor immunity is unclear. Here, we isolated three bacterial species, including Bifidobacterium pseudolongum, Lactobacillus johnsonii and Olsenella species, that significantly enhanced efficacy of immune checkpoint inhibitors in four mouse models of cancer. We found that intestinal B. pseudolongum modulated enhanced immunotherapy response through production of the metabolite inosine. Decreased gut barrier function induced by immunotherapy increased systemic translocation of inosine and activated anti-tumor T cells. The effect of inosine was dependent on T cell expression of the adenosine A2A receptor and required co-stimulation. Collectively, our study identifies a novel microbial metabolite-immune pathway that is activated by immunotherapy that may be exploited to develop microbial-based adjuvant therapies.

[Source](https://science.sciencemag.org/content/early/2020/08/12/science.abc3421)

10.1126/science.abc3421

---

## A genome compendium reveals diverse metabolic adaptations of Antarctic soil microorganisms

[Source](https://www.biorxiv.org/content/10.1101/2020.08.06.239558v1.full.pdf)

---

## Healthy soils for healthy plants for healthy humans: How beneficial microbes in the soil, food and gut are interconnected and how agriculture can contribute to human health.
 07.08.2020, _H Hirt_


The microbiota of the human gut and the plant rhizome are similar in many ways and intricately connected with each other. A healthy plant therefore affects human microbiota and human health.

[Source](https://www.embopress.org/doi/epdf/10.15252/embr.202051069)

10.15252/embr.202051069

---

## Healthy soils for healthy plants for healthy humans

[Source](https://www.embopress.org/doi/epdf/10.15252/embr.202051069)

---

## The Impact of Mercury Selection and Conjugative Genetic Elements on Community Structure and Resistance Gene Transfer

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.01846/full)

---

## Plant-microbiome interactions: from community assembly to plant health.
 13.08.2020, _P Trivedi, JE Leach, SG Tringe, T Sa, BK Singh_


Healthy plants host diverse but taxonomically structured communities of microorganisms, the plant microbiota, that colonize every accessible plant tissue. Plant-associated microbiomes confer fitness advantages to the plant host, including growth promotion, nutrient uptake, stress tolerance and resistance to pathogens. In this Review, we explore how plant microbiome research has unravelled the complex network of genetic, biochemical, physical and metabolic interactions among the plant, the associated microbial communities and the environment. We also discuss how those interactions shape the assembly of plant-associated microbiomes and modulate their beneficial traits, such as nutrient acquisition and plant health, in addition to highlighting knowledge gaps and future directions.

[Source](https://www.nature.com/articles/s41579-020-0412-1)

10.1038/s41579-020-0412-1

---

## Differential prevalence and host-association of antimicrobial resistance traits in disinfected and non-disinfected drinking water systems

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969720349809)

---

## MiDAS 3: An ecosystem-specific reference database, taxonomy and knowledge platform for activated sludge and anaerobic digesters reveals species-level microbiome composition of activated sludge.
 10.08.2020, _M Nierychlo, KS Andersen, Y Xu, N Green, C Jiang, M Albertsen, MS Dueholm, PH Nielsen_


The function of the microbiomes in wastewater treatment systems and anaerobic digesters is dictated by the physiological activity of their members and complex interactions between them. Since functional traits are often conserved at low taxonomic ranks (genus, species, strain), high resolution taxonomic classification is crucial to understand the role of microbes in any ecosystem. Here we present MiDAS 3, a comprehensive 16S rRNA gene reference database based on full-length 16S rRNA gene amplicon sequence variants (FL-ASVs) derived from activated sludge and anaerobic digester systems in Denmark. The new database proposes unique provisional names for all unclassified microorganisms down to species level, providing a new and much-needed tool for microbiome research. The MiDAS 3 database was used to analyze the microbiome in 20 Danish wastewater treatment plants with nutrient removal, sampled over 13 years. The 50 most abundant species belonged to 42 genera, including 14 genera with provisional 'midas' name. Of those, 20 have no known function in the system, which highlights the need for more efforts towards elucidating the role of important members of wastewater treatment ecosystems. The new MiDAS 3 database also forms the backbone of the MiDAS Field Guide - an online resource linking the identity of microorganisms in wastewater treatment systems to available data related to their functional importance. The new field guide contains a complete list of genera (&gt;1800) and species (&gt;4200) found in activated sludge and anaerobic digesters in Denmark, but is also relevant to wastewater systems across the world. The identity of the microbes is linked to functional information, where available, and the website provides the possibility to BLAST new sequences against the MiDAS 3 database. The MiDAS Field Guide is a collaborative platform acting as an online knowledge repository, facilitating understanding of wastewater treatment ecosystem function.

[Source](https://www.sciencedirect.com/science/article/pii/S0043135420304929)

10.1016/j.watres.2020.115955

---

## Metagenomic profiling of ammonia- and methane-oxidizing microorganisms in two sequential rapid sand filters

[Source](https://www.sciencedirect.com/science/article/pii/S0043135420308241)

---

## Shotgun metagenomics of indigenous bacteria collected from the banks of the San Jacinto River for biodegradation of aromatic waste.
 07.08.2020, _R Iyer, A Damania_


Several Eastern Harris County communities lie near the now enclosed San Jacinto River (SJR) Waste Pit Superfund, a dumping ground for chlorinated dioxins and other paper mill waste products. Currently, no active monitoring of the SJR is conducted for these toxins with the exact concentration and health impact to the area unknown. As such, remediation and monitoring efforts outside of the Superfund itself could be necessary. To better understand the possible environmental fate of these aromatics, here we provide a shotgun metagenomic analysis of the structural and putative functional diversity of the SJR microbiome from two impacted Channelview, Texas communities bordering the Superfund. Results show that the underlying SJR microbiome possesses a core of metabolic enzymes related to the β-ketoadipate and benzoate degradation pathways. This suggests possible endpoints for many aromatics found deposited in the SJR including dioxin-like compounds. However, degradation biomarkers related to the priming and initial cleavage of chlorinated dioxin-like aromatics while present, are poorly concentrated across sampled sites. This may be due in part to decreased coverage of low abundance bacterial species, but also be a contributing factor leading to increased recalcitrance of these compounds in this environment compared to other aromatics.

[Source](https://academic.oup.com/femsle/advance-article-abstract/doi/10.1093/femsle/fnaa133/5881932)

10.1093/femsle/fnaa133

---

## Viruses in wastewater: occurrence, abundance and detection methods.
 11.08.2020, _MVA Corpuz, A Buonerba, G Vigliotta, T Zarra, F Ballesteros, P Campiglia, V Belgiorno, G Korshin, V Naddeo_


This paper presents an updated and comprehensive review on the different methods used for detection and quantification of viruses in wastewater treatment systems. The analysis of viability of viruses in wastewater and sludge is another thrust of this review. Recent studies have mostly focused on determining the abundance and diversity of viruses in wastewater influents, in samples from primary, secondary, and tertiary treatment stages, and in final effluents. A few studies have also examined the occurrence and diversity of viruses in raw and digested sludge samples. Recent efforts to improve efficiency of virus detection and quantification methods in the complex wastewater and sludge matrices are highlighted in this review. A summary and a detailed comparison of the pre-treatment methods that have been utilized for wastewater and sludge samples are also presented. The role of metagenomics or sequencing analysis in monitoring wastewater systems to predict disease outbreaks, to conduct public health surveillance, to assess the efficiency of existing treatment systems in virus removal, and to re-evaluate current regulations regarding pathogenic viruses in wastewater is discussed in this paper. Challenges and future perspectives in the detection of viruses, including emerging and newly emerged viruses such as the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), in wastewater systems are discussed in this review.

[Source](https://www.sciencedirect.com/science/article/pii/S0048969720344399)

10.1016/j.scitotenv.2020.140910

---

## Viromes outperform total metagenomes in revealing the spatiotemporal patterns of agricultural soil viral communitie

[Source](https://www.biorxiv.org/content/10.1101/2020.08.06.237214v1.full.pdf)

---

## Correlative Light Electron Ion Microscopy reveal in vivo localisation of bedaquiline in <em>Mycobacterium tuberculosis</em> infected lungs

[Source](https://www.biorxiv.org/content/10.1101/2020.08.05.237537v1.full.pdf)

---

## Label-free smartphone quantitation of bacteria by darkfield imaging of light scattering in fluoropolymer micro capillary film allows portable detection of bacteriophage lysis

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0925400520309928)

---

## Raven: a de novo genome assembler for long reads

[Source](https://www.biorxiv.org/content/10.1101/2020.08.07.242461v1.full.pdf)

---

## SHAMAN: a user-friendly website for metataxonomic analysis from raw reads to statistical analysis.
 11.08.2020, _S Volant, P Lechat, P Woringer, L Motreff, P Campagne, C Malabat, S Kennedy, A Ghozlane_


Comparing the composition of microbial communities among groups of interest (e.g., patients vs healthy individuals) is a central aspect in microbiome research. It typically involves sequencing, data processing, statistical analysis and graphical display. Such an analysis is normally obtained by using a set of different applications that require specific expertise for installation, data processing and in some cases, programming skills.

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03666-4)

10.1186/s12859-020-03666-4

---

## Rapid discovery of novel prophages using biological feature engineering and machine learning

[Source](https://www.biorxiv.org/content/10.1101/2020.08.09.243022v1.full.pdf)

---

## Critical Relevance of Stochastic Effects on Low-Bacterial-Biomass 16S rRNA Gene Analysis.
 25.07.2020, _JR Erb-Downward, NR Falkowski, JC D'Souza, LM McCloskey, RA McDonald, CA Brown, K Shedden, RP Dickson, CM Freeman, KA Stringer, B Foxman, GB Huffnagle, JL Curtis, SD Adar_


The bacterial microbiome of human body sites, previously considered sterile, remains highly controversial because it can be challenging to isolate signal from noise when low-biomass samples are being analyzed. We tested the hypothesis that stochastic sequencing noise, separable from reagent contamination, is generated during sequencing on the Illumina MiSeq platform when DNA input is below a critical threshold. We first purified DNA from serial dilutions of Pseudomonas aeruginosa and from negative controls using three DNA purification kits, quantified input using droplet digital PCR, and then sequenced the 16S rRNA gene in four technical replicates. This process identified reproducible contaminant signal that was separable from an irreproducible stochastic noise, which occurred as bacterial biomass of samples decreased. This approach was then applied to authentic respiratory samples from healthy individuals (n = 22) that ranged from high to ultralow bacterial biomass. Using oral rinse, bronchoalveolar lavage (BAL) fluid, and exhaled breath condensate (EBC) samples and matched controls, we were able to demonstrate (i) that stochastic noise dominates sequencing in real-world low-bacterial-biomass samples that contain fewer than 104 copies of the 16S rRNA gene per sample, (ii) that critical examination of the community composition of technical replicates can be used to separate signal from noise, and (iii) that EBC is an irreproducible sampling modality for sampling the microbiome of the lower airways. We anticipate that these results combined with suggested methods for identifying and dealing with noisy communities will facilitate increased reproducibility while simultaneously permitting characterization of potentially important low-biomass communities.IMPORTANCE DNA contamination from external sources (reagents, environment, operator, etc.) has long been assumed to be the main cause of spurious signals that appear under low-bacterial-biomass conditions. Here, we demonstrate that contamination can be separated from another, random signal generated during low-biomass-sample sequencing. This stochastic noise is not reproduced between technical replicates; however, results for any one replicate taken alone could look like a microbial community different from the controls. Using this information, we investigated respiratory samples from healthy humans and determined the narrow range of bacterial biomass where samples transition from producing reproducible microbial sequences to ones dominated by noise. We present a rigorous approach to studies involving low-bacterial-biomass samples to detect this source of noise and provide a framework for deciding if a sample is likely to be dominated by noise. We anticipate that this work will facilitate increased reproducibility in the characterization of potentially important low-biomass communities.

[Source](https://mbio.asm.org/content/mbio/11/3/e00258-20.full.pdf)

10.1128/mBio.00258-20

---

## On the complexity of haplotyping a microbial community

[Source](https://www.biorxiv.org/content/10.1101/2020.08.10.244848v1.full.pdf)

---

## Inclusion of Oxford Nanopore long reads improves all microbial and viral metagenome-assembled genomes from a complex aquifer system.
 07.08.2020, _WA Overholt, M Hölzer, P Geesink, C Diezel, M Marz, K Küsel_


Assembling microbial and viral genomes from metagenomes is a powerful and appealing method to understand structure-function relationships in complex environments. To compare the recovery of genomes from microorganisms and their viruses from groundwater, we generated shotgun metagenomes with Illumina sequencing accompanied by long reads derived from the Oxford Nanopore Technologies (ONT) sequencing platform. Assembly and metagenome-assembled genome (MAG) metrics for both microbes and viruses were determined from an Illumina-only assembly, ONT-only assembly, and a hybrid assembly approach. The hybrid approach recovered 2x more mid to high-quality MAGs compared to the Illumina-only approach and 4x more than the ONT-only approach. A similar number of viral genomes were reconstructed using the hybrid and ONT methods, and both recovered nearly four-fold more viral genomes than the Illumina-only approach. While yielding fewer MAGs, the ONT-only approach generated MAGs with a high probability of containing rRNA genes, 3x higher than either of the other methods. Of the shared MAGs recovered from each method, the ONT-only approach generated the longest and least fragmented MAGs, while the hybrid approach yielded the most complete. This work provides quantitative data to inform a cost-benefit analysis of the decision to supplement shotgun metagenomic projects with long reads towards the goal of recovering genomes from environmentally abundant groups. This article is protected by copyright. All rights reserved.

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/epdf/10.1111/1462-2920.15186)

10.1111/1462-2920.15186

---

## Meta-analysis of 16S rRNA Microbial Data Identified Distinctive and Predictive Microbiota Dysbiosis in Colorectal Carcinoma Adjacent Tissue.
 28.07.2020, _Z Mo, P Huang, C Yang, S Xiao, G Zhang, F Ling, L Li_


As research focusing on the colorectal cancer fecal microbiome using shotgun sequencing continues, increasing evidence has supported correlations between colorectal carcinomas (CRCs) and fecal microbiome dysbiosis. However, large-scale on-site and off-site (surrounding adjacent) tissue microbiome characterization of CRC was underrepresented. Here, considering each taxon as a feature, we demonstrate a machine learning-based method to investigate tissue microbial differences among CRC, colorectal adenoma (CRA), and healthy control groups using 16S rRNA data sets retrieved from 15 studies. A total of 2,099 samples were included and analyzed in case-control comparisons. Multiple methods, including differential abundance analysis, random forest classification, cooccurrence network analysis, and Dirichlet multinomial mixture analysis, were conducted to investigate the microbial signatures. We showed that the dysbiosis of the off-site tissue of colonic cancer was distinctive and predictive. The AUCs (areas under the curve) were 80.7%, 96.0%, and 95.8% for CRC versus healthy control random forest models using stool, tissue, and adjacent tissue samples and 69.9%, 91.5%, and 89.5% for the corresponding CRA models, respectively. We also found that the microbiota ecologies of the surrounding adjacent tissues of CRC and CRA were similar to their on-site counterparts according to network analysis. Furthermore, based on the enterotyping of tissue samples, the cohort-specific microbial signature might be the crux in addressing classification generalization problems. Despite cohort heterogeneity, the dysbiosis of lesion-adjacent tissues might provide us with further perspectives in demonstrating the role of the microbiota in colorectal cancer tumorigenesis.IMPORTANCE Turbulent fecal and tissue microbiome dysbiosis of colorectal carcinoma and adenoma has been identified, and some taxa have been proven to be carcinogenic. However, the microbiomes of surrounding adjacent tissues of colonic cancerous tissues were seldom investigated uniformly on a large scale. Here, we characterize the microbiome signatures and dysbiosis of various colonic cancer sample groups. We found a high correlation between colorectal carcinoma adjacent tissue microbiomes and their on-site counterparts. We also discovered that the microbiome dysbiosis in adjacent tissues could discriminate colorectal carcinomas from healthy controls effectively. These results extend our knowledge on the microbial profile of colorectal cancer tissues and highlight microbiota dysbiosis in the surrounding tissues. They also suggest that microbial feature variations of cancerous lesion-adjacent tissues might help to reveal the microbial etiology of colonic cancer and could ultimately be applied for diagnostic and screening purposes.

[Source](https://conference.dsi.ubc.ca/)

10.1128/mSystems.00138-20

---

## Microbiome, machine learning and network models

[Source](https://conference.dsi.ubc.ca/)

---

## Annual Research Review: Critical windows - the microbiota-gut-brain axis in neurocognitive development.
 19.02.2020, _CSM Cowan, TG Dinan, JF Cryan_


The gut microbiota is a vast, complex, and fascinating ecosystem of microorganisms that resides in the human gastrointestinal tract. As an integral part of the microbiota-gut-brain axis, it is now being recognized that the microbiota is a modulator of brain and behavior, across species. Intriguingly, periods of change in the microbiota coincide with the development of other body systems and particularly the brain. We hypothesize that these times of parallel development are biologically relevant, corresponding to 'sensitive periods' or 'critical windows' in the development of the microbiota-gut-brain axis. Specifically, signals from the microbiota during these periods are hypothesized to be crucial for establishing appropriate communication along the axis throughout the life span. In other words, the microbiota is hypothesized to act like an expected input to calibrate the development of the microbiota-gut-brain axis. The absence or disruption of the microbiota during specific developmental windows would therefore be expected to have a disproportionate effect on specific functions or potentially for regulation of the system as a whole. Evidence for microbial modulation of neurocognitive development and neurodevelopmental risk is discussed in light of this hypothesis, finishing with a focus on the challenges that lay ahead for the future study of the microbiota-gut-brain axis during development.

[Source](https://researcheracademy.elsevier.com/workshop/b0864329-151a-4795-b808-8c8811c3d679)

10.1111/jcpp.13156

---

## The Gut Brain Axis across the Life Span

[Source](https://researcheracademy.elsevier.com/workshop/b0864329-151a-4795-b808-8c8811c3d679)

---

## Pushing the Envelope: Remodeling the Gram-negative Outer Membrane

[Source](https://nus-sg.zoom.us/meeting/register/tJElcu6qrzMtHNFgujqZRp5CUIRFP3PIlkQj)

---

## Singapore Bacterial Ultra Group

[Source](https://twitter.com/search?q=%23sgbug&src=typed_query)

---

## Getting to Know Your Friendly Neighbourhood Microbiologist

[Source](http://jams.org.au/index.php/jamspod/)

---

## Does not compute

[Source](https://www.theblackgoatpodcast.com/posts/does-not-compute/)

---

