## The role of immune cells in the liver tumor microenvironment: an involvement of gut microbiota-derived factors.
 02.06.2022, _T Kamiya, N Ohtani_


More than 500 species of microbiota reside in the human intestine and coexist with humans, their host. Gut microbial metabolites and components are absorbed from the intestine and influence cells in the liver, including hepatocytes and stromal cells, such as liver sinusoidal endothelial cells, hepatic stellate cells, Kupffer cells, natural killer (NK) cells, NK T cells, and other immune cells. This gut-originated axis to the liver is called the "gut-liver axis", which underscores the importance of the link between the gut and the liver. In this review, we discuss the gut microbial components and metabolites that affect cells in the liver, particularly in association with immune cells, and the related responses. We also highlight the mechanisms underlying gut microbiota-mediated liver carcinogenesis and discuss cancer prevention, including the recently clarified modulation of immune checkpoint inhibitor efficacy by the gut microbiota.

[Source](https://doi.org/10.1093/intimm/dxac020)

10.1093/intimm/dxac020

---

## Multiomic Analyses of Nascent Preterm Infant Microbiomes Differentiation Suggest Opportunities for Targeted Intervention.
 02.06.2022, _SB Orchanian, JM Gauglitz, S Wandro, KC Weldon, M Doty, K Stillwell, S Hansen, L Jiang, F Vargas, KE Rhee, JC Lumeng, PC Dorrestein, R Knight, JH Kim, SJ Song, AD Swafford_


The first week after birth is a critical time for the establishment of microbial communities for infants. Preterm infants face unique environmental impacts on their newly acquired microbiomes, including increased incidence of cesarean section delivery and exposure to antibiotics as well as delayed enteral feeding and reduced human interaction during their intensive care unit stay. Using contextualized paired metabolomics and 16S sequencing data, the development of the gut, skin, and oral microbiomes of infants is profiled daily for the first week after birth, and it is found that the skin microbiome appears robust to early life perturbation, while direct exposure of infants to antibiotics, rather than presumed maternal transmission, delays microbiome development and prevents the early differentiation based on body site regardless of delivery mode. Metabolomic analyses identify the development of all gut metabolomes of preterm infants toward full-term infant profiles, but a significant increase of primary bile acid metabolism only in the non-antibiotic treated vaginally birthed late preterm infants. This study provides a framework for future multi-omic, multibody site analyses on these high-risk preterm infant populations and suggests opportunities for monitoring and intervention, with infant antibiotic exposure as the primary driver of delays in microbiome development.

[Source](https://doi.org/10.1002/adbi.202101313)

10.1002/adbi.202101313

---

## Short chain fatty acids-producing and mucin-degrading intestinal bacteria predict the progression of early Parkinson's disease.
 01.06.2022, _H Nishiwaki, M Ito, T Hamaguchi, T Maeda, K Kashihara, Y Tsuboi, J Ueyama, T Yoshida, H Hanada, I Takeuchi, M Katsuno, M Hirayama, K Ohno_


To elucidate the relevance of gut dysbiosis in Parkinson's disease (PD) in disease progression, we made random forest models to predict the progression of PD in two years by gut microbiota in 165 PD patients. The area under the receiver operating characteristic curves (AUROCs) of gut microbiota-based models for Hoehn &amp; Yahr (HY) stages 1 and 2 were 0.799 and 0.705, respectively. Similarly, gut microbiota predicted the progression of Movement Disorder Society-Unified Parkinson's Disease Rating Scale (MDS-UPDRS) III scores in an early stage of PD with AUROC = 0.728. Decreases of short-chain fatty acid-producing genera, Fusicatenibacter, Faecalibacterium, and Blautia, as well as an increase of mucin-degrading genus Akkermansia, predicted accelerated disease progression. The four genera remained unchanged in two years in PD, indicating that the taxonomic changes were not the consequences of disease progression. PD patients with marked gut dysbiosis may thus be destined to progress faster than those without gut dysbiosis.

[Source](https://doi.org/10.1038/s41531-022-00328-5)

10.1038/s41531-022-00328-5

---

## Intestinal inflammation alters the antigen-specific immune response to a skin commensal.
 03.06.2022, _GR Merana, LR Dwyer, MO Dhariwala, A Weckel, JR Gonzalez, JN Okoro, JN Cohen, CM Tamaki, J Han, P Tasoff, Y Palacios-Calderon, CWY Ha, SV Lynch, JA Segre, HH Kong, MG Kattah, A Ma, TC Scharschmidt_


_Animals, Colitis, Immunity, Inflammation, Mice, Skin, Staphylococcus epidermidis, T-Lymphocytes, Regulatory_
Resident microbes in skin and gut predominantly impact local immune cell function during homeostasis. However, colitis-associated neutrophilic skin disorders suggest possible breakdown of this compartmentalization with disease. Using a model wherein neonatal skin colonization by Staphylococcus epidermidis facilitates generation of commensal-specific tolerance and CD4+ regulatory T cells (Tregs), we ask whether this response is perturbed by gut inflammation. Chemically induced colitis is accompanied by intestinal expansion of S. epidermidis and reduces gut-draining lymph node (dLN) commensal-specific Tregs. It also results in reduced commensal-specific Tregs in skin and skin-dLNs and increased skin neutrophils. Increased CD4+ circulation between gut and skin dLN suggests that the altered cutaneous response is initiated in the colon, and resistance to colitis-induced effects in Cd4creIl1r1fl/fl mice implicate interleukin (IL)-1 in mediating the altered commensal-specific response. These findings provide mechanistic insight into observed connections between inflammatory skin and intestinal diseases.

[Source](https://doi.org/10.1016/j.celrep.2022.110891)

10.1016/j.celrep.2022.110891

---

## Characterization of resistome in <em>Lactobacillus</em> genomic sequences from the human gut

[Source](https://doi.org/10.1016/j.jgar.2022.05.014)

---

## Clustering Analysis of the Multi-Microbial Consortium by <em>Lactobacillus</em> Species Against Vaginal Dysbiosis Among Ecuadorian Women

[Source](https://doi.org/10.3389/fcimb.2022.863208)

---

## Nutritional Intake and Gut Microbiome Composition Predict Parkinson's Disease.
 02.06.2022, _M Lubomski, X Xu, AJ Holmes, S Muller, JYH Yang, RL Davis, CM Sue_


Models to predict Parkinson's disease (PD) incorporating alterations of gut microbiome (GM) composition have been reported with varying success.

[Source](https://doi.org/10.3389/fnagi.2022.881872)

10.3389/fnagi.2022.881872

---

## Bacteriocin-Producing Probiotic Lactic Acid Bacteria in Controlling Dysbiosis of the Gut Microbiota

[Source](https://doi.org/10.3389/fcimb.2022.851140)

---

## Relationship between gut microbiota and colorectal cancer: Probiotics as a potential strategy for prevention.
 03.06.2022, _SY Lee, DY Lee, JH Kang, JH Kim, JW Jeong, HW Kim, DH Oh, SH Yoon, SJ Hur_


_Carcinogenesis, Colorectal Neoplasms, Gastrointestinal Microbiome, Humans, Prebiotics, Probiotics_
Gut microbiota comprise microorganisms residing in the gastrointestinal tract. Some of these microbiota are implicated in the progression of colorectal cancer (CRC). Here, we highlight studies on the effects of meat intake and fermented foods on characteristics of gut microbiota that can influence colitis-associated factors underlying CRC. Gut microbiota can influence the development and progression of CRC, through influencing factors such as secretion of toxins; enzymes for activating carcinogenesis (including β-glucuronidase, β-glucosidase, azoreductase, nitroreductase, and alcohol dehydrogenase); hydrogen sulfide generation; generation of reactive oxygen species and inflammation; secondary bile salt transformation; and products of protein fermentation. Additionally, some studies that the composition of gut microbiota (probiotics) or prebiotics plays an important role in the production of short chain fatty acids, inactivation enzymes for carcinogenesis, antioxidant activities, and inhibition of pathogen colonization. In this review, we discuss various explanatory mechanisms of the relationship between the multifactorial role of the gut microbiota and the development of CRC. Moreover, this review provides fundamental information on dietary fermented food and the gut microbiota, which is helpful for healthy people and those with CRC alike.

[Source](https://doi.org/10.1016/j.foodres.2022.111327)

10.1016/j.foodres.2022.111327

---

## Health benefits of microalgae and their microbiomes.
 01.06.2022, _I Krohn, S Menanteau-Ledouble, G Hageskal, Y Astafyeva, P Jouannais, JL Nielsen, M Pizzol, A Wentzel, WR Streit_


Microalgae comprise a phylogenetically very diverse group of photosynthetic unicellular pro- and eukaryotic organisms growing in marine and other aquatic environments. While they are well explored for the generation of biofuels, their potential as a source of antimicrobial and prebiotic substances have recently received increasing interest. Within this framework, microalgae may offer solutions to the societal challenge we face, concerning the lack of antibiotics treating the growing level of antimicrobial resistant bacteria and fungi in clinical settings. While the vast majority of microalgae and their associated microbiota remain unstudied, they may be a fascinating and rewarding source for novel and more sustainable antimicrobials and alternative molecules and compounds. In this review, we present an overview of the current knowledge on health benefits of microalgae and their associated microbiota. Finally, we describe remaining issues and limitation, and suggest several promising research potentials that should be given attention.

[Source](https://doi.org/10.1111/1751-7915.14082)

10.1111/1751-7915.14082

---

## <em>Lactobacillus reuteri</em> J1 prevents obesity by altering the gut microbiota and regulating bile acid metabolism in obese mice

[Source](https://doi.org/10.1039/d1fo04387k)

---

## 16S rRNA Gene Sequencing Reveals Specific Gut Microbes Common to Medicinal Insects.
 02.06.2022, _J Geng, Z Sui, W Dou, Y Miao, T Wang, X Wei, S Chen, Z Zhang, J Xiao, D Huang_


Insects have a long history of being used in medicine, with clear primary and secondary functions and less side effects, and the study and exploitation of medicinal insects have received increasing attention. Insects gut microbiota and their metabolites play an important role in protecting the hosts from other potentially harmful microbes, providing nutrients, promoting digestion and degradation, and regulating growth and metabolism of the hosts. However, there are still few studies linking the medicinal values of insects with their gut microbes. In this study, we focused on the specific gut microbiota common to medicinal insects, hoping to trace the potential connection between medicinal values and gut microbes of medicinal insects. Based on 16S rRNA gene sequencing data, we compared the gut microbiota of medicinal insects [Periplaneta americana, Protaetia (Liocola) brevitarsis (Lewis) and Musca domestica], in their medicinal stages, and non-medicinal insects (Hermetia illucens L., Tenebrio molitor, and Drosophila melanogaster), and found that the intestinal microbial richness of medicinal insects was higher, and there were significant differences in the microbial community structure between the two groups. We established a model using a random-forest method to preliminarily screen out several types of gut microbiota common to medicinal insects that may play medicinal values: Parabacteroides goldsteinii, Lactobacillus dextrinicus, Bifidobacterium longum subsp. infantis (B. infantis), and Vagococcus carniphilus. In particular, P. goldsteinii and B. infantis were most probably involved in the anti-inflammatory effects of medicinal insects. Our results revealed an association between medicinal insects and their gut microbes, providing new development directions and possibly potential tools for utilizing microbes to enhance the medicinal efficacy of medicinal insects.

[Source](https://doi.org/10.3389/fmicb.2022.892767)

10.3389/fmicb.2022.892767

---

## Antioxidant potential of <em>Pediococcus pentosaceus</em> strains from the sow milk bacterial collection in weaned piglets

[Source](https://doi.org/10.1186/s40168-022-01278-z)

---

## Composition and Diversity of Gut Bacterial Community in Different Life Stages of a Leaf Beetle <em>Gastrolina depressa</em>

[Source](https://doi.org/10.1007/s00248-022-02054-0)

---

## Gut microbiota-derived ursodeoxycholic acid from neonatal dairy calves improves intestinal homeostasis and colitis to attenuate extended-spectrum β-lactamase-producing enteroaggregative <em>Escherichia coli</em> infection

[Source](https://doi.org/10.1186/s40168-022-01269-0)

---

## Recovery of metagenome-assembled genomes from the phyllosphere of 110 rice genotypes.
 06.06.2022, _P Su, WA Wicaksono, C Li, K Michl, G Berg, D Wang, Y Xiao, R Huang, H Kang, D Zhang, T Cernava, Y Liu_


_Bacteria, Genome, Bacterial, Metagenome, Metagenomics, Oryza_
The plant microbiota plays crucial roles in sustaining plant health and productivity. Advancing plant microbiome research and designing sustainable practices for agriculture requires in-depth assessments of microorganisms associated with different host plants; however, there is little information on functional aspects of many microorganisms of interest. Therefore, we enriched microorganisms from the phyllosphere of 110 rice genotypes and subjected them to shotgun metagenomic sequencing to reconstruct bacterial genomes from the obtained datasets. The approach yielded a total of 1.34 terabases of shotgun-sequenced metagenomic data. By separately recovering bacterial genomes from each of the 110 rice genotypes, we recovered 569 non-redundant metagenome-assembled genomes (MAGs) with a completeness higher than 50% and contaminations less than 10%. The MAGs were primarily assigned to Alphaproteobacteria, Gammaproteobacteria, and Bacteroidia. The presented data provides an extended basis for microbiome analyses of plant-associated microorganisms. It is complemented by detailed metadata to facilitate implementations in ecological studies, biotechnological mining approaches, and comparative assessments with genomes or MAGs from other studies.

[Source](https://doi.org/10.1038/s41597-022-01320-7)

10.1038/s41597-022-01320-7

---

## Mercury drives microbial community assembly and ecosystem multifunctionality across a Hg contamination gradient in rice paddies.
 06.06.2022, _Q Pu, K Zhang, AJ Poulain, J Liu, R Zhang, MA Abdelhafiz, B Meng, X Feng_


_Biodiversity, Mercury, Microbiota, Oryza, Soil_
Soil microbial communities are critical for maintaining terrestrial ecosystems and fundamental ecological processes. Mercury (Hg) is a heavy metal that is toxic to microorganisms, but its effects on microbial community assembly and ecosystem multifunctionality in rice paddy ecosystems remain largely unknown. In the current study, we analyzed the microbial community structure and ecosystem multifunctionality of paddy soils across a Hg contamination gradient. The results demonstrated that Hg contamination significantly altered the microbial community structure. The microbial communities were predominantly driven by deterministic selection rather than stochastic processes. The random forest model and variation partition analysis demonstrated that the Hg level was the most important predictor of microbial profiles. Ecosystem multifunctionality decreased across the Hg concentration gradient, and multifunctionality was significantly correlated with soil biodiversity, suggesting that Hg-induced reductions in soil biodiversity led to reduced ecosystem services. A structural equation model showed that Hg contamination directly and indirectly affected ecosystem multifunctionality. The present work broadens our knowledge of the assembly of the microbiome in rice paddies across a Hg contamination gradient and highlights the significance of soil biodiversity in regulating ecosystem functions, especially in Hg-polluted rice paddies.

[Source](https://doi.org/10.1016/j.jhazmat.2022.129055)

10.1016/j.jhazmat.2022.129055

---

## The sanitary indoor environment-a potential source for intact human-associated anaerobes.
 03.06.2022, _MR Pausan, M Blohs, A Mahnert, C Moissl-Eichinger_


_Archaea, Humans, Microbiota, Oxygen, RNA, Ribosomal, 16S, Symbiosis_
A healthy human microbiome relies on the interaction with and exchange of microbes that takes place between the human body and its environment. People in high-income countries spend most of their time indoors and for this reason, the built environment (BE) might represent a potent source of commensal microbes. Anaerobic microbes are of particular interest, as researchers have not yet sufficiently clarified how the human microbiome acquires oxygen-sensitive microbes. We sampled the bathrooms in ten households and used propidium monoazide (PMA) to assess the viability of the collected prokaryotes. We compared the microbiome profiles based on 16S rRNA gene sequencing and confirmed our results by genetic and cultivation-based analyses. Quantitative and qualitative analysis revealed that most of the microbial taxa in the BE samples are human-associated. Less than 25% of the prokaryotic signatures originate from intact cells, indicating that aerobic and stress resistant taxa display an apparent survival advantage. However, we also confirmed the presence of intact, strictly anaerobic taxa on bathroom floors, including methanogenic archaea. As methanogens are regarded as highly sensitive to aerobic conditions, oxygen-tolerance experiments were performed with human-associated isolates to validate their survival. These results show that human-associated methanogens can survive oxic conditions for at least 6 h. We collected strong evidence that supports the hypothesis that obligate anaerobic taxa can survive in the BE for a limited amount of time. This suggests that the BE serves as a potential source of anaerobic human commensals.

[Source](https://doi.org/10.1038/s41522-022-00305-z)

10.1038/s41522-022-00305-z

---

## Magnetotactic bacteria and magnetofossils: ecology, evolution and environmental implications.
 03.06.2022, _P Goswami, K He, J Li, Y Pan, AP Roberts, W Lin_


_Bacteria, Ecosystem, Fresh Water, Prospective Studies_
Magnetotactic bacteria (MTB) are a group of phylogenetically diverse and morphologically varied microorganisms with a magnetoresponsive capability called magnetotaxis or microbial magnetoreception. MTB are a distinctive constituent of the microbiome of aquatic ecosystems because they use Earth's magnetic field to align themselves in a north or south facing direction and efficiently navigate to their favored microenvironments. They have been identified worldwide from diverse aquatic and waterlogged microbiomes, including freshwater, saline, brackish and marine ecosystems, and some extreme environments. MTB play important roles in the biogeochemical cycling of iron, sulphur, phosphorus, carbon and nitrogen in nature and have been recognized from in vitro cultures to sequester heavy metals like selenium, cadmium, and tellurium, which makes them prospective candidate organisms for aquatic pollution bioremediation. The role of MTB in environmental systems is not limited to their lifespan; after death, fossil magnetosomal magnetic nanoparticles (known as magnetofossils) are a promising proxy for recording paleoenvironmental change and geomagnetic field history. Here, we summarize the ecology, evolution, and environmental function of MTB and the paleoenvironmental implications of magnetofossils in light of recent discoveries.

[Source](https://doi.org/10.1038/s41522-022-00304-0)

10.1038/s41522-022-00304-0

---

## Emerging technologies in the study of the virome

[Source](https://doi.org/10.1016/j.coviro.2022.101231)

---

## EasyMap - An Interactive Web Tool for Evaluating and Comparing Associations of Clinical Variables and Microbiome Composition.
 03.06.2022, _E Dahan, VM Martin, M Yassour_


_Humans, Metadata, Microbiota, Multivariate Analysis_
One of the most common tasks in microbiome studies is comparing microbial profiles across various groups of people (e.g., sick vs. healthy). Routinely, researchers use multivariate linear regression models to address these challenges, such as linear regression packages, MaAsLin2, LEfSe, etc. In many cases, it is unclear which metadata variables should be included in the linear model, as many human-associated variables are correlated with one another. Thus, multiple models are often tested, each including a different set of variables, however the challenge of selecting the metadata variables in the final model remains. Here, we present EasyMap, an interactive online tool allowing for (1) running multiple multivariate linear regression models, on the same features and metadata; (2) visualizing the associations between microbial features and clinical metadata found in each model; and (3) comparing across the various models to identify the critical metadata variables and select the optimal model. EasyMap provides a side-by-side visualization of association results across the various models, each with additional metadata variables, enabling us to evaluate the impact of each metadata variable on the associated feature. EasyMap's interface enables filtering associations by significance, focusing on specific microbes and finding the robust associations that are found across multiple models. While EasyMap was designed to analyze microbiome data, it can handle any other tabular data with numeric features and metadata variables. EasyMap takes the common task of multivariate linear regression to the next level, with an intuitive and simple user interface, allowing for wide comparisons of multiple models to identify the robust microbial feature associations. EasyMap is available at http://yassour.rcs.huji.ac.il/easymap.

[Source](https://doi.org/10.3389/fcimb.2022.854164)

10.3389/fcimb.2022.854164

---

## Effect of cryopreservation medium conditions on growth and isolation of gut anaerobes from human faecal samples.
 07.06.2022, _A Biclot, GRB Huys, R Bacigalupe, K D'hoe, D Vandeputte, G Falony, RY Tito, J Raes_


_Cryopreservation, Culture Media, Dimethyl Sulfoxide, Feces, Humans, RNA, Ribosomal, 16S_
Novel strategies for anaerobic bacterial isolations from human faecal samples and various initiatives to generate culture collections of gut-derived bacteria have instigated considerable interest for the development of novel microbiota-based treatments. Early in the process of building a culture collection, optimal faecal sample preservation is essential to safeguard the viability of the broadest taxonomic diversity range possible. In contrast to the much more established faecal storage conditions for meta-omics applications, the impact of stool sample preservation conditions on bacterial growth recovery and isolation remains largely unexplored. In this study, aliquoted faecal samples from eleven healthy human volunteers selected based on a range of physicochemical and microbiological gradients were cryopreserved at - 80 °C either without the addition of any medium (dry condition) or in different Cary-Blair medium conditions with or without a cryoprotectant, i.e. 20% (v/v) glycerol or 5% (v/v) DMSO. Faecal aliquots were subjected to bulk 16S rRNA gene sequencing as well as dilution plating on modified Gifu Anaerobic Medium after preservation for culturable fraction profiling and generation of bacterial culture collections.

[Source](https://doi.org/10.1186/s40168-022-01267-2)

10.1186/s40168-022-01267-2

---

## BiGAMi: Bi-Objective Genetic Algorithm Fitness Function for Feature Selection on Microbiome Datasets.
 02.06.2022, _M Leske, F Bottacini, H Afli, BGN Andrade_


The relationship between the host and the microbiome, or the assemblage of microorganisms (including bacteria, archaea, fungi, and viruses), has been proven crucial for its health and disease development. The high dimensionality of microbiome datasets has often been addressed as a major difficulty for data analysis, such as the use of machine-learning (ML) and deep-learning (DL) models. Here, we present BiGAMi, a bi-objective genetic algorithm fitness function for feature selection in microbial datasets to train high-performing phenotype classifiers. The proposed fitness function allowed us to build classifiers that outperformed the baseline performance estimated by the original studies by using as few as 0.04% to 2.32% features of the original dataset. In 35 out of 42 performance comparisons between BiGAMi and other feature selection methods evaluated here (sequential forward selection, SelectKBest, and GARS), BiGAMi achieved its results by selecting 6-93% fewer features. This study showed that the application of a bi-objective GA fitness function against microbiome datasets succeeded in selecting small subsets of bacteria whose contribution to understood diseases and the host state was already experimentally proven. Applying this feature selection approach to novel diseases is expected to quickly reveal the microbes most relevant to a specific condition.

[Source](https://doi.org/10.3390/mps5030042)

10.3390/mps5030042

---

