## Public human microbiome data are dominated by highly developed countries.
 19.02.2022, _RJ Abdill, EM Adamowicz, R Blekhman_


The importance of sampling from globally representative populations has been well established in human genomics. In human microbiome research, however, we lack a full understanding of the global distribution of sampling in research studies. This information is crucial to better understand global patterns of microbiome-associated diseases and to extend the health benefits of this research to all populations. Here, we analyze the country of origin of all 444,829 human microbiome samples that are available from the world's 3 largest genomic data repositories, including the Sequence Read Archive (SRA). The samples are from 2,592 studies of 19 body sites, including 220,017 samples of the gut microbiome. We show that more than 71% of samples with a known origin come from Europe, the United States, and Canada, including 46.8% from the US alone, despite the country representing only 4.3% of the global population. We also find that central and southern Asia is the most underrepresented region: Countries such as India, Pakistan, and Bangladesh account for more than a quarter of the world population but make up only 1.8% of human microbiome samples. These results demonstrate a critical need to ensure more global representation of participants in microbiome studies.

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001536)

10.1371/journal.pbio.3001536

---

## Beyond the Microbiome: Germ-ganism? An Integrative Idea for Microbial Existence, Organization, Growth, Pathogenicity, and Therapeutics.
 07.03.2022, _ME Kambouris, S Goudoudaki, S Kritikou, A Milioni, K Karamperis, I Giavasis, GP Patrinos, A Velegraki, Y Manoussopoulos_


The advances made by microbiome research call for new vocabulary and expansion of our thinking in microbiology. For example, the life-forms presenting in both unicellular and multicellular formats invite us to rethink microbial existence, organization, growth, pathogenicity, and therapeutics in the 21st century. A view of such populations as parts of single organisms with a loose, distributed multicellular organization, introduced here as a germ-ganism, rather than communities, might open up interesting prospects for diagnostics and therapeutics innovation. This study tested and further contextualized the concept of germ-ganism using solid cultures of bacteria and fungi. Based on our findings and the literature reviewed herein, we propose that germ-ganism has synergy with a systems medicine approach by broadening host-environment interactions from cells and microorganisms to a scale of biological ecosystems. Germ-ganism also brings about the possibility of studying the multilevel impacts of novel therapeutic agents within and across networks of microbial ecosystems. The germ-ganism would lend itself, in the long term, to a veritable biocybernetics system, while in the mid-term, we anticipate it will contribute to new diagnostics and therapeutics. Biosecurity applications would be immensely affected by germ-ganism. Industrial applications of germ-ganism are of interest as a more sustainable alternative to costly solutions such as tampered strains/microorganisms. In conclusion, germ-ganism is informed by lessons from microbiome research and invites rethinking microbial existence, organization, and growth as an organism. Germ-ganism has vast ramifications for understanding pathogenicity, and clinical, biosecurity, and biotechnology applications in the current historical moment of the COVID-19 pandemic and beyond.

[Source](https://www.liebertpub.com/doi/10.1089/omi.2022.0015)

10.1089/omi.2022.0015

---

## Quantifying and Cataloguing Unknown Sequences within Human Microbiomes.
 08.03.2022, _S Modha, DL Robertson, J Hughes, RJ Orton_


Advances in genome sequencing technologies and lower costs have enabled the exploration of a multitude of known and novel environments and microbiomes. This has led to an exponential growth in the raw sequence data that are deposited in online repositories. Metagenomic and metatranscriptomic data sets are typically analysed with regard to a specific biological question. However, it is widely acknowledged that these data sets are comprised of a proportion of sequences that bear no similarity to any currently known biological sequence, and this so-called "dark matter" is often excluded from downstream analyses. In this study, a systematic framework was developed to assemble, identify, and measure the proportion of unknown sequences present in distinct human microbiomes. This framework was applied to 40 distinct studies, comprising 963 samples, and covering 10 different human microbiomes including fecal, oral, lung, skin, and circulatory system microbiomes. We found that while the human microbiome is one of the most extensively studied, on average 2% of assembled sequences have not yet been taxonomically defined. However, this proportion varied extensively among different microbiomes and was as high as 25% for skin and oral microbiomes that have more interactions with the environment. A rate of taxonomic characterization of 1.64% of unknown sequences being characterized per month was calculated from these taxonomically unknown sequences discovered in this study. A cross-study comparison led to the identification of similar unknown sequences in different samples and/or microbiomes. Both our computational framework and the novel unknown sequences produced are publicly available for future cross-referencing. Our approach led to the discovery of several novel viral genomes that bear no similarity to sequences in the public databases. Some of these are widespread as they have been found in different microbiomes and studies. Hence, our study illustrates how the systematic characterization of unknown sequences can help the discovery of novel microbes, and we call on the research community to systematically collate and share the unknown sequences from metagenomic studies to increase the rate at which the unknown sequence space can be classified.

[Source](https://journals.asm.org/doi/10.1128/msystems.01468-21)

10.1128/msystems.01468-21

---

## Alterations of lung microbiota in patients with non-small cell lung cancer.
 08.03.2022, _W Zeng, C Zhao, M Yu, H Chen, Y Pan, Y Wang, H Bao, H Ma, S Ma_


The role of lung microbiota in non-small cell lung cancer remains unclear. We investigated the characteristics and functional roles of lung microbiota in non-small cell lung cancer. Bronchoalveolar lavage fluid samples were obtained from patients with non-small cell lung cancer (n = 46) and with benign lung disease (n = 29). The differences in composition and gene expression in the microbiota between the samples were analyzed using 16s rRNA sequencing. The oncogenic genus (Veillonella) was then evaluated in the progression of lung cancer in C57 BL/6 mice. Compared to benign lung disease, the lung microbiota in non-small cell lung cancer was significantly altered, both in terms of α- and β-diversity. In terms of bacterial composition, the non-small cell lung cancer group was enriched with two Phyla (Firmicutes, Bacteroidetes) and three genera (Streptococcus, Prevotella, Veillonella). Prevotella and Veillonella were most strongly associated with non-small cell lung cancer, and Veillonella significantly promoted the progression of lung cancer in vivo. Moreover, metabolic prediction revealed that ribosomes, biosynthesis of secondary metabolites, and pyrimidine metabolism were among the enriched pathways that may be involved in the progression of non-small cell lung cancer. Overall, results suggest that the progression of non-small cell lung cancer is followed by significant changes in the composition and function of the lung microbiota. These differing genera may be potential diagnostic markers and therapeutic targets.

[Source](https://www.tandfonline.com/doi/full/10.1080/21655979.2022.2045843)

10.1080/21655979.2022.2045843

---

## Evaluation of microbiome enrichment and host DNA depletion in human vaginal samples using Oxford Nanopore's adaptive sequencing.
 08.03.2022, _M Marquet, J Zöllkau, J Pastuschek, A Viehweger, E Schleußner, O Makarewicz, MW Pletz, R Ehricht, C Brandt_


Metagenomic sequencing is promising for clinical applications to study microbial composition concerning disease or patient outcomes. Alterations of the vaginal microbiome are associated with adverse pregnancy outcomes, like preterm premature rupture of membranes and preterm birth. Methodologically these samples often have to deal with low relative amounts of prokaryotic DNA and high amounts of host DNA (&gt; 90%), decreasing the overall microbial resolution. Nanopore's adaptive sampling method offers selective DNA depletion or target enrichment to directly reject or accept DNA molecules during sequencing without specialized sample preparation. Here, we demonstrate how selective 'human host depletion' resulted in a 1.70 fold (± 0.27 fold) increase in total sequencing depth, providing higher taxonomic profiling sensitivity. At the same time, the microbial composition remains consistent with the control experiments. The complete removal of all human host sequences is not yet possible and should be considered as an ethical approval statement might still be necessary. Adaptive sampling increased microbial sequencing yield in all 15 sequenced clinical routine vaginal samples, making it a valuable tool for clinical surveillance and medical-based research, which can be used in addition to other host depletion methods before sequencing.

[Source](https://www.nature.com/articles/s41598-022-08003-8)

10.1038/s41598-022-08003-8

---

## Personal diet-microbiota interactions and weight loss.
 08.03.2022, _HM Roager, LH Christensen_


The aim of this review is to provide an overview of how person-specific interactions between diet and the gut microbiota could play a role in affecting diet-induced weight loss responses. The highly person-specific gut microbiota, which is shaped by our diet, secretes digestive enzymes and molecules that affect digestion in the colon. Therefore, weight loss responses could in part depend on personal colonic fermentation responses, which affect energy extraction of food and production of microbial metabolites, such as short-chain fatty acids (SCFAs), which exert various effects on host metabolism. Colonic fermentation is the net result of the complex interplay between availability of dietary substrates, the functional capacity of the gut microbiome and environmental (abiotic) factors in the gut such as pH and transit time. While animal studies have demonstrated that the gut microbiota can causally affect obesity, causal and mechanistic evidence from human studies is still largely lacking. However, recent human studies have proposed that the baseline gut microbiota composition may predict diet-induced weight loss-responses. In particular, individuals characterised by high relative abundance of Prevotella have been found to lose more weight on diets rich in dietary fibre compared to individuals with low Prevotella abundance. Although harnessing of personal diet-microbiota interactions holds promise for more personalised nutrition and obesity management strategies to improve human health, there is currently insufficient evidence to unequivocally link the gut microbiota and weight loss in human subjects. To move the field forward, a greater understanding of the mechanistic underpinnings of personal diet-microbiota interactions is needed.

[Source](https://www.cambridge.org/core/journals/proceedings-of-the-nutrition-society/article/personal-dietmicrobiota-interactions-and-weight-loss/FBF6F475097A485F90DFF79A43438657)

10.1017/S0029665122000805

---

## Integrated analysis of gut microbiome and host immune responses in COVID-19.
 08.03.2022, _X Xu, W Zhang, M Guo, C Xiao, Z Fu, S Yu, L Jiang, S Wang, Y Ling, F Liu, Y Tan, S Chen_


Emerging evidence indicates that the gut microbiome contributes to the host immune response to infectious diseases. Here, to explore the role of the gut microbiome in the host immune responses in COVID-19, we conducted shotgun metagenomic sequencing and immune profiling of 14 severe/critical and 24 mild/moderate COVID-19 cases as well as 31 healthy control samples. We found that the diversity of the gut microbiome was reduced in severe/critical COVID-19 cases compared to mild/moderate ones. We identified the abundance of some gut microbes altered post-SARS-CoV-2 infection and related to disease severity, such as Enterococcus faecium, Coprococcus comes, Roseburia intestinalis, Akkermansia muciniphila, Bacteroides cellulosilyticus and Blautia obeum. We further analyzed the correlation between the abundance of gut microbes and host responses, and obtained a correlation map between clinical features of COVID-19 and 16 severity-related gut microbe, including Coprococcus comes that was positively correlated with CD3+/CD4+/CD8+ lymphocyte counts. In addition, an integrative analysis of gut microbiome and the transcriptome of peripheral blood mononuclear cells (PBMCs) showed that genes related to viral transcription and apoptosis were up-regulated in Coprococcus comes low samples. Moreover, a number of metabolic pathways in gut microbes were also found to be differentially enriched in severe/critical or mild/moderate COVID-19 cases, including the superpathways of polyamine biosynthesis II and sulfur oxidation that were suppressed in severe/critical COVID-19. Together, our study highlighted a potential regulatory role of severity related gut microbes in the immune response of host.

[Source](https://link.springer.com/article/10.1007/s11684-022-0921-6)

10.1007/s11684-022-0921-6

---

## Gut microbiome modifications over time when removing in-feed antibiotics from the prophylaxis of post-weaning diarrhea in piglets.
 07.03.2022, _P Cremonesi, F Biscarini, B Castiglioni, CA Sgoifo, R Compiani, P Moroni_


Post-weaning diarrhea (PWD) in pigs has mainly an infectious basis and control strategies are centred on antibiotics added to the diet. Given concerns on the spread of multi-resistant bacteria, it is necessary to develop alternative prophylactic approaches to control PWD in piglets. The most promising alternative strategies are based on substances that act indirectly on the bacteria by stimulating the immune system or by improving gut health. The aim of this study was to evaluate the effect on the gut microbiota of feed supplemented with a mixture of essential oils (garlic and oregano) in weaning piglets, compared to traditional PWD management (in-feed antibiotics) and to a control group without any diet supplementation. The study involved 197 piglets from 18 litters in a single farm. The piglets were followed from birth to day 58 of age and were weaned at day 26. During the experimental period, the animals were monitored for weight and growth, average daily gain, morbidity and mortality. For the metataxonomics analysis, rectal samples were collected from 17 piglets from the three experimental groups at 4 different time-points (days 1, 12, 26 and 58). Results revealed that the gut microbiota in pre- and post-weaning piglets was dominated by the phyla Firmicutes (51%), Bacteroidetes (25%) and Proteobacteria (16%), which together make up for over 90% of the entire piglet core gut microbiota. The core microbiota comprised 10 taxa before weaning and 43 taxa after weaning, with 7 taxa overlapping between timepoints: two of them (Prevotella 9, p-value = 0.00095; Solobacterium p-value = 0.00821) were significantly more abundant after weaning. All alpha diversity indexes were significantly different between pre- and post-weaning, while only Shannon and Simpson diversity and equitability were significantly different between treatments. Based on the matrix of Bray-Curtis dissimilarities, samples showed clear clustering per timepoint (before and after weaning, p-value &lt; 0.001) and between treatments by timepoint (p-value = 0.0086). The oil-diet group showed a consistently higher F:B ratio at all timepoints. These results show that the pig gut microbiota changes significantly with weaning, and suggest that the use of essential oils as feed supplementation to control PWD does not seem to alter sgnificantly the microbiota nor the growth parameters of piglets, however modifications of specific taxa may occur.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0262199)

10.1371/journal.pone.0262199

---

## Complete Genome Sequences of Three Staphylococcus haemolyticus Strains Isolated from the Lung of a TGFβ1 Transgenic Mouse with Lung Fibrosis.
 07.03.2022, _AM Abdel-Hamid, CN D'Alessandro-Gabazza, T Yasuma, KKO Walden, CJ Fields, EC Gabazza, I Cann_


We report here the complete genome sequences of three Staphylococcus haemolyticus strains isolated from a mouse fibrotic lung tissue and exhibiting proapoptotic activity on human lung alveolar epithelial cells. The genomes were obtained from a combination of Illumina MiSeq and Oxford Nanopore MinION sequencing.

[Source](https://journals.asm.org/doi/10.1128/mra.01176-21)

10.1128/mra.01176-21

---

## Colitis-associated microbiota drives changes in behaviour in male mice in the absence of inflammation

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0889159122000691)

---

## Comprehensive mouse microbiota genome catalog reveals major difference to its human counterpart.
 08.03.2022, _S Kieser, EM Zdobnov, M Trajkovski_


Mouse is the most used model for studying the impact of microbiota on its host, but the repertoire of species from the mouse gut microbiome remains largely unknown. Accordingly, the similarity between human and mouse microbiomes at a low taxonomic level is not clear. We construct a comprehensive mouse microbiota genome (CMMG) catalog by assembling all currently available mouse gut metagenomes and combining them with published reference and metagenome-assembled genomes. The 41'798 genomes cluster into 1'573 species, of which 78.1% are uncultured, and we discovered 226 new genera, seven new families, and one new order. CMMG enables an unprecedented coverage of the mouse gut microbiome exceeding 86%, increases the mapping rate over four-fold, and allows functional microbiota analyses of human and mouse linking them to the driver species. Comparing CMMG to microbiota from the unified human gastrointestinal genomes shows an overlap of 62% at the genus but only 10% at the species level, demonstrating that human and mouse gut microbiota are largely distinct. CMMG contains the most comprehensive collection of consistently functionally annotated species of the mouse and human microbiome to date, setting the ground for the analysis of new and reanalysis of existing datasets at an unprecedented depth.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009947)

10.1371/journal.pcbi.1009947

---

## The microbiome and resistome of apple fruits alter in the post-harvest period.
 08.03.2022, _B Wassermann, A Abdelfattah, H Müller, L Korsten, G Berg_


A detailed understanding of antimicrobial resistance trends among all human-related environments is key to combat global health threats. In food science, however, the resistome is still little considered. Here, we studied the apple microbiome and resistome from different cultivars (Royal Gala and Braeburn) and sources (freshly harvested in South Africa and exported apples in Austrian supermarkets) by metagenomic approaches, genome reconstruction and isolate sequencing.

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-022-00402-8)

10.1186/s40793-022-00402-8

---

## Effects of Ocean Warming on the Underexplored Members of the Coral Microbiome.
 08.03.2022, _J Maire, P Buerger, WY Chan, P Deore, AM Dungan, MR Nitschke, MJH van Oppen_


The climate crisis is one of the most significant threats to marine ecosystems. It is leading to severe increases in sea surface temperatures and in the frequency and magnitude of marine heatwaves. These changing conditions are directly impacting coral reef ecosystems, which are among the most biodiverse ecosystems on Earth. Coral-associated symbionts are particularly affected because summer heatwaves cause coral bleaching-the loss of endosymbiotic microalgae (Symbiodiniaceae) from coral tissues, leading to coral starvation and death. Coral-associated Symbiodiniaceae and bacteria have been extensively studied in the context of climate change, especially in terms of community diversity and dynamics. However, data on other microorganisms and their response to climate change are scarce. Here, we review current knowledge on how increasing temperatures affect understudied coral-associated microorganisms such as archaea, fungi, viruses, and protists other than Symbiodiniaceae, as well as microbe-microbe interactions. We show that the coral-microbe symbiosis equilibrium is at risk under current and predicted future climate change and argue that coral reef conservation initiatives should include microbe-focused approaches.

[Source](https://academic.oup.com/icb/advance-article/doi/10.1093/icb/icac005/6544673)

10.1093/icb/icac005

---

## A comparison of six DNA extraction protocols for 16S, ITS, and shotgun metagenomic sequencing of microbial communities

[Source](https://www.biorxiv.org/content/10.1101/2022.03.07.483343v1.abstract)

---

## A comparison of hard and soft direct methods for DNA extraction from soil

[Source](https://www.biorxiv.org/content/10.1101/2022.03.07.483395v1.abstract)

---

