## A two-sample mendelian randomization analysis investigates associations between gut microbiota and infertility.
 17.07.2023, _T Li, W Shao, Y Wang, R Zhou, Z Yun, Y He, Y Wu_


_Female, Male, Humans, Infertility, Female, Mendelian Randomization Analysis, Gastrointestinal Microbiome, Genome-Wide Association Study, Infertility, Male_
Observational studies have provided evidence of a correlation between alterations in gut microbiota composition and infertility. However, concrete proof supporting the causal relationship is still lacking. We performed a Mendelian randomization study to assess whether genetically gut microbiota composition influences the risk of infertility. The genetic data pertaining to gut microbiota were obtained from a genome-wide association study meta-analysis, which was conducted among 24 cohorts (18,340 participants) from the international MiBioGen consortium. By the primary method of assessing causality, we have identified 2 family taxa, 2 genus taxa, and 1 order taxa that were linked to a low risk of male infertility, while 1 genus taxa were associated with a high risk of male infertility. Furthermore, we have discovered 6 genus taxa, 1 phylum taxa, 1 class taxa, 1 order taxa, and 1 family taxa that were associated with a low risk of female infertility, while 1 genus taxa were linked to a high risk of female infertility. This study successfully confirmed that there was a causal link between gut microbiota and infertility. The identification of these specific strains through genetic prediction offers a valuable insight for early diagnosis, prevention, and treatment of infertility.

[Source](https://doi.org/10.1038/s41598-023-38624-6)

10.1038/s41598-023-38624-6

---

## Targeting gut microbiota-derived kynurenine to predict and protect the remodeling of the pressure-overloaded young heart.
 17.07.2023, _B Shi, X Zhang, Z Song, Z Dai, K Luo, B Chen, Z Zhou, Y Cui, B Feng, Z Zhu, J Zheng, H Zhang, X He_


_Animals, Mice, Kynurenine, Gastrointestinal Microbiome, Heart, Fibroblasts, Metabolomics_
Pressure-overloaded left ventricular remodeling in young population is progressive and readily degenerate into heart failure. The aims of this study were to identify a plasma metabolite that predicts and is mechanistically linked to the disease. Untargeted metabolomics determined elevated plasma kynurenine (Kyn) in both the patient cohorts and the mice model, which was correlated with remodeling parameters. In vitro and in vivo evidence, combined with single-nucleus RNA sequencing (snRNA-seq), demonstrated that Kyn affected both cardiomyocytes and cardiac fibroblasts by activating aryl hydrocarbon receptors (AHR) to up-regulate hypertrophy- and fibrosis-related genes. Shotgun metagenomics and fecal microbiota transplantation revealed the existence of the altered gut microbiota-Kyn relationship. Supplementation of selected microbes reconstructed the gut microbiota, reduced plasma Kyn, and alleviated ventricular remodeling. Our data collectively discovered a gut microbiota-derived metabolite to activate AHR and its gene targets in remodeling young heart, a process that could be prevented by specific gut microbiota modulation.

[Source](https://doi.org/10.1126/sciadv.adg7417)

10.1126/sciadv.adg7417

---

## Intratumoral Microbiota Composition Regulates Chemoimmunotherapy Response in Esophageal Squamous Cell Carcinoma.
 11.07.2023, _H Wu, X Leng, Q Liu, T Mao, T Jiang, Y Liu, F Li, C Cao, J Fan, L Chen, Y Chen, Q Yao, S Lu, R Liang, L Hu, M Liu, Y Wan, Z Li, J Peng, Q Luo, H Zhou, J Yin, K Xu, M Lan, X Peng, H Lan, G Li, Y Han, X Zhang, ZJ Xiao, J Lang, G Wang, C Xu_


Neoadjuvant chemoimmunotherapy (NACI) has shown promise in the treatment of resectable esophageal squamous cell carcinoma (ESCC). The microbiomes of patients can impact therapy response, and previous studies have demonstrated that intestinal microbiota influences cancer immunotherapy by activating gut immunity. Here, we investigated the effects of intratumoral microbiota on the response of ESCC patients to NACI. Intra-tumoral microbiota signatures of β-diversity were disparate and predicted the treatment efficiency of NACI. The enrichment of Streptococcus positively correlated with GrzB+ and CD8+ T cell infiltration in tumor tissues. The abundance of Streptococcus could predict prolonged disease-free survival in ESCC. Single-cell RNA sequencing (scRNA-seq) demonstrated that responders displayed a higher proportion of CD8+ effector memory T cells but a lower proportion of CD4+ Treg cells. Mice that underwent fecal microbial transplantation (FMT) or intestinal colonization with Streptococcus from responders showed enrichment of Streptococcus in tumor tissues, elevated tumor-infiltrating CD8+ T cells, and a favorable response to anti-PD-1 treatment. Collectively, this study suggests that intratumoral Streptococcus signatures could predict NACI response and sheds light on the potential clinical utility of intratumoral microbiota for cancer immunotherapy.

[Source](https://doi.org/10.1158/0008-5472.CAN-22-2593)

10.1158/0008-5472.CAN-22-2593

---

## Metabolic and inflammatory profiles, gut microbiota and lifestyle factors in overweight and normal weight young thai adults.
 17.07.2023, _S Somnuk, S Komindr, S Monkhai, T Poolsawat, M Nakphaichit, B Wanikorn_


_Young Adult, Humans, Adult, Overweight, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Southeast Asian People, Body Mass Index, Biomarkers, Insulin, Life Style_
Obesity among young adults, especially those living in developing countries is increasing. A high body mass index (BMI) is one of the major causes of several diseases worldwide, constituting an important risk factor for non-communicable diseases (NCDs). Investigations describing the relationship between BMI, clinical and gut microbiota characteristics and lifestyle factors of overweight young adults, especially from Southeast Asian countries are limited. Metabolic and inflammatory biomarkers, fecal microbiota profiles and lifestyle factors were compared between overweight Thai young adults (n = 30, mean age 33 ± 9.48) and those with normal weight (n = 30, mean age 27 ±7.50). This study was registered with the Thai Clinical Trials Registry (TCTR20220204007). Health status including body composition, fasting glucose and insulin, lipid profiles, liver and kidney function, inflammatory biomarkers, blood pressure and fecal microbiota using 16S rRNA gene sequencing data was determined. Dietary intake was assessed using a 3-day dietary record and a food frequency questionnaire (FFQ), with physical activity levels compared using the international physical activity questionnaire (IPAQ). The overweight group had significantly higher BMI, waist-hip ratio, body fat mass, % body fat, skeletal mass, triglyceride level, C-reactive protein, insulin and blood pressure, with lower levels of high-density lipoprotein cholesterol (HDL-C) and blood urea nitrogen compared to the normal weight group. Significant differences in fecal microbiota composition at the family and genus levels were observed between the two groups. In our clinical setting, we also observed that unhealthy diets with high consumption of food rich in fat and sugar, processed meat and alcohol, and physical inactivity were associated with an increased prevalence of overweight in Thai young adults. Results provided the big picture of health and lifestyle characteristics of overweight young Thai people. Young adults should be encouraged to engage in health-promoting activities that maintain healthy bodyweight.

[Source](https://doi.org/10.1371/journal.pone.0288286)

10.1371/journal.pone.0288286

---

## Metagenomic assessment of gut microbial communities and risk of severe COVID-19.
 14.07.2023, _LH Nguyen, D Okin, DA Drew, VM Battista, SJ Jesudasen, TM Kuntz, A Bhosle, KN Thompson, T Reinicke, CH Lo, JE Woo, A Caraballo, L Berra, J Vieira, CY Huang, U Das Adhikari, M Kim, HY Sui, M Magicheva-Gupta, L McIver, MB Goldberg, DS Kwon, C Huttenhower, AT Chan, PS Lai_


_Humans, Post-Acute COVID-19 Syndrome, COVID-19, Microbiota, Metagenome, Gastrointestinal Microbiome_
The gut microbiome is a critical modulator of host immunity and is linked to the immune response to respiratory viral infections. However, few studies have gone beyond describing broad compositional alterations in severe COVID-19, defined as acute respiratory or other organ failure.

[Source](https://doi.org/10.1186/s13073-023-01202-6)

10.1186/s13073-023-01202-6

---

## Longitudinal flux balance analyses of a patient with episodic colonic inflammation reveals microbiome metabolic dynamics.
 15.07.2023, _A Basile, A Heinken, J Hertel, L Smarr, W Li, L Treu, G Valle, S Campanaro, I Thiele_


_Humans, Male, Gastrointestinal Microbiome, Microbiota, Inflammation, Liver, Anti-Bacterial Agents, Escherichia coli_
We report the first use of constraint-based microbial community modeling on a single individual with episodic inflammation of the gastrointestinal tract, who has a well documented set of colonic inflammatory biomarkers, as well as metagenomically-sequenced fecal time series covering seven dates over 16 months. Between the first two time steps the individual was treated with both steroids and antibiotics. Our methodology enabled us to identify numerous time-correlated microbial species and metabolites. We found that the individual's dynamical microbial ecology in the disease state led to time-varying in silico overproduction, compared to healthy controls, of more than 24 biologically important metabolites, including methane, thiamine, formaldehyde, trimethylamine N-oxide, folic acid, serotonin, histamine, and tryptamine. The microbe-metabolite contribution analysis revealed that some Dialister species changed metabolic pathways according to the inflammation phases. At the first time point, characterized by the highest levels of serum (complex reactive protein) and fecal (calprotectin) inflammation biomarkers, they produced L-serine or formate. The production of the compounds, through a cascade effect, was mediated by the interaction with pathogenic Escherichia coli strains and Desulfovibrio piger. We integrated the microbial community metabolic models of each time point with a male whole-body, organ-resolved model of human metabolism to track the metabolic consequences of dysbiosis at different body sites. The presence of D. piger in the gut microbiome influenced the sulfur metabolism with a domino effect affecting the liver. These results revealed large longitudinal variations in an individual's gut microbiome ecology and metabolite production, potentially impacting other organs in the body. Future simulations with more time points from an individual could permit us to assess how external drivers, such as diet change or medical interventions, drive microbial community dynamics.

[Source](https://doi.org/10.1080/19490976.2023.2226921)

10.1080/19490976.2023.2226921

---

## Exploring Co-occurrence patterns and microbial diversity in the lung microbiome of patients with non-small cell lung cancer.
 14.07.2023, _S Najafi, SA Jamalkandi, A Najafi, J Salimian, A Ahmadi_


_Humans, Carcinoma, Non-Small-Cell Lung, Lung Neoplasms, Actinobacteria, Bifidobacterium, Microbiota, Lung_
It has been demonstrated in the literature that a dysbiotic microbiome could have a negative impact on the host immune system and promote disease onset or exacerbation. Co-occurrence networks have been widely adopted to identify biomarkers and keystone taxa in the pathogenesis of microbiome-related diseases. Despite the promising results that network-driven approaches have led to in various human diseases, there is a dearth of research pertaining to key taxa that contribute to the pathogenesis of lung cancer. Therefore, our primary goal in this study is to explore co-existing relationships among members of the lung microbial community and any potential gained or lost interactions in lung cancer.

[Source](https://doi.org/10.1186/s12866-023-02931-9)

10.1186/s12866-023-02931-9

---

## Aerobic exercise training and gut microbiome-associated metabolic shifts in women with overweight: a multi-omic study.
 14.07.2023, _JE Hintikka, JP Ahtiainen, P Permi, S Jalkanen, M Lehtonen, S Pekkala_


_Adult, Humans, Female, Overweight, Gastrointestinal Microbiome, Multiomics, Exercise, Obesity, Lecithins_
Physical activity is essential in weight management, improves overall health, and mitigates obesity-related risk markers. Besides inducing changes in systemic metabolism, habitual exercise may improve gut's microbial diversity and increase the abundance of beneficial taxa in a correlated fashion. Since there is a lack of integrative omics studies on exercise and overweight populations, we studied the metabolomes and gut microbiota associated with programmed exercise in obese individuals. We measured the serum and fecal metabolites of 17 adult women with overweight during a 6-week endurance exercise program. Further, we integrated the exercise-responsive metabolites with variations in the gut microbiome and cardiorespiratory parameters. We found clear correlation with several serum and fecal metabolites, and metabolic pathways, during the exercise period in comparison to the control period, indicating increased lipid oxidation and oxidative stress. Especially, exercise caused co-occurring increase in levels of serum lyso-phosphatidylcholine moieties and fecal glycerophosphocholine. This signature was associated with several microbial metagenome pathways and the abundance of Akkermansia. The study demonstrates that, in the absence of body composition changes, aerobic exercise can induce metabolic shifts that provide substrates for beneficial gut microbiota in overweight individuals.

[Source](https://doi.org/10.1038/s41598-023-38357-6)

10.1038/s41598-023-38357-6

---

## Glycogen availability and pH variation in a medium simulating vaginal fluid influence the growth of vaginal <em>Lactobacillus</em> species and <em>Gardnerella vaginalis</em>

[Source](https://doi.org/10.1186/s12866-023-02916-8)

---

## Antibiotic-induced gut microbiota dysbiosis has a functional impact on purine metabolism

[Source](https://doi.org/10.1186/s12866-023-02932-8)

---

## Coarse graining the human gut microbiome

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(23)00224-X)

---

## Intestinal mucus and their glycans: A habitat for thriving microbiota.
 17.07.2023, _AS Luis, GC Hansson_


_Intestinal Mucosa, Mucin-2, Intestines, Mucus, Mucins, Microbiota, Polysaccharides_
The colon mucus layer is organized with an inner colon mucus layer that is impenetrable to bacteria and an outer mucus layer that is expanded to allow microbiota colonization. A major component of mucus is MUC2, a glycoprotein that is extensively decorated, especially with O-glycans. In the intestine, goblet cells are specialized in controlling glycosylation and making mucus. Some microbiota members are known to encode multiple proteins that are predicted to bind and/or cleave mucin glycans. The interactions between commensal microbiota and host mucins drive intestinal colonization, while at the same time, the microbiota can utilize the glycans on mucins and affect the colonic mucus properties. This review will examine this interaction between commensal microbes and intestinal mucins and discuss how this interplay affects health and disease.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(23)00219-6)

10.1016/j.chom.2023.05.026

---

## https://www.cell.com/trends/immunology/fulltext/S1471-4906(23)00111-4

[Source](https://doi.org/10.1016/j.it.2023.06.003)

---

## A comprehensive perspective on the interaction between gut microbiota and COVID-19 vaccines.
 14.07.2023, _M Hong, T Lan, Q Li, B Li, Y Yuan, F Xu, W Wang_


_Animals, Humans, COVID-19 Vaccines, Gastrointestinal Microbiome, Pandemics, COVID-19, Microbiota_
The efficacy of COVID-19 vaccines varies between individuals and populations, and the reasons for this are still not fully understood. Recent clinical studies and animal models have indicated that the gut microbiota may influence the immunogenicity of the vaccine and, thus, its effectiveness. This suggests that there is a bidirectional relationship between the gut microbiota and the COVID-19 vaccine, with the varying components of the microbiota either enhancing or reducing the vaccine's efficacy. To put an end to the spread of the COVID-19 pandemic, the necessity of vaccines that create powerful and long-term immunity is now more important than ever, and understanding the role of the gut microbiota in this process is essential. Conversely, COVID-19 vaccines also have a significant effect on the gut microbiota, decreasing its total number of organisms and the variety of species present. In this Review, we analyze the evidence that suggesting an interaction between the gut microbiota and COVID-19 vaccine effectiveness, consider the immunological mechanisms that may be responsible for this connection, and explore the possibility of using gut microbiota-focused interventions to improve the efficacy of COVID-19 vaccines.

[Source](https://doi.org/10.1080/19490976.2023.2233146)

10.1080/19490976.2023.2233146

---

## <em>Bacillus coagulans</em> MZY531 alleviates intestinal mucosal injury in immunosuppressive mice via modulating intestinal barrier, inflammatory response, and gut microbiota

[Source](https://doi.org/10.1038/s41598-023-38379-0)

---

## Microbiome Engineering Using Probiotic Yeast: <em>Saccharomyces boulardii</em> and the Secreted Human Lysozyme Lead to Changes in the Gut Microbiome and Metabolome of Mice

[Source](https://doi.org/10.1128/spectrum.00780-23)

---

## Neotropical bee microbiomes point to a fragmented social core and strong species-level effects.
 17.07.2023, _JG Kueneman, E Bonadies, D Thomas, DW Roubik, WT Wcislo_


_Bees, Animals, Microbiota, Social Behavior, Gastrointestinal Microbiome, Forests_
Individuals that band together create new ecological opportunities for microorganisms. In vertical transmission, theory predicts a conserved microbiota within lineages, especially social bees. Bees exhibit solitary to social behavior among and/or within species, while life cycles can be annual or perennial. Bee nests may be used over generations or only once, and foraging ecology varies widely. To assess which traits are associated with bee microbiomes, we analyzed microbial diversity within solitary and social bees of Apidae, Colletidae, and Halictidae, three bee families in Panama's tropical forests. Our analysis considered the microbiome of adult gut contents replicated through time, localities, and seasons (wet and dry) and included bee morphology and comparison to abdominal (dissected) microbiota. Diversity and distribution of tropical bee microbes (TBM) within the corbiculate bee clade were emphasized.

[Source](https://doi.org/10.1186/s40168-023-01593-z)

10.1186/s40168-023-01593-z

---

## Multi-Omics Reveals the Impact of Exogenous Short-Chain Fatty Acid Infusion on Rumen Homeostasis: Insights into Crosstalk between the Microbiome and the Epithelium in a Goat Model

[Source](https://doi.org/10.1128/spectrum.05343-22)

---

## Identification of Gut Microbiome and Metabolites Associated with Acute Diarrhea in Cats.
 10.07.2023, _H Bai, T Liu, S Wang, W Gong, L Shen, S Zhang, Z Wang_


Changes in diet and environment can lead to acute diarrhea in companion animals, but the composition and interactions of the gut microbiome during acute diarrhea remain unclear. In this multicenter case-control study, we investigated the relationship between intestinal flora and acute diarrhea in two breeds of cats. Acutely diarrheic American Shorthair (MD, n = 12) and British Shorthair (BD, n = 12) and healthy American Shorthair (MH, n = 12) and British Shorthair (BH, n = 12) cats were recruited. Gut microbial 16S rRNA sequencing, metagenomic sequencing, and untargeted metabolomic analysis were performed. We observed significant differences in beta-diversity (Adonis, P &lt; 0.05) across breeds and disease state cohorts. Profound differences in gut microbial structure and function were found between the two cat breeds. In comparison to healthy British Shorthair cats, Prevotella, Providencia, and Sutterella were enriched while Blautia, Peptoclostridium, and Tyzzerella were reduced in American Shorthair cats. In the case-control cohort, cats with acute diarrhea exhibited an increased abundance of Bacteroidota, Prevotella, and Prevotella copri and a decreased abundance of Bacilli, Erysipelotrichales, and Erysipelatoclostridiaceae (both MD and BD cats, P &lt; 0.05). Metabolomic analysis identified significant changes in the BD intestine, affecting 45 metabolic pathways. Moreover, using a random forest classifier, we successfully predicted the occurrence of acute diarrhea with an area under the curve of 0.95. Our findings indicate a distinct gut microbiome profile that is associated with the presence of acute diarrhea in cats. However, further investigations using larger cohorts of cats with diverse conditions are required to validate and extend these findings. IMPORTANCE Acute diarrhea is common in cats, and our understanding of the gut microbiome variations across breeds and disease states remains unclear. We investigated the gut microbiome of two cat breeds (British Shorthair and American Shorthair) with acute diarrhea. Our study revealed significant effects of breeds and disease states on the structure and function of the gut microbiota in cats. These findings emphasize the need to consider breed-related factors in animal nutrition and research models. Additionally, we observed an altered gut metabolome in cats with acute diarrhea, closely linked to changes in bacterial genera. We identified a panel of microbial biomarkers with high diagnostic accuracy for feline acute diarrhea. These findings provide novel insights into the diagnosis, classification, and treatment of feline gastrointestinal diseases.

[Source](https://doi.org/10.1128/spectrum.00590-23)

10.1128/spectrum.00590-23

---

## First metagenomic analysis of the Andean condor (<em>Vultur gryphus</em>) gut microbiome reveals microbial diversity and wide resistome

[Source](https://doi.org/10.7717/peerj.15235)

---

## The predictive power of phylogeny on growth rates in soil bacterial communities.
 16.07.2023, _J Walkup, C Dang, RL Mau, M Hayer, E Schwartz, BW Stone, KS Hofmockel, BJ Koch, AM Purcell, J Pett-Ridge, C Wang, BA Hungate, EM Morrissey_


Predicting ecosystem function is critical to assess and mitigate the impacts of climate change. Quantitative predictions of microbially mediated ecosystem processes are typically uninformed by microbial biodiversity. Yet new tools allow the measurement of taxon-specific traits within natural microbial communities. There is mounting evidence of a phylogenetic signal in these traits, which may support prediction and microbiome management frameworks. We investigated phylogeny-based trait prediction using bacterial growth rates from soil communities in Arctic, boreal, temperate, and tropical ecosystems. Here we show that phylogeny predicts growth rates of soil bacteria, explaining an average of 31%, and up to 58%, of the variation within ecosystems. Despite limited overlap in community composition across these ecosystems, shared nodes in the phylogeny enabled ancestral trait reconstruction and cross-ecosystem predictions. Phylogenetic relationships could explain up to 38% (averaging 14%) of the variation in growth rates across the highly disparate ecosystems studied. Our results suggest that shared evolutionary history contributes to similarity in the relative growth rates of related bacteria in the wild, allowing phylogeny-based predictions to explain a substantial amount of the variation in taxon-specific functional traits, within and across ecosystems.

[Source](https://doi.org/10.1038/s43705-023-00281-1)

10.1038/s43705-023-00281-1

---

## Potential relevance between soybean nitrogen uptake and rhizosphere prokaryotic communities under waterlogging stress.
 14.07.2023, _T Lian, L Cheng, Q Liu, T Yu, Z Cai, H Nian, M Hartmann_


Waterlogging in soil can limit the availability of nitrogen to plants by promoting denitrification and reducing nitrogen fixation and nitrification. The root-associated microorganisms that determine nitrogen availability at the root-soil interface can be influenced by plant genotype and soil type, which potentially alters the nitrogen uptake capacity of plants in waterlogged soils. In a greenhouse experiment, two soybean genotypes with contrasting capacities to resist waterlogging stress were grown in Udic Argosol and Haplic Alisol soils with and without waterlogging, respectively. Using isotope labeling, high-throughput amplicon sequencing and qPCR, we show that waterlogging negatively affects soybean yield and nitrogen absorption from fertilizer, atmosphere, and soil. These effects were soil-dependent and more pronounced in the waterlogging-sensitive than tolerant genotype. The tolerant genotype harbored more ammonia oxidizers and less nitrous oxide reducers. Anaerobic, nitrogen-fixing, denitrifying and iron-reducing bacteria such as Geobacter/Geomonas, Sphingomonas, Candidatus Koribacter, and Desulfosporosinus were proportionally enriched in association with the tolerant genotype under waterlogging. These changes in the rhizosphere microbiome might ultimately help the plant to improve nitrogen uptake under waterlogged, anoxic conditions. This research contributes to a better understanding of the adaptability of soybean genotypes under waterlogging stress and might help to formulate fertilization strategies that improve nitrogen use efficiency of soybean. Schematic representation of the effects of waterlogging on nitrogen uptake and rhizosphere microbiota in dependence of soil type and soybean genotype.

[Source](https://doi.org/10.1038/s43705-023-00282-0)

10.1038/s43705-023-00282-0

---

## Effects of nitrogen reduction combined with bio-organic fertilizer on soil bacterial community diversity of red raspberry orchard.
 13.07.2023, _X Yuan, J Zhang, F Chang, X Wang, X Zhang, H Luan, G Qi, S Guo_


_Fertilizers, RNA, Ribosomal, 16S, Rubus, Alkalies, Bacteria, Nitrogen, Phosphorus_
Understanding soil bacterial diversity under nitrogen reduction is necessary for the crucial role in soil nitrogen cycling. However, the effects of combined fertilization on soil chemical properties, microbial community structure, and yield are unknown. This study was conducted to investigate the effect of nitrogen fertilizer reduction with bio-organic fertilizer on soil bacterial community diversity of red raspberry orchard. Six treatments were set in this study: NF-100%, NF-75%, NF-50%, NF-25% and CF, no nitrogen fertilizer and bio-organic fertilizer for CK. The bacterial community structures of soil were analyzed by 16S rRNA gene amplification high-throughput sequencing technology. Nitrogen fertilizer reduction with bio-organic fertilizer increased soil organic matter (SOM), total nitrogen (TN), alkali-hydrolyzable nitrogen (AN), available phosphorus (AP), available potassium (AK), and reduced soil pH. NF-50% and NF-25% treatments increased the yield of red raspberry. Nitrogen reduction combined with bio-organic fertilizer increased the relative abundance of copiotrophic bacteria and decreased the relative abundance of oligotrophic bacteria. The increase in copiotrophic bacteria in the soil of red raspberry orchard could indicate an increase in soil nutrient availability, which have positive implications for soil fertility and production. However, nitrogen fertilizer reduction with bio-organic fertilizer altered the abundance and diversity of soil bacteria, which was reduced compared to CF treatments. The PCoA analysis of the soil bacterial community showed that the community structure of NF-25% treatment was more different from other treatments, indicating that the fertilization method changed the community structure of soil bacteria. The results of a redundancy analysis showed that SOM, pH, AN, TN, and AP were the main factors affecting the microbial community structure. Overall, the reduction of nitrogen fertilizer with bio-organic fertilizer significantly increased the soil nutrient content, reduced the relative abundance and diversity of soil bacteria, increased the relative abundance of beneficial bacteria in the soil, changed the bacterial community structure of soil, increased production and created suitable soil conditions for the red raspberry growth.

[Source](https://doi.org/10.1371/journal.pone.0283718)

10.1371/journal.pone.0283718

---

## Leaf side determines the relative importance of dispersal versus host filtering in the phyllosphere microbiome.
 12.07.2023, _W Smets, MK Chock, CM Walsh, CQ Vanderburgh, E Kau, SE Lindow, N Fierer, B Koskella_


Leaves harbor distinct microbial communities that can have an important impact on plant health and microbial ecosystems worldwide. Nevertheless, the ecological processes that shape the composition of leaf microbial communities remain unclear, with previous studies reporting contradictory results regarding the importance of bacterial dispersal versus host selection. This discrepancy could be driven in part because leaf microbiome studies typically consider the upper and lower leaf surfaces as a single entity despite these habitats possessing considerable anatomical differences. We characterized the composition of bacterial phyllosphere communities from the upper and lower leaf surfaces across 24 plant species. Leaf surface pH and stomatal density were found to shape phyllosphere community composition, and the underside of leaves had lower richness and higher abundances of core community members than upper leaf surfaces. We found fewer endemic bacteria on the upper leaf surfaces, suggesting that dispersal is more important in shaping these communities, with host selection being a more important force in microbiome assembly on lower leaf surfaces. Our study illustrates how changing the scale in which we observe microbial communities can impact our ability to resolve and predict microbial community assembly patterns on leaf surfaces. IMPORTANCE Leaves can harbor hundreds of different bacterial species that form unique communities for every plant species. Bacterial communities on leaves are really important because they can, for example, protect their host against plant diseases. Usually, bacteria from the whole leaf are considered when trying to understand these communities; however, this study shows that the upper and lower sides of a leaf have a very different impact on how these communities are shaped. It seems that the bacteria on the lower leaf side are more closely associated with the plant host, and communities on the upper leaf side are more impacted by immigrating bacteria. This can be really important when we want to treat, for example, crops in the field with beneficial bacteria or when trying to understand host-microbe interactions on the leaves.

[Source](https://doi.org/10.1128/mbio.01111-23)

10.1128/mbio.01111-23

---

## Pathogen resistance in soils associated with&nbsp;bacteriome network reconstruction through reductive soil disinfestation

[Source](https://doi.org/10.1007/s00253-023-12676-0)

---

## Detection of Microorganisms in Body Fluid Samples.
 17.07.2023, _X Ji, S Ni, G Tian, L Zhang, W Wang_


_RNA, Ribosomal, 16S, Sequence Analysis, DNA, Microbiota, Metagenome, Bacteria, High-Throughput Nucleotide Sequencing, Metagenomics, Body Fluids_
Next-generation sequencing (NGS) has been widely applied to the identification of microbiome in body fluids. The methodology of 16S rRNA amplicon sequencing is simple, fast, and cost-effective. It overcomes the problem that some microorganisms cannot be isolated or cultured. Low abundant bacteria can also be amplified and sequenced, but the resolution of classification can hardly reach species or sub-species level; moreover, this methodology is mainly used to identify bacterial populations, and other microorganisms like viruses or fungi cannot be sequenced. On the other hand, the microbiome profiling obtained by shotgun metagenomic sequencing is more comprehensive with better resolution, and more accurate classification can be expected due to higher coverage of genomic sequences from microorganisms. By combining the capture-based method with metagenomic sequencing, we can further enrich and detect low abundant microorganisms and identify the viral integration sites in host gDNA at once.

[Source](https://doi.org/10.1007/978-1-0716-3346-5_5)

10.1007/978-1-0716-3346-5_5

---

## Guidelines for Microbiome Studies in Renal Physiology.
 13.07.2023, _RR Muralitharan, M Snelson, G Meric, MT Coughlan, FZ Marques_


Gut microbiome research has increased dramatically in the last decade, including in renal health and disease. The field is moving from experiments showing mere association to causation using both forward and reverse microbiome approaches, leveraging tools such as germ-free animals, treatment with antibiotics, and faecal microbiota transplantations. However, we are still seeing a gap between discovery and translation that needs to be addressed, so that patients can benefit from microbiome-based therapies. In this guideline paper, we discuss the key considerations that affect the gut microbiome of animal and clinical studies assessing renal function, many which are often overlooked, resulting in false-positive results. For animal studies, these include suppliers, acclimatisation, baseline microbiota and its normalisation, littermates and cohort/cage effects, diet, sex differences, age, circadian differences, antibiotics and sweeteners, and models used. Clinical studies have some unique considerations, which include sampling, gut transit time, dietary records, medication, and renal phenotypes. We provide best practice guidance on sampling, storage, DNA extraction, and methods for microbial DNA sequencing (both 16S rRNA and shotgun metagenome). Finally, we discuss follow-up analyses, including tools available, metrics, and their interpretation, and the key challenges ahead in the microbiome field. By standardising study designs, methods and reporting, we will accelerate the findings from discovery to translation and result in new microbiome-based therapies that may improve renal health.

[Source](https://doi.org/10.1152/ajprenal.00072.2023)

10.1152/ajprenal.00072.2023

---

## Quantitative differences in synthetic gut microbial inoculums do not affect the final stabilized in vitro community compositions

[Source](https://doi.org/10.1128/msystems.01249-22)

---

## Activity-based protein profiling in microbes and the gut microbiome.
 10.07.2023, _L Han, PV Chang_


Activity-based protein profiling (ABPP) is a powerful chemical approach for probing protein function and enzymatic activity in complex biological systems. This strategy typically utilizes activity-based probes that are designed to bind a specific protein, amino acid residue, or protein family and form a covalent bond through a reactivity-based warhead. Subsequent analysis by mass spectrometry-based proteomic platforms that involve either click chemistry or affinity-based labeling to enrich for the tagged proteins enables identification of protein function and enzymatic activity. ABPP has facilitated elucidation of biological processes in bacteria, discovery of new antibiotics, and characterization of host-microbe interactions within physiological contexts. This review will focus on recent advances and applications of ABPP in bacteria and complex microbial communities.

[Source](https://doi.org/10.1016/j.cbpa.2023.102351)

10.1016/j.cbpa.2023.102351

---

