## Association between microbiome and the development of adverse posttraumatic neuropsychiatric sequelae after traumatic stress exposure.
 22.11.2023, _AL Zeamer, MC Salive, X An, FL Beaudoin, SL House, JS Stevens, D Zeng, TC Neylan, GD Clifford, SD Linnstaedt, SL Rauch, AB Storrow, C Lewandowski, PI Musey, PL Hendry, S Sheikh, CW Jones, BE Punches, RA Swor, LA Hudak, JL Pascual, MJ Seamon, E Harris, C Pearson, DA Peak, RC Merchant, RM Domeier, NK Rathlev, BJ O'Neil, P Sergot, LD Sanchez, SE Bruce, RC Kessler, KC Koenen, SA McLean, V Bucci, JP Haran_


_Adult, Humans, Microbiota, Stress Disorders, Post-Traumatic, Gastrointestinal Microbiome, Feces, Biological Availability_
Patients exposed to trauma often experience high rates of adverse post-traumatic neuropsychiatric sequelae (APNS). The biological mechanisms promoting APNS are currently unknown, but the microbiota-gut-brain axis offers an avenue to understanding mechanisms as well as possibilities for intervention. Microbiome composition after trauma exposure has been poorly examined regarding neuropsychiatric outcomes. We aimed to determine whether the gut microbiomes of trauma-exposed emergency department patients who develop APNS have dysfunctional gut microbiome profiles and discover potential associated mechanisms. We performed metagenomic analysis on stool samples (n = 51) from a subset of adults enrolled in the Advancing Understanding of RecOvery afteR traumA (AURORA) study. Two-, eight- and twelve-week post-trauma outcomes for post-traumatic stress disorder (PTSD) (PTSD checklist for DSM-5), normalized depression scores (PROMIS Depression Short Form 8b) and somatic symptom counts were collected. Generalized linear models were created for each outcome using microbial abundances and relevant demographics. Mixed-effect random forest machine learning models were used to identify associations between APNS outcomes and microbial features and encoded metabolic pathways from stool metagenomics. Microbial species, including Flavonifractor plautii, Ruminococcus gnavus and, Bifidobacterium species, which are prevalent commensal gut microbes, were found to be important in predicting worse APNS outcomes from microbial abundance data. Notably, through APNS outcome modeling using microbial metabolic pathways, worse APNS outcomes were highly predicted by decreased L-arginine related pathway genes and increased citrulline and ornithine pathways. Common commensal microbial species are enriched in individuals who develop APNS. More notably, we identified a biological mechanism through which the gut microbiome reduces global arginine bioavailability, a metabolic change that has also been demonstrated in the plasma of patients with PTSD.

[Source](https://doi.org/10.1038/s41398-023-02643-8)

10.1038/s41398-023-02643-8

---

## The preterm human milk microbiota fluctuates by postpartum week and is characterized by gestational age and maternal BMI.
 17.11.2023, _EJ Filatava, Z Liu, J Xie, D-B Tran, K Chen, N El Habbal, G Weinstock, Y Zhou, KE Gregory_


Despite a growing recognition that the type of nutrition received by preterm infants influences their intestinal microbiome and health outcomes, the microbiota of mother's own milk (MOM), pasteurized donor human milk (PDHM), and infant formula remain poorly characterized. In our study, we found that the structure of microbial communities, bacterial diversity, and relative abundances of specific genera were significantly different between MOM, PDHM, and formula. Additionally, our results suggest that the microbiota of MOM changes as a function of time and maternal factors. Lastly, we identified three lactotypes within MOM that have distinct microbial compositions and described the maternal factors associated with them. These findings set the stage for future research aimed at advancing our knowledge of the microbiota of preterm infant nutrition and the specific influence it may have on health outcomes.

[Source](https://doi.org/10.1128/mbio.02106-23)

10.1128/mbio.02106-23

---

## Strain-resolved metagenomic analysis of the gut as a reservoir for bloodstream infection pathogens among premature infants in Singapore

[Source](https://doi.org/10.1186/s13099-023-00583-8)

---

## <em>Porphyromonas gingivalis</em> as a promotor in the development of the alcoholic liver disease via ferroptosis

[Source](https://doi.org/10.1016/j.micinf.2023.105250)

---

## The core phageome and its interrelationship with preterm human milk lipids

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(23)01385-2)

---

## Longitudinal analysis at three oral sites links oral microbiota to clinical outcomes in allogeneic hematopoietic stem-cell transplant

[Source](https://doi.org/10.1128/spectrum.02910-23)

---

## Gut inflammation associated with age and Alzheimer’s disease pathology: a human cohort study

[Source](https://doi.org/10.1038/s41598-023-45929-z)

---

## Robust airway microbiome signatures in acute respiratory failure and hospital-acquired pneumonia

[Source](https://doi.org/10.1038/s41591-023-02617-9)

---

## Systematic mining of the human microbiome identifies antimicrobial peptides with diverse activity spectra

[Source](https://doi.org/10.1038/s41564-023-01524-6)

---

## Skin benefits of postbiotics derived from <em>Micrococcus luteus</em> derived from human skin: an untapped potential for dermatological health

[Source](https://doi.org/10.1007/s13258-023-01471-w)

---

## <em>Akkermansia muciniphila</em> attenuated lipopolysaccharide-induced acute lung injury by modulating the gut microbiota and SCFAs in mice

[Source](https://doi.org/10.1039/d3fo04051h)

---

## The Potential of Meta-Proteomics and Artificial Intelligence to Establish the Next Generation of Probiotics for Personalized Healthcare

[Source](https://doi.org/10.1021/acs.jafc.3c03834)

---

## Gut microbiome in healthy aging versus those associated with frailty

[Source](https://doi.org/10.1080/19490976.2023.2278225)

---

## Host species and habitat shape fish-associated bacterial communities: phylosymbiosis between fish and their microbiome

[Source](https://doi.org/10.1186/s40168-023-01697-6)

---

## <em>Aedes aegypti</em> microbiome composition covaries with the density of <em>Wolbachia</em> infection

[Source](https://doi.org/10.1186/s40168-023-01678-9)

---

## Sequential host-bacteria and bacteria-bacteria interactions determine the microbiome establishment of <em>Nematostella vectensis</em>

[Source](https://doi.org/10.1186/s40168-023-01701-z)

---

## Microbial symbionts are shared between ants and their associated beetles.
 16.11.2023, _C Valdivia, JA Newton, C von Beeren, S O'Donnell, DJC Kronauer, JA Russell, P Łukasik_


The transmission of microbial symbionts across animal species could strongly affect their biology and evolution, but our understanding of transmission patterns and dynamics is limited. Army ants (Formicidae: Dorylinae) and their hundreds of closely associated insect guest species (myrmecophiles) can provide unique insights into interspecific microbial symbiont sharing. Here, we compared the microbiota of workers and larvae of the army ant Eciton burchellii with those of 13 myrmecophile beetle species using 16S rRNA amplicon sequencing. We found that the previously characterized specialized bacterial symbionts of army ant workers were largely absent from ant larvae and myrmecophiles, whose microbial communities were usually dominated by Rickettsia, Wolbachia, Rickettsiella and/or Weissella. Strikingly, different species of myrmecophiles and ant larvae often shared identical 16S rRNA genotypes of these common bacteria. Protein-coding gene sequences confirmed the close relationship of Weissella strains colonizing army ant larvae, some workers and several myrmecophile species. Unexpectedly, these strains were also similar to strains infecting dissimilar animals inhabiting very different habitats: trout and whales. Together, our data show that closely interacting species can share much of their microbiota, and some versatile microbial species can inhabit and possibly transmit across a diverse range of hosts and environments.

[Source](https://doi.org/10.1111/1462-2920.16544)

10.1111/1462-2920.16544

---

## The gut microbiome mediates adaptation to scarce food in Coleoptera.
 16.11.2023, _OT Moldovan, AA Carrell, PA Bulzu, E Levei, R Bucur, C Sitar, L Faur, IC Mirea, M Șenilă, O Cadar, M Podar_


Beetles are ubiquitous cave invertebrates worldwide that adapted to scarce subterranean resources when they colonized caves. Here, we investigated the potential role of gut microbiota in the adaptation of beetles to caves from different climatic regions of the Carpathians. The beetles' microbiota was host-specific, reflecting phylogenetic and nutritional adaptation. The microbial community structure further resolved conspecific beetles by caves suggesting microbiota-host coevolution and influences by local environmental factors. The detritivore species hosted a variety of bacteria known to decompose and ferment organic matter, suggesting turnover and host cooperative digestion of the sedimentary microbiota and allochthonous-derived nutrients. The cave Carabidae, with strong mandibula, adapted to predation and scavenging of animal and plant remains, had distinct microbiota dominated by symbiotic lineages Spiroplasma or Wolbachia. All beetles had relatively high levels of fermentative Carnobacterium and Vagococcus involved in lipid accumulation and a reduction of metabolic activity, and both features characterize adaptation to caves.

[Source](https://doi.org/10.1186/s40793-023-00537-2)

10.1186/s40793-023-00537-2

---

## Interplay of gut microbiota and host epithelial mitochondrial dysfunction is necessary for the development of spontaneous intestinal inflammation in mice.
 24.11.2023, _KM Alula, AS Dowdell, B LeBere, JS Lee, CL Levens, KA Kuhn, BA Kaipparettu, WE Thompson, RS Blumberg, SP Colgan, AL Theiss_


_Humans, Animals, Mice, Crohn Disease, Gastrointestinal Microbiome, Ileitis, Inflammation, Inflammatory Bowel Diseases, Paneth Cells, Butyrates, Mitochondria, Intestinal Mucosa_
Intestinal epithelial cell (IEC) mitochondrial dysfunction involvement in inflammatory bowel diseases (IBD), including Crohn's disease affecting the small intestine, is emerging in recent studies. As the interface between the self and the gut microbiota, IECs serve as hubs of bidirectional cross-talk between host and luminal microbiota. However, the role of mitochondrial-microbiota interaction in the ileum is largely unexplored. Prohibitin 1 (PHB1), a chaperone protein of the inner mitochondrial membrane required for optimal electron transport chain function, is decreased during IBD. We previously demonstrated that mice deficient in PHB1 specifically in IECs (Phb1i∆IEC) exhibited mitochondrial impairment, Paneth cell defects, gut microbiota dysbiosis, and spontaneous inflammation in the ileum (ileitis). Mice deficient in PHB1 in Paneth cells (epithelial secretory cells of the small intestine; Phb1∆PC) also exhibited mitochondrial impairment, Paneth cell defects, and spontaneous ileitis. Here, we determined whether this phenotype is driven by Phb1 deficiency-associated ileal microbiota alterations or direct effects of loss of PHB1 in host IECs.

[Source](https://doi.org/10.1186/s40168-023-01686-9)

10.1186/s40168-023-01686-9

---

## Microbial diversity and community structure dynamics in acid mine drainage: Acidic fire with dissolved heavy metals.
 21.11.2023, _W Sajjad, N Ilahi, S Kang, A Bahadur, A Banerjee, S Zada, B Ali, M Rafiq, G Zheng_


Acid mine drainage (AMD) is one of the leading causes of environmental pollution and is linked to public health and ecological consequences. Microbes-mineral interaction generates AMD, but microorganisms can also remedy AMD pollution. Exploring the microbial response to AMD effluents may reveal survival strategies in extreme ecosystems. Three distinct sites across a mine (inside the mine, the entrance of the mine, and outside) were selected to study their heavy metal concentrations due to significant variations in pH and physicochemical characteristics, and high-throughput sequencing was carried out to investigate the microbial diversity. The metal and ion concentrations followed the order SO42-, Fe, Cu, Zn, Mg, Pb, Co, Cr, and Ni from highest to lowest, respectively. Maximum sequences were allocated to Proteobacteria and Firmicutes. Among archaea, the abundance of Thaumarchaeota and Euryarchaeota was higher outside of mine. Most of the genera (23.12 %) were unclassified and unknown. The average OTUs (operational taxonomic units) were significantly higher outside the mine; however, diversity indices were not significantly different across the mine sites. Hierarchical clustering of selective genera and nMDS ordination of OTUs displayed greater segregation resolution inside and outside of mine, whereas the entrance samples clustered with greater similarity. Heterogeneous selection might be the main driver of community composition outside the mine, whereas stochastic processes became prominent inside the mine. However, the ANOSIM test shows a relatively even distribution of community composition within and between the groups. Microbial phyla showed both positive and negative correlations with physicochemical factors. A greater number of biomarkers were reported outside of the mine. Predictive functional investigation revealed the existence of putative degradative, metabolic, and biosynthetic pathways. This study presents a rare dataset in our understanding of microbial diversity and distribution as shaped by the ecological gradient and potential novelty in phylogenetic/taxonomic diversity in AMD, with potential biotechnological applications.

[Source](https://doi.org/10.1016/j.scitotenv.2023.168635)

10.1016/j.scitotenv.2023.168635

---

## Polar lake microbiomes have distinct evolutionary histories

[Source](https://doi.org/10.1126/sciadv.ade7130)

---

## PSKR1 balances the plant growth-defence trade-off in the rhizosphere microbiome

[Source](https://doi.org/10.1038/s41477-023-01539-1)

---

## <em>Streptomyces</em> alleviate abiotic stress in plant by producing pteridic acids

[Source](https://doi.org/10.1038/s41467-023-43177-3)

---

## The pan-microbiome profiling system Taxa4Meta identifies clinical dysbiotic features and classifies diarrheal disease

[Source](https://doi.org/10.1172/JCI170859)

---

## Machine learning for microbiologists

[Source](https://doi.org/10.1038/s41579-023-00984-1)

---

