## Metabolite-based dietary supplementation in human type 1 diabetes is associated with microbiota and immune modulation.
 20.01.2022, _KJ Bell, S Saad, BJ Tillett, HM McGuire, S Bordbar, YA Yap, LT Nguyen, MR Wilkins, S Corley, S Brodie, S Duong, CJ Wright, S Twigg, BF de St Groth, LC Harrison, CR Mackay, EN Gurzov, EE Hamilton-Williams, E Mariño_


Short-chain fatty acids (SCFAs) produced by the gut microbiota have beneficial anti-inflammatory and gut homeostasis effects and prevent type 1 diabetes (T1D) in mice. Reduced SCFA production indicates a loss of beneficial bacteria, commonly associated with chronic autoimmune and inflammatory diseases, including T1D and type 2 diabetes. Here, we addressed whether a metabolite-based dietary supplement has an impact on humans with T1D. We conducted a single-arm pilot-and-feasibility trial with high-amylose maize-resistant starch modified with acetate and butyrate (HAMSAB) to assess safety, while monitoring changes in the gut microbiota in alignment with modulation of the immune system status.

[Source](https://doi.org/10.1186/s40168-021-01193-9)

10.1186/s40168-021-01193-9

---

## GWAS-associated bacteria and their metabolites appear to be causally related to the development of inflammatory bowel disease

[Source](https://doi.org/10.1038/s41430-022-01074-w)

---

## The vaginal Torquetenovirus titer varies with vaginal microbiota composition in pregnant women.
 20.01.2022, _TR Tozetto-Mendoza, MC Mendes-Correa, AF Moron, LJ Forney, IM Linhares, A Ribeiro da Silva, L Honorato, SS Witkin_


Torquetenovirus (TTV) is a nonpathogenic endogenous virus whose abundance varies with the extent of immune system activation. We determined if the TTV titer in the vagina of pregnant women was associated with vaginal microbiota composition and levels of compounds in vaginal secretions. Vaginal TTV and microbiota composition in 494 second trimester pregnant women were identified by gene amplification and analysis. Vaginal matrix metalloproteinases (MMPs), tissue inhibitors of MMP (TIMP) and lactic acid isomers were measured by ELISA. Dominance was defined as the relative abundance of a specific bacterium or species at &gt;50% of the total number of bacteria identified. Clinical data were obtained by chart review. The median log10 TTV titer was lowest when Lactobacillus species other than L. iners were dominant (&lt;1.0) as compared to when L. iners (4.1, p = 0.0001), bacteria other than lactobacilli (4.5, p = 0.0016) or no bacterium (4.7, p = 0.0009) dominated. The TTV titer was inversely proportional to L. crispatus abundance (p&lt;0.0001) and directly proportional to levels of G. vaginalis (p = 0.0008) and L. iners (p = 0.0010). The TTV titer was proportional to TIMP-1, TIMP-2, MMP-8 and MMP-9 abundance (p≤0.0002) and inversely proportional to the level of D-lactic acid (p = 0.0024). We conclude that the association between variations in the TTV titer and the relative abundance of specific bacterial species and vaginal compounds indicates that local changes in immune status likely influence vaginal fluid composition.

[Source](https://doi.org/10.1371/journal.pone.0262672)

10.1371/journal.pone.0262672

---

## Can the Acid-formation Potential of Saliva Detect a Caries-related Shift in the Oral Microbiome?
 20.01.2022, _C Frese, LS Reissfelder, S Kilian, A Felten, L Laurisch, K Schoilew, S Boutin_


To determine acid-formation potential of saliva and evaluate whether this method corresponds with microbiome composition of individuals with and without caries.

[Source](https://doi.org/10.3290/j.ohpd.b2573053)

10.3290/j.ohpd.b2573053

---

## Gut microbiome alterations and gut barrier dysfunction are associated with host immune homeostasis in COVID-19 patients.
 20.01.2022, _Z Sun, ZG Song, C Liu, S Tan, S Lin, J Zhu, FH Dai, J Gao, JL She, Z Mei, T Lou, JJ Zheng, Y Liu, J He, Y Zheng, C Ding, F Qian, Y Zheng, YM Chen_


COVID-19 is an infectious disease characterized by multiple respiratory and extrapulmonary manifestations, including gastrointestinal symptoms. Although recent studies have linked gut microbiota to infectious diseases such as influenza, little is known about the role of the gut microbiota in COVID-19 pathophysiology.

[Source](https://doi.org/10.1186/s12916-021-02212-0)

10.1186/s12916-021-02212-0

---

## Predicting Treatment Nonresponse in Hispanic/Latino Children Receiving Silver Diamine Fluoride for Caries Arrest: A Pilot Study Using Machine Learning.
 20.01.2022, _RR Ruff, B Paul, MA Sierra, F Xu, X Li, YO Crystal, D Saxena_


Objectives: Silver diamine fluoride (SDF) is a nonsurgical therapy for the arrest and prevention of dental caries with demonstrated clinical efficacy. Approximately 20% of children receiving SDF fail to respond to treatment. The objective of this study was to develop a predictive model of treatment non-response using machine learning. Methods: An observational pilot study (N = 20) consisting of children with and without active decay and who did and did not respond to silver diamine fluoride provided salivary samples and plaque from infected and contralateral sites. 16S rRNA genes from samples were amplified and sequenced on an Illumina Miseq and analyzed using QIIME. The association between operational taxonomic units and treatment non-response was assessed using lasso regression and artificial neural networks. Results: Bivariate group comparisons of bacterial abundance indicate a number of genera were significantly different between non-responders and those who responded to SDF therapy. No differences were found between non-responders and caries-active subjects. Prevotella pallens and Veillonella denticariosi were retained in full lasso models and combined with clinical variables in a six-input multilayer perceptron. Discussion: The acidogenic and acid-tolerant nature of retained bacterial species may overcome the antimicrobial effects of SDF. Further research to validate the model in larger external samples is needed.

[Source](https://doi.org/10.3389/froh.2021.695759)

10.3389/froh.2021.695759

---

## The Gut Microbiome of Adults With Type 1 Diabetes and Its Association With the Host Glycemic Control.
 19.01.2022, _S Shilo, A Godneva, M Rachmiel, T Korem, Y Bussi, D Kolobkov, T Karady, N Bar, BC Wolf, Y Glantz-Gashai, M Cohen, NZ Levin, N Shehadeh, N Gruber, N Levran, S Koren, A Weinberger, O Pinhas-Hamiel, E Segal_


Previous studies have demonstrated an association between gut microbiota composition and type 1 diabetes (T1D) pathogenesis. However, little is known about the composition and function of the gut microbiome in adults with longstanding T1D or its association with host glycemic control.

[Source](https://doi.org/10.2337/dc21-1656)

10.2337/dc21-1656

---

## Evaluation of Bacterial Diversity and Evolutionary Dynamics of Gut <em>Bifidobacterium longum</em> Isolates Obtained from Older Individuals in Hubei Province, China

[Source](https://doi.org/10.1128/spectrum.01442-21)

---

## Frontiers in Oral Mucosal Immunity and the Microbiome.
 20.01.2022, _GN Belibasakis, G Hajishengallis_


The 2nd International Conference on Oral Mucosal Immunity and the Microbiome (OMIM) took place at the Grecotel Kos Imperial Hotel, Kos, Greece, between 25th and 30th September 2021, under the auspices of the Aegean Conferences. This has only been the second Aegean Conference of this thematic, the first one having taken place in 2018 in Crete, during the same period of the year. Given the hardships in travel and heightened infection transmission risks amid the COVID-19 pandemic, the Conference was well attended by 29 international speakers across the world. For many of the participants, this was the first conference travel in the post-pandemic era, and quite significant that it has taken place on the island of Hippocrates. Stringent regional health and safety regulations had to be followed to accomplish for this in-person Conference to take place. Frontiers in Oral Health has hosted papers from presentations of the Conference, whereas the present article serves as the proceedings of the Conference with summaries of the presentations.

[Source](https://doi.org/10.3389/froh.2021.821148)

10.3389/froh.2021.821148

---

## The microbiota-gut-brain axis: pathways to better brain health. Perspectives on what we know, what we need to investigate and how to put knowledge into practice

[Source](https://doi.org/10.1007/s00018-021-04060-w)

---

## Predictors of Fecal Microbiota Transplant Failure in <em>Clostridioides difficile</em> Infection: An Updated Meta-analysis

[Source](https://doi.org/10.1097/MCG.0000000000001667)

---

## The Role of Gut Microbiota-Brain Axis in Pathophysiology of ADHD: A Systematic Review

[Source](https://doi.org/10.1177/10870547211073474)

---

## Meta-Analysis Using NGS Data: The <em>Veillonella</em> Species in Dental Caries

[Source](https://doi.org/10.3389/froh.2021.770917)

---

## Peppermint oil effects on the gut microbiome in children with functional abdominal pain

[Source](https://doi.org/10.1111/cts.13224)

---

## SER-109, an Oral Microbiome Therapy for Recurrent <em>Clostridioides difficile</em> Infection

[Source](https://doi.org/10.1056/NEJMoa2106516)

---

## <em>Lactiplantibacillus plantarum</em> CCFM1019 attenuate polycystic ovary syndrome through butyrate dependent gut-brain mechanism

[Source](https://doi.org/10.1039/d1fo01744f)

---

## Causal relationship between gut microbiota and serum vitamin D: evidence from genetic correlation and Mendelian randomization study

[Source](https://doi.org/10.1038/s41430-021-01065-3)

---

## Modulation of Gut Microbiota and Immune System by Probiotics, Pre-biotics, and Post-biotics

[Source](https://doi.org/10.3389/fnut.2021.634897)

---

## Outlook on next-generation probiotics from the human gut

[Source](https://doi.org/10.1007/s00018-021-04080-6)

---

## Seasonal shift of the gut microbiome synchronizes host peripheral circadian rhythm for physiological adaptation to a low-fat diet in the giant panda

[Source](https://doi.org/10.1016/j.celrep.2021.110203)

---

## Seasonal variations in the composition and functional profiles of gut microbiota reflect dietary changes in plateau pikas.
 20.01.2022, _C Fan, L Zhang, S Jia, X Tang, H Fu, W Li, C Liu, H Zhang, Q Cheng, Y Zhang_


Seasonal variations in gut microbiota of small mammals and how it is influenced by environmental variables is relatively poorly understood. We sampled 162 wild plateau pikas (Ochotona curzoniae) in four seasons over two and a half years and recorded the air temperature, precipitation, and nutrient content in edible vegetation at the sampling site. After conducting 16S rRNA and shotgun metagenomic sequencing, we found that the highest alpha diversity, the relative abundance of Firmicutes, and the simplest co-occurrence network occurred in winter, whereas that the highest relative abundance of Proteobacteria and the most complex network structure was observed in spring. The highest relative abundance of Verrucomicrobiota and Spirochaetota were seen in summer and autumn, respectively. Air temperature, precipitation, and the contents of crude protein, crude fiber, and polysaccharide in vegetation had significant effects on the seasonal changes in gut microbiota. Diet contributed more to microbial variation than climatic factors. Metagenomic analysis revealed that the amino acid metabolism pathway and axillary activity enzymes were most abundant in summer, while abundance of carbohydrate-binding modules and carbohydrate esterases were highest in spring. These microbial variations were related to the changes in dietary nutrition, indicating that gut microbiota of plateau pika contribute to the efficient use of food resources. This study provides new evidence of how external environmental factors affect the intestinal environment of small mammals. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1111/1749-4877.12630)

10.1111/1749-4877.12630

---

## N-acyl-homoserine lactone produced by <em>Rahnella inusitata</em> isolated from the gut of <em>Galleria mellonella</em> influences <em>Salmonella</em> phenotypes

[Source](https://doi.org/10.1007/s42770-022-00681-w)

---

## Comparative 16S Metabarcoding of Nile Tilapia Gut Microbiota from the Northern Lakes of Egypt.
 20.01.2022, _AM Serag, MS Abdel-Sabour, M El-Hadidi, M Maged, M Magdy, MF Ramadan, MH Refaat_


Nile tilapia, Oreochromis niloticus, is the principal fish bred in Egypt. A pilot study was designed to analyze the bacterial composition of the Nile tilapia fish guts from two saltwater lakes in Northern Egypt. Fish samples were obtained from two Delta lakes: Manzala (ML) and Borollus (BL). DNA was extracted, and the bacterial communities in the stomach content were classified (down to the species level) using the 16S rRNA-based analysis. From the two metagenomics libraries in this study, 1,426,740 reads of the amplicon sequence corresponding to 508 total taxonomic operational units were recorded. The most prevalent bacterial phyla were Proteobacteria, Firmicutes, Actinobacteria, and Synergistetes in all samples. Some of the strains identified belong to classes of pathogenic zoonotic bacteria. A notable difference was observed between gut bacteria of Nile tilapia fish obtained from BL and ML. There is a remarkable indication that Nile tilapia fish living in BL is heavily burdened with pathogenic microbes most remarkably those involved with methylation of mercury and its accumulation in fish organs. These pathogenic microbes could have clinical implications and correlated with many diseases. This result was also consistent with the metagenomic data's functional prediction that indicated that Nile tilapia species harboring these two Egyptian northern lakes may be exposed to numerous anthropogenic pollutants. The findings show that the host environment has a significant impact on the composition of its microbiota. The first step towards exploring the better management of this profit-making fish is recognizing the structure of the microbiome.

[Source](https://doi.org/10.1007/s12010-021-03750-2)

10.1007/s12010-021-03750-2

---

## Diversity and Distribution of Microbial Communities Associated with Reef Corals of the Malay Peninsula.
 19.01.2022, _DP Kanisan, ZBR Quek, RM Oh, L Afiq-Rosli, JN Lee, D Huang, BJ Wainwright_


Coral-associated bacteria play critical roles in the regulation of coral health and function. Environmental perturbations that alter the bacterial community structure can render the coral holobiont more susceptible and less resilient to disease. Understanding the natural variation of the coral microbiome across space and host species provides a baseline that can be used to distinguish shifts in community structure. Using a 16S rRNA gene metabarcoding approach, this study examines bacterial community structure across three scleractinian coral hosts. Our results show that corals of three regions-eastern and western Peninsular Malaysia and Singapore-host distinct bacterial communities; despite these differences, we were able to identify a core microbiome shared across all three species. This core microbiome was also present in samples previously collected in Thailand, suggesting that these core microbes play an important role in promoting and maintaining host health. For example, several have been identified as dimethylsulfoniopropionate (DMSP) metabolizers that have roles in sulfur cycling and the suppression of bacterial pathogens. Pachyseris speciosa has the most variable microbiome, followed by Porites lutea, with the composition of the Diploastrea heliopora microbiome the least variable throughout all locations. Microbial taxa associated with each region or site are likely shaped by local environmental conditions. Taken together, host identity is a major driver of differences in microbial community structure, while environmental heterogeneity shapes communities at finer scales.

[Source](https://doi.org/10.1007/s00248-022-01958-1)

10.1007/s00248-022-01958-1

---

## The Harris’&nbsp;hawk (<em>Parabuteo unicinctus</em>) as a source of pathogenic human yeasts: a potential risk to human health

[Source](https://doi.org/10.2217/fmb-2021-0166)

---

## Increased microbial diversity and decreased prevalence of common pathogens in the gut microbiomes of wild turkeys compared to domestic turkeys.
 19.01.2022, _J Craft, H Eddington, N Christman, W Pryor, JM Chaston, D Erickson, E Wilson_


Turkeys (Meleagris gallopavo) provide a globally important source of protein and constitute the second most important source of poultry meat in the world. Bacterial diseases are common in commercial poultry production causing significant production losses for farmers. Due to the increasingly recognized problems associated with large-scale/indiscriminant antibiotic use in agricultural settings, poultry producers need alternative methods to control common bacterial pathogens. In this study we compared the cecal microbiota of wild and domestic turkeys, hypothesizing that environmental pressures faced by wild birds may select for a disease-resistant microbial community. Sequence analysis of 16S rRNA genes amplified from cecal samples indicate that free-roaming wild turkeys carry a rich and variable microbiota compared to domestic turkeys raised on large-scale poultry farms. Wild turkeys also had very low levels of Staphylococcus, Salmonella and E. coli when compared to domestic turkeys. E. coli strains isolated from wild or domestic turkey cecal samples also belong to distinct phylogenetic backgrounds and differ in their propensity to carry virulence genes. E. coli strains isolated from factory-raised turkeys were far more likely to carry genes for capsule (kpsII, kpsIII) or siderophore (iroN, fyuA) synthesis than those isolated from wild turkeys. These results suggest that the microbiota of wild turkeys may provide colonization resistance against common poultry pathogens. Importance Due to the increasingly recognized problems associated with antibiotic use in agricultural settings, poultry producers need alternative methods to control common bacterial pathogens. In this study we compare the microbiota of wild and domestic turkeys. Results suggest that free ranging wild turkeys carry a distinct microbiome when compared to farm raised turkeys. The microbiome of wild birds contains very low levels of poultry pathogens compared to farm raised birds. The microbiomes of wild turkeys may be used to guide development of new ways to control disease in large scale poultry production.

[Source](https://doi.org/10.1128/AEM.01423-21)

10.1128/AEM.01423-21

---

## Comparative Evaluation of the Effect of Metformin and Insulin on Gut Microbiota and Metabolome Profiles of Type 2 Diabetic Rats Induced by the Combination of Streptozotocin and High-Fat Diet.
 20.01.2022, _N Hu, Q Zhang, H Wang, X Yang, Y Jiang, R Chen, L Wang_


Lately, an increasing number of studies have investigated the relationship between metformin and gut microbiota, suggesting that metformin exerts part of its hypoglycemic effect through the microbes. However, its underlying mechanism remains largely undetermined. In the present study, we investigated the effects of metformin on gut microbiota and metabolome profiles in serum and compared it with insulin treatment in rats with type 2 diabetes mellitus (T2DM). Diabetic rats (DM group) were induced by a combination of streptozotocin and high-fat diet (HFD). After 7 days, DM rats were treated with metformin (MET group) or insulin (INS group) for 3 weeks. The 16S rRNA sequencing of the gut microbiota and non-targeted metabolomics analysis of serum were conducted. A total of 13 bile acids (BAs) in serum were further determined and compared among different groups. The rat model of T2DM was well established with the typical diabetic symptoms, showing significantly increased blood glucose, AUC of OGTT, HOMA-IR, TC, TG, LDL-C and TBA. Metformin or insulin treatment could ameliorate symptoms of diabetes and partly recover the abnormal biochemical indicators. Compared with DM rats, the relative abundances of 13 genera were significantly changed after metformin treatment, while only three genera were changed after insulin treatment. The metformin and insulin treatments also exhibited different serum metabolome profiles in T2DM rats. Moreover, 64 differential metabolites were identified between MET and DM groups, whereas 206 were identified between INS and DM groups. Insulin treatment showed greater influence on amino acids, glycerophospholipids/glycerolipids, and acylcarnitine compared with the metformin treatment, while metformin had an important impact on BAs. Furthermore, metformin could significantly decrease the serum levels of CA, GCA, UDCA, and GUDCA, but increase the level of TLCA in DM rats. Insulin treatment significantly decreased the levels of CA, UDCA, and CDCA. Besides, several metabolites in serum or microbiota were positively or negatively correlated with some bacteria. Collectively, our findings indicated that metformin had a stronger effect on gut microbiota than insulin, while insulin treatment showed greater influence on serum metabolites, which provided novel insights into the therapeutic effects of metformin on diabetes.

[Source](https://doi.org/10.3389/fphar.2021.794103)

10.3389/fphar.2021.794103

---

## Modulating the rhizosphere microbiome by altering the cocktail of root secretions.
 20.01.2022, _W Zhang, GA Mason_



[Source](https://doi.org/10.1093/plphys/kiab480)

10.1093/plphys/kiab480

---

## Characterization of Tobacco Microbiome by Metagenomics Approach.
 20.01.2022, _RS Kumar, N Mishra, A Kumar_


Chronic consumption of tobacco in all forms, either smoked/smokeless forms, causes major health hazards to humans that include cancer, cardiovascular, lung diseases, diabetes, fertility issues, etc. Among tobacco-mediated cancers, the prominent one being the oral cancers are caused due to chronic tobacco chewing. The biochemicals present in tobacco are involved in carcinogenesis, and their presence is partly mediated by the existence of microbes in tobacco products. The microbial characterization has been evolved from classical microscopical observation to the recent development of 16S rRNA sequencing by next-generation sequencing methods. The metagenomics approach using 16S rRNA-based next-generation sequencing methods enables the detection and characterization of the complete microbial community of tobacco, including both cultivable and non-cultivable microorganisms. Identification of microbes will help in devising strategies to limit the carcinogenic compounds present in tobacco.

[Source](https://doi.org/10.1007/978-1-0716-1896-7_22)

10.1007/978-1-0716-1896-7_22

---

## Incorporating genome-based phylogeny and functional similarity into diversity assessments helps to resolve a global collection of human gut metagenomes.
 20.01.2022, _ND Youngblut, J de la Cuesta-Zuluaga, RE Ley_


Tree-based diversity measures incorporate phylogenetic or functional relatedness into comparisons of microbial communities. This can improve the identification of explanatory factors compared to tree-agnostic diversity measures. However, applying tree-based diversity measures to metagenome data is more challenging than for single-locus sequencing (e.g., 16S rRNA gene). Utilizing the Genome Taxonomy Database (GTDB) for species-level metagenome profiling allows for functional diversity measures based on genomic content or traits inferred from it. Still, it is unclear how metagenome-based assessments of microbiome diversity benefit from incorporating phylogeny or function into measures of diversity. We assessed this by measuring phylogeny-based, function-based, and tree-agnostic diversity measures from a large, global collection of human gut metagenomes composed of 30 studies and 2943 samples. We found tree-based measures to explain phenotypic variation (e.g., westernization, disease status, and gender) better or equivalent to tree-agnostic measures. Ecophylogenetic and functional diversity measures provided unique insight into how microbiome diversity was partitioned by phenotype. Tree-based measures greatly improved machine learning model performance for predicting westernization, disease status, and gender, relative to models trained solely on tree-agnostic measures. Our findings illustrate the usefulness of tree- and function-based measures for metagenomic assessments of microbial diversity, which is a fundamental component of microbiome science. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1111/1462-2920.15910)

10.1111/1462-2920.15910

---

## Single-cell Microbiomics Unveils Distribution and Patterns of Microbial Symbioses in the Natural Environment.
 20.01.2022, _V Boscaro, V Manassero, PJ Keeling, C Vannini_


Protist-bacteria associations are extremely common. Among them, those involving ciliates of the genus Euplotes are emerging as models for symbioses between prokaryotes and eukaryotes, and a great deal of information is available from cultured representatives of this system. Even so, as for most known microbial symbioses, data on natural populations is lacking, and their ecology remains largely unexplored; how well lab cultures represent actual diversity is untested. Here, we describe a survey on natural populations of Euplotes based on a single-cell microbiomic approach, focusing on taxa that include known endosymbionts of this ciliate. The results reveal an unexpected variability in symbiotic communities, with individual hosts of the same population harboring different sets of bacterial endosymbionts. Co-occurring Euplotes individuals of the same population can even have different essential symbionts, Polynucleobacter and "Candidatus Protistobacter," which might suggest that replacement events could be more frequent in nature than previously hypothesized. Accessory symbionts are even more variable: some showed a strong affinity for one host species, some for a sampling site, and two ("Candidatus Cyrtobacter" and "Candidatus Anadelfobacter") displayed an unusual pattern of competitive exclusion. These data represent the first insight into the prevalence and patterns of bacterial symbionts in natural populations of free-living protists.

[Source](https://doi.org/10.1007/s00248-021-01938-x)

10.1007/s00248-021-01938-x

---

## Adaptive and powerful microbiome multivariate association analysis via feature selection.
 20.01.2022, _K Banerjee, J Chen, X Zhan_


The important role of human microbiome is being increasingly recognized in health and disease conditions. Since microbiome data is typically high dimensional, one popular mode of statistical association analysis for microbiome data is to pool individual microbial features into a group, and then conduct group-based multivariate association analysis. A corresponding challenge within this approach is to achieve adequate power to detect an association signal between a group of microbial features and the outcome of interest across a wide range of scenarios. Recognizing some existing methods' susceptibility to the adverse effects of noise accumulation, we introduce the Adaptive Microbiome Association Test (AMAT), a novel and powerful tool for multivariate microbiome association analysis, which unifies both blessings of feature selection in high-dimensional inference and robustness of adaptive statistical association testing. AMAT first alleviates the burden of noise accumulation via distance correlation learning, and then conducts a data-adaptive association test under the flexible generalized linear model framework. Extensive simulation studies and real data applications demonstrate that AMAT is highly robust and often more powerful than several existing methods, while preserving the correct type I error rate. A free implementation of AMAT in R computing environment is available at https://github.com/kzb193/AMAT.

[Source](https://doi.org/10.1093/nargab/lqab120)

10.1093/nargab/lqab120

---

## Metabolic modeling of host-microbe interactions for therapeutics in colorectal cancer.
 20.01.2022, _P Bhalla, R Rengaswamy, D Karunagaran, GK Suraishkumar, S Sahoo_


The onset of colorectal cancer (CRC) is often attributed to gut bacterial dysbiosis, and thus gut microbiota are highly relevant in devising treatment strategies. Certain gut microbes, like Enterococcus spp., exhibit remarkable anti-neoplastic and probiotic properties, which can aid in silver nanoparticle (AgNPs) induced reactive oxygen species (ROS)-based CRC treatment. However, the effects of AgNPs on gut microbial metabolism have not been reported thus far. In this study, a detailed systems-level understanding of ROS metabolism in Enterococcus durans (E. durans), a representative gut microbe, was gained using constraint-based modeling, wherein, the critical association between ROS and folate metabolism was established. Experimental studies involving low AgNP concentration treatment of E. durans cultures confirmed these modeling predictions (an increased extracellular folate concentration by 52%, at the 9th h of microbial growth, was observed). Besides, the computational studies established various metabolic pathways involving amino acids, energy metabolites, nucleotides, and SCFAs as the key players in elevating folate levels on ROS exposure. The anti-cancer potential of E. durans was also studied through MTT analysis of HCT 116 cells treated with microbial culture (AgNP treated) supernatant. A decrease in cell viability by 19% implicated the role of microbial metabolites (primarily folate) in causing cell death. The genome-scale modeling approach was then extended to extensively model CRC metabolism, as well as CRC-E. durans interactions in the context of CRC treatment, using tissue-specific metabolic models of CRC and healthy colon. These findings on further validation can facilitate the development of robust and effective cancer therapy.

[Source](https://doi.org/10.1038/s41540-021-00210-9)

10.1038/s41540-021-00210-9

---

