## Metabolomics-Microbiome Crosstalk in the Breast Cancer Microenvironment.
 25.11.2021, _MM Al-Ansari, RH AlMalki, LA Dahabiyeh, AM Abdel Rahman_


Breast cancer, the most frequent cancer diagnosed among females, is associated with a high mortality rate worldwide. Alterations in the microbiota have been linked with breast cancer development, suggesting the possibility of discovering disease biomarkers. Metabolomics has emerged as an advanced promising analytical approach for profiling metabolic features associated with breast cancer subtypes, disease progression, and response to treatment. The microenvironment compromises non-cancerous cells such as fibroblasts and influences cancer progression with apparent phenotypes. This review discusses the role of metabolomics in studying metabolic dysregulation in breast cancer caused by the effect of the tumor microenvironment on multiple cells such as immune cells, fibroblasts, adipocytes, etc. Breast tumor cells have a unique metabolic profile through the elevation of glycolysis and the tricarboxylic acid cycle metabolism. This metabolic profile is highly sensitive to microbiota activity in the breast tissue microenvironment. Metabolomics shows great potential as a tool for monitoring metabolic dysregulation in tissue and associating the findings with microbiome expression.

[Source](https://www.mdpi.com/2218-1989/11/11/758)

10.3390/metabo11110758

---

## Aspergillus fumigatus pan-genome analysis identifies genetic variants associated with human infection.
 25.11.2021, _AE Barber, T Sae-Ong, K Kang, B Seelbinder, J Li, G Walther, G Panagiotou, O Kurzai_


Aspergillus fumigatus is an environmental saprobe and opportunistic human fungal pathogen. Despite an estimated annual occurrence of more than 300,000 cases of invasive disease worldwide, a comprehensive survey of the genomic diversity present in A. fumigatus-including the relationship between clinical and environmental isolates and how this genetic diversity contributes to virulence and antifungal drug resistance-has been lacking. In this study we define the pan-genome of A. fumigatus using a collection of 300 globally sampled genomes (83 clinical and 217 environmental isolates). We found that 7,563 of the 10,907 unique orthogroups (69%) are core and present in all isolates and the remaining 3,344 show presence/absence of variation, representing 16-22% of the genome of each isolate. Using this large genomic dataset of environmental and clinical samples, we found an enrichment for clinical isolates in a genetic cluster whose genomes also contain more accessory genes, including genes coding for transmembrane transporters and proteins with iron-binding activity, and genes involved in both carbohydrate and amino-acid metabolism. Finally, we leverage the power of genome-wide association studies to identify genomic variation associated with clinical isolates and triazole resistance as well as characterize genetic variation in known virulence factors. This characterization of the genomic diversity of A. fumigatus allows us to move away from a single reference genome that does not necessarily represent the species as a whole and better understand its pathogenic versatility, ultimately leading to better management of these infections.

[Source](https://www.nature.com/articles/s41564-021-00993-x)

10.1038/s41564-021-00993-x

---

## Human gut bacterial metabolism drives Th17 activation and colitis.
 25.11.2021, _M Alexander, QY Ang, RR Nayak, AE Bustion, M Sandy, B Zhang, V Upadhyay, KS Pollard, SV Lynch, PJ Turnbaugh_


Bacterial activation of T helper 17 (Th17) cells exacerbates mouse models of autoimmunity, but how human-associated bacteria impact Th17-driven disease remains elusive. We show that human gut Actinobacterium Eggerthella lenta induces intestinal Th17 activation by lifting inhibition of the Th17 transcription factor Rorγt through cell- and antigen-independent mechanisms. E. lenta is enriched in inflammatory bowel disease (IBD) patients and worsens colitis in a Rorc-dependent manner in mice. Th17 activation varies across E. lenta strains, which is attributable to the cardiac glycoside reductase 2 (Cgr2) enzyme. Cgr2 is sufficient to induce interleukin (IL)-17a, a major Th17 cytokine. cgr2+ E. lenta deplete putative steroidal glycosides in pure culture; related compounds are negatively associated with human IBD severity. Finally, leveraging the sensitivity of Cgr2 to dietary arginine, we prevented E. lenta-induced intestinal inflammation in mice. Together, these results support a role for human gut bacterial metabolism in driving Th17-dependent autoimmunity.

[Source](https://www.sciencedirect.com/science/article/pii/S1931312821005072)

10.1016/j.chom.2021.11.001

---

## Integrative Analysis of Metabolome and Microbiome in Patients with Progressive Alcohol-Associated Liver Disease.
 25.11.2021, _B Gao, Y Zhu, N Gao, W Shen, P Stärkel, B Schnabl_


Alcohol-associated liver disease is one of the most prevalent diseases around the world, with 10-20% of patients developing progressive liver disease. To identify the complex and correlated nature of metabolic and microbial data types in progressive liver disease, we performed an integrated analysis of the fecal and serum metabolomes with the gut microbiome in a cohort of 38 subjects, including 15 patients with progressive liver disease, 16 patients with non-progressive liver disease, and 7 control subjects. We found that although patients were generally clustered in three groups according to disease status, metabolites showed better separation than microbial species. Furthermore, eight serum metabolites were correlated with two microbial species, among which seven metabolites were decreased in patients with progressive liver disease. Five fecal metabolites were correlated with three microbial species, among which four metabolites were decreased in patients with progressive liver disease. When predicting progressive liver disease from non-progressive liver disease using correlated metabolic and microbial signatures with the random forest model, correlated serum metabolites and microbial species showed great predictive power, with the area under the receiver operating characteristic curve achieving 0.91. The multi-omics signatures identified in this study are helpful for the early identification of patients with progressive alcohol-associated liver disease, which is a key step for therapeutic intervention.

[Source](https://www.mdpi.com/2218-1989/11/11/766)

10.3390/metabo11110766

---

## Human Fecal Metabolome Reflects Differences in Body Mass Index, Physical Fitness, and Blood Lipoproteins in Healthy Older Adults.
 25.11.2021, _M Cui, A Trimigno, JL Castro-Mejía, S Reitelseder, J Bülow, RL Bechshøft, DS Nielsen, L Holm, SB Engelsen, B Khakimov_


This study investigated how body mass index (BMI), physical fitness, and blood plasma lipoprotein levels are related to the fecal metabolome in older adults. The fecal metabolome data were acquired using proton nuclear magnetic resonance spectroscopy and gas chromatography-mass spectrometry on 163 healthy older adults (65-80 years old, 80 females and 83 males). Overweight and obese subjects (BMI ≥ 27) showed higher levels of fecal amino acids (AAs) (valine, alanine, and phenylalanine) compared to normal-weight subjects (BMI ≤ 23.5). Adults classified in the high-fitness group displayed slightly lower concentrations of fecal short-chain fatty acids, propionic acid, and AAs (methionine, leucine, glutamic acid, and threonine) compared to the low-fitness group. Subjects with lower levels of cholesterol in low-density lipoprotein particles (LDLchol, ≤2.6 mmol/L) displayed higher fecal levels of valine, glutamic acid, phenylalanine, and lactic acid, while subjects with a higher level of cholesterol in high-density lipoprotein particles (HDLchol, ≥2.1 mmol/L) showed lower fecal concentration of isovaleric acid. The results from this study suggest that the human fecal metabolome, which primarily represents undigested food waste and metabolites produced by the gut microbiome, carries important information about human health and should be closely integrated to other omics data for a better understanding of the role of the gut microbiome and diet on human health and metabolism.

[Source](https://www.mdpi.com/2218-1989/11/11/717)

10.3390/metabo11110717

---

## Gut Microbiota, Microbial Metabolites and Human Physical Performance.
 25.11.2021, _S Lensu, S Pekkala_


Trillions of microbes inhabiting the gut modulate the metabolism of the host. Cross-sectional studies have reported associations between physical performance and the gut microbiota (GM). Physical activity seems to increase GM diversity and the abundance of certain health-beneficial microbes. We reviewed the evidence from longitudinal studies on the connection between physically active lifestyle or long-term exercise interventions and the GM. We made literature searches using databases of Web of Science and PubMed Medline to collect human studies showing or not the associations between the GM and exercise. Many controversies exist in the studies. However, the longitudinal studies show that frequently, medium-intensity endurance exercise has yielded most beneficial effects on the GM, but the results vary depending on the study population and exercise protocol. In addition, the literature shows that certain microbes own the potency to increase physical activity and performance. Generally, a physically active lifestyle and exercise associate with a "healthy" GM. However, in previously sedentary subjects, the exercise-induced improvements in the GM seem to disappear unless the active lifestyle is continued. Unfortunately, several studies are not controlled for the diet. Thus, in the future, more longitudinal studies on the GM and physical performance are needed, with detailed dietary information.

[Source](https://www.mdpi.com/2218-1989/11/11/716)

10.3390/metabo11110716

---

## Next steps after 15 stimulating years of human gut microbiome research.
 24.11.2021, _T Clavel, HP Horz, N Segata, M Vehreschild_


Gut microbiome research has bloomed over the past 15 years. We have learnt a lot about the complex microbial communities that colonize our intestine. Promising avenues of research and microbiome-based applications are being implemented, with the goal of sustaining host health and applying personalized disease management strategies. Despite this exciting outlook, many fundamental questions about enteric microbial ecosystems remain to be answered. Organizational measures will also need to be taken to optimize the outcome of discoveries happening at an extremely rapid pace. This article highlights our own view of the field and perspectives for the next 15 years.

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1751-7915.13970)

10.1111/1751-7915.13970

---

## The microbiome, Gastrointestinal cancer and immunotherapy.
 25.11.2021, _RC Newsome, Y Yang, C Jobin_


The gastrointestinal tract greatly contributes to global cancer burden and cancer-related deaths. The microbiota represents the population of microorganisms that live in and around the body, located primarily in the gastrointestinal tract. The microbiota has been implicated in colorectal cancer development and progression, but its role in cancer therapy for the gastrointestinal tract is less defined, especially for extra-intestinal cancers. In this review, we discuss the past 5 years of research into microbial involvement in immune-related therapies for colorectal, pancreatic, hepatic and gastric cancers, with the goal of highlighting recent advances and new areas for investigation in this field.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/jgh.15742)

10.1111/jgh.15742

---

## Mycobiota-induced IgA antibodies regulate fungal commensalism in the gut and are dysregulated in Crohn's disease.
 25.11.2021, _I Doron, M Mesko, XV Li, T Kusakabe, I Leonardi, DG Shaw, WD Fiers, WY Lin, M Bialt-DeCelie, E Román, RS Longman, J Pla, PC Wilson, ID Iliev_


Secretory immunoglobulin A (sIgA) plays an important role in gut barrier protection by shaping the resident microbiota community, restricting the growth of bacterial pathogens and enhancing host protective immunity via immunological exclusion. Here, we found that a portion of the microbiota-driven sIgA response is induced by and directed towards intestinal fungi. Analysis of the human gut mycobiota bound by sIgA revealed a preference for hyphae, a fungal morphotype associated with virulence. Candida albicans was a potent inducer of IgA class-switch recombination among plasma cells, via an interaction dependent on intestinal phagocytes and hyphal programming. Characterization of sIgA affinity and polyreactivity showed that hyphae-associated virulence factors were bound by these antibodies and that sIgA influenced C. albicans morphotypes in the murine gut. Furthermore, an increase in granular hyphal morphologies in patients with Crohn's disease compared with healthy controls correlated with a decrease in antifungal sIgA antibody titre with affinity to two hyphae-associated virulence factors. Thus, in addition to its importance in gut bacterial regulation, sIgA targets the uniquely fungal phenomenon of hyphal formation. Our findings indicate that antifungal sIgA produced in the gut can play a role in regulating intestinal fungal commensalism by coating fungal morphotypes linked to virulence, thereby providing a protective mechanism that might be dysregulated in patients with Crohn's disease.

[Source](https://www.nature.com/articles/s41564-021-00983-z)

10.1038/s41564-021-00983-z

---

## Fecal microbiota transplantation protects rotenone-induced Parkinson's disease mice via suppressing inflammation mediated by the lipopolysaccharide-TLR4 signaling pathway through the microbiota-gut-brain axis.
 20.11.2021, _Z Zhao, J Ning, XQ Bao, M Shang, J Ma, G Li, D Zhang_


Parkinson's disease (PD) is a prevalent neurodegenerative disorder, displaying not only well-known motor deficits but also gastrointestinal dysfunctions. Consistently, it has been increasingly evident that gut microbiota affects the communication between the gut and the brain in PD pathogenesis, known as the microbiota-gut-brain axis. As an approach to re-establishing a normal microbiota community, fecal microbiota transplantation (FMT) has exerted beneficial effects on PD in recent studies. Here, in this study, we established a chronic rotenone-induced PD mouse model to evaluate the protective effects of FMT treatment on PD and to explore the underlying mechanisms, which also proves the involvement of gut microbiota dysbiosis in PD pathogenesis via the microbiota-gut-brain axis.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01107-9)

10.1186/s40168-021-01107-9

---

## Xylan alleviates dietary fiber deprivation-induced dysbiosis by selectively promoting Bifidobacterium pseudocatenulatum in pigs.
 25.11.2021, _Z Wang, Y Bai, Y Pi, WJJ Gerrits, S de Vries, L Shang, S Tao, S Zhang, D Han, Z Zhu, J Wang_


Low dietary fiber intake has been shown to disturb the gut microbiome community, damage the mucus barrier, and promote pathogen susceptibility. However, little is known about the temporal response of the gut microbiome to dietary fiber deprivation and the recovery induced by dietary fiber inclusion in pigs.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01175-x)

10.1186/s40168-021-01175-x

---

## Circadian dynamics of the teleost skin immune-microbiome interface.
 19.11.2021, _AR Ellison, D Wilcockson, J Cable_


Circadian rhythms of host immune activity and their microbiomes are likely pivotal to health and disease resistance. The integration of chronotherapeutic approaches to disease mitigation in managed animals, however, is yet to be realised. In aquaculture, light manipulation is commonly used to enhance growth and control reproduction but may have unknown negative consequences for animal health. Infectious diseases are a major barrier to sustainable aquaculture and understanding the circadian dynamics of fish immunity and crosstalk with the microbiome is urgently needed.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01160-4)

10.1186/s40168-021-01160-4

---

## Honey bee genetics shape the strain-level structure of gut microbiota in social transmission.
 20.11.2021, _J Wu, H Lang, X Mu, Z Zhang, Q Su, X Hu, H Zheng_


Honey bee gut microbiota transmitted via social interactions are beneficial to the host health. Although the microbial community is relatively stable, individual variations and high strain-level diversity have been detected across honey bees. Although the bee gut microbiota structure is influenced by environmental factors, the heritability of the gut members and the contribution of the host genetics remains elusive. Considering bees within a colony are not readily genetically identical due to the polyandry of the queen, we hypothesize that the microbiota structure can be shaped by host genetics.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01174-y)

10.1186/s40168-021-01174-y

---

## Evidence for host-microbiome co-evolution in apple.
 25.11.2021, _A Abdelfattah, AJM Tack, B Wasserman, J Liu, G Berg, J Norelli, S Droby, M Wisniewski_


Plants evolved in association with a diverse community of microorganisms. The effect of plant phylogeny and domestication on host-microbiome co-evolutionary dynamics are poorly understood. Here we examined the effect of domestication and plant lineage on the composition of the endophytic microbiome of 11 Malus species, representing three major groups: domesticated apple (M. domestica), wild apple progenitors, and wild Malus species. The endophytic community of M. domestica and its wild progenitors showed higher microbial diversity and abundance than wild Malus species. Heirloom and modern cultivars harbored a distinct community composition, though the difference was not significant. A community-wide Bayesian model revealed that the endophytic microbiome of domesticated apple is an admixture of its wild progenitors, with clear evidence for microbiome introgression, especially for the bacterial community. We observed a significant correlation between the evolutionary distance of Malus species and their microbiome. This study supports co-evolution between Malus species and their microbiome during domestication. This finding has major implications for future breeding programs and our understanding of the evolution of plants and their microbiomes.

[Source](https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.17820)

10.1111/nph.17820

---

## Protective role of the Arabidopsis leaf microbiota against a bacterial pathogen.
 25.11.2021, _CM Vogel, DB Potthoff, M Schäfer, N Barandun, JA Vorholt_


The aerial parts of plants are host to taxonomically structured bacterial communities. Members of the core phyllosphere microbiota can protect Arabidopsis thaliana against foliar pathogens. However, whether plant protection is widespread and to what extent the modes of protection differ among phyllosphere microorganisms are not clear. Here, we present a systematic analysis of plant protection capabilities of the At-LSPHERE, which is a collection of &gt;200 bacterial isolates from A. thaliana, against the bacterial pathogen Pseudomonas syringae pv. tomato DC3000. In total, 224 bacterial leaf isolates were individually assessed for plant protection in a gnotobiotic system. Protection against the pathogen varied, with ~10% of leaf microbiota strains providing full protection, ~10% showing intermediate levels of protection and the remaining ~80% not markedly reducing disease phenotypes upon infection. The most protective strains were distributed across different taxonomic groups. Synthetic community experiments revealed additive effects of strains but also that a single strain can confer full protection in a community context. We also identify different mechanisms that contribute to plant protection. Although pattern-triggered immunity coreceptor signalling is involved in protection by a subset of strains, other strains protected in the absence of functional plant immunity receptors BAK1 and BKK1. Using a comparative genomics approach combined with mutagenesis, we reveal that direct bacteria-pathogen interactions contribute to plant protection by Rhizobium Leaf202. This shows that a computational approach based on the data provided can be used to identify genes of the microbiota that are important for plant protection.

[Source](https://www.nature.com/articles/s41564-021-00997-7)

10.1038/s41564-021-00997-7

---

## Emergent bacterial community properties induce enhanced drought tolerance in Arabidopsis.
 22.11.2021, _N Yang, J Nesme, HL Røder, X Li, Z Zuo, M Petersen, M Burmølle, SJ Sørensen_


Drought severely restricts plant production and global warming is further increasing drought stress for crops. Much information reveals the ability of individual microbes affecting plant stress tolerance. However, the effects of emergent bacterial community properties on plant drought tolerance remain largely unexplored. Here, we inoculated Arabidopsis plants in vivo with a four-species bacterial consortium (Stenotrophomonas rhizophila, Xanthomonas retroflexus, Microbacterium oxydans, and Paenibacillus amylolyticus, termed as SPMX), which is able to synergistically produce more biofilm biomass together than the sum of the four single-strain cultures, to investigate its effects on plant performance and rhizo-microbiota during drought. We found that SPMX remarkably improved Arabidopsis survival post 21-day drought whereas no drought-tolerant effect was observed when subjected to the individual strains, revealing emergent properties of the SPMX consortium as the underlying cause of the induced drought tolerance. The enhanced drought tolerance was associated with sustained chlorophyll content and endogenous abscisic acid (ABA) signaling. Furthermore, our data showed that the addition of SPMX helped to stabilize the diversity and structure of root-associated microbiomes, which potentially benefits plant health under drought. These SPMX-induced changes jointly confer an increased drought tolerance to plants. Our work may inform future efforts to engineer the emergent bacterial community properties to improve plant tolerance to drought.

[Source](https://www.nature.com/articles/s41522-021-00253-0)

10.1038/s41522-021-00253-0

---

## Cross-regional scale pollution of freshwater biofilms unveiled by antibiotic resistance genes.
 25.11.2021, _Y Yao, Z Liu, KK Yip, Y Pu, W Cheng, M Li, O Habimana_


A comprehensive global profile of the distribution of ARGs in freshwater biofilms is lacking. We utilized metagenomic approaches to reveal the diversity, abundance, transferability and hosts of ARGs in 96 freshwater biofilm samples from 38 sampling sites across four countries. The abundant ARGs were associated with bacitracin, multidrug, polymyxin macrolide-lincosamide-streptogramin (MLS) aminoglycoside, β-lactam, chloramphenicol, sulfonamide and tetracycline resistance, consistent with the spectrum of antibiotics commonly used in human or veterinary medicine. As expected, the resistome in freshwater biofilm habitats was significantly influenced by geographical location and human footprint. Based on the co-occurrence pattern revealed by network analysis, mdtC, kdpE, and emrB were proposed as ARG indicators in freshwater biofilms that can be used to evaluate the abundance of 46 other co-occurring ARG subtypes quantitatively. Metagenomic assembly analysis revealed that the identified ARGs were hosted by more than 46 bacterial phyla, including various pathogens, which greatly expands the knowledge of resistome diversity in freshwater biofilms. Our study points to the central roles of biofilms in harbouring ARGs. The results could enhance understanding the distribution of ARGs in freshwater habitats, thereby strengthening the global environmental risk assessment and management of ARGs.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969721069114)

10.1016/j.scitotenv.2021.151835

---

## Functional metagenomic analysis of quorum sensing signaling in a nitrifying community

[Source](https://www.nature.com/articles/s41522-021-00250-3)

---

## Multivariable association discovery in population-scale meta-omics studies.
 16.11.2021, _H Mallick, A Rahnavard, LJ McIver, S Ma, Y Zhang, LH Nguyen, TL Tickle, G Weingart, B Ren, EH Schwager, S Chatterjee, KN Thompson, JE Wilkinson, A Subramanian, Y Lu, L Waldron, JN Paulson, EA Franzosa, HC Bravo, C Huttenhower_


It is challenging to associate features such as human health outcomes, diet, environmental conditions, or other metadata to microbial community measurements, due in part to their quantitative properties. Microbiome multi-omics are typically noisy, sparse (zero-inflated), high-dimensional, extremely non-normal, and often in the form of count or compositional measurements. Here we introduce an optimized combination of novel and established methodology to assess multivariable association of microbial community features with complex metadata in population-scale observational studies. Our approach, MaAsLin 2 (Microbiome Multivariable Associations with Linear Models), uses generalized linear and mixed models to accommodate a wide variety of modern epidemiological studies, including cross-sectional and longitudinal designs, as well as a variety of data types (e.g., counts and relative abundances) with or without covariates and repeated measurements. To construct this method, we conducted a large-scale evaluation of a broad range of scenarios under which straightforward identification of meta-omics associations can be challenging. These simulation studies reveal that MaAsLin 2's linear model preserves statistical power in the presence of repeated measures and multiple covariates, while accounting for the nuances of meta-omics features and controlling false discovery. We also applied MaAsLin 2 to a microbial multi-omics dataset from the Integrative Human Microbiome (HMP2) project which, in addition to reproducing established results, revealed a unique, integrated landscape of inflammatory bowel diseases (IBD) across multiple time points and omics profiles.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009442)

10.1371/journal.pcbi.1009442

---

## Assessing saliva microbiome collection and processing methods.
 22.11.2021, _AJS Armstrong, V Parmar, MJ Blaser_


The oral microbiome has been connected with lung health and may be of significance in the progression of SARS-CoV-2 infection. Saliva-based SARS-CoV-2 tests provide the opportunity to leverage stored samples for assessing the oral microbiome. However, these collection kits have not been tested for their accuracy in measuring the oral microbiome. Saliva is highly enriched with human DNA and reducing it prior to shotgun sequencing may increase the depth of bacterial reads. We examined both the effect of saliva collection method and sequence processing on measurement of microbiome depth and diversity by 16S rRNA gene amplicon and shotgun metagenomics. We collected 56 samples from 22 subjects. Each subject provided saliva samples with and without preservative, and a subset provided a second set of samples the following day. 16S rRNA gene (V4) sequencing was performed on all samples, and shotgun metagenomics was performed on a subset of samples collected with preservative with and without human DNA depletion before sequencing. We observed that the beta diversity distances within subjects over time was smaller than between unrelated subjects, and distances within subjects were smaller in samples collected with preservative. Samples collected with preservative had higher alpha diversity measuring both richness and evenness. Human DNA depletion before extraction and shotgun sequencing yielded higher total and relative reads mapping to bacterial sequences. We conclude that collecting saliva with preservative may provide more consistent measures of the oral microbiome and depleting human DNA increases yield of bacterial sequences.

[Source](https://www.nature.com/articles/s41522-021-00254-z)

10.1038/s41522-021-00254-z

---

