## Commensal bacterial outer membrane protein A induces interleukin-22 production.
 01.06.2024, _Y Wang, VL Ngo, J Zou, AT Gewirtz_


Interleukin (IL)-22 promotes host-microbiota homeostasis. We sought to identify microbiota metabolite(s) that drive intestinal IL-22 production. We observed that exposing Peyer's patch cells (PPCs), ex vivo, to fecal supernatants (FSs) recapitulates fermentable fiber- and microbiota-dependent IL-22 production, and cellular sources thereof, thus supporting the use of this model. An interrogation of FSs generated from mice fed the fermentable fiber inulin (FS-Inu) revealed that its IL-22-inducing activity is mediated by heat-labile protein. Fractionation of FS-Inu by ion-exchange chromatography, and subsequent proteomic analysis of IL-22-inducing fractions, indicates that outer membrane protein A (OmpA) might be a microbial driver of IL-22 expression. Concomitantly, recombinant OmpA from Parabacteroides goldsteinii, which is enriched by an inulin diet, induces IL-22 production and expression of the IL-22-dependent genes REG3γ and -β, in PPCs and mice. Thus, OmpA is one bacterial inducer of IL-22 expression, potentially linking diet, mucosal immune homeostasis, and gut health.

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(24)00620-X)

10.1016/j.celrep.2024.114292

---

## Mucus production, host-microbiome interactions, hormone sensitivity, and innate immune responses modeled in human cervix chips.
 01.06.2024, _Z Izadifar, J Cotton, S Chen, V Horvath, A Stejskalova, A Gulati, NT LoGrande, B Budnik, S Shahriar, ER Doherty, Y Xie, T To, SE Gilpin, AM Sesay, G Goyal, CB Lebrilla, DE Ingber_


_Humans, Female, Cervix Uteri, Immunity, Innate, Microbiota, Host Microbial Interactions, Gardnerella vaginalis, Lactobacillus crispatus, Mucus, Lab-On-A-Chip Devices_
Modulation of the cervix by steroid hormones and commensal microbiome play a central role in the health of the female reproductive tract. Here we describe organ-on-a-chip (Organ Chip) models that recreate the human cervical epithelial-stromal interface with a functional epithelial barrier and production of mucus with biochemical and hormone-responsive properties similar to living cervix. When Cervix Chips are populated with optimal healthy versus dysbiotic microbial communities (dominated by Lactobacillus crispatus and Gardnerella vaginalis, respectively), significant differences in tissue innate immune responses, barrier function, cell viability, proteome, and mucus composition are observed that are similar to those seen in vivo. Thus, human Cervix Organ Chips represent physiologically relevant in vitro models to study cervix physiology and host-microbiome interactions, and hence may be used as a preclinical testbed for development of therapeutic interventions to enhance women's health.

[Source](https://doi.org/10.1038/s41467-024-48910-0)

10.1038/s41467-024-48910-0

---

## Gut microbiome and symptoms in females with irritable bowel syndrome: a cross-sectional analysis.
 31.05.2024, _KJ Kamp, AM Plantinga, KC Cain, RL Burr, CS Tsai, Q Wu, SY So, S Badu, T Savidge, RJ Shulman, MM Heitkemper_


_Humans, Irritable Bowel Syndrome, Female, Gastrointestinal Microbiome, Adult, Cross-Sectional Studies, Young Adult, RNA, Ribosomal, 16S, Adolescent, Middle Aged, Feces, Bacteria_
Irritable bowel syndrome (IBS), a disorder of gut-brain interaction, is associated with abdominal pain and stool frequency/character alterations that are linked to changes in microbiome composition. We tested whether taxa differentially abundant between females with IBS vs healthy control females (HC) are associated with daily gastrointestinal and psychological symptom severity. Participants (age 18-50 year) completed a 3-day food record and collected a stool sample during the follicular phase. They also completed a 28-day diary rating symptom intensity; analysis focused on the three days after the stool sample collection. 16S rRNA gene sequencing was used for bacterial identification. Taxon abundance was compared between IBS and HC using zero-inflated quantile analysis (ZINQ). We found that females with IBS (n = 67) had greater Bacteroides abundance (q = 0.003) and lower odds of Bifidobacterium presence (q = 0.036) compared to HC (n = 46) after adjusting for age, race, body mass index, fibre intake, and hormonal contraception use. Intestimonas, Oscillibacter, and Phascolarctobacterium were more often present and Christensenellaceae R-7 group, Collinsella, Coprococcus 2, Moryella, Prevotella 9, Ruminococcaceae UCG-002, Ruminococcaceae UCG-005, and Ruminococcaceae UCG-014 were less commonly present in IBS compared to HC. Despite multiple taxon differences in IBS vs HC, we found no significant associations between taxon presence or abundance and average daily symptom severity within the IBS group. This may indicate the need to account for interactions between microbiome, dietary intake, metabolites, and host factors.

[Source](https://doi.org/10.1163/18762891-bja00015)

10.1163/18762891-bja00015

---

## Gut microbiome in the Graves' disease: Comparison before and after anti-thyroid drug treatment.
 03.06.2024, _C Jeong, H Baek, J Bae, N Hwang, J Ha, YS Cho, DJ Lim_


_Humans, Graves Disease, Gastrointestinal Microbiome, Male, Female, Adult, Antithyroid Agents, Middle Aged, Feces, Case-Control Studies_
While several studies have proposed a connection between the gut microbiome and the pathogenesis of Graves's disease (GD), there has been a lack of reports on alteration in microbiome following using anti-thyroid drug treatment (ATD) to treat GD. Stool samples were collected from newly diagnosed GD patients provided at baseline and after 6 months of ATD treatment. The analysis focused on investigating the association between the changes in the gut microbiome and parameter including thyroid function, thyroid-related antibodies, and the symptom used to assess hyperthyroidism before and after treatment. A healthy control (HC) group consisting of data from 230 healthy subjects (110 males and 120 females) sourced from the open EMBL Nucleotide Sequence Database was included. Twenty-nine GD patients (14 males and 15 females) were enrolled. The analysis revealed a significant reduction of alpha diversity in GD patients. However, after ATD treatment, alpha diversity exhibited a significant increase, restored to levels comparable to the HC levels. Additionally, GD patients displayed lower levels of Firmicutes and higher levels of Bacteroidota. Following treatment, there was an increased in Firmicutes and a decrease in Bacteroidota, resembling levels found in the HC levels. The symptoms of hyperthyroidism were negatively associated with Firmicutes and positively associated with Bacteroidota. GD had significantly lower levels of Roseburia, Lachnospiraceaea, Sutterella, Escherichia-shigella, Parasuterella, Akkermansia, and Phascolarctobacterium compared to HC (all p &lt; 0.05). Post-treatment, Subdoligranulum increased (p = 0.010), while Veillonella and Christensenellaceaea R-7 group decreased (p = 0.023, p = 0.029, respectively). Anaerostipes showed a significant association with both higher smoking pack years and TSHR-Ab levels, with greater abundantce observed in smokers among GD (p = 0.16). Although reduced ratio of Firmicutes/Bacteroidetes was evident in GD, this ratio recovered after treatment. This study postulates the involvement of the gut microbiome in the pathogenesis of GD, suggesting potential restoration after treatment.

[Source](https://doi.org/10.1371/journal.pone.0300678)

10.1371/journal.pone.0300678

---

## Guild-level signature of gut microbiome for diabetic kidney disease.
 31.05.2024, _S Tang, G Wu, Y Liu, B Xue, S Zhang, W Zhang, Y Jia, Q Xie, C Liang, L Wang, H Heng, W Wei, X Shi, Y Hu, J Yang, L Zhao, X Wang, L Zhao, H Yuan_


Current microbiome signatures for chronic diseases such as diabetic kidney disease (DKD) are mainly based on low-resolution taxa such as genus or phyla and are often inconsistent among studies. In microbial ecosystems, bacterial functions are strain specific, and taxonomically different bacteria tend to form co-abundance functional groups called guilds. Here, we identified guild-level signatures for DKD by performing in-depth metagenomic sequencing and conducting genome-centric and guild-based analysis on fecal samples from 116 DKD patients and 91 healthy subjects. Redundancy analysis on 1,543 high-quality metagenome-assembled genomes (HQMAGs) identified 54 HQMAGs that were differentially distributed among the young healthy control group, elderly healthy control group, early-stage DKD patients (EDG), and late-stage DKD patients (LDG). Co-abundance network analysis classified the 54 HQMAGs into two guilds. Compared to guild 2, guild 1 contained more short-chain fatty acid biosynthesis genes and fewer genes encoding uremic toxin indole biosynthesis, antibiotic resistance, and virulence factors. Guild indices, derived from the total abundance of guild members and their diversity, delineated DKD patients from healthy subjects and between different severities of DKD. Age-adjusted partial Spearman correlation analysis showed that the guild indices were correlated with DKD disease progression and with risk indicators of poor prognosis. We further validated that the random forest classification model established with the 54 HQMAGs was also applicable for classifying patients with end-stage renal disease and healthy subjects in an independent data set. Therefore, this genome-level, guild-based microbial analysis strategy may identify DKD patients with different severity at an earlier stage to guide clinical interventions.

[Source](https://doi.org/10.1128/mbio.00735-24)

10.1128/mbio.00735-24

---

## Gut microbiome correlates of recurrent urinary tract infection: a longitudinal, multi-center study

[Source](https://www.thelancet.com/journals/eclinm/article/PIIS2589-5370(24)00069-5/fulltext)

---

## Gut microbiota-derived butyrate induces epigenetic and metabolic reprogramming in myeloid-derived suppressor cells to alleviate primary biliary cholangitis

[Source](https://www.gastrojournal.org/article/S0016-5085(24)04936-9/abstract)

---

## Gut microbiota research nexus: One Health relationship between human, animal, and environmental resistomes

[Source](https://doi.org/10.1002/mlf2.12101)

---

## Exploring the Impact of Airway Microbiome on Asthma Morbidity: A Focus on the Constructing a ‘Eubiosis Reinstatement Therapy’ for Asthma-CURE Project

[Source](https://doi.org/10.1007/s41030-024-00261-3)

---

## A Gram-negative-selective antibiotic that spares the gut microbiome

[Source](https://doi.org/10.1038/s41586-024-07502-0)

---

## <em>Bifidobacterium adolescentis</em>-derived hypaphorine alleviates acetaminophen hepatotoxicity by promoting hepatic Cry1 expression

[Source](https://doi.org/10.1186/s12967-024-05312-6)

---

## <em>Chrysanthemum morifolium</em> attenuates metabolic and alcohol-associated liver disease via gut microbiota and PPARα/γ activation

[Source](https://doi.org/10.1016/j.phymed.2024.155774)

---

## Transgenerational effects of early life stress on the fecal microbiota in mice

[Source](https://doi.org/10.1038/s42003-024-06279-2)

---

## Metagenomics and metaproteomics alterations are associated with kidney disease in opisthorchiasis hamsters fed a high-fat and high-fructose diet

[Source](https://doi.org/10.1371/journal.pone.0301907)

---

## Spatial profiles of the bacterial microbiota throughout the gastrointestinal tract of dairy goats

[Source](https://doi.org/10.1007/s00253-024-13200-8)

---

## Interaction between methanotrophy and gastrointestinal nematodes infection on the rumen microbiome of lambs.
 31.05.2024, _PS Corrêa, MA Fernandes, CR Jimenez, LW Mendes, P de Mello Tavares Lima, AL Abdalla, H Louvandini_


Complex crosstalk occurs between gastrointestinal nematodes and gut symbiotic microbiota, with consequences for animal metabolism. To investigate the connection between methane production and endoparasites, this study evaluated the effect of mixed infection with Haemonchus contortus and Trichostrongylus colubriformis on methanogenic and methanotrophic community in rumen microbiota of lambs using shotgun metagenomic and Real-time quantitative PCR (qPCR). The rumen content was collected from 6 Santa Inês lambs, (7 months old) before and after 42 days infection by esophageal tube. The metagenomic analysis showed that the infection affected the microbial community structure leading to decreased abundance of methanotrophs bacteria, i.e. α-proteobacteria and β-proteobacteria, anaerobic methanotrophic archaea (ANME), protozoa, sulfate-reducing bacteria, syntrophic bacteria with methanogens, geobacter, and genes related to pyruvate, fatty acid, nitrogen, and sulfur metabolisms, ribulose monophosphate cycle and Entner-Doudoroff Pathway. Additionally, the abundance of methanogenic archaea and the mcrA gene did not change. The co-occurrence networks enabled us to identify the interactions between each taxon in microbial communities and to determine the reshaping of rumen microbiome associations by gastrointestinal nematode infection. Besides, the correlation between ANMEs was lower in the animal's post-infection. Our findings suggest that gastrointestinal parasites potentially lead to decreased methanotrophic metabolism-related microorganisms and genes.

[Source](https://doi.org/10.1093/femsec/fiae083)

10.1093/femsec/fiae083

---

## Antibiotic-altered gut microbiota explain host memory plasticity and disrupt pace-of-life covariation for an aquatic snail.
 01.06.2024, _GL Davidson, IA Cienfuegos, S Dalesman_


_Animals, Gastrointestinal Microbiome, Anti-Bacterial Agents, Lymnaea, Memory, Snails, Phenotype_
There is mounting evidence that intestinal microbiota communities and their genes (the gut microbiome) influence how animals behave and interact with their environment, driving individual variation. Individual covariation in behavioural, physiological, and cognitive traits among individuals along a fast-slow continuum is thought to arise because these traits are linked as part of an adaptive pace-of-life strategy. Yet paradoxically, trait intercorrelation is absent or disrupted in some populations but not others. Here, we provide experimental evidence from aquatic pond snails (Lymnaea stagnalis) that environmental stressors and the gut microbiota explain host phenotypic plasticity and disrupted covariation among traits. Antibiotic exposure at varying levels of ecologically relevant concentrations had multiple effects starting with gut microbiota diversity, differential abundance, and inferred function. Memory declined in line with antibiotic concentrations that caused the most profound gut microbiota disruption, and although pace-of-life traits remained rigid, their covariation did not. Moreover, inferred microbial metabolic pathways with biologically relevant host functions explained individual and treatment variation in phenotypes. Together, our results point to the gut microbiome as a proximate mechanism influencing the emergence and maintenance of phenotypic variation within populations and highlights the need to decipher whether the gut microbiome's sensitivity to environmental pollution facilitates adaptive or maladaptive phenotypic plasticity.

[Source](https://doi.org/10.1093/ismejo/wrae078)

10.1093/ismejo/wrae078

---

## Metagenome-assembled genomes from microbiomes fermenting dairy coproducts.
 31.05.2024, _KA Walters, KS Myers, TJ Donohue, DR Noguera_


To advance knowledge of microbial communities capable of fermenting agro-industrial residues into value-added products, we report metagenomes of microbial communities from six anaerobic bioreactors that were fed a mixture of ultra-filtered milk permeate and cottage cheese acid whey. These metagenomes produced 122 metagenome-assembled genomes that represent 34 distinct taxa.

[Source](https://doi.org/10.1128/mra.00173-24)

10.1128/mra.00173-24

---

## Global biogeography of microbes driving ocean ecological status under climate change.
 03.06.2024, _Z Zhang, Q Zhang, B Chen, Y Yu, T Wang, N Xu, X Fan, J Penuelas, Z Fu, Y Deng, YG Zhu, H Qian_


_Climate Change, Oceans and Seas, Microbiota, Seawater, Ecosystem, Carbon Cycle, Biodiversity_
Microbial communities play a crucial role in ocean ecology and global biogeochemical processes. However, understanding the intricate interactions among diversity, taxonomical composition, functional traits, and how these factors respond to climate change remains a significant challenge. Here, we propose seven distinct ecological statuses by systematically considering the diversity, structure, and biogeochemical potential of the ocean microbiome to delineate their biogeography. Anthropogenic climate change is expected to alter the ecological status of the surface ocean by influencing environmental conditions, particularly nutrient and oxygen contents. Our predictive model, which utilizes machine learning, indicates that the ecological status of approximately 32.44% of the surface ocean may undergo changes from the present to the end of this century, assuming no policy interventions. These changes mainly include poleward shifts in the main taxa, increases in photosynthetic carbon fixation and decreases in nutrient metabolism. However, this proportion can decrease significantly with effective control of greenhouse gas emissions. Our study underscores the urgent necessity for implementing policies to mitigate climate change, particularly from an ecological perspective.

[Source](https://doi.org/10.1038/s41467-024-49124-0)

10.1038/s41467-024-49124-0

---

## Engineering natural microbiomes toward enhanced bioremediation by microbiome modeling.
 03.06.2024, _Z Ruan, K Chen, W Cao, L Meng, B Yang, M Xu, Y Xing, P Li, S Freilich, C Chen, Y Gao, J Jiang, X Xu_


_Microbiota, Biodegradation, Environmental, Herbicides, Soil Microbiology, Soil Pollutants, Models, Biological, Bacteria_
Engineering natural microbiomes for biotechnological applications remains challenging, as metabolic interactions within microbiomes are largely unknown, and practical principles and tools for microbiome engineering are still lacking. Here, we present a combinatory top-down and bottom-up framework to engineer natural microbiomes for the construction of function-enhanced synthetic microbiomes. We show that application of herbicide and herbicide-degrader inoculation drives a convergent succession of different natural microbiomes toward functional microbiomes (e.g., enhanced bioremediation of herbicide-contaminated soils). We develop a metabolic modeling pipeline, SuperCC, that can be used to document metabolic interactions within microbiomes and to simulate the performances of different microbiomes. Using SuperCC, we construct bioremediation-enhanced synthetic microbiomes based on 18 keystone species identified from natural microbiomes. Our results highlight the importance of metabolic interactions in shaping microbiome functions and provide practical guidance for engineering natural microbiomes.

[Source](https://doi.org/10.1038/s41467-024-49098-z)

10.1038/s41467-024-49098-z

---

## Interpretable machine learning decodes soil microbiome's response to drought stress.
 01.06.2024, _M Hagen, R Dass, C Westhues, J Blom, SJ Schultheiss, S Patz_


Extreme weather events induced by climate change, particularly droughts, have detrimental consequences for crop yields and food security. Concurrently, these conditions provoke substantial changes in the soil bacterial microbiota and affect plant health. Early recognition of soil affected by drought enables farmers to implement appropriate agricultural management practices. In this context, interpretable machine learning holds immense potential for drought stress classification of soil based on marker taxa.

[Source](https://doi.org/10.1186/s40793-024-00578-1)

10.1186/s40793-024-00578-1

---

## Macroalgal eukaryotic microbiome composition indicates novel phylogenetic diversity and broad host spectrum of oomycete pathogens.
 31.05.2024, _N Timanikova, K Fletcher, JW Han, P van West, S Woodward, GH Kim, FC Küpper, M Wenzel_


_Oomycetes, Phylogeny, Seaweed, Microbiota, RNA, Ribosomal, 18S, Symbiosis, Biodiversity, Eukaryota, Genetic Variation_
Seaweeds are important components of marine ecosystems with emerging potential in aquaculture and as sources of biofuel, food products and pharmacological compounds. However, an increasingly recognised threat to natural and industrial seaweed populations is infection with parasitic single-celled eukaryotes from the relatively understudied oomycete lineage. Here we examine the eukaryomes of diverse brown, red and green marine macroalgae collected from polar (Baffin Island), cold-temperate (Falkland Islands) and tropical (Ascension Island) locations, with a focus on oomycete and closely related diatom taxa. Using 18S rRNA gene amplicon sequencing, we show unexpected genetic and taxonomic diversity of the eukaryomes, a strong broad-brush association between eukaryome composition and geographic location, and some evidence of association between eukaryome structure and macroalgal phylogenetic relationships (phylosymbiosis). However, the oomycete fraction of the eukaryome showed disparate patterns of diversity and structure, highlighting much weaker association with geography and no evidence of phylosymbiosis. We present several novel haplotypes of the most common oomycete Eurychasma dicksonii and report for the first time a cosmopolitan distribution and absence of host specificity of this important pathogen. This indicates rich diversity in macroalgal oomycete pathogens and highlights that these pathogens may be generalist and highly adaptable to diverse environmental conditions.

[Source](https://doi.org/10.1111/1462-2920.16656)

10.1111/1462-2920.16656

---

## Combining compositional data sets introduces error in covariance network reconstruction.
 31.05.2024, _JD Brunner, AJ Robinson, PSG Chain_


Microbial communities are diverse biological systems that include taxa from across multiple kingdoms of life. Notably, interactions between bacteria and fungi play a significant role in determining community structure. However, these statistical associations across kingdoms are more difficult to infer than intra-kingdom associations due to the nature of the data involved using standard network inference techniques. We quantify the challenges of cross-kingdom network inference from both theoretical and practical points of view using synthetic and real-world microbiome data. We detail the theoretical issue presented by combining compositional data sets drawn from the same environment, e.g. 16S and ITS sequencing of a single set of samples, and we survey common network inference techniques for their ability to handle this error. We then test these techniques for the accuracy and usefulness of their intra- and inter-kingdom associations by inferring networks from a set of simulated samples for which a ground-truth set of associations is known. We show that while the two methods mitigate the error of cross-kingdom inference, there is little difference between techniques for key practical applications including identification of strong correlations and identification of possible keystone taxa (i.e. hub nodes in the network). Furthermore, we identify a signature of the error caused by transkingdom network inference and demonstrate that it appears in networks constructed using real-world environmental microbiome data.

[Source](https://doi.org/10.1093/ismeco/ycae057)

10.1093/ismeco/ycae057

---

## How to Obtain and Compare Metagenome-Assembled Genomes.
 31.05.2024, _FB Sanchez, SE Sato Guima, JC Setubal_


_Metagenome, Metagenomics, Software, Computational Biology, Databases, Genetic, Sequence Analysis, DNA, Genome, Bacterial_
Metagenome-assembled genomes, or MAGs, are genomes retrieved from metagenome datasets. In the vast majority of cases, MAGs are genomes from prokaryotic species that have not been isolated or cultivated in the lab. They, therefore, provide us with information on these species that are impossible to obtain otherwise, at least until new cultivation methods are devised. Thanks to improvements and cost reductions of DNA sequencing technologies and growing interest in microbial ecology, the rise in number of MAGs in genome repositories has been exponential. This chapter covers the basics of MAG retrieval and processing and provides a practical step-by-step guide using a real dataset and state-of-the-art tools for MAG analysis and comparison.

[Source](https://doi.org/10.1007/978-1-0716-3838-5_6)

10.1007/978-1-0716-3838-5_6

---

## A strategy for differential abundance analysis of sparse microbiome data with group-wise structured zeros.
 03.06.2024, _F Abegaz, D Abedini, F White, A Guerrieri, A Zancarini, L Dong, JA Westerhuis, F van Eeuwijk, H Bouwmeester, AK Smilde_


_Microbiota, Computational Biology, Bacteria, Plants, Algorithms_
Comparing the abundance of microbial communities between different groups or obtained under different experimental conditions using count sequence data is a challenging task due to various issues such as inflated zero counts, overdispersion, and non-normality. Several methods and procedures based on counts, their transformation and compositionality have been proposed in the literature to detect differentially abundant species in datasets containing hundreds to thousands of microbial species. Despite efforts to address the large numbers of zeros present in microbiome datasets, even after careful data preprocessing, the performance of existing methods is impaired by the presence of inflated zero counts and group-wise structured zeros (i.e. all zero counts in a group). We propose and validate using extensive simulations an approach combining two differential abundance testing methods, namely DESeq2-ZINBWaVE and DESeq2, to address the issues of zero-inflation and group-wise structured zeros, respectively. This combined approach was subsequently successfully applied to two plant microbiome datasets that revealed a number of taxa as interesting candidates for further experimental validation.

[Source](https://doi.org/10.1038/s41598-024-62437-w)

10.1038/s41598-024-62437-w

---

## A review of machine learning methods for cancer characterization from microbiome data.
 03.06.2024, _M Teixeira, F Silva, RM Ferreira, T Pereira, C Figueiredo, HP Oliveira_


Recent studies have shown that the microbiome can impact cancer development, progression, and response to therapies suggesting microbiome-based approaches for cancer characterization. As cancer-related signatures are complex and implicate many taxa, their discovery often requires Machine Learning approaches. This review discusses Machine Learning methods for cancer characterization from microbiome data. It focuses on the implications of choices undertaken during sample collection, feature selection and pre-processing. It also discusses ML model selection, guiding how to choose an ML model, and model validation. Finally, it enumerates current limitations and how these may be surpassed. Proposed methods, often based on Random Forests, show promising results, however insufficient for widespread clinical usage. Studies often report conflicting results mainly due to ML models with poor generalizability. We expect that evaluating models with expanded, hold-out datasets, removing technical artifacts, exploring representations of the microbiome other than taxonomical profiles, leveraging advances in deep learning, and developing ML models better adapted to the characteristics of microbiome data will improve the performance and generalizability of models and enable their usage in the clinic.

[Source](https://doi.org/10.1038/s41698-024-00617-7)

10.1038/s41698-024-00617-7

---

