## Bacteriophage-Bacteria Interactions in the Gut: From Invertebrates to Mammals.
 13.07.2021, _JM Kirsch, RS Brzozowski, D Faith, JL Round, PR Secor, BA Duerkop_


Bacteria and their viruses (bacteriophages or phages) interact antagonistically and beneficially in polymicrobial communities such as the guts of animals. These interactions are multifaceted and are influenced by environmental conditions. In this review, we discuss phage-bacteria interactions as they relate to the complex environment of the gut. Within the mammalian and invertebrate guts, phages and bacteria engage in diverse interactions including genetic coexistence through lysogeny, and phages directly modulate microbiota composition and the immune system with consequences that are becoming recognized as potential drivers of health and disease. With greater depth of understanding of phage-bacteria interactions in the gut and the outcomes, future phage therapies become possible. Expected final online publication date for the Annual Review of Virology, Volume 8 is September 2021. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.

[Source](https://www.annualreviews.org/doi/10.1146/annurev-virology-091919-101238)

10.1146/annurev-virology-091919-101238

---

## Helicobacter pylori infection has a detrimental impact on the efficacy of cancer immunotherapies

[Source](https://gut.bmj.com/content/early/2021/07/11/gutjnl-2020-323392)

---

## Cervicovaginal microbiota and metabolome predict preterm birth risk in an ethnically diverse cohort

[Source](https://insight.jci.org/articles/view/149257)

---

## [REVIEW][Early-life gut microbiota and its connection to metabolic health in children: Perspective on ecological drivers and need for quantitative approach

[Source](https://www.sciencedirect.com/science/article/pii/S2352396421002681)

---

## Associations of air pollution and greenness with the nasal microbiota of healthy infants: A longitudinal study

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0013935121009270?via%3Dihub)

---

## Multi-omics profiling predicts allograft function after lung transplantation

[Source](https://erj.ersjournals.com/content/early/2021/06/10/13993003.03292-2020)

---

## Metagenomic Analysis Reveals A Possible Association Between Respiratory Infection and Periodontitis

[Source](https://www.sciencedirect.com/science/article/pii/S1672022921001443)

---

## Novel mechanisms of microbial crosstalk with skin innate immunity

[Source](https://onlinelibrary.wiley.com/doi/10.1111/exd.14429)

---

## Shifts in the skin bacterial and fungal communities of healthy children transitioning through puberty

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0022202X21014214)

---

## Gut-microbiota-targeted diets modulate human immune status

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0092867421007546)

---

## Silent hypoxia in COVID-19: a gut microbiota connection

[Source](https://www.sciencedirect.com/science/article/pii/S246886732100081X)

---

## The Athlete and Gut Microbiome: Short-chain Fatty Acids as Potential Ergogenic Aids for Exercise and Training

[Source](https://www.thieme-connect.de/products/ejournals/abstract/10.1055/a-1524-2095)

---

## Identification and engraftment of new bacterial strains by shotgun metagenomic sequence analysis in patients with recurrent Clostridioides difficile infection before and after fecal microbiota transplantation and in healthy human subjects

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0251590)

---

## Kefir Ameliorates Specific Microbiota-Gut-Brain Axis Impairments in a Mouse Model Relevant to Autism Spectrum Disorder

[Source](https://www.sciencedirect.com/science/article/pii/S0889159121002683)

---

## Comparative analysis of oral-gut microbiota between captive and wild long-tailed macaque in Thailand

[Source](https://www.nature.com/articles/s41598-021-93779-4)

---

## Impact of the dietary inclusion of dried food residues on the apparent nutrient digestibility and the intestinal microbiota of dogs

[Source](https://www.tandfonline.com/doi/abs/10.1080/1745039X.2021.1949229?journalCode=gaan20)

---

## Triphenyltin exposure causes changes in health-associated gut microbiome and metabolites in marine medaka

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0269749121013336)

---

## The microbial metabolite p-Cresol induces autistic-like behaviors in mice by remodeling the gut microbiota

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01103-z)

---

## Seven years of Progress in Determining Fungal Diversity and Characterization of Fungi Isolated from the Japanese Experiment Module KIBO, International Space Station

[Source](https://onlinelibrary.wiley.com/doi/10.1111/1348-0421.12931)

---

## Virome studies of food production systems: time for ‘farm to fork’ analyses

[Source](https://www.sciencedirect.com/science/article/pii/S0958166921000987)

---

## D-MANOVA: fast distance-based multivariate analysis of variance for large-scale microbiome association studies

[Source](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab498/6320063)

---

## Umibato: estimation of time-varying microbial interaction using continuous-time regression hidden Markov model.
 13.07.2021, _S Hosoda, T Fukunaga, M Hamada_


Accumulating evidence has highlighted the importance of microbial interaction networks. Methods have been developed for estimating microbial interaction networks, of which the generalized Lotka-Volterra equation (gLVE)-based method can estimate a directed interaction network. The previous gLVE-based method for estimating microbial interaction networks did not consider time-varying interactions.

[Source](https://academic.oup.com/bioinformatics/article/37/Supplement_1/i16/6319692)

10.1093/bioinformatics/btab287

---

## Stool Studies Don’t Pass the Sniff Test: A Systematic Review of Human Gut Microbiome Research Suggests Widespread Misuse of Machine Learning

[Source](https://arxiv.org/abs/2107.03611)

---

## Characterizing Microbiomes via Sequencing of Marker Loci: Techniques To Improve Throughput, Account for Cross-Contamination, and Reduce Cost.
 13.07.2021, _JG Harrison, GD Randolph, CA Buerkle_


New approaches to characterizing microbiomes via high-throughput sequencing provide impressive gains in efficiency and cost reduction compared to approaches that were standard just a few years ago. However, the speed of method development has been such that staying abreast of the latest technological advances is challenging. Moreover, shifting laboratory protocols to include new methods can be expensive and time consuming. To facilitate adoption of new techniques, we provide a guide and review of recent advances that are relevant for single-locus sequence-based study of microbiomes-from extraction to library preparation-including a primer regarding the use of liquid-handling automation in small-scale academic settings. Additionally, we describe several amendments to published techniques to improve throughput, track contamination, and reduce cost. Notably, we suggest adding synthetic DNA molecules to each sample during nucleic acid extraction, thus providing a method of documenting incidences of cross-contamination. We also describe a dual-indexing scheme for Illumina sequencers that allows multiplexing of many thousands of samples with minimal PhiX input. Collectively, the techniques that we describe demonstrate that laboratory technology need not impose strict limitations on the scale of molecular microbial ecology studies. IMPORTANCE New methods to characterize microbiomes reduce technology-imposed limitations to study design, but many new approaches have not been widely adopted. Here, we present techniques to increase throughput and reduce contamination alongside a thorough review of current best practices.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00294-21)

10.1128/mSystems.00294-21

---

## Two-Target Quantitative PCR To Predict Library Composition for Shallow Shotgun Sequencing.
 13.07.2021, _MY Cho, M Oliva, A Spreafico, B Chen, X Wei, Y Choi, R Kaul, LL Siu, B Coburn, PHH Schneeberger_


When determining human microbiota composition, shotgun sequencing is a powerful tool that can generate high-resolution taxonomic and functional information at once. However, the technique is limited by missing information about host-to-microbe ratios observed in different body compartments. This limitation makes it difficult to plan shotgun sequencing assays, especially in the context of high sample multiplexing and limited sequencing output and is of particular importance for studies employing the recently described shallow shotgun sequencing technique. In this study, we evaluated the use of a quantitative PCR (qPCR)-based assay to predict host-to-microbe ratio prior to sequencing. Combining a two-target assay involving the bacterial 16S rRNA gene and the human beta-actin gene, we derived a model to predict human-to-microbe ratios from two sample types, including stool samples and oropharyngeal swabs. We then validated it on two independently collected sample types, including rectal swabs and vaginal secretion samples. This assay enabled accurate prediction in the validation set in a range of sample compositions between 4% and 98% nonhuman reads and observed proportions varied between -18.8% and +19.2% from the expected values. We hope that this easy-to-use assay will help researchers to plan their shotgun sequencing experiments in a more efficient way. IMPORTANCE When determining human microbiota composition, shotgun sequencing is a powerful tool that can generate large amounts of data. However, in sample compositions with low or variable microbial density, shallowing sequencing can negatively affect microbial community metrics. Here, we show that variable sequencing depth decreases measured alpha diversity at differing rates based on community composition. We then derived a model that can determine sample composition prior to sequencing using quantitative PCR (qPCR) data and validated the model using a separate sample set. We have included a tool that uses this model to be available for researchers to use when gauging shallow sequencing viability of samples.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00552-21)

10.1128/mSystems.00552-21

---

## Statistical approaches for differential expression analysis in metatranscriptomics.
 13.07.2021, _Y Zhang, KN Thompson, C Huttenhower, EA Franzosa_


Metatranscriptomics (MTX) has become an increasingly practical way to profile the functional activity of microbial communities in situ. However, MTX remains underutilized due to experimental and computational limitations. The latter are complicated by non-independent changes in both RNA transcript levels and their underlying genomic DNA copies (as microbes simultaneously change their overall abundance in the population and regulate individual transcripts), genetic plasticity (as whole loci are frequently gained and lost in microbial lineages) and measurement compositionality and zero-inflation. Here, we present a systematic evaluation of and recommendations for differential expression (DE) analysis in MTX.

[Source](https://academic.oup.com/bioinformatics/article/37/Supplement_1/i34/6319701)

10.1093/bioinformatics/btab327

---

## Benchmark of 16S rRNA gene amplicon sequencing using Japanese gut microbiome data from the V1-V2 and V3-V4 primer sets.
 13.07.2021, _S Kameoka, D Motooka, S Watanabe, R Kubo, N Jung, Y Midorikawa, NO Shinozaki, Y Sawai, AK Takeda, S Nakamura_


_Benchmarking, Gastrointestinal Microbiome, Genes, rRNA, High-Throughput Nucleotide Sequencing, Humans, Japan, RNA, Ribosomal, 16S_
16S rRNA gene amplicon sequencing (16S analysis) is widely used to analyze microbiota with next-generation sequencing technologies. Here, we compared fecal 16S analysis data from 192 Japanese volunteers using the modified V1-V2 (V12) and the standard V3-V4 primer (V34) sets to optimize the gut microbiota analysis protocol.

[Source](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-021-07746-4)

10.1186/s12864-021-07746-4

---

## In vitro models of gut digestion across childhood: current developments, challenges and future trends.
 12.07.2021, _E Fournier, C Roussel, A Dominicis, D Ley, MA Peyron, V Collado, M Mercier-Bonin, C Lacroix, M Alric, T Van de Wiele, C Chassard, L Etienne-Mesmin, S Blanquet-Diot_


The human digestion is a multi-step and multi-compartment process essential for human health, at the heart of many issues raised by academics, the medical world and industrials from the food, nutrition and pharma fields. In the first years of life, major dietary changes occur and are concomitant with an evolution of the whole child digestive tract anatomy and physiology, including colonization of gut microbiota. All these phenomena are influenced by child exposure to environmental compounds, such as drugs (especially antibiotics) and food pollutants, but also childhood infections. Due to obvious ethical, regulatory and technical limitations, in vivo approaches in animal and human are more and more restricted to favor complementary in vitro approaches. This review summarizes current knowledge on the evolution of child gut physiology from birth to 3 years old regarding physicochemical, mechanical and microbial parameters. Then, all the available in vitro models of the child digestive tract are described, ranging from the simplest static mono-compartmental systems to the most sophisticated dynamic and multi-compartmental models, and mimicking from the oral phase to the colon compartment. Lastly, we detail the main applications of child gut models in nutritional, pharmaceutical and microbiological studies and discuss the limitations and challenges facing this field of research.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0734975021001026)

10.1016/j.biotechadv.2021.107796

---

