##  enormous <em>E. coli</em> in Edinburgh

[Source](https://www.thisiscolossal.com/2022/08/luke-jerram-ecoli/)

---

## Effects of sea salt intake on metabolites, steroid hormones, and gut microbiota in rats.
 16.08.2022, _S Chanmuang, BM Kim, SY Gu, YJ Son, HG Le, YD Nam, EJ Song, KS Ham, HJ Kim_


_Animals, Female, Gastrointestinal Microbiome, Hormones, Kidney, Male, Rats, Sodium Chloride, Sodium Chloride, Dietary, Steroids_
High salt intake is positively linked to many health problems, but the effect of mineral-rich sea salt (SS) has rarely been studied. To better understand the physiological effects of SS intake, the changes in general characteristics, metabolites, steroid hormones, and gut microbiota of SS-fed rats were investigated. Male rats were fed either a normal diet (ND, control) or ND containing 1% SS or 4% SS for 5 weeks. SS intake decreased fat, spleen, liver, and body weight, and increased blood urea nitrogen (BUN), water intake, and gut salt content. Accumulated gut salt content led to a decrease in beneficial bacteria, such as Lachnospiraceae and Lactobacillus, but an increase in potentially harmful bacteria, resulting in a change in lipid metabolites associated with gut health. Interestingly, most renal lysophosphatidylcholines (LPCs) associated with many renal functions were dramatically decreased and female hormones, such as estrogens, were significantly more altered than the male hormones by high SS intake. Although further investigation is needed, these data suggest that high SS intake could be positively linked to kidney dysfunction and gut health problems, and salt-related physiological changes may be sex-specific. Additionally, these data will be useful to better under-stand the physiological effects of SS intake.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0269014)

10.1371/journal.pone.0269014

---

## The changing influence of host genetics on the leaf fungal microbiome throughout plant development.
 17.08.2022, _TL Karasov, DS Lundberg_


_Genotype, Mycobiome, Panicum, Plant Development, Plant Leaves_
Host genetics and the environment influence which fungal microbes colonize a plant. A new study in PLOS Biology finds that the relative influence of these factors changes throughout the development of the biofuel crop switchgrass growing in field settings.

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001748)

10.1371/journal.pbio.3001748

---

## Impacts of temperature and hydraulic regime on discolouration and biofilm fouling in drinking water distribution systems

[Source](https://journals.plos.org/water/article?id=10.1371/journal.pwat.0000033#abstract0)

---

## Nine quick tips for pathway enrichment analysis.
 15.08.2022, _D Chicco, G Agapito_


_Computational Biology, Databases, Factual, Humans, Software_
Pathway enrichment analysis (PEA) is a computational biology method that identifies biological functions that are overrepresented in a group of genes more than would be expected by chance and ranks these functions by relevance. The relative abundance of genes pertinent to specific pathways is measured through statistical methods, and associated functional pathways are retrieved from online bioinformatics databases. In the last decade, along with the spread of the internet, higher availability of computational resources made PEA software tools easy to access and to use for bioinformatics practitioners worldwide. Although it became easier to use these tools, it also became easier to make mistakes that could generate inflated or misleading results, especially for beginners and inexperienced computational biologists. With this article, we propose nine quick tips to avoid common mistakes and to out a complete, sound, thorough PEA, which can produce relevant and robust results. We describe our nine guidelines in a simple way, so that they can be understood and used by anyone, including students and beginners. Some tips explain what to do before starting a PEA, others are suggestions of how to correctly generate meaningful results, and some final guidelines indicate some useful steps to properly interpret PEA results. Our nine tips can help users perform better pathway enrichment analyses and eventually contribute to a better understanding of current biology.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010348)

10.1371/journal.pcbi.1010348

---

## Ten quick tips for biomarker discovery and validation analyses using machine learning.
 15.08.2022, _R Diaz-Uriarte, E Gómez de Lope, R Giugno, H Fröhlich, PV Nazarov, IA Nepomuceno-Chamorro, A Rauschenberger, E Glaab_


_Biomarkers, Biomedical Research, Computational Biology, Machine Learning_

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010357)

10.1371/journal.pcbi.1010357

---

