## Portion size can be used strategically to increase intake of vegetables and fruits in young children over multiple days: a cluster-randomized crossover trial.
 22.09.2021, _LS Roe, CE Sanchez, AD Smethers, KL Keller, BJ Rolls_


Although dietary guidelines recommend that vegetables and fruits make up half the diet, it is unclear whether serving vegetables and fruits in larger portions will have sustained effects on children's intake over multiple days.

[Source](https://www.biocodexmicrobiotafoundation.com/national-call-projects/usa)

10.1093/ajcn/nqab321

---

## https://www.biocodexmicrobiotafoundation.com/national-call-projects/usa

[Source](https://www.biocodexmicrobiotafoundation.com/national-call-projects/usa)

---

## Food for thought: does host diet affect skin microbes?
 20.09.2021, _AM Schneider, AM Nelson_


_Diet, Gastrointestinal Microbiome, Humans_

[Source](https://onlinelibrary.wiley.com/doi/10.1111/bjd.20573)

10.1111/bjd.20573

---

## Host traits, lifestyle and environment are associated with human skin bacteria.
 20.09.2021, _L Moitinho-Silva, N Boraczynski, H Emmert, H Baurecht, S Szymczak, H Schulz, D Haller, J Linseisen, C Gieger, A Peters, L Tittmann, W Lieb, C Bang, A Franke, E Rodriguez, S Weidinger_


_Bacteria, Cross-Sectional Studies, Humans, Life Style, Microbiota, RNA, Ribosomal, 16S_
The human skin offers diverse ecosystems for microbial symbionts. However, the factors shaping skin-microbiome interactions are still insufficiently characterized. This contrasts with the broader knowledge about factors influencing gut microbiota.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/bjd.20072)

10.1111/bjd.20072

---

## Deciphering the low abundance microbiota of presumed aseptic hip and knee implants.
 17.09.2021, _C Carr, H Wilcox, JP Burton, S Menon, KF Al, D O'Gorman, BA Lanting, EM Vasarhelyi, M Neufeld, MG Teeter_


16S rRNA gene sequencing of DNA extracted from clinically uninfected hip and knee implant samples has revealed polymicrobial populations. However, previous studies assessed 16S rRNA gene sequencing as a technique for the diagnosis of periprosthetic joint infections, leaving the microbiota of presumed aseptic hip and knee implants largely unstudied. These communities of microorganisms might play important roles in aspects of host health, such as aseptic loosening. Therefore, this study sought to characterize the bacterial composition of presumed aseptic joint implant microbiota using next generation 16S rRNA gene sequencing, and it evaluated this method for future investigations. 248 samples were collected from implants of 41 patients undergoing total hip or knee arthroplasty revision for presumed aseptic failure. DNA was extracted using two methodologies-one optimized for high throughput and the other for human samples-and amplicons of the V4 region of the 16S rRNA gene were sequenced. Sequencing data were analyzed and compared with ancillary specific PCR and microbiological culture. Computational tools (SourceTracker and decontam) were used to detect and compensate for environmental and processing contaminants. Microbial diversity of patient samples was higher than that of open-air controls and differentially abundant taxa were detected between these conditions, possibly reflecting a true microbiota that is present in clinically uninfected joint implants. However, positive control-associated artifacts and DNA extraction methodology significantly affected sequencing results. As well, sequencing failed to identify Cutibacterium acnes in most culture- and PCR-positive samples. These challenges limited characterization of bacteria in presumed aseptic implants, but genera were identified for further investigation. In all, we provide further support for the hypothesis that there is likely a microbiota present in clinically uninfected joint implants, and we show that methods other than 16S rRNA gene sequencing may be ideal for its characterization. This work has illuminated the importance of further study of microbiota of clinically uninfected joint implants with novel molecular and computational tools to further eliminate contaminants and artifacts that arise in low bacterial abundance samples.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0257471)

10.1371/journal.pone.0257471

---

## Next Generation Microbiome Research: Identification of Keystone Species in the Metabolic Regulation of Host-Gut Microbiota Interplay.
 21.09.2021, _H Tudela, SP Claus, M Saleh_


The community of the diverse microorganisms residing in the gastrointestinal tract, known as the gut microbiota, is exceedingly being studied for its impact on health and disease. This community plays a major role in nutrient metabolism, maintenance of the intestinal epithelial barrier but also in local and systemic immunomodulation. A dysbiosis of the gut microbiota, characterized by an unbalanced microbial ecology, often leads to a loss of essential functions that may be associated with proinflammatory conditions. Specifically, some key microbes that are depleted in dysbiotic ecosystems, called keystone species, carry unique functions that are essential for the balance of the microbiota. In this review, we discuss current understanding of reported keystone species and their proposed functions in health. We also elaborate on current and future bioinformatics tools needed to identify missing functions in the gut carried by keystone species. We propose that the identification of such keystone species functions is a major step for the understanding of microbiome dynamics in disease and toward the development of microbiome-based therapeutics.

[Source](https://www.frontiersin.org/articles/10.3389/fcell.2021.719072/full)

10.3389/fcell.2021.719072

---

## Commensal microbe-derived acetate suppresses NAFLD/NASH development via hepatic FFAR2 signalling in mice.
 21.09.2021, _R Aoki, M Onuki, K Hattori, M Ito, T Yamada, K Kamikado, YG Kim, N Nakamoto, I Kimura, JM Clarke, T Kanai, K Hase_


Non-alcoholic liver disease (NAFLD) is the hepatic manifestation of metabolic syndrome, and it can progress to non-alcoholic steatohepatitis (NASH). Alterations in the gut microbiome have been implicated in the development of NAFLD/NASH, although the underlying mechanisms remain unclear.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01125-7)

10.1186/s40168-021-01125-7

---

## 16 S rRNA gene diversity and gut microbial composition of the Indian white shrimp (Penaeus indicus).
 18.09.2021, _PK Patil, TN Vinay, SD Ghate, V Baskaran, S Avunje_


The endemic Indian white shrimp (Penaeus indicus) is an economically important crustacean species, distributed in the Indo-West Pacific region. Knowledge of its gut microbial composition helps in dietary interventions to ensure improved health and production. Here we analyzed V3-V4 hypervariable regions of the 16 S rRNA gene to examine intestinal microbiota in wild and domesticated farmed P. indicus. The study revealed that Proteobacteria, Fusobacteria, Tenericutes, and Bacteroidetes, were the dominant phyla in both the groups although there were differences in relative abundance. The dominant genera in case of the wild group were Photobacterium (29.5 %) followed by Propionigenium (13.9 %), Hypnocyclicus (13.7 %) and Vibrio (11.1 %); while Vibrio (46.5 %), Catenococcus (14 %), Propionigenium (10.3 %) and Photobacterium (8.7 %) were dominant in the farmed group. The results of the study suggest the role of environment on the relative abundance of gut bacteria. This is the first report characterizing gut microbial diversity in P. indicus, which can be used to understand the role of gut microbiota in health, nutrition, reproduction, and growth.

[Source](https://link.springer.com/article/10.1007%2Fs10482-021-01658-9)

10.1007/s10482-021-01658-9

---

## Disease-induced changes in plant microbiome assembly and functional adaptation.
 19.09.2021, _M Gao, C Xiong, C Gao, CKM Tsui, MM Wang, X Zhou, AM Zhang, L Cai_


The plant microbiome is an integral part of the host and increasingly recognized as playing fundamental roles in plant growth and health. Increasing evidence indicates that plant rhizosphere recruits beneficial microbes to the plant to suppress soil-borne pathogens. However, the ecological processes that govern plant microbiome assembly and functions in the below- and aboveground compartments under pathogen invasion are not fully understood. Here, we studied the bacterial and fungal communities associated with 12 compartments (e.g., soils, roots, stems, and fruits) of chili pepper (Capsicum annuum L.) using amplicons (16S and ITS) and metagenomics approaches at the main pepper production sites in China and investigated how Fusarium wilt disease (FWD) affects the assembly, co-occurrence patterns, and ecological functions of plant-associated microbiomes.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01138-2)

10.1186/s40168-021-01138-2

---

## Diversity and distribution of viruses inhabiting the deepest ocean on Earth.
 22.09.2021, _H Jian, Y Yi, J Wang, Y Hao, M Zhang, S Wang, C Meng, Y Zhang, H Jing, Y Wang, X Xiao_


_Archaea, Ecosystem, Oceans and Seas, Seawater, Viruses_
As the most abundant biological entities on the planet, viruses significantly influence the overall functioning of marine ecosystems. The abundance, distribution, and biodiversity of viral communities in the upper ocean have been relatively well studied, but our understanding of viruses in the hadal biosphere remains poor. Here, we established the oceanic trench viral genome dataset (OTVGD) by analysing 19 microbial metagenomes derived from seawater and sediment samples of the Mariana, Yap, and Kermadec Trenches. The trench viral communities harbored remarkably high novelty, and they were predicted to infect ecologically important microbial clades, including Thaumarchaeota and Oleibacter. Significant inter-trench and intra-trench exchange of viral communities was proposed. Moreover, viral communities in different habitats (seawater/sediment and depth-stratified ocean zones) exhibited distinct niche-dependent distribution patterns and genomic properties. Notably, microbes and viruses in the hadopelagic seawater seemed to preferably adopt lysogenic lifestyles compared to those in the upper ocean. Furthermore, niche-specific auxiliary metabolic genes were identified in the hadal viral genomes, and a novel viral D-amino acid oxidase was functionally and phylogenetically characterized, suggesting the contribution of these genes in the utilization of refractory organic matter. Together, these findings highlight the genomic novelty, dynamic movement, and environment-driven diversification of viral communities in oceanic trenches, and suggest that viruses may influence the hadal ecosystem by reprogramming the metabolism of their hosts and modulating the community of keystone microbes.

[Source](https://www.nature.com/articles/s41396-021-00994-y)

10.1038/s41396-021-00994-y

---

## Deep-sea shipwrecks represent island-like ecosystems for marine microbiomes.
 22.09.2021, _LJ Hamdan, JJ Hampel, RD Moseley, RL Mugge, A Ray, JL Salerno, M Damour_


_Biodiversity, Ecosystem, Gulf of Mexico, Microbiota_
Biogeography of macro- and micro-organisms in the deep sea is, in part, shaped by naturally occurring heterogeneous habitat features of geological and biological origin such as seeps, vents, seamounts, whale and wood-falls. Artificial features including shipwrecks and energy infrastructure shape the biogeographic patterns of macro-organisms; how they influence microorganisms is unclear. Shipwrecks may function as islands of biodiversity for microbiomes, creating a patchwork of habitats with influence radiating out into the seabed. Here we show microbiome richness and diversity increase as a function of proximity to the historic deep-sea shipwreck Anona in the Gulf of Mexico. Diversity and richness extinction plots provide evidence of an island effect on microbiomes. A halo of core taxa on the seabed was observed up to 200 m away from the wreck indicative of the transition zone from shipwreck habitat to the surrounding environment. Transition zones around natural habitat features are often small in area compared to what was observed at Anona indicating shipwrecks may exert a large sphere of influence on seabed microbiomes. Historic shipwrecks are abundant, isolated habitats with global distribution, providing a means to explore contemporary processes shaping biogeography on the seafloor. This work is a case study for how built environments impact microbial biodiversity and provides new information on how arrival of material to the seafloor shapes benthic microbiomes.

[Source](https://www.nature.com/articles/s41396-021-00978-y)

10.1038/s41396-021-00978-y

---

## The biogeographic differentiation of algal microbiomes in the upper ocean from pole to pole.
 17.09.2021, _K Martin, K Schmidt, A Toseland, CA Boulton, K Barry, B Beszteri, CPD Brussaard, A Clum, CG Daum, E Eloe-Fadrosh, A Fong, B Foster, B Foster, M Ginzburg, M Huntemann, NN Ivanova, NC Kyrpides, E Lindquist, S Mukherjee, K Palaniappan, TBK Reddy, MR Rizkallah, S Roux, K Timmermans, SG Tringe, WH van de Poll, N Varghese, KU Valentin, TM Lenton, IV Grigoriev, RM Leggett, V Moulton, T Mock_


Eukaryotic phytoplankton are responsible for at least 20% of annual global carbon fixation. Their diversity and activity are shaped by interactions with prokaryotes as part of complex microbiomes. Although differences in their local species diversity have been estimated, we still have a limited understanding of environmental conditions responsible for compositional differences between local species communities on a large scale from pole to pole. Here, we show, based on pole-to-pole phytoplankton metatranscriptomes and microbial rDNA sequencing, that environmental differences between polar and non-polar upper oceans most strongly impact the large-scale spatial pattern of biodiversity and gene activity in algal microbiomes. The geographic differentiation of co-occurring microbes in algal microbiomes can be well explained by the latitudinal temperature gradient and associated break points in their beta diversity, with an average breakpoint at 14 °C ± 4.3, separating cold and warm upper oceans. As global warming impacts upper ocean temperatures, we project that break points of beta diversity move markedly pole-wards. Hence, abrupt regime shifts in algal microbiomes could be caused by anthropogenic climate change.

[Source](https://www.nature.com/articles/s41467-021-25646-9)

10.1038/s41467-021-25646-9

---

## A 2021 taxonomy update for the family Smacoviridae.
 21.09.2021, _M Krupovic, A Varsani_


The family Smacoviridae (order Cremevirales, class Arfiviricetes, phylum Cressdnaviricota) is comprised of viruses with small circular single-stranded DNA genomes of ~2.3-3 kb in length that have primarily been identified in fecal sample of various animals. Smacovirus genomes carry two genes in ambisense orientation encoding a capsid protein and a rolling-circle replication initiation protein, respectively. We have revised the taxonomy of the family by assigning 138 new genomic sequences deposited in GenBank to already established taxa as well as 41 new species and six new genera. Furthermore, we have adopted binomial species nomenclature, conforming to the "Genus + freeform epithet" format for all 84 species from 12 genera. The updated Smacoviridae taxonomy presented in this article has been ratified by the International Committee on Taxonomy of Viruses (ICTV).

[Source](https://link.springer.com/article/10.1007%2Fs00705-021-05224-6)

10.1007/s00705-021-05224-6

---

## MIMOSA2: A metabolic network-based tool for inferring mechanism-supported relationships in microbiome-metabolome data

[Source](https://www.biorxiv.org/content/10.1101/2021.09.14.459910v1.abstract)

---

## Identifying indicator species in ecological habitats using Deep Optimal Feature Learning.
 14.09.2021, _Y Tsai, SA Baldwin, B Gopaluni_


Much of the current research on supervised modelling is focused on maximizing outcome prediction accuracy. However, in engineering disciplines, an arguably more important goal is that of feature extraction, the identification of relevant features associated with the various outcomes. For instance, in microbial communities, the identification of keystone species can often lead to improved prediction of future behavioral shifts. This paper proposes a novel feature extractor based on Deep Learning, which is largely agnostic to underlying assumptions regarding the training data. Starting from a collection of microbial species abundance counts, the Deep Learning model first trains itself to classify the selected distinct habitats. It then identifies indicator species associated with the habitats. The results are then compared and contrasted with those obtained by traditional statistical techniques. The indicator species are similar when compared at top taxonomic levels such as Domain and Phylum, despite visible differences in lower levels such as Class and Order. More importantly, when our estimated indicators are used to predict final habitat labels using simpler models (such as Support Vector Machines and traditional Artificial Neural Networks), the prediction accuracy is improved. Overall, this study serves as a preliminary step that bridges modern, black-box Machine Learning models with traditional, domain expertise-rich techniques.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0256782)

10.1371/journal.pone.0256782

---

