## Genome-scale metabolic modeling of <em>Aspergillus fumigatus</em> strains reveals growth dependencies on the lung microbiome

[Source](https://doi.org/10.1038/s41467-023-39982-5)

---

## Depletion of butyrate-producing microbes of the Firmicutes predicts nonresponse to FMT therapy in patients with recurrent <em>Clostridium difficile</em> infection

[Source](https://doi.org/10.1080/19490976.2023.2236362)

---

## Chagas disease is related to structural changes of the gut microbiota in adults with chronic infection (TRIPOBIOME Study)

[Source](https://doi.org/10.1371/journal.pntd.0011490)

---

## Study of the gut microbiome in Egyptian patients with Parkinsonâ€™s Disease

[Source](https://doi.org/10.1186/s12866-023-02933-7)

---

## Intestinal microbiota controls graft-versus-host disease independent of donor-host genetic disparity

[Source](https://www.cell.com/immunity/fulltext/S1074-7613(23)00281-9)

---

## A systematic review exploring the association between the human gut microbiota and brain connectivity in health and disease

[Source](https://doi.org/10.1038/s41380-023-02146-4)

---

## A macroecological perspective on genetic diversity in the human gut microbiome

[Source](https://doi.org/10.1371/journal.pone.0288926)

---

## Gut microbiota metabolites mediate the interplay between childhood maltreatment and psychopathology in patients with eating disorders

[Source](https://doi.org/10.1038/s41598-023-38665-x)

---

## In-depth metataxonomic investigation reveals low richness, high intervariability, and diverse phylotype candidates of archaea&nbsp;in the human urogenital tract

[Source](https://doi.org/10.1038/s41598-023-38710-9)

---

## Host microbiome-pathogen interactions in pediatric infections

[Source](https://doi.org/10.1097/QCO.0000000000000949)

---

## Intratumoural microbiome can predict the prognosis of hepatocellular carcinoma after surgery

[Source](https://doi.org/10.1002/ctm2.1331)

---

## A shotgun metagenomic analysis of the fecal microbiome in humans infected with <em>Giardia duodenalis</em>

[Source](https://doi.org/10.1186/s13071-023-05821-1)

---

## Microbiome-derived cobalamin and succinyl-CoA as biomarkers for improved screening of anal cancer

[Source](https://doi.org/10.1038/s41591-023-02407-3)

---

## Bacterial clade-specific analysis identifies distinct epithelial responses in inflammatory bowel disease

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00252-5)

---

## Multimodal immune cell phenotyping in GI biopsies reveals microbiome-related T cell modulations in human GvHD

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00253-7)

---

## Prebiotics and the risk of upper digestive tract and stomach cancers: the PrebiotiCa study

[Source](https://doi.org/10.1016/j.jand.2023.07.008)

---

## Disturbance of eucalypt forests alters the composition, function, and assembly of soil microbial communities

[Source](https://doi.org/10.1093/femsec/fiad085)

---

## Reciprocal influence of soil, phyllosphere, and aphid microbiomes

[Source](https://doi.org/10.1186/s40793-023-00515-8)

---

## Co-evolution within the plant holobiont drives host performance

[Source](https://doi.org/10.15252/embr.202357455)

---

## May microbial ecological baseline exist in continental groundwater?

[Source](https://doi.org/10.1186/s40168-023-01572-4)

---

## Evolutionary patterns of archaea predominant in acidic environment

[Source](https://doi.org/10.1186/s40793-023-00518-5)

---

## Machine learning-based approaches for cancer prediction using microbiome data.
 24.07.2023, _P Freitas, F Silva, JV Sousa, RM Ferreira, C Figueiredo, T Pereira, HP Oliveira_


_Humans, Colonic Neoplasms, Rectal Neoplasms, Stomach Neoplasms, Machine Learning, Microbiota_
Emerging evidence of the relationship between the microbiome composition and the development of numerous diseases, including cancer, has led to an increasing interest in the study of the human microbiome. Technological breakthroughs regarding DNA sequencing methods propelled microbiome studies with a large number of samples, which called for the necessity of more sophisticated data-analytical tools to analyze this complex relationship. The aim of this work was to develop a machine learning-based approach to distinguish the type of cancer based on the analysis of the tissue-specific microbial information, assessing the human microbiome as valuable predictive information for cancer identification. For this purpose, Random Forest algorithms were trained for the classification of five types of cancer-head and neck, esophageal, stomach, colon, and rectum cancers-with samples provided by The Cancer Microbiome Atlas database. One versus all and multi-class classification studies were conducted to evaluate the discriminative capability of the microbial data across increasing levels of cancer site specificity, with results showing a progressive rise in difficulty for accurate sample classification. Random Forest models achieved promising performances when predicting head and neck, stomach, and colon cancer cases, with the latter returning accuracy scores above 90% across the different studies conducted. However, there was also an increased difficulty when discriminating esophageal and rectum cancers, failing to differentiate with adequate results rectum from colon cancer cases, and esophageal from head and neck and stomach cancers. These results point to the fact that anatomically adjacent cancers can be more complex to identify due to microbial similarities. Despite the limitations, microbiome data analysis using machine learning may advance novel strategies to improve cancer detection and prevention, and decrease disease burden.

[Source](https://doi.org/10.1038/s41598-023-38670-0)

10.1038/s41598-023-38670-0

---

## Bioinformatic mining for RiPP biosynthetic gene clusters in Bacteroidales reveals possible new subfamily architectures and novel natural products

[Source](https://doi.org/10.3389/fmicb.2023.1219272)

---

## A bioinformatic analysis of zinc transporters in intestinal Lactobacillaceae.
 18.07.2023, _U Huynh, HN Nguyen, BK Trinh, J Elhaj, ML Zastrow_


As the second most abundant transition element and a crucial cofactor for many proteins, zinc is essential for the survival of all living organisms. To maintain required zinc levels and prevent toxic overload, cells and organisms have a collection of metal transport proteins for uptake and efflux of zinc. In bacteria, metal transport proteins are well defined for model organisms and many pathogens, but fewer studies have explored metal transport proteins, including those for zinc, in commensal bacteria from the gut microbiota. The healthy human gut microbiota is comprised of hundreds of species and among these, bacteria from the Lactobacillaceae family are well documented to have various beneficial effects on health. Furthermore, changes in dietary metal intake, such as for zinc and iron, are frequently correlated with changes in abundance of Lactobacillaceae. Few studies have explored zinc requirements and zinc homeostasis mechanisms in Lactobacillaceae, however. Here we applied a bioinformatics approach to identify and compare predicted zinc uptake and efflux proteins in several Lactobacillaceae genera of intestinal relevance. Few Lactobacillaceae had zinc transporters currently annotated in proteomes retrieved from the UniProt database, but protein sequence-based (BLASTP) homology searches revealed that high affinity ABC transporter genes are likely common, albeit with genus-specific domain features. P-type ATPase transporters are probably also common and some Lactobacillaceae genera code for predicted zinc efflux cation diffusion facilitators. This analysis confirms that Lactobacillaceae harbor genes for various zinc transporter homologs, and provides a foundation for systematic experimental studies to elucidate zinc homeostasis mechanisms in these bacteria.

[Source](https://doi.org/10.1093/mtomcs/mfad044)

10.1093/mtomcs/mfad044

---

## Uncovering the Carboxylated Metabolome in Gut Microbiota-Host Co-metabolism: A Chemical Derivatization-Molecular Networking Approach.
 20.07.2023, _YZ Wang, YY Chen, XZ Wu, PR Bai, N An, XL Liu, QF Zhu, YQ Feng_


Gut microbiota-host co-metabolites serve as essential mediators of communication between the host and gut microbiota. They provide nutrient sources for host cells and regulate gut microenvironment, which are associated with a variety of diseases. Analysis of gut microbiota-host co-metabolites is of great significance to explore the host-gut microbiota interaction. In this study, we integrated chemical derivatization, liquid chromatography-mass spectrometry, and molecular networking (MN) to establish a novel CD-MN strategy for the analysis of carboxylated metabolites in gut microbial-host co-metabolism. Using this strategy, 261 carboxylated metabolites from mouse feces were detected, which grouped to various classes including fatty acids, bile acids, N-acyl amino acids, benzoheterocyclic acids, aromatic acids, and other unknown small-scale molecular clusters in MN. Based on the interpretation of the bile acid cluster, a novel type of phenylacetylated conjugates of host bile acids was identified, which were mediated by gut microbiota and exhibited a strong binding ability to Farnesoid X receptor and Takeda G protein-coupled receptor 5. Our proposed strategy offers a promising platform for uncovering carboxylated metabolites in gut microbial-host co-metabolism.

[Source](https://doi.org/10.1021/acs.analchem.3c02353)

10.1021/acs.analchem.3c02353

---

