## Airway dysbiosis accelerates lung function decline in chronic obstructive pulmonary disease.
 19.05.2023, _W Liang, Y Yang, S Gong, M Wei, Y Ma, R Feng, J Gao, X Liu, F Tu, W Ma, X Yi, Z Liang, F Wang, L Wang, D Chen, W Shu, BE Miller, R Tal-Singer, GC Donaldson, JA Wedzicha, D Singh, TMA Wilkinson, CE Brightling, R Chen, N Zhong, Z Wang_


Progressive lung function decline is a hallmark of chronic obstructive pulmonary disease (COPD). Airway dysbiosis occurs in COPD, but whether it contributes to disease progression remains unknown. Here, we show, through a longitudinal analysis of two cohorts involving four UK centers, that baseline airway dysbiosis in COPD patients, characterized by the enrichment of opportunistic pathogenic taxa, associates with a rapid forced expiratory volume in 1 s (FEV1) decline over 2 years. Dysbiosis associates with exacerbation-related FEV1 fall and sudden FEV1 fall at stability, contributing to long-term FEV1 decline. A third cohort in China further validates the microbiota-FEV1-decline association. Human multi-omics and murine studies show that airway Staphylococcus aureus colonization promotes lung function decline through homocysteine, which elicits a neutrophil apoptosis-to-NETosis shift via the AKT1-S100A8/A9 axis. S. aureus depletion via bacteriophages restores lung function in emphysema mice, providing a fresh approach to slow COPD progression by targeting the airway microbiome.

[Source](https://doi.org/10.1016/j.chom.2023.04.018)

10.1016/j.chom.2023.04.018

---

## An integrated tumor, immune and microbiome atlas of colon cancer.
 19.05.2023, _J Roelands, PJK Kuppen, EI Ahmed, R Mall, T Masoodi, P Singh, G Monaco, C Raynaud, NFCC de Miranda, L Ferraro, TC Carneiro-Lobo, N Syed, A Rawat, A Awad, J Decock, W Mifsud, LD Miller, S Sherif, MG Mohamed, D Rinchai, M Van den Eynde, RW Sayaman, E Ziv, F Bertucci, MA Petkar, S Lorenz, LS Mathew, K Wang, S Murugesan, D Chaussabel, AL Vahrmeijer, E Wang, A Ceccarelli, KA Fakhro, G Zoppoli, A Ballestrero, RAEM Tollenaar, FM Marincola, J Galon, SA Khodor, M Ceccarelli, W Hendrickx, D Bedognetti_


The lack of multi-omics cancer datasets with extensive follow-up information hinders the identification of accurate biomarkers of clinical outcome. In this cohort study, we performed comprehensive genomic analyses on fresh-frozen samples from 348 patients affected by primary colon cancer, encompassing RNA, whole-exome, deep T cell receptor and 16S bacterial rRNA gene sequencing on tumor and matched healthy colon tissue, complemented with tumor whole-genome sequencing for further microbiome characterization. A type 1 helper T cell, cytotoxic, gene expression signature, called Immunologic Constant of Rejection, captured the presence of clonally expanded, tumor-enriched T cell clones and outperformed conventional prognostic molecular biomarkers, such as the consensus molecular subtype and the microsatellite instability classifications. Quantification of genetic immunoediting, defined as a lower number of neoantigens than expected, further refined its prognostic value. We identified a microbiome signature, driven by Ruminococcus bromii, associated with a favorable outcome. By combining microbiome signature and Immunologic Constant of Rejection, we developed and validated a composite score (mICRoScore), which identifies a group of patients with excellent survival probability. The publicly available multi-omics dataset provides a resource for better understanding colon cancer biology that could facilitate the discovery of personalized therapeutic approaches.

[Source](https://doi.org/10.1038/s41591-023-02324-5)

10.1038/s41591-023-02324-5

---

## Skin microbiome alterations in upper extremity secondary lymphedema.
 19.05.2023, _AC Campbell, T Fei, JE Baik, HJ Park, J Shin, K Kuonqui, S Brown, A Sarker, RP Kataru, BJ Mehrara_


_Humans, Upper Extremity, Lymphatic Vessels, Skin, Neoplasms, Lymphedema_
Lymphedema is a chronic condition that commonly occur from lymphatic injury following surgical resection of solid malignancies. While many studies have centered on the molecular and immune pathways that perpetuate lymphatic dysfunction, the role of the skin microbiome in lymphedema development remains unclear. In this study, skin swabs collected from normal and lymphedema forearms of 30 patients with unilateral upper extremity lymphedema were analyzed by 16S ribosomal RNA sequencing. Statistical models for microbiome data were utilized to correlate clinical variables with microbial profiles. Overall, 872 bacterial taxa were identified. There were no significant differences in microbial alpha diversity of the colonizing bacteria between normal and lymphedema skin samples (p = 0.25). Notably, for patients without a history of infection, a one-fold change in relative limb volume was significantly associated with a 0.58-unit increase in Bray-Curtis microbial distance between paired limbs (95%CI = 0.11,1.05, p = 0.02). Additionally, several genera, including Propionibacterium and Streptococcus, demonstrated high variability between paired samples. In summary, we demonstrate high compositional heterogeneity in the skin microbiome in upper extremity secondary lymphedema, supporting future studies into the role of host-microbe interactions on lymphedema pathophysiology.

[Source](https://doi.org/10.1371/journal.pone.0283609)

10.1371/journal.pone.0283609

---

## Isolation and identification of hyaluronan-degrading bacteria from Japanese fecal microbiota.
 19.05.2023, _H Akazawa, I Fukuda, H Kaneda, S Yoda, M Kimura, R Nomoto, S Ueda, Y Shirai, R Osawa_


_Humans, Hyaluronic Acid, East Asian People, Bacteria, Feces, Gastrointestinal Microbiome_
Hyaluronan (HA) is a high-molecular-weight glycosaminoglycan and widely distributed in all connective tissues and organs with diverse biological functions. HA has been increasingly used as dietary supplements targeted to joint and skin health for humans. We here first report isolation of bacteria from human feces that are capable of degrading HA to lower molecular weight HA oligosaccharides (oligo-HAs). The bacteria were successfully isolated via a selective enrichment method, in which the serially diluted feces of healthy Japanese donors were individually incubated in an enrichment medium containing HA, followed by the isolation of candidate strains from streaked HA-containing agar plates and selection of HA-degrading strains by measuring HA using an ELISA. Subsequent genomic and biochemical assays identified the strains as Bacteroides finegoldii, B. caccae, B. thetaiotaomicron, and Fusobacterium mortiferum. Furthermore, our HPLC analysis revealed that the strains degraded HA to oligo-HAs of various lengths. Subsequent quantitative PCR assay targeting the HA degrading bacteria showed that their distribution in the Japanese donors varied. The evidence suggests that dietary HA is degraded by the human gut microbiota with individual variation to oligo-HAs components, which are more absorbable than HA, thereby exerting its beneficial effects.

[Source](https://doi.org/10.1371/journal.pone.0284517)

10.1371/journal.pone.0284517

---

## Bacterial and metabolic phenotypes associated with inadequate response to ursodeoxycholic acid treatment in primary biliary cholangitis.
 19.05.2023, _L Martinez-Gili, A Pechlivanis, JAK McDonald, S Begum, J Badrock, JK Dyson, R Jones, G Hirschfield, SD Ryder, R Sandford, S Rushbrook, D Thorburn, SD Taylor-Robinson, MME Crossey, JR Marchesi, G Mells, E Holmes, D Jones_


_Humans, Ursodeoxycholic Acid, Liver Cirrhosis, Biliary, Dehydrocholic Acid, RNA, Ribosomal, 16S, Cholagogues and Choleretics, Gastrointestinal Microbiome, Bile Acids and Salts, Biomarkers, Phenotype, Bacteria_
Primary biliary cholangitis (PBC) is a chronic cholestatic liver disease with ursodeoxycholic acid (UDCA) as first-line treatment. Poor response to UDCA is associated with a higher risk of progressing to cirrhosis, but the underlying mechanisms are unclear. UDCA modulates the composition of primary and bacterial-derived bile acids (BAs). We characterized the phenotypic response to UDCA based on BA and bacterial profiles of PBC patients treated with UDCA. Patients from the UK-PBC cohort (n = 419) treated with UDCA for a minimum of 12-months were assessed using the Barcelona dynamic response criteria. BAs from serum, urine, and feces were analyzed using Ultra-High-Performance Liquid Chromatography-Mass Spectrometry and fecal bacterial composition measured using 16S rRNA gene sequencing. We identified 191 non-responders, 212 responders, and a subgroup of responders with persistently elevated liver biomarkers (n = 16). Responders had higher fecal secondary and tertiary BAs than non-responders and lower urinary bile acid abundances, with the exception of 12-dehydrocholic acid, which was higher in responders. The sub-group of responders with poor liver function showed lower alpha-diversity evenness, lower abundance of fecal secondary and tertiary BAs than the other groups and lower levels of phyla with BA-deconjugation capacity (Actinobacteriota/Actinomycetota, Desulfobacterota, Verrucomicrobiota) compared to responders. UDCA dynamic response was associated with an increased capacity to generate oxo-/epimerized secondary BAs. 12-dehydrocholic acid is a potential biomarker of treatment response. Lower alpha-diversity and lower abundance of bacteria with BA deconjugation capacity might be associated with an incomplete response to treatment in some patients.

[Source](https://doi.org/10.1080/19490976.2023.2208501)

10.1080/19490976.2023.2208501

---

## Gut microbiota and its relation to inflammation in patients with bipolar depression: a cross-sectional study.
 21.05.2023, _T Huang, Y Shang, C Dai, Q Zhang, S Hu, J Xie_


To explore the gut microbiota characteristics in depressed patients with bipolar disorder (BD) as well as the connection between the gut microbiota and inflammatory markers.

[Source](https://doi.org/10.1186/s12991-023-00453-2)

10.1186/s12991-023-00453-2

---

## Alterations in the Gut Microbiome and Metabolisms in Pregnancies with Fetal Growth Restriction.
 18.05.2023, _Z Tao, Y Chen, F He, J Tang, L Zhan, H Hu, Z Ding, S Ruan, Y Chen, B Chen, Y Wang, X Guo, L Xie, M Zhong, Q Huang_


Fetuses diagnosed with fetal growth restriction (FGR) are at an elevated risk of stillbirth and adulthood morbidity. Gut dysbiosis has emerged as one of the impacts of placental insufficiency, which is the main cause of FGR. This study aimed to characterize the relationships among the intestinal microbiome, metabolites, and FGR. Characterization was conducted on the gut microbiome, fecal metabolome, and human phenotypes in a cohort of 35 patients with FGR and 35 normal pregnancies (NP). The serum metabolome was analyzed in 19 patients with FGR and 31 normal pregnant women. Multidimensional data was integrated to reveal the links between data sets. A fecal microbiota transplantation mouse model was used to determine the effects of the intestinal microbiome on fetal growth and placental phenotypes. The diversity and composition of the gut microbiota were altered in patients with FGR. A group of microbial species altered in FGR closely correlated with fetal measurements and maternal clinical variables. Fecal and serum metabolism profiles were distinct in FGR patients compared to those in the NP group. Altered metabolites were identified and associated with clinical phenotypes. Integrated multi-omics analysis revealed the interactions among gut microbiota, metabolites, and clinical measurements. Microbiota from FGR gravida transplanted to mice progestationally induced FGR and placental dysfunction, including impaired spiral artery remodeling and insufficient trophoblast cell invasion. Taken together, the integration of microbiome and metabolite profiles from the human cohort indicates that patients with FGR endure gut dysbiosis and metabolic disorders, which contribute to disease pathogenesis. IMPORTANCE Downstream of the primary cause of fetal growth restriction are placental insufficiency and fetal malnutrition. Gut microbiota and metabolites appear to play an important role in the progression of gestation, while dysbiosis induces maternal and fetal complications. Our study elaborates the significant differences in microbiota profiles and metabolome characteristics between women with FGR and normal pregnancies. This is the first attempt so far that reveals the mechanistic links in multi-omics in FGR, providing a novel insight into host-microbe interaction in placenta-derived diseases.

[Source](https://doi.org/10.1128/spectrum.00076-23)

10.1128/spectrum.00076-23

---

## 16S rRNA-Based Microbiota Profiling Assists Conventional Culture Analysis of Airway Samples from Pediatric Cystic Fibrosis Patients.
 18.05.2023, _M Kristensen, EM de Koff, ML Chu, S Groendijk, GA Tramper-Stranders, KM de Winter-de Groot, HM Janssens, HA Tiddens, M van Westreenen, EAM Sanders, BHGM Arets, CK van der Ent, SMPJ Prevaes, D Bogaert_


16S-based sequencing provides broader information on the respiratory microbial community than conventional culturing. However, it (often) lacks species- and strain-level information. To overcome this issue, we used 16S rRNA-based sequencing results from 246 nasopharyngeal samples obtained from 20 infants with cystic fibrosis (CF) and 43 healthy infants, which were all 0 to 6 months old, and compared them to both standard (blind) diagnostic culturing and a 16S-sequencing-informed "targeted" reculturing approach. Using routine culturing, we almost uniquely detected Moraxella catarrhalis, Staphylococcus aureus, and Haemophilus influenzae (42%, 38%, and 33% of samples, respectively). Using the targeted reculturing approach, we were able to reculture 47% of the top-5 operational taxonomical units (OTUs) in the sequencing profiles. In total, we identified 60 species from 30 genera with a median of 3 species per sample (range, 1 to 8). We also identified up to 10 species per identified genus. The success of reculturing the top-5 genera present from the sequencing profile depended on the genus. In the case of Corynebacterium being in the top 5, we recultured them in 79% of samples, whereas for Staphylococcus, this value was only 25%. The success of reculturing was also correlated with the relative abundance of those genera in the corresponding sequencing profile. In conclusion, revisiting samples using 16S-based sequencing profiles to guide a targeted culturing approach led to the detection of more potential pathogens per sample than conventional culturing and may therefore be useful in the identification and, consequently, treatment of bacteria considered relevant for the deterioration or exacerbation of disease in patients like those with CF. IMPORTANCE Early and effective treatment of pulmonary infections in cystic fibrosis is vital to prevent chronic lung damage. Although microbial diagnostics and treatment decisions are still based on conventional culture methods, research is gradually focusing more on microbiome and metagenomic-based approaches. This study compared the results of both methods and proposed a way to combine the best of both worlds. Many species can relatively easily be recultured based on the 16S-based sequencing profile, and it provides more in-depth information about the microbial composition of a sample than that obtained through routine (blind) diagnostic culturing. Still, well-known pathogens can be missed by both routine diagnostic culture methods as well as by targeted reculture methods, sometimes even when they are highly abundant, which may be a consequence of either sample storage conditions or antibiotic treatment at the time of sampling.

[Source](https://doi.org/10.1128/spectrum.04057-22)

10.1128/spectrum.04057-22

---

## Single-Molecule Approach to 16S rRNA for Vaginal Microbiome Signatures in Response to Metronidazole Treatment.
 18.05.2023, _H Qin, J Jiao, D A, M Hua, K Han, H Du, Z Wang, J Li, D Zhang, B Xiao, C Chen_


Bacterial vaginosis (BV) is the most common infection of the lower reproductive tract among women of reproductive age, characterized by a depletion of health-associated Lactobacillus and an overgrowth of anaerobes. Metronidazole has been recommended as a first-line therapy for treating BV for decades. Although most cases are cured by the treatment, recurrent infections of BV seriously affect women's reproductive health. Until now, limited information on the vaginal microbiota has been explored at the species level. Here, we adopted a single molecular sequencing approach for the 16S rRNA gene, named FLAST (full-length assembly sequencing technology), to analyze the human vaginal microbiota that improved species-level resolution for taxonomy and identified microbiota alterations in the vaginal tract in response to treatment with metronidazole. Appling high-throughput sequencing, we identified 96 and 189 novel full-length 16S rRNA gene sequences in Lactobacillus and Prevotella, respectively, which had not previously been reported in vaginal samples. Moreover, we found that Lactobacillus iners was significantly enriched in the cured group before metronidazole treatment, and that was maintained in a high frequency after the treatment, suggesting an important role for this species in response to metronidazole treatment. Our research also highlights the importance of the single-molecule paradigm for progressing the field of microbiology and applying these insights to better understand the dynamic microbiota during BV treatment. Subsequent novel treatment approaches should be proposed to improve BV treatment outcomes, optimize the vaginal microbiome, and reduce gynecological and obstetric sequelae. IMPORTANCE Bacterial vaginosis (BV) is a common infectious disease of the reproductive tract. Metronidazole treatment, as the first line of treatment, frequently fails at recovery of the microbiome. However, the precise types of Lactobacillus and other bacteria involved in BV remain unclear, and this has resulted in a failure to identify potential markers to predict clinic outcomes. In this study, we adopted a 16S rRNA gene full-length assembly sequencing technology for the taxonomy analysis and evaluation of vaginal microbiota before and after treatment with metronidazole. We additionally identified 96 and 189 novel 16S rRNA gene sequences in Lactobacillus and Prevotella species, respectively, in vaginal samples, which improves our understanding of the vaginal microbiota. Moreover, we found that the abundance of Lactobacillus iners and Prevotella bivia before treatment was associated with a lack of cure. These potential biomarkers will help to facilitate future studies aimed at improving BV treatment outcomes, optimize the vaginal microbiome, and reduce adverse sexual and reproductive outcomes.

[Source](https://doi.org/10.1128/spectrum.01706-22)

10.1128/spectrum.01706-22

---

## The presence of intratumoral <em>Porphyromonas gingivalis</em> correlates with a previously defined pancreatic adenocarcinoma, immune cell expression phenotype and with tumor resident, adaptive immune receptor features

[Source](https://doi.org/10.1093/carcin/bgad033)

---

## Microbiome is the key to preventing anastomotic leak in colorectal surgery.
 17.05.2023, _JM Kinross_



[Source](https://doi.org/10.1136/gutjnl-2022-329281)

10.1136/gutjnl-2022-329281

---

## The aerobiome-health axis: a paradigm shift in bioaerosol thinking

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(23)00130-0)

---

## Impact of the host microbiota on fungal infections: new possibilities for intervention?
 21.05.2023, _EWL Chow, L Mei Pang, Y Wang_


Many human fungal pathogens are opportunistic. They are primarily benign residents of the human body and only become infectious when the host's immunity and microbiome are compromised. Bacteria dominate the human microbiome, playing an essential role in keeping fungi harmless and acting as the first line of defense against fungal infection. The Human Microbiome Project, launched by NIH in 2007, has stimulated extensive investigation and significantly advanced our understanding of the molecular mechanisms governing the interaction between bacteria and fungi, providing valuable insights for developing future antifungal strategies by exploiting the interaction. This review summarizes recent progress in this field and discusses new possibilities and challenges. We must seize the opportunities presented by researching bacterial-fungal interplay in the human microbiome to address the global spread of drug-resistant fungal pathogens and the drying pipelines of effective antifungal drugs.

[Source](https://doi.org/10.1016/j.addr.2023.114896)

10.1016/j.addr.2023.114896

---

## Malnourished Microbes: Host-Microbiome Interactions in Child Undernutrition.
 19.05.2023, _HJ Jones, CD Bourke, JR Swann, RC Robertson_


Childhood undernutrition is a major global health burden that is only partially resolved by nutritional interventions. Both chronic and acute forms of child undernutrition are characterized by derangements in multiple biological systems including metabolism, immunity, and endocrine systems. A growing body of evidence supports a role of the gut microbiome in mediating these pathways influencing early life growth. Observational studies report alterations in the gut microbiome of undernourished children, while preclinical studies suggest that this can trigger intestinal enteropathy, alter host metabolism, and disrupt immune-mediated resistance against enteropathogens, each of which contribute to poor early life growth. Here, we compile evidence from preclinical and clinical studies and describe the emerging pathophysiological pathways by which the early life gut microbiome influences host metabolism, immunity, intestinal function, endocrine regulation, and other pathways contributing to child undernutrition. We discuss emerging microbiome-directed therapies and consider future research directions to identify and target microbiome-sensitive pathways in child undernutrition. Expected final online publication date for the Annual Review of Nutrition, Volume 43 is August 2023. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.

[Source](https://doi.org/10.1146/annurev-nutr-061121-091234)

10.1146/annurev-nutr-061121-091234

---

## A narrative review from gut to lungs: non-small cell lung cancer and the gastrointestinal microbiome.
 19.05.2023, _H Shah, TL Ng_


The gut microbiome has emerged as an important gateway to improving therapeutic outcomes in lung cancer, especially for immunotherapy. Our objective is to review the impact of the bidirectional relationship between the gut microbiome, lung cancer, and the immune system, and to identify areas of future research.

[Source](https://doi.org/10.21037/tlcr-22-595)

10.21037/tlcr-22-595

---

## The interplay between oral microbiota, gut microbiota and systematic diseases.
 21.05.2023, _X Tan, Y Wang, T Gong_


Over the past two decades, the importance of microbiota in health and disease has become evident. The human gut microbiota and oral microbiota are the largest and second-largest microbiome in the human body, respectively, and they are physically connected as the oral cavity is the beginning of the digestive system. Emerging and exciting evidence has shown complex and important connections between gut microbiota and oral microbiota. The interplay of the two microbiomes may contribute to the pathological processes of many diseases, including diabetes, rheumatoid arthritis, nonalcoholic fatty liver disease, inflammatory bowel disease, pancreatic cancer, colorectal cancer, and so on. In this review, we discuss possible routes and factors of oral microbiota to affect gut microbiota, and the contribution of this interplay between oral and gut microbiota to systemic diseases. Although most studies are association studies, recently, there have been increasing mechanistic investigations. This review aims to enhance the interest in the connection between oral and gut microbiota, and shows the tangible impact of this connection on human health.

[Source](https://doi.org/10.1080/20002297.2023.2213112)

10.1080/20002297.2023.2213112

---

## Exposing new taxonomic variation with inflammation - a murine model-specific genome database for gut microbiome researchers.
 22.05.2023, _I Leleiwi, J Rodriguez-Ramos, M Shaffer, A Sabag-Daigle, K Kokkinias, RM Flynn, RA Daly, LFM Kop, LM Solden, BMM Ahmer, MA Borton, KC Wrighton_


_Humans, Animals, Mice, Gastrointestinal Microbiome, Disease Models, Animal, Mice, Inbred CBA, Microbiota, Inflammation, Bacteroidetes_
The murine CBA/J mouse model widely supports immunology and enteric pathogen research. This model has illuminated Salmonella interactions with the gut microbiome since pathogen proliferation does not require disruptive pretreatment of the native microbiota, nor does it become systemic, thereby representing an analog to gastroenteritis disease progression in humans. Despite the value to broad research communities, microbiota in CBA/J mice are not represented in current murine microbiome genome catalogs.

[Source](https://doi.org/10.1186/s40168-023-01529-7)

10.1186/s40168-023-01529-7

---

## Protective effect of L-pipecolic acid on constipation in C57BL/6 mice based on gut microbiome and serum metabolomic.
 22.05.2023, _H Li, HY Xiao, LP Yuan, B Yan, Y Pan, PP Tian, WJ Zhang_


_Mice, Animals, Loperamide, Gastrointestinal Microbiome, Serotonin, Quality of Life, Mice, Inbred C57BL, Constipation, Water_
Functional constipation (FC) in children affects their growth, development and quality of life. L-pipecolic acid (L-PA) was decreased in FC children based on gut microbiome and serum metabolomic. In this study, loperamide-induced constipation in mice was used to evaluate the effects of L-PA on constipated mice.

[Source](https://doi.org/10.1186/s12866-023-02880-3)

10.1186/s12866-023-02880-3

---

## Alterations of the gut microbiome and metabolic profile in CVB3-induced mice acute viral myocarditis.
 22.05.2023, _Q Kong, L Chen, X Zeng, F Lu, Y Huang, W Wu_


_Animals, Mice, Myocarditis, Gastrointestinal Microbiome, Chromatography, Liquid, Tandem Mass Spectrometry, Metabolome, Metabolomics, Steroids, Hormones, RNA, Ribosomal, 16S_
Acute viral myocarditis (AVMC) is an inflammatory disease of the myocardium. Evidence indicates that dysbiosis of gut microbiome and related metabolites intimately associated with cardiovascular diseases through the gut-heart axis.

[Source](https://doi.org/10.1186/s12866-023-02863-4)

10.1186/s12866-023-02863-4

---

## Seaweed polysaccharides treatment alleviates injury of inflammatory responses and gut barrier in LPS-induced mice.
 21.05.2023, _Y He, M Xu, S Lu, W Zou, Y Wang, M Fakhar-E-Alam Kulyar, M Iqbal, K Li_


Gastrointestinal (GI) disease is a common digestive tract disease effects health of millions of human globally each year, thus the role of intestinal microflora had been emphasized. Seaweed polysaccharides featured a wide range of pharmacological activities, such as antioxidant activity and pharmacological action, but whether they can alleviate the dysbiosis of gut microbial ecology caused by lipopolysaccharide (LPS) exposure has not been well conducted. In this study, we investigated the effects of different concentration of seaweed polysaccharides on LPS-induced intestinal disorder by using hematoxylin and eosin (H&amp;E) staining and 16S rRNA high-throughput sequencing. Histopathological results indicated that the intestinal structure in the LPS-induced group was damaged. Furthermore, LPS exposure not only reduced the intestinal microbial diversity in mice but also induced considerable transformation in its composition, along with significant increase in pathogenic bacteria (Helicobacter, Citrobacter and Mucispirillum) and decrease in beneficial bacteria (Firmicutes, Lactobacillus, Akkermansia and Parabacteroides). Nonetheless, seaweed polysaccharides administration could recover the gut microbial dysbiosis and the loss of gut microbial diversity induced by LPS exposure. In summary, seaweed polysaccharides were effective against LPS-induced intestinal damage in mice via the modulation of intestinal microecology.

[Source](https://doi.org/10.1016/j.micpath.2023.106159)

10.1016/j.micpath.2023.106159

---

## An entomopathogenic fungus exploits its host humoral antibacterial immunity to minimize bacterial competition in the hemolymph.
 22.05.2023, _JL Wang, J Sun, YJ Song, HH Zheng, GJ Wang, WX Luo, L Li, XS Liu_


_Animals, Hemolymph, Moths, Insecta, Anti-Bacterial Agents, Bacteria_
The insect hemolymph (blood-equivalent fluid), composed of a large number of hemocytes (blood cells) and a variety of soluble immune effectors, is hostile for pathogens including fungi. In order to survive in the insect hemocoel (body cavity), the entomopathogenic fungus (EPF) has evolved two classical coping strategies, namely evasion and suppression of the host immune reactions. However, it remains unclear whether EPF has other ways of coping with host immunity.

[Source](https://doi.org/10.1186/s40168-023-01538-6)

10.1186/s40168-023-01538-6

---

## Engineered symbiotic bacteria interfering <em>Nosema</em> redox system inhibit microsporidia parasitism in honeybees

[Source](https://doi.org/10.1038/s41467-023-38498-2)

---

## New insights into microbial and metabolite signatures of coral bleaching.
 20.05.2023, _H Sun, Z Xie, X Yang, B Yang, B Liao, J Yin, B Xiao_


Coral bleaching and coral reef degradation have been severely increased due to anthropogenic impacts, especially global warming. Studies have indicated the key role of host-microbiome symbiotic relationships for the coral holobiont health and development, although not all of the mechanisms of interaction have been fully explored. Here, we explore bacterial and metabolic shifts within coral holobionts under thermal stress, and its correlation with bleaching. Our results showed obvious signs of coral bleaching after 13 days of heating treatment, and a more-complex co-occurrence network was observed in the coral-associated bacterial community of the heating group. The bacterial community and metabolites changed significantly under thermal stress, and genera Flavobacterium, Shewanella and Psychrobacter increased from &lt;0.1 % to 43.58 %, 6.95 % and 6.35 %, respectively. Bacteria potentially associated with stress tolerance, biofilm formation and mobile elements decreased from 80.93 %, 62.15 % and 49.27 % to 56.28 %, 28.41 % and 18.76 %, respectively. The differentially expressed metabolites of corals after heating treatment, such as Cer(d18:0/17:0), 1-Methyladenosine, Trp-P-1 and Marasmal, were associated with cell cycle regulation and antioxidant properties. Our results can contribute to our current understanding on the correlations between coral-symbiotic bacteria, metabolites and the coral physiological response to thermal stress. These new insights into the metabolomics of heat-stressed coral holobionts may expand our knowledge on the mechanisms underlying bleaching.

[Source](https://doi.org/10.1016/j.scitotenv.2023.164258)

10.1016/j.scitotenv.2023.164258

---

## Feeding hempseed cake alters the bovine gut, respiratory and reproductive microbiota.
 22.05.2023, _TM Winders, DB Holman, KN Schmidt, SM Luecke, DJ Smith, BW Neville, CR Dahlen, KC Swanson, S Amat_


_Humans, Cattle, Animals, Female, Infant, Animal Feed, RNA, Ribosomal, 16S, Diet, Silage, Reproduction, Zea mays, Rumen_
A growing number of studies have investigated the feasibility of utilizing hemp by-products as livestock feedstuffs; however, their impact on livestock microbiomes remains unexplored. Here, we evaluated the effects of feeding hempseed cake on the gastrointestinal, respiratory, and reproductive microbiota in beef heifers. Angus-crossbred heifers (19-months old, initial body weight = 494 ± 10 kg [SE]) were fed a corn-based finishing diet containing 20% hempseed cake as a substitute for 20% corn dried distillers' grains with solubles (DM basis; Control; n = 16/group) for 111 days until slaughter. Ruminal fluid and deep nasopharyngeal swabs (days 0, 7, 42, 70 and 98), and vaginal and uterine swabs (at slaughter) were collected, and the microbiota assessed using 16S rRNA gene sequencing. Diet affected the community structure of the ruminal (d 7-98; 0.06 ≤ R2 ≤ 0.12; P &lt; 0.05), nasopharyngeal (d 98; R2 = 0.18; P &lt; 0.001), and vaginal (R2 = 0.06; P &lt; 0.01) microbiota. Heifers fed hempseed cake had increased microbial diversity in the rumen, reduced microbial richness in the vagina, and greater microbial diversity and richness in the uterus. In addition to the distinct microbial communities in the rumen, nasopharynx, vagina and uterus, we identified 28 core taxa that were shared (≥ 60% of all samples) across these sampling locations. Feeding hempseed cake appeared to alter the bovine gut, respiratory and reproductive microbiota. Our results suggest that future research aiming to evaluate the use of hemp by-products in livestock diet should consider their impact on animal microbiome and microbiome mediated animal health and reproductive efficiency. Our findings also highlight the need for research evaluating the impact of hemp-associated food and personal care products on the human microbiome.

[Source](https://doi.org/10.1038/s41598-023-35241-1)

10.1038/s41598-023-35241-1

---

## Composition of mucus- and digesta-associated bacteria in growing pigs with and without diarrhea differed according to the presence of colonic inflammation

[Source](https://doi.org/10.1186/s12866-023-02874-1)

---

## The microbiome and gene expression of honey bee workers are affected by a diet containing pollen substitutes.
 22.05.2023, _JE Powell, P Lau, J Rangel, R Arnott, T De Jong, NA Moran_


_Bees, Animals, Microbiota, Gastrointestinal Microbiome, Diet, Dietary Proteins, Bacteria, Gene Expression_
Pollen is the primary source of dietary protein for honey bees. It also includes complex polysaccharides in its outer coat, which are largely indigestible by bees but can be metabolized by bacterial species within the gut microbiota. During periods of reduced availability of floral pollen, supplemental protein sources are frequently provided to managed honey bee colonies. The crude proteins in these supplemental feeds are typically byproducts from food manufacturing processes and are rarely derived from pollen. Our experiments on the impact of different diets showed that a simplified pollen-free diet formulated to resemble the macronutrient profile of a monofloral pollen source resulted in larger microbial communities with reduced diversity, reduced evenness, and reduced levels of potentially beneficial hive-associated bacteria. Furthermore, the pollen-free diet sharply reduced the expression of genes central to honey bee development. In subsequent experiments, we showed that these shifts in gene expression may be linked to colonization by the gut microbiome. Lastly, we demonstrated that for bees inoculated with a defined gut microbiota, those raised on an artificial diet were less able to suppress infection from a bacterial pathogen than those that were fed natural pollen. Our findings demonstrate that a pollen-free diet significantly impacts the gut microbiota and gene expression of honey bees, indicating the importance of natural pollen as a primary protein source.

[Source](https://doi.org/10.1371/journal.pone.0286070)

10.1371/journal.pone.0286070

---

## Amplicon-based metagenomic association analysis of gut microbiota in relation to egg-laying period and breeds of hens.
 22.05.2023, _XY Wang, JX Meng, WX Ren, H Ma, G Liu, R Liu, HL Geng, Q Zhao, XX Zhang, HB Ni_


_Animals, Female, Gastrointestinal Microbiome, Chickens, RNA, Ribosomal, 16S, Microbiota, Cyanobacteria_
The gut microbiota plays an essential role in maintaining gut homeostasis and improving performance, with the composition of microbial communities visibly differing across different laying stages in hens and significantly correlating with egg production. To gain further insights into the association between microbial community characteristics and laying periods in Hy-Line variety brown and Isa brown laying hens, we conducted a 16S rRNA amplicon sequencing survey.

[Source](https://doi.org/10.1186/s12866-023-02857-2)

10.1186/s12866-023-02857-2

---

## Diversity and compositional differences of the airborne microbiome in a biophilic indoor environment

[Source](https://doi.org/10.1038/s41598-023-34928-9)

---

## Engineering the Crop Microbiota Through Host Genetics.
 17.05.2023, _C Escudero-Martinez, D Bulgarelli_


The microbiota populating the plant-soil continuum defines an untapped resource for sustainable crop production. The host plant is a driver for the taxonomic composition and function of these microbial communities. In this review, we illustrate how the host genetic determinants of the microbiota have been shaped by plant domestication and crop diversification. We discuss how the heritable component of microbiota recruitment may represent, at least partially, a selection for microbial functions underpinning the growth, development, and health of their host plants and how the magnitude of this heritability is influenced by the environment. We illustrate how host-microbiota interactions can be treated as an external quantitative trait and review recent studies associating crop genetics with microbiota-based quantitative traits. We also explore the results of reductionist approaches, including synthetic microbial communities, to establish causal relationships between microbiota and plant phenotypes. Lastly, we propose strategies to integrate microbiota manipulation into crop selection programs. Although a detailed understanding of when and how heritability for microbiota composition can be deployed for breeding purposes is still lacking, we argue that advances in crop genomics are likely to accelerate wider applications of plant-microbiota interactions in agriculture. Expected final online publication date for the Annual Review of Phytopathology, Volume 61 is September 2023. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.

[Source](https://doi.org/10.1146/annurev-phyto-021621-121447)

10.1146/annurev-phyto-021621-121447

---

## Root microbiome diversity and structure of the Sonoran desert buffelgrass (<em>Pennisetum ciliare L.</em>)

[Source](https://doi.org/10.1371/journal.pone.0285978)

---

## Long-read sequencing for the metagenomic analysis of microbiomes.
 19.05.2023, _T Free_


_High-Throughput Nucleotide Sequencing, Microbiota, Metagenome, Sequence Analysis, DNA, Metagenomics_
One technology, long-read sequencing, and one research field, microbiome studies, have risen to prominence over the last decade. But how can one be used in the other? What changes are being wrought? And what limitations remain? [Formula: see text].

[Source](https://doi.org/10.2144/btn-2023-0028)

10.2144/btn-2023-0028

---

