## Association between microbiome and the development of adverse posttraumatic neuropsychiatric sequelae after traumatic stress exposure.
 21.11.2023, _AL Zeamer, MC Salive, X An, FL Beaudoin, SL House, JS Stevens, D Zeng, TC Neylan, GD Clifford, SD Linnstaedt, SL Rauch, AB Storrow, C Lewandowski, PI Musey, PL Hendry, S Sheikh, CW Jones, BE Punches, RA Swor, LA Hudak, JL Pascual, MJ Seamon, E Harris, C Pearson, DA Peak, RC Merchant, RM Domeier, NK Rathlev, BJ O'Neil, P Sergot, LD Sanchez, SE Bruce, RC Kessler, KC Koenen, SA McLean, V Bucci, JP Haran_


_Adult, Humans, Microbiota, Stress Disorders, Post-Traumatic, Gastrointestinal Microbiome, Feces, Biological Availability_
Patients exposed to trauma often experience high rates of adverse post-traumatic neuropsychiatric sequelae (APNS). The biological mechanisms promoting APNS are currently unknown, but the microbiota-gut-brain axis offers an avenue to understanding mechanisms as well as possibilities for intervention. Microbiome composition after trauma exposure has been poorly examined regarding neuropsychiatric outcomes. We aimed to determine whether the gut microbiomes of trauma-exposed emergency department patients who develop APNS have dysfunctional gut microbiome profiles and discover potential associated mechanisms. We performed metagenomic analysis on stool samples (n = 51) from a subset of adults enrolled in the Advancing Understanding of RecOvery afteR traumA (AURORA) study. Two-, eight- and twelve-week post-trauma outcomes for post-traumatic stress disorder (PTSD) (PTSD checklist for DSM-5), normalized depression scores (PROMIS Depression Short Form 8b) and somatic symptom counts were collected. Generalized linear models were created for each outcome using microbial abundances and relevant demographics. Mixed-effect random forest machine learning models were used to identify associations between APNS outcomes and microbial features and encoded metabolic pathways from stool metagenomics. Microbial species, including Flavonifractor plautii, Ruminococcus gnavus and, Bifidobacterium species, which are prevalent commensal gut microbes, were found to be important in predicting worse APNS outcomes from microbial abundance data. Notably, through APNS outcome modeling using microbial metabolic pathways, worse APNS outcomes were highly predicted by decreased L-arginine related pathway genes and increased citrulline and ornithine pathways. Common commensal microbial species are enriched in individuals who develop APNS. More notably, we identified a biological mechanism through which the gut microbiome reduces global arginine bioavailability, a metabolic change that has also been demonstrated in the plasma of patients with PTSD.

[Source](https://doi.org/10.1038/s41398-023-02643-8)

10.1038/s41398-023-02643-8

---

## The preterm human milk microbiota fluctuates by postpartum week and is characterized by gestational age and maternal BMI.
 17.11.2023, _EJ Filatava, Z Liu, J Xie, D-B Tran, K Chen, N El Habbal, G Weinstock, Y Zhou, KE Gregory_


Despite a growing recognition that the type of nutrition received by preterm infants influences their intestinal microbiome and health outcomes, the microbiota of mother's own milk (MOM), pasteurized donor human milk (PDHM), and infant formula remain poorly characterized. In our study, we found that the structure of microbial communities, bacterial diversity, and relative abundances of specific genera were significantly different between MOM, PDHM, and formula. Additionally, our results suggest that the microbiota of MOM changes as a function of time and maternal factors. Lastly, we identified three lactotypes within MOM that have distinct microbial compositions and described the maternal factors associated with them. These findings set the stage for future research aimed at advancing our knowledge of the microbiota of preterm infant nutrition and the specific influence it may have on health outcomes.

[Source](https://doi.org/10.1128/mbio.02106-23)

10.1128/mbio.02106-23

---

## Strain-resolved metagenomic analysis of the gut as a reservoir for bloodstream infection pathogens among premature infants in Singapore

[Source](https://doi.org/10.1186/s13099-023-00583-8)

---

## <em>Porphyromonas gingivalis</em> as a promotor in the development of the alcoholic liver disease via ferroptosis

[Source](https://doi.org/10.1016/j.micinf.2023.105250)

---

## The core phageome and its interrelationship with preterm human milk lipids

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(23)01385-2)

---

## Longitudinal analysis at three oral sites links oral microbiota to clinical outcomes in allogeneic hematopoietic stem-cell transplant

[Source](https://doi.org/10.1128/spectrum.02910-23)

---

## Gut inflammation associated with age and Alzheimerâ€™s disease pathology: a human cohort study

[Source](https://doi.org/10.1038/s41598-023-45929-z)

---

## Robust airway microbiome signatures in acute respiratory failure and hospital-acquired pneumonia.
 13.11.2023, _E Montassier, GD Kitsios, JE Radder, Q Le Bastard, BJ Kelly, A Panzer, SV Lynch, CS Calfee, RP Dickson, A Roquilly_


Respiratory microbial dysbiosis is associated with acute respiratory distress syndrome (ARDS) and hospital-acquired pneumonia (HAP) in critically ill patients. However, we lack reproducible respiratory microbiome signatures that can increase our understanding of these conditions and potential treatments. Here, we analyze 16S rRNA sequencing data from 2,177 respiratory samples collected from 1,029 critically ill patients (21.7% with ARDS and 26.3% with HAP) and 327 healthy controls, sourced from 17 published studies. After data harmonization and pooling of individual patient data, we identified microbiota signatures associated with ARDS, HAP and prolonged mechanical ventilation. Microbiota signatures for HAP and prolonged mechanical ventilation were characterized by depletion of a core group of microbes typical of healthy respiratory samples, and the ARDS microbiota signature was distinguished by enrichment of potentially pathogenic respiratory microbes, including Pseudomonas and Staphylococcus. Using machine learning models, we identified clinically informative, three- and four-factor signatures that predicted ARDS, HAP and prolonged mechanical ventilation with relatively high accuracy (area under the curve of 0.751, 0.72 and 0.727, respectively). We validated the signatures in an independent prospective cohort of 136 patients on mechanical ventillation and found that patients with microbiome signatures associated with ARDS, HAP or prolonged mechanical ventilation had longer times to successful extubation than patients lacking these signatures (hazard ratios of 1.56 (95% confidence interval (CI) 1.07-2.27), 1.51 (95% CI 1.02-2.23) and 1.50 (95% CI 1.03-2.18), respectively). Thus, we defined and validated robust respiratory microbiome signatures associated with ARDS and HAP that may help to identify promising targets for microbiome therapeutic modulation in critically ill patients.

[Source](https://doi.org/10.1038/s41591-023-02617-9)

10.1038/s41591-023-02617-9

---

## Systematic mining of the human microbiome identifies antimicrobial peptides with diverse activity spectra.
 17.11.2023, _AM King, Z Zhang, E Glassey, P Siuti, J Clardy, CA Voigt_


Human-associated bacteria secrete modified peptides to control host physiology and remodel the microbiota species composition. Here we scanned 2,229 Human Microbiome Project genomes of species colonizing skin, gastrointestinal tract, urogenital tract, mouth and trachea for gene clusters encoding RiPPs (ribosomally synthesized and post-translationally modified peptides). We found 218 lanthipeptides and 25 lasso peptides, 70 of which were synthesized and expressed in E. coli and 23 could be purified and functionally characterized. They were tested for activity against bacteria associated with healthy human flora and pathogens. New antibiotics were identified against strains implicated in skin, nasal and vaginal dysbiosis as well as from oral strains selectively targeting those in the gut. Extended- and narrow-spectrum antibiotics were found against methicillin-resistant Staphylococcus aureus and vancomycin-resistant Enterococci. Mining natural products produced by human-associated microbes will enable the elucidation of ecological relationships and may be a rich resource for antimicrobial discovery.

[Source](https://doi.org/10.1038/s41564-023-01524-6)

10.1038/s41564-023-01524-6

---

## Skin benefits of postbiotics derived from <em>Micrococcus luteus</em> derived from human skin: an untapped potential for dermatological health

[Source](https://doi.org/10.1007/s13258-023-01471-w)

---

## <em>Akkermansia muciniphila</em> attenuated lipopolysaccharide-induced acute lung injury by modulating the gut microbiota and SCFAs in mice

[Source](https://doi.org/10.1039/d3fo04051h)

---

## The Potential of Meta-Proteomics and Artificial Intelligence to Establish the Next Generation of Probiotics for Personalized Healthcare

[Source](https://doi.org/10.1021/acs.jafc.3c03834)

---

## Gut microbiome in healthy aging versus those associated with frailty

[Source](https://doi.org/10.1080/19490976.2023.2278225)

---

## Host species and habitat shape fish-associated bacterial communities: phylosymbiosis between fish and their microbiome

[Source](https://doi.org/10.1186/s40168-023-01697-6)

---

## <em>Aedes aegypti</em> microbiome composition covaries with the density of <em>Wolbachia</em> infection

[Source](https://doi.org/10.1186/s40168-023-01678-9)

---

## Sequential host-bacteria and bacteria-bacteria interactions determine the microbiome establishment of <em>Nematostella vectensis</em>

[Source](https://doi.org/10.1186/s40168-023-01701-z)

---

## Microbial symbionts are shared between ants and their associated beetles

[Source](https://doi.org/10.1111/1462-2920.16544)

---

## The gut microbiome mediates adaptation to scarce food in Coleoptera

[Source](https://doi.org/10.1186/s40793-023-00537-2)

---

## Interplay of gut microbiota and host epithelial mitochondrial dysfunction is necessary for the development of spontaneous intestinal inflammation in mice

[Source](https://doi.org/10.1186/s40168-023-01686-9)

---

## Microbial diversity and community structure dynamics in acid mine drainage: Acidic fire with dissolved heavy metals

[Source](https://doi.org/10.1016/j.scitotenv.2023.168635)

---

## Polar lake microbiomes have distinct evolutionary histories

[Source](https://doi.org/10.1126/sciadv.ade7130)

---

## PSKR1 balances the plant growth-defence trade-off in the rhizosphere microbiome

[Source](https://doi.org/10.1038/s41477-023-01539-1)

---

## <em>Streptomyces</em> alleviate abiotic stress in plant by producing pteridic acids

[Source](https://doi.org/10.1038/s41467-023-43177-3)

---

## The pan-microbiome profiling system Taxa4Meta identifies clinical dysbiotic features and classifies diarrheal disease

[Source](https://doi.org/10.1172/JCI170859)

---

## Machine learning for microbiologists

[Source](https://doi.org/10.1038/s41579-023-00984-1)

---

