## <em>Neisseria</em> species as pathobionts in bronchiectasis

[Source](https://doi.org/10.1016/j.chom.2022.08.005)

---

## The gut microbiome molecular complex in human health and disease.
 15.09.2022, _P Wilmes, C Martin-Gallausiaux, M Ostaszewski, VTE Aho, PV Novikova, CC Laczny, JG Schneider_


The human gut microbiome produces a functional complex of biomolecules, including nucleic acids, (poly)peptides, structural molecules, and metabolites. This impacts human physiology in multiple ways, especially by triggering inflammatory pathways in disease. At present, much remains to be learned about the identity of key effectors and their causal roles.

[Source](https://doi.org/10.1016/j.chom.2022.08.016)

10.1016/j.chom.2022.08.016

---

## Codiversification of gut microbiota with humans.
 15.09.2022, _TA Suzuki, JL Fitzstevens, VT Schmidt, H Enav, KE Huus, M Mbong Ngwese, A Grießhammer, A Pfleiderer, BR Adegbite, JF Zinsou, M Esen, TP Velavan, AA Adegnika, LH Song, TD Spector, AL Muehlbauer, N Marchi, H Kang, L Maier, R Blekhman, L Ségurel, G Ko, ND Youngblut, P Kremsner, RE Ley_


The gut microbiomes of human populations worldwide have many core microbial species in common. However, within a species, some strains can show remarkable population specificity. The question is whether such specificity arises from a shared evolutionary history (codiversification) between humans and their microbes. To test for codiversification of host and microbiota, we analyzed paired gut metagenomes and human genomes for 1225 individuals in Europe, Asia, and Africa, including mothers and their children. Between and within countries, a parallel evolutionary history was evident for humans and their gut microbes. Moreover, species displaying the strongest codiversification independently evolved traits characteristic of host dependency, including reduced genomes and oxygen and temperature sensitivity. These findings all point to the importance of understanding the potential role of population-specific microbial strains in microbiome-mediated disease phenotypes.

[Source](https://doi.org/10.1126/science.abm7759)

10.1126/science.abm7759

---

## Gut microbiome of multiple sclerosis patients and paired household healthy controls reveal associations with disease risk and course

[Source](https://doi.org/10.1016/j.cell.2022.08.021)

---

## Composition and diverse differences of intestinal microbiota in ulcerative colitis patients.
 17.09.2022, _S Zhu, M Han, S Liu, L Fan, H Shi, P Li_


To explore the composition of the intestinal microbiota in ulcerative colitis (UC) patients and to identify differences in the microbiota between patients with active disease and those in remission.

[Source](https://doi.org/10.3389/fcimb.2022.953962)

10.3389/fcimb.2022.953962

---

## A systematic review on omics data (metagenomics, metatranscriptomics, and metabolomics) in the role of microbiome in gallbladder disease.
 17.09.2022, _P Di Carlo, N Serra, R Alduina, R Guarino, A Craxì, A Giammanco, T Fasciana, A Cascio, CM Sergi_


Microbiotas are the range of microorganisms (mainly bacteria and fungi) colonizing multicellular, macroscopic organisms. They are crucial for several metabolic functions affecting the health of the host. However, difficulties hamper the investigation of microbiota composition in cultivating microorganisms in standard growth media. For this reason, our knowledge of microbiota can benefit from the analysis of microbial macromolecules (DNA, transcripts, proteins, or by-products) present in various samples collected from the host. Various omics technologies are used to obtain different data. Metagenomics provides a taxonomical profile of the sample. It can also be used to obtain potential functional information. At the same time, metatranscriptomics can characterize members of a microbiome responsible for specific functions and elucidate genes that drive the microbiotas relationship with its host. Thus, while microbiota refers to microorganisms living in a determined environment (taxonomy of microorganisms identified), microbiome refers to the microorganisms and their genes living in a determined environment and, of course, metagenomics focuses on the genes and collective functions of identified microorganisms. Metabolomics completes this framework by determining the metabolite fluxes and the products released into the environment. The gallbladder is a sac localized under the liver in the human body and is difficult to access for bile and tissue sampling. It concentrates the bile produced in the hepatocytes, which drains into bile canaliculi. Bile promotes fat digestion and is released from the gallbladder into the upper small intestine in response to food. Considered sterile originally, recent data indicate that bile microbiota is associated with the biliary tract's inflammation and carcinogenesis. The sample size is relevant for omic studies of rare diseases, such as gallbladder carcinoma. Although in its infancy, the study of the biliary microbiota has begun taking advantage of several omics strategies, mainly based on metagenomics, metabolomics, and mouse models. Here, we show that omics analyses from the literature may provide a more comprehensive image of the biliary microbiota. We review studies performed in this environmental niche and focus on network-based approaches for integrative studies.

[Source](https://doi.org/10.3389/fphys.2022.888233)

10.3389/fphys.2022.888233

---

## Variability of strain engraftment and predictability of microbiome composition after fecal microbiota transplantation across different diseases.
 15.09.2022, _G Ianiro, M Punčochář, N Karcher, S Porcari, F Armanini, F Asnicar, F Beghini, A Blanco-Míguez, F Cumbo, P Manghi, F Pinto, L Masucci, G Quaranta, S De Giorgi, GD Sciumè, S Bibbò, F Del Chierico, L Putignani, M Sanguinetti, A Gasbarrini, M Valles-Colomer, G Cammarota, N Segata_


Fecal microbiota transplantation (FMT) is highly effective against recurrent Clostridioides difficile infection and is considered a promising treatment for other microbiome-related disorders, but a comprehensive understanding of microbial engraftment dynamics is lacking, which prevents informed applications of this therapeutic approach. Here, we performed an integrated shotgun metagenomic systematic meta-analysis of new and publicly available stool microbiomes collected from 226 triads of donors, pre-FMT recipients and post-FMT recipients across eight different disease types. By leveraging improved metagenomic strain-profiling to infer strain sharing, we found that recipients with higher donor strain engraftment were more likely to experience clinical success after FMT (P = 0.017) when evaluated across studies. Considering all cohorts, increased engraftment was noted in individuals receiving FMT from multiple routes (for example, both via capsules and colonoscopy during the same treatment) as well as in antibiotic-treated recipients with infectious diseases compared with antibiotic-naïve patients with noncommunicable diseases. Bacteroidetes and Actinobacteria species (including Bifidobacteria) displayed higher engraftment than Firmicutes except for six under-characterized Firmicutes species. Cross-dataset machine learning predicted the presence or absence of species in the post-FMT recipient at 0.77 average AUROC in leave-one-dataset-out evaluation, and highlighted the relevance of microbial abundance, prevalence and taxonomy to infer post-FMT species presence. By exploring the dynamics of microbiome engraftment after FMT and their association with clinical variables, our study uncovered species-specific engraftment patterns and presented machine learning models able to predict donors that might optimize post-FMT specific microbiome characteristics for disease-targeted FMT protocols.

[Source](https://doi.org/10.1038/s41591-022-01964-3)

10.1038/s41591-022-01964-3

---

## Drivers and determinants of strain dynamics following fecal microbiota transplantation.
 15.09.2022, _TSB Schmidt, SS Li, OM Maistrenko, W Akanni, LP Coelho, S Dolai, A Fullam, AM Glazek, R Hercog, H Herrema, F Jung, S Kandels, A Orakov, R Thielemann, M von Stetten, T Van Rossum, V Benes, TJ Borody, WM de Vos, CY Ponsioen, M Nieuwdorp, P Bork_


Fecal microbiota transplantation (FMT) is a therapeutic intervention for inflammatory diseases of the gastrointestinal tract, but its clinical mode of action and subsequent microbiome dynamics remain poorly understood. Here we analyzed metagenomes from 316 FMTs, sampled pre and post intervention, for the treatment of ten different disease indications. We quantified strain-level dynamics of 1,089 microbial species, complemented by 47,548 newly constructed metagenome-assembled genomes. Donor strain colonization and recipient strain resilience were mostly independent of clinical outcomes, but accurately predictable using LASSO-regularized regression models that accounted for host, microbiome and procedural variables. Recipient factors and donor-recipient complementarity, encompassing entire microbial communities to individual strains, were the main determinants of strain population dynamics, providing insights into the underlying processes that shape the post-FMT gut microbiome. Applying an ecology-based framework to our findings indicated parameters that may inform the development of more effective, targeted microbiome therapies in the future, and suggested how patient stratification can be used to enhance donor microbiota colonization or the displacement of recipient microbes in clinical practice.

[Source](https://doi.org/10.1038/s41591-022-01913-0)

10.1038/s41591-022-01913-0

---

## Long-term ecological and evolutionary dynamics in the gut microbiomes of carbapenemase-producing Enterobacteriaceae colonized subjects.
 15.09.2022, _JTL Kang, JJY Teo, D Bertrand, A Ng, A Ravikrishnan, M Yong, OT Ng, K Marimuthu, SL Chen, KR Chng, YH Gan, N Nagarajan_


Long-term colonization of the gut microbiome by carbapenemase-producing Enterobacteriaceae (CPE) is a growing area of public health concern as it can lead to community transmission and rapid increase in cases of life-threatening CPE infections. Here, leveraging the observation that many subjects are decolonized without interventions within a year, we used longitudinal shotgun metagenomics (up to 12 timepoints) for detailed characterization of ecological and evolutionary dynamics in the gut microbiome of a cohort of CPE-colonized subjects and family members (n = 46; 361 samples). Subjects who underwent decolonization exhibited a distinct ecological shift marked by recovery of microbial diversity, key commensals and anti-inflammatory pathways. In addition, colonization was marked by elevated but unstable Enterobacteriaceae abundances, which exhibited distinct strain-level dynamics for different species (Escherichia coli and Klebsiella pneumoniae). Finally, comparative analysis with whole-genome sequencing data from CPE isolates (n = 159) helped identify substrain variation in key functional genes and the presence of highly similar E. coli and K. pneumoniae strains with variable resistance profiles and plasmid sharing. These results provide an enhanced view into how colonization by multi-drug-resistant bacteria associates with altered gut ecology and can enable transfer of resistance genes, even in the absence of overt infection and antibiotic usage.

[Source](https://doi.org/10.1038/s41564-022-01221-w)

10.1038/s41564-022-01221-w

---

## Structural and functional dysbiosis of gut microbiota in Tibetan subjects with coronary heart disease.
 17.09.2022, _Q Cheng, C Fan, F Liu, Y Li, H Hou, Y Ma, Y Tan, Y Li, Y Hai, T Wu, L Zhang, Y Zhang_


The gut microbiota plays a crucial role in coronary heart disease (CHD). However, only a few studies focusing on the relationship between gut microbiota and CHD in ethnic populations are available. Here, we employed shotgun sequencing of the gut metagenome to analyze the taxonomic composition and functional annotation of the gut microbiota of 14 CHD patients, 13 patients with non-stenosis coronary heart disease (NCHD), and 18 healthy controls (HT) in Tibetan subjects. We found that the α-diversity of the gut microbiota was not significantly different among the three groups., whereas β-diversity was significantly altered in the CHD group compared with HT. Based on the receiver operating characteristic curve (ROC) analysis, the relative abundance of Proteobacteria species effectively distinguished patients with CHD from the control group. Most of the enriched species belonged to Proteobacteria. The pathways that contributed the most to the differences between groups were amino acid metabolism-related pathways, especially lysine biosynthesis. The enzymes of the lysine biosynthesis pathway, including K01714 and K00821, were significantly decreased in the CHD group. Our findings increase the understanding of the association between CHD pathogenesis and gut microbiota in the Tibetan population, thus paving the way for the development of improved diagnostic methods and treatments for Tibetan patients with CHD.

[Source](https://doi.org/10.1016/j.ygeno.2022.110483)

10.1016/j.ygeno.2022.110483

---

## Growth of succinate consumer <em>Dialister hominis</em> is supported by <em>Bacteroides thetaiotaomicron</em>

[Source](https://doi.org/10.1016/j.anaerobe.2022.102642)

---

## RhoB affects colitis through modulating cell signaling and intestinal microbiome.
 16.09.2022, _J Yang, G Pei, X Sun, Y Xiao, C Miao, L Zhou, B Wang, L Yang, M Yu, ZS Zhang, ET Keller, Z Yao, Q Wang_


The pathogenesis of inflammatory bowel diseases (IBD) is multifactorial, and diagnostic and treatment strategies for IBD remain to be developed. RhoB regulates multiple cell functions; however, its role in colitis is unexplored.

[Source](https://doi.org/10.1186/s40168-022-01347-3)

10.1186/s40168-022-01347-3

---

## Diet fuelling inflammatory bowel diseases: preclinical and clinical concepts.
 16.09.2022, _TE Adolph, J Zhang_


The diet and gut microbiota have been extensively interrogated as a fuel for gut inflammation in inflammatory bowel diseases (IBDs) in the last few years. Here, we review how specific nutrients, typically enriched in a Western diet, instigate or deteriorate experimental gut inflammation in a genetically susceptible host and we discuss microbiota-dependent and independent mechanisms. We depict the study landscape of nutritional trials in paediatric and adult IBD and delineate common grounds for dietary advice. Conclusively, the diet reflects a critical rheostat of microbial dysbiosis and gut inflammation in IBD. Dietary restriction by exclusive enteral nutrition, with or without a specific exclusion diet, is effectively treating paediatric Crohn's disease, while adult IBD trials are less conclusive. Insights into molecular mechanisms of nutritional therapy will change the perception of IBD and will allow us to enter the era of precision nutrition. To achieve this, we discuss the need for carefully designed nutritional trials with scientific rigour comparable to medical trials, which also requires action from stake holders. Establishing evidence-based dietary therapy for IBD does not only hold promise to avoid long-term immunosuppression, but to provide a widely accessible therapy at low cost. Identification of dietary culprits disturbing gut health also bears the potential to prevent IBD and allows informed decision making in food politics.

[Source](https://doi.org/10.1136/gutjnl-2021-326575)

10.1136/gutjnl-2021-326575

---

## Recent findings in Akkermansia muciniphila-regulated metabolism and its role in intestinal diseases.
 16.09.2022, _MJ Liu, JY Yang, ZH Yan, S Hu, JQ Li, ZX Xu, YP Jian_


The mammalian gastrointestinal tract is colonized with a majority of gut microbes, affecting host metabolism and homeostasis. Gut microbiota plays a vital role in nutrient exchange, signaling transduction between intestinal epithelial cells, and resistance to pathogen invasion. Gut microbiota is divided into mucus layer bacteria and intestinal lumen bacteria based on the colonization distribution. Akkermansia muciniphila (A. muciniphila) prefers to colonize in the intestinal mucus layer, and specifically degrades mucins to produce short-chain fatty acids, providing energy for the host and promoting colonization of the bacterium itself. Degradation of mucins prompts the host to compensate for the production of more mucins, thereby maintaining the dynamics of these proteins. In the intestinal micro-ecosystem, A. muciniphila is non-pathogenic, and its colonization with suitable abundance contributes to the development of immune system, thus promoting intestinal health. The mechanisms by which A. muciniphila bears a protective role in the host intestine are currently unclear. In this review, we summarize the microenvironment for the colonization of A. muciniphila, physiological characteristics and pathophysiological impact of A. muciniphila on intestinal diseases, such as irritable bowel syndrome, inflammatory bowel diseases, and intestinal tumors. We also provided updates for current studies on signals that A. muciniphila enhances intestinal barrier integrity and regulates immune response. Together, we conclude that A. muciniphila is a promising probiotic, which could be a microbial target for the treatment of multiple intestinal diseases.

[Source](https://doi.org/10.1016/j.clnu.2022.08.029)

10.1016/j.clnu.2022.08.029

---

## Recent findings in <em>Akkermansia muciniphila</em>-regulated metabolism and its role in intestinal diseases

[Source](https://doi.org/10.1016/j.clnu.2022.08.029)

---

## Comprehensive analysis of microbiome, metabolome and transcriptome revealed the mechanisms of <em>Moringa oleifera</em> polysaccharide on preventing ulcerative colitis

[Source](https://doi.org/10.1016/j.ijbiomac.2022.09.100)

---

## Probiotics ameliorates pulmonary inflammation via modulating gut microbiota and rectifying Th17/Treg imbalance in a rat model of PM2.5 induced lung injury

[Source](https://doi.org/10.1016/j.ecoenv.2022.114060)

---

## Short-term supplementation with ω-3 polyunsaturated fatty acids modulates primarily mucolytic species from the gut luminal mucin niche in a human fermentation system

[Source](https://doi.org/10.1080/19490976.2022.2120344)

---

## Interactive Effect of Combined Intermittent and Sustained Hypoxia and High-Fat Diet on the Colonic Mucosal Microbiome and Host Gene Expression in Mice

[Source](https://doi.org/10.2147/NSS.S370957)

---

## Fungi are more transient than bacteria in caterpillar gut microbiomes

[Source](https://www.nature.com/articles/s41598-022-19855-5)

---

## Adapting to climate with limited genetic diversity: Nucleotide, DNA methylation and microbiome variation among populations of the social spider <em>Stegodyphus dumicola</em>

[Source](https://doi.org/10.1111/mec.16696)

---

## Filtering ASVs/OTUs via mutual information-based microbiome network analysis

[Source](https://doi.org/10.1186/s12859-022-04919-0)

---

## Batch effects removal for microbiome data via conditional quantile regression

[Source](https://doi.org/10.1038/s41467-022-33071-9)

---

## Phylogenetic Analysis of Anti-CRISPR and Member Addition in the Families

[Source](https://doi.org/10.1007/s12033-022-00558-1)

---

## Shallow shotgun sequencing of the microbiome recapitulates 16S amplicon results and provides functional insights

[Source](https://doi.org/10.1111/1755-0998.13713)

---

## SnapShot: The Expobiome Map

[Source](https://doi.org/10.1016/j.chom.2022.08.015)

---

