## Influence of Intratumor Microbiome on Clinical Outcome and Immune Processes in Prostate Cancer.
 09.09.2020, _J Ma, A Gnanasekar, A Lee, WT Li, M Haas, J Wang-Rodriguez, EY Chang, M Rajasekaran, WM Ongkeko_


Although 1 in 9 American men will receive a diagnosis of prostate cancer (PC), most men with this diagnosis will not die from it, as most PCs are indolent. However, there is a subset of patients in which the once-indolent PC becomes metastatic and eventually, fatal. In this study, we analyzed microbial compositions of intratumor bacteria in PC to determine the influence of the microbiome on metastatic growth. Using large-scale RNA-sequencing data and corresponding clinical data, we correlated the abundance of microbes to immune pathways and PC risk factors, identifying specific microbes that either significantly deter or contribute to cancer aggressiveness. Interestingly, most of the microbes we found appeared to play anti-tumor roles in PC. Since these anti-tumor microbes were overrepresented in tumor samples, we believe that microbes thrive in the tumor microenvironment, outcompete cancer cells, and directly mitigate tumor growth by recruiting immune cells. These include Listeria monocytogenes, Methylobacterium radiotolerans JCM 2831, Xanthomonas albilineans GPE PC73, and Bradyrhizobium japonicum, which are negatively correlated with Gleason score, Tumor-Node-Metastasis (TNM) stage, prostate-specific antigen (PSA) level, and Androgen Receptor (AR) expression, respectively. We also identified microbes that contribute to tumor growth and are positively correlated with genomic alterations, dysregulated immune-associated (IA) genes, and prostate cancer stem cells (PCSC) genes.

[Source](https://doi.org/10.3390/cancers12092524)

10.3390/cancers12092524

---

## Experimental (co)evolution in a multi-species microbial community results in local maladaptation

[Source](https://doi.org/10.1111/ele.13599)

---

## Mothers of Preterm Infants Have Individualized Breast Milk Microbiota that Changes Temporally Based on Maternal Characteristics.
 05.09.2020, _MR Asbury, J Butcher, JK Copeland, S Unger, N Bando, EM Comelli, V Forte, A Kiss, L LeMay-Nedjelski, PM Sherman, A Stintzi, C Tomlinson, PW Wang, DL O'Connor_


Mother's milk contains complex microbial communities thought to be important for colonizing a preterm infant's gastrointestinal tract. However, little is known about the microbiota in the preterm mother's milk and factors influencing its composition. We characterized the temporal dynamics of microbial communities in 490 breast milk samples from 86 mothers of preterm infants (born &lt;1,250g) over the first 8 weeks postpartum. Highly individualized microbial communities were identified in each mother's milk that changed temporally with notable alterations in predicted microbial functions. However, pre-pregnancy BMI, delivery mode, and antibiotics were associated with changes in these microbial dynamics. Individual classes of antibiotics and their duration of exposure during prenatal and postpartum periods showed unique relationships with microbial taxa abundance and diversity in mother's milk. These results highlight the temporal complexity of the preterm mother's milk microbiota and its relationship with maternal characteristics as well as the importance of discussing antibiotic stewardship for mothers.

[Source](https://doi.org/10.1016/j.chom.2020.08.001)

10.1016/j.chom.2020.08.001

---

## The meconium microbiota shares more features with the amniotic fluid microbiota than the maternal fecal and vaginal microbiota

[Source](https://doi.org/10.1080/19490976.2020.1794266)

---

## Stratum Corneum Lipidomics Analysis Reveals Altered Ceramide Profile in Atopic Dermatitis Patients Across Body Sites With Correlated Changes in Skin Microbiome.
 04.09.2020, _H Emmert, H Baurecht, F Thielking, D Stölzl, E Rodriguez, I Harder, E Proksch, S Weidinger_


Atopic dermatitis (AD) is driven by the interplay between a dysfunctional epidermal barrier and a skewed cutaneous immune dysregulation. As part of the complex skin barrier dysfunction abnormalities in lipid organization and microbiome composition have been described. We set out to systematically investigate the composition of the stratum corneum lipidome, skin microbiome, and skin physiology parameters at three different body sites in patients with AD and healthy volunteers.

[Source](https://doi.org/10.1111/exd.14185)

10.1111/exd.14185

---

## Diverse vaginal microbiome was associated with pro-inflammatory vaginal milieu among pregnant women in Uganda

[Source](https://doi.org/10.1016/j.humic.2020.100076)

---

## Diet Diurnally Regulates Small Intestinal Microbiome-Epithelial-Immune Homeostasis and Enteritis.
 05.09.2020, _T Tuganbaev, U Mor, S Bashiardes, T Liwinski, SP Nobs, A Leshem, M Dori-Bachash, CA Thaiss, EY Pinker, K Ratiner, L Adlung, S Federici, C Kleimeyer, C Moresi, T Yamada, Y Cohen, X Zhang, H Massalha, E Massasa, Y Kuperman, PA Koni, A Harmelin, N Gao, S Itzkovitz, K Honda, H Shapiro, E Elinav_


Throughout a 24-h period, the small intestine (SI) is exposed to diurnally varying food- and microbiome-derived antigenic burdens but maintains a strict immune homeostasis, which when perturbed in genetically susceptible individuals, may lead to Crohn disease. Herein, we demonstrate that dietary content and rhythmicity regulate the diurnally shifting SI epithelial cell (SIEC) transcriptional landscape through modulation of the SI microbiome. We exemplify this concept with SIEC major histocompatibility complex (MHC) class II, which is diurnally modulated by distinct mucosal-adherent SI commensals, while supporting downstream diurnal activity of intra-epithelial IL-10+ lymphocytes regulating the SI barrier function. Disruption of this diurnally regulated diet-microbiome-MHC class II-IL-10-epithelial barrier axis by circadian clock disarrangement, alterations in feeding time or content, or epithelial-specific MHC class II depletion leads to an extensive microbial product influx, driving Crohn-like enteritis. Collectively, we highlight nutritional features that modulate SI microbiome, immunity, and barrier function and identify dietary, epithelial, and immune checkpoints along this axis to be potentially exploitable in future Crohn disease interventions.

[Source](https://doi.org/10.1016/j.cell.2020.08.027)

10.1016/j.cell.2020.08.027

---

## Microbiota-modulated CART+ enteric neurons autonomously regulate blood glucose.
 02.09.2020, _PA Muller, F Matheis, M Schneeberger, Z Kerner, V Jové, D Mucida_


The gut microbiota affects tissue physiology, metabolism, and function of both the immune and nervous systems. We found that intrinsic enteric-associated neurons (iEAN) in mice are functionally adapted to the intestinal segment they occupy; ileal and colonic neurons are more responsive to microbial colonization than duodenal neurons. Specifically, a microbially-responsive subset of viscerofugal CART+ neurons, enriched in the ileum and colon, modulated feeding and glucose metabolism. These CART+ neurons send axons to the prevertebral ganglia and are poly-synaptically connected to the liver and pancreas. Microbiota depletion led to NLRP6- and Caspase 11-dependent loss of CART+ neurons, and impaired glucose regulation. Hence, iEAN subsets appear to be capable of regulating blood glucose levels independently from the central nervous system.

[Source](https://doi.org/10.1126/science.abd6176.)

10.1126/science.abd6176

---

## Succinate Produced by Intestinal Microbes Promotes Specification of Tuft Cells to Suppress Ileal Inflammation.
 25.08.2020, _A Banerjee, CA Herring, B Chen, H Kim, AJ Simmons, AN Southard-Smith, MM Allaman, JR White, MC Macedonia, ET Mckinley, MA Ramirez Solano, EA Scoville, Q Liu, KT Wilson, RJ Coffey, MK Washington, JA Goettel, KS Lau_


Countries endemic for parasitic infestations have a lower incidence of Crohn's disease (CD) than non-endemic countries, and there have been anecdotal reports of the beneficial effects of helminths in CD patients. Tuft cells in the small intestine sense and direct the immune response against eukaryotic parasites. We investigated the activities of tuft cells in patients with CD and mouse models of intestinal inflammation.

[Source](https://doi.org/10.1053/j.gastro.2020.08.029)

10.1053/j.gastro.2020.08.029

---

## Gut microbiome improves postoperative cognitive function by decreasing permeability of the blood-brain barrier in aged mice.
 08.09.2020, _J Wen, Y Ding, L Wang, Y Xiao_


Increased permeability of blood-brain barrier (BBB) is a major pathophysiological mechanism of postoperative cognitive dysfunction (POCD) in the elderly. The reduced beneficial gut microbiome due to aging results in a decline in the production of sodium butyrate (NaB), which might enhance the BBB permeability. The present study investigated whether gut microbiome or NaB could improve the postoperative cognitive function in aged and gut dysbiosis mouse model.

[Source](https://doi.org/10.1016/j.brainresbull.2020.08.017)

10.1016/j.brainresbull.2020.08.017

---

## Indolepropionic Acid, a Metabolite of the Microbiome, Has Cytostatic Properties in Breast Cancer by Activating AHR and PXR Receptors and Inducing Oxidative Stress.
 28.08.2020, _Z Sári, E Mikó, T Kovács, L Jankó, T Csonka, G Lente, É Sebő, J Tóth, D Tóth, P Árkosy, A Boratkó, G Ujlaki, M Török, I Kovács, J Szabó, B Kiss, G Méhes, JJ Goedert, P Bai_


Oncobiotic transformation of the gut microbiome may contribute to the risk of breast cancer. Recent studies have provided evidence that the microbiome secretes cytostatic metabolites that inhibit the proliferation, movement, and metastasis formation of cancer cells. In this study, we show that indolepropionic acid (IPA), a bacterial tryptophan metabolite, has cytostatic properties. IPA selectively targeted breast cancer cells, but it had no effects on non-transformed, primary fibroblasts. In cell-based and animal experiments, we showed that IPA supplementation reduced the proportions of cancer stem cells and the proliferation, movement, and metastasis formation of cancer cells. These were achieved through inhibiting epithelial-to-mesenchymal transition, inducing oxidative and nitrosative stress, and boosting antitumor immune response. Increased oxidative/nitrosative stress was due to the IPA-mediated downregulation of nuclear factor erythroid 2-related factor 2 (NRF2), upregulation of inducible nitric oxide synthase (iNOS), and enhanced mitochondrial reactive species production. Increased oxidative/nitrosative stress led to cytostasis and reductions in cancer cell stem-ness. IPA exerted its effects through aryl hydrocarbon receptor (AHR) and pregnane X receptor (PXR) receptors. A higher expression of PXR and AHR supported better survival in human breast cancer patients, highlighting the importance of IPA-elicited pathways in cytostasis in breast cancer. Furthermore, AHR activation and PXR expression related inversely to cancer cell proliferation level and to the stage and grade of the tumor. The fecal microbiome's capacity for IPA biosynthesis was suppressed in women newly diagnosed with breast cancer, especially with stage 0. Bacterial indole biosynthesis showed correlation with lymphocyte infiltration to tumors in humans. Taken together, we found that IPA is a cytostatic bacterial metabolite, the production of which is suppressed in human breast cancer. Bacterial metabolites, among them, IPA, have a pivotal role in regulating the progression but not the initiation of the disease.

[Source](https://doi.org/10.3390/cancers12092411)

10.3390/cancers12092411

---

## Angiogenin maintains gut microbe homeostasis by balancing α-Proteobacteria and Lachnospiraceae.
 26.08.2020, _D Sun, R Bai, W Zhou, Z Yao, Y Liu, S Tang, X Ge, L Luo, C Luo, GF Hu, J Sheng, Z Xu_


Antimicrobial peptides (AMPs) play essential roles in maintaining gut health and are associated with IBD. This study is to elucidate the effect of angiogenin (ANG), an intestine-secreted AMP, on gut microbiota and its relevance with IBD.

[Source](https://doi.org/10.1136/gutjnl-2019-320135)

10.1136/gutjnl-2019-320135

---

## Gut Microbiota-Associated Activation of TLR5 Induces Apolipoprotein A1 Production in the Liver.
 21.08.2020, _JHC Yiu, KS Chan, J Cheung, J Li, Y Liu, Y Wang, WLW Fung, J Cai, SW Cheung, B Dorweiler, EYF Wan, P Tso, A Xu, CWH Woo_


Rationale: Dysbiosis of gut microbiota plays an important role in cardiovascular diseases but the molecular mechanisms are complex. An association between gut microbiome and the variance in HDL-cholesterol (HDL-C) level was suggested in a human study. Besides, dietary fat was shown to increase both HDL-C and LDL-C levels. We speculate that certain types of gut bacteria responding to dietary fat may help to regulate HDL-C level and potentially affect atherosclerotic development. Objective: We aimed to investigate whether and how high-fat diet (HFD)-associated gut microbiota regulated HDL level. Methods and Results: We found that HFD increased gut flagellated bacteria population in mice. The increase in HDL-C levels was adopted by mice receiving fecal microbiome transplantion from HFD-fed mouse donors. HFD led to increased hepatic but not circulating flagellin, and deletion of TLR5, a receptor sensing flagellin, suppressed HFD-stimulated HDL-C and apolipoprotein A1 (ApoA1) levels. Overexpression of TLR5 in the liver of TLR5-knockout mice was able to partially restore the production of ApoA1 and HDL-C levels. Mechanistically, TLR5 activation by flagellin in primary hepatocytes stimulated ApoA1 production through the transcriptional activation responding to the binding of NFkappaB on Apoa1 promoter region. Furthermore, oral supplementation of flagellin was able to stimulate hepatic ApoA1 production and HDL-C level, and decrease atherosclerotic lesion size in apolipoprotein E-deficient (Apoe-/-) mice without triggering hepatic and systemic inflammation. The stimulation of ApoA1 production was also seen in human ApoA1 transgenic mice treated with oral flagellin. Conclusions: Our finding suggests that commensal flagellated bacteria in gut can facilitate ApoA1 and HDL productions in liver through activation of TLR5 in hepatocytes. Hepatic TLR5 may be a potential drug target to increase ApoA1.

[Source](https://doi.org/10.1161/CIRCRESAHA.120.317362)

10.1161/CIRCRESAHA.120.317362

---

## Cross-generational effects of climate change on the microbiome of a photosynthetic sponge.
 01.09.2020, _HM Luter, M Andersen, E Versteegen, P Laffy, S Uthicke, JJ Bell, NS Webster_


Coral reefs are facing increasing pressure from rising seawater temperatures and ocean acidification. Sponges have been proposed as possible winners in the face of climate change, however little is known about the mechanisms underpinning their predicted tolerance. Here we assessed whether microbiome mediated cross-generational acclimatisation could enable the photosynthetic sponge Carteriospongia foliascens to survive under future climate scenarios. To achieve this, we first established the potential for vertical (cross-generational) transmission of symbionts. Sixty-four amplicon sequence variants accounting for &gt;90% of the total C. foliascens microbial community were present across adult, larval and juvenile life stages, showing that a large proportion of the microbiome is vertically acquired and maintained. When C. foliascens were exposed to climate scenarios projected for 2050 and 2100, the host remained visibly unaffected (i.e. no necrosis/bleaching) and the overall microbiome was not significantly different among treatments in adult tissue, the respective larvae or recruits transplanted amongst climate treatments. However, indicator species analysis revealed that parental exposure to future climate scenarios altered the presence and abundance of a small suite of microbial taxa in the recruits, thereby revealing the potential for microbiome mediated cross-generational acclimatisation through both symbiont shuffling and symbiont switching within a vertically acquired microbiome. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1111/1462-2920.15222)

10.1111/1462-2920.15222

---

## Symbionts shape host innate immunity in honeybees.
 26.08.2020, _RD Horak, SP Leonard, NA Moran_


The gut microbiome plays a critical role in the health of many animals. Honeybees are no exception, as they host a core microbiome that affects their nutrition and immune function. However, the relationship between the honeybee immune system and its gut symbionts is poorly understood. Here, we explore how the beneficial symbiont Snodgrassella alvi affects honeybee immune gene expression. We show that both live and heat-killed S. alvi protect honeybees from the opportunistic pathogen Serratia marcescens and lead to the expression of host antimicrobial peptides. Honeybee immune genes respond differently to live S. alvi compared to heat-killed S. alvi, the latter causing a more extensive immune expression response. We show a preference for Toll pathway upregulation over the Imd pathway in the presence of both live and heat-killed S. alvi. Finally, we find that live S. alvi aids in clearance of S. marcescens from the honeybee gut, supporting a potential role for the symbiont in colonization resistance. Our results show that colonization by the beneficial symbiont S. alvi triggers a replicable honeybee immune response. These responses may benefit the host and the symbiont, by helping to regulate gut microbial members and preventing overgrowth or invasion by opportunists.

[Source](http://doi.org/10.1098/rspb.2020.1184)

10.1098/rspb.2020.1184

---

## Parental Microbiota Modulates Offspring Development, Body Mass and Fecundity in a Polyphagous Fruit Fly

[Source](https://doi.org/10.3390/microorganisms8091289)

---

## Fast and Facile Biodegradation of Polystyrene by the Gut Microbial Flora of Plesiophthalmus davidis Larvae.
 02.09.2020, _S Woo, I Song, HJ Cha_


Polystyrene (PS), which accounts for a significant fraction of plastic wastes, is difficult to biodegrade due to its unique molecular structure. Therefore, biodegradation and chemical modification of PS are limited. In this study, we report PS biodegradation by the larvae of the darkling beetle Plesiophthalmus davidis (Coleoptera: Tenebrionidae). In 14 days, P. davidis ingested 34.27 ± 4.04 mg of Styrofoam (PS foam) per larva and survived by feeding only on Styrofoam. Fourier transform infrared spectroscopy confirmed that the ingested Styrofoam was oxidized. Gel permeation chromatography analysis indicated the decrease in average molecular weight of the residual PS in the frass compared with the feed Styrofoam. When the extracted gut flora was cultured for 20 days with PS films, biofilm and cavities were observed by scanning electron microscopy and atomic force microscopy. X-ray photoelectron spectroscopy (XPS) studies revealed that C-O bonding was introduced into the biodegraded PS film. Serratia sp. strain WSW (KCTC 82146), which was isolated from the gut flora, also formed a biofilm and cavities on the PS film in 20 days, but its degradation was less prominent than the gut flora. XPS confirmed that C-O and C=O bonds were introduced into the biodegraded PS film by Serratia sp. WSW. Microbial community analysis revealed that Serratia was in the gut flora in significant amounts and increased sixfold when the larvae were fed Styrofoam for 2 weeks. This suggests that P. davidis larvae and its gut bacteria could be used to chemically modify and rapidly degrade PS.IMPORTANCE PS is widely produced in the modern world, but it is robust against biodegradation. A few studies reported the biodegradation of PS, but most of them merely observed its weight loss; fewer were able to find its chemical modifications, which are rather direct evidence of biodegradation, by using limited organisms. Therefore, it is required to find an effective way to decompose PS using various kinds of organisms. Herein, we discovered a new PS-degrading insect species and bacterial strain, and we found that the genus that includes the PS-degrading bacterial strain occurs in significant amounts in the larval gut flora, and the proportion of this genus increased as the larvae were fed Styrofoam. Our research offers a wider selection of PS-degrading insects and the possibility of using a certain mixture of bacteria that resemble the gut flora of a PS-degrading insect to biodegrade PS, and thus could contribute to solving the global plastic crisis.

[Source](https://doi.org/10.1128/AEM.01361-20)

10.1128/AEM.01361-20

---

## Gut microbiota protects honey bees (Apis mellifera L.) against polystyrene microplastics exposure risks

[Source](https://doi.org/10.1016/j.jhazmat.2020.123828)

---

## Characteristics of microbial community of soil subjected to industrial production of antibiotics.
 09.09.2020, _M Borčinová, A Pitkina, H Marešová, V Štěpánek, A Palyzová, P Kyslík_


Ecosystems worldwide are exposed to pollutants connected to the industrial production of pharmaceuticals. The objective of this study was to study the composition and characteristics of the soil microbial communities that had been exposed to long-term selection pressure caused by the industrial production of penicillin G. Soil samples from four sites among the penicillin G production plant were analysed using 16S rRNA profiling via Illumina MiSeq platform and were compared with the control samples from four sites outside the plant. Total metagenomic DNA from the impacted soil was also used for the preparation of E. coli T1R-based fosmid library which was consequently qualitatively tested for the presence of penicillin G acylase (PGA)-encoding genes using the method of sequence homology. Analyses of alpha diversity revealed that the long-term antibiotic presence in the soil significantly increased the microbial diversity and richness in terms of Shannon diversity index (p = 0.002) and Chao estimates (p = 0.004). Principal component analysis showed that the two types of communities (on-site and control) could be separated at the phylum, class and genus level. The on-site soil was enriched in Betaproteobacteria, Deltaproteobacteria, Gemmatimonadetes, Acidobacteria and Planctomycetia, while a significant decrease in Actinobacteria was observed. Metagenomic fosmid library revealed high hit rates in identifying PGAs (14 different genes identified) and confirmed the biotechnological potential of soils impacted by anthropogenic activity. This study offers new insights into the changes in microbial communities of soils exposed to anthropogenic activity as well as indicates that those soils may represent a hotspot for biotechnologically interesting targets.

[Source](https://doi.org/10.1007/s12223-020-00819-z)

10.1007/s12223-020-00819-z

---

## Host selection shapes crop microbiome assembly and network complexity.
 27.08.2020, _C Xiong, Y Zhu, J Wang, B Singh, L Han, J Shen, P Li, G Wang, C Wu, A Ge, L Zhang, J He_


Plant microbiomes are essential to host health and productivity but the ecological processes that govern crop microbiome assembly are not fully known. Here we examined bacterial communities across 684 samples from soils (rhizosphere and bulk soil) and multiple compartment niches (rhizoplane, root endosphere, phylloplane, and leaf endosphere) in maize (Zea mays)-wheat (Triticum aestivum)/barley (Hordeum vulgare) rotation system under different fertilization practices at two contrasting sites. Our results demonstrate that microbiome assembly is shaped predominantly by compartment niche and host species rather than by site or fertilization practice. From soils to epiphytes to endophytes, host selection pressure sequentially increased and bacterial diversity and network complexity consequently reduced, with the strongest host effect in leaf endosphere. Source tracking indicates that crop microbiome is mainly derived from soils and gradually enriched and filtered at different plant compartment niches. Moreover, crop microbiomes were dominated by a few dominant taxa (~0.5% of bacterial phylotypes), with Bacilli identified as the important biomarker taxa for wheat and barley and Methylobacteriaceae for maize. Our work provides comprehensive empirical evidence on host selection, potential sources and enrichment processes for crop microbiome assembly, and has important implications for future crop management and manipulation of crop microbiome for sustainable agriculture.

[Source](https://doi.org/10.1111/nph.16890)

10.1111/nph.16890

---

## Microbiome of root vegetables-a source of gluten-degrading bacteria.
 02.09.2020, _V Kõiv, K Adamberg, S Adamberg, I Sumeri, S Kasvandik, V Kisand, Ü Maiväli, T Tenson_


Gluten is a cereal protein that is incompletely digested by human proteolytic enzymes that create immunogenic peptides that accumulate in the gastrointestinal tract (GIT). Although both environmental and human bacteria have been shown to expedite gluten hydrolysis, gluten intolerance is a growing concern. Here we hypothesize that together with food, we acquire environmental bacteria that could impact our GIT with gluten-degrading bacteria. Using in vitro gastrointestinal simulation conditions, we evaluated the capacity of endophytic bacteria that inhabit root vegetables, potato (Solanum tuberosum), carrot (Daucus sativus), beet (Beta vulgaris), and topinambur (Jerusalem artichoke) (Helianthus tuberosus), to resist these conditions and degrade gluten. By 16S rDNA sequencing, we discovered that bacteria from the families Enterobacteriaceae, Bacillaceae, and Clostridiaceae most effectively multiply in conditions similar to the human GIT (microoxic conditions, 37 °C) while utilizing vegetable material and gluten as nutrients. Additionally, we used stomach simulation (1 h, pH 3) and intestinal simulation (1 h, bile salts 0.4%) treatments. The bacteria that survived this treatment retained the ability to degrade gluten epitopes but at lower levels. Four bacterial strains belonging to species Bacillus pumilus, Clostridium subterminale, and Clostridium sporogenes isolated from vegetable roots produced proteases with postproline cleaving activity that successfully neutralized the toxic immunogenic epitopes. KEY POINTS: • Bacteria from root vegetables can degrade gluten. • Some of these bacteria can resist conditions mimicking gastrointestinal tract.

[Source](https://doi.org/10.1007/s00253-020-10852-0)

10.1007/s00253-020-10852-0

---

## Environmental stability impacts the differential sensitivity of marine microbiomes to increases in temperature and acidity.
 05.09.2020, _Z Wang, D Tsementzi, TC Williams, DL Juarez, SK Blinebry, NS Garcia, BK Sienkiewicz, KT Konstantinidis, ZI Johnson, DE Hunt_


Ambient conditions shape microbiome responses to both short- and long-duration environment changes through processes including physiological acclimation, compositional shifts, and evolution. Thus, we predict that microbial communities inhabiting locations with larger diel, episodic, and annual variability in temperature and pH should be less sensitive to shifts in these climate-change factors. To test this hypothesis, we compared responses of surface ocean microbes from more variable (nearshore) and more constant (offshore) sites to short-term factorial warming (+3 °C) and/or acidification (pH -0.3). In all cases, warming alone significantly altered microbial community composition, while acidification had a minor influence. Compared with nearshore microbes, warmed offshore microbiomes exhibited larger changes in community composition, phylotype abundances, respiration rates, and metatranscriptomes, suggesting increased sensitivity of microbes from the less-variable environment. Moreover, while warming increased respiration rates, offshore metatranscriptomes yielded evidence of thermal stress responses in protein synthesis, heat shock proteins, and regulation. Future oceans with warmer waters may enhance overall metabolic and biogeochemical rates, but they will host altered microbial communities, especially in relatively thermally stable regions of the oceans.

[Source](https://doi.org/10.1038/s41396-020-00748-2)

10.1038/s41396-020-00748-2

---

## Insights into the Vertical Stratification of Microbial Ecological Roles across the Deepest Seawater Column on Earth.
 01.09.2020, _CX Xue, J Liu, DJ Lea-Smith, G Rowley, H Lin, Y Zheng, XY Zhu, J Liang, W Ahmad, JD Todd, XH Zhang_


The Earth's oceans are a huge body of water with physicochemical properties and microbial community profiles that change with depth, which in turn influences their biogeochemical cycling potential. The differences between microbial communities and their functional potential in surface to hadopelagic water samples are only beginning to be explored. Here, we used metagenomics to investigate the microbial communities and their potential to drive biogeochemical cycling in seven different water layers down the vertical profile of the Challenger Deep (0-10,500 m) in the Mariana Trench, the deepest natural point in the Earth's oceans. We recovered 726 metagenome-assembled genomes (MAGs) affiliated to 27 phyla. Overall, biodiversity increased in line with increased depth. In addition, the genome size of MAGs at ≥4000 m layers was slightly larger compared to those at 0-2000 m. As expected, surface waters were the main source of primary production, predominantly from Cyanobacteria. Intriguingly, microbes conducting an unusual form of nitrogen metabolism were identified in the deepest waters (&gt;10,000 m), as demonstrated by an enrichment of genes encoding proteins involved in dissimilatory nitrate to ammonia conversion (DNRA), nitrogen fixation and urea transport. These likely facilitate the survival of ammonia-oxidizing archaea α lineage, which are typically present in environments with a high ammonia concentration. In addition, the microbial potential for oxidative phosphorylation and the glyoxylate shunt was enhanced in &gt;10,000 m waters. This study provides novel insights into how microbial communities and their genetic potential for biogeochemical cycling differs through the Challenger deep water column, and into the unique adaptive lifestyle of microbes in the Earth's deepest seawater.

[Source](https://doi.org/10.3390/microorganisms8091309)

10.3390/microorganisms8091309

---

## The microbial community of a biofilm lining the wall of a pristine cave in Western New Guinea.
 07.09.2020, _P Turrini, M Tescari, D Visaggio, M Pirolo, GA Lugli, M Ventura, E Frangipani, P Visca_


Caves are extreme environments inhabited by microbial communities adapted to thrive oligotrophic conditions. Cave microbes are organised in complex ecological networks and have developed survival strategies involving the production and release of a large variety of secondary metabolites, including antibiotic-like compounds. In this study, the structure and the metabolic features of a biofilm-like microbial community lining the walls of a pristine karst cavity (the Yumugi river cave) located in a remote region of the Western New Guinea were investigated. 16S rRNA and shotgun sequence analyses highlighted the prevalence of chemoorganotrophic phyla (Proteobacteria, Actinobacteria, Firmicutes and Acidobacteria), consistent with metabolic predictions inferred from the cave metagenome analysis. Few clinically relevant antimicrobial resistance genes were detected. A culture-based approach allowed the isolation of some heterotrophic members of the bacterial community, and antimicrobial susceptibility testing revealed an overall high level of resistance to different antimicrobials classes. Isolates presumptively representing new uncharacterized members of genus Pseudomonas displayed interesting antibiotic properties against Gram-positive indicator strains. Our work supports the hypothesis that caves represent a reservoir for new bacterial species and drug discovery research.

[Source](https://doi.org/10.1016/j.micres.2020.126584)

10.1016/j.micres.2020.126584

---

## Airborne bacteria in outdoor air and air of mechanically ventilated buildings at city-scale in Hong Kong across seasons

[Source](https://doi.org/10.1021/acs.est.9b07623)

---

## Phages and their potential to modulate the microbiome and immunity.
 09.09.2020, _S Federici, SP Nobs, E Elinav_


Bacteriophages (hence termed phages) are viruses that target bacteria and have long been considered as potential future treatments against antibiotic-resistant bacterial infection. However, the molecular nature of phage interactions with bacteria and the human host has remained elusive for decades, limiting their therapeutic application. While many phages and their functional repertoires remain unknown, the advent of next-generation sequencing has increasingly enabled researchers to decode new lytic and lysogenic mechanisms by which they attack and destroy bacteria. Furthermore, the last decade has witnessed a renewed interest in the utilization of phages as therapeutic vectors and as a means of targeting pathogenic or commensal bacteria or inducing immunomodulation. Importantly, the narrow host range, immense antibacterial repertoire, and ease of manipulating phages may potentially allow for their use as targeted modulators of pathogenic, commensal and pathobiont members of the microbiome, thereby impacting mammalian physiology and immunity along mucosal surfaces in health and in microbiome-associated diseases. In this review, we aim to highlight recent advances in phage biology and how a mechanistic understanding of phage-bacteria-host interactions may facilitate the development of novel phage-based therapeutics. We provide an overview of the challenges of the therapeutic use of phages and how these could be addressed for future use of phages as specific modulators of the human microbiome in a variety of infectious and noncommunicable human diseases.

[Source](https://doi.org/10.1038/s41423-020-00532-4)

10.1038/s41423-020-00532-4

---

## Generalized estimating equation modeling on correlated microbiome sequencing data with longitudinal measures.
 08.09.2020, _B Chen, W Xu_


Existing models for assessing microbiome sequencing such as operational taxonomic units (OTUs) can only test predictors' effects on OTUs. There is limited work on how to estimate the correlations between multiple OTUs and incorporate such relationship into models to evaluate longitudinal OTU measures. We propose a novel approach to estimate OTU correlations based on their taxonomic structure, and apply such correlation structure in Generalized Estimating Equations (GEE) models to estimate both predictors' effects and OTU correlations. We develop a two-part Microbiome Taxonomic Longitudinal Correlation (MTLC) model for multivariate zero-inflated OTU outcomes based on the GEE framework. In addition, longitudinal and other types of repeated OTU measures are integrated in the MTLC model. Extensive simulations have been conducted to evaluate the performance of the MTLC method. Compared with the existing methods, the MTLC method shows robust and consistent estimation, and improved statistical power for testing predictors' effects. Lastly we demonstrate our proposed method by implementing it into a real human microbiome study to evaluate the obesity on twins.

[Source](https://doi.org/10.1371/journal.pcbi.1008108)

10.1371/journal.pcbi.1008108

---

## The Nubeam reference-free approach to analyze metagenomic sequencing reads.
 04.09.2020, _H Dai, Y Guan_


We present Nubeam (nucleotide be amatrix) as a novel reference-free approach to analyze short sequencing reads. Nubeam represents nucleotides by matrices, transforms a read into a product of matrices, and assigns numbers to reads based on the product matrix. Nubeam capitalizes on the noncommutative property of matrix multiplication, such that different reads are assigned different numbers and similar reads similar numbers. A sample, which is a collection of reads, becomes a collection of numbers that form an empirical distribution. We demonstrate that the genetic difference between samples can be quantified by the distance between empirical distributions. Nubeam includes the k-mer method as a special case, but unlike the k-mer method, it is convenient for Nubeam to account for GC bias and nucleotide quality. As a reference-free approach, Nubeam avoids reference bias, and mapping bias and can work with organisms without reference genomes. Thus, Nubeam is ideal to analyze data sets from metagenomics whole genome shotgun (WGS) sequencing, where the amount of unmapped reads is substantial. When applied to a WGS sequencing data set to quantify distances between metagenomics samples from various human body habitats, Nubeam recapitulates findings made by mapping-based methods and sheds light on contributions of unmapped reads. Nubeam is also useful in analyzing 16S rRNA sequencing data, which is a more prevalent type of data set in metagenomics studies. In our analysis, Nubeam recapitulated the findings that natural microbiota in mouse gut are resilient under challenges, and Nubeam detected differences in vaginal microbiota between cases of polycystic ovary syndrome and healthy controls.

[Source](https://doi.org/10.1101/gr.261750.120)

10.1101/gr.261750.120

---

## The archives are half-empty: an assessment of the availability of microbial community sequencing data.
 04.09.2020, _SD Jurburg, M Konzack, N Eisenhauer, A Heintz-Buschart_


As DNA sequencing has become more popular, the public genetic repositories where sequences are archived have experienced explosive growth. These repositories now hold invaluable collections of sequences, e.g., for microbial ecology, but whether these data are reusable has not been evaluated. We assessed the availability and state of 16S rRNA gene amplicon sequences archived in public genetic repositories (SRA, EBI, and DDJ). We screened 26,927 publications in 17 microbiology journals, identifying 2015 16S rRNA gene sequencing studies. Of these, 7.2% had not made their data public at the time of analysis. Among a subset of 635 studies sequencing the same gene region, 40.3% contained data which was not available or not reusable, and an additional 25.5% contained faults in data formatting or data labeling, creating obstacles for data reuse. Our study reveals gaps in data availability, identifies major contributors to data loss, and offers suggestions for improving data archiving practices.

[Source](https://doi.org/10.1038/s42003-020-01204-9)

10.1038/s42003-020-01204-9

---

## An effective culturomics approach to study the gut microbiota of mammals

[Source](https://doi.org/10.1016/j.resmic.2020.09.001)

---

## Activity-based cell sorting reveals responses of uncultured archaea and bacteria to substrate amendment.
 05.09.2020, _NJ Reichart, ZJ Jay, V Krukenberg, AE Parker, RL Spietz, R Hatzenpichler_


Metagenomic studies have revolutionized our understanding of the metabolic potential of uncultured microorganisms in various ecosystems. However, many of these genomic predictions have yet to be experimentally tested, and the functional expression of genomic potential often remains unaddressed. In order to obtain a more thorough understanding of cell physiology, novel techniques capable of testing microbial metabolism under close to in situ conditions must be developed. Here, we provide a benchmark study to demonstrate that bioorthogonal non-canonical amino acid tagging (BONCAT) in combination with fluorescence-activated cell sorting (FACS) and 16S rRNA gene sequencing can be used to identify anabolically active members of a microbial community incubated in the presence of various growth substrates or under changing physicochemical conditions. We applied this approach to a hot spring sediment microbiome from Yellowstone National Park (Wyoming, USA) and identified several microbes that changed their activity levels in response to substrate addition, including uncultured members of the phyla Thaumarchaeota, Acidobacteria, and Fervidibacteria. Because shifts in activity in response to substrate amendment or headspace changes are indicative of microbial preferences for particular growth conditions, results from this and future BONCAT-FACS studies could inform the development of cultivation media to specifically enrich uncultured microbes. Most importantly, BONCAT-FACS is capable of providing information on the physiology of uncultured organisms at as close to in situ conditions as experimentally possible.

[Source](https://doi.org/10.1038/s41396-020-00749-1)

10.1038/s41396-020-00749-1

---

## An Ingestible Self-Polymerizing System for Targeted Sampling of Gut Microbiota and Biomarkers.
 27.08.2020, _L Chen, L Gruzinskyte, SL Jørgensen, A Boisen, SK Srivastava_


A proof-of-concept for the fabrication of a self-polymerizing system for sampling of gut microbiome in the upper gastrointestinal (GI) tract is presented. An orally ingestible microdevice is loaded with the self-polymerizing reaction mixture to entrap gut microbiota and biomarkers. This polymerization reaction is activated in the aqueous environment, like fluids in the intestinal lumen, and causes site-specific microsampling in the gastrointestinal tract. The sampled microbiota and protein biomarkers can be isolated and analyzed via high-throughput multiomic analyses. The study utilizes a hollow microdevice (Su-8, ca. 250 μm), loaded with an on-board reaction mixture (iron chloride, ascorbic acid, and poly(ethylene glycol) diacrylate monomers) for diacrylate polymerization in the gut of an animal model. An enteric-coated rat capsule was used to orally gavage these microdevices in a rat model, thereby, protecting the microdevices in the stomach (pH 2), but releasing them in the intestine (pH 6.6). Upon capsule disintegration, the microdevices were released in the presence of luminal fluids (in the small intestine region), where iron chloride reacts with ascorbic acid, to initiate poly(ethylene glycol) diacrylate polymerization via a free radical mechanism. Upon retrieval of the microdevices, gut microbiota was found to be entrapped in the polymerized hydrogel matrix, and genomic content was analyzed via 16s rRNA amplicon sequencing. Herein, the results show that the bacterial composition recovered from the microdevices closely resemble the bacterial composition of the gut microenvironment to which the microdevice is exposed. Further, histological assessment showed no signs of local tissue inflammation or toxicity. This study lays a strong foundation for the development of untethered, non-invasive microsampling technologies in the gut and advances our understanding of host-gut microbiome interactions, leading to a better understanding of their commensal behavior and associated GI disease progression in the near future.

[Source](https://doi.org/10.1021/acsnano.0c05426)

10.1021/acsnano.0c05426

---

## Aging and the Combined effects of ADRA2B and CB1 deletions on Affective Working Memory.
 23.03.2020, _B Fairfield, N Mammarella, L Fontanella, A Sarra, M D'Aurora, L Stuppia, V Gatta_


Many studies have found that memory for affective material is better than memory for neutral information and memory for positive material compared to negative material is better in older adults. Behavioral, neurophysiological as well as single polymorphism differences have been advanced to account for these effects. Here, we aimed to examine whether the combination of two polymorphisms (ADRA2B and CB1) in older adults influences active maintenance and manipulation of emotional information in aging working memory. We examined genotype data from 207 older adults (56 double deletion carriers, 116 single deletion carriers and 35 no deletion carriers) who performed a verbal operation span-like task with positive, negative and neutral words. We found that subjects carrying both ADRA2B and CB1 variants generally remembered a higher number of words. In addition, double carriers showed positivity effects while single carriers showed more general emotional enhancement effects, especially as strings lengthened. These findings are amongst the first to suggest a haplotype account of positivity effects in older adults' memory.

[Source](https://microbiomedigest.com/2020/09/10/september-10-2020/?like_comment=25380&_wpnonce=5c02214802)

10.1038/s41598-019-40108-5

---

## <span>Like</span>

[Source](https://microbiomedigest.com/2020/09/10/september-10-2020/?like_comment=25380&_wpnonce=5c02214802)

---

