## Gut microbiota drives colon cancer risk associated with diet: a comparative analysis of meat-based and pesco-vegetarian diets

[Source](https://doi.org/10.1186/s40168-024-01900-2)

---

## Decoding the diagnostic and therapeutic potential of microbiota using pan-body pan-disease microbiomics.
 28.09.2024, _GP Schmartz, J Rehner, MP Gund, V Keller, LG Molano, S Rupf, M Hannig, T Berger, E Flockerzi, B Seitz, S Fleser, S Schmitt-Grohé, S Kalefack, M Zemlin, M Kunz, F Götzinger, C Gevaerd, T Vogt, J Reichrath, L Diehl, A Hecksteden, T Meyer, C Herr, A Gurevich, D Krug, J Hegemann, K Bozhueyuek, TAM Gulder, C Fu, C Beemelmanns, JM Schattenberg, OV Kalinina, A Becker, M Unger, N Ludwig, M Seibert, ML Stein, NL Hanna, MC Martin, F Mahfoud, M Krawczyk, SL Becker, R Müller, R Bals, A Keller_


_Humans, Microbiota, Metagenome, Metagenomics, Bacteria, Feces, Male, Female, Multigene Family, Saliva, Adult_
The human microbiome emerges as a promising reservoir for diagnostic markers and therapeutics. Since host-associated microbiomes at various body sites differ and diseases do not occur in isolation, a comprehensive analysis strategy highlighting the full potential of microbiomes should include diverse specimen types and various diseases. To ensure robust data quality and comparability across specimen types and diseases, we employ standardized protocols to generate sequencing data from 1931 prospectively collected specimens, including from saliva, plaque, skin, throat, eye, and stool, with an average sequencing depth of 5.3 gigabases. Collected from 515 patients, these samples yield an average of 3.7 metagenomes per patient. Our results suggest significant microbial variations across diseases and specimen types, including unexpected anatomical sites. We identify 583 unexplored species-level genome bins (SGBs) of which 189 are significantly disease-associated. Of note, the existence of microbial resistance genes in one specimen was indicative of the same resistance genes in other specimens of the same patient. Annotated and previously undescribed SGBs collectively harbor 28,315 potential biosynthetic gene clusters (BGCs), with 1050 significant correlations to diseases. Our combinatorial approach identifies distinct SGBs and BGCs, emphasizing the value of pan-body pan-disease microbiomics as a source for diagnostic and therapeutic strategies.

[Source](https://doi.org/10.1038/s41467-024-52598-7)

10.1038/s41467-024-52598-7

---

## Deep longitudinal lower respiratory tract microbiome profiling reveals genome-resolved functional and evolutionary dynamics in critical illness.
 27.09.2024, _M Cheng, Y Xu, X Cui, X Wei, Y Chang, J Xu, C Lei, L Xue, Y Zheng, Z Wang, L Huang, M Zheng, H Luo, Y Leng, C Jiang_


_Humans, Critical Illness, Microbiota, Intensive Care Units, Metagenome, Metagenomics, Longitudinal Studies, Male, Female, Plasmids, Genome, Bacterial, Respiratory System, Aged, Middle Aged, Bacteria, Pneumonia, Evolution, Molecular_
The lower respiratory tract (LRT) microbiome impacts human health, especially among critically ill patients. However, comprehensive characterizations of the LRT microbiome remain challenging due to low microbial mass and host contamination. We develop a chelex100-based low-biomass microbial-enrichment method (CMEM) that enables deep metagenomic profiling of LRT samples to recover near-complete microbial genomes. We apply the method to 453 longitudinal LRT samples from 157 intensive care unit (ICU) patients in three geographically distant hospitals. We recover 120 high-quality metagenome-assembled genomes (MAGs) and associated plasmids without culturing. We detect divergent longitudinal microbiome dynamics and hospital-specific dominant opportunistic pathogens and resistomes in pneumonia patients. Diagnosed pneumonia and the ICU stay duration were associated with the abundance of specific antibiotic-resistance genes (ARGs). Moreover, CMEM can serve as a robust tool for genome-resolved analyses. MAG-based analyses reveal strain-specific resistome and virulome among opportunistic pathogen strains. Evolutionary analyses discover increased mobilome in prevailing opportunistic pathogens, highly conserved plasmids, and new recombination hotspots associated with conjugative elements and prophages. Integrative analysis with epidemiological data reveals frequent putative inter-patient strain transmissions in ICUs. In summary, we present a genome-resolved functional, transmission, and evolutionary landscape of the LRT microbiota in critically ill patients.

[Source](https://doi.org/10.1038/s41467-024-52713-8)

10.1038/s41467-024-52713-8

---

## Small Bowel Microbiome Profiles in Overweight and Obese Humans.
 26.09.2024, _L Li_



[Source](https://doi.org/10.14309/ajg.0000000000003071)

10.14309/ajg.0000000000003071

---

## Intratumoral microbiota as a novel prognostic indicator in bladder cancer.
 27.09.2024, _Y Zhang, H Lin, L Liang, S Jin, J Lv, Y Zhou, F Xu, F Liu, N Feng_


_Urinary Bladder Neoplasms, Humans, Prognosis, Microbiota, Tumor Microenvironment, In Situ Hybridization, Fluorescence, Biomarkers, Tumor, Female, Male, Middle Aged, Aged_
Microbes are important components of the tumor microenvironment and have a close relationship with tumors. However, there is still a lack of research on the intratumoral microbiota in bladder cancer and its impact on the tumor immune microenvironment. In this study, we used fluorescence in situ hybridization (FISH) and observed a substantial presence of microbiota in bladder cancer tissues, with greater abundance compared to that in normal bladder tissues. Based on the BIC database, we found that the microbiome of bladder cancer is highly diverse and its structure is significantly different from that of other tumors. To investigate the relationships among the intratumoral microbiota, tumor immunity, and prognosis in bladder cancer patients, we analyzed bladder cancer-specific differentially expressed immune- and antimicrobial-related genes from the ImmPort, TISIDB, and TCGA databases. We identified 11 hub genes and constructed a prognostic risk model. Further analysis revealed differences at the family and genus levels between distinct groups. Using LEfSe analysis, we identified six hub biomarkers and developed a novel microbial-based scoring system. The scoring system allows subgrouping of bladder cancer patients, with significant differences in prognosis, immune cell infiltration, tumor mutation burden, and immune checkpoints among different groups. Further FISH and immunofluorescence co-staining experiments initially verified that the specific distribution of microorganisms and M2 macrophages in bladder cancer may be closely related to the poor prognosis of patients. In conclusion, this study revealed the characteristics of the intratumoral microbiota in bladder cancer and identified potential prognostic targets for clinical application.

[Source](https://doi.org/10.1038/s41598-024-72918-7)

10.1038/s41598-024-72918-7

---

## Virulence Factors and Susceptibility to Ciprofloxacin, Vancomycin, Triclosan, and Chlorhexidine among Enterococci from Clinical Specimens, Food, and Wastewater.
 28.09.2024, _DB Gorski, J Vlainić, I Škrlec, S Novak, Ž Novosel, Z Biloglav, V Plečko, I Kosalec_


Enterococcus faecalis and E. faecium are opportunistic pathogens commonly found in the microbiota of humans and other animals as well as in the environment. This article presents the results of antimicrobial susceptibility testing using phenotypic methods (broth microdilution and standardized disk diffusion) on selected clinical, food, and wastewater isolates of E. faecalis and E. faecium. The isolates were divided into subgroups based on their sensitivity to the following antibiotics: vancomycin (VAN) and ciprofloxacin (CIP), and biocides triclosan (TCL) and chlorhexidine (CHX). The study also investigated in vitro virulence factors, including biofilm formation ability, cell surface hydrophobicity (CSH) and β-hemolysis, to explore aspects of pathogenesis. In our study, regardless of the isolation source, VAN-resistant (VAN-R) and CIP-resistant (CIP-R) E. faecalis and E. faecium were detected. The highest proportion of CIP-R strains was found among clinical isolates of E. faecalis and E. faecium, with clinical E. faecium also showing the highest proportion of VAN-R strains. But the highest proportion of VAN-R E. faecalis strains was found in wastewater samples. The highest TCL MIC90 values for E. faecalis were found in wastewater isolates, while for E. faecium, the highest TCL MIC90 values were observed in food isolates. The highest CHX MIC90 values for both E. faecalis and E. faecium were identified in clinical specimens. The results obtained for E. faecalis did not indicate differences in TCL MIC and CHX MIC values with respect to sensitivity to VAN and CIP. Higher CHX MIC50 and CHX MIC90 values were obtained for CIP-R and VAN-R E. faecium. Among the tested isolates, 97.75% of the E. faecalis isolates produced biofilm, while 72.22% of the E. faecium isolates did so as well. In biofilm-forming strength categories III and IV, statistically significantly higher proportions of CIP-susceptible (CIP-S) and VAN-susceptible (VAN-S) E. faecalis were determined. In category III, there is no statistically significant difference in E. faecium CIP sensitivity. In category IV, we had a significantly higher proportion of CIP-R strains. On the other hand, the association between the moderate or strong category of biofilm formation and E. faecium VAN susceptibility was not significant. E. faecalis isolated from wastewater had a CSH index (HI) ≥ 50%, categorizing them as "moderate", while all the other strains were categorized as "low" based on the CSH index. Among the E. faecalis isolates, cell surface hydrophobicity indices differed significantly across isolation sources. In contrast, E. faecium isolates showed similar hydrophobicity indices across isolation sources, with no significant difference found. Moreover, no correlation was found between the enterococcal cell surface hydrophobicity and biofilm formation in vitro. After anaerobic incubation, β-hemolytic activity was confirmed in 19.10% of the E. faecalis and 3.33% of the E. faecium strains.

[Source](https://doi.org/10.1038/d41573-024-00157-6)

10.3390/microorganisms12091808

---

## Identifying antimicrobials in the human microbiome

[Source](https://doi.org/10.1038/d41573-024-00157-6)

---

## <em>Akkermansia muciniphila </em>as a Potential Guardian against Oral Health Diseases: A Narrative Review

[Source](https://doi.org/10.3390/nu16183075)

---

## Intratumoral microbiota in colorectal cancer: focus on specific distribution and potential mechanisms.
 28.09.2024, _J Long, J Wang, C Xiao, F You, Y Jiang, X Li_


_Colorectal Neoplasms, Humans, Microbiota, Animals, Tumor Microenvironment_
Colorectal cancer (CRC) is one of the most prevalent and lethal malignant tumors globally, posing significant health risks and societal burdens. Recently, advancements in next-generation sequencing technology have identified CRC intratumoral microbiota, thereby opening up novel avenues for further research. This review synthesizes the current advancements in CRC intratumoral microbiota and their impact on CRC progression and discusses the disparities in the relative abundance and community composition of CRC intratumoral microbiota across various colorectal tumors based on their anatomical location and molecular subtypes, as well as the tumor stages, and spatial tumor distribution. Intratumoral microbiota predominantly influence CRC development by modulating colonic epithelial cells, tumor cells, and the tumor microenvironment. Mechanistically, they can cause DNA damage, apoptosis and epithelial-mesenchymal transition. The effects of different intratumoral microbiota on CRC have been shown to be two-fold. In the future, to address the limitations of existing studies, it is important to develop comprehensive experimental protocols and suitable in vitro models for elucidating more mechanisms of intratumoral microbiota on CRC, which will facilitate the clinical application of microbe-related therapeutic strategies in CRC and potentially other tumors.

[Source](https://doi.org/10.1186/s12964-024-01831-3)

10.1186/s12964-024-01831-3

---

## Bacterial extracellular vesicles at the interface of gut microbiota and immunity.
 28.09.2024, _I Melo-Marques, SM Cardoso, N Empadinhas_


_Extracellular Vesicles, Gastrointestinal Microbiome, Humans, Bacteria, Animals, Dysbiosis, Gastrointestinal Tract_
Bacterial extracellular vesicles (BEVs) are nano-sized lipid-shielded structures released by bacteria and that play an important role in intercellular communication. Their broad taxonomic origins and varying cargo compositions suggest their active participation in significant biological mechanisms. Specifically, they are involved in directly modulating microbial ecosystems, competing with other organisms, contributing to pathogenicity, and influencing the immunity of their hosts. This review examines the mechanisms that underlie the modulatory effects of BEVs on gut dynamics and immunity. Understanding how BEVs modulate microbiota composition and functional imbalances is crucial, as gut dysbiosis is implicated not only in the pathogenesis of various gastrointestinal, metabolic, and neurological diseases, but also in reducing resistance to colonization by enteric pathogens, which is particularly concerning given the current antimicrobial resistance crisis. This review summarizes recent advancements in the field of BEVs to encourage further research into these enigmatic entities. This will facilitate a better understanding of intra- and interkingdom communication phenomena and reveal promising therapeutic approaches.

[Source](https://doi.org/10.1080/19490976.2024.2396494)

10.1080/19490976.2024.2396494

---

## GV-971 prevents severe acute pancreatitis by remodeling the microbiota-metabolic-immune axis.
 27.09.2024, _X Chen, X Chen, D Yan, N Zhang, W Fu, M Wu, F Ge, J Wang, X Li, M Geng, J Wang, D Tang, J Liu_


_Animals, Gastrointestinal Microbiome, Male, Mice, Pancreatitis, Mice, Inbred C57BL, Disease Models, Animal, Macrophages, Fatty Acids, Volatile, Fecal Microbiota Transplantation, Humans, Metabolomics_
Despite recent advances, severe acute pancreatitis (SAP) remains a lethal inflammation with limited treatment options. Here, we provide compelling evidence of GV-971 (sodium oligomannate), an anti-Alzheimer's medication, as being a protective agent in various male mouse SAP models. Microbiome sequencing, along with intestinal microbiota transplantation and mass cytometry technology, unveil that GV-971 reshapes the gut microbiota, increasing Faecalibacterium populations and modulating both peripheral and intestinal immune systems. A metabolomics analysis of cecal contents from GV-971-treated SAP mice further identifies short-chain fatty acids, including propionate and butyrate, as key metabolites in inhibiting macrophage M1 polarization and subsequent lethal inflammation by blocking the MAPK pathway. These findings suggest GV-971 as a promising therapeutic for SAP by targeting the microbiota metabolic immune axis.

[Source](https://doi.org/10.1038/s41467-024-52398-z)

10.1038/s41467-024-52398-z

---

## Fecal Microbiota Transplantation-Mediated Rebalancing of the Gut-Brain Axis Alleviates Cisplatin-Induced Neuropathic Pain

[Source](https://doi.org/10.1021/acschemneuro.4c00267)

---

## FMT rescues mice from DSS-induced colitis in a STING-dependent manner

[Source](https://doi.org/10.1080/19490976.2024.2397879)

---

## Aged garlic oligosaccharides modulate host metabolism and gut microbiota to alleviate high-fat and high-cholesterol diet-induced atherosclerosis in ApoE(-/-) mice

[Source](https://doi.org/10.1016/j.foodchem.2024.141409)

---

## Phytoactives for Obesity Management: Integrating Nanomedicine for Its Effective Delivery.
 27.09.2024, _AK Mandal, A Sahoo, WH Almalki, SS Almujri, A Alhamyani, A Aodah, NK Alruwaili, SZBS Abdul Kadir, RK Mandal, RA Almalki, JA Lal, M Rahman_


Obesity is a global health concern that requires urgent investigation and management. While synthetic anti-obesity medications are available, they come with a high risk of side-effects and variability in their efficacy. Therefore, natural compounds are increasingly being used to treat obesity worldwide. The proposition that naturally occurring compounds, mainly polyphenols, can be effective and safer for obesity management through food and nutrient fortification is strongly supported by extensive experimental research. This review focuses on the pathogenesis of obesity while reviewing the efficacy of an array of phytoactives used for obesity treatment. It details mechanisms such as enzyme inhibition, energy expenditure, appetite suppression, adipocyte differentiation, lipid metabolism, and modulation of gut microbiota. Comprehensive in vitro, in vivo, and preclinical studies underscore the promise of phytoactives in combating obesity, which have been thoroughly reviewed. However, challenges, such as poor bioavailability and metabolism, limit their potential. Advances in nanomedicines may overcome these constraints, offering a new avenue for enhancing the efficacy of phytoactives. Nonetheless, rigorous and targeted clinical trials are essential before applying phytoactives as a primary treatment for obesity.

[Source](https://doi.org/10.1093/nutrit/nuae136)

10.1093/nutrit/nuae136

---

## Exploring microbial diversity and biosynthetic potential in zoo and wildlife animal microbiomes.
 28.09.2024, _GP Schmartz, J Rehner, MJ Schuff, LG Molano, SL Becker, M Krawczyk, A Tagirdzhanov, A Gurevich, R Francke, R Müller, V Keller, A Keller_


_Animals, Animals, Wild, Animals, Zoo, Gastrointestinal Microbiome, Anti-Bacterial Agents, Microbiota, Bacteria, Multigene Family, Humans, Biodiversity, Drug Resistance, Bacterial, Vancomycin, Biosynthetic Pathways_
Understanding human, animal, and environmental microbiota is essential for advancing global health and combating antimicrobial resistance (AMR). We investigate the oral and gut microbiota of 48 animal species in captivity, comparing them to those of wildlife animals. Specifically, we characterize the microbiota composition, metabolic pathways, AMR genes, and biosynthetic gene clusters (BGCs) encoding the production of specialized metabolites. Our results reveal a high diversity of microbiota, with 585 novel species-level genome bins (SGBs) and 484 complete BGCs identified. Functional gene analysis of microbiomes shows diet-dependent variations. Furthermore, by comparing our findings to wildlife-derived microbiomes, we observe the impact of captivity on the animal microbiome, including examples of converging microbiome compositions. Importantly, our study identifies AMR genes against commonly used veterinary antibiotics, as well as resistance to vancomycin, a critical antibiotic in human medicine. These findings underscore the importance of the 'One Health' approach and the potential for zoonotic transmission of pathogenic bacteria and AMR. Overall, our study contributes to a better understanding of the complexity of the animal microbiome and highlights its BGC diversity relevant to the discovery of novel antimicrobial compounds.

[Source](https://doi.org/10.1038/s41467-024-52669-9)

10.1038/s41467-024-52669-9

---

## Fur microbiome as a putative source of symbiotic bacteria in sucking lice.
 27.09.2024, _J Martin Říhová, S Gupta, E Nováková, V Hypša_


_Animals, Symbiosis, Microbiota, Phylogeny, RNA, Ribosomal, 16S, Phthiraptera, Bacteria_
Symbiosis between insects and bacteria has been established countless times. While it is well known that the symbionts originated from a variety of different bacterial taxa, it is usually difficult to determine their environmental source and a route of their acquisition by the host. In this study, we address this question using a model of Neisseriaceae symbionts in rodent lice. These bacteria established their symbiosis independently with different louse taxa (Polyplax, Hoplopleura, Neohaematopinus), most likely from the same environmental source. We first applied amplicon analysis to screen for candidate source bacterium in the louse environment. Since lice are permanent ectoparasites, often specific to the particular host, we screened various microbiomes associated with three rodent species (Microtus arvalis, Clethrionomys glareolus, and Apodemus flavicollis). The analyzed samples included fur, skin, spleen, and other ectoparasites sampled from these rodents. The fur microbiome data revealed a Neisseriaceae bacterium, closely related to the known louse symbionts. The draft genomes of the environmental Neisseriaceae, assembled from all three rodent hosts, converged to a remarkably small size of approximately 1.4 Mbp, being even smaller than the genomes of the related symbionts. Our results suggest that the rodent fur microbiome can serve as a source for independent establishment of bacterial symbiosis in associated louse species. We further propose a hypothetical scenario of the genome evolution during the transition of a free-living bacterium to the member of the rodent fur-associated microbiome and subsequently to the facultative and obligate louse symbionts.

[Source](https://doi.org/10.1038/s41598-024-73026-2)

10.1038/s41598-024-73026-2

---

## Fecal microbiota transplantation from patients with polycystic ovary syndrome induces metabolic disorders and ovarian dysfunction in germ-free mice.
 28.09.2024, _F Huang, Y Deng, M Zhou, R Tang, P Zhang, R Chen_


_Polycystic Ovary Syndrome, Animals, Fecal Microbiota Transplantation, Female, Mice, Gastrointestinal Microbiome, Humans, Dysbiosis, Feces, Metabolic Diseases, Bacteria, Insulin Resistance, Ovary, Germ-Free Life, Disease Models, Animal, Adult_
Dysbiosis of the microbiome is a key hallmark of polycystic ovary syndrome (PCOS). However, the interaction between the host and microbiome and its relevance to the pathogenesis of PCOS remain unclear.

[Source](https://doi.org/10.1186/s12866-024-03513-z)

10.1186/s12866-024-03513-z

---

## Dietary fiber intake impacts gut bacterial and viral populations in a hypertensive mouse model

[Source](https://doi.org/10.1080/19490976.2024.2407047)

---

## Effect of shipping on the microbiome of donor mice used to reconstitute germ-free recipients.
 28.09.2024, _ZL McAdams, J Yates, G Turner, RA Dorfmeyer, M Wight-Carter, J Amos-Landgraf, CL Franklin, AC Ericsson_


The gut microbiota (GM) influences multiple processes during host development and maintenance. To study these events, fecal microbiota transfer (FMT) to germ-free (GF) recipients is often performed. Mouse models of disease are also susceptible to GM-dependent effects, and cryo-repositories often store feces from donated mouse strains. Shipping live mice may affect the GM and result in an inaccurate representation of the baseline GM. We hypothesize that the use of such fecal samples for FMT would transfer shipping-induced changes in the donor GM to GF recipients. To test this, donor mice originating from two suppliers were shipped to the University of Missouri. Fecal samples collected pre- and post-shipping were used to inoculate GF mice. Pre- and post-shipping fecal samples from donors, and fecal and/or cecal contents were collected from recipients at one and two weeks post-FMT. 16S rRNA sequencing revealed supplier-dependent effects of shipping on the donor microbiome. FMT efficiency was independent of shipping timepoint or supplier, resulting in transmission of shipping-induced changes to recipient mice, however the effect of supplier-origin microbiome remained evident. While shipping may cause subtle changes in fecal samples collected for FMT, such effects are inconsistent among supplier-origin GMs and minor in comparison to other biological variables.

[Source](https://doi.org/10.1080/29933935.2024.2363858)

10.1080/29933935.2024.2363858

---

## Bacterial microbiome diversity along poultry slaughtering lines: insights from chicken carcasses and environmental sources.
 27.09.2024, _AE Telli, Y Biçer, N Telli, G Sönmez, G Turkal, İ Güzel_


This study aimed to determine the bacterial diversity of chicken carcasses and their surrounding environment at various stages along a poultry slaughter line.

[Source](https://doi.org/10.2478/jvetres-2024-0051)

10.2478/jvetres-2024-0051

---

## The Gut Microbiome of Two Wild Bumble Bee Species Native of South America: <em>Bombus pauloensis</em> and <em>Bombus bellicosus</em>

[Source](https://doi.org/10.1007/s00248-024-02430-y)

---

## Gill-associated ammonia oxidizers are widespread in teleost fish.
 26.09.2024, _W Mes, S Lücker, MS Jetten, H Siepel, M Gorissen, MA van Kessel_


Recent advances in sequencing methods have greatly expanded the knowledge of teleost-associated microorganisms. While fish-gut microbiomes are comparatively well studied, less attention has gone toward other, external organ-microbiome associations. Gills are particularly interesting to investigate due to their functions in gas exchange, osmoregulation, and nitrogen excretion. We recently discovered a branchial symbiosis between nitrogen-cycling bacteria and teleosts (zebrafish and carp), in which ammonia-oxidizing Nitrosomonas and denitrifying bacteria together convert toxic ammonia excreted by the fish into harmless dinitrogen (N2) gas. This symbiosis can function as a "natural biofilter" in fish gills and can potentially occur in all ammonotelic fish species, but it remains unknown how widespread this symbiosis is. In this study, we analyzed all publicly available gill microbiome data sets and checked for the presence of Nitrosomonas. We discovered that more than half of the described fish gill microbiomes contain 16S rRNA gene sequences of ammonia-oxidizing bacteria (AOB). The presence of gill-specific AOB was shown in both wild and aquacultured fish, as well as in marine and freshwater fish species. Based on these findings, we propose that ammonia oxidizers are widespread in teleost fish gills. These gill-associated AOB can significantly affect fish nitrogen excretion, and the widespread nature of this association suggests that the gill-associated AOB can have similar impacts on more fish species. Future research should address the contribution of these microorganisms to fish nitrogen metabolism and the fundamental characteristics of this novel symbiosis.IMPORTANCERecent advances in sequencing have increased our knowledge of teleost-associated microbiota, but the gill microbiome has received comparatively little attention. We recently discovered a consortium of nitrogen-cycling bacteria in the gills of common carp and zebrafish, which are able to convert (toxic) ammonia into harmless dinitrogen gas. These microorganisms thus function as a natural nitrogen biofilter. We analyzed all available gill microbiome data sets to determine how widespread gill-associated ammonia-oxidizing bacteria (AOB) are. More than half of the data sets contained AOB, representing both aquacultured and wild fish from freshwater and marine habitats. In total, 182 amplicon sequencing variants were obtained, of which 115 were found specifically in the gills and not the environmental microbiomes. As gill-associated AOB are apparently widespread in teleost fish, it is important to study their impact on host nitrogen excretion and the potential to reduce ammonia accumulation in (recirculating) aquaculture of relevant fish species.

[Source](https://doi.org/10.1128/spectrum.00295-24)

10.1128/spectrum.00295-24

---

## Automated annotation of scientific texts for ML-based keyphrase extraction and validation.
 27.09.2024, _OO Amusat, H Hegde, CJ Mungall, A Giannakou, NP Byers, D Gunter, K Fagnan, L Ramakrishnan_


_Machine Learning, Data Curation, Data Mining, Metadata_
Advanced omics technologies and facilities generate a wealth of valuable data daily; however, the data often lack the essential metadata required for researchers to find, curate, and search them effectively. The lack of metadata poses a significant challenge in the utilization of these data sets. Machine learning (ML)-based metadata extraction techniques have emerged as a potentially viable approach to automatically annotating scientific data sets with the metadata necessary for enabling effective search. Text labeling, usually performed manually, plays a crucial role in validating machine-extracted metadata. However, manual labeling is time-consuming and not always feasible; thus, there is a need to develop automated text labeling techniques in order to accelerate the process of scientific innovation. This need is particularly urgent in fields such as environmental genomics and microbiome science, which have historically received less attention in terms of metadata curation and creation of gold-standard text mining data sets. In this paper, we present two novel automated text labeling approaches for the validation of ML-generated metadata for unlabeled texts, with specific applications in environmental genomics. Our techniques show the potential of two new ways to leverage existing information that is only available for select documents within a corpus to validate ML models, which can then be used to describe the remaining documents in the corpus. The first technique exploits relationships between different types of data sources related to the same research study, such as publications and proposals. The second technique takes advantage of domain-specific controlled vocabularies or ontologies. In this paper, we detail applying these approaches in the context of environmental genomics research for ML-generated metadata validation. Our results show that the proposed label assignment approaches can generate both generic and highly specific text labels for the unlabeled texts, with up to 44% of the labels matching with those suggested by a ML keyword extraction algorithm.

[Source](https://doi.org/10.1093/database/baae093)

10.1093/database/baae093

---

## SpeciateIT and vSpeciateDB: novel, fast, and accurate per sequence 16S rRNA gene taxonomic classification of vaginal microbiota.
 28.09.2024, _JB Holm, P Gajer, J Ravel_


_RNA, Ribosomal, 16S, Microbiota, Vagina, Female, Algorithms, Humans, Software, Databases, Genetic_
Clustering of sequences into operational taxonomic units (OTUs) and denoising methods are a mainstream stopgap to taxonomically classifying large numbers of 16S rRNA gene sequences. Environment-specific reference databases generally yield optimal taxonomic assignment.

[Source](https://doi.org/10.1186/s12859-024-05930-3)

10.1186/s12859-024-05930-3

---

## Implementing high-throughput insect barcoding in microbiome studies: impact of non-destructive DNA extraction on microbiome reconstruction.
 28.09.2024, _V Andriienko, M Buczek, R Meier, A Srivathsan, P Łukasik, MR Kolasa_


_Animals, DNA Barcoding, Taxonomic, Microbiota, Insecta, RNA, Ribosomal, 16S, DNA, Bacterial, Bacteria, Polymerase Chain Reaction, Biodiversity, High-Throughput Nucleotide Sequencing_
Symbiotic relationships with diverse microorganisms are crucial for many aspects of insect biology. However, while our understanding of insect taxonomic diversity and the distribution of insect species in natural communities is limited, we know much less about their microbiota. In the era of rapid biodiversity declines, as researchers increasingly turn towards DNA-based monitoring, developing and broadly implementing approaches for high-throughput and cost-effective characterization of both insect and insect-associated microbial diversity is essential. We need to verify whether approaches such as high-throughput barcoding, a powerful tool for identifying wild insects, would permit subsequent microbiota reconstruction in these specimens.

[Source](https://doi.org/10.7717/peerj.18025)

10.7717/peerj.18025

---

## A Metagenomics Pipeline to Characterize Self-Collected Vaginal Microbiome Samples.
 28.09.2024, _K Thomas-White, EE Hilt, G Olmschenk, M Gong, CD Phillips, C Jarvis, N Sanford, J White, P Navarro_


Vaginitis is a widespread issue for women worldwide, yet current diagnostic tools are lacking. Bacterial vaginosis (BV) is the most prevalent type of vaginitis, found in 10-50% of reproductive-aged women. Current diagnostic methods for BV rely on clinical criteria, microscopy, or the detection of a few microbes by qPCR. However, many vaginal infections lack a single etiological agent and are characterized by changes in the vaginal microbiome community structure (e.g., BV is defined as a loss of protective lactobacilli resulting in an overgrowth of anaerobic bacteria). Shotgun metagenomic sequencing provides a comprehensive view of all the organisms present in the vaginal microbiome (VMB), allowing for a better understanding of all potential etiologies. Here, we describe a robust VMB metagenomics sequencing test with a sensitivity of 93.1%, a specificity of 90%, a negative predictive value of 93.4%, and a positive predictive value of 89.6% certified by Clinical Laboratory Improvement Amendments (CLIA), the College of American Pathologist (CAP), and the Clinical Laboratory Evaluation Program (CLEP). We sequenced over 7000 human vaginal samples with this pipeline and described general findings and comparisons to US census data.

[Source](https://doi.org/10.3390/diagnostics14182039)

10.3390/diagnostics14182039

---

## Microbe-drug association prediction model based on graph convolution and attention networks.
 27.09.2024, _B Wang, T Wang, X Du, J Li, J Wang, P Wu_


_Humans, Deep Learning, Microbiota, Computational Biology, Neural Networks, Computer, Drug Discovery, Pharmaceutical Preparations_
The human microbiome plays a key role in drug development and precision medicine, but understanding its complex interactions with drugs remains a challenge. Identifying microbe-drug associations not only enhances our understanding of their mechanisms but also aids in drug discovery and repurposing. Traditional experiments are expensive and time-consuming, making computational methods for predicting microbe-drug associations a new trend. Currently, computational methods specifically designed for this task are still scarce. Therefore, to address the shortcomings of traditional experimental methods in predicting potential microbe-drug associations, this paper proposes a new prediction model named GCNATMDA. The model combines two deep learning models, Graph Convolutional Network and Graph Attention Network, and aims to reveal potential relationships between microbes and drugs by learning related features. Thus improve the efficiency and accuracy of prediction. We first integrated the microbe-drug association matrix from the existing dataset, and then combined the calculated microbe-drug characteristic matrix as the model input. The GCN module is used to dig deeper into the potential characterization of microbes and drugs, while the GAT module further learns the more complex interactions between them and generates the corresponding score matrix. The experimental results show that the GCNATMDA model achieves 96.59% and 93.01% in AUC and AUPR evaluation indexes, respectively, which is significantly better than the existing prediction models. In addition, the reliability of the prediction results is verified by a series of experiments.

[Source](https://doi.org/10.1038/s41598-024-71834-0)

10.1038/s41598-024-71834-0

---

