## Strain-level functional variation in the human gut microbiota based on bacterial binding to artificial food particles.
 16.02.2021, _ML Patnode, JL Guruge, JJ Castillo, GA Couture, V Lombard, N Terrapon, B Henrissat, CB Lebrilla, JI Gordon_


Greater understanding of the spatial relationships between members of the human gut microbiota and available nutrients is needed to gain deeper insights about community dynamics and expressed functions. Therefore, we generated a panel of artificial food particles with each type composed of microscopic paramagnetic beads coated with a fluorescent barcode and one of 60 different dietary or host glycan preparations. Analysis of 160 Bacteroides and Parabacteroides strains disclosed diverse strain-specific and glycan-specific binding phenotypes. We identified carbohydrate structures that correlated with binding by specific bacterial strains in vitro and noted strain-specific differences in the catabolism of glycans that mediate adhesion. Mixed in vitro cultures revealed that these adhesion phenotypes are maintained in more complex communities. Additionally, orally administering glycan beads to gnotobiotic mice confirmed specificity in glycan binding. This approach should facilitate analyses of how strains occupying the same physical niche interact, and it should advance the development of synbiotics, more nutritious foods, and microbiota-based diagnostics.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(21)00037-8)

10.1016/j.chom.2021.01.007

---

## Debaryomyces is enriched in Crohn's disease intestinal tissue and impairs healing in mice.
 16.03.2021, _U Jain, AM Ver Heul, S Xiong, MH Gregory, EG Demers, JT Kern, CW Lai, BD Muegge, DAG Barisas, JS Leal-Ekman, P Deepak, MA Ciorba, TC Liu, DA Hogan, P Debbas, J Braun, DPB McGovern, DM Underhill, TS Stappenbeck_


Alterations of the mycobiota composition associated with Crohn's disease (CD) are challenging to link to defining elements of pathophysiology, such as poor injury repair. Using culture-dependent and -independent methods, we discovered that Debaryomyces hansenii preferentially localized to and was abundant within incompletely healed intestinal wounds of mice and inflamed mucosal tissues of CD human subjects. D. hansenii cultures from injured mice and inflamed CD tissues impaired colonic healing when introduced into injured conventionally raised or gnotobiotic mice. We reisolated D. hansenii from injured areas of these mice, fulfilling Koch's postulates. Mechanistically, D. hansenii impaired mucosal healing through the myeloid cell-specific type 1 interferon-CCL5 axis. Taken together, we have identified a fungus that inhabits inflamed CD tissue and can lead to dysregulated mucosal healing.

[Source](https://science.sciencemag.org/content/371/6534/1154)

10.1126/science.abd0919

---

## Dysbiosis in Metabolic Genes of the Gut Microbiomes of Patients with an Ileo-anal Pouch Resembles That Observed in Crohn's Disease.
 03.03.2021, _V Dubinsky, L Reshef, K Rabinowitz, K Yadgar, L Godny, K Zonensain, N Wasserberg, I Dotan, U Gophna_


Crohn's disease (CD), ulcerative colitis (UC), and pouchitis are multifactorial and chronic inflammatory bowel diseases (IBD). Pouchitis develops in former UC patients after proctocolectomy and ileal-pouch-anal anastomosis and is characterized by inflammation of the previously normal small intestine comprising the pouch. The extent to which microbial functional alteration (dysbiosis) in pouchitis resembles that of CD or UC has not been investigated, and the pathogenesis of pouchitis remains unknown. We collected 208 fecal metagenomes from 69 patients with a pouch (normal pouch and pouchitis) and compared them to publicly available metagenomes of patients with CD (n = 88), patients with UC (n = 76), and healthy controls (n = 56). Patients with pouchitis presented the highest alterations in species, metabolic pathways, and enzymes, which was correlated with intestinal inflammation. Ruminococcus gnavus strains encoding mucin-degrading glycoside hydrolases were highly enriched in pouchitis. Butyrate and secondary bile acid biosynthesis pathways were decreased in IBD phenotypes and were especially low in pouchitis. Pathways such as amino acid biosynthesis and degradation of aromatic compounds and sugars, encoded by members of the Enterobacteriaceae, were enriched in pouch and CD but not in UC. We developed microbial feature-based classifiers that can distinguish between patients with a normal pouch and pouchitis and identified species and genes that were predictive of pouchitis. We propose that the noninflamed pouch is already dysbiotic and microbially is similar to CD. Our study reveals microbial functions that outline the pathogenesis of pouchitis and suggests bacterial groups and functions that could be targeted for intervention to attenuate small intestinal inflammation present in pouchitis and CD.IMPORTANCE Crohn's disease (CD), ulcerative colitis (UC), and pouchitis are chronic inflammatory conditions of the bowel. Pouchitis develops in former UC patients after proctocolectomy and ileal-pouch-anal anastomosis and is characterized by inflammation of the previously normal small intestine comprising the pouch. The extent to which microbial dysbiosis in patients with pouchitis resembles that of CD or UC and the pathogenesis of pouchitis remains unclear. We investigated the functions in the gut microbiomes of these patients using metagenomics. We found that the noninflamed pouch is already dysbiotic and microbially is similar to CD. Our study reveals microbial functions with a potential role in pouchitis pathogenesis such as depletion in butyrate and secondary bile acid synthesis and enrichment of amino acid synthesis and degradation of aromatic compounds, encoded by members of the Enterobacteriaceae We developed microbial feature-based classifiers that can distinguish between patients with a normal pouch and pouchitis and identified species and genes that were predictive of pouchitis. We suggest species and functions that could be targeted for intervention to attenuate small intestinal inflammation present in pouchitis and CD.

[Source](https://msystems.asm.org/content/6/2/e00984-20)

10.1128/mSystems.00984-20

---

## Human-associated microbiota suppress invading bacteria even under disruption by antibiotics.
 13.03.2021, _AD Letten, M Baumgartner, KR Pfrunder-Cardozo, JM Levine, AR Hall_


In light of their adverse impacts on resident microbial communities, it is widely predicted that broad-spectrum antibiotics can promote the spread of resistance by releasing resistant strains from competition with other strains and species. We investigated the competitive suppression of a resistant strain of Escherichia coli inoculated into human-associated communities in the presence and absence of the broad and narrow spectrum antibiotics rifampicin and polymyxin B, respectively. We found strong evidence of community-level suppression of the resistant strain in the absence of antibiotics and, despite large changes in community composition and abundance following rifampicin exposure, suppression of the invading resistant strain was maintained in both antibiotic treatments. Instead, the strength of competitive suppression was more strongly associated with the source community (stool sample from individual human donor). This suggests microbiome composition strongly influences the competitive suppression of antibiotic-resistant strains, but at least some antibiotic-associated disruption can be tolerated before competitive release is observed. A deeper understanding of this association will aid the development of ecologically-aware strategies for managing antibiotic resistance.

[Source](https://www.nature.com/articles/s41396-021-00929-7)

10.1038/s41396-021-00929-7

---

## Key bacterial taxa and metabolic pathways affecting gut short-chain fatty acid profiles in early life.
 16.03.2021, _N Tsukuda, K Yahagi, T Hara, Y Watanabe, H Matsumoto, H Mori, K Higashi, H Tsuji, S Matsumoto, K Kurokawa, T Matsuki_


Infant gut microbiota development affects the host physiology throughout life, and short-chain fatty acids (SCFAs) are promising key metabolites mediating microbiota-host relationships. Here, we investigated dense longitudinally collected faecal samples from 12 subjects during the first 2 years (n = 1048) to identify early life gut SCFA patterns and their relationships with the microbiota. Our results revealed three distinct phases of progression in the SCFA profiles: early phase characterised by low acetate and high succinate, middle-phase characterised by high lactate and formate and late-phase characterised by high propionate and butyrate. Assessment of the SCFA-microbiota relationships revealed that faecal butyrate is associated with increased Clostridiales and breastfeeding cessation, and that diverse and personalised assemblage of Clostridiales species possessing the acetyl-CoA pathway play major roles in gut butyrate production. We also found an association between gut formate and some infant-type bifidobacterial species, and that human milk oligosaccharides (HMO)-derived fucose is the substrate for formate production during breastfeeding. We identified genes upregulated in fucose and fucosylated HMO utilisation in infant-type bifidobacteria. Notably, bifidobacteria showed interspecific and intraspecific variation in the gene repertoires, and cross-feeding of fucose contributed to gut formate production. This study provides an insight into early life SCFA-microbiota relationships, which is an important step for developing strategies for modulating lifelong health.

[Source](https://www.nature.com/articles/s41396-021-00937-7)

10.1038/s41396-021-00937-7

---

## Tonsillar Microbiota: a Cross-Sectional Study of Patients with Chronic Tonsillitis or Tonsillar Hypertrophy

[Source](https://msystems.asm.org/content/6/2/e01302-20)

---

## Long-term evolution and short-term adaptation of microbiota strains and sub-strains in mice

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312821000470)

---

## ContigExtender: a new approach to improving de novo sequence assembly for viral metagenomics data

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04038-2)

---

