## Assessment of disinfectant efficacy in reducing microbial growth.
 29.06.2022, _AA Alajlan, LE Mukhtar, AS Almussallam, AM Alnuqaydan, NS Albakiri, TF Almutari, KM Bin Shehail, FS Aldawsari, SM Alajel_


_COVID-19, Chlorhexidine, Disinfectants, Escherichia coli, Hand Sanitizers, Humans, Sterilization_
The incidence of hospital- and community-acquired infections has been dramatically increased worldwide. Accordingly, hands hygiene and the use of disinfectants have been increased leading to the expansion in hand sanitizers production to meet public demand. This study was conducted to assess the efficiency of common disinfectants in the market of Riyadh, Saudi Arabia in inhibiting the microbial growth during the time of Coronavirus disease 2019 (COVID-19) pandemic. Five bacterial strains of commonly hospital-acquired infections (Pseudomonas aeruginosa, Escherichia coli, Salmonella enteritidis, Staphylococcus aureus, and Enterococcus faecalis) (ATCC reference strains and clinical isolates) were examined for their susceptibility against 18 disinfectants collected from the Saudi market. The tested 18 disinfectants were broadly clustered into different groups based on their active chemical composition as following: 12 products contained alcohol, 2 products had chlorhexidine, 3 products contained mixed concentration of alcohol/chlorhexidine and 1 product had a mixture of chlorhexidine/Hexamidine/Chlorocresol. By measuring the minimum inhibitory concentration (MIC) and the minimum bactericidal concentration (MBC), our results revealed that all the 18 disinfectants have reduced the microbial growth of all the tested strains. Generally, the MICs and the MBCs for the clinical strains are higher than those of the reference strains. Taken together, our findings showed that all tested products have high disinfectants' killing rate against microbes of different origins, which suggest the high quality of these disinfectants and the good surveillance practice by the local authorities in Saudi Arabia.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0269850)

10.1371/journal.pone.0269850

---

## Metagenomic analysis of the gut microbiota in piglets either challenged or not with enterotoxigenic Escherichia coli reveals beneficial effects of probiotics on microbiome composition, resistome, digestive function and oxidative stress responses.
 28.06.2022, _P Apiwatsiri, P Pupa, W Sirichokchatchawan, V Sawaswong, P Nimsamer, S Payungporn, DJ Hampson, N Prapasarakul_


_Animals, Enterotoxigenic Escherichia coli, Escherichia coli Infections, Gastrointestinal Microbiome, Microbiota, Oxidative Stress, Probiotics, Swine, Swine Diseases, Weaning_
This study used metagenomic analysis to investigate the gut microbiota and resistome in piglets that were or were not challenged with enterotoxigenic Escherichia coli (ETEC) and had or had not received dietary supplementation with microencapsulated probiotics. The 72 piglets belonged to six groups that were either non-ETEC challenged (groups 1-3) or ETEC challenged (receiving 5ml of 109 CFU/ml pathogenic ETEC strain L3.2 one week following weaning at three weeks of age: groups 4-6). On five occasions at 2, 5, 8, 11, and 14 days of piglet age, groups 2 and 5 were supplemented with 109 CFU/ml of multi-strain probiotics (Lactiplantibacillus plantarum strains 22F and 25F, and Pediococcus acidilactici 72N) while group 4 received 109 CFU/ml of P. acidilactici 72N. Group 3 received 300mg/kg chlortetracycline in the weaner diet to mimic commercial conditions. Rectal faecal samples were obtained for metagenomic and resistome analysis at 2 days of age, and at 12 hours and 14 days after the timing of post-weaning challenge with ETEC. The piglets were all euthanized at 42 days of age. The piglets in groups 2 and 5 were enriched with several desirable microbial families, including Lactobacillaceae, Lachnospiraceae and Ruminococcaceae, while piglets in group 3 had increases in members of the Bacteroidaceae family and exhibited an increase in tetW and tetQ genes. Group 5 had less copper and multi-biocide resistance. Mobile genetic elements IncQ1 and IncX4 were the most prevalent replicons in antibiotic-fed piglets. Only groups 6 and 3 had the integrase gene (intl) class 2 and 3 detected, respectively. The insertion sequence (IS) 1380 was prevalent in group 3. IS3 and IS30, which are connected to dietary intake, were overrepresented in group 5. Furthermore, only group 5 showed genes associated with detoxification, with enrichment of genes associated with oxidative stress, glucose metabolism, and amino acid metabolism compared to the other groups. Overall, metagenomic analysis showed that employing a multi-strain probiotic could transform the gut microbiota, reduce the resistome, and boost genes associated with food metabolism.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0269959#sec002)

10.1371/journal.pone.0269959

---

## Phylogenetic diversity analysis of shotgun metagenomic reads describes gut microbiome development and treatment effects in the post-weaned pig.
 28.06.2022, _D Gaio, MZ DeMaere, K Anantanawat, GJ Eamens, L Falconer, TA Chapman, S Djordjevic, AE Darling_


_Animals, Anti-Bacterial Agents, Dysbiosis, Female, Gastrointestinal Microbiome, Microbiota, Phylogeny, Probiotics, Swine, Weaning_
Intensive farming practices can increase exposure of animals to infectious agents against which antibiotics are used. Orally administered antibiotics are well known to cause dysbiosis. To counteract dysbiotic effects, numerous studies in the past two decades sought to understand whether probiotics are a valid tool to help re-establish a healthy gut microbial community after antibiotic treatment. Although dysbiotic effects of antibiotics are well investigated, little is known about the effects of intramuscular antibiotic treatment on the gut microbiome and a few studies attempted to study treatment effects using phylogenetic diversity analysis techniques. In this study we sought to determine the effects of two probiotic- and one intramuscularly administered antibiotic treatment on the developing gut microbiome of post-weaning piglets between their 3rd and 9th week of life. Shotgun metagenomic sequences from over 800 faecal time-series samples derived from 126 post-weaning piglets and 42 sows were analysed in a phylogenetic framework. Differences between individual hosts such as breed, litter, and age, were found to be important contributors to variation in the community composition. Host age was the dominant factor in shaping the gut microbiota of piglets after weaning. The post-weaning pig gut microbiome appeared to follow a highly structured developmental program with characteristic post-weaning changes that can distinguish hosts that were born as little as two days apart in the second month of life. Treatment effects of the antibiotic and probiotic treatments were found but were subtle and included a higher representation of Mollicutes associated with intramuscular antibiotic treatment, and an increase of Lactobacillus associated with probiotic treatment. The discovery of correlations between experimental factors and microbial community composition is more commonly addressed with OTU-based methods and rarely analysed via phylogenetic diversity measures. The latter method, although less intuitive than the former, suffers less from library size normalization biases, and it proved to be instrumental in this study for the discovery of correlations between microbiome composition and host-, and treatment factors.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0270372#sec002)

10.1371/journal.pone.0270372

---

## Using axenic and gnotobiotic insects to examine the role of different microbes on the development and reproduction of the kissing bug <em>Rhodnius prolixus</em> (Hemiptera: Reduviidae)

[Source](https://www.biorxiv.org/content/10.1101/2022.06.29.497934v1.abstract?%3Fcollection=)

---

## MicNet toolbox: Visualizing and unraveling a microbial network.
 28.06.2022, _N Favila, D Madrigal-Trejo, D Legorreta, J Sánchez-Pérez, L Espinosa-Asuar, LE Eguiarte, V Souza_


_Cluster Analysis, Computational Biology, Microbial Consortia, Microbiota, Software_
Applications of network theory to microbial ecology are an emerging and promising approach to understanding both global and local patterns in the structure and interplay of these microbial communities. In this paper, we present an open-source python toolbox which consists of two modules: on one hand, we introduce a visualization module that incorporates the use of UMAP, a dimensionality reduction technique that focuses on local patterns, and HDBSCAN, a clustering technique based on density; on the other hand, we have included a module that runs an enhanced version of the SparCC code, sustaining larger datasets than before, and we couple the resulting networks with network theory analyses to describe the resulting co-occurrence networks, including several novel analyses, such as structural balance metrics and a proposal to discover the underlying topology of a co-occurrence network. We validated the proposed toolbox on 1) a simple and well described biological network of kombucha, consisting of 48 ASVs, and 2) we validate the improvements of our new version of SparCC. Finally, we showcase the use of the MicNet toolbox on a large dataset from Archean Domes, consisting of more than 2,000 ASVs. Our toolbox is freely available as a github repository (https://github.com/Labevo/MicNetToolbox), and it is accompanied by a web dashboard (http://micnetapplb-1212130533.us-east-1.elb.amazonaws.com) that can be used in a simple and straightforward manner with relative abundance data. This easy-to-use implementation is aimed to microbial ecologists with little to no experience in programming, while the most experienced bioinformatics will also be able to manipulate the source code's functions with ease.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0259756)

10.1371/journal.pone.0259756

---

## Evaluating the impact of sample storage, handling, and technical ability on the decay and recovery of SARS-CoV-2 in wastewater.
 28.06.2022, _RE Beattie, AD Blackwood, T Clerkin, C Dinga, RT Noble_


_COVID-19, Humans, Pandemics, RNA, Viral, SARS-CoV-2, Waste Water, Wastewater-Based Epidemiological Monitoring_
Wastewater based epidemiology (WBE) is useful for tracking and monitoring the level of disease prevalence in a community and has been used extensively to complement clinical testing during the current COVID-19 pandemic. Despite the numerous benefits, sources of variability in sample storage, handling, and processing methods can make WBE data difficult to generalize. We performed an experiment to determine sources of variability in WBE data including the impact of storage time, handling, and processing techniques on the concentration of SARS-CoV-2 in wastewater influent from three wastewater treatment plants (WWTP) in North Carolina over 19 days. The SARS-CoV-2 concentration in influent samples held at 4°C did not degrade significantly over the 19-day experiment. Heat pasteurization did not significantly impact the concentration of SARS-CoV-2 at two of the three WWTP but did reduce viral recovery at the WWTP with the smallest population size served. On each processing date, one filter from each sample was processed immediately while a replicate filter was frozen at -80°C. Once processed, filters previously frozen were found to contain slightly higher concentrations (&lt;0.2 log copies/L) than their immediately processed counterparts, indicating freezing filters is a viable method for delayed quantification and may even improve recovery at WWTP with low viral concentrations. Investigation of factors contributing to variability during sample processing indicated that analyst experience level contributed significantly (p&lt;0.001) to accepted droplet generation while extraction efficiency and reverse transcription efficiency contributed significantly (p&lt;0.05) to day-to-day SARS-CoV-2 variability. This study provides valuable practical information for minimizing decay and/or loss of SARS CoV-2 in wastewater influent while adhering to safety procedures, promoting efficient laboratory workflows, and accounting for sources of variability.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0270659)

10.1371/journal.pone.0270659

---

## NPR’s Short Wave Podcast: Let’s Get Crafty With Agar Art

[Source](https://www.npr.org/2022/06/22/1106686014/lets-get-crafty-with-agar-art)

---

## Ten simple rules for improving communication among scientists.
 01.07.2022, _C Bautista, N Alfuraiji, A Drangowska-Way, K Gangwani, A de Flamingh, PE Bourne_


_Communication, Retrospective Studies, Social Networking_
Communication is a fundamental part of scientific development and methodology. With the advancement of the internet and social networks, communication has become rapid and sometimes overwhelming, especially in science. It is important to provide scientists with useful, effective, and dynamic tools to establish and build a fluid communication framework that allows for scientific advancement. Therefore, in this article, we present advice and recommendations that can help promote and improve science communication while respecting an adequate balance in the degree of commitment toward collaborative work. We have developed 10 rules shown in increasing order of commitment that are grouped into 3 key categories: (1) speak (based on active participation); (2) join (based on joining scientific groups); and (3) assess (based on the analysis and retrospective consideration of the weaknesses and strengths). We include examples and resources that provide actionable strategies for involvement and engagement with science communication, from basic steps to more advanced, introspective, and long-term commitments. Overall, we aim to help spread science from within and encourage and engage scientists to become involved in science communication effectively and dynamically.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010130)

10.1371/journal.pcbi.1010130

---

## You can’t report your feelings: The hidden labor of managing threats to safety by women in global public health fieldwork

[Source](https://journals.plos.org/globalpublichealth/article?id=10.1371/journal.pgph.0000153)

---

