## A possible role for taste receptor cells in surveying the oral microbiome.
 13.01.2023, _EM Heisey, LP McCluskey_


Taste receptor cells are sensory specialists that detect chemicals in food and drink. An exciting new report in PLOS Biology suggests that some taste cells could also be involved in immune surveillance like counterparts in the intestine.

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001953)

10.1371/journal.pbio.3001953

---

## Type II taste cells participate in mucosal immune surveillance.
 13.01.2023, _Y Qin, SR Palayyan, X Zheng, S Tian, RF Margolskee, SK Sukumaran_


The oral microbiome is second only to its intestinal counterpart in diversity and abundance, but its effects on taste cells remains largely unexplored. Using single-cell RNASeq, we found that mouse taste cells, in particular, sweet and umami receptor cells that express taste 1 receptor member 3 (Tas1r3), have a gene expression signature reminiscent of Microfold (M) cells, a central player in immune surveillance in the mucosa-associated lymphoid tissue (MALT) such as those in the Peyer's patch and tonsils. Administration of tumor necrosis factor ligand superfamily member 11 (TNFSF11; also known as RANKL), a growth factor required for differentiation of M cells, dramatically increased M cell proliferation and marker gene expression in the taste papillae and in cultured taste organoids from wild-type (WT) mice. Taste papillae and organoids from knockout mice lacking Spib (SpibKO), a RANKL-regulated transcription factor required for M cell development and regeneration on the other hand, failed to respond to RANKL. Taste papillae from SpibKO mice also showed reduced expression of NF-κB signaling pathway components and proinflammatory cytokines and attracted fewer immune cells. However, lipopolysaccharide-induced expression of cytokines was strongly up-regulated in SpibKO mice compared to their WT counterparts. Like M cells, taste cells from WT but not SpibKO mice readily took up fluorescently labeled microbeads, a proxy for microbial transcytosis. The proportion of taste cell subtypes are unaltered in SpibKO mice; however, they displayed increased attraction to sweet and umami taste stimuli. We propose that taste cells are involved in immune surveillance and may tune their taste responses to microbial signaling and infection.

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001647)

10.1371/journal.pbio.3001647

---

## How our microbiome is shaped by family, friends and even neighbours.
 19.01.2023, _E Callaway_



[Source](https://www.nature.com/articles/d41586-023-00118-w)

10.1038/d41586-023-00118-w

---

## The person-to-person transmission landscape of the gut and oral microbiomes.
 18.01.2023, _M Valles-Colomer, A Blanco-Míguez, P Manghi, F Asnicar, L Dubois, D Golzato, F Armanini, F Cumbo, KD Huang, S Manara, G Masetti, F Pinto, E Piperni, M Punčochář, L Ricci, M Zolfo, O Farrant, A Goncalves, M Selma-Royo, AG Binetti, JE Becerra, B Han, J Lusingu, J Amuasi, L Amoroso, A Visconti, CM Steves, M Falchi, M Filosi, A Tett, A Last, Q Xu, N Qin, H Qin, J May, D Eibach, MV Corrias, M Ponzoni, E Pasolli, TD Spector, E Domenici, MC Collado, N Segata_


The human microbiome is an integral component of the human body and a co-determinant of several health conditions1,2. However, the extent to which interpersonal relations shape the individual genetic makeup of the microbiome and its transmission within and across populations remains largely unknown3,4. Here, capitalizing on more than 9,700 human metagenomes and computational strain-level profiling, we detected extensive bacterial strain sharing across individuals (more than 10 million instances) with distinct mother-to-infant, intra-household and intra-population transmission patterns. Mother-to-infant gut microbiome transmission was considerable and stable during infancy (around 50% of the same strains among shared species (strain-sharing rate)) and remained detectable at older ages. By contrast, the transmission of the oral microbiome occurred largely horizontally and was enhanced by the duration of cohabitation. There was substantial strain sharing among cohabiting individuals, with 12% and 32% median strain-sharing rates for the gut and oral microbiomes, and time since cohabitation affected strain sharing more than age or genetics did. Bacterial strain sharing additionally recapitulated host population structures better than species-level profiles did. Finally, distinct taxa appeared as efficient spreaders across transmission modes and were associated with different predicted bacterial phenotypes linked with out-of-host survival capabilities. The extent of microorganism transmission that we describe underscores its relevance in human microbiome studies5, especially those on non-infectious, microbiome-associated diseases.

[Source](https://www.nature.com/articles/s41586-022-05620-1)

10.1038/s41586-022-05620-1

---

## DUCNP@Mn-MOF/FOE as a Highly Selective and Bioavailable Drug Delivery System for Synergistic Combination Cancer Therapy.
 18.01.2023, _X Zhao, S He, B Li, B Liu, Y Shi, W Cong, F Gao, J Li, F Wang, K Liu, C Sheng, J Su, HG Hu_


Heterostructures comprising lanthanide-doped upconversion nanoparticles (DUCNPs) and metal-organic frameworks (MOFs) are emerging as promising nanosystems for integrating medical diagnosis and treatment. Here, the DUCNP@Mn-MOF nanocarrier was developed, which showed good efficiency for loading and delivering a cytotoxic antitumor agent (3-F-10-OH-evodiamine, FOE). The combined advantages of the pH-responsive and peroxidase-like properties of Mn-MOF and the unique optical features of DUCNPs granted the DUCNP@Mn-MOF/FOE system synergistic chemodynamic and chemotherapeutic effects. The DUCNP@Mn-MOF nanocarrier effectively overcame the intrinsic limitations of FOE, such as its unfavorable physicochemical properties and limited in vivo potency. This complexed nanosystem was responsive to the tumor microenvironment and showed excellent tumor targeting capability. Thus, DUCNP@Mn-MOF/FOE exhibited highly selective and bioavailable drug delivery properties and is promising for cancer therapy. In a mouse breast cancer model, DUCNP@Mn-MOF/FOE inhibited tumor growth without significant toxicity. Therefore, the proposed nanosystem represents a promising theragnostic platform for multimodal combination diagnosis and therapy of tumors.

[Source](https://www.nature.com/articles/s41574-022-00794-0)

10.1021/acs.nanolett.2c04042

---

## The gut microbiota in obesity and weight management: microbes as friends or foe?

[Source](https://www.nature.com/articles/s41574-022-00794-0)

---

## Dynamics of the gut microbiome, IgA response, and plasma metabolome in the development of pediatric celiac disease.
 18.01.2023, _K Girdhar, YD Dogru, Q Huang, Y Yang, V Tolstikov, A Raisingani, M Chrudinova, J Oh, K Kelley, JF Ludvigsson, MA Kiebish, NW Palm, J Ludvigsson, E Altindis_


_Animals, Child, Preschool, Humans, Mice, Atrophy, Celiac Disease, Cytokines, Gastrointestinal Microbiome, Glutens, Immunoglobulin A, Metabolome, Mice, Inbred C57BL, Prospective Studies_
Celiac disease (CD) is an autoimmune disorder triggered by gluten consumption. Almost all CD patients possess human leukocyte antigen (HLA) DQ2/DQ8 haplotypes; however, only a small subset of individuals carrying these alleles develop CD, indicating the role of environmental factors in CD pathogenesis. The main objective of this study was to determine the contributory role of gut microbiota and microbial metabolites in CD onset. To this end, we obtained fecal samples from a prospective cohort study (ABIS) at ages 2.5 and 5 years. Samples were collected from children who developed CD after the final sample collection (CD progressors) and healthy children matched by age, HLA genotype, breastfeeding duration, and gluten-exposure time (n=15-16). We first used 16S sequencing and immunoglobulin-A sequencing (IgA-seq) using fecal samples obtained from the same children (i) 16 controls and 15 CD progressors at age 2.5 and (ii) 13 controls and 9 CD progressors at age 5. We completed the cytokine profiling, and plasma metabolomics using plasma samples obtained at age 5 (n=7-9). We also determined the effects of one microbiota-derived metabolite, taurodeoxycholic acid (TDCA), on the small intestines and immune cell composition in vivo.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01429-2)

10.1186/s40168-022-01429-2

---

## Multi-omics profiles of the intestinal microbiome in irritable bowel syndrome and its bowel habit subtypes.
 12.01.2023, _JP Jacobs, V Lagishetty, MC Hauer, JS Labus, TS Dong, R Toma, M Vuyisich, BD Naliboff, JM Lackner, A Gupta, K Tillisch, EA Mayer_


_Humans, Gastrointestinal Microbiome, Irritable Bowel Syndrome, Multiomics, RNA, Ribosomal, 16S, Feces, Habits_
Irritable bowel syndrome (IBS) is a common gastrointestinal disorder that is thought to involve alterations in the gut microbiome, but robust microbial signatures have been challenging to identify. As prior studies have primarily focused on composition, we hypothesized that multi-omics assessment of microbial function incorporating both metatranscriptomics and metabolomics would further delineate microbial profiles of IBS and its subtypes.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01450-5)

10.1186/s40168-022-01450-5

---

## Microbiome Features Differentiating Unsupervised-Stratification-Based Clusters of Patients with Abnormal Glycometabolism.
 18.01.2023, _T Xu, X Wang, Y Chen, H Li, L Zhao, X Ding, C Zhang_


The alteration of gut microbiota structure plays a pivotal role in the pathogenesis of abnormal glycometabolism. However, the microbiome features identified in patient groups stratified solely based on glucose levels remain controversial among different studies. In this study, we stratified 258 participants (discovery cohort) into three clusters according to an unsupervised method based on 16 clinical parameters involving the levels of blood glucose, insulin, and lipid. We found 67 cluster-specific microbiome features (i.e., amplicon sequence variants [ASVs]) based on 16S rRNA gene V3-V4 region sequencing. Specifically, ASVs belonging to Barnesville and Alistipes were enriched in cluster 1, in which participants had the lowest blood glucose levels, high insulin sensitivity, and a high-fecal short-chain fatty acid concentration. ASVs belonging to Prevotella copri and Ruminococcus gnavus were enriched in cluster 2, which was characterized by a moderate level of blood glucose, serious insulin resistance, and high levels of cholesterol and triglyceride. Cluster 3 was characterized by a high level of blood glucose and insulin deficiency, enriched with ASVs in P. copri and Bacteroides vulgatus. In addition, machine learning classifiers using the 67 cluster-specific ASVs were used to distinguish individuals in one cluster from those in the other two clusters both in discovery and testing cohorts (n = 83). Therefore, microbiome features identified based on the unsupervised stratification of patients with more inclusive clinical parameters may better reflect microbiota alterations associated with the progression of abnormal glycometabolism. IMPORTANCE The gut microbiota is altered in patients with type 2 diabetes (T2D) and prediabetes. The association of particular bacteria with T2D, however, varied among studies, which has made it challenging to develop precision medicine approaches for the prevention and alleviation of T2D. Blood glucose level is the only parameter in clustering patients when identifying the T2D-related bacteria in previous studies. This stratification ignores the fact that patients within the same blood glucose range differ in their insulin resistance and dyslipidemia, which also may be related to disordered gut microbiota. In addition to parameters of blood glucose levels, we also used additional parameters involving insulin and lipid levels to stratify participants into three clusters and further identified cluster-specific microbiome features. We further validated the association between these microbiome features and glycometabolism with an independent cohort. This study highlights the importance of stratification of patients with blood glucose, insulin, and lipid levels when identifying the microbiome features associated with the progression of abnormal glycometabolism.

[Source](https://journals.asm.org/doi/10.1128/mbio.03487-22)

10.1128/mbio.03487-22

---

## Gut Bacterial Dysbiosis in Irritable Bowel Syndrome: a Case-Control Study and a Cross-Cohort Analysis Using Publicly Available Data Sets

[Source](https://journals.asm.org/doi/10.1128/spectrum.02125-22)

---

## Harnessing gut microbes for glycan detection and quantification.
 19.01.2023, _JL Modesto, VH Pearce, GE Townsend_


_Animals, Humans, Gastrointestinal Microbiome, Polysaccharides, Glycomics, Mammals_
Glycans facilitate critical biological functions and control the mammalian gut microbiota composition by supplying differentially accessible nutrients to distinct microbial subsets. Therefore, identifying unique glycan substrates that support defined microbial populations could inform therapeutic avenues to treat diseases via modulation of the gut microbiota composition and metabolism. However, examining heterogeneous glycan mixtures for individual microbial substrates is hindered by glycan structural complexity and diversity, which presents substantial challenges to glycomics approaches. Fortuitously, gut microbes encode specialized sensor proteins that recognize unique glycan structures and in-turn activate predictable, specific, and dynamic transcriptional responses. Here, we harness this microbial machinery to indicate the presence and abundance of compositionally similar, yet structurally distinct glycans, using a transcriptional reporter we develop. We implement these tools to examine glycan mixtures, isolate target molecules for downstream characterization, and quantify the recovered products. We assert that this toolkit could dramatically enhance our understanding of the mammalian intestinal environment and identify host-microbial interactions critical for human health.

[Source](https://www.nature.com/articles/s41467-022-35626-2)

10.1038/s41467-022-35626-2

---

## The gut-microbiota-brain axis in a Spanish population in the aftermath of the COVID-19 pandemic: microbiota composition linked to anxiety, trauma, and depression profiles

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2022.2162306)

---

## Expanded catalogue of metagenome-assembled genomes reveals resistome characteristics and athletic performance-associated microbes in horse.
 13.01.2023, _C Li, X Li, R Guo, W Ni, K Liu, Z Liu, J Dai, Y Xu, S Abduriyim, Z Wu, Y Zeng, B Lei, Y Zhang, Y Wang, W Zeng, Q Zhang, C Chen, J Qiao, C Liu, S Hu_


_Horses, Humans, Animals, Metagenome, Genome, Bacterial, Gastrointestinal Microbiome, Drug Resistance, Microbial, Athletic Performance, Metagenomics_
As a domesticated species vital to humans, horses are raised worldwide as a source of mechanical energy for sports, leisure, food production, and transportation. The gut microbiota plays an important role in the health, diseases, athletic performance, and behaviour of horses.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01448-z)

10.1186/s40168-022-01448-z

---

## The bovine foot skin microbiota is associated with host genotype and the development of infectious digital dermatitis lesions.
 12.01.2023, _V Bay, A Gillespie, E Ganda, NJ Evans, SD Carter, L Lenzi, A Lucaci, S Haldenby, M Barden, BE Griffiths, E Sánchez-Molano, R Bicalho, G Banos, A Darby, G Oikonomou_


_Female, Cattle, Animals, Digital Dermatitis, RNA, Ribosomal, 16S, Genome-Wide Association Study, Cattle Diseases, Communicable Diseases, Microbiota, Genotype_
Bovine Digital Dermatitis (BDD) is a prevalent infectious disease, causing painful foot skin lesions and lameness in cattle. We describe herein the bovine foot skin microbiota and its associations with BDD using 16S rRNA gene amplicon and shotgun metagenomic sequencing on samples from 259 dairy cows from three UK dairy farms.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01440-7)

10.1186/s40168-022-01440-7

---

## A framework for the targeted recruitment of crop-beneficial soil taxa based on network analysis of metagenomics data.
 12.01.2023, _M Berihu, TS Somera, A Malik, S Medina, E Piombo, O Tal, M Cohen, A Ginatt, M Ofek-Lalzar, A Doron-Faigenboim, M Mazzola, S Freilich_


The design of ecologically sustainable and plant-beneficial soil systems is a key goal in actively manipulating root-associated microbiomes. Community engineering efforts commonly seek to harness the potential of the indigenous microbiome through substrate-mediated recruitment of beneficial members. In most sustainable practices, microbial recruitment mechanisms rely on the application of complex organic mixtures where the resources/metabolites that act as direct stimulants of beneficial groups are not characterized. Outcomes of such indirect amendments are unpredictable regarding engineering the microbiome and achieving a plant-beneficial environment.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01438-1)

10.1186/s40168-022-01438-1

---

## Diversity and functional analysis of rumen and fecal microbial communities associated with dietary changes in crossbreed dairy cattle.
 13.01.2023, _FM Kibegwa, RC Bett, CK Gachuiri, E Machuka, F Stomeo, FD Mujibi_


The objective of this study was to investigate the effect of varying roughage and concentrate proportions, in diet of crossbreed dairy cattle, on the composition and associated functional genes of rumen and fecal microbiota. We also explored fecal samples as a proxy for rumen liquor samples. Six crossbred dairy cattle were reared on three diets with an increasing concentrate and reducing roughage amount in three consecutive 10-day periods. After each period, individual rumen liquor and fecal samples were collected and analyzed through shotgun metagenomic sequencing. Average relative abundance of identified Operational Taxonomic Units (OTU) and microbial functional roles from all animals were compared between diets and sample types (fecal and rumen liquor). Results indicated that dietary modifications significantly affected several rumen and fecal microbial OTUs. In the rumen, an increase in dietary concentrate resulted in an upsurge in the abundance of Proteobacteria, while reducing the proportions of Bacteroidetes and Firmicutes. Conversely, changes in microbial composition in fecal samples were not consistent with dietary modification patterns. Microbial functional pathway classification identified that carbohydrate metabolism and protein metabolism pathways dominated microbial roles. Assessment of dietary effects on the predicted functional roles of these microbiota revealed that a high amount of dietary concentrate resulted in an increase in central carbohydrate metabolism and a corresponding reduction in protein synthesis. Moreover, we identified several microbial stress-related responses linked to dietary changes. Bacteroides and Clostridium genera were the principal hosts of these microbial functions. Therefore, the roughage to concentrate proportion has more influence on the microbial composition and microbial functional genes in rumen samples than fecal samples. As such, we did not establish a significant relationship between the rumen and fecal metagenome profiles, and the rumen and fecal microbiota from one animal did not correlate more than those from different animals.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274371)

10.1371/journal.pone.0274371

---

## Multi-amplicon microbiome data analysis pipelines for mixed orientation sequences using QIIME2: Assessing reference database, variable region and pre-processing bias in classification of mock bacterial community samples.
 13.01.2023, _KA Maki, B Wolff, L Varuzza, SJ Green, JJ Barb_


Microbiome research relies on next-generation sequencing and on downstream data analysis workflows. Several manufacturers have introduced multi-amplicon kits for microbiome characterization, improving speciation, but present unique challenges for analysis. The goal of this methodology study was to develop two analysis pipelines specific to mixed-orientation reads from multi-hypervariable (V) region amplicons. A secondary aim was to assess agreement with expected abundance, considering database and variable region. Mock community sequence data (n = 41) generated using the Ion16S™ Metagenomics Kit and Ion Torrent Sequencing Platform were analyzed using two workflows. Amplicons from V2, V3, V4, V6-7, V8 and V9 were deconvoluted using a specialized plugin based on CutPrimers. A separate workflow using Cutadapt is also presented. Three reference databases (Ribosomal Database Project, Greengenes and Silva) were used for taxonomic assignment. Bray-Curtis, Euclidean and Jensen-Shannon distance measures were used to evaluate overall annotation consistency, and specific taxon agreement was determined by calculating the ratio of observed to expected relative abundance. Reads that mapped to regions V2-V9 varied for both CutPrimers and Cutadapt-based methods. Within the CutPrimers-based pipeline, V3 amplicons had the best agreement with the expected distribution, tested using global distance measures, while V9 amplicons had the worst agreement. Accurate taxonomic annotation varied by genus-level taxon and V region analyzed. For the first time, we present a microbiome analysis pipeline that employs a specialized plugin to allow microbiome researchers to separate multi-amplicon data from the Ion16S Metagenomics Kit into V-specific reads. We also present an additional analysis workflow, modified for Ion Torrent mixed orientation reads. Overall, the global agreement of amplicons with the expected mock community abundances differed across V regions and reference databases. Benchmarking data should be referenced when planning a microbiome study to consider these biases related to sequencing and data analysis for multi-amplicon sequencing kits.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0280293)

10.1371/journal.pone.0280293

---

