## Global, distinctive, and personal changes in molecular and microbial profiles by specific fibers in humans.
 10.06.2022, _SM Lancaster, B Lee-McMullen, CW Abbott, JV Quijada, D Hornburg, H Park, D Perelman, DJ Peterson, M Tang, A Robinson, S Ahadi, K Contrepois, CJ Hung, M Ashland, T McLaughlin, A Boonyanit, A Horning, JL Sonnenburg, MP Snyder_


Dietary fibers act through the microbiome to improve cardiovascular health and prevent metabolic disorders and cancer. To understand the health benefits of dietary fiber supplementation, we investigated two popular purified fibers, arabinoxylan (AX) and long-chain inulin (LCI), and a mixture of five fibers. We present multiomic signatures of metabolomics, lipidomics, proteomics, metagenomics, a cytokine panel, and clinical measurements on healthy and insulin-resistant participants. Each fiber is associated with fiber-dependent biochemical and microbial responses. AX consumption associates with a significant reduction in LDL and an increase in bile acids, contributing to its observed cholesterol reduction. LCI is associated with an increase in Bifidobacterium. However, at the highest LCI dose, there is increased inflammation and elevation in the liver enzyme alanine aminotransferase. This study yields insights into the effects of fiber supplementation and the mechanisms behind fiber-induced cholesterol reduction, and it shows effects of individual, purified fibers on the microbiome.

[Source](https://doi.org/10.1016/j.chom.2022.03.036)

10.1016/j.chom.2022.03.036

---

## Stitching together a healthy gut microbiome with fiber.
 09.06.2022, _TM Schmidt_


Inter-individual variability in the gut microbiome confounds efforts to understand host responses to dietary fiber. In this issue of Cell Host &amp; Microbe, Lancaster et al. report individual and generalized host and microbiome responses in an interventional study of fiber supplements, motivating consideration of an alternative classification of fiber.

[Source](https://doi.org/10.1016/j.chom.2022.05.007)

10.1016/j.chom.2022.05.007

---

## Maternal gut microbiome-induced IgG regulates neonatal gut microbiome and immunity.
 10.06.2022, _KZ Sanidad, M Amir, A Ananthanarayanan, A Singaraju, NB Shiland, HS Hong, N Kamada, N Inohara, G Núñez, MY Zeng_


The gut microbiome elicits antigen-specific immunoglobulin G (IgG) at steady state that cross-reacts to pathogens to confer protection against systemic infection. The role of gut microbiome-specific IgG antibodies in the development of the gut microbiome and immunity against enteric pathogens in early life, however, remains largely undefined. In this study, we show that gut microbiome-induced maternal IgG is transferred to the neonatal intestine through maternal milk via the neonatal Fc receptor and directly inhibits Citrobacter rodentium colonization and attachment to the mucosa. Enhanced neonatal immunity against oral C. rodentium infection was observed after maternal immunization with a gut microbiome-derived IgG antigen, outer membrane protein A, or induction of IgG-inducing gut bacteria. Furthermore, by generating a gene-targeted mouse model with complete IgG deficiency, we demonstrate that IgG knockout neonates are more susceptible to C. rodentium infection and exhibit alterations of the gut microbiome that promote differentiation of interleukin-17A-producing γδ T cells in the intestine, which persist into adulthood and contribute to increased disease severity in a dextran sulfate sodium-induced mouse model of colitis. Together, our studies have defined a critical role for maternal gut microbiome-specific IgG antibodies in promoting immunity against enteric pathogens and shaping the development of the gut microbiome and immune cells in early life.

[Source](https://doi.org/10.1126/sciimmunol.abh3816)

10.1126/sciimmunol.abh3816

---

## Altered Metabolomics in Bipolar Depression With Gastrointestinal Symptoms.
 10.06.2022, _XJ Guo, YB Xiong, Y Jia, XH Cui, WZ Wu, JS Tian, H Yang, Y Ren_


Although gastrointestinal (GI) symptoms are very common in patients with bipolar disorder (BD), Few studies have researched the pathomechanism behind these symptoms. In the present study, we aim at elucidate the pathomechanism of GI symptoms in BD through metabolomic analysis.

[Source](https://doi.org/10.3389/fpsyt.2022.861285)

10.3389/fpsyt.2022.861285

---

## Meta-analysis of mucosal microbiota reveals universal microbial signatures and dysbiosis in gastric carcinogenesis.
 10.06.2022, _C Liu, SK Ng, Y Ding, Y Lin, W Liu, SH Wong, JJ Sung, J Yu_


The consistency of the associations between gastric mucosal microbiome and gastric cancer across studies remained unexamined. We aimed to identify universal microbial signatures in gastric carcinogenesis through a meta-analysis of gastric microbiome from multiple studies. Compositional and ecological profiles of gastric microbes across stages of gastric carcinogenesis were significantly altered. Meta-analysis revealed that opportunistic pathobionts Fusobacterium, Parvimonas, Veillonella, Prevotella and Peptostreptococcus were enriched in GC, while commensals Bifidobacterium, Bacillus and Blautia were depleted in comparison to SG. The co-occurring correlation strengths of GC-enriched bacteria were increased along disease progression while those of GC-depleted bacteria were decreased. Eight bacterial taxa, including Veillonella, Dialister, Granulicatella, Herbaspirillum, Comamonas, Chryseobacterium, Shewanella and Helicobacter, were newly identified by this study as universal biomarkers for robustly discriminating GC from SG, with an area under the curve (AUC) of 0.85. Moreover, H. pylori-positive samples exhibited reduced microbial diversity, altered microbiota community and weaker interactions among gastric microbes. Our meta-analysis demonstrated comprehensive and generalizable gastric mucosa microbial features associated with histological stages of gastric carcinogenesis, including GC associated bacteria, diagnostic biomarkers, bacterial network alteration and H. pylori influence.

[Source](https://doi.org/10.1038/s41388-022-02377-9)

10.1038/s41388-022-02377-9

---

## <em>Gardnerella vaginalis</em> induces matrix metalloproteinases in the cervicovaginal epithelium through TLR-2 activation

[Source](https://doi.org/10.1016/j.jri.2022.103648)

---

## <em>Lactobacillus rhamnosus</em> colonisation antagonizes <em>Candida albicans</em> by forcing metabolic adaptations that compromise pathogenicity

[Source](https://doi.org/10.1038/s41467-022-30661-5)

---

## Honeys with anti-inflammatory capacity can alter the elderly gut microbiota in an <em>ex vivo</em> gut model

[Source](https://doi.org/10.1016/j.foodchem.2022.133229)

---

## Human colon cancer-derived <em>Clostridioides difficile</em> strains drive colonic tumorigenesis in mice

[Source](https://aacrjournals.org/cancerdiscovery/article/doi/10.1158/2159-8290.CD-21-1273/700781/)

---

## Variation in gut microbiome structure across the annual hibernation cycle in a wild primate

[Source](https://doi.org/10.1093/femsec/fiac070)

---

## A high-throughput DNA sequencing study of fecal bacteria of seven Mexican horse breeds

[Source](https://doi.org/10.1007/s00203-022-03009-2)

---

## Species- and site-specific circulating bacterial DNA in Subantarctic sentinel mussels <em>Aulacomya atra</em> and <em>Mytilus platensis</em>

[Source](https://doi.org/10.1038/s41598-022-13774-1)

---

## The unsung roles of microbial secondary metabolite effectors in the plant disease cacophony

[Source](https://doi.org/10.1016/j.pbi.2022.102233)

---

## Plant-microbe interactions in the rhizosphere via a circular metabolic economy

[Source](https://doi.org/10.1093/plcell/koac163)

---

## Phyllosphere bacterial and fungal communities vary with host species identity, plant traits and seasonality in a subtropical forest

[Source](https://doi.org/10.1186/s40793-022-00423-3)

---

## Nitrogen availability determines plant growth promotion and the induction of root branching by the probiotic fungus <em>Trichoderma atroviride</em> in <em>Arabidopsis</em> seedlings

[Source](https://doi.org/10.1007/s00203-022-03004-7)

---

## Antibiotic resistant bacteria and genes in wastewater treatment plants: From occurrence to treatment strategies.
 09.06.2022, _S Li, BS Ondon, SH Ho, J Jiang, F Li_


This study aims to discuss the following: (1) occurrence and proliferation of antibiotic resistance in wastewater treatment plants (WWTPs); (2) factors influencing antibiotic resistance bacteria and genes in WWTPs; (3) tools to assess antibiotic resistance in WWTPs; (4) environmental contamination of antibiotic resistant bacteria (ARB) and antibiotic resistance genes (ARGs) from WWTPs; (5) effects of ARB and ARGs from WWTPs on human health; and (6) treatment strategies. In general, resistant and multi-resistant bacteria, including Enterobacteriaceae, Pseudomonas aeruginosa, and Escherichia coli, exist in various processes of WWTPs. The existence of ARB and ARGs results from the high concentration of antibiotics in wastewater, which promote selective pressures on the local bacteria present in WWTPs. Thus, improving wastewater treatment technology and avoiding the misuse of antibiotics is critical to overcoming the threat of proliferation of ARBs and ARGs. Numerous factors can affect the development of ARB and ARGs in WWTPs. Abiotic factors can affect the bacterial community dynamics, thereby, affecting the applicability of ARB during the wastewater treatment process. Furthermore, the organic loads and other nutrients influence bacterial survival and growth. Specifically, molecular methods for the rapid characterization and detection of ARBs or their genes comprise DNA sequencing, real-time PCR, simple and multiplex PCR, and hybridization-based technologies, including micro- and macro-arrays. The reuse of effluent from WWTPs for irrigation is an efficient method to overcome water scarcity. However, there are also some potential environmental risks associated with this practice, such as increase in the levels of antibiotic resistance in the soil microbiome. Human mortality rates may significantly increase, as ARB can lead to resistance among several types of antibiotics or longer treatment times. Some treatment technologies, such as anaerobic and aerobic treatment, coagulation, membrane bioreactors, and disinfection processes, are considered potential techniques to restrict antibiotic resistance in the environment.

[Source](https://doi.org/10.1016/j.scitotenv.2022.156544)

10.1016/j.scitotenv.2022.156544

---

## Lessons From Insect Fungiculture: From Microbial Ecology to Plastics Degradation.
 10.06.2022, _MO Barcoto, A Rodrigues_


Anthropogenic activities have extensively transformed the biosphere by extracting and disposing of resources, crossing boundaries of planetary threat while causing a global crisis of waste overload. Despite fundamental differences regarding structure and recalcitrance, lignocellulose and plastic polymers share physical-chemical properties to some extent, that include carbon skeletons with similar chemical bonds, hydrophobic properties, amorphous and crystalline regions. Microbial strategies for metabolizing recalcitrant polymers have been selected and optimized through evolution, thus understanding natural processes for lignocellulose modification could aid the challenge of dealing with the recalcitrant human-made polymers spread worldwide. We propose to look for inspiration in the charismatic fungal-growing insects to understand multipartite degradation of plant polymers. Independently evolved in diverse insect lineages, fungiculture embraces passive or active fungal cultivation for food, protection, and structural purposes. We consider there is much to learn from these symbioses, in special from the community-level degradation of recalcitrant biomass and defensive metabolites. Microbial plant-degrading systems at the core of insect fungicultures could be promising candidates for degrading synthetic plastics. Here, we first compare the degradation of lignocellulose and plastic polymers, with emphasis in the overlapping microbial players and enzymatic activities between these processes. Second, we review the literature on diverse insect fungiculture systems, focusing on features that, while supporting insects' ecology and evolution, could also be applied in biotechnological processes. Third, taking lessons from these microbial communities, we suggest multidisciplinary strategies to identify microbial degraders, degrading enzymes and pathways, as well as microbial interactions and interdependencies. Spanning from multiomics to spectroscopy, microscopy, stable isotopes probing, enrichment microcosmos, and synthetic communities, these strategies would allow for a systemic understanding of the fungiculture ecology, driving to application possibilities. Detailing how the metabolic landscape is entangled to achieve ecological success could inspire sustainable efforts for mitigating the current environmental crisis.

[Source](https://doi.org/10.3389/fmicb.2022.812143)

10.3389/fmicb.2022.812143

---

## Insights into plastic biodegradation: community composition and functional capabilities of the superworm (<em>Zophobas morio</em>) microbiome in styrofoam feeding trials

[Source](https://doi.org/10.1099/mgen.0.000842)

---

## DNA metabarcoding across disciplines: sequencing our way to greater understanding across scales of biological organization.
 10.06.2022, _AM Forsman, AE Savage, BD Hoenig, MR Gaither_


DNA metabarcoding describes the use of targeted DNA (i.e., amplicon) sequencing to identify community constituents from a complex sample containing genetic material from multiple organisms, such as water, soil, gut contents, microbiomes, or biofilms. This molecular approach for characterizing mixed DNA samples relies on the development of "universal primers" that allow for effective amplification of target sequences across a broad range of taxa. Armed with optimized lab protocols and rigorous bioinformatics tools, DNA metabarcoding can produce a wealth of information about the hidden biodiversity of various sample types by probing for organisms' molecular footprints. DNA metabarcoding has received considerable popular press over the last few years because of gut microbiome studies in humans and beyond. However, there are many other applications that are continually integrating molecular biology with other fields of study to address questions that have previously been unanswerable, for both prokaryotic and eukaryotic targets. For example, we can now sample mostly-digested gut contents from virtually any organism to learn about ontogeny and foraging ecology. Water samples collected from different locations can be filtered to extract eDNA (i.e., environmental DNA), revealing the biodiversity of fishes and other taxa targeted by carefully selected primer sets. This universal primer metabarcoding approach has even been extended to looking at diverse gene families within single species, which is particularly useful for complex immune system genetics. The purpose of this SICB symposium was to bring together researchers using DNA metabarcoding approaches to (a) showcase the diversity of applications of this technique for addressing questions spanning ecology, evolution, and physiology, and (b) to spark connections among investigators from different fields that are utilizing similar approaches to facilitate optimization and standardization of metabarcoding methods and analyses. The resulting manuscripts from this symposium represent a great diversity of metabarcoding applications and taxonomic groups of interest.

[Source](https://doi.org/10.1093/icb/icac090)

10.1093/icb/icac090

---

## Systematic cross-biospecimen evaluation of DNA extraction kits for long- and short-read multi-metagenomic sequencing studies

[Source](https://doi.org/10.1016/j.gpb.2022.05.006)

---

