## a regularly-updated collection of FREE, ONLINE microbiome events

[Source](https://twitter.com/svetlana_up/status/1404769552551104514)

---

## #PhDchat

[Source](https://twitter.com/hashtag/PhDchat?src=hash&ref_src=twsrc%5Etfw)

---

## @AcademicChatter

[Source](https://twitter.com/AcademicChatter?ref_src=twsrc%5Etfw)

---

## https://t.co/TeQjcaaKg5

[Source](https://t.co/TeQjcaaKg5)

---

## July 14, 2021

[Source](https://twitter.com/Prokaryota/status/1415343343278690304?ref_src=twsrc%5Etfw)

---

## Prospective correlation between the patient microbiome with response to and development of immune-mediated adverse effects to immunotherapy in lung cancer

[Source](https://link.springer.com/epdf/10.1186/s12885-021-08530-z)

---

## Probiotics impact the antibiotic resistance gene reservoir along the human GI tract in a person-specific and antibiotic-dependent manner

[Source](https://www.nature.com/articles/s41564-021-00920-0)

---

## @Elinav_Lab

[Source](https://twitter.com/Elinav_Lab?ref_src=twsrc%5Etfw)

---

## @Elinav_DKFZ

[Source](https://twitter.com/Elinav_DKFZ?ref_src=twsrc%5Etfw)

---

## @NatureMicrobiol

[Source](https://twitter.com/NatureMicrobiol?ref_src=twsrc%5Etfw)

---

## @LabMihar

[Source](https://twitter.com/LabMihar?ref_src=twsrc%5Etfw)

---

## @rvaldesmas

[Source](https://twitter.com/rvaldesmas?ref_src=twsrc%5Etfw)

---

## @SuezLab

[Source](https://twitter.com/SuezLab?ref_src=twsrc%5Etfw)

---

## https://t.co/LJ1efUaiNI

[Source](https://t.co/LJ1efUaiNI)

---

## July 5, 2021

[Source](https://twitter.com/Elinav_Lab/status/1412078883680272384?ref_src=twsrc%5Etfw)

---

## Identification and engraftment of new bacterial strains by shotgun metagenomic sequence analysis in patients with recurrent <em>Clostridioides difficile </em>infection before and after fecal microbiota transplantation and in healthy human subjects 

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0251590)

---

## Differences in the individual curative effect of acupuncture for obese women with polycystic ovary syndrome based on metagenomic analysis: study protocol for a randomized controlled trial 

[Source](https://trialsjournal.biomedcentral.com/articles/10.1186/s13063-021-05426-y)

---

## Comparative Analysis of 16S rRNA Gene and Metagenome Sequencing in Pediatric Gut Microbiomes 

[Source](https://www.readcube.com/articles/10.3389/fmicb.2021.670336)

---

## #drawing

[Source](https://twitter.com/hashtag/drawing?src=hash&ref_src=twsrc%5Etfw)

---

## #bacteria

[Source](https://twitter.com/hashtag/bacteria?src=hash&ref_src=twsrc%5Etfw)

---

## #gut

[Source](https://twitter.com/hashtag/gut?src=hash&ref_src=twsrc%5Etfw)

---

## #sciart

[Source](https://twitter.com/hashtag/sciart?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/A6yBV0RCfY

[Source](https://t.co/A6yBV0RCfY)

---

## June 28, 2021

[Source](https://twitter.com/enigrox/status/1409533734173528064?ref_src=twsrc%5Etfw)

---

## Spread of Carbapenem-Resistant <em>Klebsiella pneumoniae </em>in an Intensive Care Unit: A Whole-Genome Sequence-Based Prospective Observational Study

[Source](https://journals.asm.org/doi/epdf/10.1128/Spectrum.00058-21)

---

## Linezolid- and multidrug-resistant enterococci in raw commercial dog food, Europe, 2019–2020 

[Source](https://wwwnc.cdc.gov/eid/article/27/8/20-4933_article)

---

## Identification of pathogen(s) in infectious diseases using shotgun metagenomic sequencing and conventional culture: a comparative study

[Source](https://peerj.com/articles/11699/)

---

## Predicting antibiotic resistance gene abundance in activated sludge using shotgun metagenomics and machine learning

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0043135421005820)

---

## Assessing the relationship between the rumen microbiota and feed efficiency in Nellore steers.
 18.07.2021, _DRG Lopes, M de Souza Duarte, AJ La Reau, IZ Chaves, TA de Oliveira Mendes, E Detmann, CBP Bento, MEZ Mercadante, SFM Bonilha, G Suen, HC Mantovani_


Ruminants rely upon a complex community of microbes in their rumen to convert host-indigestible feed into nutrients. However, little is known about the association between the rumen microbiota and feed efficiency traits in Nellore (Bos indicus) cattle, a breed of major economic importance to the global beef market. Here, we compare the composition of the bacterial, archaeal and fungal communities in the rumen of Nellore steers with high and low feed efficiency (FE) phenotypes, as measured by residual feed intake (RFI).

[Source](https://jasbsci.biomedcentral.com/articles/10.1186/s40104-021-00599-7)

10.1186/s40104-021-00599-7

---

## An osmotic laxative renders mice susceptible to prolonged <em>Clostridioides difficile </em>colonization and hinders clearance

[Source](https://www.biorxiv.org/content/10.1101/2021.07.13.452287v1)

---

## Tutorial: assessing metagenomics software with the CAMI benchmarking toolkit.
 18.05.2021, _F Meyer, TR Lesker, D Koslicki, A Fritz, A Gurevich, AE Darling, A Sczyrba, A Bremges, AC McHardy_


_Animals, Benchmarking, Computer Simulation, Databases, Genetic, Gastrointestinal Microbiome, Metagenome, Metagenomics, Mice, Phylogeny, Reference Standards, Reproducibility of Results, Software_
Computational methods are key in microbiome research, and obtaining a quantitative and unbiased performance estimate is important for method developers and applied researchers. For meaningful comparisons between methods, to identify best practices and common use cases, and to reduce overhead in benchmarking, it is necessary to have standardized datasets, procedures and metrics for evaluation. In this tutorial, we describe emerging standards in computational meta-omics benchmarking derived and agreed upon by a larger community of researchers. Specifically, we outline recent efforts by the Critical Assessment of Metagenome Interpretation (CAMI) initiative, which supplies method developers and applied researchers with exhaustive quantitative data about software performance in realistic scenarios and organizes community-driven benchmarking challenges. We explain the most relevant evaluation metrics for assessing metagenome assembly, binning and profiling results, and provide step-by-step instructions on how to generate them. The instructions use simulated mouse gut metagenome data released in preparation for the second round of CAMI challenges and showcase the use of a repository of tool results for CAMI datasets. This tutorial will serve as a reference for the community and facilitate informative and reproducible benchmarking in microbiome research.

[Source](https://www.biorxiv.org/content/10.1101/2021.07.12.451567v1)

10.1038/s41596-020-00480-3

---

## Critical Assessment of Metagenome Interpretation – the second round of challenges

[Source](https://www.biorxiv.org/content/10.1101/2021.07.12.451567v1)

---

## Disentangling the mechanisms shaping the surface ocean microbiota.
 04.02.2021, _R Logares, IM Deutschmann, PC Junger, CR Giner, AK Krabberød, TSB Schmidt, L Rubinat-Ripoll, M Mestre, G Salazar, C Ruiz-González, M Sebastián, C de Vargas, SG Acinas, CM Duarte, JM Gasol, R Massana_


_Archaea, Bacteria, Eukaryota, Microbiota, Oceans and Seas, Phylogeography, Plankton, RNA, Ribosomal, 16S, RNA, Ribosomal, 18S, Spatial Analysis, Temperature, Water Microbiology_
The ocean microbiota modulates global biogeochemical cycles and changes in its configuration may have large-scale consequences. Yet, the underlying ecological mechanisms structuring it are unclear. Here, we investigate how fundamental ecological mechanisms (selection, dispersal and ecological drift) shape the smallest members of the tropical and subtropical surface-ocean microbiota: prokaryotes and minute eukaryotes (picoeukaryotes). Furthermore, we investigate the agents exerting abiotic selection on this assemblage as well as the spatial patterns emerging from the action of ecological mechanisms. To explore this, we analysed the composition of surface-ocean prokaryotic and picoeukaryotic communities using DNA-sequence data (16S- and 18S-rRNA genes) collected during the circumglobal expeditions Malaspina-2010 and TARA-Oceans.

[Source](https://www.biorxiv.org/content/10.1101/2021.07.13.452182v1)

10.1186/s40168-020-00827-8

---

## Disentangling environmental effects in microbial association networks

[Source](https://www.biorxiv.org/content/10.1101/2021.07.13.452182v1)

---

## Microbiome Virtual International Conference #MVIF

[Source](https://www.microbiome-vif.org/)

---

## @RuthLeyMicro

[Source](https://twitter.com/RuthLeyMicro?ref_src=twsrc%5Etfw)

---

## https://t.co/PHH4kULBJH

[Source](https://t.co/PHH4kULBJH)

---

## pic.twitter.com/Djq9xp3Iaj

[Source](https://t.co/Djq9xp3Iaj)

---

## July 14, 2021

[Source](https://twitter.com/MicrobiomeVIF/status/1415218080016330752?ref_src=twsrc%5Etfw)

---

