## Opportunities and challenges of using metagenomic data to bring uncultured microbes into cultivation.
 17.05.2022, _S Liu, CD Moon, N Zheng, S Huws, S Zhao, J Wang_


_Bacteria, Genomics, Metagenome, Metagenomics, Microbiota_
Although there is now an extensive understanding of the diversity of microbial life on earth through culture-independent metagenomic DNA sequence analyses, the isolation and cultivation of microbes remains critical to directly study them and confirm their metabolic and physiological functions, and their ecological roles. The majority of environmental microbes are as yet uncultured however; therefore, bringing these rare or poorly characterized groups into culture is a priority to further understand microbiome functions. Moreover, cultivated isolates may find utility in a range of applications, such as new probiotics, biocontrol agents, and agents for industrial processes. The growing abundance of metagenomic and meta-transcriptomic sequence information from a wide range of environments provides more opportunities to guide the isolation and cultivation of microbes of interest. In this paper, we discuss a range of successful methodologies and applications that have underpinned recent metagenome-guided isolation and cultivation of microbe efforts. These approaches include determining specific culture conditions to enrich for taxa of interest, to more complex strategies that specifically target the capture of microbial species through antibody engineering and genome editing strategies. With the greater degree of genomic information now available from uncultivated members, such as via metagenome-assembled genomes, the theoretical understanding of their cultivation requirements will enable greater possibilities to capture these and ultimately gain a more comprehensive understanding of the microbiomes. Video Abstract.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01272-5)

10.1186/s40168-022-01272-5

---

## Deep learning of a bacterial and archaeal universal language of life enables transfer learning and illuminates microbial dark matter.
 17.05.2022, _A Hoarfrost, A Aptekmann, G Farfañuk, Y Bromberg_


_Archaea, Bacteria, Deep Learning, Genome, Archaeal, Genome, Bacterial, Proteins, Sequence Analysis_
The majority of microbial genomes have yet to be cultured, and most proteins identified in microbial genomes or environmental sequences cannot be functionally annotated. As a result, current computational approaches to describe microbial systems rely on incomplete reference databases that cannot adequately capture the functional diversity of the microbial tree of life, limiting our ability to model high-level features of biological sequences. Here we present LookingGlass, a deep learning model encoding contextually-aware, functionally and evolutionarily relevant representations of short DNA reads, that distinguishes reads of disparate function, homology, and environmental origin. We demonstrate the ability of LookingGlass to be fine-tuned via transfer learning to perform a range of diverse tasks: to identify novel oxidoreductases, to predict enzyme optimal temperature, and to recognize the reading frames of DNA sequence fragments. LookingGlass enables functionally relevant representations of otherwise unknown and unannotated sequences, shedding light on the microbial dark matter that dominates life on Earth.

[Source](https://www.nature.com/articles/s41467-022-30070-8)

10.1038/s41467-022-30070-8

---

## Sex differences in global metabolomic profiles of COVID-19 patients.
 19.05.2022, _RD Escarcega, P Honarpisheh, GD Colpo, HW Ahnstedt, L Couture, S Juneja, G Torres, GJ Ortiz, J Sollome, N Tabor, BP Ganesh, HA Choi, F Liu, LD McCullough, AS Tsvetkov_


_Biomarkers, COVID-19, Female, Humans, Male, Metabolomics, SARS-CoV-2, Sex Characteristics_
Coronavirus disease (COVID-19), caused by SARS-CoV-2, leads to symptoms ranging from asymptomatic disease to death. Although males are more susceptible to severe symptoms and higher mortality due to COVID-19, patient sex has rarely been examined. Sex-associated metabolic changes may implicate novel biomarkers and therapeutic targets to treat COVID-19. Here, using serum samples, we performed global metabolomic analyses of uninfected and SARS-CoV-2-positive male and female patients with severe COVID-19. Key metabolic pathways that demonstrated robust sex differences in COVID-19 groups, but not in controls, involved lipid metabolism, pentose pathway, bile acid metabolism, and microbiome-related metabolism of aromatic amino acids, including tryptophan and tyrosine. Unsupervised statistical analysis showed a profound sexual dimorphism in correlations between patient-specific clinical parameters and their global metabolic profiles. Identification of sex-specific metabolic changes in severe COVID-19 patients is an important knowledge source for researchers striving for development of potential sex-associated biomarkers and druggable targets for COVID-19 patients.

[Source](https://www.nature.com/articles/s41419-022-04861-2)

10.1038/s41419-022-04861-2

---

## Does the uterine microbiota affect the reproductive outcomes in women with recurrent implantation failures?
 19.05.2022, _LK Keburiya, VY Smolnikova, TV Priputnevich, VV Muravieva, AB Gordeev, DY Trofimov, ES Shubina, TO Kochetkova, MS Rogacheva, EA Kalinina, GT Sukhikh_


_Embryo Implantation, Embryo Transfer, Female, Fertilization in Vitro, Humans, Microbiota, Pregnancy, Pregnancy Rate_
Inefficiency of in vitro fertilization (IVF) programs can be caused by implantation failures. The uterine microbiota can influence the implantation process. However, it still remains unclear whether opportunistic microorganisms detected in the endometrium have a negative impact on the implantation success. The aim of our study was to evaluate the influence of the uterine microbiota on the embryo implantation success in patients undergoing assisted reproductive technologies.

[Source](https://bmcwomenshealth.biomedcentral.com/articles/10.1186/s12905-022-01750-w)

10.1186/s12905-022-01750-w

---

## Impact of antibiotics on off-target infant gut microbiota and resistance genes in cohort studies.
 14.05.2022, _RM Lebeaux, JC Madan, QP Nguyen, MO Coker, EF Dade, Y Moroishi, TJ Palys, BD Ross, MM Pettigrew, HG Morrison, MR Karagas, AG Hoen_


Young children are frequently exposed to antibiotics, with the potential for collateral consequences to the gut microbiome. The impact of antibiotic exposures to off-target microbes (i.e., bacteria not targeted by treatment) and antibiotic resistance genes (ARGs) is poorly understood.

[Source](https://www.nature.com/articles/s41390-022-02104-w)

10.1038/s41390-022-02104-w

---

## Should we modulate the neonatal microbiome and what should be the goal?
 14.05.2022, _N van Best, MG Dominguez-Bello, MW Hornef, E Jašarević, K Korpela, TD Lawley_


_Gastrointestinal Microbiome, Goals, Microbiota_

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01281-4)

10.1186/s40168-022-01281-4

---

## The respiratory microbiome associated with chronic obstructive pulmonary disease comorbidity in non-small cell lung cancer.
 17.05.2022, _M Shimizu, A Miyanaga, M Seike, K Matsuda, M Matsumoto, R Noro, K Fujita, Y Mano, N Furuya, K Kubota, A Gemma_


Research has shown that some microbiomes are linked to cancer. Hence, we hypothesize that alterations in the respiratory microbiome might be associated with lung cancer.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/1759-7714.14463)

10.1111/1759-7714.14463

---

## SKIOME Project: a curated collection of skin microbiome datasets enriched with study-related metadata.
 19.05.2022, _G Agostinetto, D Bozzi, D Porro, M Casiraghi, M Labra, A Bruno_


_Databases, Factual, Humans, Information Storage and Retrieval, Metadata, Microbiota, RNA, Ribosomal, 16S_
Large amounts of data from microbiome-related studies have been (and are currently being) deposited on international public databases. These datasets represent a valuable resource for the microbiome research community and could serve future researchers interested in integrating multiple datasets into powerful meta-analyses. However, this huge amount of data lacks harmonization and it is far from being completely exploited in its full potential to build a foundation that places microbiome research at the nexus of many subdisciplines within and beyond biology. Thus, it urges the need for data accessibility and reusability, according to findable, accessible, interoperable and reusable (FAIR) principles, as supported by National Microbiome Data Collaborative and FAIR Microbiome. To tackle the challenge of accelerating discovery and advances in skin microbiome research, we collected, integrated and organized existing microbiome data resources from human skin 16S rRNA amplicon-sequencing experiments. We generated a comprehensive collection of datasets, enriched in metadata, and organized this information into data frames ready to be integrated into microbiome research projects and advanced post-processing analyses, such as data science applications (e.g. machine learning). Furthermore, we have created a data retrieval and curation framework built on three different stages to maximize the retrieval of datasets and metadata associated with them. Lastly, we highlighted some caveats regarding metadata retrieval and suggested ways to improve future metadata submissions. Overall, our work resulted in a curated skin microbiome datasets collection accompanied by a state-of-the-art analysis of the last 10 years of the skin microbiome field. Database URL: https://github.com/giuliaago/SKIOMEMetadataRetrieval.

[Source](https://academic.oup.com/database/article/doi/10.1093/database/baac033/6586378)

10.1093/database/baac033

---

## Dynamic microbiome and metabolome analyses reveal the interaction between gut microbiota and anti-PD-1 based immunotherapy in hepatocellular carcinoma.
 17.05.2022, _H Wu, X Zheng, T Pan, X Yang, X Chen, B Zhang, L Peng, C Xie_


Hepatocellular carcinoma (HCC) is a severe disease with high mortality and global incidence. However, the interaction between the gut microbiome and combined immunotherapy for HCC is yet unclear. In this prospective clinical study, patients with unresectable HCC who had not received systemic treatment previously were recruited. Fecal and serum samples were collected at the baseline point and before each subsequent administration as specified. Between October 20, 2019 and February 2, 2021, 61 patients were screened for eligibility, of whom 35 patients were finally included in this study. Alpha diversity of fecal samples from patients who responded to immunotherapy was higher than that of non-responders at baseline. However, the prominent alpha-diversity between responders and non-responders became similar as early as week 6 after treatment. The beta diversity of inter-group did not show significant difference at the 9th week after treatment. Alpha-D-Glucose was the only serum metabolite that differed between the responders and non-responders after 3 months. Responder-enriched Ruminococcus showed a positive correlation with serum galactaric acid, while Klebsiella was positively associated with 3-methylindole and lenticin (all P&lt;0.01). The machine learning classifier based on serum metabolites were more able to discriminate HCC patients who potentially benefited from immunotherapy at baseline (AUC 0.793, 95% CI: 0.632-0.954) than the classifier of gut microbiome. In conclusion, gut microbiome biomarkers are associated with the response to anti-PD-1 based immunotherapy in HCC patients. Classifiers based on gut microbiota and serum metabolites are feasible. This article is protected by copyright. All rights reserved.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/ijc.34118)

10.1002/ijc.34118

---

## Exploring the multifactorial aspects of Gut Microbiome in Parkinson's Disease.
 18.05.2022, _S Pavan, AN Prabhu, S Prasad Gorthi, B Das, A Mutreja, V Shetty, T Ramamurthy, M Ballal_


Advanced research in health science has broadened our view in approaching and understanding the pathophysiology of diseases and has also revolutionised diagnosis and treatment. Ever since the establishment of Braak's hypothesis in the propagation of alpha-synuclein from the distant olfactory and enteric nervous system towards the brain in Parkinson's Disease (PD), studies have explored and revealed the involvement of altered gut microbiota in PD. This review recapitulates the gut microbiome associated with PD severity, duration, motor and non-motor symptoms, and antiparkinsonian treatment from recent literature. Gut microbial signatures in PD are potential predictors of the disease and are speculated to be used in early diagnosis and treatment. In brief, the review also emphasises on implications of the prebiotic, probiotic, faecal microbiota transplantation, and dietary interventions as alternative treatments in modulating the disease symptoms in PD.

[Source](https://link.springer.com/article/10.1007/s12223-022-00977-2)

10.1007/s12223-022-00977-2

---

## Gut microbiota of homing pigeons shows summer-winter variation under constant diet indicating a substantial effect of temperature

[Source](https://www.biorxiv.org/content/10.1101/2022.05.18.492417v1.abstract)

---

## Bumblebees with the socially transmitted microbiome: A novel model organism for gut microbiota research.
 14.05.2022, _ZJ Zhang, H Zheng_


Eusocial bumble and honey bees are important pollinators for global ecology and the agricultural economy. Although both the bumble and honey bees possess similar and host-restricted gut microbiota, they differ in aspects of morphology, autonomy, physiology, behavior, and life cycle. The social bee gut bacteria exhibit host specificity that is likely a result of long-term co-evolution. The unique life cycle of bumblebees is key for the acquisition and development of their gut microbiota, and affects the strain-level diversity of the core bacterial species. Studies on bumblebee gut bacteria show that they retain less functional capacity for carbohydrate metabolism compared with that of the honeybee. We discuss the potential roles of the bumblebee gut microbiota against pathogenic threats and the application of host-specific probiotics for bumblebees. Given the advantages of the bumblebee microbiome, including the simple structure and host specificity, and the ease of manipulating bumblebee colonies, we propose that bumblebees may provide a valuable system for understanding the general principles of host-microbe interactions, gut-brain axis, and vertical transmission.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/1744-7917.13040)

10.1111/1744-7917.13040

---

## Adaptation of gut microbiome and host metabolic systems to lignocellulosic degradation in bamboo rats.
 19.05.2022, _K Xiao, X Liang, H Lu, X Li, Z Zhang, X Lu, H Wang, Y Meng, A Roy, W Luo, X Shen, DM Irwin, Y Shen_


Bamboo rats (Rhizomys pruinosus) are among the few mammals that lives on a bamboo-based diet which is mainly composed of lignocellulose. However, the mechanisms of adaptation of their gut microbiome and metabolic systems in the degradation of lignocellulose are largely unknown. Here, we conducted a multi-omics analysis on bamboo rats to investigate the interaction between their gut microbiomes and metabolic systems in the pre- and post-weaning periods, and observed significant relationships between dietary types, gut microbiome, serum metabolome and host gene expression. For comparison, published gut microbial data from the famous bamboo-eating giant panda (Ailuropoda melanoleuca) were also used for analysis. We found that the adaptation of the gut microbiome of the bamboo rat to a lignocellulose diet is related to a member switch in the order Bacteroidales from family Bacteroidaceae to family Muribaculaceae, while for the famous bamboo-eating giant panda, several aerobes and facultative anaerobes increase after weaning. The conversion of bacteria with an increased relative abundance in bamboo rats after weaning enriched diverse carbohydrate-active enzymes (CAZymes) associated with lignocellulose degradation and functionally enhanced the biosynthesis of amino acids and B vitamins. Meanwhile, the circulating concentration of short-chain fatty acids (SCFAs) derived metabolites and the metabolic capacity of linoleic acid in the host were significantly elevated. Our findings suggest that fatty acid metabolism, including linoleic acid and SCFAs, are the main energy sources for bamboo rats in response to the low-nutrient bamboo diet.

[Source](https://www.nature.com/articles/s41396-022-01247-2)

10.1038/s41396-022-01247-2

---

## Comparative Analysis of Gut Microbiota Between Healthy and Diarrheic Horses.
 19.05.2022, _Y Li, Y Lan, S Zhang, X Wang_


Increasing evidence reveals the importance of gut microbiota in animals for regulating intestinal homeostasis, metabolism, and host health. The gut microbial community has been reported to be closely related to many diseases, but information regarding diarrheic influence on gut microbiota in horses remains scarce. This study investigated and compared gut microbial changes in horses during diarrhea. The results showed that the alpha diversity of gut microbiota in diarrheic horses decreased observably, accompanied by obvious shifts in taxonomic compositions. The dominant bacterial phyla (Firmicutes, Bacteroidetes, Spirochaetes, and Kiritimatiellaeota) and genera (uncultured_bacterium_f_Lachnospiraceae, uncultured_bacterium_f_p-251-o5, Lachnospiraceae_AC2044_group, and Treponema_2) in the healthy and diarrheic horses were same regardless of health status but different in abundances. Compared with the healthy horses, the relative abundances of Planctomycetes, Tenericutes, Firmicutes, Patescibacteria, and Proteobacteria in the diarrheic horses were observably decreased, whereas Bacteroidetes, Verrucomicrobia, and Fibrobacteres were dramatically increased. Moreover, diarrhea also resulted in a significant reduction in the proportions of 31 genera and a significant increase in the proportions of 14 genera. Taken together, this study demonstrated that the gut bacterial diversity and abundance of horses changed significantly during diarrhea. Additionally, these findings also demonstrated that the dysbiosis of gut microbiota may be an important driving factor of diarrhea in horses.

[Source](https://www.frontiersin.org/articles/10.3389/fvets.2022.882423/full)

10.3389/fvets.2022.882423

---

## Large Comparative Analyses of Primate Body Site Microbiomes Indicate that the Oral Microbiome Is Unique among All Body Sites and Conserved among Nonhuman Primates.
 19.05.2022, _AE Asangba, L Mugisha, J Rukundo, RJ Lewis, A Halajian, L Cortés-Ortiz, RE Junge, MT Irwin, J Karlson, A Perkin, M Watsa, G Erkenswick, KL Bales, DL Patton, AJ Jasinska, E Fernandez-Duque, SR Leigh, RM Stumpf_


The study of the mammalian microbiome serves as a critical tool for understanding host-microbial diversity and coevolution and the impact of bacterial communities on host health. While studies of specific microbial systems (e.g., in the human gut) have rapidly increased, large knowledge gaps remain, hindering our understanding of the determinants and levels of variation in microbiomes across multiple body sites and host species. Here, we compare microbiome community compositions from eight distinct body sites among 17 phylogenetically diverse species of nonhuman primates (NHPs), representing the largest comparative study of microbial diversity across primate host species and body sites. Analysis of 898 samples predominantly acquired in the wild demonstrated that oral microbiomes were unique in their clustering, with distinctive divergence from all other body site microbiomes. In contrast, all other body site microbiomes clustered principally by host species and differentiated by body site within host species. These results highlight two key findings: (i) the oral microbiome is unique compared to all other body site microbiomes and conserved among diverse nonhuman primates, despite their considerable dietary and phylogenetic differences, and (ii) assessments of the determinants of host-microbial diversity are relative to the level of the comparison (i.e., intra-/inter-body site, -host species, and -individual), emphasizing the need for broader comparative microbial analyses across diverse hosts to further elucidate host-microbial dynamics, evolutionary and biological patterns of variation, and implications for human-microbial coevolution. IMPORTANCE The microbiome is critical to host health and disease, but much remains unknown about the determinants, levels, and evolution of host-microbial diversity. The relationship between hosts and their associated microbes is complex. Most studies to date have focused on the gut microbiome; however, large gaps remain in our understanding of host-microbial diversity, coevolution, and levels of variation in microbiomes across multiple body sites and host species. To better understand the patterns of variation and evolutionary context of host-microbial communities, we conducted one of the largest comparative studies to date, which indicated that the oral microbiome was distinct from the microbiomes of all other body sites and convergent across host species, suggesting conserved niche specialization within the Primates order. We also show the importance of host species differences in shaping the microbiome within specific body sites. This large, comparative study contributes valuable information on key patterns of variation among hosts and body sites, with implications for understanding host-microbial dynamics and human-microbial coevolution.

[Source](https://journals.asm.org/doi/10.1128/spectrum.01643-21)

10.1128/spectrum.01643-21

---

## Novel Chloroflexi genomes from the deepest ocean reveal metabolic strategies for the adaptation to deep-sea habitats.
 14.05.2022, _R Liu, X Wei, W Song, L Wang, J Cao, J Wu, T Thomas, T Jin, Z Wang, W Wei, Y Wei, H Zhai, C Yao, Z Shen, J Du, J Fang_


_Carbon, Chloroflexi, Ecosystem, Oceans and Seas, Sulfur_
The deep sea harbors the majority of the microbial biomass in the ocean and is a key site for organic matter (OM) remineralization and storage in the biosphere. Microbial metabolism in the deep ocean is greatly controlled by the generally depleted but periodically fluctuating supply of OM. Currently, little is known about metabolic potentials of dominant deep-sea microbes to cope with the variable OM inputs, especially for those living in the hadal trenches-the deepest part of the ocean.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01263-6)

10.1186/s40168-022-01263-6

---

## CBEA: Competitive balances for taxonomic enrichment analysis.
 18.05.2022, _QP Nguyen, AG Hoen, HR Frost_


Research in human-associated microbiomes often involves the analysis of taxonomic count tables generated via high-throughput sequencing. It is difficult to apply statistical tools as the data is high-dimensional, sparse, and compositional. An approachable way to alleviate high-dimensionality and sparsity is to aggregate variables into pre-defined sets. Set-based analysis is ubiquitous in the genomics literature and has demonstrable impact on improving interpretability and power of downstream analysis. Unfortunately, there is a lack of sophisticated set-based analysis methods specific to microbiome taxonomic data, where current practice often employs abundance summation as a technique for aggregation. This approach prevents comparison across sets of different sizes, does not preserve inter-sample distances, and amplifies protocol bias. Here, we attempt to fill this gap with a new single-sample taxon enrichment method that uses a novel log-ratio formulation based on the competitive null hypothesis commonly used in the enrichment analysis literature. Our approach, titled competitive balances for taxonomic enrichment analysis (CBEA), generates sample-specific enrichment scores as the scaled log-ratio of the subcomposition defined by taxa within a set and the subcomposition defined by its complement. We provide sample-level significance testing by estimating an empirical null distribution of our test statistic with valid p-values. Herein, we demonstrate, using both real data applications and simulations, that CBEA controls for type I error, even under high sparsity and high inter-taxa correlation scenarios. Additionally, CBEA provides informative scores that can be inputs to downstream analyses such as prediction tasks.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010091)

10.1371/journal.pcbi.1010091

---

## Clustering compositional data using Dirichlet mixture model.
 19.05.2022, _S Pal, C Heumann_


A model-based clustering method for compositional data is explored in this article. Most methods for compositional data analysis require some kind of transformation. The proposed method builds a mixture model using Dirichlet distribution which works with the unit sum constraint. The mixture model uses a hard EM algorithm with some modification to overcome the problem of fast convergence with empty clusters. This work includes a rigorous simulation study to evaluate the performance of the proposed method over varied dimensions, number of clusters, and overlap. The performance of the model is also compared with other popular clustering algorithms often used for compositional data analysis (e.g. KMeans, Gaussian mixture model (GMM) Gaussian Mixture Model with Hard EM (Hard GMM), partition around medoids (PAM), Clustering Large Applications based on Randomized Search (CLARANS), Density-Based Spatial Clustering of Applications with Noise (DBSCAN) etc.) for simulated data as well as two real data problems coming from the business and marketing domain and physical science domain, respectively. The study has shown promising results exploiting different distributional patterns of compositional data.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0268438)

10.1371/journal.pone.0268438

---

## Generation and application of pseudo-long reads for metagenome assembly.
 19.05.2022, _M Sim, J Lee, S Wy, N Park, D Lee, D Kwon, J Kim_


_High-Throughput Nucleotide Sequencing, Humans, Metagenome, Metagenomics, Microbiota, Sequence Analysis, DNA_
Metagenomic assembly using high-throughput sequencing data is a powerful method to construct microbial genomes in environmental samples without cultivation. However, metagenomic assembly, especially when only short reads are available, is a complex and challenging task because mixed genomes of multiple microorganisms constitute the metagenome. Although long read sequencing technologies have been developed and have begun to be used for metagenomic assembly, many metagenomic studies have been performed based on short reads because the generation of long reads requires higher sequencing cost than short reads.

[Source](https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giac044/6586818)

10.1093/gigascience/giac044

---

## A comparison of six DNA extraction protocols for 16S, ITS, and shotgun metagenomic sequencing of microbial communities

[Source](https://www.biorxiv.org/content/10.1101/2022.03.07.483343v2.abstract)

---

## Estimating Microbial Population Data from Optical Density

[Source](https://www.biorxiv.org/content/10.1101/2021.12.22.473948v2.abstract)

---

