## Human postprandial responses to food and potential for precision nutrition.
 12.06.2020, _SE Berry, AM Valdes, DA Drew, F Asnicar, M Mazidi, J Wolf, J Capdevila, G Hadjigeorgiou, R Davies, H Al Khatib, C Bonnett, S Ganesh, E Bakker, D Hart, M Mangino, J Merino, I Linenberg, P Wyatt, JM Ordovas, CD Gardner, LM Delahanty, AT Chan, N Segata, PW Franks, TD Spector_


Metabolic responses to food influence risk of cardiometabolic disease, but large-scale high-resolution studies are lacking. We recruited n = 1,002 twins and unrelated healthy adults in the United Kingdom to the PREDICT 1 study and assessed postprandial metabolic responses in a clinical setting and at home. We observed large inter-individual variability (as measured by the population coefficient of variation (s.d./mean, %)) in postprandial responses of blood triglyceride (103%), glucose (68%) and insulin (59%) following identical meals. Person-specific factors, such as gut microbiome, had a greater influence (7.1% of variance) than did meal macronutrients (3.6%) for postprandial lipemia, but not for postprandial glycemia (6.0% and 15.4%, respectively); genetic variants had a modest impact on predictions (9.5% for glucose, 0.8% for triglyceride, 0.2% for C-peptide). Findings were independently validated in a US cohort (n = 100 people). We developed a machine-learning model that predicted both triglyceride (r = 0.47) and glycemic (r = 0.77) responses to food intake. These findings may be informative for developing personalized diet strategies. The ClinicalTrials.gov registration identifier is NCT03479866.

[Source](https://www.nature.com/articles/s41591-020-0934-0)

10.1038/s41591-020-0934-0

---

## Shotgun sequencing of honey DNA can describe honey bee derived environmental signatures and the honey bee hologenome complexity.
 10.06.2020, _S Bovo, VJ Utzeri, A Ribani, R Cabbri, L Fontanesi_


Honey bees are large-scale monitoring tools due to their extensive environmental exploration. In their activities and from the hive ecosystem complex, they get in close contact with many organisms whose traces can be transferred into the honey, which can represent an interesting reservoir of environmental DNA (eDNA) signatures and information useful to analyse the honey bee hologenome complexity. In this study, we tested a deep shotgun sequencing approach of honey DNA coupled with a specifically adapted bioinformatic pipeline. This methodology was applied to a few honey samples pointing out DNA sequences from 191 organisms spanning different kingdoms or phyla (viruses, bacteria, plants, fungi, protozoans, arthropods, mammals). Bacteria included the largest number of species. These multi-kingdom signatures listed common hive and honey bee gut microorganisms, honey bee pathogens, parasites and pests, which resembled a complex interplay that might provide a general picture of the honey bee pathosphere. Based on the Apis mellifera filamentous virus genome diversity (the most abundant detected DNA source) we obtained information that could define the origin of the honey at the apiary level. Mining Apis mellifera sequences made it possible to identify the honey bee subspecies both at the mitochondrial and nuclear genome levels.

[Source](https://www.nature.com/articles/s41598-020-66127-1)

10.1038/s41598-020-66127-1

---

## Low Turnover of Soil Bacterial rRNA at Low Temperatures.
 12.06.2020, _MD Schostag, CN Albers, CS Jacobsen, A Priemé_


Ribosomal RNA (rRNA) is used widely to investigate potentially active microorganisms in environmental samples, including soil microorganisms and other microbial communities that are subjected to pronounced seasonal variation in temperature. This raises a question about the turnover of intracellular microbial rRNA at environmentally relevant temperatures. We analyzed the turnover at four temperatures of RNA isolated from soil bacteria amended with 14C-labeled uridine. We found that the half-life of recently produced RNA increased from 4.0 days at 20°C to 15.8 days at 4°C, and 215 days at -4°C, while no degradation was detected at -18°C during a 1-year period. We discuss the implications of the strong temperature dependency of rRNA turnover for interpretation of microbiome data based on rRNA isolated from environmental samples.

[Source](https://doi.org/10.3389/fmicb.2020.00962)

10.3389/fmicb.2020.00962

---

## Diatom Modulation of Microbial Consortia Through Use of Two Unique Secondary Metabolites

[Source](https://doi.org/10.1101/2020.06.11.144840)

---

## Complementary sampling methods for coral histology, metabolomics, and microbiome

[Source](https://doi.org/10.1111/2041-210X.13431)

---

## Phage infection mediates inhibition of bystander bacteria

[Source](https://doi.org/10.1101/2020.05.11.077669)

---

## Depth-related variability in viral communities in highly stratified sulfidic mine tailings.
 12.06.2020, _SM Gao, A Schippers, N Chen, Y Yuan, MM Zhang, Q Li, B Liao, WS Shu, LN Huang_


Recent studies have significantly expanded our knowledge of viral diversity and functions in the environment. Exploring the ecological relationships between viruses, hosts, and the environment is a crucial first step towards a deeper understanding of the complex and dynamic interplays among them.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00848-3)

10.1186/s40168-020-00848-3

---

## GAD: A Python Script for Dividing Genome Annotation Files into Feature-Based Files.
 11.06.2020, _N Yasser, A Karam_


Nowadays, the manipulation and analysis of genomic data stored in publicly accessible repositories have become a daily task in genomics and bioinformatics laboratories. Due to the enormous advancement in the field of genome sequencing and the emergence of many projects, bioinformaticians have pushed for the creation of a variety of programs and pipelines that will automatically analyze such big data, in particular the pipelines of gene annotation. Dealing with annotation files using easy and simple programs is very important, particularly for non-developers, enhancing the genomic data analysis acceleration. One of the first tasks required to work with genomic annotation files is to extract different features. In this regard, we have developed GAD (https://github.com/bio-projects/GAD) using Python to be a fast, easy, and controlled script that has a high ability to handle annotation files such as GFF3 and GTF. GAD is a cross-platform graphical interface tool used to extract genome features such as intergenic regions, upstream, and downstream genes. Besides, GAD finds all names of ambiguous sequence ontology, and either extracts them or considers them as genes or transcripts. The results are produced in a variety of file formats, such as BED, GTF, GFF3, and FASTA, supported by other bioinformatics programs. The GAD can handle large sizes of different genomes and an infinite number of files with minimal user effort. Therefore, our script could be integrated into various pipelines in all genomic laboratories to accelerate data analysis.

[Source](https://link.springer.com/article/10.1007/s12539-020-00378-4)

10.1007/s12539-020-00378-4

---

