## The structure of prevacuolar compartments in <em>Neurospora crassa</em> as observed with super-resolution microscopy 

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0282989)

---

## Phylogenetic trees of closely related bacterial species and subspecies based on frequencies of short nucleotide sequences.
 26.04.2023, _Y Nakano, Y Domon, K Yamagishi_


_Phylogeny, Base Sequence, Escherichia coli, RNA, Ribosomal, 16S, Biological Evolution, Bacteria, Shigella_
Bacterial phylogenetic analyses are commonly performed to explore the evolutionary relationships among various bacterial species and genera based on their 16S rRNA gene sequences; however, these results are limited by mosaicism, intragenomic heterogeneity, and difficulties in distinguishing between related species. In this study, we aimed to perform genome-wide comparisons of different bacterial species, namely Escherichia coli, Shigella, Yersinia, Klebsiella, and Neisseria spp., based on their K-mer profiles to construct phylogenetic trees. Pentanucleotide frequency analyses (512 patterns of 5 nucleotides each) were performed to distinguish between highly similar species. Moreover, Escherichia albertii strains were clearly distinguished from E. coli and Shigella, despite being closely related to enterohemorrhagic E. coli in the phylogenetic tree. In addition, our phylogenetic tree of Ipomoea species based on pentamer frequency in chloroplast genomes was correlated with previously reported morphological similarities. Furthermore, a support vector machine clearly classified E. coli and Shigella genomes based on their pentanucleotide profiles. These results suggest that phylogenetic analyses based on penta- or hexamer profiles are a useful methodology for microbial phylogenetic studies. In addition, we introduced an R application, Phy5, which generates a phylogenetic tree based on genome-wide comparisons of pentamer profiles. The online version of Phy5 can be accessed at https://phy5.shinyapps.io/Phy5R/ and its command line version Phy5cli can be downloaded at https://github.com/YoshioNakano2021/phy5.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0268847)

10.1371/journal.pone.0268847

---

## Unmasking the tissue-resident eukaryotic DNA virome in humans.
 28.04.2023, _L Pyöriä, D Pratas, M Toppinen, K Hedman, A Sajantila, MF Perdomo_


_Humans, DNA, Viral, Eukaryota, Virome, Viruses, Genome, Human, Organ Specificity_
Little is known on the landscape of viruses that reside within our cells, nor on the interplay with the host imperative for their persistence. Yet, a lifetime of interactions conceivably have an imprint on our physiology and immune phenotype. In this work, we revealed the genetic make-up and unique composition of the known eukaryotic human DNA virome in nine organs (colon, liver, lung, heart, brain, kidney, skin, blood, hair) of 31 Finnish individuals. By integration of quantitative (qPCR) and qualitative (hybrid-capture sequencing) analysis, we identified the DNAs of 17 species, primarily herpes-, parvo-, papilloma- and anello-viruses (&gt;80% prevalence), typically persisting in low copies (mean 540 copies/ million cells). We assembled in total 70 viral genomes (&gt;90% breadth coverage), distinct in each of the individuals, and identified high sequence homology across the organs. Moreover, we detected variations in virome composition in two individuals with underlying malignant conditions. Our findings reveal unprecedented prevalences of viral DNAs in human organs and provide a fundamental ground for the investigation of disease correlates. Our results from post-mortem tissues call for investigation of the crosstalk between human DNA viruses, the host, and other microbes, as it predictably has a significant impact on our health.

[Source](https://academic.oup.com/nar/article/51/7/3223/7084602?login=true)

10.1093/nar/gkad199

---

## Predicting partner fitness based on spatial structuring in a light-driven microbial community

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011045)

---

## Evaluation of alcohol-free mouthwash for studies of the oral microbiome.
 01.05.2023, _Y Yano, E Vogtmann, AH Shreves, SJ Weinstein, A Black, N Diaz-Mayoral, Y Wan, W Zhou, X Hua, CL Dagnall, A Hutchinson, K Jones, BD Hicks, K Wyatt, M Brotzman, N Gerlanc, WY Huang, PS Albert, N Wentzensen, CC Abnet_


_Humans, Mouthwashes, RNA, Ribosomal, 16S, Microbiota, Ethanol, Bacteria_
Oral bacteria play important roles in human health and disease. Oral samples collected using ethanol-containing mouthwash are widely used for oral microbiome studies. However, ethanol is flammable and not ideal for transportation/storage in large quantities, and some individuals may avoid ethanol due to the burning sensation or due to various personal, medical, religious, and/or cultural factors. Here, we compared ethanol-free and ethanol-containing mouthwashes using multiple microbiome metrics and assessed the stability of the mouthwash samples stored up to 10 days before processing. Forty volunteers provided oral wash samples collected using ethanol-free and ethanol-containing mouthwashes. From each sample, one aliquot was immediately frozen, one was stored at 4°C for 5 days and frozen, while the third aliquot was stored for 5 days at 4°C and 5 days at ambient temperature to mimic shipping delays and then frozen. DNA was extracted, the 16S rRNA gene V4 region was amplified and sequenced, and bioinformatic processing was performed using QIIME 2. Microbiome metrics measured in the two mouthwash types were very similar, with intraclass correlation coefficients (ICCs) for alpha and beta diversity metrics greater than 0.85. Relative abundances of some taxa were significantly different, but ICCs of the top four most abundant phyla and genera were high (&gt; 0.75) for the comparability of the mouthwashes. Stability during delayed processing was also high for both mouthwashes based on alpha and beta diversity measures and relative abundances of the top four phyla and genera (ICCs ≥ 0.90). These results demonstrate ethanol-free mouthwash performs similarly to ethanol-containing mouthwash for microbial analyses, and both mouthwashes are stable for at least 10 days without freezing prior to laboratory processing. Ethanol-free mouthwash is suitable for collecting and shipping oral wash samples, and these results have important implications for planning future epidemiologic studies of the oral microbiome.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0284956)

10.1371/journal.pone.0284956

---

## A systematically biosynthetic investigation of lactic acid bacteria reveals diverse antagonistic bacteriocins that potentially shape the human microbiome.
 29.04.2023, _D Zhang, J Zhang, S Kalimuthu, J Liu, ZM Song, BB He, P Cai, Z Zhong, C Feng, P Neelakantan, YX Li_


_Female, Humans, Bacteriocins, Lactobacillales, Microbiota, Metagenome, Anti-Bacterial Agents_
Lactic acid bacteria (LAB) produce various bioactive secondary metabolites (SMs), which endow LAB with a protective role for the host. However, the biosynthetic potentials of LAB-derived SMs remain elusive, particularly in their diversity, abundance, and distribution in the human microbiome. Thus, it is still unknown to what extent LAB-derived SMs are involved in microbiome homeostasis.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01540-y)

10.1186/s40168-023-01540-y

---

## Positive mood-related gut microbiota in a long-term closed environment: a multiomics study based on the “Lunar Palace 365” experiment

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01506-0)

---

## <em>Wolbachia</em> infection in native populations of<em> Blattella germanica</em> and <em>Periplaneta americana</em>

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0284704)

---

## Bacterial and fungal community composition and community-level physiological profiles in forest soils

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0284817)

---

## Predicting microbial community compositions in wastewater treatment plants using artificial neural networks.
 01.05.2023, _X Liu, Y Nie, XL Wu_


_Wastewater, Sewage, Microbiota, Water Purification, Polyphosphates_
Activated sludge (AS) of wastewater treatment plants (WWTPs) is one of the world's largest artificial microbial ecosystems and the microbial community of the AS system is closely related to WWTPs' performance. However, how to predict its community structure is still unclear.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01519-9)

10.1186/s40168-023-01519-9

---

## Open ocean and coastal strains of the N2-fixing cyanobacterium UCYN-A have distinct transcriptomes.
 04.05.2023, _MDC Muñoz-Marín, JD Magasin, JP Zehr_


_Seawater, Transcriptome, Cyanobacteria, Nitrogen Fixation, Oceans and Seas, Nitrogen_
Decades of research on marine N2 fixation focused on Trichodesmium, which are generally free-living cyanobacteria, but in recent years the endosymbiotic cyanobacterium Candidatus Atelocyanobacterium thalassa (UCYN-A) has received increasing attention. However, few studies have shed light on the influence of the host versus the habitat on UCYN-A N2 fixation and overall metabolism. Here we compared transcriptomes from natural populations of UCYN-A from oligotrophic open-ocean versus nutrient-rich coastal waters, using a microarray that targets the full genomes of UCYN-A1 and UCYN-A2 and known genes for UCYN-A3. We found that UCYN-A2, usually regarded as adapted to coastal environments, was transcriptionally very active in the open ocean and appeared to be less impacted by habitat change than UCYN-A1. Moreover, for genes with 24 h periodic expression we observed strong but inverse correlations among UCYN-A1, A2, and A3 to oxygen and chlorophyll, which suggests distinct host-symbiont relationships. Across habitats and sublineages, genes for N2 fixation and energy production had high transcript levels, and, intriguingly, were among the minority of genes that kept the same schedule of diel expression. This might indicate different regulatory mechanisms for genes that are critical to the symbiosis for the exchange of nitrogen for carbon from the host. Our results underscore the importance of N2 fixation in UCYN-A symbioses across habitats, with consequences for community interactions and global biogeochemical cycles.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0272674)

10.1371/journal.pone.0272674

---

## Meta-organism gene expression reveals that the impact of nitrate enrichment on coral larvae is mediated by their associated Symbiodiniaceae and prokaryotic assemblages.
 29.04.2023, _H Tong, F Zhang, J Sun, SE McIlroy, W Zhang, Y Wang, H Huang, G Zhou, PY Qian_


_Animals, Anthozoa, Nitrates, Larva, Coral Reefs, Dinoflagellida, Symbiosis, Transcriptome_
Coral meta-organisms consist of the coral, and its associated Symbiodiniaceae (dinoflagellate algae), bacteria, and other microbes. Corals can acquire photosynthates from Symbiodiniaceae, whilst Symbiodiniaceae uses metabolites from corals. Prokaryotic microbes provide Symbiodiniaceae with nutrients and support the resilience of corals as meta-organisms. Eutrophication is a major cause of coral reef degradation; however, its effects on the transcriptomic response of coral meta-organisms remain unclear, particularly for prokaryotic microbes associated with corals in the larval stage. To understand acclimation of the coral meta-organism to elevated nitrate conditions, we analyzed the physiological and transcriptomic responses of Pocillopora damicornis larvae, an ecologically important scleractinian coral, after 5 days of exposure to elevated nitrate levels (5, 10, 20, and 40 µM).

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01495-0)

10.1186/s40168-023-01495-0

---

## Ten simple rules for working with other people's code.
 24.04.2023, _C Pilgrim, P Kent, K Hosseini, E Chalstrey_



[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011031)

10.1371/journal.pcbi.1011031

---

## Predicting microbe organisms using data of living micro forms of life and hybrid microbes classifier.
 24.04.2023, _A Raza, F Rustam, HUR Siddiqui, IT Diez, I Ashraf_


_Humans, Algorithms, Seroepidemiologic Studies, Machine Learning_
Microbe organisms make up approximately 60% of the earth's living matter and the human body is home to millions of microbe organisms. Microbes are microbial threats to health and may lead to several diseases in humans like toxoplasmosis and malaria. The microbiological toxoplasmosis disease in humans is widespread, with a seroprevalence of 3.6-84% in sub-Saharan Africa. This necessitates an automated approach for microbe organisms detection. The primary objective of this study is to predict microbe organisms in the human body. A novel hybrid microbes classifier (HMC) is proposed in this study which is based on a decision tree classifier and extra tree classifier using voting criteria. Experiments involve different machine learning and deep learning models for detecting ten different living microforms of life. Results suggest that the proposed HMC approach achieves a 98% accuracy score, 98% geometric mean score, 97% precision score, and 97% Cohen Kappa score. The proposed model outperforms employed models, as well as, existing state-of-the-art models. Moreover, the k-fold cross-validation corroborates the results as well. The research helps microbiologists identify the type of microbe organisms with high accuracy and prevents many diseases through early detection.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0284522)

10.1371/journal.pone.0284522

---

## iPHoP: An integrated machine learning framework to maximize host prediction for metagenome-derived viruses of archaea and bacteria.
 05.05.2023, _S Roux, AP Camargo, FH Coutinho, SM Dabdoub, BE Dutilh, S Nayfach, A Tritt_


_Archaea, Metagenome, Viruses, Bacteria, Metagenomics, Machine Learning, Genome, Viral_
The extraordinary diversity of viruses infecting bacteria and archaea is now primarily studied through metagenomics. While metagenomes enable high-throughput exploration of the viral sequence space, metagenome-derived sequences lack key information compared to isolated viruses, in particular host association. Different computational approaches are available to predict the host(s) of uncultivated viruses based on their genome sequences, but thus far individual approaches are limited either in precision or in recall, i.e., for a number of viruses they yield erroneous predictions or no prediction at all. Here, we describe iPHoP, a two-step framework that integrates multiple methods to reliably predict host taxonomy at the genus rank for a broad range of viruses infecting bacteria and archaea, while retaining a low false discovery rate. Based on a large dataset of metagenome-derived virus genomes from the IMG/VR database, we illustrate how iPHoP can provide extensive host prediction and guide further characterization of uncultivated viruses.

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3002083)

10.1371/journal.pbio.3002083

---

