## Advances and challenges in cataloging the human gut virome.
 14.07.2022, _J Li, F Yang, M Xiao, A Li_


The human gut virome, which is often referred to as the "dark matter" of the gut microbiome, remains understudied. A better understanding of the composition and variations of the gut virome across populations is critical for exploring its impact on diseases and health. A series of advances in the characterization of human gut virome have unveiled high genetic diversity and various functional potentials of gut viruses. Here, we summarize the recently available human gut virome databases and discuss their features, procedures, and challenges with the intention to provide a reference to researchers to use while choosing a profiling database. We also propose a "best practice" for cataloging the viral population.

[Source](https://doi.org/10.1016/j.chom.2022.06.003)

10.1016/j.chom.2022.06.003

---

## Diversity of the repetitive DNA fraction in Cestrum, the genus with the largest genomes within Solanaceae.
 09.07.2022, _TB de Souza, LM Parteka, R de Assis, ALL Vanzela_


Cestrum species present large genomes (2 C = ~ 24 pg), a high occurrence of B chromosomes and great diversity in heterochromatin bands. Despite this diversity, karyotypes maintain the chromosome number 2n = 16 (except when they present B chromosomes), and a relative similarity in chromosome morphology and symmetry. To deepen our knowledge of the Cestrum genome composition, low-coverage sequencing data of C. strigilatum and C. elegans were compared, including cytogenomic analyses of seven species.

[Source](https://doi.org/10.1099/mgen.0.000847)

10.1007/s11033-022-07728-z

---

## Genomic diversity of genus <em>Limosilactobacillus</em>

[Source](https://doi.org/10.1099/mgen.0.000847)

---

## Comparison of the respiratory tract microbiome in hospitalized COVID-19 patients with different disease severity.
 15.07.2022, _J Chen, X Liu, W Liu, C Yang, R Jia, Y Ke, J Guo, L Jia, C Wang, Y Chen_


Little is known about the characteristics of respiratory tract microbiome in Coronavirus disease 2019 (COVID-19) inpatients with different severity. We conducted a study that expected to clarify these characteristics as much as possible. A cross-sectional study was conducted to characterize respiratory tract microbial communities of 69 COVID-19 inpatients from 64 nasopharyngeal swabs and 5 sputum specimens using 16S ribosomal RNA (rRNA) gene V3-V4 region sequencing. The bacterial profiles were analyzed to find potential biomarkers by the two-step method, the combination of random forest model and the linear discriminant analysis effect size (LEfSe), and explore the connections with clinical characteristics by Spearman's rank test. Compared with mild COVID-19 patients, severe patients had significantly decreased bacterial diversity (P values were less than 0.05 in the alpha and beta diversity) and relative lower abundance of opportunistic pathogens, including Actinomyces, Prevotella, Rothia, Streptococcus, Veillonella. Eight potential biomarkers including Treponema, Leptotrichia, Lachnoanaerobaculum, Parvimonas, Alloprevotella, Porphyromonas, Gemella and Streptococcus were found to distinguish the mild COVID-19 patients from the severe COVID-19 patients. The genera of Actinomyces and Prevotella were negatively correlated with age in two groups. Intensive Care Unit (ICU) admission, neutrophil count (GRA) and lymphocyte count (LYMPH) were significantly correlated with different genera in the two groups. In addition, there were a positive correlation between Klebsiella and white blood cell count (WBC) in two groups. The respiratory tract microbiome had significant differences in COVID-19 patients with different severity. The value of the respiratory tract microbiome as predictive biomarkers for COVID-19 severity deserves further exploration. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1002/jmv.28002)

10.1002/jmv.28002

---

## An international report on bacterial communities in esophageal squamous cell carcinoma.
 15.07.2022, _J Nomburg, S Bullman, D Nasrollahzadeh, EA Collisson, B Abedi-Ardekani, LO Akoko, JR Atkins, GC Buckle, S Gopal, N Hu, B Kaimila, M Khoshnia, R Malekzadeh, D Menya, BT Mmbaga, S Moody, G Mulima, BP Mushi, J Mwaiselage, A Mwanga, Y Newton, DL Ng, A Radenbaugh, DS Rwakatema, M Selekwa, J Sch√ºz, PR Taylor, C Vaske, A Goldstein, MR Stratton, V McCormack, P Brennan, JA DeCaprio, M Meyerson, EJ Mmbaga, K Van Loon_


The incidence of esophageal squamous cell carcinoma (ESCC) is disproportionately high in the eastern corridor of Africa and parts of Asia. Emerging research has identified a potential association between poor oral health and ESCC. One possible link between poor oral health and ESCC involves the alteration of the microbiome. We performed an integrated analysis of four independent sequencing efforts of ESCC tumors from patients from high- and low-incidence regions of the world. Using whole genome sequencing (WGS) and RNA sequencing (RNAseq) of ESCC tumors from 61 patients in Tanzania, we identified a community of bacteria, including members of the genera Fusobacterium, Selenomonas, Prevotella, Streptococcus, Porphyromonas, Veillonella, and Campylobacter, present at high abundance in ESCC tumors. We then characterized the microbiome of 238 ESCC tumor specimens collected in two additional independent sequencing efforts consisting of patients from other high-ESCC incidence regions (Tanzania, Malawi, Kenya, Iran, China). This analysis revealed similar ESCC-associated bacterial communities in these cancers. Because these genera are traditionally considered members of the oral microbiota, we next explored whether there was a relationship between the synchronous saliva and tumor microbiomes of ESCC patients in Tanzania. Comparative analyses revealed that paired saliva and tumor microbiomes were significantly similar with a specific enrichment of Fusobacterium and Prevotella in the tumor microbiome. Together, these data indicate that cancer-associated oral bacteria are associated with ESCC tumors at the time of diagnosis and support a model in which oral bacteria are present in high abundance in both saliva and tumors of some ESCC patients.

[Source](https://doi.org/10.1002/ijc.34212)

10.1002/ijc.34212

---

## Phenolic-rich beverages reduce bacterial TMA formation in an ex vivo-in vitro colonic fermentation model

[Source](https://doi.org/10.1039/d2fo01159j)

---

## <em>Lactobacillus paragasseri</em> OLL2809 Improves Depression-Like Behavior and Increases Beneficial Gut Microbes in Mice

[Source](https://doi.org/10.3389/fnins.2022.918953)

---

## In silico identification of two peptides with antibacterial activity against multidrug-resistant <em>Staphylococcus aureus</em>

[Source](https://doi.org/10.1038/s41522-022-00320-0)

---

## How Dietary Fibre, Acting via the Gut Microbiome, Lowers Blood Pressure

[Source](https://doi.org/10.1007/s11906-022-01216-2)

---

## Low FODMAP diet in irritable bowel syndrome: a review of recent clinical trials and meta-analyses

[Source](https://doi.org/10.1097/MCO.0000000000000854)

---

## Prebiotic effects of plant-derived (poly)phenols on host metabolism: Is there a role for short-chain fatty acids?

[Source](https://doi.org/10.1080/10408398.2022.2100315)

---

## Unveiling biogeographic patterns in the worldwide distributed <em>Ceratitis capitata</em> (medfly) using populations genomics and microbiome composition

[Source](https://doi.org/10.1111/mec.16616)

---

## Differential oral microbiome in nonhuman primates from periodontitis susceptible and resistant matrilines

[Source](https://doi.org/10.1111/omi.12377)

---

## Chicken jejunal microbiota improves growth performance by mitigating intestinal inflammation

[Source](https://doi.org/10.1186/s40168-022-01299-8)

---

## Comparison Analysis of Different DNA Extraction Methods on Suitability for Long-Read Metagenomic Nanopore Sequencing

[Source](https://doi.org/10.3389/fcimb.2022.919903)

---

## KOMB: K-core based de novo characterization of copy number variation in microbiomes.
 16.07.2022, _A Balaji, N Sapoval, C Seto, RA Leo Elworth, Y Fu, MG Nute, T Savidge, S Segarra, TJ Treangen_


Characterizing metagenomes via kmer-based, database-dependent taxonomic classification has yielded key insights into underlying microbiome dynamics. However, novel approaches are needed to track community dynamics and genomic flux within metagenomes, particularly in response to perturbations. We describe KOMB, a novel method for tracking genome level dynamics within microbiomes. KOMB utilizes K-core decomposition to identify Structural variations (SVs), specifically, population-level Copy Number Variation (CNV) within microbiomes. K-core decomposition partitions the graph into shells containing nodes of induced degree at least K, yielding reduced computational complexity compared to prior approaches. Through validation on a synthetic community, we show that KOMB recovers and profiles repetitive genomic regions in the sample. KOMB is shown to identify functionally-important regions in Human Microbiome Project datasets, and was used to analyze longitudinal data and identify keystone taxa in Fecal Microbiota Transplantation (FMT) samples. In summary, KOMB represents a novel graph-based, taxonomy-oblivious, and reference-free approach for tracking CNV within microbiomes. KOMB is open source and available for download at https://gitlab.com/treangenlab/komb.

[Source](https://doi.org/10.1016/j.csbj.2022.06.019)

10.1016/j.csbj.2022.06.019

---

## CASPER: An Integrated Software Platform for Rapid Development of CRISPR Tools.
 14.07.2022, _B Mendoza, T Fry, D Dooley, J Herman, CT Trinh_


Both academic and enterprise software solutions exist for designing CRISPR targets. They offer advantages when designing guide RNAs (gRNAs) but often focus on a select number of model organisms. Those that offer a wide variety of organisms can be limited in support of alternative endonucleases and downstream analyses such as multitargeting and population analyses to interrogate a microbiome. To accommodate broad CRISPR utilization, we developed a flexible platform software CRISPR Associated Software for Pathway Engineering and Research (CASPER) for gRNA generation and analysis in any organism and with any CRISPR-Cas system. CASPER combines traditional gRNA design tools with unique functions such as multiple Cas-type gRNA generation and evaluation of spacer redundancy in a single species or microbiome. The analyses have implications for strain-, species-, or genus-specific CRISPR diagnostic probe design and microbiome manipulation. The novel features of CASPER are packaged in a user-friendly interface to create a computational environment for researchers to streamline the utility of CRISPR-Cas systems.

[Source](https://doi.org/10.1089/crispr.2022.0025)

10.1089/crispr.2022.0025

---

## Integrating Choline and Specific Intestinal Microbiota to Classify Type 2 Diabetes in Adults: A Machine Learning Based Metagenomics Study.
 16.07.2022, _Q Zeng, M Zhao, F Wang, Y Li, H Li, J Zheng, X Chen, X Zhao, L Ji, X Gao, C Liu, Y Wang, S Cheng, J Xu, B Pan, J Sun, Y Li, D Li, Y He, L Zheng_


_Adult, Betaine, Choline, Cross-Sectional Studies, Diabetes Mellitus, Type 2, Gastrointestinal Microbiome, Glucose, Humans, Machine Learning, Metagenomics, Methylamines, Prediabetic State_
Emerging evidence is examining the precise role of intestinal microbiota in the pathogenesis of type 2 diabetes. The aim of this study was to investigate the association of intestinal microbiota and microbiota-generated metabolites with glucose metabolism systematically in a large cross-sectional study in China. 1160 subjects were divided into three groups based on their glucose level: normal glucose group (n=504), prediabetes group (n=394), and diabetes group (n=262). Plasma concentrations of TMAO, choline, betaine, and carnitine were measured. Intestinal microbiota was measured in a subgroup of 161 controls, 144 prediabetes and 56 diabetes by using metagenomics sequencing. We identified that plasma choline [Per SD of log-transformed change: odds ratio 1.36 (95 confidence interval 1.16, 1.58)] was positively, while betaine [0.77 (0.66, 0.89)] was negatively associated with diabetes, independently of TMAO. Individuals with diabetes could be accurately distinguished from controls by integrating data on choline, and certain microbiota species, as well as traditional risk factors (AUC=0.971). KOs associated with the carbohydrate metabolism pathway were enhanced in individuals with high choline level. The functional shift in the carbohydrate metabolism pathway in high choline group was driven by species Ruminococcus lactaris, Coprococcus catus and Prevotella copri. We demonstrated the potential ability for classifying diabetic population by choline and specific species, and provided a novel insight of choline metabolism linking the microbiota to impaired glucose metabolism and diabetes.

[Source](https://doi.org/10.3389/fendo.2022.906310)

10.3389/fendo.2022.906310

---

