## Genomic population structure associated with repeated escape of <em>Salmonella enterica </em>ATCC14028s from the laboratory into nature

[Source](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009820)

---

## Role of the mobilome in the global dissemination of the carbapenem resistance gene blaNDM 

[Source](https://www.biorxiv.org/content/10.1101/2021.01.14.426698v2.abstract?%3Fcollection=)

---

## Existence of log-phase <em>Escherichia coli</em> persisters and lasting memory of a starvation pulse

[Source](https://www.biorxiv.org/content/10.1101/2020.09.17.301598v3.abstract?%3Fcollection=)

---

## Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes.
 29.09.2021, _V Aggarwala, I Mogno, Z Li, C Yang, GJ Britton, A Chen-Liaw, J Mitcham, G Bongers, D Gevers, JC Clemente, JF Colombel, A Grinspan, J Faith_


Fecal microbiota transplantation (FMT) has been successfully applied to treat recurrent Clostridium difficile infection in humans, but a precise method to measure which bacterial strains stably engraft in recipients and evaluate their association with clinical outcomes is lacking. We assembled a collection of &gt;1,000 different bacterial strains that were cultured from the fecal samples of 22 FMT donors and recipients. Using our strain collection combined with metagenomic sequencing data from the same samples, we developed a statistical approach named Strainer for the detection and tracking of bacterial strains from metagenomic sequencing data. We applied Strainer to evaluate a cohort of 13 FMT longitudinal clinical interventions and detected stable engraftment of 71% of donor microbiota strains in recipients up to 5 years post-FMT. We found that 80% of recipient gut bacterial strains pre-FMT were eliminated by FMT and that post-FMT the strains present persisted up to 5 years later, together with environmentally acquired strains. Quantification of donor bacterial strain engraftment in recipients independently explained (precision 100%, recall 95%) the clinical outcomes (relapse or success) after initial and repeat FMT. We report a compendium of bacterial species and strains that consistently engraft in recipients over time that could be used in defined live biotherapeutic products as an alternative to FMT. Our analytical framework and Strainer can be applied to systematically evaluate either FMT or defined live bacterial therapeutic studies by quantification of strain engraftment in recipients.

[Source](https://www.nature.com/articles/s41564-021-00966-0)

10.1038/s41564-021-00966-0

---

## A role for arthropods as vectors of multidrug-resistant Enterobacterales in surgical site infections from South Asia.
 29.09.2021, _B Hassan, M Ijaz, A Khan, K Sands, GI Serfas, L Clayfield, MM El-Bouseary, G Lai, E Portal, A Khan, WJ Watkins, J Parkhill, TR Walsh_


Understanding how multidrug-resistant Enterobacterales (MDRE) are transmitted in low- and middle-income countries (LMICs) is critical for implementing robust policies to curb the increasing burden of antimicrobial resistance (AMR). Here, we analysed samples from surgical site infections (SSIs), hospital surfaces (HSs) and arthropods (summer and winter 2016) to investigate the incidence and transmission of MDRE in a public hospital in Pakistan. We investigated Enterobacterales containing resistance genes (blaCTX-M-15, blaNDM and blaOXA-48-like) for identification, antimicrobial susceptibility testing and whole-genome sequencing. Genotypes, phylogenetic relationships and transmission events for isolates from different sources were investigated using single-nucleotide polymorphism (SNP) analysis with a cut-off of ≤20 SNPs. Escherichia coli (14.3%), Klebsiella pneumoniae (10.9%) and Enterobacter cloacae (16.3%) were the main MDRE species isolated. The carbapenemase gene blaNDM was most commonly detected, with 15.5%, 15.1% and 13.3% of samples positive in SSIs, HSs and arthropods, respectively. SNP (≤20) and spatiotemporal analysis revealed linkages in bacteria between SSIs, HSs and arthropods supporting the One Health approach to underpin infection control policies across LMICs and control AMR.

[Source](https://www.nature.com/articles/s41564-021-00965-1)

10.1038/s41564-021-00965-1

---

## Response of soil microbiome structure and its network profiles to four soil amendments in monocropping strawberry greenhouse.
 29.09.2021, _S Liu, MH Khan, Z Yuan, S Hussain, H Cao, Y Liu_


With the constant surge of strawberry cultivation and human demand, widespread concern has been expressed about the severe soil and plant health problems caused by continuous strawberry cropping, particularly monocropping in greenhouses. Effective microorganisms (EM) and Bacillus subtilis (BS) have been extensively commercialized as biological control agents (BCAs) to promote plant growth and yield enhancement. However, their effects on soil microbes are obscure. To regulate the microbial community in continuous cropping strawberry soils, we developed four soil amendments based on these two BCAs by adding low and high contents of compost. The amplicon sequencing of bacterial and fungal ribosomal markers was applied to study the response of the soil microbiome structure. We noticed a sharp increase in bacterial diversity after adding EM-treated high compost and BS-treated low compost, while there was no significant change in fungal diversity among treatments. Through taxonomic classification and FUNGuild analysis, we found that the application of soil amendments resulted in a significant decline in the relative abundance of fungal plant pathogens (Rhizopus, Penicillium and Fusarium) in the soils; accordingly, the metabolic functions of a range of detrimental fungi were inhibited. Correlation analysis indicated that soil microbial community was indirectly driven by soil physicochemical properties. Co-occurrence networks revealed that soil amendments contributed to the connectivity of bacterial network, and EM-treated with high compost was the most complex and balanced. Collectively, EM-treated high compost and BS-treated low compost can well regulate the microbial community structure and thus maintain soil health.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0245180)

10.1371/journal.pone.0245180

---

## Truncated denitrifiers dominate the denitrification pathway in tundra soil metagenomes

[Source](https://www.biorxiv.org/content/10.1101/2020.12.21.419267v3.abstract?%3Fcollection=)

---

## Response of <em>Endozoicomonas montiporae</em> to heat stress and coral host lysates

[Source](https://www.biorxiv.org/content/10.1101/2021.09.27.461970v1.abstract?%3Fcollection=)

---

## Enhanced copper-resistance gene repertoire in <em>Alteromonas macleodii</em> strains isolated from copper-treated marine coatings

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0257800)

---

## Bacterial community structure of early-stage biofilms is dictated by temporal succession rather than substrate types in the southern coastal seawater of India

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0257961)

---

## Ecological stochasticity and phage induction diversify bacterioplankton communities at the microscale

[Source](https://www.biorxiv.org/content/10.1101/2021.09.27.461956v1.abstract?%3Fcollection=)

---

