## A practice of using five-colour chart to guide the control of COVID-19 and resumption of work in Zhejiang Province, China.
 14.06.2021, _F He, X Shang, F Ling, Z Chen, T Fu, J Lin, Z Wang_


_COVID-19, China, Color, Disease Outbreaks, Epidemics, Humans, Pandemics, Quarantine, Risk Assessment_
Since the outbreak of COVID-19 in December 2019 in Wuhan, Zhejiang has become the province with the largest number of cases. The aim of this article is to present Zhejiang province's experience of establishing an accurate and smart control mechanism for epidemic prevention and control and resumption of work and production using a 'five-colour epidemic chart'. The number of confirmed cases, proportion of local cases, and occurrence of clustered outbreaks were used as evaluation indicators to calculate the county-level epidemic risk and were assigned different weight coefficients; the absence of cases for 3 and 7 consecutive days was used as the adjustment index. When the first chart was published on February 9, there were 1 very-high-risk, 12 high-risk, and 12 low-risk counties. Under the five-colour chart, Zhejiang began to adopt precise measures to prevent and control the epidemic and resume work and production. By February 24, the low-risk counties had expanded to 82, with no high-risk and very-high-risk counties. The epidemic situation in Zhejiang province has been effectively controlled. The experience of epidemic prevention and control in Zhejiang is worthy to be emulated and learned by other countries and regions.

[Source](https://doi.org/10.3785/j.issn.1008-9292.2020.02.02)

10.1038/s41598-021-90808-0

---

## Management Of COVID-19: The Zhejiang Experience

[Source](https://doi.org/10.3785/j.issn.1008-9292.2020.02.02)

---

## Metatranscriptomic Characterization of COVID-19 Identified A Host Transcriptional Classifier Associated With Immune Signaling.
 26.06.2020, _H Zhang, JW Ai, W Yang, X Zhou, F He, S Xie, W Zeng, Y Li, Y Yu, X Gou, Y Li, X Wang, H Su, T Xu, W Zhang_


The recent identification of a novel coronavirus, also known as SARS-CoV-2, has caused a global outbreak of respiratory illnesses. The rapidly developing pandemic has posed great challenges to diagnosis of this novel infection. However, little is known about the metatranscriptomic characteristics of patients with Coronavirus Disease 2019 (COVID-19).

[Source](https://doi.org/10.1093/cid/ciaa663)

10.1093/cid/ciaa663

---

## The lung tissue microbiota features of 20 deceased patients with COVID-19.
 10.01.2021, _J Fan, X Li, Y Gao, J Zhou, S Wang, B Huang, J Wu, Q Cao, Y Chen, Z Wang, D Luo, T Zhou, R Li, Y Shang, X Nie_


_Betacoronavirus, COVID-19, Coronavirus Infections, Critical Illness, Humans, Microbiota, Pandemics, Pneumonia, Viral, SARS-CoV-2_

[Source](https://doi.org/10.1016/j.jinf.2020.06.047)

10.1016/j.jinf.2020.06.047

---

## The Lung Tissue Microbiota Features Of 20 Deceased Patients With COVID-19

[Source](https://doi.org/10.1016/j.jinf.2020.06.047)

---

## Nasopharyngeal Microbiota Profiling of SARS-CoV-2 Infected Patients.
 28.09.2020, _F De Maio, B Posteraro, FR Ponziani, P Cattani, A Gasbarrini, M Sanguinetti_


We analyzed the bacterial communities of the nasopharynx in 40 SARS-CoV-2 infected and uninfected patients. All infected patients had a mild COVID-19 disease. We did not find statistically significant differences in either bacterial richness and diversity or composition. These findings suggest a nasopharyngeal microbiota at least early resilient to SARS-CoV-2 infection.

[Source](https://doi.org/10.1186/s12575-020-00131-7)

10.1186/s12575-020-00131-7

---

## Metagenomic Next-Generation Sequencing of Nasopharyngeal Specimens Collected from Confirmed and Suspect COVID-19 Patients.
 14.12.2020, _HH Mostafa, JA Fissel, B Fanelli, Y Bergman, V Gniazdowski, M Dadlani, KC Carroll, RR Colwell, PJ Simner_


_Bacteria, COVID-19, Coinfection, Computational Biology, High-Throughput Nucleotide Sequencing, Humans, Metagenome, Metagenomics, Microbiota, Nasopharynx, RNA, Viral, SARS-CoV-2, Specimen Handling_
Metagenomic next-generation sequencing (mNGS) offers an agnostic approach for emerging pathogen detection directly from clinical specimens. In contrast to targeted methods, mNGS also provides valuable information on the composition of the microbiome and might uncover coinfections that may associate with disease progression and impact prognosis. To evaluate the use of mNGS for detecting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and/or other infecting pathogens, we applied direct Oxford Nanopore long-read third-generation metatranscriptomic and metagenomic sequencing. Nasopharyngeal (NP) swab specimens from 50 patients under investigation for CoV disease 2019 (COVID-19) were sequenced, and the data were analyzed by the CosmosID bioinformatics platform. Further, we characterized coinfections and the microbiome associated with a four-point severity index. SARS-CoV-2 was identified in 77.5% (31/40) of samples positive by RT-PCR, correlating with lower cycle threshold (Ct) values and fewer days from symptom onset. At the time of sampling, possible bacterial or viral coinfections were detected in 12.5% of SARS-CoV-2-positive specimens. A decrease in microbial diversity was observed among COVID-19-confirmed patients (Shannon diversity index, P = 0.0082; Chao richness estimate, P = 0.0097; Simpson diversity index, P = 0.018), and differences in microbial communities were linked to disease severity (P = 0.022). Furthermore, statistically significant shifts in the microbiome were identified among SARS-CoV-2-positive and -negative patients, in the latter of whom a higher abundance of Propionibacteriaceae (P = 0.028) and a reduction in the abundance of Corynebacterium accolens (P = 0.025) were observed. Our study corroborates the growing evidence that increased SARS-CoV-2 RNA detection from NP swabs is associated with the early stages rather than the severity of COVID-19. Further, we demonstrate that SARS-CoV-2 causes a significant change in the respiratory microbiome. This work illustrates the utility of mNGS for the detection of SARS-CoV-2, for diagnosing coinfections without viral target enrichment or amplification, and for the analysis of the respiratory microbiome.IMPORTANCE SARS-CoV-2 has presented a rapidly accelerating global public health crisis. The ability to detect and analyze viral RNA from minimally invasive patient specimens is critical to the public health response. Metagenomic next-generation sequencing (mNGS) offers an opportunity to detect SARS-CoV-2 from nasopharyngeal (NP) swabs. This approach also provides information on the composition of the respiratory microbiome and its relationship to coinfections or the presence of other organisms that may impact SARS-CoV-2 disease progression and prognosis. Here, using direct Oxford Nanopore long-read third-generation metatranscriptomic and metagenomic sequencing of NP swab specimens from 50 patients under investigation for COVID-19, we detected SARS-CoV-2 sequences by applying the CosmosID bioinformatics platform. Further, we characterized coinfections and detected a decrease in the diversity of the microbiomes in these patients. Statistically significant shifts in the microbiome were identified among COVID-19-positive and -negative patients, in the latter of whom a higher abundance of Propionibacteriaceae and a reduction in the abundance of Corynebacterium accolens were observed. Our study also corroborates the growing evidence that increased SARS-CoV-2 RNA detection from NP swabs is associated with the early stages of disease rather than with severity of disease. This work illustrates the utility of mNGS for the detection and analysis of SARS-CoV-2 from NP swabs without viral target enrichment or amplification and for the analysis of the respiratory microbiome.

[Source](https://doi.org/10.1128/mBio.01969-20)

10.1128/mBio.01969-20

---

## Feasibility of using alternative swabs and storage solutions for paired SARS-CoV-2 detection and microbiome analysis in the hospital environment.
 26.05.2021, _JJ Minich, F Ali, C Marotz, P Belda-Ferre, L Chiang, JP Shaffer, CS Carpenter, D McDonald, J Gilbert, SM Allard, EE Allen, R Knight, DA Sweeney, AD Swafford_


_Biological Transport, COVID-19 Nucleic Acid Testing, Ethanol, Feasibility Studies, Humans, Intensive Care Units, Limit of Detection, Microbiota, RNA, Ribosomal, 16S, RNA, Viral, Ribonucleases, SARS-CoV-2, Specimen Handling_
Determining the role of fomites in the transmission of SARS-CoV-2 is essential in the hospital setting and will likely be important outside of medical facilities as governments around the world make plans to ease COVID-19 public health restrictions and attempt to safely reopen economies. Expanding COVID-19 testing to include environmental surfaces would ideally be performed with inexpensive swabs that could be transported safely without concern of being a source of new infections. However, CDC-approved clinical-grade sampling supplies and techniques using a synthetic swab are expensive, potentially expose laboratory workers to viable virus and prohibit analysis of the microbiome due to the presence of antibiotics in viral transport media (VTM). To this end, we performed a series of experiments comparing the diagnostic yield using five consumer-grade swabs (including plastic and wood shafts and various head materials including cotton, synthetic, and foam) and one clinical-grade swab for inhibition to RNA. For three of these swabs, we evaluated performance to detect SARS-CoV-2 in twenty intensive care unit (ICU) hospital rooms of patients including COVID-19+ patients. All swabs were placed in 95% ethanol and further evaluated in terms of RNase activity. SARS-CoV-2 was measured both directly from the swab and from the swab eluent.

[Source](https://doi.org/10.1186/s40168-020-00960-4)

10.1186/s40168-020-00960-4

---

## Variability in digestive and respiratory tract Ace2 expression is associated with the microbiome.
 27.03.2021, _ST Koester, N Li, DM Lachance, NM Morella, N Dey_


_Adenomatous Polyposis Coli Protein, Angiotensin-Converting Enzyme 2, Animals, Colon, Female, Gastrointestinal Microbiome, Gene Expression, Interleukin-10, Intestine, Small, Lung, Male, Mice, Mice, Inbred C57BL, Mice, Knockout_
COVID-19 (coronavirus disease 2019) patients exhibiting gastrointestinal symptoms are reported to have worse prognosis. Ace2 (angiotensin-converting enzyme 2), the gene encoding the host protein to which SARS-CoV-2 spike proteins bind, is expressed in the gut and therefore may be a target for preventing or reducing severity of COVID-19. Here we test the hypothesis that Ace2 expression in the gastrointestinal and respiratory tracts is modulated by the microbiome. We used quantitative PCR to profile Ace2 expression in germ-free mice, conventional raised specific pathogen-free mice, and gnotobiotic mice colonized with different microbiota. Intestinal Ace2 expression levels were significantly higher in germ-free mice compared to conventional mice. A similar trend was observed in the respiratory tract. Intriguingly, microbiota depletion via antibiotics partially recapitulated the germ-free phenotype, suggesting potential for microbiome-mediated regulation of Ace2 expression. Variability in intestinal Ace2 expression was observed in gnotobiotic mice colonized with different microbiota, partially attributable to differences in microbiome-encoded proteases and peptidases. Together, these data suggest that the microbiome may be one modifiable factor determining COVID-19 infection risk and disease severity.

[Source](https://doi.org/10.1371/journal.pone.0248730)

10.1371/journal.pone.0248730

---

## Variability In Digestive And Respiratory Tract Ace2 expression Is associated with the microbiome

[Source](https://doi.org/10.1371/journal.pone.0248730)

---

## Diversity and genomic determinants of the microbiomes associated with COVID-19 and non-COVID respiratory diseases.
 25.05.2021, _MN Hoque, MS Rahman, R Ahmed, MS Hossain, MS Islam, T Islam, MA Hossain, AZ Siddiki_


The novel coronavirus disease 2019 (COVID-19) is a rapidly emerging and highly transmissible disease caused by the Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2). Understanding the microbiomes associated with the upper respiratory tract infection (URTI), chronic obstructive pulmonary disease (COPD) and COVID-19 diseases has clinical interest. We hypothesize that microbiome diversity and composition, and their genomic features are associated with different pathological conditions of these human respiratory tract diseases. To test this hypothesis, we analyzed 21 RNASeq metagenomic data including eleven COVID-19 (BD = 6 and China = 5), six COPD (UK = 6) and four URTI (USA = 4) samples to unravel the microbiome diversity and related genomic metabolic functions. The metagenomic data mapped to 534 bacterial, 60 archaeal and 61 viral genomes with distinct variation in the microbiome composition across the samples (COVID-19 &gt; COPD &gt; URTI). Notably, 94.57%, 80.0% and 24.59% bacterial, archaeal and viral genera shared between the COVID-19 and non-COVID samples, respectively. However, the COVID-19 related samples had sole association with 16 viral genera other than SARS-CoV-2. Strain-level virome profiling revealed 660 and 729 strains in COVID-19 and non-COVID samples, respectively, and of them 34.50% strains shared between the conditions. Functional annotation of the metagenomic data identified the association of several biochemical pathways related to basic metabolism (amino acid and energy), ABC transporters, membrane transport, virulence, disease and defense, regulation of virulence, programmed cell death, and primary immunodeficiency. We also detected 30 functional gene groups/classes associated with resistance to antibiotics and toxic compounds (RATC) in both COVID-19 and non-COVID microbiomes. Furthermore, we detected comparatively higher abundance of cobalt-zinc-cadmium resistance (CZCR) and multidrug resistance to efflux pumps (MREP) genes in COVID-19 metagenome. The profiles of microbiome diversity and associated microbial genomic features found in both COVID-19 and non-COVID (COPD and URTI) samples might be helpful in developing microbiome-based diagnostics and therapeutics for COVID-19 and non-COVID respiratory diseases. However, future studies might be carried out to explore the microbiome dynamics and the cross-talk between host and microbiomes employing larger volume of samples from different ethnic groups and geoclimatic conditions.

[Source](https://doi.org/10.1038/s41598-021-85750-0)

10.1016/j.genrep.2021.101200

---

## Functional Profiling Of COVID-19 Respiratory Tract Microbiomes

[Source](https://doi.org/10.1038/s41598-021-85750-0)

---

## Signatures of COVID-19 severity and immune response in the respiratory tract microbiome.
 29.05.2021, _C Merenstein, G Liang, SA Whiteside, AG Cobián-Güemes, MS Merlino, LJ Taylor, A Glascock, K Bittinger, C Tanes, J Graham-Wooten, LA Khatib, AS Fitzgerald, S Reddy, AE Baxter, JR Giles, DA Oldridge, NJ Meyer, EJ Wherry, JE McGinniss, FD Bushman, RG Collman_


Viral infection of the respiratory tract can be associated with propagating effects on the airway microbiome, and microbiome dysbiosis may influence viral disease.

[Source](https://doi.org/10.1101/2021.04.02.21254514)

10.1101/2021.04.02.21254514

---

## Signatures Of COVID-19 Severity And Immune Response In The Respiratory Tract microbiome

[Source](https://doi.org/10.1101/2021.04.02.21254514)

---

## Plasma microbiome in COVID-19 subjects: an indicator of gut barrier defects and dysbiosis.
 14.04.2021, _R Prasad, MJ Patton, JL Floyd, CP Vieira, S Fortmann, M DuPont, A Harbour, CS Jeremy, J Wright, R Lamendella, BR Stevens, MB Grant_


The gut is a well-established route of infection and target for viral damage by SARS-CoV-2. This is supported by the clinical observation that about half of COVID-19 patients exhibit gastrointestinal ( GI ) symptoms. We asked whether the analysis of plasma could provide insight into gut barrier dysfunction in patients with COVID-19 infection. Plasma samples of COVID-19 patients (n=30) and healthy control (n=16) were collected during hospitalization. Plasma microbiome was analyzed using 16S rRNA sequencing, metatranscriptomic analysis, and gut permeability markers including FABP-2, PGN and LPS in both patient cohorts. Almost 65% (9 out 14) COVID-19 patients showed abnormal presence of gut microbes in their bloodstream. Plasma samples contained predominately Proteobacteria, Firmicutes, and Actinobacteria . The abundance of gram-negative bacteria ( Acinetobacter, Nitrospirillum, Cupriavidus, Pseudomonas, Aquabacterium, Burkholderia, Caballeronia, Parabhurkholderia, Bravibacterium, and Sphingomonas ) was higher than the gram-positive bacteria ( Staphylococcus and Lactobacillus ) in COVID-19 subjects. The levels of plasma gut permeability markers FABP2 (1282±199.6 vs 838.1±91.33; p=0.0757), PGN (34.64±3.178 vs 17.53±2.12; p&lt;0.0001), and LPS (405.5±48.37 vs 249.6±17.06; p=0.0049) were higher in COVID-19 patients compared to healthy subjects. These findings support that the intestine may represent a source for bacteremia and may contribute to worsening COVID-19 outcomes. Therapies targeting the gut and prevention of gut barrier defects may represent a strategy to improve outcomes in COVID-19 patients.

[Source](https://doi.org/10.1101/2021.04.06.438634)

10.1101/2021.04.06.438634

---

## Plasma Microbiome In COVID-19 Subjects: An Indicator Of Gut Barrier Defects And dysbiosis

[Source](https://doi.org/10.1101/2021.04.06.438634)

---

## Evaluation of superinfection, antimicrobial usage, and airway microbiome with metagenomic sequencing in COVID-19 patients: A cohort study in Shanghai.
 11.06.2021, _Q Miao, Y Ma, Y Ling, W Jin, Y Su, Q Wang, J Pan, Y Zhang, H Chen, J Yuan, H Wu, B Hu_


In COVID-19 patients, information regarding superinfection, antimicrobial assessment, and the value of metagenomic sequencing (MS) could help develop antimicrobial stewardship.

[Source](https://doi.org/10.1016/j.jmii.2021.03.015)

10.1016/j.jmii.2021.03.015

---

## Evaluation Of Superinfection, Antimicrobial Usage, And Airway Microbiome With metagenomic sequencing in COVID-19 patients: A cohort study in Shanghai

[Source](https://doi.org/10.1016/j.jmii.2021.03.015)

---

## COVID-19 Influences Lung Microbiota Dynamics And Favors The Emergence Of Rare infectious diseases: A case report of <em>Hafnia alvei</em> pneumonia

[Source](https://doi.org/10.1016/j.jcrc.2021.04.008)

---

## Impaired immune signaling and changes in the lung microbiome precede secondary bacterial pneumonia in COVID-19.
 27.05.2021, _A Tsitsiklis, B Zha, A Byrne, C DeVoe, S Levan, E Rackaityte, S Sunshine, E Mick, R Ghale, A Jauregui, N Neff, A Sarma, P Serpa, T Deiss, A Kistler, S Carrillo, KM Ansel, A Leligdowicz, S Christenson, N Jones, B Wu, S Darmanis, M Matthay, S Lynch, J DeRisi, C Consortium, C Hendrickson, K Kangelaris, M Krummel, P Woodruff, D Erle, O Rosenberg, C Calfee, C Langelier_


Secondary bacterial infections, including ventilator-associated pneumonia (VAP), lead to worse clinical outcomes and increased mortality following viral respiratory infections including in patients with coronavirus disease 2019 (COVID-19). Using a combination of tracheal aspirate bulk and single-cell RNA sequencing (scRNA-seq) we assessed lower respiratory tract immune responses and microbiome dynamics in 28 COVID-19 patients, 15 of whom developed VAP, and eight critically ill uninfected controls. Two days before VAP onset we observed a transcriptional signature of bacterial infection. Two weeks prior to VAP onset, following intubation, we observed a striking impairment in immune signaling in COVID-19 patients who developed VAP. Longitudinal metatranscriptomic analysis revealed disruption of lung microbiome community composition in patients with VAP, providing a connection between dysregulated immune signaling and outgrowth of opportunistic pathogens. These findings suggest that COVID-19 patients who develop VAP have impaired antibacterial immune defense detectable weeks before secondary infection onset.

[Source](https://doi.org/10.1038/s41598-021-89516-6)

10.21203/rs.3.rs-380803/v1

---

## The Lower Respiratory Tract Microbiome Of Critically Ill Patients With COVID-19

[Source](https://doi.org/10.1038/s41598-021-89516-6)

---

## Metagenomic analysis reveals oropharyngeal microbiota alterations in patients with COVID-19.
 31.05.2021, _S Ma, F Zhang, F Zhou, H Li, W Ge, R Gan, H Nie, B Li, Y Wang, M Wu, D Li, D Wang, Z Wang, Y You, Z Huang_


_Adult, Bacteria, COVID-19, Female, Humans, Male, Metagenomics, Microbiota, Middle Aged, Oropharynx, SARS-CoV-2_
COVID-19 remains a serious emerging global health problem, and little is known about the role of oropharynx commensal microbes in infection susceptibility and severity. Here, we present the oropharyngeal microbiota characteristics identified by shotgun metagenomic sequencing analyses of oropharynx swab specimens from 31 COVID-19 patients, 29 influenza B patients, and 28 healthy controls. Our results revealed a distinct oropharyngeal microbiota composition in the COVID-19 patients, characterized by enrichment of opportunistic pathogens such as Veillonella and Megasphaera and depletion of Pseudopropionibacterium, Rothia, and Streptococcus. Based on the relative abundance of the oropharyngeal microbiome, we built a microbial classifier to distinguish COVID-19 patients from flu patients and healthy controls with an AUC of 0.889, in which Veillonella was identified as the most prominent biomarker for COVID-19 group. Several members of the genus Veillonella, especially Veillonella parvula which was highly enriched in the oropharynx of our COVID-19 patients, were also overrepresented in the BALF of COVID-19 patients, indicating that the oral cavity acts as a natural reservoir for pathogens to induce co-infections in the lungs of COVID-19 patients. We also found the increased ratios of Klebsiella sp., Acinetobacter sp., and Serratia sp. were correlated with both disease severity and elevated systemic inflammation markers (neutrophil-lymphocyte ratio, NLR), suggesting that these oropharynx microbiota alterations may impact COVID-19 severity by influencing the inflammatory response. Moreover, the oropharyngeal microbiome of COVID-19 patients exhibited a significant enrichment in amino acid metabolism and xenobiotic biodegradation and metabolism. In addition, all 26 drug classes of antimicrobial resistance genes were detected in the COVID-19 group, and were significantly enriched in critical cases. In conclusion, we found that oropharyngeal microbiota alterations and functional differences were associated with COVID-19 severity.

[Source](https://doi.org/10.1038/s41392-021-00614-3)

10.1038/s41392-021-00614-3

---

## Metagenomic Analysis Reveals Oropharyngeal Microbiota Alterations In Patients With COVID-19

[Source](https://doi.org/10.1038/s41392-021-00614-3)

---

## Acute SARS-CoV-2 Infection Is Associated With An Expansion Of Bacteria Pathogens In the nose including <em>Pseudomonas aeruginosa</em>

[Source](https://doi.org/10.1101/2021.05.20.445008)

---

## Validation of High-Sensitivity Severe Acute Respiratory Syndrome Coronavirus 2 Testing for Stool-Toward the New Normal for Fecal Microbiota Transplantation.
 17.06.2021, _A Babiker, JM Ingersoll, MW Adelman, AS Webster, KJ Broder, V Stittleburg, JJ Waggoner, CS Kraft, MH Woodworth_


_COVID-19, COVID-19 Testing, Centers for Disease Control and Prevention, U.S., Fecal Microbiota Transplantation, Feces, Humans, Nasopharynx, RNA, Viral, Reverse Transcriptase Polymerase Chain Reaction, SARS-CoV-2, Tissue Donors, United States_
Mounting evidence demonstrates potential for fecal-oral transmission of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The US Food and Drug Administration now requires SARS-CoV-2 testing of potential feces donors before the use of stool manufactured for fecal microbiota transplantation. We sought to develop and validate a high-sensitivity SARS-CoV-2 reverse transcriptase polymerase chain reaction (RT-PCR) procedure for testing stool specimens.

[Source](https://doi.org/10.1186/s12575-021-00148-6)

10.14309/ctg.0000000000000363

---

## Nasopharyngeal Microbiota In SARS-CoV-2 Positive And Negative Patients

[Source](https://doi.org/10.1186/s12575-021-00148-6)

---

## SARS-CoV-2 detection status associates with bacterial community composition in patients and the hospital environment.
 12.06.2021, _C Marotz, P Belda-Ferre, F Ali, P Das, S Huang, K Cantrell, L Jiang, C Martino, RE Diner, G Rahman, D McDonald, G Armstrong, S Kodera, S Donato, G Ecklu-Mensah, N Gottel, MC Salas Garcia, LY Chiang, RA Salido, JP Shaffer, MK Bryant, K Sanders, G Humphrey, G Ackermann, N Haiminen, KL Beck, HC Kim, AP Carrieri, L Parida, Y Vázquez-Baeza, FJ Torriani, R Knight, J Gilbert, DA Sweeney, SM Allard_


_COVID-19, Hospitals, Humans, Pandemics, Phylogeny, RNA, Ribosomal, 16S, RNA, Viral, SARS-CoV-2_
SARS-CoV-2 is an RNA virus responsible for the coronavirus disease 2019 (COVID-19) pandemic. Viruses exist in complex microbial environments, and recent studies have revealed both synergistic and antagonistic effects of specific bacterial taxa on viral prevalence and infectivity. We set out to test whether specific bacterial communities predict SARS-CoV-2 occurrence in a hospital setting.

[Source](https://doi.org/10.1186/s40168-021-01083-0)

10.1186/s40168-021-01083-0

---

## SARS-CoV-2 Detection Status Associates With Bacterial Community Composition In patients and the hospital environment

[Source](https://doi.org/10.1186/s40168-021-01083-0)

---

## Gnotobiotic Rats Reveal That Gut Microbiota Regulates Colonic mRNA of Ace2, the Receptor for SARS-CoV-2 Infectivity.
 18.12.2020, _T Yang, S Chakraborty, P Saha, B Mell, X Cheng, JY Yeo, X Mei, G Zhou, J Mandal, R Golonka, BS Yeoh, V Putluri, DWB Piyarathna, N Putluri, CG McCarthy, CF Wenceslau, A Sreekumar, AT Gewirtz, M Vijay-Kumar, B Joe_


_Angiotensin-Converting Enzyme 2, Animals, Betacoronavirus, COVID-19, Colitis, Colon, Coronavirus Infections, Correlation of Data, Gastrointestinal Microbiome, Gene Expression Regulation, Germ-Free Life, Pandemics, Peptidyl-Dipeptidase A, Pneumonia, Viral, RNA, Messenger, Rats, SARS-CoV-2_

[Source](https://doi.org/10.1161/HYPERTENSIONAHA.120.15360)

10.1161/HYPERTENSIONAHA.120.15360

---

## Temporal landscape of human gut RNA and DNA virome in SARS-CoV-2 infection and severity.
 27.04.2021, _T Zuo, Q Liu, F Zhang, YK Yeoh, Y Wan, H Zhan, GCY Lui, Z Chen, AYL Li, CP Cheung, N Chen, W Lv, RWY Ng, EYK Tso, KSC Fung, V Chan, L Ling, G Joynt, DSC Hui, FKL Chan, PKS Chan, SC Ng_


_COVID-19, Child, Preschool, DNA, Gastrointestinal Microbiome, Humans, RNA, SARS-CoV-2, Virome_
Coronavirus disease 2019 (COVID-19) caused by the enveloped RNA virus SARS-CoV-2 primarily affects the respiratory and gastrointestinal tracts. SARS-CoV-2 was isolated from fecal samples, and active viral replication was reported in human intestinal cells. The human gut also harbors an enormous amount of resident viruses (collectively known as the virome) that play a role in regulating host immunity and disease pathophysiology. Understanding gut virome perturbation that underlies SARS-CoV-2 infection and severity is an unmet need.

[Source](https://doi.org/10.1053/j.gastro.2020.05.048)

10.1186/s40168-021-01008-x

---

## Alterations In Gut Microbiota Of Patients With COVID-19 During Time Of Hospitalization

[Source](https://doi.org/10.1053/j.gastro.2020.05.048)

---

## Alterations in Fecal Fungal Microbiome of Patients With COVID-19 During Time of Hospitalization until Discharge.
 05.05.2021, _T Zuo, H Zhan, F Zhang, Q Liu, EYK Tso, GCY Lui, N Chen, A Li, W Lu, FKL Chan, PKS Chan, SC Ng_


_Adult, Aged, Aspergillus flavus, Aspergillus niger, Betacoronavirus, COVID-19, Candida, Candida albicans, Case-Control Studies, Community-Acquired Infections, Coronavirus Infections, DNA, Fungal, Feces, Female, Fungi, Gastrointestinal Microbiome, Humans, Male, Metagenomics, Middle Aged, Mycobiome, Nasopharynx, Pandemics, Patient Discharge, Pneumonia, Pneumonia, Viral, SARS-CoV-2, Time Factors, Young Adult_
Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infects intestinal cells, and might affect the intestinal microbiota. We investigated changes in the fecal fungal microbiomes (mycobiome) of patients with SARS-CoV-2 infection during hospitalization and on recovery.

[Source](https://doi.org/10.1053/j.gastro.2020.06.048)

10.1053/j.gastro.2020.06.048

---

## Gut microbiota composition reflects disease severity and dysfunctional immune responses in patients with COVID-19.
 15.03.2021, _YK Yeoh, T Zuo, GC Lui, F Zhang, Q Liu, AY Li, AC Chung, CP Cheung, EY Tso, KS Fung, V Chan, L Ling, G Joynt, DS Hui, KM Chow, SSS Ng, TC Li, RW Ng, TC Yip, GL Wong, FK Chan, CK Wong, PK Chan, SC Ng_


_Adult, Bacteria, C-Reactive Protein, COVID-19, Cytokines, DNA, Bacterial, Dysbiosis, Female, Gastrointestinal Microbiome, Gastrointestinal Tract, Hong Kong, Humans, Immunity, Male, SARS-CoV-2, Severity of Illness Index, Transferases_
Although COVID-19 is primarily a respiratory illness, there is mounting evidence suggesting that the GI tract is involved in this disease. We investigated whether the gut microbiome is linked to disease severity in patients with COVID-19, and whether perturbations in microbiome composition, if any, resolve with clearance of the SARS-CoV-2 virus.

[Source](https://doi.org/10.1136/gutjnl-2020-323020)

10.1136/gutjnl-2020-323020

---

## Gut Microbiota Composition Reflects Disease Severity And Dysfunctional Immune responses in patients with COVID-19

[Source](https://doi.org/10.1136/gutjnl-2020-323020)

---

## The faecal metabolome in COVID-19 patients is altered and associated with clinical features and gut microbes.
 09.03.2021, _L Lv, H Jiang, Y Chen, S Gu, J Xia, H Zhang, Y Lu, R Yan, L Li_


_Adult, Bacteria, COVID-19, Cohort Studies, Dysbiosis, Feces, Female, Fungi, Gas Chromatography-Mass Spectrometry, Gastrointestinal Microbiome, Humans, Male, Metabolome, Middle Aged, SARS-CoV-2_
Although SARS-CoV-2 can invade the intestine, though its effect on digestion and absorption is not fully understood. In the present study, 56 COVID-19 patients and 47 age- and sex-matched healthy subjects were divided into a discovery cohort and a validation cohort. Blood, faeces and clinical information were collected from the patients in the hospital and at discharge. The faecal metabolome was analysed using gas chromatography-mass spectrometry, and Spearman's correlation analyses of clinical features, the serum metabolome, and the faecal micro- and mycobiota were conducted. The results showed that, the faeces of COVID-19 patients were enriched with important nutrients that should be metabolized or absorbed, such as sucrose and 2-palmitoyl-glycerol; diet-related components that cannot be synthesized by humans, such as 1,5-anhydroglucitol and D-pinitol; and harmful metabolites, such as oxalate, were also detected. In contrast, purine metabolites such as deoxyinosine and hypoxanthine, low-water-soluble long-chain fatty alcohols/acids such as behenic acid, compounds rarely occurring in nature such as D-allose and D-arabinose, and microbe-related compounds such as 2,4-di-tert-butylphenol were depleted in the faeces of COVID-19 patients. Moreover, these metabolites significantly correlated with altered serum metabolites such as oxalate and gut microbesincluding Ruminococcaceae, Actinomyces, Sphingomonas and Aspergillus. Although levels of several faecal metabolites, such as sucrose, 1,5-anhydroglucitol and D-pinitol, of discharged patients were not different from those of healthy controls (HCs), those of oxalate and 2-palmitoyl-glycerol did differ. Therefore, alterations in the faecal metabolome of COVID-19 patients may reflect malnutrition and intestinal inflammation and warrant greater attention. The results of present study provide new insights into the pathogenesis and treatment of COVID-19.

[Source](https://doi.org/10.1016/j.aca.2021.338267)

10.1016/j.aca.2021.338267

---

## The Faecal Metabolome In COVID-19 Patients Is Altered And Associated With Clinical features and gut microbes

[Source](https://doi.org/10.1016/j.aca.2021.338267)

---

## 16S rRNA Gene Sequencing Of Rectal Swab In Patients Affected By COVID-19

[Source](https://doi.org/10.1371/journal.pone.0247041)

---

## Investigation Of Nasal/Oropharyngeal Microbial Community Of COVID-19 Patients By 16S rDNA Sequencing

[Source](https://doi.org/10.3390/ijerph18042174)

---

## Integrated Gut Virome And Bacteriome Dynamics In COVID-19 Patients

[Source](https://doi.org/10.1080/19490976.2021.1887722)

---

## Six-month Follow-up Of Gut Microbiota Richness In Patients With COVID-19

[Source](https://doi.org/10.1136/gutjnl-2021-324090)

---

## Gut Mycobiota Alterations In Patients With COVID-19 And H1N1 Infections And Their associations with clinical features

[Source](https://doi.org/10.1038/s42003-021-02036-x)

---

## Gut Microbiome And Resistome Changes During The First Wave Of The COVID-19 Pandemic in comparison with pre-pandemic travel-related changes

[Source](https://doi.org/10.1093/jtm/taab067)

---

## Gut Microbiota May Not Be Fully Restored In Recovered COVID-19 Patients After 3-Month Recovery

[Source](https://doi.org/10.3389/fnut.2021.638825)

---

## Gut Microbiome Composition Is Associated With COVID-19 Disease Severity

[Source](https://doi.org/10.1053/j.gastro.2021.05.006)

---

## The Gut Microbiome Of COVID-19 Recovered Patients Returns To Uninfected Status In A minority-dominated United States cohort

[Source](https://doi.org/10.1080/19490976.2021.1926840)

---

## Intestinal Permeability Changes With Bacterial Translocation As Key Events modulating systemic host immune response to SARS-CoV-2: A working hypothesis

[Source](https://doi.org/10.1016/j.dld.2020.09.009)

---

## The Potential Application Of Probiotics And Prebiotics For The Prevention And treatment of COVID-19

[Source](https://doi.org/10.1038/s41538-020-00078-9)

---

## The Gut Microbiome: An Under-recognised Contributor To The COVID-19 Pandemic?

[Source](https://doi.org/10.1177/1756284820974914)

---

## Is Gut Microbiota Dysbiosis A Predictor Of Increased Susceptibility To Poor Outcome of COVID-19 Patients? An Update

[Source](https://doi.org/10.3390/microorganisms90100530)

---

## Do An Altered Gut Microbiota And An Associated Leaky Gut Affect COVID-19 Severity?

[Source](https://doi.org/10.1128/mBio.03022-20)

---

## The Microbiome Of Bat Guano: For What Is This Knowledge Important?

[Source](https://doi.org/10.1007/s00253-021-11143-y)

---

## Microbiota Modulation Of The Gut-Lung Axis In COVID-19

[Source](https://doi.org/10.3389/fimmu.2021.635471)

---

## Next-Generation Sequencing Reveals The Progression Of COVID-19

[Source](https://doi.org/10.3389/fcimb.2021.632490)

---

## Metagenomics Approaches To Investigate The Gut Microbiome Of COVID-19 Patients

[Source](https://doi.org/10.1177/1177932221999428)

---

## https://www.frontiersin.org/articles/10.3389/fmicb.2020.01388/full

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.01388/full)

---

## https://www.frontiersin.org/articles/10.3389/fcimb.2020.576551/full

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2020.576551/full)

---

## https://www.sciencedirect.com/science/article/pii/S0306987720333673

[Source](https://www.sciencedirect.com/science/article/pii/S0306987720333673)

---

## https://www.dldjournalonline.com/article/S1590-8658(20)30474-6/fulltext

[Source](https://www.dldjournalonline.com/article/S1590-8658(20)30474-6/fulltext)

---

## https://theconversation.com/a-healthy-microbiome-builds-a-strong-immune-system-that-could-help-defeat-covid-19-145668

[Source](https://theconversation.com/a-healthy-microbiome-builds-a-strong-immune-system-that-could-help-defeat-covid-19-145668)

---

## https://www.medrxiv.org/content/10.1101/2021.01.05.20249061v1

[Source](https://www.medrxiv.org/content/10.1101/2021.01.05.20249061v1)

---

## https://www.newsweek.com/gut-microbes-could-predict-how-seriously-ill-covid-19-patient-might-get-preliminary-study-suggests-1500568

[Source](https://www.newsweek.com/gut-microbes-could-predict-how-seriously-ill-covid-19-patient-might-get-preliminary-study-suggests-1500568)

---

## https://theconversation.com/gut-reaction-how-the-gut-microbiome-may-influence-the-severity-of-covid-19-139094

[Source](https://theconversation.com/gut-reaction-how-the-gut-microbiome-may-influence-the-severity-of-covid-19-139094)

---

## https://www.frontiersin.org/articles/10.3389/fmed.2021.652464/full

[Source](https://www.frontiersin.org/articles/10.3389/fmed.2021.652464/full)

---

## <span>Like</span>

[Source](https://microbiomedigest.com/2021/06/21/microbiome-and-covid19-some-research-highlights/?like_comment=27894&_wpnonce=7c6d7910a2)

---

## https://www.frontiersin.org/articles/10.3389/fmicb.2021.637430/full

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2021.637430/full)

---

## <span>Like</span>

[Source](https://microbiomedigest.com/2021/06/21/microbiome-and-covid19-some-research-highlights/?like_comment=27895&_wpnonce=9f21138a25)

---

