## Tet2 deficiency drives liver microbiome dysbiosis triggering Tc1 cell autoimmune hepatitis.
 06.06.2022, _SP Pandey, MJ Bender, AC McPherson, CM Phelps, LM Sanchez, M Rana, L Hedden, KA Sangani, L Chen, JH Shapira, M Siller, C Goel, EF Verdú, B Jabri, A Chang, UR Chandran, SJ Mullett, SG Wendell, AD Singhi, JS Tilstra, JF Pierre, GE Arteel, R Hinterleitner, M Meisel_


The triggers that drive interferon-γ (IFNγ)-producing CD8 T cell (Tc1 cell)-mediated autoimmune hepatitis (AIH) remain obscure. Here, we show that lack of hematopoietic Tet methylcytosine dioxygenase 2 (Tet2), an epigenetic regulator associated with autoimmunity, results in the development of microbiota-dependent AIH-like pathology, accompanied by hepatic enrichment of aryl hydrocarbon receptor (AhR) ligand-producing pathobionts and rampant Tc1 cell immunity. We report that AIH-like disease development is dependent on both IFNγ and AhR signaling, as blocking either reverts ongoing AIH-like pathology. Illustrating the critical role of AhR-ligand-producing pathobionts in this condition, hepatic translocation of the AhR ligand indole-3-aldehyde (I3A)-releasing Lactobacillus reuteri is sufficient to trigger AIH-like pathology. Finally, we demonstrate that I3A is required for L. reuteri-induced Tc1 cell differentiation in vitro and AIH-like pathology in vivo, both of which are restrained by Tet2 within CD8 T cells. This AIH-disease model may contribute to the development of therapeutics to alleviate AIH.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312822002669)

10.1016/j.chom.2022.05.006

---

## Mucosal host–microbe interactions associate with clinical phenotypes in inflammatory bowel disease

[Source](https://www.biorxiv.org/content/10.1101/2022.06.04.494807v1.abstract)

---

## Gut microbiome mediates the protective effects of exercise after myocardial infarction.
 07.06.2022, _Q Zhou, J Deng, X Pan, D Meng, Y Zhu, Y Bai, C Shi, Y Duan, T Wang, X Li, JP Sluijter, J Xiao_


_Animals, Feces, Gastrointestinal Microbiome, Mice, Mice, Inbred C57BL, Microbiota, Myocardial Infarction_
Gut microbiota plays important roles in health maintenance and diseases. Physical exercise has been demonstrated to be able to modulate gut microbiota. However, the potential role of gut microbiome in exercise protection to myocardial infarction (MI) remains unclear.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01271-6)

10.1186/s40168-022-01271-6

---

## Turtle ants harbor metabolically versatile microbiomes with conserved functions across development and phylogeny.
 06.06.2022, _B Béchade, Y Hu, JG Sanders, CS Cabuslay, P Łukasik, BR Williams, VJ Fiers, R Lu, JT Wertz, JA Russell_


Gut bacterial symbionts can support animal nutrition by facilitating digestion and providing valuable metabolites. However, changes in symbiotic roles between immature and adult stages are not well documented, especially in ants. Here, we explored the metabolic capabilities of microbiomes sampled from herbivorous turtle ant (Cephalotes sp.) larvae and adult workers through (meta)genomic screening and in vitro metabolic assays. We reveal that larval guts harbor bacterial symbionts with impressive metabolic capabilities, including catabolism of plant and fungal recalcitrant dietary fibers and energy-generating fermentation. Additionally, several members of the specialized adult gut microbiome, sampled downstream of an anatomical barrier that dams large food particles, show a conserved potential to depolymerize many dietary fibers. Symbionts from both life stages have the genomic capacity to recycle nitrogen and synthesize amino acids and B-vitamins. With help of their gut symbionts, including several bacteria likely acquired from the environment, turtle ant larvae may aid colony digestion and contribute to colony-wide nitrogen, B-vitamin and energy budgets. In addition, the conserved nature of the digestive capacities among adult-associated symbionts suggests that nutritional ecology of turtle ant colonies has long been shaped by specialized, behaviorally-transferred gut bacteria with over 45 million years of residency.

[Source](https://academic.oup.com/femsec/advance-article/doi/10.1093/femsec/fiac068/6602351)

10.1093/femsec/fiac068

---

## Alpine headwaters emerging from glaciers and rock glaciers host different bacterial communities: Ecological implications for the future.
 29.04.2020, _M Tolotti, L Cerasino, C Donati, M Pindo, M Rogora, R Seppi, D Albanese_


_Biodiversity, Ecosystem, Ice Cover, Italy, Rivers_
Mountain glacier shrinkage represents a major effect of the current global warming and 80-100% of the Alpine glaciers are predicted to vanish within the next few decades. As the thawing rate of mountain permafrost ice is much lower than for glacier ice, a shift from glacial to periglacial dynamics is predicted for Alpine landscapes during the 21st century. Despite the growing literature on the impacts of deglaciation on Alpine hydrology and ecosystems, chemical and biological features of waters emerging from Alpine rock glaciers (i.e. permafrost landforms composed by a mixture of ice and debris) have been poorly investigated so far, and knowledge on microbial biodiversity of headwaters is still sparse. A set of glacier-, rock glacier- and groundwater/precipitation-fed streams was investigated in the Italian Central Alps in late summer 2016, aiming at exploring bacterial community composition and diversity in epilithic and surface sediment biofilm and at verifying the hypothesis that rock glacier-fed headwaters represent peculiar ecosystems from both a chemical and biological point of view. Rock glacier-fed waters showed high values of electrical conductivity and trace elements related to their bedrock lithology, and their highly diverse bacterial assemblages significantly differed from those detected in glacier-fed streams. Bacterial taxonomic composition appeared to be mainly related to water and substrate type, as well as to water chemistry, the latter including concentrations of nutrients and trace metals. The results of this study confirm the chemical and biological peculiarity of rock glacier-fed waters compared to glacial waters, and suggest a potential driving role of thawing permafrost in modulating future ecological traits of Alpine headwaters within the context of progressing deglaciation.

[Source](https://www.biorxiv.org/content/10.1101/2022.06.04.494832v1.abstract)

10.1016/j.scitotenv.2020.137101

---

## Summer dynamics of microbial diversity on a mountain glacier

[Source](https://www.biorxiv.org/content/10.1101/2022.06.04.494832v1.abstract)

---

## MIAOME: Human microbiome affect the host epigenome.
 06.06.2022, _L Wang, W Zhang, X Wu, X Liang, L Cao, J Zhai, Y Yang, Q Chen, H Liu, J Zhang, Y Ding, F Zhu, J Tang_


Besides the genetic factors having tremendous influences on the regulations of the epigenome, the microenvironmental factors have recently gained extensive attention for their roles in affecting the host epigenome. There are three major types of microenvironmental factors: microbiota-derived metabolites (MDM), microbiota-derived components (MDC) and microbiota-secreted proteins (MSP). These factors can regulate host physiology by modifying host gene expression through the three highly interconnected epigenetic mechanisms (e.g. histone modifications, DNA modifications, and non-coding RNAs). However, no database was available to provide the comprehensive factors of these types. Herein, a database entitled 'Human Microbiome Affect The Host Epigenome (MIAOME)' was constructed. Based on the types of epigenetic modifications confirmed in the literature review, the MIAOME database captures 1068 (63 genus, 281 species, 707 strains, etc.) human microbes, 91 unique microbiota-derived metabolites &amp; components (16 fatty acids, 10 bile acids, 10 phenolic compounds, 10 vitamins, 9 tryptophan metabolites, etc.) derived from 967 microbes; 50 microbes that secreted 40 proteins; 98 microbes that directly influence the host epigenetic modification, and provides 3 classifications of the epigenome, including (1) 4 types of DNA modifications, (2) 20 histone modifications and (3) 490 ncRNAs regulations, involved in 160 human diseases. All in all, MIAOME has compiled the information on the microenvironmental factors influence host epigenome through the scientific literature and biochemical databases, and allows the collective considerations among the different types of factors. It can be freely assessed without login requirement by all users at: http://miaome.idrblab.net/ttd/.

[Source](https://www.sciencedirect.com/science/article/pii/S2001037022001805)

10.1016/j.csbj.2022.05.024

---

## Beta-diversity distance matrices for microbiome sample size and power calculations — How to obtain good estimates

[Source](https://www.sciencedirect.com/science/article/pii/S2001037022001490)

---

## phyloMDA: an R package for phylogeny-aware microbiome data analysis

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-04744-5)

---

## MeShClust v3.0: high-quality clustering of DNA sequences using the mean shift algorithm and alignment-free identity scores.
 06.06.2022, _HZ Girgis_


Tools for accurately clustering biological sequences are among the most important tools in computational biology. Two pioneering tools for clustering sequences are CD-HIT and UCLUST, both of which are fast and consume reasonable amounts of memory; however, there is a big room for improvement in terms of cluster quality. Motivated by this opportunity for improving cluster quality, we applied the mean shift algorithm in MeShClust v1.0. The mean shift algorithm is an instance of unsupervised learning. Its strong theoretical foundation guarantees the convergence to the true cluster centers. Our implementation of the mean shift algorithm in MeShClust v1.0 was a step forward. In this work, we scale up the algorithm by adapting an out-of-core strategy while utilizing alignment-free identity scores in a new tool: MeShClust v3.0.

[Source](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-022-08619-0)

10.1186/s12864-022-08619-0

---

