## gutSMASH

[Source](https://gutsmash.bioinformatics.nl/)

---

## AMPSphere

[Source](https://zenodo.org/record/4574469#.YEI4HhERWV4)

---

##  Green Algorithms calculator

[Source](http://www.green-algorithms.org)

---

## Comparison of new versus reused Harmonic scalpel performance in laparoscopic appendectomy in patients with acute appendicitis-a randomized clinical trial.
 09.02.2021, _J Mihanović, NL Šikić, I Mrklić, Z Katušić, R Karlo, M Jukić, A Jerončić, Z Pogorelić_


The aim of this study was to compare lateral thermal damage of the appendix and clinical outcomes after laparoscopic appendectomy using new versus reused Harmonic scalpels.

[Source](https://pratheepaj.github.io/diffTop/)

10.1007/s00423-020-02039-y

---

## a reproducible research appendix

[Source](https://pratheepaj.github.io/diffTop/)

---

## The Transbiome project

[Source](http://gigasciencejournal.com/blog/transbiome-guest-post/)

---

## here

[Source](https://en.goteo.org/project/transbiome)

---

## Link between microbiota and hypertension: Focus on LPS/TLR4 pathway in endothelial dysfunction and vascular inflammation, and therapeutic implication of probiotics.
 22.02.2021, _A Grylls, K Seidler, J Neil_


High blood pressure (BP) presents a significant public health challenge. Recent findings suggest that altered microbiota can exert a hypertensive effect on the host. One of the possible mechanisms involved is the chronic translocation of its components, mainly lipopolysaccharides (LPS) into systemic circulation leading to metabolic endotoxemia. In animal models, LPS has been commonly used to induce endothelial dysfunction and vascular inflammation. In human studies, plasma LPS concentration has been positively correlated with hypertension, however, the mechanistic link has not been fully elucidated. It is hypothesised here that the LPS-induced direct alterations to the vascular endothelium and resulting hypertension are possible targets for probiotic intervention. The methodology of this review involved a systematic search of the literature with critical appraisal of papers. Three tranches of search were performed: 1) existing review papers; 2) primary mechanistic animal, in vitro and human studies; and 3) primary intervention studies. A total of 70 peer-reviewed papers were included across the three tranches and critically appraised using SIGN50 for human studies and the ARRIVE guidelines for animal studies. The extracted information was coded into key themes and summarized in a narrative analysis. Results highlight the role of LPS in the activation of endothelial toll-like receptor 4 (TLR4) initiating a cascade of interrelated signalling pathways including: 1) Nicotinamide adenine dinucleotide phosphate (NADPH) oxidase/ Reactive oxygen species (ROS)/ Endothelial nitric oxide synthase (eNOS) pathway leading to endothelial dysfunction; and 2) Mitogen-Activated Protein Kinase (MAPK) and Nuclear factor kappa B (NF-κB) pathways leading to vascular inflammation. Findings from animal intervention studies suggest an improvement in vasorelaxation, vascular inflammation and hypertension following probiotic supplementation, which was mediated by downregulation of LPS-induced pathways. Randomised controlled trials (RCTs) and systematic reviews provided some evidence for the anti-inflammatory effect of probiotics with statistically significant antihypertensive effect in clinical samples and may offer a viable intervention for the management of hypertension.

[Source](https://www.biorxiv.org/content/10.1101/2021.02.25.432841v1.full.pdf)

10.1016/j.biopha.2021.111334

---

## A systematic analysis of metabolic pathways in the human gut microbiota

[Source](https://www.biorxiv.org/content/10.1101/2021.02.25.432841v1.full.pdf)

---

## Strain inheritance and neonatal gut microbiota development: A meta-analysis.
 10.03.2021, _D Podlesny, WF Fricke_


As many inflammatory and metabolic disorders have been associated with structural deficits of the human gut microbiota, the principles and mechanisms that govern its initialization and development are of considerable scientific interest and clinical relevance. However, our current understanding of the developing gut microbiota dynamics remains incomplete. We carried out a large-scale, comprehensive meta-analysis of over 1900 available metagenomic shotgun samples from neonates, infants, adolescents, and their families, using our recently introduced SameStr program for strain-level microbiota profiling and the detection of microbial strain transfer and persistence. We found robust associations between fecal microbiota composition and age, as well as delivery mode, which was measurable for up to two years of life. C-section was associated with increased relative abundances of non-gut species and delayed transition from a predominantly oxygen-tolerant to intolerant microbial community. Unsupervised networks based on shared strain profiles generated family-specific clusters connecting infants, their siblings, parents and grandparents and, in one case, suggested strain transfer between neonates from the same hospital ward, but could also be used to identify potentially mislabeled metagenome samples. Vaginally delivered newborns shared more strains with their mothers than C-section infants, but strain sharing was reduced if mothers underwent antibiotic treatment. Shared strains persisted in infants throughout the first year of life and belonged to the same bacterial species as strains that were shared between adults and their parents. Irrespective of delivery type, older children shared strains with their mothers and fathers and, into adulthood, could be accurately distinguished from unrelated sample pairs. Prominent fecal commensal bacteria were both among frequently transferred (e.g. Bacteroides and Sutterella) and newly acquired taxa (e.g. Blautia, Faecalibacterium, and Ruminococcus). Our meta-analysis presents a more detailed and comprehensive picture of the highly dynamic neonatal and infant fecal microbiota development than previous studies and presents evidence for taxonomic and functional compositional differences early in life between infants born naturally or by C-section, which persist well into adolescence.

[Source](https://www.sciencedirect.com/science/article/pii/S1438422121000126)

10.1016/j.ijmm.2021.151483

---

## Adaptive ecological processes and metabolic independence drive microbial colonization and resilience in the human gut

[Source](https://www.biorxiv.org/content/10.1101/2021.03.02.433653v1.full.pdf)

---

## Effect of fecal microbiota transplantation on neurological restoration in a spinal cord injury mouse model: involvement of brain-gut axis

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01007-y)

---

## Association between antibiotics and gut microbiome dysbiosis in children: systematic review and meta-analysis.
 06.03.2021, _L McDonnell, A Gilkes, M Ashworth, V Rowland, TH Harries, D Armstrong, P White_


Antibiotics in childhood have been linked with diseases including asthma, juvenile arthritis, type 1 diabetes, Crohn's disease and mental illness. The underlying mechanisms are thought related to dysbiosis of the gut microbiome. We conducted a systematic review of the association between antibiotics and disruption of the pediatric gut microbiome. Searches used MEDLINE, EMBASE and Web of Science. Eligible studies: association between antibiotics and gut microbiome dysbiosis; children 0-18 years; molecular techniques of assessment; outcomes of microbiome richness, diversity or composition. Quality assessed by Newcastle-Ottawa Scale or Cochrane Risk of Bias Tool. Meta-analysis where possible. A total of 4,668 publications identified: 12 in final analysis (5 randomized controlled trials (RCTs), 5 cohort studies, 2 cross-sectional studies). Microbiome richness was measured in 3 studies, species diversity in 6, and species composition in 10. Quality of evidence was good or fair. 5 studies found a significant reduction in diversity and 3 a significant reduction in richness. Macrolide exposure was associated with reduced richness for twice as long as penicillin. Significant reductions were seen in Bifidobacteria (5 studies) and Lactobacillus (2 studies), and significant increases in Proteobacteria such as E. coli (4 studies). A meta-analysis of RCTs of the effect of macrolide (azithromycin) exposure on the gut microbiome found a significant reduction in alpha-diversity (Shannon index: mean difference -0.86 (95% CI -1.59, -0.13). Antibiotic exposure was associated with reduced microbiome diversity and richness, and with changes in bacterial abundance. The potential for dysbiosis in the microbiome should be taken into account when prescribing antibiotics for children.Systematic review registration number: CRD42018094188.

[Source](https://www.tandfonline.com/doi/epub/10.1080/19490976.2020.1870402?needAccess=true)

10.1080/19490976.2020.1870402

---

## Bi- and Multi-directional Gene Transfer in the Natural Populations of Polyvalent Bacteriophages, and Their Host Species Spectrum Representing Foodborne Versus Other Human and/or Animal Pathogens.
 23.01.2021, _E Gabashvili, S Kobakhidze, S Koulouris, T Robinson, M Kotetishvili_


Unraveling the trends of phage-host versus phage-phage coevolution is critical for avoiding possible undesirable outcomes from the use of phage preparations intended for therapeutic, food safety or environmental safety purposes. We aimed to investigate a phenomenon of intergeneric recombination and its trajectories across the natural populations of phages predominantly linked to foodborne pathogens. The results from the recombination analyses, using a large array of the recombination detection algorithms imbedded in SplitsTree, RDP4, and Simplot software packages, provided strong evidence (fit: 100; P ≤ 0.014) for both bi- and multi-directional intergeneric recombination of the genetic loci involved collectively in phage morphogenesis, host specificity, virulence, replication, and persistence. Intergeneric recombination was determined to occur not only among conspecifics of the virulent versus temperate phages but also between the phages with these different lifestyles. The recombining polyvalent phages were suggested to interact with fairly large host species networks, including sometimes genetically very distinct species, such as e.g., Salmonella enterica and/or Escherichia coli versus Staphylococcus aureus or Yersinia pestis. Further studies are needed to understand whether phage-driven intergeneric recombination can lead to undesirable changes of intestinal and other microbiota in humans and animals.

[Source](https://mmbr.asm.org/content/mmbr/85/1/e00027-19.full.pdf)

10.1007/s12560-021-09460-6

---

## Salmonella versus the Microbiome

[Source](https://mmbr.asm.org/content/mmbr/85/1/e00027-19.full.pdf)

---

## Strain-Resolved Dynamics of the Lung Microbiome in Patients with Cystic Fibrosis.
 11.03.2021, _M Dmitrijeva, CR Kahlert, R Feigelman, RL Kleiner, O Nolte, WC Albrich, F Baty, C von Mering_


In cystic fibrosis, dynamic and complex communities of microbial pathogens and commensals can colonize the lung. Cultured isolates from lung sputum reveal high inter- and intraindividual variability in pathogen strains, sequence variants, and phenotypes; disease progression likely depends on the precise combination of infecting lineages. Routine clinical protocols, however, provide a limited overview of the colonizer populations. Therefore, a more comprehensive and precise identification and characterization of infecting lineages could assist in making corresponding decisions on treatment. Here, we describe longitudinal tracking for four cystic fibrosis patients who exhibited extreme clinical phenotypes and, thus, were selected from a pilot cohort of 11 patients with repeated sampling for more than a year. Following metagenomics sequencing of lung sputum, we find that the taxonomic identity of individual colonizer lineages can be easily established. Crucially, even superficially clonal pathogens can be subdivided into multiple sublineages at the sequence level. By tracking individual allelic differences over time, an assembly-free clustering approach allows us to reconstruct multiple lineage-specific genomes with clear structural differences. Our study showcases a culture-independent shotgun metagenomics approach for longitudinal tracking of sublineage pathogen dynamics, opening up the possibility of using such methods to assist in monitoring disease progression through providing high-resolution routine characterization of the cystic fibrosis lung microbiome.IMPORTANCE Cystic fibrosis patients frequently suffer from recurring respiratory infections caused by colonizing pathogenic and commensal bacteria. Although modern therapies can sometimes alleviate respiratory symptoms by ameliorating residual function of the protein responsible for the disorder, management of chronic respiratory infections remains an issue. Here, we propose a minimally invasive and culture-independent method to monitor microbial lung content in patients with cystic fibrosis at minimal additional effort on the patient's part. Through repeated sampling and metagenomics sequencing of our selected cystic fibrosis patients, we successfully classify infecting bacterial lineages and deconvolute multiple lineage variants of the same species within a given patient. This study explores the application of modern computational methods for deconvoluting lineages in the cystic fibrosis lung microbiome, an environment known to be inhabited by a heterogeneous pathogen population that complicates management of the disorder.

[Source](https://mbio.asm.org/content/mbio/12/2/e02863-20.full.pdf)

10.1128/mBio.02863-20

---

## Resilience of the oral microbiome.
 10.03.2021, _WG Wade_


The human mouth harbors a complex microbiota, the composition of which is potentially influenced by a wide range of factors, including the intake of food and drink, the availability of endogenous nutrients, the host immune system, drug treatments, and systemic diseases. Despite these possible influences, the oral microbiota is remarkably resilient, particularly in comparison with the microbiota of the large intestine. Diet, with the exception of excessive and/or frequent consumption of fermentable carbohydrate or supplementation with nitrate, has minimal impact on the composition of the oral bacterial community. The common oral diseases dental caries and the periodontal diseases is associated with modification of the oral microbiota primarily as a result of the ecological changes induced by excessive acid production and inflammation, respectively. Systemically-administered antimicrobials have only a small effect on the composition of the oral bacterial community, and while locally delivered antimicrobials can have some clinical benefits, the biofilm lifestyle of oral bacteria lends them substantial resistance to the agents used. Saliva plays an important role in oral microbial ecology, by supplying nutrients and providing protection against colonization by nonoral organisms. Dry mouth is one condition that has a major effect on the microbiota, resulting in increased colonization by opportunistic pathogens. Some systemic diseases do affect the oral microbiome, notably diabetes, in which raised levels of glucose in saliva and tissue impact on bacterial nutrition.

[Source](https://onlinelibrary.wiley.com/doi/epdf/10.1111/prd.12365)

10.1111/prd.12365

---

## Skin microbiota and clinical associations in Netherton sindrome

[Source](https://www.jidinnovations.org/action/showPdf?pii=S2667-0267%2821%2900008-4)

---

## Long-term stability of the urogenital microbiota of asymptomatic European women.
 28.02.2021, _M Ksiezarek, S Ugarcina-Perovic, J Rocha, F Grosso, L Peixe_


To date, information on healthy female urinary microbiota is available mostly at genus level and at one time point. However, profound species-level characterization of healthy urinary microbiome and its stability over time are essential for further correct interpretation of its role in healthy urogenital tract. In this study, we investigated female urogenital microbiome (FUM) at two timepoints (within 2.5-year interval) in young asymptomatic European women. We used culturomics with accurate isolates' identification (MALDI-TOF MS and gene markers sequencing) to understand species stability within healthy FUM.

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-021-02123-3)

10.1186/s12866-021-02123-3

---

## <em>Limosilactobacillus urinaemulieris </em>sp. nov. and <em>Limosilactobacillus portuensis</em> sp. nov. isolated from urine of healthy women

[Source](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.004726)

---

## The structure and diversity of strain-level variation in vaginal bacteria.
 03.03.2021, _BA Tortelli, AL Lewis, JC Fay_


The vaginal microbiome plays an important role in human health and species of vaginal bacteria have been associated with reproductive disease. Strain-level variation is also thought to be important, but the diversity, structure and evolutionary history of vaginal strains is not as well characterized. We developed and validated an approach to measure strain variation from metagenomic data based on SNPs within the core genomes for six species of vaginal bacteria: Gardnerella vaginalis, Lactobacillus crispatus, Lactobacillus iners, Lactobacillus jensenii, Lactobacillus gasseri and Atopobium vaginae. Despite inhabiting the same environment, strain diversity and structure varies across species. All species except L. iners are characterized by multiple distinct groups of strains. Even so, strain diversity is lower in the Lactobacillus species, consistent with a more recent colonization of the human vaginal microbiome. Both strain diversity and the frequency of multi-strain samples is related to species-level diversity of the microbiome in which they occur, suggesting similar ecological factors influencing diversity within the vaginal niche. We conclude that the structure of strain-level variation provides both the motivation and means of testing whether strain-level differences contribute to the function and health consequences of the vaginal microbiome.

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000543)

10.1099/mgen.0.000543

---

## Obesity is associated with lower bacterial vaginosis prevalence in menopausal but not pre-menopausal women in a retrospective analysis of the Women's Interagency HIV Study.
 09.03.2021, _E Daubert, KM Weber, AL French, D Seidman, K Michel, D Gustafson, K Murphy, CA Muzny, M Alcaide, A Sheth, AA Adimora, GT Spear_


The vaginal microbiota is known to impact women's health, but the biological factors that influence the composition of the microbiota are not fully understood. We previously observed that levels of glycogen in the lumen of the vagina were higher in women that had a high body mass index (BMI). Vaginal glycogen is thought to impact the composition of the vaginal microbiota. We therefore sought to determine if BMI was associated having or not having bacterial vaginosis (BV), as determined by the Amsel criteria. We also hypothesized that increased blood glucose levels could lead to the previously-observed higher vaginal glycogen levels and therefore investigated if hemoglobin A1c levels were associated with BV. We analyzed data from the Women's Interagency HIV Study using multiple multivariable (GEE) logistic regression models to assess the relationship between BMI, BV and blood glucose. Women with a BMI &gt;30 kg/m2 (obese) had a lower rate (multivariable adjusted OR 0.87 (0.79-0.97), p = 0.009) of BV compared to the reference group (BMI 18.5-24.9 kg/m2). There was a significantly lower rate of BV in post-menopausal obese women compared to the post-menopausal reference group, but not in pre-menopausal women. HIV- post-menopausal obese women had a significantly lower rate of BV, but this was not seen in HIV+ post-menopausal obese women. Pre-menopausal women with a higher hemoglobin A1c (≥6.5%) had a significantly lower rate (multivariable adjusted OR 0.66 (0.49-0.91), p = 0.010) of BV compared to pre-menopausal women with normal hemoglobin A1c levels (&lt;5.7%), but there was no difference in post-menopausal women. This study shows an inverse association of BMI with BV in post-menopausal women and hemoglobin A1c with BV in pre-menopausal women. Further studies are needed to confirm these relationships in other cohorts across different reproductive stages and to identify underlying mechanisms for these observed associations.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0248136)

10.1371/journal.pone.0248136

---

## Bacterial biofilm formation on vaginal ring pessaries used for pelvic organ prolapse.
 04.03.2021, _FG Gould, MP Carey, EL Plummer, GL Murray, JA Danielewski, SN Tabrizi, SM Garland_


The objective of this study was to characterize the bacterial biofilm on vaginal ring pessaries used to treat pelvic organ prolapse and investigate the relationship between biofilm phenotype and patient symptoms and clinical signs that are suggestive of inflammation.

[Source](https://link.springer.com/article/10.1007/s00192-021-04717-x)

10.1007/s00192-021-04717-x

---

## Long-term storage of feces at −80 °C versus −20 °C is negligible for 16S rRNA amplicon profiling of the equine bacterial microbiome

[Source](https://peerj.com/articles/10837/)

---

## Differences in the gut microbiomes of dogs and wolves: roles of antibiotics and starch.
 10.03.2021, _Y Liu, B Liu, C Liu, Y Hu, C Liu, X Li, X Li, X Zhang, DM Irwin, Z Wu, Z Chen, Q Jin, S Zhang_


Dogs are domesticated wolves. Change of living environment, such as diet and veterinary care may affect the gut bacterial flora of dogs. The aim of this study was to assess the gut bacterial diversity and function in dogs compared with captive wolves. We surveyed the gut bacterial diversity of 27 domestic dogs, which were fed commercial dog food, and 31 wolves, which were fed uncooked meat, by 16S rRNA sequencing. In addition, we collected fecal samples from 5 dogs and 5 wolves for shotgun metagenomic sequencing to explore changes in the functions of their gut microbiome.

[Source](https://bmcvetres.biomedcentral.com/articles/10.1186/s12917-021-02815-y)

10.1186/s12917-021-02815-y

---

## An Expanded Gene Catalog of Mouse Gut Metagenomes.
 25.02.2021, _J Zhu, H Ren, H Zhong, X Li, Y Zou, M Han, M Li, L Madsen, K Kristiansen, L Xiao_


High-quality and comprehensive reference gene catalogs are essential for metagenomic research. The rather low diversity of samples used to construct existing catalogs of the mouse gut metagenome limits the numbers of identified genes in existing catalogs. We therefore established an expanded catalog of genes in the mouse gut metagenome (EMGC) containing &gt;5.8 million genes by integrating 88 newly sequenced samples, 86 mouse gut-related bacterial genomes, and 3 existing gene catalogs. EMGC increases the number of nonredundant genes by more than 1 million genes compared to the so-far most extensive catalog. More than 60% of the genes in EMGC were assigned to Bacteria, with 54.20% being assigned to a phylum and 35.33% to a genus, while 30.39% were annotated at the KEGG orthology level. Nine hundred two metagenomic species (MGS) assigned to 122 taxa are identified based on the EMGC. The EMGC-based analysis of samples from groups of mice originating from different animal providers, housing laboratories, and genetic strains substantiated that diet is a major contributor to differences in composition and functional potential of the gut microbiota irrespective of differences in environment and genetic background. We envisage that EMGC will serve as a valuable reference data set for future metagenomic studies in mice.IMPORTANCE We established an expanded gene catalog of the mouse gut metagenome not only to increase the sample size compared to that in existing catalogs but also to provide a more comprehensive reference data set of the mouse gut microbiome for bioinformatic analysis. The expanded gene catalog comprises more than 5.8 million unique genes, as well as a wide range of taxonomic and functional information. Particularly, the analysis of metagenomic species with the expanded gene catalog reveals a great novelty of mouse gut-inhabiting microbial species. We envisage that the expanded gene catalog of the mouse gut metagenome will serve as a valuable bioinformatic resource for future gut metagenomic studies in mice.

[Source](https://msphere.asm.org/content/msph/6/1/e01119-20.full.pdf)

10.1128/mSphere.01119-20

---

## Long-term evolution and short-term adaptation of microbiota strains and sub-strains in mice.
 04.03.2021, _B Yilmaz, C Mooser, I Keller, H Li, J Zimmermann, L Bosshard, T Fuhrer, M Gomez de Agüero, NF Trigo, H Tschanz-Lischer, JP Limenitakis, WD Hardt, KD McCoy, B Stecher, L Excoffier, U Sauer, SC Ganal-Vonarburg, AJ Macpherson_


Isobiotic mice, with an identical stable microbiota composition, potentially allow models of host-microbial mutualism to be studied over time and between different laboratories. To understand microbiota evolution in these models, we carried out a 6-year experiment in mice colonized with 12 representative taxa. Increased non-synonymous to synonymous mutation rates indicate positive selection in multiple taxa, particularly for genes annotated for nutrient acquisition or replication. Microbial sub-strains that evolved within a single taxon can stably coexist, consistent with niche partitioning of ecotypes in the complex intestinal environment. Dietary shifts trigger rapid transcriptional adaptation to macronutrient and micronutrient changes in individual taxa and alterations in taxa biomass. The proportions of different sub-strains are also rapidly altered after dietary shift. This indicates that microbial taxa within a mouse colony adapt to changes in the intestinal environment by long-term genomic positive selection and short-term effects of transcriptional reprogramming and adjustments in sub-strain proportions.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(21)00047-0)

10.1016/j.chom.2021.02.001

---

## The gut mycobiome of healthy mice is shaped by the environment and correlates with metabolic outcomes in response to diet.
 10.03.2021, _TS Mims, QA Abdallah, JD Stewart, SP Watts, CT White, TV Rousselle, A Gosain, A Bajwa, JC Han, KA Willis, JF Pierre_


As an active interface between the host and their diet, the gut microbiota influences host metabolic adaptation; however, the contributions of fungi have been overlooked. Here, we investigate whether variations in gut mycobiome abundance and composition correlate with key features of host metabolism. We obtained animals from four commercial sources in parallel to test if differing starting mycobiomes can shape host adaptation in response to processed diets. We show that the gut mycobiome of healthy mice is shaped by the environment, including diet, and significantly correlates with metabolic outcomes. We demonstrate that exposure to processed diet leads to persistent differences in fungal communities that significantly associate with differential deposition of body mass in male mice compared to mice fed standardized diet. Fat deposition in the liver, transcriptional adaptation of metabolically active tissues and serum metabolic biomarker levels are linked with alterations in fungal community diversity and composition. Specifically, variation in fungi from the genera Thermomyces and Saccharomyces most strongly associate with metabolic disturbance and weight gain. These data suggest that host-microbe metabolic interactions may be influenced by variability in the mycobiome. This work highlights the potential significance of the gut mycobiome in health and has implications for human and experimental metabolic studies.

[Source](https://www.nature.com/articles/s42003-021-01820-z)

10.1038/s42003-021-01820-z

---

## Functional genetics of human gut commensal <em>Bacteroides thetaiotaomicron</em> reveals metabolic requirements for growth across environments

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247%2821%2900103-0#%20)

---

## Tea polyphenols regulate gut microbiota dysbiosis induced by antibiotic in mice.
 01.03.2021, _J Li, C Chen, H Yang, X Yang_


Tea polyphenols (TPs) are now widely used in foods for various biological activities. However, they are rarely used in foods to regulate gut microbiota dysbiosis induced by antibiotics. We assessed the regulation of TPs on gut microbiota with an antibiotic-induced intestinal flora disorder mouse model. The mice were orally administered with cefixime for 8 days, then received TPs for 28 days. We found that the antibiotic had a profound impact on the gut microbiota. Compared with the normal group, significant decreases in the species richness and diversity and the production of short-chain fatty acids (SCFAs) were still observed 28 days after the antibiotic treatment, although there was no significant difference in the colonic mucosa. TPs significantly alleviated the decrease of the richness and diversity of gut microbiota caused by the antibiotic treatment, and significantly increased the relative abundance of beneficial microbes such as Lactobacillus, Akkermansia, Blautia, Roseburia, and Eubacterium. The function prediction showed that TPs significantly decreased the relative abundance of genes related to human diseases, yet significantly increased the relative abundance of genes related to cell growth and death, cell motility, and energy metabolism. These showed that TPs could regulate the gut microbiota dysbiosis induced by antibiotics, thus decreasing the risk of diseases such as obesity, cancer, and diabetes. These suggest that TPs have a great potential to be used as a functional food ingredient to prevent or reduce adverse effects of antibiotics.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0963996921000508)

10.1016/j.foodres.2021.110153

---

## Designing a home for beneficial plant microbiomes.
 08.03.2021, _C Song, K Jin, JM Raaijmakers_


The plant microbiome comprises a highly diverse community of saprotrophic, mutualistic, and pathogenic microbes that can affect plant growth and plant health. There is substantial interest to exploit beneficial members of plant microbiomes for new sustainable management strategies in crop production. However, poor survival and colonization of plant tissues by introduced microbial isolates as well as lack of expression of the plant growth-promoting or disease-suppressive traits at the right time and place are still major limitations for successful implementation of microbiomes in future agricultural practices and plant breeding programs. Similar to building a home for humans, we discuss different strategies of building a home for beneficial plant microbiomes, here referred to as the 'MicrobiHome'.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S136952662100025X)

10.1016/j.pbi.2021.102025

---

## Polymicrobial interaction between <em>Lactobacillus</em> and <em>Saccharomyces cerevisiae</em>: coexistence-relevant mechanisms

[Source](https://www.tandfonline.com/doi/abs/10.1080/1040841X.2021.1893265)

---

## Viromes outperform total metagenomes in revealing the spatiotemporal patterns of agricultural soil viral communities.
 22.02.2021, _C Santos-Medellin, LA Zinke, AM Ter Horst, DL Gelardi, SJ Parikh, JB Emerson_


Viruses are abundant yet understudied members of soil environments that influence terrestrial biogeochemical cycles. Here, we characterized the dsDNA viral diversity in biochar-amended agricultural soils at the preplanting and harvesting stages of a tomato growing season via paired total metagenomes and viral size fraction metagenomes (viromes). Size fractionation prior to DNA extraction reduced sources of nonviral DNA in viromes, enabling the recovery of a vaster richness of viral populations (vOTUs), greater viral taxonomic diversity, broader range of predicted hosts, and better access to the rare virosphere, relative to total metagenomes, which tended to recover only the most persistent and abundant vOTUs. Of 2961 detected vOTUs, 2684 were recovered exclusively from viromes, while only three were recovered from total metagenomes alone. Both viral and microbial communities differed significantly over time, suggesting a coupled response to rhizosphere recruitment processes and/or nitrogen amendments. Viral communities alone were also structured along an 18 m spatial gradient. Overall, our results highlight the utility of soil viromics and reveal similarities between viral and microbial community dynamics throughout the tomato growing season yet suggest a partial decoupling of the processes driving their spatial distributions, potentially due to differences in dispersal, decay rates, and/or sensitivities to soil heterogeneity.

[Source](https://www.nature.com/articles/s41396-021-00897-y)

10.1038/s41396-021-00897-y

---

## Biases in Viral Metagenomics-Based Detection, Cataloguing and Quantification of Bacteriophage Genomes in Human Faeces, a Review

[Source](https://www.mdpi.com/2076-2607/9/3/524)

---

## High molecular weight DNA extraction strategies for long-read sequencing of complex metagenomes

[Source](https://www.biorxiv.org/content/10.1101/2021.03.03.433801v1.full.pdf)

---

## AMPSphere: the worldwide survey of prokaryotic antimicrobial peptides

[Source](https://zenodo.org/record/4574469#.YEBJOBERWV4)

---

## A Statistical Perspective on the Challenges in Molecular Microbial Biology

[Source](https://arxiv.org/abs/2103.04198)

---

## An introduction to agent-based models as an accessible surrogate to field-based research and teaching.
 01.12.2020, _KJ Murphy, S Ciuti, A Kane_


There are many barriers to fieldwork including cost, time, and physical ability. Unfortunately, these barriers disproportionately affect minority communities and create a disparity in access to fieldwork in the natural sciences. Travel restrictions, concerns about our carbon footprint, and the global lockdown have extended this barrier to fieldwork across the community and led to increased anxiety about gaps in productivity, especially among graduate students and early-career researchers. In this paper, we discuss agent-based modeling as an open-source, accessible, and inclusive resource to substitute for lost fieldwork during COVID-19 and for future scenarios of travel restrictions such as climate change and economic downturn. We describe the benefits of Agent-Based models as a teaching and training resource for students across education levels. We discuss how and why educators and research scientists can implement them with examples from the literature on how agent-based models can be applied broadly across life science research. We aim to amplify awareness and adoption of this technique to broaden the diversity and size of the agent-based modeling community in ecology and evolutionary research. Finally, we discuss the challenges facing agent-based modeling and discuss how quantitative ecology can work in tandem with traditional field ecology to improve both methods.

[Source](https://www.biorxiv.org/content/10.1101/2021.03.08.434372v1.full.pdf)

10.1002/ece3.6848

---

## The carbon footprint of bioinformatics

[Source](https://www.biorxiv.org/content/10.1101/2021.03.08.434372v1.full.pdf)

---

## Make Interactive Complex Heatmaps in R 

[Source](https://www.biorxiv.org/content/10.1101/2021.03.08.434289v2.full.pdf)

---

## Machine learning applications in microbial ecology, human microbiome studies, and environmental monitoring.
 10.03.2021, _RB Ghannam, SM Techtmann_


Advances in nucleic acid sequencing technology have enabled expansion of our ability to profile microbial diversity. These large datasets of taxonomic and functional diversity are key to better understanding microbial ecology. Machine learning has proven to be a useful approach for analyzing microbial community data and making predictions about outcomes including human and environmental health. Machine learning applied to microbial community profiles has been used to predict disease states in human health, environmental quality and presence of contamination in the environment, and as trace evidence in forensics. Machine learning has appeal as a powerful tool that can provide deep insights into microbial communities and identify patterns in microbial community data. However, often machine learning models can be used as black boxes to predict a specific outcome, with little understanding of how the models arrived at predictions. Complex machine learning algorithms often may value higher accuracy and performance at the sacrifice of interpretability. In order to leverage machine learning into more translational research related to the microbiome and strengthen our ability to extract meaningful biological information, it is important for models to be interpretable. Here we review current trends in machine learning applications in microbial ecology as well as some of the important challenges and opportunities for more broad application of machine learning to understanding microbial communities.

[Source](https://www.sciencedirect.com/science/article/pii/S2001037021000325)

10.1016/j.csbj.2021.01.028

---

## Tutorial: assessing metagenomics software with the CAMI benchmarking toolkit.
 02.03.2021, _F Meyer, TR Lesker, D Koslicki, A Fritz, A Gurevich, AE Darling, A Sczyrba, A Bremges, AC McHardy_


Computational methods are key in microbiome research, and obtaining a quantitative and unbiased performance estimate is important for method developers and applied researchers. For meaningful comparisons between methods, to identify best practices and common use cases, and to reduce overhead in benchmarking, it is necessary to have standardized datasets, procedures and metrics for evaluation. In this tutorial, we describe emerging standards in computational meta-omics benchmarking derived and agreed upon by a larger community of researchers. Specifically, we outline recent efforts by the Critical Assessment of Metagenome Interpretation (CAMI) initiative, which supplies method developers and applied researchers with exhaustive quantitative data about software performance in realistic scenarios and organizes community-driven benchmarking challenges. We explain the most relevant evaluation metrics for assessing metagenome assembly, binning and profiling results, and provide step-by-step instructions on how to generate them. The instructions use simulated mouse gut metagenome data released in preparation for the second round of CAMI challenges and showcase the use of a repository of tool results for CAMI datasets. This tutorial will serve as a reference for the community and facilitate informative and reproducible benchmarking in microbiome research.

[Source](https://www.nature.com/articles/s41596-020-00480-3)

10.1038/s41596-020-00480-3

---

## STATegra: Multi-Omics Data Integration – A Conceptual Scheme With a Bioinformatics Pipeline

[Source](https://www.frontiersin.org/articles/10.3389/fgene.2021.620453/full)

---

## Exploring bacterial diversity via a curated and searchable snapshot of archived DNA sequences

[Source](https://www.biorxiv.org/content/10.1101/2021.03.02.433662v1.full.pdf)

---

## clinker &amp; clustermap.js: automatic generation of gene cluster comparison figures

[Source](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btab007/6103786)

---

## gapseq: informed prediction of bacterial metabolic pathways and reconstruction of accurate metabolic models.
 11.03.2021, _J Zimmermann, C Kaleta, S Waschina_


Genome-scale metabolic models of microorganisms are powerful frameworks to predict phenotypes from an organism's genotype. While manual reconstructions are laborious, automated reconstructions often fail to recapitulate known metabolic processes. Here we present gapseq ( https://github.com/jotech/gapseq ), a new tool to predict metabolic pathways and automatically reconstruct microbial metabolic models using a curated reaction database and a novel gap-filling algorithm. On the basis of scientific literature and experimental data for 14,931 bacterial phenotypes, we demonstrate that gapseq outperforms state-of-the-art tools in predicting enzyme activity, carbon source utilisation, fermentation products, and metabolic interactions within microbial communities.

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02295-1)

10.1186/s13059-021-02295-1

---

## mSystems Thinking Series: Diversity, Equity, and Inclusion in Microbiology 

[Source](https://asmusa-org.zoom.us/webinar/register/WN_9ZIMp_o6QUapQpZWDnmTaA)

---

## The Microbiome Centers Consortium Seminar Series 

[Source](https://docs.google.com/document/d/1FZWTdsocs9wGpbABF4KOjq6CfIWqtF18RI8_r7gWYfI/edit)

---

## Americas Webinar: Mother’s Milk &amp; Baby’s Bacteria: Discoveries from the CHILD Cohort Study

[Source](https://www.isrhml.com/i4a/ams/meetings/index.cfm?controller=meetings&action=startRegistration&conferenceID=94&reginit=1&pageID=3436)

---

## Get R Done: R Tutorials video series

[Source](https://www.youtube.com/playlist?list=PLmNgrNF3pZqg2GPmcIAWDa1A2-cAfbX37)

---

## Machine Learning in Genomics: Tools, Resources, Clinical Applications and Ethics

[Source](https://www.genome.gov/event-calendar/Machine-Learning-in-Genomics-Tools-Resources-Clinical-Applications-and-Ethics)

---

## February 2021 Pod: Global Chang‪e‬

[Source](https://podcasts.apple.com/se/podcast/february-2021-pod-global-change/id1509550298?i=1000511103116)

---

## Enhancers with Molly Gasperini 

[Source](https://bioinformatics.chat/enhancers)

---

## Matthew Schechter: Metagenomics History Part 1 and 2

[Source](https://anchor.fm/bioinfopod/episodes/Matthew-Schechter-Metagenomics-History-Part-1-eppqeh)

---

## Ep. 33 Oyster Microbiome with Dr. Rachel Diner 

[Source](https://planetb612.fm/ep-33-oyster-microbiome-with-dr-rachel-diner/)

---

## UMich Microbial Masterpieces 

[Source](https://sites.google.com/umich.edu/microbialmasterpieces)

---

