## Mechanism of Dyspnea during Exercise in Children with Corrected Congenital Heart Disease.
 24.01.2022, _M Chlif, MM Ammar, NB Said, L Sergey, S Ahmaidi, F Alassery, H Hamam_


_Child, Dyspnea, Exercise, Exercise Test, Heart Defects, Congenital, Humans, Oxygen Consumption_
This study will evaluate cardiorespiratory and peripheral muscle function and their relationship with subjective dyspnea threshold after the surgical correction of congenital heart disease in children. Thirteen children with surgically repaired congenital heart disease were recruited. Each participant performed an incremental exercise test on a cycle ergometer until exhaustion. Gas exchanges were continuously sampled to measure the maximal aerobic parameters and ventilatory thresholds. The functional capacity of the subjects was assessed with a 6 min walk test. At the end of the exercise test, isokinetic Cybex Norm was used to evaluate the strength and endurance of the knee extensor muscle in the leg. Dyspnea was subjectively scored with a visual analog scale during the last 15 s of each exercise step. Oxygen consumption measured at the dyspnea score/VO2 relationship located at the dyspnea threshold, at which dyspnea suddenly increased. Results: The maximal and submaximal values of the parameters describing the exercise and the peripheral muscular performances were: VO2 Peak: 33.8 ± 8.9 mL·min-1·kg-1; HR: 174 ± 9 b·min-1; VEmax: 65.68 ± 15.9 L·min-1; P max: 117 ± 27 W; maximal voluntary isometric force MVIF: 120.8 ± 41.9 N/m; and time to exhaustion Tlim: 53 ± 21 s. Oxygen consumption measured at the dyspnea threshold was related to VO2 Peak (R2 = 0.74; p &lt; 0.01), Tlim (R2 = 0.78; p &lt; 0.01), and the distance achieved during the 6MWT (R2 = 0.57; p &lt; 0.05). Compared to the theoretical maximal values for the power output, VO2, and HR, the surgical correction did not repair the exercise performance. After the surgical correction of congenital heart disease, exercise performance was impeded by alterations of the cardiorespiratory function and peripheral local factors. A subjective evaluation of the dyspnea threshold is a reliable criterion that allows the prediction of exercise capacity in subjects suffering from congenital heart disease.

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

10.3390/ijerph19010099

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/x4RUYfkN0P

[Source](https://t.co/x4RUYfkN0P)

---

## The usefulness of the endoscopic surgical skill qualification system in laparoscopic right hemicolectomy: a single-center, retrospective analysis with propensity score matching.
 16.01.2023, _M Iwamoto, Y Makutani, Y Yane, H Ushijima, Y Yoshioka, T Wada, K Daito, T Tokoro, Y Chiba, K Ueda, J Kawamura_


Recently, a new certification system called the Endoscopic Surgical Skill Qualification System (ESSQS) has been launched in Japan to improve surgical safety. This study aimed to determine whether ESSQS-qualified surgeons affect the short- and long-term outcomes of laparoscopic right hemicolectomy.

[Source](https://twitter.com/svetlana_up/status/1612362315789393920?ref_src=twsrc%5Etfw)

10.1007/s00423-023-02810-x

---

## January 9, 2023

[Source](https://twitter.com/svetlana_up/status/1612362315789393920?ref_src=twsrc%5Etfw)

---

## Longitudinal, Multi-Platform Metagenomics Yields a High-Quality Genomic Catalog and Guides an In Vitro Model for Cheese Communities

[Source](https://journals.asm.org/doi/10.1128/msystems.00701-22)

---

## Human milk oligosaccharides, antimicrobial drugs, and the gut microbiota of term neonates: observations from the KOALA birth cohort study.
 13.01.2023, _DJM Barnett, MF Endika, CE Klostermann, F Gu, C Thijs, A Nauta, HA Schols, H Smidt, ICW Arts, J Penders_


_Infant, Infant, Newborn, Female, Animals, Pregnancy, Humans, Child, Milk, Human, Gastrointestinal Microbiome, Phascolarctidae, Cohort Studies, RNA, Ribosomal, 16S, Breast Feeding, Prebiotics, Anti-Infective Agents, Oligosaccharides, Anti-Bacterial Agents_
The infant gut microbiota affects childhood health. This pioneer microbiota may be vulnerable to antibiotic exposures, but could be supported by prebiotic oligosaccharides found in breast milk and some infant formulas. We sought to characterize the effects of several exposures on the neonatal gut microbiota, including human milk oligosaccharides (HMOs), galacto-oligosaccharides (GOS), and infant/maternal antimicrobial exposures. We profiled the stool microbiota of 1023 one-month-old infants from the KOALA Birth Cohort using 16S rRNA gene amplicon sequencing. We quantified 15 HMOs in breast milk from the mothers of 220 infants, using high-performance liquid chromatography-mass spectrometry. Both breastfeeding and antibiotic exposure decreased gut microbial diversity, but each was associated with contrasting shifts in microbiota composition. Other factors associated with microbiota composition included C-section, homebirth, siblings, and exposure to animals. Neither infant exposure to oral antifungals nor maternal exposure to antibiotics during pregnancy were associated with infant microbiota composition. Four distinct groups of breast milk HMO compositions were evident, corresponding to maternal Secretor status and Lewis group combinations defined by the presence/absence of certain fucosylated HMOs. However, we found the strongest evidence for microbiota associations between two non-fucosylated HMOs: 6'-sialyllactose (6'-SL) and lacto-N-hexaose (LNH), which were associated with lower and higher relative abundances of Bifidobacterium, respectively. Among 111 exclusively formula-fed infants, the GOS-supplemented formula was associated with a lower relative abundance of Clostridium perfringens. In conclusion, the gut microbiota is sensitive to some prebiotic and antibiotic exposures during early infancy and understanding their effects could inform future strategies for safeguarding a health-promoting infant gut microbiota.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2022.2164152)

10.1080/19490976.2022.2164152

---

## Joint Analysis of Microbial and Immune Cell Abundance in Liver Cancer Tissue Using a Gene Expression Profile Deconvolution Algorithm Combined With Foreign Read Remapping.
 16.07.2022, _D Ai, Y Xing, Q Zhang, Y Wang, X Liu, G Liu, LC Xia_


_Algorithms, Bacteria, Computational Biology, Humans, Liver Neoplasms, Software, Transcriptome, Tumor Microenvironment_
Recent transcriptomics and metagenomics studies showed that tissue-infiltrating immune cells and bacteria interact with cancer cells to shape oncogenesis. This interaction and its effects remain to be elucidated. However, it is technically difficult to co-quantify immune cells and bacteria in their respective microenvironments. To address this challenge, we herein report the development of a complete a bioinformatics pipeline, which accurately estimates the number of infiltrating immune cells using a novel Particle Swarming Optimized Support Vector Regression (PSO-SVR) algorithm, and the number of infiltrating bacterial using foreign read remapping and the GRAMMy algorithm. It also performs systematic differential abundance analyses between tumor-normal pairs. We applied the pipeline to a collection of paired liver cancer tumor and normal samples, and we identified bacteria and immune cell species that were significantly different between tissues in terms of health status. Our analysis showed that this dual model of microbial and immune cell abundance had a better differentiation (84%) between healthy and diseased tissue. Caldatribacterium sp., Acidaminococcaceae sp., Planctopirus sp., Desulfobulbaceae sp.,Nocardia farcinica as well as regulatory T cells (Tregs), resting mast cells, monocytes, M2 macrophases, neutrophils were identified as significantly different (Mann Whitney Test, FDR&lt; 0.05). Our open-source software is freely available from GitHub at https://github.com/gutmicrobes/PSO-SVR.git.

[Source](https://twitter.com/hashtag/GutMicrobes?src=hash&ref_src=twsrc%5Etfw)

10.3389/fimmu.2022.853213

---

## #GutMicrobes

[Source](https://twitter.com/hashtag/GutMicrobes?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/xMMQLXIVp6

[Source](https://t.co/xMMQLXIVp6)

---

## SPCIS: Standardized Plant Community with Introduced Status database.
 13.01.2023, _L Petri, EM Beaury, J Corbin, K Peach, H Sofaer, IS Pearse, R Early, DT Barnett, I Ibáñez, RK Peet, M Schafale, TR Wentworth, JP Vanderhorst, DN Zaya, G Spyreas, BA Bradley_


The movement of plant species across the globe exposes native communities to new species introductions. While introductions are pervasive, two aspects of variability underlie patterns and processes of biological invasions at macroecological scales. First, only a portion of introduced species become invaders capable of substantially impacting ecosystems. Second, species that do become invasive at one location may not be invasive in others; impacts depend on invader abundance and recipient species and conditions. Accounting for these phenomena is essential to accurately understand the patterns of plant invasion and explain the idiosyncratic results reflected in the literature on biological invasions. The lack of community-level richness and the abundance of data spanning broad scales and environmental conditions have until now hindered our understanding of invasions at a macroecological scale. To address this limitation, we leveraged quantitative surveys of plant communities in the USA and integrated and harmonized nine datasets into the Standardized Plant Community with Introduced Status (SPCIS) database. The database contains 14,056 unique taxa identified within 83,391 sampling units, of which 52.6% have at least one introduced species. The SPCIS database includes comparable information on plant species occurrence, abundance, and native status across the 50 U.S. States and Puerto Rico. SPCIS can be used to answer macro-scale questions about native plant communities and interactions with invasive plants. There are no copyright restrictions on the data, and we ask the users of this dataset to cite this paper, the respective paper(s) corresponding to the dataset sampling design (all references are provided in Data S1: Metadata S1: Class II-B-2), and the references described in Data S1: Metadata S1: Class III-B-4 as applicable to the dataset being utilized.

[Source](https://twitter.com/_david_barnett_?ref_src=twsrc%5Etfw)

10.1002/ecy.3947

---

## @_david_barnett_

[Source](https://twitter.com/_david_barnett_?ref_src=twsrc%5Etfw)

---

## @MF_Endika

[Source](https://twitter.com/MF_Endika?ref_src=twsrc%5Etfw)

---

## Allele-specific expression analysis for complex genetic phenotypes applied to a unique dilated cardiomyopathy cohort.
 15.01.2023, _D van Beek, J Verdonschot, K Derks, H Brunner, TM de Kok, ICW Arts, S Heymans, M Kutmon, M Adriaens_


_Humans, Alleles, Cardiomyopathy, Dilated, Gene Expression Regulation, Quantitative Trait Loci, Genome-Wide Association Study, Phenotype, Polymorphism, Single Nucleotide_
Allele-specific expression (ASE) analysis detects the relative abundance of alleles at heterozygous loci as a proxy for cis-regulatory variation, which affects the personal transcriptome and proteome. This study describes the development and application of an ASE analysis pipeline on a unique cohort of 87 well phenotyped and RNA sequenced patients from the Maastricht Cardiomyopathy Registry with dilated cardiomyopathy (DCM), a complex genetic disorder with a remaining gap in explained heritability. Regulatory processes for which ASE is a proxy might explain this gap. We found an overrepresentation of known DCM-associated genes among the significant results across the cohort. In addition, we were able to find genes of interest that have not been associated with DCM through conventional methods such as genome-wide association or differential gene expression studies. The pipeline offers RNA sequencing data processing, individual and population level ASE analyses as well as group comparisons and several intuitive visualizations such as Manhattan plots and protein-protein interaction networks. With this pipeline, we found evidence supporting the case that cis-regulatory variation contributes to the phenotypic heterogeneity of DCM. Additionally, our results highlight that ASE analysis offers an additional layer to conventional genomic and transcriptomic analyses for candidate gene identification and biological insight.

[Source](https://twitter.com/MaCSBio?ref_src=twsrc%5Etfw)

10.1038/s41598-023-27591-7

---

## @MaCSBio

[Source](https://twitter.com/MaCSBio?ref_src=twsrc%5Etfw)

---

## Effects of hatching system on chick quality, welfare and health of young breeder flock offspring.
 15.01.2023, _R Molenaar, N Stockhofe-Zurwieden, MF Giersberg, TB Rodenburg, B Kemp, H van den Brand, IC de Jong_


Alternative hatching systems have been developed for broiler chickens to provide immediately feed and water after hatch and reduce the number or severity of early life stressors. Besides beneficial effects of these alternative hatching systems on chick quality and performance, broiler health and welfare may be positively affected as well. Especially offspring from young broiler breeder flocks may benefit, as they have been shown to be more sensitive to preturbations than offspring from older breeder flocks. This study evaluated effects of hatching systems on chick quality, health and welfare of young breeder flock offspring, using 3 different hatching systems: conventional hatchery-hatched (HH), hatchery-fed (HF), and on-farm hatching (OH). A total of 24 pens were used in a completely randomized block design, with 8 pens per hatching system and 30 chickens per pen. Chick quality at hatch and performance until 35 d of age was improved in the HF and OH compared to HH treatment, but only minor effects were found on the welfare indicators: footpad dermatitis, hock burn, cleanliness, skin lesion and gait score. No effect was observed on the dynamics of a humoral immune response after NCD vaccination, given at d 0 and 14 of age, as no differences between NCD titers were found at d 18. Animals were vaccinated with a live attenuated infectious bronchitis vaccine virus (IBV) at d 28 to address treatment related differences to disease resilience. The expressions of inflammation and epithelial integrity related genes in the trachea and histo-pathological changes in the trachea were examined at 3 d after vaccine administration. No differences between treatment groups were observed. Although beneficial effects of HF and OH systems were found for young breeder flock offspring on chick quality at hatch and body weight posthatch, only one effect of alternative hatching systems on welfare and health indicators were found. No effect of hatching system on humoral immune response or disease resilience was found.

[Source](https://twitter.com/WUR?ref_src=twsrc%5Etfw)

10.1016/j.psj.2022.102448

---

## @WUR

[Source](https://twitter.com/WUR?ref_src=twsrc%5Etfw)

---

## @MIB_WUR

[Source](https://twitter.com/MIB_WUR?ref_src=twsrc%5Etfw)

---

## @MaastrichtUMC

[Source](https://twitter.com/MaastrichtUMC?ref_src=twsrc%5Etfw)

---

## January 10, 2023

[Source](https://twitter.com/penderslab/status/1612753647300911104?ref_src=twsrc%5Etfw)

---

## The gut microbiota in multiple sclerosis varies with disease activity

[Source](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-022-01148-1)

---

## Intestinal gas production by the gut microbiota: A review

[Source](https://www.sciencedirect.com/science/article/pii/S1756464622004376)

---

## Gestational diabetes is driven by microbiota-induced inflammation months before diagnosis

[Source](https://gut.bmj.com/content/early/2023/01/10/gutjnl-2022-328406)

---

## The gut microbiota is an emerging target for improving brain health during ageing

[Source](https://www.cambridge.org/core/journals/gut-microbiome/article/gut-microbiota-is-an-emerging-target-for-improving-brain-health-during-ageing/E9572C5EB8BC237511139BB2A603F795)

---

## #GutMicrobiome

[Source](https://twitter.com/hashtag/GutMicrobiome?src=hash&ref_src=twsrc%5Etfw)

---

## #Brain

[Source](https://twitter.com/hashtag/Brain?src=hash&ref_src=twsrc%5Etfw)

---

## #Aging

[Source](https://twitter.com/hashtag/Aging?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/Z0fgsS1Pzv

[Source](https://t.co/Z0fgsS1Pzv)

---

## @BoehmeMarcus

[Source](https://twitter.com/BoehmeMarcus?ref_src=twsrc%5Etfw)

---

## @Lauriemcox

[Source](https://twitter.com/Lauriemcox?ref_src=twsrc%5Etfw)

---

## @CambridgeUP

[Source](https://twitter.com/CambridgeUP?ref_src=twsrc%5Etfw)

---

## pic.twitter.com/NcfAT46ADK

[Source](https://t.co/NcfAT46ADK)

---

## January 10, 2023

[Source](https://twitter.com/ktguzz/status/1612769830138179587?ref_src=twsrc%5Etfw)

---

## Commensal Lactobacilli Metabolically Contribute to Cervical Epithelial Homeostasis in a Species-Specific Manner

[Source](https://journals.asm.org/doi/10.1128/msphere.00452-22)

---

## Preterm birth is associated with xenobiotics and predicted by the vaginal metabolome

[Source](https://www.nature.com/articles/s41564-022-01293-8)

---

## Gut microbiota of the young ameliorates physical fitness of the aged in mice

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01386-w)

---

## Catheterization of mice triggers resurgent urinary tract infection seeded by a bladder reservoir of <em>Acinetobacter baumanni</em>

[Source](https://www.science.org/doi/10.1126/scitranslmed.abn8134)

---

## Prebiotic intervention with HAMSAB in untreated essential hypertensive patients assessed in a phase II randomized trial 

[Source](https://twitter.com/FZMarques/status/1612579178104381440)

---

## <em>Akkermansia muciniphila</em> protects mice against an emerging tick-borne viral pathogen 

[Source](https://www.nature.com/articles/s41564-022-01279-6)

---

## Beehives possess their own distinct microbiomes 

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-023-00460-6)

---

## Differential carbon utilization enables co-existence of recently speciated Campylobacteraceae in the cow rumen epithelial microbiome.
 12.01.2023, _CR Strachan, XA Yu, V Neubauer, AJ Mueller, M Wagner, Q Zebeli, E Selberherr, MF Polz_


The activities of different microbes in the cow rumen have been shown to modulate the host's ability to utilize plant biomass, while the host-rumen interface has received little attention. As datasets collected worldwide have pointed to Campylobacteraceae as particularly abundant members of the rumen epithelial microbiome, we targeted this group in a subset of seven cows with meta- and isolate genome analysis. We show that the dominant Campylobacteraceae lineage has recently speciated into two populations that were structured by genome-wide selective sweeps followed by population-specific gene import and recombination. These processes led to differences in gene expression and enzyme domain composition that correspond to the ability to utilize acetate, the main carbon source for the host, at the cost of inhibition by propionate. This trade-off in competitive ability further manifests itself in differential dynamics of the two populations in vivo. By exploring population-level adaptations that otherwise remain cryptic in culture-independent analyses, our results highlight how recent evolutionary dynamics can shape key functional roles in the rumen microbiome.

[Source](https://www.nature.com/articles/s41564-022-01300-y)

10.1038/s41564-022-01300-y

---

## Ecological landscapes guide the assembly of optimal microbial communities.
 13.01.2023, _AB George, KS Korolev_


_Humans, Soil Microbiology, Microbiota, Microbial Consortia, Agriculture_
Assembling optimal microbial communities is key for various applications in biofuel production, agriculture, and human health. Finding the optimal community is challenging because the number of possible communities grows exponentially with the number of species, and so an exhaustive search cannot be performed even for a dozen species. A heuristic search that improves community function by adding or removing one species at a time is more practical, but it is unknown whether this strategy can discover an optimal or nearly optimal community. Using consumer-resource models with and without cross-feeding, we investigate how the efficacy of search depends on the distribution of resources, niche overlap, cross-feeding, and other aspects of community ecology. We show that search efficacy is determined by the ruggedness of the appropriately-defined ecological landscape. We identify specific ruggedness measures that are both predictive of search performance and robust to noise and low sampling density. The feasibility of our approach is demonstrated using experimental data from a soil microbial community. Overall, our results establish the conditions necessary for the success of the heuristic search and provide concrete design principles for building high-performing microbial consortia.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010570)

10.1371/journal.pcbi.1010570

---

## Over-optimism in unsupervised microbiome analysis: Insights from network learning and clustering.
 06.01.2023, _T Ullmann, S Peschel, P Finger, CL Müller, AL Boulesteix_


In recent years, unsupervised analysis of microbiome data, such as microbial network analysis and clustering, has increased in popularity. Many new statistical and computational methods have been proposed for these tasks. This multiplicity of analysis strategies poses a challenge for researchers, who are often unsure which method(s) to use and might be tempted to try different methods on their dataset to look for the "best" ones. However, if only the best results are selectively reported, this may cause over-optimism: the "best" method is overly fitted to the specific dataset, and the results might be non-replicable on validation data. Such effects will ultimately hinder research progress. Yet so far, these topics have been given little attention in the context of unsupervised microbiome analysis. In our illustrative study, we aim to quantify over-optimism effects in this context. We model the approach of a hypothetical microbiome researcher who undertakes four unsupervised research tasks: clustering of bacterial genera, hub detection in microbial networks, differential microbial network analysis, and clustering of samples. While these tasks are unsupervised, the researcher might still have certain expectations as to what constitutes interesting results. We translate these expectations into concrete evaluation criteria that the hypothetical researcher might want to optimize. We then randomly split an exemplary dataset from the American Gut Project into discovery and validation sets multiple times. For each research task, multiple method combinations (e.g., methods for data normalization, network generation, and/or clustering) are tried on the discovery data, and the combination that yields the best result according to the evaluation criterion is chosen. While the hypothetical researcher might only report this result, we also apply the "best" method combination to the validation dataset. The results are then compared between discovery and validation data. In all four research tasks, there are notable over-optimism effects; the results on the validation data set are worse compared to the discovery data, averaged over multiple random splits into discovery/validation data. Our study thus highlights the importance of validation and replication in microbiome analysis to obtain reliable results and demonstrates that the issue of over-optimism goes beyond the context of statistical testing and fishing for significance.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010820)

10.1371/journal.pcbi.1010820

---

## Ten simple rules for using public biological data for your research.
 09.01.2023, _VH Oza, JH Whitlock, EJ Wilk, A Uno-Antonison, B Wilk, M Gajapathy, TC Howton, A Trull, L Ianov, EA Worthey, BN Lasseigne_


_Reproducibility of Results, Information Storage and Retrieval_
With an increasing amount of biological data available publicly, there is a need for a guide on how to successfully download and use this data. The 10 simple rules for using public biological data are: (1) use public data purposefully in your research; (2) evaluate data for your use case; (3) check data reuse requirements and embargoes; (4) be aware of ethics for data reuse; (5) plan for data storage and compute requirements; (6) know what you are downloading; (7) download programmatically and verify integrity; (8) properly cite data; (9) make reprocessed data and models Findable, Accessible, Interoperable, and Reusable (FAIR) and share; and (10) make pipelines and code FAIR and share. These rules are intended as a guide for researchers wanting to make use of available data and to increase data reuse and reproducibility.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010749)

10.1371/journal.pcbi.1010749

---

## SemiBin2: self-supervised contrastive learning leads to better MAGs for short- and long-read sequencing 

[Source](https://www.biorxiv.org/content/10.1101/2023.01.09.523201v1)

---

## https://t.co/noR7crphRT

[Source](https://t.co/noR7crphRT)

---

## January 10, 2023

[Source](https://twitter.com/luispedrocoelho/status/1612757265072676865?ref_src=twsrc%5Etfw)

---

