## The change of gut microbiota in MDD patients under SSRIs treatment.
 22.07.2021, _Y Shen, X Yang, G Li, J Gao, Y Liang_


The alterations in the gut microbiota have been reported to be correlated with the development of depression. The purpose of this study was to investigate the changes of intestinal microbiota in depressed patients after antidepressant treatment. We recruited 30 MDD patients (MDD group) and 30 healthy controls (control group). The MDD group received individualized treatment with escitalopram at a maximum dose of 20 mg/day. After depressive symptoms improved to a HAMD scale score &gt; 50%, a fecal sample was collected again and used as the follow-up group. The differences of gut microbiota between patients and controls, the characteristics of gut microbiota under treatment and the potential differences in metabolic functions were thus investigated. The Firmicutes/Bacteroidetes ratio was significantly different within three groups, and the ratio of follow-up group was significantly lower than those of the other two groups. Alpha diversity was significantly higher in MDD group than those of the other groups, and the alpha diversity was not significantly different between control and follow-up groups. The beta diversity of some patients resembled participants in the control group. The metabolic function of gut microbiota after treatment was still different from that of the control group. This study suggests that the intestinal flora of depressed patients has a tendency to return to normal under escitalopram treatment.

[Source](https://doi.org/10.1038/s41598-021-94481-1)

10.1038/s41598-021-94481-1

---

## The Change Of Gut Microbiota In MDD Patients Under SSRIs Treatment

[Source](https://doi.org/10.1038/s41598-021-94481-1)

---

## Minimal-moderate variation of human oral virome and microbiome in IgA deficiency.
 22.07.2021, _MJ de la Cruz Peña, LI Gonzalez-Granado, I Garcia-Heredia, LM Carballa, M Martinez-Garcia_


Immunoglobulin A (IgA) is the dominant antibody found in our mucosal secretions and has long been recognized to play an important role in protecting our epithelium from pathogens. Recently, IgA has been shown to be involved in gut homeostatic regulation by 'recognizing' and shaping our commensal microbes. Paradoxically, yet selective IgA-deficiency is often described as asymptomatic and there is a paucity of studies only focused on the mice and human gut microbiome context fully ignoring other niches of our body and our commensal viruses. Here, we used as a model the human oral cavity and employed a holistic view and studied the impact of IgA deficiency and also common variable IgA and IgM immunodeficiencies (CVID), on both the human virome and microbiome. Unexpectedly, metagenomic and experimental data in human IgA deficiency and CVID indicate minimal-moderate changes in microbiome and virome composition compared to healthy control group and point out to a rather functional, resilient oral commensal viruses and microbes. However, a significant depletion (two fold) of bacterial cells (p-value &lt; 0.01) and viruses was observed in IgA-deficiency. Our results demonstrate that, within the limits of our cohort, IgA role is not critical for maintaining a rather functional salivary microbiome and suggest that IgA is not a major influence on the composition of abundant commensal microbes.

[Source](https://doi.org/10.1038/s41598-021-94507-8)

10.1038/s41598-021-94507-8

---

## Minimal-moderate Variation Of Human Oral Virome And Microbiome In IgA Deficiency

[Source](https://doi.org/10.1038/s41598-021-94507-8)

---

## Weight shapes the intestinal microbiome in preterm infants: results of a prospective observational study.
 22.07.2021, _FH Heida, EMW Kooi, J Wagner, TY Nguyen, JBF Hulscher, AGJF van Zoonen, AF Bos, HJM Harmsen, MC de Goffau_


The intestinal microbiome in preterm infants differs markedly from term infants. It is unclear whether the microbiome develops over time according to infant specific factors.

[Source](https://doi.org/10.1186/s12866-021-02279-y)

10.1186/s12866-021-02279-y

---

## Weight Shapes The Intestinal Microbiome In Preterm Infants: Results Of A Prospective observational study

[Source](https://doi.org/10.1186/s12866-021-02279-y)

---

## Functional heterogeneity in the fermentation capabilities of the healthy human gut microbiota.
 21.07.2021, _T Gurry, LTT Nguyen, X Yu, EJ Alm_


The human gut microbiota is known for its highly heterogeneous composition across different individuals. However, relatively little is known about functional differences in its ability to ferment complex polysaccharides. Through ex vivo measurements from healthy human donors, we show that individuals vary markedly in their microbial metabolic phenotypes (MMPs), mirroring differences in their microbiota composition, and resulting in the production of different quantities and proportions of Short Chain Fatty Acids (SCFAs) from the same inputs. We also show that aspects of these MMPs can be predicted from composition using 16S rRNA sequencing. From experiments performed using the same dietary fibers in vivo, we demonstrate that an ingested bolus of fiber is almost entirely consumed by the microbiota upon passage. We leverage our ex vivo data to construct a model of SCFA production and absorption in vivo, and argue that inter-individual differences in quantities of absorbed SCFA are directly related to differences in production. Though in vivo studies are required to confirm these data in the context of the gut, in addition to in vivo read outs of SCFAs produced in response to specific fiber spike-ins, these data suggest that optimizing SCFA production in a given individual through targeted fiber supplementation requires quantitative understanding of their MMP.

[Source](https://doi.org/10.1371/journal.pone.0254004)

10.1371/journal.pone.0254004

---

## Functional Heterogeneity In The Fermentation Capabilities Of The Healthy Human Gut microbiota

[Source](https://doi.org/10.1371/journal.pone.0254004)

---

## Characterization of the blood and neutrophil-specific microbiomes and exploration of potential bacterial biomarkers for sepsis in surgical patients.
 21.07.2021, _C Wang, Q Li, C Tang, X Zhao, Q He, X Tang, J Ren_


Recent studies have demonstrated the presence of a circulating microbiome in the blood of healthy subjects and chronic inflammatory patients. However, our knowledge regarding the blood microbiome and its potential roles in surgical patients remains very limited. The objective of this study was to determine the blood microbial landscape in surgical patients and to explore its potential associations with postoperative sepsis.

[Source](https://doi.org/10.1002/iid3.483)

10.1002/iid3.483

---

## Characterization Of The Blood And Neutrophil-specific Microbiomes And Exploration Of potential bacterial biomarkers for sepsis in surgical patients

[Source](https://doi.org/10.1002/iid3.483)

---

## Cognitive function improvement after fecal microbiota transplantation in in Alzheimer's dementia patient: a case report.
 22.07.2021, _SH Park, JH Lee, J Shin, JS Kim, B Cha, S Lee, KS Kwon, YW Shin, SH Choi_


After fecal microbiota transplantation (FMT) to treat Clostridioides difficile infection (CDI), cognitive improvement is noticeable, suggesting an essential association between the gut microbiome and neural function. Although it is known that the gut microbiome is linked with cognitive function, whether FMT may lead to cognitive improvement in patients with neurodegenerative disorders remains to be elucidated. We present the case of a 90-year-old woman with Alzheimer's dementia and severe CDI who underwent FMT. Cognitive function testing (Mini-Mental State Examination, Montreal Cognitive Assessment, and Clinical Dementia Rating assessment) was performed one month before FMT and one week and one month after FMT. We collected the patients' fecal samples before FMT and 3 weeks after FMT to compare the microbiota composition. The 16S rRNA gene amplicons were analyzed using the QIIME2 platform (version 2020.2) and Phyloseq R package. The linear discriminant analysis effect size was performed to determine the taxonomic difference between pre- and post-FMT. Functional biomarker analysis using the Kruskal-Wallis H test was performed between the pre- and post-FMT. The cognitive function tests after FMT showed an improvement compared to the tests before the procedure. FMT changed the microbiota composition in recipient feces. We found that the genera were reported to be associated with cognitive function. In addition, short-chain fatty acids was found to be significantly different between before and after FMT. This finding suggests the presence of an association between the gut microbiome and cognitive function. Further, it emphasizes the need for clinical awareness regarding the effect of FMT on the brain-gut-microbiome axis and its potential as a therapy for patients with dementia.

[Source](https://doi.org/10.1080/03007995.2021.1957807)

10.1080/03007995.2021.1957807

---

## Cognitive Function Improvement After Fecal Microbiota Transplantation In In Alzheimer’s dementia patient: a case report

[Source](https://doi.org/10.1080/03007995.2021.1957807)

---

## Amplicon and Metagenomic Analysis of Middle East Respiratory Syndrome (MERS) Coronavirus and the Microbiome in Patients with Severe MERS.
 21.07.2021, _W Aljabr, M Alruwaili, R Penrice-Randal, A Alrezaihi, AJ Harrison, Y Ryan, E Bentley, B Jones, BY Alhatlani, D AlShahrani, Z Mahmood, NY Rickett, B Alosaimi, A Naeem, S Alamri, H Alsran, ME Hamed, X Dong, AM Assiri, AR Alrasheed, M Hamza, MW Carroll, M Gemmell, A Darby, I Donovan-Banfield, JP Stewart, DA Matthews, AD Davidson, JA Hiscox_


Middle East respiratory syndrome coronavirus (MERS-CoV) is a zoonotic infection that emerged in the Middle East in 2012. Symptoms range from mild to severe and include both respiratory and gastrointestinal illnesses. The virus is mainly present in camel populations with occasional zoonotic spill over into humans. The severity of infection in humans is influenced by numerous factors, and similar to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), underlying health complications can play a major role. Currently, MERS-CoV and SARS-CoV-2 are coincident in the Middle East and thus a rapid way of sequencing MERS-CoV to derive genotype information for molecular epidemiology is needed. Additionally, complicating factors in MERS-CoV infections are coinfections that require clinical management. The ability to rapidly characterize these infections would be advantageous. To rapidly sequence MERS-CoV, an amplicon-based approach was developed and coupled to Oxford Nanopore long read length sequencing. This and a metagenomic approach were evaluated with clinical samples from patients with MERS. The data illustrated that whole-genome or near-whole-genome information on MERS-CoV could be rapidly obtained. This approach provided data on both consensus genomes and the presence of minor variants, including deletion mutants. The metagenomic analysis provided information of the background microbiome. The advantage of this approach is that insertions and deletions can be identified, which are the major drivers of genotype change in coronaviruses. IMPORTANCE Middle East respiratory syndrome coronavirus (MERS-CoV) emerged in late 2012 in Saudi Arabia. The virus is a serious threat to people not only in the Middle East but also in the world and has been detected in over 27 countries. MERS-CoV is spreading in the Middle East and neighboring countries, and approximately 35% of reported patients with this virus have died. This is the most severe coronavirus infection so far described. Saudi Arabia is a destination for many millions of people in the world who visit for religious purposes (Umrah and Hajj), and so it is a very vulnerable area, which imposes unique challenges for effective control of this epidemic. The significance of our study is that clinical samples from patients with MERS were used for rapid in-depth sequencing and metagenomic analysis using long read length sequencing.

[Source](https://doi.org/10.1128/mSphere.00219-21)

10.1128/mSphere.00219-21

---

## Dental Plaque Microbial Resistomes of Periodontal Health and Disease and Their Changes after Scaling and Root Planing Therapy.
 21.07.2021, _Y Kang, B Sun, Y Chen, Y Lou, M Zheng, Z Li_


The human oral microbial community has been considered a reservoir of antibiotic resistance. Currently, the effects of periodontitis and the scaling and root planing (SRP) treatment on the performance of antibiotic-resistant genes (ARGs) and metal-resistant genes (MRGs) in the dental plaque microbiota are not well characterized. To explore this issue, we selected 48 healthy-state (HS), 40 periodontitis-state (PS; before treatment), and 24 resolved-state (RS; after SRP treatment) metagenomic data of dental plaque samples from the Sequence Read Archive (SRA) database. NetShift analysis identified Fretibacterium fastidiosum, Tannerella forsythia, and Campylobacter rectus as key drivers during dental plaque microbiota alteration in the progression of periodontitis. Periodontitis and SRP treatment resulted in an increase in the number of ARGs and MRGs in dental plaque and significantly altered the composition of ARG and MRG profiles. Bacitracin, beta-lactam, macrolide-lincosamide-streptogramin (MLS), tetracycline, and multidrug resistance genes were the main classes of ARGs with high relative abundance, whereas multimetal, iron, chromium, and copper resistance genes were the primary types of MRGs in dental plaque microbiota. The cooccurrence of ARGs, MRGs, and mobile genetic elements (MGEs) indicated that a coselection phenomenon exists in the resistomes of dental plaque microbiota. Overall, our data provide new insights into the standing of the distribution of ARGs and MRGs in oral microbiota of periodontitis patients, and it was possible to contribute to the understanding of the complicated correlations among microorganisms, resistomes, and MGEs. IMPORTANCE The emergence and development of resistance to antibiotics in periodontal pathogens have affected the success rate of treatment for periodontitis. The development of new antibacterial strategies is urgently needed to help control and treat periodontal disease, and dental plaque microbiome studies offer a promising new angle of attack. In this study, we investigated the dental plaque microbiota and resistomes in periodontal health and disease states and their changes after SRP therapy. This is the first analysis of the profile of the microbial community and antibiotic and metal resistance genes in dental plaque by the metagenomic approach, to the best of our knowledge. Monitoring the profile of these resistomes has huge potential to provide reference levels for proper antibiotics use and the development of new antimicrobial strategies in periodontitis therapy and thereby improve actual efficacy of the treatment regimens.

[Source](https://doi.org/10.1128/mSphere.00162-21)

10.1128/mSphere.00162-21

---

## Archaea Are Rare and Uncommon Members of the Mammalian Skin Microbiome.
 20.07.2021, _AK Umbach, AA Stegelmeier, JD Neufeld_


Although previous research demonstrates that skin-associated archaea are rarely detected within human skin microbiome data, exist at relatively low abundance, and are primarily affiliated with the Methanobacteriota and Halobacteriota phyla, other studies suggest that archaea are consistently detected and relatively abundant on human skin, with skin "archaeomes" dominated by putative ammonia oxidizers of the Nitrososphaeria class (Thermoproteota phylum, formerly Thaumarchaeota). Here, we evaluated new and existing 16S rRNA gene sequence data sourced from mammalian skin and skin-associated surfaces and generated with two commonly used universal prokaryotic primer sets to assess archaeal prevalence, relative abundance, and taxonomic distribution. Archaeal 16S rRNA gene sequences were detected in only 17.5% of 1,688 samples by high-throughput sequence data, with most of the archaeon-positive samples associated with nonhuman mammalian skin. Only 5.9% of human-associated skin sample data sets contained sequences affiliated with archaeal 16S rRNA genes. When detected, the relative abundance of sequences affiliated with archaeal amplicon sequence variants (ASVs) was less than 1% for most mammalian skin samples and did not exceed 2% for any samples. Although several computer keyboard microbial profiles were dominated by Nitrososphaeria sequences, all other skin microbiome data sets tested were primarily composed of sequences affiliated with Methanobacteriota and Halobacteriota phyla. Our findings revise downward recent estimates of human skin archaeal distributions and relative abundances, especially those affiliated with the Nitrososphaeria, reflecting a limited and infrequent archaeal presence within the mammalian skin microbiome. IMPORTANCE The current state of research on mammalian skin-associated archaea is limited, with the few papers focusing on potential skin archaeal communities often in disagreement with each other. As such, there is no consensus on the prevalence or taxonomic composition of archaea on mammalian skin. Mammalian skin health is in part influenced by its complex microbiota and consortium of bacteria and potential archaea. Without a clear foundational analysis and characterization of the mammalian skin archaeome, it will be difficult for future research to explore the potential impact of skin-associated archaea on skin health and function. The current work provides a much-needed analysis of the mammalian skin archaeome and contributes to building a foundation from which further discussion and exploration of the skin archaeome might continue.

[Source](https://doi.org/10.1128/mSystems.00642-21)

10.1128/mSystems.00642-21

---

## Association between Human Genetic Variants and the Vaginal Bacteriome of Pregnant Women.
 20.07.2021, _W Fan, H Kan, HY Liu, TL Wang, YN He, M Zhang, YX Li, YJ Li, W Meng, Q Li, AQ Hu, YJ Zheng_


The influence of human genetic variants on the vaginal bacterial traits (VBTs) of pregnant women is still unknown. Using a genome-wide association approach based on the 16S rRNA bacteriome analysis, a total of 72 host genetic variant (single nucleotide polymorphisms [SNPs], indels, or copy number variations [CNVs])-VBT associations were found that reached the genome-wide significance level (P &lt; 5 × 10-8) with an acceptable genomic inflation factor λ of &lt;1.1. The majority of these SNPs that reached the genome-wide significance level had a relatively low minor allele frequency (MAF), and only seven of them had MAFs greater than 0.05. rs303212, located at the IFIT1 gene on chromosome 10, was the most eye-catching variant, which had a genome-wide association with the relative abundance (RAB) of Actinobacteria and Bifidobacteriaceae and also had a suggestive association with the RAB of a few common vaginal bacteria including Actinobacteriota, Firmicutes, Lactobacillus, and Gardnerella vaginalis and the beta diversity weighted UniFrac (P &lt; 1 × 10-5). The findings of the study suggest that the vaginal bacteriome may be influenced by a number of genetic variants across the human genome and that interferon signaling may have an important influence on vaginal bacterial communities during pregnancy. IMPORTANCE Knowledge about the influence of host genetics on the vaginal bacteriome in pregnancy is still limited. Although a number of environmental and behavioral factors may exert influences on the structure of vaginal bacterial communities, the vaginal bacteriome often undergoes a relatively fixed transition to a more stable and less diverse state as the menstrual cycle stops, which raises questions on the effects of human genetics. We utilized a genome-wide approach to identify the associations between genetic variants and multiple VBTs and performed enrichment analyses. The human genetics during pregnancy may be involved in multiple pathways. The results may disclose innate functional factors involved in shaping the vaginal bacteriome during pregnancy and provide insight into the establishment of specific strategies for prevention and clinical treatment of pregnancy complications.

[Source](https://doi.org/10.1128/mSystems.00158-21)

10.1128/mSystems.00158-21

---

## Population-wide diversity and stability of serum antibody epitope repertoires against human microbiota.
 20.07.2021, _T Vogl, S Klompus, S Leviatan, IN Kalka, A Weinberger, C Wijmenga, J Fu, A Zhernakova, RK Weersma, E Segal_


Serum antibodies can recognize both pathogens and commensal gut microbiota. However, our current understanding of antibody repertoires is largely based on DNA sequencing of the corresponding B-cell receptor genes, and actual bacterial antigen targets remain incompletely characterized. Here we have profiled the serum antibody responses of 997 healthy individuals against 244,000 rationally selected peptide antigens derived from gut microbiota and pathogenic and probiotic bacteria. Leveraging phage immunoprecipitation sequencing (PhIP-Seq) based on phage-displayed synthetic oligo libraries, we detect a wide breadth of individual-specific as well as shared antibody responses against microbiota that associate with age and gender. We also demonstrate that these antibody epitope repertoires are more longitudinally stable than gut microbiome species abundances. Serum samples of more than 200 individuals collected five years apart could be accurately matched and could serve as an immunologic fingerprint. Overall, our results suggest that systemic antibody responses provide a non-redundant layer of information about microbiota beyond gut microbial species composition.

[Source](https://doi.org/10.1038/s41591-021-01409-3)

10.1038/s41591-021-01409-3

---

## Population-wide Diversity And Stability Of Serum Antibody Epitope Repertoires against human microbiota

[Source](https://doi.org/10.1038/s41591-021-01409-3)

---

## Dynamic Colonization of Microbes and Their Functions after Fecal Microbiota Transplantation for Inflammatory Bowel Disease.
 20.07.2021, _ND Chu, JW Crothers, LTT Nguyen, SM Kearney, MB Smith, Z Kassam, C Collins, R Xavier, PL Moses, EJ Alm_


For fecal microbiota transplantation (FMT) to be successful in immune diseases like inflammatory bowel disease, it is assumed that therapeutic microbes and their beneficial functions and immune interactions must colonize a recipient patient and persist in sufficient quantity and for a sufficient period of time to produce a clinical benefit. Few studies, however, have comprehensively profiled the colonization and persistence of transferred microbes along with the transfer of their microbial functions and interactions with the host immune system. Using 16S, metagenomic, and immunoglobulin A (IgA) sequencing, we analyzed hundreds of longitudinal microbiome samples from a randomized controlled trial of 12 patients with ulcerative colitis who received fecal transplant or placebo for 12 weeks. We uncovered diverse competitive dynamics among donor and patient strains, showing that persistence of transferred microbes is far from static. Indeed, one patient experienced a dramatic loss of donor bacteria 10 weeks into the trial, coinciding with a bloom of pathogenic bacteria and worsening symptoms. We evaluated the transfer of microbial functions, including desired ones, such as butyrate production, and unintended ones, such as antibiotic resistance. By profiling bacteria coated with IgA, we identified bacteria associated with inflammation and found that microbial interactions with the host immune system can be transferred across people, which could play a role in gut microbiome therapeutics for immune-related diseases. Our findings shed light on the colonization dynamics of gut microbes and their functions in the context of FMT to treat a complex disease-information that may provide a foundation for developing more-targeted therapeutics. IMPORTANCE Fecal microbiota transplantation (FMT)-transferring fecal microbes from a healthy donor to a sick patient-has shown promise for gut diseases such as inflammatory bowel disease. Unlike pharmaceuticals, however, fecal transplants are complex mixtures of living organisms, which must then interact with the microbes and immune system of the recipient. We sought to understand these interactions by tracking the microbes of 12 inflammatory bowel disease patients who received fecal transplants for 12 weeks. We uncovered a range of dynamics. For example, one patient experienced successful transfer of donor bacteria, only to lose them after 10 weeks. We similarly evaluated transfer of microbial functions, including how they interacted with the recipient's immune system. Our findings shed light on the colonization dynamics of gut microbes, as well as their functions in the context of FMT-information that may provide a critical foundation for the development of more-targeted therapeutics.

[Source](https://doi.org/10.1128/mBio.00975-21)

10.1128/mBio.00975-21

---

## Tryptophan metabolites modulate inflammatory bowel disease and colorectal cancer by affecting immune system.
 22.07.2021, _M Ala_


Tryptophan is an essential amino acid, going through three different metabolic pathways in the intestines. Indole pathway in the gut microbiota, serotonin system in the enterochromaffin cells and kynurenine pathway in the immune cells and intestinal lining are the three arms of tryptophan metabolism in the intestines. Clinical, in vivo and in vitro studies showed that each one of these arms has a significant impact on IBD. This review explains how different metabolites of tryptophan are involved in the pathophysiology of IBD and colorectal cancer, as a major complication of IBD. Indole metabolites alleviate colitis and protect against colorectal cancer while serotonin arm follows a more complicated and receptor-specific pattern. Indole metabolites and kynurenine interact with aryl hydrocarbon receptor (AHR) to induce T regulatory cells differentiation, confine Th17 and Th1 response and produce anti-inflammatory mediators. Kynurenine decreases tumor-infiltrating CD8+ cells and mediates tumor cells immune evasion. Serotonin system also increases colorectal cancer cells proliferation and metastasis while, indole metabolites can profoundly decrease colorectal cancer growth. Targeted therapy for tryptophan metabolites may improve the management of IBD and colorectal cancer, e.g. supplementation of indole metabolites such as indole-3-carbinol (I3C), inhibition of kynurenine monooxygenase (KMO) and selective stimulation or inhibition of specific serotonergic receptors can mitigate colitis. Furthermore, it will be explained how indole metabolites supplementation, inhibition of indoleamine 2,3-dioxygenase 1 (IDO1), KMO and serotonin receptors can protect against colorectal cancer. Additionally, extensive molecular interactions between tryptophan metabolites and intracellular signaling pathways will be thoroughly discussed.

[Source](https://doi.org/10.1080/08830185.2021.1954638)

10.1080/08830185.2021.1954638

---

## Tryptophan Metabolites Modulate Inflammatory Bowel Disease And Colorectal Cancer By affecting immune system

[Source](https://doi.org/10.1080/08830185.2021.1954638)

---

## From gut microbiota to host appetite: gut microbiota-derived metabolites as key regulators.
 21.07.2021, _H Han, B Yi, R Zhong, M Wang, S Zhang, J Ma, Y Yin, J Yin, L Chen, H Zhang_


Feelings of hunger and satiety are the key determinants for maintaining the life of humans and animals. Disturbed appetite control may disrupt the metabolic health of the host and cause various metabolic disorders. A variety of factors have been implicated in appetite control, including gut microbiota, which develop the intricate interactions to manipulate the metabolic requirements and hedonic feelings. Gut microbial metabolites and components act as appetite-related signaling molecules to regulate appetite-related hormone secretion and the immune system, or act directly on hypothalamic neurons. Herein, we summarize the effects of gut microbiota on host appetite and consider the potential molecular mechanisms. Furthermore, we propose that the manipulation of gut microbiota represents a clinical therapeutic potential for lessening the development and consequence of appetite-related disorders. Video abstract.

[Source](https://doi.org/10.1186/s40168-021-01093-y)

10.1186/s40168-021-01093-y

---

## From Gut Microbiota To Host Appetite: Gut Microbiota-derived Metabolites As Key regulators

[Source](https://doi.org/10.1186/s40168-021-01093-y)

---

## The Roles of Cytoplasmic Lipid Droplets in Modulating Intestinal Uptake of Dietary Fat.
 20.07.2021, _AS Zembroski, C Xiao, KK Buhman_


Dietary fat absorption is required for health but also contributes to hyperlipidemia and metabolic disease when dysregulated. One step in the process of dietary fat absorption is the formation of cytoplasmic lipid droplets (CLDs) in small intestinal enterocytes; these CLDs serve as dynamic triacylglycerol storage organelles that influence the rate at which dietary fat is absorbed. Recent studies have uncovered novel factors regulating enterocyte CLD metabolism that in turn influence the absorption of dietary fat. These include peroxisome proliferator-activated receptor α activation, compartmentalization of different lipid pools, the gut microbiome, liver X receptor and farnesoid X receptor activation, obesity, and physiological factors stimulating CLD mobilization. Understanding how enterocyte CLD metabolism is regulated is key in modulating the absorption of dietary fat in the prevention of hyperlipidemia and its associated metabolic disorders. Expected final online publication date for the Annual Review of Nutrition, Volume 41 is September 2021. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.

[Source](https://doi.org/10.1146/annurev-nutr-110320-013657)

10.1146/annurev-nutr-110320-013657

---

## Intestinal Microbiota in the SARS-CoV-2 Infection: What Is Known?
 21.07.2021, _PB Rodrigues, AB Dos Santos Pereira Gomes, LM Genaro, LB Pascoal, APD de Souza, RF Leal, MAR Vinolo_


_COVID-19, China, Dysbiosis, Gastrointestinal Microbiome, Humans, SARS-CoV-2_
The Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), the etiological agent of COVID-19, emerged last year in China and quickly spread to millions of people around the world. This virus infects cells in different tissues and causes pulmonary (e.g., pneumonia and acute respiratory distress syndrome), neurological, cardiovascular, and intestinal manifestations, which can be the result of a direct viral effect or secondary to endothelial, thrombotic, or immunological alterations. In this chapter, we discuss recent studies which highlighted the relevance of the intestinal microbiota for other infectious respiratory diseases. We present the "altered microbiota" (dysbiotic) as a point of connection between conditions that are risk factors for the development of severe forms of COVID-19. In addition, we describe the findings of recent studies reporting alterations of microbiota composition in COVID-19 patients and speculate on how this may impact in development of the disease.

[Source](https://doi.org/10.1007/978-3-030-71697-4_7)

10.1007/978-3-030-71697-4_7

---

## The Microbiota-Gut-Brain Axis and Epilepsy.
 19.07.2021, _Q Yue, M Cai, B Xiao, Q Zhan, C Zeng_


Honoured as the second genome in humans, the gut microbiota is involved in a constellation of physiological and pathological processes, including those related to the central nervous system. The communication between the gut microbiota and the brain is realized by a complex bidirectional connection, known as the "microbiota-gut-brain axis", via neuroendocrine, immunological, and direct neural mechanisms. Recent studies indicate that gut dysfunction/dysbiosis is presumably involved in the pathogenesis of and susceptibility to epilepsy. In addition, the reconstruction of the intestinal microbiome through, for example, faecal microbiota transplantation, probiotic intervention, and a ketogenic diet, has exhibited beneficial effects on drug-resistant epilepsy. The purposes of this review are to provide a brief overview of the microbiota-gut-brain axis and to synthesize what is known about the involvement of the gut microbiota in the pathogenesis and treatment of epilepsy, to bring new insight into the pathophysiology of epilepsy and to present a preliminary discussion of novel therapeutic options for epilepsy based on the gut microbiota.

[Source](https://doi.org/10.1007/s10571-021-01130-2)

10.1007/s10571-021-01130-2

---

## Microbiota links to immunotherapy toxicity.
 20.07.2021, _S Seton-Rogers_



[Source](https://doi.org/10.1038/s41568-021-00390-w)

10.1038/s41568-021-00390-w

---

## Forming Consensus To Advance Urobiome Research.
 20.07.2021, _L Brubaker, JF Gourdine, NY Siddiqui, A Holland, T Halverson, R Limeria, D Pride, L Ackerman, CS Forster, KM Jacobs, KJ Thomas-White, C Putonti, Q Dong, M Weinstein, ES Lukacz, L Karstens, AJ Wolfe_


Urobiome research has the potential to advance the understanding of a wide range of diseases, including lower urinary tract symptoms and kidney disease. Many scientific areas have benefited from early research method consensus to facilitate the greater, common good. This consensus document, developed by a group of expert investigators currently engaged in urobiome research (UROBIOME 2020 conference participants), aims to promote standardization and advances in this field by the adoption of common core research practices. We propose a standardized nomenclature as well as considerations for specimen collection, preservation, storage, and processing. Best practices for urobiome study design include our proposal for standard metadata elements as part of core metadata collection. Although it is impractical to follow fixed analytical procedures when analyzing urobiome data, we propose guidelines to document and report data originating from urobiome studies. We offer this first consensus document with every expectation of subsequent revision as our field progresses.

[Source](https://doi.org/10.1128/mSystems.01371-20)

10.1128/mSystems.01371-20

---

## A high-protein diet containing inulin/oligofructose supports body weight gain associated with lower energy expenditure and carbohydrate oxidation, and alters faecal microbiota in C57BL/6 mice.
 22.07.2021, _F Koch, M Derno, M Langhammer, A Tuchscherer, HM Hammon, M Mielenz, CC Metges, B Kuhla_


Prebiotic supplements and high-protein (HP) diets reduce body weight and modulate intestinal microbiota. Our aim was to elucidate the combined effect of an inulin/oligofructose (FOS) and HP diet on body weight gain, energy metabolism and faecal microbiota. Forty male C57BL/6NCrl mice were fed a control (C) diet for 2 weeks and allocated to a C or HP (40 % protein) diet including no or 10 % inulin/FOS (C + I and HP + I) for 4 weeks. Inulin/FOS was added in place of starch and cellulose. Body weight, food intake, faecal energy and nitrogen were determined. Indirect calorimetry and faecal microbiota analysis were performed after 3 weeks on diets. Body weight gain of HP-fed mice was 36 % lower than HP + I- and C-fed mice (P &lt; 0⋅05). Diet digestibility and food conversion efficiency were higher in HP + I- than HP-fed mice (P &lt; 0⋅01), while food intake was comparable between groups. Total energy expenditure (heat production) was 25 % lower in HP + I- than in C-, HP- and C + I-fed mice (P &lt; 0⋅001). Carbohydrate oxidation tended to be 24 % higher in HP- than in HP + I-fed mice (P &lt; 0⋅05). Faecal nitrogen excretion was 31-45 % lower in C-, C + I- and HP + I- than in HP-fed mice (P &lt; 0⋅05). Faecal Bacteroides-Prevotella DNA was 2⋅3-fold higher in C + I- and HP + I- relative to C-fed mice (P &lt; 0⋅05), but Clostridium leptum DNA abundances was 79 % lower in HP + I- than in HP-fed mice (P &lt; 0⋅05). We suggest that the higher conversion efficiency of dietary energy of HP + I but not C + I-fed mice is caused by higher digestibility and lower heat production, resulting in increased body mass.

[Source](https://doi.org/10.1017/jns.2021.42)

10.1017/jns.2021.42

---

## A High-protein Diet Containing Inulin/oligofructose Supports Body Weight Gain associated with lower energy expenditure and carbohydrate oxidation, and alters faecal microbiota in C57BL/6 mice

[Source](https://doi.org/10.1017/jns.2021.42)

---

## Immunosuppressive Activity Is Attenuated By <em>Astragalus</em> <em>polysaccharides</em> Via remodeling the gut microenvironment in melanoma mice

[Source](https://doi.org/10.1111/cas.15078)

---

## The interaction effect between tea polyphenols and intestinal microbiota: Role in ameliorating neurological diseases.
 21.07.2021, _M Hong, R Zhang, Y Liu, Z Wu, P Weng_


Tea polyphenols (TP) are one of the most functional and bioactive substances in tea. The interactions between TP and intestinal microbiota suggest that probiotics intervention is a useful method to ameliorate neurological diseases. Now, numerous researches have suggested that TP plays a significant role in modulating intestinal bacteria, especially in the area of sustaining a stable state of intestinal microbial function and abundance. Furthermore, homeostatic intestinal bacteria can enhance the immunity of the host. The close reciprocity between intestinal microbiota and the central nervous system provides a new chance for TP to modulate neural-related diseases depending on intestinal microbiota. Therefore, based on the bidirectional relationship between the brain and the intestines, this review provides a new clue to solve insomnia symptoms and related neurological diseases that will enable us to better study the bidirectional effects of TP and intestinal microbiota on the improvement of host health. PRACTICAL APPLICATIONS: This review provides a new clue to solve insomnia symptoms and related neurological diseases that will enable us to better study bidirectional effects of TP and intestinal microbiota on the improvement of host health.

[Source](https://doi.org/10.1111/jfbc.13870)

10.1111/jfbc.13870

---

## The Interaction Effect Between Tea Polyphenols And Intestinal Microbiota: Role In ameliorating neurological diseases

[Source](https://doi.org/10.1111/jfbc.13870)

---

## Nutritional Interventions and the Gut Microbiome in Children.
 20.07.2021, _S Mehta, SL Huey, D McDonald, R Knight, JL Finkelstein_


The gut microbiome plays an integral role in health and disease, and diet is a major driver of its composition, diversity, and functional capacity. Given the dynamic development of the gut microbiome in infants and children, it is critical to address two major questions: (a) Can diet modify the composition, diversity, or function of the gut microbiome, and (b) will such modification affect functional/clinical outcomes including immune function, cognitive development, and overall health? We synthesize the evidence on the effect of nutritional interventions on the gut microbiome in infants and children across 26 studies. Findings indicate the need to study older children, assess the whole intestinal tract, and harmonize methods and interpretation of findings, which are critical for informing meaningful clinical and public health practice. These findings are relevant for precision health, may help identify windows of opportunity for intervention, and may inform the design and delivery of such interventions. Expected final online publication date for the Annual Review of Nutrition, Volume 41 is September 2021. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.

[Source](https://doi.org/10.1146/annurev-nutr-021020-025755)

10.1146/annurev-nutr-021020-025755

---

## Interspecific variation and functional traits of the gut microbiome in spiders from the wild: The largest effort so far.
 21.07.2021, _K Tyagi, I Tyagi, V Kumar_


Spiders being one of the most diverse group in phylum arthropod are of great importance due to their role as predators, silk producer, and in medicinal applications. Spiders in prey-predator relationships play a crucial role in balancing the food-chain of any ecosystem; therefore it is essential to characterize the gut microbiota of spiders collected from natural environments. In the present work, the largest effort so far has been made to characterize the gut microbiota of 35 spider species belonging to four different families using 16S amplicon targeting sequencing. Further, we compared the gut microbiota composition including endosymbiont abundance in spider species collected from different geographical locations. The results obtained revealed the presence of genera like Acinetobacter (15%), V7clade (9%), Wolbachia (8%), Pseudomonas (5%), Bacillus (6%). Although comparative analysis revealed that the gut bacterial composition in all the spider families has a similar pattern, in terms of community richness and evenness. The bacterial diversity in the spider family, Lycosidae are more diverse than in Salticidae, Tetragnathidae and Araneidae. Furthermore, it was observed that the abundance of endosymbiont genera, i.e. Wolbachia and Rickettsia, leads to shift in the abundance of other bacterial taxa and may cause sexual alterations in spider species. Moreover, predicted functional analysis based on PICRUSt2 reveals that gut microbiota of spider species were involved in functions like metabolism of carbohydrates, cofactors and vitamins, amino acids; biosynthesis of organic compounds, fatty acids, lipids etc. Based on the results obtained, it can be said that different locations do not correlate with community composition of gut microbiota in spider species collected from natural environments.

[Source](https://doi.org/10.1371/journal.pone.0251790)

10.1371/journal.pone.0251790

---

## Interspecific Variation And Functional Traits Of The Gut Microbiome In Spiders From the wild: The largest effort so far.

[Source](https://doi.org/10.1371/journal.pone.0251790)

---

## <em>Vibrio vulnificus</em> Induces The Death Of A Major Bacterial Species In The Mouse Gut via cyclo-Phe-Pro.

[Source](https://doi.org/10.1186/s40168-021-01095-w)

---

## Maternal cecal microbiota transfer rescues early-life antibiotic-induced enhancement of type 1 diabetes in mice.
 21.07.2021, _XS Zhang, YS Yin, J Wang, T Battaglia, K Krautkramer, WV Li, J Li, M Brown, M Zhang, MH Badri, AJS Armstrong, CM Strauch, Z Wang, I Nemet, N Altomare, JC Devlin, L He, JT Morton, JA Chalk, K Needles, V Liao, J Mount, H Li, KV Ruggles, RA Bonneau, MG Dominguez-Bello, F Bäckhed, SL Hazen, MJ Blaser_


Early-life antibiotic exposure perturbs the intestinal microbiota and accelerates type 1 diabetes (T1D) development in the NOD mouse model. Here, we found that maternal cecal microbiota transfer (CMT) to NOD mice after early-life antibiotic perturbation largely rescued the induced T1D enhancement. Restoration of the intestinal microbiome was significant and persistent, remediating the antibiotic-depleted diversity, relative abundance of particular taxa, and metabolic pathways. CMT also protected against perturbed metabolites and normalized innate and adaptive immune effectors. CMT restored major patterns of ileal microRNA and histone regulation of gene expression. Further experiments suggest a gut-microbiota-regulated T1D protection mechanism centered on Reg3γ, in an innate intestinal immune network involving CD44, TLR2, and Reg3γ. This regulation affects downstream immunological tone, which may lead to protection against tissue-specific T1D injury.

[Source](https://doi.org/10.1016/j.chom.2021.06.014)

10.1016/j.chom.2021.06.014

---

## Maternal Cecal Microbiota Transfer Rescues Early-life Antibiotic-induced Enhancement of type 1 diabetes in mice.

[Source](https://doi.org/10.1016/j.chom.2021.06.014)

---

## Comprehensive Cultivation of the Swine Gut Microbiome Reveals High Bacterial Diversity and Guides Bacterial Isolation in Pigs.
 20.07.2021, _X Wang, S Howe, X Wei, F Deng, T Tsai, J Chai, Y Xiao, H Yang, CV Maxwell, Y Li, J Zhao_


Despite the substantial progress made in human gut culturomics, little is known about the culturability of the swine gut microbiota. In this study, we cultured swine gut microbiota using 53 bacterial cultivation methods with different medium and gas combinations from three pigs at four different growth stages. Both culture-dependent (CD; colony mixtures from each method) and culture-independent (CI; original fecal suspensions) samples were subjected to 16S rRNA gene amplicon sequencing. Increasing microbial diversities were observed in both CI and CD samples from successive growth stages. While a total of 378, 482, 565, and 555 bacterial amplicon sequence variants (ASVs) were observed in the CI samples, higher microbial diversities (415, 675, 808, and 823 observed ASVs) were detected using the CD methods at the lactation, nursery, growing, and finishing stages, respectively. We constructed reference culture maps showing the preferred cultivation conditions for specific bacterial taxa and examined the effects of culturing factors such as oxygen, medium, donor pig age, antibiotics, and blood culture preincubation on swine gut microbiota cultivation. We focused on a wide range of beneficial bacteria, chose 1,299 colonies based on the reference map, and Sanger sequenced their 16S rRNA genes. These isolates clustered into 148 different bacterial taxa covering 28 genera. We observed 11, 19, 33, and 25 pairs of cooccurring ASVs in both CD and CI samples at four successive growth stages. This study provides guidance in culturing the swine gut microbiota of interest, which is critical when characterizing their functions in this important animal species. IMPORTANCE The swine gut microbiome has been the focus of many investigations due to the fact that pigs serve as both an excellent biomedical model for human diseases and an important protein source. Substantial progress has been made in swine gut microbiome studies using next-generation sequencing-based culture-independent approaches, but little is known about the culturability of the swine gut microbiota. To understand their roles in swine production, it is critical to culture bacterial strains of interest. In this study, we cultured the gut microbiota from pigs at different growth stages using 53 bacterial cultivation methods with different medium and gas combinations. This study provides evidence that the swine gut microbiota is much more diverse based on a culture-dependent approach than previously known. It provides preliminary guidance for isolating certain bacteria of interest from pigs, which is critical in establishing causal relationships between the gut microbiota and the health status of pigs.

[Source](https://doi.org/10.1128/mSystems.00477-21)

10.1128/mSystems.00477-21

---

## Drought stress and plant ecotype drive microbiome recruitment in switchgrass rhizosheath.
 22.07.2021, _TY Liu, N Ye, X Wang, D Das, Y Tan, X You, M Long, T Hu, L Dai, J Zhang, MX Chen_


The rhizosheath, a layer of soil grains that adheres firmly to roots, is beneficial for plant growth and adaptation to drought environments. Switchgrass is a perennial C4 grass that can form contact rhizosheath under drought conditions. In this study, we characterized the microbiomes of four different rhizocompartments of two switchgrass ecotypes (Alamo and Kanlow) grown under drought or well-watered conditions via 16S rRNA amplicon sequencing. These four rhizocompartments, the bulk soil, rhizosheath soil, rhizoplane, and root endosphere, harbored both distinct and overlapping microbial communities. The root compartments (rhizoplane and root endosphere) displayed low-complexity communities dominated by Proteobacteria and Firmicutes. Compared to bulk soil, Cyanobacteria and Bacteroidetes were selectively enriched, while Proteobacteria and Firmicutes were selectively depleted, in rhizosheath soil. Taxa from Proteobacteria or Firmicutes were specifically selected in Alamo or Kanlow rhizosheath soil. Following drought stress, Citrobacter and Acinetobacter were further enriched in rhizosheath soil, suggesting that rhizosheath microbiome assembly is driven by drought stress. Additionally, the ecotype-specific recruitment of rhizosheath microbiome reveals their differences in drought-stress responses. Collectively, these results shed light on rhizosheath microbiome recruitment in switchgrass and lay the foundation for the improvement of drought tolerance in switchgrass by regulating the rhizosheath microbiome. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1111/jipb.13154)

10.1111/jipb.13154

---

## Drought Stress And Plant Ecotype Drive Microbiome Recruitment In Switchgrass

[Source](https://doi.org/10.1111/jipb.13154)

---

## Microbe-dependent heterosis in maize.
 21.07.2021, _MR Wagner, C Tang, F Salvato, KM Clouse, A Bartlett, S Vintila, L Phillips, S Sermons, M Hoffmann, PJ Balint-Kurti, M Kleiner_


Hybrids account for nearly all commercially planted varieties of maize and many other crop plants because crosses between inbred lines of these species produce first-generation [F1] offspring that greatly outperform their parents. The mechanisms underlying this phenomenon, called heterosis or hybrid vigor, are not well understood despite over a century of intensive research. The leading hypotheses-which focus on quantitative genetic mechanisms (dominance, overdominance, and epistasis) and molecular mechanisms (gene dosage and transcriptional regulation)-have been able to explain some but not all of the observed patterns of heterosis. Abiotic stressors are known to impact the expression of heterosis; however, the potential role of microbes in heterosis has largely been ignored. Here, we show that heterosis of root biomass and other traits in maize is strongly dependent on the belowground microbial environment. We found that, in some cases, inbred lines perform as well by these criteria as their F1 offspring under sterile conditions but that heterosis can be restored by inoculation with a simple community of seven bacterial strains. We observed the same pattern for seedlings inoculated with autoclaved versus live soil slurries in a growth chamber and for plants grown in steamed or fumigated versus untreated soil in the field. In a different field site, however, soil steaming increased rather than decreased heterosis, indicating that the direction of the effect depends on community composition, environment, or both. Together, our results demonstrate an ecological phenomenon whereby soil microbes differentially impact the early growth of inbred and hybrid maize.

[Source](https://doi.org/10.1073/pnas.2021965118)

10.1073/pnas.2021965118

---

## Prioritizing host phenotype to understand microbiome heritability in plants.
 21.07.2021, _MR Wagner_


Breeders and evolutionary geneticists have grappled with the complexity of the "genotype-to-phenotype map" for decades. Now, recent studies highlight the relevance of this concept for understanding heritability of plant microbiomes. Because host phenotype is a more proximate cause of microbiome variation than host genotype, microbiome heritability varies across plant anatomy and development. Fine-scale variation of plant traits within organs suggests that the well-established concept of "microbiome compartment" should be refined. Additionally, recent work shows that the balance of deterministic processes (including host genetic effects) vs. stochastic processes also varies over time and space. Together, these findings suggest that re-centering plant phenotype-both as a predictor and a readout of microbiome function-will accelerate new insights into microbiome heritability.

[Source](https://doi.org/10.1111/nph.17622)

10.1111/nph.17622

---

## Prioritizing Host Phenotype To Understand Microbiome Heritability In Plants

[Source](https://doi.org/10.1111/nph.17622)

---

## On the robustness of inference of association with the gut microbiota in stool, rectal swab and mucosal tissue samples.
 22.07.2021, _S Sun, X Zhu, X Huang, HJ Murff, RM Ness, DL Seidner, AA Sorgen, IC Blakley, C Yu, Q Dai, MA Azcarate-Peril, MJ Shrubsole, AA Fodor_


The gut microbiota plays an important role in human health and disease. Stool, rectal swab and rectal mucosal tissue samples have been used in individual studies to survey the microbial community but the consequences of using these different sample types are not completely understood. In this study, we report differences in stool, rectal swab and rectal mucosal tissue microbial communities with shotgun metagenome sequencing of 1397 stool, swab and mucosal tissue samples from 240 participants. The taxonomic composition of stool and swab samples was distinct, but less different to each other than mucosal tissue samples. Functional profile differences between stool and swab samples are smaller, but mucosal tissue samples remained distinct from the other two types. When the taxonomic and functional profiles were used for inference in association with host phenotypes of age, sex, body mass index (BMI), antibiotics or non-steroidal anti-inflammatory drugs (NSAIDs) use, hypothesis testing using either stool or rectal swab gave broadly significantly correlated results, but inference performed on mucosal tissue samples gave results that were generally less consistent with either stool or swab. Our study represents an important resource for determination of how inference can change for taxa and pathways depending on the choice of where to sample within the human gut.

[Source](https://doi.org/10.1038/s41598-021-94205-5)

10.1038/s41598-021-94205-5

---

## On The Robustness Of Inference Of Association With The Gut Microbiota In Stool, rectal swab and mucosal tissue samples

[Source](https://doi.org/10.1038/s41598-021-94205-5)

---

## Amplicon Sequence Variants Artificially Split Bacterial Genomes into Separate Clusters.
 21.07.2021, _PD Schloss_


Amplicon sequencing variants (ASVs) have been proposed as an alternative to operational taxonomic units (OTUs) for analyzing microbial communities. ASVs have grown in popularity, in part because of a desire to reflect a more refined level of taxonomy since they do not cluster sequences based on a distance-based threshold. However, ASVs and the use of overly narrow thresholds to identify OTUs increase the risk of splitting a single genome into separate clusters. To assess this risk, I analyzed the intragenomic variation of 16S rRNA genes from the bacterial genomes represented in an rrn copy number database, which contained 20,427 genomes from 5,972 species. As the number of copies of the 16S rRNA gene increased in a genome, the number of ASVs also increased. There was an average of 0.58 ASVs per copy of the 16S rRNA gene for full-length 16S rRNA genes. It was necessary to use a distance threshold of 5.25% to cluster full-length ASVs from the same genome into a single OTU with 95% confidence for genomes with 7 copies of the 16S rRNA, such as Escherichia coli. This research highlights the risk of splitting a single bacterial genome into separate clusters when ASVs are used to analyze 16S rRNA gene sequence data. Although there is also a risk of clustering ASVs from different species into the same OTU when using broad distance thresholds, these risks are of less concern than artificially splitting a genome into separate ASVs and OTUs. IMPORTANCE 16S rRNA gene sequencing has engendered significant interest in studying microbial communities. There has been tension between trying to classify 16S rRNA gene sequences to increasingly lower taxonomic levels and the reality that those levels were defined using more sequence and physiological information than is available from a fragment of the 16S rRNA gene. Furthermore, the naming of bacterial taxa reflects the biases of those who name them. One motivation for the recent push to adopt ASVs in place of OTUs in microbial community analyses is to allow researchers to perform their analyses at the finest possible level that reflects species-level taxonomy. The current research is significant because it quantifies the risk of artificially splitting bacterial genomes into separate clusters. Far from providing a better representation of bacterial taxonomy and biology, the ASV approach can lead to conflicting inferences about the ecology of different ASVs from the same genome.

[Source](https://doi.org/10.1128/mSphere.00191-21)

10.1128/mSphere.00191-21

---

## Linking plasmid-based beta-lactamases to their bacterial hosts using single-cell fusion PCR.
 22.07.2021, _PJ Diebold, FN New, M Hovan, MJ Satlin, IL Brito_


The horizonal transfer of plasmid-encoded genes allows bacteria to adapt to constantly shifting environmental pressures, bestowing functional advantages to their bacterial hosts such as antibiotic resistance, metal resistance, virulence factors, and polysaccharide utilization. However, common molecular methods such as short- and long-read sequencing of microbiomes cannot associate extrachromosomal plasmids with the genome of the host bacterium. Alternative methods to link plasmids to host bacteria are either laborious, expensive, or prone to contamination. Here we present the One-step Isolation and Lysis PCR (OIL-PCR) method, which molecularly links plasmid-encoded genes with the bacterial 16S rRNA gene via fusion PCR performed within an emulsion. After validating this method, we apply it to identify the bacterial hosts of three clinically relevant beta-lactamases within the gut microbiomes of neutropenic patients, as they are particularly vulnerable multidrug-resistant infections. We successfully detect the known association of a multi-drug resistant plasmid with Klebsiella pneumoniae, as well as the novel associations of two low-abundance genera, Romboutsia and Agathobacter. Further investigation with OIL-PCR confirmed that our detection of Romboutsia is due to its physical association with Klebsiella as opposed to directly harboring the beta-lactamase genes. Here we put forth a robust, accessible, and high-throughput platform for sensitively surveying the bacterial hosts of mobile genes, as well as detecting physical bacterial associations such as those occurring within biofilms and complex microbial communities.

[Source](https://doi.org/10.7554/eLife.66834)

10.7554/eLife.66834

---

## Linking Plasmid-based Beta-lactamases To Their Bacterial Hosts Using Single-cell fusion PCR

[Source](https://doi.org/10.7554/eLife.66834)

---

## An integrated machine learning framework for a discriminative analysis of schizophrenia using multi-biological data.
 20.07.2021, _PF Ke, DS Xiong, JH Li, ZL Pan, J Zhou, SJ Li, J Song, XY Chen, GX Li, J Chen, XB Li, YP Ning, FC Wu, K Wu_


Finding effective and objective biomarkers to inform the diagnosis of schizophrenia is of great importance yet remains challenging. Relatively little work has been conducted on multi-biological data for the diagnosis of schizophrenia. In this cross-sectional study, we extracted multiple features from three types of biological data, including gut microbiota data, blood data, and electroencephalogram data. Then, an integrated framework of machine learning consisting of five classifiers, three feature selection algorithms, and four cross validation methods was used to discriminate patients with schizophrenia from healthy controls. Our results show that the support vector machine classifier without feature selection using the input features of multi-biological data achieved the best performance, with an accuracy of 91.7% and an AUC of 96.5% (p &lt; 0.05). These results indicate that multi-biological data showed better discriminative capacity for patients with schizophrenia than single biological data. The top 5% discriminative features selected from the optimal model include the gut microbiota features (Lactobacillus, Haemophilus, and Prevotella), the blood features (superoxide dismutase level, monocyte-lymphocyte ratio, and neutrophil count), and the electroencephalogram features (nodal local efficiency, nodal efficiency, and nodal shortest path length in the temporal and frontal-parietal brain areas). The proposed integrated framework may be helpful for understanding the pathophysiology of schizophrenia and developing biomarkers for schizophrenia using multi-biological data.

[Source](https://doi.org/10.1038/s41598-021-94007-9)

10.1038/s41598-021-94007-9

---

## An Integrated Machine Learning Framework For A Discriminative Analysis Of schizophrenia using multi-biological data.

[Source](https://doi.org/10.1038/s41598-021-94007-9)

---

## An Intestinal Gut Organ Culture System for Analyzing Host-Microbiota Interactions.
 20.07.2021, _S Azriel, H Bootz, A Shemesh, S Amidror, N Yissachar_


The structure of the gut tissue facilitates close and mutualistic interactions between the host and the gut microbiota. These cross-talks are crucial for maintaining local and systemic homeostasis; changes to gut microbiota composition (dysbiosis) associate with a wide array of human diseases. Methods for dissecting host-microbiota interactions encompass an inherent tradeoff among preservation of physiological tissue structure (when using in vivo animal models) and the level of control over the experiment factors (as in simple in vitro cell culture systems). To address this tradeoff, Yissachar et al. recently developed an intestinal organ culture system. The system preserves a naive colon tissue construction and cellular mechanisms and it also permits tight experimental control, facilitating experimentations that cannot be readily performed in vivo. It is optimal for dissecting short-term responses of various gut components (such as epithelial, immunological and neuronal elements) to luminal perturbations (including anaerobic or aerobic microbes, whole microbiota samples from mice or humans, drugs and metabolites). Here, we present a detailed description of an optimized protocol for organ culture of multiple gut fragments using a custom-made gut culture device. Host responses to luminal perturbations can be visualized by immunofluorescence staining of tissue sections or whole-mount tissue fragments, fluorescence in-situ hybridization (FISH), or time-lapse imaging. This system supports a wide array of readouts, including next-generation sequencing, flow cytometry, and various cellular and biochemical assays. Overall, this three-dimensional organ culture system supports the culture of large, intact intestinal tissues and has broad applications for high-resolution analysis and visualization of host-microbiota interactions in the local gut environment.

[Source](https://doi.org/10.3791/62779)

10.3791/62779

---

