## Ultra-deep sequencing of Hadza hunter-gatherers recovers vanishing gut microbes.
 29.06.2023, _MM Carter, MR Olm, BD Merrill, D Dahan, S Tripathi, SP Spencer, FB Yu, S Jain, N Neff, AR Jha, ED Sonnenburg, JL Sonnenburg_


The gut microbiome modulates immune and metabolic health. Human microbiome data are biased toward industrialized populations, limiting our understanding of non-industrialized microbiomes. Here, we performed ultra-deep metagenomic sequencing on 351 fecal samples from the Hadza hunter-gatherers of Tanzania and comparative populations in Nepal and California. We recovered 91,662 genomes of bacteria, archaea, bacteriophages, and eukaryotes, 44% of which are absent from existing unified datasets. We identified 124 gut-resident species vanishing in industrialized populations and highlighted distinct aspects of the Hadza gut microbiome related to in situ replication rates, signatures of selection, and strain sharing. Industrialized gut microbes were found to be enriched in genes associated with oxidative stress, possibly a result of microbiome adaptation to inflammatory processes. This unparalleled view of the Hadza gut microbiome provides a valuable resource, expands our understanding of microbes capable of colonizing the human gut, and clarifies the extensive perturbation induced by the industrialized lifestyle.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(23)00597-4)

10.1016/j.cell.2023.05.046

---

## A genetic system for <em>Akkermansia muciniphila</em> reveals a role for mucin foraging in gut colonization and host sterol biosynthesis gene expression

[Source](https://doi.org/10.1038/s41564-023-01407-w)

---

## The first international Rome consensus conference on gut microbiota and faecal microbiota transplantation in inflammatory bowel disease.
 20.06.2023, _LR Lopetuso, S Deleu, L Godny, V Petito, P Puca, F Facciotti, H Sokol, G Ianiro, L Masucci, M Abreu, I Dotan, SP Costello, A Hart, TH Iqbal, S Paramsothy, M Sanguinetti, S Danese, H Tilg, F Cominelli, TT Pizarro, A Armuzzi, G Cammarota, A Gasbarrini, S Vermeire, F Scaldaferri_


Several randomised clinical trials (RCTs) performing faecal microbiota transplantation (FMT) for the management of inflammatory bowel disease (IBD), particularly for ulcerative colitis, have recently been published, but with major variations in study design. These include differences in administered dose, route and frequency of delivery, type of placebo and evaluated endpoints. Although the overall outcomes appear to be promising, they are highly dependent on both donor and recipient factors.

[Source](https://doi.org/10.1136/gutjnl-2023-329948)

10.1136/gutjnl-2023-329948

---

## Global branches and local states of the human gut microbiome define associations with environmental and intrinsic factors.
 28.06.2023, _J Tap, F Lejzerowicz, A Cotillard, M Pichaud, D McDonald, SJ Song, R Knight, P Veiga, M Derrien_


_Humans, Gastrointestinal Microbiome, Cross-Sectional Studies, Bacteroides, RNA, Ribosomal, 16S_
The gut microbiome is important for human health, yet modulation requires more insight into inter-individual variation. Here, we explored latent structures of the human gut microbiome across the human lifespan, applying partitioning, pseudotime, and ordination approaches to &gt;35,000 samples. Specifically, three major gut microbiome branches were identified, within which multiple partitions were observed in adulthood, with differential abundances of species along branches. Different compositions and metabolic functions characterized the branches' tips, reflecting ecological differences. An unsupervised network analysis from longitudinal data from 745 individuals showed that partitions exhibited connected gut microbiome states rather than over-partitioning. Stability in the Bacteroides-enriched branch was associated with specific ratios of Faecalibacterium:Bacteroides. We also showed that associations with factors (intrinsic and extrinsic) could be generic, branch- or partition-specific. Our ecological framework for cross-sectional and longitudinal data allows a better understanding of overall variation in the human gut microbiome and disentangles factors associated with specific configurations.

[Source](https://doi.org/10.1038/s41467-023-38558-7)

10.1038/s41467-023-38558-7

---

## Key bacterial taxa determine longitudinal dynamics of aromatic amino acid catabolism in infants' gut.
 27.06.2023, _MF Laursen, AK Sinha, M Pedersen, HM Roager_


_Humans, Infant, Gastrointestinal Microbiome, Bacteria, Fatty Acids, Volatile, Propionates, Feces, Amino Acids, Aromatic_
The development of the gut microbiota in early life is linked to metabolic, neuronal, and immunological development. Recent studies have shown that bacterial production of short-chain fatty acids (SCFAs) and aromatic amino acid (AAA) catabolites in the gut can mediate host-microbe interactions. However, the dynamics of these microbiota-derived metabolites and the key bacterial taxa producing AAA catabolites during infancy are largely unknown. Here, we investigated the longitudinal dynamics of the microbiota and microbiota-derived SCFAs and AAA catabolites in more than 200 fecal samples from 25 healthy breast- or mixed-fed Danish infants during the first 6 months of life. We found that the gut microbiota composition and metabolism were highly individual but showed significant development over time. SCFAs and specific groups of AAA catabolites showed distinct temporal abundance patterns. Furthermore, we identified bacterial taxa responsible for the generation of AAA catabolites by associating the dynamics of gut microbial taxa and AAA catabolites and subsequently validating these associations in vitro by cultivation of strains representing the associated taxa. In addition to specific Bifidobacterium species being the main producers of aromatic lactic acids, we identified Peptostreptococcus anaerobius as the main producer of aromatic propionic acids, Ruminococcus gnavus as a main producer of tryptamine, and Enterococcus species as main tyramine producers in infants' gut. Thus, our results showcase the temporal dynamics of key gut microbial metabolites in early life and demonstrate that the appearance and abundance of specific AAA catabolites result from the appearance and abundance of specific key bacterial taxa in infants' gut.

[Source](https://doi.org/10.1080/19490976.2023.2221426)

10.1080/19490976.2023.2221426

---

## Low diversity microbiota and an increased metabolism of arginine and aromatic amino acids: a hallmark of hepatic encephalopathy in western Mexican patients with alcohol-associated cirrhosis

[Source](https://doi.org/10.1093/jambio/lxad113)

---

## Alterations in the gut virome are associated with type 2 diabetes and diabetic nephropathy.
 26.06.2023, _G Fan, F Cao, T Kuang, H Yi, C Zhao, L Wang, J Peng, Z Zhuang, T Xu, Y Luo, Y Xie, H Li, K Zhang, Y Zeng, X Zhang, S Peng, X Qiu, D Zhou, H Liang, B Yang, J Kang, Y Liu, Y Zhang_


_Humans, Gastrointestinal Microbiome, Diabetic Nephropathies, Virome, Diabetes Mellitus, Type 2, Bacteriophages, Bacteria_
Although changes in gut microbiome have been associated with the development of T2D and its complications, the role of the gut virome remains largely unknown. Here, we characterized the gut virome alterations in T2D and its complications diabetic nephropathy (DN) by metagenomic sequencing of fecal viral-like particles. Compared with controls, T2D subjects, especially those with DN, had significantly lower viral richness and diversity. 81 viral species were identified to be significantly altered in T2D subjects, including a decrease in some phages (e.g. Flavobacterium phage and Cellulophaga phaga). DN subjects were depleted of 12 viral species, including Bacteroides phage, Anoxybacillus virus and Brevibacillus phage, and enriched in 2 phages (Shigella phage and Xylella phage). Multiple viral functions, particularly those of phage lysing host bacteria, were markedly reduced in T2D and DN. Strong viral-bacterial interactions in healthy controls were disrupted in both T2D and DN. Moreover, the combined use of gut viral and bacterial markers achieved a powerful diagnostic performance for T2D and DN, with AUC of 99.03% and 98.19%, respectively. Our results suggest that T2D and its complication DN are characterized by a significant decrease in gut viral diversity, changes in specific virus species, loss of multiple viral functions, and disruption of viral-bacterial correlations. The combined gut viral and bacterial markers have diagnostic potential for T2D and DN.

[Source](https://doi.org/10.1080/19490976.2023.2226925)

10.1080/19490976.2023.2226925

---

## Species-Specific Analysis of Bacterial Vaginosis-Associated Bacteria.
 22.06.2023, _J Osei Sekyere, AB Oyenihi, J Trama, ME Adelson_


Vaginal dysbiosis in women reduces the abundance of Lactobacillus species and increases that of anaerobic fastidious bacteria. This dysbiotic condition in the vagina, called bacterial vaginosis (BV), can be symptomatic with odorous vaginal discharges or asymptomatic and affects a third of women of reproductive age. Three unclassified bacterial species designated BV-associated bacteria 1, 2, and 3 (BVAB-1, -2, and -3) in 2005 were found to be highly preponderant in the vagina of females with BV. Here, we used sequence homology and phylogenetics analyses to identify the actual species of BVAB-1, -2, and -3 and found BVAB-1 to be Clostridiales genomosp. BVAB-1, BVAB-2 to be Oscillospiraceae bacterium strain CHIC02, and BVAB-3 to be Mageeibacillus indolicus, respectively. These are anaerobic and uncultured species that can be identified only through metagenomics. Long-read sequencing of BV specimens can also enable a genomic reassembly of these species' genomes from metagenomes. Species-specific identification of these pathogens and the availability of their genomes from assembled metagenomes will advance our understanding of their biology, facilitate the design of sensitive diagnostics and drugs, and enhance the treatment of BV. IMPORTANCE For many years since 2005, BVAB, an important pathogen of the female vaginal tract that is associated with BV, has been identified using PCR without knowing its actual species. Without a full genome of these pathogens, a better understanding of their pathogenicity, treatment, resistance, and diagnostics cannot be reached. In this analysis, we use the DNA of BVAB-1, -2, and -3 to determine their actual species to enhance further research into their pathogenicity, resistance, diagnosis, and treatment.

[Source](https://doi.org/10.1128/spectrum.04676-22)

10.1128/spectrum.04676-22

---

## Gut microbiota in nonalcoholic fatty liver disease: a PREDIMED-Plus trial sub analysis.
 23.06.2023, _AM Gómez-Pérez, P Ruiz-Limón, J Salas-Salvadó, J Vioque, D Corella, M Fitó, J Vidal, A Atzeni, L Torres-Collado, A Álvarez-Sala, MÁ Martínez, A Goday, D Benaiges, J García-Gavilán, MR Bernal López, I Moreno-Indias, FJ Tinahones_


_Humans, Fibrosis, Gastrointestinal Microbiome, Liver, Metabolic Syndrome, Non-alcoholic Fatty Liver Disease_
To evaluate the changes in the gut microbiota associated with changes in the biochemical markers of nonalcoholic fatty liver disease (NAFLD) after a lifestyle intervention with the Mediterranean diet. Participants (n = 297) from two centers of PREDIMED-Plus trial (Prevención con Dieta Mediterránea) were divided into three different groups based on the change tertile in the Hepatic Steatosis Index (HSI) or the Fibrosis-4 score (FIB-4) between baseline and one year of intervention. One-year changes in HSI were: tertile 1 (T1) (-24.9 to -7.51), T2 (-7.5 to -1.86), T3 (-1.85 to 13.64). The most significant differences in gut microbiota within the year of intervention were observed in the T1 and T3. According to the FIB-4, participants were categorized in non-suspected fibrosis (NSF) and with indeterminate or suspected fibrosis (SF). NSF participants showed higher abundances of Alcaligenaceae, Bacteroidaceae, Bifidobacteriaceae, Clostridiaceae, Enterobacteriaceae, Peptostreptococcaceae, Verrucomicrobiaceae compared to those with SF. Then, participants were divided depending on the FIB-4 tertile of change: T1 (-89.60 to -5.57), T2 (-5.56 to 11.4), and T3 (11.41 to 206.24). FIB-4 T1 showed a decrease in Akkermansia and an increase in Desulfovibrio. T2 had an increase in Victivallaceae, Clostridiaceae, and Desulfovibrio. T3 showed a decrease in Enterobacteriaceae, and an increase in Sutterella, Faecalibacterium, and Blautia. A relation between biochemical index changes of NAFLD/NASH (HSI and FIB-4) and gut microbiota changes were found. These observations highlight the importance of lifestyle intervention in the modulation of gut microbiota and the management of metabolic syndrome and its hepatic manifestations.

[Source](https://doi.org/10.1080/19490976.2023.2223339)

10.1080/19490976.2023.2223339

---

## Associations of Circulating Estrogens and Estrogen Metabolites with Fecal and Oral Microbiome in Postmenopausal Women in the Ghana Breast Health Study.
 21.06.2023, _Z Wu, RM Pfeiffer, DA Byrd, Y Wan, D Ansong, JN Clegg-Lamptey, B Wiafe-Addai, L Edusei, E Adjei, N Titiloye, F Dedey, F Aitpillah, J Oppong, V Vanderpuye, E Osei-Bonsu, CL Dagnall, K Jones, A Hutchinson, BD Hicks, TU Ahearn, R Knight, R Biritwum, J Yarney, S Wiafe, B Awuah, K Nyarko, M Garcia-Closas, R Sinha, JD Figueroa, LA Brinton, B Trabert, E Vogtmann_


The human fecal and oral microbiome may play a role in the etiology of breast cancer through modulation of endogenous estrogen metabolism. This study aimed to investigate associations of circulating estrogens and estrogen metabolites with the fecal and oral microbiome in postmenopausal African women. A total of 117 women with fecal (N = 110) and oral (N = 114) microbiome data measured by 16S rRNA gene sequencing, and estrogens and estrogen metabolites data measured by liquid chromatography tandem mass spectrometry were included. The outcomes were measures of the microbiome and the independent variables were the estrogens and estrogen metabolites. Estrogens and estrogen metabolites were associated with the fecal microbial Shannon index (global P &lt; 0.01). In particular, higher levels of estrone (β = 0.36, P = 0.03), 2-hydroxyestradiol (β = 0.30, P = 0.02), 4-methoxyestrone (β = 0.51, P = 0.01), and estriol (β = 0.36, P = 0.04) were associated with higher levels of the Shannon index, while 16alpha-hydroxyestrone (β = -0.57, P &lt; 0.01) was inversely associated with the Shannon index as indicated by linear regression. Conjugated 2-methoxyestrone was associated with oral microbial unweighted UniFrac as indicated by MiRKAT (P &lt; 0.01) and PERMANOVA, where conjugated 2-methoxyestrone explained 2.67% of the oral microbial variability, but no other estrogens or estrogen metabolites were associated with any other beta diversity measures. The presence and abundance of multiple fecal and oral genera, such as fecal genera from families Lachnospiraceae and Ruminococcaceae, were associated with several estrogens and estrogen metabolites as indicated by zero-inflated negative binomial regression. Overall, we found several associations of specific estrogens and estrogen metabolites and the fecal and oral microbiome. IMPORTANCE Several epidemiologic studies have found associations of urinary estrogens and estrogen metabolites with the fecal microbiome. However, urinary estrogen concentrations are not strongly correlated with serum estrogens, a known risk factor for breast cancer. To better understand whether the human fecal and oral microbiome were associated with breast cancer risk via the regulation of estrogen metabolism, we conducted this study to investigate the associations of circulating estrogens and estrogen metabolites with the fecal and oral microbiome in postmenopausal African women. We found several associations of parent estrogens and several estrogen metabolites with the microbial communities, and multiple individual associations of estrogens and estrogen metabolites with the presence and abundance of multiple fecal and oral genera, such as fecal genera from families Lachnospiraceae and Ruminococcaceae, which have estrogen metabolizing properties. Future large, longitudinal studies to investigate the dynamic changes of the fecal and oral microbiome and estrogen relationship are needed.

[Source](https://doi.org/10.1128/spectrum.01572-23)

10.1128/spectrum.01572-23

---

## Meta-analysis of the effects of proton pump inhibitors on the human gut microbiota.
 28.06.2023, _J Zhang, C Zhang, Q Zhang, L Yu, W Chen, Y Xue, Q Zhai_


_Humans, Proton Pump Inhibitors, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Feces, Microbiota, Bacteria, Clostridiales_
Mounting evidence has linked changes in human gut microbiota to proton pump inhibitor (PPI) use. Accordingly, multiple studies have analyzed the gut microbiomes of PPI users, but PPI-microbe interactions are still understudied. Here, we performed a meta-analysis of four studies with available 16S rRNA gene amplicon sequencing data to uncover the potential changes in human gut microbes among PPI users. Despite some differences, we found common features of the PPI-specific microbiota, including a decrease in the Shannon diversity index and the depletion of bacteria from the Ruminococcaceae and Lachnospiraceae families, which are crucial short-chain fatty acid-producers. Through training based on multiple studies, using a random forest classification model, we further verified the representativeness of the six screened gut microbial genera and 20 functional genes as PPI-related biomarkers, with AUC values of 0.748 and 0.879, respectively. Functional analysis of the PPI-associated 16S rRNA microbiome revealed enriched carbohydrate- and energy-associated genes, mostly encoding fructose-1,6-bisphosphatase and pyruvate dehydrogenase, among others. In this study, we have demonstrated alterations in bacterial abundance and functional metabolic potential related to PPI use, as a basis for future studies on PPI-induced adverse effects.

[Source](https://doi.org/10.1186/s12866-023-02895-w)

10.1186/s12866-023-02895-w

---

## Gut microbiomes of captive primates show phylosymbiosis, respond to dietary sugar reduction, and select for host-specific dietary microbes.
 24.06.2023, _SL Bornbusch, CR Muletz-Wolz, E Lopez-Bondarchuk, MT Maslanka, EL Kendrick_


Host-associated microbiomes are influenced by evolutionary history and proximate factors such as diet and environment. Zoos house animals in relatively standardized and manipulatable environments, making zoo populations valuable for studying microbiomes. Using a small population of five closely related primate species housed under nearly identical environments, we investigated gut microbiome variation regarding (a) congruence between host evolutionary history and gut bacterial composition (i.e. phylosymbiosis), (b) a longitudinal reduction in dietary sugar intake, and (c) ingestion of bacteria from dietary sources. We found that the primate gut microbiomes varied across individuals and showed phylosymbiosis. When animals were fed diets with reduced sugar and increased fiber, we found host-specific changes in taxonomically distinct microbes (Phascolarctobacterium, Megasphaera, and Sharpea). Yet, these bacterial genera share similar functional potential (fiber degradation), indicating that the distinct bacterial communities may fulfill similar functions. Although all individuals received the same diet, the diet-associated bacteria in primate gut microbiomes were distinct across individuals of different species, suggesting a mechanism that selects for unique dietary microbes to persist in animal guts. Our findings show that the microbiomes of a small, captive primate population housed under uniform environmental conditions still show patterns congruent with combined influences of evolutionary history and diet.

[Source](https://doi.org/10.1093/femsec/fiad069)

10.1093/femsec/fiad069

---

## Characterization of two keystone taxa, sulfur-oxidizing, and nitrate-reducing bacteria, by tracking their role transitions in the benzo[a]pyrene degradative microbiome

[Source](https://doi.org/10.1186/s40168-023-01583-1)

---

## Inferring microbial co-occurrence networks from amplicon data: a systematic evaluation

[Source](https://doi.org/10.1128/msystems.00961-22)

---

## Human microbiome cultivation expands with AI.
 22.06.2023, _M Selma-Royo, N Segata, L Ricci_



[Source](https://doi.org/10.1038/s41587-023-01852-2)

10.1038/s41587-023-01852-2

---

## GutBug: A Tool for Prediction of Human Gut Bacteria Mediated Biotransformation of Biotic and Xenobiotic Molecules Using Machine Learning.
 27.06.2023, _AS Malwe, GN Srivastava, VK Sharma_


_Humans, Xenobiotics, Bacteria, Gastrointestinal Microbiome, Pharmaceutical Preparations, Biotransformation, Machine Learning_
Dietary components and bioactive molecules present in functional foods and nutraceuticals provide various beneficial effects including modulation of host gut microbiome. These metabolites along with orally administered drugs can be potentially bio-transformed by gut microbiome, which can alter their bioavailability and intended biological or pharmacological activity resulting in individual or population-specific variation in drug and dietary responses. Experimental determination of microbiome-mediated metabolism of orally ingested molecules is difficult due to the enormous diversity and complexity of the gut microbiome. To address this problem, we developed "GutBug", a web-based resource that predicts all possible bacterial metabolic enzymes that can potentially biotransform xenobiotics and biotic molecules using a combination of machine learning, neural networks and chemoinformatic methods. Using 3,457 enzyme substrates for training and a curated database of 363,872 enzymes from ∼700 gut bacterial strains, GutBug can predict complete EC number of the bacterial enzymes involved in a biotransformation reaction of the given molecule along with the reaction centres with accuracies between 0.78 and 0.97 across different reaction classes. Validation of GutBug's performance using 27 molecules known to be biotransformed by human gut bacteria, including complex polysaccharides, flavonoids, and oral drugs further attests to GutBug's accuracy and utility. Thus, GutBug enhances our understanding of various metabolite-gut bacterial interactions and their resultant effects on the human host health across populations, which will find enormous applications in diet design and intervention, identification and administration of new prebiotics, development of nutraceutical products, and improvements in drug designing. GutBug is available at https://metabiosys.iiserb.ac.in/gutbug.

[Source](https://doi.org/10.1016/j.jmb.2023.168056)

10.1016/j.jmb.2023.168056

---

## Comparison of DNA extraction methods for 16S rRNA gene sequencing in the analysis of the human gut microbiome.
 26.06.2023, _C Elie, M Perret, H Hage, E Sentausa, A Hesketh, K Louis, A Fritah-Lafont, P Leissner, C Vachon, H Rostaing, F Reynier, G Gervasi, A Saliou_


_Humans, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Genes, rRNA, DNA, Microbiota_
The gut microbiome is widely analyzed using high-throughput sequencing, such as 16S rRNA gene amplicon sequencing and shotgun metagenomic sequencing (SMS). DNA extraction is known to have a large impact on the metagenomic analyses. The aim of this study was to compare DNA extraction protocols for 16S sequencing. In that context, four commonly used DNA extraction methods were compared for the analysis of the gut microbiota. Commercial versions were evaluated against modified protocols using a stool preprocessing device (SPD, bioMérieux) upstream DNA extraction. Stool samples from nine healthy volunteers and nine patients with a Clostridium difficile infection were extracted with all protocols and 16S sequenced. Protocols were ranked using wet- and dry-lab criteria, including quality controls of the extracted genomic DNA, alpha-diversity, accuracy using a mock community of known composition and repeatability across technical replicates. SPD improved overall efficiency of three of the four tested protocols compared with their commercial version, in terms of DNA extraction yield, sample alpha-diversity, and recovery of Gram-positive bacteria. The best overall performance was obtained for the S-DQ protocol, SPD combined with the DNeasy PowerLyser PowerSoil protocol from QIAGEN. Based on this evaluation, we strongly believe that the use of such stool preprocessing device improves both the standardization and the quality of the DNA extraction in the human gut microbiome studies.

[Source](https://doi.org/10.1038/s41598-023-33959-6)

10.1038/s41598-023-33959-6

---

## Optimizing metaproteomics database construction: lessons from a study of the vaginal microbiome.
 23.06.2023, _EM Lee, S Srinivasan, SO Purvine, TL Fiedler, OP Leiser, SC Proll, SS Minot, BL Deatherage Kaiser, DN Fredricks_


Metaproteomics, a method for untargeted, high-throughput identification of proteins in complex samples, provides functional information about microbial communities and can tie functions to specific taxa. Metaproteomics often generates less data than other omics techniques, but analytical workflows can be improved to increase usable data in metaproteomic outputs. Identification of peptides in the metaproteomic analysis is performed by comparing mass spectra of sample peptides to a reference database of protein sequences. Although these protein databases are an integral part of the metaproteomic analysis, few studies have explored how database composition impacts peptide identification. Here, we used cervicovaginal lavage (CVL) samples from a study of bacterial vaginosis (BV) to compare the performance of databases built using six different strategies. We evaluated broad versus sample-matched databases, as well as databases populated with proteins translated from metagenomic sequencing of the same samples versus sequences from public repositories. Smaller sample-matched databases performed significantly better, driven by the statistical constraints on large databases. Additionally, large databases attributed up to 34% of significant bacterial hits to taxa absent from the sample, as determined orthogonally by 16S rRNA gene sequencing. We also tested a set of hybrid databases which included bacterial proteins from NCBI RefSeq and translated bacterial genes from the samples. These hybrid databases had the best overall performance, identifying 1,068 unique human and 1,418 unique bacterial proteins, ~30% more than a database populated with proteins from typical vaginal bacteria and fungi. Our findings can help guide the optimal identification of proteins while maintaining statistical power for reaching biological conclusions. IMPORTANCE Metaproteomic analysis can provide valuable insights into the functions of microbial and cellular communities by identifying a broad, untargeted set of proteins. The databases used in the analysis of metaproteomic data influence results by defining what proteins can be identified. Moreover, the size of the database impacts the number of identifications after accounting for false discovery rates (FDRs). Few studies have tested the performance of different strategies for building a protein database to identify proteins from metaproteomic data and those that have largely focused on highly diverse microbial communities. We tested a range of databases on CVL samples and found that a hybrid sample-matched approach, using publicly available proteins from organisms present in the samples, as well as proteins translated from metagenomic sequencing of the samples, had the best performance. However, our results also suggest that public sequence databases will continue to improve as more bacterial genomes are published.

[Source](https://doi.org/10.1128/msystems.00678-22)

10.1128/msystems.00678-22

---

## Assumptions on decision making and environment can yield multiple steady states in microbial community models.
 26.06.2023, _A Theorell, J Stelling_


_Humans, Models, Biological, Computer Simulation, Microbiota, Genome, Decision Making_
Microbial community simulations using genome scale metabolic networks (GSMs) are relevant for many application areas, such as the analysis of the human microbiome. Such simulations rely on assumptions about the culturing environment, affecting if the culture may reach a metabolically stationary state with constant microbial concentrations. They also require assumptions on decision making by the microbes: metabolic strategies can be in the interest of individual community members or of the whole community. However, the impact of such common assumptions on community simulation results has not been investigated systematically.

[Source](https://doi.org/10.1186/s12859-023-05325-w)

10.1186/s12859-023-05325-w

---

## EMBED: Essential MicroBiomE Dynamics, a dimensionality reduction approach for longitudinal microbiome studies.
 28.06.2023, _M Shahin, B Ji, PD Dixit_


_Microbiota, Bacteria_
Dimensionality reduction offers unique insights into high-dimensional microbiome dynamics by leveraging collective abundance fluctuations of multiple bacteria driven by similar ecological perturbations. However, methods providing lower-dimensional representations of microbiome dynamics both at the community and individual taxa levels are not currently available. To that end, we present EMBED: Essential MicroBiomE Dynamics, a probabilistic nonlinear tensor factorization approach. Like normal mode analysis in structural biophysics, EMBED infers ecological normal modes (ECNs), which represent the unique orthogonal modes capturing the collective behavior of microbial communities. Using multiple real and synthetic datasets, we show that a very small number of ECNs can accurately approximate microbiome dynamics. Inferred ECNs reflect specific ecological behaviors, providing natural templates along which the dynamics of individual bacteria may be partitioned. Moreover, the multi-subject treatment in EMBED systematically identifies subject-specific and universal abundance dynamics that are not detected by traditional approaches. Collectively, these results highlight the utility of EMBED as a versatile dimensionality reduction tool for studies of microbiome dynamics.

[Source](https://doi.org/10.1038/s41540-023-00285-6)

10.1038/s41540-023-00285-6

---

## Ordering taxa in image convolution networks improves microbiome-based machine learning accuracy.
 23.06.2023, _O Shtossel, H Isakov, S Turjeman, O Koren, Y Louzoun_


_Humans, Gastrointestinal Microbiome, Artificial Intelligence, RNA, Ribosomal, 16S, Microbiota, Machine Learning_
The human gut microbiome is associated with a large number of disease etiologies. As such, it is a natural candidate for machine-learning-based biomarker development for multiple diseases and conditions. The microbiome is often analyzed using 16S rRNA gene sequencing or shotgun metagenomics. However, several properties of microbial sequence-based studies hinder machine learning (ML), including non-uniform representation, a small number of samples compared with the dimension of each sample, and sparsity of the data, with the majority of taxa present in a small subset of samples. We show here using a graph representation that the cladogram structure is as informative as the taxa frequency. We then suggest a novel method to combine information from different taxa and improve data representation for ML using microbial taxonomy. iMic (image microbiome) translates the microbiome to images through an iterative ordering scheme, and applies convolutional neural networks to the resulting image. We show that iMic has a higher precision in static microbiome gene sequence-based ML than state-of-the-art methods. iMic also facilitates the interpretation of the classifiers through an explainable artificial intelligence (AI) algorithm to iMic to detect taxa relevant to each condition. iMic is then extended to dynamic microbiome samples by translating them to movies.

[Source](https://doi.org/10.1080/19490976.2023.2224474)

10.1080/19490976.2023.2224474

---

