## A small molecule produced by Lactobacillus species blocks Candida albicans filamentation by inhibiting a DYRK1-family kinase.
 25.10.2021, _J MacAlpine, M Daniel-Ivad, Z Liu, J Yano, NM Revie, RT Todd, PJ Stogios, H Sanchez, TR O'Meara, TA Tompkins, A Savchenko, A Selmecki, AO Veri, DR Andes, PL Fidel, N Robbins, J Nodwell, L Whitesell, LE Cowen_


The fungus Candida albicans is an opportunistic pathogen that can exploit imbalances in microbiome composition to invade its human host, causing pathologies ranging from vaginal candidiasis to fungal sepsis. Bacteria of the genus Lactobacillus are colonizers of human mucosa and can produce compounds with bioactivity against C. albicans. Here, we show that some Lactobacillus species produce a small molecule under laboratory conditions that blocks the C. albicans yeast-to-filament transition, an important virulence trait. It remains unexplored whether the compound is produced in the context of the human host. Bioassay-guided fractionation of Lactobacillus-conditioned medium linked this activity to 1-acetyl-β-carboline (1-ABC). We use genetic approaches to show that filamentation inhibition by 1-ABC requires Yak1, a DYRK1-family kinase. Additional biochemical characterization of structurally related 1-ethoxycarbonyl-β-carboline confirms that it inhibits Yak1 and blocks C. albicans biofilm formation. Thus, our findings reveal Lactobacillus-produced 1-ABC can prevent the yeast-to-filament transition in C. albicans through inhibition of Yak1.

[Source](https://doi.org/10.1038/s41467-021-26390-w)

10.1038/s41467-021-26390-w

---

## Bifidobacterium species associated with breastfeeding produce aromatic lactic acids in the infant gut.
 31.10.2021, _MF Laursen, M Sakanaka, N von Burg, U Mörbe, D Andersen, JM Moll, CT Pekmez, A Rivollier, KF Michaelsen, C Mølgaard, MV Lind, LO Dragsted, T Katayama, HL Frandsen, AM Vinggaard, MI Bahl, S Brix, W Agace, TR Licht, HM Roager_


Breastfeeding profoundly shapes the infant gut microbiota, which is critical for early life immune development, and the gut microbiota can impact host physiology in various ways, such as through the production of metabolites. However, few breastmilk-dependent microbial metabolites mediating host-microbiota interactions are currently known. Here, we demonstrate that breastmilk-promoted Bifidobacterium species convert aromatic amino acids (tryptophan, phenylalanine and tyrosine) into their respective aromatic lactic acids (indolelactic acid, phenyllactic acid and 4-hydroxyphenyllactic acid) via a previously unrecognized aromatic lactate dehydrogenase (ALDH). The ability of Bifidobacterium species to convert aromatic amino acids to their lactic acid derivatives was confirmed using monocolonized mice. Longitudinal profiling of the faecal microbiota composition and metabolome of Danish infants (n = 25), from birth until 6 months of age, showed that faecal concentrations of aromatic lactic acids are correlated positively with the abundance of human milk oligosaccharide-degrading Bifidobacterium species containing the ALDH, including Bifidobacterium longum, B. breve and B. bifidum. We further demonstrate that faecal concentrations of Bifidobacterium-derived indolelactic acid are associated with the capacity of these samples to activate in vitro the aryl hydrocarbon receptor (AhR), a receptor important for controlling intestinal homoeostasis and immune responses. Finally, we show that indolelactic acid modulates ex vivo immune responses of human CD4+ T cells and monocytes in a dose-dependent manner by acting as an agonist of both the AhR and hydroxycarboxylic acid receptor 3 (HCA3). Our findings reveal that breastmilk-promoted Bifidobacterium species produce aromatic lactic acids in the gut of infants and suggest that these microbial metabolites may impact immune function in early life.

[Source](https://doi.org/10.1038/s41564-021-00970-4)

10.1038/s41564-021-00970-4

---

## Bifidobacterium catabolism of human milk oligosaccharides overrides endogenous competitive exclusion driving colonization and protection.
 31.10.2021, _BE Heiss, AM Ehrlich, MX Maldonado-Gomez, DH Taft, JA Larke, ML Goodson, CM Slupsky, DJ Tancredi, HE Raybould, DA Mills_


Understanding how exogenous microbes stably colonize the animal gut is essential to reveal mechanisms of action and tailor effective probiotic treatments. Bifidobacterium species are naturally enriched in the gastrointestinal tract of breast-fed infants. Human milk oligosaccharides (HMOs) are associated with this enrichment. However, direct mechanistic proof of the importance of HMOs in this colonization is lacking given milk contains additional factors that impact the gut microbiota. This study examined mice supplemented with the HMO 2'fucosyllactose (2'FL) together with a 2'FL-consuming strain, Bifidobacterium pseudocatenulatum MP80. 2'FL supplementation creates a niche for high levels of B.p. MP80 persistence, similar to Bifidobacterium levels seen in breast-fed infants. This synergism impacted gut microbiota composition, activated anti-inflammatory pathways and protected against chemically-induced colitis. These results demonstrate that bacterial-milk glycan interactions alone drive enrichment of beneficial Bifidobacterium and provide a model for tunable colonization thus facilitating insight into mechanisms of health promotion by bifidobacteriain neonates.

[Source](https://doi.org/10.1080/19490976.2021.1986666)

10.1080/19490976.2021.1986666

---

## Mother's Milk Microbiome Shaping Fecal and Skin Microbiota in Infants with Food Allergy and Atopic Dermatitis: A Pilot Analysis.
 26.10.2021, _M Gołębiewski, E Łoś-Rycharska, M Sikora, T Grzybowski, M Gorzkiewicz, A Krogulska_


The child microbiome, including gut and skin communities, is shaped by a multitude of factors, and breastfeeding is one of the most essential. Food allergy (FA) and atopic dermatitis (AD) are among the most common diseases in pediatrics, with the prevalence of each up to 6% and 20%, respectively. Therefore, we aimed at finding differences between the fecal and skin microbiomes of FA and AD patients in the context of breastfeeding, by means of the Illumina sequencing of 16S rRNA gene fragment libraries amplified from the total DNA isolated from samples collected from allergic and healthy infants. We also analyzed milk samples from the mothers of the examined children and searched for patterns of incidence suggesting milk influence on an infant's allergy status. Here we show that a mother's milk influences her child's fecal and skin microbiomes and identify Acinetobacter as the taxon whose abundance is correlated with milk and child-derived samples. We demonstrate that breastfeeding makes allergic children's fecal and skin communities more similar to those of healthy infants than in the case of formula-feeding. We also identify signature taxa that might be important in maintaining health or allergy development.

[Source](https://doi.org/10.3390/nu13103600)

10.3390/nu13103600

---

## Early-life formula feeding is associated with infant gut microbiota alterations and an increased antibiotic resistance load.

[Source](https://doi.org/10.1093/ajcn/nqab353)

---

## A Prospective, Open-Label Pilot Study of Concurrent Male Partner Treatment for Bacterial Vaginosis.
 27.10.2021, _EL Plummer, LA Vodstrcil, M Doyle, JA Danielewski, GL Murray, G Fehler, CK Fairley, DM Bulach, SM Garland, EPF Chow, JS Hocking, CS Bradshaw_


Up to 50% of women receiving first-line antibiotics for bacterial vaginosis (BV) experience recurrence within 12 weeks. Evidence suggests that reinfection from an untreated regular sexual partner contributes to recurrence. We conducted a pilot study of 34 heterosexual couples to describe the impact of concurrent partner treatment on the composition of the genital microbiota over a 12-week period. We also determined the acceptability and tolerability of concurrent partner treatment and obtained preliminary estimates of the efficacy of the intervention to inform a randomized controlled trial (RCT). Women received first-line antibiotic treatment for BV (i.e., oral metronidazole or intravaginal clindamycin), and their male partner received oral metronidazole, 400 mg, and 2% clindamycin cream applied topically to penile skin, both twice daily for 7 days. The genital microbiota was characterized at three anatomical sites (women, vaginal; men, cutaneous penile and first-pass urine [representing the urethra]) using 16S rRNA gene sequencing. Immediately posttreatment, concurrent partner treatment significantly reduced the abundance of BV-associated bacteria (false-discovery rate [FDR] corrected P value &lt; 0.05) and altered the overall microbiota composition of all three anatomical sites (P = 0.001). Suppression of BV-associated bacteria was sustained in the majority (81%) of women over the 12-week period (FDR P value &lt; 0.05), despite BV-associated bacteria reemerging at both genital sites in men. In this cohort of women at high risk for recurrence, five recurred within 12 weeks of treatment (17%; 95% confidence interval [CI], 6 to 34%). Importantly, men tolerated and adhered to combination therapy. Our findings provide support for an RCT of combined oral and topical male partner treatment for BV. IMPORTANCE Recurrence of BV following standard treatment is unacceptably high. Posttreatment recurrence is distressing for women, and it imposes a considerable burden on the health care system. Recurrences result in multiple presentations to clinical services and repeated antibiotic use, and the associated obstetric and gynecological sequelae are significant. New treatments to improve long-term BV cure are urgently needed. Here, we used 16S rRNA gene sequencing to investigate changes in the microbiota composition at three genital sites (vagina, penile skin, and male urethra) of heterosexual couples undergoing concurrent partner treatment for bacterial vaginosis (BV). We found that concurrent partner treatment immediately and significantly altered the composition of the genital microbiota of both partners, with a reduction in BV-associated bacteria seen at all three sites. BV cure at 12 weeks posttreatment was higher than expected. These microbiological data provide evidence for continued investigation of partner treatment as a strategy to improve BV cure.

[Source](https://doi.org/10.1128/mBio.02323-21)

10.1128/mBio.02323-21

---

## Tungsten enzymes play a role in detoxifying food and antimicrobial aldehydes in the human gut microbiome.
 25.10.2021, _GJ Schut, MP Thorgersen, FL Poole, DK Haja, S Putumbaka, MWW Adams_


Tungsten (W) is a metal that is generally thought to be seldom used in biology. We show here that a W-containing oxidoreductase (WOR) family is diverse and widespread in the microbial world. Surprisingly, WORs, along with the tungstate-specific transporter Tup, are abundant in the human gut microbiome, which contains 24 phylogenetically distinct WOR types. Two model gut microbes containing six types of WOR and Tup were shown to assimilate W. Two of the WORs were natively purified and found to contain W. The enzymes catalyzed the conversion of toxic aldehydes to the corresponding acid, with one WOR carrying out an electron bifurcation reaction coupling aldehyde oxidation to the simultaneous reduction of NAD+ and of the redox protein ferredoxin. Such aldehydes are present in cooked foods and are produced as antimicrobials by gut microbiome metabolism. This aldehyde detoxification strategy is dependent on the availability of W to the microbe. The functions of other WORs in the gut microbiome that do not oxidize aldehydes remain unknown. W is generally beyond detection (&lt;6 parts per billion) in common foods and at picomolar concentrations in drinking water, suggesting that W availability could limit some gut microbial functions and might be an overlooked micronutrient.

[Source](https://doi.org/10.1073/pnas.2109008118)

10.1073/pnas.2109008118

---

## Commensal bacteria promote endocrine resistance in prostate cancer through androgen biosynthesis.
 15.10.2021, _N Pernigoni, E Zagato, A Calcinotto, M Troiani, RP Mestre, B Calì, G Attanasio, J Troisi, M Minini, S Mosole, A Revandkar, E Pasquini, AR Elia, D Bossi, A Rinaldi, P Rescigno, P Flohr, J Hunt, A Neeb, L Buroni, C Guo, J Welti, M Ferrari, M Grioni, J Gauthier, RZ Gharaibeh, A Palmisano, GM Lucchini, E D'Antonio, S Merler, M Bolis, F Grassi, A Esposito, M Bellone, A Briganti, M Rescigno, JP Theurillat, C Jobin, S Gillessen, J de Bono, A Alimonti_


_Aged, Aged, 80 and over, Androgen Antagonists, Androgens, Animals, Anti-Bacterial Agents, Bacteria, Cell Line, Tumor, Fecal Microbiota Transplantation, Gastrointestinal Microbiome, Host Microbial Interactions, Humans, Male, Mice, Mice, Inbred C57BL, Mice, Inbred NOD, Mice, SCID, Middle Aged, Neoplasms, Experimental, Prevotella, Prostatic Neoplasms, Castration-Resistant, Symbiosis, Xenograft Model Antitumor Assays_
[Figure: see text].

[Source](https://doi.org/10.1126/science.abf8403)

10.1126/science.abf8403

---

## Vertebrate host phylogeny influences gut archaeal diversity.
 31.10.2021, _ND Youngblut, GH Reischer, S Dauser, S Maisch, C Walzer, G Stalder, AH Farnleitner, RE Ley_


Commonly used 16S rRNA gene primers do not detect the full range of archaeal diversity present in the vertebrate gut. As a result, several questions regarding the archaeal component of the gut microbiota remain, including which Archaea are host-associated, the specificities of such associations and the major factors influencing archaeal diversity. Using 16S rRNA gene amplicon sequencing with primers that specifically target Archaea, we obtained sufficient sequence data from 185 gastrointestinal samples collected from 110 vertebrate species that span five taxonomic classes (Mammalia, Aves, Reptilia, Amphibia and Actinopterygii), of which the majority were wild. We provide evidence for previously undescribed Archaea-host associations, including Bathyarchaeia and Methanothermobacter, the latter of which was prevalent among Aves and relatively abundant in species with higher body temperatures, although this association could not be decoupled from host phylogeny. Host phylogeny explained archaeal diversity more strongly than diet, while specific taxa were associated with both factors, and cophylogeny was significant and strongest for mammalian herbivores. Methanobacteria was the only class predicted to be present in the last common ancestors of mammals and all host species. Further analysis indicated that Archaea-Bacteria interactions have a limited effect on archaeal diversity. These findings expand our current understanding of Archaea-vertebrate associations.

[Source](https://doi.org/10.1038/s41564-021-00980-2)

10.1038/s41564-021-00980-2

---

## Gut microbiota dysbiosis contributes to the development of chronic obstructive pulmonary disease.
 29.10.2021, _N Li, Z Dai, Z Wang, Z Deng, J Zhang, J Pu, W Cao, T Pan, Y Zhou, Z Yang, J Li, B Li, P Ran_


Dysbiosis of the gut microbiome is involved in the pathogenesis of various diseases, but the contribution of gut microbes to the progression of chronic obstructive pulmonary disease (COPD) is still poorly understood.

[Source](https://doi.org/10.1186/s12931-021-01872-z)

10.1186/s12931-021-01872-z

---

## Oral Administration of <em>Latilactobacillus sakei </em>ADM14 Improves Lipid Metabolism and Fecal Microbiota Profile Associated With Metabolic Dysfunction in a High-Fat Diet Mouse Model.

[Source](https://doi.org/10.3389/fmicb.2021.746601)

---

## Discrimination of Bacterial Community Structures among Healthy, Gingivitis, and Periodontitis Statuses through Integrated Metatranscriptomic and Network Analyses.
 26.10.2021, _T Nemoto, T Shiba, K Komatsu, T Watanabe, M Shimogishi, M Shibasaki, T Koyanagi, T Nagai, S Katagiri, Y Takeuchi, T Iwata_


Periodontal disease is an inflammatory condition caused by polymicrobial infection. The inflammation is initiated at the gingiva (gingivitis) and then extends to the alveolar bone, leading to tooth loss (periodontitis). Previous studies have shown differences in bacterial composition between periodontal healthy and diseased sites. However, bacterial metabolic activities during the health-to-periodontitis microbiome shift are still inadequately understood. This study was performed to investigate the bacterial characteristics of healthy, gingivitis, and periodontitis statuses through metatranscriptomic analysis. Subgingival plaque samples of healthy, gingivitis, and periodontitis sites in the same oral cavity were collected from 21 patients. Bacterial compositions were then determined based on 16S rRNA reads; taxonomic and functional profiles derived from genes based on mRNA reads were estimated. The results showed clear differences in bacterial compositions and functional profiles between healthy and periodontitis sites. Co-occurrence networks were constructed for each group by connecting two bacterial species if their mRNA abundances were positively correlated. The clustering coefficient values were 0.536 for healthy, 0.600 for gingivitis, and 0.371 for periodontitis sites; thus, network complexity increased during gingivitis development, whereas it decreased during progression to periodontitis. Taxa, including Eubacterium nodatum, Eubacterium saphenum, Filifactor alocis, and Fretibacterium fastidiosum, showed greater transcriptional activities than those of red complex bacteria, in conjunction with disease progression. These taxa were associated with periodontal disease progression, and the health-to-periodontitis microbiome shift was accompanied by alterations in bacterial network structure and complexity. IMPORTANCE The characteristics of the periodontal microbiome influence clinical periodontal status. Gingivitis involves reversible gingival inflammation without alveolar bone resorption. In contrast, periodontitis is an irreversible disease characterized by inflammatory destruction in both soft and hard tissues. An imbalance of the microbiome is present in both gingivitis and periodontitis. However, differences in microbiomes and their functional activities in the healthy, gingivitis, and periodontitis statuses are still inadequately understood. Furthermore, some inflamed gingival statuses do not consistently cause attachment loss. In this study, metatranscriptomic analyses were used to investigate the specific bacterial composition and gene expression patterns of the microbiomes of the healthy, gingivitis, and periodontitis statuses. In addition, co-occurrence network analysis revealed that the gingivitis site included features of networks observed in both the healthy and periodontitis sites. These results provide transcriptomic evidence to support gingivitis as an intermediate state between the healthy and periodontitis statuses.

[Source](https://doi.org/10.1128/mSystems.00886-21)

10.1128/mSystems.00886-21

---

## Commensal segmented filamentous bacteria-derived retinoic acid primes host defense to intestinal infection.
 22.10.2021, _V Woo, EM Eshleman, S Hashimoto-Hill, J Whitt, SE Wu, L Engleman, T Rice, R Karns, JE Qualls, DB Haslam, BA Vallance, T Alenghat_


Interactions between the microbiota and mammalian host are essential for defense against infection, but the microbial-derived cues that mediate this relationship remain unclear. Here, we find that intestinal epithelial cell (IEC)-associated commensal bacteria, segmented filamentous bacteria (SFB), promote early protection against the pathogen Citrobacter rodentium, independent of CD4+ T cells. SFB induced histone modifications in IECs at sites enriched for retinoic acid receptor motifs, suggesting that SFB may enhance defense through retinoic acid (RA). Consistent with this, inhibiting RA signaling suppressed SFB-induced protection. Intestinal RA levels were elevated in SFB mice, despite the inhibition of mammalian RA production, indicating that SFB directly modulate RA. Interestingly, RA was produced by intestinal bacteria, and the loss of bacterial-intrinsic aldehyde dehydrogenase activity decreased the RA levels and increased infection. These data reveal RA as an unexpected microbiota-derived metabolite that primes innate defense and suggests that pre- and probiotic approaches to elevate RA could prevent or combat infections.

[Source](https://doi.org/10.1016/j.chom.2021.09.010)

10.1016/j.chom.2021.09.010

---

## Intestinal microbiota shapes gut physiology and regulates enteric neurons and glia.
 30.10.2021, _FA Vicentini, CM Keenan, LE Wallace, C Woods, JB Cavin, AR Flockton, WB Macklin, J Belkind-Gerson, SA Hirota, KA Sharkey_


The intestinal microbiota plays an important role in regulating gastrointestinal (GI) physiology in part through interactions with the enteric nervous system (ENS). Alterations in the gut microbiome frequently occur together with disturbances in enteric neural control in pathophysiological conditions. However, the mechanisms by which the microbiota regulates GI function and the structure of the ENS are incompletely understood. Using a mouse model of antibiotic (Abx)-induced bacterial depletion, we sought to determine the molecular mechanisms of microbial regulation of intestinal function and the integrity of the ENS. Spontaneous reconstitution of the Abx-depleted microbiota was used to assess the plasticity of structure and function of the GI tract and ENS. Microbiota-dependent molecular mechanisms of ENS neuronal survival and neurogenesis were also assessed.

[Source](https://doi.org/10.1186/s40168-021-01165-z)

10.1186/s40168-021-01165-z

---

## <em>Lactobacillus rhamnosus</em> GG Derived Extracellular Vesicles Modulate Gut Microbiota and Attenuate Inflammatory in DSS-Induced Colitis Mice.

[Source](https://doi.org/10.3390/nu13103319)

---

## Phage Treatment Trial to Eradicate LA-MRSA from Healthy Carrier Pigs.
 29.10.2021, _H Tuomala, M Verkola, A Meller, J Van der Auwera, S Patpatia, A Järvinen, M Skurnik, A Heikinheimo, S Kiljunen_


The increase of livestock-associated methicillin-resistant Staphylococcus aureus (LA-MRSA) causes a threat to human health. LA-MRSA can be transmitted from animals to animal caretakers, which may further spread MRSA to communities and health care facilities. The objective of this work was to study the efficacy of phage treatment in the eradication of LA-MRSA from healthy carrier pigs. A total of 19 MRSA -positive weanling pigs were assigned to a test (n = 10) and a control group (n = 9). A phage cocktail containing three Staphylococcus phages, or a control buffer was administered to the nares and skin of the pigs three times every two days, after which the phage and MRSA levels in nasal and skin swab samples were monitored for a three-week period. The sensitivity of the strains isolated during the follow-up period to the phage cocktail and each phage individually was analyzed and the pig sera were tested for antibodies against the phages used in the cocktail. The phage treatment did not cause any side effects to the pigs. Phages were found in the skin and nasal samples on the days following the phage applications, but there was no reduction in the MRSA levels in the sampled animals. Phage-resistant strains or phage-specific antibodies were not detected during the experiment. The MRSA load in these healthy carrier animals was only 10-100 CFU/swab or nasal sample, which was likely below the replication threshold of phages. The effectiveness of phage treatment to eradicate MRSA from the pigs could thus not be (reliably) determined.

[Source](https://doi.org/10.3390/v13101888)

10.3390/v13101888

---

## Characterization of the Eukaryotic Virome of Mice from Different Sources.
 28.10.2021, _C Zhang, M Burch, K Wylie, B Herter, CL Franklin, AC Ericsson_


Accumulating studies show that the host microbiome influences the development or progression of many diseases. The eukaryotic virome, as a key component of the microbiome, plays an important role in host health and disease in humans and animals, including research animals designed to model human disease. To date, the majority of research on the microbiome has focused on bacterial populations, while less attention has been paid to the viral component. Members of the eukaryotic virome interact with the commensal bacterial microbiome through trans-kingdom interactions, and influence host immunity and disease phenotypes as a collective microbial ecosystem. As such, differences in the virome may affect the reproducibility of animal models, and supplementation of the virome may enhance the translatability of animal models of human disease. However, there are minimal empirical data regarding differences in the virome of mice from different commercial sources. Our hypotheses were that the mice obtained from pet store sources and lab mice differ in their eukaryotic virome, and that lab mice from different sources would also have different viromes. To test this hypothesis, the ViroCap platform was used to characterize the eukaryotic virome in multiple tissues of mice from different sources including three sources of laboratory mice and two pet stores. As expected, pet store mice harbored a much greater diversity within the virome compared to lab mice. This included an ostensibly novel norovirus strain identified in one source of these mice. Viruses found in both laboratory and pet store populations included four strains of endogenous retroviruses and murine astrovirus with the latter being restricted to one source of lab mice. Considering the relatively high richness virome within different samples from healthy humans, these data suggest that mouse models from alternative sources may be more translational to the human condition. Moreover, these data demonstrate that, by characterizing the eukaryotic murine virome from different sources, novel viruses may be identified for use as field strains in biomedical research.

[Source](https://doi.org/10.3390/microorganisms9102064)

10.3390/microorganisms9102064

---

## Supplemental L-Arginine Improves the Embryonic Intestine Development and Microbial Succession in a Chick Embryo Model.

[Source](https://doi.org/10.3389/fnut.2021.692305)

---

## Plastic-Degrading Potential across the Global Microbiome Correlates with Recent Pollution Trends.
 27.10.2021, _J Zrimec, M Kokina, S Jonasson, F Zorrilla, A Zelezniak_


Biodegradation is a plausible route toward sustainable management of the millions of tons of plastic waste that have accumulated in terrestrial and marine environments. However, the global diversity of plastic-degrading enzymes remains poorly understood. Taking advantage of global environmental DNA sampling projects, here we constructed hidden Markov models from experimentally verified enzymes and mined ocean and soil metagenomes to assess the global potential of microorganisms to degrade plastics. By controlling for false positives using gut microbiome data, we compiled a catalogue of over 30,000 nonredundant enzyme homologues with the potential to degrade 10 different plastic types. While differences between the ocean and soil microbiomes likely reflect the base compositions of these environments, we find that ocean enzyme abundance increases with depth as a response to plastic pollution and not merely taxonomic composition. By obtaining further pollution measurements, we observed that the abundance of the uncovered enzymes in both ocean and soil habitats significantly correlates with marine and country-specific plastic pollution trends. Our study thus uncovers the earth microbiome's potential to degrade plastics, providing evidence of a measurable effect of plastic pollution on the global microbial ecology as well as a useful resource for further applied research. IMPORTANCE Utilization of synthetic biology approaches to enhance current plastic degradation processes is of crucial importance, as natural plastic degradation processes are very slow. For instance, the predicted lifetime of a polyethylene terephthalate (PET) bottle under ambient conditions ranges from 16 to 48 years. Moreover, although there is still unexplored diversity in microbial communities, synergistic degradation of plastics by microorganisms holds great potential to revolutionize the management of global plastic waste. To this end, the methods and data on novel plastic-degrading enzymes presented here can help researchers by (i) providing further information about the taxonomic diversity of such enzymes as well as understanding of the mechanisms and steps involved in the biological breakdown of plastics, (ii) pointing toward the areas with increased availability of novel enzymes, and (iii) giving a basis for further application in industrial plastic waste biodegradation. Importantly, our findings provide evidence of a measurable effect of plastic pollution on the global microbial ecology.

[Source](https://doi.org/10.1128/mBio.02155-21)

10.1128/mBio.02155-21

---

## Acrylate protects a marine bacterium from grazing by a ciliate predator.
 31.10.2021, _ZJ Teng, P Wang, XL Chen, R Guillonneau, CY Li, SB Zou, J Gong, KW Xu, L Han, C Wang, DJ Scanlan, Y Chen, YZ Zhang_


Cleavage of dimethylsulfoniopropionate (DMSP) can deter herbivores in DMSP-producing eukaryotic algae; however, it is unclear whether a parallel defence mechanism operates in marine bacteria. Here we demonstrate that the marine bacterium Puniceibacterium antarcticum SM1211, which does not use DMSP as a carbon source, has a membrane-associated DMSP lyase, DddL. At high concentrations of DMSP, DddL causes an accumulation of acrylate around cells through the degradation of DMSP, which protects against predation by the marine ciliate Uronema marinum. The presence of acrylate can alter the grazing preference of U. marinum to other bacteria in the community, thereby influencing community structure.

[Source](https://doi.org/10.1038/s41564-021-00981-1)

10.1038/s41564-021-00981-1

---

## The fungal root endophyte Serendipita vermifera displays inter-kingdom synergistic beneficial effects with the microbiota in Arabidopsis thaliana and barley.
 26.10.2021, _LK Mahdi, S Miyauchi, C Uhlmann, R Garrido-Oter, G Langen, S Wawra, Y Niu, R Guan, S Robertson-Albertyn, D Bulgarelli, JE Parker, A Zuccaro_


Plant root-associated bacteria can confer protection against pathogen infection. By contrast, the beneficial effects of root endophytic fungi and their synergistic interactions with bacteria remain poorly defined. We demonstrate that the combined action of a fungal root endophyte from a widespread taxon with core bacterial microbiota members provides synergistic protection against an aggressive soil-borne pathogen in Arabidopsis thaliana and barley. We additionally reveal early inter-kingdom growth promotion benefits which are host and microbiota composition dependent. Using RNA-sequencing, we show that these beneficial activities are not associated with extensive host transcriptional reprogramming but rather with the modulation of expression of microbial effectors and carbohydrate-active enzymes.

[Source](https://doi.org/10.1038/s41396-021-01138-y)

10.1038/s41396-021-01138-y

---

## Spatial Characterization of Microbial Communities on Multi-Species Leafy Greens Grown Simultaneously in the Vegetable Production Systems on the International Space Station.
 26.10.2021, _ME Hummerick, CLM Khodadad, AR Dixit, LE Spencer, GJ Maldonado-Vasquez, JL Gooden, CJ Spern, JA Fischer, N Dufour, RM Wheeler, MW Romeyn, TM Smith, GD Massa, Y Zhang_


The establishment of steady-state continuous crop production during long-term deep space missions is critical for providing consistent nutritional and psychological benefits for the crew, potentially improving their health and performance. Three technology demonstrations were completed achieving simultaneous multi-species plant growth and the concurrent use of two Veggie units on the International Space Station (ISS). Microbiological characterization using molecular and culture-based methods was performed on leaves and roots from two harvests of three leafy greens, red romaine lettuce (Lactuca sativa cv. 'Outredgeous'); mizuna mustard, (Brassica rapa var japonica); and green leaf lettuce, (Lactuca sativa cv. Waldmann's) and associated rooting pillow components and Veggie chamber surfaces. Culture based enumeration and pathogen screening indicated the leafy greens were safe for consumption. Surface samples of the Veggie facility and plant pillows revealed low counts of bacteria and fungi and are commonly isolated on ISS. Community analysis was completed with 16S rRNA amplicon sequencing. Comparisons between pillow components, and plant tissue types from VEG-03D, E, and F revealed higher diversity in roots and rooting substrate than the leaves and wick. This work provides valuable information for food production-related research on the ISS and the impact of the plant microbiome on this unique closed environment.

[Source](https://doi.org/10.3390/life11101060)

10.3390/life11101060

---

## Relationships between community composition, productivity and invasion resistance in semi-natural bacterial microcosms.
 23.10.2021, _ML Jones, DW Rivett, A Pascual-García, T Bell_


Common garden experiments that inoculate a standardised growth medium with synthetic microbial communities (i.e. constructed from individual isolates or using dilution cultures) suggest that the ability of the community to resist invasions by additional microbial taxa can be predicted by the overall community productivity (broadly defined as cumulative cell density and/or growth rate). However, to the best of our knowledge, no common garden study has yet investigated the relationship between microbial community composition and invasion resistance in microcosms whose compositional differences reflect natural, rather than laboratory-designed, variation. We conducted experimental invasions of two bacterial strains (Pseudomonas fluorescens and Pseudomonas putida) into laboratory microcosms inoculated with 680 different mixtures of bacteria derived from naturally occurring microbial communities collected in the field. Using 16S rRNA gene amplicon sequencing to characterise microcosm starting composition, and high-throughput assays of community phenotypes including productivity and invader survival, we determined that productivity is a key predictor of invasion resistance in natural microbial communities, substantially mediating the effect of composition on invasion resistance. The results suggest that similar general principles govern invasion in artificial and natural communities, and that factors affecting resident community productivity should be a focal point for future microbial invasion experiments.

[Source](https://doi.org/10.7554/eLife.71811)

10.7554/eLife.71811

---

## High-efficiency delivery of CRISPR-Cas9 by engineered probiotics enables precise microbiome editi

[Source](https://doi.org/10.15252/msb.202110335)

---

## Visualizing transfer of microbial biomolecules by outer membrane vesicles in microbe-host-communication in vivo.
 28.10.2021, _M Bittel, P Reichert, I Sarfati, A Dressel, S Leikam, S Uderhardt, I Stolzer, TA Phu, M Ng, NK Vu, S Tenzer, U Distler, S Wirtz, V Rothhammer, MF Neurath, RL Raffai, C Günther, S Momma_


The intestinal microbiota influences mammalian host physiology in health and disease locally in the gut but also in organs devoid of direct contact with bacteria such as the liver and brain. Extracellular vesicles (EVs) or outer membrane vesicles (OMVs) released by microbes are increasingly recognized for their potential role as biological shuttle systems for inter-kingdom communication. However, physiologically relevant evidence for the transfer of functional biomolecules from the intestinal microbiota to individual host cells by OMVs in vivo is scarce. By introducing Escherichia coli engineered to express Cre-recombinase (E. coliCre ) into mice with a Rosa26.tdTomato-reporter background, we leveraged the Cre-LoxP system to report the transfer of bacterial OMVs to recipient cells in vivo. Colonizing the intestine of these mice with E. coliCre , resulted in Cre-recombinase induced fluorescent reporter gene-expression in cells along the intestinal epithelium, including intestinal stem cells as well as mucosal immune cells such as macrophages. Furthermore, even far beyond the gut, bacterial-derived Cre induced extended marker gene expression in a wide range of host tissues, including the heart, liver, kidney, spleen, and brain. Together, our findings provide a method and proof of principle that OMVs can serve as a biological shuttle system for the horizontal transfer of functional biomolecules between bacteria and mammalian host cells.

[Source](https://doi.org/10.1002/jev2.12159)

10.1002/jev2.12159

---

