## Citizen-science reveals changes in the oral microbiome in Spain through age and lifestyle factors.
 23.05.2022, _JR Willis, E Saus, S Iraola-Guzmán, E Ksiezopolska, L Cozzuto, LA Bejarano, N Andreu-Somavilla, M Alloza-Trabado, A Blanco, A Puig-Sola, E Broglio, C Carolis, J Ponomarenko, J Hecht, T Gabaldón_


_Bacteria, Humans, Life Style, Microbiota, Middle Aged, Mouth, Spain_
The relevance of the human oral microbiome to our understanding of human health has grown in recent years as microbiome studies continue to develop. Given the links of the oral cavity with the digestive, respiratory and circulatory systems, the composition of the oral microbiome is relevant beyond just oral health, impacting systemic processes across the body. However, we still have a very limited understanding about intrinsic and extrinsic factors that shape the composition of the healthy oral microbiome. Here, we followed a citizen-science approach to assess the relative impact on the oral microbiome of selected biological, social, and lifestyle factors in 1648 Spanish individuals. We found that the oral microbiome changes across age, with middle ages showing a more homogeneous composition, and older ages showing more diverse microbiomes with increased representation of typically low abundance taxa. By measuring differences within and between groups of individuals sharing a given parameter, we were able to assess the relative impact of different factors in driving specific microbial compositions. Chronic health disorders present in the analyzed population were the most impactful factors, followed by smoking and the presence of yeasts in the oral cavity. Finally, we corroborate findings in the literature that relatives tend to have more similar oral microbiomes, and show for the first time a similar effect for classmates. Multiple intrinsic and extrinsic factors jointly shape the oral microbiome. Comparative analysis of metabarcoding data from a large sample set allows us to disentangle the individual effects.

[Source](https://doi.org/10.1038/s41522-022-00279-y)

10.1038/s41522-022-00279-y

---

## Gut Microbiome Was Highly Related to the Regulation of Metabolism in Lung Adenocarcinoma Patients

[Source](https://doi.org/10.3389/fonc.2022.790467)

---

## Host-Population Microbial Diversity Scaling of Chinese Gut Microbiomes in Gout Patients.
 20.05.2022, _J Zhou, Y Yuan, P Xu, B Yi, H Chen, W Su_


Gout is a prevalent chronic inflammatory disease that affects the life of tens of millions of people worldwide, and it typically presents as gout arthritis, gout stone, or even kidney damage. Research has revealed its connection with the gut microbiome, although exact mechanism is still unclear. Studies have shown the decline of microbiome diversity in gout patients and change of microbiome compositions between the gout patients and healthy controls. Nevertheless, how diversity changes across host individuals at a cohort (population) level has not been investigated to the best of our knowledge. Here we apply the diversity-area relationship (DAR), which is an extension to the classic SAR (species-area relationship) and establishes the power-function model between microbiome diversity and the number of individuals within cohort, to comparatively investigate diversity scaling (changes) of gut microbiome in gout patients and healthy controls. The DAR modeling with a study involving 83 subjects (41 gout patients) revealed that the potential number of microbial species in gout patients is only 70% of that in the healthy control (2790 vs 3900) although the difference may not be statistically significant. The other DAR parameters including diversity scaling and similarity parameters did not show statistically significant differences. We postulate that the high resilience of gut microbiome may explain the lack of significant gout-disease effects on gut microbial diversity at the population level. The lack of statistically significant difference between the gout patients and healthy controls at host population (cohort) level is different from the previous findings at individual level in the existing literature.

[Source](https://doi.org/10.1177/11769343221095858)

10.1177/11769343221095858

---

## Combined Microbiome and Metabolome Analysis Reveals a Novel Interplay Between Intestinal Flora and Serum Metabolites in Lung Cancer.
 20.05.2022, _S Chen, R Gui, XH Zhou, JH Zhang, HY Jiang, HT Liu, YF Fu_


_Feces, Gastrointestinal Microbiome, Humans, Lung Neoplasms, Metabolome, Quality of Life, RNA, Ribosomal, 16S, Valine_
As the leading cause of cancer death, lung cancer seriously endangers human health and quality of life. Although many studies have reported the intestinal microbial composition of lung cancer, little is known about the interplay between intestinal microbiome and metabolites and how they affect the development of lung cancer. Herein, we combined 16S ribosomal RNA (rRNA) gene sequencing and liquid chromatography-mass spectrometry (LC-MS) technology to analyze intestinal microbiota composition and serum metabolism profile in a cohort of 30 lung cancer patients with different stages and 15 healthy individuals. Compared with healthy people, we found that the structure of intestinal microbiota in lung cancer patients had changed significantly (Adonis, p = 0.021). In order to determine how intestinal flora affects the occurrence and development of lung cancer, the Spearman rank correlation test was used to find the connection between differential microorganisms and differential metabolites. It was found that as thez disease progressed, L-valine decreased. Correspondingly, the abundance of Lachnospiraceae_UCG-006, the genus with the strongest association with L-valine, also decreased in lung cancer groups. Correlation analysis showed that the gut microbiome and serum metabolic profile had a strong synergy, and Lachnospiraceae_UCG-006 was closely related to L-valine. In summary, this study described the characteristics of intestinal flora and serum metabolic profiles of lung cancer patients with different stages. It revealed that lung cancer may be the result of the mutual regulation of L-valine and Lachnospiraceae_UCG-006 through the aminoacyl-tRNA biosynthesis pathway, and proposed that L-valine may be a potential marker for the diagnosis of lung cancer.

[Source](https://doi.org/10.3389/fcimb.2022.885093)

10.3389/fcimb.2022.885093

---

## Decreased Diversity of Gut Microbiota Is Associated with Decreased Muscle Mass and Function in Older Adults Residing in a Nursing Home

[Source](https://doi.org/10.1007/s12603-022-1792-x)

---

## The role of the mucin-glycan foraging <em>Ruminococcus gnavus</em> in the communication between the gut and the brain

[Source](https://doi.org/10.1080/19490976.2022.2073784)

---

## A compilation of fecal microbiome shotgun metagenomics from hematopoietic cell transplantation patients

[Source](https://doi.org/10.1038/s41597-022-01302-9)

---

## The gut microbiota: stable bioreactor of variable composition?

[Source](https://doi.org/10.1016/j.tem.2022.04.005)

---

## Characterizing the mucin-degrading capacity of the human gut microbiota

[Source](https://doi.org/10.1038/s41598-022-11819-z)

---

## Mapping microbiome-redox spectrum and evaluating Microbial-Redox Index in chronic gastritis

[Source](https://doi.org/10.1038/s41598-022-12431-x)

---

## Endometrial microbiome: sampling, assessment, and possible impact on embryo implantation

[Source](https://doi.org/10.1038/s41598-022-12095-7)

---

## Temporal relationship among adiposity, gut microbiota, and insulin resistance in a longitudinal human cohort

[Source](https://doi.org/10.1186/s12916-022-02376-3)

---

## The respiratory microbiome associated with chronic obstructive pulmonary disease comorbidity in non-small cell lung cancer

[Source](https://doi.org/10.1111/1759-7714.14463)

---

## Cervicovaginal loads of <em>Gardnerella</em> spp. are increased in immunocompetent women with persistent high-risk human papillomavirus infection

[Source](https://doi.org/10.1099/jmm.0.001527)

---

## Dysbiotic relationship between arthritis and the oral-gut microbiome. A critical review

[Source](https://doi.org/10.1111/jre.13002)

---

## Intestinal microbiota composition of children with infection with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and multisystem inflammatory syndrome (MIS-C).
 21.05.2022, _C Suskun, O Kilic, D Yilmaz Ciftdogan, S Guven, A Karbuz, A Ozkaya Parlakay, Y Kara, E Kacmaz, A Sahin, A Boga, D Kizmaz Isancli, B Gulhan, SK Yuksek, E Kiral, G Bozan, MO Arslanoglu, MC Kizil, M Dinleyici, T Us, A Varis, M Kaya, Y Vandenplas, EC Dinleyici_


Microbiota composition may play a role in the development, prognosis, or post-infection of COVID-19. There are studies evaluating the microbiota composition at the time of diagnosis and during the course of COVID-19, especially in adults, while studies in children are limited and no study available in children with multisystem inflammatory syndrome in children (MIS-C). This study was planned to compare intestinal microbiota composition in children diagnosed with MIS-C and acute COVID-19 infection with healthy children. In this prospective multicenter study, 25 children diagnosed with MIS-C, 20 with COVID-19 infection, and 19 healthy children were included. Intestinal microbiota composition was evaluated by 16 s rRNA gene sequencing. We observed changes of diversity, richness, and composition of intestinal microbiota in MIS-C cases compared to COVID-19 cases and in the healthy controls. The Shannon index was higher in the MIS-C group than the healthy controls (p &lt; 0.01). At phylum level, in the MIS-C group, a significantly higher relative abundance of Bacteroidetes and lower abundance of Firmicutes was found compared to the control group. Intestinal microbiota composition changed in MIS-C cases compared to COVID-19 and healthy controls, and Faecalibacterium prausnitzii decreased; Bacteroides uniformis, Bacteroides plebeius, Clostridium ramosum, Eubacterium dolichum, Eggerthella lenta, Bacillus thermoamylovorans, Prevotella tannerae, and Bacteroides coprophilus were dominant in children with MIS-C. At species level, we observed decreased Faecalibacterium prausnitzii, and increased Eubacterium dolichum, Eggerthella lenta, and Bacillus thermoamylovorans in children with MIS-C and increased Bifidobacterium adolescentis and Dorea formicigenerasus in the COVID-19 group. Our study is the first to evaluate the microbiota composition in MIS-C cases. There is a substantial change in the composition of the gut microbiota: (1) reduction of F. prausnitzii in children with MIS-C and COVID-19; (2) an increase of Eggerthella lenta which is related with autoimmunity; and (3) the predominance of E. dolichum is associated with metabolic dysfunctions and obesity in children with MIS-C.

[Source](https://doi.org/10.1007/s00431-022-04494-9)

10.1007/s00431-022-04494-9

---

## Super Dominant Pathobiontic Bacteria in the Nasopharyngeal Microbiota Cause Secondary Bacterial Infection in COVID-19 Patients.
 23.05.2022, _T Qin, Y Wang, J Deng, B Xu, X Zhu, J Wang, H Zhou, N Zhao, F Jin, H Ren, H Wang, Q Li, X Xu, Y Guo, R Li, Y Xiong, X Wang, J Guo, H Zheng, X Hou, K Wan, J Zhang, J Lu, B Kan, J Xu_


Coronavirus disease 2019 (COVID-19) is a respiratory infectious disease responsible for many infections worldwide. Differences in respiratory microbiota may correlate with disease severity. Samples were collected from 20 severe and 51 mild COVID-19 patients. High-throughput sequencing of the 16S rRNA gene was used to analyze the bacterial community composition of the upper and lower respiratory tracts. The indices of diversity were analyzed. When one genus accounted for &gt;50% of reads from a sample, it was defined as a super dominant pathobiontic bacterial genus (SDPG). In the upper respiratory tract, uniformity indices were significantly higher in the mild group than in the severe group (P &lt; 0.001). In the lower respiratory tract, uniformity indices, richness indices, and the abundance-based coverage estimator were significantly higher in the mild group than in the severe group (P &lt; 0.001). In patients with severe COVID-19, SDPGs were detected in 40.7% of upper and 63.2% of lower respiratory tract samples. In patients with mild COVID-19, only 10.8% of upper and 8.5% of lower respiratory tract samples yielded SDPGs. SDPGs were present in both upper and lower tracts in seven patients (35.0%), among which six (30.0%) patients possessed the same SDPG in the upper and lower tracts. However, no patients with mild infections had an SDPG in both tracts. Staphylococcus, Corynebacterium, and Acinetobacter were the main SDPGs. The number of SDPGs identified differed significantly between patients with mild and severe COVID-19 (P &lt; 0.001). SDPGs in nasopharyngeal microbiota cause secondary bacterial infection in COVID-19 patients and aggravate pneumonia. IMPORTANCE The nasopharyngeal microbiota is composed of a variety of not only the true commensal bacterial species but also the two-face pathobionts, which are one a harmless commensal bacterial species and the other a highly invasive and deadly pathogen. In a previous study, we found that the diversity of nasopharyngeal microbiota was lost in severe influenza patients. We named the genus that accounted for over 50% of microbiota abundance as super dominant pathobiontic genus, which could invade to cause severe pneumonia, leading to high fatality. Similar phenomena were found here for SARS-CoV-2 infection. The diversity of nasopharyngeal microbiota was lost in severe COVID-19 infection patients. SDPGs in nasopharyngeal microbiota were frequently detected in severe COVID-19 patients. Therefore, the SDPGs in nasopharynx microbiota might invade into low respiratory and be responsible for secondary bacterial pneumonia in patients with SARS-CoV-2 infection.

[Source](https://doi.org/10.1128/spectrum.01956-21)

10.1128/spectrum.01956-21

---

## Increased Abundance of <em>Achromobacter xylosoxidans</em> and <em>Bacillus cereus</em> in Upper Airway Transcriptionally Active Microbiome of COVID-19 Mortality Patients Indicates Role of Co-Infections in Disease Severity and Outcome

[Source](https://doi.org/10.1128/spectrum.02311-21)

---

## In vitro anti-tuberculosis effect of probiotic <em>Lacticaseibacillus rhamnosus</em> PMC203 isolated from vaginal microbiota

[Source](https://doi.org/10.1038/s41598-022-12413-z)

---

## Commensal bacteria-derived extracellular vesicles suppress ulcerative colitis through regulating the macrophages polarization and remodeling the gut microbiota.
 20.05.2022, _L Liang, C Yang, L Liu, G Mai, H Li, L Wu, M Jin, Y Chen_


_Animals, Clostridium butyricum, Colitis, Colitis, Ulcerative, Colon, Cytokines, Dextran Sulfate, Disease Models, Animal, Extracellular Vesicles, Gastrointestinal Microbiome, Macrophages, Mice, Mice, Inbred C57BL, RNA, Ribosomal, 16S_
The extracellular vesicles (EVs) traffic constitutes an essential pathway of cellular communication. And the molecules in EVs produced by procaryotes help in maintaining homeostasis, addressing microbial imbalance and infections, and regulating the immune system. Despite the fact that Clostridium butyricum (C. butyricum) is commonly used for treating ulcerative colitis (UC), the potential role of C. butyricum-secreted EVs in commensals-host crosstalk remains unclear.

[Source](https://doi.org/10.1186/s12934-022-01812-6)

10.1186/s12934-022-01812-6

---

## Adaptation of the gut pathobiont <em>Enterococcus faecalis</em> to deoxycholate and taurocholate bile acids

[Source](https://doi.org/10.1038/s41598-022-12552-3)

---

## The OmpA of commensal <em>Escherichia coli</em> of CRC patients affects apoptosis of the HCT116 colon cancer cell line

[Source](https://doi.org/10.1186/s12866-022-02540-y)

---

## Bee foraging preferences, microbiota and pathogens revealed by direct shotgun metagenomics of honey

[Source](https://doi.org/10.1111/1755-0998.13626)

---

## Time for a change! A spotlight on the many neglected facets of sponge microbial biotechnology

[Source](https://doi.org/10.2174/1389201023666220516103715)

---

## Dietary wheat and reduced methane yield are linked to rumen microbiome changes in dairy cows

[Source](https://doi.org/10.1371/journal.pone.0268157)

---

## Role of Insect Gut Microbiota in Pesticide Degradation: A Review.
 21.05.2022, _JA Siddiqui, MM Khan, BS Bamisile, M Hafeez, M Qasim, MT Rasheed, MA Rasheed, S Ahmad, MI Shahid, Y Xu_


Insect pests cause significant agricultural and economic losses to crops worldwide due to their destructive activities. Pesticides are designed to be poisonous and are intentionally released into the environment to combat the menace caused by these noxious pests. To survive, these insects can resist toxic substances introduced by humans in the form of pesticides. According to recent findings, microbes that live in insect as symbionts have recently been found to protect their hosts against toxins. Symbioses that have been formed are between the pests and various microbes, a defensive mechanism against pathogens and pesticides. Insects' guts provide unique conditions for microbial colonization, and resident bacteria can deliver numerous benefits to their hosts. Insects vary significantly in their reliance on gut microbes for basic functions. Insect digestive tracts are very different in shape and chemical properties, which have a big impact on the structure and composition of the microbial community. Insect gut microbiota has been found to contribute to feeding, parasite and pathogen protection, immune response modulation, and pesticide breakdown. The current review will examine the roles of gut microbiota in pesticide detoxification and the mechanisms behind the development of resistance in insects to various pesticides. To better understand the detoxifying microbiota in agriculturally significant pest insects, we provided comprehensive information regarding the role of gut microbiota in the detoxification of pesticides.

[Source](https://doi.org/10.3389/fmicb.2022.870462)

10.3389/fmicb.2022.870462

---

## The <em>Brassica napus</em> seed microbiota is cultivar-specific and transmitted via paternal breeding lines

[Source](https://doi.org/10.1111/1751-7915.14077)

---

## Assembly, Core Microbiota, and Function of the Rhizosphere Soil and Bark Microbiota in <em>Eucommia ulmoides</em>

[Source](https://doi.org/10.3389/fmicb.2022.855317)

---

## Acquisition of a complex root microbiome reshapes the transcriptomes of rice plants

[Source](https://doi.org/10.1111/nph.18261)

---

## Water deficit affects inter-kingdom microbial connections in plant rhizosphere

[Source](https://doi.org/10.1111/1462-2920.16031)

---

## Facilitating Reforestation Through the Plant Microbiome: Perspectives from the Phyllosphere.
 23.05.2022, _PE Busby, G Newcombe, AS Neat, C Averill_


Tree planting and natural regeneration contribute to the ongoing effort to restore Earth's forests. Our review addresses how the plant microbiome can enhance the survival of planted and naturally regenerating seedlings and serve in long-term forest carbon capture and the conservation of biodiversity. We focus on fungal leaf endophytes, ubiquitous defensive symbionts that protect against pathogens. We first show that fungal and oomycetous pathogen richness varies greatly for tree species native to the United States (n = 0-876 known pathogens per US tree species), with nearly half of tree species either without pathogens in these major groups or with unknown pathogens. Endophytes are insurance against the poorly known and changing threat of tree pathogens. Next, we reviewed studies of plant-phyllosphere feedback, but knowledge gaps prevented us from evaluating whether adding conspecific leaf litter to planted seedlings promotes defensive symbiosis, analogous to adding soil to promote positive feedback. Finally, we discuss research priorities for integrating the plant microbiome into efforts to expand Earth's forests. Expected final online publication date for the Annual Review of Phytopathology, Volume 60 is August 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.

[Source](https://doi.org/10.1146/annurev-phyto-021320-010717)

10.1146/annurev-phyto-021320-010717

---

## The standardisation of the approach to metagenomic human gut analysis: from sample collection to microbiome profiling.
 23.05.2022, _N Szóstak, A Szymanek, J Havránek, K Tomela, M Rakoczy, A Samelak-Czajka, M Schmidt, M Figlerowicz, J Majta, K Milanowska-Zabel, L Handschuh, A Philips_


_DNA, Humans, Metagenome, Metagenomics, Microbiota, RNA, Ribosomal, 16S, Reproducibility of Results_
In recent years, the number of metagenomic studies increased significantly. Wide range of factors, including the tremendous community complexity and variability, is contributing to the challenge in reliable microbiome community profiling. Many approaches have been proposed to overcome these problems making hardly possible to compare results of different studies. The significant differences between procedures used in metagenomic research are reflected in a variation of the obtained results. This calls for the need for standardisation of the procedure, to reduce the confounding factors originating from DNA isolation, sequencing and bioinformatics analyses in order to ensure that the differences in microbiome composition are of a true biological origin. Although the best practices for metagenomics studies have been the topic of several publications and the main aim of the International Human Microbiome Standard (IHMS) project, standardisation of the procedure for generating and analysing metagenomic data is still far from being achieved. To highlight the difficulties in the standardisation of metagenomics methods, we thoroughly examined each step of the analysis of the human gut microbiome. We tested the DNA isolation procedure, preparation of NGS libraries for next-generation sequencing, and bioinformatics analysis, aimed at identifying microbial taxa. We showed that the homogenisation time is the leading factor impacting sample diversity, with the recommendation for a shorter homogenisation time (10 min). Ten minutes of homogenisation allows for better reflection of the bacteria gram-positive/gram-negative ratio, and the obtained results are the least heterogenous in terms of beta-diversity of samples microbial composition. Besides increasing the homogenisation time, we observed further potential impact of the library preparation kit on the gut microbiome profiling. Moreover, our analysis revealed that the choice of the library preparation kit influences the reproducibility of the results, which is an important factor that has to be taken into account in every experiment. In this study, a tagmentation-based kit allowed for obtaining the most reproducible results. We also considered the choice of the computational tool for determining the composition of intestinal microbiota, with Kraken2/Bracken pipeline outperforming MetaPhlAn2 in our in silico experiments. The design of an experiment and a detailed establishment of an experimental protocol may have a serious impact on determining the taxonomic profile of the intestinal microbiome community. Results of our experiment can be helpful for a wide range of studies that aim to better understand the role of the gut microbiome, as well as for clinical purposes.

[Source](https://doi.org/10.1038/s41598-022-12037-3)

10.1038/s41598-022-12037-3

---

## Locality-Sensitive Hashing-Based k-Mer Clustering for Identification of Differential Microbial Markers Related to Host Phenotype.
 18.05.2022, _W Han, H Tang, Y Ye_


Microbial organisms play important roles in many aspects of human health and diseases. Encouraged by the numerous studies that show the association between microbiomes and human diseases, computational and machine learning methods have been recently developed to generate and utilize microbiome features for prediction of host phenotypes such as disease versus healthy cancer immunotherapy responder versus nonresponder. We have previously developed a subtractive assembly approach, which focuses on extraction and assembly of differential reads from metagenomic data sets that are likely sampled from differential genomes or genes between two groups of microbiome data sets (e.g., healthy vs. disease). In this article, we further improved our subtractive assembly approach by utilizing groups of k-mers with similar abundance profiles across multiple samples. We implemented a locality-sensitive hashing (LSH)-enabled approach (called kmerLSHSA) to group billions of k-mers into k-mer coabundance groups (kCAGs), which were subsequently used for the retrieval of differential kCAGs for subtractive assembly. Testing of the kmerLSHSA approach on simulated data sets and real microbiome data sets showed that, compared with the conventional approach that utilizes all genes, our approach can quickly identify differential genes that can be used for building promising predictive models for microbiome-based host phenotype prediction. We also discussed other potential applications of LSH-enabled clustering of k-mers according to their abundance profiles across multiple microbiome samples.

[Source](https://doi.org/10.1089/cmb.2021.0640)

10.1089/cmb.2021.0640

---

