## Analysis of the diversity of intestinal microbiome and its potential value as a biomarker in patients with schizophrenia: A cohort study

[Source](https://www.sciencedirect.com/science/article/pii/S0165178120307769)

---

## A longitudinal study of the development of the saliva microbiome in infants 2 days to 5 years compared to the microbiome in adolescents.
 28.06.2020, _P Lif Holgerson, A Esberg, A Sjödin, CE West, I Johansson_


Understanding oral microbiota programming attracts increasing interest due to its importance for oral health and potential associations with systemic diseases. Here the oral microbiota was longitudinally characterized in children from 2 days (n = 206) to 5 years of age and in young adults (n = 175) by sequencing of the v3-v4 region of the 16S rRNA gene from saliva extracted DNA. Alpha diversity increased by age, with 2-day- and 3-month-old infants in one sub-group, and 18-month- and 3-year-old children in another. Firmicutes decreased up to 3 years of age, whereas Proteobacteria, Actinobacteria, Bacteroidetes and Fusobacteria abundances increased. Abiotrophia, Actinomyces, Capnocytophaga, Corynebacterium, Fusobacterium, Kingella, Leptotrichia, Neisseria and Porphyromonas appeared from 18-months of age. This was paralleled by expansions in the core microbiome that continued up to adulthood. The age-related microbiota transformation was paralleled by functional alterations, e.g., changed metabolic pathways that reflected e.g., breastfeeding and increasing proportions of anaerobic species. Oral microbiotas differed by feeding mode and weakly by mode of delivery, but not gender, pacifier use or cleaning method or probiotic intake. The study shows that the saliva microbiota is diverse 2 days after birth and under transformation up to 5 years of age and beyond, with fluctuations possibly reflecting age-related environmental influences.

[Source](https://www.nature.com/articles/s41598-020-66658-7)

10.1038/s41598-020-66658-7

---

## Longitudinal analysis reveals transition barriers between dominant ecological states in the gut microbiome.
 26.06.2020, _R Levy, AT Magis, JC Earls, O Manor, T Wilmanski, J Lovejoy, SM Gibbons, GS Omenn, L Hood, ND Price_


The Pioneer 100 Wellness Project involved quantitatively profiling 108 participants' molecular physiology over time, including genomes, gut microbiomes, blood metabolomes, blood proteomes, clinical chemistries, and data from wearable devices. Here, we present a longitudinal analysis focused specifically around the Pioneer 100 gut microbiomes. We distinguished a subpopulation of individuals with reduced gut diversity, elevated relative abundance of the genus Prevotella, and reduced levels of the genus Bacteroides We found that the relative abundances of Bacteroides and Prevotella were significantly correlated with certain serum metabolites, including omega-6 fatty acids. Primary dimensions in distance-based redundancy analysis of clinical chemistries explained 18.5% of the variance in bacterial community composition, and revealed a Bacteroides/Prevotella dichotomy aligned with inflammation and dietary markers. Finally, longitudinal analysis of gut microbiome dynamics within individuals showed that direct transitions between Bacteroides-dominated and Prevotella-dominated communities were rare, suggesting the presence of a barrier between these states. One implication is that interventions seeking to transition between Bacteroides- and Prevotella-dominated communities will need to identify permissible paths through ecological state-space that circumvent this apparent barrier.

[Source](https://www.pnas.org/content/117/24/13839)

10.1073/pnas.1922498117

---

## Genome-wide associations of human gut microbiome variation and implications for causal inference analyses.
 27.06.2020, _DA Hughes, R Bacigalupe, J Wang, MC Rühlemann, RY Tito, G Falony, M Joossens, S Vieira-Silva, L Henckaerts, L Rymenans, C Verspecht, S Ring, A Franke, KH Wade, NJ Timpson, J Raes_


Recent population-based1-4 and clinical studies5 have identified a range of factors associated with human gut microbiome variation. Murine quantitative trait loci6, human twin studies7 and microbiome genome-wide association studies1,3,8-12 have provided evidence for genetic contributions to microbiome composition. Despite this, there is still poor overlap in genetic association across human studies. Using appropriate taxon-specific models, along with support from independent cohorts, we show an association between human host genotype and gut microbiome variation. We also suggest that interpretation of applied analyses using genetic associations is complicated by the probable overlap between genetic contributions and heritable components of host environment. Using faecal 16S ribosomal RNA gene sequences and host genotype data from the Flemish Gut Flora Project (n = 2,223) and two German cohorts (FoCus, n = 950; PopGen, n = 717), we identify genetic associations involving multiple microbial traits. Two of these associations achieved a study-level threshold of P = 1.57 × 10-10; an association between Ruminococcus and rs150018970 near RAPGEF1 on chromosome 9, and between Coprococcus and rs561177583 within LINC01787 on chromosome 1. Exploratory analyses were undertaken using 11 other genome-wide associations with strong evidence for association (P &lt; 2.5 × 10-8) and a previously reported signal of association between rs4988235 (MCM6/LCT) and Bifidobacterium. Across these 14 single-nucleotide polymorphisms there was evidence of signal overlap with other genome-wide association studies, including those for age at menarche and cardiometabolic traits. Mendelian randomization analysis was able to estimate associations between microbial traits and disease (including Bifidobacterium and body composition); however, in the absence of clear microbiome-driven effects, caution is needed in interpretation. Overall, this work marks a growing catalogue of genetic associations that will provide insight into the contribution of host genotype to gut microbiome. Despite this, the uncertain origin of association signals will likely complicate future work looking to dissect function or use associations for causal inference analysis.

[Source](https://www.nature.com/articles/s41564-020-0743-8)

10.1038/s41564-020-0743-8

---

## Microarray Analysis of Gene Expression Involved in Butyrate-Resistant Colorectal Carcinoma HCT116 Cells.
 29.06.2020, _C Khonthun, N Saikachain, S Popluechai, K Kespechara, A Hiranyakas, M Srikummool, D Surangkul_


Resistance to chemotherapeutic agents is usually found in cancer stem cells (CSCs) and cancer stem-like cells that are often regarded as the target for cancer monitoring. However, the different patterns of their transcriptomic profiling is still unclear.

[Source](https://www.thelancet.com/journals/lanonc/article/PIIS1470-2045(20)30351-X/fulltext)

10.31557/APJCP.2020.21.6.1739

---

## Cancer and the microbiome

[Source](https://www.thelancet.com/journals/lanonc/article/PIIS1470-2045(20)30351-X/fulltext)

---

## Influences of Ingredients and Bakers on the Bacteria and Fungi in Sourdough Starters and Bread.
 23.03.2020, _AT Reese, AA Madden, M Joossens, G Lacaze, RR Dunn_


Sourdough starters are naturally occurring microbial communities in which the environment, ingredients, and bakers are potential sources of microorganisms. The relative importance of these pools remains unknown. Here, bakers from two continents used a standardized recipe and ingredients to make starters that were then baked into breads. We characterized the fungi and bacteria associated with the starters, bakers' hands, and ingredients using 16S and internal transcribed spacer (ITS) rRNA gene amplicon sequencing and then measured dough acidity and bread flavor. Starter communities were much less uniform than expected, and this variation manifested in the flavor of the bread. Starter communities were most similar to those found in flour but shared some species with the bakers' skin. While humans likely contribute microorganisms to the starters, the reverse also appears to be true. This bidirectional exchange of microorganisms between starters and bakers highlights the importance of microbial diversity on bodies and in our environments as it relates to foods.IMPORTANCE Sourdough starters are complex communities of yeast and bacteria which confer characteristic flavor and texture to sourdough bread. The microbes present in starters can be sourced from ingredients or the baking environment and are typically consistent over time. Herein, we show that even when the recipe and ingredients for starter and bread are identical, different bakers around the globe produce highly diverse starters which then alter bread acidity and flavor. Much of the starter microbial community comes from bread flour, but the diversity is also associated with differences in the microbial community on the hands of bakers. These results indicate that bakers may be a source for yeast and bacteria in their breads and/or that bakers' jobs are reflected in their skin microbiome.

[Source](https://msphere.asm.org/content/5/1/e00950-19#:~:text=The%20bacterial%20and%20fungal%20species,bread%20(1%E2%80%933).&text=Canonically%2C%20one%20or%20several%20Lactobacillus,the%20yeasts%20produce%20carbon%20dioxide.)

10.1128/mSphere.00950-19

---

## Effects of zinc pollution and compost amendment on the root microbiome of a metal tolerant poplar clone

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.01677/abstract)

---

## New insights into the distributions of nitrogen fixation and diazotrophs revealed by high-resolution sensing and sampling methods.
 25.06.2020, _W Tang, E Cerdán-García, H Berthelot, D Polyviou, S Wang, A Baylay, H Whitby, H Planquette, M Mowlem, J Robidart, N Cassar_


Nitrogen availability limits marine productivity across large ocean regions. Diazotrophs can supply new nitrogen to the marine environment via nitrogen (N2) fixation, relieving nitrogen limitation. The distributions of diazotrophs and N2 fixation have been hypothesized to be generally controlled by temperature, phosphorus, and iron availability in the global ocean. However, even in the North Atlantic where most research on diazotrophs and N2 fixation has taken place, environmental controls remain contentious. Here we measure diazotroph composition, abundance, and activity at high resolution using newly developed underway sampling and sensing techniques. We capture a diazotrophic community shift from Trichodesmium to UCYN-A between the oligotrophic, warm (25-29 °C) Sargasso Sea and relatively nutrient-enriched, cold (13-24 °C) subpolar and eastern American coastal waters. Meanwhile, N2 fixation rates measured in this study are among the highest ever recorded globally and show significant increase with phosphorus availability across the transition from the Gulf Stream into subpolar and coastal waters despite colder temperatures and higher nitrate concentrations. Transcriptional patterns in both Trichodesmium and UCYN-A indicate phosphorus stress in the subtropical gyre. Over this iron-replete transect spanning the western North Atlantic, our results suggest that temperature is the major factor controlling the diazotrophic community structure while phosphorous drives N2 fixation rates. Overall, the occurrence of record-high UCYN-A abundance and peak N2 fixation rates in the cold coastal region where nitrate concentrations are highest (~200 nM) challenges current paradigms on what drives the distribution of diazotrophs and N2 fixation.

[Source](https://www.nature.com/articles/s41396-020-0703-6)

10.1038/s41396-020-0703-6

---

## Language of plant-microbe-microbe interactions in rhizospheric ecosystems

[Source](https://doi.org/10.1016/B978-0-12-818469-1.00005-5)

---

## Editorial: Microbial Ecology in Reservoirs and Lakes

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.01348/full)

---

## Natural Holobiome Engineering by Using Native Extreme Microbiome to Counteract the Climate Change Effects

[Source](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7287022/)

---

