## <span style="color:#ff6600;">The Recovery, Interpretation and Use of Ancient Pathogen Genomes</span>

[Source](https://doi.org/10.1016/j.cub.2020.08.081)

---

## Functional diversity of microbial ecologies estimated from ancient human coprolites and dental calculus.
 05.10.2020, _DK Jacobson, TP Honap, C Monroe, J Lund, BA Houk, AC Novotny, C Robin, E Marini, CM Lewis_


Human microbiome studies are increasingly incorporating macroecological approaches, such as community assembly, network analysis and functional redundancy to more fully characterize the microbiome. Such analyses have not been applied to ancient human microbiomes, preventing insights into human microbiome evolution. We address this issue by analysing published ancient microbiome datasets: coprolites from Rio Zape (n = 7; 700 CE Mexico) and historic dental calculus (n = 44; 1770-1855 CE, UK), as well as two novel dental calculus datasets: Maya (n = 7; 170 BCE-885 CE, Belize) and Nuragic Sardinians (n = 11; 1400-850 BCE, Italy). Periodontitis-associated bacteria (Treponema denticola, Fusobacterium nucleatum and Eubacterium saphenum) were identified as keystone taxa in the dental calculus datasets. Coprolite keystone taxa included known short-chain fatty acid producers (Eubacterium biforme, Phascolarctobacterium succinatutens) and potentially disease-associated bacteria (Escherichia, Brachyspira). Overlap in ecological profiles between ancient and modern microbiomes was indicated by similarity in functional response diversity profiles between contemporary hunter-gatherers and ancient coprolites, as well as parallels between ancient Maya, historic UK, and modern Spanish dental calculus; however, the ancient Nuragic dental calculus shows a distinct ecological structure. We detected key ecological signatures from ancient microbiome data, paving the way to expand understanding of human microbiome evolution. This article is part of the theme issue 'Insights into health and disease from ancient biomolecules'.

[Source](http://dx.doi.org/10.1098/rstb.2019.0586)

10.1098/rstb.2019.0586

---

## Investigating the demographic history of Japan using ancient oral microbiota.
 05.10.2020, _R Eisenhofer, H Kanzawa-Kiriyama, KI Shinoda, LS Weyrich_


While microbial communities in the human body (microbiota) are now commonly associated with health and disease in industrialised populations, we know very little about how these communities co-evolved and changed with humans throughout history and deep prehistory. We can now examine these communities by sequencing ancient DNA preserved within calcified dental plaque (calculus), providing insights into the origins of disease and their links to human history. Here, we examine ancient DNA preserved within dental calculus samples and their associations with two major cultural periods in Japan: the Jomon period hunter-gatherers approximately 3000 years before present (BP) and the Edo period agriculturalists 400-150 BP. We investigate how human oral microbiomes have changed in Japan through time and explore the presence of microorganisms associated with oral diseases (e.g. periodontal disease, dental caries) in ancient Japanese populations. Finally, we explore oral microbial strain diversity and its potential links to ancient demography in ancient Japan by performing phylogenomic analysis of a widely conserved oral species-Anaerolineaceae oral taxon 439. This research represents, to our knowledge, the first study of ancient oral microbiomes from Japan and demonstrates that the analysis of ancient dental calculus can provide key information about the origin of non-infectious disease and its deep roots with human demography. This article is part of the theme issue 'Insights into health and disease from ancient biomolecules'.

[Source](https://doi.org/10.1098/rstb.2019.0578)

10.1098/rstb.2019.0578

---

## Distinct human gut microbial taxonomic signatures uncovered with different sample processing and microbial cell disruption methods for metaproteomic analysis

[Source](https://doi.org/10.1101/2020.10.08.331066)

---

## Deciphering diet-gut microbiota-host interplay: Investigations of pectin

[Source](https://doi.org/10.1016/j.tifs.2020.10.010)

---

## Microbes and Environment: Global Warming Reverting the Frozen Zombies

[Source](https://doi.org/10.1007/978-3-030-49732-3_24)

---

## Taxonomic Diversity of Bacteria and Their Filterable Forms in the Soils of Eastern Antarctica (Larsemann Hills and Bunger Hills)

[Source](https://doi.org/10.1134/S0026261720050136)

---

## Long-term warming effects on the microbiome and nitrogen fixation associated with the moss Racomitrium lanuginosum in Icelandic tundra

[Source](https://doi.org/10.1101/838581)

---

## Diversity of Eukaryotic Microorganisms in the Drainage Waters of a Coal Open-Cast Mine

[Source](https://doi.org/10.1134/S0026261720050100)

---

## Dissimilarity of microbial diversity of pond water, shrimp intestine and sediment in Aquamimicry system

[Source](https://doi.org/10.1186/s13568-020-01119-y)

---

## Comparative diversity analysis of halophiles at two polar saltern systems in Indramayu, West Java, Indonesia

[Source](https://doi.org/10.1111/lam.13401)

---

## Mechanistic model of nutrient uptake explains dichotomy between marine oligotrophic and copiotrophic bacteria

[Source](https://doi.org/10.1101/2020.10.08.331785)

---

## Microbial diversity of seafood

[Source](https://doi.org/10.1016/j.cofs.2020.09.005)

---

## The first head‐tailed virus, MFTV1, produced by a hyperthermophilic methanogenic deep‐sea archaea

[Source](https://doi.org/10.1111/1462-2920.15271)

---

## Single-virus genomics and beyond

[Source](https://doi.org/10.1038/s41579-020-00444-0)

---

## A Rapid Automated Process for Organizing Bacterial Cluster Segments Using Deep Neural Networks

[Source](https://ieeexplore.ieee.org/abstract/document/9214173)

---

## FACS-iChip: a high-efficiency iChip system for microbial ‘dark matter’ mining

[Source](https://doi.org/10.1007/s42995-020-00067-7)

---

## Comparison of Nucleic Acid Extraction Methods for a Viral Metagenomics Analysis of Respiratory Viruses

[Source](https://doi.org/10.3390/microorganisms8101539)

---

## Advancing science or advancing careers? Researchers’ opinions on success indicators

[Source](https://doi.org/10.1101/2020.06.22.165654)

---

## <span style="color:#ff6600;">Ten simple rules for annotating sequencing experiments</span>

[Source](https://doi.org/10.1371/journal.pcbi.1008260)

---

