## Dysbiosis of a microbiota-immune metasystem in critical illness is associated with nosocomial infections.
 09.03.2023, _J Schlechte, AZ Zucoloto, IL Yu, CJ Doig, MJ Dunbar, KD McCoy, B McDonald_


Critically ill patients in intensive care units experience profound alterations of their gut microbiota that have been linked to a high risk of hospital-acquired (nosocomial) infections and adverse outcomes through unclear mechanisms. Abundant mouse and limited human data suggest that the gut microbiota can contribute to maintenance of systemic immune homeostasis, and that intestinal dysbiosis may lead to defects in immune defense against infections. Here we use integrated systems-level analyses of fecal microbiota dynamics in rectal swabs and single-cell profiling of systemic immune and inflammatory responses in a prospective longitudinal cohort study of critically ill patients to show that the gut microbiota and systemic immunity function as an integrated metasystem, where intestinal dysbiosis is coupled to impaired host defense and increased frequency of nosocomial infections. Longitudinal microbiota analysis by 16s rRNA gene sequencing of rectal swabs and single-cell profiling of blood using mass cytometry revealed that microbiota and immune dynamics during acute critical illness were highly interconnected and dominated by Enterobacteriaceae enrichment, dysregulated myeloid cell responses and amplified systemic inflammation, with a lesser impact on adaptive mechanisms of host defense. Intestinal Enterobacteriaceae enrichment was coupled with impaired innate antimicrobial effector responses, including hypofunctional and immature neutrophils and was associated with an increased risk of infections by various bacterial and fungal pathogens. Collectively, our findings suggest that dysbiosis of an interconnected metasystem between the gut microbiota and systemic immune response may drive impaired host defense and susceptibility to nosocomial infections in critical illness.

[Source](https://doi.org/10.1038/s41591-023-02243-5)

10.1038/s41591-023-02243-5

---

## Altered infective competence of the human gut microbiome in COVID-19 

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01472-7)

---

## Impact of high altitude on composition and functional profiling of oral microbiome in Indian male population.
 13.03.2023, _M Kumari, B Bhushan, MR Eslavath, AK Srivastava, RC Meena, R Varshney, L Ganju_


The oral cavity of human contains bacteria that are critical for maintaining the homeostasis of the body. External stressors such as high altitude (HA) and low oxygen affect the human gut, skin and oral microbiome. However, compared to the human gut and skin microbiome, studies demonstrating the impact of altitude on human oral microbiota are currently scarce. Alterations in the oral microbiome have been reported to be associated with various periodontal diseases. In light of the increased occurrence of HA oral health related problems, the effect of HA on the oral salivary microbiome was investigated. We conducted a pilot study in 16 male subjects at two different heights i.e., H1 (210 m) and H2 (4420 m). Total of 31 saliva samples,16 at H1 and 15 at H2 were analyzed by utilizing the 16S rRNA high-throughput sequencing, to explore the relationship between the HA environment and salivary microbiota. The preliminary results suggesting that, the most abundant microbiome at the phylum level are: Firmicutes, Bacteroidetes, Proteobacteria, and Actinobacteria. Interestingly, 11 genera were identified at the both heights with different relative abundances. In addition, the salivary microbiome was more diverse at H1 compared to H2 as demonstrated by decreased alpha diversity. Further, predicted functional results indicate that microbial metabolic profiles significantly decreased at H2 as compared to H1, including two major metabolic pathways involving carbohydrates, and amino acids. Our findings show that HA induces shifts in the composition and structure of human oral microbiota which can affect host health homeostasis.

[Source](https://doi.org/10.1038/s41598-023-30963-8)

10.1038/s41598-023-30963-8

---

## Incidence of Breast Cancer and <em>Enterococcus</em> Infection: A Retrospective Analysis

[Source](https://doi.org/10.14740/wjon1551)

---

## Multi-omics studies in interpreting the evolving standard model for immune functions.
 11.03.2023, _D Ganguly_


A standard model that is able to generalize data on myriad involvement of the immune system in organismal physio-pathology and to provide a unified evolutionary teleology for immune functions in multicellular organisms remains elusive. A number of such 'general theories of immunity' have been proposed based on contemporaneously available data, starting with the usual description of self-nonself discrimination, followed by the 'danger model' and the more recent 'discontinuity theory.' More recent data deluge on involvement of immune mechanisms in a wide variety of clinical contexts, a number of which fail to get readily accommodated into the available teleologic standard models, makes deriving a standard model of immunity more challenging. But technological advances enabling multi-omics investigations into an ongoing immune response, covering genome, epigenome, coding and regulatory transcriptome, proteome, metabolome and tissue-resident microbiome, bring newer opportunities for developing a more integrative insight into immunocellular mechanisms within different clinical contexts. The new ability to map the heterogeneity of composition, trajectory and endpoints of immune responses, in both health and disease, also necessitates incorporation into the potential standard model of immune functions, which again can only be achieved through multi-omics probing of immune responses and integrated analyses of the multi-dimensional data.

[Source](https://doi.org/10.1093/bfgp/elad003)

10.1093/bfgp/elad003

---

## Understanding the bacterial compositional network associations between oral and gut microbiome within healthy Koreans.
 10.03.2023, _J Jeong, K Ahn, S Mun, K Yun, YT Kim, W Jung, KE Lee, MY Kim, Y Ahn, K Han_


Oral microbial ecosystem could influence intestinal diseases, but there have been insufficient studies demonstrating the association of microbial composition between the oral cavity and the intestinal system. Thus, we aimed to investigate the compositional network within the oral microbiome related to gut enterotype from saliva and stool samples collected from 112 healthy Korean subjects. Here, we performed bacterial 16S amplicon sequencing from clinical samples. Then, we determined oral microbiome type related to individual's gut enterotype for healthy Korean. The co-occurrence analysis was performed to interactivity prediction of microbiome within saliva samples. As a result, it could be classified into two Korean oral microbiome types (KO) and four oral-gut-associated microbiome types (KOGA) according to distribution and significant differences of oral microflora. The co-occurrence analysis showed various bacterial compositional networks linked around Streptococcus and Haemophilus within healthy subjects. The present study was first approach in healthy Koreans to identify the oral microbiome types related to the gut microbiome and investigate their characteristics. Hence, we suggest that our results could be potential healthy control data for identifying differences in microbial composition between healthy people and oral disease patients and studying microbial association with the gut microbial environment (oral-gut microbiome axis).

[Source](https://doi.org/10.1080/20002297.2023.2186591)

10.1080/20002297.2023.2186591

---

## A transcriptomic insight into the human sperm microbiome through next-generation sequencing.
 10.03.2023, _C Corral-Vazquez, J Blanco, R Aiese Cigliano, S Zaida, F Vidal, E Anton_


The purpose of this study is to provide novel information through Next Generation Sequencing (NGS) for the characterization of viral and bacterial RNA cargo of human sperm cells from healthy fertile donors. For this, RNA-seq raw data of poly(A) RNA from 12 sperm samples from fertile donors were aligned to microbiome databases using the GAIA software. Species of viruses and bacteria were quantified in Operational Taxonomic Units (OTU) and filtered by minimal expression level (&gt;1% OTU in at least one sample). Mean expression values (and their standard deviation) of each species were estimated. A Hierarchical Cluster Analysis (HCA) and a Principal Component Analysis (PCA) were performed to detect common microbiome patterns among samples. Sixteen microbiome species, families, domains, and orders surpassed the established expression threshold. Of the 16 categories, nine corresponded to viruses (23.07% OTU) and seven to bacteria (2.77% OTU), among which the Herperviriales order and Escherichia coli were the most abundant, respectively. HCA and PCA displayed four clusters of samples with a differentiated microbiome fingerprint. This work represents a pilot study into the viruses and bacteria that make up the human sperm microbiome. Despite the high variability observed, some patterns of similarity among individuals were identified. Further NGS studies under standardized methodological procedures are necessary to achieve a deep knowledge of the semen microbiome and its implications in male fertility.

[Source](https://doi.org/10.1080/19396368.2023.2183912)

10.1080/19396368.2023.2183912

---

## Metformin alleviates liver fibrosis in mice by enriching <em>Lactobacillus</em> sp. MF-1 in the gut microbiota

[Source](https://doi.org/10.1016/j.bbadis.2023.166664)

---

## Microbiota alters the metabolome in an age- and sex- dependent manner in mice

[Source](https://doi.org/10.1038/s41467-023-37055-1)

---

## IL-22 alters gut microbiota composition and function to increase aryl hydrocarbon receptor activity in mice and humans.
 13.03.2023, _JS Mar, N Ota, ND Pokorzynski, Y Peng, A Jaochico, D Sangaraju, E Skippington, AN Lekkerkerker, ME Rothenberg, MW Tan, T Yi, ME Keir_


_Humans, Animals, Mice, Gastrointestinal Microbiome, Receptors, Aryl Hydrocarbon, Interleukins, Indoles_
IL-22 is induced by aryl hydrocarbon receptor (AhR) signaling and plays a critical role in gastrointestinal barrier function through effects on antimicrobial protein production, mucus secretion, and epithelial cell differentiation and proliferation, giving it the potential to modulate the microbiome through these direct and indirect effects. Furthermore, the microbiome can in turn influence IL-22 production through the synthesis of L-tryptophan (L-Trp)-derived AhR ligands, creating the prospect of a host-microbiome feedback loop. We evaluated the impact IL-22 may have on the gut microbiome and its ability to activate host AhR signaling by observing changes in gut microbiome composition, function, and AhR ligand production following exogenous IL-22 treatment in both mice and humans.

[Source](https://doi.org/10.1186/s40168-023-01486-1)

10.1186/s40168-023-01486-1

---

## Decreased Paneth cell α-defensins promote fibrosis in a choline-deficient L-amino acid-defined high-fat diet-induced mouse model of nonalcoholic steatohepatitis via disrupting intestinal microbiota

[Source](https://doi.org/10.1038/s41598-023-30997-y)

---

## Microbiota and functional analyses of nitrogen-fixing bacteria in root-knot nematode parasitism of plants.
 13.03.2023, _Y Li, S Lei, Z Cheng, L Jin, T Zhang, LM Liang, L Cheng, Q Zhang, X Xu, C Lan, C Lu, M Mo, KQ Zhang, J Xu, B Tian_


_Animals, Nitrogen-Fixing Bacteria, Plant Diseases, Nematoda, Plants, Microbiota, Solanum lycopersicum, Bacteria, Nitrogen, Plant Roots_
Root-knot nematodes (RKN) are among the most important root-damaging plant-parasitic nematodes, causing severe crop losses worldwide. The plant rhizosphere and root endosphere contain rich and diverse bacterial communities. However, little is known about how RKN and root bacteria interact to impact parasitism and plant health. Determining the keystone microbial taxa and their functional contributions to plant health and RKN development is important for understanding RKN parasitism and developing efficient biological control strategies in agriculture.

[Source](https://doi.org/10.1186/s40168-023-01484-3)

10.1186/s40168-023-01484-3

---

## Non-volatile metabolites mediating plant interactions with insect herbivores.
 09.03.2023, _M Luo, B Li, G Jander, S Zhou_


Non-volatile metabolites constitute the bulk of plant biomass. From the perspective of plant-insect interactions, these structurally diverse compounds including nutritious core metabolites and defensive specialized metabolites. In this review, we synthesize current literature on multiple scales of plant-insect interactions mediated by non-volatile metabolites. At the molecular level, functional genetics studies have revealed a large collection of receptors targeting plant non-volatile metabolites in model insect species and agricultural pests. By contrast, examples of plant receptors of insect-derived molecules remain sparse. For insect herbivores, plant non-volatile metabolites function beyond the dichotomy of core metabolite being nutrients and specialized metabolites being defensive compounds. Insect feeding tends to elicit evolutionarily conserved regulation of plant specialized metabolism, whereas its effect on plant core metabolism varies widely based the interacting species. Finally, several recent studies have demonstrated that non-volatile metabolites can mediate tripartite communication on the community scale, facilitated by physical connections established by direct root-to-root communication, parasitic plants, arbuscular mycorrhizae, and the rhizosphere microbiome. Recent advances in both plant and insect molecular biology, will facilitate further research on the role of non-volatile metabolites in mediating plant-insect interactions.

[Source](https://doi.org/10.1111/tpj.16180)

10.1111/tpj.16180

---

## Comparative metagenomic analysis reveals rhizosphere microbial community composition and functions help protect grapevines against salt stress.
 10.03.2023, _B Wang, X Wang, Z Wang, K Zhu, W Wu_


Soil salinization is a serious abiotic stress for grapevines. The rhizosphere microbiota of plants can help counter the negative effects caused by salt stress, but the distinction between rhizosphere microbes of salt-tolerant and salt-sensitive varieties remains unclear.

[Source](https://doi.org/10.3389/fmicb.2023.1102547)

10.3389/fmicb.2023.1102547

---

## Emerging Pathways for Engineering the Rhizosphere Microbiome for Optimal Plant Health.
 09.03.2023, _S Yang, H Liu, P Xie, T Wen, Q Shen, J Yuan_


The increasing impacts of global climate change on crop performance pose a significant threat to global food security. The rhizosphere microbiomes intimately interact with the plant and can largely facilitate plants in growth promotion and stress resistance via multiple mechanisms. This review focuses on approaches for harnessing the rhizosphere microbiomes to produce beneficial effects toward enhanced crop productivity, including the use of organic and inorganic amendments, and microbial inoculants. Emerging methods, such as the utilization of synthetic microbial consortia, host-mediated microbiome engineering, prebiotics made from specific plant root exudates, and crop breeding to promote beneficial plant-microbiome interactions, are highlighted. Updating our knowledge in this field is critical for understanding and improving plant-microbiome interactions, thereby enhancing plant adaptiveness to changing environmental conditions.

[Source](https://doi.org/10.1021/acs.jafc.2c08758)

10.1021/acs.jafc.2c08758

---

## A bayesian zero-inflated dirichlet-multinomial regression model for multivariate compositional count data.
 10.03.2023, _MD Koslovsky_


The Dirichlet-multinomial (DM) distribution plays a fundamental role in modern statistical methodology development and application. Recently, the DM distribution and its variants have been used extensively to model multivariate count data generated by high-throughput sequencing technology in omics research due to its ability to accommodate the compositional structure of the data as well as overdispersion. A major limitation of the DM distribution is that it is unable to handle excess zeros typically found in practice which may bias inference. To fill this gap, we propose a novel Bayesian zero-inflated DM model for multivariate compositional count data with excess zeros. We then extend our approach to regression settings and embed sparsity-inducing priors to perform variable selection for high-dimensional covariate spaces. Throughout, modeling decisions are made to boost scalability without sacrificing interpretability or imposing limiting assumptions. Extensive simulations and an application to a human gut microbiome data set are presented to compare the performance of the proposed method to existing approaches. We provide an accompanying R package with a user-friendly vignette to apply our method to other data sets. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1111/biom.13853)

10.1111/biom.13853

---

## Determining the most accurate 16S rRNA hypervariable region for taxonomic identification from respiratory samples.
 13.03.2023, _R López-Aladid, L Fernández-Barat, V Alcaraz-Serrano, L Bueno-Freire, N Vázquez, R Pastor-Ibáñez, A Palomeque, P Oscanoa, A Torres_


_Humans, RNA, Ribosomal, 16S, Sequence Analysis, DNA, Bacteria, Microbiota, Respiratory System, High-Throughput Nucleotide Sequencing_
16S rRNA gene profiling, which contains nine hypervariable regions (V1-V9), is the gold standard for identifying taxonomic units by high-throughput sequencing. Microbiome studies combine two or more region sequences (usually V3-V4) to increase the resolving power for identifying bacterial taxa. We compare the resolving powers of V1-V2, V3-V4, V5-V7, and V7-V9 to improve microbiome analyses in sputum samples from patients with chronic respiratory diseases. DNA were isolated from 33 human sputum samples, and libraries were created using a QIASeq screening panel intended for Illumina platforms (16S/ITS; Qiagen Hilden, Germany). The analysis included a mock community as a microbial standard control (ZymoBIOMICS). We used the Deblur algorithm to identify bacterial amplicon sequence variants (ASVs) at the genus level. Alpha diversity was significantly higher for V1-V2, V3-V4, and V5-V7 compared with V7-V9, and significant compositional dissimilarities in the V1-V2 and V7-V9 analyses versus the V3-V4 and V5-V7 analyses. A cladogram confirmed these compositional differences, with the latter two being very similar in composition. The combined hypervariable regions showed significant differences when discriminating between the relative abundances of bacterial genera. The area under the curve revealed that V1-V2 had the highest resolving power for accurately identifying respiratory bacterial taxa from sputum samples. Our study confirms that 16S rRNA hypervariable regions provide significant differences for taxonomic identification in sputum. Comparing the taxa of microbial community standard control with the taxa samples, V1-V2 combination exhibits the most sensitivity and specificity. Thus, while third generation full-length 16S rRNA sequencing platforms become more available, the V1-V2 hypervariable regions can be used for taxonomic identification in sputum.

[Source](https://doi.org/10.1038/s41598-023-30764-z)

10.1038/s41598-023-30764-z

---

