## <em>Aggregatibacter</em> is inversely associated with inflammatory mediators in sputa of patients with chronic airway diseases and reduces inflammation <em>in vitro</em>

[Source](https://doi.org/10.1186/s12931-024-02983-z)

---

## Oral Microbiota Development in the First 60 Months: A Longitudinal Study.
 12.10.2024, _K Yama, S Morishima, K Tsutsumi, R Jo, Y Aita, T Inokuchi, T Okuda, D Watai, K Ohara, M Maruyama, T Chikazawa, T Iwamoto, Y Kakizawa, T Oniki_


Childhood is considered crucial in the establishment of future oral microbiota. However, the precise period of oral microbiota development remains unclear. This study aimed to identify the progression of oral microbiota formation in children. We longitudinally investigated the salivary microbiota of 54 children across 13 time points from 1 wk to 60 mo (5 y) old and their parents at 2 time points as a representative sample of the adult microbiota. Using next-generation sequencing, we obtained 10,000 gene sequences of the 16s rRNA V1-V2 region for each sample. The detection rate in children of 110 operational taxonomic units commonly detected in more than 85% of mothers and fathers, defined as the main constituent bacteria, was 25% at 1 wk old, increased to 80% between 6 and 18 mo old, and reached approximately 90% by 36 mo old. Early main constituent bacteria detected at 1 wk old were limited to Streptococcus, Rothia, and Gemella. At 6 to 18 mo old, the detection rates of various main constituent bacteria, including Neisseria, Haemophilus, and Fusobacterium, increased. UniFrac distance analysis showed that the oral microbiota of children approached that of adults at 6 to 18 mo old. In the weighted UniFrac distance index, unlike the unweighted index, there were no significant changes in children between 36 and 60 mo old from adults, and microbiota formation at 60 mo old was sufficiently advanced to be included within the range of adult individual differences. Our findings suggest that the initial 36 mo, particularly the period from 6 to 18 mo old, consists of a time window for oral microbiota maturation. In addition, the development of microbiota during this period may be critical for future oral disease prevention.

[Source](https://doi.org/10.1177/00220345241272011)

10.1177/00220345241272011

---

## Microbiome-derived metabolites in early to mid-pregnancy and risk of gestational diabetes: a metabolome-wide association study.
 13.10.2024, _SM Susarla, O Fiehn, I Thiele, AL Ngo, DK Barupal, RF Chehab, A Ferrara, Y Zhu_


_Humans, Female, Diabetes, Gestational, Pregnancy, Adult, Prospective Studies, Metabolome, Case-Control Studies, Microbiota, Metabolomics_
Pre-diagnostic disturbances in the microbiome-derived metabolome have been associated with an increased risk of diabetes in non-pregnant populations. However, the roles of microbiome-derived metabolites, the end-products of microbial metabolism, in gestational diabetes (GDM) remain understudied. We examined the prospective association of microbiome-derived metabolites in early to mid-pregnancy with GDM risk in a diverse population.

[Source](https://doi.org/10.1186/s12916-024-03606-6)

10.1186/s12916-024-03606-6

---

## Unique Gut Microbiome and Metabolic Profiles in Chinese Workers Exposed to Dust: Insights from a Case-Control Study

[Source](https://doi.org/10.1097/JOM.0000000000003243)

---

## Dysbiotic signatures and diagnostic potential of gut microbial markers for inflammatory bowel disease in Korean population

[Source](https://doi.org/10.1038/s41598-024-74002-6)

---

## A search for bacteria identified from cerebrospinal fluid shunt infections in previous surgical events

[Source](https://doi.org/10.1371/journal.pone.0311605)

---

## Microbiome and metabolome patterns after lung transplantation reflect underlying disease and chronic lung allograft dysfunction

[Source](https://doi.org/10.1186/s40168-024-01893-y)

---

## Reexamining the role of <em>Fusobacterium nucleatum</em> subspecies in clinical and experimental studies

[Source](https://doi.org/10.1080/19490976.2024.2415490)

---

## Host-microbe interaction paradigms in acute and recurrent vulvovaginal candidiasis

[Source](https://doi.org/10.1016/j.chom.2024.08.018)

---

## Balancing act: counteracting adverse drug effects on the microbiome

[Source](https://doi.org/10.1016/j.tim.2024.09.011)

---

## Gut microbiome and bladder cancer: A new link through nitrosamine metabolism

[Source](https://doi.org/10.1016/j.chom.2024.09.003)

---

## Prostate cancer microenvironment: multidimensional regulation of immune cells, vascular system, stromal cells, and microbiota

[Source](https://doi.org/10.1186/s12943-024-02137-1)

---

## Gut and oral microbiota in gynecological cancers: interaction, mechanism, and therapeutic value

[Source](https://doi.org/10.1038/s41522-024-00577-7)

---

## Effects of complex probiotics on intestinal function and its regulatory mechanism in patients with constipation

[Source](https://doi.org/10.1163/18762891-bja00039)

---

## <em>Lactiplantibacillus plantarum</em> P101 Alleviated Alcohol-Induced Hepatic Lipid Accumulation in Mice via AMPK Signaling Pathway: Gut Microbiota and Metabolomics Analysis

[Source](https://doi.org/10.1007/s12602-024-10373-6)

---

## Unlocking the genetic influence on milk variation and its potential implication for infant health

[Source](https://doi.org/10.1016/j.xgen.2024.100676)

---

## The gut microbiome and dietary fibres: implications in obesity, cardiometabolic diseases and cancer

[Source](https://doi.org/10.1038/s41579-024-01108-z)

---

## Microbiome-based therapeutics for Parkinson’s disease

[Source](https://doi.org/10.1016/j.neurot.2024.e00462)

---

## Gut bacteria of lepidopteran herbivores facilitate digestion of plant toxins

[Source](https://doi.org/10.1073/pnas.2412165121)

---

## Sea cucumbers and their symbiotic microbiome have evolved to feed on seabed sediments.
 11.10.2024, _W Pan, X Wang, C Ren, X Jiang, S Gong, Z Xie, NK Wong, X Li, J Huang, D Fan, P Luo, Y Yang, X Ren, S Yu, Z Qin, X Wu, D Huo, B Ma, Y Liu, X Zhang, Z E, J Liang, H Sun, L Yuan, X Liu, C Cheng, H Long, J Li, Y Wang, C Hu, T Chen_


_Animals, Symbiosis, Geologic Sediments, Sea Cucumbers, Glycoside Hydrolases, Gastrointestinal Microbiome, Holothuria, Phylogeny, Biological Evolution, Ecosystem, Feeding Behavior, Microbiota, Bacteria_
Sea cucumbers are predominant deposit feeders in benthic ecosystems, providing protective benefits to coral reefs by reducing disease prevalence. However, how they receive sufficient nutrition from seabed sediments remains poorly understood. Here, we investigate Holothuria leucospilota, an ecologically significant tropical sea cucumber, to elucidate digestive mechanisms underlying marine deposit-feeding. Genomic analysis reveals intriguing evolutionary adaptation characterized by an expansion of digestive carbohydrase genes and a contraction of digestive protease genes, suggesting specialization in digesting microalgae. Developmentally, two pivotal dietary shifts, namely, from endogenous nutrition to planktonic feeding, and from planktonic feeding to deposit feeding, induce changes in digestive tract enzyme profiles, with adults mainly expressing carbohydrases and lipases. A nuanced symbiotic relationship exists between gut microbiota and the host, namely, specific resident bacteria supply crucial enzymes for food digestion, while other bacteria are digested and provide assimilable nutrients. Our study further identifies Holothuroidea lineage-specific lysozymes that are restrictedly expressed in the intestines to support bacterial digestion. Overall, this work advances our knowledge of the evolutionary innovations in the sea cucumber digestive system which enable them to efficiently utilize nutrients from seabed sediments and promote food recycling within marine ecosystems.

[Source](https://doi.org/10.1038/s41467-024-53205-5)

10.1038/s41467-024-53205-5

---

## Comprehensive profile of the companion animal gut microbiome integrating reference-based and reference-free methods

[Source](https://doi.org/10.1093/ismejo/wrae201)

---

## Quantitative Synthesis of Microbe-Driven Acclimation and Adaptation in Wild Vertebrates.
 12.10.2024, _G Martin Bideguren, O Razgour, A Alberdi_


Microorganisms associated with animals harbour a unique set of functional traits pivotal for the normal functioning of their hosts. This realisation has led researchers to hypothesise that animal-associated microbial communities may boost the capacity of their hosts to acclimatise and adapt to environmental changes, two eco-evolutionary processes with significant applied relevance. Aiming to assess the importance of microorganisms for wild vertebrate conservation, we conducted a quantitative systematic review to evaluate the scientific evidence for the contribution of gut microorganisms to the acclimation and adaptation capacity of wild vertebrate hosts. After screening 1974 publications, we scrutinised the 109 studies that met the inclusion criteria based on 10 metrics encompassing study design, methodology and reproducibility. We found that the studies published so far were not able to resolve the contribution of gut microorganisms due to insufficient study design and research methods for addressing the hypothesis. Our findings underscore the limited application to date of microbiome knowledge in vertebrate conservation and management, highlighting the need for a paradigm shift in research approaches. Considering these results, we advocate for a shift from observational studies to experimental manipulations, where fitness or related indicators are measured, coupled with an update in molecular techniques used to analyse microbial functions. In addition, closer collaboration with conservation managers and practitioners from the inception of the project is needed to encourage meaningful application of microbiome knowledge in adaptive wildlife conservation management.

[Source](https://doi.org/10.1111/eva.70025)

10.1111/eva.70025

---

## The role of foraging pollinators in assembling the flower microbiota and transmitting the fire blight pathogen <em>Erwinia amylovora</em>

[Source](https://doi.org/10.1111/1462-2920.16702)

---

## Distinct microbial communities supported by iron oxidation.
 10.10.2024, _K Burkartová, A Hlaváček, S Skoblia, L Falteisek_


_Iron, Oxidation-Reduction, Bacteria, Hydrogen-Ion Concentration, Biodiversity, Microbiota, Ecosystem, Mining, Phylogeny, Archaea_
Microbial biostalactites and streamers commonly grow at iron seepages in abandoned mines worldwide. This study addresses the diversity and composition of these simple prokaryotic communities, which thrive in pH ranges from 2.4 to 6.6 across six different mines. Our analysis of 85 communities reveals that a pH of approximately 3.2 is a critical threshold where alpha and beta diversity change discretely. Below this pH, the average number of ASVs per sample is 2.91 times lower than above this boundary. Autotrophs, heterotrophs, and symbionts of eukaryotes originate from nearly non-overlapping species pools in the two habitat types that differ only in pH. Communities below pH 3.2 further divide into two distinct groups, differing in diversity, taxonomic, and functional composition. Both types of communities coexist within the same stalactites, likely corresponding to zones where the capillary structure of the stalactite is either perfused or clogged. These findings indicate that microbial community structure can be significantly influenced by the intricate spatial organization of the ecosystem, rather than solely by measurable environmental parameters.

[Source](https://doi.org/10.1111/1462-2920.16706)

10.1111/1462-2920.16706

---

## Normalization of short-chain fatty acid concentration by bacterial count of stool samples improves discrimination between eubiotic and dysbiotic gut microbiota caused by <em>Clostridioides difficile </em>infection

[Source](https://doi.org/10.1080/19490976.2024.2415488)

---

## SmT/SHM-seq: simultaneously capturing spatial transcriptome and microbiome information in plants.
 12.10.2024, _P Mu, W Li, LP Tran, X Li_



[Source](https://doi.org/10.1016/j.tplants.2024.09.010)

10.1016/j.tplants.2024.09.010

---

## High accuracy meets high throughput for near full-length 16S ribosomal RNA amplicon sequencing on the Nanopore platform

[Source](https://doi.org/10.1093/pnasnexus/pgae411)

---

## A deployable film method to enable replicable sampling of low-abundance environmental microbiomes.
 11.10.2024, _P Mankiewicz Ledins, EZ Lin, C Bhattacharya, KJG Pollitt, AH Dyson, EM Hénaff_


_Microbiota, Humans, Specimen Handling, Dimethylpolysiloxanes, DNA, Bacterial, Environmental Monitoring, Environmental Microbiology_
Urbanizing global populations spend over 90% of their time indoors where microbiome abundance and diversity are low. Chronic exposure to microbiomes with low abundance and diversity have demonstrated negative long-term impacts on human health. Sequencing-based analyses of environmental nucleic acids are critical to understanding the impact of the indoor microbiome on human health, however low DNA yields indoors, alongside sample collection and processing inconsistencies, currently challenge study replicability. This study presents a comparative assessment of a novel, passive, easily replicable sampling strategy using polydimethylsiloxane (PDMS) sheets alongside a representative swab-based collection protocol. Deployable, customizable PDMS films designed for whole-sample insertion into standardized extraction kits demonstrated 43% higher DNA yields per sample, and 76% higher yields per cm2 of sampler over swab-based protocols. These results indicate that this accessible, scalable method enables sufficient DNA collection to comprehensively evaluate indoor microbiome exposures and potential human health impacts using smaller, more space efficient samplers, representing an attractive alternative to swab-based collection. In addition, this process reduces the manual steps required for microbiome sampling which could address inter-study variability, transform the current microbiome sampling paradigm, and ultimately benefit the replicability and accessibility of microbiome exposure studies.

[Source](https://doi.org/10.1038/s41598-024-72341-y)

10.1038/s41598-024-72341-y

---

