## A microbiota-modulated checkpoint directs immunosuppressive intestinal T cells into cancers.
 12.06.2023, _M Fidelle, C Rauber, C Alves Costa Silva, AL Tian, I Lahmar, AM de La Varende, L Zhao, C Thelemaque, I Lebhar, M Messaoudene, E Pizzato, R Birebent, MD Mbogning Fonkou, S Zoppi, A Reni, C Dalban, M Leduc, G Ferrere, S Durand, P Ly, A Silvin, K Mulder, CA Dutertre, F Ginhoux, S Yonekura, MP Roberti, M Tidjani-Alou, S Terrisse, J Chen, O Kepp, A Schippers, N Wagner, J Suárez-Gosálvez, S Kobold, JE Fahrner, C Richard, J Bosq, L Lordello, G Vitali, N Galleron, B Quinquis, E Le Chatelier, L Blanchard, JP Girard, A Jarry, N Gervois, E Godefroy, N Labarrière, R Koschny, R Daillère, B Besse, C Truntzer, F Ghiringhelli, N Coatnoan, V Mhanna, D Klatzmann, D Drubay, L Albiges, AM Thomas, N Segata, FX Danlos, A Marabelle, B Routy, L Derosa, G Kroemer, L Zitvogel_


_Humans, Immunoglobulins, Integrins, Immunosuppressive Agents, Neoplasms_
Antibiotics (ABX) compromise the efficacy of programmed cell death protein 1 (PD-1) blockade in cancer patients, but the mechanisms underlying their immunosuppressive effects remain unknown. By inducing the down-regulation of mucosal addressin cell adhesion molecule 1 (MAdCAM-1) in the ileum, post-ABX gut recolonization by Enterocloster species drove the emigration of enterotropic α4β7+CD4+ regulatory T 17 cells into the tumor. These deleterious ABX effects were mimicked by oral gavage of Enterocloster species, by genetic deficiency, or by antibody-mediated neutralization of MAdCAM-1 and its receptor, α4β7 integrin. By contrast, fecal microbiota transplantation or interleukin-17A neutralization prevented ABX-induced immunosuppression. In independent lung, kidney, and bladder cancer patient cohorts, low serum levels of soluble MAdCAM-1 had a negative prognostic impact. Thus, the MAdCAM-1-α4β7 axis constitutes an actionable gut immune checkpoint in cancer immunosurveillance.

[Source](https://doi.org/10.1126/science.abo2296)

10.1126/science.abo2296

---

## Using gut microbiota as a diagnostic tool for colorectal cancer: machine learning techniques reveal promising results.
 09.06.2023, _F Lu, T Lei, J Zhou, H Liang, P Cui, T Zuo, L Ye, H Chen, J Huang_


_Humans, Gastrointestinal Microbiome, Colorectal Neoplasms, RNA, Ribosomal, 16S, Bayes Theorem, Feces, Escherichia coli, Machine Learning, Prevotella_
Introduction. Increasing evidence suggests a correlation between gut microbiota and colorectal cancer (CRC).Hypothesis/Gap Statement. However, few studies have used gut microbiota as a diagnostic biomarker for CRC.Aim. The objective of this study was to explore whether a machine learning (ML) model based on gut microbiota could be used to diagnose CRC and identify key biomarkers in the model.Methodology. We sequenced the 16S rRNA gene from faecal samples of 38 participants, including 17 healthy subjects and 21 CRC patients. Eight supervised ML algorithms were used to diagnose CRC based on faecal microbiota operational taxonomic units (OTUs), and the models were evaluated in terms of identification, calibration and clinical practicality for optimal modelling parameters. Finally, the key gut microbiota was identified using the random forest (RF) algorithm.Results. We found that CRC was associated with the dysregulation of gut microbiota. Through a comprehensive evaluation of supervised ML algorithms, we found that different algorithms had significantly different prediction performance using faecal microbiomes. Different data screening methods played an important role in optimization of the prediction models. We found that naïve Bayes algorithms [NB, accuracy=0.917, area under the curve (AUC)=0.926], RF (accuracy=0.750, AUC=0.926) and logistic regression (LR, accuracy=0.750, AUC=0.889) had high predictive potential for CRC. Furthermore, important features in the model, namely s__metagenome_g__Lachnospiraceae_ND3007_group (AUC=0.814), s__Escherichia_coli_g__Escherichia-Shigella (AUC=0.784) and s__unclassified_g__Prevotella (AUC=0.750), could each be used as diagnostic biomarkers of CRC.Conclusions. Our results suggested an association between gut microbiota dysregulation and CRC, and demonstrated the feasibility of the gut microbiota to diagnose cancer. The bacteria s__metagenome_g__Lachnospiraceae_ND3007_group, s__Escherichia_coli_g__Escherichia-Shigella and s__unclassified_g__Prevotella were key biomarkers for CRC.

[Source](https://doi.org/10.1099/jmm.0.001699)

10.1099/jmm.0.001699

---

## Outer membrane utilisomes mediate glycan uptake in gut Bacteroidetes

[Source](https://doi.org/10.1038/s41586-023-06146-w)

---

## Skin maturation from birth to 10 years of age: Structure, function, composition and microbiome.
 11.06.2023, _GN Stamatas, PF Roux, E Boireau-Adamezyk, I Lboukili, T Oddos_


Infant and adult skin physiology differ in many ways; however, limited data exist for older children. To further investigate the maturation processes of healthy skin during childhood. Skin parameters were recorded in 80 participants of four age groups: babies (0-2 years), young children (3-6 years), older children (7-&lt;10 years) and adults (25-40 years). Overall, skin barrier function continues to mature, reaching adult levels of transepidermal water loss (TEWL), lipid compactness, stratum corneum (SC) thickness and corneocyte size by the age of about 6 years. Higher levels of lactic acid and lower levels of total amino acids in the SC of babies and young children further indicate higher cell turnover rates. In all age groups, TEWL and skin surface hydration values remain higher on the face compared with the arm. Skin becomes darker and contains higher levels of melanin with increasing age. The composition of skin microbiome of the dorsal forearm in all children groups is distinct from that in adults, with Firmicutes predominating in the former and Proteobacteria in the latter. Skin physiology, along with the skin microbiome, continues to mature during early childhood in a site-specific manner.

[Source](https://doi.org/10.1111/exd.14843)

10.1111/exd.14843

---

## Ecological principles of fecal microbiota transplantation.
 10.06.2023, _F Dini-Andreote, GF Custer_


Fecal microbiota transplantation (FMT) is a procedure used to treat diseases related to microbiome imbalances. We discuss how ecological principles can inform the design of FMT clinical trials and contribute to data interpretation. This effort will promote a better understanding of microbiome engraftment and assist the development of clinical protocols.

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(23)00162-2)

10.1016/j.tim.2023.05.009

---

## Human Genetics Influences Microbiome Composition Involved in Asthma Exacerbations despite Inhaled Corticosteroid Treatment.
 10.06.2023, _J Perez-Garcia, A Espuela-Ortiz, JM Hernández-Pérez, R González-Pérez, P Poza-Guedes, E Martin-Gonzalez, C Eng, O Sardón-Prado, E Mederos-Luis, P Corcuera-Elosegui, I Sánchez-Machín, J Korta-Murua, J Villar, EG Burchard, F Lorenzo-Diaz, M Pino-Yanes_


The upper-airway microbiome is involved in asthma exacerbations despite inhaled corticosteroid (ICS) treatment. Although human genetics regulates microbiome composition, its influence on asthma-related airway bacteria remains unknown.

[Source](https://www.jacionline.org/article/S0091-6749(23)00748-0/pdf)

10.1016/j.jaci.2023.05.021

---

## Gut bacterial metabolism contributes to host global purine homeostasis.
 06.06.2023, _K Kasahara, RL Kerby, Q Zhang, M Pradhan, M Mehrabian, AJ Lusis, G Bergström, F Bäckhed, FE Rey_


The microbes and microbial pathways that influence host inflammatory disease progression remain largely undefined. Here, we show that variation in atherosclerosis burden is partially driven by gut microbiota and is associated with circulating levels of uric acid (UA) in mice and humans. We identify gut bacterial taxa spanning multiple phyla, including Bacillota, Fusobacteriota, and Pseudomonadota, that use multiple purines, including UA as carbon and energy sources anaerobically. We identify a gene cluster that encodes key steps of anaerobic purine degradation and that is widely distributed among gut-dwelling bacteria. Furthermore, we show that colonization of gnotobiotic mice with purine-degrading bacteria modulates levels of UA and other purines in the gut and systemically. Thus, gut microbes are important drivers of host global purine homeostasis and serum UA levels, and gut bacterial catabolism of purines may represent a mechanism by which gut bacteria influence health.

[Source](https://doi.org/10.1016/j.chom.2023.05.011)

10.1016/j.chom.2023.05.011

---

## The relative abundance of fecal bacterial species belonging to the Firmicutes and Bacteroidetes phyla is related to plasma levels of bile acids in young adults

[Source](https://doi.org/10.1007/s11306-023-02016-8)

---

## The effect of vitamin D supplementation on the gut microbiome in older Australians - Results from analyses of the D-Health Trial.
 11.06.2023, _H Pham, M Waterhouse, S Rahman, C Baxter, B Duarte Romero, DSA McLeod, PR Ebeling, DR English, G Hartel, RL O'Connell, JC van der Pols, AJ Venn, PM Webb, DC Whiteman, F Huygens, RE Neale_


_Aged, Female, Humans, Male, Australia, Bacteroidetes, Dietary Supplements, Double-Blind Method, Firmicutes, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Vitamin D, Aged, 80 and over_
Observational studies suggest a link between vitamin D and the composition of the gut microbiome, but there is little evidence from randomized controlled trials of vitamin D supplementation. We analyzed data from the D-Health Trial, a randomized, double-blind, placebo-controlled trial. We recruited 21,315 Australians aged 60-84 y and randomized them to 60,000 IU of vitamin D3 or placebo monthly for 5 y. Stool samples were collected from a sample of 835 participants (417 in the placebo and 418 in the vitamin D group) approximately 5 y after randomization. We characterized the gut microbiome using 16S rRNA gene sequencing. We used linear regression to compare alpha diversity indices (i.e. Shannon index (primary outcome), richness, inverse Simpson index), and the ratio of Firmicutes to Bacteroidetes between the two groups. We analyzed between-sample (beta) diversity (i.e. Bray Curtis distance and UniFrac index) using principal coordinate analysis and used PERMANOVA to test for significant clustering according to randomization group. We also assessed the difference in the abundance of the 20 most abundant genera between the two groups using negative binomial regression model with adjustment for multiple testing. Approximately half the participants included in this analysis were women (mean age 69.4 y). Vitamin D supplementation did not alter the Shannon diversity index (mean 3.51 versus 3.52 in the placebo and vitamin D groups, respectively, p = 0.50). Similarly, there was little difference between the groups for other alpha diversity indices, the abundance of different genera, and the Firmicutes-to-Bacteroidetes ratio. We did not observe clustering of bacterial communities according to randomization group. In conlusion, monthly doses of 60,000 IU of vitamin D supplementation for 5 y did not alter the composition of the gut microbiome in older Australians.

[Source](https://doi.org/10.1080/19490976.2023.2221429)

10.1080/19490976.2023.2221429

---

## Impaired flux of bile acids from the liver to the gut reveals microbiome-immune interactions associated with liver damage

[Source](https://doi.org/10.1038/s41522-023-00398-0)

---

## Gut microbiota dysbiosis promotes the development of epithelial ovarian cancer via regulating Hedgehog signaling pathway.
 10.06.2023, _X Hu, X Xu, X Zeng, R Jin, S Wang, H Jiang, Y Tang, G Chen, J Wei, T Chen, Q Chen_


_Humans, Female, Animals, Mice, Carcinoma, Ovarian Epithelial, Hedgehog Proteins, Ovarian Neoplasms, NF-kappa B, Gastrointestinal Microbiome, Dysbiosis, Cell Line, Tumor, Signal Transduction, Inflammation_
Epithelial ovarian cancer (EOC) is the most lethal gynecological cancer, which remains a threat to female health at all ages. Hypotheses for EOC development include the continuous presence of inflammation, in which microbiota and inflammatory cytokines participate in cancer-related signaling pathway activation. Hedgehog (Hh) signaling is prominent for EOC progression, and interacts with inflammation response related to gut microbiota (GM). However, the precise roles of GM during this process are unknown. Here, we showed that the GM from patients with EOC differed from that of healthy women and had GM dysbiosis. We found that EOC modeling may lead to GM changes in mice, and it restored after the administration of GM from healthy controls, while GM from patients with EOC further exacerbated GM dysbiosis. Furthermore, we found that GM from EOC markedly promoted tumor progression and activated Hh signaling; meanwhile, it increased the extent of inflammation and activated NF-κB signaling, but GM from healthy controls improved them. Our results demonstrate how GM dysbiosis promoted EOC progression by activating Hh signaling mediated by TLR4/NF-κB signaling. We anticipate our assay to be a new thought for exploring the role of GM in EOC development. Furthermore, improving GM dysbiosis is a novel therapeutic approach for delaying EOC development.

[Source](https://doi.org/10.1080/19490976.2023.2221093)

10.1080/19490976.2023.2221093

---

## Microbes with higher metabolic independence are enriched in human gut microbiomes under stress.
 11.06.2023, _I Veseli, YT Chen, MS Schechter, C Vanni, EC Fogarty, AR Watson, B Jabri, R Blekhman, AD Willis, MK Yu, A Fernàndez-Guerra, J Füssel, AM Eren_


A wide variety of human diseases are associated with loss of microbial diversity in the human gut, inspiring a great interest in the diagnostic or therapeutic potential of the microbiota. However, the ecological forces that drive diversity reduction in disease states remain unclear, rendering it difficult to ascertain the role of the microbiota in disease emergence or severity. One hypothesis to explain this phenomenon is that microbial diversity is diminished as disease states select for microbial populations that are more fit to survive environmental stress caused by inflammation or other host factors. Here, we tested this hypothesis on a large scale, by developing a software framework to quantify the enrichment of microbial metabolisms in complex metagenomes as a function of microbial diversity. We applied this framework to over 400 gut metagenomes from individuals who are healthy or diagnosed with inflammatory bowel disease (IBD). We found that high metabolic independence (HMI) is a distinguishing characteristic of microbial communities associated with individuals diagnosed with IBD. A classifier we trained using the normalized copy numbers of 33 HMI-associated metabolic modules not only distinguished states of health versus IBD, but also tracked the recovery of the gut microbiome following antibiotic treatment, suggesting that HMI is a hallmark of microbial communities in stressed gut environments.

[Source](https://doi.org/10.1101/2023.05.10.540289)

10.1101/2023.05.10.540289

---

## Evolutionary implications of host genetic control for engineering beneficial microbiomes.
 10.06.2023, _LP Henry, J Bergelson_


Engineering new functions in the microbiome requires understanding how host genetic control and microbe-microbe interactions shape the microbiome. One key genetic mechanism underlying host control is the immune system. The immune system can promote stability in the composition of the microbiome by reshaping the ecological dynamics of its members, but the degree of stability will depend on the interplay between ecological context, immune system development, and higher-order microbe-microbe interactions. The eco-evolutionary interplay affecting composition and stability should inform the strategies used to engineer new functions in the microbiome. We conclude with recent methodological developments that provide an important path forward for both engineering new functionality in the microbiome and broadly understanding how ecological interactions shape evolutionary processes in complex biological systems.

[Source](https://doi.org/10.1016/j.coisb.2023.100455)

10.1016/j.coisb.2023.100455

---

## It's all relative: analyzing microbiome compositions, its significance, pathogenesis and microbiota derived biofilms: Challenges and opportunities for disease intervention.
 08.06.2023, _S Zaidi, K Ali, AU Khan_


_Humans, Bacteria, Biofilms, Dysbiosis, Gastrointestinal Microbiome, Microbiota, Infant, Newborn_
Concept of microorganisms has largely been perceived from their pathogenic view point. Nevertheless, it is being gradually revisited in terms of its significance to human health and now appears to be the most dominant force that shapes the immune system of the human body and also determines an individual's predisposition to diseases. Human inhabits bacterial diversity (which is predominant among all microbial communities in human body) occupying 0.3% of body mass, known as microbiota. On birth, a part of microbiota that child obtains is essentially a mother's legacy. So, the review was initiated with this critical topic of microbiotal inheritance. Since, each body site has distinct physiological specifications; therefore, they contain discrete microbiome composition that has been separately discussed along with dysbiosis-induced pathologies originating in different body organs. Factors affecting microbiome composition and may cause dysbiosis like antibiotics, delivery, feeding method etc. as well as the strategies that immune system adopts to prevent dysbiosis have been highlighted. We also tried to bring into attention the topic of dysbiosis induced biofilms, that enables cohort to survive stresses, evolve, disseminate and infection resurgence that is still in dormancy. Eventually, we put spotlight on microbiome significance in medical therapeutics. We didn't merely confine article to gut microbiota, that is being studied more extensively. Numerous community forms at diverse body sites are inter-related, and being exposed to awfully variable perturbations appear to be challenging to evaluate perturbation risks holistically. All aspects have been elaborately discussed to achieve a global depiction of human microbiota in order to meet urgent necessity for protocol standardisation. Demonstrates that environmental challenges (antibiotic use, alterations in diet, stress, smoking etc.) might cause dysbiosis i.e. transition of healthy microbiome composition to the one in which pathogenic microorganisms become more abundant, and eventually results in an infected state.

[Source](https://doi.org/10.1007/s00203-023-03589-7)

10.1007/s00203-023-03589-7

---

## Gut microbiome helps honeybee (<em>Apis mellifera</em>) resist the stress of toxic nectar plant (<em>Bidens pilosa</em>) exposure: Evidence for survival and immunity

[Source](https://doi.org/10.1111/1462-2920.16436)

---

## Regulation of host gene expression by gastrointestinal tract microbiota in Chinook Salmon (<em>Oncorhynchus tshawytscha</em>)

[Source](https://doi.org/10.1111/mec.17039)

---

## The impacts of ocean acidification, warming and their interactive effects on coral prokaryotic symbionts

[Source](https://doi.org/10.1186/s40793-023-00505-w)

---

## Association analysis of gut microbiota-metabolites-neuroendocrine changes in male rats acute exposure to simulated altitude of 5500&nbsp;m

[Source](https://doi.org/10.1038/s41598-023-35573-y)

---

## Microbiomes of Blood-Feeding Triatomines in the Context of Their Predatory Relatives and the Environment

[Source](https://doi.org/10.1128/spectrum.01681-23)

---

## Probiotic interventions promote metabolic health in high fat-fed hamsters in association with gut microbiota and endocannabinoidome alterations

[Source](https://doi.org/10.3920/BM2022.0080)

---

## Strains of <em>Faecalibacterium prausnitzii </em>and its extracts reduce blood glucose levels, percent HbA1c, and improve glucose tolerance without causing hypoglycemic side effects in diabetic and prediabetic mice

[Source](https://doi.org/10.1136/bmjdrc-2022-003101)

---

## Effects of a low FODMAP diet on gut microbiota in individuals with treated celiac disease having persistent gastrointestinal symptoms – a randomized controlled trial

[Source](https://doi.org/10.1017/S0007114523001253)

---

## Designed α-sheet peptides disrupt uropathogenic <em>E. coli</em> biofilms rendering bacteria susceptible to antibiotics and immune cells

[Source](https://doi.org/10.1038/s41598-023-36343-6)

---

## ‘Multi-omics’ data integration: applications in probiotics studies

[Source](https://doi.org/10.1038/s41538-023-00199-x)

---

## Micro-/bio-/nano-/syn-encapsulations and co-treatments of bioactive microbial feed supplementation in augmenting finfish health and aquaculture nutrition: a review

[Source](https://doi.org/10.3920/BM2022.0087)

---

## Taxonomic analysis reveals host preference of rare fungi in endophytes of <em>Vitis vinifera</em> from the Czech Republic

[Source](https://doi.org/10.1007/s12223-023-01066-8)

---

## Metaproteomics reveals functional partitioning and vegetational variation among permafrost-affected Arctic soil bacterial communities.
 05.06.2023, _SE Miller, AS Colman, JR Waldbauer_


Microbial activity in Arctic soils controls the cycling of significant stores of organic carbon and nutrients. We studied in situ processes in Alaskan soils using original metaproteomic methods in order to relate important heterotrophic functions to microbial taxa and to understand the microbial response to Arctic greening. Major bacterial groups show strong metabolic specialization in organic topsoils. α-/β-/γ-Proteobacteria specialized in the acquisition of small, soluble compounds, whereas Acidobacteria, Actinobacteria, and other detritosphere groups specialized in the degradation of plant-derived polymers. α-/β-/γ-Proteobacteria dominated the expression of transporters for common root exudates and limiting nitrogenous compounds, supporting an ecological model of dependence upon plants for carbon and competition with plants for nitrogen. Detritosphere groups specialized in distinct substrates, with Acidobacteria producing the most enzymes for hemicellulose depolymerization. Acidobacteria was the most active group across the three plant ecotypes sampled-the largely nonvascular, lower biomass intertussock and the largely vascular, higher biomass tussock and shrub. Functional partitioning among bacterial groups was stable between plant ecotypes, but certain functions associated with α-/β-/γ-Proteobacteria were more strongly expressed in higher biomass ecotypes. We show that refined metaproteomic approaches can elucidate soil microbial ecology as well as biogeochemical trajectories of major carbon stocks.IMPORTANCEThe Arctic is warming twice as fast as the rest of the planet, and Arctic soils currently store twice as much carbon as the entire atmosphere-two facts that make understanding how Arctic soil microbial communities are responding to climate change particularly urgent. Greening of vegetation cover across the Arctic landscape is one of the most prominent climate-driven shifts in Arctic terrestrial ecology, with potentially profound effects on biogeochemical cycling by the soil microbiome. Here we use metaproteomics to document microbial metabolic functions that drive soil carbon and nutrient cycling processes in an Arctic tundra landscape. We identify functional roles among bacterial taxonomic groups that are largely stable across vegetation types, with certain functions strongly expressed by rhizosphere groups reflecting a community metabolic response to greening.

[Source](https://doi.org/10.1128/msystems.01238-22)

10.1128/msystems.01238-22

---

## A new approach of microbiome monitoring in the built environment: feasibility analysis of condensation capture.
 12.06.2023, _JT Hampton-Marcell, A Ghosh, MJ Gukeh, CM Megaridis_


_Humans, RNA, Ribosomal, 16S, Feasibility Studies, Microbiota, Environmental Microbiology, Aerosols, Air Microbiology_
Humans emit approximately 30 million microbial cells per hour into their immediate vicinity. However, sampling of aerosolized microbial taxa (aerobiome) remains largely uncharacterized due to the complexity and limitations of sampling techniques, which are highly susceptible to low biomass and rapid sample degradation. Recently, there has been an interest in developing technology that collects naturally occurring water from the atmosphere, even within the built environment. Here, we analyze the feasibility of indoor aerosol condensation collection as a method to capture and analyze the aerobiome.

[Source](https://doi.org/10.1186/s40168-023-01555-5)

10.1186/s40168-023-01555-5

---

## A bioinformatics tool for identifying intratumoral microbes from the ORIEN dataset.
 11.06.2023, _C Wang, A Ma, ME McNutt, R Hoyd, CE Wheeler, LA Robinson, CHF Chan, Y Zakharia, RD Dodd, CM Ulrich, S Hardikar, ML Churchman, AA Tarhini, EA Singer, AP Ikeguchi, MD McCarter, N Denko, G Tinoco, M Husain, N Jin, AEG Osman, I Eljilany, AC Tan, SS Coleman, L Denko, G Riedlinger, BP Schneider, D Spakowicz, Q Ma_


Evidence supports significant interactions among microbes, immune cells, and tumor cells in at least 10-20% of human cancers, emphasizing the importance of further investigating these complex relationships. However, the implications and significance of tumor-related microbes remain largely unknown. Studies have demonstrated the critical roles of host microbes in cancer prevention and treatment responses. Understanding interactions between host microbes and cancer can drive cancer diagnosis and microbial therapeutics (bugs as drugs). Computational identification of cancer-specific microbes and their associations is still challenging due to the high dimensionality and high sparsity of intratumoral microbiome data, which requires large datasets containing sufficient event observations to identify relationships, and the interactions within microbial communities, the heterogeneity in microbial composition, and other confounding effects that can lead to spurious associations. To solve these issues, we present a bioinformatics tool, MEGA, to identify the microbes most strongly associated with 12 cancer types. We demonstrate its utility on a dataset from a consortium of 9 cancer centers in the Oncology Research Information Exchange Network (ORIEN). This package has 3 unique features: species-sample relations are represented in a heterogeneous graph and learned by a graph attention network; it incorporates metabolic and phylogenetic information to reflect intricate relationships within microbial communities; and it provides multiple functionalities for association interpretations and visualizations. We analyzed 2704 tumor RNA-seq samples and MEGA interpreted the tissue-resident microbial signatures of each of 12 cancer types. MEGA can effectively identify cancer-associated microbial signatures and refine their interactions with tumors.

[Source](https://doi.org/10.1101/2023.05.24.541982)

10.1101/2023.05.24.541982

---

## DeepBiomarker2: Prediction of alcohol and substance use disorder risk in post-traumatic stress disorder patients using electronic medical records and multiple social determinants of health.
 11.06.2023, _O Miranda, P Fan, X Qi, H Wang, MD Brannock, T Kosten, ND Ryan, L Kirisci, L Wang_


Prediction of high-risk events amongst patients with mental disorders is critical for personalized interventions. In our previous study, we developed a deep learning-based model, DeepBiomarker by utilizing electronic medical records (EMR) to predict the outcomes of patients with suicide-related events in post-traumatic stress disorder (PTSD) patients. Methods We improved our deep learning model to develop DeepBiomarker2 through data integration of multimodal information: lab tests, medication use, diagnosis, and social determinants of health (SDoH) parameters (both individual and neighborhood level) from EMR data for outcome prediction. We further refined our contribution analysis for identifying key factors. We applied DeepBiomarker2 to analyze EMR data of 38,807 patients from University of Pittsburgh Medical Center diagnosed with PTSD to determine their risk of developing alcohol and substance use disorder (ASUD). Results DeepBiomarker2 predicted whether a PTSD patient will have a diagnosis of ASUD within the following 3 months with a c-statistic (receiver operating characteristic AUC) of 0·93. We used contribution analysis technology to identify key lab tests, medication use and diagnosis for ASUD prediction. These identified factors imply that the regulation of the energy metabolism, blood circulation, inflammation, and microbiome is involved in shaping the pathophysiological pathways promoting ASUD risks in PTSD patients. Our study found protective medications such as oxybutynin, magnesium oxide, clindamycin, cetirizine, montelukast and venlafaxine all have a potential to reduce risk of ASUDs. Discussion DeepBiomarker2 can predict ASUD risk with high accuracy and can further identify potential risk factors along with medications with beneficial effects. We believe that our approach will help in personalized interventions of PTSD for a variety of clinical scenarios.

[Source](https://doi.org/10.21203/rs.3.rs-2949487/v1)

10.21203/rs.3.rs-2949487/v1

---

## Latent Dirichlet Allocation modeling of environmental microbiomes.
 12.06.2023, _A Kim, S Sevanto, ER Moore, N Lubbers_


_Microbiota, Microbial Interactions, Phenotype_
Interactions between stressed organisms and their microbiome environments may provide new routes for understanding and controlling biological systems. However, microbiomes are a form of high-dimensional data, with thousands of taxa present in any given sample, which makes untangling the interaction between an organism and its microbial environment a challenge. Here we apply Latent Dirichlet Allocation (LDA), a technique for language modeling, which decomposes the microbial communities into a set of topics (non-mutually-exclusive sub-communities) that compactly represent the distribution of full communities. LDA provides a lens into the microbiome at broad and fine-grained taxonomic levels, which we show on two datasets. In the first dataset, from the literature, we show how LDA topics succinctly recapitulate many results from a previous study on diseased coral species. We then apply LDA to a new dataset of maize soil microbiomes under drought, and find a large number of significant associations between the microbiome topics and plant traits as well as associations between the microbiome and the experimental factors, e.g. watering level. This yields new information on the plant-microbial interactions in maize and shows that LDA technique is useful for studying the coupling between microbiomes and stressed organisms.

[Source](https://doi.org/10.1371/journal.pcbi.1011075)

10.1371/journal.pcbi.1011075

---

## When does a Lotka-Volterra model represent microbial interactions? Insights from in vitro nasal bacterial communities.
 06.06.2023, _S Dedrick, V Warrier, KP Lemon, B Momeni_


To alter microbial community composition for therapeutic purposes, an accurate and reliable modeling framework capable of predicting microbial community outcomes is required. Lotka-Volterra (LV) equations have been utilized to describe a breadth of microbial communities, yet, the conditions in which this modeling framework is successful remain unclear. Here, we propose that a set of simple in vitro experiments-growing each member in cell-free spent medium obtained from other members-can be used as a test to decide whether an LV model is appropriate for describing microbial interactions of interest. We show that for LV to be a good candidate, the ratio of growth rate to carrying capacity of each isolate when grown in the cell-free spent media of other isolates should remain constant. Using an in vitro community of human nasal bacteria as a tractable system, we find that LV can be a good approximation when the environment is low-nutrient (i.e., when growth is limited by the availability of nutrients) and complex (i.e., when multiple resources, rather than a few, determine growth). These findings can help clarify the range of applicability of LV models and reveal when a more complex model may be necessary for predictive modeling of microbial communities.IMPORTANCEAlthough mathematical modeling can be a powerful tool to draw useful insights in microbial ecology, it is crucial to know when a simplified model adequately represents the interactions of interest. Here, we take advantage of bacterial isolates from the human nasal passages as a tractable model system and conclude that the commonly used Lotka-Volterra model can represent interactions among microbes well when the environment is complex (with many interaction mediators) and low-nutrient. Our work highlights the importance of considering both realism and simplicity when choosing a model to represent microbial interactions.

[Source](https://doi.org/10.1128/msystems.00757-22)

10.1128/msystems.00757-22

---

