## Autobiography of a gut bacterium.
 24.11.2022, _F Schmidt_


Recordings of transient transcriptional events shed light on the gut microbiome.

[Source](https://doi.org/10.1126/science.adf4442)

10.1126/science.adf4442

---

## Untargeted Fecal Metabolomic Analyses across an Industrialization Gradient Reveal Shared Metabolites and Impact of Industrialization on Fecal Microbiome-Metabolome Interactions.
 23.11.2022, _JJ Haffner, M Katemauswa, TS Kagone, E Hossain, D Jacobson, K Flores, AR Parab, AJ Obregon-Tito, RY Tito, LM Reyes, L Troncoso-Corzo, E Guija-Poma, N Meda, H Carabin, TP Honap, K Sankaranarayanan, CM Lewis, LI McCall_


The metabolome is a central determinant of human phenotypes and includes the plethora of small molecules produced by host and microbiome or taken up from exogenous sources. However, studies of the metabolome have so far focused predominantly on urban, industrialized populations. Through an untargeted metabolomic analysis of 90 fecal samples from human individuals from Africa and the Americas-the birthplace and the last continental expansion of our species, respectively-we characterized a shared human fecal metabolome. The majority of detected metabolite features were ubiquitous across populations, despite any geographic, dietary, or behavioral differences. Such shared metabolite features included hyocholic acid and cholesterol. However, any characterization of the shared human fecal metabolome is insufficient without exploring the influence of industrialization. Here, we show chemical differences along an industrialization gradient, where the degree of industrialization correlates with metabolomic changes. We identified differential metabolite features such as amino acid-conjugated bile acids and urobilin as major metabolic correlates of these behavioral shifts. Additionally, coanalyses with over 5,000 publicly available human fecal samples and cooccurrence probability analyses with the gut microbiome highlight connections between the human fecal metabolome and gut microbiome. Our results indicate that industrialization significantly influences the human fecal metabolome, but diverse human lifestyles and behavior still maintain a shared human fecal metabolome. This study represents the first characterization of the shared human fecal metabolome through untargeted analyses of populations along an industrialization gradient. IMPORTANCE As the world becomes increasingly industrialized, understanding the biological consequences of these lifestyle shifts and what it means for past, present, and future human health is critical. Indeed, industrialization is associated with rises in allergic and autoimmune health conditions and reduced microbial diversity. Exploring these health effects on a chemical level requires consideration of human lifestyle diversity, but understanding the significance of any differences also requires knowledge of what molecular components are shared between human groups. Our study reveals the key chemistry of the human gut as defined by varied industrialization-based differences and ubiquitous shared features. Ultimately, these novel findings extend our knowledge of human molecular biology, especially as it is influenced by lifestyle and behavior, and provide steps toward understanding how human biology has changed over our species' history.

[Source](https://doi.org/10.1128/msystems.00710-22)

10.1128/msystems.00710-22

---

## Diet standardization reduces intra-individual microbiome variation.
 25.11.2022, _C Delaroque, GD Wu, C Compher, J Ni, L Albenberg, Q Liu, Y Tian, AD Patterson, JD Lewis, AT Gewirtz, B Chassaing_


The human gut microbiota is highly heterogenous between individuals and also exhibits considerable day-to-day variation within individuals. We hypothesized that diet contributed to such inter- and/or intra-individual variance. Hence, we investigated the extent to which diet normalization impacted microbiota heterogeneity. We leveraged the control arm of our recently reported controlled-feeding study in which nine healthy individuals consumed a standardized additive-free diet for 10 days. Diet normalization did not impact inter-individual differences but reduced the extent of intra-individual day-to-day variation in fecal microbiota composition. Such decreased heterogeneity reflected individual-specific enrichment and depletion of an array of taxa microbiota members and was paralleled by a trend toward reduced intra-individual variance in fecal LPS and flagellin, which, collectively, reflect microbiota's pro-inflammatory potential. Yet, the microbiota of some subjects did not change significantly over the course of the study, suggesting heterogeneity in microbiota resilience to dietary stress or that baseline diets of some subjects were perhaps similar to our study's standardized diet. Collectively, our results indicate that short-term diet heterogeneity contributes to day-to-day intra-individual microbiota composition variance.

[Source](https://doi.org/10.1080/19490976.2022.2149047)

10.1080/19490976.2022.2149047

---

## Investigation of the gut microbiome, bile acid composition and host immunoinflammatory response in a model of azoxymethane-induced colon cancer at discrete timepoints

[Source](https://doi.org/10.1038/s41416-022-02062-4)

---

## A comparative study to determine the association of gut microbiome with schizophrenia in Zhejiang, China.
 27.11.2022, _F Yan, L Xia, L Xu, L Deng, G Jin_


With the rapid progress of high-throughput sequencing technology, characterization of schizophrenia (SZ) with underlying probing of the gut microbiome can explore pathogenic mechanisms, estimate disease risk, and allow customization of therapeutic and prophylactic modalities. In this study, we compared the differences in gut microbial diversity and composition between 50 SZ subjects and 50 healthy matched subjects in Zhejiang, China via targeted next-generation sequencing (16S rRNA amplicon).

[Source](https://doi.org/10.1186/s12888-022-04328-w)

10.1186/s12888-022-04328-w

---

## Gut Microbiome Analysis for Personalized Nutrition: The State of Science

[Source](https://doi.org/10.1002/mnfr.202200476)

---

## The nonindustrialised microbiome in a modern world

[Source](https://doi.org/10.1042/CS20220203)

---

## A gut microbial metabolite of dietary polyphenols reverses obesity-driven hepatic steatosis

[Source](https://doi.org/10.1073/pnas.2202934119)

---

## <em>Akkermansia muciniphila</em> ameliorates depressive disorders in a murine alcohol-LPS (mALPS) model

[Source](https://doi.org/10.1039/d2fo01478e)

---

## The Kitty Microbiome Project: Defining the Healthy Fecal "Core Microbiome" in Pet Domestic Cats.
 27.11.2022, _HH Ganz, G Jospin, CA Rojas, AL Martin, K Dahlhausen, DD Kingsbury, CX Osborne, Z Entrolezo, S Redner, B Ramirez, JA Eisen, M Leahy, C Keaton, J Wong, J Gardy, JK Jarett_


Here, we present a taxonomically defined fecal microbiome dataset for healthy domestic cats (Felis catus) fed a range of commercial diets. We used this healthy reference dataset to explore how age, diet, and living environment correlate with fecal microbiome composition. Thirty core bacterial genera were identified. Prevotella, Bacteroides, Collinsella, Blautia, and Megasphaera were the most abundant, and Bacteroides, Blautia, Lachnoclostridium, Sutterella, and Ruminococcus&amp;amp;nbsp;gnavus were the most prevalent. While community composition remained relatively stable across different age classes, the number of core taxa present decreased significantly with age. Fecal microbiome composition varied with host diet type. Cats fed kibble had a slightly, but significantly greater number of core taxa compared to cats not fed any kibble. The core microbiomes of cats fed some raw food contained taxa not as highly prevalent or abundant as cats fed diets that included kibble. Living environment also had a large effect on fecal microbiome composition. Cats living in homes differed significantly from those in shelters and had a greater portion of their microbiomes represented by core taxa. Collectively our work reinforces the findings that age, diet, and living environment are important factors to consider when defining a core microbiome in a population.

[Source](https://doi.org/10.3390/vetsci9110635)

10.3390/vetsci9110635

---

## Bioinformatic Approaches for Describing the Oral Microbiota.
 25.11.2022, _K Biswas, MW Taylor, DTJ Broderick_


_Computational Biology, Microbiota, RNA, Ribosomal, 16S_
Cultivation-independent (molecular) analysis of the oral microbiota can provide a comprehensive picture of microbial community composition, yet there is an at-times bewildering array of approaches that can be employed. This chapter introduces some of the key considerations when undertaking microbiota research and describes two alternative bioinformatic pipelines for conducting such studies. The descriptions are based on analysis of bacterial 16S ribosomal RNA gene sequences, but can be easily adapted for analysis of other microbial taxa such as fungi.

[Source](https://doi.org/10.1007/978-1-0716-2780-8_8)

10.1007/978-1-0716-2780-8_8

---

## Predicting butyrate- and propionate-forming bacteria of gut microbiota from sequencing data.
 25.11.2022, _B Kircher, S Woltemate, F Gutzki, D Schlüter, R Geffers, H Bähre, M Vital_


_Humans, Gastrointestinal Microbiome, Butyrates, Propionates, RNA, Ribosomal, 16S, Fatty Acids, Volatile, Bacteria_
The bacteria-derived short-chain fatty acids (SCFAs) butyrate and propionate play important (distinct) roles in health and disease, and understanding the ecology of respective bacteria on a community-wide level is a top priority in microbiome research. Applying sequence data (metagenomics and 16S rRNA gene) to predict SCFAs production in vitro and in vivo, a clear split between butyrate- and propionate-forming bacteria was detected with only very few taxa exhibiting pathways for the production of both SCFAs. After in vitro growth of fecal communities from distinct donors (n = 8) on different substrates (n = 7), abundances of bacteria exhibiting pathways correlated with respective SCFA concentrations, in particular in the case of butyrate. For propionate, correlations were weaker, indicating that its production is less imprinted into the core metabolism compared with butyrate-forming bacteria. Longitudinal measurements in vivo (n = 5 time-points from 20 subjects) also revealed a correlation between abundances of pathway-carrying bacteria and concentrations of the two SCFAs. Additionally, lower bacterial cell concentrations, together with higher stool moisture, promoted overall bacterial activity (measured by flow cytometry and coverage patterns of metagenome-assembled genomes) that led to elevated SCFA concentrations with over-proportional levels of butyrate. Predictions on pathway abundances based on 16S rRNA gene data using our in-house database worked well, yielding similar results as metagenomic-based analyses. Our study indicates that stimulating growth of butyrate- and propionate-producing bacteria directly leads to more production of those compounds, which is governed by two functionally distinct bacterial groups facilitating the development of precision intervention strategies targeting either metabolite.

[Source](https://doi.org/10.1080/19490976.2022.2149019)

10.1080/19490976.2022.2149019

---

## Machine learning for data integration in human gut microbiome.
 26.11.2022, _P Li, H Luo, B Ji, J Nielsen_


_Humans, Gastrointestinal Microbiome, Microbiota, Metagenomics, Dysbiosis, Machine Learning_
Recent studies have demonstrated that gut microbiota plays critical roles in various human diseases. High-throughput technology has been widely applied to characterize the microbial ecosystems, which led to an explosion of different types of molecular profiling data, such as metagenomics, metatranscriptomics and metabolomics. For analysis of such data, machine learning algorithms have shown to be useful for identifying key molecular signatures, discovering potential patient stratifications, and particularly for generating models that can accurately predict phenotypes. In this review, we first discuss how dysbiosis of the intestinal microbiota is linked to human disease development and how potential modulation strategies of the gut microbial ecosystem can be used for disease treatment. In addition, we introduce categories and workflows of different machine learning approaches, and how they can be used to perform integrative analysis of multi-omics data. Finally, we review advances of machine learning in gut microbiome applications and discuss related challenges. Based on this we conclude that machine learning is very well suited for analysis of gut microbiome and that these approaches can be useful for development of gut microbe-targeted therapies, which ultimately can help in achieving personalized and precision medicine.

[Source](https://doi.org/10.1186/s12934-022-01973-4)

10.1186/s12934-022-01973-4

---

## Impact of innovative nanoadditives on biodigesters microbiome.
 23.11.2022, _FY Hassaneen, RZ Abdallah, MS Abdallah, N Ahmed, SMM Abd Elaziz, MA El-Mokhtar, MS Badary, R Siam, NK Allam_


Nanoparticles (NPs) supplementation to biodigesters improves the digestibility of biowaste and the generation of biogas. This study investigates the impact of innovative nanoadditives on the microbiome of biodigesters. Fresh cow manure was anaerobically incubated in a water bath under mesophilic conditions for 30 days. Three different NPs (zinc ferrite, zinc ferrite with 10% carbon nanotubes and zinc ferrite with 10% C76 fullerene) were separately supplemented to the biodigesters at the beginning of the incubation period. Methane and hydrogen production were monitored daily. Manure samples were collected from the digesters at different time points and the microbial communities inside the biodigesters were investigated via real-time PCR and 16 S rRNA gene amplicon-sequencing. The results indicate that zinc ferrite NPs enhanced biogas production the most. The microbial community was significantly affected by NPs addition in terms of archaeal and bacterial 16 S rRNAgene copy numbers. The three ZF formulations NPs augmented the abundance of members within the hydrogenotrophic methanogenic phyla Methanobacteriaceae. While Methanomassiliicoccacaea were enriched in ZF/C76 supplemented biodigester due to a significant increase in hydrogen partial pressure, probably caused by the enrichment of Spirochaetaceae (genus Treponema). Overall, NPs supplementation significantly enriched acetate-producing members within Hungateiclostridiaceae in ZF/CNTs, Dysgonomonadaceae in ZF and Spirochaetaceae ZF/C76 biodigesters.

[Source](https://doi.org/10.1111/1751-7915.14173)

10.1111/1751-7915.14173

---

## Microbiome analysis in Lascaux Cave in relation to black stain alterations of rock surfaces and collembola.
 25.11.2022, _L Alonso, T Pommier, L Simon, F Maucourt, J Doré, A Dubost, V Trân Van, G Minard, C Valiente Moro, CJ Douady, Y Moënne-Loccoz_


Anthropization of Palaeolithic caves open for tourism may favour collembola invasion and result in the formation of black stains attributed to pigmented fungi. However, ecological processes underpinning black stain formation are not fully understood. Here, we tested the hypotheses that black stains from the Apse room of Lascaux Cave display a specific microbiota enriched in pigmented fungi, and that collembola thriving on the stains have the potential to consume and disseminate these black fungi. Metabarcoding showed that the microbiota of black stains and neighbouring unstained parts strongly differed, with in black stains a higher prevalence of Ochroconis and other pigmented fungi and the strong regression of Pseudomonas bacteria (whose isolates inhibited in vitro the growth of pigmented fungi). Isotopic analyses indicated that Folsomia candida collembola thriving on stains could feed on black stain in situ and assimilate the pigmented fungi they were fed with in vitro. They could carry these fungi and disseminate them when tested with complex black stains from Lascaux. This shows that black stain formation is linked to the development of pigmented fungi, which coincides with the elimination of antagonistic pseudomonads, and points towards a key role of F. candida collembola in the dynamics of pigmented fungi.

[Source](https://doi.org/10.1111/1758-2229.13133)

10.1111/1758-2229.13133

---

