## Assessing Bacterial Diversity in the Rhizosphere of Thymus zygis Growing in the Sierra Nevada National Park (Spain) through Culture-Dependent and Independent Approaches.
 22.02.2019, _J Pascual, S Blanco, M García-López, A García-Salamanca, SA Bursakov, O Genilloud, GF Bills, JL Ramos, P van Dillewijn_


_Genetic Variation, Microbiota, Molecular Typing, Parks, Recreational, Plant Roots, RNA, Bacterial, RNA, Ribosomal, 16S, Rhizosphere, Sequence Analysis, RNA, Soil Microbiology, Spain, Thymus Plant_
Little is known of the bacterial communities associated with the rhizosphere of wild plant species found in natural settings. The rhizosphere bacterial community associated with wild thyme, Thymus zygis L., plants was analyzed using cultivation, the creation of a near-full length 16S rRNA gene clone library and 454 amplicon pyrosequencing. The bacterial community was dominated by Proteobacteria (mostly Alphaproteobacteria and Betaproteobacteria), Actinobacteria, Acidobacteria, and Gemmatimonadetes. Although each approach gave a different perspective of the bacterial community, all classes/subclasses detected in the clone library and the cultured bacteria could be found in the pyrosequencing datasets. However, an exception caused by inconclusive taxonomic identification as a consequence of the short read length of pyrotags together with the detection of singleton sequences which corresponded to bacterial strains cultivated from the same sample highlight limitations and considerations which should be taken into account when analysing and interpreting amplicon datasets. Amplicon pyrosequencing of replicate rhizosphere soil samples taken a year later permit the definition of the core microbiome associated with Thymus zygis plants. Abundant bacterial families and predicted functional profiles of the core microbiome suggest that the main drivers of the bacterial community in the Thymus zygis rhizosphere are related to the nutrients originating from the plant root and to their participation in biogeochemical cycles thereby creating an intricate relationship with this aromatic plant to allow for a feedback ecological benefit.

[Source](https://www.nature.com/articles/s41579-020-0368-1.epdf)

10.1371/journal.pone.0146558

---

## Diversity within species: interpreting strains in microbiomes

[Source](https://www.nature.com/articles/s41579-020-0368-1.epdf)

---

## Earth microbial co-occurrence network reveals interconnection pattern across microbiomes

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00857-2)

---

## Local oral and nasal microbiome diversity in age-related macular degeneration.
 13.03.2020, _J Rullo, PM Far, M Quinn, N Sharma, S Bae, I Irrcher, S Sharma_


Age-related macular degeneration (AMD) is a chronic degenerative disease of the retina. Recent reports have highlighted the potential role of mucosal surface microbes in the pathogenesis of AMD. In this case-control study, the composition of the nasal and oral microbiota in newly diagnosed neovascular age-related macular degeneration cases (6 male, 7 female) was compared to controls without retinal diseases (2 male, 3 female). PCR amplification of 16S rRNA genes was performed with universal primers amplifying the V4 variable region (515F-806R). Distinct microbial community characterization was achieved using Principal Coordinates Analysis (PCoA) of the Bray-Curtis index with comparative analysis between cases and controls performed within QIIME 2. Sequencing of all cases and controls revealed clear separation with strong beta diversity between oral and nasal microbial communities (p &lt; 0.001). Microbial composition differed between cases and controls in both oral and nasal samples. The top three oral microbes identified as different compared to controls included Burkholderiales (7.41 log2fold change, p = 3.29E-05), Actinomyceataceae (6.22 log2fold change, p = 3.73E-06) and Gemella (5.28 log2fold change, p = 0.0002). The top three nasal microbes identified as different compared to controls included Rothia (13.6 log2fold change, p = 3.63E-18), Actinobacteria (10.29 log2fold change, p = 9.81E-10) and Propionibacteriales (8.73 log2fold change, p = 6.74E-09). These relative shifts in communities of bacteria detected in newly diagnosed neovascular AMD patients may suggest additional mechanistic links in disease pathogenesis.

[Source](https://www.nature.com/articles/s41598-020-60674-3)

10.1038/s41598-020-60674-3

---

## Lactobacilli Have a Niche in the Human Nose.
 27.05.2020, _I De Boeck, MFL van den Broek, CN Allonsius, I Spacova, S Wittouck, K Martens, S Wuyts, E Cauwenberghs, K Jokicevic, D Vandenheuvel, T Eilers, M Lemarcq, C De Rudder, S Thys, JP Timmermans, AV Vroegop, A Verplaetse, T Van de Wiele, F Kiekens, PW Hellings, OM Vanderveken, S Lebeer_


Although an increasing number of beneficial microbiome members are characterized for the human gut and vagina, beneficial microbes are underexplored for the human upper respiratory tract (URT). In this study, we demonstrate that taxa from the beneficial Lactobacillus genus complex are more prevalent in the healthy URT than in patients with chronic rhinosinusitis (CRS). Several URT-specific isolates are cultured, characterized, and further explored for their genetic and functional properties related to adaptation to the URT. Catalase genes are found in the identified lactobacilli, which is a unique feature within this mostly facultative anaerobic genus. Moreover, one of our isolated strains, Lactobacillus casei AMBR2, contains fimbriae that enable strong adherence to URT epithelium, inhibit the growth and virulence of several URT pathogens, and successfully colonize nasal epithelium of healthy volunteers. This study thus demonstrates that specific lactobacilli are adapted to the URT and could have a beneficial keystone function in this habitat.

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(20)30627-6)

10.1016/j.celrep.2020.107674

---

## Resistant maltodextrin intake reduces virulent metabolites in the gut environment: randomized control study in a Japanese cohort

[Source](https://www.medrxiv.org/content/10.1101/2020.05.25.20112508v1.full.pdf)

---

## One year cross-sectional study in adult and neonatal intensive care units reveals the bacterial and antimicrobial resistance genes profiles in patients and hospital surfaces.
 03.06.2020, _AP Christoff, AFR Sereia, GNF Cruz, DC Bastiani, VL Silva, C Hernandes, APM Nascente, AAD Reis, RG Viessi, ADSP Marques, BS Braga, TPL Raduan, MDV Martino, FG Menezes, LFV Oliveira_


Several studies have shown the ubiquitous presence of bacteria in hospital surfaces, staff, and patients. Frequently, these bacteria are related to HAI (healthcare-associated infections) and carry antimicrobial resistance (AMR). These HAI-related bacteria contribute to a major public health issue by increasing patient morbidity and mortality during or after hospital stay. Bacterial high-throughput amplicon gene sequencing along with identification of AMR genes, as well as whole genome sequencing (WGS), are biotechnological tools that allow multiple-sample screening for a diversity of bacteria. In this paper, we used these methods to perform a one-year cross sectional profiling of bacteria and AMR genes in adult and neonatal intensive care units (ICU and NICU) in a Brazilian public, tertiary hospital. Our results showed high abundances of HAI-related bacteria such as S. epidermidis, S. aureus, K. pneumoniae, A. baumannii complex, E. coli, E. faecalis, and P. aeruginosa in patients and hospital surfaces. Most abundant AMR genes detected throughout ICU and NICU were mecA, blaCTX-M-1 group, blaSHV-like, and blaKPC-like. We found that NICU environment and patients were more widely contaminated with pathogenic bacteria than ICU. Patient samples, despite the higher bacterial load, have lower bacterial diversity than environmental samples in both units. Finally, we also identified contamination hotspots in the hospital environment showing constant frequencies of bacterial and AMR contamination throughout the year. Whole genome sequencing (WGS), 16S rRNA oligotypes, and AMR identification allowed a high-resolution characterization of the hospital microbiome profile.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234127)

10.1371/journal.pone.0234127

---

## Cecal microbiome composition and metabolic function in probiotic treated broilers.
 03.06.2020, _DR Rodrigues, W Briggs, A Duff, K Chasser, R Murugesan, C Pender, S Ramirez, L Valenzuela, L Bielke_


Probiotics have become increasingly popular in the poultry industry as a promising nutritional intervention to promote the modulation of intestinal microbial communities and their metabolic activities as a means of improving health and performance. This study aimed to determine the influence of different probiotic formulations on the taxonomic and metabolic profiling of cecal microbial communities, as well as to define associations between cecal microbiota and growth parameters in 21 and 42-day-old broilers. Probiotics investigated included a synbiotic (SYNBIO), a yeast-based probiotic (YEAST), and three single-strain formulations of spore-forming Bacillus amyloliquefaciens (SINGLE1), B. subtilis (SINGLE2) and B. licheniformis (SINGLE3). Dietary inclusion of SYNBIO, YEAST, SINGLE2, and SINGLE3 into the diets supported a significant stimulation of BW and BWG by 7 days of age. Besides, SYNBIO reduced the overall mortality rate by 42d (p&lt;0.05). No significant variation was observed among different probiotic-based formulations for cecal microbiota composition. However, there was a treatment-specific effect on the metabolic profiles, with a particular beneficial metabolic adaptation by the microbiota when supplemented by SYNBIO and SINGLE2. Furthermore, the population of Lactobacillales was identified to be strongly associated with lower Enterobacteriales colonization, higher BW means, and lower mortality rate of growing broilers. Overall, the results emphasize that probiotic supplementation may enhance the microbial energy metabolism in the ceca of young broilers.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0225921)

10.1371/journal.pone.0225921

---

## Characterization of native<em> Escherichia coli</em> populations from bovine vagina of healthy heifers and cows with postpartum uterine disease

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0228294)

---

## Structural and functional changes of gut microbiota in ovariectomized rats and their correlations with altered bone mass.
 02.06.2020, _S Ma, J Qin, Y Hao, Y Shi, L Fu_


As a critical factor involved in the maintenance of physiological homeostasis, the gut microbiota (GM) reportedly plays a key role in bone development. To date, the association between the GM and steroid deficiency-induced osteoporosis remains poorly understood. Forty female Sprague Dawley rats were divided into an ovariectomy (OVX) or control group. We performed 16S rRNA and metagenome sequencing, to compare diversity, taxonomic differences, and functional genes. The GM composition did not change in the control group and the number of operational taxonomic units increased significantly following ovariectomy. Alpha diversity, determined by ACE estimator, CHAO estimator, the Shannon index, and the Simpson index showed an increasing trend after ovariectomy. Samples in the OVX group were well clustered both pre- and post-ovariectomy, as demonstrated by principal coordinate 1 (PC1) and PC2. Functional genes of GM, including those involved in synthesis and metabolism of carbohydrates and nucleotides, microbial structure, and heme, as well as hemin uptake and utilization, increased at the early stage of osteoporosis. We observed that Ruminococcus flavefaciens exhibited the greatest variation in abundance among the GM and this was also associated with osteoclastic indicators and the estrobolome. Specific changes in fecal microbiota are associated with the pathogenesis of steroid deficiency-induced osteoporosis.

[Source](https://www.aging-us.com/article/103290/text)

10.18632/aging.103290

---

## Structural and Functional Changes of Gut Microbiota in Ovariectomized Rats and Their Correlations With Altered Bone Mass

[Source](https://www.aging-us.com/article/103290/text)

---

## Dissecting the effect of continuous cropping of potato on soil bacterial communities as revealed by high-throughput sequencing.
 01.06.2020, _J Zhao, D Zhang, Y Yang, Y Pan, D Zhao, J Zhu, L Zhang, Z Yang_


Plant rhizosphere-associated bacterial communities play key roles in affecting host health in response to diverse biotic stresses. Currently, the effect of continuous cropping of potato on soil bacterial communities and physiochemical parameters has not been well documented. Herein, we compared bacterial composition and diversity in rotationally and continuously (5, 10, and 30 years) cropped soils, and clarified the correlations between soil properties and the bacterial communities revealed by Illumina MiSeq sequencing. Our results demonstrated that Proteobacteria, Actinobacteria and Firmicutes were the predominant phyla in all the tested soil samples. While the abundance of Proteobacteria showed an increase, the abundance of Actinobacteria and Firmicutes displayed a reduction with the increase of continuous cropping years. At the genus level, as continuous cropping years increasing, the abundance of Pseudarthrobacter, Bacillus and Pseudomonas decreased, but the abundance of Rhodanobacte, Sphingobium, Mizugakiibacter and Devosia increased. Our results also demonstrated that the abundance of plant growth-promoting rhizobacteria in the rotationally cropped soil was significantly higher than that of continuously cropped soil. Furthermore, our results showed that soil organic matter, available nitrogen, available phosphorus and available potassium were significantly correlated with bacterial community distribution. Overall, our work provides a comprehensive view of altered structure and composition of bacterial communities between the continuously cropped soil and rotationally cropped soil.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0233356)

10.1371/journal.pone.0233356

---

## Increasing flow rate reduces biofouling and colonization by filamentous bacteria in drippers fed with reclaimed wastewater

[Source](https://www.biorxiv.org/content/10.1101/2020.06.02.130013v1.full.pdf)

---

## Integration of phage and yeast display platforms: A reliable and cost effective approach for binning of peptides as displayed on-phage.
 01.06.2020, _P Pandya, RO Sayers, JP Ting, S Morshedian, C Torres, JS Cudal, K Zhang, JR Fitchett, Q Zhang, FF Zhang, J Wang, JD Durbin, JJ Carrillo, A Espada, H Broughton, Y Qian, S Afshar_


Hundreds of target specific peptides are routinely discovered by peptide display platforms. However, due to the high cost of peptide synthesis only a limited number of peptides are chemically made for further analysis. Here we describe an accurate and cost effective method to bin peptides on-phage based on binding region(s), without any requirement for peptide or protein synthesis. This approach, which integrates phage and yeast display platforms, requires display of target and its alanine variants on yeast. Flow cytometry was used to detect binding of peptides on-phage to the target on yeast. Once hits were identified, they were synthesized to confirm their binding region(s) by HDX (Hydrogen deuterium exchange) and crystallography. Moreover, we have successfully shown that this approach can be implemented as part of a panning process to deplete non-functional peptides. This technique can be applied to any target that can be successfully displayed on yeast; it narrows down the number of peptides requiring synthesis; and its utilization during selection results in enrichment of peptide population against defined binding regions on the target.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0233961)

10.1371/journal.pone.0233961

---

## Freshwater viral metagenome reveals novel and functional phage-borne antibiotic resistance genes.
 02.06.2020, _K Moon, JH Jeon, I Kang, KS Park, K Lee, CJ Cha, SH Lee, JC Cho_


Antibiotic resistance developed by bacteria is a significant threat to global health. Antibiotic resistance genes (ARGs) spread across different bacterial populations through multiple dissemination routes, including horizontal gene transfer mediated by bacteriophages. ARGs carried by bacteriophages are considered especially threatening due to their prolonged persistence in the environment, fast replication rates, and ability to infect diverse bacterial hosts. Several studies employing qPCR and viral metagenomics have shown that viral fraction and viral sequence reads in clinical and environmental samples carry many ARGs. However, only a few ARGs have been found in viral contigs assembled from metagenome reads, with most of these genes lacking effective antibiotic resistance phenotypes. Owing to the wide application of viral metagenomics, nevertheless, different classes of ARGs are being continuously found in viral metagenomes acquired from diverse environments. As such, the presence and functionality of ARGs encoded by bacteriophages remain up for debate.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00863-4)

10.1186/s40168-020-00863-4

---

## Estimating Microbial Interaction Network: Zero-inflated Latent Ising Model Based Approach

[Source](https://www.biorxiv.org/content/10.1101/2020.06.02.130914v1.full.pdf)

---

## Assessment of statistical methods from single cell, bulk RNA-seq and metagenomics applied to microbiome data

[Source](https://www.biorxiv.org/content/10.1101/2020.01.15.907964v2.full.pdf)

---

## Updated ATLAS of Biochemistry with New Metabolites and Improved Enzyme Prediction Power

[Source](https://pubs.acs.org/doi/pdf/10.1021/acssynbio.0c00052)

---

## Inferring Active Metabolic Pathways from Proteomics and Essentiality Dat

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(20)30699-9)

---

## May 2020 Pod: Discovering novel resistance genes and how bacteria become virulent

[Source](https://podcasts.apple.com/se/podcast/may-2020-pod-discovering-novel-resistance-genes-how/id1509550298?i=1000476249284&l=en)

---

