## Integrated metagenomic and metabolomic analysis reveals distinct gut-microbiome-derived phenotypes in early-onset colorectal cancer.
 11.08.2022, _C Kong, L Liang, G Liu, L Du, Y Yang, J Liu, D Shi, X Li, Y Ma_


The incidence of early-onset colorectal cancer (EO-CRC) is steadily increasing. Here, we aimed to characterise the interactions between gut microbiome, metabolites and microbial enzymes in EO-CRC patients and evaluate their potential as non-invasive biomarkers for EO-CRC.

[Source](https://gut.bmj.com/content/early/2022/08/11/gutjnl-2022-327156)

10.1136/gutjnl-2022-327156

---

## A decrease in functional microbiomes represented as <em>Faecalibacterium</em> affects immune homeostasis in long-term stable liver transplant patients

[Source](https://doi.org/10.1080/19490976.2022.2102885)

---

## Longitudinal study of the vaginal microbiome in pregnancies involving preterm labor.
 11.08.2022, _Y Sakabe, H Nishizawa, A Kato, Y Noda, A Ohwaki, H Yoshizawa, T Kato, T Sekiya, T Fujii, H Kurahashi_


Alterations in the vaginal bacterial flora reflect the status of various obstetric conditions and are associated with mechanisms that underlie certain pregnancy-associated complications. These changes are also a predictive biomarker for clinical outcomes of these adverse events.

[Source](https://doi.org/10.20407/fmj.2021-017)

10.20407/fmj.2021-017

---

## Oral microbiome and serological analyses on association of Alzheimer's disease and periodontitis.
 11.08.2022, _KL Fu, MJ Chiu, N Wara-Aswapati, CN Yang, LC Chang, YL Guo, YH Ni, YW Chen_


To investigate the association between Alzheimer's disease (AD) and periodontitis in the aspects of periodontal status, serological markers and oral microbiome.

[Source](https://doi.org/10.1111/odi.14348)

10.1111/odi.14348

---

## Ultra-processed foods and human health: from epidemiological evidence to mechanistic insights.
 11.08.2022, _B Srour, MC Kordahi, E Bonazzi, M Deschasaux-Tanguy, M Touvier, B Chassaing_


Epidemiological studies have suggested a role for ultra-processed foods in numerous chronic inflammatory diseases such as inflammatory bowel diseases and metabolic syndrome. Preclinical and clinical studies are accumulating to better decipher the effects of various aspects of food processing and formulation on the aetiology of chronic, debilitating inflammatory diseases. In this Review, we provide an overview of the current data that highlight an association between ultra-processed food consumption and various chronic diseases, with a focus on epidemiological evidence and mechanistic insights involving the intestinal microbiota.

[Source](https://doi.org/10.1016/S2468-1253(22)00169-8)

10.1016/S2468-1253(22)00169-8

---

## Dietary Fiber Modulates the Release of Gut Bacterial Products Preventing Cognitive Decline in an Alzheimer’s Mouse Model

[Source](https://doi.org/10.1007/s10571-022-01268-7)

---

## Functional and safety characterization of beneficial vaginal lactic acid bacteria for the design of vaginal hygiene products.
 11.08.2022, _JA Silva, A Marchesi, MC Aristimuño Ficosecco, MEF Nader-Macías_


The aim was to complement the evaluation of functional and safety properties of beneficial vaginal lactic acid bacteria to select the most adequate strains as potential probiotics for their inclusion in a vaginal probiotic formula.

[Source](https://doi.org/10.1111/jam.15752)

10.1111/jam.15752

---

## Butyrate-producing <em>Eubacterium rectale</em> suppresses lymphomagenesis by alleviating the TNF-induced TLR4/MyD88/NF-κB axis

[Source](https://doi.org/10.1016/j.chom.2022.07.003)

---

## The double-edged sword of probiotic supplementation on gut microbiota structure in <em>Helicobacter pylori</em> management

[Source](https://doi.org/10.1080/19490976.2022.2108655)

---

## Functional Redundancy in Bat Microbial Assemblage in the Presence of the White Nose Pathogen

[Source](https://doi.org/10.1007/s00248-022-02098-2)

---

## Non-steroidal anti-inflammatory drugs caused an outbreak of inflammation and oxidative stress with changes in the gut microbiota in rainbow trout (<em>Oncorhynchus mykiss</em>)

[Source](https://doi.org/10.1016/j.scitotenv.2022.157921)

---

## Bees under interactive stressors: The novel insecticides flupyradifurone and sulfoxaflor along with the fungicide azoxystrobin disrupt the gut microbiota of honey bees and increase opportunistic bacterial pathogens.
 11.08.2022, _Y Al Naggar, B Singavarapu, RJ Paxton, T Wubet_


The gut microbiome plays an important role in bee health and disease. But it can be disrupted by pesticides and in-hive chemicals, putting honey bee health in danger. We used a controlled and fully crossed laboratory experimental design to test the effects of a 10-day period of chronic exposure to field-realistic sublethal concentrations of two nicotinic acetylcholine receptor agonist insecticides (nACHRs), namely flupyradifurone (FPF) and sulfoxaflor (Sulf), and a fungicide, azoxystrobin (Azoxy), individually and in combination, on the survival of individual honey bee workers and the composition of their gut microbiota (fungal and bacterial diversity). Metabarcoding was used to examine the gut microbiota on days 0, 5, and 10 of pesticide exposure to determine how the microbial response varies over time; to do so, the fungal ITS2 fragment and the V4 region of the bacterial 16S rRNA were targeted. We found that FPF has a negative impact on honey bee survival, but interactive (additive or synergistic) effects between either insecticide and the fungicide on honey bee survival were not statistically significant. Pesticide treatments significantly impacted the microbial community composition. The fungicide Azoxy substantially reduced the Shannon diversity of fungi after chronic exposure for 10 days. The relative abundance of the top 10 genera of the bee gut microbiota was also differentially affected by the fungicide, insecticides, and fungicide-insecticide combinations. Gut microbiota dysbiosis was associated with an increase in the relative abundance of opportunistic pathogens such as Serratia spp. (e.g. S. marcescens), which can have devastating consequences for host health such as increased susceptibility to infection and reduced lifespan. Our findings raise concerns about the long-term impact of novel nACHR insecticides, particularly FPF, on pollinator health and recommend a novel methodology for a refined risk assessment that includes the potential effects of agrochemicals on the gut microbiome of bees.

[Source](https://doi.org/10.1016/j.scitotenv.2022.157941)

10.1016/j.scitotenv.2022.157941

---

## Host genotype controls ecological change in the leaf fungal microbiome.
 12.08.2022, _A VanWallendael, GMN Benucci, PB da Costa, L Fraser, A Sreedasyam, F Fritschi, TE Juenger, JT Lovell, G Bonito, DB Lowry_


Leaf fungal microbiomes can be fundamental drivers of host plant success, as they contain pathogens that devastate crop plants and taxa that enhance nutrient uptake, discourage herbivory, and antagonize pathogens. We measured leaf fungal diversity with amplicon sequencing across an entire growing season in a diversity panel of switchgrass (Panicum virgatum). We also sampled a replicated subset of genotypes across 3 additional sites to compare the importance of time, space, ecology, and genetics. We found a strong successional pattern in the microbiome shaped both by host genetics and environmental factors. Further, we used genome-wide association (GWA) mapping and RNA sequencing to show that 3 cysteine-rich receptor-like kinases (crRLKs) were linked to a genetic locus associated with microbiome structure. We confirmed GWAS results in an independent set of genotypes for both the internal transcribed spacer (ITS) and large subunit (LSU) ribosomal DNA markers. Fungal pathogens were central to microbial covariance networks, and genotypes susceptible to pathogens differed in their expression of the 3 crRLKs, suggesting that host immune genes are a principal means of controlling the entire leaf microbiome.

[Source](https://doi.org/10.1371/journal.pbio.3001681)

10.1371/journal.pbio.3001681

---

## <em>Suaeda salsa</em> Root-Associated Microorganisms Could Effectively Improve Maize Growth and Resistance under Salt Stress

[Source](https://doi.org/10.1128/spectrum.01349-22)

---

## Regulation of immune complex formation and signalling by FERONIA, a busy goddess in plant-microbe interactions.
 11.08.2022, _Z Duan, W Liu, K Li, W Duan, S Zhu, J Xing, T Chen, X Luo_


Being sessile in soil, plant cells rely on cell-surface receptors to sense and transduce environmental stimulus signals into intracellular responses. FERONIA (FER), a Catharanthus roseus receptor-like kinase 1-like protein, has emerged as a versatile regulator of plant growth, development, and stress responses. In recent years, accumulating studies have witnessed rapid advances in dissecting the mechanisms underlying the interaction between FER and its partners in response to pathogen invasion, particularly regulation of immune complex formation and signalling. Moreover, hormonal signalling, rhizosphere microbiota and other constituents are also extensively involved in these processes.

[Source](https://doi.org/10.1111/mpp.13256)

10.1111/mpp.13256

---

## A systematic, complexity-reduction approach to dissect the kombucha tea microbiome.
 12.08.2022, _X Huang, Y Xin, T Lu_


One defining goal of microbiome research is to uncover mechanistic causation that dictates the emergence of structural and functional traits of microbiomes. However, the extraordinary degree of ecosystem complexity has hampered the realization of the goal. Here, we developed a systematic, complexity-reducing strategy to mechanistically elucidate the compositional and metabolic characteristics of microbiome by using the kombucha tea microbiome as an example. The strategy centered around a two-species core that was abstracted from but recapitulated the native counterpart. The core was convergent in its composition, coordinated on temporal metabolic patterns, and capable for pellicle formation. Controlled fermentations uncovered the drivers of these characteristics, which were also demonstrated translatable to provide insights into the properties of communities with increased complexity and altered conditions. This work unravels the pattern and process underlying the kombucha tea microbiome, providing a potential conceptual framework for mechanistic investigation of microbiome behaviors.

[Source](https://doi.org/10.7554/eLife.76401)

10.7554/eLife.76401

---

## ProbeTools: designing hybridization probes for targeted genomic sequencing of diverse and hypervariable viral taxa.
 11.08.2022, _KS Kuchinski, J Duan, C Himsworth, W Hsiao, NA Prystajecky_


Sequencing viruses in many specimens is hindered by excessive background material from hosts, microbiota, and environmental organisms. Consequently, enrichment of target genomic material is necessary for practical high-throughput viral genome sequencing. Hybridization probes are widely used for enrichment in many fields, but their application to viral sequencing faces a major obstacle: it is difficult to design panels of probe oligo sequences that broadly target many viral taxa due to their rapid evolution, extensive diversity, and genetic hypervariability. To address this challenge, we created ProbeTools, a package of bioinformatic tools for generating effective viral capture panels, and for assessing coverage of target sequences by probe panel designs in silico. In this study, we validated ProbeTools by designing a panel of 3600 probes for subtyping the hypervariable haemagglutinin (HA) and neuraminidase (NA) genome segments of avian-origin influenza A viruses (AIVs). Using in silico assessment of AIV reference sequences and in vitro capture on egg-cultured viral isolates, we demonstrated effective performance by our custom AIV panel and ProbeTools' suitability for challenging viral probe design applications.

[Source](https://doi.org/10.1186/s12864-022-08790-4)

10.1186/s12864-022-08790-4

---

## A machine learning framework for discovery and enrichment of metagenomics metadata from open access publications.
 12.08.2022, _M Nassar, AB Rogers, F Talo', S Sanchez, Z Shafique, RD Finn, J McEntyre_


Metagenomics is a culture-independent method for studying the microbes inhabiting a particular environment. Comparing the composition of samples (functionally/taxonomically), either from a longitudinal study or cross-sectional studies, can provide clues into how the microbiota has adapted to the environment. However, a recurring challenge, especially when comparing results between independent studies, is that key metadata about the sample and molecular methods used to extract and sequence the genetic material are often missing from sequence records, making it difficult to account for confounding factors. Nevertheless, these missing metadata may be found in the narrative of publications describing the research. Here, we describe a machine learning framework that automatically extracts essential metadata for a wide range of metagenomics studies from the literature contained in Europe PMC. This framework has enabled the extraction of metadata from 114,099 publications in Europe PMC, including 19,900 publications describing metagenomics studies in European Nucleotide Archive (ENA) and MGnify. Using this framework, a new metagenomics annotations pipeline was developed and integrated into Europe PMC to regularly enrich up-to-date ENA and MGnify metagenomics studies with metadata extracted from research articles. These metadata are now available for researchers to explore and retrieve in the MGnify and Europe PMC websites, as well as Europe PMC annotations API.

[Source](https://doi.org/10.1093/gigascience/giac077)

10.1093/gigascience/giac077

---

## MetaProClust-MS1: an MS1 Profiling Approach for Large-Scale Microbiome Screening.
 11.08.2022, _CMA Simopoulos, Z Ning, L Li, MM Khamis, X Zhang, M Lavallée-Adam, D Figeys_


Metaproteomics is used to explore the functional dynamics of microbial communities. However, acquiring metaproteomic data by tandem mass spectrometry (MS/MS) is time-consuming and resource-intensive, and there is a demand for computational methods that can be used to reduce these resource requirements. We present MetaProClust-MS1, a computational framework for microbiome feature screening developed to prioritize samples for follow-up MS/MS. In this proof-of-concept study, we tested and compared MetaProClust-MS1 results on gut microbiome data, from fecal samples, acquired using short 15-min MS1-only chromatographic gradients and MS1 spectra from longer 60-min gradients to MS/MS-acquired data. We found that MetaProClust-MS1 identified robust gut microbiome responses caused by xenobiotics with significantly correlated cluster topologies of comparable data sets. We also used MetaProClust-MS1 to reanalyze data from both a clinical MS/MS diagnostic study of pediatric patients with inflammatory bowel disease and an experiment evaluating the therapeutic effects of a small molecule on the brain tissue of Alzheimer's disease mouse models. MetaProClust-MS1 clusters could distinguish between inflammatory bowel disease diagnoses (ulcerative colitis and Crohn's disease) using samples from mucosal luminal interface samples and identified hippocampal proteome shifts of Alzheimer's disease mouse models after small-molecule treatment. Therefore, we demonstrate that MetaProClust-MS1 can screen both microbiomes and single-species proteomes using only MS1 profiles, and our results suggest that this approach may be generalizable to any proteomics experiment. MetaProClust-MS1 may be especially useful for large-scale metaproteomic screening for the prioritization of samples for further metaproteomic characterization, using MS/MS, for instance, in addition to being a promising novel approach for clinical diagnostic screening. IMPORTANCE Growing evidence suggests that human gut microbiome composition and function are highly associated with health and disease. As such, high-throughput metaproteomic studies are becoming more common in gut microbiome research. However, using a conventional long liquid chromatography (LC)-MS/MS gradient metaproteomics approach as an initial screen in large-scale microbiome experiments can be slow and expensive. To combat this challenge, we introduce MetaProClust-MS1, a computational framework for microbiome screening using MS1-only profiles. In this proof-of-concept study, we show that MetaProClust-MS1 identifies clusters of gut microbiome treatments using MS1-only profiles similar to those identified using MS/MS. Our approach allows researchers to prioritize samples and treatments of interest for further metaproteomic analyses and may be generally applicable to any proteomic analysis. In particular, this approach may be especially useful for large-scale metaproteomic screening or in clinical settings where rapid diagnostic evidence is required.

[Source](https://doi.org/10.1128/msystems.00381-22)

10.1128/msystems.00381-22

---

