## Dietary fiber and probiotics influence the gut microbiome and melanoma immunotherapy response.
 23.12.2021, _CN Spencer, JL McQuade, V Gopalakrishnan, JA McCulloch, M Vetizou, AP Cogdill, MAW Khan, X Zhang, MG White, CB Peterson, MC Wong, G Morad, T Rodgers, JH Badger, BA Helmink, MC Andrews, RR Rodrigues, A Morgun, YS Kim, J Roszik, KL Hoffman, J Zheng, Y Zhou, YB Medik, LM Kahn, S Johnson, CW Hudgens, K Wani, PO Gaudreau, AL Harris, MA Jamal, EN Baruch, E Perez-Guijarro, CP Day, G Merlino, B Pazdrak, BS Lochmann, RA Szczepaniak-Sloane, R Arora, J Anderson, CM Zobniw, E Posada, E Sirmans, J Simon, LE Haydu, EM Burton, L Wang, M Dang, K Clise-Dwyer, S Schneider, T Chapman, NAS Anang, S Duncan, J Toker, JC Malke, IC Glitza, RN Amaria, HA Tawbi, A Diab, MK Wong, SP Patel, SE Woodman, MA Davies, MI Ross, JE Gershenwald, JE Lee, P Hwu, V Jensen, Y Samuels, R Straussman, NJ Ajami, KC Nelson, L Nezi, JF Petrosino, PA Futreal, AJ Lazar, J Hu, RR Jenq, MT Tetzlaff, Y Yan, WS Garrett, C Huttenhower, P Sharma, SS Watowich, JP Allison, L Cohen, G Trinchieri, CR Daniel, JA Wargo_


[Figure: see text].

[Source](https://doi.org/10.1126/science.aaz7015)

10.1126/science.aaz7015

---

## Association of the gut microbiota and fecal short-chain fatty acids with skeletal muscle mass and strength in children.
 24.12.2021, _F Chen, Q Li, Y Chen, Y Wei, J Liang, Y Song, L Shi, J Wang, L Mao, B Zhang, Z Zhang_


We aimed to investigate whether the gut microbiota and fecal short-chain fatty acids (SCFAs) are associated with skeletal muscle mass and strength in healthy Chinese children aged 6-9 years. In this study, 412 children were enrolled. 16S rRNA gene sequencing was used to characterize the gut microbiota compositions. Fecal SCFAs were quantified using high-performance liquid chromatography. Dual X-ray absorptiometry was used to measure the total body lean soft tissue mass (TSM), total body fat mass (TBF), appendicular skeletal muscle mass (ASM), and appendicular fat mass (AFM). TSM/height2 (TSMI), ASM/height2 (ASMI), TSM/weight (TSMR), ASM/weight (ASMR), and the ratio of TSM/TBF and ASM/AFM were calculated. Handgrip strength (HGS) was measured using the Jamar® Plus+ Hand Dynamometer. A multiple regression analysis after adjustment for covariates and multiple test correction showed some operational taxonomic units in partial least squares models identified by Multivariate methods with Unbiased Variable selection analysis such as genera of Faecalibacterium, Lachnospira, Lachnospiraceae_ND3007_group, and Lachnospiraceae_UCG-004 were positively correlated with at least one measure of TSM, TSMI, ASM, ASMI, and ASMI Z-score (β: 0.103-0.143, pFDR : .008-.032) but negatively correlated with at least one measure of TSMR, TSM/TBF, ASMR, ASM/AFM, and ASMR Z-score (β: -0.185 to 0.124, pFDR = .008-.045). Children with higher fecal butyric acid, acetic acid, and total SCFA levels exhibited higher TSM, ASM, TSMI, ASMI, and ASMI Z-score and lower TSM/TBF, ASM/AFM, TSMR, ASMR, and ASMR Z-score. However, after additional adjustment for TBF or body mass index, only the associations for Faecalitalea and Pyramidobacter still existed. Mediation analysis suggested that total body fat significantly mediated 66.3%-95.3% of the estimated association of microbiota and SCFAs with TSM, ASM, and ASMI Z-score. Our results suggest that the associations of gut microbiota and SCFAs with skeletal muscle quality in children may largely depend upon on total body fat content.

[Source](https://doi.org/10.1096/fj.202002697RRR)

10.1096/fj.202002697RRR

---

## Characteristics of Fecal Microbiota and Machine Learning Strategy for Fecal Invasive Biomarkers in Pediatric Inflammatory Bowel Disease.
 25.12.2021, _X Wang, Y Xiao, X Xu, L Guo, Y Yu, N Li, C Xu_


Early diagnosis and treatment of pediatric Inflammatory bowel disease (PIBD) is challenging due to the complexity of the disease and lack of disease specific biomarkers. The novel machine learning (ML) technique may be a useful tool to provide a new route for the identification of early biomarkers for the diagnosis of PIBD.

[Source](https://doi.org/10.3389/fcimb.2021.711884)

10.3389/fcimb.2021.711884

---

## The microbial <em>gbu</em> gene cluster links cardiovascular disease risk associated with red meat consumption to microbiota L-carnitine catabolism

[Source](https://doi.org/10.1038/s41564-021-01010-x)

---

## Gut Microbiota Exceeds Cervical Microbiota for Early Diagnosis of Endometriosis.
 25.12.2021, _L Huang, B Liu, Z Liu, W Feng, M Liu, Y Wang, D Peng, X Fu, H Zhu, Z Cui, L Xie, Y Ma_


The diagnosis of endometriosis is typically delayed by years for the unexclusive symptom and the traumatic diagnostic method. Several studies have demonstrated that gut microbiota and cervical mucus potentially can be used as auxiliary diagnostic biomarkers. However, none of the previous studies has compared the robustness of endometriosis classifiers based on microbiota of different body sites or demonstrated the correlation among microbiota of gut, cervical mucus, and peritoneal fluid of endometriosis, searching for alternative diagnostic approaches. Herein, we enrolled 41 women (control, n = 20; endometriosis, n = 21) and collected 122 well-matched samples, derived from feces, cervical mucus, and peritoneal fluid, to explore the nature of microbiome of endometriosis patients. Our results indicated that microbial composition is remarkably distinguished between three body sites, with 19 overlapped taxa. Moreover, endometriosis patients harbor distinct microbial communities versus control group especially in feces and peritoneal fluid, with increased abundance of pathogens in peritoneal fluid and depletion of protective microbes in feces. Particularly, genera of Ruminococcus and Pseudomonas were identified as potential biomarkers in gut and peritoneal fluid, respectively. Furthermore, novel endometriosis classifiers were constructed based on taxa selected by a robust machine learning method. These results demonstrated that gut microbiota exceeds cervical microbiota in diagnosing endometriosis. Collectively, this study reveals important insights into the microbial profiling in different body sites of endometriosis, which warrant future exploration into the role of microbiota in endometriosis and highlighted values on gut microbiota in early diagnosis of endometriosis.

[Source](https://doi.org/10.3389/fcimb.2021.788836)

10.3389/fcimb.2021.788836

---

## Natural selection for imprecise vertical transmission in host-microbiota systems.
 24.12.2021, _M Bruijning, LP Henry, SKG Forsberg, CJE Metcalf, JF Ayroles_


How and when the microbiome modulates host adaptation remains an evolutionary puzzle, despite evidence that the extended genetic repertoire of the microbiome can shape host phenotypes and fitness. One complicating factor is that the microbiome is often transmitted imperfectly across host generations, leading to questions about the degree to which the microbiome contributes to host adaptation. Here, using an evolutionary model, we demonstrate that decreasing vertical transmission fidelity can increase microbiome variation, and thus phenotypic variation, across hosts. When the most beneficial microbial genotypes change unpredictably from one generation to the next (for example, in variable environments), hosts can maximize fitness by increasing the microbiome variation among offspring, as this improves the chance of there being an offspring with the right microbial combination for the next generation. Imperfect vertical transmission can therefore be adaptive in varying environments. We characterize how selection on vertical transmission is shaped by environmental conditions, microbiome changes during host development and the contribution of other factors to trait variation. We illustrate how environmentally dependent microbial effects can favour intermediate transmission and set our results in the context of examples from natural systems. We also suggest research avenues to empirically test our predictions. Our model provides a basis to understand the evolutionary pathways that potentially led to the wide diversity of microbe transmission patterns found in nature.

[Source](https://doi.org/10.1038/s41559-021-01593-y)

10.1038/s41559-021-01593-y

---

## Comparison of the airway microbiota in children with chronic suppurative lung disease.
 30.12.2021, _B Ahmed, MJ Cox, L Cuthbertson, P James, L Gardner, W Cookson, J Davies, M Moffatt, A Bush_


_Adolescent, Child, Child, Preschool, Cystic Fibrosis, Humans, Infant, Microbiota, RNA, Ribosomal, 16S, Sputum, Thorax_
The airway microbiota is important in chronic suppurative lung diseases, such as primary ciliary dyskinesia (PCD) and cystic fibrosis (CF). This comparison has not previously been described but is important because difference between the two diseases may relate to the differing prognoses and lead to pathological insights and potentially, new treatments.

[Source](https://doi.org/10.1136/bmjresp-2021-001106)

10.1136/bmjresp-2021-001106

---

## VicRK and CovR polymorphisms in <em>Streptococcus mutans</em> strains associated with cardiovascular infections

[Source](https://doi.org/10.1099/jmm.0.001457)

---

## Oral Colonization by <em>Entamoeba gingivalis</em> and <em>Trichomonas tenax</em>: A PCR-Based Study in Health, Gingivitis, and Periodontitis

[Source](https://doi.org/10.3389/fcimb.2021.782805)

---

## Probiotics Evaluation in Oncological Surgery: A Systematic Review of 36 Randomized Controlled Trials Assessing 21 Diverse Formulations.
 26.12.2021, _E Cogo, M Elsayed, V Liang, K Cooley, C Guerin, A Psihogios, P Papadogianis_


Objectives were to evaluate probiotics safety and efficacy in oncological surgery.

[Source](https://doi.org/10.3390/curroncol28060435)

10.3390/curroncol28060435

---

## Untangling human milk oligosaccharides and infant gut microbiome.
 25.12.2021, _AC Masi, CJ Stewart_


The developing gut microbiome in infancy plays a key role in shaping the host immune system and metabolic state, and human milk is the main factor influencing its composition. Human milk does not only serve to feed the baby, but also to help the new-born adapt to its new environment and microbial exposures. Human milk protects the infant by providing multiple bioactive molecules, including human milk oligosaccharides (HMOs), which are the third most abundant solid component after lipids and lactose. The infant is unable to digest HMOs, so they reach the small and large intestines intact where they have many roles, including acting as prebiotics. Bifidobacterium spp. are the main, but not the only, commensals equipped with genes for HMO degradation. In this review we will outline the HMOs structures and functions, list the genes needed for their digestion, and describe the main strategies adopted by bacteria for their utilization.

[Source](https://doi.org/10.1016/j.isci.2021.103542)

10.1016/j.isci.2021.103542

---

## Periodontal Disease: The Good, The Bad, and The Unknown.
 25.12.2021, _LM Sedghi, M Bacino, YL Kapila_


Periodontal disease is classically characterized by progressive destruction of the soft and hard tissues of the periodontal complex, mediated by an interplay between dysbiotic microbial communities and aberrant immune responses within gingival and periodontal tissues. Putative periodontal pathogens are enriched as the resident oral microbiota becomes dysbiotic and inflammatory responses evoke tissue destruction, thus inducing an unremitting positive feedback loop of proteolysis, inflammation, and enrichment for periodontal pathogens. Keystone microbial pathogens and sustained gingival inflammation are critical to periodontal disease progression. However, recent studies have revealed the importance of previously unidentified microbes involved in disease progression, including various viruses, phages and bacterial species. Moreover, newly identified immunological and genetic mechanisms, as well as environmental host factors, including diet and lifestyle, have been discerned in recent years as further contributory factors in periodontitis. These factors have collectively expanded the established narrative of periodontal disease progression. In line with this, new ideologies related to maintaining periodontal health and treating existing disease have been explored, such as the application of oral probiotics, to limit and attenuate disease progression. The role of systemic host pathologies, such as autoimmune disorders and diabetes, in periodontal disease pathogenesis has been well noted. Recent studies have additionally identified the reciprocated importance of periodontal disease in potentiating systemic disease states at distal sites, such as in Alzheimer's disease, inflammatory bowel diseases, and oral cancer, further highlighting the importance of the oral cavity in systemic health. Here we review long-standing knowledge of periodontal disease progression while integrating novel research concepts that have broadened our understanding of periodontal health and disease. Further, we delve into innovative hypotheses that may evolve to address significant gaps in the foundational knowledge of periodontal disease.

[Source](https://doi.org/10.3389/fcimb.2021.766944)

10.3389/fcimb.2021.766944

---

## Clinical implications of preterm infant gut microbiome development.
 24.12.2021, _DB Healy, CA Ryan, RP Ross, C Stanton, EM Dempsey_


Perturbations to the infant gut microbiome during the first weeks to months of life affect growth, development and health. In particular, assembly of an altered intestinal microbiota during infant development results in an increased risk of immune and metabolic diseases that can persist into childhood and potentially into adulthood. Most research into gut microbiome development has focused on full-term babies, but health-related outcomes are also important for preterm babies. The systemic physiological immaturity of very preterm gestation babies (born earlier than 32 weeks gestation) results in numerous other microbiome-organ interactions, the mechanisms of which have yet to be fully elucidated or in some cases even considered. In this Perspective, we compare assembly of the intestinal microbiome in preterm and term infants. We focus in particular on the clinical implications of preterm infant gut microbiome composition and discuss the prospects for microbiome diagnostics and interventions to improve the health of preterm babies.

[Source](https://doi.org/10.1038/s41564-021-01025-4)

10.1038/s41564-021-01025-4

---

## Genomic characterization of the dominating Beta variant carrying vaccinated (Oxford-AstraZeneca) and non-vaccinated COVID-19 patient samples in Bangladesh: A metagenomics and whole genome approach.
 23.12.2021, _MM Rahaman, MMH Sarkar, MS Rahman, MR Islam, I Islam, O Saha, S Akter, TA Banu, I Jahan, MA Habib, B Goswami, L Bari, MA Malek, MS Khan_


Bangladesh experiences a second wave of COVID-19 since March 2021, despite the nationwide vaccination drive with ChAdOx1 (Oxford-AstraZeneca) vaccine from early February 2021. Here, we characterized 19 nasopharyngeal swab (NPS) samples from COVID-19 suspect patients using genomic and metagenomic approach. Screening for SARS-CoV-2 by RT-PCR and metagenomic sequencing revealed 17 samples of COVID-19 positive (vaccinated=10, non-vaccinated=7) and 2 samples of COVID-19 negative. We did not find any significant correlation between associated factors including vaccination status, age or sex of the patients, diversity or abundance of the co-infected organisms/pathogens, and the abundance of SARS-CoV-2. Though the first wave of the pandemic was dominated by clade 20B, Beta,V2 (South African variant) dominated the second wave (January 2021 to May 2021), while the third wave (May 2021 to September 2021) was responsible for Delta variants of the epidemic in Bangladesh including both vaccinated and unvaccinated infections. Noteworthy, the RBD region of S protein of all the isolates harbored similar substitutions including K417N, E484K and N501Y that signify the Beta, while D614G, D215G, D80A, A67V, L18F and A701V substitutions were commonly found in the non-RBD region of Spike proteins. ORF7b and ORF3a genes underwent a positive selection (dN/dS ratio 1.77 and 1.24, respectively), while the overall S protein of the Bangladeshi SARS-CoV-2 isolates underwent negative selection pressure (dN/dS=0.621). Furthermore, we found different bacterial co-infection like Streptococcus agalactiae, Neisseria meningitidis, Elizabethkingia anophelis, Stenotrophomonas maltophilia, Klebsiella pneumoni and Pseudomonas plecoglossicida, expressing a number of antibiotic resistance genes such as tetA and tetM. Overall, this approach provides valuable insights on the SARS-CoV-2 genomes and microbiome composition from both vaccinated and non-vaccinated patients in Bangladesh. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1002/jmv.27537)

10.1002/jmv.27537

---

## Concepts and Consequences of a Core Gut Microbiota for Animal Growth and Development.
 23.12.2021, _D Perlman, M Martínez-Álvaro, S Moraïs, I Altshuler, LH Hagen, E Jami, R Roehe, PB Pope, I Mizrahi_


Animal microbiomes are occasionally considered as an extension of host anatomy, physiology, and even their genomic architecture. Their compositions encompass variable and constant portions when examined across multiple hosts. The latter, termed the core microbiome, is viewed as more accommodated to its host environment and suggested to benefit host fitness. Nevertheless, discrepancies in its definitions, characteristics, and importance to its hosts exist across studies. We survey studies that characterize the core microbiome, detail its current definitions and available methods to identify it, and emphasize the crucial need to upgrade and standardize the methodologies among studies. We highlight ruminants as a case study and discuss the link between the core microbiome and host physiology and genetics, as well as potential factors that shape it. We conclude with main directives of action to better understand the host-core microbiome axis and acquire the necessary insights into its controlled modulation. Expected final online publication date for the Annual Review of Animal Biosciences, Volume 10 is February 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.

[Source](https://doi.org/10.1146/annurev-animal-013020-020412)

10.1146/annurev-animal-013020-020412

---

## Playing by their own rules: the vertebrate archaeome.
 24.12.2021, _SM Hird_


In a recent paper, Youngblut et al. used comparative methods to study the archaeal component of the vertebrate microbiome. This study showed that the vertebrate archaeome contains abundant variation, differs from the vertebrate bacteriome and lays a foundation for future comparative studies.

[Source](https://doi.org/10.1016/j.tim.2021.12.002)

10.1016/j.tim.2021.12.002

---

## Integration of whole genome sequencing and transcriptomics reveals a complex picture of the reestablishment of insecticide resistance in the major malaria vector <em>Anopheles coluzzii</em>

[Source](https://doi.org/10.1371/journal.pgen.1009970)

---

## Comparative Study of Intestinal Microbiota Composition of Six Edible Fish Species.
 25.12.2021, _T Ofek, M Lalzar, S Laviad-Shitrit, I Izhaki, M Halpern_


Intensive freshwater aquaculture in the Spring Valley, Israel, is implemented mainly in earthen fishponds and reservoirs that are stocked with a variety of edible fish species. Here we sampled six different healthy fish species from these intensive aquacultures. The fish were hybrid striped bass, European bass, red drum (all carnivores), hybrid tilapia, flathead grey mullet (both herbivores), and common carp (an omnivore). Significant differences were found among the intestinal microbiota of the six studied fish species. The microbiota composition diversity was strongly related to the trophic level of the fish, such that there was a significant difference between the carnivore and the herbivore species, while the omnivore species was not significantly different from either group. The most abundant genus in the majority of the fishes' intestinal microbiota was Cetobacterium. Furthermore, we found that beside Cetobacterium, a unique combination of taxa with relative abundance &gt;10% characterized the intestine microbiota of each fish species: unclassified Mycoplasmataceae, Aeromonas, and Vibrio (hybrid striped bass); Turicibacter and Clostridiaceae 1 (European bass); Vibrio (red drum); ZOR0006-Firmicutes (hybrid tilapia); unclassified Mycoplasmataceae and unclassified Vibrionaceae (flathead grey mullet); and Aeromonas (common carp). We conclude that each fish species has a specific bacterial genera combination that characterizes it. Moreover, diet and the trophic level of the fish have a major influence on the gut microbiota of healthy fish that grow in intensive freshwater aquaculture.

[Source](https://doi.org/10.3389/fmicb.2021.760266)

10.3389/fmicb.2021.760266

---

## Captivity Influences the Gut Microbiome of <em>Rhinopithecus roxellana</em>

[Source](https://doi.org/10.3389/fmicb.2021.763022)

---

## Assessment of Changes in the Oral Microbiome That Occur in Dogs with Periodontal Disease.
 26.12.2021, _R Santibáñez, C Rodríguez-Salas, C Flores-Yáñez, D Garrido, P Thomson_


The oral microbiome in dogs is a complex community. Under some circumstances, it contributes to periodontal disease, a prevalent inflammatory disease characterized by a complex interaction between oral microbes and the immune system. Porphyromonas and Tannerella spp. are usually dominant in this disease. How the oral microbiome community is altered in periodontal disease, especially sub-dominant microbial populations is unclear. Moreover, how microbiome functions are altered in this disease has not been studied. In this study, we compared the composition and the predicted functions of the microbiome of the cavity of healthy dogs to those with from periodontal disease. The microbiome of both groups clustered separately, indicating important differences. Periodontal disease resulted in a significant increase in Bacteroidetes and reductions in Actinobacteria and Proteobacteria. Porphyromonas abundance increased 2.7 times in periodontal disease, accompanied by increases in Bacteroides and Fusobacterium. It was predicted that aerobic respiratory processes are decreased in periodontal disease. Enrichment in fermentative processes and anaerobic glycolysis were suggestive of an anaerobic environment, also characterized by higher lipopolysaccharide biosynthesis. This study contributes to a better understanding of how periodontal disease modifies the oral microbiome and makes a prediction of the metabolic pathways that contribute to the inflammatory process observed in periodontal disease.

[Source](https://doi.org/10.3390/vetsci8120291)

10.3390/vetsci8120291

---

## Evaluation of Microbiome Alterations Following Consumption of BIOHM, a Novel Probiotic.
 24.12.2021, _MA Ghannoum, TS McCormick, M Retuerto, G Bebek, S Cousineau, L Hartman, C Barth, K Schrom_


Gastrointestinal microbiome dysbiosis may result in harmful effects on the host, including those caused by inflammatory bowel diseases (IBD). The novel probiotic BIOHM, consisting of Bifidobacterium breve, Saccharomyces boulardii, Lactobacillus acidophilus, L. rhamnosus, and amylase, was developed to rebalance the bacterial-fungal gut microbiome, with the goal of reducing inflammation and maintaining a healthy gut population. To test the effect of BIOHM on human subjects, we enrolled a cohort of 49 volunteers in collaboration with the Fermentation Festival group (Santa Barbara, CA, USA). The profiles of gut bacterial and fungal communities were assessed via stool samples collected at baseline and following 4 weeks of once-a-day BIOHM consumption. Mycobiome analysis following probiotic consumption revealed an increase in Ascomycota levels in enrolled individuals and a reduction in Zygomycota levels (p value &lt; 0.01). No statistically significant difference in Basidiomycota was detected between pre- and post-BIOHM samples and control abundance profiles (p &gt; 0.05). BIOHM consumption led to a significant reduction in the abundance of Candida genus in tested subjects (p value &lt; 0.013), while the abundance of C. albicans also trended lower than before BIOHM use, albeit not reaching statistical significance. A reduction in the abundance of Firmicutes at the phylum level was observed following BIOHM use, which approached levels reported for control individuals reported in the Human Microbiome Project data. The preliminary results from this clinical study suggest that BIOHM is capable of significantly rebalancing the bacteriome and mycobiome in the gut of healthy individuals, suggesting that further trials examining the utility of the BIOHM probiotic in individuals with gastrointestinal symptoms, where dysbiosis is considered a source driving pathogenesis, are warranted.

[Source](https://doi.org/10.3390/cimb43030148)

10.3390/cimb43030148

---

## <em>Lactobacillus rhamnosus</em> attenuates Thai chili extracts induced gut inflammation and dysbiosis despite capsaicin bactericidal effect against the probiotics, a possible toxicity of high dose capsaicin

[Source](https://doi.org/10.1371/journal.pone.0261189)

---

## May the dark be with roots: A perspective on how root illumination may bias <em>in vitro</em> research on plant-environment interactions

[Source](https://doi.org/10.1111/nph.17936)

---

## Site and land-use associations of soil bacteria and fungi define core and indicative taxa.
 23.12.2021, _G Florian, H Martin, M Johanna, H Anna-Sofia, E Jürg, G Andreas, M Reto G, F Beat, W Franco_


Soil microbial diversity has major influences on ecosystem functions and services. However, due to its complexity and uneven distribution of abundant and rare taxa, quantification of soil microbial diversity remains challenging and thereby impeding its integration into long-term monitoring programs. Using metabarcoding, we analyzed soil bacterial and fungal communities at thirty long-term soil monitoring sites from the three land-use types arable land, permanent grassland, and forest with a yearly sampling between snowmelt and first fertilization over five years. Unlike soil microbial biomass and alpha-diversity, microbial community compositions and structures were site- and land-use-specific with CAP reclassification success rates of 100%. The temporally stable site core communities included 38.5% of bacterial and 33.1% of fungal OTUs covering 95.9% and 93.2% of relative abundances. We characterized bacterial and fungal core communities and their land-use associations at the family-level. In general, fungal families revealed stronger land-use associations as compared to bacteria. This is likely due to a stronger vegetation effect on fungal core taxa, while bacterial core taxa were stronger related to soil properties. The assessment of core communities can be used to form cultivation-independent reference lists of microbial taxa, which may facilitate the development of microbial indicators for soil quality and the use of soil microbiota for long-term soil biomonitoring.

[Source](https://doi.org/10.1093/femsec/fiab165)

10.1093/femsec/fiab165

---

## tascCODA: Bayesian Tree-Aggregated Analysis of Compositional Amplicon and Single-Cell Data.
 25.12.2021, _J Ostner, S Carcy, CL Müller_


Accurate generative statistical modeling of count data is of critical relevance for the analysis of biological datasets from high-throughput sequencing technologies. Important instances include the modeling of microbiome compositions from amplicon sequencing surveys and the analysis of cell type compositions derived from single-cell RNA sequencing. Microbial and cell type abundance data share remarkably similar statistical features, including their inherent compositionality and a natural hierarchical ordering of the individual components from taxonomic or cell lineage tree information, respectively. To this end, we introduce a Bayesian model for tree-aggregated amplicon and single-cell compositional data analysis (tascCODA) that seamlessly integrates hierarchical information and experimental covariate data into the generative modeling of compositional count data. By combining latent parameters based on the tree structure with spike-and-slab Lasso penalization, tascCODA can determine covariate effects across different levels of the population hierarchy in a data-driven parsimonious way. In the context of differential abundance testing, we validate tascCODA's excellent performance on a comprehensive set of synthetic benchmark scenarios. Our analyses on human single-cell RNA-seq data from ulcerative colitis patients and amplicon data from patients with irritable bowel syndrome, respectively, identified aggregated cell type and taxon compositional changes that were more predictive and parsimonious than those proposed by other schemes. We posit that tascCODA constitutes a valuable addition to the growing statistical toolbox for generative modeling and analysis of compositional changes in microbial or cell population data.

[Source](https://doi.org/10.3389/fgene.2021.766405)

10.3389/fgene.2021.766405

---

