## Metataxonomic investigation of the microbial community in the trachea and oropharynx of healthy controls and diabetic patients using endotracheal tubes.
 05.11.2021, _SY Cho, JH Choi, SH Lee, YS Choi, SW Hwang, YJ Kim_


Although the study of respiratory microbiota has been an active field of research, obtaining the appropriate respiratory samples for healthy controls remains to be a challenge. As such, this study aims to evaluate the use of endotracheal tube washing as a viable control for sputum samples.

[Source](https://doi.org/10.1371/journal.pone.0259596)

10.1371/journal.pone.0259596

---

## Characterization of the endometrial, cervicovaginal and anorectal microbiota in post-menopausal women with endometrioid and serous endometrial cancers.
 05.11.2021, _GM Gressel, M Usyk, M Frimer, DYS Kuo, RD Burk_


To characterize the microbiota of postmenopausal women undergoing hysterectomy for endometrioid (EAC) or uterine serous cancers (USC) compared to controls with non-malignant conditions.

[Source](https://doi.org/10.1371/journal.pone.0259188)

10.1371/journal.pone.0259188

---

## Fecal Microbiota Transplantation Relieves Gastrointestinal and Autism Symptoms by Improving the Gut Microbiota in an Open-Label Study

[Source](https://doi.org/10.3389/fcimb.2021.759435)

---

## Evaluation of vaginal microbiota in women admitted to the hospital for premature labour.
 05.11.2021, _B Arena, MD Daccò_


The aim of this study was to evaluate the vaginal microbiota of women admitted to the hospital for premature labour and to compare the flora of those who responded to the tocolytic treatment with the flora of those who did not respond to the treatment and delivered prematurely Materials: the hospital records of 245 women admitted to the division of Obstetrics and Gynaecology of 'Guglielmo da Saliceto' Hospital in Piacenza for premature labour, between 24 completed weeks and 36 weeks plus 6 days of pregnancy, were reviewed and the results of vaginal swabs collected on admission were evaluated.

[Source](https://doi.org/10.23750/abm.v92i5.9925)

10.23750/abm.v92i5.9925

---

## The vaginal microbiota and innate immunity after local excisional treatment for cervical intraepithelial neoplasia.
 05.11.2021, _A Mitra, DA MacIntyre, M Paraskevaidi, AB Moscicki, V Mahajan, A Smith, YS Lee, D Lyons, E Paraskevaidis, JR Marchesi, PR Bennett, M Kyrgiou_


Vaginal microbiota (VMB) composition is altered in women with cervical intra-epithelial neoplasia (CIN) compared to healthy controls and is associated with disease progression. However, the impact of CIN excision on the VMB and innate immunity is not known. This observational study aims to explore the impact of CIN excision on the VMB, antimicrobial peptides (AMP) and proinflammatory cytokines.

[Source](https://doi.org/10.1186/s13073-021-00977-w)

10.1186/s13073-021-00977-w

---

## Comparative analysis of human facial skin microbiome between topical sites compared to entire face.
 04.11.2021, _H Lee, J Jeong, Y Oh, CJ Lee, S Mun, DG Lee, H Jo, YM Heo, C Baek, CY Heo, SM Kang, K Han_


Skin is an essential outer barrier and supports the growth of commensal microorganisms that protects a host from the offense of foreign toxic organisms. With the rapid development of next-generation sequencing (NGS)-based applications, skin microbiome research for facial health care has reached industry growth, such as therapy and cosmetic product development. Despite the acceleration of skin microbiome research, experimental standardization protocol has not yet been established in the facial site and method of sampling.

[Source](https://doi.org/10.1007/s13258-021-01180-2)

10.1007/s13258-021-01180-2

---

## Lung Tissue Microbiome Is Associated With Clinical Outcomes of Idiopathic Pulmonary Fibrosis.
 05.11.2021, _HY Yoon, SJ Moon, JW Song_


Background: Several studies using bronchoalveolar lavage fluid (BALF) reported that lung microbial communities were associated with the development and clinical outcome of idiopathic pulmonary fibrosis (IPF). However, the microbial communities in IPF lung tissues are not well known. This study is aimed to investigate bacterial microbial communities in lung tissues and determine their impact on the clinical outcomes of patients with IPF. Methods: Genomic DNA extracted from lung tissues of patients with IPF (n = 20; 10 non-survivors) and age- and sex-matched controls (n = 20) was amplified using fusion primers targeting the V3 and V4 regions of the 16S RNA genes with indexing barcodes. Results: Mean age of IPF subjects was 63.3 yr, and 65% were male. Alpha diversity indices did not significantly differ between IPF patients and controls, or between IPF non-survivors and survivors. The relative abundance of Lactobacillus, Paracoccus, and Akkermansia was increased, whereas that of Caulobacter, Azonexus, and Undibacterium decreased in patients with IPF compared with that in the controls. A decreased relative abundance of Pelomonas (odds ratio [OR], 0.352, p = 0.027) and Azonexus (OR, 0.013, p = 0.046) was associated with a diagnosis of IPF in the multivariable logistic analysis adjusted by age and gender. Multivariable Cox analysis adjusted for age and forced vital capacity (FVC) revealed that higher relative abundance of Streptococcus (hazard ratio [HR], 1.993, p = 0.044), Sphingomonas (HR, 57.590, p = 0.024), and Clostridium (HR, 37.189, p = 0.038) was independently associated with IPF mortality. The relative abundance of Curvibacter (r = 0.590) and Thioprofundum (r = 0.373) was correlated positively, whereas that of Anoxybacillus (r = -0.509) and Enterococcus (r = -0.593) was correlated inversely with FVC. In addition, the relative abundance of the Aquabacterium (r = 0.616) and Peptoniphilus (r = 0.606) genera was positively correlated, whereas that of the Fusobacterium (r = -0.464) and Phycicoccus (r = -0.495) genera was inversely correlated with distance during the 6-min walking test. Conclusions: The composition of the microbiome in lung tissues differed between patients with IPF and controls and was associated with the diagnosis, mortality, and disease severity of IPF.

[Source](https://doi.org/10.3389/fmed.2021.744523)

10.3389/fmed.2021.744523

---

## Mechanisms underlying interactions between two abundant oral commensal bacteria.
 04.11.2021, _D Perera, A McLean, V Morillo-López, K Cloutier-Leblanc, E Almeida, K Cabana, J Mark Welch, M Ramsey_


Complex polymicrobial biofilm communities are abundant in nature particularly in the human oral cavity where their composition and fitness can affect health. While the study of these communities during disease is essential and prevalent, little is known about interactions within the healthy plaque community. Here we describe interactions between two of the most abundant species in this healthy microbiome, Haemophilus parainfluenzae and Streptococcus mitis. We discovered that H. parainfluenzae typically exists adjacent to mitis group streptococci in vivo with which it is also positively correlated based on microbiome data. By comparing in vitro coculture data to ex vivo microscopy we revealed that this co-occurrence is density dependent and further influenced by H2O2 production. We discovered that H. parainfluenzae utilizes a more redundant, multifactorial response to H2O2 than related microorganisms and that this system's integrity enhances streptococcal fitness. Our results indicate that mitis group streptococci are likely the in vivo source of NAD for H. parainfluenzae and also evoke patterns of carbon utilization in vitro for H. parainfluenzae similar to those observed in vivo. Our findings describe mechanistic interactions between two of the most abundant and prevalent members of healthy supragingival plaque that contribute to their in vivo survival.

[Source](https://doi.org/10.1038/s41396-021-01141-3)

10.1038/s41396-021-01141-3

---

## Epigenetic adaptations of the masticatory mucosa to periodontal inflammation.
 04.11.2021, _GM Richter, J Kruppa, HG Keceli, ET Ataman-Duruel, C Graetz, N Pischon, G Wagner, C Rendenbach, Y Jockel-Schneider, O Martins, C Bruckmann, I Staufenbiel, A Franke, RM Nohutcu, S Jepsen, H Dommisch, AS Schaefer_


In mucosal barrier interfaces, flexible responses of gene expression to long-term environmental changes allow adaptation and fine-tuning for the balance of host defense and uncontrolled not-resolving inflammation. Epigenetic modifications of the chromatin confer plasticity to the genetic information and give insight into how tissues use the genetic information to adapt to environmental factors. The oral mucosa is particularly exposed to environmental stressors such as a variable microbiota. Likewise, persistent oral inflammation is the most important intrinsic risk factor for the oral inflammatory disease periodontitis and has strong potential to alter DNA-methylation patterns. The aim of the current study was to identify epigenetic changes of the oral masticatory mucosa in response to long-term inflammation that resulted in periodontitis.

[Source](https://doi.org/10.1186/s13148-021-01190-7)

10.1186/s13148-021-01190-7

---

## Increase of SARS-CoV-2 RNA load in faecal samples prompts for rethinking of SARS-CoV-2 biology and COVID-19 epidemiology.
 05.08.2021, _M Petrillo, C Brogna, S Cristoni, M Querci, O Piazza, G Van den Eede_


_Biology, COVID-19, Feces, Humans, RNA, Viral, SARS-CoV-2_
Background Scientific evidence for the involvement of human microbiota in the development of COVID-19 disease has been reported recently. SARS-CoV-2 RNA presence in human faecal samples and SARS-CoV-2 activity in faeces from COVID-19 patients have been observed. Methods Starting from these observations, an experimental design was developed to cultivate in vitro faecal microbiota from infected individuals, to monitor the presence of SARS-CoV-2, and to collect data on the relationship between faecal bacteria and the virus. Results Our results indicate that SARS-CoV-2 replicates in vitro in bacterial growth medium, that the viral replication follows bacterial growth and it is influenced by the administration of specific antibiotics. SARS-CoV-2-related peptides have been detected in 30-day bacterial cultures and characterised. Discussion Our observations are compatible with a 'bacteriophage-like' behaviour of SARS-CoV-2, which, to our knowledge has not been observed or described before. These results are unexpected and hint towards a novel hypothesis on the biology of SARS-CoV-2 and on the COVID-19 epidemiology. The discovery of possible new modes of action of SARS-CoV-2 has far-reaching implications for the prevention and the treatment of the disease.

[Source](https://doi.org/10.12688/f1000research.52540.2)

10.12688/f1000research.52540.3

---

## Microbiota-derived acetate enables the metabolic fitness of the brain innate immune system during health and disease.
 03.11.2021, _D Erny, N Dokalis, C Mezö, A Castoldi, O Mossad, O Staszewski, M Frosch, M Villa, V Fuchs, A Mayer, J Neuber, J Sosat, S Tholen, O Schilling, A Vlachos, T Blank, M Gomez de Agüero, AJ Macpherson, EJ Pearce, M Prinz_


As tissue macrophages of the central nervous system (CNS), microglia constitute the pivotal immune cells of this organ. Microglial features are strongly dependent on environmental cues such as commensal microbiota. Gut bacteria are known to continuously modulate microglia maturation and function by the production of short-chain fatty acids (SCFAs). However, the precise mechanism of this crosstalk is unknown. Here we determined that the immature phenotype of microglia from germ-free (GF) mice is epigenetically imprinted by H3K4me3 and H3K9ac on metabolic genes associated with substantial functional alterations including increased mitochondrial mass and specific respiratory chain dysfunctions. We identified acetate as the essential microbiome-derived SCFA driving microglia maturation and regulating the homeostatic metabolic state, and further showed that it is able to modulate microglial phagocytosis and disease progression during neurodegeneration. These findings indicate that acetate is an essential bacteria-derived molecule driving metabolic pathways and functions of microglia during health and perturbation.

[Source](https://doi.org/10.1016/j.cmet.2021.10.010)

10.1016/j.cmet.2021.10.010

---

## Phage-delivered CRISPR-Cas9 for strain-specific depletion and genomic deletions in the gut microbiome.
 03.11.2021, _KN Lam, P Spanogiannopoulos, P Soto-Perez, M Alexander, MJ Nalley, JE Bisanz, RR Nayak, AM Weakley, FB Yu, PJ Turnbaugh_


Mechanistic insights into the role of the human microbiome in the predisposition to and treatment of disease are limited by the lack of methods to precisely add or remove microbial strains or genes from complex communities. Here, we demonstrate that engineered bacteriophage M13 can be used to deliver DNA to Escherichia coli within the mouse gastrointestinal (GI) tract. Delivery of a programmable exogenous CRISPR-Cas9 system enables the strain-specific depletion of fluorescently marked isogenic strains during competitive colonization and genomic deletions that encompass the target gene in mice colonized with a single strain. Multiple mechanisms allow E. coli to escape targeting, including loss of the CRISPR array or even the entire CRISPR-Cas9 system. These results provide a robust and experimentally tractable platform for microbiome editing, a foundation for the refinement of this approach to increase targeting efficiency, and a proof of concept for the extension to other phage-bacterial pairs of interest.

[Source](https://doi.org/10.1016/j.celrep.2021.109930)

10.1016/j.celrep.2021.109930

---

## Cell-surface xyloglucan recognition and hydrolysis by the human gut commensal <em>Bacteroides uniformis</em>

[Source](https://doi.org/10.1128/AEM.01566-21)

---

## Dysbiosis of Gut Microbiome Is Associated With Rupture of Cerebral Aneurysms.
 03.11.2021, _S Kawabata, M Takagaki, H Nakamura, H Oki, D Motooka, S Nakamura, T Nishida, E Terada, N Izutsu, T Takenaka, Y Matsui, S Yamada, K Asai, A Tateishi, T Umehara, Y Yano, Y Bamba, K Matsumoto, T Kishikawa, Y Okada, T Iida, H Kishima_


Environmental factors are important with respect to the rupture of cerebral aneurysms. However, the relationship between the gut microbiome, an environmental factor, and aneurysm rupture is unclear. Therefore, we compared the gut microbiome in patients with unruptured intracranial aneurysms (UIAs) and ruptured aneurysms (RAs) to identify the specific bacteria causing the rupture of cerebral aneurysms.

[Source](https://doi.org/10.1161/STROKEAHA.121.034792)

10.1161/STROKEAHA.121.034792

---

## Effect of Long-Term and Short-Term Imbalanced Zn Manipulation on Gut Microbiota and Screening for Microbial Markers Sensitive to Zinc Status.
 03.11.2021, _L Chen, Z Wang, P Wang, X Yu, H Ding, Z Wang, J Feng_


Zinc (Zn) imbalance is a common single-nutrient disorder worldwide, but little is known about the short-term and long-term effects of imbalanced dietary zinc in the intestinal microbiome. Here, 3-week-old C57BL/6 mice were fed diets supplemented with Zn at the doses of 0 (low Zn), 30 (control Zn), 150 (high Zn), and 600 mg/kg of body weight (excess Zn) for 4 weeks (short term) and 8 weeks (long term). The gut bacterial composition at the phyla, genus, and species levels were changed as the result of the imbalanced Zn diet (e.g., Lactobacillus reuteri and Akkermansia muciniphila). Moreover, pathways including carbohydrate, glycan, and nucleotide metabolism were decreased by a short-term low-Zn diet. Valeriate production was suppressed by a long-term low-Zn diet. Pathways such as drug resistance and infectious diseases were upregulated in high- and excess-Zn diets over 4-week and 8-week intervals. Long-term zinc fortification doses, especially at the high-Zn level, suppressed the abundance of short-chain fatty acids (SCFAs)-producing genera as well as the concentrations of metabolites. Finally, Melainabacteria (phylum) and Desulfovibrio sp. strain ABHU2SB (species) were identified to be potential markers for Zn status with high accuracy (area under the curve [AUC], &gt;0.8). Collectively, this study identified significant changes in gut microbial composition and its metabolite concentration in altered Zn-fed mice and the relevant microbial markers for Zn status. IMPORTANCE Zn insufficiency is an essential health problem in developing countries. To prevent the occurrence of zinc deficit, zinc fortification and supplementation are widely used. However, in developed countries, the amounts of Zn consumed often exceed the tolerable upper intake limit. Our results demonstrated that dietary Zn is an essential mediator of microbial community structure and that both Zn deficiency and Zn overdose can generate a dysbiosis in the gut microbiota. Moreover, specific microbial biomarkers of Zn status were identified and correlated with serum Zn level. Our study found that a short-term low-Zn diet (0 mg/kg) and a long-term high-zinc diet (150 mg/kg) had obvious negative effects in a mouse model. Thus, these results indicate that the provision and duration of supplemental Zn should be approached with caution.

[Source](https://doi.org/10.1128/Spectrum.00483-21)

10.1128/Spectrum.00483-21

---

## Metagenomic and network analysis revealed wide distribution of antibiotic resistance genes in monkey gut microbiota.
 06.11.2021, _Y Yan, H Li, A Fayyaz, Y Gai_


The emergence and spread of drug-resistant microorganisms that have acquired new resistance mechanisms, leading to antibiotic resistance, continue to threaten the health of humans and animals worldwide. Non-human primates (NHPs), as close living relatives of human beings in the world, have a high degree of genetic and physiological similarity to humans. However, despite its importance, we lack a comprehensive characterization or understanding of the similarities and differences of the antibiotic resistance genes of the gut microbiome carried by non-human primates and humans. In the present study, the diversity and abundance of antibiotic resistance genes carried by the gut microbiota of cynomolgus monkeys (Macaca fascicularis) were investigated by metagenomic analysis. In total, 60 resistance types conferring resistance to 11 categories of antibiotics were identified in the gut microbiome of cynomolgus monkeys. Interestingly, the composition and abundance of ARGs carried by the gut microbiota of cynomolgus monkeys can be significantly affected by dietary changes. Moreover, we found that all ARG types carried by humans are also present in cynomolgus monkeys. The tetracycline resistance gene tet(37) is evolutionarily conserved and highly homologous. Taken together, our study provides a comprehensive overview of the diversity and richness of ARGs in the gut microbiota of cynomolgus monkeys and underlines the potentially crucial role of diet in the gut health of monkeys and humans.

[Source](https://doi.org/10.1128/mSystems.00717-21)

10.1016/j.micres.2021.126895

---

## Dietary Modulation Alters Susceptibility to<em> Listeria monocytogenes</em> and <em>Salmonella</em> Typhimurium with or without a Gut Microbiota

[Source](https://doi.org/10.1128/mSystems.00717-21)

---

## Alterations in Gut Microbiome Composition and Function in Irritable Bowel Syndrome and Increased Probiotic Abundance with Daily Supplementation.
 05.11.2021, _J Phan, D Nair, S Jain, T Montagne, DV Flores, A Nguyen, S Dietsche, S Gombar, P Cotter_


Irritable bowel syndrome (IBS) is characterized by abdominal discomfort and irregular bowel movements and stool consistency. As such, the gut microbiome has been posited as being influential for the syndrome. However, identifying microbial features associated with IBS symptom heterogeneity is difficult without large cohorts. Our aim was to identify microbial features associated with IBS and IBS subtypes compared to healthy controls and to determine if a synbiotic supplementation intervention could decrease the proportion of those microbial features. Stool samples from 490 individuals with IBS (including all dominant subtypes) and 122 individuals without IBS were analyzed with metagenomic sequencing. One hundred thirty-four IBS subjects were followed over time while receiving daily synbiotic supplementation, the composition of which varied between participants. IBS participants had significantly lower alpha diversity, an enrichment in Gram-negative bacteria, and a reduction in pathways associated with short-chain fatty acid and vitamin synthesis. Shigella species were significantly associated with IBS, while Eubacterium rectale and Faecalibacterium prausnitzii were associated with healthy controls. Random forest identified unique and overlapping microbial features associated with each IBS subtype. Longitudinal assessment of 134 IBS subjects receiving synbiotic supplements demonstrated a significant difference in microbial features and an increase in probiotic abundance across time. We identified microbial features that differentiate healthy and IBS subtypes. Synbiotic supplementation in IBS subjects did not result in alpha diversity change in the microbiome but did demonstrate changes in microbial features. Future work is needed to determine if the observed microbiome changes are associated with IBS symptom improvement. IMPORTANCE An estimated 35 million people in the United States and 11.5% of the population globally are affected by IBS. Immunity, genetics, environment, diet, small intestinal bacterial overgrowth (SIBO), and the gut microbiome are all factors that contribute to the onset or triggers of IBS. With strong supporting evidence that the gut microbiome may influence symptoms associated with IBS, elucidating the important microbes that contribute to the symptoms and severity is important to make decisions for targeted treatment. As probiotics have become more common in treating IBS symptoms, identifying effective probiotics may help inform future studies and treatment.

[Source](https://doi.org/10.1128/mSystems.01215-21)

10.1128/mSystems.01215-21

---

## The Future Is Big-and Small: Remote Sensing Enables Cross-Scale Comparisons of Microbiome Dynamics and Ecological Consequences.
 05.11.2021, _DS Beatty, LR Aoki, OJ Graham, B Yang_


Coupling remote sensing with microbial omics-based approaches provides a promising new frontier for scientists to scale microbial interactions across space and time. These data-rich, interdisciplinary methods allow us to better understand interactions between microbial communities and their environments and, in turn, their impact on ecosystem structure and function. Here, we highlight current and novel examples of applying remote sensing, machine learning, spatial statistics, and omics data approaches to marine, aquatic, and terrestrial systems. We emphasize the importance of integrating biochemical and spatiotemporal environmental data to move toward a predictive framework of microbiome interactions and their ecosystem-level effects. Finally, we emphasize lessons learned from our collaborative research with recommendations to foster productive and interdisciplinary teamwork.

[Source](https://doi.org/10.1128/mSystems.01106-21)

10.1128/mSystems.01106-21

---

## Comprehensive functional core microbiome comparison in genetically obese and lean hosts under the same environment.
 02.11.2021, _M Martínez-Álvaro, A Zubiri-Gaitán, P Hernández, M Greenacre, A Ferrer, A Blasco_


Our study provides an exhaustive comparison of the microbiome core functionalities (captured by 3,936 microbial gene abundances) between hosts with divergent genotypes for intramuscular lipid deposition. After 10 generations of divergent selection for intramuscular fat in rabbits and 4.14 phenotypic standard deviations (SD) of selection response, we applied a combination of compositional and multivariate statistical techniques to identify 122 cecum microbial genes with differential abundances between the lines (ranging from -0.75 to +0.73 SD). This work elucidates that microbial biosynthesis lipopolysaccharides, peptidoglycans, lipoproteins, mucin components, and NADH reductases, amongst others, are influenced by the host genetic determination for lipid accretion in muscle. We also differentiated between host-genetically influenced microbial mechanisms regulating lipid deposition in body or intramuscular reservoirs, with only 28 out of 122 MGs commonly contributing to both. Importantly, the results of this study are of relevant interest for the efficient development of strategies fighting obesity.

[Source](https://doi.org/10.1038/s42003-021-02784-w)

10.1038/s42003-021-02784-w

---

## <em>Vescimonas </em>gen. nov.,<em> Vescimonas coprocola </em>sp. nov.,<em> Vescimonas fastidiosa</em> sp. nov., <em>Pusillimonas</em> gen. nov. and <em>Pusillimonas</em> <em>faecalis </em>sp. nov. isolated from human faeces

[Source](https://doi.org/10.1099/ijsem.0.005066)

---

## Engineering temporal dynamics in microbial communities.
 05.11.2021, _C Ronda, HH Wang_


Microbial communities are a key part to tackling global challenges in human health, environmental conservation, and sustainable agriculture in the coming decade. Recent advances in synthetic biology to study and modify microbial communities have led to important insights into their physiology and ecology. Understanding how targeted changes to microbial communities result in reproducible alterations of the community's intrinsic fluctuations and function is important for mechanistic reconstruction of microbiomes. Studies of synthetic microbial consortia and comparative analysis of communities in normal and disrupted states have revealed ecological principles that can be leveraged to engineer communities towards desired functions. Tools enabling temporal modulation and sensing of the community dynamics offer precise spatiotemporal control of functions, help to dissect microbial interaction networks, and improve predictions of population temporal dynamics. Here we discuss recent advances to manipulate microbiome dynamics through control of specific strain engraftment and abundance, modulation of cell-cell signaling for tuning population dynamics, infiltration of new functions in the existing community with in situ engineering, and in silico modeling of microbial consortia to predict community function and ecology.

[Source](https://doi.org/10.1016/j.mib.2021.10.009)

10.1016/j.mib.2021.10.009

---

## From a parts list to assembly instructions and an operating manual: how small host models can re-write microbiome theory

[Source](https://doi.org/10.1016/j.mib.2021.10.006)

---

## Microbiota-derived extracellular vesicles in interkingdom communication in the gut.
 05.11.2021, _N Díaz-Garrido, J Badia, L Baldomà_


The intestine is fundamental in controlling human health. Intestinal epithelial and immune cells are continuously exposed to millions of microbes that greatly impact on intestinal epithelial barrier and immune function. This microbial community, known as gut microbiota, is now recognized as an important partner of the human being that actively contribute to essential functions of the intestine but also of distal organs. In the gut ecosystem, bidirectional microbiota-host communication does not involve direct cell contacts. Both microbiota and host-derived extracellular vesicles (EVs) are key players of such interkingdom crosstalk. There is now accumulating body of evidence that bacterial secreted vesicles mediate microbiota functions by transporting and delivering into host cells effector molecules that modulate host signalling pathways and cell processes. Consequently, vesicles released by the gut microbiota may have great influence on health and disease. Here we review current knowledge on microbiota EVs and specifically highlight their role in controlling host metabolism, intestinal barrier integrity and immune training.

[Source](https://doi.org/10.1002/jev2.12161)

10.1002/jev2.12161

---

## Multiple roles of short-chain fatty acids in Alzheimer disease.
 04.11.2021, _H Chen, L Meng, L Shen_


Alzheimer disease (AD) is the most common form of neurodegenerative disease in older adults and has a complicated etiology. Recently, the roles of short-chain fatty acids (SCFAs), the main metabolites generated by fermentation of dietary fiber by gut microbiota, in the pathogenesis of AD have attracted considerable interest. This study analyzed the multiple roles of SCFAs in AD pathogenesis from five aspects, including: 1) epigenetic regulation; 2) modulation of neuroinflammation; 3) maintenance of the blood-brain barrier (BBB); 4) regulation of brain metabolism; and 5) interference in amyloid protein formation. According to the currently available evidence, SCFAs, particularly butyrate, cause important biological effects that interfere with the development of AD. However, the effect of other SCFAs, such as propionate, on AD might be either beneficial or harmful to different pathways, indicating that the role of SCFAs in the pathogenesis of AD is rather complicated and warrants further investigations.

[Source](https://doi.org/10.1016/j.nut.2021.111499)

10.1016/j.nut.2021.111499

---

## Role of gut microbiota in functional constipation.
 05.11.2021, _S Zhang, R Wang, D Li, L Zhao, L Zhu_


Functional constipation (FC) is common, yet the etiology is not clear. Accumulating evidence suggests an association between FC and abnormal gut microbiota. The relationship between the gut microbiota and the gut transit is likely bidirectional. This review summarizes the current evidence regarding the impact of gut microbiota on the pathogenesis of FC. By modulating the colonic motility, secretion, and absorption, gut microbiota may contribute to the development of FC through microbial metabolic activities involving bile acids, short-chain fatty acids, 5-hydroxytryptamine, and methane. In support of the key roles of the gut microbiota in FC, treatment with probiotics, prebiotics, synbiotics, and traditional Chinese medicine often result in compositional and functional changes in the gut microbiota. Further studies on the pathogenesis of FC and the therapeutic mechanism of microecological agents will provide a knowledge base for better management of FC.

[Source](https://doi.org/10.1093/gastro/goab035)

10.1093/gastro/goab035

---

## Gut Feelings: The Microbiota-Gut-Brain Axis on Steroids.
 03.11.2021, _SY So, TC Savidge_


The intricate connection between central and enteric nervous systems is well established with emerging evidence linking gut microbiota function as a significant new contributor to gut-brain axis signaling. Several microbial signals contribute to altered gut-brain communications, with steroids representing an important biological class that impacts central and enteric nervous system function. Neuroactive steroids contribute pathologically to neurological disorders, including dementia and depression, by modulating the activity of neuroreceptors. However, limited information is available on the influence of neuroactive steroids on the enteric nervous system and gastrointestinal function. In this review, we outline how steroids can modulate enteric nervous system function by focusing on their influence on different receptors that are present in the intestine in health and disease. We also highlight the potential role of the gut microbiota in modulating neuroactive steroid signaling along the gut-brain axis.

[Source](https://doi.org/10.1152/ajpgi.00294.2021)

10.1152/ajpgi.00294.2021

---

## Urobiome: An outlook on the metagenome of urological diseases.
 03.11.2021, _R Shoemaker, J Kim_


The urinary tract likely plays a role in the development of various urinary diseases due to the recently recognized notion that urine is not sterile. In this mini review, we summarize the current literature regarding the urinary microbiome and mycobiome and its relationship to various urinary diseases. It has been recently discovered that the healthy urinary tract contains a host of microorganisms, creating a urinary microbiome. The relative abundance and type of bacteria varies, but generally, deviations in the standard microbiome are observed in individuals with urologic diseases, such as bladder cancer, benign prostatic hyperplasia, urgency urinary incontinence, overactive bladder syndrome, interstitial cystitis, bladder pain syndrome, and urinary tract infections. However, whether this change is causative, or correlative has yet to be determined. In summary, the urinary tract hosts a complex microbiome. Changes in this microbiome may be indicative of urologic diseases and can be tracked to predict, prevent, and treat them in individuals. However, current analytical and sampling collection methods may present limitations to the development in the understanding of the urinary microbiome and its relationship with various urinary diseases. Further research on the differences between healthy and diseased microbiomes, the long-term effects of antibiotic treatments on the urobiome, and the effect of the urinary mycobiome on general health will be important in developing a comprehensive understanding of the urinary microbiome and its relationship to the human body.

[Source](https://doi.org/10.4111/icu.20210312)

10.4111/icu.20210312

---

## Utilizing a reductionist model to study host-microbe interactions in intestinal inflammation.
 04.11.2021, _AM Tsou, JA Goettel, B Bao, A Biswas, YH Kang, NS Redhu, K Peng, GG Putzel, J Saltzman, R Kelly, J Gringauz, J Barends, M Hatazaki, SM Frei, R Emani, Y Huang, Z Shen, JG Fox, JN Glickman, BH Horwitz, SB Snapper_


The gut microbiome is altered in patients with inflammatory bowel disease, yet how these alterations contribute to intestinal inflammation is poorly understood. Murine models have demonstrated the importance of the microbiome in colitis since colitis fails to develop in many genetically susceptible animal models when re-derived into germ-free environments. We have previously shown that Wiskott-Aldrich syndrome protein (WASP)-deficient mice (Was-/-) develop spontaneous colitis, similar to human patients with loss-of-function mutations in WAS. Furthermore, we showed that the development of colitis in Was-/- mice is Helicobacter dependent. Here, we utilized a reductionist model coupled with multi-omics approaches to study the role of host-microbe interactions in intestinal inflammation.

[Source](https://doi.org/10.1186/s40168-021-01161-3)

10.1186/s40168-021-01161-3

---

## Disruption of autophagy by increased 5-HT alters gut microbiota and enhances susceptibility to experimental colitis and Crohn's disease.
 05.11.2021, _S Haq, H Wang, J Grondin, S Banskota, JK Marshall, II Khan, U Chauhan, F Cote, YH Kwon, D Philpott, JH Brumell, M Surette, GR Steinberg, WI Khan_


[Figure: see text].

[Source](https://doi.org/10.1126/sciadv.abi6442)

10.1126/sciadv.abi6442

---

## Differences in the gut microbiota composition of rats fed with soybean protein and their derived peptides.
 03.11.2021, _W Li, H Li, Y Zhang, C Zhang, J Zhang, X Liu_


Current studies regarding the effect of different nitrogen sources on gut microbiota have thus far disregarded the ability of probiotics and coliforms to compete for protein. This study aimed to investigate the differences in the utilization of soybean protein (SPro) and its derived peptides (SPep) by the gut microbiota of Sprague Dawley (SD) rats. The SPro and SPep prepared in this study showed extensive SPro molecular weight distribution, while that of SPep was minimal, ranging between 150 and 1000 Da and primarily consisting of two to five amino acids. The cecum microflora composition of the rats was determined via 16S rDNA amplicon sequencing, showing that the SPro and SPep significantly increased the abundance and uniformity of the gut microbiota after 35 days of feeding. The Firmicutes/Bacteroidetes (F/B) ratios of the SPep, SPro, and casein groups were 2.49 ± 0.60, 2.98 ± 1.12, and 2.59 ± 0.74, respectively. Although the rats fed with SPro and SPep displayed similar gut microbiome structures, SPep significantly promoted Lactobacillus and Phascolarctobacterium growth. The results showed that SPep significantly increased the diversity of the gut microbiota and elevated the probiotic proportion. PRACTICAL APPLICATION: SPro and SPep are two nutritious and high-quality nitrogen sources. The results showed that SPro and SPep regulated the structure of gut microbiota in rats, and the effect of SPep was better. This study provides a theoretical basis for developing SPep functional foods able to regulate gut microbiota and maintain health.

[Source](https://doi.org/10.1111/1750-3841.15948)

10.1111/1750-3841.15948

---

## Unraveling differences in fecal microbiota stability in mammals: from high variable carnivores and consistently stable herbivores.
 05.11.2021, _F Zoelzer, AL Burger, PW Dierkes_


Through the rapid development in DNA sequencing methods and tools, microbiome studies on a various number of species were performed during the last decade. This advance makes it possible to analyze hundreds of samples from different species at the same time in order to obtain a general overview of the microbiota. However, there is still uncertainty on the variability of the microbiota of different animal orders and on whether certain bacteria within a species are subject to greater fluctuations than others. This is largely due to the fact that the analysis in most extensive comparative studies is based on only a few samples per species or per study site. In our study, we aim to close this knowledge gap by analyzing multiple individual samples per species including two carnivore suborders Canoidea and Feloidea as well as the orders of herbivore Perissodactyla and Artiodactyla held in different zoos. To assess microbial diversity, 621 fecal samples from 31 species were characterized by sequencing the V3-V4 region of the 16S rRNA gene using Illumina MiSeq.

[Source](https://doi.org/10.1186/s42523-021-00141-0)

10.1186/s42523-021-00141-0

---

## The Role Transition of Dietary Species Richness in Modulating the Gut Microbial Assembly and Postweaning Performance of a Generalist Herbivore.
 05.11.2021, _G Li, C Shi, Y Song, H Chu, Z Zhang_


When facing a food shortage, generalist herbivores can respond by expanding their dietary species richness (DSR) to maximize energy collection, regardless of whether forages are preferred or not. Higher DSR usually indicates higher nutrient adequacy and better health. However, the high-DSR diet containing a large proportion of preferred species or a large proportion of less-preferred species means different things to an animal. It is still unknown how different shift patterns in DSR would affect distinctly the performance of animals via altering gut microbiota. We examined the gut microbial composition, diversity, community assembly processes, and performance of a generalist herbivore, Lasiopodomys brandtii, in a feeding experiment with increased levels of simulated DSR shifting from preferred plant species to less preferred ones. We found the survival rate and body growth of Brandt's voles showed a dome-shaped association with DSR: species performance increased initially with the increase of preferred plant species but declined with the increase of less-preferred food items. Several microbial taxa and functions closely related to the metabolism of amino acids and short-chain fatty acids also showed a dome-shaped association with DSR, which is consistent with the observation of performance change. However, the alpha diversities of gut microbiota increased linearly with DSR. The null model and phylogenetic analysis suggested that stochastic processes dominate at low DSR diets, whereas deterministic processes prevail at high DSR diets. These results suggest that the role of DSR in regulating animal performance by gut microbiota depends on the number of preferred forage items. IMPORTANCE The plant species diversity varies greatly under the influence of both climate change and human disturbance, which may negatively affect the productivity as well as the variability of organisms (e.g., small herbivores) at the next trophic level. It is still unknown how gut microbiota of small herbivores respond to such changes in dietary species richness. Our manipulative food experiment revealed that dietary species richness can affect the composition, functions, and community assembly of gut microbiota of Brandt's vole in a nonlinear way. Given the fast-growing interest in therapeutic diets to treat dysbiosis and to improve health conditions, our study highlights the need to consider not just the variety of consumed food but also the principles of rational nutrition.

[Source](https://doi.org/10.1128/mSystems.00979-21)

10.1128/mSystems.00979-21

---

## Age Associated Microbiome and Microbial Metabolites Modulation and Its Association With Systemic Inflammation in a Rhesus Macaque Model

[Source](https://doi.org/10.3389/fimmu.2021.748397)

---

## Strain-level analysis reveals the vertical microbial transmission during the life cycle of bumblebee.
 04.11.2021, _Q Su, Q Wang, X Mu, H Chen, Y Meng, X Zhang, L Zheng, X Hu, Y Zhai, H Zheng_


Microbial acquisition and development of the gut microbiota impact the establishment of a healthy host-microbes symbiosis. Compared with other animals, the eusocial bumblebees and honeybees possess a simple, recurring, and similar set of gut microbiota. However, all bee gut phylotypes have high strain-level diversity. Gut communities of different bee species are composed of host-specific groups of strains. The variable genomic regions among strains of the same species often confer critical functional differences, such as carbon source utilization, essential for the natural selection of specific strains. The annual bumblebee colony founded by solitary queens enables tracking the transmission routes of gut bacteria during development stages.

[Source](https://doi.org/10.1186/s40168-021-01163-1)

10.1186/s40168-021-01163-1

---

## Social convergence of gut microbiomes in vampire bats.
 06.11.2021, _K Yarlagadda, I Razik, RS Malhi, GG Carter_


The 'social microbiome' can fundamentally shape the costs and benefits of group-living, but understanding social transmission of microbes in free-living animals is challenging due to confounding effects of kinship and shared environments (e.g. highly associated individuals often share the same spaces, food and water). Here, we report evidence for convergence towards a social microbiome among introduced common vampire bats, Desmodus rotundus, a highly social species in which adults feed only on blood, and engage in both mouth-to-body allogrooming and mouth-to-mouth regurgitated food sharing. Shotgun sequencing of samples from six zoos in the USA, 15 wild-caught bats from a colony in Belize and 31 bats from three colonies in Panama showed that faecal microbiomes were more similar within colonies than between colonies. To assess microbial transmission, we created an experimentally merged group of the Panama bats from the three distant sites by housing these bats together for four months. In this merged colony, we found evidence that dyadic gut microbiome similarity increased with both clustering and oral contact, leading to microbiome convergence among introduced bats. Our findings demonstrate that social interactions shape microbiome similarity even when controlling for past social history, kinship, environment and diet.

[Source](https://doi.org/10.1098/rsbl.2021.0389)

10.1098/rsbl.2021.0389

---

## Shrimp AHPND Causing <em>Vibrio anguillarum</em> Infection: Quantitative Diagnosis and Identifying Antagonistic Bacteria

[Source](https://doi.org/10.1007/s10126-021-10079-8)

---

## Naturally occurring fire coral clones demonstrate a genetic and environmental basis of microbiome composition.
 05.11.2021, _CE Dubé, M Ziegler, A Mercière, E Boissin, S Planes, CA Bourmaud, CR Voolstra_


Coral microbiomes are critical to holobiont functioning, but much remains to be understood about how prevailing environment and host genotype affect microbial communities in ecosystems. Resembling human identical twin studies, we examined bacterial community differences of naturally occurring fire coral clones within and between contrasting reef habitats to assess the relative contribution of host genotype and environment to microbiome structure. Bacterial community composition of coral clones differed between reef habitats, highlighting the contribution of the environment. Similarly, but to a lesser extent, microbiomes varied across different genotypes in identical habitats, denoting the influence of host genotype. Predictions of genomic function based on taxonomic profiles suggest that environmentally determined taxa supported a functional restructuring of the microbial metabolic network. In contrast, bacteria determined by host genotype seemed to be functionally redundant. Our study suggests microbiome flexibility as a mechanism of environmental adaptation with association of different bacterial taxa partially dependent on host genotype.

[Source](https://doi.org/10.1038/s41467-021-26543-x)

10.1038/s41467-021-26543-x

---

## Dry and Rainy Seasons Significantly Alter the Gut Microbiome Composition and Reveal a Key <em>Enterococcus</em> sp. (Lactobacillales: Enterococcaceae) Core Component in <em>Spodoptera frugiperda</em> (Lepidoptera: Noctuidae) Corn Strain From Northwestern Colombia

[Source](https://doi.org/10.1093/jisesa/ieab076)

---

## Invasive Amphibian Gut Microbiota and Functions Shift Differentially in an Expanding Population but Remain Conserved Across Established Populations

[Source](https://doi.org/10.1007/s00248-021-01896-4)

---

## Microbiome Studies in Non-human Primates.
 06.11.2021, _JM Brenchley, AM Ortiz_


Observations of differing bacterial, intestinal microbiomes in people living with HIV have propelled interest in contributions of the microbiome to HIV disease. Non-human primate (NHP) models of HIV infection provide a controlled setting for assessing contributions of the microbiome by standardizing environmental confounders. We provide an overview of the findings of microbiome contributions to aspects of HIV disease derived from these animal models.

[Source](https://doi.org/10.1007/s11904-021-00584-9)

10.1007/s11904-021-00584-9

---

## Regulation divergences of <em>Lactobacillus fermentum</em> PCC and<em> Lactobacillus paracasei</em> 431 on penicillin-induced upper respiratory tract microbial dysbiosis in BALB/c mice

[Source](https://doi.org/10.1039/d0fo02981e)

---

## Dietary calcium phosphate strongly impacts gut microbiome changes elicited by inulin and galacto-oligosaccharides consumption.
 04.11.2021, _J Fuhren, M Schwalbe, J Boekhorst, C Rösch, HA Schols, M Kleerebezem_


Fructo-oligosaccharides (FOS), inulin, and galacto-oligosaccharides (GOS) are widely recognized prebiotics that profoundly affect the intestinal microbiota, including stimulation of bifidobacteria and lactobacilli, and are reported to elicit several health benefits. The combination of dietary FOS and inulin with calcium phosphate was reported to stimulate commensal Lactobacillus populations and protect the host against pathogenic Enterobacteriaceae, but little is known about the effects of GOS in diets with a different level of calcium phosphate.

[Source](https://doi.org/10.1186/s40168-021-01148-0)

10.1186/s40168-021-01148-0

---

## Effects of growth promoting microorganisms on tomato seedlings growing in different media conditions.
 06.11.2021, _R Pokluda, L Ragasová, M Jurica, A Kalisz, M Komorowska, M Niemiec, A Sekara_


Plant growth-promoting microbes (PGPM) play vital roles in maintaining crop fitness and soil health in stressed environments. Research have included analysis-based cultivation of soil-microbial-plant relationships to clarify microbiota potential. The goal of the research was to (i) evaluate the symbiotic microorganism effects on tomato seedling fitness under stressed conditions simulating a fragile soil susceptible to degradation; (ii) compare the plant-microbial interactions after inoculation with microbial isolates and fungi-bacteria consortia; (iii) develop an effective crop-microbial network, which improves soil and plant status. The experimental design included non-inoculated treatments with peat and sand at ratios of 50:50, 70:30, 100:0 (v:v), inoculated treatments with arbuscular mycorrhizal fungi (AMF) and Azospirillum brasilense (AZ) using the aforementioned peat:sand ratios; and treatment with peat co-inoculated with AMF and Saccharothrix tamanrassetensis (S). AMF + AZ increased root fresh weight in peat substrate compared to the control (4.4 to 3.3 g plant-1). An increase in shoot fresh weight was detected in the AMF + AZ treatment with a 50:50 peat:sand ratio (10.1 to 8.5 g plant-1). AMF + AZ reduced antioxidant activity (DPPH) (18-34%) in leaves, whereas AMF + S had the highest DPPH in leaves and roots (45%). Total leaf phenolic content was higher in control with a decreased proportion of peat. Peroxidase activity was enhanced in AMF + AZ and AMF + S treatments, except for AMF + AZ in peat. Microscopic root assays revealed the ability of AMF to establish strong fungal-tomato symbiosis; the colonization rate was 78-89%. AMF + AZ accelerated K and Mg accumulation in tomato leaves in treatments reflecting soil stress. To date, there has been no relevant information regarding the successful AMF and Saccharothrix co-inoculation relationship. This study confirmed that AMF + S could increase the P, S, and Fe status of seedlings under high organic C content conditions. The improved tomato growth and nutrient acquisition demonstrated the potential of PGPM colonization under degraded soil conditions.

[Source](https://doi.org/10.1371/journal.pone.0259380)

10.1371/journal.pone.0259380

---

## Inter-Kingdom Networks of Canola Microbiome Reveal <em>Bradyrhizobium</em> as Keystone Species and Underline the Importance of Bulk Soil in Microbial Studies to Enhance Canola Production

[Source](https://doi.org/10.1007/s00248-021-01905-6)

---

## City-Scale Meta-Analysis of Indoor Airborne Microbiota Reveals that Taxonomic and Functional Compositions Vary with Building Types.
 05.11.2021, _Y Zhou, MHY Leung, X Tong, JYY Lee, PKH Lee_


Currently, there is a lack of understanding on the variations of the indoor airborne microbiotas of different building types within a city, and how operational taxonomic unit (OTU)- and amplicon sequence variant (ASV)-based analyses of the 16S rRNA gene sequences affect interpretation of the indoor airborne microbiota results. Therefore, in this study, the indoor airborne bacterial microbiotas between commercial buildings, residences, and subways within the same city were compared using both OTU- and ASV-based analytic methods. Our findings suggested that indoor airborne bacterial microbiota compositions were significantly different between building types regardless of the bioinformatics method used. The processes of ecological drift and random dispersal consistently played significant roles in the assembly of the indoor microbiota across building types. Abundant taxa tended to be more centralized in the correlation network of each building type, highlighting their importance. Taxonomic changes between the microbiotas of different building types were also linked to changes in their inferred metabolic function capabilities. Overall, the results imply that customized strategies are necessary to manage indoor airborne bacterial microbiotas for each building type or even within each specific building.

[Source](https://doi.org/10.1021/acs.est.1c03941)

10.1021/acs.est.1c03941

---

## Heavy metals and microbiome are negligible drivers than mobile genetic elements in determining particle-attached and free-living resistomes in the Yellow River.
 04.11.2021, _Q Yu, J Yang, W Su, T Li, T Feng, H Li_


Suspended particles in water can shelter both microorganisms and contaminants. However, the emerging pollutants antibiotic resistance genes (ARGs) in free-living (FL) or particle-attached (PA) bacteria in aquatic environments are less explored. In this study, we compared the free-living and particle-attached ARGs during four seasons in the Yellow River using high-throughput quantitative PCR techniques and 16S rRNA gene sequencing. Our results demonstrated that both the free-living water and particles were dominated by tetracycline and beta-lactamase resistance genes. The PA-ARGs had a higher absolute abundance than FL-ARGs in the Yellow River, regardless of the season. Both PA-ARGs and FL-ARGs had the highest absolute abundance and diversity during winter. Mobile genetic elements (MGEs) were the dominant driver for both size-fractionated ARGs. However, the microbiome had less influence on PA-ARG profiles than the FL-ARG profiles, while the effects of the heavy metals on ARGs were negligible. The community assembly of both FL-ARG and PA-ARG can be explained by neutral processes. Several opportunistic pathogens (e.g., Escherichia coli) associated with human health exhibited a higher relative abundance in the particles than during a free-living lifestyle. Parts of these pathogens were potential ARG hosts. As such, it is important to monitor the ARGs and opportunistic pathogens from size-fractionated bacteria and develop targeted strategies to manage ARG dissemination and opportunistic pathogens to ensure public health.

[Source](https://doi.org/10.1016/j.jhazmat.2021.127564)

10.1016/j.jhazmat.2021.127564

---

## Assessment of the link between evidence and crime scene through soil bacterial and fungal microbiome: A mock case in forensic study.
 04.11.2021, _S Karadayı_


In forensic studies, soil traces can be used to find clues to the origin of an unknown sample or the relationship between a crime scene and a suspect and can provide invaluable evidence as they frequently adhere to objects, with high persistence. In this study, it was aimed to investigate the potential of the bacterial and fungal microbiome diversity of the soil to be used as legitimate evidence in the resolution of homicide cases. Within the scope of a mock homicide case scenario, a total of 12 soil samples were collected, including two evidence samples, three crime scene samples and seven non-crime scene related control samples. Both bacterial and fungal microbiome profiles of these samples were analysed using Illumina NovaSeq platform. The resulting sequences were analysed using QIIME 2 microbiome bioinformatics platform. Beta diversity analysis was performed to determine the difference between samples. In bacterial community analyses, it has been observed that it is difficult to distinguish evidence samples and crime scene samples from control samples at phylum and class level, whereas differentiation could be made at genus and species level. Fungal community analyses allowed to distinguish evidence samples and crime scene samples from control samples at both phylum and class and genus and species level. Principal coordinate analysis (PCoA) results showed that distance between evidence samples and crime scene reference samples was closer to each other than non-crime scene related control samples. The results of this study showed that bacterial and especially fungal DNA in soil has the potential to contribute effectively to the resolution of forensic cases. Thus, it has been understood that it is possible to establish a relationship between the case and the crime scene with the help of microbiome analyses on soil samples obtained in homicide cases.

[Source](https://doi.org/10.1016/j.forsciint.2021.111060)

10.1016/j.forsciint.2021.111060

---

## Bakta: rapid and standardized annotation of bacterial genomes via alignment-free sequence identification.
 05.11.2021, _O Schwengers, L Jelonek, MA Dieckmann, S Beyvers, J Blom, A Goesmann_


Command-line annotation software tools have continuously gained popularity compared to centralized online services due to the worldwide increase of sequenced bacterial genomes. However, results of existing command-line software pipelines heavily depend on taxon-specific databases or sufficiently well annotated reference genomes. Here, we introduce Bakta, a new command-line software tool for the robust, taxon-independent, thorough and, nonetheless, fast annotation of bacterial genomes. Bakta conducts a comprehensive annotation workflow including the detection of small proteins taking into account replicon metadata. The annotation of coding sequences is accelerated via an alignment-free sequence identification approach that in addition facilitates the precise assignment of public database cross-references. Annotation results are exported in GFF3 and International Nucleotide Sequence Database Collaboration (INSDC)-compliant flat files, as well as comprehensive JSON files, facilitating automated downstream analysis. We compared Bakta to other rapid contemporary command-line annotation software tools in both targeted and taxonomically broad benchmarks including isolates and metagenomic-assembled genomes. We demonstrated that Bakta outperforms other tools in terms of functional annotations, the assignment of functional categories and database cross-references, whilst providing comparable wall-clock runtimes. Bakta is implemented in Python 3 and runs on MacOS and Linux systems. It is freely available under a GPLv3 license at https://github.com/oschwengers/bakta. An accompanying web version is available at https://bakta.computational.bio.

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000685)

10.1099/mgen.0.000685

---

## Compositional Data Analysis of Microbiome and Any-Omics Datasets: A Validation of the Additive Logratio Transformation.
 06.11.2021, _M Greenacre, M Martínez-Álvaro, A Blasco_


Microbiome and omics datasets are, by their intrinsic biological nature, of high dimensionality, characterized by counts of large numbers of components (microbial genes, operational taxonomic units, RNA transcripts, etc.). These data are generally regarded as compositional since the total number of counts identified within a sample is irrelevant. The central concept in compositional data analysis is the logratio transformation, the simplest being the additive logratios with respect to a fixed reference component. A full set of additive logratios is not isometric, that is they do not reproduce the geometry of all pairwise logratios exactly, but their lack of isometry can be measured by the Procrustes correlation. The reference component can be chosen to maximize the Procrustes correlation between the additive logratio geometry and the exact logratio geometry, and for high-dimensional data there are many potential references. As a secondary criterion, minimizing the variance of the reference component's log-transformed relative abundance values makes the subsequent interpretation of the logratios even easier. On each of three high-dimensional omics datasets the additive logratio transformation was performed, using references that were identified according to the abovementioned criteria. For each dataset the compositional data structure was successfully reproduced, that is the additive logratios were very close to being isometric. The Procrustes correlations achieved for these datasets were 0.9991, 0.9974, and 0.9902, respectively. We thus demonstrate, for high-dimensional compositional data, that additive logratios can provide a valid choice as transformed variables, which (a) are subcompositionally coherent, (b) explain 100% of the total logratio variance and (c) come measurably very close to being isometric. The interpretation of additive logratios is much simpler than the complex isometric alternatives and, when the variance of the log-transformed reference is very low, it is even simpler since each additive logratio can be identified with a corresponding compositional component.

[Source](https://doi.org/10.3389/fmicb.2021.727398)

10.3389/fmicb.2021.727398

---

## Machine Learning Based Metagenomic Prediction of Inflammatory Bowel Disease.
 05.11.2021, _A Mihajlović, K Mladenović, T Lončar-Turukalo, S Brdar_


In this study, we investigate faecal microbiota composition, in an attempt to evaluate performance of classification algorithms in identifying Inflammatory Bowel Disease (IBD) and its two types: Crohn's disease (CD) and ulcerative colitis (UC). From many investigated algorithms, a random forest (RF) classifier was selected for detailed evaluation in three-class (CD versus UC versus nonIBD) classification task and two binary (nonIBD versus IBD and CD versus UC) classification tasks. We dealt with class imbalance, performed extensive parameter search, dimensionality reduction and two-level classification. In three-class classification, our best model reaches F1 score of 91% in average, which confirms the strong connection of IBD and gastrointestinal microbiome. Among most important features in three-class classification are species Staphylococcus hominis, Porphyromonas endodontalis, Slackia piriformis and genus Bacteroidetes.

[Source](https://doi.org/10.3233/SHTI210591)

10.3233/SHTI210591

---

## Evaluating de Novo Assembly and Binning Strategies for Time Series Drinking Water Metagenomes.
 03.11.2021, _S Vosloo, L Huo, CL Anderson, Z Dai, M Sevillano, A Pinto_


Reconstructing microbial genomes from metagenomic short-read data can be challenging due to the unknown and uneven complexity of microbial communities. This complexity encompasses highly diverse populations, which often includes strain variants. Reconstructing high-quality genomes is a crucial part of the metagenomic workflow, as subsequent ecological and metabolic inferences depend on their accuracy, quality, and completeness. In contrast to microbial communities in other ecosystems, there has been no systematic assessment of genome-centric metagenomic workflows for drinking water microbiomes. In this study, we assessed the performance of a combination of assembly and binning strategies for time series drinking water metagenomes that were collected over 6 months. The goal of this study was to identify the combination of assembly and binning approaches that result in high-quality and -quantity metagenome-assembled genomes (MAGs), representing most of the sequenced metagenome. Our findings suggest that the metaSPAdes coassembly strategies had the best performance, as they resulted in larger and less fragmented assemblies, with at least 85% of the sequence data mapping to contigs greater than 1 kbp. Furthermore, a combination of metaSPAdes coassembly strategies and MetaBAT2 produced the highest number of medium-quality MAGs while capturing at least 70% of the metagenomes based on read recruitment. Utilizing different assembly/binning approaches also assists in the reconstruction of unique MAGs from closely related species that would have otherwise collapsed into a single MAG using a single workflow. Overall, our study suggests that leveraging multiple binning approaches with different metaSPAdes coassembly strategies may be required to maximize the recovery of good-quality MAGs. IMPORTANCE Drinking water contains phylogenetic diverse groups of bacteria, archaea, and eukarya that affect the esthetic quality of water, water infrastructure, and public health. Taxonomic, metabolic, and ecological inferences of the drinking water microbiome depend on the accuracy, quality, and completeness of genomes that are reconstructed through the application of genome-resolved metagenomics. Using time series metagenomic data, we present reproducible genome-centric metagenomic workflows that result in high-quality and -quantity genomes, which more accurately signifies the sequenced drinking water microbiome. These genome-centric metagenomic workflows will allow for improved taxonomic and functional potential analysis that offers enhanced insights into the stability and dynamics of drinking water microbial communities.

[Source](https://doi.org/10.1128/Spectrum.01434-21)

10.1128/Spectrum.01434-21

---

## Evaluation of the effect of antibiotics on gut microbiota in early life based on culturomics, SMRT sequencing and metagenomics sequencing methods.
 02.11.2021, _L Wang, J Zhang, M Zhou, Q Chen, X Yang, Y Hou, M Huang, C Man, Y Jiang_


Symbiotic gut microbiota in early life plays a vital role in human health, and changes in its communication and function are associated with various complex disorders. In this study, we analyzed the gut flora communication of 6 infants at 4 months of age and determined the disturbances related to antibiotic treatment. By the culturomics and Single Molecule Real-time sequencing methods, a total of 6234 strains were divided into 16 genera and 45 species. The alpha diversity of culturable microorganisms in amoxicillin-treated infants was significantly less than that in healthy infants (p &lt;0.05), as indicated by Chao 1, observed species and Faith's PD index. According to metagenomics, the dominant genus and species were Bifidobacterium and B. longum in the healthy group. After treatment with amoxicillin, the dominant genus and species shifted to Enterococcus and E. faecium. Based on the functional annotation of metagenomics, amoxicillin affected the metabolic function of the gut microbiome by activating carbohydrate and lipid metabolism and inhibiting amino acid metabolism. Besides, the intake of antibiotics in early life increased the risk of neurodegenerative disease, virus infectious disease and antimicrobial resistance. The Antibiotic Resistance Genes Database annotation result indicated that the abundance of drug-resistance genes in the antibiotic group was higher than that in the healthy group. These genes were associated with resistance to bacitracin, most of which were associated with K. pneumonia. These findings can provide guidance in the clinic on the proper usage of antibiotics.

[Source](https://doi.org/10.1039/d1ay01106e)

10.1039/d1ay01106e

---

