## Dipeptide-catalysed Michael reaction under physiological conditions: Examination of potential bioorthogonality.
 16.03.2024, _F Schuster, BW Grau, HG Xu, A Mokhir, SB Tsogoeva_


Reactions for drug synthesis under cell-like conditions or even inside living cells can potentially be used e.g., to minimize toxic side effects, to maximize bioactive compound efficacy and/or to address drug delivery problems. Those reactions should be bioorthogonal to enable the generation of drug-like compounds with sufficiently good yields. In the known bioorthogonal Michael reactions, using thiols and phosphines as nucleophiles (e.g., in CS and CP bond formation reactions) is very common. No bioorthogonal Michael addition with a carbon nucleophile is known yet. Therefore, the development of such a reaction might be interesting for future drug discovery research. In this work, the metal-free Michael addition between cyclohexanone and various trans-β-nitrostyrenes (CC bond formation reaction), catalysed by a dipeptide salt H-Pro-Phe-O-Na+, was investigated for the first time in the presence of glutathione (GSH) and in phosphate-buffered saline (PBS). We demonstrated that with electron-withdrawing substituents on the aromatic ring and in β-position of the trans-β-nitrostyrene yields up to 64% can be obtained under physiological conditions, indicating a potential bioorthogonality of the studied Michael reaction. In addition, the selected Michael products demonstrated activity against human ovarian cancer cells A2780. This study opens up a new vista for forming bioactive compounds via CC bond formation Michael reactions under physiological (cell-like) conditions.

[Source](https://twitter.com/svetlana_up?ref_src=twsrc%5Etfw)

10.1016/j.bmc.2024.117650

---

## @svetlana_up

[Source](https://twitter.com/svetlana_up?ref_src=twsrc%5Etfw)

---

## https://t.co/LqO6nON3zv

[Source](https://t.co/LqO6nON3zv)

---

## March 8, 2024

[Source](https://twitter.com/outreachy/status/1766100995757531296?ref_src=twsrc%5Etfw)

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## @hutlab

[Source](https://twitter.com/hutlab?ref_src=twsrc%5Etfw)

---

## @Kun_L_H

[Source](https://twitter.com/Kun_L_H?ref_src=twsrc%5Etfw)

---

## @Roche

[Source](https://twitter.com/Roche?ref_src=twsrc%5Etfw)

---

## @gettesfaw

[Source](https://twitter.com/gettesfaw?ref_src=twsrc%5Etfw)

---

## https://t.co/Oytn0cS9ZI

[Source](https://t.co/Oytn0cS9ZI)

---

## pic.twitter.com/hJdw0USPop

[Source](https://t.co/hJdw0USPop)

---

## March 1, 2024

[Source](https://twitter.com/MicrobiomeVIF/status/1763521203740786793?ref_src=twsrc%5Etfw)

---

## Expanding the human gut microbiome atlas of Africa

[Source](https://www.biorxiv.org/content/10.1101/2024.03.13.584859v1)

---

## https://t.co/mlagTMmjIK

[Source](https://t.co/mlagTMmjIK)

---

## pic.twitter.com/cLDfH3zcpc

[Source](https://t.co/cLDfH3zcpc)

---

## March 15, 2024

[Source](https://twitter.com/DylanMaghini/status/1768732511402893611?ref_src=twsrc%5Etfw)

---

## The individual response to antibiotics and diet — insights into gut microbial resilience and host metabolism

[Source](https://www.nature.com/articles/s41574-024-00966-0)

---

## The impact of different antimicrobial exposures on the gut microbiome in the ARMORD observational study

[Source](https://www.medrxiv.org/content/10.1101/2024.03.06.24303874v1)

---

## Multi-omics analysis of the gut microbiome and metabolites associated with the psychoneurological symptom cluster in children with cancer receiving chemotherapy

[Source](https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-024-05066-1)

---

## Longitudinal analysis of the gut microbiome in adolescent patients with anorexia nervosa: microbiome-related factors associated with clinical outcome

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2024.2304158)

---

## The contribution of respiratory microbiome analysis to a treatable traits model of care.
 24.01.2020, _SL Taylor, HE O'Farrell, JL Simpson, IA Yang, GB Rogers_


_Airway Management, Humans, Metagenomics, Microbiological Phenomena, Microbiota, Models, Organizational, Patient Care Management, Pulmonary Disease, Chronic Obstructive, Respiratory System_
The composition of the airway microbiome in patients with chronic airway diseases, such as severe asthma, chronic obstructive pulmonary disease (COPD), bronchiectasis and cystic fibrosis (CF), has the potential to inform a precision model of clinical care. Patients with these conditions share overlapping disease characteristics, including airway inflammation and airflow limitation. The clinical management of chronic respiratory conditions is increasingly moving away from a one-size-fits-all model based on primary diagnosis, towards care targeting individual disease traits, and is particularly useful for subgroups of patients who respond poorly to conventional therapies. Respiratory microbiome analysis is an important potential contributor to such a 'treatable traits' approach, providing insight into both microbial drivers of airways disease, and the selective characteristics of the changing lower airway environment. We explore the potential to integrate respiratory microbiome analysis into a treatable traits model of clinical care and provide a practical guide to the application and clinical interpretation of respiratory microbiome analysis.

[Source](https://www.biorxiv.org/content/10.1101/2023.12.04.569909v4)

10.1111/resp.13411

---

## Healthy microbiome – moving towards functional interpretation

[Source](https://www.biorxiv.org/content/10.1101/2023.12.04.569909v4)

---

## Sociobiome – Individual and neighborhood socioeconomic status influence the gut microbiome in a multi-ethnic population in the US

[Source](https://www.nature.com/articles/s41522-024-00491-y)

---

## Cryptic diversity of cellulose-degrading gut bacteria in industrialized humans.
 18.03.2024, _S Moraïs, S Winkler, A Zorea, L Levin, FSP Nagies, N Kapust, E Lamed, A Artan-Furman, DN Bolam, MP Yadav, EA Bayer, WF Martin, I Mizrahi_


_Animals, Humans, Cellulose, Bacteria, Gastrointestinal Microbiome, Polysaccharides, Diet, Mammals_
Humans, like all mammals, depend on the gut microbiome for digestion of cellulose, the main component of plant fiber. However, evidence for cellulose fermentation in the human gut is scarce. We have identified ruminococcal species in the gut microbiota of human populations that assemble functional multienzymatic cellulosome structures capable of degrading plant cell wall polysaccharides. One of these species, which is strongly associated with humans, likely originated in the ruminant gut and was subsequently transferred to the human gut, potentially during domestication where it underwent diversification and diet-related adaptation through the acquisition of genes from other gut microbes. Collectively, these species are abundant and widespread among ancient humans, hunter-gatherers, and rural populations but are rare in populations from industrialized societies thus indicating potential disappearance in response to the westernized lifestyle.

[Source](https://www.science.org/doi/10.1126/science.adj9223)

10.1126/science.adj9223

---

## Comprehensive analyses of a large human gut Bacteroidales culture collection reveal species and strain level diversity and evolution.
 18.03.2024, _ZJ Zhang, CG Cole, MJ Coyne, H Lin, N Dylla, RC Smith, E Waligurski, R Ramaswamy, C Woodson, V Burgo, JC Little, D Moran, A Rose, M McMillin, E McSpadden, A Sundararajan, AM Sidebottom, EG Pamer, LE Comstock_


Species of the Bacteroidales order are among the most abundant and stable bacterial members of the human gut microbiome with diverse impacts on human health. While Bacteroidales strains and species are genomically and functionally diverse, order-wide comparative analyses are lacking. We cultured and sequenced the genomes of 408 Bacteroidales isolates from healthy human donors representing nine genera and 35 species and performed comparative genomic, gene-specific, mobile gene, and metabolomic analyses. Families, genera, and species could be grouped based on many distinctive features. However, we also show extensive DNA transfer between diverse families, allowing for shared traits and strain evolution. Inter- and intra-specific diversity is also apparent in the metabolomic profiling studies. This highly characterized and diverse Bacteroidales culture collection with strain-resolved genomic and metabolomic analyses can serve as a resource to facilitate informed selection of strains for microbiome reconstitution.

[Source](https://www.biorxiv.org/content/10.1101/2024.03.08.584156v1)

10.1101/2024.03.08.584156

---

## https://t.co/SQMbCohqoQ

[Source](https://t.co/SQMbCohqoQ)

---

## Changes in frequency of benign convulsions with mild gastroenteritis and their viral causes before and during the COVID-19 pandemic: a single-center study.
 19.03.2024, _H Na, S Lee, SH Kim, YO Kim_


Benign convulsions with mild gastroenteritis (CwG) are prevalent in young children during the winter. Early in the coronavirus disease 2019 (COVID-19) pandemic, viral gastroenteritis occurrence decreased and seasonal variation was lost, which can change CwG.

[Source](https://twitter.com/SZhangJerry/status/1769189530043596833?ref_src=twsrc%5Etfw)

10.3345/cep.2023.01137

---

## March 17, 2024

[Source](https://twitter.com/SZhangJerry/status/1769189530043596833?ref_src=twsrc%5Etfw)

---

## A history of repeated antibiotic usage leads to microbiota-dependent mucus defects

[Source](https://www.biorxiv.org/content/10.1101/2024.03.07.583875v1)

---

## Global abundance patterns, diversity, and ecology of Patescibacteria in wastewater treatment plants.
 18.03.2024, _H Hu, JM Kristensen, CW Herbold, P Pjevac, K Kitzinger, B Hausmann, MKD Dueholm, PH Nielsen, M Wagner_


_Wastewater, RNA, Ribosomal, 16S, Bacteria, Microbiota, Water Purification_
Microorganisms are responsible for nutrient removal and resource recovery in wastewater treatment plants (WWTPs), and their diversity is often studied by 16S rRNA gene amplicon sequencing. However, this approach underestimates the abundance and diversity of Patescibacteria due to the low coverage of commonly used PCR primers for this highly divergent bacterial phylum. Therefore, our current understanding of the global diversity, distribution, and ecological role of Patescibacteria in WWTPs is very incomplete. This is particularly relevant as Patescibacteria are considered to be associated with microbial host cells and can therefore influence the abundance and temporal variability of other microbial groups that are important for WWTP functioning.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01769-1)

10.1186/s40168-024-01769-1

---

## @MicrobiomeJ

[Source](https://twitter.com/MicrobiomeJ?ref_src=twsrc%5Etfw)

---

## Small cells - big issues: biological implications and preclinical advancements in small cell lung cancer.
 28.02.2024, _A Solta, B Ernhofer, K Boettiger, Z Megyesfalvi, S Heeke, MA Hoda, C Lang, C Aigner, FR Hirsch, K Schelch, B Döme_


_Humans, Small Cell Lung Carcinoma, Lung Neoplasms, Immunotherapy, Transcription Factors_
Current treatment guidelines refer to small cell lung cancer (SCLC), one of the deadliest human malignancies, as a homogeneous disease. Accordingly, SCLC therapy comprises chemoradiation with or without immunotherapy. Meanwhile, recent studies have made significant advances in subclassifying SCLC based on the elevated expression of the transcription factors ASCL1, NEUROD1, and POU2F3, as well as on certain inflammatory characteristics. The role of the transcription regulator YAP1 in defining a unique SCLC subset remains to be established. Although preclinical analyses have described numerous subtype-specific characteristics and vulnerabilities, the so far non-existing clinical subtype distinction may be a contributor to negative clinical trial outcomes. This comprehensive review aims to provide a framework for the development of novel personalized therapeutic approaches by compiling the most recent discoveries achieved by preclinical SCLC research. We highlight the challenges faced due to limited access to patient material as well as the advances accomplished by implementing state-of-the-art models and methodologies.

[Source](https://twitter.com/DOME_Vienna?ref_src=twsrc%5Etfw)

10.1186/s12943-024-01953-9

---

## @DOME_Vienna

[Source](https://twitter.com/DOME_Vienna?ref_src=twsrc%5Etfw)

---

## @Jance89

[Source](https://twitter.com/Jance89?ref_src=twsrc%5Etfw)

---

## Community shift from phototrophic to chemotrophic sulfide oxidation following anoxic holomixis in a stratified seawater lake.
 13.11.2018, _P Pjevac, M Korlević, JS Berg, E Bura-Nakić, I Ciglenečki, R Amann, S Orlić_


_Anaerobiosis, Bacteria, Biota, DNA, Bacterial, Metagenomics, Molecular Sequence Data, Oxidation-Reduction, Phototrophic Processes, Seawater, Sequence Analysis, DNA, Sulfides_
Most stratified sulfidic holomictic lakes become oxygenated after annual turnover. In contrast, Lake Rogoznica, on the eastern Adriatic coast, has been observed to undergo a period of water column anoxia after water layer mixing and establishment of holomictic conditions. Although Lake Rogoznica's chemistry and hydrography have been studied extensively, it is unclear how the microbial communities typically inhabiting the oxic epilimnion and a sulfidic hypolimnion respond to such a drastic shift in redox conditions. We investigated the impact of anoxic holomixis on microbial diversity and microbially mediated sulfur cycling in Lake Rogoznica with an array of culture-independent microbiological methods. Our data suggest a tight coupling between the lake's chemistry and occurring microorganisms. During stratification, anoxygenic phototrophic sulfur bacteria were dominant at the chemocline and in the hypolimnion. After an anoxic mixing event, the anoxygenic phototrophic sulfur bacteria entirely disappeared, and the homogeneous, anoxic water column was dominated by a bloom of gammaproteobacterial sulfur oxidizers related to the GSO/SUP05 clade. This study is the first report of a community shift from phototrophic to chemotrophic sulfide oxidizers as a response to anoxic holomictic conditions in a seasonally stratified seawater lake.

[Source](https://twitter.com/ppjevac?ref_src=twsrc%5Etfw)

10.1128/AEM.02435-14

---

## @ppjevac

[Source](https://twitter.com/ppjevac?ref_src=twsrc%5Etfw)

---

## @MichiWagner4

[Source](https://twitter.com/MichiWagner4?ref_src=twsrc%5Etfw)

---

## @msdueholm

[Source](https://twitter.com/msdueholm?ref_src=twsrc%5Etfw)

---

## @PerHNielsen

[Source](https://twitter.com/PerHNielsen?ref_src=twsrc%5Etfw)

---

## @_ocilisni_

[Source](https://twitter.com/_ocilisni_?ref_src=twsrc%5Etfw)

---

## #CraigHerbold

[Source](https://twitter.com/hashtag/CraigHerbold?src=hash&ref_src=twsrc%5Etfw)

---

## #KathiKitzinger

[Source](https://twitter.com/hashtag/KathiKitzinger?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/38eA5DEc3u

[Source](https://t.co/38eA5DEc3u)

---

## March 16, 2024

[Source](https://twitter.com/Huifeng_Hu/status/1768940203186532493?ref_src=twsrc%5Etfw)

---

## Challenges in computational discovery of bioactive peptides in 'omics data.
 08.03.2024, _LP Coelho, CD Santos-Júnior, C de la Fuente-Nunez_


Peptides have a plethora of activities in biological systems that can potentially be exploited biotechnologically. Several peptides are used clinically, as well as in industry and agriculture. The increase in available 'omics data has recently provided a large opportunity for mining novel enzymes, biosynthetic gene clusters, and molecules. While these data primarily consist of DNA sequences, other types of data provide important complementary information. Due to their size, the approaches proven successful at discovering novel proteins of canonical size cannot be naïvely applied to the discovery of peptides. Peptides can be encoded directly in the genome as short open reading frames (smORFs), or they can be derived from larger proteins by proteolysis. Both of these peptide classes pose challenges as simple methods for their prediction result in large numbers of false positives. Similarly, functional annotation of larger proteins, traditionally based on sequence similarity to infer orthology and then transferring functions between characterized proteins and uncharacterized ones, cannot be applied for short sequences. The use of these techniques is much more limited and alternative approaches based on machine learning are used instead. Here, we review the limitations of traditional methods as well as the alternative methods that have recently been developed for discovering novel bioactive peptides with a focus on prokaryotic genomes and metagenomes.

[Source](https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/pmic.202300105)

10.1002/pmic.202300105

---

## Challenges in computational discovery of bioactive peptides in ’omics data – Luis Pedro Coelho

[Source](https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/pmic.202300105)

---

## For long-term sustainable software in bioinformatics.
 18.03.2024, _LP Coelho_


_Computational Biology, Software_

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011920)

10.1371/journal.pcbi.1011920

---

## Data-driven prediction of colonization outcomes for complex microbial communities.
 18.03.2024, _L Wu, XW Wang, Z Tao, T Wang, W Zuo, Y Zeng, YY Liu, L Dai_


_Humans, Microbiota, Feces, Enterococcus faecium, Microbial Interactions, Enterococcus faecalis_
Microbial interactions can lead to different colonization outcomes of exogenous species, be they pathogenic or beneficial in nature. Predicting the colonization of exogenous species in complex communities remains a fundamental challenge in microbial ecology, mainly due to our limited knowledge of the diverse mechanisms governing microbial dynamics. Here, we propose a data-driven approach independent of any dynamics model to predict colonization outcomes of exogenous species from the baseline compositions of microbial communities. We systematically validate this approach using synthetic data, finding that machine learning models can predict not only the binary colonization outcome but also the post-invasion steady-state abundance of the invading species. Then we conduct colonization experiments for commensal gut bacteria species Enterococcus faecium and Akkermansia muciniphila in hundreds of human stool-derived in vitro microbial communities, confirming that the data-driven approaches can predict the colonization outcomes in experiments. Furthermore, we find that while most resident species are predicted to have a weak negative impact on the colonization of exogenous species, strongly interacting species could significantly alter the colonization outcomes, e.g., Enterococcus faecalis inhibits the invasion of E. faecium invasion. The presented results suggest that the data-driven approaches are powerful tools to inform the ecology and management of microbial communities.

[Source](https://www.nature.com/articles/s41467-024-46766-y)

10.1038/s41467-024-46766-y

---

## pic.twitter.com/oeRFQGfBDx

[Source](https://t.co/oeRFQGfBDx)

---

## Changes in frequency of benign convulsions with mild gastroenteritis and their viral causes before and during the COVID-19 pandemic: a single-center study.
 19.03.2024, _H Na, S Lee, SH Kim, YO Kim_


Benign convulsions with mild gastroenteritis (CwG) are prevalent in young children during the winter. Early in the coronavirus disease 2019 (COVID-19) pandemic, viral gastroenteritis occurrence decreased and seasonal variation was lost, which can change CwG.

[Source](https://twitter.com/alifcoder/status/1769389993464824185?ref_src=twsrc%5Etfw)

10.3345/cep.2023.01137

---

## March 17, 2024

[Source](https://twitter.com/alifcoder/status/1769389993464824185?ref_src=twsrc%5Etfw)

---

