## The cancer chemotherapeutic 5-fluorouracil is a potent Fusobacterium nucleatum inhibitor and its activity is modified by intratumoral microbiota.
 21.11.2022, _KD LaCourse, M Zepeda-Rivera, AG Kempchinsky, A Baryiames, SS Minot, CD Johnston, S Bullman_


_Humans, Fusobacterium nucleatum, Escherichia coli, Fluorouracil, Microbiota, Colorectal Neoplasms_
Fusobacterium nucleatum (Fn) is a dominant bacterial species in colorectal cancer (CRC) tissue that is associated with cancer progression and poorer patient prognosis. Following a small-molecule inhibitor screen of 1,846 bioactive compounds against a Fn CRC isolate, we find that 15% of inhibitors are antineoplastic agents including fluoropyrimidines. Validation of these findings reveals that 5-fluorouracil (5-FU), a first-line CRC chemotherapeutic, is a potent inhibitor of Fn CRC isolates. We also identify members of the intratumoral microbiota, including Escherichia coli, that are resistant to 5-FU. Further, CRC E. coli isolates can modify 5-FU and relieve 5-FU toxicity toward otherwise-sensitive Fn and human CRC epithelial cells. Lastly, we demonstrate that ex vivo patient CRC tumor microbiota undergo community disruption after 5-FU exposure and have the potential to deplete 5-FU levels, reducing local drug efficacy. Together, these observations argue for further investigation into the role of the CRC intratumoral microbiota in patient response to chemotherapy.

[Source](https://doi.org/10.1016/j.celrep.2022.111625)

10.1016/j.celrep.2022.111625

---

## The cancer chemotherapeutic 5-fluorouracil is a potent <em>Fusobacterium nucleatum </em>inhibitor and its activity is modified by intratumoral microbiota

[Source](https://doi.org/10.1016/j.celrep.2022.111625)

---

## Enterococci enhance <em>Clostridioides difficile</em> pathogenesis

[Source](https://doi.org/10.1038/s41586-022-05438-x)

---

## Host biology, ecology and the environment influence microbial biomass and diversity in 101 marine fish species.
 21.11.2022, _JJ Minich, A Härer, J Vechinski, BW Frable, ZR Skelton, E Kunselman, MA Shane, DS Perry, A Gonzalez, D McDonald, R Knight, TP Michael, EE Allen_


_Animals, Biomass, Fishes, Ecology, Microbiota, Gills, Vertebrates, Mammals_
Fish are the most diverse and widely distributed vertebrates, yet little is known about the microbial ecology of fishes nor the biological and environmental factors that influence fish microbiota. To identify factors that explain microbial diversity patterns in a geographical subset of marine fish, we analyzed the microbiota (gill tissue, skin mucus, midgut digesta and hindgut digesta) from 101 species of Southern California marine fishes, spanning 22 orders, 55 families and 83 genera, representing ~25% of local marine fish diversity. We compare alpha, beta and gamma diversity while establishing a method to estimate microbial biomass associated with these host surfaces. We show that body site is the strongest driver of microbial diversity while microbial biomass and diversity is lowest in the gill of larger, pelagic fishes. Patterns of phylosymbiosis are observed across the gill, skin and hindgut. In a quantitative synthesis of vertebrate hindguts (569 species), we also show that mammals have the highest gamma diversity when controlling for host species number while fishes have the highest percent of unique microbial taxa. The composite dataset will be useful to vertebrate microbiota researchers and fish biologists interested in microbial ecology, with applications in aquaculture and fisheries management.

[Source](https://doi.org/10.1038/s41467-022-34557-2)

10.1038/s41467-022-34557-2

---

## Genomic ancestry, diet and microbiomes of Upper Palaeolithic hunter-gatherers from San Teodoro cave.
 22.11.2022, _G Scorrano, SH Nielsen, DL Vetro, R Sawafuji, M Mackie, A Margaryan, AK Fotakis, C Martínez-Labarga, PF Fabbri, ME Allentoft, M Carra, F Martini, O Rickards, JV Olsen, MW Pedersen, E Cappellini, M Sikora_


_Humans, Animals, Proteomics, Dental Calculus, Diet, Genomics, Microbiota_
Recent improvements in the analysis of ancient biomolecules from human remains and associated dental calculus have provided new insights into the prehistoric diet and genetic diversity of our species. Here we present a multi-omics study, integrating metagenomic and proteomic analyses of dental calculus, and human ancient DNA analysis of the petrous bones of two post-Last Glacial Maximum (LGM) individuals from San Teodoro cave (Italy), to reconstruct their lifestyle and the post-LGM resettlement of Europe. Our analyses show genetic homogeneity in Sicily during the Palaeolithic, representing a hitherto unknown Italian genetic lineage within the previously identified Villabruna cluster. We argue that this lineage took refuge in Italy during the LGM, followed by a subsequent spread to central-western Europe. Analysis of dental calculus showed a diet rich in animal proteins which is also reflected on the oral microbiome composition. Our results demonstrate the power of this approach in the study of prehistoric humans and will enable future research to reach a more holistic understanding of the population dynamics and ecology.

[Source](https://doi.org/10.1038/s42003-022-04190-2)

10.1038/s42003-022-04190-2

---

## Gut microbiota analysis for prediction of clinical relapse in Crohn's disease.
 22.11.2022, _S Buffet-Bataillon, G Bouguen, F Fleury, V Cattoir, Y Le Cunff_


_Humans, Gastrointestinal Microbiome, Crohn Disease, Fatty Acids, Volatile, Feces, Clostridiales, Chronic Disease, Firmicutes, Recurrence_
The role of intestinal bacterial microbiota has been described as key in the pathophysiology of Crohn's disease (CD). CD is characterized by frequent relapses after periods of remission which are not entirely understood. In this paper, we investigate whether the heterogeneity in microbiota profiles in CD patients could be a suitable predictor for these relapses. This prospective observational study involved 259 CD patients, in which 41 provided an additional total of 62 consecutive fecal samples, with an average interval of 25 weeks in between each of these samples. Fecal microbiota was analyzed by massive genomic sequencing through 16 S rRNA amplicon sampling. We found that our 259 CD patients could be split into three distinct subgroups of microbiota (G1, G2, G3). From G1 to G3, we noticed a progressive decrease in alpha diversity (p ≤ 0.0001) but no change in the fecal calprotectin (FC) level. Focusing on the 103 consecutive samples from 41 CD patients, we showed that the patients microbiota profiles were remarkably stable over time and associated with increasing symptom severity. Investigating further this microbiota/severity association revealed that the first signs of aggravation are (1) a loss of the main anti-inflammatory Short-Chain Fatty Acids (SCFAs) Roseburia, Eubacterium, Subdoligranumum, Ruminococcus (P &lt; 0.05), (2) an increase in pro-inflammatory pathogens Proteus, Finegoldia (P &lt; 0.05) while (3) an increase of other minor SCFA producers such as Ezakiella, Anaerococcus, Megasphaera, Anaeroglobus, Fenollaria (P &lt; 0.05). Further aggravation of clinical signs is significantly linked to the subsequent loss of these minor SCFAs species and to an increase in other proinflammatory Proteobacteria such as Klebsiella, Pseudomonas, Salmonella, Acinetobacter, Hafnia and proinflammatory Firmicutes such as Staphylococcus, Enterococcus, Streptococcus. (P &lt; 0.05). To our knowledge, this is the first study (1) specifically identifying subgroups of microbiota profiles in CD patients, (2) relating these groups to the evolution of symptoms over time and (3) showing a two-step process in CD symptoms' worsening. This paves the way towards a better understanding of patient-to-patient heterogeneity, as well as providing early warning signals of future aggravation of the symptoms and eventually adapting empirically treatments.

[Source](https://doi.org/10.1038/s41598-022-23757-x)

10.1038/s41598-022-23757-x

---

## Implications of oral streptococcal bacteriophages in autism spectrum disorder.
 22.11.2022, _Z Tong, X Zhou, Y Chu, T Zhang, J Zhang, X Zhao, Z Wang, R Ding, Q Meng, J Yu, J Wang, Y Kang_


_Humans, Autism Spectrum Disorder, Streptococcus Phages, Gastrointestinal Microbiome, Mouth, Bacteriophages_
Growing evidence suggests altered oral and gut microbiota in autism spectrum disorder (ASD), but little is known about the alterations and roles of phages, especially within the oral microbiota in ASD subjects. We enrolled ASD (n = 26) and neurotypical subjects (n = 26) with their oral hygiene controlled, and the metagenomes of both oral and fecal samples (n = 104) are shotgun-sequenced and compared. We observe extensive and diverse oral phageome comparable to that of the gut, and clear signals of mouth-to-gut phage strain transfer within individuals. However, the overall phageomes of the two sites are widely different and show even less similarity in the oral communities between ASD and control subjects. The ASD oral phageome exhibits significantly reduced abundance and alpha diversity, but the Streptococcal phages there are atypically enriched, often dominating the community. The over-representation of Streptococcal phages is accompanied by enriched oral Streptococcal virulence factors and Streptococcus bacteria, all exhibiting a positive correlation with the severity of ASD clinical manifestations. These changes are not observed in the parallel sampling of the gut flora, suggesting a previously unknown oral-specific association between the excessive Streptococcal phage enrichment and ASD pathogenesis. The findings provide new evidence for the independent microbiome-mouth-brain connection, deepen our understanding of how the growth dynamics of bacteriophages and oral microbiota contribute to ASD, and point to novel effective therapeutics.

[Source](https://doi.org/10.1038/s41522-022-00355-3)

10.1038/s41522-022-00355-3

---

## Reproducible changes in the anorexia nervosa gut microbiota following inpatient therapy remain distinct from non-eating disorder controls

[Source](https://doi.org/10.1080/19490976.2022.2143217)

---

## Nasal microbiome disruption and recovery after mupirocin treatment in <em>Staphylococcus aureus</em> carriers and noncarriers

[Source](https://doi.org/10.1038/s41598-022-21453-4)

---

## BlpC-mediated selfish program leads to rapid loss of <em>Streptococcus pneumoniae</em> clonal diversity during infection

[Source](https://doi.org/10.1016/j.chom.2022.10.015)

---

## Treasures from trash in cancer research

[Source](https://doi.org/10.18632/oncotarget.28308)

---

## Human Gut Metagenomes Encode Diverse GH156 Sialidases

[Source](https://doi.org/10.1128/aem.01755-22)

---

## Species-Specific Patterns of Gut Metabolic Modules in Dutch Individuals with Different Dietary Habits

[Source](https://doi.org/10.1128/msphere.00512-22)

---

## Effect of the intratumoral microbiota on spatial and cellular heterogeneity in cancer

[Source](https://doi.org/10.1038/s41586-022-05435-0)

---

## Broad-spectrum antibiotics associated gut microbiome disturbance impairs T cell immunity and promotes lung cancer metastasis: a retrospective study

[Source](https://doi.org/10.1186/s12885-022-10307-x)

---

## Diet-induced modifications to human microbiome reshape colonic homeostasis in irritable bowel syndrome

[Source](https://doi.org/10.1016/j.celrep.2022.111657)

---

## The dynamic lung microbiome in health and disease

[Source](https://doi.org/10.1038/s41579-022-00821-x)

---

## Modulation of gut microbiota, blood metabolites, and disease resistance by dietary β-glucan in rainbow trout (<em>Oncorhynchus mykiss</em>)

[Source](https://doi.org/10.1186/s42523-022-00209-5)

---

## Living in honey: bacterial and fungal communities in honey of sympatric populations of <em>Apis mellifera</em> and the stingless bee <em>Melipona beecheii</em>, in Yucatan, Mexico

[Source](https://doi.org/10.1007/s00203-022-03319-5)

---

## The involvement of organic acids in soil fertility, plant health and environment sustainability

[Source](https://doi.org/10.1007/s00203-022-03321-x)

---

## Investigating the impact of database choice on the accuracy of metagenomic read classification for the rumen microbiome

[Source](https://doi.org/10.1186/s42523-022-00207-7)

---

## dbCAN-seq update: CAZyme gene clusters and substrates in microbiomes

[Source](https://doi.org/10.1093/nar/gkac1068)

---

## Gene co-expression network analysis of the human gut commensal bacterium <em>Faecalibacterium prausnitzii</em> in R-Shiny

[Source](https://doi.org/10.1371/journal.pone.0271847)

---

