## EST-SSR Marker Development and Full-Length Transcriptome Sequence Analysis of Tiger Lily (Lilium lancifolium Thunb).
 08.02.2022, _M Sun, Y Zhao, X Shao, J Ge, X Tang, P Zhu, J Wang, T Zhao_


The fast advancement and deployment of sequencing technologies after the Human Genome Project have greatly increased our knowledge of the eukaryotic genome sequences. However, due to technological concerns, high-quality genomic data has been confined to a few key organisms. Moreover, our understanding of which portions of genomes make up genes and which transcript isoforms synthesize these genes is scarce. Therefore, the current study has been designed to explore the reliability of the tiger lily (Lilium lancifolium Thunb) transcriptome. The PacBio-SMRT was used for attaining the complete transcriptomic profile. We obtained a total of 815,624 CCS (Circular Consensus Sequence) reads with an average length of 1295 bp. The tiger lily transcriptome has been sequenced for the first time using third-generation long-read technology. Furthermore, unigenes (38,707), lncRNAs (6852), and TF members (768) were determined based on the transcriptome data, followed by evaluating SSRs (3319). It has also been revealed that 105 out of 128 primer pairs effectively amplified PCR products. Around 15,608 transcripts were allocated to 25 distinct KOG Clusters, and 10,706 unigenes were grouped into 52 functional categories in the annotated transcripts. Until now, no tiger lily lncRNAs have been discovered. Results of this study may serve as an extensive set of reference transcripts and help us learn more about the transcriptomes of tiger lilies and pave the path for further research.

[Source](https://lebeerlab.com/the-story-behind-latilactobacillus-fragifolii-a-new-bacterial-species-isolated-in-our-lab/)

10.1155/2022/7641048

---

## read the full 🍓

[Source](https://lebeerlab.com/the-story-behind-latilactobacillus-fragifolii-a-new-bacterial-species-isolated-in-our-lab/)

---

## 'A landmark in psychiatric progress'? The role of evidence in the rise and fall of insulin coma therapy.
 22.12.2021, _R Freudenthal, J Moncrieff_


This paper examines the evidence behind the use and decline of insulin coma therapy as a treatment for schizophrenia and how this was viewed by the psychiatric profession. The paper demonstrates that, from the time of its introduction, there was considerable debate regarding the evidence for insulin treatment, and scepticism about its purported benefits. The randomized trials conducted in the 1950s were the result, rather than the origins, of this debate. Although insulin treatment was subsequently abandoned, it was still regarded as a historic moment in the modernization of psychiatry. Then, as now, evidence does not speak for itself, and insulin continued to be incorporated into the story of psychiatric progress even after it was shown to be ineffective.

[Source](https://lebeerlab.com/the-story-behind-latilactobacillus-fragifolii-a-new-bacterial-species-isolated-in-our-lab/)

10.1177/0957154X211062538

---

## story behind the paper

[Source](https://lebeerlab.com/the-story-behind-latilactobacillus-fragifolii-a-new-bacterial-species-isolated-in-our-lab/)

---

## Microbial community composition associated with early developmental stages of the Indian white shrimp, Penaeus indicus.
 07.02.2022, _TN Vinay, PK Patil, R Aravind, PSS Anand, V Baskaran, CP Balasubramanian_


Gut microbiota is known to influence the physiology, health, nutrient absorption, reproduction, and other metabolic activities of aquatic organisms. Microbial composition can influence intestinal immunity and are considered as health indicators. Information on gut microbial composition provides potential application possibilities to improve shrimp health and production. In the absence of such information for Penaeus indicus, the present study reports the microbial community structure associated with its early developmental stages. Bacterial community associated with the early developmental stages (egg, nauplii, zoea, mysis, PL1, PL6 and PL12) from two hatchery cycles were analysed employing 16S rRNA high throughput sequencing. Proteobacteria and Bacteroidetes, were the two dominant phyla in P. indicus development stages. Sequential sampling revealed the constant change in the bacterial composition at genus level. Alteromonas was dominant in egg and nauplii stage, whilst Ascidiaceihabitans (formerly Roseobacter) was the dominant genera in both PL6 and PL12. The bacterial composition was highly dynamic in early stages and our study suggests that the mysis stage is the critical phase in transforming the microbial composition and it gets stabilised by early post larval stages. This is the first report on the composition of microbiota in early developmental stages of P. indicus. Based on these results the formation of microbial composition seems to be influenced by feeding at early stages. The study provides valuable information to device intervention strategies for healthy seed production.

[Source](https://oaepublish.com/mrr/journal/volume_issues_archive/2022/1/163)

10.1007/s00438-022-01865-7

---

## the Microbiome Research Reports

[Source](https://oaepublish.com/mrr/journal/volume_issues_archive/2022/1/163)

---

## Mechanism of Dyspnea during Exercise in Children with Corrected Congenital Heart Disease.
 24.01.2022, _M Chlif, MM Ammar, NB Said, L Sergey, S Ahmaidi, F Alassery, H Hamam_


_Child, Dyspnea, Exercise, Exercise Test, Heart Defects, Congenital, Humans, Oxygen Consumption_
This study will evaluate cardiorespiratory and peripheral muscle function and their relationship with subjective dyspnea threshold after the surgical correction of congenital heart disease in children. Thirteen children with surgically repaired congenital heart disease were recruited. Each participant performed an incremental exercise test on a cycle ergometer until exhaustion. Gas exchanges were continuously sampled to measure the maximal aerobic parameters and ventilatory thresholds. The functional capacity of the subjects was assessed with a 6 min walk test. At the end of the exercise test, isokinetic Cybex Norm was used to evaluate the strength and endurance of the knee extensor muscle in the leg. Dyspnea was subjectively scored with a visual analog scale during the last 15 s of each exercise step. Oxygen consumption measured at the dyspnea score/VO2 relationship located at the dyspnea threshold, at which dyspnea suddenly increased. Results: The maximal and submaximal values of the parameters describing the exercise and the peripheral muscular performances were: VO2 Peak: 33.8 ± 8.9 mL·min-1·kg-1; HR: 174 ± 9 b·min-1; VEmax: 65.68 ± 15.9 L·min-1; P max: 117 ± 27 W; maximal voluntary isometric force MVIF: 120.8 ± 41.9 N/m; and time to exhaustion Tlim: 53 ± 21 s. Oxygen consumption measured at the dyspnea threshold was related to VO2 Peak (R2 = 0.74; p &lt; 0.01), Tlim (R2 = 0.78; p &lt; 0.01), and the distance achieved during the 6MWT (R2 = 0.57; p &lt; 0.05). Compared to the theoretical maximal values for the power output, VO2, and HR, the surgical correction did not repair the exercise performance. After the surgical correction of congenital heart disease, exercise performance was impeded by alterations of the cardiorespiratory function and peripheral local factors. A subjective evaluation of the dyspnea threshold is a reliable criterion that allows the prediction of exercise capacity in subjects suffering from congenital heart disease.

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

10.3390/ijerph19010099

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/ue0WMYAmda

[Source](https://t.co/ue0WMYAmda)

---

## February 3, 2022

[Source](https://twitter.com/MicrobiomeVIF/status/1489279775105552386?ref_src=twsrc%5Etfw)

---

## Functional and evolutionary significance of unknown genes from uncultivated taxa

[Source](https://www.biorxiv.org/content/10.1101/2022.01.26.477801v1)

---

## https://t.co/emyTjq1ewc

[Source](https://t.co/emyTjq1ewc)

---

## @cgmlaboratory

[Source](https://twitter.com/cgmlaboratory?ref_src=twsrc%5Etfw)

---

## @BigDataBiology

[Source](https://twitter.com/BigDataBiology?ref_src=twsrc%5Etfw)

---

## @BorkLab

[Source](https://twitter.com/BorkLab?ref_src=twsrc%5Etfw)

---

## @SunagawaLab

[Source](https://twitter.com/SunagawaLab?ref_src=twsrc%5Etfw)

---

## @CBGP_Madrid

[Source](https://twitter.com/CBGP_Madrid?ref_src=twsrc%5Etfw)

---

## @CaixaResearch

[Source](https://twitter.com/CaixaResearch?ref_src=twsrc%5Etfw)

---

## pic.twitter.com/JqINkcfPSX

[Source](https://t.co/JqINkcfPSX)

---

## January 28, 2022

[Source](https://twitter.com/alvarordr/status/1487026445327155201?ref_src=twsrc%5Etfw)

---

## The adult microbiome of healthy and otitis patients: Definition of the core healthy and diseased ear microbiomes

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0262806)

---

## Intestinal <em>Akkermansia muciniphila</em> predicts clinical response to PD-1 blockade in patients with advanced non-small-cell lung cancer

[Source](https://www.nature.com/articles/s41591-021-01655-5)

---

## Associations and recovery dynamics of the nasopharyngeal microbiota during influenza‑like illness in the aging population

[Source](https://www.nature.com/articles/s41598-022-05618-9.epdf)

---

## Genomic diversity and antimicrobial resistance of <em>Prevotella</em> species isolated from chronic lung disease airways 

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000754)

---

## Effect of host genetics on the gut microbiome in 7,738 participants of the Dutch Microbiome Project

[Source](https://www.nature.com/articles/s41588-021-00992-y)

---

## @SashaZhernakova

[Source](https://twitter.com/SashaZhernakova?ref_src=twsrc%5Etfw)

---

## @serenasanna1

[Source](https://twitter.com/serenasanna1?ref_src=twsrc%5Etfw)

---

## @ealopera14

[Source](https://twitter.com/ealopera14?ref_src=twsrc%5Etfw)

---

## @AKurilshikov

[Source](https://twitter.com/AKurilshikov?ref_src=twsrc%5Etfw)

---

## @Shixian00000

[Source](https://twitter.com/Shixian00000?ref_src=twsrc%5Etfw)

---

## https://t.co/6ks3jufdwl

[Source](https://t.co/6ks3jufdwl)

---

## February 3, 2022

[Source](https://twitter.com/A_van_der_graaf/status/1489268723064180743?ref_src=twsrc%5Etfw)

---

## The gut bacterial community potentiates <em>Clostridioides difficile </em>infection severity 

[Source](https://www.biorxiv.org/content/10.1101/2022.01.31.478599v1)

---

## Emergent evolutionary forces in spatial models of microbial growth in the human gut microbiota

[Source](https://www.biorxiv.org/content/10.1101/2021.07.15.452569v2)

---

## @oliviamghosh

[Source](https://twitter.com/oliviamghosh?ref_src=twsrc%5Etfw)

---

## https://t.co/AKr5gUEixx

[Source](https://t.co/AKr5gUEixx)

---

## pic.twitter.com/6q3eKO0Q1g

[Source](https://t.co/6q3eKO0Q1g)

---

## January 31, 2022

[Source](https://twitter.com/benjaminhgood/status/1488292593985146882?ref_src=twsrc%5Etfw)

---

## Microbiome-based therapeutics

[Source](https://www.nature.com/articles/s41579-021-00667-9)

---

## Multimodal interactions of drugs, natural compounds and pollutants with the gut microbiota

[Source](https://www.nature.com/articles/s41579-022-00681-5.epdf)

---

## Gut microbiome and health: mechanistic insights

[Source](https://gut.bmj.com/content/gutjnl/early/2022/01/31/gutjnl-2021-326789.full.pdf)

---

## <em>Gardnerella vaginalis </em>induces matrix metalloproteinases in the cervicovaginal epithelium through TLR-2 activation 

[Source](https://www.ajog.org/article/S0002-9378(21)01290-4/fulltext)

---

## Extended bacterial diversity of the urinary microbiome of reproductive-age healthy European women captured by culturomics and long-read amplicon sequencing

[Source](https://www.biorxiv.org/content/10.1101/2022.01.19.476882v1)

---

## @MagdalenaKsi

[Source](https://twitter.com/MagdalenaKsi?ref_src=twsrc%5Etfw)

---

## @joanacmrocha

[Source](https://twitter.com/joanacmrocha?ref_src=twsrc%5Etfw)

---

## @BeteCappelli

[Source](https://twitter.com/BeteCappelli?ref_src=twsrc%5Etfw)

---

## @marciasousa95

[Source](https://twitter.com/marciasousa95?ref_src=twsrc%5Etfw)

---

## @TeresaPGRibeiro

[Source](https://twitter.com/TeresaPGRibeiro?ref_src=twsrc%5Etfw)

---

## @filipa_grosso

[Source](https://twitter.com/filipa_grosso?ref_src=twsrc%5Etfw)

---

## @LpeixeLuisa

[Source](https://twitter.com/LpeixeLuisa?ref_src=twsrc%5Etfw)

---

## https://t.co/BxegxWSyEc

[Source](https://t.co/BxegxWSyEc)

---

## January 21, 2022

[Source](https://twitter.com/svetlana_up/status/1484325769505808387?ref_src=twsrc%5Etfw)

---

## Phylogenetic analysis and antimicrobial susceptibility profile of uropathogens

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0262952)

---

## Multidrug-resistant Uro-associated <em>Escherichia coli </em>Populations and Recurrent Urinary Tract Infections in Patients Performing Clean Intermittent Self-catheterisation

[Source](https://www.sciencedirect.com/science/article/pii/S2666168322000088)

---

## <em>Latilactobacillus fragifolii</em> sp. nov., isolated from leaves of a strawberry plant (<em>Fragaria x ananassa</em>)

[Source](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.005193)

---

## Ecology and potential functions of plant-associated microbial communities in cold environments

[Source](https://academic.oup.com/femsec/article/98/1/fiab161/6462900)

---

## @FEMSmicro

[Source](https://twitter.com/FEMSmicro?ref_src=twsrc%5Etfw)

---

## #H2020

[Source](https://twitter.com/hashtag/H2020?src=hash&ref_src=twsrc%5Etfw)

---

## #MSCA

[Source](https://twitter.com/hashtag/MSCA?src=hash&ref_src=twsrc%5Etfw)

---

## #FreezingBioprotector

[Source](https://twitter.com/hashtag/FreezingBioprotector?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/7c8vm4Tai3

[Source](https://t.co/7c8vm4Tai3)

---

## February 2, 2022

[Source](https://twitter.com/MalekMarian1/status/1488817503484751873?ref_src=twsrc%5Etfw)

---

## Gut microbiome of the largest living rodent harbors unprecedented enzymatic systems to degrade plant polysaccharides

[Source](https://www.nature.com/articles/s41467-022-28310-y)

---

## Pesticide-induced disturbances of bee gut microbiotas 

[Source](https://academic.oup.com/femsre/advance-article-abstract/doi/10.1093/femsre/fuab056/6517452)

---

## Effect of a freeze-dried coffee solution in a high-fat diet-induced obesity model in rats: Impact on inflammatory response, lipid profile, and gut microbiota

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0262270)

---

## Gut microbiota and butyrate contribute to nonalcoholic fatty liver disease in premenopause due to estrogen deficiency

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0262855)

---

## BIOME-Preserve: A novel storage and transport medium for preserving anaerobic microbiota samples for culture recovery.
 28.01.2022, _ER Hyde, H Lozano, S Cox_


Sequencing-based protocols for studying the human microbiome have unearthed a wealth of information about the relationship between the microbiome and human health. But these microbes cannot be leveraged as therapeutic targets without culture-based studies to phenotype species of interest and to establish culture collections for use in animal models. Traditional sample collection protocols are focused on preserving nucleic acids and metabolites and are largely inappropriate for preserving sensitive anaerobic bacteria for later culture recovery. Here we introduce a novel microbiome preservation kit (BIOME-Preserve) that facilitates recovery of anaerobic bacteria from human stool. Using a combination of culture recovery and shallow whole-genome shotgun sequencing, we characterized the anaerobes cultured from fresh human stool and from human stool held at room temperature in BIOME-Preserve for up to 120 hours. We recovered several species of interest to microbiome researchers, including Bifidobacterium spp., Bacteroides spp., Blautia spp., Eubacterium halii (now Anaerobutyricum hallii), Akkermansia muciniphila, and Faecalibacterium prausnitzii. We also demonstrated that freezing at -80°C did not adversely affect our ability to culture organisms from BIOME-Preserve, suggesting that it is appropriate both as a transport medium and as a medium for longer-term ultra-cold storage. Together, our results suggest BIOME-Preserve is practical for the collection, transport, and culture of anaerobic bacteria from human samples and can help enable researchers to better understand the link between the microbiome and human health and how to leverage that link through novel microbiome-based therapeutics.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0261820)

10.1371/journal.pone.0261820

---

## Species-targeted sorting and cultivation of commensal bacteria from the gut microbiome using flow cytometry under anaerobic conditions.
 08.02.2022, _S Bellais, M Nehlich, M Ania, A Duquenoy, W Mazier, G van den Engh, J Baijer, NS Treichel, T Clavel, I Belotserkovsky, V Thomas_


There is a growing interest in using gut commensal bacteria as "next generation" probiotics. However, this approach is still hampered by the fact that there are few or no strains available for specific species that are difficult to cultivate. Our objective was to adapt flow cytometry and cell sorting to be able to detect, separate, isolate, and cultivate new strains of commensal species from fecal material. We focused on the extremely oxygen sensitive (EOS) species Faecalibacterium prausnitzii and the under-represented, health-associated keystone species Christensenella minuta as proof-of-concept.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01206-7)

10.1186/s40168-021-01206-7

---

## Evaluation of taxonomic profiling methods for long-read shotgun metagenomic sequencing datasets

[Source](https://www.biorxiv.org/content/10.1101/2022.01.31.478527v1)

---

## Evaluating supervised and unsupervised background noise correction in human gut microbiome data.
 07.02.2022, _L Briscoe, B Balliu, S Sankararaman, E Halperin, NR Garud_


The ability to predict human phenotypes and identify biomarkers of disease from metagenomic data is crucial for the development of therapeutics for microbiome-associated diseases. However, metagenomic data is commonly affected by technical variables unrelated to the phenotype of interest, such as sequencing protocol, which can make it difficult to predict phenotype and find biomarkers of disease. Supervised methods to correct for background noise, originally designed for gene expression and RNA-seq data, are commonly applied to microbiome data but may be limited because they cannot account for unmeasured sources of variation. Unsupervised approaches address this issue, but current methods are limited because they are ill-equipped to deal with the unique aspects of microbiome data, which is compositional, highly skewed, and sparse. We perform a comparative analysis of the ability of different denoising transformations in combination with supervised correction methods as well as an unsupervised principal component correction approach that is presently used in other domains but has not been applied to microbiome data to date. We find that the unsupervised principal component correction approach has comparable ability in reducing false discovery of biomarkers as the supervised approaches, with the added benefit of not needing to know the sources of variation apriori. However, in prediction tasks, it appears to only improve prediction when technical variables contribute to the majority of variance in the data. As new and larger metagenomic datasets become increasingly available, background noise correction will become essential for generating reproducible microbiome analyses.

[Source](https://twitter.com/hashtag/metagenomic?src=hash&ref_src=twsrc%5Etfw)

10.1371/journal.pcbi.1009838

---

## #metagenomic

[Source](https://twitter.com/hashtag/metagenomic?src=hash&ref_src=twsrc%5Etfw)

---

## @PacBio

[Source](https://twitter.com/PacBio?ref_src=twsrc%5Etfw)

---

## The roles of recombination and selection in shaping genomic divergence in an incipient ecological species complex.
 04.02.2022, _MJ Wersebe, RE Sherman, PD Jeyasingh, LJ Weider_


Speciation genomic studies have revealed that genomes of diverging lineages are shaped jointly by the actions of gene flow and selection. These evolutionary forces acting in concert with processes such as recombination and genome features such as gene density shape a mosaic landscape of divergence. We investigated the roles of recombination and gene density in shaping the patterns of differentiation and divergence between the cyclically parthenogenetic ecological sister-taxa, Daphnia pulicaria and Daphnia pulex. First, we assembled a phased chromosome-scale genome assembly using trio-binning for D. pulicaria and constructed a genetic map using an F2-intercross panel to understand sex-specific recombination rate heterogeneity. Finally, we used a ddRADseq dataset with broad geographic sampling of D. pulicaria, D. pulex, and their hybrids to understand the patterns of genome-scale divergence and demographic parameters. Our study provides the first sex-specific estimates of recombination rates for a cyclical parthenogen, and unlike other eukaryotic species, we observed male-biased heterochiasmy in D. pulicaria, which may be related to this somewhat unique breeding mode. Additionally, regions of high gene density and recombination are generally more divergent than regions of suppressed recombination. Outlier analysis indicated that divergent genomic regions are likely driven by selection on D. pulicaria, the derived lineage colonizing a novel lake habitat. Together, our study supports a scenario of selection acting on genes related to local adaptation shaping genome-wide patterns of differentiation despite high local recombination rates in this species complex. Finally, we discuss the limitations of our data in light of demographic uncertainty.

[Source](https://twitter.com/hashtag/HiFi?src=hash&ref_src=twsrc%5Etfw)

10.1111/mec.16383

---

## #HiFi

[Source](https://twitter.com/hashtag/HiFi?src=hash&ref_src=twsrc%5Etfw)

---

## @nanopore

[Source](https://twitter.com/nanopore?ref_src=twsrc%5Etfw)

---

## @ctitusbrown

[Source](https://twitter.com/ctitusbrown?ref_src=twsrc%5Etfw)

---

## @saltyscientist

[Source](https://twitter.com/saltyscientist?ref_src=twsrc%5Etfw)

---

## https://t.co/eBBDiOj3fw

[Source](https://t.co/eBBDiOj3fw)

---

## February 2, 2022

[Source](https://twitter.com/DPortik/status/1488965535505084419?ref_src=twsrc%5Etfw)

---

## AGAMEMNON: an Accurate metaGenomics And MEtatranscriptoMics quaNtificatiON analysis suite

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02610-4)

---

## McComedy: A user-friendly tool for next-generation individual-based modeling of microbial consumer-resource systems

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009777)

---

## VeChat: Correcting errors in long reads using variation graphs

[Source](https://www.biorxiv.org/content/10.1101/2022.01.30.478352v1)

---

## ReadBouncer: Precise and Scalable Adaptive Sampling for Nanopore Sequencing

[Source](https://www.biorxiv.org/content/10.1101/2022.02.01.478636v1)

---

## @LetsTalkSciComm

[Source](https://twitter.com/LetsTalkSciComm?ref_src=twsrc%5Etfw)

---

## https://t.co/8X2UI79CFo

[Source](https://t.co/8X2UI79CFo)

---

## February 2, 2022

[Source](https://twitter.com/svetlana_up/status/1488722393212551173?ref_src=twsrc%5Etfw)

---

