## Interspecies interactions in bacterial colonies are determined by physiological traits and the environment

[Source](https://doi.org/10.1101/623017)

---

## Bacterial colonization reprograms the neonatal gut metabolome.
 14.04.2020, _K Bittinger, C Zhao, Y Li, E Ford, ES Friedman, J Ni, CV Kulkarni, J Cai, Y Tian, Q Liu, AD Patterson, D Sarkar, SHJ Chan, C Maranas, A Saha-Shah, P Lund, BA Garcia, LM Mattei, JS Gerber, MA Elovitz, A Kelly, P DeRusso, D Kim, CE Hofstaedter, M Goulian, H Li, FD Bushman, BS Zemel, GD Wu_


Initial microbial colonization and later succession in the gut of human infants are linked to health and disease later in life. The timing of the appearance of the first gut microbiome, and the consequences for the early life metabolome, are just starting to be defined. Here, we evaluated the gut microbiome, proteome and metabolome in 88 African-American newborns using faecal samples collected in the first few days of life. Gut bacteria became detectable using molecular methods by 16 h after birth. Detailed analysis of the three most common species, Escherichia coli, Enterococcus faecalis and Bacteroides vulgatus, did not suggest a genomic signature for neonatal gut colonization. The appearance of bacteria was associated with reduced abundance of approximately 50 human proteins, decreased levels of free amino acids and an increase in products of bacterial fermentation, including acetate and succinate. Using flux balance modelling and in vitro experiments, we provide evidence that fermentation of amino acids provides a mechanism for the initial growth of E. coli, the most common early colonizer, under anaerobic conditions. These results provide a deep characterization of the first microbes in the human gut and show how the biochemical environment is altered by their appearance.

[Source](https://www.nature.com/articles/s41564-020-0694-0)

10.1038/s41564-020-0694-0

---

## Salivary Factors that Maintain the Normal Oral Commensal Microflora.
 14.04.2020, _GH Carpenter_


The oral microbiome is one of the most stable ecosystems in the body and yet the reasons for this are still unclear. As well as being stable, it is also highly diverse which can be ascribed to the variety of niches available in the mouth. Previous studies have focused on the microflora in disease-either caries or periodontitis-and only recently have they considered factors that maintain the normal microflora. This has led to the perception that the microflora proliferate in nutrient-rich periods during oral processing of foods and drinks and starves in between times. In this review, evidence is presented which shows that the normal flora are maintained on a diet of salivary factors including urea, lactate, and salivary protein degradation. These factors are actively secreted by salivary glands which suggests these factors are important in maintaining normal commensals in the mouth. In addition, the immobilization of SIgA in the mucosal pellicle indicates a mechanism to retain certain bacteria that does not rely on the bacterial-centric mechanisms such as adhesins. By examining the salivary metabolome, it is clear that protein degradation is a key nutrient and the availability of free amino acids increases resistance to environmental stresses.

[Source](https://doi.org/10.1177/0022034520915486)

10.1177/0022034520915486

---

## Metagenomic Next-Generation Sequencing of Rectal Swabs for the Surveillance of Antimicrobial Resistant Organisms on the Illumina Miseq and Oxford MinION Platforms

[Source](https://doi.org/10.1101/2020.04.16.044214)

---

## Mortality of native and invasive ladybirds co-infected by ectoparasitic and entomopathogenic fungi

[Source](https://doi.org/10.1101/2020.04.17.045930)

---

## The Microbes Found in the Honey of New York City Beehives

[Source](https://doi.org/10.1101/2020.04.16.042762)

---

## Correlation between autochthonous microbial communities and key odorants during the fermentation of red pepper (<em>Capsicum annuum L.</em>)

[Source](https://doi.org/10.1016/j.fm.2020.103510)

---

## Microbial Versatility in Varied Environments: Microbes in Sensitive Environments

[Source](https://doi.org/10.1007/978-981-15-3028-9)

---

## The seasonal variation of microbial communities in drinking water sources in Shanghai

[Source](https://doi.org/10.1016/j.jclepro.2020.121604)

---

## LDPE microplastics significantly alter the temporal turnover of soil microbial communities

[Source](https://doi.org/10.1016/j.scitotenv.2020.138682)

---

## High abundance of <em>Ralstonia solanacearum</em> changed tomato rhizosphere microbiome and metabolome

[Source](https://doi.org/10.1186/s12870-020-02365-9)

---

## First evidence of virus-like particles in the bacterial symbionts of <em>Bryozoa</em>

[Source](https://doi.org/10.1101/2020.04.16.045880)

---

## Microbial community development on model particles in the deep sulfidic waters of the Black Sea

[Source](https://doi.org/10.1111/1462-2920.15024)

---

## The stepwise assembly of the neonatal virome is modulated by breastfeeding

[Source](https://doi.org/10.1038/s41586-020-2192-1)

---

## Bayesian classification, anomaly detection, and survival analysis using network inputs with application to the microbiome

[Source](https://arxiv.org/abs/2004.04765)

---

## Expanding the diversity of bacterioplankton isolates and modeling isolation efficacy with large scale dilution-to-extinction cultivation

[Source](https://doi.org/10.1101/2020.04.17.046896)

---

## Catalyzing rapid discovery of gold-precipitating bacterial lineages with university students

[Source](https://doi.org/10.7717/peerj.8925)

---

## Science-ing from home

[Source](https://www.nature.com/articles/d41586-020-00935-3?WT.ec_id=NATURE-20200416&utm_source=nature_etoc&utm_medium=email&utm_campaign=20200416&sap-outbound-id=EEEC46E999825D4EB57C42CB14A79A64C6B18124)

---

## Lean, mean, learning machines

[Source](https://doi.org/10.1038/s41579-020-0357-4)

---

