## Different Characteristics in Gut Microbiome between Advanced Adenoma Patients and Colorectal Cancer Patients by Metagenomic Analysis.
 01.12.2022, _S Han, J Zhuang, Y Pan, W Wu, K Ding_


The occurrence and development of colorectal cancer (CRC) and advanced adenoma (AA) are closely related to the gut microbiome, and AA has a high cancerization progression rate to CRC. Current studies have revealed that bacteriological analysis cannot identify CRC from AA. The objective was to explore microbial targets that could identify CRC and AA from a microecological perspective and to figure out the best way to identify CRC based on fecal microbes. The metagenomic sequencing data were used to describe the gut microbiome profile and analyze the differences between microbial abundance and microbial single nucleotide polymorphism (SNP) characteristics in AA and CRC patients. It was found that there were no significant differences in the diversity between the two groups. The abundance of bacteria (e.g., Firmicutes, Clostridia, and Blautia), fungi (Hypocreales), archaea (Methanosarcina, Methanoculleus, and Methanolacinia), and viruses (Alphacoronavirus, Sinsheimervirus, and Gammaretrovirus) differed between AA and CRC patients. Multiple machine-learning algorithms were used to establish prediction models, aiming to identify CRC and AA. The accuracy of the random forest (RF) model based on the gut microbiome was 86.54%. Nevertheless, the accuracy of SNP was 92.31% in identifying CRC from AA. In conclusion, using microbial SNP was the best method to identify CRC, it was superior to using the gut microbiome, and it could provide new targets for CRC screening. IMPORTANCE There are differences in characteristic microorganisms between AA and CRC. However, current studies have indicated that bacteriological analysis cannot identify CC from AA, and thus, we wondered if there were some other targets that could be used to identify CRC from AA in the gut microbiome. The differences of SNPs in the gut microbiota of intraindividuals were significantly smaller than those of interindividuals. In addition, compared with intestinal microbes, SNP was less affected by time with certain stability. It was discovered that microbial SNP was better than the gut microbiome for identifying CRC from AA. Therefore, screening characteristic microbial SNP could provide a new research direction for identifying CRC from AA.

[Source](https://doi.org/10.1128/spectrum.01593-22)

10.1128/spectrum.01593-22

---

## The Spatial Features and Temporal Changes in the Gut Microbiota of a Healthy Chinese Population.
 01.12.2022, _W Zhang, N Han, T Zhang, Y Qiang, X Peng, X Li, B Kan_


In this study, we aimed to understand the characteristics of the gut microbial composition in a healthy Chinese population and to evaluate if they differed across different regions. In addition, we aimed to understand the changes in the gut microbial composition over time. We collected 239 fecal samples from healthy Chinese adults living in four regions and performed a 1-year time cohort study in a small population in Beijing. The Chinese gut microbiota share 34 core bacterial genera and 39 core bacterial species, which exist in all collected samples. Several disease-related microorganisms (DRMs), virulence factors, and antibiotic resistance genes were found in one or more healthy Chinese samples. Differences in gut microbiota were observed in samples from different regions, locations, individuals, and time points. Compared to other factors, time was associated with a lower degree of change in the gut microbiota. Our findings revealed spatial and temporal changes in the gut microbiota of healthy Chinese individuals. Compared to fecal microbiomes of 152 samples in the publicly released the Human Microbiome Project (HMP) project from the United States, samples in this study have higher variability in the fecal microbiome, with higher richness, Shannon diversity indices, and Pielou evenness indexes, at both the genus and species levels. The microbiota data obtained in this study will provide a detailed basis for further understanding the composition of the gut microbiota in the healthy Chinese population. IMPORTANCE China accounts for approximately 1/5th of the world's total population. Differences in environment, ethnicity, and living habits could impart unique features to the structure of the gut microbiota of Chinese individuals. In 2016, we started to investigate healthy Chinese people and their gut microbiomes. Phase I results for 16S rRNA amplicons have been released. However, owing to the limitations of 16S rRNA amplicon sequencing, the gut microbiome of a healthy Chinese population could not be examined thoroughly at the species level, and the detailed changes in the gut microbiota over time need to be investigated. To address these knowledge gaps, we started a phase II study and investigated the basis for variations in the gut microbiome composition in a healthy Chinese population at the species level using shotgun metagenomics technology. In the phase II study, we also conducted a time scale analysis of fecal samples from healthy Chinese subjects, as a pioneered study, which quantitatively clarified the changes in the gut microbiota at both the spatial and temporal levels and elucidated the distribution pattern of DRMs in healthy Chinese individuals.

[Source](https://doi.org/10.1128/spectrum.01310-22)

10.1128/spectrum.01310-22

---

## The Gut Microbiota is Important for the Maintenance of Blood-Cerebrospinal Fluid Barrier Integrity.
 01.12.2022, _EG Knox, CMK Lynch, YS Lee, CM O'Driscoll, G Clarke, JF Cryan, MR Aburto_


The gut microbiota communicates with the brain though several pathways including the vagus nerve, immune system, microbial metabolites, and through the endocrine system. Pathways along the humoral/immune gut-microbiota-brain axis comprise of a series of vascular and epithelial barriers including the intestinal epithelial barrier, gut-vascular barrier, blood brain barrier, and blood-cerebrospinal fluid barrier. Of these barriers, the relationship between the gut microbiota and blood-cerebrospinal fluid barrier is yet to be fully defined. Here, using a germ-free mouse model, we aimed to assess the relationship between the gut microbiota and the integrity of the blood-cerebrospinal fluid barrier which is localized to the choroid plexus epithelium. Using confocal microscopy, we visualized the tight junction protein zonula occludens-1, an integral aspect of choroid plexus integrity, as well as the choroid plexus fenestrated capillaries. Quantification of tight junction proteins via network analysis led to the observation that there was a decrease in the zonula occludens-1 network organization in germ-free mice, however, we did not observe any differences in capillary structure. Taken together, these data indicate that the blood- cerebrospinal fluid barrier is another barrier along the gut-microbiota-brain axis. Future studies are required to elucidate its relative contribution in signaling from microbiota to the brain.

[Source](https://doi.org/10.1111/ejn.15878)

10.1111/ejn.15878

---

## Evaluation of co-circulating pathogens and microbiome from COVID-19 infections.
 01.12.2022, _JB Thissen, MD Morrison, N Mulakken, WC Nelson, C Daum, S Messenger, DA Wadford, C Jaing_


Co-infections or secondary infections with SARS-CoV-2 have the potential to affect disease severity and morbidity. Additionally, the potential influence of the nasal microbiome on COVID-19 illness is not well understood. In this study, we analyzed 203 residual samples, originally submitted for SARS-CoV-2 testing, for the presence of viral, bacterial, and fungal pathogens and non-pathogens using a comprehensive microarray technology, the Lawrence Livermore Microbial Detection Array (LLMDA). Eighty-seven percent of the samples were nasopharyngeal samples, and 23% of the samples were oral, nasal and oral pharyngeal swabs. We conducted bioinformatics analyses to examine differences in microbial populations of these samples, as a proxy for the nasal and oral microbiome, from SARS-CoV-2 positive and negative specimens. We found 91% concordance with the LLMDA relative to a diagnostic RT-qPCR assay for detection of SARS-CoV-2. Sixteen percent of all the samples (32/203) revealed the presence of an opportunistic bacterial or frank viral pathogen with the potential to cause co-infections. The two most detected bacteria, Streptococcus pyogenes and Streptococcus pneumoniae, were present in both SARS-CoV-2 positive and negative samples. Human metapneumovirus was the most prevalent viral pathogen in the SARS-CoV-2 negative samples. Sequence analysis of 16S rRNA was also conducted to evaluate bacterial diversity and confirm LLMDA results.

[Source](https://doi.org/10.1371/journal.pone.0278543)

10.1371/journal.pone.0278543

---

## A Clinical Study Provides the First Direct Evidence That Interindividual Variations in Fecal β-Lactamase Activity Affect the Gut Mycobiota Dynamics in Response to β-Lactam Antibiotics

[Source](https://doi.org/10.1128/mbio.02880-22)

---

## Clostridia isolated from helminth-colonized humans promote the life cycle of <em>Trichuris</em> species

[Source](https://doi.org/10.1016/j.celrep.2022.111725)

---

## Zooming into the structure of the microbiome

[Source](https://doi.org/10.1038/s41579-022-00834-6)

---

## <em>Saussurea involucrata</em> oral liquid regulates gut microbiota and serum metabolism during alleviation of collagen-induced arthritis in rats

[Source](https://doi.org/10.1002/ptr.7681)

---

## Prebiotic oligofructose protects against high-fat diet-induced obesity by changing the gut microbiota, intestinal mucus production, glycosylation and secretion.
 03.12.2022, _P Paone, F Suriano, C Jian, K Korpela, NM Delzenne, M Van Hul, A Salonen, PD Cani_


_Mice, Animals, Prebiotics, Diet, High-Fat, Gastrointestinal Microbiome, Glycosylation, Diabetes Mellitus, Type 2, Obesity, Bacteroidetes, Mucus_
Obesity is a major risk factor for the development of type 2 diabetes and cardiovascular diseases, and gut microbiota plays a key role in influencing the host energy homeostasis. Moreover, obese mice have a different gut microbiota composition, associated with an alteration of the intestinal mucus layer, which represents the interface between the bacteria and the host. We previously demonstrated that prebiotic treatment with oligofructose (FOS) counteracted the effects of diet-induced obesity, together with changes in the gut microbiota composition, but it is not known if the intestinal mucus layer could be involved. In this study, we found that, in addition to preventing high-fat diet (HFD) induced obesity in mice, the treatment with FOS increased the expression of numerous genes involved in mucus production, glycosylation and secretion, the expression of both secreted and transmembrane mucins, and the differentiation and number of goblet cells. These results were associated with significant changes in the gut microbiota composition, with FOS significantly increasing the relative and absolute abundance of the bacterial genera Odoribacter, Akkermansia, two unknown Muribaculaceae and an unknown Ruminococcaceae. Interestingly, all these bacterial genera had a negative association with metabolic parameters and a positive association with markers of the mucus layer. Our study shows that FOS treatment is able to prevent HFD-induced metabolic disorders, at least in part, by acting on all the processes of the mucus production. These data suggest that targeting the mucus and the gut microbiota by using prebiotics could help to prevent or mitigate obesity and related disorders.

[Source](https://doi.org/10.1080/19490976.2022.2152307)

10.1080/19490976.2022.2152307

---

## Predicting Personalized Responses to Dietary Fiber Interventions: Opportunities for Modulation of the Gut Microbiome to Improve Health.
 29.11.2022, _CR Kok, D Rose, R Hutkins_


Inadequate dietary fiber consumption has become common across industrialized nations, accompanied by changes in gut microbial composition and a dramatic increase in chronic metabolic diseases. The human gut microbiome harbors genes that are required for the digestion of fiber, resulting in the production of end products that mediate gastrointestinal and systemic benefits to the host. Thus, the use of fiber interventions has attracted increasing interest as a strategy to modulate the gut microbiome and improve human health. However, considerable interindividual differences in gut microbial composition have resulted in variable responses toward fiber interventions. This variability has led to observed nonresponder individuals and highlights the need for personalized approaches to effectively redirect the gut ecosystem. In this review, we summarize strategies used to address the responder and nonresponder phenomenon in dietary fiber interventions and propose a targeted approach to identify predictive features based on knowledge of fiber metabolism and machine learning approaches. Expected final online publication date for the Annual Review of Food Science and Technology, Volume 14 is March 2023. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.

[Source](https://doi.org/10.1146/annurev-food-060721-015516)

10.1146/annurev-food-060721-015516

---

## Gut dysbiosis induces the development of mastitis through a reduction in host anti-inflammatory enzyme activity by endotoxemia.
 03.12.2022, _C Zhao, X Hu, L Bao, K Wu, Y Zhao, K Xiang, S Li, Y Wang, M Qiu, L Feng, X Meng, N Zhang, Y Fu_


_Female, Humans, Animals, Cattle, Mice, Dysbiosis, Endotoxemia, Lipopolysaccharides, Alkaline Phosphatase, Escherichia coli, Mastitis, Anti-Inflammatory Agents, Inflammation, Proteobacteria_
Mounting experimental evidence has shown that the gut microbiota plays a significant role in the pathogenesis of mastitis, and clinical investigations have found that the occurrence of mastitis is correlated with ruminal dysbiosis. However, the underlying mechanism by which the ruminal microbiota participates in the development of mastitis remains unknown.

[Source](https://doi.org/10.1186/s40168-022-01402-z)

10.1186/s40168-022-01402-z

---

## Chronic clinical signs of upper respiratory tract disease associate with gut and respiratory microbiomes in a cohort of domestic felines

[Source](https://doi.org/10.1371/journal.pone.0268730)

---

## Bio-Organic Fertilizer Promotes Pear Yield by Shaping the Rhizosphere Microbiome Composition and Functions.
 01.12.2022, _Z Wang, T Yang, X Mei, N Wang, X Li, Q Yang, C Dong, G Jiang, J Lin, Y Xu, Q Shen, A Jousset, S Banerjee_


Bio-organic fertilizers (BOF) containing both organic amendments and beneficial microorganisms have been consistently shown to improve soils fertility and yield. However, the exact mechanisms which link amendments and yields remain disputed, and the complexity of bio-organic fertilizers may work in parallel in several ways. BOF may directly improve yield by replenishing soil nutrients or introducing beneficial microbial genes or indirectly by altering the soil microbiome to enrich native beneficial microorganisms. In this work, we aim to disentangle the relative contributions of direct and indirect effects on pear yield. We treated pear trees with either chemical fertilizer or organic fertilizer with/without the plant-beneficial bacterium Bacillus velezensis SQR9. We then assessed, in detail, soil physicochemical and biological properties (metagenome sequencing) as well as pear yield. We then evaluated the relative importance of direct and indirect effects of soil amendments on pear yield. Both organic treatments increased plant yield by up to 20%, with the addition of bacteria tripling the increase driven by organic fertilizer alone. This increase could be linked to alterations in soil physicochemical properties, bacterial community function, and metabolism. Supplementation of organic fertilizer SQR9 increased rhizosphere microbiome richness and functional diversity. Fertilizer-sensitive microbes and functions responded as whole guilds. Pear yield was most positively associated with the Mitsuaria- and Actinoplanes-dominated ecological clusters and with gene clusters involved in ion transport and secondary metabolite biosynthesis. Together, these results suggested that bio-organic fertilizers mainly act indirectly on plant yield by creating soil chemical properties which promote a plant-beneficial microbiome. IMPORTANCE Bio-organic fertilization is a widely used, eco-friendly, sustainable approach to increasing plant productivity in the agriculture and fruit industries. However, it remains unclear whether the promotion of fruit productivity is related to specific changes in microbial inoculants, the resident microbiome, and/or the physicochemical properties of rhizosphere soils. We found that bio-organic fertilizers alter soil chemical properties, thus manipulating specific microbial taxa and functions within the rhizosphere microbiome of pear plants to promote yield. Our work unveils the ecological mechanisms which underlie the beneficial impacts of bio-organic fertilizers on yield promotion in fruit orchards, which may help in the design of more efficient biofertilizers to promote sustainable fruit production.

[Source](https://doi.org/10.1128/spectrum.03572-22)

10.1128/spectrum.03572-22

---

## Impacts of maize hybrids with different nitrogen use efficiency on root-associated microbiota based on distinct rhizosphere soil metabolites.
 01.12.2022, _K Li, L Chen, W Shi, C Hu, Y Sha, G Feng, E Wang, W Chen, X Sui, G Mi_


Plant genotypes shape root-associated microbiota that affect plant nutrient acquisition and productivity. It is unclear how maize hybrids modify root-associated microbiota and their functions and relationship with nitrogen use efficiency (NUE) by regulating rhizosphere soil metabolites. Here, two N-efficient (NE) (ZD958, DMY3) and two N-inefficient (NIE) maize hybrids (YD9953, LY99) were used to investigate this issue under low N (60 kg N ha-1 , LN) and high N (180 kg N ha-1 , HN) field conditions. NE hybrids had higher yield than NIE hybrids under LN but not HN. NE and NIE hybrids recruited only distinct root-associated bacterial microbiota in LN. The bacterial network stability was stronger in NE than NIE hybrids. Compared with NIE hybrids, NE hybrids recruited more bacterial taxa that have been described as plant growth-promoting rhizobacteria (PGPR), and less related to denitrification and N competition; this resulted in low N2 O emission and high rhizosphere NO3 - -N accumulation. NE and NIE hybrids had distinct rhizosphere soil metabolite patterns, and their specific metabolites were closely related to microbiota and specific genera under LN. Our findings reveal the relationships among plant NUE, rhizosphere soil metabolites, root-associated microbiota, and soil nutrient cycling, and this information is informative for breeding NE crops. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1111/1462-2920.16293)

10.1111/1462-2920.16293

---

## Coupled C, H, N, S and Fe biogeochemical cycles operating in the continental deep subsurface of the Iberian Pyrite Belt

[Source](https://doi.org/10.1111/1462-2920.16291)

---

## Long-Read Sequencing Improves Recovery of Picoeukaryotic Genomes and Zooplankton Marker Genes from Marine Metagenomes.
 30.11.2022, _NV Patin, KD Goodwin_


Long-read sequencing offers the potential to improve metagenome assemblies and provide more robust assessments of microbial community composition and function than short-read sequencing. We applied Pacific Biosciences (PacBio) CCS (circular consensus sequencing) HiFi shotgun sequencing to 14 marine water column samples and compared the results with those for short-read metagenomes from the corresponding environmental DNA samples. We found that long-read metagenomes varied widely in quality and biological information. The community compositions of the corresponding long- and short-read metagenomes were frequently dissimilar, suggesting higher stochasticity and/or bias associated with PacBio sequencing. Long reads provided few improvements to the assembly qualities, gene annotations, and prokaryotic metagenome-assembled genome (MAG) binning results. However, only long reads produced high-quality eukaryotic MAGs and contigs containing complete zooplankton marker gene sequences. These results suggest that high-quality long-read metagenomes can improve marine community composition analyses and provide important insight into eukaryotic phyto- and zooplankton genetics, but the benefits may be outweighed by the inconsistent data quality. IMPORTANCE Ocean microbes provide critical ecosystem services, but most remain uncultivated. Their communities can be studied through shotgun metagenomic sequencing and bioinformatic analyses, including binning draft microbial genomes. However, most sequencing to date has been done using short-read technology, which rarely yields genome sequences of key microbes like SAR11. Long-read sequencing can improve metagenome assemblies but is hampered by technological shortcomings and high costs. In this study, we compared long- and short-read sequencing of marine metagenomes. We found a wide range of long-read metagenome qualities and minimal improvements to microbiome analyses. However, long reads generated draft genomes of eukaryotic algal species and provided full-length marker gene sequences of zooplankton species, including krill and copepods. These results suggest that long-read sequencing can provide greater genetic insight into the wide diversity of eukaryotic phyto- and zooplankton that interact as part of and with the marine microbiome.

[Source](https://doi.org/10.1128/msystems.00595-22)

10.1128/msystems.00595-22

---

## The Application of Spearman Partial Correlation for Screening Predictors of Weight Loss in a Multiomics Dataset

[Source](https://doi.org/10.1089/omi.2022.0135)

---

## Sampling of Human Microbiomes to Screen for Antibiotic-Producing Commensals.
 01.12.2022, _B Torres Salazar, A Lange, L Camus, S Heilbronner_


_Humans, Anti-Bacterial Agents, Microbiota, Skin, Nose, Soil_
Soil-derived microorganisms have been sampled intensively throughout the last decades in order to discover bacterial strains that produce new antibiotics. The increasing emergence of multidrug-resistant bacteria and the constant high demand for new antibiotic classes are leading to the sampling and investigation of new microbiomes that contain antimicrobial producers. Human-associated microbiomes are therefore gaining more and more attention. This chapter presents a detailed description of how human microbiomes can be sampled and how microbiota members from skin and nasal samples can be isolated. Different methods for antimicrobial compound screening are presented.

[Source](https://doi.org/10.1007/978-1-0716-2855-3_3)

10.1007/978-1-0716-2855-3_3

---

