## Distinct Changes Occur in the Human Breast Milk Microbiome Between Early and Established Lactation in Breastfeeding Guatemalan Mothers.
 03.03.2021, _E Gonzalez, NJB Brereton, C Li, L Lopez Leyva, NW Solomons, LB Agellon, ME Scott, KG Koski_


Human breast milk contains a diverse community of bacteria, but as breast milk microbiome studies have largely focused on mothers from high income countries where few women breastfeed to 6 months, the temporal changes in the breast milk microbiome that occur during later lactation stages have not been explored. For this cross-sectional study, microbiota from breast milk samples of Mam-Mayan mothers living in eight remote rural communities in the Western Highlands of Guatemala were analyzed. All mothers delivered vaginally and breastfed their infants for 6 months. Breast milk from 76 unrelated mothers was used to compare two lactation stages, either "early" (6-46 days post-partum, n = 33) or "late" (109-184 days post-partum, n = 43). Breast milk microbial communities were assessed using 16S ribosomal RNA gene sequencing and lactation stages were compared using DESeq2 differential abundance analysis. A total of 1,505 OTUs were identified, including 287 which could be annotated as putative species. Among several maternal factors, lactation stage explained microbiome variance and inertia in ordination with the most significance (p &lt; 0.001). Differential abundance analysis identified 137 OTUs as significantly higher in either early or late lactation. These included a general shift from Staphylococcus and Streptococcus species in early lactation to Sphingobium and Pseudomonas species in late lactation. Species enriched in early lactation included putative commensal bacteria known to colonize the infant oral and intestinal tracts whereas species enriched in late lactation had a uniform functional trait associated with aromatic compound degradation. Differentially abundant species also included several species which have not previously been reported within breast milk, such as Janthinobacterium agaricidamnosum, Novosphingobium clariflavum, Ottowia beijingensis, and Flavobacterium cucumis. These discoveries describe temporal changes to the breast milk microbiome of healthy Guatemalan mothers from early to late lactation. Collectively, these findings illustrate how studying under-represented human populations might advance our understanding of factors that modulate the human milk microbiome in low and middle income countries (LMIC).

[Source](https://doi.org/10.3389/fmicb.2021.557180)

10.3389/fmicb.2021.557180

---

## Spermidine-induced recovery of human dermal structure and barrier function by skin microbiome.
 22.02.2021, _G Kim, M Kim, M Kim, C Park, Y Yoon, DH Lim, H Yeo, S Kang, YG Lee, NI Beak, J Lee, S Kim, JY Kwon, WW Choi, C Lee, KW Yoon, H Park, DG Lee_


An unbalanced microbial ecosystem on the human skin is closely related to skin diseases and has been associated with inflammation and immune responses. However, little is known about the role of the skin microbiome on skin aging. Here, we report that the Streptococcus species improved the skin structure and barrier function, thereby contributing to anti-aging. Metagenomic analyses showed the abundance of Streptococcus in younger individuals or those having more elastic skin. Particularly, we isolated Streptococcus pneumoniae, Streptococcus infantis, and Streptococcus thermophilus from face of young individuals. Treatment with secretions of S. pneumoniae and S. infantis induced the expression of genes associated with the formation of skin structure and the skin barrier function in human skin cells. The application of culture supernatant including Streptococcal secretions on human skin showed marked improvements on skin phenotypes such as elasticity, hydration, and desquamation. Gene Ontology analysis revealed overlaps in spermidine biosynthetic and glycogen biosynthetic processes. Streptococcus-secreted spermidine contributed to the recovery of skin structure and barrier function through the upregulation of collagen and lipid synthesis in aged cells. Overall, our data suggest the role of skin microbiome into anti-aging and clinical applications.

[Source](https://doi.org/10.1038/s42003-020-01619-4)

10.1038/s42003-020-01619-4

---

## Influence of dapivirine vaginal ring use on cervicovaginal immunity and functional microbiome in adolescent girls.
 12.02.2021, _C Farr Zuend, L Noël-Romas, S Hoger, S McCorriser, G Westmacott, J Marrazzo, SL Hillier, C Dezzutti, K Squires, KE Bunge, A Burgener_


The antiretroviral-based dapivirine vaginal ring reduced HIV risk among women in phase III clinical trials. However, limited data exists on the impact of dapivirine on the vaginal microenvironment in adolescents.

[Source](https://journals.lww.com/aidsonline/Fulltext/2021/03010/Influence_of_dapivirine_vaginal_ring_use_on.3.aspx)

10.1097/QAD.0000000000002751

---

## Mapping the entire functionally active endometrial microbiota.
 18.02.2021, _A Sola-Leyva, E Andrés-León, NM Molina, LC Terron-Camero, J Plaza-Díaz, MJ Sáez-Lara, MC Gonzalvo, R Sánchez, S Ruíz, L Martínez, S Altmäe_


Does endometrium harbour functionally active microorganisms and whether the microbial composition differs between proliferative and mid-secretory phases?

[Source](https://doi.org/10.1093/humrep/deaa372)

10.1093/humrep/deaa372

---

## Genomic relatedness and clinical significance of <em>Streptococcus mitis </em>strains isolated from the urogenital tract of sexual partners

[Source](https://doi.org/10.1099/mgen.0.000535)

---

## Identification of an N-acetylneuraminic acid-presenting bacteria isolated from a human microbiome.
 04.03.2021, _Z Han, PS Thuy-Boun, W Pfeiffer, VF Vartabedian, A Torkamani, JR Teijaro, DW Wolan_


N-Acetylneuraminic acid is the most abundant sialic acid (SA) in humans and is expressed as the terminal sugar on intestinal mucus glycans. Several pathogenic bacteria harvest and display host SA on their own surfaces to evade Siglec-mediated host immunity. While previous studies have identified bacterial enzymes associated with SA catabolism, no reported methods permit the selective labeling, tracking, and quantitation of SA-presenting microbes within complex multi-microbial systems. We combined metabolic labeling, click chemistry, 16S rRNA gene, and whole-genome sequencing to track and identify SA-presenting microbes from a cultured human fecal microbiome. We isolated a new strain of Escherichia coli that incorporates SA onto its own surface and encodes for the nanT, neuA, and neuS genes necessary for harvesting and presenting SA. Our method is applicable to the identification of SA-presenting bacteria from human, animal, and environmental microbiomes, as well as providing an entry point for the investigation of surface-expressed SA-associated structures.

[Source](https://doi.org/10.1038/s41598-021-83875-w)

10.1038/s41598-021-83875-w

---

## Fecal microbiota transplantation in HIV: A pilot placebo-controlled study.
 22.02.2021, _S Serrano-Villar, A Talavera-Rodríguez, MJ Gosalbes, N Madrid, JA Pérez-Molina, RJ Elliott, B Navia, VF Lanza, A Vallejo, M Osman, F Dronda, S Budree, J Zamora, C Gutiérrez, M Manzano, MJ Vivancos, R Ron, J Martínez-Sanz, S Herrera, U Ansa, A Moya, S Moreno_


Changes in the microbiota have been linked to persistent inflammation during treated HIV infection. In this pilot double-blind study, we study 30 HIV-infected subjects on antiretroviral therapy (ART) with a CD4/CD8 ratio &lt; 1 randomized to either weekly fecal microbiota capsules or placebo for 8 weeks. Stool donors were rationally selected based on their microbiota signatures. We report that fecal microbiota transplantation (FMT) is safe, not related to severe adverse events, and attenuates HIV-associated dysbiosis. FMT elicits changes in gut microbiota structure, including significant increases in alpha diversity, and a mild and transient engraftment of donor's microbiota during the treatment period. The greater engraftment seems to be achieved by recent antibiotic use before FMT. The Lachnospiraceae and Ruminococcaceae families, which are typically depleted in people with HIV, are the taxa more robustly engrafted across time-points. In exploratory analyses, we describe a significant amelioration in the FMT group in intestinal fatty acid-binding protein (IFABP), a biomarker of intestinal damage that independently predicts mortality. Gut microbiota manipulation using a non-invasive and safe strategy of FMT delivery is feasible and deserves further investigation. Trial number: NCT03008941.

[Source](https://doi.org/10.1038/s41467-021-21472-1)

10.1038/s41467-021-21472-1

---

## Gastrointestinal microbiota composition predicts peripheral inflammatory state during treatment of human tuberculosis.
 22.02.2021, _MF Wipperman, SK Bhattarai, CK Vorkas, VS Maringati, Y Taur, L Mathurin, K McAulay, SC Vilbrun, D Francois, J Bean, KF Walsh, C Nathan, DW Fitzgerald, MS Glickman, V Bucci_


The composition of the gastrointestinal microbiota influences systemic immune responses, but how this affects infectious disease pathogenesis and antibiotic therapy outcome is poorly understood. This question is rarely examined in humans due to the difficulty in dissociating the immunologic effects of antibiotic-induced pathogen clearance and microbiome alteration. Here, we analyze data from two longitudinal studies of tuberculosis (TB) therapy (35 and 20 individuals) and a cross sectional study from 55 healthy controls, in which we collected fecal samples (for microbiome analysis), sputum (for determination of Mycobacterium tuberculosis (Mtb) bacterial load), and peripheral blood (for transcriptomic analysis). We decouple microbiome effects from pathogen sterilization by comparing standard TB therapy with an experimental TB treatment that did not reduce Mtb bacterial load. Random forest regression to the microbiome-transcriptome-sputum data from the two longitudinal datasets reveals that renormalization of the TB inflammatory state is associated with Mtb pathogen clearance, increased abundance of Clusters IV and XIVa Clostridia, and decreased abundance of Bacilli and Proteobacteria. We find similar associations when applying machine learning to peripheral gene expression and microbiota profiling in the independent cohort of healthy individuals. Our findings indicate that antibiotic-induced reduction in pathogen burden and changes in the microbiome are independently associated with treatment-induced changes of the inflammatory response of active TB, and the response to antibiotic therapy may be a combined effect of pathogen killing and microbiome driven immunomodulation.

[Source](https://doi.org/10.1038/s41467-021-21475-y)

10.1038/s41467-021-21475-y

---

## Regulation of neonatal IgA production by the maternal microbiota.
 23.02.2021, _Q Mu, BK Swartwout, M Edwards, J Zhu, G Lee, K Eden, X Cabana-Puig, DK McDaniel, J Mao, L Abdelhamid, RM Brock, IC Allen, CM Reilly, XM Luo_


Infants are prone to enteric infections due to an underdeveloped immune system. The maternal microbiota, through shaping the neonatal microbiota, helps establish a strong immune system in infants. We and others have observed the phenomenon of enhanced early neonatal immunoglobulin A (IgA) production in preweaning immunocompetent mice nursed by immunodeficient dams. Here, we show that this enhancement of IgA in neonates results from maternally derived microbiota. In addition, we have found that the neonatal IgA production can be induced by Lactobacillus reuteri, which is enriched in the milk of immunodeficient dams. Moreover, we show that while the production of neonatal IgA is dependent on neonatal T cells, the immunodeficient maternal microbiota-mediated enhancement of neonatal IgA has a T cell-independent component. Indeed, this enhancement may be dependent on type 3 innate lymphoid cells in the neonatal small intestinal lamina propria. Interestingly, maternal microbiota-induced neonatal IgA does not cross-react with common enteric pathogens. Future investigations will determine the functional consequences of having this extra IgA.

[Source](https://doi.org/10.1073/pnas.2015691118)

10.1073/pnas.2015691118

---

## Lysates of <em>Methylococcus capsulatus</em> Bath induce a lean-like microbiota, intestinal FoxP3+RORγt+IL-17+ Tregs and improve metabolism

[Source](https://doi.org/10.1038/s41467-021-21408-9)

---

## Sex-dependent modulation of immune development by secretory IgA-coated Lactobacillus reuteri isolated from breast milk.
 22.02.2021, _C Qi, M Ding, S Li, Q Zhou, D Li, R Yu, J Sun_


Lactobacilli, commonly present in human breast milk, appear to colonize the neonatal gut and provide protection to infants against various infections, thereby promoting immune development. This study examined the potential probiotic role of breast milk-derived Lactobacillus reuteri FN041 in immune development in mice. The FN041 were gavaged either to BALB/c dams (n = 6/group) during the lactation period or to their offspring (n = 6/sex per intervention) after weaning separately (cointervention). All interventions induced increased intestinal barriers in 5-wk-old offspring, especially in the females. Immunoglobulin A plasmocytes in ileal tissue and secretory IgA (sIgA) in ileal contents increased in all 5-wk-old offspring of cointervention. The activation of mRNA expression of 17 genes was sex-dependent, especially in 5-wk-old offspring. Broader genes were regulated in female mice. The effect of cointervention on the Shannon index of total microbiota is sex-related. The Shannon index of sIgA-coated microbiota increased in both sexes. The sIgA-coated microbiota showed intergroup differences according to β diversity, especially in female mice that showed an increase in Bifidobacterium of Actinobacteria. The sIgA-coated Bifidobacteria was positively correlated with mRNA expression of Tlr9. The sIgA-coated Lactobacillus in male offspring was negatively correlated with mRNA expression of Cldn2. In conclusion, L. reuteri FN041 promoted the production of intestinal sIgA and the expression of genes related to antimicrobial peptides in the offspring and enhanced the function of the mucosal barrier, depending on sex and treatment manner.

[Source](https://doi.org/10.3168/jds.2020-19437)

10.3168/jds.2020-19437

---

## Oral berberine improves brain dopa/dopamine levels to ameliorate Parkinson's disease by regulating gut microbiota.
 24.02.2021, _Y Wang, Q Tong, SR Ma, ZX Zhao, LB Pan, L Cong, P Han, R Peng, H Yu, Y Lin, TL Gao, JW Shou, XY Li, XF Zhang, ZW Zhang, J Fu, BY Wen, JB Yu, X Cao, JD Jiang_


The phenylalanine-tyrosine-dopa-dopamine pathway provides dopamine to the brain. In this process, tyrosine hydroxylase (TH) is the rate-limiting enzyme that hydroxylates tyrosine and generates levodopa (L-dopa) with tetrahydrobiopterin (BH4) as a coenzyme. Here, we show that oral berberine (BBR) might supply H• through dihydroberberine (reduced BBR produced by bacterial nitroreductase) and promote the production of BH4 from dihydrobiopterin; the increased BH4 enhances TH activity, which accelerates the production of L-dopa by the gut bacteria. Oral BBR acts in a way similar to vitamins. The L-dopa produced by the intestinal bacteria enters the brain through the circulation and is transformed to dopamine. To verify the gut-brain dialog activated by BBR's effect, Enterococcus faecalis or Enterococcus faecium was transplanted into Parkinson's disease (PD) mice. The bacteria significantly increased brain dopamine and ameliorated PD manifestation in mice; additionally, combination of BBR with bacteria showed better therapeutic effect than that with bacteria alone. Moreover, 2,4,6-trimethyl-pyranylium tetrafluoroborate (TMP-TFB)-derivatized matrix-assisted laser desorption mass spectrometry (MALDI-MS) imaging of dopamine identified elevated striatal dopamine levels in mouse brains with oral Enterococcus, and BBR strengthened the imaging intensity of brain dopamine. These results demonstrated that BBR was an agonist of TH in Enterococcus and could lead to the production of L-dopa in the gut. Furthermore, a study of 28 patients with hyperlipidemia confirmed that oral BBR increased blood/fecal L-dopa by the intestinal bacteria. Hence, BBR might improve the brain function by upregulating the biosynthesis of L-dopa in the gut microbiota through a vitamin-like effect.

[Source](https://doi.org/10.1038/s41392-020-00456-5)

10.1038/s41392-020-00456-5

---

## <em>Lactobacillus reuteri</em>-derived extracellular vesicles maintain intestinal immune homeostasis against lipopolysaccharide-induced inflammatory responses in broilers

[Source](https://doi.org/10.1186/s40104-020-00532-4)

---

## Dicer-like proteins influence Arabidopsis root microbiota independent of RNA-directed DNA methylation.
 04.03.2021, _R Kaushal, L Peng, SK Singh, M Zhang, X Zhang, JI Vílchez, Z Wang, D He, Y Yang, S Lv, Z Xu, RJL Morcillo, W Wang, W Huang, PW Paré, CP Song, JK Zhu, R Liu, W Zhong, P Ma, H Zhang_


Plants are naturally associated with root microbiota, which are microbial communities influential to host fitness. Thus, it is important to understand how plants control root microbiota. Epigenetic factors regulate the readouts of genetic information and consequently many essential biological processes. However, it has been elusive whether RNA-directed DNA methylation (RdDM) affects root microbiota assembly.

[Source](https://doi.org/10.1186/s40168-020-00966-y)

10.1186/s40168-020-00966-y

---

## Succession of the Resident Soil Microbial Community in Response to Periodic Inoculations.
 27.02.2021, _Z Wang, Z Chen, GA Kowalchuk, Z Xu, X Fu, EE Kuramae_


To maintain the beneficial effects of microbial inoculants on plant and soil, repeated inoculation represents a promising option. Until now, the impacts of one-off inoculation on the native microbiome have been explored, but it remains unclear how long and to what extent the periodic inoculations would affect the succession of the resident microbiome in bulk soil. Here we examined the dynamic responses of plant growth, soil functions and resident bacterial community in the bulk soil to periodic inoculations of phosphate-solubilizing and N2-fixing bacteria alone or in combination. Compared to single-strain inoculation, co-inoculation better stimulated plant growth and soil nutrients. However, the benefits from inoculants did not increase with repeated inoculations and were not maintained after transplanting to a different site. In response to microbial inoculants, three patterns of shifts in bacterial composition were observed - fold increased, fold decreased, and resilience. The periodic inoculations impacted the succession course of resident bacterial communities in bulk soil, mainly driven by changes in soil pH and nitrate, resulting in the development of three main cluster types throughout the investigation. The single and mixed inoculants transiently modulated the variation in the resident community in association with soil pH and C/N, but finally the community established and showed resilience to following inoculations. Consequently, the necessity of repeated inoculations should be reconsidered, and while the different microbial inoculants showed distinct impacts on resident microbiome succession, communities ultimately exhibited resilience.IMPORTANCEIntroducing beneficial microbes to the plant-soil system is an environmentally friendly approach to improve crop yield and soil environment. Numerous studies have attempted to reveal the impacts of inoculation on rhizosphere microbiome. However, little is known about the effectiveness of periodic inoculations on soil functioning. In addition, the impact persistence of repeated inoculations on the native community remains unclear. Here, we track the succession traits of resident microbiome in the bulk soil across a growing season and identify the taxa clusters that diversely respond to periodic inoculation. Crucially, we compare the development of resident community composition with and without inoculation, thus providing new insight into understanding the interactions between resident microbes and intruders. Finally, we conclude that initial inoculation plays a more important role in influencing the whole system, and the native microbial community exhibits traits of resilience, but no resistance, to the subsequent inoculations.

[Source](https://aem.asm.org/content/early/2021/02/22/AEM.00046-21)

10.1128/AEM.00046-21

---

## Climate change alters temporal dynamics of alpine soil microbial functioning and biogeochemical cycling via earlier snowmelt.
 23.02.2021, _AAD Broadbent, HSK Snell, A Michas, WJ Pritchard, L Newbold, I Cordero, T Goodall, N Schallhart, R Kaufmann, RI Griffiths, M Schloter, M Bahn, RD Bardgett_


Soil microbial communities regulate global biogeochemical cycles and respond rapidly to changing environmental conditions. However, understanding how soil microbial communities respond to climate change, and how this influences biogeochemical cycles, remains a major challenge. This is especially pertinent in alpine regions where climate change is taking place at double the rate of the global average, with large reductions in snow cover and earlier spring snowmelt expected as a consequence. Here, we show that spring snowmelt triggers an abrupt transition in the composition of soil microbial communities of alpine grassland that is closely linked to shifts in soil microbial functioning and biogeochemical pools and fluxes. Further, by experimentally manipulating snow cover we show that this abrupt seasonal transition in wide-ranging microbial and biogeochemical soil properties is advanced by earlier snowmelt. Preceding winter conditions did not change the processes that take place during snowmelt. Our findings emphasise the importance of seasonal dynamics for soil microbial communities and the biogeochemical cycles that they regulate. Moreover, our findings suggest that earlier spring snowmelt due to climate change will have far reaching consequences for microbial communities and nutrient cycling in these globally widespread alpine ecosystems.

[Source](https://doi.org/10.1038/s41396-021-00922-0)

10.1038/s41396-021-00922-0

---

## Microbial Community Structure Driven by a Volcanic Gradient in Glaciers of the Antarctic Archipelago South Shetland

[Source](https://doi.org/10.3390/microorganisms9020392)

---

## Effect of polymer type on the colonization of plastic pellets by marine bacteria.
 28.02.2021, _J Hansen, J Melchiorsen, N Ciacotich, L Gram, EC Sonnenschein_


Plastic is omnipresent in the oceans and serves as surface for biofilm-forming microorganisms. Plastic debris comprises different polymers, which may influence microbial colonization and here, we evaluated if polymer type affects bacterial biofilm formation. Quantifying the biofilm on polyethylene (PE), polypropylene (PP) or polystyrene (PS) pellets by six marine bacterial strains (Vibrio, Pseudoalteromonas, Phaeobacter) demonstrated that each strain had a unique colonization behavior with either a preference for PS or PP over the other polymer types or no preference for a specific plastic type. PE, PP, and PS pellets were exposed to natural seawater microbiota using free-living or total communities as inoculum. Microbial assembly as determined by 16S rRNA (V4) amplicon sequencing was affected by the composition of the initial inoculum and also by the plastic type. Known polymer and hydrocarbon degraders such as Paraglaciecola, Oleibacter, and Hydrogenophaga were found in the plastic biofilms. Thus, on a community level, bacterial colonization on plastic is influenced by the microorganisms as well as the polymer type, and also individual strains can demonstrate polymer-specific colonization.

[Source](https://doi.org/10.1093/femsle/fnab026)

10.1093/femsle/fnab026

---

## Lifestyle of sponge symbiont phages by host prediction and correlative microscopy.
 19.02.2021, _MT Jahn, T Lachnit, SM Markert, C Stigloher, L Pita, M Ribes, BE Dutilh, U Hentschel_


Bacteriophages (phages) are ubiquitous elements in nature, but their ecology and role in animals remains little understood. Sponges represent the oldest known extant animal-microbe symbiosis and are associated with dense and diverse microbial consortia. Here we investigate the tripartite interaction between phages, bacterial symbionts, and the sponge host. We combined imaging and bioinformatics to tackle important questions on who the phage hosts are and what the replication mode and spatial distribution within the animal is. This approach led to the discovery of distinct phage-microbe infection networks in sponge versus seawater microbiomes. A new correlative in situ imaging approach ('PhageFISH-CLEM') localised phages within bacterial symbiont cells, but also within phagocytotically active sponge cells. We postulate that the phagocytosis of free virions by sponge cells modulates phage-bacteria ratios and ultimately controls infection dynamics. Prediction of phage replication strategies indicated a distinct pattern, where lysogeny dominates the sponge microbiome, likely fostered by sponge host-mediated virion clearance, while lysis dominates in seawater. Collectively, this work provides new insights into phage ecology within sponges, highlighting the importance of tripartite animal-phage-bacterium interplay in holobiont functioning. We anticipate that our imaging approach will be instrumental to further understanding of viral distribution and cellular association in animal hosts.

[Source](https://doi.org/10.1038/s41396-021-00900-6)

10.1038/s41396-021-00900-6

---

## Bacteria and Metabolic Potential in Karst Caves Revealed by Intensive Bacterial Cultivation and Genome Assembly.
 01.03.2021, _HZ Zhu, ZF Zhang, N Zhou, CY Jiang, BJ Wang, L Cai, HM Wang, SJ Liu_


Karst caves are widely distributed subsurface systems, and the microbiomes therein are proposed to be the driving force for cave evolution and biogeochemical cycling. In past years, culture-independent studies on the microbiomes of cave systems have been conducted, yet intensive microbial cultivation is still needed to validate the sequence-derived hypothesis and to disclose the microbial functions in cave ecosystems. In this study, the microbiomes of two karst caves in Guizhou Province in southwest China were examined. A total of 3,562 bacterial strains were cultivated from rock, water, and sediment samples, and 329 species (including 14 newly described species) of 102 genera were found. We created a cave bacterial genome collection of 218 bacterial genomes from a karst cave microbiome through the extraction of 204 database-derived genomes and de novo sequencing of 14 new bacterial genomes. The cultivated genome collection obtained in this study and the metagenome data from previous studies were used to investigate the bacterial metabolism and potential involvement in the carbon, nitrogen, and sulfur biogeochemical cycles in the cave ecosystem. New N2-fixing Azospirillum and alkane-oxidizing Oleomonas species were documented in the karst cave microbiome. Two pcaIJ clusters of the β-ketoadipate pathway that were abundant in both the cultivated microbiomes and the metagenomic data were identified, and their representatives from the cultivated bacterial genomes were functionally demonstrated. This large-scale cultivation of a cave microbiome represents the most intensive collection of cave bacterial resources to date and provides valuable information and diverse microbial resources for future cave biogeochemical research.IMPORTANCE Karst caves are oligotrophic environments that are dark and humid and have a relatively stable annual temperature. The diversity of bacteria and their metabolisms are crucial for understanding the biogeochemical cycling in cave ecosystems. We integrated large-scale bacterial cultivation with metagenomic data mining to explore the compositions and metabolisms of the microbiomes in two karst cave systems. Our results reveal the presence of a highly diversified cave bacterial community, and 14 new bacterial species were described and their genomes sequenced. In this study, we obtained the most intensive collection of cultivated microbial resources from karst caves to date and predicted the various important routes for the biogeochemical cycling of elements in cave ecosystems.

[Source](https://aem.asm.org/content/87/6/e02440-20)

10.1128/AEM.02440-20

---

## MetaHiC phage-bacteria infection network reveals active cycling phages of the healthy human gut.
 26.02.2021, _M Marbouty, A Thierry, GA Millot, R Koszul_


Bacteriophages play important roles in regulating the intestinal human microbiota composition, dynamics and homeostasis, and characterizing their bacterial hosts is needed to understand their impact. We applied a metagenomic Hi-C approach on 10 healthy human gut samples to unveil a large infection network encompassing more than 6,000 interactions bridging a metagenomic assembled genomes (MAGs) and a phage sequence, allowing to study in situ phage-host ratio. Whereas three-quarter of these sequences likely correspond to dormant prophages, 5% exhibit a much higher coverage than their associated MAG, representing potentially actively replicating phages. We detected 17 sequences of members of the crAss-like phage family, whose hosts diversity remained until recently relatively elusive. For each of them, a unique bacterial host was identified, all belonging to different genus of Bacteroidetes. Therefore, metaHiC deciphers infection network of microbial population with a high-specificity paving the way to dynamic analysis of mobile genetic elements in complex ecosystems.

[Source](https://elifesciences.org/articles/60608)

10.7554/eLife.60608

---

## On the evolutionary origins of host-microbe associations.
 23.02.2021, _M Sieber, A Traulsen, H Schulenburg, AE Douglas_


Many microorganisms with high prevalence in host populations are beneficial to the host and maintained by specialized transmission mechanisms. Although microbial promotion of host fitness and specificity of the associations undoubtedly enhance microbial prevalence, it is an open question whether these symbiotic traits are also a prerequisite for the evolutionary origin of prevalent microbial taxa. To address this issue, we investigate how processes without positive microbial effects on host fitness or host choice can influence the prevalence of certain microbes in a host population. Specifically, we develop a theoretical model to assess the conditions under which particular microbes can become enriched in animal hosts even when they are not providing a specific benefit to a particular host. We find increased prevalence of specific microbes in a host when both show some overlap in their lifecycles, and especially when both share dispersal routes across a patchy habitat distribution. Our results emphasize that host enrichment per se is not a reliable indicator of beneficial host-microbe interactions. The resulting increase in time spent associated with a host may nevertheless give rise to new selection conditions, which can favor microbial adaptations toward a host-associated lifestyle, and, thus, it could be the foundation for subsequent evolution of mutually beneficial coevolved symbioses.

[Source](https://doi.org/10.1073/pnas.2016487118)

10.1073/pnas.2016487118

---

## Ecology-guided prediction of cross-feeding interactions in the human gut microbiome.
 01.03.2021, _A Goyal, T Wang, V Dubinkina, S Maslov_


Understanding a complex microbial ecosystem such as the human gut microbiome requires information about both microbial species and the metabolites they produce and secrete. These metabolites are exchanged via a large network of cross-feeding interactions, and are crucial for predicting the functional state of the microbiome. However, till date, we only have information for a part of this network, limited by experimental throughput. Here, we propose an ecology-based computational method, GutCP, using which we predict hundreds of new experimentally untested cross-feeding interactions in the human gut microbiome. GutCP utilizes a mechanistic model of the gut microbiome with the explicit exchange of metabolites and their effects on the growth of microbial species. To build GutCP, we combine metagenomic and metabolomic measurements from the gut microbiome with optimization techniques from machine learning. Close to 65% of the cross-feeding interactions predicted by GutCP are supported by evidence from genome annotations, which we provide for experimental testing. Our method has the potential to greatly improve existing models of the human gut microbiome, as well as our ability to predict the metabolic profile of the gut.

[Source](https://doi.org/10.1038/s41467-021-21586-6)

10.1038/s41467-021-21586-6

---

## Using a multiple-delivery-mode training approach to develop local capacity and infrastructure for advanced bioinformatics in Africa.
 25.02.2021, _V Ras, G Botha, S Aron, K Lennard, I Allali, S Claassen-Weitz, KS Mwaikono, D Kennedy, JR Holmes, G Rendon, S Panji, CJ Fields, N Mulder_


With more microbiome studies being conducted by African-based research groups, there is an increasing demand for knowledge and skills in the design and analysis of microbiome studies and data. However, high-quality bioinformatics courses are often impeded by differences in computational environments, complicated software stacks, numerous dependencies, and versions of bioinformatics tools along with a lack of local computational infrastructure and expertise. To address this, H3ABioNet developed a 16S rRNA Microbiome Intermediate Bioinformatics Training course, extending its remote classroom model. The course was developed alongside experienced microbiome researchers, bioinformaticians, and systems administrators, who identified key topics to address. Development of containerised workflows has previously been undertaken by H3ABioNet, and Singularity containers were used here to enable the deployment of a standard replicable software stack across different hosting sites. The pilot ran successfully in 2019 across 23 sites registered in 11 African countries, with more than 200 participants formally enrolled and 106 volunteer staff for onsite support. The pulling, running, and testing of the containers, software, and analyses on various clusters were performed prior to the start of the course by hosting classrooms. The containers allowed the replication of analyses and results across all participating classrooms running a cluster and remained available posttraining ensuring analyses could be repeated on real data. Participants thus received the opportunity to analyse their own data, while local staff were trained and supported by experienced experts, increasing local capacity for ongoing research support. This provides a model for delivering topic-specific bioinformatics courses across Africa and other remote/low-resourced regions which overcomes barriers such as inadequate infrastructures, geographical distance, and access to expertise and educational materials.

[Source](https://doi.org/10.1371/journal.pcbi.1008640)

10.1371/journal.pcbi.1008640

---

## Amplicon-based skin microbiome profiles collected by tape stripping with different adhesive film dressings: a comparative study.
 23.02.2021, _K Ogai, K Shibata, N Takahashi, K Ogura, S Okamoto, J Sugama_


Medical film dressings have been used to obtain skin microbiota for skin microbiome studies, although their adhesive force may be so strong that the skin could be injured when applied to those who have fragile skin, such as older people. Several products with less adhesive force are available, although their applicability for skin microbiome studies remains unknown. This study aimed to test whether the dressings with less adhesive force could be used for amplicon-based skin microbiome studies. A set of three different film dressings, with acrylic, urethane, or silicone adhesive, was applied to the back skin of nine healthy young participants. The copy number of the 16S ribosomal RNA (rRNA) gene, microbial compositions, and alpha and beta diversity indices were analyzed by amplicon analysis of the 16S rRNA gene using next-generation sequencing and were compared among the three film dressings.

[Source](https://doi.org/10.1186/s12866-021-02122-4)

10.1186/s12866-021-02122-4

---

## High-Resolution Differentiation of Enteric Bacteria in Premature Infant Fecal Microbiomes Using a Novel rRNA Amplicon.
 17.02.2021, _J Graf, N Ledala, MJ Caimano, E Jackson, D Gratalo, D Fasulo, MD Driscoll, S Coleman, AP Matson_


Identifying and tracking microbial strains as microbiomes evolve are major challenges in the field of microbiome research. We utilized a new sequencing kit that combines DNA extraction with PCR amplification of a large region of the rRNA operon and downstream bioinformatic data analysis. Longitudinal microbiome samples of coadmitted twins from two different neonatal intensive care units (NICUs) were analyzed using an ∼2,500-base amplicon that spans the 16S and 23S rRNA genes and mapped to a new, custom 16S-23S rRNA database. Amplicon sequence variants (ASVs) inferred using DADA2 provided sufficient resolution for the differentiation of rRNA variants from closely related but not previously sequenced Klebsiella, Escherichia coli, and Enterobacter strains, among the first bacteria colonizing the gut of these infants after admission to the NICU. Distinct ASV groups (fingerprints) were monitored between coadmitted twins over time, demonstrating the potential to track the source and spread of both commensals and pathogens. The high-resolution taxonomy obtained from long amplicon sequencing enables the tracking of strains temporally and spatially as microbiomes are established in infants in the hospital environment.IMPORTANCE Achieving strain-level resolution is a major obstacle for source tracking and temporal studies of microbiomes. In this study, we describe a novel deep-sequencing approach that provides species- and strain-level resolution of the neonatal microbiome. Using Klebsiella, E. coli, and Enterobacter as examples, we could monitor their temporal dynamics after antibiotic treatment and in pairs of twins. The strain-level resolution, combined with the greater sequencing depth and decreased cost per read of PacBio Sequel 2, enables this advantageous source- and strain-tracking analysis method to be implemented widely across more complex microbiomes.

[Source](https://mbio.asm.org/content/12/1/e03656-20)

10.1128/mBio.03656-20

---

## https://archive.vn/SScWt

[Source](https://archive.vn/SScWt)

---

## Sulfurized Polyacrylonitrile for High-Performance Lithium-Sulfur Batteries: In-Depth Computational Approach Revealing Multiple Sulfur's Reduction Pathways and Hidden Li+ Storage Mechanisms for Extra Discharge Capacity.
 14.01.2021, _S Perez Beltran, PB Balbuena_


Like no other sulfur host material, polyacrylonitrile-derived sulfurized carbon (SPAN) promises improved electrochemical performance for lithium-sulfur batteries, based on its compatibility with carbonate solvents and ability to prevent self-discharge and shuttle effect. However, a complete understanding of the SPAN's lithiation mechanism is still missing because its structural features vary widely with synthesis conditions, and its electrochemical performance deviates from elemental sulfur. This study continues our research on the elucidation of the SPAN's structural characteristics and lithiation mechanisms via computational approaches. Our models reproduce most experimental data regarding carbon's graphitization level and conjugated ordering, sulfur-carbon covalent bonding, sulfur loading, and elemental composition. Our simulations emulate the discharge voltage observed in experiments for the first discharge, which reveals that sulfur follows multiple reduction pathways based on its interaction with the carbon backbone. Sulfur reduction takes place above 1.0 V vs Li/Li+ mostly in the SPAN-like material, with no long-chain lithium polysulfide formation. Below 1.0 V vs Li/Li+, the backbone's electrochemical activity occurs via multiple C-Li and N-Li interactions, mostly with edge carbon atoms and pyridinic nitrogen. Moreover, we identify Li+ binding sites throughout the graphitized backbone that might lead to prohibited energy costs for Li+ deintercalation, which may explain the irreversible capacity loss between the first and second discharges. This work improves understanding of lithiation mechanisms in sulfurized carbon, which is useful for rationally designing SPAN synthesis pathways tailored to increase sulfur loading and enhanced electrochemical performance.

[Source](https://microbiomedigest.com/2021/03/04/march-4-2021/?like_comment=27154&_wpnonce=6493d7301e)

10.1021/acsami.0c17537

---

## <span>Like</span>

[Source](https://microbiomedigest.com/2021/03/04/march-4-2021/?like_comment=27154&_wpnonce=6493d7301e)

---

