## Inferring the effectiveness of government interventions against COVID-19

[Source](https://science.sciencemag.org/content/early/2020/12/15/science.abd9338.full)

---

## COVID 19 mortality: Probable role of microbiome to explain disparity.
 14.12.2020, _P Kumar, B Chander_


_Bacterial Infections, Biodiversity, COVID-19, Developing Countries, Environmental Microbiology, Health Status Disparities, Humans, Interferon Type I, Microbiota, Models, Biological, Pandemics, Poverty, Rural Population, SARS-CoV-2_
There is a significant difference between COVID 19 associated mortality between different countries. Generally the number of deaths per million population are higher in the developed countries despite better health care efficiency, drinking water quality and expected healthy life span (HALE) at the time of birth. Developing and underdeveloped countries on the other hand have lower mortality even with higher rural and slum populations along with incidence of diarrhea because of lack of sanitation. We analyzed data from 122 countries out of which 80 were high or upper middle income and 42 were low or low middle income countries. There was statistically significant positive correlation between COVID 19 deaths /million population and water current score, health efficiency, and HALE. Statistically significant negative correlation was observed with % rural population and fraction of diarrhea because of inadequate sanitation for all ages. Moreover analysis of 51 countries showed that there is significant negative correlation between COVID 19 deaths /million population and proportion of total population living in slums. We propose that high microbial exposure particularly gram negative bacteria can possibly induce interferon type I which might have a protective effect against COVID 19 since the countries with less mortality also tend to have lack of sanitation and high incidence of attendant diseases. So, far none of the predictive models have taken into account immune status of populations engendered by environmental microbial exposure or microbiome. There might be a need to look at dynamics of COVID 19 pandemic using immune perspective. The approach can potentially inform better policies including interventions.

[Source](https://www.sciencedirect.com/science/article/pii/S0306987720321125)

10.1016/j.mehy.2020.110209

---

## Theory of (antimicrobial) relativity: When competitors determine a species’ drug sensitivity.

[Source](https://www.biorxiv.org/content/10.1101/2020.12.14.422780v1.abstract)

---

## Metagenomic Next-Generation Sequencing of Nasopharyngeal Specimens Collected from Confirmed and Suspect COVID-19 Patients.
 14.12.2020, _HH Mostafa, JA Fissel, B Fanelli, Y Bergman, V Gniazdowski, M Dadlani, KC Carroll, RR Colwell, PJ Simner_


_Bacteria, COVID-19, Coinfection, Computational Biology, High-Throughput Nucleotide Sequencing, Humans, Metagenome, Metagenomics, Microbiota, Nasopharynx, RNA, Viral, SARS-CoV-2, Specimen Handling_
Metagenomic next-generation sequencing (mNGS) offers an agnostic approach for emerging pathogen detection directly from clinical specimens. In contrast to targeted methods, mNGS also provides valuable information on the composition of the microbiome and might uncover coinfections that may associate with disease progression and impact prognosis. To evaluate the use of mNGS for detecting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and/or other infecting pathogens, we applied direct Oxford Nanopore long-read third-generation metatranscriptomic and metagenomic sequencing. Nasopharyngeal (NP) swab specimens from 50 patients under investigation for CoV disease 2019 (COVID-19) were sequenced, and the data were analyzed by the CosmosID bioinformatics platform. Further, we characterized coinfections and the microbiome associated with a four-point severity index. SARS-CoV-2 was identified in 77.5% (31/40) of samples positive by RT-PCR, correlating with lower cycle threshold (Ct) values and fewer days from symptom onset. At the time of sampling, possible bacterial or viral coinfections were detected in 12.5% of SARS-CoV-2-positive specimens. A decrease in microbial diversity was observed among COVID-19-confirmed patients (Shannon diversity index, P = 0.0082; Chao richness estimate, P = 0.0097; Simpson diversity index, P = 0.018), and differences in microbial communities were linked to disease severity (P = 0.022). Furthermore, statistically significant shifts in the microbiome were identified among SARS-CoV-2-positive and -negative patients, in the latter of whom a higher abundance of Propionibacteriaceae (P = 0.028) and a reduction in the abundance of Corynebacterium accolens (P = 0.025) were observed. Our study corroborates the growing evidence that increased SARS-CoV-2 RNA detection from NP swabs is associated with the early stages rather than the severity of COVID-19. Further, we demonstrate that SARS-CoV-2 causes a significant change in the respiratory microbiome. This work illustrates the utility of mNGS for the detection of SARS-CoV-2, for diagnosing coinfections without viral target enrichment or amplification, and for the analysis of the respiratory microbiome.IMPORTANCE SARS-CoV-2 has presented a rapidly accelerating global public health crisis. The ability to detect and analyze viral RNA from minimally invasive patient specimens is critical to the public health response. Metagenomic next-generation sequencing (mNGS) offers an opportunity to detect SARS-CoV-2 from nasopharyngeal (NP) swabs. This approach also provides information on the composition of the respiratory microbiome and its relationship to coinfections or the presence of other organisms that may impact SARS-CoV-2 disease progression and prognosis. Here, using direct Oxford Nanopore long-read third-generation metatranscriptomic and metagenomic sequencing of NP swab specimens from 50 patients under investigation for COVID-19, we detected SARS-CoV-2 sequences by applying the CosmosID bioinformatics platform. Further, we characterized coinfections and detected a decrease in the diversity of the microbiomes in these patients. Statistically significant shifts in the microbiome were identified among COVID-19-positive and -negative patients, in the latter of whom a higher abundance of Propionibacteriaceae and a reduction in the abundance of Corynebacterium accolens were observed. Our study also corroborates the growing evidence that increased SARS-CoV-2 RNA detection from NP swabs is associated with the early stages of disease rather than with severity of disease. This work illustrates the utility of mNGS for the detection and analysis of SARS-CoV-2 from NP swabs without viral target enrichment or amplification and for the analysis of the respiratory microbiome.

[Source](https://mbio.asm.org/content/11/6/e01969-20)

10.1128/mBio.01969-20

---

## Variation in oral microbiome is associated with future risk of lung cancer among never-smokers.
 15.12.2020, _HD Hosgood, Q Cai, X Hua, J Long, J Shi, Y Wan, Y Yang, C Abnet, BA Bassig, W Hu, BT Ji, M Klugman, Y Xiang, YT Gao, JY Wong, W Zheng, N Rothman, XO Shu, Q Lan_


To prospectively investigate whether diversity in oral microbiota is associated with risk of lung cancer among never-smokers.

[Source](https://thorax.bmj.com/content/early/2020/11/15/thoraxjnl-2020-215542)

10.1136/thoraxjnl-2020-215542

---

## Effect of an 8-week Exercise Training on Gut Microbiota in Physically Inactive Older Women.
 15.12.2020, _F Zhong, X Wen, M Yang, HY Lai, H Momma, L Cheng, X Sun, R Nagatomi, C Huang_


Exercise can alter the composition of gut microbiota. However, studies examining the effects of exercise on gut microbiota in the elderly are lacking. This study aims to investigate whether an 8-week exercise training affect gut microbiota in physically inactive elderly women. Fourteen women were randomly assigned to either exercise group or control group. Repeated-measures analysis of variance was used to reveal changes in gut microbiota. Alpha diversity did not change significantly. A tendency to form 2 clusters was observed for operational taxonomic units (OTU) after intervention. At phylum, class, and order levels, a significant difference was observed between two groups for Fusobacteria (F=5.257, P=0.045), Betaproteobacteria (F=5.149, P=0.047), and Bifidobacteriales (F=7.624, P=0.020). A significant interaction was observed between two groups for Actinobacteria (F=8.434, P=0.016). At family and genus levels, a signiﬁcant main effect of groups was observed in Bifidobacteriaceae (F=7.624, P=0.020), Bifidobacterium (F=7.404, P=0.022), and Gemmiger (F=5.881, P=0.036). These findings indicate that an 8-week exercise training may induce partial changes in relative abundance and OTU clustering of gut microbiota in physically inactive elderly women. Also, exercise may increase the abundance of bacteria associated with anti-inflammation such as Verrucomicrobia, reduce the abundance of bacteria associated with pro-inflammation such as Proteobacteria.

[Source](https://www.thieme-connect.de/products/ejournals/abstract/10.1055/a-1301-7011)

10.1055/a-1301-7011

---

## Vitamin D metabolites and the gut microbiome in older men.
 14.12.2020, _RL Thomas, L Jiang, JS Adams, ZZ Xu, J Shen, S Janssen, G Ackermann, D Vanderschueren, S Pauwels, R Knight, ES Orwoll, DM Kado_


_Aged, Aged, 80 and over, Butyrates, Calcifediol, Calcitriol, Cross-Sectional Studies, DNA, Bacterial, Feces, Gastrointestinal Microbiome, Humans, Independent Living, Intestinal Mucosa, Male, Phylogeny, RNA, Ribosomal, 16S_
The vitamin D receptor is highly expressed in the gastrointestinal tract where it transacts gene expression. With current limited understanding of the interactions between the gut microbiome and vitamin D, we conduct a cross-sectional analysis of 567 older men quantifying serum vitamin D metabolites using LC-MSMS and defining stool sub-Operational Taxonomic Units from16S ribosomal RNA gene sequencing data. Faith's Phylogenetic Diversity and non-redundant covariate analyses reveal that the serum 1,25(OH)2D level explains 5% of variance in α-diversity. In β-diversity analyses using unweighted UniFrac, 1,25(OH)2D is the strongest factor assessed, explaining 2% of variance. Random forest analyses identify 12 taxa, 11 in the phylum Firmicutes, eight of which are positively associated with either 1,25(OH)2D and/or the hormone-to-prohormone [1,25(OH)2D/25(OH)D] "activation ratio." Men with higher levels of 1,25(OH)2D and higher activation ratios, but not 25(OH)D itself, are more likely to possess butyrate producing bacteria that are associated with better gut microbial health.

[Source](https://www.nature.com/articles/s41467-020-19793-8)

10.1038/s41467-020-19793-8

---

## Longitudinal profiling of the macaque vaginal microbiome reveals similarities to diverse human vaginal communities: implications for use as a pre-clinical model for bacterial vaginosis

[Source](https://www.biorxiv.org/content/10.1101/2020.12.14.422805v1.abstract)

---

## The earthworm microbiome is resilient to exposure to biocidal metal nanoparticles.
 14.12.2020, _E Swart, T Goodall, P Kille, DJ Spurgeon, C Svendsen_


_Animals, Gastrointestinal Microbiome, Metal Nanoparticles, Microbiota, Oligochaeta, Soil_
Environmental pollution can disrupt the interactions between animals and their symbiotic bacteria, which can lead to adverse effects on the host even in the absence of direct chemical toxicity. It is therefore crucial to understand how environmental pollutants affect animal microbiomes, especially for those chemicals that are designed to target microbes. Here, we study the effects of two biocidal nanoparticles (NPs) (Ag and CuO) on the soil bacterial community and the resident gut microbiome of the earthworm Eisenia fetida over a 28-day period using metabarcoding techniques. Exposures to NPs were conducted following OECD test guidelines and effects on earthworm reproduction and juvenile biomass were additionally recorded in order to compare effects on the host to effects on microbiomes. By employing a full concentration series, we were able to link pollutants to microbiome effects in high resolution. Multivariate analysis, differential abundance analysis and species sensitivity distribution analysis showed that Ag-NPs are more toxic to soil bacteria than CuO-NPs. In contrast to the strong effects of CuO-NPs and Ag-NPs on the soil bacterial community, the earthworm gut microbiome is largely resilient to exposure to biocidal NPs. Despite this buffering effect, CuO-NPs did negatively affect the relative abundance of some earthworm symbionts, including 'Candidatus Lumbricincola'. Changes in the soil bacterial community and the earthworm microbiome occur at total copper concentrations often found or modelled to occur in agricultural fields, demonstrating that soil bacterial communities and individual taxa in the earthworm microbiome may be at risk from environmental copper exposure including in nanomaterial form.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0269749120363211)

10.1016/j.envpol.2020.115633

---

## Environmental microbiome mapping as a strategy to improve quality and safety in the food industry

[Source](https://www.sciencedirect.com/science/article/abs/pii/S2214799320301405)

---

## Nitrosophilus alvini gen. nov., sp. nov., a hydrogen-oxidizing chemolithoautotroph isolated from a deep-sea hydrothermal vent in the East Pacific Rise, inferred by a genome-based taxonomy of the phylum "Campylobacterota".
 10.12.2020, _T Shiotani, S Mino, W Sato, S Nishikawa, M Yonezawa, SM Sievert, T Sawabe_


A novel bacterium, strain EPR55-1T, was isolated from a deep-sea hydrothermal vent on the East Pacific Rise. The cells were motile rods. Growth was observed at temperatures between 50 and 60°C (optimum, 60°C), at pH values between 5.4 and 8.6 (optimum, pH 6.6) and in the presence of 2.4-3.2% (w/v) NaCl (optimum, 2.4%). The isolate used molecular hydrogen as its sole electron donor, carbon dioxide as its sole carbon source, ammonium as its sole nitrogen source, and thiosulfate, sulfite (0.01 to 0.001%, w/v) or elemental sulfur as its sole sulfur source. Nitrate, nitrous oxide (33%, v/v), thiosulfate, molecular oxygen (0.1%, v/v) or elemental sulfur could serve as the sole electron acceptor to support growth. Phylogenetic analyses based on both 16S rRNA gene sequences and whole genome sequences indicated that strain EPR55-1T belonged to the family Nitratiruptoraceae of the class "Campylobacteria", but it had the distinct phylogenetic relationship with the genus Nitratiruptor. On the basis of the physiological and molecular characteristics of the isolate, the name Nitrosophilus alvini gen. nov. sp. nov. is proposed, with EPR55-1T as the type strain (= JCM 32893T = KCTC 15925T). In addition, it is shown that "Nitratiruptor labii" should be transferred to the genus Nitrtosophilus; the name Nitrosophilus labii comb. nov. (JCM 34002T = DSM 111345T) is proposed for this organism. Furthermore, 16S rRNA gene-based and genome-based analyses showed that Cetia pacifica is phylogenetically associated with Caminibacter species. We therefore propose the reclassification of Cetia pacifica as Caminibacter pacificus comb. nov. (DSM 27783T = JCM 19563T). Additionally, AAI thresholds for genus classification and the reclassification of subordinate taxa within "Campylobacteria" are also evaluated, based on the analyses using publicly available genomes of all the campylobacterial species.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0241366)

10.1371/journal.pone.0241366

---

## A rapid approach to profiling diverse fungal communities using the MinION™ nanopore sequencer.
 14.12.2020, _KK Mafune, BJ Godfrey, DJ Vogt, KA Vogt_


_DNA, Fungal, Fungi, Mycobiome, Nanopore Sequencing_
The Oxford Nanopore Technologies MinION™ sequencer holds the capability to generate long amplicon reads; however, only a small amount of information is available regarding methodological approaches and the ability to identify a broad diversity of fungal taxa. To assess capabilities, three fungal mock communities were sequenced, each of which had varying ratios of 16 taxa. The data were processed through our selected pipeline. The MinION recovered all mock community members, when mixed at equal ratios. When a taxon was represented at a lower ratio, it was not recovered or decreased in relative abundance. Despite high error rates, highly accurate consensus sequences can be derived. This methodological approach identified all mock community taxa, demonstrating the MinION can be used as a practical alternative to profile fungal communities.

[Source](https://www.future-science.com/doi/10.2144/btn-2019-0072)

10.2144/btn-2019-0072

---

## Microbe-seq: high-throughput, single-microbe genomics with strain resolution, applied to a human gut microbiome

[Source](https://www.biorxiv.org/content/10.1101/2020.12.14.422699v1.abstract)

---

## Oral Microbiome Diversity and Species Abundance Linked to Lung Cancer in Nonsmokers

[Source](https://www.genengnews.com/news/oral-microbiome-diversity-and-species-abundance-linked-to-lung-cancer-in-nonsmokers/)

---

