## <em> </em>Bladder cancer, inflammageing and microbiomes

[Source](https://doi.org/10.1038/s41585-022-00611-3)

---

## Gut associated metabolites and their roles in <em>Clostridioides difficile </em>pathogenesis

[Source](https://doi.org/10.1080/19490976.2022.2094672)

---

## Oral and gut dysbiosis leads to functional alterations in Parkinson’s disease

[Source](https://doi.org/10.1038/s41531-022-00351-6)

---

## Transcriptional programming in a<em> Bacteroides</em> consortium

[Source](https://doi.org/10.1038/s41467-022-31614-8)

---

## Gut microbiota composition in colorectal cancer patients is genetically regulated

[Source](https://doi.org/10.1038/s41598-022-15230-6)

---

## Relative contributions of the host genome, microbiome, and environment to the metabolic profile

[Source](https://doi.org/10.1007/s13258-022-01277-2)

---

## Effects of probiotic Lactiplantibacillus plantarum IMC 510 supplementation on metabolic factors in otherwise healthy overweight and obese individuals.
 07.07.2022, _MM Coman, L Miorelli, MV Micioni Di Bonaventura, C Cifani, C Salvesi, A Amedei, S Silvi, MC Verdenelli_


Probiotic supplementation approach offers the possibility to shape the gut microbiota (GM), enabling the development of innovative formulations able to improve intestinal wellbeing and consequently the related body weight modulation and energy metabolism. In the present clinical study, a new potential probiotic supplement based on Lactiplantibacillus plantarum IMC 510 was studied for weight management.

[Source](https://doi.org/10.1111/jam.15703)

10.1111/jam.15703

---

## Effects of probiotic <em>Lactiplantibacillus plantarum </em>IMC 510 supplementation on metabolic factors in otherwise healthy overweight and obese individuals

[Source](https://doi.org/10.1111/jam.15703)

---

## Grape seed and skin extract, a potential prebiotic with anti-obesity effect through gut microbiota modulation.
 06.07.2022, _M Mokrani, K Charradi, F Limam, E Aouani, MC Urdaci_


Obesity is a worldwide health problem and a significant risk factor for diabetes and cardiovascular diseases. Gut microbiota (GM) plays an essential role in obesity, and prebiotics such as polyphenols could be one way to improve microbial dysbiosis-induced obesity.

[Source](https://doi.org/10.1186/s13099-022-00505-0)

10.1186/s13099-022-00505-0

---

## <em>Lactobacillus acidophilus</em> LA85 ameliorates cyclophosphamide-induced immunosuppression by modulating Notch and TLR4/NF-κB signal pathways and remodeling the gut microbiota

[Source](https://doi.org/10.1039/d1fo04331e)

---

## Transplantation of bacteriophages from ulcerative colitis patients shifts the gut bacteriome and exacerbates the severity of DSS colitis.
 07.07.2022, _A Sinha, Y Li, MK Mirzaei, M Shamash, R Samadfam, IL King, CF Maurice_


Inflammatory bowel diseases (IBDs) including Crohn's disease (CD) and ulcerative colitis (UC) are characterized by chronic and debilitating gut inflammation. Altered bacterial communities of the intestine are strongly associated with IBD initiation and progression. The gut virome, which is primarily composed of bacterial viruses (bacteriophages, phages), is thought to be an important factor regulating and shaping microbial communities in the gut. While alterations in the gut virome have been observed in IBD patients, the contribution of these viruses to alterations in the bacterial community and heightened inflammatory responses associated with IBD patients remains largely unknown.

[Source](https://doi.org/10.1186/s40168-022-01275-2)

10.1186/s40168-022-01275-2

---

## <em>Rickettsia helvetica </em>infection is associated with microbiome modulation in <em>Ixodes ricinus </em>collected from humans in Serbia

[Source](https://doi.org/10.1038/s41598-022-15681-x)

---

## Microplastic exposure across trophic levels: effects on the host-microbiota of freshwater organisms.
 06.07.2022, _JE Varg, D Outomuro, W Kunce, L Kuehrer, R Svanbäck, F Johansson_


Microplastics are a pervasive pollutant widespread in the sea and freshwater from anthropogenic sources, and together with the presence of pesticides, they can have physical and chemical effects on aquatic organisms and on their microbiota. Few studies have explored the combined effects of microplastics and pesticides on the host-microbiome, and more importantly, the effects across multiple trophic levels. In this work, we studied the effects of exposure to microplastics and the pesticide deltamethrin on the diversity and abundance of the host-microbiome across a three-level food chain: daphnids-damselfly-dragonflies. Daphnids were the only organism exposed to 1 µm microplastic beads, and they were fed to damselfly larvae. Those damselfly larvae were exposed to deltamethrin and then fed to the dragonfly larvae. The microbiotas of the daphnids, damselflies, and dragonflies were analyzed.

[Source](https://doi.org/10.1186/s40793-022-00429-x)

10.1186/s40793-022-00429-x

---

## Estuarine plastisphere as an overlooked source of N(2)O production

[Source](https://doi.org/10.1038/s41467-022-31584-x)

---

## Uncovering Hidden Members and Functions of the Soil Microbiome Using De Novo Metaproteomics.
 06.07.2022, _JY Lee, HD Mitchell, MC Burnet, R Wu, SC Jenson, ED Merkley, ES Nakayasu, CD Nicora, JK Jansson, KE Burnum-Johnson, SH Payne_


Metaproteomics has been increasingly utilized for high-throughput characterization of proteins in complex environments and has been demonstrated to provide insights into microbial composition and functional roles. However, significant challenges remain in metaproteomic data analysis, including creation of a sample-specific protein sequence database. A well-matched database is a requirement for successful metaproteomics analysis, and the accuracy and sensitivity of PSM identification algorithms suffer when the database is incomplete or contains extraneous sequences. When matched DNA sequencing data of the sample is unavailable or incomplete, creating the proteome database that accurately represents the organisms in the sample is a challenge. Here, we leverage a de novo peptide sequencing approach to identify the sample composition directly from metaproteomic data. First, we created a deep learning model, Kaiko, to predict the peptide sequences from mass spectrometry data and trained it on 5 million peptide-spectrum matches from 55 phylogenetically diverse bacteria. After training, Kaiko successfully identified organisms from soil isolates and synthetic communities directly from proteomics data. Finally, we created a pipeline for metaproteome database generation using Kaiko. We tested the pipeline on native soils collected in Kansas, showing that the de novo sequencing model can be employed as an alternative and complementary method to construct the sample-specific protein database instead of relying on (un)matched metagenomes. Our pipeline identified all highly abundant taxa from 16S rRNA sequencing of the soil samples and uncovered several additional species which were strongly represented only in proteomic data.

[Source](https://doi.org/10.1021/acs.jproteome.2c00334)

10.1021/acs.jproteome.2c00334

---

## Privacy Risks in Microbiome Research: Public Perspectives before and during a Global Pandemic.
 08.07.2022, _A Shin, H Xu_


We assessed public perspectives of microbiome research privacy risks before and after a nationwide emergency was declared in the United States regarding the Covid-19 pandemic. From January to July of 2020, we conducted an online survey of perceived privacy risks of microbiome research among U.S. adults. Among 3,106 participants (the preemergency group), most expressed that the microbiome posed privacy risks similar to those associated with DNA (60.3%) or medical records (50.6%) and that they would prefer detailed explanations (70.2%) of risk in consent materials. Only 8.9% reported moderate to high familiarity with microbiome privacy risks. In adjusted analyses, individuals who participated in the study after the Covid-19 emergency was declared (the Covid-19 emergency group) were less likely to express that microbiome privacy risks were similar to those of DNA or medical records and more likely to report familiarity with the privacy risks of microbiomes. There was a trend toward increased concern after the Covid-19 emergency was declared (p = 0.053). Overall, the study revealed that many U.S. adults believe that microbiome privacy risks are similar to those associated with DNA or medical records, and they prefer detailed explanations in consent documents. Individuals who participated after the Covid-19 emergency was declared reported greater knowledge of microbiome privacy risks but had more concern.

[Source](https://doi.org/10.1002/eahr.500132)

10.1002/eahr.500132

---

## <em> </em>Metagenomics Approaches to Investigate the Neonatal Gut Microbiome

[Source](https://doi.org/10.3389/fped.2022.886627)

---

## <em> </em>Before you go: a packing list for portable DNA sequencing of microbiomes and metagenomes

[Source](https://doi.org/10.1099/mic.0.001220)

---

## A Bayesian method for identifying associations between response variables and bacterial community composition

[Source](https://doi.org/10.1371/journal.pcbi.1010108)

---

## GenBank as a Source to Monitor and Analyze Host-Microbiome Data

[Source](https://doi.org/10.1093/bioinformatics/btac487)

---

## Phylogenies of the 16S rRNA gene and its hypervariable regions lack concordance with core genome phylogenies.
 07.07.2022, _HB Hassler, B Probert, C Moore, E Lawson, RW Jackson, BT Russell, VP Richards_


The 16S rRNA gene is used extensively in bacterial phylogenetics, in species delineation, and now widely in microbiome studies. However, the gene suffers from intragenomic heterogeneity, and reports of recombination and an unreliable phylogenetic signal are accumulating. Here, we compare core gene phylogenies to phylogenies constructed using core gene concatenations to estimate the strength of signal for the 16S rRNA gene, its hypervariable regions, and all core genes at the intra- and inter-genus levels. Specifically, we perform four intra-genus analyses (Clostridium, n = 65; Legionella, n = 47; Staphylococcus, n = 36; and Campylobacter, n = 17) and one inter-genus analysis [41 core genera of the human gut microbiome (31 families, 17 orders, and 12 classes), n = 82].

[Source](https://doi.org/10.1186/s40168-022-01295-y)

10.1186/s40168-022-01295-y

---

