## https://t.co/n3ChfNAE2S

[Source](https://t.co/n3ChfNAE2S)

---

## @merenbey

[Source](https://twitter.com/merenbey?ref_src=twsrc%5Etfw)

---

## pic.twitter.com/72YxwKWIvq

[Source](https://t.co/72YxwKWIvq)

---

## October 31, 2023

[Source](https://twitter.com/MicrobiomeVIF/status/1719311652875612671?ref_src=twsrc%5Etfw)

---

## A human urothelial microtissue model reveals shared colonization and survival strategies between uropathogens and commensals.
 10.11.2023, _C Flores, J Ling, A Loh, RG Maset, A Aw, IJ White, R Fernando, JL Rohn_


_Animals, Humans, Escherichia coli, Fimbriae Proteins, Adhesins, Escherichia coli, Urinary Tract Infections, Escherichia coli Infections_
Urinary tract infection is among the most common infections worldwide, typically studied in animals and cell lines with limited uropathogenic strains. Here, we assessed diverse bacterial species in a human urothelial microtissue model exhibiting full stratification, differentiation, innate epithelial responses, and urine tolerance. Several uropathogens invaded intracellularly, but also commensal Escherichia coli, suggesting that invasion is a shared survival strategy, not solely a virulence hallmark. The E. coli adhesin FimH was required for intracellular bacterial community formation, but not for invasion. Other shared lifestyles included filamentation (Gram-negatives), chaining (Gram-positives), and hijacking of exfoliating cells, while biofilm-like aggregates were formed mainly with Pseudomonas and Proteus. Urothelial cells expelled invasive bacteria in Rab-/LC3-decorated structures, while highly cytotoxic/invasive uropathogens, but not commensals, disrupted host barrier function and strongly induced exfoliation and cytokine production. Overall, this work highlights diverse species-/strain-specific infection strategies and corresponding host responses in a human urothelial microenvironment, providing insights at the microtissue, cell, and molecular level.

[Source](https://www.science.org/doi/full/10.1126/sciadv.adi9834)

10.1126/sciadv.adi9834

---

## @ScienceAdvances

[Source](https://twitter.com/ScienceAdvances?ref_src=twsrc%5Etfw)

---

## @CarlosEBFlores

[Source](https://twitter.com/CarlosEBFlores?ref_src=twsrc%5Etfw)

---

## #UTI

[Source](https://twitter.com/hashtag/UTI?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/gGjoGsuhHW

[Source](https://t.co/gGjoGsuhHW)

---

## @GlobalUti

[Source](https://twitter.com/GlobalUti?ref_src=twsrc%5Etfw)

---

## Autism intervention meta-analysis of early childhood studies (Project AIM): updated systematic review and secondary analysis.
 14.11.2023, _M Sandbank, K Bottema-Beutel, S Crowley LaPoint, JI Feldman, DJ Barrett, N Caldwell, K Dunham, J Crank, S Albarran, T Woynaroski_


To summarize the breadth and quality of evidence supporting commonly recommended early childhood autism interventions and their estimated effects on developmental outcomes.

[Source](https://twitter.com/JennyRohn/status/1722341766584549741?ref_src=twsrc%5Etfw)

10.1136/bmj-2023-076733

---

## November 8, 2023

[Source](https://twitter.com/JennyRohn/status/1722341766584549741?ref_src=twsrc%5Etfw)

---

## <em>Gardnerella pickettii </em>sp. nov. (formerly <em>Gardnerella</em> genomic species 3) and <em>Gardnerella greenwoodii </em>sp. nov. (formerly <em>Gardnerella </em>genomic species 8) isolated from female urinary microbiome 

[Source](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.006140)

---

## <em>Lactobacillus isalae</em> sp. nov., isolated from the female reproductive tract 

[Source](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.006038)

---

## Identification of a DNA-cytosine methyltransferase that impacts global transcription to promote group B streptococcal vaginal colonization

[Source](https://journals.asm.org/doi/10.1128/mbio.02306-23)

---

## @KdoranLab

[Source](https://twitter.com/KdoranLab?ref_src=twsrc%5Etfw)

---

## https://t.co/s4Tqnp9Kvj

[Source](https://t.co/s4Tqnp9Kvj)

---

## November 1, 2023

[Source](https://twitter.com/HaiderSManzer/status/1719688498884604319?ref_src=twsrc%5Etfw)

---

## Aggression: A gut reaction? The effects of the gut microbiome on aggression – Atara Uzan-Yulzari – bioRxiv

[Source](https://www.biorxiv.org/content/10.1101/2023.10.26.564110v1)

---

## @KorenLab

[Source](https://twitter.com/KorenLab?ref_src=twsrc%5Etfw)

---

## @evanelliottlab

[Source](https://twitter.com/evanelliottlab?ref_src=twsrc%5Etfw)

---

## https://t.co/Zs4Y7ziPQH

[Source](https://t.co/Zs4Y7ziPQH)

---

## October 31, 2023

[Source](https://twitter.com/SondraTurjeman/status/1719239282747908560?ref_src=twsrc%5Etfw)

---

## Establishment of tongue microbiota by 18 months of age and determinants of its microbial profile.
 11.10.2023, _S Kageyama, J Ma, M Furuta, T Takeshita, M Asakawa, Y Okabe, Y Yamashita_


Acquisition of oral bacteria early in life is considered to have an important role in the establishment of oral microbiota. In this study, we examined tongue swab samples collected from 216 infants at the 18-month checkup, along with samples collected from them and from their mothers at the 4-month checkup. The bacterial composition of each sample was determined using PacBio single-molecule long-read sequencing of the full-length 16S rRNA gene and amplicon sequence variant (ASV) analysis. At 18 months of age, the tongue microbiota was dominated by Streptococcus salivarius, Neisseria perflava, and Granulicatella adiacens; and the overall bacterial composition was more similar to that of the mothers than at 4 months of age. The bacterial composition at 18 months of age was classified into three microbiota profiles: S. salivarius-dominant, Neisseria-dominant, and infant profile dominated by 4 months specific ASVs such as S. salivarius, Streptococcus lactarius, and Streptococcus peroris. Those with infant profiles exhibited significantly higher rates of current breastfeeding, dental caries or white spot lesions, and dental plaque accumulation compared to infants with the other profiles. Comparison between the S. salivarius- and Neisseria-dominant profiles showed that incomplete weaning, low intake of fruits, and frequent intake of sweetened beverages or sweet snacks were significantly associated with the S. salivarius-dominant profile. Our findings, based on a high-resolution profiling approach, demonstrate that the foundation of the adult tongue microbiota is established by 18 months of age, and the bacterial profile is possibly determined by dietary habits during this period. IMPORTANCE Understanding the development of oral microbiota early in life and the factors that influence it is important for preventing the establishment of dysbiotic oral microbiota later in life. This study demonstrates that the tongue microbiota undergoes early development from 4 to 18 months of age and converges into two types of microbiota showing indications of adult characteristics, with either S. salivarius or Neisseria-dominance. Interestingly, their divergence was strongly determined by their weaning status and the dietary frequencies of sweetened beverages, snacks, and fruits, suggesting that dietary habits during this period might influence the establishment of the oral microbiota. These findings may contribute to the development of novel preventive strategies against oral microbiota-related diseases.

[Source](https://journals.asm.org/doi/full/10.1128/mbio.01337-23)

10.1128/mbio.01337-23

---

## Pet cats may shape the antibiotic resistome of their owner's gut and living environment.
 30.10.2023, _Y Yang, X Hu, S Cai, N Hu, Y Yuan, Y Wu, Y Wang, J Mi, X Liao_


_Animals, Humans, Cats, Anti-Bacterial Agents, Genes, Bacterial, Aminoglycosides, Tetracycline, Enterobacteriaceae_
Companion animals can contribute to the physical and mental health of people and often live in very close association with their owners. However, the antibiotic resistome carried by companion animals and the impact they have on their owners and living environment remain unclear. In this study, we compared the ARG profiles of cats, humans, and their living environments using metagenomic analysis to identify the core ARGs in the cat and human gut and explore the potential impact of cats on ARGs in the human gut through the environment.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01679-8)

10.1186/s40168-023-01679-8

---

## The impact of kit, environment and sampling contamination on the observed microbiome of bovine milk

[Source](https://www.biorxiv.org/content/10.1101/2023.11.07.566052v1)

---

## Early life exposure to broccoli sprouts confers stronger protection against enterocolitis development in an immunological mouse model of inflammatory bowel disease

[Source](https://journals.asm.org/doi/10.1128/msystems.00688-23)

---

## Daily fluctuation of colonic microbiome in response to nutrient substrates in a pig model 

[Source](https://www.nature.com/articles/s41522-023-00453-w)

---

## Assessing the impact of anti-microbial showerheads on the prevalence and abundance of opportunistic pathogens in shower water and shower water-associated aerosols

[Source](https://www.frontiersin.org/articles/10.3389/frmbi.2023.1292571/full)

---

## Global abundance patterns, diversity, and ecology of Patescibacteria in wastewater treatment plant

[Source](https://www.biorxiv.org/content/10.1101/2023.10.25.562895v1)

---

## A high-throughput and low-waste viability assay for microbes.
 03.11.2023, _CT Meyer, GK Lynch, DF Stamo, EJ Miller, A Chatterjee, JM Kralj_


Counting viable cells is a universal practice in microbiology. The colony-forming unit (CFU) assay has remained the gold standard to measure viability across disciplines, but it is time-intensive and resource-consuming. Here we describe the geometric viability assay (GVA) that replicates CFU measurements over 6 orders of magnitude while reducing over 10-fold the time and consumables required. GVA computes a sample's viable cell count on the basis of the distribution of embedded colonies growing inside a pipette tip. GVA is compatible with Gram-positive and Gram-negative planktonic bacteria (Escherichia coli, Pseudomonas aeruginosa and Bacillus subtilis), biofilms and fungi (Saccharomyces cerevisiae). Laborious CFU experiments such as checkerboard assays, treatment time-courses and drug screens against slow-growing cells are simplified by GVA. The ease and low cost of GVA evinces that it can replace existing viability assays and enable viability measurements at previously impractical scales.

[Source](https://www.nature.com/articles/s41564-023-01513-9)

10.1038/s41564-023-01513-9

---

## Comparison of gene clustering criteria reveals intrinsic uncertainty in pangenome analyses.
 07.11.2023, _S Manzano-Morales, Y Liu, S González-Bodí, J Huerta-Cepas, J Iranzo_


_Phylogeny, Reproducibility of Results, Uncertainty, Genome Size, Cluster Analysis_
A key step for comparative genomics is to group open reading frames into functionally and evolutionarily meaningful gene clusters. Gene clustering is complicated by intraspecific duplications and horizontal gene transfers that are frequent in prokaryotes. In consequence, gene clustering methods must deal with a trade-off between identifying vertically transmitted representatives of multicopy gene families, which are recognizable by synteny conservation, and retrieving complete sets of species-level orthologs. We studied the implications of adopting homology, orthology, or synteny conservation as formal criteria for gene clustering by performing comparative analyses of 125 prokaryotic pangenomes.

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03089-3)

10.1186/s13059-023-03089-3

---

## Comparison of gene clustering criteria reveals intrinsic uncertainty in pangenome analyses – Saioa Manzano-Morales

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03089-3)

---

## ACR: metagenome-assembled prokaryotic and eukaryotic genome refinement tool.
 02.11.2023, _HJ Seong, JJ Kim, WJ Sul_


_Metagenome, Eukaryota, Microbiota, Algorithms, Metagenomics, Cluster Analysis_
Microbial genome recovery from metagenomes can further explain microbial ecosystem structures, functions and dynamics. Thus, this study developed the Additional Clustering Refiner (ACR) to enhance high-purity prokaryotic and eukaryotic metagenome-assembled genome (MAGs) recovery. ACR refines low-quality MAGs by subjecting them to iterative k-means clustering predicated on contig abundance and increasing bin purity through validated universal marker genes. Synthetic and real-world metagenomic datasets, including short- and long-read sequences, evaluated ACR's effectiveness. The results demonstrated improved MAG purity and a significant increase in high- and medium-quality MAG recovery rates. In addition, ACR seamlessly integrates with various binning algorithms, augmenting their strengths without modifying core features. Furthermore, its multiple sequencing technology compatibilities expand its applicability. By efficiently recovering high-quality prokaryotic and eukaryotic genomes, ACR is a promising tool for deepening our understanding of microbial communities through genome-centric metagenomics.

[Source](https://academic.oup.com/bib/article-abstract/24/6/bbad381/7330501)

10.1093/bib/bbad381

---

## https://t.co/RVEkkmv13G

[Source](https://t.co/RVEkkmv13G)

---

## Protocol update and baseline characteristics for the TRIal to slow the Progression of Diabetes (TRIPOD) randomized controlled trial.
 15.11.2023, _J Bairavi, DS Gardner, E Yeoh, KW Tham, M Gandhi, NC Tan, PC Lee, RSM Lim, T Adamjee, YB Cheung, YM Bee, EA Finkelstein_


Type 2 diabetes (T2D), a major risk factor for cardiovascular disease and other adverse health conditions, is on the rise in Singapore. TRIPOD is a randomized controlled trial aimed to determine whether complementing usual care with an evidence-based diabetes management package (DMP) -comprising access to an evidence-based app, health coaching, pedometer, glucometer and weighing scale, with or without a financial rewards scheme (M-POWER rewards), can improve mean HbA1c levels at months 6 and 12.

[Source](https://twitter.com/HoonJeSeong1/status/1719231372362215759?ref_src=twsrc%5Etfw)

10.1186/s13063-023-07770-7

---

## October 31, 2023

[Source](https://twitter.com/HoonJeSeong1/status/1719231372362215759?ref_src=twsrc%5Etfw)

---

