## Spot-light on microbiota in obesity and cancer

[Source](https://www.nature.com/articles/s41366-021-00866-7)

---

## Microbiome analysis, the immune response and transplantation in the era of next generation sequencing.
 08.08.2021, _S Kanangat, I Skaljic_


The human gastrointestinal tract, skin and mucosal surfaces are inhabited by a complex system of bacteria, viruses, fungi, archaea, protists, and eukaryotic parasites with predominance of bacteria and bacterial viruses (bacteriophages). Collectively these microbes form the microbiota of the microecosystem of humans. Recent advancement in technologies for nucleic acid isolation from various environmental samples, feces and body secretions and advancements in shotgun throughput massive parallel DNA and RNA sequencing along with 16S ribosomal gene sequencing have unraveled the identity of otherwise unknown microbial entities constituting the human microecosystem. The improved transcriptome analysis, technological developments in biochemical analytical methods and availability of complex bioinformatics tools have allowed us to begin to understand the metabolome of the microbiome and the biochemical pathways and potential signal transduction pathways in human cells in response to microbial infections and their products. Also, developments in human whole genome sequencing, targeted gene sequencing of histocompatibility genes and other immune response associated genes by Next Generation Sequencing (NGS) have allowed us to have a better conceptualization of immune responses, and alloimmune responses. These modern technologies have enabled us to dive into the intricate relationship between commensal symbiotic and pathogenic microbiome and immune system. For the most part, the commensal symbiotic microbiota helps to maintain normal immune homeostasis besides providing healthy nutrients, facilitating digestion, and protecting the skin, mucosal and intestinal barriers. However, changes in diets, administration of therapeutic agents like antibiotics, chemotherapeutic agents, immunosuppressants etc. along with certain host factors including human histocompatibility antigens may alter the microbial ecosystem balance by causing changes in microbial constituents, hierarchy of microbial species and even dysbiosis. Such alterations may cause immune dysregulation, breach of barrier protection and lead to immunopathogenesis rather than immune homeostasis. The effects of human microbiome on immunity, health and disease are currently under intense research with cutting edge technologies in molecular biology, biochemistry, and bioinformatics along with tremendous ability to characterize immune response at single cell level. This review will discuss the contemporary status on human microbiome immune system interactions and their potential effects on health, immune homeostasis and allograft transplantation.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S019888592100183X)

10.1016/j.humimm.2021.07.009

---

## Type 1 diabetes in pregnancy is associated with distinct changes in the composition and function of the gut microbiome

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01104-y)

---

## Metataxonomic review to elucidate the role of the microbiome in Celiac disease across the gastrointestinal tract

[Source](https://www.biorxiv.org/content/10.1101/2021.08.08.455560v1.abstract?%3Fcollection=)

---

## Microbiota and sleep: awakening the gut feeling.
 08.08.2021, _P Sen, A Molinero-Perez, KJ O'Riordan, CP McCafferty, KD O'Halloran, JF Cryan_


Various lifestyle and environmental factors are known to influence sleep. Increasingly, evidence points to a role for the microbiota in regulating brain and behaviour. This article explores how the microbiota-gut-brain axis affects sleep directly and indirectly. We summarize the possible molecular mechanisms underlying sleep-microbiome interactions and discuss how various factors interact with the gut microbiota to influence sleep. Furthermore, we present the current evidence of alterations of the microbiota-gut-brain axis in various sleep disorders and pathologies where comorbid sleep disturbances are common. Since manipulating the gut microbiota could potentially improve sleep, we outline ways in which this can be achieved.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1471491421001854)

10.1016/j.molmed.2021.07.004

---

## Effects of Smoking on Inflammatory Markers in a Healthy Population as Analyzed via the Gut Microbiota

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2021.633242/full)

---

## The link between the gut microbiota and Parkinson's Disease: A systematic mechanism review with focus on α-synuclein transport.
 09.08.2021, _SD Nielsen, NM Pearson, K Seidler_


Research has suggested a link between the gut microbiota and Parkinson's Disease (PD), and an early involvement of gastrointestinal dysfunction has been reported in patients. A mechanism review was performed to investigate whether the neurodegenerative cascade begins in the gut; mediated by gut dysbiosis and retrograde transport of α-synuclein. This review provides a summary of microbiome composition associated with PD, and evaluates pathophysiological mechanisms from animal and in vitro models of PD.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0006899321004662)

10.1016/j.brainres.2021.147609

---

## Gut brain axis: an insight into microbiota role in Parkinson's disease.
 09.08.2021, _SA Moustafa, S Mohamed, A Dawood, J Azar, E Elmorsy, NAM Rizk, M Salama_


Parkinson's disease (PD) is one of the most common progressive neurodegenerative diseases. It is characterized neuropathologically by the presence of alpha-synuclein containing Lewy Bodies in the substantia nigra of the brain with loss of dopaminergic neurons in the pars compacta of the substantia nigra. The presence of alpha-synuclein aggregates in the substantia nigra and the enteric nervous system (ENS) drew attention to the possibility of a correlation between the gut microbiota and Parkinson's disease. The gut-brain axis is a two-way communication system, which explains how through the vagus nerve, the gut microbiota can affect the central nervous system (CNS), including brain functions related to the ENS, as well as how CNS can alter various gut secretions and immune responses. As a result, this dysbiosis or alteration in gut microbiota can be an early sign of PD with reported changes in short chain fatty acids, bile acids, and lipids. This gave rise to the use of probiotics and faecal microbiota transplantation as alternative approaches to improve the symptoms of patients with PD. The aim of this review is to discuss investigations that have been done to explore the gastrointestinal involvement in Parkinson's disease, the effect of dysbiosis, and potential therapeutic strategies for PD.

[Source](https://link.springer.com/article/10.1007%2Fs11011-021-00808-2)

10.1007/s11011-021-00808-2

---

## Describing the intestinal microbiota of Holstein Fasciola-positive and -negative cattle from a hyperendemic area of fascioliasis in central Colombia.
 09.08.2021, _AL Ramírez, G Herrera, M Muñoz, L Vega, L Cruz-Saavedra, D García-Corredor, M Pulido-Medellín, DM Bulla-Castañeda, JC Giraldo, MC Bernal, J Fernández-Manrique, LR Vásquez-Arteaga, JD Ramírez_


The ability to identify compositional changes in the intestinal microbiota of parasitized hosts is important for understanding the physiological processes that may affect animal productivity. Within the field of host-parasite interactions, many studies have suggested that helminths can influence the microbial composition of their hosts via their immunomodulatory effects. Bovine fascioliasis is a helminthiasis widely studied by immunologists, but with little information available regarding gut microbial communities. Thus, we aimed to describe the composition of the intestinal microbiota of Holstein Fasciola-positive and -negative cattle using parasitological methods and ELISA (enzyme-linked immunosorbent assay). Bovine fecal samples (n = 65) were obtained from livestock slaughter plants in the Cundi-Boyacense Colombian highlands (a hyperendemic region for bovine fascioliasis) and studied by amplicon-based next-generation 16S-rRNA and 18S-rRNA gene sequencing. From these samples, 35 were Fasciola hepatica-negative and, 30 were F. hepatica-positive in our detection analysis. Our results showed a reduction in the relative abundance of Bacteroidetes and Ascomycota in the Fasciola-positive samples, along with decreased relative abundances of the commensal taxa previously associated with fermentation and digestion processes. However, metabolomic approaches and functional analyzes of the intestinal microbiota are necessary to support this hypothesis. These findings are a small first step in the development of research aimed at understanding how microbial populations in bovines are modulated in liver helminth infections.

[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0009658)

10.1371/journal.pntd.0009658

---

## The gut microbiome of healthy Vietnamese adults and children is a major reservoir for resistance genes against critical antimicrobials.
 10.08.2021, _J Pereira-Dias, C Nguyen Ngoc Minh, C Tran Thi Hong, T Nguyen Thi Nguyen, T Ha Thanh, C Zellmer, H Chung The, L Pike, EE Higginson, S Baker_


Antimicrobials are a key group of therapeutic agents. Given the animal/human population density and high antimicrobial consumption rate in Southeast Asia, the region is a focal area for monitoring antimicrobial resistance (AMR). Hypothesizing that the gastrointestinal tract of healthy individuals in Vietnam is a major source of AMR genes that may be transferred to pathogens, we performed shotgun metagenomic sequencing on fecal samples from 42 healthy Vietnamese people (21 children and 21 adults). We compared their microbiome profiles by age group and determined the composition of AMR genes. An analysis of the taxonomic profiles in the gut microbiome showed a clear differentiation by age, with young children (age &lt;2 years) exhibiting a unique structure in comparison to adults and older children. We identified a total of 132 unique AMR genes, with MLS class resistance genes (ermB and lnuC) and tetracycline resistance genes being the almost ubiquitous across the study population. Notably, samples from younger children were significantly associated with a greater number of AMR genes than other age groups, including key signature genes associated with AMR pathogens (e.g., blaCTX-M, mphA). Our data suggest that the gut microbiome of those living in Vietnam, particularly young children, is a substantial reservoir of AMR genes, which can be transferred to circulating enteric pathogens. Our data supports the generation of longitudinal cohort studies of those living in urban and rural areas of developing countries to understand the behavior of these AMR reservoirs and their role in generating MDR and XDR pathogens.

[Source](https://academic.oup.com/jid/advance-article/doi/10.1093/infdis/jiab398/6347305)

10.1093/infdis/jiab398

---

## Characteristics of gut microbiota in people with obesity.
 10.08.2021, _M Duan, Y Wang, Q Zhang, R Zou, M Guo, H Zheng_


Obesity is the cause of cardiovascular diseases and other diseases, leading to increased medical costs, and causing a great burden to individuals, families and society. The prevalence of obesity is increasing and has become a global health problem. There is growing evidence that gut microbiota plays an important role in obesity. In this article, we revealed the differences in the gut microbiota between 21 people with obesity and 21 control subjects, and predicted the functional potential changes by 16S rRNA sequencing of the fecal bacteria of the subjects.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0255446)

10.1371/journal.pone.0255446

---

## Post-weaning shifts in microbiome composition and metabolism revealed by over 25 000 pig gut metagenome-assembled genomes.
 09.08.2021, _D Gaio, MZ DeMaere, K Anantanawat, TA Chapman, SP Djordjevic, AE Darling_


Using a previously described metagenomics dataset of 27 billion reads, we reconstructed over 50 000 metagenome-assembled genomes (MAGs) of organisms resident in the porcine gut, 46.5 % of which were classified as &gt;70 % complete with a &lt;10 % contamination rate, and 24.4 % were nearly complete genomes. Here, we describe the generation and analysis of those MAGs using time-series samples. The gut microbial communities of piglets appear to follow a highly structured developmental programme in the weeks following weaning, and this development is robust to treatments including an intramuscular antibiotic treatment and two probiotic treatments. The high resolution we obtained allowed us to identify specific taxonomic 'signatures' that characterize the gut microbial development immediately after weaning. Additionally, we characterized the carbohydrate repertoire of the organisms resident in the porcine gut. We tracked the abundance shifts of 294 carbohydrate active enzymes, and identified the species and higher-level taxonomic groups carrying each of these enzymes in their MAGs. This knowledge can contribute to the design of probiotics and prebiotic interventions as a means to modify the piglet gut microbiome.

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000501)

10.1099/mgen.0.000501

---

## Effects of dietary rosemary extract supplementation on growth performance, nutrient digestibility, antioxidant capacity, intestinal morphology and microbiota of weaning pigs.
 09.08.2021, _M Yang, Y Yin, F Wang, X Bao, L Long, B Tan, Y Yin, J Chen_


Rosemary (Rosmarinus officinalis L.) extract (RE) has multiple pharmacological and biological activities, including use as a food additive and medicine. This study was conducted to investigate the effects of dietary RE supplementation on the growth performance, nutrient digestibility, antioxidant capacity, intestinal morphology and microbiota of weaning piglets. A total of 192 crossbred weaned piglets [Duroc × (Large White × Landrace)] (initial body weight = 6.65 ± 0.33 kg, weaned days = 23 ± 1 d) were group housed (six pigs per pen; n = 8 pens/treatment). Pigs were fed a corn-soybean meal-based control diet or the basal diet supplemented with 100, 200 or 400 mg/kg RE. Pigs were allowed ad libitum access to fed for 21-d. The growth performance and apparent total tract digestibility of nutrients, and intestinal morphology and antioxidant status were evaluated. The components of the microbial microflora were also determined in the cecal samples. Compared with the control, dietary supplementation with RE increased the final body weight, average daily gain and average daily feed intake (linear, P = 0.038, 0.016, and 0.009, respectively), and decreased the diarrhea ratio in piglets (linear, P &lt; 0.05). The digestibility of crude protein (linear, P = 0.034) and gross energy (linear, P = 0.046) increased with treatment with RE. Piglets fed RE showed longer villus height (linear, P = 0.037 and 0.028, respectively) and villus height/crypt depth (linear, P = 0.004 and 0.012; quadratic, P = 0.023 and 0.036, respectively) in the jejunum and ileum, in addition to a lesser crypt depth in the jejunum (linear, P = 0.019) and ileum (quadratic, P = 0.042). The addition of RE increased the activity of superoxide dismutase (linear, P = 0.035 and 0.008 respectively) and glutathione peroxidase activity (linear, P = 0.027 and 0.039 respectively), and decreased the content of malondialdehyde (linear, P = 0.041 and 0.013; quadratic, P = 0.023 and 0.005 respectively) in the serum and liver. Dietary RE supplementation, compared with the control, increased the number of Bifidobacterium (linear, P = 0.034) and Bacteroidetes (linear, P = 0.029), while decreased Escherichia coli (linear, P = 0.008; quadratic, P = 0.014) in the cecal contents. Thus, dietary RE supplementation can improve growth performance, nutrient digestibility, antioxidant capacity, intestinal morphology and the microbiota in weaned piglets, and 200 mg/kg may be considered the optimum dosage.

[Source](https://academic.oup.com/jas/advance-article-abstract/doi/10.1093/jas/skab237/6346706?)

10.1093/jas/skab237

---

## Changes in the gut microbiota: a possible factor influencing peripheral blood immune indexes in non-obese diabetic mice.
 09.08.2021, _Y Wu, Q You, J Fei, J Wu_


Intestinal flora changes were found in patients and animals with type 1 diabetes (T1D). However, few studies have provided any explicit clues of changes in highly disease related commensal microbiota before disease onset and their relationships with disordered peripheral immune cells. We conducted 16S rRNA microbiota analysis of non-obese diabetic (NOD) mice from weaning to diabetes onset to identify highly disease related microbes and performed Spearman correlation analysis between anomalous flora and peripheral immune cells. We found NOD mice had increased exclusive bacteria and decreased community richness or diversity, besides, with the features of decreased abundance of Bacteroidetes and increased abundance of Firmicutes, Proteobacteria or Deferribacteres and remarkable fluctuations of genus relative abundance. Furthermore, kinds of highly T1D related genus and their strong correlations with peripheral immune cells, especially neutrophils, were discovered. Microbial changes in NOD mice differed from that of ICR mice and highly disease associated microbes have strong correlations with the peripheral neutrophil ratio, which provide evidence that neutrophils are possibly involved in the pathogenesis of T1D.

[Source](https://link.springer.com/article/10.1007%2Fs10482-021-01632-5)

10.1007/s10482-021-01632-5

---

## The gut bacteria of an invasive amphibian respond to the dual challenges of range-expansion and parasite attack

[Source](https://www.biorxiv.org/content/10.1101/2020.11.16.385690v3.abstract?%3Fcollection=)

---

## Fishing for the Virome of Tropical Tuna.
 10.08.2021, _E Gadoin, C Desnues, S Monteil-Bouchard, T Bouvier, JC Auguet, E Roque d'Orbcastel, Y Bettarel_


While planktonic viruses have received much attention in recent decades, knowledge of the virome of marine organisms, especially fish, still remains rudimentary. This is notably the case with tuna, which are among the most consumed fish worldwide and represent considerable economic, social and nutritional value. Yet the composition of the tuna virome and its biological and environmental determinants remain unknown. To begin to address this gap, we investigated the taxonomic diversity of viral communities inhabiting the skin mucus, gut and liver of two major tropical tuna species (skipjack and yellowfin) in individuals fished in the Atlantic and Indian Oceans. While we found significant differences in the virome composition between the organs, this was totally independent of the tuna species or sex. The tuna virome was mainly dominated by eukaryotic viruses in the digestive organs (gut and liver), while bacteriophages were predominant in the mucus. We observed the presence of specific viral families in each organ, some previously identified as fish or human pathogens (e.g., Iridoviridae, Parvoviridae, Alloherpesviridae, Papillomaviridae). Interestingly, we also detected a 'core virome' that was shared by all the organs and was mainly composed of Caudovirales, Microviridae and Circoviridae. These results show that tuna host a mosaic of viral niches, whose establishment, role and circulation remain to be elucidated.

[Source](https://www.mdpi.com/1999-4915/13/7/1291)

10.3390/v13071291

---

## Colonization by the Red Imported Fire Ant, Solenopsis invicta, Modifies Soil Bacterial Communities.
 09.08.2021, _NV Travanty, EL Vargo, CS Apperson, L Ponnusamy_


The long-standing association between insects and microorganisms has been especially crucial to the evolutionary and ecological success of social insect groups. Notably, research on the interaction of the two social forms (monogyne and polygyne) of the red imported fire ant (RIFA), Solenopsis invicta Buren, with microbes in its soil habitat is presently limited. In this study, we characterized bacterial microbiomes associated with RIFA nest soils and native (RIFA-negative) soils to better understand the effects of colonization of RIFA on soil microbial communities. Bacterial community fingerprints of 16S rRNA amplicons using denaturing gradient gel electrophoresis revealed significant differences in the structure of the bacterial communities between RIFA-positive and RIFA-negative soils at 0 and 10 cm depths. Illumina sequencing of 16S rRNA amplicons provided fine-scale analysis to test for effects of RIFA colonization, RIFA social form, and soil depth on the composition of the bacterial microbiomes of the soil and RIFA workers. Our results showed the bacterial community structure of RIFA-colonized soils to be significantly different from native soil communities and to evidence elevated abundances of several taxa, including Actinobacteria. Colony social form was not found to be a significant factor in nest or RIFA worker microbiome compositions. RIFA workers and nest soils were determined to have markedly different bacterial communities, with RIFA worker microbiomes being characterized by high abundances of a Bartonella-like endosymbiont and Entomoplasmataceae. Cloning and sequencing of the 16S rRNA gene revealed the Bartonella sp. to be a novel bacterium.

[Source](https://link.springer.com/article/10.1007%2Fs00248-021-01826-4)

10.1007/s00248-021-01826-4

---

## Shotgun proteomic profiling of dormant, ‘non-culturable’ Mycobacterium tuberculosis

[Source](https://www.biorxiv.org/content/10.1101/2021.08.06.455493v1.abstract?%3Fcollection=)

---

## A clinically validated human saliva metatranscriptomic test for global systems biology studies

[Source](https://www.biorxiv.org/content/10.1101/2021.08.03.454950v2.abstract?%3Fcollection=)

---

## Sample Preparation for Detection of Different Bacterial Strains by Matrix-Assisted Laser Desorption/Ionization Time-of-Flight (MALDI-TOF) Mass Spectrometry.
 11.08.2021, _D Chac, M Kordahi, L Brettner, RW DePaolo_


_Bacteria, Humans, Lasers, Specimen Handling, Spectrometry, Mass, Matrix-Assisted Laser Desorption-Ionization_
Matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-TOF MS) provides a fast and easy means to identify culturable microorganisms to the species level. The sample preparation of microbial colonies for MALDI-TOF analysis requires a suitable protein extraction method. While standard MALDI-TOF sample preparation methods are well suited for the identification of and the discrimination between microorganisms belonging to different species, they are not disruptive enough to allow the discrimination between different strains of the same microorganism. More disruptive protein extraction methods lead to better discrimination power because they allow a better breakdown of bacterial cell membrane and a more efficient extraction of conserved microbial proteins that are specific to each species and strain. Here we describe how to extract proteins from single microbial colonies using formic acid and acetonitrile to disrupt cells prior to placing them on a target plate for MALDI-TOF MS analysis. Contrary to other sample preparation methods for MALDI-TOF MS, this approach allows the discrimination between different strains of microorganisms of the same species. Our approach also provides the groundwork data for building algorithms that allow the detection of specific microbial strains of interest, with a great potential for diagnostic applications in clinical settings. © 2021 Wiley Periodicals LLC. Basic Protocol: Protein extraction and MALDI-TOF bio-typing of phenotypically distinct bacterial species.

[Source](https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/cpz1.212)

10.1002/cpz1.212

---

