## The human microbiome and COVID-19: A systematic review.
 02.07.2021, _S Yamamoto, M Saito, A Tamura, D Prawisuda, T Mizutani, H Yotsuyanagi_


_Bronchoalveolar Lavage Fluid, COVID-19, Dysbiosis, Feces, Gastrointestinal Microbiome, Humans, Microbiota, Nasopharynx_
Human microbiotas are communities of microorganisms living in symbiosis with humans. They play an important role in the host immune response to respiratory viral infection. However, evidence on the human microbiome and coronavirus disease (COVID-19) relationship is insufficient. The aim of this systematic literature review was to evaluate existing evidence on the association between the microbiome and COVID-19 in humans and summarize these data in the pandemic era.

[Source](https://doi.org/10.1371/journal.pone.0253293)

10.1371/journal.pone.0253293

---

## The gut microbiome and type 2 diabetes status in the Multiethnic Cohort.
 23.06.2021, _G Maskarinec, P Raquinio, BS Kristal, VW Setiawan, LR Wilkens, AA Franke, U Lim, L Le Marchand, TW Randolph, JW Lampe, MAJ Hullar_


The gut microbiome may play a role in inflammation associated with type 2 diabetes (T2D) development. This cross-sectional study examined its relation with glycemic status within a subset of the Multiethnic Cohort (MEC) and estimated the association of circulating bacterial endotoxin (measured as plasma lipopolysaccharide-binding protein (LBP)) with T2D, which may be mediated by C-reactive protein (CRP).

[Source](https://doi.org/10.1371/journal.pone.0250855)

10.1371/journal.pone.0250855

---

## Niche partitioning facilitates coexistence of closely related gut bacteria

[Source](https://doi.org/10.1101/2021.03.12.434400)

---

## Successional Stages in Infant Gut Microbiota Maturation

[Source](https://doi.org/10.1101/2021.06.25.450009)

---

## Fetal meconium does not have a detectable microbiota before birth

[Source](https://doi.org/10.1038/s41564-021-00904-0)

---

## Developmental intestinal microbiome alterations in canine fading puppy syndrome: a prospective observational study

[Source](https://doi.org/10.1038/s41522-021-00222-7)

---

## Gut microbiome alterations in high-fat-diet-fed mice are associated with antibiotic tolerance

[Source](https://doi.org/10.1038/s41564-021-00912-0)

---

## Enteric viruses nucleic acids distribution along the digestive tract of rhesus macaques with idiopathic chronic diarrhea

[Source](https://doi.org/10.1101/2021.06.24.449827)

---

## Microbiomes of commercially-available pine nuts and sesame seeds

[Source](https://doi.org/10.1371/journal.pone.0252605)

---

## Spatial patterns in phage-Rhizobium coevolutionary interactions across regions of common bean domestication

[Source](https://doi.org/10.1038/s41396-021-00907-z)

---

## <span style="color:#ff6600;">Genomic insights into diverse bacterial taxa that degrade extracellular DNA in marine sediments</span>

[Source](https://doi.org/10.1038/s41564-021-00917-9)

---

## High viral abundance and low diversity are associated with increased CRISPR-Cas prevalence across microbial ecosystems

[Source](https://doi.org/10.1101/2021.06.24.449667)

---

## Metagenomic compendium of 189,680 DNA viruses from the human gut microbiome.
 30.06.2021, _S Nayfach, D Páez-Espino, L Call, SJ Low, H Sberro, NN Ivanova, AD Proal, MA Fischbach, AS Bhatt, P Hugenholtz, NC Kyrpides_


Bacteriophages have important roles in the ecology of the human gut microbiome but are under-represented in reference databases. To address this problem, we assembled the Metagenomic Gut Virus catalogue that comprises 189,680 viral genomes from 11,810 publicly available human stool metagenomes. Over 75% of genomes represent double-stranded DNA phages that infect members of the Bacteroidia and Clostridia classes. Based on sequence clustering we identified 54,118 candidate viral species, 92% of which were not found in existing databases. The Metagenomic Gut Virus catalogue improves detection of viruses in stool metagenomes and accounts for nearly 40% of CRISPR spacers found in human gut Bacteria and Archaea. We also produced a catalogue of 459,375 viral protein clusters to explore the functional potential of the gut virome. This revealed tens of thousands of diversity-generating retroelements, which use error-prone reverse transcription to mutate target genes and may be involved in the molecular arms race between phages and their bacterial hosts.

[Source](https://doi.org/10.1038/s41564-021-00928-6)

10.1038/s41564-021-00928-6

---

## SCAPP: an algorithm for improved plasmid assembly in metagenomes.
 01.07.2021, _D Pellow, A Zorea, M Probst, O Furman, A Segal, I Mizrahi, R Shamir_


_Algorithms, Humans, Metagenome, Metagenomics, Plasmids, Sequence Analysis, DNA, Software_
Metagenomic sequencing has led to the identification and assembly of many new bacterial genome sequences. These bacteria often contain plasmids: usually small, circular double-stranded DNA molecules that may transfer across bacterial species and confer antibiotic resistance. These plasmids are generally less studied and understood than their bacterial hosts. Part of the reason for this is insufficient computational tools enabling the analysis of plasmids in metagenomic samples.

[Source](https://doi.org/10.1186/s40168-021-01068-z)

10.1186/s40168-021-01068-z

---

## A standardized archaeal taxonomy for the Genome Taxonomy Database.
 30.06.2021, _C Rinke, M Chuvochina, AJ Mussig, PA Chaumeil, AA Davín, DW Waite, WB Whitman, DH Parks, P Hugenholtz_


The accrual of genomic data from both cultured and uncultured microorganisms provides new opportunities to develop systematic taxonomies based on evolutionary relationships. Previously, we established a bacterial taxonomy through the Genome Taxonomy Database. Here, we propose a standardized archaeal taxonomy that is derived from a 122-concatenated-protein phylogeny that resolves polyphyletic groups and normalizes ranks based on relative evolutionary divergence. The resulting archaeal taxonomy, which forms part of the Genome Taxonomy Database, is stable for a range of phylogenetic variables including marker gene selection, inference methods, corrections for rate heterogeneity and compositional bias, tree rooting scenarios and expansion of the genome database. Rank normalization is shown to robustly correct for substitution rates varying up to 30-fold using simulated datasets. Taxonomic curation follows the rules of the International Code of Nomenclature of Prokaryotes while taking into account proposals to formally recognize the rank of phylum and to use genome sequences as type material. This taxonomy is based on 2,392 archaeal genomes, 93.3% of which required one or more changes to their existing taxonomy, mainly owing to incomplete classification. We identify 16 archaeal phyla and reclassify 3 major monophyletic units from the former Euryarchaeota and one phylum that unites the Thaumarchaeota-Aigarchaeota-Crenarchaeota-Korarchaeota (TACK) superphylum into a single phylum.

[Source](https://doi.org/10.1038/s41564-021-00918-8)

10.1038/s41564-021-00918-8

---

## Is the open access citation advantage real? A systematic review of the citation of open access and subscription-based articles

[Source](https://doi.org/10.1371/journal.pone.0253129)

---

