## Functional profiling of COVID-19 respiratory tract microbiomes.
 26.03.2021, _N Haiminen, F Utro, E Seabolt, L Parida_


In response to the ongoing global pandemic, characterizing the molecular-level host interactions of the new coronavirus SARS-CoV-2 responsible for COVID-19 has been at the center of unprecedented scientific focus. However, when the virus enters the body it also interacts with the micro-organisms already inhabiting the host. Understanding the virus-host-microbiome interactions can yield additional insights into the biological processes perturbed by viral invasion. Alterations in the gut microbiome species and metabolites have been noted during respiratory viral infections, possibly impacting the lungs via gut-lung microbiome crosstalk. To better characterize microbial functions in the lower respiratory tract during COVID-19 infection, we carry out a functional analysis of previously published metatranscriptome sequencing data of bronchoalveolar lavage fluid from eight COVID-19 cases, twenty-five community-acquired pneumonia patients, and twenty healthy controls. The functional profiles resulting from comparing the sequences against annotated microbial protein domains clearly separate the cohorts. By examining the associated metabolic pathways, distinguishing functional signatures in COVID-19 respiratory tract microbiomes are identified, including decreased potential for lipid metabolism and glycan biosynthesis and metabolism pathways, and increased potential for carbohydrate metabolism pathways. The results include overlap between previous studies on COVID-19 microbiomes, including decrease in the glycosaminoglycan degradation pathway and increase in carbohydrate metabolism. The results also suggest novel connections to consider, possibly specific to the lower respiratory tract microbiome, calling for further research on microbial functions and host-microbiome interactions during SARS-CoV-2 infection.

[Source](https://doi.org/10.1038/s41598-021-85750-0)

10.1038/s41598-021-85750-0

---

## Mild CF Lung Disease is Associated with Bacterial Community Stability

[Source](https://doi.org/10.1101/2021.03.23.436717)

---

## On-person adaptive evolution of <em>Staphylococcus aureus</em> during atopic dermatitis increases disease severity

[Source](https://doi.org/10.1101/2021.03.24.436824)

---

## The Unique Impact of COVID-19 on Human Gut Microbiome Research

[Source](https://doi.org/10.3389/fmed.2021.652464)

---

## Variability in digestive and respiratory tract <em>Ace2</em> expression is associated with the microbiome

[Source](https://doi.org/10.1371/journal.pone.0248730)

---

## Towards a mechanistic understanding of reciprocal drug–microbiome interactions

[Source](https://doi.org/10.15252/msb.202010116)

---

## The Microbiome Meets Nanotechnology: Opportunities and Challenges in Developing New Diagnostic Devices

[Source](https://doi.org/10.1002/adma.202006104)

---

## The capacity to produce hydrogen sulfide (H2S) via cysteine degradation is ubiquitous in the human gut microbiome

[Source](https://doi.org/10.1101/2021.03.24.436781)

---

## Species richness determines <em>C. difficile</em> invasion outcome in synthetic human gut communities

[Source](https://doi.org/10.1101/2021.03.23.436677)

---

## Gut microbiota determines the social behavior of mice and induces metabolic and inflammatory changes in their adipose tissue

[Source](https://doi.org/10.1038/s41522-021-00193-9)

---

## Improvement of loperamide-induced slow transit constipation by <em>Bifidobacterium bifidum</em> G9-1 is mediated by the correction of butyrate production and neurotransmitter profile due to improvement in dysbiosis

[Source](https://doi.org/10.1371/journal.pone.0248584)

---

## A Preliminary Comparison on Faecal Microbiomes of Free-Ranging Large Baleen (<em>Balaenoptera musculus</em>, <em>B. physalus</em>, <em>B. borealis</em>) and Toothed (<em>Physeter macrocephalus</em>) Whales

[Source](https://doi.org/10.1007/s00248-021-01729-4)

---

## Host–microbial systems as glass cannons: Explaining microbiome stability in corals exposed to extrinsic perturbations

[Source](https://doi.org/10.1111/1365-2656.13466)

---

## Fish predation on corals promotes the dispersal of coral symbionts

[Source](https://doi.org/10.1186/s42523-021-00086-4)

---

## Global analysis of the apple fruit microbiome: Are all apples the same?

[Source](https://doi.org/10.1111/1462-2920.15469)

---

## Evaluating domestication and ploidy effects on the assembly of the wheat bacterial microbiome.
 18.03.2021, _HML Wipf, D Coleman-Derr_


While numerous studies implicate the microbiome in host fitness, contributions of host evolution to microbial recruitment remain largely uncharacterized. Past work has shown that plant polyploidy and domestication can influence plant biotic and abiotic interactions, yet impacts on broader microbiome assembly are still unknown for many crop species. In this study, we utilized three approaches-two field studies and one greenhouse-based experiment-to determine the degree to which patterns in bacterial community assembly in wheat (Triticum sp.) roots and rhizospheres are attributable to the host factors of ploidy level (2n, 4n, 6n) and domestication status (cultivated vs. wild). Profiling belowground bacterial communities with 16S rRNA gene amplicon sequencing, we analyzed patterns in diversity and composition. From our initial analyses of a subsetted dataset, we observed that host ploidy level was statistically significant in explaining variation in alpha and beta diversity for rhizosphere microbiomes, as well as correlated with distinct phylum-level shifts in composition, in the field. Using a reduced complexity field soil inoculum and controlled greenhouse conditions, we found some evidence suggesting that genomic lineage and ploidy level influence root alpha and beta diversity (p-value&lt;0.05). However, in a follow-up field experiment using an expanded set of Triticum genomes that included both wild and domesticated varieties, we did not find a strong signal for either diploid genome lineages, domestication status, or ploidy level in shaping rhizosphere bacterial communities. Taken together, these results suggest that while host ploidy and domestication may have some minor influence on microbial assembly, these impacts are subtle and difficult to assess in belowground compartments for wheat varieties. By improving our understanding of the degree to which host ploidy and cultivation factors shape the plant microbiome, this research informs perspectives on what key driving forces may underlie microbiome structuring, as well as where future efforts may be best directed towards fortifying plant growth by microbial means. The greatest influence of the host on the wheat microbiome appeared to occur in the rhizosphere compartment, and we suggest that future work focuses on this environment to further characterize how host genomic and phenotypic changes influence plant-microbe communications.

[Source](https://doi.org/10.1371/journal.pone.0248030)

10.1371/journal.pone.0248030

---

## Evaluating domestication and ploidy effects on the assembly of the wheat bacterial microbiome, Wipf and Coleman-Derr

[Source](https://doi.org/10.1371/journal.pone.0248030)

---

## Uncharted biosynthetic potential of the ocean microbiome

[Source](https://doi.org/10.1101/2021.03.24.436479)

---

## The microbiome of the Black Sea water column analyzed by shotgun and genome centric metagenomics

[Source](https://doi.org/10.1186/s40793-021-00374-1)

---

## Revealing taxon-specific heavy metal-resistance mechanisms in denitrifying phosphorus removal sludge using genome-centric metaproteomics

[Source](https://doi.org/10.1186/s40168-021-01016-x)

---

## Forensic Microbiome Database: A Tool for Forensic Geolocation Meta-Analysis Using Publicly Available 16S rRNA Microbiome Sequencing

[Source](https://doi.org/10.3389/fmicb.2021.644861)

---

## ProgPerm: Progressive permutation for a dynamic representation of the robustness of microbiome discoveries

[Source](https://doi.org/10.1186/s12859-021-04061-3)

---

## Model-Based Microbiome Data Ordination: A Variational Approximation Approach

[Source](https://doi.org/10.1080/10618600.2021.1882467)

---

## Blue poo: impact of gut transit time on the gut microbiome using a novel marker

[Source](https://doi.org/10.1136/gutjnl-2020-323877)

---

## Benchmarking DNA isolation kits used in analyses of the urinary microbiome

[Source](https://doi.org/10.1038/s41598-021-85482-1)

---

## Choice of library preparation affects sequence quality, genome assembly, and precise <em>in silico</em> prediction of virulence genes in shiga toxin-producing <em>Escherichia coli</em>

[Source](https://doi.org/10.1371/journal.pone.0242294)

---

