## Different Characteristics in Gut Microbiome between Advanced Adenoma Patients and Colorectal Cancer Patients by Metagenomic Analysis.
 01.12.2022, _S Han, J Zhuang, Y Pan, W Wu, K Ding_


The occurrence and development of colorectal cancer (CRC) and advanced adenoma (AA) are closely related to the gut microbiome, and AA has a high cancerization progression rate to CRC. Current studies have revealed that bacteriological analysis cannot identify CRC from AA. The objective was to explore microbial targets that could identify CRC and AA from a microecological perspective and to figure out the best way to identify CRC based on fecal microbes. The metagenomic sequencing data were used to describe the gut microbiome profile and analyze the differences between microbial abundance and microbial single nucleotide polymorphism (SNP) characteristics in AA and CRC patients. It was found that there were no significant differences in the diversity between the two groups. The abundance of bacteria (e.g., Firmicutes, Clostridia, and Blautia), fungi (Hypocreales), archaea (Methanosarcina, Methanoculleus, and Methanolacinia), and viruses (Alphacoronavirus, Sinsheimervirus, and Gammaretrovirus) differed between AA and CRC patients. Multiple machine-learning algorithms were used to establish prediction models, aiming to identify CRC and AA. The accuracy of the random forest (RF) model based on the gut microbiome was 86.54%. Nevertheless, the accuracy of SNP was 92.31% in identifying CRC from AA. In conclusion, using microbial SNP was the best method to identify CRC, it was superior to using the gut microbiome, and it could provide new targets for CRC screening. IMPORTANCE There are differences in characteristic microorganisms between AA and CRC. However, current studies have indicated that bacteriological analysis cannot identify CC from AA, and thus, we wondered if there were some other targets that could be used to identify CRC from AA in the gut microbiome. The differences of SNPs in the gut microbiota of intraindividuals were significantly smaller than those of interindividuals. In addition, compared with intestinal microbes, SNP was less affected by time with certain stability. It was discovered that microbial SNP was better than the gut microbiome for identifying CRC from AA. Therefore, screening characteristic microbial SNP could provide a new research direction for identifying CRC from AA.

[Source](https://doi.org/10.1128/spectrum.01593-22)

10.1128/spectrum.01593-22

---

## The Spatial Features and Temporal Changes in the Gut Microbiota of a Healthy Chinese Population.
 01.12.2022, _W Zhang, N Han, T Zhang, Y Qiang, X Peng, X Li, B Kan_


In this study, we aimed to understand the characteristics of the gut microbial composition in a healthy Chinese population and to evaluate if they differed across different regions. In addition, we aimed to understand the changes in the gut microbial composition over time. We collected 239 fecal samples from healthy Chinese adults living in four regions and performed a 1-year time cohort study in a small population in Beijing. The Chinese gut microbiota share 34 core bacterial genera and 39 core bacterial species, which exist in all collected samples. Several disease-related microorganisms (DRMs), virulence factors, and antibiotic resistance genes were found in one or more healthy Chinese samples. Differences in gut microbiota were observed in samples from different regions, locations, individuals, and time points. Compared to other factors, time was associated with a lower degree of change in the gut microbiota. Our findings revealed spatial and temporal changes in the gut microbiota of healthy Chinese individuals. Compared to fecal microbiomes of 152 samples in the publicly released the Human Microbiome Project (HMP) project from the United States, samples in this study have higher variability in the fecal microbiome, with higher richness, Shannon diversity indices, and Pielou evenness indexes, at both the genus and species levels. The microbiota data obtained in this study will provide a detailed basis for further understanding the composition of the gut microbiota in the healthy Chinese population. IMPORTANCE China accounts for approximately 1/5th of the world's total population. Differences in environment, ethnicity, and living habits could impart unique features to the structure of the gut microbiota of Chinese individuals. In 2016, we started to investigate healthy Chinese people and their gut microbiomes. Phase I results for 16S rRNA amplicons have been released. However, owing to the limitations of 16S rRNA amplicon sequencing, the gut microbiome of a healthy Chinese population could not be examined thoroughly at the species level, and the detailed changes in the gut microbiota over time need to be investigated. To address these knowledge gaps, we started a phase II study and investigated the basis for variations in the gut microbiome composition in a healthy Chinese population at the species level using shotgun metagenomics technology. In the phase II study, we also conducted a time scale analysis of fecal samples from healthy Chinese subjects, as a pioneered study, which quantitatively clarified the changes in the gut microbiota at both the spatial and temporal levels and elucidated the distribution pattern of DRMs in healthy Chinese individuals.

[Source](https://doi.org/10.1128/spectrum.01310-22)

10.1128/spectrum.01310-22

---

## The Gut Microbiota is Important for the Maintenance of Blood-Cerebrospinal Fluid Barrier Integrity.
 01.12.2022, _EG Knox, CMK Lynch, YS Lee, CM O'Driscoll, G Clarke, JF Cryan, MR Aburto_


The gut microbiota communicates with the brain though several pathways including the vagus nerve, immune system, microbial metabolites, and through the endocrine system. Pathways along the humoral/immune gut-microbiota-brain axis comprise of a series of vascular and epithelial barriers including the intestinal epithelial barrier, gut-vascular barrier, blood brain barrier, and blood-cerebrospinal fluid barrier. Of these barriers, the relationship between the gut microbiota and blood-cerebrospinal fluid barrier is yet to be fully defined. Here, using a germ-free mouse model, we aimed to assess the relationship between the gut microbiota and the integrity of the blood-cerebrospinal fluid barrier which is localized to the choroid plexus epithelium. Using confocal microscopy, we visualized the tight junction protein zonula occludens-1, an integral aspect of choroid plexus integrity, as well as the choroid plexus fenestrated capillaries. Quantification of tight junction proteins via network analysis led to the observation that there was a decrease in the zonula occludens-1 network organization in germ-free mice, however, we did not observe any differences in capillary structure. Taken together, these data indicate that the blood- cerebrospinal fluid barrier is another barrier along the gut-microbiota-brain axis. Future studies are required to elucidate its relative contribution in signaling from microbiota to the brain.

[Source](https://doi.org/10.1111/ejn.15878)

10.1111/ejn.15878

---

## Evaluation of co-circulating pathogens and microbiome from COVID-19 infections.
 05.12.2022, _JB Thissen, MD Morrison, N Mulakken, WC Nelson, C Daum, S Messenger, DA Wadford, C Jaing_


_Humans, COVID-19, SARS-CoV-2, RNA, Ribosomal, 16S, Coinfection, COVID-19 Testing, Microbiota_
Co-infections or secondary infections with SARS-CoV-2 have the potential to affect disease severity and morbidity. Additionally, the potential influence of the nasal microbiome on COVID-19 illness is not well understood. In this study, we analyzed 203 residual samples, originally submitted for SARS-CoV-2 testing, for the presence of viral, bacterial, and fungal pathogens and non-pathogens using a comprehensive microarray technology, the Lawrence Livermore Microbial Detection Array (LLMDA). Eighty-seven percent of the samples were nasopharyngeal samples, and 23% of the samples were oral, nasal and oral pharyngeal swabs. We conducted bioinformatics analyses to examine differences in microbial populations of these samples, as a proxy for the nasal and oral microbiome, from SARS-CoV-2 positive and negative specimens. We found 91% concordance with the LLMDA relative to a diagnostic RT-qPCR assay for detection of SARS-CoV-2. Sixteen percent of all the samples (32/203) revealed the presence of an opportunistic bacterial or frank viral pathogen with the potential to cause co-infections. The two most detected bacteria, Streptococcus pyogenes and Streptococcus pneumoniae, were present in both SARS-CoV-2 positive and negative samples. Human metapneumovirus was the most prevalent viral pathogen in the SARS-CoV-2 negative samples. Sequence analysis of 16S rRNA was also conducted to evaluate bacterial diversity and confirm LLMDA results.

[Source](https://doi.org/10.1371/journal.pone.0278543)

10.1371/journal.pone.0278543

---

## A Clinical Study Provides the First Direct Evidence That Interindividual Variations in Fecal β-Lactamase Activity Affect the Gut Mycobiota Dynamics in Response to β-Lactam Antibiotics

[Source](https://doi.org/10.1128/mbio.02880-22)

---

## Clostridia isolated from helminth-colonized humans promote the life cycle of <em>Trichuris</em> species

[Source](https://doi.org/10.1016/j.celrep.2022.111725)

---

## Zooming into the structure of the microbiome

[Source](https://doi.org/10.1038/s41579-022-00834-6)

---

## <em>Saussurea involucrata</em> oral liquid regulates gut microbiota and serum metabolism during alleviation of collagen-induced arthritis in rats

[Source](https://doi.org/10.1002/ptr.7681)

---

## Prebiotic oligofructose protects against high-fat diet-induced obesity by changing the gut microbiota, intestinal mucus production, glycosylation and secretion

[Source](https://doi.org/10.1080/19490976.2022.2152307)

---

## Predicting Personalized Responses to Dietary Fiber Interventions: Opportunities for Modulation of the Gut Microbiome to Improve Health

[Source](https://doi.org/10.1146/annurev-food-060721-015516)

---

## Gut dysbiosis induces the development of mastitis through a reduction in host anti-inflammatory enzyme activity by endotoxemia

[Source](https://doi.org/10.1186/s40168-022-01402-z)

---

## Chronic clinical signs of upper respiratory tract disease associate with gut and respiratory microbiomes in a cohort of domestic felines

[Source](https://doi.org/10.1371/journal.pone.0268730)

---

## Bio-Organic Fertilizer Promotes Pear Yield by Shaping the Rhizosphere Microbiome Composition and Functions

[Source](https://doi.org/10.1128/spectrum.03572-22)

---

## Impacts of maize hybrids with different nitrogen use efficiency on root-associated microbiota based on distinct rhizosphere soil metabolites

[Source](https://doi.org/10.1111/1462-2920.16293)

---

## Coupled C, H, N, S and Fe biogeochemical cycles operating in the continental deep subsurface of the Iberian Pyrite Belt

[Source](https://doi.org/10.1111/1462-2920.16291)

---

## Long-Read Sequencing Improves Recovery of Picoeukaryotic Genomes and Zooplankton Marker Genes from Marine Metagenomes

[Source](https://doi.org/10.1128/msystems.00595-22)

---

## The Application of Spearman Partial Correlation for Screening Predictors of Weight Loss in a Multiomics Dataset

[Source](https://doi.org/10.1089/omi.2022.0135)

---

## Sampling of Human Microbiomes to Screen for Antibiotic-Producing Commensals.
 05.12.2022, _B Torres Salazar, A Lange, L Camus, S Heilbronner_


_Humans, Anti-Bacterial Agents, Microbiota, Skin, Nose, Soil_
Soil-derived microorganisms have been sampled intensively throughout the last decades in order to discover bacterial strains that produce new antibiotics. The increasing emergence of multidrug-resistant bacteria and the constant high demand for new antibiotic classes are leading to the sampling and investigation of new microbiomes that contain antimicrobial producers. Human-associated microbiomes are therefore gaining more and more attention. This chapter presents a detailed description of how human microbiomes can be sampled and how microbiota members from skin and nasal samples can be isolated. Different methods for antimicrobial compound screening are presented.

[Source](https://doi.org/10.1007/978-1-0716-2855-3_3)

10.1007/978-1-0716-2855-3_3

---

