## Fecal microbiota transplantation for the improvement of metabolism in obesity: The FMT-TRIM double-blind placebo-controlled pilot trial.
 09.03.2020, _EW Yu, L Gao, P Stastka, MC Cheney, J Mahabamunuge, M Torres Soto, CB Ford, JA Bryant, MR Henn, EL Hohmann_


There is intense interest about whether modulating gut microbiota can impact systemic metabolism. We investigated the safety of weekly oral fecal microbiota transplantation (FMT) capsules from healthy lean donors and their ability to alter gut microbiota and improve metabolic outcomes in patients with obesity.

[Source](https://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.1003051)

10.1371/journal.pmed.1003051

---

## Diet influences the functions of the human intestinal microbiome.
 09.03.2020, _M De Angelis, I Ferrocino, FM Calabrese, F De Filippis, N Cavallo, S Siragusa, S Rampelli, R Di Cagno, K Rantsiou, L Vannini, N Pellegrini, C Lazzi, S Turroni, N Lorusso, M Ventura, M Chieppa, E Neviani, P Brigidi, PW O'Toole, D Ercolini, M Gobbetti, L Cocolin_


Gut microbes programme their metabolism to suit intestinal conditions and convert dietary components into a panel of small molecules that ultimately affect host physiology. To unveil what is behind the effects of key dietary components on microbial functions and the way they modulate host-microbe interaction, we used for the first time a multi-omic approach that goes behind the mere gut phylogenetic composition and provides an overall picture of the functional repertoire in 27 fecal samples from omnivorous, vegan and vegetarian volunteers. Based on our data, vegan and vegetarian diets were associated to the highest abundance of microbial genes/proteins responsible for cell motility, carbohydrate- and protein-hydrolyzing enzymes, transport systems and the synthesis of essential amino acids and vitamins. A positive correlation was observed when intake of fiber and the relative fecal abundance of flagellin were compared. Microbial cells and flagellin extracted from fecal samples of 61 healthy donors modulated the viability of the human (HT29) colon carcinoma cells and the host response through the stimulation of the expression of Toll-like receptor 5, lectin RegIIIα and three interleukins (IL-8, IL-22 and IL-23). Our findings concretize a further and relevant milestone on how the diet may prevent/mitigate disease risk.

[Source](https://www.nature.com/articles/s41598-020-61192-y)

10.1038/s41598-020-61192-y

---

## <em>Bacillus coagulans</em> SANK 70258 suppresses <em>Enterobacteriaceae</em> in the microbiota of ulcerative colitis in vitro and enhances butyrogenesis in healthy microbiota

[Source](https://link.springer.com/article/10.1007/s00253-020-10506-1)

---

## Gut epithelial metabolism as a key driver of intestinal dysbiosis associated with noncommunicable diseases.
 03.03.2020, _CD Shelton, MX Byndloss_


In high-income countries, the leading causes of death are non-communicable diseases (NCD), such as obesity, cancer and cardiovascular disease. An important feature of most NCDs is inflammation-induced gut dysbiosis characterized by a shift in the microbial community structure from obligate to facultative anaerobes such as Proteobacteria. This microbial imbalance can contribute to disease pathogenesis by either a depletion in or the production of microbiota derived metabolites. However, little is known about the mechanism by which inflammation mediated changes in host physiology disrupt the microbial ecosystem in our large intestine leading to disease. Recent work by our group suggests that during gut homeostasis, epithelial hypoxia derived from PPARγ-dependent β-oxidation of microbiota-derived short-chain fatty acids limits oxygen availability in the colon, thereby maintaining a balanced microbial community. During inflammation, disruption in gut anaerobiosis drives an expansion of facultative anaerobic Enterobacteriaceae, regardless of their pathogenic potential. Therefore, our research group is currently exploring the concept that dysbiosis-associated expansion of Enterobacteriaceae can be viewed as a microbial signature of epithelial dysfunction and may play a greater role in different models of NCDs, including diet-induced obesity, atherosclerosis and inflammation-associated colorectal cancer.

[Source](https://iai.asm.org/content/early/2020/02/26/IAI.00939-19.abstract)

10.1128/IAI.00939-19

---

## Etiologic features of diarrheagenic microbes in stool specimens from patients with acute diarrhea in Thailand.
 05.03.2020, _K Okada, W Wongboot, W Kamjumphol, N Suebwongsa, P Wangroongsarb, P Kluabwang, N Chuenchom, W Swaddiwudhipong, T Wongchai, W Manosuthi, N Assawapatchara, P Khum-On, P Udompat, C Thanee, S Kitsaran, L Jirapong, C Jaiwong, S Nedsuwan, C Siripipattanamongkol, PA Okada, S Chantaroj, S Komukai, S Hamada_


Many microbial species have been recognized as enteropathogens for humans. Here, we predicted the causative agents of acute diarrhea using data from multiplex quantitative PCR (qPCR) assays targeting 19 enteropathogens. For this, a case-control study was conducted at eight hospitals in Thailand. Stool samples and clinical data were collected from 370 hospitalized patients with acute diarrhea and 370 non-diarrheal controls. Multiple enteropathogens were detected in 75.7% and 13.0% of diarrheal stool samples using multiplex qPCR and bacterial culture methods, respectively. Asymptomatic carriers of enteropathogens were found among 87.8% and 45.7% of individuals by qPCR and culture methods, respectively. These results suggested the complexity of identifying causative agents of diarrhea. An analysis using the quantification cut-off values for clinical relevance drastically reduced pathogen-positive stool samples in control subjects from 87.8% to 0.5%, whereas 48.9% of the diarrheal stool samples were positive for any of the 11 pathogens. Among others, rotavirus, norovirus GII, Shigella/EIEC, and Campylobacter were strongly associated with acute diarrhea (P-value &lt; 0.001). Characteristic clinical symptoms, epidemic periods, and age-related susceptibility to infection were observed for some enteropathogens. Investigations based on qPCR approaches covering a broad array of enteropathogens might thus improve our understanding of diarrheal disease etiology and epidemiological trends.

[Source](https://www.nature.com/articles/s41598-020-60711-1)

10.1038/s41598-020-60711-1

---

## Raman-activated sorting of antibiotic-resistant bacteria in human gut microbiota.
 01.03.2020, _Y Wang, J Xu, L Kong, B Li, H Li, WE Huang, C Zheng_


The antibiotic-resistant bacteria (ARB) and antibiotic-resistant genes (ARGs) in human gut microbiota have significant impact on human health. Whilst high throughput metagenomic sequencing reveals genotypes of bacteria communities, the functionality, phenotype and heterogeneity of human gut microbiota are still elusive. In this study, we applied Raman microscopy with deuterium isotope probing (Raman-DIP) to detect metabolic active ARB(MA-ARB) in situ at the single-cell level in human gut microbiota from two healthy adults. We analyzed the relative abundances of MA-ARB under different concentrations of amoxicillin, cephalexin, tetracycline, florfenicol and vancomycin. To establish the link between phenotypes and genotypes of the MA-ARB, Raman-activated cell sorting (RACS) was used to sort MA-ARB from human gut microbiota, and mini-metagenomic DNA of the sorted bacteria was amplified, sequenced and analyzed. The sorted MA-ARB and their associated ARGs were identified. We revealed that a genotype does not necessarily lead to a phenotype due to the lack of gene expression. Our results suggest a strong relation between ARB in human gut microbiota and personal medical history. This study demonstrates that the toolkit of Raman-DIP, RACS and DNA sequencing can be useful to unravel both phenotypes and genotypes of ARB in human gut microbiota at the single-cell level. This article is protected by copyright. All rights reserved.

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1462-2920.14962)

10.1111/1462-2920.14962

---

## Metagenomic analysis reveals the microbiome and resistome in migratory birds.
 03.03.2020, _J Cao, Y Hu, F Liu, Y Wang, Y Bi, N Lv, J Li, B Zhu, GF Gao_


Antibiotic-resistant pathogens pose high risks to human and animal health worldwide. In recent years, the role of gut microbiota as a reservoir of antibiotic resistance genes (ARGs) in humans and animals has been increasingly investigated. However, the structure and function of the gut bacterial community, as well as the ARGs they carry in migratory birds remain unknown.

[Source](https://microbiomejournal.biomedcentral.com/track/pdf/10.1186/s40168-019-0781-8)

10.1186/s40168-019-0781-8

---

## Lack of evidence for microbiota in the placental and fetal tissues of rhesus macaques

[Source](https://www.biorxiv.org/content/10.1101/2020.03.05.980052v1)

---

## Diarrhoeal events can trigger long-term<em> Clostridium difficile</em> colonization with recurrent blooms

[Source](https://www.nature.com/articles/s41564-020-0668-2)

---

## Long-Term Phytoremediation of Coastal Saline Soil Reveals Plant Species-Specific Patterns of Microbial Community Recruitment.
 08.03.2020, _X Wang, R Sun, Y Tian, K Guo, H Sun, X Liu, H Chu, B Liu_


Soil salinization is one of the major land degradation processes that decreases soil fertility and crop production worldwide. In this study, a long-term coastal saline soil remediation experiment was conducted with three salt-tolerant plant species: Lycium chinense Mill. (LCM), Tamarix chinensis Lour. (TCL), and Gossypium hirsutum Linn. (GHL). The three plant species successfully remediated the saline soil but showed different efficacies. The archaeal, bacterial, and fungal communities in barren soil and in four rhizocompartments (distal-rhizosphere soil, proximal-rhizosphere soil, rhizoplane, and endosphere) of the three plant species were assessed. All three plant species significantly decreased the richness of the archaeal communities but increased that of the bacterial and fungal communities in both the rhizosphere and rhizoplane compared with those in the barren soil. The archaeal and bacterial community structures were strongly influenced by the rhizocompartment, while specific fungal communities were recruited by different plant species. The microbial taxa whose abundance either increased or decreased significantly during remediation were identified. Soil electrical conductivity (EC) was identified as the main factor driving the variation in microbial community composition between the remediated and barren soil, and total nitrogen (TN), total carbon (TC), and available potassium (AK) were the main factors driving the differences among plant species. This report provides new insights into the responses of the root zone microbial communities of different salt-tolerant plant species during phytoremediation.IMPORTANCE Despite knowing that phytoremediation by salt-tolerant plants is an effective technology for ameliorating saline soils and that microorganisms contribute significantly to plant stress tolerance and soil fertility, we still lack a comprehensive understanding of how microbes respond to the growth of salt-tolerant plants and the subsequent decline in soil salinity. The results of this study revealed different response patterns among bacterial, archaeal, and fungal communities and indicated that the decline in archaeal abundance might be a sign of successful remediation of coastal saline soils. The recruitment of specific fungal communities by different plant species indicated the importance of fungi in plant species-specific remediation functions. We also identified the taxa that may play key roles during remediation, and these taxa could potentially be used as indicators of phytoremediation. Overall, these findings highlight the importance of microbes in the phytoremediation of saline soil and suggest that the mechanisms involved are plant species specific.

[Source](https://msystems.asm.org/content/5/2/e00741-19)

10.1128/mSystems.00741-19

---

## Impact of Soil Microbial Amendments on Tomato Rhizosphere Microbiome and Plant Growth in Field Soil.
 07.03.2020, _A Nuzzo, A Satpute, U Albrecht, SL Strauss_


There is increased interest by the agricultural industry in microbial amendments that leverage natural beneficial interactions between plants and soil microbes to improve crop production. However, translating fundamental knowledge from laboratory experiments into efficient field application often has mixed results, and there is less clarity about the interaction between added microbes and the native microbial community, where microorganisms belonging to the same phylogenic clades often reside. In this study, four commercially available microbial amendments were examined in two greenhouse experiments using field soil to assess their impact on tomato plant growth and the native soil microbial communities. The amendments contained different formulations of plant growth-promoting bacteria (Lactobacilli, Rhizobia, etc.), yeasts, and mycorrhizal fungi. The application of the tested amendments in greenhouse conditions resulted in no significant impact on plant growth. A deeper statistical analysis detected variations in the microbial communities that accounted only for 0.25% of the total species, particularly in native taxa not related to the inoculated species and represented less than 1% of the total variance. This suggests that under commercial field conditions, additional confounding variables may play a role in the efficacy of soil microbial amendments. This study confirms the necessity of more in-depth validation requirements for the formulations of soil microbial amendments before delivery to the agricultural market in order to leverage their benefits for the producers, the consumers, and the environment.

[Source](https://link.springer.com/article/10.1007/s00248-020-01497-7)

10.1007/s00248-020-01497-7

---

## DeepMAsED: Evaluating the quality of metagenomic assemblies.
 25.02.2020, _O Mineeva, M Rojas-Carulla, RE Ley, B Schölkopf, ND Youngblut_


Methodological advances in metagenome assembly are rapidly increasing in the number of published metagenome assemblies. However, identifying misassemblies is challenging due to a lack of closely related reference genomes that can act as pseudo ground truth. Existing reference-free methods are no longer maintained, can make strong assumptions that may not hold across a diversity of research projects, and have not been validated on large scale metagenome assemblies.

[Source](https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btaa124/5756210)

10.1093/bioinformatics/btaa124

---

## A method for achieving complete microbial genomes and better quality bins from metagenomics dat

[Source](https://www.biorxiv.org/content/biorxiv/early/2020/03/07/2020.03.05.979740.full.pdf)

---

## Estimating relative biomasses of organisms in microbiota using "phylopeptidomics".
 09.03.2020, _O Pible, F Allain, V Jouffret, K Culotta, G Miotello, J Armengaud_


There is an important need for the development of fast and robust methods to quantify the diversity and temporal dynamics of microbial communities in complex environmental samples. Because tandem mass spectrometry allows rapid inspection of protein content, metaproteomics is increasingly used for the phenotypic analysis of microbiota across many fields, including biotechnology, environmental ecology, and medicine.

[Source](https://microbiomejournal.biomedcentral.com/track/pdf/10.1186/s40168-020-00797-x)

10.1186/s40168-020-00797-x

---

