## Androgen-induced gut dysbiosis disrupts glucolipid metabolism and endocrinal functions in polycystic ovary syndrome.
 07.05.2021, _Q Han, J Wang, W Li, ZJ Chen, Y Du_


The characteristics of polycystic ovary syndrome (PCOS), a common reproductive endocrinal disorder, are high incidence, complicated aetiology and poor therapeutic effects. PCOS patients frequently exhibit gut dysbiosis; however, its roles in the regulation of metabolic and endocrinal balances in PCOS pathophysiology are not clear.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01046-5)

10.1186/s40168-021-01046-5

---

## Diet, obesity, and the gut microbiome as determinants modulating metabolic outcomes in a non-human primate model 

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01069-y)

---

## Metagenomic Analysis of Common Intestinal Diseases Reveals Relationships among Microbial Signatures and Powers Multidisease Diagnostic Models.
 05.05.2021, _P Jiang, S Wu, Q Luo, XM Zhao, WH Chen_


Common intestinal diseases such as Crohn's disease (CD), ulcerative colitis (UC), and colorectal cancer (CRC) share clinical symptoms and altered gut microbes, necessitating cross-disease comparisons and the use of multidisease models. Here, we performed meta-analyses on 13 fecal metagenome data sets of the three diseases. We identified 87 species and 65 pathway markers that were consistently changed in multiple data sets of the same diseases. According to their overall trends, we grouped the disease-enriched marker species into disease-specific and disease-common clusters and revealed their distinct phylogenetic relationships; species in the CD-specific cluster were phylogenetically related, while those in the CRC-specific cluster were more distant. Strikingly, UC-specific species were phylogenetically closer to CRC, likely because UC patients have higher risk of CRC. Consistent with their phylogenetic relationships, marker species had similar within-cluster and different between-cluster metabolic preferences. A portion of marker species and pathways correlated with an indicator of leaky gut, suggesting a link between gut dysbiosis and human-derived contents. Marker species showed more coordinated changes and tighter inner-connections in cases than the controls, suggesting that the diseased gut may represent a stressed environment and pose stronger selection on gut microbes. With the marker species and pathways, we constructed four high-performance (including multidisease) models with an area under the receiver operating characteristic curve (AUROC) of 0.87 and true-positive rates up to 90%, and explained their putative clinical applications. We identified consistent microbial alterations in common intestinal diseases, revealed metabolic capacities and the relationships among marker bacteria in distinct states, and supported the feasibility of metagenome-derived multidisease diagnosis.IMPORTANCE Gut microbes have been identified as potential markers in distinguishing patients from controls in colorectal cancer, ulcerative colitis, and Crohn's disease individually, whereas there lacks a systematic analysis to investigate the exclusive microbial shifts of these enteropathies with similar clinical symptoms. Our meta-analysis and cross-disease comparisons identified consistent microbial alterations in each enteropathy, revealed microbial ecosystems among marker bacteria in distinct states, and demonstrated the necessity and feasibility of metagenome-based multidisease classifications. To the best of our knowledge, this is the first study to construct multiclass models for these common intestinal diseases.

[Source](https://msystems.asm.org/content/6/3/e00112-21)

10.1128/mSystems.00112-21

---

## Individuals at risk for rheumatoid arthritis harbor differential intestinal bacteriophage communities with distinct metabolic potential.
 06.05.2021, _MR Mangalea, D Paez-Espino, K Kieft, A Chatterjee, ME Chriswell, JA Seifert, ML Feser, MK Demoruelle, A Sakatos, K Anantharaman, KD Deane, KA Kuhn, VM Holers, BA Duerkop_


Rheumatoid arthritis (RA) is an autoimmune disease characterized in seropositive individuals by the presence of anti-cyclic citrullinated protein (CCP) antibodies. RA is linked to the intestinal microbiota, yet the association of microbes with CCP serology and their contribution to RA is unclear. We describe intestinal phage communities of individuals at risk for developing RA, with or without anti-CCP antibodies, whose first-degree relatives have been diagnosed with RA. We show that at-risk individuals harbor intestinal phage compositions that diverge based on CCP serology, are dominated by Streptococcaceae, Bacteroidaceae, and Lachnospiraceae phages, and may originate from disparate ecosystems. These phages encode unique repertoires of auxiliary metabolic genes, which associate with anti-CCP status, suggesting that these phages directly influence the metabolic and immunomodulatory capability of the microbiota. This work sets the stage for the use of phages as preclinical biomarkers and provides insight into a possible microbial-based causation of RA disease development.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(21)00148-7)

10.1016/j.chom.2021.03.020

---

## Integrative microbiomics in bronchiectasis exacerbations.
 06.05.2021, _M Mac Aogáin, JK Narayana, PY Tiew, NABM Ali, VFL Yong, TK Jaggi, AYH Lim, HR Keir, AJ Dicker, KX Thng, A Tsang, FX Ivan, ME Poh, M Oriano, S Aliberti, F Blasi, TB Low, TH Ong, B Oliver, YH Giam, A Tee, MS Koh, JA Abisheganaden, K Tsaneva-Atanasova, JD Chalmers, SH Chotirmall_


Bronchiectasis, a progressive chronic airway disease, is characterized by microbial colonization and infection. We present an approach to the multi-biome that integrates bacterial, viral and fungal communities in bronchiectasis through weighted similarity network fusion ( https://integrative-microbiomics.ntu.edu.sg ). Patients at greatest risk of exacerbation have less complex microbial co-occurrence networks, reduced diversity and a higher degree of antagonistic interactions in their airway microbiome. Furthermore, longitudinal interactome dynamics reveals microbial antagonism during exacerbation, which resolves following treatment in an otherwise stable multi-biome. Assessment of the Pseudomonas interactome shows that interaction networks, rather than abundance alone, are associated with exacerbation risk, and that incorporation of microbial interaction data improves clinical prediction models. Shotgun metagenomic sequencing of an independent cohort validated the multi-biome interactions detected in targeted analysis and confirmed the association with exacerbation. Integrative microbiomics captures microbial interactions to determine exacerbation risk, which cannot be appreciated by the study of a single microbial group. Antibiotic strategies probably target the interaction networks rather than individual microbes, providing a fresh approach to the understanding of respiratory infection.

[Source](https://www.nature.com/articles/s41591-021-01289-7)

10.1038/s41591-021-01289-7

---

## The International Scientific Association of Probiotics and Prebiotics (ISAPP) consensus statement on the definition and scope of postbiotics.
 05.05.2021, _S Salminen, MC Collado, A Endo, C Hill, S Lebeer, EMM Quigley, ME Sanders, R Shamir, JR Swann, H Szajewska, G Vinderola_


In 2019, the International Scientific Association for Probiotics and Prebiotics (ISAPP) convened a panel of experts specializing in nutrition, microbial physiology, gastroenterology, paediatrics, food science and microbiology to review the definition and scope of postbiotics. The term 'postbiotics' is increasingly found in the scientific literature and on commercial products, yet is inconsistently used and lacks a clear definition. The purpose of this panel was to consider the scientific, commercial and regulatory parameters encompassing this emerging term, propose a useful definition and thereby establish a foundation for future developments. The panel defined a postbiotic as a "preparation of inanimate microorganisms and/or their components that confers a health benefit on the host". Effective postbiotics must contain inactivated microbial cells or cell components, with or without metabolites, that contribute to observed health benefits. The panel also discussed existing evidence of health-promoting effects of postbiotics, potential mechanisms of action, levels of evidence required to meet the stated definition, safety and implications for stakeholders. The panel determined that a definition of postbiotics is useful so that scientists, clinical triallists, industry, regulators and consumers have common ground for future activity in this area. A generally accepted definition will hopefully lead to regulatory clarity and promote innovation and the development of new postbiotic products.

[Source](https://www.nature.com/articles/s41575-021-00440-6)

10.1038/s41575-021-00440-6

---

## Fungal taxonomy and sequence-based nomenclature.
 29.04.2021, _R Lücking, MC Aime, B Robbertse, AN Miller, T Aoki, HA Ariyawansa, G Cardinali, PW Crous, IS Druzhinina, DM Geiser, DL Hawksworth, KD Hyde, L Irinyi, R Jeewon, PR Johnston, PM Kirk, E Malosso, TW May, W Meyer, HR Nilsson, M Öpik, V Robert, M Stadler, M Thines, D Vu, AM Yurkov, N Zhang, CL Schoch_


The identification and proper naming of microfungi, in particular plant, animal and human pathogens, remains challenging. Molecular identification is becoming the default approach for many fungal groups, and environmental metabarcoding is contributing an increasing amount of sequence data documenting fungal diversity on a global scale. This includes lineages represented only by sequence data. At present, these taxa cannot be formally described under the current nomenclature rules. By considering approaches used in bacterial taxonomy, we propose solutions for the nomenclature of taxa known only from sequences to facilitate consistent reporting and communication in the literature and public sequence repositories.

[Source](https://www.nature.com/articles/s41564-021-00888-x)

10.1038/s41564-021-00888-x

---

## A Streptococcus Quorum Sensing System Enables Suppression of Innate Immunity.
 07.05.2021, _KM Rahbari, JC Chang, MJ Federle_


Some bacterial pathogens utilize cell-cell communication systems, such as quorum sensing (QS), to coordinate genetic programs during host colonization and infection. The human-restricted pathosymbiont Streptococcus pyogenes (group A streptococcus [GAS]) uses the Rgg2/Rgg3 QS system to modify the bacterial surface, enabling biofilm formation and lysozyme resistance. Here, we demonstrate that innate immune cell responses to GAS are substantially altered by the QS status of the bacteria. We found that macrophage activation, stimulated by multiple agonists and assessed by cytokine production and NF-κB activity, was substantially suppressed upon interaction with QS-active GAS but not QS-inactive bacteria. Neither macrophage viability nor bacterial adherence, internalization, or survival were altered by the QS activation status, yet tumor necrosis factor alpha (TNF-α), interleukin 6 (IL-6), and interferon beta (IFN-β) levels and NF-κB reporter activity were drastically lower following infection with QS-active GAS. Suppression required contact between viable bacteria and macrophages. A QS-regulated biosynthetic gene cluster (BGC) in the GAS genome, encoding several putative enzymes, was also required for macrophage modulation. Our findings suggest a model wherein upon contact with macrophages, QS-active GAS produce a BGC-derived factor capable of suppressing inflammatory responses. The suppressive capability of QS-active GAS is abolished after treatment with a specific QS inhibitor. These observations suggest that interfering with the ability of bacteria to collaborate via QS can serve as a strategy to counteract microbial efforts to manipulate host defenses.IMPORTANCEStreptococcus pyogenes is restricted to human hosts and commonly causes superficial diseases such as pharyngitis; it can also cause severe and deadly manifestations including necrotizing skin disease or severe postinfectious sequelae like rheumatic heart disease. Understanding the complex mechanisms used by this pathogen to manipulate host defenses could aid in developing new therapeutics to treat infections. Here, we examine the impact of a bacterial cell-cell communication system, which is highly conserved across S. pyogenes, on host innate immune responses. We find that S. pyogenes uses this system to suppress macrophage proinflammatory cytokine responses in vitro Interference with this communication system could serve as a strategy to disarm bacteria and maintain an effective immune response.

[Source](https://mbio.asm.org/content/12/3/e03400-20)

10.1128/mBio.03400-20

---

## Tamock: simulation of habitat-specific benchmark data in metagenomics.
 07.05.2021, _SM Gerner, AB Graf, T Rattei_


_Algorithms, Benchmarking, Metagenome, Metagenomics, Sequence Analysis, DNA, Software_
Simulated metagenomic reads are widely used to benchmark software and workflows for metagenome interpretation. The results of metagenomic benchmarks depend on the assumptions about their underlying ecosystems. Conclusions from benchmark studies are therefore limited to the ecosystems they mimic. Ideally, simulations are therefore based on genomes, which resemble particular metagenomic communities realistically.

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04154-z)

10.1186/s12859-021-04154-z

---

## Whole lung tissue is the preferred sampling method for amplicon-based characterization of murine lung microbiota.
 06.05.2021, _JM Baker, KJ Hinkle, RA McDonald, CA Brown, NR Falkowski, GB Huffnagle, RP Dickson_


Low-biomass microbiome studies (such as those of the lungs, placenta, and skin) are vulnerable to contamination and sequencing stochasticity, which obscure legitimate microbial signal. While human lung microbiome studies have rigorously identified sampling strategies that reliably capture microbial signal from these low-biomass microbial communities, the optimal sampling strategy for characterizing murine lung microbiota has not been empirically determined. Performing accurate, reliable characterization of murine lung microbiota and distinguishing true microbial signal from noise in these samples will be critical for further mechanistic microbiome studies in mice.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01055-4)

10.1186/s40168-021-01055-4

---

