## Major data analysis errors invalidate cancer microbiome findings.
 09.10.2023, _A Gihawi, Y Ge, J Lu, D Puiu, A Xu, CS Cooper, DS Brewer, M Pertea, SL Salzberg_


We re-analyzed the data from a recent large-scale study that reported strong correlations between DNA signatures of microbial organisms and 33 different cancer types and that created machine-learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (i) errors in the genome database and the associated computational methods led to millions of false-positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (ii) errors in the transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine-learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.IMPORTANCERecent reports showing that human cancers have a distinctive microbiome have led to a flurry of papers describing microbial signatures of different cancer types. Many of these reports are based on flawed data that, upon re-analysis, completely overturns the original findings. The re-analysis conducted here shows that most of the microbes originally reported as associated with cancer were not present at all in the samples. The original report of a cancer microbiome and more than a dozen follow-up studies are, therefore, likely to be invalid.

[Source](https://journals.asm.org/doi/10.1128/mbio.01607-23)

10.1128/mbio.01607-23

---

## Relative abundance data can misrepresent heritability of the microbiome.
 11.10.2023, _M Bruijning, JF Ayroles, LP Henry, B Koskella, KM Meyer, CJE Metcalf_


Host genetics can shape microbiome composition, but to what extent it does, remains unclear. Like any other complex trait, this important question can be addressed by estimating the heritability (h2) of the microbiome-the proportion of variance in the abundance in each taxon that is attributable to host genetic variation. However, unlike most complex traits, microbiome heritability is typically based on relative abundance data, where taxon-specific abundances are expressed as the proportion of the total microbial abundance in a sample.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01669-w)

10.1186/s40168-023-01669-w

---

## Discrimination exposure impacts unhealthy processing of food cues: crosstalk between the brain and gut

[Source](https://www.nature.com/articles/s44220-023-00134-9)

---

## Vertical transmission of gut bacteria in commercial chickens is limited.
 12.10.2023, _N Shterzer, N Rothschild, Y Sbehat, J Dayan, D Eytan, Z Uni, E Mills_


The existence of vertical transmission in chickens under commercial settings, where chicks are raised separately from adults, is unclear. To answer this question, the fecal microbiota of chicks hatched and grown separately was compared with their mothers' microbiota. Most amplicon sequence variants (ASVs) identified in hens were not detected at all in chicks up to two weeks of age by 16S rDNA sequencing, and those that were detected had a low incidence among the chicks. Nevertheless, a few ASVs that were common with the hens were highly prevalent among the chicks, implying that they were efficiently transmitted to chicks. These ASVs were culturable from the reproductive tract of hens and eggshells. Furthermore, interventions attempting to disrupt transmission resulted in a reduction in the prevalence of specific phylogenetic groups in chicks. To conclude, vertical transmission in commercial poultry grown separately from adults likely exists but is not efficient, possibly resulting in impairment of microbiota function. This implies that artificial exposure to adult bacterial strains might improve microbiota functioning.

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-023-00272-6)

10.1186/s42523-023-00272-6

---

## The role of family and environment in determining the skin microbiome of captive aquatic frogs, Xenopus laevis

[Source](https://www.biorxiv.org/content/10.1101/2023.10.05.561135v1)

---

## Vertical transmission of gut bacteria in commercial chickens is limited.
 12.10.2023, _N Shterzer, N Rothschild, Y Sbehat, J Dayan, D Eytan, Z Uni, E Mills_


The existence of vertical transmission in chickens under commercial settings, where chicks are raised separately from adults, is unclear. To answer this question, the fecal microbiota of chicks hatched and grown separately was compared with their mothers' microbiota. Most amplicon sequence variants (ASVs) identified in hens were not detected at all in chicks up to two weeks of age by 16S rDNA sequencing, and those that were detected had a low incidence among the chicks. Nevertheless, a few ASVs that were common with the hens were highly prevalent among the chicks, implying that they were efficiently transmitted to chicks. These ASVs were culturable from the reproductive tract of hens and eggshells. Furthermore, interventions attempting to disrupt transmission resulted in a reduction in the prevalence of specific phylogenetic groups in chicks. To conclude, vertical transmission in commercial poultry grown separately from adults likely exists but is not efficient, possibly resulting in impairment of microbiota function. This implies that artificial exposure to adult bacterial strains might improve microbiota functioning.

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-023-00272-6)

10.1186/s42523-023-00272-6

---

## On pickles: biological and sociocultural links between fermented foods and the human gut microbiome.
 04.12.2021, _A Flachs, JD Orkin_


_Fermentation, Fermented Foods, Food Microbiology, Gastrointestinal Microbiome, Humans, Microbiota_
The composition of the human microbiome varies considerably in diversity and density across communities as a function of the foods we eat and the places we live. While all foods contain microbes, humans directly shape this microbial ecology through fermentation. Fermented foods are produced from microbial reactions that depend on local environmental conditions, fermentation practices, and the manner in which foods are prepared and consumed. These interactions are of special interest to ethnobiologists because they link investigations of how people shape and know the world around them to local knowledge, food traditions, local flora, and microbial taxa.

[Source](https://www.biorxiv.org/content/10.1101/2023.10.09.561628v1)

10.1186/s13002-021-00458-9

---

## MetaQuad: Shared Informative Variants Discovery in Metagenomic Samples

[Source](https://www.biorxiv.org/content/10.1101/2023.10.09.561628v1)

---

## Insertion sequence driven evolution of the microbiota revealed by a novel metagenomics pipeline

[Source](https://www.biorxiv.org/content/10.1101/2023.10.06.561241v1)

---

## A comparison of short-read, HiFi long-read, and hybrid strategies for genome-resolved metagenomics

[Source](https://www.biorxiv.org/content/10.1101/2023.10.04.560907v1)

---

## The newest Oxford Nanopore R10.4.1 full-length 16S rRNA sequencing enables the accurate resolution of species-level microbial community profiling.
 06.10.2023, _T Zhang, H Li, S Ma, J Cao, H Liao, Q Huang, W Chen_


The long-read amplicon provides a species-level solution for the community. With the improvement of nanopore flowcells, the accuracy of Oxford Nanopore Technologies (ONT) R10.4.1 has been substantially enhanced, with an average of approximately 99%. To evaluate its effectiveness on amplicons, three types of microbiomes were analyzed by 16S ribosomal RNA (hereinafter referred to as "16S") amplicon sequencing using Novaseq, Pacbio sequel II, and Nanopore PromethION platforms (R9.4.1 and R10.4.1) in the current study. We showed the error rate, recall, precision, and bias index in the mock sample. The error rate of ONT R10.4.1 was greatly reduced, with a better recall in the case of the synthetic community. Meanwhile, in different types of environmental samples, ONT R10.4.1 analysis resulted in a composition similar to Pacbio data. We found that classification tools and databases influence ONT data. Based on these results, we conclude that the ONT R10.4.1 16S amplicon can also be used for application in environmental samples. IMPORTANCE The long-read amplicon supplies the community with a species-level solution. Due to the high error rate of nanopore sequencing early on, it has not been frequently used in 16S studies. Oxford Nanopore Technologies (ONT) introduced the R10.4.1 flowcell with Q20+ reagent to achieve more than 99% accuracy as sequencing technology advanced. However, there has been no published study on the performance of commercial PromethION sequencers with R10.4.1 flowcells on 16S sequencing or on the impact of accuracy improvement on taxonomy (R9.4.1 to R10.4.1) using 16S ONT data. In this study, three types of microbiomes were investigated by 16S ribosomal RNA (rRNA) amplicon sequencing using Novaseq, Pacbio sequel II, and Nanopore PromethION platforms (R9.4.1 and R10.4.1). In the mock sample, we displayed the error rate, recall, precision, and bias index. We observed that the error rate in ONT R10.4.1 is significantly lower, especially when deletions are involved. First and foremost, R10.4.1 and Pacific Bioscience platforms reveal a similar microbiome in environmental samples. This study shows that the R10.4.1 full-length 16S rRNA sequences allow for species identification of environmental microbiota.

[Source](https://journals.asm.org/doi/abs/10.1128/aem.00605-23)

10.1128/aem.00605-23

---

## Challenges and opportunities in sharing microbiome data and analyses.
 02.10.2023, _C Huttenhower, RD Finn, AC McHardy_


Microbiome data, metadata and analytical workflows have become 'big' in terms of volume and complexity. Although the infrastructure and technologies to share data have been established, the interdisciplinary and multi-omic nature of the field can make resources difficult to identify and use. Following best practices for data deposition requires substantial effort, with sometimes little obvious reward. Gaps remain where microbiome-specific resources for data sharing or reproducibility do not yet exist. We outline available best practices, challenges to their adoption and opportunities in data sharing in microbiome research. We showcase examples of best practices and advocate for their enforcement and incentivization for data sharing. This includes recognition of data curation and sharing endeavours by individuals, institutions, journals and funders. Opportunities for progress include enabling microbiome-specific databases to incorporate future methods for data analysis, integration and reuse.

[Source](https://www.nature.com/articles/s41564-023-01484-x)

10.1038/s41564-023-01484-x

---

