## Microbial species interactions determine community diversity in fluctuating environments

[Source](https://www.biorxiv.org/content/10.1101/2020.07.22.216010v2.full.pdf)

---

## The host-associated archaeome.
 21.07.2020, _G Borrel, JF Brug√®re, S Gribaldo, RA Schmitz, C Moissl-Eichinger_


Host-associated microbial communities have an important role in shaping the health and fitness of plants and animals. Most studies have focused on the bacterial, fungal or viral communities, but often the archaeal component has been neglected. The archaeal community, the so-called archaeome, is now increasingly recognized as an important component of host-associated microbiomes. It is composed of various lineages, including mainly Methanobacteriales and Methanomassiliicoccales (Euryarchaeota), as well as representatives of the Thaumarchaeota. Host-archaeome interactions have mostly been delineated from methanogenic archaea in the gastrointestinal tract, where they contribute to substantial methane production and are potentially also involved in disease-relevant processes. In this Review, we discuss the diversity and potential roles of the archaea associated with protists, plants and animals. We also present the current understanding of the archaeome in humans, the specific adaptations involved in interaction with the resident microbial community as well as with the host, and the roles of the archaeome in both health and disease.

[Source](https://www.nature.com/articles/s41579-020-0407-y)

10.1038/s41579-020-0407-y

---

## Nitrate as a potential prebiotic for the oral microbiome

[Source](https://www.nature.com/articles/s41598-020-69931-x)

---

## Deep skin dysbiosis in vitiligo patients: link with mitochondrial and immune changes

[Source](https://www.medrxiv.org/content/10.1101/2020.07.29.20163469v1.full.pdf)

---

## The microbiome: An emerging key player in aging and longevity

[Source](https://www.sciencedirect.com/science/article/pii/S2468501120300146)

---

## Immunoglobulin recognition of fecal bacteria in stunted and non-stunted children: findings from the Afribiota study

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00890-1)

---

## Changes in the vaginal microbiota across a gradient of urbanization

[Source](https://www.nature.com/articles/s41598-020-69111-x)

---

## Comparison between a novel molecular tool and conventional methods for diagnostic classification of bacterial vaginosis: is integration of the two approaches necessary for a better evaluation?

[Source](http://www.newmicrobiologica.org/PUB/allegati_pdf/AOP/496N171.pdf)

---

## Genome-wide mutagenesis identifies factors involved in <em>Enterococcus faecalis</em> vaginal adherence and persistence

[Source](https://www.biorxiv.org/content/10.1101/2020.04.30.069468v2.full.pdf)

---

## Association between Vaginal Bacterial Microbiota and Vaginal Yeast Colonization

[Source](https://academic.oup.com/jid/article-abstract/doi/10.1093/infdis/jiaa459/5878087)

---

## The microbiome in urological diseases

[Source](https://icurology.org/DOIx.php?id=10.4111/icu.2020.61.4.338)

---

## FimH and Anti-Adhesive Therapeutics: A Disarming Strategy Against Uropathogens

[Source](https://www.mdpi.com/2079-6382/9/7/397)

---

## Characterization of urinary microbiome in patients with bladder cancer: Results from a single-institution, feasibility study

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1078143920301393)

---

## Resistance to change: AMR gene dynamics on a commercial pig farm with high antimicrobial usage

[Source](https://www.nature.com/articles/s41598-020-58659-3)

---

## Vaginal Microbiota Changes During Estrous Cycle in Dairy Heifers

[Source](https://www.frontiersin.org/articles/10.3389/fvets.2020.00371/full)

---

## The gut microbiome switches mutant p53 from tumour-suppressive to oncogenic

[Source](https://www.nature.com/articles/s41586-020-2541-0)

---

## An Enteric Pathogen Subverts Colonization Resistance by Evading Competition for Amino Acids in the Gut

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30359-0)

---

## Consumption of a Western-Style Diet Modulates the Response 

[Source](https://msystems.asm.org/content/msys/5/4/e00317-20.full.pdf)

---

## Consumption of a Western-Style Diet Modulates the Response of the Murine Gut Microbiome to Ciprofloxacin.
 30.07.2020, _DJ Cabral, JI Wurster, BJ Korry, S Penumutchu, P Belenky_


Dietary composition and antibiotic use have major impacts on the structure and function of the gut microbiome, often resulting in dysbiosis. Despite this, little research has been done to explore the role of host diet as a determinant of antibiotic-induced microbiome disruption. Here, we utilize a multi-omic approach to characterize the impact of Western-style diet consumption on ciprofloxacin-induced changes to gut microbiome structure and transcriptional activity. We found that Western diet consumption dramatically increased Bacteroides abundances and shifted the community toward the metabolism of simple sugars and mucus glycoproteins. Mice consuming a Western-style diet experienced a greater expansion of Firmicutes following ciprofloxacin treatment than those eating a control diet. Transcriptionally, we found that ciprofloxacin reduced the abundance of tricarboxylic acid (TCA) cycle transcripts on both diets, suggesting that carbon metabolism plays a key role in the response of the gut microbiome to this antibiotic. Despite this, we observed extensive diet-dependent differences in the impact of ciprofloxacin on microbiota function. In particular, at the whole-community level we detected an increase in starch degradation, glycolysis, and pyruvate fermentation following antibiotic treatment in mice on the Western diet, which we did not observe in mice on the control diet. Similarly, we observed diet-specific changes in the transcriptional activity of two important commensal bacteria, Akkermansia muciniphila and Bacteroides thetaiotaomicron, involving diverse cellular processes such as nutrient acquisition, stress responses, and capsular polysaccharide (CPS) biosynthesis. These findings demonstrate that host diet plays a role in determining the impacts of ciprofloxacin on microbiome composition and microbiome function.IMPORTANCE Due to the growing incidence of disorders related to antibiotic-induced dysbiosis, it is essential to determine how our "Western"-style diet impacts the response of the microbiome to antibiotics. While diet and antibiotics have profound impacts on gut microbiome composition, little work has been done to examine their combined effects. Previous work has shown that nutrient availability, influenced by diet, plays an important role in determining the extent of antibiotic-induced disruption to the gut microbiome. Thus, we hypothesize that the Western diet will shift microbiota metabolism toward simple sugar and mucus degradation and away from polysaccharide utilization. Because of bacterial metabolism's critical role in antibiotic susceptibility, this change in baseline metabolism will impact how the structure and function of the microbiome are impacted by ciprofloxacin exposure. Understanding how diet modulates antibiotic-induced microbiome disruption will allow for the development of dietary interventions that can alleviate many of the microbiome-dependent complications of antibiotic treatment.

[Source](https://msystems.asm.org/content/msys/5/4/e00317-20.full.pdf)

10.1128/mSystems.00317-20

---

## of the Murine Gut Microbiome to Ciprofloxacin

[Source](https://msystems.asm.org/content/msys/5/4/e00317-20.full.pdf)

---

## Determining the primary sources of fecal pollution using microbial source tracking assays combined with land-use information in the Edwards Aquifer.
 30.07.2020, _J Hinojosa, J Green, F Estrada, J Herrera, T Mata, D Phan, ABMT Pasha, A Matta, D Johnson, V Kapoor_


The Edwards Aquifer serves as a primary source of drinking water to more than 2 million people in south-central Texas, and as a karst aquifer, is vulnerable to human and animal fecal contamination which poses a serious risk to human and environmental health. A one-year study (Jan 2018 - Feb 2019) was conducted to determine the primary sources of fecal pollution along the Balcones and Leon Creek within the Edwards Aquifer recharge and contributing zones using general (E. coli, enterococci, and universal Bacteriodales) and host-associated (human-, dog-, cow- and chicken/duck-associated Bacteriodales) microbial source tracking (MST) assays. Additionally, sites were classified based on surrounding land use as a potential source predictor and marker levels were correlated with rain events and water quality parameters. Levels for the three general indicators were highest and exhibited similar trends across the sampling sites, suggesting that the sole use of these markers is not sufficient for specific fecal source identification. Among the host-associated markers, highest concentrations were observed for the dog marker (BacCan) in the Leon Creek area and the cow marker (BacCow) in the Balcones Creek area. Additionally, Chicken/Duck-Bac, BacCan and BacCow all exhibited higher concentrations during the spring season and the end of fall/early winter. Relatively lower concentrations were observed for the human-associated markers (HF183 and BacHum), however, levels were higher in the Leon Creek area and highest following rainfall events. Additionally, relatively higher levels in HF183 and BacHum were observed at sites having greater human population and septic tank density and may be attributed to leaks or breaks in these infrastructures. This study is the first to examine and compare fecal contamination at rural and urban areas in the recharge and contributing zones of the Edwards Aquifer using a molecular MST approach targeting Bacteroidales 16S rRNA gene-based assays. The Bacteroidales marker assays, when combined with land use and weather information, can allow for a better understanding of the sources and fluxes of fecal contamination, which can help devise effective mitigation measures to protect water quality.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S004313542030748X)

10.1016/j.watres.2020.116211

---

## Diverse Microorganisms in Sediment and Groundwater Are Implicated in Extracellular Redox Processes Based on Genomic Analysis of Bioanode Communities

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.01694/full)

---

## Discovering microbiota and volatile compounds of surstr√∂mming, the traditional Swedish sour herring.
 17.06.2020, _L Belleggia, L Aquilanti, I Ferrocino, V Milanoviƒá, C Garofalo, F Clementi, L Cocolin, M Mozzon, R Foligni, MN Haouet, S Scuota, M Framboas, A Osimani_


In this study, the microbiota of ready-to-eat surstr√∂mming from three Swedish producers were studied using a combined approach. The pH values of the samples ranged between 6.67 ¬± 0.01 and 6.98 ¬± 0.01, whereas their aw values were between 0.911 ¬± 0.001 and 0.940 ¬± 0.001. The acetic acid concentration was between 0.289 ¬± 0.009 g/100 g and 0.556 ¬± 0.036 g/100 g. Very low concentrations of lactic acid were measured. Viable counting revealed the presence of mesophilic aerobes, mesophilic lactobacilli and lactococci as well as halophilic lactobacilli and lactococci, coagulase-negative staphylococci, halophilic aerobes and anaerobes. Negligible counts for Enterobacteriaceae, Pseudomonadaceae and total eumycetes were observed, whereas no sulfite-reducing anaerobes were detected. Listeria monocytogenes and Salmonella spp. were absent in all samples. Multiplex real-time PCR revealed the absence of the bont/A, bont/B, bont/E, bont/F, and 4gyrB (CP) genes, which encode botulinic toxins, in all the samples analyzed. Metagenomic sequencing revealed the presence of a core microbiota dominated by Halanaerobium praevalens, Alkalibacterium gilvum, Carnobacterium spp., Tetragenococcus halophilus, Clostridiisalibacter spp. and Porphyromonadaceae. Psychrobacter celer, Ruminococcaceae, Marinilactibacillus psychrotolerans, Streptococcus infantis and Salinivibrio costicola were detected as minor OTUs. GC-MS analysis of volatile components revealed the massive presence of trimethylamine and sulphur compounds. Moreover, 1,2,4-trithiolane, phenols, ketones, aldehydes, alcohols, esters and long chain aliphatic hydrocarbons were also detected. The data obtained allowed pro-technological bacteria, which are well-adapted to saline environments, to be discovered for the first time. Further analyses are needed to better clarify the extent of the contribution of either the microbiota or autolytic enzymes of the fish flesh in the aroma definition.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0740002020300927#bib60)

10.1016/j.fm.2020.103503

---

## Viable bacterial communities on hospital window components in patient rooms

[Source](https://peerj.com/articles/9580/)

---

## Microbiome and Metagenome Analyses of a Closed Habitat during Human Occupation.
 30.07.2020, _GB Malli Mohan, CW Parker, C Urbaniak, NK Singh, A Hood, JJ Minich, R Knight, M Rucker, K Venkateswaran_


Microbial contamination during long-term confinements of space exploration presents potential risks for both crew members and spacecraft life support systems. A novel swab kit was used to sample various surfaces from a submerged, closed, analog habitat to characterize the microbial populations. Samples were collected from various locations across the habitat which were constructed from various surface materials (linoleum, dry wall, particle board, glass, and metal), and microbial populations were examined by culture, quantitative PCR (qPCR), microbiome 16S rRNA gene sequencing, and shotgun metagenomics. Propidium monoazide (PMA)-treated samples identified the viable/intact microbial population of the habitat. The cultivable microbial population ranged from below the detection limit to 106 CFU/sample, and their identity was characterized using Sanger sequencing. Both 16S rRNA amplicon and shotgun sequencing were used to characterize the microbial dynamics, community profiles, and functional attributes (metabolism, virulence, and antimicrobial resistance). The 16S rRNA amplicon sequencing revealed abundance of viable (after PMA treatment) Actinobacteria (Brevibacterium, Nesternkonia, Mycobacterium, Pseudonocardia, and Corynebacterium), Firmicutes (Virgibacillus, Staphylococcus, and Oceanobacillus), and Proteobacteria (especially Acinetobacter) on linoleum, dry wall, and particle board (LDP) surfaces, while members of Firmicutes (Leuconostocaceae) and Proteobacteria (Enterobacteriaceae) were high on the glass/metal surfaces. Nonmetric multidimensional scaling determined from both 16S rRNA and metagenomic analyses revealed differential microbial species on LDP surfaces and glass/metal surfaces. The shotgun metagenomic sequencing of samples after PMA treatment showed bacterial predominance of viable Brevibacterium (53.6%), Brachybacterium (7.8%), Pseudonocardia (9.9%), Mycobacterium (3.7%), and Staphylococcus (2.1%), while fungal analyses revealed Aspergillus and Penicillium dominance.IMPORTANCE This study provides the first assessment of monitoring cultivable and viable microorganisms on surfaces within a submerged, closed, analog habitat. The results of the analyses presented herein suggest that the surface material plays a role in microbial community structure, as the microbial populations differed between LDP and metal/glass surfaces. The metal/glass surfaces had less-complex community, lower bioburden, and more closely resembled the controls. These results indicated that material choice is crucial when building closed habitats, even if they are simply analogs. Finally, while a few species were associated with previously cultivated isolates from the International Space Station and MIR spacecraft, the majority of the microbial ecology of the submerged analog habitat differs greatly from that of previously studied analog habitats.

[Source](https://msystems.asm.org/content/msys/5/4/e00367-20.full-text.pdf)

10.1128/mSystems.00367-20

---

## Viral Metagenomic Content Reflects Seawater Ecological Quality in the Coastal Zone.
 29.07.2020, _A Tsiola, G Michoud, S Fodelianakis, I Karakassis, G Kotoulas, A Pavlidou, C Pavloudi, P Pitta, N Simboura, D Daffonchio, M Tsapakis_


Viruses interfere with their host's metabolism through the expression of auxiliary metabolic genes (AMGs) that, until now, are mostly studied under large physicochemical gradients. Here, we focus on coastal marine ecosystems and we sequence the viral metagenome (virome) of samples with discrete levels of human-driven disturbances. We aim to describe the relevance of viromics with respect to ecological quality status, defined by the classic seawater trophic index (TRIX). Neither viral (family level) nor bacterial (family level, based on 16S rRNA sequencing) community structure correlated with TRIX. AMGs involved in the Calvin and tricarboxylic acid cycles were found at stations with poor ecological quality, supporting viral lysis by modifying the host's energy supply. AMGs involved in "non-traditional" energy-production pathways (3HP, sulfur oxidation) were found irrespective of ecological quality, highlighting the importance of recognizing the prevalent metabolic paths and their intermediate byproducts. Various AMGs explained the variability between stations with poor vs. good ecological quality. Our study confirms the pivotal role of the virome content in ecosystem functioning, acting as a "pool" of available functions that may be transferred to the hosts. Further, it suggests that AMGs could be used as an ultra-sensitive metric of energy-production pathways with relevance in the vulnerable coastal zone and its ecological quality.

[Source](https://www.mdpi.com/1999-4915/12/8/806/htm)

10.3390/v12080806

---

## Evaluating metagenomics tools for genome binning with real metagenomic datasets and CAMI datasets.
 30.07.2020, _Y Yue, H Huang, Z Qi, HM Dou, XY Liu, TF Han, Y Chen, XJ Song, YH Zhang, J Tu_


Shotgun metagenomics based on untargeted sequencing can explore the taxonomic profile and the function of unknown microorganisms in samples, and complement the shortage of amplicon sequencing. Binning assembled sequences into individual groups, which represent microbial genomes, is the key step and a major challenge in metagenomic research. Both supervised and unsupervised machine learning methods have been employed in binning. Genome binning belonging to unsupervised method clusters contigs into individual genome bins by machine learning methods without the assistance of any reference databases. So far a lot of genome binning tools have emerged. Evaluating these genome tools is of great significance to microbiological research. In this study, we evaluate 15 genome binning tools containing 12 original binning tools and 3 refining binning tools by comparing the performance of these tools on chicken gut metagenomic datasets and the first CAMI challenge datasets.

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03667-3)

10.1186/s12859-020-03667-3

---

## Untangling Species-Level Composition of Complex Bacterial Communities through a Novel Metagenomic Approach.
 30.07.2020, _C Milani, G Alessandri, M Mangifesta, L Mancabelli, GA Lugli, F Fontana, G Longhi, R Anzalone, A Viappiani, S Duranti, F Turroni, R Costi, A Annicchiarico, A Morini, L Sarli, MC Ossiprandi, D van Sinderen, M Ventura_


16S small-subunit (SSU) rRNA gene-based bacterial profiling is the gold standard for cost-effective taxonomic reconstruction of complex bacterial populations down to the genus level. However, it has been proven ineffective in clinical and research settings requiring higher taxonomic resolution. We therefore developed a bacterial profiling method based on the internal transcribed spacer (ITS) region employing optimized primers and a comprehensive ITS database for accurate cataloguing of bacterial communities at (sub)species resolution. Performance of the microbial ITS profiling pipeline was tested through analysis of host-associated, food, and environmental matrices, while its efficacy in clinical settings was assessed through analysis of mucosal biopsy specimens of colorectal cancer, leading to the identification of putative novel biomarkers. The data collected indicate that the proposed pipeline represents a major step forward in cost-effective identification and screening of microbial biomarkers at (sub)species level, with relevant impact in research, industrial, and clinical settings.IMPORTANCE We developed a novel method for accurate cataloguing of bacterial communities at (sub)species level involving amplification of the internal transcribed spacer (ITS) region through optimized primers, followed by next-generation sequencing and taxonomic classification of amplicons by means of a comprehensive database of bacterial ITS sequences. Host-associated, food, and environmental matrices were employed to test the performance of the microbial ITS profiling pipeline. Moreover, mucosal biopsy samples from colorectal cancer patients were analyzed to demonstrate the scientific relevance of this profiling approach in a clinical setting through identification of putative novel biomarkers. The results indicate that the ITS-based profiling pipeline proposed here represents a key metagenomic tool with major relevance for research, industrial, and clinical settings.

[Source](https://msystems.asm.org/content/msys/5/4/e00404-20.full.pdf)

10.1128/mSystems.00404-20

---

## Proteogenomics of rare taxonomic phyla: A prospective treasure trove of protein coding genes.
 30.12.2016, _D Kumar, AK Mondal, R Kutum, D Dash_


_Archaeal Proteins, Bacterial Proteins, Gene Transfer, Horizontal, Genome, Archaeal, Genome, Bacterial, Humans, Molecular Sequence Annotation, Open Reading Frames, Phylogeny, Proteome, Proteomics_
Sustainable innovations in sequencing technologies have resulted in a torrent of microbial genome sequencing projects. However, the prokaryotic genomes sequenced so far are unequally distributed along their phylogenetic tree; few phyla contain the majority, the rest only a few representatives. Accurate genome annotation lags far behind genome sequencing. While automated computational prediction, aided by comparative genomics, remains a popular choice for genome annotation, substantial fraction of these annotations are erroneous. Proteogenomics utilizes protein level experimental observations to annotate protein coding genes on a genome wide scale. Benefits of proteogenomics include discovery and correction of gene annotations regardless of their phylogenetic conservation. This not only allows detection of common, conserved proteins but also the discovery of protein products of rare genes that may be horizontally transferred or taxonomy specific. Chances of encountering such genes are more in rare phyla that comprise a small number of complete genome sequences. We collated all bacterial and archaeal proteogenomic studies carried out to date and reviewed them in the context of genome sequencing projects. Here, we present a comprehensive list of microbial proteogenomic studies, their taxonomic distribution, and also urge for targeted proteogenomics of underexplored taxa to build an extensive reference of protein coding genes.

[Source](https://www.biorxiv.org/content/10.1101/2020.07.27.224071v1.full.pdf)

10.1002/pmic.201500263

---

## Automated prediction and annotation of small proteins in microbial genomes

[Source](https://www.biorxiv.org/content/10.1101/2020.07.27.224071v1.full.pdf)

---

## Accurate and sensitive detection of microbial eukaryotes from metagenomic shotgun sequencing data

[Source](https://www.biorxiv.org/content/10.1101/2020.07.22.216580v1.full.pdf)

---

## MicrobeAnnotator: a user-friendly, comprehensive microbial genome annotation pipeline

[Source](https://www.biorxiv.org/content/10.1101/2020.07.20.211847v1.full.pdf)

---

## Topological and kernel-based microbial phenotype prediction from MALDI-TOF mass spectra.
 17.07.2020, _C Weis, M Horn, B Rieck, A Cu√©nod, A Egli, K Borgwardt_


Microbial species identification based on matrix-assisted laser desorption ionization time-of-flight (MALDI-TOF) mass spectrometry (MS) has become a standard tool in clinical microbiology. The resulting MALDI-TOF mass spectra also harbour the potential to deliver prediction results for other phenotypes, such as antibiotic resistance. However, the development of machine learning algorithms specifically tailored to MALDI-TOF MS-based phenotype prediction is still in its infancy. Moreover, current spectral pre-processing typically involves a parameter-heavy chain of operations without analyzing their influence on the prediction results. In addition, classification algorithms lack quantification of uncertainty, which is indispensable for predictions potentially influencing patient treatment.

[Source](https://academic.oup.com/bioinformatics/article/36/Supplement_1/i30/5870489)

10.1093/bioinformatics/btaa429

---

## NetCoMi: Network Construction and Comparison for Microbiome Data in R

[Source](https://www.biorxiv.org/content/10.1101/2020.07.15.195248v1.full.pdf)

---

## Metagenomic approaches in microbial ecology: an update on whole-genome and marker gene sequencing analyses.
 24.07.2020, _AE P√©rez-Cobas, L Gomez-Valero, C Buchrieser_


Metagenomics and marker gene approaches, coupled with high-throughput sequencing technologies, have revolutionized the field of microbial ecology. Metagenomics is a culture-independent method that allows the identification and characterization of organisms from all kinds of samples. Whole-genome shotgun sequencing analyses the total DNA of a chosen sample to determine the presence of micro-organisms from all domains of life and their genomic content. Importantly, the whole-genome shotgun sequencing approach reveals the genomic diversity present, but can also give insights into the functional potential of the micro-organisms identified. The marker gene approach is based on the sequencing of a specific gene region. It allows one to describe the microbial composition based on the taxonomic groups present in the sample. It is frequently used to analyse the biodiversity of microbial ecosystems. Despite its importance, the analysis of metagenomic sequencing and marker gene data is quite a challenge. Here we review the primary workflows and software used for both approaches and discuss the current challenges in the field.

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000409)

10.1099/mgen.0.000409

---

## Machine learning for drug development with Marinka Zitnik

[Source](https://bioinformatics.chat/drug-development)

---

