## Resistome expansion in disease-associated human gut microbiomes.
 31.07.2023, _S Fredriksen, S de Warle, P van Baarlen, J Boekhorst, JM Wells_


_Humans, Gastrointestinal Microbiome, Microbiota, Anti-Bacterial Agents, Drug Resistance, Microbial, Genes, Bacterial_
The resistome, the collection of antibiotic resistance genes (ARGs) in a microbiome, is increasingly recognised as relevant to the development of clinically relevant antibiotic resistance. Many metagenomic studies have reported resistome differences between groups, often in connection with disease and/or antibiotic treatment. However, the consistency of resistome associations with antibiotic- and non-antibiotic-treated diseases has not been established. In this study, we re-analysed human gut microbiome data from 26 case-control studies to assess the link between disease and the resistome.

[Source](https://doi.org/10.1186/s40168-023-01610-1)

10.1186/s40168-023-01610-1

---

## A microbial causal mediation analytic tool for health disparity and applications in body mass index

[Source](https://doi.org/10.1186/s40168-023-01608-9)

---

## Alterations of gut microbiota in biopsy-proven diabetic nephropathy and a long history of diabetes without kidney damage

[Source](https://doi.org/10.1038/s41598-023-39444-4)

---

## Assessment of infant outgrowth of cow’s milk allergy in relation to the faecal microbiome and metaproteome

[Source](https://doi.org/10.1038/s41598-023-39260-w)

---

## Analysis of bacterial vaginosis, the vaginal microbiome, and sexually transmitted infections following the provision of menstrual cups in Kenyan schools: Results of a nested study within a cluster randomized controlled trial

[Source](https://doi.org/10.1371/journal.pmed.1004258)

---

## Evolutionary modeling suggests that addictions may be driven by competition-induced microbiome dysbiosis

[Source](https://doi.org/10.1038/s42003-023-05099-0)

---

## Alterations in the gut microbiome implicate key taxa and metabolic pathways across inflammatory arthritis phenotypes

[Source](https://doi.org/10.1126/scitranslmed.abn4722)

---

## Increased genital mucosal cytokines in Canadian women associate with higher antigen-presenting cells, inflammatory metabolites, epithelial barrier disruption, and the depletion of <em>L. crispatus</em>

[Source](https://doi.org/10.1186/s40168-023-01594-y)

---

## Insulin alleviates murine colitis through microbiome alterations and bile acid metabolism

[Source](https://doi.org/10.1186/s12967-023-04214-3)

---

## The Vaginal Microbiota of Pregnant Women Varies with Gestational Age, Maternal Age, and Parity

[Source](https://doi.org/10.1128/spectrum.03429-22)

---

## Unique microbial landscape in the human oropharynx during different types of acute respiratory tract infections

[Source](https://doi.org/10.1186/s40168-023-01597-9)

---

## Probiotics to prevent necrotising enterocolitis in very preterm or very low birth weight infants

[Source](https://doi.org/10.1002/14651858.CD005496.pub6)

---

## A gut feeling for drugs that have metabolic benefits

[Source](https://doi.org/10.1038/s41467-023-40167-3)

---

## Antibiotic perturbations to the gut microbiome

[Source](https://doi.org/10.1038/s41579-023-00933-y)

---

## The global human gut microbiome: genes, lifestyles, and diet.
 29.07.2023, _M Parizadeh, MC Arrieta_


A growing number of human gut microbiome studies consistently describe differences between human populations. Here, we review how factors related to host genetics, ethnicity, lifestyle, and geographic location help explain this variation. Studies from contrasting environmental scenarios point to diet and lifestyle as the most influential. The effect of human migration and displacement demonstrates how the microbiome adapts to newly adopted lifestyles and contributes to the profound biological and health consequences attributed to migration. This information strongly suggests against a universal scale for healthy or dysbiotic gut microbiomes, and prompts for additional microbiome population surveys, particularly from less industrialized nations. Considering these important differences will be critical for designing strategies to diagnose and restore dysbiosis in various human populations.

[Source](https://doi.org/10.1016/j.molmed.2023.07.002)

10.1016/j.molmed.2023.07.002

---

## 5-Aminosalicylic acid alters the gut microbiota and altered microbiota transmitted vertically to offspring have protective effects against colitis.
 31.07.2023, _H Wada, J Miyoshi, S Kuronuma, Y Nishinarita, N Oguri, N Hibi, O Takeuchi, Y Akimoto, STM Lee, M Matsuura, T Kobayashi, T Hibi, T Hisamatsu_


_Humans, Animals, Mice, Mesalamine, Gastrointestinal Microbiome, Colitis, Colitis, Ulcerative, Inflammation, Inflammatory Bowel Diseases, Anti-Inflammatory Agents, Dextran Sulfate, Disease Models, Animal, Colon, Mice, Inbred C57BL_
Although many therapeutic options are available for inflammatory bowel disease (IBD), 5-aminosalicylic acid (5-ASA) is still the key medication, particularly for ulcerative colitis (UC). However, the mechanism of action of 5-ASA remains unclear. The intestinal microbiota plays an important role in the pathophysiology of IBD, and we hypothesized that 5-ASA alters the intestinal microbiota, which promotes the anti-inflammatory effect of 5-ASA. Because intestinal inflammation affects the gut microbiota and 5-ASA can change the severity of inflammation, assessing the impact of inflammation and 5-ASA on the gut microbiota is not feasible in a clinical study of patients with UC. Therefore, we undertook a translational study to demonstrate a causal link between 5-ASA administration and alterations of the intestinal microbiota. Furthermore, by rigorously controlling environmental confounders and excluding the effect of 5-ASA itself with a vertical transmission model, we observed that the gut microbiota altered by 5-ASA affected host mucosal immunity and decreased susceptibility to dextran sulfate sodium-induce colitis. Although the potential intergenerational transmission of epigenetic changes needs to be considered in this study, these findings suggested that alterations in the intestinal microbiota induced by 5-ASA directed the host immune system towards an anti-inflammatory state, which underlies the mechanism of 5-ASA efficacy.

[Source](https://doi.org/10.1038/s41598-023-39491-x)

10.1038/s41598-023-39491-x

---

## Presence of an ultra-small microbiome in fermented cabbages.
 25.07.2023, _HW Lee, SR Yoon, YM Dang, M Kang, K Lee, JH Ha, JW Bae_


_Humans, Brassica, RNA, Ribosomal, 16S, Microbiota, Vegetables, Bacteria, DNA_
Ultramicrobacteria (UMB), also known as ultra-small bacteria, are tiny bacteria with a size less than 0.1 µm3. They have a high surface-to-volume ratio and are found in various ecosystems, including the human body. UMB can be classified into two types: one formed through cell contraction and the other that maintains a small size. The ultra-small microbiome (USM), which may contain UMB, includes all bacteria less than 0.2 µm in size and is difficult to detect with current methods. However, it poses a potential threat to food hygiene, as it can pass through sterilization filters and exist in a viable but non-culturable (VBNC) state. The data on the USM of foods is limited. Some bacteria, including pathogenic species, are capable of forming UMB under harsh conditions, making it difficult to detect them through conventional culture techniques.

[Source](https://doi.org/10.7717/peerj.15680)

10.7717/peerj.15680

---

## The long-term gut bacterial signature of a wild primate is associated with a timing effect of pre- and postnatal maternal glucocorticoid levels.
 31.07.2023, _S Anzà, D Schneider, R Daniel, M Heistermann, S Sangmaneedet, J Ostner, O Schülke_


_Female, Animals, Pregnancy, Humans, Glucocorticoids, Gastrointestinal Microbiome, Cross-Sectional Studies, Primates, Bacteria, Prenatal Exposure Delayed Effects_
During development, elevated levels of maternal glucocorticoids (GCs) can have detrimental effects on offspring morphology, cognition, and behavior as well as physiology and metabolism. Depending on the timing of exposure, such effects may vary in strength or even reverse in direction, may alleviate with age, or may concern more stable and long-term programming of phenotypic traits. Maternal effects on gut bacterial diversity, composition, and function, and the persistence of such effects into adulthood of long-lived model species in the natural habitats remain underexplored.

[Source](https://doi.org/10.1186/s40168-023-01596-w)

10.1186/s40168-023-01596-w

---

## Early-life prophylactic antibiotic treatment disturbs the stability of the gut microbiota and increases susceptibility to H9N2 AIV in chicks.
 30.07.2023, _X Liang, Z Zhang, H Wang, X Lu, W Li, H Lu, A Roy, X Shen, DM Irwin, Y Shen_


_Animals, Anti-Bacterial Agents, Gastrointestinal Microbiome, Influenza A Virus, H9N2 Subtype, Chickens, Microbiota_
Antibiotics are widely used for prophylactic therapy and for improving the growth performance of chicken. The problem of bacterial drug resistance caused by antibiotic abuse has previously attracted extensive attention; however, the influence of early-day use of prophylactic antibiotics on the gut microflora and on the disease resistance ability in chicks has not been explored. Here, we comprehensively evaluate the growth performance, gut microbial dynamics, level of antibiotic resistance genes (ARGs) in the gut microbial community, and resistance to H9N2 avian influenza virus (AIV) in chickens following long-term and short-term early-day prophylactic antibiotic treatment.

[Source](https://doi.org/10.1186/s40168-023-01609-8)

10.1186/s40168-023-01609-8

---

## Human-influenced diets affect the gut microbiome of wild baboons.
 26.07.2023, _M Moy, L Diakiw, KR Amato_


_Animals, Humans, Gastrointestinal Microbiome, Papio, RNA, Ribosomal, 16S, Diet, Primates, Hominidae, Papio anubis, Monosaccharides_
Industrialized diets that incorporate processed foods and are often high in simple sugars and fats and low in fiber have myriad health impacts, many of which may operate via impacts on the gut microbiota. Examining how these diets affect the gut microbiota can be challenging given that lab animal models experience altered environmental contexts, and human studies include a suite of co-varying cultural and environmental factors that are likely to shape the gut microbiota alongside diet. To complement these approaches, we compare the microbiomes of wild populations of olive baboons (Papio anubis) with differential access to human trash high in processed foods, simple sugars, and fats in Rwanda's Akagera National Park. Baboons are a good model system since their microbiomes are compositionally similar to those of humans. Additionally, this population inhabits a common environment with different social groups consuming qualitatively different amounts of human trash, limiting variation in non-dietary factors. Using 16S rRNA gene amplicon sequencing we find that baboons with unlimited access to human trash have reduced microbial alpha diversity and reduced relative abundances of fiber-degrading taxa such as Ruminococcaceae, Prevotellaceae, and Lachnospiraceae. In contrast, baboons with limited access to human trash have a microbiome more similar to that of baboons with no access to human trash. Our results suggest that while a human-influenced diet high in processed foods, simple sugars, and fats is sufficient to alter the microbiome in wild baboons, there is a minimum threshold of dietary alteration that must occur before the microbiome is substantially altered. We recommend that data from wild primate populations such as these be used to complement ongoing research on diet-microbiome-health interactions in humans and lab animal models.

[Source](https://doi.org/10.1038/s41598-023-38895-z)

10.1038/s41598-023-38895-z

---

## Expanding the range of the respiratory infectome in Australian feedlot cattle with and without respiratory disease using metatranscriptomics.
 30.07.2023, _BP Brito, MJ Frost, K Anantanawat, F Jaya, T Batterham, SP Djordjevic, WS Chang, EC Holmes, AE Darling, PD Kirkland_


_Animals, Cattle, Rhinitis, Australia, Respiratory Tract Diseases, Viruses, Cattle Diseases_
Bovine respiratory disease (BRD) is one of the most common diseases in intensively managed cattle, often resulting in high morbidity and mortality. Although several pathogens have been isolated and extensively studied, the complete infectome of the respiratory complex consists of a more extensive range unrecognised species. Here, we used total RNA sequencing (i.e., metatranscriptomics) of nasal and nasopharyngeal swabs collected from animals with and without BRD from two cattle feedlots in Australia.

[Source](https://doi.org/10.1186/s40168-023-01591-1)

10.1186/s40168-023-01591-1

---

## Social network position is a major predictor of ant behavior, microbiota composition, and brain gene expression

[Source](https://doi.org/10.1371/journal.pbio.3002203)

---

## The root microbiome: techniques for exploration and agricultural applications.
 31.07.2023, _A Cannon_


_Agriculture, Microbiota_
Standfirst With an ever-growing global population, the pursuit of sustainable agriculture has become paramount. What if the solution lies right beneath our feet? Enter the root microbiome, the hidden hero poised to revolutionize agriculture and bring us closer to a greener future. [Formula: see text].

[Source](https://doi.org/10.2144/btn-2023-0057)

10.2144/btn-2023-0057

---

## Recently-formed Antarctic lakes host less diverse benthic bacterial and diatom communities than their older counterparts.
 29.07.2023, _J Kollár, K Kopalová, J Kavan, K Vrbická, D Nývlt, L Nedbalová, M Stibal, TJ Kohler_


Glacier recession is creating new waterbodies in proglacial forelands worldwide, including Antarctica. Yet, it is unknown how microbial communities of recently formed 'young' waterbodies (originating decades to a few centuries ago) compare with established 'old' counterparts (millennia ago). Here, we compared benthic microbial communities of different lake types on James Ross Island, Antarctic Peninsula, using 16S rDNA metabarcoding and light microscopy to explore bacterial and diatom communities, respectively. We found that the older lakes host significantly more diverse bacterial and diatom communities compared to the young ones. To identify potential mechanisms for these differences, linear models and dbRDA analyses suggested combinations of water temperature, pH, and conductivity to be the most important factors for diversity and community structuring, while differences in geomorphological and hydrological stability, though more difficult to quantify, are likely also influential. These results, along with an indicator species analysis, suggest that physical and chemical constraints associated with individual lakes histories are likely more influential to the assembly of the benthic microbial communities than lake age alone. Collectively, these results improve our understanding of microbial community drivers in Antarctic freshwaters, and help predict how the microbial landscape may shift with future habitat creation within a changing environment.

[Source](https://doi.org/10.1093/femsec/fiad087)

10.1093/femsec/fiad087

---

## Greengenes2 unifies microbial data in a single reference tree

[Source](https://doi.org/10.1038/s41587-023-01845-1)

---

## CheckM2: a rapid, scalable and accurate tool for assessing microbial genome quality using machine learning.
 27.07.2023, _A Chklovski, DH Parks, BJ Woodcroft, GW Tyson_


Advances in sequencing technologies and bioinformatics tools have dramatically increased the recovery rate of microbial genomes from metagenomic data. Assessing the quality of metagenome-assembled genomes (MAGs) is a critical step before downstream analysis. Here, we present CheckM2, an improved method of predicting genome quality of MAGs using machine learning. Using synthetic and experimental data, we demonstrate that CheckM2 outperforms existing tools in both accuracy and computational speed. In addition, CheckM2's database can be rapidly updated with new high-quality reference genomes, including taxa represented only by a single genome. We also show that CheckM2 accurately predicts genome quality for MAGs from novel lineages, even for those with reduced genome size (for example, Patescibacteria and the DPANN superphylum). CheckM2 provides accurate genome quality predictions across bacterial and archaeal lineages, giving increased confidence when inferring biological conclusions from MAGs.

[Source](https://doi.org/10.1038/s41592-023-01940-w)

10.1038/s41592-023-01940-w

---

## Lectin-Seq: A method to profile lectin-microbe interactions in native communities.
 31.07.2023, _RL McPherson, CR Isabella, RL Walker, D Sergio, S Bae, T Gaca, S Raman, LTT Nguyen, DA Wesener, M Halim, MG Wuo, A Dugan, R Kerby, S Ghosh, FE Rey, C Dhennezel, G Pishchany, V Lensch, H Vlamakis, EJ Alm, RJ Xavier, LL Kiessling_


_Humans, Lectins, Immunity, Innate_
Soluble human lectins are critical components of innate immunity. Genetic models suggest that lectins influence host-resident microbiota, but their specificity for commensal and mutualist species is understudied. Elucidating lectins' roles in regulating microbiota requires an understanding of which microbial species they bind within native communities. To profile human lectin recognition, we developed Lectin-Seq. We apply Lectin-Seq to human fecal microbiota using the soluble mannose-binding lectin (MBL) and intelectin-1 (hItln1). Although each lectin binds a substantial percentage of the samples (10 to 20%), the microbial interactomes of MBL and hItln1 differ markedly in composition and diversity. MBL binding is highly selective for a small subset of species commonly associated with humans. In contrast, hItln1's interaction profile encompasses a broad range of lower-abundance species. Our data uncover stark differences in the commensal recognition properties of human lectins.

[Source](https://doi.org/10.1126/sciadv.add8766)

10.1126/sciadv.add8766

---

## A mixed-model approach for estimating drivers of microbiota community composition and differential taxonomic abundance.
 25.07.2023, _AR Sweeny, H Lemon, A Ibrahim, KA Watt, K Wilson, DZ Childs, DH Nussey, A Free, L McNally_


Next-generation sequencing (NGS) and metabarcoding approaches are increasingly applied to wild animal populations, but there is a disconnect between the widely applied generalized linear mixed model (GLMM) approaches commonly used to study phenotypic variation and the statistical toolkit from community ecology typically applied to metabarcoding data. Here, we describe the suitability of a novel GLMM-based approach for analyzing the taxon-specific sequence read counts derived from standard metabarcoding data. This approach allows decomposition of the contribution of different drivers to variation in community composition (e.g., age, season, individual) via interaction terms in the model random-effects structure. We provide guidance to implementing this approach and show how these models can identify how responsible specific taxonomic groups are for the effects attributed to different drivers. We applied this approach to two cross-sectional data sets from the Soay sheep population of St. Kilda. GLMMs showed agreement with dissimilarity-based approaches highlighting the substantial contribution of age and minimal contribution of season to microbiota community compositions, and simultaneously estimated the contribution of other technical and biological factors. We further used model predictions to show that age effects were principally due to increases in taxa of the phylum Bacteroidetes and declines in taxa of the phylum Firmicutes. This approach offers a powerful means for understanding the influence of drivers of community structure derived from metabarcoding data. We discuss how our approach could be readily adapted to allow researchers to estimate contributions of additional factors such as host or microbe phylogeny to answer emerging questions surrounding the ecological and evolutionary roles of within-host communities. IMPORTANCE NGS and fecal metabarcoding methods have provided powerful opportunities to study the wild gut microbiome. A wealth of data is, therefore, amassing across wild systems, generating the need for analytical approaches that can appropriately investigate simultaneous factors at the host and environmental scale that determine the composition of these communities. Here, we describe a generalized linear mixed-effects model (GLMM) approach to analyze read count data from metabarcoding of the gut microbiota, allowing us to quantify the contributions of multiple host and environmental factors to within-host community structure. Our approach provides outputs that are familiar to a majority of field ecologists and can be run using any standard mixed-effects modeling packages. We illustrate this approach using two metabarcoding data sets from the Soay sheep population of St. Kilda investigating age and season effects as worked examples.

[Source](https://doi.org/10.1128/msystems.00040-23)

10.1128/msystems.00040-23

---

