## Large-scale association analyses identify host factors influencing human gut microbiome composition.
 19.01.2021, _A Kurilshikov, C Medina-Gomez, R Bacigalupe, D Radjabzadeh, J Wang, A Demirkan, CI Le Roy, JA Raygoza Garay, CT Finnicum, X Liu, DV Zhernakova, MJ Bonder, TH Hansen, F Frost, MC Rühlemann, W Turpin, JY Moon, HN Kim, K Lüll, E Barkan, SA Shah, M Fornage, J Szopinska-Tokov, ZD Wallen, D Borisevich, L Agreus, A Andreasson, C Bang, L Bedrani, JT Bell, H Bisgaard, M Boehnke, DI Boomsma, RD Burk, A Claringbould, K Croitoru, GE Davies, CM van Duijn, L Duijts, G Falony, J Fu, A van der Graaf, T Hansen, G Homuth, DA Hughes, RG Ijzerman, MA Jackson, VWV Jaddoe, M Joossens, T Jørgensen, D Keszthelyi, R Knight, M Laakso, M Laudes, LJ Launer, W Lieb, AJ Lusis, AAM Masclee, HA Moll, Z Mujagic, Q Qibin, D Rothschild, H Shin, SJ Sørensen, CJ Steves, J Thorsen, NJ Timpson, RY Tito, S Vieira-Silva, U Völker, H Völzke, U Võsa, KH Wade, S Walter, K Watanabe, S Weiss, FU Weiss, O Weissbrod, HJ Westra, G Willemsen, H Payami, DMAE Jonkers, A Arias Vasquez, EJC de Geus, KA Meyer, J Stokholm, E Segal, E Org, C Wijmenga, HL Kim, RC Kaplan, TD Spector, AG Uitterlinden, F Rivadeneira, A Franke, MM Lerch, L Franke, S Sanna, M D'Amato, O Pedersen, AD Paterson, R Kraaij, J Raes, A Zhernakova_


To study the effect of host genetics on gut microbiome composition, the MiBioGen consortium curated and analyzed genome-wide genotypes and 16S fecal microbiome data from 18,340 individuals (24 cohorts). Microbial composition showed high variability across cohorts: only 9 of 410 genera were detected in more than 95% of samples. A genome-wide association study of host genetic variation regarding microbial taxa identified 31 loci affecting the microbiome at a genome-wide significant (P &lt; 5 × 10-8) threshold. One locus, the lactase (LCT) gene locus, reached study-wide significance (genome-wide association study signal: P = 1.28 × 10-20), and it showed an age-dependent association with Bifidobacterium abundance. Other associations were suggestive (1.95 × 10-10 &lt; P &lt; 5 × 10-8) but enriched for taxa showing high heritability and for genes expressed in the intestine and brain. A phenome-wide association study and Mendelian randomization identified enrichment of microbiome trait loci in the metabolic, nutrition and environment domains and suggested the microbiome might have causal effects in ulcerative colitis and rheumatoid arthritis.

[Source](https://www.nature.com/articles/s41588-020-00763-1)

10.1038/s41588-020-00763-1

---

## Genome-wide association study in 8,956 German individuals identifies influence of ABO histo-blood groups on gut microbiome.
 19.01.2021, _MC Rühlemann, BM Hermes, C Bang, S Doms, L Moitinho-Silva, LB Thingholm, F Frost, F Degenhardt, M Wittig, J Kässens, FU Weiss, A Peters, K Neuhaus, U Völker, H Völzke, G Homuth, S Weiss, H Grallert, M Laudes, W Lieb, D Haller, MM Lerch, JF Baines, A Franke_


The intestinal microbiome is implicated as an important modulating factor in multiple inflammatory1,2, neurologic3 and neoplastic diseases4. Recent genome-wide association studies yielded inconsistent, underpowered and rarely replicated results such that the role of human host genetics as a contributing factor to microbiome assembly and structure remains uncertain5-11. Nevertheless, twin studies clearly suggest host genetics as a driver of microbiome composition11. In a genome-wide association analysis of 8,956 German individuals, we identified 38 genetic loci to be associated with single bacteria and overall microbiome composition. Further analyses confirm the identified associations of ABO histo-blood groups and FUT2 secretor status with Bacteroides and Faecalibacterium spp. Mendelian randomization analysis suggests causative and protective effects of gut microbes, with clade-specific effects on inflammatory bowel disease. This holistic investigative approach of the host, its genetics and its associated microbial communities as a 'metaorganism' broaden our understanding of disease etiology, and emphasize the potential for implementing microbiota in disease treatment and management.

[Source](https://www.nature.com/articles/s41588-020-00747-1)

10.1038/s41588-020-00747-1

---

## Fecal microbiome and metabolome differ in healthy and food-allergic twins.
 19.01.2021, _R Bao, LA Hesser, Z He, X Zhou, KC Nadeau, CR Nagler_


BACKGROUNDThere has been a striking generational increase in the prevalence of food allergies. We have proposed that this increase can be explained, in part, by alterations in the commensal microbiome.METHODSTo identify bacterial signatures and metabolic pathways that may influence the expression of this disease, we collected fecal samples from a unique, well-controlled cohort of twins concordant or discordant for food allergy. Samples were analyzed by integrating 16S rRNA gene amplicon sequencing and liquid chromatography-tandem mass spectrometry metabolite profiling.RESULTSA bacterial signature of 64 operational taxonomic units (OTUs) distinguished healthy from allergic twins; the OTUs enriched in the healthy twins were largely taxa from the Clostridia class. We detected significant enrichment in distinct metabolite pathways in each group. The enrichment of diacylglycerol in healthy twins is of particular interest for its potential as a readily measurable fecal biomarker of health. In addition, an integrated microbial-metabolomic analysis identified a significant association between healthy twins and Phascolarctobacterium faecium and Ruminococcus bromii, suggesting new possibilities for the development of live microbiome-modulating biotherapeutics.CONCLUSIONTwin pairs exhibited significant differences in their fecal microbiomes and metabolomes through adulthood, suggesting that the gut microbiota may play a protective role in patients with food allergies beyond the infant stage.TRIAL REGISTRATIONParticipants in this study were recruited as part of an observational study (ClinicalTrials.gov NCT01613885) at multiple sites from 2014 to 2018.FUNDINGThis work was supported by the Sunshine Charitable Foundation; the Moss Family Foundation; the National Institute of Allergy and Infectious Diseases (NIAID) (R56AI134923 and R01AI 140134); the Sean N. Parker Center for Allergy and Asthma Research; the National Heart, Lung, and Blood Institute (R01 HL 118612); the Orsak family; the Kepner family; and the Stanford Institute for Immunity, Transplant and Infection.

[Source](https://www.jci.org/articles/view/141935)

10.1172/JCI141935

---

## Hepatic steatosis relates to gastrointestinal microbiota changes in obese girls with polycystic ovary syndrome.
 19.01.2021, _B Jobira, DN Frank, LJ Silveira, L Pyle, MM Kelsey, Y Garcia-Reyes, CE Robertson, D Ir, KJ Nadeau, M Cree-Green_


Hepatic steatosis (HS) is common in adolescents with obesity and polycystic ovary syndrome (PCOS). Gut microbiota are altered in adults with obesity, HS, and PCOS, which may worsen metabolic outcomes, but similar data is lacking in youth.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0245219)

10.1371/journal.pone.0245219

---

## Microbiota modulates gut immunity and promotes baculovirus infection in Helicoverpa armigera.
 19.01.2021, _C Yuan, L Xing, M Wang, Z Hu, Z Zou_


Baculoviruses are natural enemies of agricultural and forest insect pests and play an important role in biological pest control. Oral infection by baculovirus in the insect midgut is necessary for establishing systemic infection and eventually killing the insect. Since the insect midgut continuously encounters microbiota, the gut microbiota could affect baculovirus infection. Here, we demonstrated that gut microbiota modulates immune responses and promotes baculovirus infection in the cotton bollworm, Helicoverpa armigera. After oral infection, numerous host immunity-related genes including genes encoding Toll and immune deficiency (IMD) pathway components were upregulated in the midgut. Elimination of the gut microbiota significantly increased the resistance to viral infection in H. armigera. Quantitative real-time reverse transcription polymerase chain reaction and proteomic analysis showed that downregulation of the antiviral factor prophenoloxidase (PPO) could be mediated by microbiota during infection. It implied that midgut microbiota diminishes the expression of PPO to facilitate viral infection in H. armigera. Our findings revealed that the microbiota plays an important role in modulating the resistance of H. armigera to baculovirus infection, providing new insights in applying biopesticide.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/1744-7917.12894)

10.1111/1744-7917.12894

---

## Immunological design of commensal communities to treat intestinal infection and inflammation.
 19.01.2021, _RL Brown, MLY Larkinson, TB Clarke_


The immunological impact of individual commensal species within the microbiota is poorly understood limiting the use of commensals to treat disease. Here, we systematically profile the immunological fingerprint of commensals from the major phyla in the human intestine (Actinobacteria, Bacteroidetes, Firmicutes and Proteobacteria) to reveal taxonomic patterns in immune activation and use this information to rationally design commensal communities to enhance antibacterial defenses and combat intestinal inflammation. We reveal that Bacteroidetes and Firmicutes have distinct effects on intestinal immunity by differentially inducing primary and secondary response genes. Within these phyla, the immunostimulatory capacity of commensals from the Bacteroidia class (Bacteroidetes phyla) reflects their robustness of TLR4 activation and Bacteroidia communities rely solely on this receptor for their effects on intestinal immunity. By contrast, within the Clostridia class (Firmicutes phyla) it reflects the degree of TLR2 and TLR4 activation, and communities of Clostridia signal via both of these receptors to exert their effects on intestinal immunity. By analyzing the receptors, intracellular signaling components and transcription factors that are engaged by different commensal species, we identify canonical NF-κB signaling as a critical rheostat which grades the degree of immune stimulation commensals elicit. Guided by this immunological analysis, we constructed a cross-phylum consortium of commensals (Bacteroides uniformis, Bacteroides ovatus, Peptostreptococcus anaerobius and Clostridium histolyticum) which enhances innate TLR, IL6 and macrophages-dependent defenses against intestinal colonization by vancomycin resistant Enterococci, and fortifies mucosal barrier function during pathological intestinal inflammation through the same pathway. Critically, the setpoint of intestinal immunity established by this consortium is calibrated by canonical NF-κB signaling. Thus, by profiling the immunological impact of major human commensal species our work paves the way for rational microbiota reengineering to protect against antibiotic resistant infections and to treat intestinal inflammation.

[Source](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1009191)

10.1371/journal.ppat.1009191

---

## Ticks Resist Skin Commensals with Immune Factor of Bacterial Origin.
 15.01.2021, _BM Hayes, AD Radkov, F Yarza, S Flores, J Kim, Z Zhao, KW Lexa, L Marnin, J Biboy, V Bowcut, W Vollmer, JHF Pedra, S Chou_


Ticks transmit a diverse array of microbes to vertebrate hosts, including human pathogens, which has led to a human-centric focus in this vector system. Far less is known about pathogens of ticks themselves. Here, we discover that a toxin in blacklegged ticks (Ixodes scapularis) horizontally acquired from bacteria-called domesticated amidase effector 2 (dae2)-has evolved to kill mammalian skin microbes with remarkable efficiency. Secreted into the saliva and gut of ticks, Dae2 limits skin-associated staphylococci in ticks while feeding. In contrast, Dae2 has no intrinsic ability to kill Borrelia burgdorferi, the tick-borne Lyme disease bacterial pathogen. These findings suggest ticks resist their own pathogens while tolerating symbionts. Thus, just as tick symbionts can be pathogenic to humans, mammalian commensals can be harmful to ticks. Our study underscores how virulence is context-dependent and bolsters the idea that "pathogen" is a status and not an identity.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0092867420314495)

10.1016/j.cell.2020.10.042

---

## The effect of taxonomic classification by full-length 16S rRNA sequencing with a synthetic long-read technology.
 19.01.2021, _J Jeong, K Yun, S Mun, WH Chung, SY Choi, YD Nam, MY Lim, CP Hong, C Park, Y Ahn, K Han_


Characterizing the microbial communities inhabiting specimens is one of the primary objectives of microbiome studies. A short-read sequencing platform for reading partial regions of the 16S rRNA gene is most commonly used by reducing the cost burden of next-generation sequencing (NGS), but misclassification at the species level due to its length being too short to consider sequence similarity remains a challenge. Loop Genomics recently proposed a new 16S full-length-based synthetic long-read sequencing technology (sFL16S). We compared a 16S full-length-based synthetic long-read (sFL16S) and V3-V4 short-read (V3V4) methods using 24 human GUT microbiota samples. Our comparison analyses of sFL16S and V3V4 sequencing data showed that they were highly similar at all classification resolutions except the species level. At the species level, we confirmed that sFL16S showed better resolutions than V3V4 in analyses of alpha-diversity, relative abundance frequency and identification accuracy. Furthermore, we demonstrated that sFL16S could overcome the microbial misidentification caused by different sequence similarity in each 16S variable region through comparison the identification accuracy of Bifidobacterium, Bacteroides, and Alistipes strains classified from both methods. Therefore, this study suggests that the new sFL16S method is a suitable tool to overcome the weakness of the V3V4 method.

[Source](https://www.nature.com/articles/s41598-020-80826-9)

10.1038/s41598-020-80826-9

---

