## The oral microbiome and lung cancer risk: An analysis of 3 prospective cohort studies.
 05.08.2022, _E Vogtmann, X Hua, G Yu, V Purandare, AG Hullings, D Shao, Y Wan, S Li, CL Dagnall, K Jones, BD Hicks, A Hutchinson, JG Caporaso, W Wheeler, DP Sandler, LEB Freeman, LM Liao, WY Huang, ND Freedman, NE Caporaso, R Sinha, MH Gail, J Shi, CC Abnet_


Previous studies suggested associations between the oral microbiome and lung cancer, but studies were predominantly cross-sectional and underpowered.

[Source](https://doi.org/10.1093/jnci/djac149)

10.1093/jnci/djac149

---

## Antibiotic resistance genes in the gut microbiota of mothers and linked neonates with or without sepsis from low- and middle-income countries

[Source](https://doi.org/10.1038/s41564-022-01184-y)

---

## Crosstalk between mucosal microbiota, host gene expression, and sociomedical factors in the progression of colorectal cancer

[Source](https://doi.org/10.1038/s41598-022-17823-7)

---

## Using Culture-Enriched Phenotypic Metagenomics for Targeted High-Throughput Monitoring of the Clinically Important Fraction of the β-Lactam Resistome

[Source](https://doi.org/10.1021/acs.est.2c03627)

---

## <em>Porphyromonas gingivalis</em> and dental stem cells crosstalk amplify inflammation and bone loss in the periodontitis niche

[Source](https://doi.org/10.1002/jcp.30848)

---

## On the effect of flavonoids and dietary fibre in lingonberries on atherosclerotic plaques, lipid profiles and gut microbiota composition in <em>Apoe(-/-)</em> mice

[Source](https://doi.org/10.1080/09637486.2022.2106358)

---

## The Gut-Brain-Microbiome Connection: Can Probiotics Decrease Anxiety and Depression?

[Source](https://doi.org/10.1080/01612840.2022.2106525)

---

## The effect of <em>Lactobacillus rhamnosus</em> B10 on alcoholic liver injury and intestinal microbiota in alcohol-induced mice model

[Source](https://doi.org/10.1111/jfbc.14372)

---

## The effect of young blood plasma administration on gut microbiota in middle-aged rats

[Source](https://doi.org/10.1007/s00203-022-03154-8)

---

## Quantitative microbial population study reveals geographical differences in bacterial symbionts of <em>Ixodes ricinus</em>

[Source](https://doi.org/10.1186/s40168-022-01276-1)

---

## Host taxonomy determines the composition, structure, and diversity of the earthworm cast microbiome under homogenous feeding conditions.
 04.08.2022, _M Aira, M Pérez-Losada, KA Crandall, J Domínguez_


Host evolutionary history is a key factor shaping the earthworm cast microbiome, although its effect can be shadowed by the earthworm's diet. To untangle dietary from taxon effects, we raised nine earthworm species on a uniform diet of cow manure and compared cast microbiome across species while controlling for diet. Our results showed that, under controlled laboratory conditions, earthworm microbiomes are species-specific, more diverse than that of the controlled diet, and mainly comprised of native bacteria (i.e., not acquired from the diet). Furthermore, diet has a medium to large convergence effect on microbiome composition since earthworms shared 16 to 74% of their bacterial amplicon sequence variants (ASV). The inter-species core microbiome included 10 ASVs, while their intra-species core microbiomes were larger and varied in ASV richness (24-48%) and sequence abundance across earthworm species. This specificity in core microbiomes and variable degree of similarity in bacterial composition suggest that phylosymbiosis could determine earthworm microbiome assembly. However, lack of congruence between the earthworm phylogeny and the microbiome dendrogram suggests that a consistent diet fed over several generations may have weakened potential phylosymbiotic effects. Thus, cast microbiome assembly in earthworms seem to be the result of an interplay among host phylogeny and diet.

[Source](https://doi.org/10.1093/femsec/fiac093)

10.1093/femsec/fiac093

---

## Ruminal Microbiota Determines the High-Fiber Utilization of Ruminants: Evidence from the Ruminal Microbiota Transplant.
 04.08.2022, _X Chen, F Yan, T Liu, Y Zhang, X Li, M Wang, C Zhang, X Xu, L Deng, J Yao, S Wu_


The rumen, which contains a series of prokaryotes and eukaryotes with high abundance, determines the high ability to degrade complex carbohydrates in ruminants. Using 16S rRNA gene sequencing, we compared the ruminal microbiota of dairy goats with that in the foregut and colon of mice and found more Bacteroides identified in the rumen, which helps ruminants to utilize plant-derived polysaccharides, cellulose, and other structural carbohydrates. Furthermore, high-fiber diets did not significantly increase intestinal fiber-degrading bacteria in mice, but did produce higher levels of ruminal fiber-degrading bacteria in dairy goats. Through rumen microbe transplantation (RMT), we found that rumen-derived fiber-degrading bacteria can colonize the intestines of mice to exert their fiber-degrading function, but their colonization efficiency is affected by diet. Additionally, the colonization of these fiber-degrading bacteria in the colon may involve higher content of butyrate in the colon, protecting the colonic epithelial barrier and promoting energy metabolism. Overall, the fiber degradation function of rumen bacteria through RMT was verified, and our results provide new insights into isolating the functional and beneficial fiber-degrading bacteria in the rumen, providing a theoretical basis for the role of dietary fiber in intestinal health. IMPORTANCE Ruminants have a powerful progastric digestive system that converts structural carbohydrates into nutrients useful to humans. It is well known that this phenomenon is due to the fact that the rumen of ruminants is a natural microbial fermenter, which can ferment structural carbohydrates such as cellulose and hemicellulose and transform them into volatile fatty acids to supply energy for host. However, monogastric animals have an inherent disadvantage in utilizing fiber, so screening rumen-derived fiber-degrading bacteria as a fermentation strain for biological feed is needed in an attempt at improving the fiber digestibility of monogastric animals. In this study, a ruminal microbiota transplant experiment from goats to mice proves that ruminal microbiota could serve as a key factor in utilization of high-fiber diets and provides a new perspective for the development of probiotics with fiber degradation function from the rumen and the importance of the use of prebiotics during the intake of probiotics.

[Source](https://doi.org/10.1128/spectrum.00446-22)

10.1128/spectrum.00446-22

---

## Computational approach to modeling microbiome landscapes associated with chronic human disease progression.
 04.08.2022, _L Li, J Sohn, RJ Genco, J Wactawski-Wende, S Goodison, PI Diaz, Y Sun_


A microbial community is a dynamic system undergoing constant change in response to internal and external stimuli. These changes can have significant implications for human health. However, due to the difficulty in obtaining longitudinal samples, the study of the dynamic relationship between the microbiome and human health remains a challenge. Here, we introduce a novel computational strategy that uses massive cross-sectional sample data to model microbiome landscapes associated with chronic disease development. The strategy is based on the rationale that each static sample provides a snapshot of the disease process, and if the number of samples is sufficiently large, the footprints of individual samples populate progression trajectories, which enables us to recover disease progression paths along a microbiome landscape by using computational approaches. To demonstrate the validity of the proposed strategy, we developed a bioinformatics pipeline and applied it to a gut microbiome dataset available from a Crohn's disease study. Our analysis resulted in one of the first working models of microbial progression for Crohn's disease. We performed a series of interrogations to validate the constructed model. Our analysis suggested that the model recapitulated the longitudinal progression of microbial dysbiosis during the known clinical trajectory of Crohn's disease. By overcoming restrictions associated with complex longitudinal sampling, the proposed strategy can provide valuable insights into the role of the microbiome in the pathogenesis of chronic disease and facilitate the shift of the field from descriptive research to mechanistic studies.

[Source](https://doi.org/10.1371/journal.pcbi.1010373)

10.1371/journal.pcbi.1010373

---

## Guideline for the analysis of the microbial communities of the human upper airways.
 05.08.2022, _L Mancabelli, T Ciociola, GA Lugli, C Tarracchini, F Fontanta, A Viappiani, F Turroni, A Ticinesi, T Meschi, S Conti, M Ventura, C Milani_


The recent COVID-19 pandemic prompted a rapid-growing interest in the investigation of the human microbiota of the upper airways. In fact, the resident microbial community of this body district may have an influence on the onset of SARS-CoV-2 infection and its clinical course in terms of presence, symptom severity, and outcomes. However, several microbiological methodologies are available to study the human microbiota, reflecting the extensive fragmentation of methodological approaches. We investigate the impact of two critical steps that can induce biases in the downstream analyses, i.e. sampling method and microbial DNA extraction kit employed. We observed major discrepancies regarding the total amount of prokaryotic DNA that could be retrieved from a biological sample and the proportion between bacterial DNA and human host DNA. Moreover, shotgun DNA sequencing and taxonomic profile reconstruction also revealed correlations between sampling methods and the procedures applied for microbial DNA extraction. Based on all the data collected in this study, we formulate indications regarding the most efficient and reliable methodological procedures for the metagenomic analyses of the upper airways' microbiota to maximize accuracy and reproducibility.

[Source](https://doi.org/10.1080/20002297.2022.2103282)

10.1080/20002297.2022.2103282

---

