## Analysis of Immune, Microbiota and Metabolome Maturation in Infants in a Clinical Trial of Lactobacillus Paracasei CBA L74-fermented Formula. 

[Source](https://doi.org/10.1038/s41467-020-16582-1)

---

## Ecological succession in the vaginal microbiota during pregnancy and birth.
 03.06.2020, _MA Rasmussen, J Thorsen, MG Dominguez-Bello, MJ Blaser, MS Mortensen, AD Brejnrod, SA Shah, MH Hjelmsø, J Lehtimäki, U Trivedi, H Bisgaard, SJ Sørensen, J Stokholm_


The mother's vaginal microbiota represents the first microbes to which a child is exposed when delivered vaginally. However, little is known about the composition and development of the vaginal microbiota during pregnancy and birth. Here, we analyzed the vaginal microbiota of 57 women in pregnancy week 24, 36 and at birth after rupture of membranes but before delivery, and further compared the composition with that of the gut and airways of the 1-week-old child. The vaginal community structure had dramatic changes in bacterial diversity and taxonomic distribution, yet carried an individual-specific signature. The relative abundance of most bacterial taxa increased stepwise from week 24 of pregnancy until birth, with a gradual decline of Lactobacillus. Mother-to-child vertical transfer, as suggested by sharing, was modest, with the strongest transfer being for Clostridiales followed by Lactobacillales and Enterobacteriales. In conclusion, late gestation is associated with an increase in maternal vaginal microbiota diversity, and vaginal bacteria at birth only modestly predict the composition of the neonatal microbiota.

[Source](https://doi.org/10.1038/s41396-020-0686-3)

10.1038/s41396-020-0686-3

---

## Ecological Succession in the Vaginal Microbiota During Pregnancy and Birth.

[Source](https://doi.org/10.1038/s41396-020-0686-3)

---

## Physicians perspectives about medical sources of information: protocol for an overview of systematic reviews.
 16.06.2020, _G Urrea, N Carvajal-Juliá, C Arcos, J Pérez-Bracchiglione_


Despite the growing availability of evidence and sources of information, it is not clear what are the physicians preferences for filling gaps in their medical knowledge.

[Source](https://doi.org/10.1093/ajcn/nqaa076)

10.5867/medwave.2020.05.7921

---

## Neonatal Diet Alters Fecal Microbiota and Metabolome Profiles at Different Ages in Infants Fed Breast Milk or Formula.

[Source](https://doi.org/10.1093/ajcn/nqaa076)

---

## Human milk fungi: environmental determinants and inter-kingdom associations with milk bacteria in the CHILD Cohort Study.
 09.06.2020, _S Moossavi, K Fehr, H Derakhshani, H Sbihi, B Robertson, L Bode, J Brook, SE Turvey, TJ Moraes, AB Becker, PJ Mandhane, MR Sears, E Khafipour, P Subbarao, MB Azad_


Fungi constitute an important yet frequently neglected component of the human microbiota with a possible role in health and disease. Fungi and bacteria colonise the infant gastrointestinal tract in parallel, yet most infant microbiome studies have ignored fungi. Milk is a source of diverse and viable bacteria, but few studies have assessed the diversity of fungi in human milk.

[Source](https://doi.org/10.1186/s12866-020-01829-0)

10.1186/s12866-020-01829-0

---

## Human Milk Fungi: Environmental Determinants and Inter-Kingdom Associations With Milk Bacteria in the CHILD Cohort Study.

[Source](https://doi.org/10.1186/s12866-020-01829-0)

---

## Profiles of Human Milk Oligosaccharides and Their Relations to the Milk Microbiota of Breastfeeding Mothers in Dubai.
 12.06.2020, _C Ayoub Moubareck, M Lootah, M Tahlak, K Venema_


The composition of human breast milk is affected by several factors, including genetics, geographic location and maternal nutrition. This study investigated the human milk oligosaccharides (HMOs) of breastfeeding mothers living in Dubai and their relations with the milk microbiota. A total of 30 breast milk samples were collected from healthy Emirati and UAE-expatriates at Latifa Hospital. HMO profiling was performed using UHPLC-MS. Microbiota profiles were determined by sequencing amplicons of the V3-V4 region of the 16S rRNA gene. HMO concentrations were significantly higher in Emirati, and dropped with the lactation period in both groups of mothers. The Le (a-b+)-secretor (Le+Se+) type was the most abundant in Dubai mothers (60%), followed by the Le(a-b-)-secretor (Le-Se+) type (23%). Bifidobacterium and Lactobacillus were considerably lower in Dubai-based mothers, while Pseudomonas and Delftia (Hydrogenophaga) were detected at a higher abundance compared to mothers from other countries. Atopobium was correlated with sialyl-lacto-N-tetraose c, Leptotrichia and Veillonella were correlated with 6'-sialyl-lactose, and Porphyromonas was correlated with lacto-N-hexaose. The study highlights the HMO profiles of breastfeeding mothers in Dubai and reveals few correlations with milk microbial composition. Targeted genomic analyses may help in determining whether these differences are due to genetic variations or to sociocultural and environmental factors.

[Source](https://www.mdpi.com/2072-6643/12/6/1727)

10.3390/nu12061727

---

## Human milk microbiota development during lactation and its relation to maternal geographic location and gestational hypertensive status.
 16.06.2020, _Y Wan, J Jiang, M Lu, W Tong, R Zhou, J Li, J Yuan, F Wang, D Li_


Bacteria in human milk could directly seed the infant intestinal microbiota, while information about how milk microbiota develops during lactation and how geographic location, gestational hypertensive status, and maternal age influence this process is limited. Here, we collected human milk samples from mothers of term infants at the first day, 2 weeks, and 6 weeks postpartum from 117 longitudinally followed-up mothers (age: 28.7 ± 3.6 y) recruited from three cities in China. We found that milk microbial diversity and richness were the highest in colostrum but gradually decreased over lactation. Microbial composition changed across lactation and exhibited more discrete compositional patterns in 2-week and 6-week milk samples compared with colostrum samples. At phylum level, the abundance of Proteobacteria increased during lactation, while Firmicutes showed the opposite trend. At genus level, Staphylococcus, Streptococcus, Acinetobacter, Pseudomonas, and Lactobacillus were predominant in colostrum samples and showed distinct variations across lactation. Maternal geographic location was significantly associated with the milk microbiota development and the abundance of predominant genus. In addition, milk from mothers with gestational prehypertension had a different and less diverse microbial community at genus level in early lactation times, and contained less Lactobacillus in the 2-week milk samples than those from normotensive mothers. Findings of our study outlined the human milk microbial diversity and community development over lactation, and underscored the importance of maternal geographic locations and gestational hypertensive status on milk microbiota, which might have important implications in the establishment of the infant intestinal microbiota via breastfeeding.

[Source](https://doi.org/10.1080/19490976.2020.1760711)

10.1080/19490976.2020.1760711

---

## Human Milk Microbiota Development During Lactation and Its Relation to Maternal Geographic Location and Gestational Hypertensive Status.

[Source](https://doi.org/10.1080/19490976.2020.1760711)

---

## Skin microbiota analysis-inspired development of novel anti-infectives.
 10.06.2020, _Y Liu, Y Liu, Z Du, L Zhang, J Chen, Z Shen, Q Liu, J Qin, H Lv, H Wang, L He, J Liu, Q Huang, Y Sun, M Otto, M Li_


The alarming spread of antimicrobial resistance requires the development of novel anti-infective drugs. Despite the recent research focus on the human microbiome and its likely value to understand and exploit inter-bacterial inhibitory phenomena as a source for antimicrobial strategies, the human microbiota has barely been investigated for the purpose of drug development.

[Source](https://doi.org/10.1186/s40168-020-00866-1)

10.1186/s40168-020-00866-1

---

## Skin Microbiota Analysis-Inspired Development of Novel Anti-Infectives.

[Source](https://doi.org/10.1186/s40168-020-00866-1)

---

## Skin Microbiome Analysis for Forensic Human Identification: What Do We Know So Far?
 12.06.2020, _P Tozzo, G D'Angiolella, P Brun, I Castagliuolo, S Gino, L Caenazzo_


Microbiome research is a highly transdisciplinary field with a wide range of applications and methods for studying it, involving different computational approaches and models. The fact that different people host radically different microbiota highlights forensic perspectives in understanding what leads to this variation and what regulates it, in order to effectively use microbes as forensic evidence. This narrative review provides an overview of some of the main scientific works so far produced, focusing on the potentiality of using skin microbiome profiling for human identification in forensics. This review was performed following the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines. The examined literature clearly ascertains that skin microbial communities, although personalized, vary systematically across body sites and time, with intrapersonal differences over time smaller than interpersonal ones, showing such a high degree of spatial and temporal variability that the degree and nature of this variability can constitute in itself an important parameter useful in distinguishing individuals from one another. Even making the effort to organically synthesize all results achieved until now, it is quite evident that these results are still the pieces of a puzzle, which is not yet complete.

[Source](https://doi.org/10.3390/microorganisms8060873)

10.3390/microorganisms8060873

---

## Protecting the outside: biological tools to manipulate the skin microbiota.
 28.05.2020, _JN O'Sullivan, MC Rea, C Hill, RP Ross_


Interest surrounding the role that skin microbes play in various aspects of human health has recently experienced a timely surge, particularly among researchers, clinicians and consumer-focused industries. The world is now approaching a post-antibiotic era where conventional antibacterial therapeutics have shown a loss in effectiveness due to overuse, leading to the looming antibiotic resistance crisis. The increasing threat posed by antibiotic resistance is compounded by an inadequate discovery rate of new antibiotics and has, in turn, resulted in global interest for alternative solutions. Recent studies have demonstrated that imbalances in skin microbiota are associated with assorted skin diseases and infections. Specifically, restoration of this ecosystem imbalance results in an alleviation of symptoms, achieved simply by applying bacteria normally found in abundance on healthy skin to the skin of those deficient in beneficial bacteria. The aim of this review is to discuss the currently available literature on biological tools that have the potential to manipulate the skin microbiota, with particular focus on bacteriocins, phage therapy, antibiotics, probiotics and targets of the gut-skin axis. This review will also address how the skin microbiota protects humans from invading pathogens in the external environment while discussing novel strategies to manipulate the skin microbiota to avoid and/or treat various disease states.

[Source](https://doi.org/10.1093/femsec/fiaa085)

10.1093/femsec/fiaa085

---

## Protecting the Outside: Biological Tools to Manipulate the Skin Microbiota.

[Source](https://doi.org/10.1093/femsec/fiaa085)

---

## Recovery of Vaginal Microbiota After Standard Treatment for Bacterial Vaginosis Infection: An Observational Study.
 12.06.2020, _L Lehtoranta, AA Hibberd, J Reimari, J Junnila, N Yeung, J Maukonen, G Crawford, AC Ouwehand_


Vaginal microbiota dysbiosis and bacterial vaginosis (BV) affect negatively women's health. Understanding vaginal microbiota fluctuations in BV during and after antibiotic treatment would facilitate accurate decision-making on the treatment regimen, avoid unnecessary antibiotic use, and potentially mitigate recurrence. We investigated vaginal microbiota composition of 30 women with BV before and after 5-day metronidazole treatment and compared the results with 30 healthy women. Vaginal microbiota was assessed by Nugent score and analyzed by 16S rRNA gene sequencing in swabs on baseline Day 1, and on Day 8 and 15, after completion of antibiotic treatment by women with BV. Prior to antibiotic treatment (Day 1), BV-positive women were dominated by Lactobacillusiners (25.8%), Prevotellatimonensis/bivia (18.0%), and Gardnerella vaginalis (14.6%), whereas healthy women were dominated by L.iners (37.5%) and Lactobacilluscrispatus/acidophilus (19.2%). On Day 8, L.iners abundance increased in BV-treated women being significantly higher compared with healthy women (67.8% vs. 37.5%, p = 0.049). On Day 15, the relative abundance of all microbial taxa was similar between the groups. Vaginal microbiota of women with BV shifted to resemble that of healthy controls after metronidazole. Sequencing analysis provides more in-depth understanding of changes in vaginal microbiota. The role of L.iners in vaginal health and dysbiosis requires further investigations.

[Source](https://doi.org/10.3390/microorganisms8060875)

10.3390/microorganisms8060875

---

## Alterations in Vaginal Microbiota and Associated Metabolome in Women with Recurrent Implantation Failure.
 09.06.2020, _M Fu, X Zhang, Y Liang, S Lin, W Qian, S Fan_


Recurrent implantation failure (RIF) refers to repeated failure to become pregnant after transferring embryos with normal morphology. However, the pathogenesis of RIF remains unrevealed, especially for those without any pathological features. In this study, we characterized the vaginal microbiota and metabolomes of patients with unexplained RIF, while patients who achieved clinical pregnancy in the first frozen embryo transfer (FET) cycle were used as controls. Based on 16S rRNA gene sequencing of the vaginal microbiota, the vaginal Lactobacillus showed a significant positive correlation with the pregnancy rate, and the RIF group presented higher microbial α-diversity than the control group (P value = 0.016). The metabolomic profile identified 2,507 metabolites, of which 37 were significantly different between the two groups (P value &lt; 0.05, variable importance for the projection [VIP] &gt; 1). Among them, 2',3-cyclic UMP and inositol phosphate were the top two metabolites that were higher in the RIF group, while glycerophospholipids and benzopyran were important metabolites that were lower in the RIF group. A lack of lysobisphosphatidic acid and prostaglandin metabolized from glycerophospholipids will lead to deferred implantation and embryo crowding. Benzopyran, as a selective estrogen receptor modulator, may affect the outcome of pregnancy. All of the changes in metabolite profiles may result in or from the differential microbiota compositions in RIF patients. In conclusion, significant differences were presented in the vaginal microbiota and metabolomes between patients with unexplained RIF and women who became pregnant in the first FET cycle. For the first time, this study elaborates the possible pathogenesis of RIF by investigating the vaginal microbiota and metabolites in RIF patients.IMPORTANCEIn vitro fertilization-embryo transfer (IVF-ET) is now widely applied for treating infertility, and unexplained recurrent implantation failure (RIF) has become a substantial challenge. We hypothesize that vaginal microbial dysbiosis is associated with RIF, as it is linked to many female reproductive diseases. In this study, we characterized the vaginal microbiota and metabolomes of patients with unexplained RIF, while patients who achieved clinical pregnancy in the first IVF cycle were set as controls. In general, significant differences were discovered in the vaginal microbiota and metabolomes between the two groups. This study is the first detailed elaboration of the vaginal microbiota and metabolites associated with RIF. We believe that our findings will inspire researchers to consider the dynamics of microbiomes related to the microenvironment as a critical feature for future studies of nosogenesis not only for RIF but also for other reproductive diseases.

[Source](https://mbio.asm.org/content/11/3/e03242-19)

10.1128/mBio.03242-19

---

## Dynamic interactions of influenza viruses in Hong Kong during 1998-2018.
 16.06.2020, _W Yang, EHY Lau, BJ Cowling_


Influenza epidemics cause substantial morbidity and mortality every year worldwide. Currently, two influenza A subtypes, A(H1N1) and A(H3N2), and type B viruses co-circulate in humans and infection with one type/subtype could provide cross-protection against the others. However, it remains unclear how such ecologic competition via cross-immunity and antigenic mutations that allow immune escape impact influenza epidemic dynamics at the population level. Here we develop a comprehensive model-inference system and apply it to study the evolutionary and epidemiological dynamics of the three influenza types/subtypes in Hong Kong, a city of global public health significance for influenza epidemic and pandemic control. Utilizing long-term influenza surveillance data since 1998, we are able to estimate the strength of cross-immunity between each virus-pairs, the timing and frequency of punctuated changes in population immunity in response to antigenic mutations in influenza viruses, and key epidemiological parameters over the last 20 years including the 2009 pandemic. We find evidence of cross-immunity in all types/subtypes, with strongest cross-immunity from A(H1N1) against A(H3N2). Our results also suggest that A(H3N2) may undergo antigenic mutations in both summers and winters and thus monitoring the virus in both seasons may be important for vaccine development. Overall, our study reveals intricate epidemiological interactions and underscores the importance of simultaneous monitoring of population immunity, incidence rates, and viral genetic and antigenic changes.

[Source](https://doi.org/10.1093/cid/ciaa709)

10.1371/journal.pcbi.1007989

---

## Alterations of the Gut Microbiota in Patients With COVID-19 or H1N1 Influenza.

[Source](https://doi.org/10.1093/cid/ciaa709)

---

## Alterations in the Gut Microbiome and Suppression of Histone Deacetylases by Resveratrol Are Associated with Attenuation of Colonic Inflammation and Protection Against Colorectal Cancer.
 12.06.2020, _HR Alrafas, PB Busbee, KN Chitrala, M Nagarkatti, P Nagarkatti_


Inflammatory bowel disease (IBD) is known to significantly increase the risk for development of colorectal cancer (CRC), suggesting inflammation and cancer development are closely intertwined. Thus, agents that suppress inflammation may prevent the onset of cancer. In the current study, we used resveratrol, an anti-inflammatory stilbenoid, to study the role of microbiota in preventing inflammation-driven CRC. Resveratrol treatment in the azoxymethane (AOM) and dextran sodium sulphate (DSS) CRC murine model caused an increase in anti-inflammatory CD4 + FOXP3 + (Tregs) and CD4 + IL10 + cells, a decrease in proinflammatory Th1 and Th17 cells, and attenuated CRC development. Gut microbial profile studies demonstrated that resveratrol altered the gut microbiome and short chain fatty acid (SCFA), with modest increases in n-butyric acid and a potential butyrate precursor isobutyric acid. Fecal transfer from resveratrol-treated CRC mice and butyrate supplementation resulted in attenuation of disease and suppression of the inflammatory T cell response. Data also revealed both resveratrol and sodium butyrate (BUT) were capable of inhibiting histone deacetylases (HDACs), correlating with Treg induction. Analysis of The Cancer Genome Atlas (TCGA) datasets revealed increased expression of Treg-specific transcription factor FoxP3 or anti-inflammatory IL-10 resulted in an increase in 5-year survival of patients with CRC. These data suggest that alterations in the gut microbiome lead to an anti-inflammatory T cell response, leading to attenuation of inflammation-driven CRC.

[Source](https://doi.org/10.3390/jcm9061796)

10.3390/jcm9061796

---

## Long-term dietary intervention reveals resilience of the gut microbiota despite changes in diet and weight.
 14.06.2020, _GK Fragiadakis, HC Wastyk, JL Robinson, ED Sonnenburg, JL Sonnenburg, CD Gardner_


With the rising rates of obesity and associated metabolic disorders, there is a growing need for effective long-term weight-loss strategies, coupled with an understanding of how they interface with human physiology. Interest is growing in the potential role of gut microbes as they pertain to responses to different weight-loss diets; however, the ways that diet, the gut microbiota, and long-term weight loss influence one another is not well understood.

[Source](https://doi.org/10.1093/ajcn/nqaa046)

10.1093/ajcn/nqaa046

---

## Long-term Dietary Intervention Reveals Resilience of the Gut Microbiota Despite Changes in Diet and Weight.

[Source](https://doi.org/10.1093/ajcn/nqaa046)

---

## Intraoral Microbial Metabolism and Association with Host Taste Perception.
 21.05.2020, _A Gardner, PW So, GH Carpenter_


Metabolomics has been identified as a means of functionally assessing the net biological activity of a particular microbial community. Considering the oral microbiome, such an approach remains largely underused. While the current knowledge of the oral microbiome is constantly expanding, there are several deficits in knowledge particularly relating to their interactions with their host. This work uses nuclear magnetic resonance spectroscopy to investigate metabolic differences between oral microbial metabolism of endogenous (i.e., salivary protein) and exogenous (i.e., dietary carbohydrates) substrates. It also investigated whether microbial generation of different metabolites may be associated with host taste perception. This work found that in the absence of exogenous substrate, oral bacteria readily catabolize salivary protein and generate metabolic profiles similar to those seen in vivo. Important metabolites such as acetate, butyrate, and propionate are generated at relatively high concentrations. Higher concentrations of metabolites were generated by tongue biofilm compared to planktonic salivary bacteria. Thus, as has been postulated, metabolite production in proximity to taste receptors could reach relatively high concentrations. In the presence of 0.25 M exogenous sucrose, increased catabolism was observed with increased concentrations of a range of metabolites relating to glycolysis (lactate, pyruvate, succinate). Additional pyruvate-derived molecules such as acetoin and alanine were also increased. Furthermore, there was evidence that individual taste sensitivity to sucrose was related to differences in the metabolic fate of sucrose in the mouth. High-sensitivity perceivers appeared more inclined toward continual citric acid cycle activity postsucrose, whereas low-sensitivity perceivers had a more efficient conversion of pyruvate to lactate. This work collectively indicates that the oral microbiome exists in a complex balance with the host, with fluctuating metabolic activity depending on nutrient availability. There is preliminary evidence of an association between host behavior (sweet taste perception) and oral catabolism of sugar.

[Source](https://doi.org/10.1177/0022034520917142)

10.1177/0022034520917142

---

## Combined Prebiotic and Microbial Intervention Improves Oral Cholera Vaccination Responses in a Mouse Model of Childhood Undernutrition.

[Source](https://doi.org/10.1016/j.chom.2020.04.008)

---

## Commensal Bacteria Modulate Immunoglobulin A Binding in Response to Host Nutrition.
 16.06.2020, _KE Huus, KC Bauer, EM Brown, T Bozorgmehr, SE Woodward, A Serapio-Palacios, RCT Boutin, C Petersen, BB Finlay_


Immunoglobulin (Ig) A controls host-microbial homeostasis in the gut. IgA recognition of beneficial bacteria is decreased in acutely undernourished children, but the factors driving these changes in IgA targeting are unknown. Child undernutrition is a global health challenge that is exacerbated by poor sanitation and intestinal inflammation. To understand how nutrition impacts immune-microbe interactions, we used a mouse model of undernutrition with or without fecal-oral exposure and assessed IgA-bacterial targeting from weaning to adulthood. In contrast to healthy control mice, undernourished mice fail to develop IgA recognition of intestinal Lactobacillus. Glycan-mediated interactions between Lactobacillus and host antibodies are lost in undernourished mice due to rapid bacterial adaptation. Lactobacillus adaptations occur in direct response to nutritional pressure, independently of host IgA, and are associated with reduced mucosal colonization and with bacterial mutations in carbohydrate processing genes. Together these data indicate that diet-driven bacterial adaptations shape IgA recognition in the gut.

[Source](https://doi.org/10.1016/j.chom.2020.03.012)

10.1016/j.chom.2020.03.012

---

## The interplay between the gut microbiota and NLRP3 activation affects the severity of acute pancreatitis in mice.
 12.06.2020, _X Li, C He, N Li, L Ding, H Chen, J Wan, X Yang, L Xia, W He, H Xiong, X Shu, Y Zhu, N Lu_


Early dysbiosis of the gut microbiota is associated with the severity of acute pancreatitis (AP), although the underlying mechanism is unclear. Here, we investigated the role of crosstalk between NLRP3 and the gut microbiota in the development of AP utilizing gut microbiota deficient mice, as well as NLRP3 knockout (KO) mouse models. Pancreatic damage and systemic inflammation were improved in antibiotic-treated (Abx) and germ-free (GF) mice, accompanied by weakened activity of the intestinal NLRP3 inflammasome. Interestingly, fecal microbiota transplantation (FMT) reactivated the intestinal NLRP3 inflammasome and exacerbated the disease in Abx and GF mice. Although the gut barrier in GF and Abx mice was disrupted, gut microbiota deficiency ameliorated the severity of AP, probably due to the reduction in bacterial translocation from the gut to the pancreas. The composition of the gut microbiota was significantly different between NLRP3 KO mice and wild-type (WT) mice at baseline, and there were alterations in response to the induction of AP. While a dramatic shift in the gut microbiota with overgrowth of Escherichia-Shigella was observed in WT mice suffering from AP, there was no significant change in NLRP3 KO mice with or without AP, suggesting that NLRP3 deficiency counteracts AP-induced microbial disturbance. With a strengthened gut barrier and decreased systemic inflammation, NLRP3 KO mice showed less severe AP, as revealed by reduced pancreatic neutrophilic infiltration and necrosis. Taken together, these results identified the bidirectional modulation between the gut microbiota and NLRP3 in the progression of AP, which suggests the interplay of the host and microbiome during AP.

[Source](https://doi.org/10.1080/19490976.2020.1770042)

10.1080/19490976.2020.1770042

---

## Structural and functional changes of gut microbiota in ovariectomized rats and their correlations with altered bone mass.
 02.06.2020, _S Ma, J Qin, Y Hao, Y Shi, L Fu_


As a critical factor involved in the maintenance of physiological homeostasis, the gut microbiota (GM) reportedly plays a key role in bone development. To date, the association between the GM and steroid deficiency-induced osteoporosis remains poorly understood. Forty female Sprague Dawley rats were divided into an ovariectomy (OVX) or control group. We performed 16S rRNA and metagenome sequencing, to compare diversity, taxonomic differences, and functional genes. The GM composition did not change in the control group and the number of operational taxonomic units increased significantly following ovariectomy. Alpha diversity, determined by ACE estimator, CHAO estimator, the Shannon index, and the Simpson index showed an increasing trend after ovariectomy. Samples in the OVX group were well clustered both pre- and post-ovariectomy, as demonstrated by principal coordinate 1 (PC1) and PC2. Functional genes of GM, including those involved in synthesis and metabolism of carbohydrates and nucleotides, microbial structure, and heme, as well as hemin uptake and utilization, increased at the early stage of osteoporosis. We observed that Ruminococcus flavefaciens exhibited the greatest variation in abundance among the GM and this was also associated with osteoclastic indicators and the estrobolome. Specific changes in fecal microbiota are associated with the pathogenesis of steroid deficiency-induced osteoporosis.

[Source](https://doi.org/10.18632/aging.103290)

10.18632/aging.103290

---

## Comparative assessment of faecal microbial composition and metabonome of swine, farmers and human control.
 06.06.2020, _SC Tan, CW Chong, IKS Yap, KL Thong, CSJ Teh_


The gastrointestinal tract of humans and swine consist of a wide range of bacteria which interact with hosts metabolism. Due to the differences in co-evolution and co-adaptation, a large fraction of the gut microbiome is host-specific. In this study, we evaluated the effect of close human-animal interaction to the faecal metagenome and metabonome of swine, farmer and human control. Three distinct clusters were observed based on T-RFLP-derived faecal microbial composition. However, 16S-inferred faecal microbiota and metabolic profiles showed that only human control was significantly different from the swine (P &lt; 0.05). The metabonome of farmers and human controls were highly similar. Notably, higher trimethylamine N-oxide (TMAO) and butyrate were detected in human control and swine, respectively. The relative abundance of TMAO was positively correlated with Prevotella copri. Overall, we compared and established the relationship between the metabolites and microbiota composition of swine, farmers and human control. Based on the data obtained, we deduced that long term occupational exposure to swine and farm environment had affected the gut bacterial composition of farmers. Nonetheless, the effect was less prominent in the metabolite profiles, suggesting the gut bacteria expressed high functional plasticity and are therefore resilience to the level of community shift detected.

[Source](https://doi.org/10.1038/s41598-020-65891-4)

10.1038/s41598-020-65891-4

---

## Staphylococcus nepalensis, a commensal of the oral microbiota of domestic cats, is a reservoir of transferrable antimicrobial resistance.
 10.06.2020, _AL Andrade-Oliveira, CC Rossi, T Souza-Silva, M Giambiagi-deMarval_


Staphylococcus nepalensis is a commensal bacterium from the oral microbiota of domestic cats, with a still obscure clinical importance. In this work, we analysed the ability of feline strains of S. nepalensis to transfer antimicrobial resistance genes to Staphylococcus aureus isolated from humans through plasmids. To this end, we first analysed all publicly available genomes from cat staphylococci using computational methods to build a pan-resistome. Genes that encode resistance to erythromycin, gentamicin, mupirocin and tetracycline, common to human and cat staphylococci and previously described to be located in mobile genetic elements, were chosen for the next analyses. We studied 15 strains of S. nepalensis, which were shown to be genetically different by GTG5-PCR. As observed by disc diffusion, resistance to tetracycline was widespread (80 %), followed by resistance to erythromycin (40 %), gentamicin (27 %) and mupirocin (7 %). The strains were positive for several antimicrobial resistance genes and more than half of them harboured plasmids. The loss of plasmids and resistance genes in some strains were induced by stress with SDS. Through conjugation experiments, we observed that these plasmids can be transferred to S. aureus, thus increasing its potential to resist drug therapy. Our findings show that S. nepalensis, an underestimated inhabitant of the cat microbiota, can be a reservoir of antimicrobial resistance genes for S. aureus and, like many other staphylococci, be an overlooked and silent threat to their animal hosts and humans living with them.

[Source](https://doi.org/10.1099/mic.0.000940)

10.1099/mic.0.000940

---

## Staphylococcus nepalensis, a Commensal of the Oral Microbiota of Domestic Cats, Is a Reservoir of Transferrable Antimicrobial Resistance.

[Source](https://doi.org/10.1099/mic.0.000940)

---

## Increasing the Hindgut Carbohydrate/Protein Ratio by Cecal Infusion of Corn Starch or Casein Hydrolysate Drives Gut Microbiota-Related Bile Acid Metabolism To Stimulate Colonic Barrier Function.
 03.06.2020, _Y Pi, C Mu, K Gao, Z Liu, Y Peng, W Zhu_


Dietary high protein and low carbohydrate levels compromise colonic microbiota and bile acid metabolism, which underlies a detrimental gut environment. However, it remains unclear if the diet-induced changes in colonic health are due to a change in hindgut nutrient availability and what key intermediates link the microbe-epithelium dialogue. To specifically alter the hindgut nutrient substrate availability, here we used a cecally cannulated pig model to infuse corn starch and casein hydrolysate directly into the cecum to generate a stepwise change of carbohydrate/nitrogenous compound (C/N) ratio. Pigs were cecally infused daily with either saline (Control), corn starch (Starch), or casein hydrolysate (Casein) (n = 8 per group), respectively, for 19 days. After infusion, C/N ratios in colonic digesta were 16.33, 12.56, and 8.54 for the starch, control, and casein groups, respectively (P &lt; 0.05). Relative to the control group, casein infusion showed greater abundance of the bacteria (Eubacterium) capable of bile acid 7α-dehydroxylation (baiJ), higher levels of expression of bacterial genes encoding the baiJ enzyme, and higher levels of secondary bile acid (deoxycholic acid [DCA] and lithocholic acid [LCA]), while the starch infusion showed the opposite effect. Correspondingly, casein infusion downregulated expression of genes encoding tight junction proteins (ZO-1 and OCLD) and upregulated expression of genes encoding epidermal growth factor receptor (EGFR). The ratio of C/N was linearly related with the concentrations of DCA and LCA and gene expression levels of ZO-1, occludin, and EGFR. Caco-2 cell experiments further showed that DCA and LCA downregulated expression of genes involved in barrier function (ZO-1 and OCLD) and upregulated the gene expression of EGFR and Src. Inhibition of EGFR and Src could abolish DCA- and LCA-induced downregulation of ZO-1, indicating that DCA and LCA impair gut barrier function via enhancing the EGFR-Src pathway. These results suggest that the ratio of C/N in the large intestine is an important determinant of microbial metabolism and gut barrier function in the colon. The findings provide evidence that microbe-related secondary bile acid metabolism may mediate the interplay between microbes and gut barrier function.IMPORTANCE High-fiber or high-protein diets could alter gut microbiota and health in the large intestine, but factors involved in the effects remain unclear. The present study for the first time demonstrates that the starch- and casein-induced C/N ratio in the hindgut is an important factor. Using the cannulated pig model, we found that the distinct C/N ratio induced by cecal infusion of corn starch or casein hydrolysate was linearly correlated with microbial metabolites (secondary bile acids) and tight junction proteins (ZO-1 and OCLD). Cell culture study further demonstrates that the gut microbial metabolites (DCA and LCA) could impair the intestinal barrier function via the EGFR-Src pathway. These suggest that DCA and LCA were key metabolites mediating microbe-epithelium dialogue when the hindgut C/N ratios were altered by cecal infusion of corn starch or casein hydrolysate. These findings provide new insight into the impact of C/N ratio in the large intestine on colonic health and provide a new framework for therapeutic strategy in gut health through targeted manipulation of hindgut microbiota by increasing the carbohydrate level in the large intestine.

[Source](https://msystems.asm.org/content/5/3/e00176-20)

10.1128/mSystems.00176-20

---

## The microbial biogeography of the gastrointestinal tract of preterm and term lambs.
 08.06.2020, _CYL Chong, T Vatanen, M Oliver, FH Bloomfield, JM O'Sullivan_


Nutritional supplementation is a common clinical intervention to support the growth of preterm infants. There is little information on how nutritional supplementation interacts with the developing microbiome of the small intestine, the major site for nutrient metabolism and absorption. We investigated the effect of preterm birth and nutritional supplementation on the mucosal and luminal microbiota along the gastrointestinal tract (GIT) of preterm lambs. Preterm lambs (n = 24) were enterally supplemented with branched-chain amino acids (BCAAs), carbohydrate (maltodextrin), or water for two weeks from birth. Term lambs (n = 7) received water. Mucosal scrapings and luminal samples were collected from the duodenum, jejunum, ileum (small intestine) and colon at six weeks post-term age and analysed by 16S rRNA amplicon sequencing. Anatomical site explained 54% (q = 0.0004) of the variance and differences between the term and preterm groups explained 5.7% (q = 0.024) of the variance in microbial beta-diversities. The colon was enriched with Tenericutes and Verrucomicrobia compared to the small intestine, while Actinobacteria, and superphylum Patescibacteria were present in higher abundance in the small intestine compared to the colon. Our findings highlight that early-life short-term nutritional supplementation in preterm lambs does not alter the microbial community residing in the small intestine and colon.

[Source](https://doi.org/10.1038/s41598-020-66056-z)

10.1038/s41598-020-66056-z

---

## The Microbial Biogeography of the Gastrointestinal Tract of Preterm and Term Lambs.

[Source](https://doi.org/10.1038/s41598-020-66056-z)

---

## Microbial Ecology of Atlantic Salmon (Salmo salar) Hatcheries: Impacts of the Built Environment on Fish Mucosal Microbiota.
 11.06.2020, _JJ Minich, GD Poore, K Jantawongsri, C Johnston, K Bowie, J Bowman, R Knight, B Nowak, EE Allen_


Successful rearing of fish in hatcheries is critical for conservation, recreational fishing, commercial fishing through wild stock enhancements, and aquaculture production. Flowthrough (FT) hatcheries require more water than recirculating aquaculture systems (RAS), which enable up to 99% of their water to be recycled, thus significantly reducing environmental impacts. Here, we evaluated the biological and physical microbiome interactions of three Atlantic salmon hatcheries (RAS n = 2, FT n = 1). Gill, skin, and digesta from six juvenile fish along with tank biofilms and water were sampled from tanks in each of the hatcheries (60 fish across 10 tanks) to assess the built environment and mucosal microbiota using 16S rRNA gene sequencing. The water and tank biofilm had more microbial richness than fish mucus, while skin and digesta from RAS fish had 2 times the richness of FT fish. Body sites each had unique microbiomes (P &lt; 0.001) and were influenced by hatchery system type (P &lt; 0.001), with RAS being more similar. A strong association between the tank and fish microbiome was observed. Water and tank biofilm richness was positively correlated with skin and digesta richness. Strikingly, the gill, skin, and digesta communities were more similar to that in the origin tank biofilm than those in all other experimental tanks, suggesting that the tank biofilm has a direct influence on fish-associated microbial communities. Lastly, microbial diversity and mucous cell density were positively associated with fish growth and length. The results from this study provide evidence for a link between the tank microbiome and the fish microbiome, with the skin microbiome as an important intermediate.IMPORTANCE Atlantic salmon, Salmo salar, is the most farmed marine fish worldwide, with an annual production of 2,248 million metric tons in 2016. Salmon hatcheries are increasingly changing from flowthrough toward recirculating aquaculture system (RAS) design to accommodate more control over production along with improved environmental sustainability due to lower impacts on water consumption. To date, microbiome studies of hatcheries have focused either on the fish mucosal microbiota or on the built environment microbiota but have not combined the two to understand their interactions. Our study evaluates how the water and tank biofilm microbiota influences the fish microbiota across three mucosal environments (gill, skin, and digesta). Results from this study highlight how the built environment is a unique source of microbes to colonize fish mucus and, furthermore, how this can influence fish health. Further studies can use this knowledge to engineer built environments to modulate fish microbiota for beneficial phenotypes.

[Source](https://aem.asm.org/content/86/12/e00411-20)

10.1128/AEM.00411-20

---

## Dysbiosis in the Gut Microbiota of Soil Fauna Explains the Toxicity of Tire Tread Particles.
 16.06.2020, _J Ding, D Zhu, HT Wang, SB Lassen, QL Chen, G Li, M Lv, YG Zhu_


Tread particles (TPs) from vehicle tires are widely distributed in soil ecosystems; therefore, there is an urgent need to evaluate their effects on soil biota. In the present study, the soil worm Enchytraeus crypticus was incubated for 21 days in soil microcosms containing increasing concentrations of TPs (0, 0.0048%, 0.024%, 0.12%, 0.6%, and 3% of dry soil weight). High concentrations of zinc (Zn, 9407.4 mg kg-1) and polycyclic aromatic hydrocarbons (PAHs, 46.8 mg kg-1) were detected in the TPs, which resulted in their increased concentrations in soils amended with TPs. We demonstrated that TPs had an adverse effect on the survival (decreased by more than 25%) and reproduction (decreased by more than 50%) of the soil worms. Moreover, TP exposure disturbed the microbiota of the worm guts and surrounding soil. In addition, a covariation between bacterial and fungal communities was observed in the worm guts after exposure to TPs. Further analysis showed that TP exposure caused an enrichment of microbial genera associated with opportunistic pathogenesis in the worm guts. The combined results from this study indicate that TPs might threaten the terrestrial ecosystem by affecting soil fauna and their gut microbiota.

[Source](https://doi.org/10.1021/acs.est.0c00917)

10.1021/acs.est.0c00917

---

## Dietary Adaptation of Microbiota in Drosophila Requires NF-κB-Dependent Control of the Translational Regulator 4E-BP.
 10.06.2020, _C Vandehoef, M Molaei, J Karpac_


Dietary nutrients shape complex interactions between hosts and their commensal gut bacteria, further promoting flexibility in host-microbiota associations that can drive nutritional symbiosis. However, it remains less clear if diet-dependent host signaling mechanisms also influence these associations. Using Drosophila, we show here that nuclear factor κB (NF-κB)/Relish, an innate immune transcription factor emerging as a signaling node linking nutrient-immune-metabolic interactions, is vital to adapt gut microbiota species composition to host diet macronutrient composition. We find that Relish is required within midgut enterocytes to amplify host-Lactobacillus associations, an important bacterial mediator of nutritional symbiosis, and thus modulate microbiota composition in response to dietary adaptation. Relish limits diet-dependent transcriptional inducibility of the cap-dependent translation inhibitor 4E-BP/Thor to control microbiota composition. Furthermore, maintaining cap-dependent translation in response to dietary adaptation is critical to amplify host-Lactobacillus associations. These results highlight that NF-κB-dependent host signaling mechanisms, in coordination with host translation control, shape diet-microbiota interactions.

[Source](https://doi.org/10.1016/j.celrep.2020.107736)

10.1016/j.celrep.2020.107736

---

## Responses of soil and earthworm gut bacterial communities to heavy metal contamination.
 16.06.2020, _P Liu, Y Yang, M Li_


The large accumulation of heavy metals in the soil surrounding steel factories has become a severe environmental problem. However, few studies have focused on how the earthworm gut microbiota responds to heavy metals in the soil. This study used research sites at a steel factory in Nanjing, China, to investigate how the soil bacterial community and earthworm gut microbiota respond differently to heavy metal contamination using Illumina high-throughput sequencing targeting 16S rRNA genes. The bacterial community of earthworm guts showed a distinct structure compared with that of the soil, featuring a higher relative abundance of Proteobacteria (45.7%) and Bacteroidetes (18.8%). The bacterial community in the earthworm gut appeared more susceptible to heavy metal contamination compared with the soil community. For example, we identified 38 OTUs (Operational taxonomic units) significantly influenced by contamination among 186 abundant OTUs in the soil, whereas 63 out of the 127 abundant OTUs in the earthworm gut were altered significantly under contamination. This susceptibility may be partly explained by the lower alpha diversity and distinct microbial interactions in the gut. In addition, the accumulation of heavy metals also stimulated the growth of potential plant growth promoting bacteria (PGPB) in the earthworm gut, especially those related to indole-3-acetic acid (IAA) and 1-aminocyclopropane-1-carboxylic acid deaminase (ACCD) production, which may potentially benefit the phyto-remediation of heavy metals. These results contribute to our understanding of the soil biota and its interactions under heavy metal contamination and may provide further insights into the phyto-remediation of metal-contaminated soil.

[Source](https://doi.org/10.1016/j.envpol.2020.114921)

10.1016/j.envpol.2020.114921

---

## Effect of life stage and pesticide exposure on the gut microbiota of Aedes albopictus and Culex pipiens L.
 16.06.2020, _EO Juma, BF Allan, CH Kim, C Stone, C Dunlap, EJ Muturi_


Pesticides commonly contaminate the aquatic environments inhabited by mosquito juveniles. However, their role in shaping the mosquito microbiota is not well understood. We hypothesized that environmentally relevant concentrations of atrazine, permethrin and malathion will mediate a shift in the mosquito gut bacterial community structure due to their toxic effect on the aquatic bacterial communities, and reduce mosquito gut bacterial diversity by enriching pesticide-degrading bacterial communities over susceptible taxa. Illumina MiSeq sequencing of the V3-V4 hypervariable regions of the 16 S rRNA gene was used to characterize the microbial communities of larval and adult stages of the two mosquito species and the water samples from microcosms treated with each of the pesticides, separately. Bacterial community composition differed by sample type (larval stage vs. adult stage) and water sampling date (day 3 vs. day 7), but not by pesticide treatment. In larval stages, bacterial OTU richness was highest in samples exposed to malathion, intermediate in permethrin, and lowest in controls. Bacterial richness was significantly higher in larval stages compared to adult stages for all treatments. This study provides a primer for future studies evaluating mosquito microbial responses to exposures to chemical pesticides and the possible implications for mosquito ecology.

[Source](https://doi.org/10.1038/s41598-020-66452-5)

10.1038/s41598-020-66452-5

---

## Effect of Life Stage and Pesticide Exposure on the Gut Microbiota of Aedes Albopictus and Culex Pipiens L.

[Source](https://doi.org/10.1038/s41598-020-66452-5)

---

## Caterpillar gut and host plant phylloplane mycobiomes differ: a new perspective on fungal involvement in insect guts.
 10.06.2020, _D Višňovská, P Pyszko, M Šigut, M Kostovčík, M Kolařík, N Kotásková, P Drozd_


Compared with the highly diverse microbiota of leaves, herbivorous insects exhibit impoverished gut microbial communities. Research to date has focused on the bacterial component of these gut microbiomes, neglecting the fungal component. As caterpillar gut bacterial microbiomes are derived strongly from their diet, we hypothesized that their mycobiomes would reflect the host leaf mycobiomes. Using the ITS2 rDNA and V5-V6 16S rRNA gene regions for DNA metabarcoding of caterpillar gut and host leaf sample pairs we compared their mycobiome genus diversity and compositions and identified genera associated with caterpillar guts. Leaves and caterpillar guts harbored different mycobiomes with quite low qualitative similarity (Jaccard index = 38.03%). The fungal genera most significantly associated with the caterpillar gut included Penicillium, Mucor, and unidentified Saccharomycetales, whereas leaf-associated genera included Holtermanniella, Gibberella (teleomorph of Fusarium), and Seimatosporium. Although caterpillar gut and leaf mycobiomes had similar genus richness overall, this indicator was not correlated for individual duplets. Moreover, as more samples entered the analysis, mycobiome richness increased more rapidly in caterpillar guts than in leaves. The results suggest that the mycobiota of the caterpillar gut differs from that of their feeding substrate; further, the mycobiomes appear to be richer than the well-studied bacterial microbiotas.

[Source](https://doi.org/10.1093/femsec/fiaa116)

10.1093/femsec/fiaa116

---

## Caterpillar Gut and Host Plant Phylloplane Mycobiomes Differ: A New Perspective on Fungal Involvement in Insect Guts.

[Source](https://doi.org/10.1093/femsec/fiaa116)

---

## Poly(propylene imine) dendrimers can bind to PEGylated albumin at PEG and albumin surface: Biophysical examination of a PEGylated platform to transport cationic dendritic nanoparticles.
 16.06.2020, _K Ciepluch, R Biehl, M Bryszewska, M Arabski_


Cationic dendrimers are considered one of the best drug transporters in the body. However, in order to improve their biocompatibility, modification of them is required to reduce toxicity. In this way, many dendrimers may lose their original properties, for example, anticancer. To improve biocompatibility of dendrimers, it is possible to complex them with albumin, as is done very often in drug delivery. However, the interaction of dendrimers with albumin can lead to protein structure disruption or no complexation at all. Therefore, the investigation of the interaction between cationic poly-(propylene imine) dendrimers and polyethylene glycol (PEG)-albumin by fluorescence, circular dichroism, small angle X-ray scattering (SAXS), and transmission electron microscopy was carried out. Results show that cationic dendrimers bind to PEGylated albumin at PEG and albumin surfaces. The obtained results for 5k-PEG indicate a preferential binding of the dendrimers to PEG. For 20k-PEG binding of dendrimers to PEG and protein could induce a collapse of the PEG chain onto the protein surface. This opens up new possibilities to the use of PEGylated albumin as a platform to carry dendrimers without changing the albumin structure and improve the pharmacokinetic properties of dendrimers without further modification.

[Source](https://doi.org/10.1093/femsec/fiaa056)

10.1002/bip.23386

---

## Application of the Fungicide Iprodione: Accelerated Biodegradation or Toxicity?

[Source](https://doi.org/10.1093/femsec/fiaa056)

---

## Host genotype explains rhizospheric microbial community composition: the case of wild cotton metapopulations (Gossypium hirsutum L.) in Mexico.
 03.06.2020, _A Hernández-Terán, M Navarro-Díaz, M Benítez, R Lira, A Wegier, AE Escalante_


The rhizosphere provides several benefits to the plant host being a strong determinant for its health, growth, and productivity. Nonetheless, the factors behind the assembly of the microbial communities associated with the rhizosphere such as the role of plant genotypes is not completely understood. In this study we tested the role that intraspecific genetic variation has in rhizospheric microbial community assemblages, using genetically distinct wild cotton populations as a model of study. We followed a common garden experiment including five wild cotton populations, controlling for plant genotypes, environmental conditions and soil microbial community inoculum, to test for microbial differences associated with genetic variation of the plant hosts. Microbial communities of the treatments were characterized by culture independent 16SrRNA gene amplicon sequencing with Illumina MiSeq platform. We analyzed microbial community diversity (alpha and beta), and diversity structure of such communities, determined by co-occurrence networks. Results show that different plant genotypes select for different and specific microbial communities from a common inoculum. Although we found common amplicon sequence variants (ASVs) to all plant populations (235) we also found unique ASVs for different populations that could be related to potential functional role of such ASVs in the rhizosphere.

[Source](https://doi.org/10.1093/femsec/fiaa109)

10.1093/femsec/fiaa109

---

## Host Genotype Explains Rhizospheric Microbial Community Composition: The Case of Wild Cotton Metapopulations (Gossypium Hirsutum L.) in Mexico.

[Source](https://doi.org/10.1093/femsec/fiaa109)

---

## Microbiome approaches provide the key to biologically control postharvest pathogens and storability of fruits and vegetables.
 16.06.2020, _P Kusstatscher, T Cernava, A Abdelfattah, J Gokul, L Korsten, G Berg_


Microbes play an important role in plants and interact closely with their host starting from sprouting seeds, continuing during growth, and after harvest. The discovery of their importance for plant and postharvest health initiated a biotechnological development of various antagonistic bacteria and fungi for disease control. Nevertheless, their application often showed inconsistent effects. Recently, high-throughput sequencing-based techniques including advanced microscopy reveal fruits and vegetables as holobionts. At harvest, all fruits and vegetables harbor a highly abundant and specific microbiota including beneficial, pathogenic and spoilage microorganisms. Especially, a high microbial diversity and resilient microbial networks were shown to be linked to fruit and vegetable health, while diseased products showed severe dysbioses. Field and postharvest handling of fruits and vegetables was shown to affect the indigenous microbiome and therefore has a substantial impact on the storability of fruits and vegetables. Microbiome tracking can be implemented as a new tool to evaluate and assess all postharvest processes and contribute to fruit and vegetable health. Here we summarize current research advancements in the emerging field of postharvest microbiomes and elaborate its importance. The generated knowledge provides profound insights into postharvest microbiome dynamics and sets a new basis for targeted, microbiome-driven and sustainable control strategies.

[Source](https://doi.org/10.1093/femsec/fiaa119)

10.1093/femsec/fiaa119

---

## Microbiome Approaches Provide the Key to Biologically Control Postharvest Pathogens and Storability of Fruits and Vegetables. 

[Source](https://doi.org/10.1093/femsec/fiaa119)

---

## The Plant Microbiome: From Ecology to Reductionism and Beyond.
 12.06.2020, _CR Fitzpatrick, I Salas-González, JM Conway, OM Finkel, S Gilbert, D Russ, PJPL Teixeira, JL Dangl_


Methodological advances over the past two decades have propelled plant microbiome research, allowing the field to comprehensively test ideas proposed over a century ago and generate many new hypotheses. Studying the distribution of microbial taxa and genes across plant habitats has revealed the importance of various ecological and evolutionary forces shaping plant microbiota. In particular, selection imposed by plant habitats strongly shapes the diversity and composition of microbiota and leads to microbial adaptation associated with navigating the plant immune system and utilizing plant-derived resources. Reductionist approaches have demonstrated that the interaction between plant immunity and the plant microbiome is, in fact, bidirectional and that plants, microbiota, and the environment shape a complex chemical dialogue that collectively orchestrates the plant microbiome. The next stage in plant microbiome research will require the integration of ecological and reductionist approaches to establish a general understanding of the assembly and function in both natural and managed environments. Expected final online publication date for the Annual Review of Microbiology, Volume 74 is September 8, 2020. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.

[Source](https://www.annualreviews.org/doi/10.1146/annurev-micro-022620-014327)

10.1146/annurev-micro-022620-014327

---

## Persistence of wastewater antibiotic resistant bacteria and their genes in human fecal material.
 01.06.2020, _N Scaccia, I Vaz-Moreira, CM Manaia_


Domestic wastewater is a recognized source of antibiotic resistant bacteria and antibiotic resistance genes (ARB&amp;ARGs), whose risk of transmission to humans cannot be ignored. The fitness of wastewater ARB in the complex fecal microbiota of a healthy human was investigated in feces-based microcosm assays (FMAs). FMAs were inoculated with two wastewater isolates, Escherichia coli strain A2FCC14 (MLST ST131) and Enterococcus faecium strain H1EV10 (MLST ST78), harboring the ARGs blaTEM, blaCTX, blaOXA-A and vanA, respectively. The FMAs, incubated in the presence or absence of oxygen or in the presence or absence of the antibiotics cefotaxime or vancomycin, were monitored based on cultivation, ARGs quantification and bacterial community analysis. The fecal bacterial community was dominated by members of the phyla Firmicutes, Bacteroidetes, Actinobacteria, Proteobacteria and Verrucomicrobia. The ARGs harbored by the wastewater isolates could be quantified after one week, in FMAs incubated under both aerobic and anaerobic conditions. These observations were not significantly different in FMAs incubated anaerobically, supplemented with sub-inhibitory concentrations of cefotaxime or vancomycin. The observation that ARGs of wastewater ARB persisted in presence of the human fecal microbiota for at least one week supports the hypothesis of a potential transmission to humans, a topic that deserves further investigation.

[Source](https://doi.org/10.1093/femsec/fiaa058)

10.1093/femsec/fiaa058

---

## Persistence of Wastewater Antibiotic Resistant Bacteria and Their Genes in Human Fecal Material.

[Source](https://doi.org/10.1093/femsec/fiaa058)

---

## Microorganisms populating the water-related indoor biome.
 13.06.2020, _M Novak Babič, C Gostinčar, N Gunde-Cimerman_


Modernisation of our households created novel opportunities for microbial growth and thus changed the array of microorganisms we come in contact with. While many studies have investigated microorganisms in the air and dust, tap water, another major input of microbial propagules, has received far less attention. The quality of drinking water in developed world is strictly regulated to prevent immediate danger to human health. However, fungi, algae, protists and bacteria of less immediate concern are usually not screened for. These organisms can thus use water as a vector of transmission into the households, especially if they are resistant to various water treatment procedures. Good tolerance of unfavourable abiotic conditions is also important for survival once microbes enter the household. Limitation of water availability, high or low temperatures, application of antimicrobial chemicals and other measures are taken to prevent indoor microbial overgrowth. These conditions, together with a large number of novel chemicals in our homes, shape the diversity and abundance of indoor microbiota through constant selection of the most resilient species, resulting in a substantial overlap in diversity of indoor and natural extreme environments. At least in fungi, extremotolerance has been linked to human pathogenicity, explaining why many species found in novel indoor habitats (such as dishwasher) are notable opportunistic pathogens. As a result, microorganisms that often enter our households with water and are then enriched in novel indoor habitats might have a hitherto underestimated impact on the well-being of the increasingly indoor-bound human population. KEY POINTS: Domestic environment harbours a large diversity of microorganisms. Microbiota of water-related indoor habitats mainly originates from tap water. Bathrooms, kitchens and household appliances select for polyextremotolerant species. Many household-related microorganisms are human opportunistic pathogens.

[Source](https://doi.org/10.1007/s00253-020-10719-4)

10.1007/s00253-020-10719-4

---

## Introducing Lu-1, a Novel Lactobacillus jensenii Phage Abundant in the Urogenital Tract.
 16.06.2020, _T Miller-Ensminger, R Mormando, L Maskeri, JW Shapiro, AJ Wolfe, C Putonti_


Bacteriophages (phages) play a key role in shaping microbial communities, including those of the human body. Phages are abundant members of the urogenital tract, most often persisting through the lysogenic life cycle as prophages integrated within the genomes of their bacterial hosts. While numerous studies of the urogenital microbiota have focused on the most abundant bacterial member of this niche-Lactobacillus species-very little is known about Lactobacillus phages. Focusing on Lactobacillus jensenii strains from the urinary tract, we identified numerous prophages related to the previously characterized Lv-1 phage from a vaginal L. jensenii strain. Furthermore, we identified a new L. jensenii phage, Lu-1. Evidence suggests that both phages are abundant within the urogenital tract. CRISPR spacer sequences matching to Lv-1 and Lu-1 prophages were identified. While first detected in urinary isolates, the Lu-1 phage was also discovered in L. jensenii isolates from vaginal and perineal swabs, and both phages were found in metagenomic data sets. The prevalence of these phages in the isolates suggests that both phages are active members of the urogenital microbiota.

[Source](https://doi.org/10.1371/journal.pone.0234159)

10.1371/journal.pone.0234159

---

## Bacteriophage-mediated manipulation of the gut microbiome - promises and presents limitations.
 04.06.2020, _TS Rasmussen, AK Koefoed, RR Jakobsen, L Deng, JL Castro-Mejía, A Brunse, H Neve, FK Vogensen, DS Nielsen_


Gut microbiome (GM) composition and function are linked to human health and disease, and routes for manipulating the GM have become an area of intense research. Due to its high treatment efficacy, the use of fecal microbiota transplantation (FMT) is generally accepted as a promising experimental treatment for patients suffering from GM imbalances (dysbiosis), e.g. caused by recurrent Clostridioides difficile infections (rCDI). Mounting evidence suggests that bacteriophages (phages) play a key role in successful FMT treatment by restoring the dysbiotic bacterial GM. As a refinement to FMT, removing the bacterial component of donor feces by sterile filtration, also referred to as fecal virome transplantation (FVT), decreases the risk of invasive infections caused by bacteria. However, eukaryotic viruses and prophage-encoded virulence factors remain a safety issue. Recent in vivo studies show how cascading effects are initiated when phage communities are transferred to the gut by e.g. FVT, which leads to changes in the GM composition, host metabolome, and improve host health such as alleviating symptoms of obesity and type-2-diabetes (T2D). In this review, we discuss the promises and limitations of FVT along with the perspectives of using FVT to treat various diseases associated with GM dysbiosis.

[Source](https://doi.org/10.1093/femsre/fuaa020)

10.1093/femsre/fuaa020

---

## Bacteriophage-mediated Manipulation of the Gut Microbiome – Promises and Presents Limitations.

[Source](https://doi.org/10.1093/femsre/fuaa020)

---

## gutMEGA: a database of the human gut MEtaGenome Atlas.
 04.06.2020, _Q Zhang, K Yu, S Li, X Zhang, Q Zhao, X Zhao, Z Liu, H Cheng, ZX Liu, X Li_


The gut microbiota plays important roles in human health through regulating both physiological homeostasis and disease emergence. The accumulation of metagenomic sequencing studies enables us to better understand the temporal and spatial variations of the gut microbiota under different physiological and pathological conditions. However, it is inconvenient for scientists to query and retrieve published data; thus, a comprehensive resource for the quantitative gut metagenome is urgently needed. In this study, we developed gut MEtaGenome Atlas (gutMEGA), a well-annotated comprehensive database, to curate and host published quantitative gut microbiota datasets from Homo sapiens. By carefully curating the gut microbiota composition, phenotypes and experimental information, gutMEGA finally integrated 59 132 quantification events for 6457 taxa at seven different levels (kingdom, phylum, class, order, family, genus and species) under 776 conditions. Moreover, with various browsing and search functions, gutMEGA provides a fast and simple way for users to obtain the relative abundances of intestinal microbes among phenotypes. Overall, gutMEGA is a convenient and comprehensive resource for gut metagenome research, which can be freely accessed at http://gutmega.omicsbio.info.

[Source](https://doi.org/10.1093/bib/bbaa082)

10.1093/bib/bbaa082

---

## Engineering CRISPR/Cas9 to mitigate abundant host contamination for 16S rRNA gene-based amplicon sequencing.
 09.06.2020, _L Song, K Xie_


High-throughput sequencing of bacterial 16S rRNA gene (16S-seq) is a useful and common method for studying bacterial community structures. However, contamination of the 16S rRNA genes from the mitochondrion and plastid hinders the sensitive bacterial 16S-seq in plant microbiota profiling, especially for some plant species such as rice. To date, efficiently mitigating such host contamination without a bias is challenging in 16S rRNA gene-based amplicon sequencing.

[Source](https://doi.org/10.1186/s40168-020-00859-0)

10.1186/s40168-020-00859-0

---

## Optimization and standardization of the culturomics technique for human microbiome exploration.
 16.06.2020, _A Diakite, G Dubourg, N Dione, P Afouda, S Bellali, II Ngom, C Valles, ML Tall, JC Lagier, D Raoult_


Culturomics is a high-throughput culture approach that has dramatically contributed to the recent renewal of culture. While metagenomics enabled substantial advances in exploring the microbiota, culturomics significantly expanded our knowledge regarding the bacterial gut repertoire through the discovery and the description of hundreds of new taxa. While this approach relies on the variation of culture conditions and media, we have tested so far more than 300 conditions since the beginning of culturomics studies. In this context, we aimed herein to identify the most profitable conditions for optimizing culturomics approach. For this purpose, we have analysed a set of 58 culturomics conditions that were previously applied to 8 faecal specimens, enabling the isolation of 497 bacterial species. As a result, we were able to reduce the number of conditions used to isolate these 497 of more than a half (i.e. to 25 culture conditions). We have also established a list of the 16 conditions that allowed to capture 98% of the total number of species previously isolated. These data constitute a methodological starting point for culture-based microbiota studies by improving the culturomics workflow without any loss of captured bacterial diversity.

[Source](https://doi.org/10.1038/s41598-020-66738-8)

10.1038/s41598-020-66738-8

---

## Optimization and Standardization of the Culturomics Technique for Human Microbiome Exploration.

[Source](https://doi.org/10.1038/s41598-020-66738-8)

---

## Direct detection of mRNA expression in microbial cells by fluorescence in situ hybridization using RNase H-assisted rolling circle amplification.
 16.06.2020, _H Takahashi, K Horio, S Kato, T Kobori, K Watanabe, T Aki, Y Nakashimada, Y Okamura_


Meta-analyses using next generation sequencing is a powerful strategy for studying microbiota; however, it cannot clarify the role of individual microbes within microbiota. To know which cell expresses what gene is important for elucidation of the individual cell's function in microbiota. In this report, we developed novel fluorescence in situ hybridization (FISH) procedure using RNase-H-assisted rolling circle amplification to visualize mRNA of interest in microbial cells without reverse transcription. Our results show that this method is applicable to both Gram-negative and Gram-positive microbes without any noise from DNA, and it is possible to visualize the target mRNA expression directly at the single-cell level. Therefore, our procedure, when combined with data of meta-analyses, can help to understand the role of individual microbes in the microbiota.

[Source](https://doi.org/10.1038/s41598-020-65864-7)

10.1038/s41598-020-65864-7

---

