## Fecal glycoprotein 2 is a marker of gut microbiota dysbiosis and systemic inflammation.
 23.10.2024, _F Frost, S Weiss, J Hertel, M Rühlemann, C Bang, A Franke, M Nauck, M Dörr, H Völzke, D Roggenbuck, P Schierack, U Völker, G Homuth, AA Aghdassi, M Sendler, MM Lerch, FU Weiss_


Antimicrobial autoantigenic glycoprotein 2 (GP2) is an important component of the innate immune system which originates from the exocrine pancreas as well as from the small intestines. The relationship of GP2 with the intestinal microbiome as well as the systemic implications of increased fecal GP2 levels are, however, still unclear. Therefore, fecal samples from 2,812 individuals of the Study of Health in Pomerania (SHIP) were collected to determine GP2 levels (enzyme-linked immunosorbent assay) and gut microbiota profiles (16 S rRNA gene sequencing). These data were correlated and associated with highly standardised and comprehensive phenotypic data of the study participants.

[Source](https://doi.org/10.1186/s13099-024-00657-1)

10.1186/s13099-024-00657-1

---

## Detrimental effect of an early exposure to antibiotics on the outcomes of immunotherapy in a multi-tumor cohort of patients

[Source](https://doi.org/10.1093/oncolo/oyae284)

---

## Upper respiratory microbial communities of healthy populations are shaped by niche and age.
 23.10.2024, _S Zelasko, MH Swaney, S Sandstrom, TC Davenport, CM Seroogy, JE Gern, LR Kalan, CR Currie_


_Humans, Microbiota, Mouth, Adult, Infant, Female, Bacteria, Male, Age Factors, Child, Preschool, Healthy Volunteers, Metagenomics, Respiratory System, Metagenome_
Alterations in upper respiratory microbiomes have been implicated in shaping host health trajectories, including by limiting mucosal pathogen colonization. However, limited comparative studies of respiratory microbiome development and functioning across age groups have been performed. Herein, we perform shotgun metagenomic sequencing paired with pathogen inhibition assays to elucidate differences in nasal and oral microbiome composition and intermicrobial interactions across healthy 24-month-old infant (n = 229) and adult (n = 100) populations.

[Source](https://doi.org/10.1186/s40168-024-01940-8)

10.1186/s40168-024-01940-8

---

## Intestinal Fatty Acid-Binding Protein as a Marker of Necrotizing Enterocolitis Incidence and Severity: A Scoping Review.
 22.10.2024, _L Sbragia, IJN Gualberto, J Xia, R Gadde, A Saulsbery, S Hameedi, ALF Mársico Dalto, OO Olutoye_


Necrotizing enterocolitis (NEC) is a severe inflammatory disease of the gastrointestinal tract and one of the most common life-threatening emergencies affecting newborns. Intestinal fatty acid-binding protein (I-FABP) has been used as a possible marker of intestinal damage in NEC. We aimed to carry out a scoping review of all publications that explore the role of I-FABP in NEC to inspire new research into the potential utility of I-FABP as a marker of NEC.

[Source](https://doi.org/10.1038/s41598-024-75157-y)

10.1016/j.jss.2024.09.059

---

## Dysbiosis of the initial stool microbiota increases the risk of developing necrotizing enterocolitis or feeding intolerance in newborns

[Source](https://doi.org/10.1038/s41598-024-75157-y)

---

## Resource sharing of an infant gut microbiota synthetic community in combinations of human milk oligosaccharides.
 18.10.2024, _A Ioannou, MD Berkhout, WT Scott, B Blijenberg, S Boeren, M Mank, J Knol, C Belzer_


Quickly after birth, the gut microbiota is shaped via species acquisition and resource pressure. Breastmilk, and more specifically, human milk oligosaccharides are a determining factor in the formation of microbial communities and the interactions between bacteria. Prominent human milk oligosaccharide degraders have been rigorously characterized, but it is not known how the gut microbiota is shaped as a complex community. Here, we designed BIG-Syc, a synthetic community of 13 strains from the gut of vaginally born, breastfed infants. BIG-Syc replicated key compositional, metabolic, and proteomic characteristics of the gut microbiota of infants. Upon fermentation of a 4 and 5 human milk oligosaccharide mix, BIG-Syc demonstrated different compositional and proteomic profiles, with Bifidobacterium infantis and Bifidobacterium bifidum suppressing one another. The mix of 5 human milk oligosaccharides resulted in a more diverse composition with dominance of B. bifidum, whereas that with 4 human milk oligosaccharides supported the dominance of B. infantis, in 4 of 6 replicates. Reintroduction of bifidobacteria to BIG-Syc led to their engraftment and establishment of their niche. Based on proteomics and genome-scale metabolic models, we reconstructed the carbon source utilization and metabolite and gas production per strain. BIG-Syc demonstrated teamwork as cross-feeders utilized simpler carbohydrates, organic acids, and gases released from human milk oligosaccharide degraders. Collectively, our results showed that human milk oligosaccharides prompt resource-sharing for their complete degradation while leading to a different compositional and functional profile in the community. At the same time, BIG-Syc proved to be an accurate model for the representation of intra-microbe interactions.

[Source](https://doi.org/10.1093/ismejo/wrae209)

10.1093/ismejo/wrae209

---

## Aging-induced dysbiosis worsens sepsis severity but is attenuated by probiotics in D-galactose-administered mice with cecal ligation and puncture model.
 20.10.2024, _C Pinitchun, W Panpetch, T Bhunyakarnjanarat, K Udompornpitak, HT Do, P Visitchanakun, DL Wannigama, S Udomkarnjananun, M Sukprasansap, T Tencomnao, P Tangtanatakul, A Leelahavanichkul_


_Animals, Galactose, Dysbiosis, Mice, Probiotics, Sepsis, Aging, Mice, Inbred C57BL, Disease Models, Animal, Male, Cecum, Gastrointestinal Microbiome, RAW 264.7 Cells, Lacticaseibacillus rhamnosus, Ligation, Feces, Punctures, Cytokines, Sepsis-Associated Encephalopathy_
Despite the well-established effects of aging on brain function and gut dysbiosis (an imbalance in gut microbiota), the influence of aging on sepsis-associated encephalopathy (SAE) and the role of probiotics in this context remain less understood.

[Source](https://doi.org/10.1371/journal.pone.0311774)

10.1371/journal.pone.0311774

---

## Multi-omics analysis reveals the interplay between pulmonary microbiome and host in immunocompromised patients with sepsis-induced acute lung injury.
 18.10.2024, _F Lu, T Huang, R Chen, H Yin_


The mechanisms behind the high inflammatory state and immunocompromise in severe sepsis remain unclear. While microbiota's role in immune regulation is known, the impact of pulmonary microbiota on sepsis progression is not fully understood. This study aims to investigate pulmonary microbial characteristics in septic patients and their relationship with host immune-related genes and clinical features. Fifty-four sepsis patients were divided into the immunocompromised host (ICH) group (n = 18) and the control group (n = 36). Bronchoalveolar lavage fluid (BALF) was analyzed using metagenomic next-generation sequencing (mNGS) to assess the pulmonary microbiome, and transcriptomic sequencing evaluated host gene expression. The pulmonary microbiota network in the ICH group showed notable alterations. Symbiotic bacteria like Streptococcus salivarius and Streptococcus oralis were key taxa in the control group. In contrast, opportunistic pathogens such as Campylobacter concisus and Prevotella melaninogenica, typically linked to infections in various body sites, dominated in the ICH group. Transcriptomic analysis revealed differential genes between the two groups. The downregulated differential genes in the ICH group were primarily enriched in pathways related to T-cell activation and the Type I interferon signaling pathway, both crucial for the immune system. Further correlation analysis identified significant associations between certain microbes and host genes, as well as clinical indicators, particularly with species like Campylobacter concisus, Streptococcus salivarius, Streptococcus oralis, and several species of Veillonella. These findings suggest that alterations in the pulmonary microbiome, especially the presence of opportunistic pathogens, may contribute to immune dysregulation in immunocompromised septic patients, warranting further research to explore causal relationships.

[Source](https://doi.org/10.1128/spectrum.01424-24)

10.1128/spectrum.01424-24

---

## A <em>Blautia producta</em> specific gFET-based aptasensor for quantitative monitoring of microbiome quality

[Source](https://doi.org/10.1039/d4nh00281d)

---

## Alterations of the gut microbiome in HIV infection highlight human anelloviruses as potential predictors of immune recovery.
 20.10.2024, _C Boukadida, A Peralta-Prado, M Chávez-Torres, K Romero-Mora, A Rincon-Rubio, S Ávila-Ríos, D Garrido-Rodríguez, G Reyes-Terán, S Pinto-Cardoso_


_Humans, Gastrointestinal Microbiome, HIV Infections, Feces, Cross-Sectional Studies, Male, Longitudinal Studies, Adult, HIV-1, Female, Anelloviridae, Dysbiosis, Middle Aged, CD4 Lymphocyte Count, Bacteriophages, Bacteria, CD4-Positive T-Lymphocytes, Virome_
HIV-1 infection is characterized by a massive depletion of mucosal CD4 T cells that triggers a cascade of events ultimately linking gut microbial dysbiosis to HIV-1 disease progression and pathogenesis. The association between HIV infection and the enteric virome composition is less characterized, although viruses are an essential component of the gut ecosystem. Here, we performed a cross-sectional analysis of the fecal viral (eukaryotic viruses and bacteriophages) and bacterial microbiome in people with HIV (PWH) and in HIV-negative individuals. To gain further insight into the association between the gut microbiome composition, HIV-associated immunodeficiency, and immune recovery, we carried out a longitudinal study including 14 PWH who initiated antiretroviral therapy (ART) and were followed for 24 months with samplings performed at baseline (before ART) and at 2, 6, 12, and 24 months post-ART initiation.

[Source](https://doi.org/10.1186/s40168-024-01925-7)

10.1186/s40168-024-01925-7

---

## Gut microbiota determines the fate of dietary fiber-targeted interventions in host health.
 20.10.2024, _W Wang, Z Fan, Q Yan, T Pan, J Luo, Y Wei, B Li, Z Fang, W Lu_


_Dietary Fiber, Gastrointestinal Microbiome, Humans, Prebiotics, Animals, Bacteria, Fatty Acids, Volatile, Bile Acids and Salts_
Epidemiological investigation confirmed that the intake of dietary fiber (DF) is closely related to human health, and the most important factor affecting the physiological function of DF, besides its physicochemical properties, is the gut microbiota. This paper mainly summarizes the interaction between DF and gut microbiota, including the influence of DF on the colonization of gut microbiota based on its different physicochemical properties, and the physiological role of gut microbiota in destroying the complex molecular structure of DF by encoding carbohydrate-active enzymes, thus producing small molecular products that affect the metabolism of the host. Taking cardiovascular disease (Atherosclerosis and hypertension), liver disease, and immune diseases as examples, it is confirmed that some DF, such as fructo-oligosaccharide, galactooligosaccharide, xylo-oligosaccharide, and inulin, have prebiotic-like physiological effects. These effects are dependent on the metabolites produced by the gut microbiota. Therefore, this paper further explores how DF affects the gut microbiota's production of substances such as short-chain fatty acids, bile acids, and tryptophan metabolites, and provides a preliminary explanation of the mechanisms associated with their impact on host health. Finally, based on the structural properties of DF and the large heterogeneity in the composition of the population gut microbiota, it may be a future trend to utilize DF and the gut microbiota to correlate host health for precision nutrition by combining the information from population disease databases.

[Source](https://doi.org/10.1080/19490976.2024.2416915)

10.1080/19490976.2024.2416915

---

## Vaginal lactobacilli produce anti-inflammatory β-carboline compounds.
 18.10.2024, _VJ Glick, CA Webber, LE Simmons, MC Martin, M Ahmad, CH Kim, AND Adams, S Bang, MC Chao, NC Howard, SM Fortune, M Verma, M Jost, LK Beura, MJ James, SY Lee, CM Mitchell, J Clardy, KH Kim, S Gopinath_


The optimal vaginal microbiome is a Lactobacillus-dominant community. Apart from Lactobacillus iners, the presence of Lactobacillus species is associated with reduced vaginal inflammation and reduced levels of pro-inflammatory cytokines. Loss of Lactobacillus-dominance is associated with inflammatory conditions, such as bacterial vaginosis (BV). We have identified that Lactobacillus crispatus, a key vaginal bacterial species, produces a family of β-carboline compounds with anti-inflammatory activity. These compounds suppress nuclear factor κB (NF-κB) and interferon (IFN) signaling downstream of multiple pattern recognition receptors in primary human cells and significantly dampen type I IFN receptor (IFNAR) activation in monocytes. Topical application of an anti-inflammatory β-carboline compound, perlolyrine, was sufficient to significantly reduce vaginal inflammation in a mouse model of genital herpes infection. These compounds are enriched in cervicovaginal lavage (CVL) of healthy people compared with people with BV. This study identifies a family of compounds by which vaginal lactobacilli mediate host immune homeostasis and highlights a potential therapeutic avenue for vaginal inflammation.

[Source](https://doi.org/10.1016/j.chom.2024.09.014)

10.1016/j.chom.2024.09.014

---

## EphA2 blockage ALW-II-41-27 alleviates atherosclerosis by remodeling gut microbiota to regulate bile acid metabolism.
 23.10.2024, _C Lu, D Liu, Q Wu, J Zeng, Y Xiong, T Luo_


_Animals, Atherosclerosis, Gastrointestinal Microbiome, Bile Acids and Salts, Mice, Receptor, EphA2, Diet, High-Fat, Male, Humans, Disease Models, Animal, Plaque, Atherosclerotic, Mice, Inbred C57BL, Bacteria, Dysbiosis_
Coronary artery disease (CAD), a critical condition resulting from systemic inflammation, metabolic dysfunction, and gut microbiota dysbiosis, poses a global public health challenge. ALW-II-41-27, a specific inhibitor of the EphA2 receptor, has shown anti-inflammatory prosperities. However, the impact of ALW-II-41-27 on atherosclerosis has not been elucidated. This study aimed to examine the roles of pharmacologically inhibiting EphA2 and the underlying mechanism in ameliorating atherosclerosis. ALW-II-41-27 was administered to apoE-/- mice fed a high-fat diet via intraperitoneal injection. We first discovered that ALW-II-41-27 led to a significant reduction in atherosclerotic plaques, evidenced by reduced lipid and macrophage accumulation, alongside an increase in collagen and smooth muscle cell content. ALW-II-41-27 also significantly lowered plasma and hepatic cholesterol levels, as well as the colonic inflammation. Furthermore, gut microbiota was analyzed by metagenomics and plasma metabolites by untargeted metabolomics. ALW-II-41-27-treated mice enriched Enterococcus, Akkermansia, Eggerthella and Lactobaccilus, accompanied by enhanced secondary bile acids production. To explore the causal link between ALW-II-41-27-associated gut microbiota and atherosclerosis, fecal microbiota transplantation was employed. Mice that received ALW-II-41-27-treated mouse feces exhibited the attenuated atherosclerotic plaque. In clinical, lower plasma DCA and HDCA levels were determined in CAD patients using quantitative metabolomics and exhibited a negative correlation with higher monocytes EphA2 expression. Our findings underscore the potential of ALW-II-41-27 as a novel therapeutic agent for atherosclerosis, highlighting its capacity to modulate gut microbiota composition and bile acid metabolism, thereby offering a promising avenue for CAD.

[Source](https://doi.org/10.1038/s41522-024-00585-7)

10.1038/s41522-024-00585-7

---

## <em>Lactobacillus rhamnosus</em> GG and <em>Bifidobacterium animalis</em> subsp. lactis BB-12 promote infected wound healing via regulation of the wound microenvironment

[Source](https://doi.org/10.1111/1751-7915.70031)

---

## Ameliorative effects of <em>Akkermansia muciniphila</em> on anxiety-like behavior and cognitive deficits in a rat model of Alzheimer’s disease

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0006899324005341?via%3Dihub)

---

## Bacterial and Symbiodiniaceae communities' variation in corals with distinct traits and geographical distribution.
 19.10.2024, _LB Villela, AW da Silva-Lima, APB Moreira, YRA Aiube, FV Ribeiro, HDM Villela, ME Majzoub, M Amario, RL de Moura, T Thomas, RS Peixoto, PS Salomon_


_Animals, Anthozoa, Microbiota, Coral Reefs, RNA, Ribosomal, 16S, Bacteria, Brazil, Symbiosis, Phylogeny, Dinoflagellida, High-Throughput Nucleotide Sequencing, Ecosystem_
Coral microbiomes play crucial roles in holobiont homeostasis and adaptation. The host's ability to populate broad ecological niches and to cope with environmental changes seems to be related to the flexibility of the coral microbiome. By means of high-throughput DNA sequencing we characterized simultaneously both bacterial (16S rRNA) and Symbiodiniaceae (ITS2) communities of four reef-building coral species (Mussismilia braziliensis, Mussismilia harttii, Montastraea cavernosa, and Favia gravida) that differ in geographic distribution and niche specificity. Samples were collected in a marginal reef system (Abrolhos, Brazil) in four sites of contrasting irradiance and turbidity. Biological filters governed by the host are important in shaping corals' microbiome structure. More structured associated microbial communities by reef site tend to occur in coral species with broader geographic and depth ranges, especially for Symbiodiniaceae, whereas the endemic and habitat-specialist host, M. braziliensis, has relatively more homogenous bacterial communities with more exclusive members. Our findings lend credence to the hypothesis that higher microbiome flexibility renders corals more adaptable to diverse environments, a trend that should be investigated in more hosts and reef areas.

[Source](https://doi.org/10.1038/s41598-024-70121-2)

10.1038/s41598-024-70121-2

---

## The Role of Geography, Diet, and Host Phylogeny on the Gut Microbiome in the Hawaiian Honeycreeper Radiation.
 18.10.2024, _MS Costantini, E Videvall, JT Foster, MCI Medeiros, JD Gillece, EH Paxton, LH Crampton, HL Mounce, AX Wang, RC Fleischer, MG Campana, FA Reed_


The animal gut microbiome can have a strong influence on the health, fitness, and behavior of its hosts. The composition of the gut microbial community can be influenced by factors such as diet, environment, and evolutionary history (phylosymbiosis). However, the relative influence of these factors is unknown in most bird species. Furthermore, phylosymbiosis studies have largely focused on clades that diverged tens of millions of years ago, and little is known about the degree of gut microbiome divergence in more recent species radiations. This study explores the drivers of microbiome variation across the unique and recent Hawaiian honeycreeper radiation (Fringillidae: Drepanidinae). Fecal samples were collected from 14 extant species spanning the main islands of the Hawaiian archipelago and were sequenced using three metabarcoding markers to characterize the gut microbiome, invertebrate diet, and plant diet of Hawaiian honeycreepers. We then used these metabarcoding data and the honeycreeper host phylogeny to evaluate their relative roles in shaping the gut microbiome. Microbiome variation across birds was highly individualized; however, source island had a small but significant effect on microbiome structure. The microbiomes did not recapitulate the host phylogenetic tree, indicating that evolutionary history does not strongly influence microbiome structure in the honeycreeper clade. These results expand our understanding of the roles of diet, geography, and phylogeny on avian microbiome structure, while also providing important ecological information about the diet and gut microbiota of wild Hawaiian honeycreepers.

[Source](https://doi.org/10.1002/ece3.70372)

10.1002/ece3.70372

---

## New insights into the enteric methane production based on the archaeal genome map of ruminant gastrointestinal tract.
 19.10.2024, _Y Tang, X Liu, S Zhu, M Jia, JX Liu, HZ Sun_


As one of the important components of ruminant gastrointestinal tract (GIT) microbiome, archaea are involved in many biological processes, especially methanogenesis. However, the diversity and individual metabolic characteristics of archaea in this habitat remain largely understudied, partly due to the lack of a unified reference genome catalog.

[Source](https://doi.org/10.1016/j.jare.2024.09.016)

10.1016/j.jare.2024.09.016

---

## A metabolic modeling-based framework for predicting trophic dependencies in native rhizobiomes of crop plants.
 23.10.2024, _AA Ginatt, M Berihu, E Castel, S Medina, G Carmi, A Faigenboim-Doron, I Sharon, O Tal, S Droby, T Somera, M Mazzola, H Eizenberg, S Freilich_


_Rhizosphere, Malus, Microbiota, Plant Roots, Soil Microbiology, Bacteria, Crops, Agricultural, Metabolomics, Models, Biological_
The exchange of metabolites (i.e., metabolic interactions) between bacteria in the rhizosphere determines various plant-associated functions. Systematically understanding the metabolic interactions in the rhizosphere, as well as in other types of microbial communities, would open the door to the optimization of specific predefined functions of interest, and therefore to the harnessing of the functionality of various types of microbiomes. However, mechanistic knowledge regarding the gathering and interpretation of these interactions is limited. Here, we present a framework utilizing genomics and constraint-based modeling approaches, aiming to interpret the hierarchical trophic interactions in the soil environment. 243 genome scale metabolic models of bacteria associated with a specific disease-suppressive vs disease-conducive apple rhizospheres were drafted based on genome-resolved metagenomes, comprising an in silico native microbial community. Iteratively simulating microbial community members' growth in a metabolomics-based apple root-like environment produced novel data on potential trophic successions, used to form a network of communal trophic dependencies. Network-based analyses have characterized interactions associated with beneficial vs non-beneficial microbiome functioning, pinpointing specific compounds and microbial species as potential disease supporting and suppressing agents. This framework provides a means for capturing trophic interactions and formulating a range of testable hypotheses regarding the metabolic capabilities of microbial communities within their natural environment. Essentially, it can be applied to different environments and biological landscapes, elucidating the conditions for the targeted manipulation of various microbiomes, and the execution of countless predefined functions.

[Source](https://doi.org/10.7554/eLife.94558)

10.7554/eLife.94558

---

## Unravelling the microbiome of wild flowering plants: a comparative study of leaves and flowers in alpine ecosystems.
 21.10.2024, _DK Ramakrishnan, F Jauernegger, D Hoefle, C Berg, G Berg, A Abdelfattah_


_Plant Leaves, Microbiota, Bacteria, Fungi, Flowers, Soil Microbiology, Ecosystem, Austria, Altitude, Magnoliopsida, Biodiversity, Soil_
While substantial research has explored rhizosphere and phyllosphere microbiomes, knowledge on flower microbiome, particularly in wild plants remains limited. This study explores into the diversity, abundance, and composition of bacterial and fungal communities on leaves and flowers of wild flowering plants in their natural alpine habitat, considering the influence of environmental factors.

[Source](https://doi.org/10.1186/s12866-024-03574-0)

10.1186/s12866-024-03574-0

---

## Algae-fungi symbioses and bacteria-fungi co-exclusion drive tree species-specific differences in canopy bark microbiomes.
 17.10.2024, _J Freudenthal, K Dumack, S Schaffer, M Schlegel, M Bonkowski_


With over 3 trillion trees, forest ecosystems comprise nearly one-third of the terrestrial surface of the Earth. Very little attention has been given to the exploration of the above-ground plant microbiome of trees, its complex trophic interactions, and variations among tree species. To address this knowledge gap, we applied a primer-independent shotgun metatranscriptomic approach to assess the entire living canopy bark microbiome comprising prokaryotic and eukaryotic primary producers, decomposers, and various groups of consumers. With almost 1500 genera, we found a high microbial diversity on three tree species with distinct bark textures: oak (Quercus robur), linden (Tilia cordata), both with rough bark, and maple (Acer pseudoplatanus) with smooth bark. Core co-occurrence network analysis revealed a rich food web dominated by algal primary producers, and bacterial and fungal decomposers, sustaining a diverse community of consumers, including protists, microscopic metazoans and predatory bacteria. Whereas maple accommodated a depauperate microbiome, oak and linden accommodated a richer microbiome mainly differing in their relative community composition: Bacteria exhibited an increased dominance on linden, whereas co-occurring algae and fungi dominated on oak, highlighting the importance of algal-fungal lichen symbioses even at the microscopic scale. Further, due to bacteria-fungi co-exclusion, bacteria on bark are not the main beneficiaries of algae-derived carbon compounds as it is known from aquatic systems.

[Source](https://doi.org/10.1093/ismejo/wrae206)

10.1093/ismejo/wrae206

---

## Agricultural practices influence soil microbiome assembly and interactions at different depths identified by machine learning.
 23.10.2024, _Y Mo, R Bier, X Li, M Daniels, A Smith, L Yu, J Kan_


_Soil Microbiology, Machine Learning, Agriculture, Microbiota, Fungi, Soil, Bacteria_
Agricultural practices affect soil microbes which are critical to soil health and sustainable agriculture. To understand prokaryotic and fungal assembly under agricultural practices, we use machine learning-based methods. We show that fertility source is the most pronounced factor for microbial assembly especially for fungi, and its effect decreases with soil depths. Fertility source also shapes microbial co-occurrence patterns revealed by machine learning, leading to fungi-dominated modules sensitive to fertility down to 30 cm depth. Tillage affects soil microbiomes at 0-20 cm depth, enhancing dispersal and stochastic processes but potentially jeopardizing microbial interactions. Cover crop effects are less pronounced and lack depth-dependent patterns. Machine learning reveals that the impact of agricultural practices on microbial communities is multifaceted and highlights the role of fertility source over the soil depth. Machine learning overcomes the linear limitations of traditional methods and offers enhanced insights into the mechanisms underlying microbial assembly and distributions in agriculture soils.

[Source](https://doi.org/10.1038/s42003-024-07059-8)

10.1038/s42003-024-07059-8

---

## Community successional patterns and inter-kingdom interactions during granular biofilm development.
 23.10.2024, _M de Celis, O Modin, L Arregui, F Persson, A Santos, I Belda, BM Wilén, R Liébana_


_Biofilms, Sewage, Bioreactors, Bacteria, Microbial Interactions, Eukaryota, Microbiota_
Aerobic granular sludge is a compact and efficient biofilm process used for wastewater treatment which has received much attention and is currently being implemented worldwide. The microbial associations and their ecological implications occurring during granule development, especially those involving inter-kingdom interactions, are poorly understood. In this work, we monitored the prokaryote and eukaryote community composition and structure during the granulation of activated sludge for 343 days in a sequencing batch reactor (SBR) and investigated the influence of abiotic and biotic factors on the granule development. Sludge granulation was accomplished with low-wash-out dynamics at long settling times, allowing for the microbial communities to adapt to the SBR environmental conditions. The sludge granulation and associated changes in microbial community structure could be divided into three stages: floccular, intermediate, and granular. The eukaryotic and prokaryotic communities showed parallel successional dynamics, with three main sub-communities identified for each kingdom, dominating in each stage of sludge granulation. Although inter-kingdom interactions were shown to affect community succession during the whole experiment, during granule development random factors like the availability of settlement sites or drift acquired increasing importance. The prokaryotic community was more affected by deterministic factors, including reactor conditions, while the eukaryotic community was to a larger extent shaped by biotic interactions (including inter-kingdom interactions) and stochasticity.

[Source](https://doi.org/10.1038/s41522-024-00581-x)

10.1038/s41522-024-00581-x

---

## Metagenome-based microbial community analysis of urine-derived fertilizer.
 21.10.2024, _NN Woldeyohannis, AF Desta_


_Humans, Fertilizers, Urine, Bacteria, Struvite, Metagenome, Microbiota, Metagenomics, DNA, Bacterial, Phylogeny_
Phosphorus is essential for food production and its supply is limited. Urine is an excellent source of phosphorus and one way to produce fertilizer is through conversion of urine to struvite (MgNH3PO4.6H2O). The present study aimed to understand the bacterial portion of the microbial community composition and dynamics of plasmid-mediated antimicrobial resistant genes during the optimized process of struvite production from composite human urine. Samples for DNA extraction was collected from fresh urine, stored urine and struvite during the process of struvite production. Shotgun metagenomic analysis was employed to understand the bacterial community. The most dominant phyla in the fresh and stored urine samples were Pseudomonadata, which comprised of 60% and 43% respectively, followed by Bacillota, comprised of 25% and 39% respectively. The struvite sample was dominated by the phylum Bacilliota (61%), Pseudomonadota (18%) and bacteroidota (12%). Members of the above phyla persisted in dominating each sample accordingly. Member of the family Morganellaceae was dominant in the fresh sample while the stored urine and struvite samples were dominated by the family Clostridiaceae. A decrease of members of the class Gammaproteobacteria was observed from the fresh to the struvite sample though not statistically significant. The genus Pseudomonas remained to be the most dominant member of Gammaproteobacteria in the fresh and stored urine sample with OTU count of 12,116 and 6,155 with a marked decrease by half in the stored sample. On the other hand, members of the genera Clostridium, Enterococcus, Bacteroides in the stored samples and Clostridium, Alkaliphilus and Pseudomonas in the struvite samples were dominant. 96% of the identified genera were shared in all the samples and the antimicrobial resistance genes (ARGs) identified in the fresh urine were shared by the struvite but not by the stored urine (e.g. sul, cat, aph and aac members). The presence of high abundance of ARGs in struvite needs attention in the persistence and transmissibility of the ARGs before application for agriculture.

[Source](https://doi.org/10.1186/s12866-024-03578-w)

10.1186/s12866-024-03578-w

---

## Metabolic cross-feeding interactions modulate the dynamic community structure in microbial fuel cell under variable organic loading wastewaters.
 17.10.2024, _N Srinak, P Chiewchankaset, S Kalapanulak, P Panichnumsin, T Saithong_


The efficiency of microbial fuel cells (MFCs) in industrial wastewater treatment is profoundly influenced by the microbial community, which can be disrupted by variable industrial operations. Although microbial guilds linked to MFC performance under specific conditions have been identified, comprehensive knowledge of the convergent community structure and pathways of adaptation is lacking. Here, we developed a microbe-microbe interaction genome-scale metabolic model (mmGEM) based on metabolic cross-feeding to study the adaptation of microbial communities in MFCs treating sulfide-containing wastewater from a canned-pineapple factory. The metabolic model encompassed three major microbial guilds: sulfate-reducing bacteria (SRB), methanogens (MET), and sulfide-oxidizing bacteria (SOB). Our findings revealed a shift from an SOB-dominant to MET-dominant community as organic loading rates (OLRs) increased, along with a decline in MFC performance. The mmGEM accurately predicted microbial relative abundance at low OLRs (L-OLRs) and adaptation to high OLRs (H-OLRs). The simulations revealed constraints on SOB growth under H-OLRs due to reduced sulfate-sulfide (S) cycling and acetate cross-feeding with SRB. More cross-fed metabolites from SRB were diverted to MET, facilitating their competitive dominance. Assessing cross-feeding dynamics under varying OLRs enabled the execution of practical scenario-based simulations to explore the potential impact of elevated acidity levels on SOB growth and MFC performance. This work highlights the role of metabolic cross-feeding in shaping microbial community structure in response to high OLRs. The insights gained will inform the development of effective strategies for implementing MFC technology in real-world industrial environments.

[Source](https://doi.org/10.1371/journal.pcbi.1012533)

10.1371/journal.pcbi.1012533

---

## Evaluation of inter- and intra-variability in gut health markers in healthy adults using an optimised faecal sampling and processing method.
 23.10.2024, _K Kruger, Y Myeonghyun, N van der Wielen, DE Kok, GJ Hooiveld, S Keshtkar, M Diepeveen-de Bruin, MGJ Balvers, M Grootte-Bromhaar, K Mudde, NTHN Ly, Y Vermeiren, LCPGM de Groot, RCH de Vos, GB Gonzales, WT Steegenga, MPH van Trijp_


_Humans, Feces, Adult, Biomarkers, Male, Female, Gastrointestinal Microbiome, Healthy Volunteers, Specimen Handling, Middle Aged, Young Adult_
Despite advances in gut health research, the variability of important gut markers within individuals over time remains underexplored. We investigated the intra-individual variation of various faecal gut health markers using an optimised processing protocol aimed at reducing variability. Faecal samples from ten healthy adults over three consecutive days demonstrated marker-specific intra-individual coefficients of variation (CV%), namely: stool consistency (16.5%), water content (5.7%), pH (3.9%), total SCFAs (17.2%), total BCFAs (27.4%), total bacteria and fungi copies (40.6% and 66.7%), calprotectin and myeloperoxidase (63.8% and 106.5%), and untargeted metabolites (on average 40%). For thirteen microbiota genera, including Bifidobacterium and Akkermansia, variability exceeded 30%, whereas microbiota diversity was less variable (Phylogenetic Diversity 3.3%, Inverse Simpson 17.2%). Mill-homogenisation of frozen faeces significantly reduced the replicates CV% for total SCFAs (20.4-7.5%) and total BCFAs (15.9-7.8%), and untargeted metabolites compared to faecal hammering only, without altering mean concentrations. Our results show the potential need for repeated sampling to accurately represent specific gut health markers. We also demonstrated the effectiveness of optimised preprocessing of human stool samples in reducing overall analytical variability.

[Source](https://doi.org/10.1038/s41598-024-75477-z)

10.1038/s41598-024-75477-z

---

## LineageFilter: Improved Proteotyping of Complex Samples Using Metaproteomics and Machine Learning.
 19.10.2024, _H Hachemi, J Armengaud, L Grenga, O Pible_


Metaproteomics is a powerful tool to characterize how microbiota function by analyzing their proteic content by tandem mass spectrometry. Given the complexity of these samples, accurately assessing their taxonomical composition without prior information based solely on peptide sequences remains a challenge. Here, we present LineageFilter, a new python-based AI software for refined proteotyping of complex samples using metaproteomics interpreted data and machine learning. Given a tentative list of taxa, their abundances, and the scores associated with their identified peptides, LineageFilter computes a comprehensive set of features for each identified taxon at all taxonomical ranks. Its machine-learning model then assesses the likelihood of each taxon's presence based on these features, enabling improved proteotyping and sample-specific database construction.

[Source](https://doi.org/10.1021/acs.jproteome.4c00184)

10.1021/acs.jproteome.4c00184

---

