## Global Outbreaks and Responses

[Source](https://www.biomedcentral.com/collections/globaloutbreaksandresponses)

---

## Fungal diseases as neglected pathogens: A wake-up call to public health officials

[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0007964)

---

## Inducible cell-to-cell signaling for tunable dynamics in microbial communities

[Source](https://www.nature.com/articles/s41467-020-15056-8)

---

## The insect antimicrobial peptide cecropin A disrupts uropathogenic Escherichia coli biofilms

[Source](https://www.nature.com/articles/s41522-020-0116-3)

---

## Using comparative genomics to understand molecular features of carbapenem-resistant Acinetobacter baumannii from South Korea causing invasive infections and their clinical implications

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229416)

---

## Factors associated with sufficient knowledge of antibiotics and antimicrobial resistance in the Japanese general population

[Source](https://www.nature.com/articles/s41598-020-60444-1)

---

## Faeces and vaccines could be the key to tackling antimicrobial resistance

[Source](https://www.bmj.com/content/368/bmj.m731)

---

## A survey of extended-spectrum beta-lactamase-producing Enterobacteriaceae in urban wetlands in southwestern Nigeria as a step towards generating prevalence maps of antimicrobial resistance

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229451)

---

## Viable bacterial colonization is highly limited in the human intestine in utero

[Source](https://www.nature.com/articles/s41591-020-0761-3)

---

## Perinatal Antibiotic Exposure Affects the Transmission between Maternal and Neonatal Microbiota and Is Associated with Early-Onset Sepsis

[Source](https://msphere.asm.org/content/msph/5/1/e00984-19.full.pdf)

---

## Combined analysis of microbial metagenomic and metatranscriptomic sequencing data to assess in situ physiological conditions in the premature infant gut

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229537)

---

## A microbiome and metabolomic signature of phases of cutaneous healing identified by profiling sequential acute wounds of human skin: An exploratory study

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229545)

---

## Single-Cell Genomics and the Oral Microbiome

[Source](https://journals.sagepub.com/doi/abs/10.1177/0022034520907380)

---

## Insights Obtained by Culturing Saccharibacteria With Their Bacterial Hosts

[Source](https://journals.sagepub.com/doi/abs/10.1177/0022034520905792)

---

## Electronic Cigarette Aerosol Modulates the Oral Microbiome and Increases Risk of Infection

[Source](https://www.cell.com/iscience/fulltext/S2589-0042(20)30068-7)

---

## Oral Microbiome Changes Implicated in Gum Inflammation in E-Cigarette Users

[Source](https://www.genomeweb.com/sequencing/oral-microbiome-changes-implicated-gum-inflammation-e-cigarette-users#.XmCCpy2ZPOQ)

---

## A salivary metabolite signature that reflects gingival host-microbe interactions: instability predicts gingivitis susceptibility

[Source](https://www.nature.com/articles/s41598-020-59988-z)

---

## The Role of Lung and Gut Microbiota in the Pathology of Asthma

[Source](https://www.sciencedirect.com/science/article/pii/S1074761320300340)

---

## Endogenous murine microbiota member Faecalibaculum rodentium and its human homologue protect from intestinal tumour growth

[Source](https://www.nature.com/articles/s41564-019-0649-5)

---

## Escherichia coli adhesion portion FimH functions as an adjuvant for cancer immunotherapy

[Source](https://www.nature.com/articles/s41467-020-15030-4)

---

## Temporal changes in gut microbiota profile in children with acute lymphoblastic leukemia prior to commencement-, during-, and post-cessation of chemotherapy

[Source](https://bmccancer.biomedcentral.com/articles/10.1186/s12885-020-6654-5)

---

## Interplay between Dietary Polyphenols and Oral and Gut Microbiota in the Development of Colorectal Cancer

[Source](https://www.mdpi.com/2072-6643/12/3/625)

---

## Short-term supplementation of celecoxib-shifted butyrate production on a simulated model of the gut microbial ecosystem and ameliorated in vitro inflammation

[Source](https://www.nature.com/articles/s41522-020-0119-0)

---

## Crosstalk Between Gut Microbiota and Innate Immunity and Its Implication in Autoimmune Diseases

[Source](https://www.frontiersin.org/articles/10.3389/fimmu.2020.00282/full)

---

## An empirically derived method for measuring human gut microbiome alpha diversity: Demonstrated utility in predicting health-related outcomes among a human clinical sample

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229204)

---

## Proof-of-concept study of the efficacy of a microbiota-directed complementary food formulation (MDCF) for treating moderate acute malnutrition

[Source](https://bmcpublichealth.biomedcentral.com/articles/10.1186/s12889-020-8330-8)

---

## Microbiota as Predictor of Mortality in Allogeneic Hematopoietic-Cell Transplantation

[Source](https://www.nejm.org/doi/10.1056/NEJMoa1900623)

---

## Age-Related Variation of Bacterial and Fungal Communities in Different Body Habitats across the Young, Elderly, and Centenarians in Sardinia

[Source](https://msphere.asm.org/content/msph/5/1/e00558-19.full.pdf)

---

## Response of the human gut and saliva microbiome to urbanization in Cameroon

[Source](https://www.nature.com/articles/s41598-020-59849-9)

---

## Therapeutic Effects of the In Vitro Cultured Human Gut Microbiota as Transplants on Altering Gut Microbiota and Improving Symptoms Associated with Autism Spectrum Disorder

[Source](https://link.springer.com/article/10.1007/s00248-020-01494-w)

---

## A preliminary study on the potential of Nanopore MinION and Illumina MiSeq 16S rRNA gene sequencing to characterize building-dust microbiomes

[Source](https://www.nature.com/articles/s41598-020-59771-0)

---

## Spatial metabolomics of in situ hostâ€“microbe interactions at the micrometre scale

[Source](https://www.nature.com/articles/s41564-019-0664-6)

---

## Global chemical effects of the microbiome include new bile-acid conjugations

[Source](https://www.nature.com/articles/s41586-020-2047-9)

---

## PM2.5 exposure perturbs lung microbiome and its metabolic profile in mice

[Source](https://www.sciencedirect.com/science/article/pii/S0048969720309426)

---

## Growth hormone deficiency and excess alter the gut microbiome in adult male mice

[Source](https://academic.oup.com/endo/advance-article-abstract/doi/10.1210/endocr/bqaa026/5758063)

---

## Dissociation of Adaptive Thermogenesis from Glucose Homeostasis in Microbiome-Deficient Mice

[Source](https://www.cell.com/cell-metabolism/pdf/S1550-4131(20)30013-9.pdf)

---

## Fecal IgA Levels Are Determined by Strain-Level Differences in Bacteroides ovatus and Are Modifiable by Gut Microbiota Manipulation

[Source](https://www.sciencedirect.com/science/article/pii/S193131282030055X)

---

## Can rumen bacteria communicate to each other?

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00796-y)

---

## Entomopathogenic nematode-associated microbiota: from monoxenic paradigm to pathobiome

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00800-5)

---

## The canine oral microbiome: variation in bacterial populations across different niches

[Source](https://link.springer.com/article/10.1186/s12866-020-1704-3)

---

## Evolutionary analysis of Mycobacterium bovis genotypes across Africa suggests co-evolution with livestock and humans

[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0008081)

---

## Nanopore sequencing of microbial communities reveals the potential role of sea lice as a reservoir for fish pathogens

[Source](https://www.nature.com/articles/s41598-020-59747-0)

---

## Insights on aquatic microbiome of the Indian Sundarbans mangrove areas

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0221543)

---

## Ganoderma lucidum cultivation affect microbial community structure of soil, wood segments and tree roots.
 05.03.2020, _F Ren, Y Zhang, H Yu, YA Zhang_


The popular medicinal mushroom Ganoderma lucidum (Fr.) Karst. [Ling Zhi] has been widely used for the general promotion of health and longevity in Asian countries. Continuous cultivation may affect soil microbe and soil properties. However, the effect of G. lucidum cultivation on related wood segments, soil and tree roots microbial communities and soil properties is remain unknown. In our study, the microbial communities of soils, wood segments, and tree roots before and after G. lucidum cultivation were investigated by Illumina Miseq sequencing of both ITS and 16S rDNA, and taxonomic composition of eukaryotic and prokaryotic microorganisms were observed. Indices of microbial richness, diversity and evenness significantly differed between before and after G. lucidum cultivation. Each of the investigated sampling type harbored a distinctive microbial community and differed remarkably before and after G. lucidum cultivation. Ascomycota and Basidiomycota (fungi), Proteobacteria and Actinobacteria (bacteria) showed significant differences after Ling Zhi cultivation. The soil property values also changed after cultivation. The redundancy analysis (RDA) showed that both the fungal and bacterial community structure significantly correlated with soil humus, pH, nitrogen, carbon and trace elements (Fe, Zn, Mn, Cu) contents. The results indicated that G. lucidum cultivation may have significant differed the associated microbial community structures and soil properties. The study will provide useful information for G. lucidum cultivation and under-forest economic development.

[Source](https://www.nature.com/articles/s41598-020-60362-2)

10.1038/s41598-020-60362-2

---

## Effect of long-term destocking on soil fungal functional groups and interactions with plants

[Source](https://link.springer.com/article/10.1007/s11104-020-04452-0)

---

## Comparative Analysis of Root Microbiomes of Rice Cultivars with High and Low Methane Emissions Reveals Differences in Abundance of Methanogenic Archaea and Putative Upstream Fermenters.
 27.02.2020, _Z Liechty, C Santos-MedellÃ­n, J Edwards, B Nguyen, D Mikhail, S Eason, G Phillips, V Sundaresan_


Rice cultivation worldwide accounts for âˆ¼7 to 17% of global methane emissions. Methane cycling in rice paddies is a microbial process not only involving methane producers (methanogens) and methane metabolizers (methanotrophs) but also other microbial taxa that affect upstream processes related to methane metabolism. Rice cultivars vary in their rates of methane emissions, but the influence of rice genotypes on methane cycling microbiota has been poorly characterized. Here, we profiled the rhizosphere, rhizoplane, and endosphere microbiomes of a high-methane-emitting cultivar (Sabine) and a low-methane-emitting cultivar (CLXL745) throughout the growing season to identify variations in the archaeal and bacterial communities relating to methane emissions. The rhizosphere of the high-emitting cultivar was enriched in methanogens compared to that in the low emitter, whereas the relative abundances of methanotrophs between the cultivars were not significantly different. Further analysis of cultivar-sensitive taxa identified families enriched in the high emitter that are associated with methanogenesis-related processes. The high emitter had greater relative abundances of sulfate-reducing and iron-reducing taxa which peak earlier in the season than methanogens and are necessary to lower soil oxidation reduction potential before methanogenesis can occur. The high emitter also had a greater abundance of fermentative taxa which produce methanogenesis precursors (acetate, CO2, and H2). Furthermore, the high emitter was enriched in taxa related to acetogenesis which compete with methanogens for CO2 and H2 These taxa were enriched in a spatio-specific manner and reveal a complex network of microbial interactions on which plant genotype-dependent factors can act to affect methanogenesis and methane emissions.IMPORTANCE Rice cultivation is a major source of anthropogenic emissions of methane, a greenhouse gas with a potentially severe impact on climate change. Emission variation between rice cultivars suggests the feasibility of breeding low-emission rice, but there is a limited understanding of how genotypes affect the microbiota involved in methane cycling. Here, we show that the root microbiome of the high-emitting cultivar is enriched both in methanogens and in taxa associated with fermentation, iron, and sulfate reduction and acetogenesis, processes that support methanogenesis. Understanding how cultivars affect microbes with methanogenesis-related functions is vital for understanding the genetic basis for methane emission in rice and can aid in the development of breeding programs that reduce the environmental impact of rice cultivation.

[Source](https://msystems.asm.org/content/msys/5/1/e00897-19.full.pdf)

10.1128/mSystems.00897-19

---

## Fungal diversity and community composition of wheat rhizosphere and non-rhizosphere soils from three different agricultural production regions of South Africa

[Source](https://www.sciencedirect.com/science/article/pii/S0929139319309485)

---

## Microbial and volatile profiling of soils suppressive to Fusarium culmorum of wheat.
 19.02.2020, _A Ossowicki, V Tracanna, MLC Petrus, G van Wezel, JM Raaijmakers, MH Medema, P Garbeva_


In disease-suppressive soils, microbiota protect plants from root infections. Bacterial members of this microbiota have been shown to produce specific molecules that mediate this phenotype. To date, however, studies have focused on individual suppressive soils and the degree of natural variability of soil suppressiveness remains unclear. Here, we screened a large collection of field soils for suppressiveness to Fusarium culmorum using wheat (Triticum aestivum) as a model host plant. A high variation of disease suppressiveness was observed, with 14% showing a clear suppressive phenotype. The microbiological basis of suppressiveness to F. culmorum was confirmed by gamma sterilization and soil transplantation. Amplicon sequencing revealed diverse bacterial taxonomic compositions and no specific taxa were found exclusively enriched in all suppressive soils. Nonetheless, co-occurrence network analysis revealed that two suppressive soils shared an overrepresented bacterial guild dominated by various Acidobacteria. In addition, our study revealed that volatile emission may contribute to suppression, but not for all suppressive soils. Our study raises new questions regarding the possible mechanistic variability of disease-suppressive phenotypes across physico-chemically different soils. Accordingly, we anticipate that larger-scale soil profiling, along with functional studies, will enable a deeper understanding of disease-suppressive microbiomes.

[Source](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2019.2527#d3e1244)

10.1098/rspb.2019.2527

---

## Industrial development as a key factor explaining variances in soil and grass phyllosphere microbiomes in urban green spaces.
 02.03.2020, _ZZ Yan, QL Chen, YJ Zhang, JZ He, HW Hu_


Microbiota in urban green spaces underpin ecosystem services that are essential to environmental health and human wellbeing. However, the factors shaping the microbial communities in urban green spaces, especially those associated with turf grass phyllosphere, remain poorly understood. The lack of this knowledge greatly limits our ability to assess ecological, social and recreational benefits of urban green spaces in the context of global urbanization. In this study, we used amplicon sequencing to characterize soil and grass phyllosphere bacterial communities in 40 urban green spaces and three minimally disturbed national parks in Victoria, Australia. The results indicated that urbanization might have shown different impacts on soil and grass phyllosphere microbial communities. The bacterial diversity in soil but not in grass phyllosphere was significantly higher in urban green spaces than in national parks. Principal coordinate analysis revealed significant differences in the overall patterns of bacterial community composition between urban green spaces and national parks for both soil and grass phyllosphere. Industrial development, as represented by the number of industries in the region, was identified as a key driver shaping the bacterial community profiles in urban green spaces. Variation partitioning analysis suggested that industrial factors together with their interaction with other factors explained 20% and 28% of the variances in soil and grass phyllosphere bacterial communities, respectively. The findings highlight the importance of industrial development in driving the spatial patterns of urban microbiomes, and have important implication for the management of microbiomes in urban green spaces.

[Source](https://www.sciencedirect.com/science/article/pii/S026974911937558X)

10.1016/j.envpol.2020.114201

---

## Plant communities mediate the interactive effects of invasion and drought on soil microbial communities.
 20.02.2020, _C Fahey, A Koyama, PM Antunes, K Dunfield, SL Flory_


Soil microbiomes could play a major role in ecosystem responses to escalating anthropogenic global change. However, we currently have a poor understanding of how soil microbes will respond to interacting global change factors and if responses will be mediated by changes in plant community structure. We used a field experiment to assess changes in soil fungal and bacterial communities in response to plant invasion, experimental drought, and their combination. In addition, we evaluated the relative importance of direct versus indirect pathways of invasion and drought through changes in associated plant communities with structural equation models. We found that fungal communities were interactively structured by invasion and drought, where fungal richness was lowest with invasion under ambient conditions but highest with invasion under drought conditions. Bacterial richness was lower under drought but unaffected by invasion. Changes in the plant community, including lower plant richness and higher root biomass, moderated the direct effects of invasion on microbial richness. Fungal and bacterial functional groups, including pathogens, mutualists, and nitrogen metabolizers, were also influenced by plant community changes. In sum, plant communities mediated the effects of interacting global change drivers on soil microbial community structure, with significant potential consequences for community dynamics and ecosystem functions.

[Source](https://www.nature.com/articles/s41396-020-0614-6)

10.1038/s41396-020-0614-6

---

## Methyl t-butyl ether-degrading bacteria for bioremediation and biocontrol purposes.
 28.02.2020, _G d'Errico, V Aloj, V Ventorino, A Bottiglieri, E Comite, A Ritieni, R Marra, S Bolletti Censi, GR Flematti, O Pepe, F Vinale_


A total of fifteen potential methyl t-butyl ether (MtBE)-degrading bacterial strains were isolated from contaminated soil. They have been identified as belonging to the genera Bacillus, Pseudomonas, Kocuria, Janibacter, Starkeya, Bosea, Mycolicibacterium, and Rhodovarius. Bacillus aryabhattai R1B, S. novella R8b, and M. mucogenicum R8i were able to grow using MtBE as carbon source, exhibiting different growth behavior and contaminant degradation ability. Their biocontrol ability was tested against various fungal pathogens. Both S. novella R8b and B. aryabhattai were effective in reducing the development of necrotic areas on leaves within 48 hours from Botritys cinerea and Alternaria alternata inoculation. Whereas, M. mucogenicum effectively controlled B. cinerea after 72 hours. Similar results were achieved using Pythium ultimum, in which the application of isolated bacteria increased seed germination. Only M. mucogenicum elicited tomato plants resistance against B. cinerea. This is the first report describing the occurrence of bioremediation and biocontrol activities in M. mucogenicum, B. aryabhattai and S. novella species. The production of maculosin and its antibiotic activity against Rhizoctonia solani has been reported for first time from S. novella. Our results highlight the importance of multidisciplinary approaches to achieve a consistent selection of bacterial strains useful for plant protection and bioremediation purposes.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0228936)

10.1371/journal.pone.0228936

---

## Uncovering microbial responses to sharp geochemical gradients in a terrace contaminated by acid mine drainage

[Source](https://www.sciencedirect.com/science/article/pii/S0269749119368253)

---

## SnapShot: Microbial Extremophiles.
 24.02.2020, _AK Schmid, T Allers, J DiRuggiero_


Extremophiles are remarkable examples of life's resilience, thriving in hot springs at boiling temperatures, in brine lakes saturated with salt, and in the driest deserts. We review the biogeography, currently known limits of life, and molecular adaptations to extremes. See the online interactive map for additional detail on biogeography, environmental microbiology, and exemplary species. To view this SnapShot, open or download the PDF.

[Source](https://www.cell.com/cell/pdf/S0092-8674(20)30099-4.pdf)

10.1016/j.cell.2020.01.018

---

## Denitrification performance and microbial communities of solid-phase denitrifying reactors using poly (butylene succinate)/bamboo powder composite.
 05.03.2020, _W Qi, MJ Taherzadeh, Y Ruan, Y Deng, JS Chen, HF Lu, XY Xu_


This study explored the denitrification performance of solid-phase denitrification (SPD) systems packed with poly (butylene succinate)/bamboo powder composite to treat synthetic aquaculture wastewater under different salinity conditions (0â€° Vs. 25â€°). The results showed composite could achieve the maximum denitrification rates of 0.22 kg (salinity, 0â€°) and 0.34 kg NO3--N m-3 d-1 (salinity, 25â€°) over 200-day operation. No significant nitrite accumulation and less dissolved organic carbon (DOC) release (&lt;15 mg/L) were found. The morphological and spectroscopic analyses demonstrated the mixture composites degradation. Microbial community analysis showed that Acidovorax, Simplicispira, Denitromonas, SM1A02, Marinicella and Formosa were the dominant genera for denitrifying bacteria, while Aspergillus was the major genus for denitrifying fungus. The co-network analysis also indicated the interactions between bacterial and fungal community played an important role in composite degradation and denitrification. The outcomes provided a potential strategy of DOC control and cost reduction for aquaculture nitrate removal by SPD.

[Source](https://www.sciencedirect.com/science/article/pii/S0960852420303023)

10.1016/j.biortech.2020.123033

---

## Meta-omics Provides Insights into the Impact of Hydrocarbon Contamination on Microbial Mat Functioning.
 20.02.2020, _J AubÃ©, P Senin, P Bonin, O Pringault, C Jeziorski, O Bouchez, C Klopp, R Guyoneaud, M GoÃ±i-Urriza_


Photosynthetic microbial mats are stable, self-supported communities. Due to their coastal localization, these mats are frequently exposed to hydrocarbon contamination and are able to grow on it. To decipher how this contamination disturbs the functioning of microbial mats, we compared two mats: a contaminated mat exposed to chronic petroleum contamination and a reference mat. The taxonomic and metabolic structures of the mats in spring and fall were determined using metagenomic and metatranscriptomic approaches. Extremely high contamination disturbed the seasonal variations of the mat. ABC transporters, two-component systems, and type IV secretion system-related genes were overabundant in the contaminated mats. Xenobiotic degradation metabolism was minor in the metagenomes of both mats, and only the expression of genes involved in polycyclic aromatic hydrocarbon degradation was higher in the contaminated mat. Interestingly, the expression rates of genes involved in hydrocarbon activation decreased during the 1-year study period, concomitant with the decrease in easily degradable hydrocarbons, suggesting a transient effect of hydrocarbon contamination. Alteromonadales and Oceanospirillales hydrocarbonoclastic bacteria appeared to be key in hydrocarbon remediation in the contaminated mat. Overall, the contaminated microbial mat was able to cope with hydrocarbon contamination and displayed an adaptive functioning that modified seasonal behaviour.

[Source](https://link.springer.com/article/10.1007/s00248-020-01493-x)

10.1007/s00248-020-01493-x

---

## Role of microbial diversity for sustainable pyrite oxidation control in acid and metalliferous drainage prevention.
 05.03.2020, _OT Ogbughalu, S Vasileiadis, RC Schumann, AR Gerson, J Li, RSC Smart, MD Short_


Acid and metalliferous drainage (AMD) remains a challenging issue for the mining sector. AMD management strategies have attempted to shift from treatment of acid leachates post-generation to more sustainable at-source prevention. Here, the efficacy of microbial-geochemical at-source control approach was investigated over a period of 84 weeks. Diverse microbial communities were stimulated using organic carbon amendment in a simulated silicate-containing sulfidic mine waste rock environment. Mineral waste in the unamended leach system generated AMD quickly and throughout the study, with known lithotrophic iron- and sulfur-oxidising microbes dominating column communities. The organic-amended mineral waste column showed suppressed metal dissolution and AMD generation. Molecular DNA-based next generation sequencing confirmed a less diverse lithotrophic community in the acid-producing control, with a more diverse microbial community under organic amendment comprising organotrophic iron/sulfur-reducers, autotrophs, hydrogenotrophs and heterotrophs. Time-series multivariate statistical analyses displayed distinct ecological patterns in microbial diversity between AMD- and non-AMD-environments. Focused ion beam-TEM micrographs and elemental mapping showed that silicate-stabilised passivation layers were successfully established across pyrite surfaces in organic-amended treatments, with these layers absent in unamended controls. Organic amendment and resulting increases in microbial abundance and diversity played an important role in sustaining these passivating layers in the long-term.

[Source](https://www.sciencedirect.com/science/article/pii/S0304389420303265)

10.1016/j.jhazmat.2020.122338

---

## Critical care management of adults with community-acquired severe respiratory viral infection.
 28.02.2020, _YM Arabi, R Fowler, FG Hayden_


With the expanding use of molecular assays, viral pathogens are increasingly recognized among critically ill adult patients with community-acquired severe respiratory illness; studies have detected respiratory viral infections (RVIs) in 17-53% of such patients. In addition, novel pathogens including zoonotic coronaviruses like the agents causing Severe Acute Respiratory Syndrome (SARS), Middle East Respiratory Syndrome (MERS) and the 2019 novel coronavirus (2019 nCoV) are still being identified. Patients with severe RVIs requiring ICU care present typically with hypoxemic respiratory failure. Oseltamivir is the most widely used neuraminidase inhibitor for treatment of influenza; data suggest that early use is associated with reduced mortality in critically ill patients with influenza. At present, there are no antiviral therapies of proven efficacy for other severe RVIs. Several adjunctive pharmacologic interventions have been studied for their immunomodulatory effects, including macrolides, corticosteroids, cyclooxygenase-2 inhibitors, sirolimus, statins, anti-influenza immune plasma, and vitamin C, but none is recommended at present in severe RVIs. Evidence-based supportive care is the mainstay for management of severe respiratory viral infection. Non-invasive ventilation in patients with severe RVI causing acute hypoxemic respiratory failure and pneumonia is associated with a high likelihood of transition to invasive ventilation. Limited existing knowledge highlights the need for data regarding supportive care and adjunctive pharmacologic therapy that is specific for critically ill patients with severe RVI. There is a need for more pragmatic and efficient designs to test different therapeutics both individually and in combination.

[Source](https://www.biomedcentral.com/collections/coronavirus)

10.1007/s00134-020-05943-5

---

## Coronavirus research highlights

[Source](https://www.biomedcentral.com/collections/coronavirus)

---

## Fecal Viral Community Responses to High-Fat Diet in Mice.
 27.02.2020, _A Schulfer, TM Santiago-Rodriguez, M Ly, JM Borin, J Chopyk, MJ Blaser, DT Pride_


Alterations in diet can have significant impact on the host, with high-fat diet (HFD) leading to obesity, diabetes, and inflammation of the gut. Although membership and abundances in gut bacterial communities are strongly influenced by diet, substantially less is known about how viral communities respond to dietary changes. Examining fecal contents of mice as the mice were transitioned from normal chow to HFD, we found significant changes in the relative abundances and the diversity in the gut of bacteria and their viruses. Alpha diversity of the bacterial community was significantly diminished in response to the diet change but did not change significantly in the viral community. However, the diet shift significantly impacted the beta diversity in both the bacterial and viral communities. There was a significant shift away from the relatively abundant Siphoviridae accompanied by increases in bacteriophages from the Microviridae family. The proportion of identified bacteriophage structural genes significantly decreased after the transition to HFD, with a conserved loss of integrase genes in all four experimental groups. In total, this study provides evidence for substantial changes in the intestinal virome disproportionate to bacterial changes, and with alterations in putative viral lifestyles related to chromosomal integration as a result of shift to HFD.IMPORTANCE Prior studies have shown that high-fat diet (HFD) can have profound effects on the gastrointestinal (GI) tract microbiome and also demonstrate that bacteria in the GI tract can affect metabolism and lean/obese phenotypes. We investigated whether the composition of viral communities that also inhabit the GI tract are affected by shifts from normal to HFD. We found significant and reproducible shifts in the content of GI tract viromes after the transition to HFD. The differences observed in virome community membership and their associated gene content suggest that these altered viral communities are populated by viruses that are more virulent toward their host bacteria. Because HFD also are associated with significant shifts in GI tract bacterial communities, we believe that the shifts in the viral community may serve to drive the changes that occur in associated bacterial communities.

[Source](https://msphere.asm.org/content/msph/5/1/e00833-19.full.pdf)

10.1128/mSphere.00833-19

---

## Rapid MinION profiling of preterm microbiota and antimicrobial-resistant pathogens.
 05.03.2020, _RM Leggett, C Alcon-Giner, D Heavens, S Caim, TC Brook, M Kujawska, S Martin, N Peel, H Acford-Palmer, L Hoyles, P Clarke, LJ Hall, MD Clark_


The MinION sequencing platform offers near real-time analysis of DNA sequence; this makes the tool attractive for deployment in fieldwork or clinical settings. We used the MinION platform coupled to the NanoOK RT software package to perform shotgun metagenomic sequencing and profile mock communities and faecal samples from healthy and ill preterm infants. Using Nanopore data, we reliably classified a 20-species mock community and captured the diversity of the immature gut microbiota over time and in response to interventions such as probiotic supplementation, antibiotic treatment or episodes of suspected sepsis. We also performed rapid real-time runs to assess gut-associated microbial communities in critically ill and healthy infants, facilitated by NanoOK RT software package, which analysed sequences as they were generated. Our pipeline reliably identified pathogenic bacteria (that is, Klebsiella pneumoniae and Enterobacter cloacae) and their corresponding antimicrobial resistance gene profiles within as little as 1 h of sequencing. Results were confirmed using pathogen isolation, whole-genome sequencing and antibiotic susceptibility testing, as well as mock communities and clinical samples with known antimicrobial resistance genes. Our results demonstrate that MinION (including cost-effective Flongle flow cells) with NanoOK RT can process metagenomic samples to a rich dataset in &lt; 5 h, which creates a platform for future studies aimed at developing these tools and approaches in clinical settings with a focus on providing tailored patient antimicrobial treatment options.

[Source](https://www.nature.com/articles/s41564-019-0626-z)

10.1038/s41564-019-0626-z

---

## Assembly-free single-molecule sequencing recovers complete virus genomes from natural microbial communities.
 20.02.2020, _J Beaulaurier, E Luo, JM Eppley, P Den Uyl, X Dai, A Burger, DJ Turner, M Pendleton, S Juul, E Harrington, EF DeLong_


Viruses are the most abundant biological entities on Earth and play key roles in host ecology, evolution, and horizontal gene transfer. Despite recent progress in viral metagenomics, the inherent genetic complexity of virus populations still poses technical difficulties for recovering complete virus genomes from natural assemblages. To address these challenges, we developed an assembly-free, single-molecule nanopore sequencing approach enabling direct recovery of complete viral genome sequences from environmental samples. Our method yielded thousands of full-length, high-quality draft virus genome sequences that were not recovered using standard short-read assembly approaches. Additionally, our analyses discriminated between populations whose genomes had identical direct terminal repeats, versus those with circularly permuted repeats at their termini, thus providing new insight into native virus reproduction and genome packaging. Novel DNA sequences were discovered whose repeat structures, gene contents, and concatemer lengths suggest they are phage-inducible chromosomal islands, which are packaged as concatemers in phage particles, with lengths that match the size ranges of co-occurring phage genomes. Our new virus sequencing strategy can provide previously unavailable information about the genome structures, population biology, and ecology of naturally occurring viruses and viral parasites.

[Source](https://genome.cshlp.org/content/early/2020/02/19/gr.251686.119.abstract)

10.1101/gr.251686.119

---

## Rapid inference of antibiotic resistance and susceptibility by genomic neighbour typing.
 05.03.2020, _K BÅ™inda, A Callendrello, KC Ma, DR MacFadden, T Charalampous, RS Lee, L Cowley, CB Wadsworth, YH Grad, G Kucherov, J O'Grady, M Baym, WP Hanage_


Surveillance of drug-resistant bacteria is essential for healthcare providers to deliver effective empirical antibiotic therapy. However, traditional molecular epidemiology does not typically occur on a timescale that could affect patient treatment and outcomes. Here, we present a method called 'genomic neighbour typing' for inferring the phenotype of a bacterial sample by identifying its closest relatives in a database of genomes with metadata. We show that this technique can infer antibiotic susceptibility and resistance for both Streptococcus pneumoniae and Neisseria gonorrhoeae. We implemented this with rapid k-mer matching, which, when used on Oxford Nanopore MinION data, can run in real time. This resulted in the determination of resistance within 10 min (91% sensitivity and 100% specificity for S. pneumoniae and 81% sensitivity and 100% specificity for N. gonorrhoeae from isolates with a representative database) of starting sequencing, and within 4 h of sample collection (75% sensitivity and 100% specificity for S. pneumoniae) for clinical metagenomic sputum samples. This flexible approach has wide application for pathogen surveillance and may be used to greatly accelerate appropriate empirical antibiotic treatment.

[Source](https://www.nature.com/articles/s41564-019-0656-6)

10.1038/s41564-019-0656-6

---

## Tough microbe's DNA could be a lasting archive.
 03.03.2020, _S Nadis_



[Source](https://science.sciencemag.org/content/367/6480/840)

10.1126/science.367.6480.840

---

## Inflammatory bowel disease is associated with increased gut-to-blood penetration of short-chain fatty acids: A new, non-invasive marker of a functional intestinal lesion.
 01.08.2019, _K Jaworska, M Konop, K Bielinska, T Hutsch, M Dziekiewicz, A Banaszkiewicz, M Ufnal_


What is the central question of this study? 'Leaky gut' has been found in intestinal and extra-intestinal diseases. However, functional evaluation of intestinal permeability is not widely used as a diagnostic marker, possibly owing to significant limitations of currently used permeability assays. There is an unmet need for development of a new, non-invasive test to assess intestinal function. What is the main finding and its importance? We show that an increased blood-to-stool ratio of the concentration of gut bacteria-produced short-chain fatty acids may be used as a marker of gut permeability. Our findings lay the groundwork for establishing a new, non-invasive, risk-free diagnostic tool in diseases associated with intestinal barrier malfunction, such as inflammatory bowel disease.

[Source](https://www.kpbs.org/news/2020/feb/24/how-bacteria-can-sculpt-gut/)

10.1113/EP087773

---

## How Bacteria Can â€˜Sculpt The Gutâ€™

[Source](https://www.kpbs.org/news/2020/feb/24/how-bacteria-can-sculpt-gut/)

---

## The bee microbiome can fight back against fungi that cause Colony Collapse Disorder

[Source](https://massivesci.com/articles/bacteria-bees-colony-collapse-disorder-pathogens-biocontrol/)

---

## Ancient gut microbiomes shed light on human evolution

[Source](https://www.newswise.com/articles/ancient-gut-microbiomes-shed-light-on-human-evolution)

---

## Launch of the Marine Microbiome Roadmap

[Source](https://www.nioz.nl/en/news/launch-of-the-marine-microbiome-roadmap)

---

## Understanding Microbiome Resilience on a Changing Planet

[Source](https://vp.research.msu.edu/feature/understanding-microbiome-resilience-changing-planet)

---

## Adaptations in gut microbes helped our ancestors thrive in new geographical territories: Study

[Source](https://www.newkerala.com/news/2020/29079.htm)

---

## Scientists Announce VIRGO, First Comprehensive Gene Catalog To Decode the Vaginal Microbiome

[Source](https://www.prnewswire.com/news-releases/scientists-announce-virgo-first-comprehensive-gene-catalog-to-decode-the-vaginal-microbiome-301011341.html)

---

## Metabolomics Reveals Connection Between Gut Microbiome and Blood

[Source](https://www.metabolon.com/who-we-are/blog/metabolomics-reveals-connection-between-gut-microbiome-and-blood)

---

## Study shows universally positive effect of cover crops on soil microbiome

[Source](https://phys.org/news/2020-02-universally-positive-effect-crops-soil.html)

---

## New Center of Microbiome Science is in the works at Ohio State

[Source](https://news.osu.edu/new-center-of-microbiome-science-is-in-the-works-at-ohio-state/)

---

## Common diseases may be communicable through microbiome, says UBC researcher

[Source](https://www.ubyssey.ca/science/microbiome-communicable-diseases/)

---

## The microbes in your mouth, and a reminder to floss and go to the dentist

[Source](https://www.sciencedaily.com/releases/2020/03/200302162248.htm)

---

## For Your Microbiome, How You Eat Could Be As Important As What You Eat

[Source](https://www.technologynetworks.com/immunology/news/for-your-microbiome-how-you-eat-could-be-as-important-as-what-you-eat-331627)

---

## Study: Infectious gut bacteria may predict UTI risk

[Source](https://news.cornell.edu/stories/2020/02/study-infectious-gut-bacteria-may-predict-uti-risk)

---

## The Best Way to Avoid Getting Sick Is to Wash Your Hands â€” But Youâ€™re Probably Doing It Wrong

[Source](https://www.livestrong.com/article/13725173-how-and-when-to-wash-your-hands/)

---

## DuPont carries on microbiome momentum with metabolic disease collaboration

[Source](https://www.nutraingredients.com/Article/2020/02/19/DuPont-enters-microbiome-metabolic-disease-collaboration#)

---

## Invest 5% of research funds in ensuring data are reusable

[Source](https://www.nature.com/articles/d41586-020-00505-7)

---

