## iNetModels 2.0: an interactive visualization and database of multi-omics data.
 13.04.2021, _M Arif, C Zhang, X Li, C Güngör, B Çakmak, M Arslantürk, A Tebani, B Özcan, O Subaş, W Zhou, B Piening, H Turkez, L Fagerberg, N Price, L Hood, M Snyder, J Nielsen, M Uhlen, A Mardinoglu_


It is essential to reveal the associations between various omics data for a comprehensive understanding of the altered biological process in human wellness and disease. To date, very few studies have focused on collecting and exhibiting multi-omics associations in a single database. Here, we present iNetModels, an interactive database and visualization platform of Multi-Omics Biological Networks (MOBNs). This platform describes the associations between the clinical chemistry, anthropometric parameters, plasma proteomics, plasma metabolomics, as well as metagenomics for oral and gut microbiome obtained from the same individuals. Moreover, iNetModels includes tissue- and cancer-specific Gene Co-expression Networks (GCNs) for exploring the connections between the specific genes. This platform allows the user to interactively explore a single feature's association with other omics data and customize its particular context (e.g. male/female specific). The users can also register their data for sharing and visualization of the MOBNs and GCNs. Moreover, iNetModels allows users who do not have a bioinformatics background to facilitate human wellness and disease research. iNetModels can be accessed freely at https://inetmodels.com without any limitation.

[Source](https://science.sciencemag.org/content/371/6536/eabc4552)

10.1093/nar/gkab254

---

## The microbiome and human cancer

[Source](https://science.sciencemag.org/content/371/6536/eabc4552)

---

## A framework for microbiome science in public health

[Source](https://www.nature.com/articles/s41591-021-01258-0)

---

## Niche and local geography shape the pangenome of wastewater- and livestock-associated Enterobacteriaceae

[Source](https://advances.sciencemag.org/content/7/15/eabe3868)

---

## Immune System, Microbiota, and Microbial Metabolites: The Unresolved Triad in Colorectal Cancer Microenvironment

[Source](https://www.frontiersin.org/articles/10.3389/fimmu.2021.612826/full)

---

## Microbiome and colorectal cancer: A review of the past, present, and future

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0960740421000499)

---

## Microbiota-derived short chain fatty acids modulate microglia and promote Aβ plaque deposition

[Source](https://elifesciences.org/articles/59826)

---

## Intestinal microbiota drives cholestasis-induced specific hepatic gene expression patterns

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2021.1911534)

---

## When your host shuts down: larval diapause impacts host-microbiome interactions in Nasonia vitripennis

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01037-6)

---

## Gut microbiota-derived inosine from dietary barley leaf supplementation attenuates colitis through PPARγ signaling activation

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01028-7)

---

## Important ecophysiological roles of non-dominant Actinobacteria in plant residue decomposition, especially in less fertile soils

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01032-x)

---

## Rhizosphere community selection reveals bacteria associated with reduced root disease

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00997-5)

---

## Shotgun metagenomics reveals distinct functional diversity and metabolic capabilities between 12 000-year-old permafrost and active layers on Muot da Barba Peider (Swiss Alps)

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000558)

---

## The Polar Night Shift: Annual Dynamics and Drivers of Microbial Community Structure in the Arctic Ocean

[Source](https://www.biorxiv.org/content/10.1101/2021.04.08.436999v2.abstract)

---

## Seafloor incubation experiment with deep-sea hydrothermal vent fluid reveals effect of pressure and lag time on autotrophic microbial communities.
 20.02.2021, _CS Fortunato, DA Butterfield, B Larson, N Lawrence-Slavas, CK Algar, L Zeigler Allen, JF Holden, G Proskurowski, E Reddington, LC Stewart, BD Topçuoğlu, JJ Vallino, JA Huber_


Depressurization and sample processing delays may impact the outcome of shipboard microbial incubations of samples collected from the deep sea. To address this knowledge gap, we developed an ROV-powered incubator instrument to carry out and compare results from in situ and shipboard RNA Stable Isotope Probing (RNA-SIP) experiments to identify the key chemolithoautotrophic microbes and metabolisms in diffuse, low-temperature venting fluids from Axial Seamount. All the incubations showed microbial uptake of labeled bicarbonate primarily by thermophilic autotrophic Epsilonbacteraeota that oxidized hydrogen coupled with nitrate reduction. However, the in situ seafloor incubations showed higher abundances of transcripts annotated for aerobic processes suggesting that oxygen was lost from the hydrothermal fluid samples prior to shipboard analysis. Furthermore, transcripts for thermal stress proteins such as heat shock chaperones and proteases were significantly more abundant in the shipboard incubations suggesting that depressurization induced thermal stress in the metabolically active microbes in these incubations. Together, results indicate that while the autotrophic microbial communities in the shipboard and seafloor experiments behaved similarly, there were distinct differences that provide new insight into the activities of natural microbial assemblages under near-native conditions in the ocean.Importance: Diverse microbial communities drive biogeochemical cycles in Earth's ocean, yet studying these organisms and processes is often limited by technological capabilities, especially in the deep ocean. In this study, we used a novel marine microbial incubator instrument capable of in situ experimentation to investigate microbial primary producers at deep-sea hydrothermal vents. We carried out identical stable isotope probing experiments coupled to RNA sequencing both on the seafloor and on the ship to examine thermophilic, microbial autotrophs in venting fluids from an active submarine volcano. Our results indicate that microbial communities were significantly impacted by the effects of depressurization and sample processing delay, with shipboard microbial communities more stressed compared to seafloor incubations. Differences in metabolism were also apparent and are likely linked to the chemistry of the fluid at the beginning of the experiment. Microbial experimentation in the natural habitat provides new insights into understanding microbial activities in the ocean.

[Source](https://aem.asm.org/content/87/9/e00078-21)

10.1128/AEM.00078-21

---

## Seafloor Incubation Experiment with Deep-Sea Hydrothermal Vent Fluid Reveals Effect of Pressure and Lag Time on Autotrophic Microbial Communities

[Source](https://aem.asm.org/content/87/9/e00078-21)

---

## Plasmid fitness costs are caused by specific genetic conflicts

[Source](https://www.biorxiv.org/content/10.1101/2021.04.10.439128v2.abstract)

---

## Comprehensive benchmarking of tools to identify phages in metagenomic shotgun sequencing data

[Source](https://www.biorxiv.org/content/10.1101/2021.04.12.438782v1.abstract)

---

