## The influence of placenta microbiota of normal term pregnant women on immune regulation during pregnancy.
 03.03.2024, _P Yang, T Lu, X Liang, T Huang, L Wu, Z He, X Xiao, S Fan_


The concerted regulation of placenta microbiota and the immune responses secures the occurrence and development of pregnancy, while few studies reported this correlation. This study aimed to explore the relationship between the placenta microbiota and immune regulation during pregnancy.

[Source](https://doi.org/10.1186/s12884-024-06353-x)

10.1186/s12884-024-06353-x

---

## Effect of gut microbiome modulation on muscle function and cognition: the PROMOTe randomised controlled trial.
 03.03.2024, _M Ni Lochlainn, RCE Bowyer, JM Moll, MP García, S Wadge, AF Baleanu, A Nessa, A Sheedy, G Akdag, D Hart, G Raffaele, PT Seed, C Murphy, SDR Harridge, AA Welch, C Greig, K Whelan, CJ Steves_


Studies suggest that inducing gut microbiota changes may alter both muscle physiology and cognitive behaviour. Gut microbiota may play a role in both anabolic resistance of older muscle, and cognition. In this placebo controlled double blinded randomised controlled trial of 36 twin pairs (72 individuals), aged ≥60, each twin pair are block randomised to receive either placebo or prebiotic daily for 12 weeks. Resistance exercise and branched chain amino acid (BCAA) supplementation is prescribed to all participants. Outcomes are physical function and cognition. The trial is carried out remotely using video visits, online questionnaires and cognitive testing, and posting of equipment and biological samples. The prebiotic supplement is well tolerated and results in a changed gut microbiome [e.g., increased relative Bifidobacterium abundance]. There is no significant difference between prebiotic and placebo for the primary outcome of chair rise time (β = 0.579; 95% CI -1.080-2.239 p = 0.494). The prebiotic improves cognition (factor score versus placebo (β = -0.482; 95% CI,-0.813, -0.141; p = 0.014)). Our results demonstrate that cheap and readily available gut microbiome interventions may improve cognition in our ageing population. We illustrate the feasibility of remotely delivered trials for older people, which could reduce under-representation of older people in clinical trials. ClinicalTrials.gov registration: NCT04309292.

[Source](https://doi.org/10.1038/s41467-024-46116-y)

10.1038/s41467-024-46116-y

---

## A metagenomic catalog of the early-life human gut virome.
 03.03.2024, _S Zeng, A Almeida, S Li, J Ying, H Wang, Y Qu, R Paul Ross, C Stanton, Z Zhou, X Niu, D Mu, S Wang_


Early-life human gut microbiome is a pivotal driver of gut homeostasis and infant health. However, the viral component (known as "virome") remains mostly unexplored. Here, we establish the Early-Life Gut Virome (ELGV), a catalog of 160,478 non-redundant DNA and RNA viral sequences from 8130 gut virus-like particles (VLPs) enriched or bulk metagenomes in the first three years of life. By clustering, 82,141 viral species are identified, 68.3% of which are absent in existing databases built mainly from adults, and 64 and 8 viral species based on VLPs-enriched and bulk metagenomes, respectively, exhibit potentials as biomarkers to distinguish infants from adults. With the largest longitudinal population of infants profiled by either VLPs-enriched or bulk metagenomic sequencing, we track the inherent instability and temporal development of the early-life human gut virome, and identify differential viruses associated with multiple clinical factors. The mother-infant shared virome and interactions between gut virome and bacteriome early in life are further expanded. Together, the ELGV catalog provides the most comprehensive and complete metagenomic blueprint of the early-life human gut virome, facilitating the discovery of pediatric disease-virome associations in future.

[Source](https://doi.org/10.1038/s41467-024-45793-z)

10.1038/s41467-024-45793-z

---

## Exploring the gut DNA virome in fecal immunochemical test stool samples reveals associations with lifestyle in a large population-based study

[Source](https://doi.org/10.1038/s41467-024-46033-0)

---

## Bisphenol A exposure affects specific gut taxa and drives microbiota dynamics in childhood obesity.
 01.03.2024, _A Lopez-Moreno, K Cerk, L Rodrigo, A Suarez, M Aguilera, A Ruiz-Rodriguez_


Cumulative xenobiotic exposure has an environmental and human health impact which is currently assessed under the One Health approach. Bisphenol A (BPA) exposure and its potential link with childhood obesity that has parallelly increased during the last decades deserve special attention. It stands during prenatal or early life and could trigger comorbidities and non-communicable diseases along life. Accumulation in the nature of synthetic chemicals supports the "environmental obesogen" hypothesis, such as BPA. This estrogen-mimicking xenobiotic has shown endocrine disruptive and obesogenic effects accompanied by gut microbiota misbalance that is not yet well elucidated. This study aimed to investigate specific microbiota taxa isolated and selected by direct BPA exposure and reveal its role on the overall children microbiota community and dynamics, driving toward specific obesity dysbiosis. A total of 333 BPA-resistant isolated species obtained through culturing after several exposure conditions were evaluated for their role and interplay with the global microbial community. The selected BPA-cultured taxa biomarkers showed a significant impact on alpha diversity. Specifically, Clostridium and Romboutsia were positively associated promoting the richness of microbiota communities, while Intestinibacter, Escherichia-Shigella, Bifidobacterium, and Lactobacillus were negatively associated. Microbial community dynamics and networks analyses showed differences according to the study groups. The normal-weight children group exhibited a more enriched, structured, and connected taxa network compared to overweight and obese groups, which could represent a more resilient community to xenobiotic substances. In this sense, subnetwork analysis generated with the BPA-cultured genera showed a correlation between taxa connectivity and more diverse potential enzymatic BPA degradation capacities.IMPORTANCEOur findings indicate how gut microbiota taxa with the capacity to grow in BPA were differentially represented within differential body mass index children study groups and how these taxa affected the overall dynamics toward patterns of diversity generally recognized in dysbiosis. Community network and subnetwork analyses corroborated the better connectedness and stability profiles for normal-weight group compared to the overweight and obese groups.

[Source](https://doi.org/10.1128/msystems.00957-23)

10.1128/msystems.00957-23

---

## <em>Prevotella</em>, a dominant bacterium in young people with stage Ⅲ periodontitis, related to the arachidonic acid metabolism pathway

[Source](https://doi.org/10.1016/j.micinf.2024.105316)

---

## Dual membrane-spanning anti-sigma factors regulate vesiculation in <em>Bacteroides thetaiotaomicron</em>

[Source](https://doi.org/10.1073/pnas.2321910121)

---

## <em>Blautia coccoides </em>is a Newly Identified Bacterium Increased by Leucine Deprivation and has a Novel Function in Improving Metabolic Disorders

[Source](https://doi.org/10.1002/advs.202309255)

---

## Fecal Microbiota Transplantation in Amyotrophic Lateral Sclerosis: Clinical Protocol and Evaluation of Microbiota Immunity Axis

[Source](https://doi.org/10.1007/978-1-0716-3662-6_27)

---

## Effect of <em>Lactobacillus reuteri</em> NCIMB 30351 drops on symptoms of infantile functional gastrointestinal disorders and gut microbiota in early infants: Results from a randomized, placebo-controlled clinical trial

[Source](https://doi.org/10.1007/s00431-024-05473-y)

---

## Gut microbiota facilitate adaptation of invasive moths to new host plants

[Source](https://doi.org/10.1093/ismejo/wrae031)

---

## A Synthetic Microbiome Based on Dominant Microbes in Wild Rice Rhizosphere to Promote Sulfur Utilization

[Source](https://doi.org/10.1186/s12284-024-00695-y)

---

## Life at the borderlands: microbiomes of interfaces critical to One Health

[Source](https://doi.org/10.1093/femsre/fuae008)

---

## Neural network establishes co-occurrence links between transformation products of the contaminant and the soil microbiome

[Source](https://doi.org/10.1016/j.scitotenv.2024.171287)

---

## Transforming Big Data into AI-ready data for nutrition and obesity research

[Source](https://doi.org/10.1002/oby.23989)

---

## Explainable artificial intelligence for microbiome data analysis in colorectal cancer biomarker identification

[Source](https://doi.org/10.3389/fmicb.2024.1348974)

---

## A metatranscriptomics strategy for efficient characterization of the microbiome in human tissues with low microbial biomass.
 01.03.2024, _J Pereira-Marques, RM Ferreira, C Figueiredo_


The high background of host RNA poses a major challenge to metatranscriptome analysis of human samples. Hence, metatranscriptomics has been mainly applied to microbe-rich samples, while its application in human tissues with low ratio of microbial to host cells has yet to be explored. Since there is no computational workflow specifically designed for the taxonomic and functional analysis of this type of samples, we propose an effective metatranscriptomics strategy to accurately characterize the microbiome in human tissues with a low ratio of microbial to host content. We experimentally generated synthetic samples with well-characterized bacterial and host cell compositions, and mimicking human samples with high and low microbial loads. These synthetic samples were used for optimizing and establishing the workflow in a controlled setting. Our results show that the integration of the taxonomic analysis of optimized Kraken 2/Bracken with the functional analysis of HUMAnN 3 in samples with low microbial content, enables the accurate identification of a large number of microbial species with a low false-positive rate, while improving the detection of microbial functions. The effectiveness of our metatranscriptomics workflow was demonstrated in synthetic samples, simulated datasets, and most importantly, human gastric tissue specimens, thus providing a proof of concept for its applicability on mucosal tissues of the gastrointestinal tract. The use of an accurate and reliable metatranscriptomics approach for human tissues with low microbial content will expand our understanding of the functional activity of the mucosal microbiome, uncovering critical interactions between the microbiome and the host in health and disease.

[Source](https://doi.org/10.1080/19490976.2024.2323235)

10.1080/19490976.2024.2323235

---

## On the limits of 16S rRNA gene-based metagenome prediction and functional profiling.
 01.03.2024, _MS Matchado, M Rühlemann, S Reitmeier, T Kacprowski, F Frost, D Haller, J Baumbach, M List_


_Humans, Metagenome, RNA, Ribosomal, 16S, Genes, rRNA, Microbiota, Algorithms_
Molecular profiling techniques such as metagenomics, metatranscriptomics or metabolomics offer important insights into the functional diversity of the microbiome. In contrast, 16S rRNA gene sequencing, a widespread and cost-effective technique to measure microbial diversity, only allows for indirect estimation of microbial function. To mitigate this, tools such as PICRUSt2, Tax4Fun2, PanFP and MetGEM infer functional profiles from 16S rRNA gene sequencing data using different algorithms. Prior studies have cast doubts on the quality of these predictions, motivating us to systematically evaluate these tools using matched 16S rRNA gene sequencing, metagenomic datasets, and simulated data. Our contribution is threefold: (i) using simulated data, we investigate if technical biases could explain the discordance between inferred and expected results; (ii) considering human cohorts for type two diabetes, colorectal cancer and obesity, we test if health-related differential abundance measures of functional categories are concordant between 16S rRNA gene-inferred and metagenome-derived profiles and; (iii) since 16S rRNA gene copy number is an important confounder in functional profiles inference, we investigate if a customised copy number normalisation with the rrnDB database could improve the results. Our results show that 16S rRNA gene-based functional inference tools generally do not have the necessary sensitivity to delineate health-related functional changes in the microbiome and should thus be used with care. Furthermore, we outline important differences in the individual tools tested and offer recommendations for tool selection.

[Source](https://doi.org/10.1099/mgen.0.001203)

10.1099/mgen.0.001203

---

