## Effects of ultra-processed foods on the microbiota-gut-brain axis: The bread-and-butter issue.
 25.04.2023, _Z Song, R Song, Y Liu, Z Wu, X Zhang_


_Brain-Gut Axis, Food, Processed, Bread, Food Additives, Sugars_
The topic of gut microbiota and the microbiota-gut-brain (MGB) axis has become the forefront of research and reports in the past few years. The gut microbiota is a dynamic interface between the environment, food, and the host, reflecting the health status as well as maintaining normal physiological metabolism. Modern ultra-processed foods (UPF) contain large quantities of saturated and trans fat, added sugar, salt, and food additives that seriously affect the gut and physical health. In addition, these unhealthy components directly cause changes in gut microbiota functions and microbial metabolism, subsequently having the potential to impact the neural network. This paper reviews an overview of the link between UPF ingredients and the MGB axis. Considerable studies have examined that high intake of trans fat, added sugar and salt have deleterious effects on gut and brain functions, but relatively less focus has been placed on the impact of food additives on the MGB axis. Data from several studies suggest that food additives might be linked to metabolic diseases and inflammation. They may also alter the gut microbiota composition and microbial metabolites, which potentially affect cognition and behavior. Therefore, we emphasize that food additives including emulsifiers, artificial sweeteners, colorants, and preservatives interact with the gut microbiota and their possible effects on altering the brain and behavior based on the latest research. Future studies should further investigate whether gut dysbiosis mediates the effect of UPF on brain diseases and behavior. This thesis here sheds new light on future research pointing to the potentially detrimental effects of processed food consumption on brain health.

[Source](https://doi.org/10.1016/j.foodres.2023.112730)

10.1016/j.foodres.2023.112730

---

## The emerging role of the small intestinal microbiota in human health and disease.
 25.04.2023, _RAAA Ruigrok, RK Weersma, A Vich Vila_


_Humans, Gastrointestinal Microbiome, Intestine, Small, Microbiota, Feces_
The human gut microbiota continues to demonstrate its importance in human health and disease, largely owing to the countless number of studies investigating the fecal microbiota. Underrepresented in these studies, however, is the role played by microbial communities found in the small intestine, which, given the essential function of the small intestine in nutrient absorption, host metabolism, and immunity, is likely highly relevant. This review provides an overview of the methods used to study the microbiota composition and dynamics along different sections of the small intestine. Furthermore, it explores the role of the microbiota in facilitating the small intestine in its physiological functions and discusses how disruption of the microbial equilibrium can influence disease development. The evidence suggests that the small intestinal microbiota is an important regulator of human health and its characterization has the potential to greatly advance gut microbiome research and the development of novel disease diagnostics and therapeutics.

[Source](https://doi.org/10.1080/19490976.2023.2201155)

10.1080/19490976.2023.2201155

---

## Gut microbiome can predict chemoradiotherapy efficacy in patients with esophageal squamous cell carcinoma.
 22.04.2023, _T Sasaki, Y Matsumoto, K Murakami, S Endo, T Toyozumi, R Otsuka, K Kinoshita, J Hu, S Iida, H Morishita, Y Nishioka, A Nakano, M Uesato, H Matsubara_


The gut microbiome plays an important role in cancer pathogenesis and therapy. Some studies have reported that specific bacteria in tumor tissues may contribute to the prognosis and treatment of esophageal squamous cell carcinoma (ESCC). However, there is limited evidence that the gut microbiome is associated with ESCC. This study assessed the utility of the gut microbiome as a predictive marker of the therapeutic effect in patients with ESCC undergoing chemo-radiotherapy (CRT).

[Source](https://doi.org/10.1007/s10388-023-01004-0)

10.1007/s10388-023-01004-0

---

## Multi-omics signatures in new-onset diabetes predict metabolic response to dietary inulin: findings from an observational study followed by an interventional trial.
 25.04.2023, _N Ďásková, I Modos, M Krbcová, M Kuzma, H Pelantová, J Hradecký, M Heczková, M Bratová, P Videňská, P Šplíchalová, M Králová, M Heniková, J Potočková, A Ouřadová, R Landberg, T Kühn, M Cahová, J Gojda_


_Humans, Inulin, Diabetes Mellitus, Type 2, Case-Control Studies, Cross-Sectional Studies, Multiomics, Obesity, Overweight_
The metabolic performance of the gut microbiota contributes to the onset of type 2 diabetes. However, targeted dietary interventions are limited by the highly variable inter-individual response. We hypothesized (1) that the composition of the complex gut microbiome and metabolome (MIME) differ across metabolic spectra (lean-obese-diabetes); (2) that specific MIME patterns could explain the differential responses to dietary inulin; and (3) that the response can be predicted based on baseline MIME signature and clinical characteristics.

[Source](https://doi.org/10.1038/s41387-023-00235-5)

10.1038/s41387-023-00235-5

---

## Comparison of Conjunctival Sac Microbiome between Low and High Myopic Eyes.
 21.04.2023, _K Xiao, Z Chen, Q Long_


Microbial communities played a vital role in maintaining homeostasis of ocular surface. However, no studies explored the myopia-associated conjunctiva microbiota changes until now. In this study, conjunctival sac swab specimens were collected from 12 eyes of low myopia (LM), and 14 eyes of high myopia (HM) patients. The V3-V4 region of the 16S rRNA gene was amplified and then sequenced. Statistical analysis was performed to investigate differences in the taxonomy and diversity between two groups. Compared to LM, higher Ocular Surface Disease Index (OSDI) scores were observed in HM group. The Shannon index of the HM was lower than that of the LM group (P = 0.017). Principle coordinate analysis and Partial Least Squares Discrimination Analysis showed distinct microbiome composition between two groups. At the phylum level, there were higher relative abundances of Proteobacteria (68.27% vs 38.51%) and lower abundances of Actinobacteria (3.71% vs 9.19%) in HM, compared to LM group (P = 0.031, 0.010, respectively). At the genus level, the abundances of Acinetobacter in HM (18.16%) were significantly higher than the LM (6.52%) group (P = 0.011). Actinobacteria levels were negatively correlated with the myopic spherical equivalent and OSDI scores. Moreover, positive correlations were found between Proteobacteria levels and OSDI scores, Acinetobacter levels were positively correlated with myopic spherical equivalent and OSDI scores. In conclusion, HM Patients have bacterial microbiota imbalance in the conjunctival sac, compared with LM patients. Proteobacteria, Actinobacteria, Acinetobacter may play roles in the HM associated ocular surface irritation.

[Source](https://doi.org/10.1007/s12275-023-00045-5)

10.1007/s12275-023-00045-5

---

## Helical Tomotherapy of Lymph Node-negative Early-stage Breast Cancer After Breast-conserving Surgery: Long-term Results.
 25.04.2023, _F Zwicker, R Klepper, H Hauswald, S Hoefel, L Raether, PE Huber, J Debus, M Schempp_


Adjuvant radiotherapy is an integral part of the interdisciplinary curative treatment of breast cancer. We aimed to examine the long-term clinical results of helical tomotherapy in female patients with local restricted, lymph node negative breast cancer after breast-conserving surgery.

[Source](https://doi.org/10.1093/procel/pwad020)

10.21873/anticanres.16365

---

## Sentinel or accomplice – gut microbiota and microglia cross-talk in disorders of gut-brain interaction

[Source](https://doi.org/10.1093/procel/pwad020)

---

## Replacing ground Rhodes grass hay with soyhulls in the pelleted diet: effects on ingestive behavior, nutrient digestibility, blood metabolites, growth performance, and economic viability of intensive fattening Lohi lambs.
 25.04.2023, _M Khan, MA Rashid, MS Yousaf, S Naveed, I Mohsin, HU Rehman_


This study aimed to evaluate the effects of replacing soyhulls (SH) for ground Rhodes grass hay (RGH) in a pelleted diet on ingestive behavior, digestibility, blood metabolites, growth performance, and economic viability of fattening Lohi lambs. A total of thirty male lambs (age: 5 months; body weight 20.4 ± 0.24 kg) were allotted to one of the three diets (n = 10 lambs/diet) under a completely randomized design. The diets were with 25% RGH inclusion (control), replacing 15% RGH with 15% SH as a fiber source (SH-15), and containing only 25% SH inclusion on a dry basis (SH-25). Ingestive behavior parameters like time spent (min/day), bouts frequency (number/day), and bout length (min/bout) for feeding, drinking, rumination, chewing, standing, and lying were not influenced (P &gt; 0.05) by replacing RGH with SH. The dry matter (DM) and neutral detergent fiber (NDF) chewing rate, rumination rate, and feeding efficiency were also not changed (P &gt; 0.05) by dietary treatments, whereas total dry matter and NDF intakes, and their rumination efficiencies were lower (P &lt; 0.05) for the SH-25 than for the rest of the treatments. The digestibility of NDF and acid detergent fiber (ADF) was higher (P &lt; 0.05) for the SH-25 than for the rest of the treatments. The concentrations of blood metabolites like glucose, blood nitrogen, and cholesterol were not changed (P &gt; 0.05) among the treatment groups. Similarly, performance parameters like dry matter intake, initial and final body weight, average daily gain, and feed conversion ratio were similar (P &gt; 0.05) across the treatments. However, the incidence of loose fecal consistency was higher (P &lt; 0.05) for SH-25 than for the control. The values of economic efficiency were better for SH-25-fed lambs than those fed the rest of the treatments. Based on the results, substituting SH for RGH in a pelleted diet improved the digestibility of fiber fractions, and economics without affecting the growth performance, and blood metabolites of fattening lambs. However, lower rumination efficiency and loos fecal consistency are evidencing less effectiveness of SH fiber.

[Source](https://doi.org/10.1080/19490976.2023.2197837)

10.1007/s11250-023-03578-4

---

## Effects of dietary fibers or probiotics on functional constipation symptoms and roles of gut microbiota: a double-blinded randomized placebo trial

[Source](https://doi.org/10.1080/19490976.2023.2197837)

---

## Use of probiotic lactobacilli in the treatment of vaginal infections: In vitro and in vivo investigations.
 21.04.2023, _P Liu, Y Lu, R Li, X Chen_


_Infant, Newborn, Female, Humans, Lactobacillus, Premature Birth, Vagina, Vulvovaginitis, Probiotics_
The vaginal microbiome is a distinct component of the human microbiome that is colonized by a wide variety of microorganisms. Lactobacilli are the most frequently identified microorganisms in the healthy human vagina. These Gram-positive bacilli can acidify the vaginal microenvironment, inhibit the proliferation of other pathogenic microorganisms, and promote the maintenance of a eubiotic vaginal microbiome. However, a vaginal flora with a reduced proportion or abundance of lactobacilli is associated with various vaginal infections that have been linked to serious health consequences such as infertility, preterm birth, pelvic inflammatory disease, premature rupture of membranes, and miscarriage. Due to their "Generally Recognized as Safe" classification and critical role in vaginal health, probiotic lactobacilli have been widely used as an alternative or adjunct to traditional antibiotic therapy for the treatment of vaginal infections and restoration of the vaginal microbiome. This review focuses on the significant role of probiotic lactobacilli in the vaginal microenvironment and discusses the use of probiotic lactobacilli in the treatment of female vaginal infections in vitro and in vivo.

[Source](https://doi.org/10.3389/fcimb.2023.1153894)

10.3389/fcimb.2023.1153894

---

## The hindgut microbiome contributes to host oxidative stress in postpartum dairy cows by affecting glutathione synthesis process.
 25.04.2023, _F Gu, S Zhu, J Hou, Y Tang, JX Liu, Q Xu, HZ Sun_


_Humans, Female, Cattle, Animals, Mice, RNA, Ribosomal, 16S, Glutamine, Cysteine, Postpartum Period, Microbiota, Glutathione, Oxidative Stress, Glutamates_
Dairy cows are susceptible to postpartum systemic oxidative stress (OS), which leads to significant production loss and metabolic disorders. The gut microbiota has been linked to host health and stress levels. However, to what extent the gut microbiota is associated with postpartum OS remains unknown. In this study, the contribution of the fecal microbiota to postpartum systemic OS and its underlying mechanisms were investigated by integrating 16S rRNA gene sequencing, metagenomics, and metabolomics in postpartum dairy cattle and by transplanting fecal microbiota from cattle to mice.

[Source](https://doi.org/10.1186/s40168-023-01535-9)

10.1186/s40168-023-01535-9

---

## Gut microbiome responds compositionally and functionally to the seasonal diet variations in wild gibbons

[Source](https://doi.org/10.1038/s41522-023-00388-2)

---

## Metagenomic and metatranscriptomic exploration of the Egyptian Red Sea sponge <em>Theonella</em> sp. associated microbial community

[Source](https://doi.org/10.1016/j.margen.2023.101032)

---

## <em>Vibrio cholerae</em> Invasion Dynamics of the Chironomid Host Are Strongly Influenced by Aquatic Cell Density and Can Vary by Strain

[Source](https://doi.org/10.1128/spectrum.02652-22)

---

## Antibiotic Conditioning Shapes Pseudosterile Mouse Models by Deleting Colonic Microbes Rather than Small Intestinal Microbes

[Source](https://doi.org/10.1128/spectrum.00814-23)

---

## Microbiota-mediated nitrogen fixation and microhabitat homeostasis in aerial root-mucilage.
 25.04.2023, _Z Pang, X Mao, S Zhou, S Yu, G Liu, C Lu, J Wan, L Hu, P Xu_


_Humans, Nitrogen Fixation, Polysaccharides, Microbiota, Bacteria, Rhizosphere, Plants, Homeostasis, Plant Roots, Soil Microbiology_
Plants sustain intimate relationships with diverse microbes. It is well-recognized that these plant-associated microbiota shape individual performance and fitness of host plants, but much remains to be explored regarding how they exert their function and maintain their homeostasis.

[Source](https://doi.org/10.1186/s40168-023-01525-x)

10.1186/s40168-023-01525-x

---

## Metapangenomics of wild and cultivated banana microbiome reveals a plethora of host-associated protective functions.
 25.04.2023, _S Singh, SA Aghdam, RM Lahowetz, AMV Brown_


Microbiomes are critical to plants, promoting growth, elevating stress tolerance, and expanding the plant's metabolic repertoire with novel defense pathways. However, generally microbiomes within plant tissues, which intimately interact with their hosts, remain poorly characterized. These endospheres have become a focus in banana (Musa spp.)-an important plant for study of microbiome-based disease protection. Banana is important to global food security, while also being critically threatened by pandemic diseases. Domestication and clonal propagation are thought to have depleted protective microbiomes, whereas wild relatives may hold promise for new microbiome-based biological controls. The goal was to compare metapangenomes enriched from 7 Musa genotypes, including wild and cultivated varieties grown in sympatry, to assess the host associations with root and leaf endosphere functional profiles.

[Source](https://doi.org/10.1186/s40793-023-00493-x)

10.1186/s40793-023-00493-x

---

## Effects of vermicompost leachate on nitrogen, phosphorus, and microbiome in a food waste bioponic system

[Source](https://doi.org/10.1016/j.jenvman.2023.117860)

---

## The effect of wheat genotype on the microbiome is more evident in roots and varies through time.
 22.04.2023, _L Quiza, J Tremblay, AP Pagé, CW Greer, CJ Pozniak, R Li, B Haug, SM Hemmingsen, M St-Arnaud, E Yergeau_


Crop breeding has traditionally ignored the plant-associated microbial communities. Considering the interactions between plant genotype and associated microbiota is of value since different genotypes of the same crop often harbor distinct microbial communities which can influence the plant phenotype. However, recent studies have reported contrasting results, which led us to hypothesize that the effect of genotype is constrained by growth stages, sampling year and plant compartment. To test this hypothesis, we sampled bulk soil, rhizosphere soil and roots of 10 field-grown wheat genotypes, twice per year, for 4 years. DNA was extracted and regions of the bacterial 16 S rRNA and CPN60 genes and the fungal ITS region were amplified and sequenced. The effect of genotype was highly contingent on the time of sampling and on the plant compartment sampled. Only for a few sampling dates, were the microbial communities significantly different across genotypes. The effect of genotype was most often significant for root microbial communities. The three marker genes used provided a highly coherent picture of the effect of genotype. Taken together, our results confirm that microbial communities in the plant environment strongly vary across compartments, growth stages, and years, and that this can mask the effect of genotype.

[Source](https://doi.org/10.1038/s43705-023-00238-4)

10.1038/s43705-023-00238-4

---

## Disentangling temporal associations in marine microbial networks.
 24.04.2023, _IM Deutschmann, AK Krabberød, F Latorre, E Delage, C Marrasé, V Balagué, JM Gasol, R Massana, D Eveillard, S Chaffron, R Logares_


_Ecosystem, Bacteria, Microbial Consortia, Microbiota, Seasons, Microbial Interactions_
Microbial interactions are fundamental for Earth's ecosystem functioning and biogeochemical cycling. Nevertheless, they are challenging to identify and remain barely known. Omics-based censuses are helpful in predicting microbial interactions through the statistical inference of single (static) association networks. Yet, microbial interactions are dynamic and we have limited knowledge of how they change over time. Here, we investigate the dynamics of microbial associations in a 10-year marine time series in the Mediterranean Sea using an approach inferring a time-resolved (temporal) network from a single static network.

[Source](https://doi.org/10.1186/s40168-023-01523-z)

10.1186/s40168-023-01523-z

---

## Evaluating the Reproducibility of Amplicon Sequencing Data Derived from Deep-Sea Cold Seep Sediment-Associated Microbiota.
 19.04.2023, _J Kong, J Feng, L Sun, S Zhang_


Benefiting from the rapid developments and wide applications of high-throughput sequencing, great advancements have been made in investigating microbiota, which are highly diverse and play key roles in both element cycling and the energy flow of ecosystems. There have been inherent limitations of amplicon sequencing that could introduce uncertainty and raise concerns about the accuracy and reproducibility of this technology. However, studies focusing on the reproducibility of amplicon sequencing are limited, especially in characterizing microbial communities in deep-sea sediments. To evaluate reproducibility, 118 deep-sea sediment samples were used for 16S rRNA gene sequencing in technical replicates (repeated measurements of the same sample) that demonstrate the variability of amplicon sequencing. The average occurrence-based overlaps were 35.98% and 27.02% between two and among three technical replicates, respectively, whereas their abundance-based overlaps reached 84.88% and 83.16%, respectively. Although variations of alpha and beta diversity indices were found between/among technical replicates, alpha diversity indices were similar across samples, and the average beta diversity indices were much smaller for technical replicates than among samples. Moreover, clustering methods (i.e., operational taxonomic units [OTUs] and amplicon sequence variants [ASVs]) were shown to have little impact on the alpha and beta diversity patterns of microbial communities. Taken together, although there are variations between/among technical replicates, amplicon sequencing is still a powerful tool with which to reveal diversity patterns of microbiota in deep-sea sediments. IMPORTANCE The reproducibility of amplicon sequencing is vital for whether the diversities of microbial communities could be accurately estimated. Thus, reproducibility influences the drawing of sound ecological conclusions. Nevertheless, few studies have focused on the reproducibility of microbial communities that are characterized by amplicon sequencing, and studies focusing on microbiota in deep-sea sediments have been especially lacking. In this study, we evaluated the reproducibility of amplicon sequencing targeting microbiota in deep-sea sediments of cold seep. Our results revealed that there were variations between/among technical replicates and that amplicon sequencing was still a powerful tool with which to characterize the diversities of microbial communities in deep-sea sediments. This study provides valuable guidelines for the reproducibility evaluation of future work in experimental design and interpretation.

[Source](https://doi.org/10.1128/spectrum.04048-22)

10.1128/spectrum.04048-22

---

## BioMateriOME: to understand microbe-material interactions within sustainable, living architectures.
 25.04.2023, _B Delgado Corrales, R Kaiser, P Nerlich, A Agraviador, A Sherry_


_Humans, Microbiota, Built Environment, Social Sciences_
BioMateriOME evolved from a prototyping process which was informed from discussions between a team of designers, architects and microbiologists, when considering constructing with biomaterials or human cohabitation with novel living materials in the built environment. The prototype has two elements (i) BioMateriOME-Public (BMP), an interactive public materials library, and (ii) BioMateriOME-eXperimental (BMX), a replicated materials library for rigorous microbiome experimentation. The prototype was installed into the OME, a unique experimental living house, in order to (1) gain insights into society's perceptions of living materials, and (2) perform a comparative analysis of indoor surface microbiome development on novel biomaterials in contrast to conventional indoor surfaces, respectively. This review summarizes the BioMateriOME prototype and its use as a tool in combining microbiology, design, architecture and social science. The use of microbiology and biological components in the fabrication of biomaterials is provided, together with an appreciation of the microbial communities common to conventional indoor surfaces, and how these communities may change in response to the implementation of living materials in our homes. Societal perceptions of microbiomes and biomaterials, are considered within the framework of healthy architecture. Finally, features of architectural design with microbes in mind are introduced, with the possibility of codifying microbial surveillance into design and construction benchmarks, standards and regulations toward healthier buildings and their occupants.

[Source](https://doi.org/10.1016/bs.aambs.2022.11.001)

10.1016/bs.aambs.2022.11.001

---

## Deciphering associations between gut microbiota and clinical factors using microbial modules.
 21.04.2023, _R Wang, X Zheng, F Song, MH Wong, KS Leung, L Cheng_


Human gut microbiota plays a vital role in maintaining body health. The dysbiosis of gut microbiota is associated with a variety of diseases. It is critical to uncover the associations between gut microbiota and disease states as well as other intrinsic or environmental factors. However, inferring alterations of individual microbial taxa based on relative abundance data likely leads to false associations and conflicting discoveries in different studies. Moreover, the effects of underlying factors and microbe-microbe interactions could lead to the alteration of larger sets of taxa. It might be more robust to investigate gut microbiota using groups of related taxa instead of the composition of individual taxa.

[Source](https://doi.org/10.1093/bioinformatics/btad213)

10.1093/bioinformatics/btad213

---

## SituSeq: an offline protocol for rapid and remote Nanopore 16S rRNA amplicon sequence analysis.
 24.04.2023, _J Zorz, C Li, A Chakraborty, DA Gittins, T Surcon, N Morrison, R Bennett, A MacDonald, CRJ Hubert_


Microbiome analysis through 16S rRNA gene sequencing is a crucial tool for understanding the microbial ecology of any habitat or ecosystem. However, workflows require large equipment, stable internet, and extensive computing power such that most of the work is performed far away from sample collection in both space and time. Performing amplicon sequencing and analysis at sample collection would have positive implications in many instances including remote fieldwork and point-of-care medical diagnoses. Here we present SituSeq, an offline and portable workflow for the sequencing and analysis of 16S rRNA gene amplicons using Nanopore sequencing and a standard laptop computer. SituSeq was validated by comparing Nanopore 16S rRNA gene amplicons, Illumina 16S rRNA gene amplicons, and Illumina metagenomes, sequenced using the same environmental DNA. Comparisons revealed consistent community composition, ecological trends, and sequence identity across platforms. Correlation between the abundance of taxa in each taxonomic level in Illumina and Nanopore data sets was high (Pearson's r &gt; 0.9), and over 70% of Illumina 16S rRNA gene sequences matched a Nanopore sequence with greater than 97% sequence identity. On board a research vessel on the open ocean, SituSeq was used to analyze amplicon sequences from deep sea sediments less than 2 h after sequencing, and 8 h after sample collection. The rapidly available results informed decisions about subsequent sampling in near real-time while the offshore expedition was still underway. SituSeq is a portable and user-friendly workflow that helps to bring the power of microbial genomics and diagnostics to many more researchers and situations.

[Source](https://doi.org/10.1038/s43705-023-00239-3)

10.1038/s43705-023-00239-3

---

## Integrating Multi-Omics Data to Construct Reliable Interconnected Models of Signaling, Gene Regulatory, and Metabolic Pathways.
 21.04.2023, _K Kumar, D Bhowmik, S Mandloi, A Gautam, A Lahiri, N Biswas, S Paul, S Chakrabarti_


_Humans, Multiomics, MicroRNAs, Signal Transduction, Metabolic Networks and Pathways, Neoplasms_
Alteration of the status of the metabolic enzymes could be a probable way to regulate metabolic reprogramming, which is a critical cellular adaptation mechanism especially for cancer cells. Coordination among biological pathways, such as gene-regulatory, signaling, and metabolic pathways is crucial for regulating metabolic adaptation. Also, incorporation of resident microbial metabolic potential in human body can influence the interplay between the microbiome and the systemic or tissue metabolic environments. Systemic framework for model-based integration of multi-omics data can ultimately improve our understanding of metabolic reprogramming at holistic level. However, the interconnectivity and novel meta-pathway regulatory mechanisms are relatively lesser explored and understood. Hence, we propose a computational protocol that utilizes multi-omics data to identify probable cross-pathway regulatory and protein-protein interaction (PPI) links connecting signaling proteins or transcription factors or miRNAs to metabolic enzymes and their metabolites using network analysis and mathematical modeling. These cross-pathway links were shown to play important roles in metabolic reprogramming in cancer scenarios.

[Source](https://doi.org/10.1007/978-1-0716-3008-2_6)

10.1007/978-1-0716-3008-2_6

---

## Interactions between the lipidome and genetic and environmental factors in autism

[Source](https://doi.org/10.1038/s41591-023-02271-1)

---

