## Diet at birth is critical for healthy growth, independent of effects on the gut microbiota.
 30.07.2024, _LJW van den Elsen, A Rekima, MA Lynn, C Isnard, S Machado, N Divakara, D Patalwala, A Middleton, N Stevens, F Servant, R Burcelin, DJ Lynn, V Verhasselt_


_Animals, Gastrointestinal Microbiome, Mice, Colostrum, Animals, Newborn, Female, Diet, Lactation, Pregnancy, Mice, Inbred C57BL, Male, Milk, Malnutrition, Bacteria_
Colostrum is the first milk for a newborn. Its high content in microbiota shaping compounds and its intake at the time of gut microbiota seeding suggests colostrum may be critical in the establishment of a healthy microbiota. There is also accumulating evidence on the importance of the gut microbiota for healthy growth. Here, we aimed to investigate the contribution of colostrum, and colostrum-induced microbiota to growth promotion. Addressing this question is highly significant because (1) globally, less than half of the newborns are fully colostrum fed (2) the evidence for the importance of the microbiota for the prevention of undernutrition has only been demonstrated in juvenile or adult pre-clinical models while stunting already starts before weaning.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01852-7)

10.1186/s40168-024-01852-7

---

## Comparative analysis of microbiome in coronal and root caries.
 02.08.2024, _T Gondo, N Hiraishi, A Takeuchi, D Moyes, Y Shimada_


_Humans, Microbiota, Root Caries, Dental Caries, RNA, Ribosomal, 16S, DNA, Bacterial, Middle Aged, Male, Female, Adult, Dentin, Aged_
The global rise in the elderly population has increased the prevalence of root caries. Streptococcus mutans, Lactobacilli and Actinomyces are considered the primary pathogens of dental caries in culture-based studies. This study aimed to investigate bacterial profiles in coronal and root caries lesions and determine the association of specific bacterial genera at each site.

[Source](https://bmcoralhealth.biomedcentral.com/articles/10.1186/s12903-024-04670-3)

10.1186/s12903-024-04670-3

---

## Gut microbiota carcinogen metabolism causes distal tissue tumours.
 31.07.2024, _B Roje, B Zhang, E Mastrorilli, A Kovačić, L Sušak, I Ljubenkov, E Ćosić, K Vilović, A Meštrović, EL Vukovac, V Bučević-Popović, Ž Puljiz, I Karaman, J Terzić, M Zimmermann_


Exposure to environmental pollutants and human microbiome composition are important predisposition factors for tumour development1,2. Similar to drug molecules, pollutants are typically metabolized in the body, which can change their carcinogenic potential and affect tissue distribution through altered toxicokinetics3. Although recent studies demonstrated that human-associated microorganisms can chemically convert a wide range of xenobiotics and influence the profile and tissue exposure of resulting metabolites4,5, the effect of microbial biotransformation on chemical-induced tumour development remains unclear. Here we show that the depletion of the gut microbiota affects the toxicokinetics of nitrosamines, which markedly reduces the development and severity of nitrosamine-induced urinary bladder cancer in mice6,7. We causally linked this carcinogen biotransformation to specific gut bacterial isolates in vitro and in vivo using individualized bacterial culture collections and gnotobiotic mouse models, respectively. We tested gut communities from different human donors to demonstrate that microbial carcinogen metabolism varies between individuals and we showed that this metabolic activity applies to structurally related nitrosamine carcinogens. Altogether, these results indicate that gut microbiota carcinogen metabolism may be a contributing factor for chemical-induced carcinogenesis, which could open avenues to target the microbiome for improved predisposition risk assessment and prevention of cancer.

[Source](https://www.nature.com/articles/s41586-024-07754-w)

10.1038/s41586-024-07754-w

---

## Time-Dependent Shifts in Intestinal Bacteriome, Klebsiella Colonization and Incidence of Antibiotic-Resistance Genes after Allogeneic Hematopoietic Stem Cell Transplantation.
 29.07.2024, _OV Goloshchapov, AB Chukhlovin, DE Polev, YA Eismont, DS Bug, AV Kusakin, OV Kosarev, RV Klementeva, VV Gostev, VA Ageevets, NP Volkov, AS Ipatova, IS Moiseev, AA Spiridonova, SV Sidorenko, AD Kulagin_


Dose-intensive cytostatic therapy and antibiotic treatment in allogeneic hematopoietic stem cell transplantation (allo-HSCT) cause severe abnormalities in a composition of gut microbiota as well as the emergence of antibiotic resistance. The data on the longitudinal recovery of major bacterial phyla and the expansion of genes associated with antibiotic resistance are limited. We collected regular stool samples during the first year after allo-HSCT from 12 adult patients with oncohematological disorders after allo-HSCT and performed 16SrRNA sequencing, multiplex PCR, conventional bacteriology and CHROMagar testing. We observed a decline in Shannon microbiota diversity index as early as day 0 of allo-HSCT (p = 0.034) before any administration of antibiotics, which persisted up to 1 year after transplantation, when the Shannon index returned to pre-transplant levels (p = 0.91). The study confirmed the previously shown decline in Bacillota (Firmicutes) genera and the expansion of E. coli/Shigella, Klebsiella and Enterococci. The recovery of Firmicutes was slower than that of other phyla and occurred only a year post-transplant. A positive correlation was observed between the expansion of E. coli/Shigella genera and blaKPC, blaCTX-M-1 and blaTEM (p &lt; 0.001), Klebsiella spp. and blaOXA-48-like, blaNDM, blaCTX-M-1, blaTEM, and blaSHV (p &lt; 0.001), Pseudomonas spp. and blaNDM (p = 0.002), Enterococcus spp. and blaOXA-48-like, blaNDM, blaCTX-M-1, blaSHV (p &lt; 0.01). The correlation was observed between the expansion of Enterobacterales and and carbapenemase-positive CHROMagar samples (p &lt; 0.001). Samples positive for carbapenem-resitant bacteria were at their maximum levels on day +30, and were gradually diminishing one year after allo-HSCT. From day +30 to +60, all isolated K. pneumoniae strains in fecal samples proved to be resistant to the main antibiotic groups (carbapenems, aminoglycosides, fluoroquinolones, third-generation cephalosporins). One year after HSCT, we documented the spontaneous decolonization of K. pneumoniae. The sensitivity of molecular biology techniques in the search for total and antibiotic-resistant Klebsiella seems to be superior to common bacteriological cultures. Future studies should be focused on searching for novel approaches to the efficient reconstitution and/or maintenance of strictly anaerobic microbiota in oncological patients.

[Source](https://www.mdpi.com/2227-9059/12/7/1566)

10.3390/biomedicines12071566

---

## Comparative analysis of the treatment-naive microbiome across rheumatic diseases to predict MTX treatment response

[Source](https://www.biorxiv.org/content/10.1101/2024.07.31.605913v1.abstract)

---

## Meta-analysis identifies common gut microbiota associated with multiple sclerosis.
 02.08.2024, _Q Lin, Y Dorsett, A Mirza, H Tremlett, L Piccio, EE Longbrake, SN Choileain, DA Hafler, LM Cox, HL Weiner, T Yamamura, K Chen, Y Wu, Y Zhou_


_Humans, Multiple Sclerosis, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Bacteria, Adult, Male, Female, Case-Control Studies_
Previous studies have identified a diverse group of microbial taxa that differ between patients with multiple sclerosis (MS) and the healthy population. However, interpreting findings on MS-associated microbiota is challenging, as there is no true consensus. It is unclear whether there is gut microbiota commonly altered in MS across studies.

[Source](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-024-01364-x)

10.1186/s13073-024-01364-x

---

## Ganoderma lucidum alleviates kidney aging by modulating gut microbiota

[Source](https://www.biorxiv.org/content/10.1101/2024.07.30.605752v1)

---

## Dietary fiber content in clinical ketogenic diets modifies the gut microbiome and seizure resistance in mice

[Source](https://www.biorxiv.org/content/10.1101/2024.07.31.606041v1)

---

## Effects of antibiotic cocktail on the fecal microbiota and their potential correlation of local immune response

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-024-03424-z)

---

## Practical applications of soil microbiota to improve ecosystem restoration: current knowledge and future directions

[Source](https://onlinelibrary.wiley.com/doi/full/10.1111/brv.13124)

---

## Impact of host species on assembly, composition, and functional profiles of phycosphere microbiomes

[Source](https://journals.asm.org/doi/10.1128/msystems.00583-24)

---

## Metaproteomic analysis decodes trophic interactions of microorganisms in the dark ocean

[Source](https://www.nature.com/articles/s41467-024-50867-z)

---

## Microbial communities on microplastics from seawater and mussels: Insights from the northern Adriatic Sea.
 02.08.2024, _Ž Kolenc, M Kovač Viršek, A Klančnik, N Janecko_


Microplastics, synthetic solid particles of different sizes (&lt; 5 mm), pose a major challenge to marine ecosystems. Introducing microplastics into the marine environment leads to the formation of complex microbial communities, a topic of growing interest in environmental research. For this study, we selected an area in the northern Adriatic Sea, less affected by human activities, to understand how pristine environmental conditions influence microbial colonization of microplastics. Samples of coastal seawater and Mediterranean mussels (Mytilus galloprovincialis) were collected in a mussel farm near Debeli rtič of the Slovenian coast. Microplastics were isolated, visually and chemically analyzed and DNA was extracted for metagenomics. In the marine water column, 12.7 microplastics per m3 water column and 0.58 microplastics per individual mussel were found. Sufficient DNA was available to analyze six particles, five originating from seawater, and one from a mussel. This was the first-ever sequenced microplastic particle from a mussel. Genera of Pseudomonas and Serratia were identified in all samples. In one of the samples, the most abundant was a marine genus Pseudoalteromonas, while in another sample Campylobacter was present with &gt;30 % abundance. The microbiomes of the mussel- and seawater-isolated particles were similar, suggesting a common microbial colonization pattern, which may have implications for the transfer of microplastic-associated microbes, including potential pathogens, through the food web to the consumers. Microplastic pollution is a complex issue requiring further research, especially regarding microbial biofilms, pathogen colonization and the potential of pathogen transmission via microplastic particles. Our findings enhance the understanding of microplastic pollution in the Adriatic Sea and stress the necessity for comprehensive strategies to mitigate the impact on marine ecosystems.

[Source](https://www.sciencedirect.com/science/article/pii/S004896972405280X)

10.1016/j.scitotenv.2024.175130

---

## Organic fertilization co-selects genetically linked antibiotic and metal(loid) resistance genes in global soil microbiome.
 20.06.2024, _ZT Liu, RA Ma, D Zhu, KT Konstantinidis, YG Zhu, SY Zhang_


_Soil Microbiology, Microbiota, Fertilizers, Metagenome, Drug Resistance, Microbial, Soil, Genes, Bacterial, Metals, Anti-Bacterial Agents, Agriculture_
Antibiotic resistance genes (ARGs) and metal(loid) resistance genes (MRGs) coexist in organic fertilized agroecosystems based on their correlations in abundance, yet evidence for the genetic linkage of ARG-MRGs co-selected by organic fertilization remains elusive. Here, an analysis of 511 global agricultural soil metagenomes reveals that organic fertilization correlates with a threefold increase in the number of diverse types of ARG-MRG-carrying contigs (AMCCs) in the microbiome (63 types) compared to non-organic fertilized soils (22 types). Metatranscriptomic data indicates increased expression of AMCCs under higher arsenic stress, with co-regulation of the ARG-MRG pairs. Organic fertilization heightens the coexistence of ARG-MRG in genomic elements through impacting soil properties and ARG and MRG abundances. Accordingly, a comprehensive global map was constructed to delineate the distribution of coexistent ARG-MRGs with virulence factors and mobile genes in metagenome-assembled genomes from agricultural lands. The map unveils a heightened relative abundance and potential pathogenicity risks (range of 4-6) for the spread of coexistent ARG-MRGs in Central North America, Eastern Europe, Western Asia, and Northeast China compared to other regions, which acquire a risk range of 1-3. Our findings highlight that organic fertilization co-selects genetically linked ARGs and MRGs in the global soil microbiome, and underscore the need to mitigate the spread of these co-resistant genes to safeguard public health.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(21)00585-7)

10.1038/s41467-024-49165-5

---

## A global metagenomic map of urban microbiomes and antimicrobial resistance

[Source](https://www.cell.com/cell/fulltext/S0092-8674(21)00585-7)

---

## Solving genomic puzzles: computational methods for metagenomic binning.
 02.08.2024, _V Mallawaarachchi, A Wickramarachchi, H Xue, B Papudeshi, SR Grigson, G Bouras, RE Prahl, A Kaphle, A Verich, B Talamantes-Becerra, EA Dinsdale, RA Edwards_


_Metagenomics, Computational Biology, Metagenome, Algorithms, Genomics_
Metagenomics involves the study of genetic material obtained directly from communities of microorganisms living in natural environments. The field of metagenomics has provided valuable insights into the structure, diversity and ecology of microbial communities. Once an environmental sample is sequenced and processed, metagenomic binning clusters the sequences into bins representing different taxonomic groups such as species, genera, or higher levels. Several computational tools have been developed to automate the process of metagenomic binning. These tools have enabled the recovery of novel draft genomes of microorganisms allowing us to study their behaviors and functions within microbial communities. This review classifies and analyzes different approaches of metagenomic binning and different refinement, visualization, and evaluation techniques used by these methods. Furthermore, the review highlights the current challenges and areas of improvement present within the field of research.

[Source](https://academic.oup.com/bib/article/25/5/bbae372/7724458)

10.1093/bib/bbae372

---

