## Metaproteomic portrait of the healthy human gut microbiota.
 02.07.2024, _A Tanca, A Palomba, G Fiorito, M Abbondio, D Pagnozzi, S Uzzau_


_Humans, Gastrointestinal Microbiome, Proteomics, Bacteria, Feces, Bacterial Proteins, Healthy Volunteers, Proteome, Metabolic Networks and Pathways_
Gut metaproteomics can provide direct evidence of microbial functions actively expressed in the colonic environments, contributing to clarify the role of the gut microbiota in human physiology. In this study, we re-analyzed 10 fecal metaproteomics datasets of healthy individuals from different continents and countries, with the aim of identifying stable and variable gut microbial functions and defining the contribution of specific bacterial taxa to the main metabolic pathways. The "core" metaproteome included 182 microbial functions and 83 pathways that were identified in all individuals analyzed. Several enzymes involved in glucose and pyruvate metabolism, along with glutamate dehydrogenase, acetate kinase, elongation factors G and Tu and DnaK, were the proteins with the lowest abundance variability in the cohorts under study. On the contrary, proteins involved in chemotaxis, response to stress and cell adhesion were among the most variable functions. Random-effect meta-analysis of correlation trends between taxa, functions and pathways revealed key ecological and molecular associations within the gut microbiota. The contribution of specific bacterial taxa to the main biological processes was also investigated, finding that Faecalibacterium is the most stable genus and the top contributor to anti-inflammatory butyrate production in the healthy gut microbiota. Active production of other mucosal immunomodulators facilitating host tolerance was observed, including Roseburia flagellin and lipopolysaccharide biosynthetic enzymes expressed by members of Bacteroidota. Our study provides a detailed picture of the healthy human gut microbiota, contributing to unveil its functional mechanisms and its relationship with nutrition, immunity, and environmental stressors.

[Source](https://doi.org/10.1038/s41522-024-00526-4)

10.1038/s41522-024-00526-4

---

## Microbiome and Growth in Infants with Congenital Heart Disease.
 29.06.2024, _MP Fundora, C Calamaro, Y Wu, AM Brown, A St John, R Keiffer, Y Xiang, K Liu, S Gillespie, PW Denning, K Sanders-Lewis, B Seitter, J Bai_


To profile the gut microbiome in infants with congenital heart disease undergoing cardiac surgery compared with matched infants and to investigate the association with growth (weight, length, and head circumference).

[Source](https://www.jpeds.com/article/S0022-3476(24)00272-5/fulltext)

10.1016/j.jpeds.2024.114169

---

## Driving gut microbiota enterotypes through host genetics.
 01.07.2024, _C Larzul, J Estellé, M Borey, F Blanc, G Lemonnier, Y Billon, MG Thiam, B Quinquis, N Galleron, D Jardet, J Lecardonnel, F Plaza Oñate, C Rogel-Gaillard_


_Animals, Gastrointestinal Microbiome, Swine, Feces, Bacteria, Metagenomics, Prevotella, Ruminococcus, Treponema_
Population stratification based on interindividual variability in gut microbiota composition has revealed the existence of several ecotypes named enterotypes in humans and various animal species. Enterotypes are often associated with environmental factors including diet, but knowledge of the role of host genetics remains scarce. Moreover, enterotypes harbor functionalities likely associated with varying abilities and susceptibilities of their host. Previously, we showed that under controlled conditions, 60-day-old pig populations consistently split into two enterotypes with either Prevotella and Mitsuokella (PM enterotype) or Ruminococcus and Treponema (RT enterotype) as keystone taxa. Here, our aim was to rely on pig as a model to study the influence of host genetics to assemble enterotypes, and to provide clues on enterotype functional differences and their links with growth traits.

[Source](https://doi.org/10.1186/s40168-024-01827-8)

10.1186/s40168-024-01827-8

---

## Obesity-associated microbiomes instigate visceral adipose tissue inflammation by recruitment of distinct neutrophils.
 30.06.2024, _D Shantaram, R Hoyd, AM Blaszczak, L Antwi, A Jalilvand, VP Wright, J Liu, AJ Smith, D Bradley, W Lafuse, Y Liu, NF Williams, O Snyder, C Wheeler, B Needleman, S Brethauer, S Noria, D Renton, KA Perry, P Nagareddy, D Wozniak, S Mahajan, PSJB Rana, M Pietrzak, LS Schlesinger, DJ Spakowicz, WA Hsueh_


_Intra-Abdominal Fat, Animals, Obesity, Humans, Neutrophils, Diet, High-Fat, Mice, Inflammation, Gastrointestinal Microbiome, Male, Mice, Inbred C57BL, Female, Feces, Microbiota, Th1 Cells, Neutrophil Infiltration_
Neutrophils are increasingly implicated in chronic inflammation and metabolic disorders. Here, we show that visceral adipose tissue (VAT) from individuals with obesity contains more neutrophils than in those without obesity and is associated with a distinct bacterial community. Exploring the mechanism, we gavaged microbiome-depleted mice with stool from patients with and without obesity during high-fat or normal diet administration. Only mice receiving high-fat diet and stool from subjects with obesity show enrichment of VAT neutrophils, suggesting donor microbiome and recipient diet determine VAT neutrophilia. A rise in pro-inflammatory CD4+ Th1 cells and a drop in immunoregulatory T cells in VAT only follows if there is a transient spike in neutrophils. Human VAT neutrophils exhibit a distinct gene expression pattern that is found in different human tissues, including tumors. VAT neutrophils and bacteria may be a novel therapeutic target for treating inflammatory-driven complications of obesity, including insulin resistance and colon cancer.

[Source](https://doi.org/10.1038/s41467-024-48935-5)

10.1038/s41467-024-48935-5

---

## Fecal, Duodenal and Tumor Microbiota Composition of Esophageal Carcinoma Patients, a Longitudinal Prospective Cohort.
 26.06.2024, _T van den Ende, NC de Clercq, M Davids, R Goedegebuure, BH Doeve, G Ebrahimi, J Buijsen, R Hoekstra, NH Mohammad, MF Bijlsma, M Nieuwdorp, HWM van Laarhoven_


The microbiome has been associated with chemotherapy and immune checkpoint inhibitor (ICI) efficacy. How this pertains to resectable esophageal carcinoma (EC) is unknown. Our aim was to identify microbial signatures in resectable EC associated with response to neoadjuvant chemoradiotherapy (nCRT) with or without ICI.

[Source](https://doi.org/10.1093/jnci/djae153)

10.1093/jnci/djae153

---

## Role of the intestinal microbiota in contributing to weight disorders and associated comorbidities.
 28.06.2024, _M Van Hul, AM Neyrinck, A Everard, A Abot, LB Bindels, NM Delzenne, C Knauf, PD Cani_


SUMMARYThe gut microbiota is a major factor contributing to the regulation of energy homeostasis and has been linked to both excessive body weight and accumulation of fat mass (i.e., overweight, obesity) or body weight loss, weakness, muscle atrophy, and fat depletion (i.e., cachexia). These syndromes are characterized by multiple metabolic dysfunctions including abnormal regulation of food reward and intake, energy storage, and low-grade inflammation. Given the increasing worldwide prevalence of obesity, cachexia, and associated metabolic disorders, novel therapeutic strategies are needed. Among the different mechanisms explaining how the gut microbiota is capable of influencing host metabolism and energy balance, numerous studies have investigated the complex interactions existing between nutrition, gut microbes, and their metabolites. In this review, we discuss how gut microbes and different microbiota-derived metabolites regulate host metabolism. We describe the role of the gut barrier function in the onset of inflammation in this context. We explore the importance of the gut-to-brain axis in the regulation of energy homeostasis and glucose metabolism but also the key role played by the liver. Finally, we present specific key examples of how using targeted approaches such as prebiotics and probiotics might affect specific metabolites, their signaling pathways, and their interactions with the host and reflect on the challenges to move from bench to bedside.

[Source](https://doi.org/10.1128/cmr.00045-23)

10.1128/cmr.00045-23

---

## The gut microbiome in disorders of gut-brain interaction.
 04.07.2024, _N Kraimi, T Ross, J Pujo, G De Palma_


_Gastrointestinal Microbiome, Humans, Brain-Gut Axis, Gastrointestinal Diseases, Brain, Animals, Gastrointestinal Tract_
Functional gastrointestinal disorders (FGIDs), chronic disorders characterized by either abdominal pain, altered intestinal motility, or their combination, have a worldwide prevalence of more than 40% and impose a high socioeconomic burden with a significant decline in quality of life. Recently, FGIDs have been reclassified as disorders of gut-brain interaction (DGBI), reflecting the key role of the gut-brain bidirectional communication in these disorders and their impact on psychological comorbidities. Although, during the past decades, the field of DGBIs has advanced significantly, the molecular mechanisms underlying DGBIs pathogenesis and pathophysiology, and the role of the gut microbiome in these processes are not fully understood. This review aims to discuss the latest body of literature on the complex microbiota-gut-brain interactions and their implications in the pathogenesis of DGBIs. A better understanding of the existing communication pathways between the gut microbiome and the brain holds promise in developing effective therapeutic interventions for DGBIs.

[Source](https://doi.org/10.1111/febs.17200)

10.1080/19490976.2024.2360233

---

## The role of the gut microbiome in disorders of gut-brain interaction

[Source](https://doi.org/10.1111/febs.17200)

---

## Multi-omics signatures reveal genomic and functional heterogeneity of Cutibacterium acnes in normal and diseased skin.
 03.07.2024, _T Yu, X Xu, Y Liu, X Wang, S Wu, Z Qiu, X Liu, X Pan, C Gu, S Wang, L Dong, W Li, X Yao_


Cutibacterium acnes is the most abundant bacterium of the human skin microbiome since adolescence, participating in both skin homeostasis and diseases. Here, we demonstrate individual and niche heterogeneity of C. acnes from 1,234 isolate genomes. Skin disease (atopic dermatitis and acne) and body site shape genomic differences of C. acnes, stemming from horizontal gene transfer and selection pressure. C. acnes harbors characteristic metabolic functions, fewer antibiotic resistance genes and virulence factors, and a more stable genome compared with Staphylococcus epidermidis. Integrated genome, transcriptome, and metabolome analysis at the strain level unveils the functional characteristics of C. acnes. Consistent with the transcriptome signature, C. acnes in a sebum-rich environment induces toxic and pro-inflammatory effects on keratinocytes. L-carnosine, an anti-oxidative stress metabolite, is up-regulated in the C. acnes metabolome from atopic dermatitis and attenuates skin inflammation. Collectively, our study reveals the joint impact of genes and the microenvironment on C. acnes function.

[Source](https://doi.org/10.1016/j.chom.2024.06.002)

10.1016/j.chom.2024.06.002

---

## Vaginal anaerobes are associated with cervicitis: a case-control study.
 29.06.2024, _EL Plummer, LA Vodstrcil, JA Danielewski, GL Murray, ML Doyle, RL Latimer, CK Fairley, EP Chow, SM Garland, CS Bradshaw_


Cervicitis is associated with important reproductive sequelae. Primary causes include chlamydia and gonorrhoea, but a known sexually transmitted infection (STI) is not identified in &gt;50% of cases (i.e. STI-negative cervicitis). Bacterial vaginosis (BV) and specific BV-associated bacteria have also been associated with cervicitis, but data are limited. We investigated the association between STI-negative cervicitis and vaginal microbiota composition.

[Source](https://www.journalofinfection.com/article/S0163-4453(24)00144-0/fulltext)

10.1016/j.jinf.2024.106210

---

## The Impact of Oral Diseases on Respiratory Health and the Influence of Respiratory Infections on the Oral Microbiome.
 27.06.2024, _A Gupta, LM Saleena, P Kannan, A Shivachandran_


The objective of this review is to examine the relationship between oral diseases and respiratory health, investigating how oral microbiome disruptions contribute to respiratory tract infections. Additionally, it aims to explore the impact of respiratory disease symptoms and treatments on the oral microbiome.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0300571224003828?via%3Dihub)

10.1016/j.jdent.2024.105213

---

## <em>Lactobacillus acidophilus </em>protects against <em>Corynebacterium pseudotuberculosis </em>infection by regulating the autophagy of macrophages and maintaining gut microbiota homeostasis in C57BL/6 mice

[Source](https://doi.org/10.1128/msystems.00484-24)

---

## Exploring novel microbial metabolites and drugs for inhibiting <em>Clostridioides difficile</em>

[Source](https://doi.org/10.1128/msphere.00273-24)

---

## Ketogenic diet-induced bile acids protect against obesity through reduced calorie absorption.
 27.06.2024, _X Li, J Yang, X Zhou, C Dai, M Kong, L Xie, C Liu, Y Liu, D Li, X Ma, Y Dai, Y Sun, Z Jian, X Guo, X Lin, Y Li, L Sun, X Liu, L Jin, H Tang, Y Zheng, S Hong_


The low-carbohydrate ketogenic diet (KD) has long been practiced for weight loss, but the underlying mechanisms remain elusive. Gut microbiota and metabolites have been suggested to mediate the metabolic changes caused by KD consumption, although the particular gut microbes or metabolites involved are unclear. Here, we show that KD consumption enhances serum levels of taurodeoxycholic acid (TDCA) and tauroursodeoxycholic acid (TUDCA) in mice to decrease body weight and fasting glucose levels. Mechanistically, KD feeding decreases the abundance of a bile salt hydrolase (BSH)-coding gut bacterium, Lactobacillus murinus ASF361. The reduction of L. murinus ASF361 or inhibition of BSH activity increases the circulating levels of TDCA and TUDCA, thereby reducing energy absorption by inhibiting intestinal carbonic anhydrase 1 expression, which leads to weight loss. TDCA and TUDCA treatments have been found to protect against obesity and its complications in multiple mouse models. Additionally, the associations among the abovementioned bile acids, microbial BSH and metabolic traits were consistently observed both in an observational study of healthy human participants (n = 416) and in a low-carbohydrate KD interventional study of participants who were either overweight or with obesity (n = 25). In summary, we uncover a unique host-gut microbiota metabolic interaction mechanism for KD consumption to decrease body weight and fasting glucose levels. Our findings support TDCA and TUDCA as two promising drug candidates for obesity and its complications in addition to a KD.

[Source](https://doi.org/10.1038/s42255-024-01072-1)

10.1038/s42255-024-01072-1

---

## How Diet and Lifestyle Can Fine-Tune Gut Microbiomes for Healthy Aging.
 01.07.2024, _M Tamayo, M Olivares, P Ruas-Madiedo, A Margolles, JC Espín, I Medina, MV Moreno-Arribas, S Canals, CR Mirasso, S Ortín, H Beltrán-Sanchez, A Palloni, FA Tomás-Barberán, Y Sanz_


_Humans, Gastrointestinal Microbiome, Life Style, Diet, Healthy Aging, Probiotics, Aging_
Many physical, social, and psychological changes occur during aging that raise the risk of developing chronic diseases, frailty, and dependency. These changes adversely affect the gut microbiota, a phenomenon known as microbe-aging. Those microbiota alterations are, in turn, associated with the development of age-related diseases. The gut microbiota is highly responsive to lifestyle and dietary changes, displaying a flexibility that also provides anactionable tool by which healthy aging can be promoted. This review covers, firstly, the main lifestyle and socioeconomic factors that modify the gut microbiota composition and function during healthy or unhealthy aging and, secondly, the advances being made in defining and promoting healthy aging, including microbiome-informed artificial intelligence tools, personalized dietary patterns, and food probiotic systems.

[Source](https://www.annualreviews.org/content/journals/10.1146/annurev-food-072023-034458)

10.1146/annurev-food-072023-034458

---

## Gut microbiota signatures of vulnerability to food addiction in mice and humans.
 26.06.2024, _S Samulėnaitė, A García-Blanco, J Mayneris-Perxachs, L Domingo-Rodríguez, J Cabana-Domínguez, N Fernàndez-Castillo, E Gago-García, L Pineda-Cirera, A Burokas, J Espinosa-Carrasco, S Arboleya, J Latorre, C Stanton, K Hosomi, J Kunisawa, B Cormand, JM Fernández-Real, R Maldonado, E Martín-García_


Food addiction is a multifactorial disorder characterised by a loss of control over food intake that may promote obesity and alter gut microbiota composition. We have investigated the potential involvement of the gut microbiota in the mechanisms underlying food addiction.

[Source](https://doi.org/10.1136/gutjnl-2023-331445)

10.1136/gutjnl-2023-331445

---

## Phenogrouping heart failure with preserved or mildly reduced ejection fraction using electronic health record data.
 05.07.2024, _F Soltani, DA Jenkins, A Kaura, J Bradley, N Black, JP Farrant, SG Williams, A Mulla, B Glampson, J Davies, D Papadimitriou, K Woods, AD Shah, MR Thursz, B Williams, FW Asselbergs, EK Mayer, C Herbert, S Grant, N Curzen, I Squire, T Johnson, K O'Gallagher, AM Shah, D Perera, R Kharbanda, RS Patel, KM Channon, R Lee, N Peek, J Mayet, CA Miller_


_Humans, Heart Failure, Electronic Health Records, Female, Male, Aged, Stroke Volume, Middle Aged, Risk Assessment, Ventricular Function, Left, United Kingdom, Risk Factors, Prognosis, Aged, 80 and over, Databases, Factual, Unsupervised Machine Learning, Hospitalization, Time Factors, Comorbidity, Cause of Death, Phenotype, Data Mining_
Heart failure (HF) with preserved or mildly reduced ejection fraction includes a heterogenous group of patients. Reclassification into distinct phenogroups to enable targeted interventions is a priority. This study aimed to identify distinct phenogroups, and compare phenogroup characteristics and outcomes, from electronic health record data.

[Source](https://doi.org/10.1128/msphere.00233-24)

10.1186/s12872-024-03987-9

---

## Group membership, not diet, structures the composition and functional potential of the gut microbiome in a wild primate

[Source](https://doi.org/10.1128/msphere.00233-24)

---

## Maternal transmission as a microbial symbiont sieve, and the absence of lactation in male mammals.
 30.06.2024, _BT Fagan, GWA Constable, R Law_


_Animals, Lactation, Female, Male, Symbiosis, Gastrointestinal Microbiome, Milk, Pregnancy, Mammals, Maternal Inheritance_
Gut microbiomes of mammals carry a complex symbiotic assemblage of microorganisms. Feeding newborn infants milk from the mammary gland allows vertical transmission of the parental milk microbiome to the offspring's gut microbiome. This has benefits, but also has hazards for the host population. Using mathematical models, we demonstrate that biparental vertical transmission enables deleterious microbial elements to invade host populations. In contrast, uniparental vertical transmission acts as a sieve, preventing these invasions. Moreover, we show that deleterious symbionts generate selection on host modifier genes that keep uniparental transmission in place. Since microbial transmission occurs during birth in placental mammals, subsequent transmission of the milk microbiome needs to be maternal to avoid the spread of deleterious elements. This paper therefore argues that viviparity and the hazards from biparental transmission of the milk microbiome, together generate selection against male lactation in placental mammals.

[Source](https://doi.org/10.1038/s41467-024-49559-5)

10.1038/s41467-024-49559-5

---

## A metagenomic assessment of bacterial community in spices sold open-air markets in Saint-Louis, Senegal.
 29.06.2024, _S Sané, AAM Diouara, S Coundoul, SD Tene, A Kane, SF Wade, A Tamba, M Diop, MN Mbaye, F Thiam, M Dieng, M Mbengue, CM Nguer, AD Sarr, AS Ndao, C Touré Kane_


_Spices, Senegal, Metagenomics, RNA, Ribosomal, 16S, Bacteria, Humans, Metagenome, Microbiota, Curcuma_
Natural spices play an essential role in human nutrition and well-being. However, their processing on different scales can expose them to potential sources of contamination. This study aimed to describe the bacterial community genomic footprint in spices sold in Senegal. Spice samples were collected in August 2022 in Saint-Louis, Senegal. The genomic region coding bacterial 16S rRNA was then amplified and sequenced using Oxford Nanopore Technology (ONT). Sequencing was carried out on two batches of samples, one containing part of the "Local Spices or Herbs" (n = 10), and the other, a mixture of 7 spices, Curcuma, Thyme and the other part of the "Local Spices or Herbs" (n = 39). Results showed high bacterial diversity and the predominance of Escherichia coli and Salmonella enterica in samples, with total reads of 65,744 and 165,325 for the two batches, respectively. The sample category "Homemade mixture of food condiments ", which includes all "Local Spices or Herbs" samples, showed remarkable bacterial diversity. These were followed by Curcuma, a blend of 7 spices and thyme. Also, the different categories of spices studied show similarities in their bacterial composition. These results highlight the microbial community's highly diverse genomic profile, including pathogenic bacteria, in spice samples.

[Source](https://doi.org/10.1038/s41598-024-65756-0)

10.1038/s41598-024-65756-0

---

## Compositionally aware estimation of cross-correlations for microbiome data.
 30.06.2024, _IT Jensen, L Janss, S Radutoiu, R Waagepetersen_


_Microbiota, Algorithms, Humans_
In the field of microbiome studies, it is of interest to infer correlations between abundances of different microbes (here referred to as operational taxonomic units, OTUs). Several methods taking the compositional nature of the sequencing data into account exist. However, these methods cannot infer correlations between OTU abundances and other variables. In this paper we introduce the novel methods SparCEV (Sparse Correlations with External Variables) and SparXCC (Sparse Cross-Correlations between Compositional data) for quantifying correlations between OTU abundances and either continuous phenotypic variables or components of other compositional datasets, such as transcriptomic data. SparCEV and SparXCC both assume that the average correlation in the dataset is zero. Iterative versions of SparCEV and SparXCC are proposed to alleviate bias resulting from deviations from this assumption. We compare these new methods to empirical Pearson cross-correlations after applying naive transformations of the data (log and log-TSS). Additionally, we test the centered log ratio transformation (CLR) and the variance stabilising transformation (VST). We find that CLR and VST outperform naive transformations, except when the correlation matrix is dense. SparCEV and SparXCC outperform CLR and VST when the number of OTUs is small and perform similarly to CLR and VST for large numbers of OTUs. Adding the iterative procedure increases accuracy for SparCEV and SparXCC for all cases, except when the average correlation in the dataset is close to zero or the correlation matrix is dense. These results are consistent with our theoretical considerations.

[Source](https://doi.org/10.1371/journal.pone.0305032)

10.1371/journal.pone.0305032

---

## Bioinformatic Workflows for Metaproteomics.
 28.06.2024, _T Holstein, T Muth_


_Proteomics, Computational Biology, Workflow, Software, Microbiota, Humans, Databases, Protein, Metagenomics_
The strong influence of microbiomes on areas such as ecology and human health has become widely recognized in the past years. Accordingly, various techniques for the investigation of the composition and function of microbial community samples have been developed. Metaproteomics, the comprehensive analysis of the proteins from microbial communities, allows for the investigation of not only the taxonomy but also the functional and quantitative composition of microbiome samples. Due to the complexity of the investigated communities, methods developed for single organism proteomics cannot be readily applied to metaproteomic samples. For this purpose, methods specifically tailored to metaproteomics are required. In this work, a detailed overview of current bioinformatic solutions and protocols in metaproteomics is given. After an introduction to the proteomic database search, the metaproteomic post-processing steps are explained in detail. Ten specific bioinformatic software solutions are focused on, covering various steps including database-driven identification and quantification as well as taxonomic and functional assignment.

[Source](https://doi.org/10.1007/978-1-0716-3910-8_16)

10.1007/978-1-0716-3910-8_16

---

## Floria: fast and accurate strain haplotyping in metagenomes.
 30.06.2024, _J Shaw, JS Gounot, H Chen, N Nagarajan, YW Yu_


_Metagenome, Metagenomics, Haplotypes, Software, Humans, Genome, Bacterial, Microbiota, Bacteria, High-Throughput Nucleotide Sequencing, Sequence Analysis, DNA_
Shotgun metagenomics allows for direct analysis of microbial community genetics, but scalable computational methods for the recovery of bacterial strain genomes from microbiomes remains a key challenge. We introduce Floria, a novel method designed for rapid and accurate recovery of strain haplotypes from short and long-read metagenome sequencing data, based on minimum error correction (MEC) read clustering and a strain-preserving network flow model. Floria can function as a standalone haplotyping method, outputting alleles and reads that co-occur on the same strain, as well as an end-to-end read-to-assembly pipeline (Floria-PL) for strain-level assembly. Benchmarking evaluations on synthetic metagenomes show that Floria is &gt; 3× faster and recovers 21% more strain content than base-level assembly methods (Strainberry) while being over an order of magnitude faster when only phasing is required. Applying Floria to a set of 109 deeply sequenced nanopore metagenomes took &lt;20 min on average per sample and identified several species that have consistent strain heterogeneity. Applying Floria's short-read haplotyping to a longitudinal gut metagenomics dataset revealed a dynamic multi-strain Anaerostipes hadrus community with frequent strain loss and emergence events over 636 days. With Floria, accurate haplotyping of metagenomic datasets takes mere minutes on standard workstations, paving the way for extensive strain-level metagenomic analyses.

[Source](https://doi.org/10.1093/bioinformatics/btae252)

10.1093/bioinformatics/btae252

---

## Enhancing antimicrobial resistance detection with MetaGeneMiner: Targeted gene extraction from metagenomes.
 27.06.2024, _C Liu, Z Tang, L Li, Y Kang, Y Teng, Y Yu_


Accurately and efficiently extracting microbial genomic sequences from complex metagenomic data is crucial for advancing our understanding in fields such as clinical diagnostics, environmental microbiology, and biodiversity. As sequencing technologies evolve, this task becomes increasingly challenging due to the intricate nature of microbial communities and the vast amount of data generated. Especially in intensive care units (ICUs), infections caused by antibiotic-resistant bacteria are increasingly prevalent among critically ill patients, significantly impacting the effectiveness of treatments and patient prognoses. Therefore, obtaining timely and accurate information about infectious pathogens is of paramount importance for the treatment of patients with severe infections, which enables precisely targeted anti-infection therapies, and a tool that can extract microbial genomic sequences from metagenomic dataset would be of help.

[Source](https://doi.org/10.1097/CM9.0000000000003182)

10.1097/CM9.0000000000003182

---

## Benchmarking low- and high-throughput protein cleanup and digestion methods for human fecal metaproteomics.
 27.06.2024, _A Tanca, MA Deledda, L De Diego, M Abbondio, S Uzzau_


The application of fecal metaproteomics to large-scale studies of the gut microbiota requires high-throughput analysis and standardized experimental protocols. Although high-throughput protein cleanup and digestion methods are increasingly used in shotgun proteomics, no studies have yet critically compared such protocols using human fecal samples. In this study, human fecal protein extracts were processed using several different protocols based on three main approaches: filter-aided sample preparation (FASP), solid-phase-enhanced sample preparation (SP3), and suspension trapping (S-Trap). These protocols were applied in both low-throughput (i.e., microtube-based) and high-throughput (i.e., microplate-based) formats, and the final peptide mixtures were analyzed by liquid chromatography coupled to high-resolution tandem mass spectrometry. The FASP-based methods and the combination of SP3 with in-StageTips (iST) yielded the best results in terms of the number of peptides identified through a database search against gut microbiome and human sequences. The efficiency of protein digestion, the ability to preserve hydrophobic peptides and high molecular weight proteins, and the reproducibility of the methods were also evaluated for the different protocols. Other relevant variables, including interindividual variability of stool, duration of protocols, and total costs, were considered and discussed. In conclusion, the data presented here can significantly contribute to the optimization and standardization of sample preparation protocols in human fecal metaproteomics. Furthermore, the promising results obtained with the high-throughput methods are expected to encourage the development of automated workflows and their application to large-scale gut microbiome studies.IMPORTANCEFecal metaproteomics is an experimental approach that allows the investigation of gut microbial functions, which are involved in many different physiological and pathological processes. Standardization and automation of sample preparation protocols in fecal metaproteomics are essential for its application in large-scale studies. Here, we comparatively evaluated different methods, available also in a high-throughput format, enabling two key steps of the metaproteomics analytical workflow (namely, protein cleanup and digestion). The results of our study provide critical information that may be useful for the optimization of metaproteomics experimental pipelines and their implementation in laboratory automation systems.

[Source](https://doi.org/10.1128/msystems.00661-24)

10.1128/msystems.00661-24

---

## Bacterial barcoding facilitates plant microbiome studies.
 26.06.2024, _H Liu_



[Source](https://doi.org/10.1038/s41579-024-01069-3)

10.1038/s41579-024-01069-3

---

## <strong>Retraction Note: </strong>Microbiome analyses of blood and tissues suggest cancer diagnostic approach

[Source](https://doi.org/10.1038/s41586-024-07656-x)

---

