## Enhancing nutritional niche and host defenses by modifying the gut microbiome.
 16.11.2022, _Q Sun, NM Vega, B Cervantes, CP Mancuso, N Mao, MN Taylor, JJ Collins, AS Khalil, J Gore, TK Lu_


_Animals, Gastrointestinal Microbiome, Caenorhabditis elegans, Microbiota, Bacteria_
The gut microbiome is essential for processing complex food compounds and synthesizing nutrients that the host cannot digest or produce, respectively. New model systems are needed to study how the metabolic capacity provided by the gut microbiome impacts the nutritional status of the host, and to explore possibilities for altering host metabolic capacity via the microbiome. Here, we colonized the nematode Caenorhabditis elegans gut with cellulolytic bacteria that enabled C. elegans to utilize cellulose, an otherwise indigestible substrate, as a carbon source. Cellulolytic bacteria as a community component in the worm gut can also support additional bacterial species with specialized roles, which we demonstrate by using Lactobacillus plantarum to protect C. elegans against Salmonella enterica infection. This work shows that engineered microbiome communities can be used to endow host organisms with novel functions, such as the ability to utilize alternate nutrient sources or to better fight pathogenic bacteria.

[Source](https://www.embopress.org/doi/full/10.15252/msb.20209933)

10.15252/msb.20209933

---

## Evolution of immune genes is associated with the Black Death

[Source](https://www.nature.com/articles/s41586-022-05349-x)

---

## Cross-generational bacterial strain transfer to an infant after fecal microbiota transplantation to a pregnant patient: a case report

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01394-w)

---

## Age-Related Changes, Influencing Factors, and Crosstalk Between Vaginal and Gut Microbiota: A Cross-Sectional Comparative Study of Pre- and Postmenopausal Women

[Source](https://www.liebertpub.com/doi/10.1089/jwh.2022.0114)

---

## Characterization of oral microbiota in HPV and non-HPV head and neck squamous cell carcinoma and its association with patient outcomes

[Source](https://www.sciencedirect.com/science/article/pii/S1368837522005358)

---

## Making gut microbiomes from scratch

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312822005182)

---

## Impairments of intestinal arginine and NO metabolisms trigger aging-associated intestinal barrier dysfunction and ‘inflammaging’

[Source](https://www.sciencedirect.com/science/article/pii/S2213231722003007)

---

## Dietary fiber ameliorates sleep disturbance connected to the gut–brain axis

[Source](https://pubs.rsc.org/en/content/articlelanding/2022/FO/D2FO01178F)

---

## Meta-analysis of 16S rRNA microbial data identified alterations of the gut microbiota in COVID-19 patients during the acute and recovery phases

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-022-02686-9)

---

## Relationships between dietary diversity and gut microbial diversity in the elderly

[Source](https://www.wageningenacademic.com/doi/10.3920/BM2022.0054)

---

## Alteration of the faecal microbiota composition in patients with constipation: evidence of American Gut Project

[Source](https://www.wageningenacademic.com/doi/10.3920/BM2022.0009)

---

## Gut microbiome, cognitive function and brain structure: a multi-omics integration analysis.
 16.11.2022, _X Liang, Y Fu, WT Cao, Z Wang, K Zhang, Z Jiang, X Jia, CY Liu, HR Lin, H Zhong, Z Miao, W Gou, M Shuai, Y Huang, S Chen, B Zhang, YM Chen, JS Zheng_


_Humans, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Cross-Sectional Studies, Case-Control Studies, Cognition, Brain_
Microbiome-gut-brain axis may be involved in the progression of age-related cognitive impairment and relevant brain structure changes, but evidence from large human cohorts is lacking. This study was aimed to investigate the associations of gut microbiome with cognitive impairment and brain structure based on multi-omics from three independent populations.

[Source](https://translationalneurodegeneration.biomedcentral.com/articles/10.1186/s40035-022-00323-z)

10.1186/s40035-022-00323-z

---

## Microbiome insights into pediatric familial adenomatous polyposis.
 16.11.2022, _TM Attard, S Septer, CE Lawson, MI Attard, STM Lee, S Umar_


_Humans, Child, Longitudinal Studies, Adenomatous Polyposis Coli, Biopsy, Microbiota, Tumor Microenvironment_
Individuals with familial adenomatous polyposis (FAP) harbor numerous polyps with inevitable early progression to colon cancer. Complex microbiotic-tumor microenvironment perturbations suggest a dysbiotic relationship between polyp and microbiome. In this study, we performed comprehensive analyses of stool and tissue microbiome of pediatric FAP subjects and compared with unaffected cohabiting relatives through 16S V4 region amplicon sequencing and machine learning platforms.

[Source](https://ojrd.biomedcentral.com/articles/10.1186/s13023-022-02569-2)

10.1186/s13023-022-02569-2

---

## The crosstalk between intestinal bacterial microbiota and immune cells in colorectal cancer progression.
 15.11.2022, _X Wen, X Ye, X Yang, R Jiang, C Qian, X Wang_


Different types of cells that are involved in tumor immunity play a significant part in antitumor therapy. The intestinal microbiota consist of the trillions of diverse microorganisms that inhabit the gastrointestinal tract. Recently, much emphasis has been paid to the link between these symbionts and colorectal cancer (CRC). This association might be anything from oncogenesis and cancer development to resistance or susceptibility to chemotherapeutic medicines. Cancer patients have a significantly different microbial composition in their guts compared to healthy persons. The microbiome may play a role in the development and development of cancer through the modulation of tumor immunosurveillance, as shown by these studies; however, the specific processes underlying this role are still poorly understood. This review focuses on the relationship between the intestinal bacterial microbiota and immune cells to determine how the commensal microbiome influences the initiation and development of CRC.

[Source](https://link.springer.com/article/10.1007/s12094-022-02995-5)

10.1007/s12094-022-02995-5

---

## Epigenetic regulation by metabolites from the gut microbiome.
 15.11.2022, _C Marín-Tello, P Jintaridth, F Sanchez, C González, L Zelada-Castillo, A Vásquez-Arqueros, A Guevara-Vásquez, A Vieira_


The gut microbiome can metabolise food components, such as dietary fibres and various phytochemicals; and the microbiome can also synthesise some nutrients, for example B vitamins. The metabolites produced by bacteria and other micro-organisms in the colon can have implications for health and disease risk. Some of these metabolites are epigenetically active, and can contribute to changes in the chemical modification and structure of chromatin by affecting the activity and expression of epigenetically-active enzymes, for example histone deacetylases and DNA methyltransferases. The epigenetic activity of such gut microbiome metabolites is reviewed herein.

[Source](https://www.wageningenacademic.com/doi/10.3920/BM2022.0006)

10.3920/BM2022.0006

---

## A tetrapeptide from maize combined with probiotics exerted strong anti-inflammatory effects and modulated gut microbiota in DSS-induced colitis mice.
 14.11.2022, _Z Li, S Zhang, L Xu, X Fang, Y Wan, D Yu, Y Guo_


Ulcerative colitis (UC) is a chronic inflammatory bowel disease characterized by recurrent gastrointestinal inflammation caused by abnormal immune response, and patients usually have intestinal flora imbalance. At present, the pathogenesis of UC is not well understood, and it appears that there is chronic activation of the immune and inflammatory cascade in genetically susceptible individuals. Some food supplements such as specific peptides and probiotics have been investigated and shown the potential for the treatment of UC. The purpose of this study is to investigate the therapeutic effect and potential mechanism of tetrapeptide from maize (TPM) and probiotic treatment on dextran sulfate sodium (DSS)-induced UC in C57BL/6J mice. Our results indicated that the therapeutic effects of TPM and probiotics are positively associated with a reduction in pro-inflammatory cytokine levels and restoration of the gut microbiota. Treatment with TPM or probiotics effectively alleviated the adverse effects of UC, including weight loss, shortened colon length, and colon and kidney tissue damage in mice. Additionally, both TPM and probiotics significantly reduced pro-inflammatory cytokine levels and oxidative stress in UC mice, and the effect was more pronounced when both were used together. Moreover, co-treatment with TPM and probiotics increased the diversity of gut microbes in UC mice, reduced the ratio of Firmicutes to Bacteroidetes (F/B) and increased the abundance of bacterial species, including Muribaculaceae, Alistipes, Ligilactobacillus and Lactobacillus, and has been shown to be beneficial for a variety of pathological conditions.

[Source](https://pubs.rsc.org/en/content/articlelanding/2022/FO/D2FO02678C)

10.1039/d2fo02678c

---

## IGF1R is a mediator of sex-specific metabolism in mice: Effects of age and high-fat diet.
 13.11.2022, _P Pérez-Matute, IP López, M Íñiguez, E Recio-Fernández, R Torrens, S Piñeiro-Hermida, E Alfaro-Arnedo, L Chau, C Walz, A Hoeflich, JA Oteo, JG Pichel_


_Female, Mice, Male, Animals, Diet, High-Fat, Obesity, Adiposity, Insulin Resistance, Insulin_
We aimed to investigate the short and long-term metabolic consequences of IGF1R systemic gene deficiency in mice.

[Source](https://www.frontiersin.org/articles/10.3389/fendo.2022.1033208/full)

10.3389/fendo.2022.1033208

---

## Nutrient dynamics in coral symbiosis depend on both the relative and absolute abundance of Symbiodiniaceae species.
 10.11.2022, _SE McIlroy, CP terHorst, M Teece, MA Coffroth_


_Animals, Anthozoa, Symbiosis, Dinoflagellida, Nitrogen, Carbon, Nutrients, Coral Reefs_
Symbionts provide a variety of reproductive, nutritional, and defensive resources to their hosts, but those resources can vary depending on symbiont community composition. As genetic techniques open our eyes to the breadth of symbiont diversity within myriad microbiomes, symbiosis research has begun to consider what ecological mechanisms affect the identity and relative abundance of symbiont species and how this community structure impacts resource exchange among partners. Here, we manipulated the in hospite density and relative ratio of two species of coral endosymbionts (Symbiodinium microadriaticum and Breviolum minutum) and used stable isotope enrichment to trace nutrient exchange with the host, Briareum asbestinum.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01382-0)

10.1186/s40168-022-01382-0

---

## Function-Based Rhizosphere Assembly along a Gradient of Desiccation in the Former Aral Sea.
 15.11.2022, _WA Wicaksono, D Egamberdieva, C Berg, M Mora, P Kusstatscher, T Cernava, G Berg_


The desiccation of the Aral Sea represents one of the largest human-made environmental regional disasters. The salt- and toxin-enriched dried-out basin provides a natural laboratory for studying ecosystem functioning and rhizosphere assembly under extreme anthropogenic conditions. Here, we investigated the prokaryotic rhizosphere communities of the native pioneer plant Suaeda acuminata (C.A.Mey.) Moq. in comparison to bulk soil across a gradient of desiccation (5, 10, and 40 years) by metagenome and amplicon sequencing combined with quantitative PCR (qPCR) analyses. The rhizosphere effect was evident due to significantly higher bacterial abundances but less diversity in the rhizosphere compared to bulk soil. Interestingly, in the highest salinity (5 years of desiccation), rhizosphere functions were mainly provided by archaeal communities. Along the desiccation gradient, we observed a significant change in the rhizosphere microbiota, which was reflected by (i) a decreasing archaeon-bacterium ratio, (ii) replacement of halophilic archaea by specific plant-associated bacteria, i.e., Alphaproteobacteria and Actinobacteria, and (iii) an adaptation of specific, potentially plant-beneficial biosynthetic pathways. In general, both bacteria and archaea were found to be involved in carbon cycling and fixation, as well as methane and nitrogen metabolism. Analysis of metagenome-assembled genomes (MAGs) showed specific signatures for production of osmoprotectants, assimilatory nitrate reduction, and transport system induction. Our results provide evidence that rhizosphere assembly by cofiltering specific taxa with distinct traits is a mechanism which allows plants to thrive under extreme conditions. Overall, our findings highlight a function-based rhizosphere assembly, the importance of plant-microbe interactions in salinated soils, and their exploitation potential for ecosystem restoration approaches. IMPORTANCE The desertification of the Aral Sea basin in Uzbekistan and Kazakhstan represents one of the most serious anthropogenic environmental disasters of the last century. Since the 1960s, the world's fourth-largest inland body of water has been constantly shrinking, which has resulted in an extreme increase of salinity accompanied by accumulation of many hazardous and carcinogenic substances, as well as heavy metals, in the dried-out basin. Here, we investigated bacterial and archaeal communities in the rhizosphere of pioneer plants by combining classic molecular methods with amplicon sequencing as well as metagenomics for functional insights. By implementing a desiccation gradient, we observed (i) remarkable differences in the archaeon-bacterium ratio of plant rhizosphere samples, (ii) replacement of archaeal indicator taxa during succession, and (iii) the presence of specific, potentially plant-beneficial biosynthetic pathways in archaea present during the early stages. In addition, our results provide hitherto-undescribed insights into the functional redundancy between plant-associated archaea and bacteria.

[Source](https://journals.asm.org/doi/10.1128/msystems.00739-22)

10.1128/msystems.00739-22

---

## Metagenomics versus total RNA sequencing: most accurate data-processing tools, microbial identification accuracy and perspectives for ecological assessments.
 14.11.2022, _CA Hempel, N Wright, J Harvie, JS Hleap, SJ Adamowicz, D Steinke_


_Metagenomics, Microbiota, Sequence Analysis, RNA, High-Throughput Nucleotide Sequencing, RNA, Ribosomal, 16S_
Metagenomics and total RNA sequencing (total RNA-Seq) have the potential to improve the taxonomic identification of diverse microbial communities, which could allow for the incorporation of microbes into routine ecological assessments. However, these target-PCR-free techniques require more testing and optimization. In this study, we processed metagenomics and total RNA-Seq data from a commercially available microbial mock community using 672 data-processing workflows, identified the most accurate data-processing tools, and compared their microbial identification accuracy at equal and increasing sequencing depths. The accuracy of data-processing tools substantially varied among replicates. Total RNA-Seq was more accurate than metagenomics at equal sequencing depths and even at sequencing depths almost one order of magnitude lower than those of metagenomics. We show that while data-processing tools require further exploration, total RNA-Seq might be a favorable alternative to metagenomics for target-PCR-free taxonomic identifications of microbial communities and might enable a substantial reduction in sequencing costs while maintaining accuracy. This could be particularly an advantage for routine ecological assessments, which require cost-effective yet accurate methods, and might allow for the incorporation of microbes into ecological assessments.

[Source](https://academic.oup.com/nar/article/50/16/9279/6671113)

10.1093/nar/gkac689

---

## Metagenome-assembled genome extraction and analysis from microbiomes using KBase.
 14.11.2022, _D Chivian, SP Jungbluth, PS Dehal, EM Wood-Charlson, RS Canon, BH Allen, MM Clark, T Gu, ML Land, GA Price, WJ Riehl, MW Sneddon, R Sutormin, Q Zhang, RW Cottingham, CS Henry, AP Arkin_


Uncultivated Bacteria and Archaea account for the vast majority of species on Earth, but obtaining their genomes directly from the environment, using shotgun sequencing, has only become possible recently. To realize the hope of capturing Earth's microbial genetic complement and to facilitate the investigation of the functional roles of specific lineages in a given ecosystem, technologies that accelerate the recovery of high-quality genomes are necessary. We present a series of analysis steps and data products for the extraction of high-quality metagenome-assembled genomes (MAGs) from microbiomes using the U.S. Department of Energy Systems Biology Knowledgebase (KBase) platform ( http://www.kbase.us/ ). Overall, these steps take about a day to obtain extracted genomes when starting from smaller environmental shotgun read libraries, or up to about a week from larger libraries. In KBase, the process is end-to-end, allowing a user to go from the initial sequencing reads all the way through to MAGs, which can then be analyzed with other KBase capabilities such as phylogenetic placement, functional assignment, metabolic modeling, pangenome functional profiling, RNA-Seq and others. While portions of such capabilities are available individually from other resources, the combination of the intuitive usability, data interoperability and integration of tools in a freely available computational resource makes KBase a powerful platform for obtaining MAGs from microbiomes. While this workflow offers tools for each of the key steps in the genome extraction process, it also provides a scaffold that can be easily extended with additional MAG recovery and analysis tools, via the KBase software development kit (SDK).

[Source](https://www.nature.com/articles/s41596-022-00747-x)

10.1038/s41596-022-00747-x

---

## SourceFinder: a Machine-Learning-Based Tool for Identification of Chromosomal, Plasmid, and Bacteriophage Sequences from Assemblies.
 15.11.2022, _D Aytan-Aktug, V Grigorjev, J Szarvas, PTLC Clausen, P Munk, M Nguyen, JJ Davis, FM Aarestrup, O Lund_


High-throughput genome sequencing technologies enable the investigation of complex genetic interactions, including the horizontal gene transfer of plasmids and bacteriophages. However, identifying these elements from assembled reads remains challenging due to genome sequence plasticity and the difficulty in assembling complete sequences. In this study, we developed a classifier, using random forest, to identify whether sequences originated from bacterial chromosomes, plasmids, or bacteriophages. The classifier was trained on a diverse collection of 23,211 chromosomal, plasmid, and bacteriophage sequences from hundreds of bacterial species. In order to adapt the classifier to incomplete sequences, each complete sequence was subsampled into 5,000 nucleotide fragments and further subdivided into k-mers. This three-class classifier succeeded in identifying chromosomes, plasmids, and bacteriophages using k-mer distributions of complete and partial genome sequences, including simulated metagenomic scaffolds with minimum performance of 0.939 area under the receiver operating characteristic curve (AUC). This classifier, implemented as SourceFinder, has been made available as an online web service to help the community with predicting the chromosomal, plasmid, and bacteriophage sources of assembled bacterial sequence data (https://cge.food.dtu.dk/services/SourceFinder/). IMPORTANCE Extra-chromosomal genes encoding antimicrobial resistance, metal resistance, and virulence provide selective advantages for bacterial survival under stress conditions and pose serious threats to human and animal health. These accessory genes can impact the composition of microbiomes by providing selective advantages to their hosts. Accurately identifying extra-chromosomal elements in genome sequence data are critical for understanding gene dissemination trajectories and taking preventative measures. Therefore, in this study, we developed a random forest classifier for identifying the source of bacterial chromosomal, plasmid, and bacteriophage sequences.

[Source](https://journals.asm.org/doi/10.1128/spectrum.02641-22)

10.1128/spectrum.02641-22

---

## TaxiBGC: a Taxonomy-Guided Approach for Profiling Experimentally Characterized Microbial Biosynthetic Gene Clusters and Secondary Metabolite Production Potential in Metagenomes.
 15.11.2022, _VK Gupta, U Bakshi, D Chang, AR Lee, JM Davis, S Chandrasekaran, YS Jin, MF Freeman, J Sung_


Biosynthetic gene clusters (BGCs) in microbial genomes encode bioactive secondary metabolites (SMs), which can play important roles in microbe-microbe and host-microbe interactions. Given the biological significance of SMs and the current profound interest in the metabolic functions of microbiomes, the unbiased identification of BGCs from high-throughput metagenomic data could offer novel insights into the complex chemical ecology of microbial communities. Currently available tools for predicting BGCs from shotgun metagenomes have several limitations, including the need for computationally demanding read assembly, predicting a narrow breadth of BGC classes, and not providing the SM product. To overcome these limitations, we developed taxonomy-guided identification of biosynthetic gene clusters (TaxiBGC), a command-line tool for predicting experimentally characterized BGCs (and inferring their known SMs) in metagenomes by first pinpointing the microbial species likely to harbor them. We benchmarked TaxiBGC on various simulated metagenomes, showing that our taxonomy-guided approach could predict BGCs with much-improved performance (mean F1 score, 0.56; mean PPV score, 0.80) compared with directly identifying BGCs by mapping sequencing reads onto the BGC genes (mean F1 score, 0.49; mean PPV score, 0.41). Next, by applying TaxiBGC on 2,650 metagenomes from the Human Microbiome Project and various case-control gut microbiome studies, we were able to associate BGCs (and their SMs) with different human body sites and with multiple diseases, including Crohn's disease and liver cirrhosis. In all, TaxiBGC provides an in silico platform to predict experimentally characterized BGCs and their SM production potential in metagenomic data while demonstrating important advantages over existing techniques. IMPORTANCE Currently available bioinformatics tools to identify BGCs from metagenomic sequencing data are limited in their predictive capability or ease of use to even computationally oriented researchers. We present an automated computational pipeline called TaxiBGC, which predicts experimentally characterized BGCs (and infers their known SMs) in shotgun metagenomes by first considering the microbial species source. Through rigorous benchmarking techniques on simulated metagenomes, we show that TaxiBGC provides a significant advantage over existing methods. When demonstrating TaxiBGC on thousands of human microbiome samples, we associate BGCs encoding bacteriocins with different human body sites and diseases, thereby elucidating a possible novel role of this antibiotic class in maintaining the stability of microbial ecosystems throughout the human body. Furthermore, we report for the first time gut microbial BGC associations shared among multiple pathologies. Ultimately, we expect our tool to facilitate future investigations into the chemical ecology of microbial communities across diverse niches and pathologies.

[Source](https://journals.asm.org/doi/10.1128/msystems.00925-22)

10.1128/msystems.00925-22

---

## Optimization of Low-Biomass Sample Collection and Quantitative PCR-Based Titration Impact 16S rRNA Microbiome Resolution.
 15.11.2022, _BGJ Clokie, A Elsheshtawy, A Albalat, A Nylund, A Beveridge, CJ Payne, S MacKenzie_


The major aquatic interface between host and environment in teleost finfish species is the gill. The diversity of this infraclass, high complexity of the organ, and its direct exposure to the surrounding environment make it an ideal candidate for furthering our understanding of the intertwined relationships between host and microbiome. Capturing the structure and diversity of bacterial communities from this low-biomass, inhibitor-rich tissue can, however, prove challenging. Lessons learned in doing so are directly applicable to similar sample types in other areas of microbiology. Through the development of a quantitative PCR assay for both host material and 16S rRNA genes, we tested and developed a robust method for low-biomass sample collection which minimized host DNA contamination. Quantification of 16S rRNA facilitated not only the screening of samples prior to costly library construction and sequencing but also the production of equicopy libraries based on 16S rRNA gene copies. A significant increase in diversity of bacteria captured was achieved, providing greater information on the true structure of the microbial community. Such findings offer important information for determining functional processes. Results were confirmed across fresh, brackish, and marine environs with four different fish species, with results showing broad homology between samples, demonstrating the robustness of the approach. Evidence presented is widely applicable to samples similar in composition, such as sputum or mucus, or those that are challenging due to the inherent inclusion of inhibitors. IMPORTANCE The interaction between the fish gill and surrounding bacteria-rich water provides an intriguing model for examining the interaction between the fish, free-floating bacteria, and the bacterial microbiome on the gill surface. Samples that are inherently low in bacteria, or that have components that inhibit the ability to produce libraries that identify the components of microbial communities, present significant challenges. Gill samples present both of these types of challenges. We developed methods for quantifying both the bacterial and host DNA material and established a sampling method which both reduced inhibitor content and maximized bacterial diversity. By quantifying and normalizing bacteria prior to library construction, we showed significant improvements with regards to the fidelity of the final data. Our results support wide-ranging applications for analyzing samples of similar composition, such as mucus and sputum, in other microbiological spheres.

[Source](https://journals.asm.org/doi/10.1128/spectrum.02255-22)

10.1128/spectrum.02255-22

---

