## Finding Candida auris in public metagenomic repositories.
 22.01.2024, _JE Mario-Vasquez, UR Bagal, E Lowe, A Morgulis, J Phan, DJ Sexton, S Shiryev, R Slatkevičius, R Welsh, AP Litvintseva, M Blumberg, R Agarwala, NA Chow_


_Humans, Candida, Candidiasis, Candida auris, Prospective Studies, Metagenomics, Antifungal Agents_
Candida auris is a newly emerged multidrug-resistant fungus capable of causing invasive infections with high mortality. Despite intense efforts to understand how this pathogen rapidly emerged and spread worldwide, its environmental reservoirs are poorly understood. Here, we present a collaborative effort between the U.S. Centers for Disease Control and Prevention, the National Center for Biotechnology Information, and GridRepublic (a volunteer computing platform) to identify C. auris sequences in publicly available metagenomic datasets. We developed the MetaNISH pipeline that uses SRPRISM to align sequences to a set of reference genomes and computes a score for each reference genome. We used MetaNISH to scan ~300,000 SRA metagenomic runs from 2010 onwards and identified five datasets containing C. auris reads. Finally, GridRepublic has implemented a prospective C. auris molecular monitoring system using MetaNISH and volunteer computing.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0291406)

10.1371/journal.pone.0291406

---

## A pilot study of fecal pH and redox as functional markers in the premature infant gut microbiome.
 27.01.2024, _J Letourneau, L Walker, SH Han, LA David, N Younge_


_Infant, Infant, Newborn, Humans, Infant, Premature, Gastrointestinal Microbiome, Pilot Projects, RNA, Ribosomal, 16S, Oxidation-Reduction, Alkalosis, Biomarkers, Hydrogen-Ion Concentration_
The infant gut microbiome is a crucial factor in health and development. In preterm infants, altered gut microbiome composition and function have been linked to serious neonatal complications such as necrotizing enterocolitis and sepsis, which can lead to long-term disability. Although many studies have described links between microbiome composition and disease risk, there is a need for biomarkers to identify infants at risk of these complications in practice. In this pilot study, we obtained stool samples from preterm infant participants longitudinally during the first postnatal months, and measured pH and redox, as well as SCFA content and microbiome composition by 16S rRNA gene amplicon sequencing. These outcomes were compared to clinical data to better understand the role of pH and redox in infant gut microbiome development and overall health, and to assess the potential utility of pH and redox as biomarkers. We found that infants born earlier or exposed to antibiotics exhibited increased fecal pH, and that redox potential increased with postnatal age. These differences may be linked to changes in SCFA content, which was correlated with pH and increased with age. Microbiome composition was also related to birth weight, age, pH, and redox. Our findings suggest that pH and redox may serve as biomarkers of metabolic state in the preterm infant gut.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0290598)

10.1371/journal.pone.0290598

---

## Assembly, stability, and dynamics of the infant gut microbiome are linked to bacterial strains and functions in mother’s milk

[Source](https://www.biorxiv.org/content/10.1101/2024.01.28.577594v1.abstract)

---

## A genome-wide association study reveals the relationship between human genetic variation and the nasal microbiome.
 30.01.2024, _X Liu, X Tong, L Zou, Y Ju, M Liu, M Han, H Lu, H Yang, J Wang, Y Zong, W Liu, X Xu, X Jin, L Xiao, H Jia, R Guo, T Zhang_


The nasal cavity harbors diverse microbiota that contributes to human health and respiratory diseases. However, whether and to what extent the host genome shapes the nasal microbiome remains largely unknown. Here, by dissecting the human genome and nasal metagenome data from 1401 healthy individuals, we demonstrated that the top three host genetic principal components strongly correlated with the nasal microbiota diversity and composition. The genetic association analyses identified 63 genome-wide significant loci affecting the nasal microbial taxa and functions, of which 2 loci reached study-wide significance (p &lt; 1.7 × 10-10): rs73268759 within CAMK2A associated with genus Actinomyces and family Actinomycetaceae; and rs35211877 near POM121L12 with Gemella asaccharolytica. In addition to respiratory-related diseases, the associated loci are mainly implicated in cardiometabolic or neuropsychiatric diseases. Functional analysis showed the associated genes were most significantly expressed in the nasal airway epithelium tissue and enriched in the calcium signaling and hippo signaling pathway. Further observational correlation and Mendelian randomization analyses consistently suggested the causal effects of Serratia grimesii and Yokenella regensburgei on cardiometabolic biomarkers (cystine, glutamic acid, and creatine). This study suggested that the host genome plays an important role in shaping the nasal microbiome.

[Source](https://www.nature.com/articles/s42003-024-05822-5)

10.1038/s42003-024-05822-5

---

## Differential peripheral immune signatures elicited by vegan versus ketogenic diets in humans.
 30.01.2024, _VM Link, P Subramanian, F Cheung, KL Han, A Stacy, L Chi, BA Sellers, G Koroleva, AB Courville, S Mistry, A Burns, R Apps, KD Hall, Y Belkaid_


Nutrition has broad impacts on all physiological processes. However, how nutrition affects human immunity remains largely unknown. Here we explored the impact of a dietary intervention on both immunity and the microbiota by performing a post hoc analysis of a clinical trial in which each of the 20 participants sequentially consumed vegan or ketogenic diets for 2 weeks ( NCT03878108 ). Using a multiomics approach including multidimensional flow cytometry, transcriptomic, proteomic, metabolomic and metagenomic datasets, we assessed the impact of each diet, and dietary switch, on host immunity and the microbiota. Our data revealed that overall, a ketogenic diet was associated with a significant upregulation of pathways and enrichment in cells associated with the adaptive immune system. In contrast, a vegan diet had a significant impact on the innate immune system, including upregulation of pathways associated with antiviral immunity. Both diets significantly and differentially impacted the microbiome and host-associated amino acid metabolism, with a strong downregulation of most microbial pathways following ketogenic diet compared with baseline and vegan diet. Despite the diversity of participants, we also observed a tightly connected network between datasets driven by compounds associated with amino acids, lipids and the immune system. Collectively, this work demonstrates that in diverse participants 2 weeks of controlled dietary intervention is sufficient to significantly and divergently impact host immunity, which could have implications for precision nutritional interventions. ClinicalTrials.gov registration: NCT03878108 .

[Source](https://www.nature.com/articles/s41591-023-02761-2)

10.1038/s41591-023-02761-2

---

## Metagenomic analyses of 7000 to 5500 years old coprolites excavated from the Torihama shell-mound site in the Japanese archipelago.
 27.01.2024, _L Nishimura, A Tanino, M Ajimoto, T Katsumura, M Ogawa, K Koganebuchi, D Waku, M Kumagai, R Sugimoto, H Nakaoka, H Oota, I Inoue_


_Humans, Japan, Metagenome, Metagenomics, Genomics, Archaeology_
Coprolites contain various kinds of ancient DNAs derived from gut micro-organisms, viruses, and foods, which can help to determine the gut environment of ancient peoples. Their genomic information should be helpful in elucidating the interaction between hosts and microbes for thousands of years, as well as characterizing the dietary behaviors of ancient people. We performed shotgun metagenomic sequencing on four coprolites excavated from the Torihama shell-mound site in the Japanese archipelago. The coprolites were found in the layers of the Early Jomon period, corresponding stratigraphically to 7000 to 5500 years ago. After shotgun sequencing, we found that a significant number of reads showed homology with known gut microbe, viruses, and food genomes typically found in the feces of modern humans. We detected reads derived from several types of phages and their host bacteria simultaneously, suggesting the coexistence of viruses and their hosts. The food genomes provide biological evidence for the dietary behavior of the Jomon people, consistent with previous archaeological findings. These results indicate that ancient genomic analysis of coprolites is useful for understanding the gut environment and lifestyle of ancient peoples.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0295924)

10.1371/journal.pone.0295924

---

## Unravelling the ancient fungal DNA from the Iceman’s gut

[Source](https://www.biorxiv.org/content/10.1101/2024.01.24.576930v1)

---

## Streptococcus anginosus promotes gastric inflammation, atrophy, and tumorigenesis in mice

[Source](https://www.cell.com/cell/fulltext/S0092-8674(24)00006-0)

---

## Gut complement induced by the microbiota combats pathogens and spares commensals

[Source](https://www.cell.com/cell/fulltext/S0092-8674(24)00001-1)

---

## Complement in breast milk modifies offspring gut microbiota to promote infant health

[Source](https://www.cell.com/cell/fulltext/S0092-8674(23)01384-3)

---

## Antidiabetic effects and mechanism of γ-polyglutamic acid on type II diabetes mice

[Source](https://www.sciencedirect.com/science/article/pii/S0141813024006123)

---

## Bacterial microbiome associated with cigarette beetle Lasioderma serricorne (F.) and its microbial plasticity in relation to diet sources

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0289215)

---

## Metagenomics reveals the temporal dynamics of the rumen resistome and microbiome in goat kids

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01733-5)

---

## Thermal fluctuations independently modulate physiological plasticity and the dynamics of the gut microbiome in a tropical rocky shore oyster

[Source](https://www.biorxiv.org/content/10.1101/2023.02.08.527599v2.abstract)

---

## Diversity and structure of the deep-sea sponge microbiome in the equatorial Atlantic Ocean

[Source](https://www.biorxiv.org/content/10.1101/2024.01.24.577104v1.abstract)

---

## Community succession of the grapevine fungal microbiome in the annual growth cycle.
 31.05.2022, _D Liu, K Howell_


_Fungi, Mycobiome, Vitis, Wine_
Microbial ecology and activity in wine production influences grapevine health and productivity, conversion of sugar to ethanol during fermentation, wine aroma, wine quality and distinctiveness. Fungi in the vineyard ecosystem are not well described. Here, we characterized the spatial and temporal dynamics of fungal communities associated with the grapevine (grapes, flowers, leaves, and roots) and soils over an annual growth cycle in two vineyards to investigate the influences of grape habitat, plant developmental stage (flowering, fruit set, veraison, and harvest), vineyards, and climatic conditions. Fungi were influenced by both the grapevine habitat and plant development stage. The core microbiome was prioritized over space and time, and the identified core members drove seasonal community succession. The developmental stage of veraison, where the grapes undergo a dramatic change in metabolism and start accumulating sugar, coincided with a distinct shift in fungal communities. Co-occurrence networks showed strong correlations between the plant microbiome, the soil microbiome, and weather indices. Our study describes the complex ecological dynamics that occur in microbial assemblages over a growing season and highlight succession of the core community in vineyards.

[Source](https://www.biorxiv.org/content/10.1101/2024.01.15.574562v2.abstract)

10.1111/1462-2920.15172

---

## Temporal and spatial dynamics within the fungal microbiome of grape fermentation

[Source](https://www.biorxiv.org/content/10.1101/2024.01.15.574562v2.abstract)

---

## Autotrophic biofilms sustained by deeply sourced groundwater host diverse bacteria implicated in sulfur and hydrogen metabolism.
 29.01.2024, _LE Valentin-Alvarado, SC Fakra, AJ Probst, JR Giska, AL Jaffe, LM Oltrogge, J West-Roberts, J Rowland, M Manga, DF Savage, C Greening, BJ Baker, JF Banfield_


_Ecosystem, Bacteria, Sulfides, Oxidation-Reduction, Groundwater, Sulfur, Biofilms, Hydrogen, Phylogeny_
Biofilms in sulfide-rich springs present intricate microbial communities that play pivotal roles in biogeochemical cycling. We studied chemoautotrophically based biofilms that host diverse CPR bacteria and grow in sulfide-rich springs to investigate microbial controls on biogeochemical cycling.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01704-w)

10.1186/s40168-023-01704-w

---

## Viroid-like colonists of human microbiomes.
 31.01.2024, _IN Zheludev, RC Edgar, MJ Lopez-Galiano, M de la Peña, A Babaian, AS Bhatt, AZ Fire_


Here, we describe the "Obelisks," a previously unrecognised class of viroid-like elements that we first identified in human gut metatranscriptomic data. "Obelisks" share several properties: (i) apparently circular RNA ∼1kb genome assemblies, (ii) predicted rod-like secondary structures encompassing the entire genome, and (iii) open reading frames coding for a novel protein superfamily, which we call the "Oblins". We find that Obelisks form their own distinct phylogenetic group with no detectable sequence or structural similarity to known biological agents. Further, Obelisks are prevalent in tested human microbiome metatranscriptomes with representatives detected in ∼7% of analysed stool metatranscriptomes (29/440) and in ∼50% of analysed oral metatranscriptomes (17/32). Obelisk compositions appear to differ between the anatomic sites and are capable of persisting in individuals, with continued presence over &gt;300 days observed in one case. Large scale searches identified 29,959 Obelisks (clustered at 90% nucleotide identity), with examples from all seven continents and in diverse ecological niches. From this search, a subset of Obelisks are identified to code for Obelisk-specific variants of the hammerhead type-III self-cleaving ribozyme. Lastly, we identified one case of a bacterial species ( Streptococcus sanguinis ) in which a subset of defined laboratory strains harboured a specific Obelisk RNA population. As such, Obelisks comprise a class of diverse RNAs that have colonised, and gone unnoticed in, human, and global microbiomes.

[Source](https://www.biorxiv.org/content/10.1101/2024.01.20.576352v1)

10.1101/2024.01.20.576352

---

## Spatially-assembled binary carbon anode synergizing directional electron transfer and enriched microbe accommodation for wastewater treatment and energy conversion: From simulation to experiments.
 31.01.2024, _M Yin, B Fu, T Xu, X Cao, X Huang, X Zhang_


Bioelectrochemical systems (BESs) hold prospects in wastewater energy and resource recovery. Anode optimization is important for simultaneous enhancement of wastewater energy conversion and effluent quality in BESs. In this study, a multi-physics model coupling fluid flow, organic degradation and electrochemical process was constructed to guide the design and optimization of BES anodes. Based on the multi-physics simulation, spatially-assembled binary carbon anodes composed of three-dimensional carbon mesh skeleton and granular activated carbon were proposed and established. The granular activated carbon conducive to microbe accommodation played a vital role in improving effluent water quality, while the carbon mesh skeleton favoring electron collection and transfer could enhance the bioelectricity output. With an average chemical oxygen demand (COD) removal rate of 0.442 kg m-3 d-1, a maximum power density of 20.6 W m-3 was achieved in the optimized composite anode BES, which was 25% and 154% higher than carbon mesh skeleton BES and granular activated carbon BES. Electroactive bacteria were enriched in composite anodes and performed important functions related to microbial metabolism and energy production. The spatially-assembled binary carbon anode with low carbon mesh packing density was more cost-effective with a daily energy output per anode cost of 221 J d-1 RMB-1. This study not only provides a cost-efficient alternative anode to simultaneously improve organic degradation and power generation performance, but also demonstrates the potential of multi-physics simulation in offering theoretical support and prediction for BES configuration design as well as optimization.

[Source](https://www.biorxiv.org/content/10.1101/2024.01.28.577602v1)

10.1016/j.watres.2024.121104

---

## DeepGOMeta: Predicting functions for microbes

[Source](https://www.biorxiv.org/content/10.1101/2024.01.28.577602v1)

---

## Predicting microbial genome-scale metabolic networks directly from 16S rRNA gene sequences

[Source](https://www.biorxiv.org/content/10.1101/2024.01.26.576649v1)

---

## Unraveling metagenomics through long-read sequencing: a comprehensive review.
 31.01.2024, _C Kim, M Pongpanich, T Porntaveetus_


_RNA, Ribosomal, 16S, High-Throughput Nucleotide Sequencing, Metagenomics, Sequence Analysis, DNA, Genomics_
The study of microbial communities has undergone significant advancements, starting from the initial use of 16S rRNA sequencing to the adoption of shotgun metagenomics. However, a new era has emerged with the advent of long-read sequencing (LRS), which offers substantial improvements over its predecessor, short-read sequencing (SRS). LRS produces reads that are several kilobases long, enabling researchers to obtain more complete and contiguous genomic information, characterize structural variations, and study epigenetic modifications. The current leaders in LRS technologies are Pacific Biotechnologies (PacBio) and Oxford Nanopore Technologies (ONT), each offering a distinct set of advantages. This review covers the workflow of long-read metagenomics sequencing, including sample preparation (sample collection, sample extraction, and library preparation), sequencing, processing (quality control, assembly, and binning), and analysis (taxonomic annotation and functional annotation). Each section provides a concise outline of the key concept of the methodology, presenting the original concept as well as how it is challenged or modified in the context of LRS. Additionally, the section introduces a range of tools that are compatible with LRS and can be utilized to execute the LRS process. This review aims to present the workflow of metagenomics, highlight the transformative impact of LRS, and provide researchers with a selection of tools suitable for this task.

[Source](https://translational-medicine.biomedcentral.com/articles/10.1186/s12967-024-04917-1)

10.1186/s12967-024-04917-1

---

## Non-prescription therapeutics for IBS: where are we?

[Source](https://www.gutmicrobiotaforhealth.com/non-prescription-therapeutics-for-ibs-where-are-we/)

---

