## Microbial competition reduces metabolic interaction distances to the low µm-range.
 20.10.2020, _RJ van Tatenhove-Pel, T Rijavec, A Lapanje, I van Swam, E Zwering, JA Hernandez-Valdes, OP Kuipers, C Picioreanu, B Teusink, H Bachmann_


Metabolic interactions between cells affect microbial community compositions and hence their function in ecosystems. It is well-known that under competition for the exchanged metabolite, concentration gradients constrain the distances over which interactions can occur. However, interaction distances are typically quantified in two-dimensional systems or without accounting for competition or other metabolite-removal, conditions which may not very often match natural ecosystems. We here analyze the impact of cell-to-cell distance on unidirectional cross-feeding in a three-dimensional aqueous system with competition for the exchanged metabolite. Effective interaction distances were computed with a reaction-diffusion model and experimentally verified by growing a synthetic consortium of 1 µm-sized metabolite producer, receiver, and competitor cells in different spatial structures. We show that receivers cannot interact with producers located on average 15 µm away from them, as product concentration gradients flatten close to producer cells. We developed an aggregation protocol and varied the receiver cells' product affinity, to show that within producer-receiver aggregates even low-affinity receiver cells could interact with producers. These results show that competition or other metabolite-removal of a public good in a three-dimensional system reduces metabolic interaction distances to the low µm-range, highlighting the importance of concentration gradients as physical constraint for cellular interactions.

[Source](https://doi.org/10.1038/s41396-020-00806-9)

10.1038/s41396-020-00806-9

---

## Developmental patterns in the nasopharyngeal microbiome during infancy are associated with asthma risk.
 24.10.2020, _HHF Tang, A Lang, SM Teo, LM Judd, R Gangnon, MD Evans, KE Lee, R Vrtis, PG Holt, RF Lemanske, DJ Jackson, KE Holt, M Inouye, JE Gern_


Studies indicate that the nasal microbiome may correlate strongly with the presence or future risk of childhood asthma.

[Source](https://doi.org/10.1016/j.jaci.2020.10.009)

10.1016/j.jaci.2020.10.009

---

## Distinct Fecal and Plasma Metabolites in Children with Autism Spectrum Disorders and Their Modulation after Microbiota Transfer Therapy.
 28.10.2020, _DW Kang, JB Adams, T Vargason, M Santiago, J Hahn, R Krajmalnik-Brown_


Accumulating evidence has strengthened a link between dysbiotic gut microbiota and autism. Fecal microbiota transplant (FMT) is a promising therapy to repair dysbiotic gut microbiota. We previously performed intensive FMT called microbiota transfer therapy (MTT) for children with autism spectrum disorders (ASD) and observed a substantial improvement of gastrointestinal and behavioral symptoms. We also reported modulation of the gut microbiome toward a healthy one. In this study, we report comprehensive metabolite profiles from plasma and fecal samples of the children who participated in the MTT trial. With 619 plasma metabolites detected, we found that the autism group had distinctive metabolic profiles at baseline. Eight metabolites (nicotinamide riboside, IMP, iminodiacetate, methylsuccinate, galactonate, valylglycine, sarcosine, and leucylglycine) were significantly lower in the ASD group at baseline, while caprylate and heptanoate were significantly higher in the ASD group. MTT drove global shifts in plasma profiles across various metabolic features, including nicotinate/nicotinamide and purine metabolism. In contrast, for 669 fecal metabolites detected, when correcting for multiple hypotheses, no metabolite was significantly different at baseline. Although not statistically significant, p-cresol sulfate was relatively higher in the ASD group at baseline, and after MTT, the levels decreased and were similar to levels in typically developing (TD) controls. p-Cresol sulfate levels were inversely correlated with Desulfovibrio, suggesting a potential role of Desulfovibrio on p-cresol sulfate modulation. Further studies of metabolites in a larger ASD cohort, before and after MTT, are warranted, as well as clinical trials of other therapies to address the metabolic changes which MTT was not able to correct.IMPORTANCE Despite the prevalence of autism and its extensive impact on our society, no U.S. Food and Drug Administration-approved treatment is available for this complex neurobiological disorder. Based on mounting evidences that support a link between autism and the gut microbiome, we previously performed a pioneering open-label clinical trial using intensive fecal microbiota transplant. The therapy significantly improved gastrointestinal and behavioral symptoms. Comprehensive metabolomic measurements in this study showed that children with autism spectrum disorder (ASD) had different levels of many plasma metabolites at baseline compared to those in typically developing children. Microbiota transfer therapy (MTT) had a systemic effect, resulting in substantial changes in plasma metabolites, driving a number of metabolites to be more similar to those from typically developing children. Our results provide evidence that changes in metabolites are one mechanism of the gut-brain connection mediated by the gut microbiota and offer plausible clinical evidence for a promising autism treatment and biomarkers.

[Source](https://msphere.asm.org/content/5/5/e00314-20)

10.1128/mSphere.00314-20

---

## Gut Microbiome in Children from Indigenous and Urban Communities in México: Different Subsistence Models, Different Microbiomes

[Source](https://doi.org/10.3390/microorganisms8101592)

---

## Gut microbiome profiling of a rural and urban South African cohort reveals biomarkers of a population in lifestyle transition

[Source](https://doi.org/10.1186/s12866-020-02017-w)

---

## Maturation of the gut microbiome during the first year of life contributes to the protective farm effect on childhood asthma

[Source](https://doi.org/10.1038/s41591-020-1095-x)

---

## Fecal Microbiota Changes in Patients With Postpartum Depressive Disorder

[Source](https://doi.org/10.3389/fcimb.2020.567268)

---

## Tunable dynamics of B cell selection in gut germinal centres

[Source](https://doi.org/10.1038/s41586-020-2865-9)

---

## Acute liver failure is regulated by MYC- and microbiome-dependent programs

[Source](https://doi.org/10.1038/s41591-020-1102-2)

---

## Epithelial TLR4 signaling activates DUOX2 to induce microbiota-driven tumorigenesis

[Source](https://doi.org/10.1053/j.gastro.2020.10.031)

---

## Gut Microbiome Alters the Activity of Liver Cytochromes P450 in Mice With Sex-Dependent Differences

[Source](https://doi.org/10.3389/fphar.2020.01303)

---

## Distinct Polysaccharide Utilization Determines Interspecies Competition between Intestinal Prevotella spp

[Source](https://doi.org/10.1016/j.chom.2020.09.012)

---

## FXR in the dorsal vagal complex is sufficient and necessary for upper small intestinal microbiome-mediated changes of TCDCA to alter insulin action in rats.
 22.10.2020, _SY Zhang, RJW Li, YM Lim, B Batchuluun, H Liu, TMZ Waise, TKT Lam_


Conjugated bile acids are metabolised by upper small intestinal microbiota, and serum levels of taurine-conjugated bile acids are elevated and correlated with insulin resistance in people with type 2 diabetes. However, whether changes in taurine-conjugated bile acids are necessary for small intestinal microbiome to alter insulin action remain unknown.

[Source](http://dx.doi.org/10.1136/gutjnl-2020-321757)

10.1136/gutjnl-2020-321757

---

## To the Land and Beyond: Crab Microbiomes as a Paradigm for the Evolution of Terrestrialization.
 30.10.2020, _S Cannicci, S Fratini, N Meriggi, G Bacci, A Iannucci, A Mengoni, D Cavalieri_


The transition to terrestrial environments by formerly aquatic species has occurred repeatedly in many animal phyla and lead to the vast diversity of extant terrestrial species. The differences between aquatic and terrestrial habitats are enormous and involved remarkable morphological and physiological changes. Convergent evolution of various traits is evident among phylogenetically distant taxa, but almost no information is available about the role of symbiotic microbiota in such transition. Here, we suggest that intertidal and terrestrial brachyuran crabs are a perfect model to study the evolutionary pathways and the ecological role of animal-microbiome symbioses, since their transition to land is happening right now, through a number of independent lineages. The microorganisms colonizing the gut of intertidal and terrestrial crabs are expected to play a major role to conquer the land, by reducing water losses and permitting the utilization of novel food sources. Indeed, it has been shown that the microbiomes hosted in the digestive system of terrestrial isopods has been critical to digest plant items, but nothing is known about the microbiomes present in the gut of truly terrestrial crabs. Other important physiological regulations that could be facilitated by microbiomes are nitrogen excretion and osmoregulation in the new environment. We also advocate for advances in comparative and functional genomics to uncover physiological aspects of these ongoing evolutionary processes. We think that the multidisciplinary study of microorganisms associated with terrestrial crabs will shed a completely new light on the biological and physiological processes involved in the sea-land transition.

[Source](https://doi.org/10.3389/fmicb.2020.575372)

10.3389/fmicb.2020.575372

---

## The direct and indirect effects of environmental toxicants on the health of bumblebees and their microbiomes.
 28.10.2020, _JA Rothman, KA Russell, L Leger, QS McFrederick, P Graystock_


Bumblebees (Bombus spp.) are important and widespread insect pollinators, but the act of foraging on flowers can expose them to harmful pesticides and chemicals such as oxidizers and heavy metals. How these compounds directly influence bee survival and indirectly affect bee health via the gut microbiome is largely unknown. As toxicants in floral nectar and pollen take many forms, we explored the genomes of bee-associated microbes for their potential to detoxify cadmium, copper, selenate, the neonicotinoid pesticide imidacloprid, and hydrogen peroxide-which have all been identified in floral nectar and pollen. We then exposed Bombus impatiens workers to varying concentrations of these chemicals via their diet and assayed direct effects on bee survival. Using field-realistic doses, we further explored the indirect effects on bee microbiomes. We found multiple putative genes in core gut microbes that may aid in detoxifying harmful chemicals. We also found that while the chemicals are largely toxic at levels within and above field-realistic concentrations, the field-realistic concentrations-except for imidacloprid-altered the composition of the bee microbiome, potentially causing gut dysbiosis. Overall, our study shows that chemicals found in floral nectar and pollen can cause bee mortality, and likely have indirect, deleterious effects on bee health via their influence on the bee microbiome.

[Source](https://doi.org/10.1098/rspb.2020.0980)

10.1098/rspb.2020.0980

---

## A Very Hungry Caterpillar: Polyethylene Metabolism and Lipid Homeostasis in Larvae of the Greater Wax Moth (Galleria mellonella).
 26.10.2020, _CM LeMoine, HC Grove, CM Smith, BJ Cassone_


Larvae of the greater wax moth (Galleria mellonella) possess the remarkable ability to consume and rapidly degrade low-density polyethylene. Previous studies have investigated the involvement of the animal's microbiome, but little is known about the host's actual role and if it benefits from biodegradation of this synthetic polymer. We used a combination of RNA sequencing and biochemical approaches to assess caterpillars fed honeycomb, fed polyethylene (PE), or starved for up to 72 h. Sequencing of gut transcripts revealed PE-fed larvae retain an expression profile consistent with normal intestinal function but also show distinct molecular signatures indicative of enhanced fatty acid metabolism (FAM). Further, quantification of total lipid content validated the impact of a PE diet on FAM; in contrast to lipid-depleted starved animals, PE-fed caterpillars maintain lipid reserves similar to honeycomb-fed larvae. Additionally, we found the activity of putative enzymes involved in lipid oxidation (e.g., alcohol dehydrogenase) are considerably higher in PE-fed larvae, indicating that on a functional level, these caterpillars are inducing pathways to effectively metabolize PE. Overall, we put forward a hypothesized model where the similarity in chemical structure between PE and its natural honeycomb diet has endowed larvae of G. mellonella with the extraordinary capability to derive energy from PE as an exclusive food source through pre-existing metabolic pathways.

[Source](https://doi.org/10.1021/acs.est.0c04386)

10.1021/acs.est.0c04386

---

## Deciphering the Microbial Taxonomy and Functionality of Two Diverse Mangrove Ecosystems and Their Potential Abilities To Produce Bioactive Compounds.
 07.11.2020, _S Liao, Y Wang, H Liu, G Fan, SK Sahu, T Jin, J Chen, P Zhang, L Gram, ML Strube, Q Shi, SMY Lee, X Liu_


Mangroves, as important and special ecosystems, create unique ecological environments for examining the microbial gene capacity and potential for producing bioactive compounds. However, little is known about the biogeochemical implications of microbiomes in mangrove ecosystems, especially the variations between pristine and anthropogenic mangroves. To elucidate this, we investigated the microbial taxonomic and functional shifts of the mangrove microbiomes and their potential for bioactive compounds in two different coastal mangrove ecosystems in southern China. A gene catalogue, including 87 million unique genes, was constructed, based on deep shotgun metagenomic sequencing. Differentially enriched bacterial and archaeal taxa between pristine mangroves (Guangxi) and anthropogenic mangroves (Shenzhen) were found. The Nitrospira and ammonia-oxidizing archaea, specifically, were more abundant in Shenzhen mangroves, while sulfate-reducing bacteria and methanogens were more abundant in Guangxi mangroves. The results of functional analysis were consistent with the taxonomic results, indicating that the Shenzhen mangrove microbiome has a higher abundance of genes involved in nitrogen metabolism while the Guangxi mangrove microbiome has a higher capacity for sulfur metabolism and methanogenesis. Biosynthetic gene clusters were identified in the metagenome data and in hundreds of de novo reconstructed nonredundant microbial genomes, respectively. Notably, we found different biosynthetic potential in different taxa, and we identified three high quality and novel Acidobacteria genomes with a large number of BGCs. In total, 67,278 unique genes were annotated with antibiotic resistance, indicating the prevalence and persistence in multidrug-resistant genes in the mangrove microbiome.IMPORTANCE This study comprehensively described the taxonomy and functionality of mangrove microbiomes, including their capacity for secondary metabolite biosynthesis and their ability to resist antibiotics. The microbial taxonomic and functional characteristics differed between geographical locations, corresponding to the environmental condition of two diverse mangrove regions. A large number of microbial biosynthetic gene clusters encoding novel bioactivities were found, and this can serve as a valuable resource to guide novel bioactive compound discovery for potential clinical uses.

[Source](https://msystems.asm.org/content/5/5/e00851-19)

10.1128/mSystems.00851-19

---

## Introducing the Mangrove Microbiome Initiative: Identifying Microbial Research Priorities and Approaches To Better Understand, Protect, and Rehabilitate Mangrove Ecosystems.
 28.10.2020, _SM Allard, MT Costa, AN Bulseco, V Helfer, LGE Wilkins, C Hassenrück, K Zengler, M Zimmer, N Erazo, JL Mazza Rodrigues, N Duke, VMM Melo, I Vanwonterghem, H Junca, HM Makonde, DJ Jiménez, TCL Tavares, M Fusi, D Daffonchio, CM Duarte, RS Peixoto, AS Rosado, JA Gilbert, J Bowman_


Mangrove ecosystems provide important ecological benefits and ecosystem services, including carbon storage and coastline stabilization, but they also suffer great anthropogenic pressures. Microorganisms associated with mangrove sediments and the rhizosphere play key roles in this ecosystem and make essential contributions to its productivity and carbon budget. Understanding this nexus and moving from descriptive studies of microbial taxonomy to hypothesis-driven field and lab studies will facilitate a mechanistic understanding of mangrove ecosystem interaction webs and open opportunities for microorganism-mediated approaches to mangrove protection and rehabilitation. Such an effort calls for a multidisciplinary and collaborative approach, involving chemists, ecologists, evolutionary biologists, microbiologists, oceanographers, plant scientists, conservation biologists, and stakeholders, and it requires standardized methods to support reproducible experiments. Here, we outline the Mangrove Microbiome Initiative, which is focused around three urgent priorities and three approaches for advancing mangrove microbiome research.

[Source](https://msystems.asm.org/content/5/5/e00658-20)

10.1128/mSystems.00658-20

---

## The water chemistry and microbiome of household wells in Medawachchiya, Sri Lanka, an area with high prevalence of chronic kidney disease of unknown origin (CKDu).
 30.10.2020, _LK McDonough, KT Meredith, C Nikagolla, RJ Middleton, JK Tan, AV Ranasinghe, F Sierro, RB Banati_


Chronic kidney disease (CKD) of unknown etiology (CKDu) mostly affects agricultural communities in Central America, South Asia, Africa, but likely also in North America and Australia. One such area with increased CKDu prevalence is the Medawachchiya District Secretariat Division of the Anuradhapura District in the North Central Province of Sri Lanka. Recent research has focused on the presence of various microbial pathogens in drinking water as potential causal or contributing factors to CKDu, yet no study to date has performed a more comprehensive microbial and water chemistry assessment of household wells used for domestic water supply in areas of high CKDu prevalence. In this study, we describe the chemical composition and total microbial content in 30 domestic household wells in the Medawachchiya District Secretariat Division. While the chemical composition in the tested wells mostly lies within standard drinking water limits, except for high levels of fluoride (F), magnesium (Mg), sodium (Na), chloride (Cl) and calcium (Ca) in some samples, we find a frequent presence of cyanotoxin-producing Microcystis, confirming earlier studies in Sri Lanka. Since the total microbial content of drinking water also directly influences the composition of the human gut microbiome, it can be considered an important determinant of health. Several bacterial phyla were previously reported in the gut microbiome of patients with CKD. Using these bacteria phyla to define operational taxonomic units, we found that these bacteria also occur in the microbiome of the sampled well water. Based on available environmental data, our study demonstrates associations between the abundances of these bacteria with geographical distribution, well water temperature and likely fertilizer use in the local surface water catchment area of the individual household wells. Our results reinforce the recommendation that household wells with stagnant or infrequently used water should be purged prior to use for drinking water, bathing and irrigation. The latter is suggested because of the reported potential accumulation of bacterial toxins by agricultural crops. The observation that bacteria previously found in chronic kidney disease patients are also present in household wells requires a more detailed systematic study of both the human gut and drinking water microbiomes in CKDu patients, in relation to disease prevalence and progression.

[Source](https://doi.org/10.1038/s41598-020-75336-7)

10.1038/s41598-020-75336-7

---

## Sediment Metagenomes as Time Capsules of Lake Microbiomes.
 05.11.2020, _RE Garner, I Gregory-Eaves, DA Walsh_


The reconstruction of ecological time series from lake sediment archives can retrace the environmental impact of human activities. Molecular genetic approaches in paleolimnology have provided unprecedented access to DNA time series, which record evidence of the microbial ecologies that underlaid historical lake ecosystems. Such studies often rely on single-gene surveys, and consequently, the full diversity of preserved microorganisms remains unexplored. In this study, we probed the diversity archived in contemporary and preindustrial sediments by comparative shotgun metagenomic analysis of surface water and sediment samples from three eastern Canadian lakes. In a strategy that was aimed at disentangling historical DNA from the indigenous sediment background, microbial preservation signals were captured by mapping sequence similarities between sediment metagenome reads and reference surface water metagenome assemblies. We detected preserved Cyanobacteria, diverse bacterioplankton, microeukaryotes, and viruses in sediment metagenomes. Among the preserved microorganisms were important groups never before reported in paleolimnological reconstructions, including bacteriophages (Caudovirales) and ubiquitous freshwater Betaproteobacteria (Polynucleobacter and Limnohabitans). In contrast, ultramicroscopic Actinobacteria ("Candidatus Nanopelagicales") and Alphaproteobacteria (Pelagibacterales) were apparently not well preserved in sediment metagenomes even though they were numerically dominant in surface water metagenomes. Overall, our study explored a novel application of whole-metagenome shotgun sequencing for discovering the DNA remains of a broad diversity of microorganisms preserved in lake sediments. The recovery of diverse microbial time series supports the taxonomic expansion of microbiome reconstructions and the development of novel microbial paleoindicators.IMPORTANCE Lakes are critical freshwater resources under mounting pressure from climate change and other anthropogenic stressors. The reconstruction of ecological time series from sediment archives with paleolimnological techniques has been shown to be an effective means of understanding how humans are modifying lake ecosystems over extended timescales. In this study, we combined shotgun DNA sequencing with a novel comparative analysis of surface water and sediment metagenomes to expose the diversity of microorganisms preserved in lake sediments. The detection of DNA from a broad diversity of preserved microbes serves to more fully reconstruct historical microbiomes and describe preimpact lake conditions.

[Source](https://msphere.asm.org/content/5/6/e00512-20)

10.1128/mSphere.00512-20

---

## Bioaerosols in the Athens Metro: Metagenetic insights into the PM 10 microbiome in a naturally ventilated subway station

[Source](https://doi.org/10.1016/j.envint.2020.106186)

---

## Biotic Interactions Are More Important than Propagule Pressure in Microbial Community Invasions.
 31.10.2020, _MBN Albright, S Sevanto, V Gallegos-Graves, J Dunbar_


Microbial probiotics are intended to improve functions in diverse ecosystems, yet probiotics often fail to establish in a preexisting microbiome. This is a species invasion problem. The relative importance of the two major factors controlling establishment in this context-propagule pressure (inoculation dose and frequency) and biotic interactions (composition of introduced and resident communities)-is unknown. We tested the effect of these factors in driving microbial composition and functioning following 12 microbial community invasions (e.g., introductions of many microbial invaders) in microcosms. Ecosystem functioning over a 30-day postinvasion period was assessed by measuring activity (respiration) and environment modification (dissolved organic carbon abundance). To test the dependence on environmental context, experiments were performed in two resource environments. In both environments, biotic interactions were more important than propagule pressure in driving microbial composition and community function, but the magnitude of effect varied by environment. Successful invaders comprised approximately 8% of the total number of operational taxonomic units (OTUs). Bacteria were better invaders than fungi, with average relative abundances of 7.4% ± 6.8% and 1.5% ± 1.4% of OTUs, respectively. Common bacterial invaders were associated with stress response traits. The most resilient bacterial and fungal families, in other words, those least impacted by invasions, were linked to antimicrobial resistance or production traits. Illuminating the principles that determine community composition and functioning following microbial invasions is key to efficient community engineering.IMPORTANCE With increasing frequency, humans are introducing new microbes into preexisting microbiomes to alter functioning. Example applications include modification of microflora in human guts for better health and those of soil for food security and/or climate management. Probiotic applications are often approached as trial-and-error endeavors and have mixed outcomes. We propose that increased success in microbiome engineering may be achieved with a better understanding of microbial invasions. We conducted a microbial community invasion experiment to test the relative importance of propagule pressure and biotic interactions in driving microbial community composition and ecosystem functioning in microcosms. We found that biotic interactions were more important than propagule pressure in determining the impact of microbial invasions. Furthermore, the principles for community engineering vary among organismal groups (bacteria versus fungi).

[Source](https://mbio.asm.org/content/11/5/e02089-20.long)

10.1128/mBio.02089-20

---

## Microbiological, immunological, and histological changes in the gut of Salmonella Enteritidis-challenged rats fed goat cheese containing Lactobacillus rhamnosus EM1107.
 02.11.2020, _FRL Rolim, CJB Oliveira, OC de Freitas Neto, KMO Dos Santos, GCB Guerra, RV Rodrigues, POA de Assis, DFS Araújo, VAG de Carvalho, MLP Lemos, NMV da Silva, JKB Soares, HEM Garcia, EL de Souza, F de A L Souza, MEG de Barros, MEG de Oliveira, RCRE Queiroga_


Cheeses are able to serve as suitable matrices for supplying probiotics to consumers, enabling appropriate conditions for bacteria to survive gastric transit and reach the gut, where they are assumed to promote beneficial processes. The present study aimed to evaluate the microbiological, immunological, and histological changes in the gut of Salmonella Enteritidis-challenged rats fed goat cheese supplemented with the probiotic strain Lactobacillus rhamnosus EM1107. Thirty male albino Wistar rats were randomly distributed into 5 experimental groups with 6 animals each: negative (NC) and positive (PtC) control groups, control goat cheese (CCh), goat cheese added with L. rhamnosus EM1107 (LrCh), and L. rhamnosus EM1107 only (EM1107). All animals, except NC group were challenged with Salmonella Enteritidis (109 colony-forming units in 1 mL of saline through oral gavage). Microbial composition was assessed with high-throughput 16S rRNA sequencing by means of Illumina MiSeq (Illumina, San Diego, CA). Nuclear factor kappa B (NF-κB) from the animal cecum tissue was determined by real-time PCR and interleukins (TNF-α, IL-1β, IL-10, and IFN-γ) by means of ELISA. Myeloperoxidase and malondialdehyde levels were determined biochemically. The administration of the L. rhamnosus EM1107 probiotic strain, either as a pure culture or added to a cheese matrix, was able to reduce Salmonella colonization in the intestinal lumen and lessen tissue damage compared with rats from PtC group. In addition, the use of cheese for the probiotic strain delivery (LrCh) was associated with a marked shift in the gut microbiota composition toward the increase of beneficial organisms such as Blautia and Lactobacillus and a reduction in NF-κB expression. These findings support our hypothesis that cheeses might be explored as functional matrices for the efficacious delivery of probiotic strains to consumers.

[Source](https://doi.org/10.3168/jds.2020-18820)

10.3168/jds.2020-18820

---

## Comparative Genome Analysis of Bifidobacterium longum subsp. infantis Strains Reveals Variation in Human Milk Oligosaccharide Utilization Genes among Commercial Probiotics.
 29.10.2020, _RM Duar, G Casaburi, RD Mitchell, LNC Scofield, CA Ortega Ramirez, D Barile, BM Henrick, SA Frese_


Dysbiosis is associated with acute and long-term consequences for neonates. Probiotics can be effective in limiting the growth of bacteria associated with dysbiosis and promoting the healthy development of the infant microbiome. Given its adaptation to the infant gut, and promising data from animal and in vitro models, Bifidobacterium longum subsp. infantis is an attractive candidate for use in infant probiotics. However, strain-level differences in the ability of commercialized strains to utilize human milk oligosaccharides (HMOs) may have implications in the performance of strains in the infant gut. In this study, we characterized twelve B. infantis probiotic strains and identified two main variants in one of the HMO utilization gene clusters. Some strains possessed the full repertoire of HMO utilization genes (H5-positive strains), while H5-negative strains lack an ABC-type transporter known to bind core HMO structures. H5-positive strains achieved significantly superior growth on lacto-N-tetraose and lacto-N-neotetraose. In vitro, H5-positive strains had a significant fitness advantage over H5-negative strains, which was also observed in vivo in breastfed infants. This work provides evidence of the functional implications of genetic differences among B. infantis strains and highlights that genotype and HMO utilization phenotype should be considered when selecting a strain for probiotic use in infants.

[Source](https://doi.org/10.3390/nu12113247)

10.3390/nu12113247

---

## Persistence of Suspected Probiotic Organisms in Preterm Infant Gut Microbiota Weeks After Probiotic Supplementation in the NICU.
 30.10.2020, _EI Yousuf, M Carvalho, SE Dizzell, S Kim, E Gunn, J Twiss, L Giglia, C Stuart, EK Hutton, KM Morrison, JC Stearns_


Probiotics are becoming a prevalent supplement to prevent necrotizing enterocolitis in infants born preterm. However, little is known about the ability of these live bacterial supplements to colonize the gut or how they affect endogenous bacterial strains and the overall gut community. We capitalized on a natural experiment resulting from a policy change that introduced the use of probiotics to preterm infants in a single Neonatal Intensive Care Unit. We used amplicon sequence variants (ASVs) derived from the v3 region of the 16S rRNA gene to compare the prevalence and abundance of Bifidobacterium and Lactobacillus in the gut of preterm infants who were and were not exposed to a probiotic supplement in-hospital. Infants were followed to 5 months corrected age. In the probiotic-exposed infants, ASVs belonging to species of Bifidobacterium appeared at high relative abundance during probiotic supplementation and persisted for up to 5 months. In regression models that controlled for the confounding effects of age and antibiotic exposure, probiotic-exposed infants had a higher abundance of the suspected probiotic bifidobacteria than unexposed infants. Conversely, the relative abundance of Lactobacillus was similar between preterm groups over time. Lactobacillus abundance was inversely related to antibiotic exposure. Furthermore, the overall gut microbial community of the probiotic-exposed preterm infants at term corrected age clustered more closely to samples collected from 10-day old full-term infants than to samples from unexposed preterm infants at term age. In conclusion, routine in-hospital administration of probiotics to preterm infants resulted in the potential for colonization of the gut with probiotic organisms post-discharge and effects on the gut microbiome as a whole. Further research is needed to fully discriminate probiotic bacterial strains from endogenous strains and to explore their functional role in the gut microbiome and in infant health.

[Source](https://doi.org/10.3389/fmicb.2020.574137)

10.3389/fmicb.2020.574137

---

## Model-based and phylogenetically adjusted quantification of metabolic interaction between microbial species.
 30.10.2020, _TJ Lam, M Stamboulian, W Han, Y Ye_


Microbial community members exhibit various forms of interactions. Taking advantage of the increasing availability of microbiome data, many computational approaches have been developed to infer bacterial interactions from the co-occurrence of microbes across diverse microbial communities. Additionally, the introduction of genome-scale metabolic models have also enabled the inference of cooperative and competitive metabolic interactions between bacterial species. By nature, phylogenetically similar microbial species are more likely to share common functional profiles or biological pathways due to their genomic similarity. Without properly factoring out the phylogenetic relationship, any estimation of the competition and cooperation between species based on functional/pathway profiles may bias downstream applications. To address these challenges, we developed a novel approach for estimating the competition and complementarity indices for a pair of microbial species, adjusted by their phylogenetic distance. An automated pipeline, PhyloMint, was implemented to construct competition and complementarity indices from genome scale metabolic models derived from microbial genomes. Application of our pipeline to 2,815 human-gut associated bacteria showed high correlation between phylogenetic distance and metabolic competition/cooperation indices among bacteria. Using a discretization approach, we were able to detect pairs of bacterial species with cooperation scores significantly higher than the average pairs of bacterial species with similar phylogenetic distances. A network community analysis of high metabolic cooperation but low competition reveals distinct modules of bacterial interactions. Our results suggest that niche differentiation plays a dominant role in microbial interactions, while habitat filtering also plays a role among certain clades of bacterial species.

[Source](https://doi.org/10.1371/journal.pcbi.1007951)

10.1371/journal.pcbi.1007951

---

## Comprehensive screening of genomic and metagenomic data reveals a large diversity of tetracycline resistance genes.
 30.10.2020, _F Berglund, ME Böhm, A Martinsson, S Ebmeyer, T Österlund, A Johnning, DGJ Larsson, E Kristiansson_


Tetracyclines are broad-spectrum antibiotics used to prevent or treat a variety of bacterial infections. Resistance is often mediated through mobile resistance genes, which encode one of the three main mechanisms: active efflux, ribosomal target protection or enzymatic degradation. In the last few decades, a large number of new tetracycline-resistance genes have been discovered in clinical settings. These genes are hypothesized to originate from environmental and commensal bacteria, but the diversity of tetracycline-resistance determinants that have not yet been mobilized into pathogens is unknown. In this study, we aimed to characterize the potential tetracycline resistome by screening genomic and metagenomic data for novel resistance genes. By using probabilistic models, we predicted 1254 unique putative tetracycline resistance genes, representing 195 gene families (&lt;70 % amino acid sequence identity), whereof 164 families had not been described previously. Out of 17 predicted genes selected for experimental verification, 7 induced a resistance phenotype in an Escherichia coli host. Several of the predicted genes were located on mobile genetic elements or in regions that indicated mobility, suggesting that they easily can be shared between bacteria. Furthermore, phylogenetic analysis indicated several events of horizontal gene transfer between bacterial phyla. Our results also suggested that acquired efflux pumps originate from proteobacterial species, while ribosomal protection genes have been mobilized from Firmicutes and Actinobacteria. This study significantly expands the knowledge of known and putatively novel tetracycline resistance genes, their mobility and evolutionary history. The study also provides insights into the unknown resistome and genes that may be encountered in clinical settings in the future.

[Source](https://doi.org/10.1099/mgen.0.000455)

10.1099/mgen.0.000455

---

## Naught all zeros in sequence count data are the same.
 29.10.2020, _JD Silverman, K Roche, S Mukherjee, LA David_


Genomic studies feature multivariate count data from high-throughput DNA sequencing experiments, which often contain many zero values. These zeros can cause artifacts for statistical analyses and multiple modeling approaches have been developed in response. Here, we apply different zero-handling models to gene-expression and microbiome datasets and show models can disagree substantially in terms of identifying the most differentially expressed sequences. Next, to rationally examine how different zero handling models behave, we developed a conceptual framework outlining four types of processes that may give rise to zero values in sequence count data. Last, we performed simulations to test how zero handling models behave in the presence of these different zero generating processes. Our simulations showed that simple count models are sufficient across multiple processes, even when the true underlying process is unknown. On the other hand, a common zero handling technique known as "zero-inflation" was only suitable under a zero generating process associated with an unlikely set of biological and experimental conditions. In concert, our work here suggests several specific guidelines for developing and choosing state-of-the-art models for analyzing sparse sequence count data.

[Source](https://doi.org/10.1016/j.csbj.2020.09.014)

10.1016/j.csbj.2020.09.014

---

## Microbiomic Analysis on Low Abundant Respiratory Biomass Samples; Improved Recovery of Microbial DNA From Bronchoalveolar Lavage Fluid.
 31.10.2020, _M Saladié, JA Caparrós-Martín, P Agudelo-Romero, PAB Wark, SM Stick, F O'Gara_


In recent years the study of the commensal microbiota is driving a remarkable paradigm shift in our understanding of human physiology. However, intrinsic technical difficulties associated with investigating the Microbiomics of some body niches are hampering the development of new knowledge. This is particularly the case when investigating the functional role played by the human microbiota in modulating the physiology of key organ systems. A major hurdle in investigating specific Microbiome communities is linked to low bacterial density and susceptibility to bias caused by environmental contamination. To prevent such inaccuracies due to background processing noise, harmonized tools for Microbiomic and bioinformatics practices have been recommended globally. The fact that the impact of this undesirable variability is negatively correlated with the DNA concentration in the sample highlights the necessity to improve existing DNA isolation protocols. In this report, we developed and tested a protocol to more efficiently recover bacterial DNA from low volumes of bronchoalveolar lavage fluid obtained from infants and adults. We have compared the efficiency of the described method with that of a commercially available kit for microbiome analysis in body fluids. We show that this new methodological approach performs better in terms of extraction efficiency. As opposed to commercial kits, the DNA extracts obtained with this new protocol were clearly distinguishable from the negative extraction controls in terms of 16S copy number and Microbiome community profiles. Altogether, we described a cost-efficient protocol that can facilitate microbiome research in low-biomass human niches.

[Source](https://doi.org/10.3389/fmicb.2020.572504)

10.3389/fmicb.2020.572504

---

## Jumping the green wall: The use of PNA-DNA clamps to enhance microbiome sampling depth in wildlife microbiome research.
 05.11.2020, _L Víquez-R, R Fleischer, K Wilhelm, M Tschapka, S Sommer_


As microbiome research moves away from model organisms to wildlife, new challenges for microbiome high-throughput sequencing arise caused by the variety of wildlife diets. High levels of contamination are commonly observed emanating from the host (mitochondria) or diet (chloroplast). Such high contamination levels affect the overall sequencing depth of wildlife samples thus decreasing statistical power and leading to poor performance in downstream analysis. We developed an amplification protocol utilizing PNA-DNA clamps to maximize the use of resources and to increase the sampling depth of true microbiome sequences in samples with high levels of plastid contamination. We chose two study organisms, a bat (Leptonyteris yerbabuenae) and a bird (Mimus parvulus), both relying on heavy plant-based diets that sometimes lead to traces of plant-based fecal material producing high contamination signals from chloroplasts and mitochondria. On average, our protocol yielded a 13-fold increase in bacterial sequence amplification compared with the standard protocol (Earth Microbiome Protocol) used in wildlife research. For both focal species, we were able to increase significantly the percentage of sequences available for downstream analyses after the filtering of plastids and mitochondria. Our study presents the first results obtained by using PNA-DNA clamps to block the PCR amplification of chloroplast and mitochondrial DNA from the diet in the gut microbiome of wildlife. The method involves a cost-effective molecular technique instead of the filtering out of unwanted sequencing reads. As 33% and 26% of birds and bats, respectively, have a plant-based diet, the tool that we present here will optimize the sequencing and analysis of wild microbiomes.

[Source](https://doi.org/10.1002/ece3.6814)

10.1002/ece3.6814

---

