## Preservation of conjugated primary bile acids by oxygenation of the small intestinal microbiota in vitro.
 10.05.2024, _J Firrman, ES Friedman, A Hecht, WC Strange, AB Narrowe, K Mahalak, GD Wu, L Liu_


Bile acids play a critical role in the emulsification of dietary lipids, a critical step in the primary function of the small intestine, which is the digestion and absorption of food. Primary bile acids delivered into the small intestine are conjugated to enhance functionality, in part, by increasing aqueous solubility and preventing passive diffusion of bile acids out of the gut lumen. Bile acid function can be disrupted by the gut microbiota via the deconjugation of primary bile acids by bile salt hydrolases (BSHs), leading to their conversion into secondary bile acids through the expression of bacterial bile acid-inducible genes, a process often observed in malabsorption due to small intestinal bacterial overgrowth. By modeling the small intestinal microbiota in vitro using human small intestinal ileostomy effluent as the inocula, we show here that the infusion of physiologically relevant levels of oxygen, normally found in the proximal small intestine, reduced deconjugation of primary bile acids, in part, through the expansion of bacterial taxa known to have a low abundance of BSHs. Further recapitulating the small intestinal bile acid composition of the small intestine, limited conversion of primary into secondary bile acids was observed. Remarkably, these effects were preserved among four separate communities, each inoculated with a different small intestinal microbiota, despite a high degree of taxonomic variability under both anoxic and aerobic conditions. In total, these results provide evidence for a previously unrecognized role that the oxygenated environment of the small intestine plays in the maintenance of normal digestive physiology.

[Source](https://doi.org/10.1128/mbio.00943-24)

10.1128/mbio.00943-24

---

## HIV rebound in HIV controllers is associated with a specific fecal microbiome profile.
 10.05.2024, _Y Cai, O Podlaha, SG Deeks, C Brinson, MN Ramgopal, E DeJesus, A Mills, P Shalit, M Abdel-Mohsen, L Zhang, CR de Vries, E Vendrame, D SenGupta, JJ Wallin_


HIV infection is associated with gut dysbiosis, and microbiome variability may affect HIV control when antiretroviral therapy (ART) is stopped. The TLR7 agonist, vesatolimod, was previously associated with a modest delay in viral rebound following analytical treatment interruption in HIV controllers (HCs). Using a retrospective analysis of fecal samples from HCs treated with vesatolimod or placebo (NCT03060447), people with chronic HIV (CH; NCT02858401) or without HIV (PWOH), we examined fecal microbiome profile in HCs before/after treatment, and in CH and PWOH. Microbiome diversity and abundance were compared between groups to investigate the association between specific phyla/species, immune biomarkers, and viral outcomes during treatment interruption. Although there were no significant differences in gut microbiome diversity between people with and without HIV, HCs, and CH shared common features that distinguished them from PWOH. there was a trend toward greater microbiome diversity among HCs. Treatment with vesatolimod reduced dysbiosis in HCs. Firmicutes positively correlated with T-cell activation, while Bacteroidetes and Euryarchaeota inversely correlated with TLR7-mediated immune activation. Specific types of fecal microbiome abundance (e.g. Alistipes putredinis) positively correlated with HIV rebound. In conclusion, variability in the composition of the fecal microbiome is associated with markers of immune activation following vesatolimod treatment and ART interruption.

[Source](https://doi.org/10.1002/eji.202350809)

10.1002/eji.202350809

---

## Comparative analysis of the tumor microbiome, molecular profiles, and immune cell abundances by HPV status in mucosal head and neck cancers and their impact on survival.
 12.05.2024, _R Upadhyay, A Dhakal, C Wheeler, R Hoyd, M Jagjit Singh, V Karivedu, P Bhateja, M Bonomi, S Valentin, ME Gamez, DJ Konieczkowski, S Baliga, JC Grecula, DM Blakaj, E Gogineni, DL Mitchell, NC Denko, D Spakowicz, SR Jhawar_


_Humans, Head and Neck Neoplasms, Female, Papillomavirus Infections, Male, Microbiota, Tumor Microenvironment, Squamous Cell Carcinoma of Head and Neck, Prognosis, Middle Aged, Papillomaviridae, Aged_
Head and Neck Squamous Cell Carcinoma (HNSCC) comprises a diverse group of tumors with variable treatment response and prognosis. The tumor microenvironment (TME), which includes microbiome and immune cells, can impact outcomes. Here, we sought to relate the presence of specific microbes, gene expression, and tumor immune infiltration using tumor transcriptomics from The Cancer Genome Atlas (TCGA) and associate these with overall survival (OS). RNA sequencing (RNAseq) from HNSCC tumors in TCGA was processed through the exogenous sequences in tumors and immune cells (exotic) pipeline to identify and quantify low-abundance microbes. The detection of the Papillomaviridae family of viruses assessed HPV status. All statistical analyses were performed using R. A total of 499 RNAseq samples from TCGA were analyzed. HPV was detected in 111 samples (22%), most commonly Alphapapillomavirus 9 (90.1%). The presence of Alphapapillomavirus 9 was associated with improved OS [HR = 0.60 (95%CI: 0.40-0.89, p = .01)]. Among other microbes, Yersinia pseudotuberculosis was associated with the worst survival (HR = 3.88; p = .008), while Pseudomonas viridiflava had the best survival (HR = 0.05; p = .036). Microbial species found more abundant in HPV- tumors included several gram-negative anaerobes. HPV- tumors had a significantly higher abundance of M0 (p &lt; .001) and M2 macrophages (p = .035), while HPV+ tumors had more T regulatory cells (p &lt; .001) and CD8+ T-cells (p &lt; .001). We identified microbes in HNSCC tumor samples significantly associated with survival. A greater abundance of certain anaerobic microbes was seen in HPV tumors and pro-tumorigenic macrophages. These findings suggest that TME can be used to predict patient outcomes and may help identify mechanisms of resistance to systemic therapies.

[Source](https://doi.org/10.1080/15384047.2024.2350249)

10.1080/15384047.2024.2350249

---

## Metatranscriptome and resistome of the endodontic microbiome.
 08.05.2024, _ET Pinheiro, L Karygianni, GTM Candeiro, BG Vilela, LO Dantas, ACC Pereira, BPFA Gomes, T Attin, T Thurnheer, G Russo_


In this study, we used metatranscriptomics for the first time to investigate microbial composition, functional signatures, and antimicrobial resistance (AR) gene expression in endodontic infections.

[Source](https://www.jendodon.com/article/S0099-2399(24)00276-0/abstract)

10.1016/j.joen.2024.03.015

---

## Microbiome of periodontitis and peri-implantitis before and after therapy: Long-read 16S rRNA gene amplicon sequencing.
 08.05.2024, _PS Yu, CC Tu, N Wara-Aswapati, CY Wang, YK Tu, HH Hou, T Ueno, IH Chen, KL Fu, HY Li, YW Chen_


The microbial profiles of peri-implantitis and periodontitis (PT) are inconclusive. The controversies mainly arise from the differences in sampling sites, targeted gene fragment, and microbiome analysis techniques. The objective of this study was to explore the microbiomes of peri-implantitis (PI), control implants (CI), PT and control teeth (CT), and the microbial change of PI after nonsurgical treatment (PIAT).

[Source](https://doi.org/10.1111/jre.13269)

10.1111/jre.13269

---

## <em>Akkermansia muciniphila</em>: a deworming partner independent of type 2 immunity

[Source](https://doi.org/10.1080/19490976.2024.2338947)

---

## The gut microbiome in alcohol use disorder and alcohol-associated liver disease: A systematic review of clinical studies.
 08.05.2024, _EN Grodin, EM Burnette, C Rodriguez, JA Fulcher, LA Ray_


Evidence suggests that a relationship exists between the gut microbiome and the pathogenesis of alcohol use disorder (AUD) and alcohol-associated liver disease (AALD). This systematic review identified studies that investigated the gut microbiome in individuals with an AUD or an AALD. A search was conducted on October 27, 2022, in PubMed, Web of Science, and Embase databases. Fifty studies satisfied eligibility criteria. Most studies found evidence for gut dysbiosis in individuals with AUD and AALD. Microbiome intervention studies have mostly been conducted in AALD patients; fecal microbial transplant interventions show the most promise. Because most studies were conducted cross-sectionally, the causal relationship between the gut microbiome and alcohol use is unknown. Furthermore, almost all studies have been conducted in predominantly male populations, leaving critical questions regarding sex differences and generalizability of the findings. The study summaries and recommendations provided in this review seek to identify areas for further research and to highlight potential gut microbial interventions for treating AUD and AALD.

[Source](https://doi.org/10.1111/acer.15338)

10.1111/acer.15338

---

## Bile acid-microbiota crosstalk in hepatitis B virus infection.
 09.05.2024, _J Wang, H Xu, Z Liu, Y Cao, S Chen, R Hou, Y Zhou, Y Wang_


Hepatitis B virus (HBV) is a hepatotropic non-cytopathic virus characterized by liver-specific gene expression. HBV infection highjacks bile acid metabolism, notably impairing bile acid uptake via sodium taurocholate cotransporting polypeptide (NTCP), which is a functional receptor for HBV entry. Concurrently, HBV infection induces changes in bile acid synthesis and the size of the bile acid pool. Conversely, bile acid facilitates HBV replication and expression through the signaling molecule farnesoid X receptor (FXR), a nuclear receptor activated by bile acid. However, in HepaRG cells and primary hepatocytes, FXR agonists suppress HBV RNA expression and the synthesis and secretion of DNA. In the gut, the size and composition of the bile acid pool significantly influence the gut microbiota. In turn, the gut microbiota impacts bile acid metabolism and innate immunity, potentially promoting HBV clearance. Thus, the bile acid-gut microbiota axis represents a complex and evolving relationship in the context of HBV infection. This review explores the interplay between bile acid and gut microbiota in HBV infection and discusses the development of HBV entry inhibitors targeting NTCP.

[Source](https://doi.org/10.1111/jgh.16604)

10.1111/jgh.16604

---

## Ecological insights into hematopoiesis regulation: unraveling the influence of gut microbiota.
 10.05.2024, _K Zheng, Z Wei, W Li_


_Hematopoiesis, Gastrointestinal Microbiome, Humans, Hematopoietic Stem Cells, Animals, Signal Transduction, Bacteria, Gastrointestinal Tract, Bone Marrow_
The gut microbiota constitutes a vast ecological system within the human body, forming a mutually interdependent entity with the host. In recent years, advancements in molecular biology technologies have provided a clearer understanding of the role of the gut microbiota. They not only influence the local immune status and metabolic functions of the host's intestinal tract but also impact the functional transformation of hematopoietic stem cells (HSCs) through the gut-blood axis. In this review, we will discuss the role of the gut microbiota in influencing hematopoiesis. We analyze the interactions between HSCs and other cellular components, with a particular emphasis on the direct functional regulation of HSCs by the gut microbiota and their indirect influence through cellular components in the bone marrow microenvironment. Additionally, we propose potential control targets for signaling pathways triggered by the gut microbiota to regulate hematopoietic function, filling crucial knowledge gaps in the development of this research field.

[Source](https://doi.org/10.1080/19490976.2024.2350784)

10.1080/19490976.2024.2350784

---

## Interactions between host and gut microbiota in gestational diabetes mellitus and their impacts on offspring.
 13.05.2024, _S Wang, Z Cui, H Yang_


_Diabetes, Gestational, Humans, Gastrointestinal Microbiome, Pregnancy, Female, Dysbiosis, Fatty Acids, Volatile, Bacteria, Host Microbial Interactions, Lipopolysaccharides_
Gestational diabetes mellitus (GDM) is characterized by insulin resistance and low-grade inflammation, and most studies have demonstrated gut dysbiosis in GDM pregnancies. Overall, they were manifested as a reduction in microbiome diversity and richness, depleted short chain fatty acid (SCFA)-producing genera and a dominant of Gram-negative pathogens releasing lipopolysaccharide (LPS). The SCFAs functioned as energy substance or signaling molecules to interact with host locally and beyond the gut. LPS contributed to pathophysiology of diseases through activating Toll-like receptor 4 (TLR4) and involved in inflammatory responses. The gut microbiome dysbiosis was not only closely related with GDM, it was also vital to fetal health through vertical transmission. In this review, we summarized gut microbiota signature in GDM pregnancies of each trimester, and presented a brief introduction of microbiome derived SCFAs. We then discussed mechanisms of microbiome-host interactions in the physiopathology of GDM and associated metabolic disorders. Finally, we compared offspring microbiota composition from GDM with that from normal pregnancies, and described the possible mechanism.

[Source](https://doi.org/10.1186/s12866-024-03255-y)

10.1186/s12866-024-03255-y

---

## Mechanisms of microbiota modulation: Implications for health, disease, and therapeutic interventions.
 14.05.2024, _OP Ugwu, EU Alum, MB Okon, EI Obeagu_


_Humans, Fecal Microbiota Transplantation, Gastrointestinal Microbiome, Anti-Bacterial Agents, Probiotics, Gastrointestinal Diseases_
Microbiota modulation, the intentional change in the structure and function of the microbial community, is an emerging trajectory that holds the promise to mitigate an infinite number of health issues. The present review illustrates the underlying principles of microbiota modulation and the various applications of this fundamental process to human health, healthcare management, and pharmacologic interventions. Different strategies, directing on dietary interventions, fecal microbiota transplantation, treatment with antibiotics, bacteriophages, microbiome engineering, and modulation of the immune system, are described in detail. This therapeutic implication is reflected in clinical applications to gastrointestinal disorders and immune-mediated diseases for microbiota-modulating agents. In addition to this, the review outlines the challenges of translating researched outcomes into clinical practice to consider safety and provides insights into future research directions of this rapidly developing area.

[Source](https://doi.org/10.1097/MD.0000000000038088)

10.1097/MD.0000000000038088

---

## <em>Lactobacillus paracasei </em>Jlus66 relieves DSS-induced ulcerative colitis in a murine model by maintaining intestinal barrier integrity, inhibiting inflammation, and improving intestinal microbiota structure

[Source](https://doi.org/10.1007/s00394-024-03419-6)

---

## Conjugated linoleic acid metabolite impact in colorectal cancer: a potential microbiome-based precision nutrition approach.
 10.05.2024, _A González, A Fullaondo, J Rodríguez, C Tirnauca, I Odriozola, A Odriozola_


Colorectal cancer (CRC) is the second most deadly and the third most diagnosed cancer in both sexes worldwide. CRC pathogenesis is associated with risk factors such as genetics, alcohol, smoking, sedentariness, obesity, unbalanced diets, and gut microbiota dysbiosis. The gut microbiota is the microbial community living in symbiosis in the intestine, in a dynamic balance vital for health. Increasing evidence underscores the influence of specific gut microbiota bacterial species on CRC incidence and pathogenesis. In this regard, conjugated linoleic acid (CLA) metabolites produced by certain gut microbiota have demonstrated an anticarcinogenic effect in CRC, influencing pathways for inflammation, proliferation, and apoptosis. CLA production occurs naturally in the rumen, and human bioavailability is through the consumption of food derived from ruminants. In recent years, biotechnological attempts to increase CLA bioavailability in humans have been unfruitful. Therefore, the conversion of essential dietary linoleic acid to CLA metabolite by specific intestinal bacteria has become a promising process. This article reviews the evidence regarding CLA and CLA-producing bacteria as therapeutic agents against CRC and investigates the best strategy for increasing the yield and bioavailability of CLA. Given the potential and limitations of the present strategies, a new microbiome-based precision nutrition approach based on endogenous CLA production by human gut bacteria is proposed. A literature search in the PubMed and PubMed Central databases identified 794 papers on human gut bacteria associated with CLA production. Of these, 51 studies exploring association consistency were selected. After excluding 19 papers, due to health concerns or discrepancies between studies, 32 papers were selected for analysis, encompassing data for 38 CLA-producing bacteria, such as Bifidobacterium and Lactobacillus species. The information was analyzed by a bioinformatics food recommendation system patented by our research group, Phymofood (EP22382095). This paper presents a new microbiome-based precision nutrition approach targeting CLA-producing gut bacterial species to maximize the anticarcinogenic effect of CLA in CRC.

[Source](https://doi.org/10.1093/nutrit/nuae046)

10.1093/nutrit/nuae046

---

## Aging amplifies a gut microbiota immunogenic signature linked to heightened inflammation.
 10.05.2024, _ME Caetano-Silva, A Shrestha, AF Duff, D Kontic, PC Brewster, MC Kasperek, CH Lin, DA Wainwright, D Hernandez-Saavedra, JA Woods, MT Bailey, TW Buford, JM Allen_


Aging is associated with low-grade inflammation that increases the risk of infection and disease, yet the underlying mechanisms remain unclear. Gut microbiota composition shifts with age, harboring microbes with varied immunogenic capacities. We hypothesized the gut microbiota acts as an active driver of low-grade inflammation during aging. Microbiome patterns in aged mice strongly associated with signs of bacterial-induced barrier disruption and immune infiltration, including marked increased levels of circulating lipopolysaccharide (LPS)-binding protein (LBP) and colonic calprotectin. Ex vivo immunogenicity assays revealed that both colonic contents and mucosa of aged mice harbored increased capacity to activate toll-like receptor 4 (TLR4) whereas TLR5 signaling was unchanged. We found patterns of elevated innate inflammatory signaling (colonic Il6, Tnf, and Tlr4) and endotoxemia (circulating LBP) in young germ-free mice after 4 weeks of colonization with intestinal contents from aged mice compared with young counterparts, thus providing a direct link between aging-induced shifts in microbiota immunogenicity and host inflammation. Additionally, we discovered that the gut microbiota of aged mice exhibited unique responses to a broad-spectrum antibiotic challenge (Abx), with sustained elevation in Escherichia (Proteobacteria) and altered TLR5 immunogenicity 7 days post-Abx cessation. Together, these data indicate that old age results in a gut microbiota that differentially acts on TLR signaling pathways of the innate immune system. We found that these age-associated microbiota immunogenic signatures are less resilient to challenge and strongly linked to host inflammatory status. Gut microbiota immunogenic signatures should be thus considered as critical factors in mediating chronic inflammatory diseases disproportionally impacting older populations.

[Source](https://doi.org/10.1111/acel.14190)

10.1111/acel.14190

---

## Bacteroidota inhibit microglia clearance of amyloid-beta and promote plaque deposition in Alzheimer's disease mouse models.
 11.05.2024, _C Wasén, LC Beauchamp, J Vincentini, S Li, DS LeServe, C Gauthier, JR Lopes, TG Moreira, MN Ekwudo, Z Yin, P da Silva, RK Krishnan, O Butovsky, LM Cox, HL Weiner_


_Animals, Microglia, Alzheimer Disease, Amyloid beta-Peptides, Plaque, Amyloid, Disease Models, Animal, Female, Phagocytosis, Mice, Male, Mice, Transgenic, Bacteroides fragilis, Gastrointestinal Microbiome, Humans, Mice, Inbred C57BL, Hippocampus_
The gut microbiota and microglia play critical roles in Alzheimer's disease (AD), and elevated Bacteroides is correlated with cerebrospinal fluid amyloid-β (Aβ) and tau levels in AD. We hypothesize that Bacteroides contributes to AD by modulating microglia. Here we show that administering Bacteroides fragilis to APP/PS1-21 mice increases Aβ plaques in females, modulates cortical amyloid processing gene expression, and down regulates phagocytosis and protein degradation microglial gene expression. We further show that administering Bacteroides fragilis to aged wild-type male and female mice suppresses microglial uptake of Aβ1-42 injected into the hippocampus. Depleting murine Bacteroidota with metronidazole decreases amyloid load in aged 5xFAD mice, and activates microglial pathways related to phagocytosis, cytokine signaling, and lysosomal degradation. Taken together, our study demonstrates that members of the Bacteroidota phylum contribute to AD pathogenesis by suppressing microglia phagocytic function, which leads to impaired Aβ clearance and accumulation of amyloid plaques.

[Source](https://doi.org/10.1038/s41467-024-47683-w)

10.1038/s41467-024-47683-w

---

## <em>Prevotella copri </em>promotes vascular calcification via lipopolysaccharide through activation of NF-κB signaling pathway

[Source](https://doi.org/10.1080/19490976.2024.2351532)

---

## Temporal dynamics of fecal microbiota community succession in broiler chickens, calves, and piglets under aerobic exposure.
 08.05.2024, _P Oladele, J Ngo, T Chang, TA Johnson_


Researchers have extensively studied the effect of oxygen on the growth and survival of bacteria. However, the impact of oxygen on bacterial community structure, particularly its ability to select for taxa within the context of a complex microbial community, is still unclear. In a 21-day microcosm experiment, we investigated the effect of aerobic exposure on the fecal community structure and succession pattern in broiler, calf, and piglet feces (n = 10 for each feces type). Bacterial diversity decreased and community structure changed rapidly in the broiler microbiome (P &lt; 0.001), while the fecal community of calves and piglets, which have higher initial diversity, was stable after initial exposure but decreased in diversity after 3 days (P &lt; 0.001). The response to aerobic exposure was host animal specific, but in all three animals, the change in community structure was driven by a decrease in anaerobic species, primarily belonging to Firmicutes and Bacteroidetes (except in broilers where Bacteroidetes increased), along with an increase in aerobic species belonging to Proteobacteria and Actinobacteria. Using random forest regression, we identified microbial features that predict aerobic exposure. In all three animals, host-beneficial Prevotella-related ASVs decreased after exposure, while ASVs belonging to Acinetobacter, Corynbacterium, and Tissierella were increased. The decrease of Prevotella was rapid in broilers but delayed in calves and piglets. Knowing when these pathobionts increase in abundance after aerobic exposure could inform farm sanitation practices and could be important in designing animal experiments that modulate the microbiome.IMPORTANCEThe fecal microbial community is contained within a dynamic ecosystem of interacting microbes that varies in biotic and abiotic components across different animal species. Although oxygen affects bacterial growth, its specific impact on the structure of complex communities, such as those found in feces, and how these effects vary between different animal species are poorly understood. In this study, we demonstrate that the effect of aerobic exposure on the fecal microbiota was host-animal-specific, primarily driven by a decrease in Firmicutes and Bacteroidetes, but accompanied by an increase in Actinobacteria, Proteobacteria, and other pathobionts. Interestingly, we observed that more complex communities from pig and cattle exhibited initial resilience, while a less diverse community from broilers displayed a rapid response to aerobic exposure. Our findings offer insights that can inform farm sanitation practices, as well as experimental design, sample collection, and processing protocols for microbiome studies across various animal species.

[Source](https://doi.org/10.1128/spectrum.04084-23)

10.1128/spectrum.04084-23

---

## Characterizing the blood microbiota in healthy and febrile domestic cats via 16s rRNA sequencing.
 11.05.2024, _L Kitson, AAMJ Becker, K Hartmann, M Bergmann, P Sepulveda-Garcia, N Canales, A Muller_


_Animals, Cats, RNA, Ribosomal, 16S, Microbiota, Fever, Female, Male, Bacteria, Cat Diseases_
This study aimed to evaluate the blood bacterial microbiota in healthy and febrile cats. High-quality sequencing reads from the 16S rRNA gene variable region V3-V4 were obtained from genomic blood DNA belonging to 145 healthy cats, and 140 febrile cats. Comparisons between the blood microbiota of healthy and febrile cats revealed dominant presence of Actinobacteria, followed by Firmicutes and Proteobacteria, and a lower relative abundance of Bacteroidetes. Upon lower taxonomic levels, the bacterial composition was significantly different between healthy and febrile cats. The families Faecalibacterium and Kineothrix (Firmicutes), and Phyllobacterium (Proteobacteria) experienced increased abundance in febrile samples. Whereas Thioprofundum (Proteobacteria) demonstrated a significant decrease in abundance in febrile. The bacterial composition and beta diversity within febrile cats was different according to the affected body system (Oral/GI, systemic, skin, and respiratory) at both family and genus levels. Sex and age were not significant factors affecting the blood microbiota of febrile cats nor healthy ones. Age was different between young adult and mature adult healthy cats. Alpha diversity was unaffected by any factors. Overall, the findings suggest that age, health status and nature of disease are significant factors affecting blood microbiota diversity and composition in cats, but sex is not.

[Source](https://doi.org/10.1038/s41598-024-61023-4)

10.1038/s41598-024-61023-4

---

## Metagenomic sequencing and reconstruction of 82 microbial genomes from barley seed communities.
 13.05.2024, _KA Tshisekedi, P De Maayer, A Botes_


_Hordeum, Seeds, Metagenome, Microbiota, Metagenomics, Genome, Microbial, Genome, Bacterial, Bacteria_
Barley (Hordeum vulgare) is essential to global food systems and the brewing industry. Its physiological traits and microbial communities determine malt quality. Although microbes influence barley from seed health to fermentation, there is a gap in metagenomic insights during seed storage. Crucially, elucidating the changes in microbial composition associated with barley seeds is imperative for understanding how these fluctuations can impact seed health and ultimately, influence both agricultural yield and quality of barley-derived products. Whole metagenomes were sequenced from eight barley seed samples obtained at different storage time points from harvest to nine months. After binning, 82 metagenome-assembled genomes (MAGs) belonging to 26 distinct bacterial genera were assembled, with a substantial proportion of potential novel species. Most of our MAG dataset (61%) showed over 90% genome completeness. This pioneering barley seed microbial genome retrieval provides insights into species diversity and structure, laying the groundwork for understanding barley seed microbiome interactions at the genome level.

[Source](https://doi.org/10.1038/s41597-024-03332-x)

10.1038/s41597-024-03332-x

---

## Modulation of rhizosphere microbial community and metabolites by bio-functionalized nanoscale silicon oxide alleviates cadmium-induced phytotoxicity in bayberry plants.
 14.05.2024, _T Ahmed, L Shou, J Guo, M Noman, Y Qi, Y Yao, HA Masood, M Rizwan, MA Ali, HM Ali, B Li, X Qi_


Cadmium (Cd) is an extremely toxic heavy metal that can originate from industrial activities and accumulate in agricultural soils. This study investigates the potential of biologically synthesized silicon oxide nanoparticles (Bio-SiNPs) in alleviating Cd toxicity in bayberry plants. Bio-SiNPs were synthesized using the bacterial strain Chryseobacterium sp. RTN3 and thoroughly characterized using advanced techniques. A pot experiment results demonstrated that Cd stress substantially reduced leaves biomass, photosynthesis efficiency, antioxidant enzyme activity, and induced oxidative damage in bayberry (Myrica rubra) plants. However, Bio-SiNPs application at 200 mg kg-1 significantly enhanced plant biomass, chlorophyll content (26.4 %), net photosynthetic rate (8.6 %), antioxidant enzyme levels, and mitigated reactive oxygen species production under Cd stress. Bio-SiNPs modulated key stress-related phytohormones by increasing salicylic acid (13.2 %) and abscisic acid (13.7 %) contents in plants. Bio-SiNPs augmented Si deposition on root surfaces, preserving normal ultrastructure in leaf cells. Additionally, 16S rRNA gene sequencing demonstrated that Bio-SiNPs treatment favorably reshaped structure and abundance of specific bacterial groups (Proteobacteria, Actinobacteriota, and Acidobacteriota) in the rhizosphere. Notably, Bio-SiNPs application significantly modulated the key metabolites (phenylacetaldehyde, glycitein, maslinic acid and methylmalonic acid) under both normal and Cd stress conditions. Overall, this study highlights that bio-nanoremediation using Bio-SiNPs enhances tolerance to Cd stress in bayberry plants by beneficially modulating biochemical, microbial, and metabolic attributes.

[Source](https://doi.org/10.1016/j.scitotenv.2024.173068)

10.1016/j.scitotenv.2024.173068

---

## Phyllosphere fungal diversity generates pervasive nonadditive effects on plant performance.
 08.05.2024, _BK Almeida, EH Tran, ME Afkhami_


Plants naturally harbor diverse microbiomes that can dramatically impact their health and productivity. However, it remains unclear how fungal microbiome diversity, especially in the phyllosphere, impacts intermicrobial interactions and consequent nonadditive effects on plant productivity. Combining manipulative experiments, field collections, culturing, microbiome sequencing, and synthetic consortia, we experimentally tested for the first time how foliar fungal community diversity impacts plant productivity. We inoculated morning glories (Ipomoea hederifolia L.) with 32 phyllosphere consortia of either low or high diversity or with single fungal taxa, and measured effects on plant productivity and allocation. We found the following: (1) nonadditive effects were pervasive with 56% of fungal consortia interacting synergistically or antagonistically to impact plant productivity, including some consortia capable of generating acute synergism (e.g. &gt; 1000% increase in productivity above the additive expectation), (2) interactions among 'commensal' fungi were responsible for this nonadditivity in diverse consortia, (3) synergistic interactions were approximately four times stronger than antagonistic effects, (4) fungal diversity affected the magnitude but not frequency or direction of nonadditivity, and (5) diversity affected plant performance nonlinearly with the highest performance in low-diversity treatments. These findings highlight the importance of interpreting plant-microbiome interactions under a framework that incorporates intermicrobial interactions and nonadditive outcomes to understand natural complexity.

[Source](https://doi.org/10.1111/nph.19792)

10.1111/nph.19792

---

## Deciphering the functional and structural complexity of the Solar Lake flat mat microbial benthic communities.
 10.05.2024, _RZ Abdallah, AHA Elbehery, SF Ahmed, A Ouf, MN Malash, W Liesack, R Siam_


The Solar Lake in Taba, Egypt, encompasses one of the few modern-day microbial mats' systems metabolically analogous to Precambrian stromatolites. Solar Lake benthic communities and their adaptation to the Lake's unique limnological cycle have not been described for over two decades. In this study, we revisit the flat mat and describe the summer's shallow water versus exposed microbial community; the latter occurs in response to the seasonal partial receding of water. We employed metagenomic NovaSeq-6000 shotgun sequencing and 16S rRNA, mcrA, and dsrB quantitative PCR. A total of 292 medium-to-high-quality metagenome-assembled genomes (MAGs) were reconstructed. At the structural level, Candidatus Aenigmatarchaeota, Micrarchaeota, and Omnitrophota MAGs were exclusively detected in the shallow-water mats, whereas Halobacteria and Myxococcota MAGs were specific to the exposed microbial mat. Functionally, genes involved in reactive oxygen species (ROS) detoxification and osmotic pressure were more abundant in the exposed than in the shallow-water microbial mats, whereas genes involved in sulfate reduction/oxidation and nitrogen fixation were ubiquitously detected. Genes involved in the utilization of methylated amines for methane production were predominant when compared with genes associated with alternative methanogenesis pathways. Solar Lake methanogen MAGs belonged to Methanosarcinia, Bathyarchaeia, Candidatus Methanofastidiosales, and Archaeoglobales. The latter had the genetic capacity for anaerobic methane oxidation. Moreover, Coleofasciculus chthonoplastes, previously reported to dominate the winter shallow-water flat mat, had a substantial presence in the summer. These findings reveal the taxonomic and biochemical microbial zonation of the exposed and shallow-water Solar Lake flat mat benthic community and their capacity to ecologically adapt to the summer water recession.

[Source](https://doi.org/10.1128/msystems.00095-24)

10.1128/msystems.00095-24

---

## Microbiome processing of organic nitrogen input supports growth and cyanotoxin production of <em>Microcystis aeruginosa </em>cultures

[Source](https://doi.org/10.1093/ismejo/wrae082)

---

## Genomic fingerprints of the world's soil ecosystems.
 09.05.2024, _EB Graham, VA Garayburu-Caruso, R Wu, J Zheng, R McClure, GD Jones_


Despite the explosion of soil metagenomic data, we lack a synthesized understanding of patterns in the distribution and functions of soil microorganisms. These patterns are critical to predictions of soil microbiome responses to climate change and resulting feedbacks that regulate greenhouse gas release from soils. To address this gap, we assay 1,512 manually curated soil metagenomes using complementary annotation databases, read-based taxonomy, and machine learning to extract multidimensional genomic fingerprints of global soil microbiomes. Our objective is to uncover novel biogeographical patterns of soil microbiomes across environmental factors and ecological biomes with high molecular resolution. We reveal shifts in the potential for (i) microbial nutrient acquisition across pH gradients; (ii) stress-, transport-, and redox-based processes across changes in soil bulk density; and (iii) greenhouse gas emissions across biomes. We also use an unsupervised approach to reveal a collection of soils with distinct genomic signatures, characterized by coordinated changes in soil organic carbon, nitrogen, and cation exchange capacity and in bulk density and clay content that may ultimately reflect soil environments with high microbial activity. Genomic fingerprints for these soils highlight the importance of resource scavenging, plant-microbe interactions, fungi, and heterotrophic metabolisms. Across all analyses, we observed phylogenetic coherence in soil microbiomes-more closely related microorganisms tended to move congruently in response to soil factors. Collectively, the genomic fingerprints uncovered here present a basis for global patterns in the microbial mechanisms underlying soil biogeochemistry and help beget tractable microbial reaction networks for incorporation into process-based models of soil carbon and nutrient cycling.IMPORTANCEWe address a critical gap in our understanding of soil microorganisms and their functions, which have a profound impact on our environment. We analyzed 1,512 global soils with advanced analytics to create detailed genetic profiles (fingerprints) of soil microbiomes. Our work reveals novel patterns in how microorganisms are distributed across different soil environments. For instance, we discovered shifts in microbial potential to acquire nutrients in relation to soil acidity, as well as changes in stress responses and potential greenhouse gas emissions linked to soil structure. We also identified soils with putative high activity that had unique genomic characteristics surrounding resource acquisition, plant-microbe interactions, and fungal activity. Finally, we observed that closely related microorganisms tend to respond in similar ways to changes in their surroundings. Our work is a significant step toward comprehending the intricate world of soil microorganisms and its role in the global climate.

[Source](https://doi.org/10.1128/msystems.01112-23)

10.1128/msystems.01112-23

---

## Insights into cyanobacterial blooms through the lens of omics.
 09.05.2024, _V Van Le, QG Tran, SR Ko, HM Oh, CY Ahn_


Cyanobacteria are oxygen-producing photosynthetic bacteria that convert carbon dioxide into biomass upon exposure to sunlight. However, favorable conditions cause harmful cyanobacterial blooms (HCBs), which are the dense accumulation of biomass at the water surface or subsurface, posing threats to freshwater ecosystems and human health. Understanding the mechanisms underlying cyanobacterial bloom formation is crucial for effective management. In this regard, recent advancements in omics technologies have provided valuable insights into HCBs, which have raised expectations to develop more effective control methods in the near future. This systematic literature review aims to present the genomic architecture, adaptive mechanisms, microbial interactions, and ecological impacts of HCBs through the lens of omics. Genomic analysis indicates that the genome plasticity of cyanobacteria has enabled their resilience and effective adaptation to environmental changes. Transcriptomic investigations have revealed that cyanobacteria use various strategies for adapting to environmental stress. Additionally, metagenomic and metatranscriptomic analyses have emphasized the significant role of the microbial community in regulating HCBs. Finally, we offer perspectives on potential opportunities for further research in this field.

[Source](https://doi.org/10.1016/j.scitotenv.2024.173028)

10.1016/j.scitotenv.2024.173028

---

## fastCCLasso: a fast and efficient algorithm for estimating correlation matrix from compositional data.
 11.05.2024, _S Zhang, H Fang, T Hu_


The composition and structure of microbial communities on the body surface are closely related to human health. The interaction relationship among microbes can help us understand the formation of the microecological environment and the biological mechanism by which microorganisms influence host health. With the help of high-throughput sequencing technologies, microbial abundances in a natural environment can be directly measured without the isolation of microorganisms in culture. Sequencing experiments in microbiome studies can measure the relative abundance of microbes, which is called compositional data. Although there are already many methods for correlation analysis for compositional data, the computation time or accuracy still needs to be improved for current microbiome studies.

[Source](https://doi.org/10.1093/bioinformatics/btae314)

10.1093/bioinformatics/btae314

---

## A Standard Pipeline for Analyzing the Endometrial Microbiome.
 08.05.2024, _M de Medeiros Garcia Torres, DCF Lanza_


The endometrial microbiome is a rapidly advancing field of research, particularly in obstetrics and gynecology, as it has been found to be linked with obstetric complications and potential impacts on fertility. The diversity of microorganisms presents in the endometrium, along with their metabolites, can influence reproductive outcomes by modulating the local immune environment of the uterus. However, a major challenge in advancing our understanding of the endometrial microbiota lies in the heterogeneity of available studies, which vary in terms of patient selection, control groups, collection methods and analysis methodologies. In this study, we propose a detailed pipeline for endometrial microbiome analysis, based on the most comprehensive prospective of 64 studies that have investigated the endometrial microbiome up to the present. Additionally, our review suggests that a dominance of Lactobacilli in the endometrium may be associated with improved reproductive prognosis, including higher implantation rates and lower miscarriage rates. By establishing a standardized pipeline, we aim to facilitate future research, enabling better comparison and correlation of bacterial communities with the health status of patients, including fertility-related issues.

[Source](https://doi.org/10.1007/s43032-024-01557-0)

10.1007/s43032-024-01557-0

---

