## The respiratory tract microbiome, the pathogen load, and clinical interventions define severity of bacterial pneumonia.
 26.08.2023, _AE Pérez-Cobas, C Ginevra, C Rusniok, S Jarraud, C Buchrieser_


Bacterial pneumonia is a considerable problem worldwide. Here, we follow the inter-kingdom respiratory tract microbiome (RTM) of a unique cohort of 38 hospitalized patients (n = 97 samples) with pneumonia caused by Legionella pneumophila. The RTM composition is characterized by diversity drops early in hospitalization and ecological species replacement. RTMs with the highest bacterial and fungal loads show low diversity and pathogen enrichment, suggesting high biomass as a biomarker for secondary and/or co-infections. The RTM structure is defined by a "commensal" cluster associated with a healthy RTM and a "pathogen" enriched one, suggesting that the cluster equilibrium drives the microbiome to recovery or dysbiosis. Legionella biomass correlates with disease severity and co-morbidities, while clinical interventions influence the RTM dynamics. Fungi, archaea, and protozoa seem to contribute to progress of pneumonia. Thus, the interplay of the RTM equilibrium, the pathogen load dynamics, and clinical interventions play a critical role in patient recovery.

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00320-8)

10.1016/j.xcrm.2023.101167

---

## A phenome-wide association study (PheWAS) to identify the health impacts of 4-cresol sulfate in the Nagahama Study.
 28.08.2023, _H Ou, S Kawaguchi, K Sonomura, T Kawaguchi, S Kitada, S Yoshiji, F Brial, D Gauguier, J Xia, F Matsuda_


_Humans, Diabetes Mellitus, Type 2, Sulfuric Acid Esters, Phenomics, Endophenotypes_
Gut-microbiota derived metabolites are important regulators of host biology and metabolism. To understand the impacts of the microbial metabolite 4-cresol sulfate (4-CS) on four chronic diseases [type 2 diabetes mellitus, metabolic syndrome (MetS), non-alcoholic fatty liver disease, and chronic kidney disease (CKD)], we conducted association analyses of plasma 4-CS quantified by liquid chromatography coupled to mass spectrometry (LC-MS) in 3641 participants of the Nagahama study. Our results validated the elevation of 4-CS in CKD and identified a reducing trend in MetS. To delineate the holistic effects of 4-CS, we performed a phenome-wide association analysis (PheWAS) with 937 intermediate biological and behavioral traits. We detected associations between 4-CS and 39 phenotypes related to blood pressure regulation, hepatic and renal functions, hematology, sleep quality, intraocular pressure, ion regulation, ketone and fatty acid metabolisms, disease history and dietary habits. Among them, 19 PheWAS significant traits, including fatty acids and 14 blood pressure indices, were correlated with MetS, suggesting that 4-CS is a potential biomarker for MetS. Consistent associations of this gut microbial-derived metabolite on multiple endophenotypes underlying distinct etiopathogenesis support its role in the overall host health, with prospects of probiotic-based therapeutic solutions in chronic diseases.

[Source](https://doi.org/10.1038/s41598-023-40697-2)

10.1038/s41598-023-40697-2

---

## A meta-analysis of tissue microbial biomarkers for recurrence and metastasis in multiple cancer types.
 28.08.2023, _X Li, X Yuan, X Zhu, C Li, L Ji, K Lv, G Tian, K Ning, J Yang_


_Humans, Neoplasm Recurrence, Local, Carcinoma, Squamous Cell, Stomach Neoplasms, Microbial Interactions, Biomarkers_
Background. Local recurrence and distant metastasis are the main causes of death in patients with cancer. Only considering species abundance changes when identifying markers of recurrence and metastasis in patients hinders finding solutions.Hypothesis. Consideration of microbial abundance changes and microbial interactions facilitates the identification of microbial markers of tumour recurrence and metastasis.Aim. This study aims to simultaneously consider microbial abundance changes and microbial interactions to identify microbial markers of recurrence and metastasis in multiple cancer types.Method. One thousand one hundred and six non-RM (patients without recurrence and metastasis within 3 years after initial surgery) tissue samples and 912 RM (patients with recurrence or metastasis within 3 years after initial surgery) tissue samples representing 11 cancer types were collected from The Cancer Genome Atlas (TCGA).Results. Tumour tissue bacterial composition differed significantly among 11 cancers. Among them, the tissue microbiome of four cancers, head and neck squamous cell carcinoma (HNSC), lung squamous cell carcinoma (LUSC), stomach adenocarcinoma (STAD) and uterine corpus endometrial carcinoma (UCEC), showed relatively good performance in predicting recurrence and metastasis in patients, with areas under the receiver operating characteristic curve (AUCs) of 0.78, 0.74, 0.91 and 0.93, respectively. Considering both species abundance changes and microbial interactions for the four cancers, a combination of nine genera (Niastella, Schlesneria, Thioalkalivibrio, Phaeobacter, Sphaerotilus, Thiomonas, Lawsonia, Actinobacillus and Spiroplasma) performed best in predicting patient survival.Conclusion. Taken together, our results imply that comprehensive consideration of microbial abundance changes and microbial interactions is helpful for mining bacterial markers that carry prognostic information.

[Source](https://doi.org/10.1099/jmm.0.001744)

10.1099/jmm.0.001744

---

## Microbe Profile: <em>Ruminococcus gnavus:</em> the yin and yang of human gut symbionts

[Source](https://doi.org/10.1099/mic.0.001383)

---

## Oral <em>Porphyromonas gingivalis</em> infection affects intestinal microbiota and promotes atherosclerosis

[Source](https://doi.org/10.1111/jcpe.13864)

---

## Gut microbiota and fecal short chain fatty acids differ with adiposity and country of origin: the METS-microbiome study.
 28.08.2023, _G Ecklu-Mensah, C Choo-Kang, MG Maseng, S Donato, P Bovet, B Viswanathan, K Bedu-Addo, J Plange-Rhule, P Oti Boateng, TE Forrester, M Williams, EV Lambert, D Rae, N Sinyanya, A Luke, BT Layden, S O'Keefe, JA Gilbert, LR Dugas_


_Humans, Gastrointestinal Microbiome, Adiposity, Ghana, Microbiota, Obesity, Fatty Acids, Volatile, Glucose, Hypertension_
The relationship between microbiota, short chain fatty acids (SCFAs), and obesity remains enigmatic. We employ amplicon sequencing and targeted metabolomics in a large (n = 1904) African origin cohort from Ghana, South Africa, Jamaica, Seychelles, and the US. Microbiota diversity and fecal SCFAs are greatest in Ghanaians, and lowest in Americans, representing each end of the urbanization spectrum. Obesity is significantly associated with a reduction in SCFA concentration, microbial diversity, and SCFA synthesizing bacteria, with country of origin being the strongest explanatory factor. Diabetes, glucose state, hypertension, obesity, and sex can be accurately predicted from the global microbiota, but when analyzed at the level of country, predictive accuracy is only universally maintained for sex. Diabetes, glucose, and hypertension are only predictive in certain low-income countries. Our findings suggest that adiposity-related microbiota differences differ between low-to-middle-income compared to high-income countries. Further investigation is needed to determine the factors driving this association.

[Source](https://doi.org/10.1038/s41467-023-40874-x)

10.1038/s41467-023-40874-x

---

## Very early life microbiome and metabolome correlates with primary vaccination variability in children.
 23.08.2023, _M Shaffer, K Best, C Tang, X Liang, S Schulz, E Gonzalez, CH White, TP Wyche, J Kang, H Wesseling, BD Topçuoğlu, T Cairns, TR Sana, RM Kaufhold, JM Maritz, CH Woelk, G Swaminathan, JE Norton, ME Pichichero_


Despite multiple vaccine doses early in life, a substantial proportion of infants do not mount protective responses. In this study, we followed a cohort of children over the first 2 years of life, collecting microbiome and metabolome data longitudinally to investigate correlates of lower and higher responses to primary vaccinations. We found that the stool and nasopharyngeal microbiome developed with age, though demonstrated divergent timing and patterns in maturation. When measured at child age 2 months, evenness of genera in the stool microbiome correlated with lower vaccine responses, upregulated metabolome genes that encode for lipid A biosynthesis and oxidative phosphorylation correlated with higher vaccine responses, and abundance of phenylpyruvic acid in serum correlated with lower vaccine responses, measured 10 months later. Antibiotic exposure was associated with low vaccine response, and microbiome/metabolome features at child age 2 months, before childhood vaccinations commenced, correlated with variations in vaccine responses measured at child age 1 year. These results indicate that there may be potential to intervene before first childhood vaccinations to improve later protection. IMPORTANCE We show that simultaneous study of stool and nasopharyngeal microbiome reveals divergent timing and patterns of maturation, suggesting that local mucosal factors may influence microbiome composition in the gut and respiratory system. Antibiotic exposure in early life as occurs commonly, may have an adverse effect on vaccine responsiveness. Abundance of gut and/or nasopharyngeal bacteria with the machinery to produce lipopolysaccharide-a toll-like receptor 4 agonist-may positively affect future vaccine protection, potentially by acting as a natural adjuvant. The increased levels of serum phenylpyruvic acid in infants with lower vaccine-induced antibody levels suggest an increased abundance of hydrogen peroxide, leading to more oxidative stress in low vaccine-responding infants.

[Source](https://doi.org/10.1128/msystems.00661-23)

10.1128/msystems.00661-23

---

## Carbohydrate quality, fecal microbiota and cardiometabolic health in older adults: a cohort study.
 26.08.2023, _A Atzeni, SK Nishi, N Babio, C Belzer, P Konstanti, J Vioque, D Corella, O Castañer, J Vidal, I Moreno-Indias, L Torres-Collado, EM Asensio, M Fitó, AM Gomez-Perez, A Arias, M Ruiz-Canela, FB Hu, FJ Tinahones, J Salas-Salvadó_


_Aged, Humans, Gastrointestinal Microbiome, Cohort Studies, Microbiota, Bacteroidetes, Cardiovascular Diseases_
The impact of carbohydrate quality, measured by the carbohydrate quality index (CQI), on gut microbiota and health has been scarcely investigated. The aim of this study was to cross-sectionally and longitudinally explore the relationships between CQI, fecal microbiota, and cardiometabolic risk factors in an elderly Mediterranean population at high cardiovascular risk. At baseline and 1-year, CQI was assessed from food frequency questionnaires data, cardiometabolic risk factors were measured, and fecal microbiota profiled from 16S sequencing. Multivariable-adjusted linear regression models were fitted to assess the associations between tertiles of baseline CQI, fecal microbiota, and cardiometabolic risk factors at baseline, and between tertiles of 1-year change in CQI, 1-year change in fecal microbiota and cardiometabolic risk factors. Cross-sectionally, higher CQI was positively associated with Shannon alpha diversity index, and abundance of genera Faecalibacterium and Christensenellaceae R7 group, and negatively associated with the abundance of Odoribacter, and uncultured Rhodospirillales genera. Some of these genera were associated with higher glycated hemoglobin and lower body mass index. In addition, we observed a positive association between CQI, and some pathways related with the metabolism of butyrate precursors and plants-origin molecules. Longitudinally, 1-year improvement in CQI was associated with a concurrent increase in the abundance of genera Butyrivibrio. Increased abundance of this genera was associated with 1-year improvement in insulin status. These observations suggest that a better quality of carbohydrate intake is associated with improved metabolic health, and this improvement could be modulated by greater alpha diversity and abundance of specific genera linked to beneficial metabolic outcomes.

[Source](https://doi.org/10.1080/19490976.2023.2246185)

10.1080/19490976.2023.2246185

---

## Multi-omics analysis revealing the interplay between gut microbiome and the host following opioid use.
 26.08.2023, _U Kolli, R Jalodia, S Moidunny, PK Singh, Y Ban, J Tao, GN Cantu, E Valdes, S Ramakrishnan, S Roy_


_Animals, Mice, Gastrointestinal Microbiome, Analgesics, Opioid, Dysbiosis, Multiomics, Opioid-Related Disorders, Riboflavin, Morphine Derivatives, Lipids_
Opioid crisis is an ongoing epidemic since the past several decades in the United States. Opioid use-associated microbial dysbiosis is emerging as a key regulator of intestinal homeostasis and behavioral responses to opioid. However, the mechanistic insight into the role of microbial community in modulating host response is unavailable. To uncover the role of opioid-induced dysbiosis in disrupting intestinal homeostasis we utilized whole genome sequencing, untargeted metabolomics, and mRNA sequencing to identify changes in microbiome, metabolome, and host transcriptome respectively. Morphine treatment resulted in significant expansion of Parasuterella excrementihominis, Burkholderiales bacterium 1_1_47, Enterococcus faecalis, Enterorhabdus caecimuris and depletion of Lactobacillus johnsonii. These changes correlated with alterations in lipid metabolites and flavonoids. Significant alteration in microbial metabolism (metabolism of lipids, amino acids, vitamins and cofactors) and increased expression of virulence factors and biosynthesis of lipopolysaccharides (LPS) and lipoteichoic acid (LTA) were observed in microbiome of morphine-treated animals. In concurrence with changes in microbiome and metabolome extensive changes in innate and adaptive immune response, lipid metabolism, and gut barrier dysfunction were observed in the host transcriptome. Microbiome depleted mice displayed lower levels of inflammation, immune response and tissue destruction compared to mice harboring a dysbiotic microbiome in response to morphine treatment, thus establishing dysbiotic microbiome as mediator of morphine gut pathophysiology. Integrative analysis of multi-omics data highlighted the associations between Parasutterella excrementihominis, Burkholderiales bacterium 1_1_47, Enterococcus faecalis, Enterorhabdus caecimuris and altered levels of riboflavin, flavonoids, and lipid metabolites including phosphocholines, carnitines, bile acids, and ethanolamines with host gene expression changes involved in inflammation and barrier integrity of intestine. Omic analysis also highlighted the role of probiotic bacteria Lactobacillus johnsonii, metabolites flavonoids and riboflavin that were depleted with morphine as important factors for intestinal homeostasis. This study presents for the first time ever an interactive view of morphine-induced changes in microbial metabolism, strain level gut microbiome analysis and comprehensive view of changes in gut transcriptome. We also identified areas of potential therapeutic interventions to limit microbial dysbiosis and present a unique resource to the opioid research community.

[Source](https://doi.org/10.1080/19490976.2023.2246184)

10.1080/19490976.2023.2246184

---

## The gut microbiota reprograms intestinal lipid metabolism through long noncoding RNA Snhg9.
 28.08.2023, _Y Wang, M Wang, J Chen, Y Li, Z Kuang, C Dende, P Raj, G Quinn, Z Hu, T Srinivasan, B Hassell, KA Ruhn, CL Behrendt, T Liang, X Dou, Z Song, LV Hooper_


_Animals, Mice, Gastrointestinal Microbiome, Lipid Metabolism, RNA, Long Noncoding, Immunity, Innate, PPAR gamma, Lymphocytes, Lipids, Mammals_
The intestinal microbiota regulates mammalian lipid absorption, metabolism, and storage. We report that the microbiota reprograms intestinal lipid metabolism in mice by repressing the expression of long noncoding RNA (lncRNA) Snhg9 (small nucleolar RNA host gene 9) in small intestinal epithelial cells. Snhg9 suppressed the activity of peroxisome proliferator-activated receptor γ (PPARγ)-a central regulator of lipid metabolism-by dissociating the PPARγ inhibitor sirtuin 1 from cell cycle and apoptosis protein 2 (CCAR2). Forced expression of Snhg9 in the intestinal epithelium of conventional mice impaired lipid absorption, reduced body fat, and protected against diet-induced obesity. The microbiota repressed Snhg9 expression through an immune relay encompassing myeloid cells and group 3 innate lymphoid cells. Our findings thus identify an unanticipated role for a lncRNA in microbial control of host metabolism.

[Source](https://doi.org/10.1126/science.ade0522)

10.1126/science.ade0522

---

## <em>Staphylococcus epidermidis </em>activates keratinocyte cytokine expression and promotes skin inflammation through the production of phenol-soluble modulins 

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(23)01035-5)

---

## Targeting nonalcoholic fatty liver disease via gut microbiome-centered therapies.
 25.08.2023, _M Koning, H Herrema, M Nieuwdorp, AS Meijnikman_


Humans possess abundant amounts of microorganisms, including bacteria, fungi, viruses, and archaea, in their gut. Patients with nonalcoholic fatty liver disease (NAFLD) exhibit alterations in their gut microbiome and an impaired gut barrier function. Preclinical studies emphasize the significance of the gut microbiome in the pathogenesis of NAFLD. In this overview, we explore how adjusting the gut microbiome could serve as an innovative therapeutic strategy for NAFLD. We provide a summary of current information on untargeted techniques such as probiotics and fecal microbiota transplantation, as well as targeted microbiome-focused therapies including engineered bacteria, prebiotics, postbiotics, and phages for the treatment of NAFLD.

[Source](https://doi.org/10.1080/19490976.2023.2226922)

10.1080/19490976.2023.2226922

---

## Intracellular<em> Fusobacterium nucleatum</em> infection increases METTL3-mediated m6A methylation to promote the metastasis of esophageal squamous cell carcinoma

[Source](https://doi.org/10.1016/j.jare.2023.08.014)

---

## Characteristics of gut microbiota determine effects of specific probiotics strains in patients with functional constipation.
 25.08.2023, _H Zhang, L Sun, Z Zhao, Y Zhou, Y Liu, N Zhang, J Yan, S Wang, R Li, J Zhang, X Wang, W Li, Y Pan, M Wang, B Luo, M Li, Z Sun, Y Zhao, Y Nie_



[Source](https://doi.org/10.1097/CM9.0000000000002826)

10.1097/CM9.0000000000002826

---

## <em>Lacticaseibacillus rhamnosus</em> TF318 prevents depressive behavior in rats by inhibiting HPA-axis hyperactivity and upregulating BDNF expression

[Source](https://doi.org/10.1016/j.neulet.2023.137460)

---

## Role of mucin glycosylation in the gut microbiota-brain axis of core 3 O-glycan deficient mice.
 28.08.2023, _E Coletto, GM Savva, D Latousakis, M Pontifex, EH Crost, L Vaux, A Telatin, K Bergstrom, D Vauzour, N Juge_


_Animals, Mice, Mucins, Brain-Gut Axis, Glycosylation, Gastrointestinal Microbiome, Brain, Polysaccharides_
Alterations in intestinal mucin glycosylation have been associated with increased intestinal permeability and sensitivity to inflammation and infection. Here, we used mice lacking core 3-derived O-glycans (C3GnT-/-) to investigate the effect of impaired mucin glycosylation in the gut-brain axis. C3GnT-/- mice showed altered microbial metabolites in the caecum associated with brain function such as dimethylglycine and N-acetyl-L-tyrosine profiles as compared to C3GnT+/+ littermates. In the brain, polysialylated-neural cell adhesion molecule (PSA-NCAM)-positive granule cells showed an aberrant phenotype in the dentate gyrus of C3GnT-/- mice. This was accompanied by a trend towards decreased expression levels of PSA as well as ZO-1 and occludin as compared to C3GnT+/+. Behavioural studies showed a decrease in the recognition memory of C3GnT-/- mice as compared to C3GnT+/+ mice. Combined, these results support the role of mucin O-glycosylation in the gut in potentially influencing brain function which may be facilitated by the passage of microbial metabolites through an impaired gut barrier.

[Source](https://doi.org/10.1038/s41598-023-40497-8)

10.1038/s41598-023-40497-8

---

## Outer membrane vesicles from a mosquito commensal mediate targeted killing of <em>Plasmodium</em> parasites via the phosphatidylcholine scavenging pathway

[Source](https://doi.org/10.1038/s41467-023-40887-6)

---

## Whole genome analyses of toxicants tolerance genes of <em>Apis mellifera </em>gut-derived <em>Enterococcus faecium </em>strains

[Source](https://doi.org/10.1186/s12864-023-09590-0)

---

## The endohyphal microbiome: current progress and challenges for scaling down integrative multi-omic microbiome research.
 28.08.2023, _JM Kelliher, AJ Robinson, R Longley, LYD Johnson, BT Hanson, DP Morales, G Cailleau, P Junier, G Bonito, PSG Chain_


_Multiomics, Data Analysis, Eukaryota, Microbiota, Prokaryotic Cells_
As microbiome research has progressed, it has become clear that most, if not all, eukaryotic organisms are hosts to microbiomes composed of prokaryotes, other eukaryotes, and viruses. Fungi have only recently been considered holobionts with their own microbiomes, as filamentous fungi have been found to harbor bacteria (including cyanobacteria), mycoviruses, other fungi, and whole algal cells within their hyphae. Constituents of this complex endohyphal microbiome have been interrogated using multi-omic approaches. However, a lack of tools, techniques, and standardization for integrative multi-omics for small-scale microbiomes (e.g., intracellular microbiomes) has limited progress towards investigating and understanding the total diversity of the endohyphal microbiome and its functional impacts on fungal hosts. Understanding microbiome impacts on fungal hosts will advance explorations of how "microbiomes within microbiomes" affect broader microbial community dynamics and ecological functions. Progress to date as well as ongoing challenges of performing integrative multi-omics on the endohyphal microbiome is discussed herein. Addressing the challenges associated with the sample extraction, sample preparation, multi-omic data generation, and multi-omic data analysis and integration will help advance current knowledge of the endohyphal microbiome and provide a road map for shrinking microbiome investigations to smaller scales. Video Abstract.

[Source](https://doi.org/10.1186/s40168-023-01634-7)

10.1186/s40168-023-01634-7

---

## Microbiome dataset of bacterial and fungal communities in anthosphere of twelve different wild plants in South Korea.
 24.08.2023, _J Kim, Y Cui, H Lee, SJ Chun_


This dataset provides detailed profiles of bacterial and fungal communities associated with flowers (anthosphere) of 12 different plant species collected from remote and secluded locations characterized by a flourishing and diverse plant ecosystem. In total, 144 flower samples were collected from 12 different wild plants. Bacterial 16S rRNA and fungal ITS genes obtained using the Illumina Miseq approach were used to describe the anthosphere. Metadata and raw sequences obtained in this study are available from the National Center for Biotechnology Information (BioProject ID: PRJNA983070). Amplicon Sequence Variants (ASVs) of bacteria and fungi were analyzed using the DADA2 pipeline. After quality filtering, trimming, and removing the chimeric sequences, 2076 bacterial and 2152 fungal ASVs were identified in the anthosphere. Burkholderiales and Enterobacterales in bacteria, and Pleosporales in fungi were the predominant groups in the anthosphere regardless of the plant species. Among the twelve different plant species, Forsythia koreana exhibited the highest abundance of both bacterial and fungal groups. This dataset represents a detailed exploration of the anthosphere in the most abundant and commonly observed plant species in South Korea, and provides new insights into the microbial communities and interactions of the anthosphere.

[Source](https://doi.org/10.1016/j.dib.2023.109470)

10.1016/j.dib.2023.109470

---

## Anthropogenic activities mediate stratification and stability of microbial communities in freshwater sediments.
 28.08.2023, _X Zhou, JT Lennon, X Lu, A Ruan_


_Humans, Anthropogenic Effects, Ecosystem, Lakes, China, Earth, Planet_
Freshwater sediment microbes are crucial decomposers that play a key role in regulating biogeochemical cycles and greenhouse gas emissions. They often exhibit a highly ordered structure along depth profiles. This stratification not only reflects redox effects but also provides valuable insights into historical transitions, as sediments serve as important archives for tracing environmental history. The Anthropocene, a candidate geological epoch, has recently garnered significant attention. However, the human impact on sediment zonation under the cover of natural redox niches remains poorly understood. Dam construction stands as one of the most far-reaching anthropogenic modifications of aquatic ecosystems. Here we attempted to identify the ecological imprint of damming on freshwater sediment microbiome.

[Source](https://doi.org/10.1186/s40168-023-01612-z)

10.1186/s40168-023-01612-z

---

## Long-term agro-management strategies shape soil bacterial community structure in dryland wheat systems.
 28.08.2023, _S Singh, S Singh, SB Lukas, S Machado, A Nouri, F Calderon, ER Rieke, SB Cappellazzi_


_Soil, Triticum, Carbon Dioxide, Agriculture, Calcinosis, Carbon_
Soil microbes play a crucial role in soil organic matter decomposition and nutrient cycling and are influenced by management practices. Therefore, quantifying the impacts of various agricultural management practices on soil microbiomes and their activity is crucial for making informed management decisions. This study aimed to assess the impact of various management systems on soil bacterial abundance and diversity, soil enzyme activities and carbon mineralization potential in wheat-based systems. To accomplish this, soil samples from 0 to 15 cm depth were collected from ongoing long-term field trials in eastern Oregon region under wheat (Triticum aestivum L.)-fallow (WF), WF with different tillage (WT), wheat-pea (Pisum sativum L.) (WP), WF under different crop residue management (CR) and natural undisturbed/unmanaged grassland pasture (GP). These trials consisted of an array of treatments like tillage intensities, nitrogen rates, organic amendments, and seasonal residue burning. This study was a part of the Soil Health Institute's North American Project to Evaluate Soil Health measurements (NAPESHM). Bacterial community structure was determined using amplicon sequencing of the V4 region of 16SrRNA genes and followed the protocols of the Earth Microbiome Project. In addition, extracellular enzyme activities, and carbon mineralization potential (1d-CO2) were measured. Among different trials, 1d-CO2 in WT, WP, and CR studies averaged 53%, 51% and 87% lower than GP systems, respectively. Enzyme activities were significantly greater in GP compared to the other managements and followed similar trend as respiration. We observed higher evenness in GP and higher richness in spring residue burning treatment of CR study. Our results indicated that species evenness is perhaps a better indicator of soil health in comparison to other indices in dryland wheat systems.

[Source](https://doi.org/10.1038/s41598-023-41216-z)

10.1038/s41598-023-41216-z

---

## Microbial community structure and functional prediction in five full-scale industrial park wastewater treatment plants.
 25.08.2023, _T Tong, J Tong, K Xue, Y Li, J Yu, Y Wei_


The development of industrial parks has become an important global trend contributing significantly to economic and industrial growth. However, this growth comes at a cost, as the treatment of multisource industrial wastewater generated in these parks can be difficult owing to its complex composition. Microorganisms play a critical role in pollutant removal during industrial park wastewater treatment. Therefore, our study focused on the microbial communities in five full-scale industrial park wastewater treatment plants (WWTPs) with similar treatment processes and capacities. The results showed that denitrifying bacteria were dominant in almost every process section of all the plants, with heterotrophic denitrification being the main pathway. Moreover, autotrophic sulfur denitrification and methane oxidation denitrification may contribute to total nitrogen (TN) removal. In plants where the influent had low levels of COD and TN, dominant bacteria included oligotrophic microorganisms like Prosthecobacter (2.88 % ~ 10.02 %) and hgcI_clade (2.05 % ~ 9.49 %). Heavy metal metabolizing microorganisms, such as Norank_f__PHOS-HE36 (3.96 % ~ 5.36 %) and Sediminibacterium (1.86 % ~ 5.34 %), were prevalent in oxidation ditch and secondary settling tanks in certain plants. Functional Annotation of Prokaryotic Taxa (FAPROTAX) revealed that microbial communities in the regulation and hydrolysis tanks exhibited higher potential activity in the nitrogen (N) and sulfur (S) cycles than those in the oxidation ditch. Sulfate/sulfite reduction was common in most plants, whereas the potential occurrence of sulfide compounds and thiosulfate oxidation tended to be higher in plants with a relatively high sulfate concentration and low COD content in their influent. Our study provides a new understanding of the microbial community in full-scale industrial park WWTPs and highlights the critical role of microorganisms in the treatment of industrial wastewater.

[Source](https://doi.org/10.1016/j.scitotenv.2023.166529)

10.1016/j.scitotenv.2023.166529

---

## Deterministic Development of Soil Microbial Communities in Disturbed Soils Depends on Microbial Biomass of the Bioinoculum.
 25.08.2023, _Y Pinheiro Alves de Souza, M Schloter, W Weisser, S Schulz_


Despite its enormous importance for ecosystem services, factors driving microbial recolonization of soils after disturbance are still poorly understood. Here, we compared the microbial recolonization patterns of a disturbed, autoclaved soil using different amounts of the original non-disturbed soil as inoculum. By using this approach, we manipulated microbial biomass, but did not change microbial diversity of the inoculum. We followed the development of a new soil microbiome after reinoculation over a period of 4 weeks using a molecular barcoding approach as well as qPCR. Focus was given on the assessment of bacteria and archaea. We could show that 1 week after inoculation in all inoculated treatments bacterial biomass exceeded the values from the original soil as a consequence of high dissolved organic carbon (DOC) concentrations in the disturbed soil resulting from the disturbance. This high biomass was persistent over the complete experimental period. In line with the high DOC concentrations, in the first 2 weeks of incubation, copiotrophic bacteria dominated the community, which derived from the inoculum used. Only in the disturbed control soils which did not receive a microbial inoculum, recolonization pattern differed. In contrast, archaeal biomass did not recover over the experimental period and recolonization was strongly triggered by amount of inoculated original soil added. Interestingly, the variability between replicates of the same inoculation density decreased with increasing biomass in the inoculum, indicating a deterministic development of soil microbiomes if higher numbers of cells are used for reinoculation.

[Source](https://doi.org/10.1007/s00248-023-02285-9)

10.1007/s00248-023-02285-9

---

## Microbial communities related to the sulfur cycle in the Sansha Yongle Blue Hole

[Source](https://doi.org/10.1128/spectrum.01149-23)

---

## A study of microbial diversity in a biofertilizer consortium

[Source](https://doi.org/10.1371/journal.pone.0286285)

---

## Metagenomic profiling pipelines improve taxonomic classification for 16S amplicon sequencing data.
 28.08.2023, _AR Odom, T Faits, E Castro-Nallar, KA Crandall, WE Johnson_


_Humans, Metagenomics, RNA, Ribosomal, 16S, Metagenome, Microbiota, Bone Plates, Cercozoa, Polyarteritis Nodosa_
Most experiments studying bacterial microbiomes rely on the PCR amplification of all or part of the gene for the 16S rRNA subunit, which serves as a biomarker for identifying and quantifying the various taxa present in a microbiome sample. Several computational methods exist for analyzing 16S amplicon sequencing. However, the most-used bioinformatics tools cannot produce high quality genus-level or species-level taxonomic calls and may underestimate the potential accuracy of these calls. We used 16S sequencing data from mock bacterial communities to evaluate the sensitivity and specificity of several bioinformatics pipelines and genomic reference libraries used for microbiome analyses, concentrating on measuring the accuracy of species-level taxonomic assignments of 16S amplicon reads. We evaluated the tools DADA2, QIIME 2, Mothur, PathoScope 2, and Kraken 2 in conjunction with reference libraries from Greengenes, SILVA, Kraken 2, and RefSeq. Profiling tools were compared using publicly available mock community data from several sources, comprising 136 samples with varied species richness and evenness, several different amplified regions within the 16S rRNA gene, and both DNA spike-ins and cDNA from collections of plated cells. PathoScope 2 and Kraken 2, both tools designed for whole-genome metagenomics, outperformed DADA2, QIIME 2 using the DADA2 plugin, and Mothur, which are theoretically specialized for 16S analyses. Evaluations of reference libraries identified the SILVA and RefSeq/Kraken 2 Standard libraries as superior in accuracy compared to Greengenes. These findings support PathoScope and Kraken 2 as fully capable, competitive options for genus- and species-level 16S amplicon sequencing data analysis, whole genome sequencing, and metagenomics data tools.

[Source](https://doi.org/10.1038/s41598-023-40799-x)

10.1038/s41598-023-40799-x

---

## Amplicon-Based Microbiome Profiling: From Second- to Third-Generation Sequencing for Higher Taxonomic Resolution.
 28.08.2023, _E Notario, G Visci, B Fosso, C Gissi, N Tanaskovic, M Rescigno, M Marzano, G Pesole_


_RNA, Ribosomal, 16S, Benchmarking, Computational Biology, Microbiota, Technology_
The 16S rRNA amplicon-based sequencing approach represents the most common and cost-effective strategy with great potential for microbiome profiling. The use of second-generation sequencing (NGS) technologies has led to protocols based on the amplification of one or a few hypervariable regions, impacting the outcome of the analysis. Nowadays, comparative studies are necessary to assess different amplicon-based approaches, including the full-locus sequencing currently feasible thanks to third-generation sequencing (TGS) technologies. This study compared three different methods to achieve the deepest microbiome taxonomic characterization: (a) the single-region approach, (b) the multiplex approach, covering several regions of the target gene/region, both based on NGS short reads, and (c) the full-length approach, which analyzes the whole length of the target gene thanks to TGS long reads. Analyses carried out on benchmark microbiome samples, with a known taxonomic composition, highlighted a different classification performance, strongly associated with the type of hypervariable regions and the coverage of the target gene. Indeed, the full-length approach showed the greatest discriminating power, up to species level, also on complex real samples. This study supports the transition from NGS to TGS for the study of the microbiome, even if experimental and bioinformatic improvements are still necessary.

[Source](https://doi.org/10.3390/genes14081567)

10.3390/genes14081567

---

## A flexible quasi-likelihood model for microbiome abundance count data.
 22.08.2023, _Y Shi, H Li, C Wang, J Chen, H Jiang, YT Shih, H Zhang, Y Song, Y Feng, L Liu_


In this article, we present a flexible model for microbiome count data. We consider a quasi-likelihood framework, in which we do not make any assumptions on the distribution of the microbiome count except that its variance is an unknown but smooth function of the mean. By comparing our model to the negative binomial generalized linear model (GLM) and Poisson GLM in simulation studies, we show that our flexible quasi-likelihood method yields valid inferential results. Using a real microbiome study, we demonstrate the utility of our method by examining the relationship between adenomas and microbiota. We also provide an R package "fql" for the application of our method.

[Source](https://doi.org/10.1002/sim.9880)

10.1002/sim.9880

---

## Identification of multivariable Boolean patterns in microbiome and microbial gene composition data.
 24.08.2023, _G Golovko, K Khanipov, V Reyes, I Pinchuk, Y Fofanov_


Virtually every biological system is governed by complex relations among its components. Identifying such relations requires a rigorous or heuristics-based search for patterns among variables/features of a system. Various algorithms have been developed to identify two-dimensional (involving two variables) patterns employing correlation, covariation, mutual information, etc. It seems obvious, however, that comprehensive descriptions of complex biological systems need also to include more complicated multivariable relations, which can only be described using patterns that simultaneously embrace 3, 4, and more variables. The goal of this manuscript is to (a) introduce a novel type of associations (multivariable Boolean patterns) that can be manifested between features of complex systems but cannot be identified (described) by traditional pair-vise metrics; (b) propose patterns classification method, and (c) provide a novel definition of the pattern's strength (pattern's score) able to accommodate heterogeneous multi-omics data. To demonstrate the presence of such patterns, we performed a search for all possible 2-, 3-, and 4-dimensional patterns in historical data from the Human Microbiome Project (15 body sites) and collection of H. pylori genomes associated with gastric ulcers, gastritis, and duodenal ulcers. In all datasets under consideration, we were able to identify hundreds of statistically significant multivariable patterns. These results suggest that such patterns can be common in microbial genomics/microbiomics systems.

[Source](https://doi.org/10.1016/j.biosystems.2023.105007)

10.1016/j.biosystems.2023.105007

---

## Machine learning classification by fitting amplicon sequences to existing OTUs.
 24.08.2023, _CR Armour, KL Sovacool, WL Close, BD Topçuoğlu, J Wiens, PD Schloss_


The ability to use 16S rRNA gene sequence data to train machine learning classification models offers the opportunity to diagnose patients based on the composition of their microbiome. In some applications, the taxonomic resolution that provides the best models may require the use of de novo operational taxonomic units (OTUs) whose composition changes when new data are added. We previously developed a new reference-based approach, OptiFit, that fits new sequence data to existing de novo OTUs without changing the composition of the original OTUs. While OptiFit produces OTUs that are as high quality as de novo OTUs, it is unclear whether this method for fitting new sequence data into existing OTUs will impact the performance of classification models relative to models trained and tested only using de novo OTUs. We used OptiFit to cluster sequences into existing OTUs and evaluated model performance in classifying a dataset containing samples from patients with and without colonic screen relevant neoplasia (SRN). We compared the performance of this model to standard methods including de novo and database-reference-based clustering. We found that using OptiFit performed as well or better in classifying SRNs. OptiFit can streamline the process of classifying new samples by avoiding the need to retrain models using reclustered sequences. IMPORTANCE There is great potential for using microbiome data to aid in diagnosis. A challenge with de novo operational taxonomic unit (OTU)-based classification models is that 16S rRNA gene sequences are often assigned to OTUs based on similarity to other sequences in the dataset. If data are generated from new patients, the old and new sequences must be reclustered to OTUs and the classification model retrained. Yet there is a desire to have a single, validated model that can be widely deployed. To overcome this obstacle, we applied the OptiFit clustering algorithm to fit new sequence data to existing OTUs allowing for reuse of the model. A random forest model implemented using OptiFit performed as well as the traditional reassign and retrain approach. This result shows that it is possible to train and apply machine learning models based on OTU relative abundance data that do not require retraining or the use of a reference database.

[Source](https://doi.org/10.1128/msphere.00336-23)

10.1128/msphere.00336-23

---

## Statistical normalization methods in microbiome data with application to microbiome cancer research.
 28.08.2023, _Y Xia_


_RNA, Ribosomal, 16S, Gastrointestinal Microbiome, Microbiota, Metagenome, Research Design, Neoplasms_
Mounting evidence has shown that gut microbiome is associated with various cancers, including gastrointestinal (GI) tract and non-GI tract cancers. But microbiome data have unique characteristics and pose major challenges when using standard statistical methods causing results to be invalid or misleading. Thus, to analyze microbiome data, it not only needs appropriate statistical methods, but also requires microbiome data to be normalized prior to statistical analysis. Here, we first describe the unique characteristics of microbiome data and the challenges in analyzing them (Section 2). Then, we provide an overall review on the available normalization methods of 16S rRNA and shotgun metagenomic data along with examples of their applications in microbiome cancer research (Section 3). In Section 4, we comprehensively investigate how the normalization methods of 16S rRNA and shotgun metagenomic data are evaluated. Finally, we summarize and conclude with remarks on statistical normalization methods (Section 5). Altogether, this review aims to provide a broad and comprehensive view and remarks on the promises and challenges of the statistical normalization methods in microbiome data with microbiome cancer research examples.

[Source](https://doi.org/10.1080/19490976.2023.2244139)

10.1080/19490976.2023.2244139

---

## Elucidating the role of microbes in celiac disease through gnotobiotic modeling.
 28.08.2023, _A Caminero, EF Verdu, HJ Galipeau_


_Humans, Animals, Mice, Celiac Disease, Glutens, Gastrointestinal Microbiome_
Celiac disease (CeD) is a common immune-mediated disease triggered by the ingestion of gluten in genetically predisposed individuals. CeD is unique in that the trigger (gluten), necessary genes (HLA-DQ2 and DQ8), and the autoantigen (tissue transglutaminase) have been identified, allowing additional environmental co-factors, like the intestinal microbiota, to be studied through relevant in vivo models. Murine models for CeD have come a long way in the past decade and there are now in vitro and in vivo tools available that mimic certain aspects of clinical disease. These models, many of which express the CeD risk genes, have recently been used to study the mechanisms through which the microbiota play a role in CeD pathogenesis through a gnotobiotic approach. Historically, the generation of gnotobiology technology in mid-20th century allowed for the study of immunity and physiology under a complete absence of microbes (axenic) or known colonized status (gnotobiotic). This enabled understanding of mechanisms by which certain bacteria contribute to health and disease. With this perspective, here, we will discuss the various murine models currently being used to study CeD. We will then describe how utilizing axenic and gnotobiotic CeD models has increased our understanding of how microbes influence relevant steps of CeD pathogenesis, and explain key methodology involved in axenic and gnotobiotic modeling.

[Source](https://doi.org/10.1016/bs.mcb.2023.01.017)

10.1016/bs.mcb.2023.01.017

---

