## CAR-T cell therapy-related cytokine release syndrome and therapeutic response is modulated by the gut microbiome in hematologic malignancies.
 09.09.2022, _Y Hu, J Li, F Ni, Z Yang, X Gui, Z Bao, H Zhao, G Wei, Y Wang, M Zhang, R Hong, L Wang, W Wu, M Mohty, A Nagler, AH Chang, MRM van den Brink, MD Li, H Huang_


Immunotherapy utilizing chimeric antigen receptor T cell (CAR-T) therapy holds promise for hematologic malignancies, however, response rates and associated immune-related adverse effects widely vary among patients. Here we show, by comparing diversity and composition of the gut microbiome during different CAR-T therapeutic phases in the clinical trial ChiCTR1800017404, that the gut flora characteristically differs among patients and according to treatment stages, and might also reflect patient response to therapy in relapsed/refractory multiple myeloma (MM; n = 43), acute lympholastic leukemia (ALL; n = 23) and non-Hodgkin lymphoma (NHL; n = 12). We observe significant temporal differences in diversity and abundance of Bifidobacterium, Prevotella, Sutterella, and Collinsella between MM patients in complete remission (n = 24) and those in partial remission (n = 11). Furthermore, we find that patients with severe cytokine release syndrome present with higher abundance of Bifidobacterium, Leuconostoc, Stenotrophomonas, and Staphylococcus, which is reproducible in an independent cohort of 38 MM patients. This study has important implications for understanding the biological role of the microbiome in CAR-T treatment responsiveness of hematologic malignancy patients, and may guide therapeutic intervention to increase efficacy. The success rate of CAR-T cell therapy is high in blood cancers, yet individual patient characteristics might reduce therapeutic benefit. Here we show that therapeutic response in MM, ALL and NHL, and occurrence of severe cytokine release syndrome in multiple myeloma are associated with specific gut microbiome alterations.

[Source](https://doi.org/10.1038/s41467-022-32960-3)

10.1038/s41467-022-32960-3

---

## Untangling the link between the human gut microbiota composition and the severity of the symptoms of the COVID-19 infection.
 10.09.2022, _L Mancabelli, C Milani, F Fontana, GA Lugli, C Tarracchini, A Viappiani, T Ciociola, A Ticinesi, A Nouvenne, T Meschi, F Turroni, M Ventura_


Recent pandemic infection caused by SARS-CoV-2 (COVID-19) led the scientific community to investigate the possible causes contributing to the physiopathology of this disease. In this context, analyses of the intestinal microbiota highlighted possible correlation between host-associated bacterial communities and development of the COVID-19. Nevertheless, a detailed investigation of the role of the human microbiota in the severity of the symptoms of this disease is still lacking. This study performed a comprehensive meta-analysis of 323 fecal samples from public and novel Italian data sets based on the shotgun metagenomic approach. In detail, the comparative analyses revealed possible differences in the microbial biodiversity related to the individual health status, highlighting a species richness decrease in COVID-19 patients with a severe prognosis. Moreover, healthy subjects resulted characterized by a higher abundance of protective and health-supporting bacterial species, while patients affected by COVID-19 disease displayed a significant increase of opportunistic pathogen bacteria involved in developing putrefactive dysbiosis. Furthermore, prediction of the microbiome functional capabilities suggested that individuals affected by COVID-19 subsist in an unbalanced metabolism characterized by an overrepresentation of enzymes involved in the protein metabolism at the expense of carbohydrates-oriented pathways, which can impact on disease severity and in excessive systemic inflammation. This article is protected by copyright. All rights reserved.

[Source](https://doi.org/10.1111/1462-2920.16201)

10.1111/1462-2920.16201

---

## Trimethylamine-N-oxide (TMAO) promotes balloon injury-induced neointimal hyperplasia via upregulating Beclin1 and impairing autophagic flux

[Source](https://doi.org/10.1016/j.biopha.2022.113639)

---

## <em>Weizmannia faecalis</em> sp. nov., isolated from a human stool sample

[Source](https://doi.org/10.1007/s00203-022-03229-6)

---

## <em>Fusobacterium nucleatum</em> aggravates ulcerative colitis through promoting gut microbiota dysbiosis and dysmetabolism

[Source](https://doi.org/10.1002/JPER.22-0205)

---

## The evaluation of fecal microbiota transplantation vs vancomycin in a <em>Clostridioides difficile</em> infection model

[Source](https://doi.org/10.1007/s00253-022-12154-z)

---

## The early-life gut microbiome and vaccine efficacy

[Source](https://doi.org/10.1016/S2666-5247(22)00185-9)

---

## Crucial nuances in understanding (mis)associations between the neonatal microbiome and Cesarean delivery

[Source](https://doi.org/10.1016/j.molmed.2022.07.005)

---

## L-Theanine regulates lipid metabolism by modulating gut microbiota and bile acid metabolism

[Source](https://doi.org/10.1002/jsfa.12222)

---

## Structural and prebiotic activity analysis of the polysaccharide produced by <em>Lactobacillus helveticus</em> SNA12

[Source](https://doi.org/10.1016/j.carbpol.2022.119971)

---

## Unlocking a novel determinant of athletic performance: The role of the gut microbiota, short-chain fatty acids, and biotics in exercise

[Source](https://doi.org/10.1016/j.jshs.2022.09.002)

---

## Comparative analysis of gut microbiota and immune genes linked with the immune system of wild and captive <em>Spodoptera frugiperda</em> (Lepidoptera: Noctuidae)

[Source](https://doi.org/10.1016/j.dci.2022.104530)

---

## A global dataset of microbial community in ticks from metagenome study

[Source](https://doi.org/10.1038/s41597-022-01679-7)

---

## Exposure to dechlorane 602 induces perturbation of gut immunity and microbiota in female mice

[Source](https://doi.org/10.1016/j.envpol.2022.120141)

---

## Dynamics of bacterial and fungal communities of mango: From the tree to ready-to-Eat products

[Source](https://doi.org/10.1016/j.fm.2022.104095)

---

## The microbiome of fly organs and fly-human microbial transfer during decomposition

[Source](https://doi.org/10.1016/j.forsciint.2022.111425)

---

