## Time series strain tracking analysis post fecal transplantation identifies individual specific patterns of fecal dominant donor, recipient, and unrelated microbial strains.
 19.09.2022, _H Koo, CD Morrow_


_Anti-Bacterial Agents, Fecal Microbiota Transplantation, Feces, Nucleotides, Time Factors_
Fecal microbial transplantation (FMT) has been used with the therapeutic intent to change the functions of the gut microbial community in metabolism and host immunity. For most of these therapies, the recipients are not given antibiotics to eliminate the microbial community prior to transplant with donor fecal microbes resulting in the initial gut microbial community following FMT consisting of a consortium of donor and recipient microbes. The detailed analysis of the fecal samples from these FMT over time provides a unique opportunity to study the changes in the gut microbial strain community that occurs following the introduction of new microbial strains (donor) into an established community (recipient).

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274633)

10.1371/journal.pone.0274633

---

## Microbiome complexity shapes metabolism.
 23.09.2022, _L Litichevskiy, CA Thaiss_


_Animals, Gastrointestinal Microbiome, Germ-Free Life, Mice, Microbiota_
Equipped with a novel isolator-housed metabolic cage system, a study in PLOS Biology assessed how the metabolism of mice harboring a defined minimal microbial community (OligoMM12) differs from that of germ-free and conventionally colonized mice.

[Source](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001793)

10.1371/journal.pbio.3001793

---

## Minimal overall divergence of the gut microbiome in an adaptive radiation of <em>Cyprinodon </em>pupfishes despite potential adaptive enrichment for scale-eating

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0273177)

---

## Molecular characterization and biofilm-formation analysis of<em> Listeria monocytogenes</em>, <em>Salmonella </em>spp., and <em>Escherichia coli</em> isolated from Brazilian swine slaughterhouses

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274636)

---

## Ecology and distribution of <em>Leptospira</em> spp., reservoir hosts and environmental interaction in Sri Lanka, with identification of a new strain

[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0010757#sec002)

---

## A simple kit to use computational notebooks for more openness, reproducibility, and productivity in research

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010356)

---

## Bacterial concentration and Campylobacter spp. quantification differ when fresh or ultra-frozen samples are analysed over time using molecular biology and culture-based methods.
 20.09.2022, _F Khattak, S Galgano, J Houdijk_


_Animals, Campylobacter jejuni, Chickens, Culture Techniques, Freezing, Molecular Biology_
The study aimed to delineate the robustness of the culture-based and molecular biology methods to assess the total bacterial concentration and Campylobacter jejuni (C. jejuni) quantification in caecal content, analysed as fresh or after being stored immediately at ultra-low (-80°C) temperature at different time points (for 3, 7, 14, 28 and 62 days post collection). The caecal content was collected from birds that were artificially colonised with C. jejuni (in-vivo), and quantification was performed using both colony-forming unit (CFU) and qPCR. The results showed that storage time affected the output of culture-based analyses but mostly did not alter concentration retrieved via qPCR. After an initial ~4.5 log10 reduction in CFU observed from fresh (day 0) to frozen samples, bacterial concentration retrieved with culture-based methods seemed to be constant in samples frozen for 3 to 62 days, indicating a possible threshold for C. jejuni loss of viability due to effect of storage temperature. Ranking order analyses, revealed that the molecular biology technique was able to attribute somewhat the same relative C. jejuni concentrations to the samples analysed via qPCR. However, day 0 measurements from culture-based methods were associated with the absence of or negatively weak correlations with the rest of the time points, but ranking order was maintained from day 3 onwards. On the other hand, ranking order correlations were less constant when measuring total bacterial concentration through qPCR. The study suggests that if biological samples can't be analysed as fresh (immediately after collection) and have to be stored prior to analysis, then storage at -80°C samples be recommended to avoid the temporal-dependent effects on C. jejuni concentrations. In addition, irrespective of the method of analysis, an initial loss of CFU must be factored in when interpreting the results obtained from frozen samples.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274682)

10.1371/journal.pone.0274682

---

## Bacterial concentration and <em>Campylobacter</em> spp. quantification differ when fresh or ultra-frozen samples are analysed over time using molecular biology and culture-based methods

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274682)

---

## Robust inference of population size histories from genomic sequencing data.
 16.09.2022, _G Upadhya, M Steinrücken_


Unraveling the complex demographic histories of natural populations is a central problem in population genetics. Understanding past demographic events is of general anthropological interest, but is also an important step in establishing accurate null models when identifying adaptive or disease-associated genetic variation. An important class of tools for inferring past population size changes from genomic sequence data are Coalescent Hidden Markov Models (CHMMs). These models make efficient use of the linkage information in population genomic datasets by using the local genealogies relating sampled individuals as latent states that evolve along the chromosome in an HMM framework. Extending these models to large sample sizes is challenging, since the number of possible latent states increases rapidly. Here, we present our method CHIMP (CHMM History-Inference Maximum-Likelihood Procedure), a novel CHMM method for inferring the size history of a population. It can be applied to large samples (hundreds of haplotypes) and only requires unphased genomes as input. The two implementations of CHIMP that we present here use either the height of the genealogical tree (TMRCA) or the total branch length, respectively, as the latent variable at each position in the genome. The requisite transition and emission probabilities are obtained by numerically solving certain systems of differential equations derived from the ancestral process with recombination. The parameters of the population size history are subsequently inferred using an Expectation-Maximization algorithm. In addition, we implement a composite likelihood scheme to allow the method to scale to large sample sizes. We demonstrate the efficiency and accuracy of our method in a variety of benchmark tests using simulated data and present comparisons to other state-of-the-art methods. Specifically, our implementation using TMRCA as the latent variable shows comparable performance and provides accurate estimates of effective population sizes in intermediate and ancient times. Our method is agnostic to the phasing of the data, which makes it a promising alternative in scenarios where high quality data is not available, and has potential applications for pseudo-haploid data.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010419)

10.1371/journal.pcbi.1010419

---

## Rapid genotyping of targeted viral samples using Illumina short-read sequencing data.
 20.09.2022, _A Váradi, E Kaszab, G Kardos, E Prépost, K Szarka, L Laczkó_


_Genome, Genotype, High-Throughput Nucleotide Sequencing, Software_
The most important information about microorganisms might be their accurate genome sequence. Using current Next Generation Sequencing methods, sequencing data can be generated at an unprecedented pace. However, we still lack tools for the automated and accurate reference-based genotyping of viral sequencing reads. This paper presents our pipeline designed to reconstruct the dominant consensus genome of viral samples and analyze their within-host variability. We benchmarked our approach on numerous datasets and showed that the consensus genome of samples could be obtained reliably without further manual data curation. Our pipeline can be a valuable tool for fast identifying viral samples. The pipeline is publicly available on the project's GitHub page (https://github.com/laczkol/QVG).

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0274414)

10.1371/journal.pone.0274414

---

## Improved assessments of bulk milk microbiota composition via sample preparation and DNA extraction methods.
 21.09.2022, _Z Xue, ML Marco_


_Animals, Bacteria, Cattle, DNA, Bacterial, Endopeptidase K, Glycerol, Microbiota, Milk, RNA, Ribosomal, 16S, Reproducibility of Results, Ribonuclease, Pancreatic_
Although bacterial detection by 16S rRNA gene amplicon DNA sequencing is a widely-applied technique, standardized methods for sample preparation and DNA extraction are needed to ensure accuracy, reproducibility, and scalability for automation. To develop these methods for bovine bulk milk, we assembled and tested a bacterial cell mock community (BCMC) containing bacterial species commonly found in milk. The following protocol variations were examined:: BCMC enumeration (colony enumeration or microscopy), sample volume (200 μl to 30 ml), sample storage condition (frozen in PBS or 25% glycerol or exposure to freeze-thaw cycles), cell lysis method (bead-beating, vortex, enzymatic), and DNA extraction procedure (MagMAX Total, MagMAX CORE, and MagMAX Ultra 2.0, with and without either Proteinase K or RNase A). Cell enumeration by microscopy was more accurate for quantification of the BCMC contents. We found that least 10 mL (≥ 104 cells in high quality milk) is needed for reproducible bacterial detection by 16S rRNA gene amplicon DNA sequencing, whereas variations in storage conditions caused minor differences in the BCMC. For DNA extraction and purification, a mild lysis step (bead-beating for 10 s at 4 m/s or vortexing at 1800 rpm for 10 s) paired with the MagMAX Total kit and Proteinase K digestion provided the most accurate representation of the BCMC. Cell lysis procedures conferred the greatest changes to milk microbiota composition and these effects were confirmed to provide similar results for commercial milk samples. Overall, our systematic approach with the BCMC is broadly applicable to other milk, food, and environmental samples therefore recommended for improving accuracy of culture-independent, DNA sequence-based analyses of microbial composition in different habitats.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0267992)

10.1371/journal.pone.0267992

---

