## Intestinal Bacteriophage Therapy: Looking for Optimal Efficacy.
 20.10.2021, _F Javaudin, C Latour, L Debarbieux, Q Lamy-Besnier_


Several human intestinal microbiota studies suggest that bacteriophages, viruses infecting bacteria, play a role in gut homeostasis. Currently, bacteriophages are considered a tool to precisely engineer the intestinal microbiota, but they have also attracted considerable attention as a possible solution to fight against bacterial pathogens resistant to antibiotics. These two applications necessitate bacteriophages to reach and kill their bacterial target within the gut environment. Unfortunately, exploitable clinical data in this field are scarce. Here, we review the administration of bacteriophages to target intestinal bacteria in mammalian experimental models. While bacteriophage amplification in the gut was often confirmed, we found that in most studies, it had no significant impact on the load of the targeted bacteria. In particular, we observed that the outcome of bacteriophage treatments is linked to the behavior of the target bacteria toward each animal model. Treatment efficacy ranges from poor in asymptomatic intestinal carriage to high in intestinal disease. This broad range of efficacy underlines the difficulties to reach a consensus on the impact of bacteriophages in the gut and calls for deeper investigations of key parameters that influence the success of such interventions before launching clinical trials.

[Source](https://doi.org/10.1128/CMR.00136-21)

10.1128/CMR.00136-21

---

## Wildlife symbiotic bacteria are indicators of the health status of the host and its ecosystem.
 20.10.2021, _M Bravo, T Combes, F Martinez-Estrada, D Risco, P Gonçalves, W Garcia-Jimenez, R Cerrato, P Fernandez-Llario, J Gutierrez-Merino_


Lactic acid bacteria (LAB) are gut symbionts that can be used as a model to understand the host-microbiota crosstalk under unpredictable environmental conditions such as wildlife ecosystems. The aim of this study was to determine whether viable LAB can be informative of the health status of wild boar populations. We monitored the genotype and phenotype of LAB based on markers that included safety and phylogenetic origin, antibacterial activity and immunomodulatory properties. A LAB profile dominated by lactobacilli appears to stimulate protective immune responses and relates to strains widely used as probiotics, resulting in a potentially healthy wildlife population whereas microbiota overpopulated by enterococci was observed in a hostile environment. These enterococci were closely related to pathogenic strains that have developed mechanisms to evade innate immune system, posing a potential risk for the host health. Furthermore, our LAB isolates displayed antibacterial properties in a species-dependent manner. Nearly all of them were able to inhibit bacterial pathogens, raising the possibility of using them as a la carte antibiotic alternative in the unexplored field of wildlife disease mitigation. Our study highlights that microbiological characterization of LAB is a useful indicator of wildlife health status and the ecological origin from which they derive. Significance Statement The wildlife symbiotic microbiota is an important component to the greater for greater diversity and functionality of their bacterial populations, influencing the host health and adaptability to its ecosystem. Although many microbes are partly responsible for the development of multiple physiological processes, only certain bacterial groups such as lactic acid bacteria (LAB) have the capacity to overpopulate the gut, promoting health (or disease) when specific genetic and environmental conditions are present. LAB have been exploited in many ways due to their probiotic properties, in particular lactobacilli, however their relationship with wildlife gut-associated microbiota hosts remains to be elucidated. On the other hand, it is unclear whether LAB such as enterococci, which have been associated with detrimental health effects, could lead to disease. These important questions have not been properly addressed in the field of wildlife, and therefore, should be clearly attained.

[Source](https://doi.org/10.1128/AEM.01385-21)

10.1128/AEM.01385-21

---

## High-coverage metabolomics uncovers microbiota-driven biochemical landscape of interorgan transport and gut-brain communication in mice.
 20.10.2021, _Y Lai, CW Liu, Y Yang, YC Hsiao, H Ru, K Lu_


The mammalian gut harbors a complex and dynamic microbial ecosystem: the microbiota. While emerging studies support that microbiota regulates brain function with a few molecular cues suggested, the overall biochemical landscape of the "microbiota-gut-brain axis" remains largely unclear. Here we use high-coverage metabolomics to comparatively profile feces, blood sera, and cerebral cortical brain tissues of germ-free C57BL/6 mice and their age-matched conventionally raised counterparts. Results revealed for all three matrices metabolomic signatures owing to microbiota, yielding hundreds of identified metabolites including 533 altered for feces, 231 for sera, and 58 for brain with numerous significantly enriched pathways involving aromatic amino acids and neurotransmitters. Multicompartmental comparative analyses single out microbiota-derived metabolites potentially implicated in interorgan transport and the gut-brain axis, as exemplified by indoxyl sulfate and trimethylamine-N-oxide. Gender-specific characteristics of these landscapes are discussed. Our findings may be valuable for future research probing microbial influences on host metabolism and gut-brain communication.

[Source](https://doi.org/10.1038/s41467-021-26209-8)

10.1038/s41467-021-26209-8

---

## Analysis of gut microbiota and metabolites in patients with rheumatoid arthritis and identification of potential biomarkers.
 21.10.2021, _Y Chen, C Ma, L Liu, J He, C Zhu, F Zheng, W Dai, X Hong, D Liu, D Tang, Y Dai_


Rheumatoid arthritis (RA) is an autoimmune disease described by joint destruction, synovitis and pannus formation. The gut microbiota acts as an environmental factor that plays an important role in RA, but little research regarding the etiopathogenic mechanisms of the microbiome in RA has been carried out. We used an integrated approach of 16S rRNA gene sequencing and ultrahigh-performance liquid chromatography-mass spectrometry-based metabolomics to analyze the structure and diversity of the intestinal flora and metabolites of the gut microbiota in RA patients compared with healthy subjects. In this study, α-diversity analysis of the gut microbiota showed that there was no significant difference between the healthy control (HC) and RA groups. However, β-diversity analysis showed that there was a significant difference between the two groups. Further analysis of alteration of the gut microbiota revealed that at the phylum level, the relative abundance of p_Bacteroidetes was significantly decreased in the RA group, while that of Verrucomicrobia and Proteobacteria was significantly increased in the RA group. At the genus level, Bacteroides, Faecalibacterium and some probiotics were decreased in the RA group, while 97 genera, including Lactobacillus, Streptococcus and Akkermansia, were increased in the RA group. Seventy-four differentially abundant metabolites were identified between the HC and RA groups, and we identified two potential biomarkers (9,12-octadecadiynoic acid and 10Z-nonadecenoic acid) in RA.

[Source](https://doi.org/10.18632/aging.203641)

10.18632/aging.203641

---

## Gut-microbiota-microglia-brain interactions in Alzheimer's disease: knowledge-based, multi-dimensional characterization.
 21.10.2021, _Q Wang, PB Davis, X Qi, SG Chen, ME Gurney, G Perry, PM Doraiswamy, R Xu_


Interactions between the gut microbiota, microglia, and aging may modulate Alzheimer's disease (AD) pathogenesis but the precise nature of such interactions is not known.

[Source](https://doi.org/10.1186/s13195-021-00917-1)

10.1186/s13195-021-00917-1

---

## Alterations in the oral microbiome in HIV infection: causes, effects and potential interventions.
 22.10.2021, _S Li, B Su, QS He, H Wu, T Zhang_


A massive depletion of CD4+ T lymphocytes has been described in early and acute human immunodeficiency virus (HIV) infection, leading to an imbalance between the human microbiome and immune responses. In recent years, a growing interest in the alterations in gut microbiota in HIV infection has led to many studies; however, only few studies have been conducted to explore the importance of oral microbiome in HIV-infected individuals. Evidence has indicated the dysbiosis of oral microbiota in people living with HIV (PLWH). Potential mechanisms might be related to the immunodeficiency in the oral cavity of HIV-infected individuals, including changes in secretory components such as reduced levels of enzymes and proteins in saliva and altered cellular components involved in the reduction and dysfunction of innate and adaptive immune cells. As a result, disrupted oral immunity in HIV-infected individuals leads to an imbalance between the oral microbiome and local immune responses, which may contribute to the development of HIV-related diseases and HIV-associated non-acquired immunodeficiency syndrome comorbidities. Although the introduction of antiretroviral therapy (ART) has led to a significant decrease in occurrence of the opportunistic oral infections in HIV-infected individuals, the dysbiosis in oral microbiome persists. Furthermore, several studies with the aim to investigate the ability of probiotics to regulate the dysbiosis of oral microbiota in HIV-infected individuals are ongoing. However, the effects of ART and probiotics on oral microbiome in HIV-infected individuals remain unclear. In this article, we review the composition of the oral microbiome in healthy and HIV-infected individuals and the possible effect of oral microbiome on HIV-associated oral diseases. We also discuss how ART and probiotics influence the oral microbiome in HIV infection. We believe that a deeper understanding of composition and function of the oral microbiome is critical for the development of effective preventive and therapeutic strategies for HIV infection.

[Source](https://doi.org/10.1097/CM9.0000000000001825)

10.1097/CM9.0000000000001825

---

## Utilization efficiency of human milk oligosaccharides by human-associated <em>Akkermansia</em> is strain-dependent

[Source](https://doi.org/10.1128/AEM.01487-21)

---

## Upper Respiratory Tract Microbiome of Australian Aboriginal and Torres Strait Islander Children in Ear and Nose Health and Disease.
 20.10.2021, _A Coleman, J Zaugg, A Wood, K Cottrell, EG Håkansson, J Adams, M Brown, A Cervin, S Bialasiewicz_


The objective of this study was to examine the nasal microbiota in relation to otitis media (OM) status and nose health in Indigenous Australian children. Children 2 to 7 years of age were recruited from two northern Australian (Queensland) communities. Clinical histories were obtained through parent interviews and reviews of the medical records. Nasal cavity swab samples were obtained, and the children's ears, nose, and throat were examined. DNA was extracted and analyzed by 16S rRNA amplicon next-generation sequencing of the V3/V4 region, in combination with previously generated culture data. A total of 103 children were recruited (mean age, 4.7 years); 17 (16.8%) were healthy, i.e., normal examination results and no history of OM. The nasal microbiota differed significantly in relation to OM status and nose health. Children with historical OM had greater relative abundance of Moraxella, compared to healthy children, despite both having healthy ears at the time of swabbing. Children with healthy noses had greater relative abundance of Staphylococcus aureus, compared to those with rhinorrhea. Dolosigranulum was correlated with Corynebacterium in healthy children. Haemophilus and Streptococcus were correlated across phenotypes. Ornithobacterium was absent or was present with low relative abundance in healthy children and clustered around otopathogens. It correlated with Helcococcus and Dichelobacter. Dolosigranulum and Corynebacterium form a synergism that promotes upper respiratory tract (URT)/ear health in Indigenous Australian children. Ornithobacterium likely represents "Candidatus Ornithobacterium hominis" and in this population is correlated with a novel bacterium that appears to be related to poor URT/ear health. IMPORTANCE Recurring and chronic infections of the ear (OM) are disproportionately prevalent in disadvantaged communities across the globe and, in particular, within Indigenous communities. Despite numerous intervention strategies, OM persists as a major health issue and is the leading cause of preventable hearing loss. In disadvantaged communities, this hearing loss is associated with negative educational and social development outcomes, and consequently, poorer employment prospects and increased contact with the justice system in adulthood. Thus, a better understanding of the microbial ecology is needed in order to identify new targets to treat, as well as to prevent the infections. This study used a powerful combination of 16S rRNA gene sequencing and extended culturomics to show that Dolosigranulum pigrum, a bacterium previously identified as a candidate protective species, may require cocolonization with Corynebacterium pseudodiphtheriticum in order to prevent OM. Additionally, emerging and potentially novel pathogens and bacteria were identified.

[Source](https://journals.asm.org/doi/10.1128/Spectrum.00367-21)

10.1128/Spectrum.00367-21

---

## The Human Microbiome in immunobullous disorders and lichen planus.
 20.10.2021, _M Karimova, D Moyes, M Ide, J Setterfield_


For several decades, there has been a significant growth in the incidence of autoimmune diseases. Studies indicate that genetic factors may not be the only trigger for disease development and that dysbiosis of the microbiome may be another mechanism involved in the pathogenesis of autoimmune diseases. The role of the microbiome in the development of common skin disorders such as psoriasis, atopic dermatitis, acne, and rosacea is increasingly well understood. However, few studies have focused on lichen planus and the rare acquired immunobullous diseases (AIBD), both mucocutaneous groups of disorders linked to skin, oral and gut microbiomes. This review provides an insight into the current understanding of how the microbiome may contribute to the development of autoimmunity as well as themaintenance and exacerbation of acquired immunobullous and lichenoid diseases. These mechanisms may have implications for future preventive and therapeutic approaches.

[Source](https://doi.org/10.1111/ced.14987)

10.1111/ced.14987

---

## Emerging cellular and molecular interactions between the lung microbiota and lung diseases.
 25.10.2021, _L Xiang, X Meng_


With the discovery of the lung microbiota, its study in both pulmonary health and disease has become a vibrant area of emerging research interest. Thus far, most studies have described the lung microbiota composition in lung disease quite well, and some of these studies indicated alterations in lung microbial communities related to the onset and development of lung disease and vice versa. However, the underlying mechanisms, particularly the cellular and molecular links, are still largely unknown. In this review, we highlight the current progress in the complex cellular and molecular mechanisms by which the lung microbiome interacts with immune homeostasis and pulmonary disease pathogenesis to advance our understanding of the elaborate function of the lung microbiota in lung disease. We hope that this work can attract more attention to this still-young yet very promising field to facilitate the identification of new therapeutic targets and provide more innovative therapies. Additional accurate standard-based methodologies and technological breakthroughs are critical to propel the field forward to ultimately achieve the goal of maintaining respiratory health.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/jgh.15716)

10.1080/1040841X.2021.1992345

---

## What is unknown in using Microbiota as a Therapeutic?

[Source](https://onlinelibrary.wiley.com/doi/10.1111/jgh.15716)

---

## Isolation and Identification of Potential Probiotic Bacteria from Human Milk.
 21.10.2021, _QS Damaceno, B Gallotti, IMM Reis, YCP Totte, GB Assis, HC Figueiredo, TF Silva, V Azevedo, JR Nicoli, FS Martins_


Breast milk was long considered a sterile environment, but now it is known to harbor many bacteria that will shape the newborn microbiota. The benefits of breastfeeding to newborn health are, on some level, related to the presence of beneficial bacteria in human milk. Therefore, this study aims to investigate and isolate potential probiotics present in human milk that might be associated with improved health in infants, being potential candidates to be used in simulated human milk formula. Milk samples of 24 healthy mothers were collected at three time points: 30 min (colostrum), 5-9 days (transitional milk), and 25-30 days (mature milk) postpartum. Samples were evaluated by culturing, and the isolated bacteria were identified by MALDI-TOF MS and 16S DNA sequencing. In vitro screening for probiotics properties was performed, and the potential probiotics were mono-associated with germ-free mice to evaluate their ability to colonize the gastrointestinal tract. The microorganisms were submitted to the spray-drying process to check their viability for a potential simulated milk formula production. Seventy-seven bacteria were isolated from breast milk pertaining to four bacterial genera (Staphylococcus, Streptococcus, Leuconostoc, and Lacticaseibacillus). Four potential probiotics were selected: Lacticaseibacillus rhamnosus (n = 2) and Leuconostoc mesenteroides (n = 2). Isolates were able to colonize the gastrointestinal tract of germ-free mice and remained viable after the spray-drying process. In conclusion, breast milk harbors a unique microbiota with beneficial microorganisms that will impact the newborn gut colonization, being an essential source of probiotic candidates to be used in a formula of simulated maternal milk.

[Source](https://doi.org/10.1007/s12602-021-09866-5)

10.1007/s12602-021-09866-5

---

## Probiotics as the live microscopic fighters against <em>Helicobacter pylori</em> gastric infections

[Source](https://doi.org/10.1186/s12876-021-01977-1)

---

## The probiotic <em>Companilactobacillus crustorum</em> MN047 alleviates colitis-associated tumorigenesis via modulating the intestinal microenvironment

[Source](https://doi.org/10.1039/d1fo01531a)

---

## <em>Lactobacillus rhamnosus</em> and <em>Bifidobacterium longum</em> alleviate colitis and cognitive impairment in mice by regulating IFN-γ to IL-10 and TNF-α to IL-10 expression ratios

[Source](https://doi.org/10.1038/s41598-021-00096-x)

---

## Interaction between four galactans with different structural characteristics and gut microbiota

[Source](https://doi.org/10.1080/10408398.2021.1992605)

---

## Habitat Disturbance Linked with Host Microbiome Dispersion and <em>Bd</em> Dynamics in Temperate Amphibians

[Source](https://doi.org/10.1007/s00248-021-01897-3)

---

## The Microbial Composition of Penaeid Shrimps’ Hepatopancreas Is Modulated by Hemocyanin

[Source](https://doi.org/10.4049/jimmunol.2100746)

---

## Overabundance of <em>Asaia</em> and <em>Serratia </em>Bacteria Is Associated with Deltamethrin Insecticide Susceptibility in <em>Anopheles coluzzii </em>from Agboville, Côte d’Ivoire

[Source](https://journals.asm.org/doi/10.1128/Spectrum.00157-21)

---

## Genomics and phylogeny of the proposed phylum <em>‘Candidatus</em> Poribacteria’ associated with the excavating sponge <em>Thoosa mismalolli</em>

[Source](https://doi.org/10.1007/s10482-021-01670-z)

---

## Microbial Diversity in Cultivated and Feral Vanilla <em>Vanilla planifolia</em> Orchids Affected by Stem and Rot Disease

[Source](https://doi.org/10.1007/s00248-021-01876-8)

---

## Indirect reduction of <em>Ralstonia solanacearum</em> via pathogen helper inhibition

[Source](https://doi.org/10.1038/s41396-021-01126-2)

---

## Comparative Analysis of the Cultured and Total Bacterial Community in the Wheat Rhizosphere Microbiome Using Culture-Dependent and Culture-Independent Approaches.
 20.10.2021, _SH Youseif, FH Abd El-Megeed, EA Humm, M Maymon, AH Mohamed, SA Saleh, AM Hirsch_


Rhizosphere and root-associated bacteria are key components of crop production and sustainable agriculture. However, utilization of these beneficial bacteria is often limited by conventional culture techniques because a majority of soil microorganisms cannot be cultured using standard laboratory media. Therefore, the purpose of this study was to improve culturability and investigate the diversity of the bacterial communities from the wheat rhizosphere microbiome collected from three locations in Egypt with contrasting soil characteristics by using metagenomic analysis and improved culture-based methods. The improved strategies of the culture-dependent approach included replacing the agar in the medium with gellan gums and modifying its preparation by autoclaving the phosphate and gelling agents separately. Compared to the total operational taxonomic units (OTUs) observed from the metagenomic data sets derived from the three analyzed soils, 1.86 to 2.52% of the bacteria were recovered using the modified cultivation strategies, whereas less than 1% were obtained employing the standard cultivation protocols. Twenty-one percent of the cultivable isolates exhibited multiple plant growth-promoting (PGP) properties, including P solubilization activity and siderophore production. From the metagenomic analysis, the most abundant phyla were Proteobacteria, Actinobacteria, Chloroflexi, Bacteroidetes, and Firmicutes. Moreover, the relative abundance of the specific bacterial taxa was correlated with the soil characteristics, demonstrating the effect of the soil in modulating the plant rhizosphere microbiome. IMPORTANCE Bacteria colonizing the rhizosphere, a narrow zone of soil surrounding the root system, are known to have beneficial effects in improving the growth and stress tolerance of plants. However, most bacteria in natural environments, especially those in rhizosphere soils, are recalcitrant to cultivation using traditional techniques, and thus their roles in soil health and plant growth remain unexplored. Hence, investigating new culture media and culture conditions to bring "not-yet-cultured" species into cultivation and to identify new functions is still an important task for all microbiologists. To this end, we describe improved cultivation protocols that increase the number and diversity of cultured bacteria from the rhizosphere of wheat plants. Using such approaches will lead to new insights into culturing more beneficial bacteria that live in the plant rhizosphere, in so doing creating greater opportunities not only for field application but also for promoting sustainability.

[Source](https://journals.asm.org/doi/10.1128/Spectrum.00678-21)

10.1128/Spectrum.00678-21

---

## Mechanistic Insight into Yeast Bloom in a Lactic Acid Bacteria Relaying-Community in the Start of Sourdough Microbiota Evolution

[Source](https://journals.asm.org/doi/10.1128/Spectrum.00662-21)

---

