## The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2- Coronaviridae Study

[Source](https://www.nature.com/articles/s41564-020-0695-z)

---

## Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses.
 01.04.2020, _M Letko, A Marzi, V Munster_


Over the past 20 years, several coronaviruses have crossed the species barrier into humans, causing outbreaks of severe, and often fatal, respiratory illness. Since SARS-CoV was first identified in animal markets, global viromics projects have discovered thousands of coronavirus sequences in diverse animals and geographic regions. Unfortunately, there are few tools available to functionally test these viruses for their ability to infect humans, which has severely hampered efforts to predict the next zoonotic viral outbreak. Here, we developed an approach to rapidly screen lineage B betacoronaviruses, such as SARS-CoV and the recent SARS-CoV-2, for receptor usage and their ability to infect cell types from different species. We show that host protease processing during viral entry is a significant barrier for several lineage B viruses and that bypassing this barrier allows several lineage B viruses to enter human cells through an unknown receptor. We also demonstrate how different lineage B viruses can recombine to gain entry into human cells, and confirm that human ACE2 is the receptor for the recently emerging SARS-CoV-2.

[Source](https://www.nature.com/articles/s41564-020-0688-y)

10.1038/s41564-020-0688-y

---

## Variation in prostaglandin metabolism during growth of the diatom Thalassiosira rotula.
 31.03.2020, _V Di Dato, R Barbarinaldi, A Amato, F Di Costanzo, C Fontanarosa, A Perna, A Amoresano, F Esposito, A Cutignano, A Ianora, G Romano_


Prostaglandins (PGs) are hormone-like mediators in many physiological and pathological processes that are present in all vertebrates, in some terrestrial and aquatic invertebrates, and have also been identified in some macroalgae. They have recently been reported also in marine microalgae but their role as chemical mediators is largely unknown. Here we studied the expression pattern of the PG biosynthetic pathway during different growth phases of the centric diatom Thalassiosira rotula and assessed the release of PGs in the surrounding environment for the first time. We show that enzymes responsible for PGs formation such as cyclooxygenase, prostaglandin E synthase 2-like and prostaglandin F synthase are mainly expressed at the end of the exponential phase and that PGs are released especially during the stationary and senescent phases, suggesting a possible signaling function for these compounds. Phylogenetic analysis of the limiting enzyme, COX, indicate the presence in diatoms of more than one enzyme related to the oxidative metabolism of fatty acids belonging to the peroxidase-cyclooxygenase superfamily. These findings suggest a more complex evolution and diversity of metabolic pathways leading to the synthesis of lipid mediators in diatoms.

[Source](https://www.nature.com/articles/s41598-020-61967-3)

10.1038/s41598-020-61967-3

---

## Early Life Respiratory Syncytial Virus Infection and Asthmatic Responses.
 23.01.2020, _C Ptaschinski, NW Lukacs_


_Adaptive Immunity, Age Factors, Animals, Asthma, Disease Susceptibility, Energy Metabolism, Epigenesis, Genetic, Gastrointestinal Microbiome, Humans, Immunity, Innate, Respiratory Mucosa, Respiratory Syncytial Virus Infections, Respiratory Syncytial Viruses_
The infant's developing immune response is central to establishing a balanced system that reacts appropriately to infectious stimuli, but does not induce altered disease states with potential long-term sequelae. Respiratory syncytial virus may alter the immune system, affecting future responses. Early infection may have direct effects on the lung itself. Other early life processes contribute to the development of immune responses including assembly of the microbiome, which seems to have a particularly important role for establishing the immune environment. This review covers studies that have set up important paradigms and discusses recent data that direct research toward informative hypotheses.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0899900720300952)

10.1016/j.iac.2019.03.002

---

## Factors affecting early-life intestinal microbiota development 

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0899900720300952)

---

## Gut, oral and skin microbiome of Indian patrilineal families reveal perceptible association with age

[Source](https://www.nature.com/articles/s41598-020-62195-5)

---

## Implication of the gut microbiome composition of type 2 diabetic patients from northern China.
 31.03.2020, _Q Li, Y Chang, K Zhang, H Chen, S Tao, Z Zhang_


Emerging evidence has suggested the association of the gut microbiome with some human diseases, including type 2 diabetes (T2D). In this study, we analyzed the gut microbiota from a cohort of healthy and diabetic Chinese individuals from Northern China. Pyrosequencing of the V4V5 region of 16S rRNA genes revealed a significant decrease in the gut microbiota diversity of diabetic patients as compared to healthy individuals. Butyrate-producing bacteria such as Bifidobacterium and Akkermansia were significantly decreased in diabetic patients. Furthermore, the abundance of Dorea was significantly increased in T2D individuals and negatively correlated with the abundance of butyrate-producing bacteria. The increase of Dorea could play a role in the development of T2D and has been previously overlooked. Importantly, functional analysis of the gut microbiome revealed for the first time that increased levels of butyrate production via transferases and the degradation of several amino acids due to gut microbial metabolism have strong correlations with T2D in Northern China. Moreover, the potential of gut microbiota-based classifiers to identify individuals with a high risk for T2D has been demonstrated in this study. Taken together, our findings have revealed a previously unappreciated association of the gut microbiome with T2D and have also suggested that changes in gut microbiota may be used to identify individuals at high risk for T2D.

[Source](https://www.nature.com/articles/s41598-020-62224-3)

10.1038/s41598-020-62224-3

---

## The microbiota programs DNA methylation to control intestinal homeostasis and inflammation.
 30.03.2020, _I Ansari, G Raddatz, J Gutekunst, M Ridnik, D Cohen, M Abu-Remaileh, T Tuganbaev, H Shapiro, E Pikarsky, E Elinav, F Lyko, Y Bergman_


Although much research has been done on the diversity of the gut microbiome, little is known about how it influences intestinal homeostasis under normal and pathogenic conditions. Epigenetic mechanisms have recently been suggested to operate at the interface between the microbiota and the intestinal epithelium. We performed whole-genome bisulfite sequencing on conventionally raised and germ-free mice, and discovered that exposure to commensal microbiota induced localized DNA methylation changes at regulatory elements, which are TET2/3-dependent. This culminated in the activation of a set of 'early sentinel' response genes to maintain intestinal homeostasis. Furthermore, we demonstrated that exposure to the microbiota in dextran sodium sulfate-induced acute inflammation results in profound DNA methylation and chromatin accessibility changes at regulatory elements, leading to alterations in gene expression programs enriched in colitis- and colon-cancer-associated functions. Finally, by employing genetic interventions, we show that microbiota-induced epigenetic programming is necessary for proper intestinal homeostasis in vivo.

[Source](https://www.nature.com/articles/s41564-019-0659-3)

10.1038/s41564-019-0659-3

---

## The Contest of Microbial Pigeon Neighbors: Interspecies Competition Between Serratia marcescens and the Human Pathogen Cryptococcus neoformans 

[Source](https://www.sciencedirect.com/science/article/pii/S1878614620300477)

---

## Pseudomonas PS01 Isolated from Maize Rhizosphere Alters Root System Architecture and Promotes Plant Growth.
 31.03.2020, _TN Chu, LV Bui, MTT Hoang_


The objectives of this study were to evaluate the plant growth promoting effects on Arabidopsis by Pseudomonas sp. strains associated with rhizosphere of crop plants grown in Mekong Delta, Vietnam. Out of all the screened isolates, Pseudomonas PS01 isolated from maize rhizosphere showed the most prominent plant growth promoting effects on Arabidopsis and maize (Zeamays). We also found that PS01 altered root system architecture (RSA). The full genome of PS01 was resolved using high-throughput sequencing. Phylogenetic analysis identified PS01 as a member of the Pseudomonas putida subclade, which is closely related to Pseudomonastaiwanensis.. PS01 genome size is 5.3 Mb, assembled in 71 scaffolds comprising of 4820 putative coding sequence. PS01 encodes genes for the indole-3-acetic acid (IAA), acetoin and 2,3-butanediol biosynthesis pathways. PS01 promoted the growth of Arabidopsis and altered the root system architecture by inhibiting primary root elongation and promoting lateral root and root hair formation. By employing gene expression analysis, genetic screening and pharmacological approaches, we suggested that the plant-growth promoting effects of PS01 and the alteration of RSA might be independent of bacterial auxin and could be caused by a combination of different diffusible compounds and volatile organic compounds (VOCs). Taken together, our results suggest that PS01 is a potential candidate to be used as bio-fertilizer agent for enhancing plant growth.

[Source](https://www.mdpi.com/2076-2607/8/4/471)

10.3390/microorganisms8040471

---

## Viruses with different genome types adopt a similar strategy to pack nucleic acids based on positively charged protein domains.
 31.03.2020, _RD Requião, RL Carneiro, MH Moreira, M Ribeiro-Alves, S Rossetto, FL Palhano, T Domitrovic_


Capsid proteins often present a positively charged arginine-rich sequence at their terminal regions, which has a fundamental role in genome packaging and particle stability for some icosahedral viruses. These sequences show little to no conservation and are structurally dynamic such that they cannot be easily detected by common sequence or structure comparisons. As a result, the occurrence and distribution of positively charged domains across the viral universe are unknown. Based on the net charge calculation of discrete protein segments, we identified proteins containing amino acid stretches with a notably high net charge (Q &gt; + 17), which are enriched in icosahedral viruses with a distinctive bias towards arginine over lysine. We used viral particle structural data to calculate the total electrostatic charge derived from the most positively charged protein segment of capsid proteins and correlated these values with genome charges arising from the phosphates of each nucleotide. We obtained a positive correlation (r = 0.91, p-value &lt;0001) for a group of 17 viral families, corresponding to 40% of all families with icosahedral structures described to date. These data indicated that unrelated viruses with diverse genome types adopt a common underlying mechanism for capsid assembly based on R-arms.

[Source](https://www.nature.com/articles/s41598-020-62328-w)

10.1038/s41598-020-62328-w

---

## 5 challenges in understanding the role of the virome in health and disease.
 27.03.2020, _D Wang_



[Source](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1008318)

10.1371/journal.ppat.1008318

---

## Whole-genome sequencing of Burkholderia pseudomallei from an urban melioidosis hot spot reveals a fine-scale population structure and localised spatial clustering in the environment.
 31.03.2020, _A Rachlin, M Mayo, JR Webb, M Kleinecke, V Rigas, G Harrington, BJ Currie, M Kaestli_


Melioidosis is a severe disease caused by the environmental bacterium Burkholderia pseudomallei that affects both humans and animals throughout northern Australia, Southeast Asia and increasingly globally. While there is a considerable degree of genetic diversity amongst isolates, B. pseudomallei has a robust global biogeographic structure and genetic populations are spatially clustered in the environment. We examined the distribution and local spread of B. pseudomallei in Darwin, Northern Territory, Australia, which has the highest recorded urban incidence of melioidosis globally. We sampled soil and land runoff throughout the city centre and performed whole-genome sequencing (WGS) on B. pseudomallei isolates. By combining phylogenetic analyses, Bayesian clustering and spatial hot spot analysis our results demonstrate that some sequence types (STs) are widespread in the urban Darwin environment, while others are highly spatially clustered over a small geographic scale. This clustering matches the spatial distribution of clinical cases for one ST. Results also demonstrate a greater overall isolate diversity recovered from drains compared to park soils, further supporting the role drains may play in dispersal of B. pseudomallei STs in the environment. Collectively, knowledge gained from this study will allow for better understanding of B. pseudomallei phylogeography and melioidosis source attribution, particularly on a local level.

[Source](https://www.nature.com/articles/s41598-020-62300-8)

10.1038/s41598-020-62300-8

---

## Comprehensive evaluation of complex polymicrobial specimens using next generation sequencing and standard microbiological culture.
 31.03.2020, _LA Cummings, DR Hoogestraat, SL Rassoulian-Barrett, CA Rosenthal, SJ Salipante, BT Cookson, NG Hoffman_


Optimal clinical decision-making depends on identification of clinically relevant organisms present in a sample. Standard microbiological culture may fail to identify unusual or fastidious organisms and can misrepresent relative abundance of sample constituents. Culture-independent methods have improved our ability to deconvolute polymicrobial patient samples. We used next-generation 16S rRNA gene sequencing (NGS16S) to determine how often cultivatable organisms in complex polymicrobial samples are not reported by standard culture. Twenty consecutive bronchoalveolar lavage (BAL) samples were plated to standard and additional media; bacteria were identified by NGS16S analysis of DNA extracted directly from samples or from washed culture plates. 96% of organisms identified were cultivable, but only 21% were reported by standard culture, indicating that standard work-up provides an incomplete assessment of microbial constituents. Direct NGS16S correlated well with standard culture, identifying the same predominant organism in 50% of samples. When predominant organisms differed, NGS16S most often detected anaerobes, whose growth is unsupported by standard culture conditions for this specimen. NGS16S identified more organisms per sample and allowed identification of fastidious organisms, while culture was better at capturing organisms when bacterial load was low, and allowed incidental recovery of non-bacterial pathogens. Molecular and culture-based methods together detect more organisms than either method alone.

[Source](https://www.nature.com/articles/s41598-020-62424-x)

10.1038/s41598-020-62424-x

---

## Potential of proteomics to probe microbes.
 26.03.2020, _C Shiny Matilda, I Madhusudan, R Gaurav Isola, C Shanthi_


An organism exposed to a plethora of environmental perturbations undergoes proteomic changes which enable the characterization of total proteins in it. Much of the proteomic information is obtained from genomic data. Additional information on the proteome such as posttranslational modifications, protein-protein interactions, protein localization, metabolic pathways, and so on are deduced using proteomic tools which genomics and transcriptomics fail to offer. The proteomic analysis allows identification of precise changes in proteins, which in turn solve the complexity of microbial population providing insights into the microbial metabolism, cellular pathways, and behavior of microorganisms in new environments. Furthermore, they provide clues for the exploitation of their special features for biotechnological applications. Numerous techniques for the analysis of microbial proteome such as electrophoretic, chromatographic, mass spectrometric-based methods as well as quantitative proteomics are available which facilitate protein separation, expression, identification, and quantification of proteins. An understanding of the potential of each of the proteomic tools has created a significant impact on diverse microbiological aspects and the same has been discussed in this review.

[Source](https://onlinelibrary.wiley.com/doi/abs/10.1002/jobm.201900628)

10.1002/jobm.201900628

---

