## <strong>Fecal microbiota transplantation plus anti-PD-1 immunotherapy in advanced melanoma: a phase I trial</strong>

[Source](https://doi.org/10.1038/s41591-023-02453-x)

---

## <strong>Top-down identification of keystone taxa in the microbiome</strong>

[Source](https://doi.org/10.1038/s41467-023-39459-5)

---

## <strong>Sources of gut microbiota variation in a large longitudinal Finnish infant cohort</strong>

[Source](https://www.thelancet.com/journals/ebiom/article/PIIS2352-3964(23)00260-8/fulltext)

---

## 11<sup>th</sup>&nbsp;Microbiome R&amp;D and Business Collaboration Forum and 5<sup>th</sup>&nbsp;Probiotics &amp; Prebiotics Congress: USA

[Source](https://www.global-engage.com/event/microbiome/)

---

## A microbiome exercise

[Source](https://doi.org/10.1126/science.adi6329)

---

## Mendelian randomisation analysis reveals the possible causal relationship between infections, microbiota and clinical disease

[Source](https://doi.org/10.1136/gutjnl-2023-329787)

---

## Microbiome variance of the small bowel in Crohn’s disease

[Source](https://doi.org/10.1136/gutjnl-2022-327313)

---

## Metagenomics analysis of the neonatal intestinal resistome

[Source](https://doi.org/10.3389/fped.2023.1169651)

---

## Development of the intestinal microbiome in cystic fibrosis in early life

[Source](https://doi.org/10.1128/msphere.00046-23)

---

## Overexpression of human alpha-Synuclein leads to dysregulated microbiome/metabolites with ageing in a rat model of Parkinson disease

[Source](https://doi.org/10.1186/s13024-023-00628-1)

---

## Male-Biased Gut Microbiome and Metabolites Aggravate Colorectal Cancer Development

[Source](https://doi.org/10.1002/advs.202206238)

---

## Systematic review: Autism spectrum disorder and the gut microbiota

[Source](https://doi.org/10.1111/acps.13587)

---

## Early-life respiratory infections and developmental immunity determine lifelong lung health

[Source](https://doi.org/10.1038/s41590-023-01550-w)

---

## Ecoevolutionary processes structure milk microbiomes across the mammalian tree of life

[Source](https://doi.org/10.1073/pnas.2218900120)

---

## Shared evolutionary path in social microbiomes

[Source](https://doi.org/10.1093/molbev/msad153)

---

## Use of probiotics, prebiotics, and synbiotics in non-alcoholic fatty liver disease: A systematic review and meta-analysis

[Source](https://doi.org/10.1111/jgh.16256)

---

## <em>Akkermansia muciniphila</em> and <em>Bifidobacterium bifidum</em> Prevent NAFLD by Regulating FXR Expression and Gut Microbiota

[Source](https://doi.org/10.14218/JCTH.2022.00415)

---

## <em>Limosilactobacillus reuteri</em> inhibits the acid tolerance response in oral bacteria

[Source](https://doi.org/10.1016/j.bioflm.2023.100136)

---

## Using the collaborative cross to identify the role of host genetics in defining the murine gut microbiome

[Source](https://doi.org/10.1186/s40168-023-01552-8)

---

## Selection for environmental variance shifted the gut microbiome composition driving animal resilience

[Source](https://doi.org/10.1186/s40168-023-01580-4)

---

## Metabolic interaction models recapitulate leaf microbiota ecology.
 10.07.2023, _M Schäfer, AR Pacheco, R Künzler, M Bortfeld-Miller, CM Field, E Vayena, V Hatzimanikatis, JA Vorholt_


_Microbiota, Bacteria, Ecology, Arabidopsis, Plant Leaves_
Resource allocation affects the structure of microbiomes, including those associated with living hosts. Understanding the degree to which this dependency determines interspecies interactions may advance efforts to control host-microbiome relationships. We combined synthetic community experiments with computational models to predict interaction outcomes between plant-associated bacteria. We mapped the metabolic capabilities of 224 leaf isolates from Arabidopsis thaliana by assessing the growth of each strain on 45 environmentally relevant carbon sources in vitro. We used these data to build curated genome-scale metabolic models for all strains, which we combined to simulate &gt;17,500 interactions. The models recapitulated outcomes observed in planta with &gt;89% accuracy, highlighting the role of carbon utilization and the contributions of niche partitioning and cross-feeding in the assembly of leaf microbiomes.

[Source](https://doi.org/10.1126/science.adf5121)

10.1126/science.adf5121

---

## Microeukaryotic predators shape the wastewater microbiome.
 08.07.2023, _N Heck, J Freudenthal, K Dumack_


The physicochemical parameters that shape the prokaryotic community composition in wastewater have been extensively studied. In contrast, it is poorly understood whether and how biotic interactions affect the prokaryotic community composition in wastewater. We used metatranscriptomics data from a bioreactor sampled weekly over 14 months to investigate the wastewater microbiome, including often neglected microeukaryotes. Our analysis revealed that while prokaryotes are unaffected by seasonal changes in water temperature, they are impacted by a seasonal, temperature-induced change in the microeukaryotic community. Our findings suggest that selective predation pressure exerted by microeukaryotes is a significant factor shaping the prokaryotic community in wastewater. This study underscores the importance of investigating the entire wastewater microbiome to develop a comprehensive understanding of wastewater treatment.

[Source](https://doi.org/10.1016/j.watres.2023.120293)

10.1016/j.watres.2023.120293

---

## Functional and sequence-based metagenomics to uncover carbohydrate-degrading enzymes from composting samples.
 07.07.2023, _C Santos-Pereira, J Sousa, ÂMA Costa, AO Santos, T Rito, P Soares, R Franco-Duarte, SC Silvério, LR Rodrigues_


The renewable, abundant , and low-cost nature of lignocellulosic biomass can play an important role in the sustainable production of bioenergy and several added-value bioproducts, thus providing alternative solutions to counteract the global energetic and industrial demands. The efficient conversion of lignocellulosic biomass greatly relies on the catalytic activity of carbohydrate-active enzymes (CAZymes). Finding novel and robust biocatalysts, capable of being active under harsh industrial conditions, is thus imperative to achieve an economically feasible process. In this study, thermophilic compost samples from three Portuguese companies were collected, and their metagenomic DNA was extracted and sequenced through shotgun sequencing. A novel multi-step bioinformatic pipeline was developed to find CAZymes and characterize the taxonomic and functional profiles of the microbial communities, using both reads and metagenome-assembled genomes (MAGs) as input. The samples' microbiome was dominated by bacteria, where the classes Gammaproteobacteria, Alphaproteobacteria, and Balneolia stood out for their higher abundance, indicating that the degradation of compost biomass is mainly driven by bacterial enzymatic activity. Furthermore, the functional studies revealed that our samples are a rich reservoir of glycoside hydrolases (GH), particularly of GH5 and GH9 cellulases, and GH3 oligosaccharide-degrading enzymes. We further constructed metagenomic fosmid libraries with the compost DNA and demonstrated that a great number of clones exhibited β-glucosidase activity. The comparison of our samples with others from the literature showed that, independently of the composition and process conditions, composting is an excellent source of lignocellulose-degrading enzymes. To the best of our knowledge, this is the first comparative study on the CAZyme abundance and taxonomic/functional profiles of Portuguese compost samples. KEY POINTS: • Sequence- and function-based metagenomics were used to find CAZymes in compost samples. • Thermophilic composts proved to be rich in bacterial GH3, GH5, and GH9 enzymes. • Compost-derived fosmid libraries are enriched in clones with β-glucosidase activity.

[Source](https://doi.org/10.1007/s00253-023-12627-9)

10.1007/s00253-023-12627-9

---

## Analysis of the microbiome of the Bolivian high-altitude Lake Pastos Grandes.
 04.07.2023, _J Daga-Quisbert, F Ugarte, GK Rajarao, AJA van Maris, J Quillaguamán_


Lake Pastos Grandes in Bolivia is mainly composed of salt flats, which are sporadically and only partially submerged during the wet season. In the present study, the chemical composition of water samples of the lake and some influent rivers was determined. We found that it is likely that the lake was influenced by the dilution of metals from ancient evaporites. We performed the first metagenomic studies on this lake. Analyses on shotgun metagenomics revealed that the relative abundances of Burkholderiales and Pseudomonadales were noteworthy in the water samples, whereas the archaea belonging to the Halobacteriales and Cyanobacteria from subsection III had high abundances in the salt flat. The eukaryotes Crustacea and Diatomea exhibited the highest abundances in the water samples. We investigated further the potential effect of human activities on the nitrogen cycle mobilization in the lake and the propagation of antimicrobial resistance genes. This is the first report about the cycle in the lake. Additionally, rifamycin resistance genes and genes related to efflux pumps, which are not considered a hazard when identified in metagenomes, had the uppermost relative abundances in all sampling points. We found that Lake Pastos Grandes hitherto does not show an appreciable influence by anthropogenic actions.

[Source](https://doi.org/10.1093/femsec/fiad073)

10.1093/femsec/fiad073

---

## Metagenomic Analysis of a Concrete Bridge Reveals a Microbial Community Dominated by Halophilic Bacteria and Archaea.
 05.07.2023, _EA Kiledal, M Shaw, SW Polson, JA Maresca_


Concrete hosts a small but diverse microbiome that changes over time. Shotgun metagenomic sequencing would enable assessment of both the diversity and function of the microbial community in concrete, but a number of unique challenges make this difficult for concrete samples. The high concentration of divalent cations in concrete interferes with nucleic acid extraction, and the extremely low biomass in concrete means that DNA from laboratory contamination may be a large fraction of the sequence data. Here, we develop an improved method for DNA extraction from concrete, with higher yield and lower laboratory contamination. To show that this method provides DNA of sufficient quality and quantity to do shotgun metagenomic sequencing, DNA was extracted from a sample of concrete obtained from a road bridge and sequenced with an Illumina MiSeq system. This microbial community was dominated by halophilic Bacteria and Archaea, with enriched functional pathways related to osmotic stress responses. Although this was a pilot-scale effort, we demonstrate that metagenomic sequencing can be used to characterize microbial communities in concrete and that older concrete structures may host different microbes than recently poured concrete. IMPORTANCE Prior work on the microbial communities of concrete focused on the surfaces of concrete structures such as sewage pipes or bridge pilings, where thick biofilms were easy to observe and sample. Because the biomass inside concrete is so low, more recent analyses of the microbial communities inside concrete used amplicon sequencing methods to describe those communities. However, to understand the activity and physiology of microbes in concrete, or to develop living infrastructure, we must develop more direct methods of community analysis. The method developed here for DNA extraction and metagenomic sequencing can be used for analysis of microbial communities inside concrete and can likely be adapted for other cementitious materials.

[Source](https://doi.org/10.1128/spectrum.05112-22)

10.1128/spectrum.05112-22

---

## A catalog of bacterial reference genomes from cultivated human oral bacteria.
 06.07.2023, _W Li, H Liang, X Lin, T Hu, Z Wu, W He, M Wang, J Zhang, Z Jie, X Jin, X Xu, J Wang, H Yang, W Zhang, K Kristiansen, L Xiao, Y Zou_


_Humans, Bacteria, Mouth, Saliva, Microbiota, Streptococcus_
The oral cavity harbors highly diverse communities of microorganisms. However, the number of isolated species and high-quality genomes is limited. Here we present a Cultivated Oral Bacteria Genome Reference (COGR), comprising 1089 high-quality genomes based on large-scale aerobic and anaerobic cultivation of human oral bacteria isolated from dental plaques, tongue, and saliva. COGR covers five phyla and contains 195 species-level clusters of which 95 include 315 genomes representing species with no taxonomic annotation. The oral microbiota differs markedly between individuals, with 111 clusters being person-specific. Genes encoding CAZymes are abundant in the genomes of COGR. Members of the Streptococcus genus make up the largest proportion of COGR and many of these harbor entire pathways for quorum sensing important for biofilm formation. Several clusters containing unknown bacteria are enriched in individuals with rheumatoid arthritis, emphasizing the importance of culture-based isolation for characterizing and exploiting oral bacteria.

[Source](https://doi.org/10.1038/s41522-023-00414-3)

10.1038/s41522-023-00414-3

---

## Gene-based microbiome representation enhances host phenotype classification.
 05.07.2023, _T Deschênes, FWE Tohoundjona, PL Plante, V Di Marzo, F Raymond_


With the concomitant advances in both the microbiome and machine learning fields, the gut microbiome has become of great interest for the potential discovery of biomarkers to be used in the classification of the host health status. Shotgun metagenomics data derived from the human microbiome is composed of a high-dimensional set of microbial features. The use of such complex data for the modeling of host-microbiome interactions remains a challenge as retaining de novo content yields a highly granular set of microbial features. In this study, we compared the prediction performances of machine learning approaches according to different types of data representations derived from shotgun metagenomics. These representations include commonly used taxonomic and functional profiles and the more granular gene cluster approach. For the five case-control datasets used in this study (Type 2 diabetes, obesity, liver cirrhosis, colorectal cancer, and inflammatory bowel disease), gene-based approaches, whether used alone or in combination with reference-based data types, allowed improved or similar classification performances as the taxonomic and functional profiles. In addition, we show that using subsets of gene families from specific functional categories of genes highlight the importance of these functions on the host phenotype. This study demonstrates that both reference-free microbiome representations and curated metagenomic annotations can provide relevant representations for machine learning based on metagenomic data. IMPORTANCE Data representation is an essential part of machine learning performance when using metagenomic data. In this work, we show that different microbiome representations provide varied host phenotype classification performance depending on the dataset. In classification tasks, untargeted microbiome gene content can provide similar or improved classification compared to taxonomical profiling. Feature selection based on biological function also improves classification performance for some pathologies. Function-based feature selection combined with interpretable machine learning algorithms can generate new hypotheses that can potentially be assayed mechanistically. This work thus proposes new approaches to represent microbiome data for machine learning that can potentiate the findings associated with metagenomic data.

[Source](https://doi.org/10.1128/msystems.00531-23)

10.1128/msystems.00531-23

---

## Integrative meta-omics in Galaxy and beyond.
 10.07.2023, _VC Schiml, F Delogu, P Kumar, B Kunath, B Batut, S Mehta, JE Johnson, B Grüning, PB Pope, PD Jagtap, TJ Griffin, MØ Arntzen_


'Omics methods have empowered scientists to tackle the complexity of microbial communities on a scale not attainable before. Individually, omics analyses can provide great insight; while combined as "meta-omics", they enhance the understanding of which organisms occupy specific metabolic niches, how they interact, and how they utilize environmental nutrients. Here we present three integrative meta-omics workflows, developed in Galaxy, for enhanced analysis and integration of metagenomics, metatranscriptomics, and metaproteomics, combined with our newly developed web-application, ViMO (Visualizer for Meta-Omics) to analyse metabolisms in complex microbial communities.

[Source](https://doi.org/10.1186/s40793-023-00514-9)

10.1186/s40793-023-00514-9

---

## Compositional analysis of microbiome data using the linear decomposition model (LDM).
 06.07.2023, _YJ Hu, GA Satten_


There are compelling reasons to test compositional hypotheses about microbiome data. We present here LDM-clr, an extension of our linear decomposition model (LDM) approach to allow fitting linear models to centered-log-ratio-transformed taxa count data. As LDM-clr is implemented within the existing LDM program, it enjoys all the features supported by LDM, including a compositional analysis of differential abundance at both the taxon and community levels, while allowing for a wide range of covariates and study designs for either association or mediation analysis.

[Source](https://doi.org/10.1101/2023.05.26.542540)

10.1101/2023.05.26.542540

---

## Processing and Storage Methods Affect Oral and Gut Microbiome Composition.
 04.07.2023, _DK Superdock, W Zhang, AC Poole_


Across microbiome studies, fecal and oral samples are stored and processed in different ways, which could affect the observed microbiome composition. Here, we compared treatment methods, which included both storage conditions and processing methods, applied to samples prior to DNA extraction to determine how each affects microbial community diversity as assessed by 16S rRNA gene sequencing. We collected dental swab, saliva, and fecal samples from 10 individuals, with three technical replicates per treatment method. We assessed four methods of processing fecal samples prior to DNA extraction. We also compared different fractions of frozen saliva and dental samples to fresh samples. We found that lyophilized fecal samples, fresh whole saliva samples, and the supernatant fraction of thawed dental samples retained the highest levels of alpha diversity in samples. The supernatant fraction of thawed saliva samples had the second highest alpha diversity compared to fresh. Then we investigated the differences in microbes between different treatments at the domain and phylum levels as well as identified the amplicon sequence variants (ASVs) that were significantly different between the methods producing the highest alpha diversity and the other treatment methods. Lyophilized fecal samples had a greater prevalence of Archaea as well as a greater ratio of Firmicutes to Bacteroidetes compared to the other treatment methods. Our results provide practical considerations, not only for selection of processing method, but also for comparing results across studies that use these methods. Our findings also indicate differences in treatment method could be a confounding factor influencing the presence, absence, or differential abundance of microbes reported in conflicting studies.

[Source](https://doi.org/10.1101/2023.06.13.544865)

10.1101/2023.06.13.544865

---

## A genome catalog of the early-life human skin microbiome

[Source](https://doi.org/10.1101/2023.05.22.541509)

---

## In vitro models as a tool to study the role of gut microbiota in obesity

[Source](https://doi.org/10.1080/10408398.2023.2232022)

---

