## Nurturing futures through the maternal microbiome.
 07.08.2024, _R Pala, K Kenny_


Recently there has been growing recognition of the productive and protective features of our microbial kin and the crucial role of 'commensal' microbes in supporting and sustaining health. Current microbiological and pharmacological literature is increasingly highlighting the role of maternal gut microbiomes in the long-term health of both mothers and children. Drawing on the information and advice directed towards Australian parents from conception through the first years of a child's life, we consider its messaging about the need to secure for the foetus/future-child an enduring, optimal state of health by managing the maternal microbiome. We argue that this post-Pasteurian trend gives rise to relations of care that are, at once, newly collective and more-than-human-but also disciplinary in ways that position the maternal microbiome as a new site of scrutiny that disproportionately responsibilises and burdens mothers. We notice how microbiome research is used both to reframe motherhood as a form of micro(bial)-management and to maintain motherhood as a medicalised process. The feminist and more-than-human potential that this research can provide is missing in the way these resources are presented to parents.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/1467-9566.13828)

10.1111/1467-9566.13828

---

## The skin microbiota of preterm infants and impact of diaper change frequency.
 03.08.2024, _NE Younge, DJ Parris, D Hatch, A Barnes, DH Brandon_


_Humans, Infant, Premature, Infant, Newborn, Skin, Female, Male, Microbiota, Diapers, Infant, Feces, Hydrogen-Ion Concentration, RNA, Ribosomal, 16S, Water Loss, Insensible_
To evaluate the impact of diaper change frequency, clinical characteristics, and skin health metrics on development of the skin microbiota in preterm infants.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0306333)

10.1371/journal.pone.0306333

---

## Quantitative analysis of cis-regulatory elements in transcription with KAS-ATAC-seq.
 10.08.2024, _R Lyu, Y Gao, T Wu, C Ye, P Wang, C He_


_Animals, Mice, Transcription Factors, Transcription, Genetic, Enhancer Elements, Genetic, Chromatin, Binding Sites, Humans, DNA, Single-Stranded, Chromatin Immunoprecipitation Sequencing, Transposases, Regulatory Elements, Transcriptional, Tretinoin, Gene Expression Regulation, Cell Differentiation, Sequence Analysis, DNA, Regulatory Sequences, Nucleic Acid_
Cis-regulatory elements (CREs) are pivotal in orchestrating gene expression throughout diverse biological systems. Accurate identification and in-depth characterization of functional CREs are crucial for decoding gene regulation networks during cellular processes. In this study, we develop Kethoxal-Assisted Single-stranded DNA Assay for Transposase-Accessible Chromatin with Sequencing (KAS-ATAC-seq) to quantitatively analyze the transcriptional activity of CREs. A main advantage of KAS-ATAC-seq lies in its precise measurement of ssDNA levels within both proximal and distal ATAC-seq peaks, enabling the identification of transcriptional regulatory sequences. This feature is particularly adept at defining Single-Stranded Transcribing Enhancers (SSTEs). SSTEs are highly enriched with nascent RNAs and specific transcription factors (TFs) binding sites that define cellular identity. Moreover, KAS-ATAC-seq provides a detailed characterization and functional implications of various SSTE subtypes. Our analysis of CREs during mouse neural differentiation demonstrates that KAS-ATAC-seq can effectively identify immediate-early activated CREs in response to retinoic acid (RA) treatment. Our findings indicate that KAS-ATAC-seq provides more precise annotation of functional CREs in transcription. Future applications of KAS-ATAC-seq would help elucidate the intricate dynamics of gene regulation in diverse biological processes.

[Source](https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-024-03543-4)

10.1038/s41467-024-50680-8

---

## Vitamin A carotenoids, but not retinoids, mediate the impact of a healthy diet on gut microbial diversity

[Source](https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-024-03543-4)

---

## Gut microbial composition is altered in sarcopenia: A systematic review and meta-analysis of clinical studies.
 08.08.2024, _X Mai, S Yang, Q Chen, K Chen_


_Sarcopenia, Humans, Gastrointestinal Microbiome, Bacteria_
Increasing evidence has shown that gut microbiota (GM) was involved in the pathophysiology of musculoskeletal disorders through multiple pathways such as protein anabolism, chronic inflammation and immunity, and imbalanced metabolism. We performed a systematic review and meta-analysis of human studies to evaluate GM diversity differences between individuals with and without sarcopenia, and explore bacteria with potential to become biomarkers. PubMed, Embase and Cochrane library were systematically searched from inception to February 16, 2024. Studies were included if they (1) sampled adults with sarcopenia, and (2) performed GM analysis and reported α-diversity, β-diversity or relative abundance. The methodological quality of included studies and the certainty of evidence were assessed through the Joanna Briggs Institute critical appraisal checklist for analytical cross-sectional studies and the Grades of Recommendation, Assessment, Development and Evaluation (GRADE) Working Group system, respectively. Weighted standardized mean differences (SMDs) and corresponding 95% confidence intervals (CIs) were estimated for α-diversity indices using a fixed-effects and a random-effects model. Beta diversity and the relative abundance of GM were summarized qualitatively. A total of 19 studies involving 6,565 participants were included in this study. Compared with controls, significantly moderate decrease in microbial richness in participants with sarcopenia were found (Chao1: SMD = -0.44; 95%CI, -0.64 to -0.23, I2 = 57.23%, 13 studies; observed species: SMD = -0.68; 95%CI, -1.00 to -0.37, I2 = 66.07%, 5 studies; ACE index: SMD = -0.30; 95%CI, -0.56 to -0.04, I2 = 8.12%, 4 studies), with very low certainty of evidence. Differences in β-diversity were consistently observed in 84.6% of studies and 97.3% of participants. The detailed analysis of the gut microbial differential abundance identified a loss of Prevotellaceae, Prevotella, and Megamonas in sarcopenia compared with non-sarcopenia. In conclusion, sarcopenia was found to be associated with reduced richness of GM, and supplementing intestinal bacteria described above may contribute to preventing and treating this muscle disease. The research protocol was registered and approved in PROSPERO (CRD42023412849).

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0308360)

10.1371/journal.pone.0308360

---

## Associations of upper respiratory mucosa microbiota with Rheumatoid arthritis, autoantibodies, and disease activity.
 08.08.2024, _YB Joo, J Lee, YJ Park, SY Bang, K Kim, HS Lee_


_Humans, Arthritis, Rheumatoid, Autoantibodies, Middle Aged, Female, Male, Microbiota, RNA, Ribosomal, 16S, Respiratory Mucosa, Adult, Aged, Case-Control Studies_
The lung is recognized as a site for initiating the formation of self-antigen and autoimmune responses in rheumatoid arthritis (RA). We aimed to investigate the association of upper respiratory microbiota with RA, autoantibody production, and disease activity. Forty-six patients with RA and 17 controls were examined. Nasopharyngeal swab samples were sequenced for microbiome profiling using the V3-V4 region of the 16S rRNA gene. The microbial diversity and relative abundance were compared between RA patients and controls. Correlation analyses were conducted to evaluate the relationship between microbial abundance and clinical markers such as autoantibodies and disease activity. Microbial diversity analysis revealed no major differences between RA patients and healthy controls. However, beta diversity analysis indicated a subtle distinction in microbial composition (unweighted UniFrac distance) between the two groups (P = 0.03), hinting at a minor subset of microbiota associated with disease status. Differential abundance analysis uncovered specific taxa at various taxonomic levels, including Saccharibacteria (TM7) [O-1] (PFDR = 2.53 × 10-2), TM7 [F-1] (PFDR = 5.20 × 10-3), Microbacterium (PFDR = 3.37 × 10-4), and Stenotrophomonas (PFDR = 2.57 × 10-3). The relative abundance of ten genera correlated significantly with anti-cyclic citrullinated peptide (anti-CCP) antibody levels (PFDR &lt; 0.05) and 11 genera were significantly associated with disease activity markers, including ESR, CRP, DAS28-ESR, and DAS-CRP (PFDR &lt; 0.05). In particular, Saccharibacteria TM7 [G-3] and Peptostreptococcaceae [XI] [G-1] were correlated with all disease activity biomarkers. Dysbiosis in the upper respiratory mucosa is associated with RA, anti-CCP antibody levels, and disease activity.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0308010)

10.1371/journal.pone.0308010

---

## Effect of β-lactam antibiotics on the gut microbiota of term neonates.
 10.08.2024, _H Gu, E Tao, Y Fan, G Long, X Jia, T Yuan, L Chen, X Shu, W Zheng, M Jiang_


_Female, Humans, Infant, Newborn, Male, Bacteria, beta Lactam Antibiotics, Feces, Gastrointestinal Microbiome, RNA, Ribosomal, 16S_
β-Lactam antibiotics are a class of antibiotics commonly used to treat bacterial infections. However, the effects of β-lactam antibiotics on term neonatal intestinal flora have not been fully elucidated. Hospitalized full-term newborns receiving β-lactam antibiotics formed the antibiotic group (n = 67), while those without antibiotic treatment comprised the non-antibiotic group (n = 47). A healthy group included healthy full-term newborns (n = 16). Stool samples were collected for 16 S rDNA sequencing to analyze gut microbiota variations. Further investigation was carried out within the β-lactam antibiotic group, exploring the effects of antibiotic use on the newborns' gut microbiota in relation to the duration and type of antibiotic administration, delivery method, and feeding practices. The antibiotic group exhibited significant difference of microbial community composition compared to the other groups. Genera like Klebsiella, Enterococcus, Streptococcus, Alistipes, and Aeromonas were enriched, while Escherichia-Shigella, Clostridium sensu stricto 1, Bifidobacterium, and Parabacteroides were reduced. Klebsiella negatively correlated with Escherichia-Shigella, positively with Enterobacter, while Escherichia-Shigella negatively correlated with Enterococcus and Streptococcus. Regardless of neonatal age, β-lactam antibiotics induced an elevated abundance of Klebsiella and Enterococcus. The impact on gut microbiota varied with the duration and type of antibiotic (cefotaxime or ampicillin/sulbactam). Compared to vaginal delivery, cesarean delivery after β-lactam treatment heightened the abundance of Klebsiella, Enterobacteriaceae_Unclassified, Lactobacillales_Unclassified, and Pectobacterium. Feeding patterns minimally influenced β-lactam-induced alterations. In conclusion, β-lactam antibiotic treatment for neonatal pneumonia and sepsis markedly disrupted intestinal microbiota, favoring Klebsiella, Enterococcus, Streptococcus, Alistipes, and Aeromonas. The impact of β-lactam varied by duration, type, and delivery method, emphasizing heightened disruptions post-cesarean delivery.

[Source](https://ann-clinmicrob.biomedcentral.com/articles/10.1186/s12941-024-00730-2)

10.1186/s12941-024-00730-2

---

## Clinical efficacy of a multilamellar cream on skin physiology and microbiome in an epidermal stress model: A controlled double-blinded study 

[Source](https://onlinelibrary.wiley.com/doi/10.1111/ics.12950)

---

## Mouse adaptation of human inflammatory bowel diseases microbiota enhances colonization efficiency and alters microbiome aggressiveness depending on the recipient colonic inflammatory environment.
 09.08.2024, _SM Gray, AD Moss, JW Herzog, S Kashiwagi, B Liu, JB Young, S Sun, AP Bhatt, AA Fodor, R Balfour Sartor_


_Animals, Humans, Mice, Fecal Microbiota Transplantation, Gastrointestinal Microbiome, Inflammatory Bowel Diseases, Disease Models, Animal, Dysbiosis, Interleukin-10, Colitis, Feces, Colon, Mice, Knockout, Mice, Inbred C57BL, Female, Bacteria, Inflammation, Male_
Understanding the cause vs consequence relationship of gut inflammation and microbial dysbiosis in inflammatory bowel diseases (IBD) requires a reproducible mouse model of human-microbiota-driven experimental colitis.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01857-2)

10.1186/s40168-024-01857-2

---

## Gut microbiota composition of the isopod Ligia in South Korea exposed to expanded polystyrene pollution.
 09.08.2024, _YM Lee, KM Choi, SH Mun, JW Yoo, JH Jung_


_Gastrointestinal Microbiome, Republic of Korea, Animals, Isopoda, Water Pollutants, Chemical, Metagenomics_
Plastics pose a considerable challenge to aquatic ecosystems because of their increasing global usage and non-biodegradable properties. Coastal plastic debris can persist in ecosystems; however, its effects on resident organisms remain unclear. A metagenomic analysis of the isopoda Ligia, collected from clean (Nae-do, ND) and plastic-contaminated sites (Maemul-do, MD) in South Korea, was conducted to clarify the effects of microplastic contamination on the gut microbiota. Ligia gut microbiota's total operational taxonomic units were higher in ND than in MD. Alpha diversity did not differ significantly between the two Ligia gut microbial communities collected from ND and MD, although richness (Observed species) was lower in MD than in ND. Proteobacteria (67.47%, ND; 57.30%, MD) and Bacteroidetes (13.63%, ND; 20.76%, MD) were the most abundant phyla found at both sites. Significant different genera in Ligia from EPS-polluted sites were observed. Functional gene analysis revealed that 19 plastic degradation-related genes, including those encoding hydrogenase, esterase, and carboxylesterase, were present in the gut microbes of Ligia from MD, indicating the potential role of the Ligia gut microbiota in plastic degradation. This study provides the first comparative field evidence of the gut microbiota dynamics of plastic detritus consumers in marine ecosystems.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0308246)

10.1371/journal.pone.0308246

---

## Effects of a supplemented diet containing 7 probiotic strains (Honeybeeotic) on honeybee physiology and immune response: analysis of hemolymph cytology, phenoloxidase activity, and gut microbiome.
 09.08.2024, _P Robino, L Galosi, A Bellato, S Vincenzetti, E Gonella, I Ferrocino, E Serri, L Biagini, A Roncarati, P Nebbia, C Menzio, G Rossi_


_Animals, Bees, Dietary Supplements, Gastrointestinal Microbiome, Hemocytes, Hemolymph, Immunity, Innate, Italy, Monophenol Monooxygenase, Probiotics_
In this study, a probiotic mixture (Honeybeeotic) consisting of seven bacterial strains isolated from a unique population of honeybees (Apis mellifera ligustica) was used. That honeybee population was located in the Roti Abbey locality of the Marche Region in Italy, an area isolated from human activities, and genetic contamination from other honeybee populations. The aim was to investigate the effects of this probiotic mixture on the innate immunity and intestinal microbiome of healthy common honeybees in two hives of the same apiary. Hive A received a diet of 50% glucose syrup, while hive B received the same syrup supplemented with the probiotics, both administered daily for 1 month. To determine whether the probiotic altered the immune response, phenoloxidase activity and hemolymph cellular subtype count were investigated. Additionally, metagenomic approaches were used to analyze the effects on gut microbiota composition and function, considering the critical role the gut microbiota plays in modulating host physiology.

[Source](https://biolres.biomedcentral.com/articles/10.1186/s40659-024-00533)

10.1186/s40659-024-00533-x

---

## Survey of antimicrobial and probiotic use practices in wildlife rehabilitation in the United States.
 03.08.2024, _EA Miller, R Amato, JB Ponder, I Bueno_


_Animals, Probiotics, United States, Animals, Wild, Surveys and Questionnaires, Humans, Anti-Infective Agents, Anti-Bacterial Agents_
Antimicrobial resistance is a global health concern. As such, there have been increased efforts to monitor and standardize antimicrobial prescribing practices in humans and domestic animals. In contrast, there is relatively little known about specific prescribing practices in wild animals despite the wide use of antimicrobials and other microbial interventions, such as probiotics to treat captive wildlife. Therefore, the goal of this study was to examine current antimicrobial and probiotic use from a cross-section of wildlife rehabilitation facilities in the United States. An anonymous electronic survey was sent to 105 United States permitted wildlife facilities to collect information about admissions, current antimicrobial and probiotic use practices, and current staff knowledge and attitudes surrounding antimicrobial resistance and probiotic effectiveness. Respondents from over 50% of facilities participated in the survey (54/105), including 45 facilities that treated birds. All facilities reported using antimicrobials, including some from groups considered critically important for human medicine, for a wide range of medical conditions and prophylaxis. Among antibiotics, enrofloxacin and amoxicillin-clavulanic acid were the most commonly used. Antifungals were not as widespread, but itraconazole was the most commonly used. Over 75% of respondents said that their facilities would benefit from having standardized antimicrobial guidelines in place. Probiotics were also used in more than 50% of facilities, but there was notable disparity in opinions regarding their efficacy. The results of this survey are a first step towards understanding antimicrobial and probiotic use practices in the treatment of captive wildlife and developing an antimicrobial stewardship program for wildlife rehabilitation.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0308261)

10.1371/journal.pone.0308261

---

## The impact of management on the fecal microbiome of endangered greater sage-grouse (Centrocercus urophasianus) in a zoo-based conservation program

[Source](https://academic.oup.com/conphys/article/12/1/coae052/7728823)

---

## Microdiversity in marine pelagic ammonia-oxidizing archaeal populations in a Mediterranean long-read metagenome

[Source](https://enviromicro-journals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.16684)

---

## Production of structurally diverse sphingolipids by anaerobic marine bacteria in the euxinic Black Sea water column.
 08.08.2024, _S Ding, FAB von Meijenfeldt, NJ Bale, JS Sinninghe Damsté, L Villanueva_


Microbial lipids, used as taxonomic markers and physiological indicators, have mainly been studied through cultivation. However, this approach is limited due to the scarcity of cultures of environmental microbes, thereby restricting insights into the diversity of lipids and their ecological roles. Addressing this limitation, here we apply metalipidomics combined with metagenomics in the Black Sea, classifying and tentatively identifying 1623 lipid-like species across 18 lipid classes. We discovered over 200 novel, abundant, and structurally diverse sphingolipids in euxinic waters, including unique 1-deoxysphingolipids with long-chain fatty acids and sulfur-containing groups. Sphingolipids were thought to be rare in bacteria and their molecular and ecological functions in bacterial membranes remain elusive. However, genomic analysis focused on sphingolipid biosynthesis genes revealed that members of 38 bacterial phyla in the Black Sea can synthesize sphingolipids, representing a fourfold increase from previously known capabilities and accounting for up to 25% of the microbial community. These sphingolipids appear to be involved in oxidative stress response, cell wall remodeling and are associated with the metabolism of nitrogen-containing molecules. Our findings underscore the effectiveness of multi-omics approaches in exploring microbial chemical ecology.

[Source](https://academic.oup.com/ismej/advance-article/doi/10.1093/ismejo/wrae153/7729370)

10.1093/ismejo/wrae153

---

## Characterization of fungal microbiome structure in leaf litter compost through metagenomic profiling for harnessing the bio-organic fertilizer potential.
 09.08.2024, _S Mahongnao, P Sharma, S Nanda_


Sustainable waste management through composting has gain renewed attention since it could upcycle organic waste into valuable bio-organic fertiliser. This study explored the composition of fungal communities in leaf litter and organic waste composts ecosystems by employing advanced internal transcribed spacer (ITS) metagenomic profiling. This approach provides insights into the diversity, composition, and potential functions of these fungi, offering practical implications for optimising composting processes and enhancing sustainable waste management practices. Various organic composts were collected, including leaf litter composts, from different sources in Delhi-National Capital Region, India, and fungal microbiome composition were characterised through ITS profiling. Results revealed that leaf litter composts and cow dung manure had the highest fungal read counts, while kitchen waste compost had the lowest. Alpha diversity indices, including Chao1 and Shannon, exhibited differences in species richness and diversity among composts, though statistical significance was limited. The leaf composts had relatively higher alpha diversity than the other organic waste composts analysed. The study also identified dominant fungal genera specifically, Wallemia, Geotrichum, Pichia, Mycothermus, Mortierella, Aspergillus, Fusarium, and Basidiobolus, across the compost samples. The presence of beneficial fungal genera like Pichia, Geotrichum, Trichoderma, Mortierella, Basidiobolus, Aspergillus, and others were detected in leaf waste compost and the other organic waste composts. There was also presence of some pathogenic genera viz. Alternaria, Fusarium, and Acremonium, in these composts which underscored the need for proper composting practices and source selection to optimise soil fertility and minimise disease risks in agriculture. Remarkably, leaf compost has highest proportion of beneficial genera with least observed abundance of pathogens. On the other hand, the municipal organic waste compost has least proportion of beneficial genera with higher abundance of pathogens. Overall, these findings contributed to characterisation of composting processes, advancing waste management practices, and enhancing the use of leaf compost as a bio-organic fertiliser.

[Source](https://link.springer.com/article/10.1007/s13205-024-04028-0)

10.1007/s13205-024-04028-0

---

## Plant species influences the composition of root system microbiome and its antibiotic resistance profile in a constructed wetland receiving primary treated wastewater

[Source](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2024.1436122/full)

---

## DNA stable isotope probing reveals the impact of trophic interactions on bioaugmentation of soils with different pollution histories.
 09.08.2024, _EE Nieto, SD Jurburg, N Steinbach, S Festa, IS Morelli, BM Coppotelli, A Chatzinotas_


_Soil Microbiology, Soil, Soil Pollutants, Biodegradation, Environmental, RNA, Ribosomal, 16S, RNA, Ribosomal, 18S, Bacteria, Biomass, Carbon Isotopes, Food Chain, Polycyclic Aromatic Hydrocarbons, Isotope Labeling_
Bioaugmentation is considered a sustainable and cost-effective methodology to recover contaminated environments, but its outcome is highly variable. Predation is a key top-down control mechanism affecting inoculum establishment, however, its effects on this process have received little attention. This study focused on the impact of trophic interactions on bioaugmentation success in two soils with different pollution exposure histories. We inoculated a 13C-labelled pollutant-degrading consortium in these soils and tracked the fate of the labelled biomass through stable isotope probing (SIP) of DNA. We identified active bacterial and eukaryotic inoculum-biomass consumers through amplicon sequencing of 16S rRNA and 18S rRNA genes coupled to a novel enrichment factor calculation.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01865-2)

10.1186/s40168-024-01865-2

---

## Seed endophytes and rhizosphere microbiome of Imperata cylindrica, a pioneer plant of abandoned mine lands.
 09.08.2024, _W Mao, Y Wu, Q Li, Y Xiang, W Tang, H Hu, X Ji, H Li_


Some plant-associated microorganisms could improve host plants biotic and abiotic stress tolerance. Imperata cylindrica is a dominant pioneer plant in some abandoned mine lands with higher concentrations of heavy metal (HM). To discover the specific microbiome of I. cylindrica in this extreme environment and evaluate its role, the microbiome of I. cylindrica's seeds and rhizosphere soils from HM heavily contaminated (H) and lightly contaminated (L) sites were studied. It was found that HM-contamination significantly reduced the richness of endophytic bacteria in seeds, but increased the abundance of resistant species, such as Massilia sp. and Duganella sp. Spearman's rank correlation coefficient analysis showed that both Massilia sp. and Duganella sp. showed a significant positive correlation with Zn concentration, indicating that it may have a strong tolerance to Zn. A comparison of the microbiome of rhizosphere soils (RS) and adjacent bare soils (BS) of site H showed that I. cylindrica colonization significantly increased the diversity of fungi in rhizosphere soil and the abundance of Ascomycota associated with soil nutrient cycling. Spearman's rank correlation coefficient analysis showed that Ascomycota was positively correlated with the total nitrogen. Combined with the fact that the total nitrogen content of RS was significantly higher than that of BS, we suppose that Ascomycota may enhance the nitrogen fixation of I. cylindrica, thereby promoting its growth in such an extreme environment. In conclusion, the concentration of HM and nutrient contents in the soil significantly affected the microbial community of rhizosphere soils and seeds of I. cylindrica, in turn, the different microbiomes further affected soil HM concentration and nutrient contents. The survival of I. cylindrica in HM severely contaminated environment may mainly be through recruiting more microorganisms that can enhance its nutrition supply.

[Source](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2024.1415329/full)

10.3389/fmicb.2024.1415329

---

## Temperature-dependent trophic associations modulate soil bacterial communities along latitudinal gradients.
 08.08.2024, _X Huang, J Wang, K Dumack, K Anantharaman, B Ma, Y He, W Liu, H Di, Y Li, J Xu_


Understanding the environmental and biological mechanisms shaping latitudinal patterns in microbial diversity is challenging in the field of ecology. Although multiple hypotheses have been proposed to explain these patterns, a consensus has rarely been reached. Here, we conducted a large-scale field survey and microcosm experiments to investigate how environmental heterogeneity and putative trophic interactions (exerted by protist-bacteria associations and T4-like virus-bacteria associations) affect soil bacterial communities along a latitudinal gradient. We found that the microbial latitudinal diversity was kingdom dependent, showing decreasing, clumped, and increasing trends in bacteria, protists, and T4-like viruses, respectively. Climatic and edaphic drivers played predominant roles in structuring the bacterial communities, the intensity of the climatic effect increased sharply from 30°N to 32°N, whereas the intensity of the edaphic effect remained stable. Biotic associations were also essential in shaping the bacterial communities, with protist-bacteria associations showing a quadratic distribution, whereas virus-bacteria associations were significant only at high latitudes. The microcosm experiments further revealed that the temperature component, which is affiliated with climate conditions, is the primary regulator of trophic associations along the latitudinal gradient. Overall, our study highlights a previously underestimated mechanism of how the putative biotic interactions influence bacterial communities and their response to environmental gradients.

[Source](https://academic.oup.com/ismej/advance-article/doi/10.1093/ismejo/wrae145/7729363)

10.1093/ismejo/wrae145

---

## Models of microbiome evolution incorporating host resource provisioning

[Source](https://www.biorxiv.org/content/10.1101/2024.08.05.606723v1.full)

---

## Combining Bifidobacterium longum subsp. infantis and human milk oligosaccharides synergistically increases short chain fatty acid production ex vivo.
 07.08.2024, _F De Bruyn, K James, G Cottenet, M Dominick, J Katja_


_Humans, Milk, Human, Oligosaccharides, Fatty Acids, Volatile, Infant, Bifidobacterium longum subspecies infantis, Bifidobacterium, Probiotics, Gastrointestinal Microbiome, Feces, Female, Synbiotics, Child, Preschool_
To enhance health benefits, a probiotic can be co-administered with a metabolizable prebiotic forming a synergistic synbiotic. We assessed the synergies resulting from combining Bifidobacterium longum subsp. infantis LMG 11588 and an age-adapted blend of six human milk oligosaccharides (HMOs) in ex vivo colonic incubation bioreactors seeded with fecal background microbiota from infant and toddler donors. When HMOs were combined with B. infantis LMG 11588, they were rapidly and completely consumed. This resulted in increased short chain fatty acid (SCFA) production compared to the summed SCFA production from individual ingredients (synergy). Remarkably, HMOs were partially consumed for specific infant donors in the absence of B. infantis LMG 11588, yet all donors showed increased SCFA production upon B. infantis LMG 11588 supplementation. We found specific bacterial taxa associated with the differential response pattern to HMOs. Our study shows the importance of carefully selecting pre- and probiotic into a synergistic synbiotic that could benefit infants.

[Source](https://www.nature.com/articles/s42003-024-06628-1)

10.1038/s42003-024-06628-1

---

