## Gut microbial carbohydrate metabolism contributes to insulin resistance.
 30.08.2023, _T Takeuchi, T Kubota, Y Nakanishi, H Tsugawa, W Suda, AT Kwon, J Yazaki, K Ikeda, S Nemoto, Y Mochizuki, T Kitami, K Yugi, Y Mizuno, N Yamamichi, T Yamazaki, I Takamoto, N Kubota, T Kadowaki, E Arner, P Carninci, O Ohara, M Arita, M Hattori, S Koyasu, H Ohno_


Insulin resistance is the primary pathophysiology underlying metabolic syndrome and type 2 diabetes1,2. Previous metagenomic studies have described the characteristics of gut microbiota and their roles in metabolizing major nutrients in insulin resistance3-9. In particular, carbohydrate metabolism of commensals has been proposed to contribute up to 10% of the host's overall energy extraction10, thereby playing a role in the pathogenesis of obesity and prediabetes3,4,6. Nevertheless, the underlying mechanism remains unclear. Here we investigate this relationship using a comprehensive multi-omics strategy in humans. We combine unbiased faecal metabolomics with metagenomics, host metabolomics and transcriptomics data to profile the involvement of the microbiome in insulin resistance. These data reveal that faecal carbohydrates, particularly host-accessible monosaccharides, are increased in individuals with insulin resistance and are associated with microbial carbohydrate metabolisms and host inflammatory cytokines. We identify gut bacteria associated with insulin resistance and insulin sensitivity that show a distinct pattern of carbohydrate metabolism, and demonstrate that insulin-sensitivity-associated bacteria ameliorate host phenotypes of insulin resistance in a mouse model. Our study, which provides a comprehensive view of the host-microorganism relationships in insulin resistance, reveals the impact of carbohydrate metabolism by microbiota, suggesting a potential therapeutic target for ameliorating insulin resistance.

[Source](https://doi.org/10.1038/s41586-023-06466-x)

10.1038/s41586-023-06466-x

---

## Gut Microbiome in Patients With Early-Stage and Late-Stage Melanoma.
 02.09.2023, _RG Witt, SH Cass, T Tran, A Damania, EE Nelson, E Sirmans, EM Burton, M Chelvanambi, S Johnson, HA Tawbi, JE Gershenwald, MA Davies, C Spencer, A Mishra, MC Wong, NJ Ajami, CB Peterson, CR Daniel, JA Wargo, JL McQuade, KC Nelson_


The gut microbiome modulates the immune system and responses to immunotherapy in patients with late-stage melanoma. It is unknown whether fecal microbiota profiles differ between healthy individuals and patients with melanoma or if microbiota profiles differ among patients with different stages of melanoma. Defining gut microbiota profiles in individuals without melanoma and those with early-stage and late-stage melanoma may reveal features associated with disease progression.

[Source](https://doi.org/10.1001/jamadermatol.2023.2955)

10.1001/jamadermatol.2023.2955

---

## The upper respiratory tract microbiota of healthy adults is affected by <em>Streptococcus pneumoniae</em> carriage, smoking habits, and contact with children

[Source](https://doi.org/10.1186/s40168-023-01640-9)

---

## Salivary microbiome of healthy women of reproductive age.
 01.09.2023, _E Cauwenberghs, E Oerlemans, S Wittouck, CN Allonsius, T Gehrmann, S Ahannach, I De Boeck, I Spacova, PA Bron, G Donders, V Verhoeven, S Lebeer_


The human salivary microbial community plays a crucial role in local and systemic diseases. Biological and lifestyle factors such as menstrual cycle, oral hygiene, and smoking have been documented to impact this community. However, while hormonal contraceptives are the most prescribed drug in healthy women and intimate partners play key roles in microbial exchange between humans, their impact on the salivary microbiome of women of reproductive age have been understudied. Additionally, the role of other lifestyle factors such as diet, allergies, age, and stress on the saliva microbiome of the general population is not well understood. Here, we studied the salivary microbiome of 255 healthy women of reproductive age using self-sampling kits and 16S rRNA amplicon sequencing combined with questionnaires on lifestyle and host-related parameters. A preserved salivary bacterial community of 12 genera (Actinobacillus, Actinomyces, Alloprevotella, Campylobacter, Fusobacterium, Gemella, Granulicatella, Leptotrichia, Neisseria, Prevotella, Streptococcus, and Veillonella) was identified. Contrary to what we expected, the number of intimate partners or specific contraceptive use did not have a major impact on these bacterial communities. However, recent use of oral antibiotics was associated with a significant decrease in richness at genus level and increase in mean relative abundances of several taxa. Being stressed or nervous was associated with a significantly increased richness of the salivary microbiome at the level of amplicon sequencing variants . Nevertheless, these associations with host-related and lifestyle variables only appeared to be subtle, suggesting that the salivary microbiome is mainly driven by the buccal environment and health status of an individual. IMPORTANCE The salivary microbiome has been proven to play a crucial role in local and systemic diseases. Moreover, the effects of biological and lifestyle factors such as oral hygiene and smoking on this microbial community have already been explored. However, what was not yet well understood was the natural variation of the saliva microbiome in healthy women and how this is associated with specific use of hormonal contraception and with the number of different sexual partners with whom microbiome exchange is expected regularly. In this paper, we characterized the salivary microbiome of 255 healthy women of reproductive age using an in-depth questionnaire and self-sampling kits. Using the large metadata set, we were able to investigate the associations of several host-related and lifestyle variables with the salivary microbiome profiles. Our study shows a high preservation between individuals.

[Source](https://doi.org/10.1128/mbio.00300-23)

10.1128/mbio.00300-23

---

## Microbiome characteristics description of COVID-19 patients based on bulk RNA-seq and scRNA-Seq data.
 31.08.2023, _S Zhang, X Liu, Y Zhao, P Wang, R Yu, P Xu, Y Jiang, L Cheng_


After infection with SARS-CoV-2, the microbiome inside the human body changes dramatically. By re-annotating microbial sequences in bulk RNA-seq and scRNA-seq data of COVID-19 patients, we described the cellular microbial landscape of COVID-19 patients and identified characteristic microorganisms in various tissues. We found that Acinetobacter lwoffii was highly correlated with COVID-19 symptoms and might disrupt some pathways of patients by interacting with the host and other microbes, such as Klebsiella pneumoniae. We further identified characteristic microorganisms specific to cell type, indicating the enrichment preference of some microbes. We also revealed the co-infection of SARS-CoV-2 with hMPV, which may cause the development of COVID-19. Overall, we demonstrated that the presence of intracellular microorganisms in COVID-19 patients and the synergies between microorganisms were strongly correlated with disease progression, providing a theoretical basis for COVID-19 treatment in a certain extent.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S001048252300865X?via%3Dihub)

10.1016/j.compbiomed.2023.107400

---

## Parkinson's disease alters the composition of subgingival microbiome.
 01.09.2023, _E Yay, M Yilmaz, H Toygar, N Balci, C Alvarez Rivas, B Bolluk Kilic, A Zirh, B Paster, A Kantarci_


The current study aimed to test the hypothesis that Parkinson's disease exacerbates periodontitis by altering its microbiome.

[Source](https://doi.org/10.1080/20002297.2023.2250650)

10.1080/20002297.2023.2250650

---

## Cervicovaginal <em>Gardnerella</em> sialidase-encoding gene in persistent human papillomavirus infection

[Source](https://doi.org/10.1038/s41598-023-41469-8)

---

## Human milk oligosaccharides modulate the intestinal microbiome of healthy adults.
 06.09.2023, _JP Jacobs, ML Lee, DJ Rechtman, AK Sun, C Autran, V Niklas_


_Adult, Female, Humans, Infant, Newborn, Gastrointestinal Microbiome, Milk, Human, Oligosaccharides, RNA, Ribosomal, 16S_
Human milk contains over 200 distinct oligosaccharides, which are critical to shaping the developing neonatal gut microbiome. To investigate whether a complex mixture of human milk oligosaccharides (HMOs) would similarly modulate the adult gut microbiome, HMO-Concentrate derived from pooled donor breast milk was administered orally to 32 healthy adults for 7 days followed by 21 days of monitoring. Fecal samples were collected for 16S rRNA gene sequencing, shotgun metagenomics, and metabolomics analyses. HMO-Concentrate induced dose-dependent Bifidobacterium expansion, reduced microbial diversity, and altered microbial gene content. Following HMO cessation, a microbial succession occurred with diverse taxonomic changes-including Bacteroides expansion-that persisted through day 28. This was associated with altered microbial gene content, shifts in serum metabolite levels, and increased circulating TGFÎ² and IL-10. Incubation of cultured adult microbiota with HMO-Concentrate induced dose-dependent compositional shifts that were not recapitulated by individual HMOs or defined mixtures of the 10 most abundant HMOs in HMO-Concentrate at their measured concentrations. These findings support that pooled donor HMOs can exert direct effects on adult gut microbiota and that complex mixtures including low abundance HMOs present in donor milk may be required for maximum effect.Registration: ClinicalTrials.gov NCT05516225.

[Source](https://doi.org/10.1038/s41598-023-41040-5)

10.1038/s41598-023-41040-5

---

## Intestinal microbiome and metabolome signatures in patients with chronic granulomatous disease.
 02.09.2023, _P Chandrasekaran, Y Han, CS Zerbe, T Heller, SS DeRavin, SA Kreuzberg, BE Marciano, Y Siu, DR Jones, RS Abraham, MC Stephens, AM Tsou, S Snapper, S Conlan, P Subramanian, M Quinones, C Grou, V Calderon, C Deming, JW Leiding, DE Arnold, BR Logan, LM Griffith, A Petrovic, TI Mousallem, N Kapoor, JR Heimall, JL Barnum, M Kapadia, N Wright, A Rayes, S Chandra, LA Broglie, D Chellapandian, CL Deal, E Grunebaum, SS Lim, K Mallhi, RA Marsh, L Murguia-Favela, S Parikh, F Touzot, MJ Cowan, CC Dvorak, E Haddad, DB Kohn, L Notarangelo, SY Pai, JM Puck, MA Pulsipher, TR Torgerson, EM Kang, HL Malech, JA Segre, CE Bryant, SM Holland, EL Falcone_


CGD is caused by defects in any of the 6 subunits forming the nicotinamide adenine dinucleotide phosphate (NADPH) oxidase complex 2 (NOX2), leading to severely reduced or absent phagocyte-derived ROS production. Almost 50% of patients with chronic granulomatous disease (CGD) have IBD (CGD-IBD). While conventional IBD therapies can treat CGD-IBD, their benefits must be weighed against the risk of infection. Understanding the impact of NOX2 defects on the intestinal microbiota may lead to the identification of novel CGD-IBD treatments.

[Source](https://www.jacionline.org/article/S0091-6749(23)01101-6/pdf)

10.1016/j.jaci.2023.07.022

---

## The curious case of <em>Prevotella copri</em>

[Source](https://doi.org/10.1080/19490976.2023.2249152)

---

## COVID-19 Vaccine Acceptance Among Healthcare Students in Vietnam, Based on Health Belief Model.
 07.09.2023, _DT Tieu Mai, PT Thuy_


Healthcare students have a high risk of acquiring COVID-19 while practicing in medical facilities, and their health-related decisions might considerably impact the people around them. In the circumstance of many people refusing vaccination, the delay in accepting the COVID-19 vaccine by this group could be a barrier to providing effective immunity to the entire population against the COVID-19 pandemic.

[Source](https://doi.org/10.1007/s11274-023-03742-w)

10.1177/23779608231196662

---

## Gut microbiota and microbiota-derived metabolites in colorectal cancer: enemy or friend

[Source](https://doi.org/10.1007/s11274-023-03742-w)

---

## N-Acetylglucosamine and its dimer ameliorate inflammation in murine colitis by strengthening the gut barrier function.
 01.09.2023, _SI Choi, YC Shin, JS Lee, YC Yoon, JM Kim, MK Sung_


Ulcerative colitis (UC) is a chronic gastrointestinal disease whose incidence is increasing rapidly worldwide. Anti-inflammatory medications, including 5-aminosalicylic acid (5-ASA), corticosteroids, and immunosuppressants, are used for its treatment; however, new alternatives would be required due to the serious side effects of some of these medications. N-Acetylglucosamine (NAG) is an amino sugar composed of mucin that is secreted by intestinal epithelial cells. It is also used to promote the growth of intestinal bacteria. The current study aimed to determine the efficacy of NAG against dextran sulfate sodium (DSS)-induced chronic colitis and elucidate its mechanism of action. Mice were randomly divided into control, DSS, 0.1% sulfasalazine, 0.1% NAG, 0.3% NAG, and 0.3% NAG-dimer (NAG-D) groups, and results showed that colitis-induced body weight loss, disease activity, colonic tissue damage, colon length shortening, and the loss of mucin-secreting area were significantly improved in the NAG-D group. The intestinal permeability indicator, serum CD 14 level, and expression of the tight junction protein, occludin, were both improved in the 0.3% NAG group. Inflammatory biomarkers, including GATA3, IFN-Î³, p-IÎºBÎ±, COX2, TGF-Î²1, and Smad7, were significantly lower in the 0.3% NAG and NAG-D groups than in the DSS group. The intestinal microbial composition was most significantly altered in the 0.3% NAG group, showing decreased ratios of pathogenic bacteria, such as Betaproteobacteria, especially Burkholderiales. The results overall suggested that NAG or NAG-D supplementation can alleviate inflammation by strengthening the intestinal barrier function and maintaining gut microbiota homeostasis in a DSS-induced colitis mouse model.

[Source](https://doi.org/10.1039/d3fo00282a)

10.1039/d3fo00282a

---

## Omega-3 nanoemulgel in prevention of radiation-induced oral mucositis and its associated effect on microbiome: a randomized clinical trial.
 02.09.2023, _BM Morsy, S El Domiaty, MAM Meheissen, LA Heikal, MA Meheissen, NM Aly_


_Humans, Quality of Life, Dysbiosis, Stomatitis, Mucositis, Microbiota_
Oral mucositis (OM) is recognized as one of the most frequent debilitating sequelae encountered by head and neck cancer (HNC) patients treated by radiotherapy. This results in severe mucosal tissue inflammation and oral ulcerations that interfere with patient's nutrition, quality of life (QoL) and survival. Omega-3 (Ï-3) polyunsaturated fatty acids (PUFAs) have recently gained special interest in dealing with oral diseases owing to its anti-inflammatory, anti-oxidant and wound healing properties. Thus, this study aims to assess topical Omega-3 nanoemulgel efficacy in prevention of radiation-induced oral mucositis and regulation of oral microbial dysbiosis.

[Source](https://doi.org/10.1186/s12903-023-03276-5)

10.1186/s12903-023-03276-5

---

## <em>Lactiplantibacillus plantarum</em> ELF051 Alleviates Antibiotic-Associated Diarrhea by Regulating Intestinal Inflammation and Gut Microbiota

[Source](https://doi.org/10.1007/s12602-023-10150-x)

---

## Microbiomes and metabolomes of dominant coral reef primary producers illustrate a potential role for immunolipids in marine symbioses.
 05.09.2023, _H Mannochio-Russo, SOI Swift, KK Nakayama, CB Wall, EC Gentry, M Panitchpakdi, AM Caraballo-Rodriguez, AT Aron, D Petras, K Dorrestein, TK Dorrestein, TM Williams, EM Nalley, NT Altman-Kurosaki, M Martinelli, JY Kuwabara, JL Darcy, VS Bolzani, L Wegley Kelly, C Mora, JY Yew, AS Amend, M McFall-Ngai, NA Hynson, PC Dorrestein, CE Nelson_


_Animals, Coral Reefs, Symbiosis, Metabolome, Anthozoa, Microbiota, Seaweed_
The dominant benthic primary producers in coral reef ecosystems are complex holobionts with diverse microbiomes and metabolomes. In this study, we characterize the tissue metabolomes and microbiomes of corals, macroalgae, and crustose coralline algae via an intensive, replicated synoptic survey of a single coral reef system (Waimea Bay, O'ahu, Hawaii) and use these results to define associations between microbial taxa and metabolites specific to different hosts. Our results quantify and constrain the degree of host specificity of tissue metabolomes and microbiomes at both phylum and genus level. Both microbiome and metabolomes were distinct between calcifiers (corals and CCA) and erect macroalgae. Moreover, our multi-omics investigations highlight common lipid-based immune response pathways across host organisms. In addition, we observed strong covariation among several specific microbial taxa and metabolite classes, suggesting new metabolic roles of symbiosis to further explore.

[Source](https://doi.org/10.1038/s42003-023-05230-1)

10.1038/s42003-023-05230-1

---

## Alteration of diet microbiota limits the experimentally evolved immune priming response in flour beetles, but not pathogen resistance

[Source](https://doi.org/10.1111/jeb.14213)

---

## Mammals show distinct functional gut microbiome dynamics to identical series of environmental stressors.
 31.08.2023, _A Koziol, I Odriozola, A Leonard, R Eisenhofer, C San JosÃ©, O Aizpurua, A Alberdi_


The ability of the gut microbiome has been posited as an additional axis of animals' phenotypic plasticity. However, whether and how such plasticity varies across hosts with different biological features remains unclear. We performed a captivity experiment to compare how the taxonomic, phylogenetic, and functional microbial dynamics varied across a series of temperature and dietary disturbances in two mammals: the insectivorous-specialist Crocidura russula and the omnivorous-generalist Apodemus sylvaticus. Combining genome-resolved metagenomics, metabolic pathway distillation and joint species distribution modeling, we observed that, although microbiome alpha diversity of both species remained stable, C. russula exhibited substantially higher variability and directionality of microbial responses than A. sylvaticus. Our results indicate that the intrinsic properties (e.g., diversity and functional redundancy) of microbial communities coupled with physiological attributes (e.g., thermal plasticity) of hosts shape the taxonomic, phylogenetic, and functional response of gut microbiomes to environmental stressors, which might influence their contribution to the acclimation and adaptation capacity of animal hosts. IMPORTANCE In our manuscript, we report the first interspecific comparative study about the plasticity of the gut microbiota. We conducted a captivity experiment that exposed wild-captured mammals to a series of environmental challenges over 45 days. We characterized their gut microbial communities using genome-resolved metagenomics and modeled how the taxonomic, phylogenetic, and functional microbial dynamics varied across a series of disturbances in both species. Our results indicate that the intrinsic properties (e.g., diversity and functional redundancy) of microbial communities coupled with physiological attributes (e.g., thermal plasticity) of hosts shape the taxonomic, phylogenetic, and functional response of gut microbiomes to environmental stressors, which might influence their contribution to the acclimation and adaptation capacity of animal hosts.

[Source](https://doi.org/10.1128/mbio.01606-23)

10.1128/mbio.01606-23

---

## The hologenome of <em>Daphnia magna </em>reveals possible DNA methylation and microbiome-mediated evolution of the host genome

[Source](https://doi.org/10.1093/nar/gkad685)

---

## Impact of intraspecific variation in insect microbiomes on host phenotype and evolution.
 02.09.2023, _C Lange, S Boyer, TM Bezemer, MC Lefort, MK Dhami, E Biggs, R Groenteman, SV Fowler, Q Paynter, AM Verdecia Mogena, M Kaltenpoth_


Microbes can be an important source of phenotypic plasticity in insects. Insect physiology, behaviour, and ecology are influenced by individual variation in the microbial communities held within the insect gut, reproductive organs, bacteriome, and other tissues. It is becoming increasingly clear how important the insect microbiome is for insect fitness, expansion into novel ecological niches, and novel environments. These investigations have garnered heightened interest recently, yet a comprehensive understanding of how intraspecific variation in the assembly and function of these insect-associated microbial communities can shape the plasticity of insects is still lacking. Most research focuses on the core microbiome associated with a species of interest and ignores intraspecific variation. We argue that microbiome variation among insects can be an important driver of evolution, and we provide examples showing how such variation can influence fitness and health of insects, insect invasions, their persistence in new environments, and their responses to global environmental changes. A and B are two stages of an individual or a population of the same species. The drivers lead to a shift in the insect associated microbial community, which has consequences for the host. The complex interplay of those consequences affects insect adaptation and evolution and influences insect population resilience or invasion.

[Source](https://doi.org/10.1038/s41396-023-01500-2)

10.1038/s41396-023-01500-2

---

## Why do hosts malfunction without microbes? Missing benefits versus evolutionary addiction.
 31.08.2023, _TJ Hammer_


Microbes are widely recognized to be vital to host health. This new consensus rests, in part, on experiments showing how hosts malfunction when microbes are removed. More and more microbial dependencies are being discovered, even in fundamental processes such as development, immunity, physiology, and behavior. But why do they exist? The default explanation is that microbes are beneficial; when hosts lose microbes, they also lose benefits. Here I call attention to evolutionary addiction, whereby a host trait evolves a need for microbes without having been improved by them. Evolutionary addiction should be considered when interpreting microbe-removal experiments, as it is a distinct and potentially common process. Further, it may have unique implications for the evolution and stability of host-microbe interactions.

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(23)00224-X)

10.1016/j.tim.2023.07.012

---

## Different microbial assemblage colonized on microplastics and clay particles in aerobic sludge treatment.
 02.09.2023, _Z Zhang, H Zhang, H Qiang, P Liu, X Guo, L Zhu_


In this study, a combination of property analysis and high-throughput sequencing was used to investigate the microbial colonization ability and their community structures and functions in polypropylene microplastics (PPMPs), polystyrene microplastics (PSMPs) and montmorillonite (MMT), respectively as the representatives of artificial and natural substrates in aerobic sludge treatment. After 45 d of incubation, the surface properties of substrates were altered with the increased oxygen functional groups and surface roughness, indicating microbial settlement. Moreover, MPs had different microbial structures from that of MMT, and PSMPs exhibited higher microbial diversity and abundance than PPMPs and MMT. Also, these substrates changed the inherent ecological niche in sludge. Especially, the abundance of some pathogens (e.g., Pseudomonas, Klebsiella and Flavobacterium) was increased in MPs, and the disease risk of Kyoto Encyclopedia of Genes and Genomes metabolic pathway (e.g., Infectious diseases: Bacterial, Infectious diseases: Parasitic and Immune diseases) was higher. Also, the presence of MPs inhibited the decomposition of organic matter including soluble chemical oxygen demand and protein compared to natural substrates. The findings revealed the crucial vector role of MPs for microbes and the effect on aerobic sludge treatment, highlighting the necessity of MP removal in sludge.

[Source](https://doi.org/10.1016/j.scitotenv.2023.166473)

10.1016/j.scitotenv.2023.166473

---

## Metagenomic insights into the microbiota involved in lactate and butyrate production and manipulating their synthesis in alfalfa silage.
 03.09.2023, _R Li, M Hu, X Jiang, C Xu_


Lactate and butyrate are important indicators of silage quality. However, the microorganisms and mechanisms responsible for lactate and butyrate production in silage are not well documented.

[Source](https://doi.org/10.1093/jambio/lxad197)

10.1093/jambio/lxad197

---

## RSim: A reference-based normalization method via rank similarity.
 01.09.2023, _B Yuan, S Wang_


Microbiome sequencing data normalization is crucial for eliminating technical bias and ensuring accurate downstream analysis. However, this process can be challenging due to the high frequency of zero counts in microbiome data. We propose a novel reference-based normalization method called normalization via rank similarity (RSim) that corrects sample-specific biases, even in the presence of many zero counts. Unlike other normalization methods, RSim does not require additional assumptions or treatments for the high prevalence of zero counts. This makes it robust and minimizes potential bias resulting from procedures that address zero counts, such as pseudo-counts. Our numerical experiments demonstrate that RSim reduces false discoveries, improves detection power, and reveals true biological signals in downstream tasks such as PCoA plotting, association analysis, and differential abundance analysis.

[Source](https://doi.org/10.1371/journal.pcbi.1011447)

10.1371/journal.pcbi.1011447

---

## Computation-aided studies related to the induction of specialized metabolite biosynthesis in microbial co-cultures: An introductory overview

[Source](https://www.csbj.org/article/S2001-0370(23)00286-6/fulltext)

---

## Analyzing microbial evolution through gene and genome phylogenies.
 05.09.2023, _S Teichman, MD Lee, AD Willis_


Microbiome scientists critically need modern tools to explore and analyze microbial evolution. Often this involves studying the evolution of microbial genomes as a whole. However, different genes in a single genome can be subject to different evolutionary pressures, which can result in distinct gene-level evolutionary histories. To address this challenge, we propose to treat estimated gene-level phylogenies as data objects, and present an interactive method for the analysis of a collection of gene phylogenies. We use a local linear approximation of phylogenetic tree space to visualize estimated gene trees as points in low-dimensional Euclidean space, and address important practical limitations of existing related approaches, allowing an intuitive visualization of complex data objects. We demonstrate the utility of our proposed approach through microbial data analyses, including by identifying outlying gene histories in strains of Prevotella , and by contrasting Streptococcus phylogenies estimated using different gene sets. Our method is available as an open-source R package, and assists with estimating, visualizing and interacting with a collection of bacterial gene phylogenies. dimension reduction, microbiome, non-Euclidean, statistical genetics, visualization.

[Source](https://doi.org/10.1101/2023.08.15.553440)

10.1101/2023.08.15.553440

---

