## Spatially restricted immune and microbiota-driven adaptation of the gut.
 20.11.2024, _T Mayassi, C Li, Å Segerstolpe, EM Brown, R Weisberg, T Nakata, H Yano, P Herbst, D Artis, DB Graham, RJ Xavier_


The intestine is characterized by an environment in which host requirements for nutrient and water absorption are consequently paired with the requirements to establish tolerance to the outside environment. To better understand how the intestine functions in health and disease, large efforts have been made to characterize the identity and composition of cells from different intestinal regions1-8. However, the robustness, nature of adaptability and extent of resilience of the transcriptional landscape and cellular underpinning of the intestine in space are still poorly understood. Here we generated an integrated resource of the spatial and cellular landscape of the murine intestine in the steady and perturbed states. Leveraging these data, we demonstrated that the spatial landscape of the intestine was robust to the influence of the microbiota and was adaptable in a spatially restricted manner. Deploying a model of spatiotemporal acute inflammation, we demonstrated that both robust and adaptable features of the landscape were resilient. Moreover, highlighting the physiological relevance and value of our dataset, we identified a region of the middle colon characterized by an immune-driven multicellular spatial adaptation of structural cells to the microbiota. Our results demonstrate that intestinal regionalization is characterized by robust and resilient structural cell states and that the intestine can adapt to environmental stress in a spatially controlled manner through the crosstalk between immunity and structural cell homeostasis.

[Source](https://doi.org/10.1038/s41586-024-08216-z)

10.1038/s41586-024-08216-z

---

## Bacterial interactions underpin worsening lung function in cystic fibrosis-associated infections.
 22.11.2024, _DW Rivett, LR Hatfield, H Gavillet, M Hardman, C van der Gast_


Chronic lung infections are the primary cause of morbidity and early mortality in cystic fibrosis (CF) and, as such, have been the subject of a great deal of research. Subsequently, they have become one of the key paradigms for polymicrobial infections. The literature, however, has traditionally focused on the presence of pathogens in isolation or univariate measures like number of species to predict decline of lung function and ignores large swathes of data. Here, we suggest that looking at the interactions between species identified by 16S rRNA gene sequencing, rather than at species singularly, could elucidate hitherto unknown properties of these complicated infections. To confirm this, pooled samples from studies conducted by our laboratory, sequenced using the same pipeline, were used to assess microbiome-wide associations to lung function. We found pathogenic interactions between species were limited to the most abundant species, which were composed of canonical CF pathogens (including Pseudomonas, Staphylococcus, Stenotrophomonas, and Achromobacter) and commensals. This observation is crucial for better understanding of polymicrobial infections and treatment of these conditions while providing a simple framework for expanding this research into other disease states. The adoption of ecological principles into infection science can provide better understanding and options to those suffering from chronic conditions. The statistical ecology approach presented here enables clear hypotheses from observational data that can be ratified through subsequent manipulative experimental studies. Moreover, it can also be used to support the design and construction of clinically relevant in vitro models of polymicrobial infections.

[Source](https://doi.org/10.1128/mbio.01456-24)

10.1128/mbio.01456-24

---

## Aerobic bacterial group as an early-stage biomarker from faecal samples of patients with colorectal cancer without distant metastasis

[Source](https://doi.org/10.1163/18762891-bja00051)

---

## The Parkinson's disease drug entacapone disrupts gut microbiome homoeostasis via iron sequestration.
 28.11.2024, _FC Pereira, X Ge, JM Kristensen, RH Kirkegaard, K Maritsch, D Szamosvári, S Imminger, D Seki, JB Shazzad, Y Zhu, M Decorte, B Hausmann, D Berry, K Wasmund, A Schintlmeister, T Böttcher, JX Cheng, M Wagner_


_Gastrointestinal Microbiome, Humans, Iron, Parkinson Disease, Feces, Catechols, Nitriles, Homeostasis, Bacteria, Metagenomics, Antiparkinson Agents_
Many human-targeted drugs alter the gut microbiome, leading to implications for host health. However, the mechanisms underlying these effects are not well known. Here we combined quantitative microbiome profiling, long-read metagenomics, stable isotope probing and single-cell chemical imaging to investigate the impact of two widely prescribed drugs on the gut microbiome. Physiologically relevant concentrations of entacapone, a treatment for Parkinson's disease, or loxapine succinate, used to treat schizophrenia, were incubated ex vivo with human faecal samples. Both drugs significantly impact microbial activity, more so than microbial abundance. Mechanistically, entacapone can complex and deplete available iron resulting in gut microbiome composition and function changes. Microbial growth can be rescued by replenishing levels of microbiota-accessible iron. Further, entacapone-induced iron starvation selected for iron-scavenging gut microbiome members encoding antimicrobial resistance and virulence genes. These findings reveal the impact of two under-investigated drugs on whole microbiomes and identify metal sequestration as a mechanism of drug-induced microbiome disturbance.

[Source](https://doi.org/10.1038/s41564-024-01853-0)

10.1038/s41564-024-01853-0

---

## Coffee consumption is associated with intestinal <em>Lawsonibacter asaccharolyticus</em> abundance and prevalence across multiple cohorts

[Source](https://doi.org/10.1038/s41564-024-01858-9)

---

## Enteric bacterial infection stimulates remodelling of bile metabolites to promote intestinal homeostasis.
 28.11.2024, _T Zhang, Y Hasegawa, MK Waldor_


_Animals, Mice, Homeostasis, Citrobacter rodentium, Enterobacteriaceae Infections, Metabolome, Gastrointestinal Microbiome, Bile, Liver, Listeria monocytogenes, Mice, Inbred C57BL, Intestines, Metabolomics, Listeriosis, Bile Acids and Salts, Specific Pathogen-Free Organisms_
The liver makes bile, an aqueous solution critical for fat absorption, which is secreted into the duodenum. Despite extensive studies on bile salts, other components of bile are less well characterized. Here we used global metabolomic analysis on bile from specific-pathogen-free, germ-free, Citrobacter rodentium-infected or Listeria monocytogenes-infected mice and identified a metabolome of 812 metabolites that were altered by both microbiota and enteric infection. Hepatic transcriptomics identified enteric-infection-triggered pathways that probably underlie bile remodelling. Enteric infection increased levels of four dicarboxylates in bile, including itaconate. Analysis of Acod1-/- mice indicated that increased itaconate also increased tuft cell abundance, altered microbiota composition and function as detected by metagenomic analysis, and modulated host defence, leading to reduced Vibrio cholerae colonization. Our data suggest that enteric-infection-associated signals are relayed between the intestine and liver and induce transcriptional programmes that shape the bile metabolome, modifying the immunomodulatory and host defence functions of bile.

[Source](https://doi.org/10.1038/s41564-024-01862-z)

10.1038/s41564-024-01862-z

---

## Microbiome-Based Colon Cancer Patient Stratification and Survival Analysis.
 23.11.2024, _J Smyth, J Godet, A Choudhary, A Das, GV Gkoutos, A Acharjee_


_Humans, Gastrointestinal Microbiome, Colonic Neoplasms, Male, Female, Middle Aged, Survival Analysis, Aged, Kaplan-Meier Estimate, Prognosis, Cluster Analysis_
Colorectal cancer (CRC) is any cancer that starts in the colon or the rectum and presents a significant health concern. It is the third most diagnosed and the second deadliest cancer, with an estimated 153,020 new cases and 52,550 deaths in 2023. The severity of colon cancer may be attributed to its ability to avoid the host immune system and growth suppressors, its asymptomatic nature in the early stages, its association with a continually ageing population and unfavourable diet and obesity. The composition of the gut microbiome plays an important role in the development of CRC and presents as an important target in early detection and in predicting treatment outcomes in CRC. This study aims to identify microbiome-specific derived clusters in CRC patients and conduct subsequent survival analysis using the specific microbiome features within clusters.

[Source](https://doi.org/10.1002/cam4.70434)

10.1002/cam4.70434

---

## Why HPV16? Why, now, HPV42? How the discovery of HPV42 in rare cancers provides an opportunity to challenge our understanding about the transition between health and disease for common members of the healthy microbiota.
 19.11.2024, _IG Bravo, S Belkhir, P Paget-Bailly_


In 2022, a bioinformatic, agnostic approach identified HPV42 as causative agent of a rare cancer, later confirmed experimentally. This unexpected association offers an opportunity to reconsider our understanding about papillomavirus infections and cancers. We have expanded our knowledge about the diversity of papillomaviruses and the diseases they cause.Yet we still lack answers to fundamental questions, such as what makes HPV16 different from the closely related HPV31 or HPV33; or why the very divergent HPV13 and HPV32 cause focal epithelial hyperplasia, while HPV6 or HPV42 do not, despite their evolutionary relatedness. Certain members of the healthy skin microbiota are associated to rare clinical conditions. We propose that a focus on cellular phenotypes, most often transient and influenced by intrinsic and extrinsic factors, may help understand the continuum between health and disease. A conceptual switch is required towards an interpretation of biology as a diversity of states connected by transition probabilities, rather than quasi-deterministic programs. Under this perspective, papillomaviruses may only trigger malignant transformation when specific viral genotypes interact with precise cellular states. Drawing on Canguilhem's concepts of normal and pathological, we suggest that understanding the transition between fluid cellular states can illuminate how commensal-like infections transition from benign to malignant.

[Source](https://doi.org/10.1093/femsre/fuae029)

10.1093/femsre/fuae029

---

## Acarbose impairs gut <em>Bacteroides</em> growth by targeting intracellular glucosidases

[Source](https://doi.org/10.1128/mbio.01506-24)

---

## More than just a number: the gut microbiota and brain function across the extremes of life.
 24.11.2024, _ND Nuzum, C Deady, S Kittel-Schneider, JF Cryan, SM O'Mahony, G Clarke_


_Gastrointestinal Microbiome, Humans, Brain, Animals, Brain-Gut Axis, Alzheimer Disease, Neurodegenerative Diseases_
Understanding the interrelationship between the gut microbiota and host physiology, although still in its relative infancy, has taken important steps forward over the past decade. In the context of brain disorders including those characterized by neurodevelopmental and neurodegenerative changes there have been important advances. However, initially research involved correlational analyses, had limited translational scope, and lacked functional assessments. Thus, largescale longitudinal clinical investigations that assess causation and underlying mechanisms via in depth analysis methods are needed. In neurodegeneration research, strong causal evidence now links the gut microbiome to Alzheimer's (AD), and Parkinson's Disease (PD), as supported by human-to-animal transplantation studies. Longitudinal interventions are being conducted in AD, PD, amyotrophic lateral sclerosis, Huntington's disease, and multiple sclerosis. Neurodevelopmental research has also seen a boon in microbiome-related clinical research including in autism, Attention-deficit/hyperactivity disorder, and schizophrenia, which is confirming prior animal model work regarding the key time-windows in the gut microbiome important for infant cognition. While recent research advances represent important progress, fundamental knowledge gaps and obstacles remain. Knowing how and why the gut microbiome changes at the extremes of life will develop our mechanistic understanding and help build the evidence base as we strive toward counteracting microbial missteps with precision therapeutic interventions.

[Source](https://doi.org/10.1080/19490976.2024.2418988)

10.1080/19490976.2024.2418988

---

## Microbiota-derived small molecule genotoxins: host interactions and ecological impact in the gut ecosystem.
 23.11.2024, _EL Zechner, S Kienesberger_


_Humans, Gastrointestinal Microbiome, Bacteria, Animals, Host Microbial Interactions, Mutagens, Polyketides, Enterotoxins, Peptides_
The human intestinal tract is densely colonized by a microbial community that is subject to intense competition. Bacteria in this complex habitat seek to outcompete their neighbors for nutrients and eliminate competitors with antibacterial toxins. Antagonism can be mediated by diverse effectors including toxic proteins and small molecule inhibitors that are released extracellularly or delivered by specialized secretion systems to targeted cells. Two prototypical microbiota-derived enterotoxins, colibactin and tilimycin, and the newly discovered family of indolimines represent an expanding group of non-proteinaceous small molecules which specifically target DNA. In addition to cell killing, they generate mutations and genome instability in intoxicated microbes and host cells alike. They have been studied in detail because of their direct toxicity to human cells and important etiological roles in intestinal pathologies. Increasing evidence, however, reveals that these commensal genotoxins are also mediators of interbacterial antagonism, which impacts gut microbial ecology. In this review, we illustrate the functional versatility of commensal genotoxins in the gut ecosystem.

[Source](https://doi.org/10.1080/19490976.2024.2430423)

10.1080/19490976.2024.2430423

---

## Citizen science as an instrument for women’s health research

[Source](https://doi.org/10.1038/s41591-024-03371-2)

---

## Normal Gut Microbiomes in Diverse Populations: Clinical Implications.
 18.11.2024, _O Shete, TS Ghosh_


The human microbiome is a sensor and modulator of physiology and homeostasis. Remarkable tractability underpins the promise of therapeutic manipulation of the microbiome. However, the definition of a normal or healthy microbiome has been elusive. This is in part due to the underrepresentation of minority groups and major global regions in microbiome studies to date. We review studies of the microbiome in different populations and highlight a commonality among health-associated microbiome signatures along with major drivers of variation. We also provide an overview of microbiome-associated therapeutic interventions for some widespread, widely studied diseases. We discuss sources of bias and the challenges associated with defining population-specific microbiome reference bases. We propose a roadmap for defining normal microbiome references that can be used for population-customized microbiome therapeutics and diagnostics.

[Source](https://doi.org/10.1146/annurev-med-051223-031809)

10.1146/annurev-med-051223-031809

---

## Synthetic Ecosystems: From the Test Tube to the Biosphere

[Source](https://doi.org/10.1021/acssynbio.4c00384)

---

## <em>Limosilactobacillus reuteri</em> ameliorates maternal separation stress in newborn mice and alters subsequent adult behaviour

[Source](https://doi.org/10.1163/18762891-bja00054)

---

## Sodium oligomannate disrupts the adherence of Ribhigh bacteria to gut epithelia to block SAA-triggered Th1 inflammation in 5XFAD transgenic mice.
 21.11.2024, _X Wang, Z Xie, J Yuan, E Jin, W Lian, S Chang, G Sun, Z Feng, H Xu, C Du, X Yang, A Xia, J Qiu, Q Zhang, F Lin, J Liu, L Li, X Du, Z Xiao, Z Yi, Z Luo, C Ge, R Li, M Zheng, Y Jiang, T Wang, J Zhang, Q Guo, M Geng_


Sodium oligomannate (GV-971), an oligosaccharide drug approved in China for treating mild-to-moderate Alzheimer's disease (AD), was previously found to recondition the gut microbiota and limit altered peripheral Th1 immunity in AD transgenic mice. As a follow-up study, we here made advances by pinpointing a Lactobacillus murinus (L.m.) strain that highly expressed a gene encoding a putative adhesin containing Rib repeats (Ribhigh-L.m.) particularly enriched in 5XFAD transgenic mice. Mechanistically, Ribhigh-L.m. adherence to the gut epithelia upregulated fecal metabolites, among which lactate ranked as the top candidate. Excess lactate stimulated the epithelial production of serum amyloid A (SAA) in the gut via the GPR81-NFκB axis, contributing to peripheral Th1 activation. Moreover, GV-971 disrupted the adherence of Ribhigh-L.m. to gut epithelia via direct binding to Rib, which corrected the excess lactate, reduced SAA, and alleviated Th1-skewed inflammation. Together, we gained further insights into the molecular link between gut bacteria and AD progression and the mechanism of GV-971 in treating AD.

[Source](https://doi.org/10.1038/s41421-024-00725-5)

10.1038/s41421-024-00725-5

---

## <em>B. vulgatus</em> ameliorates high-fat diet-induced obesity through modulating intestinal serotonin synthesis and lipid absorption in mice

[Source](https://doi.org/10.1080/19490976.2024.2423040)

---

## Multiomics of yaks reveals significant contribution of microbiome into host metabolism.
 24.11.2024, _S Yang, J Zheng, H Mao, P Vinitchaikul, D Wu, J Chai_


_Animals, Cattle, Rumen, Metagenomics, Bacteria, Gastrointestinal Microbiome, Metabolomics, Methane, Animal Feed, Metabolome, Microbiota, Multiomics_
An intensive feeding system might improve the production cycle of yaks. However, how intensive feeding system contributes to yak growth is unclear. Here, multi-omics, including rumen metagenomics, rumen and plasma metabolomics, were performed to classify the regulatory mechanisms of intensive feeding system on yaks. Increased growth performance were observed. Rumen metagenomics revealed that Clostridium, Methanobrevibacter, Piromyces and Anaeromyces increased in the intensively fed yaks, contributing to amino acid and carbohydrate metabolism. The grazing yaks had more cellulolytic microbes. These microbiomes were correlated with the pathways of "Alanine aspartate and glutamate metabolism" and "Pyruvate metabolism". Intensive feeding increased methane degradation functions, while grazing yaks had higher methyl metabolites associated with methane production. These rumen microbiomes and their metabolites resulted in changes in plasma metabolome, finally influencing yaks' growth. Thus, an intensive feeding system altered the rumen microbiome and metabolism as well as host metabolism, resulting in improvements of yak growth.

[Source](https://doi.org/10.1038/s41522-024-00609-2)

10.1038/s41522-024-00609-2

---

## Single-strain probiotics enhance growth, anti-pathogen immunity, and resistance to <em>Nocardia seriolae </em>in grey mullet (<em>Mugil cephalus</em>) via gut microbiota modulation

[Source](https://doi.org/10.1186/s42523-024-00353-0)

---

## The gut microbiota of three avian species living in sympatry.
 23.11.2024, _H Pereira, N Chakarov, BA Caspers, M Gilles, W Jones, T Mijoro, S Zefania, T Székely, O Krüger, JI Hoffman_


_Animals, Gastrointestinal Microbiome, Sympatry, Species Specificity, Madagascar_
Evolutionary divergence and genetic variation are often linked to differences in microbial community structure and diversity. While environmental factors and diet heavily influence gut microbial communities, host species contributions are harder to quantify. Closely related species living in sympatry provide a unique opportunity to investigate species differences without the confounding effects of habitat and dietary variation. We therefore compared and contrasted the gut microbiota of three sympatric plover species: the widespread Kittlitz's and white-fronted plovers (Anarhynchus pecuarius and A. marginatus) and the endemic and vulnerable Madagascar plover (A. thoracicus).

[Source](https://doi.org/10.1186/s12862-024-02329-9)

10.1186/s12862-024-02329-9

---

## Butyrate reduces epithelial barrier dysfunction induced by the foodborne mycotoxin deoxynivalenol in cell monolayers derived from pig jejunum organoids.
 27.11.2024, _J Alberge, E Mussard, C Al-Ayoubi, C Lencina, C Marrauld, L Cauquil, CS Achard, I Mateos, I Alassane-Kpembi, IP Oswald, L Soler, S Combes, M Beaumont_


_Animals, Trichothecenes, Swine, Jejunum, Organoids, Intestinal Mucosa, Butyrates, Gastrointestinal Microbiome, Tight Junctions, Epithelial Cells, Fusarium, Immunity, Innate, Mycotoxins, Goblet Cells_
The foodborne mycotoxin deoxynivalenol (DON) produced by Fusarium species threats animal and human health through disruption of the intestinal barrier. Targeting the gut microbiota and its products appears as a promising strategy to mitigate DON intestinal toxicity. In this study, we investigated whether the bacterial metabolite butyrate could alleviate epithelial barrier disruption induced by DON. We used a model of cell monolayers derived from porcine jejunum organoids allowing to reproduce the cellular complexity of the intestinal epithelium. Our results show that DON dose-dependently disrupted the epithelial barrier integrity, reduced epithelial differentiation, and altered innate immune defenses. Butyrate attenuated the DON-induced increase in paracellular permeability. Butyrate also prevented epithelial barrier dysfunction triggered by anisomycin, a ribosome inhibitor like DON. Moreover, butyrate partially counteracted the effects of DON on tight junctions (TJP1, OCLN), innate epithelial defenses (PTGS2, CD14, TLR4, TLR5), and absorptive cell functions (CA2, VIL1, NHE3, CFTR). In contrast, butyrate did not prevent the toxic effects of DON on mitochondrial metabolism, proliferation and goblet cell functions. Taken together, our results demonstrate that the bacterial metabolite butyrate is able to reduce DON-induced epithelial barrier disruption.

[Source](https://doi.org/10.1080/19490976.2024.2430424)

10.1080/19490976.2024.2430424

---

## Trophic modulation of endophytes by rhizosphere protists.
 20.11.2024, _R Asiloglu, SO Bodur, SO Samuel, M Aycan, J Murase, N Harada_


The plant-microbe interactions, which is crucial for plant health and productivity, mainly occur in rhizosphere: a narrow zone of soil surrounding roots of living plants. The rhizosphere hosts one of the most intense habitats for microbial prey-predator interactions, especially between predatory protists and bacteria. Here, based on two key facts, microbial predators modulate rhizobacterial community composition, and the rhizobacterial community is the primary source of root microbiome, endophytes; we hypothesized that predation upon rhizobacteria would modulate the community composition of endophytic bacteria. The effects of three taxonomically distinct axenic protist species (Acanthamoeba castellanii, Vermamoeba vermiformis, and Heteromita globosa) were tested in this study. To examine the robustness of the hypotheses, the experiments were conducted in three soil types characterized by distinct bacterial communities and physicochemical properties. The bacterial community compositions were analyzed with high throughput sequencing. Bacterial gene abundances were estimated with a real-time-PCR method. The results showed that protists modulated endophytic communities, which originated in the rhizosphere soil. The modulation of endophytic communities by protists showed chaotic patterns rather than a deterministic effect under different soil types. The observed chaotic dynamics were further confirmed with an additional experiment, in which chaos was triggered by changes in the dilution rates of soil nutrients. Furthermore, the presence of predators enhanced the root colonization of endophytes. Our findings identify a key mechanism for the modulation of root endophytes and enhance understanding of underground plant-microbe interactions, which can lead to open new avenues for modulating the root microbiome to enhance crop production.

[Source](https://doi.org/10.1093/ismejo/wrae235)

10.1093/ismejo/wrae235

---

## Metagenome-based diversity and functional analysis of culturable microbes in sugarcane.
 19.11.2024, _H Lin, L Wu, L Zhang, QK Ta, P Liu, J Song, X Yang_


Sugarcane is a key crop for sugar and energy production, and understanding the diversity of its associated microbes is crucial for optimizing its growth and health. However, there is a lack of thorough investigation and use of microbial resources in sugarcane. This study conducted a comprehensive analysis of culturable microbes and their functional features in different tissues and rhizosphere soil of four diverse sugarcane species using metagenomics techniques. The results revealed significant microbial diversity in sugarcane's tissues and rhizosphere soil, including several important biomarker bacterial taxa identified, which are reported to engage in several processes that support plant growth, such as nitrogen fixation, phosphate solubilization, and the production of plant hormones. The Linear discriminant analysis Effect Size (LEfSe) studies identified unique microbial communities in different parts of the same sugarcane species, particularly Burkholderia, which exhibited significant variations across the sugarcane species. Microbial analysis of carbohydrate-active enzymes (CAZymes) indicated that genes related to sucrose metabolism were mostly present in specific bacterial taxa, including Burkholderia, Pseudomonas, Paraburkholderia, and Chryseobacterium. This study improves understanding of the diversities and functions of endophytes and rhizosphere soil microbes in sugarcane. Moreover, the approaches and findings of this study provide valuable insights for microbiome research and the use of comparable technologies in other agricultural fields.

[Source](https://doi.org/10.1128/spectrum.01982-24)

10.1128/spectrum.01982-24

---

## Root hair developmental regulators orchestrate drought triggered microbiome changes and the interaction with beneficial Rhizobiaceae.
 23.11.2024, _Z Wang, Z Li, Y Zhang, J Liao, K Guan, J Zhai, P Meng, X Tang, T Dong, Y Song_


_Plant Roots, Droughts, Microbiota, Stress, Physiological, Rhizobium, Arabidopsis, Metagenome, Transcriptome, Symbiosis_
Drought is one of the most serious abiotic stresses, and emerging evidence suggest plant microbiome affects plant drought tolerance. However, there is a lack of genetic evidence regarding whether and how plants orchestrate the dynamic assembly of the microbiome upon drought. By utilizing mutants with enhanced or decreased root hair densities, we find that root hair regulators also affect drought induced root microbiome changes. Rhizobiaceae is a key biomarker taxa affected by root hair related mutants. We isolated and sequenced 1479 root associated microbes, and confirmed that several Rhizobium strains presented stress-alleviating activities. Metagenome, root transcriptome and root metabolome studies further reveal the multi-omic changes upon drought stress. We knocked out an ornithine cyclodeaminase (ocd) gene in Rhizobium sp. 4F10, which significantly dampens its stress alleviating ability. Our genetic and integrated multi-omics studies confirm the involvement of host genetic effects in reshaping a stress-alleviating root microbiome during drought, and provide mechanistic insights into Rhizobiaceae mediated abiotic stress protection.

[Source](https://doi.org/10.1038/s41467-024-54417-5)

10.1038/s41467-024-54417-5

---

## Functional diversity of the above-ground fungal community under long-term integrated, organic and biodynamic Vineyard Management.
 21.11.2024, _K Steng, F Roy, H Kellner, J Moll, S Tittmann, J Frotscher, J Döring_


Sustainable agriculture increasingly emphasizes the importance of microbial communities in influencing plant health and productivity. In viticulture, understanding the impact of management practices on fungal communities is critical, given their role in disease dynamics, grape and wine quality. This study investigates the effects of integrated, organic, and biodynamic management practices on the diversity and function of fungal communities in a vineyard located in Geisenheim, Germany, focusing on above-ground parts such as bark, leaves, and grapes.

[Source](https://doi.org/10.1186/s40793-024-00625-x)

10.1186/s40793-024-00625-x

---

## A functional microbiome catalogue crowdsourced from North American rivers.
 26.11.2024, _MA Borton, BB McGivern, KR Willi, BJ Woodcroft, AC Mosier, DM Singleton, T Bambakidis, A Pelly, RA Daly, F Liu, A Freiburger, JN Edirisinghe, JP Faria, R Danczak, I Leleiwi, AE Goldman, MJ Wilkins, EK Hall, C Pennacchio, S Roux, EA Eloe-Fadrosh, SP Good, MB Sullivan, EM Wood-Charlson, CS Miller, MRV Ross, CS Henry, BC Crump, JC Stegen, KC Wrighton_


Predicting elemental cycles and maintaining water quality under increasing anthropogenic influence requires knowledge of the spatial drivers of river microbiomes. However, understanding of the core microbial processes governing river biogeochemistry is hindered by a lack of genome-resolved functional insights and sampling across multiple rivers. Here we used a community science effort to accelerate the sampling, sequencing and genome-resolved analyses of river microbiomes to create the Genome Resolved Open Watersheds database (GROWdb). GROWdb profiles the identity, distribution, function and expression of microbial genomes across river surface waters covering 90% of United States watersheds. Specifically, GROWdb encompasses microbial lineages from 27 phyla, including novel members from 10 families and 128 genera, and defines the core river microbiome at the genome level. GROWdb analyses coupled to extensive geospatial information reveals local and regional drivers of microbial community structuring, while also presenting foundational hypotheses about ecosystem function. Building on the previously conceived River Continuum Concept1, we layer on microbial functional trait expression, which suggests that the structure and function of river microbiomes is predictable. We make GROWdb available through various collaborative cyberinfrastructures2,3, so that it can be widely accessed across disciplines for watershed predictive modelling and microbiome-based management practices.

[Source](https://doi.org/10.1038/s41586-024-08240-z)

10.1038/s41586-024-08240-z

---

## Experimental evolution at ecological scales allows linking of viral genotypes to specific host strains.
 23.11.2024, _MD Ramos-Barbero, B Aldeguer-Riquelme, T Viver, J Villamor, M Carrillo-Bautista, C López-Pascual, KT Konstantinidis, M Martínez-García, F Santos, R Rossello-Mora, J Antón_


Viruses shape microbial community structure and activity through the control of population diversity and cell abundances. Identifying and monitoring the dynamics of specific virus-host pairs in nature is hampered by the limitations of culture-independent approaches such as metagenomics, which do not always provide strain-level resolution, and culture-based analyses, which eliminate the ecological background and in-situ interactions. Here, we have explored the interaction of a specific "autochthonous" host strain and its viruses within a natural community. Bacterium Salinibacter ruber strain M8 was spiked into its environment of isolation, a crystallizer pond from a coastal saltern, and the viral and cellular communities were monitored for one month using culture, metagenomics, and microscopy. Metagenome sequencing indicated that the M8 abundance decreased sharply after being added to the pond, likely due to forces other than viral predation. However, the presence of M8 selected for two species of a new viral genus, Phoenicisalinivirus, for which 120 strains were isolated. During this experiment, an assemblage of closely related viral genomic variants was replaced by a single population with the ability to infect M8, a scenario which was compatible with the selection of a genomic variant from the rare biosphere. Further analysis implicated a viral genomic region putatively coding for a tail fiber protein to be responsible for M8 specificity. Our results indicate that low abundance viral genotypes provide a viral seed bank that allows for a highly specialized virus-host response within a complex ecological background.

[Source](https://doi.org/10.1093/ismejo/wrae208)

10.1093/ismejo/wrae208

---

## Selenium mitigated cadmium-induced ovarian retardation in female Procambarus clarkii by regulating vitellogenin synthesis and transfer in the hepatopancreas.
 15.11.2024, _H Yang, Y Yang, A Mo, Y Yuan_


Cadmium (Cd) is prevalent in aquatic ecosystems and accumulates in various tissues of aquatic organisms, leading to severe biological toxicity. Selenium (Se) is recognized for mitigating heavy metal toxicity, though its protective effects against Cd in aquatic crustaceans remain underexplored. This study, therefore, assessed the effects of dietary Cd (15 mg/kg) exposure and Se (6 mg/kg) supplementation on the hepatopancreas and ovaries of female crayfish to uncover the mechanisms of Cd toxicity and the protective role of Se. The results showed that Cd accumulation in the hepatopancreas caused a reduced hepatopancreas index (HPI), decreased protein content, histopathological damage, and oxidative stress, while Se supplementation reduced Cd levels, mitigated damage, and restored tissue integrity and antioxidant defenses. Transcriptomic analysis further revealed significant alterations in gene expression related to detoxification, lipid metabolism, and energy production in response to Cd exposure, which were partially or fully restored by Se supplementation. Additionally, Se alleviated Cd-induced inhibition of ovarian development, as evidenced by improved ovary index, enhanced oocyte development, and normalization of essential trace element levels. Mechanistically, Se restored the Cd-disrupted vitellogenin (Vtg) synthesis in the hepatopancreas via regulating the mRNA expression of hsp70 and genes related to the molt-inhibiting hormone (MIH) (mih, rxr, and ecr). Overall, these findings indicate that Se supplementation mitigated Cd-induced hepatopancreatic dysfunction, restored Vtg synthesis, and consequently counteracted the inhibition of ovarian development in adult female crayfish.

[Source](https://doi.org/10.1186/s40793-024-00641-x)

10.1016/j.ecoenv.2024.117339

---

## Selenium alters the gene content but not the taxonomic composition of the soil microbiome

[Source](https://doi.org/10.1186/s40793-024-00641-x)

---

## Evaluation of microbial community dynamics and chlorinated solvent biodegradation in methane-amended microcosms from an acidic aquifer.
 20.11.2024, _M Hwangbo, R Rezes, KH Chu, PB Hatzinger_


_Biodegradation, Environmental, Groundwater, Methane, Water Pollutants, Chemical, Trichloroethylene, Bacteria, Microbiota, Volatile Organic Compounds, Solvents_
Anaerobic bioremediation is rarely an effective strategy to treat chlorinated ethenes such as trichloroethene (TCE) in acidic aquifers because partial dechlorination typically results in accumulation of daughter products. Methanotrophs have the capability of oxidizing TCE and other chlorinated volatile organic compounds (CVOCs) to non-toxic products, but their occurrence, diversity, and biodegradation capabilities in acidic environments are largely unknown. This study investigated the impacts of different methane (CH4) concentrations and the presence of CVOCs on the community of acidophilic methanotrophs in microcosms prepared from acidic aquifer samples collected upgradient and downgradient of a mulch barrier installed to promote in-situ anaerobic CVOC biodegradation in Maryland, USA. The ability of indigenous methanotrophs to biodegrade CVOCs was also evaluated. Results of stable isotope probing (SIP) and Next Generation Sequencing (NGS) showed that the microbial communities in the microcosms varied by location and were affected by both CH4 concentration and the presence of different CVOCs, many of which were biodegraded by the indigenous methanotrophs. Data indicate the likelihood of aerobic cometabolic degradation of CVOCs downgradient of the mulch barrier designed for anaerobic treatment. The study extends the overall knowledge of acidophilic methanotrophs in groundwater and shows that these bacteria have significant potential for degrading CVOCs even at low CH4 concentrations.

[Source](https://doi.org/10.1007/s10532-024-10103-3)

10.1007/s10532-024-10103-3

---

## Rotavirus rewires host cell metabolic pathways toward glutamine catabolism for effective virus infection.
 24.11.2024, _S Mitra, R Datta Chaudhuri, R Sarkar, S Banerjee, A Mukherjee, R Sharma, A Gope, K Kitahara, SI Miyoshi, M Chawla-Sarkar_


_Glutamine, Humans, Rotavirus, Rotavirus Infections, Metabolic Networks and Pathways, Animals, Virus Replication, Mice, HT29 Cells, Host-Pathogen Interactions, Gastrointestinal Microbiome, Epithelial Cells_
Rotavirus (RV) accounts for 19.11% of global diarrheal deaths. Though GAVI assisted vaccine introduction has curtailed RV induced mortality, factors like RV strain diversity, differential infantile gut microbiome, malnutrition, interference from maternal antibodies and other administered vaccines, etc. often compromise vaccine efficacy. Herein emerges the need of antivirals which can be administered adjunct to vaccination to curb the socio-economic burden stemming from frequent RV infection. Cognisance of pathogen-perturbed host cellular physiology has revolutionized translational research and aided precision-based therapy, particularly for viruses, with no metabolic machinery of their own. To date there has been limited exploration of the host cellular metabolome in context of RV infection. In this study, we explored the endometabolomic landscape of human intestinal epithelial cells (HT-29) on RV-SA11 infection. Significant alteration of host cellular metabolic pathways like the nucleotide biosynthesis pathway, alanine, aspartate and glutamate metabolism pathway, the host citric acid cycle was observed in RV-SA11 infection scenario. Detailed study further revealed that RV replication is exclusively dependent on glutamine metabolism for their propagation in host cells. Glutamine metabolism generates glutamate, aspartate, and asparagine which facilitates virus infection. Abrogation of aspartate biogenesis from glutamine by use of Aminooxyacetic acid (AOAA), significantly curbed RV-SA11 infection in-vitro and in-vivo. Overall, the study improves our understanding of host-rotavirus interactome and recognizes host glutamine metabolism pathway as a suitable target for effective therapeutic intervention against RV infection.

[Source](https://doi.org/10.1080/19490976.2024.2428425)

10.1080/19490976.2024.2428425

---

## SPARTA: Interpretable functional classification of microbiomes and detection of hidden cumulative effects.
 18.11.2024, _B Ruiz, A Belcour, S Blanquart, S Buffet-Bataillon, I Le Huërou-Luron, A Siegel, Y Le Cunff_


The composition of the gut microbiota is a known factor in various diseases and has proven to be a strong basis for automatic classification of disease state. A need for a better understanding of microbiota data on the functional scale has since been voiced, as it would enhance these approaches' biological interpretability. In this paper, we have developed a computational pipeline for integrating the functional annotation of the gut microbiota into an automatic classification process and facilitating downstream interpretation of its results. The process takes as input taxonomic composition data, which can be built from 16S or whole genome sequencing, and links each component to its functional annotations through interrogation of the UniProt database. A functional profile of the gut microbiota is built from this basis. Both profiles, microbial and functional, are used to train Random Forest classifiers to discern unhealthy from control samples. SPARTA ensures full reproducibility and exploration of inherent variability by extending state-of-the-art methods in three dimensions: increased number of trained random forests, selection of important variables with an iterative process, repetition of full selection process from different seeds. This process shows that the translation of the microbiota into functional profiles gives non-significantly different performances when compared to microbial profiles on 5 of 6 datasets. This approach's main contribution however stems from its interpretability rather than its performance: through repetition, it also outputs a robust subset of discriminant variables. These selections were shown to be more consistent than those obtained by a state-of-the-art method, and their contents were validated through a manual bibliographic research. The interconnections between selected taxa and functional annotations were also analyzed and revealed that important annotations emerge from the cumulated influence of non-selected taxa.

[Source](https://doi.org/10.1371/journal.pcbi.1012577)

10.1371/journal.pcbi.1012577

---

## Micro-DeMix: a mixture beta-multinomial model for investigating the heterogeneity of the stool microbiome compositions.
 26.11.2024, _R Liu, Y Wang, D Cheng_


Extensive research has uncovered the critical role of the human gut microbiome in various aspects of health, including metabolism, nutrition, physiology, and immune function. Fecal microbiota is often used as a proxy for understanding the gut microbiome, but it represents an aggregate view, overlooking spatial variations across different gastrointestinal (GI) locations. Emerging studies with spatial microbiome data collected from specific GI regions offer a unique opportunity to better understand the spatial composition of the stool microbiome.

[Source](https://doi.org/10.1093/bioinformatics/btae667)

10.1093/bioinformatics/btae667

---

