## It's the Economy, Stupid: Applying (Micro)economic Principles to Microbiome Science.
 11.01.2022, _N Kashtan, B Bushong, JHJ Leveau_


A key challenge in microbiome science is the scale mismatch problem, which arises when the scale at which microbial communities are sampled, interrogated, and averaged is different from the scale at which individual microorganisms within those communities interact with each other and with their environment. Profiling the microbial communities in a teaspoon of soil, from a scoop of fecal matter, or along a plant leaf surface represents a scale mismatch of multiple orders of magnitude, which may limit our ability to interpret or predict species interactions and community assembly within such samples. In this Perspective, we explore how economists, who are historically and topically split along the lines of micro- and macroeconomics, deal with the scale mismatch problem, and how taking clues from (micro)economists could benefit the field of microbiomics.

[Source](https://journals.asm.org/doi/10.1128/msystems.01033-21)

10.1128/msystems.01033-21

---

## Conceptual Exchanges for Understanding Free-Living and Host-Associated Microbiomes.
 11.01.2022, _CA Pfister, SH Light, B Bohannan, T Schmidt, A Martiny, NA Hynson, S Devkota, L David, K Whiteson_


Whether a microbe is free-living or associated with a host from across the tree of life, its existence depends on a limited number of elements and electron donors and acceptors. Yet divergent approaches have been used by investigators from different fields. The "environment first" research tradition emphasizes thermodynamics and biogeochemical principles, including the quantification of redox environments and elemental stoichiometry to identify transformations and thus an underlying microbe. The increasingly common "microbe first" research approach benefits from culturing and/or DNA sequencing methods to first identify a microbe and encoded metabolic functions. Here, the microbe itself serves as an indicator for environmental conditions and transformations. We illustrate the application of both approaches to the study of microbiomes and emphasize how both can reveal the selection of microbial metabolisms across diverse environments, anticipate alterations to microbiomes in host health, and understand the implications of a changing climate for microbial function.

[Source](https://journals.asm.org/doi/10.1128/msystems.01374-21)

10.1128/msystems.01374-21

---

## Endometrial microbiota composition is associated with reproductive outcome in infertile patients.
 07.01.2022, _I Moreno, I Garcia-Grau, D Perez-Villaroya, M Gonzalez-Monfort, M Bahçeci, MJ Barrionuevo, S Taguchi, E Puente, M Dimattina, MW Lim, G Meneghini, M Aubuchon, M Leondires, A Izquierdo, M Perez-Olgiati, A Chavez, K Seethram, D Bau, C Gomez, D Valbuena, F Vilella, C Simon_


Previous evidence indicates associations between the female reproductive tract microbiome composition and reproductive outcome in infertile patients undergoing assisted reproduction. We aimed to determine whether the endometrial microbiota composition is associated with reproductive outcomes of live birth, biochemical pregnancy, clinical miscarriage or no pregnancy.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01184-w)

10.1186/s40168-021-01184-w

---

## Temporal development of the infant oral microbiome.
 11.01.2022, _J Blum, M Silva, SJ Byrne, CA Butler, GG Adams, EC Reynolds, SG Dashper_


The human oral microbiome is becoming recognized as playing roles in health and disease well beyond the oral cavity over the lifetime of the individual. The oral microbiome is hypothesized to result from specific colonization events followed by a reproducible and ordered development of complex bacterial communities. Colonization events, proliferation, succession and subsequent community development are dependent on a range of host and environmental factors, most notably the neonate diet. It is now becoming apparent that early childhood and prenatal influences can have long term effects on the development of human oral microbiomes. In this review, the temporal development of the infant human oral microbiome is examined, with the effects of prenatal and postnatal influences and the roles of specific bacteria. Dietary and environmental factors, especially breastfeeding, have a significant influence on the development of the infant oral microbiome. The evidence available regarding the roles and functions of early colonizing bacteria is still limited, and gaps in knowledge where further research is needed to elucidate these specific roles in relation to health and disease still exist.

[Source](https://www.tandfonline.com/doi/abs/10.1080/1040841X.2021.2025042?journalCode=imby20)

10.1080/1040841X.2021.2025042

---

## Effect of antibiotic pretreatment on bacterial engraftment after Fecal Microbiota Transplant (FMT) in IBS-D.
 11.01.2022, _P Singh, EJ Alm, JM Kelley, V Cheng, M Smith, Z Kassam, J Nee, J Iturrino, A Lembo_


Fecal microbiota transplantation (FMT) is an attractive strategy to correct microbial dysbiosis in diarrhea-predominant irritable bowel syndrome (IBS-D). Although the mechanism of FMT is thought to be bacterial engraftment, the best approach to achieve engraftment after FMT in IBS-D (and other diseases) is not clear. We evaluated the effect of FMT (with or without pretreatment with antibiotics) on gut microbiome and symptoms in patients with IBS-D. In this randomized, placebo-controlled, single-center study, 44 patients with IBS-D with a least moderate severity (IBS severity scoring system, i.e., IBS-SSS, ≥175) were randomly assigned to one of four groups: single-dose oral FMT alone, single-dose oral FMT following a 7-day pretreatment course of Ciprofloxacin and Metronidazole (CM-FMT) or Rifaximin (R-FMT), or Placebo FMT. Primary endpoint was engraftment post-FMT and secondary endpoints were changes in IBS-SSS, and IBS-quality of life (IBS-QOL) at week 10. Median engraftment was significantly different among the three FMT groups (P = .013). Engraftment post-FMT was significantly higher in the FMT alone arm (15.5%) compared to that in R-FMT group (5%, P = .04) and CM-FMT group (2.4%, P = .002). The mean change in IBS-SSS and IBS-QOL from baseline were not significantly different among the four groups or between the three FMT groups combined vs. placebo at week 10. In summary, antibiotic pretreatment significantly reduced bacterial engraftment after FMT in patients with IBS-D.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2021.2020067)

10.1080/19490976.2021.2020067

---

## The gut microbiota can orchestrate the signaling pathways in colorectal cancer.
 10.01.2022, _ALB Bennedsen, S Furbo, T Bjarnsholt, H Raskov, I Gögenur, L Kvich_


Current evidence suggests that bacteria contribute to the development of certain cancers, such as colorectal cancer (CRC), partly by stimulating chronic inflammation. However, little is known about the bacterial impact on molecular pathways in CRC. Recent studies have demonstrated how specific bacteria can influence the major CRC-related pathways, i.e., Wnt, PI3K-Akt, MAPK, TGF-β, EGFR, mTOR, and p53. In order to advance the current understanding and facilitate the choice of pathways to investigate, we have systematically collected and summarized the current knowledge within bacterial-altered major pathways in CRC. Several pro-tumorigenic and anti-tumorigenic bacterial species and their respective metabolites interfere with the major signaling pathways addressed in this review. Not surprisingly, some of these studies investigated known CRC drivers, such as Escherichia coli, Fusobacterium nucleatum, and Bacteroides fragilis. Interestingly, some metabolites produced by bacterial species typically considered pathogenic, ex Vibrio cholera, displayed anti-tumorigenic activities, emphasizing the caution needed when classifying healthy and unhealthy microorganisms. The results collectively emphasize the complexity of the relationship between the microbiota and the tumorigenesis of CRC, and future studies should verify these findings in more realistic models, such as organoids, which constitute a promising platform. Moreover, future trials should investigate the clinical potential of preventive modulation of the gut microbiota regarding CRC development.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/apm.13206)

10.1111/apm.13206

---

## Age-Associated Gut Dysbiosis, Marked by Loss of Butyrogenic Potential, Correlates With Altered Plasma Tryptophan Metabolites in Older People Living With HIV.
 11.01.2022, _S Ghare, R Singhal, V Bryant, S Gautam, CC Tirumala, PK Srisailam, A Reyes-Vega, D Ghooray, CJ McClain, K Hoffman, J Petrosino, K Bryant, V Govind, R Cohen, RL Cook, S Barve_


Imbalance in tryptophan (TRP) metabolism and its neuroactive metabolites, serotonin and kynurenine (KYN), is a known pathogenic mechanism underlying neurocognitive impairment. Gut microbiota plays an important role in TRP metabolism, and the production of these neuroactive molecules affects neurocognitive function. Although both HIV infection and normal aging independently induce gut dysbiosis and influence TRP metabolism, their interactive effects on compositional/functional changes in gut microbiota and consequent alterations in TRP metabolites remain largely undetermined.

[Source](https://journals.lww.com/jaids/Fulltext/2022/02011/Age_Associated_Gut_Dysbiosis,_Marked_by_Loss_of.9.aspx)

10.1097/QAI.0000000000002866

---

## Intestinal Microbiota and Host Cooperate for Adaptation as a Hologenome.
 11.01.2022, _H Zhou, L Yang, J Ding, R Dai, C He, K Xu, L Luo, L Xiao, Y Zheng, C Han, FT Akinyemi, CF Honaker, Y Zhang, PB Siegel, H Meng_


Multiomic analyses reported here involved two lines of chickens, from a common founder population, that had undergone long-term selection for high (HWS) or low (LWS) 56-day body weight. In these lines that differ by around 15-fold in body weight, we observed different compositions of intestinal microbiota in the holobionts and variation in DNA methylation, mRNA expression, and microRNA profiles in the ceca. Insulin-like growth factor 2 mRNA-binding protein 1 (IGF2BP1) was the most upregulated gene in HWS ceca with its expression likely affected by the upregulation of expression of gga-miR-2128 and a methylated region near its transcription start site (388 bp). Correlation analysis showed that IGF2BP1 expression was associated with an abundance of microbes, such as Lactobacillus and Methanocorpusculum. These findings suggest that IGF2BP1 was regulated in the hologenome in adapting to long-term artificial selection for body weight. Our study provides evidence that adaptation of the holobiont can occur in the microbiome as well as in the epigenetic profile of the host. IMPORTANCE The hologenome concept has broadened our perspectives for studying host-microbe coevolution. The multiomic analyses reported here involved two lines of chickens, from a common founder population, that had undergone long-term selection for high (HWS) or low (LWS) 56-day body weight. In these lines that differ by around 15-fold in body weight, we observed different compositions of intestinal microbiota in the holobionts, and variation in DNA methylation, mRNA expression, and microRNA profiles in ceca. The insulin-like growth factor 2 mRNA-binding protein 1 (IGF2BP1) was the most upregulated gene in HWS ceca with its expression likely affected by a methylated region near its transcription start site and the upregulation of expression of gga-miR-2128. Correlation analysis also showed that IGF2BP1 expression was associated with the abundance of microbes, such as Lactobacillus and Methanocorpusculum. These findings suggest that IGF2BP1 was regulated in the hologenome in response to long-term artificial selection for body weight. Our study shows that the holobiont may adapt in both the microbiome and the host's epigenetic profile.

[Source](https://journals.asm.org/doi/10.1128/msystems.01261-21)

10.1128/msystems.01261-21

---

## The bacterial density of clinical rectal swabs is highly variable, correlates with sequencing contamination, and predicts patient risk of extraintestinal infection.
 10.01.2022, _R Chanderraj, CA Brown, K Hinkle, N Falkowski, RJ Woods, RP Dickson_


In ecology, population density is a key feature of community analysis. Yet in studies of the gut microbiome, bacterial density is rarely reported. Studies of hospitalized patients commonly use rectal swabs for microbiome analysis, yet variation in their bacterial density-and the clinical and methodologic significance of this variation-remains undetermined. We used an ultra-sensitive quantification approach-droplet digital PCR (ddPCR)-to quantify bacterial density in rectal swabs from 118 hospitalized patients. We compared bacterial density with bacterial community composition (via 16S rRNA amplicon sequencing) and clinical data to determine if variation in bacterial density has methodological, clinical, and prognostic significance.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01190-y)

10.1186/s40168-021-01190-y

---

## Consistent changes in the intestinal microbiota of Atlantic salmon fed insect meal diets.
 11.01.2022, _Y Li, K Gajardo, A Jaramillo-Torres, TM Kortner, Å Krogdahl_


Being part of fish's natural diets, insects have become a practical alternative feed ingredient for aquaculture. While nutritional values of insects have been extensively studied in various fish species, their impact on the fish microbiota remains to be fully explored. In an 8-week freshwater feeding trial, Atlantic salmon (Salmo salar) were fed either a commercially relevant reference diet or an insect meal diet wherein black soldier fly (Hermetia illucens) larvae meal comprised 60% of total ingredients. Microbiota of digesta and mucosa origin from the proximal and distal intestine were collected and profiled along with feed and water samples.

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-021-00159-4)

10.1186/s42523-021-00159-4

---

## Maternal regulation of biliary disease in neonates via gut microbial metabolites.
 11.01.2022, _JJ Jee, L Yang, P Shivakumar, PP Xu, R Mourya, U Thanekar, P Yu, Y Zhu, Y Pan, H Wang, X Duan, Y Ye, B Wang, Z Jin, Y Liu, Z Cao, M Watanabe-Chailland, LE Romick-Rosendale, M Wagner, L Fei, Z Luo, NJ Ollberding, ST Tang, JA Bezerra_


Maternal seeding of the microbiome in neonates promotes a long-lasting biological footprint, but how it impacts disease susceptibility in early life remains unknown. We hypothesized that feeding butyrate to pregnant mice influences the newborn's susceptibility to biliary atresia, a severe cholangiopathy of neonates. Here, we show that butyrate administration to mothers renders newborn mice resistant to inflammation and injury of bile ducts and improves survival. The prevention of hepatic immune cell activation and survival trait is linked to fecal signatures of Bacteroidetes and Clostridia and increases glutamate/glutamine and hypoxanthine in stool metabolites of newborn mice. In human neonates with biliary atresia, the fecal microbiome signature of these bacteria is under-represented, with suppression of glutamate/glutamine and increased hypoxanthine pathways. The direct administration of butyrate or glutamine to newborn mice attenuates the disease phenotype, but only glutamine renders bile duct epithelial cells resistant to cytotoxicity by natural killer cells. Thus, maternal intake of butyrate influences the fecal microbial population and metabolites in newborn mice and the phenotypic expression of experimental biliary atresia, with glutamine promoting survival of bile duct epithelial cells.

[Source](https://www.nature.com/articles/s41467-021-27689-4)

10.1038/s41467-021-27689-4

---

## Reduced calorie diet combined with NNMT inhibition establishes a distinct microbiome in DIO mice.
 11.01.2022, _A Dimet-Wiley, Q Wu, JT Wiley, A Eswar, H Neelakantan, T Savidge, S Watowich_


Treatment with a nicotinamide N-methyltransferase inhibitor (NNMTi; 5-amino-1-methylquinolinium) combined with low-fat diet (LD) promoted dramatic whole-body adiposity and weight loss in diet-induced obese (DIO) mice, rapidly normalizing these measures to age-matched lean animals, while LD switch alone was unable to restore these measures to age-matched controls in the same time frame. Since mouse microbiome profiles often highly correlate with body weight and fat composition, this study was designed to test whether the cecal microbiomes of DIO mice treated with NNMTi and LD were comparable to the microbiomes of age-matched lean counterparts and distinct from microbiomes of DIO mice maintained on a high-fat Western diet (WD) or subjected to LD switch alone. There were minimal microbiome differences between lean and obese controls, suggesting that diet composition and adiposity had limited effects. However, DIO mice switched from an obesity-promoting WD to an LD (regardless of treatment status) displayed several genera and phyla differences compared to obese and lean controls. While alpha diversity measures did not significantly differ between groups, beta diversity principal coordinates analyses suggested that mice from the same treatment group were the most similar. K-means clustering analysis of amplicon sequence variants by animal demonstrated that NNMTi-treated DIO mice switched to LD had a distinct microbiome pattern that was highlighted by decreased Erysipelatoclostridium and increased Lactobacillus relative abundances compared to vehicle counterparts; these genera are tied to body weight and metabolic regulation. Additionally, Parasutterella relative abundance, which was increased in both the vehicle- and NNMTi-treated LD-switched groups relative to the controls, significantly correlated with several adipose tissue metabolites' abundances. Collectively, these results provide a novel foundation for future investigations.

[Source](https://www.nature.com/articles/s41598-021-03670-5)

10.1038/s41598-021-03670-5

---

## Comparison of Gut Microbiota and Metabolic Status of Sows With Different Litter Sizes During Pregnancy.
 11.01.2022, _J Chen, F Li, W Yang, S Jiang, Y Li_


The experiment was conducted to compare the differences of gut microbiota and metabolic status of sows with different litter sizes on days 30 and 110 of gestation, and uncover the relationship between the composition of maternal gut microbiota during gestation and sow reproductive performance. Twenty-six Large White × Landrace crossbred multiparous sows (2nd parity) with similar back fat thickness and body weight were assigned to two groups [high-reproductive performance group (HP group) and low-reproductive performance group (LP group)] according to their litter sizes and fed a common gestation diet. Results showed that compared with LP sows, HP sows had significantly lower plasma levels of triglyceride (TG) on gestation d 30 (P &lt; 0.05), but had significantly higher plasma levels of TG, non-esterified fatty acid, tumor necrosis factor-α, and immunoglobulin M on gestation d 110 (P &lt; 0.05). Consistently, HP sows revealed increased alpha diversity and butyrate-producing genera, as well as fecal butyrate concentration, on gestation d 30; HP sows showed significantly different microbiota community structure with LP sows (P &lt; 0.05) and had markedly higher abundance of Firmicutes (genera Christensenellaceae_R-7_group and Terrisporobacter) which were positively related with litter size on gestation d 110 than LP sows (P &lt; 0.05). In addition, plasma biochemical parameters, plasma cytokines, and fecal microbiota shifted dramatically from gestation d 30 to d 110. Therefore, our findings demonstrated that microbial abundances and community structures differed significantly between sows with different litter sizes and gestation stages, which was associated with changes in plasma biochemical parameters, inflammatory factors, and immunoglobulin. Moreover, these findings revealed that there was a significant correlation between litter size and gut microbiota of sows, and provided a microbial perspective to improve sow reproductive performance in pig production.

[Source](https://www.frontiersin.org/articles/10.3389/fvets.2021.793174/full)

10.3389/fvets.2021.793174

---

## Modulation of gut microbiota impacts diet-induced and drug-induced alopecia in mice.
 06.01.2022, _S Lam, J Zhang, K Yang, LC Chu, W Zhu, W Tang, FKL Chan, PKS Chan, WKK Wu, SC Ng_



[Source](https://gut.bmj.com/content/early/2022/01/05/gutjnl-2021-326320)

10.1136/gutjnl-2021-326320

---

## Sex-specific differences in the gut microbiota and fecal metabolites in an adolescent valproic acid-induced rat autism model

[Source](https://imrpress.com/journal/FBL/26/12/10.52586/5051)

---

## Host tp53 mutation induces gut dysbiosis eliciting inflammation through disturbed sialic acid metabolism.
 10.01.2022, _JG Lee, S Lee, J Jeon, HG Kong, HJ Cho, JH Kim, SY Kim, MJ Oh, D Lee, N Seo, KH Park, K Yu, HJ An, CM Ryu, JS Lee_


Host tp53 mutations are frequently found during the early stages of colitis-associated colorectal cancer (CAC), but whether such mutations induce gut microbiota dysbiosis and chronic intestinal inflammation that contributes to the development of CAC, remains unknown.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01191-x)

10.1186/s40168-021-01191-x

---

## Whole-Genome Duplication and Host Genotype Affect Rhizosphere Microbial Communities.
 11.01.2022, _JCB Ponsford, CJ Hubbard, JG Harrison, L Maignien, CA Buerkle, C Weinig_


The composition of microbial communities found in association with plants is influenced by host phenotype and genotype. However, the ways in which specific genetic architectures of host plants shape microbiomes are unknown. Genome duplication events are common in the evolutionary history of plants and influence many important plant traits, and thus, they may affect associated microbial communities. Using experimentally induced whole-genome duplication (WGD), we tested the effect of WGD on rhizosphere bacterial communities in Arabidopsis thaliana. We performed 16S rRNA amplicon sequencing to characterize differences between microbiomes associated with specific host genetic backgrounds (Columbia versus Landsberg) and ploidy levels (diploid versus tetraploid). We modeled relative abundances of bacterial taxa using a hierarchical Bayesian approach. We found that host genetic background and ploidy level affected rhizosphere community composition. We then tested to what extent microbiomes derived from a specific genetic background or ploidy level affected plant performance by inoculating sterile seedlings with microbial communities harvested from a prior generation. We found a negative effect of the tetraploid Columbia microbiome on growth of all four plant genetic backgrounds. These findings suggest an interplay between host genetic background and ploidy level and bacterial community assembly with potential ramifications for host fitness. Given the prevalence of ploidy-level variation in both wild and managed plant populations, the effects on microbiomes of this aspect of host genetic architecture could be a widespread driver of differences in plant microbiomes. IMPORTANCE Plants influence the composition of their associated microbial communities, yet the underlying host-associated genetic determinants are typically unknown. Genome duplication events are common in the evolutionary history of plants and affect many plant traits. Using Arabidopsis thaliana, we characterized how whole-genome duplication affected the composition of rhizosphere bacterial communities and how bacterial communities associated with two host plant genetic backgrounds and ploidy levels affected subsequent plant growth. We observed an interaction between ploidy level and genetic background that affected both bacterial community composition and function. This research reveals how genome duplication, a widespread genetic feature of both wild and crop plant species, influences bacterial assemblages and affects plant growth.

[Source](https://journals.asm.org/doi/10.1128/msystems.00973-21)

10.1128/msystems.00973-21

---

## A Goldilocks Principle for the Gut Microbiome: Taxonomic Resolution Matters for Microbiome-Based Classification of Colorectal Cancer.
 11.01.2022, _CR Armour, BD Topçuoğlu, A Garretto, PD Schloss_


Colorectal cancer is a common and deadly disease in the United States accounting for over 50,000 deaths in 2020. This progressive disease is highly preventable with early detection and treatment, but many people do not comply with the recommended screening guidelines. The gut microbiome has emerged as a promising target for noninvasive detection of colorectal cancer. Most microbiome-based classification efforts utilize taxonomic abundance data from operational taxonomic units (OTUs) or amplicon sequence variants (ASVs) with the goal of increasing taxonomic resolution. However, it is unknown which taxonomic resolution is optimal for microbiome-based classification of colorectal cancer. To address this question, we used a reproducible machine learning framework to quantify classification performance of models based on data annotated to phylum, class, order, family, genus, OTU, and ASV levels. We found that model performance increased with increasing taxonomic resolution, up to the family level where performance was equal (P &gt; 0.05) among family (mean area under the receiver operating characteristic curve [AUROC], 0.689), genus (mean AUROC, 0.690), and OTU (mean AUROC, 0.693) levels before decreasing at the ASV level (P &lt; 0.05; mean AUROC, 0.676). These results demonstrate a trade-off between taxonomic resolution and prediction performance, where coarse taxonomic resolution (e.g., phylum) is not distinct enough, but fine resolution (e.g., ASV) is too individualized to accurately classify samples. Similar to the story of Goldilocks and the three bears (L. B. Cauley, Goldilocks and the Three Bears, 1981), mid-range resolution (i.e., family, genus, and OTU) is "just right" for optimal prediction of colorectal cancer from microbiome data. IMPORTANCE Despite being highly preventable, colorectal cancer remains a leading cause of cancer-related death in the United States. Low-cost, noninvasive detection methods could greatly improve our ability to identify and treat early stages of disease. The microbiome has shown promise as a resource for detection of colorectal cancer. Research on the gut microbiome tends to focus on improving our ability to profile species and strain level taxonomic resolution. However, we found that finer resolution impedes the ability to predict colorectal cancer based on the gut microbiome. These results highlight the need for consideration of the appropriate taxonomic resolution for microbiome analyses and that finer resolution is not always more informative.

[Source](https://journals.asm.org/doi/10.1128/mbio.03161-21)

10.1128/mbio.03161-21

---

## Towards a metagenomics machine learning interpretable model for understanding the transition from adenoma to colorectal cancer.
 11.01.2022, _CS Casimiro-Soriguer, C Loucera, M Peña-Chilet, J Dopazo_


Gut microbiome is gaining interest because of its links with several diseases, including colorectal cancer (CRC), as well as the possibility of being used to obtain non-intrusive predictive disease biomarkers. Here we performed a meta-analysis of 1042 fecal metagenomic samples from seven publicly available studies. We used an interpretable machine learning approach based on functional profiles, instead of the conventional taxonomic profiles, to produce a highly accurate predictor of CRC with better precision than those of previous proposals. Moreover, this approach is also able to discriminate samples with adenoma, which makes this approach very promising for CRC prevention by detecting early stages in which intervention is easier and more effective. In addition, interpretable machine learning methods allow extracting features relevant for the classification, which reveals basic molecular mechanisms accounting for the changes undergone by the microbiome functional landscape in the transition from healthy gut to adenoma and CRC conditions. Functional profiles have demonstrated superior accuracy in predicting CRC and adenoma conditions than taxonomic profiles and additionally, in a context of explainable machine learning, provide useful hints on the molecular mechanisms operating in the microbiota behind these conditions.

[Source](https://www.nature.com/articles/s41598-021-04182-y)

10.1038/s41598-021-04182-y

---

## Library Preparation and Sequencing Platform Introduce Bias in Metagenomic-Based Characterizations of Microbiomes

[Source](https://www.biorxiv.org/content/10.1101/592154v2.abstract)

---

