## The complexities of the diet-microbiome relationship: advances and perspectives.
 24.01.2021, _ER Leeming, P Louca, R Gibson, C Menni, TD Spector, CI Le Roy_


Personalised dietary modulation of the gut microbiota may be key to disease management. Current investigations provide a broad understanding of the impact of diet on the composition and activity of the gut microbiota, yet detailed knowledge in applying diet as an actionable tool remains limited. Further to the relative novelty of the field, approaches are yet to be standardised and extremely heterogeneous research outcomes have ensued. This may be related to confounders associated with complexities in capturing an accurate representation of both diet and the gut microbiota. This review discusses the intricacies and current methodologies of diet-microbial relations, the implications and limitations of these investigative approaches, and future considerations that may assist in accelerating applications. New investigations should consider improved collection of dietary data, further characterisation of mechanistic interactions, and an increased focus on -omic technologies such as metabolomics to describe the bacterial and metabolic activity of food degradation, together with its crosstalk with the host. Furthermore, clinical evidence with health outcomes is required before therapeutic dietary strategies for microbial amelioration can be made. The potential to reach detailed understanding of diet-microbiota relations may depend on re-evaluation, progression, and unification of research methodologies, which consider the complexities of these interactions.

[Source](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-020-00813-7)

10.1186/s13073-020-00813-7

---

## A mixed community of skin microbiome representatives influences cutaneous processes more than individual members.
 26.01.2021, _KH Loomis, SK Wu, A Ernlund, K Zudock, A Reno, K Blount, DK Karig_


Skin, the largest organ of the human body by weight, hosts a diversity of microorganisms that can influence health. The microbial residents of the skin are now appreciated for their roles in host immune interactions, wound healing, colonization resistance, and various skin disorders. Still, much remains to be discovered in terms of the host pathways influenced by skin microorganisms, as well as the higher-level skin properties impacted through these microbe-host interactions. Towards this direction, recent efforts using mouse models point to pronounced changes in the transcriptional profiles of the skin in response to the presence of a microbial community. However, there is a need to quantify the roles of microorganisms at both the individual and community-level in healthy human skin. In this study, we utilize human skin equivalents to study the effects of individual taxa and a microbial community in a precisely controlled context. Through transcriptomics analysis, we identify key genes and pathways influenced by skin microbes, and we also characterize higher-level impacts on skin processes and properties through histological analyses.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00963-1)

10.1186/s40168-020-00963-1

---

## Analysis of the Composition and Functions of the Microbiome in Diabetic Foot Osteomyelitis Based on 16S rRNA and Metagenome Sequencing Technology.
 25.01.2021, _M Zou, Y Cai, P Hu, Y Cao, X Luo, X Fan, B Zhang, X Wu, N Jiang, Q Lin, H Zhou, Y Xue, F Gao_


_Aged, Diabetic Foot, Female, Humans, Male, Metagenome, Metagenomics, Microbiota, Middle Aged, Osteomyelitis, RNA, Bacterial, RNA, Ribosomal, 16S, Sequence Analysis, RNA_
Metagenome sequencing has not been used in infected bone specimens. This prospective observational study explored the microbiome and its function in patients with diabetic foot osteomyelitis (DFO) and posttraumatic foot osteomyelitis (PFO) based on 16S rRNA sequencing and metagenome sequencing technologies. Spearman analysis was used to explore the correlation between dominant species and clinical indicators of patients with DFO. High-throughput sequencing showed that all the specimens were polymicrobial. The microbial diversity was significantly higher in the DFO group than in the PFO group. Firmicutes, Prevotellaceae, and Prevotella were the most abundant microbes in the DFO group. The most abundant microbes in the PFO group were Proteobacteria, Halomonadaceae, and HalomonasPrevotella denticola, Prevotella jejuni, and Prevotella fusca had positive correlation with the duration of diabetic foot infection (DFI_d). Proteus vulgaris was positively correlated with the infection index, while Bacteroides fragilis was negatively correlated. The microbial functional genes were more abundant in the DFO group than in the PFO group. Metagenome sequencing is feasible for the analysis of the microbiome in infected bone specimens. Gram-negative bacteria and anaerobes are dominant in DFO.

[Source](https://diabetes.diabetesjournals.org/content/69/11/2423)

10.2337/db20-0503

---

## Gut microbiota condition the therapeutic efficacy of trastuzumab in HER2-positive breast cancer.
 23.01.2021, _M Di Modica, G Gargari, V Regondi, A Bonizzi, S Arioli, B Belmonte, L De Cecco, E Fasano, F Bianchi, A Bertolotti, C Tripodo, L Villani, F Corsi, S Guglielmetti, A Balsari, T Triulzi, E Tagliabue_


Emerging evidence indicates that gut microbiota affect the response to anticancer therapies by modulating the host immune system. In this study, we investigated the impact of the gut microbiota on immune-mediated trastuzumab antitumor efficacy in preclinical models of HER2-positive breast cancer (BC) and in 24 patients with primary HER2-positive BC undergoing trastuzumab-containing neoadjuvant treatment. In mice, the antitumor activity of trastuzumab was impaired by antibiotic administration or fecal microbiota transplantation from antibiotic-treated donors. Modulation of the intestinal microbiota was reflected in tumors by impaired recruitment of CD4+ T cells and GZMB+ cells after trastuzumab treatment. Antibiotics caused reductions in dendritic cell (DC) activation and the release of IL12p70 upon trastuzumab treatment, a mechanism that was necessary for trastuzumab effectiveness in our model. In patients, lower α-diversity and lower abundance of Lachnospiraceae, Turicibacteriaceae, Bifidobacteriaceae and Prevotellaceae characterized nonresponsive patients (NR) compared to those who achieved pathological complete response (R), similar to antibiotic-treated mice. The transfer of fecal microbiota from R and NR into mice bearing HER2-positive BC recapitulated the response to trastuzumab observed in patients. Fecal microbiota β-diversity segregated patients according to response and positively correlated with immune signature related to interferon, IL12-NO, activated CD4+ T cells and activated DC in tumors. Overall, our data reveal the direct involvement of the gut microbiota in trastuzumab efficacy, suggesting that manipulation of the gut microbiota is an optimal future strategy to achieve a therapeutic effect or to exploit its potential as a biomarker for treatment response.

[Source](https://cancerres.aacrjournals.org/content/early/2021/01/21/0008-5472.CAN-20-1659.short)

10.1158/0008-5472.CAN-20-1659

---

## Role of the gut microbiota in type 2 diabetes and related diseases.
 26.01.2021, _G Yang, J Wei, P Liu, Q Zhang, Y Tian, G Hou, L Meng, Y Xin, X Jiang_


Type 2 diabetes is the fastest-growing metabolic disease in the world. Many clinical studies have found that type 2 diabetes patients have metabolic disorders and chronic inflammatory states accompanied by disturbances in the gut microbiota. The gut microbiota plays an important role in body metabolism and immune regulation, and disturbances in the gut microbiota in conjunction with destruction of the intestinal barrier in type 2 diabetes patients causes damage to multiple organs. Therefore, the gut microbiota may be a new therapeutic target for treating type 2 diabetes and related diseases. In this review, we introduce the characteristics of the gut microbiota in type 2 diabetes and related diseases, as well as highlight the potential molecular mechanisms of their effects on intestinal barrier disruption, metabolic disorders, and chronic inflammation. Finally, we summarize an intestinal microecological therapeutic strategy, with a focus on shaping the intestinal bacteria, to improve the malignant progress of type 2 diabetes and related diseases. Author Summary. Type 2 diabetes(T2D)is the fastest-growing metabolic disease in the world. Many clinical studies have found that T2D patients have metabolic disorders and chronic inflammatory states, accompanied by disturbances of the gut microbiota and increased intestinal permeability. The number of human gut microbiota is more than 10 times of human cells, and they play an important role in the body's metabolism and immune regulation. The abnormal intestinal metabolites and intestinal barrier disruption caused by the gut microbiota dysbiosis in the T2D facilitate intestinal bacteria and their harmful metabolites entering the circulatory system. The abnormal entering will cause the damage to multiple organs through disturbing insulin sensitivity, glucose metabolism, and immune homeostasis. Therefore, the gut microbiota may be a new therapeutic target for improving T2D and its related diseases. In this review, we introduce the compositional characteristics of the gut microbiota in T2D, and highlight some new molecular mechanisms of their effects on intestinal barrier disruption, metabolic disorders and chronic inflammation in T2D and its related diseases. Finally, we summarize an intestinal microecological therapeutic strategy, with a focus on shaping the intestinal bacteria, to improve the malignant progress of T2D and related diseases.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0026049521000123)

10.1016/j.metabol.2021.154712

---

## Seasonal shifts in the gut microbiome indicate plastic responses to diet in wild geladas.
 24.01.2021, _A Baniel, KR Amato, JC Beehner, TJ Bergman, A Mercer, RF Perlman, L Petrullo, L Reitsema, S Sams, A Lu, N Snyder-Mackler_


Adaptive shifts in gut microbiome composition are one route by which animals adapt to seasonal changes in food availability and diet. However, outside of dietary shifts, other potential environmental drivers of gut microbial composition have rarely been investigated, particularly in organisms living in their natural environments.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00977-9)

10.1186/s40168-020-00977-9

---

## Taxonomic and functional adaption of the gastrointestinal microbiome of goats kept at high altitude (4800 m) under intensive or extensive rearing conditions.
 20.01.2021, _K Zhang, C He, Y Xu, C Zhang, C Li, X Jing, M Wang, Y Yang, L Suo, P Kalds, J Song, X Wang, D Brugger, Y Wu, Y Chen_


The gut microbiota composition is influenced by the diet as well as the environment in both wild and domestic animals. We studied the effects of two feeding systems on the rumen and hindgut microbiome of semi-feral Tibetan goats kept at high altitude (∼4800 m) using 16S rRNA gene and metagenomic sequencing. Intensive drylot feeding resulted in significantly higher zootechnical performance, narrower ruminal acetate: propionate ratios and a drop in the average rumen pH at slaughter to ∼5.04. Hindgut microbial adaption appeared to be more diverse in the drylot group suggesting a higher influx of undegraded complex non-starch polysaccharides from the rumen. Despite their higher fiber levels in the diet, grazing goats exhibited lower counts of Methanobrevibacter and genes associated with the hydrogenotrophic methanogenesis pathway, presumably reflecting the scarce dietary conditions (low energy density) when rearing goats on pasture from extreme alpine environments. These conditions appeared to promote a relevant abundance of bacitracin genes. In parallel, we recognized a significant increase in the abundance of antibiotic resistance genes in the digestive tracts of drylot animals. In summary, this study provides a deeper insight into the metataxonomic and functional adaption of the gastrointestinal microbiome of goats subject to intensive drylot and extensive pasture rearing conditions at high altitude.

[Source](https://academic.oup.com/femsec/advance-article/doi/10.1093/femsec/fiab009/6104461)

10.1093/femsec/fiab009

---

## Shifts in bacterial communities and antibiotic resistance genes in surface water and gut microbiota of guppies (Poecilia reticulata) in the upper Rio Uberabinha, Brazil.
 26.01.2021, _J Jia, G Gomes-Silva, M Plath, BB Pereira, C UeiraVieira, Z Wang_


Anthropogenic activities especially water pollution can affect the diversity and composition of microbial communities and promote the spread of antibiotic resistance genes (ARGs). In this study, water samples and guppies (Poecilia reticulata) were sampled from six sampling sites along the Uberabinha River in southeastern Brazil, both microbial communities and ARGs of surface waters and intestinal microbiota of guppies (Poecilia reticulata) were detected. According to the results of 16S rRNA amplicon sequencing, Proteobacteria, Bacteroidetes, Firmicutes and Actinobacteria were dominant phyla in both water and intestinal microbiota, but the abundance of putative pathogens was higher at heavily polluted sites. Up to 83% of bacteria in intestinal microbiota originated from water microbiota; this proportion was relatively higher in less polluted compared to polluted environments. ARGs providing resistance of tetracyclines and quinolones were dominant in both water and gut microbiota. The relative abundances of class I integrons and ARGs were as high as 1.74 × 10-1/16S rRNA copies and 3.61 × 10-1/16S rRNA copies, respectively, at heavily polluted sites. Correlation analysis suggests that integrons and bacteria play key roles in explaining the widespread occurrence of ARGs in the surface, but not in intestinal microbiota. We could rule out the class I integrons a potential intermediary bridge for ARGs between both types of microbiomes. Our results highlight the tight link in microbial communities and ARGs between ambient microbiota of stream ecosystems and intestinal microbiota of fish. Our study could have far-reaching consequences for fisheries and consumer safety and calls for investigations of gut microbiota of target species of both commercial fisheries and recreational (hobby) angling.

[Source](https://www.sciencedirect.com/science/article/pii/S014765132100066X)

10.1016/j.ecoenv.2021.111955

---

## Microdiversity and phylogeographic diversification of bacterioplankton in pelagic freshwater systems revealed through long-read amplicon sequencing.
 26.01.2021, _Y Okazaki, S Fujinaga, MM Salcher, C Callieri, A Tanaka, A Kohzu, H Oyagi, H Tamaki, SI Nakano_


Freshwater ecosystems are inhabited by members of cosmopolitan bacterioplankton lineages despite the disconnected nature of these habitats. The lineages are delineated based on &gt; 97% 16S rRNA gene sequence similarity, but their intra-lineage microdiversity and phylogeography, which are key to understanding the eco-evolutional processes behind their ubiquity, remain unresolved. Here, we applied long-read amplicon sequencing targeting nearly full-length 16S rRNA genes and the adjacent ribosomal internal transcribed spacer sequences to reveal the intra-lineage diversities of pelagic bacterioplankton assemblages in 11 deep freshwater lakes in Japan and Europe.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00974-y)

10.1186/s40168-020-00974-y

---

## Microbiota entrapped in recently-formed ice: Paradana Ice Cave, Slovenia.
 22.01.2021, _J Mulec, A Oarga-Mulec, L Holko, L Pašić, AN Kopitar, T Eleršek, A Mihevc_


Paradana is one of the biggest ice caves in Slovenia, with an estimated ice volume of 8,000 m3. Reflecting climatological conditions, the cave ice undergoes repeated freeze-thaw cycles and regular yearly deposition of fresh ice. Three distinct ice block samples, collected from the frozen lake in May 2016, were analysed to obtain data on ice physicochemical properties and the composition of associated microbiota. Isotopic composition of the ice samples (18O, 2H) and a local meteoric water line (LMWL) constructed for monthly precipitation at Postojna were used to estimate the isotopic composition of the water that formed the ice, which had high values of deuterium excess and low concentrations of chloride, sulphate and nitrate. The values of total organic carbon (1.93-3.95 mg/l) within the ice blocks fall within the range of those measured in karst streams. Total cell count in the ice was high and the proportion of cell viability increased along the depth gradient and ranged from 4.67 × 104 to 1.52 × 105 cells/ml and from 51.0 to 85.4%, respectively. Proteobacteria represented the core of the cave-ice microbiome (55.9-79.1%), and probably play an essential role in this ecosystem. Actinobacteria was the second most abundant phylum (12.0-31.4%), followed in abundance by Bacteroidetes (2.8-4.3%). Ice phylotypes recorded amounted to 442 genera, but only 43 genera had abundances greater than 0.5%. Most abundant were Pseudomonas, a well-known ice dweller, and Lysobacter, which previously was not reported in this context. Finally, two xanthophytes, Chloridella glacialis and Ellipsoidion perminimum, known from polar environments, were cultured from the ice. This indicates that the abundance and ecological role of phototrophs in such environments might be greater than previously deduced.

[Source](https://www.nature.com/articles/s41598-021-81528-6)

10.1038/s41598-021-81528-6

---

## Sea‐ice microbial communities in the Central Arctic Ocean: Limited responses to short‐term pCO2 perturbations

[Source](https://aslopubs.onlinelibrary.wiley.com/doi/10.1002/lno.11690)

---

## Bacterial community assemblages in classroom floor dust of 50 public schools in a large city: characterization using 16S rRNA sequences and associations with environmental factors

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00954-2)

---

## Plastics and the microbiome: impacts and solutions

[Source](https://link.springer.com/article/10.1186/s40793-020-00371-w)

---

## Understanding the host-microbe interactions using metabolic modeling

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00955-1)

---

