## Relationships of the very low birth weight infant microbiome with neurodevelopment at 2 and 4 years of age.
 28.10.2022, _A Sarkar, SM Prescott, S Dutra, JY Yoo, J Gordon, E Shaffer, D McSkimming, ME Groer_


_Infant, Newborn, Infant, Adult, Humans, Child, Preschool, Infant, Very Low Birth Weight, Intensive Care Units, Neonatal, Gestational Age, Birth Weight, Microbiota, Anti-Bacterial Agents_
Very low birth weight (VLBW) infants (&lt;1500 g) are at risk for poor neurodevelopmental outcomes depending on gestational age (GA), birth weight (BW), and morbidity in early life. The contribution of the gut microbiome is not well understood. Stool samples were collected weekly in the neonatal intensive care unit (NICU) from 24 VLBW infants for 6 weeks after admission and then again at 2 and 4 years of age. The Battelle Development Inventory-2 Screening Test (BDI-2 ST) was administered at 2- and 4-year time points. VLBW infants had dysbiotic microbiota in the NICU that progressed for most to an adult-type microbiota by 4 years of age. The BDI-2 ST results at age of 2 years triggered referral for further testing in 14 toddlers (70%), and by 4 years of age only seven of these 14 continued to require referral. Both NICU infant stool diversity and particular microbial amplicon sequence variants were associated with BDI-2 ST subscales, particularly for cognition, adaptive, and communication subscales, when controlled for GA, BW, and antibiotic exposure. Network analysis of the NICU infant stool microbial ecology showed differences in children needing neurodevelopmental referral. The results of this preliminary study indicate that the neonatal gut microbiome plays a role in early cognitive and behavioral neurodevelopment.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/dev.22317)

10.1002/dev.22317

---

## The lower airways microbiota and antimicrobial peptides indicate dysbiosis in sarcoidosis.
 22.10.2022, _KS Knudsen, S Lehmann, R Nielsen, S Tangedal, A Paytuvi-Gallart, W Sanseverino, EMH Martinsen, PS Hiemstra, TM Eagan_


_Humans, Dysbiosis, beta-Defensins, Antimicrobial Peptides, Bronchoalveolar Lavage Fluid, RNA, Ribosomal, 16S, Microbiota, Sarcoidosis, Lung, Bacteria, Protease Inhibitors, Adenosine Monophosphate_
The role of the pulmonary microbiome in sarcoidosis is unknown. The objectives of this study were the following: (1) examine whether the pulmonary fungal and bacterial microbiota differed in patients with sarcoidosis compared with controls; (2) examine whether there was an association between the microbiota and levels of the antimicrobial peptides (AMPs) in protected bronchoalveolar lavage (PBAL).

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01362-4)

10.1186/s40168-022-01362-4

---

## Profiling the fungal microbiome after FMT for graft-versus-host disease: insights from a phase 1 interventional study.
 24.10.2022, _YF van Lier, T Rolling, GK Armijo, B Zhai, NJE Haverkate, E Meijer, E Nur, B Blom, JU Peled, MRM van den Brink, TM Hohl, MD Hazenberg, KA Markey_


Disruption of the intestinal bacterial microbiota is frequently observed in the context of allogeneic hematopoietic cell transplantation and is particularly pronounced in patients that develop graft-versus-host disease (GvHD). Donor fecal microbiota transplantation restores gut microbial diversity and reduces graft-versus-host disease in hematopoietic cell transplant recipients. The composition of the intestinal fungal community in GvHD patients, and whether fungal taxa are transferred during FMT is currently unknown. We performed a secondary analysis of our clinical trial on FMT in steroid-refractory GvHD with a focus on the mycobiota. We characterized the fecal mycobiota of the seventeen patients and of the healthy donors using internal transcribed spacer (ITS)-1 amplicon sequencing. The donor who provided the majority of FMT material in our study represents an n-of-one study of the intestinal flora over time. In this donor, mycobiota composition fluctuated over time, while the bacterial microbiota remained stable over sixteen months. Fungal DNA was more frequently detected in baseline stool samples from patients with steroid-refractory GvHD than in patients with steroid-dependent GvHD. We could detect fungal taxa in the majority of samples, but did not see evidence of mycobiota transfer from donor to recipient. Our study demonstrates the feasibility of profiling the mycobiota alongside the more-traditional bacterial microbiota, establishes the methodology, and provides a first insight into the mycobiota composition of GvHD patients.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S2666636722017067)

10.1016/j.jtct.2022.10.011

---

## Colorectal cancer, Vitamin D and microbiota: A double-blind Phase II randomized trial (ColoViD) in colorectal cancer patients.
 24.10.2022, _F Bellerba, D Serrano, J Harriet, C Pozzi, N Segata, A NabiNejad, E Piperni, P Gnagnarella, D Macis, V Aristarco, CA Accornero, P Manghi, AG Gonzaga, R Biffi, L Bottiglieri, C Trovato, MG Zampino, F Corso, R Bellocco, S Raimondi, M Rescigno, S Gandini_


Several studies suggest a role of gut microbiota in colorectal cancer (CRC) initiation and progression. Vitamin D (vitD) blood levels are also inversely correlated with CRC risk and prognosis. However, these factors' interplay remains unknown.

[Source](https://www.sciencedirect.com/science/article/pii/S1476558622000689?via%3Dihub)

10.1016/j.neo.2022.100842

---

## Natural and after colon washing fecal samples: the two sides of the coin for investigating the human gut microbiome.
 27.10.2022, _E Piancone, B Fosso, M Marzano, M De Robertis, E Notario, A Oranger, C Manzari, S Bruno, G Visci, G Defazio, AM D'Erchia, E Filomena, D Maio, M Minelli, I Vergallo, M Minelli, G Pesole_


_Humans, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Feces, Microbiota, DNA, Ribosomal, Colon_
To date several studies address the important role of gut microbiome and its interplay with the human host in the health and disease status. However, the selection of a universal sampling matrix representative of the microbial biodiversity associated with the gastrointestinal (GI) tract, is still challenging. Here we present a study in which, through a deep metabarcoding analysis of the 16S rRNA gene, we compared two sampling matrices, feces (F) and colon washing feces (CWF), in order to evaluate their relative effectiveness and accuracy in representing the complexity of the human gut microbiome. A cohort of 30 volunteers was recruited and paired F and CWF samples were collected from each subject. Alpha diversity analysis confirmed a slightly higher biodiversity of CWF compared to F matched samples. Likewise, beta diversity analysis proved that paired F and CWF microbiomes were quite similar in the same individual, but remarkable inter-individual variability occurred among the microbiomes of all participants. Taxonomic analysis in matched samples was carried out to investigate the intra and inter individual/s variability. Firmicutes, Bacteroidota, Proteobacteria and Actinobacteriota were the main phyla in both F and CWF samples. At genus level, Bacteirodetes was the most abundant in F and CWF samples, followed by Faecalibacterium, Blautia and Escherichia-Shigella. Our study highlights an inter-individual variability greater than intra-individual variability for paired F and CWF samples. Indeed, an overall higher similarity was observed across matched F and CWF samples, suggesting, as expected, a remarkable overlap between the microbiomes inferred using the matched F and CWF samples. Notably, absolute quantification of total 16S rDNA by droplet digital PCR (ddPCR) revealed comparable overall microbial load between paired F and CWF samples. We report here the first comparative study on fecal and colon washing fecal samples for investigating the human gut microbiome and show that both types of samples may be used equally for the study of the gut microbiome. The presented results suggest that the combined use of both types of sampling matrices could represent a suitable choice to obtain a more complete overview of the human gut microbiota for addressing different biological and clinical questions.

[Source](https://www.nature.com/articles/s41598-022-20888-z)

10.1038/s41598-022-20888-z

---

## Effects of heat stress on 16S rDNA, metagenome and metabolome in Holstein cows at different growth stages.
 26.10.2022, _L Feng, Y Zhang, W Liu, D Du, W Jiang, Z Wang, Z Wang, Z Hu_


_Animals, Cattle, Female, Diet, DNA, Ribosomal, Heat-Shock Response, Lactation, Metabolome, Metagenome, Milk_
Heat stress is an important issue in dairy cattle feeding management affecting summer health and economic efficiency. This experiment combined 16S rDNA sequencing(3,864,982 tags, 30 sequencing data), metagenomic sequencing(1,269,441,128 reads, 18 sequencing data), metabolomics analysis(72 sequencing data) and blood index analysis. Ten cows in each animal type (growing heifers, heifers, and lactating cows) were selected for sample collection in April and August. Here, we characterized both the changes in metabolites, rumen microbial communities and their functional potential and the effects of heat stress on serum biochemical, immune, oxidative stress, and hormonal indices derived from rumen fluid and serum samples from cows during different growth stages and in different climates. The generated data expand the resources for the rumen microbiome related to heat stress and age and provide useful datasets for research on developing therapeutic strategies to achieve high summer milk production in cows. These datasets will help researchers study the effects of heat stress on the physiological metabolism of Holstein cows and the time-dependent changes associated with growth stages.

[Source](https://www.nature.com/articles/s41597-022-01777-6)

10.1038/s41597-022-01777-6

---

## Tick transmission of Borrelia burgdorferi to the murine host is not influenced by environmentally acquired midgut microbiota

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01378-w)

---

## Mining the microbiota to identify gut commensals modulating neuroinflammation in a mouse model of multiple sclerosis

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01364-2)

---

## Microbiomes associated with avian malaria survival differ between susceptible Hawaiian honeycreepers and sympatric malaria-resistant introduced birds

[Source](https://onlinelibrary.wiley.com/doi/10.1111/mec.16743)

---

## Impact of Germinated Brown Rice and Brown Rice on Metabolism, Inflammation, and Gut Microbiome in High Fat Diet-Induced Insulin Resistant Mice

[Source](https://pubs.acs.org/doi/10.1021/acs.jafc.2c06662)

---

## Dietary Lactobacillus reuteri prevent from inflammation mediated apoptosis of liver via improving intestinal microbiota and bile acid metabolism

[Source](https://www.sciencedirect.com/science/article/abs/pii/S030881462202605X)

---

## Specific metabolites drive the deterministic assembly of diseased rhizosphere microbiome through weakening microbial degradation of autotoxin

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01375-z)

---

## Low-abundance populations distinguish microbiome performance in plant cell wall deconstruction

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01377-x)

---

## Community assembly patterns and processes of microbiome responses to habitats and Mytilopsis sallei invasion in the tidal zones of the Pearl River Estuary

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969722067754)

---

## LotuS2: an ultrafast and highly accurate tool for amplicon sequencing analysis

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01365-1)

---

## Double emulsions as a high-throughput enrichment and isolation platform for slower-growing microbes

[Source](https://www.biorxiv.org/content/10.1101/2022.10.23.513397v1.abstract)

---

## A novel metagenome-derived viral RNA polymerase and its application in a cell-free expression system for metagenome screening

[Source](https://www.nature.com/articles/s41598-022-22383-x)

---

