## Mechanism of Dyspnea during Exercise in Children with Corrected Congenital Heart Disease.
 24.01.2022, _M Chlif, MM Ammar, NB Said, L Sergey, S Ahmaidi, F Alassery, H Hamam_


_Child, Dyspnea, Exercise, Exercise Test, Heart Defects, Congenital, Humans, Oxygen Consumption_
This study will evaluate cardiorespiratory and peripheral muscle function and their relationship with subjective dyspnea threshold after the surgical correction of congenital heart disease in children. Thirteen children with surgically repaired congenital heart disease were recruited. Each participant performed an incremental exercise test on a cycle ergometer until exhaustion. Gas exchanges were continuously sampled to measure the maximal aerobic parameters and ventilatory thresholds. The functional capacity of the subjects was assessed with a 6 min walk test. At the end of the exercise test, isokinetic Cybex Norm was used to evaluate the strength and endurance of the knee extensor muscle in the leg. Dyspnea was subjectively scored with a visual analog scale during the last 15 s of each exercise step. Oxygen consumption measured at the dyspnea score/VO2 relationship located at the dyspnea threshold, at which dyspnea suddenly increased. Results: The maximal and submaximal values of the parameters describing the exercise and the peripheral muscular performances were: VO2 Peak: 33.8 ± 8.9 mL·min-1·kg-1; HR: 174 ± 9 b·min-1; VEmax: 65.68 ± 15.9 L·min-1; P max: 117 ± 27 W; maximal voluntary isometric force MVIF: 120.8 ± 41.9 N/m; and time to exhaustion Tlim: 53 ± 21 s. Oxygen consumption measured at the dyspnea threshold was related to VO2 Peak (R2 = 0.74; p &lt; 0.01), Tlim (R2 = 0.78; p &lt; 0.01), and the distance achieved during the 6MWT (R2 = 0.57; p &lt; 0.05). Compared to the theoretical maximal values for the power output, VO2, and HR, the surgical correction did not repair the exercise performance. After the surgical correction of congenital heart disease, exercise performance was impeded by alterations of the cardiorespiratory function and peripheral local factors. A subjective evaluation of the dyspnea threshold is a reliable criterion that allows the prediction of exercise capacity in subjects suffering from congenital heart disease.

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

10.3390/ijerph19010099

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## @JenWargoMD

[Source](https://twitter.com/JenWargoMD?ref_src=twsrc%5Etfw)

---

## @NikolayOskolkov

[Source](https://twitter.com/NikolayOskolkov?ref_src=twsrc%5Etfw)

---

## @SondraTurjeman

[Source](https://twitter.com/SondraTurjeman?ref_src=twsrc%5Etfw)

---

## @MireiaVallesc

[Source](https://twitter.com/MireiaVallesc?ref_src=twsrc%5Etfw)

---

## https://t.co/5lRK9Mj8hN

[Source](https://t.co/5lRK9Mj8hN)

---

## https://t.co/EAD6ZZoxTC

[Source](https://t.co/EAD6ZZoxTC)

---

## pic.twitter.com/3c240LsCsC

[Source](https://t.co/3c240LsCsC)

---

## December 21, 2023

[Source](https://twitter.com/MicrobiomeVIF/status/1737811126937645489?ref_src=twsrc%5Etfw)

---

## Vaginal <em>Lactobacillus </em>fatty acid response mechanisms reveal a novel strategy for bacterial vaginosis treatment

[Source](https://www.biorxiv.org/content/10.1101/2023.12.30.573720v1)

---

## #microbiome

[Source](https://twitter.com/hashtag/microbiome?src=hash&ref_src=twsrc%5Etfw)

---

## #Lactobacillus

[Source](https://twitter.com/hashtag/Lactobacillus?src=hash&ref_src=twsrc%5Etfw)

---

## Characteristics and hormonal contraception use patterns of levonorgestrel 13.5 mg intrauterine device users in Sweden 2014-2016.
 16.10.2023, _M Linder, J Schoendorf, F Pisa, R Schulze-Rath, CE Cesta_


The levonorgestrel-releasing intrauterine device containing 13.5 mg of levonorgestrel (LNG 13.5 mg IUD), Jaydess®, was approved for contraception by the European Medicines Agency in 2013. We aimed to describe the characteristics of new users of LNG 13.5 mg IUD in Sweden within the first 3 years after approval (2014-2016).

[Source](https://twitter.com/hashtag/WomensHealth?src=hash&ref_src=twsrc%5Etfw)

10.1016/j.contraception.2023.110309

---

## #WomensHealth

[Source](https://twitter.com/hashtag/WomensHealth?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/8Rx5qpeXtz

[Source](https://t.co/8Rx5qpeXtz)

---

## Rosie, a Health Education Question-and-Answer Chatbot for New Mothers: Randomized Pilot Study.
 12.01.2024, _QC Nguyen, EM Aparicio, M Jasczynski, A Channell Doig, X Yue, H Mane, N Srikanth, FXM Gutierrez, N Delcid, X He, J Boyd-Graber_


Stark disparities exist in maternal and child outcomes and there is a need to provide timely and accurate health information.

[Source](https://twitter.com/mzhu_ology/status/1742564944120516923?ref_src=twsrc%5Etfw)

10.2196/51361

---

## January 3, 2024

[Source](https://twitter.com/mzhu_ology/status/1742564944120516923?ref_src=twsrc%5Etfw)

---

## BilR is a gut microbial enzyme that reduces bilirubin to urobilinogen.
 08.01.2024, _B Hall, S Levy, K Dufault-Thompson, G Arp, A Zhong, GM Ndjite, A Weiss, D Braccia, C Jenkins, MR Grant, S Abeysinghe, Y Yang, MD Jermain, CH Wu, B Ma, X Jiang_


_Infant, Newborn, Adult, Humans, Bilirubin, Urobilinogen, Gastrointestinal Microbiome, Liver, Bacteria_
Metabolism of haem by-products such as bilirubin by humans and their gut microbiota is essential to human health, as excess serum bilirubin can cause jaundice and even neurological damage. The bacterial enzymes that reduce bilirubin to urobilinogen, a key step in this pathway, have remained unidentified. Here we used biochemical analyses and comparative genomics to identify BilR as a gut-microbiota-derived bilirubin reductase that reduces bilirubin to urobilinogen. We delineated the BilR sequences from similar reductases through the identification of key residues critical for bilirubin reduction and found that BilR is predominantly encoded by Firmicutes species. Analysis of human gut metagenomes revealed that BilR is nearly ubiquitous in healthy adults, but prevalence is decreased in neonates and individuals with inflammatory bowel disease. This discovery sheds light on the role of the gut microbiome in bilirubin metabolism and highlights the significance of the gut-liver axis in maintaining bilirubin homeostasis.

[Source](https://www.nature.com/articles/s41564-023-01549-x)

10.1038/s41564-023-01549-x

---

## Influence of feeding practices in the composition and functionality of infant gut microbiota and its relationship with health.
 06.01.2024, _M Martínez-Martínez, M Martínez-Martínez, R Soria-Guerra, S Gamiño-Gutiérrez, C Senés-Guerrero, A Santacruz, R Flores-Ramírez, A Salazar-Martínez, D Portales-Pérez, H Bach, F Martínez-Gutiérrez_


_Infant, Newborn, Infant, Female, Child, Humans, Gastrointestinal Microbiome, Cross-Sectional Studies, Volatile Organic Compounds, Breast Feeding, Immunoglobulin A, Infant Formula_
Establishing the infant's gut microbiota has long-term implications on health and immunity. Breastfeeding is recognized as the best practice of infant nutrition in comparison with formula feeding. We evaluated the effects of the primary feeding practices by analyzing the infant growth and the potential association with gut diseases. A cross-sectional and observational study was designed. This study included 55 mothers with infants, who were divided according to their feeding practices in breastfeeding (BF), formula feeding (FF), and combined breast and formula feeding (CF). Anthropometric measurements of the participants were recorded. Additionally, non-invasive fecal samples from the infants were collected to analyze the microbiota by sequencing, immunoglobulin A (IgA) concentration (ELISA), and volatile organic compounds (gas chromatography with an electronic nose). Results showed that the microbiota diversity in the BF group was the highest compared to the other two groups. The IgA levels in the BF group were twice as high as those in the FF group. Moreover, the child´s growth in the BF group showed the best infant development when the data were compared at birth to the recollection time, as noted by the correlation with a decreased concentration of toxic volatile organic compounds. Interestingly, the CF group showed a significant difference in health status when the data were compared with the FF group. We conclude that early health practices influence children's growth, which is relevant to further research about how those infants' health evolved.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0294494)

10.1371/journal.pone.0294494

---

## Mucin-degrading gut bacteria promote anti-parasitic immunity

[Source](https://www.biorxiv.org/content/10.1101/2022.02.28.482289v2)

---

## Host genetic regulation of human gut microbial structural variation 

[Source](https://www.nature.com/articles/s41586-023-06893-w)

---

## Deciphering the gut microbiome of grass carp through multi-omics approach.
 06.01.2024, _M Li, H Liang, H Yang, Q Ding, R Xia, J Chen, W Zhou, Y Yang, Z Zhang, Y Yao, C Ran, Z Zhou_


_Animals, Gastrointestinal Microbiome, Carps, Multiomics, Microbiota, Proteobacteria, Fusobacteria, Bacteroidetes, Firmicutes, Fusobacterium, RNA, Ribosomal, 16S, Mammals_
Aquaculture plays an important role in global protein supplies and food security. The ban on antibiotics as feed additive proposes urgent need to develop alternatives. Gut microbiota plays important roles in the metabolism and immunity of fish and has the potential to give rise to novel solutions for challenges confronted by fish culture. However, our understanding of fish gut microbiome is still lacking.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01715-7)

10.1186/s40168-023-01715-7

---

## Spotting disease disrupts the microbiome of infected purple sea urchins, <em>Strongylocentrotus purpuratus</em>

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-023-03161-9)

---

## <em>Desulfovibrio vulgaris </em>interacts with novel gut epithelial immune receptor LRRC19 and exacerbates colitis 

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01722-8)

---

## Individualized microbiotas dictate the impact of dietary fiber on colitis sensitivity.
 08.01.2024, _E Bonazzi, A Bretin, L Vigué, F Hao, AD Patterson, AT Gewirtz, B Chassaing_


_Humans, Mice, Animals, Psyllium, Inulin, Colitis, Dietary Fiber, Gastrointestinal Microbiome, Inflammatory Bowel Diseases_
The observation that the intestinal microbiota is central in the development of IBD suggests that dietary fiber, the microbiota's primary source of nourishment, could play a central role in these diseases. Accordingly, enriching diets with specific soluble fibers remodels microbiota and modulates colitis sensitivity. In humans, a recent study suggests that the microbiota of select IBD patients might influence the impacts they would experience upon fiber exposure. We sought here to define the extent to which individual microbiotas varied in their responsiveness to purified soluble fiber inulin and psyllium. Moreover, the extent to which such variance might impact proneness to colitis.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01724-6)

10.1186/s40168-023-01724-6

---

## A consortia of clinical<em> E. coli</em> strains with distinct in vitro adherent/invasive properties establish their own co-colonization niche and shape the intestinal microbiota in inflammation-susceptible mice

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01710-y)

---

## A unified compendium of prokaryotic and viral genomes from over 300 anaerobic digestion microbiomes.
 05.01.2024, _VB Centurion, A Rossi, E Orellana, G Ghiotto, B Kakuk, MS Morlino, A Basile, G Zampieri, L Treu, S Campanaro_


The anaerobic digestion process degrades organic matter into simpler compounds and occurs in strictly anaerobic and microaerophilic environments. The process is carried out by a diverse community of microorganisms where each species has a unique role and it has relevant biotechnological applications since it is used for biogas production. Some aspects of the microbiome, including its interaction with phages, remains still unclear: a better comprehension of the community composition and role of each species is crucial for a cured understanding of the carbon cycle in anaerobic systems and improving biogas production.

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-023-00545-2)

10.1186/s40793-023-00545-2

---

## Cellulomonas alba sp. nov. and Cellulomonas edaphi sp. nov., isolated from wetland soils.
 12.01.2024, _S Park, I Kim, G Chhetri, Y Jung, H Woo, T Seo_


Two novel strains were isolated from wetland soils in Goyang, Republic of Korea. The two Gram-stain-positive, facultatively anaerobic, rod-shaped bacterial-type strains were designated MW4T and MW9T. Phylogenomic analysis based on whole-genome sequences suggested that both strains belonged to the genus Cellulomonas. The cells of strain MW4T were non-motile and grew at 20-40 °C (optimum, 35 °C), at pH 6.0-10.0 (optimum, pH 8.0) and in the presence of 0-1.0% NaCl (optimum, 0 %). The cells of strain MW9T were non-motile and grew at 20-40 °C (optimum, 35 °C), at pH 5.0-9.0 (optimum, pH 8.0) and in the presence of 0-1.0% NaCl (optimum, 0 %). The average nucleotide identity (77.1-88.1 %) and digital DNA-DNA hybridization values (21.0-34.8 %) between the two novel strains and with their closely related strains fell within the range for the genus Cellulomonas. The novel strains MW4T and MW9T and reference strains possessed alkane synthesis gene clusters (oleA, oleB, oleC and oleD). Phylogenomic, phylogenetic, average nucleotide identity, digital DNA-DNA hybridization, physiological and biochemical data indicated that the novel strains were distinct from other members of the family Cellulomonadaceae. We propose the names Cellulomonas alba sp. nov. (type strain MW4T=KACC 23260T=TBRC 17645T) and Cellulomons edaphi sp. nov. (type strain MW9T=KACC 23261T=TBRC 17646T) for the two strains.

[Source](https://twitter.com/hashtag/anaerobic?src=hash&ref_src=twsrc%5Etfw)

10.1099/ijsem.0.006235

---

## #anaerobic

[Source](https://twitter.com/hashtag/anaerobic?src=hash&ref_src=twsrc%5Etfw)

---

## Mathematical modeling of fluid dynamics in in vitro gut fermentation systems: A new tool to improve the interpretation of microbial metabolism.
 12.01.2024, _J Lessard-Lord, J Lupien-Meilleur, C Roussel, B Gosselin-Cliche, C Silvestri, V Di Marzo, D Roy, E Rousseau, Y Desjardins_


In vitro systems are widely employed to assess the impact of dietary compounds on the gut microbiota and their conversion into beneficial bacterial metabolites. However, the complex fluid dynamics and multi-segmented nature of these systems can complicate the comprehensive analysis of dietary compound fate, potentially confounding physical dilution or washout with microbial catabolism. In this study, we developed fluid dynamics models based on sets of ordinary differential equations to simulate the behavior of an inert compound within two commonly used in vitro systems: the continuous two-stage PolyFermS system and the semi-continuous multi-segmented SHIME® system as well as into various declinations of those systems. The models were validated by investigating the fate of blue dextran, demonstrating excellent agreement between experimental and modeling data (with r2 values ranging from 0.996 to 0.86 for different approaches). As a proof of concept for the utility of fluid dynamics models in in vitro system, we applied generated models to interpret metabolomic data of procyanidin A2 (ProA2) generated from the addition of proanthocyanidin (PAC)-rich cranberry extract to both the PolyFermS and SHIME® systems. The results suggested ProA2 degradation by the gut microbiota when compared to the modeling of an inert compound. Models of fluid dynamics developed in this study provide a foundation for comprehensive analysis of gut metabolic data in commonly utilized in vitro PolyFermS and SHIME® bioreactor systems and can enable a more accurate understanding of the contribution of bacterial metabolism to the variability in the concentration of target metabolites.

[Source](https://twitter.com/hashtag/digestion?src=hash&ref_src=twsrc%5Etfw)

10.1096/fj.202301739RR

---

## #digestion

[Source](https://twitter.com/hashtag/digestion?src=hash&ref_src=twsrc%5Etfw)

---

## Mathematical modeling of fluid dynamics in in vitro gut fermentation systems: A new tool to improve the interpretation of microbial metabolism.
 12.01.2024, _J Lessard-Lord, J Lupien-Meilleur, C Roussel, B Gosselin-Cliche, C Silvestri, V Di Marzo, D Roy, E Rousseau, Y Desjardins_


In vitro systems are widely employed to assess the impact of dietary compounds on the gut microbiota and their conversion into beneficial bacterial metabolites. However, the complex fluid dynamics and multi-segmented nature of these systems can complicate the comprehensive analysis of dietary compound fate, potentially confounding physical dilution or washout with microbial catabolism. In this study, we developed fluid dynamics models based on sets of ordinary differential equations to simulate the behavior of an inert compound within two commonly used in vitro systems: the continuous two-stage PolyFermS system and the semi-continuous multi-segmented SHIME® system as well as into various declinations of those systems. The models were validated by investigating the fate of blue dextran, demonstrating excellent agreement between experimental and modeling data (with r2 values ranging from 0.996 to 0.86 for different approaches). As a proof of concept for the utility of fluid dynamics models in in vitro system, we applied generated models to interpret metabolomic data of procyanidin A2 (ProA2) generated from the addition of proanthocyanidin (PAC)-rich cranberry extract to both the PolyFermS and SHIME® systems. The results suggested ProA2 degradation by the gut microbiota when compared to the modeling of an inert compound. Models of fluid dynamics developed in this study provide a foundation for comprehensive analysis of gut metabolic data in commonly utilized in vitro PolyFermS and SHIME® bioreactor systems and can enable a more accurate understanding of the contribution of bacterial metabolism to the variability in the concentration of target metabolites.

[Source](https://twitter.com/hashtag/microbiomes?src=hash&ref_src=twsrc%5Etfw)

10.1096/fj.202301739RR

---

## #microbiomes

[Source](https://twitter.com/hashtag/microbiomes?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/GhTcYWnDnf

[Source](https://t.co/GhTcYWnDnf)

---

## Mathematical modeling of fluid dynamics in in vitro gut fermentation systems: A new tool to improve the interpretation of microbial metabolism.
 12.01.2024, _J Lessard-Lord, J Lupien-Meilleur, C Roussel, B Gosselin-Cliche, C Silvestri, V Di Marzo, D Roy, E Rousseau, Y Desjardins_


In vitro systems are widely employed to assess the impact of dietary compounds on the gut microbiota and their conversion into beneficial bacterial metabolites. However, the complex fluid dynamics and multi-segmented nature of these systems can complicate the comprehensive analysis of dietary compound fate, potentially confounding physical dilution or washout with microbial catabolism. In this study, we developed fluid dynamics models based on sets of ordinary differential equations to simulate the behavior of an inert compound within two commonly used in vitro systems: the continuous two-stage PolyFermS system and the semi-continuous multi-segmented SHIME® system as well as into various declinations of those systems. The models were validated by investigating the fate of blue dextran, demonstrating excellent agreement between experimental and modeling data (with r2 values ranging from 0.996 to 0.86 for different approaches). As a proof of concept for the utility of fluid dynamics models in in vitro system, we applied generated models to interpret metabolomic data of procyanidin A2 (ProA2) generated from the addition of proanthocyanidin (PAC)-rich cranberry extract to both the PolyFermS and SHIME® systems. The results suggested ProA2 degradation by the gut microbiota when compared to the modeling of an inert compound. Models of fluid dynamics developed in this study provide a foundation for comprehensive analysis of gut metabolic data in commonly utilized in vitro PolyFermS and SHIME® bioreactor systems and can enable a more accurate understanding of the contribution of bacterial metabolism to the variability in the concentration of target metabolites.

[Source](https://twitter.com/hashtag/Microbiome?src=hash&ref_src=twsrc%5Etfw)

10.1096/fj.202301739RR

---

## #Microbiome

[Source](https://twitter.com/hashtag/Microbiome?src=hash&ref_src=twsrc%5Etfw)

---

## Efficient SARS-CoV-2 infection antagonization by rhACE2 ectodomain multimerized onto the Avidin-Nucleic-Acid-NanoASsembly.
 03.01.2024, _S Bernardotto, I Frasson, S Faravelli, A Morelli, E Schiavon, GY Moscatiello, MB Violatto, A Pinnola, A Canciani, A Mattarei, G Rossi, M Brini, L Pasetto, V Bonetto, P Bigini, F Forneris, SN Richter, M Morpurgo_


_Animals, Humans, Mice, COVID-19, SARS-CoV-2, Avidin, Angiotensin-Converting Enzyme 2, Nucleic Acids, Tissue Distribution, Protein Binding, Receptors, Virus, Lipids_
Nanodecoy systems based on analogues of viral cellular receptors assembled onto fluid lipid-based membranes of nano/extravescicles are potential new tools to complement classic therapeutic or preventive antiviral approaches. The need for lipid-based membranes for transmembrane receptor anchorage may pose technical challenges along industrial translation, calling for alternative geometries for receptor multimerization. Here we developed a semisynthetic self-assembling SARS-CoV-2 nanodecoy by multimerizing the biotin labelled virus cell receptor -ACE2- ectodomain onto a poly-avidin nanoparticle (NP) based on the Avidin-Nucleic-Acid-NanoASsembly-ANANAS. The ability of the assembly to prevent SARS-CoV-2 infection in human lung cells and the affinity of the ACE2:viral receptor-binding domain (RBD) interaction were measured at different ACE2:NP ratios. At ACE2:NP = 30, 90 % SARS-CoV-2 infection inhibition at ACE2 nanomolar concentration was registered on both Wuhan and Omicron variants, with ten-fold higher potency than the monomeric protein. Lower and higher ACE2 densities were less efficient suggesting that functional recognition between multi-ligand NPs and multi-receptor virus surfaces requires optimal geometrical relationships. In vivo studies in mice showed that the biodistribution and safety profiles of the nanodecoy are potentially suitable for preventing viral infection upon nasal instillation. Viral receptor multimerization using ANANAS is a convenient process which, in principle, could be rapidly adapted to counteract also other viral infections.

[Source](https://twitter.com/DiBio_UniPD?ref_src=twsrc%5Etfw)

10.1016/j.biomaterials.2023.122394

---

## @DiBio_UniPD

[Source](https://twitter.com/DiBio_UniPD?ref_src=twsrc%5Etfw)

---

## @LauraTreu

[Source](https://twitter.com/LauraTreu?ref_src=twsrc%5Etfw)

---

## A unified compendium of prokaryotic and viral genomes from over 300 anaerobic digestion microbiomes.
 05.01.2024, _VB Centurion, A Rossi, E Orellana, G Ghiotto, B Kakuk, MS Morlino, A Basile, G Zampieri, L Treu, S Campanaro_


The anaerobic digestion process degrades organic matter into simpler compounds and occurs in strictly anaerobic and microaerophilic environments. The process is carried out by a diverse community of microorganisms where each species has a unique role and it has relevant biotechnological applications since it is used for biogas production. Some aspects of the microbiome, including its interaction with phages, remains still unclear: a better comprehension of the community composition and role of each species is crucial for a cured understanding of the carbon cycle in anaerobic systems and improving biogas production.

[Source](https://twitter.com/guido_zampieri?ref_src=twsrc%5Etfw)

10.1186/s40793-023-00545-2

---

## @guido_zampieri

[Source](https://twitter.com/guido_zampieri?ref_src=twsrc%5Etfw)

---

## @orellanaest

[Source](https://twitter.com/orellanaest?ref_src=twsrc%5Etfw)

---

## @GabrieleGhiotto

[Source](https://twitter.com/GabrieleGhiotto?ref_src=twsrc%5Etfw)

---

## pic.twitter.com/tghPIRUyvZ

[Source](https://t.co/tghPIRUyvZ)

---

## Rosie, a Health Education Question-and-Answer Chatbot for New Mothers: Randomized Pilot Study.
 12.01.2024, _QC Nguyen, EM Aparicio, M Jasczynski, A Channell Doig, X Yue, H Mane, N Srikanth, FXM Gutierrez, N Delcid, X He, J Boyd-Graber_


Stark disparities exist in maternal and child outcomes and there is a need to provide timely and accurate health information.

[Source](https://twitter.com/campanarostef/status/1742576845915070538?ref_src=twsrc%5Etfw)

10.2196/51361

---

## January 3, 2024

[Source](https://twitter.com/campanarostef/status/1742576845915070538?ref_src=twsrc%5Etfw)

---

## Unraveling the interplay between root exudates, microbiota, and rhizosheath formation in pearl millet.
 06.01.2024, _A Alahmad, M Harir, S Fochesato, J Tulumello, A Walker, M Barakat, PMS Ndour, P Schmitt-Kopplin, L Cournac, L Laplaze, T Heulin, W Achouak_


_Pennisetum, RNA, Ribosomal, 16S, Plant Roots, Microbiota, Soil, Plants, Exudates and Transudates, Soil Microbiology, Rhizosphere_
The rhizosheath, a cohesive soil layer firmly adhering to plant roots, plays a vital role in facilitating water and mineral uptake. In pearl millet, rhizosheath formation is genetically controlled and influenced by root exudates. Here, we investigated the impact of root exudates on the microbiota composition, interactions, and assembly processes, and rhizosheath structure in pearl millet using four distinct lines with contrasting soil aggregation abilities.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-023-01727-3)

10.1186/s40168-023-01727-3

---

## mEnrich-seq: methylation-guided enrichment sequencing of bacterial taxa of interest from microbiome.
 04.01.2024, _L Cao, Y Kong, Y Fan, M Ni, A Tourancheau, M Ksiezarek, EA Mead, T Koo, M Gitman, XS Zhang, G Fang_


Metagenomics has enabled the comprehensive study of microbiomes. However, many applications would benefit from a method that sequences specific bacterial taxa of interest, but not most background taxa. We developed mEnrich-seq (in which 'm' stands for methylation and seq for sequencing) for enriching taxa of interest from metagenomic DNA before sequencing. The core idea is to exploit the self versus nonself differentiation by natural bacterial DNA methylation and rationally choose methylation-sensitive restriction enzymes, individually or in combination, to deplete host and background taxa while enriching targeted taxa. This idea is integrated with library preparation procedures and applied in several applications to enrich (up to 117-fold) pathogenic or beneficial bacteria from human urine and fecal samples, including species that are hard to culture or of low abundance. We assessed 4,601 bacterial strains with mapped methylomes so far and showed broad applicability of mEnrich-seq. mEnrich-seq provides microbiome researchers with a versatile and cost-effective approach for selective sequencing of diverse taxa of interest.

[Source](https://www.nature.com/articles/s41592-023-02125-1)

10.1038/s41592-023-02125-1

---

## Host genetic determinants drive compartment-specific assembly of tea plant microbiomes.
 01.11.2022, _X Tan, H Xie, J Yu, Y Wang, J Xu, P Xu, B Ma_


_Soil Microbiology, Camellia sinensis, Genome-Wide Association Study, Bacteria, Plant Roots, Microbiota, Tea_
Diverse host factors drive microbial variation in plant-associated environments, whereas their genetic mechanisms remain largely unexplored. To address this, we coupled the analyses of plant genetics and microbiomes in this study. Using 100 tea plant (Camellia sinensis) cultivars, the microbiomes of rhizosphere, root endosphere and phyllosphere showed clear compartment-specific assembly, whereas the subpopulation differentiation of tea cultivars exhibited small effects on microbial variation in each compartment. Through microbiome genome-wide association studies, we examined the interactions between tea genetic loci and microbial variation. Notably, genes related to the cell wall and carbon catabolism were heavily linked to root endosphere microbial composition, whereas genes related to the metabolism of metal ions and small organic molecules were overrepresented in association with rhizosphere microbial composition. Moreover, a set of tea genetic variants, including the cytoskeleton-related formin homology interacting protein 1 gene, were strongly associated with the β-diversity of phyllosphere microbiomes, implying their interactions with the overall structure of microbial communities. Our results create a catalogue of tea genetic determinants interacting with microbiomes and reveal the compartment-specific microbiome assembly driven by host genetics.

[Source](https://www.biorxiv.org/content/10.1101/2023.12.27.573469v1)

10.1111/pbi.13897

---

## A catalogue of small proteins from the global microbiome 

[Source](https://www.biorxiv.org/content/10.1101/2023.12.27.573469v1)

---

## https://t.co/otRNegCrYR

[Source](https://t.co/otRNegCrYR)

---

## Mycoplasma pneumoniae epidemic in Denmark, October to December, 2023.
 12.01.2024, _AC Nordholm, B Søborg, P Jokelainen, K Lauenborg Møller, L Flink Sørensen, T Grove Krause, S Anker Uldum, HD Emborg_


We report a surge of patients, especially children and adolescents, with respiratory disease caused by Mycoplasma pneumoniae in Denmark since October 2023. While the surge has reached an epidemic level, no impact on hospital capacity has been observed; only 14% (446/3,195) of cases, primarily adults, required hospitalisation. Macrolide resistance was detected in less than 2% of samples tested. Timely monitoring of hospitalisations linked to M. pneumoniae infections has been established to inform the healthcare system, decisionmakers and the public.

[Source](https://twitter.com/BigDataBiology/status/1740567854985203910?ref_src=twsrc%5Etfw)

10.2807/1560-7917.ES.2024.29.2.2300707

---

## December 29, 2023

[Source](https://twitter.com/BigDataBiology/status/1740567854985203910?ref_src=twsrc%5Etfw)

---

