## LEGO-based microscope

[Source](https://github.com/tobetz/LegoMicroscope)

---

## Microbiome meta-analysis and cross-disease comparison enabled by the SIAMCAT machine learning toolbox.
 03.04.2021, _J Wirbel, K Zych, M Essex, N Karcher, E Kartal, G Salazar, P Bork, S Sunagawa, G Zeller_


The human microbiome is increasingly mined for diagnostic and therapeutic biomarkers using machine learning (ML). However, metagenomics-specific software is scarce, and overoptimistic evaluation and limited cross-study generalization are prevailing issues. To address these, we developed SIAMCAT, a versatile R toolbox for ML-based comparative metagenomics. We demonstrate its capabilities in a meta-analysis of fecal metagenomic studies (10,803 samples). When naively transferred across studies, ML models lost accuracy and disease specificity, which could however be resolved by a novel training set augmentation strategy. This reveals some biomarkers to be disease-specific, with others shared across multiple conditions. SIAMCAT is freely available from siamcat.embl.de .

[Source](https://siamcat.embl.de)

10.1186/s13059-021-02306-1

---

## SIAMCAT

[Source](https://siamcat.embl.de)

---

## Prevalence of Bacillus cereus bacteriophages in fermented foods and characterization of phage JBP901.
 03.10.2011, _H Shin, N Bandara, E Shin, S Ryu, KP Kim_


_Bacillus Phages, Bacillus cereus, Fermentation, Food, Food Microbiology, Genome, Viral, Korea, Myoviridae_
Bacillus cereus is widely distributed in the environment and is responsible for foodborne illnesses, causing diarrhea and vomiting. In this study, the prevalence of B. cereus bacteriophages in Korean fermented foods was studied using 6 reference strains, and JBP901 was further characterized. In a total of 47 samples, 19 (40.4%) contained more than one kind of B. cereus phage. Fourteen phages were purified and determined for host lysis profile. Among them, JBP901 was specific to B. cereus group strains (forming plaques in 13 of 14 isolates) and not able to lyse other tested Gram-positive and Gram-negative strains including Bacillus subtilis. Transmission electron microscopy analysis suggested that JBP901 is a member of the family Myoviridae. Restriction enzyme digestion pattern analysis and one-step growth study indicated a genome size of approximately 150 kbp and a burst size of 100. Furthermore, bacterial lysis studies revealed that JBP901 could be used to control the growth of B. cereus in liquid culture and in food. This is the first report to examine the prevalence of phages in fermented foods in Korea, and suggests that fermented foods are useful sources of B. cereus phage isolation; moreover, B. cereus phages could provide an additional tool to tackle problems associated with B. cereus contamination.

[Source](https://github.com/ellinium/plaque_size_tool)

10.1016/j.resmic.2011.07.001

---

## bacteriophage plaques size tool

[Source](https://github.com/ellinium/plaque_size_tool)

---

## In situ novel environment assay reveals acoustic exploration as a repeatable behavioral response in migratory bats.
 18.04.2021, _T Schabacker, O Lindecke, S Rizzi, L Marggraf, G Pētersons, CC Voigt, L Snijders_


Integrating information on species-specific sensory perception with spatial activity provides a high-resolution understanding of how animals explore environments, yet frequently used exploration assays commonly ignore sensory acquisition as a measure for exploration. Echolocation is an active sensing system used by hundreds of mammal species, primarily bats. As echolocation call activity can be reliably quantified, bats present an excellent model system to investigate intraspecific variation in environmental cue sampling. Here, we developed an in situ roost-like novel environment assay for tree-roosting bats. We repeatedly tested 52 individuals of the migratory bat species, Pipistrellus nathusii, across 24 h, to examine the role of echolocation when crawling through a maze-type arena and test for consistent intraspecific variation in sensory-based exploration. We reveal a strong correlation between echolocation call activity and spatial activity. Moreover, we show that during the exploration of the maze, individuals consistently differed in spatial activity as well as echolocation call activity, given their spatial activity, a behavioral response we term 'acoustic exploration'. Acoustic exploration was correlated with other exploratory behaviors, but not with emergence latency. We here present a relevant new measure for exploration behavior and provide evidence for consistent (short-term) intra-specific variation in the level at which wild bats collect information from a novel environment.

[Source](https://nsurp.org/)

10.1038/s41598-021-87588-y

---

## <strong>OPEN #NSURP2021 CALL</strong>

[Source](https://nsurp.org/)

---

## Microbiome research in general and business newspapers: How many microbiome articles are published and which study designs make the news the most?
 12.04.2021, _A Prados-Bo, G Casino_


The microbiome is a matter of interest for science, consumers and business. Our objective is to quantify that interest in academic journals and newspapers, both quantitatively and by study design. We calculated the number of articles on the microbiome from the total number of biomedicine articles featured in both PubMed and Spanish science news agency SINC, from 2008 to 2018. We used the Factiva database to identify news stories on microbiome papers in three general newspapers (The New York Times, The Times and El País) and three business newspapers (The Wall Street Journal, the Financial Times and Expansión), from 2007 to 2019. Then, we compared news stories with microbiome papers in PubMed, while also analyzing the frequencies of five study design types, both in the newspapers and in the papers themselves. Microbiome papers represented 0.8% of biomedicine papers in PubMed from 2008 to 2018 (increasing from 0.4% to 1.4%), while microbiome news published by SINC represented 1.6% of total biomedical news stories during the same period (increasing from 0.2% to 2.2%). The number of news stories on microbiome papers correlated with the number of microbiome papers (0.91, p &lt; 0.001) featured in general newspapers, but not in business ones. News stories on microbiome papers represented 78.9% and 42.7% of all microbiome articles in general and business newspapers, respectively. Both media outlet types tended to over-report observational studies in humans while under-reporting environmental studies, while the representation of systematic reviews of randomized controlled trials, randomized controlled trials and animal/laboratory studies was similar when comparing newspapers and PubMed. The microbiome is receiving increasing attention in academic journals and newspapers. News stories on the microbiome in general and business newspapers are mostly based on research findings and are more interested in observational studies in humans and less in environmental studies compared to PubMed.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0249835)

10.1371/journal.pone.0249835

---

## Prokaryotic taxonomy and nomenclature in the age of big sequence data.
 07.04.2021, _P Hugenholtz, M Chuvochina, A Oren, DH Parks, RM Soo_


The classification of life forms into a hierarchical system (taxonomy) and the application of names to this hierarchy (nomenclature) is at a turning point in microbiology. The unprecedented availability of genome sequences means that a taxonomy can be built upon a comprehensive evolutionary framework, a longstanding goal of taxonomists. However, there is resistance to adopting a single framework to preserve taxonomic freedom, and ever increasing numbers of genomes derived from uncultured prokaryotes threaten to overwhelm current nomenclatural practices, which are based on characterised isolates. The challenge ahead then is to reach a consensus on the taxonomic framework and to adapt and scale the existing nomenclatural code, or create a new code, to systematically incorporate uncultured taxa into the chosen framework.

[Source](https://www.nature.com/articles/s41396-021-00941-x)

10.1038/s41396-021-00941-x

---

## The metabolic network of the last bacterial common ancestor.
 17.04.2021, _JC Xavier, RE Gerhards, JLE Wimmer, J Brueckner, FDK Tria, WF Martin_


Bacteria are the most abundant cells on Earth. They are generally regarded as ancient, but due to striking diversity in their metabolic capacities and widespread lateral gene transfer, the physiology of the first bacteria is unknown. From 1089 reference genomes of bacterial anaerobes, we identified 146 protein families that trace to the last bacterial common ancestor, LBCA, and form the conserved predicted core of its metabolic network, which requires only nine genes to encompass all universal metabolites. Our results indicate that LBCA performed gluconeogenesis towards cell wall synthesis, and had numerous RNA modifications and multifunctional enzymes that permitted life with low gene content. In accordance with recent findings for LUCA and LACA, analyses of thousands of individual gene trees indicate that LBCA was rod-shaped and the first lineage to diverge from the ancestral bacterial stem was most similar to modern Clostridia, followed by other autotrophs that harbor the acetyl-CoA pathway.

[Source](https://www.nature.com/articles/s42003-021-01918-4)

10.1038/s42003-021-01918-4

---

## Designing a high-resolution, LEGO-based microscope for an educational setting

[Source](https://www.biorxiv.org/content/10.1101/2021.04.11.439311v1)

---

## Genome-Based Targeted Sequencing as a Reproducible Microbial Community Profiling Assay.
 08.04.2021, _J Benjamino, B Leopold, D Phillips, MD Adams_


Current sequencing-based methods for profiling microbial communities rely on marker gene (e.g., 16S rRNA) or metagenome shotgun sequencing (mWGS) analysis. We present an approach based on a single-primer extension reaction using a highly multiplexed oligonucleotide probe pool. This approach, termed MA-GenTA (microbial abundances from genome tagged analysis), enables quantitative, straightforward, cost-effective microbiome profiling that combines desirable features of both 16S rRNA and mWGS strategies. The use of multiple probes per target genome and rigorous probe design criteria enabled robust determination of relative abundance. To test the utility of the MA-GenTA assay, probes were designed for 830 genome sequences representing bacteria present in mouse stool specimens. Comparison of the MA-GenTA data with mWGS data demonstrated excellent correlation down to 0.01% relative abundance and a similar number of organisms detected per sample. Despite the incompleteness of the reference database, nonmetric multidimensional scaling (NMDS) clustering based on the Bray-Curtis dissimilarity metric of sample groups was consistent between MA-GenTA, mWGS, and 16S rRNA data sets. MA-GenTA represents a potentially useful new method for microbiome community profiling based on reference genomes.IMPORTANCE New methods for profiling the microbial communities can create new approaches to understanding the composition and function of those communities. In this study, we combined bacterial genome-specific probe design with a highly multiplexed single primer extension reaction as a new method to profile microbial communities, using stool from various mouse strains as a test case. This method, termed MA-GenTA, was benchmarked against 16S rRNA gene sequencing and metagenome sequencing methods and delivered similar relative abundance and clustering data. Since the probes were generated from reference genomes, MA-GenTA was also able to provide functional pathway data for the stool microbiome in the assayed samples. The method is more informative than 16S rRNA analysis while being less costly than metagenome shotgun sequencing.

[Source](https://msphere.asm.org/content/msph/6/2/e01325-20.full.pdf)

10.1128/mSphere.01325-20

---

## Microbial ACBP/DBI-like genes are rare in the human gut microbiome and show no links with obesity.
 10.04.2021, _AM Thomas, F Asnicar, G Kroemer, N Segata_


Acyl coenzyme A (CoA) binding protein (ACBP), also called diazepam-binding inhibitor (DBI) is a phylogenetically conserved protein that is expressed by all eukaryotic species as well as by some bacteria. Since elevated ACBP/DBI levels play a major role in the inhibition of autophagy, increase in appetite and lipoanabolism that accompany obesity, we wondered whether ACBP/DBI produced by the human microbiome might affect host weight. We found that the genomes of bacterial commensals rarely contain ACBP/DBI homologues, which are rather encoded by genomes of some pathogenic or environmental taxa that were not prevalent in human feces. Exhaustive bioinformatic analyses of 1,899 gut samples from healthy individuals refuted the hypothesis that bacterial ACBP/DBI might affect the BMI in a physiological context. Thus, the physiological regulation of BMI is unlikely to be affected by microbial ACBP/DBI-like proteins. However, at the speculative level, it remains possible that ACBP/DBI produced by potential pathogenic bacteria might enhance their virulence by inhibiting autophagy and hence subverting innate immune responses.ImportanceAcyl coenzyme A (CoA) binding protein (ACBP) can be encoded by several organisms across the domains of life, including microbes, and has shown to play major roles in human metabolic processes. However, little is known about its presence in the human gut microbiome and whether its microbial counterpart could also play a role in human metabolism. In the present study, we found that microbial ACBP/DBI sequences were rarely present in the gut microbiome across multiple metagenomic datasets. Microbes that carried ACBP/DBI in the human gut microbiome included Saccharomyces cerevisiae, Lautropia mirabilis and Comamonas kerstersii, but these microorganisms were not associated with body-mass index, further indicating an unconvincing role for microbial ACBP/DBI in human metabolism.

[Source](https://aem.asm.org/content/early/2021/04/05/AEM.00471-21)

10.1128/AEM.00471-21

---

## The long-term genetic stability and individual specificity of the human gut microbiome

[Source](https://www.cell.com/cell/fulltext/S0092-8674(21)00352-4)

---

## Emulating the gut–liver axis: Dissecting the microbiome’s effect on drug metabolism using multiorgan-on-chip models

[Source](https://www.sciencedirect.com/science/article/pii/S2451965021000223)

---

## The human urobiome

[Source](https://link.springer.com/article/10.1007/s00335-021-09862-8)

---

## Urinary microbiota of women with recurrent urinary tract infection: collection and culture methods

[Source](https://link.springer.com/article/10.1007%2Fs00192-021-04780-4)

---

## Characterization of the first vaginal <em>Lactobacillus crispatus </em>genomes isolated in Brazil

[Source](https://peerj.com/articles/11079/)

---

## Comparative Genome Analyses of <em>Lactobacillus crispatus</em> Isolates from Different Ecological Niches Reveal an Adaptation of This Species to the Human Vaginal Environment

[Source](https://aem.asm.org/content/aem/87/8/e02899-20.full.pdf)

---

## <em>Lacticaseibacillus rhamnosus</em> GR-1, a.k.a. <em>Lactobacillus rhamnosus</em> GR-1: Past and Future Perspectives

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(21)00069-X)

---

## Probiotic<em> Lactobacillu</em>s sp. Strains Inhibit Growth, Adhesion, Biofilm Formation, and Gene Expression of Bacterial Vaginosis-Inducing <em>Gardnerella vaginalis</em>

[Source](https://www.mdpi.com/2076-2607/9/4/728)

---

## Dental biofilm microbiota dysbiosis is associated with the risk of acute graft-versus-host disease after allogeneic hematopoietic stem cell transplantation

[Source](https://www.medrxiv.org/content/10.1101/2021.02.04.21251019v2.full.pdf)

---

## Shotgun metagenomics reveals distinct functional diversity and metabolic capabilities between 12 000-year-old permafrost and active layers on Muot da Barba Peider (Swiss Alps).
 13.04.2021, _C Perez-Mon, W Qi, S Vikram, A Frossard, T Makhalanyane, D Cowan, B Frey_


The warming-induced thawing of permafrost promotes microbial activity, often resulting in enhanced greenhouse gas emissions. The ability of permafrost microorganisms to survive the in situ sub-zero temperatures, their energetic strategies and their metabolic versatility in using soil organic materials determine their growth and functionality upon thawing. Hence, functional characterization of the permafrost microbiome, particularly in the underexplored mid-latitudinal alpine regions, is a crucial first step in predicting its responses to the changing climate, and the consequences for soil-climate feedbacks. In this study, for the first time, the functional potential and metabolic capabilities of a temperate mountain permafrost microbiome from central Europe has been analysed using shotgun metagenomics. Permafrost and active layers from the summit of Muot da Barba Peider (MBP) [Swiss Alps, 2979 m above sea level (a.s.l.)] revealed a strikingly high functional diversity in the permafrost (north-facing soils at a depth of 160 cm). Permafrost metagenomes were enriched in stress-response genes (e.g. cold-shock genes, chaperones), as well as in genes involved in cell defence and competition (e.g. antiviral proteins, antibiotics, motility, nutrient-uptake ABC transporters), compared with active-layer metagenomes. Permafrost also showed a higher potential for the synthesis of carbohydrate-active enzymes, and an overrepresentation of genes involved in fermentation, carbon fixation, denitrification and nitrogen reduction reactions. Collectively, these findings demonstrate the potential capabilities of permafrost microorganisms to thrive in cold and oligotrophic conditions, and highlight their metabolic versatility in carbon and nitrogen cycling. Our study provides a first insight into the high functional gene diversity of the central European mountain permafrost microbiome. Our findings extend our understanding of the microbial ecology of permafrost and represent a baseline for future investigations comparing the functional profiles of permafrost microbial communities at different latitudes.

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000558)

10.1099/mgen.0.000558

---

## Plaque Size Tool: an automated plaque analysis tool for simplifying and standardising bacteriophage plaque morphology measurements

[Source](https://www.biorxiv.org/content/10.1101/2021.04.12.439404v1.full.pdf)

---

## The gut virome of healthy children during the first year of life is diverse and dynamic.
 14.04.2021, _B Taboada, P Morán, A Serrano-Vázquez, P Iša, L Rojas-Velázquez, H Pérez-Juárez, S López, J Torres, C Ximenez, CF Arias_


In this work, we determined the diversity and dynamics of the gut virome of infants during the first year of life. Fecal samples were collected monthly, from birth to one year of age, from three healthy children living in a semi-rural village in Mexico. Most of the viral reads were classified into six families of bacteriophages including five dsDNA virus families of the order Caudovirales, with Siphoviridae and Podoviridae being the most abundant. Eukaryotic viruses were detected as early as two weeks after birth and remained present all along the first year of life. Thirty-four different eukaryotic virus families were found, where eight of these families accounted for 98% of all eukaryotic viral reads: Anelloviridae, Astroviridae, Caliciviridae, Genomoviridae, Parvoviridae, Picornaviridae, Reoviridae and the plant-infecting viruses of the Virgaviridae family. Some viruses in these families are known human pathogens, and it is surprising that they were found during the first year of life in infants without gastrointestinal symptoms. The eukaryotic virus species richness found in this work was higher than that observed in previous studies; on average between 7 and 24 virus species were identified per sample. The richness and abundance of the eukaryotic virome significantly increased during the second semester of life, probably because of an increased environmental exposure of infants with age. Our findings suggest an early and permanent contact of infants with a diverse array of bacteriophages and eukaryotic viruses, whose composition changes over time. The bacteriophages and eukaryotic viruses found in these children could represent a metastable virome, whose potential influence on the development of the infant's immune system or on the health of the infants later in life, remains to be investigated.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0240958)

10.1371/journal.pone.0240958

---

## Taxonomic analysis of metagenomic data with kASA.
 30.03.2021, _S Weging, A Gogol-Döring, I Grosse_


The taxonomic analysis of sequencing data has become important in many areas of life sciences. However, currently available tools for that purpose either consume large amounts of RAM or yield insufficient quality and robustness. Here, we present kASA, a k-mer based tool capable of identifying and profiling metagenomic DNA or protein sequences with high computational efficiency and a user-definable memory footprint. We ensure both high sensitivity and precision by using an amino acid-like encoding of k-mers together with a range of multiple k's. Custom algorithms and data structures optimized for external memory storage enable a full-scale taxonomic analysis without compromise on laptop, desktop, and HPCC.

[Source](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkab200/6204649)

10.1093/nar/gkab200

---

## Microbiome meta-analysis and cross-disease comparison enabled by the SIAMCAT machine learning toolbox.
 03.04.2021, _J Wirbel, K Zych, M Essex, N Karcher, E Kartal, G Salazar, P Bork, S Sunagawa, G Zeller_


The human microbiome is increasingly mined for diagnostic and therapeutic biomarkers using machine learning (ML). However, metagenomics-specific software is scarce, and overoptimistic evaluation and limited cross-study generalization are prevailing issues. To address these, we developed SIAMCAT, a versatile R toolbox for ML-based comparative metagenomics. We demonstrate its capabilities in a meta-analysis of fecal metagenomic studies (10,803 samples). When naively transferred across studies, ML models lost accuracy and disease specificity, which could however be resolved by a novel training set augmentation strategy. This reveals some biomarkers to be disease-specific, with others shared across multiple conditions. SIAMCAT is freely available from siamcat.embl.de .

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02306-1)

10.1186/s13059-021-02306-1

---

## Integrating pan-genome with metagenome for microbial community profiling.
 13.04.2021, _C Zhong, C Chen, L Wang, K Ning_


Advances in sequencing technology have led to the increased availability of genomes and metagenomes, which has greatly facilitated microbial pan-genome and metagenome analysis in the community. In line with this trend, studies on microbial genomes and phenotypes have gradually shifted from individuals to environmental communities. Pan-genomics and metagenomics are powerful strategies for in-depth profiling study of microbial communities. Pan-genomics focuses on genetic diversity, dynamics, and phylogeny at the multi-genome level, while metagenomics profiles the distribution and function of culture-free microbial communities in special environments. Combining pan-genome and metagenome analysis can reveal the microbial complicated connections from an individual complete genome to a mixture of genomes, thereby extending the catalog of traditional individual genomic profile to community microbial profile. Therefore, the combination of pan-genome and metagenome approaches has become a promising method to track the sources of various microbes and decipher the population-level evolution and ecosystem functions. This review summarized the pan-genome and metagenome approaches, the combined strategies of pan-genome and metagenome, and applications of these combined strategies in studies of microbial dynamics, evolution, and function in communities. We discussed emerging strategies for the study of microbial communities that integrate information in both pan-genome and metagenome. We emphasized studies in which the integrating pan-genome with metagenome approach improved the understanding of models of microbial community profiles, both structural and functional. Finally, we illustrated future perspectives of microbial community profile: more advanced analytical techniques, including big-data based artificial intelligence, will lead to an even better understanding of the patterns of microbial communities.

[Source](https://www.sciencedirect.com/science/article/pii/S2001037021000726)

10.1016/j.csbj.2021.02.021

---

## Connecting structure to function with the recovery of over 1000 high-quality metagenome-assembled genomes from activated sludge using long-read sequencing.
 19.04.2021, _CM Singleton, F Petriglieri, JM Kristensen, RH Kirkegaard, TY Michaelsen, MH Andersen, Z Kondrotaite, SM Karst, MS Dueholm, PH Nielsen, M Albertsen_


_Bacteria, Bioreactors, Denmark, Genome, Bacterial, High-Throughput Nucleotide Sequencing, Metagenome, Metagenomics, Microbiota, Phylogeny, RNA, Ribosomal, 16S, RNA, Ribosomal, 23S, RNA, Ribosomal, 5S, Sewage, Waste Water, Water Purification_
Microorganisms play crucial roles in water recycling, pollution removal and resource recovery in the wastewater industry. The structure of these microbial communities is increasingly understood based on 16S rRNA amplicon sequencing data. However, such data cannot be linked to functional potential in the absence of high-quality metagenome-assembled genomes (MAGs) for nearly all species. Here, we use long-read and short-read sequencing to recover 1083 high-quality MAGs, including 57 closed circular genomes, from 23 Danish full-scale wastewater treatment plants. The MAGs account for ~30% of the community based on relative abundance, and meet the stringent MIMAG high-quality draft requirements including full-length rRNA genes. We use the information provided by these MAGs in combination with &gt;13 years of 16S rRNA amplicon sequencing data, as well as Raman microspectroscopy and fluorescence in situ hybridisation, to uncover abundant undescribed lineages belonging to important functional groups.

[Source](https://www.nature.com/articles/s41467-021-22203-2)

10.1038/s41467-021-22203-2

---

## Evaluation of the effects of library preparation procedure and sample characteristics on the accuracy of metagenomic profiles 

[Source](https://www.biorxiv.org/content/10.1101/2021.04.12.439578v1.full.pdf)

---

## Democratising deep learning for microscopy with ZeroCostDL4Mic.
 16.04.2021, _L von Chamier, RF Laine, J Jukkala, C Spahn, D Krentzel, E Nehme, M Lerche, S Hernández-Pérez, PK Mattila, E Karinou, S Holden, AC Solak, A Krull, TO Buchholz, ML Jones, LA Royer, C Leterrier, Y Shechtman, F Jug, M Heilemann, G Jacquemet, R Henriques_


Deep Learning (DL) methods are powerful analytical tools for microscopy and can outperform conventional image processing pipelines. Despite the enthusiasm and innovations fuelled by DL technology, the need to access powerful and compatible resources to train DL networks leads to an accessibility barrier that novice users often find difficult to overcome. Here, we present ZeroCostDL4Mic, an entry-level platform simplifying DL access by leveraging the free, cloud-based computational resources of Google Colab. ZeroCostDL4Mic allows researchers with no coding expertise to train and apply key DL networks to perform tasks including segmentation (using U-Net and StarDist), object detection (using YOLOv2), denoising (using CARE and Noise2Void), super-resolution microscopy (using Deep-STORM), and image-to-image translation (using Label-free prediction - fnet, pix2pix and CycleGAN). Importantly, we provide suitable quantitative tools for each network to evaluate model performance, allowing model optimisation. We demonstrate the application of the platform to study multiple biological processes.

[Source](https://www.nature.com/articles/s41467-021-22518-0)

10.1038/s41467-021-22518-0

---

## Derivation of Induced Pluripotent Stem Cell (iPSC) Lines from Patient-Specific Peripheral Blood Mononuclear Cells (PBMC) Using Episomal Vectors.
 27.03.2021, _VBR Konala, S Nandakumar, H Surendran, R Pal_


Inherited retinal diseases (IRDs) are a diverse group of rare eye disorders, resulting in vision loss or blindness. The underlying reason is mutation in one or more than 250 different genes associated with the development and normal physiology of retina largely comprising of rod/cone photoreceptors and retinal pigment epithelium. Interestingly, the sub retinal region of an eye has been shown to be immune privileged, broadening the scope of cell-replacement therapies for patients suffering from retinal degeneration. Several groups around the globe, including ours, have demonstrated safety and efficacy in preclinical studies by employing various approaches of retinal cell therapy. This had largely been possible with the advent of induced pluripotent stem cells (iPSC)-reprogrammed from adult somatic cells, that serves as a starting material for generating retinal cells de novo. Here, we describe a detailed procedure for reprogramming peripheral blood mononuclear cells (PBMC) into iPSC using episomal vectors without any physical disruption in the host genome. The lines thus created were tested for sterility, cytogenetic stability, identity, absence of episomal plasmids and further authenticated for pluripotency and tri-lineage differentiation capacity by embryoid body formation and immunocytochemistry. We believe that this feeder-cell free, animal-product free and gene-insertion free protocol would help people to develop and bank patient-specific cell lines for autologous cell therapies for incurable rare diseases.

[Source](http://www.plasmidbiologysociety.org/plasmids-around-the-globe-2021/)

10.1007/7651_2021_385

---

## Plasmids around the globe

[Source](http://www.plasmidbiologysociety.org/plasmids-around-the-globe-2021/)

---

## Spring 2021 Microbiome Informatics webinars

[Source](https://u.osu.edu/coms/webinar-series/)

---

## EMBARK Spring Webinars

[Source](https://antimicrobialresistance.eu/2021/03/26/embark-spring-webinars/)

---

## BISMiS Live

[Source](https://www.bismis.net/)

---

## here

[Source](https://us02web.zoom.us/webinar/register/WN__5Y1ZI5HTy-xEO0sBnXK0Q)

---

## Berkeley Lab Live Science

[Source](https://www.eventbrite.com/e/berkeley-lab-live-science-exploring-earth-tickets-150785558665)

---

## ISAPP 2021 Virtual Annual Meeting 

[Source](https://isappscience.org/for-scientists/annual-meeting/2021-annual-meeting-open-registration/)

---

## Open Online R Stream

[Source](https://www.wvbauer.com/doku.php/live_streams)

---

## RaukR: Advanced R for Bioinformatics Summer School

[Source](https://nbisweden.github.io/workshop-RaukR-2106/)

---

