## The Epigenetics of Food Allergy.
 08.07.2020, _C Chang, H Wu, Q Lu_


_Epigenesis, Genetic, Epigenomics, Food Hypersensitivity, Humans, Immune Tolerance_
Food allergy is a global health problem, particularly in developed countries. It is mainly mediated by Th2 cell and IgE produced by B cells. While the pathogenesis of IgE-mediated food allergy is quite straightforward, the factors that lead to the development of food allergies at any age in children and adults are unclear. Recent studies have revealed that genetics, epigenetics, and environmental exposures contribute to the development of atopy. In this chapter, we discuss the interplay between these three key elements, reveal how epigenetic modifications may mediate genetic susceptibility of food allergies, and explain why epigenetic modifications may be the key in environmental factors mediated-gene expression, leading to the loss of immune tolerance and eventually, the initiation of food allergies. It should be noted that the study of the role of epigenetics in food allergy is still in its infancy, and lags behind research on epigenetics in other fields such as cancer and autoimmune diseases. One of the reasons for this may be the extreme complexity and variability of clinical presentation of food allergy, ranging from less severe forms such as oral allergy syndrome to full-blown anaphylaxis. Research on early exposure has disrupted the previous thinking of avoidance of food allergies to prevent sensitization in children, instead leading to recommendations that early introduction to foods may, in fact, induce tolerance. However, clear and unequivocal guidelines on how to approach this in the clinical setting have not been developed. The coming of the epigenetic era in food allergies is to provide better understanding of pathogenesis of food allergy, as well as providing therapeutic and preventive strategies for this very common condition.

[Source](https://www.mdpi.com/journal/cells/special_issues/Microbiome_Cells)

10.1007/978-981-15-3449-2_5

---

## Interplay of Gene Expression Regulation and Microbiome

[Source](https://www.mdpi.com/journal/cells/special_issues/Microbiome_Cells)

---

## Anna Karenina and the subgingival microbiome associated with periodontitis

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01056-3)

---

## Irisin ameliorates endoplasmic reticulum stress and liver fibrosis through inhibiting PERK-mediated destabilization of HNRNPA1 in hepatic stellate cells.
 05.05.2021, _X Liao, W Zhan, R Li, T Tian, L Yu, Q Yang_


Liver fibrosis is a common consequence of chronic liver diseases involved with the activation of hepatic stellate cells (HSCs) and endoplasmic reticulum (ER) stress. Irisin is a small polypeptide hormone that shows beneficial effects on metabolic disorders. The current study aimed to investigate the biological function of irisin on hepatic fibrosis. A mouse model of carbon tetrachloride (CCl4)-induced hepatic fibrosis was established. CCl4-treated mice showed elevated serum levels of AST and ALT, increased collagen accumulation, induced ER stress, and upregulated expressions of pro-fibrotic proteins in the liver compared to the controls. The administration of irisin, however, ameliorated CCl4-induced hepatic fibrosis in both cultured HSCs and mice. PKR-like ER kinase (PERK) is a key component of the ER stress-associated signaling pathway. We found that irisin treatment improved the stability of heterogeneous nuclear ribonucleoprotein A1 (HNRNPA1) via regulating the phosphorylation of PERK in mouse livers and isolated HSCs. Also, the knockdown of HNRNPA1 eliminated the hepatoprotective effects of irisin on hepatic fibrosis and ER stress. In summary, this study showed that irisin alleviated ER stress and hepatic fibrosis by inhibiting PERK-mediated HNRNPA1 destabilization, suggesting that irisin may represent a promising therapeutic strategy for patients with liver fibrosis.

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2021.650893/full)

10.1515/hsz-2020-0251

---

## The Association Between Intestinal Bacteria and Allergic Diseases—Cause or Consequence?

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2021.650893/full)

---

## Exploring the Modulatory Effects of Gut Microbiota in Anti-Cancer Therapy.
 02.05.2021, _W Li, X Deng, T Chen_


In the recent decade, gut microbiota has received growing interest due to its role in human health and disease. On the one hand, by utilizing the signaling pathways of the host and interacting with the immune system, the gut microbiota is able to maintain the homeostasis in human body. This important role is mainly modulated by the composition of microbiota, as a normal microbiota composition is responsible for maintaining the homeostasis of human body, while an altered microbiota profile could contribute to several pathogenic conditions and may further lead to oncogenesis and tumor progression. Moreover, recent insights have especially focused on the important role of gut microbiota in current anticancer therapies, including chemotherapy, radiotherapy, immunotherapy and surgery. Research findings have indicated a bidirectional interplay between gut microbiota and these therapeutic methods, in which the implementation of different therapeutic methods could lead to different alterations in gut microbiota, and the presence of gut microbiota could in turn contribute to different therapeutic responses. As a result, manipulating the gut microbiota to reduce the therapy-induced toxicity may provide an adjuvant therapy to achieve a better therapeutic outcome. Given the complex role of gut microbiota in cancer treatment, this review summarizes the interactions between gut microbiota and anticancer therapies, and demonstrates the current strategies for reshaping gut microbiota community, aiming to provide possibilities for finding an alternative approach to lower the damage and improve the efficacy of cancer therapy.

[Source](https://www.frontiersin.org/articles/10.3389/fonc.2021.644454/full)

10.3389/fonc.2021.644454

---

## The interplay between Helicobacter pylori and gastrointestinal microbiota.
 03.05.2021, _CC Chen, JM Liou, YC Lee, TC Hong, EM El-Omar, MS Wu_


The complex population of microbes in the human gastrointestinal (GI) tract interacts with itself and with the host, exerting a deep influence on health and disease development. The development of modern sequencing technology has enabled us to gain insight into GI microbes. Helicobacter pylori colonization significantly affects the gastric microenvironment, which in turn affects gastric microbiota and may be correlated with colonic microbiota changes. Crosstalk between H. pylori and GI commensal flora may play a role in H. pylori-related carcinogenicity and extragastric manifestations. We review current knowledge on how H. pylori shapes GI microbiota with a specific focus on its impact on the stomach and colon. We also review current evidence on colonic microbiota changes attributed to eradication therapy based on the clinical studies performed to date.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2021.1909459)

10.1080/19490976.2021.1909459

---

## Do Antibiotics Cause Obesity Through Long-term Alterations in the Gut Microbiome? A Review of Current Evidence.
 04.05.2021, _N Vallianou, M Dalamaga, T Stratigou, I Karampela, C Tsigalou_


In this review, we summarize current evidence on the association between antibiotics and the subsequent development of obesity through modulation of the gut microbiome. Particular emphasis is given on (i) animal and human studies and their limitations; (ii) the reservoir of antibiotics in animal feed, emerging antibiotic resistance, gut dysbiosis, and obesity; (iii) the role of infections, specifically viral infections, as a cause of obesity; and (iv) the potential therapeutic approaches other than antibiotics to modulate gut microbiome.

[Source](https://link.springer.com/article/10.1007%2Fs13679-021-00438-w)

10.1007/s13679-021-00438-w

---

## Gut microbiota contributes to the development of hypertension in a genetic mouse model of systemic lupus erythematosus

[Source](https://bpspubs.onlinelibrary.wiley.com/doi/10.1111/bph.15512)

---

## Infant gut microbiota modulation by human milk disaccharides in humanized microbiome mice.
 03.05.2021, _A Rubio-Del-Campo, R Gozalbo-Rovira, EM Moya-Gonzálvez, J Alberola, J Rodríguez-Díaz, MJ Yebra_


Human milk glycans present a unique diversity of structures that suggest different mechanisms by which they may affect the infant microbiome development. A humanized mouse model generated by infant fecal transplantation was utilized here to evaluate the impact of fucosyl-α1,3-GlcNAc (3FN), fucosyl-α1,6-GlcNAc, lacto-N-biose (LNB) and galacto-N-biose on the fecal microbiota and host-microbiota interactions. 16S rRNA amplicon sequencing showed that certain bacterial genera significantly increased (Ruminococcus and Oscillospira) or decreased (Eubacterium and Clostridium) in all disaccharide-supplemented groups. Interestingly, cluster analysis differentiates the consumption of fucosyl-oligosaccharides from galactosyl-oligosaccharides, highlighting the disappearance of Akkermansia genus in both fucosyl-oligosaccharides. An increment of the relative abundance of Coprococcus genus was only observed with 3FN. As well, LNB significantly increased the relative abundance of Bifidobacterium, whereas the absolute levels of this genus, as measured by quantitative real-time PCR, did not significantly increase. OTUs corresponding to the species Bifidobacterium longum, Bifidobacterium adolescentis and Ruminococcus gnavus were not present in the control after the 3-week intervention, but were shared among the donor and specific disaccharide groups, indicating that their survival is dependent on disaccharide supplementation. The 3FN-feeding group showed increased levels of butyrate and acetate in the colon, and decreased levels of serum HDL-cholesterol. 3FN also down-regulated the pro-inflammatory cytokine TNF-α and up-regulated the anti-inflammatory cytokines IL-10 and IL-13, and the Toll-like receptor 2 in the large intestine tissue. The present study revealed that the four disaccharides show efficacy in producing beneficial compositional shifts of the gut microbiota and in addition, the 3FN demonstrated physiological and immunomodulatory roles.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2021.1914377)

10.1080/19490976.2021.1914377

---

## Description and comparison of the skin and ear canal microbiota of non-allergic and allergic German shepherd dogs using next generation sequencing.
 03.05.2021, _N Apostolopoulos, SP Glaeser, R Bagwe, S Janssen, U Mayer, C Ewers, P Kämpfer, R Neiger, N Thom_


Atopic dermatitis is one of the most common skin diseases in dogs. Pathogenesis is complex and incompletely understood. Skin colonizing bacteria likely play an important role in the severity of this disease. Studying the canine skin microbiota using traditional microbiological methods has many limitations which can be overcome by molecular procedures. The aim of this study was to describe the bacterial microbiota of the skin and ear canals of healthy non-allergic and allergic German shepherd dogs (GSDs) without acute flare or concurrent skin infection and to compare both. Bacterial 16S rRNA gene amplicon sequence data revealed no differences of bacterial community patterns between the different body sites (axilla, front dorsal interdigital skin, groin, and ear canals) in non-allergic dogs. The microbiota at the different body sites of non-allergic GSDs showed no significant differences. Only for the samples obtained from the axilla the bacterial microbiota of allergic dogs was characterized by a lower species richness compared to that of non-allergic dogs and the bacterial community composition of the skin and ear canals of allergic dogs showed body site specific differences compared to non-allergic dogs. Actinobacteria was the most abundant phylum identified from the non-allergic dogs and Proteobacteria from allergic dogs. Macrococcus spp. were more abundant on non-allergic skin while Sphingomonas spp. were more abundant on the allergic skin. Forward step redundancy analysis of metadata indicated that the household the dogs came from had the strongest impact on the composition of the skin microbiome followed by sex, host health status and body site.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0250695)

10.1371/journal.pone.0250695

---

## Integrating taxonomic, functional, and strain-level profiling of diverse microbial communities with bioBakery 3.
 05.05.2021, _F Beghini, LJ McIver, A Blanco-Míguez, L Dubois, F Asnicar, S Maharjan, A Mailyan, P Manghi, M Scholz, AM Thomas, M Valles-Colomer, G Weingart, Y Zhang, M Zolfo, C Huttenhower, EA Franzosa, N Segata_


Culture-independent analyses of microbial communities have progressed dramatically in the last decade, particularly due to advances in methods for biological profiling via shotgun metagenomics. Opportunities for improvement continue to accelerate, with greater access to multi-omics, microbial reference genomes, and strain-level diversity. To leverage these, we present bioBakery 3, a set of integrated, improved methods for taxonomic, strain-level, functional, and phylogenetic profiling of metagenomes newly developed to build on the largest set of reference sequences now available. Compared to current alternatives, MetaPhlAn 3 increases the accuracy of taxonomic profiling, and HUMAnN 3 improves that of functional potential and activity. These methods detected novel disease-microbiome links in applications to CRC (1262 metagenomes) and IBD (1635 metagenomes and 817 metatranscriptomes). Strain-level profiling of an additional 4077 metagenomes with StrainPhlAn 3 and PanPhlAn 3 unraveled the phylogenetic and functional structure of the common gut microbe Ruminococcus bromii, previously described by only 15 isolate genomes. With open-source implementations and cloud-deployable reproducible workflows, the bioBakery 3 platform can help researchers deepen the resolution, scale, and accuracy of multi-omic profiling for microbial community studies.

[Source](https://elifesciences.org/articles/65088)

10.7554/eLife.65088

---

## Validation and standardization of DNA extraction and library construction methods for metagenomics-based human fecal microbiome measurements.
 06.05.2021, _DM Tourlousse, K Narita, T Miura, M Sakamoto, A Ohashi, K Shiina, M Matsuda, D Miura, M Shimamura, Y Ohyama, A Yamazoe, Y Uchino, K Kameyama, S Arioka, J Kataoka, T Hisada, K Fujii, S Takahashi, M Kuroiwa, M Rokushima, M Nishiyama, Y Tanaka, T Fuchikami, H Aoki, S Kira, R Koyanagi, T Naito, M Nishiwaki, H Kumagai, M Konda, K Kasahara, M Ohkuma, H Kawasaki, Y Sekiguchi, J Terauchi_


_DNA, Humans, Metagenomics, Microbiota, Reference Standards, Reproducibility of Results, Sequence Analysis, DNA_
Validation and standardization of methodologies for microbial community measurements by high-throughput sequencing are needed to support human microbiome research and its industrialization. This study set out to establish standards-based solutions to improve the accuracy and reproducibility of metagenomics-based microbiome profiling of human fecal samples.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01048-3https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01048-3)

10.1186/s40168-021-01048-3

---

