## Towards systematic evaluation of epidemic responses during humanitarian crises: a scoping review of existing public health evaluation frameworks.
 05.03.2020, _A Warsame, K Blanchet, F Checchi_


Epidemics continue to pose a significant public health threat to populations in low and middle-income countries. However, little is known about the appropriateness and performance of response interventions in such settings. We undertook a rapid scoping review of public health evaluation frameworks for emergency settings in order to judge their suitability for assessing epidemic response. Our search identified a large variety of frameworks. However, very few are suitable for framing the response to an epidemic, or its evaluation. We propose a generic epidemic framework that draws on elements of existing frameworks. We believe that this framework may potentially be of use in closing the gap between increasing global epidemic risk and the ability to respond effectively.

[Source](https://www.biomedcentral.com/collections/globaloutbreaksandresponses)

10.1136/bmjgh-2019-002109

---

## Global Outbreaks and Responses

[Source](https://www.biomedcentral.com/collections/globaloutbreaksandresponses)

---

## Fungal diseases as neglected pathogens: A wake-up call to public health officials.
 28.02.2020, _ML Rodrigues, JD Nosanchuk_



[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0007964)

10.1371/journal.pntd.0007964

---

## Inducible cell-to-cell signaling for tunable dynamics in microbial communities.
 05.03.2020, _A Miano, MJ Liao, J Hasty_


The last decade has seen bacteria at the forefront of biotechnological innovation, with applications including biomolecular computing, living therapeutics, microbiome engineering and microbial factories. These emerging applications are all united by the need to precisely control complex microbial dynamics in spatially extended environments, requiring tools that can bridge the gap between intracellular and population-level coordination. To address this need, we engineer an inducible quorum sensing system which enables precise tunability of bacterial dynamics both at the population and community level. As a proof-of-principle, we demonstrate the advantages of this system when genetically equipped for cargo delivery. In addition, we exploit the absence of cross-talk with respect to the majority of well-characterized quorum sensing systems to demonstrate inducibility of multi-strain communities. More broadly, this work highlights the unexplored potential of remotely inducible quorum sensing systems which, coupled to any gene of interest, may facilitate the translation of circuit designs into applications.

[Source](https://www.nature.com/articles/s41467-020-15056-8)

10.1038/s41467-020-15056-8

---

## The insect antimicrobial peptide cecropin A disrupts uropathogenic Escherichia coli biofilms.
 04.03.2020, _M Kalsy, M Tonk, M Hardt, U Dobrindt, A Zdybicka-Barabas, M Cytrynska, A Vilcinskas, K Mukherjee_


Current antibiotics cannot eradicate uropathogenic Escherichia coli (UPEC) biofilms, leading to recurrent urinary tract infections. Here, we show that the insect antimicrobial peptide cecropin A (CecA) can destroy planktonic and sessile biofilm-forming UPEC cells, either alone or when combined with the antibiotic nalidixic acid (NAL), synergistically clearing infection in vivo without off-target cytotoxicity. The multi-target mechanism of action involves outer membrane permeabilization followed by biofilm disruption triggered by the inhibition of efflux pump activity and interactions with extracellular and intracellular nucleic acids. These diverse targets ensure that resistance to the CecA + NAL combination emerges slowly. The antimicrobial mechanisms of CecA, thus, extend beyond pore-forming activity to include an unanticipated biofilm-eradication process, offering an alternative approach to combat antibiotic-resistant UPEC infections.

[Source](https://www.nature.com/articles/s41522-020-0116-3)

10.1038/s41522-020-0116-3

---

## Using comparative genomics to understand molecular features of carbapenem-resistant Acinetobacter baumannii from South Korea causing invasive infections and their clinical implications.
 28.02.2020, _MH Kim, H Jeong, YM Sim, S Lee, D Yong, CM Ryu, JY Choi_


Acinetobacter baumannii is a highly potent nosocomial pathogen that is associated with increased in-hospital mortality. Here, we investigated the changes in molecular characteristics of carbapenem-resistant A. baumannii (CRAB) isolated from the blood samples of patients admitted to a tertiary hospital in South Korea from January 2009 to July 2015. Whole genome sequencing using the Illumina MiSeq platform and multi-locus sequence typing (MLST) were performed for 98 CRAB clinical isolates. In silico analyses for the prediction of antimicrobial resistance and virulence factor genes were performed. Plasmid sequences, including complete forms, were reconstructed from the sequence reads. Epidemiologic data were collected from the hospital database. MLST using the Oxford scheme revealed 10 sequence types of CRAB, of which ST191 was the dominant type (n = 59). Although blaOXA-23 was shared by most analysed strains, the compositions of antimicrobial resistance determinants differed among sequence types. ST447 and ST451/ST1809 with a few resistance genes were isolated during the later years of the study period. The number of virulence genes increased, while that of ST191 did not change significantly over the investigation period. Intriguingly MLST types, compositions of antimicrobial resistance genes, and virulence genes had no association with clinical outcomes of CRAB bacteraemia. In conclusion, active changes in or accumulations of antimicrobial resistance determinants and virulence genes in CRAB were not observed during the research period. Molecular characteristics of CRAB had no association with clinical outcomes of CRAB bacteraemia.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229416)

10.1371/journal.pone.0229416

---

## Factors associated with sufficient knowledge of antibiotics and antimicrobial resistance in the Japanese general population.
 04.03.2020, _S Tsuzuki, N Fujitsuka, K Horiuchi, S Ijichi, Y Gu, Y Fujitomo, R Takahashi, N Ohmagari_


We conducted two online surveys about antibiotics targeted at the Japanese general population in March 2017 and February 2018. In total, 6,982 participants completed the questionnaire. Factors associated with knowledge of antibiotics, knowledge of antimicrobial resistance (AMR) and appropriate behavioural changes were evaluated by a machine learning approach using DataRobot. Factors strongly associated with three dependent variables in the model were extracted based on permuation importance. We found that the strongest determinant of knowledge of antibiotics and AMR was education level. Knowledge of antibiotics was strongly associated with the frequency of internet use. Exposure to primary information was associated with motivation for appropriate behavioural changes. Improving the availability of primary information would be a beneficial intervention. Individuals lacking higher education and without opportunities to obtain primary information should be considered a target population for effective interventions.

[Source](https://www.nature.com/articles/s41598-020-60444-1)

10.1038/s41598-020-60444-1

---

## Faeces and vaccines could be the key to tackling antimicrobial resistance.
 02.03.2020, _E Mahase_



[Source](https://www.bmj.com/content/368/bmj.m731)

10.1136/bmj.m731

---

## A survey of extended-spectrum beta-lactamase-producing Enterobacteriaceae in urban wetlands in southwestern Nigeria as a step towards generating prevalence maps of antimicrobial resistance.
 04.03.2020, _OO Adelowo, OO Ikhimiukor, C Knecht, J Vollmers, M Bhatia, AK Kaster, JA MÃ¼ller_


In many countries, emission of insufficiently treated wastewater into water bodies appears to be an important factor in spreading clinically relevant antimicrobial resistant bacteria. In this study, we looked for the presence of Enterobacteriaceae strains with resistance to 3rd generation cephalosporin antibiotics in four urban wetlands in southwestern Nigeria by isolation, whole genome sequencing and qPCR enumeration of marker genes. Genome analysis of multi-drug resistant and potentially pathogenic Escherichia coli isolates (members of the widely distributed ST10 complex) revealed the presence of the extended spectrum beta-lactamase gene blaCTX-M-15 on self-transmissible IncF plasmids. The gene was also present together with a blaTEM-1B gene on self-transmissible IncH plasmids in multi-drug resistant Enterobacter cloacae isolates. A Citrobacter freundii isolate carried blaTEM-1B on an IncR-type plasmid without discernable conjugation apparatus. All strains were isolated from a wetland for which previous qPCR enumeration of marker genes, in particular the ratio of intI1 to 16S rRNA gene copy numbers, had indicated a strong anthropogenic impact. Consistent with the isolation origin, qPCR analysis in this study showed that the blaCTX-M gene was present at an abundance of 1x10-4 relative to bacterial 16S rRNA gene copy numbers. The results indicate that contamination of these urban aquatic ecosystems with clinically relevant antibiotic resistant bacteria is substantial in some areas. Measures should therefore be put in place to mitigate the propagation of clinically relevant antimicrobial resistance within the Nigerian aquatic ecosystems.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229451)

10.1371/journal.pone.0229451

---

## Viable bacterial colonization is highly limited in the human intestine in utero.
 25.02.2020, _E Rackaityte, J Halkias, EM Fukui, VF Mendoza, C Hayzelden, ED Crawford, KE Fujimura, TD Burt, SV Lynch_


Mucosal immunity develops in the human fetal intestine by 11-14 weeks of gestation, yet whether viable microbes exist in utero and interact with the intestinal immune system is unknown. Bacteria-like morphology was identified in pockets of human fetal meconium at mid-gestation by scanning electron microscopy (n = 4), and a sparse bacterial signal was detected by 16S rRNA sequencing (n = 40 of 50) compared to environmental controls (n = 87). Eighteen taxa were enriched in fetal meconium, with Micrococcaceae (n = 9) and Lactobacillus (n = 6) the most abundant. Fetal intestines dominated by Micrococcaceae exhibited distinct patterns of T cell composition and epithelial transcription. Fetal Micrococcus luteus, isolated only in the presence of monocytes, grew on placental hormones, remained viable within antigen presenting cells, limited inflammation ex vivo and possessed genomic features linked with survival in the fetus. Thus, viable bacteria are highly limited in the fetal intestine at mid-gestation, although strains with immunomodulatory capacity are detected in subsets of specimens.

[Source](https://www.nature.com/articles/s41591-020-0761-3)

10.1038/s41591-020-0761-3

---

## Perinatal Antibiotic Exposure Affects the Transmission between Maternal and Neonatal Microbiota and Is Associated with Early-Onset Sepsis.
 27.02.2020, _P Zhou, Y Zhou, B Liu, Z Jin, X Zhuang, W Dai, Z Yang, X Feng, Q Zhou, Y Liu, X Xu, L Zhang_


Intrapartum antibiotic prophylaxis reduces the risk of infection to a mother and neonate, but antibiotic-mediated maternal and neonatal microbiota dysbiosis increases other health risks to newborn infants. We studied the impact of perinatal antibiotic prophylaxis on the microbiota in mothers and newborns with full-term or preterm delivery. Ninety-eight pregnant women and their neonates were divided into the following four groups: full term without antibiotic exposure (FT), full term with antibiotic exposure (FTA), preterm without antibiotic exposure (PT), and preterm with antibiotic exposure (PTA). Bacterial composition was analyzed by sequencing the 16S rRNA gene from maternal vaginal swabs (V) and neonatal meconium (F). The results showed that in maternal vaginal and neonatal meconium microbiota, FT and PT groups had a higher load of Lactobacillus spp. than did the FTA and PTA groups. In addition, whether in the mother or newborn, the dissimilarity in microbiota between FT and PT was the lowest compared to that between other groups. Compared to the FT and PT groups, the dissimilarity in microbial structures between the vagina and meconium decreased in the FTA and PTA groups. The health outcome of infants reveals an association between early-onset sepsis and antibiotic-mediated microbiota dysbiosis. In conclusion, perinatal antibiotic exposure is related to the establishment of gut microbiota and health risks in newborns. Promoting the rational usage of antibiotics with pregnant women will improve neonatal health.IMPORTANCE Perinatal antibiotic prophylaxis is an effective method for preventing group B Streptococcus (GBS) infection in newborns. Antibiotic exposure unbalances women's vaginal microbiota, which is associated with the establishment of the newborn gut microbiota. However, the influence of perinatal antibiotic exposure on neonatal gut microbiota colonization and health outcomes remains unclear. In this study, we found that perinatal antibiotic exposure induced microbiota dysbiosis in a woman's vagina and the neonatal gut, and we highlight a significant decrease in the abundance of Lactobacillus spp. The influence of antibiotic use on the microbiota was greater than that from gestational age. Additionally, full-term newborns without antibiotic exposure had no evidence of early-onset sepsis, whereas in full-term or preterm newborns with antibiotic exposure before birth, at least one infant was diagnosed with early-onset sepsis. These results suggest an association between perinatal antibiotic exposure and microbial dysbiosis in maternal vaginal and neonatal gut environments, which may be related to the occurrence of early-onset sepsis.

[Source](https://msphere.asm.org/content/msph/5/1/e00984-19.full.pdf)

10.1128/mSphere.00984-19

---

## Combined analysis of microbial metagenomic and metatranscriptomic sequencing data to assess in situ physiological conditions in the premature infant gut.
 04.03.2020, _Y Sher, MR Olm, T Raveh-Sadka, CT Brown, R Sher, B Firek, R Baker, MJ Morowitz, JF Banfield_


Microbes alter their transcriptomic profiles in response to the environment. The physiological conditions experienced by a microbial community can thus be inferred using meta-transcriptomic sequencing by comparing transcription levels of specifically chosen genes. However, this analysis requires accurate reference genomes to identify the specific genes from which RNA reads originate. In addition, such an analysis should avoid biases in transcript counts related to differences in organism abundance. In this study we describe an approach to address these difficulties. Sample-specific meta-genomic assembled genomes (MAGs) were used as reference genomes to accurately identify the origin of RNA reads, and transcript ratios of genes with opposite transcription responses were compared to eliminate biases related to differences in organismal abundance, an approach hereafter named the "diametric ratio" method. We used this approach to probe the environmental conditions experienced by Escherichia spp. in the gut of 4 premature infants, 2 of whom developed necrotizing enterocolitis (NEC), a severe inflammatory intestinal disease. We analyzed twenty fecal samples taken from four premature infants (4-6 time points from each infant), and found significantly higher diametric ratios of genes associated with low oxygen levels in samples of infants later diagnosed with NEC than in samples without NEC. We also show this method can be used for examining other physiological conditions, such as exposure to nitric oxide and osmotic pressure. These study results should be treated with caution, due to the presence of confounding factors that might also distinguish between NEC and control infants. Nevertheless, together with benchmarking analyses, we show here that the diametric ratio approach can be applied for evaluating the physiological conditions experienced by microbes in situ. Results from similar studies can be further applied for designing diagnostic methods to detect NEC in its early developmental stages.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229537)

10.1371/journal.pone.0229537

---

## A microbiome and metabolomic signature of phases of cutaneous healing identified by profiling sequential acute wounds of human skin: An exploratory study.
 27.02.2020, _M Ashrafi, Y Xu, H Muhamadali, I White, M Wilkinson, K Hollywood, M Baguneid, R Goodacre, A Bayat_


Profiling skin microbiome and metabolome has been utilised to gain further insight into wound healing processes. The aims of this multi-part temporal study in 11 volunteers were to analytically profile the dynamic wound tissue and headspace metabolome and sequence microbial communities in acute wound healing at days 0, 7, 14, 21 and 28, and to investigate their relationship to wound healing, using non-invasive quantitative devices. Metabolites were obtained using tissue extraction, sorbent and polydimethylsiloxane patches and analysed using GCMS. PCA of wound tissue metabolome clearly separated time points with 10 metabolites of 346 being involved in separation. Analysis of variance-simultaneous component analysis identified a statistical difference between the wound headspace metabolome, sites (P = 0.0024) and time points (P&lt;0.0001), with 10 out of the 129 metabolites measured involved with this separation between sites and time points. A reciprocal relationship between Staphylococcus spp. and Propionibacterium spp. was observed at day 21 (P&lt;0.05) with a statistical correlation between collagen and Propionibacterium (r = 0.417; P = 0.038) and Staphylococcus (r = -0.434; P = 0.03). Procrustes analysis showed a statistically significant similarity between wound headspace and tissue metabolome with non-invasive wound devices. This exploratory study demonstrates the temporal and dynamic nature of acute wound metabolome and microbiome presenting a novel class of biomarkers that correspond to wound healing, with further confirmatory studies now necessary.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229545)

10.1371/journal.pone.0229545

---

## Single-Cell Genomics and the Oral Microbiome.
 24.02.2020, _M Balachandran, KL Cross, M Podar_


The human oral cavity is one of the first environments where microbes have been discovered and studied since the dawn of microbiology. Nevertheless, approximately 200 types of bacteria from the oral microbiota have remained uncultured in the laboratory. Some are associated with a healthy oral microbial community, while others are linked to oral diseases, from dental caries to gum disease. Single-cell genomics has enabled inferences on the physiology, virulence, and evolution of such uncultured microorganisms and has further enabled isolation and cultivation of several novel oral bacteria, including the discovery of novel interspecies interactions. This review summarizes some of the more recent advances in this field, which is rapidly moving toward physiologic characterization of single cells and ultimately cultivation of the yet uncultured. A combination of traditional microbiological approaches with genomic-based physiologic predictions and isolation strategies may lead to the oral microbiome being the first complex microbial community to have all its members cultivable in the laboratory. Studying the biology of the individual microbes when in association with other members of the community, in controlled laboratory conditions and in vivo, should lead to a better understanding of oral dysbiosis and its prevention and reversion.

[Source](https://journals.sagepub.com/doi/abs/10.1177/0022034520907380)

10.1177/0022034520907380

---

## Insights Obtained by Culturing Saccharibacteria With Their Bacterial Hosts.
 27.02.2020, _B Bor, AJ Collins, PP Murugkar, S Balasubramanian, TT To, EL Hendrickson, JK Bedree, FB Bidlack, CD Johnston, W Shi, JS McLean, X He, FE Dewhirst_


Oral microbiome research has moved from asking "Who's there?" to "What are they doing?" Understanding what microbes "do" involves multiple approaches, including obtaining genomic information and examining the interspecies interactions. Recently we isolated a human oral Saccharibacteria (TM7) bacterium, HMT-952, strain TM7x, which is an ultrasmall parasite of the oral bacterium Actinomyces odontolyticus. The host-parasite interactions, such as phage-bacterium or Saccharibacteria-host bacterium, are understudied areas with large potential for insight. The Saccharibacteria phylum is a member of Candidate Phyla Radiation, a large lineage previously devoid of cultivated members. However, expanding our understanding of Saccharibacteria-host interactions requires examining multiple phylogenetically distinct Saccharibacteria-host pairs. Here we report the isolation of 3 additional Saccharibacteria species from the human oral cavity in binary coculture with their bacterial hosts. They were obtained by filtering ultrasmall Saccharibacteria cells free of other larger bacteria and inoculating them into cultures of potential host bacteria. The binary cocultures obtained could be stably passaged and studied. Complete closed genomes were obtained and allowed full genome analyses. All have small genomes (&lt;1 Mb) characteristic of parasitic species and dramatically limited de novo synthetic pathway capabilities but include either restriction modification or CRISPR-Cas systems as part of an innate defense against foreign DNA. High levels of gene synteny exist among Saccharibacteria species. Having isolates growing in coculture with their hosts allowed time course studies of growth and parasite-host interactions by phase contrast, fluorescence in situ hybridization, and scanning electron microscopy. The cells of the 4 oral Saccharibacteria species are ultrasmall and could be seen attached to their larger Actinobacteria hosts. Parasite attachment appears to lead to host cell death and lysis. The successful cultivation of Saccharibacteria species has significantly expanded our understanding of these ultrasmall Candidate Phyla Radiation bacteria.

[Source](https://journals.sagepub.com/doi/abs/10.1177/0022034520905792)

10.1177/0022034520905792

---

## Electronic Cigarette Aerosol Modulates the Oral Microbiome and Increases Risk of Infection.
 27.02.2020, _S Pushalkar, B Paul, Q Li, J Yang, R Vasconcelos, S Makwana, JM GonzÃ¡lez, S Shah, C Xie, MN Janal, E Queiroz, M Bederoff, J Leinwand, J Solarewicz, F Xu, E Aboseria, Y Guo, D Aguallo, C Gomez, A Kamer, D Shelley, Y Aphinyanaphongs, C Barber, T Gordon, P Corby, X Li, D Saxena_


The trend of e-cigarette use among teens is ever increasing. Here we show the dysbiotic oral microbial ecology in e-cigarette users influencing the local host immune environment compared with non-smoker controls and cigarette smokers. Using 16S rRNA high-throughput sequencing, we evaluated 119 human participants, 40 in each of the three cohorts, and found significantly altered beta-diversity in e-cigarette users (p = 0.006) when compared with never smokers or tobacco cigarette smokers. The abundance of Porphyromonas and Veillonella (p = 0.008) was higher among vapers. Interleukin (IL)-6 and IL-1Î² were highly elevated in e-cigarette users when compared with non-users. Epithelial cell-exposed e-cigarette aerosols were more susceptible for infection. In vitro infection model of premalignant Leuk-1 and malignant cell lines exposed to e-cigarette aerosol and challenged by Porphyromonas gingivalis and Fusobacterium nucleatum resulted in elevated inflammatory response. Our findings for the first time demonstrate that e-cigarette users are more prone to infection.

[Source](https://www.cell.com/iscience/fulltext/S2589-0042(20)30068-7)

10.1016/j.isci.2020.100884

---

## Oral Microbiome Changes Implicated in Gum Inflammation in E-Cigarette Users

[Source](https://www.genomeweb.com/sequencing/oral-microbiome-changes-implicated-gum-inflammation-e-cigarette-users#.XmCCpy2ZPOQ)

---

## A salivary metabolite signature that reflects gingival host-microbe interactions: instability predicts gingivitis susceptibility.
 28.02.2020, _MM Fernandez-Gutierrez, S Imangaliyev, A Prodan, BG Loos, BJF Keijser, M Kleerebezem_


Several proteins and peptides in saliva were shown to stimulate gingival wound repair, but the role of salivary metabolites in this process remains unexplored. In vitro gingival re-epithelialization kinetics were determined using unstimulated saliva samples from healthy individuals collected during an experimental gingivitis study. Elastic net regression with stability selection identified a specific metabolite signature in a training dataset that was associated with the observed re-epithelialization kinetics and enabled its prediction for all saliva samples obtained in the clinical study. This signature encompassed ten metabolites, including plasmalogens, diacylglycerol and amino acid derivatives, which reflect enhanced host-microbe interactions. This association is in agreement with the positive correlation of the metabolite signature with the individual's gingival bleeding index. Remarkably, intra-individual signature-variation over time was associated with elevated risk for gingivitis development. Unravelling how these metabolites stimulate wound repair could provide novel avenues towards therapeutic approaches in patients with impaired wound healing capacity.

[Source](https://www.nature.com/articles/s41598-020-59988-z)

10.1038/s41598-020-59988-z

---

## The Role of Lung and Gut Microbiota in the Pathology of Asthma.
 20.02.2020, _W Barcik, RCT Boutin, M Sokolowska, BB Finlay_


Asthma is a common chronic respiratory disease affecting more than 300 million people worldwide. Clinical features of asthma and its immunological and molecular etiology vary significantly among patients. An understanding of the complexities of asthma has evolved to the point where precision medicine approaches, including microbiome analysis, are being increasingly recognized as an important part of disease management. Lung and gut microbiota play several important roles in the development, regulation, and maintenance of healthy immune responses. Dysbiosis and subsequent dysregulation of microbiota-related immunological processes affect the onset of the disease, its clinical characteristics, and responses to treatment. Bacteria and viruses are the most extensively studied microorganisms relating to asthma pathogenesis, but other microbes, including fungi and even archaea, can potently influence airway inflammation. This review focuses on recently discovered connections between lung and gut microbiota, including bacteria, fungi, viruses, and archaea, and their influence on asthma.

[Source](https://www.sciencedirect.com/science/article/pii/S1074761320300340)

10.1016/j.immuni.2020.01.007

---

## Endogenous murine microbiota member Faecalibaculum rodentium and its human homologue protect from intestinal tumour growth.
 06.03.2020, _E Zagato, C Pozzi, A Bertocchi, T Schioppa, F Saccheri, S Guglietta, B Fosso, L Melocchi, G Nizzoli, J Troisi, M Marzano, B Oresta, I Spadoni, K Atarashi, S Carloni, S Arioli, G Fornasa, F Asnicar, N Segata, S Guglielmetti, K Honda, G Pesole, W Vermi, G Penna, M Rescigno_


The microbiota has been shown to promote intestinal tumourigenesis, but a possible anti-tumourigenic effect has also been postulated. Here, we demonstrate that changes in the microbiota and mucus composition are concomitant with tumourigenesis. We identified two anti-tumourigenic strains of the microbiota-Faecalibaculum rodentium and its human homologue, Holdemanella biformis-that are strongly under-represented during tumourigenesis. Reconstitution of ApcMin/+ or azoxymethane- and dextran sulfate sodium-treated mice with an isolate of F. rodentium (F. PB1) or its metabolic products reduced tumour growth. Both F. PB1 and H. biformis produced short-chain fatty acids that contributed to control protein acetylation and tumour cell proliferation by inhibiting calcineurin and NFATc3 activation in mouse and human settings. We have thus identified endogenous anti-tumourigenic bacterial strains with strong diagnostic, therapeutic and translational potential.

[Source](https://www.nature.com/articles/s41564-019-0649-5)

10.1038/s41564-019-0649-5

---

## Escherichia coli adhesion portion FimH functions as an adjuvant for cancer immunotherapy.
 05.03.2020, _W Zhang, L Xu, HB Park, J Hwang, M Kwak, PCW Lee, G Liang, X Zhang, J Xu, JO Jin_


Induction of antigen-specific immune activation by the maturation of dendritic cells (DCs) is a strategy used for cancer immunotherapy. In this study, we find that FimH, which is an Escherichia coli adhesion portion, induces toll-like receptor 4-dependent and myeloid differentiation protein 2-independent DC maturation in mice in vivo. A combined treatment regimen with FimH and antigen promotes antigen-specific immune activation, including proliferation of T cells, production of IFN-Î³ and TNF-Î±, and infiltration of effector T cells into tumors, which consequently inhibits tumor growth in mice in vivo against melanoma and carcinoma. In addition, combined therapeutic treatment of anti-PD-L1 antibodies and FimH treatment efficiently inhibits CT26 tumor growth in BALB/c mice. Finally, FimH promotes human peripheral blood DC activation and syngeneic T-cell proliferation and activation. Taken together, these findings demonstrate that FimH can be a useful adjuvant for cancer immunotherapy.

[Source](https://www.nature.com/articles/s41467-020-15030-4)

10.1038/s41467-020-15030-4

---

## Temporal changes in gut microbiota profile in children with acute lymphoblastic leukemia prior to commencement-, during-, and post-cessation of chemotherapy.
 05.03.2020, _LL Chua, R Rajasuriar, YAL Lim, YL Woo, P Loke, H Ariffin_


Alteration in gut microbiota has been recently linked with childhood leukemia and the use of chemotherapy. Whether the perturbed microbiota community is restored after disease remission and cessation of cancer treatment has not been evaluated. This study examines the chronological changes of gut microbiota in children with acute lymphoblastic leukemia (ALL) prior to the start-, during-, and following cessation of chemotherapy.

[Source](https://bmccancer.biomedcentral.com/articles/10.1186/s12885-020-6654-5)

10.1186/s12885-020-6654-5

---

## Interplay between Dietary Polyphenols and Oral and Gut Microbiota in the Development of Colorectal Cancer.
 03.03.2020, _C Cueva, M Silva, I Pinillos, B BartolomÃ©, MV Moreno-Arribas_


Colorectal cancer (CRC) is the third most diagnosed type of cancer worldwide. Dietary features play an important role in its development, and the involvement of human microbial communities in this pathology has also recently been recognized. Individuals with CRC display alterations in gut bacterial composition and a notably higher abundance of putative oral bacteria in colonic tumors. Many experimental studies and preclinical evidence propose that dietary polyphenols have a relevant role in CRC development and progression, mainly attributed to their immunomodulatory activities. Furthermore, polyphenols can modulate oral and gut microbiota, and in turn, intestinal microbes catabolize polyphenols to release metabolites that are often more active and better absorbed than the original phenolic compounds. The current study aimed to review and summarize current knowledge on the role of microbiota and the interactions between dietary polyphenols and microbiota in relation to CRC development. We have highlighted the mechanisms by which dietary polyphenols and/or their microbial metabolites exert their action on the pathogenesis and prevention of CRC as modulators of the composition and/or activity of oral and intestinal microbiota, including novel screening biomarkers and possible nutritional therapeutic implications.

[Source](https://www.mdpi.com/2072-6643/12/3/625)

10.3390/nu12030625

---

## Short-term supplementation of celecoxib-shifted butyrate production on a simulated model of the gut microbial ecosystem and ameliorated in vitro inflammation.
 05.03.2020, _E Hernandez-Sanabria, E Heiremans, M Calatayud Arroyo, R Props, L Leclercq, J Snoeys, T Van de Wiele_


Celecoxib has been effective in the prevention and treatment of chronic inflammatory disorders through inhibition of altered cyclooxygenase-2 (COX-2) pathways. Despite the benefits, continuous administration may increase risk of cardiovascular events. Understanding microbiome-drug-host interactions is fundamental for improving drug disposition and safety responses of colon-targeted formulations, but little information is available on the bidirectional interaction between individual microbiomes and celecoxib. Here, we conducted in vitro batch incubations of human faecal microbiota to obtain a mechanistic proof-of-concept of the short-term impact of celecoxib on activity and composition of colon bacterial communities. Celecoxib-exposed microbiota shifted metabolic activity and community composition, whereas total transcriptionally active bacterial population was not significantly changed. Butyrate production decreased by 50% in a donor-dependent manner, suggesting that celecoxib impacts in vitro fermentation. Microbiota-derived acetate has been associated with inhibition of cancer markers and our results suggest uptake of acetate for bacterial functions when celecoxib was supplied, which potentially favoured bacterial competition for acetyl-CoA. We further assessed whether colon microbiota modulates anti-inflammatory efficacy of celecoxib using a simplified inflammation model, and a novel in vitro simulation of the enterohepatic metabolism. Celecoxib was responsible for only 5% of the variance in bacterial community composition but celecoxib-exposed microbiota preserved barrier function and decreased concentrations of IL-8 and CXCL16 in a donor-dependent manner in our two models simulating gut inflammatory milieu. Our results suggest that celecoxib-microbiome-host interactions may not only elicit adaptations in community composition but also in microbiota functionality, and these may need to be considered for guaranteeing efficient COX-2 inhibition.

[Source](https://www.nature.com/articles/s41522-020-0119-0)

10.1038/s41522-020-0119-0

---

## Crosstalk Between Gut Microbiota and Innate Immunity and Its Implication in Autoimmune Diseases

[Source](https://www.frontiersin.org/articles/10.3389/fimmu.2020.00282/full)

---

## An empirically derived method for measuring human gut microbiome alpha diversity: Demonstrated utility in predicting health-related outcomes among a human clinical sample.
 02.03.2020, _SL Hagerty, KE Hutchison, CA Lowry, AD Bryan_


The human gut microbiome has emerged as a potential key factor involved in the manifestation of physical and mental health. Despite an explosion of cross-disciplinary interest in researching the gut microbiome, there remains to be a gold-standard method for operationalizing gut microbiome alpha diversity. Given researchers' interest in examining the relationships among gut microbiome alpha diversity and health-related outcomes of interest, a way of operationalizing the microbiome that yields a numeric value, which could be used in common statistical approaches, is needed. Thus, the current study aims to provide methodological guidance for how to operationalize microbiome alpha diversity. Findings suggest that alpha diversity of the human gut microbiome is comprised of two sub-constructs (richness and evenness), and we propose a step-by-step method of creating alpha diversity composite measures based on this key insight. Finally, we demonstrate that our empirically derived richness and evenness composite measures are significantly associated with health-related variables of interest (alcohol use, symptoms of depression) among a human clinical sample.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229204)

10.1371/journal.pone.0229204

---

## Proof-of-concept study of the efficacy of a microbiota-directed complementary food formulation (MDCF) for treating moderate acute malnutrition.
 25.02.2020, _I Mostafa, NN Nahar, MM Islam, S Huq, M Mustafa, M Barratt, JI Gordon, T Ahmed_


Childhood undernutrition remains a significant global health challenge accounting for over half of all under 5 child mortality. Moderate acute malnutrition (MAM), which leads to wasting [weight-for-length z-scores (WLZ) between - 2 and - 3], affects 33 million children under 5 globally and more than 2 million in Bangladesh alone. We have previously reported that acute malnutrition in this population is associated with gut microbiota immaturity, and in a small, 1-month pre-proof-of-concept (POC) study demonstrated that a microbiota-directed complementary food formulation (MDCF-2) was able to repair this immaturity, promote weight gain and increase plasma biomarkers and mediators of healthy growth. Here we describe the design controlled feeding study that tests whether MDCF-2 exhibits superior efficacy (ponderal growth, host biomarkers of a biological state) than a conventional Ready-to-use Supplementary Food (RUSF) in children with MAM over intervention period of 3 months.

[Source](https://bmcpublichealth.biomedcentral.com/articles/10.1186/s12889-020-8330-8)

10.1186/s12889-020-8330-8

---

## Microbiota as Predictor of Mortality in Allogeneic Hematopoietic-Cell Transplantation.
 28.02.2020, _JU Peled, ALC Gomes, SM Devlin, ER Littmann, Y Taur, AD Sung, D Weber, D Hashimoto, AE Slingerland, JB Slingerland, M Maloy, AG Clurman, CK Stein-Thoeringer, KA Markey, MD Docampo, M Burgos da Silva, N Khan, A Gessner, JA Messina, K Romero, MV Lew, A Bush, L Bohannon, DG Brereton, E Fontana, LA Amoretti, RJ Wright, GK Armijo, Y Shono, M Sanchez-Escamilla, N Castillo Flores, A Alarcon Tomas, RJ Lin, L YÃ¡Ã±ez San Segundo, GL Shah, C Cho, M Scordo, I Politikos, K Hayasaka, Y Hasegawa, B Gyurkocza, DM Ponce, JN Barker, MA Perales, SA Giralt, RR Jenq, T Teshima, NJ Chao, E Holler, JB Xavier, EG Pamer, MRM van den Brink_


Relationships between microbiota composition and clinical outcomes after allogeneic hematopoietic-cell transplantation have been described in single-center studies. Geographic variations in the composition of human microbial communities and differences in clinical practices across institutions raise the question of whether these associations are generalizable.

[Source](https://www.nejm.org/doi/10.1056/NEJMoa1900623)

10.1056/NEJMoa1900623

---

## Age-Related Variation of Bacterial and Fungal Communities in Different Body Habitats across the Young, Elderly, and Centenarians in Sardinia

[Source](https://msphere.asm.org/content/msph/5/1/e00558-19.full.pdf)

---

## Response of the human gut and saliva microbiome to urbanization in Cameroon.
 27.02.2020, _A Lokmer, S Aflalo, N Amougou, S Lafosse, A Froment, FE Tabe, M Poyet, M Groussin, R Said-Mohamed, L SÃ©gurel_


Urban populations from highly industrialized countries are characterized by a lower gut bacterial diversity as well as by changes in composition compared to rural populations from less industrialized countries. To unveil the mechanisms and factors leading to this diversity loss, it is necessary to identify the factors associated with urbanization-induced shifts at a smaller geographical scale, especially in less industrialized countries. To do so, we investigated potential associations between a variety of dietary, medical, parasitological and socio-cultural factors and the gut and saliva microbiomes of 147 individuals from three populations along an urbanization gradient in Cameroon. We found that the presence of Entamoeba sp., a commensal gut protozoan, followed by stool consistency, were major determinants of the gut microbiome diversity and composition. Interestingly, urban individuals have retained most of their gut eukaryotic and bacterial diversity despite significant changes in diet compared to the rural areas, suggesting that the loss of bacterial microbiome diversity observed in industrialized areas is likely associated with medication. Finally, we observed a weak positive correlation between the gut and the saliva microbiome diversity and composition, even though the saliva microbiome is mainly shaped by habitat-related factors.

[Source](https://www.nature.com/articles/s41598-020-59849-9)

10.1038/s41598-020-59849-9

---

## Therapeutic Effects of the In Vitro Cultured Human Gut Microbiota as Transplants on Altering Gut Microbiota and Improving Symptoms Associated with Autism Spectrum Disorder.
 26.02.2020, _K Chen, Y Fu, Y Wang, L Liao, H Xu, A Zhang, J Zhang, L Fan, J Ren, B Fang_


Autism spectrum disorder (ASD) is a brain-based neurodevelopmental disorder characterized by behavioral abnormalities. Accumulating studies show that the gut microbiota plays a vital role in the pathogenesis of ASD, and gut microbiota transplantation (GMT) is a promising technique for the treatment of ASD. In clinical applications of GMT, it is challenging to obtain effective transplants because of the high costs of donor selection and heterogeneity of donors' gut microbiota, which can cause different clinical responses. In vitro batch culture is a fast, easy-to-operate, and repeatable method to culture gut microbiota. Thus, the present study investigates the feasibility of treating ASD with in vitro cultured gut microbiota as transplants. We cultured gut microbiota via the in vitro batch culture method and performed GMT in the maternal immune activation (MIA)-induced ASD mouse model with original donor microbiota and in vitro cultured microbiota. Open field, three-chamber social, marble burying, and self-grooming tests were used for behavioral improvement assessment. Serum levels of chemokines were detected. Microbial total DNA was extracted from mouse fecal samples, and 16S rDNA was sequenced using Illumina. Our results showed that GMT treatment with original and cultured donor gut microbiota significantly ameliorated anxiety-like and repetitive behaviors and improved serum levels of chemokines including GRO-Î± (CXCL1), MIP-1Î± (CCL3), MCP-3 (CCL7), RANTES (CCL5), and Eotaxin (CCL11) in ASD mice. Meanwhile, the gut microbial communities of the two groups that received GMT treatment were changed compared with the ASD mice groups. In the group treated with in vitro cultured donor gut microbiota, there was a significant decrease in the relative abundance of key differential taxa, including S24-7, Clostridiaceae, Prevotella_other, and Candidatus Arthromitus. The relative abundance of these taxa reached close to the level of healthy mice. Prevotella_other also decreased in the group treated with original donor gut microbiota, with a significant increase in Ruminococcaceae and Oscillospira. The present study demonstrated that GMT with in vitro cultured microbiota also improved behavioral abnormalities and chemokine disorders in an ASD mouse model compared with GMT with original donor gut microbiota. In addition, it significantly modified several key differential taxa in gut microbial composition.

[Source](https://link.springer.com/article/10.1007/s00248-020-01494-w)

10.1007/s00248-020-01494-w

---

## A preliminary study on the potential of Nanopore MinION and Illumina MiSeq 16S rRNA gene sequencing to characterize building-dust microbiomes

[Source](https://www.nature.com/articles/s41598-020-59771-0)

---

## Spatial metabolomics of in situ host-microbe interactions at the micrometre scale.
 27.02.2020, _B Geier, EM Sogin, D Michellod, M Janda, M Kompauer, B Spengler, N Dubilier, M Liebeke_


Spatial metabolomics describes the location and chemistry of small molecules involved in metabolic phenotypes, defence molecules and chemical interactions in natural communities. Most current techniques are unable to spatially link the genotype and metabolic phenotype of microorganisms in situ at a scale relevant to microbial interactions. Here, we present a spatial metabolomics pipeline (metaFISH) that combines fluorescence in situ hybridization (FISH) microscopy and high-resolution atmospheric-pressure matrix-assisted laser desorption/ionization mass spectrometry to image host-microbe symbioses and their metabolic interactions. The metaFISH pipeline aligns and integrates metabolite and fluorescent images at the micrometre scale to provide a spatial assignment of host and symbiont metabolites on the same tissue section. To illustrate the advantages of metaFISH, we mapped the spatial metabolome of a deep-sea mussel and its intracellular symbiotic bacteria at the scale of individual epithelial host cells. Our analytical pipeline revealed metabolic adaptations of the epithelial cells to the intracellular symbionts and variation in metabolic phenotypes within a single symbiont 16S rRNA phylotype, and enabled the discovery of specialized metabolites from the host-microbe interface. metaFISH provides a culture-independent approach to link metabolic phenotypes to community members in situ and is a powerful tool for microbiologists across fields.

[Source](https://www.nature.com/articles/s41564-019-0664-6)

10.1038/s41564-019-0664-6

---

## Global chemical effects of the microbiome include new bile-acid conjugations.
 06.03.2020, _RA Quinn, AV Melnik, A Vrbanac, T Fu, KA Patras, MP Christy, Z Bodai, P Belda-Ferre, A Tripathi, LK Chung, M Downes, RD Welch, M Quinn, G Humphrey, M Panitchpakdi, KC Weldon, A Aksenov, R da Silva, J Avila-Pacheco, C Clish, S Bae, H Mallick, EA Franzosa, J Lloyd-Price, R Bussell, T Thron, AT Nelson, M Wang, E Leszczynski, F Vargas, JM Gauglitz, MJ Meehan, E Gentry, TD Arthur, AC Komor, O Poulsen, BS Boland, JT Chang, WJ Sandborn, M Lim, N Garg, JC Lumeng, RJ Xavier, BI Kazmierczak, R Jain, M Egan, KE Rhee, D Ferguson, M Raffatellu, H Vlamakis, GG Haddad, D Siegel, C Huttenhower, SK Mazmanian, RM Evans, V Nizet, R Knight, PC Dorrestein_


A mosaic of cross-phylum chemical interactions occurs between all metazoans and their microbiomes. A number of molecular families that are known to be produced by the microbiome have a marked effect on the balance between health and disease1-9. Considering the diversity of the human microbiome (which numbers over 40,000 operational taxonomic units10), the effect of the microbiome on the chemistry of an entire animal remains underexplored. Here we use mass spectrometry informatics and data visualization approaches11-13 to provide an assessment of the effects of the microbiome on the chemistry of an entire mammal by comparing metabolomics data from germ-free and specific-pathogen-free mice. We found that the microbiota affects the chemistry of all organs. This included the amino acid conjugations of host bile acids that were used to produce phenylalanocholic acid, tyrosocholic acid and leucocholic acid, which have not previously been characterized despite extensive research on bile-acid chemistry14. These bile-acid conjugates were also found in humans, and were enriched in patients with inflammatory bowel disease or cystic fibrosis. These compounds agonized the farnesoid X receptor in vitro, and mice gavaged with the compounds showed reduced expression of bile-acid synthesis genes in vivo. Further studies are required to confirm whether these compounds have a physiological role in the host, and whether they contribute to gut diseases that are associated with microbiome dysbiosis.

[Source](https://www.nature.com/articles/s41586-020-2047-9)

10.1038/s41586-020-2047-9

---

## PM2.5 exposure perturbs lung microbiome and its metabolic profile in mice

[Source](https://www.sciencedirect.com/science/article/pii/S0048969720309426)

---

## Growth hormone deficiency and excess alter the gut microbiome in adult male mice.
 26.02.2020, _EA Jensen, JA Young, Z Jackson, J Busken, EO List, RK Carroll, JJ Kopchick, ER Murphy, DE Berryman_


The gut microbiome has been implicated in host metabolism, endocrinology, and pathophysiology. Furthermore, several studies have shown that gut bacteria impact host growth, partially mediated through the growth hormone (GH)/insulin-like growth factor-1 (IGF-1) axis. Yet, no study to date has examined the specific role of GH on the gut microbiome. Our study thus characterized the adult gut microbial profile and intestinal phenotype in GH gene-disrupted (GH-/-) mice (a model of GH deficiency) and bovine GH transgenic (bGH) mice (a model of chronic, excess GH action) at six months of age. Both the GH-/- and bGH mice had altered microbial signatures, which were in opposing directions at the phylum and genus levels. For example, GH-/- mice had significantly reduced abundance in Proteobacteria, Campylobacterota, and Actinobacteria phyla, whereas bGH mice exhibited a trending increase in those phyla compared to respective controls. Analysis of maturity of the microbial community demonstrated that lack of GH resulted in a significantly more immature microbiome while excess GH increases microbial maturity. Several common bacterial genera were shared, although in opposing directions, between the two mouse lines (e.g. decreased in GH-/- mice and increased in bGH mice), suggesting an association with GH. Likewise, metabolic pathways like acetate, butyrate, heme B, and folate biosynthesis were predicted to be impacted by GH. This study is the first to characterize the gut microbiome in mouse lines with altered GH action and indicates that GH may play a role in the growth of certain microbiota thus impacting microbial maturation and metabolic function.

[Source](https://academic.oup.com/endo/advance-article-abstract/doi/10.1210/endocr/bqaa026/5758063)

10.1210/endocr/bqaa026

---

## Dissociation of Adaptive Thermogenesis from Glucose Homeostasis in Microbiome-Deficient Mice.
 05.03.2020, _TI Krisko, HT Nicholls, CJ Bare, CD Holman, GG Putzel, RS Jansen, N Sun, KY Rhee, AS Banks, DE Cohen_


Recent studies suggest that a key mechanism whereby the gut microbiome influences energy balance and glucose homeostasis is through the recruitment of brown and beige adipocytes, primary mediators of the adaptive thermogenic response. To test this, we assessed energy expenditure and glucose metabolism in two complementary mouse models of gut microbial deficiency, which were exposed to a broad range of thermal and dietary stresses. Neither ablation of the gut microbiome, nor the substantial microbial perturbations induced by cold ambient temperatures, influenced energy expenditure during cold exposure or high-fat feeding. Nevertheless, we demonstrated a critical role for gut microbial metabolism in maintaining euglycemia through the production of amino acid metabolites that optimized hepatic TCA (tricarboxylic acid) cycle fluxes in support of gluconeogenesis. These results distinguish the dispensability of the gut microbiome for the regulation of energy expenditure from its critical contribution to the maintenance of glucose homeostasis.

[Source](https://www.cell.com/cell-metabolism/pdf/S1550-4131(20)30013-9.pdf)

10.1016/j.cmet.2020.01.012

---

## Fecal IgA Levels Are Determined by Strain-Level Differences in Bacteroides ovatus and Are Modifiable by Gut Microbiota Manipulation.
 28.02.2020, _C Yang, I Mogno, EJ Contijoch, JN Borgerding, V Aggarwala, Z Li, S Siu, EK Grasset, DS Helmus, MC Dubinsky, S Mehandru, A Cerutti, JJ Faith_


Fecal IgA production depends on colonization by a gut microbiota. However, the bacterial strains that drive gut IgA production remain largely unknown. Here, we assessed the IgA-inducing capacity of a diverse set of human gut microbial strains by monocolonizing mice with each strain. We identified Bacteroides ovatus as the species that best induced gut IgA production. However, this induction varied bimodally across different B. ovatus strains. The high IgA-inducing B. ovatus strains preferentially elicited more IgA production in the large intestine through the T cell-dependent B cell-activation pathway. Remarkably, a low-IgA phenotype in mice could be robustly and consistently converted into a high-IgA phenotype by transplanting a multiplex cocktail of high IgA-inducing B. ovatus strains but not individual ones. Our results highlight the critical importance of microbial strains in driving phenotype variation in the mucosal immune system and provide a strategy to robustly modify a gut immune phenotype, including IgA production.

[Source](https://www.sciencedirect.com/science/article/pii/S193131282030055X)

10.1016/j.chom.2020.01.016

---

## Can rumen bacteria communicate to each other?
 28.02.2020, _MY Won, LB Oyama, SJ Courtney, CJ Creevey, SA Huws_


The rumen contains a myriad of microbes whose primary role is to degrade and ferment dietary nutrients, which then provide the host with energy and nutrients. Rumen microbes commonly attach to ingested plant materials and form biofilms for effective plant degradation. Quorum sensing (QS) is a well-recognised form of bacterial communication in most biofilm communities, with homoserine lactone (AHL)-based QS commonly being used by Gram-negative bacteria alone and AI-2 Lux-based QS communication being used to communicate across Gram-negative and Gram-positive bacteria. However, bacterial cell to cell communication in the rumen is poorly understood. In this study, rumen bacterial genomes from the Hungate collection and Genbank were prospected for QS-related genes. To check that the discovered QS genes are actually expressed in the rumen, we investigated expression levels in rumen metatranscriptome datasets.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00796-y)

10.1186/s40168-020-00796-y

---

## Entomopathogenic nematode-associated microbiota: from monoxenic paradigm to pathobiome.
 04.03.2020, _JC Ogier, S PagÃ¨s, M Frayssinet, S Gaudriault_


The holistic view of bacterial symbiosis, incorporating both host and microbial environment, constitutes a major conceptual shift in studies deciphering host-microbe interactions. Interactions between Steinernema entomopathogenic nematodes and their bacterial symbionts, Xenorhabdus, have long been considered monoxenic two partner associations responsible for the killing of the insects and therefore widely used in insect pest biocontrol. We investigated this "monoxenic paradigm" by profiling the microbiota of infective juveniles (IJs), the soil-dwelling form responsible for transmitting Steinernema-Xenorhabdus between insect hosts in the parasitic lifecycle.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00800-5)

10.1186/s40168-020-00800-5

---

## The canine oral microbiome: variation in bacterial populations across different niches.
 06.03.2020, _A Ruparell, T Inui, R Staunton, C Wallis, O Deusch, LJ Holcombe_


Microbiota from different niches within the canine oral cavity were profiled and compared. Supragingival plaque and stimulated saliva, were collected alongside samples from the buccal and tongue dorsum mucosa, from 14 Labrador retrievers at three timepoints within a 1 month timeframe. The V3-V4 region of the 16S rRNA gene was sequenced via Illumina MiSeq.

[Source](https://link.springer.com/article/10.1186/s12866-020-1704-3)

10.1186/s12866-020-1704-3

---

## Evolutionary analysis of Mycobacterium bovis genotypes across Africa suggests co-evolution with livestock and humans.
 02.03.2020, _OF Inlamea, P Soares, CY Ikuta, MB Heinemann, SJ AchÃ¡, A Machado, JS Ferreira Neto, M Correia-Neves, T Rito_


Mycobacterium bovis is the pathogenic agent responsible for bovine tuberculosis (bTB), a zoonotic disease affecting mostly cattle, but also transmittable to humans and wildlife. Genetic studies on M. bovis allow to detect possible routes of bTB transmission and the identification of genetic reservoirs that may provide an essential framework for public health action. We used a database with 1235 M. bovis genotypes collected from different regions in Africa with 45 new Mozambican samples. Our analyses, based on phylogeographic and population genetics' approaches, allowed to identify two clear trends. First, the genetic diversity of M. bovis is geographically clustered across the continent, with the only incidences of long-distance sharing of genotypes, between South Africa and Algeria, likely due to recent European introductions. Second, there is a broad gradient of diversity from Northern to Southern Africa with a diversity focus on the proximity to the Near East, where M. bovis likely emerged with animal domestication in the last 10,000 years. Diversity indices are higher in Eastern Africa, followed successively by Northern, Central, Southern and Western Africa, roughly correlating with the regional archaeological records of introduction of animal domesticates. Given this scenario M. bovis in Africa was probably established millennia ago following a concomitant spread with cattle, sheep and goat. Such scenario could translate into long-term locally adapted lineages across Africa. This work describes a novel scenario for the spread of M. bovis in Africa using the available genetic data, opening the field to further studies using higher resolution genomic data.

[Source](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0008081)

10.1371/journal.pntd.0008081

---

## Nanopore sequencing of microbial communities reveals the potential role of sea lice as a reservoir for fish pathogens.
 28.02.2020, _AT GonÃ§alves, R Collipal-Matamal, V Valenzuela-MuÃ±oz, G NuÃ±ez-AcuÃ±a, D Valenzuela-Miranda, C Gallardo-EscÃ¡rate_


Caligus rogercresseyi is a copepod ectoparasite with a high prevalence in salmon farms in Chile, causing severe welfare and economic concerns to the sector. Information on the parasite's underpinning mechanisms to support its life strategy is recently being investigated. Due to the critical role of microbiota, this study aimed to characterize the microbiota community associated with C. rogercresseyi from different regions with salmon aquaculture in Chile. Using third-generation sequencing with Nanopore technology (MinION) the full 16S rRNA gene from sea lice obtained from 8 areas distributed over the three main aquaculture regions were sequenced. Microbiota of the parasite is mainly comprised of members of phyla Proteobacteria and Bacteroidetes, and a core microbiota community with 147 taxonomical features was identified, and it was present in sea lice from the three regions. This community accounted for 19% of total identified taxa but more than 70% of the total taxonomical abundance, indicating a strong presence in the parasite. Several taxa with bioactive compound secretory capacity were identified, such as members of genus Pseudoalteromonas and Dokdonia, suggesting a possible role of the lice microbiota during the host infestation processes. Furthermore, the microbiota community was differentially associated with the salmon production, where several potential pathogens such as Vibrio, Tenacibaculum, and Aeromonas in Los Lagos, AysÃ©n, and Magallanes region were identified. Notably, the Chilean salmon industry was initially established in the Los Lagos region but it's currently moving to the south, where different oceanographic conditions coexist with lice populations. The results originated by this study will serve as foundation to investigate putative role of sea lice as vectors for fish pathogens and also as reservoirs for antibiotic-resistant genes.

[Source](https://www.nature.com/articles/s41598-020-59747-0)

10.1038/s41598-020-59747-0

---

## Insights on aquatic microbiome of the Indian Sundarbans mangrove areas.
 25.02.2020, _PK Dhal, GA Kopprio, A GÃ¤rdes_


Anthropogenic perturbations have strong impact on water quality and ecological health of mangrove areas of Indian Sundarbans. Diversity in microbial community composition is important causes for maintaining the health of the mangrove ecosystem. However, microbial communities of estuarine water in Indian Sundarbans mangrove areas and environmental determinants that contribute to those communities were seldom studied.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0221543)

10.1371/journal.pone.0221543

---

## Ganoderma lucidum cultivation affect microbial community structure of soil, wood segments and tree roots.
 05.03.2020, _F Ren, Y Zhang, H Yu, YA Zhang_


The popular medicinal mushroom Ganoderma lucidum (Fr.) Karst. [Ling Zhi] has been widely used for the general promotion of health and longevity in Asian countries. Continuous cultivation may affect soil microbe and soil properties. However, the effect of G. lucidum cultivation on related wood segments, soil and tree roots microbial communities and soil properties is remain unknown. In our study, the microbial communities of soils, wood segments, and tree roots before and after G. lucidum cultivation were investigated by Illumina Miseq sequencing of both ITS and 16S rDNA, and taxonomic composition of eukaryotic and prokaryotic microorganisms were observed. Indices of microbial richness, diversity and evenness significantly differed between before and after G. lucidum cultivation. Each of the investigated sampling type harbored a distinctive microbial community and differed remarkably before and after G. lucidum cultivation. Ascomycota and Basidiomycota (fungi), Proteobacteria and Actinobacteria (bacteria) showed significant differences after Ling Zhi cultivation. The soil property values also changed after cultivation. The redundancy analysis (RDA) showed that both the fungal and bacterial community structure significantly correlated with soil humus, pH, nitrogen, carbon and trace elements (Fe, Zn, Mn, Cu) contents. The results indicated that G. lucidum cultivation may have significant differed the associated microbial community structures and soil properties. The study will provide useful information for G. lucidum cultivation and under-forest economic development.

[Source](https://www.nature.com/articles/s41598-020-60362-2)

10.1038/s41598-020-60362-2

---

## Effect of long-term destocking on soil fungal functional groups and interactions with plants

[Source](https://link.springer.com/article/10.1007/s11104-020-04452-0)

---

## Comparative Analysis of Root Microbiomes of Rice Cultivars with High and Low Methane Emissions Reveals Differences in Abundance of Methanogenic Archaea and Putative Upstream Fermenters.
 27.02.2020, _Z Liechty, C Santos-MedellÃ­n, J Edwards, B Nguyen, D Mikhail, S Eason, G Phillips, V Sundaresan_


Rice cultivation worldwide accounts for â¼7 to 17% of global methane emissions. Methane cycling in rice paddies is a microbial process not only involving methane producers (methanogens) and methane metabolizers (methanotrophs) but also other microbial taxa that affect upstream processes related to methane metabolism. Rice cultivars vary in their rates of methane emissions, but the influence of rice genotypes on methane cycling microbiota has been poorly characterized. Here, we profiled the rhizosphere, rhizoplane, and endosphere microbiomes of a high-methane-emitting cultivar (Sabine) and a low-methane-emitting cultivar (CLXL745) throughout the growing season to identify variations in the archaeal and bacterial communities relating to methane emissions. The rhizosphere of the high-emitting cultivar was enriched in methanogens compared to that in the low emitter, whereas the relative abundances of methanotrophs between the cultivars were not significantly different. Further analysis of cultivar-sensitive taxa identified families enriched in the high emitter that are associated with methanogenesis-related processes. The high emitter had greater relative abundances of sulfate-reducing and iron-reducing taxa which peak earlier in the season than methanogens and are necessary to lower soil oxidation reduction potential before methanogenesis can occur. The high emitter also had a greater abundance of fermentative taxa which produce methanogenesis precursors (acetate, CO2, and H2). Furthermore, the high emitter was enriched in taxa related to acetogenesis which compete with methanogens for CO2 and H2 These taxa were enriched in a spatio-specific manner and reveal a complex network of microbial interactions on which plant genotype-dependent factors can act to affect methanogenesis and methane emissions.IMPORTANCE Rice cultivation is a major source of anthropogenic emissions of methane, a greenhouse gas with a potentially severe impact on climate change. Emission variation between rice cultivars suggests the feasibility of breeding low-emission rice, but there is a limited understanding of how genotypes affect the microbiota involved in methane cycling. Here, we show that the root microbiome of the high-emitting cultivar is enriched both in methanogens and in taxa associated with fermentation, iron, and sulfate reduction and acetogenesis, processes that support methanogenesis. Understanding how cultivars affect microbes with methanogenesis-related functions is vital for understanding the genetic basis for methane emission in rice and can aid in the development of breeding programs that reduce the environmental impact of rice cultivation.

[Source](https://msystems.asm.org/content/msys/5/1/e00897-19.full.pdf)

10.1128/mSystems.00897-19

---

## Fungal diversity and community composition of wheat rhizosphere and non-rhizosphere soils from three different agricultural production regions of South Africa

[Source](https://www.sciencedirect.com/science/article/pii/S0929139319309485)

---

## Microbial and volatile profiling of soils suppressive to Fusarium culmorum of wheat.
 19.02.2020, _A Ossowicki, V Tracanna, MLC Petrus, G van Wezel, JM Raaijmakers, MH Medema, P Garbeva_


In disease-suppressive soils, microbiota protect plants from root infections. Bacterial members of this microbiota have been shown to produce specific molecules that mediate this phenotype. To date, however, studies have focused on individual suppressive soils and the degree of natural variability of soil suppressiveness remains unclear. Here, we screened a large collection of field soils for suppressiveness to Fusarium culmorum using wheat (Triticum aestivum) as a model host plant. A high variation of disease suppressiveness was observed, with 14% showing a clear suppressive phenotype. The microbiological basis of suppressiveness to F. culmorum was confirmed by gamma sterilization and soil transplantation. Amplicon sequencing revealed diverse bacterial taxonomic compositions and no specific taxa were found exclusively enriched in all suppressive soils. Nonetheless, co-occurrence network analysis revealed that two suppressive soils shared an overrepresented bacterial guild dominated by various Acidobacteria. In addition, our study revealed that volatile emission may contribute to suppression, but not for all suppressive soils. Our study raises new questions regarding the possible mechanistic variability of disease-suppressive phenotypes across physico-chemically different soils. Accordingly, we anticipate that larger-scale soil profiling, along with functional studies, will enable a deeper understanding of disease-suppressive microbiomes.

[Source](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2019.2527#d3e1244)

10.1098/rspb.2019.2527

---

## Industrial development as a key factor explaining variances in soil and grass phyllosphere microbiomes in urban green spaces.
 02.03.2020, _ZZ Yan, QL Chen, YJ Zhang, JZ He, HW Hu_


Microbiota in urban green spaces underpin ecosystem services that are essential to environmental health and human wellbeing. However, the factors shaping the microbial communities in urban green spaces, especially those associated with turf grass phyllosphere, remain poorly understood. The lack of this knowledge greatly limits our ability to assess ecological, social and recreational benefits of urban green spaces in the context of global urbanization. In this study, we used amplicon sequencing to characterize soil and grass phyllosphere bacterial communities in 40 urban green spaces and three minimally disturbed national parks in Victoria, Australia. The results indicated that urbanization might have shown different impacts on soil and grass phyllosphere microbial communities. The bacterial diversity in soil but not in grass phyllosphere was significantly higher in urban green spaces than in national parks. Principal coordinate analysis revealed significant differences in the overall patterns of bacterial community composition between urban green spaces and national parks for both soil and grass phyllosphere. Industrial development, as represented by the number of industries in the region, was identified as a key driver shaping the bacterial community profiles in urban green spaces. Variation partitioning analysis suggested that industrial factors together with their interaction with other factors explained 20% and 28% of the variances in soil and grass phyllosphere bacterial communities, respectively. The findings highlight the importance of industrial development in driving the spatial patterns of urban microbiomes, and have important implication for the management of microbiomes in urban green spaces.

[Source](https://www.sciencedirect.com/science/article/pii/S026974911937558X)

10.1016/j.envpol.2020.114201

---

## Plant communities mediate the interactive effects of invasion and drought on soil microbial communities.
 20.02.2020, _C Fahey, A Koyama, PM Antunes, K Dunfield, SL Flory_


Soil microbiomes could play a major role in ecosystem responses to escalating anthropogenic global change. However, we currently have a poor understanding of how soil microbes will respond to interacting global change factors and if responses will be mediated by changes in plant community structure. We used a field experiment to assess changes in soil fungal and bacterial communities in response to plant invasion, experimental drought, and their combination. In addition, we evaluated the relative importance of direct versus indirect pathways of invasion and drought through changes in associated plant communities with structural equation models. We found that fungal communities were interactively structured by invasion and drought, where fungal richness was lowest with invasion under ambient conditions but highest with invasion under drought conditions. Bacterial richness was lower under drought but unaffected by invasion. Changes in the plant community, including lower plant richness and higher root biomass, moderated the direct effects of invasion on microbial richness. Fungal and bacterial functional groups, including pathogens, mutualists, and nitrogen metabolizers, were also influenced by plant community changes. In sum, plant communities mediated the effects of interacting global change drivers on soil microbial community structure, with significant potential consequences for community dynamics and ecosystem functions.

[Source](https://www.nature.com/articles/s41396-020-0614-6)

10.1038/s41396-020-0614-6

---

## Methyl t-butyl ether-degrading bacteria for bioremediation and biocontrol purposes.
 28.02.2020, _G d'Errico, V Aloj, V Ventorino, A Bottiglieri, E Comite, A Ritieni, R Marra, S Bolletti Censi, GR Flematti, O Pepe, F Vinale_


A total of fifteen potential methyl t-butyl ether (MtBE)-degrading bacterial strains were isolated from contaminated soil. They have been identified as belonging to the genera Bacillus, Pseudomonas, Kocuria, Janibacter, Starkeya, Bosea, Mycolicibacterium, and Rhodovarius. Bacillus aryabhattai R1B, S. novella R8b, and M. mucogenicum R8i were able to grow using MtBE as carbon source, exhibiting different growth behavior and contaminant degradation ability. Their biocontrol ability was tested against various fungal pathogens. Both S. novella R8b and B. aryabhattai were effective in reducing the development of necrotic areas on leaves within 48 hours from Botritys cinerea and Alternaria alternata inoculation. Whereas, M. mucogenicum effectively controlled B. cinerea after 72 hours. Similar results were achieved using Pythium ultimum, in which the application of isolated bacteria increased seed germination. Only M. mucogenicum elicited tomato plants resistance against B. cinerea. This is the first report describing the occurrence of bioremediation and biocontrol activities in M. mucogenicum, B. aryabhattai and S. novella species. The production of maculosin and its antibiotic activity against Rhizoctonia solani has been reported for first time from S. novella. Our results highlight the importance of multidisciplinary approaches to achieve a consistent selection of bacterial strains useful for plant protection and bioremediation purposes.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0228936)

10.1371/journal.pone.0228936

---

## Uncovering microbial responses to sharp geochemical gradients in a terrace contaminated by acid mine drainage

[Source](https://www.sciencedirect.com/science/article/pii/S0269749119368253)

---

## SnapShot: Microbial Extremophiles.
 24.02.2020, _AK Schmid, T Allers, J DiRuggiero_


Extremophiles are remarkable examples of life's resilience, thriving in hot springs at boiling temperatures, in brine lakes saturated with salt, and in the driest deserts. We review the biogeography, currently known limits of life, and molecular adaptations to extremes. See the online interactive map for additional detail on biogeography, environmental microbiology, and exemplary species. To view this SnapShot, open or download the PDF.

[Source](https://www.cell.com/cell/pdf/S0092-8674(20)30099-4.pdf)

10.1016/j.cell.2020.01.018

---

## Denitrification performance and microbial communities of solid-phase denitrifying reactors using poly (butylene succinate)/bamboo powder composite.
 05.03.2020, _W Qi, MJ Taherzadeh, Y Ruan, Y Deng, JS Chen, HF Lu, XY Xu_


This study explored the denitrification performance of solid-phase denitrification (SPD) systems packed with poly (butylene succinate)/bamboo powder composite to treat synthetic aquaculture wastewater under different salinity conditions (0â° Vs. 25â°). The results showed composite could achieve the maximum denitrification rates of 0.22 kg (salinity, 0â°) and 0.34 kg NO3--N m-3 d-1 (salinity, 25â°) over 200-day operation. No significant nitrite accumulation and less dissolved organic carbon (DOC) release (&lt;15 mg/L) were found. The morphological and spectroscopic analyses demonstrated the mixture composites degradation. Microbial community analysis showed that Acidovorax, Simplicispira, Denitromonas, SM1A02, Marinicella and Formosa were the dominant genera for denitrifying bacteria, while Aspergillus was the major genus for denitrifying fungus. The co-network analysis also indicated the interactions between bacterial and fungal community played an important role in composite degradation and denitrification. The outcomes provided a potential strategy of DOC control and cost reduction for aquaculture nitrate removal by SPD.

[Source](https://www.sciencedirect.com/science/article/pii/S0960852420303023)

10.1016/j.biortech.2020.123033

---

## Meta-omics Provides Insights into the Impact of Hydrocarbon Contamination on Microbial Mat Functioning.
 20.02.2020, _J AubÃ©, P Senin, P Bonin, O Pringault, C Jeziorski, O Bouchez, C Klopp, R Guyoneaud, M GoÃ±i-Urriza_


Photosynthetic microbial mats are stable, self-supported communities. Due to their coastal localization, these mats are frequently exposed to hydrocarbon contamination and are able to grow on it. To decipher how this contamination disturbs the functioning of microbial mats, we compared two mats: a contaminated mat exposed to chronic petroleum contamination and a reference mat. The taxonomic and metabolic structures of the mats in spring and fall were determined using metagenomic and metatranscriptomic approaches. Extremely high contamination disturbed the seasonal variations of the mat. ABC transporters, two-component systems, and type IV secretion system-related genes were overabundant in the contaminated mats. Xenobiotic degradation metabolism was minor in the metagenomes of both mats, and only the expression of genes involved in polycyclic aromatic hydrocarbon degradation was higher in the contaminated mat. Interestingly, the expression rates of genes involved in hydrocarbon activation decreased during the 1-year study period, concomitant with the decrease in easily degradable hydrocarbons, suggesting a transient effect of hydrocarbon contamination. Alteromonadales and Oceanospirillales hydrocarbonoclastic bacteria appeared to be key in hydrocarbon remediation in the contaminated mat. Overall, the contaminated microbial mat was able to cope with hydrocarbon contamination and displayed an adaptive functioning that modified seasonal behaviour.

[Source](https://link.springer.com/article/10.1007/s00248-020-01493-x)

10.1007/s00248-020-01493-x

---

## Role of microbial diversity for sustainable pyrite oxidation control in acid and metalliferous drainage prevention.
 05.03.2020, _OT Ogbughalu, S Vasileiadis, RC Schumann, AR Gerson, J Li, RSC Smart, MD Short_


Acid and metalliferous drainage (AMD) remains a challenging issue for the mining sector. AMD management strategies have attempted to shift from treatment of acid leachates post-generation to more sustainable at-source prevention. Here, the efficacy of microbial-geochemical at-source control approach was investigated over a period of 84 weeks. Diverse microbial communities were stimulated using organic carbon amendment in a simulated silicate-containing sulfidic mine waste rock environment. Mineral waste in the unamended leach system generated AMD quickly and throughout the study, with known lithotrophic iron- and sulfur-oxidising microbes dominating column communities. The organic-amended mineral waste column showed suppressed metal dissolution and AMD generation. Molecular DNA-based next generation sequencing confirmed a less diverse lithotrophic community in the acid-producing control, with a more diverse microbial community under organic amendment comprising organotrophic iron/sulfur-reducers, autotrophs, hydrogenotrophs and heterotrophs. Time-series multivariate statistical analyses displayed distinct ecological patterns in microbial diversity between AMD- and non-AMD-environments. Focused ion beam-TEM micrographs and elemental mapping showed that silicate-stabilised passivation layers were successfully established across pyrite surfaces in organic-amended treatments, with these layers absent in unamended controls. Organic amendment and resulting increases in microbial abundance and diversity played an important role in sustaining these passivating layers in the long-term.

[Source](https://www.sciencedirect.com/science/article/pii/S0304389420303265)

10.1016/j.jhazmat.2020.122338

---

## Perspectives on monoclonal antibody therapy as potential therapeutic intervention for Coronavirus disease-19 (COVID-19).
 05.03.2020, _B Shanmugaraj, K Siriwattananon, K Wangkanont, W Phoolcharoen_


Last decade witnessed the outbreak of many life-threatening human pathogens including Nipah, Ebola, Chikungunya, Zika, Middle East respiratory syndrome coronavirus (MERS-CoV), Severe Acute respiratory syndrome coronavirus (SARS-CoV) and more recently novel coronavirus (2019-nCoV or SARS-CoV-2). The disease condition associated with novel coronavirus, referred to as Coronavirus disease (COVID-19). The emergence of novel coronavirus in 2019 in Wuhan, China marked the third highly pathogenic coronavirus infecting humans in the 21st century. The continuing emergence of coronaviruses at regular intervals poses a significant threat to human health and economy. Ironically, even after a decade of research on coronavirus, still there are no licensed vaccines or therapeutic agents to treat coronavirus infection which highlights an urgent need to develop effective vaccines or post-exposure prophylaxis to prevent future epidemics. Several clinical, genetic and epidemiological features of COVID-19 resemble SARS-CoV infection. Hence, the research advancements on SARS-CoV treatment might help scientific community in quick understanding of this virus pathogenesis and develop effective therapeutic/prophylactic agents to treat and prevent this infection. Monoclonal antibodies represent the major class of biotherapeutics for passive immunotherapy to fight against viral infection. The therapeutic potential of monoclonal antibodies has been well recognized in the treatment of many diseases. Here, we summarize the potential monoclonal antibody based therapeutic intervention for COVID-19 by considering the existing knowledge on the neutralizing monoclonal antibodies against similar coronaviruses SARS-CoV and MERS-CoV. Further research on COVID-19 pathogenesis could identify appropriate therapeutic targets to develop specific anti-virals against this newly emerging pathogen.

[Source](https://www.biomedcentral.com/collections/coronavirus)

10.12932/AP-200220-0773

---

## Coronavirus research highlights

[Source](https://www.biomedcentral.com/collections/coronavirus)

---

## Fecal Viral Community Responses to High-Fat Diet in Mice.
 27.02.2020, _A Schulfer, TM Santiago-Rodriguez, M Ly, JM Borin, J Chopyk, MJ Blaser, DT Pride_


Alterations in diet can have significant impact on the host, with high-fat diet (HFD) leading to obesity, diabetes, and inflammation of the gut. Although membership and abundances in gut bacterial communities are strongly influenced by diet, substantially less is known about how viral communities respond to dietary changes. Examining fecal contents of mice as the mice were transitioned from normal chow to HFD, we found significant changes in the relative abundances and the diversity in the gut of bacteria and their viruses. Alpha diversity of the bacterial community was significantly diminished in response to the diet change but did not change significantly in the viral community. However, the diet shift significantly impacted the beta diversity in both the bacterial and viral communities. There was a significant shift away from the relatively abundant Siphoviridae accompanied by increases in bacteriophages from the Microviridae family. The proportion of identified bacteriophage structural genes significantly decreased after the transition to HFD, with a conserved loss of integrase genes in all four experimental groups. In total, this study provides evidence for substantial changes in the intestinal virome disproportionate to bacterial changes, and with alterations in putative viral lifestyles related to chromosomal integration as a result of shift to HFD.IMPORTANCE Prior studies have shown that high-fat diet (HFD) can have profound effects on the gastrointestinal (GI) tract microbiome and also demonstrate that bacteria in the GI tract can affect metabolism and lean/obese phenotypes. We investigated whether the composition of viral communities that also inhabit the GI tract are affected by shifts from normal to HFD. We found significant and reproducible shifts in the content of GI tract viromes after the transition to HFD. The differences observed in virome community membership and their associated gene content suggest that these altered viral communities are populated by viruses that are more virulent toward their host bacteria. Because HFD also are associated with significant shifts in GI tract bacterial communities, we believe that the shifts in the viral community may serve to drive the changes that occur in associated bacterial communities.

[Source](https://msphere.asm.org/content/msph/5/1/e00833-19.full.pdf)

10.1128/mSphere.00833-19

---

## Rapid MinION profiling of preterm microbiota and antimicrobial-resistant pathogens.
 06.03.2020, _RM Leggett, C Alcon-Giner, D Heavens, S Caim, TC Brook, M Kujawska, S Martin, N Peel, H Acford-Palmer, L Hoyles, P Clarke, LJ Hall, MD Clark_


The MinION sequencing platform offers near real-time analysis of DNA sequence; this makes the tool attractive for deployment in fieldwork or clinical settings. We used the MinION platform coupled to the NanoOK RT software package to perform shotgun metagenomic sequencing and profile mock communities and faecal samples from healthy and ill preterm infants. Using Nanopore data, we reliably classified a 20-species mock community and captured the diversity of the immature gut microbiota over time and in response to interventions such as probiotic supplementation, antibiotic treatment or episodes of suspected sepsis. We also performed rapid real-time runs to assess gut-associated microbial communities in critically ill and healthy infants, facilitated by NanoOK RT software package, which analysed sequences as they were generated. Our pipeline reliably identified pathogenic bacteria (that is, Klebsiella pneumoniae and Enterobacter cloacae) and their corresponding antimicrobial resistance gene profiles within as little as 1 h of sequencing. Results were confirmed using pathogen isolation, whole-genome sequencing and antibiotic susceptibility testing, as well as mock communities and clinical samples with known antimicrobial resistance genes. Our results demonstrate that MinION (including cost-effective Flongle flow cells) with NanoOK RT can process metagenomic samples to a rich dataset in &lt; 5 h, which creates a platform for future studies aimed at developing these tools and approaches in clinical settings with a focus on providing tailored patient antimicrobial treatment options.

[Source](https://www.nature.com/articles/s41564-019-0626-z)

10.1038/s41564-019-0626-z

---

## Assembly-free single-molecule sequencing recovers complete virus genomes from natural microbial communities.
 20.02.2020, _J Beaulaurier, E Luo, JM Eppley, P Den Uyl, X Dai, A Burger, DJ Turner, M Pendleton, S Juul, E Harrington, EF DeLong_


Viruses are the most abundant biological entities on Earth and play key roles in host ecology, evolution, and horizontal gene transfer. Despite recent progress in viral metagenomics, the inherent genetic complexity of virus populations still poses technical difficulties for recovering complete virus genomes from natural assemblages. To address these challenges, we developed an assembly-free, single-molecule nanopore sequencing approach enabling direct recovery of complete viral genome sequences from environmental samples. Our method yielded thousands of full-length, high-quality draft virus genome sequences that were not recovered using standard short-read assembly approaches. Additionally, our analyses discriminated between populations whose genomes had identical direct terminal repeats, versus those with circularly permuted repeats at their termini, thus providing new insight into native virus reproduction and genome packaging. Novel DNA sequences were discovered whose repeat structures, gene contents, and concatemer lengths suggest they are phage-inducible chromosomal islands, which are packaged as concatemers in phage particles, with lengths that match the size ranges of co-occurring phage genomes. Our new virus sequencing strategy can provide previously unavailable information about the genome structures, population biology, and ecology of naturally occurring viruses and viral parasites.

[Source](https://genome.cshlp.org/content/early/2020/02/19/gr.251686.119.abstract)

10.1101/gr.251686.119

---

## Rapid inference of antibiotic resistance and susceptibility by genomic neighbour typing.
 06.03.2020, _K BÅinda, A Callendrello, KC Ma, DR MacFadden, T Charalampous, RS Lee, L Cowley, CB Wadsworth, YH Grad, G Kucherov, J O'Grady, M Baym, WP Hanage_


Surveillance of drug-resistant bacteria is essential for healthcare providers to deliver effective empirical antibiotic therapy. However, traditional molecular epidemiology does not typically occur on a timescale that could affect patient treatment and outcomes. Here, we present a method called 'genomic neighbour typing' for inferring the phenotype of a bacterial sample by identifying its closest relatives in a database of genomes with metadata. We show that this technique can infer antibiotic susceptibility and resistance for both Streptococcus pneumoniae and Neisseria gonorrhoeae. We implemented this with rapid k-mer matching, which, when used on Oxford Nanopore MinION data, can run in real time. This resulted in the determination of resistance within 10 min (91% sensitivity and 100% specificity for S. pneumoniae and 81% sensitivity and 100% specificity for N. gonorrhoeae from isolates with a representative database) of starting sequencing, and within 4 h of sample collection (75% sensitivity and 100% specificity for S. pneumoniae) for clinical metagenomic sputum samples. This flexible approach has wide application for pathogen surveillance and may be used to greatly accelerate appropriate empirical antibiotic treatment.

[Source](https://www.nature.com/articles/s41564-019-0656-6)

10.1038/s41564-019-0656-6

---

## Tough microbe's DNA could be a lasting archive.
 03.03.2020, _S Nadis_



[Source](https://science.sciencemag.org/content/367/6480/840)

10.1126/science.367.6480.840

---

## Inflammatory bowel disease is associated with increased gut-to-blood penetration of short-chain fatty acids: A new, non-invasive marker of a functional intestinal lesion.
 01.08.2019, _K Jaworska, M Konop, K Bielinska, T Hutsch, M Dziekiewicz, A Banaszkiewicz, M Ufnal_


What is the central question of this study? 'Leaky gut' has been found in intestinal and extra-intestinal diseases. However, functional evaluation of intestinal permeability is not widely used as a diagnostic marker, possibly owing to significant limitations of currently used permeability assays. There is an unmet need for development of a new, non-invasive test to assess intestinal function. What is the main finding and its importance? We show that an increased blood-to-stool ratio of the concentration of gut bacteria-produced short-chain fatty acids may be used as a marker of gut permeability. Our findings lay the groundwork for establishing a new, non-invasive, risk-free diagnostic tool in diseases associated with intestinal barrier malfunction, such as inflammatory bowel disease.

[Source](https://www.kpbs.org/news/2020/feb/24/how-bacteria-can-sculpt-gut/)

10.1113/EP087773

---

## How Bacteria Can âSculpt The Gutâ

[Source](https://www.kpbs.org/news/2020/feb/24/how-bacteria-can-sculpt-gut/)

---

## The bee microbiome can fight back against fungi that cause Colony Collapse Disorder

[Source](https://massivesci.com/articles/bacteria-bees-colony-collapse-disorder-pathogens-biocontrol/)

---

## Ancient gut microbiomes shed light on human evolution

[Source](https://www.newswise.com/articles/ancient-gut-microbiomes-shed-light-on-human-evolution)

---

## Launch of the Marine Microbiome Roadmap

[Source](https://www.nioz.nl/en/news/launch-of-the-marine-microbiome-roadmap)

---

## Understanding Microbiome Resilience on a Changing Planet

[Source](https://vp.research.msu.edu/feature/understanding-microbiome-resilience-changing-planet)

---

## Adaptations in gut microbes helped our ancestors thrive in new geographical territories: Study

[Source](https://www.newkerala.com/news/2020/29079.htm)

---

## Scientists Announce VIRGO, First Comprehensive Gene Catalog To Decode the Vaginal Microbiome

[Source](https://www.prnewswire.com/news-releases/scientists-announce-virgo-first-comprehensive-gene-catalog-to-decode-the-vaginal-microbiome-301011341.html)

---

## Metabolomics Reveals Connection Between Gut Microbiome and Blood

[Source](https://www.metabolon.com/who-we-are/blog/metabolomics-reveals-connection-between-gut-microbiome-and-blood)

---

## Study shows universally positive effect of cover crops on soil microbiome

[Source](https://phys.org/news/2020-02-universally-positive-effect-crops-soil.html)

---

## New Center of Microbiome Science is in the works at Ohio State

[Source](https://news.osu.edu/new-center-of-microbiome-science-is-in-the-works-at-ohio-state/)

---

## Common diseases may be communicable through microbiome, says UBC researcher

[Source](https://www.ubyssey.ca/science/microbiome-communicable-diseases/)

---

## The microbes in your mouth, and a reminder to floss and go to the dentist

[Source](https://www.sciencedaily.com/releases/2020/03/200302162248.htm)

---

## For Your Microbiome, How You Eat Could Be As Important As What You Eat

[Source](https://www.technologynetworks.com/immunology/news/for-your-microbiome-how-you-eat-could-be-as-important-as-what-you-eat-331627)

---

## Study: Infectious gut bacteria may predict UTI risk

[Source](https://news.cornell.edu/stories/2020/02/study-infectious-gut-bacteria-may-predict-uti-risk)

---

## The Best Way to Avoid Getting Sick Is to Wash Your Hands â But Youâre Probably Doing It Wrong

[Source](https://www.livestrong.com/article/13725173-how-and-when-to-wash-your-hands/)

---

## DuPont carries on microbiome momentum with metabolic disease collaboration

[Source](https://www.nutraingredients.com/Article/2020/02/19/DuPont-enters-microbiome-metabolic-disease-collaboration#)

---

## Invest 5% of research funds in ensuring data are reusable

[Source](https://www.nature.com/articles/d41586-020-00505-7)

---

