## Microbiome Forum FOR EVERYONE

[Source](https://www.microbiome-vif.org/)

---

## Mechanism of Dyspnea during Exercise in Children with Corrected Congenital Heart Disease.
 24.01.2022, _M Chlif, MM Ammar, NB Said, L Sergey, S Ahmaidi, F Alassery, H Hamam_


_Child, Dyspnea, Exercise, Exercise Test, Heart Defects, Congenital, Humans, Oxygen Consumption_
This study will evaluate cardiorespiratory and peripheral muscle function and their relationship with subjective dyspnea threshold after the surgical correction of congenital heart disease in children. Thirteen children with surgically repaired congenital heart disease were recruited. Each participant performed an incremental exercise test on a cycle ergometer until exhaustion. Gas exchanges were continuously sampled to measure the maximal aerobic parameters and ventilatory thresholds. The functional capacity of the subjects was assessed with a 6 min walk test. At the end of the exercise test, isokinetic Cybex Norm was used to evaluate the strength and endurance of the knee extensor muscle in the leg. Dyspnea was subjectively scored with a visual analog scale during the last 15 s of each exercise step. Oxygen consumption measured at the dyspnea score/VO2 relationship located at the dyspnea threshold, at which dyspnea suddenly increased. Results: The maximal and submaximal values of the parameters describing the exercise and the peripheral muscular performances were: VO2 Peak: 33.8 ± 8.9 mL·min-1·kg-1; HR: 174 ± 9 b·min-1; VEmax: 65.68 ± 15.9 L·min-1; P max: 117 ± 27 W; maximal voluntary isometric force MVIF: 120.8 ± 41.9 N/m; and time to exhaustion Tlim: 53 ± 21 s. Oxygen consumption measured at the dyspnea threshold was related to VO2 Peak (R2 = 0.74; p &lt; 0.01), Tlim (R2 = 0.78; p &lt; 0.01), and the distance achieved during the 6MWT (R2 = 0.57; p &lt; 0.05). Compared to the theoretical maximal values for the power output, VO2, and HR, the surgical correction did not repair the exercise performance. After the surgical correction of congenital heart disease, exercise performance was impeded by alterations of the cardiorespiratory function and peripheral local factors. A subjective evaluation of the dyspnea threshold is a reliable criterion that allows the prediction of exercise capacity in subjects suffering from congenital heart disease.

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

10.3390/ijerph19010099

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## @MicroRocket

[Source](https://twitter.com/MicroRocket?ref_src=twsrc%5Etfw)

---

## @tejpalgill

[Source](https://twitter.com/tejpalgill?ref_src=twsrc%5Etfw)

---

## @Brooke_C_Wilson

[Source](https://twitter.com/Brooke_C_Wilson?ref_src=twsrc%5Etfw)

---

## @ameliamcg

[Source](https://twitter.com/ameliamcg?ref_src=twsrc%5Etfw)

---

## @SuEgan4

[Source](https://twitter.com/SuEgan4?ref_src=twsrc%5Etfw)

---

## https://t.co/MzsTYidE6J

[Source](https://t.co/MzsTYidE6J)

---

## March 28, 2022

[Source](https://twitter.com/MicrobiomeVIF/status/1508364474415263751?ref_src=twsrc%5Etfw)

---

## Global Distribution of mcr Gene Variants in 214K Metagenomic Samples

[Source](https://journals.asm.org/doi/10.1128/msystems.00105-22)

---

## Symposium Review: After the storm—Perspectives on the taxonomy of<em> Lactobacillaceae</em> 

[Source](https://www.sciencedirect.com/science/article/pii/S2666910222000503)

---

## Assessing the Effect of Smokeless Tobacco Consumption on Oral Microbiome in Healthy and Oral Cancer Patients

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2022.841465/full)

---

## https://t.co/jrz3zlJF20

[Source](https://t.co/jrz3zlJF20)

---

## @vks_mBioSys

[Source](https://twitter.com/vks_mBioSys?ref_src=twsrc%5Etfw)

---

## @ashoks773

[Source](https://twitter.com/ashoks773?ref_src=twsrc%5Etfw)

---

## April 1, 2022

[Source](https://twitter.com/VishnuPrasoodan/status/1509796156674809860?ref_src=twsrc%5Etfw)

---

## Unravelling specific diet and gut microbial contributions to inflammatory bowel disease

[Source](https://www.biorxiv.org/content/10.1101/2022.04.03.486886v1)

---

## Effects of caloric restriction on the gut microbiome are linked with immune senescence

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01249-4)

---

## Tap water as a natural vehicle for microorganisms shaping the human gut microbiome

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.15988)

---

## Diet-gut microbiota interactions on cardiovascular disease

[Source](https://www.sciencedirect.com/science/article/pii/S2001037022001039)

---

## Robust Variation in Infant Gut Microbiome Assembly Across a Spectrum of Lifestyles

[Source](https://www.biorxiv.org/content/10.1101/2022.03.30.486467v1)

---

## https://t.co/hYDKUuczdw

[Source](https://t.co/hYDKUuczdw)

---

## April 3, 2022

[Source](https://twitter.com/MattagenOlmics/status/1510742683781763078?ref_src=twsrc%5Etfw)

---

## Extensive metagenomic analysis of the porcine gut resistome to identify indicators reflecting antimicrobial resistance

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01241-y)

---

## Modulation of the gut microbiome by novel synthetic glycans for the production of propionate and the reduction of cardiometabolic risk factors

[Source](https://www.biorxiv.org/content/10.1101/2022.04.04.487010v1)

---

## The phyllosphere microbiome shifts toward combating melanose pathogen

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01234-x)

---

## Near-neutral pH increased n-caprylate production in a microbiome with product inhibition of methanogenesis

[Source](https://www.biorxiv.org/content/10.1101/2022.04.01.486710v1)

---

## Next-generation sequencing: insights to advance clinical investigations of the microbiome.
 04.04.2022, _CR Wensel, JL Pluznick, SL Salzberg, CL Sears_


_High-Throughput Nucleotide Sequencing, Humans, Metagenomics, Microbiota, RNA, Ribosomal, 16S, Sequence Analysis, RNA_
Next-generation sequencing (NGS) technology has advanced our understanding of the human microbiome by allowing for the discovery and characterization of unculturable microbes with prediction of their function. Key NGS methods include 16S rRNA gene sequencing, shotgun metagenomic sequencing, and RNA sequencing. The choice of which NGS methodology to pursue for a given purpose is often unclear for clinicians and researchers. In this Review, we describe the fundamentals of NGS, with a focus on 16S rRNA and shotgun metagenomic sequencing. We also discuss pros and cons of each methodology as well as important concepts in data variability, study design, and clinical metadata collection. We further present examples of how NGS studies of the human microbiome have advanced our understanding of human disease pathophysiology across diverse clinical contexts, including the development of diagnostics and therapeutics. Finally, we share insights as to how NGS might further be integrated into and advance microbiome research and clinical care in the coming years.

[Source](https://www.jci.org/articles/view/154944)

10.1172/JCI154944

---

## Metagenomics-enabled microbial surveillance.
 05.04.2022, _KKK Ko, KR Chng, N Nagarajan_


_Animals, COVID-19, Humans, Metagenomics, Pandemics, Viruses, Zoonoses_
Lessons learnt from the COVID-19 pandemic include increased awareness of the potential for zoonoses and emerging infectious diseases that can adversely affect human health. Although emergent viruses are currently in the spotlight, we must not forget the ongoing toll of morbidity and mortality owing to antimicrobial resistance in bacterial pathogens and to vector-borne, foodborne and waterborne diseases. Population growth, planetary change, international travel and medical tourism all contribute to the increasing frequency of infectious disease outbreaks. Surveillance is therefore of crucial importance, but the diversity of microbial pathogens, coupled with resource-intensive methods, compromises our ability to scale-up such efforts. Innovative technologies that are both easy to use and able to simultaneously identify diverse microorganisms (viral, bacterial or fungal) with precision are necessary to enable informed public health decisions. Metagenomics-enabled surveillance methods offer the opportunity to improve detection of both known and yet-to-emerge pathogens.

[Source](https://www.nature.com/articles/s41564-022-01089-w)

10.1038/s41564-022-01089-w

---

## Evolution, characterization, and immune response function of long-chain acyl-CoA synthetase genes in rainbow trout (Oncorhynchus mykiss) under hypoxic stress.
 06.04.2022, _F Ma, YZL Ma, RMX Chen_


Long-chain acyl-CoA synthetases (Acsls), members of the acyl-activating enzyme superfamily, haves been systematically characterized in mammals and certain fishes, but the research on their involvement in reproductive development and hypoxic stress response in rainbow trout remains limited. In this study, we investigated the acsl gene structure and physical and chemical characteristics and the evolutionary relationship among acsl genes using the NCBI/Ensembl database. Using hypoxia treatment experiment, acsl gene expression in various organs and its regulation were investigated. A total of 11 acsl genes were identified in rainbow trout. Phylogenetic analyses found that acsl genes in rainbow trout were clustered into two clades: acsl3/4 and acsl1/2/5/6, and the additional gene duplication observed resulted from the third round of genome duplication unique to teleosts. Multiple sequence alignment and conserved motif analyses showed that the sequence of acsl proteins was highly conserved. Real-time quantitative PCR (RT-qPCR) showed that the acsl genes were highly expressed in immune tissues (liver and head kidney). Under hypoxia, the expression of acsl genes was upregulated, suggesting that they enhance the tolerance to hypoxia and are involved in the immune response in rainbow trout. Our study provides valuable insights into teleost evolution and effects of hypoxia on biological immunity and form a basis for further research on the functional characteristics of acsl genes.

[Source](https://twitter.com/hashtag/bioinformatics?src=hash&ref_src=twsrc%5Etfw)

10.1016/j.cbpb.2022.110737

---

## #bioinformatics

[Source](https://twitter.com/hashtag/bioinformatics?src=hash&ref_src=twsrc%5Etfw)

---

## MicroRNAs as novel biomarkers for rivaroxaban therapeutic drug monitoring.
 06.04.2022, _E Rytkin, IV Bure, PO Bochkov, KA Akmalova, KB Mirzaev, MS Cherniaeva, OD Ostroumova, VV Smirnov, EA Grishina, AG Saribekian, IN Sychev, DA Sychev_


The aim of this study is to assess micro-RNAs miR-142 and miR-39 as potential biomarkers for drug-monitoring of rivaroxaban among elderly patients with atrial fibrillation.

[Source](https://twitter.com/hashtag/training?src=hash&ref_src=twsrc%5Etfw)

10.1515/dmpt-2021-0118

---

## #training

[Source](https://twitter.com/hashtag/training?src=hash&ref_src=twsrc%5Etfw)

---

## Preclinical Safety Evaluation: Acute and Repeated-Dose Toxicity of a New Intranasal Recombinant Vector Vaccine TB/FLU-04L Against Tuberculosis.
 06.04.2022, _K Stosman, K Sivak, A Aleksandrov, Z Buzitskaya, T Rassokha, M Stukova_


Vaccination against tuberculosis is one of the most successful medical measures to reduce morbidity and mortality. The BCG vaccine has been in use for more than 100 years, but its efficacy is still controversial. New vaccine candidates may offer better protection than available BCG vaccine. In this work, we studied the acute and the repeated-dose toxicity study of a new vector vaccine TB/Flu-04L against tuberculosis.

[Source](https://twitter.com/hashtag/materials?src=hash&ref_src=twsrc%5Etfw)

10.1055/a-1771-5985

---

## #materials

[Source](https://twitter.com/hashtag/materials?src=hash&ref_src=twsrc%5Etfw)

---

## @ISBSIB

[Source](https://twitter.com/ISBSIB?ref_src=twsrc%5Etfw)

---

## Breedbase: a digital ecosystem for modern plant breeding.
 06.04.2022, _N Morales, AC Ogbonna, BJ Ellerbrock, GJ Bauchet, T Tantikanjana, IY Tecle, AF Powell, D Lyon, N Menda, CC Simoes, S Saha, P Hosmani, M Flores, N Panitz, RS Preble, A Agbona, I Rabbi, P Kulakow, P Peteti, R Kawuki, W Esuma, M Kanaabi, DM Chelangat, E Uba, A Olojede, J Onyeka, T Shah, M Karanja, C Egesi, H Tufan, A Paterne, A Asfaw, JL Jannink, M Wolfe, CL Birkett, DJ Waring, JM Hershberger, MA Gore, KR Robbins, T Rife, C Courtney, J Poland, E Arnaud, MA Laporte, H Kulembeka, K Salum, E Mrema, A Brown, S Bayo, B Uwimana, V Akech, C Yencho, B de Boeck, H Campos, R Swennen, JD Edwards, LA Mueller_


Modern breeding methods integrate next-generation sequencing (NGS) and phenomics to identify plants with the best characteristics and greatest genetic merit for use as parents in subsequent breeding cycles to ultimately create improved cultivars able to sustain high adoption rates by farmers. This data-driven approach hinges on strong foundations in data management, quality control, and analytics. Of crucial importance is a central database able to 1) track breeding materials, 2) store experimental evaluations, 3) record phenotypic measurements using consistent ontologies, 4) store genotypic information, and 5) implement algorithms for analysis, prediction and selection decisions. Because of the complexity of the breeding process, breeding databases also tend to be complex, difficult, and expensive to implement and maintain. Here, we present a breeding database system, Breedbase (https://breedbase.org/). Originally initiated as Cassavabase (https://cassavabase.org/) with the NextGen Cassava project (https://www.nextgencassava.org/), and later developed into a crop-agnostic system, it is presently used by dozens of different crops and projects. The system is web-based and is available as open source software. It is available on GitHub (https://github.com/solgenomics/) and packaged in a Docker image for deployment (https://dockerhub.com/breedbase/). The Breedbase system enables breeding programs to better manage and leverage their data for decision making within a fully integrated digital ecosystem.

[Source](https://twitter.com/hashtag/github?src=hash&ref_src=twsrc%5Etfw)

10.1093/g3journal/jkac078

---

## #github

[Source](https://twitter.com/hashtag/github?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/xtRon0hIzP

[Source](https://t.co/xtRon0hIzP)

---

## March 31, 2022

[Source](https://twitter.com/geertvangeest/status/1509559476839497728?ref_src=twsrc%5Etfw)

---

## Software testing in microbial bioinformatics: a call to action

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000790)

---

## Machine Learning and Deep Learning Applications in Metagenomic Taxonomy and Functional Annotation

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2022.811495/full)

---

## Unifying the known and unknown microbial coding sequence space

[Source](https://elifesciences.org/articles/67667)

---

## NSURP

[Source](https://nsurp.org/)

---

## @the_real_KVD

[Source](https://twitter.com/the_real_KVD?ref_src=twsrc%5Etfw)

---

## @microbeMinded

[Source](https://twitter.com/microbeMinded?ref_src=twsrc%5Etfw)

---

## @BenjaminGVincen

[Source](https://twitter.com/BenjaminGVincen?ref_src=twsrc%5Etfw)

---

## https://t.co/W2nLX6lwwJ

[Source](https://t.co/W2nLX6lwwJ)

---

## April 5, 2022

[Source](https://twitter.com/NSURPorg/status/1511178781951619074?ref_src=twsrc%5Etfw)

---

