## Human microbiome variation associated with race and ethnicity emerges as early as 3 months of age

[Source](https://doi.org/10.1371/journal.pbio.3002230)

---

## Mapping the T cell repertoire to a complex gut bacterial community

[Source](https://doi.org/10.1038/s41586-023-06431-8)

---

## Comparison of the relative impacts of acute consumption of an inulin-enriched diet, milk kefir or a commercial probiotic product on the human gut microbiome and metabolome.
 19.08.2023, _LH Walsh, AM Walsh, I Garcia-Perez, F Crispie, A Costabile, R Ellis, J Finlayson, LA Finnegan, MJ Claesson, E Holmes, PD Cotter_


It has been established that the human gut microbiota is central to health, and, consequently, there has been a growing desire to positively modulate its composition and/or function through, for example, the use of fermented foods, prebiotics or probiotics. Here, we compare the relative impact of the daily consumption of an inulin-enriched diet (n = 10), a commercial probiotic-containing fermented milk product (FMP) (n = 10), or a traditional kefir FMP (n = 9), over a 28-day period on the gut microbiome and urine metabolome of healthy human adults. None of the treatments resulted in significant changes to clinical parameters or biomarkers tested. However, shotgun metagenomic analysis revealed that kefir consumption resulted in a significant change in taxonomy, in the form of an increased abundance of the sub-dominant FMP-associated species Lactococcus raffinolactis, which further corresponded to shifts in the urine metabolome. Overall, our results indicated that daily consumption of a single portion of kefir alone resulted in detectable changes to the gut microbiota and metabolome of consumers.

[Source](https://www.nature.com/articles/s41538-023-00216-z)

10.1038/s41538-023-00216-z

---

## High-resolution strain-level microbiome composition analysis from short reads.
 19.08.2023, _H Liao, Y Ji, Y Sun_


_Microbiota, Metagenome, Metagenomics, Software_
Bacterial strains under the same species can exhibit different biological properties, making strain-level composition analysis an important step in understanding the dynamics of microbial communities. Metagenomic sequencing has become the major means for probing the microbial composition in host-associated or environmental samples. Although there are a plethora of composition analysis tools, they are not optimized to address the challenges in strain-level analysis: highly similar strain genomes and the presence of multiple strains under one species in a sample. Thus, this work aims to provide a high-resolution and more accurate strain-level analysis tool for short reads.

[Source](https://doi.org/10.1186/s40168-023-01615-w)

10.1186/s40168-023-01615-w

---

## Insights into the Evolution of Host Association through the Isolation and Characterization of a Novel Human Periodontal Pathobiont, Desulfobulbus oralis.
 27.12.2019, _KL Cross, P Chirania, W Xiong, CJ Beall, JG Elkins, RJ Giannone, AL Griffen, AM Guss, RL Hettich, SS Joshi, EM Mokrzan, RK Martin, IB Zhulin, EJ Leys, M Podar_


_Adaptation, Biological, Deltaproteobacteria, Evolution, Molecular, Gene Expression Profiling, Gene Transfer, Horizontal, Genome, Bacterial, Gingiva, Humans, Ohio, Periodontitis, Phylogeny, Proteome_
The human oral microbiota encompasses representatives of many bacterial lineages that have not yet been cultured. Here we describe the isolation and characterization of previously uncultured Desulfobulbus oralis, the first human-associated representative of its genus. As mammalian-associated microbes rarely have free-living close relatives, D. oralis provides opportunities to study how bacteria adapt and evolve within a host. This sulfate-reducing deltaproteobacterium has adapted to the human oral subgingival niche by curtailing its physiological repertoire, losing some biosynthetic abilities and metabolic independence, and by dramatically reducing environmental sensing and signaling capabilities. The genes that enable free-living Desulfobulbus to synthesize the potent neurotoxin methylmercury were also lost by D. oralis, a notably positive outcome of host association. However, horizontal gene acquisitions from other members of the microbiota provided novel mechanisms of interaction with the human host, including toxins like leukotoxin and hemolysins. Proteomic and transcriptomic analysis revealed that most of those factors are actively expressed, including in the subgingival environment, and some are secreted. Similar to other known oral pathobionts, D. oralis can trigger a proinflammatory response in oral epithelial cells, suggesting a direct role in the development of periodontal disease.IMPORTANCE Animal-associated microbiota likely assembled as a result of numerous independent colonization events by free-living microbes followed by coevolution with their host and other microbes. Through specific adaptation to various body sites and physiological niches, microbes have a wide range of contributions, from beneficial to disease causing. Desulfobulbus oralis provides insights into genomic and physiological transformations associated with transition from an open environment to a host-dependent lifestyle and the emergence of pathogenicity. Through a multifaceted mechanism triggering a proinflammatory response, D. oralis is a novel periodontal pathobiont. Even though culture-independent approaches can provide insights into the potential role of the human microbiome "dark matter," cultivation and experimental characterization remain important to studying the roles of individual organisms in health and disease.

[Source](https://www.biorxiv.org/content/10.1101/2023.08.08.552500v2.abstract)

10.1128/mBio.02061-17

---

## Profiling novel lateral gene transfer events in the human microbiome

[Source](https://www.biorxiv.org/content/10.1101/2023.08.08.552500v2.abstract)

---

## Major data analysis errors invalidate cancer microbiome findings.
 14.08.2023, _A Gihawi, Y Ge, J Lu, D Puiu, A Xu, CS Cooper, DS Brewer, M Pertea, SL Salzberg_


We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.

[Source](https://www.biorxiv.org/content/10.1101/2023.07.28.550993v1)

10.1101/2023.07.28.550993

---

## Microdiversity of the vaginal microbiome is associated with preterm birth.
 21.08.2023, _J Liao, L Shenhav, JA Urban, M Serrano, B Zhu, GA Buck, T Korem_


_Infant, Newborn, Pregnancy, Humans, Female, Premature Birth, Microbiota, Metagenome, Acclimatization, Biological Evolution_
Preterm birth (PTB) is the leading cause of neonatal morbidity and mortality. The vaginal microbiome has been associated with PTB, yet the mechanisms underlying this association are not fully understood. Understanding microbial genetic adaptations to selective pressures, especially those related to the host, may yield insights into these associations. Here, we analyze metagenomic data from 705 vaginal samples collected during pregnancy from 40 women who delivered preterm spontaneously and 135 term controls from the Multi-Omic Microbiome Study-Pregnancy Initiative. We find that the vaginal microbiome of pregnancies that ended preterm exhibited unique genetic profiles. It was more genetically diverse at the species level, a result which we validate in an additional cohort, and harbored a higher richness and diversity of antimicrobial resistance genes, likely promoted by transduction. Interestingly, we find that Gardnerella species drove this higher genetic diversity, particularly during the first half of the pregnancy. We further present evidence that Gardnerella spp. underwent more frequent recombination and stronger purifying selection in genes involved in lipid metabolism. Overall, our population genetics analyses reveal associations between the vaginal microbiome and PTB and suggest that evolutionary processes acting on vaginal microbes may play a role in adverse pregnancy outcomes such as PTB.

[Source](https://doi.org/10.1038/s41467-023-40719-7)

10.1038/s41467-023-40719-7

---

## Dysbiosis of gut microbiota during fecal stream diversion in patients with colorectal cancer.
 21.08.2023, _SY Lee, HM Park, CH Kim, HR Kim_


The effect of fecal stream diversion on the gut microbiota is still uncertain. The present study was designed to assess the effect of fecal stream diversion on the composition of the gut microbiota in patients with colorectal cancer. We included patients undergoing left-sided colorectal cancer surgery with (ileostomy group) or without (control group) diverting ileostomy. Fecal samples were collected from 10 patients in each group before surgery (t1) and after ileostomy repair in the ileostomy group and 6-12 months after the initial surgery in the control group (t2). The fecal microbiota was assessed using 16S rRNA sequencing, and changes in the composition of the fecal microbiota were compared between the two groups.

[Source](https://doi.org/10.1186/s13099-023-00566-9)

10.1186/s13099-023-00566-9

---

## The gut microbiome contributes to somatic morphine withdrawal behavior and implicates a TLR2 mediated mechanism.
 20.08.2023, _B Truitt, G Venigalla, P Singh, S Singh, J Tao, I Chupikova, S Roy_


The ongoing opioid epidemic has left millions of people suffering from opioid use disorder due to the over-prescription of highly addictive substances. Chronic opioid exposure leads to dependence, where the absence of the drug results in negative symptoms of withdrawal, often driving patients to continue drug use; however, few therapeutic strategies are currently available to combat the cycle of addiction and the severity of morphine withdrawal. This study investigates the microbiome as a potential therapeutic target for morphine withdrawal, as gut dysbiosis caused by morphine use has been proven to contribute to other aspects of opioid use disorders, such as tolerance. Results show that although the microbiome during morphine withdrawal trends toward recovery from morphine-induced dysbiosis, there continues to be a disruption in the alpha and beta diversity as well as the abundance of gram-positive bacteria that may still contribute to the severity of morphine withdrawal symptoms. Germ-free mice lacking the microbiome did not develop somatic withdrawal symptoms, indicating that the microbiome is necessary for the development of somatic withdrawal behavior. Notably, only TLR2 but not TLR4 whole-body knockout models display less withdrawal severity, implicating that the microbiome, through a gram-positive, TLR2 mediated mechanism, drives opioid-induced somatic withdrawal behavior.

[Source](https://doi.org/10.1080/19490976.2023.2242610)

10.1080/19490976.2023.2242610

---

## Association of the human gut microbiota with vascular stiffness.
 19.08.2023, _RRC Cuadrat, T Goris, A Birukov, F Eichelmann, BGN Andrade, C Bang, A Franke, C Wittenbecher, MB Schulze_


_Humans, Gastrointestinal Microbiome, Cross-Sectional Studies, Prospective Studies, Vascular Stiffness, Cancer Vaccines_
Gut microbiota metabolites have been mechanistically linked to inflammatory pathway activation and atherosclerosis, which are major causes of vascular stiffness (VS). Aiming to investigate if the gut microbiome might be involved in VS development, we performed a cross-sectional study (n = 3,087), nested within the population-based European Prospective Investigations into Cancer and Nutrition (EPIC) Potsdam. We investigated the correlation of the gut microbiota (alpha diversity and taxa abundance) with 3 vascular stiffness measures: carotid-femoral (PWV), aortic augmentation index (AIX) and ankle-brachial index (ABI). Shannon index was not significantly associated with VS but the number of observed Amplicon Sequence Variants (ASV) was positively associated with PWV and AIX. We found a total of 19 ASVs significantly associated with at least one VS measure in multivariable-adjusted models. One ASV (classified as Sutterella wadsworthensis) was associated with 2 VS measures, AIX (- 0.11 ± 0.04) and PWV (-0.14 ± 0.03). Other examples of ASVs associated with VS were Collinsella aerofaciens, previously reported to be affected by diet and Bacteroides uniformis, commercially available as probiotics. In conclusion, our study suggests a potential role of individual components of the gut microbiota in the aetiology of VS.

[Source](https://doi.org/10.1038/s41598-023-40178-6)

10.1038/s41598-023-40178-6

---

## Gut microbiota and intervertebral disc degeneration: a bidirectional two-sample Mendelian randomization study.
 16.08.2023, _Z Geng, J Wang, G Chen, J Liu, J Lan, Z Zhang, J Miao_


_Humans, Gastrointestinal Microbiome, Genome-Wide Association Study, Intervertebral Disc Degeneration, Mendelian Randomization Analysis, Reproducibility of Results_
Although previous studies have suggested a close association between gut microbiota (GM) and intervertebral disc degeneration (IVDD), the causal relationship between them remains unclear. Hence, we thoroughly investigate their causal relationship by means of a two-sample Mendelian randomization (MR) study, aiming to determine the impact of gut microbiota on the risk of developing intervertebral disc degeneration.

[Source](https://doi.org/10.1186/s13018-023-04081-0)

10.1186/s13018-023-04081-0

---

## The gut microbiota-brain axis in neurological disorder.
 21.08.2023, _H Ullah, S Arbab, Y Tian, CQ Liu, Y Chen, L Qijie, MIU Khan, IU Hassan, K Li_


The gut microbiota (GM) plays an important role in the physiology and pathology of the host. Microbiota communicate with different organs of the organism by synthesizing hormones and regulating body activity. The interaction of the central nervous system (CNS) and gut signaling pathways includes chemical, neural immune and endocrine routes. Alteration or dysbiosis in the gut microbiota leads to different gastrointestinal tract disorders that ultimately impact host physiology because of the abnormal microbial metabolites that stimulate and trigger different physiologic reactions in the host body. Intestinal dysbiosis leads to a change in the bidirectional relationship between the CNS and GM, which is linked to the pathogenesis of neurodevelopmental and neurological disorders. Increasing preclinical and clinical studies/evidence indicate that gut microbes are a possible susceptibility factor for the progression of neurological disorders, including Alzheimer's disease (AD), Parkinson's disease (PD), multiple sclerosis (MS) and autism spectrum disorder (ASD). In this review, we discuss the crucial connection between the gut microbiota and the central nervous system, the signaling pathways of multiple biological systems and the contribution of gut microbiota-related neurological disorders.

[Source](https://doi.org/10.1038/s41564-023-01455-2)

10.3389/fnins.2023.1225875

---

## Microbiome alterations in autism spectrum disorder

[Source](https://doi.org/10.1038/s41564-023-01455-2)

---

## Rosmarinic acid alleviates intestinal inflammatory damage and inhibits endoplasmic reticulum stress and smooth muscle contraction abnormalities in intestinal tissues by regulating gut microbiota.
 18.08.2023, _K Li, J Wu, S Xu, X Li, Y Zhang, XJ Gao_


The host-bacterial interactions play the key role in inflammatory bowel disease (IBD). Dysbiosis of the intestinal flora can lead to pathological changes in the intestine. Rosmarinic acid (RA) is a natural phenolic acid compound with antioxidant, anti-cancer, anti-inflammatory, anti-apoptotic, anti-fibrotic, and anti-bacterial activities that has a palliative effect on acute IBD. We have established an in vivo model for mice. Histological staining was performed to directly observe RA alterations in the intestinal tract. The alteration of RA on mouse intestinal flora was observed by 16S rRNA high-throughput sequencing, and the effect of RA on intestinal mechanism of action was detected by qPCR and western blot. The results showed that RA had a significant protective effect on the intestine. RA upregulated the abundance of Lactobacillus johnsonii and Candidatus Arthromitus sp SFB-mouse-NL and downregulated the abundance of Bifidobacterium pseudolongum, Escherichia coli, and Romboutsia ilealis. RA downregulated the expressions of ROCK, RhoA, CaM, MLC, MLCK, ZEB1, ZO-1, ZO-2, occludin, E-cadherin, IL-1β, IL-6, TNF-α, GRP78, PERK, IRE1, ATF6, CHOP, Caspase12, Caspase9, Caspase3, Bax, Cytc, RIPK1, RIPK3, MLKL, and upregulated the expression of IL-10 and Bcl-2. These results displayed that RA inhibited the inflammation, which is caused by tight junction damage, by repairing intestinal flora dysbiosis, relieved endoplasmic reticulum stress, inhibited cell death, and corrected smooth muscle contractile dysregulation. The results of this study revealed RA could have a protective effect on the small intestine of mice by regulating intestinal flora. IMPORTANCE Inflammatory bowel disease (IBD) is a chronic, relapsing, remitting disorder of the gastrointestinal system. In this study, we investigated the protective effects of rosmarinic acid on the intestinal tract. The results showed that RA was effective in reducing inflammatory damage, endoplasmic reticulum stress, smooth muscle contraction abnormalities, and regulating intestinal flora disorders.

[Source](https://doi.org/10.1128/spectrum.01914-23)

10.1128/spectrum.01914-23

---

## <em>Prevotella copri </em>alleviates sarcopenia via attenuating muscle mass loss and function decline

[Source](https://onlinelibrary.wiley.com/doi/full/10.1002/jcsm.13313)

---

## The Influence of Protein Secretomes of <em>Enterococcus durans</em> on ex vivo Human Gut Microbiome

[Source](https://doi.org/10.1007/s12602-023-10136-9)

---

## Kombucha tea as an anti-hyperglycemic agent in humans with diabetes – a randomized controlled pilot investigation

[Source](https://doi.org/10.3389/fnut.2023.1190248)

---

## The unique gut microbiome of giant pandas involved in protein metabolism contributes to the host’s dietary adaption to bamboo

[Source](https://doi.org/10.1186/s40168-023-01603-0)

---

## Systematic review of cnidarian microbiomes reveals insights into the structure, specificity, and fidelity of marine associations

[Source](https://doi.org/10.1038/s41467-023-39876-6)

---

## Toward an Integrated Understanding of the Lepidoptera Microbiome

[Source](https://doi.org/10.1146/annurev-ento-020723-102548)

---

## Characterization of bacterial community in tobacco leaves at flue-curing and redrying processing stages

[Source](https://doi.org/10.1038/s41598-023-40502-0)

---

## Inter-comparison of marine microbiome sampling protocols

[Source](https://doi.org/10.1038/s43705-023-00278-w)

---

## Discovery of a novel bacterial class with the capacity to drive sulfur cycling and microbiome structure in a paleo-ocean analog

[Source](https://doi.org/10.1038/s43705-023-00287-9)

---

## MT-MAG: Accurate and interpretable machine learning for complete or partial taxonomic assignments of metagenomeassembled genomes.
 21.08.2023, _W Li, L Kari, Y Yu, LA Hug_


_Animals, Cattle, Female, Humans, Metagenome, Benchmarking, Computational Biology, Gastrointestinal Microbiome, Machine Learning_
We propose MT-MAG, a novel machine learning-based software tool for the complete or partial hierarchically-structured taxonomic classification of metagenome-assembled genomes (MAGs). MT-MAG is alignment-free, with k-mer frequencies being the only feature used to distinguish a DNA sequence from another (herein k = 7). MT-MAG is capable of classifying large and diverse metagenomic datasets: a total of 245.68 Gbp in the training sets, and 9.6 Gbp in the test sets analyzed in this study. In addition to complete classifications, MT-MAG offers a "partial classification" option, whereby a classification at a higher taxonomic level is provided for MAGs that cannot be classified to the Species level. MT-MAG outputs complete or partial classification paths, and interpretable numerical classification confidences of its classifications, at all taxonomic ranks. To assess the performance of MT-MAG, we define a "weighted classification accuracy," with a weighting scheme reflecting the fact that partial classifications at different ranks are not equally informative. For the two benchmarking datasets analyzed (genomes from human gut microbiome species, and bacterial and archaeal genomes assembled from cow rumen metagenomic sequences), MT-MAG achieves an average of 87.32% in weighted classification accuracy. At the Species level, MT-MAG outperforms DeepMicrobes, the only other comparable software tool, by an average of 34.79% in weighted classification accuracy. In addition, MT-MAG is able to completely classify an average of 67.70% of the sequences at the Species level, compared with DeepMicrobes which only classifies 47.45%. Moreover, MT-MAG provides additional information for sequences that it could not classify at the Species level, resulting in the partial or complete classification of 95.13%, of the genomes in the datasets analyzed. Lastly, unlike other taxonomic assignment tools (e.g., GDTB-Tk), MT-MAG is an alignment-free and genetic marker-free tool, able to provide additional bioinformatics analysis to confirm existing or tentative taxonomic assignments.

[Source](https://doi.org/10.1371/journal.pone.0283536)

10.1371/journal.pone.0283536

---

## BinaRena: a dedicated interactive platform for human-guided exploration and binning of metagenomes.
 21.08.2023, _MJ Pavia, A Chede, Z Wu, H Cadillo-Quiroz, Q Zhu_


_Humans, Metagenome, Microbiota, Algorithms, Biological Evolution, Diarrhea_
Exploring metagenomic contigs and "binning" them into metagenome-assembled genomes (MAGs) are essential for the delineation of functional and evolutionary guilds within microbial communities. Despite the advances in automated binning algorithms, their capabilities in recovering MAGs with accuracy and biological relevance are so far limited. Researchers often find that human involvement is necessary to achieve representative binning results. This manual process however is expertise demanding and labor intensive, and it deserves to be supported by software infrastructure.

[Source](https://doi.org/10.1186/s40168-023-01625-8)

10.1186/s40168-023-01625-8

---

## Detection of viable commensal bacteria in murine melanoma tumors by culturomics.
 14.08.2023, _CM Phelps, JH Shapira, CR Laughlin, M Meisel_


Emerging evidence suggests the tumor microbiome at gut-distal sites can modulate tumor immunity and response to cancer immunotherapy. However, detection of commensal bacteria at gut-distal tumor sites is challenging given their low abundance. Here, we present a culturomics approach to facilitate recovery of phylogenetically diverse live commensal bacteria within gut-distal melanoma tumors. We describe steps for media preparation, tissue isolation, tissue homogenization, and host cell lysis. We then detail broth expansion culture followed by agar culture and single-colony 16S rRNA sequencing. For complete details on the use and execution of this protocol, please refer to Bender and McPherson et al. (2023).1.

[Source](https://doi.org/10.1016/j.xpro.2023.102492)

10.1016/j.xpro.2023.102492

---

