## Macrophages in Microbial Pathogenesis: Commonalities of Defense Evasion Mechanisms.
 15.11.2021, _S Pandey, S Kant, M Khawary, D Tripathi_


Macrophages are key arsenals of the immune system against invaders. After compartmental isolation of a pathogen in phagosomes, the host immune response attempts to neutralize the pathogen. However, pathogens possess the ability to subvert these assaults and can also convert macrophages into their replicative niche. The multiple host defense evasion mechanisms employed by these pathogens like phagosome maturation arrest, molecular mimicry through secretory antigens, interference with host signaling, active radical neutralization, inhibition of phagosome acidification, alteration of programmed cell death and many other mechanisms. Macrophage biology as a part of the host-pathogen interaction has expanded rapidly in the past decade. The present review aims to shed some light upon the macrophage defense evasion strategies employed by infecting pathogens. We have also incorporated recent knowledge in the field of macrophage dynamics during infection and evolutionary perspectives of macrophage dynamics.

[Source](https://journals.asm.org/doi/10.1128/IAI.00291-21)

10.1128/IAI.00291-21

---

## The Influence of FUT2 and FUT3 Polymorphisms and Nasopharyngeal Microbiome on Respiratory Infections in Breastfed Bangladeshi Infants from the Microbiota and Health Study.
 13.11.2021, _A Binia, L Siegwald, S Sultana, M Shevlyakova, G Lefebvre, F Foata, S Combremont, A Charpagne, K Vidal, N Sprenger, M Rahman, A Palleja, AC Eklund, HB Nielsen, H Brüssow, SA Sarker, O Sakwinska_


Acute respiratory infections (ARIs) are one of the most common causes of morbidity and mortality in young children. The aim of our study was to examine whether variation in maternal FUT2 (α1,2-fucosyltransferase 2) and FUT3 (α1,3/4-fucosyltransferase 3) genes, which shape fucosylated human milk oligosaccharides (HMOs) in breast milk, are associated with the occurrence of ARIs in breastfed infants as well as the influence of the nasopharyngeal microbiome on ARI risk. Occurrences of ARIs were prospectively recorded in a cohort of 240 breastfed Bangladeshi infants from birth to 2 years. Secretor and Lewis status was established by sequencing of FUT2/3 genes. The nasopharyngeal microbiome was characterized by shotgun metagenomics, complemented by specific detection of respiratory pathogens; 88.6% of mothers and 91% of infants were identified as secretors. Maternal secretor status was associated with reduced ARI incidence among these infants in the period from birth to 6 months (incidence rate ratio [IRR], 0.66; 95% confidence interval [CI], 0.47 to 0.94; P = 0.020), but not at later time periods. The nasopharyngeal microbiome, despite precise characterization to the species level, was not predictive of subsequent ARIs. The observed risk reduction of ARIs among infants of secretor mothers during the predominant breastfeeding period is consistent with the hypothesis that fucosylated oligosaccharides in human milk contribute to protection against respiratory infections. However, we found no evidence that modulation of the nasopharyngeal microbiome influenced ARI risk. IMPORTANCE The observed risk reduction of acute respiratory infections (ARIs) among infants of secretor mothers during the predominant breastfeeding period is consistent with the hypothesis that fucosylated oligosaccharides in human milk contribute to protection against respiratory infections. Respiratory pathogens were only weak modulators of risk, and the nasopharyngeal microbiome did not influence ARI risk, suggesting that the associated protective effects of human milk oligosaccharides (HMOs) are not conveyed via changes in the nasopharyngeal microbiome. Our observations add to the evidence for a role of fucosylated HMOs in protection against respiratory infections in exclusively or predominantly breastfed infants in low-resource settings. There is no indication that the nasopharyngeal microbiome substantially modulates the risk of subsequent mild ARIs. Larger studies are needed to provide mechanistic insights on links between secretor status, HMOs, and risk of respiratory infections.

[Source](https://journals.asm.org/doi/10.1128/mSphere.00686-21)

10.1128/mSphere.00686-21

---

## Fecal Microbiota Transplantation Commonly Failed in Children with Co-Morbidities.
 11.11.2021, _R Kellermayer, Q Wu, D Nagy-Szakal, K Queliza, FD Ihekweazu, CE Bocchini, AR Magee, N Oezguen, JK Spinler, EB Hollister, RJ Shulman, J Versalovic, RA Luna, TC Savidge_


Fecal microbiota transplantation (FMT) is arguably the most effective treatment for recurrent Clostridioides difficile infection (rCDI). Clinical reports on pediatric FMT have not systematically evaluated microbiome restoration in patients with co-morbidities. Here we determined whether FMT recipient age and underlying co-morbidity influenced clinical outcomes and microbiome restoration when treated from shared fecal donor sources.

[Source](https://journals.lww.com/jpgn/Abstract/9000/Fecal_Microbiota_Transplantation_Commonly_Failed.95495.aspx)

10.1097/MPG.0000000000003336

---

## Current and future applications of fecal microbiota transplantation for children.
 15.11.2021, _CC Chen, CH Chiu_


Fecal microbiota transplantation (FMT) is a new and adequate route to modify the microbial ecosystem in gastrointestinal tract of the hosts. Intestinal microbiota is highly associated with human health and disease. According to the reports of human clinical trials or case series, the application of FMT ranged from Clostridiodes difficile infection (CDI), inflammatory bowel disease (IBD), irritable bowel syndrome, refractory diarrhea, diabetes mellitus, metabolic syndrome, and even neurologic diseases, including Parkinson disease, and neuropsychiatric disorder (autism spectrum disorder, ASD). Although the current allowed indication of FMT is CDI in Taiwan, more application and development are expectable in the future. There is a relative rare data available for children in application of fecal microbiota transplantation. Thus, we review previous published research inspecting FMT in children, and address particular considerations when conducting FMT in pediatric patients.

[Source](https://www.sciencedirect.com/science/article/pii/S2319417021001530)

10.1016/j.bj.2021.11.004

---

## Microbial influences on gut development and gut-brain communication.
 10.11.2021, _L Ye, JF Rawls_


The developmental programs that build and sustain animal forms also encode the capacity to sense and adapt to the microbial world within which they evolved. This is abundantly apparent in the development of the digestive tract, which typically harbors the densest microbial communities of the body. Here, we review studies in human, mouse, zebrafish and Drosophila that are revealing how the microbiota impacts the development of the gut and its communication with the nervous system, highlighting important implications for human and animal health.

[Source](https://journals.biologists.com/dev/article-abstract/148/21/dev194936/273402/Microbial-influences-on-gut-development-and-gut?redirectedFrom=fulltext)

10.1242/dev.194936

---

## Characterization and predictive functional profiles on metagenomic 16S rRNA data of liver transplant recipients: A longitudinal study.
 15.11.2021, _S Sivaraj, JK Copeland, A Malik, E Pasini, M Angeli, A Azhie, S Husain, D Kumar, J Allard, DS Guttman, A Humar, M Bhat_


Long-term survival after Liver Transplantation (LT) is often compromised by infectious and metabolic complications. We aimed to delineate alterations in intestinal microbiome (IM) over time that could contribute to medical complications compromising long-term survival following LT. Fecal samples from LT recipients were collected at 3 months (3M) and 6 months (6M) post-LT. The bacterial DNA was extracted using E.Z.N.A.® Stool DNA Kit and 16S rRNA gene sequencing at V4 hypervariable region was performed. DADA2 and Phyloseq was implemented to analyze the taxonomic composition. Differentially abundant taxa were identified by metagenomeSeq and LEfSe. Piphillin, an Inferred functional metagenomic analysis tool was used to study the bacterial functional content. For comparison, healthy samples were extracted from NCBI and analyzed similarly. The taxonomic &amp; functional profiles of LT recipients were validated with metagenomic sequencing data from animals exposed to immunosuppressants using Venny. Our findings provide a new perspective on longitudinal increase in specific IM communities post-LT along with an increase in bacterial genes associated with metabolic and infectious disease. This article is protected by copyright. All rights reserved.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/ctr.14534)

10.1111/ctr.14534

---

## Changes to human faecal microbiota after international travel.
 15.11.2021, _C Kampmann, J Dicksved, L Engstrand, H Rautelin_


The aim was to investigate whether travelling to less-resourced destinations influences the composition of faecal microbiota in generally healthy adults.

[Source](https://www.sciencedirect.com/science/article/pii/S1477893921002404)

10.1016/j.tmaid.2021.102199

---

## Cognate recognition of microbial antigens defines constricted CD4+ T cell receptor repertoires in the inflamed colon.
 11.11.2021, _M Muschaweck, L Kopplin, F Ticconi, A Schippers, A Iljazovic, EJC Gálvez, AT Abdallah, N Wagner, IG Costa, T Strowig, O Pabst_


Key aspects of intestinal T cells, including their antigen specificity and their selection by the microbiota and other intestinal antigens, as well as the contribution of individual T cell clones to regulatory and effector functions, remain unresolved. Here we tracked adoptively transferred T cell populations to specify the interrelation of T cell receptor repertoire and the gut antigenic environment. We show that dominant TCRα clonotypes were shared between interferon-γ- and interleukin-17-producing but not regulatory Foxp3+ T cells. Identical TCRα clonotypes accumulated in the colon of different individuals, whereas antibiotics or defined colonization correlated with the expansion of distinct expanded T cell clonotypes. Our results demonstrate key aspects of intestinal CD4+ T cell activation and suggest that few microbial species exert a dominant effect on the intestinal T cell repertoire during colitis. We speculate that dominant proinflammatory T cell clones might provide a therapeutic target in human inflammatory bowel disease.

[Source](https://www.sciencedirect.com/science/article/pii/S1074761321003460)

10.1016/j.immuni.2021.08.014

---

## An aniline-substituted bile salt analog protects both mice and hamsters from multiple Clostridioides difficile strains.
 15.11.2021, _JR Phan, DM Do, MC Truong, C Ngo, JH Phan, SK Sharma, A Schilke, CC Mefferd, JV Villarama, D Lai, A Consul, BP Hedlund, SM Firestine, E Abel-Santos_


Clostridioides difficile infection (CDI) is the major identifiable cause of antibiotic-associated diarrhea. The emergence of hypervirulent C. difficile strains has led to increases in both hospital- and community-acquired CDI. Furthermore, CDI relapse from hypervirulent strains can reach up to 25%. Thus, standard treatments are rendered less effective, making new methods of prevention and treatment more critical. Previously, the bile salt analog CamSA was shown to inhibit spore germination in vitro and protect mice and hamsters from C. difficile strain 630. Here, we show that CamSA was less active at preventing spore germination of other C. difficile ribotypes, including the hypervirulent strain R20291. Strain-specific in vitro germination activity of CamSA correlated with its ability to prevent CDI in mice. Additional bile salt analogs were screened for in vitro germination inhibition activity against strain R20291, and the most active compounds were tested against other strains. An aniline-substituted bile salt analog, (CaPA), was found to be a better anti-germinant than CamSA against eight different C. difficile strains. In addition, CaPA was capable of reducing, delaying, or preventing murine CDI signs in all strains tested. CaPA-treated mice showed no obvious toxicity and showed minor effects on their gut microbiome. CaPA's efficacy was further confirmed by its ability to prevent CDI in hamsters infected with strain 630. These data suggest that C. difficile spores respond to germination inhibitors in a strain-dependent manner. However, careful screening can identify anti-germinants with broad CDI prophylaxis activity.

[Source](https://journals.asm.org/doi/10.1128/AAC.01435-21)

10.1128/AAC.01435-21

---

## Analysis of Cow-Calf Microbiome Transfer Routes and Microbiome Diversity in the Newborn Holstein Dairy Calf Hindgut.
 12.11.2021, _H Zhu, M Yang, JJ Loor, A Elolimy, L Li, C Xu, W Wang, S Yin, Y Qu_


Hindgut microorganisms in newborn calves play an important role in the development of immunity and metabolism, and optimization of performance. However, knowledge of the extent to which microbiome colonization of the calf intestine is dependent on maternal characteristics is limited. In this study, placenta, umbilical cord, amniotic fluid, colostrum, cow feces, and calf meconium samples were collected from 6 Holstein cow-calf pairs. Microbial composition was analyzed by 16S rRNA gene high-throughput sequencing, and maternal transfer characteristics assessed using SourceTracker based on Gibbs sampling to fit the joint distribution using the mean proportions of each sample with meconium as the "sink" and other sample types as different "sources." Alpha and beta diversity analyses revealed sample type-specific microbiome features: microbial composition of the placenta, umbilical cord, amniotic fluid, colostrum, and calf feces were similar, but differed from cow feces (p &lt; 0.05). Compared with profiles of meconium vs. placenta, meconium vs. umbilical cord, and meconium vs. colostrum, differences between the meconium and amniotic fluid were most obvious. SourceTracker analysis revealed that 23.8 ± 2.21% of the meconium OTUs matched those of umbilical cord samples, followed by the meconium-placenta pair (15.57 ± 2.2%), meconium-colostrum pair (14.4 ± 1.9%), and meconium-amniotic fluid pair (11.2 ± 1.7%). The matching ratio between meconium and cow feces was the smallest (10.5 ± 1%). Overall, our data indicated that the composition of the meconium microflora was similar compared with multiple maternal sites including umbilical cord, placenta, colostrum, and amniotic fluid. The umbilical cord microflora seemed to contribute the most to colonization of the fecal microflora of calves. Bacteria with digestive functions such as cellulose decomposition and rumen fermentation were mainly transmitted during the maternal transfer process.

[Source](https://www.frontiersin.org/articles/10.3389/fnut.2021.736270/full)

10.3389/fnut.2021.736270

---

## Multi-omic evaluation of metabolic alterations in multiple sclerosis identifies shifts in aromatic amino acid metabolism.
 11.11.2021, _KC Fitzgerald, MD Smith, S Kim, ES Sotirchos, MD Kornberg, M Douglas, B Nourbakhsh, J Graves, R Rattan, L Poisson, M Cerghet, EM Mowry, E Waubant, S Giri, PA Calabresi, P Bhargava_


The circulating metabolome provides unique insights into multiple sclerosis (MS) pathophysiology, but existing studies are relatively small or characterized limited metabolites. We test for differences in the metabolome between people with MS (PwMS; n = 637 samples) and healthy controls (HC; n = 317 samples) and assess the association between metabolomic profiles and disability in PwMS. We then assess whether metabolic differences correlate with changes in cellular gene expression using publicly available scRNA-seq data and whether identified metabolites affect human immune cell function. In PwMS, we identify striking abnormalities in aromatic amino acid (AAA) metabolites (p = 2.77E-18) that are also strongly associated with disability (p = 1.01E-4). Analysis of scRNA-seq data demonstrates altered AAA metabolism in CSF and blood-derived monocyte cell populations in PwMS. Treatment with AAA-derived metabolites in vitro alters monocytic endocytosis and pro-inflammatory cytokine production. We identify shifts in AAA metabolism resulting in the reduced production of immunomodulatory metabolites and increased production of metabotoxins in PwMS.

[Source](https://www.sciencedirect.com/science/article/pii/S266637912100286X)

10.1016/j.xcrm.2021.100424

---

## It takes guts to learn: machine learning techniques for disease detection from the gut microbiome.
 15.11.2021, _KD Curry, MG Nute, TJ Treangen_


Associations between the human gut microbiome and expression of host illness have been noted in a variety of conditions ranging from gastrointestinal dysfunctions to neurological deficits. Machine learning (ML) methods have generated promising results for disease prediction from gut metagenomic information for diseases including liver cirrhosis and irritable bowel disease, but have lacked efficacy when predicting other illnesses. Here, we review current ML methods designed for disease classification from microbiome data. We highlight the computational challenges these methods have effectively overcome and discuss the biological components that have been overlooked to offer perspectives on future work in this area.

[Source](https://portlandpress.com/emergtoplifesci/article/doi/10.1042/ETLS20210213/230242/It-takes-guts-to-learn-machine-learning-techniques)

10.1042/ETLS20210213

---

