## CRISPR-Cas systems are widespread accessory elements across bacterial and archaeal plasmids

[Source](https://www.biorxiv.org/content/10.1101/2021.06.04.447074v1.abstract?%3Fcollection=)

---

## Indoor bacterial, fungal and viral species and functional genes in urban and rural schools in Shanxi Province, China–association with asthma, rhinitis and rhinoconjunctivitis in high school students

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01091-0)

---

## Insight into the ecology of vaginal bacteria through integrative analyses of metagenomic and metatranscriptomic data

[Source](https://www.biorxiv.org/content/10.1101/2021.06.17.448822v1.abstract?%3Fcollection=)

---

## Review: microbial transformations of human bile acids.
 18.06.2021, _DV Guzior, RA Quinn_


Bile acids play key roles in gut metabolism, cell signaling, and microbiome composition. While the liver is responsible for the production of primary bile acids, microbes in the gut modify these compounds into myriad forms that greatly increase their diversity and biological function. Since the early 1960s, microbes have been known to transform human bile acids in four distinct ways: deconjugation of the amino acids glycine or taurine, and dehydroxylation, dehydrogenation, and epimerization of the cholesterol core. Alterations in the chemistry of these secondary bile acids have been linked to several diseases, such as cirrhosis, inflammatory bowel disease, and cancer. In addition to the previously known transformations, a recent study has shown that members of our gut microbiota are also able to conjugate amino acids to bile acids, representing a new set of "microbially conjugated bile acids." This new finding greatly influences the diversity of bile acids in the mammalian gut, but the effects on host physiology and microbial dynamics are mostly unknown. This review focuses on recent discoveries investigating microbial mechanisms of human bile acids and explores the chemical diversity that may exist in bile acid structures in light of the new discovery of microbial conjugations. Video Abstract.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01101-1)

10.1186/s40168-021-01101-1

---

## Microbial transformations of human bile acids

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01101-1)

---

## Targeted high-resolution taxonomic identification of <em>Bifidobacterium longum </em>subsp<em>. infantis</em> using human milk oligosaccharide metabolizing genes

[Source](https://www.biorxiv.org/content/10.1101/2021.06.04.447087v1.abstract?%3Fcollection=)

---

## An integrated gene catalog and over 10,000 metagenome-assembled genomes from the gastrointestinal microbiome of ruminants.
 18.06.2021, _F Xie, W Jin, H Si, Y Yuan, Y Tao, J Liu, X Wang, C Yang, Q Li, X Yan, L Lin, Q Jiang, L Zhang, C Guo, C Greening, R Heller, LL Guan, PB Pope, Z Tan, W Zhu, M Wang, Q Qiu, Z Li, S Mao_


Gastrointestinal tract (GIT) microbiomes in ruminants play major roles in host health and thus animal production. However, we lack an integrated understanding of microbial community structure and function as prior studies. are predominantly biased towards the rumen. Therefore, to acquire a microbiota inventory of the discrete GIT compartments, In this study, we used shotgun metagenomics to profile the microbiota of 370 samples that represent 10 GIT regions of seven ruminant species.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01078-x)

10.1186/s40168-021-01078-x

---

## Leveraging host-genetics and gut microbiota to determine immunocompetence in pigs

[Source](https://www.biorxiv.org/content/10.1101/2021.06.08.447584v1.abstract?%3Fcollection=)

---

## Joint contributions of the gut microbiota and host genetics to feed efficiency in chickens.
 05.06.2021, _C Wen, W Yan, C Mai, Z Duan, J Zheng, C Sun, N Yang_


Feed contributes most to livestock production costs. Improving feed efficiency is crucial to increase profitability and sustainability for animal production. Host genetics and the gut microbiota can both influence the host phenotype. However, the association between the gut microbiota and host genetics and their joint contribution to feed efficiency in chickens is largely unclear.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01040-x)

10.1186/s40168-021-01040-x

---

## Holobiont nitrogen control and its potential for eutrophication resistance in an obligate photosymbiotic jellyfish

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01075-0)

---

## Microbial metabolism and adaptations in <em>Atribacteria</em>-dominated methane hydrate sediments

[Source](https://www.biorxiv.org/content/10.1101/536078v2.abstract?%3Fcollection=)

---

## Seed microbiota revealed by a large-scale meta-analysis including 50 plant species

[Source](https://www.biorxiv.org/content/10.1101/2021.06.08.447541v1.abstract?%3Fcollection=)

---

## The activity and functions of subarctic soil microbial communities vary across vegetation types

[Source](https://www.biorxiv.org/content/10.1101/2021.06.12.448001v1.abstract?%3Fcollection=)

---

## Revealing the full biosphere structure and versatile metabolic functions in the deepest ocean sediment of the Challenger Deep

[Source](https://www.biorxiv.org/content/10.1101/2021.06.05.447043v1.abstract?%3Fcollection=)

---

## Mechanistic insights into the success of xenobiotic degraders resolved from metagenomes of microbial enrichment cultures

[Source](https://www.biorxiv.org/content/10.1101/2021.03.03.433815v2.abstract?%3Fcollection=)

---

## Low shifts in salinity determined assembly processes and network stability of microeukaryotic plankton communities in a subtropical urban reservoir

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01079-w)

---

## Linking genomic and physiological characteristics of psychrophilic <em>Arthrobacter</em> to metagenomic data to explain global environmental distribution

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01084-z)

---

## Genomic adaptations enabling <em>Acidithiobacillus</em> distribution across wide-ranging hot spring temperatures and pHs

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01090-1)

---

## Metagenome-based comparisons of decay rates and host-specificity of fecal microbial communities for improved microbial source tracking

[Source](https://www.biorxiv.org/content/10.1101/2021.06.17.448865v1.abstract?%3Fcollection=)

---

## Expanding clinical phage microbiology: Simulating phage inhalation for respiratory tract infections

[Source](https://www.biorxiv.org/content/10.1101/2021.06.14.448272v1.abstract?%3Fcollection=)

---

## Ultra-accurate microbial amplicon sequencing with synthetic long reads.
 14.06.2021, _BJ Callahan, D Grinevich, S Thakur, MA Balamotis, TB Yehezkel_


Out of the many pathogenic bacterial species that are known, only a fraction are readily identifiable directly from a complex microbial community using standard next generation DNA sequencing. Long-read sequencing offers the potential to identify a wider range of species and to differentiate between strains within a species, but attaining sufficient accuracy in complex metagenomes remains a challenge.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01072-3)

10.1186/s40168-021-01072-3

---

## Evaluation of RNAlater as a field-compatible preservation method for metaproteomic analyses of bacteria-animal symbioses

[Source](https://www.biorxiv.org/content/10.1101/2021.06.16.448770v1.abstract?%3Fcollection=)

---

## Predicting microbiomes through a deep latent space.
 19.06.2021, _B García-Jiménez, J Muñoz, S Cabello, J Medina, MD Wilkinson_


Microbial communities influence their environment by modifying the availability of compounds, such as nutrients or chemical elicitors. Knowing the microbial composition of a site is therefore relevant to improve productivity or health. However, sequencing facilities are not always available, or may be prohibitively expensive in some cases. Thus, it would be desirable to computationally predict the microbial composition from more accessible, easily-measured features.

[Source](https://www.biorxiv.org/content/10.1101/2021.06.17.448886v1.abstract?%3Fcollection=)

10.1093/bioinformatics/btaa971

---

## Predicting microbiome compositions through deep learning

[Source](https://www.biorxiv.org/content/10.1101/2021.06.17.448886v1.abstract?%3Fcollection=)

---

