## <em>Aggregatibacter</em> is inversely associated with inflammatory mediators in sputa of patients with chronic airway diseases and reduces inflammation <em>in vitro</em>

[Source](https://doi.org/10.1186/s12931-024-02983-z)

---

## Oral Microbiota Development in the First 60 Months: A Longitudinal Study.
 12.10.2024, _K Yama, S Morishima, K Tsutsumi, R Jo, Y Aita, T Inokuchi, T Okuda, D Watai, K Ohara, M Maruyama, T Chikazawa, T Iwamoto, Y Kakizawa, T Oniki_


Childhood is considered crucial in the establishment of future oral microbiota. However, the precise period of oral microbiota development remains unclear. This study aimed to identify the progression of oral microbiota formation in children. We longitudinally investigated the salivary microbiota of 54 children across 13 time points from 1 wk to 60 mo (5 y) old and their parents at 2 time points as a representative sample of the adult microbiota. Using next-generation sequencing, we obtained 10,000 gene sequences of the 16s rRNA V1-V2 region for each sample. The detection rate in children of 110 operational taxonomic units commonly detected in more than 85% of mothers and fathers, defined as the main constituent bacteria, was 25% at 1 wk old, increased to 80% between 6 and 18 mo old, and reached approximately 90% by 36 mo old. Early main constituent bacteria detected at 1 wk old were limited to Streptococcus, Rothia, and Gemella. At 6 to 18 mo old, the detection rates of various main constituent bacteria, including Neisseria, Haemophilus, and Fusobacterium, increased. UniFrac distance analysis showed that the oral microbiota of children approached that of adults at 6 to 18 mo old. In the weighted UniFrac distance index, unlike the unweighted index, there were no significant changes in children between 36 and 60 mo old from adults, and microbiota formation at 60 mo old was sufficiently advanced to be included within the range of adult individual differences. Our findings suggest that the initial 36 mo, particularly the period from 6 to 18 mo old, consists of a time window for oral microbiota maturation. In addition, the development of microbiota during this period may be critical for future oral disease prevention.

[Source](https://doi.org/10.1177/00220345241272011)

10.1177/00220345241272011

---

## Microbiome-derived metabolites in early to mid-pregnancy and risk of gestational diabetes: a metabolome-wide association study

[Source](https://doi.org/10.1186/s12916-024-03606-6)

---

## Unique Gut Microbiome and Metabolic Profiles in Chinese Workers Exposed to Dust: Insights from a Case-Control Study.
 11.10.2024, _X Qian, Y Liu, X Wei, X Chen, G Rong, X Hu_


This study aimed to identify distinct gut microbiome and serum metabolic features in workers exposed to dust compared to healthy controls.

[Source](https://doi.org/10.1097/JOM.0000000000003243)

10.1097/JOM.0000000000003243

---

## Dysbiotic signatures and diagnostic potential of gut microbial markers for inflammatory bowel disease in Korean population.
 13.10.2024, _HS Kim, SJ Oh, BK Kim, JE Kim, BH Kim, YK Park, BG Yang, JY Lee, JW Bae, CK Lee_


_Humans, Gastrointestinal Microbiome, Dysbiosis, Female, Male, Republic of Korea, Adult, Middle Aged, Biomarkers, RNA, Ribosomal, 16S, Feces, Inflammatory Bowel Diseases, Colitis, Ulcerative, Metagenomics, Crohn Disease, Case-Control Studies, Cross-Sectional Studies, Young Adult, Aged_
Fecal samples were collected from 640 individuals in Korea, including 523 patients with IBD (223 with Crohn's disease [CD] and 300 with ulcerative colitis [UC]) and 117 healthy controls. The samples were subjected to cross-sectional gut metagenomic analysis using 16 S rRNA sequencing and bioinformatics analysis. Patients with IBD, particularly those with CD, exhibited significantly lower alpha diversities than the healthy subjects. Differential abundance analysis revealed dysbiotic signatures, characterized by an expansion of the genus Escherichia-Shigella in patients with CD. Functional annotations showed that functional pathways related to bacterial pathogenesis and production of hydrogen sulfide (H2S) were strongly upregulated in patients with CD. A dysbiosis score, calculated based on functional characteristics, highly correlated with disease severity. Markers distinguishing between healthy subjects and patients with IBD showed accurate classification based on a small number of microbial taxa, which may be used to diagnose ambiguous cases. These findings confirm the taxonomic and functional dysbiosis of the gut microbiota in patients with IBD, especially those with CD. Taxa indicative of dysbiosis may have significant implications for future clinical research on the management and diagnosis of IBD.

[Source](https://doi.org/10.1038/s41598-024-74002-6)

10.1038/s41598-024-74002-6

---

## A search for bacteria identified from cerebrospinal fluid shunt infections in previous surgical events

[Source](https://doi.org/10.1371/journal.pone.0311605)

---

## Microbiome and metabolome patterns after lung transplantation reflect underlying disease and chronic lung allograft dysfunction.
 12.10.2024, _C Martin, KS Mahan, TD Wiggen, AJ Gilbertsen, MI Hertz, RC Hunter, RA Quinn_


_Humans, Lung Transplantation, Metabolome, Microbiota, Bronchoalveolar Lavage Fluid, Male, Female, Middle Aged, Adult, RNA, Ribosomal, 16S, Allografts, Aged, Bacteria, Lung, Metabolomics, Lung Diseases, Cystic Fibrosis, Pulmonary Disease, Chronic Obstructive_
Progression of chronic lung disease may lead to the requirement for lung transplant (LTx). Despite improvements in short-term survival after LTx, chronic lung allograft dysfunction (CLAD) remains a critical challenge for long-term survival. This study investigates the molecular and microbial relationships between underlying lung disease and the development of CLAD in bronchoalveolar lavage fluid (BALF) from subjects post-LTx, which is crucial for tailoring treatment strategies specific to allograft dysfunctions.

[Source](https://doi.org/10.1186/s40168-024-01893-y)

10.1186/s40168-024-01893-y

---

## Reexamining the role of <em>Fusobacterium nucleatum</em> subspecies in clinical and experimental studies

[Source](https://doi.org/10.1080/19490976.2024.2415490)

---

## Host-microbe interaction paradigms in acute and recurrent vulvovaginal candidiasis.
 13.10.2024, _J MacAlpine, MS Lionakis_


_Candidiasis, Vulvovaginal, Humans, Female, Vagina, Candida, Recurrence, Host Microbial Interactions, Neutrophils, Host-Pathogen Interactions, Animals, Microbiota_
Candida spp. are members of the human mucosal microbiota that can cause opportunistic diseases ranging from superficial infections to life-threatening invasive candidiasis. In humans, the most common infection caused by Candida spp. is vulvovaginal candidiasis (VVC), which affects &gt;70% of women at least once in their lifetime. Of those women, ∼5%-10% develop recurrent VVC (RVVC). In this review, we summarize our current understanding of the host and fungal factors that contribute to susceptibility to VVC and RVVC. We synthesize key findings that support the notion that disease symptoms are driven by neutrophil-associated dysfunction and immunopathology and describe how antifungal immune mechanisms in the vagina are distinct from other mucosal barrier sites. Finally, we highlight key, unanswered research areas within the field that can help us better understand the immunopathogenesis of this infection and facilitate the development of novel preventive, therapeutic, and/or vaccination strategies to combat these common, poorly understood diseases.

[Source](https://doi.org/10.1016/j.chom.2024.08.018)

10.1016/j.chom.2024.08.018

---

## Balancing act: counteracting adverse drug effects on the microbiome.
 12.10.2024, _J de la Cuesta-Zuluaga, P Müller, L Maier_


The human gut microbiome, a community of microbes that plays a crucial role in our wellbeing, is highly adaptable but also vulnerable to drug treatments. This vulnerability can have serious consequences for the host, for example, increasing susceptibility to infections, immune, metabolic, and cognitive disorders. However, the microbiome's adaptability also provides opportunities to prevent, protect, or even reverse drug-induced damage. Recently, several innovative approaches have emerged aimed at minimizing the collateral damage of drugs on the microbiome. Here, we outline these approaches, discuss their applicability in different treatment scenarios, highlight current challenges, and suggest avenues that may lead to an effective protection of the microbiome.

[Source](https://doi.org/10.1016/j.tim.2024.09.011)

10.1016/j.tim.2024.09.011

---

## Gut microbiome and bladder cancer: A new link through nitrosamine metabolism.
 10.10.2024, _S Mani_


_Urinary Bladder Neoplasms, Gastrointestinal Microbiome, Humans, Nitrosamines, Animals, Carcinogens, Butylhydroxybutylnitrosamine, Mice_
A recent Nature paper1 reveals that gut microbes metabolize N-butyl-N-(4-hydroxybutyl)-nitrosamine (BBN) into the bladder carcinogen N-n-butyl-N-(3-carboxypropyl)-nitrosamine (BCPN) in the intestines, establishing a direct link between gut microbial activity and the development of bladder cancer.

[Source](https://doi.org/10.1016/j.chom.2024.09.003)

10.1016/j.chom.2024.09.003

---

## Prostate cancer microenvironment: multidimensional regulation of immune cells, vascular system, stromal cells, and microbiota

[Source](https://doi.org/10.1186/s12943-024-02137-1)

---

## Gut and oral microbiota in gynecological cancers: interaction, mechanism, and therapeutic value.
 13.10.2024, _J Wu, J Li, M Yan, Z Xiang_


_Humans, Female, Genital Neoplasms, Female, Gastrointestinal Microbiome, Dysbiosis, Mouth, Microbiota_
Gynecologic cancers develop from the female reproductive organs. Microbial dysbiosis in the gut and oral cavity can communicate with each other through various ways, leading to mucosal destruction, inflammatory response, genomic instability, and ultimately inducing cancer and worsening. Here, we introduce the mechanisms of interactions between gut and oral microbiota and their changes in the development of gynecologic tumors. In addition, new therapeutic approaches based on microbiota modulation are discussed.

[Source](https://doi.org/10.1038/s41522-024-00577-7)

10.1038/s41522-024-00577-7

---

## Effects of complex probiotics on intestinal function and its regulatory mechanism in patients with constipation

[Source](https://doi.org/10.1163/18762891-bja00039)

---

## <em>Lactiplantibacillus plantarum</em> P101 Alleviated Alcohol-Induced Hepatic Lipid Accumulation in Mice via AMPK Signaling Pathway: Gut Microbiota and Metabolomics Analysis

[Source](https://doi.org/10.1007/s12602-024-10373-6)

---

## Unlocking the genetic influence on milk variation and its potential implication for infant health.
 10.10.2024, _C Nussbaum, S Kim-Hellmuth_


_Humans, Milk, Human, Infant Health, Infant, Female, Gastrointestinal Microbiome_
Human milk has long been recognized for its critical role in infant and maternal health. In this issue of Cell Genomics, Johnson et al.1 apply a human genetics and genomics approach to shed light on the complex relationship between maternal genetics, milk variation, and the infant gut microbiome.

[Source](https://doi.org/10.1016/j.xgen.2024.100676)

10.1016/j.xgen.2024.100676

---

## The gut microbiome and dietary fibres: implications in obesity, cardiometabolic diseases and cancer.
 10.10.2024, _NM Delzenne, LB Bindels, AM Neyrinck, J Walter_


Dietary fibres constitute a heterogeneous class of nutrients that are key in the prevention of various chronic diseases. Most dietary fibres are fermented by the gut microbiome and may, thereby, modulate the gut microbial ecology and metabolism, impacting human health. Dietary fibres may influence the occurrence of specific bacterial taxa, with this effect varying between individuals. The effect of dietary fibres on microbial diversity is a matter of debate. Most intervention studies with dietary fibres in the context of obesity and related metabolic disorders reveal the need for an accurate assessment of the microbiome to better understand the variable response to dietary fibres. Epidemiological studies confirm that a high dietary fibre intake is strongly associated with a reduced occurrence of many types of cancer. However, there is a need to determine the impact of intervention with specific dietary fibres on cancer risk, therapy efficacy and toxicity, as well as in cancer cachexia. In this Review, we summarize the mechanisms by which the gut microbiome can mediate the physiological benefits of dietary fibres in the contexts of obesity, cardiometabolic diseases and cancer, their incidence being clearly linked to low dietary fibre intake.

[Source](https://doi.org/10.1038/s41579-024-01108-z)

10.1038/s41579-024-01108-z

---

## Microbiome-based therapeutics for Parkinson’s disease

[Source](https://doi.org/10.1016/j.neurot.2024.e00462)

---

## Gut bacteria of lepidopteran herbivores facilitate digestion of plant toxins.
 11.10.2024, _N Zhang, Z Qian, J He, X Shen, X Lei, C Sun, J Fan, GW Felton, Y Shao_


_Animals, Herbivory, Gastrointestinal Microbiome, Bombyx, Morus, Symbiosis, Lepidoptera, Spodoptera, Bacteria, Digestion_
Lepidopterans commonly feed on plant material, being the most significant insect herbivores in nature. Despite plant resistance to herbivory, such as producing toxic secondary metabolites, herbivores have developed mechanisms encoded in their genomes to tolerate or detoxify plant defensive compounds. Recent studies also highlight the role of gut microbiota in mediating detoxification in herbivores; however, convincing evidence supporting the significant contribution of gut symbionts is rare in Lepidoptera. Here, we show that the growth of various lepidopteran species was inhibited by a mulberry-derived secondary metabolite, 1-deoxynojirimycin (DNJ); as expected, the specialist silkworm Bombyx mori grew well, but interestingly, gut microbiota of early-instar silkworms was affected by the DNJ level, and several bacterial species responded positively to enriched DNJ. Among these, a bacterial strain isolated from the silkworm gut (Pseudomonas fulva ZJU1) can degrade and utilize DNJ as the sole energy source, and after inoculation into nonspecialists (e.g., beet armyworm Spodoptera exigua), P. fulva ZJU1 increased host resistance to DNJ and significantly promoted growth. We used genomic and transcriptomic analyses to identify genes potentially involved in DNJ degradation, and CRISPR-Cas9-mediated mutagenesis verified the function of ilvB, a key binding protein, in metabolizing DNJ. Furthermore, the ilvB deletion mutant, exhibiting normal bacterial growth, could no longer enhance nonspecialist performance, supporting a role in DNJ degradation in vivo. Therefore, our study demonstrated causality between the gut microbiome and detoxification of plant chemical defense in Lepidoptera, facilitating a mechanistic understanding of host-microbe relationships across this complex, abundant insect group.

[Source](https://doi.org/10.1073/pnas.2412165121)

10.1073/pnas.2412165121

---

## Sea cucumbers and their symbiotic microbiome have evolved to feed on seabed sediments.
 14.10.2024, _W Pan, X Wang, C Ren, X Jiang, S Gong, Z Xie, NK Wong, X Li, J Huang, D Fan, P Luo, Y Yang, X Ren, S Yu, Z Qin, X Wu, D Huo, B Ma, Y Liu, X Zhang, Z E, J Liang, H Sun, L Yuan, X Liu, C Cheng, H Long, J Li, Y Wang, C Hu, T Chen_


_Animals, Symbiosis, Geologic Sediments, Sea Cucumbers, Glycoside Hydrolases, Gastrointestinal Microbiome, Holothuria, Phylogeny, Biological Evolution, Ecosystem, Feeding Behavior, Microbiota, Bacteria_
Sea cucumbers are predominant deposit feeders in benthic ecosystems, providing protective benefits to coral reefs by reducing disease prevalence. However, how they receive sufficient nutrition from seabed sediments remains poorly understood. Here, we investigate Holothuria leucospilota, an ecologically significant tropical sea cucumber, to elucidate digestive mechanisms underlying marine deposit-feeding. Genomic analysis reveals intriguing evolutionary adaptation characterized by an expansion of digestive carbohydrase genes and a contraction of digestive protease genes, suggesting specialization in digesting microalgae. Developmentally, two pivotal dietary shifts, namely, from endogenous nutrition to planktonic feeding, and from planktonic feeding to deposit feeding, induce changes in digestive tract enzyme profiles, with adults mainly expressing carbohydrases and lipases. A nuanced symbiotic relationship exists between gut microbiota and the host, namely, specific resident bacteria supply crucial enzymes for food digestion, while other bacteria are digested and provide assimilable nutrients. Our study further identifies Holothuroidea lineage-specific lysozymes that are restrictedly expressed in the intestines to support bacterial digestion. Overall, this work advances our knowledge of the evolutionary innovations in the sea cucumber digestive system which enable them to efficiently utilize nutrients from seabed sediments and promote food recycling within marine ecosystems.

[Source](https://doi.org/10.1038/s41467-024-53205-5)

10.1038/s41467-024-53205-5

---

## Comprehensive profile of the companion animal gut microbiome integrating reference-based and reference-free methods

[Source](https://doi.org/10.1093/ismejo/wrae201)

---

## Quantitative Synthesis of Microbe-Driven Acclimation and Adaptation in Wild Vertebrates.
 12.10.2024, _G Martin Bideguren, O Razgour, A Alberdi_


Microorganisms associated with animals harbour a unique set of functional traits pivotal for the normal functioning of their hosts. This realisation has led researchers to hypothesise that animal-associated microbial communities may boost the capacity of their hosts to acclimatise and adapt to environmental changes, two eco-evolutionary processes with significant applied relevance. Aiming to assess the importance of microorganisms for wild vertebrate conservation, we conducted a quantitative systematic review to evaluate the scientific evidence for the contribution of gut microorganisms to the acclimation and adaptation capacity of wild vertebrate hosts. After screening 1974 publications, we scrutinised the 109 studies that met the inclusion criteria based on 10 metrics encompassing study design, methodology and reproducibility. We found that the studies published so far were not able to resolve the contribution of gut microorganisms due to insufficient study design and research methods for addressing the hypothesis. Our findings underscore the limited application to date of microbiome knowledge in vertebrate conservation and management, highlighting the need for a paradigm shift in research approaches. Considering these results, we advocate for a shift from observational studies to experimental manipulations, where fitness or related indicators are measured, coupled with an update in molecular techniques used to analyse microbial functions. In addition, closer collaboration with conservation managers and practitioners from the inception of the project is needed to encourage meaningful application of microbiome knowledge in adaptive wildlife conservation management.

[Source](https://doi.org/10.1111/eva.70025)

10.1111/eva.70025

---

## The role of foraging pollinators in assembling the flower microbiota and transmitting the fire blight pathogen <em>Erwinia amylovora</em>

[Source](https://doi.org/10.1111/1462-2920.16702)

---

## Distinct microbial communities supported by iron oxidation.
 10.10.2024, _K Burkartová, A Hlaváček, S Skoblia, L Falteisek_


_Iron, Oxidation-Reduction, Bacteria, Hydrogen-Ion Concentration, Biodiversity, Microbiota, Ecosystem, Mining, Phylogeny, Archaea_
Microbial biostalactites and streamers commonly grow at iron seepages in abandoned mines worldwide. This study addresses the diversity and composition of these simple prokaryotic communities, which thrive in pH ranges from 2.4 to 6.6 across six different mines. Our analysis of 85 communities reveals that a pH of approximately 3.2 is a critical threshold where alpha and beta diversity change discretely. Below this pH, the average number of ASVs per sample is 2.91 times lower than above this boundary. Autotrophs, heterotrophs, and symbionts of eukaryotes originate from nearly non-overlapping species pools in the two habitat types that differ only in pH. Communities below pH 3.2 further divide into two distinct groups, differing in diversity, taxonomic, and functional composition. Both types of communities coexist within the same stalactites, likely corresponding to zones where the capillary structure of the stalactite is either perfused or clogged. These findings indicate that microbial community structure can be significantly influenced by the intricate spatial organization of the ecosystem, rather than solely by measurable environmental parameters.

[Source](https://doi.org/10.1111/1462-2920.16706)

10.1111/1462-2920.16706

---

## Normalization of short-chain fatty acid concentration by bacterial count of stool samples improves discrimination between eubiotic and dysbiotic gut microbiota caused by <em>Clostridioides difficile </em>infection

[Source](https://doi.org/10.1080/19490976.2024.2415488)

---

## SmT/SHM-seq: simultaneously capturing spatial transcriptome and microbiome information in plants.
 12.10.2024, _P Mu, W Li, LP Tran, X Li_



[Source](https://doi.org/10.1016/j.tplants.2024.09.010)

10.1016/j.tplants.2024.09.010

---

## High accuracy meets high throughput for near full-length 16S ribosomal RNA amplicon sequencing on the Nanopore platform.
 11.10.2024, _X Lin, K Waring, H Ghezzi, C Tropini, J Tyson, RM Ziels_


Small subunit (SSU) ribosomal RNA (rRNA) gene amplicon sequencing is a foundational method in microbial ecology. Currently, short-read platforms are commonly employed for high-throughput applications of SSU rRNA amplicon sequencing, but at the cost of poor taxonomic classification due to limited fragment lengths. The Oxford Nanopore Technologies (ONT) platform can sequence full-length SSU rRNA genes, but its lower raw-read accuracy has so-far limited accurate taxonomic classification and de novo feature generation. Here, we present a sequencing workflow, termed ssUMI, that combines unique molecular identifier (UMI)-based error correction with newer (R10.4+) ONT chemistry and sample barcoding to enable high throughput near full-length SSU rRNA (e.g. 16S rRNA) amplicon sequencing. The ssUMI workflow generated near full-length 16S rRNA consensus sequences with 99.99% mean accuracy using a minimum subread coverage of 3×, surpassing the accuracy of Illumina short reads. The consensus sequences generated with ssUMI were used to produce error-free de novo sequence features with no false positives with two microbial community standards. In contrast, Nanopore raw reads produced erroneous de novo sequence features, indicating that UMI-based error correction is currently necessary for high-accuracy microbial profiling with R10.4+ ONT sequencing chemistries. We showcase the cost-competitive scalability of the ssUMI workflow by sequencing 87 time-series wastewater samples and 27 human gut samples, obtaining quantitative ecological insights that were missed by short-read amplicon sequencing. ssUMI, therefore, enables accurate and low-cost full-length 16S rRNA amplicon sequencing on Nanopore, improving accessibility to high-resolution microbiome science.

[Source](https://doi.org/10.1093/pnasnexus/pgae411)

10.1093/pnasnexus/pgae411

---

## A deployable film method to enable replicable sampling of low-abundance environmental microbiomes.
 15.10.2024, _P Mankiewicz Ledins, EZ Lin, C Bhattacharya, KJG Pollitt, AH Dyson, EM Hénaff_


_Microbiota, Humans, Specimen Handling, Dimethylpolysiloxanes, DNA, Bacterial, Environmental Monitoring, Environmental Microbiology_
Urbanizing global populations spend over 90% of their time indoors where microbiome abundance and diversity are low. Chronic exposure to microbiomes with low abundance and diversity have demonstrated negative long-term impacts on human health. Sequencing-based analyses of environmental nucleic acids are critical to understanding the impact of the indoor microbiome on human health, however low DNA yields indoors, alongside sample collection and processing inconsistencies, currently challenge study replicability. This study presents a comparative assessment of a novel, passive, easily replicable sampling strategy using polydimethylsiloxane (PDMS) sheets alongside a representative swab-based collection protocol. Deployable, customizable PDMS films designed for whole-sample insertion into standardized extraction kits demonstrated 43% higher DNA yields per sample, and 76% higher yields per cm2 of sampler over swab-based protocols. These results indicate that this accessible, scalable method enables sufficient DNA collection to comprehensively evaluate indoor microbiome exposures and potential human health impacts using smaller, more space efficient samplers, representing an attractive alternative to swab-based collection. In addition, this process reduces the manual steps required for microbiome sampling which could address inter-study variability, transform the current microbiome sampling paradigm, and ultimately benefit the replicability and accessibility of microbiome exposure studies.

[Source](https://doi.org/10.1038/s41598-024-72341-y)

10.1038/s41598-024-72341-y

---

