## Paternal microbiome perturbations impact offspring fitness.
 04.05.2024, _A Argaw-Denboba, TSB Schmidt, M Di Giacomo, B Ranjan, S Devendran, E Mastrorilli, CT Lloyd, D Pugliese, V Paribeni, J Dabin, A Pisaniello, S Espinola, A Crevenna, S Ghosh, N Humphreys, O Boruc, P Sarkies, M Zimmermann, P Bork, JA Hackett_


The gut microbiota operates at the interface of host-environment interactions to influence human homoeostasis and metabolic networks1-4. Environmental factors that unbalance gut microbial ecosystems can therefore shape physiological and disease-associated responses across somatic tissues5-9. However, the systemic impact of the gut microbiome on the germline-and consequently on the F1 offspring it gives rise to-is unexplored10. Here we show that the gut microbiota act as a key interface between paternal preconception environment and intergenerational health in mice. Perturbations to the gut microbiota of prospective fathers increase the probability of their offspring presenting with low birth weight, severe growth restriction and premature mortality. Transmission of disease risk occurs via the germline and is provoked by pervasive gut microbiome perturbations, including non-absorbable antibiotics or osmotic laxatives, but is rescued by restoring the paternal microbiota before conception. This effect is linked with a dynamic response to induced dysbiosis in the male reproductive system, including impaired leptin signalling, altered testicular metabolite profiles and remapped small RNA payloads in sperm. As a result, dysbiotic fathers trigger an elevated risk of in utero placental insufficiency, revealing a placental origin of mammalian intergenerational effects. Our study defines a regulatory 'gut-germline axis' in males, which is sensitive to environmental exposures and programmes offspring fitness through impacting placenta function.

[Source](https://doi.org/10.1038/s41586-024-07336-w)

10.1038/s41586-024-07336-w

---

## Oral bacteria relative abundance in faeces increases due to gut microbiota depletion and is linked with patient outcomes.
 04.05.2024, _C Liao, T Rolling, A Djukovic, T Fei, V Mishra, H Liu, C Lindberg, L Dai, B Zhai, JU Peled, MRM van den Brink, TM Hohl, JB Xavier_


The detection of oral bacteria in faecal samples has been associated with inflammation and intestinal diseases. The increased relative abundance of oral bacteria in faeces has two competing explanations: either oral bacteria invade the gut ecosystem and expand (the 'expansion' hypothesis), or oral bacteria transit through the gut and their relative increase marks the depletion of other gut bacteria (the 'marker' hypothesis). Here we collected oral and faecal samples from mouse models of gut dysbiosis (antibiotic treatment and DSS-induced colitis) and used 16S ribosomal RNA sequencing to determine the abundance dynamics of oral bacteria. We found that the relative, but not absolute, abundance of oral bacteria increases, reflecting the 'marker' hypothesis. Faecal microbiome datasets from diverse patient cohorts, including healthy individuals and patients with allogeneic haematopoietic cell transplantation or inflammatory bowel disease, consistently support the 'marker' hypothesis and explain associations between oral bacterial abundance and patient outcomes consistent with depleted gut microbiota. By distinguishing between the two hypotheses, our study guides the interpretation of microbiome compositional data and could potentially identify cases where therapies are needed to rebuild the resident microbiome rather than protect against invading oral bacteria.

[Source](https://doi.org/10.1038/s41564-024-01680-3)

10.1038/s41564-024-01680-3

---

## Human gut microbiota and their production of endocannabinoid-like mediators are directly affected by a dietary oil.
 05.05.2024, _C Roussel, M Sola, J Lessard-Lord, N Nallabelli, P Généreux, C Cavestri, O Azeggouar Wallen, R Villano, F Raymond, N Flamand, C Silvestri, V Di Marzo_


_Humans, Gastrointestinal Microbiome, Bacteria, Endocannabinoids, Colon, Ileum, Fatty Acids, Omega-3, Plant Oils, Dietary Supplements, Adult, Male_
Dietary omega-3 polyunsaturated fatty acids (n-3 PUFAs) and the gut microbiome affect each other. We investigated the impact of supplementation with Buglossoides arvensis oil (BO), rich in stearidonic acid (SDA), on the human gut microbiome. Employing the Mucosal Simulator of the Human Intestinal Microbial Ecosystem (M-SHIME), we simulated the ileal and ascending colon microbiomes of four donors. Our results reveal two distinct microbiota clusters influenced by BO, exhibiting shared and contrasting shifts. Notably, Bacteroides and Clostridia abundance underwent similar changes in both clusters, accompanied by increased propionate production in the colon. However, in the ileum, cluster 2 displayed a higher metabolic activity in terms of BO-induced propionate levels. Accordingly, a triad of bacterial members involved in propionate production through the succinate pathway, namely Bacteroides, Parabacteroides, and Phascolarctobacterium, was identified particularly in this cluster, which also showed a surge of second-generation probiotics, such as Akkermansia, in the colon. Finally, we describe for the first time the capability of gut bacteria to produce N-acyl-ethanolamines, and particularly the SDA-derived N-stearidonoyl-ethanolamine, following BO supplementation, which also stimulated the production of another bioactive endocannabinoid-like molecule, commendamide, in both cases with variations across individuals. Spearman correlations enabled the identification of bacterial genera potentially involved in endocannabinoid-like molecule production, such as, in agreement with previous reports, Bacteroides in the case of commendamide. This study suggests that the potential health benefits on the human microbiome of certain dietary oils may be amenable to stratified nutrition strategies and extend beyond n-3 PUFAs to include microbiota-derived endocannabinoid-like mediators.

[Source](https://doi.org/10.1080/19490976.2024.2335879)

10.1080/19490976.2024.2335879

---

## Leveraging genomic signatures of oral microbiome-associated antibiotic resistance genes for diagnosing pancreatic cancer.
 02.05.2024, _X Shen, X Zhu, H Liu, R Yuan, Q Guo, P Zhao_


_Humans, Pancreatic Neoplasms, Microbiota, Carcinoma, Pancreatic Ductal, Female, Male, Mouth, Middle Aged, Drug Resistance, Microbial, Aged, Genomics_
Growing evidence has increasingly suggested a potential linkage between the oral microbiome and various diseases, including pancreatic ductal adenocarcinoma (PDAC). However, the utilization of gene-level information derived from the oral microbiome for diagnosing PDAC remains unexplored. In this study, we sought to investigate the novel potential of leveraging genomic signatures associated with antibiotic resistance genes (ARGs) within the oral microbiome for the diagnosis of PDAC. By conducting an analysis of oral microbiome samples obtained from PDAC patients, we successfully identified specific ARGs that displayed distinct sequence abundance profiles correlated with the presence of PDAC. In the healthy group, three ARGs were found to be enriched, whereas 21 ARGs were enriched in PDAC patients. Remarkably, these ARGs from oral microbiome exhibited promising diagnostic capabilities for PDAC (AUROC = 0.79), providing a non-invasive and early detection method. Our findings not only provide novel modal data for diagnosing PDAC but also shed light on the intricate interplay between the oral microbiome and PDAC.

[Source](https://doi.org/10.1371/journal.pone.0302361)

10.1371/journal.pone.0302361

---

## Deciphering the Pancreatic Cancer Microbiome in Mainland China: Impact of <em>Exiguobacterium/Bacillus </em>ratio on Tumor Progression and Prognostic Significance

[Source](https://doi.org/10.1016/j.phrs.2024.107197)

---

## Emergent antibiotic persistence in a spatially structured synthetic microbial mutualism

[Source](https://doi.org/10.1093/ismejo/wrae075)

---

## Microplastics and microbiota: Unraveling the hidden environmental challenge.
 02.05.2024, _J Demarquoy_


_Microplastics, Humans, Gastrointestinal Microbiome, Dysbiosis, Environmental Health, Environmental Exposure, Animals_
This editorial explores the intricate relationship between microplastics (MPs) and gut microbiota, emphasizing the complexity and environmental health implications. The gut microbiota, a crucial component of gastrointestinal health, is examined in the context of potential microbial degradation of MPs. Furthermore, dysbiosis induced by MPs emerges as a consensus, disrupting the balance of gut microbiota and decreasing diversity. The mechanisms triggering dysbiosis, including physical interactions and chemical composition, are under investigation. Ongoing research addresses the consequences of MPs on immune fun-ction, nutrient metabolism, and overall host health. The bidirectional relationship between MPs and gut microbiota has significant implications for environmental and human health. Despite uncertainties, MPs negatively impact gut microbiota and health. Further research is essential to unravel the complex interactions and assess the long-term consequences of MPs on both environmental and human well-being.

[Source](https://doi.org/10.3748/wjg.v30.i16.2191)

10.3748/wjg.v30.i16.2191

---

## Genome plasticity shapes the ecology and evolution of <em>Phocaeicola dorei </em>and <em>Phocaeicola vulgatus</em>

[Source](https://doi.org/10.1038/s41598-024-59148-7)

---

## A catalog of ethanol-producing microbes in humans.
 03.05.2024, _B Mbaye, RM Wasfy, MT Alou, P Borentain, R Gerolami, JC Dufour, M Million_


Aim: Endogenous ethanol production emerges as a mechanism of nonalcoholic steatohepatitis, obesity, diabetes and auto-brewery syndrome. Methods: To identify ethanol-producing microbes in humans, we used the NCBI taxonomy browser and the PubMed database with an automatic query and manual verification. Results: 85 ethanol-producing microbes in human were identified. Saccharomyces cerevisiae, Candida and Pichia were the most represented fungi. Enterobacteriaceae was the most represented bacterial family with mainly Escherichia coli and Klebsiella pneumoniae. Species of the Lachnospiraceae and Clostridiaceae family, of the Lactobacillales order and of the Bifidobacterium genus were also identified. Conclusion: This catalog will help the study of ethanol-producing microbes in human in the pathophysiology, diagnosis, prevention and management of human diseases associated with endogenous ethanol production.

[Source](https://www.futuremedicine.com/doi/abs/10.2217/fmb-2023-0250?journalCode=fmb)

10.2217/fmb-2023-0250

---

## Direct metagenomics investigation of non-surgical hard-to-heal wounds: a review.
 07.05.2024, _M Morsli, F Salipante, C Magnan, C Dunyach-Remy, A Sotto, JP Lavigne_


_Humans, Metagenomics, Bacteria, High-Throughput Nucleotide Sequencing, Wound Healing, Microbiota, Pressure Ulcer, Diabetic Foot, Wound Infection, Varicose Ulcer_
Non-surgical chronic wounds, including diabetes-related foot diseases (DRFD), pressure injuries (PIs) and venous leg ulcers (VLU), are common hard-to-heal wounds. Wound evolution partly depends on microbial colonisation or infection, which is often confused by clinicians, thereby hampering proper management. Current routine microbiology investigation of these wounds is based on in vitro culture, focusing only on a limited panel of the most frequently isolated bacteria, leaving a large part of the wound microbiome undocumented.

[Source](https://doi.org/10.1186/s12941-024-00698-z)

10.1186/s12941-024-00698-z

---

## A consortium of <em>Hordeum vulgare </em>and gut microbiota against non-alcoholic fatty liver disease via data-driven analysis

[Source](https://doi.org/10.1080/21691401.2024.2347380)

---

## Amphibian tolerance to arsenic: microbiome-mediated insights.
 07.05.2024, _IF Cordeiro, CGC Lemes, AB Sanchez, AK da Silva, CH de Paula, RC de Matos, DF Ribeiro, JP de Matos, CCM Garcia, M Beirão, CG Becker, MRS Pires, LM Moreira_


_Animals, Arsenic, Microbiota, Skin, Amphibians, Bacteria, Permeability_
Amphibians are often recognized as bioindicators of healthy ecosystems. The persistence of amphibian populations in heavily contaminated environments provides an excellent opportunity to investigate rapid vertebrate adaptations to harmful contaminants. Using a combination of culture-based challenge assays and a skin permeability assay, we tested whether the skin-associated microbiota may confer adaptive tolerance to tropical amphibians in regions heavily contaminated with arsenic, thus supporting the adaptive microbiome principle and immune interactions of the amphibian mucus. At lower arsenic concentrations (1 and 5 mM As3+), we found a significantly higher number of bacterial isolates tolerant to arsenic from amphibians sampled at an arsenic contaminated region (TES) than from amphibians sampled at an arsenic free region (JN). Strikingly, none of the bacterial isolates from our arsenic free region tolerated high concentrations of arsenic. In our skin permeability experiment, where we tested whether a subset of arsenic-tolerant bacterial isolates could reduce skin permeability to arsenic, we found that isolates known to tolerate high concentrations of arsenic significantly reduced amphibian skin permeability to this metalloid. This pattern did not hold true for bacterial isolates with low arsenic tolerance. Our results describe a pattern of environmental selection of arsenic-tolerant skin bacteria capable of protecting amphibians from intoxication, which helps explain the persistence of amphibian populations in water bodies heavily contaminated with arsenic.

[Source](https://doi.org/10.1038/s41598-024-60879-w)

10.1038/s41598-024-60879-w

---

## Combined effect of seasons and life history in an anuran strengthens the response and relationship between their physiology and gut microbiota

[Source](https://doi.org/10.1038/s41598-024-60105-7)

---

## Gut microbiome and resistome characterization of pigs treated with commonly used post-weaning diarrhea treatments.
 07.05.2024, _J Guitart-Matas, M Ballester, L Fraile, L Darwich, N Giler-Baquerizo, J Tarres, S López-Soria, Y Ramayo-Caldas, L Migura-Garcia_


The global burden of antimicrobial resistance demands additional measures to ensure the sustainable and conscious use of antimicrobials. For the swine industry, the post-weaning period is critical and for many years, antimicrobials have been the most effective strategy to control and treat post-weaning related infections. Among them, post-weaning diarrhea causes vast economic losses, as it severely compromises piglets' health and growth performance. In this study, 210 piglets were transferred from a farm with recurrent cases of post-weaning diarrhea to an experimental farm and divided into six different treatment groups to determine the effect of the different treatments on the growth performance and survival, the microbiome, and the resistome in a cross-sectional and longitudinal study. The different treatments included antimicrobials trimethoprim/sulfamethoxazole, colistin, and gentamicin, an oral commercial vaccine, a control with water acidification, and an untreated control. An extra group remained at the farm of origin following the implemented amoxicillin routine treatment. A total of 280 fecal samples from pigs at four different sampling times were selected for metagenomics: before weaning-treatment at the farm of origin, and three days, two weeks, and four weeks post-treatment.

[Source](https://doi.org/10.1186/s42523-024-00307-6)

10.1186/s42523-024-00307-6

---

## Effects of microbiome transplants on fly microbiome.
 03.05.2024, _J Ferreira_


_Animals, Microbiota, Drosophila melanogaster, Fecal Microbiota Transplantation, Gastrointestinal Microbiome_

[Source](https://doi.org/10.1038/s41684-024-01369-4)

10.1038/s41684-024-01369-4

---

## Multi-omics reveal the gut microbiota-mediated severe foraging environment adaption of small wild ruminants in the Three-River-Source National Park, China.
 02.05.2024, _H Liu, X Zhao, S Xu, L Zhao, X Han, X Xu, N Zhao, L Hu, C Luo, X Wang, Q Zhang, T Guo_


The Tibetan antelope (Pantholops hodgsonii), blue sheep (Pseudois nayaur), and Tibetan sheep (Ovis aries) are the dominant small ruminants in the Three-River-Source National Park (TRSNP). However, knowledge about the association between gut microbiota and host adaptability remains poorly understood. Herein, multi-omics sequencing approaches were employed to investigate the gut microbiota-mediated forage adaption in these ruminants. The results revealed that although wild ruminants (WR) of P. hodgsoni and P. nayaur were faced with severe foraging environments with significantly low vegetation coverage and nutrition, the apparent forage digestibility of dry matter, crude protein, and acid detergent fiber was significantly higher than that of O. aries. The 16s rRNA sequencing showed that the gut microbiota in WR underwent convergent evolution, and alpha diversity in these two groups was significantly higher than that in O. aries. Moreover, indicator species, including Bacteroidetes and Firmicutes, exhibited positive relationships with apparent forage digestibility, and their relative abundances were enriched in the gut of WR. Enterotype analysis further revealed that enterotype 1 belonged to WR, and the abundance of fatty acid synthesis metabolic pathway-related enzyme genes was significantly higher than enterotype 2, represented by O. aries. Besides, the metagenomic analysis identified 14 pathogenic bacterial species, among which 10 potentially pathogenic bacteria were significantly enriched in the gut microbiota of O. aries. Furthermore, the cellulolytic strains and genes encoding cellulase and hemicellulase were significantly enriched in WR. In conclusion, our results provide new evidence of gut microbiota to facilitate wildlife adaption in severe foraging environments of the TRSNP, China.

[Source](https://doi.org/10.1111/1749-4877.12830)

10.1111/1749-4877.12830

---

## Sea cucumber (<em>Holothuria glaberrima</em>) intestinal microbiome dataset from Puerto Rico, generated by shotgun sequencing

[Source](https://doi.org/10.1016/j.dib.2024.110421)

---

## Depletion of protective microbiota promotes the incidence of fruit disease.
 01.05.2024, _X Luo, K Sun, HR Li, XY Zhang, YT Pan, DL Luo, YB Wu, HJ Jiang, XH Wu, CY Ma, CC Dai, W Zhang_


Plant-associated microbiomes play important roles in plant health and productivity. However, despite fruits being directly linked to plant productivity, little is known about the microbiomes of fruits and their potential association with fruit health. Here, by integrating 16S rRNA gene, ITS high-throughput sequencing data and microbiological culturable approaches, we reported that roots and fruits (pods) of peanut, a typical plant that bears fruits underground, recruit different bacterial and fungal communities independently of cropping conditions, and that the incidence of pod disease under monocropping conditions is attributed to the depletion of Bacillus genus and enrichment of Aspergillus genus in geocarposphere. On this basis, we constructed a synthetic community (SynCom) consisting of three Bacillus strains from geocarposphere soil under rotation conditions with high culturable abundance. Comparative transcriptome, microbiome profiling and plant phytohormone signaling analysis reveal that the SynCom exhibited more effective Aspergillus growth inhibition and pod disease control than individual strain, which was underpinned by a combination of molecular mechanisms related to fungal cell proliferation interference, mycotoxins biosynthesis impairment and jasmonic acid-mediated plant immunity activation. Overall, our results reveal the filter effect of plant organs on the microbiome, and that depletion of key protective microbial community promotes the fruit disease incidence.

[Source](https://doi.org/10.1093/ismejo/wrae071)

10.1093/ismejo/wrae071

---

## Analysis of Salt Stress on Soil Microbial Community Composition and Its Correlation with Active Components in the Rhizosphere of <em>Acanthopanax senticosus</em>

[Source](https://doi.org/10.1007/s00284-024-03687-9)

---

## Mechanisms and implications of bacterial-fungal competition for soil resources.
 01.05.2024, _C Wang, Y Kuzyakov_


Elucidating complex interactions between bacteria and fungi that determine microbial community structure, composition, and functions in soil, as well as regulate carbon (C) and nutrient fluxes, is crucial to understand biogeochemical cycles. Among the various interactions, competition for resources is the main factor determining the adaptation and niche differentiation between these two big microbial groups in soil. This is because C and energy limitations for microbial growth are a rule rather than an exception. Here, we review the C and energy demands of bacteria and fungi - the two major kingdoms in soil - the mechanisms of their competition for these and other resources, leading to niche differentiation, and the global change impacts on this competition. The normalized microbial utilization preference showed that bacteria are 1.4-5 times more efficient in the uptake of simple organic compounds as substrates, whereas fungi are 1.1-4.1 times more effective in utilizing complex compounds. Accordingly, bacteria strongly outcompete fungi for simple substrates, while fungi take advantage of complex compounds. Bacteria also compete with fungi for the products released during the degradation of complex substrates. Based on these specifics, we differentiated spatial, temporal, and chemical niches for these two groups in soil. The competition will increase under the main five global changes including elevated CO2, N deposition, soil acidification, global warming, and drought. Elevated CO2, N deposition, and warming increase bacterial dominance, whereas soil acidification and drought increase fungal competitiveness.

[Source](https://doi.org/10.1093/ismejo/wrae073)

10.1093/ismejo/wrae073

---

## Potential coupling of microbial methane, nitrogen, and sulphur cycling in the Okinawa Trough cold seep sediments.
 01.05.2024, _Y Chen, X Dong, Z Sun, C Xu, X Zhang, S Qin, W Geng, H Cao, B Zhai, X Li, N Wu_


The Okinawa Trough (OT) is a back-arc basin with a wide distribution of active cold seep systems. However, our understanding of the metabolic function of microbial communities in the cold seep sediments of the OT remains limited. In this study, we investigated the vertical profiles of functional genes involved in methane, nitrogen, and sulphur cycling in the cold seep sediments of the OT. Furthermore, we explored the possible coupling mechanisms between these biogeochemical cycles. The study revealed that the majority of genes associated with the nitrogen and sulphur cycles were most abundant in the surface sediment layers. However, only the key genes responsible for sulphur disproportionation (sor), nitrogen fixation (nifDKH), and methane metabolism (mcrABG) were more prevalent within sulfate-methane transition zone (SMTZ). Significant positive correlations (P &lt; 0.05) were observed between functional genes involved in sulphur oxidation, thiosulphate disproportionation with denitrification, and dissimilatory nitrate reduction to ammonium (DNRA), as well as between AOM/methanogenesis and nitrogen fixation, and between sulphur disproportionation and AOM. A genome of Filomicrobium (class Alphaproteobacteria) has demonstrated potential in chemoautotrophic activities, particularly in coupling DNRA and denitrification with sulphur oxidation. Additionally, the characterized sulfate reducers such as Syntrophobacterales have been found to be capable of utilizing nitrate as an electron acceptor. The predominant methanogenic/methanotrophic groups in the OT sediments were identified as H2-dependent methylotrophic methanogens (Methanomassiliicoccales and Methanofastidiosales) and ANME-1a. This study offered a thorough understanding of microbial ecosystems in the OT cold seep sediments, emphasizing their contribution to nutrient cycling.IMPORTANCEThe Okinawa Trough (OT) is a back-arc basin formed by extension within the continental lithosphere behind the Ryukyu Trench arc system. Cold seeps are widespread in the OT. While some studies have explored microbial communities in OT cold seep sediments, their metabolic potential remains largely unknown. In this study, we used metagenomic analysis to enhance comprehension of the microbial community's role in nutrient cycling and proposed hypotheses on the coupling process and mechanisms involved in biogeochemical cycles. It was revealed that multiple metabolic pathways can be performed by a single organism or microbes that interact with each other to carry out various biogeochemical cycling. This data set provided a genomic road map on microbial nutrient cycling in OT sediment microbial communities.

[Source](https://doi.org/10.1128/spectrum.03490-23)

10.1128/spectrum.03490-23

---

## mbDecoda: a debiased approach to compositional data analysis for microbiome surveys.
 05.05.2024, _Y Zong, H Zhao, T Wang_


_Microbiota, Humans, Algorithms, Data Analysis_
Potentially pathogenic or probiotic microbes can be identified by comparing their abundance levels between healthy and diseased populations, or more broadly, by linking microbiome composition with clinical phenotypes or environmental factors. However, in microbiome studies, feature tables provide relative rather than absolute abundance of each feature in each sample, as the microbial loads of the samples and the ratios of sequencing depth to microbial load are both unknown and subject to considerable variation. Moreover, microbiome abundance data are count-valued, often over-dispersed and contain a substantial proportion of zeros. To carry out differential abundance analysis while addressing these challenges, we introduce mbDecoda, a model-based approach for debiased analysis of sparse compositions of microbiomes. mbDecoda employs a zero-inflated negative binomial model, linking mean abundance to the variable of interest through a log link function, and it accommodates the adjustment for confounding factors. To efficiently obtain maximum likelihood estimates of model parameters, an Expectation Maximization algorithm is developed. A minimum coverage interval approach is then proposed to rectify compositional bias, enabling accurate and reliable absolute abundance analysis. Through extensive simulation studies and analysis of real-world microbiome datasets, we demonstrate that mbDecoda compares favorably with state-of-the-art methods in terms of effectiveness, robustness and reproducibility.

[Source](https://doi.org/10.1093/bib/bbae205)

10.1093/bib/bbae205

---

## EnsembleSeq: a workflow towards real-time, rapid, and simultaneous multi-kingdom-amplicon sequencing for holistic and resource-effective microbiome research at scale.
 30.04.2024, _S Nagpal, SS Mande, H Hooda, U Dutta, B Taneja_


Bacterial communities are often concomitantly present with numerous microorganisms in the human body and other natural environments. Amplicon-based microbiome studies have generally paid skewed attention, that too at a rather shallow genus level resolution, to the highly abundant bacteriome, with interest now forking toward the other microorganisms, particularly fungi. Given the generally sparse abundance of other microbes in the total microbiome, simultaneous sequencing of amplicons targeting multiple microbial kingdoms could be possible even with full multiplexing. Guiding studies are currently needed for performing and monitoring multi-kingdom-amplicon sequencing and data capture at scale. Aiming to address these gaps, amplification of full-length bacterial 16S rRNA gene and entire fungal internal-transcribed spacer (ITS) region was performed for human saliva samples (n = 96, including negative and positive controls). Combined amplicon DNA libraries were prepared for nanopore sequencing using a major fraction of 16S molecules and a minor fraction of ITS amplicons. Sequencing was performed in a single run of an R10.4.1 flow cell employing the latest V14 chemistry. An approach for real-time monitoring of the species saturation using dynamic rarefaction was designed as a guiding determinant of optimal run time. Real-time saturation monitoring for both bacterial and fungal species enabled the completion of sequencing within 30 hours, utilizing less than 60% of the total nanopores. Approximately 5 million high quality (HQ) taxonomically assigned reads were generated (~4.2 million bacterial and 0.7 million fungal), providing a wider (beyond bacteriome) snapshot of human oral microbiota at species-level resolution. Among the more than 400 bacterial and 240 fungal species identified in the studied samples, the species of Streptococcus (e.g., Streptococcus mitis and Streptococcus oralis) and Candida (e.g., Candida albicans and Candida tropicalis) were observed to be the dominating microbes in the oral cavity, respectively. This conformed well with the previous reports of the human oral microbiota. EnsembleSeq provides a proof-of-concept toward the identification of both fungal and bacterial species simultaneously in a single fully multiplexed nanopore sequencing run in a time- and resource-effective manner. Details of this workflow, along with the associated codebase, are provided to enable large-scale application for a holistic species-level microbiome study.

[Source](https://doi.org/10.1128/spectrum.04150-23)

10.1128/spectrum.04150-23

---

## mi-Mic: a novel multi-layer statistical test for microbiota-disease associations.
 04.05.2024, _O Shtossel, S Finkelstein, Y Louzoun_


_Humans, Microbiota, Disease, Statistics as Topic_
mi-Mic, a novel approach for microbiome differential abundance analysis, tackles the key challenges of such statistical tests: a large number of tests, sparsity, varying abundance scales, and taxonomic relationships. mi-Mic first converts microbial counts to a cladogram of means. It then applies a priori tests on the upper levels of the cladogram to detect overall relationships. Finally, it performs a Mann-Whitney test on paths that are consistently significant along the cladogram or on the leaves. mi-Mic has much higher true to false positives ratios than existing tests, as measured by a new real-to-shuffle positive score.

[Source](https://doi.org/10.1186/s13059-024-03256-0)

10.1186/s13059-024-03256-0

---

## Unveiling microbial diversity: harnessing long-read sequencing technology.
 30.04.2024, _DP Agustinho, Y Fu, VK Menon, GA Metcalf, TJ Treangen, FJ Sedlazeck_


Long-read sequencing has recently transformed metagenomics, enhancing strain-level pathogen characterization, enabling accurate and complete metagenome-assembled genomes, and improving microbiome taxonomic classification and profiling. These advancements are not only due to improvements in sequencing accuracy, but also happening across rapidly changing analysis methods. In this Review, we explore long-read sequencing's profound impact on metagenomics, focusing on computational pipelines for genome assembly, taxonomic characterization and variant detection, to summarize recent advancements in the field and provide an overview of available analytical methods to fully leverage long reads. We provide insights into the advantages and disadvantages of long reads over short reads and their evolution from the early days of long-read sequencing to their recent impact on metagenomics and clinical diagnostics. We further point out remaining challenges for the field such as the integration of methylation signals in sub-strain analysis and the lack of benchmarks.

[Source](https://doi.org/10.1038/s41592-024-02262-1)

10.1038/s41592-024-02262-1

---

