## Linking microbial genes to plasma and stool metabolites uncovers host-microbial interactions underlying ulcerative colitis disease course.
 12.01.2024, _M Schirmer, M Stražar, J Avila-Pacheco, DF Rojas-Tapias, EM Brown, E Temple, A Deik, K Bullock, S Jeanfavre, K Pierce, S Jin, R Invernizzi, MM Pust, Z Costliow, DR Mack, AM Griffiths, T Walters, BM Boyle, S Kugathasan, H Vlamakis, J Hyams, L Denson, CB Clish, RJ Xavier_


Understanding the role of the microbiome in inflammatory diseases requires the identification of microbial effector molecules. We established an approach to link disease-associated microbes to microbial metabolites by integrating paired metagenomics, stool and plasma metabolomics, and culturomics. We identified host-microbial interactions correlated with disease activity, inflammation, and the clinical course of ulcerative colitis (UC) in the Predicting Response to Standardized Colitis Therapy (PROTECT) pediatric inception cohort. In severe disease, metabolite changes included increased dipeptides and tauro-conjugated bile acids (BAs) and decreased amino-acid-conjugated BAs in stool, whereas in plasma polyamines (N-acetylputrescine and N1-acetylspermidine) increased. Using patient samples and Veillonella parvula as a model, we uncovered nitrate- and lactate-dependent metabolic pathways, experimentally linking V. parvula expansion to immunomodulatory tryptophan metabolite production. Additionally, V. parvula metabolizes immunosuppressive thiopurine drugs through xdhA xanthine dehydrogenase, potentially impairing the therapeutic response. Our findings demonstrate that the microbiome contributes to disease-associated metabolite changes, underscoring the importance of these interactions in disease pathology and treatment.

[Source](https://doi.org/10.1016/j.chom.2023.12.013)

10.1016/j.chom.2023.12.013

---

## Incorporating metabolic activity, taxonomy and community structure to improve microbiome-based predictive models for host phenotype prediction.
 15.01.2024, _M Monshizadeh, Y Ye_


_Humans, Gastrointestinal Microbiome, Phylogeny, Diabetes Mellitus, Type 2, Microbiota, Phenotype_
We developed MicroKPNN, a prior-knowledge guided interpretable neural network for microbiome-based human host phenotype prediction. The prior knowledge used in MicroKPNN includes the metabolic activities of different bacterial species, phylogenetic relationships, and bacterial community structure, all in a shallow neural network. Application of MicroKPNN to seven gut microbiome datasets (involving five different human diseases including inflammatory bowel disease, type 2 diabetes, liver cirrhosis, colorectal cancer, and obesity) shows that incorporation of the prior knowledge helped improve the microbiome-based host phenotype prediction. MicroKPNN outperformed fully connected neural network-based approaches in all seven cases, with the most improvement of accuracy in the prediction of type 2 diabetes. MicroKPNN outperformed a recently developed deep-learning based approach DeepMicro, which selects the best combination of autoencoder and machine learning approach to make predictions, in all of the seven cases. Importantly, we showed that MicroKPNN provides a way for interpretation of the predictive models. Using importance scores estimated for the hidden nodes, MicroKPNN could provide explanations for prior research findings by highlighting the roles of specific microbiome components in phenotype predictions. In addition, it may suggest potential future research directions for studying the impacts of microbiome on host health and diseases. MicroKPNN is publicly available at https://github.com/mgtools/MicroKPNN.

[Source](https://doi.org/10.1080/19490976.2024.2302076)

10.1080/19490976.2024.2302076

---

## Microbiota and mucosal gene expression of fecal microbiota transplantation or placebo treated patients with chronic pouchitis.
 15.01.2024, _AK Hartikainen, I Khan, EK Karjalainen, L Renkonen-Sinisalo, P Arkkila, J Jalanka, AH Lepistö, R Satokari_


_Humans, Fecal Microbiota Transplantation, Pouchitis, Colitis, Ulcerative, Gastrointestinal Microbiome, Microbiota, Gene Expression, Feces_
Altered microbiota and impaired host immune function have been linked to the pathogenesis of pouchitis. We used 16S rRNA gene sequencing and RNA sequencing data from a previous randomized clinical trial (RCT) on fecal microbiota transplantation (FMT) therapy in 26 chronic pouchitis patients with one-year follow-up. We analyzed changes in both luminal and mucosal microbiota composition, as well as in host mucosal gene expression to gain insights into the host-microbiota interactions possibly underlying clinical outcomes of the patients. Antibiotic type and pattern of use were significant drivers of the luminal microbiota at baseline. Differential gene expression analysis indicated transition from ileal to colonic gene expression in the pouch, and upregulation in inflammation- and immune system-related pathways in the pouch. At 4 weeks, the non-relapsed FMT patients had a lower microbiota dissimilarity to the donor than the non-relapsed placebo patients (p = .02). While two FMT-treated patients showed a shift toward the donor's microbiota during the one-year follow-up, the overall FMT microbiota modulation effect was low. Patient's luminal and mucosal microbiota profiles were unstable in both FMT and placebo groups. Expression of the chemokine receptor CXCR4 was downregulated at 52 weeks compared to the baseline in the non-relapsed patients in both FMT and placebo groups. Microbiota modulation by FMT seems to be low in this patient group. The microbiota composition or alterations did not explain the relapse status of the patients. Some evidence for remission-related host gene expression pattern was found; specifically, CXCR4 expression may have a role in sustained remission.

[Source](https://doi.org/10.1080/19490976.2023.2295445)

10.1080/19490976.2023.2295445

---

## Methodology for biomarker discovery with reproducibility in microbiome data using machine learning.
 15.01.2024, _D Rojas-Velazquez, S Kidwai, AD Kraneveld, A Tonda, D Oberski, J Garssen, A Lopez-Rincon_


In recent years, human microbiome studies have received increasing attention as this field is considered a potential source for clinical applications. With the advancements in omics technologies and AI, research focused on the discovery for potential biomarkers in the human microbiome using machine learning tools has produced positive outcomes. Despite the promising results, several issues can still be found in these studies such as datasets with small number of samples, inconsistent results, lack of uniform processing and methodologies, and other additional factors lead to lack of reproducibility in biomedical research. In this work, we propose a methodology that combines the DADA2 pipeline for 16s rRNA sequences processing and the Recursive Ensemble Feature Selection (REFS) in multiple datasets to increase reproducibility and obtain robust and reliable results in biomedical research.

[Source](https://doi.org/10.1038/s41598-023-50601-7)

10.1186/s12859-024-05639-3

---

## A robust microbiome signature for autism spectrum disorder across different studies using machine learning

[Source](https://doi.org/10.1038/s41598-023-50601-7)

---

## The spatial dissimilarities and connections of the microbiota in the upper and lower respiratory tract of beef cattle.
 16.12.2023, _Z Zhang, C Zhang, Y Zhong, S Yang, F Deng, Y Li, J Chai_


_Cattle, Animals, Nasopharynx, Nose, Bacteria, Cattle Diseases, Lung, Microbiota_
Bovine respiratory disease (BRD) causes morbidity and mortality in cattle. The critical roles of the respiratory microbiota in BRD have been widely studied. The nasopharynx was the most popular sampling niche for BRD pathogen studies. The oral cavity and other niches within the respiratory tract, such as nostrils and lung, are less assessed. In this study, oropharyngeal swabs (OS), nasal swabs (NS), nasopharyngeal swabs (NP), and bronchoalveolar lavage (BAL) were collected from calves located in four countries and analyzed for investigation of the dissimilarities and connections of the respiratory microbiota. The results showed that the microbial diversity, structure, and composition in the upper and lower respiratory tract in beef cattle from China, the USA, Canada, and Italy were significantly different. The microbial taxa for each sampling niche were specific and associated with their local physiology and geography. The signature microbiota for OS, NS, NP, and BAL were identified using the LEfSe algorithm. Although the spatial dissimilarities among the respiratory niches existed, the microbial connections were observed in beef cattle regardless of geography. Notably, the nostril and nasopharynx had more similar microbiomes compared to lung communities. The major bacterial immigration patterns in the bovine respiratory tract were estimated and some of them were associated with geography. In addition, the contribution of oral microbiota to the nasal and lung ecosystems was confirmed. Lastly, microbial interactions were characterized to reveal the correlation between the commercial microbiota and BRD-associated pathogens. In conclusion, shared airway microbiota among niches and geography provides the possibility to investigate the common knowledge for bovine respiratory health and diseases. In spite of the dissimilarities of the respiratory microbiota in cattle, the spatial connections among these sampling niches not only allow us to deeply understand the airway ecosystem but also benefit the research and development of probiotics for BRD.

[Source](https://www.cysticfibrosisjournal.com/article/S1569-1993(23)01735-6/fulltext)

10.3389/fcimb.2023.1269726

---

## Nasal lavage microbiome, but not nasal swab microbiome, correlates with sinonasal inflammation in children with cystic fibrosis

[Source](https://www.cysticfibrosisjournal.com/article/S1569-1993(23)01735-6/fulltext)

---

## A Gpr35-tuned gut microbe-brain metabolic axis regulates depressive-like behavior.
 10.01.2024, _L Cheng, H Wu, X Cai, Y Zhang, S Yu, Y Hou, Z Yin, Q Yan, Q Wang, T Sun, G Wang, Y Yuan, X Zhang, H Hao, X Zheng_


Gene-environment interactions shape behavior and susceptibility to depression. However, little is known about the signaling pathways integrating genetic and environmental inputs to impact neurobehavioral outcomes. We report that gut G-protein-coupled receptor, Gpr35, engages a microbe-to-brain metabolic pathway to modulate neuronal plasticity and depressive behavior in mice. Psychological stress decreases intestinal epithelial Gpr35, genetic deletion of which induces depressive-like behavior in a microbiome-dependent manner. Gpr35-/- mice and individuals with depression have increased Parabacteroides distasonis, and its colonization to wild-type mice induces depression. Gpr35-/- and Parabacteroides distasonis-colonized mice show reduced indole-3-carboxaldehyde (IAld) and increased indole-3-lactate (ILA), which are produced from opposing branches along the bacterial catabolic pathway of tryptophan. IAld and ILA counteractively modulate neuroplasticity in the nucleus accumbens, a brain region linked to depression. IAld supplementation produces anti-depressant effects in mice with stress or gut epithelial Gpr35 deficiency. Together, these findings elucidate a gut microbe-brain signaling mechanism that underlies susceptibility to depression.

[Source](https://doi.org/10.1016/j.chom.2023.12.009)

10.1016/j.chom.2023.12.009

---

## The salivary microbiota of patients with acute lower respiratory tract infection-A multicenter cohort study.
 15.01.2024, _MB Rogers, A Harner, M Buhay, B Firek, B Methé, A Morris, OMP Palmer, SB Promes, RL Sherwin, L Southerland, AR Vieira, S Yende, MJ Morowitz, DT Huang_


_Adult, Humans, Prospective Studies, Microbiota, Respiratory Tract Infections, Gastrointestinal Microbiome, Feces, RNA, Ribosomal, 16S_
The human microbiome contributes to health and disease, but the oral microbiota is understudied relative to the gut microbiota. The salivary microbiota is easily accessible, underexplored, and may provide insight into response to infections. We sought to determine the composition, association with clinical features, and heterogeneity of the salivary microbiota in patients with acute lower respiratory tract infection (LRTI). We conducted a multicenter prospective cohort study of 147 adults with acute LRTI presenting to the emergency department of seven hospitals in three states (Pennsylvania, Michigan, and Ohio) between May 2017 and November 2018. Salivary samples were collected in the emergency department, at days 2-5 if hospitalized, and at day 30, as well as fecal samples if patients were willing. We compared salivary microbiota profiles from patients to those of healthy adult volunteers by sequencing and analyzing bacterial 16-rRNA. Compared to healthy volunteers, the salivary microbiota of patients with LRTI was highly distinct and strongly enriched with intestinal anaerobes such as Bacteroidaceae, Ruminococcaceae, and Lachnospiraceae (e.g., mean 10% relative abundance of Bacteroides vs &lt; 1% in healthy volunteers). Within the LRTI population, COPD exacerbation was associated with altered salivary microbiota composition compared to other LRTI conditions. The largest determinant of microbiota variation within the LRTI population was geography (city in which the hospital was located).

[Source](https://doi.org/10.1371/journal.pone.0290062)

10.1371/journal.pone.0290062

---

## Clinical Relevance of Vaginal and Endometrial Microbiome Investigation in Women with Repeated Implantation Failure and Recurrent Pregnancy Loss.
 13.01.2024, _X Gao, YV Louwers, JSE Laven, S Schoenmakers_


_Pregnancy, Female, Humans, Clinical Relevance, Vagina, Uterus, Lactobacillus, Microbiota, Abortion, Habitual_
Recent studies have investigated if and how the vaginal and endometrial microbiome might affect endometrial receptivity and reproductive health. Although there is no consensus on the existence of a core uterine microbiome yet, evidence shows that the dominance of Lactobacillus spp. in the female reproductive tract is generally associated with eubiosis and improved chances of successful implantation and an ongoing pregnancy. Conversely, vaginal and endometrial dysbiosis can cause local inflammation and an increase of pro-inflammatory cytokines, compromising the integrity and receptivity of the endometrial mucosa and potentially hampering successful embryonic implantation. This review provides a critical appraisal of the influence of the vaginal and endometrial microbiome as parts of the female reproductive tract on fertility outcomes, focusing on repeated implantation failure (RIF) and recurrent pregnancy loss (RPL). It seems that RIF as well as RPL are both associated with an increase in microbiome diversity and a loss of Lactobacillus dominance in the lower female reproductive system.

[Source](https://doi.org/10.1093/humrep/dead274)

10.3390/ijms25010622

---

## The endometrial microbiota and early pregnancy loss

[Source](https://doi.org/10.1093/humrep/dead274)

---

## Multi-omics of the gut microbial ecosystem in patients with microsatellite-instability-high gastrointestinal cancer resistant to immunotherapy.
 09.01.2024, _S Cheng, Z Han, D Dai, F Li, X Zhang, M Lu, Z Lu, X Wang, J Zhou, J Li, X Guo, P Song, C Qiu, W Shen, Q Zhang, N Zhu, X Wang, Y Tan, Y Kou, X Yin, L Shen, Z Peng_


Despite the encouraging efficacy of anti-PD-1/PD-L1 immunotherapy in microsatellite-instability-high/deficient mismatch repair (MSI-H/dMMR) advanced gastrointestinal cancer, many patients exhibit primary or acquired resistance. Using multi-omics approaches, we interrogate gut microbiome, blood metabolome, and cytokines/chemokines of patients with MSI-H/dMMR gastrointestinal cancer (N = 77) at baseline and during the treatment. We identify a number of microbes (e.g., Porphyromonadaceae) and metabolites (e.g., arginine) highly associated with primary resistance to immunotherapy. An independent validation cohort (N = 39) and mouse model are used to further confirm our findings. A predictive machine learning model for primary resistance is also built and achieves an accuracy of 0.79 on the external validation set. Furthermore, several microbes are pinpointed that gradually changed during the process of acquired resistance. In summary, our study demonstrates the essential role of gut microbiome in drug resistance, and this can be utilized as a preventative diagnosis tool and therapeutic target in the future.

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00572-4)

10.1016/j.xcrm.2023.101355

---

## The microbial metabolite urolithin A reduces <em>Clostridioides difficile </em>toxin expression and toxin-induced epithelial damage

[Source](https://doi.org/10.1128/msystems.01255-23)

---

## Impact of Periodontitis on the Leakage of Oral Bacteria to the Gut

[Source](https://doi.org/10.1177/00220345231221709)

---

## Changes in the fecal microbiota of breast cancer patients based on 16S rRNA gene sequencing: a systematic review and meta-analysis.
 13.01.2024, _B Luan, F Ge, X Lu, Z Li, H Zhang, J Wu, Q Yang, L Chen, W Zhang, W Chen_


Breast cancer (BC) is a devastating disease for women. Microbial influences may be involved in the development and progression of breast cancer. This study aimed to investigate the difference in intestinal flora abundance between breast cancer patients and healthy controls (HC) based on previous 16S ribosomal RNA (rRNA) gene sequencing results, which have been scattered and inconsistent in previous studies.

[Source](https://doi.org/10.1007/s12094-023-03373-5)

10.1007/s12094-023-03373-5

---

## <em>Bacteroides </em>and related species: The keystone taxa of the human gut microbiota

[Source](https://doi.org/10.1016/j.anaerobe.2024.102819)

---

## Enhancing microbiome research in sub-Saharan Africa.
 11.01.2024, _KI Ayeni, D Berry, CN Ezekiel, B Warth_


While there are lighthouse examples of microbiome research in sub-Saharan Africa (SSA), a significant proportion of local researchers face several challenges. Here, we highlight prevailing issues limiting microbiome research in SSA and suggest potential technological, societal, and research-based solutions. We emphasize the need for considerable investment in infrastructures, training, and appropriate funding to democratize modern technologies with a view to providing useful data to improve human health.

[Source](https://www.cell.com/trends/microbiology/fulltext/S0966-842X(23)00321-9)

10.1016/j.tim.2023.11.003

---

## Early microbial intervention reshapes phenotypes of newborn <em>Bos taurus </em>through metabolic regulations

[Source](https://doi.org/10.1093/gigascience/giad118)

---

## Gut microbiota variations in wild yellow baboons (<em>Papio cynocephalus</em>) are associated with sex and habitat disturbance

[Source](https://doi.org/10.1038/s41598-023-50126-z)

---

## Genetic mutation in <em>Escherichia coli </em>genome during adaptation to the murine intestine is optimized for the host diet

[Source](https://doi.org/10.1128/msystems.01123-23)

---

## <em>Candidatus </em>Methanosphaera massiliense sp. nov., a methanogenic archaeal species found in a human fecal sample and prevalent in pigs and red kangaroos

[Source](https://doi.org/10.1128/spectrum.05141-22)

---

## A novel plant growth-promoting rhizobacterium, <em>Rhizosphaericola mali </em>gen. nov., sp. nov., isolated from healthy apple tree soil

[Source](https://doi.org/10.1038/s41598-024-51492-y)

---

## Manipulation of the seagrass-associated microbiome reduces disease severity.
 09.01.2024, _OJ Graham, EM Adamczyk, S Schenk, P Dawkins, S Burke, E Chei, K Cisz, S Dayal, J Elstner, ALP Hausner, T Hughes, O Manglani, M McDonald, C Mikles, A Poslednik, A Vinton, L Wegener Parfrey, CD Harvell_


Host-associated microbes influence host health and function and can be a first line of defence against infections. While research increasingly shows that terrestrial plant microbiomes contribute to bacterial, fungal, and oomycete disease resistance, no comparable experimental work has investigated marine plant microbiomes or more diverse disease agents. We test the hypothesis that the eelgrass (Zostera marina) leaf microbiome increases resistance to seagrass wasting disease. From field eelgrass with paired diseased and asymptomatic tissue, 16S rRNA gene amplicon sequencing revealed that bacterial composition and richness varied markedly between diseased and asymptomatic tissue in one of the two years. This suggests that the influence of disease on eelgrass microbial communities may vary with environmental conditions. We next experimentally reduced the eelgrass microbiome with antibiotics and bleach, then inoculated plants with Labyrinthula zosterae, the causative agent of wasting disease. We detected significantly higher disease severity in eelgrass with a native microbiome than an experimentally reduced microbiome. Our results over multiple experiments do not support a protective role of the eelgrass microbiome against L. zosterae. Further studies of these marine host-microbe-pathogen relationships may continue to show new relationships between plant microbiomes and diseases.

[Source](https://doi.org/10.1111/1462-2920.16582)

10.1111/1462-2920.16582

---

## Assessment of DNA extraction methods for human gut mycobiome analysis.
 12.01.2024, _P Rintarhat, YJ Cho, H Koh, S Park, EJ Lee, H Lim, J Noh, DW Lee, WH Jung_


The gut mycobiome plays an important role in the health and disease of the human gut, but its exact function is still under investigation. While there is a wealth of information available on the bacterial community of the human gut microbiome, research on the fungal community is still relatively limited. In particular, technical methodologies for mycobiome analysis, especially the DNA extraction method for human faecal samples, varied in different studies. In the current study, two commercial kits commonly used in DNA extraction, the QIAamp® Fast DNA Stool Mini Kit and DNeasy PowerSoil Pro Kit, and one manual method, the International Human Microbiome Standards Protocol Q, were compared. Furthermore, the effectiveness of two different bead-beating machines, the Mini-Beadbeater-16 and FastPrep-24TM 5G, was compared in parallel. A mock fungal community with a known composition of fungal strains was also generated and included to compare different DNA extraction methods. Our results suggested that the method using the DNeasy PowerSoil Pro Kit and Mini-Beadbeater-16 provides the best results to extract DNA from human faecal samples. Based on our data, we propose a standard operating procedure for DNA extraction from human faecal samples for mycobiome analysis.

[Source](https://doi.org/10.1098/rsos.231129)

10.1098/rsos.231129

---

## NanoDeep: a deep learning framework for nanopore adaptive sampling on microbial sequencing.
 10.01.2024, _Y Lin, Y Zhang, H Sun, H Jiang, X Zhao, X Teng, J Lin, B Shu, H Sun, Y Liao, J Zhou_


_Humans, Deep Learning, Nanopores, Nanopore Sequencing, Gene Library, Genome, Bacterial_
Nanopore sequencers can enrich or deplete the targeted DNA molecules in a library by reversing the voltage across individual nanopores. However, it requires substantial computational resources to achieve rapid operations in parallel at read-time sequencing. We present a deep learning framework, NanoDeep, to overcome these limitations by incorporating convolutional neural network and squeeze and excitation. We first showed that the raw squiggle derived from native DNA sequences determines the origin of microbial and human genomes. Then, we demonstrated that NanoDeep successfully classified bacterial reads from the pooled library with human sequence and showed enrichment for bacterial sequence compared with routine nanopore sequencing setting. Further, we showed that NanoDeep improves the sequencing efficiency and preserves the fidelity of bacterial genomes in the mock sample. In addition, NanoDeep performs well in the enrichment of metagenome sequences of gut samples, showing its potential applications in the enrichment of unknown microbiota. Our toolkit is available at https://github.com/lysovosyl/NanoDeep.

[Source](https://doi.org/10.1093/bib/bbad499)

10.1093/bib/bbad499

---

## BactInt: A domain driven transfer learning approach for extracting inter-bacterial associations from biomedical text.
 10.01.2024, _K Das Baksi, V Pokhrel, AE Pudavar, SS Mande, BK Kuntal_


The healthy as well as dysbiotic state of an ecosystem like human body is known to be influenced not only by the presence of the bacterial groups in it, but also with respect to the associations within themselves. Evidence reported in biomedical text serves as a reliable source for identifying and ascertaining such inter bacterial associations. However, the complexity of the reported text as well as the ever-increasing volume of information necessitates development of methods for automated and accurate extraction of such knowledge.

[Source](https://doi.org/10.1016/j.compbiolchem.2023.108012)

10.1016/j.compbiolchem.2023.108012

---

## The African Human Microbiome Portal: a public web portal of curated metagenomic metadata

[Source](https://doi.org/10.1093/database/baad092)

---

## MK-BMC: a Multi-Kernel framework with Boosted distance metrics for Microbiome data for Classification.
 10.01.2024, _H Xu, T Wang, Y Miao, M Qian, Y Yang, S Wang_


Research on human microbiome have suggested associations with human health, opening opportunities to predict health outcomes using microbiome. Studies have also suggested that diverse forms of taxa such as rare taxa that are evolutionally-related and abundant taxa that are evolutionally-unrelated could be associated with or predictive of a health outcome. Although prediction models were developed for microbiome data, no prediction models currently exist that use multiple forms of microbiome-outcome associations.

[Source](https://doi.org/10.1093/bioinformatics/btad757)

10.1093/bioinformatics/btad757

---

