## <strong>Fecal microbiota transplantation plus anti-PD-1 immunotherapy in advanced melanoma: a phase I trial</strong>

[Source](https://doi.org/10.1038/s41591-023-02453-x)

---

## <strong>Top-down identification of keystone taxa in the microbiome</strong>

[Source](https://doi.org/10.1038/s41467-023-39459-5)

---

## <strong>Sources of gut microbiota variation in a large longitudinal Finnish infant cohort</strong>

[Source](https://www.thelancet.com/journals/ebiom/article/PIIS2352-3964(23)00260-8/fulltext)

---

## 11<sup>th</sup>&nbsp;Microbiome R&amp;D and Business Collaboration Forum and 5<sup>th</sup>&nbsp;Probiotics &amp; Prebiotics Congress: USA

[Source](https://www.global-engage.com/event/microbiome/)

---

## A microbiome exercise

[Source](https://doi.org/10.1126/science.adi6329)

---

## Mendelian randomisation analysis reveals the possible causal relationship between infections, microbiota and clinical disease

[Source](https://doi.org/10.1136/gutjnl-2023-329787)

---

## Microbiome variance of the small bowel in Crohn’s disease

[Source](https://doi.org/10.1136/gutjnl-2022-327313)

---

## Metagenomics analysis of the neonatal intestinal resistome

[Source](https://doi.org/10.3389/fped.2023.1169651)

---

## Development of the intestinal microbiome in cystic fibrosis in early life

[Source](https://doi.org/10.1128/msphere.00046-23)

---

## Overexpression of human alpha-Synuclein leads to dysregulated microbiome/metabolites with ageing in a rat model of Parkinson disease

[Source](https://doi.org/10.1186/s13024-023-00628-1)

---

## Male-Biased Gut Microbiome and Metabolites Aggravate Colorectal Cancer Development

[Source](https://doi.org/10.1002/advs.202206238)

---

## Systematic review: Autism spectrum disorder and the gut microbiota

[Source](https://doi.org/10.1111/acps.13587)

---

## Early-life respiratory infections and developmental immunity determine lifelong lung health.
 06.07.2023, _CM Lloyd, S Saglani_


Respiratory infections are common in infants and young children. However, the immune system develops and matures as the child grows, thus the effects of infection during this time of dynamic change may have long-term consequences. The infant immune system develops in conjunction with the seeding of the microbiome at the respiratory mucosal surface, at a time that the lungs themselves are maturing. We are now recognizing that any disturbance of this developmental trajectory can have implications for lifelong lung health. Here, we outline our current understanding of the molecular mechanisms underlying relationships between immune and structural cells in the lung with the local microorganisms. We highlight the importance of gaining greater clarity as to what constitutes a healthy respiratory ecosystem and how environmental exposures influencing this network will aid efforts to mitigate harmful effects and restore lung immune health.

[Source](https://doi.org/10.1038/s41590-023-01550-w)

10.1038/s41590-023-01550-w

---

## Ecoevolutionary processes structure milk microbiomes across the mammalian tree of life.
 12.07.2023, _MM Keady, RR Jimenez, M Bragg, JCP Wagner, SL Bornbusch, ML Power, CR Muletz-Wolz_


_Animals, Female, Pregnancy, Milk, Gastrointestinal Microbiome, RNA, Ribosomal, 16S, Placenta, Microbiota, Mammals, Bacteria, Eutheria_
Milk production is an ancient adaptation that unites all mammals. Milk contains a microbiome that can contribute to offspring health and microbial-immunological development. We generated a comprehensive milk microbiome dataset (16S rRNA gene) for the class Mammalia, representing 47 species from all placental superorders, to determine processes structuring milk microbiomes. We show that across Mammalia, milk exposes offspring to maternal bacterial and archaeal symbionts throughout lactation. Deterministic processes of environmental selection accounted for 20% of milk microbiome assembly processes; milk microbiomes were similar from mammals with the same host superorder (Afrotheria, Laurasiathera, Euarchontoglires, and Xenarthra: 6%), environment (marine captive, marine wild, terrestrial captive, and terrestrial wild: 6%), diet (carnivore, omnivore, herbivore, and insectivore: 5%), and milk nutrient content (sugar, fat, and protein: 3%). We found that diet directly and indirectly impacted milk microbiomes, with indirect effects being mediated by milk sugar content. Stochastic processes, such as ecological drift, accounted for 80% of milk microbiome assembly processes, which was high compared to mammalian gut and mammalian skin microbiomes (69% and 45%, respectively). Even amid high stochasticity and indirect effects, our results of direct dietary effects on milk microbiomes provide support for enteromammary trafficking, representing a mechanism by which bacteria are transferred from the mother's gut to mammary gland and then to offspring postnatally. The microbial species present in milk reflect both selective pressures and stochastic processes at the host level, exemplifying various ecological and evolutionary factors acting on milk microbiomes, which, in turn, set the stage for offspring health and development.

[Source](https://doi.org/10.1073/pnas.2218900120)

10.1073/pnas.2218900120

---

## Shared evolutionary path in social microbiomes.
 04.07.2023, _N Frazão, I Gordo_


Social networks can influence the ecology of gut bacteria, shaping the species composition of the gut microbiome in humans and other animals. Gut commensals evolve and can adapt at a rapid pace when colonizing healthy hosts. Here, we aimed at assessing the impact of host-to-host bacterial transmission on Escherichia coli evolution in the mammalian gut. Using an in vivo experimental evolution approach in mice, we found a transmission rate of 7% (±3% 2SE) of E. coli cells per day between hosts inhabiting the same household. Consistent with the predictions of a simple population genetics model of mutation-selection-migration, the level of shared events resulting from within host evolution is greatly enhanced in co-housed mice, showing that hosts undergoing the same diet and habit are not only expected to have similar microbiome species compositions but also similar microbiome evolutionary dynamics. Furthermore, we estimated the rate of mutation accumulation of E. coli to be 3.0 × 10-3 (±0.8 × 10-3 2SE) mutations/genome/generation, irrespective of the social context of the regime. Our results reveal the impact of bacterial migration across hosts in shaping the adaptive evolution of new strains colonizing gut microbiomes.

[Source](https://doi.org/10.1093/molbev/msad153)

10.1093/molbev/msad153

---

## Use of probiotics, prebiotics, and synbiotics in non-alcoholic fatty liver disease: A systematic review and meta-analysis.
 06.07.2023, _L Rong, D Ch'ng, P Jia, KKF Tsoi, SH Wong, JJY Sung_


Patients with non-alcoholic fatty liver disease (NAFLD) exhibit compositional changes in their gut microbiome, which represents a potential therapeutic target. Probiotics, prebiotics, and synbiotics are microbiome-targeted therapies that have been proposed as treatment for NAFLD. We aim to systematically review the effects of these therapies in liver-related outcomes of NAFLD patients.

[Source](https://doi.org/10.1111/jgh.16256)

10.1111/jgh.16256

---

## <em>Akkermansia muciniphila</em> and <em>Bifidobacterium bifidum</em> Prevent NAFLD by Regulating FXR Expression and Gut Microbiota

[Source](https://doi.org/10.14218/JCTH.2022.00415)

---

## <em>Limosilactobacillus reuteri</em> inhibits the acid tolerance response in oral bacteria

[Source](https://doi.org/10.1016/j.bioflm.2023.100136)

---

## Using the collaborative cross to identify the role of host genetics in defining the murine gut microbiome

[Source](https://doi.org/10.1186/s40168-023-01552-8)

---

## Selection for environmental variance shifted the gut microbiome composition driving animal resilience

[Source](https://doi.org/10.1186/s40168-023-01580-4)

---

## Metabolic interaction models recapitulate leaf microbiota ecology

[Source](https://doi.org/10.1126/science.adf5121)

---

## Microeukaryotic predators shape the wastewater microbiome

[Source](https://doi.org/10.1016/j.watres.2023.120293)

---

## Functional and sequence-based metagenomics to uncover carbohydrate-degrading enzymes from composting samples

[Source](https://doi.org/10.1007/s00253-023-12627-9)

---

## Analysis of the microbiome of the Bolivian high-altitude Lake Pastos Grandes

[Source](https://doi.org/10.1093/femsec/fiad073)

---

## Metagenomic Analysis of a Concrete Bridge Reveals a Microbial Community Dominated by Halophilic Bacteria and Archaea

[Source](https://doi.org/10.1128/spectrum.05112-22)

---

## A catalog of bacterial reference genomes from cultivated human oral bacteria

[Source](https://doi.org/10.1038/s41522-023-00414-3)

---

## Gene-based microbiome representation enhances host phenotype classification

[Source](https://doi.org/10.1128/msystems.00531-23)

---

## Integrative meta-omics in Galaxy and beyond

[Source](https://doi.org/10.1186/s40793-023-00514-9)

---

## Compositional analysis of microbiome data using the linear decomposition model (LDM).
 06.07.2023, _YJ Hu, GA Satten_


There are compelling reasons to test compositional hypotheses about microbiome data. We present here LDM-clr, an extension of our linear decomposition model (LDM) approach to allow fitting linear models to centered-log-ratio-transformed taxa count data. As LDM-clr is implemented within the existing LDM program, it enjoys all the features supported by LDM, including a compositional analysis of differential abundance at both the taxon and community levels, while allowing for a wide range of covariates and study designs for either association or mediation analysis.

[Source](https://doi.org/10.1101/2023.05.26.542540)

10.1101/2023.05.26.542540

---

## Processing and Storage Methods Affect Oral and Gut Microbiome Composition.
 04.07.2023, _DK Superdock, W Zhang, AC Poole_


Across microbiome studies, fecal and oral samples are stored and processed in different ways, which could affect the observed microbiome composition. Here, we compared treatment methods, which included both storage conditions and processing methods, applied to samples prior to DNA extraction to determine how each affects microbial community diversity as assessed by 16S rRNA gene sequencing. We collected dental swab, saliva, and fecal samples from 10 individuals, with three technical replicates per treatment method. We assessed four methods of processing fecal samples prior to DNA extraction. We also compared different fractions of frozen saliva and dental samples to fresh samples. We found that lyophilized fecal samples, fresh whole saliva samples, and the supernatant fraction of thawed dental samples retained the highest levels of alpha diversity in samples. The supernatant fraction of thawed saliva samples had the second highest alpha diversity compared to fresh. Then we investigated the differences in microbes between different treatments at the domain and phylum levels as well as identified the amplicon sequence variants (ASVs) that were significantly different between the methods producing the highest alpha diversity and the other treatment methods. Lyophilized fecal samples had a greater prevalence of Archaea as well as a greater ratio of Firmicutes to Bacteroidetes compared to the other treatment methods. Our results provide practical considerations, not only for selection of processing method, but also for comparing results across studies that use these methods. Our findings also indicate differences in treatment method could be a confounding factor influencing the presence, absence, or differential abundance of microbes reported in conflicting studies.

[Source](https://doi.org/10.1101/2023.06.13.544865)

10.1101/2023.06.13.544865

---

## A genome catalog of the early-life human skin microbiome

[Source](https://doi.org/10.1101/2023.05.22.541509)

---

## In vitro models as a tool to study the role of gut microbiota in obesity

[Source](https://doi.org/10.1080/10408398.2023.2232022)

---

