## Genomic structure predicts metabolite dynamics in microbial communities.
 27.01.2022, _K Gowda, D Ping, M Mani, S Kuehn_


The metabolic activities of microbial communities play a defining role in the evolution and persistence of life on Earth, driving redox reactions that give rise to global biogeochemical cycles. Community metabolism emerges from a hierarchy of processes, including gene expression, ecological interactions, and environmental factors. In wild communities, gene content is correlated with environmental context, but predicting metabolite dynamics from genomes remains elusive. Here, we show, for the process of denitrification, that metabolite dynamics of a community are predictable from the genes each member of the community possesses. A simple linear regression reveals a sparse and generalizable mapping from gene content to metabolite dynamics for genomically diverse bacteria. A consumer-resource model correctly predicts community metabolite dynamics from single-strain phenotypes. Our results demonstrate that the conserved impacts of metabolic genes can predict community metabolite dynamics, enabling the prediction of metabolite dynamics from metagenomes, designing denitrifying communities, and discovering how genome evolution impacts metabolism.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(21)01542-7)

10.1016/j.cell.2021.12.036

---

## Inhalable antibiotic resistomes emitted from hospitals: metagenomic insights into bacterial hosts, clinical relevance, and environmental risks.
 28.01.2022, _D Wu, L Jin, J Xie, H Liu, J Zhao, D Ye, XD Li_


Threats of antimicrobial resistance (AMR) to human health are on the rise worldwide. Airborne fine particulate matter (PM2.5), especially those emitted from hospitals, could serve as a substantial yet lesser-known environmental medium of inhalable antibiotic resistomes. A genome-centric understanding of the hosting bacterial taxa, mobility potential, and consequent risks of the resistomes is needed to reveal the health relevance of PM2.5-associated AMR from clinical settings.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01197-5)

10.1186/s40168-021-01197-5

---

## Long-read metagenomic sequencing reveals shifts in associations of antibiotic resistance genes with mobile genetic elements from sewage to activated sludge.
 30.01.2022, _D Dai, C Brown, H Bürgmann, DGJ Larsson, I Nambi, T Zhang, CF Flach, A Pruden, PJ Vikesland_


There is concern that the microbially rich activated sludge environment of wastewater treatment plants (WWTPs) may contribute to the dissemination of antibiotic resistance genes (ARGs). We applied long-read (nanopore) sequencing to profile ARGs and their neighboring genes to illuminate their fate in the activated sludge treatment by comparing their abundance, genetic locations, mobility potential, and bacterial hosts within activated sludge relative to those in influent sewage across five WWTPs from three continents.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01216-5)

10.1186/s40168-021-01216-5

---

## Understanding microbiome alterations in autism.
 21.01.2022, _K O'Leary_



[Source](https://www.nature.com/articles/d41591-022-00028-w)

10.1038/d41591-022-00028-w

---

## 3β-Hydroxysteroid dehydrogenase expressed by gut microbes degrades testosterone and is linked to depression in males.
 02.02.2022, _D Li, R Liu, M Wang, R Peng, S Fu, A Fu, J Le, Q Yao, T Yuan, H Chi, X Mu, T Sun, H Liu, P Yan, S Wang, S Cheng, Z Deng, Z Liu, G Wang, Y Li, T Liu_


Testosterone deficiency can lead to depressive symptoms in humans; however, the causes of this deficiency are incompletely understood. Here, we isolated Mycobacterium neoaurum from the fecal samples of testosterone-deficient patients with depression and showed that this strain could degrade testosterone in vitro. Furthermore, gavaging rats with M. neoaurum reduced their serum and brain testosterone levels and induced depression-like behaviors. We identified the gene encoding 3β-hydroxysteroid dehydrogenase (3β-HSD) as the enzyme causing testosterone degradation. Introducing 3β-HSD into Escherichia coli enhanced its ability to degrade testosterone. Gavaging rats with 3β-HSD-producing E. coli reduced their serum and brain testosterone levels and caused depression-like behaviors. Finally, compared with 16.67% of participants without depression, 42.99% (46/107) of the fecal samples of patients with depression harbored 3β-HSD, and 60.87% (28/46) of these fecal samples expressed 3β-HSD. These results suggest that 3β-HSD expressed by gut microbes may be associated with depressive symptoms due to testosterone degradation.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(22)00037-3)

10.1016/j.chom.2022.01.001

---

## Early-life viral infections are associated with disadvantageous immune and microbiota profiles and recurrent respiratory infections.
 21.01.2022, _WAA de Steenhuijsen Piters, RL Watson, EM de Koff, R Hasrat, K Arp, MLJN Chu, PCM de Groot, MA van Houten, EAM Sanders, D Bogaert_


The respiratory tract is populated by a specialized microbial ecosystem, which is seeded during and directly following birth. Perturbed development of the respiratory microbial community in early-life has been associated with higher susceptibility to respiratory tract infections (RTIs). Given a consistent gap in time between first signs of aberrant microbial maturation and the observation of the first RTIs, we hypothesized that early-life host-microbe cross-talk plays a role in this process. We therefore investigated viral presence, gene expression profiles and nasopharyngeal microbiota from birth until 12 months of age in 114 healthy infants. We show that the strongest dynamics in gene expression profiles occurred within the first days of life, mostly involving Toll-like receptor (TLR) and inflammasome signalling. These gene expression dynamics coincided with rapid microbial niche differentiation. Early asymptomatic viral infection co-occurred with stronger interferon activity, which was related to specific microbiota dynamics following, including early enrichment of Moraxella and Haemophilus spp. These microbial trajectories were in turn related to a higher number of subsequent (viral) RTIs over the first year of life. Using a multi-omic approach, we found evidence for species-specific host-microbe interactions related to consecutive susceptibility to RTIs. Although further work will be needed to confirm causality of our findings, together these data indicate that early-life viral encounters could impact subsequent host-microbe cross-talk, which is linked to later-life infections.

[Source](https://www.nature.com/articles/s41564-021-01043-2)

10.1038/s41564-021-01043-2

---

## Meconium Microbiome of Very Preterm Infants across Germany 

[Source](https://journals.asm.org/doi/10.1128/msphere.00808-21)

---

## Comparative metabolomic analysis reveals shared and unique chemical interactions in sponge holobionts

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01220-9)

---

## A multi-omics approach unravels metagenomic and metabolic alterations of a probiotic and synbiotic additive in rainbow trout (Oncorhynchus mykiss) 

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01221-8)

---

## MUON: multimodal omics analysis framework

[Source](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02577-8)

---

## A Toast to All the Rejects

[Source](https://www.theatlantic.com/family/archive/2022/01/celebrate-your-rejections-failures/621327/)

---

