## Commensal bacterial outer membrane protein A induces interleukin-22 production.
 01.06.2024, _Y Wang, VL Ngo, J Zou, AT Gewirtz_


Interleukin (IL)-22 promotes host-microbiota homeostasis. We sought to identify microbiota metabolite(s) that drive intestinal IL-22 production. We observed that exposing Peyer's patch cells (PPCs), ex vivo, to fecal supernatants (FSs) recapitulates fermentable fiber- and microbiota-dependent IL-22 production, and cellular sources thereof, thus supporting the use of this model. An interrogation of FSs generated from mice fed the fermentable fiber inulin (FS-Inu) revealed that its IL-22-inducing activity is mediated by heat-labile protein. Fractionation of FS-Inu by ion-exchange chromatography, and subsequent proteomic analysis of IL-22-inducing fractions, indicates that outer membrane protein A (OmpA) might be a microbial driver of IL-22 expression. Concomitantly, recombinant OmpA from Parabacteroides goldsteinii, which is enriched by an inulin diet, induces IL-22 production and expression of the IL-22-dependent genes REG3γ and -β, in PPCs and mice. Thus, OmpA is one bacterial inducer of IL-22 expression, potentially linking diet, mucosal immune homeostasis, and gut health.

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(24)00620-X)

10.1016/j.celrep.2024.114292

---

## Mucus production, host-microbiome interactions, hormone sensitivity, and innate immune responses modeled in human cervix chips.
 01.06.2024, _Z Izadifar, J Cotton, S Chen, V Horvath, A Stejskalova, A Gulati, NT LoGrande, B Budnik, S Shahriar, ER Doherty, Y Xie, T To, SE Gilpin, AM Sesay, G Goyal, CB Lebrilla, DE Ingber_


_Humans, Female, Cervix Uteri, Immunity, Innate, Microbiota, Host Microbial Interactions, Gardnerella vaginalis, Lactobacillus crispatus, Mucus, Lab-On-A-Chip Devices_
Modulation of the cervix by steroid hormones and commensal microbiome play a central role in the health of the female reproductive tract. Here we describe organ-on-a-chip (Organ Chip) models that recreate the human cervical epithelial-stromal interface with a functional epithelial barrier and production of mucus with biochemical and hormone-responsive properties similar to living cervix. When Cervix Chips are populated with optimal healthy versus dysbiotic microbial communities (dominated by Lactobacillus crispatus and Gardnerella vaginalis, respectively), significant differences in tissue innate immune responses, barrier function, cell viability, proteome, and mucus composition are observed that are similar to those seen in vivo. Thus, human Cervix Organ Chips represent physiologically relevant in vitro models to study cervix physiology and host-microbiome interactions, and hence may be used as a preclinical testbed for development of therapeutic interventions to enhance women's health.

[Source](https://doi.org/10.1038/s41467-024-48910-0)

10.1038/s41467-024-48910-0

---

## Gut microbiome and symptoms in females with irritable bowel syndrome: a cross-sectional analysis.
 31.05.2024, _KJ Kamp, AM Plantinga, KC Cain, RL Burr, CS Tsai, Q Wu, SY So, S Badu, T Savidge, RJ Shulman, MM Heitkemper_


_Humans, Irritable Bowel Syndrome, Female, Gastrointestinal Microbiome, Adult, Cross-Sectional Studies, Young Adult, RNA, Ribosomal, 16S, Adolescent, Middle Aged, Feces, Bacteria_
Irritable bowel syndrome (IBS), a disorder of gut-brain interaction, is associated with abdominal pain and stool frequency/character alterations that are linked to changes in microbiome composition. We tested whether taxa differentially abundant between females with IBS vs healthy control females (HC) are associated with daily gastrointestinal and psychological symptom severity. Participants (age 18-50 year) completed a 3-day food record and collected a stool sample during the follicular phase. They also completed a 28-day diary rating symptom intensity; analysis focused on the three days after the stool sample collection. 16S rRNA gene sequencing was used for bacterial identification. Taxon abundance was compared between IBS and HC using zero-inflated quantile analysis (ZINQ). We found that females with IBS (n = 67) had greater Bacteroides abundance (q = 0.003) and lower odds of Bifidobacterium presence (q = 0.036) compared to HC (n = 46) after adjusting for age, race, body mass index, fibre intake, and hormonal contraception use. Intestimonas, Oscillibacter, and Phascolarctobacterium were more often present and Christensenellaceae R-7 group, Collinsella, Coprococcus 2, Moryella, Prevotella 9, Ruminococcaceae UCG-002, Ruminococcaceae UCG-005, and Ruminococcaceae UCG-014 were less commonly present in IBS compared to HC. Despite multiple taxon differences in IBS vs HC, we found no significant associations between taxon presence or abundance and average daily symptom severity within the IBS group. This may indicate the need to account for interactions between microbiome, dietary intake, metabolites, and host factors.

[Source](https://doi.org/10.1163/18762891-bja00015)

10.1163/18762891-bja00015

---

## Gut microbiome in the Graves' disease: Comparison before and after anti-thyroid drug treatment.
 31.05.2024, _C Jeong, H Baek, J Bae, N Hwang, J Ha, YS Cho, DJ Lim_


_Humans, Graves Disease, Gastrointestinal Microbiome, Male, Female, Adult, Antithyroid Agents, Middle Aged, Feces, Case-Control Studies_
While several studies have proposed a connection between the gut microbiome and the pathogenesis of Graves's disease (GD), there has been a lack of reports on alteration in microbiome following using anti-thyroid drug treatment (ATD) to treat GD. Stool samples were collected from newly diagnosed GD patients provided at baseline and after 6 months of ATD treatment. The analysis focused on investigating the association between the changes in the gut microbiome and parameter including thyroid function, thyroid-related antibodies, and the symptom used to assess hyperthyroidism before and after treatment. A healthy control (HC) group consisting of data from 230 healthy subjects (110 males and 120 females) sourced from the open EMBL Nucleotide Sequence Database was included. Twenty-nine GD patients (14 males and 15 females) were enrolled. The analysis revealed a significant reduction of alpha diversity in GD patients. However, after ATD treatment, alpha diversity exhibited a significant increase, restored to levels comparable to the HC levels. Additionally, GD patients displayed lower levels of Firmicutes and higher levels of Bacteroidota. Following treatment, there was an increased in Firmicutes and a decrease in Bacteroidota, resembling levels found in the HC levels. The symptoms of hyperthyroidism were negatively associated with Firmicutes and positively associated with Bacteroidota. GD had significantly lower levels of Roseburia, Lachnospiraceaea, Sutterella, Escherichia-shigella, Parasuterella, Akkermansia, and Phascolarctobacterium compared to HC (all p &lt; 0.05). Post-treatment, Subdoligranulum increased (p = 0.010), while Veillonella and Christensenellaceaea R-7 group decreased (p = 0.023, p = 0.029, respectively). Anaerostipes showed a significant association with both higher smoking pack years and TSHR-Ab levels, with greater abundantce observed in smokers among GD (p = 0.16). Although reduced ratio of Firmicutes/Bacteroidetes was evident in GD, this ratio recovered after treatment. This study postulates the involvement of the gut microbiome in the pathogenesis of GD, suggesting potential restoration after treatment.

[Source](https://doi.org/10.1371/journal.pone.0300678)

10.1371/journal.pone.0300678

---

## Guild-level signature of gut microbiome for diabetic kidney disease.
 31.05.2024, _S Tang, G Wu, Y Liu, B Xue, S Zhang, W Zhang, Y Jia, Q Xie, C Liang, L Wang, H Heng, W Wei, X Shi, Y Hu, J Yang, L Zhao, X Wang, L Zhao, H Yuan_


Current microbiome signatures for chronic diseases such as diabetic kidney disease (DKD) are mainly based on low-resolution taxa such as genus or phyla and are often inconsistent among studies. In microbial ecosystems, bacterial functions are strain specific, and taxonomically different bacteria tend to form co-abundance functional groups called guilds. Here, we identified guild-level signatures for DKD by performing in-depth metagenomic sequencing and conducting genome-centric and guild-based analysis on fecal samples from 116 DKD patients and 91 healthy subjects. Redundancy analysis on 1,543 high-quality metagenome-assembled genomes (HQMAGs) identified 54 HQMAGs that were differentially distributed among the young healthy control group, elderly healthy control group, early-stage DKD patients (EDG), and late-stage DKD patients (LDG). Co-abundance network analysis classified the 54 HQMAGs into two guilds. Compared to guild 2, guild 1 contained more short-chain fatty acid biosynthesis genes and fewer genes encoding uremic toxin indole biosynthesis, antibiotic resistance, and virulence factors. Guild indices, derived from the total abundance of guild members and their diversity, delineated DKD patients from healthy subjects and between different severities of DKD. Age-adjusted partial Spearman correlation analysis showed that the guild indices were correlated with DKD disease progression and with risk indicators of poor prognosis. We further validated that the random forest classification model established with the 54 HQMAGs was also applicable for classifying patients with end-stage renal disease and healthy subjects in an independent data set. Therefore, this genome-level, guild-based microbial analysis strategy may identify DKD patients with different severity at an earlier stage to guide clinical interventions.

[Source](https://doi.org/10.1128/mbio.00735-24)

10.1128/mbio.00735-24

---

## Gut microbiome correlates of recurrent urinary tract infection: a longitudinal, multi-center study

[Source](https://www.thelancet.com/journals/eclinm/article/PIIS2589-5370(24)00069-5/fulltext)

---

## Gut microbiota-derived butyrate induces epigenetic and metabolic reprogramming in myeloid-derived suppressor cells to alleviate primary biliary cholangitis.
 29.05.2024, _R Wang, B Li, B Huang, Y Li, Q Liu, Z Lyu, R Chen, Q Qian, X Liang, X Pu, Y Wu, Y Chen, Q Miao, Q Wang, M Lian, X Xiao, PSC Leung, ME Gershwin, Z You, X Ma, R Tang_


Gut dysbiosis and myeloid-derived suppressor cells (MDSCs) are implicated in primary biliary cholangitis (PBC) pathogenesis. However, it remains unknown whether gut microbiota or their metabolites can modulate MDSCs homeostasis to rectify immune dysregulation in PBC.

[Source](https://www.gastrojournal.org/article/S0016-5085(24)04936-9/abstract)

10.1053/j.gastro.2024.05.014

---

## Gut microbiota research nexus: One Health relationship between human, animal, and environmental resistomes.
 01.06.2024, _Y Fu, Q Dou, K Smalla, Y Wang, TA Johnson, KK Brandt, Z Mei, M Liao, SA Hashsham, A Schäffer, H Smidt, T Zhang, H Li, R Stedtfeld, H Sheng, B Chai, M Virta, X Jiang, F Wang, YG Zhu, JM Tiedje_


The emergence and rapid spread of antimicrobial resistance is of global public health concern. The gut microbiota harboring diverse commensal and opportunistic bacteria that can acquire resistance via horizontal and vertical gene transfers is considered an important reservoir and sink of antibiotic resistance genes (ARGs). In this review, we describe the reservoirs of gut ARGs and their dynamics in both animals and humans, use the One Health perspective to track the transmission of ARG-containing bacteria between humans, animals, and the environment, and assess the impact of antimicrobial resistance on human health and socioeconomic development. The gut resistome can evolve in an environment subject to various selective pressures, including antibiotic administration and environmental and lifestyle factors (e.g., diet, age, gender, and living conditions), and interventions through probiotics. Strategies to reduce the abundance of clinically relevant antibiotic-resistant bacteria and their resistance determinants in various environmental niches are needed to ensure the mitigation of acquired antibiotic resistance. With the help of effective measures taken at the national, local, personal, and intestinal management, it will also result in preventing or minimizing the spread of infectious diseases. This review aims to improve our understanding of the correlations between intestinal microbiota and antimicrobial resistance and provide a basis for the development of management strategies to mitigate the antimicrobial resistance crisis.

[Source](https://doi.org/10.1002/mlf2.12101)

10.1002/mlf2.12101

---

## Exploring the Impact of Airway Microbiome on Asthma Morbidity: A Focus on the Constructing a ‘Eubiosis Reinstatement Therapy’ for Asthma-CURE Project

[Source](https://doi.org/10.1007/s41030-024-00261-3)

---

## A Gram-negative-selective antibiotic that spares the gut microbiome

[Source](https://doi.org/10.1038/s41586-024-07502-0)

---

## <em>Bifidobacterium adolescentis</em>-derived hypaphorine alleviates acetaminophen hepatotoxicity by promoting hepatic Cry1 expression

[Source](https://doi.org/10.1186/s12967-024-05312-6)

---

## <em>Chrysanthemum morifolium</em> attenuates metabolic and alcohol-associated liver disease via gut microbiota and PPARα/γ activation

[Source](https://doi.org/10.1016/j.phymed.2024.155774)

---

## Transgenerational effects of early life stress on the fecal microbiota in mice

[Source](https://doi.org/10.1038/s42003-024-06279-2)

---

## Metagenomics and metaproteomics alterations are associated with kidney disease in opisthorchiasis hamsters fed a high-fat and high-fructose diet

[Source](https://doi.org/10.1371/journal.pone.0301907)

---

## Spatial profiles of the bacterial microbiota throughout the gastrointestinal tract of dairy goats

[Source](https://doi.org/10.1007/s00253-024-13200-8)

---

## Interaction between methanotrophy and gastrointestinal nematodes infection on the rumen microbiome of lambs

[Source](https://doi.org/10.1093/femsec/fiae083)

---

## Antibiotic-altered gut microbiota explain host memory plasticity and disrupt pace-of-life covariation for an aquatic snail

[Source](https://doi.org/10.1093/ismejo/wrae078)

---

## Metagenome-assembled genomes from microbiomes fermenting dairy coproducts

[Source](https://doi.org/10.1128/mra.00173-24)

---

## Global biogeography of microbes driving ocean ecological status under climate change

[Source](https://doi.org/10.1038/s41467-024-49124-0)

---

## Engineering natural microbiomes toward enhanced bioremediation by microbiome modeling

[Source](https://doi.org/10.1038/s41467-024-49098-z)

---

## Interpretable machine learning decodes soil microbiome’s response to drought stress

[Source](https://doi.org/10.1186/s40793-024-00578-1)

---

## Macroalgal eukaryotic microbiome composition indicates novel phylogenetic diversity and broad host spectrum of oomycete pathogens

[Source](https://doi.org/10.1111/1462-2920.16656)

---

## Combining compositional data sets introduces error in covariance network reconstruction

[Source](https://doi.org/10.1093/ismeco/ycae057)

---

## How to Obtain and Compare Metagenome-Assembled Genomes

[Source](https://doi.org/10.1007/978-1-0716-3838-5_6)

---

## A strategy for differential abundance analysis of sparse microbiome data with group-wise structured zeros

[Source](https://doi.org/10.1038/s41598-024-62437-w)

---

## A review of machine learning methods for cancer characterization from microbiome data

[Source](https://doi.org/10.1038/s41698-024-00617-7)

---

