## Mapping the gut microbial structural variations in healthy aging within the Chinese population.
 09.11.2024, _L Shen, H Zhao, Y Xi, Z Wang, K Deng, W Gou, K Zhang, W Hu, J Tang, F Xu, Z Jiang, Y Fu, Y Zhu, D Zhou, YM Chen, JS Zheng_


Mapping gut microbial structural variants (SVs) during human aging may provide fundamental knowledge and mechanistic understanding of the gut microbiome's relationship with healthy aging. We characterize gut microbial SVs from 3,230 Chinese participants, identifying key SVs associated with aging, healthy aging, and age-related chronic diseases. Our findings reveal a pattern of copy number loss in aging-related SVs, with 35 core SVs consistently detected. Additionally, eight SVs distinguish healthy from unhealthy aging, regardless of age. Notably, a 3-kbp deletion SV of Bifidobacterium pseudocatenulatum, encoding plant polysaccharide degradation, is regulated by plant-based diet and contributes to healthy aging through bile acid metabolism. Our analysis also connects SVs to age-related diseases, such as chronic kidney disease, via genes in the methionine-homocysteine pathway. This study deepens our understanding of the gut microbiome's role in aging and could inform future efforts to enhance lifespan and healthspan.

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(24)01319-6)

10.1016/j.celrep.2024.114968

---

## Association between butyrate-producing gut bacteria and the risk of infectious disease hospitalisation: results from two observational, population-based microbiome studies.
 06.09.2024, _RFJ Kullberg, I Wikki, BW Haak, A Kauko, H Galenkamp, H Peters-Sengers, JM Butler, AS Havulinna, J Palmu, D McDonald, C Benchraka, MI Abdel-Aziz, M Prins, AH Maitland van der Zee, BJ van den Born, P Jousilahti, WM de Vos, V Salomaa, R Knight, L Lahti, M Nieuwdorp, T Niiranen, WJ Wiersinga_


_Humans, Middle Aged, Adult, Hospitalization, Male, Female, Gastrointestinal Microbiome, Aged, Finland, Butyrates, RNA, Ribosomal, 16S, Netherlands, Young Adult, Bacteria, Feces, Adolescent, Communicable Diseases, Cohort Studies, Risk Factors_
Microbiota alterations are common in patients hospitalised for severe infections, and preclinical models have shown that anaerobic butyrate-producing gut bacteria protect against systemic infections. However, the relationship between microbiota disruptions and increased susceptibility to severe infections in humans remains unclear. We investigated the relationship between gut microbiota and the risk of future infection-related hospitalisation in two large population-based cohorts.

[Source](https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(24)00304-5/fulltext)

10.1016/S2666-5247(24)00079-X

---

## Reassessing the role of butyrate-producing bacteria in infection risk

[Source](https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(24)00304-5/fulltext)

---

## Microbiome science of human excrement composting.
 09.11.2024, _J Meilander, JG Caporaso_


Linear waste management systems are unsustainable and contribute to environmental degradation, economic inequity, and health disparities. Among the array of environmental challenges stemming from anthropogenic impacts, the management of human excrement (human feces and urine) stands as a significant concern. Over two billion people do not have access to adequate sanitation, signifying a global public health crisis. Composting is the microbial biotechnology aimed at cycling organic waste, including human excrement, for improved public health, agricultural productivity and safety, and environmental sustainability. Applications of modern microbiome -omics and related technologies have the capacity to support continued advances in composting science and praxis. In this article, we review literature focused on applications of microbiome technologies to study composting systems and reactions. The studies we survey generally fall into the categories of animal manure composting, biosolids composting, and human excrement composting. We review experiments utilizing microbiome technologies to investigate strategies for enhancing pathogen suppression and accelerating the biodegradation of organic matter. Additionally, we explore studies focused on the bioengineering potential of microbes as inoculants to facilitate degradation of toxins such as pharmaceuticals or per- and polyfluoroalkyl substances. The findings from these studies underscore the importance of advancing our understanding of composting processes through the integration of emerging microbiome -omics technologies. We conclude that work to-date has demonstrated exciting basic and applied science potential from studying compost microbiomes, with promising implications for enhancing global environmental sustainability and public health.

[Source](https://academic.oup.com/ismej/advance-article/doi/10.1093/ismejo/wrae228/7888797)

10.1093/ismejo/wrae228

---

## Microbiota-derived lysophosphatidylcholine alleviates Alzheimer's disease pathology via suppressing ferroptosis.
 07.11.2024, _X Zha, X Liu, M Wei, H Huang, J Cao, S Liu, X Bian, Y Zhang, F Xiao, Y Xie, W Wang, C Zhang_


Alzheimer's disease (AD) is a pervasive neurodegenerative disorder, and new approaches for its prevention and therapy are critically needed. Here, we elucidate a gut-microbiome-brain axis that offers actionable perspectives for achieving this objective. Using the 5xFAD mouse model, we identify increased Clostridium abundance and decreased Bacteroides abundance as key features associated with β-amyloid (Aβ) burden. Treatment with Bacteroides ovatus, or its associated metabolite lysophosphatidylcholine (LPC), significantly reduces Aβ load and ameliorates cognitive impairment. Mechanistically, LPC acts through the orphan receptor GPR119, inhibiting ACSL4 expression, thereby suppressing ferroptosis and ameliorating AD pathologies. Analysis of fecal and serum samples from individuals with AD also reveals diminished levels of Bacteroides and LPC. This study thus identifies a B.ovatus-triggered pathway regulating AD pathologies and indicates that the use of single gut microbiota, metabolite, or small molecule compound may complement current prevention and treatment approaches for AD.

[Source](https://www.cell.com/cell-metabolism/fulltext/S1550-4131(24)00402-9)

10.1016/j.cmet.2024.10.006

---

## Probiotics and their metabolite spermidine enhance IFN-γ+CD4+ T cell immunity to inhibit hepatitis B virus.
 13.11.2024, _T Wang, Y Fan, S Tan, Z Wang, M Li, X Guo, X Yu, Q Lin, X Song, L Xu, L Li, S Li, L Gao, X Liang, C Li, C Ma_


The therapeutic potential of commensal microbes and their metabolites is promising in the functional cure of chronic hepatitis B virus (HBV) infection, which is defined as hepatitis B surface antigen (HBsAg) loss. Here, using both specific-pathogen-free and germ-free mice, we report that probiotics significantly promote the decline of HBsAg and inhibit HBV replication by enhancing intestinal homeostasis and provoking intrahepatic interferon (IFN)-γ+CD4+ T cell immune response. Depletion of CD4+ T cells or blockage of IFN-γ abolishes probiotics-mediated HBV inhibition. Specifically, probiotics-derived spermidine accumulates in the gut and transports to the liver, where it exhibits a similar anti-HBV effect. Mechanistically, spermidine enhances IFN-γ+CD4+ T cell immunity by autophagy. Strikingly, administration of probiotics in HBV patients reveals a preliminary trend to accelerate the decline of serum HBsAg. In conclusion, probiotics and their derived spermidine promote HBV clearance via autophagy-enhanced IFN-γ+CD4+ T cell immunity, highlighting the therapeutic potential of probiotics and spermidine for the functional cure of HBV patients.

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(24)00593-7)

10.1016/j.xcrm.2024.101822

---

## The phageome of patients with ulcerative colitis treated with donor fecal microbiota reveals markers associated with disease remission.
 20.10.2024, _ME Majzoub, S Paramsothy, C Haifer, R Parthasarathy, TJ Borody, RW Leong, MA Kamm, NO Kaakoush_


_Humans, Colitis, Ulcerative, Fecal Microbiota Transplantation, Bacteriophages, Gastrointestinal Microbiome, Feces, Double-Blind Method, Male, Female, Metagenomics, Adult, Dysbiosis, Middle Aged, Virome, Remission Induction, Anti-Bacterial Agents, Biomarkers_
Bacteriophages are influential within the human gut microbiota, yet they remain understudied relative to bacteria. This is a limitation of studies on fecal microbiota transplantation (FMT) where bacteriophages likely influence outcome. Here, using metagenomics, we profile phage populations - the phageome - in individuals recruited into two double-blind randomized trials of FMT in ulcerative colitis. We leverage the trial designs to observe that phage populations behave similarly to bacterial populations, showing temporal stability in health, dysbiosis in active disease, modulation by antibiotic treatment and by FMT. We identify a donor bacteriophage putatively associated with disease remission, which on genomic analysis was found integrated in a bacterium classified to Oscillospiraceae, previously isolated from a centenarian and predicted to produce vitamin B complex except B12. Our study provides an in-depth assessment of phage populations during different states and suggests that bacteriophage tracking has utility in identifying determinants of disease activity and resolution.

[Source](https://www.nature.com/articles/s41467-024-53454-4)

10.1038/s41467-024-53454-4

---

## Chronic exposure to polycyclic aromatic hydrocarbons alters skin virome composition and virus-host interactions.
 11.11.2024, _S Du, X Tong, MHY Leung, RJ Betts, AC Woo, P Bastien, N Misra, L Aguilar, C Clavaud, PKH Lee_


_Skin, Female, Humans, Virome, Polycyclic Aromatic Hydrocarbons, Microbiota, Bacteria, Viruses, China, Adult, Metagenome, Host Microbial Interactions, Air Pollutants_
Exposure to polycyclic aromatic hydrocarbons (PAHs) in polluted air influences the composition of the skin microbiome, which in turn is associated with altered skin phenotypes. However, the interactions between PAH exposure and viromes are unclear. This study aims to elucidate how PAH exposure affects the composition and function of skin viruses, their role in shaping the metabolism of bacterial hosts, and the subsequent effects on skin phenotype. We analyzed metagenomes from cheek skin swabs collected from 124 Chinese women in our previous study and found that the viruses associated with the two microbiome cutotypes had distinct diversities, compositions, functions, and lifestyles following PAH exposure. Moreover, exposure to high concentrations of PAHs substantially increased interactions between viruses and certain biodegrading bacteria. Under high-PAH exposure, the viruses were enriched in xenobiotic degradation functions, and there was evidence suggesting that the insertion of bacteriophage-encoded auxiliary metabolic genes into hosts aids biodegradation. Under low-PAH exposure conditions, the interactions followed the "Piggyback-the-Winner" model, with Cutibacterium acnes being "winners," whereas under high-PAH exposure, they followed the "Piggyback-the-Persistent" model, with biodegradation bacteria being "persistent." These findings highlight the impact of air pollutants on skin bacteria and viruses, their interactions, and their modulation of skin health. Understanding these intricate relationships could provide insights for developing targeted strategies to maintain skin health in polluted environments, emphasizing the importance of mitigating pollutant exposure and harnessing the potential of viruses to help counteract the adverse effects.

[Source](https://academic.oup.com/ismej/article/18/1/wrae218/7841796)

10.1093/ismejo/wrae218

---

## mbDriver: identifying driver microbes in microbial communities based on time-series microbiome data.
 11.11.2024, _X Tan, F Xue, C Zhang, T Wang_


_Humans, Microbiota, Computational Biology, Algorithms, Gastrointestinal Microbiome_
Alterations in human microbial communities are intricately linked to the onset and progression of diseases. Identifying the key microbes driving these community changes is crucial, as they may serve as valuable biomarkers for disease prevention, diagnosis, and treatment. However, there remains a need for further research to develop effective methods for addressing this critical task. This is primarily because defining the driver microbe requires consideration not only of each microbe's individual contributions but also their interactions. This paper introduces a novel framework, called mbDriver, for identifying driver microbes based on microbiome abundance data collected at discrete time points. mbDriver comprises three main components: (i) data preprocessing of time-series abundance data using smoothing splines based on the negative binomial distribution, (ii) parameter estimation for the generalized Lotka-Volterra (gLV) model using regularized least squares, and (iii) quantification of each microbe's contribution to the community's steady state by manipulating the causal graph implied by gLV equations. The performance of nonparametric spline-based denoising and regularized least squares estimation is comprehensively evaluated on simulated datasets, demonstrating superiority over existing methods. Furthermore, the practical applicability and effectiveness of mbDriver are showcased using a dietary fiber intervention dataset and an ulcerative colitis dataset. Notably, driver microbes identified in the dietary fiber intervention dataset exhibit significant effects on the abundances of short-chain fatty acids, while those identified in the ulcerative colitis dataset show a significant correlation with metabolism-related pathways.

[Source](https://academic.oup.com/bib/article/25/6/bbae580/7889309)

10.1093/bib/bbae580

---

## AMRColab - a user-friendly antimicrobial resistance detection and visualization tool.
 23.10.2024, _SD Lam, S Di Gregorio, MY Ang, E Griffiths, TZMT Jamaluddin, S Nathan, HM Neoh_


_Computational Biology, Humans, Methicillin-Resistant Staphylococcus aureus, Drug Resistance, Bacterial, Software, Anti-Bacterial Agents, Genome, Bacterial_
Antimicrobial resistance (AMR) poses a significant threat to global public health, with the potential to cause millions of deaths annually by 2050. Effective surveillance of AMR pathogens is crucial for monitoring and predicting their behaviour in response to antibiotics. However, many public health professionals lack the necessary bioinformatics skills and resources to analyse pathogen genomes effectively. To address this challenge, we developed AMRColab, an open-access bioinformatics analysis suite hosted on Google Colaboratory. AMRColab enables users with limited or no bioinformatics training to detect and visualize AMR determinants in pathogen genomes using a 'plug-and-play' approach. The platform integrates established bioinformatics tools such as AMRFinderPlus and hAMRonization, allowing users to analyse, compare and visualize trends in AMR pathogens easily. A trial run using methicillin-resistant Staphylococcus aureus (MRSA) strains demonstrated AMRColab's effectiveness in identifying AMR determinants and facilitating comparative analysis across strains. A workshop was conducted and feedback from participants indicated high confidence in using AMRColab and a willingness to incorporate it into their research. AMRColab's user-friendly interface and modular design make it accessible to a diverse audience, including medical laboratory technologists, medical doctors and public health scientists, regardless of their bioinformatics expertise. Future improvements to AMRColab will include enhanced visualization tools, multilingual support and the establishment of an online community platform. AMRColab represents a significant step towards democratizing AMR surveillance and empowering public health professionals to combat AMR effectively.

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.001308#)

10.1099/mgen.0.001308

---

## Seven-year performance of a clinical metagenomic next-generation sequencing test for diagnosis of central nervous system infections.
 12.11.2024, _P Benoit, N Brazer, M de Lorenzi-Tognon, E Kelly, V Servellita, M Oseguera, J Nguyen, J Tang, C Omura, J Streithorst, M Hillberg, D Ingebrigtsen, K Zorn, MR Wilson, T Blicharz, AP Wong, B O'Donovan, B Murray, S Miller, CY Chiu_


Metagenomic next-generation sequencing (mNGS) of cerebrospinal fluid (CSF) is an agnostic method for broad-based diagnosis of central nervous system (CNS) infections. Here we analyzed the 7-year performance of clinical CSF mNGS testing of 4,828 samples from June 2016 to April 2023 performed by the University of California, San Francisco (UCSF) clinical microbiology laboratory. Overall, mNGS testing detected 797 organisms from 697 (14.4%) of 4,828 samples, consisting of 363 (45.5%) DNA viruses, 211 (26.4%) RNA viruses, 132 (16.6%) bacteria, 68 (8.5%) fungi and 23 (2.9%) parasites. We also extracted clinical and laboratory metadata from a subset of the samples (n = 1,164) from 1,053 UCSF patients. Among the 220 infectious diagnoses in this subset, 48 (21.8%) were identified by mNGS alone. The sensitivity, specificity and accuracy of mNGS testing for CNS infections were 63.1%, 99.6% and 92.9%, respectively. mNGS testing exhibited higher sensitivity (63.1%) than indirect serologic testing (28.8%) and direct detection testing from both CSF (45.9%) and non-CSF (15.0%) samples (P &lt; 0.001 for all three comparisons). When only considering diagnoses made by CSF direct detection testing, the sensitivity of mNGS testing increased to 86%. These results justify the routine use of diagnostic mNGS testing for hospitalized patients with suspected CNS infection.

[Source](https://www.nature.com/articles/s41591-024-03275-1)

10.1038/s41591-024-03275-1

---

