## Reporting guidelines for human microbiome research: the STORMS checklist

[Source](https://doi.org/10.1038/s41591-021-01552-x)

---

## Honey bee genetics shape the strain-level structure of gut microbiota in social transmission

[Source](https://doi.org/10.1186/s40168-021-01174-y)

---

## Metatranscriptomics to characterize respiratory virome, microbiome, and host response directly from clinical samples

[Source](https://doi.org/10.1016/j.crmeth.2021.100091)

---

## Response of Lung Microbiota to Changes of Pulmonary Innate Immunity Under Healthy Conditions

[Source](https://doi.org/10.1164/rccm.202109-2187LE)

---

## Small intestinal bacterial overgrowth in Alzheimer’s disease

[Source](https://doi.org/10.1007/s00702-021-02440-x)

---

## Periodontitis Salivary Microbiota Worsens Colitis

[Source](https://doi.org/10.1177/00220345211049781)

---

## The Effects of Moderate Alcohol Consumption on Circulating Metabolites and Gut Microbiota in Patients With Coronary Artery Disease

[Source](https://doi.org/10.3389/fcvm.2021.767692)

---

## Integrative Analysis of LGR5/6 Gene Variants, Gut Microbiota Composition and Osteoporosis Risk in Elderly Population

[Source](https://doi.org/10.3389/fmicb.2021.765008)

---

## Comparison of the Gut Microbiota in Patients with Benign and Malignant Breast Tumors: A Pilot Study

[Source](https://doi.org/10.1177/11769343211057573)

---

## Temporal variability in quantitative human gut microbiome profiles and implications for clinical research

[Source](https://doi.org/10.1038/s41467-021-27098-7)

---

## Association between Parkinson’s disease and the faecal eukaryotic microbiota

[Source](https://doi.org/10.1038/s41531-021-00244-0)

---

## Micronutrient supplements can promote disruptive protozoan and fungal communities in the developing infant gut

[Source](https://doi.org/10.1038/s41467-021-27010-3)

---

## An integrative understanding of the large metabolic shifts induced by antibiotics in critical illness

[Source](https://doi.org/10.1080/19490976.2021.1993598)

---

## Efficacy and Outcomes of Fecal Microbiota Transplantation for Recurrent<em> Clostridioides difficile </em>Infection in Children with Inflammatory Bowel Disease

[Source](https://doi.org/10.1093/ecco-jcc/jjab202)

---

## The anti-inflammatory effect of bacterial short chain fatty acids is partially mediated by endocannabinoids.
 23.11.2021, _A Vijay, A Kouraki, S Gohir, J Turnbull, A Kelly, V Chapman, DA Barrett, WJ Bulsiewicz, AM Valdes_


The endocannabinoid (EC) system has pleiotropic functions in the body. It plays a key role in energy homeostasis and the development of metabolic disorders being a mediator in the relationship between the gut microbiota and host metabolism. In the current study we explore the functional interactions between the endocannabinoid system and the gut microbiome in modulating inflammatory markers. Using data from a 6 week exercise intervention (treatment n = 38 control n = 40) and a cross sectional validation cohort (n = 35), we measured the associations of 2-arachidonoylglycerol (2-AG), anandamide (AEA), N-oleoylethanolamine (OEA) and N-palmitoylethanolamine (PEA) with gut microbiome composition, gut derived metabolites (SCFAs) and inflammatory markers both cross-sectionally and longitudinally. At baseline AEA and OEA were positively associated with alpha diversity (β(SE) = .32 (.06), P = .002; .44 (.04), P &lt; .001) and with SCFA producing bacteria such as Bifidobacterium (2-AG β(SE) = .21 (.10), P &lt; .01; PEA β(SE) = .23 (.08), P &lt; .01), Coprococcus 3 and Faecalibacterium (PEA β(SE) = .29 (.11), P = .01; .25 (.09), P &lt; .01) and negatively associated with Collinsella (AEA β(SE) = -.31 (.12), P = .004). Additionally, we found AEA to be positively associated with SCFA Butyrate (β(SE) = .34 (.15), P = .01). AEA, OEA and PEA all increased significantly with the exercise intervention but remained constant in the control group. Changes in AEA correlated with SCFA butyrate and increases in AEA and PEA correlated with decreases in TNF-ɑ and IL-6 statistically mediating one third of the effect of SCFAs on these cytokines. Our data show that the anti-inflammatory effects of SCFAs are partly mediated by the EC system suggesting that there may be other pathways involved in the modulation of the immune system via the gut microbiome.

[Source](https://doi.org/10.1080/19490976.2021.1997559)

10.1080/19490976.2021.1997559

---

## Bile acids as regulatory molecules and potential targets in metabolic diseases.
 21.11.2021, _AJ Xie, CT Mai, YZ Zhu, XC Liu, Y Xie_


Bile acids are important hydroxylated steroids that are synthesized in the liver from cholesterol for intestinal absorption of lipids and other fatty-nutrient. They also display remarkable and immense functions such as regulating immune responses, managing the apoptosis of cells, participating in glucose metabolism, and so on. Some bile acids were used for the treatment or prevention of diseases such as gallstones, primary biliary cirrhosis, and colorectal cancer. Meanwhile, the accumulation of toxic bile acids leads to apoptosis, necrosis, and inflammation. Alteration of bile acids metabolism, as well as the gut microbiota that interacted with bile acids, contributes to the pathogenesis of metabolic diseases. Therefore, the purpose of this review is to summarize the current functions and pre-clinical or clinical applications of bile acids, and to further discuss the alteration of bile acids in metabolic disorders as well as the manipulation of bile acids metabolism as potential therapeutic targets.

[Source](https://doi.org/10.1016/j.lfs.2021.120152)

10.1016/j.lfs.2021.120152

---

## Renaissance for Phage-Based Bacterial Control.
 22.11.2021, _C Schwarz, J Mathieu, JA Laverde Gomez, P Yu, PJJ Alvarez_


Bacteriophages (phages) are an underutilized biological resource with vast potential for pathogen control and microbiome editing. Phage research and commercialization have increased rapidly in biomedical and agricultural industries, but adoption has been limited elsewhere. Nevertheless, converging advances in DNA sequencing, bioinformatics, microbial ecology, and synthetic biology are now poised to broaden phage applications beyond pathogen control toward the manipulation of microbial communities for defined functional improvements. Enhancements in sequencing combined with network analysis make it now feasible to identify and disrupt microbial associations to elicit desirable shifts in community structure or function, indirectly modulate species abundance, and target hub or keystone species to achieve broad functional shifts. Sequencing and bioinformatic advancements are also facilitating the use of temperate phages for safe gene delivery applications. Finally, integration of synthetic biology stands to create novel phage chassis and modular genetic components. While some fundamental, regulatory, and commercialization barriers to widespread phage use remain, many major challenges that have impeded the field now have workable solutions. Thus, a new dawn for phage-based (chemical-free) precise biocontrol and microbiome editing is on the horizon to enhance, suppress, or modulate microbial activities important for public health, food security, and more sustainable energy production and water reuse.

[Source](https://doi.org/10.1021/acs.est.1c06232)

10.1021/acs.est.1c06232

---

## Sulphur metabolism in colon cancer tissues: a case report and literature review.
 24.11.2021, _H Fukuoka, T Andou, T Moriya, K Narita, K Kasahara, D Miura, Y Sekiguchi, S Suzuki, K Nakagawa, M Ozawa, A Ishibe, I Endo_


_Aged, Chromatography, Liquid, Colonic Neoplasms, Female, Humans, Intestinal Mucosa, Sulfur, Tandem Mass Spectrometry_
Sulphur-containing compounds have been linked to colorectal cancer by factors such as the presence of methyl mercaptan in intestinal gas and long-term dietary intake associated with sulphur-metabolizing microbiota. Therefore, this current case report hypothesized that active sulphur metabolism in colorectal cancer results in the formation of sulphur compounds in the intestine and, thus, examined sulphur metabolites possibly associated with sulphur respiration in colon cancer tissues. The patient was a 73-year-old female that underwent laparoscopic right hemicolectomy for ascending colon cancer. During the surgery, colon cancer tissues and normal intestinal mucosa samples were collected. After optimizing the sample concentrations for homogenization (pre-treatment), the samples were stabilized using a hydroxyphenyl-containing derivative and the relevant metabolites were quantified using liquid chromatography with tandem mass spectrometry. The results showed that cysteine persulfide and cysteine trisulfide levels were higher in colon cancer tissues than in normal mucosal tissues. Thus, sulphur metabolism, possibly sulphur respiration, is enhanced in colon cancer tissues.

[Source](https://doi.org/10.1177/03000605211059936)

10.1177/03000605211059936

---

## Dietary fatty acids in gut health: Absorption, metabolism and function.
 18.11.2021, _E Xu, C Chen, J Fu, L Zhu, J Shu, M Jin, Y Wang, X Zong_


In biological responses, fatty acids (FA) are absorbed and metabolized in the form of substrates for energy production. The molecular structures (number of double bonds and chain length) and composition of dietary FA impact digestion, absorption and metabolism, and the biological roles of FA. Recently, increasing evidence indicates that FA are essentially utilized as an energy source and are signaling molecules that exert physiological activity of gut microbiota and immune responses. In addition, FA could serve as natural ligands for orphan G protein-coupled receptors (GPCR), also called free fatty acid receptors (FFAR), which intertwine metabolic and immune systems via multiple mechanisms. The present review explores the recent findings on FA absorption and its impact on gut health, particularly addressing the mechanism by which dietary FA potentially influences intestinal microbiota and epithelial functions. Also, this work attempts to uncover research ideas for devising future strategies for manipulating the composition of dietary FA to regulate gut health and support a normal immune system for metabolic and immune disorders.

[Source](https://doi.org/10.1016/j.aninu.2021.09.010)

10.1016/j.aninu.2021.09.010

---

## The search for aliens within us: a review of evidence and theory regarding the foetal microbiome.
 17.11.2021, _R Cariño, L Takayasu, W Suda, H Masuoka, K Hirayama, S Konishi, M Umezaki_


The microbiome is believed to be established during the birthing process through exposure to the maternal microbiome and immediate external environment. The absence of a microbiome prior to birth is based on the sterile womb hypothesis, which was formulated at the beginning of the 20th century and is supported primarily by the culture-based approach in microbiological studies.Findings of bacterial presence in products of fertilization such as the placenta, amniotic fluid, foetal membranes, and umbilical cord blood in studies using next-generation DNA sequencing technologies began to challenge the sterile nature of the intrauterine environment during gestation. These studies have been mainly criticized by their approach to contamination and inconclusive evidence of viability. The implications of bacterial presence in utero are far reaching in medicine and basic sciences. If commensal bacteria exist in the foetus, antibiotic therapies in pregnancy particularly for asymptomatic cases will need to be re-evaluated. Experimental studies utilizing gnotobiology may also be impacted by a realignment of theory.This review of existing literature aims to provide insight into the existence of bacteria in utero, specifically the foetal microbiome through analysis of experimental evidence and theoretical concepts, and to suggest approaches that may further provide clarity into this inquiry.

[Source](https://doi.org/10.1080/1040841X.2021.1999903)

10.1080/1040841X.2021.1999903

---

## Human immune diversity: from evolution to modernity.
 19.11.2021, _A Liston, S Humblet-Baron, D Duffy, A Goris_


The extreme diversity of the human immune system, forged and maintained throughout evolutionary history, provides a potent defense against opportunistic pathogens. At the same time, this immune variation is the substrate upon which a plethora of immune-associated diseases develop. Genetic analysis suggests that thousands of individually weak loci together drive up to half of the observed immune variation. Intense selection maintains this genetic diversity, even selecting for the introgressed Neanderthal or Denisovan alleles that have reintroduced variation lost during the out-of-Africa migration. Variations in age, sex, diet, environmental exposure, and microbiome each potentially explain the residual variation, with proof-of-concept studies demonstrating both plausible mechanisms and correlative associations. The confounding interaction of many of these variables currently makes it difficult to assign definitive contributions. Here, we review the current state of play in the field, identify the key unknowns in the causality of immune variation, and identify the multidisciplinary pathways toward an improved understanding.

[Source](https://doi.org/10.1038/s41590-021-01058-1)

10.1038/s41590-021-01058-1

---

## Human Milk Oligosaccharides as a Missing Piece in Combating Nutritional Issues during Exclusive Breastfeeding.
 20.11.2021, _V Sudarma, B Hegar, A Hidayat, R Agustina_


Extensive studies have shown that breast milk is the best source of nutrition for infants, especially during the first six months, because it fulfills almost all of their nutritional needs. Among the many functional building blocks in breast milk, human milk oligosaccharides (HMOs) have been receiving more attention recently. Furthermore, it is the third most common group of compounds in human milk, and studies have demonstrated the health benefits it provides for infants, including improved nutritional status. HMOs were previously known as the 'bifidus factor' due to their 'bifidogenic' or prebiotic effects, which enabled the nourishment of the gastrointestinal microbiota. Healthy gastrointestinal microbiota are intestinal health substrates that increase nutrient absorption and reduce the incidence of diarrhea. In addition, HMOs, directly and indirectly, protect infants against infections and strengthen their immune system, leading to a positive energy balance and promoting normal growth. Non-modifiable factors, such as genetics, and modifiable factors (e.g., maternal health, diet, nutritional status, environment) can influence the HMO profile. This review provides an overview of the current understanding of how HMOs can contribute to the prevention and treatment of nutritional issues during exclusive breastfeeding.

[Source](https://doi.org/10.5223/pghn.2021.24.6.501)

10.5223/pghn.2021.24.6.501

---

## Advancements in the Pharmaceutical Applications of Probiotics: Dosage Forms and Formulation Technology.
 24.11.2021, _KC Baral, R Bajracharya, SH Lee, HK Han_


_Ecosystem, Gastrointestinal Microbiome, Pharmaceutical Preparations, Probiotics, Technology_
Probiotics have demonstrated their high potential to treat and/or prevent various diseases including neurodegenerative disorders, cancers, cardiovascular diseases, and inflammatory diseases. Probiotics are also effective against multidrug-resistant pathogens and help maintain a balanced gut microbiota ecosystem. Accordingly, the global market of probiotics is growing rapidly, and research efforts to develop probiotics into therapeutic adjuvants are gaining momentum. However, because probiotics are living microorganisms, many biological and biopharmaceutical barriers limit their clinical application. Probiotics may lose their activity in the harsh gastric conditions of the stomach or in the presence of bile salts. Moreover, they easily lose their viability under thermal or oxidative stress during their preparation and storage. Therefore, stable formulations of probiotics are required to overcome the various physicochemical, biopharmaceutical, and biological barriers and to maximize their therapeutic effectiveness and clinical applicability. This review provides an overview of the pharmaceutical applications of probiotics and covers recent formulation approaches to optimize the delivery of probiotics with particular emphasis on various dosage forms and formulation technologies.

[Source](https://doi.org/10.2147/IJN.S337427)

10.2147/IJN.S337427

---

## <em>Lactobacillus rhamnosus </em>strain LRH05 intervention ameliorated body weight gain and adipose inflammation via modulating the gut microbiota in high-fat diet-induced obese mice

[Source](https://doi.org/10.1002/mnfr.202100348)

---

## Bifidobacterium infantis Maintains Genome Stability in Ulcerative Colitis via Regulating Anaphase-Promoting Complex Subunit 7.
 20.11.2021, _T Han, X Hu, K Li, D Zhang, Y Zhang, J Li_


Probiotics represents a promising intestinal microbiota-targeted therapeutic method for the treatment of ulcerative colitis (UC). Several lines of evidence implicate that Bifidobacterium infantis serves as a probiotic strain with proven efficacy in maintaining the remission of UC. However, the exact mechanisms underlying the beneficial effects of B. infantis on UC progression have yet to be elucidated. Herein, we provide evidence that B. infantis acts as a key predisposing factor for the maintenance of host genome stability. First, we showed that the fecal microbiota transplantation (FMT) of UC-derived feces contributes to more severely DNA damage in dextran sodium sulfate (DSS)-induced mice likely due to mucosa-associated microbiota alterations, as reflected by the rapid appearance of DNA double strand breaks (DSBs), a typical marker of genome instability. Genomic DNA damage analysis of colon tissues derived from healthy controls, patients with UC or dysplasia, and colitis associated cancer (CAC) patients, revealed an enhanced level of DSBs with aggravation in the degree of the intestinal mucosal lesions. To evaluate whether B. infantis modulates the host genome stability, we employed the DSS-induced colitis model and a TNFα-induced intestinal epithelial cell model. Following the administration of C57BL/6 mice with B. infantis via oral gavage, we found that the development of DSS-induced colitis in mice was significantly alleviated, in contrast to the colitis model group. Notably, B. infantis administration decreased DSB levels in both DSS-induced colitis and TNF-treated colonial cell model. Accordingly, our bioinformatic and functional studies demonstrated that B. infantis altered signal pathways involved in ubiquitin-mediated proteolysis, transcriptional misregulation in cancer, and the bacterial invasion of epithelial cells. Mechanistically, B. infantis upregulated anaphase-promoting complex subunit 7 (APC7), which was significantly suppressed in colitis condition, to activate the DNA repair pathway and alter the genome stability, while downregulation of APC7 abolished the efficiency of B. infantis treatment to induce a decrease in the level of DSBs in TNFα-induced colonial cells. Collectively, our results support that B. infantis orchestrates a molecular network involving in APC7 and genome stability, to control UC development at the clinical, biological, and mechanistic levels. Supplying B. infantis and targeting its associated pathway will yield valuable insight into the clinical management of UC patients.

[Source](https://doi.org/10.3389/fmicb.2021.761113)

10.3389/fmicb.2021.761113

---

## <em>Bifidobacterium infantis </em>Maintains Genome Stability in Ulcerative Colitis via Regulating Anaphase-Promoting Complex Subunit 7

[Source](https://doi.org/10.3389/fmicb.2021.761113)

---

## Galacto-oligosaccharide Supplementation Modulates Pathogen-Commensal Competition between <em>Streptococcus agalactiae</em> and <em>Streptococcus salivarius</em>

[Source](https://doi.org/10.1002/cbic.202100559)

---

## Metagenome-assembled genomes and gene catalog from the chicken gut microbiome aid in deciphering antibiotic resistomes.
 19.11.2021, _Y Feng, Y Wang, B Zhu, GF Gao, Y Guo, Y Hu_


Gut microbial reference genomes and gene catalogs are necessary for understanding the chicken gut microbiome. Here, we assembled 12,339 microbial genomes and constructed a gene catalog consisting of ~16.6 million genes by integrating 799 public chicken gut microbiome samples from ten countries. We found that 893 and 38 metagenome-assembled genomes (MAGs) in our dataset were putative novel species and genera, respectively. In the chicken gut, Lactobacillus aviarius and Lactobacillus crispatus were the most common lactic acid bacteria, and glycoside hydrolases were the most abundant carbohydrate-active enzymes (CAZymes). Antibiotic resistome profiling results indicated that Chinese chicken samples harbored a higher relative abundance but less diversity of antimicrobial resistance genes (ARGs) than European samples. We also proposed the effects of geography and host species on the gut resistome. Our study provides the largest integrated metagenomic dataset from the chicken gut to date and demonstrates its value in exploring chicken gut microbial genes.

[Source](https://doi.org/10.1038/s42003-021-02827-2)

10.1038/s42003-021-02827-2

---

## Contrasting microbiome dynamics of putative denitrifying bacteria in two octocoral species exposed to dissolved organic carbon (DOC) and warming.
 17.11.2021, _N Xiang, C Hassenrück, C Pogoreutz, N Rädecker, SM Simancas-Giraldo, CR Voolstra, C Wild, A Gärdes_


Mutualistic nutrient cycling in the coral-algae symbiosis depends on limited nitrogen (N) availability for algal symbionts. Denitrifying prokaryotes capable of reducing nitrate or nitrite to dinitrogen could thus support coral holobiont functioning by limiting N availability. Octocorals show some of the highest denitrification rates among reef organisms, however little is known about the community structures of associated denitrifiers and their response to environmental fluctuations. Combining 16S rRNA gene amplicon sequencing with nirS in-silico PCR and quantitative PCR, we found differences in bacterial community dynamics between two octocorals exposed to excess dissolved organic carbon (DOC) and concomitant warming. While bacterial communities of the gorgonian Pinnigorgia flava remained largely unaffected by DOC and warming, the soft coral Xenia umbellata exhibited a pronounced shift towards Alphaproteobacteria dominance under excess DOC. Likewise, the relative abundance of denitrifiers was not altered in P. flava, but decreased by one order of magnitude in X. umbellata under excess DOC likely due to decreased proportions of Ruegeria spp. Given that holobiont C:N ratios remained stable in P. flava but showed a pronounced increase with excess DOC in X. umbellata host, our results suggest that microbial community dynamics may reflect the nutritional status of the holobiont. Hence, denitrifier abundance may be directly linked to N availability. This suggests a passive regulation of N cycling microbes, which could help stabilize nutrient limitation in the coral-algal symbiosis and thereby support holobiont functioning in a changing environment. Importance Octocorals are important members of reef-associated benthic communities that can rapidly replace scleractinian corals as the dominant ecosystem engineers on degraded reefs. Considering the substantial change in the (a)biotic environment that is commonly driving reef degradation, maintaining a dynamic and metabolically diverse microbial community might contribute to octocoral acclimatization and ecological adaptation. Nitrogen (N) cycling microbes, in particular denitrifying prokaryotes, may support holobiont functioning by limiting internal N availability, but little is known about the identity and (a)biotic drivers of octocoral-associated denitrifiers. Here, we show contrasting dynamics of bacterial communities associated with two common octocoral species, the soft coral Xenia umbellata and the gorgonian Pinnigorgia flava after a six-week exposure to excess dissolved organic carbon (DOC) under concomitant warming conditions. The specific responses of denitrifier communities associated with the two octocoral species aligned with the nutritional status of holobiont members. This suggests a passive regulation of this microbial trait based on N availability in the coral holobiont.

[Source](https://doi.org/10.1128/AEM.01886-21)

10.1128/AEM.01886-21

---

## Emergent bacterial community properties induce enhanced drought tolerance in <em>Arabidopsis</em>

[Source](https://doi.org/10.1038/s41522-021-00253-0)

---

## Drops join to make a stream: high-throughput nanoscale cultivation to grasp the lettuce root microbiome.
 19.11.2021, _A Persyn, A Mueller, S Goormachtig_


Root endospheres house complex and diverse bacterial communities, of which many strains have not been cultivated yet by means of the currently available isolation techniques. The Prospector® (General Automation Lab Technologies, San Carlos, CA, USA), an automated and high-throughput bacterial cultivation system, was applied to analyse the root endomicrobiome of lettuce (Lactuca sativa L.). By using deep sequencing, we compared the results obtained with the Prospector and the traditional solid medium culturing and extinction methods. We found that the species richness did not differ and that the amount of previously uncultured bacteria did not increase, but that the bacterial diversity isolated by the three methods varied. In addition, the tryptic soy broth and King's B media provided a lower, but different, diversity of bacteria than that of Reasoner's 2A (R2A) medium when used within the Prospector system and the number of unique bacterial strains did not weigh up against those isolated with the R2A medium. Thus, to cultivate as broad a variety of bacteria as possible, divergent isolation techniques should be used in parallel. Thanks to its speed and limited manual requirements, the Prospector is a valuable system to enlarge root microbiome culture collections.

[Source](https://doi.org/10.1111/1758-2229.13014)

10.1111/1758-2229.13014

---

## Pan-genome analysis identifies intersecting roles for <em>Pseudomonas</em> specialized metabolites in potato pathogen inhibition

[Source](https://doi.org/10.7554/eLife.71900)

---

## Small-scale agricultural grassland management can affect soil fungal community structure as much as continental scale geographic patterns.
 18.11.2021, _A Fox, F Widmer, A Barreiro, M Jongen, M Musyoki, Â Vieira, J Zimmermann, C Cruz, LM Dimitrova-Mårtensson, F Rasche, L Silva, A Lüscher_


A European transect was established, ranging from Sweden to the Azores, to determine the relative influence of geographic factors and agricultural small-scale management on the grassland soil microbiome. Within each of five countries (factor 'Country'), which maximized a range of geographic factors, two differing growth condition regions (factor 'GCR') were selected: a favorable region with conditions allowing for high plant biomass production and a contrasting less favorable region with a markedly lower potential. Within each region, grasslands of contrasting management intensities (factor 'MI') were defined: intensive and extensive, from which soil samples were collected. Across the transect, 'MI' was a strong differentiator of fungal community structure, having a comparable effect to continental scale geographic factors ('Country'). 'MI' was also a highly significant driver of bacterial community structure, but 'Country' was clearly the stronger driver. For both, 'GCR' was the weakest driver. Also at the regional level, strong effects of MI occurred on various measures of the soil microbiome (i.e. OTU richness, management-associated indicator OTUs), though the effects were largely regional-specific. Our results illustrate the decisive influence of grassland MI on soil microbial community structure, over both regional and continental scales, and, thus, highlight the importance of preserving rare extensive grasslands.

[Source](https://doi.org/10.1093/femsec/fiab148)

10.1093/femsec/fiab148

---

## Geographic and position-based variations in phyllospheric bacterial communities present on flue-cured tobacco.
 18.11.2021, _J Ye, Y Ding, X Qi, J Xu, X Yang, Z Zhang_


Although tobacco leaves (TLs) contain abundant bacteria, how the geography and leaf position of TLs affect these bacteria is unclear. Here, TLs at different positions from Henan (HN, strong flavor style) and Yunnan (YN, fresh flavor style) provinces were collected, and the bacteria were characterized by Illumina sequencing at harvest and 1 year of storage. Bacterial communities were very different between TLs originating from different geographical areas and positions, and beta diversity analysis showed that leaf position was the most important factor for phyllospheric bacterial communities, followed by geographical area and storage time. At the genus level, Subdoligranulum, Thermus, and Acinetobacter were obviously more abundant in HN than in YN, while Blautia and Ruminococcus were significantly more abundant in YN. These differences in bacterial communities decreased after 1 year of storage, indicating that the microbiota tends to become similar during tobacco processing. Storage time also affected the phyllospheric bacteria of TLs, as the bacterial communities shifted significantly on both HN and YN TLs after 1 year of storage. Significant differences in the predicted genes were also observed between the different geographic locations and leaf positions. Potential human pathogens, including Acinetobacter, Methylobacterium, and Escherichia-Shigella, were greatly different between TLs originating from different areas and positions. These data suggested that geographic variations and positions were associated with phyllospheric bacterial communities on TLs, which may be related to not only the flavor style and quality of TLs but also the potential health risks to humans. KEY POINTS: • Tobacco leaf position and tobacco growth location affected bacterial communities. • Microbial communities of TLs shifted significantly after one year of storage. • Potential human pathogens differed at different leaf positions and growth locations.

[Source](https://doi.org/10.1007/s00253-021-11671-7)

10.1007/s00253-021-11671-7

---

## Improved quantitative microbiome profiling for environmental antibiotic resistance surveillance.
 22.11.2021, _A Ott, M Quintela-Baluja, AM Zealand, G O'Donnell, MRM Haniffah, DW Graham_


Understanding environmental microbiomes and antibiotic resistance (AR) is hindered by over reliance on relative abundance data from next-generation sequencing. Relative data limits our ability to quantify changes in microbiomes and resistomes over space and time because sequencing depth is not considered and makes data less suitable for Quantitative Microbial Risk Assessments (QMRA), critical in quantifying environmental AR exposure and transmission risks.

[Source](https://doi.org/10.1186/s40793-021-00391-0)

10.1186/s40793-021-00391-0

---

## Sediment Microbiota in Response to Circuitry of Sediment Microbial Fuel Cells, Revealed by 16S rRNA Gene Amplicon Sequencing.
 19.11.2021, _Y Takemura, M Aoki, T Tran P, N Tomioka, K Kubota, N Matsuura, K Syutsubo_


Information about sediment microbiota affected by sediment microbial fuel cells (SMFC) is limited. A laboratory-scale SMFC was applied to a eutrophic lake sediment under closed-circuit/open-circuit conditions. We analyzed the prokaryotes in the sediment adhering to the anode material. The archaeal family Methanoperedenaceae was a predominant group under closed-circuit conditions.

[Source](https://doi.org/10.1128/MRA.00984-21)

10.1128/MRA.00984-21

---

## Bacterial response to spatial gradients of algal-derived nutrients in a porous microplate

[Source](https://doi.org/10.1038/s41396-021-01147-x)

---

## Microbial ecology of tourist Paleolithic caves.
 18.11.2021, _Z Bontemps, L Alonso, T Pommier, M Hugoni, Y Moenne-Loccoz_


Microorganisms colonize caves extensively, and in caves open for tourism they may cause alterations on wall surfaces. This is a major concern in caves displaying Paleolithic art, which is usually fragile and may be irremediably damaged by microbial alterations. Therefore, many caves were closed for preservation purposes, e.g. Lascaux (France), Altamira (Spain), while others were never opened to the public to avoid microbial contamination, e.g. Chauvet Cave (France), etc. The recent development of high-throughput sequencing technologies allowed several descriptions of cave microbial diversity and prompted the writing of this review, which focuses on the cave microbiome for the three domains of life (Bacteria, Archaea, microeukaryotes), the impact of tourism-related anthropization on microorganisms in Paleolithic caves, and the development of microbial alterations on the walls of these caves. This review shows that the microbial phyla prevalent in pristine caves are similar to those evidenced in water, soil, plant and metazoan microbiomes, but specificities at lower taxonomic levels remain to be clarified. Most of the data relates to Bacteria and Fungi, while other microeukaryotes and Archaea are poorly documented. Tourism may cause shifts in the microbiota of Paleolithic caves, but larger-scale investigation are required as these shifts may differ from one cave to the next. Finally, different types of alterations can occur in caves, especially in Paleolithic caves. Many microorganisms potentially involved have been identified, but diversity analyses of these alterations have not always included a comparison with neighboring unaltered zones as controls, making such associations uncertain. It is expected that omics technologies will also allow a better understanding of the functional diversities of the cave microbiome. This will be needed to decipher microbiome dynamics in response to touristic frequentation, to guide cave management, and to identify the most appropriate reclamation approaches to mitigate microbial alterations in tourist Paleolithic caves.

[Source](https://doi.org/10.1016/j.scitotenv.2021.151492)

10.1016/j.scitotenv.2021.151492

---

## Multivariate statistical monitoring system for microbial population dynamics.
 17.11.2021, _K Ishiya, S Aburatani_


Microbiomes in their natural environments vary dynamically with changing environmental conditions. The detection of these dynamic changes in microbial populations is critical for understanding the impact of environmental changes on the microbial community. Here, we propose a novel method to detect time-series changes in the microbiome, based on multivariate statistical process control. By focusing on the interspecies structures, this approach enables the robust detection of time-series changes in a microbiome composed of a large number of microbial species. Applying this approach to empirical human gut microbiome data, we accurately traced time-series changes in microbiota composition induced by a dietary intervention trial. This method was also excellent for tracking the recovery process after the intervention. Our approach can be useful for monitoring dynamic changes in complex microbial communities.

[Source](https://doi.org/10.1088/1478-3975/ac3ad6)

10.1088/1478-3975/ac3ad6

---

## GMrepo v2: a curated human gut microbiome database with special focus on disease markers and cross-dataset comparison.
 17.11.2021, _D Dai, J Zhu, C Sun, M Li, J Liu, S Wu, K Ning, LJ He, XM Zhao, WH Chen_


GMrepo (data repository for Gut Microbiota) is a database of curated and consistently annotated human gut metagenomes. Its main purposes are to increase the reusability and accessibility of human gut metagenomic data, and enable cross-project and phenotype comparisons. To achieve these goals, we performed manual curation on the meta-data and organized the datasets in a phenotype-centric manner. GMrepo v2 contains 353 projects and 71,642 runs/samples, which are significantly increased from the previous version. Among these runs/samples, 45,111 and 26,531 were obtained by 16S rRNA amplicon and whole-genome metagenomics sequencing, respectively. We also increased the number of phenotypes from 92 to 133. In addition, we introduced disease-marker identification and cross-project/phenotype comparison. We first identified disease markers between two phenotypes (e.g. health versus diseases) on a per-project basis for selected projects. We then compared the identified markers for each phenotype pair across datasets to facilitate the identification of consistent microbial markers across datasets. Finally, we provided a marker-centric view to allow users to check if a marker has different trends in different diseases. So far, GMrepo includes 592 marker taxa (350 species and 242 genera) for 47 phenotype pairs, identified from 83 selected projects. GMrepo v2 is freely available at: https://gmrepo.humangut.info.

[Source](https://doi.org/10.1093/nar/gkab1019)

10.1093/nar/gkab1019

---

## metaSNV v2: detection of SNVs and subspecies in prokaryotic metagenomes.
 18.11.2021, _T Van Rossum, PI Costea, L Paoli, R Alves, R Thielemann, S Sunagawa, P Bork_


Taxonomic analysis of microbial communities is well supported at the level of species and strains. However, species can contain significant phenotypic diversity and strains are rarely widely shared across global populations. Stratifying the diversity between species and strains can identify "subspecies", which are a useful intermediary. High-throughput identification and profiling of subspecies is not yet supported in the microbiome field. Here, we use an operational definition of subspecies based on SNV patterns within species to identify and profile subspecies in metagenomes, along with their distinctive SNVs and genes. We incorporate this method into metaSNV v2, which extends existing SNV-calling software to support further SNV interpretation for population genetics. These new features support microbiome analyses to link SNV profiles with host phenotype or environment and niche-specificity. We demonstrate subspecies identification in marine and faecal metagenomes. In the latter, we analyse 70 species in 7,524 adult and infant subjects, supporting a common subspecies population structure in the human gut microbiome and illustrating some limits in subspecies calling.

[Source](https://doi.org/10.1093/bioinformatics/btab789)

10.1093/bioinformatics/btab789

---

## Assessing saliva microbiome collection and processing methods.
 22.11.2021, _AJS Armstrong, V Parmar, MJ Blaser_


The oral microbiome has been connected with lung health and may be of significance in the progression of SARS-CoV-2 infection. Saliva-based SARS-CoV-2 tests provide the opportunity to leverage stored samples for assessing the oral microbiome. However, these collection kits have not been tested for their accuracy in measuring the oral microbiome. Saliva is highly enriched with human DNA and reducing it prior to shotgun sequencing may increase the depth of bacterial reads. We examined both the effect of saliva collection method and sequence processing on measurement of microbiome depth and diversity by 16S rRNA gene amplicon and shotgun metagenomics. We collected 56 samples from 22 subjects. Each subject provided saliva samples with and without preservative, and a subset provided a second set of samples the following day. 16S rRNA gene (V4) sequencing was performed on all samples, and shotgun metagenomics was performed on a subset of samples collected with preservative with and without human DNA depletion before sequencing. We observed that the beta diversity distances within subjects over time was smaller than between unrelated subjects, and distances within subjects were smaller in samples collected with preservative. Samples collected with preservative had higher alpha diversity measuring both richness and evenness. Human DNA depletion before extraction and shotgun sequencing yielded higher total and relative reads mapping to bacterial sequences. We conclude that collecting saliva with preservative may provide more consistent measures of the oral microbiome and depleting human DNA increases yield of bacterial sequences.

[Source](https://doi.org/10.1038/s41522-021-00254-z)

10.1038/s41522-021-00254-z

---

## Analysing microbiome intervention design studies: Comparison of alternative multivariate statistical methods.
 21.11.2021, _M Khomich, I Måge, I Rud, I Berget_


The diet plays a major role in shaping gut microbiome composition and function in both humans and animals, and dietary intervention trials are often used to investigate and understand these effects. A plethora of statistical methods for analysing the differential abundance of microbial taxa exists, and new methods are constantly being developed, but there is a lack of benchmarking studies and clear consensus on the best multivariate statistical practices. This makes it hard for a biologist to decide which method to use. We compared the outcomes of generic multivariate ANOVA (ASCA and FFMANOVA) against statistical methods commonly used for community analyses (PERMANOVA and SIMPER) and methods designed for analysis of count data from high-throughput sequencing experiments (ALDEx2, ANCOM and DESeq2). The comparison is based on both simulated data and five published dietary intervention trials representing different subjects and study designs. We found that the methods testing differences at the community level were in agreement regarding both effect size and statistical significance. However, the methods that provided ranking and identification of differentially abundant operational taxonomic units (OTUs) gave incongruent results, implying that the choice of method is likely to influence the biological interpretations. The generic multivariate ANOVA tools have the flexibility needed for analysing multifactorial experiments and provide outputs at both the community and OTU levels; good performance in the simulation studies suggests that these statistical tools are also suitable for microbiome data sets.

[Source](https://doi.org/10.1371/journal.pone.0259973)

10.1371/journal.pone.0259973

---

## Technical note: overcoming host contamination in bovine vaginal metagenomic samples with Nanopore adaptive sequencing.
 18.11.2021, _CT Ong, EM Ross, GB Boe-Hansen, C Turni, BJ Hayes, AE Tabor_


Animal metagenomic studies, in which host-associated microbiomes are profiled, are an increasingly important contribution to our understanding of the physiological functions, health and susceptibility to diseases of livestock. One of the major challenges in these studies is host DNA contamination, which limits the sequencing capacity for metagenomic content and reduces the accuracy of metagenomic profiling. This is the first study comparing the effectiveness of different sequencing methods for profiling bovine vaginal metagenomic samples. We compared the new method of Oxford Nanopore Technologies (ONT) adaptive sequencing, which can be used to target or eliminate defined genetic sequences, to standard ONT sequencing, Illumina 16S rDNA amplicon sequencing, and Illumina shotgun sequencing. The efficiency of each method in recovering the metagenomic data and recalling the metagenomic profiles was assessed. ONT adaptive sequencing yielded a higher amount of metagenomic data than the other methods per 1 Gb of sequence data. The increased sequencing efficiency of ONT adaptive sequencing consequently reduced the amount of raw data needed to provide sufficient coverage for the metagenomic samples with high host-to-microbe DNA ratio. Additionally, the long reads generated by ONT adaptive sequencing retained the continuity of read information, which benefited the in-depth annotations for both taxonomical and functional profiles of the metagenome. The different methods resulted in the identification of different taxa. Genera Clostridium, which was identified at low abundances and categorised under Order "Unclassified Clostridiales" when using the 16S rDNA amplicon sequencing method, was identified to be the dominant genera in the sample when sequenced with the three other methods. Additionally, higher numbers of annotated genes were identified with ONT adaptive sequencing, which also produced high coverage on most of the commonly annotated genes. This study illustrates the advantages of ONT adaptive sequencing in improving the amount of metagenomic data derived from microbiome samples with high host-to-microbe DNA ratio and the advantage of long reads in preserving intact information for accurate annotations.

[Source](https://doi.org/10.1093/jas/skab344)

10.1093/jas/skab344

---

## Mian: Interactive Web-Based Microbiome Data Table Visualization and Machine Learning Platform.
 17.11.2021, _BT Jin, F Xu, RT Ng, JC Hogg_


Mian is a web application to interactively visualize, run statistical tools, and train machine learning models on operational taxonomic unit (OTU) or amplicon sequence variant (ASV) datasets to identify key taxonomic groups, diversity trends, or taxonomic composition shifts in the context of provided categorical or numerical sample metadata. Tools, including Fisher's exact test, Boruta feature selection, alpha and beta diversity, and random forest and deep neural network classifiers, facilitate open-ended data exploration and hypothesis generation on microbial datasets.

[Source](https://doi.org/10.1093/bioinformatics/btab754)

10.1093/bioinformatics/btab754

---

