## Primate phageomes are structured by superhost phylogeny and environment

[Source](https://www.pnas.org/content/118/15/e2013535118)

---

## Fungal-induced glycolysis in macrophages promotes colon cancer by enhancing innate lymphoid cell secretion of IL-22.
 16.02.2021, _Y Zhu, T Shi, X Lu, Z Xu, J Qu, Z Zhang, G Shi, S Shen, Y Hou, Y Chen, T Wang_


Incorporation of microbiome data has recently become important for prevention, diagnosis, and treatment of colorectal cancer, and several species of bacteria were shown to be associated with carcinogenesis. However, the role of commensal fungi in colon cancer remains poorly understood. Here, we report that mice lacking the c-type lectin Dectin-3 (Dectin-3-/- ) show increased tumorigenesis and Candida albicans burden upon chemical induction. Elevated C. albicans load triggered glycolysis in macrophages and interleukin-7 (IL-7) secretion. IL-7 induced IL-22 production in RORγt+ (group 3) innate lymphoid cells (ILC3s) via aryl hydrocarbon receptor and STAT3. Consistently, IL-22 frequency in tumor tissues of colon cancer patients positively correlated with fungal burden, indicating the relevance of this regulatory axis in human disease. These results establish a C. albicans-driven crosstalk between macrophages and innate lymphoid cells in the intestine and expand our understanding on how commensal mycobiota regulate host immunity and promote tumorigenesis.

[Source](https://www.embopress.org/doi/full/10.15252/embj.2020105320)

10.15252/embj.2020105320

---

## A method for detection of SARS-CoV-2 RNA in healthy human stool: a validation study.
 07.04.2021, _MP Coryell, M Iakiviak, N Pereira, PP Murugkar, J Rippe, DB Williams, T Heald-Sargent, LN Sanchez-Pinto, J Chavez, JL Hastie, RL Sava, CZ Lien, TT Wang, WJ Muller, MA Fischbach, PE Carlson_


Faecal shedding of SARS-CoV-2 has raised concerns about transmission through faecal microbiota transplantation procedures. Validation parameters of authorised tests for SARS-CoV-2 RNA detection in respiratory samples are described in product labelling, whereas the published methods for SARS-CoV-2 detection from faecal samples have not permitted a robust description of the assay parameters. We aimed to develop and validate a test specifically for detection of SARS-CoV-2 in human stool.

[Source](https://www.sciencedirect.com/science/article/pii/S2666524721000598)

10.1016/S2666-5247(21)00059-8

---

## The Ocular Microbiome and Microbiota and their Effects on Ocular Surface Pathophysiology and Disorders.
 05.04.2021, _P Aragona, C Baudouin, JM Benitez Del Castillo, E Messmer, S Barabino, J Merayo-Lloves, F Brignole-Baudouin, L Inferrera, M Rolando, R Mencucci, M Rescigno, S Bonini, M Labetoulle_


The ocular surface flora performs an important role in the defense mechanisms of the ocular surface system. Its regulation of the immunological activity and the barrier effect against pathogen invasion are remarkable. Composition of the flora differs according to the methods of investigation, because the microbiome, composed of the genetic material of bacteria, fungi, viruses, protozoa, and eukaryotes on the ocular surface, differs from the microbiota, which is the community of microorganisms that colonize the ocular surface. The observed composition of the ocular surface flora depends on harvesting and examining methods, whether with traditional culture or with more refined genetic analysis, based on rRNA and DNA sequencing. Environment, diet, sex, and age influence the microbial flora composition, thus complicating the analysis of the baseline status. Moreover, potentially pathogenic organisms can affect its composition like also various disorders, including chronic inflammation, and therapies applied to the ocular surface. A better understanding of the composition and function of microbial communities at the ocular surface could bring new insights and clarify the epidemiology and pathology of ocular surface dynamics in health and disease. The purpose of this review is to provide an up-to-date overview of knowledge about this topic.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0039625721000837)

10.1016/j.survophthal.2021.03.010

---

## Multi-dimensional study of the oral microbiome, metabolite, and immunologic environment in peanut allergy.
 05.04.2021, _HE Ho, Y Chun, S Jeong, O Jumreornvong, SH Sicherer, S Bunyavanich_


The oral mucosa is the initial interface between food antigens, microbiota and mucosal immunity, yet little is known about oral host-environment dynamics in food allergy.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0091674921005509)

10.1016/j.jaci.2021.03.028

---

## Temporal Variation in Staphylococcus aureus Protein A Genotypes from Nose and Skin in Atopic Dermatitis Patients.
 06.04.2021, _MMF van Mierlo, SGMA Pasmans, JEE Totté, J de Wit, BL Herpers, MC Vos, CHW Klaassen, LM Pardo_


Staphylococcus aureus colonization is associated with disease severity in patients with atopic dermatitis (AD).

[Source](https://www.karger.com/Article/FullText/515235)

10.1159/000515235

---

## Sex-Based Differences in Fecal Short-Chain Fatty Acid and Gut Microbiota in Irritable Bowel Syndrome Patients.
 06.04.2021, _Q Sun, Z Liu, L Zhang, H Wei, L Song, S Zhu, M He, L Duan_


Irritable bowel syndrome (IBS) is characterized by recurrent abdominal pain and altered defecation, with a female predominance. Alterations in fecal short-chain fatty acids (SCFAs) is associated with IBS. However, little is known regarding sex-based differences in this regard.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/1751-2980.12988)

10.1111/1751-2980.12988

---

## Ecological and evolutionary responses to antibiotic treatment in the human gut microbiota.
 06.04.2021, _JH Pennycook, PD Scanlan_


The potential for antibiotics to affect the ecology and evolution of the human gut microbiota is well recognised and has wide-ranging implications for host health. Here, we review the findings of key studies that surveyed the human gut microbiota during antibiotic treatment. We find several broad patterns including the loss of diversity, disturbance of community composition, suppression of bacteria in the Actinobacteria phylum, amplification of bacteria in the Bacteroidetes phylum, and promotion of antibiotic resistance. Such changes to the microbiota were often, but not always, recovered following the end of treatment. However, many studies reported unique and/or contradictory results, which highlights our inability to meaningfully predict or explain the effects of antibiotic treatment on the human gut microbiome. This problem arises from variation between existing studies in three major categories: differences in dose, class, and combinations of antibiotic treatments used; differences in demographics, lifestyles, and locations of subjects; and differences in measurements, analyses, and reporting styles used by researchers. To overcome this, we suggest two integrated approaches: (i) a top-down approach focused on building predictive models through large sample sizes, deep metagenomic sequencing, and effective collaboration; and (ii) a bottom-up reductionist approach focused on testing hypotheses using model systems.

[Source](https://academic.oup.com/femsre/advance-article/doi/10.1093/femsre/fuab018/6212228)

10.1093/femsre/fuab018

---

## The Fecal and Serum Metabolomics of Giant Pandas Based on Untargeted Metabolomics.
 05.04.2021, _S Yang, Y Huang, C Li, L Jin, W Deng, S Zhao, D Wu, Y He, B Li, Z Yu, T Li, Z Zhang, X Pan, H Zhang, L Zou_


Little is comprehensively known or understood about giant panda fecal and serum metabolites, which could serve as important indicators of the physiological metabolism of giant pandas. Therefore, we determined the contents of fecal and serum metabolites of giant pandas based on an untargeted metabolome. Four hundred and 955 metabolites were detected in the feces and serum of giant panda, respectively. Glycerophospholipid and choline metabolism were the main metabolic pathways in feces and serum. A significant correlation between the gut microbiota and fecal metabolites was found (P &lt; 0.01). Fecal metabolites were not greatly affected by the age or gender of giant pandas, but serum metabolites were significantly affected by age and gender. The majority of different metabolites caused by age were higher in serum of younger giant pandas, including fatty acids, lipids, metabolites of bile acids, and intermediate products of vitamin D3. The majority of different metabolites caused by gender included fatty acids, phosphatidylcholine (PC), phosphatidylserine (PS), and phosphatidylethanolamine (PE). A separate feeding diet should be considered according to different ages and genders of giant panda. Therefore, our results could provide helpful suggestions to further protect captive giant pandas.

[Source](https://bioone.org/journals/zoological-science/volume-38/issue-2/zs200106/The-Fecal-and-Serum-Metabolomics-of-Giant-Pandas-Based-on/10.2108/zs200106.short)

10.2108/zs200106

---

## Water Activities of Acid Brine Lakes Approach the Limit for Life.
 05.04.2021, _KC Benison, WK O'Neill, D Blain, JE Hallsworth_


Water activity is an important characteristic for describing unusual waters and is a determinant of habitability for microorganisms. However, few empirical studies of water activity have been done for natural waters exhibiting an extreme chemistry. Here, we investigate water activity for acid brines from Western Australia and Chile with pH as low as 1.4, salinities as high as 32% total dissolved solids, and complex chemical compositions. These acid brines host diverse communities of extremophilic microorganisms, including archaea, bacteria, algae, and fungi, according to metagenomic analyses. For the most extreme brine, its water activity (0.714) was considerably lower than that of saturated (pure) NaCl brine. This study provides a thermodynamic insight into life within end-member natural waters that lie at, or possibly beyond, the very edge of habitable space on Earth.

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1462-2920.15499)

10.1089/ast.2020.2334

---

## Astrobiology of life on Earth

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1462-2920.15499)

---

## Impact of UV Radiation on the Raman Signal of Cystine: Implications for the Detection of S-rich Organics on Mars.
 11.03.2021, _V Megevand, JC Viennet, E Balan, M Gauthier, P Rosier, M Morand, Y Garino, M Guillaumet, S Pont, O Beyssac, S Bernard_


Traces of life may have been preserved in ancient martian rocks in the form of molecular fossils. Yet the surface of Mars is continuously exposed to intense UV radiation detrimental to the preservation of organics. Because the payload of the next rovers going to Mars to seek traces of life will comprise Raman spectroscopy tools, laboratory simulations that document the effect of UV radiation on the Raman signal of organics appear critically needed. The experiments conducted here evidence that UV radiation is directly responsible for the increase of disorder and for the creation of electronic defects and radicals within the molecular structure of S-rich organics such as cystine, enhancing the contribution of light diffusion processes to the Raman signal. The present results suggest that long exposure to UV radiation would ultimately be responsible for the total degradation of the Raman signal of cystine. Yet because the degradation induced by UV is not instantaneous, it should be possible to detect freshly excavated S-rich organics with the Raman instruments on board the rovers. Alternatively, given the very short lifetime of organic fluorescence (nanoseconds) compared to most mineral luminescence (micro- to milliseconds), exploiting fluorescence signals might allow the detection of S-rich organics on Mars. In any case, as illustrated here, we should not expect to detect pristine S-rich organic compounds on Mars, but rather by-products of their degradation.

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.15494)

10.1089/ast.2020.2340

---

## Mars’ surface is not universally biocidal

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.15494)

---

## Draft Genome Sequences and Antimicrobial Profiles of Three Staphylococcus epidermidis Strains from Neonatal Blood Samples.
 02.04.2021, _I Afeke, A Moustafa, M Hirose, M Becker, H Busch, A Kuenstner, A Faehnrich, AS Ablordey, C Haertel, KH Amega-Aho, MT Badr, J Rupp, S Ibrahim_


Data on molecular characterization of coagulase-negative staphylococci causing neonatal sepsis in low-income countries are highly limited. This report highlights the isolation of three Staphylococcus epidermidis non-genome assembly strains (NGASs) from blood samples from neonates with unknown transmission sources. Pathogenic factors and sources of transmission of these strains warrant further investigation.

[Source](https://mra.asm.org/content/10/13/e00170-21)

10.1128/MRA.00170-21

---

## CoCoNet: An Efficient Deep Learning Tool for Viral Metagenome Binning.
 06.04.2021, _CG Arisdakessian, O Nigro, G Steward, G Poisson, M Belcaid_


Metagenomic approaches hold the potential to characterize microbial communities and unravel the intricate link between the microbiome and biological processes. Assembly is one of the most critical steps in metagenomics experiments. It consists of transforming overlapping DNA sequencing reads into sufficiently accurate representations of the community's genomes. This process is computationally difficult and commonly results in genomes fragmented across many contigs. Computational binning methods are used to mitigate fragmentation by partitioning contigs based on their sequence composition, abundance, or chromosome organization into bins representing the community's genomes. Existing binning methods have been principally tuned for bacterial genomes and do not perform favorably on viral metagenomes.

[Source](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab213/6211156)

10.1093/bioinformatics/btab213

---

## Rank normalization empowers a t-test for microbiome differential abundance analysis while controlling for false discoveries.
 06.04.2021, _ML Davis, Y Huang, K Wang_


A major task in the analysis of microbiome data is to identify microbes associated with differing biological conditions. Before conducting analysis, raw data must first be adjusted so that counts from different samples are comparable. A typical approach is to estimate normalization factors by which all counts in a sample are multiplied or divided. However, the inherent variation associated with estimation of normalization factors are often not accounted for in subsequent analysis, leading to a loss of precision. Rank normalization is a nonparametric alternative to the estimation of normalization factors in which each count for a microbial feature is replaced by its intrasample rank. Although rank normalization has been successfully applied to microarray analysis in the past, it has yet to be explored for microbiome data, which is characterized by high frequencies of 0s, strongly correlated features and compositionality. We propose to use rank normalization as an alternative to the estimation of normalization factors and examine its performance when paired with a two-sample t-test. On a rigorous 3rd-party benchmarking simulation, it is shown to offer strong control over the false discovery rate, and at sample sizes greater than 50 per treatment group, to offer an improvement in performance over commonly used normalization factors paired with t-tests, Wilcoxon rank-sum tests and methodologies implemented by R packages. On two real datasets, it yielded valid and reproducible results that were strongly in agreement with the original findings and the existing literature, further demonstrating its robustness and future potential. Availability: The data underlying this article are available online along with R code and supplementary materials at https://github.com/matthewlouisdavisBioStat/Rank-Normalization-Empowers-a-T-Test.

[Source](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbab059/6210069)

10.1093/bib/bbab059

---

## Discovering multiple types of DNA methylation from bacteria and microbiome using nanopore sequencing.
 06.04.2021, _A Tourancheau, EA Mead, XS Zhang, G Fang_


Bacterial DNA methylation occurs at diverse sequence contexts and plays important functional roles in cellular defense and gene regulation. Existing methods for detecting DNA modification from nanopore sequencing data do not effectively support de novo study of unknown bacterial methylomes. In this work, we observed that a nanopore sequencing signal displays complex heterogeneity across methylation events of the same type. To enable nanopore sequencing for broadly applicable methylation discovery, we generated a training dataset from an assortment of bacterial species and developed a method, named nanodisco ( https://github.com/fanglab/nanodisco ), that couples the identification and fine mapping of the three forms of methylation into a multi-label classification framework. We applied it to individual bacteria and the mouse gut microbiome for reliable methylation discovery. In addition, we demonstrated the use of DNA methylation for binning metagenomic contigs, associating mobile genetic elements with their host genomes and identifying misassembled metagenomic contigs.

[Source](https://www.nature.com/articles/s41592-021-01109-3)

10.1038/s41592-021-01109-3

---

