## The evolution of coexistence from competition in experimental co-cultures of Escherichia coli and Saccharomyces cerevisiae.
 23.10.2020, _JN Barber, AL Sezmis, LC Woods, TD Anderson, JM Voss, MJ McDonald_


Microbial communities are comprised of many species that coexist on small spatial scales. This is difficult to explain because many interspecies interactions are competitive, and ecological theory predicts that one species will drive the extinction of another species that competes for the same resource. Conversely, evolutionary theory proposes that natural selection can lead to coexistence by driving competing species to use non-overlapping resources. However, evolutionary escape from extinction may be slow compared to the rate of competitive exclusion. Here, we use experimental co-cultures of Escherichia coli and Saccharomyces cerevisiae to study the evolution of coexistence in species that compete for resources. We find that while E. coli usually outcompetes S. cerevisiae in co-culture, a few populations evolved stable coexistence after ~1000 generations of coevolution. We sequenced S. cerevisiae and E. coli populations, identified multi-hit genes, and engineered alleles from these genes into several genetic backgrounds, finding that some mutations modified interactions between E. coli and S. cerevisiae. Together, our data demonstrate that coexistence can evolve, de novo, from intense competition between two species with no history of coevolution.

[Source](https://www.nature.com/articles/s41396-020-00810-z)

10.1038/s41396-020-00810-z

---

## Quantifying the effects of antibiotic treatment on the extracellular polymer network of antimicrobial resistant and sensitive biofilms using multiple particle tracking.
 17.02.2021, _LC Powell, M Abdulkarim, J Stokniene, QE Yang, TR Walsh, KE Hill, M Gumbleton, DW Thomas_


Novel therapeutics designed to target the polymeric matrix of biofilms requires innovative techniques to accurately assess their efficacy. Here, multiple particle tracking (MPT) was developed to characterize the physical and mechanical properties of antimicrobial resistant (AMR) bacterial biofilms and to quantify the effects of antibiotic treatment. Studies employed nanoparticles (NPs) of varying charge and size (40-500 nm) in Pseudomonas aeruginosa PAO1 and methicillin-resistant Staphylococcus aureus (MRSA) biofilms and also in polymyxin B (PMB) treated Escherichia coli biofilms of PMB-sensitive (PMBSens) IR57 and PMB-resistant (PMBR) PN47 strains. NP size-dependent and strain-related differences in the diffusion coefficient values of biofilms were evident between PAO1 and MRSA. Dose-dependent treatment effects induced by PMB in PMBSens E. coli biofilms included increases in diffusion and creep compliance (P &lt; 0.05), not evident in PMB treatment of PMBR E. coli biofilms. Our results highlight the ability of MPT to quantify the diffusion and mechanical effects of antibiotic therapies within the AMR biofilm matrix, offering a valuable tool for the pre-clinical screening of anti-biofilm therapies.

[Source](https://www.nature.com/articles/s41522-020-00172-6)

10.1038/s41522-020-00172-6

---

## Biofilms harbour Clostridioides difficile, serving as a reservoir for recurrent infection.
 17.02.2021, _C Normington, IB Moura, JA Bryant, DJ Ewin, EV Clark, MJ Kettle, HC Harris, W Spittal, G Davis, MR Henn, CB Ford, MH Wilcox, AM Buckley_


C. difficile infection (CDI) is a worldwide healthcare problem with ~30% of cases failing primary therapy, placing a burden on healthcare systems and increasing patient morbidity. We have little understanding of why these therapies fail. Here, we use a clinically validated in vitro gut model to assess the contribution of biofilms towards recurrent disease and to investigate biofilm microbiota-C. difficile interactions. Initial experiments show that C. difficile cells became associated with the colonic biofilm microbiota and are not depleted by vancomycin or faecal microbiota transplant therapies. We observe that transferring biofilm encased C. difficile cells into a C. difficile naïve but CDI susceptible model induces CDI. Members of the biofilm community can impact C. difficile biofilm formation by acting either antagonistically or synergistically. We highlight the importance of biofilms as a reservoir for C. difficile, which can be a cause for recurrent infections.

[Source](https://www.nature.com/articles/s41522-021-00184-w)

10.1038/s41522-021-00184-w

---

## The contribution of gut bacterial metabolites in the human immune signaling pathway of non-communicable diseases.
 16.02.2021, _F Hosseinkhani, A Heinken, I Thiele, PW Lindenburg, AC Harms, T Hankemeier_


The interaction disorder between gut microbiota and its host has been documented in different non-communicable diseases (NCDs) such as metabolic syndrome, neurodegenerative disease, and autoimmune disease. The majority of these altered interactions arise through metabolic cross-talk between gut microbiota and host immune system, inducing a low-grade chronic inflammation that characterizes all NCDs. In this review, we discuss the contribution of bacterial metabolites to immune signaling pathways involved in NCDs. We then review recent advances that aid to rationally design microbial therapeutics. A deeper understanding of these intersections between host and gut microbiota metabolism using metabolomics-based system biology platform promises to reveal the fundamental mechanisms that drive metabolic predispositions to disease and suggest new avenues to use microbial therapeutic opportunities for NCDs treatment and prevention. Abbreviations: NCDs: non-communicable disease, IBD: inflammatory bowel disease, IL: interleukin, T2D: type 2 diabetes, SCFAs: short-chain fatty acids, HDAC: histone deacetylases, GPCR: G-protein coupled receptors, 5-HT: 5-hydroxytryptamine receptor signaling, DCs: dendritic cells, IECs: intestinal epithelial cells, T-reg: T regulatory cell, NF-κB: nuclear factor κB, TNF-α: tumor necrosis factor alpha, Th: T helper cell, CNS: central nervous system, ECs: enterochromaffin cells, NSAIDs: non-steroidal anti-inflammatory drugs, AhR: aryl hydrocarbon receptor, IDO: indoleamine 2,3-dioxygenase, QUIN: quinolinic acid, PC: phosphatidylcholine, TMA: trimethylamine, TMAO: trimethylamine N-oxide, CVD: cardiovascular disease, NASH: nonalcoholic steatohepatitis, BAs: bile acids, FXR: farnesoid X receptor, CDCA: chenodeoxycholic acid, DCA: deoxycholic acid, LCA: lithocholic acid, UDCA: ursodeoxycholic acid, CB: cannabinoid receptor, COBRA: constraint-based reconstruction and analysis.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2021.1882927)

10.1080/19490976.2021.1882927

---

## The effect of acute moderate-intensity exercise on the serum and fecal metabolomes and the gut microbiota of cross-country endurance athletes.
 15.02.2021, _M Tabone, C Bressa, JA García-Merino, D Moreno-Pérez, EC Van, FA Castelli, F Fenaille, M Larrosa_


Physical exercise can produce changes in the microbiota, conferring health benefits through mechanisms that are not fully understood. We sought to determine the changes driven by exercise on the gut microbiota and on the serum and fecal metabolome using 16S rRNA gene analysis and untargeted metabolomics. A total of 85 serum and 12 fecal metabolites and six bacterial taxa (Romboutsia, Escherichia coli TOP498, Ruminococcaceae UCG-005, Blautia, Ruminiclostridium 9 and Clostridium phoceensis) were modified following a controlled acute exercise session. Among the bacterial taxa, Ruminiclostridium 9 was the most influenced by fecal and serum metabolites, as revealed by linear multivariate regression analysis. Exercise significantly increased the fecal ammonia content. Functional analysis revealed that alanine, aspartate and glutamate metabolism and the arginine and aminoacyl-tRNA biosynthesis pathways were the most relevant modified pathways in serum, whereas the phenylalanine, tyrosine and tryptophan biosynthesis pathway was the most relevant pathway modified in feces. Correlation analysis between fecal and serum metabolites suggested an exchange of metabolites between both compartments. Thus, the performance of a single exercise bout in cross-country non-professional athletes produces significant changes in the microbiota and in the serum and fecal metabolome, which may have health implications.

[Source](https://www.nature.com/articles/s41598-021-82947-1)

10.1038/s41598-021-82947-1

---

## Candida albicans promotes tooth decay by inducing oral microbial dysbiosis.
 07.11.2020, _Q Du, B Ren, J He, X Peng, Q Guo, L Zheng, J Li, H Dai, V Chen, L Zhang, X Zhou, X Xu_


Candida albicans has been detected in root carious lesions. The current study aimed to explore the action of this fungal species on the microbial ecology and the pathogenesis of root caries. Here, by analyzing C. albicans in supragingival dental plaque collected from root carious lesions and sound root surfaces of root-caries subjects as well as caries-free individuals, we observed significantly increased colonization of C. albicans in root carious lesions. Further in vitro and animal studies showed that C. albicans colonization increased the cariogenicity of oral biofilm by altering its microbial ecology, leading to a polymicrobial biofilm with enhanced acidogenicity, and consequently exacerbated tooth demineralization and carious lesion severity. More importantly, we demonstrated that the cariogenicity-promoting activity of C. albicans was dependent on PHR2. Deletion of PHR2 restored microbial equilibrium and led to a less cariogenic biofilm as demonstrated by in vitro artificial caries model or in vivo root-caries rat model. Our data indicate the critical role of C. albicans infection in the occurrence of root caries. PHR2 is the major factor that determines the ecological impact and caries-promoting activity of C. albicans in a mixed microbial consortium.

[Source](https://www.nature.com/articles/s41396-020-00823-8)

10.1038/s41396-020-00823-8

---

## Characterization of oral virome and microbiome revealed distinctive microbiome disruptions in paediatric patients with hand, foot and mouth disease.
 22.02.2021, _SX Ho, N Min, EPY Wong, CY Chong, JJH Chu_


While the underlying determinants are unclear, hand, foot and mouth disease (HFMD) presents a wide spectrum of clinical manifestations with varying severity in different individuals. Recently, many studies identified the human microbiome as a critical factor in the pathogenesis of various diseases. Therefore, we here investigated the ecological dynamics of the oral microbiome changes during the HFMD infection. After targeted enrichment of all known vertebrate viruses, the virome profiles of symptomatic and asymptomatic HFMD patients were examined and revealed to be significantly altered from those of healthy individuals, with nine discriminative viruses detected. Further characterization of the prokaryotic microbiome revealed an elevated level of Streptococcus sp. as the most important signature of the symptomatic HFMD cohort, positively correlating to the level of enterovirus A RNA. In addition, we found that while coxsackievirus A5 is detected in saliva RNA of all asymptomatic cases, coxsackievirus A6 dominates the majority of the symptomatic cohort.

[Source](https://www.nature.com/articles/s41522-021-00190-y)

10.1038/s41522-021-00190-y

---

## Characterization of the human skin resistome and identification of two microbiota cutotypes.
 23.02.2021, _Z Li, J Xia, L Jiang, Y Tan, Y An, X Zhu, J Ruan, Z Chen, H Zhen, Y Ma, Z Jie, L Xiao, H Yang, J Wang, K Kristiansen, X Xu, L Jin, C Nie, J Krutmann, X Liu, J Wang_


The human skin microbiota is considered to be essential for skin homeostasis and barrier function. Comprehensive analyses of its function would substantially benefit from a catalog of reference genes derived from metagenomic sequencing. The existing catalog for the human skin microbiome is based on samples from limited individuals from a single cohort on reference genomes, which limits the coverage of global skin microbiome diversity.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00995-7)

10.1186/s40168-020-00995-7

---

## A gene co-association network regulating gut microbial communities in a Duroc pig population

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00994-8)

---

## Modulating gut microbiota in a mouse model of Graves’ orbitopathy and its impact on induced disease

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00952-4)

---

## Multi-compartment metabolomics and metagenomics reveal major hepatic and intestinal disturbances in cancer cachectic mice

[Source](https://onlinelibrary.wiley.com/doi/pdf/10.1002/jcsm.12684)

---

## Expanded catalog of microbial genes and metagenome-assembled genomes from the pig gut microbiome

[Source](https://www.nature.com/articles/s41467-021-21295-0)

---

## Mining zebrafish microbiota reveals key community-level resistance against fish pathogen infection

[Source](https://www.nature.com/articles/s41396-020-00807-8)

---

## From exploration to remediation: A microbial perspective for innovation in mining

[Source](https://www.sciencedirect.com/science/article/pii/S0012825221000623)

---

## A Combined Metagenomics and Metatranscriptomics Approach to Unravel Costa Rican Cocoa Box Fermentation Processes Reveals Yet Unreported Microbial Species and Functionalities

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2021.641185/full)

---

## Mechanistic strategies of microbial communities regulating lignocellulose deconstruction in a UK salt marsh

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00964-0)

---

## Taxonomic and functional analyses of intact microbial communities thriving in extreme, astrobiology-relevant, anoxic sites

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00989-5)

---

## Genomic evidence for sulfur intermediates as new biogeochemical hubs in a model aquatic microbial ecosystem

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-00999-x)

---

## The Seagrass Holobiont: What We Know and What We Still Need to Disclose for Its Possible Use as an Ecological Indicator

[Source](https://www.mdpi.com/2073-4441/13/4/406/htm)

---

## Spatiotemporal variations and relationships of phosphorus, phosphomonoesterases, and bacterial communities in sediments from two Chilean rivers

[Source](https://www.sciencedirect.com/science/article/pii/S0048969721008494)

---

## Resource partitioning of phytoplankton metabolites that support bacterial heterotrophy

[Source](https://www.nature.com/articles/s41396-020-00811-y)

---

## Next-generation diagnostics: virus capture facilitates a sensitive viral diagnosis for epizootic and zoonotic pathogens including SARS-CoV-2

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00973-z)

---

## Preliminary Assessment of Viral Metagenome from Cancer Tissue and Blood from Patients with Lung Adenocarcinoma.
 17.02.2021, _HZ Cai, H Zhang, J Yang, J Zeng, H Wang_


Here, using viral metagenomic method, we investigated the composition of virome in blood and cancer tissue samples which were collected from 25 patients with lung adenocarcinoma. Results indicated virus sequences showing similarity to human pegivirus (HPgV), anellovirus, human endogenous retrovirus (HERV) and polyomavirus were recovered from this cohort. Three different complete genomes of HPgV were acquired from the blood samples and one complete genome of polyomavirus was determined from the cancer tissue sample. Phylogenetic analysis indicated that the 3 HPgV strains all belonged to genotype 3 and the polyomavirus showed the highest sequence identity (99.73%) to Trichodysplasia spinulosa-associated polyomavirus. PCR screening results indicated the 3 HPgVs were present in 5 out of the 25 blood samples and the polyomavirus only existed in a cancer tissue sample pool. Whether infections with viruses have association with lung cancer needs further study with larger size of sampling. This article is protected by copyright. All rights reserved.

[Source](https://onlinelibrary.wiley.com/doi/abs/10.1002/jmv.26887)

10.1002/jmv.26887

---

## PathoFact: a pipeline for the prediction of virulence factors and antimicrobial resistance genes in metagenomic data.
 23.02.2021, _L de Nies, S Lopes, SB Busi, V Galata, A Heintz-Buschart, CC Laczny, P May, P Wilmes_


Pathogenic microorganisms cause disease by invading, colonizing, and damaging their host. Virulence factors including bacterial toxins contribute to pathogenicity. Additionally, antimicrobial resistance genes allow pathogens to evade otherwise curative treatments. To understand causal relationships between microbiome compositions, functioning, and disease, it is essential to identify virulence factors and antimicrobial resistance genes in situ. At present, there is a clear lack of computational approaches to simultaneously identify these factors in metagenomic datasets.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00993-9)

10.1186/s40168-020-00993-9

---

## CANTARE: finding and visualizing network-based multi-omic predictive models.
 23.02.2021, _JC Siebert, M Saint-Cyr, SJ Borengasser, BD Wagner, CA Lozupone, C Görg_


One goal of multi-omic studies is to identify interpretable predictive models for outcomes of interest, with analytes drawn from multiple omes. Such findings could support refined biological insight and hypothesis generation. However, standard analytical approaches are not designed to be "ome aware." Thus, some researchers analyze data from one ome at a time, and then combine predictions across omes. Others resort to correlation studies, cataloging pairwise relationships, but lacking an obvious approach for cohesive and interpretable summaries of these catalogs.

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04016-8)

10.1186/s12859-021-04016-8

---

## kLDM: Inferring Multiple Metagenomic Association Networks based on the Variation of Environmental Factors.
 19.02.2021, _Y Yang, X Wang, K Xie, C Zhu, N Chen, T Chen_


Identification of significant biological relationships or patterns is central to many metagenomic studies. Methods that estimate association networks have been proposed for this purpose; however, they assume that associations are static, neglecting the fact that relationships in a microbial ecosystem may vary with changes in environmental factors, which can result in inaccurate estimations. Therefore, this paper proposes a computational model, called the k-Lognormal-Dirichlet-Multinomial (kLDM) model, which estimates multiple association networks that correspond to specific environmental conditions, and simultaneously infers microbe-microbe and environmental factor-microbe associations for each network. The effectiveness of the kLDM model was demonstrated on synthetic data, a colorectal cancer (CRC) dataset, the Tara Oceans dataset, and the American Gut Project dataset. The results reveal that the widely-used Spearman's rank correlation coefficient (SCC) method performed much worse than the other methods, indicating the importance of separating samples by environmental conditions. Cancer fecal samples were then compared with cancer-free samples, and the estimation achieved by kLDM exhibited fewer associations among microbes but stronger associations between specific bacteria, especially five CRC-associated operational taxonomic units (OTUs), indicating gut microbe translocation in cancer patients. Some environmental factor-dependent associations were then found within a marine eukaryotic community. Finally, the gut microbial heterogeneity of irritable bowel disease (IBD) patients was detected. These results demonstrate that kLDM can elucidate the complex associations within microbial ecosystems. The kLDM program, R, and Python scripts, together with all experimental datasets, are accessible at https://github.com/tinglab/kLDM.git.

[Source](https://www.sciencedirect.com/science/article/pii/S1672022921000206)

10.1016/j.gpb.2020.06.015

---

## Cross-species RNA-seq for deciphering host-microbe interactions.
 18.02.2021, _AJ Westermann, J Vogel_


The human body is constantly exposed to microorganisms, which entails manifold interactions between human cells and diverse commensal or pathogenic bacteria. The cellular states of the interacting cells are decisive for the outcome of these encounters such as whether bacterial virulence programmes and host defence or tolerance mechanisms are induced. This Review summarizes how next-generation RNA sequencing (RNA-seq) has become a primary technology to study host-microbe interactions with high resolution, improving our understanding of the physiological consequences and the mechanisms at play. We illustrate how the discriminatory power and sensitivity of RNA-seq helps to dissect increasingly complex cellular interactions in time and space down to the single-cell level. We also outline how future transcriptomics may answer currently open questions in host-microbe interactions and inform treatment schemes for microbial disorders.

[Source](https://www.nature.com/articles/s41576-021-00326-y)

10.1038/s41576-021-00326-y

---

## A community resource for paired genomic and metabolomic data mining.
 24.02.2021, _MA Schorn, S Verhoeven, L Ridder, F Huber, DD Acharya, AA Aksenov, G Aleti, JA Moghaddam, AT Aron, S Aziz, A Bauermeister, KD Bauman, M Baunach, C Beemelmanns, JM Beman, MV Berlanga-Clavero, AA Blacutt, HB Bode, A Boullie, A Brejnrod, TS Bugni, A Calteau, L Cao, VJ Carrión, R Castelo-Branco, S Chanana, AB Chase, MG Chevrette, LV Costa-Lotufo, JM Crawford, CR Currie, B Cuypers, T Dang, T de Rond, AM Demko, E Dittmann, C Du, C Drozd, JC Dujardin, RJ Dutton, A Edlund, DP Fewer, N Garg, JM Gauglitz, EC Gentry, L Gerwick, E Glukhov, H Gross, M Gugger, DG Guillén Matus, EJN Helfrich, BF Hempel, JS Hur, M Iorio, PR Jensen, KB Kang, L Kaysser, NL Kelleher, CS Kim, KH Kim, I Koester, GM König, T Leao, SR Lee, YY Lee, X Li, JC Little, KN Maloney, D Männle, C Martin H, AC McAvoy, WW Metcalf, H Mohimani, C Molina-Santiago, BS Moore, MW Mullowney, M Muskat, LF Nothias, EC O'Neill, EI Parkinson, D Petras, J Piel, EC Pierce, K Pires, R Reher, D Romero, MC Roper, M Rust, H Saad, C Saenz, LM Sanchez, SJ Sørensen, M Sosio, RD Süssmuth, D Sweeney, K Tahlan, RJ Thomson, NJ Tobias, AE Trindade-Silva, GP van Wezel, M Wang, KC Weldon, F Zhang, N Ziemert, KR Duncan, M Crüsemann, S Rogers, PC Dorrestein, MH Medema, JJJ van der Hooft_



[Source](https://www.biorxiv.org/content/10.1101/2020.03.25.007666v2.abstract)

10.1038/s41589-020-00724-z

---

## Cross-sectional analysis of the microbiota of the human gut and their direct environment (exposome) in a household cohort in northern Vietnam

[Source](https://www.biorxiv.org/content/10.1101/2021.02.23.432416v1.abstract)

---

## HumGut: A comprehensive Human Gut prokaryotic genomes collection filtered by metagenome data

[Source](https://www.biorxiv.org/content/10.1101/2020.03.25.007666v2.abstract)

---

## Comparative analysis of 16S rRNA gene and metagenome sequencing in pediatric gut microbiomes

[Source](https://www.biorxiv.org/content/10.1101/2021.02.20.432118v1.abstract)

---

## Unifying the known and unknown microbial coding sequence space

[Source](https://www.biorxiv.org/content/10.1101/2020.06.30.180448v5.abstract)

---

