## Capturing the antibiotic resistome of preterm infants reveals new benefits of probiotic supplementation.
 29.08.2022, _AK Guitor, EI Yousuf, AR Raphenya, EK Hutton, KM Morrison, AG McArthur, GD Wright, JC Stearns_


_Anti-Bacterial Agents, Bacteria, Dietary Supplements, Female, Humans, Infant, Infant, Newborn, Infant, Premature, Premature Birth, Probiotics_
Probiotic use in preterm infants can mitigate the impact of antibiotic exposure and reduce rates of certain illnesses; however, the benefit on the gut resistome, the collection of antibiotic resistance genes, requires further investigation. We hypothesized that probiotic supplementation of early preterm infants (born &lt; 32-week gestation) while in hospital reduces the prevalence of antibiotic resistance genes associated with pathogenic bacteria in the gut. We used a targeted capture approach to compare the resistome from stool samples collected at the term corrected age of 40 weeks for two groups of preterm infants (those that routinely received a multi-strain probiotic during hospitalization and those that did not) with samples from full-term infants at 10 days of age to identify if preterm birth or probiotic supplementation impacted the resistome. We also compared the two groups of preterm infants up to 5 months of age to identify persistent antibiotic resistance genes.

[Source](https://doi.org/10.1186/s40168-022-01327-7)

10.1186/s40168-022-01327-7

---

## Human gut microbiota after bariatric surgery alters intestinal morphology and glucose absorption in mice independently of obesity.
 25.08.2022, _FF Anhê, S Zlitni, SY Zhang, BS Choi, CY Chen, KP Foley, NG Barra, MG Surette, L Biertho, D Richard, A Tchernof, TKT Lam, A Marette, J Schertzer_


Bariatric surgery is an effective treatment for type 2 diabetes (T2D) that changes gut microbial composition. We determined whether the gut microbiota in humans after restrictive or malabsorptive bariatric surgery was sufficient to lower blood glucose.

[Source](https://doi.org/10.1136/gutjnl-2022-328185)

10.1136/gutjnl-2022-328185

---

## Temporal dynamics of oropharyngeal microbiome among SARS-CoV-2 patients reveals continued dysbiosis even after Viral Clearance.
 28.08.2022, _SK Paine, UK Rout, C Bhattacharyya, D Parai, M Alam, RR Nanda, D Tripathi, P Choudhury, CN Kundu, S Pati, D Bhattacharya, A Basu_


_Anti-Bacterial Agents, Bacteria, COVID-19, Coinfection, Dysbiosis, Humans, Microbiota, Oropharynx, Pneumonia, Bacterial, SARS-CoV-2, Sepsis_
The severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) pandemic has posed multiple challenges to global public health. Clinical features and sequela of SARS-CoV-2 infection include long-term and short-term complications often clinically indistinguishable from bacterial sepsis and acute lung infection. Post-hoc studies of previous SARS outbreaks postulate secondary bacterial infections with microbial dysbiosis. Oral microbial dysbiosis, particularly the altered proportion of Firmicutes and Proteobacteria, observed in other respiratory virus infection, like influenza, has shown to be associated with increased morbidity and mortality. Oropharynx and lung share similar kinds of bacterial species. We hypothesized that alteration in the Human Oropharyngeal Microbiome in SARS-CoV-2 patients can be a clinical indicator of bacterial infection related complications. We made a longitudinal comparison of oropharyngeal microbiome of 20 SARS-CoV-2 patients over a period of 30 days; at three time points, with a 15 days interval; contrasting them with a matched group of 10 healthy controls. Present observation indicates that posterior segment of the oropharyngeal microbiome is a key reservoir for bacteria causing pneumonia and chronic lung infection on SARS-CoV-2 infection. Oropharyngeal microbiome is indeed altered and its α-diversity decreases, indicating reduced stability, in all SARS-CoV-2 positive individuals right at Day-1; i.e. within ~24 h of post clinical diagnosis. The dysbiosis persists long-term (30 days) irrespective of viral clearance and/or administration of antibiotics. There is a severe depletion of commensal bacteria phyla like Firmicutes among the patients and that depletion is compensated by higher proportion of bacteria associated with sepsis and severe lung infection from phyla Proteobacteria. We also found elevated proportions of certain genus that have previously been shown to be causal for lung pneumonia in studies of model organisms and human autopsies' including Stenotrophomonas, Acenetobactor, Enterobactor, Klebsiella and Chryseobacterium that were to be elevated among the cases. We also show that responses to the antibiotics (Azithromycin and Doxycycline) are not uniform for all individuals.

[Source](https://doi.org/10.1038/s41522-022-00330-y)

10.1038/s41522-022-00330-y

---

## Comparison of Metagenomic Sequencing and the NanoString nCounter Analysis System for the Characterization of Bacterial and Viral Communities in Vaginal Samples.
 24.08.2022, _TN Wylie, J Schrimpf, H Gula, BN Herter, KM Wylie_


DNA sequencing assays have been used to characterize the vaginal microbiome and to identify associations with clinical outcomes. The purpose of this study was to evaluate the utility of the NanoString nCounter platform, a more efficient assay compared to sequencing, for the characterization of vaginal microbial communities. A panel of NanoString nCounter probes was designed to detect common vaginal bacteria and viruses with relevance to reproductive health. A defined synthetic community of microbes and 43 clinical samples were interrogated with NanoString nCounter assays and compared to known compositions or metagenomic shotgun sequencing (MSS) results. The NanoString nCounter platform and MSS were able to distinguish closely related microbes. In clinical samples, the relative abundance of bacterial species was similar between the two assays. The assays sometimes disagreed when targets were present at low abundance. More viruses were detected by MSS than by nCounter. However, the nCounter assays are able to provide results in about 30 h with minimal hands-on time, whereas MSS requires at least 138 to 178 h with extensive hands-on time. The reagent cost for the two assays was similar, but the overall cost of the nCounter was lower due to the minimal hands-on time. MSS can be used to inform the design of a targeted multiplex panel for the assessment of vaginal microbial communities, thereby allowing for more cost-effective and rapid screening of patient samples for research studies. The sensitivity for low abundance microbes could be improved, possibly by adding additional target amplification cycles before nCounter assessment. This approach has potential as an assay with both research and clinical applications. IMPORTANCE Metagenomic shotgun sequencing can inform the design of a targeted multiplex panel by which the NanoString nCounter platform can assess vaginal microbial communities, thereby allowing for more cost-effective and rapid screening of patient samples.

[Source](https://doi.org/10.1128/msphere.00197-22)

10.1128/msphere.00197-22

---

## Oral Microbiota from Periodontitis Promote Oral Squamous Cell Carcinoma Development via γδ T Cell Activation.
 24.08.2022, _W Wei, J Li, X Shen, J Lyu, C Yan, B Tang, W Ma, H Xie, L Zhao, L Cheng, Y Deng, Y Li_


Oral squamous cell carcinoma (OSCC) is a fatal disease, and periodontitis is associated with OSCC development. However, the pathogenesis in the context of OSCC with periodontitis has not been fully understood. Here, we demonstrated that periodontitis promoted OSCC development, accompanied by alterations in the oral bacterial community and the tumor immune microenvironment. The oral microbiota from periodontitis maintained the dominant position throughout the whole process of OSCC with periodontitis, of which Porphyromonas was the most abundant genus. The oral microbiota from periodontitis could activate interleukin-17-positive (IL-17+) γδ T cells directly. The activated γδ T cells were necessary for the IL-17/signal transducer and activator of transcription 3 (STAT3) pathway and promoted M2-tumor-associated macrophage (TAM) infiltration in OSCC proliferation. Our data provide insight into the carcinogenesis of OSCC with periodontitis by outlining the tumor-associated immune response shaped by the oral microbiota from periodontitis. Thus, oral commensal bacteria and IL-17+ γδ T cells might be potential targets for monitoring and treating OSCC. IMPORTANCE The work reveals the role of the oral microbiota from periodontitis in carcinogenesis. Furthermore, our study provides insight into the pathogenesis of OSCC with periodontitis by outlining the tumor-associated immune response shaped by the oral microbiota from periodontitis, which might identify new research and intervention targets for OSCC with periodontitis.

[Source](https://doi.org/10.1128/msystems.00469-22)

10.1128/msystems.00469-22

---

## Gut Microbiota and Coronary Plaque Characteristics.
 24.08.2022, _A Nakajima, S Mitomo, H Yuki, M Araki, LM Seegers, I McNulty, H Lee, D Kuter, M Ishibashi, K Kobayashi, J Dijkstra, H Onishi, H Yabushita, S Matsuoka, H Kawamoto, Y Watanabe, K Tanaka, S Chou, T Naganuma, M Okutsu, S Tahara, N Kurita, S Nakamura, S Das, S Nakamura, IK Jang_


Background The relationship between gut microbiota and in vivo coronary plaque characteristics has not been reported. This study was conducted to investigate the relationship between gut microbiota and coronary plaque characteristics in patients with coronary artery disease. Methods and Results Patients who underwent both optical coherence tomography and intravascular ultrasound imaging and provided stool and blood specimens were included. The composition of gut microbiota was evaluated using 16S rRNA sequencing. A total of 55 patients were included. At the genus level, 2 bacteria were associated with the presence of thin-cap fibroatheroma, and 9 bacteria were associated with smaller fibrous cap thickness. Among them, some bacteria had significant associations with inflammatory/prothrombotic biomarkers. Dysgonomonas had a positive correlation with interleukin-6, Paraprevotella had a positive correlation with fibrinogen and negative correlation with high-density lipoprotein cholesterol, Succinatimonas had positive correlations with fibrinogen and homocysteine, and Bacillus had positive correlations with fibrinogen and high-sensitivity C-reactive protein. In addition, Paraprevotella, Succinatimonas, and Bacillus were also associated with greater plaque volume. Ten bacteria were associated with larger fibrous cap thickness. Some were associated with protective biomarker changes; Anaerostipes had negative correlations with trimethylamine N-oxide, tumor necrosis factor α, and interleukin-6, and Dielma had negative correlations with trimethylamine N-oxide, white blood cells, plasminogen activator inhibitor-1, and homocysteine, and a positive correlation with high-density lipoprotein cholesterol. Conclusions Bacteria that were associated with vulnerable coronary plaque phenotype and greater plaque burden were identified. These bacteria were also associated with elevated inflammatory or prothrombotic biomarkers. Registration URL: https://www.umin.ac.jp/ctr/; Unique identifier: UMIN000041692.

[Source](https://doi.org/10.1161/JAHA.122.026036)

10.1161/JAHA.122.026036

---

## Toward FAIR Representations of Microbial Interactions.
 25.08.2022, _AR Pacheco, C Pauvert, D Kishore, D Segrè_


Despite an ever-growing number of data sets that catalog and characterize interactions between microbes in different environments and conditions, many of these data are neither easily accessible nor intercompatible. These limitations present a major challenge to microbiome research by hindering the streamlined drawing of inferences across studies. Here, we propose guiding principles to make microbial interaction data more findable, accessible, interoperable, and reusable (FAIR). We outline specific use cases for interaction data that span the diverse space of microbiome research, and discuss the untapped potential for new insights that can be fulfilled through broader integration of microbial interaction data. These include, among others, the design of intercompatible synthetic communities for environmental, industrial, or medical applications, and the inference of novel interactions from disparate studies. Lastly, we envision potential trajectories for the deployment of FAIR microbial interaction data based on existing resources, reporting standards, and current momentum within the community.

[Source](https://doi.org/10.1128/msystems.00659-22)

10.1128/msystems.00659-22

---

## Gut microbiome-mediated mechanisms for reducing cholesterol levels: implications for ameliorating cardiovascular disease.
 25.08.2022, _B Jia, Y Zou, X Han, JW Bae, CO Jeon_


Cardiovascular disease (CVD) is a health problem worldwide, and elevated cholesterol levels are a key risk factor for the disease. Dysbiotic gut microbiota has been shown to be associated with CVD development. However, the beneficial effects of healthy microbiota in decreasing cholesterol levels have not been summarized. Herein, we begin by discussing the potential mechanisms by which the gut microbiota reduces cholesterol levels. We further sketch the application of probiotics from the genera Lactobacillus and Bifidobacterium in reducing cholesterol levels in clinical studies. Finally, we present the cholesterol-lowering function of beneficial commensal microbes, such as Akkermansia and Bacteroides spp., as these microbes have potential to be the next-generation probiotics (NGPs). The information reviewed in this paper will help people to understand how the gut microbiome might alter cholesterol metabolism and enable the development of NGPs to prevent and treat CVD.

[Source](https://doi.org/10.1016/j.tim.2022.08.003)

10.1016/j.tim.2022.08.003

---

## Type 2 diabetes, gut microbiome, and systems biology: A novel perspective for a new era.
 26.08.2022, _YE Martínez-López, DA Esquivel-Hernández, JP Sánchez-Castañeda, D Neri-Rosario, R Guardado-Mendoza, O Resendis-Antonio_


_Diabetes Mellitus, Type 2, Diet, Gastrointestinal Microbiome, Humans, Microbiota, Systems Biology_
The association between the physio-pathological variables of type 2 diabetes (T2D) and gut microbiota composition suggests a new avenue to track the disease and improve the outcomes of pharmacological and non-pharmacological treatments. This enterprise requires new strategies to elucidate the metabolic disturbances occurring in the gut microbiome as the disease progresses. To this end, physiological knowledge and systems biology pave the way for characterizing microbiota and identifying strategies in a move toward healthy compositions. Here, we dissect the recent associations between gut microbiota and T2D. In addition, we discuss recent advances in how drugs, diet, and exercise modulate the microbiome to favor healthy stages. Finally, we present computational approaches for disentangling the metabolic activity underlying host-microbiota codependence. Altogether, we envision that the combination of physiology and computational modeling of microbiota metabolism will drive us to optimize the diagnosis and treatment of T2D patients in a personalized way.

[Source](https://doi.org/10.1080/19490976.2022.2111952)

10.1080/19490976.2022.2111952

---

## Human gut bifidobacteria inhibit the growth of the opportunistic fungal pathogen <em>Candida albicans</em>

[Source](https://doi.org/10.1093/femsec/fiac095)

---

## Orally administered Odoribacter laneus improves glucose control and inflammatory profile in obese mice by depleting circulating succinate.
 28.08.2022, _I Huber-Ruano, E Calvo, J Mayneris-Perxachs, MM Rodríguez-Peña, V Ceperuelo-Mallafré, L Cedó, C Núñez-Roa, J Miro-Blanch, M Arnoriaga-Rodríguez, A Balvay, C Maudet, P García-Roves, O Yanes, S Rabot, GM Grimaud, A De Prisco, A Amoruso, JM Fernández-Real, J Vendrell, S Fernández-Veledo_


_Animals, Bacteroidetes, Blood Glucose, Diabetes Mellitus, Type 2, Diet, High-Fat, Humans, Inflammation, Mice, Mice, Inbred C57BL, Mice, Obese, Obesity, Succinic Acid_
Succinate is produced by both human cells and by gut bacteria and couples metabolism to inflammation as an extracellular signaling transducer. Circulating succinate is elevated in patients with obesity and type 2 diabetes and is linked to numerous complications, yet no studies have specifically addressed the contribution of gut microbiota to systemic succinate or explored the consequences of reducing intestinal succinate levels in this setting.

[Source](https://doi.org/10.1186/s40168-022-01306-y)

10.1186/s40168-022-01306-y

---

## Orally administered <em>Odoribacter laneus </em>improves glucose control and inflammatory profile in obese mice by depleting circulating succinate

[Source](https://doi.org/10.1186/s40168-022-01306-y)

---

## Experimental methods modestly impact interpretation of the effect of environmental exposures on the larval zebrafish gut microbiome.
 29.08.2022, _K Stagaman, KD Kasschau, RL Tanguay, TJ Sharpton_


_Animals, DNA, Environmental Exposure, Gastrointestinal Microbiome, Larva, Zebrafish_
Rapidly growing fields, such as microbiome science, often lack standardization of procedures across research groups. This is especially the case for microbiome investigations in the zebrafish (Danio rerio) model system, which is quickly becoming a workhorse system for understanding the exposure-microbiome-physiology axis. To guide future investigations using this model system, we defined how various experimental decisions affect the outcomes of studies on the effects of exogenous exposure on the zebrafish gut microbiome. Using a model toxicant, benzo[a]pyrene (BaP), we assessed how each of two dissection methods (gut dissection vs. whole fish), three DNA extraction kits (Qiagen Blood &amp; Tissue, Macherey-Nagel NucleoSpin, and Qiagen PowerSoil), and inclusion of PCR replicates (single vs. pooled triplicate reactions) affected our interpretation of how exposure influences the diversity and composition of the gut microbiome, as well as our ability to identify microbiome biomarkers of exposure. We found that inclusion of PCR replicates had the smallest effect on our final interpretations, and the effects of dissection method and DNA extraction kit had significant effects in specific contexts, primarily in the cases of identifying microbial biomarkers.

[Source](https://doi.org/10.1038/s41598-022-18532-x)

10.1038/s41598-022-18532-x

---

## A Metabologenomic approach reveals alterations in the gut microbiota of a mouse model of Alzheimer's disease.
 27.08.2022, _F Favero, E Barberis, M Gagliardi, S Espinoza, L Contu, S Gustincich, F Boccafoschi, C Borsotti, D Lim, V Rubino, F Mignone, E Pasolli, M Manfredi, S Zucchelli, D Corà, M Corazzari_


_Alzheimer Disease, Animals, Disease Models, Animal, Dysbiosis, Gastrointestinal Microbiome, Humans, Mice, Neurodegenerative Diseases_
The key role played by host-microbiota interactions on human health, disease onset and progression, and on host response to treatments has increasingly emerged in the latest decades. Indeed, dysbiosis has been associated to several human diseases such as obesity, diabetes, cancer and also neurodegenerative disease, such as Parkinson, Huntington and Alzheimer's disease (AD), although whether causative, consequence or merely an epiphenomenon is still under investigation. In the present study, we performed a metabologenomic analysis of stool samples from a mouse model of AD, the 3xTgAD. We found a significant change in the microbiota of AD mice compared to WT, with a longitudinal divergence of the F/B ratio, a parameter suggesting a gut dysbiosis. Moreover, AD mice showed a significant decrease of some amino acids, while data integration revealed a dysregulated production of desaminotyrosine (DAT) and dihydro-3-coumaric acid. Collectively, our data show a dysregulated gut microbiota associated to the onset and progression of AD, also indicating that a dysbiosis can occur prior to significant clinical signs, evidenced by early SCFA alterations, compatible with gut inflammation.

[Source](https://doi.org/10.1371/journal.pone.0273036)

10.1371/journal.pone.0273036

---

## Intraintestinal Analysis of the Functional Activity of Microbiomes and Its Application to the Common Marmoset Intestine

[Source](https://doi.org/10.1128/msystems.00520-22)

---

## A comprehensive comparison of fecal microbiota in three ecological bird groups of raptors, waders, and waterfowl.
 26.08.2022, _C Zhao, L Liu, L Gao, L Bai_


Gut microbiota plays a vital role in maintaining the health and immunity of wild birds. However, less is known about the comparison of fecal microbiota between different ecological groups of wild birds, particularly in the Yellow River National Wetland in Baotou, China, an important transit point for birds migrating all over the East Asia-Australian and Central Asian flyways. In this study, we characterized the fecal microbiota and potential microbial function in nine bird species of raptors, waders, and waterfowl using 16S rRNA gene amplicon sequencing to reveal the microbiota differences and interaction patterns. The results indicated that there was no significant difference in α-diversity, but a significant difference in β-diversity between the three groups of birds. The fecal bacterial microbiota was dominated by Firmicutes, Proteobacteria, Actinobacteria, and Bacteroidetes in all groups of birds. Furthermore, we identified five bacterial genera that were significantly higher in raptors, five genera that were significantly higher in waders, and two genera that were more abundant in waterfowl. The bacterial co-occurrence network results revealed 15 and 26 key genera in raptors and waterfowls, respectively. The microbial network in waterfowl exhibited a stronger correlation pattern than that in raptors. PICRUSt2 predictions indicated that fecal bacterial function was significantly enriched in the antibiotic biosynthesis pathway in all three groups. Metabolic pathways related to cell motility (bacterial chemotaxis and flagellar assembly) were significantly more abundant in raptors than in waders, whereas waders were enriched in lipid metabolism (synthesis and degradation of ketone bodies and fatty acid biosynthesis). The fecal microbiota in waterfowl harbored more abundant vitamin B6 metabolism, RNA polymerase, and tyrosine and tryptophan biosynthesis. This comparative study revealed the microbial community structure, microbial co-occurrence patterns, and potential functions, providing a better understanding of the ecology and conservation of wild birds. Future studies may focus on unraveling metagenomic functions and dynamics along with the migration routine or different seasons by metagenomics or metatranscriptomics.

[Source](https://doi.org/10.3389/fmicb.2022.919111)

10.3389/fmicb.2022.919111

---

## IHNV Infection Induces Strong Mucosal Immunity and Changes of Microbiota in Trout Intestine.
 29.08.2022, _Z Huang, M Zhan, G Cheng, R Lin, X Zhai, H Zheng, Q Wang, Y Yu, Z Xu_


_Animals, Fish Diseases, Immunity, Mucosal, Infectious hematopoietic necrosis virus, Inflammation, Intestinal Mucosa, Mammals, Microbiota, Oncorhynchus mykiss, RNA, Ribosomal, 16S, Rhabdoviridae Infections_
The fish intestinal mucosa is among the main sites through which environmental microorganisms interact with the host. Therefore, this tissue not only constitutes the first line of defense against pathogenic microorganisms but also plays a crucial role in commensal colonization. The interaction between the mucosal immune system, commensal microbiota, and viral pathogens has been extensively described in the mammalian intestine. However, very few studies have characterized these interactions in early vertebrates such as teleosts. In this study, rainbow trout (Oncorhynchus mykiss) was infected with infectious hematopoietic necrosis virus (IHNV) via a recently developed immersion method to explore the effects of viral infection on gut immunity and microbial community structure. IHNV successfully invaded the gut mucosa of trout, resulting in severe tissue damage, inflammation, and an increase in gut mucus. Moreover, viral infection triggered a strong innate and adaptive immune response in the gut, and RNA-seq analysis indicated that both antiviral and antibacterial immune pathways were induced, suggesting that the viral infection was accompanied by secondary bacterial infection. Furthermore, 16S rRNA sequencing also revealed that IHNV infection induced severe dysbiosis, which was characterized by large increases in the abundance of Bacteroidetes and pathobiont proliferation. Moreover, the fish that survived viral infection exhibited a reversal of tissue damage and inflammation, and their microbiome was restored to its pre-infection state. Our findings thus demonstrated that the relationships between the microbiota and gut immune system are highly sensitive to the physiological changes triggered by viral infection. Therefore, opportunistic bacterial infection must also be considered when developing strategies to control viral infection.

[Source](https://doi.org/10.3390/v14081838)

10.3390/v14081838

---

## Wildfire-dependent changes in soil microbiome diversity and function.
 29.08.2022, _AR Nelson, AB Narrowe, CC Rhoades, TS Fegel, RA Daly, HK Roth, RK Chu, KK Amundson, RB Young, AS Steindorff, SJ Mondo, IV Grigoriev, A Salamov, T Borch, MJ Wilkins_


Forest soil microbiomes have crucial roles in carbon storage, biogeochemical cycling and rhizosphere processes. Wildfire season length, and the frequency and size of severe fires have increased owing to climate change. Fires affect ecosystem recovery and modify soil microbiomes and microbially mediated biogeochemical processes. To study wildfire-dependent changes in soil microbiomes, we characterized functional shifts in the soil microbiota (bacteria, fungi and viruses) across burn severity gradients (low, moderate and high severity) 1 yr post fire in coniferous forests in Colorado and Wyoming, USA. We found severity-dependent increases of Actinobacteria encoding genes for heat resistance, fast growth, and pyrogenic carbon utilization that might enhance post-fire survival. We report that increased burn severity led to the loss of ectomycorrhizal fungi and less tolerant microbial taxa. Viruses remained active in post-fire soils and probably influenced carbon cycling and biogeochemistry via turnover of biomass and ecosystem-relevant auxiliary metabolic genes. Our genome-resolved analyses link post-fire soil microbial taxonomy to functions and reveal the complexity of post-fire soil microbiome activity.

[Source](https://doi.org/10.1038/s41564-022-01203-y)

10.1038/s41564-022-01203-y

---

## Fungal succession on the decomposition of three plant species from a Brazilian mangrove.
 29.08.2022, _MA Moitinho, JB Chiaramonte, L Bononi, T Gumiere, IS Melo, RG Taketani_


_Avicennia, Brazil, Combretaceae, Microbiota, Plant Leaves, Plants, Rhizophoraceae_
Leaf decomposition is the primary process in release of nutrients in the dynamic mangrove habitat, supporting the ecosystem food webs. On most environments, fungi are an essential part of this process. However, due to the peculiarities of mangrove forests, this group is currently neglected. Thus, this study tests the hypothesis that fungal communities display a specific succession pattern in different mangrove species and this due to differences in their ecological role. A molecular approach was employed to investigate the dynamics of the fungal community during the decomposition of three common plant species (Rhizophora mangle, Laguncularia racemosa, and Avicennia schaueriana) from a mangrove habitat located at the southeast of Brazil. Plant material was the primary driver of fungi communities, but time also was marginally significant for the process, and evident changes in the fungal community during the decomposition process were observed. The five most abundant classes common to all the three plant species were Saccharomycetes, Sordariomycetes, Tremellomycetes, Eurotiomycetes, and Dothideomycetes, all belonging to the Phylum Ascomycota. Microbotryomycetes class were shared only by A. schaueriana and L. racemosa, while Agaricomycetes class were shared by L. racemosa and R. mangle. The class Glomeromycetes were shared by A. schaueriana and R. mangle. The analysis of the core microbiome showed that Saccharomycetes was the most abundant class. In the variable community, Sordariomycetes was the most abundant one, mainly in the Laguncularia racemosa plant. The results presented in this work shows a specialization of the fungal community regarding plant material during litter decomposition which might be related to the different chemical composition and rate of degradation.

[Source](https://doi.org/10.1038/s41598-022-18667-x)

10.1038/s41598-022-18667-x

---

## Fungi-on-a-Chip: microfluidic platforms for single-cell studies on fungi.
 24.08.2022, _F Richter, S Bindschedler, M Calonne-Salmon, S Declerck, P Junier, CE Stanley_


Fungi, with their enormous diversity, bear essential roles both in nature and our everyday lives. They inhabit a range of ecosystems, such as soil, where they are involved in organic matter degradation and bioremediation processes. More recently, fungi have been recognised as key components of the microbiome in other eukaryotes, such as humans, where they play a fundamental role not only in human pathogenesis, but also likely as commensals. In the food sector, fungi are used either directly or as fermenting agents and are often key players in the biotechnological industry, where they are responsible for the production of both bulk chemicals and antibiotics. Although the macroscopic fruiting bodies are immediately recognisable by most observers, the structure, function and interactions of fungi with other microbes at the microscopic scale still remain largely hidden. Herein, we shed light on new advances in the emerging field of Fungi-on-a-Chip microfluidic technologies for single-cell studies on fungi. We discuss the development and application of microfluidic tools in the fields of medicine and biotechnology, as well as in-depth biological studies having significance for ecology and general natural processes. Finally, a future perspective is provided, highlighting new frontiers in which microfluidic technology can benefit this field.

[Source](https://doi.org/10.1093/femsre/fuac039)

10.1093/femsre/fuac039

---

## Climate-Endangered Arctic Epishelf Lake Harbors Viral Assemblages with Distinct Genetic Repertoires.
 25.08.2022, _M Labbé, M Thaler, TM Pitot, JZ Rapp, WF Vincent, AI Culley_


Milne Fiord, located on the coastal margin of the Last Ice Area (LIA) in the High Arctic (82°N, Canada), harbors an epishelf lake, a rare type of ice-dependent ecosystem in which a layer of freshwater overlies marine water connected to the open ocean. This microbe-dominated ecosystem faces catastrophic change due to the deterioration of its ice environment related to warming temperatures. We produced the first assessment of viral abundance, diversity, and distribution in this vulnerable ecosystem and explored the niches available for viral taxa and the functional genes underlying their distribution. We found that the viral community in the freshwater layer was distinct from, and more diverse than, the community in the underlying seawater and contained a different set of putative auxiliary metabolic genes, including the sulfur starvation-linked gene tauD and the gene coding for patatin-like phospholipase. The halocline community resembled the freshwater more than the marine community, but harbored viral taxa unique to this layer. We observed distinct viral assemblages immediately below the halocline, at a depth that was associated with a peak of prasinophyte algae and the viral family Phycodnaviridae. We also assembled 15 complete circular genomes, including a putative Pelagibacter phage with a marine distribution. It appears that despite its isolated and precarious situation, the varied niches in this epishelf lake support a diverse viral community, highlighting the importance of characterizing underexplored microbiota in the Last Ice Area before these ecosystems undergo irreversible change. IMPORTANCE Viruses are key to understanding polar aquatic ecosystems, which are dominated by microorganisms. However, studies of viral communities are challenging to interpret because the vast majority of viruses are known only from sequence fragments, and their taxonomy, hosts, and genetic repertoires are unknown. Our study establishes a basis for comparison that will advance understanding of viral ecology in diverse global environments, particularly in the High Arctic. Rising temperatures in this region mean that researchers have limited time remaining to understand the biodiversity and biogeochemical cycles of ice-dependent environments and the consequences of these rapid, irreversible changes. The case of the Milne Fiord epishelf lake has special urgency because of the rarity of this type of "floating lake" ecosystem and its location in the Last Ice Area, a region of thick sea ice with global importance for conservation efforts.

[Source](https://doi.org/10.1128/aem.00228-22)

10.1128/aem.00228-22

---

## Profiling of the microbiota in the remaining sports drink and orange juice in plastic bottles after direct drinking.
 25.08.2022, _M Kawachi, A Wakui, N Kaku, N Takahashi, S Maruyama, J Washio, Y Abiko, G Mayanagi, K Tanaka, N Takahashi, T Sato_


The survival of bacteria in the sports drink and orange juice remaining in and at the mouth of bottles after direct drinking was examined after immediately drinking and incubation at 37°C for 24 h.

[Source](https://doi.org/10.1016/j.job.2022.08.003)

10.1016/j.job.2022.08.003

---

## K-Clique Multiomics Framework: A Novel Protocol to Decipher the Role of Gut Microbiota Communities in Nutritional Intervention Trials.
 29.08.2022, _C Mengucci, L Nissen, G Picone, C Malpuech-Brugère, C Orfila, L Ricciardiello, A Bordoni, F Capozzi, A Gianotti_


The availability of omics data providing information from different layers of complex biological processes that link nutrition to human health would benefit from the development of integrated approaches combining holistically individual omics data, including those associated with the microbiota that impacts the metabolisation and bioavailability of food components. Microbiota must be considered as a set of populations of interconnected consortia, with compensatory capacities to adapt to different nutritional intake. To study the consortium nature of the microbiome, we must rely on specially designed data analysis tools. The purpose of this work is to propose the construction of a general correlation network-based explorative tool, suitable for nutritional clinical trials, by integrating omics data from faecal microbial taxa, stool metabolome (1H NMR spectra) and GC-MS for stool volatilome. The presented approach exploits a descriptive paradigm necessary for a true multiomics integration of data, which is a powerful tool to investigate the complex physiological effects of nutritional interventions.

[Source](https://doi.org/10.3390/metabo12080736)

10.3390/metabo12080736

---

## Principal microbial groups: compositional alternative to phylogenetic grouping of microbiome data.
 25.08.2022, _A Boyraz, V Pawlowsky-Glahn, JJ Egozcue, AC Acar_


Statistical and machine learning techniques based on relative abundances have been used to predict health conditions and to identify microbial biomarkers. However, high dimensionality, sparsity and the compositional nature of microbiome data represent statistical challenges. On the other hand, the taxon grouping allows summarizing microbiome abundance with a coarser resolution in a lower dimension, but it presents new challenges when correlating taxa with a disease. In this work, we present a novel approach that groups Operational Taxonomical Units (OTUs) based only on relative abundances as an alternative to taxon grouping. The proposed procedure acknowledges the compositional data making use of principal balances. The identified groups are called Principal Microbial Groups (PMGs). The procedure reduces the need for user-defined aggregation of $\textrm{OTU}$s and offers the possibility of working with coarse group of $\textrm{OTU}$s, which are not present in a phylogenetic tree. PMGs can be used for two different goals: (1) as a dimensionality reduction method for compositional data, (2) as an aggregation procedure that provides an alternative to taxon grouping for construction of microbial balances afterward used for disease prediction. We illustrate the procedure with a cirrhosis study data. PMGs provide a coherent data analysis for the search of biomarkers in human microbiota. The source code and demo data for PMGs are available at: https://github.com/asliboyraz/PMGs.

[Source](https://doi.org/10.1093/bib/bbac328)

10.1093/bib/bbac328

---

## Postnatal growth and gut microbiota development influenced early childhood growth in preterm infants.
 27.08.2022, _JS Tadros, A Llerena, A Sarkar, R Johnson, EM Miller, HL Gray, TTB Ho_


Preterm infants are at high risk for growth failure and childhood weight problems due to the disruption of normal intrauterine growth and nutrition. Early nutritional support and microbiome acquisition can play an important role in childhood growth.

[Source](https://doi.org/10.1007/978-1-0716-2549-1_21)

10.3389/fped.2022.850629

---

## Analysis of Gut Microbiome Using Fecal Samples

[Source](https://doi.org/10.1007/978-1-0716-2549-1_21)

---

