## Lessons and recommendations from three decades as an NSF REU site: A call for systems-based assessment.
 28.03.2020, _AL McDevitt, MV Patel, AM Ellison_


For more than 30 years, the US National Science Foundation's Research Experiences for Undergraduates (REU) program has supported thousands of undergraduate researchers annually and provides many students with their first research experiences in field ecology or evolution. REUs embed students in scientific communities where they apprentice with experienced researchers, build networks with their peers, and help students understand research cultures and how to work within them. REUs are thought to provide formative experiences for developing researchers that differ from experiences in a college classrooms, laboratories, or field trips. REU assessments have improved through time but they are largely ungrounded in educational theory. Thus, evaluation of long-term impacts of REUs remains limited and best practices for using REUs to enhance student learning are repeatedly re-invented. We describe how one sociocultural learning framework, cultural-historical activity theory (CHAT), could be used to guide data collection to characterize the effects of REU programs on participant's learning in an educationally meaningful context. CHAT embodies a systems approach to assessment that accounts for social and cultural factors that influence learning. We illustrate how CHAT has guided assessment of the Harvard Forest Summer Research Program in Ecology (HF-SRPE), one of the longest-running REU sites in the United States. Characterizing HF-SRPE using CHAT helped formalize thoughts and language for the program evaluation, reflect on potential barriers to success, identify assessment priorities, and revealed important oversights in data collection.

[Source](https://nsurp.org/)

10.1002/ece3.6136

---

## National Summer Undergraduate Research Project

[Source](https://nsurp.org/)

---

## <em>Salmonella</em> finds a way: Metabolic versatility of <em>Salmonella enterica</em> serovar Typhimurium in diverse host environments

[Source](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1008540)

---

## Phylogenetic background and habitat drive the genetic diversification of <em>Escherichia coli</em>

[Source](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008866)

---

## The microbiome of diabetic foot ulcers: a comparison of swab and tissue biopsy wound sampling techniques using 16S rRNA gene sequencing.
 19.06.2020, _J Travis, M Malone, H Hu, A Baten, K Johani, F Huygens, K Vickery, K Benkendorff_


Health-care professionals need to collect wound samples to identify potential pathogens that contribute to wound infection. Obtaining appropriate samples from diabetic foot ulcers (DFUs) where there is a suspicion of infection is of high importance. Paired swabs and tissue biopsies were collected from DFUs and both sampling techniques were compared using 16S rRNA gene sequencing.

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-020-01843-2)

10.1186/s12866-020-01843-2

---

## MAPK-dependent hormonal signaling plasticity contributes to overcoming <em>Bacillus thuringiensis </em>toxin action in an insect host

[Source](https://www.nature.com/articles/s41467-020-16608-8?)

---

## Investigation of the effects of probiotic, <em>Bacillus subtilis</em> on stress reactions in laying hens using infrared thermography

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234117)

---

## Gut carriage of antimicrobial resistance genes in women exposed to small-scale poultry farms in rural Uganda: A feasibility study

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0229699#abstract0)

---

## Anti-microbial activity of whole blood and plasma collected from Anna’s Hummingbirds (<em>Calypte anna</em>) against three different microbes

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234239)

---

## Bacteriophages specific to Shiga toxin-producing <em>Escherichia coli</em> exist in goat feces and associated environments on an organic produce farm in Northern California, USA

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234438)

---

## Survival of <em>Escherichia coli </em>O157 in autoclaved and natural sandy soil mesocosms

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234562)

---

## Geometry and evolution of the ecological niche in plant-associated microbes

[Source](https://www.nature.com/articles/s41467-020-16778-5?)

---

## Meta-analysis of the impacts of global change factors on soil microbial diversity and functionality

[Source](https://www.nature.com/articles/s41467-020-16881-7?)

---

## Neighboring plants divergently modulate effects of loss-of-function in maize mycorrhizal phosphate uptake on host physiology and root fungal microbiota.
 19.06.2020, _I Fabiańska, L Pesch, E Koebke, N Gerlach, M Bucher_


Maize, a main crop worldwide, establishes a mutualistic symbiosis with arbuscular mycorrhizal (AM) fungi providing nutrients to the roots from soil volumes which are normally not in reach of the non-colonized root. The mycorrhizal phosphate uptake pathway (MPU) spans from extraradical hyphae to root cortex cells housing fungal arbuscules and promotes the supply of phosphate to the mycorrhizal host in exchange for photosynthetic carbon. This symbiotic association with the mycobiont has been shown to affect plant host nutritional status and growth performance. However, whether and how the MPU affects the root microbial community associated with mycorrhizal hosts in association with neighboring plants, remains to be demonstrated. Here the maize germinal Mu transposon insertion mutant pht1;6, defective in mycorrhiza-specific Pi transporter PHT1;6 gene, and wild type B73 (wt) plants were grown in mono- and mixed culture and examined under greenhouse and field conditions. Disruption of the MPU in pht1;6 resulted in strongly diminished growth performance, in reduced P allocation to photosynthetic source leaves, and in imbalances in leaf elemental composition beyond P. At the microbial community level a loss of MPU activity had a minor effect on the root-associated fungal microbiome which was almost fully restricted to AM fungi of the Glomeromycotina. Moreover, while wt grew better in presence of pht1;6, pht1;6 accumulated little biomass irrespective of whether it was grown in mono- or mixed culture and despite of an enhanced fungal colonization of its roots in co-culture with wt. This suggested that a functional MPU is prerequisite to maintain maize growth and that neighboring plants competed for AM fungal Pi in low P soil. Thus future strategies towards improving yield in maize populations on soils with low inputs of P fertilizer could be realized by enhancing MPU at the individual plant level while leaving the root-associated fungal community largely unaffected.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0232633)

10.1371/journal.pone.0232633

---

## Comparative analysis of viruses in four bee species collected from agricultural, urban, and natural landscapes

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234431)

---

## bigPint: A Bioconductor visualization package that makes big data pint-sized.
 16.06.2020, _L Rutter, D Cook_


Interactive data visualization is imperative in the biological sciences. The development of independent layers of interactivity has been in pursuit in the visualization community. We developed bigPint, a data visualization package available on Bioconductor under the GPL-3 license (https://bioconductor.org/packages/release/bioc/html/bigPint.html). Our software introduces new visualization technology that enables independent layers of interactivity using Plotly in R, which aids in the exploration of large biological datasets. The bigPint package presents modernized versions of scatterplot matrices, volcano plots, and litre plots through the implementation of layered interactivity. These graphics have detected normalization issues, differential expression designation problems, and common analysis errors in public RNA-sequencing datasets. Researchers can apply bigPint graphics to their data by following recommended pipelines written in reproducible code in the user manual. In this paper, we explain how we achieved the independent layers of interactivity that are behind bigPint graphics. Pseudocode and source code are provided. Computational scientists can leverage our open-source code to expand upon our layered interactive technology and/or apply it in new ways toward other computational biology tasks.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007912)

10.1371/journal.pcbi.1007912

---

## Subduing the influence of PCR inhibitors on amplifying aged, degraded, and low copy number DNA: PCR enhancer cocktail-p and rescue PCR.
 16.06.2020, _BM Kemp, B Bingham, R Frome, M Labonte, E Palmer, ES Parsons, KW Gobalet, J Rosenthal_


PCR inhibitors are a formidable problem to the study of aged, degraded, and/or low copy number DNA. As a result, there is a need to find alternate methods that ameliorate the efficacy of PCR. In this study, we attempted to use genetic methods to identify the species of salmonid (Oncorhynchus spp.) remains recovered from archaeological sites along the Feather River located in northern California, United States. In the process of doing so, we compared the efficacy of a PCR enhancer cocktail called "PEC-P" and a reagent rich PCR recipe called "rescue PCR" over standard PCR. Across all treatments (full concentration and 1:10 dilute eluates subjected to standard PCR, PEC-P, and rescue PCR) species identification was possible for 74 of 93 archaeological fish specimens (79.6%). Overall, six of the 93 samples (6.5%) consistently yielded species identification across all treatments. The species of ten specimens (10.8%) were uniquely identified from amplicons produced with either PEC-P or rescue PCR or both. Notably, the species of seven samples (7.5%) were uniquely identified with standard PCR over the alternative treatments. Considering both full concentration and 1:10 dilute eluates (N = 186), standard PCR performed as well as PEC-P (p = 0.1451) and rescue (p = 0.6753). Yet, considering results from full concentration eluates alone (N = 93), PEC-P (60.2%) outperformed both standard PCR (44.1%; p = 0.0277) and rescue PCR (40.9%; p = 0.0046). Stochasticity observed in our study cautions us against choosing a "best" performing method of those explored here and suggests their respective potentials to improve success may be sample dependent. When working with samples compromised by PCR inhibitors, it is useful to have alternative methodologies for subduing the problem. Both PEC-P and rescue PCR represent useful alternative methods for the study of aged, degraded, and/or low copy number DNA samples compromised by PCR inhibitors.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234745)

10.1371/journal.pone.0234745

---

## Click-to-Capture: A method for enriching viable <em>Staphylococcus aureus </em>using bio-orthogonal labeling of surface proteins

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234542)

---

