## Divergent age-associated and metabolism-associated gut microbiome signatures modulate cardiovascular disease risk.
 06.06.2024, _T Wang, Z Shi, H Ren, M Xu, J Lu, F Yang, C Ye, K Wu, M Chen, X Xu, D Liu, L Kong, R Zheng, J Zheng, M Li, Y Xu, Z Zhao, Y Chen, H Yang, J Wang, G Ning, J Li, H Zhong, Y Bi, W Wang_


Insight into associations between the gut microbiome with metabolism and aging is crucial for tailoring interventions to promote healthy longevity. In a discovery cohort of 10,207 individuals aged 40-93 years, we used 21 metabolic parameters to classify individuals into five clusters, termed metabolic multimorbidity clusters (MCs), that represent different metabolic subphenotypes. Compared to the cluster classified as metabolically healthy (MC1), clusters classified as 'obesity-related mixed' (MC4) and 'hyperglycemia' (MC5) exhibited an increased 11.1-year cardiovascular disease (CVD) risk by 75% (multivariable-adjusted hazard ratio (HR): 1.75, 95% confidence interval (CI): 1.43-2.14) and by 117% (2.17, 1.72-2.74), respectively. These associations were replicated in a second cohort of 9,061 individuals with a 10.0-year follow-up. Based on analysis of 4,491 shotgun fecal metagenomes from the discovery cohort, we found that gut microbial composition was associated with both MCs and age. Next, using 55 age-specific microbial species to capture biological age, we developed a gut microbial age (MA) metric, which was validated in four external cohorts comprising 4,425 metagenomic samples. Among individuals aged 60 years or older, the increased CVD risk associated with MC4 or MC5, as compared to MC1, MC2 or MC3, was exacerbated in individuals with high MA but diminished in individuals with low MA, independent of age, sex and other lifestyle and dietary factors. This pattern, in which younger MA appears to counteract the CVD risk attributable to metabolic dysfunction, implies a modulating role of MA in cardiovascular health for metabolically unhealthy older people.

[Source](https://doi.org/10.1038/s41591-024-03038-y)

10.1038/s41591-024-03038-y

---

## Quantifying variation across 16S rRNA gene sequencing runs in human microbiome studies.
 08.06.2024, _AJ Hoisington, CE Stamper, JC Ellis, CA Lowry, LA Brenner_


_Humans, RNA, Ribosomal, 16S, Feces, Microbiota, Sequence Analysis, DNA, DNA, Bacterial, Bacteria, Reproducibility of Results, Mouth, High-Throughput Nucleotide Sequencing_
Recent microbiome research has incorporated a higher number of samples through more participants in a study, longitudinal studies, and metanalysis between studies. Physical limitations in a sequencing machine can result in samples spread across sequencing runs. Here we present the results of sequencing nearly 1000 16S rRNA gene sequences in fecal (stabilized and swab) and oral (swab) samples from multiple human microbiome studies and positive controls that were conducted with identical standard operating procedures. Sequencing was performed in the same center across 18 different runs. The simplified mock community showed limitations in accuracy, while precision (e.g., technical variation) was robust for the mock community and actual human positive control samples. Technical variation was the lowest for stabilized fecal samples, followed by fecal swab samples, and then oral swab samples. The order of technical variation stability was inverse of DNA concentrations (e.g., highest in stabilized fecal samples), highlighting the importance of DNA concentration in reproducibility and urging caution when analyzing low biomass samples. Coefficients of variation at the genus level also followed the same trend for lower variation with higher DNA concentrations. Technical variation across both sample types and the two human sampling locations was significantly less than the observed biological variation. Overall, this research providing comparisons between technical and biological variation, highlights the importance of using positive controls, and provides semi-quantified data to better understand variation introduced by sequencing runs. KEY POINTS: • Mock community and positive control accuracy were lower than precision. • Samples with lower DNA concentration had increased technical variation across sequencing runs. • Biological variation was significantly higher than technical variation due to sequencing runs.

[Source](https://doi.org/10.1007/s00253-024-13198-z)

10.1007/s00253-024-13198-z

---

## Gut microbiome changes associated with chronic pancreatitis and pancreatic cancer: A systematic review and Meta-analysis.
 07.06.2024, _J Hong, Y Fu, X Chen, Y Zhang, X Li, T Li, Y Liu, M Fan, R Lin_


The study of changes in the microbiome in chronic pancreatitis (CP) and pancreatic ductal adenocarcinoma (PDAC) holds significant potential for developing noninvasive diagnostic tools as well as innovative interventions to alter the progression of diseases. This systematic review and meta-analysis aimed to analyze in detail the taxonomic and functional characteristics of the gut microbiome in patients with CP and PDAC.

[Source](https://doi.org/10.1097/JS9.0000000000001724)

10.1097/JS9.0000000000001724

---

## Gut microbial signatures of patients with diarrhea-predominant irritable bowel syndrome and their healthy relatives.
 07.06.2024, _J Chen, H Lan, C Li, Y Xie, X Cheng, R Xia, C Ke, X Liang_


Irritable bowel syndrome (IBS) is a prevalent gastrointestinal disorder, encompassing diarrhea-predominant irritable bowel syndrome (IBS-D). Here, we utilized 16S rDNA gene sequencing to identify potential microbial drivers of IBS-D.

[Source](https://doi.org/10.1093/jambio/lxae118)

10.1093/jambio/lxae118

---

## Two weeks of acarbose treatment show no effect on gut microbiome composition in patients with type 2 diabetes: a randomised, placebo-controlled, double-blind, crossover study.
 06.06.2024, _NB Dalsgaard, LS Gasbjerg, LS Hansen, D Nielsen, T Rasmussen, FK Knop_


The alpha-glucosidase inhibitor acarbose is approved for the treatment of type 2 diabetes (T2D). It acts in the lumen of the gut by reducing intestinal hydrolysis and absorption of ingested carbohydrates. This reduces postprandial blood glucose concentration and increases the content of carbohydrates in the distal parts of the intestine potentially influencing gut microbiome (GM) composition and thereby possibly impacting the gut microbiome (GM) dysbiosis associated with T2D. Here, we investigated the effect of acarbose on GM composition in patients with T2D.

[Source](https://doi.org/10.1530/EC-24-0052)

10.1530/EC-24-0052

---

## A conserved inhibitory interdomain interaction regulates DNA-binding activities of hybrid two-component systems in<em> Bacteroides</em>

[Source](https://doi.org/10.1128/mbio.01220-24)

---

## Multi-omics Analysis Reveals Key Gut Microbiota and Metabolites Closely Associated with Huntington's Disease.
 08.06.2024, _SX Qian, YF Bao, XY Li, Y Dong, XL Zhang, ZY Wu_


Dysbiosis of the gut microbiota is closely associated with neurodegenerative diseases, including Huntington's disease (HD). Gut microbiome-derived metabolites are key factors in host-microbiome interactions. This study aimed to investigate the crucial gut microbiome and metabolites in HD and their correlations. Fecal and serum samples from 11 to 26 patients with HD, respectively, and 16 and 23 healthy controls, respectively, were collected. The fecal samples were used for shotgun metagenomics while the serum samples for metabolomics analysis. Integrated analysis of the metagenomics and metabolomics data was also conducted. Firmicutes, Bacteroidota, Proteobacteria, Uroviricota, Actinobacteria, and Verrucomicrobia were the dominant phyla. At the genus level, the presence of Bacteroides, Faecalibacterium, Parabacteroides, Alistipes, Dialister, and Christensenella was higher in HD patients, while the abundance of Lachnospira, Roseburia, Clostridium, Ruminococcus, Blautia, Butyricicoccus, Agathobaculum, Phocaeicola, Coprococcus, and Fusicatenibacter decreased. A total of 244 differential metabolites were identified and found to be enriched in the glycerophospholipid, nucleotide, biotin, galactose, and alpha-linolenic acid metabolic pathways. The AUC value from the integrated analysis (1) was higher than that from the analysis of the gut microbiota (0.8632). No significant differences were found in the ACE, Simpson, Shannon, Sobs, and Chao indexes between HD patients and controls. Our study determined crucial functional gut microbiota and potential biomarkers associated with HD pathogenesis, providing new insights into the role of the gut microbiota-brain axis in HD occurrence and development.

[Source](https://doi.org/10.1007/s12035-024-04271-9)

10.1007/s12035-024-04271-9

---

## Unravelling the role of intratumoral bacteria in digestive system cancers: current insights and future perspectives.
 07.06.2024, _W Lu, A Aihaiti, P Abudukeranmu, Y Liu, H Gao_


_Humans, Digestive System Neoplasms, Bacteria, Gastrointestinal Microbiome, Animals_
Recently, research on the human microbiome, especially concerning the bacteria within the digestive system, has substantially advanced. This exploration has unveiled a complex interplay between microbiota and health, particularly in the context of disease. Evidence suggests that the gut microbiome plays vital roles in digestion, immunity and the synthesis of vitamins and neurotransmitters, highlighting its significance in maintaining overall health. Conversely, disruptions in these microbial communities, termed dysbiosis, have been linked to the pathogenesis of various diseases, including digestive system cancers. These bacteria can influence cancer progression through mechanisms such as DNA damage, modulation of the tumour microenvironment, and effects on the host's immune response. Changes in the composition and function within the tumours can also impact inflammation, immune response and cancer therapy effectiveness. These findings offer promising avenues for the clinical application of intratumoral bacteria for digestive system cancer treatment, including the potential use of microbial markers for early cancer detection, prognostication and the development of microbiome-targeted therapies to enhance treatment outcomes. This review aims to provide a comprehensive overview of the pivotal roles played by gut microbiome bacteria in the development of digestive system cancers. Additionally, we delve into the specific contributions of intratumoral bacteria to digestive system cancer development, elucidating potential mechanisms and clinical implications. Ultimately, this review underscores the intricate interplay between intratumoral bacteria and digestive system cancers, underscoring the pivotal role of microbiome research in transforming diagnostic, prognostic and therapeutic paradigms for digestive system cancers.

[Source](https://doi.org/10.1186/s12967-024-05320-6)

10.1186/s12967-024-05320-6

---

## Effect of microplastics pollution on the gut microbiome of anecic and endogeic earthworms.
 07.06.2024, _C Papazlatani, P Garbeva, E Huerta-Lwanga_


Microplastic (MP) pollution constitutes an emerging type of pollution threatening both aquatic and terrestrial ecosystems. The impact on aquatic ecosystems has been extensively studied, but the effect on terrestrial ecosystems and their inhabitants is mostly underexplored. In this study we explored the effect of microplastic pollution on gut bacterial microbiome of endogeic (Aporrectodea caliginosa) and anecic (Lumbricus terrestris) earthworms. The experiments were performed in sandy soil with 0.2% of low-density polyethylene microplastics (LDPE MPs). We observed that the endogeic earthworms had 100% survival while anecic earthworms survived 25 days in the control (i.e. absence of MPs) and 21 days in the treatment with LDPE MPs. The main driver of shifts in the diversity and composition of the bacterial communities in the gut of tested earthworms was the life style of the worms, followed by the presence of MPs. The bacterial microbiome diversity was significantly different among the 2 types of earthworms and the highest bacterial diversity was found in the gut of the endogeic earthworms. The effect of microplastics on gut bacterial microbiome was clearly observed in the changes in the relative abundance of several phyla and families of the bacterial communities in both types of earthworms, although it was most evident in the anecic earthworms. The Actinobacteriota, Proteobacteria and Firmicutes were the main groups enhanced in the microplastic treatments, suggesting enrichment of the bacterial communities with potential plastic degraders.

[Source](https://doi.org/10.1093/femsle/fnae040)

10.1093/femsle/fnae040

---

## Novel indigenous probiotic isolated from healthy Pacific White shrimp <em>Litopenaeus vannamei</em> intestine in differing stages based on metagenomic and screening approach

[Source](https://doi.org/10.1016/j.fsi.2024.109678)

---

## <em>Citrobacter rodentium</em> infection impairs dopamine metabolism and exacerbates the pathology of Parkinson’s disease in mice

[Source](https://doi.org/10.1186/s12974-024-03145-0)

---

## Gut microbiota depletion delays somatic peripheral nerve development and impairs neuromuscular junction maturation

[Source](https://doi.org/10.1080/19490976.2024.2363015)

---

## Unravelling the impact of environmental factors in shaping plant microbiomes

[Source](https://doi.org/10.1111/1751-7915.14504)

---

## GROND: a quality-checked and publicly available database of full-length 16S-ITS-23S rRNA operon sequences

[Source](https://doi.org/10.1099/mgen.0.001255)

---

## Machine learning enabled design features of antimicrobial peptides selectively targeting peri-implant disease progression.
 09.06.2024, _K Boone, N Tjokro, KN Chu, C Chen, ML Snead, C Tamerler_


Peri-implantitis is a complex infectious disease that manifests as progressive loss of alveolar bone around the dental implants and hyper-inflammation associated with microbial dysbiosis. Using antibiotics in treating peri-implantitis is controversial because of antibiotic resistance threats, the non-selective suppression of pathogens and commensals within the microbial community, and potentially serious systemic sequelae. Therefore, conventional treatment for peri-implantitis comprises mechanical debridement by nonsurgical or surgical approaches with adjunct local microbicidal agents. Consequently, current treatment options may not prevent relapses, as the pathogens either remain unaffected or quickly re-emerge after treatment. Successful mitigation of disease progression in peri-implantitis requires a specific mode of treatment capable of targeting keystone pathogens and restoring bacterial community balance toward commensal species. Antimicrobial peptides (AMPs) hold promise as alternative therapeutics through their bacterial specificity and targeted inhibitory activity. However, peptide sequence space exhibits complex relationships such as sparse vector encoding of sequences, including combinatorial and discrete functions describing peptide antimicrobial activity. In this paper, we generated a transparent Machine Learning (ML) model that identifies sequence-function relationships based on rough set theory using simple summaries of the hydropathic features of AMPs. Comparing the hydropathic features of peptides according to their differential activity for different classes of bacteria empowered predictability of antimicrobial targeting. Enriching the sequence diversity by a genetic algorithm, we generated numerous candidate AMPs designed for selectively targeting pathogens and predicted their activity using classifying rough sets. Empirical growth inhibition data is iteratively fed back into our ML training to generate new peptides, resulting in increasingly more rigorous rules for which peptides match targeted inhibition levels for specific bacterial strains. The subsequent top scoring candidates were empirically tested for their inhibition against keystone and accessory peri-implantitis pathogens as well as an oral commensal bacterium. A novel peptide, VL-13, was confirmed to be selectively active against a keystone pathogen. Considering the continually increasing number of oral implants placed each year and the complexity of the disease progression, prevalence of peri-implant diseases continues to rise. Our approach offers transparent ML-enabled paths towards developing antimicrobial peptide-based therapies targeting the changes in the microbial communities that can beneficially impact disease progression.

[Source](https://doi.org/10.3389/fdmed.2024.1372534)

10.3389/fdmed.2024.1372534

---

## An exploratory<em> in silico</em> analysis of bacteriocin gene clusters in the urobiome

[Source](https://doi.org/10.20517/mrr.2023.78)

---

