## Altered gut microbiota as potential biomarker biomarkers for autism spectrum disorder in early childhood.
 04.06.2023, _Y Zhao, Y Wang, F Meng, X Chen, T Chang, H Huang, F He, Y Zheng_


Gastrointestinal (GI) disorders are widely recorded in autism spectrum disorder (ASD), and ASD with GI symptoms is a vital subtype of this disease. Growing evidence suggests altered gut microbiota biomarkers in ASD, but little is known about the gut microbiota of individuals with ASD with GI Symptoms, particularly in early childhood. In our study, the gut microbiota of 36 individuals with ASD along with GI symptoms and 40 typically developing (TD) children were compared using 16S rRNA gene sequencing. The microbial diversity and composition were found to differ between the two groups. Compared to TD, the gut microbiota of ASD patients with GI symptoms exhibited decreased alpha diversity and depletion of butyrate-producing bacteria (e.g., Faecalibacterium and Coprococcus). In addition, microbial functional analysis showed abnormality in several gut metabolic models and gut brain models of ASD with GI symptoms, including SCFAshort-chain fatty acid (SCFA) synthesis/degradation and neurotoxin-related p-cresol degradation, which are closely associated with ASD-related behaviors in animal models. Furthermore, we constructed a Support Vector Machine classification model, which robustly discriminated individuals with ASD and GI symptoms from TD individuals in a validation set (AUC = 0.88). Our findings provide a deep insight into the roles of the disturbed gut ecosystem in individuals with ASD and GI symptoms aged 3-6 years. Our classification model supports gut microbiota as a potential biomarker for the early identification of ASD and interventions targeting particular gut-beneficial microbiota.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0306452223002026?via%3Dihub)

10.1016/j.neuroscience.2023.04.029

---

## The composition of environmental microbiota in three tree fruit packing facilities changed over seasons and contained taxa indicative of <em>L. monocytogenes</em> contamination

[Source](https://doi.org/10.1186/s40168-023-01544-8)

---

## Succession and determinants of the early life nasopharyngeal microbiota in a South African birth cohort

[Source](https://doi.org/10.1186/s40168-023-01563-5)

---

## Secreted <em>Akkermansia muciniphila</em> threonyl-tRNA synthetase functions to monitor and modulate immune homeostasis

[Source](https://doi.org/10.1016/j.chom.2023.05.007)

---

## Gut Microbiome Composition is associated with future onset of Crohn’s Disease in Healthy First-Degree Relatives

[Source](https://www.gastrojournal.org/article/S0016-5085(23)00805-3/pdf)

---

## Metagenome Investigation of Ocular Microbiota of Cataract Patients With and Without Type 2 Diabetes.
 05.06.2023, _Z Shao, X Shan, L Jing, W Wang, W Li, Z Ren, BN Zhang, Y Huang_


_Humans, Metagenome, Diabetes Mellitus, Type 2, Microbiota, Bacteria, Cataract, Conjunctiva_
Our objective was to investigate differences in the ocular surface bacterial composition in cataract patients with and without type 2 diabetes (T2D).

[Source](https://doi.org/10.1167/tvst.12.6.1)

10.1167/tvst.12.6.1

---

## Longitudinal Gut Microbiota Dysbiosis Underlies Olanzapine-Induced Weight Gain.
 01.06.2023, _L Qian, X He, Y Liu, F Gao, W Lu, Y Fan, Y Gao, W Wang, F Zhu, Y Wang, X Ma_


Olanzapine is one of the most effective medicines available for stabilizing schizophrenia spectrum disorders. However, it has been reported to show the greatest propensity for inducing body weight gain and producing metabolic side effects, which cause a great burden in patients with psychiatric disorders. Since the gut microbiota has a profound impact on the initiation and development of metabolic diseases, we conducted a longitudinal study to explore its role in olanzapine-induced obesity and metabolic abnormalities. Female Sprague-Dawley rats were treated with different doses of olanzapine, and metabolic and inflammatory markers were measured. Olanzapine significantly induced body weight gain (up to a 2.1-fold change), which was accompanied by hepatic inflammation and increased plasma triglyceride levels (up to a 2.9-fold change), as well as gut microbiota dysbiosis. Subsequently, fuzzy c-means clustering was used to characterize three clusters of longitudinal trajectories for microbial fluctuations: (i) genera continuing to increase, (ii) genera continuing to decrease, and (iii) genera temporarily changing. Among them, Enterorhabdus (r = 0.38), Parasutterella (r = 0.43), and Prevotellaceae UCG-001 (r = 0.52) positively correlated with body weight gain. In addition, two MetaCyc metabolic pathways were identified as associated with olanzapine-induced body weight gain, including the superpathway of glucose and xylose degradation and the superpathway of l-threonine biosynthesis. In conclusion, we demonstrate that olanzapine can directly alter the gut microbiota and rapidly induce dysbiosis, which is significantly associated with body weight gain. This may suggest gut microbiota targets in future studies on metabolic abnormalities caused by olanzapine. IMPORTANCE Olanzapine is one of the most effective second-generation antipsychotics for stabilizing schizophrenia spectrum disorders. However, olanzapine has multiple drug-induced metabolic side effects, including weight gain. This study provides insight to the gut microbiota target in olanzapine-induced obesity. Specifically, we explored the longitudinal gut microbiota trajectories of female Sprague-Dawley rats undergoing olanzapine treatment. We showed that olanzapine treatment causes a dynamic alteration of gut microbiota diversity. Additionally, we identified three genera, Parasutterella, Enterorhabdus, and Prevotellaceae UCG-001, that may play an important role in olanzapine-induced obesity. In this case, the supply or removal of specific elements of the gut microbiota may represent a promising avenue for treatment of olanzapine-related metabolic side effects.

[Source](https://doi.org/10.1128/spectrum.00058-23)

10.1128/spectrum.00058-23

---

## Metagenome-mining indicates an association between bacteriocin presence and strain diversity in the infant gut.
 03.06.2023, _I Ormaasen, K Rudi, DB Diep, L Snipen_


_Humans, Infant, Bacteriocins, Metagenome, Gastrointestinal Microbiome_
Our knowledge about the ecological role of bacterial antimicrobial peptides (bacteriocins) in the human gut is limited, particularly in relation to their role in the diversification of the gut microbiota during early life. The aim of this paper was therefore to address associations between bacteriocins and bacterial diversity in the human gut microbiota. To investigate this, we did an extensive screening of 2564 healthy human gut metagenomes for the presence of predicted bacteriocin-encoding genes, comparing bacteriocin gene presence to strain diversity and age.

[Source](https://doi.org/10.1186/s12864-023-09388-0)

10.1186/s12864-023-09388-0

---

## The Microbiome in Advanced Melanoma: Where Are We Now?
 03.06.2023, _DD Fortman, D Hurd, D Davar_


This review summarizes recent data linking gut microbiota composition to ICI outcomes and gut microbiota-specific interventional clinical trials in melanoma.

[Source](https://doi.org/10.1007/s11912-023-01431-3)

10.1007/s11912-023-01431-3

---

## Human gut microbiome: A primer for the clinician.
 04.06.2023, _S Kedia, V Ahuja_


The human host gets tremendously influenced by a genetically and phenotypically distinct and heterogeneous constellation of microbial species-the human microbiome-the gut being one of the most densely populated and characterized site for these organisms. Microbiome science has advanced rapidly, technically with respect to the analytical methods and biologically with respect to its mechanistic influence in health and disease states. A clinician conducting a microbiome study should be aware of the nuances related to microbiome research, especially with respect to the technical and biological factors that can influence the interpretation of research outcomes. Hence, this review is an attempt to detail these aspects of the human gut microbiome, with emphasis on its determinants in a healthy state.

[Source](https://doi.org/10.1002/jgh3.12902)

10.1002/jgh3.12902

---

## Role of gut microbiota and bacterial metabolites in mucins of colorectal cancer.
 05.06.2023, _M Gu, W Yin, J Zhang, J Yin, X Tang, J Ling, Z Tang, W Yin, X Wang, Q Ni, Y Zhu, T Chen_


_Humans, Gastrointestinal Microbiome, Colorectal Neoplasms, Mucins, Bacteria, Intestinal Mucosa_
Colorectal cancer (CRC) is a major health burden, accounting for approximately 10% of all new cancer cases worldwide. Accumulating evidence suggests that the crosstalk between the host mucins and gut microbiota is associated with the occurrence and development of CRC. Mucins secreted by goblet cells not only protect the intestinal epithelium from microorganisms and invading pathogens but also provide a habitat for commensal bacteria. Conversely, gut dysbiosis results in the dysfunction of mucins, allowing other commensals and their metabolites to pass through the intestinal epithelium, potentially triggering host responses and the subsequent progression of CRC. In this review, we summarize how gut microbiota and bacterial metabolites regulate the function and expression of mucin in CRC and novel treatment strategies for CRC.

[Source](https://doi.org/10.3389/fcimb.2023.1119992)

10.3389/fcimb.2023.1119992

---

## Bacteriophage targeting microbiota alleviates non-alcoholic fatty liver disease induced by high alcohol-producing <em>Klebsiella pneumoniae</em>

[Source](https://www.nature.com/articles/s41467-023-39028-w)

---

## <em>Clostridium butyricum</em> improves cognitive dysfunction in ICV-STZ-induced Alzheimer’s disease mice via suppressing TLR4 signaling pathway through the gut-brain axis

[Source](https://doi.org/10.1371/journal.pone.0286086)

---

## Regulation of gut microbiota by vitamin C, vitamin E and β-carotene.
 01.06.2023, _XY Li, L Meng, L Shen, HF Ji_


_Vitamin E, Ascorbic Acid, beta Carotene, Gastrointestinal Microbiome, Vitamins, Antioxidants, Vitamin A, Vitamin K_
Vitamin C (VC), vitamin E (VE) and β-carotene (βC) are representative dietary antioxidants, which exist in daily diet and can increase the antioxidant capacity of body fluids, cells and tissues. The health benefits of vitamins like VC, VE and βC are widely demonstrated. Given that the strong associations between the gut microbiota and host health or a range of diseases has been extensively reported, it is important to explore the modulatory effects of known vitamins on the gut microbiota. Herein, this article reviews the effects of VC, VE and βC on the gut microbiota. Totally, 19 studies were included, of which eight were related to VC, nine to VE, and six to βC. Overall, VC, VE and βC can provide health benefits to the host by modulating the composition and metabolic activity of the gut microbiota, improving intestinal barrier function and maintaining the normal function of the immune system. Two perspectives are proposed for future studies: i) roles of known antioxidant activity of vitamins in regulating the gut microbiota and its molecular mechanism need to be further studied; ii) causal relationships between the regulatory effects of vitamins on gut microbiota and host health still remains to be further verified.

[Source](https://doi.org/10.1016/j.foodres.2023.112749)

10.1016/j.foodres.2023.112749

---

## Diversity of the Pacific Ocean coral reef microbiome.
 05.06.2023, _PE Galand, HJ Ruscheweyh, G Salazar, C Hochart, N Henry, BCC Hume, PH Oliveira, A Perdereau, K Labadie, C Belser, E Boissin, S Romac, J Poulain, G Bourdin, G Iwankow, C Moulin, EJ Armstrong, DA Paz-García, M Ziegler, S Agostini, B Banaigs, E Boss, C Bowler, C de Vargas, E Douville, M Flores, D Forcioli, P Furla, E Gilson, F Lombard, S Pesant, S Reynaud, OP Thomas, R Troublé, D Zoccola, CR Voolstra, RV Thurber, S Sunagawa, P Wincker, D Allemand, S Planes_


_Animals, Coral Reefs, Pacific Ocean, Anthozoa, Biodiversity, Microbiota, Fishes, Plankton_
Coral reefs are among the most diverse ecosystems on Earth. They support high biodiversity of multicellular organisms that strongly rely on associated microorganisms for health and nutrition. However, the extent of the coral reef microbiome diversity and its distribution at the oceanic basin-scale remains to be explored. Here, we systematically sampled 3 coral morphotypes, 2 fish species, and planktonic communities in 99 reefs from 32 islands across the Pacific Ocean, to assess reef microbiome composition and biogeography. We show a very large richness of reef microorganisms compared to other environments, which extrapolated to all fishes and corals of the Pacific, approximates the current estimated total prokaryotic diversity for the entire Earth. Microbial communities vary among and within the 3 animal biomes (coral, fish, plankton), and geographically. For corals, the cross-ocean patterns of diversity are different from those known for other multicellular organisms. Within each coral morphotype, community composition is always determined by geographic distance first, both at the island and across ocean scale, and then by environment. Our unprecedented sampling effort of coral reef microbiomes, as part of the Tara Pacific expedition, provides new insight into the global microbial diversity, the factors driving their distribution, and the biocomplexity of reef ecosystems.

[Source](https://doi.org/10.1038/s41467-023-38500-x)

10.1038/s41467-023-38500-x

---

## Effects of <em>Aeromonas hydrophila</em> infection on the intestinal microbiota, transcriptome, and metabolomic of common carp (<em>Cyprinus carpio</em>)

[Source](https://doi.org/10.1016/j.fsi.2023.108876)

---

## Genital tract microbiome dynamics are associated with time of <em>Chlamydia</em> infection in mice

[Source](https://doi.org/10.1038/s41598-023-36130-3)

---

## The core mangrove microbiome reveals shared taxa potentially involved in nutrient cycling and promoting host survival

[Source](https://doi.org/10.1186/s40793-023-00499-5)

---

## Formation and Analysis of Mono-species and Polymicrobial Oral Biofilms in Flow-Cell Models.
 06.06.2023, _J Neilands, G Svensäter, G Boisen, C Robertsson, C Wickström, JR Davies_


_Humans, Biofilms, Saliva, Periodontitis, Adhesins, Bacterial, Bacteria_
The oral microbiota, which is known to include at least 600 different bacterial species, is found on the teeth and mucosal surfaces as multi-species communities or biofilms. The oral surfaces are covered with a pellicle of proteins absorbed from saliva, and biofilm formation is initiated when primary colonizers, which express surface adhesins that bind to specific salivary components, attach to the oral tissues. Further development then proceeds through co-aggregation of additional species. Over time, the composition of oral biofilms, which varies between different sites throughout the oral cavity, is determined by a combination of environmental factors such as the properties of the underlying surface, nutrient availability and oxygen levels, and bacterial interactions within the community. A complex equilibrium between biofilm communities and the host is responsible for the maintenance of a healthy biofilm phenotype (eubiosis). In the face of sustained environmental perturbation, however, biofilm homeostasis can break down giving rise to dysbiosis, which is associated with the development of oral diseases such as caries and periodontitis.In vitro models have an important part to play in increasing our understanding of the complex processes involved in biofilm development in oral health and disease, and the requirements for experimental system, microbial complexity, and analysis techniques will necessarily vary depending on the question posed. In this chapter we describe some current and well-established methods used in our laboratory for studying oral bacteria in biofilm models which can be adapted to suit the needs of individual users.

[Source](https://doi.org/10.1007/978-1-0716-3243-7_2)

10.1007/978-1-0716-3243-7_2

---

## MAIRA: Protein-based Analysis of MinION Reads on a Laptop.
 06.06.2023, _C Bağcı, B Albrecht, DH Huson_


_High-Throughput Nucleotide Sequencing, Sequence Analysis, DNA, Software, Microcomputers, Microbiota_
Third-generation sequencing technologies are being increasingly used in microbiome research and this has given rise to new challenges in computational microbiome analysis. Oxford Nanopore's MinION is a portable sequencer that streams data that can be basecalled on-the-fly. Here we give an introduction to the MAIRA software, which is designed to analyze MinION sequencing reads from a microbiome sample, as they are produced in real-time, on a laptop. The software processes reads in batches and updates the presented analysis after each batch. There are two analysis steps: First, protein alignments are calculated to determine which genera might be present in a sample. When strong evidence for a genus is found, then, in a second step, a more detailed analysis is performed by aligning the reads against the proteins of all species in the detected genus. The program presents a detailed analysis of species, antibiotic resistance genes, and virulence factors.

[Source](https://doi.org/10.1007/978-1-0716-3072-3_11)

10.1007/978-1-0716-3072-3_11

---

## Recovery and Analysis of Long-Read Metagenome-Assembled Genomes.
 06.06.2023, _K Arumugam, I Bessarab, MAS Haryono, RBH Williams_


_Metagenome, Metagenomics, Microbiota, Sequence Analysis, DNA, Base Sequence, High-Throughput Nucleotide Sequencing_
The development of long-read nucleic acid sequencing is beginning to make very substantive impact on the conduct of metagenome analysis, particularly in relation to the problem of recovering the genomes of member species of complex microbial communities. Here we outline bioinformatics workflows for the recovery and characterization of complete genomes from long-read metagenome data and some complementary procedures for comparison of cognate draft genomes and gene quality obtained from short-read sequencing and long-read sequencing.

[Source](https://doi.org/10.1007/978-1-0716-3072-3_12)

10.1007/978-1-0716-3072-3_12

---

## Evaluation of 96-well high-throughput DNA extraction methods for 16S rRNA gene metabarcoding.
 31.05.2023, _MP Chapuis, L Benoit, M Galan_


Gaining meaningful insights into bacterial communities associated with animal hosts requires the provision of high-quality nucleic acids. Although many studies have compared DNA extraction methods for samples with low bacterial biomass (e.g. water) or specific PCR inhibitors (e.g. plants), DNA extraction bias in samples without inherent technical constraint (e.g. animal samples) has received little attention. Furthermore, there is an urgent need to identify a DNA extraction methods in a high-throughput format that decreases the cost and time for processing large numbers of samples. We here evaluated five DNA extraction protocols, using silica membrane-based spin columns and a 96-well microplate format and based on either mechanical or enzymatic lysis or a combination of both, using three bacterial mock communities and Illumina sequencing of the V4 region of the 16SrRNA gene. Our results showed that none of the DNA extraction methods fully eliminated bias associated with unequal lysis efficiencies. However, we identified a DNA extraction method with a lower bias for each mock community standard. Of these methods, those including an enzymatic lysis showed biases specific to some bacteria. Altogether, these results again demonstrate the importance of DNA extraction standardization to be able to compare the microbiome results of different samples. In this attempt, we advise for the use of the 96-well DNeasy Blood and Tissue kit (Qiagen) with a zirconia bead-beating procedure, which optimizes altogether the cost, handling time and bacteria-specific effects associated with enzymatic lysis.

[Source](https://doi.org/10.1111/1755-0998.13812)

10.1111/1755-0998.13812

---

