## Microbiome definition re-visited: old concepts and new challenges.
 01.07.2020, _G Berg, D Rybakova, D Fischer, T Cernava, MC Vergès, T Charles, X Chen, L Cocolin, K Eversole, GH Corral, M Kazou, L Kinkel, L Lange, N Lima, A Loy, JA Macklin, E Maguin, T Mauchline, R McClure, B Mitter, M Ryan, I Sarand, H Smidt, B Schelkle, H Roume, GS Kiran, J Selvin, RSC Souza, L van Overbeek, BK Singh, M Wagner, A Walsh, A Sessitsch, M Schloter_


The field of microbiome research has evolved rapidly over the past few decades and has become a topic of great scientific and public interest. As a result of this rapid growth in interest covering different fields, we are lacking a clear commonly agreed definition of the term "microbiome." Moreover, a consensus on best practices in microbiome research is missing. Recently, a panel of international experts discussed the current gaps in the frame of the European-funded MicrobiomeSupport project. The meeting brought together about 40 leaders from diverse microbiome areas, while more than a hundred experts from all over the world took part in an online survey accompanying the workshop. This article excerpts the outcomes of the workshop and the corresponding online survey embedded in a short historical introduction and future outlook. We propose a definition of microbiome based on the compact, clear, and comprehensive description of the term provided by Whipps et al. in 1988, amended with a set of novel recommendations considering the latest technological developments and research findings. We clearly separate the terms microbiome and microbiota and provide a comprehensive discussion considering the composition of microbiota, the heterogeneity and dynamics of microbiomes in time and space, the stability and resilience of microbial networks, the definition of core microbiomes, and functionally relevant keystone species as well as co-evolutionary principles of microbe-host and inter-species interactions within the microbiome. These broad definitions together with the suggested unifying concepts will help to improve standardization of microbiome studies in the future, and could be the starting point for an integrated assessment of data resulting in a more rapid transfer of knowledge from basic science into practice. Furthermore, microbiome standards are important for solving new challenges associated with anthropogenic-driven changes in the field of planetary health, for which the understanding of microbiomes might play a key role. Video Abstract.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00875-0)

10.1186/s40168-020-00875-0

---

## Large-scale metabolic interaction network of the mouse and human gut microbiota.
 01.07.2020, _R Lim, JJT Cabatbat, TLP Martin, H Kim, S Kim, J Sung, CM Ghim, PJ Kim_


The role of our gut microbiota in health and disease is largely attributed to the collective metabolic activities of the inhabitant microbes. A system-level framework of the microbial community structure, mediated through metabolite transport, would provide important insights into the complex microbe-microbe and host-microbe chemical interactions. This framework, if adaptable to both mouse and human systems, would be useful for mechanistic interpretations of the vast amounts of experimental data from gut microbiomes in murine animal models, whether humanized or not. Here, we constructed a literature-curated, interspecies network of the mammalian gut microbiota for mouse and human hosts, called NJC19. This network is an extensive data resource, encompassing 838 microbial species (766 bacteria, 53 archaea, and 19 eukaryotes) and 6 host cell types, interacting through 8,224 small-molecule transport and macromolecule degradation events. Moreover, we compiled 912 negative associations between organisms and metabolic compounds that are not transportable or degradable by those organisms. Our network may facilitate experimental and computational endeavors for the mechanistic investigations of host-associated microbial communities.

[Source](https://www.nature.com/articles/s41597-020-0516-5)

10.1038/s41597-020-0516-5

---

## Very early-life exposure to microbiota-induced TNF drives the maturation of neonatal pre-cDC1.
 17.06.2020, _A Köhler, S Delbauve, J Smout, D Torres, V Flamand_


Induction of immune protection against pathogens is particularly crucial during the neonatal period dominated by anti-inflammatory and tolerance immunity. The preclinical study was carried out to determine whether environmental factors such as microbiota may influence early life immunity by impacting the development and the functional maturation of precursors of type 1 conventional dendritic cells (pre-cDC1), endowed with regulatory properties.

[Source](https://gut.bmj.com/content/early/2020/06/15/gutjnl-2019-319700)

10.1136/gutjnl-2019-319700

---

## Arrhythmic Gut Microbiome Signatures Predict Risk of Type 2 Diabetes

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312820303437)

---

## Ranking microbiome variance in inflammatory bowel disease: a large longitudinal intercontinental study.
 15.06.2020, _AG Clooney, J Eckenberger, E Laserna-Mendieta, KA Sexton, MT Bernstein, K Vagianos, M Sargent, FJ Ryan, C Moran, D Sheehan, RD Sleator, LE Targownik, CN Bernstein, F Shanahan, MJ Claesson_


The microbiome contributes to the pathogenesis of inflammatory bowel disease (IBD) but the relative contribution of different lifestyle and environmental factors to the compositional variability of the gut microbiota is unclear.

[Source](https://gut.bmj.com/content/early/2020/06/11/gutjnl-2020-321106)

10.1136/gutjnl-2020-321106

---

## Cholesterol Metabolism by Uncultured Human Gut Bacteria Influences Host Cholesterol Level.
 16.06.2020, _DJ Kenny, DR Plichta, D Shungin, N Koppel, AB Hall, B Fu, RS Vasan, SY Shaw, H Vlamakis, EP Balskus, RJ Xavier_


The human microbiome encodes extensive metabolic capabilities, but our understanding of the mechanisms linking gut microbes to human metabolism remains limited. Here, we focus on the conversion of cholesterol to the poorly absorbed sterol coprostanol by the gut microbiota to develop a framework for the identification of functional enzymes and microbes. By integrating paired metagenomics and metabolomics data from existing cohorts with biochemical knowledge and experimentation, we predict and validate a group of microbial cholesterol dehydrogenases that contribute to coprostanol formation. These enzymes are encoded by ismA genes in a clade of uncultured microorganisms, which are prevalent in geographically diverse human cohorts. Individuals harboring coprostanol-forming microbes have significantly lower fecal cholesterol levels and lower serum total cholesterol with effects comparable to those attributed to variations in lipid homeostasis genes. Thus, cholesterol metabolism by these microbes may play important roles in reducing intestinal and serum cholesterol concentrations, directly impacting human health.

[Source](https://www.sciencedirect.com/science/article/pii/S193131282030295X)

10.1016/j.chom.2020.05.013

---

## Amending microbiota by targeting intestinal inflammation with TNF blockade attenuates development of colorectal cancer

[Source](https://www.nature.com/articles/s43018-020-0078-7)

---

## A Systematic Analysis of Mosquito-Microbiome Biosynthetic Gene Clusters Reveals Antimalarial Siderophores that Reduce Mosquito Reproduction Capacity

[Source](https://www.sciencedirect.com/science/article/abs/pii/S2451945620302270)

---

## Microbial succession during the transition from active to inactive stages of deep-sea hydrothermal vent sulfide chimneys

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00851-8)

---

## Phase-variable capsular polysaccharides and lipoproteins modify bacteriophage susceptibility in Bacteroides thetaiotaomicron

[Source](https://www.nature.com/articles/s41564-020-0746-5)

---

## ATLAS: a Snakemake workflow for assembly, annotation, and genomic binning of metagenome sequence data

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03585-4)

---

## High-Throughput Stool Metaproteomics: Method and Application to Human Specimens

[Source](https://msystems.asm.org/content/5/3/e00200-20)

---

## Interindividual Variation in Dietary Carbohydrate Metabolism by Gut Bacteria Revealed with Droplet Microfluidic Culture

[Source](https://msystems.asm.org/content/5/3/e00864-19.abstract)

---

