## Blood type and the microbiome- untangling a complex relationship with lessons from pathogens.
 14.07.2020, _KL Arnolds, CG Martin, CA Lozupone_


The complex communities of microbes that constitute the human microbiome are influenced by host and environmental factors. Here, we address how a fundamental aspect of human biology, blood type, contributes to shaping this microscopic ecosystem. Although this question remains largely unexplored, we glean insights from decades of work describing relationships between pathogens and blood type. The bacterial strategies, molecular mechanisms, and host responses that shaped those relationships may parallel those that characterize how blood type and commensals interact. Understanding these nuanced interactions will expand our capacity to analyze and manipulate the human microbiome.

[Source](https://www.sciencedirect.com/science/article/pii/S1369527420300801)

10.1016/j.mib.2020.06.008

---

## <em>In vivo</em> synthesis of bacterial amyloid curli contributes to joint inflammation during <em>S</em>. Typhimurium infection

[Source](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1008591)

---

## Proteomic advances in salivary diagnostics.
 14.07.2020, _E Pappa, K Vougas, J Zoidakis, H Vastardis_


Saliva is identified as functional equivalent to serum, reflecting the physiological state of the body, as well as hormonal, emotional, nutritional and metabolic alterations. The application of mass spectrometry based approaches has allowed a thorough characterization of the saliva proteome and led to the discovery of putative biomarkers. Several salivary biomarkers have been recently explored as potentially useful screening tools in patients diagnosed with metabolic disorders. In this review, we provide an overview of saliva proteomics studies, with a focus on diabetes, and we explore the evidence for the utility of well identified markers for the diagnosis and monitoring of the disease. Emerging approaches in salivary diagnostics that may significantly advance the field of diabetes research are also highlighted.

[Source](https://www.sciencedirect.com/science/article/pii/S1570963920301412)

10.1016/j.bbapap.2020.140494

---

## Tumor Microbiome Diversity and Composition Influence Pancreatic Cancer Outcomes.
 13.06.2020, _E Riquelme, Y Zhang, L Zhang, M Montiel, M Zoltan, W Dong, P Quesada, I Sahin, V Chandra, A San Lucas, P Scheet, H Xu, SM Hanash, L Feng, JK Burks, KA Do, CB Peterson, D Nejman, CD Tzeng, MP Kim, CL Sears, N Ajami, J Petrosino, LD Wood, A Maitra, R Straussman, M Katz, JR White, R Jenq, J Wargo, F McAllister_


_Adult, Aged, Animals, Bacteria, Carcinoma, Pancreatic Ductal, Cell Line, Tumor, Cohort Studies, Fecal Microbiota Transplantation, Feces, Female, Gastrointestinal Microbiome, Humans, Male, Mice, Mice, Inbred C57BL, Middle Aged, Pancreatic Neoplasms, RNA, Ribosomal, 16S, Sequence Analysis, RNA, Survival Rate_
Most patients diagnosed with resected pancreatic adenocarcinoma (PDAC) survive less than 5 years, but a minor subset survives longer. Here, we dissect the role of the tumor microbiota and the immune system in influencing long-term survival. Using 16S rRNA gene sequencing, we analyzed the tumor microbiome composition in PDAC patients with short-term survival (STS) and long-term survival (LTS). We found higher alpha-diversity in the tumor microbiome of LTS patients and identified an intra-tumoral microbiome signature (Pseudoxanthomonas-Streptomyces-Saccharopolyspora-Bacillus clausii) highly predictive of long-term survivorship in both discovery and validation cohorts. Through human-into-mice fecal microbiota transplantation (FMT) experiments from STS, LTS, or control donors, we were able to differentially modulate the tumor microbiome and affect tumor growth as well as tumor immune infiltration. Our study demonstrates that PDAC microbiome composition, which cross-talks to the gut microbiome, influences the host immune response and natural history of the disease.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0016508520349295)

10.1016/j.cell.2019.07.008

---

## The Influence of Gut and Tumor Microbiome on Pancreatic Cancer Outcomes

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0016508520349295)

---

## Inulin alleviates hypersaline-stress induced oxidative stress and dysbiosis of gut microbiota in Nile tilapia (Oreochromis niloticus)

[Source](https://www.sciencedirect.com/science/article/pii/S0044848620314782)

---

## Microbial Community Composition and Function in Sediments from the Pearl River Mouth Basin

[Source](https://link.springer.com/article/10.1007/s11802-020-4225-7)

---

## Protist Interactions and Seasonal Dynamics in the Coast of Yantai, Northern Yellow Sea of China as Revealed by Metabarcoding

[Source](https://link.springer.com/article/10.1007/s11802-020-4461-x)

---

## Substrate regulation leads to differential responses of microbial ammonia-oxidizing communities to ocean warming

[Source](https://www.nature.com/articles/s41467-020-17366-3?utm_source=ncomms_etoc&utm_medium=email&utm_campaign=toc_41467_11_1&utm_content=20200715&WT.ec_id=NCOMMS-20200715&sap-outbound-id=4057EC46EB3D8DB950200D2F9993DE89BE0E491A)

---

## Ecogenomics of groundwater viruses suggests niche differentiation linked to specific environmental tolerance

[Source](https://www.biorxiv.org/content/10.1101/2020.07.14.203604v1.abstract?%3Fcollection=)

---

## Can interaction between silicon and non–rhizobial bacteria benefit in improving nodulation and nitrogen fixation in salinity–stressed legumes? A review

[Source](https://www.sciencedirect.com/science/article/pii/S2452219820301415)

---

## Reductive soil disinfestation incorporated with organic residue combination significantly improves soil microbial activity and functional diversity than sole residue incorporation

[Source](https://link.springer.com/article/10.1007/s00253-020-10778-7)

---

## Microbial clicks and combobulation: integrating microbes without passports and visas for managing soilborne diseases

[Source](https://www.tandfonline.com/doi/abs/10.1080/03235408.2020.1791464)

---

## Arbuscular mycorrhizal fungi improve mineral nutrition and tolerance of olive tree to Verticillium wilt

[Source](https://www.tandfonline.com/doi/abs/10.1080/03235408.2020.1792603)

---

## Responses of microbial communities and metabolic activities in the rhizosphere during phytoremediation of Cd-contaminated soil

[Source](https://www.sciencedirect.com/science/article/pii/S0147651320307971)

---

## MnO<sub>2</sub>/tourmaline composites as efficient cathodic catalysts enhance bioelectroremediation of contaminated river sediment and shape biofilm microbiomes in sediment microbial fuel cells

[Source](https://www.sciencedirect.com/science/article/pii/S0926337320307463)

---

## Reductive soil disinfestation incorporated with organic residue combination significantly improves soil microbial activity and functional diversity than sole residue incorporation

[Source](https://link.springer.com/article/10.1007/s00253-020-10778-7)

---

## Revegetation approach and plant identity unequally affect structure, ecological network and function of soil microbial community in a highly acidified mine tailings pond

[Source](https://www.sciencedirect.com/science/article/pii/S0048969720343175)

---

## Nitrogen enrichment stimulates wetland plant responses whereas salt amendments alter sediment microbial communities and biogeochemical responses

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0235225)

---

## Identification of sulfate-reducing and methanogenic microbial taxa in anaerobic bioreactors from industrial wastewater treatment plants using next-generation sequencing and gene clone library analyses.
 13.07.2020, _K Jantharadej, W Mhuantong, T Limpiyakorn, S Mongkolsuk, K Sirikanchana, BB Suwannasilp_


An understanding of microbial communities present in anaerobic bioreactors can strongly facilitate the development of approaches to control undesirable microorganisms, such as sulfate-reducing bacteria (SRB), in the system. In this study, overall microbial communities present in anaerobic bioreactors from seven industrial wastewater treatment plants (including food, pulp and paper industries) were investigated using 16S rRNA gene amplicon sequencing (MiSeq, Illumina). The dominant methanogens identified in the anaerobic bioreactors treating industrial wastewater were Methanobacterium and Methanosaeta; Methanospirillum was a predominant methanogen in the anaerobic sludge digester. Hydrogenotrophic and acetoclastic methanogens were detected at similar relative abundances in the anaerobic covered lagoons treating starch wastewater, whereas hydrogenotrophic methanogens were the predominant methanogens present in the sludge digester. SRB communities were further investigated using dsrB gene clone libraries. The results indicated the presence of SRB, such as uncultured Desulfobulbus sp., Syntrophobacter fumaroxidans, Syntrophorhabdus sp. PtaB.Bin027, and Desulfovibrio fructosivarans JJ. Incomplete-oxidizing SRB were the predominant SRB in all of the anaerobic bioreactors treating wastewater. In contrast, similar relative abundances of complete and incomplete-oxidizing SRB were observed in the sludge digester. The results of this study can further facilitate the development of SRB-controlling strategies to improve the efficiency of wastewater treatment.

[Source](https://www.tandfonline.com/doi/abs/10.1080/10934529.2020.1789409)

10.1080/10934529.2020.1789409

---

## Determination of the inhibitory concentration level of fat, oil, and grease (FOG) towards bacterial and archaeal communities in anaerobic digestion

[Source](https://www.sciencedirect.com/science/article/pii/S1364032120303233)

---

## PARGT: a software tool for predicting antimicrobial resistance in bacteria.
 08.07.2020, _AS Chowdhury, DR Call, SL Broschat_


With the ever-increasing availability of whole-genome sequences, machine-learning approaches can be used as an alternative to traditional alignment-based methods for identifying new antimicrobial-resistance genes. Such approaches are especially helpful when pathogens cannot be cultured in the lab. In previous work, we proposed a game-theory-based feature evaluation algorithm. When using the protein characteristics identified by this algorithm, called 'features' in machine learning, our model accurately identified antimicrobial resistance (AMR) genes in Gram-negative bacteria. Here we extend our study to Gram-positive bacteria showing that coupling game-theory-identified features with machine learning achieved classification accuracies between 87% and 90% for genes encoding resistance to the antibiotics bacitracin and vancomycin. Importantly, we present a standalone software tool that implements the game-theory algorithm and machine-learning model used in these studies.

[Source](https://search.proquest.com/openview/c2f241520d44b2f87fb458e65d890e7a/1?pq-origsite=gscholar&cbl=2041939)

10.1038/s41598-020-67949-9

---

## Classification of 16S rRNA reads is improved using a niche-specific database constructed by near-full length sequencing.
 13.07.2020, _PR Myer, TG McDaneld, LA Kuehn, KD Dedonder, MD Apley, SF Capik, BV Lubbers, GP Harhay, DM Harhay, JW Keele, MT Henniger, BA Clemmons, TPL Smith_


Surveys of microbial populations in environmental niches of interest often utilize sequence variation in the gene encoding the ribosomal small subunit (the 16S rRNA gene). Generally, these surveys target the 16S genes using semi-degenerate primers to amplify portions of a subset of bacterial species, sequence the amplicons in bulk, and assign to putative taxonomic categories by comparison to databases purporting to connect specific sequences in the main variable regions of the gene to specific organisms. Due to sequence length constraints of the most popular bulk sequencing platforms, the primers selected amplify one to three of the nine variable regions, and taxonomic assignment is based on relatively short stretches of sequence (150-500 bases). We demonstrate that taxonomic assignment is improved through reduced unassigned reads by including a survey of near-full-length sequences specific to the target environment, using a niche of interest represented by the upper respiratory tract (URT) of cattle. We created a custom Bovine URT database from these longer sequences for assignment of shorter, less expensive reads in comparisons of the upper respiratory tract among individual animals. This process improves the ability to detect changes in the microbial populations of a given environment, and the accuracy of defining the content of that environment at increasingly higher taxonomic resolution.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0235498)

10.1371/journal.pone.0235498

---

