## Microbiome Forum FOR EVERYONE

[Source](https://www.microbiome-vif.org/)

---

## Mechanism of Dyspnea during Exercise in Children with Corrected Congenital Heart Disease.
 24.01.2022, _M Chlif, MM Ammar, NB Said, L Sergey, S Ahmaidi, F Alassery, H Hamam_


_Child, Dyspnea, Exercise, Exercise Test, Heart Defects, Congenital, Humans, Oxygen Consumption_
This study will evaluate cardiorespiratory and peripheral muscle function and their relationship with subjective dyspnea threshold after the surgical correction of congenital heart disease in children. Thirteen children with surgically repaired congenital heart disease were recruited. Each participant performed an incremental exercise test on a cycle ergometer until exhaustion. Gas exchanges were continuously sampled to measure the maximal aerobic parameters and ventilatory thresholds. The functional capacity of the subjects was assessed with a 6 min walk test. At the end of the exercise test, isokinetic Cybex Norm was used to evaluate the strength and endurance of the knee extensor muscle in the leg. Dyspnea was subjectively scored with a visual analog scale during the last 15 s of each exercise step. Oxygen consumption measured at the dyspnea score/VO2 relationship located at the dyspnea threshold, at which dyspnea suddenly increased. Results: The maximal and submaximal values of the parameters describing the exercise and the peripheral muscular performances were: VO2 Peak: 33.8 ± 8.9 mL·min-1·kg-1; HR: 174 ± 9 b·min-1; VEmax: 65.68 ± 15.9 L·min-1; P max: 117 ± 27 W; maximal voluntary isometric force MVIF: 120.8 ± 41.9 N/m; and time to exhaustion Tlim: 53 ± 21 s. Oxygen consumption measured at the dyspnea threshold was related to VO2 Peak (R2 = 0.74; p &lt; 0.01), Tlim (R2 = 0.78; p &lt; 0.01), and the distance achieved during the 6MWT (R2 = 0.57; p &lt; 0.05). Compared to the theoretical maximal values for the power output, VO2, and HR, the surgical correction did not repair the exercise performance. After the surgical correction of congenital heart disease, exercise performance was impeded by alterations of the cardiorespiratory function and peripheral local factors. A subjective evaluation of the dyspnea threshold is a reliable criterion that allows the prediction of exercise capacity in subjects suffering from congenital heart disease.

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

10.3390/ijerph19010099

---

## #MVIF

[Source](https://twitter.com/hashtag/MVIF?src=hash&ref_src=twsrc%5Etfw)

---

## @MicroRocket

[Source](https://twitter.com/MicroRocket?ref_src=twsrc%5Etfw)

---

## @tejpalgill

[Source](https://twitter.com/tejpalgill?ref_src=twsrc%5Etfw)

---

## @Brooke_C_Wilson

[Source](https://twitter.com/Brooke_C_Wilson?ref_src=twsrc%5Etfw)

---

## @ameliamcg

[Source](https://twitter.com/ameliamcg?ref_src=twsrc%5Etfw)

---

## @SuEgan4

[Source](https://twitter.com/SuEgan4?ref_src=twsrc%5Etfw)

---

## https://t.co/MzsTYidE6J

[Source](https://t.co/MzsTYidE6J)

---

## March 28, 2022

[Source](https://twitter.com/MicrobiomeVIF/status/1508364474415263751?ref_src=twsrc%5Etfw)

---

## Global Distribution of mcr Gene Variants in 214K Metagenomic Samples

[Source](https://journals.asm.org/doi/10.1128/msystems.00105-22)

---

## Symposium Review: After the storm—Perspectives on the taxonomy of<em> Lactobacillaceae</em> 

[Source](https://www.sciencedirect.com/science/article/pii/S2666910222000503)

---

## Assessing the Effect of Smokeless Tobacco Consumption on Oral Microbiome in Healthy and Oral Cancer Patients

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2022.841465/full)

---

## https://t.co/jrz3zlJF20

[Source](https://t.co/jrz3zlJF20)

---

## @vks_mBioSys

[Source](https://twitter.com/vks_mBioSys?ref_src=twsrc%5Etfw)

---

## @ashoks773

[Source](https://twitter.com/ashoks773?ref_src=twsrc%5Etfw)

---

## April 1, 2022

[Source](https://twitter.com/VishnuPrasoodan/status/1509796156674809860?ref_src=twsrc%5Etfw)

---

## Unravelling specific diet and gut microbial contributions to inflammatory bowel disease

[Source](https://www.biorxiv.org/content/10.1101/2022.04.03.486886v1)

---

## Effects of caloric restriction on the gut microbiome are linked with immune senescence

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01249-4)

---

## Tap water as a natural vehicle for microorganisms shaping the human gut microbiome

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.15988)

---

## Diet-gut microbiota interactions on cardiovascular disease

[Source](https://www.sciencedirect.com/science/article/pii/S2001037022001039)

---

## Robust Variation in Infant Gut Microbiome Assembly Across a Spectrum of Lifestyles

[Source](https://www.biorxiv.org/content/10.1101/2022.03.30.486467v1)

---

## https://t.co/hYDKUuczdw

[Source](https://t.co/hYDKUuczdw)

---

## April 3, 2022

[Source](https://twitter.com/MattagenOlmics/status/1510742683781763078?ref_src=twsrc%5Etfw)

---

## Extensive metagenomic analysis of the porcine gut resistome to identify indicators reflecting antimicrobial resistance

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01241-y)

---

## Modulation of the gut microbiome by novel synthetic glycans for the production of propionate and the reduction of cardiometabolic risk factors

[Source](https://www.biorxiv.org/content/10.1101/2022.04.04.487010v1)

---

## The phyllosphere microbiome shifts toward combating melanose pathogen

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01234-x)

---

## Near-neutral pH increased n-caprylate production in a microbiome with product inhibition of methanogenesis

[Source](https://www.biorxiv.org/content/10.1101/2022.04.01.486710v1)

---

## Next-generation sequencing: insights to advance clinical investigations of the microbiome.
 04.04.2022, _CR Wensel, JL Pluznick, SL Salzberg, CL Sears_


_High-Throughput Nucleotide Sequencing, Humans, Metagenomics, Microbiota, RNA, Ribosomal, 16S, Sequence Analysis, RNA_
Next-generation sequencing (NGS) technology has advanced our understanding of the human microbiome by allowing for the discovery and characterization of unculturable microbes with prediction of their function. Key NGS methods include 16S rRNA gene sequencing, shotgun metagenomic sequencing, and RNA sequencing. The choice of which NGS methodology to pursue for a given purpose is often unclear for clinicians and researchers. In this Review, we describe the fundamentals of NGS, with a focus on 16S rRNA and shotgun metagenomic sequencing. We also discuss pros and cons of each methodology as well as important concepts in data variability, study design, and clinical metadata collection. We further present examples of how NGS studies of the human microbiome have advanced our understanding of human disease pathophysiology across diverse clinical contexts, including the development of diagnostics and therapeutics. Finally, we share insights as to how NGS might further be integrated into and advance microbiome research and clinical care in the coming years.

[Source](https://www.jci.org/articles/view/154944)

10.1172/JCI154944

---

## Metagenomics-enabled microbial surveillance.
 05.04.2022, _KKK Ko, KR Chng, N Nagarajan_


_Animals, COVID-19, Humans, Metagenomics, Pandemics, Viruses, Zoonoses_
Lessons learnt from the COVID-19 pandemic include increased awareness of the potential for zoonoses and emerging infectious diseases that can adversely affect human health. Although emergent viruses are currently in the spotlight, we must not forget the ongoing toll of morbidity and mortality owing to antimicrobial resistance in bacterial pathogens and to vector-borne, foodborne and waterborne diseases. Population growth, planetary change, international travel and medical tourism all contribute to the increasing frequency of infectious disease outbreaks. Surveillance is therefore of crucial importance, but the diversity of microbial pathogens, coupled with resource-intensive methods, compromises our ability to scale-up such efforts. Innovative technologies that are both easy to use and able to simultaneously identify diverse microorganisms (viral, bacterial or fungal) with precision are necessary to enable informed public health decisions. Metagenomics-enabled surveillance methods offer the opportunity to improve detection of both known and yet-to-emerge pathogens.

[Source](https://www.nature.com/articles/s41564-022-01089-w)

10.1038/s41564-022-01089-w

---

## Bioinformatics Detection of Modulators Controlling Splicing Factor-Dependent Intron Retention in the Human Brain.
 07.04.2022, _SX Chen, E Simpson, JL Reiter, Y Liu_


Alternative RNA splicing is an important means of genetic control and transcriptome diversity. However, when alternative splicing events are studied independently, coordinated splicing modulated by common factors is often not recognized. As a result, the molecular mechanisms of how splicing regulators promote or repress splice site recognition in a context-dependent manner are not well understood. The functional coupling between multiple gene regulatory layers suggests that splicing is modulated by additional genetic or epigenetic components. Here, we developed a bioinformatics approach to identify causal modulators of splicing activity based on the variation of gene expression in large RNA sequencing datasets. We applied this approach in a neurological context with hundreds of dorsolateral prefrontal cortex samples. Our model is strengthened with the incorporation of genetic variants to impute gene expression in a Mendelian randomization-based approach. We identified novel modulators of the splicing factor SRSF1, including UIMC1 and the long noncoding RNA CBR3-AS1, that function over dozens of SRSF1 intron retention splicing targets. This strategy can be widely used to identify modulators of RNA-binding proteins involved in tissue-specific alternative splicing. This article is protected by copyright. All rights reserved.

[Source](https://twitter.com/hashtag/bioinformatics?src=hash&ref_src=twsrc%5Etfw)

10.1002/humu.24379

---

## #bioinformatics

[Source](https://twitter.com/hashtag/bioinformatics?src=hash&ref_src=twsrc%5Etfw)

---

## Reimagining diversity in multiple myeloma clinical trials.
 07.04.2022, _D Habr, M Corsaro_


Despite evidence that multiple myeloma disproportionately impacts ethnic minority groups and elderly patients, they remain underrepresented in therapeutic clinical trials. This commentary summarizes potential strategies that could be adopted by the pharmaceutical industry to ensure diversity in clinical trials of emerging therapies. These could include building trust through tailored education, optimizing awareness and exposure to clinical trial opportunities, ensuring access and extending outreach, and promoting clinical trial diversity by design. This article is protected by copyright. All rights reserved.

[Source](https://twitter.com/hashtag/training?src=hash&ref_src=twsrc%5Etfw)

10.1002/hon.2997

---

## #training

[Source](https://twitter.com/hashtag/training?src=hash&ref_src=twsrc%5Etfw)

---

## Nondestructive optical feedback systems for use during infrared laser sealing of blood vessels.
 07.04.2022, _NC Giglio, NM Fried_


High-power infrared lasers are capable of sealing blood vessels during surgery. A real-time diagnostic feedback system utilizing diffuse optical transmission is characterized by nondestructive identification of vessel seals.

[Source](https://twitter.com/hashtag/materials?src=hash&ref_src=twsrc%5Etfw)

10.1002/lsm.23548

---

## #materials

[Source](https://twitter.com/hashtag/materials?src=hash&ref_src=twsrc%5Etfw)

---

## @ISBSIB

[Source](https://twitter.com/ISBSIB?ref_src=twsrc%5Etfw)

---

## GNN-Surrogate: A Hierarchical and Adaptive Graph Neural Network for Parameter Space Exploration of Unstructured-Mesh Ocean Simulations.
 07.04.2022, _N Shi, J Xu, SW Wurster, H Guo, J Woodring, LP Van Roekel, HW Shen_


We propose GNN-Surrogate, a graph neural network-based surrogate model to explore the parameter space of ocean climate simulations. Parameter space exploration is important for domain scientists to understand the influence of input parameters (e.g., wind stress) on the simulation output (e.g., temperature). The exploration requires scientists to exhaust the complicated parameter space by running a batch of computationally expensive simulations. Our approach improves the efficiency of parameter space exploration with a surrogate model that predicts the simulation outputs accurately and efficiently. Specifically, GNN-Surrogate predicts the output field with given simulation parameters so scientists can explore the simulation parameter space with visualizations from user-specified visual mappings. Moreover, our graph-based techniques are designed for unstructured meshes, making the exploration of simulation outputs on irregular grids efficient. For efficient training, we generate hierarchical graphs and use adaptive resolutions. We give quantitative and qualitative evaluations on the MPAS-Ocean simulation to demonstrate the effectiveness and efficiency of GNN-Surrogate. Source code is publicly available at https://github.com/trainsn/GNN-Surrogate.

[Source](https://twitter.com/hashtag/github?src=hash&ref_src=twsrc%5Etfw)

10.1109/TVCG.2022.3165345

---

## #github

[Source](https://twitter.com/hashtag/github?src=hash&ref_src=twsrc%5Etfw)

---

## https://t.co/xtRon0hIzP

[Source](https://t.co/xtRon0hIzP)

---

## March 31, 2022

[Source](https://twitter.com/geertvangeest/status/1509559476839497728?ref_src=twsrc%5Etfw)

---

## Software testing in microbial bioinformatics: a call to action

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000790)

---

## Machine Learning and Deep Learning Applications in Metagenomic Taxonomy and Functional Annotation

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2022.811495/full)

---

## Unifying the known and unknown microbial coding sequence space

[Source](https://elifesciences.org/articles/67667)

---

## NSURP

[Source](https://nsurp.org/)

---

## @the_real_KVD

[Source](https://twitter.com/the_real_KVD?ref_src=twsrc%5Etfw)

---

## @microbeMinded

[Source](https://twitter.com/microbeMinded?ref_src=twsrc%5Etfw)

---

## @BenjaminGVincen

[Source](https://twitter.com/BenjaminGVincen?ref_src=twsrc%5Etfw)

---

## https://t.co/W2nLX6lwwJ

[Source](https://t.co/W2nLX6lwwJ)

---

## April 5, 2022

[Source](https://twitter.com/NSURPorg/status/1511178781951619074?ref_src=twsrc%5Etfw)

---

