## Sequence variant analysis reveals poor correlations in microbial taxonomic abundance between humans and mice after gnotobiotic transfer.
 21.12.2020, _F Fouladi, EM Glenny, EC Bulik-Sullivan, MCB Tsilimigras, M Sioda, SA Thomas, Y Wang, Z Djukic, Q Tang, LM Tarantino, CM Bulik, AA Fodor, IM Carroll_


_Animals, Fecal Microbiota Transplantation, Feces, Gastrointestinal Microbiome, Germ-Free Life, Humans, Mice, Microbiota, RNA, Ribosomal, 16S_
Transplanting human gut microbiotas into germ-free (GF) mice is a popular approach to disentangle cause-and-effect relationships between enteric microbes and disease. Algorithm development has enabled sequence variant (SV) identification from 16S rRNA gene sequence data. SV analyses can identify which donor taxa colonize recipient GF mice, and how SV abundance in humans is replicated in these mice. Fecal microbiotas from 8 human subjects were used to generate 77 slurries, which were transplanted into 153 GF mice. Strong correlations between fecal and slurry microbial communities were observed; however, only 42.15 ± 9.95% of SVs successfully transferred from the donor to the corresponding recipient mouse. Firmicutes had a particularly low transfer rate and SV abundance was poorly correlated between donor and recipient pairs. Our study confirms human fecal microbiotas colonize formerly GF mice, but the engrafted community only partially resembles the input human communities. Our findings emphasize the importance of reporting a standardized transfer rate and merit the exploration of other animal models or in silico tools to understand the relationships between human gut microbiotas and disease.

[Source](https://www.nature.com/articles/s41396-020-0645-z)

10.1038/s41396-020-0645-z

---

## Application of ecological and evolutionary theory to microbiome community dynamics across systems – McDonald, Marchesi &amp; Koskella

[Source](https://royalsocietypublishing.org/doi/10.1098/rspb.2020.2886)

---

## Salivary and Urinary Metabolome in Pediatric Obesity and Metabolic Syndrome

[Source](https://link.springer.com/chapter/10.1007/978-3-030-53370-0_19)

---

## Cesarean delivery is associated with decreased beneficial gut microbes and increased pathobionts and butyrate excretion: a longitudinal study over the first year of life.

[Source](https://obgyn.onlinelibrary.wiley.com/doi/10.1111/1471-0528.16633)

---

## Future directions for studying resilience of the oral ecosystem

[Source](https://www.nature.com/articles/s41415-020-2407-8)

---

## Maternal diet alters human milk oligosaccharide composition with implications for the milk metagenome

[Source](https://www.nature.com/articles/s41598-020-79022-6)

---

## Mapping the Human Milk Microbiome: Impetus for a Long-Awaited Renaissance in Maternal and Infant Nutrition Research?

[Source](https://academic.oup.com/jn/advance-article/doi/10.1093/jn/nxaa373/6039459)

---

## Breast Milk Modulates Transgenerational Immune Inheritance

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0092867420306255)

---

## An Immunologic Mode of Multigenerational Transmission Governs a Gut Treg Setpoint

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0092867420304931)

---

## The Lung Microbiome and Pneumonia

[Source](https://academic.oup.com/jid/advance-article/doi/10.1093/infdis/jiaa702/6039519)

---

## Microbiome: A Supportive or a Leading Actor in Lung Cancer?

[Source](https://www.karger.com/Article/FullText/511556)

---

## Skin and Gut Microbiome in Psoriasis: Gaining Insight Into the Pathophysiology of It and Finding Novel Therapeutic Strategies

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.589726/full)

---

## An association between chronic widespread pain and the gut microbiome

[Source](https://academic.oup.com/rheumatology/advance-article/doi/10.1093/rheumatology/keaa847/6040753)

---

## The Gut Microbiome as a Reservoir for Antimicrobial Resistance

[Source](https://academic.oup.com/jid/advance-article/doi/10.1093/infdis/jiaa497/6039355)

---

## Dietary modification of gut microbial metabolism protects the kidney

[Source](https://www.nature.com/articles/s41581-020-00364-5)

---

## Gut Microbiome Imbalance and Neuroinflammation: Impact of COVID‐19 on Parkinson’s Disease

[Source](https://onlinelibrary.wiley.com/doi/10.1002/mds.28231)

---

## US nativity and dietary acculturation impact the gut microbiome in a diverse US population

[Source](https://www.nature.com/articles/s41396-020-0630-6)

---

## Gut Microbiota Status in COVID-19: An Unrecognized Player?

[Source](https://www.frontiersin.org/articles/10.3389/fcimb.2020.576551/full)

---

## Honeybee gut microbiota modulates host behaviors and neurological processes

[Source](https://www.biorxiv.org/content/10.1101/2020.12.19.423587v1.abstract)

---

## Honeybee genetics shape the strain-level structure of gut microbiota in social transmission

[Source](https://www.biorxiv.org/content/10.1101/2020.12.17.423353v1.abstract)

---

## Description of the microbiota in epidermal mucus and skin of sharks (Ginglymostoma cirratum and Negaprion brevirostris) and one stingray (Hypanus americanus)

[Source](https://peerj.com/articles/10240/)

---

## Roundup causes embryonic development failure and alters metabolic pathways and gut microbiota functionality in non-target species

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00943-5)

---

## Influence of vintage, geographic location and cultivar on the structure of microbial communities associated with the grapevine rhizosphere in vineyards of San Juan Province, Argentina

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0243848)

---

## Restoring degraded microbiome function with self-assembled communities

[Source](https://academic.oup.com/femsec/article/96/12/fiaa225/5956484)

---

## Combining whole-genome shotgun sequencing and rRNA gene amplicon analyses to improve detection of microbe-microbe interaction networks in plant leaves.
 21.12.2020, _J Regalado, DS Lundberg, O Deusch, S Kersten, T Karasov, K Poersch, G Shirsekar, D Weigel_


_Genes, rRNA, Germany, Microbiota, Plant Leaves, RNA, Ribosomal, 16S_
Microorganisms from all domains of life establish associations with plants. Although some harm the plant, others antagonize pathogens or prime the plant immune system, support the acquisition of nutrients, tune plant hormone levels, or perform additional services. Most culture-independent plant microbiome research has focused on amplicon sequencing of the 16S rRNA gene and/or the internal transcribed spacer (ITS) of rRNA genomic loci, which show the relative abundance of the microbes to each other. Here, we describe shotgun sequencing of 275 wild Arabidopsis thaliana leaf microbiomes from southwest Germany, with additional bacterial 16S and eukaryotic ITS1 rRNA amplicon data from 176 of these samples. Shotgun data, which unlike the amplicon data capture the ratio of microbe to plant DNA, enable scaling of microbial read abundances to reflect the microbial load on the host. In a more cost-effective hybrid strategy, we show they also allow a similar scaling of amplicon data to overcome compositionality problems. Our wild plants were dominated by bacterial sequences, with eukaryotes contributing only a minority of reads. Microbial membership showed weak associations with both site of origin and plant genotype, both of which were highly confounded in this dataset. There was large variation among microbiomes, with one extreme comprising samples of low complexity and a high load of microorganisms typical of infected plants, and the other extreme being samples of high complexity and a low microbial load. Critically, considering absolute microbial load led to fundamentally different conclusions about microbiome assembly and the interaction networks among major taxa.

[Source](https://www.nature.com/articles/s41396-020-0665-8)

10.1038/s41396-020-0665-8

---

## A joint modeling approach for longitudinal microbiome data improves ability to detect microbiome associations with disease – PLOS Computation Biology

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008473)

---

## Predicting Nash equilibria for microbial metabolic interactions.
 14.12.2020, _J Cai, T Tan, SH Joshua Chan_


Microbial metabolic interactions impact ecosystems, human health and biotechnology profoundly. However, their determination remains elusive, invoking an urgent need for predictive models seamlessly integrating metabolism with evolutionary principles that shape community interactions.

[Source](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa1014/6033579)

10.1093/bioinformatics/btaa1014

---

## microeco: An R package for data mining in microbial community ecology

[Source](https://academic.oup.com/femsec/advance-article/doi/10.1093/femsec/fiaa255/6041020)

---

## Multi-omics data integration considerations and study design for biological systems and disease.
 21.12.2020, _S Graw, K Chappell, CL Washam, A Gies, J Bird, MS Robeson, SD Byrum_


With the advancement of next-generation sequencing and mass spectrometry, there is a growing need for the ability to merge biological features in order to study a system as a whole. Features such as the transcriptome, methylome, proteome, histone post-translational modifications and the microbiome all influence the host response to various diseases and cancers. Each of these platforms have technological limitations due to sample preparation steps, amount of material needed for sequencing, and sequencing depth requirements. These features provide a snapshot of one level of regulation in a system. The obvious next step is to integrate this information and learn how genes, proteins, and/or epigenetic factors influence the phenotype of a disease in context of the system. In recent years, there has been a push for the development of data integration methods. Each method specifically integrates a subset of omics data using approaches such as conceptual integration, statistical integration, model-based integration, networks, and pathway data integration. In this review, we discuss considerations of the study design for each data feature, the limitations in gene and protein abundance and their rate of expression, the current data integration methods, and microbiome influences on gene and protein expression. The considerations discussed in this review should be regarded when developing new algorithms for integrating multi-omics data.

[Source](https://pubs.rsc.org/en/content/articlelanding/2021/MO/D0MO00041H)

10.1039/d0mo00041h

---

## Improved microbial community characterization of 16S rRNA via metagenome hybridization capture enrichment

[Source](https://www.biorxiv.org/content/10.1101/2020.12.18.423101v1.abstract)

---

