## Microbiome-associated human genetic variants impact phenome-wide disease risk.
 26.07.2022, _RHG Markowitz, AL LaBella, M Shi, A Rokas, JA Capra, JF Ferguson, JD Mosley, SR Bordenstein_


_Disease, Gastrointestinal Microbiome, Genetic Variation, Genome, Human, Humans, Phenomics, Phenotype_
Human genetic variation associates with the composition of the gut microbiome, yet its influence on clinical traits remains largely unknown. We analyzed the consequences of nearly a thousand gut microbiome-associated variants (MAVs) on phenotypes reported in electronic health records from tens of thousands of individuals. We discovered and replicated associations of MAVs with neurological, metabolic, digestive, and circulatory diseases. Five significant MAVs in these categories correlate with the relative abundance of microbes down to the strain level. We also demonstrate that these relationships are independently observed and concordant with microbe by disease associations reported in case-control studies. Moreover, a selective sweep and population differentiation impacted some disease-linked MAVs. Combined, these findings establish triad relationships among the human genome, microbiome, and disease. Consequently, human genetic influences may offer opportunities for precision diagnostics of microbiome-associated diseases but also highlight the relevance of genetic background for microbiome modulation and therapeutics.

[Source](https://www.pnas.org/doi/full/10.1073/pnas.2200551119)

10.1073/pnas.2200551119

---

## Microbial risk score for capturing microbial characteristics, integrating multi-omics data, and predicting disease risk.
 09.08.2022, _C Wang, LN Segal, J Hu, B Zhou, RB Hayes, J Ahn, H Li_


_COVID-19, Crohn Disease, Disease Susceptibility, Humans, Microbiota, Risk Factors_
With the rapid accumulation of microbiome-wide association studies, a great amount of microbiome data are available to study the microbiome's role in human disease and advance the microbiome's potential use for disease prediction. However, the unique features of microbiome data hinder its utility for disease prediction.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01310-2)

10.1186/s40168-022-01310-2

---

## A framework for testing the impact of co-infections on host gut microbiomes

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-022-00198-5)

---

## Vaginal microbiota and personal risk factors associated with HPV status conversion-A new approach to reduce the risk of cervical cancer?
 09.08.2022, _Z Yang, Y Zhang, A Stubbe-Espejel, Y Zhao, M Liu, J Li, Y Zhao, G Tong, N Liu, L Qi, A Hutchins, S Lin, Y Li_


Vaginal microbiota (VMB) is associated with changes in Human papilloma virus (HPV) status, which consequently influences the risk of cervical cancer. This association was often confounded by personal risk factors. This pilot research aimed to explore the relationship between vaginal microbiota, personal risk factors and their interactions with HPV status conversion to identify the vaginal microbiota that was associated with HPV clearance under heterogeneous personal risk factors. A total of 38 women participated by self-collecting a cervicovaginal mucus (CVM) sample that was sent for metagenomics sequencing. Most of the participants also filled in personal risk factors questionnaire through an eHealth platform and authorized the use of their previous HPV genotyping results stored in this eHealth platform. Based on the two HPV results, the participants were grouped into three cohorts, namely HPV negative, HPV persistent infection, and HPV status conversion. The relative abundance of VMB and personal factors were compared among these three cohorts. A correlation investigation was performed between VMB and the significant personal factors to characterize a robustness of the panel for HPV status change using R programming. At baseline, 12 participants were HPV-negative, and 22 were HPV-positive. Within one year, 18 women remained HPV-positive, 12 were HPV-negative and 4 participants showed HPV clearance. The factors in the eHealth questionnaire were systematically evaluated which identified several factors significantly associated with persistent HPV infection, including age, salary, history of reproductive tract infection, and the total number of sexual partners. Concurrent vaginal microbiome samples suggest that a candidate biomarker panel consisting of Lactobacillus gasseri, Streptococcus agalactiae, and Timona prevotella bacteria, which may be associated with HPV clearance. This pilot study indicates a stable HPV status-related vaginal microbe environment. To establish a robust biomarker panel for clinical use, larger cohorts will be recruited into follow-up studies.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0270521)

10.1371/journal.pone.0270521

---

## The role of the gut microbiota in multiple sclerosis.
 05.08.2022, _J Correale, R Hohlfeld, SE Baranzini_


During the past decade, research has revealed that the vast community of micro-organisms that inhabit the gut - known as the gut microbiota - is intricately linked to human health and disease, partly as a result of its influence on systemic immune responses. Accumulating evidence demonstrates that these effects on immune function are important in neuroinflammatory diseases, such as multiple sclerosis (MS), and that modulation of the microbiome could be therapeutically beneficial in these conditions. In this Review, we examine the influence that the gut microbiota have on immune function via modulation of serotonin production in the gut and through complex interactions with components of the immune system, such as T cells and B cells. We then present evidence from studies in mice and humans that these effects of the gut microbiota on the immune system are important in the development and course of MS. We also consider how strategies for manipulating the composition of the gut microbiota could be used to influence disease-related immune dysfunction and form the basis of a new class of therapeutics. The strategies discussed include the use of probiotics, supplementation with bacterial metabolites, transplantation of faecal matter or defined microbial communities, and dietary intervention. Carefully designed studies with large human cohorts will be required to gain a full understanding of the microbiome changes involved in MS and to develop therapeutic strategies that target these changes.

[Source](https://www.nature.com/articles/s41582-022-00697-8)

10.1038/s41582-022-00697-8

---

## Specific gut microbiota alterations in essential tremor and its difference from Parkinson's disease.
 09.08.2022, _P Zhang, P Huang, J Du, Y He, J Liu, G He, S Cui, W Zhang, G Li, S Chen_


Essential tremor (ET) is the most common movement disorder and share overlapping symptoms with Parkinson's disease (PD), making differential diagnosis challenging. Gut dysbiosis is regarded crucial in the pathogenesis of PD. Since ET patients also has comorbidity in gastrointestinal disorders, the relationship between gut microbiota and ET really worth investigating and may help distinguishing ET from PD. Fecal samples from 54 ET, 67 de novo PD and 54 normal controls (NC) were collected for 16S ribosomal RNA gene sequencing and quantitative real-time PCR. ET showed lower species richness (Chao1 index) than NC and PD. ET was with Bacteroides-dominant enterotype, while PD was with Ruminococcus-dominant enterotype. Compared with NC, 7 genera were significantly reduced in ET, 4 of which (Ruminococcus, Romboutsia, Mucispirillum, and Aeromonas) were identified to be distinctive with an area under the curve (AUC) of 0.705. Compared to PD, 26 genera were found significantly different from ET, 4 of which (Bacteroides, Fusobacterium, Phascolarctobacterium, and Lachnospira) were found distinguishable with an AUC of 0.756. Clinical association results indicated that Proteus was associated with disease severity (TETRAS) of ET, while Klebsiella was linked to depression and anxiety in ET. Functional predictions revealed that 4 Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways were altered in ET. This study reveals gut dysbiosis in ET and it provides new insight into the pathogenesis of ET and helps distinguishing ET from PD.

[Source](https://www.nature.com/articles/s41531-022-00359-y)

10.1038/s41531-022-00359-y

---

## Identification of clinical and ecological determinants of strain engraftment after fecal microbiota transplantation using metagenomics.
 05.08.2022, _D Podlesny, M Durdevic, S Paramsothy, NO Kaakoush, C Högenauer, G Gorkiewicz, J Walter, WF Fricke_


Fecal microbiota transplantation (FMT) is a promising therapeutic approach for microbiota-associated pathologies, but our understanding of the post-FMT microbiome assembly process and its ecological and clinical determinants is incomplete. Here we perform a comprehensive fecal metagenome analysis of 14 FMT trials, involving five pathologies and &gt;250 individuals, and determine the origins of strains in patients after FMT. Independently of the underlying clinical condition, conspecific coexistence of donor and recipient strains after FMT is uncommon and donor strain engraftment is strongly positively correlated with pre-FMT recipient microbiota dysbiosis. Donor strain engraftment was enhanced through antibiotic pretreatment and bowel lavage and dependent on donor and recipient ɑ-diversity; strains from relatively abundant species were more likely and from predicted oral, oxygen-tolerant, and gram-positive species less likely to engraft. We introduce a general mechanistic framework for post-FMT microbiome assembly in alignment with ecological theory, which can guide development of optimized, more targeted, and personalized FMT therapies.

[Source](https://www.sciencedirect.com/science/article/pii/S2666379122002543)

10.1016/j.xcrm.2022.100711

---

## Alterations in gut microbiota of esophageal squamous cell carcinoma patients.
 19.07.2022, _MK Cheung, GGL Yue, S Lauw, CSY Li, MY Yung, SC Ng, HC Yip, HS Kwan, PWY Chiu, CBS Lau_


Esophageal squamous cell carcinoma (ESCC) is the most common histological subtype of esophageal cancer worldwide. Patients with ESCC display an altered esophageal microbiota compared with healthy individuals; however, little is known about the gut microbiota in ESCC.

[Source](https://onlinelibrary.wiley.com/doi/full/10.1111/jgh.15941)

10.1111/jgh.15941

---

## Targeted suppression of human IBD-associated gut microbiota commensals by phage consortia for treatment of intestinal inflammation.
 09.08.2022, _S Federici, S Kredo-Russo, R Valdés-Mas, D Kviatcovsky, E Weinstock, Y Matiuhin, Y Silberberg, K Atarashi, M Furuichi, A Oka, B Liu, M Fibelman, IN Weiner, E Khabra, N Cullin, N Ben-Yishai, D Inbar, H Ben-David, J Nicenboim, N Kowalsman, W Lieb, E Kario, T Cohen, YF Geffen, L Zelcbuch, A Cohen, U Rappo, I Gahali-Sass, M Golembo, V Lev, M Dori-Bachash, H Shapiro, C Moresi, A Cuevas-Sierra, G Mohapatra, L Kern, D Zheng, SP Nobs, J Suez, N Stettner, A Harmelin, N Zak, S Puttagunta, M Bassan, K Honda, H Sokol, C Bang, A Franke, C Schramm, N Maharshak, RB Sartor, R Sorek, E Elinav_


_Animals, Bacteriophages, Colitis, Gastrointestinal Microbiome, Humans, Inflammation, Inflammatory Bowel Diseases, Klebsiella pneumoniae, Mice_
Human gut commensals are increasingly suggested to impact non-communicable diseases, such as inflammatory bowel diseases (IBD), yet their targeted suppression remains a daunting unmet challenge. In four geographically distinct IBD cohorts (n = 537), we identify a clade of Klebsiella pneumoniae (Kp) strains, featuring a unique antibiotics resistance and mobilome signature, to be strongly associated with disease exacerbation and severity. Transfer of clinical IBD-associated Kp strains into colitis-prone, germ-free, and colonized mice enhances intestinal inflammation. Stepwise generation of a lytic five-phage combination, targeting sensitive and resistant IBD-associated Kp clade members through distinct mechanisms, enables effective Kp suppression in colitis-prone mice, driving an attenuated inflammation and disease severity. Proof-of-concept assessment of Kp-targeting phages in an artificial human gut and in healthy volunteers demonstrates gastric acid-dependent phage resilience, safety, and viability in the lower gut. Collectively, we demonstrate the feasibility of orally administered combination phage therapy in avoiding resistance, while effectively inhibiting non-communicable disease-contributing pathobionts.

[Source](https://www.sciencedirect.com/science/article/pii/S0092867422008509)

10.1016/j.cell.2022.07.003

---

## Camel milk microbiota: A culture-independent assessment.
 08.08.2022, _R Rahmeh, A Akbar, H Alomirah, M Kishk, A Al-Ateeqi, S Al-Milhm, A Shajan, B Akbar, S Al-Merri, M Alotaibi, A Esposito_


Camel milk is renowned for its nutritional value and its therapeutic properties. It is considered a promising alternative to bovine milk due to its higher nutritional benefits, hypoallergenic characteristics and greater digestibility in the human gastrointestinal system. This study reports camel milk's bacterial and fungal microbiota, and the effect of geographical location and season on its bacterial community. We sequenced the V3-V4 regions of the16S rRNA gene for bacteria and the internal transcribed spacer (ITS) for fungi. A total of 134 samples of dromedary raw camel milk were collected from south, north and middle Kuwait during two seasons. Raw camel milk showed a diversified bacterial community, with 1196 genera belonging to 33 phyla. The four most predominant phyla of bacteria were Proteobacteria, Firmicutes, Actinobacteria and Bacteroidota. The core microbiota of raw camel milk, represented by the dominant genera shared by the majority of samples, was constituted by the genera Schlegelella, Paenibacillus, Lactobacillus, unclassified Comamonadaceae, Pediococcus, Moraxella, Acinetobacter, Staphylococcus, Enterococcus, Pseudomonas, Streptococcus, unclassified Micrococcaceae, Rothia, unclassified Sphingomonadaceae, unclassified Neisseriaceae and Sphingomonas. The fungal population was assessed in 14 raw camel milk samples, and comprised 87 genera belonging to 3 phyla. The genera Penicillium, Cladosporium, Candida, Aspergillus, Alternaria and Fusarium, dominated the fungal community. These findings shed light on raw camel milk's core bacterial and fungal microbiome. The geographical location and the season had a significant impact on the diversity and composition of camel milk microbiome.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0963996922006871?)

10.1016/j.foodres.2022.111629

---

## Effect of processing on the anti-inflammatory efficacy of cocoa in a high fat diet-induced mouse model of obesity

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0955286322001851)

---

## The impacts of viral infection and subsequent antimicrobials on the microbiome-resistome of growing pigs.
 07.08.2022, _TN Gaire, C Odland, B Zhang, T Ray, E Doster, J Nerem, S Dee, P Davies, N Noyes_


_Animals, Anti-Bacterial Agents, Anti-Infective Agents, Coinfection, Metagenomics, Microbiota, Porcine Reproductive and Respiratory Syndrome, Porcine respiratory and reproductive syndrome virus, Swine_
Antimicrobials are used in food-producing animals for purposes of preventing, controlling, and/or treating infections. In swine, a major driver of antimicrobial use is porcine reproductive and respiratory syndrome (PRRS), which is caused by a virus that predisposes infected animals to secondary bacterial infections. Numerous antimicrobial protocols are used to treat PRRS, but we have little insight into how these treatment schemes impact antimicrobial resistance (AMR) dynamics within the fecal microbiome of commercial swine. The aim of this study was to determine whether different PRRS-relevant antimicrobial treatment protocols were associated with differences in the fecal microbiome and resistome of growing pigs. To accomplish this, we used a metagenomics approach to characterize and compare the longitudinal wean-to-market resistome and microbiome of pigs challenged with PRRS virus and then exposed to different antimicrobial treatments, and a group of control pigs not challenged with PRRS virus and having minimal antimicrobial exposure. Genomic DNA was extracted from pen-level composite fecal samples from each treatment group and subjected to metagenomic sequencing and microbiome-resistome bioinformatic and statistical analysis. Microbiome-resistome profiles were compared over time and between treatment groups.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01312-0)

10.1186/s40168-022-01312-0

---

## Soil microbial community structures are shaped by agricultural systems revealing little temporal variation.
 08.08.2022, _A Fox, F Widmer, A Lüscher_


Many studies in soil microbial ecology are undertaken with a single sampling event, with the influence of temporal progression rarely being considered. Under field conditions, soil samples were taken from different agricultural systems; a sown grassland to maize rotation (MC), an intensively managed permanent grassland (INT), as well as extensively managed permanent grasslands with high (EXT_HP), low to sufficient (EXT_LP) and deficient available P (EXT_DP), six times throughout the 2017 growing season. Thus, this study aimed to determine if any differences in soil microbiome structures between both sharply contrasting (MC - INT - EXT), slightly differing (EXT_HP - EXT_DP) and quite similar (EXT_HP - EXT_LP and EXT_LP - EXT_DP) agricultural systems persist through changing growth conditions within the growing season. For both fungal and bacterial community structure, the influence of agricultural system (√CV = 0.256 and 0.145, respectively, both at least P &lt; 0.01) was much greater than that of temporal progression (√CV = 0.065 and 0.042, respectively, both P &lt; 0.001). Importantly, nearly all agricultural systems persistently harbored significantly distinct fungal community structures across each of the six sampling events (all at least P &lt; 0.05). There were not as many pairwise differences in bacterial community structure between the agricultural systems, but some did persist (MC and EXT_HP ∼ EXT_DP, all P &lt; 0.001). Additionally, persistent indicator fungal OTUs (IndVal &gt;0.7, P ≤ 0.05) associated to each agricultural system (except EXT_LP) were found in each of the six sampling events. These results highlight the temporal stability of pairwise differences in soil microbiome structures between established agricultural systems through changing plant growth conditions, even between those with a comparable management regime. This is a highly relevant finding in informing the sampling strategy of studies in soil microbial ecology as well as for designing efficient soil biodiversity monitoring systems.

[Source](https://www.sciencedirect.com/science/article/pii/S0013935122012427)

10.1016/j.envres.2022.113915

---

## The streamwater microbiome encodes hydrologic data across scales.
 09.08.2022, _DR URycki, M Bassiouni, SP Good, BC Crump, B Li_


Many fundamental questions in hydrology remain unanswered due to the limited information that can be extracted from existing data sources. Microbial communities constitute a novel type of environmental data, as they are comprised of many thousands of taxonomically and functionally diverse groups known to respond to both biotic and abiotic environmental factors. As such, these microscale communities reflect a range of macroscale conditions and characteristics, some of which also drive hydrologic regimes. Here, we assess the extent to which streamwater microbial communities (as characterized by 16S gene amplicon sequence abundance) encode information about catchment hydrology across scales. We analyzed 64 summer streamwater DNA samples collected from subcatchments within the Willamette, Deschutes, and John Day river basins in Oregon, USA, which range 0.03-29,000 km2 in area and 343-2334 mm/year of precipitation. We applied information theory to quantify the breadth and depth of information about common hydrologic metrics encoded within microbial taxa. Of the 256 microbial taxa that spanned all three watersheds, we found 9.6 % (24.5/256) of taxa, on average, shared information with a given hydrologic metric, with a median 15.6 % (range = 12.4-49.2 %) reduction in uncertainty of that metric based on knowledge of the microbial biogeography. All of the hydrologic metrics we assessed, including daily discharge at different time lags, mean monthly discharge, and seasonal high and low flow durations were encoded within the microbial community. Summer microbial taxa shared the most information with winter mean flows. Our study demonstrates quantifiable relationships between streamwater microbial taxa and hydrologic metrics at different scales, likely resulting from the integration of multiple overlapping drivers of each. Streamwater microbial communities are rich sources of information that may contribute fresh insight to unresolved hydrologic questions.

[Source](https://www.sciencedirect.com/science/article/pii/S0048969722050100)

10.1016/j.scitotenv.2022.157911

---

## Lake iTag measurements over nineteen years, introducing the limony dataset

[Source](https://www.biorxiv.org/content/10.1101/2022.08.04.502869v1.abstract)

---

