## Association between microbiome and the development of adverse posttraumatic neuropsychiatric sequelae after traumatic stress exposure.
 22.11.2023, _AL Zeamer, MC Salive, X An, FL Beaudoin, SL House, JS Stevens, D Zeng, TC Neylan, GD Clifford, SD Linnstaedt, SL Rauch, AB Storrow, C Lewandowski, PI Musey, PL Hendry, S Sheikh, CW Jones, BE Punches, RA Swor, LA Hudak, JL Pascual, MJ Seamon, E Harris, C Pearson, DA Peak, RC Merchant, RM Domeier, NK Rathlev, BJ O'Neil, P Sergot, LD Sanchez, SE Bruce, RC Kessler, KC Koenen, SA McLean, V Bucci, JP Haran_


_Adult, Humans, Microbiota, Stress Disorders, Post-Traumatic, Gastrointestinal Microbiome, Feces, Biological Availability_
Patients exposed to trauma often experience high rates of adverse post-traumatic neuropsychiatric sequelae (APNS). The biological mechanisms promoting APNS are currently unknown, but the microbiota-gut-brain axis offers an avenue to understanding mechanisms as well as possibilities for intervention. Microbiome composition after trauma exposure has been poorly examined regarding neuropsychiatric outcomes. We aimed to determine whether the gut microbiomes of trauma-exposed emergency department patients who develop APNS have dysfunctional gut microbiome profiles and discover potential associated mechanisms. We performed metagenomic analysis on stool samples (n = 51) from a subset of adults enrolled in the Advancing Understanding of RecOvery afteR traumA (AURORA) study. Two-, eight- and twelve-week post-trauma outcomes for post-traumatic stress disorder (PTSD) (PTSD checklist for DSM-5), normalized depression scores (PROMIS Depression Short Form 8b) and somatic symptom counts were collected. Generalized linear models were created for each outcome using microbial abundances and relevant demographics. Mixed-effect random forest machine learning models were used to identify associations between APNS outcomes and microbial features and encoded metabolic pathways from stool metagenomics. Microbial species, including Flavonifractor plautii, Ruminococcus gnavus and, Bifidobacterium species, which are prevalent commensal gut microbes, were found to be important in predicting worse APNS outcomes from microbial abundance data. Notably, through APNS outcome modeling using microbial metabolic pathways, worse APNS outcomes were highly predicted by decreased L-arginine related pathway genes and increased citrulline and ornithine pathways. Common commensal microbial species are enriched in individuals who develop APNS. More notably, we identified a biological mechanism through which the gut microbiome reduces global arginine bioavailability, a metabolic change that has also been demonstrated in the plasma of patients with PTSD.

[Source](https://doi.org/10.1038/s41398-023-02643-8)

10.1038/s41398-023-02643-8

---

## The preterm human milk microbiota fluctuates by postpartum week and is characterized by gestational age and maternal BMI.
 17.11.2023, _EJ Filatava, Z Liu, J Xie, D-B Tran, K Chen, N El Habbal, G Weinstock, Y Zhou, KE Gregory_


Despite a growing recognition that the type of nutrition received by preterm infants influences their intestinal microbiome and health outcomes, the microbiota of mother's own milk (MOM), pasteurized donor human milk (PDHM), and infant formula remain poorly characterized. In our study, we found that the structure of microbial communities, bacterial diversity, and relative abundances of specific genera were significantly different between MOM, PDHM, and formula. Additionally, our results suggest that the microbiota of MOM changes as a function of time and maternal factors. Lastly, we identified three lactotypes within MOM that have distinct microbial compositions and described the maternal factors associated with them. These findings set the stage for future research aimed at advancing our knowledge of the microbiota of preterm infant nutrition and the specific influence it may have on health outcomes.

[Source](https://doi.org/10.1128/mbio.02106-23)

10.1128/mbio.02106-23

---

## Strain-resolved metagenomic analysis of the gut as a reservoir for bloodstream infection pathogens among premature infants in Singapore

[Source](https://doi.org/10.1186/s13099-023-00583-8)

---

## <em>Porphyromonas gingivalis</em> as a promotor in the development of the alcoholic liver disease via ferroptosis

[Source](https://doi.org/10.1016/j.micinf.2023.105250)

---

## The core phageome and its interrelationship with preterm human milk lipids

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(23)01385-2)

---

## Longitudinal analysis at three oral sites links oral microbiota to clinical outcomes in allogeneic hematopoietic stem-cell transplant

[Source](https://doi.org/10.1128/spectrum.02910-23)

---

## Gut inflammation associated with age and Alzheimerâ€™s disease pathology: a human cohort study

[Source](https://doi.org/10.1038/s41598-023-45929-z)

---

## Robust airway microbiome signatures in acute respiratory failure and hospital-acquired pneumonia

[Source](https://doi.org/10.1038/s41591-023-02617-9)

---

## Systematic mining of the human microbiome identifies antimicrobial peptides with diverse activity spectra

[Source](https://doi.org/10.1038/s41564-023-01524-6)

---

## Skin benefits of postbiotics derived from <em>Micrococcus luteus</em> derived from human skin: an untapped potential for dermatological health

[Source](https://doi.org/10.1007/s13258-023-01471-w)

---

## <em>Akkermansia muciniphila</em> attenuated lipopolysaccharide-induced acute lung injury by modulating the gut microbiota and SCFAs in mice

[Source](https://doi.org/10.1039/d3fo04051h)

---

## The Potential of Meta-Proteomics and Artificial Intelligence to Establish the Next Generation of Probiotics for Personalized Healthcare

[Source](https://doi.org/10.1021/acs.jafc.3c03834)

---

## Gut microbiome in healthy aging versus those associated with frailty

[Source](https://doi.org/10.1080/19490976.2023.2278225)

---

## Host species and habitat shape fish-associated bacterial communities: phylosymbiosis between fish and their microbiome

[Source](https://doi.org/10.1186/s40168-023-01697-6)

---

## <em>Aedes aegypti</em> microbiome composition covaries with the density of <em>Wolbachia</em> infection

[Source](https://doi.org/10.1186/s40168-023-01678-9)

---

## Sequential host-bacteria and bacteria-bacteria interactions determine the microbiome establishment of <em>Nematostella vectensis</em>

[Source](https://doi.org/10.1186/s40168-023-01701-z)

---

## Microbial symbionts are shared between ants and their associated beetles

[Source](https://doi.org/10.1111/1462-2920.16544)

---

## The gut microbiome mediates adaptation to scarce food in Coleoptera

[Source](https://doi.org/10.1186/s40793-023-00537-2)

---

## Interplay of gut microbiota and host epithelial mitochondrial dysfunction is necessary for the development of spontaneous intestinal inflammation in mice

[Source](https://doi.org/10.1186/s40168-023-01686-9)

---

## Microbial diversity and community structure dynamics in acid mine drainage: Acidic fire with dissolved heavy metals

[Source](https://doi.org/10.1016/j.scitotenv.2023.168635)

---

## Polar lake microbiomes have distinct evolutionary histories

[Source](https://doi.org/10.1126/sciadv.ade7130)

---

## PSKR1 balances the plant growth-defence trade-off in the rhizosphere microbiome.
 17.11.2023, _S Song, Z Morales Moreira, AL Briggs, XC Zhang, AC Diener, CH Haney_


Microbiota benefit their hosts by improving nutrient uptake and pathogen protection. How host immunity restricts microbiota while avoiding autoimmunity is poorly understood. Here we show that the Arabidopsis phytosulfokine receptor 1 (pskr1) mutant displays autoimmunity (plant stunting, defence-gene expression and reduced rhizosphere bacterial growth) in response to growth-promoting Pseudomonas fluorescens. Microbiome profiling and microbiota colonization showed that PSKR1-mediated reduction in bacterial growth and stunting is largely specific to Pseudomonas. Transcriptional profiling demonstrated that PSKR1 regulates the growth-defence trade-off during Pseudomonas colonization: PSKR1 upregulates plant photosynthesis and root growth but suppresses salicylic-acid-mediated defences. Genetic epistasis experiments showed that pskr1 stunting and restriction of bacterial growth are salicylic acid dependent. Finally, we showed that Pseudomonas, but not other bacteria, induces PSKR1 expression in roots, suggesting that Pseudomonas might manipulate plant signalling to promote its colonization. Our data demonstrate a genetic mechanism to coordinate beneficial functions of the microbiome while preventing autoimmunity.

[Source](https://doi.org/10.1038/s41477-023-01539-1)

10.1038/s41477-023-01539-1

---

## <em>Streptomyces</em> alleviate abiotic stress in plant by producing pteridic acids

[Source](https://doi.org/10.1038/s41467-023-43177-3)

---

## The pan-microbiome profiling system Taxa4Meta identifies clinical dysbiotic features and classifies diarrheal disease.
 14.11.2023, _Q Wu, S Badu, SY So, TJ Treangen, TC Savidge_


Targeted metagenomic sequencing is an emerging strategy to survey disease- specific microbiome biomarkers for clinical diagnosis and prognosis. However, this approach often yields inconsistent or conflicting results due to inadequate study power and sequencing bias. We introduce Taxa4Meta, a bioinformatics pipeline explicitly designed to compensate for technical and demographic bias. We designed and validated Taxa4Meta for accurate taxonomic profiling of 16S rRNA amplicon data acquired from different sequencing strategies. Taxa4Meta offers significant potential in identifying clinical dysbiotic features that can reliably predict human disease, validated comprehensively via re-analysis of individual patient 16S datasets. We leveraged the power of Taxa4Meta's pan-microbiome profiling to generate 16S-based classifiers that exhibited excellent utility for stratification of diarrheal patients with Clostridioides difficile infection, irritable bowel syndrome or inflammatory bowel diseases, which represent common misdiagnoses and pose significant challenges for clinical management. We believe that Taxa4Meta represents a new "best practices" approach to individual microbiome surveys that can be used to define gut dysbiosis at a population-scale level.

[Source](https://doi.org/10.1172/JCI170859)

10.1172/JCI170859

---

## Machine learning for microbiologists.
 15.11.2023, _F Asnicar, AM Thomas, A Passerini, L Waldron, N Segata_


Machine learning is increasingly important in microbiology where it is used for tasks such as predicting antibiotic resistance and associating human microbiome features with complex host diseases. The applications in microbiology are quickly expanding and the machine learning tools frequently used in basic and clinical research range from classification and regression to clustering and dimensionality reduction. In this Review, we examine the main machine learning concepts, tasks and applications that are relevant for experimental and clinical microbiologists. We provide the minimal toolbox for a microbiologist to be able to understand, interpret and use machine learning in their experimental and translational activities.

[Source](https://doi.org/10.1038/s41579-023-00984-1)

10.1038/s41579-023-00984-1

---

