## The intestinal 3M (microbiota, metabolism, metabolome) Zeitgeist - from fundamentals to future challenges.
 05.10.2021, _P Dey, SR Chaudhuri, T Efferth, S Pal_


The role of the intestine in human health and disease has historically been neglected and was mostly attributed to digestive and absorptive functions. In the past two decades, however, discoveries related to human nutrition and intestinal host-microbe reciprocal interaction have established the essential role of intestinal health in the pathogenesis of chronic diseases and the overall wellbeing. That transfer of gut microbiota could be a means of disease phenotype transfer has revolutionized our understanding of chronic disease pathogenesis. This narrative review highlights the major concepts related to intestinal microbiota, metabolism, and metabolome (3M) that have facilitated our fundamental understanding of the association between the intestine and human health and disease. In line with increased interest of microbiota-dependent modulation of human health by dietary phytochemicals, we have also discussed the emerging concepts beyond the phytochemical bioactivities which emphasizes the integral role of microbial metabolites of parent phytochemicals at extraintestinal tissues. Finally, this review concludes with challenges and future prospects in defining the 3 M interactions and has emphasized the fact that, it takes 'guts' to stay healthy.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0891584921007462)

10.1016/j.freeradbiomed.2021.09.026

---

## Gut microbiota changes in preeclampsia, abnormal placental growth and healthy pregnant women.
 05.10.2021, _L Huang, M Cai, L Li, X Zhang, Y Xu, J Xiao, Q Huang, G Luo, Z Zeng, C Jin, Y Jin, J He, W Yang_


Preeclampsia (PE) is a condition of high blood pressure that is usually concurrent with proteinuria in pregnancy. PE complicates the management of both maternal and fetal health and contributes to most adverse pregnancy outcomes, but the mechanism underlying the development of PE remains unclear. In this study, we performed a case-control study to compare the gut microbiota of PE (n = 26), abnormal placental growth (APG, n = 25) and healthy pregnant women (n = 28) and analyzed the potential pathogenic role of gut microbiota in PE progression.

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-021-02327-7)

10.1186/s12866-021-02327-7

---

## Refinement of microbiota analysis of specimens from patients with respiratory infections using next-generation sequencing.
 05.10.2021, _H Ikegami, S Noguchi, K Fukuda, K Akata, K Yamasaki, T Kawanami, H Mukae, K Yatera_


Next-generation sequencing (NGS) technologies have been applied in bacterial flora analysis. However, there is no standardized protocol, and the optimal clustering threshold for estimating bacterial species in respiratory infection specimens is unknown. This study was conducted to investigate the optimal threshold for clustering 16S ribosomal RNA gene sequences into operational taxonomic units (OTUs) by comparing the results of NGS technology with those of the Sanger method, which has a higher accuracy of sequence per single read than NGS technology. This study included 45 patients with pneumonia with aspiration risks and 35 patients with lung abscess. Compared to Sanger method, the concordance rates of NGS technology (clustered at 100%, 99%, and 97% homology) with the predominant phylotype were 78.8%, 71.3%, and 65.0%, respectively. With respect to the specimens dominated by the Streptococcus mitis group, containing several important causative agents of pneumonia, Bray Curtis dissimilarity revealed that the OTUs obtained at 100% clustering threshold (versus those obtained at 99% and 97% thresholds; medians of 0.35, 0.69, and 0.71, respectively) were more similar to those obtained by the Sanger method, with statistical significance (p &lt; 0.05). Clustering with 100% sequence identity is necessary when analyzing the microbiota of respiratory infections using NGS technology.

[Source](https://www.nature.com/articles/s41598-021-98985-8)

10.1038/s41598-021-98985-8

---

## Experimental Dermatology Special Issue with Focus on ‘Skin microbiome and antimicrobial peptides’

[Source](https://onlinelibrary.wiley.com/toc/16000625/2021/30/10)

---

## Gut microbiome in pediatric acute leukemia: from predisposition to cure.
 05.10.2021, _R Masetti, E Muratore, D Leardini, D Zama, S Turroni, P Brigidi, S Esposito, A Pession_


The gut microbiome (GM) has emerged as a key factor in the genesis and progression of many diseases. The intestinal bacterial composition is also able to influence treatment-related side effects and even efficacy of oncological therapies. Acute leukemia (AL) is the most common cancer among children and the most frequent cause of death from cancer during childhood. Outcome has improved considerably over the past four decades, with the current long-term survival for acute lymphoblastic leukemia being approximately 90%. However, several acute toxicities and long-term sequela are associated with the multimodal therapy-protocols applied in these patients. Specific GM configurations could contribute in the multistep developmental hypothesis for leukemogenesis. Moreover, GM alterations occur during the AL therapeutic course and are associated with treatment-related complications, especially during hematopoietic stem cell transplantation. The GM perturbation could last even after the removal of microbiome-modifying factors, like antibiotics, chemotherapeutic drugs or allo-immune reactions, contributing to several health-related issues in AL survivors. The aim of this paper is to provide a comprehensive review of the chronological changes of GM in children with AL, from predisposition to cure. The underpinning biological processes and the potential interventions to modulate the GM towards a potentially health-promoting configuration are also highlighted.

[Source](https://ashpublications.org/bloodadvances/article/doi/10.1182/bloodadvances.2021005129/477172/Gut-microbiome-in-pediatric-acute-leukemia-from)

10.1182/bloodadvances.2021005129

---

## Effects of moderate exercise on hepatic amino acid and fatty acid composition, liver transcriptome, and intestinal microbiota in channel catfish (Ictalurus punctatus).
 04.10.2021, _L Zhang, Y Yu, L Dong, J Gan, T Mao, T Liu, X Li, L He_


Previous research on swimming exercise in fish has focused on muscle building and quality of flesh. However, the effects of hepatic amino acid and fatty acid composition, liver gene expression profile, and intestinal microbiota are poorly understood. In this study, channel catfish (Ictalurus punctatus) were subjected to a 4-week swimming exercise, and liver transcriptome and intestinal microbiota analyses were performed to broaden our understanding of fish under exercise. When compared to non-exercised channel catfish (N-EXF), exercised channel catfish (EXF) had improved arachidonic acid (C20:4n6; ARA), docosahexaenoic acid (C22:6n3; DHA), aspartic acid (Asp) and glycine (Gly). The liver transcriptome analysis revealed 2912 differentially expressed genes and numerous enriched signaling pathways including those involved in nutrient synthesis, such as biosynthesis of unsaturated fatty acids and amino acids; glucose metabolism, such as glycolysis/gluconeogenesis, insulin signaling, and AMPK signaling pathways; and oxygen transport, such as HIF-1, PI3K-Akt, and MAPK signaling pathways. In addition, bacterial 16S rRNA gene sequencing data revealed that long-term exercise increased bacterial diversity and richness, and changed the intestinal microbial composition in channel catfish. In summary, this study provides insights into hepatic metabolic pathways, candidate genes, and intestinal microbiota underlying the long-term exercised channel catfish.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1744117X21001337)

10.1016/j.cbd.2021.100921

---

## Metagenomic Sequencing Reveals that High-Grain Feeding Alters the Composition and Metabolism of Cecal Microbiota and Induces Cecal Mucosal Injury in Sheep.
 05.10.2021, _F Xie, L Xu, Y Wang, S Mao_


The cecum serves as an additional fermentation site for ruminants, but it lacks buffering capacity and has a relatively simple epithelial structure compared to the rumen. The role of high-grain (HG) diets in manipulating the rumen microbiome has been well elucidated, yet the microbial response to such diets in the cecum and the subsequent microbe-host interactions remain largely unexplored. Here, we describe the modification of the cecal microbiome and host epithelial gene expression based on data from 20 sheep grouped to feed an HG diet for 7, 14, and 28 days. Our data indicate that the alteration of cecal microbial fermentation was manifested by a decrease in luminal pH and an increase in acetate and butyrate concentrations following the diet change to HG. We further demonstrate that the alteration of the microbiome was driven by microbes that are likely acetate producers (e.g., Blautia spp. and Akkermansia spp.) and butyrate producers (e.g., Anaerostipes spp. and Roseburia spp.). Moreover, the core microbiota in the cecal microbiome was predominantly maintained after HG diet feeding, while the specific populations of the cecal microbiomes adaptively varied at the species and genomic levels time dependently. Association analysis suggests that the perturbations of the cecal microbiome under the HG diet were closely linked to the variations in the two key enzymes that catalyze the conversion of pyruvate to acetyl-CoA and urease enzymes that hydrolyze urea into ammonia, alongside mucosal inflammatory responses. Overall, our findings here provide novel insights into understanding microbiome-host interactions in the hindgut of ruminants. IMPORTANCE High-grain (HG) diets are known to alter the rumen microbiome. However, the responses of the hindgut microbiota and its epithelial function need further investigation in ruminants. Using 20 sheep as the experimental model, we found that the microbial fermentation pattern of the cecum changed after switching to the HG diet. The taxa of the acetate and butyrate producers increased with the feeding time. Moreover, enzymes engaged in carbon and nitrogen metabolisms of the cecal microbiome are altered. The expression of epithelial genes related to volatile fatty acid (VFA) absorption and metabolism, cytokines, and tight junction proteins, alongside light microscopy visualization of epithelial tissue, suggested that the HG diet may induce cecal mucosal inflammatory responses. Our findings reveal cecal microbial and metabolic perturbations in response to HG diets in sheep and provide a new reference for the research on hindgut microbial homeostasis and host health in ruminants.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00915-21)

10.1128/mSystems.00915-21

---

## Biodegradable and conventional microplastics exhibit distinct microbiome, functionality, and metabolome changes in soil.
 04.10.2021, _Y Sun, C Duan, N Cao, C Ding, Y Huang, J Wang_


Environmental concerns with liberal petroleum-based plastic use have led to demand for sustainable biodegradable alternatives. However, the inadequate end-of-life treatment of plastics may emit microplastics, either conventional or biodegradable, to the terrestrial environment. It is essential to evaluate the possible effects of conventional and biodegradable microplastics on the composition and function of soil microbial communities. Therefore, we conducted a soil microcosm experiment with polyethylene (PE), polystyrene (PS), polylactide (PLA), or polybutylene succinate (PBS) microplastics. The soil microbiome and metabolome were evaluated via 16S rRNA gene sequencing, metagenomics, and untargeted metabolomics. We reported that the presence of conventional or biodegradable microplastics can significantly alter soil microbial community composition. Compared to the control soils, the microbiome in PBS and PLA amended soils exhibited higher potential for uptake of exogenous carbohydrates and amino acids, but a reduced capacity for related metabolic function, potentially due to catabolite repression. No differences in soil metabolome can be observed between conventional microplastic treatments and the control. The potential reason may be that the functional diversity was unaffected by PE and PS microplastics, while the biodegradable particles promoted the soil microbial multifunctionality. Our findings systematically shed light on the influence of conventional and biodegradable microplastics on soil microorganisms, facilitating microplastic regulation.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0304389421022500)

10.1016/j.jhazmat.2021.127282

---

## Soil microbiota and microarthropod communities in oil contaminated sites in the European Subarctic.
 05.10.2021, _EN Melekhina, ES Belykh, MY Markarova, AA Taskaeva, EE Rasova, OA Baturina, MR Kabilov, IO Velegzhaninov_


The present comprehensive study aimed to estimate the aftermath of oil contamination and the efficacy of removing the upper level of polluted soil under the conditions of the extreme northern taiga of northeastern European Russia. Soil samples from three sites were studied. Two sites were contaminated with the contents of a nearby sludge collector five years prior to sampling. The highly contaminated upper soil level was removed from one of them. The other was left for self-restoration. A chemical analysis of the soils was conducted, and changes in the composition of the soil zoocoenosis and bacterial and fungal microbiota were investigated. At both contaminated sites, a decrease in the abundance and taxonomic diversity of indicator groups of soil fauna, oribatid mites and collembolans compared to the background site were found. The pioneer eurytopic species Oppiella nova, Proisotoma minima and Xenyllodes armatus formed the basis of the microarthropod populations in the contaminated soil. A complete change in the composition of dominant taxonomic units was observed in the microbiota, both the bacterial and fungal communities. There was an increase in the proportion of representatives of Proteobacteria and Actinobacteria in polluted soils compared to the background community. Hydrocarbon-degrading bacteria-Alcanivorax, Rhodanobacter ginsengisoli, Acidobacterium capsulatum, and Acidocella-and fungi-Amorphotheca resinae abundances greatly increased in oil-contaminated soil. Moreover, among both bacteria and fungi, a sharp increase in the abundance of uncultivated organisms that deserve additional attention as potential oil degraders or organisms with a high resistance to oil contamination were observed. The removal of the upper soil level was partly effective in terms of decreasing the oil product concentration (from approximately 21 to 2.6 g/kg of soil) and preventing a decrease in taxonomic richness but did not prevent alterations in the composition of the microbiota or zoocoenosis.

[Source](https://www.nature.com/articles/s41598-021-98680-8)

10.1038/s41598-021-98680-8

---

## Fast and persistent responses of alpine permafrost microbial communities to in situ warming.
 05.10.2021, _C Perez-Mon, B Stierli, M Plötze, B Frey_


Global warming in mid-latitude alpine regions results in permafrost thawing, together with greater availability of carbon and nutrients in soils and frequent freeze-thaw cycles. Yet it is unclear how these multifactorial changes will shape the 1 m-deep permafrost microbiome in the future, and how this will in turn modulate microbially-mediated feedbacks between mountain soils and climate (e.g. soil CO2 emissions). To unravel the responses of the alpine permafrost microbiome to in situ warming, we established a three-year experiment in a permafrost monitoring summit in the Alps. Specifically, we simulated conditions of warming by transplanting permafrost soils from a depth of 160 cm either to the active-layer topsoils in the north-facing slope or in the warmer south-facing slope, near the summit. qPCR-based and amplicon sequencing analyses indicated an augmented microbial abundance in the transplanted permafrost, driven by the increase in copiotrophic prokaryotic taxa (e.g. Noviherbaspirillum and Massilia) and metabolically versatile psychrotrophs (e.g. Tundrisphaera and Granulicella); which acclimatized to the changing environment and potentially benefited from substrates released upon thawing. Metabolically restricted Patescibacteria lineages vastly decreased with warming, as reflected in the loss of α-diversity in the transplanted soils. Ascomycetous sapro-pathotrophs (e.g. Tetracladium) and a few lichenized fungi (e.g. Aspicilia) expanded in the transplanted permafrost, particularly in soils transplanted to the warmer south-facing slope, replacing basidiomycetous yeasts (e.g. Glaciozyma). The transplantation-induced loosening of microbial association networks in the permafrost could potentially indicate lesser cooperative interactions between neighboring microorganisms. Broader substrate-use microbial activities measured in the transplanted permafrost could relate to altered soil C dynamics. The three-year simulated warming did not, however, enhance heterotrophic respiration, which was limited by the carbon-depleted permafrost conditions. Collectively, our quantitative findings suggest the vulnerability of the alpine permafrost microbiome to warming, which might improve predictions on microbially-modulated transformations of mountain soil ecosystems under the future climate.

[Source](https://www.sciencedirect.com/science/article/pii/S0048969721057983)

10.1016/j.scitotenv.2021.150720

---

## Environmental dynamics of hospital microbiome upon transfer from a major hospital to a new facility.
 04.10.2021, _A Ashokan, JM Choo, SL Taylor, D Lagana, DR Shaw, MS Warner, SL Wesselingh, GB Rogers_


Infection control is critical to safe hospital care. However, how bacteria within nosocomial environments relate to space utilisation and occupancy remains poorly understood. Our aim was to characterise the hospital microbiome in the context of the closure of a tertiary hospital and the opening of a new facility.

[Source](https://www.journalofinfection.com/article/S0163-4453(21)00490-4/fulltext)

10.1016/j.jinf.2021.09.020

---

## Predicting Microbiome Metabolism and Interactions through Integrating Multidisciplinary Principles.
 05.10.2021, _CM Schmidt, P Ghadermazi, SHJ Chan_


In this Commentary, we will discuss some of the current trends and challenges in modeling microbiome metabolism. A focus will be the state of the art in the integration of metabolic networks, ecological and evolutionary principles, and spatiotemporal considerations, followed by envisioning integrated frameworks incorporating different principles and data to generate predictive models in the future.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00768-21)

10.1128/mSystems.00768-21

---

## Uniform Manifold Approximation and Projection (UMAP) Reveals Composite Patterns and Resolves Visualization Artifacts in Microbiome Data.
 05.10.2021, _G Armstrong, C Martino, G Rahman, A Gonzalez, Y Vázquez-Baeza, G Mishne, R Knight_


Microbiome data are sparse and high dimensional, so effective visualization of these data requires dimensionality reduction. To date, the most commonly used method for dimensionality reduction in the microbiome is calculation of between-sample microbial differences (beta diversity), followed by principal-coordinate analysis (PCoA). Uniform Manifold Approximation and Projection (UMAP) is an alternative method that can reduce the dimensionality of beta diversity distance matrices. Here, we demonstrate the benefits and limitations of using UMAP for dimensionality reduction on microbiome data. Using real data, we demonstrate that UMAP can improve the representation of clusters, especially when the clusters are composed of multiple subgroups. Additionally, we show that UMAP provides improved correlation of biological variation along a gradient with a reduced number of coordinates of the resulting embedding. Finally, we provide parameter recommendations that emphasize the preservation of global geometry. We therefore conclude that UMAP should be routinely used as a complementary visualization method for microbiome beta diversity studies. IMPORTANCE UMAP provides an additional method to visualize microbiome data. The method is extensible to any beta diversity metric used with PCoA, and our results demonstrate that UMAP can indeed improve visualization quality and correspondence with biological and technical variables of interest. The software to perform this analysis is available under an open-source license and can be obtained at https://github.com/knightlab-analyses/umap-microbiome-benchmarking; additionally, we have provided a QIIME 2 plugin for UMAP at https://github.com/biocore/q2-umap.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00691-21)

10.1128/mSystems.00691-21

---

