## Maternal antibiotic prophylaxis during cesarean section has a limited impact on the infant gut microbiome

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(24)00268-3)

---

## Microbial assimilatory sulfate reduction-mediated H2S: an overlooked role in Crohn's disease development.
 16.08.2024, _W Luo, M Zhao, M Dwidar, Y Gao, L Xiang, X Wu, MH Medema, S Xu, X Li, H Schäfer, M Chen, R Feng, Y Zhu_


_Crohn Disease, Humans, Gastrointestinal Microbiome, Hydrogen Sulfide, Animals, Mice, Sulfates, Escherichia coli, Feces, Dysbiosis, Colon, Metagenomics, Oxidation-Reduction, Disease Models, Animal, Female_
H2S imbalances in the intestinal tract trigger Crohn's disease (CD), a chronic inflammatory gastrointestinal disorder characterized by microbiota dysbiosis and barrier dysfunction. However, a comprehensive understanding of H2S generation in the gut, and the contributions of both microbiota and host to systemic H2S levels in CD, remain to be elucidated. This investigation aimed to enhance comprehension regarding the sulfidogenic potential of both the human host and the gut microbiota.

[Source](https://doi.org/10.1186/s40168-024-01873-2)

10.1186/s40168-024-01873-2

---

## Characterization of the vaginal microbiota in Italian women with endometriosis: preliminary study.
 16.08.2024, _R Sessa, S Filardo, MF Viscardi, G Brandolino, L Muzii, M Di Pietro, MG Porpora_


This cross-sectional study aims to assess the interplay between the vaginal microbiota and endometriosis.

[Source](https://doi.org/10.1007/s00404-024-07631-x)

10.1007/s00404-024-07631-x

---

## Profiles and interactions of gut microbiome and intestinal microRNAs in pediatric Crohn's disease.
 16.08.2024, _Y Lv, C Zhen, A Liu, Y Hu, G Yang, C Xu, Y Lou, Q Cheng, Y Luo, J Yu, Y Fang, H Zhao, K Peng, Y Yu, J Lou, J Chen, Y Ni_


Gut dysbiosis is closely related to dysregulated microRNAs (miRNAs) in the intestinal epithelial cells, which plays an important role in the pathogenesis of Crohn's disease (CD). We investigated the relationship between fecal gut microbiome (GM) and intestinal tissue miRNAs in different stages of pediatric CD. Metagenomic analysis and miRNA sequencing were conducted to examine the GM and intestinal miRNA profiles of CD patients before and after clinical induction therapy and the controls. Twenty-seven newly diagnosed, therapy-naïve pediatric patients with active CD and 11 non-inflammatory bowel disease (IBD) controls were recruited in this study. Among CD patients, 11 patients completed induction treatment and reached clinical remission. Both GM and miRNA profiles were significantly changed between CD patients and controls. Seven key bacteria were identified at species level including Defluviitalea raffinosedens, Thermotalea metallivorans, Roseburia intestinalis, Dorea sp. AGR2135, Escherichia coli, Shigella sonnei, and Salmonella enterica, the exact proportions of which were further validated by real-time quantitative PCR analysis. Eight key miRNAs were also identified including hsa-miR-215-5p, hsa-miR-194-5p, hsa-miR-12135, hsa-miR-509-3-5p, hsa-miR-212-5p, hsa-miR-4448, hsa-miR-501-3p, and hsa-miR-503-5p. The functional enrichment analysis of differential miRNAs indicated the significantly altered cyclin protein, cyclin-dependent protein, and cell cycle pathway. The close interactions between seven key bacteria and eight key miRNAs were further investigated by miRNA target prediction. The association between specific miRNA expressions and key gut bacteria at different stages of CD supported their important roles as potential molecular biomarkers. Understanding the relationship between them will help us to explore the molecular mechanisms of CD.

[Source](https://doi.org/10.1128/msystems.00783-24)

10.1128/msystems.00783-24

---

## Regional variation and adaptive evolution in <em>Bifidobacterium pseudocatenulatum</em>: Insights into genomic and functional diversity in human gut

[Source](https://doi.org/10.1016/j.foodres.2024.114840)

---

## Effects of physical exercise on <em>Akkermansia muciniphila</em>: a systematic review of human and animal studies

[Source](https://doi.org/10.1163/18762891-bja00031)

---

## Microbiome modification for personalized treatment of dysbiotic diseases

[Source](https://www.cell.com/cell-host-microbe/abstract/S1931-3128(24)00282-8)

---

## Combining Gut Microbiota Modulation and Enzymatic-Triggered Colonic Delivery by Prebiotic Nanoparticles Improves Mouse Colitis Therapy

[Source](https://doi.org/10.34133/bmr.0062)

---

## Effect of chronic alcohol consumption on oral microbiota in rats with periodontitis

[Source](https://doi.org/10.7717/peerj.17795)

---

## Fecal microbiota transplantation accelerates restoration of florfenicol-disturbed intestinal microbiota in a fish model

[Source](https://doi.org/10.1038/s42003-024-06727-z)

---

## Structural, Biochemical, and Phylogenetic Analysis of Bacterial and Fungal Carbohydrate Esterase Family 15 Glucuronoyl Esterases in the Rumen

[Source](https://doi.org/10.1007/s10930-024-10221-0)

---

## Harnessing the plant microbiome for sustainable crop production

[Source](https://doi.org/10.1038/s41579-024-01079-1)

---

## The impact of filamentous plant pathogens on the host microbiota

[Source](https://doi.org/10.1186/s12915-024-01965-3)

---

## Metagenomic insights into the prokaryotic communities of heavy metal-contaminated hypersaline soils

[Source](https://doi.org/10.1016/j.scitotenv.2024.175497)

---

## Ecological phage therapy: Can bacteriophages help rapidly restore the soil microbiome?

[Source](https://doi.org/10.1002/ece3.70185)

---

## Do restoration strategies in mangroves recover microbial diversity? A case study in the Yucatan peninsula

[Source](https://doi.org/10.1371/journal.pone.0307929)

---

## Metagenome sequencing and 107 microbial genomes from seamount sediments along the Yap and Mariana trenches.
 17.08.2024, _Y Zhang, H Jing_


_Metagenome, Geologic Sediments, Phylogeny, Bacteria, Archaea, Genome, Microbial, Genome, Archaeal, Genome, Bacterial_
Microbes in the sediments across a series of seamounts along the island arc of the Yap and Mariana trenches were investigated by metagenome. In this study, we reconstructed 107 metagenome-assembled genomes (MAGs), including 100 bacteria and 7 archaea. All the MAGs exhibited &gt;75% completeness and &lt;10% contamination, with 26 MAGs being classified as 'nearly complete' (completeness &gt;90%), while 50 falling within 80-90% range and 31 between 75-80% complete. Phylogenomic analysis revealed that 86% (n = 92) of these MAGs represented new taxa at different taxonomical levels. The species composition of these MAGs was most consistent with the previous reports, with the most abundant phyla being Proteobacteria (n = 39), Methylomirabilota (n = 27), and Nitrospirota (n = 7). These draft genomes provided novel data on species diversity and function in the seamount microbial community, which will provide reference data for extensive comparative genomic studies across crucial phylogenetic groups worldwide.

[Source](https://doi.org/10.1038/s41597-024-03762-7)

10.1038/s41597-024-03762-7

---

## Metrological evaluation of DNA extraction method effects on the bacterial microbiome and resistome in sputum.
 16.08.2024, _A Benčič, N Toplak, S Koren, A Bogožalec Košir, M Milavec, V Tomič, D Lužnik, T Dreo_


Targeted high-throughput sequencing (HTS) has revolutionized the way we look at bacterial communities. It can be used for the species-specific detection of bacteria as well as for the determination of the microbiome and resistome and can be applied to samples from almost any environment. However, the results of targeted HTS can be influenced by many factors, which poses a major challenge for its use in clinical diagnostics. In this study, we investigated the impact of the DNA extraction method on the determination of the bacterial microbiome and resistome by targeted HTS using principles from metrology and diagnostics such as repeatability and analytical sensitivity. Sputum samples spiked with Acinetobacter baumannii, Klebsiella pneumoniae, and Pseudomonas aeruginosa at three different concentrations (103-106 cells/mL) were used. DNA was extracted from each sample on 2 separate days in three replicates each using three different extraction methods based on cetrimonium bromide, magnetic beads, and silica membranes. All three spiked bacteria were detected in sputum, and the DNA extraction method had no significant effect on detection. However, the DNA extraction method had significant effects on the composition of the microbiome and the resistome. The sequencing results were repeatable in the majority of cases. The silica membrane-based DNA extraction kit provided the most repeatable results and the highest diversity of the microbiome and resistome. Targeted HTS has been shown to be a reliable tool for determining the microbiome and resistome; however, the method of DNA extraction should be carefully selected to minimize its impact on the results.

[Source](https://doi.org/10.1128/msystems.00735-24)

10.1128/msystems.00735-24

---

## AI in microbiome research: Where have we been, where are we going?
 15.08.2024, _GK Gerber_


_Artificial Intelligence, Humans, Microbiota, Software_
Artificial intelligence (AI), a subdiscipline of computer science that develops machines or software that mimic characteristically human cognitive functions, is currently undergoing a revolution. In this commentary article, I will give my perspective on the evolution of the field and my thoughts on how AI may impact microbiome research in the next decade.

[Source](https://www.cell.com/cell-host-microbe/abstract/S1931-3128(24)00280-4)

10.1016/j.chom.2024.07.021

---

## Modeling Microbial Communities: Perspective and Challenges.
 16.08.2024, _L Raajaraam, K Raman_


_Microbiota, Models, Biological, Machine Learning_
Microbial communities are immensely important due to their widespread presence and profound impact on various facets of life. Understanding these complex systems necessitates mathematical modeling, a powerful tool for simulating and predicting microbial community behavior. This review offers a critical analysis of metabolic modeling and highlights key areas that would greatly benefit from broader discussion and collaboration. Moreover, we explore the challenges and opportunities linked to the intricate nature of these communities, spanning data generation, modeling, and validation. We are confident that ongoing advancements in modeling techniques, such as machine learning, coupled with interdisciplinary collaborations, will unlock the full potential of microbial communities across diverse applications.

[Source](https://doi.org/10.1021/acssynbio.4c00116)

10.1021/acssynbio.4c00116

---

