## Diet-omics in the Study of Urban and Rural Crohn disease Evolution (SOURCE) cohort

[Source](https://www.nature.com/articles/s41467-024-48106-6)

---

## Improvements in Gut Microbiome Composition and Clinical Symptoms Following Familial Fecal Microbiota Transplantation in a Nineteen-Year-Old Adolescent With Severe Autism

[Source](https://www.journalmc.org/index.php/JMC/article/view/4209)

---

## Interplay between particle size and microbial ecology in the gut microbiome

[Source](https://www.biorxiv.org/content/10.1101/2024.04.26.591376v1)

---

## Shotgun metagenomic analysis of saliva microbiome suggests <em>Mogibacterium</em> as a factor associated with chronic bacterial osteomyelitis

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0302569)

---

## Pesticide exposure and the microbiota-gut-brain axis

[Source](https://www.nature.com/articles/s41396-023-01450-9)

---

## Short-term exposure to antibiotics begets long-term disturbance in gut microbial metabolism and molecular ecological networks

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01795-z)

---

## Synthesis of recovery patterns in microbial communities across environments

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01802-3)

---

## Cooperative interactions between invader and resident microbial community members weaken the negative diversity-invasion relationship

[Source](https://onlinelibrary.wiley.com/doi/10.1111/ele.14433)

---

## High speciation rate of niche specialists in hot springs

[Source](https://www.nature.com/articles/s41396-023-01447-4)

---

## Effects of reduced snowpack due to climate warming on abiotic and biotic soil properties in alpine and boreal forest systems

[Source](https://journals.plos.org/climate/article)

---

## Microbiota recovery in a chronosquences of impoverished Cerrado soils with biosolids applications 

[Source](https://www.sciencedirect.com/science/article/pii/S004896972403105X)

---

## Melatonin promotes the recovery of apple plants after waterlogging by shaping the structure and function of the rhizosphere microbiome

[Source](https://onlinelibrary.wiley.com/doi/10.1111/pce.14903)

---

## Antimony efflux underpins phosphorus cycling and resistance of phosphate-solubilizing bacteria in mining soils.
 22.07.2023, _S Liu, J Zeng, H Yu, C Wang, Y Yang, J Wang, Z He, Q Yan_


_Antimony, Soil, Phosphates, Phosphorus, Phylogeny, Environmental Monitoring, Soil Pollutants, Metals, Heavy, Bacteria, Microbiota, China, Soil Microbiology_
Microorganisms play crucial roles in phosphorus (P) turnover and P bioavailability increases in heavy metal-contaminated soils. However, microbially driven P-cycling processes and mechanisms of their resistance to heavy metal contaminants remain poorly understood. Here, we examined the possible survival strategies of P-cycling microorganisms in horizontal and vertical soil samples from the world's largest antimony (Sb) mining site, which is located in Xikuangshan, China. We found that total soil Sb and pH were the primary factors affecting bacterial community diversity, structure and P-cycling traits. Bacteria with the gcd gene, encoding an enzyme responsible for gluconic acid production, largely correlated with inorganic phosphate (Pi) solubilization and significantly enhanced soil P bioavailability. Among the 106 nearly complete bacterial metagenome-assembled genomes (MAGs) recovered, 60.4% carried the gcd gene. Pi transportation systems encoded by pit or pstSCAB were widely present in gcd-harboring bacteria, and 43.8% of the gcd-harboring bacteria also carried the acr3 gene encoding an Sb efflux pump. Phylogenetic and potential horizontal gene transfer (HGT) analyses of acr3 indicated that Sb efflux could be a dominant resistance mechanism, and two gcd-harboring MAGs appeared to acquire acr3 through HGT. The results indicated that Sb efflux could enhance P cycling and heavy metal resistance in Pi-solubilizing bacteria in mining soils. This study provides novel strategies for managing and remediating heavy metal-contaminated ecosystems.

[Source](https://www.nature.com/articles/s41396-023-01445-6)

10.1038/s41396-023-01445-6

---

## Antibiotic exposure alters the honeybee gut microbiota and may interfere with the honeybee behavioral caste transition.
 08.05.2024, _Z Zhang, X Mu, Q Cao, Y Zhai, L Zheng, Y Liu, H Zheng, X Zhang_


Behavioral division is essential for the sustainability and reproduction of honeybee populations. While accumulating evidence has documented that antibiotic exposure interferes with bee behavioral divisions, how the gut microbiome, host physiology, and genetic regulation are implicated in this process remains understudied. Here, by constructing single-cohort colonies, we validated that the gut microbiota varied in composition between age-matched nurse and forager bees. Perturbing the gut microbiota with a low dose of antibiotic retained the gut bacterial size, but the structure of the microbial community continuously diverged from the control group after antibiotic treatment. Fewer foragers were observed in the antibiotic groups in the field experiment. A combinatorial effect of decreased gut metabolic gene repertoires, reduced brain neurotransmitter titers, and downregulated brain immune genes could potentially be related to behavioral tasks transition delay. This work indicates that the disturbance to both the gut microbiome and host physiologies after antibiotic exposure may have implications on social behavior development, highlighting the need for further research focusing on antibiotic pollution threatening the honeybee population's health.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/1744-7917.13374)

10.1111/1744-7917.13374

---

## Meta-analysis of the Cetacea gut microbiome: Diversity, co-evolution, and interaction with the anthropogenic pathobiome.
 07.05.2024, _E Radaelli, G Palladino, E Nanetti, D Scicchitano, S Rampelli, S Airoldi, M Candela, M Marangi_


Despite their critical roles in marine ecosystems, only few studies have addressed the gut microbiome (GM) of cetaceans in a comprehensive way. Being long-living apex predators with a carnivorous diet but evolved from herbivorous ancestors, cetaceans are an ideal model for studying GM-host evolutionary drivers of symbiosis and represent a valuable proxy of overall marine ecosystem health. Here, we investigated the GM of eight different cetacean species, including both Odontocetes (toothed whales) and Mysticetes (baleen whales), by means of 16S rRNA-targeted amplicon sequencing. We collected faecal samples from free-ranging cetaceans circulating within the Pelagos Sanctuary (North-western Mediterranean Sea) and we also included publicly available cetacean gut microbiome sequences. Overall, we show a clear GM trajectory related to host phylogeny and taxonomy (i.e., phylosymbiosis), with remarkable GM variations which may reflect adaptations to different diets between baleen and toothed whales. While most samples were found to be infected by protozoan parasites of potential anthropic origin, we report that this phenomenon did not lead to severe GM dysbiosis. This study underlines the importance of both host phylogeny and diet in shaping the GM of cetaceans, highlighting the role of neutral processes as well as environmental factors in the establishment of this GM-host symbiosis. Furthermore, the presence of potentially human-derived protozoan parasites in faeces of free-ranging cetaceans emphasizes the importance of these animals as bioindicators of anthropic impact on marine ecosystems.

[Source](https://www.sciencedirect.com/science/article/pii/S0048969724030900)

10.1016/j.scitotenv.2024.172943

---

## Comparative analysis of gut microbiome of mangrove brachyuran crabs revealed patterns of phylosymbiosis and codiversification.
 07.05.2024, _CTT Tsang, TKL Hui, NM Chung, WT Yuen, LM Tsang_


The acquisition of microbial symbionts enables animals to rapidly adapt to and exploit novel ecological niches, thus significantly enhancing the evolutionary fitness and success of their hosts. However, the dynamics of host-microbe interactions and their evolutionary implications remain largely underexplored in marine invertebrates. Crabs of the family Sesarmidae (Crustacea: Brachyura) are dominant inhabitants of mangrove forests and are considered keystone species there. Their rapid diversification, particularly after adopting a plant-feeding lifestyle, is believed to have been facilitated by symbiotic gut microbes, enabling successful colonization of intertidal and terrestrial environments. To investigate the patterns and mechanisms shaping the microbial communities and the role of microbes in the evolution of Sesarmidae, we characterized and compared the gut microbiome compositions across 43 crab species from Sesarmidae and other mangrove-associated families using 16S metabarcoding. We found that the gut microbiome assemblages in crabs are primarily determined by host identity, with a secondary influence from environmental factors such as microhabitat and sampling location, and to a lesser extent influenced by biological factors such as sex and gut region. While patterns of phylosymbiosis (i.e. when microbial community relationships recapitulate the phylogeny of their hosts) were consistently observed in all beta-diversity metrics analysed, the strength of phylosymbiosis varied across crab families. This suggests that the bacterial assemblages in each family were differentially shaped by different degrees of host filtering and/or other evolutionary processes. Notably, Sesarmidae displayed signals of cophylogeny with its core gut bacterial genera, which likely play crucial functional roles in their hosts by providing lignocellulolytic enzymes, essential amino acids, and fatty acids supplementation. Our results support the hypothesis of microbial contribution to herbivory and terrestrialization in mangrove crabs, highlighting the tight association and codiversification of the crab holobiont.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/mec.17377)

10.1111/mec.17377

---

## A perfusion host-microbe bioreactor (HMB) system that captures dynamic interactions of secreted metabolites between epithelial cells cocultured with a human gut anaerobe.
 08.05.2024, _J Yang, J Cassaday, TP Wyche, B Squadroni, W Newhard, H Trinh, D Cabral, E Hett, TR Sana, K Lee, S Kasper_


The human microbiota impacts a variety of diseases and responses to therapeutics. Due to a lack of robust in vitro models, detailed mechanistic explanations of host-microbiota interactions cannot often be recapitulated. We describe the design and development of a novel, versatile and modular in vitro system that enables indirect coculture of human epithelial cells with anaerobic bacteria for the characterization of host-microbe secreted metabolite interactions. This system was designed to compartmentalize anaerobes and human cells in separate chambers conducive to each organism's requisite cell growth conditions. Using perfusion, fluidic mixing, and automated sample collection, the cells continuously received fresh media, while in contact with their corresponding compartments conditioned supernatant. Supernatants from each chamber were collected in a cell-free time-resolved fashion. The system sustained low oxygen conditions in the anaerobic chamber, while also supporting the growth of a representative anaerobe (Bacteroides thetaiotaomicron) and a human colonic epithelial cell line (Caco-2) in the aerobic chamber. Caco-2 global gene expression changes in response to coculture with B. thetaiotaomicron was characterized using RNA sequencing. Extensive, targeted metabolomics analysis of over 150 central carbon metabolites was performed on the serially collected supernatants. We observed broad metabolite changes in host-microbe coculture, compared to respective mono-culture controls. These effects were dependent both on sampling time and the compartment probed (apical vs. basolateral). Coculturing resulted in the depletion of several important metabolites, including guanine, uridine 5'-monophosphate, asparagine, and thiamine. Additionally, while Caco-2 cells cultured alone predominantly affected the basolateral metabolite milieu, increased abundance of 2,3-dihydroxyisovalerate and thymine on the basolateral side, occurred when the cells were cocultured with B. thetaiotaomicron. Thus, our system can capture the dynamic, competitive and cooperative processes between host cells and gut microbes.

[Source](https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/bit.28730)

10.1002/bit.28730

---

## Stable, fluorescent markers for tracking synthetic communities and assembly dynamics.
 10.05.2024, _B Jorrin, TL Haskett, HE Knights, A Martyn, TJ Underwood, J Dolliver, R Ledermann, PS Poole_


_Microbiota, DNA Transposable Elements, Rhizosphere, Plasmids, Plant Roots, Proteobacteria, Flow Cytometry, Luminescent Proteins, Soil Microbiology_
After two decades of extensive microbiome research, the current forefront of scientific exploration involves moving beyond description and classification to uncovering the intricate mechanisms underlying the coalescence of microbial communities. Deciphering microbiome assembly has been technically challenging due to their vast microbial diversity but establishing a synthetic community (SynCom) serves as a key strategy in unravelling this process. Achieving absolute quantification is crucial for establishing causality in assembly dynamics. However, existing approaches are primarily designed to differentiate a specific group of microorganisms within a particular SynCom.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01792-2)

10.1186/s40168-024-01792-2

---

## Sterile sentinels and MinION sequencing capture active soil microbial communities that differentiate crop rotations.
 10.05.2024, _SR Erlandson, PM Ewing, SL Osborne, RM Lehman_


Soil microbial communities are difficult to measure and critical to soil processes. The bulk soil microbiome is highly diverse and spatially heterogeneous, which can make it difficult to detect and monitor the responses of microbial communities to differences or changes in management, such as different crop rotations in agricultural research. Sampling a subset of actively growing microbes should promote monitoring how soil microbial communities respond to management by reducing the variation contributed by high microbial spatial and temporal heterogeneity and less active microbes. We tested an in-growth bag method using sterilized soil in root-excluding mesh, "sterile sentinels," for the capacity to differentiate between crop rotations. We assessed the utility of different incubation times and compared colonized sentinels to concurrently sampled bulk soils for the statistical power to differentiate microbial community composition in low and high diversity crop rotations. We paired this method with Oxford Nanopore MinION sequencing to assess sterile sentinels as a standardized, fast turn-around monitoring method.

[Source](https://environmentalmicrobiome.biomedcentral.com/articles/10.1186/s40793-024-00571-8)

10.1186/s40793-024-00571-8

---

