## The gut virome of healthy children during the first year of life is diverse and dynamic.
 14.09.2021, _B Taboada, P Morán, A Serrano-Vázquez, P Iša, L Rojas-Velázquez, H Pérez-Juárez, S López, J Torres, C Ximenez, CF Arias_


_Bacteriophages, DNA Viruses, Feces, Gastrointestinal Diseases, Gastrointestinal Tract, Humans, Infant, Infant, Newborn, Mexico, Virome_
In this work, we determined the diversity and dynamics of the gut virome of infants during the first year of life. Fecal samples were collected monthly, from birth to one year of age, from three healthy children living in a semi-rural village in Mexico. Most of the viral reads were classified into six families of bacteriophages including five dsDNA virus families of the order Caudovirales, with Siphoviridae and Podoviridae being the most abundant. Eukaryotic viruses were detected as early as two weeks after birth and remained present all along the first year of life. Thirty-four different eukaryotic virus families were found, where eight of these families accounted for 98% of all eukaryotic viral reads: Anelloviridae, Astroviridae, Caliciviridae, Genomoviridae, Parvoviridae, Picornaviridae, Reoviridae and the plant-infecting viruses of the Virgaviridae family. Some viruses in these families are known human pathogens, and it is surprising that they were found during the first year of life in infants without gastrointestinal symptoms. The eukaryotic virus species richness found in this work was higher than that observed in previous studies; on average between 7 and 24 virus species were identified per sample. The richness and abundance of the eukaryotic virome significantly increased during the second semester of life, probably because of an increased environmental exposure of infants with age. Our findings suggest an early and permanent contact of infants with a diverse array of bacteriophages and eukaryotic viruses, whose composition changes over time. The bacteriophages and eukaryotic viruses found in these children could represent a metastable virome, whose potential influence on the development of the infant's immune system or on the health of the infants later in life, remains to be investigated.

[Source](https://www.biorxiv.org/content/10.1101/2023.03.05.531203v1.abstract)

10.1371/journal.pone.0240958

---

## The influence of early life exposures on the infant gut virome

[Source](https://www.biorxiv.org/content/10.1101/2023.03.05.531203v1.abstract)

---

## Nurturing the Early Life Gut Microbiome and Immune Maturation for Long Term Health.
 30.11.2021, _SK Dogra, C Kwong Chung, D Wang, O Sakwinska, S Colombo Mottaz, N Sprenger_


Early life is characterized by developmental milestones such as holding up the head, turning over, sitting up and walking that are typically achieved sequentially in specific time windows. Similarly, the early gut microbiome maturation can be characterized by specific temporal microorganism acquisition, colonization and selection with differential functional features over time. This orchestrated microbial sequence occurs from birth during the first years of age before the microbiome reaches an adult-like composition and function between 3 and 5 years of age. Increasingly, these different steps of microbiome development are recognized as crucial windows of opportunity for long term health, primarily linked to appropriate immune and metabolic development. For instance, microbiome disruptors such as preterm and Cesarean-section birth, malnutrition and antibiotic use are associated with increased risk to negatively affect long-term immune and metabolic health. Different age discriminant microbiome taxa and functionalities are used to describe age-appropriate microbiome development, and advanced modelling techniques enable an understanding and visualization of an optimal microbiome maturation trajectory. Specific microbiome features can be related to later health conditions, however, whether such features have a causal relationship is the topic of intense research. Early life nutrition is an important microbiome modulator, and 'Mother Nature' provides the model with breast milk as the sole source of nutrition for the early postnatal period, while dietary choices during the prenatal and weaning period are to a large extent guided by tradition and culture. Increasing evidence suggests prenatal maternal diet and infant and child nutrition impact the infant microbiome trajectory and immune competence development. The lack of a universal feeding reference for such phases represents a knowledge gap, but also a great opportunity to provide adequate nutritional guidance to maintain an age-appropriate microbiome for long term health. Here, we provide a narrative review and perspective on our current understanding of age-appropriate microbiome maturation, its relation to long term health and how nutrition shapes and influences this relationship.

[Source](https://www.biorxiv.org/content/10.1101/2023.03.06.531289v1.abstract)

10.3390/microorganisms9102110

---

## Maternal diet and gut microbiome composition modulate early life immune responses

[Source](https://www.biorxiv.org/content/10.1101/2023.03.06.531289v1.abstract)

---

## Smoking-induced subgingival dysbiosis precedes clinical signs of periodontal disease.
 07.03.2023, _R Tamashiro, L Strange, K Schnackenberg, J Santos, H Gadalla, L Zhao, EC Li, E Hill, B Hill, GS Sidhu, M Kirst, C Walker, GP Wang_


Smoking accelerates periodontal disease and alters the subgingival microbiome. However, the relationship between smoking-associated subgingival dysbiosis and progression of periodontal disease is not well understood. Here, we sampled 233 subgingival sites longitudinally from 8 smokers and 9 non-smokers over 6-12 months, analyzing 804 subgingival plaque samples using 16 rRNA sequencing. At equal probing depths, the microbial richness and diversity of the subgingival microbiome was higher in smokers compared to non-smokers, but these differences decreased as probing depths increased. The overall subgingival microbiome of smokers differed significantly from non-smokers at equal probing depths, which was characterized by colonization of novel minority microbes and a shift in abundant members of the microbiome to resemble periodontally diseased communities enriched with pathogenic bacteria. Temporal analysis showed that microbiome in shallow sites were less stable than deeper sites, but temporal stability of the microbiome was not significantly affected by smoking status or scaling and root planing. We identified 7 taxa-Olsenella sp., Streptococcus cristatus, Streptococcus pneumoniae, Streptococcus parasanguinis, Prevotella sp., Alloprevotella sp., and a Bacteroidales sp. that were significantly associated with progression of periodontal disease. Taken together, these results suggest that subgingival dysbiosis in smokers precedes clinical signs of periodontal disease, and support the hypothesis that smoking accelerates subgingival dysbiosis to facilitate periodontal disease progression.

[Source](https://www.nature.com/articles/s41598-023-30203-z)

10.1038/s41598-023-30203-z

---

## Gut microbiota and risk of five common cancers: A univariable and multivariable Mendelian randomization study.
 07.03.2023, _Z Wei, B Yang, T Tang, Z Xiao, F Ye, X Li, S Wu, JG Huang, S Jiang_


Previous studies have linked gut microbiota with cancer etiology, but the associations for specific gut microbiota are causal or owing to bias remain to be elucidated.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/cam4.5772)

10.1002/cam4.5772

---

## The role of the sex hormone-gut microbiome axis in tumor immunotherapy.
 08.03.2023, _L Wang, L Tang, D Zhai, M Song, W Li, S Xu, S Jiang, H Meng, J Liang, Y Wang, B Zhang_


_Humans, Gastrointestinal Microbiome, Immunotherapy, Gonadal Steroid Hormones, Immune Checkpoint Inhibitors, Neoplasms_
Accumulating evidence suggested that both gut microbiome and sex play a critical role in the efficacy of immune checkpoint blockade therapy. Considering the reciprocal relationship between sex hormones and gut microbiome, the sex hormone-gut microbiome axis may participate in the regulation of the response to immune checkpoint inhibitors (ICIs). In this review, it was attempted to summarize the current knowledge about the influences of both sex and gut microbiome on the antitumor efficacy of ICIs and describe the interaction between sex hormones and gut microbiome. Accordingly, this review discussed the potential of enhancing the antitumor efficacy of ICIs through regulating the levels of sex hormones through manipulation of gut microbiome. Collectively, this review provided reliable evidence concerning the role of the sex hormone-gut microbiome axis in tumor immunotherapy.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2023.2185035)

10.1080/19490976.2023.2185035

---

## The Influence of the Microbiome on Aggressive Behavior: an Insight into Age Related Aggression.
 07.03.2023, _L Gulledge, D Oyebode, JR Donaldson_


Aggression is a complex psychological program that is influenced by genetics, environment, and psychological history. Research has shown that the hormonal levels in the body and the development of the brain can be major predictors of aggression. This review highlights recent studies that have connected the gut microbiome to alterations in hormones and brain development and how this can impact aggression. This paper also provides a systematic review on studies that directly assess the connection between the gut microbiome and aggression and reviews these connections in relation to age. We conclude with future directions that are needed to further determine the link between the microbiome and aggression among adolescents.

[Source](https://academic.oup.com/femsle/advance-article/doi/10.1093/femsle/fnac114/7070714)

10.1093/femsle/fnac114

---

## Gnotobiotic mice housing conditions critically influence the phenotype associated with transfer of faecal microbiota in a context of obesity

[Source](https://gut.bmj.com/content/early/2022/09/01/gutjnl-2021-326475)

---

## The coral microbiome: Towards an understanding of the molecular mechanisms of coral-microbiota interactions.
 07.03.2023, _AR Mohamed, MA Ochsenkühn, A Kazlak, A Moustafa, SA Amin_


Corals live in a complex, multi-partite symbiosis with diverse microbes across kingdoms, some of which are implicated in vital functions, such as those related to resilience against climate change. However, knowledge gaps and technical challenges limit our understanding of the nature and functional significance of complex symbiotic relationships within corals. Here, we provide an overview of the complexity of the coral microbiome focusing on taxonomic diversity and functions of well-studied and cryptic microbes. Mining the coral literature indicate that while corals collectively harbor a third of all marine bacterial phyla, known bacterial symbionts and antagonists of corals represent a minute fraction of this diversity and that these taxa cluster into select genera, suggesting selective evolutionary mechanisms enabled these bacteria to gain a niche within the holobiont. Recent advances in coral microbiome research aimed at leveraging microbiome manipulation to increase coral's fitness to help mitigate heat stress-related mortality are discussed. Then, insights into the potential mechanisms through which microbiota can communicate with and modify host responses are examined by describing known recognition patterns, potential microbially-derived coral epigenome effector proteins and coral gene regulation. Finally, the power of omics tools used to study corals are highlighted with emphasis on an integrated host-microbiota multiomics framework to understand the underlying mechanisms during symbiosis and climate change-driven dysbiosis.

[Source](https://academic.oup.com/femsre/advance-article/doi/10.1093/femsre/fuad005/7071893)

10.1093/femsre/fuad005

---

## Comparison of the gut microbiome and resistome in captive African and Asian elephants on the same diet.
 07.03.2023, _X Feng, R Hua, W Zhang, Y Liu, C Luo, T Li, X Chen, H Zhu, Y Wang, Y Lu_


Elephants are endangered species and threatened with extinction. They are monogastric herbivorous, hindgut fermenters and their digestive strategy requires them to consume large amounts of low quality forage. The gut microbiome is important to their metabolism, immune regulation, and ecological adaptation. Our study investigated the structure and function of the gut microbiota as well as the antibiotic resistance genes (ARGs) in captive African and Asian elephants on the same diet. Results showed that captive African and Asian elephants had distinct gut bacterial composition. MetaStats analysis showed that the relative abundance of Spirochaetes (FDR = 0.00) and Verrucomicrobia (FDR = 0.01) at the phylum level as well as Spirochaetaceae (FDR = 0.01) and Akkermansiaceae (FDR = 0.02) at the family level varied between captive African and Asian elephants. Among the top ten functional subcategories at level 2 (57 seed pathway) of Kyoto Encyclopedia of Genes and Genomes (KEGG) database, the relative gene abundance of cellular community-prokaryotes, membrane transport, and carbohydrate metabolism in African elephants were significantly lower than those in Asian elephants (0.98 vs. 1.03%, FDR = 0.04; 1.25 vs. 1.43%, FDR = 0.03; 3.39 vs. 3.63%; FDR = 0.02). Among the top ten functional subcategories at level 2 (CAZy family) of CAZy database, MetaStats analysis showed that African elephants had higher relative gene abundance of Glycoside Hydrolases family 28 (GH 28) compared to Asian elephants (0.10 vs. 0.08%, FDR = 0.03). Regarding the antibiotic resistance genes carried by gut microbes, MetaStats analysis showed that African elephants had significantly higher relative abundance of vanO (FDR = 0.00), tetQ (FDR = 0.04), and efrA (FDR = 0.04) than Asian elephants encoding resistance for glycopeptide, tetracycline, and macrolide/rifamycin/fluoroquinolone antibiotic, respectively. In conclusion, captive African and Asian elephants on the same diet have distinct gut microbial communities. Our findings established the ground work for future research on improving gut health of captive elephants.

[Source](https://www.frontiersin.org/articles/10.3389/fvets.2023.986382/full)

10.3389/fvets.2023.986382

---

## Core and conditionally rare taxa as indicators of agricultural drainage ditch and stream health and function.
 07.03.2023, _Y Shi, IUH Khan, D Radford, G Guo, M Sunohara, E Craiovan, DR Lapen, P Pham, W Chen_


The freshwater microbiome regulates aquatic ecological functionality, nutrient cycling, pathogenicity, and has the capacity to dissipate and regulate pollutants. Agricultural drainage ditches are ubiquitous in regions where field drainage is necessary for crop productivity, and as such, are first-line receptors of agricultural drainage and runoff. How bacterial communities in these systems respond to environmental and anthropogenic stressors are not well understood. In this study, we carried out a three year study in an agriculturally dominated river basin in eastern Ontario, Canada to explore the spatial and temporal dynamics of the core and conditionally rare taxa (CRT) of the instream bacterial communities using a 16S rRNA gene amplicon sequencing approach. Water samples were collected from nine stream and drainage ditch sites that represented the influence of a range of upstream land uses.

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-023-02755-7)

10.1186/s12866-023-02755-7

---

## Species invasions shift microbial phenology in a two-decade freshwater time series

[Source](https://www.pnas.org/eprint/4YR8GPQQIH4PKZTAXRE6/full)

---

## Controlling taxa abundance improves metatranscriptomics differential analysis.
 07.03.2023, _Z Ji, L Ma_


A common task in analyzing metatranscriptomics data is to identify microbial metabolic pathways with differential RNA abundances across multiple sample groups. With information from paired metagenomics data, some differential methods control for either DNA or taxa abundances to address their strong correlation with RNA abundance. However, it remains unknown if both factors need to be controlled for simultaneously.

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-023-02799-9)

10.1186/s12866-023-02799-9

---

## coda4microbiome: compositional data analysis for microbiome cross-sectional and longitudinal studies.
 08.03.2023, _ML Calle, M Pujolassos, A Susin_


_Infant, Humans, Cross-Sectional Studies, Algorithms, Data Analysis, Longitudinal Studies, Microbiota_
One of the main challenges of microbiome analysis is its compositional nature that if ignored can lead to spurious results. Addressing the compositional structure of microbiome data is particularly critical in longitudinal studies where abundances measured at different times can correspond to different sub-compositions.

[Source](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-023-05205-3)

10.1186/s12859-023-05205-3

---

## You can move, but you can’t hide: identification of mobile genetic elements with geNomad

[Source](https://www.biorxiv.org/content/10.1101/2023.03.05.531206v1)

---

## Assembling the perfect bacterial genome using Oxford Nanopore and Illumina sequencing.
 06.03.2023, _RR Wick, LM Judd, KE Holt_


_Animals, Nanopores, High-Throughput Nucleotide Sequencing, Genome, Bacterial, Oryzias, Technology_
A perfect bacterial genome assembly is one where the assembled sequence is an exact match for the organism's genome-each replicon sequence is complete and contains no errors. While this has been difficult to achieve in the past, improvements in long-read sequencing, assemblers, and polishers have brought perfect assemblies within reach. Here, we describe our recommended approach for assembling a bacterial genome to perfection using a combination of Oxford Nanopore Technologies long reads and Illumina short reads: Trycycler long-read assembly, Medaka long-read polishing, Polypolish short-read polishing, followed by other short-read polishing tools and manual curation. We also discuss potential pitfalls one might encounter when assembling challenging genomes, and we provide an online tutorial with sample data (github.com/rrwick/perfect-bacterial-genome-tutorial).

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010905)

10.1371/journal.pcbi.1010905

---

