## The CRCbiome study: a large prospective cohort study examining the role of lifestyle and the gut microbiome in colorectal cancer screening participants.
 22.08.2021, _AS Kværner, E Birkeland, C Bucher-Johannessen, E Vinberg, JI Nordby, H Kangas, V Bemanian, P Ellonen, E Botteri, E Natvig, T Rognes, E Hovig, R Lyle, OH Ambur, WM de Vos, S Bultman, A Hjartåker, R Landberg, M Song, HS Blix, G Ursin, KR Randel, T de Lange, G Hoff, Ø Holme, P Berstad, TB Rounge_


Colorectal cancer (CRC) screening reduces CRC incidence and mortality. However, current screening methods are either hampered by invasiveness or suboptimal performance, limiting their effectiveness as primary screening methods. To aid in the development of a non-invasive screening test with improved sensitivity and specificity, we have initiated a prospective biomarker study (CRCbiome), nested within a large randomized CRC screening trial in Norway. We aim to develop a microbiome-based classification algorithm to identify advanced colorectal lesions in screening participants testing positive for an immunochemical fecal occult blood test (FIT). We will also examine interactions with host factors, diet, lifestyle and prescription drugs. The prospective nature of the study also enables the analysis of changes in the gut microbiome following the removal of precancerous lesions.

[Source](https://www.nature.com/articles/s41579-021-00562-3)

10.1186/s12885-021-08640-8

---

## Host specificity of the gut microbiome

[Source](https://www.nature.com/articles/s41579-021-00562-3)

---

## Harnessing the power of host-microbe symbioses to address grand challenges.
 14.09.2021, _U Hentschel_



[Source](https://www.nature.com/articles/s41579-021-00619-3)

10.1038/s41579-021-00619-3

---

## The intratumoral microbiome: Characterization methods and functional impact.
 13.09.2021, _CJF Heymann, JM Bard, MF Heymann, D Heymann, C Bobin-Dubigeon_


Live-pathogenic bacteria, which were identified inside tumors hundreds year ago, are key elements in modern cancer research. As they have a relatively accessible genome, they offer a multitude of metabolic engineering opportunities, useful in several clinical fields. Better understanding of the tumor microenvironment and its associated microbiome would help conceptualize new metabolically engineered species, triggering efficient therapeutic responses against cancer. Unfortunately, given the low microbial biomass nature of tumors, characterizing the tumor microbiome remains a challenge. Tumors have a high host versus bacterial DNA ratio, making it extremely complex to identify tumor-associated bacteria. Nevertheless, with the improvements in next-generation analytic tools, recent studies demonstrated the existence of intratumor bacteria inside defined tumors. It is now proven that each cancer subtype has a unique microbiome, characterized by bacterial communities with specific metabolic functions. This review provides a brief overview of the main approaches used to characterize the tumor microbiome, and of the recently proposed functions of intracellular bacteria identified in oncological entities. The therapeutic aspects of live-pathogenic microbes are also discussed, regarding the tumor microenvironment of each cancer type.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0304383521004560)

10.1016/j.canlet.2021.09.009

---

## Oral hygiene and oral microbiota in children and young people with neurological impairment and oropharyngeal dysphagia

[Source](https://www.nature.com/articles/s41598-021-97425-x)

---

## The Application of The Skin Virome for Human Identification

[Source](https://www.biorxiv.org/content/10.1101/2021.09.10.459834v1.abstract)

---

## Vaginal microbiome topic modeling of laboring Ugandan women with and without fever.
 14.09.2021, _M Movassagh, LM Bebell, K Burgoine, C Hehnly, L Zhang, K Moran, K Sheldon, SA Sinnar, E Mbabazi-Kabachelor, E Kumbakumba, J Bazira, M Ochora, R Mulondo, BK Nsubuga, AD Weeks, M Gladstone, P Olupot-Olupot, J Ngonzi, DJ Roberts, FA Meier, RA Irizarry, JR Broach, SJ Schiff, JN Paulson_


The composition of the maternal vaginal microbiome influences the duration of pregnancy, onset of labor, and even neonatal outcomes. Maternal microbiome research in sub-Saharan Africa has focused on non-pregnant and postpartum composition of the vaginal microbiome. Here we aimed to illustrate the relationship between the vaginal microbiome of 99 laboring Ugandan women and intrapartum fever using routine microbiology and 16S ribosomal RNA gene sequencing from two hypervariable regions (V1-V2 and V3-V4). To describe the vaginal microbes associated with vaginal microbial communities, we pursued two approaches: hierarchical clustering methods and a novel Grades of Membership (GoM) modeling approach for vaginal microbiome characterization. Leveraging GoM models, we created a basis composed of a preassigned number of microbial topics whose linear combination optimally represents each patient yielding more comprehensive associations and characterization between maternal clinical features and the microbial communities. Using a random forest model, we showed that by including microbial topic models we improved upon clinical variables to predict maternal fever. Overall, we found a higher prevalence of Granulicatella, Streptococcus, Fusobacterium, Anaerococcus, Sneathia, Clostridium, Gemella, Mobiluncus, and Veillonella genera in febrile mothers, and higher prevalence of Lactobacillus genera (in particular L. crispatus and L. jensenii), Acinobacter, Aerococcus, and Prevotella species in afebrile mothers. By including clinical variables with microbial topics in this model, we observed young maternal age, fever reported earlier in the pregnancy, longer labor duration, and microbial communities with reduced Lactobacillus diversity were associated with intrapartum fever. These results better defined relationships between the presence or absence of intrapartum fever, demographics, peripartum course, and vaginal microbial topics, and expanded our understanding of the impact of the microbiome on maternal and potentially neonatal outcome risk.

[Source](https://www.nature.com/articles/s41522-021-00244-1)

10.1038/s41522-021-00244-1

---

## Plasma metagenomic sequencing to detect and quantify bacterial DNA in ICU patients suspected of sepsis: a proof-of-principle study.
 12.09.2021, _MT Kisat, A Odenheimer-Bergman, H Markus, B Joseph, SN Srivatsan, T Contente-Cuomo, Z Khalpey, P Keim, T O'Keeffe, R Askari, A Salim, P Rhee, M Murtaza_


Timely recognition of sepsis and identification of pathogens can improve outcomes in critical care patients but microbial cultures have low accuracy and long turnaround times. In this proof-of-principle study, we describe metagenomic sequencing and analysis of non-human DNA in plasma. We hypothesized that quantitative analysis of bacterial DNA (bDNA) levels in plasma can enable detection and monitoring of pathogens.

[Source](https://journals.lww.com/jtrauma/Abstract/9000/Plasma_metagenomic_sequencing_to_detect_and.97305.aspx)

10.1097/TA.0000000000003396

---

## Plasma metagenomic sequencing to detect and quantify bacterial DNA in ICU patients suspected of sepsis

[Source](https://journals.lww.com/jtrauma/Abstract/9000/Plasma_metagenomic_sequencing_to_detect_and.97305.aspx)

---

## The gut microbiome in konzo.
 13.09.2021, _MS Bramble, N Vashist, A Ko, S Priya, C Musasa, A Mathieu, A Spencer, M Lupamba Kasendue, P Mamona Dilufwasayo, K Karume, J Nsibu, H Manya, MNA Uy, B Colwell, M Boivin, JPB Mayambu, D Okitundu, A Droit, D Mumba Ngoyi, R Blekhman, D Tshala-Katumbay, E Vilain_


Konzo, a distinct upper motor neuron disease associated with a cyanogenic diet and chronic malnutrition, predominately affects children and women of childbearing age in sub-Saharan Africa. While the exact biological mechanisms that cause this disease have largely remained elusive, host-genetics and environmental components such as the gut microbiome have been implicated. Using a large study population of 180 individuals from the Democratic Republic of the Congo, where konzo is most frequent, we investigate how the structure of the gut microbiome varied across geographical contexts, as well as provide the first insight into the gut flora of children affected with this debilitating disease using shotgun metagenomic sequencing. Our findings indicate that the gut microbiome structure is highly variable depending on region of sampling, but most interestingly, we identify unique enrichments of bacterial species and functional pathways that potentially modulate the susceptibility of konzo in prone regions of the Congo.

[Source](https://www.nature.com/articles/s41467-021-25694-1)

10.1038/s41467-021-25694-1

---

## High-salt diet mediates interplay between NK cells and gut microbiota to induce potent tumor immunity.
 13.09.2021, _ZA Rizvi, R Dalal, S Sadhu, Y Kumar, S Kumar, SK Gupta, MR Tripathy, DK Rathore, A Awasthi_


[Figure: see text].

[Source](https://www.science.org/doi/10.1126/sciadv.abg5016)

10.1126/sciadv.abg5016

---

## The Firmicutes/Bacteroidetes ratio of the human gut microbiota is associated with prostate enlargement.
 13.09.2021, _K Takezawa, K Fujita, M Matsushita, D Motooka, K Hatano, E Banno, N Shimizu, T Takao, S Takada, K Okada, S Fukuhara, H Kiuchi, H Uemura, S Nakamura, Y Kojima, N Nonomura_


The pathophysiology of the prostate enlargement underlying lower urinary tract symptoms is unknown. Meanwhile, the gut microbiota can contribute to various host conditions. We hypothesized that the gut microbiota plays a role in prostate enlargement.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/pros.24223)

10.1002/pros.24223

---

## Gut microbiota from green tea polyphenol-dosed mice improves intestinal epithelial homeostasis and ameliorates experimental colitis.
 13.09.2021, _Z Wu, S Huang, T Li, N Li, D Han, B Zhang, ZZ Xu, S Zhang, J Pang, S Wang, G Zhang, J Zhao, J Wang_


_Animals, Colitis, Dextran Sulfate, Disease Models, Animal, Gastrointestinal Microbiome, Homeostasis, Mice, Mice, Inbred C57BL, Polyphenols, Tea_
Alteration of the gut microbiota may contribute to the development of inflammatory bowel disease (IBD). Epigallocatechin-3-gallate (EGCG), a major bioactive constituent of green tea, is known to be beneficial in IBD alleviation. However, it is unclear whether the gut microbiota exerts an effect when EGCG attenuates IBD.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01115-9)

10.1186/s40168-021-01115-9

---

## Differential perturbations of gut microbial profiles and co-occurrence networks among phases of methamphetamine-induced conditioned place preference.
 12.09.2021, _Y Wang, J Zhang, C Deji, J Fan, X Miao, S Li, X Zeng, F Guan_


The gut-brain axis provides a pathway for the interaction between gut microbiota and methamphetamine (METH) addiction. However, the gut microbial signatures during different phases of METH use remain unclear. In the present study, we established models of acquisition, extinction, and reinstatement of METH-induced conditioned place preference (CPP) in male mice and detected the gut microbiome profiles of the fecal samples at the three phases by 16S rRNA gene sequencing. Our results revealed that the richness of the gut microbiome increased following repeated METH administration, and it decreased after 4 weeks of abstinence. The microbial richness remained at a low level after one METH challenge at the reinstatement phase. The abundance of several genera including Prevotella, Bacteroides, and Lactobacillus differentially altered among phases of METH-induced CPP. The co-occurrence networks of the gut microbiome became weaker and more unstable during the development of METH-induced CPP at the extinction and reinstatement phases. Notably, the predicted gene functions of short-chain fatty acid metabolism, which were correlated with the abundance of Prevotella, Bacteroides, and Lactobacillus, were found differentially enriched among phases of METH-induced CPP. Our findings highlight a potential association between perturbations of the gut microbiome and different phases of METH use.

[Source](https://onlinelibrary.wiley.com/doi/10.1002/jnr.24963)

10.1002/jnr.24963

---

## Gut microbiota regulation of P-glycoprotein in the intestinal epithelium in maintenance of homeostasis.
 13.09.2021, _SE Foley, C Tuohy, M Dunford, MJ Grey, H De Luca, C Cawley, RL Szabady, A Maldonado-Contreras, JM Houghton, DV Ward, RJ Mrsny, BA McCormick_


_ATP Binding Cassette Transporter, Subfamily B, ATP Binding Cassette Transporter, Subfamily B, Member 1, Animals, Gastrointestinal Microbiome, Homeostasis, Humans, Intestinal Mucosa, Mice_
P-glycoprotein (P-gp) plays a critical role in protection of the intestinal epithelia by mediating efflux of drugs/xenobiotics from the intestinal mucosa into the gut lumen. Recent studies bring to light that P-gp also confers a critical link in communication between intestinal mucosal barrier function and the innate immune system. Yet, despite knowledge for over 10 years that P-gp plays a central role in gastrointestinal homeostasis, the precise molecular mechanism that controls its functional expression and regulation remains unclear. Here, we assessed how the intestinal microbiome drives P-gp expression and function.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01137-3)

10.1186/s40168-021-01137-3

---

## Vitamin A supplementation ameliorates ulcerative colitis in gut microbiota-dependent manner.
 13.09.2021, _B Pang, H Jin, N Liao, J Li, C Jiang, J Shi_


Ulcerative colitis (UC), is a chronic relapsing inflammatory condition of the gastrointestinal track. The purpose of this study is to explore whether Vitamin A (VA) can treat UC and its mechanisms. A mouse model of UC was established using 3.0% (w/v) dextran sodium sulfate (DSS). VA was used to treat UC by intragastric administration of 5000 international unit (IU) retinyl acetate. Fecal microbiota transplantation (FMT) was also used to treat the UC model mice to verify the effect of influenced gut microbiota. The content of short-chain fatty acids (SCFAs) in cecal contents was quantitatively detected by gas chromatography and mass spectrometry. VA supplementation significantly ameliorated UC. 16S rRNA sequencing indicated that VA-treated mice exhibited much more abundant gut microbial diversity and flora composition. Targeted metabolomics analysis manifested the increased production of SCFAs in VA-treated mice. Gut microbiota depletion and FMT results confirmed the gut microbiota-dependent mechanism as that VA relieved UC via regulating gut microbiota: increase in SCFA-producing genera and decrease in UC-related genera. The restore of intestinal barrier and the inhibition of inflammation were also found to contribute to the amelioration of UC by VA. It was concluded that a VA supplement was enough to cause a significant change in gut microbiota and amelioration of UC.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0963996921004671)

10.1016/j.foodres.2021.110568

---

## Dynamics of the fecal microbiome and antimicrobial resistome in commercial piglets during the weaning period.
 14.09.2021, _P Suriyaphol, JKH Chiu, N Yimpring, P Tunsagool, W Mhuantong, R Chuanchuen, I Bessarab, RBH Williams, RT Ong, G Suriyaphol_


This study aimed to characterize the alteration of the fecal microbiome and antimicrobial resistance (AMR) determinants in 24 piglets at day 3 pre-weaning (D. - 3), weaning day (D.0), days 3 (D.3) and 8 post-weaning (D.8), using whole-genome shotgun sequencing. Distinct clusters of microbiomes and AMR determinants were observed at D.8 when Prevotella (20.9%) was the major genus, whereas at D. - 3-D.3, Alistipes (6.9-12.7%) and Bacteroides (5.2-8.5%) were the major genera. Lactobacillus and Escherichia were notably observed at D. - 3 (1.2%) and D. - 3-D.3 (0.2-0.4%), respectively. For AMR, a distinct cluster of AMR determinants was observed at D.8, mainly conferring resistance to macrolide-lincosamide-streptogramin (mefA), β-lactam (cfxA6 and aci1) and phenicol (rlmN). In contrast, at D. - 3-D.3, a high abundance of determinants with aminoglycoside (AMG) (sat, aac(6')-aph(2''), aadA and acrF), β-lactam (fus-1, cepA and mrdA), multidrug resistance (MDR) (gadW, mdtE, emrA, evgS, tolC and mdtB), phenicol (catB4 and cmlA4), and sulfonamide patterns (sul3) was observed. Canonical correlation analysis (CCA) plot associated Escherichia coli with aac(6')-aph(2''), emrA, mdtB, catB4 and cmlA4 at D. - 3, D.0 and/or D.3 whereas at D.8 associations between Prevotella and mefA, cfxA6 and aci1 were identified. The weaning age and diet factor played an important role in the microbial community composition.

[Source](https://www.nature.com/articles/s41598-021-97586-9)

10.1038/s41598-021-97586-9

---

## A statistical model for describing and simulating microbial community profiles.
 13.09.2021, _S Ma, B Ren, H Mallick, YS Moon, E Schwager, S Maharjan, TL Tickle, Y Lu, RN Carmody, EA Franzosa, L Janson, C Huttenhower_


Many methods have been developed for statistical analysis of microbial community profiles, but due to the complex nature of typical microbiome measurements (e.g. sparsity, zero-inflation, non-independence, and compositionality) and of the associated underlying biology, it is difficult to compare or evaluate such methods within a single systematic framework. To address this challenge, we developed SparseDOSSA (Sparse Data Observations for the Simulation of Synthetic Abundances): a statistical model of microbial ecological population structure, which can be used to parameterize real-world microbial community profiles and to simulate new, realistic profiles of known structure for methods evaluation. Specifically, SparseDOSSA's model captures marginal microbial feature abundances as a zero-inflated log-normal distribution, with additional model components for absolute cell counts and the sequence read generation process, microbe-microbe, and microbe-environment interactions. Together, these allow fully known covariance structure between synthetic features (i.e. "taxa") or between features and "phenotypes" to be simulated for method benchmarking. Here, we demonstrate SparseDOSSA's performance for 1) accurately modeling human-associated microbial population profiles; 2) generating synthetic communities with controlled population and ecological structures; 3) spiking-in true positive synthetic associations to benchmark analysis methods; and 4) recapitulating an end-to-end mouse microbiome feeding experiment. Together, these represent the most common analysis types in assessment of real microbial community environmental and epidemiological statistics, thus demonstrating SparseDOSSA's utility as a general-purpose aid for modeling communities and evaluating quantitative methods. An open-source implementation is available at http://huttenhower.sph.harvard.edu/sparsedossa2.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008913)

10.1371/journal.pcbi.1008913

---

