## Human gut mycobiota tune immunity via CARD9-dependent induction of anti-fungal IgG antibodies.
 06.02.2021, _I Doron, I Leonardi, XV Li, WD Fiers, A Semon, M Bialt-DeCelie, M Migaud, IH Gao, WY Lin, T Kusakabe, A Puel, ID Iliev_


Antibodies mediate natural and vaccine-induced immunity against viral and bacterial pathogens, whereas fungi represent a widespread kingdom of pathogenic species for which neither vaccine nor neutralizing antibody therapies are clinically available. Here, using a multi-kingdom antibody profiling (multiKAP) approach, we explore the human antibody repertoires against gut commensal fungi (mycobiota). We identify species preferentially targeted by systemic antibodies in humans, with Candida albicans being the major inducer of antifungal immunoglobulin G (IgG). Fungal colonization of the gut induces germinal center (GC)-dependent B cell expansion in extraintestinal lymphoid tissues and generates systemic antibodies that confer protection against disseminated C. albicans or C. auris infection. Antifungal IgG production depends on the innate immunity regulator CARD9 and CARD9+CX3CR1+ macrophages. In individuals with invasive candidiasis, loss-of-function mutations in CARD9 are associated with impaired antifungal IgG responses. These results reveal an important role of gut commensal fungi in shaping the human antibody repertoire through CARD9-dependent induction of host-protective antifungal IgG.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(21)00059-3)

10.1016/j.cell.2021.01.016

---

## Functional and taxonomic comparison of mouse and human gut microbiotas using extensive culturing and metagenomics

[Source](https://www.biorxiv.org/content/10.1101/2021.02.11.430759v1.full.pdf)

---

## High intake of vegetables is linked to lower white blood cell profile and the effect is mediated by the gut microbiome.
 14.02.2021, _C Menni, P Louca, SE Berry, A Vijay, S Astbury, ER Leeming, R Gibson, F Asnicar, G Piccinno, J Wolf, R Davies, M Mangino, N Segata, TD Spector, AM Valdes_


Chronic inflammation, which can be modulated by diet, is linked to high white blood cell counts and correlates with higher cardiometabolic risk and risk of more severe infections, as in the case of COVID-19.

[Source](https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-021-01913-w)

10.1186/s12916-021-01913-w

---

## The Reactobiome Unravels a New Paradigm in Human Gut Microbiome Metabolism

[Source](https://www.biorxiv.org/content/10.1101/2021.02.01.428114v1.full.pdf)

---

## Spatial analysis of gut microbiome reveals a distinct ecological niche associated with the mucus layer.
 11.02.2021, _K Duncan, K Carey-Ewend, S Vaishnava_


Mucus-associated bacterial communities are critical for determining disease pathology and promoting colonization resistance. Yet the key ecological properties of mucus resident communities remain poorly defined. Using an approach that combines in situ hybridization, laser microdissection and 16s rRNA sequencing of spatially distinct regions of the mouse gut lumen, we discovered that a dense microbial community resembling a biofilm is embedded in the mucus layer. The mucus-associated biofilm-like community excluded bacteria belonging to phylum Proteobacteria. Additionally, it was significantly more diverse and consisted of bacterial species that were unique to it. By employing germ-free mice deficient in T and B lymphocytes we found that formation of biofilm-like structure was independent of adaptive immunity. Instead the integrity of biofilm-like community depended on Gram-positive commensals such as Clostridia. Additionally, biofilm-like community in the mucus lost fewer Clostridia and showed smaller bloom of Proteobacteria compared to the lumen upon antibiotic treatment. When subjected to time-restricted feeding biofilm-like structure significantly enhanced in size and showed enrichment of Clostridia. Taken together our work discloses that mucus-associated biofilm-like community represents a specialized community that is structurally and compositionally distinct that excludes aerobic bacteria while enriching for anaerobic bacteria such as Clostridia, exhibits enhanced stability to antibiotic treatment and that can be modulated by dietary changes.

[Source](https://www.tandfonline.com/doi/epub/10.1080/19490976.2021.1874815)

10.1080/19490976.2021.1874815

---

## Diversity and dynamism of IgA-microbiota interactions.
 11.02.2021, _KE Huus, C Petersen, BB Finlay_


IgA mediates microbial homeostasis at the intestinal mucosa. Within the gut, IgA acts in a context-dependent manner to both prevent and promote bacterial colonization and to influence bacterial gene expression, thus providing exquisite control of the microbiota. IgA-microbiota interactions are highly diverse across individuals and populations, yet the factors driving this variation remain poorly understood. In this Review, we summarize evidence for the host, bacterial and environmental factors that influence IgA-microbiota interactions. Recent advances have helped to clarify the antigenic specificity and immune selection of intestinal IgA and have highlighted the importance of microbial glycan recognition. Furthermore, emerging evidence suggests that diet and nutrition play an important role in shaping IgA recognition of the microbiota. IgA-microbiota interactions are disrupted during both overnutrition and undernutrition and may be altered dynamically in response to diet, with potential implications for host health. We situate this research in the context of outstanding questions and future directions in order to better understand the fascinating paradigm of IgA-microbiota homeostasis.

[Source](https://www.nature.com/articles/s41577-021-00506-1)

10.1038/s41577-021-00506-1

---

## Antibiotic-associated dysbiosis affects the ability of the gut microbiota to control intestinal inflammation upon fecal microbiota transplantation in experimental colitis models.
 12.02.2021, _F Strati, M Pujolassos, C Burrello, MR Giuffrè, G Lattanzi, F Caprioli, J Troisi, F Facciotti_


The gut microbiota plays a central role in host physiology and in several pathological mechanisms in humans. Antibiotics compromise the composition and functions of the gut microbiota inducing long-lasting detrimental effects on the host. Recent studies suggest that the efficacy of different clinical therapies depends on the action of the gut microbiota. Here, we investigated how different antibiotic treatments affect the ability of the gut microbiota to control intestinal inflammation upon fecal microbiota transplantation in an experimental colitis model and in ex vivo experiments with human intestinal biopsies.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00991-x)

10.1186/s40168-020-00991-x

---

## Adherent-invasive <em>E. coli</em> metabolism of propanediol in Crohn’s disease regulates phagocytes to drive intestinal inflammation 

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312821000329)

---

## Adults with Prader-Willi syndrome exhibit a unique microbiota profile.
 11.02.2021, _WJ Dahl, J Auger, Z Alyousif, JL Miller, TA Tompkins_


Adults with Prader-Willi syndrome (PWS) require less energy intake to maintain body weight than the general adult population. This, combined with their altered gastrointestinal transit time, may impact microbiota composition. The aim of the study was to determine if the fecal microbiota composition of adults with PWS differed from non-affected adults. Using usual diet/non-interventional samples, fecal microbiota composition was analyzed using 16S rRNA gene amplicon sequencing and data from adults with PWS were merged with four other adult cohorts that differed by geographical location and age. QIIME 2™ sample-classifier, machine learning algorithms were used to cross-train the samples and predict from which dataset the taxonomic profiles belong. Taxa that most distinguished between all datasets were extracted and a visual inspection of the R library PiratePlots was performed to select the taxa that differed in abundance specific to PWS.

[Source](https://bmcresnotes.biomedcentral.com/articles/10.1186/s13104-021-05470-6)

10.1186/s13104-021-05470-6

---

## Population density affects the outcome of competition in co-cultures of <em>Gardnerella </em>species isolated from the human vaginal microbiome

[Source](https://www.biorxiv.org/content/10.1101/2021.02.09.430506v1.full.pdf)

---

## Lactic acid-containing products for bacterial vaginosis and their impact on the vaginal microbiota: A systematic review.
 11.02.2021, _EL Plummer, CS Bradshaw, M Doyle, CK Fairley, GL Murray, D Bateson, L Masson, J Slifirski, G Tachedjian, LA Vodstrcil_


The vaginal microbiota in bacterial vaginosis (BV) typically has low abundance of lactic acid producing lactobacilli. Lactic acid has properties that may make it effective for treating BV and/or restoring an optimal lactobacillus-dominated vaginal microbiota. We conducted a systematic review to describe the effect of intravaginal lactic acid-containing products on BV cure, and their impact on vaginal microbiota composition (PROSPERO registration: CRD42018115982).

[Source](https://journals.plos.org/plosone/article/related?id=10.1371/journal.pone.0246953)

10.1371/journal.pone.0246953

---

## Functional and Taxonomic Dysbiosis of the Gut, Urine, and Semen Microbiomes in Male Infertility.
 12.02.2021, _SD Lundy, N Sangwan, NV Parekh, MKP Selvam, S Gupta, P McCaffrey, K Bessoff, A Vala, A Agarwal, ES Sabanegh, SC Vij, C Eng_


Little is known about the role of the genitourinary and gastrointestinal microbiota in the pathogenesis of male infertility.

[Source](https://www.europeanurology.com/article/S0302-2838(21)00019-1/fulltext)

10.1016/j.eururo.2021.01.014

---

## The microbiome and host mucosal interactions in urinary tract diseases.
 05.02.2021, _B Jones-Freeman, M Chonwerawong, VR Marcelino, AV Deshpande, SC Forster, MR Starkey_


The urinary tract consists of the bladder, ureters, and kidneys, and is an essential organ system for filtration and excretion of waste products and maintaining systemic homeostasis. In this capacity, the urinary tract is impacted by its interactions with other mucosal sites, including the genitourinary and gastrointestinal systems. Each of these sites harbors diverse ecosystems of microbes termed the microbiota, that regulates complex interactions with the local and systemic immune system. It remains unclear whether changes in the microbiota and associated metabolites may be a consequence or a driver of urinary tract diseases. Here, we review the current literature, investigating the impact of the microbiota on the urinary tract in homeostasis and disease including urinary stones, acute kidney injury, chronic kidney disease, and urinary tract infection. We propose new avenues for exploration of the urinary microbiome using emerging technology and discuss the potential of microbiome-based medicine for urinary tract conditions.

[Source](https://www.nature.com/articles/s41385-020-00372-5)

10.1038/s41385-020-00372-5

---

## Epidemiology of urinary tract infections in children: Causative bacteria and antimicrobial therapy.
 05.02.2021, _Y Tanaka, T Oishi, S Ono, M Kono, A Kato, I Miyata, N Ohno, K Ouchi_


Urinary tract infections (UTIs) are the most common bacterial infections in children. This study aimed to review characteristics of causative bacteria and effectiveness of antimicrobial therapy in children with febrile UTIs.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/ped.14639)

10.1111/ped.14639

---

## Repeatability and reproducibility assessment in a large-scale population-based microbiota study: case study on human milk microbiota

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00998-4)

---

## Respiratory microbiome in mechanically ventilated patients: a narrative review.
 11.02.2021, _M Fromentin, JD Ricard, D Roux_


The respiratory microbiome has been less explored than the gut microbiome. Despite the speculated importance of dysbiosis of the microbiome in ventilator-associated pneumonia (VAP) and acute respiratory distress syndrome (ARDS), only few studies have been performed in invasively ventilated ICU patients. And only the results of small cohorts have been published. An overlap exists between bacterial populations observed in the lower respiratory tract and the oropharyngeal tract. The bacterial microbiota is characterized by relatively abundant bacteria difficult to cultivate by standard methods. Under mechanical ventilation, a dysbiosis occurs with a drop overtime in diversity. During VAP development, lung dysbiosis is characterized by a shift towards a dominant bacterial pathogen (mostly Proteobacteria) whereas enrichment of gut-associated bacteria mainly Enterobacteriaceae is the specific feature discriminating ARDS patients. However, the role of this dysbiosis in VAP and ARDS pathogenesis is not yet fully understood. A more in-depth analysis of the interplay between bacteria, virus and fungi and a better understanding of the host-microbiome interaction could provide a more comprehensive view of the role of the microbiome in VAP and ARDS pathogenesis. Priority should be given to validate a consensual and robust methodology for respiratory microbiome research and to conduct longitudinal studies. A deeper understanding of microbial interplay should be a valuable guide for care of ARDS and VAP preventive/therapeutic strategies. We present a review on the current knowledge and expose perspectives and potential clinical applications of respiratory microbiome research in mechanically ventilated patients.

[Source](https://link.springer.com/article/10.1007%2Fs00134-020-06338-2)

10.1007/s00134-020-06338-2

---

## Genomic and functional analyses of fungal and bacterial consortia that enable lignocellulose breakdown in goat gut microbiomes.
 02.02.2021, _X Peng, SE Wilken, TS Lankiewicz, SP Gilmore, JL Brown, JK Henske, CL Swift, A Salamov, K Barry, IV Grigoriev, MK Theodorou, DL Valentine, MA O'Malley_


The herbivore digestive tract is home to a complex community of anaerobic microbes that work together to break down lignocellulose. These microbiota are an untapped resource of strains, pathways and enzymes that could be applied to convert plant waste into sugar substrates for green biotechnology. We carried out more than 400 parallel enrichment experiments from goat faeces to determine how substrate and antibiotic selection influence membership, activity, stability and chemical productivity of herbivore gut communities. We assembled 719 high-quality metagenome-assembled genomes (MAGs) that are unique at the species level. More than 90% of these MAGs are from previously unidentified herbivore gut microorganisms. Microbial consortia dominated by anaerobic fungi outperformed bacterially dominated consortia in terms of both methane production and extent of cellulose degradation, which indicates that fungi have an important role in methane release. Metabolic pathway reconstructions from MAGs of 737 bacteria, archaea and fungi suggest that cross-domain partnerships between fungi and methanogens enabled production of acetate, formate and methane, whereas bacterially dominated consortia mainly produced short-chain fatty acids, including propionate and butyrate. Analyses of carbohydrate-active enzyme domains present in each anaerobic consortium suggest that anaerobic bacteria and fungi employ mostly complementary hydrolytic strategies. The division of labour among herbivore anaerobes to degrade plant biomass could be harnessed for industrial bioprocessing.

[Source](https://www.nature.com/articles/s41564-020-00861-0)

10.1038/s41564-020-00861-0

---

## Metagenomics of antimicrobial and heavy metal resistance in the cecal microbiome of fattening pigs raised without antibiotics.
 06.02.2021, _P Tunsagool, W Mhuantong, S Tangphatsornruang, N Am-In, R Chuanchuen, T Luangtongkum, G Suriyaphol_


This study aimed to detect the cecal microbiome, antimicrobial resistance (AMR) and heavy metal resistance genes (MRGs) in fattening pigs raised under antibiotic-free (ABF) conditions compared with ordinary industrial pigs (control, C) using whole-genome shotgun sequencing. ABF pigs showed the enrichment of Prevotella (33%) and Lactobacillus (13%), whereas Escherichia coli (40%), Fusobacterium and Bacteroides (each at 4%) were notably observed in the C group. Distinct clusters of cecal microbiota of ABF and C pigs were revealed; however, microbiota of some C pigs (C1) appeared in the same cluster as ABF and were totally separated from the remaining C pigs (C2). For AMR genes, the highest abundance tet(Q) (35.7%) and mef(A) (12.7%) were markedly observed in the ABF group whereas tet(Q) (26.2%) and tet(W) (10.4%) were shown in the C group. tet(Q) was positively correlated to Prevotella in ABF and C1 samples. In the C2 group, the prominent tet(W) was positively correlated to Fusobacterium and Bacteroides Pigs have never received tetracycline but pregnant sows used chlortetracycline once 7 d before parturition. Chromosomal Cu and Zn resistance genes were also shown in both groups regardless the received Cu and Zn feed additives. A higher abundance of multi-metal resistance genes was observed in the C group (44%) compared with the ABF group (41%). In conclusion, the microbiome clusters in some C pigs were similar to that in ABF pigs. High abundant tetracycline resistance genes interrelated to major bacteria were observed in both ABF and C pigs. MRGs were also observed.IMPORTANCE: Owing to the increased problem of AMR in farm animals, raising farm animals without antibiotics is one method that could solve this problem. Our study showed that only some tetracycline and macrolide resistance genes, tet(Q), tet(W) and mef(A), were markedly abundant in ABF and C groups. The tet(Q) and tet(W) genes interrelated to different predominant bacteria in each group, showing the potential role of major bacteria as reservoirs of AMR genes. In addition, chromosomal Cu and Zn resistance genes were also observed in both pig groups, not depending on the use of Cu and Zn additives in both farms. The association of MRGs and AMR genotypes and phenotypes together with the method to re-sensitize bacteria to antibiotics should be studied further to unveil the cause of high resistance genes and solve the problems.

[Source](https://aem.asm.org/content/early/2021/02/02/AEM.02684-20)

10.1128/AEM.02684-20

---

## Unravelling the Role of Rumen Microbial Communities, Genes, and Activities on Milk Fatty Acid Profile Using a Combination of Omics Approaches.
 10.02.2021, _S Stergiadis, I Cabeza-Luna, M Mora-Ortiz, RD Stewart, RJ Dewhurst, DJ Humphries, M Watson, R Roehe, MD Auffret_


Milk products are an important component of human diets, with beneficial effects for human health, but also one of the major sources of nutritionally undesirable saturated fatty acids (SFA). Recent discoveries showing the importance of the rumen microbiome on dairy cattle health, metabolism and performance highlight that milk composition, and potentially milk SFA content, may also be associated with microorganisms, their genes and their activities. Understanding these mechanisms can be used for the development of cost-effective strategies for the production of milk with less SFA. This work aimed to compare the rumen microbiome between cows producing milk with contrasting FA profile and identify potentially responsible metabolic-related microbial mechanisms. Forty eight Holstein dairy cows were fed the same total mixed ration under the same housing conditions. Milk and rumen fluid samples were collected from all cows for the analysis of fatty acid profiles (by gas chromatography), the abundances of rumen microbiome communities and genes (by whole-genome-shotgun metagenomics), and rumen metabolome (using 500 MHz nuclear magnetic resonance). The following groups: (i) 24 High-SFA (66.9-74.4% total FA) vs. 24 Low-SFA (60.2-66.6%% total FA) cows, and (ii) 8 extreme High-SFA (69.9-74.4% total FA) vs. 8 extreme Low-SFA (60.2-64.0% total FA) were compared. Rumen of cows producing milk with more SFA were characterized by higher abundances of the lactic acid bacteria Lactobacillus, Leuconostoc, and Weissella, the acetogenic Proteobacteria Acetobacter and Kozakia, Mycobacterium, two fungi (Cutaneotrichosporon and Cyphellophora), and at a lesser extent Methanobrevibacter and the protist Nannochloropsis. Cows carrying genes correlated with milk FA also had higher concentrations of butyrate, propionate and tyrosine and lower concentrations of xanthine and hypoxanthine in the rumen. Abundances of rumen microbial genes were able to explain between 76 and 94% on the variation of the most abundant milk FA. Metagenomics and metabolomics analyses highlighted that cows producing milk with contrasting FA profile under the same diet, also differ in their rumen metabolic activities in relation to adaptation to reduced rumen pH, carbohydrate fermentation, and protein synthesis and metabolism.

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.590441/full)

10.3389/fmicb.2020.590441

---

## Evaluation of the ocular surface mycobiota in clinically normal horses

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0246537)

---

## Colonization at birth with human CST IV cervicovaginal microbiota alters development and increases neonatal mortality in mice

[Source](https://www.biorxiv.org/content/10.1101/2021.02.01.429213v1.full.pdf)

---

## Novel gut pathobionts confound results in a widely used mouse model of human inflammatory disease

[Source](https://www.biorxiv.org/content/10.1101/2021.02.09.430393v1)

---

## Low-dose exposure of glyphosate-based herbicides disrupt the urine metabolome and its interaction with gut microbiota

[Source](https://www.nature.com/articles/s41598-021-82552-2)

---

## Microbiota control of maternal behavior regulates early postnatal growth of offspring

[Source](https://advances.sciencemag.org/content/7/5/eabe6563)

---

## Lactic acid bacteria as probiotics for the nose?

[Source](https://sfamjournals.onlinelibrary.wiley.com/doi/epdf/10.1111/1751-7915.13759)

---

## <em>Lactobacillus</em> bile salt hydrolase substrate specificity governs bacterial fitness and host colonization

[Source](https://www.pnas.org/content/pnas/118/6/e2017709118.full.pdf)

---

## Probiotic nasal spray development by spray drying

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0939641120303416)

---

## Microbiota analysis of rural and urban surface waters and sediments in Bangladesh identifies human waste as driver of antibiotic resistance

[Source](https://www.biorxiv.org/content/10.1101/2021.02.04.429629v1.full.pdf)

---

## Depth-discrete metagenomics reveals the roles of microbes in biogeochemical cycling in the tropical freshwater Lake Tanganyika

[Source](https://www.nature.com/articles/s41396-021-00898-x)

---

## Trace gas oxidizers are widespread and active members of soil microbial communities

[Source](https://www.nature.com/articles/s41564-020-00811-w)

---

## Bacterial communities regulate temporal variations of the antibiotic resistome in soil following manure amendment.
 08.02.2021, _J Cheng, X Tang, C Liu_


The increasing emergence of antibiotic-resistant genes (ARGs) represents a global threat to human health. Land application of animal manure is known to contribute considerably to the propagation and dispersal of antibiotic resistance in agro-ecosystems. Yet, the primary determinants of the fate of the soil resistome remain obscure. In this study, a pot experiment was conducted to examine temporal changes in ARGs, mobile genetic elements (MGEs), and bacterial communities in a weakly developed loamy soil (an entisol known as calcareous purple soil) upon addition of pig or chicken manure. On the day of manure application, substantial increases in the diversity and relative abundance of ARGs were observed in soil amended with raw pig manure. At the same time, no obvious changes were observed for soil amended with chicken manure. Antibiotic resistance in pig manure-amended soils rapidly decreased over time to a level that was still higher than that of unamended soil at 100 days after manure application. The results of the Mantel test and Procrustes analysis indicated that ARG profiles in soil were significantly correlated with the structure of the bacterial phylogeny. Variation partitioning analysis further revealed that the bacterial community played a major role in regulating the temporal changes in ARGs in soil following manure application. Increased numbers and relative abundances of MGEs and their significant positive correlations with ARGs were observed, which suggest that a potential contribution from lateral gene transfer to the persistence and spread of ARGs should not be overlooked. Overall, our findings provide a better understanding of the mechanisms underlying the dynamics of ARGs in entisols following manure application and have practical implications for managing manure applications in entisols of the study area and other areas.

[Source](https://link.springer.com/article/10.1007%2Fs11356-021-12746-8)

10.1007/s11356-021-12746-8

---

## Quantifying Variation in Bacterial Reproductive Fitness: a High-Throughput Method.
 12.02.2021, _PM Frey, J Baer, J Bergada-Pijuan, C Lawless, PK Bühler, RD Kouyos, KP Lemon, AS Zinkernagel, SD Brugger_


To evaluate changes in reproductive fitness of bacteria, e.g., after acquisition of antimicrobial resistance, a low-cost high-throughput method to analyze bacterial growth on agar is desirable for broad usability. In our bacterial quantitative fitness analysis (BaQFA), arrayed cultures are spotted on agar and photographed sequentially while growing. These time-lapse images are analyzed using a purpose-built open-source software to derive normalized image intensity (NI) values for each culture spot. Subsequently, a Gompertz growth model is fitted to NI values, and fitness is calculated from model parameters. To represent a range of clinically important pathogenic bacteria, we used different strains of Enterococcus faecium, Escherichia coli, and Staphylococcus aureus, with and without antimicrobial resistance. Relative competitive fitness (RCF) was defined as the mean fitness ratio of two strains growing competitively on one plate.BaQFA permitted the accurate construction of growth curves from bacteria grown on semisolid agar plates and fitting of Gompertz models. Normalized image intensity values showed a strong association with the total CFU/ml count per spotted culture (P &lt; 0.001) for all strains of the three species. BaQFA showed relevant reproductive fitness differences between individual strains, suggesting substantially higher fitness of methicillin-resistant S. aureus JE2 than Cowan (RCF, 1.58; P &lt; 0.001). Similarly, the vancomycin-resistant E. faecium ST172b showed higher competitive fitness than susceptible E. faecium ST172 (RCF, 1.59; P &lt; 0.001). Our BaQFA method allows detection of fitness differences between bacterial strains and may help to estimate epidemiological antimicrobial persistence or contribute to the prediction of clinical outcomes in severe infections.IMPORTANCE Reproductive fitness of bacteria is a major factor in the evolution and persistence of antimicrobial resistance and may play an important role in severe infections. With a computational approach to quantify fitness in bacteria growing competitively on agar plates, our high-throughput method has been designed to obtain additional phenotypic data for antimicrobial resistance analysis at a low cost. Furthermore, our bacterial quantitative fitness analysis (BaQFA) enables the investigation of a link between bacterial fitness and clinical outcomes in severe invasive bacterial infections. This may allow future use of our method for patient management and risk stratification of clinical outcomes. Our proposed method uses open-source software and a hardware setup that can utilize consumer electronics. This will enable a wider community of researchers, including those from low-resource countries, where the burden of antimicrobial resistance is highest, to obtain valuable information about emerging bacterial strains.

[Source](https://msystems.asm.org/content/6/1/e01323-20)

10.1128/mSystems.01323-20

---

## Enabling multiscale variation analysis with genome graphs

[Source](https://www.biorxiv.org/content/10.1101/2021.02.03.429603v1.full.pdf)

---

## HMD-ARG: hierarchical multi-task deep learning for annotating antibiotic resistance genes.
 12.02.2021, _Y Li, Z Xu, W Han, H Cao, R Umarov, A Yan, M Fan, H Chen, CM Duarte, L Li, PL Ho, X Gao_


The spread of antibiotic resistance has become one of the most urgent threats to global health, which is estimated to cause 700,000 deaths each year globally. Its surrogates, antibiotic resistance genes (ARGs), are highly transmittable between food, water, animal, and human to mitigate the efficacy of antibiotics. Accurately identifying ARGs is thus an indispensable step to understanding the ecology, and transmission of ARGs between environmental and human-associated reservoirs. Unfortunately, the previous computational methods for identifying ARGs are mostly based on sequence alignment, which cannot identify novel ARGs, and their applications are limited by currently incomplete knowledge about ARGs.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01002-3)

10.1186/s40168-021-01002-3

---

## BAGET 2.0: an updated web tool for the effortless retrieval of prokaryotic gene context and sequence.
 03.02.2021, _B Hepp, V Da Cunha, F Lorieux, J Oberto_


The retrieval of a single gene sequence and context from completely sequenced bacterial and archaeal genomes constitutes an intimidating task for the wet bench biologist. Existing web-based genome browsers are either too complex for routine use or only provide a subset of the available prokaryotic genomes.

[Source](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab082/6126806)

10.1093/bioinformatics/btab082

---

## Metagenomics workflow for hybrid assembly, differential coverage binning, metatranscriptomics and pathway analysis (MUFFIN).
 09.02.2021, _R Van Damme, M Hölzer, A Viehweger, B Müller, E Bongcam-Rudloff, C Brandt_


Metagenomics has redefined many areas of microbiology. However, metagenome-assembled genomes (MAGs) are often fragmented, primarily when sequencing was performed with short reads. Recent long-read sequencing technologies promise to improve genome reconstruction. However, the integration of two different sequencing modalities makes downstream analyses complex. We, therefore, developed MUFFIN, a complete metagenomic workflow that uses short and long reads to produce high-quality bins and their annotations. The workflow is written by using Nextflow, a workflow orchestration software, to achieve high reproducibility and fast and straightforward use. This workflow also produces the taxonomic classification and KEGG pathways of the bins and can be further used for quantification and annotation by providing RNA-Seq data (optionally). We tested the workflow using twenty biogas reactor samples and assessed the capacity of MUFFIN to process and output relevant files needed to analyze the microbial community and their function. MUFFIN produces functional pathway predictions and, if provided de novo metatranscript annotations across the metagenomic sample and for each bin. MUFFIN is available on github under GNUv3 licence: https://github.com/RVanDamme/MUFFIN.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008716)

10.1371/journal.pcbi.1008716

---

## PATO: Pangenome Analysis Toolkit

[Source](https://www.biorxiv.org/content/10.1101/2021.01.30.428878v1.full.pdf)

---

## Struo2: efficient metagenome profiling database construction for ever-expanding microbial genome datasets 

[Source](https://www.biorxiv.org/content/10.1101/2021.02.10.430604v1.full.pdf)

---

## The Gut Microbiome, Health &amp; Disease. You truly are what you eat

[Source](https://docs.google.com/forms/d/e/1FAIpQLScfGagez5gkvGNj-Svb-HwrrgHsFvw3PRBDGVSZLGhz8csIDA/viewform)

---

## Human Gut Bacteria, Bile Acids, and Host Physiology

[Source](https://ntu-sg.zoom.us/meeting/register/tJ0rfumuqz4jG91AYyev1mw8Z34sLccAp3xS)

---

## MicroSeminars 2021

[Source](https://microseminar.wordpress.com/)

---

## Biofilm Baithak – An India-specific Meetup 

[Source](https://docs.google.com/forms/d/e/1FAIpQLSd665rqYDJ6nJ67wcgSue0OCcedTTWL3_CdIzjjbsHYUahqxg/viewform)

---

## <em>Klebsiella</em> Virtual Seminar Series 2021 #KlebClub2021

[Source](https://twitter.com/OlayaRendueles/status/1357375425324924928)

---

## Bioinformatics for Biologists: An Introduction to Linux, Bash Scripting, and R

[Source](https://www.futurelearn.com/courses/linux-for-bioinformatics)

---

## Next Generation Sequencing Bioinformatics course

[Source](https://redcap.h3abionet.org/redcap/surveys/?s=4K9PKXFLMC)

---

## Biobanking

[Source](https://www.fun-mooc.fr/courses/course-v1:pasteur+96019+session01/about)

---

## Building Biomes with Jack Gilbert

[Source](https://open.spotify.com/episode/6famPVMm31a40q0HHoGtCI)

---

## Reverse mentoring with Martin Dougherty

[Source](https://your-digital-mentor.simplecast.com/episodes/bonus-episode-reverse-mentoring)

---

## Post-docs Part 2: Hiring 

[Source](https://soundcloud.com/the-corresponding-author/episode-17-post-docs-part-2-hiring)

---

## Green algae and fatty acids 

[Source](https://podcasts.google.com/feed/aHR0cDovL3R3aW1zaG93Lm1pY3JvYmV3b3JsZC5saWJzeW5wcm8uY29tL3Jzcw/episode/MDgyN2U0N2ItYTA2Yy00Yjc0LWFmZmEtNTNhZjQ3OWNiMTMz)

---

## Dr. Heer Mehta: Antibiotic Resistance and Experimental Evolution

[Source](https://anchor.fm/bioinfopod/episodes/Dr--Heer-Mehta-Antibiotic-Resistance-and-Experimental-Evolution-ep8bqf)

---

## Microbe Crayon Project

[Source](https://anneamadden.wordpress.com/about/new-design/microbe-crayon-project/)

---

## Drawing the #BacterialAlphabet

[Source](https://twitter.com/NoemieMatthey/status/1357086103174590465)

---

## A portrait of budding yeasts: A symbol of the arts, sciences and a whole greater than the sum of its parts.
 10.02.2021, _JL Steenwyk_



[Source](https://twitter.com/hashtag/sciart?src=hash&ref_src=twsrc%5Etfw)

10.1002/yea.3518

---

## #sciart

[Source](https://twitter.com/hashtag/sciart?src=hash&ref_src=twsrc%5Etfw)

---

## #BacterialAlphabet

[Source](https://twitter.com/hashtag/BacterialAlphabet?src=hash&ref_src=twsrc%5Etfw)

---

## Efficient Transformation of Somatic Embryos and Regeneration of Cork Oak Plantlets with A Gene (CsTL1) Encoding a Chestnut Thaumatin-Like Protein.
 13.02.2021, _V Cano, MT Martínez, JL Couselo, E Varas, FJ Vieitez, E Corredoira_


We present a reproducible procedure for transforming somatic embryos of cork oak with the CsTL1 gene that codes for a thaumatin-like protein, in order to confer tolerance to Phytophthora cinnamomi. Different concentrations/combinations of the antibiotics carbenicillin and cefotaxime, as bacteriostatic agents, and kanamycin, as a selective agent, were tested. A lethal dose of 125 mg/L kanamycin was employed to select transgenic somatic embryos, and carbenicillin was used as a bacteriostatic agent at a concentration of 300 mg/L, which does not inhibit somatic embryo proliferation. The transformation efficiency was clearly genotype-dependent and was higher for the TGR3 genotype (17%) than for ALM80 (4.5%) and ALM6 (2%). Insertion of the transgenes in genomic DNA was confirmed by PCR analysis, whereas expression of the CsTL1 gene was evaluated by semi-quantitative real-time PCR (qPCR) analysis. A vitrification treatment successfully cryopreserved the transgenic lines generated. The antifungal activity of the thaumatin-like protein expressed by the gene CsTL1 was evaluated in an in vitro bioassay with the oomycete P. cinnamomi. Of the eight transgenic lines analyzed, seven survived for between one or two times longer than non-transgenic plantlets. Expression of the CsTL1 gene and plantlet survival days were correlated, and survival was generally greater in plantlets that strongly expressed the CsTL1 gene.

[Source](https://twitter.com/hashtag/Agrobacterium?src=hash&ref_src=twsrc%5Etfw)

10.3390/ijms22041757

---

## #Agrobacterium

[Source](https://twitter.com/hashtag/Agrobacterium?src=hash&ref_src=twsrc%5Etfw)

---

## @Mart1_100dra

[Source](https://twitter.com/Mart1_100dra?ref_src=twsrc%5Etfw)

---

## SciComm speaks.
 17.04.2018, __



[Source](https://twitter.com/hashtag/scicomm?src=hash&ref_src=twsrc%5Etfw)

10.1126/science.aat7933

---

## #scicomm

[Source](https://twitter.com/hashtag/scicomm?src=hash&ref_src=twsrc%5Etfw)

---

## pic.twitter.com/OwpJJmAJoM

[Source](https://t.co/OwpJJmAJoM)

---

## Empiric treatment for peritonsillar abscess: A single-center experience with medical therapy alone.
 13.02.2021, _AL Zebolsky, J Dewey, EJ Swayze, S Moffatt, CD Sullivan_


Compare the use of medical therapy alone (MTA) with surgical therapy (ST) for the empiric treatment of peritonsillar abscess (PTA).

[Source](https://twitter.com/NoemieMatthey/status/1357086103174590465?ref_src=twsrc%5Etfw)

10.1016/j.amjoto.2021.102954

---

## February 3, 2021

[Source](https://twitter.com/NoemieMatthey/status/1357086103174590465?ref_src=twsrc%5Etfw)

---

