## The oral microbiome and lung cancer risk: An analysis of 3 prospective cohort studies.
 05.08.2022, _E Vogtmann, X Hua, G Yu, V Purandare, AG Hullings, D Shao, Y Wan, S Li, CL Dagnall, K Jones, BD Hicks, A Hutchinson, JG Caporaso, W Wheeler, DP Sandler, LEB Freeman, LM Liao, WY Huang, ND Freedman, NE Caporaso, R Sinha, MH Gail, J Shi, CC Abnet_


Previous studies suggested associations between the oral microbiome and lung cancer, but studies were predominantly cross-sectional and underpowered.

[Source](https://doi.org/10.1093/jnci/djac149)

10.1093/jnci/djac149

---

## Antibiotic resistance genes in the gut microbiota of mothers and linked neonates with or without sepsis from low- and middle-income countries

[Source](https://doi.org/10.1038/s41564-022-01184-y)

---

## Crosstalk between mucosal microbiota, host gene expression, and sociomedical factors in the progression of colorectal cancer.
 04.08.2022, _N Kim, JA Gim, BJ Lee, BI Choi, HS Yoon, SH Kim, MK Joo, JJ Park, C Kim_


Various omics-based biomarkers related to the occurrence, progression, and prognosis of colorectal cancer (CRC) have been identified. In this study, we attempted to identify gut microbiome-based biomarkers and detect their association with host gene expression in the initiation and progression of CRC by integrating analysis of the gut mucosal metagenome, RNA sequencing, and sociomedical factors. We performed metagenome and RNA sequencing on colonic mucosa samples from 13 patients with advanced CRC (ACRC), 10 patients with high-risk adenoma (HRA), and 7 normal control (NC) individuals. All participants completed a questionnaire on sociomedical factors. The interaction and correlation between changes in the microbiome and gene expression were assessed using bioinformatic analysis. When comparing HRA and NC samples, which can be considered to represent the process of tumor initiation, 28 genes and five microbiome species were analyzed with correlation plots. When comparing ACRC and HRA samples, which can be considered to represent the progression of CRC, seven bacterial species and 21 genes were analyzed. When comparing ACRC and NC samples, 16 genes and five bacterial species were analyzed, and four correlation plots were generated. A network visualizing the relationship between bacterial and host gene expression in the initiation and progression of CRC indicated that Clostridium spiroforme and Tyzzerella nexilis were hub bacteria in the development and progression of CRC. Our study revealed the interactions of and correlation between the colonic mucosal microbiome and host gene expression to identify potential roles of the microbiome in the initiation and progression of CRC. Our results provide gut microbiome-based biomarkers that may be potential diagnostic markers and therapeutic targets in patients with CRC.

[Source](https://doi.org/10.1038/s41598-022-17823-7)

10.1038/s41598-022-17823-7

---

## Using Culture-Enriched Phenotypic Metagenomics for Targeted High-Throughput Monitoring of the Clinically Important Fraction of the β-Lactam Resistome.
 05.08.2022, _Z Zhang, G Zhang, F Ju_


High bacterial community diversity and complexity greatly challenge the cost-efficient monitoring of clinically prevalent antibiotic-resistant bacteria, which are usually present as rare and important populations involved in the environmental dissemination of clinical resistance. Here, we introduce culture-enriched phenotypic metagenomics that integrates culture enrichment, phenotypic screening, and metagenomic analyses as an emerging standardized methodology for targeted resistome monitoring and apply it to decipher the extended-spectrum β-lactam resistome in a municipal wastewater treatment plant (WWTP) and its receiving river. The results showed that clinically prevalent carbapenemase genes (e.g., the NDM and KPC families) and extended-spectrum β-lactamase genes (e.g., the CTX-M, TEM, and OXA families) were prevalent in the WWTP and showed prominent potential in horizontal dissemination. Strikingly, carbapenem and polymyxin resistance genes co-occurred in the highly virulent nosocomial pathogens Enterobacter kobei and Citrobacter freundii. Overall, this study exemplifies phenotypic metagenomics for high-throughput surveillance of a targeted clinically important fraction of antibiotic resistomes and substantially expands current knowledge on extended-spectrum β-lactam resistance in WWTPs.

[Source](https://doi.org/10.1021/acs.est.2c03627)

10.1021/acs.est.2c03627

---

## <em>Porphyromonas gingivalis</em> and dental stem cells crosstalk amplify inflammation and bone loss in the periodontitis niche

[Source](https://doi.org/10.1002/jcp.30848)

---

## On the effect of flavonoids and dietary fibre in lingonberries on atherosclerotic plaques, lipid profiles and gut microbiota composition in <em>Apoe(-/-)</em> mice

[Source](https://doi.org/10.1080/09637486.2022.2106358)

---

## The Gut-Brain-Microbiome Connection: Can Probiotics Decrease Anxiety and Depression?
 05.08.2022, _J Maybee, T Pearson, L Elliott_


Anxiety and depression are highly prevalent mood disorders worldwide. Complete remission of symptoms is often difficult to achieve, despite following recommended treatment guidelines. Numerous antidepressants and anxiolytics exist, and new drugs are being developed constantly, yet the incidence of common mood disorders continues to rise. Despite the prevalence of these issues, mental health treatment has not evolved much in recent years. An exciting area of research uncovered in the past decade is the gut-brain-microbiome axis, a bi-directional communication pathway. Because the human microbiome is closely related to mood, research is being done to investigate whether probiotic supplementation could potentially affect symptoms of anxiety and depression.

[Source](https://doi.org/10.1080/01612840.2022.2106525)

10.1080/01612840.2022.2106525

---

## The effect of <em>Lactobacillus rhamnosus</em> B10 on alcoholic liver injury and intestinal microbiota in alcohol-induced mice model

[Source](https://doi.org/10.1111/jfbc.14372)

---

## The effect of young blood plasma administration on gut microbiota in middle-aged rats

[Source](https://doi.org/10.1007/s00203-022-03154-8)

---

## Quantitative microbial population study reveals geographical differences in bacterial symbionts of <em>Ixodes ricinus</em>

[Source](https://doi.org/10.1186/s40168-022-01276-1)

---

## Host taxonomy determines the composition, structure, and diversity of the earthworm cast microbiome under homogenous feeding conditions.
 04.08.2022, _M Aira, M Pérez-Losada, KA Crandall, J Domínguez_


Host evolutionary history is a key factor shaping the earthworm cast microbiome, although its effect can be shadowed by the earthworm's diet. To untangle dietary from taxon effects, we raised nine earthworm species on a uniform diet of cow manure and compared cast microbiome across species while controlling for diet. Our results showed that, under controlled laboratory conditions, earthworm microbiomes are species-specific, more diverse than that of the controlled diet, and mainly comprised of native bacteria (i.e., not acquired from the diet). Furthermore, diet has a medium to large convergence effect on microbiome composition since earthworms shared 16 to 74% of their bacterial amplicon sequence variants (ASV). The inter-species core microbiome included 10 ASVs, while their intra-species core microbiomes were larger and varied in ASV richness (24-48%) and sequence abundance across earthworm species. This specificity in core microbiomes and variable degree of similarity in bacterial composition suggest that phylosymbiosis could determine earthworm microbiome assembly. However, lack of congruence between the earthworm phylogeny and the microbiome dendrogram suggests that a consistent diet fed over several generations may have weakened potential phylosymbiotic effects. Thus, cast microbiome assembly in earthworms seem to be the result of an interplay among host phylogeny and diet.

[Source](https://doi.org/10.1093/femsec/fiac093)

10.1093/femsec/fiac093

---

## Ruminal Microbiota Determines the High-Fiber Utilization of Ruminants: Evidence from the Ruminal Microbiota Transplant.
 04.08.2022, _X Chen, F Yan, T Liu, Y Zhang, X Li, M Wang, C Zhang, X Xu, L Deng, J Yao, S Wu_


The rumen, which contains a series of prokaryotes and eukaryotes with high abundance, determines the high ability to degrade complex carbohydrates in ruminants. Using 16S rRNA gene sequencing, we compared the ruminal microbiota of dairy goats with that in the foregut and colon of mice and found more Bacteroides identified in the rumen, which helps ruminants to utilize plant-derived polysaccharides, cellulose, and other structural carbohydrates. Furthermore, high-fiber diets did not significantly increase intestinal fiber-degrading bacteria in mice, but did produce higher levels of ruminal fiber-degrading bacteria in dairy goats. Through rumen microbe transplantation (RMT), we found that rumen-derived fiber-degrading bacteria can colonize the intestines of mice to exert their fiber-degrading function, but their colonization efficiency is affected by diet. Additionally, the colonization of these fiber-degrading bacteria in the colon may involve higher content of butyrate in the colon, protecting the colonic epithelial barrier and promoting energy metabolism. Overall, the fiber degradation function of rumen bacteria through RMT was verified, and our results provide new insights into isolating the functional and beneficial fiber-degrading bacteria in the rumen, providing a theoretical basis for the role of dietary fiber in intestinal health. IMPORTANCE Ruminants have a powerful progastric digestive system that converts structural carbohydrates into nutrients useful to humans. It is well known that this phenomenon is due to the fact that the rumen of ruminants is a natural microbial fermenter, which can ferment structural carbohydrates such as cellulose and hemicellulose and transform them into volatile fatty acids to supply energy for host. However, monogastric animals have an inherent disadvantage in utilizing fiber, so screening rumen-derived fiber-degrading bacteria as a fermentation strain for biological feed is needed in an attempt at improving the fiber digestibility of monogastric animals. In this study, a ruminal microbiota transplant experiment from goats to mice proves that ruminal microbiota could serve as a key factor in utilization of high-fiber diets and provides a new perspective for the development of probiotics with fiber degradation function from the rumen and the importance of the use of prebiotics during the intake of probiotics.

[Source](https://doi.org/10.1128/spectrum.00446-22)

10.1128/spectrum.00446-22

---

## Computational approach to modeling microbiome landscapes associated with chronic human disease progression.
 04.08.2022, _L Li, J Sohn, RJ Genco, J Wactawski-Wende, S Goodison, PI Diaz, Y Sun_


A microbial community is a dynamic system undergoing constant change in response to internal and external stimuli. These changes can have significant implications for human health. However, due to the difficulty in obtaining longitudinal samples, the study of the dynamic relationship between the microbiome and human health remains a challenge. Here, we introduce a novel computational strategy that uses massive cross-sectional sample data to model microbiome landscapes associated with chronic disease development. The strategy is based on the rationale that each static sample provides a snapshot of the disease process, and if the number of samples is sufficiently large, the footprints of individual samples populate progression trajectories, which enables us to recover disease progression paths along a microbiome landscape by using computational approaches. To demonstrate the validity of the proposed strategy, we developed a bioinformatics pipeline and applied it to a gut microbiome dataset available from a Crohn's disease study. Our analysis resulted in one of the first working models of microbial progression for Crohn's disease. We performed a series of interrogations to validate the constructed model. Our analysis suggested that the model recapitulated the longitudinal progression of microbial dysbiosis during the known clinical trajectory of Crohn's disease. By overcoming restrictions associated with complex longitudinal sampling, the proposed strategy can provide valuable insights into the role of the microbiome in the pathogenesis of chronic disease and facilitate the shift of the field from descriptive research to mechanistic studies.

[Source](https://doi.org/10.1371/journal.pcbi.1010373)

10.1371/journal.pcbi.1010373

---

## Guideline for the analysis of the microbial communities of the human upper airways.
 05.08.2022, _L Mancabelli, T Ciociola, GA Lugli, C Tarracchini, F Fontanta, A Viappiani, F Turroni, A Ticinesi, T Meschi, S Conti, M Ventura, C Milani_


The recent COVID-19 pandemic prompted a rapid-growing interest in the investigation of the human microbiota of the upper airways. In fact, the resident microbial community of this body district may have an influence on the onset of SARS-CoV-2 infection and its clinical course in terms of presence, symptom severity, and outcomes. However, several microbiological methodologies are available to study the human microbiota, reflecting the extensive fragmentation of methodological approaches. We investigate the impact of two critical steps that can induce biases in the downstream analyses, i.e. sampling method and microbial DNA extraction kit employed. We observed major discrepancies regarding the total amount of prokaryotic DNA that could be retrieved from a biological sample and the proportion between bacterial DNA and human host DNA. Moreover, shotgun DNA sequencing and taxonomic profile reconstruction also revealed correlations between sampling methods and the procedures applied for microbial DNA extraction. Based on all the data collected in this study, we formulate indications regarding the most efficient and reliable methodological procedures for the metagenomic analyses of the upper airways' microbiota to maximize accuracy and reproducibility.

[Source](https://doi.org/10.1080/20002297.2022.2103282)

10.1080/20002297.2022.2103282

---

