## SARS-CoV-2 viral load is associated with increased disease severity and mortality

[Source](https://www.nature.com/articles/s41467-020-19057-5)

---

## Multi-omics analyses of radiation survivors identify radioprotective microbes and metabolites.
 03.11.2020, _H Guo, WC Chou, Y Lai, K Liang, JW Tam, WJ Brickey, L Chen, ND Montgomery, X Li, LM Bohannon, AD Sung, NJ Chao, JU Peled, ALC Gomes, MRM van den Brink, MJ French, AN Macintyre, GD Sempowski, X Tan, RB Sartor, K Lu, JPY Ting_


Ionizing radiation causes acute radiation syndrome, which leads to hematopoietic, gastrointestinal, and cerebrovascular injuries. We investigated a population of mice that recovered from high-dose radiation to live normal life spans. These "elite-survivors" harbored distinct gut microbiota that developed after radiation and protected against radiation-induced damage and death in both germ-free and conventionally housed recipients. Elevated abundances of members of the bacterial taxa Lachnospiraceae and Enterococcaceae were associated with postradiation restoration of hematopoiesis and gastrointestinal repair. These bacteria were also found to be more abundant in leukemia patients undergoing radiotherapy, who also displayed milder gastrointestinal dysfunction. In our study in mice, metabolomics revealed increased fecal concentrations of microbially derived propionate and tryptophan metabolites in elite-survivors. The administration of these metabolites caused long-term radioprotection, mitigation of hematopoietic and gastrointestinal syndromes, and a reduction in proinflammatory responses.

[Source](https://science.sciencemag.org/content/370/6516/eaay9097)

10.1126/science.aay9097

---

## Dietary simple sugars alter microbial ecology in the gut and promote colitis in mice.
 29.10.2020, _S Khan, S Waliullah, V Godfrey, MAW Khan, RA Ramachandran, BL Cantarel, C Behrendt, L Peng, LV Hooper, H Zaki_


The higher prevalence of inflammatory bowel disease (IBD) in Western countries points to Western diet as a possible IBD risk factor. High sugar, which is linked to many noncommunicable diseases, is a hallmark of the Western diet, but its role in IBD remains unknown. Here, we studied the effects of simple sugars such as glucose and fructose on colitis pathogenesis in wild-type and Il10-/- mice. Wild-type mice fed 10% glucose in drinking water or high-glucose diet developed severe colitis induced by dextran sulfate sodium. High-glucose-fed Il10-/- mice also developed a worsened colitis compared to glucose-untreated Il10-/- mice. Short-term intake of high glucose or fructose did not trigger inflammatory responses in healthy gut but markedly altered gut microbiota composition. In particular, the abundance of the mucus-degrading bacteria Akkermansia muciniphila and Bacteroides fragilis was increased. Consistently, bacteria-derived mucolytic enzymes were enriched leading to erosion of the colonic mucus layer of sugar-fed wild-type and Il10-/- mice. Sugar-induced exacerbation of colitis was not observed when mice were treated with antibiotics or maintained in a germ-free environment, suggesting that altered microbiota played a critical role in sugar-induced colitis pathogenesis. Furthermore, germ-free mice colonized with microbiota from sugar-treated mice showed increased colitis susceptibility. Together, these data suggest that intake of simple sugars predisposes to colitis and enhances its pathogenesis via modulation of gut microbiota in mice.

[Source](https://stm.sciencemag.org/content/12/567/eaay6218)

10.1126/scitranslmed.aay6218

---

## Deep-Learning Resources for Studying Glycan-Mediated Host-Microbe Interactions.
 29.10.2020, _D Bojar, RK Powers, DM Camacho, JJ Collins_


Glycans, the most diverse biopolymer, are shaped by evolutionary pressures stemming from host-microbe interactions. Here, we present machine learning and bioinformatics methods to leverage the evolutionary information present in glycans to gain insights into how pathogens and commensals interact with hosts. By using techniques from natural language processing, we develop deep-learning models for glycans that are trained on a curated dataset of 19,299 unique glycans and can be used to study and predict glycan functions. We show that these models can be utilized to predict glycan immunogenicity and the pathogenicity of bacterial strains, as well as investigate glycan-mediated immune evasion via molecular mimicry. We also develop glycan-alignment methods and use these to analyze virulence-determining glycan motifs in the capsular polysaccharides of bacterial pathogens. These resources enable one to identify and study glycan motifs involved in immunogenicity, pathogenicity, molecular mimicry, and immune evasion, expanding our understanding of host-microbe interactions.

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30562-X)

10.1016/j.chom.2020.10.004

---

## Mucin-derived O-glycans supplemented to diet mitigate diverse microbiota perturbations.
 22.10.2020, _KM Pruss, A Marcobal, AM Southwick, D Dahan, SA Smits, JA Ferreyra, SK Higginbottom, ED Sonnenburg, PC Kashyap, B Choudhury, L Bode, JL Sonnenburg_


Microbiota-accessible carbohydrates (MACs) are powerful modulators of microbiota composition and function. These substrates are often derived from diet, such as complex polysaccharides from plants or human milk oligosaccharides (HMOs) during breastfeeding. Host-derived mucus glycans on gut-secreted mucin proteins serve as a continuous endogenous source of MACs for resident microbes; here we investigate the potential role of purified, orally administered mucus glycans in maintaining a healthy microbial community. In this study, we liberated and purified O-linked glycans from porcine gastric mucin and assessed their efficacy in shaping the recovery of a perturbed microbiota in a mouse model. We found that porcine mucin glycans (PMGs) and HMOs enrich for taxonomically similar resident microbes. We demonstrate that PMGs aid recovery of the microbiota after antibiotic treatment, suppress Clostridium difficile abundance, delay the onset of diet-induced obesity, and increase the relative abundance of resident Akkermansia muciniphila. In silico analysis revealed that genes associated with mucus utilization are abundant and diverse in prevalent gut commensals and rare in enteric pathogens, consistent with these glycan-degrading capabilities being selected for during host development and throughout the evolution of the host-microbe relationship. Importantly, we identify mucus glycans as a novel class of prebiotic compounds that can be used to mitigate perturbations to the microbiota and provide benefits to host physiology.

[Source](https://www.nature.com/articles/s41396-020-00798-6)

10.1038/s41396-020-00798-6

---

## Microbiota-Derived Short-Chain Fatty Acids Promote LAMTOR2-Mediated Immune Responses in Macrophages.
 04.11.2020, _T Wu, H Li, C Su, F Xu, G Yang, K Sun, M Xu, N Lv, B Meng, Y Liu, L Hu, Y Liu, Y Gao, H Wang, Y Lan, D Xu, J Li_


ABSRTACTKlebsiella pneumoniae is a common cause of human-pneumonia-derived sepsis with high morbidity and mortality. The microbiota promotes and maintains host immune homeostasis. The mechanisms by which the gut microbiota affects the host defenses in the respiratory system systematically, however, remain poorly understood. Here, we show that gut microbiota depletion increases susceptibility to extracellular K. pneumoniae infections in terms of increased bacterial burdens in lung and decreased survival rates. Oral supplementation with gut microbiota-derived short-chain fatty acids (SCFAs), subsequently activating G protein-coupled receptor 43 (GPCR43), enhances a macrophage's capacity to phagocytose invading K. pneumoniae Furthermore, SCFAs and GPR43 increase macrophage bacterial clearance by upregulating LAMTOR2, which is further identified as an antibacterial effector and elucidated to facilitate phagosome-lysosome fusion and extracellular signal-regulated kinase (ERK) phosphorylation. Lastly, conditional ablation of Lamtor2 in macrophages decreases their antimicrobial activity, even though mice were pretreated with exogenous SCFA supplementation.IMPORTANCE These observations highlight that SCFAs promote macrophage elimination of K. pneumoniae via a LAMTOR2-dependent signal pathway and suggest that it is possible to intervene in K. pneumoniae pneumonia by targeting the gut microbiota.

[Source](https://msystems.asm.org/content/5/6/e00587-20)

10.1128/mSystems.00587-20

---

## A probiotic yeast-based immunotherapy against Clostridioides difficile infection.
 29.10.2020, _K Chen, Y Zhu, Y Zhang, T Hamza, H Yu, A Saint Fleur, J Galen, Z Yang, H Feng_


Antibiotic-resistant Clostridioides difficile is an anaerobic Gram-positive bacterium that colonizes the colon and is responsible for more than 29,000 deaths in the United States each year. Hence, C. difficile infection (CDI) poses an urgent threat to public health. Antibody-mediated neutralization of TcdA and TcdB toxins, the major virulence factors of CDI, represents an effective strategy to combat the disease without invoking antibiotic resistance. However, current antitoxin approaches are mostly based on parenteral infusion of monoclonal antibodies that are costly, narrow spectrum, and not optimized against the intestinal disease. Here, we engineered probiotic Saccharomyces boulardii to constitutively secrete a single tetra-specific antibody that potently and broadly neutralized both toxins and demonstrated protection against primary and recurrent CDI in both prophylactic and therapeutic mouse models of disease. This yeast immunotherapy is orally administered, can be used concurrently with antibiotics, and may have potential as a prophylactic against CDI risk and as a therapeutic for patients with CDI.

[Source](https://stm.sciencemag.org/content/12/567/eaax4905)

10.1126/scitranslmed.aax4905

---

## Host specificity of microbiome assembly and its fitness effects in phytoplankton.
 24.10.2020, _SL Jackrel, JW Yang, KC Schmidt, VJ Denef_


Insights into symbiosis between eukaryotic hosts and their microbiomes have shifted paradigms on what determines host fitness, ecology, and behavior. Questions remain regarding the roles of host versus environment in shaping microbiomes, and how microbiome composition affects host fitness. Using a model system in ecology, phytoplankton, we tested whether microbiomes are host-specific, confer fitness benefits that are host-specific, and remain conserved in time in their composition and fitness effects. We used an experimental approach in which hosts were cleaned of bacteria and then exposed to bacterial communities from natural environments to permit recruitment of microbiomes. We found that phytoplankton microbiomes consisted of a subset of taxa recruited from these natural environments. Microbiome recruitment was host-specific, with host species explaining more variation in microbiome composition than environment. While microbiome composition shifted and then stabilized over time, host specificity remained for dozens of generations. Microbiomes increased host fitness, but these fitness effects were host-specific for only two of the five species. The shifts in microbiome composition over time amplified fitness benefits to the hosts. Overall, this work solidifies the importance of host factors in shaping microbiomes and elucidates the temporal dynamics of microbiome compositional and fitness effects.

[Source](https://www.nature.com/articles/s41396-020-00812-x)

10.1038/s41396-020-00812-x

---

## Microbial ecology of the newly discovered serpentinite-hosted Old City hydrothermal field (southwest Indian ridge).
 03.11.2020, _A Lecoeuvre, B Ménez, M Cannat, V Chavagnac, E Gérard_


Lost City (mid-Atlantic ridge) is a unique oceanic hydrothermal field where carbonate-brucite chimneys are colonized by a single phylotype of archaeal Methanosarcinales, as well as sulfur- and methane-metabolizing bacteria. So far, only one submarine analog of Lost City has been characterized, the Prony Bay hydrothermal field (New Caledonia), which nonetheless shows more microbiological similarities with ecosystems associated with continental ophiolites. This study presents the microbial ecology of the 'Lost City'-type Old City hydrothermal field, recently discovered along the southwest Indian ridge. Five carbonate-brucite chimneys were sampled and subjected to mineralogical and geochemical analyses, microimaging, as well as 16S rRNA-encoding gene and metagenomic sequencing. Dominant taxa and metabolisms vary between chimneys, in conjunction with the predicted redox state, while potential formate- and CO-metabolizing microorganisms as well as sulfur-metabolizing bacteria are always abundant. We hypothesize that the variable environmental conditions resulting from the slow and diffuse hydrothermal fluid discharge that currently characterizes Old City could lead to different microbial populations between chimneys that utilize CO and formate differently as carbon or electron sources. Old City discovery and this first description of its microbial ecology opens up attractive perspectives for understanding environmental factors shaping communities and metabolisms in oceanic serpentinite-hosted ecosystems.

[Source](https://www.nature.com/articles/s41396-020-00816-7)

10.1038/s41396-020-00816-7

---

## Roles of bacteriophages, plasmids and CRISPR immunity in microbial community dynamics revealed using time-series integrated meta-omics.
 03.11.2020, _S Martínez Arbas, S Narayanasamy, M Herold, LA Lebrun, MR Hoopmann, S Li, TJ Lam, BJ Kunath, ND Hicks, CM Liu, LB Price, CC Laczny, JD Gillece, JM Schupp, PS Keim, RL Moritz, K Faust, H Tang, Y Ye, A Skupin, P May, EEL Muller, P Wilmes_


Viruses and plasmids (invasive mobile genetic elements (iMGEs)) have important roles in shaping microbial communities, but their dynamic interactions with CRISPR-based immunity remain unresolved. We analysed generation-resolved iMGE-host dynamics spanning one and a half years in a microbial consortium from a biological wastewater treatment plant using integrated meta-omics. We identified 31 bacterial metagenome-assembled genomes encoding complete CRISPR-Cas systems and their corresponding iMGEs. CRISPR-targeted plasmids outnumbered their bacteriophage counterparts by at least fivefold, highlighting the importance of CRISPR-mediated defence against plasmids. Linear modelling of our time-series data revealed that the variation in plasmid abundance over time explained more of the observed community dynamics than phages. Community-scale CRISPR-based plasmid-host and phage-host interaction networks revealed an increase in CRISPR-mediated interactions coinciding with a decrease in the dominant 'Candidatus Microthrix parvicella' population. Protospacers were enriched in sequences targeting genes involved in the transmission of iMGEs. Understanding the factors shaping the fitness of specific populations is necessary to devise control strategies for undesirable species and to predict or explain community-wide phenotypes.

[Source](https://www.nature.com/articles/s41564-020-00794-8)

10.1038/s41564-020-00794-8

---

## Detecting Gene Ontology misannotations using taxon-specific rate ratio comparisons.
 22.10.2020, _X Wei, C Zhang, PL Freddolino, Y Zhang_


Many protein function databases are built on automated or semi-automated curations and can contain various annotation errors. The correction of such misannotations is critical to improving the accuracy and reliability of the databases.

[Source](https://academic.oup.com/bioinformatics/article-abstract/36/16/4383/5848643)

10.1093/bioinformatics/btaa548

---

## phyloFlash: Rapid Small-Subunit rRNA Profiling and Targeted Assembly from Metagenomes.
 28.10.2020, _HR Gruber-Vodicka, BKB Seah, E Pruesse_


The small-subunit rRNA (SSU rRNA) gene is the key marker in molecular ecology for all domains of life, but it is largely absent from metagenome-assembled genomes that often are the only resource available for environmental microbes. Here, we present phyloFlash, a pipeline to overcome this gap with rapid, SSU rRNA-centered taxonomic classification, targeted assembly, and graph-based binning of full metagenomic assemblies. We show that a cleanup of artifacts is pivotal even with a curated reference database. With such a filtered database, the general-purpose mapper BBmap extracts SSU rRNA reads five times faster than the rRNA-specialized tool SortMeRNA with similar sensitivity and higher selectivity on simulated metagenomes. Reference-based targeted assemblers yielded either highly fragmented assemblies or high levels of chimerism, so we employ the general-purpose genomic assembler SPAdes. Our optimized implementation is independent of reference database composition and has satisfactory levels of chimera formation. phyloFlash quickly processes Illumina (meta)genomic data, is straightforward to use, even as part of high-throughput quality control, and has user-friendly output reports. The software is available at https://github.com/HRGV/phyloFlash (GPL3 license) and is documented with an online manual.IMPORTANCE To track organisms across all domains of life, the SSU rRNA gene is the gold standard. Many environmental microbes are known only from high-throughput sequence data, but the SSU rRNA gene, the key to visualization by molecular probes and link to existing literature, is often missing from metagenome-assembled genomes (MAGs). The easy-to-use phyloFlash software suite tackles this gap with rapid, SSU rRNA-centered taxonomic classification, targeted assembly, and graph-based linking to MAGs. Starting from a cleaned reference database, phyloFlash profiles the taxonomic diversity and assembles the sorted SSU rRNA reads. The phyloFlash design is domain agnostic and covers eukaryotes, archaea, and bacteria alike. phyloFlash also provides utilities to visualize multisample comparisons and to integrate the recovered SSU rRNAs in a metagenomics workflow by linking them to MAGs using assembly graph parsing.

[Source](https://msystems.asm.org/content/5/5/e00920-20)

10.1128/mSystems.00920-20

---

## Large-Scale Metagenome Assembly Reveals Novel Animal-Associated Microbial Genomes, Biosynthetic Gene Clusters, and Other Genetic Diversity.
 04.11.2020, _ND Youngblut, J de la Cuesta-Zuluaga, GH Reischer, S Dauser, N Schuster, C Walzer, G Stalder, AH Farnleitner, RE Ley_


Large-scale metagenome assemblies of human microbiomes have produced a vast catalogue of previously unseen microbial genomes; however, comparatively few microbial genomes derive from other vertebrates. Here, we generated 5,596 metagenome-assembled genomes (MAGs) from the gut metagenomes of 180 predominantly wild animal species representing 5 classes, in addition to 14 existing animal gut metagenome data sets. The MAGs comprised 1,522 species-level genome bins (SGBs), most of which were novel at the species, genus, or family level, and the majority were enriched in host versus environment metagenomes. Many traits distinguished SGBs enriched in host or environmental biomes, including the number of antimicrobial resistance genes. We identified 1,986 diverse biosynthetic gene clusters; only 23 clustered with any MIBiG database references. Gene-based assembly revealed tremendous gene diversity, much of it host or environment specific. Our MAG and gene data sets greatly expand the microbial genome repertoire and provide a broad view of microbial adaptations to the vertebrate gut.IMPORTANCE Microbiome studies on a select few mammalian species (e.g., humans, mice, and cattle) have revealed a great deal of novel genomic diversity in the gut microbiome. However, little is known of the microbial diversity in the gut of other vertebrates. We studied the gut microbiomes of a large set of mostly wild animal species consisting of mammals, birds, reptiles, amphibians, and fish. Unfortunately, we found that existing reference databases commonly used for metagenomic analyses failed to capture the microbiome diversity among vertebrates. To increase database representation, we applied advanced metagenome assembly methods to our animal gut data and to many public gut metagenome data sets that had not been used to obtain microbial genomes. Our resulting genome and gene cluster collections comprised a great deal of novel taxonomic and genomic diversity, which we extensively characterized. Our findings substantially expand what is known of microbial genomic diversity in the vertebrate gut.

[Source](https://msystems.asm.org/content/5/6/e01045-20)

10.1128/mSystems.01045-20

---

## Origins and genetic legacy of prehistoric dogs.
 03.11.2020, _A Bergström, L Frantz, R Schmidt, E Ersmark, O Lebrasseur, L Girdland-Flink, AT Lin, J Storå, KG Sjögren, D Anthony, E Antipina, S Amiri, G Bar-Oz, VI Bazaliiskii, J Bulatović, D Brown, A Carmagnini, T Davy, S Fedorov, I Fiore, D Fulton, M Germonpré, J Haile, EK Irving-Pease, A Jamieson, L Janssens, I Kirillova, LK Horwitz, J Kuzmanovic-Cvetković, Y Kuzmin, RJ Losey, DL Dizdar, M Mashkour, M Novak, V Onar, D Orton, M Pasarić, M Radivojević, D Rajković, B Roberts, H Ryan, M Sablin, F Shidlovskiy, I Stojanović, A Tagliacozzo, K Trantalidou, I Ullén, A Villaluenga, P Wapnish, K Dobney, A Götherström, A Linderholm, L Dalén, R Pinhasi, G Larson, P Skoglund_


Dogs were the first domestic animal, but little is known about their population history and to what extent it was linked to humans. We sequenced 27 ancient dog genomes and found that all dogs share a common ancestry distinct from present-day wolves, with limited gene flow from wolves since domestication but substantial dog-to-wolf gene flow. By 11,000 years ago, at least five major ancestry lineages had diversified, demonstrating a deep genetic history of dogs during the Paleolithic. Coanalysis with human genomes reveals aspects of dog population history that mirror humans, including Levant-related ancestry in Africa and early agricultural Europe. Other aspects differ, including the impacts of steppe pastoralist expansions in West and East Eurasia and a near-complete turnover of Neolithic European dog ancestry.

[Source](https://science.sciencemag.org/content/370/6516/557)

10.1126/science.aba9572

---

