## Safety, Feasibility, and Advantages of Oral Microbiota Transplantation: The First Clinical Case.
 14.06.2024, _OV Goloshchapov, AB Chukhlovin, DS Bug, DE Polev, OV Kosarev, RV Klementeva, EA Izmailova, IV Kazantsev, MS Khalipskaia, МО Goloshchapova, OS Yudintseva, IM Barkhatov, NV Petukhova, LS Zubarovskaya, AD Kulagin, IS Moiseev_


The pilot clinical study presented demonstrates the possibility, safety, and effectiveness of oral microbiota transplantation from a healthy donor to a patient with neuroblastoma to prevent chemotherapy-induced oral mucositis. A 6-month-old patient with a diagnosis of retroperitoneal neuroblastoma was treated according to the NB 2004 protocol. Due to the development of severe oral mucositis, it was decided to perform oral microbiota transplantation. During the next 3 chemotherapy cycles and conditioning regimen before autologous hematopoietic cell transplantation (auto-HCT), the patient was repeatedly injected per os with donor saliva from her healthy mother. Oral microbiota transplantation was shown to effectively prevent the development of oral mucositis after chemotherapy, and only grade 1 oral mucositis developed after auto-HCT. In all loci of the oral cavity, there was a decreased abundance of bacteria from the Staphylococcaceae, Micrococcaceae, and Xanthomonadaceae families. Conversely, there was an increase in the relative abundance of Streptococcaceae and certain other bacterial taxa. In conclusion, the transplantation of maternal saliva in this patient prevented severe mucositis and was accompanied by a compositional change of the patient's oral microbiota. No adverse events due to the transplantation of maternal saliva were noted.

[Source](https://journals.lww.com/jpho-online/fulltext/9900/safety,_feasibility,_and_advantages_of_oral.438.aspx)

10.1097/MPH.0000000000002896

---

## Oral microbial dysbiosis in patients with oral cavity cancers.
 20.06.2024, _O Unlu, M Demirci, T Paksoy, AB Eden, HD Tansuker, A Dalmizrak, C Aktan, F Senel, AV Sunter, O Yigit, BO Cakir, A Kantarci_


_Humans, Female, Male, Middle Aged, Dysbiosis, Mouth Neoplasms, Saliva, Case-Control Studies, Surveys and Questionnaires, Aged, Microbiota, Adult, RNA, Ribosomal, 16S, Oral Health_
The pathogenesis of oral cavity cancers is complex. We tested the hypothesis that oral microbiota dysbiosis is associated with oral cavity cancer.

[Source](https://link.springer.com/article/10.1007/s00784-024-05770-8)

10.1007/s00784-024-05770-8

---

## Nasal cathelicidin is expressed in early life and is increased during mild, but not severe respiratory syncytial virus infection

[Source](https://www.nature.com/articles/s41598-024-64446-1)

---

## Tissue-resident bacteria in metabolic diseases: emerging evidence and challenges.
 19.06.2024, _L Massier, N Musat, M Stumvoll, V Tremaroli, R Chakaroun, P Kovacs_


Although the impact of the gut microbiome on health and disease is well established, there is controversy regarding the presence of microorganisms such as bacteria and their products in organs and tissues. However, recent contamination-aware findings of tissue-resident microbial signatures provide accumulating evidence in support of bacterial translocation in cardiometabolic disease. The latter provides a distinct paradigm for the link between microbial colonizers of mucosal surfaces and host metabolism. In this Perspective, we re-evaluate the concept of tissue-resident bacteria including their role in metabolic low-grade tissue and systemic inflammation. We examine the limitations and challenges associated with studying low bacterial biomass samples and propose experimental and analytical strategies to overcome these issues. Our Perspective aims to encourage further investigation of the mechanisms linking tissue-resident bacteria to host metabolism and their potentially actionable health implications for prevention and treatment.

[Source](https://www.nature.com/articles/s42255-024-01065-0)

10.1038/s42255-024-01065-0

---

## Faecal microbiota of schoolchildren is associated with nutritional status and markers of inflammation: a double-blinded cluster-randomized controlled trial using multi-micronutrient fortified rice

[Source](https://www.nature.com/articles/s41467-024-49093-4)

---

## Short-chain fatty acid valerate reduces voluntary alcohol intake in male mice.
 21.06.2024, _SC Bokoliya, J Russell, Y Dorsett, HA Panier, V Singh, L Daddi, H Yuan, LR Dedon, Z Liu, Y Zhou, Z Min, JR Barson, J Covault, JA Bubier, Y Zhou_


_Animals, Male, Mice, Gastrointestinal Microbiome, gamma-Aminobutyric Acid, Fatty Acids, Volatile, Alcohol Drinking, Amygdala, Ethanol, Mice, Inbred C57BL, Disease Models, Animal, Binge Drinking, Pentanoic Acids_
Despite serious health and social consequences, effective intervention strategies for habitual alcohol binge drinking are lacking. The development of novel therapeutic and preventative approaches is highly desirable. Accumulating evidence in the past several years has established associations between the gut microbiome and microbial metabolites with drinking behavior, but druggable targets and their underlying mechanism of action are understudied.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01829-6)

10.1186/s40168-024-01829-6

---

## Sleep promoting and omics exploration on probiotics fermented <em>Gastrodia elata</em> Blume

[Source](https://www.nature.com/articles/s41538-024-00277-8)

---

## Metagenomic approach revealed the mobility and co-occurrence of antibiotic resistomes between non-intensive aquaculture environment and human.
 18.06.2024, _L Tian, G Fang, G Li, L Li, T Zhang, Y Mao_


_Aquaculture, Humans, Metagenomics, Anti-Bacterial Agents, Animals, Bacteria, Metagenome, Fishes, Drug Resistance, Bacterial, Drug Resistance, Microbial, Genes, Bacterial, Interspersed Repetitive Sequences_
Aquaculture is an important food source worldwide. The extensive use of antibiotics in intensive large-scale farms has resulted in resistance development. Non-intensive aquaculture is another aquatic feeding model that is conducive to ecological protection and closely related to the natural environment. However, the transmission of resistomes in non-intensive aquaculture has not been well characterized. Moreover, the influence of aquaculture resistomes on human health needs to be further understood. Here, metagenomic approach was employed to identify the mobility of aquaculture resistomes and estimate the potential risks to human health.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01824-x)

10.1186/s40168-024-01824-x

---

## Bacterial community and cyanotoxin gene distribution of the Winam Gulf, Lake Victoria, Kenya

[Source](https://enviromicro-journals.onlinelibrary.wiley.com/doi/10.1111/1758-2229.13297)

---

## Genome-wide screen of genetic determinants that govern Escherichia coli growth and persistence in lake water.
 21.06.2024, _N Teteneva, A Sanches-Medeiros, V Sourjik_


_Escherichia coli, Lakes, Sigma Factor, Genome, Bacterial, DNA Transposable Elements, Bacterial Proteins, Escherichia coli Proteins, Gene Expression Regulation, Bacterial, Water Microbiology_
Although enteric bacteria normally reside within the animal intestine, the ability to persist extraintestinally is an essential part of their overall lifestyle, and it might contribute to transmission between hosts. Despite this potential importance, few genetic determinants of extraintestinal growth and survival have been identified, even for the best-studied model, Escherichia coli. In this work, we thus used a genome-wide library of barcoded transposon insertions to systematically identify functional clusters of genes that are crucial for E. coli fitness in lake water. Our results revealed that inactivation of pathways involved in maintaining outer membrane integrity, nucleotide biosynthesis, and chemotaxis negatively affected E. coli growth or survival in this extraintestinal environment. In contrast, inactivation of another group of genes apparently benefited E. coli growth or persistence in filtered lake water, resulting in higher abundance of these mutants. This group included rpoS, which encodes the general stress response sigma factor, as well as genes encoding several other global transcriptional regulators and RNA chaperones, along with several poorly annotated genes. Based on this co-enrichment, we identified these gene products as novel positive regulators of RpoS activity. We further observed that, despite their enhanced growth, E. coli mutants with inactive RpoS had reduced viability in lake water, and they were not enriched in the presence of the autochthonous microbiota. This highlights the duality of the general stress response pathway for E. coli growth outside the host.

[Source](https://academic.oup.com/ismej/advance-article/doi/10.1093/ismejo/wrae096/7693287)

10.1093/ismejo/wrae096

---

## Root-associated Streptomyces produce galbonolides to modulate plant immunity and promote rhizosphere colonization.
 19.06.2024, _C Nicolle, D Gayrard, A Noël, M Hortala, A Amiel, S Grat, A Le Ru, G Marti, JL Pernodet, S Lautru, B Dumas, T Rey_


The rhizosphere, which serves as the primary interface between plant roots and the soil, constitutes an ecological niche for a huge diversity of microbial communities. Currently, there is little knowledge on the nature and the function of the different metabolites released by rhizospheric microbes to facilitate colonization of this highly competitive environment. Here, we demonstrate how the production of galbonolides, a group of polyene macrolides that inhibit plant and fungal inositol phosphorylceramide synthase (IPCS), empowers the rhizospheric Streptomyces strain AgN23, to thrive in the rhizosphere by triggering the plant's defence mechanisms. Metabolomic analysis of AgN23-inoculated Arabidopsis roots revealed a strong induction in the production of an indole alkaloid, camalexin, which is a major phytoalexin in Arabidopsis. By using a plant mutant compromized in camalexin synthesis, we show that camalexin production is necessary for the successful colonization of the rhizosphere by AgN23. Conversely, hindering galbonolides biosynthesis in AgN23 knock-out mutant resulted in loss of inhibition of IPCS, a deficiency in plant defence activation, notably the production of camalexin, and a strongly reduced development of the mutant bacteria in the rhizosphere. Together, our results identified galbonolides as important metabolites mediating rhizosphere colonization by Streptomyces.

[Source](https://academic.oup.com/ismej/advance-article/doi/10.1093/ismejo/wrae112/7696154)

10.1093/ismejo/wrae112

---

## Organic fertilization co-selects genetically linked antibiotic and metal(loid) resistance genes in global soil microbiome.
 20.06.2024, _ZT Liu, RA Ma, D Zhu, KT Konstantinidis, YG Zhu, SY Zhang_


_Soil Microbiology, Microbiota, Fertilizers, Metagenome, Drug Resistance, Microbial, Soil, Genes, Bacterial, Metals, Anti-Bacterial Agents, Agriculture_
Antibiotic resistance genes (ARGs) and metal(loid) resistance genes (MRGs) coexist in organic fertilized agroecosystems based on their correlations in abundance, yet evidence for the genetic linkage of ARG-MRGs co-selected by organic fertilization remains elusive. Here, an analysis of 511 global agricultural soil metagenomes reveals that organic fertilization correlates with a threefold increase in the number of diverse types of ARG-MRG-carrying contigs (AMCCs) in the microbiome (63 types) compared to non-organic fertilized soils (22 types). Metatranscriptomic data indicates increased expression of AMCCs under higher arsenic stress, with co-regulation of the ARG-MRG pairs. Organic fertilization heightens the coexistence of ARG-MRG in genomic elements through impacting soil properties and ARG and MRG abundances. Accordingly, a comprehensive global map was constructed to delineate the distribution of coexistent ARG-MRGs with virulence factors and mobile genes in metagenome-assembled genomes from agricultural lands. The map unveils a heightened relative abundance and potential pathogenicity risks (range of 4-6) for the spread of coexistent ARG-MRGs in Central North America, Eastern Europe, Western Asia, and Northeast China compared to other regions, which acquire a risk range of 1-3. Our findings highlight that organic fertilization co-selects genetically linked ARGs and MRGs in the global soil microbiome, and underscore the need to mitigate the spread of these co-resistant genes to safeguard public health.

[Source](https://www.nature.com/articles/s41467-024-49165-5)

10.1038/s41467-024-49165-5

---

## Conceptualizing soil fauna effects on labile and stabilized soil organic matter.
 20.06.2024, _G Angst, A Potapov, FX Joly, Š Angst, J Frouz, P Ganault, N Eisenhauer_


_Soil, Soil Microbiology, Animals, Ecosystem, Organic Chemicals, Minerals, Carbon, Microbiota_
Fauna is highly abundant and diverse in soils worldwide, but surprisingly little is known about how it affects soil organic matter stabilization. Here, we review how the ecological strategies of a multitude of soil faunal taxa can affect the formation and persistence of labile (particulate organic matter, POM) and stabilized soil organic matter (mineral-associated organic matter, MAOM). We propose three major mechanisms - transformation, translocation, and grazing on microorganisms - by which soil fauna alters factors deemed essential in the formation of POM and MAOM, including the quantity and decomposability of organic matter, soil mineralogy, and the abundance, location, and composition of the microbial community. Determining the relevance of these mechanisms to POM and MAOM formation in cross-disciplinary studies that cover individual taxa and more complex faunal communities, and employ physical fractionation, isotopic, and microbiological approaches is essential to advance concepts, models, and policies focused on soil organic matter and effectively manage soils as carbon sinks, nutrient stores, and providers of food.

[Source](https://www.nature.com/articles/s41467-024-49240-x)

10.1038/s41467-024-49240-x

---

## The first ITS1 profiling of honey samples from the Southeast Asian region Lombok, Bali and Banggi Island.
 21.06.2024, _S Ullah, F Huyop, RA Wahab, N Huda, HA Oyewusi, IGA Sujana, S Saloko, AASPR Andriani, MAN Mohamad, AA Abdul Hamid, MH Mohd Nasir, NS Antara, IBW Gunam_


_Honey, Bees, Animals, Mycobiome, Asia, Southeastern, DNA, Intergenic, Fungi, Pollen, Islands, Southeast Asian People_
Southern Asian flowers offer honeybees a diversity of nectar. Based on its geographical origin, honey quality varies. Traditional methods are less authentic than DNA-based identification. The origin of honey is determined by pollen, polyphenolic, and macro-microorganisms. In this study, amplicon sequencing targets macro-microorganisms in eDNA using the ITS1 region to explore honey's geographical location and authentication. The variety of honey samples was investigated using ITS1 with Illumina sequencing. For all four honey samples, raw sequence reads showed 979,380 raw ITS1 amplicon reads and 375 ASVs up to the phylum level. The highest total number of 202 ASVs up to phylum level identified Bali honey with 211,189 reads, followed by Banggi honey with 309,207 a total number of 111 ASVs, and Lombok represents only 63 ASVs up to phylum level with several read 458,984. Based on Shannon and Chao1, honey samples from Bali (B2) and (B3) exhibited higher diversity than honey from Lombok (B1) and green honey from Sabah (B4), while the Simpson index showed that Banggi honey (B4) had higher diversity. Honey samples had significant variance in mycobiome taxonomic composition and abundance. Zygosaccharomyces and Aspergillus were the main genera found in Lombok honey, with percentages of 68.81% and 29.76% respectively. Bali honey samples (B2 and B3) were identified as having a significant amount of the genus Aureobasidium, accounting for 40.81% and 25% of the readings, respectively. The microbiome composition of Banggi honey (B4) showed a high presence of Zygosaccharomyces 45.17% and Aureobasidium 35.24%. The ITS1 analysis effectively distinguishes between honey samples of different origins and its potential as a discriminatory tool for honey origin and authentication purposes.

[Source](https://www.nature.com/articles/s41598-024-64838-3)

10.1038/s41598-024-64838-3

---

## Asgard archaeal selenoproteome reveals a roadmap for the archaea-to-eukaryote transition of selenocysteine incorporation machinery.
 19.06.2024, _B Huang, Y Xiao, Y Zhang_


Selenocysteine (Sec) is encoded by the UGA codon that normally functions as a stop signal and is specifically incorporated into selenoproteins via a unique recoding mechanism. The translational recoding of UGA as Sec is directed by an unusual RNA structure, the Sec insertion sequence (SECIS) element. Although archaea and eukaryotes adopt a similar Sec encoding machinery, the SECIS elements have no similarities to each other with regard to sequence and structure. We analyzed more than 400 Asgard archaeal genomes to examine the occurrence of both Sec encoding system and selenoproteins in this archaeal superphylum, the closest prokaryotic relatives of eukaryotes. A comprehensive map of Sec utilization trait has been generated, providing the most detailed understanding of the use of this nonstandard amino acid in Asgard archaea so far. By characterizing the selenoproteomes of all organisms, several selenoprotein-rich phyla and species were identified. Most Asgard archaeal selenoprotein genes possess eukaryotic SECIS-like structures with varying degrees of diversity. Moreover, euryarchaeal SECIS elements might originate from Asgard archaeal SECIS elements via lateral gene transfer, indicating a complex and dynamic scenario of the evolution of SECIS element within archaea. Finally, a roadmap for the transition of eukaryotic SECIS elements from archaea was proposed, and selenophosphate synthetase may serve as a potential intermediate for the generation of ancestral eukaryotic SECIS element. Our results offer new insights into a deeper understanding of the evolution of Sec insertion machinery.

[Source](https://academic.oup.com/ismej/advance-article/doi/10.1093/ismejo/wrae111/7696153)

10.1093/ismejo/wrae111

---

## Strain tracking in complex microbiomes using synteny analysis reveals per-species modes of evolution.
 19.06.2024, _H Enav, I Paz, RE Ley_


Microbial species diversify into strains through single-nucleotide mutations and structural changes, such as recombination, insertions and deletions. Most strain-comparison methods quantify differences in single-nucleotide polymorphisms (SNPs) and are insensitive to structural changes. However, recombination is an important driver of phenotypic diversification in many species, including human pathogens. We introduce SynTracker, a tool that compares microbial strains using genome synteny-the order of sequence blocks in homologous genomic regions-in pairs of metagenomic assemblies or genomes. Genome synteny is a rich source of genomic information untapped by current strain-comparison tools. SynTracker has low sensitivity to SNPs, has no database requirement and is robust to sequencing errors. It outperforms existing tools when tracking strains in metagenomic data and is particularly suited for phages, plasmids and other low-data contexts. Applied to single-species datasets and human gut metagenomes, SynTracker, combined with an SNP-based tool, detects strains enriched in either point mutations or structural changes, providing insights into microbial evolution in situ.

[Source](https://www.nature.com/articles/s41587-024-02276-2)

10.1038/s41587-024-02276-2

---

## High-throughput single-cell transcriptomics of bacteria using combinatorial barcoding.
 17.06.2024, _KD Gaisser, SN Skloss, LM Brettner, L Paleologu, CM Roco, AB Rosenberg, M Hirano, RW DePaolo, G Seelig, A Kuchina_


Microbial split-pool ligation transcriptomics (microSPLiT) is a high-throughput single-cell RNA sequencing method for bacteria. With four combinatorial barcoding rounds, microSPLiT can profile transcriptional states in hundreds of thousands of Gram-negative and Gram-positive bacteria in a single experiment without specialized equipment. As bacterial samples are fixed and permeabilized before barcoding, they can be collected and stored ahead of time. During the first barcoding round, the fixed and permeabilized bacteria are distributed into a 96-well plate, where their transcripts are reverse transcribed into cDNA and labeled with the first well-specific barcode inside the cells. The cells are mixed and redistributed two more times into new 96-well plates, where the second and third barcodes are appended to the cDNA via in-cell ligation reactions. Finally, the cells are mixed and divided into aliquot sub-libraries, which can be stored until future use or prepared for sequencing with the addition of a fourth barcode. It takes 4 days to generate sequencing-ready libraries, including 1 day for collection and overnight fixation of samples. The standard plate setup enables single-cell transcriptional profiling of up to 1 million bacterial cells and up to 96 samples in a single barcoding experiment, with the possibility of expansion by adding barcoding rounds. The protocol requires experience in basic molecular biology techniques, handling of bacterial samples and preparation of DNA libraries for next-generation sequencing. It can be performed by experienced undergraduate or graduate students. Data analysis requires access to computing resources, familiarity with Unix command line and basic experience with Python or R.

[Source](https://www.nature.com/articles/s41596-024-01007-w)

10.1038/s41596-024-01007-w

---

## SeqCode in the golden age of prokaryotic systematics.
 19.06.2024, _DJ Jiménez, AS Rosado_


The SeqCode is a new code of prokaryotic nomenclature that was developed to validate taxon names using genome sequences as type material. The present article provides an independent view about the SeqCode, highlighting its history, current status, basic features, pros and cons, and use to date. We also discuss important topics to consider for validation of novel prokaryotic taxon names using genomes as type material. Owing to significant advances in metagenomics and cultivation methods, hundreds of novel prokaryotic species are expected to be discovered in the coming years. This manuscript aims to stimulate and enrich the debate around the use of the SeqCode in the upcoming golden age of prokaryotic taxon discovery and systematics.

[Source](https://academic.oup.com/ismej/advance-article/doi/10.1093/ismejo/wrae109/7696152)

10.1093/ismejo/wrae109

---

## Plasma metabolomics changes comparing daytime to overnight infusions of home parenteral nutrition in adult patients with short bowel syndrome: Secondary analysis of a clinical trial.
 20.06.2024, _HS Dashti, M Sevilla-Gonzalez, KM Mogensen, MF Winkler, C Compher_


_Humans, Female, Male, Middle Aged, Short Bowel Syndrome, Parenteral Nutrition, Home, Metabolomics, Adult, Circadian Rhythm_
Home parenteral nutrition (HPN) is often cycled nocturnally and is expected to result in glucose intolerance and sleep disruption partly due to circadian misalignment. This study aimed to define the metabolic response when HPN is cycled during the daytime compared to overnight.

[Source](https://www.nature.com/articles/s41575-024-00954-9)

10.1016/j.clnesp.2024.04.025

---

## Digestive Disease Week 2024 

[Source](https://www.nature.com/articles/s41575-024-00954-9)

---

## Barriers and enablers to the effective implementation of omics research in low- and middle-income countries 

[Source](https://www.nature.com/articles/s41587-024-02274-4)

---

