## A microbiological survey of handwashing sinks in the hospital built environment reveals differences in patient room and healthcare personnel sinks.
 21.05.2020, _LC Franco, W Tanner, C Ganim, T Davy, J Edwards, R Donlan_


Handwashing sinks and their associated premise plumbing are an ideal environment for pathogen-harboring biofilms to grow and spread throughout facilities due to the connected system of wastewater plumbing. This study was designed to understand the distribution of pathogens and antibiotic resistant organisms (ARO) within and among handwashing sinks in healthcare settings, using culture-dependent methods to quantify Pseudomonas aeruginosa, opportunistic pathogens capable of growth on a cefotaxime-containing medium (OPP-C), and carbapenem-resistant Enterobacteriaceae (CRE). Isolates from each medium identified as P. aeruginosa or Enterobacteriaceae were tested for susceptibility to aztreonam, ceftazidime, and meropenem; Enterobacteriaceae were also tested against ertapenem and cefotaxime. Isolates exhibiting resistance or intermediate resistance were designated ARO. Pathogens were quantified at different locations within handwashing sinks and compared in quantity and distribution between healthcare personnel (HCP) and patient room (PR) sinks. ARO were compared between samples within a sink (biofilm vs planktonic samples) and between sink types (HCP vs. PR). The drain cover was identified as a reservoir within multiple sinks that was often colonized by pathogens despite daily sink cleaning. P. aeruginosa and OPP-C mean log10 CFU/cm2 counts were higher in p-trap and tail pipe biofilm samples from HCP compared to PR sinks (2.77 ± 2.39 vs. 1.23 ± 1.62 and 5.27 ± 1.10 vs. 4.74 ± 1.06) for P. aeruginosa and OPP-C, respectively. P. aeruginosa and OPP-C mean log10 CFU/ml counts were also higher (p &lt; 0.05) in HCP compared to PR sinks p-trap water (2.21 ± 1.52 vs. 0.89 ± 1.44 and 3.87 ± 0.78 vs. 3.21 ± 1.11) for P. aeruginosa and OPP-C, respectively. However, a greater percentage of ARO were recovered from PR sinks compared to HCP sinks (p &lt; 0.05) for Enterobacteriaceae (76.4 vs. 32.9%) and P. aeruginosa (25.6 vs. 0.3%). This study supports previous work citing that handwashing sinks are reservoirs for pathogens and ARO and identifies differences in pathogen and ARO quantities between HCP and PR sinks, despite the interconnected premise plumbing.

[Source](https://www.nature.com/articles/s41598-020-65052-7)

10.1038/s41598-020-65052-7

---

## Interaction between microbiota and immunity in health and disease.
 20.05.2020, _D Zheng, T Liwinski, E Elinav_


The interplay between the commensal microbiota and the mammalian immune system development and function includes multifold interactions in homeostasis and disease. The microbiome plays critical roles in the training and development of major components of the host's innate and adaptive immune system, while the immune system orchestrates the maintenance of key features of host-microbe symbiosis. In a genetically susceptible host, imbalances in microbiota-immunity interactions under defined environmental contexts are believed to contribute to the pathogenesis of a multitude of immune-mediated disorders. Here, we review features of microbiome-immunity crosstalk and their roles in health and disease, while providing examples of molecular mechanisms orchestrating these interactions in the intestine and extra-intestinal organs. We highlight aspects of the current knowledge, challenges and limitations in achieving causal understanding of host immune-microbiome interactions, as well as their impact on immune-mediated diseases, and discuss how these insights may translate towards future development of microbiome-targeted therapeutic interventions.

[Source](https://www.nature.com/articles/s41422-020-0332-7)

10.1038/s41422-020-0332-7

---

## A Comprehensive Review on Natural Bioactive Compounds and Probiotics as Potential Therapeutics in Food Allergy Treatment

[Source](https://www.frontiersin.org/articles/10.3389/fimmu.2020.00996/full)

---

## Interaction between microbiota and immunity in health and disease.
 20.05.2020, _D Zheng, T Liwinski, E Elinav_


The interplay between the commensal microbiota and the mammalian immune system development and function includes multifold interactions in homeostasis and disease. The microbiome plays critical roles in the training and development of major components of the host's innate and adaptive immune system, while the immune system orchestrates the maintenance of key features of host-microbe symbiosis. In a genetically susceptible host, imbalances in microbiota-immunity interactions under defined environmental contexts are believed to contribute to the pathogenesis of a multitude of immune-mediated disorders. Here, we review features of microbiome-immunity crosstalk and their roles in health and disease, while providing examples of molecular mechanisms orchestrating these interactions in the intestine and extra-intestinal organs. We highlight aspects of the current knowledge, challenges and limitations in achieving causal understanding of host immune-microbiome interactions, as well as their impact on immune-mediated diseases, and discuss how these insights may translate towards future development of microbiome-targeted therapeutic interventions.

[Source](https://www.tandfonline.com/doi/full/10.1080/03036758.2020.1760896)

10.1038/s41422-020-0332-7

---

## The intestinal microbiota in health and disease

[Source](https://www.tandfonline.com/doi/full/10.1080/03036758.2020.1760896)

---

## Longitudinal Study of Oral Microbiome Variation in Twins.
 22.05.2020, _M Freire, A Moustafa, DM Harkins, MG Torralba, Y Zhang, P Leong, R Saffery, M Bockmann, C Kuelbs, T Hughes, JM Craig, KE Nelson_


Humans are host to a multitude of microorganisms that rapidly populate the body at birth, subject to a complex interplay that is dependent on host genetics, lifestyle, and environment. The host-associated microbiome, including the oral microbiome, presents itself in a complex ecosystem important to health and disease. As the most common chronic disease globally, dental caries is induced by host-microbial dysbiosis in children and adults. Multiple biological and environmental factors are likely to impact disease predisposition, onset, progression, and severity, yet longitudinal studies able to capture these influences are missing. To investigate how host genetics and environment influenced the oral microbial communities over time, we profiled supragingival plaque microbiomes of dizygotic and monozygotic twins during 3 visits over 12-months. Dental plaque DNA samples were amplified by targeting the 16S rRNA gene V4 region, and microbial findings were correlated with clinical, diet and genetic metadata. We observed that the oral microbiome variances were shaped primarily by the environment when compared to host genetics. Among the environmental factors shaping microbial changes of our subjects, significant metadata included age of the subject, and the age by which subjects initiated brushing habits, and the types of actions post-brushing. Relevant heritability of the microbiome included Actinomyces and Capnocytophaga in monozygotic twins and Kingella in dizygotic twins. Corynebacterium and Veillonella abundances were associated with age, whereas Aggregatibacter was associated with younger subjects. Streptococcus abundance showed an inverse association over time, and Selenomonas abundances increased with brushing frequency per day. Unraveling the exact biological mechanisms in caries has the potential to reveal novel host-microbial biomarkers, pathways, and targets important to effective preventive measures, and early disease control in children.

[Source](https://www.nature.com/articles/s41598-020-64747-1)

10.1038/s41598-020-64747-1

---

## Skin Microbiome in Atopic Dermatitis.
 18.05.2020, _SM Edslev, T Agner, PS Andersen_


Atopic dermatitis is a common inflammatory skin disease with a complex pathogenesis that includes imbalanced immune system signalling, impaired skin barrier and enhanced Staphylococcus aureus skin colonization. The skin bacterial communities are characterized by increasing abundance of S. aureus, leading to reduced diversity compared with the bacterial communities on healthy skin, and increasing disease severity. In contrast, fungal communities are richer and more diverse on the skin of patients with atopic dermatitis, although distribution of the most common species is similar in patients and controls. Filaggrin deficiency in atopic dermatitis skin might be related to the enhanced skin colonization by S. aureus. In addition, S. aureus expressing variant virulence factors have been shown to elicit atopic dermatitis-like phenotypes in mice, indicating that specific S. aureus strains can induce flare-ups. This review aims to provide an overview of the recent literature on the skin microbiome in atopic dermatitis.

[Source](https://doi.org/10.2340/00015555-3514)

10.2340/00015555-3514

---

## Gut microbiome diversity is an independent predictor of survival in cervical cancer patients receiving chemoradiation

[Source](https://www.medrxiv.org/content/10.1101/2020.05.18.20105809v1)

---

## The intestinal virome in children with cystic fibrosis differs from healthy controls.
 25.05.2020, _MJ Coffey, I Low, S Stelzer-Braid, B Wemheuer, M Garg, T Thomas, A Jaffe, WD Rawlinson, CY Ooi_


Intestinal bacterial dysbiosis is evident in children with cystic fibrosis (CF) and intestinal viruses may be contributory, given their influence on bacterial species diversity and biochemical cycles. We performed a prospective, case-control study on children with CF and age and gender matched healthy controls (HC), to investigate the composition and function of intestinal viral communities. Stool samples were enriched for viral DNA and RNA by viral extraction, random amplification and purification before sequencing (Illumina MiSeq). Taxonomic assignment of viruses was performed using Vipie. Functional annotation was performed using Virsorter. Inflammation was measured by calprotectin and M2-pyruvate kinase (M2-PK). Eight CF and eight HC subjects were included (50% male, mean age 6.9 ± 3.0 and 6.4 ± 5.3 years, respectively, p = 0.8). All CF subjects were pancreatic insufficient. Regarding the intestinal virome, no difference in Shannon index between CF and HC was identified. Taxonomy-based beta-diversity (presence-absence Bray-Curtis dissimilarity) was significantly different between CF and HC (R2 = 0.12, p = 0.001). Myoviridae, Faecalibacterium phage FP Taranis and unclassified Gokushovirinae were significantly decreased in CF compared with HC (q&lt;0.05). In children with CF (compared to HC), the relative abundance of genes annotated to (i) a peptidoglycan-binding domain of the peptidoglycan hydrolases (COG3409) was significantly increased (q&lt;0.05) and (ii) capsid protein (F protein) (PF02305.16) was significantly decreased (q&lt;0.05). Picornavirales, Picornaviridae, and Enterovirus were found to positively correlate with weight and BMI (r = 0.84, q = 0.01). Single-stranded DNA viruses negatively correlated with M2-PK (r = -0.86, q = 0.048). Children with CF have an altered intestinal virome compared to well-matched HC, with both taxonomic and predicted functional changes. Further exploration of Faecalibacterium phages, Gokushovirinae and phage lysins are warranted. Intestinal viruses and their functions may have important clinical implications for intestinal inflammation and growth in children with CF, potentially providing novel therapeutic targets.

[Source](https://doi.org/10.1371/journal.pone.0233557)

10.1371/journal.pone.0233557

---

## The human gut microbiota and depression: widely reviewed, yet poorly understood

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0165032720307588)

---

## The Firmicutes/Bacteroidetes Ratio: A Relevant Marker of Gut Dysbiosis in Obese Patients?
 22.05.2020, _F Magne, M Gotteland, L Gauthier, A Zazueta, S Pesoa, P Navarrete, R Balamurugan_


The gut microbiota is emerging as a promising target for the management or prevention of inflammatory and metabolic disorders in humans. Many of the current research efforts are focused on the identification of specific microbial signatures, more particularly for those associated with obesity, type 2 diabetes, and cardiovascular diseases. Some studies have described that the gut microbiota of obese animals and humans exhibits a higher Firmicutes/Bacteroidetes ratio compared with normal-weight individuals, proposing this ratio as an eventual biomarker. Accordingly, the Firmicutes/Bacteroidetes ratio is frequently cited in the scientific literature as a hallmark of obesity. The aim of the present review was to discuss the validity of this potential marker, based on the great amount of contradictory results reported in the literature. Such discrepancies might be explained by the existence of interpretative bias generated by methodological differences in sample processing and DNA sequence analysis, or by the generally poor characterization of the recruited subjects and, more particularly, the lack of consideration of lifestyle-associated factors known to affect microbiota composition and/or diversity. For these reasons, it is currently difficult to associate the Firmicutes/Bacteroidetes ratio with a determined health status and more specifically to consider it as a hallmark of obesity.

[Source](https://www.mdpi.com/2072-6643/12/5/1474)

10.3390/nu12051474

---

## Who Inhabits Our Gut? Introducing the Human Gut Microbiota

[Source](https://link.springer.com/chapter/10.1007/978-3-030-43246-1_5)

---

## A Robust Metatranscriptomic Technology for Population-Scale Studies of Diet, Gut Microbiome, and Human Health.
 01.11.2019, _A Hatch, J Horne, R Toma, BL Twibell, KM Somerville, B Pelle, KP Canfield, M Genkin, G Banavar, A Perlina, H Messier, N Klitgord, M Vuyisich_


A functional readout of the gut microbiome is necessary to enable precise control of the gut microbiome's functions, which support human health and prevent or minimize a wide range of chronic diseases. Stool metatranscriptomic analysis offers a comprehensive functional view of the gut microbiome, but despite its usefulness, it has rarely been used in clinical studies due to its complexity, cost, and bioinformatic challenges. This method has also received criticism due to potential intrasample variability, rapid changes, and RNA degradation. Here, we describe a robust and automated stool metatranscriptomic method, called Viomega, which was specifically developed for population-scale studies. Viomega includes sample collection, ambient temperature sample preservation, total RNA extraction, physical removal of ribosomal RNAs (rRNAs), preparation of directional Illumina libraries, Illumina sequencing, taxonomic classification based on a database of &gt;110,000 microbial genomes, and quantitative microbial gene expression analysis using a database of ~100 million microbial genes. We applied this method to 10,000 human stool samples and performed several small-scale studies to demonstrate sample stability and consistency. In summary, Viomega is an inexpensive, high-throughput, automated, and accurate sample-to-result stool metatranscriptomic technology platform for large-scale studies and a wide range of applications.

[Source](https://link.springer.com/chapter/10.1007/978-3-030-43246-1_6)

10.1155/2019/1718741

---

## Variability and Stability of the Human Gut Microbiome

[Source](https://link.springer.com/chapter/10.1007/978-3-030-43246-1_6)

---

## Health impact of the Anthropocene: the complex relationship between gut microbiota, epigenetics, and human health, using obesity as an example.
 07.05.2020, _C Torp Austvoll, V Gallo, D Montag_


The growing prevalence of obesity worldwide poses a public health challenge in the current geological epoch, the Anthropocene. Global changes caused by urbanisation, loss of biodiversity, industrialisation, and land-use are happening alongside microbiota dysbiosis and increasing obesity prevalence. How alterations of the gut microbiota are associated with obesity and the epigenetic mechanism mediating this and other health outcome associations are in the process of being unveiled. Epigenetics is emerging as a key mechanism mediating the interaction between human body and the environment in producing disease. Evidence suggests that the gut microbiota plays a role in obesity as it contributes to different mechanisms, such as metabolism, body weight and composition, inflammatory responses, insulin signalling, and energy extraction from food. Consistently, obese people tend to have a different epigenetic profile compared to non-obese. However, evidence is usually scattered and there is a growing need for a structured framework to conceptualise this complexity and to help shaping complex solutions. In this paper, we propose a framework to analyse the observed associations between the alterations of microbiota and health outcomes and the role of epigenetic mechanisms underlying them using obesity as an example, in the current context of global changes within the Anthropocene.

[Source](https://link.springer.com/chapter/10.1007/978-3-030-43246-1_3)

10.1017/gheg.2020.2

---

## How to Analyse Microbial Communities?

[Source](https://link.springer.com/chapter/10.1007/978-3-030-43246-1_3)

---

## [Gut Microbiota and Pancreatobiliary System].
 25.05.2020, _KH Ryu_


The gut microbiota is part of the human body that is involved in body metabolism and the occurrence of various diseases. Detecting and analyzing their genetic information (microbiome) is as important as analyzing human genes. The core microbiome, the key functional genes shared by all humans, helps better understand the physiology of the human body. Information on the gut microbiome of a diseased person can help diagnose and treat disease. The pancreatobiliary system releases functional antimicrobial substances, such as bile acids and antimicrobial peptides, which affect the gut microbiota directly. In response, the gut microbiota influences pancreatobiliary secretion by controlling the generation and emission of substances through indirect signaling. This crosstalk maintains homeostasis of the pancreatobiliary system secretion and microbiota. Dysbiosis and disease can occur if this fails to work properly. Bile acid therapy has been used widely and may affect the microbial environment in the intestine. An association of the gut microbiota has been reported in many cases of pancreatobiliary diseases, including malignant tumors. Traditionally, most pancreatobiliary diseases are accompanied by infections from the gut microbiota, which is an important target for treatment. The pancreatobiliary system can control its function through physical and drug therapy. This may be a new pioneering field in the study or treatment of the gut microbiota.

[Source](https://link.springer.com/chapter/10.1007/978-3-030-43246-1_11)

10.4166/kjg.2020.75.5.231

---

## Treating the Gut Microbiome as a System

[Source](https://link.springer.com/chapter/10.1007/978-3-030-43246-1_11)

---

## Application of metabolomics to the study of irritable bowel syndrome

[Source](https://doi.org/10.1111/nmo.13884)

---

## Distinct actions of the fermented beverage kefir on host behaviour, immunity and microbiome gut-brain modules in the mouse.
 19.05.2020, _M van de Wouw, AM Walsh, F Crispie, L van Leuven, JM Lyte, M Boehme, G Clarke, TG Dinan, PD Cotter, JF Cryan_


Mounting evidence suggests a role for the gut microbiota in modulating brain physiology and behaviour, through bi-directional communication, along the gut-brain axis. As such, the gut microbiota represents a potential therapeutic target for influencing centrally mediated events and host behaviour. It is thus notable that the fermented milk beverage kefir has recently been shown to modulate the composition of the gut microbiota in mice. It is unclear whether kefirs have differential effects on microbiota-gut-brain axis and whether they can modulate host behaviour per se.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00846-5)

10.1186/s40168-020-00846-5

---

## Physical activity shapes the intestinal microbiome and immunity of healthy mice but has no protective effects against colitis in MUC2-/- mice

[Source](https://www.biorxiv.org/content/10.1101/2020.05.24.113290v1.abstract)

---

## Synergetic responses of intestinal microbiota and epithelium to dietary inulin supplementation in pigs.
 21.05.2020, _J He, H Xie, D Chen, B Yu, Z Huang, X Mao, P Zheng, Y Luo, J Yu, J Luo, H Yan_


Inulin is a soluble dietary fiber that has been implicated in regulating the intestinal health. Here, we describe a synergetic response of intestinal microbiota and epithelial functions to increased intake of inulin in a porcine model.

[Source](https://link.springer.com/article/10.1007%2Fs00394-020-02284-3)

10.1007/s00394-020-02284-3

---

## No correlation between microbiota composition and blood parameters in nesting flatback turtles (Natator depressus).
 20.05.2020, _TF Scheelings, RJ Moore, TTH Van, M Klaassen, RD Reina_


The microbiota is considered critical for normal vertebrate homeostasis and it may exert its effects at a local level within the gastrointestinal tract, or systemically through the production of bacterial metabolites. To date, investigations into the role that the microbiota plays in reptile physiology are rare. To address this knowledge gap, we explored the relationship between differences in microbial communities to see if they accounted for differences in haematology and biochemistry values, in different populations of nesting flatback turtles (Natator depressus). We found that microbiota composition was not correlated to any of the blood analytes we measured in flatbacks. This study is the first of its kind in reptiles and highlights the need for further investigations to determine mechanisms by which the microbiota influences the physiology and health of reptiles.

[Source](https://www.nature.com/articles/s41598-020-65321-5)

10.1038/s41598-020-65321-5

---

## Linking perturbations to temporal changes in diversity, stability, and compositions of neonatal calf gut microbiota: prediction of diarrhea.
 23.05.2020, _T Ma, C Villot, D Renaud, A Skidmore, E Chevaux, M Steele, LL Guan_


Perturbations in early life gut microbiota can have long-term impacts on host health. In this study, we investigated antimicrobial-induced temporal changes in diversity, stability, and compositions of gut microbiota in neonatal veal calves, with the objective of identifying microbial markers that predict diarrhea. A total of 220 samples from 63 calves in first 8 weeks of life were used in this study. The results suggest that increase in diversity and stability of gut microbiota over time was a feature of "healthy" (non-diarrheic) calves during early life. Therapeutic antimicrobials delayed the temporal development of diversity and taxa-function robustness (a measure of microbial stability). In addition, predicted genes associated with beta lactam and cationic antimicrobial peptide resistance were more abundant in gut microbiota of calves treated with therapeutic antimicrobials. Random forest machine learning algorithm revealed that Trueperella, Streptococcus, Dorea, uncultured Lachnospiraceae, Ruminococcus 2, and Erysipelatoclostridium may be key microbial markers that can differentiate "healthy" and "unhealthy" (diarrheic) gut microbiota, as they predicted early life diarrhea with an accuracy of 84.3%. Our findings suggest that diarrhea in veal calves may be predicted by the shift in early life gut microbiota, which may provide an opportunity for early intervention (e.g., prebiotics or probiotics) to improve calf health with reduced usage of antimicrobials.

[Source](https://www.nature.com/articles/s41396-020-0678-3)

10.1038/s41396-020-0678-3

---

## The microbiome as a biosensor: functional profiles elucidate hidden stress in hosts.
 22.05.2020, _A Zolti, SJ Green, N Sela, Y Hadar, D Minz_


Microbial communities are highly responsive to environmental cues, and both their structure and activity can be altered in response to changing conditions. We hypothesized that host-associated microbial communities, particularly those colonizing host surfaces, can serve as in situ sensors to reveal environmental conditions experienced by both microorganisms and the host. For a proof-of-concept, we studied a model plant-soil system and employed a non-deterministic gene-centric approach. A holistic analysis was performed using plants of two species and irrigation with water of low quality to induce host stress. Our analyses examined the genetic potential (DNA) and gene expression patterns (RNA) of plant-associated microbial communities, as well as transcriptional profiling of host plants.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00850-9)

10.1186/s40168-020-00850-9

---

## 16S rRNA Gene Diversity in the Salt Crust of Salar de Uyuni, Bolivia, the World's Largest Salt Flat.
 22.05.2020, _WT Pecher, FL Martínez, P DasSarma, D Guzmán, S DasSarma_


Salar de Uyuni is a vast, high-altitude salt flat in Bolivia with extreme physico-geochemical properties approaching multiple limits of life. Evidence for diverse halophilic bacteria and archaea was found in its surface and near-surface salt crust using 16S amplicon analysis, providing a snapshot of prokaryotic life.

[Source](https://doi.org/ 10.1128/MRA.00374-20)

10.1128/MRA.00374-20

---

## Fungal diversity and its contribution to the biodeterioration of mural paintings in two 1700-year-old tombs of China

[Source](https://www.sciencedirect.com/science/article/pii/S0964830520303814)

---

## Characterizing Microbial Signatures on Sculptures and Paintings of Similar Provenance.
 22.05.2020, _MG Torralba, C Kuelbs, KJ Moncera, R Roby, KE Nelson_


The preservation of artwork challenges museums, collectors, and art enthusiasts. Currently, reducing moisture, adjusting the type of lighting, and preventing the formation of mold are primary methods to preserving and preventing deterioration. Other methods such as ones based in detailed knowledge of molecular biology such as microbial community characterization using polymerase chain reaction (PCR) and sequencing have yet to be explored. Such molecular biology approaches are essential to explore as some environmental bacteria are capable of oxidizing nonpolar chemical substances rich in hydrocarbons such as oil-based paints. Using 16S rDNA Illumina Sequencing, we demonstrate a novel finding that there are differing bacterial communities for artwork from roughly the same era when comparing paintings on wood, paintings on canvases, and sculptures made of stone and marble. We also demonstrate that there are specific genera such as Aeromonas known for having oxidase positive strains, present on paintings on wood and paintings on canvas that could potentially be responsible for deterioration and fading as such organisms produce water or hydrogen peroxide as a byproduct of cytochrome c oxidase activity. The advantages of these genomics-based approaches to characterizing the microbial population on deteriorating artwork provides immense potential by identifying potentially damaging species that may not be detected using conventional methods in addition to addressing challenges to identification, restoration, and preservation efforts.

[Source](https://doi.org/10.1007/s00248-020-01504-x)

10.1007/s00248-020-01504-x

---

## Proteomic and Metabolomic Correlates of Healthy Dietary Patterns: The Framingham Heart Study.
 22.05.2020, _ME Walker, RJ Song, X Xu, RE Gerszten, D Ngo, CB Clish, L Corlin, J Ma, V Xanthakis, PF Jacques, RS Vasan_


Data on proteomic and metabolomic signatures of healthy dietary patterns are limited. We evaluated the cross-sectional association of serum proteomic and metabolomic markers with three dietary patterns: the Alternative Healthy Eating Index (AHEI), the Dietary Approaches to Stop Hypertension (DASH) diet; and a Mediterranean-style (MDS) diet. We examined participants from the Framingham Offspring Study (mean age; 55 years; 52% women) who had complete proteomic (n = 1713) and metabolomic (n = 2284) data; using food frequency questionnaires to derive dietary pattern indices. Proteins and metabolites were quantified using the SomaScan platform and liquid chromatography/tandem mass spectrometry; respectively. We used multivariable-adjusted linear regression models to relate each dietary pattern index (independent variables) to each proteomic and metabolomic marker (dependent variables). Of the 1373 proteins; 103 were associated with at least one dietary pattern (48 with AHEI; 83 with DASH; and 8 with MDS; all false discovery rate [FDR] ≤ 0.05). We identified unique associations between dietary patterns and proteins (17 with AHEI; 52 with DASH; and 3 with MDS; all FDR ≤ 0.05). Significant proteins enriched biological pathways involved in cellular metabolism/proliferation and immune response/inflammation. Of the 216 metabolites; 65 were associated with at least one dietary pattern (38 with AHEI; 43 with DASH; and 50 with MDS; all FDR ≤ 0.05). All three dietary patterns were associated with a common signature of 24 metabolites (63% lipids). Proteins and metabolites associated with dietary patterns may help characterize intermediate phenotypes that provide insights into the molecular mechanisms mediating diet-related disease. Our findings warrant replication in independent populations.

[Source](https://www.mdpi.com/2072-6643/12/5/1476)

10.3390/nu12051476

---

## ARTS 2.0: feature updates and expansion of the Antibiotic Resistant Target Seeker for comparative genome mining.
 19.05.2020, _MD Mungan, M Alanjary, K Blin, T Weber, MH Medema, N Ziemert_


Multi-drug resistant pathogens have become a major threat to human health and new antibiotics are urgently needed. Most antibiotics are derived from secondary metabolites produced by bacteria. In order to avoid suicide, these bacteria usually encode resistance genes, in some cases within the biosynthetic gene cluster (BGC) of the respective antibiotic compound. Modern genome mining tools enable researchers to computationally detect and predict BGCs that encode the biosynthesis of secondary metabolites. The major challenge now is the prioritization of the most promising BGCs encoding antibiotics with novel modes of action. A recently developed target-directed genome mining approach allows researchers to predict the mode of action of the encoded compound of an uncharacterized BGC based on the presence of resistant target genes. In 2017, we introduced the 'Antibiotic Resistant Target Seeker' (ARTS). ARTS allows for specific and efficient genome mining for antibiotics with interesting and novel targets by rapidly linking housekeeping and known resistance genes to BGC proximity, duplication and horizontal gene transfer (HGT) events. Here, we present ARTS 2.0 available at http://arts.ziemertlab.com. ARTS 2.0 now includes options for automated target directed genome mining in all bacterial taxa as well as metagenomic data. Furthermore, it enables comparison of similar BGCs from different genomes and their putative resistance genes.

[Source](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkaa374/5840579)

10.1093/nar/gkaa374

---

## Draft Genome Sequence of a Novel Species of Halococcus (Strain IIIV-5B), an Endophytic Archaeon Isolated from the Leaf Tissue of Avicennia germinans.
 22.05.2020, _J Zayas-Rivera, Y Rivera-Lopez, M Velázquez-Méndez, N Romero-Oliveras, R Montalvo-Rodríguez_


Strain IIIV-5B was isolated from the leaf tissue of the black mangrove, Avicennia germinans This microorganism belongs to the genus Halococcus Here, we present the draft genome sequence of strain IIIV-5B, a novel species of this genus. The 3,869,808-bp genome has a G+C content of 63.9% and around 3,812 coding sequences.

[Source](https://doi.org/10.1128/MRA.00421-20)

10.1128/MRA.00421-20

---

## Precise phylogenetic analysis of microbial isolates and genomes from metagenomes using PhyloPhlAn 3.0.
 19.05.2020, _F Asnicar, AM Thomas, F Beghini, C Mengoni, S Manara, P Manghi, Q Zhu, M Bolzan, F Cumbo, U May, JG Sanders, M Zolfo, E Kopylova, E Pasolli, R Knight, S Mirarab, C Huttenhower, N Segata_


Microbial genomes are available at an ever-increasing pace, as cultivation and sequencing become cheaper and obtaining metagenome-assembled genomes (MAGs) becomes more effective. Phylogenetic placement methods to contextualize hundreds of thousands of genomes must thus be efficiently scalable and sensitive from closely related strains to divergent phyla. We present PhyloPhlAn 3.0, an accurate, rapid, and easy-to-use method for large-scale microbial genome characterization and phylogenetic analysis at multiple levels of resolution. PhyloPhlAn 3.0 can assign genomes from isolate sequencing or MAGs to species-level genome bins built from &gt;230,000 publically available sequences. For individual clades of interest, it reconstructs strain-level phylogenies from among the closest species using clade-specific maximally informative markers. At the other extreme of resolution, it scales to large phylogenies comprising &gt;17,000 microbial species. Examples including Staphylococcus aureus isolates, gut metagenomes, and meta-analyses demonstrate the ability of PhyloPhlAn 3.0 to support genomic and metagenomic analyses.

[Source](https://www.nature.com/articles/s41467-020-16366-7)

10.1038/s41467-020-16366-7

---

