## Diagnostic and prognostic potential of the oral and gut microbiome for lung adenocarcinoma.
 30.09.2021, _MY Lim, S Hong, KH Hwang, EJ Lim, JY Han, YD Nam_



[Source](https://onlinelibrary.wiley.com/doi/10.1002/ctm2.508)

10.1002/ctm2.508

---

## Individuals with substance use disorders have a distinct oral microbiome pattern

[Source](https://www.sciencedirect.com/science/article/pii/S2666354621000740)

---

## The Dynamics of Skin Microbiota in Shoulder Surgery Infections

[Source](https://onlinelibrary.wiley.com/doi/10.1111/apm.13185)

---

## Mucins, gut microbiota, and postbiotics role in colorectal cancer

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2021.1974795)

---

## Age and the aging process significantly alter the small bowel microbiome

[Source](https://www.sciencedirect.com/science/article/pii/S2211124721012195)

---

## Guidelines for reporting on animal fecal transplantation (GRAFT) studies: recommendations from a systematic review of murine transplantation protocols

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2021.1979878)

---

## Alpha-synuclein alters the faecal viromes of rats in a gut-initiated model of Parkinsonâ€™s disease

[Source](https://www.nature.com/articles/s42003-021-02666-1)

---

## First trimester gut microbiome induces Inflammation-dependent gestational diabetes phenotype in mice

[Source](https://www.medrxiv.org/content/10.1101/2021.09.17.21262268v1)

---

## High-resolution genetic mapping of the mucosa-associated gut microbiome in hybrid mice provides novel insight on the impact of host genetics

[Source](https://www.biorxiv.org/content/10.1101/2021.09.28.462095v1)

---

## Effects of microplastics on soil microbiome: The impacts of polymer type, shape, and concentration

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048969721055935)

---

## Long-read metagenomics of multiple displacement amplified DNA of low-biomass human gut phageomes by SACRA preprocessing chimeric reads

[Source](https://academic.oup.com/dnaresearch/advance-article/doi/10.1093/dnares/dsab019/6377780)

---

## BiG-MAP: an Automated Pipeline To Profile Metabolic Gene Cluster Abundance and Expression in Microbiomes.
 28.09.2021, _V Pascal Andreu, HE Augustijn, K van den Berg, JJJ van der Hooft, MA Fischbach, MH Medema_


Microbial gene clusters encoding the biosynthesis of primary and secondary metabolites play key roles in shaping microbial ecosystems and driving microbiome-associated phenotypes. Although effective approaches exist to evaluate the metabolic potential of such bacteria through identification of these metabolic gene clusters in their genomes, no automated pipelines exist to profile the abundance and expression levels of such gene clusters in microbiome samples to generate hypotheses about their functional roles, and to find associations with phenotypes of interest. Here, we describe BiG-MAP, a bioinformatic tool to profile abundance and expression levels of gene clusters across metagenomic and metatranscriptomic data and evaluate their differential abundance and expression under different conditions. To illustrate its usefulness, we analyzed 96 metagenomic samples from healthy and caries-associated human oral microbiome samples and identified 252 gene clusters, including unreported ones, that were significantly more abundant in either phenotype. Among them, we found the muc operon, a gene cluster known to be associated with tooth decay. Additionally, we found a putative reuterin biosynthetic gene cluster from a Streptococcus strain to be enriched but not exclusively found in healthy samples; metabolomic data from the same samples showed masses with fragmentation patterns consistent with (poly)acrolein, which is known to spontaneously form from the products of the reuterin pathway and has been previously shown to inhibit pathogenic Streptococcus mutans strains. Thus, we show how BiG-MAP can be used to generate new hypotheses on potential drivers of microbiome-associated phenotypes and prioritize the experimental characterization of relevant gene clusters that may mediate them. IMPORTANCE Microbes play an increasingly recognized role in determining host-associated phenotypes by producing small molecules that interact with other microorganisms or host cells. The production of these molecules is often encoded in syntenic genomic regions, also known as gene clusters. With the increasing numbers of (multi)omics data sets that can help in understanding complex ecosystems at a much deeper level, there is a need to create tools that can automate the process of analyzing these gene clusters across omics data sets. This report presents a new software tool called BiG-MAP, which allows assessing gene cluster abundance and expression in microbiome samples using metagenomic and metatranscriptomic data. Here, we describe the tool and its functionalities, as well as its validation using a mock community. Finally, using an oral microbiome data set, we show how it can be used to generate hypotheses regarding the functional roles of gene clusters in mediating host phenotypes.

[Source](https://journals.asm.org/doi/10.1128/mSystems.00937-21)

10.1128/mSystems.00937-21

---

## Increasing the power of interpretation for soil metaproteomics data.
 30.09.2021, _V Jouffret, G Miotello, K Culotta, S Ayrault, O Pible, J Armengaud_


Soil and sediment microorganisms are highly phylogenetically diverse but are currently largely under-represented in public molecular databases. Their functional characterization by means of metaproteomics is usually performed using metagenomic sequences acquired for the same sample. However, such hugely diverse metagenomic datasets are difficult to assemble; in parallel, theoretical proteomes from isolates available in generic databases are of high quality. Both these factors advocate for the use of theoretical proteomes in metaproteomics interpretation pipelines. Here, we examined a number of database construction strategies with a view to increasing the outputs of metaproteomics studies performed on soil samples.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01139-1)

10.1186/s40168-021-01139-1

---

