## Gut metagenomes of Asian octogenarians reveal metabolic potential expansion and distinct microbial species associated with aging phenotypes.
 08.09.2024, _A Ravikrishnan, I Wijaya, E Png, KR Chng, EXP Ho, AHQ Ng, AN Mohamed Naim, JS Gounot, SP Guan, JL Hanqing, L Guan, C Li, JY Koh, PF de Sessions, WP Koh, L Feng, TP Ng, A Larbi, AB Maier, BK Kennedy, N Nagarajan_


_Humans, Gastrointestinal Microbiome, Singapore, Male, Aged, 80 and over, Female, Aging, Metagenome, Asian People, Phenotype, Metagenomics, Bacteria, Bacteroides, Cohort Studies, Feces_
While rapid demographic changes in Asia are driving the incidence of chronic aging-related diseases, the limited availability of high-quality in vivo data hampers our ability to understand complex multi-factorial contributions, including gut microbial, to healthy aging. Leveraging a well-phenotyped cohort of community-living octogenarians in Singapore, we used deep shotgun-metagenomic sequencing for high-resolution taxonomic and functional characterization of their gut microbiomes (n = 234). Joint species-level analysis with other Asian cohorts identified distinct age-associated shifts characterized by reduction in microbial richness, and specific Alistipes and Bacteroides species enrichment (e.g., Alistipes shahii and Bacteroides xylanisolvens). Functional analysis confirmed these changes correspond to metabolic potential expansion in aging towards alternate pathways synthesizing and utilizing amino-acid precursors, vis-à-vis dominant microbial guilds producing butyrate in gut from pyruvate (e.g., Faecalibacterium prausnitzii, Roseburia inulinivorans). Extending these observations to key clinical markers helped identify &gt;10 robust microbial associations to inflammation, cardiometabolic and liver health, including potential probiotic species (e.g., Parabacteroides goldsteinii) and pathobionts (e.g., Klebsiella pneumoniae), highlighting the microbiome's role as biomarkers and potential targets for promoting healthy aging.

[Source](https://doi.org/10.1038/s41467-024-52097-9)

10.1038/s41467-024-52097-9

---

## Integrated analysis of facial microbiome and skin physio-optical properties unveils cutotype-dependent aging effects.
 07.09.2024, _C Sun, G Hu, L Yi, W Ge, Q Yang, X Yang, Y He, Z Liu, WH Chen_


_Humans, Microbiota, Skin, Face, Middle Aged, Skin Aging, Female, Adult, Male, Bacteria, Aged, Aging, Metagenome, Young Adult, High-Throughput Nucleotide Sequencing, Sebum_
Our facial skin hosts millions of microorganisms, primarily bacteria, crucial for skin health by maintaining the physical barrier, modulating immune response, and metabolizing bioactive materials. Aging significantly influences the composition and function of the facial microbiome, impacting skin immunity, hydration, and inflammation, highlighting potential avenues for interventions targeting aging-related facial microbes amidst changes in skin physiological properties.

[Source](https://doi.org/10.1186/s40168-024-01891-0)

10.1186/s40168-024-01891-0

---

## Regulation of <em>Bacteroides acidifaciens</em> by the aryl hydrocarbon receptor in IL-22-producing immune cells has sex-dependent consequential impact on colitis

[Source](https://doi.org/10.3389/fimmu.2024.1444045)

---

## The role of KLF5 in gut microbiota and lung adenocarcinoma: unveiling programmed cell death pathways and prognostic biomarkers.
 08.09.2024, _Q Fang, M Xu, W Yao, R Wu, R Han, S Kawakita, A Shen, S Guan, J Zhang, X Sun, M Zhou, N Li, Q Sun, CS Dong_


Lung adenocarcinoma (LUAD) is the most important subtype of lung cancer. It is well known that the gut microbiome plays an important role in the pathophysiology of various diseases, including cancer, but little research has been done on the intestinal microbiome associated with LUAD. Utilizing bioinformatics tools and data analysis, we identified novel potential prognostic biomarkers for LUAD. To integrate differentially expressed genes and clinical significance modules, we used a weighted correlation network analysis system. According to the Peryton database and the gutMGene database, the composition and structure of gut microbiota in LUAD patients differed from those in healthy individuals. LUAD was associated with 150 gut microbiota and 767 gut microbiota targets, with Krüppel-like factor 5 (KLF5) being the most closely related. KLF5 was associated with immune status and correlated well with the prognosis of LUAD patients. The identification of KLF5 as a potential prognostic biomarker suggests its utility in improving risk stratification and guiding personalized treatment strategies for LUAD patients. Altogether, KLF5 could be a potential prognostic biomarker in LUAD.

[Source](https://doi.org/10.1007/s12672-024-01257-w)

10.1007/s12672-024-01257-w

---

## Healthy first-degree relatives from multiplex families versus simplex families have a higher subclinical intestinal inflammation, a distinct fecal microbial signature, and harbor a higher risk of developing Crohn’s disease

[Source](https://www.gastrojournal.org/article/S0016-5085(24)05414-3/pdf)

---

## Reversions mask the contribution of adaptive evolution in microbiomes

[Source](https://doi.org/10.7554/eLife.93146)

---

## The hidden base of the iceberg: gut peptidoglycome dynamics is foundational to its influence on the host

[Source](https://doi.org/10.1080/19490976.2024.2395099)

---

## Benefits and concerns of probiotics: an overview of the potential genotoxicity of the colibactin-producing <em>Escherichia coli </em>Nissle 1917 strain

[Source](https://doi.org/10.1080/19490976.2024.2397874)

---

## Microbial Extracellular Vesicles in Host-Microbiota Interactions

[Source](https://doi.org/10.1007/978-3-031-62036-2_19)

---

## A probiotic <em>Limosilactobacillus fermentum </em>GR-3 mitigates colitis-associated tumorigenesis in mice via modulating gut microbiome

[Source](https://doi.org/10.1038/s41538-024-00307-5)

---

## Pathophysiological and Clinical Potential of Human Microbiome: Microbe-based Therapeutic Insights

[Source](https://doi.org/10.2174/0113892010314433240823113111)

---

## Soil volatilomics uncovers tight linkage between soybean presence and soil omics profiles in agricultural fields

[Source](https://doi.org/10.1038/s41598-024-70873-x)

---

## Meddling with the microbiota: Fungal tricks to infect plant hosts

[Source](https://doi.org/10.1016/j.pbi.2024.102622)

---

## A comprehensive catalog encompassing 1376 species-level genomes reveals the core community and functional diversity of anammox microbiota

[Source](https://doi.org/10.1016/j.watres.2024.122356)

---

## Revealing the intricate temporal dynamics and adaptive responses of prokaryotic and eukaryotic microbes in the coastal South China Sea

[Source](https://doi.org/10.1016/j.scitotenv.2024.176019)

---

## Biodegradation of low-density polyethylene by mixed fungi composed of <em>Alternaria </em>sp. and <em>Trametes </em>sp. isolated from landfill sites

[Source](https://doi.org/10.1186/s12866-024-03477-0)

---

## Using patterns of shared taxa to infer bacterial dispersal in human living environment in urban and rural areas

[Source](https://doi.org/10.1128/aem.00903-24)

---

## Annotation-free prediction of microbial dioxygen utilization

[Source](https://doi.org/10.1128/msystems.00763-24)

---

## MRGM: an enhanced catalog of mouse gut microbial genomes substantially broadening taxonomic and functional landscapes

[Source](https://doi.org/10.1080/19490976.2024.2393791)

---

## A unified web cloud computing platform MiMedSurv for microbiome causal mediation analysis with survival responses

[Source](https://doi.org/10.1038/s41598-024-71852-y)

---

## A quantitative approach to measure and predict microbiome response to antibiotics

[Source](https://doi.org/10.1128/msphere.00488-24)

---

## Conceptual Parallels: Microbiome Research and Ancient Medicine.
 09.09.2024, _L Sumrall, MA O'Malley_


_Humans, Microbiota, Biomedical Research_
The concepts currently operating in much medical microbiome research bear a curious resemblance to an ancient tradition of Western medicine. This tradition, humoral medicine, is concerned with the four humors: yellow and black bile, phlegm, blood. Both humoral medicine and medical microbiome research use notions of imbalance and balance for broad explanations of disease and health. Both traditions also hold that the composition of humors or microbiomes determines bodily as well as mental states. Causality in each system is often conceived teleologically, meaning that humors or microbiomes "function for" the maintenance of the whole. And ultimately, each framework situates the humors or microbiomes in a multilevel interactionist theory that conceptualizes individual health within a broader environmental context. As well as critically assessing the parallels between these systems, this article sketches some explanations of how they may have arisen. The authors also evaluate the implications of these similarities for the future of medical microbiome research and suggest ways in which the field might move forward.

[Source](https://doi.org/10.1093/bioinformatics/btae397)

10.1353/pbm.2024.a936218

---

## Metagenomic functional profiling: to sketch or not to sketch?

[Source](https://doi.org/10.1093/bioinformatics/btae397)

---

## TolRad, a model for predicting radiation tolerance using Pfam annotations, identifies novel radiosensitive bacterial species from reference genomes and MAGs

[Source](https://doi.org/10.1128/spectrum.03838-23)

---

## Improving viral annotation with artificial intelligence.
 04.09.2024, _ZN Flamholz, C Li, L Kelly_


Viruses of bacteria, "phages," are fundamental, poorly understood components of microbial community structure and function. Additionally, their dependence on hosts for replication positions phages as unique sensors of ecosystem features and environmental pressures. High-throughput sequencing approaches have begun to give us access to the diversity and range of phage populations in complex microbial community samples, and metagenomics is currently the primary tool with which we study phage populations. The study of phages by metagenomic sequencing, however, is fundamentally limited by viral diversity, which results in the vast majority of viral genomes and metagenome-annotated genomes lacking annotation. To harness bacteriophages for applications in human and environmental health and disease, we need new methods to organize and annotate viral sequence diversity. We recently demonstrated that methods that leverage self-supervised representation learning can supplement statistical sequence representations for remote viral protein homology detection in the ocean virome and propose that consideration of the functional content of viral sequences allows for the identification of similarity in otherwise sequence-diverse viruses and viral-like elements for biological discovery. In this review, we describe the potential and pitfalls of large language models for viral annotation. We describe the need for new approaches to annotate viral sequences in metagenomes, the fundamentals of what protein language models are and how one can use them for sequence annotation, the strengths and weaknesses of these models, and future directions toward developing better models for viral annotation more broadly.

[Source](https://doi.org/10.1128/mbio.03206-23)

10.1128/mbio.03206-23

---

## Sampling the Maize (<em>Zea mays</em>) Leaf Microbiome

[Source](https://doi.org/10.1101/pdb.prot108581)

---

## Sampling Root-Associated Microbiome Communities of Maize (<em>Zea mays</em>)

[Source](https://doi.org/10.1101/pdb.prot108580)

---

## Sampling and Analysis of the Maize Microbiome

[Source](https://doi.org/10.1101/pdb.top108463)

---

## Manipulating the Maize (<em>Zea mays</em>) Microbiome

[Source](https://doi.org/10.1101/pdb.prot108584)

---

