## Fecal Microbiota Nutrient Utilization Potential Suggests Mucins as Drivers for Initial Gut Colonization of Mother-Child Shared Bacteria.
 16.01.2021, _M Nilsen, A Lokmic, IL Angell, KCL Carlsen, KH Carlsen, G Haugen, G Hedlin, CM Jonassen, BJ Marsland, B Nordlund, EM Rehbinder, CM Saunders, HO Skjerven, L Snipen, AC Staff, C Söderhäll, R Vettukattil, K Rudi_


The nutritional drivers for mother-child sharing of bacteria, and corresponding longitudinal trajectory of the infant gut microbiota development are not yet completely settled. We therefore aimed to characterize the mother-child sharing and the inferred nutritional utilization potential for the gut microbiota from a large unselected cohort. We analyzed in depth gut microbiota in 100 mother-child pairs enrolled antenatally from the general population-based PreventADALL cohort. Fecal samples collected at gestational week 18 for mothers and at birth (meconium), 3, 6, and 12 months for infants were analyzed by reduced metagenome sequencing to determine metagenome size and taxonomic composition. The nutrient utilization potential was determined based on the Virtual Metabolic Human (VMH, www.vmh.life) database. The estimated median metagenome size was ∼150 million base-pairs (bp) for mothers and ∼20 million bp at birth for the children. Longitudinal analyses revealed mother-child sharing (P &lt; 0.05, chi-square test) from birth up to 6 months for 3 prevalent Bacteroides species (prevalence &gt; 25% for all age groups). In multivariate ANOVA, the mother-child shared Bacteroides were associated with vaginal delivery (1.7% explained variance, P = 0.0001). Both vaginal delivery and mother-child sharing were associated with host derived mucins as nutrient sources. The age-related increase in metagenome size corresponded to an increased diversity in nutrient utilization, with dietary polysaccharides as the main age-related factor. Our results support host derived mucins as potential selection means for mother-child sharing of initial colonizers, while the age-related increase in diversity being associated with dietary polysaccharides.IMPORTANCE The initial bacterial colonization of human infants is crucial for lifelong health. Understanding factors driving this colonization will therefore be of high importance. Here we have used a novel high taxonomic resolution approach to deduce the nutrient utilization potential of the infant gut microbiota in a large longitudinal mother-child cohort. We found mucins as potential selection means for the initial colonization of mother-child shared bacteria, while the transition to a more adult-like microbiota was associated with dietary polysaccharide utilization potential. This knowledge will be important for future understanding of the importance of diet in shaping the gut microbiota composition and development during infancy.

[Source](https://aem.asm.org/content/early/2021/01/12/AEM.02201-20.long)

10.1128/AEM.02201-20

---

## Metagenomic insights of the infant microbiome community structure and function across multiple sites in the United States.
 27.01.2021, _G Casaburi, RM Duar, H Brown, RD Mitchell, S Kazi, S Chew, O Cagney, RL Flannery, KG Sylvester, SA Frese, BM Henrick, SL Freeman_


The gut microbiome plays an important role in early life, protecting newborns from enteric pathogens, promoting immune system development and providing key functions to the infant host. Currently, there are limited data to broadly assess the status of the US healthy infant gut microbiome. To address this gap, we performed a multi-state metagenomic survey and found high levels of bacteria associated with enteric inflammation (e.g. Escherichia, Klebsiella), antibiotic resistance genes, and signatures of dysbiosis, independent of location, age, and diet. Bifidobacterium were less abundant than generally expected and the species identified, including B. breve, B. longum and B. bifidum, had limited genetic capacity to metabolize human milk oligosaccharides (HMOs), while B. infantis strains with a complete capacity for HMOs utilization were found to be exceptionally rare. Considering microbiome composition and functional capacity, this survey revealed a previously unappreciated dysbiosis that is widespread in the contemporary US infant gut microbiome.

[Source](https://doi.org/10.1038/s41598-020-80583-9)

10.1038/s41598-020-80583-9

---

## Gut bacteria-derived 5-hydroxyindole is a potent stimulant of intestinal motility via its action on L-type calcium channels.
 22.01.2021, _B Waclawiková, A Bullock, M Schwalbe, C Aranzamendi, SA Nelemans, G van Dijk, S El Aidy_


Microbial conversion of dietary or drug substrates into small bioactive molecules represents a regulatory mechanism by which the gut microbiota alters intestinal physiology. Here, we show that a wide variety of gut bacteria can metabolize the dietary supplement and antidepressant 5-hydroxytryptophan (5-HTP) to 5-hydroxyindole (5-HI) via the tryptophanase (TnaA) enzyme. Oral administration of 5-HTP results in detection of 5-HI in fecal samples of healthy volunteers with interindividual variation. The production of 5-HI is inhibited upon pH reduction in in vitro studies. When administered orally in rats, 5-HI significantly accelerates the total gut transit time (TGTT). Deciphering the underlying mechanisms of action reveals that 5-HI accelerates gut contractility via activation of L-type calcium channels located on the colonic smooth muscle cells. Moreover, 5-HI stimulation of a cell line model of intestinal enterochromaffin cells results in significant increase in serotonin production. Together, our findings support a role for bacterial metabolism in altering gut motility and lay the foundation for microbiota-targeted interventions.

[Source](https://doi.org/10.1371/journal.pbio.3001070)

10.1371/journal.pbio.3001070

---

## Gut CD4+ T cell phenotypes are a continuum molded by microbes, not by TH archetypes

[Source](https://doi.org/10.1038/s41590-020-00836-7)

---

## Regulation of Gut Microbiota Disrupts the Glucocorticoid Receptor Pathway and Inflammation-related Pathways in the Mouse Hippocampus.
 19.01.2021, _X Rao, L Liu, H Wang, Y Yu, W Li, T Chai, W Zhou, P Ji, J Song, H Wei, P Xie_


An increasing number of studies have recently indicated the important effects of gut microbes on various functions of the central nervous system. However, the underlying mechanisms by which gut microbiota regulate brain functions and behavioral phenotypes remain largely unknown. We therefore used isobaric tags for relative and absolute quantitation (iTRAQ)-based quantitative proteomic analysis to obtain proteomic profiles of the hippocampus in germ-free (GF), colonized GF, and specific pathogen-free (SPF) mice. We then integrated the resulting proteomic data with previously reported mRNA microarray data, to further explore the effects of gut microbes on host brain functions. We identified that 61 proteins were upregulated and 242 proteins were downregulated in GF mice compared with SPF mice. Of these, 124 proteins were significantly restored following gut microbiota colonization. Bioinformatic analysis of these significant proteins indicated that the glucocorticoid receptor signaling pathway and inflammation-related pathways were the most enriched disrupted pathways. This study provides new insights into the pathological mechanisms of gut microbiota-regulated diseases.

[Source](https://doi.org/10.5607/en20055)

10.5607/en20055

---

## Decreased Ecological Resistance of the Gut Microbiota in Response to Clindamycin Challenge in Mice Colonized with the Fungus Candida albicans.
 21.01.2021, _L Markey, A Pugliese, T Tian, F Roy, K Lee, CA Kumamoto_


The mammalian gut microbiota is a complex community of microorganisms which typically exhibits remarkable stability. As the gut microbiota has been shown to affect many aspects of host health, the molecular keys to developing and maintaining a "healthy" gut microbiota are highly sought after. Yet, the qualities that define a microbiota as healthy remain elusive. We used the ability to resist change in response to antibiotic disruption, a quality we refer to as ecological resistance, as a metric for the health of the bacterial microbiota. Using a mouse model, we found that colonization with the commensal fungus Candida albicans decreased the ecological resistance of the bacterial microbiota in response to the antibiotic clindamycin such that increased microbiota disruption was observed in C. albicans-colonized mice compared to that in uncolonized mice. C. albicans colonization resulted in decreased alpha diversity and small changes in abundance of bacterial genera prior to clindamycin challenge. Strikingly, co-occurrence network analysis demonstrated that C. albicans colonization resulted in sweeping changes to the co-occurrence network structure, including decreased modularity and centrality and increased density. Thus, C. albicans colonization resulted in changes to the bacterial microbiota community and reduced its ecological resistance.IMPORTANCECandida albicans is the most common fungal member of the human gut microbiota, yet its ability to interact with and affect the bacterial gut microbiota is largely uncharacterized. Previous reports showed limited changes in microbiota composition as defined by bacterial species abundance as a consequence of C. albicans colonization. We also observed only a few bacterial genera that were significantly altered in abundance in C. albicans-colonized mice; however, C. albicans colonization significantly changed the structure of the bacterial microbiota co-occurrence network. Additionally, C. albicans colonization changed the response of the bacterial microbiota ecosystem to a clinically relevant perturbation, challenge with the antibiotic clindamycin.

[Source](https://msphere.asm.org/content/6/1/e00982-20)

10.1128/mSphere.00982-20

---

## Gut Commensal Segmented Filamentous Bacteria Fine-Tune T Follicular Regulatory Cells to Modify the Severity of Systemic Autoimmune Arthritis

[Source](https://doi.org/10.4049/jimmunol.2000663)

---

## Microbial metabolism of L-tyrosine protects against allergic airway inflammation.
 26.01.2021, _TP Wypych, C Pattaroni, O Perdijk, C Yap, A Trompette, D Anderson, DJ Creek, NL Harris, BJ Marsland_


The constituents of the gut microbiome are determined by the local habitat, which itself is shaped by immunological pressures, such as mucosal IgA. Using a mouse model of restricted antibody repertoire, we identified a role for antibody-microbe interactions in shaping a community of bacteria with an enhanced capacity to metabolize L-tyrosine. This model led to increased concentrations of p-cresol sulfate (PCS), which protected the host against allergic airway inflammation. PCS selectively reduced CCL20 production by airway epithelial cells due to an uncoupling of epidermal growth factor receptor (EGFR) and Toll-like receptor 4 (TLR4) signaling. Together, these data reveal a gut microbe-derived metabolite pathway that acts distally on the airway epithelium to reduce allergic airway responses, such as those underpinning asthma.

[Source](https://doi.org/10.1038/s41590-020-00856-3)

10.1038/s41590-020-00856-3

---

## Host development overwhelms environmental dispersal in governing the ecological succession of zebrafish gut microbiota.
 20.01.2021, _F Xiao, W Zhu, Y Yu, Z He, B Wu, C Wang, L Shu, X Li, H Yin, J Wang, P Juneau, X Zheng, Y Wu, J Li, X Chen, D Hou, Z Huang, J He, G Xu, L Xie, J Huang, Q Yan_


Clarifying mechanisms underlying the ecological succession of gut microbiota is a central theme of gut ecology. Under experimental manipulations of zebrafish hatching and rearing environments, we test our core hypothesis that the host development will overwhelm environmental dispersal in governing fish gut microbial community succession due to host genetics, immunology, and gut nutrient niches. We find that zebrafish developmental stage substantially explains the gut microbial community succession, whereas the environmental effects do not significantly affect the gut microbiota succession from larvae to adult fish. The gut microbiotas of zebrafish are clearly separated according to fish developmental stages, and the degree of homogeneous selection governing gut microbiota succession is increasing with host development. This study advances our mechanistic understanding of the gut microbiota assembly and succession by integrating the host and environmental effects, which also provides new insights into the gut ecology of other aquatic animals.

[Source](https://doi.org/10.1038/s41522-020-00176-2)

10.1038/s41522-020-00176-2

---

## Quantifying fluorescent glycan uptake to elucidate strain-level variability in foraging behaviors of rumen bacteria.
 27.01.2021, _L Klassen, G Reintjes, JP Tingley, DR Jones, JH Hehemann, AD Smith, TD Schwinghamer, C Arnosti, L Jin, TW Alexander, C Amundsen, D Thomas, R Amann, TA McAllister, DW Abbott_


Gut microbiomes, such as the microbial community that colonizes the rumen, have vast catabolic potential and play a vital role in host health and nutrition. By expanding our understanding of metabolic pathways in these ecosystems, we will garner foundational information for manipulating microbiome structure and function to influence host physiology. Currently, our knowledge of metabolic pathways relies heavily on inferences derived from metagenomics or culturing bacteria in vitro. However, novel approaches targeting specific cell physiologies can illuminate the functional potential encoded within microbial (meta)genomes to provide accurate assessments of metabolic abilities. Using fluorescently labeled polysaccharides, we visualized carbohydrate metabolism performed by single bacterial cells in a complex rumen sample, enabling a rapid assessment of their metabolic phenotype. Specifically, we identified bovine-adapted strains of Bacteroides thetaiotaomicron that metabolized yeast mannan in the rumen microbiome ex vivo and discerned the mechanistic differences between two distinct carbohydrate foraging behaviors, referred to as "medium grower" and "high grower." Using comparative whole-genome sequencing, RNA-seq, and carbohydrate-active enzyme fingerprinting, we could elucidate the strain-level variability in carbohydrate utilization systems of the two foraging behaviors to help predict individual strategies of nutrient acquisition. Here, we present a multi-faceted study using complimentary next-generation physiology and "omics" approaches to characterize microbial adaptation to a prebiotic in the rumen ecosystem. Video abstract.

[Source](https://doi.org/10.1186/s40168-020-00975-x)

10.1186/s40168-020-00975-x

---

## Complementary Roles of Wood-Inhabiting Fungi and Bacteria Facilitate Deadwood Decomposition.
 13.01.2021, _V Tláskal, V Brabcová, T Větrovský, M Jomura, R López-Mondéjar, LM Oliveira Monteiro, JP Saraiva, ZR Human, T Cajthaml, U Nunes da Rocha, P Baldrian_


Forests accumulate and store large amounts of carbon (C), and a substantial fraction of this stock is contained in deadwood. This transient pool is subject to decomposition by deadwood-associated organisms, and in this process it contributes to CO2 emissions. Although fungi and bacteria are known to colonize deadwood, little is known about the microbial processes that mediate carbon and nitrogen (N) cycling in deadwood. In this study, using a combination of metagenomics, metatranscriptomics, and nutrient flux measurements, we demonstrate that the decomposition of deadwood reflects the complementary roles played by fungi and bacteria. Fungi were found to dominate the decomposition of deadwood and particularly its recalcitrant fractions, while several bacterial taxa participate in N accumulation in deadwood through N fixation, being dependent on fungal activity with respect to deadwood colonization and C supply. Conversely, bacterial N fixation helps to decrease the constraints of deadwood decomposition for fungi. Both the CO2 efflux and N accumulation that are a result of a joint action of deadwood bacteria and fungi may be significant for nutrient cycling at ecosystem levels. Especially in boreal forests with low N stocks, deadwood retention may help to improve the nutritional status and fertility of soils.IMPORTANCE Wood represents a globally important stock of C, and its mineralization importantly contributes to the global C cycle. Microorganisms play a key role in deadwood decomposition, since they possess enzymatic tools for the degradation of recalcitrant plant polymers. The present paradigm is that fungi accomplish degradation while commensalist bacteria exploit the products of fungal extracellular enzymatic cleavage, but this assumption was never backed by the analysis of microbial roles in deadwood. This study clearly identifies the roles of fungi and bacteria in the microbiome and demonstrates the importance of bacteria and their N fixation for the nutrient balance in deadwood as well as fluxes at the ecosystem level. Deadwood decomposition is shown as a process where fungi and bacteria play defined, complementary roles.

[Source](https://msystems.asm.org/content/6/1/e01078-20)

10.1128/mSystems.01078-20

---

## Natural Bacterial Assemblages in Arabidopsis thaliana Tissues Become More Distinguishable and Diverse during Host Development.
 20.01.2021, _K Beilsmith, M Perisin, J Bergelson_


To study the spatial and temporal dynamics of bacterial colonization under field conditions, we planted and sampled Arabidopsis thaliana during 2 years at two Michigan sites and surveyed colonists by sequencing 16S rRNA gene amplicons. Mosaic and dynamic assemblages revealed the plant as a patchwork of tissue habitats that differentiated with age. Although assemblages primarily varied between roots and shoots, amplicon sequence variants (ASVs) also differentiated phyllosphere tissues. Increasing assemblage diversity indicated that variants dispersed more widely over time, decreasing the importance of stochastic variation in early colonization relative to tissue differences. As tissues underwent developmental transitions, the root and phyllosphere assemblages became more distinct. This pattern was driven by common variants rather than those restricted to a particular tissue or transiently present at one developmental stage. Patterns also depended critically on fine phylogenetic resolution: when ASVs were grouped at coarse taxonomic levels, their associations with host tissue and age weakened. Thus, the observed spatial and temporal variation in colonization depended upon bacterial traits that were not broadly shared at the family level. Some colonists were consistently more successful at entering specific tissues, as evidenced by their repeatable spatial prevalence distributions across sites and years. However, these variants did not overtake plant assemblages, which instead became more even over time. Together, these results suggested that the increasing effect of tissue type was related to colonization bottlenecks for specific ASVs rather than to their ability to dominate other colonists once established.IMPORTANCE Developing synthetic microbial communities that can increase plant yield or deter pathogens requires basic research on several fronts, including the efficiency with which microbes colonize plant tissues, how plant genes shape the microbiome, and the microbe-microbe interactions involved in community assembly. Findings on each of these fronts depend upon the spatial and temporal scales at which plant microbiomes are surveyed. In our study, phyllosphere tissues housed increasingly distinct microbial assemblages as plants aged, indicating that plants can be considered collections of tissue habitats in which microbial colonists-natural or synthetic-are established with differing success. Relationships between host genes and community diversity might vary depending on when samples are collected, given that assemblages grew more diverse as plants aged. Both spatial and temporal trends weakened when colonists were grouped by family, suggesting that functional rather than taxonomic profiling will be necessary to understand the basis for differences in colonization success.

[Source](https://mbio.asm.org/content/12/1/e02723-20)

10.1128/mBio.02723-20

---

## Colonization of naïve roots from Populus tremula x alba involves successive waves of fungi and bacteria with different trophic abilities.
 16.01.2021, _F Fracchia, L Mangeot-Peter, L Jacquot, F Martin, C Veneault-Fourrey, A Deveau_


Through their roots, trees interact with a highly complex community of microorganisms belonging to various trophic guilds and contributing to tree nutrition, development and protection against stresses. Tree roots select for specific microbial species from the bulk soil communities. The root microbiome formation is a dynamic process but little is known on how the different microorganisms colonize the roots and how the selection occurs. To decipher if the final composition of the root microbiome is the product of several waves of colonization by different guilds of microorganisms, we planted sterile rooted cuttings of Gray Poplar obtained from plantlets propagated in axenic conditions in natural poplar stand-soil. We analyzed the root microbiome at different time points between 2 and 50 days of culture by combining high-throughput Illumina MiSeq sequencing of fungal rDNA ITS and bacterial 16S rRNA amplicons with Confocal Laser Scanning Microscope observations. The microbial colonisation of poplar roots took place in three stages but the dynamic was different between bacteria and fungi. Root bacterial communities were clearly different from the soil after two days of culture. By contrast, if fungi were also already colonizing roots after two days, the initial communities were very close to the one of the soil and were dominated by saprotrophs. Those were slowly replaced by endophytes and ectomycorhizal fungi. The replacement of the most abundant fungal and bacterial community members observed in poplar roots along time suggest potential competition effect between microorganisms and/or a selection by the host.IMPORTANCE The tree root microbiome is composed of a very diverse set of bacterial and fungal communities. These microorganisms have a profound impact on tree growth, development and protection against different types of stress. They mainly originate from the bulk soil and colonize the root system which provides a unique nutrient rich-environment for a diverse assemblage of microbial communities. In order to better understand how the tree root microbiome is shaped along time, we observed the composition of root-associated microbial communities of naïve plantlets of poplar transferred in natural soil. The composition of the final root microbiome rely on a series of colonization stages characterized by the dominance of different fungal guilds and bacterial community members along time. Our observations suggest an early stabilization of bacterial communities, whereas fungal communities are established following a more gradual pattern.

[Source](https://aem.asm.org/content/early/2021/01/12/AEM.02541-20.long)

10.1128/AEM.02541-20

---

## The role of host promiscuity in the invasion process of a seaweed holobiont.
 22.01.2021, _G Bonthond, T Bayer, SA Krueger-Hadfield, N Stärck, G Wang, M Nakaoka, S Künzel, F Weinberger_


Invasive species are co-introduced with microbiota from their native range and also interact with microbiota found in the novel environment to which they are introduced. Host flexibility toward microbiota, or host promiscuity, is an important trait underlying terrestrial plant invasions. To test whether host promiscuity may be important in macroalgal invasions, we experimentally simulated an invasion in a common garden setting, using the widespread invasive macroalga Agarophyton vermiculophyllum as a model invasive seaweed holobiont. After disturbing the microbiota of individuals from native and non-native populations with antibiotics, we monitored the microbial succession trajectories in the presence of a new source of microbes. Microbial communities were strongly impacted by the treatment and changed compositionally and in terms of diversity but recovered functionally by the end of the experiment in most respects. Beta-diversity in disturbed holobionts strongly decreased, indicating that different populations configure more similar -or more common- microbial communities when exposed to the same conditions. This decline in beta-diversity occurred not only more rapidly, but was also more pronounced in non-native populations, while individuals from native populations retained communities more similar to those observed in the field. This study demonstrates that microbial communities of non-native A. vermiculophyllum are more flexibly adjusted to the environment and suggests that an intraspecific increase in host promiscuity has promoted the invasion process of A. vermiculophyllum. This phenomenon may be important among invasive macroalgal holobionts in general.

[Source](https://doi.org/10.1038/s41396-020-00878-7)

10.1038/s41396-020-00878-7

---

## Sugarcane/peanut intercropping system improves the soil quality and increases the abundance of beneficial microbes.
 15.01.2021, _X Tang, J Jiang, Z Huang, H Wu, J Wang, L He, F Xiong, R Zhong, J Liu, Z Han, R Tang, L He_


Sugarcane/peanut intercropping is a highly efficient planting pattern in South China. However, the effects of sugarcane/peanut intercropping on soil quality need to be clarified. This study characterized the soil microbial community and the soil quality in sugarcane/peanut intercropping systems by the Illumina MiSeq platform. The results showed that the intercropping sugarcane (IS) system significantly increased the total N (TN), available N (AN), available P (AP), pH value, and acid phosphatase activity (ACP), but it had little effect on the total P (TP), total K (TK), available K (AK), organic matter (OM), urease activity, protease activity, catalase activity, and sucrase activity, compared with those in monocropping sugarcane (MS) and monocropping peanut (MP) systems. Both intercropping peanut (IP) and IS soils contained more bacteria and fungi than soils in the MP and MS fields, and the microbes identified were mainly Chloroflexi and Acidobacteria, respectively. Intercropping significantly increased the number of unique microbes in IS soils (68 genera), compared with the numbers in the IP (14), MS (17), and MP (16) systems. The redundancy analysis revealed that the abundances of culturable Acidobacteriaceae subgroup 1, nonculturable DA111, and culturable Acidobacteria were positively correlated with the measured soil quality in the intercropping system. Furthermore, the sugarcane/peanut intercropping significantly increased the economic benefit by 87.84% and 36.38%, as compared with that of the MP and MS, respectively. These results suggest that peanut and sugarcane intercropping increases the available N and P content by increasing the abundance of rhizospheric microbes, especially Acidobacteriaceae subgroup 1, DA111, and Acidobacteria.

[Source](https://doi.org/10.1002/jobm.202000750)

10.1002/jobm.202000750

---

## Variation of rhizosphere microbial community in continuous mono-maize seed production.
 22.01.2021, _Y Zhao, W Fu, C Hu, G Chen, Z Xiao, Y Chen, Z Wang, H Cheng_


Soil microbe is crucial to a healthy soil, therefore its diversities and abundances under different conditions are still need fully understand.The aims of the study were to characterize the community structure and diversity of microbe in the rhizosphere soil after continuous maize seed production, and the relationship between the disease incidence of four diseases and the variation of the rhizosphere microbe. The results showed that different fungal and bacterial species were predominant in different cropping year, and long-term maize seed production had a huge impact on structure and diversity of soil microbial. Ascomycota and Mortierellomycota were the dominant fungal phyla and Mortierella and Ascomycetes represented for a large proportion of genus. A relative increase of Fusarium and Gibberella and a relative decrease of Mortierella, Chrysosporium, Podospora, and Chaetomium were observed with the increase of cropping year. Pathogenic Fusarium, Curvularia, Curvularia-lunata, Cladosporium, Gibberella-baccata, and Plectosphaerellaceae were over-presented and varied at different continuous cropping year, led to different maize disease incidence. Proteobacteria and Actinobacteria ranked in the top two of all bacterial phyla, and genus Pseudarthrobacter, Roseiflexus and RB41 dominated top 3. Haliangium and Streptomyces decreased with the continuous cropping year and mono-cropping of maize seed production increased disease incidence with the increase of cropping year, while the major disease was different. Continuous cropping of maize seed production induced the decrease of protective microbe and biocontrol genera, while pathogenic pathogen increased, and maize are in danger of pathogen invasion. Field management show great effects on soil microbial community.

[Source](https://doi.org/10.1038/s41598-021-81228-1)

10.1038/s41598-021-81228-1

---

## The diversity and function of sourdough starter microbiomes.
 26.01.2021, _EA Landis, AM Oliverio, EA McKenney, LM Nichols, N Kfoury, M Biango-Daniels, LK Shell, AA Madden, L Shapiro, S Sakunala, K Drake, A Robbat, M Booker, RR Dunn, N Fierer, BE Wolfe_


Humans have relied on sourdough starter microbial communities to make leavened bread for thousands of years, but only a small fraction of global sourdough biodiversity has been characterized. Working with a community-scientist network of bread bakers, we determined the microbial diversity of 500 sourdough starters from four continents. In sharp contrast with widespread assumptions, we found little evidence for biogeographic patterns in starter communities. Strong co-occurrence patterns observed in situ and recreated in vitro demonstrate that microbial interactions shape sourdough community structure. Variation in dough rise rates and aromas were largely explained by acetic acid bacteria, a mostly overlooked group of sourdough microbes. Our study reveals the extent of microbial diversity in an ancient fermented food across diverse cultural and geographic backgrounds.

[Source](https://elifesciences.org/articles/61644)

10.7554/eLife.61644

---

## Optimization of probiotic therapeutics using machine learning in an artificial human gastrointestinal tract

[Source](https://doi.org/10.1038/s41598-020-79947-y)

---

## How reliably do metagenome-assembled genomes (MAGs) represent natural populations? Insights from comparing MAGs against isolate genomes derived from the same fecal sample.
 16.01.2021, _A Meziti, LM Rodriguez-R, JK Hatt, A Peña-Gonzalez, K Levy, KT Konstantinidis_


The recovery of metagenome-assembled genomes (MAGs) from metagenomic data has recently become a common task for microbial studies. The strengths and limitations of the underlying bioinformatics algorithms are well appreciated by now based on performance tests with mock datasets of known composition. However, these mock datasets do not capture the complexity and diversity often observed within natural populations, since their construction typically relies on only a single genome of a given organism. Further, it remains unclear if MAGs can recover population variable (e.g., shared by &gt;10% but &lt;90% of the members of the population) as efficiently as core genes (e.g., shared by &gt;90% of the members). To address these issues, we compared the gene variability of pathogenic Escherichia coli isolates from eight diarrheal samples, for which the isolate was the causative agent, against their corresponding MAGs recovered from the companion metagenomic dataset. Our analysis revealed that MAGs with completeness estimates near 95% captured only 77% of the population core genes and 50% of the variable genes, on average. Further, about 5% of the genes of these MAG were conservatively identified as missing in the isolate and were of different (non-Enterobacteriaceae) taxonomic origin, suggesting errors at the genome binning step, even though contamination estimates based on commonly used pipelines were only 1.5%. Therefore, the quality of MAGs may oftentimes be worse than estimated, and we offer examples of how to recognize and improve such MAGs to sufficient quality by -for instance- employing only contigs longer than 1,000bp for binning.IMPORTANCE Metagenome assembly and recovery of metagenome-assembled genomes (MAGs) have recently become common tasks for microbiome studies across environmental and clinical settings. However, to what extent MAGs can capture the genes of the population they represent remains speculative. Current approaches to evaluate MAG quality are limited to the recovery and copy number of universal, housekeeping genes but these genes represent a small fraction of the total genome, leaving the majority of the genome essentially inaccessible. If MAG quality in reality is lower than these approaches would estimate, this could have dramatic consequences for all downstream analyses and interpretations. In this study, we evaluated this issue using a novel approach that employs comparisons of MAGs to isolate genomes derived from the same samples. Further, our samples originated from a diarrhea case-control study, and thus our results are relevant for recovering the virulence factors of pathogens from metagenomic datasets.

[Source](https://aem.asm.org/content/early/2021/01/12/AEM.02593-20)

10.1128/AEM.02593-20

---

## Precision modification of the human gut microbiota targeting surface-associated proteins.
 22.01.2021, _R Marcos-Fernández, L Ruiz, A Blanco-Míguez, A Margolles, B Sánchez_


This work describes a new procedure that allows the targeted modification of the human gut microbiota by using antibodies raised against bacterial surface-associated proteins specific to the microorganism of interest. To this end, a polyclonal antibody recognising the surface-associated protein Surface Layer Protein A of Lactobacillus acidophilus DSM20079T was developed. By conjugating this antibody with fluorescent probes and magnetic particles, we were able to specifically identify this bacterium both in a synthetic, and in real gut microbiotas by means of a flow cytometry approach. Further, we demonstrated the applicability of this antibody to deplete complex human gut microbiotas from L. acidophilus in a single step. L. acidophilus was found to interact with other bacteria both in synthetic and in real microbiotas, as reflected by its concomitant depletion together with other species. Further optimization of the procedure including a trypsin step enabled to achieve the selective and complete isolation of this species. Depleting a single species from a gut microbiota, using antibodies recognizing specific cell surface elements of the target organism, will open up novel ways to tackle research on the specific immunomodulatory and metabolic contributions of a bacterium of interest in the context of a complex human gut microbiota, including the investigation into therapeutic applications by adding/depleting a key bacterium. This represents the first work in which an antibody/flow-cytometry based application enabled the targeted edition of human gut microbiotas, and represents the basis for the design of precision microbiome-based therapies.

[Source](https://doi.org/10.1038/s41598-020-80187-3)

10.1038/s41598-020-80187-3

---

## High-throughput cultivation and identification of bacteria from the plant root microbiota.
 14.01.2021, _J Zhang, YX Liu, X Guo, Y Qin, R Garrido-Oter, P Schulze-Lefert, Y Bai_


Cultivating native bacteria from roots of plants grown in a given environment is essential for dissecting the functions of the root microbiota for plant growth and health with strain-specific resolution. In this study, we established a straightforward protocol for high-throughput bacterial isolation from fresh root samples using limiting dilution to ensure that most cultured bacteria originated from only one microorganism. This is followed by strain characterization using a two-sided barcode polymerase chain reaction system to identify pure and heterogeneous bacterial cultures. Our approach overcomes multiple difficulties of traditional bacterial isolation and identification methods, such as obtaining bacteria with diverse growth rates while greatly increasing throughput. To facilitate data processing, we developed an easy-to-use bioinformatic pipeline called 'Culturome' ( https://github.com/YongxinLiu/Culturome ) and a graphical user interface web server ( http://bailab.genetics.ac.cn/culturome/ ). This protocol allows any research group (two or three lab members without expertise in bioinformatics) to systematically cultivate root-associated bacteria within 8-9 weeks.

[Source](https://doi.org/10.1038/s41596-020-00444-7)

10.1038/s41596-020-00444-7

---

