## The effects of antibiotic exposures on the gut resistome during hematopoietic cell transplantation in children.
 02.04.2024, _SM Heston, RR Young, K Jenkins, PL Martin, A Stokhuyzen, DV Ward, SK Bhattarai, V Bucci, M Arshad, NJ Chao, PC Seed, MS Kelly_


_Child, Humans, Anti-Bacterial Agents, Gastrointestinal Microbiome, Bacteria, Fluoroquinolones, Hematopoietic Stem Cell Transplantation_
Antibiotic resistance is a global threat driven primarily by antibiotic use. We evaluated the effects of antibiotic exposures on the gut microbiomes and resistomes of children at high risk of colonization by antibiotic-resistant bacteria. We performed shotgun metagenomic sequencing of 691 serially collected fecal samples from 80 children (&lt;18 years) undergoing hematopoietic cell transplantation. We evaluated the effects of aerobic (cefepime, vancomycin, fluoroquinolones, aminoglycosides, macrolides, and trimethoprim-sulfamethoxazole) and anaerobic (piperacillin-tazobactam, carbapenems, metronidazole, and clindamycin) antibiotic exposures on the diversity and composition of the gut microbiome and resistome. We identified 372 unique antibiotic resistance genes (ARGs); the most frequent ARGs identified encode resistance to tetracyclines (n = 88), beta-lactams (n = 84), and fluoroquinolones (n = 79). Both aerobic and anaerobic antibiotic exposures were associated with a decrease in the number of bacterial species (aerobic, β = 0.71, 95% CI: 0.64, 0.79; anaerobic, β = 0.66, 95% CI: 0.53, 0.82) and the number of unique ARGs (aerobic, β = 0.81, 95% CI: 0.74, 0.90; anaerobic, β = 0.73, 95% CI: 0.61, 0.88) within the gut metagenome. However, only antibiotic regimens that included anaerobic activity were associated with an increase in acquisition of new ARGs (anaerobic, β = 1.50; 95% CI: 1.12, 2.01) and an increase in the relative abundance of ARGs in the gut resistome (anaerobic, β = 1.62; 95% CI: 1.15, 2.27). Specific antibiotic exposures were associated with distinct changes in the number and abundance of ARGs for individual antibiotic classes. Our findings detail the impact of antibiotics on the gut microbiome and resistome and demonstrate that anaerobic antibiotics are particularly likely to promote acquisition and expansion of antibiotic-resistant bacteria.

[Source](https://doi.org/10.1080/19490976.2024.2333748)

10.1080/19490976.2024.2333748

---

## Clinically used broad-spectrum antibiotics compromise inflammatory monocyte-dependent antibacterial defense in the lung.
 02.04.2024, _PJ Dörner, H Anandakumar, I Röwekamp, F Fiocca Vernengo, B Millet Pascual-Leone, M Krzanowski, J Sellmaier, U Brüning, R Fritsche-Guenther, L Pfannkuch, F Kurth, M Milek, V Igbokwe, U Löber, B Gutbier, M Holstein, GA Heinz, MF Mashreghi, LN Schulte, AB Klatt, S Caesar, SM Wienhold, S Offermanns, M Mack, M Witzenrath, S Jordan, D Beule, JA Kirwan, SK Forslund, N Wilck, H Bartolomaeus, MM Heimesaat, B Opitz_


_Humans, Mice, Animals, Anti-Bacterial Agents, Monocytes, Anti-Infective Agents, Klebsiella pneumoniae, Lung_
Hospital-acquired pneumonia (HAP) is associated with high mortality and costs, and frequently caused by multidrug-resistant (MDR) bacteria. Although prior antimicrobial therapy is a major risk factor for HAP, the underlying mechanism remains incompletely understood. Here, we demonstrate that antibiotic therapy in hospitalized patients is associated with decreased diversity of the gut microbiome and depletion of short-chain fatty acid (SCFA) producers. Infection experiments with mice transplanted with patient fecal material reveal that these antibiotic-induced microbiota perturbations impair pulmonary defense against MDR Klebsiella pneumoniae. This is dependent on inflammatory monocytes (IMs), whose fatty acid receptor (FFAR)2/3-controlled and phagolysosome-dependent antibacterial activity is compromized in mice transplanted with antibiotic-associated patient microbiota. Collectively, we characterize how clinically relevant antibiotics affect antimicrobial defense in the context of human microbiota, and reveal a critical impairment of IM´s antimicrobial activity. Our study provides additional arguments for the rational use of antibiotics and offers mechanistic insights for the development of novel prophylactic strategies to protect high-risk patients from HAP.

[Source](https://doi.org/10.1038/s41467-024-47149-z)

10.1038/s41467-024-47149-z

---

## The gut metabolite indole-3-propionic acid activates ERK1 to restore social function and hippocampal inhibitory synaptic transmission in a 16p11.2 microdeletion mouse model

[Source](https://doi.org/10.1186/s40168-024-01755-7)

---

## Vaginal microbiomes of breast cancer survivors treated with aromatase inhibitors with and without vulvovaginal symptoms.
 02.04.2024, _P Prasanchit, P Pongchaikul, P Lertsittichai, C Tantitham, J Manonai_


_Female, Humans, Breast Neoplasms, Cancer Survivors, Aromatase Inhibitors, Breast, Vagina, Microbiota, Menopause_
Genitourinary syndrome of menopause (GSM) is the leading cause of vaginal symptoms in breast cancer survivors treated with aromatase inhibitors. However, there are currently no effective treatment options available for women with a history of breast cancer. Recent research has established that changes in the vaginal microbiome may be linked to GSM. Most studies have assessed the microbiome without accounting for the estrogen status. It remains unknown whether the vaginal microbiome differ among patients with a low estrogenic state with and without vulvovaginal symptoms. To address such research questions, our study compares the vaginal microbiomes among breast cancer survivors treated with aromatase inhibitors with and without vulvovaginal symptoms. A total of 50 breast cancer survivors treated with aromatase inhibitors were recruited, among whom 25 had vulvovaginal symptoms and 25 had no vulvovaginal symptoms. Vaginal swabs were collected. DNA extraction, followed by sequencing of the V3-V4 regions of the 16S ribosomal RNA gene, were performed. Differential abundance analysis was conducted by linear discriminant analysis effect size. Taxonomy assignment, alpha diversity and beta diversity were examined. The relative abundance of genus Sneathia and genus Gardnerella was significantly increased in vulvovaginal symptoms group with no differences in bacterial diversity and richness.

[Source](https://doi.org/10.1038/s41598-024-58118-3)

10.1038/s41598-024-58118-3

---

## Integrated analysis of gut metabolome, microbiome, and brain function reveal the role of gut-brain axis in longevity.
 02.04.2024, _B Jiao, Z Ouyang, Q Liu, T Xu, M Wan, G Ma, L Zhou, J Guo, J Wang, B Tang, Z Zhao, L Shen_


_Aged, 80 and over, Humans, Brain-Gut Axis, Gastrointestinal Microbiome, Metabolome, Microbiota, Brain_
The role of microbiota-gut-brain axis in modulating longevity remains undetermined. Here, we performed a multiomics analysis of gut metagenomics, gut metabolomics, and brain functional near-infrared spectroscopy (fNIRS) in a cohort of 164 participants, including 83 nonagenarians (NAs) and 81 non-nonagenarians (NNAs) matched with their spouses and offspring. We found that 438 metabolites were significantly different between the two groups; among them, neuroactive compounds and anti-inflammatory substances were enriched in NAs. In addition, increased levels of neuroactive metabolites in NAs were significantly associated with NA-enriched species that had three corresponding biosynthetic potentials: Enterocloster asparagiformis, Hungatella hathewayi and Oxalobacter formigenes. Further analysis showed that the altered gut microbes and metabolites were linked to the enhanced brain connectivity in NAs, including the left dorsolateral prefrontal cortex (DLPFC)-left premotor cortex (PMC), left DLPFC-right primary motor area (M1), and right inferior frontal gyrus (IFG)-right M1. Finally, we found that neuroactive metabolites, altered microbe and enhanced brain connectivity contributed to the cognitive preservation in NAs. Our findings provide a comprehensive understanding of the microbiota-gut-brain axis in a long-lived population and insights into the establishment of a microbiome and metabolite homeostasis that can benefit human longevity and cognition by enhancing functional brain connectivity.

[Source](https://doi.org/10.1080/19490976.2024.2331434)

10.1080/19490976.2024.2331434

---

## Exploring the diversity of vaginal microbiota between healthy women and cervical cancer patients in India.
 29.03.2024, _A Kamble, S Naik, M Talathi, D Jadhav, M Sakharkar, J Yang, O Prakash, R Kaul-Ghanekar_


_Humans, Female, Uterine Cervical Neoplasms, RNA, Ribosomal, 16S, Vagina, India, Escherichia coli, Microbiota_
Introduction. Cervicovaginal diversity has been reported as a predictive biomarker for cervical cancer risk. We recently reported the bio-therapeutic potential of vaginal probiotics from healthy Indian women against vaginal pathogens, isolated from the invasive cervical cancer (ICC) patients.Gap Statement. The cervicovaginal microflora from cervical cancer patients has not yet been reported from Indian population.Aim. The present study aimed at comparing the cervicovaginal microbiome between healthy controls (HC) and ICC patients from the Indian population.Methodology. In total, 30 vaginal swabs (15 from HC and 15 from ICC) were subjected to 16S rRNA gene sequencing. Alpha diversity was evaluated by Shannon and Chao1 index; and beta diversity by principle coordinate analysis (PCoA) of weighted and unweighted UniFrac distances. The relative abundance of the microbial taxa was done according to linear discriminant analysis effect size (LEfSe).Results. Predominance of Staphylococcus spp. in ICC and Lactobacillus gasseri in HC groups was observed. Alpha-diversity was found to be higher in ICC as compared to HC but was statistically non-significant. LEfSe analysis revealed Bacteroides fragilis and Escherichia coli as the marker genera in ICC with a marked decrease in Lactobacillus sp. Contrarily, in HC, L. gasseri, L. iners and L. fermentum were found to be abundant.Conclusion. Differences in the vaginal microbiome between healthy and ICC women could help in the early prediction of cervical cancer risk and thus in designing prevention strategies.

[Source](https://doi.org/10.1099/jmm.0.001819)

10.1099/jmm.0.001819

---

## Oral pathobiont <em>Klebsiella</em> chaperon usher pili provide site-specific adaptation for the inflamed gut mucosa

[Source](https://doi.org/10.1080/19490976.2024.2333463)

---

## Hypervirulent <em>Klebsiella pneumoniae</em> employs genomic island encoded toxins against bacterial competitors in the gut

[Source](https://doi.org/10.1093/ismejo/wrae054)

---

## Microbiome bacterial influencers of host immunity and response to immunotherapy

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(24)00133-2)

---

## Unlocking the Potential of Probiotics: A Comprehensive Review on Research, Production, and Regulation of Probiotics

[Source](https://doi.org/10.1007/s12602-024-10247-x)

---

## Mulberry leaf and konjac flour compound dietary fiber improves digestion and metabolism in elderly mice with high-fish-protein diet by regulating gut microbiota structure and intestinal tissue repair.
 30.03.2024, _L Deng, G Zhong, Q Wang, Z Zhu, Y Peng_


Ensuring sufficient protein intake, efficient digestion, and optimal absorption are crucial for the elderly. This study aims to investigate the potential of a compound dietary fiber, consisting of mulberry leaf and konjac flour (CMK), to enhance the digestion and absorption of a high-fish-protein diet in elderly mice. Results showed that CMK effectively reduced the number of unique peptide segments, generated short-chain fatty acids (SCFA) in feces, improved the content of glutamic pyruvic transaminase (GPT), glutamic oxaloacetic transaminase (GOT), amino acid, and urea nitrogen in serum, activated the contents of pepsin, trypsin, and erepsin, and enhanced the expression of glutamate dehydrogenase (GDH), peptide transporter 1 (PepT1), and aminopeptidase N (APN). Furthermore, CMK demonstrated its ability to decrease the levels of tumor necrosis factor-α (TNF-α), interleukin-1β (IL-1β), interleukin-10 (IL-10), lipopolysaccharide (LPS), and lipopolysaccharide binding protein (LBP), while increase the abundance of beneficial bacteria, such as Lactobacillus and Blautia. In conclusion, CMK proved effective in enhancing the digestion and metabolism of protein in elderly mice through the regulation of gut microbiota structure and intestinal tissue repair.

[Source](https://doi.org/10.1016/j.fochx.2024.101295)

10.1016/j.fochx.2024.101295

---

## Gut Microbiota and Blood Metabolites Related to Fiber Intake and Type 2 Diabetes.
 01.04.2024, _Z Wang, BA Peters, B Yu, ML Grove, T Wang, X Xue, B Thyagarajan, ML Daviglus, E Boerwinkle, G Hu, Y Mossavar-Rahmani, CR Isasi, R Knight, RD Burk, RC Kaplan, Q Qi_


_Humans, Gastrointestinal Microbiome, Diabetes Mellitus, Type 2, Diet, Bacteria, Dietary Fiber_
Consistent evidence suggests diabetes-protective effects of dietary fiber intake. However, the underlying mechanisms, particularly the role of gut microbiota and host circulating metabolites, are not fully understood. We aimed to investigate gut microbiota and circulating metabolites associated with dietary fiber intake and their relationships with type 2 diabetes (T2D).

[Source](https://doi.org/10.1161/CIRCRESAHA.123.323634)

10.1161/CIRCRESAHA.123.323634

---

## Colitis reduces active social engagement in mice and is ameliorated by supplementation with human microbiota members.
 02.04.2024, _DG Brown, M Murphy, R Cadeddu, R Bell, A Weis, T Chiaro, K Klag, J Morgan, H Coon, WZ Stephens, M Bortolato, JL Round_


_Humans, Male, Mice, Animals, Autism Spectrum Disorder, Social Participation, Microbiota, Colitis, Gastrointestinal Diseases, Dietary Supplements_
Multiple neurological disorders are associated with gastrointestinal (GI) symptoms, including autism spectrum disorder (ASD). However, it is unclear whether GI distress itself can modify aspects of behavior. Here, we show that mice that experience repeated colitis have impaired active social engagement, as measured by interactions with a foreign mouse, even though signs of colitis were no longer present. We then tested the hypothesis that individuals with ASD harbor a microbiota that might differentially influence GI health by performing microbiota transplantation studies into male germfree animals, followed by induction of colitis. Animals that harbor a microbiota from ASD individuals have worsened gut phenotypes when compared to animals colonized with microbiotas from familial neurotypical (NT) controls. We identify the enrichment of Blautia species in all familial NT controls and observe an association between elevated abundance of Bacteroides uniformis and reductions in intestinal injury. Oral treatment with either of these microbes reduces colon injury in mice. Finally, provision of a Blautia isolate from a NT control ameliorates gut injury-associated active social engagement in mice. Collectively, our data demonstrate that past intestinal distress is associated with changes in active social behavior in mice that can be ameliorated by supplementation of members of the human microbiota.

[Source](https://doi.org/10.1038/s41467-024-46733-7)

10.1038/s41467-024-46733-7

---

## Disentangling direct vs indirect effects of microbiome manipulations in a habitat-forming marine holobiont

[Source](https://doi.org/10.1038/s41522-024-00503-x)

---

## Microplastics dampen the self-renewal of hematopoietic stem cells by disrupting the gut microbiota-hypoxanthine-Wnt axis.
 31.03.2024, _L Jiang, Y Ye, Y Han, Q Wang, H Lu, J Li, W Qian, X Zeng, Z Zhang, Y Zhao, J Shi, Y Luo, Y Qiu, J Sun, J Sheng, H Huang, P Qian_


Microplastics (MPs) are contaminants ubiquitously found in the global biosphere that enter the body through inhalation or ingestion, posing significant risks to human health. Recent studies emerge that MPs are present in the bone marrow and damage the hematopoietic system. However, it remains largely elusive about the specific mechanisms by which MPs affect hematopoietic stem cells (HSCs) and their clinical relevance in HSC transplantation (HSCT). Here, we established a long-term MPs intake mouse model and found that MPs caused severe damage to the hematopoietic system. Oral gavage administration of MPs or fecal transplantation of microbiota from MPs-treated mice markedly undermined the self-renewal and reconstitution capacities of HSCs. Mechanistically, MPs did not directly kill HSCs but disrupted gut structure and permeability, which eventually ameliorated the abundance of Rikenellaceae and hypoxanthine in the intestine and inactivated the HPRT-Wnt signaling in bone marrow HSCs. Furthermore, administration of Rikenellaceae or hypoxanthine in mice as well as treatment of WNT10A in the culture system substantially rescued the MPs-induced HSC defects. Finally, we validated in a cohort of human patients receiving allogenic HSCT from healthy donors, and revealed that the survival time of patients was negatively correlated with levels of MPs, while positively with the abundance of Rikenellaceae, and hypoxanthine in the HSC donors' feces and blood. Overall, our study unleashes the detrimental roles and mechanisms of MPs in HSCs, which provides potential strategies to prevent hematopoietic damage from MPs and serves as a fundamental critique for selecting suitable donors for HSCT in clinical practice.

[Source](https://doi.org/10.1038/s41421-024-00665-0)

10.1038/s41421-024-00665-0

---

## Immune interactions between mosquitoes and microbes during midgut colonization.
 29.03.2024, _JA Cai, GK Christophides_


Mosquitoes encounter diverse microbes during their lifetime, including symbiotic bacteria, shaping their midgut ecosystem. The organization of the midgut supports microbiota persistence while defending against potential pathogens. The influx of nutrients during blood feeding triggers bacterial proliferation, challenging host homeostasis. Immune responses, aimed at controlling bacterial overgrowth, impact blood-borne pathogens like malaria parasites. However, parasites deploy evasion strategies against mosquito immunity. Leveraging these mechanisms could help engineer malaria-resistant mosquitoes, offering a transformative tool for malaria elimination.

[Source](https://doi.org/10.1016/j.cois.2024.101195)

10.1016/j.cois.2024.101195

---

## A microbial knowledge graph-based deep learning model for predicting candidate microbes for target hosts.
 02.04.2024, _J Pan, Z Zhang, Y Li, J Yu, Z You, C Li, S Wang, M Zhu, F Ren, X Zhang, Y Sun, S Wang_


_Humans, Deep Learning, Pattern Recognition, Automated, Neural Networks, Computer, Algorithms, Bacteria_
Predicting interactions between microbes and hosts plays critical roles in microbiome population genetics and microbial ecology and evolution. How to systematically characterize the sophisticated mechanisms and signal interplay between microbes and hosts is a significant challenge for global health risks. Identifying microbe-host interactions (MHIs) can not only provide helpful insights into their fundamental regulatory mechanisms, but also facilitate the development of targeted therapies for microbial infections. In recent years, computational methods have become an appealing alternative due to the high risk and cost of wet-lab experiments. Therefore, in this study, we utilized rich microbial metagenomic information to construct a novel heterogeneous microbial network (HMN)-based model named KGVHI to predict candidate microbes for target hosts. Specifically, KGVHI first built a HMN by integrating human proteins, viruses and pathogenic bacteria with their biological attributes. Then KGVHI adopted a knowledge graph embedding strategy to capture the global topological structure information of the whole network. A natural language processing algorithm is used to extract the local biological attribute information from the nodes in HMN. Finally, we combined the local and global information and fed it into a blended deep neural network (DNN) for training and prediction. Compared to state-of-the-art methods, the comprehensive experimental results show that our model can obtain excellent results on the corresponding three MHI datasets. Furthermore, we also conducted two pathogenic bacteria case studies to further indicate that KGVHI has excellent predictive capabilities for potential MHI pairs.

[Source](https://doi.org/10.1093/bib/bbae119)

10.1093/bib/bbae119

---

## Towards a unified medical microbiome ecology of the OMU for metagenomes and the OTU for microbes.
 01.04.2024, _ZS Ma_


_Animals, Humans, Metagenome, Microbiota, Gastrointestinal Microbiome, Biodiversity, Sequence Analysis, DNA, Metagenomics_
Metagenomic sequencing technologies offered unprecedented opportunities and also challenges to microbiology and microbial ecology particularly. The technology has revolutionized the studies of microbes and enabled the high-profile human microbiome and earth microbiome projects. The terminology-change from microbes to microbiomes signals that our capability to count and classify microbes (microbiomes) has achieved the same or similar level as we can for the biomes (macrobiomes) of plants and animals (macrobes). While the traditional investigations of macrobiomes have usually been conducted through naturalists' (Linnaeus &amp; Darwin) naked eyes, and aerial and satellite images (remote-sensing), the large-scale investigations of microbiomes have been made possible by DNA-sequencing-based metagenomic technologies. Two major types of metagenomic sequencing technologies-amplicon sequencing and whole-genome (shotgun sequencing)-respectively generate two contrastingly different categories of metagenomic reads (data)-OTU (operational taxonomic unit) tables representing microorganisms and OMU (operational metagenomic unit), a new term coined in this article to represent various cluster units of metagenomic genes.

[Source](https://doi.org/10.1186/s12859-023-05591-8)

10.1186/s12859-023-05591-8

---

## An in-depth evaluation of metagenomic classifiers for soil microbiomes.
 31.03.2024, _NR Edwin, AH Fitzpatrick, F Brennan, F Abram, O O'Sullivan_


Recent endeavours in metagenomics, exemplified by projects such as the human microbiome project and TARA Oceans, have illuminated the complexities of microbial biomes. A robust bioinformatic pipeline and meticulous evaluation of their methodology have contributed to the success of these projects. The soil environment, however, with its unique challenges, requires a specialized methodological exploration to maximize microbial insights. A notable limitation in soil microbiome studies is the dearth of soil-specific reference databases available to classifiers that emulate the complexity of soil communities. There is also a lack of in-vitro mock communities derived from soil strains that can be assessed for taxonomic classification accuracy.

[Source](https://doi.org/10.1186/s40793-024-00561-w)

10.1186/s40793-024-00561-w

---

## Generalized reporter score-based enrichment analysis for omics data.
 30.03.2024, _C Peng, Q Chen, S Tan, X Shen, C Jiang_


_Benchmarking, Biomedical Research, Databases, Factual, Metabolome, Microbiota_
Enrichment analysis contextualizes biological features in pathways to facilitate a systematic understanding of high-dimensional data and is widely used in biomedical research. The emerging reporter score-based analysis (RSA) method shows more promising sensitivity, as it relies on P-values instead of raw values of features. However, RSA cannot be directly applied to multi-group and longitudinal experimental designs and is often misused due to the lack of a proper tool. Here, we propose the Generalized Reporter Score-based Analysis (GRSA) method for multi-group and longitudinal omics data. A comparison with other popular enrichment analysis methods demonstrated that GRSA had increased sensitivity across multiple benchmark datasets. We applied GRSA to microbiome, transcriptome and metabolome data and discovered new biological insights in omics studies. Finally, we demonstrated the application of GRSA beyond functional enrichment using a taxonomy database. We implemented GRSA in an R package, ReporterScore, integrating with a powerful visualization module and updatable pathway databases, which is available on the Comprehensive R Archive Network (https://cran.r-project.org/web/packages/ReporterScore). We believe that the ReporterScore package will be a valuable asset for broad biomedical research fields.

[Source](https://doi.org/10.1093/bib/bbae116)

10.1093/bib/bbae116

---

