## Microbiome for Mars: surveying microbiome connections to healthcare with implications for long-duration human spaceflight, virtual workshop, July 13, 2020.
 11.01.2021, _M LaPelusa, D Donoviel, SE Branzini, PE Carlson, S Culler, AK Cheema, R Kaddurah-Daouk, D Kelly, I de Cremoux, R Knight, R Krajmalnik-Brown, SL Mayo, SK Mazmanian, EA Mayer, JF Petrosino, K Garrison_


The inaugural "Microbiome for Mars" virtual workshop took place on July 13, 2020. This event assembled leaders in microbiome research and development to discuss their work and how it may relate to long-duration human space travel. The conference focused on surveying current microbiome research, future endeavors, and how this growing field could broadly impact human health and space exploration. This report summarizes each speaker's presentation in the order presented at the workshop.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00951-5)

10.1186/s40168-020-00951-5

---

## Identifying background microbiomes in an evidence recovery laboratory: A preliminary study

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1355030621000022)

---

## Comparison of viromes in vaginal secretion from pregnant women with and without vaginitis.
 11.01.2021, _HT Zhang, H Wang, HS Wu, J Zeng, Y Yang_


Although some studies have investigated the bacterial community in vaginal tract of pregnant women, there are few reports about the viral community (virome) in this type of microenvironment.

[Source](https://virologyj.biomedcentral.com/articles/10.1186/s12985-020-01482-z)

10.1186/s12985-020-01482-z

---

## <span class="has-inline-color has-vivid-green-cyan-color">Microbiome connections with host metabolism and habitual diet from 1,098 deeply phenotyped individuals</span>

[Source](https://www.nature.com/articles/s41591-020-01183-8)

---

## Gut Microbiota Dysbiosis Is Associated with Elevated Bile Acids in Parkinson's Disease.
 07.01.2021, _P Li, BA Killinger, E Ensink, I Beddows, A Yilmaz, N Lubben, J Lamp, M Schilthuis, IE Vega, R Woltjer, JA Pospisilik, P Brundin, L Brundin, SF Graham, V Labrie_


The gut microbiome can impact brain health and is altered in Parkinson's disease (PD). The vermiform appendix is a lymphoid tissue in the cecum implicated in the storage and regulation of the gut microbiota. We sought to determine whether the appendix microbiome is altered in PD and to analyze the biological consequences of the microbial alterations. We investigated the changes in the functional microbiota in the appendix of PD patients relative to controls (n = 12 PD, 16 C) by metatranscriptomic analysis. We found microbial dysbiosis affecting lipid metabolism, including an upregulation of bacteria responsible for secondary bile acid synthesis. We then quantitatively measure changes in bile acid abundance in PD relative to the controls in the appendix (n = 15 PD, 12 C) and ileum (n = 20 PD, 20 C). Bile acid analysis in the PD appendix reveals an increase in hydrophobic and secondary bile acids, deoxycholic acid (DCA) and lithocholic acid (LCA). Further proteomic and transcriptomic analysis in the appendix and ileum corroborated these findings, highlighting changes in the PD gut that are consistent with a disruption in bile acid control, including alterations in mediators of cholesterol homeostasis and lipid metabolism. Microbially derived toxic bile acids are heightened in PD, which suggests biliary abnormalities may play a role in PD pathogenesis.

[Source](https://www.mdpi.com/2218-1989/11/1/29)

10.3390/metabo11010029

---

## The Combined Effects of Magnesium Oxide and Inulin on Intestinal Microbiota and Cecal Short-Chain Fatty Acids

[Source](https://www.mdpi.com/2072-6643/13/1/152/htm)

---

## Effect of host breeds on gut microbiome and serum metabolome in meat rabbits.
 12.01.2021, _X Ye, L Zhou, Y Zhang, S Xue, QF Gan, S Fang_


Gut microbial compositional and functional variation can affect health and production performance of farm animals. Analysing metabolites in biological samples provides information on the basic mechanisms that affect the well-being and production traits in farm animals. However, the extent to which host breeds affect the gut microbiome and serum metabolome in meat rabbits is still unknown. In this study, the differences in phylogenetic composition and functional capacities of gut microbiota in two commercial rabbit breeds Elco and Ira were determined by 16S rRNA gene and metagenomic sequencing. The alternations in serum metabolome in the two rabbit breeds were detected using ultra-performance liquid chromatography system coupled with quadrupole time of flight mass spectrometry (UPLC-QTOFMS).

[Source](https://link.springer.com/article/10.1186/s12917-020-02732-6)

10.1186/s12917-020-02732-6

---

## Differential response of digesta- and mucosa-associated intestinal microbiota to dietary insect meal during the seawater phase of Atlantic salmon

[Source](https://animalmicrobiome.biomedcentral.com/articles/10.1186/s42523-020-00071-3)

---

## Diversity and composition of the Panax ginseng rhizosphere microbiome in various cultivation modesand ages.
 12.01.2021, _AZ Tong, W Liu, Q Liu, GQ Xia, JY Zhu_


Continuous cropping of ginseng (Panax ginseng Meyer) cultivated in farmland for an extended period gives rise to soil-borne disease. The change in soil microbial composition is a major cause of soil-borne diseases and an obstacle to continuous cropping. The impact of cultivation modes and ages on the diversity and composition of the P. ginseng rhizosphere microbial community and technology suitable for cropping P. ginseng in farmland are still being explored.

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-020-02081-2)

10.1186/s12866-020-02081-2

---

## Eukaryotic community diversity and pathogenic eukaryotes in a full-scale drinking water treatment plant determined by 18S rRNA and metagenomic sequencing

[Source](https://link.springer.com/article/10.1007/s11356-020-12079-y)

---

## Microbial community structure in aquifers associated with arsenic: analysis of 16S rRNA and arsenite oxidase genes

[Source](https://peerj.com/articles/10653/)

---

## Fungi Present in Antarctic Deep-Sea Sediments Assessed Using DNA Metabarcoding.
 06.01.2021, _MB Ogaki, OHB Pinto, R Vieira, AA Neto, P Convey, M Carvalho-Silva, CA Rosa, PEAS CÃ¢mara, LH Rosa_


We assessed fungal diversity in deep-sea sediments obtained from different depths in the Southern Ocean using the internal transcribed spacer 2 (ITS2) region of nuclear ribosomal DNA by metabarcoding through high-throughput sequencing (HTS). We detected 655,991 DNA reads representing 263 fungal amplicon sequence variants (ASVs), dominated by Ascomycota, Basidiomycota, Mortierellomycota, Mucoromycota, Chytridiomycota and Rozellomycota, confirming that deep-sea sediments can represent a hotspot of fungal diversity in Antarctica. The community diversity detected included 17 dominant fungal ASVs, 62 intermediate and 213 rare. The dominant fungi included taxa of Mortierella, Penicillium, Cladosporium, Pseudogymnoascus, Phaeosphaeria and Torula. Despite the extreme conditions of the Southern Ocean benthos, the total fungal community detected in these marine sediments displayed high indices of diversity and richness, and moderate dominance, which varied between the different depths sampled. The highest diversity indices were obtained in sediments from 550 m and 250 m depths. Only 49 ASVs (18.63%) were detected at all the depths sampled, while 16 ASVs were detected only in the deepest sediment sampled at 1463 m. Based on sequence identities, the fungal community included some globally distributed taxa, primarily recorded otherwise from terrestrial environments, suggesting transport from these to deep marine sediments. The assigned taxa included symbionts, decomposers and plant-, animal- and human-pathogenic fungi, suggesting that deep-sea sediments host a complex fungal diversity, although metabarcoding does not itself confirm that living or viable organisms are present.

[Source](https://link.springer.com/article/10.1007/s00248-020-01658-8)

10.1007/s00248-020-01658-8

---

## Comparative analysis of impact of human occupancy on indoor microbiomes.
 12.01.2021, _L Cao, L Yang, CS Swanson, S Li, Q He_


Educational facilities serve as community hubs and consequently hotspots for exposure to pathogenic microorganisms. Therefore, it is of critical importance to understand processes shaping the indoor microbiomes in educational facilities to protect public health by reducing potential exposure risks of students and the broader community. In this study, the indoor surface bacterial microbiomes were characterized in two multifunctional university buildings with contrasting levels of human occupancy, of which one was recently constructed with minimal human occupancy while the other had been in full operation for six years. Higher levels of human occupancy in the older building were shown to result in greater microbial abundance in the indoor environment and greater proportion of the indoor surface bacterial microbiomes contributed from human-associated microbiota, particularly the skin microbiota. It was further revealed that human-associated microbiota had greater influence on the indoor surface bacterial microbiomes in areas of high occupancy than areas of low occupancy. Consistent with minimal impact from human occupancy in a new construction, the indoor microbiomes in the new building exhibited significantly lower influence from human-associated microbiota than in the older building, with microbial taxa originating from soil and plants representing the dominant constituents of the indoor surface bacterial microbiomes. In contrast, microbial taxa in the older building with extensive human occupancy were represented by constituents of the human microbiota, likely from occupants. These findings provide insights into processes shaping the indoor microbiomes which will aid the development of effective strategies to control microbial exposure risks of occupants in educational facilities.

[Source](https://link.springer.com/article/10.1007/s11783-020-1383-1)

10.1007/s11783-020-1383-1

---

## Re-purposing software for functional characterization of the microbiome.
 12.01.2021, _LJ Gardiner, N Haiminen, F Utro, L Parida, E Seabolt, R Krishna, JH Kaufman_


Widespread bioinformatic resource development generates a constantly evolving and abundant landscape of workflows and software. For analysis of the microbiome, workflows typically begin with taxonomic classification of the microorganisms that are present in a given environment. Additional investigation is then required to uncover the functionality of the microbial community, in order to characterize its currently or potentially active biological processes. Such functional analysis of metagenomic data can be computationally demanding for high-throughput sequencing experiments. Instead, we can directly compare sequencing reads to a functionally annotated database. However, since reads frequently match multiple sequences equally well, analyses benefit from a hierarchical annotation tree, e.g. for taxonomic classification where reads are assigned to the lowest taxonomic unit.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00971-1)

10.1186/s40168-020-00971-1

---

## Mining underutilized whole-genome sequencing projects to improve 16S rRNA databases

[Source](https://www.biorxiv.org/content/10.1101/2021.01.01.425045v1)

---

## Meta-Apo improves accuracy of 16S-amplicon-based prediction of microbiome function.
 11.01.2021, _G Jing, Y Zhang, W Cui, L Liu, J Xu, X Su_


Due to their much lower costs in experiment and computation than metagenomic whole-genome sequencing (WGS), 16S rRNA gene amplicons have been widely used for predicting the functional profiles of microbiome, via software tools such as PICRUSt 2. However, due to the potential PCR bias and gene profile variation among phylogenetically related genomes, functional profiles predicted from 16S amplicons may deviate from WGS-derived ones, resulting in misleading results.

[Source](https://link.springer.com/article/10.1186/s12864-020-07307-1)

10.1186/s12864-020-07307-1

---

## Exploring protocol bias in airway microbiome studies: one versus two PCR steps and 16S rRNA gene region V3 V4 versus V4.
 05.01.2021, _C Drengenes, TML Eagan, I Haaland, HG Wiker, R Nielsen_


Studies on the airway microbiome have been performed using a wide range of laboratory protocols for high-throughput sequencing of the bacterial 16S ribosomal RNA (16S rRNA) gene. We sought to determine the impact of number of polymerase chain reaction (PCR) steps (1- or 2- steps) and choice of target marker gene region (V3 V4 and V4) on the presentation of the upper and lower airway microbiome. Our analyses included lllumina MiSeq sequencing following three setups: Setup 1 (2-step PCR; V3 V4 region), Setup 2 (2-step PCR; V4 region), Setup 3 (1-step PCR; V4 region). Samples included oral wash, protected specimen brushes and protected bronchoalveolar lavage (healthy and obstructive lung disease), and negative controls.

[Source](https://link.springer.com/article/10.1186/s12864-020-07252-z)

10.1186/s12864-020-07252-z

---

## Publisher Correction: Quantitative image analysis of microbial communities with BiofilmQ.
 10.01.2021, _R Hartmann, H Jeckel, E Jelli, PK Singh, S Vaidya, M Bayer, DKH Rode, L Vidakovic, F DÃ­az-Pascual, JCN Fong, A DragoÅ¡, O Lamprecht, JG ThÃ¶ming, N Netter, S HÃ¤ussler, CD Nadell, V Sourjik, ÃT KovÃ¡cs, FH Yildiz, K Drescher_



[Source](https://www.nature.com/articles/s41564-020-00817-4)

10.1038/s41564-021-00863-6

---

## Quantitative image analysis of microbial communities with BiofilmQ

[Source](https://www.nature.com/articles/s41564-020-00817-4)

---

## Do the bugs in our guts make us sick, sad and obese?

[Source](https://www.thestar.com/life/health_wellness/advice/2021/01/09/do-the-bugs-in-our-guts-make-us-sick-sad-and-obese.html)

---

## New microbiome research finds associations between microbes, diet, and metabolic health

[Source](https://www.nutritionaloutlook.com/view/new-microbiome-research-finds-associations-between-microbes-diet-and-metabolic-health)

---

