## A prevalent and culturable microbiota links ecological balance to clinical stability of the human lung after transplantation

[Source](https://www.biorxiv.org/content/10.1101/2020.05.21.106211v1.abstract?%3Fcollection=)

---

## A quantitative sequencing framework for absolute abundance measurements of mucosal and lumenal microbial communities.
 23.05.2020, _JT Barlow, SR Bogatyrev, RF Ismagilov_


A fundamental goal in microbiome studies is determining which microbes affect host physiology. Standard methods for determining changes in microbial taxa measure relative, rather than absolute abundances. Moreover, studies often analyze only stool, despite microbial diversity differing substantially among gastrointestinal (GI) locations. Here, we develop a quantitative framework to measure absolute abundances of individual bacterial taxa by combining the precision of digital PCR with the high-throughput nature of 16S rRNA gene amplicon sequencing. In a murine ketogenic-diet study, we compare microbial loads in lumenal and mucosal samples along the GI tract. Quantitative measurements of absolute (but not relative) abundances reveal decreases in total microbial loads on the ketogenic diet and enable us to determine the differential effects of diet on each taxon in stool and small-intestine mucosa samples. This rigorous quantitative microbial analysis framework, appropriate for diverse GI locations enables mapping microbial biogeography of the mammalian GI tract and more accurate analyses of changes in microbial taxa in microbiome studies.

[Source](https://www.nature.com/articles/s41467-020-16224-6)

10.1038/s41467-020-16224-6

---

## The interaction between microbiome and host central nervous system: the gut-brain axis as a potential new therapeutic target in the treatment of obesity and cardiometabolic disease.
 22.05.2020, _M Wijdeveld, M Nieuwdorp, R IJzerman_


The role of the intestinal microbiota in host cardiometabolic health and disease has gained significant attention over recent decades. Previous studies have shown effects on metabolic health through gut microbiota modulation; this suggests diverse interaction pathways that constitute the communication between gut microbiota and host central nervous system, the so-called gut-brain axis.

[Source](https://pubmed.ncbi.nlm.nih.gov/32441219/?from_term=microbiota+&from_sort=date&from_pos=3)

10.1080/14728222.2020.1761958

---

## Gut Microbiota and Disorders of the Central Nervous System

[Source](https://pubmed.ncbi.nlm.nih.gov/32441219/?from_term=microbiota+&from_sort=date&from_pos=3)

---

## Intestinal fungi are causally implicated in microbiome assembly and immune development in mice.
 23.05.2020, _E van Tilburg Bernardes, VK Pettersen, MW Gutierrez, I Laforest-Lapointe, NG Jendzjowsky, JB Cavin, FA Vicentini, CM Keenan, HR Ramay, J Samara, WK MacNaughton, RJA Wilson, MM Kelly, KD McCoy, KA Sharkey, MC Arrieta_


The gut microbiome consists of a multi-kingdom microbial community. Whilst the role of bacteria as causal contributors governing host physiological development is well established, the role of fungi remains to be determined. Here, we use germ-free mice colonized with defined species of bacteria, fungi, or both to differentiate the causal role of fungi on microbiome assembly, immune development, susceptibility to colitis, and airway inflammation. Fungal colonization promotes major shifts in bacterial microbiome ecology, and has an independent effect on innate and adaptive immune development in young mice. While exclusive fungal colonization is insufficient to elicit overt dextran sulfate sodium-induced colitis, bacterial and fungal co-colonization increase colonic inflammation. Ovalbumin-induced airway inflammation reveals that bacterial, but not fungal colonization is necessary to decrease airway inflammation, yet fungi selectively promotes macrophage infiltration in the airway. Together, our findings demonstrate a causal role for fungi in microbial ecology and host immune functionality, and therefore prompt the inclusion of fungi in therapeutic approaches aimed at modulating early life microbiomes.

[Source](https://www.nature.com/articles/s41467-020-16431-1)

10.1038/s41467-020-16431-1

---

## Linking perturbations to temporal changes in diversity, stability, and compositions of neonatal calf gut microbiota: prediction of diarrhea.
 23.05.2020, _T Ma, C Villot, D Renaud, A Skidmore, E Chevaux, M Steele, LL Guan_


Perturbations in early life gut microbiota can have long-term impacts on host health. In this study, we investigated antimicrobial-induced temporal changes in diversity, stability, and compositions of gut microbiota in neonatal veal calves, with the objective of identifying microbial markers that predict diarrhea. A total of 220 samples from 63 calves in first 8 weeks of life were used in this study. The results suggest that increase in diversity and stability of gut microbiota over time was a feature of "healthy" (non-diarrheic) calves during early life. Therapeutic antimicrobials delayed the temporal development of diversity and taxa-function robustness (a measure of microbial stability). In addition, predicted genes associated with beta lactam and cationic antimicrobial peptide resistance were more abundant in gut microbiota of calves treated with therapeutic antimicrobials. Random forest machine learning algorithm revealed that Trueperella, Streptococcus, Dorea, uncultured Lachnospiraceae, Ruminococcus 2, and Erysipelatoclostridium may be key microbial markers that can differentiate "healthy" and "unhealthy" (diarrheic) gut microbiota, as they predicted early life diarrhea with an accuracy of 84.3%. Our findings suggest that diarrhea in veal calves may be predicted by the shift in early life gut microbiota, which may provide an opportunity for early intervention (e.g., prebiotics or probiotics) to improve calf health with reduced usage of antimicrobials.

[Source](https://www.nature.com/articles/s41396-020-0678-3)

10.1038/s41396-020-0678-3

---

## Continental-scale microbiome study reveals different environmental characteristics determining microbial richness and composition/quantity in hotel rooms

[Source](https://www.biorxiv.org/content/10.1101/849430v1.abstract)

---

## Extreme storms cause rapid, context-dependent shifts in nearshore subtropical bacterial communities

[Source](https://www.biorxiv.org/content/10.1101/801886v2.abstract?%3Fcollection=)

---

## Mimicking climate warming effects on Alaskan soil microbial communities via gradual temperature increase.
 23.05.2020, _MB Ballhausen, R Hewitt, MC Rillig_


Climate change can trigger shifts in community structure and may therefore pose a severe threat to soil microbial communities, especially in high northern latitudes such as the Arctic. Arctic soils are covered by snow and ice throughout most of the year. This insulation shields them from high temperature variability and low surface temperatures. If this protective layer thaws, these soils are predicted to warm up at 1.5x to 4x the rate of other terrestrial biomes. In this study, we sampled arctic soils from sites with different elevations in Alaska, incubated them for 5 months with a simulated, gradual or abrupt temperature increase of +5 Â°C, and compared bacterial and fungal community compositions after the incubation. We hypothesized that the microbial communities would not significantly change with a gradual temperature treatment, whereas an abrupt temperature increase would decrease microbial diversity and shift community composition. The only differences in community composition that we observed were, however, related to the two elevations. The abrupt and gradual temperature increase treatments did not change the microbial community composition as compared to the control indicating resistance of the microbial community to changes in temperature. This points to the potential importance of microbial dormancy and resting stages in the formation of a "buffer" against elevated temperatures. Microbial resting stages might heavily contribute to microbial biomass and thus drive the responsiveness of arctic ecosystems to climate change.

[Source](https://www.nature.com/articles/s41598-020-65329-x)

10.1038/s41598-020-65329-x

---

## Deciphering the Root Endosphere Microbiome of the Desert Plant Alhagi sparsifolia for Drought Resistance-Promoting Bacteria.
 20.05.2020, _L Zhang, W Zhang, Q Li, R Cui, Z Wang, Y Wang, YZ Zhang, W Ding, X Shen_


Drought is among the most destructive abiotic stresses limiting crop growth and yield worldwide. Although most research has focused on the contribution of plant-associated microbial communities to plant growth and disease suppression, far less is known about the microbes involved in drought resistance among desert plants. In the present study, we applied 16S rRNA gene amplicon sequencing to determine the structure of rhizosphere and root endosphere microbiomes of Alhagi sparsifolia Compared to those of the rhizosphere, endosphere microbiomes had lower diversity but contained several taxa with higher relative abundance; many of these taxa were also present in the roots of other desert plants. We isolated a Pseudomonas strain (LTGT-11-2Z) that was prevalent in root endosphere microbiomes of A. sparsifolia and promoted drought resistance during incubation with wheat. Complete genome sequencing of LTGT-11-2Z revealed 1-aminocyclopropane-1-carboxylate deaminases, siderophore, spermidine, and colanic acid biosynthetic genes, as well as type VI secretion system (T6SS) genes, which are likely involved in biofilm formation and plant-microbe interactions. Together, these results indicate that drought-enduring plants harbor bacterial endophytes favorable to plant drought resistance, and they suggest that novel endophytic bacterial taxa and gene resources may be discovered among these desert plants.IMPORTANCE Understanding microbe-mediated plant resistance to drought is important for sustainable agriculture. We performed 16S rRNA gene amplicon sequencing and culture-dependent functional analyses of Alhagi sparsifolia rhizosphere and root endosphere microbiomes and identified key endophytic bacterial taxa and their genes facilitating drought resistance in wheat. This study improves our understanding of plant drought resistance and provides new avenues for drought resistance improvement in crop plants under field conditions.

[Source](https://aem.asm.org/content/early/2020/03/23/AEM.02863-19.abstract)

10.1128/AEM.02863-19

---

## Deciphering the root endosphere microbiome of the desert plant Alhagi sparsifolia for drought resistance-promoting bacteria

[Source](https://aem.asm.org/content/early/2020/03/23/AEM.02863-19.abstract)

---

## Drought and plant litter chemistry alter microbial gene expression and metabolite production.
 23.05.2020, _AA Malik, T Swenson, C Weihe, EW Morrison, JBH Martiny, EL Brodie, TR Northen, SD Allison_


Drought represents a significant stress to microorganisms and is known to reduce microbial activity and organic matter decomposition in Mediterranean ecosystems. However, we lack a detailed understanding of the drought stress response of microbial decomposers. Here we present metatranscriptomic and metabolomic data on the physiological response of in situ microbial communities on plant litter to long-term drought in Californian grass and shrub ecosystems. We hypothesised that drought causes greater microbial allocation to stress tolerance relative to growth pathways. In grass litter, communities from the decade-long ambient and reduced precipitation treatments had distinct taxonomic and functional profiles. The most discernable physiological signatures of drought were production or uptake of compatible solutes to maintain cellular osmotic balance, and synthesis of capsular and extracellular polymeric substances as a mechanism to retain water. The results show a clear functional response to drought in grass litter communities with greater allocation to survival relative to growth that could affect decomposition under drought. In contrast, communities on chemically more diverse and complex shrub litter had smaller physiological differences in response to long-term drought but higher investment in resource acquisition traits across precipitation treatments, suggesting that the functional response to drought is constrained by substrate quality. Our findings suggest, for the first time in a field setting, a trade off between microbial drought stress tolerance, resource acquisition and growth traits in plant litter microbial communities.

[Source](https://www.nature.com/articles/s41396-020-0683-6)

10.1038/s41396-020-0683-6

---

## Genome-scale metabolic reconstruction of the symbiosis between a leguminous plant and a nitrogen-fixing bacterium

[Source](https://www.nature.com/articles/s41467-020-16484-2)

---

