## Maternal breast milk microbiota and immune markers in relation to subsequent development of celiac disease in offspring.
 26.04.2022, _J Štšepetova, K Simre, A Tagoma, O Uibo, A Peet, H Siljander, V Tillmann, M Knip, R Mändar, R Uibo_


_Bacteria, Celiac Disease, Child, Child, Preschool, Female, Humans, Lactobacillus reuteri, Lipopolysaccharide Receptors, Microbiota, Milk, Human_
The potential impact of the composition of maternal breast milk is poorly known in children who develop celiac disease (CD). The aim of our study was to compare the microbiota composition and the concentrations of immune markers in breast milk from mothers whose offspring carried the genetic predisposition to CD, and whether they did or did not develop CD during follow-up for the first 3 years of life. Maternal breast milk samples [CD children (n = 6) and healthy children (n = 18)] were collected 3 months after delivery. Enzyme-linked immunosorbent assays were used to measure TGF-β1, TGF-β2, sIgA, MFG-E8 and sCD14. For microbiota analysis, next generation (Illumina) sequencing, real-time PCR and denaturing gradient gel electrophoresis were used. Phylotype abundance and the Shannon 'H' diversity index were significantly higher in breast milk samples in the CD group. There was higher prevalence of the phyla Bacteroidetes and Fusobacteria, the classes Clostridia and Fusobacteriia, and the genera Leptotrichia, Anaerococcus, Sphingomonas, Actynomyces and Akkermansia in the CD group. The immunological markers were differently associated with some Gram-negative bacterial genera and species (Chryseobacterium, Sphingobium) as well as Gram-positive species (Lactobacillus reuteri, Bifidobacterium animalis). In conclusion, the microbiota in breast milk from mothers of genetically predisposed offspring who presented CD showed a higher bacterial phylotype abundance and diversity, as well as a different bacterial composition, as compared with the mothers of unaffected offspring. These immune markers showed some associations with bacterial composition and may influence the risk for development of CD beyond early childhood.

[Source](https://www.nature.com/articles/s41598-022-10679-x)

10.1038/s41598-022-10679-x

---

## The gut fungal and bacterial microbiota in pediatric patients with inflammatory bowel disease introduced to treatment with anti-tumor necrosis factor-α.
 26.04.2022, _R Ventin-Holmberg, M Höyhtyä, S Saqib, K Korpela, A Nikkonen, A Salonen, WM de Vos, KL Kolho_


_Bacteria, Child, Crohn Disease, Humans, Inflammatory Bowel Diseases, Infliximab, Leukocyte L1 Antigen Complex, Microbiota, Tumor Necrosis Factor-alpha_
Pediatric inflammatory bowel disease (PIBD) is a globally increasing chronic inflammatory disease associated with an imbalanced intestinal microbiota and treated with several treatment options, including anti-tumor necrosis factor alpha (TNF-α), such as infliximab (IFX). Up to half of the patients do not respond to the drug and there are no methods for response prediction. Our aim was to predict IFX response from the gut microbiota composition since this is largely unexplored in PIBD. The gut microbiota of 30 PIBD patients receiving IFX was studied by MiSeq sequencing targeting 16S and ITS region from fecal samples collected before IFX and two and six weeks after the start of treatment. The response to IFX induction was determined by fecal calprotectin value &lt; 100 µg/g at week six. The bacterial microbiota differed significantly between response groups, with higher relative abundance of butyrate-producing bacteria in responders compared to non-responders at baseline, validated by high predictive power (area under curve = 0.892) for baseline Ruminococcus and calprotectin. Additionally, non-responders had higher abundance of Candida, while responders had higher abundance of Saccharomyces at the end of the study. The gut microbiota composition in PIBD patients could predict response to IFX treatment in the future.

[Source](https://www.nature.com/articles/s41598-022-10548-7)

10.1038/s41598-022-10548-7

---

## Integrated fecal microbiome-metabolome signatures reflect stress and serotonin metabolism in irritable bowel syndrome.
 25.04.2022, _Z Mujagic, M Kasapi, DM Jonkers, I Garcia-Perez, L Vork, ZZRM Weerts, JI Serrano-Contreras, A Zhernakova, A Kurilshikov, J Scotcher, E Holmes, C Wijmenga, D Keszthelyi, JK Nicholson, JM Posma, AA Masclee_


_Feces, Gastrointestinal Microbiome, Humans, Irritable Bowel Syndrome, Metabolome, Microbiota, Serotonin_
To gain insight into the complex microbiome-gut-brain axis in irritable bowel syndrome (IBS), several modalities of biological and clinical data must be combined. We aimed to identify profiles of fecal microbiota and metabolites associated with IBS and to delineate specific phenotypes of IBS that represent potential pathophysiological mechanisms. Fecal metabolites were measured using proton nuclear magnetic resonance (1H-NMR) spectroscopy and gut microbiome using shotgun metagenomic sequencing (MGS) in a combined dataset of 142 IBS patients and 120 healthy controls (HCs) with extensive clinical, biological and phenotype information. Data were analyzed using support vector classification and regression and kernel t-SNE. Microbiome and metabolome profiles could distinguish IBS and HC with an area-under-the-receiver-operator-curve of 77.3% and 79.5%, respectively, but this could be improved by combining microbiota and metabolites to 83.6%. No significant differences in predictive ability of the microbiome-metabolome data were observed between the three classical, stool pattern-based, IBS subtypes. However, unsupervised clustering showed distinct subsets of IBS patients based on fecal microbiome-metabolome data. These clusters could be related plasma levels of serotonin and its metabolite 5-hydroxyindoleacetate, effects of psychological stress on gastrointestinal (GI) symptoms, onset of IBS after stressful events, medical history of previous abdominal surgery, dietary caloric intake and IBS symptom duration. Furthermore, pathways in metabolic reaction networks were integrated with microbiota data, that reflect the host-microbiome interactions in IBS. The identified microbiome-metabolome signatures for IBS, associated with altered serotonin metabolism and unfavorable stress response related to GI symptoms, support the microbiota-gut-brain link in the pathogenesis of IBS.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2022.2063016)

10.1080/19490976.2022.2063016

---

## Diversity and distribution of sulfur metabolic genes in the human gut microbiome and their association with colorectal cancer.
 23.04.2022, _PG Wolf, ES Cowley, A Breister, S Matatov, L Lucio, P Polak, JM Ridlon, HR Gaskins, K Anantharaman_


_Bacteria, Carcinoma, Colorectal Neoplasms, Gastrointestinal Microbiome, Humans, Sulfates, Sulfur, Taurine_
Recent evidence implicates microbial sulfidogenesis as a potential trigger of colorectal cancer (CRC), highlighting the need for comprehensive knowledge of sulfur metabolism within the human gut. Microbial sulfidogenesis produces genotoxic hydrogen sulfide (H2S) in the human colon using inorganic (sulfate) and organic (taurine/cysteine/methionine) substrates; however, the majority of studies have focused on sulfate reduction using dissimilatory sulfite reductases (Dsr).

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01242-x)

10.1186/s40168-022-01242-x

---

## Clinical significance and intestinal microbiota composition in immunocompromised children with norovirus gastroenteritis.
 23.04.2022, _PC Lin, YSH Yang, SC Lin, MC Lu, YT Tsai, SC Lu, SH Chen, SY Chen_


_Caliciviridae Infections, Child, Feces, Gastroenteritis, Gastrointestinal Microbiome, Genotype, Humans, Infant, Norovirus, RNA, Viral_
Norovirus (NoV) infection is common in pediatric patients with immunodeficiency and is more likely to cause severe disease. Objective Our study aims to figure out the clinical differences and distribution of intestinal microbiota in immunocompromised children with NoV gastroenteritis.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0266876)

10.1371/journal.pone.0266876

---

## Microbial Diversity and Composition in Six Different Gastrointestinal Sites among Participants Undergoing Upper Gastrointestinal Endoscopy in Henan, China.
 25.04.2022, _M Li, D Shao, J Zhou, J Gu, J Qin, X Li, C Hao, W Wei_


The objective of this study was to describe and compare the dynamic microbiota characteristics in the gastrointestinal (GI) tract in Chinese participants via high-throughput sequencing techniques. The study collected saliva, esophageal swab, cardia biopsy, noncardia biopsy, gastric juice, and fecal specimens from 40 participants who underwent upper GI tract cancer screening in Linzhou (Henan, China) in August 2019. The V4 region of 16S rRNA genes was amplified and sequenced using the Illumina MiniSeq platform. The observed amplicon sequence variants (ASVs) gradually decreased from saliva to esophageal swab, cardia biopsy, noncardia biopsy, and gastric juice specimens and then increased from gastric juice to fecal specimens (P &lt; 0.05). Each GI site had its own microbial characteristics that overlapped those of adjacent sites. Characteristic genera for each site were as follows: Neisseria and Prevotella in saliva, Streptococcus and Haemophilus in the esophagus, Helicobacter in the noncardia, Pseudomonas in gastric juice, Faecalibacterium, Roseburia, and Blautia in feces, and Weissella in the cardia. Helicobacter pylori-positive participants had decreased observed ASVs (cardia, P &lt; 0.01; noncardia, P &lt; 0.001) and Shannon index values (cardia, P &lt; 0.001; noncardia, P &lt; 0.001) compared with H. pylori-negative participants both in cardia and noncardia specimens. H. pylori infection played a more important role in the microbial composition of noncardia than of cardia specimens. In gastric juice, the gastric pH and H. pylori infection had similar additive effects on the microbial diversity and composition. These results show that each GI site has its own microbial characteristics that overlap those of adjacent sites and that differences and commonalities between and within microbial compositions coexist, providing essential foundations for the continuing exploration of disease-associated microbiota. IMPORTANCE Upper gastrointestinal (UGI) tract cancer is one of the most common cancers worldwide, while limited attention has been paid to the UGI microbiota. Microbial biomarkers, such as Fusobacteria nucleatum and Helicobacter pylori, bring new ideas for early detection of UGI tract cancer, which may be a highly feasible method to reduce its disease burden. This study revealed that each gastrointestinal site had its own microbial characteristics that overlapped those of adjacent sites. There were significant differences between the microbial compositions of the UGI sites and feces. Helicobacter pylori played a more significant role in the microbial composition of the noncardia stomach than in that of the cardia. Gastric pH and Helicobacter pylori had similar additive effects on the microbial diversity of gastric juice. These findings played a key role in delineating the microbiology spectrum of the gastrointestinal tract and provided baseline information for future microbial exploration covering etiology, primary screening, treatment, outcome, and health care products.

[Source](https://journals.asm.org/doi/10.1128/spectrum.00645-21)

10.1128/spectrum.00645-21

---

## A multi-omics machine learning framework in predicting the survival of colorectal cancer patients.
 25.04.2022, _M Yang, H Yang, L Ji, X Hu, G Tian, B Wang, J Yang_


Colorectal cancer (CRC), the 3rd most universal cancer globally, accounts for approximately 10% of newly diagnosed cancer incidences each year. Identifying biomarkers associated with CRC survival and predicting the survival of CRC patients are critical for personalized therapy. Existing studies on CRC survival are mainly based on single omics, studies using multi-omics to predict CRC survival are still vacant. To fill in this gap, we aim to identify biomarkers associated with CRC survival at mRNA, miRNA and tissue microbiome levels, and to evaluate the accuracy of potential biomarkers in predicting CRC survival.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0010482522003080)

10.1016/j.compbiomed.2022.105516

---

## The gut microbiome as a modulator of healthy ageing.
 26.04.2022, _TS Ghosh, F Shanahan, PW O'Toole_


The gut microbiome is a contributory factor in ageing-related health loss and in several non-communicable diseases in all age groups. Some age-linked and disease-linked compositional and functional changes overlap, while others are distinct. In this Review, we explore targeted studies of the gut microbiome of older individuals and general cohort studies across geographically distinct populations. We also address the promise of the targeted restoration of microorganisms associated with healthier ageing.

[Source](https://www.nature.com/articles/s41575-022-00605-x)

10.1038/s41575-022-00605-x

---

## Multiple sclerosis patients have an altered gut mycobiome and increased fungal to bacterial richness.
 26.04.2022, _M Yadav, S Ali, RL Shrode, SK Shahi, SN Jensen, J Hoang, S Cassidy, H Olalde, N Guseva, M Paullus, C Cherwin, K Wang, T Cho, J Kamholz, AK Mangalam_


Trillions of microbes such as bacteria, fungi, and viruses exist in the healthy human gut microbiome. Although gut bacterial dysbiosis has been extensively studied in multiple sclerosis (MS), the significance of the fungal microbiome (mycobiome) is an understudied and neglected part of the intestinal microbiome in MS. The aim of this study was to characterize the gut mycobiome of patients with relapsing-remitting multiple sclerosis (RRMS), compare it to healthy controls, and examine its association with changes in the bacterial microbiome. We characterized and compared the mycobiome of 20 RRMS patients and 33 healthy controls (HC) using Internal Transcribed Spacer 2 (ITS2) and compared mycobiome interactions with the bacterial microbiome using 16S rRNA sequencing. Our results demonstrate an altered mycobiome in RRMS patients compared with HC. RRMS patients showed an increased abundance of Basidiomycota and decreased Ascomycota at the phylum level with an increased abundance of Candida and Epicoccum genera along with a decreased abundance of Saccharomyces compared to HC. We also observed an increased ITS2/16S ratio, altered fungal and bacterial associations, and altered fungal functional profiles in MS patients compared to HC. This study demonstrates that RRMS patients had a distinct mycobiome with associated changes in the bacterial microbiome compared to HC. There is an increased fungal to bacterial ratio as well as more diverse fungal-bacterial interactions in RRMS patients compared to HC. Our study is the first step towards future studies in delineating the mechanisms through which the fungal microbiome can influence MS disease.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0264556)

10.1371/journal.pone.0264556

---

## Cancer immunotherapy resistance: The impact of microbiome-derived short-chain fatty acids and other emerging metabolites.
 26.04.2022, _F Hersi, SM Elgendy, SA Al Shamma, RT Atell, O Sadiek, HA Omar_


The landscape of cancer therapy has undergone dramatic changes over the past decade. Immune checkpoint inhibitors (ICIs) among various cancer immunotherapies have transformed the treatment paradigm for cancer therapy and improved the survival of patients. Nevertheless, oncologists are faced with key challenges that need to be overcome, such as the unpredictability of patient response to these therapies and the many immune-related adverse effects (irAEs). One major factor contributing to patient response to treatment is the composition of their gut microbiota. Many studies reported the role of gut microbiota in modulating immunotherapy. In particular, microbiota-derived metabolites, mainly short-chain fatty acids (SCFAs), have been the highlights of many studies exploring the association between the gut microbiome and patient sensitivity to cancer immunotherapy. This review discusses the role of gut microbiota-derived metabolites on patient response to ICIs and their potential use as predictive biomarkers and therapeutic targets to fine tune, regulate, and enhance cancer immunotherapy.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0024320522002739)

10.1016/j.lfs.2022.120573

---

## Oral administration of Akkermansia muciniphila elevates systemic antiaging and anticancer metabolites.
 26.07.2021, _C Grajeda-Iglesias, S Durand, R Daillère, K Iribarren, F Lemaitre, L Derosa, F Aprahamian, N Bossut, N Nirmalathasan, F Madeo, L Zitvogel, G Kroemer_


_Administration, Oral, Akkermansia, Animals, Fatty Acids, Volatile, Gastrointestinal Microbiome, Gastrointestinal Tract, Hydroxybutyrates, Liver, Metabolome, Metabolomics, Mice, Inbred C57BL, Pasteurization, Polyamines, Probiotics, Spermidine_
The presence of Akkermansia muciniphila (Akk) in the human gut is associated with good health, leanness and fitness. Mouse experimentation has demonstrated positive effects for Akk, which counteracts aging, mediates antiobesity and antidiabetic effects, dampens inflammation and improves anticancer immunosurveillance. Clinical trials have confirmed antidiabetic effects for Akk. Here, we investigated the time-dependent effects of oral administration of Akk (which was live or pasteurized) and other bacteria to mice on the metabolome of the ileum, colon, liver and blood plasma. Metabolomics was performed by a combination of chromatographic and mass spectrometric methods, yielding a total of 1.637.227 measurements. Akk had major effects on metabolism, causing an increase in spermidine and other polyamines in the gut and in the liver. Pasteurized Akk (Akk-past) was more efficient than live Akk in elevating the intestinal concentrations of polyamines, short-chain fatty acids, 2-hydroxybutyrate, as well multiple bile acids, which also increased in the circulation. All these metabolites have previously been associated with human health, providing a biochemical basis for the beneficial effects of Akk.

[Source](https://www.aging-us.com/article/202739/text)

10.18632/aging.202739

---

## Lower abundance of Bacteroides and metabolic dysfunction are highly associated with the post-weaning diarrhea in piglets.
 25.04.2022, _W Ren, B Yu, J Yu, P Zheng, Z Huang, J Luo, X Mao, J He, H Yan, J Wu, D Chen, Y Luo_


Growing evidences show a direct link between diarrhea and disorders of gut microbiota in pigs. However, whether there are microbial markers associated with post-weaning diarrhea remains unknown. In the current study, we compared the microbial community, functions and metabolites between healthy weaned piglets (group H, n=7) and piglets with post-weaning diarrhea (group D, n=7), in order to find out diarrhea associated microbial markers. Each of 7 fecal samples was collected from H and D piglets (weaned at 21 d and sampled at 26 d). The metagenomic and untargeted metabolomic analysis revealed that the microbial composition, function and metabolic profile in D pigs was considerably reshaped, including the reduced abundance and number of Bacteroides, which significantly correlated with the diarrhea status of host. The carbohydrate metabolism, biosynthesis and metabolism, lipid metabolism, amino acid metabolism, and the activity of glycan and carbohydrates digestion related enzymes showed extensively down-regulated in D pigs compared with H pigs. Diarrhea significantly changed the metabolic profiles of fecal microbiota, and most of the altered metabolites were negatively or positively correlated with the change in the abundance of Bacteroides. In conclusion, the lower abundance of Bacteroides and its associated metabolic dysfunction may be regarded as microbial markers of physiological post-weaning diarrhea in piglets.

[Source](https://link.springer.com/article/10.1007/s11427-021-2068-6)

10.1007/s11427-021-2068-6

---

## Shotgun metagenomic sequencing revealed the prebiotic potential of a grain-based diet in mice.
 26.04.2022, _A Jangid, S Fukuda, Y Suzuki, TD Taylor, H Ohno, T Prakash_


In the present study, we elucidated the effect of grain-based (GB) diet containing both soluble and insoluble fibers and purified ingredients-based (PIB) diet containing only insoluble fiber, namely cellulose on mice gut microbiome using whole shotgun based metagenomic sequencing. Although the fiber content in both diet types is the same (5%) the presence of soluble fiber only in the GB diet differentiates it from the PIB diet. The taxonomic analysis of sequenced reads reveals a significantly higher enrichment of probiotic Lactobacilli in the GB group as compared to the PIB group. Further, the enhancement of energy expensive cellular processes namely, cell cycle control, cell division, chromosome partitioning, and transcription is observed in the GB group which could be due to the metabolization of the soluble fiber for faster energy production. In contrast, a higher abundance of cellulolytic bacterial community namely, the members of family Lachnospiraceae and Ruminococcaceae and the metabolism functions are found in the PIB group. The PIB group shows a significant increase in host-derived oligosaccharide metabolism functions indicating that they might first target the host-derived oligosaccharides and self-stored glycogen in addition to utilising the available cellulose. In addition to the beneficial microbial community variations, both the groups also exhibited an increased abundance of opportunistic pathobionts which could be due to an overall low amount of fiber in the diet. Furthermore, backtracing analysis identified probiotic members of Lactobacillus, viz., L. crispatus ST1, L. fermentum CECT 5716, L. gasseri ATCC 33323, L. johnsonii NCC 533 and L. reuteri 100-23 in the GB group, while Bilophila wadsworthia 3_1_6, Desulfovibrio piger ATCC 29098, Clostridium symbiosum WAL-14163, and Ruminococcaceae bacterium D16 in the PIB group. These data suggest that Lactobacilli, a probiotic community of microorganisms, are the predominant functional contributors in the gut of GB diet-fed mice, whereas pathobionts too coexisted with commensals in the gut microbiome of the PIB group. Thus at 5% fiber, GB modifies the gut microbial ecology more effectively than PIB and the inclusion of soluble fiber in the GB diet may be one of the primary factors responsible for this impact.

[Source](https://www.nature.com/articles/s41598-022-10762-3)

10.1038/s41598-022-10762-3

---

## Carbendazim shapes microbiome and enhances resistome in the earthworm gut.
 23.04.2022, _J Song, T Li, Z Zheng, W Fu, Z Long, N Shi, Y Han, L Zhang, Y Yu, H Fang_


_Animals, Anti-Bacterial Agents, Bacteria, Benzimidazoles, Carbamates, Fungicides, Industrial, Genes, Bacterial, Microbiota, Oligochaeta, Soil_
It is worrisome that several pollutants can enhance the abundance of antibiotic resistance genes (ARGs) in the environment, including agricultural fungicides. As an important bioindicator for environmental risk assessment, earthworm is still a neglected focus that the effects of the fungicide carbendazim (CBD) residues on the gut microbiome and resistome are largely unknown. In this study, Eisenia fetida was selected to investigate the effects of CBD in the soil-earthworm systems using shotgun metagenomics and qPCR methods.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01261-8)

10.1186/s40168-022-01261-8

---

## Fungal signatures of oral disease reflect environmental degradation in a facultative avian scavenger

[Source](https://www.sciencedirect.com/science/article/pii/S0048969722024901)

---

## Whole-Genome Shotgun Metagenomic Sequencing Reveals Distinct Gut Microbiome Signatures of Obese Cats.
 25.04.2022, _X Ma, E Brinker, EC Graff, W Cao, AL Gross, AK Johnson, C Zhang, DR Martin, X Wang_


Overweight and obesity are growing health problems in domestic cats, increasing the risks of insulin resistance, lipid dyscrasias, neoplasia, cardiovascular disease, and decreasing longevity. The signature of obesity in the feline gut microbiota has not been studied at the whole-genome metagenomic level. We performed whole-genome shotgun metagenomic sequencing in the fecal samples of eight overweight/obese and eight normal cats housed in the same research environment. We obtained 271 Gbp of sequences and generated a 961-Mbp de novo reference contig assembly, with 1.14 million annotated microbial genes. In the obese cat microbiome, we discovered a significant reduction in microbial diversity (P &lt; 0.01) and Firmicutes abundance (P = 0.005), as well as decreased Firmicutes/Bacteroidetes ratios (P = 0.02), which is the inverse of obese human/mouse microbiota. Linear discriminant analysis and quantitative PCR (qPCR) validation revealed significant increases of Bifidobacterium sp., Olsenella provencensis, Dialister sp.CAG:486, and Campylobacter upsaliensis as the hallmark of obese microbiota among 400 enriched species, whereas 1,525 bacterial species have decreased abundance in the obese microbiome. Phascolarctobacterium succinatutens and an uncharacterized Erysipelotrichaceae bacterium are highly abundant (&gt;0.05%) in the normal gut with over 400-fold depletion in the obese microbiome. Fatty acid synthesis-related pathways are significantly overrepresented in the obese compared with the normal cat microbiome. In conclusion, we discovered dramatically decreased microbial diversity in obese cat gut microbiota, suggesting potential dysbiosis. A panel of seven significantly altered, highly abundant species can serve as a microbiome indicator of obesity. Our findings in the obese cat microbiome composition, abundance, and functional capacities provide new insights into feline obesity. IMPORTANCE Obesity affects around 45% of domestic cats, and licensed drugs for treating feline obesity are lacking. Physical exercise and calorie restrictions are commonly used for weight loss but with limited efficacy. Through comprehensive analyses of normal and obese cat gut bacteria flora, we identified dramatic shifts in the obese gut microbiome, including four bacterial species significantly enriched and two species depleted in the obese cats. The key bacterial community and functional capacity alterations discovered from this study will inform new weight management strategies for obese cats, such as evaluations of specific diet formulas that alter the microbiome composition, and the development of prebiotics and probiotics that promote the increase of beneficial species and the depletion of obesity-associated species. Interestingly, these bacteria identified in our study were also reported to affect the weight loss success in human patients, suggesting translational potential in human obesity.

[Source](https://journals.asm.org/doi/10.1128/spectrum.00837-22)

10.1128/spectrum.00837-22

---

## Species divergence in gut-restricted bacteria of social bees.
 25.04.2022, _Y Li, SP Leonard, JE Powell, NA Moran_


SignificanceThe diversification of host-associated microbial communities depends on barriers to gene flow, imposed by confinement to different hosts or by niche partitioning within single hosts. However, most gut microbiomes are too complex to disentangle the diversification processes. Taking advantage of the simple gut microbiomes of social bees, we demonstrate that bee gut bacteria have diversified both between host species and within single host species through the acquisition of different ecological niches within the same gut. Our study further shows that gut microbiomes differ in spatial distributions within the same host, possibility due to adaptation to specified nutritional niches, such as urea utilization.

[Source](https://www.pnas.org/doi/full/10.1073/pnas.2115013119)

10.1073/pnas.2115013119

---

## EggVio: a user friendly and versatile pipeline for assembly and functional annotation of shallow depth sequenced samples

[Source](https://www.biorxiv.org/content/10.1101/2022.04.23.489251v1.abstract)

---

## FoodMicrobionet v4: a large, integrated, open and transparent database for food bacterial communities

[Source](https://www.biorxiv.org/content/10.1101/2022.01.19.476946v2.abstract)

---

## Run-on sequencing reveals nascent transcriptomics of the human microbiome

[Source](https://www.biorxiv.org/content/10.1101/2022.04.22.489220v1.abstract)

---

