## Noninvasive, microbiome-based diagnosis of inflammatory bowel disease.
 04.10.2024, _J Zheng, Q Sun, M Zhang, C Liu, Q Su, L Zhang, Z Xu, W Lu, J Ching, W Tang, CP Cheung, AL Hamilton, AL Wilson O'Brien, SC Wei, CN Bernstein, DT Rubin, EB Chang, M Morrison, MA Kamm, FKL Chan, J Zhang, SC Ng_


Despite recent progress in our understanding of the association between the gut microbiome and inflammatory bowel disease (IBD), the role of microbiome biomarkers in IBD diagnosis remains underexplored. Here we developed a microbiome-based diagnostic test for IBD. By utilization of metagenomic data from 5,979 fecal samples with and without IBD from different geographies and ethnicities, we identified microbiota alterations in IBD and selected ten and nine bacterial species for construction of diagnostic models for ulcerative colitis and Crohn's disease, respectively. These diagnostic models achieved areas under the curve &gt;0.90 for distinguishing IBD from controls in the discovery cohort, and maintained satisfactory performance in transethnic validation cohorts from eight populations. We further developed a multiplex droplet digital polymerase chain reaction test targeting selected IBD-associated bacterial species, and models based on this test showed numerically higher performance than fecal calprotectin in discriminating ulcerative colitis and Crohn's disease from controls. Here we discovered universal IBD-associated bacteria and show the potential applicability of a multibacteria biomarker panel as a noninvasive tool for IBD diagnosis.

[Source](https://doi.org/10.1038/s41591-024-03280-4)

10.1038/s41591-024-03280-4

---

## Advancing microbiome research in Māori populations: insights from recent literature exploring the gut microbiomes of underrepresented and Indigenous peoples.
 04.10.2024, _ET Silk, SB Bayer, M Foster, NC Roy, MW Taylor, T Vatanen, RB Gearry_


The gut microbiome plays vital roles in human health, including mediating metabolism, immunity, and the gut-brain axis. Many ethnicities remain underrepresented in gut microbiome research, with significant variation between Indigenous and non-Indigenous peoples due to dietary, socioeconomic, health, and urbanization differences. Although research regarding the microbiomes of Indigenous peoples is increasing, Māori microbiome literature is lacking despite widespread inequities that Māori populations face. These inequities likely contribute to gut microbiome differences that exacerbate negative health outcomes. Characterizing the gut microbiomes of underrepresented populations is necessary to inform efforts to address health inequities. However, for microbiome research to be culturally responsible and meaningful, study design must improve to better protect the rights and interests of Indigenous peoples. Here, we discuss barriers to Indigenous participation in research and the role disparities may play in shaping the gut microbiomes of Indigenous peoples, with a particular focus on implications for Māori and areas for improvement.

[Source](https://doi.org/10.1128/msystems.00909-24)

10.1128/msystems.00909-24

---

## Salivary microbiota reflecting changes in subgingival microbiota.
 04.10.2024, _J-S Jung, J-K Kook, S-N Park, YK Lim, GH Choi, S Kim, S Ji_


The purpose of this study was to determine whether subgingival microbial changes according to the severity of periodontal disease and following the non-surgical periodontal treatment of periodontitis are reflected in the saliva microbiota. Subgingival and saliva samples were collected from 7 periodontally healthy controls, 14 patients with gingivitis, 12 with moderate periodontitis, and 18 with severe periodontitis. Among subjects who received treatment, seven moderate and seven severe periodontitis patients were selected for post-treatment microbial analysis, and their samples were analyzed at baseline and 6 months after treatment. The V3 and V4 regions of the 16S rRNA gene were sequenced, and correlations of the relative abundance of phyla and health- or periodontitis-dominant species between subgingival plaque and saliva samples were analyzed using Spearman signed-rank tests. Alpha diversity was higher in saliva than subgingival plaque samples, and beta diversity was significantly different between the two samples. However, levels of phyla and most health- or periodontitis-dominant species in salivary microbiota were significantly correlated with those in subgingival plaque. The improvement in clinical parameters following treatment was accompanied by a microbial shift not only in subgingival plaque but also in saliva. The abundance of 2 phyla including Bacteroidetes, 6 genera including Porphyromonas and Treponema, and 11 species including Porphyromonas gingivalis, Tannerella forsythia, and Filifactor alocis was significantly reduced in saliva following treatment. These results indicate that the salivary microbiota can reflect changes in the subgingival microbiota, suggesting that saliva can be used as a diagnostic tool to monitor the periodontal health status of individuals.

[Source](https://doi.org/10.1128/spectrum.01030-24)

10.1128/spectrum.01030-24

---

## Differences in the human gut microbiota with varying depressive symptom severity scores.
 05.10.2024, _Y Masamura, R Kubo, Y Midorikawa, NO Shinozaki, S Watanabe, S Maekawa, AK Takeda, T Ohta_


Depression is a prevalent mental health disorder, and its incidence has increased further because of the coronavirus disease 2019 (COVID-19) pandemic. The gut microbiome has been suggested as a potential target for mental health treatment because of the bidirectional communication system between the brain and gastrointestinal tract, known as the gut-brain axis. We aimed to investigate the relationship between the human gut microbiome and depression screening by analyzing the abundance and types of microbiomes among individuals living in Japan, where mental health awareness and support may differ from those in other countries owing to cultural factors. We used a data-driven approach to evaluate the gut microbiome of participants who underwent commercial gut microbiota testing services and completed a questionnaire survey that included a test for scoring depressive tendencies. Our data analysis results indicated that no significant differences in gut microbiome composition were found among the groups based on their depression screening scores. However, the results also indicated the potential existence of a few differentially abundant bacterial taxa. Specifically, the detected bacterial changes in abundance suggest that the Bifidobacteriaceae, Streptococcaceae, and Veillonellaceae families are candidates for differentially abundant bacteria. Our findings should contribute to the growing body of research on the relationship between gut microbiome and mental health, highlighting the potential of microbiome-based interventions for depression treatment. The limitations of this study include the lack of clear medical information on the participants' diagnoses. Future research could benefit from a larger sample size and more detailed clinical information.

[Source](https://doi.org/10.12938/bmfh.2023-049)

10.12938/bmfh.2023-049

---

## Sophorolipid: An Effective Biomolecule for Targeting Microbial Biofilms.
 07.10.2024, _S Dhadwal, S Handa, M Chatterjee, IM Banat_


_Biofilms, Surface-Active Agents, Oleic Acids, Anti-Bacterial Agents, Bacteria_
Biofilms are microbial aggregates encased in a matrix that is attached to biological or nonbiological surfaces and constitute serious problems in food, medical, and marine industries and can have major negative effects on both health and the economy. Biofilm's complex microbial community provides a resistant environment that is difficult to eradicate and is extremely resilient to antibiotics and sanitizers. There are various conventional techniques for combating biofilms, including, chemical removal, physical or mechanical removal, use of antibiotics and disinfectants to destroy biofilm producing organisms. In contrast to free living planktonic cells, biofilms are very resistant to these methods. Hence, new strategies that differ from traditional approaches are urgently required. Microbial world offers a wide range of effective "green" compounds such as biosurfactants. They outperform synthetic surfactants in terms of biodegradability, superior stabilization, and reduced toxicity concerns. They also have better antiadhesive and anti-biofilm capabilities which can be used to treat biofilm-related problems. Sophorolipids (SLs) are a major type of biosurfactants that have gained immense interest in the healthcare industries because of their antiadhesive and anti-biofilm properties. Sophorolipids may therefore prove to be attractive substances that can be used in biomedical applications as adjuvant to other antibiotics against some infections through growth inhibition and/or biofilm disruption.

[Source](https://doi.org/10.1007/s00284-024-03892-6)

10.1007/s00284-024-03892-6

---

## Microbiome-based treatment helps ease severe malnutrition

[Source](https://doi.org/10.1126/science.adt5693)

---

## A multivalent mRNA-LNP vaccine protects against <em>Clostridioides difficile </em>infection

[Source](https://doi.org/10.1126/science.adn4955)

---

## Safety and Tolerability of CP101, a full spectrum, oral microbiome therapeutic for the prevention of recurrent <em>C. difficile </em>infection: A Phase 2 Randomized Controlled Trial

[Source](https://doi.org/10.1053/j.gastro.2024.09.030)

---

## Recruitment of complete crAss-like phage genomes reveals their presence in chicken viromes, few human-specific phages, and lack of universal detection.
 03.10.2024, _MD Ramos-Barbero, C Gómez-Gómez, G Vique, L Sala-Comorera, L Rodríguez-Rubio, M Muniesa_


The order Crassvirales, which includes the prototypical crAssphage (p-crAssphage), is predominantly associated with humans, rendering it the most abundant and widely distributed group of DNA phages in the human gut. The reported human specificity and wide global distribution of p-crAssphage makes it a promising human fecal marker. However, the specificity for the human gut as well as the geographical distribution around the globe of other members of the order Crassvirales remains unknown. To determine this, a recruitment analysis using 91 complete, non-redundant genomes of crAss-like phages in human and animal viromes revealed that only 13 crAss-like phages among the 91 phages analyzed were highly specific to humans, and p-crAssphage was not in this group. Investigations to elucidate whether any characteristic of the phages was responsible for their prevalence in humans showed that the 13 human crAss-like phages do not share a core genome. Phylogenomic analysis placed them in three independent families, indicating that within the Crassvirales group, human specificity is likely not a feature of a common ancestor but rather was introduced on separate/independent occasions in their evolutionary history. The 13 human crAss-like phages showed variable geographical distribution across human metagenomes worldwide, with some being more prevalent in certain countries than in others, but none being universally identified. The varied geographical distribution and the absence of a phylogenetic relationship among the human crAss-like phages are attributed to the emergence and dissemination of their bacterial host, the symbiotic human strains of Bacteroides, across various human populations occupying diverse ecological niches worldwide.

[Source](https://doi.org/10.1093/ismejo/wrae192)

10.1093/ismejo/wrae192

---

## Structural and functional differences of gut microbiota in <em>Pomacea canaliculata</em> from different geographical locations and habitats

[Source](https://doi.org/10.1002/ece3.70283)

---

## Rhizosphere microbial community structure in high-producing, low-input switchgrass families.
 05.10.2024, _C Stonoha-Arther, K Panke-Buisse, AJ Duff, A Molodchenko, MD Casler_


_Panicum, Rhizosphere, Microbiota, Nitrogen, Soil Microbiology, Biomass, RNA, Ribosomal, 16S, Fertilizers_
Switchgrass (Panicum virgatum L.) is a native, low-input North American perennial crop primarily grown for bioenergy, livestock forage, and industrial fiber. To achieve no-input switchgrass production that meets biomass needs, several switchgrass genotypes have been identified that have a low or negative response to nitrogen fertilizer, i.e., the biomass accumulation with added nitrogen is less than or equal to that when grown without nitrogen. In order to improve the viability of low-input switchgrass production, a more detailed understanding of the biogeochemical mechanisms active in these select genotypes is needed. 16S and ITS amplicon sequencing and qPCR of key functional genes were applied to switchgrass rhizospheres to elucidate microbial community structure in high-producing, no-input switchgrass families. Rhizosphere microbial community structure differed strongly between sites, and nitrogen responsiveness.

[Source](https://doi.org/10.1371/journal.pone.0308753)

10.1371/journal.pone.0308753

---

## Dynamic alterations and ecological implications of rice rhizosphere bacterial communities induced by an insect-transmitted reovirus across space and time.
 06.10.2024, _N Wu, W Shi, L Zhang, H Wang, W Liu, Y Ren, X Li, Z Gao, X Wang_


_Oryza, Rhizosphere, Reoviridae, Plant Diseases, Bacteria, Soil Microbiology, Microbiota, Animals, RNA, Ribosomal, 16S, Plant Roots, Insecta, Plant Viruses_
Cereal diseases caused by insect-transmitted viruses are challenging to forecast and control because of their intermittent outbreak patterns, which are usually attributed to increased population densities of vector insects due to cereal crop rotations and indiscriminate use of pesticides, and lack of resistance in commercial varieties. Root microbiomes are known to significantly affect plant health, but there are significant knowledge gaps concerning epidemics of cereal virus diseases at the microbiome-wide scale under a variety of environmental and biological factors.

[Source](https://doi.org/10.1186/s40168-024-01910-0)

10.1186/s40168-024-01910-0

---

## Microbiome-induced reprogramming in post-transcriptional landscape using nanopore direct RNA sequencing.
 04.10.2024, _Z Xu, X Zheng, J Fan, Y Jiao, S Huang, Y Xie, S Xu, Y Lu, A Liu, R Liu, Y Yang, GZ Luo, T Pan, X Wang_


It has been widely recognized that the microbiota has the capacity to shape host gene expression and physiological functions. However, there remains a paucity of comprehensive study revealing the host transcriptional landscape regulated by the microbiota. Here, we comprehensively examined mRNA landscapes in mouse tissues (brain and cecum) from specific-pathogen-free and germ-free mice using nanopore direct RNA sequencing. Our results show that the microbiome has global influence on a host's RNA modifications (m6A, m5C, Ψ), isoform generation, poly(A) tail length, and transcript abundance in both brain and cecum tissues. Moreover, the microbiome exerts tissue-specific effects on various post-transcriptional regulatory processes. In addition, the microbiome impacts the coordination of multiple RNA modifications in host brain and cecum tissues. In conclusion, we establish the relationship between microbial regulation and gene expression. Our results help the understanding of the mechanisms by which the microbiome reprograms host gene expression.

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(24)01149-5)

10.1016/j.celrep.2024.114798

---

## rRNA operon improves species-level classification of bacteria and microbial community analysis compared to 16S rRNA.
 04.10.2024, _S Won, S Cho, H Kim_


Precise identification of species is fundamental in microbial genomics and is crucial for understanding the microbial communities. While the 16S rRNA gene, particularly its V3-V4 regions, has been extensively employed for microbial identification, however has limitations in achieving species-level resolution. Advancements in long-read sequencing technologies have highlighted the rRNA operon as a more accurate marker for microbial classification and analysis than the 16S rRNA gene. This study aims to compare the accuracy of species classification and microbial community analysis using the rRNA operon versus the 16S rRNA gene. We evaluated the species classification accuracy of the rRNA operon,16S rRNA gene, and 16S rRNA V3-V4 regions using a BLAST-based method and a k-mer matching-based method with public data available from NCBI. We further performed simulations to model microbial community analysis. We accessed the performance using each marker in community composition estimation and differential abundance analysis. Our findings demonstrate that the rRNA operon offers an advantage over the 16S rRNA gene and its V3-V4 regions for species-level classification within the genus. When applied to microbial community analysis, the rRNA operon enables a more accurate determination of composition. Using the rRNA operon yielded more reliable results in differential abundance analysis as well.

[Source](https://doi.org/10.1128/spectrum.00931-24)

10.1128/spectrum.00931-24

---

## ASV vs OTUs clustering: Effects on alpha, beta, and gamma diversities in microbiome metabarcoding studies.
 05.10.2024, _A Fasolo, S Deb, P Stevanato, G Concheri, A Squartini_


_Microbiota, RNA, Ribosomal, 16S, Cluster Analysis, DNA Barcoding, Taxonomic, Bacteria, Biodiversity, DNA, Bacterial, Phylogeny_
In microbial community sequencing, involving bacterial ribosomal 16S rDNA or fungal ITS, the targeted genes are the basis for taxonomical assignment. The traditional bioinformatical procedure has for decades made use of a clustering protocol by which sequences are pooled into packages of shared percent identity, typically at 97%, to yield Operational Technical Units (OTUs). Progress in the data processing methods has however led to the possibility of minimizing technical sequencers errors, which were the main reason for the OTU choice, and to analyze instead the exact Amplicon Sequence Variants (ASV) which is a choice yielding much less agglomerated reads. We have tested the two procedures on the same 16S metabarcoded bacterial amplicons dataset encompassing a series of samples from 17 adjacent habitats, taken across a 700 meter-long transect of different ecological conditions unfolding in a gradient spanning from cropland, through meadows, forest and all successional transitions up to the seashore, within the same coastal area. This design allowed to scan a high biodiversity basin and to measure alpha, beta and gamma diversity of the area, to verify the effect of the bioinformatics on the same data as concerns the values of ten different ecological indexes and other parameters. Two levels of progressive OTUs clustering, (99% and 97%) were compared with the ASV data. The results showed that the OTUs clustering proportionally led to a marked underestimation of the ecological indicators values for species diversity and to a distorted behaviour of the dominance and evenness indexes with respect to the direct use of the ASV data. Multivariate ordination analyses resulted also sensitive in terms of tree topology and coherence. Overall, data support the view that reference-based OTU clustering carries several misleading disadvantageous biases, including the risk of missing novel taxa which are yet unreferenced in databases. Since its alternatives as de novo clustering have on the other hand drawbacks due to heavier computational demand and results comparability, especially for environmental studies which contain several yet uncharacterized species, the direct ASV based analysis, at least for prokaryotes, appears to warrant significand advantages in comparison to OTU clustering at every level of percent identity cutoff.

[Source](https://doi.org/10.1371/journal.pone.0309065)

10.1371/journal.pone.0309065

---

